
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Detection and functional assessment of structural variants using whole-genome re-sequencing data in Nellore cattle - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="152550B48AF423030550B4001CDA2D0F.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="scirep">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365053/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Scientific Reports">
<meta name="citation_title" content="Detection and functional assessment of structural variants using whole-genome re-sequencing data in Nellore cattle">
<meta name="citation_author" content="Natalia A Marín-Garzón">
<meta name="citation_author_institution" content="School of Agricultural and Veterinarian Sciences, São Paulo State University (UNESP), Via de Acesso Prof. Paulo Donato Castelane, Jaboticabal, 14884-900 SP Brazil">
<meta name="citation_author" content="Lucio F M Mota">
<meta name="citation_author_institution" content="School of Agricultural and Veterinarian Sciences, São Paulo State University (UNESP), Via de Acesso Prof. Paulo Donato Castelane, Jaboticabal, 14884-900 SP Brazil">
<meta name="citation_author" content="Giovana Vargas">
<meta name="citation_author_institution" content="School of Agricultural and Veterinarian Sciences, São Paulo State University (UNESP), Via de Acesso Prof. Paulo Donato Castelane, Jaboticabal, 14884-900 SP Brazil">
<meta name="citation_author" content="Leonardo M Arikawa">
<meta name="citation_author_institution" content="School of Agricultural and Veterinarian Sciences, São Paulo State University (UNESP), Via de Acesso Prof. Paulo Donato Castelane, Jaboticabal, 14884-900 SP Brazil">
<meta name="citation_author" content="Larissa F S Fonseca">
<meta name="citation_author_institution" content="School of Agricultural and Veterinarian Sciences, São Paulo State University (UNESP), Via de Acesso Prof. Paulo Donato Castelane, Jaboticabal, 14884-900 SP Brazil">
<meta name="citation_author" content="Gerardo A Fernandes Júnior">
<meta name="citation_author_institution" content="School of Agricultural and Veterinarian Sciences, São Paulo State University (UNESP), Via de Acesso Prof. Paulo Donato Castelane, Jaboticabal, 14884-900 SP Brazil">
<meta name="citation_author" content="Roberto Carvalheiro">
<meta name="citation_author_institution" content="CSIRO Agriculture and Food, Hobart, TAS 7000 Australia">
<meta name="citation_author" content="Lucia G Albuquerque">
<meta name="citation_author_institution" content="School of Agricultural and Veterinarian Sciences, São Paulo State University (UNESP), Via de Acesso Prof. Paulo Donato Castelane, Jaboticabal, 14884-900 SP Brazil">
<meta name="citation_author_institution" content="National Council for Science and Technological Development, Brasilia, 71605-001 DF Brazil">
<meta name="citation_publication_date" content="2025 Aug 19">
<meta name="citation_volume" content="15">
<meta name="citation_firstpage" content="30364">
<meta name="citation_doi" content="10.1038/s41598-025-14139-0">
<meta name="citation_pmid" content="40830176">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365053/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365053/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365053/pdf/41598_2025_Article_14139.pdf">
<meta name="description" content="Ongoing advances in genome sequencing technologies have enabled the identification of numerous structural variants (SVs) in livestock genomes, which are the main determinants of complex traits due to their impact on gene expression. Thus, this study ...">
<meta name="og:title" content="Detection and functional assessment of structural variants using whole-genome re-sequencing data in Nellore cattle">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Ongoing advances in genome sequencing technologies have enabled the identification of numerous structural variants (SVs) in livestock genomes, which are the main determinants of complex traits due to their impact on gene expression. Thus, this study ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365053/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12365053">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1038/s41598-025-14139-0"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/41598_2025_Article_14139.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12365053%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12365053/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12365053/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365053/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-scirep.jpg" alt="Scientific Reports logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Scientific Reports" title="Link to Scientific Reports" shape="default" href="http://www.nature.com/srep" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Sci Rep</button></div>. 2025 Aug 19;15:30364. doi: <a href="https://doi.org/10.1038/s41598-025-14139-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1038/s41598-025-14139-0</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Sci%20Rep%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sci%20Rep%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Sci%20Rep%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Sci%20Rep%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Detection and functional assessment of structural variants using whole-genome re-sequencing data in Nellore cattle</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mar%C3%ADn-Garz%C3%B3n%20NA%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Natalia A Marín-Garzón</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Natalia A Marín-Garzón</span></h3>
<div class="p">
<sup>1</sup>School of Agricultural and Veterinarian Sciences, São Paulo State University (UNESP), Via de Acesso Prof. Paulo Donato Castelane, Jaboticabal, 14884-900 SP Brazil </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mar%C3%ADn-Garz%C3%B3n%20NA%22%5BAuthor%5D" class="usa-link"><span class="name western">Natalia A Marín-Garzón</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mota%20LFM%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Lucio F M Mota</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Lucio F M Mota</span></h3>
<div class="p">
<sup>1</sup>School of Agricultural and Veterinarian Sciences, São Paulo State University (UNESP), Via de Acesso Prof. Paulo Donato Castelane, Jaboticabal, 14884-900 SP Brazil </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mota%20LFM%22%5BAuthor%5D" class="usa-link"><span class="name western">Lucio F M Mota</span></a>
</div>
</div>
<sup>1,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Vargas%20G%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Giovana Vargas</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Giovana Vargas</span></h3>
<div class="p">
<sup>1</sup>School of Agricultural and Veterinarian Sciences, São Paulo State University (UNESP), Via de Acesso Prof. Paulo Donato Castelane, Jaboticabal, 14884-900 SP Brazil </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Vargas%20G%22%5BAuthor%5D" class="usa-link"><span class="name western">Giovana Vargas</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Arikawa%20LM%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Leonardo M Arikawa</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Leonardo M Arikawa</span></h3>
<div class="p">
<sup>1</sup>School of Agricultural and Veterinarian Sciences, São Paulo State University (UNESP), Via de Acesso Prof. Paulo Donato Castelane, Jaboticabal, 14884-900 SP Brazil </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Arikawa%20LM%22%5BAuthor%5D" class="usa-link"><span class="name western">Leonardo M Arikawa</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Fonseca%20LFS%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Larissa F S Fonseca</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Larissa F S Fonseca</span></h3>
<div class="p">
<sup>1</sup>School of Agricultural and Veterinarian Sciences, São Paulo State University (UNESP), Via de Acesso Prof. Paulo Donato Castelane, Jaboticabal, 14884-900 SP Brazil </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Fonseca%20LFS%22%5BAuthor%5D" class="usa-link"><span class="name western">Larissa F S Fonseca</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Fernandes%20J%C3%BAnior%20GA%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Gerardo A Fernandes Júnior</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Gerardo A Fernandes Júnior</span></h3>
<div class="p">
<sup>1</sup>School of Agricultural and Veterinarian Sciences, São Paulo State University (UNESP), Via de Acesso Prof. Paulo Donato Castelane, Jaboticabal, 14884-900 SP Brazil </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Fernandes%20J%C3%BAnior%20GA%22%5BAuthor%5D" class="usa-link"><span class="name western">Gerardo A Fernandes Júnior</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Carvalheiro%20R%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Roberto Carvalheiro</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Roberto Carvalheiro</span></h3>
<div class="p">
<sup>2</sup>CSIRO Agriculture and Food, Hobart, TAS 7000 Australia </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Carvalheiro%20R%22%5BAuthor%5D" class="usa-link"><span class="name western">Roberto Carvalheiro</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Albuquerque%20LG%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Lucia G Albuquerque</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Lucia G Albuquerque</span></h3>
<div class="p">
<sup>1</sup>School of Agricultural and Veterinarian Sciences, São Paulo State University (UNESP), Via de Acesso Prof. Paulo Donato Castelane, Jaboticabal, 14884-900 SP Brazil </div>
<div class="p">
<sup>3</sup>National Council for Science and Technological Development, Brasilia, 71605-001 DF Brazil </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Albuquerque%20LG%22%5BAuthor%5D" class="usa-link"><span class="name western">Lucia G Albuquerque</span></a>
</div>
</div>
<sup>1,</sup><sup>3</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>School of Agricultural and Veterinarian Sciences, São Paulo State University (UNESP), Via de Acesso Prof. Paulo Donato Castelane, Jaboticabal, 14884-900 SP Brazil </div>
<div id="Aff2">
<sup>2</sup>CSIRO Agriculture and Food, Hobart, TAS 7000 Australia </div>
<div id="Aff3">
<sup>3</sup>National Council for Science and Technological Development, Brasilia, 71605-001 DF Brazil </div>
<div class="author-notes p"><div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div></div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 Jan 17; Accepted 2025 Jul 29; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by-nc-nd/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12365053  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40830176/" class="usa-link">40830176</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<p id="Par1">Ongoing advances in genome sequencing technologies have enabled the identification of numerous structural variants (SVs) in livestock genomes, which are the main determinants of complex traits due to their impact on gene expression. Thus, this study aimed to detect structural variants using whole genome re-sequencing (WGS) data and identify positional candidate genes and quantitative trait loci (QTL) overlapping the most frequent SV regions in Nellore cattle. The WGS from 151 representative Nellore bulls were analyzed to conduct genome-wide structural variation calling and to identify common SV regions. Gene and QTL information surrounding the most frequent SV regions was identified using the Ensembl Genes and Cattle QTL database. The identified genes were functionally classified for biological mechanisms and pathways (Gene Ontology - GO) using the panther database. A total of 215,031 SVs were identified, with most of them being copy number variants (CNV) (183,032 deletions and 14,013 duplications) and 17,986 inversions (INV). These SVs cover, on average, 4.81% of the autosomal genome. Furthermore, we found 3,752 non-redundant SV regions that are frequent in at least 5% of the bulls. These SV regions mainly correspond to CNV regions (97%) and inversion regions (3%). In total, all SV regions cover 13.13% of the total autosomal genome, with 11.4% attributed to CNV regions and 1.7% to inversion regions. Moreover, we found that 532 SV regions were common in more than 50% of the bulls evaluated and overlapped 130 QTL previously associated with economically important traits related to exterior, health, meat and carcass, milk, production, and reproduction. A total of 1,164 positional candidate genes were identified, with 204 SVRs overlapping these genes. These genes are significantly overrepresented in GO terms related to biological processes (BP), molecular functions (MF), and biochemical pathways, playing an essential role in environmental adaptation mechanisms and feed efficiency indicator traits. Our results suggest that genes surrounding SV regions play key biological functions essential to thermotolerance, immunity, metabolism, tissue integrity, and environmental adaptation in tropical regions.</p>
<section id="sec1"><h3 class="pmc_sec_title">Supplementary Information</h3>
<p>The online version contains supplementary material available at 10.1038/s41598-025-14139-0.</p></section><section id="kwd-group1" lang="en" class="kwd-group"><p><strong>Keywords:</strong> Copy number variation, Gains and losses of DNA fragments, Mobile elements, Next-generation sequencing, Zebu cattle</p></section><section id="kwd-group2" class="kwd-group"><p><strong>Subject terms:</strong> Animal breeding, Structural variation, Genetic markers</p></section></section><section id="Sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p id="Par2">The widespread developments of molecular techniques and bioinformatic algorithms have enabled the unraveling of genetic mechanisms driving differences between individuals, which are mainly attributed to single-nucleotide variants (SNVs), small insertions and deletions (indels; &lt; 50 bp), and structural variations (SVs)<sup><a href="#CR1" class="usa-link" aria-describedby="CR1">1</a>–<a href="#CR3" class="usa-link" aria-describedby="CR3">3</a></sup>. Structural variants are generally defined as inherited variations in DNA sequences larger than 50 bp that exhibit changes in the copy number, orientation, and/or chromosomal location in relation to the reference genome<sup><a href="#CR4" class="usa-link" aria-describedby="CR4">4</a>,<a href="#CR5" class="usa-link" aria-describedby="CR5">5</a></sup>. The balanced SV types correspond to the mobile elements (inversions and translocations), which cause rearrangements of DNA sequences<sup><a href="#CR1" class="usa-link" aria-describedby="CR1">1</a>,<a href="#CR5" class="usa-link" aria-describedby="CR5">5</a></sup>. On the other hand, small DNA segments, such as insertions and deletions (indels) and copy number variants (CNVs), comprise the unbalanced SVs in which gains or losses of DNA segments are observed. CNVs are the most studied SV types in livestock and encompass duplication and deletion events ranging in size from 1 Kb to 5 Mb<sup><a href="#CR3" class="usa-link" aria-describedby="CR3">3</a>,<a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>,<a href="#CR7" class="usa-link" aria-describedby="CR7">7</a></sup>.</p>
<p id="Par3">Compared to SNPs, SVs affect larger fractions of the genome and account for the greatest amount of total polymorphic content among individual genomes<sup><a href="#CR8" class="usa-link" aria-describedby="CR8">8</a></sup>. SVs may have potentially greater effects than SNPs, from modification of genes to disruptions in the regulation of gene expression, leading to recessive disorders<sup><a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>,<a href="#CR9" class="usa-link" aria-describedby="CR9">9</a></sup>. A variety of based platforms, such as comparative genomic hybridization and SNP arrays, have been commonly used for SVs-wide detection in livestock<sup><a href="#CR10" class="usa-link" aria-describedby="CR10">10</a></sup>. However, array-based approaches lack genome coverage, especially in segmental duplication regions, which are known for being hotspots for CNV formation<sup><a href="#CR11" class="usa-link" aria-describedby="CR11">11</a></sup>. The advantages of detecting SV using whole genome sequencing data (WGS) include higher coverage and resolution, more accurate prediction of copy numbers, more precise detection of breakpoints, and higher power to identify novel SVs<sup><a href="#CR11" class="usa-link" aria-describedby="CR11">11</a></sup>. In cattle, the study of structural variations has reached increasing attention because of their association with economically important phenotypes such as fatty acid profile<sup><a href="#CR12" class="usa-link" aria-describedby="CR12">12</a></sup>meat tenderness<sup><a href="#CR13" class="usa-link" aria-describedby="CR13">13</a></sup>feed conversion ratio<sup><a href="#CR14" class="usa-link" aria-describedby="CR14">14</a></sup>growth traits<sup><a href="#CR15" class="usa-link" aria-describedby="CR15">15</a></sup>milk production traits<sup><a href="#CR16" class="usa-link" aria-describedby="CR16">16</a></sup>and milk somatic cell score<sup><a href="#CR17" class="usa-link" aria-describedby="CR17">17</a></sup>.</p>
<p id="Par4">Although WGS has become more feasible in livestock, most studies regarding structural variant-wide detection are still being conducted using SNP array-based approaches. Few published studies explored the structural diversity using whole genome sequencing data, all with less than 100 animals<sup><a href="#CR18" class="usa-link" aria-describedby="CR18">18</a>–<a href="#CR24" class="usa-link" aria-describedby="CR24">24</a></sup>. Furthermore, most have been focused on the assessment of copy number variants (CNVs) rather than the other SV types. The detection of balanced and unbalanced structural variants using whole genome sequencing data may contribute to the unraveling of the molecular mechanisms underlying genetic diversity, breed adaptation, and evolution, especially in Nellore cattle, which have not been well elucidated so far. Therefore, the aims of this study were: (i) to detect structural variants using whole genome re-sequencing data of 151 representative Nellore bulls by combining calling algorithms; (ii) to discover non-redundant and highly frequent structural variant regions (SVR) in the analyzed bulls; (iii) to search for positional candidate genes and quantitative trait loci (QTL) overlapping the most frequent SVR in the population; and (iv) to assess the functional impact of positional candidate genes overlapping SVR through enriched gene ontology terms (GO terms) related to biological process (BP), molecular function (MF), cellular component (CC), and biochemical pathways.</p></section><section id="Sec2"><h2 class="pmc_sec_title">Materials and methods</h2>
<section id="Sec3"><h3 class="pmc_sec_title">DNA samples and re-sequencing</h3>
<p id="Par6">A total of 151 Nellore bulls comprised the analyzed population for the current study. The ancestors were selected based on their contributions to the genetic diversity of the Brazilian Nellore population. For that, pedigree information of 2,688,124 related individuals, with a total of 9,811 sires, from which 6,040 were founders, and 915,371 dams were used. The selected bulls for DNA re-sequencing were the ones less related to the others, those having the higher number of progenies and that have greater contributions to the genotyped population, which consists of more than 60,000 genotyped animals from five Brazilian Nellore breeding programs: DeltaGen (<a href="http://deltagen.com.br/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://deltagen.com.br/</a>), Nelore Qualitas (<a href="https://qualitas.agr.br/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://qualitas.agr.br/</a>), Cia de Melhoramento (<a href="https://www.ciademelhoramento.com.br" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ciademelhoramento.com.br</a>), PAINT (<a href="https://www.paintmga.com.br/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.paintmga.com.br/</a>), and Centro de Pesquisa de Bovinos de Corte - Instituto de Zootecnia (<a href="https://iz.agricultura.sp.gov.br/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://iz.agricultura.sp.gov.br/</a>). For further information regarding the selected ancestors, see Fernandes Junior et al.<sup><a href="#CR2" class="usa-link" aria-describedby="CR2">2</a></sup>.</p>
<p id="Par7">Genomic DNA was extracted from semen samples (100 µL) using the DNeasy Blood and Tissue DNA Extraction Kit (Qiagen, Valencia, CA, USA). Each purified DNA sample was quality-controlled by agarose gel electrophoresis (1%). Extracted DNA was quantified based on the absorbance using the NanoDrop 1000 Spectrophotometer (Thermo Fisher Scientific, Santa Clara, CA, USA, 2007).</p>
<p id="Par8">The whole-genome re-sequencing of representative bulls was performed using Illumina sequencing by synthesis technology. From all DNA samples, 52 were re-sequenced using the Hiseq X Ten and the other 99 with the NovaSeq 6000 System platform (Illumina Inc., San Diego, USA). DNA libraries were constructed with end-linked adapters, and the resulting libraries were re-sequenced by paired-end sequencing (sequencing of both ends of DNA fragments), which produced an average of 360 million reads per bull with a length of 150 Bp (2 × 150) and an average genome coverage of 14.5x, ranging from 7.8 to 26.3x. Poor quality reads were trimmed using SeqClean software (<a href="https://hpc.ilri.cgiar.org/seqclean/seqclean" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://hpc.ilri.cgiar.org/seqclean/seqclean</a>) with default parameters. The SAMtools software v. 1.22<sup><a href="#CR25" class="usa-link" aria-describedby="CR25">25</a></sup> (<a href="https://www.htslib.org/download/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.htslib.org/download/</a>) was used to index reads, and the alignment to the bovine reference genome ARS-UCD1.2 (<a href="http://www.ncbi.nlm.nih.gov/genome/?term=bos+taurus" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://www.ncbi.nlm.nih.gov/genome/?term=bos+taurus</a>) was performed using the software HISAT2 v. 2.2.0<sup><a href="#CR26" class="usa-link" aria-describedby="CR26">26</a></sup> (<a href="https://daehwankimlab.github.io/hisat2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://daehwankimlab.github.io/hisat2</a>), in which 97.9% of reads were successfully paired. Further details regarding the re-sequencing and alignment can be found in Fernandes Junior et al.<sup><a href="#CR2" class="usa-link" aria-describedby="CR2">2</a></sup> and Fernandes Junior et al.<sup><a href="#CR27" class="usa-link" aria-describedby="CR27">27</a></sup>.</p></section><section id="Sec4"><h3 class="pmc_sec_title">Detection of structural variants</h3>
<p id="Par9">Detection of structural variants (SV) was conducted by adapting the workflow proposed by Letaief et al.<sup><a href="#CR28" class="usa-link" aria-describedby="CR28">28</a></sup> in French dairy and beef cattle (Fig. <a href="#Fig1" class="usa-link">1</a>). SV calling for each sample was performed using Parliament2<sup><a href="#CR29" class="usa-link" aria-describedby="CR29">29</a></sup>, a software that integrates multiple bioinformatics tools for high-quality structural variation events calling. The consensus strategy of Parliament2 (<a href="https://github.com/fritzsedlazeck/parliament2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/fritzsedlazeck/parliament2</a>) provides information about the size, type, location, and calling method for each SV event. We used BAM and BAI files containing the whole genome nucleotide sequences of 151 Nellore bulls, as well as BAM and BAI files containing the bovine reference genome sequence (ARS-UCD1.2). For this dataset, we executed Parliament2, setting the default parameters for each sample through a single Docker image across the DNAnexus platform. A total of three bioinformatic tools were integrated through Parliament2: CNVnator v. 0.4.1<sup><a href="#CR30" class="usa-link" aria-describedby="CR30">30</a></sup> (<a href="https://github.com/abyzovlab/CNVnator" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/abyzovlab/CNVnator</a>), Lumpy v. 0.3.1<sup><a href="#CR31" class="usa-link" aria-describedby="CR31">31</a></sup> (<a href="https://github.com/arq5x/lumpy-sv" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/arq5x/lumpy-sv</a>) and Delly v. 1.2.6 <sup><a href="#CR32" class="usa-link" aria-describedby="CR32">32</a></sup> (<a href="https://github.com/dellytools/delly" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/dellytools/delly</a>). CNVnator, Lumpy, and Delly are bioinformatics tools designed for detecting structural variants (SVs) from whole-genome sequencing (WGS) data. CNVnator identifies copy number variations (CNVs) by analyzing read depth. Lumpy utilizes a hybrid approach that combines paired-end, split-read, and read-depth signals to detect various types of structural variants, including deletions, duplications, inversions, and translocations. Delly, on the other hand, employs paired-end and split-read mapping techniques to accurately identify a wide range of structural variants, including complex rearrangements.</p>
<figure class="fig xbox font-sm" id="Fig1"><h4 class="obj_head">Fig. 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365053_41598_2025_14139_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5894/12365053/c186ffd3da27/41598_2025_14139_Fig1_HTML.jpg" loading="lazy" id="d33e450" height="764" width="661" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Flowchart of pipeline used to detect structural variants (SV). <em>CNV</em> copy number variants, <em>RO</em> reciprocal overlap, <em>DEL</em> deletions, <em>DUP</em> duplications, <em>INV</em> inversions. First, SV was detected separately by each tool within each sample through the integration software Parliament2. Second, filtering was applied at the software level, following the suggested criteria for each tool. Only SVs that passed all filters were retained and compared within the sample to retain common CNVs and INVs detected by the different tools. CNVs should share at least 90% of their sequence between the three tools, and INVs should share at least 90% of their sequence between Delly and Lumpy calls. Own work.</p></figcaption></figure><p id="Par11">After computing Parliament2, we obtained VCF files containing the SV called from each bioinformatic approach for each of the 151 Nellore bulls, consisting of a total of 6 VCF files, described as follows: 2 VCF files containing CNVnator and Lumpy detected variants and a total of 4 VCF files containing each Delly detected variant type (duplications, deletions, insertions, and inversions).</p></section><section id="Sec5"><h3 class="pmc_sec_title">Filtering of called structural variants</h3>
<p id="Par12">The filtering of detected SVs was performed at individual and population levels in order to remove unreliable raw SVs and reduce the false discovery rate. For the following analyses, different sets of utilities integrated by BCFtools<sup><a href="#CR33" class="usa-link" aria-describedby="CR33">33</a></sup> and Bedtools<sup><a href="#CR34" class="usa-link" aria-describedby="CR34">34</a></sup> were used. All called SVs were first filtered at the bioinformatic tool level, following specific criteria as suggested for each SV caller.</p>
<ol class="list" style="list-style-type:decimal">
<li><p id="Par14">CNVnator: we only retained the called structural variants that were significant for the t-statistical test (p-value &lt; 0.05), for which the null hypothesis statement is that the mean read signal depth in an SV region is the same as the mean signal depth in the sample.</p></li>
<li><p id="Par15">Delly: only called SV that passed the quality filters (i.e., flagged PASS) and supported for more than 3 paired-end events (PE support &gt; 3) were kept. After filtering, the VCF containing the remaining DEL, DUP, and INS variants was joined using the option <em>concat</em> of BCFtools to obtain a single VCF containing the copy number variants (CNVs), while the VCF containing filtered INV variants was maintained separated.</p></li>
<li><p id="Par16">Lumpy: SVs with a read count supported by more than 1 split-read event or more than 3 paired-end events (SR &gt; 1 or PE &gt; 3) were maintained.</p></li>
</ol>
<p id="Par17">The SVs that passed the described criteria were compared at the individual level (i.e., within the same sample) to identify a high-confidence set of SVs for each bull (composed only of SVs common to all bioinformatic tools). As each of the three callers used has a different power to detect a specific SV type, such as CNVnator, which only detects CNVs, the within-sample analysis was performed for each SV type separately:</p>
<ol class="list" style="list-style-type:decimal">
<li><p id="Par19">Identification of high-confidence CNVs: the consensus files containing the filtered variants from CNVnator and Lumpy, and the concatenated and filtered file containing duplications, deletions, and insertions from Delly, were compared against each other using the option <em>intersect</em> of Bedtools by fitting a minimal reciprocal overlap threshold of 90%.</p></li>
<li><p id="Par20">Identification of high-confidence INVs: the consensus-filtered file containing the SVs detected from Lumpy and the one containing the inversions detected from Delly were compared against each other using the option <em>intersect</em> of Bedtools by fitting a minimal reciprocal overlap threshold of 90%.</p></li>
</ol>
<p id="Par21">The establishment of a reciprocal overlap threshold of 90% means that if at least 90% of the sequence of a certain SV detected by one caller is the same as the sequence of an SV predicted by the others, that SV must be deemed as a high-confidence SV for a certain bull.</p>
<p id="Par22">After executing the within-sample filtering, we obtained two VCF files for each bull, one containing the high-confidence inversions (INVs) and the other one containing the high-confidence copy number variants (CNVs). All VCF files containing each type of high-confidence SVs were joined using the option <em>concat</em> of BCFtools to obtain two files containing the high-confidence CNVs and INVs, respectively, for all bulls. CNVs and INVs were then submitted to additional filtering, where SVs located in sexual chromosomes and with a length shorter than 1 Kb and higher than 5 Mb were removed. The remaining CNVs and INVs were considered for downstream analysis.</p></section><section id="Sec6"><h3 class="pmc_sec_title">Identification of structural variant regions (SVRs)</h3>
<p id="Par23">The joining of structural variants into regions is required for population-wide analysis, as it allows the detection of DNA segments having different structural variation events<sup><a href="#CR35" class="usa-link" aria-describedby="CR35">35</a></sup>. In the present study, the structural variant regions (SVRs) were built by collating, separately, the two types of high-confidence SVs (CNVs and INVs) along all bulls that overlapped in at least 1 bp. For this, we used the utility <em>merge</em> of Bedtools (<a href="https://github.com/arq5x/bedtools2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/arq5x/bedtools2</a>) fitting operations such as count and collapse, which allowed the identification of samples sharing a certain region and the occurring SV event. Regions that were not shared by at least 5% of the bulls (<em>N</em> = 7) were removed. As a result, we obtained two sets, one containing the regions of copy number variants (CNVRs) and the other one with the regions of inversions (INVRs), which were then joined in a single file using the option concat of BCFtools, to facilitate the descriptive analysis of SVRs (Fig. <a href="#Fig2" class="usa-link">2</a>).</p>
<figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365053_41598_2025_14139_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5894/12365053/192b46ca942e/41598_2025_14139_Fig2_HTML.jpg" loading="lazy" id="d33e572" height="354" width="661" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>A flowchart of the pipeline is used to detect regions of structural variant (SVR). <em>CNV</em> copy number variants, <em>RO</em> reciprocal overlap, <em>DEL</em> deletions, <em>DUP</em> duplications, <em>INV</em> inversions, <em>SVR</em> structural variant regions, <em>CNVR</em> regions of copy number variants, <em>INVR</em> regions of inversions. All sets of detected high-confidence CNVs and INVs were merged across all samples with a minimal reciprocal overlap of 1 Bp. Own work.</p></figcaption></figure></section><section id="Sec7"><h3 class="pmc_sec_title">Statistical analysis and graphical visualization of SVs and SVRs</h3>
<p id="Par25">All reported descriptive statistics and plots about the results were calculated using functions and utilities of the R package tidyverse<sup><a href="#CR36" class="usa-link" aria-describedby="CR36">36</a></sup> through the Integrated Development Environment RStudio. The association of different metrics, such as the number, distribution, and (average) length of the predicted SVs and SVRs, with chromosome size and genome coverage rate, was assessed through the pairwise Pearson correlation method using the R package Performance Analytics<sup><a href="#CR37" class="usa-link" aria-describedby="CR37">37</a></sup>.</p></section><section id="Sec8"><h3 class="pmc_sec_title">Functional analysis of highly frequent SVR</h3>
<p id="Par26">The functional implications of predicted SVR were assessed by searching genes and QTLs overlapping the most frequent SVR in the 151 bulls (SVR present in more than 50% of bulls, <em>N</em> &gt; 75) using the GALLO package of R<sup><a href="#CR38" class="usa-link" aria-describedby="CR38">38</a></sup>. Genes and QTLs were retrieved from the Ensembl Genes database<sup><a href="#CR39" class="usa-link" aria-describedby="CR39">39</a></sup> and cattle database from Animal QTLdb<sup><a href="#CR40" class="usa-link" aria-describedby="CR40">40</a></sup>respectively. Only significant QTLs (p-value &lt; 0.05) and protein-coding genes overlapping the most frequent SVR were reported in this study.</p>
<p id="Par27">Functional enrichment analysis for positional candidate genes overlapping the most frequent SVR was performed through PANTHER 17.0 software (<a href="https://pantherdb.org/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://pantherdb.org/</a>). PANTHER statistical overrepresentation test was conducted using, as an analyzed list, the set of positional candidate genes harbored by the most frequent SVR (a total of 1,164 genes) and, as a reference list, the <em>Bos taurus</em> genes contained in PANTHER databases. The annotation data sets were PANTHER GO-slim Molecular Function, PANTHER GO-slim Biological Process, PANTHER GO-slim Cellular Component, PANTHER Pathway, and Reactome Pathway. A Binomial test was applied to test whether the genes overlapping the SVR were enriched for the gene ontology categories and biochemical pathways. This assesses the probability of observing a certain number of genes related to a specific biological mechanism within the found gene set, given all genes in the <em>Bos taurus</em> reference genome. As different biological functions were tested at the same time, the Bonferroni correction method for multiple testing was used, which allows for reducing the probability of false positive associations of genes and gene ontologies and/or pathways. Only genes with significant associations (p-value &lt; 0.05) with gene ontologies (GO) or pathways were reported. The Tidyverse (<a href="https://www.tidyverse.org/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.tidyverse.org/</a>) and ggplot2 (<a href="https://ggplot2.tidyverse.org/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://ggplot2.tidyverse.org/</a>) R packages were used to graphically represent the QTL type and trait ID of significant QTLs, as well as the significantly enriched GO and/or pathways, through pie charts and bar plots.</p></section></section><section id="Sec9"><h2 class="pmc_sec_title">Results and discussion</h2>
<section id="Sec10"><h3 class="pmc_sec_title">Genome-wide SV detection</h3>
<section id="Sec11"><h4 class="pmc_sec_title">Number of high-confidence SVs</h4>
<p id="Par28">A total of 215,031 high-confidence structural variants were obtained for the analyzed bulls that passed all the described filtering criteria. Statistics summary for the number and length of high-confidence SVs, including CNVs (deletions and duplications) and inversions, are presented in Table <a href="#Tab1" class="usa-link">1</a>. Although insertions were detected by the SV callers, after applying filtering criteria and executing the reciprocal overlapping into the CNV set, no insertions remained, as they were detected with low quality and corresponded to small insertions (average length of 1 bp), which are not considered as structural variation.</p>
<section class="tw xbox font-sm" id="Tab1"><h5 class="obj_head">Table 1.</h5>
<div class="caption p"><p>Statistics summary for the number and length of detected high-confidence svs.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead>
<tr>
<th align="left" rowspan="2" colspan="1">SV type</th>
<th align="left" rowspan="2" colspan="1">
<em>N</em>
</th>
<th align="left" colspan="4" rowspan="1">Length (kb)</th>
</tr>
<tr>
<th align="left" colspan="1" rowspan="1">Mean (SD)</th>
<th align="left" colspan="1" rowspan="1">Median</th>
<th align="left" colspan="1" rowspan="1">Max</th>
<th align="left" colspan="1" rowspan="1">Total</th>
</tr>
</thead>
<tbody>
<tr><td align="left" colspan="6" rowspan="1">CNV</td></tr>
<tr>
<td align="left" colspan="1" rowspan="1"> DEL</td>
<td align="center" colspan="1" rowspan="1">183,032</td>
<td align="center" colspan="1" rowspan="1">22.68 (167.31)</td>
<td align="center" colspan="1" rowspan="1">1.91</td>
<td align="center" colspan="1" rowspan="1">4916.08</td>
<td align="center" colspan="1" rowspan="1">4,150,681</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> DUP</td>
<td align="center" colspan="1" rowspan="1">14,013</td>
<td align="center" colspan="1" rowspan="1">250.61 (560.12)</td>
<td align="center" colspan="1" rowspan="1">63.98</td>
<td align="center" colspan="1" rowspan="1">4906.46</td>
<td align="center" colspan="1" rowspan="1">3,511,807</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> INV</td>
<td align="center" colspan="1" rowspan="1">17,986</td>
<td align="center" colspan="1" rowspan="1">613.03 (851.34)</td>
<td align="center" colspan="1" rowspan="1">167.45</td>
<td align="center" colspan="1" rowspan="1">4989.55</td>
<td align="center" colspan="1" rowspan="1">11,025,935</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> Total</td>
<td align="center" colspan="1" rowspan="1">215,031</td>
<td align="center" colspan="1" rowspan="1">86.91 (365.11)</td>
<td align="center" colspan="1" rowspan="1">2.21</td>
<td align="center" colspan="1" rowspan="1">4989.55</td>
<td align="center" colspan="1" rowspan="1">18,688,423</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="_fn_p33"><p><em>SV</em> structural variant, <em>CNV</em> copy number variants, <em>DEL</em> deletions, <em>DUP</em> duplications, <em>INV</em> inversions, <em>N</em> number of SVs, <em>SD</em> standard deviation, <em>Max</em> maximum, <em>Total</em> sum of all SV lengths, <em>kb</em> kilo base pairs.</p></div></div></section><p id="Par30">In this study, SVs were detected using three complementary callers, such as CNVnator (based on read-depth), Delly (based on paired-end mapping), and Lumpy (which integrates both paired-end and split-read signals). Each tool’s output was first filtered using tool-specific quality thresholds. Then, for each SV type, only the variants that were shared across all three callers within the same individual, based on a 90% reciprocal overlap threshold, were retained as high-confidence SVs. Although this approach did not allow quantifying partial overlaps between specific pairs of tools (e.g., CNVnator vs. Delly), it ensured a highly stringent consensus set, minimizing the false discovery rate by focusing exclusively on SVs consistently detected by all methods. This strategy favored precision over sensitivity and reflected the complementary nature of the tools used, each with different strengths depending on SV type and detection principle. As such, the 215,031 high-confidence structural variants reported represent a conservative yet robust subset for downstream analyses (Supplementary Table <a href="#MOESM1" class="usa-link">S1</a>).</p>
<p id="Par31">From the total number of detected SVs, 197,045 corresponded to CNVs, and the number of deletions was 13.06 times higher than the number of duplications (Table <a href="#Tab1" class="usa-link">1</a>). This result is similar to that reported by Silva et al.<sup><a href="#CR22" class="usa-link" aria-describedby="CR22">22</a></sup>who detected 11.8 times more deletions than duplications in Nellore cattle using whole genome sequencing data. It also agrees with the findings of Santana et al.<sup><a href="#CR14" class="usa-link" aria-describedby="CR14">14</a></sup>in which the number of loss events was higher than the number of gain events for the detected CNVs from SNP data in Nellore cattle, although the ratio presented here is higher than reported by the latter authors (1.18 times more losses than gains). Similarly, our result is congruent to the ratio reported by Braga et al.<sup><a href="#CR20" class="usa-link" aria-describedby="CR20">20</a></sup> in Gyr cattle (1.96 times more deletions than duplications), who detected CNVs from WGS using CNVnator and Delly software, and agrees with the findings of Hu et al.<sup><a href="#CR41" class="usa-link" aria-describedby="CR41">41</a></sup> in different cattle breeds, including Nellore, where the number of deletions was higher than the number of duplications. Interestingly, the authors pointed out that the proportion of observed deletions was higher in the <em>Bos indicus</em> than in <em>Bos taurus</em> animals. In contrast, the ratio of deletions over duplications observed in this study diverges from what was reported by Peripolli et al.<sup><a href="#CR18" class="usa-link" aria-describedby="CR18">18</a></sup>who used whole genome sequencing data from three Brazilian locally adapted taurine breeds, where the number of observed duplications was 1.85 times higher than the observed deletions.</p>
<p id="Par32">The remarkably elevated number of deletions may be related to mechanisms causing structural variations within the genome. Non-allelic homologous recombination (NAHR) has been known as the main mechanism causing deletions and translocation events<sup><a href="#CR42" class="usa-link" aria-describedby="CR42">42</a></sup>. The findings of our study, supported by the previously reported in Nellore and other <em>Bos indicus</em> breeds, may suggest that the deleted DNA fragments have caused the evolution and diversity of Nellore cattle genomes, but this can only be known by investigating the genomes of a higher number of ancestors. Besides, it is plausible that the alignment of sequences to the <em>Bos taurus</em> reference genome led to bias. The comparison of the copy number likely excluded specific sequences present in <em>Bos taurus</em> that are no longer present in the indicine cattle genomes, leading algorithms to consider those sequences as deleted fragments<sup><a href="#CR41" class="usa-link" aria-describedby="CR41">41</a></sup>. Thus, a better characterization of structural variation in indicine cattle requires the availability of a reliable <em>Bos indicus</em> reference genome.</p>
<p id="Par33">Regardless of the algorithm used to detect the structural variants from short-read sequencing, it is often expected to find a higher number of deletions than duplications. The read pair approach detects the duplicated copies in a mapped read located at a shorter distance than the fragment length, and when the length of the duplicated sequence is larger than the insert size, the algorithm tends to fail in the accurate detection of that duplication<sup><a href="#CR43" class="usa-link" aria-describedby="CR43">43</a></sup>. On the other hand, the read depth (RD) algorithm is more likely to easily detect deletions, as a decrease in the read depth or coverage at a certain genomic region signals them<sup><a href="#CR1" class="usa-link" aria-describedby="CR1">1</a></sup>. While duplications may not always result in a straightforward increase in read depth, once the duplicated region may have variable copy numbers and irregular coverage, making detection more challenging<sup><a href="#CR43" class="usa-link" aria-describedby="CR43">43</a></sup>. Furthermore, increases in the read depth or coverage are not always indicative of duplicated copies. These could be caused by alignment artifacts that cannot be differentiated from a true duplication through the RD approaches<sup><a href="#CR44" class="usa-link" aria-describedby="CR44">44</a></sup>.</p>
<p id="Par34">Combining different SV calling algorithms should reduce the false discovery rate compared to the use of a single approach. Furthermore, the application of a stringent merge strategy and filtering criteria, as conducted in this study, may ensure a more reliable identification of copy number variants<sup><a href="#CR28" class="usa-link" aria-describedby="CR28">28</a></sup>. The increase in the accuracy of detection of all types of SVs can be achieved through long-read sequencing (up to 80 Kb), in which single reads span complex structural variants, but its use on a large scale continues to be limited because of the high cost<sup><a href="#CR8" class="usa-link" aria-describedby="CR8">8</a></sup>.</p>
<p id="Par35">In relation to the balanced structural variants, inversions represented 8,36% of the total predicted high-confidence SVs (Table <a href="#Tab1" class="usa-link">1</a>). The proportion of inversions observed in the present study was lower than that reported by Boussaha et al.<sup><a href="#CR24" class="usa-link" aria-describedby="CR24">24</a></sup>who used WGS data from 380 animals of different <em>Bos taurus</em> dairy and beef cattle breeds, in which the inversions represented more than 30% of the total number of detected SVs. Our results also differ from the report by Chen et al.<sup><a href="#CR45" class="usa-link" aria-describedby="CR45">45</a></sup>where the proportion of inversions was higher than 20% of the total number of detected SVs in a study using WGS data from 308 Holstein animals. However, our results agree with the report by the latter authors on Jersey cattle, in which inversions represented 9% of the total number of SVs detected. The proportion of detected inversions in our study was higher than that reported by Lee et al.<sup><a href="#CR46" class="usa-link" aria-describedby="CR46">46</a></sup> (~ 4%), who used RD and SR approaches to detect SVs from WGS data in Holstein cattle.</p>
<p id="Par36">There is still a lack of knowledge regarding the characterization of balanced structural variants, such as inversions, in Nellore cattle. This is because most of the reported studies in this breed are based on SNP genotyping data<sup><a href="#CR12" class="usa-link" aria-describedby="CR12">12</a>,<a href="#CR14" class="usa-link" aria-describedby="CR14">14</a>,<a href="#CR22" class="usa-link" aria-describedby="CR22">22</a></sup>from which it is not possible to identify mobile elements but only copy number gains and/or copy number losses<sup><a href="#CR4" class="usa-link" aria-describedby="CR4">4</a></sup>. Nevertheless, there is an increasing interest in creating structural variation maps in cattle, with a higher number of published studies using whole genome sequencing data in taurine dairy and beef cattle than in indicine cattle.</p>
<p id="Par37">In evolutionary genomics, chromosomal inversions are known as a powerful force contributing to local adaptation and diversification, as they have the potential to protect inverted sequences from recombination, allowing the maintenance of favorable allele combinations<sup><a href="#CR47" class="usa-link" aria-describedby="CR47">47</a></sup>. Despite their evolutionary importance, there is not enough information to enable the characterization of this type of SV in cattle, and most of the studies continue focusing on deletion and duplication events. This is likely because the detection of SVs is often conducted through the RD approach, which only allows the identification of CNVs once balanced rearrangements (inversions and translocations) do not cause changes in read depth signature<sup><a href="#CR5" class="usa-link" aria-describedby="CR5">5</a>,<a href="#CR48" class="usa-link" aria-describedby="CR48">48</a></sup>. Furthermore, the complexity of the nature of these variants makes their interpretation and discovery more challenging when compared to the other SV types. The inversions and translocations’ breakpoints usually lie within complex repeats that reduce mappability, for which reliable detection requires the use of bioinformatics tools based on a combination of algorithms, such as split reads and paired reads<sup><a href="#CR48" class="usa-link" aria-describedby="CR48">48</a></sup>.</p></section><section id="Sec12"><h4 class="pmc_sec_title">Length classification of high-confidence SVs</h4>
<p id="Par38">The high-confidence SVs have an average length of 86.9 Kb (± 365.1), ranging from 1 Kb to 4,989.5 Kb (Table <a href="#Tab1" class="usa-link">1</a>). Although the CNVs, particularly the deletions, represent the highest number of the detected SVs, on average, they are the smallest (22.68 ± 167.31 Kb), while inversions are the longest ones (613.03 ± 851.34 Kb), followed by duplications (250.61 ± 560.12 Kb) (Table <a href="#Tab1" class="usa-link">1</a>). Analysis of the distribution of SVs according to different size classes revealed that most CNVs are 1–5 Kb long (73%), while inversions are distributed in all size classes, but most of them are 200 Kb long (48%) (Fig. <a href="#Fig3" class="usa-link">3</a>b). From the predicted deletions, the highest proportion is distributed into the classes 1–5 Kb and 5–10 Kb (77.2 and 16%, respectively), whereas predicted duplications are distributed in all size classes, where the lowest proportion are 5–10 Kb long (7.2%) (Table <a href="#Tab2" class="usa-link">2</a>).</p>
<figure class="fig xbox font-sm" id="Fig3"><h5 class="obj_head">Fig. 3.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365053_41598_2025_14139_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5894/12365053/398c887565f1/41598_2025_14139_Fig3_HTML.jpg" loading="lazy" id="d33e933" height="243" width="669" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Distribution of the detected structural variants according to different size classes for copy number variants (CNVs—<strong>a</strong>) and inversions (INVs—<strong>b</strong>).</p></figcaption></figure><section class="tw xbox font-sm" id="Tab2"><h5 class="obj_head">Table 2.</h5>
<div class="caption p"><p>Statistics summary of the number and proportion of each SV type according to different size classes.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Size class</th>
<th align="left" colspan="1" rowspan="1">SV type</th>
<th align="left" colspan="1" rowspan="1">
<em>N</em>
</th>
<th align="left" colspan="1" rowspan="1">Percentage (%)*</th>
</tr></thead>
<tbody>
<tr>
<td align="left" rowspan="3" colspan="1">1–5 Kb</td>
<td align="left" colspan="1" rowspan="1">DEL</td>
<td align="center" colspan="1" rowspan="1">141,259</td>
<td align="center" colspan="1" rowspan="1">77</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">DUP</td>
<td align="center" colspan="1" rowspan="1">2935</td>
<td align="center" colspan="1" rowspan="1">21</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">INV</td>
<td align="center" colspan="1" rowspan="1">2807</td>
<td align="center" colspan="1" rowspan="1">16</td>
</tr>
<tr>
<td align="left" rowspan="3" colspan="1">5–10 Kb</td>
<td align="left" colspan="1" rowspan="1">DEL</td>
<td align="center" colspan="1" rowspan="1">30,788</td>
<td align="center" colspan="1" rowspan="1">17</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">DUP</td>
<td align="center" colspan="1" rowspan="1">1013</td>
<td align="center" colspan="1" rowspan="1">7</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">INV</td>
<td align="center" colspan="1" rowspan="1">493</td>
<td align="center" colspan="1" rowspan="1">3</td>
</tr>
<tr>
<td align="left" rowspan="3" colspan="1">10–50 Kb</td>
<td align="left" colspan="1" rowspan="1">DEL</td>
<td align="center" colspan="1" rowspan="1">4593</td>
<td align="center" colspan="1" rowspan="1">2</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">DUP</td>
<td align="center" colspan="1" rowspan="1">2504</td>
<td align="center" colspan="1" rowspan="1">18</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">INV</td>
<td align="center" colspan="1" rowspan="1">2126</td>
<td align="center" colspan="1" rowspan="1">12</td>
</tr>
<tr>
<td align="left" rowspan="3" colspan="1">50–200 Kb</td>
<td align="left" colspan="1" rowspan="1">DEL</td>
<td align="center" colspan="1" rowspan="1">3322</td>
<td align="center" colspan="1" rowspan="1">2</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">DUP</td>
<td align="center" colspan="1" rowspan="1">4225</td>
<td align="center" colspan="1" rowspan="1">30</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">INV</td>
<td align="center" colspan="1" rowspan="1">3962</td>
<td align="center" colspan="1" rowspan="1">22</td>
</tr>
<tr>
<td align="left" rowspan="3" colspan="1">&gt; 200 Kb</td>
<td align="left" colspan="1" rowspan="1">DEL</td>
<td align="center" colspan="1" rowspan="1">3070</td>
<td align="center" colspan="1" rowspan="1">2</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">DUP</td>
<td align="center" colspan="1" rowspan="1">3336</td>
<td align="center" colspan="1" rowspan="1">24</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">INV</td>
<td align="center" colspan="1" rowspan="1">8598</td>
<td align="center" colspan="1" rowspan="1">48</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="_fn_p47">
<p><em>DEL</em> deletions, <em>DUP</em> duplications, <em>INV</em> inversions, <em>N</em> number of SVs, <em>Kb</em> kilo base pair.</p>
<p>*Percentage regarding the number of each SV type.</p>
</div></div></section><p id="Par41">The distribution of high-confidence CNVs in the established size classes agrees with the results of Bickhart et al.<sup><a href="#CR10" class="usa-link" aria-describedby="CR10">10</a></sup>who observed significantly shorter CNVs (&lt; 15 Kb) in a Nellore individual than the average length of the CNVs detected in different taurine individuals using WGS data. In contrast, our results differ from those reported by Lemos et al.<sup><a href="#CR12" class="usa-link" aria-describedby="CR12">12</a></sup> in Nellore cattle, who observed that most of the CNVs detected from SNP data had lengths ranging from 10 to 500 Kb and that less than 30% of the CNVs were 1–10 Kb-long. The results presented here also differ from those reported by Silva et al.<sup><a href="#CR22" class="usa-link" aria-describedby="CR22">22</a></sup> in Nellore cattle, in which the highest number of CNVs predicted from SNP data ranged from 100 to 900 Kb.</p>
<p id="Par42">It is important to note that the differences in CNV length between the report in the present study and the results reported in the literature are mostly attributed to the nature of the data used and, consequently, to the bioinformatic algorithm for CNV calling. WGS data provides higher resolution compared to SNP data, as it allows the identification of sequence reads spanning the entire genome. WGS-based algorithms enable the identification of smaller structural variants, especially CNVs, with greater power<sup><a href="#CR3" class="usa-link" aria-describedby="CR3">3</a></sup>. In contrast, CNV detection using SNP genotyping data often relies on analyzing changes in signal intensity at SNP probes. As there is a wide range of commercial SNP arrays, they all have different probe densities, which directly impact CNV results. SNP-based detection methods exhibit good performance in detecting large events, but compared to WGS-based algorithms, they tend to lack power in detecting smaller CNVs (&lt; 10 Kb) because of the low SNP probe density and resolution<sup><a href="#CR4" class="usa-link" aria-describedby="CR4">4</a></sup>.</p>
<p id="Par43">In general, the inversions are the largest structural variants detected, with an average length of 613.03 ± 851.34 Kb (median of 167.45 Kb) (Table <a href="#Tab1" class="usa-link">1</a>). The length of inversions observed in this study is higher than that reported by Chen et al.<sup><a href="#CR45" class="usa-link" aria-describedby="CR45">45</a></sup> in Holstein (median ~ 2.53 Kb) and Jersey (median ~ 1.34 Kb) cattle. Considering the observed average length of inversions, 48% are distributed into the size class &gt; 200 Kb, 22% into the size class 50–200 Kb, while a low proportion (3%) is distributed into the size class 5–10 Kb (Fig. <a href="#Fig4" class="usa-link">4</a>a; Table <a href="#Tab2" class="usa-link">2</a>). These results differ from those reported by Boussaha et al.<sup><a href="#CR24" class="usa-link" aria-describedby="CR24">24</a></sup> in European dairy and beef cattle animals, in which more than 50% of the detected inversions were distributed into the size class corresponding to 1–10 Kb.</p>
<figure class="fig xbox font-sm" id="Fig4"><h5 class="obj_head">Fig. 4.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365053_41598_2025_14139_Fig4_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5894/12365053/44cd78e5d94e/41598_2025_14139_Fig4_HTML.jpg" loading="lazy" id="d33e1168" height="318" width="669" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>(<strong>a</strong>) Number of high-confidence structural variants identified on each of the 29 autosomal chromosomes. <em>DEL</em> deletions, <em>DUP</em> duplications, <em>INV</em> inversions. (<strong>b</strong>) Average length (kb) of the predicted high-confidence structural variants across the 29 autosomes. (<strong>c</strong>) Average proportion (%) of each chromosome covered by high-confidence structural variants (average coverage rate).</p></figcaption></figure><p id="Par45">It is outstanding that the combination of tools and SV calling algorithms enabled the detection of large inversions, as they tend to be harder to identify. There are few available and reliable information regarding inversions and translocations in livestock species, especially in cattle, limiting comparisons. More studies of inversions are available in humans than in cattle, as there is an increasing interest in assessing the impact of reversed chromosome regions on phenotypes and diseases<sup><a href="#CR49" class="usa-link" aria-describedby="CR49">49</a></sup>. Accordingly, the vast number of inversions in the human genomes available in the InvFEST public database<sup><a href="#CR50" class="usa-link" aria-describedby="CR50">50</a></sup> are lower than 100 Kb long, from which variants with lengths lower than 10 Kb are considered reliable, and those larger than 10 Kb are probably false positives<sup><a href="#CR49" class="usa-link" aria-describedby="CR49">49</a></sup>.</p>
<p id="Par46">Nevertheless, genomic studies on inversions reviewed by Wellenreuther &amp; Bernatchez<sup><a href="#CR47" class="usa-link" aria-describedby="CR47">47</a></sup>showed that the average length of chromosomal inversion events in different species of plants and animals is 8.4 Mb, ranging from 130 Kb to 100 Mb. According to studies in evolutionary genomics in different species, including invertebrates and mammals, large chromosomal inversions are more favored by selection than shorter inversions, exhibiting lower rates of recombination between a larger number of genetically distant loci<sup><a href="#CR47" class="usa-link" aria-describedby="CR47">47</a>,<a href="#CR51" class="usa-link" aria-describedby="CR51">51</a></sup>.</p></section><section id="Sec13"><h4 class="pmc_sec_title">Chromosomal location and distribution of high-confidence SVs</h4>
<p id="Par47">The predicted high-confidence structural variants were not uniformly distributed across the 29 autosomal chromosomes. The top five chromosomes harboring the highest number of predicted SVs are BTA1 (<em>N</em> = 14,996), BTA2 (<em>N</em> = 13,773), BTA5 (<em>N</em> = 12,834), BTA4 (<em>N</em> = 12,16), and BTA10 (<em>N</em> = 10,372), while the top five chromosomes harboring the smallest number of SVs are BTA25 (<em>N</em> = 2,770), BTA19 (<em>N</em> = 3,145), BTA22 (<em>N</em> = 3,676), BTA27 (<em>N</em> = 4,035), and BTA24 (<em>N</em> = 4,607). The distribution of each SV type along the 29 autosomes is shown in Fig. <a href="#Fig4" class="usa-link">4</a>.</p>
<p id="Par48">Regarding each SV type, the BTA1, BTA2, and BTA5 carry the highest number of deletions, whereas BTA23, BTA15, and BTA10 carry the highest number of duplications, and BTA4, BTA10, and BTA5 harbor the highest number of inversions (Fig. <a href="#Fig4" class="usa-link">4</a>a). Our findings regarding the distribution of SV along the chromosomes are consistent with those observed by Peripolli et al.<sup><a href="#CR18" class="usa-link" aria-describedby="CR18">18</a></sup> when analyzing WGS data from three Brazilian locally adapted taurine breeds, in which the largest number of structural variants were found on BTA 1 and 15. However, the authors only predicted CNVs (duplications and deletions). The chromosomal locations observed here are also similar to those reported by Letaief et al.<sup><a href="#CR28" class="usa-link" aria-describedby="CR28">28</a></sup> in different French dairy and beef cattle breeds, in which the largest number of SVs was observed on chromosomes 1, 6, and 5 and the smallest number on chromosome 25. The authors also combined different tools for the SVs calling. The use of different bioinformatic algorithms when analyzing WGS data, as well as the filtering and validation of SVs at software, individual, and population levels, can lead to the identification of high-confidence structural variants. In contrast, the chromosomal location of SV was different from the reported in Nellore cattle from SNP and WGS data by Silva et al.<sup><a href="#CR22" class="usa-link" aria-describedby="CR22">22</a></sup>in which most predicted structural variations were located on BTA19, BTA11, BTA2, and BTA18 for SNP-based called SVs, and on BTA12, BTA15, BTA1, and BTA2 for WGS-based called SVs. Notice that the mentioned study was only focused on CNV assessment.</p></section><section id="Sec14"><h4 class="pmc_sec_title">Chromosomal extension covered by high-confidence SVs</h4>
<p id="Par49">BTA10 was ranked as the fifth, regarding the total number of SVs that carried, but the average length of its harbored SVs (728.36 Kb) was notably high in relation to the average length of SVs located on the other chromosomes, followed by SVs located on BTA9, 23, 5, and 15, with average lengths ranging from 102.98 Kb to 143.48 Kb (Fig. <a href="#Fig4" class="usa-link">4</a>b). However, the Pearson correlation did not indicate statistical evidence regarding the association between the number of SVs and the average length of SVs harbored by each chromosome (Pearson correlation score p-value = 0.197, p-value = 0.3053). Our findings were similar to those reported by Peripolli et al.<sup><a href="#CR18" class="usa-link" aria-describedby="CR18">18</a></sup>in which the longest CNV, corresponding to deletion events, was observed on chromosome 10. The BTA10 was the one having the largest proportion of average chromosome size covered by structural variation (average coverage rate = 50.4%), followed by BTA 23, 9, 15, and 28, with average coverage rates ranging from 7.6 to 11% (Fig. <a href="#Fig4" class="usa-link">4</a>c). The lowest average coverage rate was observed on BTA25, which was also the chromosome harboring the lowest number of structural variants and the smallest ones and was followed by BTA17, 19, 16, 14, 8, and 3, with average coverage rates ranging from 0.9 to 1.1%. These results differ from those reported by Letaief et al.<sup><a href="#CR28" class="usa-link" aria-describedby="CR28">28</a></sup>where the highest percentage of sequence covered by copy number variations in French dairy and beef cattle breeds was found on chromosome 27, but agree in that the least covered sequence by structural variants was located on BTA25.</p>
<p id="Par50">Pearson correlation analysis revealed a significantly high and positive association between the number of structural variants and the chromosome size (Pearson correlation score p-value = 0.87, p-value &lt; 6.8e-10), which agrees with the findings of Letaief et al.<sup><a href="#CR28" class="usa-link" aria-describedby="CR28">28</a></sup> but is different from the results reported by Boussaha et al.<sup><a href="#CR24" class="usa-link" aria-describedby="CR24">24</a></sup> in French dairy cattle. In contrast, no statistical evidence regarding correlations between the chromosome size and the average proportion of chromosome extension covered by SVs (average coverage rate) was found in this study (Pearson correlation score p-value = 0.04, p-value = 0.833).</p>
<p id="Par51">At the individual level, an average number of 1,437 SVs (± 894.7) were predicted for each bull, with a minimum number of 89 and a maximum of 3,428 SVs. The descriptive statistics about the number and length of each SV type observed in the analyzed bulls are presented in Supplementary Table S2. Structural variation encompasses, on average, 4.81% (± 4.38) of the Nellore bulls’ total genome extension. The largest genome coverage rate of SV observed in a bull was 20.27% and was largely attributed to inversion events, and the lowest genome coverage rate was 0.03% (Fig. <a href="#Fig5" class="usa-link">5</a>, Supplementary Table S2). A significant, moderate, and positive correlation was found between the number of SVs predicted per bull in each chromosome and the chromosomal coverage rate (Pearson correlation score p-value = 0.36, p-value &lt; 2.2e<sup>− 16</sup>).</p>
<figure class="fig xbox font-sm" id="Fig5"><h5 class="obj_head">Fig. 5.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365053_41598_2025_14139_Fig5_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5894/12365053/dd57d1fee80d/41598_2025_14139_Fig5_HTML.jpg" loading="lazy" id="d33e1305" height="287" width="669" alt="Fig. 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Genome coverage rate: proportion (%) of whole genome extension covered by each detected high-confidence structural variant (SV) for each Nellore bull. Each bar represents a bull and colors the structural variant type. <em>DEL</em> deletions, <em>DUP</em> duplications, <em>INV</em> inversions.</p></figcaption></figure></section></section><section id="Sec15"><h3 class="pmc_sec_title">Discovery of potential structural variant regions (SVR)</h3>
<section id="Sec16"><h4 class="pmc_sec_title">Number, chromosomal location and distribution of SVRs</h4>
<p id="Par53">High-confidence structural variants identified in the genome of bulls were used to detect genomic regions where different types of structural variants may occur. Specifically, a deletion may be observed in one bull and a duplication in another bull (across animals). Although less common, complex rearrangements may also lead to the coexistence of both deletion and duplication affecting overlapping segments within the same individual (within animal), as previously reported for certain types of structural variant architectures. The main purpose of translating structural variants into regions (SVR) is to characterize the genetic architecture underlying population history, to assess the potential impact of structural variations on gene function and expression, and to integrate information into genomic databases, allowing a more comprehensive understanding of differences between species, breeds, and populations<sup><a href="#CR35" class="usa-link" aria-describedby="CR35">35</a></sup>.</p>
<p id="Par54">Due to the different nature of SV types, two datasets were formed: one containing the detected CNVs (encompassing duplication and deletion events), and the other containing only the inversions. This enabled the proper identification of the SV event occurring in a specific region. Thus, SVRs were constructed from those two data sets by merging the SVs found in the individual genome of bulls that reciprocally overlapped in at least one base pair, resulting in two types of SVRs: CNVRs (regions of copy number variants) and INVRs (regions of inversions). In addition, only SVRs that were frequent in at least 5% of the analyzed Nellore bulls were considered for further analysis. A total of 3,752 highly frequent SVRs were identified, from which 97% are composed of CNVRs (<em>N</em> = 3,641), and 3% correspond to INVRs (<em>N</em> = 111; Fig. <a href="#Fig6" class="usa-link">6</a>a).</p>
<figure class="fig xbox font-sm" id="Fig6"><h5 class="obj_head">Fig. 6.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365053_41598_2025_14139_Fig6_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5894/12365053/bb3af22f3514/41598_2025_14139_Fig6_HTML.jpg" loading="lazy" id="d33e1343" height="320" width="669" alt="Fig. 6"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Proportion of each type of structural variant region (SVR). The proportion of SVRs corresponding to CNVR (regions of copy number variants) and INVR (regions of inversions) (<strong>a</strong>). Proportion of CVRs corresponding to CNVR-DEL (regions of deletions), CNVR-DUP (regions of duplications), and CNVR-MIXED (regions of deletions and duplications) (<strong>b</strong>).</p></figcaption></figure><p id="Par56">From the total number of detected CNVRs, the vast majority (90%, <em>N</em> = 3,286) corresponds to deletion events (CNVR-DEL), a small proportion (7%, <em>N</em> = 256) to mixed events in which both deletions and duplications occurred (CNVR-MIXED), and the lowest proportion (3%, <em>N</em> = 99) to only duplications (CNVR-DUP) (Fig. <a href="#Fig6" class="usa-link">6</a>b).</p>
<p id="Par57">The distribution of events occurring in the detected CNVRs observed in this study is different from those reported by Lemos et al.<sup><a href="#CR12" class="usa-link" aria-describedby="CR12">12</a></sup> in Nellore cattle, in which the CNVRs were composed in more balanced proportion by the three types of events (38.5% gains, 33% mixed, and 28.5% losses). The results also diverge from the reported CNVRs in Silva et al.<sup><a href="#CR22" class="usa-link" aria-describedby="CR22">22</a></sup> in Nellore cattle, where 65% of the CNVRs corresponded to mixed events, 31.5% to duplication, and a small proportion (2.89%) to deletion events. Although the CNVRs detected in Nellore cattle by Santana et al.<sup><a href="#CR14" class="usa-link" aria-describedby="CR14">14</a></sup> were attributed in higher proportion to copy number losses than copy number gains, the proportion of deletion events corresponding to the CNVRs observed in this study is higher than the reported by those authors (41.6% deletion events, 35% duplications, and 23% mixed events).</p>
<p id="Par58">Using the information on WGS in Gyr cattle, Braga et al.<sup><a href="#CR20" class="usa-link" aria-describedby="CR20">20</a></sup> found that the CNVRs detected from CNVnator calls were composed of more than 50% deletions, around 36% of duplications, and 11% of complex events (duplications and deletions). However, the results presented here are more consistent with what was reported by these authors, in which the CNVRs detected from Delly calls were attributed 70% to deletions, 7.8% to duplications, and 22.2% to complex events. Correspondingly, our findings agree with the results of Hou et al.<sup><a href="#CR52" class="usa-link" aria-describedby="CR52">52</a></sup> in different breeds, including taurine, indicine, and African cattle, in which most of the detected CNVRs were composed of loss events (61.3%), but the representation of regions of deletions observed in this study is still higher.</p>
<p id="Par59">A considerably higher number of deletions than duplications composing a CNVR was expected, given the total number of DEL calls obtained. Nevertheless, it is still surprising that a small fraction of duplications represents common events for all bulls. Interestingly, this finding is similar to the one reported by Letaief et al.<sup><a href="#CR28" class="usa-link" aria-describedby="CR28">28</a></sup> in French dairy and beef cattle, in which only 1.6% of CNVRs corresponded to duplications, 4,1% to complex events (deletions and duplications), and regions of deletions comprised more than 95%. Our results are also in concordance with the results reported in a recent study in Simmental cattle combining WGS data from short and long reads, where only 4,2% of the CNVRs were constituted by duplication regions, 39.6% by mixed events, and more than 56% by deletions<sup><a href="#CR19" class="usa-link" aria-describedby="CR19">19</a></sup>. It is important to outline that the number of the detected duplications in this study corresponded only to tandem duplications, as the algorithms and bioinformatic tools used to conduct the SV calling do not enable the identification of discontinuous duplications. Further factors that have likely driven the small number of called duplications relative to deletions were already discussed and are basically referred to as the greatest ability of SV callers to identify sequence losses in short reads sequencing data<sup><a href="#CR43" class="usa-link" aria-describedby="CR43">43</a>,<a href="#CR45" class="usa-link" aria-describedby="CR45">45</a></sup>the difficulty to differentiate duplications from alignment artifacts<sup><a href="#CR43" class="usa-link" aria-describedby="CR43">43</a></sup>and potential molecular mechanisms such as NAHR that potentially leads to more deletions<sup><a href="#CR42" class="usa-link" aria-describedby="CR42">42</a></sup>.</p>
<p id="Par60">The highest number of highly frequent SVRs was harbored by BTA1 (<em>N</em> = 261), followed by BTA6 and BTA2 with 211 SVRs each. BTA25, which carries the lowest number of SVs, has the lowest number of SVR as well (<em>N</em> = 47), followed by BTA19 and 22 with 67 SVR each (Table <a href="#Tab3" class="usa-link">3</a>; Fig. <a href="#Fig7" class="usa-link">7</a>). Regarding CNVRs, CNVR-DEL and CNVR-MIXED were uniformly distributed across the 29 autosomes, but CNVR-DUP was not observed in BTA20, BTA24, and BTA28 (Supplementary Figure <a href="#MOESM1" class="usa-link">S1</a>). The BTA1 harbored the highest number of CNVRs (<em>N</em> = 254), most of them corresponding to deletions (<em>N</em> = 235 CNVR-DEL), followed by mixed (<em>N</em> = 12 CNVR-MIXED) and duplication events (<em>N</em> = 7 CNVR-DUP) (Supplementary Table S3, Figure <a href="#MOESM1" class="usa-link">S1</a>). This result is in concordance with previous observations of CNVR distribution in Nellore cattle, reported by Lemos et al.<sup><a href="#CR12" class="usa-link" aria-describedby="CR12">12</a></sup> and Silva et al.<sup><a href="#CR22" class="usa-link" aria-describedby="CR22">22</a></sup>where the highest number of CNVRs were located on BTA1 but differs from theirs in that the smallest proportion of CNVR were located on BTA29 and BTA27, respectively. Our findings are similar to the results of Letaief et al.<sup><a href="#CR28" class="usa-link" aria-describedby="CR28">28</a></sup>where the highest and the lowest number of CNVRs were distributed on BTA1 and BTA25, respectively, and a similar distribution was observed by Peripolli et al.<sup><a href="#CR18" class="usa-link" aria-describedby="CR18">18</a></sup> in two of the three locally adapted taurine breeds. The SVRs composed of inversion events, BTA15 and BTA18 harbored the highest number of INVR, with 11 and 9 INVR, respectively, while BTA22 and BTA24 carried the lowest number of INVR (<em>N</em> = 1). INVR were distributed along the autosomes, except the BTA14, which did not harbor any INVR (Supplementary Table S3, Figure S2).</p>
<section class="tw xbox font-sm" id="Tab3"><h5 class="obj_head">Table 3.</h5>
<div class="caption p"><p>Statistics summary for the number of svrs, length and proportion of chromosome (Chr) extension covered by structural variant region (SVR) per autosomal chromosome.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead>
<tr>
<th align="left" rowspan="2" colspan="1">Chr</th>
<th align="left" rowspan="2" colspan="1">Chr size (kb)</th>
<th align="left" rowspan="2" colspan="1">
<em>N</em>
</th>
<th align="left" colspan="4" rowspan="1">Length (Kb)</th>
<th align="left" rowspan="2" colspan="1">Chr rate (%)</th>
</tr>
<tr>
<th align="left" colspan="1" rowspan="1">Mean</th>
<th align="left" colspan="1" rowspan="1">Sd</th>
<th align="left" colspan="1" rowspan="1">Max</th>
<th align="left" colspan="1" rowspan="1">Total</th>
</tr>
</thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">1</td>
<td align="left" colspan="1" rowspan="1">158,530</td>
<td align="center" colspan="1" rowspan="1">261</td>
<td align="center" colspan="1" rowspan="1">49.02</td>
<td align="center" colspan="1" rowspan="1">339.27</td>
<td align="left" colspan="1" rowspan="1">4661.89</td>
<td align="center" colspan="1" rowspan="1">12,794.97</td>
<td align="center" colspan="1" rowspan="1">8.07</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">2</td>
<td align="left" colspan="1" rowspan="1">136,230</td>
<td align="center" colspan="1" rowspan="1">211</td>
<td align="center" colspan="1" rowspan="1">117.22</td>
<td align="center" colspan="1" rowspan="1">622.57</td>
<td align="left" colspan="1" rowspan="1">6620.64</td>
<td align="center" colspan="1" rowspan="1">24,732.87</td>
<td align="center" colspan="1" rowspan="1">18.16</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">3</td>
<td align="left" colspan="1" rowspan="1">121,010</td>
<td align="center" colspan="1" rowspan="1">177</td>
<td align="center" colspan="1" rowspan="1">73.18</td>
<td align="center" colspan="1" rowspan="1">316.3</td>
<td align="left" colspan="1" rowspan="1">2458.36</td>
<td align="center" colspan="1" rowspan="1">12,952.17</td>
<td align="center" colspan="1" rowspan="1">10.7</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">4</td>
<td align="left" colspan="1" rowspan="1">120,000.60</td>
<td align="center" colspan="1" rowspan="1">196</td>
<td align="center" colspan="1" rowspan="1">68.83</td>
<td align="center" colspan="1" rowspan="1">390.78</td>
<td align="left" colspan="1" rowspan="1">4462,06</td>
<td align="center" colspan="1" rowspan="1">13,491.11</td>
<td align="center" colspan="1" rowspan="1">11.24</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">5</td>
<td align="left" colspan="1" rowspan="1">120,090</td>
<td align="center" colspan="1" rowspan="1">207</td>
<td align="center" colspan="1" rowspan="1">66.77</td>
<td align="center" colspan="1" rowspan="1">328.65</td>
<td align="left" colspan="1" rowspan="1">3102.13</td>
<td align="center" colspan="1" rowspan="1">13,821.45</td>
<td align="center" colspan="1" rowspan="1">11.51</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">6</td>
<td align="left" colspan="1" rowspan="1">117,810</td>
<td align="center" colspan="1" rowspan="1">211</td>
<td align="center" colspan="1" rowspan="1">35.11</td>
<td align="center" colspan="1" rowspan="1">195.18</td>
<td align="left" colspan="1" rowspan="1">2080.17</td>
<td align="center" colspan="1" rowspan="1">7408.53</td>
<td align="center" colspan="1" rowspan="1">6.29</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">7</td>
<td align="left" colspan="1" rowspan="1">110,680</td>
<td align="center" colspan="1" rowspan="1">190</td>
<td align="center" colspan="1" rowspan="1">71.92</td>
<td align="center" colspan="1" rowspan="1">315.48</td>
<td align="left" colspan="1" rowspan="1">2552.96</td>
<td align="center" colspan="1" rowspan="1">13,664.86</td>
<td align="center" colspan="1" rowspan="1">12.35</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">8</td>
<td align="left" colspan="1" rowspan="1">113,320</td>
<td align="center" colspan="1" rowspan="1">164</td>
<td align="center" colspan="1" rowspan="1">65.34</td>
<td align="center" colspan="1" rowspan="1">410.55</td>
<td align="left" colspan="1" rowspan="1">3885.85</td>
<td align="center" colspan="1" rowspan="1">10,715.99</td>
<td align="center" colspan="1" rowspan="1">9.46</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">9</td>
<td align="left" colspan="1" rowspan="1">105,450</td>
<td align="center" colspan="1" rowspan="1">165</td>
<td align="center" colspan="1" rowspan="1">103.22</td>
<td align="center" colspan="1" rowspan="1">500.86</td>
<td align="left" colspan="1" rowspan="1">4633.04</td>
<td align="center" colspan="1" rowspan="1">17,031.79</td>
<td align="center" colspan="1" rowspan="1">16.15</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">10</td>
<td align="left" colspan="1" rowspan="1">103,310</td>
<td align="center" colspan="1" rowspan="1">117</td>
<td align="center" colspan="1" rowspan="1">108.21</td>
<td align="center" colspan="1" rowspan="1">450.42</td>
<td align="left" colspan="1" rowspan="1">2693.91</td>
<td align="center" colspan="1" rowspan="1">12,660.45</td>
<td align="center" colspan="1" rowspan="1">12.25</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">11</td>
<td align="left" colspan="1" rowspan="1">106,980</td>
<td align="center" colspan="1" rowspan="1">108</td>
<td align="center" colspan="1" rowspan="1">192.82</td>
<td align="center" colspan="1" rowspan="1">742.09</td>
<td align="left" colspan="1" rowspan="1">4852.18</td>
<td align="center" colspan="1" rowspan="1">20,824.23</td>
<td align="center" colspan="1" rowspan="1">19.47</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">12</td>
<td align="left" colspan="1" rowspan="1">87,220</td>
<td align="center" colspan="1" rowspan="1">149</td>
<td align="center" colspan="1" rowspan="1">93.76</td>
<td align="center" colspan="1" rowspan="1">455.91</td>
<td align="left" colspan="1" rowspan="1">3494.42</td>
<td align="center" colspan="1" rowspan="1">13,970.59</td>
<td align="center" colspan="1" rowspan="1">16.02</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">13</td>
<td align="left" colspan="1" rowspan="1">83,470</td>
<td align="center" colspan="1" rowspan="1">106</td>
<td align="center" colspan="1" rowspan="1">87.24</td>
<td align="center" colspan="1" rowspan="1">492.17</td>
<td align="left" colspan="1" rowspan="1">4396.62</td>
<td align="center" colspan="1" rowspan="1">9247.47</td>
<td align="center" colspan="1" rowspan="1">11.08</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">14</td>
<td align="left" colspan="1" rowspan="1">82,400</td>
<td align="center" colspan="1" rowspan="1">108</td>
<td align="center" colspan="1" rowspan="1">91.65</td>
<td align="center" colspan="1" rowspan="1">469.05</td>
<td align="left" colspan="1" rowspan="1">3919.73</td>
<td align="center" colspan="1" rowspan="1">9898.47</td>
<td align="center" colspan="1" rowspan="1">12.01</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">15</td>
<td align="left" colspan="1" rowspan="1">85,010</td>
<td align="center" colspan="1" rowspan="1">177</td>
<td align="center" colspan="1" rowspan="1">82.34</td>
<td align="center" colspan="1" rowspan="1">344.39</td>
<td align="left" colspan="1" rowspan="1">2771.55</td>
<td align="center" colspan="1" rowspan="1">14,574.51</td>
<td align="center" colspan="1" rowspan="1">17.14</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">16</td>
<td align="left" colspan="1" rowspan="1">81,010</td>
<td align="center" colspan="1" rowspan="1">113</td>
<td align="center" colspan="1" rowspan="1">41.7</td>
<td align="center" colspan="1" rowspan="1">242.81</td>
<td align="left" colspan="1" rowspan="1">2378.82</td>
<td align="center" colspan="1" rowspan="1">4711.80</td>
<td align="center" colspan="1" rowspan="1">5.82</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">17</td>
<td align="left" colspan="1" rowspan="1">73,170</td>
<td align="center" colspan="1" rowspan="1">105</td>
<td align="center" colspan="1" rowspan="1">49.56</td>
<td align="center" colspan="1" rowspan="1">242.82</td>
<td align="left" colspan="1" rowspan="1">1657.28</td>
<td align="center" colspan="1" rowspan="1">5203.79</td>
<td align="center" colspan="1" rowspan="1">7.11</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">18</td>
<td align="left" colspan="1" rowspan="1">65,820</td>
<td align="center" colspan="1" rowspan="1">92</td>
<td align="center" colspan="1" rowspan="1">116.46</td>
<td align="center" colspan="1" rowspan="1">380.62</td>
<td align="left" colspan="1" rowspan="1">2993.72</td>
<td align="center" colspan="1" rowspan="1">10,714.26</td>
<td align="center" colspan="1" rowspan="1">16.28</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">19</td>
<td align="left" colspan="1" rowspan="1">63,450</td>
<td align="center" colspan="1" rowspan="1">67</td>
<td align="center" colspan="1" rowspan="1">99.1</td>
<td align="center" colspan="1" rowspan="1">424.04</td>
<td align="left" colspan="1" rowspan="1">3213.91</td>
<td align="center" colspan="1" rowspan="1">6640.01</td>
<td align="center" colspan="1" rowspan="1">10.46</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">20</td>
<td align="left" colspan="1" rowspan="1">71,970</td>
<td align="center" colspan="1" rowspan="1">122</td>
<td align="center" colspan="1" rowspan="1">124.22</td>
<td align="center" colspan="1" rowspan="1">794.4</td>
<td align="left" colspan="1" rowspan="1">8377.91</td>
<td align="center" colspan="1" rowspan="1">15,154.75</td>
<td align="center" colspan="1" rowspan="1">21.06</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">21</td>
<td align="left" colspan="1" rowspan="1">69,860</td>
<td align="center" colspan="1" rowspan="1">91</td>
<td align="center" colspan="1" rowspan="1">157.09</td>
<td align="center" colspan="1" rowspan="1">738.07</td>
<td align="left" colspan="1" rowspan="1">4906.43</td>
<td align="center" colspan="1" rowspan="1">14,294.91</td>
<td align="center" colspan="1" rowspan="1">20.46</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">22</td>
<td align="left" colspan="1" rowspan="1">60,770</td>
<td align="center" colspan="1" rowspan="1">67</td>
<td align="center" colspan="1" rowspan="1">88.96</td>
<td align="center" colspan="1" rowspan="1">429.23</td>
<td align="left" colspan="1" rowspan="1">3086.26</td>
<td align="center" colspan="1" rowspan="1">5960.54</td>
<td align="center" colspan="1" rowspan="1">9.81</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">23</td>
<td align="left" colspan="1" rowspan="1">52,500</td>
<td align="center" colspan="1" rowspan="1">90</td>
<td align="center" colspan="1" rowspan="1">83.86</td>
<td align="center" colspan="1" rowspan="1">476.29</td>
<td align="left" colspan="1" rowspan="1">4393.12</td>
<td align="center" colspan="1" rowspan="1">7547.36</td>
<td align="center" colspan="1" rowspan="1">14.38</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">24</td>
<td align="left" colspan="1" rowspan="1">62,320</td>
<td align="center" colspan="1" rowspan="1">88</td>
<td align="center" colspan="1" rowspan="1">117.19</td>
<td align="center" colspan="1" rowspan="1">413.8</td>
<td align="left" colspan="1" rowspan="1">2532.45</td>
<td align="center" colspan="1" rowspan="1">10,312.71</td>
<td align="center" colspan="1" rowspan="1">16.55</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">25</td>
<td align="left" colspan="1" rowspan="1">42,350</td>
<td align="center" colspan="1" rowspan="1">47</td>
<td align="center" colspan="1" rowspan="1">46.95</td>
<td align="center" colspan="1" rowspan="1">192.26</td>
<td align="left" colspan="1" rowspan="1">1019.71</td>
<td align="center" colspan="1" rowspan="1">2206.56</td>
<td align="center" colspan="1" rowspan="1">5.21</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">26</td>
<td align="left" colspan="1" rowspan="1">51,990</td>
<td align="center" colspan="1" rowspan="1">74</td>
<td align="center" colspan="1" rowspan="1">171.39</td>
<td align="center" colspan="1" rowspan="1">655.8</td>
<td align="left" colspan="1" rowspan="1">3876.00</td>
<td align="center" colspan="1" rowspan="1">12,682.51</td>
<td align="center" colspan="1" rowspan="1">24.39</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">27</td>
<td align="left" colspan="1" rowspan="1">45,610</td>
<td align="center" colspan="1" rowspan="1">72</td>
<td align="center" colspan="1" rowspan="1">114.33</td>
<td align="center" colspan="1" rowspan="1">501.51</td>
<td align="left" colspan="1" rowspan="1">3832.58</td>
<td align="center" colspan="1" rowspan="1">8231.99</td>
<td align="center" colspan="1" rowspan="1">18.05</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">28</td>
<td align="left" colspan="1" rowspan="1">45,940</td>
<td align="center" colspan="1" rowspan="1">79</td>
<td align="center" colspan="1" rowspan="1">130.32</td>
<td align="center" colspan="1" rowspan="1">489.32</td>
<td align="left" colspan="1" rowspan="1">2704.58</td>
<td align="center" colspan="1" rowspan="1">10,295.38</td>
<td align="center" colspan="1" rowspan="1">22.41</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">29</td>
<td align="left" colspan="1" rowspan="1">51,100</td>
<td align="center" colspan="1" rowspan="1">98</td>
<td align="center" colspan="1" rowspan="1">159.21</td>
<td align="center" colspan="1" rowspan="1">912.09</td>
<td align="left" colspan="1" rowspan="1">8530.07</td>
<td align="center" colspan="1" rowspan="1">15,602.19</td>
<td align="center" colspan="1" rowspan="1">30.53</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="_fn_p76"><p><em>N</em> number of SVR, <em>Sd</em> standard deviation, <em>max</em> maximum, <em>Total</em> sum of all SVRs length. Chromosomal coverage rate (%): proportion of chromosome extension covered by SVR.</p></div></div></section><figure class="fig xbox font-sm" id="Fig7"><h5 class="obj_head">Fig. 7.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365053_41598_2025_14139_Fig7_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5894/12365053/2686436af395/41598_2025_14139_Fig7_HTML.jpg" loading="lazy" id="d33e1476" height="344" width="669" alt="Fig. 7"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig7/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Distribution of non-redundant structural variant region (SVR) in more than 5% of the population across the 29 autosomes. <em>CNVR</em> regions of copy number variants, <em>INVR</em> regions of inversions.</p></figcaption></figure></section><section id="Sec17"><h4 class="pmc_sec_title">Average length and autosomal genome extension covered by SVRs</h4>
<p id="Par63">The 3,752 SVRs exhibit an average length of 89.91 Kb (± 469.09) and a median of 2.08 Kb. The average length of INVR is 396.44 Kb (± 951.03 Kb) (Table <a href="#Tab4" class="usa-link">4</a>). Regarding CNVRs, the length trend is similar to the single events, in which the region of deletions exhibits the highest number but the shortest length (25.95 ± 232.35 Kb), followed by regions of duplications (36.47 ± 91.23 Kb), and the regions of mixed events (799 ± 1.24 Kb) which represent the longest CNVR type (Table <a href="#Tab4" class="usa-link">4</a>).</p>
<section class="tw xbox font-sm" id="Tab4"><h5 class="obj_head">Table 4.</h5>
<div class="caption p"><p>Statistics summary for the number, length, and genome coverage rate for each structural variant region (SVR) type (present in at least 5% of bulls).</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead>
<tr>
<th align="left" rowspan="2" colspan="1">SVR TYPE</th>
<th align="left" rowspan="2" colspan="1">
<em>N</em>
</th>
<th align="left" colspan="5" rowspan="1">Length (Kb)</th>
<th align="left" rowspan="2" colspan="1">Chr rate (%)</th>
</tr>
<tr>
<th align="left" colspan="1" rowspan="1">Mean</th>
<th align="left" colspan="1" rowspan="1">SD</th>
<th align="left" colspan="1" rowspan="1">Median</th>
<th align="left" colspan="1" rowspan="1">Max</th>
<th align="left" colspan="1" rowspan="1">Total</th>
</tr>
</thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">CNVR-DEL</td>
<td align="center" colspan="1" rowspan="1">3286</td>
<td align="left" colspan="1" rowspan="1">25.92</td>
<td align="center" colspan="1" rowspan="1">232.35</td>
<td align="left" colspan="1" rowspan="1">1.87</td>
<td align="center" colspan="1" rowspan="1">4852.18</td>
<td align="center" colspan="1" rowspan="1">85,188.62</td>
<td align="center" colspan="1" rowspan="1">3.31</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">CNVR-DUP</td>
<td align="center" colspan="1" rowspan="1">99</td>
<td align="left" colspan="1" rowspan="1">36.47</td>
<td align="center" colspan="1" rowspan="1">91.23</td>
<td align="left" colspan="1" rowspan="1">5.03</td>
<td align="center" colspan="1" rowspan="1">759.28</td>
<td align="center" colspan="1" rowspan="1">3610.60</td>
<td align="center" colspan="1" rowspan="1">0.14</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">CNVR-MIXED</td>
<td align="center" colspan="1" rowspan="1">256</td>
<td align="left" colspan="1" rowspan="1">799</td>
<td align="center" colspan="1" rowspan="1">1244.38</td>
<td align="left" colspan="1" rowspan="1">199.56</td>
<td align="center" colspan="1" rowspan="1">8377.91</td>
<td align="center" colspan="1" rowspan="1">204,544.38</td>
<td align="center" colspan="1" rowspan="1">7.95</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">ALL-CNVR</td>
<td align="center" colspan="1" rowspan="1">3641</td>
<td align="left" colspan="1" rowspan="1">80.57</td>
<td align="center" colspan="1" rowspan="1">443.25</td>
<td align="left" colspan="1" rowspan="1">2.028</td>
<td align="center" colspan="1" rowspan="1">8377.91</td>
<td align="center" colspan="1" rowspan="1">293,343.60</td>
<td align="center" colspan="1" rowspan="1">11.4</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">INVR</td>
<td align="center" colspan="1" rowspan="1">111</td>
<td align="left" colspan="1" rowspan="1">396.44</td>
<td align="center" colspan="1" rowspan="1">951.03</td>
<td align="left" colspan="1" rowspan="1">74</td>
<td align="center" colspan="1" rowspan="1">8530.07</td>
<td align="center" colspan="1" rowspan="1">44,004.73</td>
<td align="center" colspan="1" rowspan="1">1.71</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">ALL SVR</td>
<td align="center" colspan="1" rowspan="1">3752</td>
<td align="left" colspan="1" rowspan="1">89.91</td>
<td align="center" colspan="1" rowspan="1">469.09</td>
<td align="left" colspan="1" rowspan="1">2.08</td>
<td align="center" colspan="1" rowspan="1">8530.07</td>
<td align="center" colspan="1" rowspan="1">337,348.30</td>
<td align="center" colspan="1" rowspan="1">13.12</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="_fn_p80">
<p>Chr rate (%)—Chromosomal coverage rate (%).</p>
<p><em>CNVR</em> regions of copy number variants, <em>CNVR-DEL</em> region of copy number variants composed by deletions, <em>CNVR-DUP</em> region of copy number variants composed by duplications, <em>CNVR-MIXED</em> region of copy number variants composed by deletions and duplications, <em>INVR</em> structural variant regions corresponding to inversions, <em>N</em> number of SVR, <em>Sd</em> standard deviation, <em>max</em> maximum, <em>Total</em> sum of all SVRs length.</p>
</div></div></section><p id="Par65">The longest SVR found has a length of 8,530.07 Kb, and the join of all SVRs length comprises a total of 337,348.3 Kb, which represents 13.12% of whole autosomal genome extension (attributed to CNVRs in 11.40% and INVR in 1.71%) (Table <a href="#Tab4" class="usa-link">4</a>). This result is in concordance with what Lemos et al.<sup><a href="#CR12" class="usa-link" aria-describedby="CR12">12</a></sup> previously reported, where the predicted regions with duplication and deletion events (CNVR) from SNP genotyping data represented 13% of the genome coverage. Remarkably, the authors analyzed a Nellore population from which the analyzed bulls belonged to the same population as the evaluated in this research. Thus, similar results regarding the genome coverage of structural variations were expected. However, some of our findings have differed from those previously reported by these authors, mainly due to differences in the number of animals analyzed, the data nature used, and the combination of different strategies and algorithms for structural variants calling.</p>
<p id="Par66">The total genomic coverage of CNVR found in this study was higher than the reported coverage rate in other Nellore cattle populations. For example, Santana et al.<sup><a href="#CR14" class="usa-link" aria-describedby="CR14">14</a></sup> observed a total coverage of CNVR of 9% for predictions made from SNP data, and Silva et al.<sup><a href="#CR22" class="usa-link" aria-describedby="CR22">22</a></sup> reported an autosomal coverage of CNVR of 1.5% for predictions made from SNP and NGS data. When comparing our results with the reported for other taurine and indicine breeds, our observed coverage rate was also higher than those from Bickhart et al.<sup><a href="#CR10" class="usa-link" aria-describedby="CR10">10</a></sup> (2.1%), Bickhart et al.<sup><a href="#CR23" class="usa-link" aria-describedby="CR23">23</a></sup> (3.1%), Keel et al.<sup><a href="#CR53" class="usa-link" aria-describedby="CR53">53</a></sup> (6.7%), and Hu et al.<sup><a href="#CR41" class="usa-link" aria-describedby="CR41">41</a></sup> (1.5%), and Sun et al.<sup><a href="#CR19" class="usa-link" aria-describedby="CR19">19</a></sup> (0.18%), in which CNVRs reported were those shared by diverse cattle breeds including Nellore, Brahman, Gir, and different <em>Bos taurus</em> breeds.</p>
<p id="Par67">Likewise, SV distribution according to size classes, most of the detected SVRs (73%) have lengths ranging from 1 to 5 Kb, 14% are 5–10 Kb-long, 4% are 10–50 Kb-long, 3% 50–200 Kb-long, and 6% are larger than 200 Kb. (Table <a href="#Tab5" class="usa-link">5</a>). CNVRs are mainly distributed into the classes 1–5 Kb and 5–10 Kb (Fig. <a href="#Fig3" class="usa-link">3</a>a), while INVRs are more balanced and distributed into three size classes (10 to 50 Kb, 50 to 200, and larger than 200 - Fig. <a href="#Fig8" class="usa-link">8</a>).</p>
<section class="tw xbox font-sm" id="Tab5"><h5 class="obj_head">Table 5.</h5>
<div class="caption p"><p>Number and proportion of each type of detected SVR according to different size classes.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Size class</th>
<th align="left" colspan="1" rowspan="1">SVR type</th>
<th align="left" colspan="1" rowspan="1">
<em>N</em>
</th>
<th align="left" colspan="1" rowspan="1">Percentage (%)</th>
</tr></thead>
<tbody>
<tr>
<td align="left" rowspan="3" colspan="1">1–5 kb</td>
<td align="left" colspan="1" rowspan="1">CNVR</td>
<td align="center" colspan="1" rowspan="1">2699</td>
<td align="left" colspan="1" rowspan="1">71.93</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">INVR</td>
<td align="center" colspan="1" rowspan="1">31</td>
<td align="left" colspan="1" rowspan="1">0.83</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">ALL SVR</td>
<td align="center" colspan="1" rowspan="1">2730</td>
<td align="left" colspan="1" rowspan="1">73</td>
</tr>
<tr>
<td align="left" rowspan="3" colspan="1">5–10 kb</td>
<td align="left" colspan="1" rowspan="1">CNVR</td>
<td align="center" colspan="1" rowspan="1">540</td>
<td align="left" colspan="1" rowspan="1">14.39</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">INVR</td>
<td align="center" colspan="1" rowspan="1">4</td>
<td align="left" colspan="1" rowspan="1">0.11</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">ALL SVR</td>
<td align="center" colspan="1" rowspan="1">544</td>
<td align="left" colspan="1" rowspan="1">14</td>
</tr>
<tr>
<td align="left" rowspan="3" colspan="1">10–50 kb</td>
<td align="left" colspan="1" rowspan="1">CNVR</td>
<td align="center" colspan="1" rowspan="1">132</td>
<td align="left" colspan="1" rowspan="1">3.52</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">INVR</td>
<td align="center" colspan="1" rowspan="1">13</td>
<td align="left" colspan="1" rowspan="1">0.35</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">ALL SVR</td>
<td align="center" colspan="1" rowspan="1">145</td>
<td align="left" colspan="1" rowspan="1">4</td>
</tr>
<tr>
<td align="left" rowspan="3" colspan="1">50–200 kb</td>
<td align="left" colspan="1" rowspan="1">CNVR</td>
<td align="center" colspan="1" rowspan="1">92</td>
<td align="left" colspan="1" rowspan="1">2.45</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">INVR</td>
<td align="center" colspan="1" rowspan="1">24</td>
<td align="left" colspan="1" rowspan="1">0.64</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">ALL SVR</td>
<td align="center" colspan="1" rowspan="1">116</td>
<td align="left" colspan="1" rowspan="1">6</td>
</tr>
<tr>
<td align="left" rowspan="3" colspan="1">&gt; 200 kb</td>
<td align="left" colspan="1" rowspan="1">CNVR</td>
<td align="center" colspan="1" rowspan="1">178</td>
<td align="left" colspan="1" rowspan="1">4.74</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">INVR</td>
<td align="center" colspan="1" rowspan="1">39</td>
<td align="left" colspan="1" rowspan="1">1.04</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">ALL SVR</td>
<td align="center" colspan="1" rowspan="1">217</td>
<td align="left" colspan="1" rowspan="1">6</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="_fn_p89">
<p><em>SVR</em> structural variant regions, <em>CNVR</em> copy number variation region, <em>INVR</em> structural variant regions corresponding to inversion events, <em>N</em> number of SVR.</p>
<p>Percentage (%): proportion of each SVR type in relation to the total number of detected SVR.</p>
</div></div></section><figure class="fig xbox font-sm" id="Fig8"><h5 class="obj_head">Fig. 8.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365053_41598_2025_14139_Fig8_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5894/12365053/6f44c2bea50f/41598_2025_14139_Fig8_HTML.jpg" loading="lazy" id="d33e2291" height="274" width="669" alt="Fig. 8"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig8/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Number of structural variants regions (SVR) according to different size classes. <em>CNV</em> regions of copy number variants (<strong>a</strong>). <em>INVR</em> regions of inversions (<strong>b</strong>).</p></figcaption></figure><p id="Par70">The top five longest CNVRs are located on BTA11, 26, 21, 2, and 24, with average lengths ranging from 186.99 to 116.22 Kb (Supplementary Figure S3a). Otherwise, the top five shortest CNVRs are harbored by BTA6, 16, 25, 17, and 1, with average lengths ranging from 34.23 to 49.5 Kb. It is remarkable that although BTA1 is ranked first in terms of the number, the harbored CNVR is of short lengths. This supports the results of the Pearson correlation test, which revealed that there is a trend that the higher the number of SVRs carried by each chromosome, the shorter the SVRs are (Pearson correlation coefficient p-value = -0.37, p-value = 0.04976). By their side, the top five longest regions of inversions are carried by BTA29, 10, 7, 12, and 20, while the shortest INVR is located on BTA22, 2, 16, 1, and 6, respectively (Supplementary Figure S4a).</p>
<p id="Par71">Regarding the proportion of total chromosome extension that is covered by SVRs, the BTA29 is the most covered by regions with structural variation (chromosomal coverage rate of 30.53%), followed by chromosomes 26, 28, 20, and 21, with chromosomal coverage rates ranging from 22.46 to 24.39%. On the other side, BTA25 exhibits the lowest chromosomal coverage rate (5.21%), followed by BTA6, 16, 17, and 1, with chromosomal coverage rates ranging between 5.82 and 8.07% (Fig. <a href="#Fig9" class="usa-link">9</a>; Table <a href="#Tab3" class="usa-link">3</a>). BTA26 is the most covered chromosome by CNVR at 24.34%, while BTA25 showed the lowest covered at 4.68% (Supplementary Figure S3b).</p>
<figure class="fig xbox font-sm" id="Fig9"><h5 class="obj_head">Fig. 9.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365053_41598_2025_14139_Fig9_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5894/12365053/01f3ceef77cd/41598_2025_14139_Fig9_HTML.jpg" loading="lazy" id="d33e2470" height="337" width="669" alt="Fig. 9"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig9/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Chromosomal coverage rate: percentage (%) of chromosome extension covered by structural variant regions (SVRs.)</p></figcaption></figure><p id="Par73">The maximum percentage of chromosomal coverage observed in this study is higher than the maximum percentage reported by Lemos et al.<sup><a href="#CR12" class="usa-link" aria-describedby="CR12">12</a></sup> (24.26%) in Nellore cattle, differing from their results regarding the chromosomal distribution of coverage rates. Specifically, Lemos et al.<sup><a href="#CR12" class="usa-link" aria-describedby="CR12">12</a></sup> identified BTA19, BTA23, and BTA25 as the most covered chromosomes, whereas BTA9, BTA13, and BTA8 were the least covered by CNVR. The results also differ from those reported by Silva et al.<sup><a href="#CR22" class="usa-link" aria-describedby="CR22">22</a></sup> in Nellore cattle, where CNVRs on BTA24 exhibited the greatest total length and those on BTA19 had the shortest. In our study, the highest chromosomal coverage by INVRs was observed on BTA29, whereas BTA22 exhibited sparse coverage by inversions (Supplementary Figure S4b). Notably, BTA29 did not exhibit the highest number of SVRs, nor were the longest SVRs observed on this chromosome. Rather, the highest coverage on BTA29 can be attributed to the presence of two large inversions, combined with its smallest length, which increases the proportion of its sequence affected by SVs. Our results coincided with those from Letaief et al.<sup><a href="#CR28" class="usa-link" aria-describedby="CR28">28</a></sup> on French cattle, where chromosomes 25 and 1 were the least covered by structural variants. Pearson correlation analysis revealed a significant, strong, and positive association between the number of SVRs and the chromosome size (Pearson correlation score p-value = 0.93, p value = 4.84e<sup>− 13</sup>), which is concurrent with the findings of Letaief et al.<sup><a href="#CR28" class="usa-link" aria-describedby="CR28">28</a></sup> and Silva et al.<sup><a href="#CR22" class="usa-link" aria-describedby="CR22">22</a></sup>. In contrast, no significant correlation was detected between chromosome size and the proportion of chromosomal length covered by SVRs (chromosomal coverage rate).</p></section></section><section id="Sec18"><h3 class="pmc_sec_title">Functional annotation and enrichment of SVR</h3>
<p id="Par74">Most of the identified SVRs were common for less than 50% of bulls (Fig. <a href="#Fig10" class="usa-link">10</a>), but a considerably high number of SVRs were shared by more than half of the studied population (<em>N</em> = 532). To assess the functional impact on genetic mechanisms underlying Nellore cattle structural variations, genes, and QTLs overlapping SVRs shared by more than 50% (<em>N</em> &gt; 75 bulls) were retrieved from a publicly available genomic dataset, as described in the material and methods section. Figure <a href="#Fig11" class="usa-link">11</a> illustrates the chromosomal location of the most frequent SVRs (<em>N</em> = 532). Overall, from the 532 most frequent SVRs, 96 overlapped a total of 130 QTLs, which were distributed in 6 QTL types: exterior, health, meat and carcass, milk, production, and reproduction, which are related to a total of 50 traits (Supplementary Table S4). Figure <a href="#Fig12" class="usa-link">12</a> represents the number of SVRs overlapping QTLs, according to QTL type and trait associated.</p>
<figure class="fig xbox font-sm" id="Fig10"><h4 class="obj_head">Fig. 10.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365053_41598_2025_14139_Fig10_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5894/12365053/25dbb7a24b9d/41598_2025_14139_Fig10_HTML.jpg" loading="lazy" id="d33e2533" height="311" width="669" alt="Fig. 10"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig10/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Distribution of the number of the structural variant regions (SVR) shared by different proportions of Bulls.</p></figcaption></figure><figure class="fig xbox font-sm" id="Fig11"><h4 class="obj_head">Fig. 11.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365053_41598_2025_14139_Fig11_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5894/12365053/104c1eb3150d/41598_2025_14139_Fig11_HTML.jpg" loading="lazy" id="d33e2543" height="945" width="795" alt="Fig. 11"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig11/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Chromosomal location of the most frequent structural variant regions (SVR). Blue dots represent regions of deletions, green dots are mixed regions, red dots are the regions of duplications, and black dots the regions of inversions.</p></figcaption></figure><figure class="fig xbox font-sm" id="Fig12"><h4 class="obj_head">Fig. 12.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365053_41598_2025_14139_Fig12_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5894/12365053/4728896a5bf5/41598_2025_14139_Fig12_HTML.jpg" loading="lazy" id="d33e2560" height="397" width="669" alt="Fig. 12"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig12/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Distribution of structural variants (SVs) overlapping quantitative trait loci (QTLs). The number of QTL types identified (<strong>a</strong>) and the number of SVs per trait associated with the identified QTLs (<strong>b</strong>).</p></figcaption></figure><p id="Par78">The highest number of SVRs overlapped with QTLs related to residual feed intake (<em>N</em> = 21), followed by structural soundness (<em>N</em> = 16), multiple births (<em>N</em> = 9), clinical mastitis (<em>N</em> = 7), and milk energy yield (<em>N</em> = 7). The SVRs overlapping QTLs were distributed in all chromosomes except on BTA24. The BTA6 was the chromosome harboring the largest number of QTLs overlapping SVRs (<em>N</em> = 11), followed by BTA7 (<em>N</em> = 8) and BTA3, 4, 5, and 23 with 7 QTLs each. In addition, the region named CNVR1799 (BTA6:72.92–74.15 Mb) that was present in 75 bulls overlapped a total of 6 QTLs, distributed in two QTL types: production (residual feed intake), and exterior (teat placement, foot angle, udder quality, udder depth, and degree of spotting). Regarding the INVR, 4 regions overlapped QTLs: INVR27 (BTA1:39.48–39.53 Mb) overlapping a QTL related to structural soundness, INVR131 (BTA3:20.94–22.58 Mb) overlapping a QTL related to milk yield, INVR549 (BTA15:78.78–80.14 Mb) overlapping a QTL related to liver abscess, and INVR557 (BTA16:3.11–3.12 Mb) overlapping a QTL related to structural soundness. Most of the 96 SVRs were linked to just one QTL.</p>
<p id="Par79">Regarding the gene annotation, from the 532 most frequent SVRs, a total of 205 SVRs overlapped known genes (Supplementary Table S5). A total of 1,164 positional candidate genes surrounding the SVRs were annotated. The most annotated region of copy number variants named as CNVR3428 (BTA13:57.35–57.36 Mb) harbored a total of 84 genes, followed by the region CNVR4347 (BTA18:61.87–61.90 Mb), which harbored 58 genes, and the regions, CNVR1601 (BTA5:112.69-112.71 Mb), CNVR4532 (BTA20:33.44–33.51 Mb), and CNVR4703 (BTA21:31.13–31.19 Mb), harboring 51 genes each one (Supplementary Table S5).</p>
<p id="Par80">From the SVR that overlapped positional candidate genes, there were in total 7 regions of inversions, from which the named INVR549 (BTA15:78.97–78.98 Mb) was the most annotated harboring 44 genes, followed by INVR131 (BTA3:20.94–22.58 Mb) that harbored 24 positional candidate genes (Supplementary Table S5). All the SVRs carrying known genes were distributed in all chromosomes, and the largest number of annotated genes was harbored by BTA21 (<em>N</em> = 116), followed by BTA5, 2, 13, and 18 with 109, 105, 100, and 78 annotated genes, respectively.</p>
<p id="Par81">PANTHER overrepresentation analysis through the Binomial test and Bonferroni correction method showed that from the 1,164 positional candidate genes that overlapped the most frequent SVRs, 695 were significantly enriched for three gene ontology categories and one biochemical pathway (Fig. <a href="#Fig13" class="usa-link">13</a>). The three gene ontologies were biological process (BP), with 632 genes grouped into 15 significant GO terms (Supplementary Table S6), molecular function (MF) with 139 genes grouped into 4 significant GO terms (Supplementary Table S7), and cellular component (CC) with 236 genes grouped into 2 significant GO terms (Supplementary Table S8). In addition, 26 genes were significantly associated with a biochemical pathway (Supplementary Table S9).</p>
<figure class="fig xbox font-sm" id="Fig13"><h4 class="obj_head">Fig. 13.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365053_41598_2025_14139_Fig13_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5894/12365053/0e84fb1c78e0/41598_2025_14139_Fig13_HTML.jpg" loading="lazy" id="d33e2613" height="369" width="669" alt="Fig. 13"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig13/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Results from PANTHER enrichment analysis. Number of Biological Process, Cellular Components, Molecular Function, and Biochemical Pathways (<strong>a</strong>) and the number of genes associated with GO terms (<strong>b</strong>).</p></figcaption></figure><p id="Par83">Regarding biological-related GO terms, most of the enriched genes were significantly associated with cellular process (GO:0009987, <em>N</em> = 49), followed by biological regulation (GO:0065007, <em>N</em> = 386), regulation of biological process (GO:0050789 <em>N</em> = 369), regulation of cellular process (GO:0050794, <em>N</em> = 352), and response to stimulus (GO:0050896, <em>N</em> = 236). Enriched genes were also associated with biological processes related to cell signaling and response, including cell communication (GO:0007154, <em>N</em> = 195), signaling (GO:0023052, <em>N</em> = 194), signaling transduction (GO:0007165, <em>N</em> = 184), G protein-coupled receptor signaling pathway (GO:0007186, <em>N</em> = 72), nervous system process (GO:0050877, <em>N</em> = 61), sensory perception of chemical stimulus (GO:0007606, <em>N</em> = 57), sensory perception (GO:0007600, <em>N</em> = 57), cellular response to stimulus (GO:0051716, <em>N</em> = 31), cellular response to endogenous stimulus (GO:0071495, <em>N</em> = 32), and response to endogenous stimulus (GO:0009719, <em>N</em> = 31) (Supplementary Table S6).</p>
<p id="Par84">For Molecular Function, 63 genes were significantly associated with odorant binding (GO:0005549), 129 to transmembrane signaling receptor activity (GO:0004888), 130 to signaling receptor activity (GO:0038023), and 139 to molecular transducer activity (GO:0060089) (Supplementary Table S7). With respect to cellular components, 32 genes were significantly associated with the organelle sub-compartment (GO:0031984) and 230 with the membrane (GO:0016020) (Supplementary Table S8). Finally, a total of 26 genes were significantly associated with the phospholipid metabolism pathway (Supplementary Table S9). It is outstanding that from the enriched set, there are several genes (91 genes) belonging to the olfactory receptor (OR) family, which overlapped SVRs located on BTA3, 5, 7, 15, 23, and 29. Most of these OR genes are harbored by regions of inversions, where 39 OR genes are located in the region named INVR549 (BTA15; 78.78–80.14 Mb - Supplementary Table S5) that is shared by 76 bulls and overlaps a health type QTL (liver abscess, Supplementary Table S4). The regions INVR131 (BTA3; 20.94–22.58 Mb - Supplementary Table S5) and INVR240 (BTA5:58.73–58.94 Mb; Supplementary Table S5) harbored 2 OR genes each one, and are shared by 83 and 86 bulls, respectively. Additionally, INVR131 overlaps a milk-type QTL related to milk yield (Supplementary Table S4). The other enriched OR genes are mainly harbored by copy number regions of mixed events (deletions and duplications), such as the region CNVR1946 (BTA7:8.71–10.10 Mb; Supplementary Table S5), which harbored 19 OR genes. This region is common for 101 bulls and overlaps a production type QTL related to residual feed intake (Supplementary Table S4). Similarly, 15 OR genes are located in the region CNVR5606 (BTA29:27.14–27.60 Mb; Supplementary Table S5), are observed in 87 bulls, and 7 OR genes are carried by the region CNVR4958 (BTA23:29.21–29.32 Mb; Supplementary Table S5) common for 76 bulls. Other 5 enriched OR genes overlapped the mixed CNV region CNVR1465 (BTA5:58.63–58.91 Mb; Supplementary Table S5), which is located near a reproduction QTL related to multiple births and that is common for 102 bulls. The <em>OR5M10C</em> gene is located in the region CNVR4931 (BTA23:17.75–22.14 Mb; Supplementary Table S5), which overlaps two reproduction QTLs (multiple birth and interval to first estrus after calving; Supplementary Table S4), and one health type QTL related to health complement hemolytic activity (Supplementary Table S4). Finally, the <em>OR5AE3</em> gene is harbored by a deletion region named CNVR2043 (BTA7:40.99–41.03 Mb), which is common for 82 bulls.</p>
<p id="Par85">Some authors have reported significant associations of SNPs located in the vicinity of OR genes with economically important traits in beef cattle. For example, the <em>OR5M10</em> gene, which was observed in this study within the region INVR549, was reported by Higgins et al.<sup><a href="#CR54" class="usa-link" aria-describedby="CR54">54</a></sup> near the SNP rs136789347, which was significantly associated with the average daily gain (ADG) in Irish beef cattle breeds. Here, the <em>OR5M10</em> gene was significantly enriched for 12 of the 15 GO terms related to Biological Processes and for all GO terms related to Molecular Functions (Supplementary Table S6 and S7). Seabury et al.<sup><a href="#CR55" class="usa-link" aria-describedby="CR55">55</a></sup> also reported significant associations of SNPs surrounding OR genes with average daily gain and residual feed intake in three different U.S. beef cattle breeds. In Nellore cattle, Olivieri et al.<sup><a href="#CR56" class="usa-link" aria-describedby="CR56">56</a></sup> found 3 OR genes harbored by a genomic region located on BTA15 that explained more than 1% of additive genetic variance of dry matter intake (DMI).</p>
<p id="Par86">The mixed region CNVR1465 (BTA5:58.63–58.91 Mb) carried the genes <em>OR6C1</em>, <em>OR6C1N</em>, and <em>OR6C1S</em>, which were also found by Santana et al.<sup><a href="#CR14" class="usa-link" aria-describedby="CR14">14</a></sup> overlapping CNV regions significantly associated with feed conversion rate in Nellore cattle, in a genotyped-based study. Those genes were significantly enriched for the MF GO terms transmembrane signaling receptor activity (GO:0004888), signaling receptor activity (GO:0038023), and molecular transducer activity (GO:0060089) (Supplementary Table S7). The mentioned mixed CNV region overlapped a reproduction QTL related to multiple birth traits. Although most of the studies found in the literature reveal the functional role of the OR gene family on feed efficiency-related traits, some genes belonging to the gene family have also been associated with reproductive performance in cattle<sup><a href="#CR57" class="usa-link" aria-describedby="CR57">57</a></sup>. The OR genes are also involved in sperm chemotaxis and the development of female germ cells and oocytes<sup><a href="#CR58" class="usa-link" aria-describedby="CR58">58</a></sup>.</p>
<p id="Par87">Similarly, the gene set <em>OR7A112</em>, <em>OR7A78</em>, <em>OR7A95</em>, <em>OR7A97</em>, and <em>OR7A99</em> harbored by the mixed region CNVR1946 were also reported by Braga et al.<sup><a href="#CR20" class="usa-link" aria-describedby="CR20">20</a></sup> overlapping CNVRs in Brazilian dairy Gyr cattle, from SNP and WGS data. In this study, those genes were significantly associated with 8 of the 15 BP-related GO terms, 3 MF-related GO terms, and the CC GO term membrane (GO:0016020). Among those genes, the <em>OR795</em> gene was also reported by Rafter et al.<sup><a href="#CR59" class="usa-link" aria-describedby="CR59">59</a></sup> as a flanked gene into a deletion region significantly associated with meat carcass traits in Charolais, Holstein, and Limousin cattle.</p>
<p id="Par88">The OR genes comprise the largest superfamily in the mammalian genome. They belong to the G-protein-coupled chemoreceptors (GPCRs) and are essentially expressed on countless olfactory sensory neurons within the nasal cavity<sup><a href="#CR60" class="usa-link" aria-describedby="CR60">60</a>,<a href="#CR61" class="usa-link" aria-describedby="CR61">61</a></sup>. These protein-coding genes play roles in the reception of a wide range of chemical molecules in the environment<sup><a href="#CR62" class="usa-link" aria-describedby="CR62">62</a></sup>. Odorant mechanisms are essential for animals as they send signals to the brain in order to locate feed sources, danger, or potential mating<sup><a href="#CR62" class="usa-link" aria-describedby="CR62">62</a></sup>. Besides the nasal cavity, olfactory receptors can be expressed in other organs where they bind to nutrients and metabolites to trigger physiological responses such as changes in gut motility, ventilation rate, and cellular migration<sup><a href="#CR61" class="usa-link" aria-describedby="CR61">61</a></sup>. The OR activity is mediated by the central nervous system and is influenced by hormonal, nutritional, and metabolic mechanisms regulating energy balance that trigger behaviors such as feed searching and intake<sup><a href="#CR63" class="usa-link" aria-describedby="CR63">63</a>–<a href="#CR65" class="usa-link" aria-describedby="CR65">65</a></sup>.</p>
<p id="Par89">The structural conformation of the OR gene family has been well-studied in several vertebrate species. There is evidence showing that the OR gene family has evolved due to duplication mechanisms, which has guaranteed their maintenance and survival in the vertebrates’ genomes<sup><a href="#CR1" class="usa-link" aria-describedby="CR1">1</a>,<a href="#CR66" class="usa-link" aria-describedby="CR66">66</a></sup>. Lee et al.<sup><a href="#CR62" class="usa-link" aria-describedby="CR62">62</a></sup> observed a large variation in OR gene copy number in Korean Holstein and Angus cattle. The authors found via PCR that 40% of the tested OR loci exhibited CNVs, which suggests that copy number variations of OR genes in cattle are highly frequent and that their genetic diversity confers differences in olfaction capacity and, then, in fitness. Although duplications have been described as the main mechanism driving the maintenance of the OR genes, it results particularly interesting that regions of inversions detected in this study harbored such a great number of genes belonging to the olfactory receptor superfamily. Due to the functional relevance of OR genes for adaptation and fitness, it is plausible that under selection pressure, the desirable alleles for OR genes were protected against recombination for being widely located on regions of inversion events, thus playing important roles in the adaptation of Nellore to environmental conditions.</p>
<p id="Par90">Another important gene family found overlapping the most frequent SVR is the B defensin gene family (<em>DEFB</em>), from which 6 genes (<em>DEFB1</em>,<em> DEFB10</em>,<em> DEFB119</em>,<em> DEFB124</em>,<em> DEFB13</em>, and <em>DEFB7</em>) were significantly associated with 11 biological processes GO terms, mainly related to cellular communication, signaling and response to stimulus (Supplementary Table S6). Defensins are protein-coding genes that have key functions in the innate immune system of multicellular organisms. Besides playing important roles in antimicrobial activity, these molecules act in diverse biological mechanisms such as cell signaling, ion channel transport, toxic functions, and enzyme inhibitor activities in animals<sup><a href="#CR67" class="usa-link" aria-describedby="CR67">67</a></sup>. As well as olfactory receptor genes, the immunity-related genes are known to be more likely duplicated than other genes. Our results are in concordance with those reported by Butty et al.<sup><a href="#CR21" class="usa-link" aria-describedby="CR21">21</a></sup>. These authors observed members of the <em>DEFB</em> family in their predicted CNV regions in a Holstein population from genotypes and WGS data.</p>
<p id="Par91">Similarly, in a population of Holstein and Jersey cattle, Chen et al.<sup><a href="#CR45" class="usa-link" aria-describedby="CR45">45</a></sup> identified that genes belonging to expanded gene families, such as <em>DEFB</em>, were more associated with SV regions than other genes. Particularly, the gene <em>DEFB1</em>, which was found within the mixed copy number region named CNVR5333 (27:6.18–7.28 Mb), was also reported in the two mentioned studies. The <em>DEFB1</em> was significantly related to GO term cellular response to endogenous stimulus, signaling receptor activity, and molecular transducer activity. Important genes in the vicinity of structural variant regions have been previously reported in the literature for their association with economically important phenotypes. For example, members of the secretory phospholipase A2 family of genes (<em>PLA2G2A</em>, <em>PLA2G2D1</em>, <em>PLA2G2E</em>, and <em>PLA2G5</em>), which were harbored by the mixed copy number region CNVR717 (BTA2:132.53-133.81 Mb), were also found into CNV regions significantly associated to meat carcass traits in Nellore cattle by Berton et al.<sup><a href="#CR68" class="usa-link" aria-describedby="CR68">68</a></sup>. On the other hand, four members of Adhesion G Protein-Coupled Receptors (<em>ADGRF1</em>, <em>ADGRF2</em>, <em>ADGRF4</em>, and <em>ADGRF5</em>), were harbored by the mixed region CNVR4931 (BTA23:17.75–22.14 Mb), which were linked to one health QTL related to complement hemolytic activity, and to two reproduction QTLs related to multiple birth, and interval to first estrus after calving (Supplementary Table S4). Those genes were previously reported by Li et al.<sup><a href="#CR69" class="usa-link" aria-describedby="CR69">69</a></sup> for being significantly associated with feed efficiency indicator traits in Canadian crossbred cattle, such as residual feed intake, daily dry matter intake, average daily gain, and metabolic daily gain.</p>
<p id="Par92">A large set of genes belonging to the zinc fingers binding protein family were also found near the structural variant regions. Particularly the mixed regions CNVR4338 (BTA18:57.18–60.17 Mb) and CNVR4347 (BTA18:61.81–63.40 Mb) harbored various members of that gene family, such as the genes <em>ZNF524</em>,<em> ZNF581</em>, and <em>ZNF677</em>, which were previously associated with dry matter intake, average daily gain, and feed efficiency in an experimental population of Nellore cattle by Olivieri et al.<sup><a href="#CR56" class="usa-link" aria-describedby="CR56">56</a></sup>. Zinc finger proteins usually regulate gene expression by binding to specific DNA sequences via their DNA-binding zinc finger domains and recruiting cofactors that mediate chromatin remodeling and transcriptional repression or activation. In a transcriptomic study in <em>Bos indicus</em> cattle, Samuel &amp; Dinka<sup><a href="#CR70" class="usa-link" aria-describedby="CR70">70</a></sup> observed that ZNF genes are likely involved in the regulation of the expression of olfactory receptor genes. Thus, ZNF genes may also play roles in cell signaling and communication through the sense of olfaction, which is essential for environmental adaption. The genes identified surrounding structural variants regions are more involved in functions related to biological and cell regulation, response to stimulus, and cell communication and signaling. Those functions are essential drivers of the environmental adaptation of genomes. The reported literature in Nellore cattle supports that most of the described genes are potential candidates for feed efficiency indicator traits, which reflect the biochemical mechanisms in which they are involved that have led to improved fitness.</p></section><section id="Sec19"><h3 class="pmc_sec_title">Functional insights from genes related to high-frequency SVRs</h3>
<p id="Par93">Adaptation to tropical regions represents one of the most significant challenges in cattle breeds, requiring a coordinated response to persistent heat stress, seasonal nutritional limitations, and pathogen exposure. By analyzing the 532 most frequent SVRs in Nellore cattle, we identified genomic regions enriched for genes and QTLs associated with thermotolerance, immune function, metabolic regulation, and tissue resilience, all key attributes for tropical adaptation (Supplementary Table S4 and S10).</p>
<p id="Par94">The ability to handle elevated ambient temperatures is one of the most important attributes required for cattle raised in tropical regions. We have found SVRs encompassing genes related to cellular protection and heat dissipation, which are important traits for thermal tolerance (Supplementary Table S10). Notably, <em>HSP90AB1</em> (BTA 23, 17.75–17.76 Mb), a heat shock protein essential for maintaining proteostasis under thermal stress, has been directly associated with enhanced thermotolerance in cattle populations<sup><a href="#CR71" class="usa-link" aria-describedby="CR71">71</a>,<a href="#CR72" class="usa-link" aria-describedby="CR72">72</a></sup>. Similarly, <em>PRLR</em> (BTA 20, 38.97–39.10 Mb), particularly its slick hair variants, promotes shorter hair coats that improve heat loss through convection and radiation, an adaptation widely documented in tropical breeds<sup><a href="#CR73" class="usa-link" aria-describedby="CR73">73</a>,<a href="#CR74" class="usa-link" aria-describedby="CR74">74</a></sup>. Genes such as <em>THRB</em> (BTA 27, 41.46–41.90 Mb) and <em>MC4R</em> (BTA 24, 59.16–59.17 Mb), which modulate metabolic rate and appetite, respectively, play a pivotal role in limiting endogenous heat production during thermal stress<sup><a href="#CR75" class="usa-link" aria-describedby="CR75">75</a>–<a href="#CR77" class="usa-link" aria-describedby="CR77">77</a></sup>while ion channel genes like <em>KCNMA1</em> (BTA 28, 32.62–33.39 Mb) may facilitate vasodilation and peripheral cooling<sup><a href="#CR78" class="usa-link" aria-describedby="CR78">78</a></sup>.</p>
<p id="Par95">The capacity to mitigate oxidative damage, often a by-product of heat exposure, appears to be influenced by SVRs mapping the genes <em>EGLN1</em> (BTA 28, 4.09–4.15 Mb), a key regulator of the hypoxia-inducible factor (HIF) pathway, as well as <em>NFE2L2</em> (BTA 2, 19.62–19.65 Mb), <em>JUN</em> (BTA 3, 87.27–87.27 Mb), and <em>ATF2</em> (BTA 2, 21.74–21.79 Mb) all of which are involved in activating antioxidant and cellular stress responses<sup><a href="#CR78" class="usa-link" aria-describedby="CR78">78</a>–<a href="#CR80" class="usa-link" aria-describedby="CR80">80</a></sup>. Additional support for cutaneous resilience in hot and pathogen-rich environments may be conferred by <em>DEFB1</em> (BTA 27, 6.72–6.72 Mb), while energy-sensing mechanisms mediated by <em>PRKAA1</em> (AMPK) (BTA 20, 33.67–33.7 Mb) enable rapid metabolic adaptation under heat load<sup><a href="#CR81" class="usa-link" aria-describedby="CR81">81</a>,<a href="#CR82" class="usa-link" aria-describedby="CR82">82</a></sup>. In addition to thermal regulation, SVR surround immune competence genes are also involved in the expression of traits important for adaptation to tropical production systems. SVRs overlapping the <em>BoLA</em> complex, a cornerstone of adaptive immunity, have been repeatedly linked to resistance against mastitis, tick infestation, and other tropical diseases<sup><a href="#CR83" class="usa-link" aria-describedby="CR83">83</a></sup>. A set of innate immune genes (<em>ALCAM</em>, <em>SYK</em>, <em>NLRP2</em>, and <em>FCAR</em>) was identified within SVRs, together with immunoregulatory loci such as <em>LAIR1</em>, <em>SIGLEC10</em>, <em>PILRA</em>, and <em>TNIP3</em>. These immune functional gene sets contribute to a balanced immune response by mitigating chronic inflammation in challenging environments<sup><a href="#CR84" class="usa-link" aria-describedby="CR84">84</a>,<a href="#CR85" class="usa-link" aria-describedby="CR85">85</a></sup>. Genes regulating hematopoiesis and lymphocyte development, including <em>GATA2</em>, <em>SP110</em>, <em>IL7R</em>, <em>LIFR</em>, and <em>OSMR</em>, were also found<sup><a href="#CR86" class="usa-link" aria-describedby="CR86">86</a>–<a href="#CR88" class="usa-link" aria-describedby="CR88">88</a></sup>. In parallel, <em>SLC11A2</em>, which limits iron availability to intracellular pathogens, suggests the importance of nutritional immunity for disease resistance<sup><a href="#CR89" class="usa-link" aria-describedby="CR89">89</a></sup>.</p>
<p id="Par96">Tropical forage systems often present marked seasonal fluctuations in nutrient content and availability. In this context, SVRs were enriched for genes supporting metabolic flexibility and nutritional efficiency. <em>MC4R</em> and <em>PRKAA1</em> play a central role by regulating feed intake and energy homeostasis<sup><a href="#CR76" class="usa-link" aria-describedby="CR76">76</a></sup>. At the cellular level, nutrient sensing and growth coordination are enabled by <em>IGF1R</em> (BTA 21, 7.78–8.08 Mb), <em>RPTOR</em> (BTA 19, 51.69–52.01 Mb), and <em>RICTOR</em> (BTA 20, 35.36–35.5 Mb), key components of anabolic signaling pathways<sup><a href="#CR90" class="usa-link" aria-describedby="CR90">90</a>,<a href="#CR91" class="usa-link" aria-describedby="CR91">91</a></sup>. Genes such as <em>SREBF2</em> (BTA 5, 112.83–112.89 Mb), <em>PCK1</em> (BTA 13, 58.61–58.65 Mb) and <em>GPD1</em> (BTA 22, 6.59–6.69 Mb) support lipid and glucose metabolism under nutritional stress<sup><a href="#CR92" class="usa-link" aria-describedby="CR92">92</a>–<a href="#CR94" class="usa-link" aria-describedby="CR94">94</a></sup>while detoxification enzymes encoded by <em>ALDH4A1</em> (BTA 2, 133.76–133.79 Mb), <em>AOX1</em> (BTA 2, 89.1–89.17 Mb) and <em>AHR</em> (BTA 4, 25.81–25.86 Mb) may facilitate adaptation to bioactive compounds found in diverse tropical forages<sup><a href="#CR95" class="usa-link" aria-describedby="CR95">95</a></sup>.</p>
<p id="Par97">Transcription factors such as <em>MORC3</em>, <em>ETS2</em>, <em>JUN</em>, <em>GATA2</em>, and <em>NFE2L2</em>, which are located within SVRs (Supplementary Table S10), regulate the expression of gene sets activated by environmental stressors<sup><a href="#CR96" class="usa-link" aria-describedby="CR96">96</a>–<a href="#CR99" class="usa-link" aria-describedby="CR99">99</a></sup>. Epigenetic regulators, such as <em>MYSM1</em> (BTA 3, 87.35–87.39 Mb), transcriptional repressors like <em>SIN3A</em> (BTA 21, 33.38–33.45 Mb), and stress regulators like <em>PML</em> (BTA 21, 34.58–34.63 Mb), may further fine-tune immune and metabolic responses<sup><a href="#CR100" class="usa-link" aria-describedby="CR100">100</a>,<a href="#CR101" class="usa-link" aria-describedby="CR101">101</a></sup>. Moreover, <em>NR1D2</em> (<em>REV-ERBβ</em>) (BTA 27, 41.99–42.01 Mb), a circadian regulator, suggests that alignment between metabolic activity and environmental light-dark cycles may also contribute to tropical resilience<sup><a href="#CR102" class="usa-link" aria-describedby="CR102">102</a></sup>.</p>
<p id="Par98">The maintenance of cellular integrity under chronic environmental stress is another key adaptation. SVRs overlapping <em>XRCC6</em>, <em>BLM</em>, <em>FANCI</em>, and <em>RUVBL1</em> indicate active DNA repair mechanisms that prevent genomic instability under oxidative and thermal stimulus<sup><a href="#CR103" class="usa-link" aria-describedby="CR103">103</a>–<a href="#CR106" class="usa-link" aria-describedby="CR106">106</a></sup>. Tissue health is further supported by autophagy regulators (<em>WDFY3</em> and <em>OMA1</em>) and anti-apoptotic factors such as <em>BCL2L1</em>, while redox regulators like <em>TXNIP</em>, <em>NOX4</em>, and <em>SESN3</em> ensure intracellular oxidative balance<sup><a href="#CR107" class="usa-link" aria-describedby="CR107">107</a>–<a href="#CR112" class="usa-link" aria-describedby="CR112">112</a></sup>.</p>
<p id="Par99">The structural integrity and plasticity of tissues, especially skin and connective tissue, are maintained by SVRs containing <em>FSTL1</em>, <em>ADAMTS2</em>, <em>COL15A1</em>, <em>HMCN1</em>, <em>BMP7</em>, <em>XYLT1</em>, <em>FAT4</em>, and <em>CTNNA1</em>. These genes contribute to extracellular matrix remodeling and efficient wound repair, enhancing resistance to ectoparasites and mechanical injury in extensive grazing systems<sup><a href="#CR111" class="usa-link" aria-describedby="CR111">111</a>–<a href="#CR115" class="usa-link" aria-describedby="CR115">115</a></sup>. The results demonstrate that SVRs in Nellore cattle capture functionally important genomic regions related to heat resilience, immune defense, metabolic adaptability, and tissue robustness. Considering this layer of structural genomic information into tropical breeding programs holds substantial promise for accelerating the development of resilient, high-performing animals suited to increasingly demanding environments.</p></section></section><section id="Sec20"><h2 class="pmc_sec_title">Conclusion</h2>
<p id="Par100">This study provides a comprehensive assessment of structural variation in the Nellore cattle genome, using advanced WGS algorithms to detect deletions, duplications, and inversions. We identified 215,031 structural variants, including 183,032 deletions, 14,013 duplications, and 17,986 inversions, highlighting the widespread contribution of mobile elements to the genomic architecture of the Nellore population in Brazil. Notably, we are the first to report inversion events in Nellore cattle, emphasizing the influence of mobile elements on genomic structure. Among these, 3,752 structural variants were present in over 5% of bulls, and 532 structural variants were common to over 50% of the key ancestors analyzed. Notably, 97% of these SVRs mapped to regions of copy number variation (CNVR), and 3% were inversions (INVR). Among the CNVRs, 90% were deletions, 3% were duplications, and 7% contained both.</p>
<p id="Par101">Functional annotation of these high-frequency SVRs identified 1,160 positional candidate genes overlapping QTLs involved in thermotolerance, immune regulation, metabolic flexibility, and tissue integrity, essential components of adaptation to tropical environments. The integration of such functionally relevant variants emphasizes the importance of structural genomic diversity in shaping adaptive phenotypes under environmental stressors such as heat, nutrient fluctuation, and pathogen pressure. These findings enhance our understanding of the genetic mechanisms behind Nellore cattle’s environmental adaptation due to changes in functionally significant genes. The inclusion of SV-based information in genomic evaluations may enhance the accuracy of selection for environmentally adaptive traits, complementing SNP-based analyses and offering novel opportunities for precision breeding in beef cattle.</p></section><section id="Sec21"><h2 class="pmc_sec_title">Supplementary Information</h2>
<p>Below is the link to the electronic supplementary material.</p>
<section class="sm xbox font-sm" id="MOESM1"><div class="media p"><div class="caption">
<a href="/articles/instance/12365053/bin/41598_2025_14139_MOESM1_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 1</a><sup> (1.1MB, docx) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>The authors acknowledge the São Paulo Research Foundation (FAPESP) for the financial support grants number #2009/16118-5, #2017/10630-2, #2018/20026-8 and #2022/11852-7.</p></section><section id="notes1"><h2 class="pmc_sec_title">Author contributions</h2>
<p>Conceptualization: N.A.M.G., L.F.S.F., L.M.A. and L.F.M.M.; methodology: N.A.M.G., G.V., L.F.S.F. and G.A.F.J.; resources: L.G.A.; data curation: N.A.M.G., G.V. G.A.F.J. and L.F.M.M.; writing original draft preparation: N.A.M.G.; writing review and editing: L.F.M.M. and L.M.A.; visualization: L.F.M.M. and L.M.A.; supervision: R.C. and L.G.A; project administration: L.G.A. All authors have read and agreed to the published version of the manuscript.</p></section><section id="notes2"><h2 class="pmc_sec_title">Funding</h2>
<p>This research was funded by São Paulo Research Foundation (FAPESP) grant numbers #2009/16118-5, #2017/10630-2, #2018/20026-8, and #2022/11852-7. Coordenação de Aperfeiçoamento de Pessoal de Nível Superior – Brazil (CAPES), finance code 001.</p></section><section id="notes3"><h2 class="pmc_sec_title">Data availability</h2>
<p>The data that support the findings of this study are available from Gensys breeding program (https://gensys.com.br/), but restrictions apply to the availability of these data, which were used under license for the current study, and so are not publicly available. Data are however available from the authors (Dr. Lucia G. Albuquerque email: galvao.albuquerque@unesp.br) upon reasonable request and with permission of Gensys breeding program.</p></section><section id="notes4"><h2 class="pmc_sec_title">Declarations</h2>
<section id="FPar1"><h3 class="pmc_sec_title">Competing interests</h3>
<p id="Par102">The authors declare no competing interests.</p></section><section id="FPar2"><h3 class="pmc_sec_title">Ethics declarations</h3>
<p id="Par103">The animal procedures and data sampling presented in this study were approved and performed following the Animal Care and Ethical Committee recommendations of the São Paulo State University (UNESP), School of Agricultural and Veterinary Science (protocol number 18.340/16). All the data sampling was performed following the CEUA/ FCAV-UNESP guidelines and regulations in agreement with the Regulations for the Administration of Affairs Concerning Experimental Animals (Ministry of Science and Technology, Brazil). In addition, we confirmed the statement that the study was conducted following the ARRIVE guidelines.</p></section></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="fn1">
<p><strong>Publisher’s note</strong></p>
<p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p>
</div></div></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Bickhart, D. M. &amp; Liu, G. E. The challenges and importance of structural variation detection in livestock. <em>Front. Genet.</em><strong>5</strong>, 77142 (2014).</cite> [<a href="https://doi.org/10.3389/fgene.2014.00037" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3927395/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24600474/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bickhart,%20D.%20M.%20&amp;%20Liu,%20G.%20E.%20The%20challenges%20and%20importance%20of%20structural%20variation%20detection%20in%20livestock.%20Front.%20Genet.5,%2077142%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>Fernandes Júnior, G. A. et al. Whole-genome sequencing provides new insights into genetic mechanisms of tropical adaptation in Nellore (Bos primigenius indicus). <em>Scientific Reports</em> 10, 1–7 (2020). (2020).</cite> [<a href="https://doi.org/10.1038/s41598-020-66272-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7287098/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32523018/" class="usa-link">PubMed</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>Ho, S. S., Urban, A. E. &amp; Mills, R. E. Structural variation in the sequencing era. <em>Nature Reviews Genetics</em> 21, 171–189 (2020). (2019).</cite> [<a href="https://doi.org/10.1038/s41576-019-0180-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7402362/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31729472/" class="usa-link">PubMed</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Alkan, C., Coe, B. P. &amp; Eichler, E. E. Genome structural variation discovery and genotyping. <em>Nature Reviews Genetics</em> 12, 363–376 (2011). (2011).</cite> [<a href="https://doi.org/10.1038/nrg2958" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4108431/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21358748/" class="usa-link">PubMed</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Escaramís, G., Docampo, E. &amp; Rabionet, R. A decade of structural variants: description, history and methods to detect structural variation. <em>Brief. Funct. Genomics</em>. <strong>14</strong>, 305–314 (2015).
</cite> [<a href="https://doi.org/10.1093/bfgp/elv014" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25877305/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Escaram%C3%ADs,%20G.,%20Docampo,%20E.%20&amp;%20Rabionet,%20R.%20A%20decade%20of%20structural%20variants:%20description,%20history%20and%20methods%20to%20detect%20structural%20variation.%20Brief.%20Funct.%20Genomics.%2014,%20305%E2%80%93314%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Zhang, F., Gu, W., Hurles, M. E. &amp; Lupski, J. R. Copy number variation in human health, disease, and evolution. <em>Annu. Rev. Genomics Hum. Genet.</em><strong>10</strong>, 451–481 (2009).
</cite> [<a href="https://doi.org/10.1146/annurev.genom.9.081307.164217" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4472309/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19715442/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhang,%20F.,%20Gu,%20W.,%20Hurles,%20M.%20E.%20&amp;%20Lupski,%20J.%20R.%20Copy%20number%20variation%20in%20human%20health,%20disease,%20and%20evolution.%20Annu.%20Rev.%20Genomics%20Hum.%20Genet.10,%20451%E2%80%93481%20(2009)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Freeman, J. L. et al. Copy number variation: new insights in genome diversity. <em>Genome Res.</em><strong>16</strong>, 949–961 (2006).
</cite> [<a href="https://doi.org/10.1101/gr.3677206" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16809666/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Freeman,%20J.%20L.%20et%20al.%20Copy%20number%20variation:%20new%20insights%20in%20genome%20diversity.%20Genome%20Res.16,%20949%E2%80%93961%20(2006)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>Couldrey, C. et al. Detection and assessment of copy number variation using PacBio long-read and illumina sequencing in new Zealand dairy cattle. <em>J. Dairy. Sci.</em><strong>100</strong>, 5472–5478 (2017).
</cite> [<a href="https://doi.org/10.3168/jds.2016-12199" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28456410/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Couldrey,%20C.%20et%20al.%20Detection%20and%20assessment%20of%20copy%20number%20variation%20using%20PacBio%20long-read%20and%20illumina%20sequencing%20in%20new%20Zealand%20dairy%20cattle.%20J.%20Dairy.%20Sci.100,%205472%E2%80%935478%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>Conrad, D. F. et al. Origins and functional impact of copy number variation in the human genome. <em>Nature 2009 464:7289</em> 464, 704–712 (2010).</cite> [<a href="https://doi.org/10.1038/nature08516" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3330748/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19812545/" class="usa-link">PubMed</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>Bickhart, D. M. et al. Copy number variation of individual cattle genomes using next-generation sequencing. <em>Genome Res.</em><strong>22</strong>, 778–790 (2012).
</cite> [<a href="https://doi.org/10.1101/gr.133967.111" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3317159/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22300768/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bickhart,%20D.%20M.%20et%20al.%20Copy%20number%20variation%20of%20individual%20cattle%20genomes%20using%20next-generation%20sequencing.%20Genome%20Res.22,%20778%E2%80%93790%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Zhao, M., Wang, Q., Wang, Q., Jia, P. &amp; Zhao, Z. Computational tools for copy number variation (CNV) detection using next-generation sequencing data: features and perspectives. <em>BMC Bioinform.</em><strong>14</strong>, 1–16 (2013).</cite> [<a href="https://doi.org/10.1186/1471-2105-14-S11-S1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3846878/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24564169/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhao,%20M.,%20Wang,%20Q.,%20Wang,%20Q.,%20Jia,%20P.%20&amp;%20Zhao,%20Z.%20Computational%20tools%20for%20copy%20number%20variation%20(CNV)%20detection%20using%20next-generation%20sequencing%20data:%20features%20and%20perspectives.%20BMC%20Bioinform.14,%201%E2%80%9316%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Lemos, M. V. A. et al. Association study between copy number variation and beef fatty acid profile of Nellore cattle. <em>J. Appl. Genet.</em><strong>59</strong>, 203–223 (2018).
</cite> [<a href="https://doi.org/10.1007/s13353-018-0436-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29520708/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lemos,%20M.%20V.%20A.%20et%20al.%20Association%20study%20between%20copy%20number%20variation%20and%20beef%20fatty%20acid%20profile%20of%20Nellore%20cattle.%20J.%20Appl.%20Genet.59,%20203%E2%80%93223%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>Silva, V. H. et al. Genome-Wide detection of CNVs and their association with meat tenderness in Nelore cattle. <em>PLoS One</em>. <strong>11</strong>, e0157711 (2016).
</cite> [<a href="https://doi.org/10.1371/journal.pone.0157711" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4922624/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27348523/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Silva,%20V.%20H.%20et%20al.%20Genome-Wide%20detection%20of%20CNVs%20and%20their%20association%20with%20meat%20tenderness%20in%20Nelore%20cattle.%20PLoS%20One.%2011,%20e0157711%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>Santana, M. H. A. et al. Copy number variations and genome-wide associations reveal putative genes and metabolic pathways involved with the feed conversion ratio in beef cattle. <em>J. Appl. Genet.</em><strong>57</strong>, 495–504 (2016).
</cite> [<a href="https://doi.org/10.1007/s13353-016-0344-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27001052/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Santana,%20M.%20H.%20A.%20et%20al.%20Copy%20number%20variations%20and%20genome-wide%20associations%20reveal%20putative%20genes%20and%20metabolic%20pathways%20involved%20with%20the%20feed%20conversion%20ratio%20in%20beef%20cattle.%20J.%20Appl.%20Genet.57,%20495%E2%80%93504%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>Zhou, Y. et al. Genome-wide CNV analysis reveals variants associated with growth traits in Bos indicus. <em>BMC Genom.</em><strong>17</strong>, 1–9 (2016).</cite> [<a href="https://doi.org/10.1186/s12864-016-2461-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4888316/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27245577/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhou,%20Y.%20et%20al.%20Genome-wide%20CNV%20analysis%20reveals%20variants%20associated%20with%20growth%20traits%20in%20Bos%20indicus.%20BMC%20Genom.17,%201%E2%80%939%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Chen, L., Pryce, J. E., Hayes, B. J. &amp; Daetwyler, H. D. Investigating the effect of imputed structural variants from Whole-Genome sequence on Genome-Wide association and genomic prediction in dairy cattle. <em>Animals</em><strong>11</strong>, 541 (2021).
</cite> [<a href="https://doi.org/10.3390/ani11020541" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7922624/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33669735/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chen,%20L.,%20Pryce,%20J.%20E.,%20Hayes,%20B.%20J.%20&amp;%20Daetwyler,%20H.%20D.%20Investigating%20the%20effect%20of%20imputed%20structural%20variants%20from%20Whole-Genome%20sequence%20on%20Genome-Wide%20association%20and%20genomic%20prediction%20in%20dairy%20cattle.%20Animals11,%20541%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>Durán Aguilar, M. et al. Genome-wide association study for milk somatic cell score in Holstein cattle using copy number variation as markers. <em>J. Anim. Breed. Genet.</em><strong>134</strong>, 49–59 (2017).
</cite> [<a href="https://doi.org/10.1111/jbg.12238" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27578198/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Dur%C3%A1n%20Aguilar,%20M.%20et%20al.%20Genome-wide%20association%20study%20for%20milk%20somatic%20cell%20score%20in%20Holstein%20cattle%20using%20copy%20number%20variation%20as%20markers.%20J.%20Anim.%20Breed.%20Genet.134,%2049%E2%80%9359%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Peripolli, E. et al. Assessment of copy number variants in three Brazilian locally adapted cattle breeds using whole-genome re-sequencing data. <em>Anim. Genet.</em><strong>54</strong>, 254–270 (2023).
</cite> [<a href="https://doi.org/10.1111/age.13298" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36740987/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Peripolli,%20E.%20et%20al.%20Assessment%20of%20copy%20number%20variants%20in%20three%20Brazilian%20locally%20adapted%20cattle%20breeds%20using%20whole-genome%20re-sequencing%20data.%20Anim.%20Genet.54,%20254%E2%80%93270%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Sun, T. et al. Whole genome sequencing of simmental cattle for SNP and CNV discovery. <em>BMC Genom.</em><strong>24</strong>, 1–11 (2023).</cite> [<a href="https://doi.org/10.1186/s12864-023-09248-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10077681/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37020271/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sun,%20T.%20et%20al.%20Whole%20genome%20sequencing%20of%20simmental%20cattle%20for%20SNP%20and%20CNV%20discovery.%20BMC%20Genom.24,%201%E2%80%9311%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>Braga, L. G. et al. Identification of copy number variations in the genome of dairy Gir cattle. <em>PLoS One</em>. <strong>18</strong>, e0284085 (2023).
</cite> [<a href="https://doi.org/10.1371/journal.pone.0284085" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10085049/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37036840/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Braga,%20L.%20G.%20et%20al.%20Identification%20of%20copy%20number%20variations%20in%20the%20genome%20of%20dairy%20Gir%20cattle.%20PLoS%20One.%2018,%20e0284085%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>Butty, A. M. et al. High confidence copy number variants identified in Holstein dairy cattle from whole genome sequence and genotype array data. <em>Sci. Rep.</em><strong>10</strong> (1), 10–11 (2020). (2020).
</cite> [<a href="https://doi.org/10.1038/s41598-020-64680-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7229195/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32415111/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Butty,%20A.%20M.%20et%20al.%20High%20confidence%20copy%20number%20variants%20identified%20in%20Holstein%20dairy%20cattle%20from%20whole%20genome%20sequence%20and%20genotype%20array%20data.%20Sci.%20Rep.10%20(1),%2010%E2%80%9311%20(2020).%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>Silva, J. M. et al. Genome-wide copy number variation (CNV) detection in Nelore cattle reveals highly frequent variants in genome regions harboring QTLs affecting production traits. <em>BMC Genom.</em><strong>17</strong>, 1–14 (2016).</cite> [<a href="https://doi.org/10.1186/s12864-016-2752-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4907077/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27297173/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Silva,%20J.%20M.%20et%20al.%20Genome-wide%20copy%20number%20variation%20(CNV)%20detection%20in%20Nelore%20cattle%20reveals%20highly%20frequent%20variants%20in%20genome%20regions%20harboring%20QTLs%20affecting%20production%20traits.%20BMC%20Genom.17,%201%E2%80%9314%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>Bickhart, D. M. et al. Diversity and population-genetic properties of copy number variations and multicopy genes in cattle. <em>DNA Res.</em><strong>23</strong>, 253–262 (2016).
</cite> [<a href="https://doi.org/10.1093/dnares/dsw013" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4909312/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27085184/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bickhart,%20D.%20M.%20et%20al.%20Diversity%20and%20population-genetic%20properties%20of%20copy%20number%20variations%20and%20multicopy%20genes%20in%20cattle.%20DNA%20Res.23,%20253%E2%80%93262%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>Boussaha, M. et al. Genome-Wide study of structural variants in bovine holstein, Montbéliarde and Normande dairy breeds. <em>PLoS One</em>. <strong>10</strong>, e0135931 (2015).
</cite> [<a href="https://doi.org/10.1371/journal.pone.0135931" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4552564/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26317361/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Boussaha,%20M.%20et%20al.%20Genome-Wide%20study%20of%20structural%20variants%20in%20bovine%20holstein,%20Montb%C3%A9liarde%20and%20Normande%20dairy%20breeds.%20PLoS%20One.%2010,%20e0135931%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>Li, H. et al. The sequence alignment/map format and samtools. <em>Bioinformatics</em><strong>25</strong>, 2078–2079 (2009).
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btp352" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2723002/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19505943/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li,%20H.%20et%20al.%20The%20sequence%20alignment/map%20format%20and%20samtools.%20Bioinformatics25,%202078%E2%80%932079%20(2009)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>Kim, D., Paggi, J. M., Park, C., Bennett, C. &amp; Salzberg, S. L. Graph-based genome alignment and genotyping with HISAT2 and HISAT-genotype. <em>Nat. Biotechnol.</em><strong>37</strong>, 907–915 (2019).
</cite> [<a href="https://doi.org/10.1038/s41587-019-0201-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7605509/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31375807/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kim,%20D.,%20Paggi,%20J.%20M.,%20Park,%20C.,%20Bennett,%20C.%20&amp;%20Salzberg,%20S.%20L.%20Graph-based%20genome%20alignment%20and%20genotyping%20with%20HISAT2%20and%20HISAT-genotype.%20Nat.%20Biotechnol.37,%20907%E2%80%93915%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Fernandes Júnior, G. A. et al. Imputation accuracy to whole-genome sequence in Nellore cattle. <em>Genet. Selection Evol.</em><strong>53</strong>, 1–10 (2021).</cite> [<a href="https://doi.org/10.1186/s12711-021-00622-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7953568/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33711929/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Fernandes%20J%C3%BAnior,%20G.%20A.%20et%20al.%20Imputation%20accuracy%20to%20whole-genome%20sequence%20in%20Nellore%20cattle.%20Genet.%20Selection%20Evol.53,%201%E2%80%9310%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Letaief, R. et al. Identification of copy number variation in French dairy and beef breeds using next-generation sequencing. <em>Genet. Sel. Evol.</em><strong>49</strong>, 77 (2017).
</cite> [<a href="https://doi.org/10.1186/s12711-017-0352-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5655909/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29065859/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Letaief,%20R.%20et%20al.%20Identification%20of%20copy%20number%20variation%20in%20French%20dairy%20and%20beef%20breeds%20using%20next-generation%20sequencing.%20Genet.%20Sel.%20Evol.49,%2077%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>Zarate, S. et al. Parliament2: accurate structural variant calling at scale. <em>Gigascience</em><strong>9</strong>, (2020).</cite> [<a href="https://doi.org/10.1093/gigascience/giaa145" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7751401/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33347570/" class="usa-link">PubMed</a>]</li>
<li id="CR30">
<span class="label">30.</span><cite>Abyzov, A., Urban, A. E., Snyder, M., Gerstein, M. &amp; CNVnator An approach to discover, genotype, and characterize typical and atypical CNVs from family and population genome sequencing. <em>Genome Res.</em><strong>21</strong>, 974–984 (2011).
</cite> [<a href="https://doi.org/10.1101/gr.114876.110" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3106330/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21324876/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Abyzov,%20A.,%20Urban,%20A.%20E.,%20Snyder,%20M.,%20Gerstein,%20M.%20&amp;%20CNVnator%20An%20approach%20to%20discover,%20genotype,%20and%20characterize%20typical%20and%20atypical%20CNVs%20from%20family%20and%20population%20genome%20sequencing.%20Genome%20Res.21,%20974%E2%80%93984%20(2011)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>Layer, R. M., Chiang, C., Quinlan, A. R. &amp; Hall, I. M. LUMPY: A probabilistic framework for structural variant discovery. <em>Genome Biol.</em><strong>15</strong>, 1–19 (2014).</cite> [<a href="https://doi.org/10.1186/gb-2014-15-6-r84" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4197822/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24970577/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Layer,%20R.%20M.,%20Chiang,%20C.,%20Quinlan,%20A.%20R.%20&amp;%20Hall,%20I.%20M.%20LUMPY:%20A%20probabilistic%20framework%20for%20structural%20variant%20discovery.%20Genome%20Biol.15,%201%E2%80%9319%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>Rausch, T. et al. DELLY: structural variant discovery by integrated paired-end and split-read analysis. <em>Bioinformatics</em><strong>28</strong>, i333–i339 (2012).
</cite> [<a href="https://doi.org/10.1093/bioinformatics/bts378" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3436805/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22962449/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rausch,%20T.%20et%20al.%20DELLY:%20structural%20variant%20discovery%20by%20integrated%20paired-end%20and%20split-read%20analysis.%20Bioinformatics28,%20i333%E2%80%93i339%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Danecek, P. et al. Twelve years of SAMtools and BCFtools. <em>Gigascience</em> 10, (2021).</cite> [<a href="https://doi.org/10.1093/gigascience/giab008" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7931819/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33590861/" class="usa-link">PubMed</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>Quinlan, A. R. &amp; Hall, I. M. BEDTools: a flexible suite of utilities for comparing genomic features. <em>Bioinformatics</em><strong>26</strong>, 841–842 (2010).
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btq033" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2832824/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20110278/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Quinlan,%20A.%20R.%20&amp;%20Hall,%20I.%20M.%20BEDTools:%20a%20flexible%20suite%20of%20utilities%20for%20comparing%20genomic%20features.%20Bioinformatics26,%20841%E2%80%93842%20(2010)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>Redon, R. et al. Global variation in copy number in the human genome. <em>Nature </em><strong>444</strong>, 444–454 (2006).</cite> [<a href="https://doi.org/10.1038/nature05329" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2669898/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17122850/" class="usa-link">PubMed</a>]</li>
<li id="CR36">
<span class="label">36.</span><cite>Wickham, H. et al. Welcome to the tidyverse. <em>J. Open. Source Softw.</em><strong>4</strong>, 1686 (2019).</cite> [<a href="https://scholar.google.com/scholar_lookup?Wickham,%20H.%20et%20al.%20Welcome%20to%20the%20tidyverse.%20J.%20Open.%20Source%20Softw.4,%201686%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR37">
<span class="label">37.</span><cite>Peterson, B. G., Carl, P. &amp; PerformanceAnalytics Econometric Tools for Performance and Risk Analysis. <em>CRAN: Contributed Packages</em> Preprint at (2007). 10.32614/CRAN.package.PerformanceAnalytics</cite>
</li>
<li id="CR38">
<span class="label">38.</span><cite>Fonseca, P. A. S., Suárez-Vega, A., Marras, G. &amp; Cánovas, Á. GALLO: an R package for genomic annotation and integration of multiple data sources in livestock for positional candidate loci. <em>Gigascience</em><strong>9</strong>, (2020).</cite> [<a href="https://doi.org/10.1093/gigascience/giaa149" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7772745/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33377911/" class="usa-link">PubMed</a>]</li>
<li id="CR39">
<span class="label">39.</span><cite>Harrison, P. W. et al. Ensembl 2024. <em>Nucleic Acids Res.</em><strong>52</strong>, D891–D899 (2024).
</cite> [<a href="https://doi.org/10.1093/nar/gkad1049" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10767893/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37953337/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Harrison,%20P.%20W.%20et%20al.%20Ensembl%202024.%20Nucleic%20Acids%20Res.52,%20D891%E2%80%93D899%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR40">
<span class="label">40.</span><cite>Hu, Z. L., Park, C. A. &amp; Reecy, J. M. Bringing the animal QTLdb and CorrDB into the future: meeting new challenges and providing updated services. <em>Nucleic Acids Res.</em><strong>50</strong>, D956–D961 (2022).
</cite> [<a href="https://doi.org/10.1093/nar/gkab1116" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8728226/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34850103/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hu,%20Z.%20L.,%20Park,%20C.%20A.%20&amp;%20Reecy,%20J.%20M.%20Bringing%20the%20animal%20QTLdb%20and%20CorrDB%20into%20the%20future:%20meeting%20new%20challenges%20and%20providing%20updated%20services.%20Nucleic%20Acids%20Res.50,%20D956%E2%80%93D961%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR41">
<span class="label">41.</span><cite>Hu, Y. et al. Comparative analyses of copy number variations between Bos Taurus and Bos indicus. <em>BMC Genom.</em><strong>21</strong>, 1–11 (2020).</cite> [<a href="https://doi.org/10.1186/s12864-020-07097-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7528262/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33004001/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hu,%20Y.%20et%20al.%20Comparative%20analyses%20of%20copy%20number%20variations%20between%20Bos%20Taurus%20and%20Bos%20indicus.%20BMC%20Genom.21,%201%E2%80%9311%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR42">
<span class="label">42.</span><cite>Arlt, M. F., Wilson, T. E. &amp; Glover, T. W. Replication stress and mechanisms of CNV formation. <em>Curr. Opin. Genet. Dev.</em><strong>22</strong>, 204–210 (2012).
</cite> [<a href="https://doi.org/10.1016/j.gde.2012.01.009" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3371136/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22365495/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Arlt,%20M.%20F.,%20Wilson,%20T.%20E.%20&amp;%20Glover,%20T.%20W.%20Replication%20stress%20and%20mechanisms%20of%20CNV%20formation.%20Curr.%20Opin.%20Genet.%20Dev.22,%20204%E2%80%93210%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR43">
<span class="label">43.</span><cite>Teo, S. M., Pawitan, Y., Ku, C. S., Chia, K. S. &amp; Salim, A. Statistical challenges associated with detecting copy number variations with next-generation sequencing. <em>Bioinformatics</em><strong>28</strong>, 2711–2718 (2012).
</cite> [<a href="https://doi.org/10.1093/bioinformatics/bts535" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22942022/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Teo,%20S.%20M.,%20Pawitan,%20Y.,%20Ku,%20C.%20S.,%20Chia,%20K.%20S.%20&amp;%20Salim,%20A.%20Statistical%20challenges%20associated%20with%20detecting%20copy%20number%20variations%20with%20next-generation%20sequencing.%20Bioinformatics28,%202711%E2%80%932718%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR44">
<span class="label">44.</span><cite>Mills, R. E. et al. Mapping copy number variation by population-scale genome sequencing. <em>Nature </em><strong>470</strong>, 59–65 (2011).</cite> [<a href="https://doi.org/10.1038/nature09708" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3077050/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21293372/" class="usa-link">PubMed</a>]</li>
<li id="CR45">
<span class="label">45.</span><cite>Chen, L., Chamberlain, A. J., Reich, C. M., Daetwyler, H. D. &amp; Hayes, B. J. Detection and validation of structural variations in bovine whole-genome sequence data. <em>Genet. Selection Evol.</em><strong>49</strong>, 1–13 (2017).</cite> [<a href="https://doi.org/10.1186/s12711-017-0286-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5267451/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28122487/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chen,%20L.,%20Chamberlain,%20A.%20J.,%20Reich,%20C.%20M.,%20Daetwyler,%20H.%20D.%20&amp;%20Hayes,%20B.%20J.%20Detection%20and%20validation%20of%20structural%20variations%20in%20bovine%20whole-genome%20sequence%20data.%20Genet.%20Selection%20Evol.49,%201%E2%80%9313%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR46">
<span class="label">46.</span><cite>Lee, Y. L. et al. High-resolution structural variants catalogue in a large-scale whole genome sequenced bovine family cohort data. <em>BMC Genom.</em><strong>24</strong>, 1–17 (2023).</cite> [<a href="https://doi.org/10.1186/s12864-023-09259-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10152703/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37127590/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lee,%20Y.%20L.%20et%20al.%20High-resolution%20structural%20variants%20catalogue%20in%20a%20large-scale%20whole%20genome%20sequenced%20bovine%20family%20cohort%20data.%20BMC%20Genom.24,%201%E2%80%9317%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR47">
<span class="label">47.</span><cite>Wellenreuther, M. &amp; Bernatchez, L. Eco-Evolutionary genomics of chromosomal inversions. <em>Trends Ecol. Evol.</em><strong>33</strong>, 427–440 (2018).
</cite> [<a href="https://doi.org/10.1016/j.tree.2018.04.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29731154/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wellenreuther,%20M.%20&amp;%20Bernatchez,%20L.%20Eco-Evolutionary%20genomics%20of%20chromosomal%20inversions.%20Trends%20Ecol.%20Evol.33,%20427%E2%80%93440%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR48">
<span class="label">48.</span><cite>Rasekh, M. E. et al. Discovery of large genomic inversions using long range information. <em>BMC Genom.</em><strong>18</strong>, 1–12 (2017).</cite> [<a href="https://doi.org/10.1186/s12864-016-3444-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5223412/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28073353/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rasekh,%20M.%20E.%20et%20al.%20Discovery%20of%20large%20genomic%20inversions%20using%20long%20range%20information.%20BMC%20Genom.18,%201%E2%80%9312%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR49">
<span class="label">49.</span><cite>Puig, M., Casillas, S., Villatoro, S. &amp; Cáceres, M. Human inversions and their functional consequences. <em>Brief. Funct. Genomics</em>. <strong>14</strong>, 369–379 (2015).
</cite> [<a href="https://doi.org/10.1093/bfgp/elv020" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4576756/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25998059/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Puig,%20M.,%20Casillas,%20S.,%20Villatoro,%20S.%20&amp;%20C%C3%A1ceres,%20M.%20Human%20inversions%20and%20their%20functional%20consequences.%20Brief.%20Funct.%20Genomics.%2014,%20369%E2%80%93379%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR50">
<span class="label">50.</span><cite>Martínez-Fundichely, A. et al. InvFEST, a database integrating information of polymorphic inversions in the human genome. <em>Nucleic Acids Res.</em><strong>42</strong>, D1027–D1032 (2014).
</cite> [<a href="https://doi.org/10.1093/nar/gkt1122" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3965118/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24253300/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mart%C3%ADnez-Fundichely,%20A.%20et%20al.%20InvFEST,%20a%20database%20integrating%20information%20of%20polymorphic%20inversions%20in%20the%20human%20genome.%20Nucleic%20Acids%20Res.42,%20D1027%E2%80%93D1032%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR51">
<span class="label">51.</span><cite>Dobigny, G., Britton-Davidian, J. &amp; Robinson, T. J. Chromosomal polymorphism in mammals: an evolutionary perspective. <em>Biol. Rev.</em><strong>92</strong>, 1–21 (2017).
</cite> [<a href="https://doi.org/10.1111/brv.12213" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26234165/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Dobigny,%20G.,%20Britton-Davidian,%20J.%20&amp;%20Robinson,%20T.%20J.%20Chromosomal%20polymorphism%20in%20mammals:%20an%20evolutionary%20perspective.%20Biol.%20Rev.92,%201%E2%80%9321%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR52">
<span class="label">52.</span><cite>Hou, Y. et al. Fine mapping of copy number variations on two cattle genome assemblies using high density SNP array. <em>BMC Genom.</em><strong>13</strong>, 1–10 (2012).</cite> [<a href="https://doi.org/10.1186/1471-2164-13-376" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3583728/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22866901/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hou,%20Y.%20et%20al.%20Fine%20mapping%20of%20copy%20number%20variations%20on%20two%20cattle%20genome%20assemblies%20using%20high%20density%20SNP%20array.%20BMC%20Genom.13,%201%E2%80%9310%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR53">
<span class="label">53.</span><cite>Keel, B. N., Keele, J. W. &amp; Snelling, W. M. Genome-wide copy number variation in the bovine genome detected using low coverage sequence of popular beef breeds. <em>Anim. Genet.</em><strong>48</strong>, 141–150 (2017).
</cite> [<a href="https://doi.org/10.1111/age.12519" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27775157/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Keel,%20B.%20N.,%20Keele,%20J.%20W.%20&amp;%20Snelling,%20W.%20M.%20Genome-wide%20copy%20number%20variation%20in%20the%20bovine%20genome%20detected%20using%20low%20coverage%20sequence%20of%20popular%20beef%20breeds.%20Anim.%20Genet.48,%20141%E2%80%93150%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR54">
<span class="label">54.</span><cite>Higgins, M. G. et al. GWAS and eQTL analysis identifies a SNP associated with both residual feed intake and GFRA2 expression in beef cattle. <em>Scientific Reports 2018 </em><strong>8</strong>, 14301 (2018).</cite> [<a href="https://doi.org/10.1038/s41598-018-32374-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6155370/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30250203/" class="usa-link">PubMed</a>]</li>
<li id="CR55">
<span class="label">55.</span><cite>Seabury, C. M. et al. Genome-wide association study for feed efficiency and growth traits in U.S. beef cattle. <em>BMC Genomics 2017 18:1</em> 18, 1–25 (2017).</cite> [<a href="https://doi.org/10.1186/s12864-017-3754-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5437562/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28521758/" class="usa-link">PubMed</a>]</li>
<li id="CR56">
<span class="label">56.</span><cite>Olivieri, B. F. et al. Genomic regions associated with feed efficiency indicator traits in an experimental Nellore cattle population. <em>PLoS One</em>. <strong>11</strong>, e0164390 (2016).
</cite> [<a href="https://doi.org/10.1371/journal.pone.0164390" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5070821/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27760167/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Olivieri,%20B.%20F.%20et%20al.%20Genomic%20regions%20associated%20with%20feed%20efficiency%20indicator%20traits%20in%20an%20experimental%20Nellore%20cattle%20population.%20PLoS%20One.%2011,%20e0164390%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR57">
<span class="label">57.</span><cite>Melo, T. P., De Camargo, G. M. F., De Albuquerque, L. G. &amp; Carvalheiro, R. Genome-wide association study provides strong evidence of genes affecting the reproductive performance of Nellore beef cows. <em>PLoS One</em>. <strong>12</strong>, e0178551 (2017).
</cite> [<a href="https://doi.org/10.1371/journal.pone.0178551" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5451131/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28562680/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Melo,%20T.%20P.,%20De%20Camargo,%20G.%20M.%20F.,%20De%20Albuquerque,%20L.%20G.%20&amp;%20Carvalheiro,%20R.%20Genome-wide%20association%20study%20provides%20strong%20evidence%20of%20genes%20affecting%20the%20reproductive%20performance%20of%20Nellore%20beef%20cows.%20PLoS%20One.%2012,%20e0178551%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR58">
<span class="label">58.</span><cite>Diedrichs, F. et al. Comparative molecular portraits of human unfertilized oocytes and primordial germ cells at 10 weeks of gestation. <em>Int. J. Dev. Biol.</em><strong>56</strong>, 789–797 (2013).</cite> [<a href="https://doi.org/10.1387/ijdb.120230ja" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23417401/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Diedrichs,%20F.%20et%20al.%20Comparative%20molecular%20portraits%20of%20human%20unfertilized%20oocytes%20and%20primordial%20germ%20cells%20at%2010%20weeks%20of%20gestation.%20Int.%20J.%20Dev.%20Biol.56,%20789%E2%80%93797%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR59">
<span class="label">59.</span><cite>Rafter, P. et al. Genome-wide association analyses of carcass traits using copy number variants and Raw intensity values of single nucleotide polymorphisms in cattle. <em>BMC Genom.</em><strong>22</strong>, 1–16 (2021).</cite> [<a href="https://doi.org/10.1186/s12864-021-08075-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8542340/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34688258/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rafter,%20P.%20et%20al.%20Genome-wide%20association%20analyses%20of%20carcass%20traits%20using%20copy%20number%20variants%20and%20Raw%20intensity%20values%20of%20single%20nucleotide%20polymorphisms%20in%20cattle.%20BMC%20Genom.22,%201%E2%80%9316%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR60">
<span class="label">60.</span><cite>Olender, T., Lancet, D. &amp; Nebert, D. W. Update on the olfactory receptor (OR) gene superfamily. <em>Hum. Genomics</em>. <strong>3</strong>, 87–97 (2008).
</cite> [<a href="https://doi.org/10.1186/1479-7364-3-1-87" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2752031/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19129093/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Olender,%20T.,%20Lancet,%20D.%20&amp;%20Nebert,%20D.%20W.%20Update%20on%20the%20olfactory%20receptor%20(OR)%20gene%20superfamily.%20Hum.%20Genomics.%203,%2087%E2%80%9397%20(2008)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR61">
<span class="label">61.</span><cite>Connor, E. E., Zhou, Y. &amp; Liu, G. E. The essence of appetite: does olfactory receptor variation play a role? <em>J. Anim. Sci.</em><strong>96</strong>, 1551–1558 (2018).
</cite> [<a href="https://doi.org/10.1093/jas/sky068" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6140904/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29534194/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Connor,%20E.%20E.,%20Zhou,%20Y.%20&amp;%20Liu,%20G.%20E.%20The%20essence%20of%20appetite:%20does%20olfactory%20receptor%20variation%20play%20a%20role?%20J.%20Anim.%20Sci.96,%201551%E2%80%931558%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR62">
<span class="label">62.</span><cite>Lee, K. et al. Analysis of cattle olfactory subgenome: the first detail study on the characteristics of the complete olfactory receptor repertoire of a ruminant. <em>BMC Genom.</em><strong>14</strong>, 1–11 (2013).</cite> [<a href="https://doi.org/10.1186/1471-2164-14-596" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3766653/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24004971/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lee,%20K.%20et%20al.%20Analysis%20of%20cattle%20olfactory%20subgenome:%20the%20first%20detail%20study%20on%20the%20characteristics%20of%20the%20complete%20olfactory%20receptor%20repertoire%20of%20a%20ruminant.%20BMC%20Genom.14,%201%E2%80%9311%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR63">
<span class="label">63.</span><cite>Julliard, A. K., Koborssy, A., Fadool, D., Palouzier-Paulignan, B. &amp; D. A. &amp; Nutrient sensing: another chemosensitivity of the olfactory system. <em>Front. Physiol.</em><strong>8</strong>, 269622 (2017).</cite> [<a href="https://doi.org/10.3389/fphys.2017.00468" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5506222/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28747887/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Julliard,%20A.%20K.,%20Koborssy,%20A.,%20Fadool,%20D.,%20Palouzier-Paulignan,%20B.%20&amp;%20D.%20A.%20&amp;%20Nutrient%20sensing:%20another%20chemosensitivity%20of%20the%20olfactory%20system.%20Front.%20Physiol.8,%20269622%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR64">
<span class="label">64.</span><cite>Fleischer, J., Bumbalo, R., Bautze, V., Strotmann, J. &amp; Breer, H. Expression of odorant receptor Olfr78 in enteroendocrine cells of the colon. <em>Cell. Tissue Res.</em><strong>361</strong>, 697–710 (2015).
</cite> [<a href="https://doi.org/10.1007/s00441-015-2165-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25813788/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Fleischer,%20J.,%20Bumbalo,%20R.,%20Bautze,%20V.,%20Strotmann,%20J.%20&amp;%20Breer,%20H.%20Expression%20of%20odorant%20receptor%20Olfr78%20in%20enteroendocrine%20cells%20of%20the%20colon.%20Cell.%20Tissue%20Res.361,%20697%E2%80%93710%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR65">
<span class="label">65.</span><cite>Palouzier-Paulignan, B. et al. Olfaction under metabolic influences. <em>Chem. Senses</em>. <strong>37</strong>, 769–797 (2012).
</cite> [<a href="https://doi.org/10.1093/chemse/bjs059" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3529618/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22832483/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Palouzier-Paulignan,%20B.%20et%20al.%20Olfaction%20under%20metabolic%20influences.%20Chem.%20Senses.%2037,%20769%E2%80%93797%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR66">
<span class="label">66.</span><cite>Niimura, Y. &amp; Nei, M. Extensive gains and losses of olfactory receptor genes in mammalian evolution. <em>PLoS One</em>. <strong>2</strong>, e708 (2007).
</cite> [<a href="https://doi.org/10.1371/journal.pone.0000708" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1933591/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17684554/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Niimura,%20Y.%20&amp;%20Nei,%20M.%20Extensive%20gains%20and%20losses%20of%20olfactory%20receptor%20genes%20in%20mammalian%20evolution.%20PLoS%20One.%202,%20e708%20(2007)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR67">
<span class="label">67.</span><cite>Islam, S., Akhand, M. R. N. &amp; Hasan, M. Evolutionary trend of bovine β-defensin proteins toward functionality prediction: A domain-based bioinformatics study. <em>Heliyon</em><strong>9</strong>, (2023).</cite> [<a href="https://doi.org/10.1016/j.heliyon.2023.e14158" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10015202/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36938430/" class="usa-link">PubMed</a>]</li>
<li id="CR68">
<span class="label">68.</span><cite>Berton, M. P. et al. Genome-wide association study between copy number variation regions and carcass- and meat-quality traits in Nellore cattle. <em>Anim. Prod. Sci.</em><strong>61</strong>, 731–744 (2021).</cite> [<a href="https://scholar.google.com/scholar_lookup?Berton,%20M.%20P.%20et%20al.%20Genome-wide%20association%20study%20between%20copy%20number%20variation%20regions%20and%20carcass-%20and%20meat-quality%20traits%20in%20Nellore%20cattle.%20Anim.%20Prod.%20Sci.61,%20731%E2%80%93744%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR69">
<span class="label">69.</span><cite>Li, J., Mukiibi, R., Wang, Y., Plastow, G. S. &amp; Li, C. Identification of candidate genes and enriched biological functions for feed efficiency traits by integrating plasma metabolites and imputed whole genome sequence variants in beef cattle. <em>BMC Genom.</em><strong>22</strong>, 1–12 (2021).</cite> [<a href="https://doi.org/10.1186/s12864-021-08064-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8591823/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34781903/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li,%20J.,%20Mukiibi,%20R.,%20Wang,%20Y.,%20Plastow,%20G.%20S.%20&amp;%20Li,%20C.%20Identification%20of%20candidate%20genes%20and%20enriched%20biological%20functions%20for%20feed%20efficiency%20traits%20by%20integrating%20plasma%20metabolites%20and%20imputed%20whole%20genome%20sequence%20variants%20in%20beef%20cattle.%20BMC%20Genom.22,%201%E2%80%9312%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR70">
<span class="label">70.</span><cite>Samuel, B. &amp; Dinka, H. In Silico analysis of the promoter region of olfactory receptors in cattle (Bos indicus) to understand its gene regulation. <em>Nucleosides Nucleotides Nucleic Acids</em>. <strong>39</strong>, 853–865 (2020).
</cite> [<a href="https://doi.org/10.1080/15257770.2020.1711524" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32028828/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Samuel,%20B.%20&amp;%20Dinka,%20H.%20In%20Silico%20analysis%20of%20the%20promoter%20region%20of%20olfactory%20receptors%20in%20cattle%20(Bos%20indicus)%20to%20understand%20its%20gene%20regulation.%20Nucleosides%20Nucleotides%20Nucleic%20Acids.%2039,%20853%E2%80%93865%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR71">
<span class="label">71.</span><cite>Collier, R. J., Collier, J. L., Rhoads, R. P. &amp; Baumgard, L. H. Invited review: genes involved in the bovine heat stress response. <em>J. Dairy. Sci.</em><strong>91</strong>, 445–454 (2008).
</cite> [<a href="https://doi.org/10.3168/jds.2007-0540" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18218730/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Collier,%20R.%20J.,%20Collier,%20J.%20L.,%20Rhoads,%20R.%20P.%20&amp;%20Baumgard,%20L.%20H.%20Invited%20review:%20genes%20involved%20in%20the%20bovine%20heat%20stress%20response.%20J.%20Dairy.%20Sci.91,%20445%E2%80%93454%20(2008)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR72">
<span class="label">72.</span><cite>Dikmen, S. et al. The SLICK hair locus derived from senepol cattle confers thermotolerance to intensively managed lactating Holstein cows. <em>J. Dairy. Sci.</em><strong>97</strong>, 5508–5520 (2014).
</cite> [<a href="https://doi.org/10.3168/jds.2014-8087" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24996281/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Dikmen,%20S.%20et%20al.%20The%20SLICK%20hair%20locus%20derived%20from%20senepol%20cattle%20confers%20thermotolerance%20to%20intensively%20managed%20lactating%20Holstein%20cows.%20J.%20Dairy.%20Sci.97,%205508%E2%80%935520%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR73">
<span class="label">73.</span><cite>Huson, H. J. et al. Genome-wide association study and ancestral origins of the slick-hair coat in Tropically adapted cattle. <em>Front. Genet.</em><strong>5</strong>, 79498 (2014).</cite> [<a href="https://doi.org/10.3389/fgene.2014.00101" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4010767/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24808908/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Huson,%20H.%20J.%20et%20al.%20Genome-wide%20association%20study%20and%20ancestral%20origins%20of%20the%20slick-hair%20coat%20in%20Tropically%20adapted%20cattle.%20Front.%20Genet.5,%2079498%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR74">
<span class="label">74.</span><cite>Sosa, F. et al. Inheritance of the SLICK1 allele of PRLR in cattle. <em>Anim. Genet.</em><strong>52</strong>, 887–890 (2021).
</cite> [<a href="https://doi.org/10.1111/age.13145" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34642995/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sosa,%20F.%20et%20al.%20Inheritance%20of%20the%20SLICK1%20allele%20of%20PRLR%20in%20cattle.%20Anim.%20Genet.52,%20887%E2%80%93890%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR75">
<span class="label">75.</span><cite>Weitzel, J. M. et al. Hepatic thyroid signaling of heat-stressed late pregnant and early lactating cows. <em>J. Endocrinol.</em><strong>234</strong>, 129–141 (2017).
</cite> [<a href="https://doi.org/10.1530/JOE-17-0066" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5516449/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28500083/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Weitzel,%20J.%20M.%20et%20al.%20Hepatic%20thyroid%20signaling%20of%20heat-stressed%20late%20pregnant%20and%20early%20lactating%20cows.%20J.%20Endocrinol.234,%20129%E2%80%93141%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR76">
<span class="label">76.</span><cite>Al-Thuwaini, T. M., Al‐Shuhaib, M. B. S., Lepretre, F. &amp; Dawud, H. H. Two co‐inherited novel SNPs in the MC4R gene related to live body weight and hormonal assays in Awassi and Arabi sheep breeds of Iraq. <em>Vet. Med. Sci.</em><strong>7</strong>, 897–907 (2021).
</cite> [<a href="https://doi.org/10.1002/vms3.421" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8136946/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33369226/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Al-Thuwaini,%20T.%20M.,%20Al%E2%80%90Shuhaib,%20M.%20B.%20S.,%20Lepretre,%20F.%20&amp;%20Dawud,%20H.%20H.%20Two%20co%E2%80%90inherited%20novel%20SNPs%20in%20the%20MC4R%20gene%20related%20to%20live%20body%20weight%20and%20hormonal%20assays%20in%20Awassi%20and%20Arabi%20sheep%20breeds%20of%20Iraq.%20Vet.%20Med.%20Sci.7,%20897%E2%80%93907%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR77">
<span class="label">77.</span><cite>Maurya, V. P. et al. Adaptive mechanisms of livestock to changing climate. in Climate Change Impact on Livestock: Adaptation and Mitigation 123–138 (Springer India, New Delhi, doi:10.1007/978-81-322-2265-1_9. (2015).</cite> [<a href="https://scholar.google.com/scholar_lookup?Maurya,%20V.%20P.%20et%20al.%20Adaptive%20mechanisms%20of%20livestock%20to%20changing%20climate.%20in%20Climate%20Change%20Impact%20on%20Livestock:%20Adaptation%20and%20Mitigation%20123%E2%80%93138%20(Springer%20India,%20New%20Delhi,%20doi:10.1007/978-81-322-2265-1_9.%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR78">
<span class="label">78.</span><cite>Sarubbi, J. et al. Hypothalamic neuromodulation and control of the dermal surface temperature of livestock during hyperthermia. <em>Animals</em><strong>14</strong>, 1745 (2024).
</cite> [<a href="https://doi.org/10.3390/ani14121745" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11200636/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38929364/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sarubbi,%20J.%20et%20al.%20Hypothalamic%20neuromodulation%20and%20control%20of%20the%20dermal%20surface%20temperature%20of%20livestock%20during%20hyperthermia.%20Animals14,%201745%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR79">
<span class="label">79.</span><cite>Murata, H. et al. Heat stress induces oxidative stress and activates the KEAP1-NFE2L2-ARE pathway in bovine endometrial epithelial cells. <em>Biol. Reprod.</em><strong>105</strong>, 1114–1125 (2021).
</cite> [<a href="https://doi.org/10.1093/biolre/ioab143" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34296252/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Murata,%20H.%20et%20al.%20Heat%20stress%20induces%20oxidative%20stress%20and%20activates%20the%20KEAP1-NFE2L2-ARE%20pathway%20in%20bovine%20endometrial%20epithelial%20cells.%20Biol.%20Reprod.105,%201114%E2%80%931125%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR80">
<span class="label">80.</span><cite>Zhang, Y. et al. Activation of transcription factor AP-1 in response to thermal injury in rat small intestine and IEC-6 cells. <em>BMC Gastroenterol.</em><strong>15</strong>, 83 (2015).
</cite> [<a href="https://doi.org/10.1186/s12876-015-0309-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4498520/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26162907/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhang,%20Y.%20et%20al.%20Activation%20of%20transcription%20factor%20AP-1%20in%20response%20to%20thermal%20injury%20in%20rat%20small%20intestine%20and%20IEC-6%20cells.%20BMC%20Gastroenterol.15,%2083%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR81">
<span class="label">81.</span><cite>Daneshi, M., Caton, J. S., Caixeta, L. S., Eftekhari, Z. &amp; Ward, A. K. Expression, regulation, and function of β-Defensins in the bovine mammary glands: current knowledge and future perspectives. <em>Animals</em><strong>13</strong>, 3372 (2023).
</cite> [<a href="https://doi.org/10.3390/ani13213372" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10650070/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37958127/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Daneshi,%20M.,%20Caton,%20J.%20S.,%20Caixeta,%20L.%20S.,%20Eftekhari,%20Z.%20&amp;%20Ward,%20A.%20K.%20Expression,%20regulation,%20and%20function%20of%20%CE%B2-Defensins%20in%20the%20bovine%20mammary%20glands:%20current%20knowledge%20and%20future%20perspectives.%20Animals13,%203372%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR82">
<span class="label">82.</span><cite>Wu, L. et al. AMP-Activated protein kinase (AMPK) regulates energy metabolism through modulating thermogenesis in adipose tissue. <em>Front. Physiol.</em><strong>9</strong>, 122 (2018).
</cite> [<a href="https://doi.org/10.3389/fphys.2018.00122" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5826329/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29515462/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wu,%20L.%20et%20al.%20AMP-Activated%20protein%20kinase%20(AMPK)%20regulates%20energy%20metabolism%20through%20modulating%20thermogenesis%20in%20adipose%20tissue.%20Front.%20Physiol.9,%20122%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR83">
<span class="label">83.</span><cite>Derakhshani, H., Plaizier, J. C., De Buck, J., Barkema, H. W. &amp; Khafipour, E. Association of bovine major histocompatibility complex (BoLA) gene polymorphism with colostrum and milk microbiota of dairy cows during the first week of lactation. <em>Microbiome</em><strong>6</strong>, 203 (2018).
</cite> [<a href="https://doi.org/10.1186/s40168-018-0586-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6233267/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30419937/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Derakhshani,%20H.,%20Plaizier,%20J.%20C.,%20De%20Buck,%20J.,%20Barkema,%20H.%20W.%20&amp;%20Khafipour,%20E.%20Association%20of%20bovine%20major%20histocompatibility%20complex%20(BoLA)%20gene%20polymorphism%20with%20colostrum%20and%20milk%20microbiota%20of%20dairy%20cows%20during%20the%20first%20week%20of%20lactation.%20Microbiome6,%20203%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR84">
<span class="label">84.</span><cite>Taye, M. et al. Exploring the genomes of East African indicine cattle breeds reveals signature of selection for tropical environmental adaptation traits. <em>Cogent Food Agric</em><strong>4</strong>, (2018).</cite>
</li>
<li id="CR85">
<span class="label">85.</span><cite>Colombi, D. et al. Genomic responses to Climatic challenges in beef cattle: A review. <em>Anim. Genet.</em><strong>55</strong>, 854–870 (2024).
</cite> [<a href="https://doi.org/10.1111/age.13474" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39219301/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Colombi,%20D.%20et%20al.%20Genomic%20responses%20to%20Climatic%20challenges%20in%20beef%20cattle:%20A%20review.%20Anim.%20Genet.55,%20854%E2%80%93870%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR86">
<span class="label">86.</span><cite>Lacetera, N. et al. Lymphocyte functions in dairy cows in hot environment. <em>Int. J. Biometeorol.</em><strong>50</strong>, 105–110 (2005).
</cite> [<a href="https://doi.org/10.1007/s00484-005-0273-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15991017/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lacetera,%20N.%20et%20al.%20Lymphocyte%20functions%20in%20dairy%20cows%20in%20hot%20environment.%20Int.%20J.%20Biometeorol.50,%20105%E2%80%93110%20(2005)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR87">
<span class="label">87.</span><cite>Chen, D., Tang, T. X., Deng, H., Yang, X. P. &amp; Tang, Z. H. Interleukin-7 biology and its effects on immune cells: mediator of generation, differentiation, survival, and homeostasis. <em>Front. Immunol.</em><strong>12</strong>, 747324 (2021).
</cite> [<a href="https://doi.org/10.3389/fimmu.2021.747324" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8674869/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34925323/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chen,%20D.,%20Tang,%20T.%20X.,%20Deng,%20H.,%20Yang,%20X.%20P.%20&amp;%20Tang,%20Z.%20H.%20Interleukin-7%20biology%20and%20its%20effects%20on%20immune%20cells:%20mediator%20of%20generation,%20differentiation,%20survival,%20and%20homeostasis.%20Front.%20Immunol.12,%20747324%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR88">
<span class="label">88.</span><cite>Peters, I. J. A., de Pater, E. &amp; Zhang, W. The role of GATA2 in adult hematopoiesis and cell fate determination. <em>Front. Cell. Dev. Biol.</em><strong>11</strong>, 1250827 (2023).
</cite> [<a href="https://doi.org/10.3389/fcell.2023.1250827" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10682726/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38033856/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Peters,%20I.%20J.%20A.,%20de%20Pater,%20E.%20&amp;%20Zhang,%20W.%20The%20role%20of%20GATA2%20in%20adult%20hematopoiesis%20and%20cell%20fate%20determination.%20Front.%20Cell.%20Dev.%20Biol.11,%201250827%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR89">
<span class="label">89.</span><cite>Gunshin, H. et al. Slc11a2 is required for intestinal iron absorption and erythropoiesis but dispensable in placenta and liver. <em>J. Clin. Invest.</em><strong>115</strong>, 1258 (2005).
</cite> [<a href="https://doi.org/10.1172/JCI24356" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1077176/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15849611/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gunshin,%20H.%20et%20al.%20Slc11a2%20is%20required%20for%20intestinal%20iron%20absorption%20and%20erythropoiesis%20but%20dispensable%20in%20placenta%20and%20liver.%20J.%20Clin.%20Invest.115,%201258%20(2005)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR90">
<span class="label">90.</span><cite>Zhao, Z. et al. A critical functional missense mutation (T117M) in sheep MC4R gene significantly leads to Gain-of-Function. <em>Animals</em><strong>14</strong>, 2207 (2024).
</cite> [<a href="https://doi.org/10.3390/ani14152207" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11311007/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39123733/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhao,%20Z.%20et%20al.%20A%20critical%20functional%20missense%20mutation%20(T117M)%20in%20sheep%20MC4R%20gene%20significantly%20leads%20to%20Gain-of-Function.%20Animals14,%202207%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR91">
<span class="label">91.</span><cite>Ma, Y. L. et al. Copy number variation (CNV) in the IGF1R gene across four cattle breeds and its association with economic traits. <em>Arch. Anim. Breed.</em><strong>62</strong>, 171–179 (2019).
</cite> [<a href="https://doi.org/10.5194/aab-62-171-2019" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6852844/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31807627/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ma,%20Y.%20L.%20et%20al.%20Copy%20number%20variation%20(CNV)%20in%20the%20IGF1R%20gene%20across%20four%20cattle%20breeds%20and%20its%20association%20with%20economic%20traits.%20Arch.%20Anim.%20Breed.62,%20171%E2%80%93179%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR92">
<span class="label">92.</span><cite>Liang, C. et al. Regulatory roles of SREBF1 and SREBF2 in lipid metabolism and deposition in two Chinese representative Fat-Tailed sheep breeds. <em>Anim. 2020</em>. <strong>10, </strong>1317 (2020).</cite> [<a href="https://doi.org/10.3390/ani10081317" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7460493/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32751718/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Liang,%20C.%20et%20al.%20Regulatory%20roles%20of%20SREBF1%20and%20SREBF2%20in%20lipid%20metabolism%20and%20deposition%20in%20two%20Chinese%20representative%20Fat-Tailed%20sheep%20breeds.%20Anim.%202020.%2010,%201317%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR93">
<span class="label">93.</span><cite>Huang, J. et al. Comparative transcriptome analysis reveals that PCK1 is a potential gene affecting IMF deposition in Buffalo. <em>BMC Genom.</em><strong>21</strong>, 1–12 (2020).</cite> [<a href="https://doi.org/10.1186/s12864-020-07120-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7552535/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33045988/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Huang,%20J.%20et%20al.%20Comparative%20transcriptome%20analysis%20reveals%20that%20PCK1%20is%20a%20potential%20gene%20affecting%20IMF%20deposition%20in%20Buffalo.%20BMC%20Genom.21,%201%E2%80%9312%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR94">
<span class="label">94.</span><cite>Li, G. et al. RNA-Seq analysis identifies differentially expressed genes in the longissimus dorsi of Wagyu and Chinese red steppe cattle. <em>Int. J. Mol. Sci.</em><strong>24</strong>, 387 (2023).</cite> [<a href="https://doi.org/10.3390/ijms24010387" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9820533/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36613828/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li,%20G.%20et%20al.%20RNA-Seq%20analysis%20identifies%20differentially%20expressed%20genes%20in%20the%20longissimus%20dorsi%20of%20Wagyu%20and%20Chinese%20red%20steppe%20cattle.%20Int.%20J.%20Mol.%20Sci.24,%20387%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR95">
<span class="label">95.</span><cite>Sejian, V., Bhatta, R., Gaughan, J. B. &amp; Dunshea, F. R. &amp; Lacetera, N. Adaptation of animals to heat stress. <em>animal</em><strong>12</strong>, s431–s444 (2018).</cite> [<a href="https://doi.org/10.1017/S1751731118001945" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30139399/" class="usa-link">PubMed</a>]</li>
<li id="CR96">
<span class="label">96.</span><cite>Zhong, Z. et al. MORC proteins regulate transcription factor binding by mediating chromatin compaction in active chromatin regions. <em>Genome Biol.</em><strong>24</strong>, 96 (2023).
</cite> [<a href="https://doi.org/10.1186/s13059-023-02939-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10131428/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37101218/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhong,%20Z.%20et%20al.%20MORC%20proteins%20regulate%20transcription%20factor%20binding%20by%20mediating%20chromatin%20compaction%20in%20active%20chromatin%20regions.%20Genome%20Biol.24,%2096%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR97">
<span class="label">97.</span><cite>He, F., Ru, X. &amp; Wen, T. NRF2, a transcription factor for stress response and beyond. <em>Int. J. Mol. Sci. 2020</em>. <strong>21</strong>, 4777 (2020).</cite> [<a href="https://doi.org/10.3390/ijms21134777" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7369905/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32640524/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?He,%20F.,%20Ru,%20X.%20&amp;%20Wen,%20T.%20NRF2,%20a%20transcription%20factor%20for%20stress%20response%20and%20beyond.%20Int.%20J.%20Mol.%20Sci.%202020.%2021,%204777%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR98">
<span class="label">98.</span><cite>Lee, J. et al. Activation of Ets-2 by oxidative stress induces Bcl-xL expression and accounts for glial survival in amyotrophic lateral sclerosis. <em>FASEB J.</em><strong>23</strong>, 1739–1749 (2009).
</cite> [<a href="https://doi.org/10.1096/fj.08-121046" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2698657/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19179380/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lee,%20J.%20et%20al.%20Activation%20of%20Ets-2%20by%20oxidative%20stress%20induces%20Bcl-xL%20expression%20and%20accounts%20for%20glial%20survival%20in%20amyotrophic%20lateral%20sclerosis.%20FASEB%20J.23,%201739%E2%80%931749%20(2009)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR99">
<span class="label">99.</span><cite>Rosati, V. C., Quinn, A. A., Gleadow, R. M. &amp; Blomstedt, C. K. The putative GATA transcription factor SbGATA22 as a novel regulator of Dhurrin biosynthesis. <em>Life</em><strong>14</strong>, 470 (2024).
</cite> [<a href="https://doi.org/10.3390/life14040470" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11051066/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38672741/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rosati,%20V.%20C.,%20Quinn,%20A.%20A.,%20Gleadow,%20R.%20M.%20&amp;%20Blomstedt,%20C.%20K.%20The%20putative%20GATA%20transcription%20factor%20SbGATA22%20as%20a%20novel%20regulator%20of%20Dhurrin%20biosynthesis.%20Life14,%20470%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR100">
<span class="label">100.</span><cite>Ibeagha-Awemu, E. M. &amp; Zhao, X. Epigenetic marks: regulators of livestock phenotypes and conceivable sources of missing variation in livestock improvement programs. <em>Front. Genet.</em><strong>6</strong>, 139313 (2015).</cite> [<a href="https://doi.org/10.3389/fgene.2015.00302" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4585011/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26442116/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ibeagha-Awemu,%20E.%20M.%20&amp;%20Zhao,%20X.%20Epigenetic%20marks:%20regulators%20of%20livestock%20phenotypes%20and%20conceivable%20sources%20of%20missing%20variation%20in%20livestock%20improvement%20programs.%20Front.%20Genet.6,%20139313%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR101">
<span class="label">101.</span><cite>Jiang, X. X. et al. Epigenetic regulation of antibody responses by the histone H2A deubiquitinase MYSM1. <em>Sci. Rep.</em><strong>5</strong>, 13755 (2015).
</cite> [<a href="https://doi.org/10.1038/srep13755" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4562257/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26348977/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jiang,%20X.%20X.%20et%20al.%20Epigenetic%20regulation%20of%20antibody%20responses%20by%20the%20histone%20H2A%20deubiquitinase%20MYSM1.%20Sci.%20Rep.5,%2013755%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR102">
<span class="label">102.</span><cite>Liu, H. Y. et al. Dietary conjugated Linoleic acid modulates the hepatic circadian clock program via PPARα/REV-ERBα-Mediated chromatin modification in mice. <em>Front. Nutr.</em><strong>8</strong>, 711398 (2021).
</cite> [<a href="https://doi.org/10.3389/fnut.2021.711398" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8553932/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34722605/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Liu,%20H.%20Y.%20et%20al.%20Dietary%20conjugated%20Linoleic%20acid%20modulates%20the%20hepatic%20circadian%20clock%20program%20via%20PPAR%CE%B1/REV-ERB%CE%B1-Mediated%20chromatin%20modification%20in%20mice.%20Front.%20Nutr.8,%20711398%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR103">
<span class="label">103.</span><cite>Shah, R. B. et al. FANCI functions as a repair/apoptosis switch in response to DNA crosslinks. <em>Dev. Cell.</em><strong>56</strong>, 2207–2222e7 (2021).
</cite> [<a href="https://doi.org/10.1016/j.devcel.2021.06.010" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8378530/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34256011/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Shah,%20R.%20B.%20et%20al.%20FANCI%20functions%20as%20a%20repair/apoptosis%20switch%20in%20response%20to%20DNA%20crosslinks.%20Dev.%20Cell.56,%202207%E2%80%932222e7%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR104">
<span class="label">104.</span><cite>Rovani, B. T. et al. Analysis of nuclear maturation, DNA damage and repair gene expression of bovine oocyte and cumulus cells submitted to ionizing radiation. <em>Anim. Reprod.</em><strong>20</strong>, e20230021 (2023).
</cite> [<a href="https://doi.org/10.1590/1984-3143-AR2023-0021" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10247184/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37293252/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rovani,%20B.%20T.%20et%20al.%20Analysis%20of%20nuclear%20maturation,%20DNA%20damage%20and%20repair%20gene%20expression%20of%20bovine%20oocyte%20and%20cumulus%20cells%20submitted%20to%20ionizing%20radiation.%20Anim.%20Reprod.20,%20e20230021%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR105">
<span class="label">105.</span><cite>Patel, D. S., Misenko, S. M., Her, J. &amp; Bunting, S. F. BLM helicase regulates DNA repair by counteracting RAD51 loading at DNA double-strand break sites. <em>J. Cell Biol.</em><strong>216</strong>, 3521–3534 (2017).
</cite> [<a href="https://doi.org/10.1083/jcb.201703144" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5674892/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28912125/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Patel,%20D.%20S.,%20Misenko,%20S.%20M.,%20Her,%20J.%20&amp;%20Bunting,%20S.%20F.%20BLM%20helicase%20regulates%20DNA%20repair%20by%20counteracting%20RAD51%20loading%20at%20DNA%20double-strand%20break%20sites.%20J.%20Cell%20Biol.216,%203521%E2%80%933534%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR106">
<span class="label">106.</span><cite>Gorynia, S. et al. Structural and functional insights into a Dodecameric molecular machine – The RuvBL1/RuvBL2 complex. <em>J. Struct. Biol.</em><strong>176</strong>, 279–291 (2011).
</cite> [<a href="https://doi.org/10.1016/j.jsb.2011.09.001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21933716/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gorynia,%20S.%20et%20al.%20Structural%20and%20functional%20insights%20into%20a%20Dodecameric%20molecular%20machine%20%E2%80%93%20The%20RuvBL1/RuvBL2%20complex.%20J.%20Struct.%20Biol.176,%20279%E2%80%93291%20(2011)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR107">
<span class="label">107.</span><cite>Amills, M. &amp; Bouzat, J. L. Characterization of the bovine BCL2L1 gene and related pseudogenes. <em>Anim. Genet.</em><strong>34</strong>, 457–461 (2003).
</cite> [<a href="https://doi.org/10.1046/j.0268-9146.2003.01039.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/14687078/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Amills,%20M.%20&amp;%20Bouzat,%20J.%20L.%20Characterization%20of%20the%20bovine%20BCL2L1%20gene%20and%20related%20pseudogenes.%20Anim.%20Genet.34,%20457%E2%80%93461%20(2003)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR108">
<span class="label">108.</span><cite>Wu, D. J. et al. Autophagy-linked FYVE containing protein WDFY3 interacts with TRAF6 and modulates RANKL-induced osteoclastogenesis. <em>J. Autoimmun.</em><strong>73</strong>, 73–84 (2016).
</cite> [<a href="https://doi.org/10.1016/j.jaut.2016.06.004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5003737/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27330028/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wu,%20D.%20J.%20et%20al.%20Autophagy-linked%20FYVE%20containing%20protein%20WDFY3%20interacts%20with%20TRAF6%20and%20modulates%20RANKL-induced%20osteoclastogenesis.%20J.%20Autoimmun.73,%2073%E2%80%9384%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR109">
<span class="label">109.</span><cite>Baker, M. J. et al. Stress-induced OMA1 activation and autocatalytic turnover regulate OPA1-dependent mitochondrial dynamics. <em>EMBO J.</em><strong>33</strong>, 578–593 (2014).
</cite> [<a href="https://doi.org/10.1002/embj.201386474" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3989652/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24550258/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Baker,%20M.%20J.%20et%20al.%20Stress-induced%20OMA1%20activation%20and%20autocatalytic%20turnover%20regulate%20OPA1-dependent%20mitochondrial%20dynamics.%20EMBO%20J.33,%20578%E2%80%93593%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR110">
<span class="label">110.</span><cite>Gilkerson, R., Kaur, H., Carrillo, O. &amp; Ramos, I. OMA1-Mediated mitochondrial dynamics balance organellar homeostasis upstream of cellular stress responses. <em>Int. J. Mol. Sci. 2024</em>. <strong>25</strong>, 4566 (2024).</cite> [<a href="https://doi.org/10.3390/ijms25084566" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11049825/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38674151/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gilkerson,%20R.,%20Kaur,%20H.,%20Carrillo,%20O.%20&amp;%20Ramos,%20I.%20OMA1-Mediated%20mitochondrial%20dynamics%20balance%20organellar%20homeostasis%20upstream%20of%20cellular%20stress%20responses.%20Int.%20J.%20Mol.%20Sci.%202024.%2025,%204566%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR111">
<span class="label">111.</span><cite>Porto-Neto, L. R. et al. The genetic architecture of Climatic adaptation of tropical cattle. <em>PLoS One</em>. <strong>9</strong>, e113284 (2014).
</cite> [<a href="https://doi.org/10.1371/journal.pone.0113284" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4242650/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25419663/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Porto-Neto,%20L.%20R.%20et%20al.%20The%20genetic%20architecture%20of%20Climatic%20adaptation%20of%20tropical%20cattle.%20PLoS%20One.%209,%20e113284%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR112">
<span class="label">112.</span><cite>Taye, M. et al. Whole genome detection of signature of positive selection in African cattle reveals selection for thermotolerance. <em>Anim. Sci. J.</em><strong>88</strong>, 1889–1901 (2017).
</cite> [<a href="https://doi.org/10.1111/asj.12851" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28748670/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Taye,%20M.%20et%20al.%20Whole%20genome%20detection%20of%20signature%20of%20positive%20selection%20in%20African%20cattle%20reveals%20selection%20for%20thermotolerance.%20Anim.%20Sci.%20J.88,%201889%E2%80%931901%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR113">
<span class="label">113.</span><cite>Ly, T. D. et al. The impact of inflammatory stimuli on Xylosyltransferase-I regulation in primary human dermal fibroblasts. <em>Biomedicines</em><strong>10</strong>, 1451 (2022).
</cite> [<a href="https://doi.org/10.3390/biomedicines10061451" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9220250/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35740472/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ly,%20T.%20D.%20et%20al.%20The%20impact%20of%20inflammatory%20stimuli%20on%20Xylosyltransferase-I%20regulation%20in%20primary%20human%20dermal%20fibroblasts.%20Biomedicines10,%201451%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR114">
<span class="label">114.</span><cite>Fu, C. T. et al. An evolutionarily conserved PTEN-C/EBPα-CTNNA1 axis controls myeloid development and transformation. <em>Blood</em><strong>115</strong>, 4715–4724 (2010).
</cite> [<a href="https://doi.org/10.1182/blood-2009-11-255778" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9728059/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20371743/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Fu,%20C.%20T.%20et%20al.%20An%20evolutionarily%20conserved%20PTEN-C/EBP%CE%B1-CTNNA1%20axis%20controls%20myeloid%20development%20and%20transformation.%20Blood115,%204715%E2%80%934724%20(2010)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR115">
<span class="label">115.</span><cite>Kelwick, R., Desanlis, I., Wheeler, G. N. &amp; Edwards, D. R. The ADAMTS (A disintegrin and metalloproteinase with thrombospondin motifs) family. <em>Genome Biol.</em><strong>16</strong>, 1–16 (2015).
</cite> [<a href="https://doi.org/10.1186/s13059-015-0676-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4448532/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26025392/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kelwick,%20R.,%20Desanlis,%20I.,%20Wheeler,%20G.%20N.%20&amp;%20Edwards,%20D.%20R.%20The%20ADAMTS%20(A%20disintegrin%20and%20metalloproteinase%20with%20thrombospondin%20motifs)%20family.%20Genome%20Biol.16,%201%E2%80%9316%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12365053/bin/41598_2025_14139_MOESM1_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 1</a><sup> (1.1MB, docx) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>The data that support the findings of this study are available from Gensys breeding program (https://gensys.com.br/), but restrictions apply to the availability of these data, which were used under license for the current study, and so are not publicly available. Data are however available from the authors (Dr. Lucia G. Albuquerque email: galvao.albuquerque@unesp.br) upon reasonable request and with permission of Gensys breeding program.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Scientific Reports are provided here courtesy of <strong>Nature Publishing Group</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1038/s41598-025-14139-0"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/41598_2025_Article_14139.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (5.3 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12365053/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12365053/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12365053%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365053/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12365053/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12365053/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40830176/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12365053/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40830176/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12365053/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12365053/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="eWAg7TXnXJIY7S0xZXBh0mdiZDyE39L4C3kjJQSnbOuhy1Z2bxiqx29VSMuyqzOc">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
