
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Assessing variant effect predictors and disease mechanisms in intrinsically disordered proteins - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="152552A58AF43D430552A50004C0040D.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="ploscomp">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12377588/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="PLOS Computational Biology">
<meta name="citation_title" content="Assessing variant effect predictors and disease mechanisms in intrinsically disordered proteins">
<meta name="citation_author" content="Mohamed Fawzy">
<meta name="citation_author_institution" content="MRC Human Genetics Unit, Institute of Genetics and Cancer, University of Edinburgh, Edinburgh, United Kingdom">
<meta name="citation_author" content="Joseph A Marsh">
<meta name="citation_author_institution" content="MRC Human Genetics Unit, Institute of Genetics and Cancer, University of Edinburgh, Edinburgh, United Kingdom">
<meta name="citation_publication_date" content="2025 Aug 19">
<meta name="citation_volume" content="21">
<meta name="citation_issue" content="8">
<meta name="citation_firstpage" content="e1013400">
<meta name="citation_doi" content="10.1371/journal.pcbi.1013400">
<meta name="citation_pmid" content="40828855">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12377588/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12377588/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12377588/pdf/pcbi.1013400.pdf">
<meta name="description" content="Intrinsically disordered regions (IDRs) are central to diverse cellular processes but present unique challenges for interpreting genetic variants implicated in human disease. Unlike structured protein domains, IDRs lack stable three-dimensional ...">
<meta name="og:title" content="Assessing variant effect predictors and disease mechanisms in intrinsically disordered proteins">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Intrinsically disordered regions (IDRs) are central to diverse cellular processes but present unique challenges for interpreting genetic variants implicated in human disease. Unlike structured protein domains, IDRs lack stable three-dimensional ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12377588/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12377588">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1371/journal.pcbi.1013400"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/pcbi.1013400.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12377588%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12377588/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12377588/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12377588/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-ploscomp.png" alt="PLOS Computational Biology logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to PLOS Computational Biology" title="Link to PLOS Computational Biology" shape="default" href="https://doi.org/10.1371/journal.pcbi.1013400" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">PLoS Comput Biol</button></div>. 2025 Aug 19;21(8):e1013400. doi: <a href="https://doi.org/10.1371/journal.pcbi.1013400" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1371/journal.pcbi.1013400</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22PLoS%20Comput%20Biol%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22PLoS%20Comput%20Biol%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22PLoS%20Comput%20Biol%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22PLoS%20Comput%20Biol%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Assessing variant effect predictors and disease mechanisms in intrinsically disordered proteins</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Fawzy%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Mohamed Fawzy</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Mohamed Fawzy</span></h3>
<div class="p">
<sup>1</sup>MRC Human Genetics Unit, Institute of Genetics and Cancer, University of Edinburgh, Edinburgh, United Kingdom</div>
<div>Data curation, Formal analysis, Writing – original draft</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Fawzy%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Mohamed Fawzy</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Marsh%20JA%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Joseph A Marsh</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Joseph A Marsh</span></h3>
<div class="p">
<sup>1</sup>MRC Human Genetics Unit, Institute of Genetics and Cancer, University of Edinburgh, Edinburgh, United Kingdom</div>
<div>Conceptualization, Supervision, Writing – review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Marsh%20JA%22%5BAuthor%5D" class="usa-link"><span class="name western">Joseph A Marsh</span></a>
</div>
</div>
<sup>1,</sup><sup>*</sup>
</div>
<div class="cg p">Editor: <span class="name western">Nir Ben-Tal</span><sup>2</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="aff001">
<sup>1</sup>MRC Human Genetics Unit, Institute of Genetics and Cancer, University of Edinburgh, Edinburgh, United Kingdom</div>
<div id="edit1">
<sup>2</sup>Tel Aviv University, ISRAEL</div>
<div class="author-notes p">
<div class="fn" id="cor001">
<sup>✉</sup><p class="display-inline">* E-mail: <span>joseph.marsh@ed.ac.uk</span></p>
</div>
<div class="fn" id="coi001"><p>The authors have declared that no competing interests exist.</p></div>
</div>
<h4 class="font-secondary">Roles</h4>
<div class="p">
<strong class="contrib"><span class="name western">Mohamed Fawzy</span></strong>: <span class="role">Data curation, Formal analysis, Writing – original draft</span>
</div>
<div>
<strong class="contrib"><span class="name western">Joseph A Marsh</span></strong>: <span class="role">Conceptualization, Supervision, Writing – review &amp; editing</span>
</div>
<div class="p">
<strong class="contrib"><span class="name western">Nir Ben-Tal</span></strong>: <span class="role">Editor</span>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 Apr 1; Accepted 2025 Aug 6; Collection date 2025 Aug.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© 2025 Fawzy, Marsh</div>
<p>This is an open access article distributed under the terms of the <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Creative Commons Attribution License</a>, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12377588  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40828855/" class="usa-link">40828855</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>Abstract</h2>
<p>Intrinsically disordered regions (IDRs) are central to diverse cellular processes but present unique challenges for interpreting genetic variants implicated in human disease. Unlike structured protein domains, IDRs lack stable three-dimensional conformations and are often involved in regulation through transient interactions and post-translational modifications. These features can affect both the distribution of pathogenic variants and the performance of computational tools used to predict their effects. Here, we systematically assessed the distribution of pathogenic <em>vs</em> benign missense variants across disordered, intermediate, and structured protein regions in the human proteome. Pathogenic variants were notably depleted in IDRs yet were associated with distinct molecular mechanisms, particularly dominant gain- and loss-of-function effects. We evaluated 33 variant effect predictors (VEPs), revealing widespread reductions in sensitivity for pathogenic variants in IDRs, despite high AUROC scores driven by accurate benign variant predictions. We also observed substantial discordance among VEP classifications in disordered regions, underscoring the need for region-aware thresholds and disorder-informed prediction strategies. Incorporating features reflective of IDR biology, such as transient interaction motifs and modification sites, may enhance the accuracy and interpretability of future tools.</p></section><section class="abstract" id="abstract2"><h2>Author summary</h2>
<p>Some parts of proteins, known as intrinsically disordered regions, do not fold into fixed shapes. Instead, they stay flexible and play key roles in controlling how cells work, often by forming short-lived interactions or responding to signals. Because of this flexibility, these regions are harder to study, especially when it comes to understanding how genetic changes might lead to disease. In this study, we looked at how often disease-causing mutations appear in disordered regions, and how well current computational tools can spot them. We found that these tools often struggle to detect harmful mutations in disordered parts of proteins, even though they seem to perform well overall. This is because they are good at identifying harmless variants, which are common in these regions, but tend to miss the rare ones that do cause problems. We also found that different tools often give conflicting answers when looking at the same variant in a disordered region. Our results show that new methods are needed to properly interpret genetic changes in these parts of proteins, taking into account the unique ways they work and fail.</p></section><section id="sec001"><h2 class="pmc_sec_title">Introduction</h2>
<p>Intrinsically disordered regions (IDRs) of proteins lack stable secondary or tertiary structure under physiological conditions, instead adopting flexible conformations that can shift in response to binding partners or cellular cues [<a href="#pcbi.1013400.ref001" class="usa-link" aria-describedby="pcbi.1013400.ref001">1</a>–<a href="#pcbi.1013400.ref003" class="usa-link" aria-describedby="pcbi.1013400.ref003">3</a>]. This structural plasticity allows IDRs to act as molecular hubs in diverse cellular processes such as transcriptional regulation, signal transduction, and protein–protein interactions [<a href="#pcbi.1013400.ref004" class="usa-link" aria-describedby="pcbi.1013400.ref004">4</a>]. Their interactions are typically transient and reversible, often mediated by short linear motifs, post-translational modification (PTM) sites, or segments involved in phase separation [<a href="#pcbi.1013400.ref002" class="usa-link" aria-describedby="pcbi.1013400.ref002">2</a>,<a href="#pcbi.1013400.ref005" class="usa-link" aria-describedby="pcbi.1013400.ref005">5</a>]. Although some IDRs engage in high-affinity or promiscuous binding, their flexibility enables rapid responses to cellular signals and extensive PTM, allowing fine-tuned regulation in dynamic environments [<a href="#pcbi.1013400.ref006" class="usa-link" aria-describedby="pcbi.1013400.ref006">6</a>–<a href="#pcbi.1013400.ref009" class="usa-link" aria-describedby="pcbi.1013400.ref009">9</a>]. IDRs are especially prevalent in eukaryotic proteins, with around 30–40% containing long disordered regions (&gt;30 residues) [<a href="#pcbi.1013400.ref010" class="usa-link" aria-describedby="pcbi.1013400.ref010">10</a>]. Their sequence composition, enriched in polar and charged residues, and depleted in hydrophobic residues, prevents stable folding, favouring a dynamic equilibrium of conformational states [<a href="#pcbi.1013400.ref001" class="usa-link" aria-describedby="pcbi.1013400.ref001">1</a>,<a href="#pcbi.1013400.ref011" class="usa-link" aria-describedby="pcbi.1013400.ref011">11</a>,<a href="#pcbi.1013400.ref012" class="usa-link" aria-describedby="pcbi.1013400.ref012">12</a>]. These properties make IDRs central to regulatory networks and sensitive to perturbation, consistent with their involvement in a range of human diseases, including cancer, cardiovascular, neurodegenerative, and prion disorders [<a href="#pcbi.1013400.ref009" class="usa-link" aria-describedby="pcbi.1013400.ref009">9</a>].</p>
<p>Variant effect predictors (VEPs) are computational tools designed to estimate the potential impact of genetic variants, particularly on human health and disease. VEPs vary widely in their algorithms, training data, and input features, such as evolutionary conservation, structural properties, allele frequencies, or functional assay data [<a href="#pcbi.1013400.ref013" class="usa-link" aria-describedby="pcbi.1013400.ref013">13</a>]. While VEPs are widely used, relatively little attention has been given to how their performance varies depending on whether a variant is in a structured or disordered region. Despite the functional importance of IDRs, these regions might pose significant challenges for VEPs. Unlike structured protein domains, IDRs tend to be less evolutionarily conserved and lack the fixed secondary and tertiary structures that some VEPs rely on as predictive features. As a result, VEPs may struggle to accurately classify pathogenic variants in these regions. Previous studies have shown that standard predictors such as SIFT and PolyPhen-2 exhibit a reduction of more than 10% in sensitivity for pathogenic variants in IDRs compared to structured regions [<a href="#pcbi.1013400.ref014" class="usa-link" aria-describedby="pcbi.1013400.ref014">14</a>]. However, a systematic evaluation of VEP performance across different protein structural contexts remains largely unexplored.</p>
<p>Previously, we showed that IDRs significantly influence the apparent performance of missense VEPs, often leading to inflated area under the receiver operating characteristic curve (AUROC) values [<a href="#pcbi.1013400.ref015" class="usa-link" aria-describedby="pcbi.1013400.ref015">15</a>]. This elevation in AUROC stems from the fact that disordered regions are enriched with putatively benign missense variants, which are typically under weaker evolutionary constraint and thus easier for VEPs to correctly classify as benign. However, this can be misleading: while it is not technically incorrect, it primarily reflects the ease with which VEPs handle these straightforward benign classifications rather than an ability to detect disease-causing mutations. This distinction is critical, as it suggests that the impressive AUROC values observed in proteins with long disordered regions may not indicate robust performance in identifying pathogenic variants, which is often the primary concern in clinical and research settings. In the extreme case, a predictor that labels every variant in an IDR as benign would achieve a deceptively high AUROC, despite having no real ability to detect pathogenic variants. This highlights the need to evaluate sensitivity and class balance alongside overall accuracy metrics.</p>
<p>Building on this observation, here we have further investigated the occurrence of pathogenic missense variants and the predictive performance of VEPs in IDRs. First, we systematically classified residues across the human proteome to assess the distribution of pathogenic and benign variants in disordered versus structured regions. We evaluated 33 VEPs, spanning clinical-trained, population-tuned, and population-free categories, to determine their effectiveness in identifying pathogenic variants within IDRs. Our analysis reveals significant differences in variant distribution and VEP performance across structural contexts, highlighting the limitations of current predictors in capturing the subtle functional impacts of variants in disordered regions. Furthermore, we explored the molecular mechanisms driving pathogenicity in IDRs, particularly in autosomal dominant genes, and propose strategies to enhance VEP accuracy by incorporating region-specific thresholds and IDR-specific features. These findings provide a foundation for refining computational tools to better interpret the effects of genetic variants in disordered regions, with implications for understanding their role in human disease.</p></section><section id="sec002"><h2 class="pmc_sec_title">Results and discussion</h2>
<section id="sec003"><h3 class="pmc_sec_title">Pathogenic missense variants are depleted in disordered regions</h3>
<p>To define intrinsically disordered regions, we used AlphaFold2 (AF2) pLDDT scores, which correlate inversely with structural order [<a href="#pcbi.1013400.ref016" class="usa-link" aria-describedby="pcbi.1013400.ref016">16</a>–<a href="#pcbi.1013400.ref018" class="usa-link" aria-describedby="pcbi.1013400.ref018">18</a>]. AF2 pLDDT is recognised as a robust predictor of disorder [<a href="#pcbi.1013400.ref017" class="usa-link" aria-describedby="pcbi.1013400.ref017">17</a>]. While some studies suggest that pLDDT may approach or even exceed the performance of traditional disorder prediction tools in certain contexts [<a href="#pcbi.1013400.ref019" class="usa-link" aria-describedby="pcbi.1013400.ref019">19</a>,<a href="#pcbi.1013400.ref020" class="usa-link" aria-describedby="pcbi.1013400.ref020">20</a>], benchmarking efforts such as CAID-2 [<a href="#pcbi.1013400.ref021" class="usa-link" aria-describedby="pcbi.1013400.ref021">21</a>] have shown that no single method consistently outperforms others across all benchmarks. Thus, we use pLDDT here primarily due to its consistent proteome-wide availability and demonstrated utility for defining long disordered regions.</p>
<p>Residues were classified as disordered, ordered, or intermediate based on pLDDT and local context, as illustrated in <a href="#pcbi.1013400.g001" class="usa-link">Fig 1A</a> (see Methods). Importantly, our approach uses a conservative definition: only residues within long, low-confidence stretches (≥30 residues with average pLDDT &lt;70) were classified as disordered. This threshold aims to capture regions likely to be truly unstructured under physiological conditions, such as extended flexible linkers or regulatory activation domains, rather than transiently mobile loops within structured domains. Shorter, low-confidence segments were instead assigned to an intermediate category to reduce false positives, particularly in cases where local flexibility does not imply global disorder. This strict classification likely underestimates the full extent of disorder in the proteome, but provides a high-confidence set of IDRs for downstream analysis.</p>
<figure class="fig xbox font-sm" id="pcbi.1013400.g001"><h4 class="obj_head">Fig 1. Rarity of pathogenic mutations in intrinsically disordered regions.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12377588_pcbi.1013400.g001.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/7858/12377588/e071f306ed91/pcbi.1013400.g001.jpg" loading="lazy" height="665" width="750" alt="Fig 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/pcbi.1013400.g001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>(A)</strong> AF2-predicted structure of human Rho GTPase-activating protein 32 (UniProt ID: <a href="https://www.ncbi.nlm.nih.gov/protein/A7KAX9" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">A7KAX9</a>), with disordered, intermediate, and ordered regions depicted in red, purple, and blue, respectively. <strong>(B)</strong> Proportion of residues classified as disordered, intermediate, and ordered across the human proteome. <strong>(C)</strong> Distribution of missense variants across structural regions, expressed as percentages, with pathogenic and likely pathogenic variants from ClinVar, benign and likely benign variants from ClinVar, and putatively benign population variants from gnomAD.</p></figcaption></figure><p>Applying this approach to 20,281 canonical human proteins, we found that 25.1% of residues fall within disordered regions (<a href="#pcbi.1013400.g001" class="usa-link">Fig 1B</a>). In addition, 57.7% of residues are predicted to be ordered, while 17.2% are classified as intermediate. When considering disorder at the level of human proteins, most (57.7%) are predicted to have at least one disordered region, according to our definition, while 26.2% are highly disordered, having at least 30% of their residues in disordered regions (Fig A in <a href="#pcbi.1013400.s001" class="usa-link">S1 Text</a>). These observations are broadly consistent with previous findings [<a href="#pcbi.1013400.ref018" class="usa-link" aria-describedby="pcbi.1013400.ref018">18</a>,<a href="#pcbi.1013400.ref022" class="usa-link" aria-describedby="pcbi.1013400.ref022">22</a>], noting our strict disorder definition, as well as our more permissive intermediate classification.</p>
<p>We next examined the distribution of missense variants across structural regions. Variants were grouped into three categories: pathogenic and likely pathogenic variants from ClinVar [<a href="#pcbi.1013400.ref023" class="usa-link" aria-describedby="pcbi.1013400.ref023">23</a>], benign and likely benign variants from ClinVar, and putatively benign population variants from gnomAD [<a href="#pcbi.1013400.ref024" class="usa-link" aria-describedby="pcbi.1013400.ref024">24</a>]. Pathogenic variants were strongly depleted in disordered regions, with only ~3.6% occurring at disordered residues, compared to 15.1% in intermediate and 81.3% in ordered regions (<a href="#pcbi.1013400.g001" class="usa-link">Fig 1C</a>). In contrast, ClinVar benign variants were heavily enriched in disordered regions (39.8%). Finally, 27.2% of gnomAD putatively benign variants occurred in disordered regions, showing a slight enrichment over the occurrence of disorder in the human proteome. Overall, the vast majority of variants found in disordered regions are benign or putatively benign, while only a small minority are classified as pathogenic.</p>
<p>The strong enrichment of pathogenic missense variants in ordered regions and the corresponding enrichment of benign variants in disordered regions is broadly consistent with previous findings [<a href="#pcbi.1013400.ref025" class="usa-link" aria-describedby="pcbi.1013400.ref025">25</a>–<a href="#pcbi.1013400.ref027" class="usa-link" aria-describedby="pcbi.1013400.ref027">27</a>]. This distribution reflects fundamental differences in structural constraints between these regions. In a disordered region, a single amino acid substitution is unlikely to cause major functional disruption unless it affects a specific critical residue, such as one involved in PTMs or transient binding interactions. Conversely, in a structured domain, even a single missense variant can have a profound effect by destabilising the fold, altering interaction surfaces, or disrupting active sites. This difference in mutational tolerance aligns with the expectation that proteins rely on stable structural elements for core functions, whereas disordered regions often accommodate greater sequence variation without loss or change of function. Nevertheless, despite their depletion, we identified 1550 pathogenic missense variants in disordered regions, demonstrating that such variants can and do contribute to disease.</p></section><section id="sec004"><h3 class="pmc_sec_title">Distinct mechanisms underlie pathogenic variants in disordered regions</h3>
<p>Pathogenic missense variants in disordered regions must operate through distinct molecular mechanisms compared to those in ordered regions. In globular domains, such variants often cause disease by destabilising protein structure, leading to misfolding, or affecting the active site and ultimately a complete loss of function. In contrast, in disordered regions, pathogenic variants may affect protein behaviour by subtly altering regulatory interactions or disrupting specific binding motifs, such as short linear motifs or PTMs, without necessarily abolishing the protein’s overall function [<a href="#pcbi.1013400.ref028" class="usa-link" aria-describedby="pcbi.1013400.ref028">28</a>]. Beyond direct loss of interactions, recent work has shown that mutations in IDRs can perturb the conformational ensemble of the protein. Rather than adopting a fixed structure, IDRs exist as dynamic ensembles of interconverting conformers, and single-point mutations can shift this ensemble in ways that affects function, binding preferences, or regulatory activity [<a href="#pcbi.1013400.ref029" class="usa-link" aria-describedby="pcbi.1013400.ref029">29</a>]. These effects may be indirect but are functionally significant, especially in regulatory contexts.</p>
<p>In addition, LLPS is now recognized as a key mechanism by which many disordered regions, particularly in transcription factors and RNA-binding proteins, facilitate the formation of membraneless biomolecular condensates [<a href="#pcbi.1013400.ref030" class="usa-link" aria-describedby="pcbi.1013400.ref030">30</a>]. These condensates play critical roles in organizing cellular biochemistry, concentrating macromolecules at specific genomic loci, and regulating gene expression. Pathogenic variants in activation domains or other phase-separating IDRs can alter the properties or assembly behaviour of these condensates, impeding their formation or modifying their composition and dynamics. Thus, LLPS disruption represents another important mechanism by which variants in IDRs can cause disease, especially in contexts where dynamic compartmentalisation and multivalent interactions are essential for normal function. Complementing these observations, recent proteome-wide simulations showed that pathogenic missense variants in IDRs are disproportionately located in regions with low conformational entropy [<a href="#pcbi.1013400.ref031" class="usa-link" aria-describedby="pcbi.1013400.ref031">31</a>]. These findings support the notion that even subtle shifts in the structural ensemble of IDRs can be pathogenic, especially when they occur at dynamically constrained regulatory segments.</p>
<p>Given that current VEPs tend to perform best on loss-of-function (LOF) variants compared to gain-of-function (GOF) and dominant-negative (DN) variants [<a href="#pcbi.1013400.ref032" class="usa-link" aria-describedby="pcbi.1013400.ref032">32</a>], we can speculate that they might perform less well at the identification of pathogenic variants in disordered regions. To explore this further, we first examined inheritance mode as a proxy for mechanism. Variants in autosomal recessive (AR) disorders nearly always act via LOF, while those in autosomal dominant (AD) disorders can involve LOF, GOF, or DN effects [<a href="#pcbi.1013400.ref033" class="usa-link" aria-describedby="pcbi.1013400.ref033">33</a>,<a href="#pcbi.1013400.ref034" class="usa-link" aria-describedby="pcbi.1013400.ref034">34</a>]. In <a href="#pcbi.1013400.g002" class="usa-link">Fig 2A</a>, we plot the proportion of variants in AD <em>vs</em> AR genes that occur in disordered, intermediate and ordered regions. Although the large majority of pathogenic variants are in ordered regions for both categories, it is interesting to note that AD genes show roughly double the proportion of pathogenic variants in disordered regions compared to AR genes (4.4% <em>vs</em> 2.2%, P = 3 x 10<sup>-137</sup>, Fisher’s exact test). Similarly, pathogenic variants in AD genes are also moderately enriched in intermediate regions (18.4% <em>vs</em> 11.9%, P = 1 x 10<sup>-87</sup>, Fisher’s exact test). This suggests that pathogenic variants in AR genes are more likely to act via a LOF mechanism involving structural disruption in an ordered region. In contrast, a much greater proportion of pathogenic variation in AD genes occurs in disordered regions, likely acting via more complex mechanisms related to the role of disordered regions in regulation, signalling, and interaction networks.</p>
<figure class="fig xbox font-sm" id="pcbi.1013400.g002"><h4 class="obj_head">Fig 2. Distribution and counts of pathogenic missense variants by inheritance mode and molecular mechanism.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12377588_pcbi.1013400.g002.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/7858/12377588/11291429a7ca/pcbi.1013400.g002.jpg" loading="lazy" height="359" width="787" alt="Fig 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/pcbi.1013400.g002/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>(A)</strong> Bar plot illustrating the proportions and counts of pathogenic missense variants across disordered, intermediate, and ordered structural regions in autosomal dominant (AD) and autosomal recessive (AR) genes. Counts of pathogenic variants per structural region and inheritance mode are indicated above each bar. <strong>(B)</strong> Bar plot showing the percentage and count of pathogenic missense variants associated with dominant negative (DN), gain-of-function (GOF), and loss-of-function (LOF) mechanisms within AD genes across structural regions. Counts are displayed above each bar, with error bars representing 95% binomial confidence intervals.</p></figcaption></figure><p>Next, we considered pathogenic variants from AD genes, and grouped them by molecular mechanism (LOF, GOF and DN) based on a previously published classification [<a href="#pcbi.1013400.ref032" class="usa-link" aria-describedby="pcbi.1013400.ref032">32</a>] (<a href="#pcbi.1013400.g002" class="usa-link">Fig 2B</a>). Surprisingly, LOF variants showed the highest enrichment in disordered regions, in marked contrast with our initial expectation based on the results of the AR <em>vs</em> AD variants, and our naïve expectation of how pathogenic variants in disordered regions would be likely to act. Interestingly, however, LOF variants showed the lowest representation in intermediate regions. On closer consideration, we noted that many of the disordered LOF variants in AD genes occurred in transcription factors, which have a very well-known association with haploinsufficiency [<a href="#pcbi.1013400.ref035" class="usa-link" aria-describedby="pcbi.1013400.ref035">35</a>]. Given that many transcription factors contain long disordered regions, often playing important roles in transactivation [<a href="#pcbi.1013400.ref030" class="usa-link" aria-describedby="pcbi.1013400.ref030">30</a>,<a href="#pcbi.1013400.ref036" class="usa-link" aria-describedby="pcbi.1013400.ref036">36</a>], this suggests that disruption of these functions is a common mechanism for pathogenicity of variants in disordered regions.</p>
<p>GOF variants were also enriched in disorder compared to AD variants in general, with 6.3% occurring at predicted disordered regions. Interestingly, GOF variants also showed the strongest enrichment at intermediate regions, and the lowest representation in ordered regions. This is consistent with the previous observation that GOF variants were enriched in regions with lower pLDDT values [<a href="#pcbi.1013400.ref032" class="usa-link" aria-describedby="pcbi.1013400.ref032">32</a>]. This pattern is consistent with the flexible, context-dependent nature of disordered and intermediate regions, which often mediate transient, regulated interactions. GOF variants in these regions may create new or stronger interactions, disrupt phase separation, interfere with PTM sites, or perturb short linear motifs involved in dynamic regulation. These changes can enhance or misdirect protein activity without necessarily compromising stability or folding, making disordered and intermediate regions plausible sites for such effects.</p>
<p>Several well-known GOF mutations illustrate these principles. The canonical oncogenic BRAF p.Val600Glu mutation [<a href="#pcbi.1013400.ref037" class="usa-link" aria-describedby="pcbi.1013400.ref037">37</a>,<a href="#pcbi.1013400.ref038" class="usa-link" aria-describedby="pcbi.1013400.ref038">38</a>] occurs in a region predicted to be disordered in our pipeline and activates the MAPK pathway by mimicking phosphorylation. The histone variant H3F3A p.Lys27Met [<a href="#pcbi.1013400.ref039" class="usa-link" aria-describedby="pcbi.1013400.ref039">39</a>], found in paediatric gliomas, disrupts PRC2 binding at the disordered N-terminal tail, altering the epigenetic landscape. In FOXL2, a transcription factor associated with adult granulosa cell tumours, the p.Cys134Trp variant [<a href="#pcbi.1013400.ref040" class="usa-link" aria-describedby="pcbi.1013400.ref040">40</a>] lies within a disordered region flanking the DNA-binding domain and is thought to alter gene expression via changes in interaction specificity or post-translational control. These cases illustrate how disordered regions can host GOF mutations that bypass normal regulation without affecting global protein structure.</p>
<p>In contrast, only 2.2% of DN variants occurred at disordered regions, much lower than for the other AD mechanisms. This is consistent with the DN mechanism’s strong association with oligomeric interfaces, which tend to occur in structured domains [<a href="#pcbi.1013400.ref032" class="usa-link" aria-describedby="pcbi.1013400.ref032">32</a>]. While rare, DN effects in disordered regions remain plausible in cases involving misregulation of binding motifs or competitive inhibition [<a href="#pcbi.1013400.ref033" class="usa-link" aria-describedby="pcbi.1013400.ref033">33</a>].</p></section><section id="sec005"><h3 class="pmc_sec_title">VEP performance varies across structural regions</h3>
<p>Next, we investigated how VEPs perform in distinguishing pathogenic from putatively benign missense variants across disordered, intermediate and ordered regions. We used 33 different VEPs, with strict coverage filters, so that all VEPs had predictions for all variants tested (see Methods). Although this limits the number of VEPs we can include, it means that our results will not be influenced any coverage biases of individual methods, given that many VEPs lack full coverage even within individual genes [<a href="#pcbi.1013400.ref013" class="usa-link" aria-describedby="pcbi.1013400.ref013">13</a>].</p>
<p>We grouped VEPs into three categories using a recently introduced classification scheme [ <a href="#pcbi.1013400.ref041" class="usa-link" aria-describedby="pcbi.1013400.ref041">41</a>,<a href="#pcbi.1013400.ref042" class="usa-link" aria-describedby="pcbi.1013400.ref042">42</a>], based on their potential risk of circularity due to their training [<a href="#pcbi.1013400.ref043" class="usa-link" aria-describedby="pcbi.1013400.ref043">43</a>]. <em>Clinical-trained</em> VEPs are supervised models trained directly on human variants with known clinical labels, such as pathogenic and benign annotations. These inherently have the highest risk of circularity. <em>Population-tuned</em> VEPs are not trained on clinical labels but have been optimised or calibrated using <em>human population</em> data, typically through allele frequency-based scaling or tuning. These tend to have a much lower, but non-zero susceptibility to circularity. Finally, <em>population-free</em> VEPs have not been trained or tuned on any human variant data and are therefore immune from circularity concerns. This group includes unsupervised methods, protein language models, and models based on evolutionary conservation from sequence alignments.</p>
<p>In <a href="#pcbi.1013400.g003" class="usa-link">Fig 3</a>, we show the AUROC values for disordered, intermediate and ordered regions across all predictors. In <a href="#pcbi.1013400.g003" class="usa-link">Fig 3A</a>, we group population-free and population-tuned VEPs together (noting that most are population-free and only AlphaMissense [<a href="#pcbi.1013400.ref044" class="usa-link" aria-describedby="pcbi.1013400.ref044">44</a>], UNEECON [<a href="#pcbi.1013400.ref045" class="usa-link" aria-describedby="pcbi.1013400.ref045">45</a>] and LIST-S2 [<a href="#pcbi.1013400.ref046" class="usa-link" aria-describedby="pcbi.1013400.ref046">46</a>] are population-tuned). <a href="#pcbi.1013400.g003" class="usa-link">Fig 3B</a> shows the clinical-trained VEPs, which comprise the majority of methods in our analysis. Interestingly, nearly all VEPs show the highest AUROC values in disordered regions and the lowest values in ordered regions. The only exception to this is MPC [<a href="#pcbi.1013400.ref047" class="usa-link" aria-describedby="pcbi.1013400.ref047">47</a>], which shows slightly worse performance in disordered than ordered regions, though we note that its performance across all regions remains poor compared to other VEPs.</p>
<figure class="fig xbox font-sm" id="pcbi.1013400.g003"><h4 class="obj_head">Fig 3. Performance evaluation of VEPs across structural regions.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12377588_pcbi.1013400.g003.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/7858/12377588/62d3ab54ceac/pcbi.1013400.g003.jpg" loading="lazy" height="328" width="750" alt="Fig 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/pcbi.1013400.g003/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Performance of VEPs assessed by area under the receiver operating characteristic curve (AUROC) across disordered, intermediate, and ordered protein regions. The x-axis represents AUROC values starting at 0.5, and the y-axis lists individual VEPs. Population-free and population-tuned VEPs are shown together in <strong>(A)</strong>, while clinical-trained VEPs are shown in <strong>(B)</strong>.</p></figcaption></figure><p>When comparing different VEPs, there is little evidence that any methods perform particularly well in specific regions. For example, AlphaMissense and CPT [<a href="#pcbi.1013400.ref048" class="usa-link" aria-describedby="pcbi.1013400.ref048">48</a>] show the highest AUROC of any population-tuned or population-free models across disordered, intermediate and ordered regions. Similarly, SNPred [<a href="#pcbi.1013400.ref049" class="usa-link" aria-describedby="pcbi.1013400.ref049">49</a>] and MutFormer [<a href="#pcbi.1013400.ref050" class="usa-link" aria-describedby="pcbi.1013400.ref050">50</a>] show higher AUROC values than any clinical-trained models across all three regions. Thus, methods that perform well on ordered regions also tend to perform well on disordered regions, and there does not seem to be any reason to recommend a particular VEP for ordered <em>vs</em> disordered regions. Notably, VEPs based on protein language models (<em>e.g.,</em> ESM-1b [<a href="#pcbi.1013400.ref051" class="usa-link" aria-describedby="pcbi.1013400.ref051">51</a>] and ESM-1v [<a href="#pcbi.1013400.ref052" class="usa-link" aria-describedby="pcbi.1013400.ref052">52</a>]) appear to show very similar trends across regions as those based purely on sequence alignments (<em>e.g.,</em> GEMME [<a href="#pcbi.1013400.ref053" class="usa-link" aria-describedby="pcbi.1013400.ref053">53</a>]), suggesting that neither approach has any clear advantages in ordered <em>vs</em> disordered regions. This is somewhat surprising, as protein language models have been proposed to capture contextual sequence signals that might help in disordered regions where structural information is lacking [<a href="#pcbi.1013400.ref051" class="usa-link" aria-describedby="pcbi.1013400.ref051">51</a>]. However, our findings suggest that in practice, these models do not currently offer enhanced predictive power in IDRs. This is consistent with a recent benchmark of structure-based predictors, which found that disordered regions substantially reduce predictive performance across multiple architectures and input modalities [<a href="#pcbi.1013400.ref054" class="usa-link" aria-describedby="pcbi.1013400.ref054">54</a>].</p></section><section id="sec006"><h3 class="pmc_sec_title">VEPs show low sensitivity for pathogenic variants in disordered regions</h3>
<p>The apparently superior performance of VEPs in disordered regions, as measured by AUROC, is consistent with our earlier finding that proteins with large amounts of disorder tend to have higher AUROC values, and that excluding disordered regions often lowers overall performance [<a href="#pcbi.1013400.ref015" class="usa-link" aria-describedby="pcbi.1013400.ref015">15</a>]. In that study, we showed that this effect was not due to improved classification of pathogenic variants, but rather to the high density of putatively benign variants in disordered regions. These sites are typically under weak evolutionary constraint and are more readily classified as benign by most VEPs.</p>
<p>To investigate this phenomenon further, we calculated the sensitivity and specificity at the “optimal threshold” point of ROC curve for each VEP (see Methods). The sensitivity represents the true positive rate: the fraction of pathogenic variants that are correctly classified as pathogenic using this threshold. The specificity represents the true negative rate: the fraction of putatively benign variants correctly classified as non-pathogenic.</p>
<p>In <a href="#pcbi.1013400.g004" class="usa-link">Fig 4A</a>, we plot sensitivity <em>vs</em> specificity for all VEPs in disordered, intermediate and ordered regions. Strikingly, across nearly all predictors, sensitivity for pathogenic variants in disordered regions was significantly lower compared to ordered and intermediate regions. This low sensitivity indicates that, despite high overall AUROC scores driven by accurate identification of putatively benign variants, VEPs frequently misclassify pathogenic variants within disordered sequences. At the same time, specificities are clearly much higher in disordered regions than ordered regions. That is, while VEPs are less likely to correctly classify variants as pathogenic in disordered regions, they are more likely to misclassify benign variants.</p>
<figure class="fig xbox font-sm" id="pcbi.1013400.g004"><h4 class="obj_head">Fig 4. Assessment of VEP performance using global and region-specific optimal thresholds.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12377588_pcbi.1013400.g004.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/7858/12377588/d58949e6a9df/pcbi.1013400.g004.jpg" loading="lazy" height="608" width="750" alt="Fig 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/pcbi.1013400.g004/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Performance of VEPs evaluated by sensitivity on the x-axis and specificity on the y-axis. <strong>(A)</strong> Sensitivity <em>vs</em> specificity in disordered, intermediate and ordered regions calculated using a global optimal threshold, across all structural regions. <strong>(B)</strong> Sensitivity <em>vs</em> specificity calculated using a region-specific optimal threshold.</p></figcaption></figure><p>We observe that clinical-trained VEPs tend to have higher sensitivities than population-free VEPs, although this is almost certainly driven by the circularity issue discussed earlier. Interestingly, however, we also note a strong inverse correlation between sensitivity and specificity for population-free VEPs within disordered regions. While some degree of trade-off between these metrics is expected, one plausible explanation is that population-free predictors rely predominantly on evolutionary conservation or biophysical signals that are inherently weaker in disordered regions. As a result, adjusting their decision threshold to capture more pathogenic variants (thus boosting sensitivity) inadvertently increases misclassification of benign variants, reducing specificity. Conversely, a more stringent threshold that accurately dismisses most benign variants sharply lowers sensitivity for the sparse, but functionally impactful, pathogenic variants in IDRs.</p>
<p>Given that the low sensitivity of VEPs in disordered regions means that they are likely to miss true pathogenic variants, we wondered whether the use of region-specific thresholds could be beneficial. Thus, we calculated the optimal thresholds for each VEP from the ROC curves in the same manner as before, but considering only disordered, intermediate or ordered variants (<a href="#pcbi.1013400.s002" class="usa-link">S1 Table</a>). Essentially, this results in lower thresholds for classifying variants as pathogenic in disordered regions, and higher thresholds in ordered regions (assuming a VEP where the score positively correlates with likelihood of pathogenicity). For example, for CPT, the top-performing population-free VEP, we calculate a global optimal threshold of 0.35, compared to a disordered-specific threshold of 0.24 and ordered-specific threshold of 0.44. Note that these are not meant to be thresholds for making clinical classifications; far stricter thresholds would be required for this [<a href="#pcbi.1013400.ref055" class="usa-link" aria-describedby="pcbi.1013400.ref055">55</a>]. Instead, these represent the optimal thresholds for discriminating between pathogenic and putatively benign in our dataset, and therefore suggest useful thresholds for consideration in variant prioritisation.</p>
<p>In <a href="#pcbi.1013400.g004" class="usa-link">Fig 4B</a>, we plot sensitivity <em>vs</em> specificity using these region-specific thresholds, while Fig C in <a href="#pcbi.1013400.s001" class="usa-link">S1 Text</a> shows the difference in sensitivity and specificity using region-specific vs global thresholds. The use of region-specific thresholds results in very similar sensitivities of VEPs across all three regions, generally increasing sensitivity in disordered regions and decreasing it in ordered regions. Interestingly, while specificity is also affected, the impact appears to be relatively smaller than on sensitivity in disordered regions. This suggests that the sensitivity gain in disordered regions is likely worth the relatively small impact on specificity. In contrast, the loss of sensitivity in ordered regions is not compensated for by a large specificity increase. Thus, we suggest that these region-specific thresholds are likely useful for disordered and possibly intermediate regions, but they may be of less benefit for prioritising variants in ordered regions.</p>
<p>Given that evolutionary conservation plays a key role in most VEPs, we wondered whether the reduced sensitivity of VEPs could be related to reduced conservation of pathogenic variants in disordered regions. To address this, we compare the residue-level conservation of the sites of pathogenic and putatively benign variants in disordered, intermediate and ordered regions (Fig B in <a href="#pcbi.1013400.s001" class="usa-link">S1 Text</a>). Unsurprisingly, pathogenic variants occur at far more conserved positions than putatively benign variants. Interestingly, however, both pathogenic and putatively benign variants in disordered regions are less conserved residues than those in ordered regions. Thus, the weaker conservation of pathogenic variants in disordered regions means they are less likely to be correctly classified, thus reducing sensitivity, while the weaker conservation of putatively benign variants means that they are more likely to be correctly classified, thus increasing specificity.</p>
<p>Our findings align closely with a recent benchmarking study by Luppino et al. [<a href="#pcbi.1013400.ref056" class="usa-link" aria-describedby="pcbi.1013400.ref056">56</a>], which demonstrated that deep learning-based VEPs such as AlphaMissense maintain high specificity but suffer from markedly reduced sensitivity in IDRs. This elevated false-negative rate reflects the challenge of detecting subtle functional disruptions in regions with weak evolutionary constraints, reinforcing our conclusion that current VEPs are poorly equipped to capture pathogenic effects in disordered contexts.</p></section><section id="sec007"><h3 class="pmc_sec_title">VEPs show discordant predictions in disordered regions</h3>
<p>Given the observed variability in VEP performance across structural contexts, we next investigated the consistency of their predictions for individual variants. Specifically, we asked whether VEPs tend to agree on which variants are pathogenic in intrinsically disordered regions (IDRs), compared to intermediate and ordered regions.</p>
<p>To quantify agreement, we calculated the average pairwise concordance between VEPs using Cohen’s kappa statistic. This metric quantifies inter-rater reliability while correcting for agreement expected by chance. A kappa value of 1 indicates perfect agreement, 0 corresponds to agreement no better than random, and negative values reflect systematic disagreement. This allows us to assess not just whether two predictors classify variants similarly, but whether they do so more consistently than would be expected by chance alone. We converted VEP scores to binary predictions using each method’s global optimal threshold, and calculated mean pairwise kappa values within and between VEP groups defined by training strategy: clinical-trained, population-tuned, and population-free.</p>
<p>As shown in <a href="#pcbi.1013400.g005" class="usa-link">Fig 5</a>, overall agreement was lowest in disordered regions, particularly among population-free VEPs. These models, which rely primarily on evolutionary conservation and sequence-derived features, appear to disagree more frequently in regions lacking strong structural or conservation signals. This suggests that the sparse and subtle functional constraints characteristic of IDRs lead to reduced model convergence and greater uncertainty in prediction.</p>
<figure class="fig xbox font-sm" id="pcbi.1013400.g005"><h4 class="obj_head">Fig 5. Average classification agreement among VEP groups across structural regions.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12377588_pcbi.1013400.g005.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/7858/12377588/c3ba05f3d745/pcbi.1013400.g005.jpg" loading="lazy" height="349" width="733" alt="Fig 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/pcbi.1013400.g005/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Average Cohen’s kappa values representing classification agreement among VEP groups across disordered, intermediate, and ordered protein structural regions. The x-axis lists VEP groups categorised by training methodology, and the y-axis indicates the mean Cohen’s kappa for each group within each structural region. Error bars denote the standard error of the mean.</p></figcaption></figure><p>In contrast, clinical-trained and population-tuned VEPs showed somewhat higher agreement in disordered regions. This likely reflects shared biases arising from human variant data used during model development or calibration, which can lead to convergence due to circularity rather than true predictive power. Nonetheless, even within these groups, agreement was lower in disordered regions than in ordered regions.</p>
<p>By comparison, all VEP groups showed higher concordance in intermediate and ordered regions, consistent with the stronger evolutionary and structural constraints in these contexts. Here, models are more likely to converge on consistent classifications, likely driven by clearer functional signals.</p>
<p>Together, these findings suggest that disordered regions not only pose challenges for sensitivity but also reduce agreement between VEPs, particularly those without access to human variant data. This highlights the need for specialised strategies, both in training and interpretation, to improve consistency and reliability in these structurally flexible regions.</p></section></section><section id="sec008"><h2 class="pmc_sec_title">Conclusion</h2>
<p>IDRs are crucial mediators of diverse regulatory functions, yet their inherent structural flexibility poses significant challenges for accurate interpretation of genetic variants associated with human disease. Our systematic analysis highlights fundamental limitations in current VEPs, which demonstrate notably reduced sensitivity for pathogenic variants within these disordered contexts. Although VEPs consistently achieve higher AUROC scores in IDRs, this largely reflects their proficiency at identifying benign variants, which is driven by weaker evolutionary constraints, rather than their ability to pinpoint subtle yet functionally critical pathogenic alterations. Consequently, relying solely on global thresholds risks overlooking clinically relevant variants that operate through mechanisms unique to disordered regions.</p>
<p>We demonstrate that pathogenic variants in IDRs predominantly act through distinct molecular mechanisms compared to those in structured domains, particularly involving nuanced alterations in regulatory interactions or transcriptional activity. These mechanisms are often inadequately captured by current predictors, especially those relying heavily on evolutionary conservation signals or structured-domain assumptions. Our findings strongly advocate for incorporating region-specific thresholds into VEPs to enhance their sensitivity and accuracy for disordered regions without substantially compromising specificity.</p>
<p>Furthermore, the notable discordance among predictors in classifying IDR variants underscores the importance of refining computational methodologies and training strategies tailored specifically for disordered protein regions. Future VEP development could potentially achieve improved performance and reliability by incorporating region-specific features such as transient binding motifs, PTM sites, and context-dependent structural ensembles. Recent large-scale simulations of human IDRs demonstrate that pathogenic variants are enriched in regions with low conformational entropy, providing a new structural dimension for interpreting variant effects in disordered regions [<a href="#pcbi.1013400.ref031" class="usa-link" aria-describedby="pcbi.1013400.ref031">31</a>]. In parallel, a recent proteome-wide analysis by Cagiada et al. [<a href="#pcbi.1013400.ref057" class="usa-link" aria-describedby="pcbi.1013400.ref057">57</a>] showed that pathogenic variants in IDRs more often disrupt function directly rather than stability, and that these variants are more difficult to model accurately, leading to higher false negative rates with current prediction approaches. This supports our conclusion that disordered regions involve distinct mechanisms of pathogenicity that are poorly captured by models trained predominantly on structured proteins.</p>
<p>Overall, this work provides critical insights into the complexities of interpreting genetic variation within intrinsically disordered regions. It establishes a foundational understanding for future development of more sophisticated computational tools, ultimately enhancing the accuracy of genetic variant interpretation in clinical settings and deepening our understanding of how protein disorder contributes to disease.</p></section><section id="sec009"><h2 class="pmc_sec_title">Methods</h2>
<section id="sec010"><h3 class="pmc_sec_title">Structural classification of human residues</h3>
<p>Every residue in the human proteome, considering the primary UniProt isoform of each protein-coding gene was given a structural classification of ordered, intermediate or disordered. To do this, we utilised the pLDDT derived from AF2 [<a href="#pcbi.1013400.ref016" class="usa-link" aria-describedby="pcbi.1013400.ref016">16</a>]. pLDDT inversely correlates very well with the flexibility of protein structures such that AF2 assigns low pLDDT scores for regions that are highly flexible and lack fixed 3D structures such as IDRs and linkers between ordered regions [<a href="#pcbi.1013400.ref017" class="usa-link" aria-describedby="pcbi.1013400.ref017">17</a>,<a href="#pcbi.1013400.ref018" class="usa-link" aria-describedby="pcbi.1013400.ref018">18</a>]. A residue is deemed to belong to a disordered region if its pLDDT is less than 50 and it is part of a contiguous stretch of at least 30 residues with an average pLDDT value less than 70. In contrast, a residue is classified as ordered if its pLDDT is at least 70. Finally, residues falling outside these two conditions were classified as intermediate.</p></section><section id="sec011"><h3 class="pmc_sec_title">Missense variant dataset</h3>
<p>Our dataset of missense variants was derived in essentially the same way as in our previous study [<a href="#pcbi.1013400.ref015" class="usa-link" aria-describedby="pcbi.1013400.ref015">15</a>], with pathogenic and likely pathogenic variants taken from ClinVar (August 2022) [<a href="#pcbi.1013400.ref023" class="usa-link" aria-describedby="pcbi.1013400.ref023">23</a>], and putatively benign population variants from gnomAD v2.1 [<a href="#pcbi.1013400.ref024" class="usa-link" aria-describedby="pcbi.1013400.ref024">24</a>]. Using gnomAD as a source of ‘putatively benign’ variants is preferable to clinically classified benign variants because it reduces circularity: many VEPs incorporate allele frequency information, which is often used to label clinical benign variants [<a href="#pcbi.1013400.ref058" class="usa-link" aria-describedby="pcbi.1013400.ref058">58</a>], creating a risk of inflated performance due to overlap between training and evaluation data. In contrast, using mostly rare gnomAD variants better reflects the real-world challenge of distinguishing rare benign from rare pathogenic variants while offering a larger, less biased negative class [<a href="#pcbi.1013400.ref041" class="usa-link" aria-describedby="pcbi.1013400.ref041">41</a>,<a href="#pcbi.1013400.ref059" class="usa-link" aria-describedby="pcbi.1013400.ref059">59</a>]. However, we also separately considered benign variants from ClinVar in <a href="#pcbi.1013400.g001" class="usa-link">Fig 1</a>. Importantly, we also removed all missense variants from collagen proteins, by excluding those 213 human protein-coding genes containing collagen-helices, as defined by Pfam (PF01391) [<a href="#pcbi.1013400.ref060" class="usa-link" aria-describedby="pcbi.1013400.ref060">60</a>]. It has been demonstrated previously that presence of collagen-helix containing proteins skew analyses of intrinsic disorder, since they are fibrous proteins but are consistently predicted to be disordered [<a href="#pcbi.1013400.ref061" class="usa-link" aria-describedby="pcbi.1013400.ref061">61</a>]. In our initial analysis, we found that nearly half of pathogenic missense variants in “disordered” regions according to our definition were from collagen proteins. As we do not consider collagens to be intrinsically disordered proteins, we excluded them from our study, and we strongly suggest that people take this into consideration in future studies of disease variants in disordered regions, as they have a high potential to cause confounding.</p></section><section id="sec012"><h3 class="pmc_sec_title">Assessing VEP performance and agreement</h3>
<p>We started with the set of VEPs tested in a recent benchmarking study (considering those used in the original preprint) [<a href="#pcbi.1013400.ref041" class="usa-link" aria-describedby="pcbi.1013400.ref041">41</a>]. We only retained those methods with scores available for at least 75% of missense variants present in our pathogenic and putatively benign datasets. To ensure consistency of comparisons, we only retained variants with scores shared across all VEPs.</p>
<p>For each VEP, we assessed its performance at distinguishing between pathogenic and putatively benign variants by calculating the AUROC value across different structural regions. We also calculated “optimal thresholds” for distinguishing between pathogenic and putatively benign variants, either on a global basis, or considering specific structural regions. To do this, we applied the Youden J-statistic [<a href="#pcbi.1013400.ref062" class="usa-link" aria-describedby="pcbi.1013400.ref062">62</a>] to the ROC curve and selected the threshold that maximised this value [<a href="#pcbi.1013400.ref062" class="usa-link" aria-describedby="pcbi.1013400.ref062">62</a>]. Using these thresholds, we could then assess whether each VEP predicted each variant to be pathogenic or benign in a binary manner. From this, we could calculate other metrics, including sensitivity and specificity. We also used this to calculate Cohen’s kappa, to assess the level of agreement in classification between each pair of VEPs.</p></section></section><section id="sec013"><h2 class="pmc_sec_title">Supporting information</h2>
<section class="sm xbox font-sm" id="pcbi.1013400.s001"><div class="caption p">
<span>S1 Text. Supplemental figures.</span><p>File containing additional figures complementary to the analysis shown in the main text of this manuscript.</p>
<p>(DOCX)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12377588/bin/pcbi.1013400.s001.docx" data-ga-action="click_feat_suppl" class="usa-link">pcbi.1013400.s001.docx</a><sup> (375.2KB, docx) </sup>
</div></div></section><section class="sm xbox font-sm" id="pcbi.1013400.s002"><div class="caption p">
<span>S1 Table. Global and region-specific optimal thresholds across all VEPs.</span><p>(DOCX)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12377588/bin/pcbi.1013400.s002.docx" data-ga-action="click_feat_suppl" class="usa-link">pcbi.1013400.s002.docx</a><sup> (260.1KB, docx) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgments</h2>
<p>We thank Benjamin Livesey and Mihaly Badonyi for helpful comments on the manuscript. This work has made use of the resources provided by the Edinburgh Compute and Data Facility (ECDF) (<a href="http://www.ecdf.ed.ac.uk/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://www.ecdf.ed.ac.uk/</a>).</p></section><section id="notes1"><h2 class="pmc_sec_title">Data Availability</h2>
<p>Associated datasets are available at <a href="https://doi.org/10.6084/m9.figshare.c.7747895.v1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://doi.org/10.6084/m9.figshare.c.7747895.v1</a> and the pipeline code is shared at <a href="https://github.com/drsamibioinfo/VEPS_IN_DISORDER/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/drsamibioinfo/VEPS_IN_DISORDER/</a>.</p></section><section id="funding-statement1" lang="en"><h2 class="pmc_sec_title">Funding Statement</h2>
<p>This project was supported by funding to JAM from the European Research Council (ERC) under the European Union's Horizon 2020 research and innovation programme (grant agreement No. 101001169) and by the Medical Research Council (MRC) Human Genetics Unit core grant (MC_UU_00035/9). The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.</p></section><section id="ref-list1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="ref-list1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="pcbi.1013400.ref001">
<span class="label">1.</span><cite>Das RK, Pappu RV. Conformations of intrinsically disordered proteins are influenced by linear sequence distributions of oppositely charged residues. Proc Natl Acad Sci U S A. 2013;110(33):13392–7. doi: 10.1073/pnas.1304749110

</cite> [<a href="https://doi.org/10.1073/pnas.1304749110" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3746876/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23901099/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&amp;title=Conformations%20of%20intrinsically%20disordered%20proteins%20are%20influenced%20by%20linear%20sequence%20distributions%20of%20oppositely%20charged%20residues&amp;author=RK%20Das&amp;author=RV%20Pappu&amp;volume=110&amp;issue=33&amp;publication_year=2013&amp;pages=13392-7&amp;pmid=23901099&amp;doi=10.1073/pnas.1304749110&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013400.ref002">
<span class="label">2.</span><cite>Lermyte F. Roles, Characteristics, and Analysis of Intrinsically Disordered Proteins: A Minireview. Life (Basel). 2020;10(12):320. doi: 10.3390/life10120320

</cite> [<a href="https://doi.org/10.3390/life10120320" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7761095/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33266184/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Life%20(Basel)&amp;title=Roles,%20Characteristics,%20and%20Analysis%20of%20Intrinsically%20Disordered%20Proteins:%20A%20Minireview&amp;author=F%20Lermyte&amp;volume=10&amp;issue=12&amp;publication_year=2020&amp;pages=320&amp;pmid=33266184&amp;doi=10.3390/life10120320&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013400.ref003">
<span class="label">3.</span><cite>Pajkos M, Dosztányi Z. Functions of intrinsically disordered proteins through evolutionary lenses. Prog Mol Biol Transl Sci. 2021;183:45–74. doi: 10.1016/bs.pmbts.2021.06.017

</cite> [<a href="https://doi.org/10.1016/bs.pmbts.2021.06.017" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34656334/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Prog%20Mol%20Biol%20Transl%20Sci&amp;title=Functions%20of%20intrinsically%20disordered%20proteins%20through%20evolutionary%20lenses&amp;author=M%20Pajkos&amp;author=Z%20Doszt%C3%A1nyi&amp;volume=183&amp;publication_year=2021&amp;pages=45-74&amp;pmid=34656334&amp;doi=10.1016/bs.pmbts.2021.06.017&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013400.ref004">
<span class="label">4.</span><cite>Kurgan L, Hu G, Wang K, Ghadermarzi S, Zhao B, Malhis N, et al. Tutorial: a guide for the selection of fast and accurate computational tools for the prediction of intrinsic disorder in proteins. Nat Protoc. 2023;18(11):3157–72. doi: 10.1038/s41596-023-00876-x

</cite> [<a href="https://doi.org/10.1038/s41596-023-00876-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37740110/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Protoc&amp;title=Tutorial:%20a%20guide%20for%20the%20selection%20of%20fast%20and%20accurate%20computational%20tools%20for%20the%20prediction%20of%20intrinsic%20disorder%20in%20proteins&amp;author=L%20Kurgan&amp;author=G%20Hu&amp;author=K%20Wang&amp;author=S%20Ghadermarzi&amp;author=B%20Zhao&amp;volume=18&amp;issue=11&amp;publication_year=2023&amp;pages=3157-72&amp;pmid=37740110&amp;doi=10.1038/s41596-023-00876-x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013400.ref005">
<span class="label">5.</span><cite>Lazar T, Tantos A, Tompa P, Schad E. Intrinsic protein disorder uncouples affinity from binding specificity. Protein Sci. 2022;31(11):e4455. doi: 10.1002/pro.4455

</cite> [<a href="https://doi.org/10.1002/pro.4455" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9601785/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36305763/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Protein%20Sci&amp;title=Intrinsic%20protein%20disorder%20uncouples%20affinity%20from%20binding%20specificity&amp;author=T%20Lazar&amp;author=A%20Tantos&amp;author=P%20Tompa&amp;author=E%20Schad&amp;volume=31&amp;issue=11&amp;publication_year=2022&amp;pmid=36305763&amp;doi=10.1002/pro.4455&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013400.ref006">
<span class="label">6.</span><cite>Lee T-Y, Huang H-D, Hung J-H, Huang H-Y, Yang Y-S, Wang T-H. dbPTM: an information repository of protein post-translational modification. Nucleic Acids Res. 2006;34(Database issue):D622-7. doi: 10.1093/nar/gkj083

</cite> [<a href="https://doi.org/10.1093/nar/gkj083" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1347446/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16381945/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=dbPTM:%20an%20information%20repository%20of%20protein%20post-translational%20modification&amp;author=T-Y%20Lee&amp;author=H-D%20Huang&amp;author=J-H%20Hung&amp;author=H-Y%20Huang&amp;author=Y-S%20Yang&amp;volume=34&amp;publication_year=2006&amp;pmid=16381945&amp;doi=10.1093/nar/gkj083&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013400.ref007">
<span class="label">7.</span><cite>Uversky VN. p53 Proteoforms and Intrinsic Disorder: An Illustration of the Protein Structure-Function Continuum Concept. Int J Mol Sci. 2016;17(11):1874. doi: 10.3390/ijms17111874

</cite> [<a href="https://doi.org/10.3390/ijms17111874" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5133874/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27834926/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Int%20J%20Mol%20Sci&amp;title=p53%20Proteoforms%20and%20Intrinsic%20Disorder:%20An%20Illustration%20of%20the%20Protein%20Structure-Function%20Continuum%20Concept&amp;author=VN%20Uversky&amp;volume=17&amp;issue=11&amp;publication_year=2016&amp;pages=1874&amp;pmid=27834926&amp;doi=10.3390/ijms17111874&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013400.ref008">
<span class="label">8.</span><cite>Uversky VN. Protein intrinsic disorder and structure-function continuum. Prog Mol Biol Transl Sci. 2019;166:1–17. doi: 10.1016/bs.pmbts.2019.05.003

</cite> [<a href="https://doi.org/10.1016/bs.pmbts.2019.05.003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31521229/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Prog%20Mol%20Biol%20Transl%20Sci&amp;title=Protein%20intrinsic%20disorder%20and%20structure-function%20continuum&amp;author=VN%20Uversky&amp;volume=166&amp;publication_year=2019&amp;pages=1-17&amp;pmid=31521229&amp;doi=10.1016/bs.pmbts.2019.05.003&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013400.ref009">
<span class="label">9.</span><cite>Uversky VN, Oldfield CJ, Dunker AK. Intrinsically disordered proteins in human diseases: introducing the D2 concept. Annu Rev Biophys. 2008;37:215–46. doi: 10.1146/annurev.biophys.37.032807.125924

</cite> [<a href="https://doi.org/10.1146/annurev.biophys.37.032807.125924" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18573080/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Annu%20Rev%20Biophys&amp;title=Intrinsically%20disordered%20proteins%20in%20human%20diseases:%20introducing%20the%20D2%20concept&amp;author=VN%20Uversky&amp;author=CJ%20Oldfield&amp;author=AK%20Dunker&amp;volume=37&amp;publication_year=2008&amp;pages=215-46&amp;pmid=18573080&amp;doi=10.1146/annurev.biophys.37.032807.125924&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013400.ref010">
<span class="label">10.</span><cite>Dunker AK, Lawson JD, Brown CJ, Williams RM, Romero P, Oh JS, et al. Intrinsically disordered protein. J Mol Graph Model. 2001;19(1):26–59. doi: 10.1016/s1093-3263(00)00138-8

</cite> [<a href="https://doi.org/10.1016/s1093-3263(00)00138-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11381529/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Mol%20Graph%20Model&amp;title=Intrinsically%20disordered%20protein&amp;author=AK%20Dunker&amp;author=JD%20Lawson&amp;author=CJ%20Brown&amp;author=RM%20Williams&amp;author=P%20Romero&amp;volume=19&amp;issue=1&amp;publication_year=2001&amp;pages=26-59&amp;pmid=11381529&amp;doi=10.1016/s1093-3263(00)00138-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013400.ref011">
<span class="label">11.</span><cite>Uversky VN. What does it mean to be natively unfolded?. Eur J Biochem. 2002;269(1):2–12. doi: 10.1046/j.0014-2956.2001.02649.x

</cite> [<a href="https://doi.org/10.1046/j.0014-2956.2001.02649.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11784292/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Biochem&amp;title=What%20does%20it%20mean%20to%20be%20natively%20unfolded?&amp;author=VN%20Uversky&amp;volume=269&amp;issue=1&amp;publication_year=2002&amp;pages=2-12&amp;pmid=11784292&amp;doi=10.1046/j.0014-2956.2001.02649.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013400.ref012">
<span class="label">12.</span><cite>Vovk A, Zilman A. Effects of Sequence Composition, Patterning and Hydrodynamics on the Conformation and Dynamics of Intrinsically Disordered Proteins. Int J Mol Sci. 2023;24(2):1444. doi: 10.3390/ijms24021444

</cite> [<a href="https://doi.org/10.3390/ijms24021444" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9867189/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36674958/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Int%20J%20Mol%20Sci&amp;title=Effects%20of%20Sequence%20Composition,%20Patterning%20and%20Hydrodynamics%20on%20the%20Conformation%20and%20Dynamics%20of%20Intrinsically%20Disordered%20Proteins&amp;author=A%20Vovk&amp;author=A%20Zilman&amp;volume=24&amp;issue=2&amp;publication_year=2023&amp;pages=1444&amp;pmid=36674958&amp;doi=10.3390/ijms24021444&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013400.ref013">
<span class="label">13.</span><cite>Livesey BJ, Badonyi M, Dias M, Frazer J, Kumar S, Lindorff-Larsen K, et al. Guidelines for releasing a variant effect predictor. Genome Biol. 2025;26(1):97. doi: 10.1186/s13059-025-03572-z

</cite> [<a href="https://doi.org/10.1186/s13059-025-03572-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11998465/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40234898/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genome%20Biol&amp;title=Guidelines%20for%20releasing%20a%20variant%20effect%20predictor&amp;author=BJ%20Livesey&amp;author=M%20Badonyi&amp;author=M%20Dias&amp;author=J%20Frazer&amp;author=S%20Kumar&amp;volume=26&amp;issue=1&amp;publication_year=2025&amp;pages=97&amp;pmid=40234898&amp;doi=10.1186/s13059-025-03572-z&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013400.ref014">
<span class="label">14.</span><cite>Vacic V, Markwick PRL, Oldfield CJ, Zhao X, Haynes C, Uversky VN, et al. Disease-associated mutations disrupt functionally important regions of intrinsic protein disorder. PLoS Comput Biol. 2012;8(10):e1002709. doi: 10.1371/journal.pcbi.1002709

</cite> [<a href="https://doi.org/10.1371/journal.pcbi.1002709" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3464192/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23055912/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20Comput%20Biol&amp;title=Disease-associated%20mutations%20disrupt%20functionally%20important%20regions%20of%20intrinsic%20protein%20disorder&amp;author=V%20Vacic&amp;author=PRL%20Markwick&amp;author=CJ%20Oldfield&amp;author=X%20Zhao&amp;author=C%20Haynes&amp;volume=8&amp;issue=10&amp;publication_year=2012&amp;pmid=23055912&amp;doi=10.1371/journal.pcbi.1002709&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013400.ref015">
<span class="label">15.</span><cite>Fawzy M, Marsh JA. Understanding the heterogeneous performance of variant effect predictors across human protein-coding genes. Sci Rep. 2024;14(1):26114. doi: 10.1038/s41598-024-76202-6

</cite> [<a href="https://doi.org/10.1038/s41598-024-76202-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11526010/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39478110/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sci%20Rep&amp;title=Understanding%20the%20heterogeneous%20performance%20of%20variant%20effect%20predictors%20across%20human%20protein-coding%20genes&amp;author=M%20Fawzy&amp;author=JA%20Marsh&amp;volume=14&amp;issue=1&amp;publication_year=2024&amp;pages=26114&amp;pmid=39478110&amp;doi=10.1038/s41598-024-76202-6&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013400.ref016">
<span class="label">16.</span><cite>Jumper J, Evans R, Pritzel A, Green T, Figurnov M, Ronneberger O, et al. Highly accurate protein structure prediction with AlphaFold. Nature. 2021;596(7873):583–9. doi: 10.1038/s41586-021-03819-2

</cite> [<a href="https://doi.org/10.1038/s41586-021-03819-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8371605/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34265844/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Highly%20accurate%20protein%20structure%20prediction%20with%20AlphaFold&amp;author=J%20Jumper&amp;author=R%20Evans&amp;author=A%20Pritzel&amp;author=T%20Green&amp;author=M%20Figurnov&amp;volume=596&amp;issue=7873&amp;publication_year=2021&amp;pages=583-9&amp;pmid=34265844&amp;doi=10.1038/s41586-021-03819-2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013400.ref017">
<span class="label">17.</span><cite>Tunyasuvunakool K, Adler J, Wu Z, Green T, Zielinski M, Žídek A, et al. Highly accurate protein structure prediction for the human proteome. Nature. 2021;596(7873):590–6. doi: 10.1038/s41586-021-03828-1

</cite> [<a href="https://doi.org/10.1038/s41586-021-03828-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8387240/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34293799/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Highly%20accurate%20protein%20structure%20prediction%20for%20the%20human%20proteome&amp;author=K%20Tunyasuvunakool&amp;author=J%20Adler&amp;author=Z%20Wu&amp;author=T%20Green&amp;author=M%20Zielinski&amp;volume=596&amp;issue=7873&amp;publication_year=2021&amp;pages=590-6&amp;pmid=34293799&amp;doi=10.1038/s41586-021-03828-1&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013400.ref018">
<span class="label">18.</span><cite>Alderson TR, Pritišanac I, Kolarić Đ, Moses AM, Forman-Kay JD. Systematic identification of conditionally folded intrinsically disordered regions by AlphaFold2. Proc Natl Acad Sci U S A. 2023;120(44):e2304302120. doi: 10.1073/pnas.2304302120

</cite> [<a href="https://doi.org/10.1073/pnas.2304302120" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10622901/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37878721/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&amp;title=Systematic%20identification%20of%20conditionally%20folded%20intrinsically%20disordered%20regions%20by%20AlphaFold2&amp;author=TR%20Alderson&amp;author=I%20Priti%C5%A1anac&amp;author=%C4%90%20Kolari%C4%87&amp;author=AM%20Moses&amp;author=JD%20Forman-Kay&amp;volume=120&amp;issue=44&amp;publication_year=2023&amp;pmid=37878721&amp;doi=10.1073/pnas.2304302120&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013400.ref019">
<span class="label">19.</span><cite>Wilson CJ, Choy W-Y, Karttunen M. AlphaFold2: A Role for Disordered Protein/Region Prediction?. Int J Mol Sci. 2022;23(9):4591. doi: 10.3390/ijms23094591

</cite> [<a href="https://doi.org/10.3390/ijms23094591" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9104326/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35562983/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Int%20J%20Mol%20Sci&amp;title=AlphaFold2:%20A%20Role%20for%20Disordered%20Protein/Region%20Prediction?&amp;author=CJ%20Wilson&amp;author=W-Y%20Choy&amp;author=M%20Karttunen&amp;volume=23&amp;issue=9&amp;publication_year=2022&amp;pages=4591&amp;pmid=35562983&amp;doi=10.3390/ijms23094591&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013400.ref020">
<span class="label">20.</span><cite>Piovesan D, Monzon AM, Tosatto SCE. Intrinsic protein disorder and conditional folding in AlphaFoldDB. Protein Sci. 2022;31(11):e4466. doi: 10.1002/pro.4466

</cite> [<a href="https://doi.org/10.1002/pro.4466" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9601767/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36210722/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Protein%20Sci&amp;title=Intrinsic%20protein%20disorder%20and%20conditional%20folding%20in%20AlphaFoldDB&amp;author=D%20Piovesan&amp;author=AM%20Monzon&amp;author=SCE%20Tosatto&amp;volume=31&amp;issue=11&amp;publication_year=2022&amp;pmid=36210722&amp;doi=10.1002/pro.4466&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013400.ref021">
<span class="label">21.</span><cite>Conte AD, Mehdiabadi M, Bouhraoua A, Miguel Monzon A, Tosatto SCE, Piovesan D. Critical assessment of protein intrinsic disorder prediction (CAID) - Results of round 2. Proteins. 2023;91(12):1925–34. doi: 10.1002/prot.26582

</cite> [<a href="https://doi.org/10.1002/prot.26582" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37621223/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proteins&amp;title=Critical%20assessment%20of%20protein%20intrinsic%20disorder%20prediction%20(CAID)%20-%20Results%20of%20round%202&amp;author=AD%20Conte&amp;author=M%20Mehdiabadi&amp;author=A%20Bouhraoua&amp;author=A%20Miguel%20Monzon&amp;author=SCE%20Tosatto&amp;volume=91&amp;issue=12&amp;publication_year=2023&amp;pages=1925-34&amp;pmid=37621223&amp;doi=10.1002/prot.26582&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013400.ref022">
<span class="label">22.</span><cite>Pentony MM, Jones DT. Modularity of intrinsic disorder in the human proteome. Proteins. 2010;78(1):212–21. doi: 10.1002/prot.22504

</cite> [<a href="https://doi.org/10.1002/prot.22504" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19626706/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proteins&amp;title=Modularity%20of%20intrinsic%20disorder%20in%20the%20human%20proteome&amp;author=MM%20Pentony&amp;author=DT%20Jones&amp;volume=78&amp;issue=1&amp;publication_year=2010&amp;pages=212-21&amp;pmid=19626706&amp;doi=10.1002/prot.22504&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013400.ref023">
<span class="label">23.</span><cite>Landrum MJ, Lee JM, Riley GR, Jang W, Rubinstein WS, Church DM, et al. ClinVar: public archive of relationships among sequence variation and human phenotype. Nucleic Acids Res. 2014;42(Database issue):D980-5. doi: 10.1093/nar/gkt1113

</cite> [<a href="https://doi.org/10.1093/nar/gkt1113" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3965032/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24234437/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=ClinVar:%20public%20archive%20of%20relationships%20among%20sequence%20variation%20and%20human%20phenotype&amp;author=MJ%20Landrum&amp;author=JM%20Lee&amp;author=GR%20Riley&amp;author=W%20Jang&amp;author=WS%20Rubinstein&amp;volume=42&amp;publication_year=2014&amp;pmid=24234437&amp;doi=10.1093/nar/gkt1113&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013400.ref024">
<span class="label">24.</span><cite>Karczewski KJ, Francioli LC, Tiao G, Cummings BB, Alföldi J, Wang Q, et al. The mutational constraint spectrum quantified from variation in 141,456 humans. Nature. 2020;581(7809):434–43. doi: 10.1038/s41586-020-2308-7

</cite> [<a href="https://doi.org/10.1038/s41586-020-2308-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7334197/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32461654/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=The%20mutational%20constraint%20spectrum%20quantified%20from%20variation%20in%20141,456%20humans&amp;author=KJ%20Karczewski&amp;author=LC%20Francioli&amp;author=G%20Tiao&amp;author=BB%20Cummings&amp;author=J%20Alf%C3%B6ldi&amp;volume=581&amp;issue=7809&amp;publication_year=2020&amp;pages=434-43&amp;pmid=32461654&amp;doi=10.1038/s41586-020-2308-7&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013400.ref025">
<span class="label">25.</span><cite>Feng M, Wei X, Zheng X, Liu L, Lin L, Xia M, et al. Decoding Missense Variants by Incorporating Phase Separation via Machine Learning. Nat Commun. 2024;15(1):8279. doi: 10.1038/s41467-024-52580-3

</cite> [<a href="https://doi.org/10.1038/s41467-024-52580-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11436885/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39333476/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Commun&amp;title=Decoding%20Missense%20Variants%20by%20Incorporating%20Phase%20Separation%20via%20Machine%20Learning&amp;author=M%20Feng&amp;author=X%20Wei&amp;author=X%20Zheng&amp;author=L%20Liu&amp;author=L%20Lin&amp;volume=15&amp;issue=1&amp;publication_year=2024&amp;pages=8279&amp;pmid=39333476&amp;doi=10.1038/s41467-024-52580-3&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013400.ref026">
<span class="label">26.</span><cite>Lee KE, Pulido JS, da Palma MM, Procopio R, Hufnagel RB, Reynolds M. A Comprehensive Report of Intrinsically Disordered Regions in Inherited Retinal Diseases. Genes (Basel). 2023;14(8):1601. doi: 10.3390/genes14081601

</cite> [<a href="https://doi.org/10.3390/genes14081601" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10454668/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37628652/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genes%20(Basel)&amp;title=A%20Comprehensive%20Report%20of%20Intrinsically%20Disordered%20Regions%20in%20Inherited%20Retinal%20Diseases&amp;author=KE%20Lee&amp;author=JS%20Pulido&amp;author=MM%20da%20Palma&amp;author=R%20Procopio&amp;author=RB%20Hufnagel&amp;volume=14&amp;issue=8&amp;publication_year=2023&amp;pages=1601&amp;pmid=37628652&amp;doi=10.3390/genes14081601&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013400.ref027">
<span class="label">27.</span><cite>Wong ETC, So V, Guron M, Kuechler ER, Malhis N, Bui JM, et al. Protein-Protein Interactions Mediated by Intrinsically Disordered Protein Regions Are Enriched in Missense Mutations. Biomolecules. 2020;10(8):1097. doi: 10.3390/biom10081097

</cite> [<a href="https://doi.org/10.3390/biom10081097" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7463635/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32722039/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biomolecules&amp;title=Protein-Protein%20Interactions%20Mediated%20by%20Intrinsically%20Disordered%20Protein%20Regions%20Are%20Enriched%20in%20Missense%20Mutations&amp;author=ETC%20Wong&amp;author=V%20So&amp;author=M%20Guron&amp;author=ER%20Kuechler&amp;author=N%20Malhis&amp;volume=10&amp;issue=8&amp;publication_year=2020&amp;pages=1097&amp;pmid=32722039&amp;doi=10.3390/biom10081097&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013400.ref028">
<span class="label">28.</span><cite>Iakoucheva LM, Brown CJ, Lawson JD, Obradović Z, Dunker AK. Intrinsic disorder in cell-signaling and cancer-associated proteins. J Mol Biol. 2002;323(3):573–84. doi: 10.1016/s0022-2836(02)00969-5

</cite> [<a href="https://doi.org/10.1016/s0022-2836(02)00969-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12381310/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Mol%20Biol&amp;title=Intrinsic%20disorder%20in%20cell-signaling%20and%20cancer-associated%20proteins&amp;author=LM%20Iakoucheva&amp;author=CJ%20Brown&amp;author=JD%20Lawson&amp;author=Z%20Obradovi%C4%87&amp;author=AK%20Dunker&amp;volume=323&amp;issue=3&amp;publication_year=2002&amp;pages=573-84&amp;pmid=12381310&amp;doi=10.1016/s0022-2836(02)00969-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013400.ref029">
<span class="label">29.</span><cite>Flores E, Acharya N, Castañeda CA, Sukenik S. Single-point mutations in disordered proteins: Linking sequence, ensemble, and function. Curr Opin Struct Biol. 2025;91:102987. doi: 10.1016/j.sbi.2025.102987

</cite> [<a href="https://doi.org/10.1016/j.sbi.2025.102987" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC12205430/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39914051/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Curr%20Opin%20Struct%20Biol&amp;title=Single-point%20mutations%20in%20disordered%20proteins:%20Linking%20sequence,%20ensemble,%20and%20function&amp;author=E%20Flores&amp;author=N%20Acharya&amp;author=CA%20Casta%C3%B1eda&amp;author=S%20Sukenik&amp;volume=91&amp;publication_year=2025&amp;pages=102987&amp;pmid=39914051&amp;doi=10.1016/j.sbi.2025.102987&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013400.ref030">
<span class="label">30.</span><cite>Boija A, Klein IA, Sabari BR, Dall’Agnese A, Coffey EL, Zamudio AV, et al. Transcription Factors Activate Genes through the Phase-Separation Capacity of Their Activation Domains. Cell. 2018;175(7):1842-1855.e16. doi: 10.1016/j.cell.2018.10.042

</cite> [<a href="https://doi.org/10.1016/j.cell.2018.10.042" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6295254/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30449618/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Transcription%20Factors%20Activate%20Genes%20through%20the%20Phase-Separation%20Capacity%20of%20Their%20Activation%20Domains&amp;author=A%20Boija&amp;author=IA%20Klein&amp;author=BR%20Sabari&amp;author=A%20Dall%E2%80%99Agnese&amp;author=EL%20Coffey&amp;volume=175&amp;issue=7&amp;publication_year=2018&amp;pmid=30449618&amp;doi=10.1016/j.cell.2018.10.042&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013400.ref031">
<span class="label">31.</span><cite>Tesei G, Trolle AI, Jonsson N, Betz J, Knudsen FE, Pesce F, et al. Conformational ensembles of the human intrinsically disordered proteome. Nature. 2024;626(8000):897–904. doi: 10.1038/s41586-023-07004-5

</cite> [<a href="https://doi.org/10.1038/s41586-023-07004-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38297118/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Conformational%20ensembles%20of%20the%20human%20intrinsically%20disordered%20proteome&amp;author=G%20Tesei&amp;author=AI%20Trolle&amp;author=N%20Jonsson&amp;author=J%20Betz&amp;author=FE%20Knudsen&amp;volume=626&amp;issue=8000&amp;publication_year=2024&amp;pages=897-904&amp;pmid=38297118&amp;doi=10.1038/s41586-023-07004-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013400.ref032">
<span class="label">32.</span><cite>Gerasimavicius L, Livesey BJ, Marsh JA. Loss-of-function, gain-of-function and dominant-negative mutations have profoundly different effects on protein structure. Nat Commun. 2022;13(1):3895. doi: 10.1038/s41467-022-31686-6

</cite> [<a href="https://doi.org/10.1038/s41467-022-31686-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9259657/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35794153/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Commun&amp;title=Loss-of-function,%20gain-of-function%20and%20dominant-negative%20mutations%20have%20profoundly%20different%20effects%20on%20protein%20structure&amp;author=L%20Gerasimavicius&amp;author=BJ%20Livesey&amp;author=JA%20Marsh&amp;volume=13&amp;issue=1&amp;publication_year=2022&amp;pages=3895&amp;pmid=35794153&amp;doi=10.1038/s41467-022-31686-6&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013400.ref033">
<span class="label">33.</span><cite>Backwell L, Marsh JA. Diverse Molecular Mechanisms Underlying Pathogenic Protein Mutations: Beyond the Loss-of-Function Paradigm. Annu Rev Genomics Hum Genet. 2022;23:475–98. doi: 10.1146/annurev-genom-111221-103208

</cite> [<a href="https://doi.org/10.1146/annurev-genom-111221-103208" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35395171/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Annu%20Rev%20Genomics%20Hum%20Genet&amp;title=Diverse%20Molecular%20Mechanisms%20Underlying%20Pathogenic%20Protein%20Mutations:%20Beyond%20the%20Loss-of-Function%20Paradigm&amp;author=L%20Backwell&amp;author=JA%20Marsh&amp;volume=23&amp;publication_year=2022&amp;pages=475-98&amp;pmid=35395171&amp;doi=10.1146/annurev-genom-111221-103208&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013400.ref034">
<span class="label">34.</span><cite>Badonyi M, Marsh JA. Proteome-scale prediction of molecular mechanisms underlying dominant genetic diseases. PLoS One. 2024;19(8):e0307312. doi: 10.1371/journal.pone.0307312

</cite> [<a href="https://doi.org/10.1371/journal.pone.0307312" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11341024/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39172982/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20One&amp;title=Proteome-scale%20prediction%20of%20molecular%20mechanisms%20underlying%20dominant%20genetic%20diseases&amp;author=M%20Badonyi&amp;author=JA%20Marsh&amp;volume=19&amp;issue=8&amp;publication_year=2024&amp;pmid=39172982&amp;doi=10.1371/journal.pone.0307312&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013400.ref035">
<span class="label">35.</span><cite>Seidman JG, Seidman C. Transcription factor haploinsufficiency: when half a loaf is not enough. J Clin Invest. 2002;109(4):451–5. doi: 10.1172/JCI15043

</cite> [<a href="https://doi.org/10.1172/JCI15043" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC150881/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11854316/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Invest&amp;title=Transcription%20factor%20haploinsufficiency:%20when%20half%20a%20loaf%20is%20not%20enough&amp;author=JG%20Seidman&amp;author=C%20Seidman&amp;volume=109&amp;issue=4&amp;publication_year=2002&amp;pages=451-5&amp;pmid=11854316&amp;doi=10.1172/JCI15043&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013400.ref036">
<span class="label">36.</span><cite>Wright PE, Dyson HJ. Intrinsically disordered proteins in cellular signalling and regulation. Nat Rev Mol Cell Biol. 2015;16(1):18–29. doi: 10.1038/nrm3920

</cite> [<a href="https://doi.org/10.1038/nrm3920" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4405151/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25531225/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Mol%20Cell%20Biol&amp;title=Intrinsically%20disordered%20proteins%20in%20cellular%20signalling%20and%20regulation&amp;author=PE%20Wright&amp;author=HJ%20Dyson&amp;volume=16&amp;issue=1&amp;publication_year=2015&amp;pages=18-29&amp;pmid=25531225&amp;doi=10.1038/nrm3920&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013400.ref037">
<span class="label">37.</span><cite>Ritterhouse LL, Barletta JA. BRAF V600E mutation-specific antibody: A review. Semin Diagn Pathol. 2015;32(5):400–8. doi: 10.1053/j.semdp.2015.02.010

</cite> [<a href="https://doi.org/10.1053/j.semdp.2015.02.010" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25744437/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Semin%20Diagn%20Pathol&amp;title=BRAF%20V600E%20mutation-specific%20antibody:%20A%20review&amp;author=LL%20Ritterhouse&amp;author=JA%20Barletta&amp;volume=32&amp;issue=5&amp;publication_year=2015&amp;pages=400-8&amp;pmid=25744437&amp;doi=10.1053/j.semdp.2015.02.010&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013400.ref038">
<span class="label">38.</span><cite>Davies H, Bignell GR, Cox C, Stephens P, Edkins S, Clegg S, et al. Mutations of the BRAF gene in human cancer. Nature. 2002;417(6892):949–54. doi: 10.1038/nature00766

</cite> [<a href="https://doi.org/10.1038/nature00766" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12068308/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Mutations%20of%20the%20BRAF%20gene%20in%20human%20cancer&amp;author=H%20Davies&amp;author=GR%20Bignell&amp;author=C%20Cox&amp;author=P%20Stephens&amp;author=S%20Edkins&amp;volume=417&amp;issue=6892&amp;publication_year=2002&amp;pages=949-54&amp;pmid=12068308&amp;doi=10.1038/nature00766&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013400.ref039">
<span class="label">39.</span><cite>St. Jude Children’s Research Hospital–Washington University Pediatric Cancer Genome Project, Wu G, Broniscer A, McEachron TA, Lu C, Paugh BS, Becksfort J, et al. Somatic histone H3 alterations in pediatric diffuse intrinsic pontine gliomas and non-brainstem glioblastomas. Nat Genet. 2012;44(3):251–3. doi: 10.1038/ng.1102

</cite> [<a href="https://doi.org/10.1038/ng.1102" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3288377/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22286216/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&amp;title=Somatic%20histone%20H3%20alterations%20in%20pediatric%20diffuse%20intrinsic%20pontine%20gliomas%20and%20non-brainstem%20glioblastomas&amp;author=G%20Wu&amp;author=A%20Broniscer&amp;author=TA%20McEachron&amp;author=C%20Lu&amp;author=BS%20Paugh&amp;volume=44&amp;issue=3&amp;publication_year=2012&amp;pages=251-3&amp;pmid=22286216&amp;doi=10.1038/ng.1102&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013400.ref040">
<span class="label">40.</span><cite>Herman L, Amo A, Legois B, Di Carlo C, Veitia RA, Todeschini A-L. A cellular model provides insights into the pathogenicity of the oncogenic FOXL2 somatic variant p.Cys134Trp. Br J Cancer. 2024;130(9):1453–62. doi: 10.1038/s41416-024-02613-x

</cite> [<a href="https://doi.org/10.1038/s41416-024-02613-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11059147/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38429437/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Br%20J%20Cancer&amp;title=A%20cellular%20model%20provides%20insights%20into%20the%20pathogenicity%20of%20the%20oncogenic%20FOXL2%20somatic%20variant%20p.Cys134Trp&amp;author=L%20Herman&amp;author=A%20Amo&amp;author=B%20Legois&amp;author=C%20Di%20Carlo&amp;author=RA%20Veitia&amp;volume=130&amp;issue=9&amp;publication_year=2024&amp;pages=1453-62&amp;pmid=38429437&amp;doi=10.1038/s41416-024-02613-x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013400.ref041">
<span class="label">41.</span><cite>Livesey BJ, Marsh JA. Variant effect predictor correlation with functional assays is reflective of clinical classification performance. Genome Biol. 2025;26(1):104. doi: 10.1186/s13059-025-03575-w

</cite> [<a href="https://doi.org/10.1186/s13059-025-03575-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC12016141/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40264194/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genome%20Biol&amp;title=Variant%20effect%20predictor%20correlation%20with%20functional%20assays%20is%20reflective%20of%20clinical%20classification%20performance&amp;author=BJ%20Livesey&amp;author=JA%20Marsh&amp;volume=26&amp;issue=1&amp;publication_year=2025&amp;pages=104&amp;pmid=40264194&amp;doi=10.1186/s13059-025-03575-w&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013400.ref042">
<span class="label">42.</span><cite>Pathak AK, Bora N, Badonyi M, Livesey BJ, Consortium S, Ngeow J, et al. Pervasive ancestry bias in variant effect predictors. bioRxiv. 2025:2024.05.20.594987. doi: 10.1101/2024.05.20.594987</cite> [<a href="https://doi.org/10.1101/2024.05.20.594987" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=bioRxiv&amp;title=Pervasive%20ancestry%20bias%20in%20variant%20effect%20predictors&amp;author=AK%20Pathak&amp;author=N%20Bora&amp;author=M%20Badonyi&amp;author=BJ%20Livesey&amp;author=S%20Consortium&amp;publication_year=2025&amp;doi=10.1101/2024.05.20.594987&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013400.ref043">
<span class="label">43.</span><cite>Grimm DG, Azencott C-A, Aicheler F, Gieraths U, MacArthur DG, Samocha KE, et al. The evaluation of tools used to predict the impact of missense variants is hindered by two types of circularity. Hum Mutat. 2015;36(5):513–23. doi: 10.1002/humu.22768

</cite> [<a href="https://doi.org/10.1002/humu.22768" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4409520/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25684150/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Hum%20Mutat&amp;title=The%20evaluation%20of%20tools%20used%20to%20predict%20the%20impact%20of%20missense%20variants%20is%20hindered%20by%20two%20types%20of%20circularity&amp;author=DG%20Grimm&amp;author=C-A%20Azencott&amp;author=F%20Aicheler&amp;author=U%20Gieraths&amp;author=DG%20MacArthur&amp;volume=36&amp;issue=5&amp;publication_year=2015&amp;pages=513-23&amp;pmid=25684150&amp;doi=10.1002/humu.22768&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013400.ref044">
<span class="label">44.</span><cite>Cheng J, Novati G, Pan J, Bycroft C, Žemgulytė A, Applebaum T, et al. Accurate proteome-wide missense variant effect prediction with AlphaMissense. Science. 2023;381(6664):eadg7492. doi: 10.1126/science.adg7492

</cite> [<a href="https://doi.org/10.1126/science.adg7492" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37733863/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Accurate%20proteome-wide%20missense%20variant%20effect%20prediction%20with%20AlphaMissense&amp;author=J%20Cheng&amp;author=G%20Novati&amp;author=J%20Pan&amp;author=C%20Bycroft&amp;author=A%20%C5%BDemgulyt%C4%97&amp;volume=381&amp;issue=6664&amp;publication_year=2023&amp;pmid=37733863&amp;doi=10.1126/science.adg7492&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013400.ref045">
<span class="label">45.</span><cite>Huang Y-F. Unified inference of missense variant effects and gene constraints in the human genome. PLoS Genet. 2020;16(7):e1008922. doi: 10.1371/journal.pgen.1008922

</cite> [<a href="https://doi.org/10.1371/journal.pgen.1008922" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7384676/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32667917/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20Genet&amp;title=Unified%20inference%20of%20missense%20variant%20effects%20and%20gene%20constraints%20in%20the%20human%20genome&amp;author=Y-F%20Huang&amp;volume=16&amp;issue=7&amp;publication_year=2020&amp;pmid=32667917&amp;doi=10.1371/journal.pgen.1008922&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013400.ref046">
<span class="label">46.</span><cite>Malhis N, Jacobson M, Jones SJM, Gsponer J. LIST-S2: taxonomy based sorting of deleterious missense mutations across species. Nucleic Acids Res. 2020;48(W1):W154–61. doi: 10.1093/nar/gkaa288

</cite> [<a href="https://doi.org/10.1093/nar/gkaa288" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7319545/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32352516/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=LIST-S2:%20taxonomy%20based%20sorting%20of%20deleterious%20missense%20mutations%20across%20species&amp;author=N%20Malhis&amp;author=M%20Jacobson&amp;author=SJM%20Jones&amp;author=J%20Gsponer&amp;volume=48&amp;publication_year=2020&amp;pmid=32352516&amp;doi=10.1093/nar/gkaa288&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013400.ref047">
<span class="label">47.</span><cite>Samocha KE, Kosmicki JA, Karczewski KJ, O’Donnell-Luria AH, Pierce-Hoffman E, MacArthur DG, et al. Regional missense constraint improves variant deleteriousness prediction. bioRxiv. 2017:148353. doi: 10.1101/148353</cite> [<a href="https://doi.org/10.1101/148353" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=bioRxiv&amp;title=Regional%20missense%20constraint%20improves%20variant%20deleteriousness%20prediction&amp;author=KE%20Samocha&amp;author=JA%20Kosmicki&amp;author=KJ%20Karczewski&amp;author=AH%20O%E2%80%99Donnell-Luria&amp;author=E%20Pierce-Hoffman&amp;publication_year=2017&amp;pages=148353&amp;doi=10.1101/148353&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013400.ref048">
<span class="label">48.</span><cite>Jagota M, Ye C, Albors C, Rastogi R, Koehl A, Ioannidis N, et al. Cross-protein transfer learning substantially improves disease variant prediction. Genome Biol. 2023;24(1):182. doi: 10.1186/s13059-023-03024-6

</cite> [<a href="https://doi.org/10.1186/s13059-023-03024-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10408151/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37550700/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genome%20Biol&amp;title=Cross-protein%20transfer%20learning%20substantially%20improves%20disease%20variant%20prediction&amp;author=M%20Jagota&amp;author=C%20Ye&amp;author=C%20Albors&amp;author=R%20Rastogi&amp;author=A%20Koehl&amp;volume=24&amp;issue=1&amp;publication_year=2023&amp;pages=182&amp;pmid=37550700&amp;doi=10.1186/s13059-023-03024-6&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013400.ref049">
<span class="label">49.</span><cite>Molotkov I, Koboldt DC, Artomov M. SNPred outperforms other ensemble-based SNV pathogenicity predictors and elucidates the challenges of using ClinVar for evaluation of variant classification quality. medRxiv. 2023:2023.09.07.23295192. doi: 10.1101/2023.09.07.23295192</cite> [<a href="https://doi.org/10.1101/2023.09.07.23295192" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=medRxiv&amp;title=SNPred%20outperforms%20other%20ensemble-based%20SNV%20pathogenicity%20predictors%20and%20elucidates%20the%20challenges%20of%20using%20ClinVar%20for%20evaluation%20of%20variant%20classification%20quality&amp;author=I%20Molotkov&amp;author=DC%20Koboldt&amp;author=M%20Artomov&amp;publication_year=2023&amp;doi=10.1101/2023.09.07.23295192&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013400.ref050">
<span class="label">50.</span><cite>Jiang TT, Fang L, Wang K. Deciphering “the language of nature”: A transformer-based language model for deleterious mutations in proteins. Innovation (Camb). 2023;4(5):100487. doi: 10.1016/j.xinn.2023.100487

</cite> [<a href="https://doi.org/10.1016/j.xinn.2023.100487" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10448337/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37636282/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Innovation%20(Camb)&amp;title=Deciphering%20%E2%80%9Cthe%20language%20of%20nature%E2%80%9D:%20A%20transformer-based%20language%20model%20for%20deleterious%20mutations%20in%20proteins&amp;author=TT%20Jiang&amp;author=L%20Fang&amp;author=K%20Wang&amp;volume=4&amp;issue=5&amp;publication_year=2023&amp;pages=100487&amp;pmid=37636282&amp;doi=10.1016/j.xinn.2023.100487&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013400.ref051">
<span class="label">51.</span><cite>Brandes N, Goldman G, Wang CH, Ye CJ, Ntranos V. Genome-wide prediction of disease variant effects with a deep protein language model. Nat Genet. 2023;55(9):1512–22. doi: 10.1038/s41588-023-01465-0

</cite> [<a href="https://doi.org/10.1038/s41588-023-01465-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10484790/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37563329/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&amp;title=Genome-wide%20prediction%20of%20disease%20variant%20effects%20with%20a%20deep%20protein%20language%20model&amp;author=N%20Brandes&amp;author=G%20Goldman&amp;author=CH%20Wang&amp;author=CJ%20Ye&amp;author=V%20Ntranos&amp;volume=55&amp;issue=9&amp;publication_year=2023&amp;pages=1512-22&amp;pmid=37563329&amp;doi=10.1038/s41588-023-01465-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013400.ref052">
<span class="label">52.</span><cite>Meier J, Rao R, Verkuil R, Liu J, Sercu T, Rives A. Language models enable zero-shot prediction of the effects of mutations on protein function. Advances in Neural Information Processing Systems. Curran Associates, Inc.; 2021. p. 29287–303. Available from: <a href="https://proceedings.neurips.cc/paper/2021/hash/f51338d736f95dd42427296047067694-Abstract.html" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://proceedings.neurips.cc/paper/2021/hash/f51338d736f95dd42427296047067694-Abstract.html</a></cite> [<a href="https://scholar.google.com/scholar_lookup?Meier%20J,%20Rao%20R,%20Verkuil%20R,%20Liu%20J,%20Sercu%20T,%20Rives%20A.%20Language%20models%20enable%20zero-shot%20prediction%20of%20the%20effects%20of%20mutations%20on%20protein%20function.%20Advances%20in%20Neural%20Information%20Processing%20Systems.%20Curran%20Associates,%20Inc.;%202021.%20p.%2029287%E2%80%93303.%20Available%20from:%20https://proceedings.neurips.cc/paper/2021/hash/f51338d736f95dd42427296047067694-Abstract.html" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013400.ref053">
<span class="label">53.</span><cite>Laine E, Karami Y, Carbone A. GEMME: A Simple and Fast Global Epistatic Model Predicting Mutational Effects. Mol Biol Evol. 2019;36(11):2604–19. doi: 10.1093/molbev/msz179

</cite> [<a href="https://doi.org/10.1093/molbev/msz179" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6805226/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31406981/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol%20Biol%20Evol&amp;title=GEMME:%20A%20Simple%20and%20Fast%20Global%20Epistatic%20Model%20Predicting%20Mutational%20Effects&amp;author=E%20Laine&amp;author=Y%20Karami&amp;author=A%20Carbone&amp;volume=36&amp;issue=11&amp;publication_year=2019&amp;pages=2604-19&amp;pmid=31406981&amp;doi=10.1093/molbev/msz179&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013400.ref054">
<span class="label">54.</span><cite>Sharma A, Gitter A. Exploring zero-shot structure-based protein fitness prediction. In: arXiv.org [Internet]. 23 Apr 2025.  [cited 4 Aug 2025]. Available from: <a href="https://arxiv.org/abs/2504.16886v1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://arxiv.org/abs/2504.16886v1</a></cite>
</li>
<li id="pcbi.1013400.ref055">
<span class="label">55.</span><cite>Pejaver V, Byrne AB, Feng B-J, Pagel KA, Mooney SD, Karchin R, et al. Calibration of computational tools for missense variant pathogenicity classification and ClinGen recommendations for PP3/BP4 criteria. Am J Hum Genet. 2022;109(12):2163–77. doi: 10.1016/j.ajhg.2022.10.013

</cite> [<a href="https://doi.org/10.1016/j.ajhg.2022.10.013" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9748256/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36413997/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&amp;title=Calibration%20of%20computational%20tools%20for%20missense%20variant%20pathogenicity%20classification%20and%20ClinGen%20recommendations%20for%20PP3/BP4%20criteria&amp;author=V%20Pejaver&amp;author=AB%20Byrne&amp;author=B-J%20Feng&amp;author=KA%20Pagel&amp;author=SD%20Mooney&amp;volume=109&amp;issue=12&amp;publication_year=2022&amp;pages=2163-77&amp;pmid=36413997&amp;doi=10.1016/j.ajhg.2022.10.013&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013400.ref056">
<span class="label">56.</span><cite>Luppino F, Lenz S, Chow CFW, Toth-Petroczy A. Deep learning tools predict variants in disordered regions with lower sensitivity. BMC Genomics. 2025;26(1):367. doi: 10.1186/s12864-025-11534-9

</cite> [<a href="https://doi.org/10.1186/s12864-025-11534-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11992697/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40221640/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Genomics&amp;title=Deep%20learning%20tools%20predict%20variants%20in%20disordered%20regions%20with%20lower%20sensitivity&amp;author=F%20Luppino&amp;author=S%20Lenz&amp;author=CFW%20Chow&amp;author=A%20Toth-Petroczy&amp;volume=26&amp;issue=1&amp;publication_year=2025&amp;pages=367&amp;pmid=40221640&amp;doi=10.1186/s12864-025-11534-9&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013400.ref057">
<span class="label">57.</span><cite>Cagiada M, Jonsson N, Lindorff-Larsen K. Decoding molecular mechanisms for loss of function variants in the human proteome. bioRxiv. 2024:2024.05.21.595203. doi: 10.1101/2024.05.21.595203</cite> [<a href="https://doi.org/10.1101/2024.05.21.595203" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=bioRxiv&amp;title=Decoding%20molecular%20mechanisms%20for%20loss%20of%20function%20variants%20in%20the%20human%20proteome&amp;author=M%20Cagiada&amp;author=N%20Jonsson&amp;author=K%20Lindorff-Larsen&amp;publication_year=2024&amp;doi=10.1101/2024.05.21.595203&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013400.ref058">
<span class="label">58.</span><cite>Richards S, Aziz N, Bale S, Bick D, Das S, Gastier-Foster J, et al. Standards and guidelines for the interpretation of sequence variants: a joint consensus recommendation of the American College of Medical Genetics and Genomics and the Association for Molecular Pathology. Genet Med. 2015;17(5):405–24. doi: 10.1038/gim.2015.30

</cite> [<a href="https://doi.org/10.1038/gim.2015.30" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4544753/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25741868/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genet%20Med&amp;title=Standards%20and%20guidelines%20for%20the%20interpretation%20of%20sequence%20variants:%20a%20joint%20consensus%20recommendation%20of%20the%20American%20College%20of%20Medical%20Genetics%20and%20Genomics%20and%20the%20Association%20for%20Molecular%20Pathology&amp;author=S%20Richards&amp;author=N%20Aziz&amp;author=S%20Bale&amp;author=D%20Bick&amp;author=S%20Das&amp;volume=17&amp;issue=5&amp;publication_year=2015&amp;pages=405-24&amp;pmid=25741868&amp;doi=10.1038/gim.2015.30&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013400.ref059">
<span class="label">59.</span><cite>Wu Y, Li R, Sun S, Weile J, Roth FP. Improved pathogenicity prediction for rare human missense variants. Am J Hum Genet. 2021;108(10):1891–906. doi: 10.1016/j.ajhg.2021.08.012

</cite> [<a href="https://doi.org/10.1016/j.ajhg.2021.08.012" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8546039/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34551312/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&amp;title=Improved%20pathogenicity%20prediction%20for%20rare%20human%20missense%20variants&amp;author=Y%20Wu&amp;author=R%20Li&amp;author=S%20Sun&amp;author=J%20Weile&amp;author=FP%20Roth&amp;volume=108&amp;issue=10&amp;publication_year=2021&amp;pages=1891-906&amp;pmid=34551312&amp;doi=10.1016/j.ajhg.2021.08.012&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013400.ref060">
<span class="label">60.</span><cite>Paysan-Lafosse T, Andreeva A, Blum M, Chuguransky SR, Grego T, Pinto BL, et al. The Pfam protein families database: embracing AI/ML. Nucleic Acids Res. 2025;53(D1):D523–34. doi: 10.1093/nar/gkae997

</cite> [<a href="https://doi.org/10.1093/nar/gkae997" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11701544/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39540428/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=The%20Pfam%20protein%20families%20database:%20embracing%20AI/ML&amp;author=T%20Paysan-Lafosse&amp;author=A%20Andreeva&amp;author=M%20Blum&amp;author=SR%20Chuguransky&amp;author=T%20Grego&amp;volume=53&amp;publication_year=2025&amp;pmid=39540428&amp;doi=10.1093/nar/gkae997&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013400.ref061">
<span class="label">61.</span><cite>Smithers B, Oates ME, Tompa P, Gough J. Three reasons protein disorder analysis makes more sense in the light of collagen. Protein Sci. 2016;25(5):1030–6. doi: 10.1002/pro.2913

</cite> [<a href="https://doi.org/10.1002/pro.2913" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4838654/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26941008/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Protein%20Sci&amp;title=Three%20reasons%20protein%20disorder%20analysis%20makes%20more%20sense%20in%20the%20light%20of%20collagen&amp;author=B%20Smithers&amp;author=ME%20Oates&amp;author=P%20Tompa&amp;author=J%20Gough&amp;volume=25&amp;issue=5&amp;publication_year=2016&amp;pages=1030-6&amp;pmid=26941008&amp;doi=10.1002/pro.2913&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013400.ref062">
<span class="label">62.</span><cite>Youden WJ. Index for rating diagnostic tests. Cancer. 1950;3(1):32–5. doi: 
</cite> [<a href="https://doi.org/10.1002/1097-0142(1950)3:1&amp;#x0003c;32::aid-cncr2820030106&amp;#x0003e;3.0.co;2-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15405679/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cancer&amp;title=Index%20for%20rating%20diagnostic%20tests&amp;author=WJ%20Youden&amp;volume=3&amp;issue=1&amp;publication_year=1950&amp;pages=32-5&amp;pmid=15405679&amp;doi=10.1002/1097-0142(1950)3:1&amp;#x0003c;32::aid-cncr2820030106&amp;#x0003e;3.0.co;2-3&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="caption p">
<span>S1 Text. Supplemental figures.</span><p>File containing additional figures complementary to the analysis shown in the main text of this manuscript.</p>
<p>(DOCX)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12377588/bin/pcbi.1013400.s001.docx" data-ga-action="click_feat_suppl" class="usa-link">pcbi.1013400.s001.docx</a><sup> (375.2KB, docx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="caption p">
<span>S1 Table. Global and region-specific optimal thresholds across all VEPs.</span><p>(DOCX)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12377588/bin/pcbi.1013400.s002.docx" data-ga-action="click_feat_suppl" class="usa-link">pcbi.1013400.s002.docx</a><sup> (260.1KB, docx) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>Associated datasets are available at <a href="https://doi.org/10.6084/m9.figshare.c.7747895.v1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://doi.org/10.6084/m9.figshare.c.7747895.v1</a> and the pipeline code is shared at <a href="https://github.com/drsamibioinfo/VEPS_IN_DISORDER/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/drsamibioinfo/VEPS_IN_DISORDER/</a>.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from PLOS Computational Biology are provided here courtesy of <strong>PLOS</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1371/journal.pcbi.1013400"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/pcbi.1013400.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (1.3 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12377588/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12377588/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12377588%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12377588/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12377588/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12377588/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40828855/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12377588/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40828855/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12377588/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12377588/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="IDldvZtR8pPK8FJifkxFImD2VmbHAyDejINEFpvaDG44BbXvDblUrbxkBsx1cqjw">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
