
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Non-canonical Gq activation by orexin receptor type 2 and lemborexant observed in microsecond molecular dynamics simulations - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="AE532E758AF21EA3052E75000287A2BC.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="scirep">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373736/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Scientific Reports">
<meta name="citation_title" content="Non-canonical Gq activation by orexin receptor type 2 and lemborexant observed in microsecond molecular dynamics simulations">
<meta name="citation_author" content="Paulina Dragan">
<meta name="citation_author_institution" content="Faculty of Chemistry, University of Warsaw, 02-093 Warsaw, Poland">
<meta name="citation_author" content="Valerie Gratio">
<meta name="citation_author_institution" content="INSERM UMR1149/Inflammation Research Center (CRI), Université Paris Cité, Team “From Inflammation to Cancer in Digestive Diseases (INDiD)”, DHU UNITY, 75018 Paris, France">
<meta name="citation_author_institution" content="INSERM UMR1149/Inflammation Research Center (CRI), Université Paris Cité, Flow Cytometry Platform (CytoCRI), DHU UNITY, 75018 Paris, France">
<meta name="citation_author" content="Thierry Voisin">
<meta name="citation_author_institution" content="INSERM UMR1149/Inflammation Research Center (CRI), Université Paris Cité, Team “From Inflammation to Cancer in Digestive Diseases (INDiD)”, DHU UNITY, 75018 Paris, France">
<meta name="citation_author" content="Alain Couvineau">
<meta name="citation_author_institution" content="INSERM UMR1149/Inflammation Research Center (CRI), Université Paris Cité, Team “From Inflammation to Cancer in Digestive Diseases (INDiD)”, DHU UNITY, 75018 Paris, France">
<meta name="citation_author" content="Dorota Latek">
<meta name="citation_author_institution" content="Faculty of Chemistry, University of Warsaw, 02-093 Warsaw, Poland">
<meta name="citation_publication_date" content="2025 Aug 22">
<meta name="citation_volume" content="15">
<meta name="citation_firstpage" content="30899">
<meta name="citation_doi" content="10.1038/s41598-025-03857-0">
<meta name="citation_pmid" content="40846858">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373736/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373736/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373736/pdf/41598_2025_Article_3857.pdf">
<meta name="description" content="Orexins are hypothalamic neuropeptides primarily involved in regulating the sleep/wakefulness cycle and circadian rhythm. They bind to the orexin receptor type 1 (OX1) and type 2 (OX2), well-known drug targets in the treatment of sleep ...">
<meta name="og:title" content="Non-canonical Gq activation by orexin receptor type 2 and lemborexant observed in microsecond molecular dynamics simulations">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Orexins are hypothalamic neuropeptides primarily involved in regulating the sleep/wakefulness cycle and circadian rhythm. They bind to the orexin receptor type 1 (OX1) and type 2 (OX2), well-known drug targets in the treatment of sleep ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373736/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12373736">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1038/s41598-025-03857-0"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/41598_2025_Article_3857.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12373736%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12373736/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12373736/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373736/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-scirep.jpg" alt="Scientific Reports logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Scientific Reports" title="Link to Scientific Reports" shape="default" href="http://www.nature.com/srep" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Sci Rep</button></div>. 2025 Aug 22;15:30899. doi: <a href="https://doi.org/10.1038/s41598-025-03857-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1038/s41598-025-03857-0</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Sci%20Rep%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sci%20Rep%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Sci%20Rep%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Sci%20Rep%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Non-canonical G<sub>q</sub> activation by orexin receptor type 2 and lemborexant observed in microsecond molecular dynamics simulations</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Dragan%20P%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Paulina Dragan</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Paulina Dragan</span></h3>
<div class="p">
<sup>1</sup>Faculty of Chemistry, University of Warsaw, 02-093 Warsaw, Poland </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Dragan%20P%22%5BAuthor%5D" class="usa-link"><span class="name western">Paulina Dragan</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gratio%20V%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Valerie Gratio</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Valerie Gratio</span></h3>
<div class="p">
<sup>2</sup>INSERM UMR1149/Inflammation Research Center (CRI), Université Paris Cité, Team “From Inflammation to Cancer in Digestive Diseases (INDiD)”, DHU UNITY, 75018 Paris, France </div>
<div class="p">
<sup>3</sup>INSERM UMR1149/Inflammation Research Center (CRI), Université Paris Cité, Flow Cytometry Platform (CytoCRI), DHU UNITY, 75018 Paris, France </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gratio%20V%22%5BAuthor%5D" class="usa-link"><span class="name western">Valerie Gratio</span></a>
</div>
</div>
<sup>2,</sup><sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Voisin%20T%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Thierry Voisin</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Thierry Voisin</span></h3>
<div class="p">
<sup>2</sup>INSERM UMR1149/Inflammation Research Center (CRI), Université Paris Cité, Team “From Inflammation to Cancer in Digestive Diseases (INDiD)”, DHU UNITY, 75018 Paris, France </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Voisin%20T%22%5BAuthor%5D" class="usa-link"><span class="name western">Thierry Voisin</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Couvineau%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Alain Couvineau</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Alain Couvineau</span></h3>
<div class="p">
<sup>2</sup>INSERM UMR1149/Inflammation Research Center (CRI), Université Paris Cité, Team “From Inflammation to Cancer in Digestive Diseases (INDiD)”, DHU UNITY, 75018 Paris, France </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Couvineau%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Alain Couvineau</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Latek%20D%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Dorota Latek</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Dorota Latek</span></h3>
<div class="p">
<sup>1</sup>Faculty of Chemistry, University of Warsaw, 02-093 Warsaw, Poland </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Latek%20D%22%5BAuthor%5D" class="usa-link"><span class="name western">Dorota Latek</span></a>
</div>
</div>
<sup>1,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>Faculty of Chemistry, University of Warsaw, 02-093 Warsaw, Poland </div>
<div id="Aff2">
<sup>2</sup>INSERM UMR1149/Inflammation Research Center (CRI), Université Paris Cité, Team “From Inflammation to Cancer in Digestive Diseases (INDiD)”, DHU UNITY, 75018 Paris, France </div>
<div id="Aff3">
<sup>3</sup>INSERM UMR1149/Inflammation Research Center (CRI), Université Paris Cité, Flow Cytometry Platform (CytoCRI), DHU UNITY, 75018 Paris, France </div>
<div class="author-notes p"><div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div></div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 Jan 10; Accepted 2025 May 20; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by-nc-nd/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12373736  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40846858/" class="usa-link">40846858</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<p id="Par1">Orexins are hypothalamic neuropeptides primarily involved in regulating the sleep/wakefulness cycle and circadian rhythm. They bind to the orexin receptor type 1 (OX<sub>1</sub>) and type 2 (OX<sub>2</sub>), well-known drug targets in the treatment of sleep disorders, that have recently been shown to play a significant role in different cancers. Lemborexant is one of a few orexin receptor antagonists that have been approved for the treatment of insomnia. Despite being classified as an antagonist, lemborexant may display agonist-like behavior in the non-canonical signaling pathway of the orexin receptors, as confirmed recently in cancer cell models. Here, we generated a model of OX<sub>2</sub> in complex with the full-length G<sub>q</sub> protein and used it in the molecular dynamics (MD) study. We compared the impact of lemborexant and the OX<sub>2</sub>-selective, potent agonist compound 1 on OX<sub>2</sub> activation and subsequent guanosine diphosphate (GDP) to guanosine triphosphate (GTP) exchange in the Gα<sub>q</sub> subunit. These 2 µs MD simulations showed that both ligands evoke similar, activation-like conformational changes in OX<sub>2</sub> and explained the observed lemborexant-mediated apoptosis of cancer cells. In addition, MD simulations of the active-state OX<sub>2</sub>-G<sub>q</sub> complexes allowed us to uncover a sequence of micro- and macroscale events during the activation of G<sub>q</sub> and to detect important micro- and macroswitches in the Gα subunit.</p>
<section id="kwd-group1" lang="en" class="kwd-group"><p><strong>Keywords:</strong> Orexin receptors, OX<sub>2</sub>, G<sub>q</sub> protein activation, GDP/GTP exchange, Signal transduction, Lemborexant, Molecular dynamics</p></section><section id="kwd-group2" class="kwd-group"><p><strong>Subject terms:</strong> Cellular signalling networks, Drug development, Computational chemistry</p></section></section><section id="Sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p id="Par13">Orexins<sup><a href="#CR1" class="usa-link" aria-describedby="CR1">1</a></sup> were discovered in 1998 by two independent research groups, de Lecea et al.<sup><a href="#CR2" class="usa-link" aria-describedby="CR2">2</a></sup> and Sakurai et al.<sup><a href="#CR3" class="usa-link" aria-describedby="CR3">3</a></sup> These hypothalamic neuropeptides exist in two forms: orexin-A (OxA, 33 amino acids) and orexin-B (OxB, 28 amino acids)<sup><a href="#CR4" class="usa-link" aria-describedby="CR4">4</a></sup>, and are involved in regulating the central nervous system<sup><a href="#CR5" class="usa-link" aria-describedby="CR5">5</a></sup>. Both are formed from the same precursor via proteolytic processing<sup><a href="#CR3" class="usa-link" aria-describedby="CR3">3</a></sup> but share only 46% of the same sequence<sup><a href="#CR4" class="usa-link" aria-describedby="CR4">4</a></sup>. Best known for regulating the sleep/wakefulness cycle and circadian rhythm, orexin deficiencies can cause sleep disorders<sup><a href="#CR6" class="usa-link" aria-describedby="CR6">6</a></sup>. They also influence blood pressure, locomotion, hormone secretion<sup><a href="#CR7" class="usa-link" aria-describedby="CR7">7</a></sup>, drug addiction, reward seeking, energy homeostasis<sup><a href="#CR5" class="usa-link" aria-describedby="CR5">5</a></sup>, and peripheral organ function<sup><a href="#CR8" class="usa-link" aria-describedby="CR8">8</a></sup>.</p>
<p id="Par14">Orexins bind to two G protein-coupled receptors (GPCRs), OX<sub>1</sub> and OX<sub>2</sub><sup><a href="#CR9" class="usa-link" aria-describedby="CR9">9</a></sup>, which recognize both OxA and OxB with poor selectivity<sup><a href="#CR10" class="usa-link" aria-describedby="CR10">10</a></sup>. However, other known ligands demonstrate varied subtype selectivity, e.g., seltorexant (OX<sub>2</sub>)<sup><a href="#CR11" class="usa-link" aria-describedby="CR11">11</a></sup>, ACT-335827 (OX<sub>1</sub>)<sup><a href="#CR12" class="usa-link" aria-describedby="CR12">12</a></sup>, with some binding to both receptors<sup><a href="#CR13" class="usa-link" aria-describedby="CR13">13</a></sup>. Agonist binding activates a canonical signal cascade involving the G<sub>q</sub> protein, triggering intracellular Ca<sup>2+</sup> release. While orexin receptors additionally couple with Gi/o and Gs<sup><a href="#CR14" class="usa-link" aria-describedby="CR14">14</a></sup>, our recent data indicated that OX<sub>1</sub> is coupled exclusively to G<sub>q</sub> in recombinant human embryonic kidney-OX<sub>1</sub> (HEK-OX<sub>1</sub>) cells<sup><a href="#CR15" class="usa-link" aria-describedby="CR15">15</a></sup>. A second non-canonical signaling pathway of orexin receptors leads to apoptosis. Upon activation by orexins, two immunoreceptor tyrosine-based inhibitory motifs (ITIMs) of OX<sub>1</sub> or OX<sub>2</sub> are phosphorylated, leading to the recruitment of Src homology region 2 domain-containing phosphatase-2 SHP2. This triggers a signaling cascade involving the activation of the p38 mitogen-activated protein (MAP) kinase, the translocation of the apoptosis regulator Bax in mitochondria, cytochrome C release, and the activation of caspases-3 and -7, ultimately causing cell death<sup><a href="#CR5" class="usa-link" aria-describedby="CR5">5</a></sup>.</p>
<p id="Par15">Orexin receptors are promising drug targets for sleep disorders like narcolepsy or insomnia<sup><a href="#CR16" class="usa-link" aria-describedby="CR16">16</a></sup>, with small-molecule antagonists such as suvorexant, lemborexant, or daridorexant already on the market. While orexins as agonists promote wakefulness, antagonists treat insomnia<sup><a href="#CR17" class="usa-link" aria-describedby="CR17">17</a>,<a href="#CR18" class="usa-link" aria-describedby="CR18">18</a></sup>. Additionally, the orexin system plays a role in different cancers, e.g., OX<sub>1</sub> in glioblastoma, colon, pancreas, and prostate cancers, and OX<sub>2</sub> in rat pancreas cancer, endometrial carcinomas, or cortical adenomas<sup><a href="#CR19" class="usa-link" aria-describedby="CR19">19</a>–<a href="#CR21" class="usa-link" aria-describedby="CR21">21</a></sup>. Both orexins can induce apoptosis, thereby displaying antitumoral properties<sup><a href="#CR20" class="usa-link" aria-describedby="CR20">20</a>,<a href="#CR22" class="usa-link" aria-describedby="CR22">22</a></sup>; similar results have been reported for almorexant<sup><a href="#CR23" class="usa-link" aria-describedby="CR23">23</a></sup>. Recent studies have shown that suvorexant, lemborexant, and daridorexant induce apoptosis in human colon cancer cells expressing OX<sub>1</sub>, noticeably inhibiting OxA-induced intracellular calcium release<sup><a href="#CR15" class="usa-link" aria-describedby="CR15">15</a></sup>. They also prompted apoptosis in recombinant HEK cells expressing OX<sub>2</sub><sup><a href="#CR15" class="usa-link" aria-describedby="CR15">15</a></sup>. Additionally, lemborexant induced a partial dissociation of G<sub>q</sub>, leading to its quasi-agonistic properties on apoptosis induction in cancer cells<sup><a href="#CR15" class="usa-link" aria-describedby="CR15">15</a></sup>.</p>
<p id="Par16">As the structure of the active-state OX<sub>1</sub> has not yet been solved, we used OX<sub>2</sub> in complex with G<sub>q</sub> and either a potent, OX<sub>2</sub>-selective, small molecule agonist (compound 1)<sup><a href="#CR24" class="usa-link" aria-describedby="CR24">24</a></sup> or an antagonist (lemborexant)<sup><a href="#CR25" class="usa-link" aria-describedby="CR25">25</a></sup> to study the activation of the orexin system by graphics processing unit (GPU)-accelerated all-atom MD simulations. Notably, the activation of OX<sub>2</sub> by orexins induced pro-apoptotic effects in cancer cells similar to the one induced by OX<sub>1</sub><sup><a href="#CR15" class="usa-link" aria-describedby="CR15">15</a>,<a href="#CR21" class="usa-link" aria-describedby="CR21">21</a></sup>. MD is commonly used to observe the evolution of molecular systems in time<sup><a href="#CR26" class="usa-link" aria-describedby="CR26">26</a></sup>, and can be used to study both micro- and macroscale cellular processes<sup><a href="#CR27" class="usa-link" aria-describedby="CR27">27</a></sup>, also in the context of GPCRs<sup><a href="#CR28" class="usa-link" aria-describedby="CR28">28</a>–<a href="#CR32" class="usa-link" aria-describedby="CR32">32</a></sup>. In many such cases MD is complemented with other computational methods, especially those involving the use of artificial intelligence<sup><a href="#CR33" class="usa-link" aria-describedby="CR33">33</a></sup> that have found applications in cancer diagnosis, treatment, and drug discovery, thereby accelerating the development of new treatment strategies<sup><a href="#CR34" class="usa-link" aria-describedby="CR34">34</a></sup>.</p>
<p id="Par17">The orexin system has already been included in several in-silico studies. In 2016, Yin et al. performed molecular dynamics simulations to elucidate the basis for receptor subtype selectivity in regard to small-molecule agonists<sup><a href="#CR35" class="usa-link" aria-describedby="CR35">35</a></sup>. They determined which residues affected the binding affinity of the antagonist SB-674042 and confirmed it with radioligand binding experiments<sup><a href="#CR35" class="usa-link" aria-describedby="CR35">35</a></sup>. Computational approaches such as quantitative structure–activity relationship (QSAR) models; molecular docking; and absorption, distribution, metabolism, and excretion (ADME) predictions have also been used in the design of novel OX<sub>1</sub> antagonists<sup><a href="#CR28" class="usa-link" aria-describedby="CR28">28</a></sup>, while molecular mechanics have been used to compute the binding free energies of dual orexin receptor antagonists such as lemborexant and suvorexant using previously obtained crystal structures<sup><a href="#CR36" class="usa-link" aria-describedby="CR36">36</a></sup>. Additionally, homology modeling and MD simulations were used to predict the binding mode of OxA to OX<sub>2</sub> before the corresponding Protein Data Bank (PDB) structure was released<sup><a href="#CR37" class="usa-link" aria-describedby="CR37">37</a></sup>.</p>
<p id="Par18">In our MD study, accompanied by the experimental study of Gratio et al.<sup><a href="#CR15" class="usa-link" aria-describedby="CR15">15</a></sup>, we observed similarities between the classic OX<sub>2</sub> agonist compound 1 and lemborexant-including systems, suggesting the agonist-like behaviour of lemborexant in the recombinant HEK293T cell line expressing OX<sub>2</sub><sup><a href="#CR15" class="usa-link" aria-describedby="CR15">15</a></sup> despite its well-known role as a dual OX<sub>1</sub>/OX<sub>2</sub> antagonist<sup><a href="#CR36" class="usa-link" aria-describedby="CR36">36</a></sup>. In principle, an antagonist tightly binds to the orthosteric binding site of the GPCR receptor, preventing an endogenous agonist binding and the subsequent receptor activation. Since the lemborexant orthosteric binding site in OX<sub>1</sub>/OX<sub>2</sub> is known and well-described<sup><a href="#CR36" class="usa-link" aria-describedby="CR36">36</a></sup>, and no other binding sites, e.g., allosteric, were detected<sup><a href="#CR15" class="usa-link" aria-describedby="CR15">15</a></sup>, its surprising pro-apoptotic effect induced by the receptor activation could not be explained other than as evoking activation as an agonist. Additionally, the preceding experimental study<sup><a href="#CR15" class="usa-link" aria-describedby="CR15">15</a></sup> showed that the orexin receptor was activated by lemborexant in the absence of orexin, which excludes the possibility of an allosteric mode of action of lemborexant in tested cell lines. This is further supported by the fact that the pre-treatment of HEK cells expressing OX<sub>1</sub> with antagonists such as almorexant, lemborexant, suvorexant, and daridorexant completely abolished the induction of calcium release by OxA<sup><a href="#CR15" class="usa-link" aria-describedby="CR15">15</a></sup>. The ligand-dependent preferential activation of selected pathways in different tissues (biased agonism or functional selectivity) was previously observed for many GPCR receptors, e.g., for opioid receptors and β-adrenergic receptors<sup><a href="#CR38" class="usa-link" aria-describedby="CR38">38</a></sup>.</p>
<p id="Par19">Here, we explained the agonist-like behaviour of lemborexant on the molecular level using MD. The simulation systems included OX<sub>2</sub> bound to either lemborexant or the OX<sub>2</sub>-selective, classic agonist compound 1, and the full-length Gα<sub>q</sub>, which substituted the mini-Gsqi construct present in the template (PDB: 7L1V)<sup><a href="#CR15" class="usa-link" aria-describedby="CR15">15</a>,<a href="#CR24" class="usa-link" aria-describedby="CR24">24</a>,<a href="#CR39" class="usa-link" aria-describedby="CR39">39</a></sup> and used in our previous study. Performed MD simulations additionally provided insights into the molecular basis of the OX<sub>2</sub> activation, allowing us to propose a sequence of conformational changes that the receptor and Gα<sub>q</sub> undergo during activation, e.g., the opening of the Gα<sub>q</sub> helical domain and subsequent release of GDP. Due to a high sequence identity (67% according to Clustal2.1) between two orexin receptors, our results regarding receptor activation could be extrapolated to OX<sub>1</sub>, for which even more detailed experimental evidence has recently been provided<sup><a href="#CR15" class="usa-link" aria-describedby="CR15">15</a></sup>.</p></section><section id="Sec2"><h2 class="pmc_sec_title">Methods</h2>
<section id="Sec3"><h3 class="pmc_sec_title">Preparation of simulation systems</h3>
<p id="Par20">Models of the OX<sub>2</sub>–G<sub>q</sub> systems were prepared using Modeller v. 10.4<sup><a href="#CR40" class="usa-link" aria-describedby="CR40">40</a></sup>, with the 7L1V<sup><a href="#CR24" class="usa-link" aria-describedby="CR24">24</a></sup> PDB entry used as a template for the receptor and G protein complex, and 7SQ2<sup><a href="#CR41" class="usa-link" aria-describedby="CR41">41</a></sup> used as the template for the Gα subunit. The positions of conserved residues in the orexin receptor with respect to other GPCRs were confirmed (see Supplementary Fig. S1). Two separate sets of models were generated—one containing compound 1 in the orthosteric binding site, and another containing lemborexant. Compound 1 was present in the active-state 7L1V structure of OX<sub>2</sub>. Lemborexant was positioned in the orthosteric binding site by superposing the inactive-state 7XRR PDB structure<sup><a href="#CR36" class="usa-link" aria-describedby="CR36">36</a></sup> onto the apo 7L1V structure and transferring the lemborexant molecule from the inactive-state structure to the active-state structure in order to retain its binding mode observed in 7XRR. Both simulation systems contained GDP bound to the Gα subunit. The crystal water molecules in the 7SQ2 structure were retained during the model building by Modeller but removed before building the membrane with CHARMM-GUI. The target sequence consisted of the UNIPROT<sup><a href="#CR42" class="usa-link" aria-describedby="CR42">42</a></sup> entries for OX<sub>2</sub> (<a href="https://www.ncbi.nlm.nih.gov/protein/O43614" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">O43614</a>), Gα<sub>q</sub> (<a href="https://www.ncbi.nlm.nih.gov/protein/P50148" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">P50148</a>), Gβ<sub>1</sub> (<a href="https://www.ncbi.nlm.nih.gov/protein/P62873" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">P62873</a>), and Gγ<sub>2</sub> (<a href="https://www.ncbi.nlm.nih.gov/protein/P59768" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">P59768</a>). The UNIPROT entry for OX<sub>2</sub> was compared in terms of sequence identity (67%) with OX<sub>1</sub> (<a href="https://www.ncbi.nlm.nih.gov/protein/O43613" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">O43613</a>) using ClustalOmega (<a href="https://www.ebi.ac.uk/jdispatcher/msa/clustalo" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ebi.ac.uk/jdispatcher/msa/clustalo</a>). In order to make sure the N- and C-termini of the OX<sub>2</sub> models were comparable to the 7L1V structure, all of the residues before Y<sup>1.31</sup> and after F<sup>7.68</sup> were removed. Additionally, residues G<sup>5.72</sup> to V<sup>6.20</sup> of intracellular loop 3 (ICL3) were removed (33 residues) to simplify the model. Although ICL3 is involved in signaling through the G protein<sup><a href="#CR43" class="usa-link" aria-describedby="CR43">43</a></sup>, this region is highly flexible and predicted to be intrinsically disordered<sup><a href="#CR44" class="usa-link" aria-describedby="CR44">44</a>,<a href="#CR45" class="usa-link" aria-describedby="CR45">45</a></sup>, and is absent in most GPCR structures in PDB<sup><a href="#CR43" class="usa-link" aria-describedby="CR43">43</a></sup>. It is assumed to affect the receptor specificity for the G protein type<sup><a href="#CR43" class="usa-link" aria-describedby="CR43">43</a></sup>, and then cAMP accumulation after GDP release<sup><a href="#CR46" class="usa-link" aria-describedby="CR46">46</a></sup>, neither of which were considered in this study.</p>
<p id="Par21">For each set, 500 preliminary models were generated. Ten iterations of loop refinements were then performed for each model, resulting in a total of 5000 models per set. Their quality was evaluated by comparing DOPE scores<sup><a href="#CR47" class="usa-link" aria-describedby="CR47">47</a></sup>, and the 30 lowest-energy models were further assessed in PyMOL version 2.5<sup><a href="#CR48" class="usa-link" aria-describedby="CR48">48</a></sup>. Three lemborexant-OX<sub>2</sub>-G<sub>q</sub> and three compound 1-OX<sub>2</sub>-G<sub>q</sub> models were selected for the generation of MD simulation systems, resulting in a total number of six replicas that were simulated in parallel. The protein preparation wizard in Maestro (Schrödinger Suite release 2022-3)<sup><a href="#CR49" class="usa-link" aria-describedby="CR49">49</a></sup> was used to preprocess the models, i.e., add hydrogens and optimize bonds.</p></section><section id="Sec4"><h3 class="pmc_sec_title">Molecular dynamics simulations</h3>
<p id="Par22">Inputs for the molecular dynamics simulations were prepared using CHARMM-GUI’s<sup><a href="#CR50" class="usa-link" aria-describedby="CR50">50</a></sup> Membrane Builder<sup><a href="#CR51" class="usa-link" aria-describedby="CR51">51</a></sup> (specifically, the Bilayer Builder). Each model was exported from Maestro in the PDB file format and uploaded into CHARMM-GUI. Information about the locations of disulphide bonds in the proteins was obtained from the templates and all crystal water molecules were removed. The ligands were exported from Maestro in sdf format and uploaded into CHARMM-GUI, which then automatically generated CHARMM topology and parameter files using the CHARMM General Force Field (CGenFF). Using the replacement method, the complexes were placed in a lipid bilayer consisting of a 3:1 ratio of POPC (1-palmitoyl-2-oleoyl-<em>sn</em>-glycero-3-phosphocholine) to cholesterol. Such a membrane composition has been used in other MD studies performed for GPCRs<sup><a href="#CR52" class="usa-link" aria-describedby="CR52">52</a></sup>. The periodic rectangular water box (TIP3P) was fitted to the complex, and each system was neutralized by adding Na<sup>+</sup> and Cl<sup>-</sup> ions at a concentration of 0.15 M. The number of atoms in each system was around 240,000–250,000, and the initial size of the simulation box was ca. 122 Å × 122 Å × 179 Å.</p>
<p id="Par23">The MD simulations were performed using the GPU-accelerated version of NAMD (NAMD 3.0 Alpha)<sup><a href="#CR53" class="usa-link" aria-describedby="CR53">53</a></sup> with the Charmm36m force field. The equilibration step included 10,000 steps of the steepest descent minimization, then 25,000 steps of the conjugated gradients minimization. The equilibration simulation was performed in NVT (constant particle number, volume, and temperature) using Langevin dynamics (303.15 K). The time integration step in the equilibration and production runs was set to 2 fs. The production run in NPT (constant particle number, pressure, and temperature) was performed using the Langevin piston Nose–Hoover method (1 bar, 303.15 K) and lasted for 2 µs. Every tenth frame of the computed trajectories was combined using CatDCD<sup><a href="#CR54" class="usa-link" aria-describedby="CR54">54</a></sup>, then wrapped using PBCTools<sup><a href="#CR55" class="usa-link" aria-describedby="CR55">55</a></sup> and analyzed in VMD v.1.9.3<sup><a href="#CR56" class="usa-link" aria-describedby="CR56">56</a></sup>. Further analyses were performed using PyMOL and Maestro.</p></section></section><section id="Sec5"><h2 class="pmc_sec_title">Results</h2>
<section id="Sec6"><h3 class="pmc_sec_title">Similarities in binding modes between the classic OX<sub>2</sub> agonist compound 1 and lemborexant</h3>
<p id="Par24">MD simulations of both the classic agonist compound 1<sup><a href="#CR24" class="usa-link" aria-describedby="CR24">24</a></sup> and lemborexant-including complexes of the active-state OX<sub>2</sub> with the full-length Gα<sub>q</sub> were analyzed in terms of variance in the position of ligands and complex components (Fig. <a href="#Fig1" class="usa-link">1</a>, Supplementary Fig. S2). As a dual OX<sub>1</sub>/OX<sub>2</sub> antagonist<sup><a href="#CR57" class="usa-link" aria-describedby="CR57">57</a>,<a href="#CR58" class="usa-link" aria-describedby="CR58">58</a></sup>, lemborexant was expected to display significant RMSD fluctuations in the binding site of the active-state OX<sub>2</sub> structure, especially with additional intracellular stabilization by a G protein. However, its position remained stable, with insignificant RMSD fluctuations comparable to compound 1. This suggests there could be conditions under which lemborexant may induce an active-like conformation of OX<sub>2</sub> and behave similarly to the classic agonist. The maintained active state of OX<sub>2</sub> during simulations was validated by analyzing DRW motif residues (corresponding to the DRY ‘ionic lock’) (Supplementary Figures S3, S4). In class A GPCRs, R<sup>3.50</sup> interacts with T<sup>2.39</sup> (inactive state, e.g., 7XRR<sup>33</sup>) or Y<sup>5.58</sup> (active state, e.g., 7L1U, 7L1V<sup>24</sup>)<sup><a href="#CR59" class="usa-link" aria-describedby="CR59">59</a></sup>. The conformations of the DRW motif residues in the compound 1-including complexes remained ‘on’ in replicas 1 and 3 but shifted into an intermediate-like position in replica 2 (Supplementary Fig. S3). In the lemborexant-including complexes, active-state interactions were also formed, with fewer inactive-state interactions than in compound 1-including replicas (Supplementary Fig. S4). Additional microswitches were analyzed: PVF, CYLP, and NPIIY (corresponding to the PIF, CWxP, and NPxxY motifs, respectively; Supplementary Figure S5). In the majority of systems, V<sup>3.40</sup> in the PVF motif changed from an active-like conformation to an intermediate conformation resembling neither the active nor inactive state. The Y<sup>6.48</sup> toggle switch of the CYLP motif changed conformations in the final frame of all of the lemborexant replicas but only one compound 1 replica. Although the inactive-state 7XRR and active-state 7L1V structures include the same Y<sup>6.48</sup> conformation, the changes of this microswitch are reported to be temporary<sup><a href="#CR60" class="usa-link" aria-describedby="CR60">60</a></sup>. The Y<sup>7.53</sup> toggle switch of the NPIIY motif remained in an active state in all but one system, where it adopted an intermediate conformation. Additionally, transmembrane helix 6—recognized as a key macroswitch involved in GPCR activation—did not undergo global rearrangements in any of the simulation replicas. This was assessed by measuring the distance between the Cɑ atoms of R<sup>3.50</sup> and A<sup>6.34</sup> in the models, 7L1V (11.8 Å), and 7XRR (8.3 Å)<sup><a href="#CR61" class="usa-link" aria-describedby="CR61">61</a></sup>. In all models, these distances remained greater than 11.4 Å, thus comparable to 7L1V. These results further confirmed that OX<sub>2</sub> remained active throughout the simulations.</p>
<figure class="fig xbox font-sm" id="Fig1"><h4 class="obj_head">Fig. 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373736_41598_2025_3857_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e8af/12373736/5eb1a60aefcf/41598_2025_3857_Fig1_HTML.jpg" loading="lazy" id="MO1" height="193" width="669" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Similarities in binding modes between the classic OX<sub>2</sub> agonist compound 1 and lemborexant. A comparison of the binding modes of compound 1 (left, replica 1) and lemborexant (right, replica 1). The first frame of the simulation is shown in gray with a yellow ligand, and the final frame in green with a red ligand. The final frame is meant to represent the ligand conformation at the end of the simulations and includes the key receptor residues described in Supplementary Figures S4 and S5.</p></figcaption></figure><p id="Par25">A detailed analysis of compound 1 and lemborexant binding modes (Fig. <a href="#Fig1" class="usa-link">1</a>) aligns with previous studies<sup><a href="#CR24" class="usa-link" aria-describedby="CR24">24</a>,<a href="#CR36" class="usa-link" aria-describedby="CR36">36</a>,<a href="#CR57" class="usa-link" aria-describedby="CR57">57</a>,<a href="#CR58" class="usa-link" aria-describedby="CR58">58</a></sup>. In the 7L1V structure, the phenyl rings of compound 1 were surrounded by a lipophilic patch consisting of P<sup>3.29</sup>, M<sup>4.64</sup>, T<sup>2.61</sup>, and V<sup>2.64</sup><sup><a href="#CR24" class="usa-link" aria-describedby="CR24">24</a></sup>. Lemborexant was similarly surrounded by P<sup>3.29</sup>, T<sup>2.61</sup>, and V<sup>2.64</sup>, though Asada et al.<sup><a href="#CR36" class="usa-link" aria-describedby="CR36">36</a></sup> proposed interactions with T<sup>2.61</sup> to be rather irrelevant for lemborexant binding to the inactive-state receptor, suggesting that it demonstrated a slightly different binding mode for the active-state. H<sup>7.39</sup> interacted with compound 1 (Supplementary Figs. S6, S7), as indicated by Hong et al.<sup><a href="#CR24" class="usa-link" aria-describedby="CR24">24</a></sup> and also stabilized lemborexant in 7XRR<sup>33</sup> but through π–π stacking instead of a hydrogen bond. Both ligands closely interacted with F<sup>5.42</sup> through π–π stacking and formed hydrogen bonds with Q<sup>3.32</sup>. However, extracellular loop 2 (ECL2) closure, observed previously for OX<sub>2</sub>–mini-G<sub>sqi</sub> complexes<sup><a href="#CR15" class="usa-link" aria-describedby="CR15">15</a></sup>, caused both ligands to move further inside the receptor. Consequently, the Q<sup>3.32</sup> side chain also moved towards the inside of the receptor (see Fig. <a href="#Fig1" class="usa-link">1</a>) in contrast to what can be observed in 7L1V<sup><a href="#CR24" class="usa-link" aria-describedby="CR24">24</a></sup>. Hong et al. showed that Q<sup>3.32</sup> remained stably oriented outward during a 1 μs MD simulation with OxB<sup><a href="#CR24" class="usa-link" aria-describedby="CR24">24</a></sup>. In our 2 μs MD simulations with the small-molecule agonist compound 1, we observed the closing of ECL2 after 1 μs, shifting Q<sup>3.32</sup> towards the inside of the receptor after 2 μs. Perhaps the Q<sup>3.32</sup> conformation was influenced by ligand bulkiness (small-molecule vs. peptide) and correlated with the closing of ECL2, rather than strictly by ligand type (agonist vs. antagonist) as suggested by Hong et al.<sup><a href="#CR24" class="usa-link" aria-describedby="CR24">24</a></sup> Since ECL2 remained open for peptide agonists, Q<sup>3.32</sup> could be oriented outwards<sup><a href="#CR24" class="usa-link" aria-describedby="CR24">24</a></sup>.</p></section><section id="Sec7"><h3 class="pmc_sec_title">Activation-like changes of microswitches in Gα<sub>q</sub>
</h3>
<p id="Par26">Ham et al. described residue F341<sup>G.H5.08</sup> (labeled there as F336) in helix H5 of Gα to be a microswitch affecting the GDP binding energy via switch I and the P loop (Supplementary Fig. S8), as shown in mutagenesis studies<sup><a href="#CR62" class="usa-link" aria-describedby="CR62">62</a></sup>. The F341<sup>G.H5.08</sup>A mutation or a change of F341<sup>G.H5.08</sup> to the ‘on’ position caused fast GDP/GTP exchange rates<sup><a href="#CR62" class="usa-link" aria-describedby="CR62">62</a></sup>. In our homology models, this microswitch was initially in the ‘off’ position, as expected for an inactive G protein (Fig. <a href="#Fig2" class="usa-link">2</a>A)<sup><a href="#CR62" class="usa-link" aria-describedby="CR62">62</a></sup>. During the simulations, its change to the ‘on’ position required a slight counterclockwise rotation. This was observed mostly for the lemborexant-including replicas (1 and 2) representing the furthest stage of Gα activation. In previous 2 µs simulations involving mini-G<sub>sqi</sub><sup><a href="#CR15" class="usa-link" aria-describedby="CR15">15</a></sup>, F341<sup>G.H5.08</sup> was also in the ‘on’ position at the end of the simulation. This suggested its transition may not be directly related to the opening of Gα, as these systems lacked the alpha helical domain (AHD) of Gα. Instead, this change may have been induced by receptor activation alone.</p>
<figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373736_41598_2025_3857_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e8af/12373736/2f9609138180/41598_2025_3857_Fig2_HTML.jpg" loading="lazy" id="MO2" height="601" width="669" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Changes of microswitches D277<sup>G.HG.02</sup> and F341<sup>G.H5.08</sup> and macroswitches I–III in OX<sub>2</sub> in comparison to the rhodopsin-mediated activation of Gα<sub>i</sub> (magenta). (A) The D277<sup>G.HG.02</sup> and F341<sup>G.H5.08</sup> microswitches described by Ham et al.<sup><a href="#CR62" class="usa-link" aria-describedby="CR62">62</a></sup> in the ‘off’ position in the first frame of the compound 1 replica 2 simulation (top, left) and 1GP2 PDB structure (bottom, left). The D277<sup>G.HG.02</sup> and F341<sup>G.H5.08</sup> microswitches described by Ham et al.<sup><a href="#CR62" class="usa-link" aria-describedby="CR62">62</a></sup> in the ‘on’ position in the last frame of the compound 1 replica 1 simulation (top, center), the last frame of the lemborexant replica 1 simulation (top, right), the 6CMO PDB structure (bottom, center), and the 6OS9 PDB structure (bottom, right). (B) A comparison between the positions of macroswitches I–III, the P loop, and HC and HD between the active state 6CMO PDB structure<sup><a href="#CR70" class="usa-link" aria-describedby="CR70">70</a></sup> (magenta), the first frame of compound 1 replica 1 (gray, yellow switches), and the final 2 μs frame of the compound 1 replica 1 (green, red switches) (Left). A comparison between the positions of macroswitches I–III, the P loop, and HC and HD in the active state 6CMO PDB structure<sup><a href="#CR70" class="usa-link" aria-describedby="CR70">70</a></sup> (magenta), the first frame of the lemborexant replica 1 (gray, yellow switches), and in the final 2 μs frame of the lemborexant replica 1 (green, red switches) (Right).</p></figcaption></figure><p id="Par27">Ham et al.<sup><a href="#CR62" class="usa-link" aria-describedby="CR62">62</a></sup> also described the D277<sup>G.HG.02</sup> microswitch (labeled there as D272) as important for GDP/GTP exchange. In our case, most replicas demonstrated a breaking of the GDP–D277<sup>G.HG.02</sup> interactions regardless of ligand type (Fig. <a href="#Fig2" class="usa-link">2</a>, Supplementary Figs. S9, S10). D277<sup>G.HG.02</sup> stabilized GDP binding by interacting with the guanine ring<sup><a href="#CR62" class="usa-link" aria-describedby="CR62">62</a></sup>. The breaking of this interaction was required for GDP dissociation, which changed the D277<sup>G.HG.02</sup> microswitch into the ‘on’ position—seen as the χ1 dihedral angle (N–C<sub>α</sub>–C<sub>β</sub>–C<sub>γ</sub>) change from ca. 60° to − 60° (Supplementary Figs. S7, S8). We observed the movement of the phosphoryl group, not the guanine ring, which imposed only slight changes on D277<sup>G.HG.02</sup>. Notably, GDP dissociated from Gα via its phosphate side<sup><a href="#CR63" class="usa-link" aria-describedby="CR63">63</a></sup>. D277<sup>G.HG.02</sup> was indeed in the ‘on’ position at the end of the lemborexant-including simulations. In the compound 1-including replicas, D277<sup>G.HG.02</sup> fluctuated, changing to the ‘on’ position only at around 1.70 μs for replica 3, while in the other two replicas, it switched to the ‘off’ position at around 1.00 μs.</p></section><section id="Sec8"><h3 class="pmc_sec_title">Activation-like changes of macroswitches in Gα<sub><em>q</em></sub> inducing the GDP/GTP exchange</h3>
<p id="Par28">The microsecond timescale used for MD simulations allow to observe first stages of the GPCR signal transduction involving both changes in the ligand–receptor complex and the interacting G protein or arrestin<sup><a href="#CR31" class="usa-link" aria-describedby="CR31">31</a></sup>. This required the full length Gα<sub>q</sub> to be present. To our knowledge, none of the active-state structures of GPCRs in PDB include the full-length Gα<sub>q</sub>, though the full-length Gα<sub>i</sub> is present in e.g., the rhodopsin PDB structure 6CMO (Fig. <a href="#Fig2" class="usa-link">2</a>). Only the P loop and switch II are present in their entirety in mini-G<sub>sqi</sub>, as observed when compared to the Gα topology described by Calebiro et al.<sup><a href="#CR64" class="usa-link" aria-describedby="CR64">64</a></sup> (Supplementary Fig. S8). For this reason, we constructed models with the full-length Gα<sub>q</sub> to obtain a fully functional OX<sub>2</sub>-G protein complex.</p>
<p id="Par29">Though switch II remained rather unchanged in the previous mini-G<sub>sqi</sub> simulations<sup><a href="#CR15" class="usa-link" aria-describedby="CR15">15</a></sup>, in most of the full-length Gα<sub>q</sub> simulations changes in that region were more visible, e.g., a subsequent movement of helix H2. This altered Gα–Gβγ interactions, as Gα macroswitches I and II, and the Gα N-terminal helix form a hydrophobic cleft fitting Gβγ<sup><a href="#CR64" class="usa-link" aria-describedby="CR64">64</a>–<a href="#CR66" class="usa-link" aria-describedby="CR66">66</a></sup>. These differences between results for the full-length Gα<sub>q</sub> and mini-G<sub>sqi</sub> simulations suggest that switch II movement might be connected specifically to the presence of the AHD in the former.</p>
<p id="Par30">In our MD simulations of OX<sub>2</sub> with a full-length Gα<sub>q</sub>, a noticeable difference was observed for compound 1 vs. lemborexant. At the end of the simulations, the most open Gα conformations were observed for the lemborexant-including simulations (Supplementary Figs. S11, S12). These opening movements of the Gα subunit, required for the GDP/GTP exchange and confirmed experimentally for other GPCR receptors<sup><a href="#CR67" class="usa-link" aria-describedby="CR67">67</a></sup>, were hardly observed for compound 1 (Fig. <a href="#Fig3" class="usa-link">3</a>). This may suggest that Gα opens faster in the lemborexant-including systems.</p>
<figure class="fig xbox font-sm" id="Fig3"><h4 class="obj_head">Fig. 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373736_41598_2025_3857_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e8af/12373736/8278a00e773c/41598_2025_3857_Fig3_HTML.jpg" loading="lazy" id="MO3" height="610" width="668" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Changes in Gα macroswitches inducing the opening of the GDP/GTP exchange tunnel upon compound 1 binding to OX<sub>2</sub>. The opening of a tunnel was observed near the phosphate side of GDP. From left to right: interactions between nearby residues, a surface view of the Gα subunit and GDP, hydrogen bonds determined by VMD to form between R183<sup>G.hfs2.20</sup> (switch I) and E49<sup>G.s1h1.04</sup> (P loop), and R183<sup>G.hfs2.20</sup> and E241<sup>G.s4h3.10</sup> (switch III) at a maximum 4 Å distance at an angle cutoff of 20º.</p></figcaption></figure><p id="Par31">The GDP position changed during the simulations regardless of the OX<sub>2</sub> ligand (Supplementary Fig. S13). Notably, the terminal phosphoryl group of GDP lost its interaction with R183<sup>G.hfs2.02</sup> (GproteinDb<sup><a href="#CR68" class="usa-link" aria-describedby="CR68">68</a></sup> numbering scheme) in switch I on behalf of the second phosphoryl group. The formation of hydrogen bonds between the oxygen atoms in the second phosphoryl group and R183<sup>G.hfs2.02</sup> was analyzed (Supplementary Fig. S14). In the compound 1-including complexes, hydrogen bonds between R183<sup>G.hfs2.02</sup> and GDP were more retained than in the case of lemborexant. However, the average RMSD changes for the switch I (ca. 4 Å) were comparable for both systems (Supplementary Fig. S15).</p>
<p id="Par32">The beginning of the opening of the ‘GDP/GTP exchange tunnel’ (Supplementary Files S3, S4) near the phosphate side of GDP was observed in both simulation systems, and the same way of GDP dissociation was described by Louet et al.<sup><a href="#CR63" class="usa-link" aria-describedby="CR63">63</a></sup> This was most visible for compound 1 replica 1 and lemborexant replica 1 (Figs. <a href="#Fig3" class="usa-link">3</a>, <a href="#Fig4" class="usa-link">4</a>). This opening of the exchange tunnel, along with the opening of Gα, suggested that these replicas represent the furthest stage of G protein activation regarding GDP release. In the remaining replicas, this tunnel was blocked by residues from the P loop and switch I. Namely, the formation of a salt bridge between R183<sup>G.hfs2.02</sup> (switch I) and E49<sup>G.s1h1.04</sup> (P loop) blocked the tunnel and prevented GDP from leaving the binding site (lemborexant replica 3, compound 1 replicas 2 and 3; Figs. <a href="#Fig3" class="usa-link">3</a>, <a href="#Fig4" class="usa-link">4</a>). The disruption of this salt bridge was necessary for Gα to open<sup><a href="#CR69" class="usa-link" aria-describedby="CR69">69</a></sup>. In our simulations, during Gα activation, R183<sup>G.hfs2.02</sup> in switch I moved to the outer side to interact with E241<sup>G.s4h3.10</sup> in switch III instead (lemborexant replica 2). Presumably, switch III was responsible for stabilizing R183<sup>G.hfs2.02</sup> at this stage, preventing it from interfering with GDP leaving the binding site. Interestingly, at the beginning of every simulation, E241<sup>G.s4h3.10</sup> in switch III interacted with two residues of switch II: R210<sup>G.H2.01</sup> and Q209<sup>G.s3h2.03</sup> (to a lesser extent). Thus, the release of E241<sup>G.s4h3.10</sup> from interactions with switch II might also be required<sup><a href="#CR69" class="usa-link" aria-describedby="CR69">69</a></sup>.</p>
<figure class="fig xbox font-sm" id="Fig4"><h4 class="obj_head">Fig. 4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373736_41598_2025_3857_Fig4_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e8af/12373736/d4abfee9d6e1/41598_2025_3857_Fig4_HTML.jpg" loading="lazy" id="MO4" height="607" width="669" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Changes in Gα macroswitches inducing the opening of the GDP/GTP exchange tunnel upon lemborexant binding to OX<sub>2</sub>. The opening of a tunnel was observed near the phosphate side of GDP. From left to right: interactions in nearby residues, a surface view of the Gα subunit and GDP, hydrogen bonds determined by VMD to form between R183<sup>G.hfs2.20</sup> (switch I) and E49<sup>G.s1h1.04</sup> (P loop), and R183<sup>G.hfs2.20</sup> and E241<sup>G.s4h3.10</sup> (switch III) at a maximum 4 Å distance at an angle cutoff of 20º.</p></figcaption></figure><p id="Par33">To verify if the described above direction of the observed changes in Gα is activation-like, we compared our results with the open conformation of Gα bound to rhodopsin (6CMO)<sup><a href="#CR70" class="usa-link" aria-describedby="CR70">70</a></sup> (Fig. <a href="#Fig2" class="usa-link">2</a>) in terms of Gα local and global conformational changes. A further comparison of 6CMO with 1GP2<sup><a href="#CR66" class="usa-link" aria-describedby="CR66">66</a></sup> (the inactive-state G protein with bound GDP) showed a similar opening of D277<sup>G.HG.02</sup> and F341<sup>G.H5.08</sup> as observed in our simulations (Fig. <a href="#Fig2" class="usa-link">2</a>A). Furthermore, there was a relocation of H63<sup>G.H1.12</sup> near M59<sup>G.H1.08</sup> (mentioned by Ham et al.<sup><a href="#CR62" class="usa-link" aria-describedby="CR62">62</a></sup>) from helix H1 to an adjacent loop resulting from the change in F341<sup>G.H5.08</sup>. The extension of the adjacent loop was followed by the opening of Gα (observed in full in 6CMO). This movement of H63<sup>G.H1.12</sup> was not seen in our simulations as it would require much longer simulation timescale to move AHD for such long distance observed in 6CMO (Fig. <a href="#Fig2" class="usa-link">2</a>B). However, the beginning of this H63<sup>G.H1.12</sup> translocation was observed in most simulations (Fig. <a href="#Fig2" class="usa-link">2</a>A).</p>
<p id="Par34">The release of GDP is certainly associated with the opening of Gα (6CMO vs. 1GP2). It requires the translation of the AHD to the right and bottom, in accordance with Fig. <a href="#Fig2" class="usa-link">2</a>B, which can be seen in the movement of helices HA and HD of 6CMO. Before Gα opening, residues in switches I–III (especially switch I) had to start moving in the same direction as in 6CMO. Both simulation systems showed macroswitch changes resembling those in 6CMO (Fig. <a href="#Fig2" class="usa-link">2</a>B), confirming the beginning of the G protein activation.</p>
<p id="Par35">The average RMSD values of switch III were higher for the lemborexant-including simulations than for the compound 1-including simulations (Supplementary Fig. S16). These larger conformational fluctuations correlated with the downward movement of HD and HA, causing the whole AHD to move downward, as seen in 6CMO. This movement appeared crucial for the opening of Gα (Fig. <a href="#Fig2" class="usa-link">2</a>B, Supplementary Fig. S11).</p></section><section id="Sec9"><h3 class="pmc_sec_title">Global conformational changes in G<sub>q</sub>
</h3>
<p id="Par36">Noticeable fluctuations of the AHD in both simulation systems suggest the first stage of the opening of Gα (Fig. <a href="#Fig2" class="usa-link">2</a>). Helix HD of Gα displayed fluctuations to the left or right (Fig. <a href="#Fig2" class="usa-link">2</a>B) with a simultaneous HA movement. In some replicas, HD moved right preceding Gα opening, like in 6CMO. HD movement to the left seemed to represent fluctuations of the relative position of the RD and AHD domains that did not lead to the GDP dissociation. After 1 μs, two lemborexant-including replicas showed HD moving right, but this was observed only in one compound 1-including replica. By 2 μs, only one lemborexant-including replica but two compound 1-including replicas displayed this movement. This observation indicated that fluctuations of the AHD, including HD or HA movements, likely precede the opening of Gα as this seems to be a highly dynamic process involving the simultaneous exchange of other molecules in its binding site<sup><a href="#CR70" class="usa-link" aria-describedby="CR70">70</a></sup>. Nevertheless, the simulation replicas that demonstrated rightward HD and HA fluctuations like in 6CMO also demonstrated higher fluctuations of GDP RMSD (Supplementary Figure S13), while those with leftward movement had low fluctuations of GDP RMSD. This confirmed that the global rearrangements of HD and HA observed in our simulations were indeed associated with the GDP dissociation.</p>
<p id="Par37">As mentioned above, the lemborexant systems seemed to demonstrate a faster opening of the Gα subunit, associated with the GDP dissociation, than the compound 1 systems (Supplementary Fig. 11). The way this could occur, i.e., via a hinge involving helix HF, seemed to be consistent with previously described results<sup><a href="#CR71" class="usa-link" aria-describedby="CR71">71</a></sup>. In the lemborexant-including simulations, a surface view of Gα<sub>q</sub> showed AHD gradually separating from RD, creating the GDP/GTP exchange tunnel. This was less noticeable in the compound 1 replica 1 (Fig. <a href="#Fig3" class="usa-link">3</a>, Supplementary Fig. S12). AHD remained closed in other replicas. Interestingly, the opening of Gα in the lemborexant-including simulations was accompanied by a slight rotation of the N-terminal helix of Gα, like described by Ahn et al.<sup><a href="#CR72" class="usa-link" aria-describedby="CR72">72</a></sup> on the example of two cryo-EM structures of active-state NTSR1 (Supplementary Fig. S17).</p></section></section><section id="Sec10"><h2 class="pmc_sec_title">Conclusions and discussion</h2>
<p id="Par38">The results of the all-atom MD simulations, including both mini-G<sub>sqi</sub><sup><a href="#CR15" class="usa-link" aria-describedby="CR15">15</a></sup> and full-length Gα<sub>q</sub>, suggest that lemborexant could behave as a non-typical agonist under certain conditions, rather than solely as an antagonist<sup><a href="#CR57" class="usa-link" aria-describedby="CR57">57</a>,<a href="#CR58" class="usa-link" aria-describedby="CR58">58</a></sup>. Minimal fluctuations in its position in the orthosteric binding site of OX<sub>2</sub> indicated that it could stabilize the active-state receptor conformation, similarly to the known OX2R agonist compound 1. The maintained active state of OX<sub>2</sub> during 2 μs of simulations was confirmed through a conformational analysis of selected microswitches. Recently, Tyson et al.<sup><a href="#CR73" class="usa-link" aria-describedby="CR73">73</a></sup> suggested a new molecular mechanism for inverse agonists, through which they can bind to and act through GPCR-G protein complexes. This raises the question of whether lemborexant could instead be an inverse agonist. The authors observed that the microswitches present in the structures of inverse agonists bound to the κ-opioid receptor were in a mix of active (the PIF and DRW motifs), inactive (the orthosteric and Na + binding pockets, and the CWxP motif), and intermediate states (the NPxxY motif)<sup><a href="#CR73" class="usa-link" aria-describedby="CR73">73</a></sup>. The microswitch conformations in the final frames of our simulations (Supplementary Fig. S5) were not in agreement with these observations but rather indicated the maintained active state of the receptor. On this basis, we concluded that lemborexant is unlikely to be an inverse agonist. However, there may be differences between the κ-opioid and orexin receptors that contribute to variations in inverse agonist behavior, and our simulations may not fully capture all possible receptor conformations, especially under different physiological conditions.</p>
<p id="Par39">We analyzed and summarized the observed conformational changes (Table <a href="#Tab1" class="usa-link">1</a>). Larger conformational changes of switch III correlated with the opening of Gα. This caused higher GDP position fluctuations in the lemborexant-including simulation systems in comparison to the compound 1 systems. Notably, during the first microsecond of the simulations, fluctuations of the GDP binding region were observed mostly for the lemborexant-OX<sub>2</sub>-G protein multicomplexes and only slight fluctuations were noticed for the compound 1 complexes. This suggests differences between the assumed agonist-like behavior of lemborexant and the canonical mode of action of compound 1 in G<sub>q</sub> signaling. It cannot be undoubtedly stated whether these differences refer only to the activation-like conformational changes of the studied multicomplex or also to the rate of the GDP/GTP exchange. However, we could suggest that GDP dissociation is faster for the lemborexant complexes whereas compound 1 seems to better stabilize the Gα<sub>q</sub>-GDP complex. The impact of this on the rate of the GDP/GTP exchange and subsequent G<sub>q</sub>-mediated signaling is unclear.</p>
<section class="tw xbox font-sm" id="Tab1"><h3 class="obj_head">Table 1.</h3>
<div class="caption p"><p>A comparison of the conformational changes observed in microsecond MD simulations including lemborexant and compound 1.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Conformational change</th>
<th align="left" colspan="1" rowspan="1">Compound 1 replica 1/2/3*</th>
<th align="left" colspan="1" rowspan="1">Lemborexant replica 1/2/3</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">Ligand position in the receptor binding site</td>
<td align="left" colspan="1" rowspan="1">No change</td>
<td align="left" colspan="1" rowspan="1">No change/no change/slight change</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">ECL2 closing</td>
<td align="left" colspan="1" rowspan="1">Yes/yes/yes</td>
<td align="left" colspan="1" rowspan="1">Yes/yes/yes</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Changes of macroswitch III</td>
<td align="left" colspan="1" rowspan="1">Yes/no/no</td>
<td align="left" colspan="1" rowspan="1">Yes/yes/no</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Changes of AHD position (opening of Gα)</td>
<td align="left" colspan="1" rowspan="1">Yes/no/yes</td>
<td align="left" colspan="1" rowspan="1">Yes/yes/yes</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Formation of the GDP/GTP exchange tunnel</td>
<td align="left" colspan="1" rowspan="1">Yes/no/no</td>
<td align="left" colspan="1" rowspan="1">Yes/yes/no</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Active-like direction of rotation of N-terminal helix of Gα like in NTSR1</td>
<td align="left" colspan="1" rowspan="1">Yes/no/no</td>
<td align="left" colspan="1" rowspan="1">No/no/no</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Changes of macroswitch I to ‘on’ with R183<sup>G.hfs2.02</sup> movement away from GDP</td>
<td align="left" colspan="1" rowspan="1">Yes/yes/yes</td>
<td align="left" colspan="1" rowspan="1">Yes/yes/yes</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Changes of F341<sup>G.H5.08</sup> microswitch to ‘on’</td>
<td align="left" colspan="1" rowspan="1">Yes/no/no</td>
<td align="left" colspan="1" rowspan="1">Yes/yes/no</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Changes of D277<sup>G.HG.02</sup> microswitch to ‘on’</td>
<td align="left" colspan="1" rowspan="1">Yes/yes/yes</td>
<td align="left" colspan="1" rowspan="1">Yes/yes/yes</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="_fn_p34"><p>*If differences between replicas were observed.</p></div></div></section><p id="Par40">In addition, there is the question of whether the simulations performed with an active-state apo receptor would yield similar Gα activation observations<sup><a href="#CR30" class="usa-link" aria-describedby="CR30">30</a></sup>. Though no such study has been performed for the orexin system, de Lima et al.<sup><a href="#CR74" class="usa-link" aria-describedby="CR74">74</a></sup> performed 1 µs MD simulations for an active-state apo µ-opioid receptor in complex with Gi-GDP. They observed that when the receptor was bound to Gi in a closed conformation, it switched to an inactive state and remained that way until the end of the simulation<sup><a href="#CR74" class="usa-link" aria-describedby="CR74">74</a></sup>. The authors therefore concluded that GDP kept Gi in a closed conformation, and its activation and subsequent opening could be triggered by either the presence of an agonist or a larger simulation timescale<sup><a href="#CR74" class="usa-link" aria-describedby="CR74">74</a></sup>. Previously, we also observed that the presense of an agonist is required to rebuild the receptor conformation into the fully active state, not intermediate, confirmed later with the cryo-EM structure of C5aR1<sup><a href="#CR30" class="usa-link" aria-describedby="CR30">30</a></sup>. Considering the above studies, we first observed changes in G protein microswitch conformations within 1 μs of our simulations, leading to the conclusion that it was indeed the presence of an agonist that led to G<sub>q</sub> activation. This was further confirmed by the experimental results obtained by Gratio et al.<sup><a href="#CR15" class="usa-link" aria-describedby="CR15">15</a></sup>.</p>
<p id="Par41">The results presented here show only the first stages of the GDP dissociation and G<sub>q</sub> activation accessible to MD timescales. In addition to the known role of switches I and II shown recently in metadynamics simulations of the receptor-free spontaneous G protein activation<sup><a href="#CR69" class="usa-link" aria-describedby="CR69">69</a></sup>, we described the mechanism of the beginning of the opening of Gα<sub>q</sub> and GDP dissociation. The opening of Gα<sub>q</sub> involves two major movements of AHD rightward and downward (consistent with Fig. <a href="#Fig2" class="usa-link">2</a>), in the direction known previously from the structure of the rhodopsin–Gi multicomplex (6CMO), wherein Gα is in its fully open conformation<sup><a href="#CR70" class="usa-link" aria-describedby="CR70">70</a></sup>. As we observed, one movement of AHD (rightward) could require changes of the F341<sup>G.H5.08</sup> microswitch, while the second movement (downward) could require changes in switch III. Slight changes in switch I (Fig. <a href="#Fig2" class="usa-link">2</a>), as shown in our simulations and in 6CMO, induced the translation of AHD to both sides. In addition, we analyzed the GDP binding site to determine which residues and interactions may contribute to its dissociation.</p>
<p id="Par42">The GDP dissociation required the opening of Gα and it seemed to be associated mostly with the movement of AHD to the right (Fig. <a href="#Fig4" class="usa-link">4</a>). The GDP release was mainly triggered by conformational changes near its phosphoryl end, i.e., a loss of interactions of GDP with R183<sup>G.hfs2.02</sup> located in switch I. Thus, no clear and definite changes in the D277<sup>G.HG.02</sup> microswitch were observed. Based on the above, we conclude that the sequence of conformational changes during Gα<sub>q</sub> activation mediated by an OX<sub>2</sub> agonist could be as follows (Fig. <a href="#Fig5" class="usa-link">5</a>):</p>
<ul class="list" style="list-style-type:disc">
<li><p id="Par43">changes in the F341<sup>G.H5.08</sup> microswitch,</p></li>
<li><p id="Par44">a loss of interactions between the phosphoryl end of GDP and R183<sup>G.hfs2.02</sup> (switch I) caused by changes in switch I,</p></li>
<li><p id="Par45">the formation of temporary interactions between R183<sup>G.hfs2.02</sup> and E49<sup>G.s1h1.04</sup> (P loop),</p></li>
<li><p id="Par46">a loss of interactions between R183<sup>G.hfs2.02</sup> and E49<sup>G.s1h1.04</sup>,</p></li>
<li><p id="Par47">a loss of interactions between E241<sup>G.s4h3.10</sup> (switch III) and two residues of switch II: R210<sup>G.H2.01</sup> and Q209<sup>G.s3h2.03</sup> (to a lesser extent), releasing E241<sup>G.s4h3.10</sup>,</p></li>
<li><p id="Par48">the formation of interactions between R183<sup>G.hfs2.02</sup> and E241<sup>G.s4h3.10</sup>, stabilizing the large distance between R183<sup>G.hfs2.02</sup> and GDP,</p></li>
<li><p id="Par49">the opening of the GDP/GTP exchange tunnel near the phosphoryl end of GDP,</p></li>
<li><p id="Par50">a loss of interactions between R183<sup>G.hfs2.02</sup> and E241<sup>G.s4h3.10</sup> and E49<sup>G.s1h1.04</sup></p></li>
<li><p id="Par51">changes in the D277<sup>G.HG.02</sup> microswitch,</p></li>
<li><p id="Par52">a loss of interactions between the guanine end of GDP and D277<sup>G.HG.02</sup> caused by changes in the D277<sup>G.HG.02</sup> rotamer,</p></li>
<li><p id="Par53">changes in helices HD and HA accompanied by the opening of Gα,</p></li>
<li><p id="Par54">GDP release accompanied by a further opening of Gα.</p></li>
</ul>
<figure class="fig xbox font-sm" id="Fig5"><h3 class="obj_head">Fig. 5.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373736_41598_2025_3857_Fig5_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e8af/12373736/0279819312b2/41598_2025_3857_Fig5_HTML.jpg" loading="lazy" id="MO5" height="437" width="669" alt="Fig. 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>The suggested sequence of conformational changes that take place during the activation of Gα<sub>q</sub>. The initial conformation is presented in gray, the suggested conformational change that takes place is presented in green, and the yellow dashed lines represent polar contacts.</p></figcaption></figure><p id="Par55">In the canonical G<sub>q</sub> signaling pathway, the GDP/GTP exchange initiates Gα<sub>q</sub> protein activation, followed by the breaking of most Gα–Gβγ interactions. Gα activates phospholipase C (PLC)<sup><a href="#CR75" class="usa-link" aria-describedby="CR75">75</a></sup>, which in turn converts phosphatidylinositol 4,5-bisphosphate (PIP<sub>2</sub>) into inositol 1,4,5-trisphosphate (IP<sub>3</sub>) and diacyl glycerol (DAG). IP<sub>3</sub>, acting in the cytosol, induces Ca<sup>2+</sup> release from the endoplasmic reticulum, while DAG, attached to the cell membrane, activates protein kinase C (PKC). Recent experimental results<sup><a href="#CR15" class="usa-link" aria-describedby="CR15">15</a></sup> have proven that lemborexant induces cancer cell apoptosis through a non-canonical, Ca<sup>2+</sup>-independent pathway. This suggests that lemborexant does not activate the OX<sub>1</sub>/OX<sub>2</sub>-G<sub>q</sub> complex in a classic way to release Gα for the subsequent PLC activation. Instead, Gβγ could be released or partially released to activate the Src kinase, which in turn phosphorylates the ITIM site in OX<sub>1</sub>/OX<sub>2</sub>, leading to SHP2 phosphatase recruitment, p38 MAP kinase activation, and subsequently caspase 3/7 activation. Other examples involving Gβγ have already been observed, e.g., for the Gi-coupled M2 muscarinic acetylcholine, α2-adrenergic, and somatostatin 2 receptors, and for Gs-coupled β-adrenergic receptors, which induced the ERK pathway through Gβγ-mediated stimulation of tyrosine kinases such as Src<sup><a href="#CR76" class="usa-link" aria-describedby="CR76">76</a>,<a href="#CR77" class="usa-link" aria-describedby="CR77">77</a></sup>. In contrast, the G<sub>q/11</sub>-coupled muscarinic acetylcholine receptor M1 and α1-adrenergic receptor activated the ERK-mediated apoptosis signaling by G<sub>q/11</sub><sup><a href="#CR76" class="usa-link" aria-describedby="CR76">76</a>,<a href="#CR77" class="usa-link" aria-describedby="CR77">77</a></sup>. In the case of G<sub>q</sub>-coupled P2Y<sub>1</sub> receptor, the stress-activated protein kinase (SAPK) activation (without involving p38) also evoked caspase activity, leading to apoptosis<sup><a href="#CR76" class="usa-link" aria-describedby="CR76">76</a></sup>.</p>
<p id="Par56">Here, we observed a clear difference in the GDP/GTP exchange rate leading to the opening of Gα<sub>q</sub>. The faster activation of G<sub>q</sub> by the lemborexant-OX2R complexes could be an impulse to evoke the Gβγ-mediated activation of Src. Presumably, these differences in the rate of the GDP/GTP exchange between lemborexant and compound 1 complexes affect the activation of G<sub>q</sub>, leading to the activation of signaling pathways mediated by different subunits (Gβγ vs. Gα, respectively). However, the timescale currently accessible to MD simulations is not enough to observe the evolution of the second messenger system activated by orexin receptor effectors.</p></section><section id="Sec11"><h2 class="pmc_sec_title">Supplementary Information</h2>
<section class="sm xbox font-sm" id="MOESM1"><div class="media p"><div class="caption">
<a href="/articles/instance/12373736/bin/41598_2025_3857_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">41598_2025_3857_MOESM1_ESM.pdf</a><sup> (11.3MB, pdf) </sup><p>Supplementary Information 1. Figure S1. Multiple sequence alignment of OX<sub>1</sub>, OX<sub>2</sub> (UniProt ID: O43613, O43614, respectively), NTSR1 (6OSA, 6OS9), and rhodopsin (6CMO). Figure S2. Fluctuations in the binding site area and comparison with PDB strucures. Figure S3. The formation of hydrogen bonds between R3.50 of the ‘DRW’ motif and residues T2.39 and Y5.58 in the compound 1-including simulations. Figure S4. The formation of hydrogen bonds between R3.50 of the ‘DRW’ motif and residues T2.39 and Y5.58 in the lemborexant-including simulations. Figure S5. A comparison of conformations of sequence motifs in simulations systems and in PDB structures. Figure S6. The formation of hydrogen bonds between compound 1 and the OX<sub>2</sub> receptor residues Q3.32, H7.39, and T2.61. Figure S7. The formation of hydrogen bonds between lemborexant and the OX<sub>2</sub> receptor residues Q3.32, H7.39, and T2.61. Figure S8. Multiple sequence alignment of Gα<sub>q</sub> (UniProt ID: P50148) and Gα subunits in the 7L1V and 7SQ2 PDB structures. Figure S9. Changes of the D277 (D272) and F341 (F336) microswitches conformations observed in the compound 1-including simulations. Figure S10. Changes of the D227 (D272) and F341 (F336) microswitches conformations observed in the lemborexant-including simulations. Figure S11. Changes in conformations of G protein subunits observed in MD simulations. Figure S12. Fluctuations of the AHD domain in simulations. Figure S13. Fluctuations of GDP in simulations. Figure S14. The formation of hydrogen bonds between GDP and R183<sup>G.hfs2.02</sup> in the compound 1 and lemborexant-including simulations. Figure S15. Fluctuations of the macroswitch I in simulations. Figure S16. Fluctuations of the macroswitch III in simulations. Figure S17. Changes in the orientation of N-terminal helices of Gα in the compound 1 and lemborexant-including simulations in comparison to known active-state NTSR1 structures from PDB. Figure S18. The RMSD plots and average RMSD fluctuations observed for ECL2 in the compound 1 and lemborexant-including simulations. </p>
</div></div></section><section class="sm xbox font-sm" id="MOESM2"><div class="media p"><div class="caption">
<a href="/articles/instance/12373736/bin/41598_2025_3857_MOESM2_ESM.mp4" data-ga-action="click_feat_suppl" class="usa-link">Download video file</a><sup> (45.9MB, mp4) </sup><p>Supplementary Video 1. Opening of the GDP/GTP exchange tunnel in the lemborexant-OX<sub>2</sub> complex. Here, replica 2 was shown with every 4th frame of the 2 μs simulation. Lemborexant was shown with sticks, while residues R183<sup>G.hfs2.20</sup>, E49<sup>G.s1h1.04</sup>, and E241<sup>G.s4h3.10</sup> were shown in spheres. A sudden change is visible ca. 1.8 μs (465 - 468 frames in the movie) corresponding to the sudden movement of the phosphate side of GDP to the left of E49<sup>G.s1h1.04</sup>.</p>
</div></div></section><section class="sm xbox font-sm" id="MOESM3"><div class="media p"><div class="caption">
<a href="/articles/instance/12373736/bin/41598_2025_3857_MOESM3_ESM.mp4" data-ga-action="click_feat_suppl" class="usa-link">Download video file</a><sup> (38.1MB, mp4) </sup><p>Supplementary Video 2. Opening of the Gα subunit – a side view in the lemborexant-OX<sub>2</sub> complex. Here, replica 2 was shown with every 4th frame of the 2 μs simulation. Lemborexant was shown with sticks, while residues R183<sup>G.hfs2.20</sup>, E49<sup>G.s1h1.04</sup>, and E241<sup>G.s4h3.10</sup> were shown in spheres. A sudden change is visible ca. 1.8 μs (465 - 468 frames in the movie) corresponding to the closing and final opening of the Gα subunit.</p>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>We acknowledge the High-Performance Computing Infrastructure of the University of Warsaw Biological and Chemical Research Center. We gratefully acknowledge High-Performance Computing Infrastructure PLGrid in Poland (HPC Center: ACK Cyfronet AGH) for providing computer facilities and support (PLG/2024/017063) and the LUMI supercomputer (PLL/2023/04/016464), owned by the EuroHPC Joint Undertaking, hosted by CSC (Finland) and the LUMI consortium through PLGrid, Poland. We acknowledge funds for a short-term scientific mission within the ERNEST COST Action (CA18133) (E-COST-GRANT-CA18133-37c87d4f).</p></section><section id="glossary1" class="glossary"><h2 class="pmc_sec_title">Abbreviations</h2>
<dl class="def-list">
<dt>GPCR</dt>
<dd><p id="Par2">G protein-coupled receptor</p></dd>
<dt>OX<sub>1</sub>
</dt>
<dd><p id="Par3">Orexin receptor type 1</p></dd>
<dt>OX<sub>2</sub>
</dt>
<dd><p id="Par4">Orexin receptor type 2</p></dd>
<dt>OxA</dt>
<dd><p id="Par5">Orexin-A</p></dd>
<dt>OxB</dt>
<dd><p id="Par6">Orexin-B</p></dd>
<dt>GDP</dt>
<dd><p id="Par7">Guanosine diphosphate</p></dd>
<dt>GTP</dt>
<dd><p id="Par8">Guanosine triphosphate</p></dd>
<dt>MD</dt>
<dd><p id="Par9">Molecular dynamics</p></dd>
<dt>AHD</dt>
<dd><p id="Par10">Alpha helical domain</p></dd>
<dt>RD</dt>
<dd><p id="Par11">Ras-like domain</p></dd>
<dt>MD</dt>
<dd><p id="Par12">Molecular dynamics</p></dd>
</dl></section><section id="notes1"><h2 class="pmc_sec_title">Author contributions</h2>
<p>This computational study was designed by D.L. Protein modeling, molecular dynamics simulations, and the analysis of simulation trajectories were performed by P.D. and D.L. The first draft of the manuscript was written by P.D. and D.L. Figures were designed and prepared by P.D. and D.L. Movies were prepared by D.L. The manuscript was edited and reviewed by all of the authors. All authors have given approval to the final version of the manuscript.</p></section><section id="notes2"><h2 class="pmc_sec_title">Data availability</h2>
<p>PDB structures used in the current study are available at: <a href="https://www.rcsb.org" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.rcsb.org</a> (ID: 7L1V, 7SQ2, 6OSA, 6OS9, 6CMO). Protein sequences used in the current study are available at: <a href="https://www.uniprot.org" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.uniprot.org</a> (ID: <a href="https://www.ncbi.nlm.nih.gov/protein/P50148" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">P50148</a>, <a href="https://www.ncbi.nlm.nih.gov/protein/O43613" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">O43613</a>, <a href="https://www.ncbi.nlm.nih.gov/protein/O43614" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">O43614</a>). Protein models and simulation trajectories generated during the current study are available at: 10.5281/zenodo.14035283 and 10.5281/zenodo.14035739.</p></section><section id="FPar1"><h2 class="pmc_sec_title">Competing interests</h2>
<p id="Par57">The authors declare no competing interests.</p></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="fn1">
<p><strong>Publisher’s note</strong></p>
<p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p>
</div></div></section><section id="sec12"><h2 class="pmc_sec_title">Supplementary Information</h2>
<p>The online version contains supplementary material available at 10.1038/s41598-025-03857-0.</p></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Kukkonen, J. P., Jacobson, L. H., Hoyer, D., Rinne, M. K. &amp; Borgland, S. L. International union of basic and clinical pharmacology CXIV: Orexin receptor function, nomenclature and pharmacology. <em>Pharmacol. Rev.</em><strong>76</strong>(5), 625–688. 10.1124/pharmrev.123.000953 (2024).
</cite> [<a href="https://doi.org/10.1124/pharmrev.123.000953" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38902035/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kukkonen,%20J.%20P.,%20Jacobson,%20L.%20H.,%20Hoyer,%20D.,%20Rinne,%20M.%20K.%20&amp;%20Borgland,%20S.%20L.%20International%20union%20of%20basic%20and%20clinical%20pharmacology%20CXIV:%20Orexin%20receptor%20function,%20nomenclature%20and%20pharmacology.%20Pharmacol.%20Rev.76(5),%20625%E2%80%93688.%2010.1124/pharmrev.123.000953%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>de Lecea, L. et al. The hypocretins: Hypothalamus-specific peptides with neuroexcitatory activity. <em>Proc. Natl. Acad. Sci.</em><strong>95</strong>(1), 322–327. 10.1073/pnas.95.1.322 (1998).
</cite> [<a href="https://doi.org/10.1073/pnas.95.1.322" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC18213/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9419374/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?de%20Lecea,%20L.%20et%20al.%20The%20hypocretins:%20Hypothalamus-specific%20peptides%20with%20neuroexcitatory%20activity.%20Proc.%20Natl.%20Acad.%20Sci.95(1),%20322%E2%80%93327.%2010.1073/pnas.95.1.322%20(1998)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>Sakurai, T. et al. Orexins and orexin receptors: a family of hypothalamic neuropeptides and G Protein-coupled receptors that regulate feeding behavior. <em>Cell</em><strong>92</strong>(4), 573–585. 10.1016/S0092-8674(00)80949-6 (1998).
</cite> [<a href="https://doi.org/10.1016/s0092-8674(00)80949-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9491897/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sakurai,%20T.%20et%20al.%20Orexins%20and%20orexin%20receptors:%20a%20family%20of%20hypothalamic%20neuropeptides%20and%20G%20Protein-coupled%20receptors%20that%20regulate%20feeding%20behavior.%20Cell92(4),%20573%E2%80%93585.%2010.1016/S0092-8674(00)80949-6%20(1998)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Janto, K., Prichard, J. R. &amp; Pusalavidyasagar, S. An update on dual orexin receptor antagonists and their potential role in insomnia therapeutics. <em>J. Clin. Sleep Med.</em><strong>14</strong>(08), 1399–1408. 10.5664/jcsm.7282 (2018).
</cite> [<a href="https://doi.org/10.5664/jcsm.7282" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6086961/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30092886/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Janto,%20K.,%20Prichard,%20J.%20R.%20&amp;%20Pusalavidyasagar,%20S.%20An%20update%20on%20dual%20orexin%20receptor%20antagonists%20and%20their%20potential%20role%20in%20insomnia%20therapeutics.%20J.%20Clin.%20Sleep%20Med.14(08),%201399%E2%80%931408.%2010.5664/jcsm.7282%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Couvineau, A. et al. The anti-tumoral properties of orexin/hypocretin hypothalamic neuropeptides: an unexpected therapeutic role. <em>Front. Endocrinol.</em><strong>9</strong>, 573. 10.3389/fendo.2018.00573 (2018).</cite> [<a href="https://doi.org/10.3389/fendo.2018.00573" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6170602/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30319552/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Couvineau,%20A.%20et%20al.%20The%20anti-tumoral%20properties%20of%20orexin/hypocretin%20hypothalamic%20neuropeptides:%20an%20unexpected%20therapeutic%20role.%20Front.%20Endocrinol.9,%20573.%2010.3389/fendo.2018.00573%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Chemelli, R. M. et al. Narcolepsy in orexin knockout mice: molecular genetics of sleep regulation. <em>Cell</em><strong>98</strong>(4), 437–451. 10.1016/S0092-8674(00)81973-X (1999).
</cite> [<a href="https://doi.org/10.1016/s0092-8674(00)81973-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10481909/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chemelli,%20R.%20M.%20et%20al.%20Narcolepsy%20in%20orexin%20knockout%20mice:%20molecular%20genetics%20of%20sleep%20regulation.%20Cell98(4),%20437%E2%80%93451.%2010.1016/S0092-8674(00)81973-X%20(1999)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Sutcliffe, J. G. &amp; de Lecea, L. The hypocretins: Excitatory neuromodulatory peptides for multiple homeostatic systems, including sleep and feeding. <em>J. Neurosci. Res.</em><strong>62</strong>(2), 161–168 (2000).
</cite> [<a href="https://doi.org/10.1002/1097-4547(20001015)62:2&lt;161::AID-JNR1&gt;3.0.CO;2-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11020209/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sutcliffe,%20J.%20G.%20&amp;%20de%20Lecea,%20L.%20The%20hypocretins:%20Excitatory%20neuromodulatory%20peptides%20for%20multiple%20homeostatic%20systems,%20including%20sleep%20and%20feeding.%20J.%20Neurosci.%20Res.62(2),%20161%E2%80%93168%20(2000)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>Heinonen, M. V., Purhonen, A. K., Mäkelä, K. A. &amp; Herzig, K. H. Functions of orexins in peripheral tissues. <em>Acta Physiol.</em><strong>192</strong>(4), 471–485. 10.1111/j.1748-1716.2008.01836.x (2008).</cite> [<a href="https://doi.org/10.1111/j.1748-1716.2008.01836.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18294339/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Heinonen,%20M.%20V.,%20Purhonen,%20A.%20K.,%20M%C3%A4kel%C3%A4,%20K.%20A.%20&amp;%20Herzig,%20K.%20H.%20Functions%20of%20orexins%20in%20peripheral%20tissues.%20Acta%20Physiol.192(4),%20471%E2%80%93485.%2010.1111/j.1748-1716.2008.01836.x%20(2008)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>Sakurai, T. The role of orexin in motivated behaviours. <em>Nat. Rev. Neurosci.</em><strong>15</strong>(11), 719–731. 10.1038/nrn3837 (2014).
</cite> [<a href="https://doi.org/10.1038/nrn3837" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25301357/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sakurai,%20T.%20The%20role%20of%20orexin%20in%20motivated%20behaviours.%20Nat.%20Rev.%20Neurosci.15(11),%20719%E2%80%93731.%2010.1038/nrn3837%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>Laburthe, M. &amp; Voisin, T. The orexin receptor OX1R in colon cancer: A promising therapeutic target and a new paradigm in G protein-coupled receptor signalling through ITIMs. <em>Br. J. Pharmacol.</em><strong>165</strong>(6), 1678–1687. 10.1111/j.1476-5381.2011.01510.x (2012).
</cite> [<a href="https://doi.org/10.1111/j.1476-5381.2011.01510.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3372822/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21627633/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Laburthe,%20M.%20&amp;%20Voisin,%20T.%20The%20orexin%20receptor%20OX1R%20in%20colon%20cancer:%20A%20promising%20therapeutic%20target%20and%20a%20new%20paradigm%20in%20G%20protein-coupled%20receptor%20signalling%20through%20ITIMs.%20Br.%20J.%20Pharmacol.165(6),%201678%E2%80%931687.%2010.1111/j.1476-5381.2011.01510.x%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Bonaventure, P. et al. Characterization of JNJ-42847922, a selective orexin-2 receptor antagonist, as a clinical candidate for the treatment of insomnia. <em>J. Pharmacol. Exp. Ther.</em><strong>354</strong>(3), 471–482. 10.1124/jpet.115.225466 (2015).
</cite> [<a href="https://doi.org/10.1124/jpet.115.225466" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26177655/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bonaventure,%20P.%20et%20al.%20Characterization%20of%20JNJ-42847922,%20a%20selective%20orexin-2%20receptor%20antagonist,%20as%20a%20clinical%20candidate%20for%20the%20treatment%20of%20insomnia.%20J.%20Pharmacol.%20Exp.%20Ther.354(3),%20471%E2%80%93482.%2010.1124/jpet.115.225466%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Steiner, M. A. et al. Discovery and characterization of ACT-335827, an orally available, brain penetrant orexin receptor type 1 selective antagonist. <em>ChemMedChem</em><strong>8</strong>(6), 898–903. 10.1002/cmdc.201300003 (2013).
</cite> [<a href="https://doi.org/10.1002/cmdc.201300003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23589487/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Steiner,%20M.%20A.%20et%20al.%20Discovery%20and%20characterization%20of%20ACT-335827,%20an%20orally%20available,%20brain%20penetrant%20orexin%20receptor%20type%201%20selective%20antagonist.%20ChemMedChem8(6),%20898%E2%80%93903.%2010.1002/cmdc.201300003%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>Pałasz, A. et al. Dual orexin receptor antagonists—promising agents in the treatment of sleep disorders. <em>Int. J. Neuropsychopharmacol.</em><strong>17</strong>(1), 157–168. 10.1017/S1461145713000552 (2014).
</cite> [<a href="https://doi.org/10.1017/S1461145713000552" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23702225/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Pa%C5%82asz,%20A.%20et%20al.%20Dual%20orexin%20receptor%20antagonists%E2%80%94promising%20agents%20in%20the%20treatment%20of%20sleep%20disorders.%20Int.%20J.%20Neuropsychopharmacol.17(1),%20157%E2%80%93168.%2010.1017/S1461145713000552%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>Kukkonen, J. P. &amp; Leonard, C. S. Orexin/hypocretin receptor signalling cascades. <em>Br. J. Pharmacol.</em><strong>171</strong>(2), 314–331. 10.1111/bph.12324 (2014).
</cite> [<a href="https://doi.org/10.1111/bph.12324" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3904254/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23902572/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kukkonen,%20J.%20P.%20&amp;%20Leonard,%20C.%20S.%20Orexin/hypocretin%20receptor%20signalling%20cascades.%20Br.%20J.%20Pharmacol.171(2),%20314%E2%80%93331.%2010.1111/bph.12324%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>Gratio, V., Dragan, P., Garcia, L., Saveanu, L., Nicole, P., Voisin, T., Latek, D., &amp; Couvineau, A. (2024). Pharmacodynamics of the orexin type 1 (OX<sub>1</sub>) receptor in colon cancer cell models: A two‐sided nature of antagonistic ligands resulting from partial dissociation of Gq. <em>Br. J. Pharmacol.</em> bph.17422. 10.1111/bph.17422</cite> [<a href="https://doi.org/10.1111/bph.17422" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39675769/" class="usa-link">PubMed</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Kodadek, T. &amp; Cai, D. Chemistry and biology of orexin signaling. <em>Mol. BioSyst.</em><strong>6</strong>(8), 1366–1375. 10.1039/C003468A (2010).
</cite> [<a href="https://doi.org/10.1039/c003468a" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3379554/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20532299/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kodadek,%20T.%20&amp;%20Cai,%20D.%20Chemistry%20and%20biology%20of%20orexin%20signaling.%20Mol.%20BioSyst.6(8),%201366%E2%80%931375.%2010.1039/C003468A%20(2010)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>Roch, C., Bergamini, G., Steiner, M. A. &amp; Clozel, M. Nonclinical pharmacology of daridorexant: A new dual orexin receptor antagonist for the treatment of insomnia. <em>Psychopharmacology</em><strong>238</strong>(10), 2693–2708. 10.1007/s00213-021-05954-0 (2021).
</cite> [<a href="https://doi.org/10.1007/s00213-021-05954-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8455402/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34415378/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Roch,%20C.,%20Bergamini,%20G.,%20Steiner,%20M.%20A.%20&amp;%20Clozel,%20M.%20Nonclinical%20pharmacology%20of%20daridorexant:%20A%20new%20dual%20orexin%20receptor%20antagonist%20for%20the%20treatment%20of%20insomnia.%20Psychopharmacology238(10),%202693%E2%80%932708.%2010.1007/s00213-021-05954-0%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Roth, T., Roehrs, T. &amp; Pies, R. Insomnia: Pathophysiology and implications for treatment. <em>Sleep Med. Rev.</em><strong>11</strong>(1), 71–79. 10.1016/j.smrv.2006.06.002 (2007).
</cite> [<a href="https://doi.org/10.1016/j.smrv.2006.06.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17175184/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Roth,%20T.,%20Roehrs,%20T.%20&amp;%20Pies,%20R.%20Insomnia:%20Pathophysiology%20and%20implications%20for%20treatment.%20Sleep%20Med.%20Rev.11(1),%2071%E2%80%9379.%2010.1016/j.smrv.2006.06.002%20(2007)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Couvineau, A., Nicole, P., Gratio, V. &amp; Voisin, T. Orexins/hypocretins and cancer: A neuropeptide as emerging target. <em>Molecules</em><strong>26</strong>(16), Article 16. 10.3390/molecules26164849 (2021).</cite> [<a href="https://scholar.google.com/scholar_lookup?Couvineau,%20A.,%20Nicole,%20P.,%20Gratio,%20V.%20&amp;%20Voisin,%20T.%20Orexins/hypocretins%20and%20cancer:%20A%20neuropeptide%20as%20emerging%20target.%20Molecules26(16),%20Article%2016.%2010.3390/molecules26164849%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>Yang, S. et al. Multiomics integration reveals the effect of Orexin A on glioblastoma. <em>Front. Pharmacol.</em><strong>14</strong>, 1096159. 10.3389/fphar.2023.1096159 (2023).
</cite> [<a href="https://doi.org/10.3389/fphar.2023.1096159" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9894894/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36744263/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yang,%20S.%20et%20al.%20Multiomics%20integration%20reveals%20the%20effect%20of%20Orexin%20A%20on%20glioblastoma.%20Front.%20Pharmacol.14,%201096159.%2010.3389/fphar.2023.1096159%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>Voisin, T., Firar, A. E., Avondo, V. &amp; Laburthe, M. Orexin-induced apoptosis: The key role of the seven-transmembrane domain orexin type 2 receptor. <em>Endocrinology</em><strong>147</strong>(10), 4977–4984. 10.1210/en.2006-0201 (2006).
</cite> [<a href="https://doi.org/10.1210/en.2006-0201" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16857748/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Voisin,%20T.,%20Firar,%20A.%20E.,%20Avondo,%20V.%20&amp;%20Laburthe,%20M.%20Orexin-induced%20apoptosis:%20The%20key%20role%20of%20the%20seven-transmembrane%20domain%20orexin%20type%202%20receptor.%20Endocrinology147(10),%204977%E2%80%934984.%2010.1210/en.2006-0201%20(2006)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>Voisin, T. et al. Aberrant expression of OX1 receptors for orexins in colon cancers and liver metastases: An openable gate to apoptosis. <em>Can. Res.</em><strong>71</strong>(9), 3341–3351. 10.1158/0008-5472.CAN-10-3473 (2011).</cite> [<a href="https://doi.org/10.1158/0008-5472.CAN-10-3473" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21415167/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Voisin,%20T.%20et%20al.%20Aberrant%20expression%20of%20OX1%20receptors%20for%20orexins%20in%20colon%20cancers%20and%20liver%20metastases:%20An%20openable%20gate%20to%20apoptosis.%20Can.%20Res.71(9),%203341%E2%80%933351.%2010.1158/0008-5472.CAN-10-3473%20(2011)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>Dayot, S., et al. In vitro, in vivo and ex vivo demonstration of the antitumoral role of hypocretin-1/orexin-A and almorexant in pancreatic ductal adenocarcinoma. <em>Oncotarget</em>. <strong>9</strong>(6), 6952–6967. 10.18632/oncotarget.24084 (2018).</cite> [<a href="https://doi.org/10.18632/oncotarget.24084" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5805528/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29467942/" class="usa-link">PubMed</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>Hong, C. et al. Structures of active-state orexin receptor 2 rationalize peptide and small-molecule agonist recognition and receptor activation. <em>Nat. Commun.</em><strong>12</strong>(1), 815. 10.1038/s41467-021-21087-6 (2021).
</cite> [<a href="https://doi.org/10.1038/s41467-021-21087-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7864924/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33547286/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hong,%20C.%20et%20al.%20Structures%20of%20active-state%20orexin%20receptor%202%20rationalize%20peptide%20and%20small-molecule%20agonist%20recognition%20and%20receptor%20activation.%20Nat.%20Commun.12(1),%20815.%2010.1038/s41467-021-21087-6%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>Scott, L. J. Lemborexant: First approval. <em>Drugs</em><strong>80</strong>(4), 425–432. 10.1007/s40265-020-01276-1 (2020).
</cite> [<a href="https://doi.org/10.1007/s40265-020-01276-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32096020/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Scott,%20L.%20J.%20Lemborexant:%20First%20approval.%20Drugs80(4),%20425%E2%80%93432.%2010.1007/s40265-020-01276-1%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>Hollingsworth, S. A. &amp; Dror, R. O. Molecular dynamics simulation for all. <em>Neuron</em><strong>99</strong>(6), 1129–1143. 10.1016/j.neuron.2018.08.011 (2018).
</cite> [<a href="https://doi.org/10.1016/j.neuron.2018.08.011" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6209097/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30236283/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hollingsworth,%20S.%20A.%20&amp;%20Dror,%20R.%20O.%20Molecular%20dynamics%20simulation%20for%20all.%20Neuron99(6),%201129%E2%80%931143.%2010.1016/j.neuron.2018.08.011%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Kmiecik, S. et al. Coarse-grained protein models and their applications. <em>Chem. Rev.</em><strong>116</strong>(14), 7898–7936. 10.1021/acs.chemrev.6b00163 (2016).
</cite> [<a href="https://doi.org/10.1021/acs.chemrev.6b00163" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27333362/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kmiecik,%20S.%20et%20al.%20Coarse-grained%20protein%20models%20and%20their%20applications.%20Chem.%20Rev.116(14),%207898%E2%80%937936.%2010.1021/acs.chemrev.6b00163%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Feng, Z., Ma, S., Hu, G. &amp; Xie, X.-Q. Allosteric binding site and activation mechanism of class C G-protein coupled receptors: metabotropic glutamate receptor family. <em>AAPS J.</em><strong>17</strong>(3), 737–753. 10.1208/s12248-015-9742-8 (2015).
</cite> [<a href="https://doi.org/10.1208/s12248-015-9742-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4406965/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25762450/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Feng,%20Z.,%20Ma,%20S.,%20Hu,%20G.%20&amp;%20Xie,%20X.-Q.%20Allosteric%20binding%20site%20and%20activation%20mechanism%20of%20class%20C%20G-protein%20coupled%20receptors:%20metabotropic%20glutamate%20receptor%20family.%20AAPS%20J.17(3),%20737%E2%80%93753.%2010.1208/s12248-015-9742-8%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>Wang, J. &amp; Miao, Y. Mechanistic insights into specific G protein interactions with adenosine receptors. <em>J. Phys. Chem. B</em><strong>123</strong>(30), 6462–6473. 10.1021/acs.jpcb.9b04867 (2019).
</cite> [<a href="https://doi.org/10.1021/acs.jpcb.9b04867" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7026936/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31283874/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang,%20J.%20&amp;%20Miao,%20Y.%20Mechanistic%20insights%20into%20specific%20G%20protein%20interactions%20with%20adenosine%20receptors.%20J.%20Phys.%20Chem.%20B123(30),%206462%E2%80%936473.%2010.1021/acs.jpcb.9b04867%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR30">
<span class="label">30.</span><cite>Wisniewski, S., Dragan, P., Makal, A. &amp; Latek, D. Helix 8 in chemotactic receptors of the complement system. <em>PLoS Comput. Biol.</em><strong>18</strong>(7), e1009994. 10.1371/journal.pcbi.1009994 (2022).
</cite> [<a href="https://doi.org/10.1371/journal.pcbi.1009994" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9359563/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35862436/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wisniewski,%20S.,%20Dragan,%20P.,%20Makal,%20A.%20&amp;%20Latek,%20D.%20Helix%208%20in%20chemotactic%20receptors%20of%20the%20complement%20system.%20PLoS%20Comput.%20Biol.18(7),%20e1009994.%2010.1371/journal.pcbi.1009994%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>Rodríguez-Espigares, I. et al. GPCRmd uncovers the dynamics of the 3D-GPCRome. <em>Nat. Methods</em><strong>17</strong>(8), 777–787. 10.1038/s41592-020-0884-y (2020).
</cite> [<a href="https://doi.org/10.1038/s41592-020-0884-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32661425/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rodr%C3%ADguez-Espigares,%20I.%20et%20al.%20GPCRmd%20uncovers%20the%20dynamics%20of%20the%203D-GPCRome.%20Nat.%20Methods17(8),%20777%E2%80%93787.%2010.1038/s41592-020-0884-y%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>David, Aranda-García Tomasz Maciej et al. Large scale investigation of GPCR molecular dynamics data uncovers allosteric sites and lateral gateways Nature Communications <strong>16</strong>(1). https://doi.org/10.1038/s41467-025-57034-y (2025).</cite> [<a href="https://doi.org/10.1038/s41467-025-57034-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11868581/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40016203/" class="usa-link">PubMed</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Dorota, Latek Khushil, Prajapati Paulina, Dragan Matthew, Merski Przemysław, Osial (2025) GPCRVS - AI-driven Decision Support System for GPCR Virtual Screening International Journal of Molecular Sciences 26(5) 2160-10.3390/ijms26052160 </cite> [<a href="https://doi.org/10.3390/ijms26052160" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11900134/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40076783/" class="usa-link">PubMed</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>Sanmukh, S. G., Krzykawska-Serda, M., Dragan, P., Baron, S., Lobaccaro, J.-M. A., &amp; Latek, D. <em>Is Cancer Our Equal or Our Better? Artificial Intelligence in Cancer Drug Discovery</em> (Springer International Publishing, 2024). 10.1007/16833_2024_326.</cite>
</li>
<li id="CR35">
<span class="label">35.</span><cite>Yin, J. et al. Structure and ligand-binding mechanism of the human OX1 and OX2 orexin receptors. <em>Nat. Struct. Mol. Biol.</em><strong>23</strong>(4), Article 4. 10.1038/nsmb.3183 (2016).</cite> [<a href="https://doi.org/10.1038/nsmb.3183" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26950369/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yin,%20J.%20et%20al.%20Structure%20and%20ligand-binding%20mechanism%20of%20the%20human%20OX1%20and%20OX2%20orexin%20receptors.%20Nat.%20Struct.%20Mol.%20Biol.23(4),%20Article%204.%2010.1038/nsmb.3183%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR36">
<span class="label">36.</span><cite>Asada, H. et al. Molecular basis for anti-insomnia drug design from structure of lemborexant-bound orexin 2 receptor. <em>Structure</em><strong>30</strong>(12), 1582-1589.e4. 10.1016/j.str.2022.11.001 (2022).
</cite> [<a href="https://doi.org/10.1016/j.str.2022.11.001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36417909/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Asada,%20H.%20et%20al.%20Molecular%20basis%20for%20anti-insomnia%20drug%20design%20from%20structure%20of%20lemborexant-bound%20orexin%202%20receptor.%20Structure30(12),%201582-1589.e4.%2010.1016/j.str.2022.11.001%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR37">
<span class="label">37.</span><cite>Karhu, L., Magarkar, A., Bunker, A. &amp; Xhaard, H. Determinants of orexin receptor binding and activation—a molecular dynamics study. <em>J. Phys. Chem. B</em><strong>123</strong>(12), 2609–2622. 10.1021/acs.jpcb.8b10220 (2019).
</cite> [<a href="https://doi.org/10.1021/acs.jpcb.8b10220" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6727383/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30786708/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Karhu,%20L.,%20Magarkar,%20A.,%20Bunker,%20A.%20&amp;%20Xhaard,%20H.%20Determinants%20of%20orexin%20receptor%20binding%20and%20activation%E2%80%94a%20molecular%20dynamics%20study.%20J.%20Phys.%20Chem.%20B123(12),%202609%E2%80%932622.%2010.1021/acs.jpcb.8b10220%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR38">
<span class="label">38.</span><cite>Dragan, P. &amp; Latek, D. The two-sided impact of beta-adrenergic receptor ligands on inflammation. <em>Curr. Opin. Physiol.</em><strong>41</strong>, 100779. 10.1016/j.cophys.2024.100779 (2024).</cite> [<a href="https://scholar.google.com/scholar_lookup?Dragan,%20P.%20&amp;%20Latek,%20D.%20The%20two-sided%20impact%20of%20beta-adrenergic%20receptor%20ligands%20on%20inflammation.%20Curr.%20Opin.%20Physiol.41,%20100779.%2010.1016/j.cophys.2024.100779%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR39">
<span class="label">39.</span><cite>Nehmé, R. et al. Mini-G proteins: Novel tools for studying GPCRs in their active conformation. <em>PLoS ONE</em><strong>12</strong>(4), e0175642. 10.1371/journal.pone.0175642 (2017).
</cite> [<a href="https://doi.org/10.1371/journal.pone.0175642" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5398546/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28426733/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Nehm%C3%A9,%20R.%20et%20al.%20Mini-G%20proteins:%20Novel%20tools%20for%20studying%20GPCRs%20in%20their%20active%20conformation.%20PLoS%20ONE12(4),%20e0175642.%2010.1371/journal.pone.0175642%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR40">
<span class="label">40.</span><cite>Eswar, N. et al. Comparative protein structure modeling using modeller. <em>Curr. Protoc. Bioinform.</em>10.1002/0471250953.bi0506s15 (2006).</cite> [<a href="https://doi.org/10.1002/0471250953.bi0506s15" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4186674/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18428767/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Eswar,%20N.%20et%20al.%20Comparative%20protein%20structure%20modeling%20using%20modeller.%20Curr.%20Protoc.%20Bioinform.10.1002/0471250953.bi0506s15%20(2006)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR41">
<span class="label">41.</span><cite>Waldo, G. L. et al. Kinetic scaffolding mediated by a phospholipase C–β and Gq signaling complex. <em>Science</em><strong>330</strong>(6006), 974–980. 10.1126/science.1193438 (2010).
</cite> [<a href="https://doi.org/10.1126/science.1193438" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3046049/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20966218/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Waldo,%20G.%20L.%20et%20al.%20Kinetic%20scaffolding%20mediated%20by%20a%20phospholipase%20C%E2%80%93%CE%B2%20and%20Gq%20signaling%20complex.%20Science330(6006),%20974%E2%80%93980.%2010.1126/science.1193438%20(2010)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR42">
<span class="label">42.</span><cite>The UniProt Consortium. UniProt: The universal protein knowledgebase in 2023. <em>Nucleic Acids Res.</em><strong>51</strong>(D1), D523–D531. 10.1093/nar/gkac1052 (2023).
</cite> [<a href="https://doi.org/10.1093/nar/gkac1052" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9825514/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36408920/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?The%20UniProt%20Consortium.%20UniProt:%20The%20universal%20protein%20knowledgebase%20in%202023.%20Nucleic%20Acids%20Res.51(D1),%20D523%E2%80%93D531.%2010.1093/nar/gkac1052%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR43">
<span class="label">43.</span><cite>Sadler, F. et al. Autoregulation of GPCR signalling through the third intracellular loop. <em>Nature</em><strong>615</strong>(7953), 734–741. 10.1038/s41586-023-05789-z (2023).
</cite> [<a href="https://doi.org/10.1038/s41586-023-05789-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10033409/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36890236/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sadler,%20F.%20et%20al.%20Autoregulation%20of%20GPCR%20signalling%20through%20the%20third%20intracellular%20loop.%20Nature615(7953),%20734%E2%80%93741.%2010.1038/s41586-023-05789-z%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR44">
<span class="label">44.</span><cite>Jaakola, V.-P., Prilusky, J., Sussman, J. L. &amp; Goldman, A. G protein-coupled receptors show unusual patterns of intrinsic unfolding. <em>Protein Eng. Des. Sel.</em><strong>18</strong>(2), 103–110. 10.1093/protein/gzi004 (2005).
</cite> [<a href="https://doi.org/10.1093/protein/gzi004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15790574/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jaakola,%20V.-P.,%20Prilusky,%20J.,%20Sussman,%20J.%20L.%20&amp;%20Goldman,%20A.%20G%20protein-coupled%20receptors%20show%20unusual%20patterns%20of%20intrinsic%20unfolding.%20Protein%20Eng.%20Des.%20Sel.18(2),%20103%E2%80%93110.%2010.1093/protein/gzi004%20(2005)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR45">
<span class="label">45.</span><cite>Deupi, X. Relevance of rhodopsin studies for GPCR activation. <em>Biochim. Biophys. Acta BBA Bioenerget.</em><strong>1837</strong>(5), 674–682. 10.1016/j.bbabio.2013.09.002 (2014).</cite> [<a href="https://doi.org/10.1016/j.bbabio.2013.09.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24041646/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Deupi,%20X.%20Relevance%20of%20rhodopsin%20studies%20for%20GPCR%20activation.%20Biochim.%20Biophys.%20Acta%20BBA%20Bioenerget.1837(5),%20674%E2%80%93682.%2010.1016/j.bbabio.2013.09.002%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR46">
<span class="label">46.</span><cite>Qiu, X. et al. Coupling and activation of the β1 adrenergic receptor—the role of the third intracellular loop. <em>J. Am. Chem. Soc.</em><strong>146</strong>(41), 28527–28537. 10.1021/jacs.4c11250 (2024).
</cite> [<a href="https://doi.org/10.1021/jacs.4c11250" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11487556/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39359104/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Qiu,%20X.%20et%20al.%20Coupling%20and%20activation%20of%20the%20%CE%B21%20adrenergic%20receptor%E2%80%94the%20role%20of%20the%20third%20intracellular%20loop.%20J.%20Am.%20Chem.%20Soc.146(41),%2028527%E2%80%9328537.%2010.1021/jacs.4c11250%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR47">
<span class="label">47.</span><cite>Shen, M. &amp; Sali, A. Statistical potential for assessment and prediction of protein structures. <em>Protein Sci. Publ. Protein Soc.</em><strong>15</strong>(11), 2507–2524. 10.1110/ps.062416606 (2006).</cite> [<a href="https://doi.org/10.1110/ps.062416606" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2242414/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17075131/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Shen,%20M.%20&amp;%20Sali,%20A.%20Statistical%20potential%20for%20assessment%20and%20prediction%20of%20protein%20structures.%20Protein%20Sci.%20Publ.%20Protein%20Soc.15(11),%202507%E2%80%932524.%2010.1110/ps.062416606%20(2006)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR48">
<span class="label">48.</span><cite><em>The PyMOL Molecular Graphics System, Schrödinger</em> (Version 2.4.0). (n.d.). [Computer software]. Schrödinger, LLC.</cite>
</li>
<li id="CR49">
<span class="label">49.</span><cite><em>Maestro</em>. (2022). [Computer software]. Schrödinger, LLC.</cite>
</li>
<li id="CR50">
<span class="label">50.</span><cite>Lee, J. et al. CHARMM-GUI input generator for NAMD, GROMACS, AMBER, OpenMM, and CHARMM/OpenMM simulations using the CHARMM36 additive force field. <em>J. Chem. Theory Comput.</em><strong>12</strong>(1), 405–413. 10.1021/acs.jctc.5b00935 (2016).
</cite> [<a href="https://doi.org/10.1021/acs.jctc.5b00935" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4712441/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26631602/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lee,%20J.%20et%20al.%20CHARMM-GUI%20input%20generator%20for%20NAMD,%20GROMACS,%20AMBER,%20OpenMM,%20and%20CHARMM/OpenMM%20simulations%20using%20the%20CHARMM36%20additive%20force%20field.%20J.%20Chem.%20Theory%20Comput.12(1),%20405%E2%80%93413.%2010.1021/acs.jctc.5b00935%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR51">
<span class="label">51.</span><cite>Wu, E. L. et al. CHARMM-GUI membrane builder toward realistic biological membrane simulations. <em>J. Comput. Chem.</em><strong>35</strong>(27), 1997–2004. 10.1002/jcc.23702 (2014).
</cite> [<a href="https://doi.org/10.1002/jcc.23702" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4165794/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25130509/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wu,%20E.%20L.%20et%20al.%20CHARMM-GUI%20membrane%20builder%20toward%20realistic%20biological%20membrane%20simulations.%20J.%20Comput.%20Chem.35(27),%201997%E2%80%932004.%2010.1002/jcc.23702%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR52">
<span class="label">52.</span><cite>Sengupta, D. &amp; Chattopadhyay, A. Identification of cholesterol binding sites in the Serotonin1A receptor. <em>J. Phys. Chem. B</em><strong>116</strong>(43), 12991–12996. 10.1021/jp309888u (2012).
</cite> [<a href="https://doi.org/10.1021/jp309888u" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23067252/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sengupta,%20D.%20&amp;%20Chattopadhyay,%20A.%20Identification%20of%20cholesterol%20binding%20sites%20in%20the%20Serotonin1A%20receptor.%20J.%20Phys.%20Chem.%20B116(43),%2012991%E2%80%9312996.%2010.1021/jp309888u%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR53">
<span class="label">53.</span><cite>Phillips, J. C. et al. Scalable molecular dynamics on CPU and GPU architectures with NAMD. <em>J. Chem. Phys.</em><strong>153</strong>(4), 044130. 10.1063/5.0014475 (2020).
</cite> [<a href="https://doi.org/10.1063/5.0014475" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7395834/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32752662/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Phillips,%20J.%20C.%20et%20al.%20Scalable%20molecular%20dynamics%20on%20CPU%20and%20GPU%20architectures%20with%20NAMD.%20J.%20Chem.%20Phys.153(4),%20044130.%2010.1063/5.0014475%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR54">
<span class="label">54.</span><cite>Gullingsrud, J. (version 4.0). <em>CatDCD</em> (Version 4.0) [Computer software]. Theoretical and Computational Biophysics Group.</cite>
</li>
<li id="CR55">
<span class="label">55.</span><cite>Giorgino, T., Henin, J., Lenz, O., Mura, C., &amp; Saam, J. (version 2.7). <em>PBCTools</em> (Version 2.7) [Computer software]. Theoretical and Computational Biophysics Group.</cite>
</li>
<li id="CR56">
<span class="label">56.</span><cite>Humphrey, W., Dalke, A. &amp; Schulten, K. VMD: Visual molecular dynamics. <em>J. Mol. Graph.</em><strong>14</strong>(1), 33–38. 10.1016/0263-7855(96)00018-5 (1996).
</cite> [<a href="https://doi.org/10.1016/0263-7855(96)00018-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/8744570/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Humphrey,%20W.,%20Dalke,%20A.%20&amp;%20Schulten,%20K.%20VMD:%20Visual%20molecular%20dynamics.%20J.%20Mol.%20Graph.14(1),%2033%E2%80%9338.%2010.1016/0263-7855(96)00018-5%20(1996)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR57">
<span class="label">57.</span><cite>Beuckmann, C. T. et al. In vitro and in silico characterization of lemborexant (E2006), a novel dual orexin receptor antagonist. <em>J. Pharmacol. Exp. Ther.</em><strong>362</strong>(2), 287–295. 10.1124/jpet.117.241422 (2017).
</cite> [<a href="https://doi.org/10.1124/jpet.117.241422" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28559480/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Beuckmann,%20C.%20T.%20et%20al.%20In%20vitro%20and%20in%20silico%20characterization%20of%20lemborexant%20(E2006),%20a%20novel%20dual%20orexin%20receptor%20antagonist.%20J.%20Pharmacol.%20Exp.%20Ther.362(2),%20287%E2%80%93295.%2010.1124/jpet.117.241422%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR58">
<span class="label">58.</span><cite>Yoshida, Y. et al. Design, synthesis, and structure–activity relationships of a series of novel N-aryl-2-phenylcyclopropanecarboxamide that are potent and orally active orexin receptor antagonists. <em>Bioorg. Med. Chem.</em><strong>22</strong>(21), 6071–6088. 10.1016/j.bmc.2014.08.034 (2014).
</cite> [<a href="https://doi.org/10.1016/j.bmc.2014.08.034" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25267004/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yoshida,%20Y.%20et%20al.%20Design,%20synthesis,%20and%20structure%E2%80%93activity%20relationships%20of%20a%20series%20of%20novel%20N-aryl-2-phenylcyclopropanecarboxamide%20that%20are%20potent%20and%20orally%20active%20orexin%20receptor%20antagonists.%20Bioorg.%20Med.%20Chem.22(21),%206071%E2%80%936088.%2010.1016/j.bmc.2014.08.034%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR59">
<span class="label">59.</span><cite>Hauser, A. S. et al. GPCR activation mechanisms across classes and macro/microscales. <em>Nat. Struct. Mol. Biol.</em><strong>28</strong>(11), Article 11. 10.1038/s41594-021-00674-7 (2021).</cite> [<a href="https://doi.org/10.1038/s41594-021-00674-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8580822/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34759375/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hauser,%20A.%20S.%20et%20al.%20GPCR%20activation%20mechanisms%20across%20classes%20and%20macro/microscales.%20Nat.%20Struct.%20Mol.%20Biol.28(11),%20Article%2011.%2010.1038/s41594-021-00674-7%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR60">
<span class="label">60.</span><cite>Trzaskowski, B. et al. Action of molecular switches in GPCRs—theoretical and experimental studies. <em>Curr. Med. Chem.</em><strong>19</strong>(8), 1090–1109. 10.2174/092986712799320556 (2012).
</cite> [<a href="https://doi.org/10.2174/092986712799320556" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3343417/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22300046/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Trzaskowski,%20B.%20et%20al.%20Action%20of%20molecular%20switches%20in%20GPCRs%E2%80%94theoretical%20and%20experimental%20studies.%20Curr.%20Med.%20Chem.19(8),%201090%E2%80%931109.%2010.2174/092986712799320556%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR61">
<span class="label">61.</span><cite>Kjølbye, L. R. et al. Lipid modulation of a class B GPCR: elucidating the modulatory role of PI(4,5)P2 lipids. <em>J. Chem. Inf. Model.</em><strong>62</strong>(24), 6788–6802. 10.1021/acs.jcim.2c00635 (2022).
</cite> [<a href="https://doi.org/10.1021/acs.jcim.2c00635" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36036575/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kj%C3%B8lbye,%20L.%20R.%20et%20al.%20Lipid%20modulation%20of%20a%20class%20B%20GPCR:%20elucidating%20the%20modulatory%20role%20of%20PI(4,5)P2%20lipids.%20J.%20Chem.%20Inf.%20Model.62(24),%206788%E2%80%936802.%2010.1021/acs.jcim.2c00635%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR62">
<span class="label">62.</span><cite>Ham, D. et al. Conformational switch that induces GDP release from Gi. <em>J. Struct. Biol.</em><strong>213</strong>(1), 107694. 10.1016/j.jsb.2020.107694 (2021).
</cite> [<a href="https://doi.org/10.1016/j.jsb.2020.107694" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33418033/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ham,%20D.%20et%20al.%20Conformational%20switch%20that%20induces%20GDP%20release%20from%20Gi.%20J.%20Struct.%20Biol.213(1),%20107694.%2010.1016/j.jsb.2020.107694%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR63">
<span class="label">63.</span><cite>Louet, M., Martinez, J. &amp; Floquet, N. GDP release preferentially occurs on the phosphate side in heterotrimeric G-proteins. <em>PLoS Comput. Biol.</em><strong>8</strong>(7), e1002595. 10.1371/journal.pcbi.1002595 (2012).
</cite> [<a href="https://doi.org/10.1371/journal.pcbi.1002595" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3400569/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22829757/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Louet,%20M.,%20Martinez,%20J.%20&amp;%20Floquet,%20N.%20GDP%20release%20preferentially%20occurs%20on%20the%20phosphate%20side%20in%20heterotrimeric%20G-proteins.%20PLoS%20Comput.%20Biol.8(7),%20e1002595.%2010.1371/journal.pcbi.1002595%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR64">
<span class="label">64.</span><cite>Calebiro, D., Koszegi, Z., Lanoiselée, Y., Miljus, T. &amp; O’Brien, S. G protein-coupled receptor-G protein interactions: A single-molecule perspective. <em>Physiol. Rev.</em><strong>101</strong>(3), 857–906. 10.1152/physrev.00021.2020 (2021).
</cite> [<a href="https://doi.org/10.1152/physrev.00021.2020" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33331229/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Calebiro,%20D.,%20Koszegi,%20Z.,%20Lanoisel%C3%A9e,%20Y.,%20Miljus,%20T.%20&amp;%20O%E2%80%99Brien,%20S.%20G%20protein-coupled%20receptor-G%20protein%20interactions:%20A%20single-molecule%20perspective.%20Physiol.%20Rev.101(3),%20857%E2%80%93906.%2010.1152/physrev.00021.2020%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR65">
<span class="label">65.</span><cite>Lambright, D. G. et al. The 2.0 Å crystal structure of a heterotrimeric G protein. <em>Nature</em><strong>379</strong>(6563), 311–319. 10.1038/379311a0 (1996).
</cite> [<a href="https://doi.org/10.1038/379311a0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/8552184/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lambright,%20D.%20G.%20et%20al.%20The%202.0%20%C3%85%20crystal%20structure%20of%20a%20heterotrimeric%20G%20protein.%20Nature379(6563),%20311%E2%80%93319.%2010.1038/379311a0%20(1996)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR66">
<span class="label">66.</span><cite>Wall, M. A. et al. The structure of the G protein heterotrimer Giα1β1γ2. <em>Cell</em><strong>83</strong>(6), 1047–1058. 10.1016/0092-8674(95)90220-1 (1995).
</cite> [<a href="https://doi.org/10.1016/0092-8674(95)90220-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/8521505/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wall,%20M.%20A.%20et%20al.%20The%20structure%20of%20the%20G%20protein%20heterotrimer%20Gi%CE%B11%CE%B21%CE%B32.%20Cell83(6),%201047%E2%80%931058.%2010.1016/0092-8674(95)90220-1%20(1995)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR67">
<span class="label">67.</span><cite>Dohlman, H. G. &amp; Jones, J. C. Signal activation and inactivation by the Gα helical domain: A long-neglected partner in G protein signaling. <em>Sci. Signal.</em><strong>5</strong>(226), re2. 10.1126/scisignal.2003013 (2012).
</cite> [<a href="https://doi.org/10.1126/scisignal.2003013" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3557789/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22649098/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Dohlman,%20H.%20G.%20&amp;%20Jones,%20J.%20C.%20Signal%20activation%20and%20inactivation%20by%20the%20G%CE%B1%20helical%20domain:%20A%20long-neglected%20partner%20in%20G%20protein%20signaling.%20Sci.%20Signal.5(226),%20re2.%2010.1126/scisignal.2003013%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR68">
<span class="label">68.</span><cite>Pándy-Szekeres, G. et al. The G protein database, GproteinDb. <em>Nucleic Acids Res.</em><strong>50</strong>(D1), D518–D525. 10.1093/nar/gkab852 (2022).
</cite> [<a href="https://doi.org/10.1093/nar/gkab852" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8728128/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34570219/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?P%C3%A1ndy-Szekeres,%20G.%20et%20al.%20The%20G%20protein%20database,%20GproteinDb.%20Nucleic%20Acids%20Res.50(D1),%20D518%E2%80%93D525.%2010.1093/nar/gkab852%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR69">
<span class="label">69.</span><cite>Sun, X., Singh, S., Blumer, K. J. &amp; Bowman, G. R. Simulation of spontaneous G protein activation reveals a new intermediate driving GDP unbinding. <em>Elife</em><strong>7</strong>, e38465. 10.7554/eLife.38465 (2018).
</cite> [<a href="https://doi.org/10.7554/eLife.38465" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6224197/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30289386/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sun,%20X.,%20Singh,%20S.,%20Blumer,%20K.%20J.%20&amp;%20Bowman,%20G.%20R.%20Simulation%20of%20spontaneous%20G%20protein%20activation%20reveals%20a%20new%20intermediate%20driving%20GDP%20unbinding.%20Elife7,%20e38465.%2010.7554/eLife.38465%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR70">
<span class="label">70.</span><cite>Kang, Y. et al. Cryo-EM structure of human rhodopsin bound to an inhibitory G protein. <em>Nature</em><strong>558</strong>(7711), Article 7711. 10.1038/s41586-018-0215-y (2018).</cite> [<a href="https://doi.org/10.1038/s41586-018-0215-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8054211/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29899450/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kang,%20Y.%20et%20al.%20Cryo-EM%20structure%20of%20human%20rhodopsin%20bound%20to%20an%20inhibitory%20G%20protein.%20Nature558(7711),%20Article%207711.%2010.1038/s41586-018-0215-y%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR71">
<span class="label">71.</span><cite>Dror, R. O. et al. Structural basis for nucleotide exchange in heterotrimeric G proteins. <em>Science</em><strong>348</strong>(6241), 1361–1365. 10.1126/science.aaa5264 (2015).
</cite> [<a href="https://doi.org/10.1126/science.aaa5264" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4968074/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26089515/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Dror,%20R.%20O.%20et%20al.%20Structural%20basis%20for%20nucleotide%20exchange%20in%20heterotrimeric%20G%20proteins.%20Science348(6241),%201361%E2%80%931365.%2010.1126/science.aaa5264%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR72">
<span class="label">72.</span><cite>Ahn, D., Ham, D. &amp; Chung, K. Y. The conformational transition during G protein-coupled receptor (GPCR) and G protein interaction. <em>Curr. Opin. Struct. Biol.</em><strong>69</strong>, 117–123. 10.1016/j.sbi.2021.03.013 (2021).
</cite> [<a href="https://doi.org/10.1016/j.sbi.2021.03.013" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33975155/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ahn,%20D.,%20Ham,%20D.%20&amp;%20Chung,%20K.%20Y.%20The%20conformational%20transition%20during%20G%20protein-coupled%20receptor%20(GPCR)%20and%20G%20protein%20interaction.%20Curr.%20Opin.%20Struct.%20Biol.69,%20117%E2%80%93123.%2010.1016/j.sbi.2021.03.013%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR73">
<span class="label">73.</span><cite>Tyson, A. S., et al. Molecular mechanisms of inverse agonism via κ-opioid receptor–G protein complexes. <em>Nat. Chem. Biol.</em> 1–12. 10.1038/s41589-024-01812-0 (2025).</cite> [<a href="https://doi.org/10.1038/s41589-024-01812-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC12202187/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39775170/" class="usa-link">PubMed</a>]</li>
<li id="CR74">
<span class="label">74.</span><cite>Lima, M. R. P. D., Bezerra, R. F. S., Serafim, D. D. B. &amp; Sena Junior, D. M. Dynamics of the Apo µ-opioid receptor in complex with Gi protein. <em>Int. J. Mol. Sci.</em><strong>24</strong>(17), 13430. 10.3390/ijms241713430 (2023).
</cite> [<a href="https://doi.org/10.3390/ijms241713430" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10487971/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37686252/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lima,%20M.%20R.%20P.%20D.,%20Bezerra,%20R.%20F.%20S.,%20Serafim,%20D.%20D.%20B.%20&amp;%20Sena%20Junior,%20D.%20M.%20Dynamics%20of%20the%20Apo%20%C2%B5-opioid%20receptor%20in%20complex%20with%20Gi%20protein.%20Int.%20J.%20Mol.%20Sci.24(17),%2013430.%2010.3390/ijms241713430%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR75">
<span class="label">75.</span><cite>Suh, P.-G. et al. Multiple roles of phosphoinositide-specific phospholipase C isozymes. <em>BMB Rep.</em><strong>41</strong>(6), 415–434. 10.5483/BMBRep.2008.41.6.415 (2008).
</cite> [<a href="https://doi.org/10.5483/bmbrep.2008.41.6.415" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18593525/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Suh,%20P.-G.%20et%20al.%20Multiple%20roles%20of%20phosphoinositide-specific%20phospholipase%20C%20isozymes.%20BMB%20Rep.41(6),%20415%E2%80%93434.%2010.5483/BMBRep.2008.41.6.415%20(2008)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR76">
<span class="label">76.</span><cite>Sellers, L. A. et al. Adenosine nucleotides acting at the human P2Y1 receptor stimulate mitogen-activated protein kinases and induce apoptosis. <em>J. Biol. Chem.</em><strong>276</strong>(19), 16379–16390. 10.1074/jbc.M006617200 (2001).
</cite> [<a href="https://doi.org/10.1074/jbc.M006617200" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11278310/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sellers,%20L.%20A.%20et%20al.%20Adenosine%20nucleotides%20acting%20at%20the%20human%20P2Y1%20receptor%20stimulate%20mitogen-activated%20protein%20kinases%20and%20induce%20apoptosis.%20J.%20Biol.%20Chem.276(19),%2016379%E2%80%9316390.%2010.1074/jbc.M006617200%20(2001)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR77">
<span class="label">77.</span><cite>Yanamadala, V., Negoro, H. &amp; Denker, B. Heterotrimeric G proteins and apoptosis: intersecting signaling pathways leading to context dependent phenotypes. <em>Curr. Mol. Med.</em><strong>9</strong>(5), 527–545. 10.2174/156652409788488784 (2009).
</cite> [<a href="https://doi.org/10.2174/156652409788488784" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2822437/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19601805/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yanamadala,%20V.,%20Negoro,%20H.%20&amp;%20Denker,%20B.%20Heterotrimeric%20G%20proteins%20and%20apoptosis:%20intersecting%20signaling%20pathways%20leading%20to%20context%20dependent%20phenotypes.%20Curr.%20Mol.%20Med.9(5),%20527%E2%80%93545.%2010.2174/156652409788488784%20(2009)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12373736/bin/41598_2025_3857_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">41598_2025_3857_MOESM1_ESM.pdf</a><sup> (11.3MB, pdf) </sup><p>Supplementary Information 1. Figure S1. Multiple sequence alignment of OX<sub>1</sub>, OX<sub>2</sub> (UniProt ID: O43613, O43614, respectively), NTSR1 (6OSA, 6OS9), and rhodopsin (6CMO). Figure S2. Fluctuations in the binding site area and comparison with PDB strucures. Figure S3. The formation of hydrogen bonds between R3.50 of the ‘DRW’ motif and residues T2.39 and Y5.58 in the compound 1-including simulations. Figure S4. The formation of hydrogen bonds between R3.50 of the ‘DRW’ motif and residues T2.39 and Y5.58 in the lemborexant-including simulations. Figure S5. A comparison of conformations of sequence motifs in simulations systems and in PDB structures. Figure S6. The formation of hydrogen bonds between compound 1 and the OX<sub>2</sub> receptor residues Q3.32, H7.39, and T2.61. Figure S7. The formation of hydrogen bonds between lemborexant and the OX<sub>2</sub> receptor residues Q3.32, H7.39, and T2.61. Figure S8. Multiple sequence alignment of Gα<sub>q</sub> (UniProt ID: P50148) and Gα subunits in the 7L1V and 7SQ2 PDB structures. Figure S9. Changes of the D277 (D272) and F341 (F336) microswitches conformations observed in the compound 1-including simulations. Figure S10. Changes of the D227 (D272) and F341 (F336) microswitches conformations observed in the lemborexant-including simulations. Figure S11. Changes in conformations of G protein subunits observed in MD simulations. Figure S12. Fluctuations of the AHD domain in simulations. Figure S13. Fluctuations of GDP in simulations. Figure S14. The formation of hydrogen bonds between GDP and R183<sup>G.hfs2.02</sup> in the compound 1 and lemborexant-including simulations. Figure S15. Fluctuations of the macroswitch I in simulations. Figure S16. Fluctuations of the macroswitch III in simulations. Figure S17. Changes in the orientation of N-terminal helices of Gα in the compound 1 and lemborexant-including simulations in comparison to known active-state NTSR1 structures from PDB. Figure S18. The RMSD plots and average RMSD fluctuations observed for ECL2 in the compound 1 and lemborexant-including simulations. </p>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12373736/bin/41598_2025_3857_MOESM2_ESM.mp4" data-ga-action="click_feat_suppl" class="usa-link">Download video file</a><sup> (45.9MB, mp4) </sup><p>Supplementary Video 1. Opening of the GDP/GTP exchange tunnel in the lemborexant-OX<sub>2</sub> complex. Here, replica 2 was shown with every 4th frame of the 2 μs simulation. Lemborexant was shown with sticks, while residues R183<sup>G.hfs2.20</sup>, E49<sup>G.s1h1.04</sup>, and E241<sup>G.s4h3.10</sup> were shown in spheres. A sudden change is visible ca. 1.8 μs (465 - 468 frames in the movie) corresponding to the sudden movement of the phosphate side of GDP to the left of E49<sup>G.s1h1.04</sup>.</p>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12373736/bin/41598_2025_3857_MOESM3_ESM.mp4" data-ga-action="click_feat_suppl" class="usa-link">Download video file</a><sup> (38.1MB, mp4) </sup><p>Supplementary Video 2. Opening of the Gα subunit – a side view in the lemborexant-OX<sub>2</sub> complex. Here, replica 2 was shown with every 4th frame of the 2 μs simulation. Lemborexant was shown with sticks, while residues R183<sup>G.hfs2.20</sup>, E49<sup>G.s1h1.04</sup>, and E241<sup>G.s4h3.10</sup> were shown in spheres. A sudden change is visible ca. 1.8 μs (465 - 468 frames in the movie) corresponding to the closing and final opening of the Gα subunit.</p>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>PDB structures used in the current study are available at: <a href="https://www.rcsb.org" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.rcsb.org</a> (ID: 7L1V, 7SQ2, 6OSA, 6OS9, 6CMO). Protein sequences used in the current study are available at: <a href="https://www.uniprot.org" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.uniprot.org</a> (ID: <a href="https://www.ncbi.nlm.nih.gov/protein/P50148" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">P50148</a>, <a href="https://www.ncbi.nlm.nih.gov/protein/O43613" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">O43613</a>, <a href="https://www.ncbi.nlm.nih.gov/protein/O43614" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">O43614</a>). Protein models and simulation trajectories generated during the current study are available at: 10.5281/zenodo.14035283 and 10.5281/zenodo.14035739.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Scientific Reports are provided here courtesy of <strong>Nature Publishing Group</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1038/s41598-025-03857-0"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/41598_2025_Article_3857.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (3.0 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12373736/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12373736/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12373736%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373736/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12373736/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12373736/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40846858/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12373736/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40846858/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12373736/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12373736/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="QScDsIMk5nB76XOjPv70RIWAPvKJkETRhpZo3GdBoA8hQB7HnafHqqdRiRUVM7Aj">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
