
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Personalized Genotype‐Based Approach for Treatment of Phenylketonuria - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="AE530F1A8AEED833050F1A004CB1AAF3.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="blackwellopen">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12307255/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Journal of Inherited Metabolic Disease">
<meta name="citation_title" content="Personalized Genotype‐Based Approach for Treatment of Phenylketonuria">
<meta name="citation_author" content="Polina Gundorova">
<meta name="citation_author_institution" content="University Children&#x27;s Research, University Medical Center Hamburg‐Eppendorf (UKE), Hamburg, Germany">
<meta name="citation_author_institution" content="German Center for Child and Adolescent Health (DZKJ), Partner Site Hamburg, University Medical Center Hamburg‐Eppendorf (UKE), Hamburg, Germany">
<meta name="citation_author" content="Behnam Yousefi">
<meta name="citation_author_institution" content="German Center for Child and Adolescent Health (DZKJ), Partner Site Hamburg, University Medical Center Hamburg‐Eppendorf (UKE), Hamburg, Germany">
<meta name="citation_author_institution" content="Institute of Medical Systems Bioinformatics, Center for Biomedical AI (bAIome), Center for Molecular Neurobiology (ZMNH), University Medical Center Hamburg‐Eppendorf, Hamburg, Germany">
<meta name="citation_author" content="Mathias Woidy">
<meta name="citation_author_institution" content="University Children&#x27;s Hospital, University Medical Center Hamburg‐Eppendorf (UKE), Hamburg, Germany">
<meta name="citation_author" content="Malcolm Summer Rose‐Heine">
<meta name="citation_author_institution" content="University Children&#x27;s Research, University Medical Center Hamburg‐Eppendorf (UKE), Hamburg, Germany">
<meta name="citation_author" content="Robin Khatri">
<meta name="citation_author_institution" content="Institute of Medical Systems Bioinformatics, Center for Biomedical AI (bAIome), Center for Molecular Neurobiology (ZMNH), University Medical Center Hamburg‐Eppendorf, Hamburg, Germany">
<meta name="citation_author" content="Viviane Kasten">
<meta name="citation_author_institution" content="University Children&#x27;s Research, University Medical Center Hamburg‐Eppendorf (UKE), Hamburg, Germany">
<meta name="citation_author" content="Stefan Bonn">
<meta name="citation_author_institution" content="German Center for Child and Adolescent Health (DZKJ), Partner Site Hamburg, University Medical Center Hamburg‐Eppendorf (UKE), Hamburg, Germany">
<meta name="citation_author_institution" content="Institute of Medical Systems Bioinformatics, Center for Biomedical AI (bAIome), Center for Molecular Neurobiology (ZMNH), University Medical Center Hamburg‐Eppendorf, Hamburg, Germany">
<meta name="citation_author" content="Ania Carolina Muntau">
<meta name="citation_author_institution" content="German Center for Child and Adolescent Health (DZKJ), Partner Site Hamburg, University Medical Center Hamburg‐Eppendorf (UKE), Hamburg, Germany">
<meta name="citation_author_institution" content="University Children&#x27;s Hospital, University Medical Center Hamburg‐Eppendorf (UKE), Hamburg, Germany">
<meta name="citation_author" content="Soeren Waldemar Gersting">
<meta name="citation_author_institution" content="University Children&#x27;s Research, University Medical Center Hamburg‐Eppendorf (UKE), Hamburg, Germany">
<meta name="citation_author_institution" content="German Center for Child and Adolescent Health (DZKJ), Partner Site Hamburg, University Medical Center Hamburg‐Eppendorf (UKE), Hamburg, Germany">
<meta name="citation_publication_date" content="2025 Jul 29">
<meta name="citation_volume" content="48">
<meta name="citation_issue" content="5">
<meta name="citation_firstpage" content="e70067">
<meta name="citation_doi" content="10.1002/jimd.70067">
<meta name="citation_pmid" content="40730740">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12307255/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12307255/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12307255/pdf/JIMD-48-0.pdf">
<meta name="description" content="Extensive studies have examined the clinical manifestations, pathogenic mechanisms, and genetic variations of phenylketonuria (PKU) across different populations, resulting in a substantial collection of molecular genetic data on the phenylalanine ...">
<meta name="og:title" content="Personalized Genotype‐Based Approach for Treatment of Phenylketonuria">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Extensive studies have examined the clinical manifestations, pathogenic mechanisms, and genetic variations of phenylketonuria (PKU) across different populations, resulting in a substantial collection of molecular genetic data on the phenylalanine ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12307255/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12307255">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1002/jimd.70067"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/JIMD-48-0.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12307255%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12307255/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12307255/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12307255/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-blackwellopen.png" alt="Wiley Open Access Collection logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Wiley Open Access Collection" title="Link to Wiley Open Access Collection" shape="default" href="https://doi.org/10.1002/jimd.70067" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">J Inherit Metab Dis</button></div>. 2025 Jul 29;48(5):e70067. doi: <a href="https://doi.org/10.1002/jimd.70067" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1002/jimd.70067</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22J%20Inherit%20Metab%20Dis%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22J%20Inherit%20Metab%20Dis%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22J%20Inherit%20Metab%20Dis%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22J%20Inherit%20Metab%20Dis%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Personalized Genotype‐Based Approach for Treatment of Phenylketonuria</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gundorova%20P%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Polina Gundorova</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Polina Gundorova</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>University Children's Research, University Medical Center Hamburg‐Eppendorf (UKE), Hamburg, Germany</div>
<div class="p">
<sup>
<sup>2</sup>
</sup>German Center for Child and Adolescent Health (DZKJ), Partner Site Hamburg, University Medical Center Hamburg‐Eppendorf (UKE), Hamburg, Germany</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gundorova%20P%22%5BAuthor%5D" class="usa-link"><span class="name western">Polina Gundorova</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yousefi%20B%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Behnam Yousefi</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Behnam Yousefi</span></h3>
<div class="p">
<sup>
<sup>2</sup>
</sup>German Center for Child and Adolescent Health (DZKJ), Partner Site Hamburg, University Medical Center Hamburg‐Eppendorf (UKE), Hamburg, Germany</div>
<div class="p">
<sup>
<sup>3</sup>
</sup>Institute of Medical Systems Bioinformatics, Center for Biomedical AI (bAIome), Center for Molecular Neurobiology (ZMNH), University Medical Center Hamburg‐Eppendorf, Hamburg, Germany</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yousefi%20B%22%5BAuthor%5D" class="usa-link"><span class="name western">Behnam Yousefi</span></a>
</div>
</div>
<sup>2,</sup><sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Woidy%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Mathias Woidy</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Mathias Woidy</span></h3>
<div class="p">
<sup>
<sup>4</sup>
</sup>University Children's Hospital, University Medical Center Hamburg‐Eppendorf (UKE), Hamburg, Germany</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Woidy%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Mathias Woidy</span></a>
</div>
</div>
<sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Rose%E2%80%90Heine%20MS%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Malcolm Summer Rose‐Heine</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Malcolm Summer Rose‐Heine</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>University Children's Research, University Medical Center Hamburg‐Eppendorf (UKE), Hamburg, Germany</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Rose%E2%80%90Heine%20MS%22%5BAuthor%5D" class="usa-link"><span class="name western">Malcolm Summer Rose‐Heine</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Khatri%20R%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Robin Khatri</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Robin Khatri</span></h3>
<div class="p">
<sup>
<sup>3</sup>
</sup>Institute of Medical Systems Bioinformatics, Center for Biomedical AI (bAIome), Center for Molecular Neurobiology (ZMNH), University Medical Center Hamburg‐Eppendorf, Hamburg, Germany</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Khatri%20R%22%5BAuthor%5D" class="usa-link"><span class="name western">Robin Khatri</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kasten%20V%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Viviane Kasten</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Viviane Kasten</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>University Children's Research, University Medical Center Hamburg‐Eppendorf (UKE), Hamburg, Germany</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kasten%20V%22%5BAuthor%5D" class="usa-link"><span class="name western">Viviane Kasten</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bonn%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Stefan Bonn</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Stefan Bonn</span></h3>
<div class="p">
<sup>
<sup>2</sup>
</sup>German Center for Child and Adolescent Health (DZKJ), Partner Site Hamburg, University Medical Center Hamburg‐Eppendorf (UKE), Hamburg, Germany</div>
<div class="p">
<sup>
<sup>3</sup>
</sup>Institute of Medical Systems Bioinformatics, Center for Biomedical AI (bAIome), Center for Molecular Neurobiology (ZMNH), University Medical Center Hamburg‐Eppendorf, Hamburg, Germany</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bonn%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Stefan Bonn</span></a>
</div>
</div>
<sup>2,</sup><sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Muntau%20AC%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Ania Carolina Muntau</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Ania Carolina Muntau</span></h3>
<div class="p">
<sup>
<sup>2</sup>
</sup>German Center for Child and Adolescent Health (DZKJ), Partner Site Hamburg, University Medical Center Hamburg‐Eppendorf (UKE), Hamburg, Germany</div>
<div class="p">
<sup>
<sup>4</sup>
</sup>University Children's Hospital, University Medical Center Hamburg‐Eppendorf (UKE), Hamburg, Germany</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Muntau%20AC%22%5BAuthor%5D" class="usa-link"><span class="name western">Ania Carolina Muntau</span></a>
</div>
</div>
<sup>2,</sup><sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gersting%20SW%22%5BAuthor%5D" class="usa-link" aria-describedby="id9"><span class="name western">Soeren Waldemar Gersting</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">Soeren Waldemar Gersting</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>University Children's Research, University Medical Center Hamburg‐Eppendorf (UKE), Hamburg, Germany</div>
<div class="p">
<sup>
<sup>2</sup>
</sup>German Center for Child and Adolescent Health (DZKJ), Partner Site Hamburg, University Medical Center Hamburg‐Eppendorf (UKE), Hamburg, Germany</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gersting%20SW%22%5BAuthor%5D" class="usa-link"><span class="name western">Soeren Waldemar Gersting</span></a>
</div>
</div>
<sup>1,</sup><sup>2</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="jimd70067-aff-0001">
<sup>
<sup>1</sup>
</sup>University Children's Research, University Medical Center Hamburg‐Eppendorf (UKE), Hamburg, Germany</div>
<div id="jimd70067-aff-0002">
<sup>
<sup>2</sup>
</sup>German Center for Child and Adolescent Health (DZKJ), Partner Site Hamburg, University Medical Center Hamburg‐Eppendorf (UKE), Hamburg, Germany</div>
<div id="jimd70067-aff-0003">
<sup>
<sup>3</sup>
</sup>Institute of Medical Systems Bioinformatics, Center for Biomedical AI (bAIome), Center for Molecular Neurobiology (ZMNH), University Medical Center Hamburg‐Eppendorf, Hamburg, Germany</div>
<div id="jimd70067-aff-0004">
<sup>
<sup>4</sup>
</sup>University Children's Hospital, University Medical Center Hamburg‐Eppendorf (UKE), Hamburg, Germany</div>
<div class="author-notes p">
<div class="fn" id="correspondenceTo">
<sup>*</sup><p class="display-inline">

<strong>Correspondence:</strong>
 
Polina Gundorova (<span>p.gundorova@gmail.com</span>)
</p>
</div>
<div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Revised 2025 Jun 24; Received 2025 May 2; Accepted 2025 Jul 6; Issue date 2025 Sep.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© 2025 The Author(s). <em>Journal of Inherited Metabolic Disease</em> published by John Wiley &amp; Sons Ltd on behalf of SSIEM.</div>
<p>This is an open access article under the terms of the <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by/4.0/</a> License, which permits use, distribution and reproduction in any medium, provided the original work is properly cited.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12307255  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40730740/" class="usa-link">40730740</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>ABSTRACT</h2>
<p>Extensive studies have examined the clinical manifestations, pathogenic mechanisms, and genetic variations of phenylketonuria (PKU) across different populations, resulting in a substantial collection of molecular genetic data on the phenylalanine hydroxylase (<em>PAH</em>) gene and its variants. However, many genotypes are associated with a range of clinical phenotypes, as well as variable responsiveness to sapropterin, presenting ongoing challenges for effective treatment. To address this, we enhanced the PAH activity landscapes method by incorporating high‐throughput techniques, including automated pipetting, integrated data processing via Gaussian modeling of 3D surfaces, and bioinformatics analyses with robust quality control. Using PAH activity landscapes, we visualized PAH enzymatic function across 99 common <em>PAH</em> genotypes under varying metabolic and therapeutic conditions. This deep functional phenotyping approach enabled us to identify distinct genotype subpopulations by using consensus clustering, correlate them with clinical phenotypes, and propose subpopulation‐specific treatment protocols. Our findings suggest that clinical phenotypes can be predicted and treatment regimens can be adjusted based on residual PAH function profiles. To further support personalized treatment strategies, we revised our publicly accessible <em>PAH genotype &amp; activity landscapes database</em> to share the latest insights into PAH function and patient phenotypes—namely residual enzyme activity and responsiveness to sapropterin as conveyed by two alleles. This resource underscores the translational significance of functional research in PKU and offers a practical tool to support personalized treatment in clinical settings.</p>
<section id="kwd-group1" class="kwd-group"><p><strong>Keywords:</strong> misfolding, phenylalanine hydroxylase, phenylketonuria, sapropterin dihydrochloride, tetrahydrobiopterin</p></section></section><section class="abstract" id="abstract2"><hr class="headless">
<p>Modeling the residual enzyme activity of <em>PAH</em> genotypes allows for personalized prediction of patient phenotype and BH<sub>4</sub>‐treatment response.</p>
<section class="bt xbox font-sm" id="jimd70067-blkfxd-0001"><p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/a8d4/12307255/84d737289087/JIMD-48-0-g004.jpg" loading="lazy" id="jats-graphic-1" height="492" width="394" alt="graphic file with name JIMD-48-0-g004.jpg"></p></section></section><section id="jimd70067-sec-0001"><h2 class="pmc_sec_title">1. Introduction</h2>
<p>Phenylketonuria (PKU) is a prevalent inborn error in phenylalanine (Phe) metabolism caused by a deficiency in phenylalanine hydroxylase (PAH). Despite decades of research into its clinical features, pathogenesis, genetic variability, and extensive molecular data on <em>PAH</em> variants, predicting how specific <em>PAH</em> genotypes translate into clinical outcomes and treatment responses remains challenging. To address this, we aim to study enzyme behavior in <em>PAH</em> homo‐ and compound‐heterozygous genotypes.</p>
<section id="jimd70067-sec-0002"><h3 class="pmc_sec_title">1.1. 
PKU Pathogenesis and Treatment</h3>
<p>PKU results from defects in the PAH enzyme, impairing the hydroxylation of Phe to tyrosine (Tyr) and resulting in elevated blood Phe concentrations with neurotoxic effects [<a href="#jimd70067-bib-0001" class="usa-link" aria-describedby="jimd70067-bib-0001">1</a>]. Early initiation of a Phe‐restricted diet prevents severe symptoms, but requires strict lifelong adherence. However, long‐term compliance is difficult, risking malnutrition and reducing quality of life [<a href="#jimd70067-bib-0002" class="usa-link" aria-describedby="jimd70067-bib-0002">2</a>].</p>
<p>Biallelic <em>PAH</em> variants reduce enzyme activity. Tetrahydrobiopterin (BH<sub>4</sub>), a natural PAH cofactor, was first found to lower blood Phe [<a href="#jimd70067-bib-0003" class="usa-link" aria-describedby="jimd70067-bib-0003">3</a>] and improve PAH function, leading to the definition of the BH<sub>4</sub>‐responsive PAH deficiency phenotype [<a href="#jimd70067-bib-0004" class="usa-link" aria-describedby="jimd70067-bib-0004">4</a>]. BH<sub>4</sub> acts as a chaperone, classifying PKU as a protein misfolding disorder [<a href="#jimd70067-bib-0005" class="usa-link" aria-describedby="jimd70067-bib-0005">5</a>]. <em>Sapropterin dihydrochloride</em>, a pharmacological BH<sub>4</sub> analog, offers non‐invasive treatment for responsive patients, enabling reduced dietary restrictions and, in some cases, an unrestricted diet. This therapy improves quality of life, but is only effective if residual PAH enzyme is present. Its effectiveness depends on the genotype, and even among patients with mild <em>PAH</em> variants, the response can be inconsistent [<a href="#jimd70067-bib-0006" class="usa-link" aria-describedby="jimd70067-bib-0006">6</a>].</p></section><section id="jimd70067-sec-0003"><h3 class="pmc_sec_title">1.2. Functional Studies of PAH
</h3>
<p>Functional studies are essential for translating genetic knowledge into treatments for inherited diseases. Understanding enzyme behavior in mutated protein variants enables efforts to restore activity in vitro and potentially in vivo. Development of recombinant PAH activity landscapes—3D models mapping enzyme function based on substrate Phe and cofactor BH<sub>4</sub> concentrations ([Phe] and [BH<sub>4</sub>]) [<a href="#jimd70067-bib-0007" class="usa-link" aria-describedby="jimd70067-bib-0007">7</a>]—revealed that PAH activity depends not only on genotype but also on metabolic state and BH<sub>4</sub> dosage.</p>
<p>However, a knowledge gap remains regarding enzymatic function associated with <em>PAH</em> genotypes, with limited data for single variants expressed in diverse cellular systems [<a href="#jimd70067-bib-0006" class="usa-link" aria-describedby="jimd70067-bib-0006">6</a>, <a href="#jimd70067-bib-0008" class="usa-link" aria-describedby="jimd70067-bib-0008">8</a>, <a href="#jimd70067-bib-0009" class="usa-link" aria-describedby="jimd70067-bib-0009">9</a>, <a href="#jimd70067-bib-0010" class="usa-link" aria-describedby="jimd70067-bib-0010">10</a>]. The functional PAH enzyme complex arises from transcripts from both alleles, comprising a heterotetramer assembled from two identical or nonidentical homodimers [<a href="#jimd70067-bib-0011" class="usa-link" aria-describedby="jimd70067-bib-0011">11</a>]. <em>PAH</em> patient genotypes show frequent compound heterozygosity [<a href="#jimd70067-bib-0008" class="usa-link" aria-describedby="jimd70067-bib-0008">8</a>], and the variability of the functional PAH enzyme complex, including enzymatic function, protein half‐life, and aggregation propensity, complicates the prediction of mutant PAH complex activity. In vivo analysis of PAH function using patient tissue samples is not feasible because PAH expression is limited to liver and kidney tissues. Therefore, a consistent model is required to evaluate the function of PAH enzyme expressed from two alleles, reflecting patient genotypes.</p>
<p>Bicistronic prokaryotic expression systems enable the study of PAH heterotetramers and interallelic complementation effects [<a href="#jimd70067-bib-0012" class="usa-link" aria-describedby="jimd70067-bib-0012">12</a>], but they are not suitable for investigating a wide range of genotypes, and their use is complicated by degradation of tagged recombinant proteins and expression difficulties for certain variants [<a href="#jimd70067-bib-0013" class="usa-link" aria-describedby="jimd70067-bib-0013">13</a>]. Eukaryotic systems, supported by natural chaperones, offer better stability and more accurate modeling of PAH behavior. A dual‐expression eukaryotic vector system for studying interallelic complementation [<a href="#jimd70067-bib-0014" class="usa-link" aria-describedby="jimd70067-bib-0014">14</a>] was further transferred to activity landscape format with reliable co‐expression of both alleles in a 1:1 ratio [<a href="#jimd70067-bib-0015" class="usa-link" aria-describedby="jimd70067-bib-0015">15</a>], establishing a robust model for analyzing various genotypes. This method not only enables a detailed exploration of PAH function across different [Phe] and [BH<sub>4</sub>], but also offers insights into genotype–phenotype correlations and their implications for personalized treatment strategies.</p></section><section id="jimd70067-sec-0004"><h3 class="pmc_sec_title">1.3. Personalized Approach for PKU Treatment</h3>
<p>Functional studies are crucial for determining the optimal treatment strategy in patients with PKU [<a href="#jimd70067-bib-0016" class="usa-link" aria-describedby="jimd70067-bib-0016">16</a>]. Positive BH<sub>4</sub> loading test predicts successful long‐term therapy. However, responses can vary, requiring test adjustments, such as extended duration, higher BH<sub>4</sub> dosage, or specific [Phe] [<a href="#jimd70067-bib-0017" class="usa-link" aria-describedby="jimd70067-bib-0017">17</a>], otherwise posing risks of false‐negative results. A comprehensive approach is needed to personalize testing and dietary strategies. Recent PKU research supports individualized treatment, but applying unsystematized knowledge remains challenging. This study presents a structured public resource to support personalized care, providing a workflow to analyze and identify subpopulations of patients based on their PAH residual function profile, enabling genotype‐guided therapy.</p></section></section><section id="jimd70067-sec-0005"><h2 class="pmc_sec_title">2. Materials and Methods</h2>
<section id="jimd70067-sec-0006"><h3 class="pmc_sec_title">2.1. Overexpression of PAH Protein</h3>
<p>
<em>PAH</em> gene variants were named according to reference sequences <a href="https://www.ncbi.nlm.nih.gov/nuccore/NC_000012.11" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">NC_000012.11</a>, <a href="https://www.ncbi.nlm.nih.gov/nuccore/NM_000277.3" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">NM_000277.3</a>, <a href="https://www.ncbi.nlm.nih.gov/protein/NP_000268.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">NP_000268.1</a>. A variant library (Table <a href="#jimd70067-supitem-0003" class="usa-link">S1</a>) was created using site‐directed mutagenesis (QuikChange, Stratagene, USA) and cloned into a pEF‐DEST51 vector (Invitrogen, USA) containing a Kozak sequence, modified <em>PAH</em> cDNA, and a stop codon. Missense and nonsense variants reflected predicted amino acid changes. Two splice variants were cloned based on previously reported effects: p.Gln355_Tyr356insGlyLeuGln for c.1066‐11G&gt; A [<a href="#jimd70067-bib-0018" class="usa-link" aria-describedby="jimd70067-bib-0018">18</a>] and p.Asn401Ter for c.1315 + 1G&gt; A [<a href="#jimd70067-bib-0019" class="usa-link" aria-describedby="jimd70067-bib-0019">19</a>]. For each replicate, 4 million COS‐7 cells were electroporated (Amaxa Nucleofector II, Lonza, Switzerland) with 4 μg of DNA (2 μg per allele for compound‐heterozygotes) and cultured for 26–30 h. Following trypsinization, cells were lysed (20 mM HEPES (Sigma‐Aldrich, USA), 200 mM NaCl (Chemsolute, Germany), complete mini EDTA‐free protease inhibitor (Roche, Switzerland)) through three freeze–thaw cycles.</p></section><section id="jimd70067-sec-0007"><h3 class="pmc_sec_title">2.2. 
PAH Activity Assay</h3>
<p>PAH activity was measured using modified protocols from the previous studies [<a href="#jimd70067-bib-0007" class="usa-link" aria-describedby="jimd70067-bib-0007">7</a>, <a href="#jimd70067-bib-0015" class="usa-link" aria-describedby="jimd70067-bib-0015">15</a>] in a total assay volume of 100 μL. A reaction buffer, consisting of 22.35 mM NaHEPES (Sigma‐Aldrich, USA) (pH 7.3), 1 U/μl catalase (Sigma‐Aldrich, USA), and 10 μM ferrous ammonium sulfate (Sigma‐Aldrich, USA), was combined with 10 μL cell lysate and L‐Phenylalanine. After a 5 min pre‐incubation, the reaction was initiated by adding BH<sub>4</sub> (Cayman Chemical Company, USA) solution and halted after 15 min with 2 μL of 9 N HCl (Sigma‐Aldrich, USA). Tyr production was quantified by high‐performance liquid chromatography on a Hypersil ODS‐2 column (Dionex UltiMate 3000, Thermo Fisher Scientific, USA) with a running buffer (pH 4.6) containing 1.57% ammonium hydroxide (Sigma‐Aldrich, USA) and 2% acetic acid (AppliChem, Germany) at 2 mL/min.</p></section><section id="jimd70067-sec-0008"><h3 class="pmc_sec_title">2.3. 
PAH Activity Landscapes</h3>
<p>Activity landscapes were generated by assaying each PAH genotype in biological triplicates using a 96‐well format. Automated pipetting (FreedomEvo150, Tecan, Switzerland) enabled parallel processing of twenty‐four 96‐well plate layouts combined into six 384‐well plates, allowing eight genotypes to be analyzed per run. Every run incorporated a positive wild‐type PAH (WT) and a negative no‐DNA control. Variant activity was normalized to WT and background‐subtracted using no‐DNA values. Implemented assay updates allowed for reduction of batch‐to‐batch variability and improvement of the results reproducibility.</p>
<p>[Phe] ranged from 0 to 2500 μM across plate columns and [BH<sub>4</sub>] from 0 to 250 μM across rows, providing higher resolution near physiological concentrations. Protein content in lysates was measured by Bradford assay (Bio‐Rad, USA; CLARIOstar, BMG LABTECH, Germany). PAH activity was calculated as pmol/mg/min of Tyr, and median values from triplicates were used to generate each genotype's activity landscape.</p></section><section id="jimd70067-sec-0009"><h3 class="pmc_sec_title">2.4. Data Analysis and Identification of Subpopulations</h3>
<p>Each data sample was represented by a 12 × 8 matrix, with rows and columns corresponding to [Phe] and [BH<sub>4</sub>], respectively (Figure <a href="#jimd70067-fig-0001" class="usa-link">1A</a>). Visualization was performed using 2D/3D plots (Figure <a href="#jimd70067-fig-0001" class="usa-link">1B</a>) via regular grid interpolation and Gaussian filter smoothing (<em>SciPy</em> 1.12). A color code with power‐law normalization enhanced the resolution of low residual activity peaks (5%–20%).</p>
<figure class="fig xbox font-sm" id="jimd70067-fig-0001"><h4 class="obj_head">FIGURE 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/a8d4/12307255/0abd29160d79/JIMD-48-0-g003.jpg" loading="lazy" id="jats-graphic-3" height="387" width="709" alt="FIGURE 1"></p>
<div class="p text-right font-secondary"><a href="figure/jimd70067-fig-0001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Process of data evaluation for the PAH activity landscape of human PAH WT. Data representing PAH enzyme activity at varying [Phe] and [BH<sub>4</sub>] were visualized using a color code (with the enzyme activity scale provided at the bottom) to depict activity as a percentage of the activity of the wild type. The sequence of graphs from left to right illustrates the data‐evaluation steps. The pink “X” on the 2D graphs indicates the peak activity position representing maximum activity. The solid pink line delineates 50% of enzyme activity, signifying its optimal working range. (A) Plotting the raw data (left) obtained from the PAH activity assay in a 96‐well format and triangular surface mesh (right) creates a non‐smoothed and non‐modeled 3D surface. (B) Data interpolation with a regular grid‐based algorithm, followed by smoothing via a Gaussian filter, allows for the creation of smoothed non‐modeled 3D (left) and 2D (right) panels. (C) The logarithmic transformation of the data creates a symmetrical surface (left) that can be fitted into a Gaussian model (right). (D) The exponential transformation of the data creates smoothed modeled 3D (left) and 2D (right) panels.</p></figcaption></figure><section id="jimd70067-sec-0010"><h4 class="pmc_sec_title">2.4.1. Gaussian Modelling</h4>
<p>We implemented a Gaussian modeling‐based method to reduce noise and improve the quality. We fitted a 2D Gaussian function to the landscapes in log‐scaled [Phe] and [BH<sub>4</sub>] space (Figure <a href="#jimd70067-fig-0001" class="usa-link">1C</a>) and scaled it back to the original space (Figure <a href="#jimd70067-fig-0001" class="usa-link">1D</a>). Considering <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-1" display="inline" overflow="linebreak"><mrow><mrow><mi>x</mi></mrow></mrow></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-2" display="inline" overflow="linebreak"><mrow><mrow><mi>y</mi></mrow></mrow></math></span> as [Phe] and [BH<sub>4</sub>], we modeled the residual activities (<em>z</em>) as:</p>
<table class="disp-formula p" id="jimd70067-disp-0001"><tr><td class="formula"><math id="jats-math-3" display="block" overflow="linebreak"><mrow><mrow><msup><mi>x</mi><mo>′</mo></msup><mo linebreak="goodbreak">=</mo><mi>ln</mi><mfenced close=")" open="("><mi>x</mi></mfenced></mrow></mrow></math></td></tr></table>
<table class="disp-formula p" id="jimd70067-disp-0002"><tr><td class="formula"><math id="jats-math-4" display="block" overflow="linebreak"><mrow><mrow><msup><mi>y</mi><mo>′</mo></msup><mo linebreak="goodbreak">=</mo><mi>ln</mi><mfenced close=")" open="("><mi>y</mi></mfenced></mrow></mrow></math></td></tr></table>
<table class="disp-formula p" id="jimd70067-disp-0003"><tr><td class="formula"><math id="jats-math-5" display="block" overflow="linebreak"><mrow><mrow><mover accent="true"><mi>z</mi><mo stretchy="true">^</mo></mover><mo linebreak="goodbreak">=</mo><mi>f</mi><mfenced close=")" open="(" separators=","><msup><mi>x</mi><mo>′</mo></msup><msup><mi>y</mi><mo>′</mo></msup></mfenced><mo linebreak="goodbreak">=</mo><mi>a</mi><mo>·</mo><mi>exp</mi><mfenced close=")" open="("><mrow><mo linebreak="goodbreak">−</mo><mn>0.5</mn><mspace width="0.25em"></mspace></mrow></mfenced><mo>·</mo><mfenced close=")" open="("><mrow><mi>x</mi><mo linebreak="goodbreak">−</mo><msub><mi>m</mi><mrow><mi>x</mi><mo>′</mo></mrow></msub><mo>/</mo><msup><msub><mi>s</mi><mrow><mi>x</mi><mo>′</mo></mrow></msub><mn>2</mn></msup></mrow></mfenced><mo linebreak="goodbreak">+</mo><mfenced close=")" open="("><mrow><msup><mi>y</mi><mo>′</mo></msup><mo linebreak="goodbreak">−</mo><msub><mi>m</mi><mrow><mi>y</mi><mo>′</mo></mrow></msub><mo>/</mo><msup><msub><mi>s</mi><mrow><mi>y</mi><mo>′</mo></mrow></msub><mn>2</mn></msup></mrow></mfenced></mrow></mrow></math></td></tr></table>
<p>where <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-6" display="inline" overflow="linebreak"><mrow><mrow><mi>a</mi></mrow></mrow></math></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-7" display="inline" overflow="linebreak"><mrow><mrow><msub><mi>m</mi><mrow><mi>x</mi><mo>′</mo></mrow></msub></mrow></mrow></math></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-8" display="inline" overflow="linebreak"><mrow><mrow><msub><mi>m</mi><mrow><mi>y</mi><mo>′</mo></mrow></msub></mrow></mrow></math></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-9" display="inline" overflow="linebreak"><mrow><mrow><msub><mi>s</mi><mrow><mi>x</mi><mo>′</mo></mrow></msub></mrow></mrow></math></span>, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-10" display="inline" overflow="linebreak"><mrow><mrow><msub><mi>s</mi><mrow><mi>y</mi><mo>′</mo></mrow></msub></mrow></mrow></math></span> are hyperparameters fitted using curve optimization (<em>SciPy</em> 1.12). From the fitted model, we extracted maximum residual activity (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-11" display="inline" overflow="linebreak"><mrow><mrow><mi>a</mi></mrow></mrow></math></span>), [Phe] and [BH<sub>4</sub>] at the peak (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-12" display="inline" overflow="linebreak"><mrow><mrow><mi>exp</mi><mfenced close=")" open="("><msub><mi>m</mi><mrow><mi>x</mi><mo>′</mo></mrow></msub></mfenced></mrow></mrow></math></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-13" display="inline" overflow="linebreak"><mrow><mrow><mi>exp</mi><mfenced close=")" open="("><msub><mi>m</mi><mrow><mi>y</mi><mo>′</mo></mrow></msub></mfenced></mrow></mrow></math></span>), and then normalized the activity to the WT (%WT). Peaks occurring above 1500 μM [Phe] and lower than 3% were likely artifacts. Such landscapes were considered as having no residual activity (subpopulation 0) and were excluded from the downstream analysis.</p></section><section id="jimd70067-sec-0011"><h4 class="pmc_sec_title">2.4.2. Quality Control</h4>
<p>QC analysis was performed using three features: (i) the root mean squared error (RMSE) between the original and modeled landscapes, (ii) the number of peaks (NoP) in raw landscapes, and (iii) the variation between the triplicates (VT) as the maximum ratio between the range and the median value across wells. Thresholds were defined using exponential probability distribution functions (PDFs) with parameter <em>λ</em>, as the mean added to 2/<em>λ</em> (0.26, 8.4, and 0.23 for RMSE, NoP, and VT, respectively). Samples exceeding any of the thresholds were classified as low‐quality and excluded, with borderline cases reviewed manually.</p></section><section id="jimd70067-sec-0012"><h4 class="pmc_sec_title">2.4.3. Consensus Clustering</h4>
<p>High‐quality samples with detectable residual activity were clustered to identify genotype subpopulations. To address variability and bias in unsupervised clustering, we employed consensus clustering (R package <em>ConsensusClustering</em>) [<a href="#jimd70067-bib-0020" class="usa-link" aria-describedby="jimd70067-bib-0020">20</a>, <a href="#jimd70067-bib-0021" class="usa-link" aria-describedby="jimd70067-bib-0021">21</a>, <a href="#jimd70067-bib-0022" class="usa-link" aria-describedby="jimd70067-bib-0022">22</a>], which is robust to noise and allows estimation of optimal cluster number.</p>
<p>For future sample classification, we use <em>k</em>‐nearest neighbor labeling to avoid re‐clustering the entire dataset.</p></section></section><section id="jimd70067-sec-0013"><h3 class="pmc_sec_title">2.5. Genotype–Phenotype Correlations</h3>
<p>The BioPKU database, a comprehensive resource for PAH genotypes and associated clinical data [<a href="#jimd70067-bib-0006" class="usa-link" aria-describedby="jimd70067-bib-0006">6</a>], was used to compare the in situ modeled PAH behavior with clinical phenotypes and BH<sub>4</sub>‐responsiveness. For each of 99 tested <em>PAH</em> genotypes, data was retrieved on:</p>
<p>
<em>Clinical phenotype classification</em>: classical phenylketonuria [Phe] &gt; 1200 μmol/L, mild PKU [Phe] 600–1200 μmol/L, or MHPA [Phe] &lt; 600 μmol/L; a total of 7854 patient records.</p>
<p>
<em>BH</em>
<sub>
<em>4</em>
</sub>
<em>‐responsiveness</em>: responder, slow‐responder (combined as “responders”), non‐responder; a total of 1426 patient records.</p>
<p>Genotypes with &gt; 10 patient records were included. Genotypes with variants p.Ile65Thr and p.Phe39Leu were excluded due to recent changes in the BioPKU annotations. The final dataset comprised 7756 records for phenotype correlation and 1152 for BH<sub>4</sub>‐responsiveness (Table <a href="#jimd70067-supitem-0004" class="usa-link">S2</a>).</p></section></section><section id="jimd70067-sec-0014"><h2 class="pmc_sec_title">3. Results</h2>
<p>The <em>PAH</em> genotype dataset was created based on the selection of 42 pathogenic <em>PAH</em> variants, including all 16 of the most common variants with allele frequencies of &gt; 1% and 26 rare variants. The selected variants accounted for 61.1% of all <em>PAH</em> pathogenic alleles [<a href="#jimd70067-bib-0023" class="usa-link" aria-describedby="jimd70067-bib-0023">23</a>] (Table <a href="#jimd70067-supitem-0003" class="usa-link">S1</a>). For the current study, we selected 99 <em>PAH</em> patient genotypes, of which 19 were homozygous and 80 were compound heterozygous. The summarized frequency of the 99 pathogenic <em>PAH</em> genotypes was 50.4% of all PKU patients registered in the <em>PAH genotype &amp; activity landscape database</em> [<a href="#jimd70067-bib-0023" class="usa-link" aria-describedby="jimd70067-bib-0023">23</a>] (Table <a href="#jimd70067-supitem-0004" class="usa-link">S2</a>).</p>
<p>Building on the PAH activity landscape model [<a href="#jimd70067-bib-0015" class="usa-link" aria-describedby="jimd70067-bib-0015">15</a>], we further enhanced this approach by incorporating automated pipetting, integrated data processing, and bioinformatics analyses, elevating it to a high‐throughput platform. This advancement not only quadruples data output with heightened accuracy, but also conserves resources such as chemicals, consumables, and labor. Each activity landscape was produced as a result of the integration of activity assay data in a 96‐well format with gradients of [Phe] and [BH<sub>4</sub>] from three biological replicates (Figure <a href="#jimd70067-fig-0001" class="usa-link">1A</a>). Figure <a href="#jimd70067-fig-0001" class="usa-link">1B</a> shows 2D and 3D graphical representations of the activity landscapes based on regular grid interpolation and smoothing. To account for experimental noise and improve the quality of activity landscapes, we implemented a Gaussian modeling‐based method (Figure <a href="#jimd70067-fig-0001" class="usa-link">1C</a>). This approach allowed us to model the true 3D surfaces and estimate the peak parameters more precisely by considering the behavior of the entire surface (Figure <a href="#jimd70067-fig-0001" class="usa-link">1B</a> vs. Figure <a href="#jimd70067-fig-0001" class="usa-link">1D</a>). Furthermore, to ensure sample quality, we applied quality control (QC) analysis to assess the overall data quality, potential variations across experimental replicates, and the differences between observed and modeled activity landscapes.</p>
<p>Modeling genotype‐specific profiles of residual enzyme activity in a cellular environment allows the visualization of PAH behavior, which is usually inaccessible because its expression is limited to liver tissue. In PAH activity landscapes, enzyme activity is represented by a color code. Residual activity demonstrated an initial increase with the elevation of substrate concentrations and a subsequent reduction of activity towards higher substrate concentrations—<em>substrate inhibition</em> kinetics. The abovementioned behavior was observed for both Phe and BH<sub>4</sub> and created a characteristic drop shape of a PAH activity landscape (Figure <a href="#jimd70067-fig-0001" class="usa-link">1D</a>). An increase in activity within the elevation of [BH<sub>4</sub>] in the range of 0–100 μM provides the so‐called BH<sub>4</sub>‐response, observed in patients with PKU. In practice, this shows how supplementing a BH<sub>4</sub>‐responsive patient with additional BH<sub>4</sub> would increase residual PAH function, which would subsequently lower blood [Phe]. While the position of the peak of residual enzyme activity and the prominence of substrate inhibition can vary in different <em>PAH</em> genotypes, the visual presentation of BH<sub>4</sub>‐responsiveness in activity landscapes is defined by the presence of a residual activity peak with the activity increasing in the physiological [BH<sub>4</sub>] range towards the higher [BH<sub>4</sub>].</p>
<p>The PAH activity landscape for the WT demonstrated the peak of activity that was, on average, located at [Phe] = 330 ± 15 μM, [BH<sub>4</sub>] = 101 ± 17 μM, and the working range spanned from [Phe] 80 ± 5 μM to 1372 ± 107 μM (Figure <a href="#jimd70067-fig-0001" class="usa-link">1D</a>). Among the produced PAH activity landscapes for the 99 <em>PAH</em> genotypes, 18 did not show any peak of residual activity, and 81 genotypes demonstrated various residual peaks (Table <a href="#jimd70067-supitem-0004" class="usa-link">S2</a>). The presence of a peak of residual activity in the PAH activity landscape indicated a potential response to the BH<sub>4</sub>‐treatment. However, among patients with <em>PAH</em> genotypes demonstrating residual PAH activity, 38.9% were reported in the BioPKU database as BH<sub>4</sub>‐non‐responders (Table <a href="#jimd70067-supitem-0004" class="usa-link">S2</a>). This indicated that the standard BH<sub>4</sub>‐loading test protocol may not be efficient for different <em>PAH</em> genotypes. Identifying similarities among PAH activity landscapes for various genotypes and correlating them with clinical data allows a better understanding of the mechanism of BH<sub>4</sub>‐response in PAH‐deficiencies and adjustment of existing testing and treatment schemes accordingly. Therefore, we extracted the peak parameters of activity landscapes and applied clustering to extract typical patterns of PAH activity landscapes—subpopulations of PAH activity landscapes.</p>
<section id="jimd70067-sec-0015"><h3 class="pmc_sec_title">3.1. Subpopulations of PAH Activity Landscapes</h3>
<p>First, we recognized the subpopulation that represented genotypes with no residual activity as subpopulation 0, which included 18 of the 99 <em>PAH</em> genotypes. These samples did not pass the QC and were not included in the cluster analysis because of the absence of a residual peak. For each of the 81 activity landscapes with a residual peak that passed the QC, we calculated: (i) the maximum residual activity normalized to the WT (%WT), (ii) the peak of activity coordinates in terms of [Phe] and [BH<sub>4</sub>] (μM), and (iii) the enzyme working range of 50% of the maximum activity ([Phe] μM) (Table <a href="#jimd70067-supitem-0004" class="usa-link">S2</a>). These features were used to identify and characterize subpopulations of <em>PAH</em> genotypes representing similar behaviors. Using cluster analysis, we identified five clusters (Figure <a href="#jimd70067-fig-0002" class="usa-link">2A,B</a>), each representing a corresponding subpopulation (1–5) of activity landscapes with various residual activity peaks. Furthermore, all subpopulations were associated with the clinical phenotype and potential BH<sub>4</sub> response.</p>
<figure class="fig xbox font-sm" id="jimd70067-fig-0002"><h4 class="obj_head">FIGURE 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/a8d4/12307255/a6b7afc6c616/JIMD-48-0-g002.jpg" loading="lazy" id="jats-graphic-5" height="784" width="709" alt="FIGURE 2"></p>
<div class="p text-right font-secondary"><a href="figure/jimd70067-fig-0002/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Subpopulations of PAH activity landscapes. Characteristics that define subpopulations and their clinical relevance. Box plots display the median, range from minimum to maximum, and the 25th to 75th percentiles. (A) Visual distribution of studied PAH genotypes in PAH activity landscapes based on the parameters of residual peak activity in <em>x</em>‐, <em>y</em>‐, and <em>z</em>‐side views. For better visualization, the <em>z</em>‐axis was log10 transformed. Subpopulation 0 is not shown because of the absence of a residual activity peak. Five defined subpopulations are depicted using a color code. Subpopulation 2 is located in the area of lower × (left‐shifted) and subpopulations 3 and 4 are in the area of higher × (right‐shifted). Subpopulations 3 and 4 differ in <em>z</em>‐coordinates. Subpopulation 5 groups within the highest <em>z</em>. (B) Visual distribution of studied PAH genotypes in PAH activity landscapes based on the parameters of residual peak activity <em>x</em>‐, <em>y</em>‐, and <em>z</em>‐top views. Subpopulation 0 is not shown because of the absence of a residual activity peak. Five defined subpopulations are depicted using a color code. Subpopulation 2 is located in the area of lower × (left‐shifted) and subpopulations 3 and 4 are in the area of higher × (right‐shifted). Subpopulations 3 and 4 differ by <em>y</em>‐coordinate. (C) [Phe] at peak residual activity in various subpopulations. The peak [Phe] for the PAH WT is indicated by a dashed line. Subpopulation 1 demonstrates peaks in a position similar to the WT, subpopulation 2 is characterized by left‐shifted peaks (lower [Phe]), and subpopulations 3 and 4 by right‐shifted peaks (higher [Phe]). The peaks of residual activity for genotypes in subpopulation 5 were located in the area of normal or left‐shifted peaks. (D) Maximum residual activity at the peak in various subpopulations. The two subpopulations with right‐shifted peaks, 3 and 4, differed in their maximum residual activities, with subpopulation 3 demonstrating higher values. Subpopulation 5 exhibited the highest residual peaks among all the genotypes. (E) BH<sub>4</sub>‐treatment response in subpopulations (clinical data sourced from BioPKU [<a href="#jimd70067-bib-0009" class="usa-link" aria-describedby="jimd70067-bib-0009">9</a>]). Subpopulation 0 was associated with 100% non‐responders. Genotypes from subpopulation 1 were associated with variable response rates owing to significantly reduced residual activity. In subpopulations 2 and 5, response rates close to 100% were observed owing to left‐shifted peaks and high residual peaks, respectively. Subpopulation 3 response rates were more variable owing to the right‐shifted peaks. In comparison, in subpopulation 4, right‐shifted peaks and significantly reduced residual activity led to even lower detection of BH<sub>4</sub> response. (F) Clinical phenotypes in subpopulations (clinical data sourced from BioPKU [<a href="#jimd70067-bib-0009" class="usa-link" aria-describedby="jimd70067-bib-0009">9</a>]). Phenotypes presented as classical PKU with blood [Phe] &gt; 1200 μmol/L, mild PKU with blood [Phe] ranging 600–1200 μmol/L, or MHPA with blood [Phe] &lt; 600 μmol/L. Subpopulation 0 was associated with 100% classical PKU. Subpopulations 1, 3, and 4 demonstrated a mixture of possible phenotypes, with subpopulation 4 having the greatest chance of classical PKU. Subpopulations 2 and 5 represent patients with milder phenotypes, whereas subpopulation 5 does not include any patients with classical PKU.</p></figcaption></figure><p>Analysis of the clustering results revealed that variation across genotypes was mainly based on two characteristics: [Phe] at peak activity (Figure <a href="#jimd70067-fig-0002" class="usa-link">2C</a>) and maximum residual activity (Figure <a href="#jimd70067-fig-0002" class="usa-link">2D</a>). According to the features defining each subpopulation, they can be characterized as follows:
</p>
<ul id="jimd70067-list-0001" class="list" style="list-style-type:none">
<li id="jimd70067-li-0001">
<span class="label">0</span><p class="display-inline">No residual activity,</p>
</li>
<li id="jimd70067-li-0002">
<span class="label">1</span><p class="display-inline">Reduced activity,</p>
</li>
<li id="jimd70067-li-0003">
<span class="label">2</span><p class="display-inline">Left‐shifted peak of residual activity,</p>
</li>
<li id="jimd70067-li-0004">
<span class="label">3</span><p class="display-inline">Right‐shifted peak, higher residual activity,</p>
</li>
<li id="jimd70067-li-0005">
<span class="label">4</span><p class="display-inline">Right‐shifted peak, lower residual activity,</p>
</li>
<li id="jimd70067-li-0006">
<span class="label">5</span><p class="display-inline">High residual activity.</p>
</li>
</ul>
<p>Examples of PAH activity landscapes for each subpopulation are shown on Figure <a href="#jimd70067-fig-0003" class="usa-link">3</a>; the full dataset is available on the web‐based platform at <a href="http://pah-activitylandscapes.org/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://pah‐activitylandscapes.org/</a> [<a href="#jimd70067-bib-0023" class="usa-link" aria-describedby="jimd70067-bib-0023">23</a>].</p>
<figure class="fig xbox font-sm" id="jimd70067-fig-0003"><h4 class="obj_head">FIGURE 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/a8d4/12307255/f83d12aefa58/JIMD-48-0-g001.jpg" loading="lazy" id="jats-graphic-7" height="647" width="709" alt="FIGURE 3"></p>
<div class="p text-right font-secondary"><a href="figure/jimd70067-fig-0003/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>The PAH activity landscapes of genotypes corresponding to subpopulations 0–5. One example is shown for every defined subpopulation of PAH activity landscapes, and 2D and 3D panels illustrate the residual PAH activity of the corresponding genotypes normalized to the WT maximum activity. (A) Subpopulation 0: PAH activity landscapes of genotypes exhibiting no residual PAH activity; example genotype p.[Arg408Trp];[Arg408Trp]. The genotypes of this group showed no residual enzyme activity, as the designated PAH peak activity area spanning up to 1500 μM Phe displayed no discernible peaks. A heightened Tyr concentration was observed towards higher [Phe] and [BH<sub>4</sub>], which resulted from the natural conversion of Phe into Tyr. However, this artifact does not indicate any residual PAH activity, lies outside the metabolic space existing in human liver cells, and can only be modeled in laboratory settings. The absent peak of residual activity in subpopulation 0 directly correlates with severe clinical presentation and no response to the BH<sub>4</sub>‐treatment. (B) Subpopulation 1: PAH activity landscapes of genotypes with a standard peak position and reduced PAH activity, example genotype p.[Ala104Asp];[Ser349Pro]. Typically, this subpopulation displays a PAH residual activity peak that is significantly reduced in comparison with the PAH WT but remains in a similar shape and position. Some activity landscapes of this subpopulation demonstrated discernible peaks of residual activity that were not sufficiently prominent. Genotypes of subpopulation 1 are mostly associated with MHPA or mild PKU phenotypes and a positive response to the BH<sub>4</sub>‐treatment. (C) Subpopulation 2: PAH activity landscapes exhibiting left‐shifted peaks of residual PAH activity, example genotype p.[Ile306Val];[Ile306Val]. This subpopulation was defined by left‐shifted, narrowly peaked profiles. Notably, the working range of the enzyme predominantly falls within lower Phe concentrations. Genotypes that belong to subpopulation 2 typically lead to mHPA and a positive response to the BH<sub>4</sub>‐treatment. (D) Subpopulation 3: PAH activity landscapes with right‐shifted higher peaks, example genotype p.[Arg297His]; [Arg408Trp]. The PAH activity landscapes for genotypes representing this subpopulation were associated with wide right‐shifted peaks. Despite the relatively high residual peaks, a right shift often leads to undetectable residual activity below 200 μM Phe. Genotypes of subpopulation 3 could lead to more severe clinical presentations; however, the response to the BH<sub>4</sub>‐treatment is mostly directly detectable, due to the high residual activity peaks. (E) Subpopulation 4: PAH activity landscapes with right‐shifted lower peaks, example genotype p.[Arg261Gln];[Arg261Gln]. The PAH activity landscapes for genotypes representing this subpopulation were associated with low, wide, right‐shifted residual activity peaks. The rightward shift of both the peak and working range resulted in undetectable enzyme activity below 200 μM Phe. Genotypes grouped to subpopulation 4 are characterized by more frequent severe clinical presentation and a challenging BH<sub>4</sub>‐loading test. (F) Subpopulation 5: PAH activity landscapes with high residual activity, example genotype p.[Glu390Gly];[Glu390Gly]. The subpopulation includes activity landscapes of various shapes and peak locations; however, a common characteristic of this subpopulation is high residual activity. In the case of subpopulation 5, very high residual activity peaks allow for mild phenotypes and BH<sub>4</sub>‐response.</p></figcaption></figure><p>Subpopulation 0, which showed no measurable residual activity, was excluded from the clustering analysis. Subpopulation 1 demonstrated residual activity peaks similar in position and shape to WT. In contrast, subpopulation 2 showed left‐shifted peaks, with maximum activity occurring at lower [Phe]. Subpopulations 3 and 4 were right‐shifted, with activity peaks occurring at higher [Phe] compared to WT. While subpopulation 3 had moderate residual activity, subpopulation 4 showed lower peak heights. Subpopulation 5 was characterized by high residual activity compared to other subpopulations, but with greater variability in the position of the peak along the [Phe] axis.</p>
<p>Clearly, different pathogenic variants of <em>PAH</em> tend to alter the enzyme activity profile in several ways. In addition to the absence of residual activity in subpopulation 0, some variants only reduced the activity to different extents, but others could also change the shape of the profiles, shifting the peak of residual activity to areas of higher or lower [Phe], as well as compressing or enlarging the working range of the enzyme. Left‐shifted peaks from subpopulation 2 (Figure <a href="#jimd70067-fig-0003" class="usa-link">3C</a>) were additionally characterized by a narrow working range, whereas right‐shifted peaks in subpopulations 3 and 4 (Figure <a href="#jimd70067-fig-0003" class="usa-link">3D,E</a>) were associated with widened working ranges. The reduction of enzyme activity at high substrate concentrations and <em>substrate inhibition</em> is known to occur in PAH [<a href="#jimd70067-bib-0024" class="usa-link" aria-describedby="jimd70067-bib-0024">24</a>, <a href="#jimd70067-bib-0025" class="usa-link" aria-describedby="jimd70067-bib-0025">25</a>] and is observed in the majority of PAH activity landscapes. The altered position and broadness of the activity peak are provided by an enhanced or reduced substrate inhibition effect, which could be explained by changes in the surroundings and accessibility of substrate binding sites [<a href="#jimd70067-bib-0026" class="usa-link" aria-describedby="jimd70067-bib-0026">26</a>]. Within this hypothesis, left‐shifted narrow peaks could occur when the substrate‐binding site is more accessible, which provides for higher enzyme activity with low substrate concentrations (left‐shifted peaks), but additional binding of the substrate to the enzyme‐product complexes enhances the substrate inhibition effect (narrow activity peaks). In contrast, right‐shifted broadened peaks could occur when the substrate‐binding site is less accessible, which provides for low enzyme activity at low [Phe] (right‐shifted activity peaks), but lower binding of the substrate to the enzyme‐product complex reduces the substrate inhibition effect (broad activity peaks). Further investigation of the mechanisms by which particular pathogenic variants enhance or reduce the substrate inhibition effect would require crystal structure studies on those mutants, which, however, could be complicated by the absence of data on the Phe‐activated PAH structure [<a href="#jimd70067-bib-0027" class="usa-link" aria-describedby="jimd70067-bib-0027">27</a>]. Other possible strategies would involve modeling of the variant protein 3D structures <em>in silico</em>.</p>
<p>To investigate the clinical relevance of various changes in the shapes of activity landscapes, we compared the identified subpopulation with the data on phenotype and registered BH<sub>4</sub>‐response from the BioPKU database (Figure <a href="#jimd70067-fig-0002" class="usa-link">2E,F</a>). The reference clinical cohort resulted in 7756 patients for phenotype correlation and 1152 patients for BH<sub>4</sub>‐treatment correlation (Table <a href="#jimd70067-supitem-0004" class="usa-link">S2</a>). Based on the characteristics of the corresponding subpopulation, we can determine the prognosis of the patient's phenotype and BH<sub>4</sub>‐treatment for genotypes lacking sufficient clinical evidence in the BioPKU database. Consulting PAH activity landscapes to provide patients with blood Phe levels to match the PAH working range for particular genotypes is beneficial for optimizing BH<sub>4</sub>‐testing and treatment protocols.</p>
<section id="jimd70067-sec-0016"><h4 class="pmc_sec_title">3.1.1. Genotypes With No Residual Activity: Subpopulation 0</h4>
<p>This subpopulation, represented by 18 genotypes with homozygous or compound‐heterozygous variants, consistently exhibited no residual PAH activity (Figure <a href="#jimd70067-fig-0002" class="usa-link">2D</a>, Figure <a href="#jimd70067-fig-0003" class="usa-link">3A</a>). Notably, all possible pairwise combinations—both homozygous and compound heterozygous—of the variants p.Arg111Ter, p.Arg158Gln, p.Arg243Ter, p.Arg252Trp, p.Arg261Ter, p.Glu280Lys, p.Pro281Leu, p.Ser349Pro, c.1066‐11G&gt;A (p.Gln355_Tyr356insGlyLeuGln, IVS10‐11G&gt;A), p.Arg408Trp, and c.1315+1G&gt;A (p.Asn401Ter, IVS12+1G&gt;A) were predicted to have no residual PAH activity. Consequently, they showed no response to BH<sub>4</sub> in all tested patients (Figure <a href="#jimd70067-fig-0002" class="usa-link">2E</a>) and were clinically represented as classical PKU (Figure <a href="#jimd70067-fig-0002" class="usa-link">2F</a>).</p></section><section id="jimd70067-sec-0017"><h4 class="pmc_sec_title">3.1.2. Activity Landscapes With Reduced Residual Activity: Subpopulation 1</h4>
<p>The largest subpopulation contained 24 genotypes that displayed a peak with location and working ranges resembling those of the WT PAH (Figure <a href="#jimd70067-fig-0002" class="usa-link">2C</a>, Figure <a href="#jimd70067-fig-0003" class="usa-link">3B</a>). However, in all these instances, activity is markedly diminished in comparison with WT, generally falling within the interval of 4%–11%. The peak of residual activity was located in the range of 220–360 μM Phe with a median of 330 μM Phe, which corresponds to a WT peak with [Phe] = 352 μM. Subpopulation 1 is formed mainly by genotypes comprising mild variants, such as p.Leu48Ser, p.Ala104Asp, p.Ala300Ser, p.Val388Met, or p.Tyr414Cys [<a href="#jimd70067-bib-0028" class="usa-link" aria-describedby="jimd70067-bib-0028">28</a>, <a href="#jimd70067-bib-0029" class="usa-link" aria-describedby="jimd70067-bib-0029">29</a>, <a href="#jimd70067-bib-0030" class="usa-link" aria-describedby="jimd70067-bib-0030">30</a>], which provide low residual PAH activity. Clinical data from the BioPKU database reflect the presence of residual activity, as 80% of patients are listed as BH<sub>4</sub>‐responders or slow‐responders. However, in this subpopulation, BH<sub>4</sub>‐response varies from 24% to 100%, and this inconsistency indicates difficulties in detecting the BH<sub>4</sub>‐response for some of the genotypes. Possible reasons for the inconsistent response rates could be a low residual activity provided by a certain variant, for example, p.Leu48Ser with 8% in the homozygous genotype. A similar behavior was observed in compound heterozygous genotypes that included null variants or severe missense variants, such as p.Arg408Trp, and a variant with residual activity. As p.Arg408Trp tends to promote higher aggregation [<a href="#jimd70067-bib-0031" class="usa-link" aria-describedby="jimd70067-bib-0031">31</a>] it induces a negative interallelic complementation effect, further reducing the already impaired residual activity provided by the second variant. For patients in this group, an extended test, such as a 4‐week test, or an increased BH<sub>4</sub> dosage might be considered to detect the medication response in case of the first negative BH<sub>4</sub>‐loading test.</p></section><section id="jimd70067-sec-0018"><h4 class="pmc_sec_title">3.1.3. Left‐Shifted Peaks of Residual PAH Activity: Subpopulation 2</h4>
<p>A distinguished subpopulation consisting of 14 genotypes represented specific shifts towards lower [Phe] profiles of PAH activity landscapes (Figure <a href="#jimd70067-fig-0002" class="usa-link">2C</a>, Figure <a href="#jimd70067-fig-0003" class="usa-link">3C</a>) with a median of [Phe] at the peak of 105 μM. Additionally, the enzyme working ranges for these variants are typically confined to a narrow area, whereas the residual activity is in a relatively low range of 4%–9%. Such a profile is commonly observed in genotypes with variants p.Ala403Val and p.Ile306Val. Despite low residual activity, narrow left‐shifted profiles were associated with a 100% BH<sub>4</sub>‐response and mild clinical presentation (35% mild PKU and 52% MHPA) (Figure <a href="#jimd70067-fig-0002" class="usa-link">2E,F</a>). A possible mechanism of mild phenotypes within subpopulation 2 would be that the enzyme is active in the low range of [Phe] and effectively recycles the substrate, which does not allow for [Phe] to elevate.</p>
<p>Additionally, the peak of activity resided within a relatively low [BH<sub>4</sub>] of 55–72 μM (Figure <a href="#jimd70067-supitem-0002" class="usa-link">S1</a>). First, physiological [BH<sub>4</sub>] is beneficial for patients, allowing for residual PAH activity and providing mild phenotypes. Second, this peak positioning accounts for the high rates of BH<sub>4</sub>‐responders, suggesting that the standard BH<sub>4</sub>‐loading test procedure using a 20 mg/kg dosage would effectively benefit patients without yielding false‐negative results.</p></section><section id="jimd70067-sec-0019"><h4 class="pmc_sec_title">3.1.4. Right‐Shifted Higher Peaks of Residual PAH Activity: Subpopulation 3</h4>
<p>A subpopulation of 16 genotypes displayed PAH activity landscapes, with the peak shifted to higher [Phe] (median [Phe] = 547 μM) and a broad working range (Figures <a href="#jimd70067-fig-0002" class="usa-link">2C</a> and <a href="#jimd70067-fig-0003" class="usa-link">3D</a>). The presence of right‐shifted peaks is typically indicative of severe phenotypes and challenges in detecting BH<sub>4</sub>‐response. However, in subpopulation 3, this effect was partially mitigated by the higher residual activity of 6%–16% WT, which was greater than that in subpopulations 1 and 2 (Figure <a href="#jimd70067-fig-0002" class="usa-link">2D</a>). Despite the residual activity being higher than that in subpopulation 2, the phenotypes and BH<sub>4</sub>–response reflect a more severe clinical presentation. Phenotype distribution was heterogeneous and similar to that in subpopulation 1, with 29% classical PKU, 35% mild PKU, and 53% MHPA. An 80% of patients are reported as BH<sub>4</sub>‐responders, which reflects the possibility of failing the BH<sub>4</sub>‐loading test even with a high PAH residual activity. Right‐shifted peaks could lead to an insufficient BH<sub>4</sub>‐response while being tested with low blood [Phe], for example, when the patient is already receiving dietary treatment.</p>
<p>Certain genotypes, such as compounds with p.Glu390Gly and p.Asp415Asn, display relatively high residual activity and are therefore observed in milder phenotypes and linked to a potential response to the BH<sub>4</sub>‐treatment. Several genotypes in this subpopulation were compound heterozygous for the variant p.Ile65Thr, which has a history of contradictory reports in the literature and in the BioPKU database (earlier patients reported as slow‐responders, but later changed to non‐responders), and therefore were excluded from the analysis as unreliable data. However, PAH activity landscapes of homozygous and compound heterozygous genotypes with p.Ile65Thr demonstrated prominent right‐shifted peaks of residual activity, indicating a possible BH<sub>4</sub> response. These patients, as the others with genotypes from subpopulation 3, could benefit from BH<sub>4</sub> administration when having a higher Phe load to reach the residual activity that arises with higher [Phe].</p></section><section id="jimd70067-sec-0020"><h4 class="pmc_sec_title">3.1.5. Right‐Shifted Lower Peaks of Residual PAH Activity: Subpopulation 4</h4>
<p>This subpopulation of 19 genotypes displayed a typical picture of the PAH activity landscape when the peak was shifted towards a higher [Phe] (median [Phe] = 476 μM), broadened working range, and low residual activity from 3% to 6% (Figures <a href="#jimd70067-fig-0002" class="usa-link">2C,D</a> and <a href="#jimd70067-fig-0003" class="usa-link">3E</a>). PAH enzyme activity was not evident in the low Phe concentration range. Furthermore, these peaks often lie within higher BH<sub>4</sub> concentrations (Figure <a href="#jimd70067-supitem-0002" class="usa-link">S1</a>). Mentioned characteristics are reflected by the largest share of classical PKU of 29% and the lowest BH<sub>4</sub>‐response rate of 27% among all subpopulations of potential BH<sub>4</sub>‐responders. Severe phenotypes are often reported; since the enzyme is not active at low [Phe], it cannot maintain low Phe levels. The high share of reported BH<sub>4</sub>‐non‐responders suggests that the standard BH<sub>4</sub>‐loading test procedure might not be appropriate for patients with these genotypes. It is likely that tests are often conducted when patients are already on a Phe‐restricted diet, which results in lower blood Phe levels. Consequently, the residual activity did not increase when supplemented with pharmacological concentrations of BH<sub>4</sub>, producing numerous false‐negative results in the BH<sub>4</sub>‐loading test. For such patients, it is advisable to start the BH<sub>4</sub>‐loading test at higher Phe levels to detect residual PAH activity. This recommendation can be integrated into the BH<sub>4</sub>‐treatment process. Since the peaks of activity tend to be located within the higher [BH<sub>4</sub>], increasing the dosage of BH<sub>4</sub> might be beneficial for pikely that tests are often conducted when patients are already on a Phe‐restricted diet, which results in lower blood Phe levels. Consequently, the residual activity did not increase when supplemented with pharmacological concentrations of BH<sub>4</sub>, producing numerous false‐negative results in the BH<sub>4</sub>‐loading test. For such patients, it is advisable to start the BH<sub>4</sub>‐loading test at higher Phe levels to detect residual PAH activity. This recommendation can be integrated into the BH<sub>4</sub>‐treatment process. Since the peaks of activity tend to be located within the higher [BH<sub>4</sub>], increasing the dosage of BH<sub>4</sub> might be beneficial for patients with genotypes of subpopulation 4.</p>
<p>The typical variant found in the genotypes of subpopulation 4 is p.Arg261Gln, a common pathogenic variant with an allele frequency of 9.3%, which is known to be challenging in terms of BH<sub>4</sub>‐therapy. Homozygous and compound‐heterozygous genotypes with p.Arg261Gln are often reported as classical PKU, and the BH<sub>4</sub>‐loading test does not always detect treatment response. To avoid false‐negative BH<sub>4</sub>‐loading test results, it is advisable to increase the Phe load at the beginning of the test and to use a higher dosage of the medication.</p></section><section id="jimd70067-sec-0021"><h4 class="pmc_sec_title">3.1.6. Genotypes With High Residual PAH Activity: Subpopulation 5</h4>
<p>Subpopulation 5 includes PAH activity landscapes of 8 genotypes that are heterogeneous by the shape and position of the peak, but all have high residual PAH activity ranging from 23% to 32% (Figures <a href="#jimd70067-fig-0002" class="usa-link">2D</a> and Figure <a href="#jimd70067-fig-0003" class="usa-link">3F</a>). This group includes landscapes with left‐, right‐, and non‐shifted peaks, but when reaching a certain residual activity, they all seem to provide conditions for mild phenotypes (20% mild PKU, 80% MHPA) and are in 100% of cases associated with BH<sub>4</sub>‐response. High residual activity guarantees a response to BH<sub>4</sub> therapy or provides a mild phenotype of MHPA that does not require treatment. Notably, further research is necessary to predict whether a patient with a mild genotype will require treatment; therefore, monitoring Phe levels is recommended according to treatment guidelines.</p>
<p>Variants providing exceptionally high residual activity were often p.Ala403Val, p.Glu390Gly, and p.Asp415Asn. Homozygous and some compound‐heterozygous genotypes with the mentioned variants contribute to the subpopulation 5.</p></section></section><section id="jimd70067-sec-0022"><h3 class="pmc_sec_title">3.2. 

<em>PAH</em>
 Genotype &amp; Activity Landscapes Database</h3>
<p>One of the objectives of the study was to enhance and structure the <em>PAH genotype &amp; activity landscape database</em> [<a href="#jimd70067-bib-0023" class="usa-link" aria-describedby="jimd70067-bib-0023">23</a>], resulting in a dedicated web application for analyzing the global distribution of <em>PAH</em> genotypes and accessing enzymatic functional data. With over 11 450 genotypes included, the database is regularly being updated with newly published population and in‐house experimental data. Researchers and clinicians can also submit their unpublished genotype data from PKU patient cohorts directly to the platform.</p>
<p>This application offers tools for visualizing and analyzing sequence variant distributions, exploring patient genotypes, and identifying targets for functional studies. The database has been comprehensively reorganized to include search and analysis features and integrate new functional results. It supports both research and clinical use, enabling the interpretation of <em>PAH</em> variants and access to functional data. Clinicians can directly access laboratory‐derived PAH activity landscapes, gaining insights from activity features. Each PAH activity landscape now includes a standardized verbal interpretation based on subpopulation analysis, providing clinicians with genotype‐specific guidance. Additionally, the platform fosters collaboration between researchers and healthcare providers, with the goal of improving care for patients with PAH deficiency. The initial collection of 99 PAH activity landscapes will be expanded to include frequent population‐specific genotypes and those requested by clinicians via the online interface.</p></section></section><section id="jimd70067-sec-0023"><h2 class="pmc_sec_title">4. Discussion</h2>
<p>Genotype–phenotype correlations and BH<sub>4</sub>‐responsiveness reports in the literature and the BioPKU database are often inconsistent, making it challenging to achieve robust and predictable treatment outcomes for patients carrying the same <em>PAH</em> genotype [<a href="#jimd70067-bib-0006" class="usa-link" aria-describedby="jimd70067-bib-0006">6</a>, <a href="#jimd70067-bib-0029" class="usa-link" aria-describedby="jimd70067-bib-0029">29</a>, <a href="#jimd70067-bib-0030" class="usa-link" aria-describedby="jimd70067-bib-0030">30</a>, <a href="#jimd70067-bib-0032" class="usa-link" aria-describedby="jimd70067-bib-0032">32</a>, <a href="#jimd70067-bib-0033" class="usa-link" aria-describedby="jimd70067-bib-0033">33</a>, <a href="#jimd70067-bib-0034" class="usa-link" aria-describedby="jimd70067-bib-0034">34</a>]. The primary objective of this study was to provide a translational personalized approach to treatment, aiming to reduce false‐negative BH<sub>4</sub>‐test outcomes and support the implementation of more effective BH<sub>4</sub>‐therapy strategies.</p>
<section id="jimd70067-sec-0024"><h3 class="pmc_sec_title">4.1. Strategies for Clinical Application of PAH Activity Landscapes</h3>
<p>By applying a clustering algorithm to the PAH activity landscapes with measurable residual activity, we identified patterns that offer new insights into the phenotypic variability observed in PKU. These findings have potential utility in refining diagnostic workflows and individualizing treatment strategies. While PKU care is generally guided by international and national guidelines, clinical practice often varies—particularly regarding the timing of genetic diagnostics. Below, we outline two practical scenarios in which BH<sub>4</sub>‐treatment strategies can benefit from integrating PAH activity landscapes.</p>
<section id="jimd70067-sec-0025"><h4 class="pmc_sec_title">4.1.1. Genetic Results Not Available at Time of BH<sub>4</sub>
‐Testing</h4>
<p>In many healthcare settings, newborns begin treatment before their <em>PAH</em> genotype is known. Since genetic testing can take time, clinicians often use this initial treatment phase to assess BH<sub>4</sub>‐responsiveness. By the time genetic results are received, dietary and BH<sub>4</sub>‐treatments may already be in place. Given the inconsistencies in BH<sub>4</sub>‐responsiveness reported in the BioPKU database, it is beneficial to reassess potential false‐negative BH<sub>4</sub>‐test results using functional data.</p>
<p>When initial BH<sub>4</sub>‐loading tests indicate non‐responsiveness, PAH activity landscapes can provide an additional layer of decision‐making support:</p>
<section id="jimd70067-sec-0026"><h5 class="pmc_sec_title">4.1.1.1. Confirming Non‐Responsiveness</h5>
<p>If the genotype falls into subpopulation 0 (no residual activity), BH<sub>4</sub>‐treatments are unlikely to be beneficial, and alternative therapies should be considered.</p></section><section id="jimd70067-sec-0027"><h5 class="pmc_sec_title">4.1.1.2. Reevaluating BH<sub>4</sub>
 Test Conditions</h5>
<p>If the genotype belongs to subpopulations 1–5, a positive BH<sub>4</sub>‐response may be achievable under modified testing conditions. Depending on the location of the activity peak within the landscape, adjusting the patient's pre‐test Phe levels (within clinically safe limits), extending test duration, or combining both may improve responsiveness detection. Additionally, using the full approved BH<sub>4</sub> dose of 20 mg/kg/day—especially in cases where testing is typically initiated at lower doses—can be beneficial. These modifications may increase the likelihood of identifying partial responders.</p>
<p>These recommendations are not limited to neonates or newly diagnosed patients. Older patients who were classified as non‐responders based on earlier BH<sub>4</sub>‐loading tests may also benefit from retrospective genotype‐based evaluation. If their genotype falls into subpopulations 1–5, a re‐test under optimized conditions may be warranted to ensure that treatment decisions reflect the most accurate functional data available.</p>
<p>In short, discrepancies between the BH<sub>4</sub>‐test results and genotype‐specific functional data should prompt a careful reexamination of test conditions.</p></section></section><section id="jimd70067-sec-0028"><h4 class="pmc_sec_title">4.1.2. Genotype Available Before BH<sub>4</sub>
‐Testing</h4>
<p>An alternative scenario arises when BH<sub>4</sub> becomes newly available in a clinical center, prompting loading tests in a broader patient cohort. In this case, the availability of PAH genotype data prior to testing allows for more strategic patient selection and testing design:</p>
<section id="jimd70067-sec-0029"><h5 class="pmc_sec_title">4.1.2.1. Exclusion of Non‐Responsive Genotypes</h5>
<p>Patients with genotypes previously identified as non‐responsive (subpopulation 0) can be excluded from testing, reducing unnecessary procedures.</p></section><section id="jimd70067-sec-0030"><h5 class="pmc_sec_title">4.1.2.2. Adjustment of Pre‐Test Phe Levels</h5>
<p>For genotypes with right‐shifted activity peaks (subpopulations 3 and 4), higher initial blood Phe may be necessary to detect a BH<sub>4</sub>‐response during testing.</p></section><section id="jimd70067-sec-0031"><h5 class="pmc_sec_title">4.1.2.3. Test Adaptation for Low‐Activity Genotypes</h5>
<p>Patients with significantly reduced PAH activity (subpopulations 1 and 4) may benefit from extended loading tests and/or higher doses if standard protocols do not yield positive responses.</p>
<p>As PKU treatment becomes increasingly individualized—particularly in the optimization of combined dietary and pharmacological therapy—PAH activity landscapes offer a valuable visualization tools. These landscapes can help clinicians, dietitians, and patients better understand genotype‐specific PAH functionality, ultimately supporting more effective personalized treatment strategies.</p></section></section></section><section id="jimd70067-sec-0032"><h3 class="pmc_sec_title">4.2. Limitations of the Study and Further Research Perspectives</h3>
<p>Due to the high incidence of PKU and widespread newborn screening, a massive number of <em>PAH</em> variants and genotypes have been identified. While this study analyzed 99 genotypes—covering over a half of reported PKU cases—it represents only a fraction of the 2302 known <em>PAH</em> genotypes [<a href="#jimd70067-bib-0023" class="usa-link" aria-describedby="jimd70067-bib-0023">23</a>]. Functional testing of all variants and genotypes is impractical due to resource and cost limitations. However, the aim here was to explore correlations between in situ enzyme activity and clinical outcomes.</p>
<p>Further expansion of the PAH activity landscape dataset is planned, focusing on the most frequent genotypes across populations through scientific collaborations. Additionally, personalized activity landscapes could be created upon request to support clinical decision‐making.</p>
<p>However, the long‐term value lies in translating these experimental datasets into predictive <em>in silico</em> models. This study lays the groundwork for such tools.</p>
<p>The activity landscapes approach may also be applied to other enzymes involved in inherited metabolic disorders. Visualizing residual enzyme activity and detecting patterns in enzyme behavior can deepen understanding of disease mechanisms and aid in personalizing treatment strategies. With continued data accumulation, this method could enhance both therapeutic development and computational modeling.</p></section></section><section id="jimd70067-sec-0033"><h2 class="pmc_sec_title">5. Conclusions</h2>
<p>PKU has been studied extensively for over nine decades, resulting in a detailed understanding of its molecular and clinical aspects. This wealth of data calls for effective tools to analyze and apply findings for patient benefit. While updating guidelines and implementing new strategies may take time, timely access to the latest research is essential to improve patient care and quality of life. A translational approach—bridging research and clinical practice—can be advanced through web‐based platforms that publish functional studies in real time, enabling treatment adjustments. Genetic diagnostics, increasingly implemented in PKU management, should not only confirm the diagnosis but also guide personalized therapy. This approach can serve as a model for individualized care in other rare diseases.</p></section><section id="jimd70067-sec-0034"><h2 class="pmc_sec_title">Author Contributions</h2>
<p>
<strong>Polina Gundorova:</strong> writing ‑ original draft, conceptualization, data curation, funding acquisition, investigation, methodology, project administration, validation, visualization. <strong>Behnam Yousefi:</strong> writing ‑ original draft, data curation, methodology, software, validation, visualization. <strong>Mathias Woidy:</strong> writing ‑ review and editing, methodology, software. <strong>Malcolm Summer Rose‐Heine:</strong> investigation. <strong>Robin Khatri:</strong> software. <strong>Viviane Kasten:</strong> investigation. <strong>Stefan Bonn:</strong> supervision, funding acquisition, writing ‑ review and editing. <strong>Ania Carolina Muntau:</strong> supervision. <strong>Soeren Waldemar Gersting:</strong> resources, supervision, funding acquisition, writing ‑ review and editing.</p></section><section id="jimd70067-sec-0036"><h2 class="pmc_sec_title">Conflicts of Interest</h2>
<p>All authors have read the journal's policy on disclosure of potential conflicts of interest. A.C.M. and S.W.G. are shareholders of Inuiva GmbH. A.C.M. has received consulting and speaker fees from APR, BioMarin, and PTC Therapeutics. The authors declare no conflicts of interest.</p></section><section id="sec36"><h2 class="pmc_sec_title">Supporting information</h2>
<section class="sm xbox font-sm" id="jimd70067-supitem-0001"><div class="caption p"><p>
<strong>Data S1:</strong> Supporting Information.</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12307255/bin/JIMD-48-0-s001.docx" data-ga-action="click_feat_suppl" class="usa-link">JIMD-48-0-s001.docx</a><sup> (14.8KB, docx) </sup>
</div></div></section><section class="sm xbox font-sm" id="jimd70067-supitem-0002"><div class="caption p"><p>
<strong>Figure S1:</strong> [BH<sub>4</sub>] at peak residual activity across subpopulations.</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12307255/bin/JIMD-48-0-s004.pdf" data-ga-action="click_feat_suppl" class="usa-link">JIMD-48-0-s004.pdf</a><sup> (1.3MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="jimd70067-supitem-0003"><div class="caption p"><p>
<strong>Table S1:</strong> jimd70067‐sup‐0003‐TableS1.xlsx. <em>PAH</em> gene variants.</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12307255/bin/JIMD-48-0-s003.xlsx" data-ga-action="click_feat_suppl" class="usa-link">JIMD-48-0-s003.xlsx</a><sup> (13.6KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="jimd70067-supitem-0004"><div class="caption p"><p>
<strong>Table S2:</strong> jimd70067‐sup‐0004‐TableS2.xlsx. <em>PAH</em> gene genotypes.</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12307255/bin/JIMD-48-0-s002.xlsx" data-ga-action="click_feat_suppl" class="usa-link">JIMD-48-0-s002.xlsx</a><sup> (24.2KB, xlsx) </sup>
</div></div></section></section><section id="jimd70067-sec-0035" class="ack"><h2 class="pmc_sec_title">Acknowledgments</h2>
<p>The authors express their gratitude to the UCR team for their everyday support and advice as well as to the Kinder‐UKE metabolic team for their careful work with patients and efforts to support our research. The authors would like to thank Andrea Turcati and Italo Balestra (OmegaLambdaTec) for developing the initial version of a Python script for visualizing the PAH activity landscapes. Open Access funding enabled and organized by Projekt DEAL.</p></section><section id="notes1"><section id="jimd70067-ntgp-0001" class="fn-group"><div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="jimd70067-note-0001"><p>
<strong>Funding:</strong> This work was supported by German Academic Exchange Service, Alexander von Humboldt‐Stiftung, BioMarin Pharmaceutical, Deutsche Forschungsgemeinschaft, FOR 5068 P9, Hamburg Macht Kinder Gesund foundation, Federal Ministry of Education and Research (BMBF), German Center for Child and Adolescent Health (DZKJ), 01GL2404A.</p></div></div></section></section><section id="jimd70067-sec-0038"><h2 class="pmc_sec_title">Data Availability Statement</h2>
<p>The data that support the findings of this study are openly available in PAH genotypes &amp; activity landscapes database at <a href="http://pah-activitylandscapes.org/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://pah‐activitylandscapes.org/</a>. The data evaluation pipeline is available at the GitHub repository <a href="https://github.com/imsb-uke/Phenylketonuria" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/imsb‐uke/Phenylketonuria</a>.</p></section><section id="jimd70067-bibl-0001" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="jimd70067-bibl-0001_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="jimd70067-bib-0001">
<span class="label">1.</span><cite>
Blau N., Shen N., and Carducci C., “Molecular Genetics and Diagnosis of Phenylketonuria: State of the Art,” Expert Review of Molecular Diagnostics
14, no. 6 (2014): 655–671.
</cite> [<a href="https://doi.org/10.1586/14737159.2014.923760" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24882081/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Expert%20Review%20of%20Molecular%20Diagnostics&amp;title=Molecular%20Genetics%20and%20Diagnosis%20of%20Phenylketonuria:%20State%20of%20the%20Art&amp;author=N.%20Blau&amp;author=N.%20Shen&amp;author=C.%20Carducci&amp;volume=14&amp;issue=6&amp;publication_year=2014&amp;pages=655-671&amp;pmid=24882081&amp;doi=10.1586/14737159.2014.923760&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jimd70067-bib-0002">
<span class="label">2.</span><cite>
Tomm A., Thiele A. G., Rohde C., Kirmse S., Kiess W., and Beblo S., “Executive Functions &amp; Metabolic Control in Phenylketonuria (Pku) and Mild Hyperphenylalaninemia (mHPA),” Molecular Genetics and Metabolism
143, no. 1–2 (2024): 108544.
</cite> [<a href="https://doi.org/10.1016/j.ymgme.2024.108544" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39059271/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Genetics%20and%20Metabolism&amp;title=Executive%20Functions%20&amp;%20Metabolic%20Control%20in%20Phenylketonuria%20(Pku)%20and%20Mild%20Hyperphenylalaninemia%20(mHPA)&amp;author=A.%20Tomm&amp;author=A.%20G.%20Thiele&amp;author=C.%20Rohde&amp;author=S.%20Kirmse&amp;author=W.%20Kiess&amp;volume=143&amp;issue=1%E2%80%932&amp;publication_year=2024&amp;pages=108544&amp;pmid=39059271&amp;doi=10.1016/j.ymgme.2024.108544&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jimd70067-bib-0003">
<span class="label">3.</span><cite>
Kure S., Hou D. C., Ohura T., et al., “Tetrahydrobiopterin‐Responsive Phenylalanine Hydroxylase Deficiency,” Journal of Pediatrics
135, no. 3 (1999): 375–378.
</cite> [<a href="https://doi.org/10.1016/s0022-3476(99)70138-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10484807/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Pediatrics&amp;title=Tetrahydrobiopterin%E2%80%90Responsive%20Phenylalanine%20Hydroxylase%20Deficiency&amp;author=S.%20Kure&amp;author=D.%20C.%20Hou&amp;author=T.%20Ohura&amp;volume=135&amp;issue=3&amp;publication_year=1999&amp;pages=375-378&amp;pmid=10484807&amp;doi=10.1016/s0022-3476(99)70138-1&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jimd70067-bib-0004">
<span class="label">4.</span><cite>
Muntau A. C., Röschinger W., Habich M., et al., “Tetrahydrobiopterin as an Alternative Treatment for Mild Phenylketonuria,” New England Journal of Medicine
347, no. 26 (2002): 2122–2132.
</cite> [<a href="https://doi.org/10.1056/NEJMoa021654" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12501224/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=New%20England%20Journal%20of%20Medicine&amp;title=Tetrahydrobiopterin%20as%20an%20Alternative%20Treatment%20for%20Mild%20Phenylketonuria&amp;author=A.%20C.%20Muntau&amp;author=W.%20R%C3%B6schinger&amp;author=M.%20Habich&amp;volume=347&amp;issue=26&amp;publication_year=2002&amp;pages=2122-2132&amp;pmid=12501224&amp;doi=10.1056/NEJMoa021654&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jimd70067-bib-0005">
<span class="label">5.</span><cite>
Opladen T., Abu Seda B., Rassi A., Thöny B., Hoffmann G. F., and Blau N., “Diagnosis of Tetrahydrobiopterin Deficiency Using Filter Paper Blood Spots: Further Development of the Method and 5 Years Experience,” Journal of Inherited Metabolic Disease
34, no. 3 (2011): 819–826.
</cite> [<a href="https://doi.org/10.1007/s10545-011-9300-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21416196/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Inherited%20Metabolic%20Disease&amp;title=Diagnosis%20of%20Tetrahydrobiopterin%20Deficiency%20Using%20Filter%20Paper%20Blood%20Spots:%20Further%20Development%20of%20the%20Method%20and%205%E2%80%89Years%20Experience&amp;author=T.%20Opladen&amp;author=B.%20Abu%20Seda&amp;author=A.%20Rassi&amp;author=B.%20Th%C3%B6ny&amp;author=G.%20F.%20Hoffmann&amp;volume=34&amp;issue=3&amp;publication_year=2011&amp;pages=819-826&amp;pmid=21416196&amp;doi=10.1007/s10545-011-9300-1&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jimd70067-bib-0006">
<span class="label">6.</span><cite>
Blau N., Yue W., and Perez B., “BIOPKU Database,” 2006‑2025, <a href="http://www.biopku.org/biopku/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://www.biopku.org/biopku/</a>.</cite>
</li>
<li id="jimd70067-bib-0007">
<span class="label">7.</span><cite>
Staudigl M., Gersting S. W., Danecka M. K., et al., “The Interplay Between Genotype, Metabolic State and Cofactor Treatment Governs Phenylalanine Hydroxylase Function and Drug Response,” Human Molecular Genetics
20, no. 13 (2011): 2628–2641.
</cite> [<a href="https://doi.org/10.1093/hmg/ddr165" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21527427/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Human%20Molecular%20Genetics&amp;title=The%20Interplay%20Between%20Genotype,%20Metabolic%20State%20and%20Cofactor%20Treatment%20Governs%20Phenylalanine%20Hydroxylase%20Function%20and%20Drug%20Response&amp;author=M.%20Staudigl&amp;author=S.%20W.%20Gersting&amp;author=M.%20K.%20Danecka&amp;volume=20&amp;issue=13&amp;publication_year=2011&amp;pages=2628-2641&amp;pmid=21527427&amp;doi=10.1093/hmg/ddr165&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jimd70067-bib-0008">
<span class="label">8.</span><cite>
Hillert A., Anikster Y., Belanger‐Quintana A., et al., “The Genetic Landscape and Epidemiology of Phenylketonuria,” American Journal of Human Genetics
107, no. 2 (2020): 234–250.
</cite> [<a href="https://doi.org/10.1016/j.ajhg.2020.06.006" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7413859/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32668217/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=American%20Journal%20of%20Human%20Genetics&amp;title=The%20Genetic%20Landscape%20and%20Epidemiology%20of%20Phenylketonuria&amp;author=A.%20Hillert&amp;author=Y.%20Anikster&amp;author=A.%20Belanger%E2%80%90Quintana&amp;volume=107&amp;issue=2&amp;publication_year=2020&amp;pages=234-250&amp;pmid=32668217&amp;doi=10.1016/j.ajhg.2020.06.006&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jimd70067-bib-0009">
<span class="label">9.</span><cite>
Himmelreich N., Shen N., Okun J. G., Thiel C., Hoffmann G. F., and Blau N., “Relationship Between Genotype, Phenylalanine Hydroxylase Expression and In Vitro Activity and Metabolic Phenotype in Phenylketonuria,” Molecular Genetics and Metabolism
125, no. 1–2 (2018): 86–95.
</cite> [<a href="https://doi.org/10.1016/j.ymgme.2018.06.011" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30037505/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Genetics%20and%20Metabolism&amp;title=Relationship%20Between%20Genotype,%20Phenylalanine%20Hydroxylase%20Expression%20and%20In%C2%A0Vitro%20Activity%20and%20Metabolic%20Phenotype%20in%20Phenylketonuria&amp;author=N.%20Himmelreich&amp;author=N.%20Shen&amp;author=J.%20G.%20Okun&amp;author=C.%20Thiel&amp;author=G.%20F.%20Hoffmann&amp;volume=125&amp;issue=1%E2%80%932&amp;publication_year=2018&amp;pages=86-95&amp;pmid=30037505&amp;doi=10.1016/j.ymgme.2018.06.011&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jimd70067-bib-0010">
<span class="label">10.</span><cite>
Zhang X., Ye J., Shen N., et al., “In Vitro Residual Activities in 20 Variants of Phenylalanine Hydroxylase and Genotype‐Phenotype Correlation in Phenylketonuria Patients,” Gene
707 (2019): 239–245.
</cite> [<a href="https://doi.org/10.1016/j.gene.2019.05.029" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31102715/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Gene&amp;title=In%20Vitro%20Residual%20Activities%20in%2020%20Variants%20of%20Phenylalanine%20Hydroxylase%20and%20Genotype%E2%80%90Phenotype%20Correlation%20in%20Phenylketonuria%20Patients&amp;author=X.%20Zhang&amp;author=J.%20Ye&amp;author=N.%20Shen&amp;volume=707&amp;publication_year=2019&amp;pages=239-245&amp;pmid=31102715&amp;doi=10.1016/j.gene.2019.05.029&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jimd70067-bib-0011">
<span class="label">11.</span><cite>
Leandro J., Leandro P., and Flatmark T., “Heterotetrameric Forms of Human Phenylalanine Hydroxylase: Co‐Expression of Wild‐Type and Mutant Forms in a Bicistronic System,” Biochimica et Biophysica Acta
1812, no. 5 (2011): 602–612.
</cite> [<a href="https://doi.org/10.1016/j.bbadis.2011.02.001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21315150/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biochimica%20et%20Biophysica%20Acta&amp;title=Heterotetrameric%20Forms%20of%20Human%20Phenylalanine%20Hydroxylase:%20Co%E2%80%90Expression%20of%20Wild%E2%80%90Type%20and%20Mutant%20Forms%20in%20a%20Bicistronic%20System&amp;author=J.%20Leandro&amp;author=P.%20Leandro&amp;author=T.%20Flatmark&amp;volume=1812&amp;issue=5&amp;publication_year=2011&amp;pages=602-612&amp;pmid=21315150&amp;doi=10.1016/j.bbadis.2011.02.001&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jimd70067-bib-0012">
<span class="label">12.</span><cite>
Leandro J., Simonsen N., Saraste J., Leandro P., and Flatmark T., “Phenylketonuria as a Protein Misfolding Disease: The Mutation pG46S in Phenylalanine Hydroxylase Promotes Self‐Association and Fibril Formation,” Biochimica et Biophysica Acta
1812, no. 1 (2011): 106–120.
</cite> [<a href="https://doi.org/10.1016/j.bbadis.2010.09.015" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20937381/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biochimica%20et%20Biophysica%20Acta&amp;title=Phenylketonuria%20as%20a%20Protein%20Misfolding%20Disease:%20The%20Mutation%20pG46S%20in%20Phenylalanine%20Hydroxylase%20Promotes%20Self%E2%80%90Association%20and%20Fibril%20Formation&amp;author=J.%20Leandro&amp;author=N.%20Simonsen&amp;author=J.%20Saraste&amp;author=P.%20Leandro&amp;author=T.%20Flatmark&amp;volume=1812&amp;issue=1&amp;publication_year=2011&amp;pages=106-120&amp;pmid=20937381&amp;doi=10.1016/j.bbadis.2010.09.015&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jimd70067-bib-0013">
<span class="label">13.</span><cite>
Pey A. L., Desviat L. R., Gámez A., Ugarte M., and Pérez B., “Phenylketonuria: Genotype‐Phenotype Correlations Based on Expression Analysis of Structural and Functional Mutations in PAH,” Human Mutation
21, no. 4 (2003): 370–378.
</cite> [<a href="https://doi.org/10.1002/humu.10198" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12655546/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Human%20Mutation&amp;title=Phenylketonuria:%20Genotype%E2%80%90Phenotype%20Correlations%20Based%20on%20Expression%20Analysis%20of%20Structural%20and%20Functional%20Mutations%20in%20PAH&amp;author=A.%20L.%20Pey&amp;author=L.%20R.%20Desviat&amp;author=A.%20G%C3%A1mez&amp;author=M.%20Ugarte&amp;author=B.%20P%C3%A9rez&amp;volume=21&amp;issue=4&amp;publication_year=2003&amp;pages=370-378&amp;pmid=12655546&amp;doi=10.1002/humu.10198&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jimd70067-bib-0014">
<span class="label">14.</span><cite>
Shen N., Heintz C., Thiel C., Okun J. G., Hoffmann G. F., and Blau N., “Co‐Expression of Phenylalanine Hydroxylase Variants and Effects of Interallelic Complementation on In Vitro Enzyme Activity and Genotype‐Phenotype Correlation,” Molecular Genetics and Metabolism
117 (2016): 328–335.
</cite> [<a href="https://doi.org/10.1016/j.ymgme.2016.01.004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26803807/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Genetics%20and%20Metabolism&amp;title=Co%E2%80%90Expression%20of%20Phenylalanine%20Hydroxylase%20Variants%20and%20Effects%20of%20Interallelic%20Complementation%20on%20In%C2%A0Vitro%20Enzyme%20Activity%20and%20Genotype%E2%80%90Phenotype%20Correlation&amp;author=N.%20Shen&amp;author=C.%20Heintz&amp;author=C.%20Thiel&amp;author=J.%20G.%20Okun&amp;author=G.%20F.%20Hoffmann&amp;volume=117&amp;publication_year=2016&amp;pages=328-335&amp;pmid=26803807&amp;doi=10.1016/j.ymgme.2016.01.004&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jimd70067-bib-0015">
<span class="label">15.</span><cite>
Danecka M. K., Woidy M., Zschocke J., Feillet F., Muntau A. C., and Gersting S. W., “Mapping the Functional Landscape of Frequent Phenylalanine Hydroxylase (PAH) Genotypes Promotes Personalised Medicine in Phenylketonuria,” Journal of Medical Genetics
52, no. 3 (2015): 175–185.
</cite> [<a href="https://doi.org/10.1136/jmedgenet-2014-102621" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25596310/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Medical%20Genetics&amp;title=Mapping%20the%20Functional%20Landscape%20of%20Frequent%20Phenylalanine%20Hydroxylase%20(PAH)%20Genotypes%20Promotes%20Personalised%20Medicine%20in%20Phenylketonuria&amp;author=M.%20K.%20Danecka&amp;author=M.%20Woidy&amp;author=J.%20Zschocke&amp;author=F.%20Feillet&amp;author=A.%20C.%20Muntau&amp;volume=52&amp;issue=3&amp;publication_year=2015&amp;pages=175-185&amp;pmid=25596310&amp;doi=10.1136/jmedgenet-2014-102621&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jimd70067-bib-0016">
<span class="label">16.</span><cite>
Opladen T., López‐Laso E., Cortès‐Saladelafont E., et al., “Consensus Guideline for the Diagnosis and Treatment of Tetrahydrobiopterin (BH(4)) Deficiencies,” Orphanet Journal of Rare Diseases
15, no. 1 (2020): 126.
</cite> [<a href="https://doi.org/10.1186/s13023-020-01379-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7251883/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32456656/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Orphanet%20Journal%20of%20Rare%20Diseases&amp;title=Consensus%20Guideline%20for%20the%20Diagnosis%20and%20Treatment%20of%20Tetrahydrobiopterin%20(BH(4))%20Deficiencies&amp;author=T.%20Opladen&amp;author=E.%20L%C3%B3pez%E2%80%90Laso&amp;author=E.%20Cort%C3%A8s%E2%80%90Saladelafont&amp;volume=15&amp;issue=1&amp;publication_year=2020&amp;pages=126&amp;pmid=32456656&amp;doi=10.1186/s13023-020-01379-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jimd70067-bib-0017">
<span class="label">17.</span><cite>
Muntau A. C., Adams D. J., Bélanger‐Quintana A., et al., “International Best Practice for the Evaluation of Responsiveness to Sapropterin Dihydrochloride in Patients With Phenylketonuria,” Molecular Genetics and Metabolism
127, no. 1 (2019): 1–11.
</cite> [<a href="https://doi.org/10.1016/j.ymgme.2019.04.004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31103398/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Genetics%20and%20Metabolism&amp;title=International%20Best%20Practice%20for%20the%20Evaluation%20of%20Responsiveness%20to%20Sapropterin%20Dihydrochloride%20in%20Patients%20With%20Phenylketonuria&amp;author=A.%20C.%20Muntau&amp;author=D.%20J.%20Adams&amp;author=A.%20B%C3%A9langer%E2%80%90Quintana&amp;volume=127&amp;issue=1&amp;publication_year=2019&amp;pages=1-11&amp;pmid=31103398&amp;doi=10.1016/j.ymgme.2019.04.004&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jimd70067-bib-0018">
<span class="label">18.</span><cite>
Dworniczak B., Aulehla‐Scholz C., Kalaydjieva L., Bartholomé K., Grudda K., and Horst J., “Aberrant Splicing of Phenylalanine Hydroxylase Mrna: The Major Cause for Phenylketonuria in Parts of Southern Europe,” Genomics
11, no. 2 (1991): 242–246.
</cite> [<a href="https://doi.org/10.1016/0888-7543(91)90129-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/1769645/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genomics&amp;title=Aberrant%20Splicing%20of%20Phenylalanine%20Hydroxylase%20Mrna:%20The%20Major%20Cause%20for%20Phenylketonuria%20in%20Parts%20of%20Southern%20Europe&amp;author=B.%20Dworniczak&amp;author=C.%20Aulehla%E2%80%90Scholz&amp;author=L.%20Kalaydjieva&amp;author=K.%20Bartholom%C3%A9&amp;author=K.%20Grudda&amp;volume=11&amp;issue=2&amp;publication_year=1991&amp;pages=242-246&amp;pmid=1769645&amp;doi=10.1016/0888-7543(91)90129-3&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jimd70067-bib-0019">
<span class="label">19.</span><cite>
Marvit J., DiLella A. G., Brayton K., Ledley F. D., Robson K. J. H., and Woo S. L. C., “GT to AT Transition at a Splice Donor Site Causes Skipping of the Preceding Exon in Phenylketonuria,” Nucleic Acids Research
15, no. 14 (1987): 5613–5628.
</cite> [<a href="https://doi.org/10.1093/nar/15.14.5613" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC306010/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/3615198/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Research&amp;title=GT%20to%20AT%20Transition%20at%20a%20Splice%20Donor%20Site%20Causes%20Skipping%20of%20the%20Preceding%20Exon%20in%20Phenylketonuria&amp;author=J.%20Marvit&amp;author=A.%20G.%20DiLella&amp;author=K.%20Brayton&amp;author=F.%20D.%20Ledley&amp;author=K.%20J.%20H.%20Robson&amp;volume=15&amp;issue=14&amp;publication_year=1987&amp;pages=5613-5628&amp;pmid=3615198&amp;doi=10.1093/nar/15.14.5613&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jimd70067-bib-0020">
<span class="label">20.</span><cite>
Monti S., Tamayo P., Mesirov J., and Golub T., “Consensus Clustering: A Resampling‐Based Method for Class Discovery and Visualization of Gene Expression Microarray Data,” Machine Learning
52, no. 1 (2003): 91–118.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Machine%20Learning&amp;title=Consensus%20Clustering:%20A%20Resampling%E2%80%90Based%20Method%20for%20Class%20Discovery%20and%20Visualization%20of%20Gene%20Expression%20Microarray%20Data&amp;author=S.%20Monti&amp;author=P.%20Tamayo&amp;author=J.%20Mesirov&amp;author=T.%20Golub&amp;volume=52&amp;issue=1&amp;publication_year=2003&amp;pages=91-118&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jimd70067-bib-0021">
<span class="label">21.</span><cite>
Yousefi B., Melograna F., Galazzo G., et al., “Capturing the Dynamics of Microbial Interactions Through Individual‐Specific Networks,” Frontiers in Microbiology
14 (2023): 1170391.
</cite> [<a href="https://doi.org/10.3389/fmicb.2023.1170391" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10225591/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37256048/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Microbiology&amp;title=Capturing%20the%20Dynamics%20of%20Microbial%20Interactions%20Through%20Individual%E2%80%90Specific%20Networks&amp;author=B.%20Yousefi&amp;author=F.%20Melograna&amp;author=G.%20Galazzo&amp;volume=14&amp;publication_year=2023&amp;pages=1170391&amp;pmid=37256048&amp;doi=10.3389/fmicb.2023.1170391&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jimd70067-bib-0022">
<span class="label">22.</span><cite>
Yousefi B. and Schwikowski B., “Consensus Clustering for Robust Bioinformatics Analysis,” Preprint, bioRxiv, March 21, 2024, 586064.</cite>
</li>
<li id="jimd70067-bib-0023">
<span class="label">23.</span><cite>
Gundorova P., Woidy M., Muntau A., and Gersting S. W., “<em>PAH</em> Genotype &amp; Activity Landscapes Database, 2015‑2025”, <a href="http://www.pah-activitylandscapes.org" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">www.pah‐activitylandscapes.org</a>.</cite>
</li>
<li id="jimd70067-bib-0024">
<span class="label">24.</span><cite>
Ikeda M., Levitt M., and Udenfriend S., “Phenylalanine as Substrate and Inhibitor of Tyrosine Hydroxylase,” Archives of Biochemistry and Biophysics
120, no. 2 (1967): 420–427.
</cite> [<a href="https://doi.org/10.1016/0003-9861(67)90259-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/6033458/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Archives%20of%20Biochemistry%20and%20Biophysics&amp;title=Phenylalanine%20as%20Substrate%20and%20Inhibitor%20of%20Tyrosine%20Hydroxylase&amp;author=M.%20Ikeda&amp;author=M.%20Levitt&amp;author=S.%20Udenfriend&amp;volume=120&amp;issue=2&amp;publication_year=1967&amp;pages=420-427&amp;pmid=6033458&amp;doi=10.1016/0003-9861(67)90259-7&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jimd70067-bib-0025">
<span class="label">25.</span><cite>
Dhondt J. L., Dautrevaux M., Biserte G., and Farriaux J. P., “Phenylalanine Analogues as Inhibitors of Phenylalanine‐Hydroxylase From Rat Liver. New Conclusions Concerning Kinetic Behaviors of the Enzyme,” Biochimie
60, no. 8 (1978): 787–794.
</cite> [<a href="https://doi.org/10.1016/s0300-9084(78)80023-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/728481/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biochimie&amp;title=Phenylalanine%20Analogues%20as%20Inhibitors%20of%20Phenylalanine%E2%80%90Hydroxylase%20From%20Rat%20Liver.%20New%20Conclusions%20Concerning%20Kinetic%20Behaviors%20of%20the%20Enzyme&amp;author=J.%20L.%20Dhondt&amp;author=M.%20Dautrevaux&amp;author=G.%20Biserte&amp;author=J.%20P.%20Farriaux&amp;volume=60&amp;issue=8&amp;publication_year=1978&amp;pages=787-794&amp;pmid=728481&amp;doi=10.1016/s0300-9084(78)80023-6&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jimd70067-bib-0026">
<span class="label">26.</span><cite>
Kokkonen P., Beier A., Mazurenko S., Damborsky J., Bednar D., and Prokop Z., “Substrate Inhibition by the Blockage of Product Release and Its Control by Tunnel Engineering,” RSC Chemical Biology
2, no. 2 (2021): 645–655.
</cite> [<a href="https://doi.org/10.1039/d0cb00171f" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8341658/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34458806/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=RSC%20Chemical%20Biology&amp;title=Substrate%20Inhibition%20by%20the%20Blockage%20of%20Product%20Release%20and%20Its%20Control%20by%20Tunnel%20Engineering&amp;author=P.%20Kokkonen&amp;author=A.%20Beier&amp;author=S.%20Mazurenko&amp;author=J.%20Damborsky&amp;author=D.%20Bednar&amp;volume=2&amp;issue=2&amp;publication_year=2021&amp;pages=645-655&amp;pmid=34458806&amp;doi=10.1039/d0cb00171f&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jimd70067-bib-0027">
<span class="label">27.</span><cite>
Tomé C. S., Lopes R. R., Sousa P. M. F., et al., “Structure of Full‐Length Wild‐Type Human Phenylalanine Hydroxylase by Small Angle X‐Ray Scattering Reveals Substrate‐Induced Conformational Stability,” Scientific Reports
9, no. 1 (2019): 13615.
</cite> [<a href="https://doi.org/10.1038/s41598-019-49944-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6754429/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31541188/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Scientific%20Reports&amp;title=Structure%20of%20Full%E2%80%90Length%20Wild%E2%80%90Type%20Human%20Phenylalanine%20Hydroxylase%20by%20Small%20Angle%20X%E2%80%90Ray%20Scattering%20Reveals%20Substrate%E2%80%90Induced%20Conformational%20Stability&amp;author=C.%20S.%20Tom%C3%A9&amp;author=R.%20R.%20Lopes&amp;author=P.%20M.%20F.%20Sousa&amp;volume=9&amp;issue=1&amp;publication_year=2019&amp;pages=13615&amp;pmid=31541188&amp;doi=10.1038/s41598-019-49944-x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jimd70067-bib-0028">
<span class="label">28.</span><cite>
Zschocke J., “Phenylketonuria Mutations in Europe,” Human Mutation
21, no. 4 (2003): 345–356.
</cite> [<a href="https://doi.org/10.1002/humu.10192" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12655544/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Human%20Mutation&amp;title=Phenylketonuria%20Mutations%20in%20Europe&amp;author=J.%20Zschocke&amp;volume=21&amp;issue=4&amp;publication_year=2003&amp;pages=345-356&amp;pmid=12655544&amp;doi=10.1002/humu.10192&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jimd70067-bib-0029">
<span class="label">29.</span><cite>
Trefz F. K., Scheible D., Götz H., and Frauendienst‐Egger G., “Significance of Genotype in Tetrahydrobiopterin‐Responsive Phenylketonuria,” Journal of Inherited Metabolic Disease
32, no. 1 (2009): 22–26.
</cite> [<a href="https://doi.org/10.1007/s10545-008-0940-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18956252/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Inherited%20Metabolic%20Disease&amp;title=Significance%20of%20Genotype%20in%20Tetrahydrobiopterin%E2%80%90Responsive%20Phenylketonuria&amp;author=F.%20K.%20Trefz&amp;author=D.%20Scheible&amp;author=H.%20G%C3%B6tz&amp;author=G.%20Frauendienst%E2%80%90Egger&amp;volume=32&amp;issue=1&amp;publication_year=2009&amp;pages=22-26&amp;pmid=18956252&amp;doi=10.1007/s10545-008-0940-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jimd70067-bib-0030">
<span class="label">30.</span><cite>
Trunzo R., Santacroce R., D'Andrea G., et al., “Phenylalanine Hydroxylase Deficiency in South Italy: Genotype‐Phenotype Correlations, Identification of a Novel Mutant PAH Allele and Prediction of BH4 Responsiveness,” Clinica Chimica Acta
450 (2015): 51–55.</cite> [<a href="https://doi.org/10.1016/j.cca.2015.07.014" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26210745/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Clinica%20Chimica%20Acta&amp;title=Phenylalanine%20Hydroxylase%20Deficiency%20in%20South%20Italy:%20Genotype%E2%80%90Phenotype%20Correlations,%20Identification%20of%20a%20Novel%20Mutant%20PAH%20Allele%20and%20Prediction%20of%20BH4%20Responsiveness&amp;author=R.%20Trunzo&amp;author=R.%20Santacroce&amp;author=G.%20D'Andrea&amp;volume=450&amp;publication_year=2015&amp;pages=51-55&amp;pmid=26210745&amp;doi=10.1016/j.cca.2015.07.014&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jimd70067-bib-0031">
<span class="label">31.</span><cite>
Gersting S. W., Kemter K. F., Staudigl M., et al., “Loss of Function in Phenylketonuria is Caused by Impaired Molecular Motions and Conformational Instability,” American Journal of Human Genetics
83, no. 1 (2008): 5–17.
</cite> [<a href="https://doi.org/10.1016/j.ajhg.2008.05.013" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2443833/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18538294/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=American%20Journal%20of%20Human%20Genetics&amp;title=Loss%20of%20Function%20in%20Phenylketonuria%20is%20Caused%20by%20Impaired%20Molecular%20Motions%20and%20Conformational%20Instability&amp;author=S.%20W.%20Gersting&amp;author=K.%20F.%20Kemter&amp;author=M.%20Staudigl&amp;volume=83&amp;issue=1&amp;publication_year=2008&amp;pages=5-17&amp;pmid=18538294&amp;doi=10.1016/j.ajhg.2008.05.013&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jimd70067-bib-0032">
<span class="label">32.</span><cite>
Santos L. L., Fonseca C. G., Starling A. L. P., et al., “Variations in Genotype‐Phenotype Correlations in Phenylketonuria Patients,” Genetics and Molecular Research
9, no. 1 (2010): 1–8.
</cite> [<a href="https://doi.org/10.4238/vol9-1gmr670" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20082265/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genetics%20and%20Molecular%20Research&amp;title=Variations%20in%20Genotype%E2%80%90Phenotype%20Correlations%20in%20Phenylketonuria%20Patients&amp;author=L.%20L.%20Santos&amp;author=C.%20G.%20Fonseca&amp;author=A.%20L.%20P.%20Starling&amp;volume=9&amp;issue=1&amp;publication_year=2010&amp;pages=1-8&amp;pmid=20082265&amp;doi=10.4238/vol9-1gmr670&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jimd70067-bib-0033">
<span class="label">33.</span><cite>
Zhu T., Ye J., Han L., et al., “Variations in Genotype‐Phenotype Correlations in Phenylalanine Hydroxylase Deficiency in Chinese Han Population,” Gene
529, no. 1 (2013): 80–87.
</cite> [<a href="https://doi.org/10.1016/j.gene.2013.07.079" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23932990/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Gene&amp;title=Variations%20in%20Genotype%E2%80%90Phenotype%20Correlations%20in%20Phenylalanine%20Hydroxylase%20Deficiency%20in%20Chinese%20Han%20Population&amp;author=T.%20Zhu&amp;author=J.%20Ye&amp;author=L.%20Han&amp;volume=529&amp;issue=1&amp;publication_year=2013&amp;pages=80-87&amp;pmid=23932990&amp;doi=10.1016/j.gene.2013.07.079&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jimd70067-bib-0034">
<span class="label">34.</span><cite>
Vela‐Amieva M., Abreu‐González M., González‐del Angel A., et al., “Phenylalanine Hydroxylase Deficiency in Mexico: Genotype‐Phenotype Correlations, BH4 Responsiveness and Evidence of a Founder Effect,” Clinical Genetics
88, no. 1 (2015): 62–67.
</cite> [<a href="https://doi.org/10.1111/cge.12444" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24941924/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Clinical%20Genetics&amp;title=Phenylalanine%20Hydroxylase%20Deficiency%20in%20Mexico:%20Genotype%E2%80%90Phenotype%20Correlations,%20BH4%20Responsiveness%20and%20Evidence%20of%20a%20Founder%20Effect&amp;author=M.%20Vela%E2%80%90Amieva&amp;author=M.%20Abreu%E2%80%90Gonz%C3%A1lez&amp;author=A.%20Gonz%C3%A1lez%E2%80%90del%20Angel&amp;volume=88&amp;issue=1&amp;publication_year=2015&amp;pages=62-67&amp;pmid=24941924&amp;doi=10.1111/cge.12444&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="caption p"><p>
<strong>Data S1:</strong> Supporting Information.</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12307255/bin/JIMD-48-0-s001.docx" data-ga-action="click_feat_suppl" class="usa-link">JIMD-48-0-s001.docx</a><sup> (14.8KB, docx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="caption p"><p>
<strong>Figure S1:</strong> [BH<sub>4</sub>] at peak residual activity across subpopulations.</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12307255/bin/JIMD-48-0-s004.pdf" data-ga-action="click_feat_suppl" class="usa-link">JIMD-48-0-s004.pdf</a><sup> (1.3MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="caption p"><p>
<strong>Table S1:</strong> jimd70067‐sup‐0003‐TableS1.xlsx. <em>PAH</em> gene variants.</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12307255/bin/JIMD-48-0-s003.xlsx" data-ga-action="click_feat_suppl" class="usa-link">JIMD-48-0-s003.xlsx</a><sup> (13.6KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material4_reqid_"><div class="caption p"><p>
<strong>Table S2:</strong> jimd70067‐sup‐0004‐TableS2.xlsx. <em>PAH</em> gene genotypes.</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12307255/bin/JIMD-48-0-s002.xlsx" data-ga-action="click_feat_suppl" class="usa-link">JIMD-48-0-s002.xlsx</a><sup> (24.2KB, xlsx) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>The data that support the findings of this study are openly available in PAH genotypes &amp; activity landscapes database at <a href="http://pah-activitylandscapes.org/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://pah‐activitylandscapes.org/</a>. The data evaluation pipeline is available at the GitHub repository <a href="https://github.com/imsb-uke/Phenylketonuria" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/imsb‐uke/Phenylketonuria</a>.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Journal of Inherited Metabolic Disease are provided here courtesy of <strong>Wiley</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1002/jimd.70067"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/JIMD-48-0.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (1.1 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12307255/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12307255/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12307255%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12307255/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12307255/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12307255/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40730740/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12307255/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40730740/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12307255/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12307255/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="7VPNlAHPKXxWUTFNUUcnHYAAuVEtK3rxjx4sEfxycEzbm5OqTbaU3OSp5hrieKiy">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
        <script type="module" crossorigin="" src="/static/assets/math-574fdcc6.js"></script>
    
    

    </body>
</html>
