
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            A Click Chemistry-Based Biorthogonal Approach for the Detection and Identification of Protein Lysine Malonylation for Osteoarthritis Research - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="D2E482AF8AEEDF530582AF0027B2C703.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="acsbmcau">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371503/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="ACS Bio &amp; Med Chem Au">
<meta name="citation_title" content="A Click Chemistry-Based Biorthogonal Approach for the Detection and Identification of Protein Lysine Malonylation for Osteoarthritis Research">
<meta name="citation_author" content="Anupama Binoy">
<meta name="citation_author_institution" content="Department of Biomedical Sciences, Heritage College of Osteopathic Medicine (HCOM), Ohio University, Athens, Ohio 45701, United States">
<meta name="citation_author_institution" content="Ohio Musculoskeletal and Neurological Institute (OMNI), Heritage College of Osteopathic Medicine (HCOM), Ohio University, Athens, Ohio 45701, United States">
<meta name="citation_author" content="Pandurangan Nanjan">
<meta name="citation_author_institution" content="Department of Chemistry, Amrita School of Physical Sciences, Amrita Vishwa Vidyapeetham, Coimbatore Campus, Tamilnadu 641112, India">
<meta name="citation_author" content="Kavya Chellamuthu">
<meta name="citation_author_institution" content="Department of Chemistry, Amrita School of Physical Sciences, Amrita Vishwa Vidyapeetham, Coimbatore Campus, Tamilnadu 641112, India">
<meta name="citation_author" content="Huanhuan Liu">
<meta name="citation_author_institution" content="Department of Biomedical Sciences, Heritage College of Osteopathic Medicine (HCOM), Ohio University, Athens, Ohio 45701, United States">
<meta name="citation_author_institution" content="Ohio Musculoskeletal and Neurological Institute (OMNI), Heritage College of Osteopathic Medicine (HCOM), Ohio University, Athens, Ohio 45701, United States">
<meta name="citation_author" content="Shouan Zhu">
<meta name="citation_author_institution" content="Department of Biomedical Sciences, Heritage College of Osteopathic Medicine (HCOM), Ohio University, Athens, Ohio 45701, United States">
<meta name="citation_author_institution" content="Ohio Musculoskeletal and Neurological Institute (OMNI), Heritage College of Osteopathic Medicine (HCOM), Ohio University, Athens, Ohio 45701, United States">
<meta name="citation_author_institution" content="Diabetes Institute (DI), Heritage College of Osteopathic Medicine (HCOM), Ohio University, Athens, Ohio 45701, United States">
<meta name="citation_publication_date" content="2025 May 8">
<meta name="citation_volume" content="5">
<meta name="citation_issue" content="4">
<meta name="citation_firstpage" content="582">
<meta name="citation_doi" content="10.1021/acsbiomedchemau.4c00151">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371503/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371503/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371503/pdf/bg4c00151.pdf">
<meta name="description" content="Lysine malonylation is a post-translational modification in which a malonyl group, characterized by a negatively charged carboxylate, is covalently attached to the ε-amino side chain of lysine, influencing protein structure and function. Our ...">
<meta name="og:title" content="A Click Chemistry-Based Biorthogonal Approach for the Detection and Identification of Protein Lysine Malonylation for Osteoarthritis Research">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Lysine malonylation is a post-translational modification in which a malonyl group, characterized by a negatively charged carboxylate, is covalently attached to the ε-amino side chain of lysine, influencing protein structure and function. Our ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371503/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12371503">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1021/acsbiomedchemau.4c00151"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/bg4c00151.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12371503%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12371503/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12371503/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371503/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-acsbmcau.png" alt="ACS Bio &amp; Med Chem Au logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to ACS Bio &amp; Med Chem Au" title="Link to ACS Bio &amp; Med Chem Au" shape="default" href="https://pubs.acs.org/journal/abmcb8" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">ACS Bio Med Chem Au</button></div>. 2025 May 8;5(4):582–592. doi: <a href="https://doi.org/10.1021/acsbiomedchemau.4c00151" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1021/acsbiomedchemau.4c00151</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22ACS%20Bio%20Med%20Chem%20Au%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22ACS%20Bio%20Med%20Chem%20Au%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22ACS%20Bio%20Med%20Chem%20Au%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22ACS%20Bio%20Med%20Chem%20Au%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>A Click
Chemistry-Based Biorthogonal Approach for
the Detection and Identification of Protein Lysine Malonylation for
Osteoarthritis Research</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Binoy%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Anupama Binoy</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Anupama Binoy</span></h3>
<div class="p">
<sup>†</sup>
Department
of Biomedical Sciences, Heritage College of Osteopathic Medicine (HCOM), Ohio University, Athens, Ohio
45701, United States
</div>
<div class="p">
<sup>‡</sup>
Ohio
Musculoskeletal and Neurological Institute (OMNI), Heritage College
of Osteopathic Medicine (HCOM), Ohio University, Athens, Ohio
45701, United States
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Binoy%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Anupama Binoy</span></a>
</div>
</div>
<sup>†,</sup><sup>‡</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nanjan%20P%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Pandurangan Nanjan</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Pandurangan Nanjan</span></h3>
<div class="p">
<sup>∥</sup>
Department
of Chemistry, Amrita School of Physical Sciences, Amrita Vishwa Vidyapeetham, Coimbatore Campus, Tamilnadu
641112, India
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nanjan%20P%22%5BAuthor%5D" class="usa-link"><span class="name western">Pandurangan Nanjan</span></a>
</div>
</div>
<sup>∥</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chellamuthu%20K%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Kavya Chellamuthu</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Kavya Chellamuthu</span></h3>
<div class="p">
<sup>∥</sup>
Department
of Chemistry, Amrita School of Physical Sciences, Amrita Vishwa Vidyapeetham, Coimbatore Campus, Tamilnadu
641112, India
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chellamuthu%20K%22%5BAuthor%5D" class="usa-link"><span class="name western">Kavya Chellamuthu</span></a>
</div>
</div>
<sup>∥</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Liu%20H%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Huanhuan Liu</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Huanhuan Liu</span></h3>
<div class="p">
<sup>†</sup>
Department
of Biomedical Sciences, Heritage College of Osteopathic Medicine (HCOM), Ohio University, Athens, Ohio
45701, United States
</div>
<div class="p">
<sup>‡</sup>
Ohio
Musculoskeletal and Neurological Institute (OMNI), Heritage College
of Osteopathic Medicine (HCOM), Ohio University, Athens, Ohio
45701, United States
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Liu%20H%22%5BAuthor%5D" class="usa-link"><span class="name western">Huanhuan Liu</span></a>
</div>
</div>
<sup>†,</sup><sup>‡</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhu%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Shouan Zhu</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Shouan Zhu</span></h3>
<div class="p">
<sup>†</sup>
Department
of Biomedical Sciences, Heritage College of Osteopathic Medicine (HCOM), Ohio University, Athens, Ohio
45701, United States
</div>
<div class="p">
<sup>‡</sup>
Ohio
Musculoskeletal and Neurological Institute (OMNI), Heritage College
of Osteopathic Medicine (HCOM), Ohio University, Athens, Ohio
45701, United States
</div>
<div class="p">
<sup>§</sup>
Diabetes
Institute (DI), Heritage College of Osteopathic Medicine (HCOM), Ohio University, Athens, Ohio
45701, United States
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhu%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Shouan Zhu</span></a>
</div>
</div>
<sup>†,</sup><sup>‡,</sup><sup>§,</sup><sup>*</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="aff1">
<sup>†</sup>
Department
of Biomedical Sciences, Heritage College of Osteopathic Medicine (HCOM), Ohio University, Athens, Ohio
45701, United States
</div>
<div id="aff2">
<sup>‡</sup>
Ohio
Musculoskeletal and Neurological Institute (OMNI), Heritage College
of Osteopathic Medicine (HCOM), Ohio University, Athens, Ohio
45701, United States
</div>
<div id="aff3">
<sup>§</sup>
Diabetes
Institute (DI), Heritage College of Osteopathic Medicine (HCOM), Ohio University, Athens, Ohio
45701, United States
</div>
<div id="aff4">
<sup>∥</sup>
Department
of Chemistry, Amrita School of Physical Sciences, Amrita Vishwa Vidyapeetham, Coimbatore Campus, Tamilnadu
641112, India
</div>
<div class="author-notes p"><div class="fn" id="cor1">
<sup>*</sup><p class="display-inline">
Email: <span>zhus1@ohio.edu</span>.</p>
</div></div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2024 Dec 24; Accepted 2025 Apr 28; Revised 2025 Apr 27; Collection date 2025 Aug 20.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© 2025 The Authors. Published by American Chemical Society</div>
<p>This article is licensed under CC-BY-NC-ND 4.0</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12371503</div>
<div class="ra xbox p" role="complementary" aria-label="Related or updated information about this article"><div>
<strong>Previous version available:</strong> This article is based on a previously available preprint posted on bioRxiv on December 13, 2024: "<a href="/articles/PMC11661220/" class="usa-link">A click chemistry-based biorthogonal approach for the detection and identification of protein lysine malonylation for osteoarthritis research</a>".</div></div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>Abstract</h2>
<p>Lysine malonylation is a post-translational modification
in which
a malonyl group, characterized by a negatively charged carboxylate,
is covalently attached to the ε-amino side chain of lysine,
influencing protein structure and function. Our laboratory identified
Mak upregulation in cartilage under aging and obesity, contributing
to osteoarthritis (OA). Current antibody-based detection methods face
limitations in identifying Mak targets. Here, we introduce an alkyne-functionalized
probe, MA-diyne, which metabolically incorporates into proteins, enabling
copper­(I) ion-catalyzed click reactions to conjugate labeled proteins
with azide-based fluorescent dyes or affinity purification tags. In-gel
fluorescence confirms MA-diyne incorporation into proteins across
various cell types and species, including mouse chondrocytes, adipocytes,
HEK293T cells, and . Pull-down experiments identified known Mak proteins, such as GAPDH
and Aldolase. The extent of MA-diyne modification was higher in Sirtuin
5-deficient cells, suggesting these modified proteins are Sirtuin
5 substrates. Pulse-chase experiments confirmed the dynamic nature
of the protein malonylation. Quantitative proteomics identified 1136
proteins corresponding to 8903 peptides, with 429 proteins showing
a 1-fold increase in the labeled group. Sirtuin 5 regulated 374 of
these proteins. Pull down of newly identified proteins, such as β-actin
and Stat3, was also done. This study highlights MA-diyne as a powerful
chemical tool to investigate the molecular targets and functions of
lysine malonylation under OA conditions.</p>
<section id="kwd-group1" class="kwd-group"><p><strong>Keywords:</strong> alkyne-based probe, chondrocyte, click chemistry, Sirtuin 5, lysine malonylation</p></section></section><section class="abstract" id="abstract2"><hr class="headless">
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371503_bg4c00151_0008.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d548/12371503/f2573db1745a/bg4c00151_0008.jpg" loading="lazy" height="279" width="645" alt="graphic file with name bg4c00151_0008.jpg"></a></p></section><section class="abstract" id="abstract3"><hr class="headless">
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371503_bg4c00151_0006.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d548/12371503/f2573db1745a/bg4c00151_0006.jpg" loading="lazy" id="tgr1" height="279" width="645" alt="graphic file with name bg4c00151_0006.jpg"></a></p></section><section id="sec1"><h2 class="pmc_sec_title">1. Introduction</h2>
<p>Post-translational modification
of proteins refers to the biochemical
covalent, enzymatic, and nonenzymatic addition of functional groups
on proteins following their synthesis from ribosomes. These functional groups can be small electrophilic biochemical
metabolites like phosphate, sugars, nucleotides, methyl group, and
acetyl group, long chain/short chain acyl chains like palmitoyl, malonyl
groups, radicals generated from redox reactions like <em>S</em>-nitrosylation (SNO) and <em>S</em>-glutathionylation, etc. Post-translational modifications induce changes
in amino acid chemical properties such as deamination, deamidation,
citrullination, and oxidation as well as protein autocatalysis that
results in protein backbone cleavage. It offers complex and diverse
functional roles to the existing proteome by regulating protein activity,
structure, and conformation, its location, and molecular interactions. Post-translational covalent modification of proteins
is fundamental to numerous cellular and biological functions. It plays
a significant role in regulating normal cell physiology as well as
the pathogenesis of diseases. Lysine malonylation
(Mak) is a reversible protein post-translational modification wherein
a malonyl group is added to the ε-amino group of the lysine
residue in a protein. The addition of a negatively charged carboxylate
group to the protein imparts significant changes in the protein’s
structure and function. The reversible
protein malonylation is theoretically regulated by acyltransferases
and deacylases. The enzymes responsible for catalyzing the transfer
of a malonyl group remain largely unidentified. However, recent work
by Zang et al. has provided evidence that KAT2A is involved in histone
malonylation. Meanwhile, Sirtuin 5 (Sirt5),
a class III lysine deacetylases member, is known for its lysine demalonylation
activity along with its lysine desuccinylation function.<em>
,
</em> Our lab has previously demonstrated that deficiency of Sirt5 in
mouse primary chondrocytes increases the global protein MaK level
and disrupts cellular metabolism. Identification
of the protein substrates of Sirt5 is crucial for further elucidating
the function of malonylation on the cellular metabolism and pinpointing
the downstream targets. The antimalonyl lysine antibody has been used
previously to enrich and identify several malonylated proteins.<em>
−

</em> Antibody-based enrichment is generally based on the development
of different antibody domains specific to targeting antigens with
specific PTM, followed by immunoaffinity pull-down of target peptides
from the protein lysates. However, this strategy suffers from a lack
of specificity by missing some of the target PTM site due to the presence
of adjacent PTM present on the same sequence. To overcome this challenge, we took a chemical approach, utilizing
an alkyne functionalized chemical probe to efficiently detect and
quantify protein substrates of lysine malonylation through fluorescence
visualization and further identify them using a quantitative proteomics
approach (<a href="#fig1" class="usa-link">Figure 
</a>B).
An alkyne functionalized probe allows metabolic detection of protein
substrates by utilizing the copper­(I) ion-catalyzed alkyne azide cycloaddition
click chemistry to conjugate the labeled proteins with azide-fluorescent
dyes or affinity purification tags. This
approach has been widely utilized in identifying several other types
of PTMs including lipidation like myristoylation,<em>
,
</em> succinylation,<em>
,
</em> acetylation,<em>
,
</em> and glycosylation.<em>
,
</em> Previously, Bao et al. developed
a malonic acid-based chemical probe called MalAM-yne to detect lysine
malonylation. In this study, we designed
and synthesized a novel Meldrum’s acid-based probe, i.e., MA-diyne.
Meldrum’s acid or isopropylidene malonate is a condensation
product of malonic acid and acetone. We developed this new probe by
functionalizing Meldrum acid or isopropylidene malonate with alkynylation
under control conditions (<a href="#fig1" class="usa-link">Figure 
</a>A). We hypothesize that Meldrum acid will be readily
absorbed into the cells due to its cyclic structure and subsequently
will be linearized to form a malonyl group by an unknown action of
intracellular esterases, thus leading to effective detection and identification
of the malonylated proteins.</p>
<figure class="fig xbox font-sm" id="fig1"><h3 class="obj_head">1.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371503_bg4c00151_0001.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d548/12371503/f811a8dc26ea/bg4c00151_0001.jpg" loading="lazy" id="gr1" height="545" width="736" alt="1"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>(A) Design, synthesis, and characterization
of MA-diyne. (B) Schematic
description of the workflow to detect and identify lysine malonylated
proteins using MA-diyne.</p></figcaption></figure></section><section id="sec2"><h2 class="pmc_sec_title">2. Results and Discussion</h2>
<section id="sec2.1"><h3 class="pmc_sec_title">2.1. Chemical Synthesis and Characterization</h3>
<p>Bao et al. 2013 were the first to report a chemical probe with
an alkyne handle on the malonate group named Mal-yne. However, they modified this probe to increase cell permeability
by masking the carboxylate sides with two acetoxymethyl groups and
called it MalAM-yne. In this report, we selected an alternative approach
by adding an alkyne handle to Meldrum acid or isopropylidene malonate.<em>
,
</em> The high reactivity of this molecule is attributed to the methylene
group present between the carbonyl groups, and this site was utilized
to add the propargyl unit to Meldrum acid under controlled conditions.
Thereby, we generated dipropargyl Meldrum acid or MA-diyne through
the following chemical synthesis (<a href="#fig1" class="usa-link">Figure 
</a>A). The synthesis involves dialkynylation
on commercially available Meldrum acid. The reaction is performed
under mild conditions such as DIPEA (diisopropyl ethylamine) with
anhydrous DMF solvent at room temperature. The reaction is stirred
for 24 h. After the work-up of the reaction, the product is taken
for purification. The initial purification of the MA-diyne product
is carried out with normal SiO<sub>2</sub>, which leads to ring-opening
of MA-diyne due to its acidity. Later, SiO<sub>2</sub> is neutralized
with triethylamine, and then, purification is carried out. The pure
product is characterized by <sup>1</sup>H NMR spectroscopy [<sup>1</sup>H NMR (400 MHz): δ 1.84 (s, 6H), 2.18, (t, <em>J</em> = 5.2 Hz, 2H), 2.88 (d, <em>J</em> = 2.8 Hz, 4H) ppm. The
respective assignments and full spectra of MA-diyne are given as the <a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.4c00151/suppl_file/bg4c00151_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Supporting Information</a> (Schemes SF1 and SF2).
The synthesized probe showed two distinct quaternary carbons: (i)
near two oxygen atoms and (ii) near two carbonyl groups. The same
quaternary peaks were observed in the <sup>13</sup>C NMR spectrum:
one at 107 ppm, which is near oxygen atoms, and another at 53.43 ppm,
which is near carbonyl groups. This confirms dipropargylation in the
active methylene carbon of Meldrum’s acid. The <sup>13</sup>C NMR spectrum peaks are depicted in <a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.4c00151/suppl_file/bg4c00151_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Scheme SF3</a>. Although mass spectrum analysis of dipropargyl Meldrum’s
acid revealed no M+ peak for 220.07, we observed smaller fragments
(sized 192.04, 148.05, 92.06, and 203.28) as shown in <a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.4c00151/suppl_file/bg4c00151_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Scheme SF4</a>. The reasons for the fragmentation
of the parent molecule could be attributed to the labile nature of
the lactone structure. However, the occurrence of smaller fragments
indirectly confirms the expected compound. The previous method by
Bao et al. 2013 involves a three-step strategy, where the first step
is propargylation of diethyl malonate followed by hydrolysis to propargylated
malonic acid with a yield of 30%. Then, propargylated malonic acid
is further taken for masking the carboxylate groups with bromomethyl
acetate under ambient temperature, resulting in the product formation
of the Mal-yne probe with a yield of 77%. As a result, the reported
method gives an overall yield of 23%. MA-diyne, on the other hand, was synthesized via a streamlined synthetic
approach, employing dialkynylation of Meldrum’s acid, which
is commercially available. This method proved both straightforward
(a single-step process) and expeditious, achieving a maximum yield
of 40%, markedly surpassing the 23% yield (overall) obtained with
MalAM-yne. However, the yield limitation to 40% in the present method
arises from the ring-opening reaction of Meldrum’s acid under
the specified conditions. Furthermore, cyclic structures are seen
with improved cell permeability due to their reduced conformational
flexibility.<em>
−

</em> Therefore, we hypothesized that Meldrum acid due
to its cyclic structure will be readily taken into the cell. MA-diyne
demonstrated comparable labeling efficiency over a 0.5–1 h
duration to MalAM-yne, without necessitating the protection or masking
of Mal-yne’s carboxylate groups with two acetoxymethyl (AM)
substituents (as required to form MalAM-yne). This suggests efficient
cellular uptake. The enhanced permeability of MA-diyne may be ascribed
to the stabilization of its carboxylate groups within a cyclic structure.</p></section><section id="sec2.2"><h3 class="pmc_sec_title">2.2. MA-Diyne Can Be Metabolically Incorporated
into Cellular Proteins</h3>
<p>Our first inquest was to analyze whether
MA-diyne can be metabolically incorporated into proteins similarly
to malonyl-CoA. Therefore, we incubated mouse primary chondrocytes
with different concentrations (0–200 μM) of MA-diyne
in complete medium for 6 h (a working stock of 100 mM was prepared
by dissolving MA-diyne in DMSO; DMSO was taken as the control). The
protein lysate was collected after harvesting the cells and was subjected
to azide–alkyne click chemistry to conjugate the alkyne side
group with IR680 azide dye. The clicked proteins were then resolved
on SDS-PAGE and scanned using an in-gel IR fluorescence imager. The
result showed that there was a dose-dependent increase of MA-diyne
labeling of global proteins with the optimal concentration at less
than 50 μM MA-diyne (<a href="#fig2" class="usa-link">Figure 
</a>A). To analyze the time required for the metabolic
labeling, a time-dependent experiment was conducted by incubating
the primary chondrocytes with 100 μM of MA-diyne at different
time points ranging from 0 to 12 h. MA-diyne was able to efficiently
label the proteins in no more than 2 h (<a href="#fig2" class="usa-link">Figure 
</a>B). Interestingly, we observed that MA-diyne
showed more bands in gel fluorescence than in Western blot analysis
of the same proteins from primary chondrocytes using a commercial
anti-KMal antibody (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.4c00151/suppl_file/bg4c00151_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figure S1</a>). The dose-
and time-dependent experiment demonstrated the efficient labeling
of proteins by MA-diyne. MA-diyne employs a strategy of metabolic
labeling of proteins. Since malonyl-CoA can target both cysteine and
lysine residues due to their nucleophilic properties, it is imperative
to recognize that MA-diyne may label both amino acids. In fact, previous
studies have shown that malonylation of a single cysteine residue
is critical for the activity enzymes CPT1 (a pivotal enzyme facilitating the translocation of long-chain fatty
acids from the cytoplasm to the mitochondrial matrix, modulated by
its physiological inhibitor, malonyl-CoA) and DpgA (a bacterial type III polyketide synthase that decarboxylates
and concatenates four malonyl-CoA units to yield 3,5-dihydroxyphenylacetyl-CoA).
To elucidate the chemoselectivity of MA-diyne toward lysine, we conducted
a labeling experiment in the presence and absence of hydroxylamine,
a chemical to mask cysteine residues. As depicted in <a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.4c00151/suppl_file/bg4c00151_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figure S2A,B</a>, the metabolic labeling exhibited minimal reduction
when MA-diyne-labeled protein lysates were subjected to incubation
with 1 M hydroxylamine at 95 °C for 5 min. This observation suggests
that although MA-diyne can label both lysine and cysteine side chains
on proteins, it can predominantly label lysine side chains with a
markedly greater propensity than it can cysteine groups. Another possibility
is that the lower abundance of cysteines than of lysines in the protein
may result in only a minimal reduction in labeling when cysteines
are masked by hydroxylamine. Further studies are warranted to investigate
this possibility in more detail.</p>
<figure class="fig xbox font-sm" id="fig2"><h4 class="obj_head">2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371503_bg4c00151_0002.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d548/12371503/79e384e3434f/bg4c00151_0002.jpg" loading="lazy" id="gr2" height="540" width="681" alt="2"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Assessment of the ability of MA-diyne
to metabolically label proteins.
(A) Mouse primary chondrocytes were incubated with the indicated concentration
of MA-diyne for 6 h. The cell lysates were then clicked with IR680-azide
followed by in-gel fluorescence analysis. (B) Mouse primary chondrocytes
were incubated with 100 μM MA-diyne for the indicated time points.
Cell lysates were then clicked with IR680-azide and in-gel fluorescence
analyses. β actin was used as a loading control. (C) Confocal
microscopic fluorescence image depicting the ready uptake of the MA-diyne
into the cells and subcellular localization of malonylated proteins
in the primary chondrocytes. The primary chondrocytes grown on coverslips
were labeled with 100 μM MA-diyne for 6 h and then subjected
to click chemistry with Carboxyrhodamine 110 Azide (fluorescein isothiocyanate-FITC
tag) after fixing and permeabilization. Scale bar: 50 μM. (D)
Box plot showing the concentration of intracellular malonyl-CoA in
the cells after incubation with different concentrations of MA-diyne
for 4 h. <em>n</em> = 4. Data are presented as mean ±
SEM. Three group comparisons were evaluated using two-way ANOVA. Significance
is noted as ns <em>p</em> &gt; 0.05, *<em>p</em> &lt;
0.05, and **<em>p</em> &lt; 0.01.</p></figcaption></figure><p>To confirm that the signal achieved in the primary
chondrocytes
after treatment with MA-diyne is due to metabolic labeling and not
to nonspecific binding, we used qualitative fluorescence imaging to
visualize the labeled proteins. The primary chondrocytes were cultured
on coverslips overnight and treated with different doses of MA-diyne
for 6 h. The cells were immediately fixed using ice-cold 4% paraformaldehyde
and then permeabilized. The cells were then subjected to a click reaction
using Carboxyrhodamine 110 Azide for 1 h. The cells on the coverslips
were mounted on clean glass slides using a mounting reagent with DAPI.
Control samples were generated by performing the same procedure without
MA-diyne or Carboxyrhodamine 110 Azide. The slides were imaged using
Nikon A1R confocal microscopy under 60× magnification (<a href="#fig2" class="usa-link">Figures 
</a>C and <a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.4c00151/suppl_file/bg4c00151_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">S3</a>). MA-diyne was found to be rapidly absorbed
into the cells, which was demonstrated by the intensity of labeling
only in the samples treated with MA-diyne followed by the click reaction
with Carboxyrhodamine 110 Azide but not in the control samples incubated
with Carboxyrhodamine 110 Azide only or samples treated with only
MA-diyne (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.4c00151/suppl_file/bg4c00151_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figure S3</a>). We observed widespread
protein labeling in various cell compartments, prompting us to conduct
quantitative proteomics. However, to ensure the labeling is due to
the malonyl-CoA formed by the acyclization of Meldrum acid, we estimated
the concentration of malonyl-CoA formed in the cells using an ELISA
kit after incubating the cells with MA-diyne in different concentrations
and compared it with that in control cells without MA-diyne. We observed
a significant increase in the concentration of malonyl CoA at 100
μM. Here, it is noteworthy to state the limitation of the ELISA
kit in estimating the malonyl CoA in the alkyne form. Nevertheless,
as protein labeling by MA-diyne intensifies with prolonged incubation,
the measured malonyl-CoA likely reflects the residual malonyl-CoA
remaining after conjugation with proteins rather than the total malonyl-CoA
produced from the linearization of MA-diyne.</p></section><section id="sec2.3"><h3 class="pmc_sec_title">2.3. MA-Diyne Was Dynamically Removed from the
Proteins, Representing the Reversible Nature of Lysine Malonylation</h3>
<p>Lysine modifications like succinylation, glutarylation, and malonylation
are reversible forms of protein post-translational modifications.<em>
,,
</em> Although the enzyme responsible
for adding a malonyl group to the proteins is still unknown, Sirtuin
5 (Sirt5) has been well-documented as the enzyme responsible for removing
this modification.<em>
,
</em> To examine whether the labeling
by MA-diyne is also reversible, we conducted a pulse-chase experiment.
The wild type and Sirt5 knockdown primary chondrocytes were first
incubated with 200 μM MA-diyne for 1 h. The cells were then
washed and chased with 200 μM Meldrum acid for 0–5 h.
The cells were harvested at different time points, and the protein
lysates were conjugated with IR680-azide dye. The in-gel fluorescence
imaging reveals that the labeling signal by MA-diyne in wild-type
primary chondrocytes started fading after 0.5 h of Meldrum acid incubation,
indicating the removal of MA-diyne labeling from the proteins (<a href="#fig3" class="usa-link">Figures 
</a>A and <a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.4c00151/suppl_file/bg4c00151_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">S4</a>). In comparison, in Sirt5 knockdown cells,
the labeling signal with MA-diyne became more intense after 0.5 h
than the baseline level, and there is less removal of MA-diyne from
the labeled proteins than from the wild-type cells (<a href="#fig3" class="usa-link">Figures 
</a>B and <a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.4c00151/suppl_file/bg4c00151_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">S4</a>). We also noticed that the labeling by MA-diyne was not
modulated by the endogenous levels of malonyl-CoA, evidenced by no
difference between wild type cells and cells deficient of acetyl-CoA
carboxylase (ACC1), an enzyme that produces malonyl-CoA (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.4c00151/suppl_file/bg4c00151_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figure S5</a>). These results suggested that the
metabolically modified proteins by MA-diyne could be substrates of
Sirt5 demalonylase and the process is reversible.</p>
<figure class="fig xbox font-sm" id="fig3"><h4 class="obj_head">3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371503_bg4c00151_0003.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d548/12371503/c586b073c71b/bg4c00151_0003.jpg" loading="lazy" id="gr3" height="623" width="796" alt="3"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Pulse-chase experiment
to determine the dynamic nature of lysine
malonylation. Wild-type mouse primary chondrocytes (A) and Sirt5 KD
primary chondrocytes (B) were labeled with 200 μM MA-diyne for
1 h and then pulse chased with 200 μM Meldrum acid (precursor).
The lysates were collected at the indicated time points followed by
a click reaction with IR680-azide and in gel fluorescence analysis.
β actin was used as a loading control. The same blot was probed
with an anti-Sirt5 antibody to quantify the knockdown efficiency.</p></figcaption></figure></section><section id="sec2.4"><h3 class="pmc_sec_title">2.4. MA-Diyne Successfully Identified Putative
Malonylated Proteins</h3>
<p>Our previous studies have demonstrated
the important role of Sirt5 in chondrocyte metabolism by regulating
the malonylation of metabolic enzymes.<em>
−


</em> We have also used proteomics
to identify the malonylome in chondrocytes using the traditional Kmal
PTMscan antibody-based enrichment method. We reported the enrichment of 1000 peptides corresponding to 469
proteins. Herein this report, we applied a chemoproteomics-based approach
to enrich malonylated proteins from primary chondrocytes using MA-diyne.
Wild type and Sirt5 knockdown primary chondrocytes were labeled with
MA-diyne (100 μM) for 6 h, and then, the protein lysates were
conjugated to biotin azide through an azide–alkyne copper cycloaddition
reaction. The biotin-conjugated proteins were then immunoprecipitated
using avidin agarose beads. The beads were thoroughly washed with
HPLC water to prevent detergent or protease inhibitor contamination
in the LC–MS, followed by the addition of a 9 M urea wash.
The beads were then subjected to reductive alkylation with dithiothreitol
(DTT) and iodoacetamide (IAA), on-bead trypsin digestion, and desalting.
The enriched peptides were then subjected to bottom-up quantitative
proteomics on an Orbitrap Astral Instrument. A total of 1136 proteins
corresponding to 8903 peptides across all samples (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.4c00151/suppl_file/bg4c00151_si_002.xlsx" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Supporting Information</a> excel 1) were identified, which was
2.4 times more than the proteins we identified before using Kmal PTMscan
antibody enrichment. 430 proteins were
seen to show a more than 1-fold increase in the probe group in comparison
to the control group with more than 6 unique peptides (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.4c00151/suppl_file/bg4c00151_si_002.xlsx" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Supporting Information</a> excel 2). This indicated
that MA-diyne could enrich malonylated proteins. The enrichment of
proteins in the control group accounts for the endogenously biotinylated
proteins that might have been enriched with avidin beads. Furthermore,
387 out of the 430 proteins were found to have a more than 1-fold
increase in the Sirt5 knockdown + MA-diyne group in comparison to
the wild type + MA-diyne group (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.4c00151/suppl_file/bg4c00151_si_002.xlsx" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Supporting Information</a> excel 2). This indicates that Ma-diyne successfully enriched proteins
that are regulated by Sirt5 demalonylase enzyme. It is possible that
MA-diyne may nonspecifically bind to other protein residues due to
the structural difference of Meldrum acid (precursor of MA-diyne)
from malonic acid. Therefore, we compared the 8903 peptides to those
modified sites identified by Kmal PTMscan antibody enrichment in our
previous study. Interestingly, some of the peptides sequences identified
in MA-diyne enrichment matched with the sequence identified in the
LC–MS/MS data acquired after enrichment with the Kmal PTMscan
antibody (<a href="#tbl1" class="usa-link">Table 
</a>). All these observations indicate that MA-diyne
can efficiently detect and identify lysine malonylated proteins, though
we still could completely exclude the possibility of nonspecific binding.
Since our data lack site-specific identification of malonylation,
our lab is currently exploring alternative proteomic approaches to
identify malonylated sites on proteins detected by MA-diyne in MA-diyne-treated
primary chondrocytes. Further, these proteins will be pulled down
using enrichment tags and analyzed by LC–MS/MS for the signature
satellite peaks.</p>
<section class="tw xbox font-sm" id="tbl1"><h4 class="obj_head">1. List of Manually Validated Peptides
Sequences with Modified Malonylated Sites.</h4>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<colgroup span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
</colgroup>
<thead><tr>
<th align="center" colspan="1" rowspan="1">Protein description</th>
<th align="center" colspan="1" rowspan="1">Accession
number</th>
<th align="center" colspan="1" rowspan="1">Peptide sequence
identified with Kmal enrichment</th>
<th align="center" colspan="1" rowspan="1">Corresponding
peptides enriched in MA-diyne treatment</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">L-Lactate dehydrogenase A chain</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/protein/P06151" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">P06151</a></td>
<td align="left" colspan="1" rowspan="1">IVSSKDYCVTANSK*</td>
<td align="left" colspan="1" rowspan="1">IVSSKDYCVTANSK, DYCVTANSK</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Glucose-6-phosphate Isomerase</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/protein/P06745" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">P06745</a></td>
<td align="left" colspan="1" rowspan="1">ELQAAGK*SPEDLEK</td>
<td align="left" colspan="1" rowspan="1">ELQAAGKSPEDLEK</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Annexin A1</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/protein/P10107" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">P10107</a></td>
<td align="left" colspan="1" rowspan="1">K*ALLALAK</td>
<td align="left" colspan="1" rowspan="1">KALLALAK</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Annexin A2</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/protein/P07356" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">P07356</a></td>
<td align="left" colspan="1" rowspan="1">ASM#K*GLGTDEDSLIEIICSR ELYDAGVK*R
TK*GVDEVTIVNILTNR</td>
<td align="left" colspan="1" rowspan="1">TPAQYDASELKASM#K, TPAQYDASELK,
GLGTDEDSLIEIICSR ELYDAGVKR TKGVDEVTIVNILTNR</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Calmodulin-1</td>
<td align="left" colspan="1" rowspan="1">
<a href="https://www.ncbi.nlm.nih.gov/protein/P0DP26" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">P0DP26</a>; <a href="https://www.ncbi.nlm.nih.gov/protein/P0DP27" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">P0DP27</a>;
<a href="https://www.ncbi.nlm.nih.gov/protein/P0DP28" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">P0DP28</a>
</td>
<td align="left" colspan="1" rowspan="1">EAFSLFDK*DGDGTITTK</td>
<td align="left" colspan="1" rowspan="1">EAFSLFDK, DGDGTITTK</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Protein S100-A4</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/protein/P07091" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">P07091</a></td>
<td align="left" colspan="1" rowspan="1">ELPSFLGK*R</td>
<td align="left" colspan="1" rowspan="1">ELPSFLGK</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Actin, cytoplasmic 1</td>
<td align="left" colspan="1" rowspan="1">
<a href="https://www.ncbi.nlm.nih.gov/protein/P60710" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">P60710</a>; <a href="https://www.ncbi.nlm.nih.gov/protein/P63260" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">P63260</a>
</td>
<td align="left" colspan="1" rowspan="1">K*DLYANTVLSGGTTM#YPGIADR</td>
<td align="left" colspan="1" rowspan="1">KDLYANTVLSGGTTM#YPGIADR</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Src substrate
cortactin</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/protein/Q60598" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Q60598</a></td>
<td align="left" colspan="1" rowspan="1">SAVGHEYQSK*LSK</td>
<td align="left" colspan="1" rowspan="1">SAVGHEYQSK</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Moesin</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/protein/P26041" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">P26041</a></td>
<td align="left" colspan="1" rowspan="1">AK*FYPEDVSEELIQDITQR</td>
<td align="left" colspan="1" rowspan="1">AKFYPEDVSEELIQDITQR</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Vinculin</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/protein/Q64727" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Q64727</a></td>
<td align="left" colspan="1" rowspan="1">NLGPGM#TKM#AK*</td>
<td align="left" colspan="1" rowspan="1">NLGPGMTK</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Glyceraldehyde-3-phosphate dehydrogenase</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/protein/P16858" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">P16858</a></td>
<td align="left" colspan="1" rowspan="1">VIHDNFGIVEGLM#TTVHAITATQK*TVDGPSGK</td>
<td align="left" colspan="1" rowspan="1">VIHDNFGIVEGLM#TTVHAITATQK</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Glyceraldehyde-3-phosphate dehydrogenase</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/protein/P16858" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">P16858</a></td>
<td align="left" colspan="1" rowspan="1">LVINGK*PITIFQERDPTNIK</td>
<td align="left" colspan="1" rowspan="1">LVINGKPITIFQERDPTNIK</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Glyceraldehyde-3-phosphatedehydrogenase</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/protein/P16858" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">P16858</a></td>
<td align="left" colspan="1" rowspan="1">TVDGPSGK*LWR</td>
<td align="left" colspan="1" rowspan="1">TVDGPSGKLWR</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Peptidyl-prolyl cis–trans isomerase A</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/protein/P17742" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">P17742</a></td>
<td align="left" colspan="1" rowspan="1">SIYGEK*FEDENFILK</td>
<td align="left" colspan="1" rowspan="1">SIYGEKFEDENFILK</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Pyruvate kinase PKM</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/protein/P52480" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">P52480</a></td>
<td align="left" colspan="1" rowspan="1">CCSGAIIVLTK*SGR</td>
<td align="left" colspan="1" rowspan="1">CCSGAIIVLTK</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Myosin regulatory light
chain 12B</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/protein/Q3THE2" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Q3THE2</a></td>
<td align="left" colspan="1" rowspan="1">K*GNFNYIEFTR</td>
<td align="left" colspan="1" rowspan="1">GNFNYIEFTR</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">High mobility group protein
B1</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/protein/P63158" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">P63158</a></td>
<td align="left" colspan="1" rowspan="1">K*HPDASVNFSEFSK</td>
<td align="left" colspan="1" rowspan="1">KHPDASVNFSEFSK</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Elongation factor 1-alpha 1</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/protein/P10126" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">P10126</a></td>
<td align="left" colspan="1" rowspan="1">SGK*KLEDGPK</td>
<td align="left" colspan="1" rowspan="1">THINIVVIGHVDSGK, KLEDGPK</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Elongation factor 2</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/protein/P58252" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">P58252</a></td>
<td align="left" colspan="1" rowspan="1">EDLYLK*PIQR</td>
<td align="left" colspan="1" rowspan="1">EDLYLKPIQR</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/tbl1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><p>We then conducted a pull-down experiment to validate
further that
MA-diyne could enrich malonylated proteins. The eluted protein from
the avidin beads in a separate experiment were resolved on the SDS-PAGE
followed by immunoblotting against antibodies for some already known
malonylated proteins, ALDOA (fructose-bisphosphate aldolase A) and GAPDH (glyceraldehyde-3-phosphate dehydrogenase). The detection of GAPDH and ALDOA by Western
blot analysis of the eluted proteins from the beads (<a href="#fig4" class="usa-link">Figure 
</a>A) confirmed that MA-diyne
can indeed enrich malonylated proteins. We could also pull down other
novel proteins in our proteomics list like Stat3 and β actin
that have not been reported before (<a href="#fig4" class="usa-link">Figure 
</a>B). Furthermore, we also experimented to
validate the chemoselectivity of MA-diyne by introducing a single
amino acid mutation in the GAPDH gene at position 184 by replacing
lysine amino acid with glutamate amino acid. The primary chondrocytes were transfected with wild type GAPDH as
well as GAPDH K184E plasmids, and then, the cells were incubated with
MA-diyne and the protein lysates were tagged with biotin azide via
a click reaction followed by avidin beads enrichment. The eluted proteins
from the avidin beads were subjected to Western blot and probed for
the anti-GAPDH antibody. <a href="#fig4" class="usa-link">Figure 
</a>C demonstrates that MA-diyne was unable to label GAPDH
with single amino acid mutation of K184E in comparison to wild GAPDH,
suggesting the chemoselectivity of MA-diyne to the lysine residue.</p>
<figure class="fig xbox font-sm" id="fig4"><h4 class="obj_head">4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371503_bg4c00151_0004.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d548/12371503/92ab96e79a59/bg4c00151_0004.jpg" loading="lazy" id="gr4" height="500" width="698" alt="4"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Validation
of lysine malonylated protein targets of MA-diyne. (A)
Pull down of known malonylated proteins such as GAPDH and Aldolase
A after enrichment using avidin beads. <em>n</em> = 2. (B)
Pull down of newly identified proteins after MA-diyne enrichment. <em>n</em> = 2. (C) Pull-down of MA-diyne labeled GAPDH protein after
overexpressing GAPDH wild type and GAPDH K184E plasmids in primary
chondrocytes. Primary chondrocytes were overexpressed with GAPDH wild
type and GAPDH K184E plasmids for 48 h and then labeled with MA-diyne.
The protein lysates were clicked with biotin azide, and avidin beads
were used to enrich the biotinylated proteins. The elute proteins
from the avidin beads were resolved on SDS Gel and then transferred
on a PVDF membrane. The blot was probed for anti-GAPDH antibodies.
(D) Pie chart depicting the percentage of malonylated proteins found
in different subcellular compartments by GO analysis. (E) Distribution
of enriched proteins based on their biological process by GO analysis,
(F) KEGG pathway analysis of the enriched proteins. <em>p</em> value &lt; 0.05.</p></figcaption></figure><p>Gene ontology (GO) and pathway analysis of the
identified proteins
revealed that the malonylated proteins are localized differentially
in various subcellular compartments including the cytoplasm, plasma
membranes, endoplasmic reticulum, nucleus, mitochondria, and Golgi
apparatus of the primary chondrocytes (<a href="#fig4" class="usa-link">Figure 
</a>D and <a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.4c00151/suppl_file/bg4c00151_si_002.xlsx" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Supplementary</a> excel 3). These findings align with similar subcellular localization
patterns of malonylated proteins observed in other cell types, such
as plants, prokaryotes, parasites, and mammalian cells. GO analysis based on the biological processes
revealed that the identified malonylated proteins belonged to processes
related to musculoskeletal tissues like collagen biosynthesis and
ossification. Additionally, it was revealed that malonylated proteins
are highly involved in glucose, amino acid, and pyruvate metabolic
and lipid transport processes. The identified proteins are also enriched
in biological processes like ribose phosphate metabolism, ATP production,
TCA cycle, glycolysis, and oxidative phosphorylation (<a href="#fig4" class="usa-link">Figure 
</a>E and <a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.4c00151/suppl_file/bg4c00151_si_002.xlsx" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Supplementary excel 4</a>). These findings are consistent with previous reports
on the involvement of lysine malonylation in regulation of metabolic
disorders like type 2 diabetes, cardiovascular
diseases, osteoarthritis,<em>
,
</em> etc. We also observed the role of malonylated proteins in the mechanisms
related to the quality control of proteins such as response to endoplasmic
reticulum stress, autophagy, and proteasome-mediated protein processing (<a href="#fig4" class="usa-link">Figure 
</a>E and <a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.4c00151/suppl_file/bg4c00151_si_002.xlsx" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Supplementary excel 5</a>). KEGG analysis of the
identified malonylated proteins revealed regulation of several signaling
pathways like VEGF, Wnt signaling, cGMP-PKG, PI3K-AKT, estrogen signaling,
EGFR tyrosine kinase inhibitor resistance, growth hormone synthesis
and secretion, phospholipase D signaling pathways, etc (<a href="#fig4" class="usa-link">Figure 
</a>F and <a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.4c00151/suppl_file/bg4c00151_si_002.xlsx" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Supplementary excel 5</a>).</p></section><section id="sec2.5"><h3 class="pmc_sec_title">2.5. MA-Diyne Can Detect Lysine Malonylation in
a Wide Range of Cell Types In Vitro</h3>
<p>To assess the applicability
of the MA-diyne probe in detecting malonylated proteins in different
types of cells, we analyzed the metabolic labeling profile of MA-diyne
in subcutaneous primary adipocytes (<a href="#fig5" class="usa-link">Figure 
</a>A) and HEK 293T cell lines (<a href="#fig5" class="usa-link">Figure 
</a>B). Both cells exhibited robust
MA-diyne signaling compared to the control, suggesting successful
metabolic incorporation of MA-diyne into cellular proteins. In addition,
we assessed whether MA-diyne can be used to metabolically label proteins
of . We incubated live with 1 mM MA-diyne for 6 h with constant
shaking and then lysed the worms to extract proteins. The proteins
were then conjugated with the IR680 azide. In-gel fluorescence analysis
revealed that MA-diyne resulted in a robust labeling of malonylated
proteins (<a href="#fig5" class="usa-link">Figure 
</a>C).
The route for the metabolic labeling is unclear, but it is assumed
that MA-diyne could have penetrated cells via either the esophageal route or epidermal absorption/passive
diffusion.</p>
<figure class="fig xbox font-sm" id="fig5"><h4 class="obj_head">5.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371503_bg4c00151_0005.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d548/12371503/5da78b05a82b/bg4c00151_0005.jpg" loading="lazy" id="gr5" height="517" width="721" alt="5"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>MA-diyne can detect lysine malonylated proteins in other cells
like primary subcutaneous adipocytes, HEK293T cells, and , <em>n</em> = 2.</p></figcaption></figure></section></section><section id="sec3"><h2 class="pmc_sec_title">3. Conclusions</h2>
<p>In summary, we have developed
a novel chemical probe MA-diyne for
identifying and quantifying the protein malonylation in primary chondrocytes
and several other types of cells. This probe was synthesized by adding
a propargyl group to the Meldrum acid for utilizing the conventional
chemoproteomic approach to pull down the malonylated proteins. MA-diyne
was observed to be readily uptaken into the cells, and after getting
acyclized to malonyl-CoA by the action of intracellular esterases,
it then metabolically labeled the proteins. It was also observed that
labeling by MA-diyne was dynamic and regulated by the demalonylase
enzyme, Sirt5. Moreover, the labeling by MA-diyne was observed to
be nonenzymatic and not dependent on the endogenous levels of malonyl
CoA. Quantitative proteomics could identify a significantly larger
number of proteins than our previous attempt with the Kmal PTMscan
antibody. Moreover, this further enables us to validate the enzymatic
function of several identified metabolic proteins and their role in
the progression of osteoarthritis.</p></section><section id="sec4"><h2 class="pmc_sec_title">4. Methods</h2>
<section id="sec4.1"><h3 class="pmc_sec_title">4.1. Preparation of MA-Diyne</h3>
<p>To a stirred
solution of Meldrum acid (1.0 g, 6.94 mmol, 1.0 equiv) in DMF (10
mL) was added DIPEA (6.04 mL, 34.7 mmol, 5.0 equiv) followed by propargyl
bromide (0.79 mL, 10.41 mmol, 1.5 equiv) at room temperature under
an inert atmosphere. The resulting reaction mixture was allowed to
stir at room temperature for 24 h. The reaction mixture was then diluted
with cold water (100 mL) and extracted with EtOAc (2 × 50 mL).
The combined organic extracts were washed with brine solution (100
mL), dried over anhydrous Na2SO<sub>4</sub>, filtered, and concentrated
under reduced pressure to obtain the crude. The crude was purified
by flash column chromatography [eluent: 1% MeOH in DCM] to afford
dialkynylated Meldrum acid (460 mg, 40% yield) as an off-white solid.</p></section><section id="sec4.2"><h3 class="pmc_sec_title">4.2. Isolation of Primary Chondrocytes and Culture</h3>
<p>Primary mouse chondrocytes were isolated from the articular cartilage
of the knee joints obtained from the ∼7 days-old C57BL-6 mice
according to the previously mentioned protocol. Briefly, the articular cartilage from the tibia and femur
part was extracted from the knee joints of ∼7 day-old C57BL-6
mice and dipped in a serum-free Dulbecco’s modified Eagle’s
medium (DMEM) (Life Technology, 10567014) solution at room temperature.
After thorough washes with 1× phosphate-buffered saline (PBS),
cartilage tissues were incubated in 3 mg/mL Collagenase D (Roche)
in serum-free DMEM solution for 45 min twice and then transferred
to overnight incubation in 0.5 mg/mL Collagenase D in DMEM solution
supplemented with 3% Liberase TL (Sigma). The next day, tissues were
homogenized by using pipettes to release and suspend the cells multiple
times. Finally, the homogenate was filtered through a 40 μm
strainer to remove large debris and resuspended in fresh DMEM supplemented
with 10% FBS and 1% pen–strep solution. The suspension of cells
was then plated in 60 mm cell culture dishes and allowed to expand
as passage zero cells at 37 °C and 5% CO<sub>2</sub>.</p>
<p>Other
cell lines such as primary mouse adipocytes and HEK293T were also
cultured in DMEM supplemented with 10% FBS and 1% pen–strep
solution.</p></section><section id="sec4.3"><h3 class="pmc_sec_title">4.3. Metabolic Labeling and Cell Lysis</h3>
<p>The cells were treated with MA-diyne as prescribed. The same volume
of DMSO was used as a negative control. After the stipulated incubation
time for metabolic labeling, cells were washed with ice-cold 1×
PBS three times and scraped using a plastic scraper in 1× PBS.
The cells were collected in a 1.5 mL centrifuge tube and centrifuged
at 3000 rpm. The cell pellet thus obtained was lysed using ice-cold
lysis buffer (1% NP 40, 150 mM NaCl, 50 mM <em>N</em>-(2-hydroxyethyl)­piperazine-<em>N</em>′-ethanesulfonic acid (HEPES), 2 mM MgCl<sub>2</sub>, 20% Glycerol pH 7.4, EDTA free protease inhibitor cocktail) for
15 min on ice, followed by centrifugation at 16,000<em>g</em> for 15 min at 4 °C. The supernatant was collected, and protein
concentration was estimated using a BCA protein estimation kit (Pierce).
The cell lysates were then diluted using dilution buffer (150 mM NaCl,
50 mM HEPES, 2 mM MgCl2, 20% glycerol pH 7.4, EDTA-free protease inhibitor
cocktail) to give the final concentration of 1.5 mg/mL for the click
reaction.</p></section><section id="sec4.4"><h3 class="pmc_sec_title">4.4. Copper­(I)-Catalyzed Alkyne–Azide Cycloaddition
Reaction (CuAAC)</h3>
<p>The procedure for the CuAAC click chemistry
reaction with protein lysates was performed as described in Yang et
al. 2010 with slight modification. Briefly,
a click reaction master mixture for the required volume of protein
lysates was prepared by mixing the reagents in the following order:
100 μM IR680 azide dye (click chemistry tools) for in-gel fluorescence/100
μM Biotin azide (Click chemistry tools) for streptavidin enrichment
and Western blotting, 100 μM tris­((1-benzyl-4-triazolyl)­methyl)­amine
(TBTA) (Click chemistry tools), 1 mM tris­(carboxyethyl)­phosphine (TCEP)
(Sigma), and 1 mM CuSO<sub>4</sub>. The protein lysates were incubated
with the click reaction master mix for 1.5 h at room temperature.</p></section><section id="sec4.5"><h3 class="pmc_sec_title">4.5. In-Gel Fluorescence Imaging of Proteins</h3>
<p>After the protein was incubated with the click reaction mixture
for 1.5 h, 4 volumes of ice-cold acetone were used to stop/quench
the reaction. The proteins were then incubated at −20 °C
overnight to precipitate the proteins. The protein acetone solution
was centrifuged at 6000<em>g</em> for 5 min at 4 °C.
The obtained pellet was washed with ice-cold methanol once and then
air-dried for 10 min. The protein pellets were solubilized in 1×
loading buffer with 100 mM DTT, heated at 95 °C for 10 min, and
then resolved in 10% SDS PAGE gel. The labeled proteins were visualized
by scanning the gel under the Odyssey CLx Imaging System (excitation
maxima-672 nm, emission maxima- 694 nm).</p></section><section id="sec4.6"><h3 class="pmc_sec_title">4.6. Pulse-Chase Experiment</h3>
<p>Wild-type
primary chondrocytes and Sirt5KO cells were isolated from the wild-type
or the Sirt5 conditional knockout (Sirt5-CKO-Aggrecan-Cre<sup>ERT2</sup>; Sirt5<sup>flox/flox</sup>) mice, respectively. The primary chondrocyte
cells (wild type and SirtKO) were treated with 4-hydroxy tamoxifen
for 48 h to induce the cre-lox system. The cells were labeled with
200 μM MA-diyne for 1 h. The medium was removed, and the cells
were washed thrice with 1× PBS to ensure removal of residual
MA-diyne. Fresh medium supplemented with Meldrum acid (200 μM)
was added to the cells. The cells were extracted at different time
points (0–5 h). The cells were then subjected to protein lysis
as described earlier, followed by the click reaction with IR680 azide.</p></section><section id="sec4.7"><h3 class="pmc_sec_title">4.7. Labeling the Cells for Confocal Imaging</h3>
<p>The cells were cultured on sterile coverslips in 6-well culture
dishes. The cells were treated with different concentrations of MA-diyne
for 6 h and washed with 1× PBS. The cells were then fixed using
ice-cold 4% paraformaldehyde solution for 20 min at 4 °C. The
cells on coverslips were then washed thoroughly with ice-cold 1×
PBS several times and permeabilized using 0.25% triton X-100 prepared
in 1× PBS for 25 min at room temperature. The cells were later
washed with 1× PBS, and then, 5% bovine serum albumin blocking
buffer was added to the wells and left for shaking at room temperature
for another 30 min. The cells were then incubated with the click reaction
mix containing 100 μM Carboxyrhodamine 110 Azide, 100 μM
TBTA, 1 mM TCEP, and 1 mM CuSO<sub>4</sub> in PBS for 1 h. Finally,
cells were washed with 1× PBS to remove the remaining staining,
and coverslips were mounted on clean glass slides with the DAPI-containing
mounting reagent. The image was collected in a Nikon A1R confocal
microscope at 60× magnification.</p></section><section id="sec4.8"><h3 class="pmc_sec_title">4.8. Measurement of Malonyl CoA in Cell Supernatant
Using a Malonyl CoA ELISA Kit</h3>
<p>The cells were treated with
MA-diyne for 2–4 h at different concentrations. After the incubation,
cells were trypsinized and resuspended in 1× PBS at a concentration
of 1 × 10<sup>6</sup> cells/mL. The cells were lyzed by repeated
freeze–thaw cycles and centrifuged at 2000–3000 rpm
for 20 min at 4 °C. The supernatant was collected to estimate
the malonyl CoA concentration using the mouse malonyl CoA ELISA kit
(MyBioSource). The instructions followed for the experiment were as
mentioned in the manufacturer’s protocol.</p></section><section id="sec4.9"><h3 class="pmc_sec_title">4.9. Pull down of Labeled Proteins after Streptavidin
Enrichment</h3>
<p>The cells were labeled with MA-diyne as instructed
above, and then, the protein lysates were conjugated with biotin azide
via the click reaction. The reaction was quenched using 4 volumes
of ice-cold acetone, and the solution was incubated at −20
°C overnight to precipitate the proteins. The proteins were pelleted
at 3000<em>g</em> for 5 min and washed with methanol thrice.
The protein pellet was air-dried for 10 min and solubilized in 100
μL of solubilizing buffer (4% SDS, 20 mM EDTA, and 20% glycerol
in 1× PBS) by vortexing and gentle heating. The solution was
diluted with 1× PBS to decrease the SDS concentration to 0.5%.
100 μL of prewashed (three times 0.2% SDS prepared in 1×
PBS) avidin agar beads was added to the above protein solution and
kept for incubation for 1.5 h at room temperature on a gyrator rocker.
The solution was centrifuged at 3000 rpm for 2 min, and the supernatant
was discarded. The beads were washed thoroughly as follows: 5 times
with 10 mL of 0.2% SDS in 1× PBS, 5 times with 10 mL of 1×
PBS, and finally 5 times with distilled water. The beads were boiled
in equal volume of 100 μL of elution buffer (loading buffer
+ 100 mM DTT) at 95 °C for 5 min, and the elute was loaded on
the SDS PAGE gel. The proteins were then transferred to the nitrocellulose
membrane and probed against antistreptavidin as well as the primary
antibody of choice.</p></section><section id="sec4.10"><h3 class="pmc_sec_title">4.10. Sample Preparation for LC–MS/MS</h3>
<p>The avidin beads bound to the biotinylated proteins were washed
as instructed above and stored at −80 °C before being
sent for the LC–MS/MS at Cell Signaling Technology, Inc. Proteomics
facility. The beads were washed again with HPLC water to prevent detergent
or protease inhibitor contamination in the mass spectrometer. The
beads were incubated with 9 M urea for 10 min at room temperature.
The cysteines were then subjected to reductive alkylation with dithiothreitol
(DTT) and iodoacetamide (IAA). Finally, the whole solution was diluted
with 3 volumes of 20 mM HEPES so that urea concentration was reduced
to less than 2 M. The beads were then subjected to on-bead trypsin
digestion overnight followed by the addition of 1% trifluoroacetyl
(final concentration). The peptides were then desalted using a stage
tip and eluted with 50% acetonitrile. Peptide concentration was measured
using a BCA assay, and samples were dried. The enriched peptides were
then subjected to bottom-up quantitative proteomics on an Orbitrap
Astral Instrument. Peptides were loaded directly onto an Aurora Ultimate
TS column (25 cm × 75 μm ID, 1.7 μm C18) packed with
the C18 reversed-phase resin. The column was developed with a 45 min
linear gradient of acetonitrile at 400 nL/min. “A” buffer
= 3% acetonitrile, 0.1% formic acid in water. “B” buffer
= 80% acetonitrile, 0.1% formic acid in water. Full MS parameter settings
are available upon request.</p></section><section id="sec4.11"><h3 class="pmc_sec_title">4.11. Identification of Target Proteins</h3>
<p>MS/MS spectra were evaluated by using Spectronaut software from Biognosys.
Hybrid DIA searches were performed by using factory default settings.
Searches were performed against the most recent update of the Uniprot database. Tryptic cleavage was required
with up to 4 missed cleavage sites per peptide allowed. Carbamidomethyl
modification on cysteine was kept as static modification, while oxidized
methionine and N-terminal acetylation were set as variable modifications
with a maximum of 5 modifications/peptide. All of the results were
filtered to a 1% false discovery rate at the precursor peptide level.</p></section><section id="sec4.12"><h3 class="pmc_sec_title">4.12. Bioinformatics</h3>
<p>Gene ontology analysis
(<a href="https://predictprotein.org/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://predictprotein.org/</a>) was used to identify the subcellular localization and biological
processes of the proteins enriched by MA-diyne. A Kyoto Encyclopedia
of Genes and Genomes (KEGG) pathway analysis was also conducted to
evaluate the pathways involved (<a href="https://www.genome.jp/kegg/pathway" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.genome.jp/kegg/pathway</a>).</p>
<p>Ethical approval for the animal experiments reported here
was obtained from the Institutional Animal Care and Use Committee
(IACUC) at Ohio University, and the IACUC protocol number is 20-H-014.</p></section></section><section id="sec22"><h2 class="pmc_sec_title">Supplementary Material</h2>
<section class="sm xbox font-sm" id="sifile1"><div class="media p"><div class="caption">
<a href="/articles/instance/12371503/bin/bg4c00151_si_001.pdf" data-ga-action="click_feat_suppl" class="usa-link">bg4c00151_si_001.pdf</a><sup> (496.7KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="sifile2"><div class="media p"><div class="caption">
<a href="/articles/instance/12371503/bin/bg4c00151_si_002.xlsx" data-ga-action="click_feat_suppl" class="usa-link">bg4c00151_si_002.xlsx</a><sup> (9.7MB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="sifile3"><div class="media p"><div class="caption">
<a href="/articles/instance/12371503/bin/bg4c00151_si_003.xlsx" data-ga-action="click_feat_suppl" class="usa-link">bg4c00151_si_003.xlsx</a><sup> (133.9KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="sifile4"><div class="media p"><div class="caption">
<a href="/articles/instance/12371503/bin/bg4c00151_si_004.xlsx" data-ga-action="click_feat_suppl" class="usa-link">bg4c00151_si_004.xlsx</a><sup> (8.9KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="sifile5"><div class="media p"><div class="caption">
<a href="/articles/instance/12371503/bin/bg4c00151_si_005.xlsx" data-ga-action="click_feat_suppl" class="usa-link">bg4c00151_si_005.xlsx</a><sup> (174.6KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="sifile6"><div class="media p"><div class="caption">
<a href="/articles/instance/12371503/bin/bg4c00151_si_006.xlsx" data-ga-action="click_feat_suppl" class="usa-link">bg4c00151_si_006.xlsx</a><sup> (53.7KB, xlsx) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgments</h2>
<p>We thank the following funding support: Hevolution
Foundation AGE award AGE-008 (S.Z., H.L.), National Institutes of
Health grant R01AR081804 (S.Z., H.L.), National Institutes of Health
grant R15AR080813 (S.Z., H.L.), Arthritis National Research Foundation,
American Society for Bone and Mineral Research FIRST award (S.Z.),
Rheumatology Research Foundation Innovative Award (S.Z.), and Osteopathic
Heritage Foundation Ralph S. Licklider, D.O. Endowed Professorship
(S.Z.). We thank the staff (including Tammy Mace, Angela Smith, Shawn
Rosensteel, and Scott Carpenter) at the animal facility in the Life
Science Building for their excellent care for our animals. We thank
Dr. Vishwajeet Puri, Dr. Craig Nunemaker, and Dr. Kevin Lee for providing
the cell lines. We also thank Dr. Nathaniel Szewczyk for helping us
conduct the labeling in .
We acknowledge the help from the Proteomics Core Facility at the Cell
Signaling Technology Company for their help with the proteomics assay.</p></section><section id="notes1"><p>The Supporting Information
is
available free of charge at <a href="https://pubs.acs.org/doi/10.1021/acsbiomedchemau.4c00151?goto=supporting-info" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://pubs.acs.org/doi/10.1021/acsbiomedchemau.4c00151</a>.</p>
<ul id="silist" class="list" style="list-style-type:none">
<li><p>1H NMR spectral data of the MA-diyne probe, 1H NMR full
spectrum of the MA-diyne probe, <sup>13</sup>C NMR spectrum of the
MA-diyne probe, mass spectrometry analysis of the MA-diyne probe,
summary of obtained fragments in mass spectrometry analysis, an image
of Western blot analysis probed for the anti-MaK antibody on the proteins
obtained from primary chondrocytes treated with or without ND-630,
metabolic labeling of proteins to validate the chemoselectivity of
MA-diyne, confocal microscopic fluorescence image of the cells treated
with different concentrations of MA-diyne after the click reaction
with Carboxyrhodamine 110 Azide, Relative intensity of MA-diyne labeling
in wild type and Sirt5 KD cells normalized to β actin after
the pulse-chase experiment with Meldrum acid at different time points,
and image for in-gel fluorescence depicting similar labeling when
the endogenous levels of malonyl CoA were changed (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.4c00151/suppl_file/bg4c00151_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PDF</a>)</p></li>
<li><p>PTMscan total proteome results
(<a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.4c00151/suppl_file/bg4c00151_si_002.xlsx" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">XLSX</a>)</p></li>
<li><p>Proteins
in MA-yne vs control and Sirt5KO + probe (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.4c00151/suppl_file/bg4c00151_si_003.xlsx" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">XLSX</a>)</p></li>
<li><p>Proteins identified (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.4c00151/suppl_file/bg4c00151_si_004.xlsx" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">XLSX</a>)</p></li>
<li><p>Gene ontology analysis
(<a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.4c00151/suppl_file/bg4c00151_si_005.xlsx" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">XLSX</a>)</p></li>
<li><p>KEGG
analysis (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.4c00151/suppl_file/bg4c00151_si_006.xlsx" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">XLSX</a>)</p></li>
</ul></section><section id="notes3"><h2>⊥.</h2>
<p>A.B. and
P.N. contributed equally to the work. CRediT: <strong>Anupama
Binoy</strong> conceptualization, data curation, formal analysis, investigation,
methodology, visualization, writing - original draft; <strong>Pandurangan
Nanjan</strong> data curation, investigation, software, validation,
writing - original draft; <strong>Kavya Chellamuthu</strong> data curation,
methodology; <strong>Huanhuan Liu</strong> investigation, writing - review
&amp; editing.</p></section><section id="notes2"><p>The authors
declare no competing financial interest.</p></section><section id="ref-list1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="ref-list1_sec2"><ol class="ref-list font-sm">
<li id="ref1">
<cite>Lee J. M., Hammarén H. M., Savitski M. M., Baek S. H.. Control of protein
stability by post-translational modifications. Nat. Commun. 2023;14:201. doi: 10.1038/s41467-023-35795-8.</cite> [<a href="https://doi.org/10.1038/s41467-023-35795-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9839724/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36639369/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Commun.&amp;title=Control%20of%20protein%20stability%20by%20post-translational%20modifications&amp;author=J.%20M.%20Lee&amp;author=H.%20M.%20Hammar%C3%A9n&amp;author=M.%20M.%20Savitski&amp;author=S.%20H.%20Baek&amp;volume=14&amp;publication_year=2023&amp;pages=201&amp;pmid=36639369&amp;doi=10.1038/s41467-023-35795-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref2">
<cite>Zhong Q.. et al. Protein posttranslational
modifications in health and diseases: Functions,
regulatory mechanisms, and therapeutic implications. MedComm. 2023;4:e261.  doi: 10.1002/mco2.261.</cite> [<a href="https://doi.org/10.1002/mco2.261" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10152985/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37143582/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=MedComm&amp;title=Protein%20posttranslational%20modifications%20in%20health%20and%20diseases:%20Functions,%20regulatory%20mechanisms,%20and%20therapeutic%20implications&amp;author=Q.%20Zhong&amp;volume=4&amp;publication_year=2023&amp;pages=e261&amp;pmid=37143582&amp;doi=10.1002/mco2.261&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref3">
<cite>Dutta H., Jain N.. Post-translational
modifications and their implications in cancer. Front. Oncol. 2023;13:1240115. doi: 10.3389/fonc.2023.1240115.</cite> [<a href="https://doi.org/10.3389/fonc.2023.1240115" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10546021/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37795435/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Front.%20Oncol.&amp;title=Post-translational%20modifications%20and%20their%20implications%20in%20cancer&amp;author=H.%20Dutta&amp;author=N.%20Jain&amp;volume=13&amp;publication_year=2023&amp;pages=1240115&amp;pmid=37795435&amp;doi=10.3389/fonc.2023.1240115&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref4">
<cite>Pan S., Chen R.. Pathological implication
of protein post-translational modifications
in cancer. Mol. Aspects Med. 2022;86:101097. doi: 10.1016/j.mam.2022.101097.</cite> [<a href="https://doi.org/10.1016/j.mam.2022.101097" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9378605/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35400524/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol.%20Aspects%20Med.&amp;title=Pathological%20implication%20of%20protein%20post-translational%20modifications%20in%20cancer&amp;author=S.%20Pan&amp;author=R.%20Chen&amp;volume=86&amp;publication_year=2022&amp;pages=101097&amp;pmid=35400524&amp;doi=10.1016/j.mam.2022.101097&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref5">
<cite>Zou L.. et al. Lysine
Malonylation and Its Links to Metabolism and Diseases. Aging Dis. 2023;14:84–98. doi: 10.14336/AD.2022.0711.</cite> [<a href="https://doi.org/10.14336/AD.2022.0711" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9937698/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36818560/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Aging%20Dis.&amp;title=Lysine%20Malonylation%20and%20Its%20Links%20to%20Metabolism%20and%20Diseases&amp;author=L.%20Zou&amp;volume=14&amp;publication_year=2023&amp;pages=84-98&amp;pmid=36818560&amp;doi=10.14336/AD.2022.0711&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref6">
<cite>Zhang R.. et al. Histone malonylation
is regulated by SIRT5 and KAT2A. iScience. 2023;26:106193. doi: 10.1016/j.isci.2023.106193.</cite> [<a href="https://doi.org/10.1016/j.isci.2023.106193" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9985052/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36879797/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=iScience&amp;title=Histone%20malonylation%20is%20regulated%20by%20SIRT5%20and%20KAT2A&amp;author=R.%20Zhang&amp;volume=26&amp;publication_year=2023&amp;pages=106193&amp;pmid=36879797&amp;doi=10.1016/j.isci.2023.106193&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref7">
<cite>Rardin M. J.. et al. SIRT5 regulates the mitochondrial lysine succinylome
and metabolic
networks. Cell Metab. 2013;18:920–933. doi: 10.1016/j.cmet.2013.11.013.</cite> [<a href="https://doi.org/10.1016/j.cmet.2013.11.013" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4105152/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24315375/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Metab.&amp;title=SIRT5%20regulates%20the%20mitochondrial%20lysine%20succinylome%20and%20metabolic%20networks&amp;author=M.%20J.%20Rardin&amp;volume=18&amp;publication_year=2013&amp;pages=920-933&amp;pmid=24315375&amp;doi=10.1016/j.cmet.2013.11.013&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref8">
<cite>Nishida Y.. et al. SIRT5 Regulates both Cytosolic and Mitochondrial Protein
Malonylation
with Glycolysis as a Major Target. Mol. Cell. 2015;59(2):321–332. doi: 10.1016/j.molcel.2015.05.022.</cite> [<a href="https://doi.org/10.1016/j.molcel.2015.05.022" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4571487/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26073543/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol.%20Cell&amp;title=SIRT5%20Regulates%20both%20Cytosolic%20and%20Mitochondrial%20Protein%20Malonylation%20with%20Glycolysis%20as%20a%20Major%20Target&amp;author=Y.%20Nishida&amp;volume=59&amp;issue=2&amp;publication_year=2015&amp;pages=321-332&amp;pmid=26073543&amp;doi=10.1016/j.molcel.2015.05.022&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref9">
<cite>Peng C., Lu Z., Xie Z., Cheng Z., Chen Y., Tan M., Luo H., Zhang Y., He W., Yang K.. et al. The first
identification of lysine malonylation substrates and its regulatory
enzyme. Mol. Cell. Proteomics. 2011;10:M111.012658. doi: 10.1074/mcp.M111.012658.</cite> [<a href="https://doi.org/10.1074/mcp.M111.012658" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3237090/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21908771/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol.%20Cell.%20Proteomics&amp;title=The%20first%20identification%20of%20lysine%20malonylation%20substrates%20and%20its%20regulatory%20enzyme&amp;author=C.%20Peng&amp;author=Z.%20Lu&amp;author=Z.%20Xie&amp;author=Z.%20Cheng&amp;author=Y.%20Chen&amp;volume=10&amp;publication_year=2011&amp;pages=M111.012658&amp;pmid=21908771&amp;doi=10.1074/mcp.M111.012658&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref10">
<cite>Du J.. et al. Sirt5 is a NAD-dependent protein lysine demalonylase and desuccinylase. Science. 2011;334:806–809. doi: 10.1126/science.1207861.</cite> [<a href="https://doi.org/10.1126/science.1207861" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3217313/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22076378/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Sirt5%20is%20a%20NAD-dependent%20protein%20lysine%20demalonylase%20and%20desuccinylase&amp;author=J.%20Du&amp;volume=334&amp;publication_year=2011&amp;pages=806-809&amp;pmid=22076378&amp;doi=10.1126/science.1207861&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref11">
<cite>Bock I.. et al. Detailed specificity
analysis of antibodies binding to modified histone
tails with peptide arrays. Epigenetics. 2011;6:256–263. doi: 10.4161/epi.6.2.13837.</cite> [<a href="https://doi.org/10.4161/epi.6.2.13837" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3230550/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20962581/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Epigenetics&amp;title=Detailed%20specificity%20analysis%20of%20antibodies%20binding%20to%20modified%20histone%20tails%20with%20peptide%20arrays&amp;author=I.%20Bock&amp;volume=6&amp;publication_year=2011&amp;pages=256-263&amp;pmid=20962581&amp;doi=10.4161/epi.6.2.13837&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref12">
<cite>Sletten E. M., Bertozzi C. R.. Bioorthogonal chemistry: fishing
for selectivity in
a sea of functionality. Angew. Chem., Int. Ed.
Engl. 2009;48:6974–6998. doi: 10.1002/anie.200900942.</cite> [<a href="https://doi.org/10.1002/anie.200900942" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2864149/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19714693/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Angew.%20Chem.,%20Int.%20Ed.%20Engl.&amp;title=Bioorthogonal%20chemistry:%20fishing%20for%20selectivity%20in%20a%20sea%20of%20functionality&amp;author=E.%20M.%20Sletten&amp;author=C.%20R.%20Bertozzi&amp;volume=48&amp;publication_year=2009&amp;pages=6974-6998&amp;pmid=19714693&amp;doi=10.1002/anie.200900942&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref13">
<cite>Gong X., Feng Y., Tang H.. A metabolic
labeling protocol to
enrich myristoylated proteins from Caenorhabditis elegans. STAR Protoc. 2021;2:101013. doi: 10.1016/j.xpro.2021.101013.</cite> [<a href="https://doi.org/10.1016/j.xpro.2021.101013" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8666704/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34917984/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=STAR%20Protoc.&amp;title=A%20metabolic%20labeling%20protocol%20to%20enrich%20myristoylated%20proteins%20from%20Caenorhabditis%20elegans&amp;author=X.%20Gong&amp;author=Y.%20Feng&amp;author=H.%20Tang&amp;volume=2&amp;publication_year=2021&amp;pages=101013&amp;pmid=34917984&amp;doi=10.1016/j.xpro.2021.101013&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref14">
<cite>Heal W. P., Wright M. H., Thinon E., Tate E. W.. Multifunctional
protein labeling via enzymatic N-terminal tagging and elaboration
by click chemistry. Nat. Protoc. 2012;7:105–117. doi: 10.1038/nprot.2011.425.</cite> [<a href="https://doi.org/10.1038/nprot.2011.425" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22193303/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Protoc.&amp;title=Multifunctional%20protein%20labeling%20via%20enzymatic%20N-terminal%20tagging%20and%20elaboration%20by%20click%20chemistry&amp;author=W.%20P.%20Heal&amp;author=M.%20H.%20Wright&amp;author=E.%20Thinon&amp;author=E.%20W.%20Tate&amp;volume=7&amp;publication_year=2012&amp;pages=105-117&amp;pmid=22193303&amp;doi=10.1038/nprot.2011.425&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref15">
<cite>Ibrahim L.. et al. Succinylation of a KEAP1 sensor lysine promotes NRF2 activation. Cell Chem. Biol. 2023;30:1295–1302. doi: 10.1016/j.chembiol.2023.07.014.</cite> [<a href="https://doi.org/10.1016/j.chembiol.2023.07.014" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10592117/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37619563/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Chem.%20Biol.&amp;title=Succinylation%20of%20a%20KEAP1%20sensor%20lysine%20promotes%20NRF2%20activation&amp;author=L.%20Ibrahim&amp;volume=30&amp;publication_year=2023&amp;pages=1295-1302&amp;pmid=37619563&amp;doi=10.1016/j.chembiol.2023.07.014&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref16">
<cite>Umezawa K., Tsumoto H., Kawakami K., Miura Y.. A chemical probe for
proteomic analysis and visualization of intracellular localization
of lysine-succinylated proteins. Analyst. 2022;148:95–104. doi: 10.1039/D2AN01370C.</cite> [<a href="https://doi.org/10.1039/D2AN01370C" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36468704/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Analyst&amp;title=A%20chemical%20probe%20for%20proteomic%20analysis%20and%20visualization%20of%20intracellular%20localization%20of%20lysine-succinylated%20proteins&amp;author=K.%20Umezawa&amp;author=H.%20Tsumoto&amp;author=K.%20Kawakami&amp;author=Y.%20Miura&amp;volume=148&amp;publication_year=2022&amp;pages=95-104&amp;pmid=36468704&amp;doi=10.1039/D2AN01370C&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref17">
<cite>Afonso C. F.. et al. Cysteine-Assisted Click-Chemistry for Proximity-Driven, Site-Specific
Acetylation of Histones. Angew. Chem., Int.
Ed. Engl. 2022;61:e202208543.  doi: 10.1002/anie.202208543.</cite> [<a href="https://doi.org/10.1002/anie.202208543" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9828500/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36124857/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Angew.%20Chem.,%20Int.%20Ed.%20Engl.&amp;title=Cysteine-Assisted%20Click-Chemistry%20for%20Proximity-Driven,%20Site-Specific%20Acetylation%20of%20Histones&amp;author=C.%20F.%20Afonso&amp;volume=61&amp;publication_year=2022&amp;pages=e202208543&amp;pmid=36124857&amp;doi=10.1002/anie.202208543&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref18">
<cite>Yang Y. Y., Ascano J. M., Hang H. C.. Bioorthogonal
chemical reporters
for monitoring protein acetylation. J. Am. Chem.
Soc. 2010;132:3640–3641. doi: 10.1021/ja908871t.</cite> [<a href="https://doi.org/10.1021/ja908871t" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2848987/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20192265/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Am.%20Chem.%20Soc.&amp;title=Bioorthogonal%20chemical%20reporters%20for%20monitoring%20protein%20acetylation&amp;author=Y.%20Y.%20Yang&amp;author=J.%20M.%20Ascano&amp;author=H.%20C.%20Hang&amp;volume=132&amp;publication_year=2010&amp;pages=3640-3641&amp;pmid=20192265&amp;doi=10.1021/ja908871t&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref19">
<cite>Zhang X., Zhang Y.. Applications of azide-based
bioorthogonal click chemistry in glycobiology. Molecules. 2013;18:7145–7159. doi: 10.3390/molecules18067145.</cite> [<a href="https://doi.org/10.3390/molecules18067145" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6269833/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23783454/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecules&amp;title=Applications%20of%20azide-based%20bioorthogonal%20click%20chemistry%20in%20glycobiology&amp;author=X.%20Zhang&amp;author=Y.%20Zhang&amp;volume=18&amp;publication_year=2013&amp;pages=7145-7159&amp;pmid=23783454&amp;doi=10.3390/molecules18067145&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref20">
<cite>Jiang H.. et al. Monitoring dynamic glycosylation
in vivo using supersensitive click
chemistry. Bioconjug. Chem. 2014;25:698–706. doi: 10.1021/bc400502d.</cite> [<a href="https://doi.org/10.1021/bc400502d" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3993875/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24499412/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioconjug.%20Chem.&amp;title=Monitoring%20dynamic%20glycosylation%20in%20vivo%20using%20supersensitive%20click%20chemistry&amp;author=H.%20Jiang&amp;volume=25&amp;publication_year=2014&amp;pages=698-706&amp;pmid=24499412&amp;doi=10.1021/bc400502d&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref21">
<cite>Bao X., Zhao Q., Yang T., Fung Y. M., Li X. D.. A chemical
probe for lysine malonylation. Angew. Chem.,
Int. Ed. Engl. 2013;52:4883–4886. doi: 10.1002/anie.201300252.</cite> [<a href="https://doi.org/10.1002/anie.201300252" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23533089/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Angew.%20Chem.,%20Int.%20Ed.%20Engl.&amp;title=A%20chemical%20probe%20for%20lysine%20malonylation&amp;author=X.%20Bao&amp;author=Q.%20Zhao&amp;author=T.%20Yang&amp;author=Y.%20M.%20Fung&amp;author=X.%20D.%20Li&amp;volume=52&amp;publication_year=2013&amp;pages=4883-4886&amp;pmid=23533089&amp;doi=10.1002/anie.201300252&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref22">
<cite>Kim J., Kang E.-H., Choi T.-L.. Cyclopolymerization
To Synthesize
Conjugated Polymers Containing Meldrum’s Acid as a Precursor
for Ketene Functionality. ACS Macro Lett. 2012;1:1090–1093. doi: 10.1021/mz300250b.</cite> [<a href="https://doi.org/10.1021/mz300250b" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35607044/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=ACS%20Macro%20Lett.&amp;title=Cyclopolymerization%20To%20Synthesize%20Conjugated%20Polymers%20Containing%20Meldrum%E2%80%99s%20Acid%20as%20a%20Precursor%20for%20Ketene%20Functionality&amp;author=J.%20Kim&amp;author=E.-H.%20Kang&amp;author=T.-L.%20Choi&amp;volume=1&amp;publication_year=2012&amp;pages=1090-1093&amp;pmid=35607044&amp;doi=10.1021/mz300250b&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref23">
<cite>Jin S.-H., Cho H.-N., Choi S.-K.. Cyclopolymerization of dipropargyl
isopropylidene malonate and characterization of the product. J. Polym. Sci., Part A: Polym. Chem. 1993;31:69–74. doi: 10.1002/pola.1993.080310108.</cite> [<a href="https://doi.org/10.1002/pola.1993.080310108" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Polym.%20Sci.,%20Part%20A:%20Polym.%20Chem.&amp;title=Cyclopolymerization%20of%20dipropargyl%20isopropylidene%20malonate%20and%20characterization%20of%20the%20product&amp;author=S.-H.%20Jin&amp;author=H.-N.%20Cho&amp;author=S.-K.%20Choi&amp;volume=31&amp;publication_year=1993&amp;pages=69-74&amp;doi=10.1002/pola.1993.080310108&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref24">
<cite>Wang C. K., Swedberg J. E., Harvey P. J., Kaas Q., Craik D. J.. Conformational
Flexibility Is a Determinant of Permeability for Cyclosporin. J. Phys. Chem. B. 2018;122:2261–2276. doi: 10.1021/acs.jpcb.7b12419.</cite> [<a href="https://doi.org/10.1021/acs.jpcb.7b12419" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29400464/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Phys.%20Chem.%20B&amp;title=Conformational%20Flexibility%20Is%20a%20Determinant%20of%20Permeability%20for%20Cyclosporin&amp;author=C.%20K.%20Wang&amp;author=J.%20E.%20Swedberg&amp;author=P.%20J.%20Harvey&amp;author=Q.%20Kaas&amp;author=D.%20J.%20Craik&amp;volume=122&amp;publication_year=2018&amp;pages=2261-2276&amp;pmid=29400464&amp;doi=10.1021/acs.jpcb.7b12419&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref25">
<cite>Gudmundsson O. S.. et al. Coumarinic acid-based cyclic prodrugs of opioid peptides that exhibit
metabolic stability to peptidases and excellent cellular permeability. Pharm. Res. 1999;16:7–15. doi: 10.1023/A:1018828207920.</cite> [<a href="https://doi.org/10.1023/A:1018828207920" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9950272/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Pharm.%20Res.&amp;title=Coumarinic%20acid-based%20cyclic%20prodrugs%20of%20opioid%20peptides%20that%20exhibit%20metabolic%20stability%20to%20peptidases%20and%20excellent%20cellular%20permeability&amp;author=O.%20S.%20Gudmundsson&amp;volume=16&amp;publication_year=1999&amp;pages=7-15&amp;pmid=9950272&amp;doi=10.1023/A:1018828207920&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref26">
<cite>Dougherty P. G., Sahni A., Pei D.. Understanding
Cell Penetration of
Cyclic Peptides. Chem. Rev. 2019;119:10241–10287. doi: 10.1021/acs.chemrev.9b00008.</cite> [<a href="https://doi.org/10.1021/acs.chemrev.9b00008" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6739158/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31083977/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Chem.%20Rev.&amp;title=Understanding%20Cell%20Penetration%20of%20Cyclic%20Peptides&amp;author=P.%20G.%20Dougherty&amp;author=A.%20Sahni&amp;author=D.%20Pei&amp;volume=119&amp;publication_year=2019&amp;pages=10241-10287&amp;pmid=31083977&amp;doi=10.1021/acs.chemrev.9b00008&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref27">
<cite>Liu H., Zheng G., Treber M., Dai J., Woldegiorgis G.. Cysteine-scanning
mutagenesis of muscle carnitine palmitoyltransferase I reveals a single
cysteine residue (Cys-305) is important for catalysis. J. Biol. Chem. 2005;280:4524–4531. doi: 10.1074/jbc.M400893200.</cite> [<a href="https://doi.org/10.1074/jbc.M400893200" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15579906/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Biol.%20Chem.&amp;title=Cysteine-scanning%20mutagenesis%20of%20muscle%20carnitine%20palmitoyltransferase%20I%20reveals%20a%20single%20cysteine%20residue%20(Cys-305)%20is%20important%20for%20catalysis&amp;author=H.%20Liu&amp;author=G.%20Zheng&amp;author=M.%20Treber&amp;author=J.%20Dai&amp;author=G.%20Woldegiorgis&amp;volume=280&amp;publication_year=2005&amp;pages=4524-4531&amp;pmid=15579906&amp;doi=10.1074/jbc.M400893200&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref28">
<cite>Tseng C. C., McLoughlin S. M., Kelleher N. L., Walsh C. T.. Role of the active
site cysteine of DpgA, a bacterial type III polyketide synthase. Biochemistry. 2004;43:970–980. doi: 10.1021/bi035714b.</cite> [<a href="https://doi.org/10.1021/bi035714b" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/14744141/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biochemistry&amp;title=Role%20of%20the%20active%20site%20cysteine%20of%20DpgA,%20a%20bacterial%20type%20III%20polyketide%20synthase&amp;author=C.%20C.%20Tseng&amp;author=S.%20M.%20McLoughlin&amp;author=N.%20L.%20Kelleher&amp;author=C.%20T.%20Walsh&amp;volume=43&amp;publication_year=2004&amp;pages=970-980&amp;pmid=14744141&amp;doi=10.1021/bi035714b&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref29">
<cite>Zhang Z.. et al. Identification of lysine
succinylation as a new post-translational
modification. Nat. Chem. Biol. 2011;7:58–63. doi: 10.1038/nchembio.495.</cite> [<a href="https://doi.org/10.1038/nchembio.495" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3065206/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21151122/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Chem.%20Biol.&amp;title=Identification%20of%20lysine%20succinylation%20as%20a%20new%20post-translational%20modification&amp;author=Z.%20Zhang&amp;volume=7&amp;publication_year=2011&amp;pages=58-63&amp;pmid=21151122&amp;doi=10.1038/nchembio.495&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref30">
<cite>Wang Z. A., Cole P. A.. The Chemical Biology of Reversible Lysine Post-translational
Modifications. Cell Chem. Biol. 2020;27:953–969. doi: 10.1016/j.chembiol.2020.07.002.</cite> [<a href="https://doi.org/10.1016/j.chembiol.2020.07.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7487139/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32698016/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Chem.%20Biol.&amp;title=The%20Chemical%20Biology%20of%20Reversible%20Lysine%20Post-translational%20Modifications&amp;author=Z.%20A.%20Wang&amp;author=P.%20A.%20Cole&amp;volume=27&amp;publication_year=2020&amp;pages=953-969&amp;pmid=32698016&amp;doi=10.1016/j.chembiol.2020.07.002&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref31">
<cite>Wang Y., Chen H., Zha X.. Overview of
SIRT5 as a potential
therapeutic target: Structure, function and inhibitors. Eur. J. Med. Chem. 2022;236:114363. doi: 10.1016/j.ejmech.2022.114363.</cite> [<a href="https://doi.org/10.1016/j.ejmech.2022.114363" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35436671/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Eur.%20J.%20Med.%20Chem.&amp;title=Overview%20of%20SIRT5%20as%20a%20potential%20therapeutic%20target:%20Structure,%20function%20and%20inhibitors&amp;author=Y.%20Wang&amp;author=H.%20Chen&amp;author=X.%20Zha&amp;volume=236&amp;publication_year=2022&amp;pages=114363&amp;pmid=35436671&amp;doi=10.1016/j.ejmech.2022.114363&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref32">
<cite>Zhu S., Batushansky A., Jopkiewicz A., Makosa D., Humphries K. M., Van Remmen H., Griffin T. M.. Sirt5 Deficiency Causes Posttranslational
Protein Malonylation and Dysregulated Cellular Metabolism in Chondrocytes
Under Obesity Conditions. Cartilage. 2021;13:1185s–1199s. doi: 10.1177/1947603521993209.</cite> [<a href="https://doi.org/10.1177/1947603521993209" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8804736/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33567897/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cartilage&amp;title=Sirt5%20Deficiency%20Causes%20Posttranslational%20Protein%20Malonylation%20and%20Dysregulated%20Cellular%20Metabolism%20in%20Chondrocytes%20Under%20Obesity%20Conditions&amp;author=S.%20Zhu&amp;author=A.%20Batushansky&amp;author=A.%20Jopkiewicz&amp;author=D.%20Makosa&amp;author=K.%20M.%20Humphries&amp;volume=13&amp;publication_year=2021&amp;pages=1185s-1199s&amp;pmid=33567897&amp;doi=10.1177/1947603521993209&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref33">
<cite>Liu H.. et al. Cellular carbon stress is a mediator of obesity-associated
osteoarthritis
development. Osteoarthritis Cartilage. 2021;29:1346–1350. doi: 10.1016/j.joca.2021.04.016.</cite> [<a href="https://doi.org/10.1016/j.joca.2021.04.016" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8373780/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33984464/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Osteoarthritis%20Cartilage&amp;title=Cellular%20carbon%20stress%20is%20a%20mediator%20of%20obesity-associated%20osteoarthritis%20development&amp;author=H.%20Liu&amp;volume=29&amp;publication_year=2021&amp;pages=1346-1350&amp;pmid=33984464&amp;doi=10.1016/j.joca.2021.04.016&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref34">
<cite>Liu H., Issa D. D., Zhu S.. Sirt5 deficiency
causes chondrocyte
metabolic dysfunction and osteoarthritis during aging. Osteoarthritis Cartilage. 2022;30:S330–S331. doi: 10.1016/j.joca.2022.02.444.</cite> [<a href="https://doi.org/10.1016/j.joca.2022.02.444" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Osteoarthritis%20Cartilage&amp;title=Sirt5%20deficiency%20causes%20chondrocyte%20metabolic%20dysfunction%20and%20osteoarthritis%20during%20aging&amp;author=H.%20Liu&amp;author=D.%20D.%20Issa&amp;author=S.%20Zhu&amp;volume=30&amp;publication_year=2022&amp;pages=S330-S331&amp;doi=10.1016/j.joca.2022.02.444&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref35">
<cite>Liu H., Binoy A., Ren S., Martino T. C., Miller A. E., Willis C. R. G., Veerabhadraiah S. R., Bons J., Rose J. P., Schilling B.. et al. Sirt5 regulates chondrocyte metabolism and
osteoarthritis development through protein lysine malonylation. Arthritis Rheumatol. 2025 doi: 10.1002/art.43164.</cite> [<a href="https://doi.org/10.1002/art.43164" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC12354001/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40176311/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Arthritis%20Rheumatol.&amp;title=Sirt5%20regulates%20chondrocyte%20metabolism%20and%20osteoarthritis%20development%20through%20protein%20lysine%20malonylation&amp;author=H.%20Liu&amp;author=A.%20Binoy&amp;author=S.%20Ren&amp;author=T.%20C.%20Martino&amp;author=A.%20E.%20Miller&amp;publication_year=2025&amp;pmid=40176311&amp;doi=10.1002/art.43164&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref36">
<cite>Galván-Peña S.. et al. Malonylation
of GAPDH is an inflammatory signal in macrophages. Nat. Commun. 2019;10:338. doi: 10.1038/s41467-018-08187-6.</cite> [<a href="https://doi.org/10.1038/s41467-018-08187-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6338787/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30659183/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Commun.&amp;title=Malonylation%20of%20GAPDH%20is%20an%20inflammatory%20signal%20in%20macrophages&amp;author=S.%20Galv%C3%A1n-Pe%C3%B1a&amp;volume=10&amp;publication_year=2019&amp;pages=338&amp;pmid=30659183&amp;doi=10.1038/s41467-018-08187-6&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref37">
<cite>Xu M., Tian X., Ku T., Wang G., Zhang E.. Global Identification
and Systematic Analysis of Lysine Malonylation in Maize (Zea mays
L.) Front. Plant Sci. 2021;12:728338. doi: 10.3389/fpls.2021.728338.</cite> [<a href="https://doi.org/10.3389/fpls.2021.728338" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8417889/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34490025/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Front.%20Plant%20Sci.&amp;title=Global%20Identification%20and%20Systematic%20Analysis%20of%20Lysine%20Malonylation%20in%20Maize%20(Zea%20mays%20L.)&amp;author=M.%20Xu&amp;author=X.%20Tian&amp;author=T.%20Ku&amp;author=G.%20Wang&amp;author=E.%20Zhang&amp;volume=12&amp;publication_year=2021&amp;pages=728338&amp;pmid=34490025&amp;doi=10.3389/fpls.2021.728338&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref38">
<cite>Li Z., Wu Q., Zhang Y., Zhou X., Peng X.. Systematic analysis
of lysine malonylation in Streptococcus mutans. Front. Cell Infect. Microbiol. 2022;12:1078572. doi: 10.3389/fcimb.2022.1078572.</cite> [<a href="https://doi.org/10.3389/fcimb.2022.1078572" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9742479/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36519128/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Front.%20Cell%20Infect.%20Microbiol.&amp;title=Systematic%20analysis%20of%20lysine%20malonylation%20in%20Streptococcus%20mutans&amp;author=Z.%20Li&amp;author=Q.%20Wu&amp;author=Y.%20Zhang&amp;author=X.%20Zhou&amp;author=X.%20Peng&amp;volume=12&amp;publication_year=2022&amp;pages=1078572&amp;pmid=36519128&amp;doi=10.3389/fcimb.2022.1078572&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref39">
<cite>Nie L. B.. et al. Global profiling of
protein lysine malonylation in Toxoplasma gondii
strains of different virulence and genetic backgrounds. PLoS Neglected Trop. Dis. 2022;16:e0010431.  doi: 10.1371/journal.pntd.0010431.</cite> [<a href="https://doi.org/10.1371/journal.pntd.0010431" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9135328/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35576189/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20Neglected%20Trop.%20Dis.&amp;title=Global%20profiling%20of%20protein%20lysine%20malonylation%20in%20Toxoplasma%20gondii%20strains%20of%20different%20virulence%20and%20genetic%20backgrounds&amp;author=L.%20B.%20Nie&amp;volume=16&amp;publication_year=2022&amp;pages=e0010431&amp;pmid=35576189&amp;doi=10.1371/journal.pntd.0010431&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref40">
<cite>Du Y.. et al. Lysine Malonylation Is Elevated in Type 2 Diabetic
Mouse Models and
Enriched in Metabolic Associated Proteins. Mol.
Cell. Proteomics. 2015;14:227–236. doi: 10.1074/mcp.M114.041947.</cite> [<a href="https://doi.org/10.1074/mcp.M114.041947" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4288257/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25418362/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol.%20Cell.%20Proteomics&amp;title=Lysine%20Malonylation%20Is%20Elevated%20in%20Type%202%20Diabetic%20Mouse%20Models%20and%20Enriched%20in%20Metabolic%20Associated%20Proteins&amp;author=Y.%20Du&amp;volume=14&amp;publication_year=2015&amp;pages=227-236&amp;pmid=25418362&amp;doi=10.1074/mcp.M114.041947&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref41">
<cite>Wu L. F.. et al. Global profiling of
protein lysine malonylation in mouse cardiac
hypertrophy. J. Proteomics. 2022;266:104667. doi: 10.1016/j.jprot.2022.104667.</cite> [<a href="https://doi.org/10.1016/j.jprot.2022.104667" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35788409/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Proteomics&amp;title=Global%20profiling%20of%20protein%20lysine%20malonylation%20in%20mouse%20cardiac%20hypertrophy&amp;author=L.%20F.%20Wu&amp;volume=266&amp;publication_year=2022&amp;pages=104667&amp;pmid=35788409&amp;doi=10.1016/j.jprot.2022.104667&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref42">
<cite>Zou L., Yang Y., Wang Z., Fu X., He X., Song J., Li T., Ma H., Yu T.. Lysine Malonylation
and Its Links to Metabolism and Diseases. Aging
Dis. 2023;14:84–98. doi: 10.14336/ad.2022.0711.</cite> [<a href="https://doi.org/10.14336/ad.2022.0711" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9937698/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36818560/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Aging%20Dis.&amp;title=Lysine%20Malonylation%20and%20Its%20Links%20to%20Metabolism%20and%20Diseases&amp;author=L.%20Zou&amp;author=Y.%20Yang&amp;author=Z.%20Wang&amp;author=X.%20Fu&amp;author=X.%20He&amp;volume=14&amp;publication_year=2023&amp;pages=84-98&amp;pmid=36818560&amp;doi=10.14336/ad.2022.0711&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref43">
<cite>Rowland L. A.. et al. De novo lipogenesis
fuels adipocyte autophagosome and lysosome membrane
dynamics. Nat. Commun. 2023;14:1362. doi: 10.1038/s41467-023-37016-8.</cite> [<a href="https://doi.org/10.1038/s41467-023-37016-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10011520/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36914626/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Commun.&amp;title=De%20novo%20lipogenesis%20fuels%20adipocyte%20autophagosome%20and%20lysosome%20membrane%20dynamics&amp;author=L.%20A.%20Rowland&amp;volume=14&amp;publication_year=2023&amp;pages=1362&amp;pmid=36914626&amp;doi=10.1038/s41467-023-37016-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref44">
<cite>Gosset M., Berenbaum F., Thirion S., Jacques C.. Primary culture and
phenotyping of murine chondrocytes. Nat. Protoc. 2008;3:1253–1260. doi: 10.1038/nprot.2008.95.</cite> [<a href="https://doi.org/10.1038/nprot.2008.95" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18714293/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Protoc.&amp;title=Primary%20culture%20and%20phenotyping%20of%20murine%20chondrocytes&amp;author=M.%20Gosset&amp;author=F.%20Berenbaum&amp;author=S.%20Thirion&amp;author=C.%20Jacques&amp;volume=3&amp;publication_year=2008&amp;pages=1253-1260&amp;pmid=18714293&amp;doi=10.1038/nprot.2008.95&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref45">
<cite>Yang Y. Y., Grammel M., Raghavan A. S., Charron G., Hang H. C.. Comparative
analysis of cleavable azobenzene-based affinity tags for bioorthogonal
chemical proteomics. Chem. Biol. 2010;17:1212–1222. doi: 10.1016/j.chembiol.2010.09.012.</cite> [<a href="https://doi.org/10.1016/j.chembiol.2010.09.012" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3103785/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21095571/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Chem.%20Biol.&amp;title=Comparative%20analysis%20of%20cleavable%20azobenzene-based%20affinity%20tags%20for%20bioorthogonal%20chemical%20proteomics&amp;author=Y.%20Y.%20Yang&amp;author=M.%20Grammel&amp;author=A.%20S.%20Raghavan&amp;author=G.%20Charron&amp;author=H.%20C.%20Hang&amp;volume=17&amp;publication_year=2010&amp;pages=1212-1222&amp;pmid=21095571&amp;doi=10.1016/j.chembiol.2010.09.012&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ol></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12371503/bin/bg4c00151_si_001.pdf" data-ga-action="click_feat_suppl" class="usa-link">bg4c00151_si_001.pdf</a><sup> (496.7KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12371503/bin/bg4c00151_si_002.xlsx" data-ga-action="click_feat_suppl" class="usa-link">bg4c00151_si_002.xlsx</a><sup> (9.7MB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12371503/bin/bg4c00151_si_003.xlsx" data-ga-action="click_feat_suppl" class="usa-link">bg4c00151_si_003.xlsx</a><sup> (133.9KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material4_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12371503/bin/bg4c00151_si_004.xlsx" data-ga-action="click_feat_suppl" class="usa-link">bg4c00151_si_004.xlsx</a><sup> (8.9KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material5_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12371503/bin/bg4c00151_si_005.xlsx" data-ga-action="click_feat_suppl" class="usa-link">bg4c00151_si_005.xlsx</a><sup> (174.6KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material6_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12371503/bin/bg4c00151_si_006.xlsx" data-ga-action="click_feat_suppl" class="usa-link">bg4c00151_si_006.xlsx</a><sup> (53.7KB, xlsx) </sup>
</div></div></section></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from ACS Bio &amp; Med Chem Au are provided here courtesy of <strong>American Chemical Society</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1021/acsbiomedchemau.4c00151"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/bg4c00151.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (6.2 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12371503/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12371503/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12371503%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371503/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12371503/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12371503/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12371503/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12371503/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12371503/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="ywX6e0JrmXn8qwbiFY0hf2NgAq1QYvk4rwB4xvEc1ZIpQAlZg7sJsY8o64oP6Hor">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
