
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Kinase activity of DYRK family members is required for regulating primary cilium length, stability and morphology - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="AE5329518AF191E30529510072F77196.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="commbiol">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371027/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Communications Biology">
<meta name="citation_title" content="Kinase activity of DYRK family members is required for regulating primary cilium length, stability and morphology">
<meta name="citation_author" content="Melis D Arslanhan">
<meta name="citation_author_institution" content="Department of Molecular Biology and Genetics, Koç University, Istanbul, Turkey">
<meta name="citation_author" content="Ebru Topçu">
<meta name="citation_author_institution" content="Department of Molecular Biology and Genetics, Koç University, Istanbul, Turkey">
<meta name="citation_author" content="Elif Nur Firat-Karalar">
<meta name="citation_author_institution" content="Department of Molecular Biology and Genetics, Koç University, Istanbul, Turkey">
<meta name="citation_author_institution" content="School of Medicine, Koç University, Istanbul, Turkey">
<meta name="citation_publication_date" content="2025 Aug 21">
<meta name="citation_volume" content="8">
<meta name="citation_firstpage" content="1258">
<meta name="citation_doi" content="10.1038/s42003-025-08373-5">
<meta name="citation_pmid" content="40841441">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371027/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371027/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371027/pdf/42003_2025_Article_8373.pdf">
<meta name="description" content="The dual-specificity tyrosine-phoshorylation-regulated kinase (DYRK) family are multifunctional enzymes crucial for diverse cellular processes, including signaling through the primary cilium. Their dysregulation has been implicated in various ...">
<meta name="og:title" content="Kinase activity of DYRK family members is required for regulating primary cilium length, stability and morphology">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="The dual-specificity tyrosine-phoshorylation-regulated kinase (DYRK) family are multifunctional enzymes crucial for diverse cellular processes, including signaling through the primary cilium. Their dysregulation has been implicated in various ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371027/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12371027">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1038/s42003-025-08373-5"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/42003_2025_Article_8373.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12371027%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12371027/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12371027/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371027/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-commbiol.jpg" alt="Communications Biology logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Communications Biology" title="Link to Communications Biology" shape="default" href="https://www.nature.com/commsbio/" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Commun Biol</button></div>. 2025 Aug 21;8:1258. doi: <a href="https://doi.org/10.1038/s42003-025-08373-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1038/s42003-025-08373-5</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Commun%20Biol%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Commun%20Biol%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Commun%20Biol%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Commun%20Biol%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Kinase activity of DYRK family members is required for regulating primary cilium length, stability and morphology</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Arslanhan%20MD%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Melis D Arslanhan</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Melis D Arslanhan</span></h3>
<div class="p">
<sup>1</sup>Department of Molecular Biology and Genetics, Koç University, Istanbul, Turkey </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Arslanhan%20MD%22%5BAuthor%5D" class="usa-link"><span class="name western">Melis D Arslanhan</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Top%C3%A7u%20E%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Ebru Topçu</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Ebru Topçu</span></h3>
<div class="p">
<sup>1</sup>Department of Molecular Biology and Genetics, Koç University, Istanbul, Turkey </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Top%C3%A7u%20E%22%5BAuthor%5D" class="usa-link"><span class="name western">Ebru Topçu</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Firat-Karalar%20EN%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Elif Nur Firat-Karalar</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Elif Nur Firat-Karalar</span></h3>
<div class="p">
<sup>1</sup>Department of Molecular Biology and Genetics, Koç University, Istanbul, Turkey </div>
<div class="p">
<sup>2</sup>School of Medicine, Koç University, Istanbul, Turkey </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Firat-Karalar%20EN%22%5BAuthor%5D" class="usa-link"><span class="name western">Elif Nur Firat-Karalar</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>Department of Molecular Biology and Genetics, Koç University, Istanbul, Turkey </div>
<div id="Aff2">
<sup>2</sup>School of Medicine, Koç University, Istanbul, Turkey </div>
<div class="author-notes p"><div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div></div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2024 Jul 8; Accepted 2025 Jun 11; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by-nc-nd/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12371027  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40841441/" class="usa-link">40841441</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<p id="Par1">The dual-specificity tyrosine-phoshorylation-regulated kinase (DYRK) family are multifunctional enzymes crucial for diverse cellular processes, including signaling through the primary cilium. Their dysregulation has been implicated in various cancers and developmental disorders, highlighting the need to define their interactors and cellular functions to inform targeted therapeutics. In this study, we generate the proximity interactome of DYRK3, identifying 178 proteins involved in a range of cellular processes, including primary cilium biogenesis. We then investigate the specific role of DYRK3 and its cooperation with other DYRK family members in cilium assembly and maintenance. RNAi-mediated depletion of DYRK3 and pharmacological inhibition of DYRK kinase activity using GSK-626616 (GSK) lead to elongation of the cilium, particularly its distal segment. GSK treatment also induces ciliary defects, length fluctuations, and increased ectocytosis. Co-depletion and phenotypic rescue experiments reveal that DYRK2 and DYRK3 cooperate in regulating cilium length. Moreover, inhibiting or depleting known cilium length regulators, or quantifying their ciliary levels in GSK-treated cells, reveal functional relationships of DYRKs to centriolar satellites and the IFT complex. Collectively, our findings uncover regulatory roles for DYRK3 and DYRK kinase activity in the assembly and maintenance of primary cilium with proper length, stability, and morphology.</p>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/b6f8/12371027/94869c38cdfc/42003_2025_8373_Figa_HTML.jpg" loading="lazy" id="d33e223" height="501" width="634" alt="graphic file with name 42003_2025_8373_Figa_HTML.jpg"></p>
<section id="kwd-group1" class="kwd-group"><p><strong>Subject terms:</strong> Cilia, Centrosome, Cell signalling</p></section></section><section class="abstract" id="Abs2"><hr class="headless">
<p id="Par3">DYRK3 kinase interacts with multiple centriolar satellites and ciliary components to regulate axoneme length dynamics in cooperation with DYRK2.</p></section><section id="Sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p id="Par4">The dual-specificity tyrosine-phosphorylation-regulated kinases (DYRKs) constitute an evolutionarily conserved family of kinases, with members from yeast to human. DYRKs exhibit both serine/threonine (Ser/Thr) and tyrosine (Tyr) kinase activities. DYRKs are activated by autophosphorylation of a conserved tyrosine residue in the activation loop and, once activated, phosphorylate substrates on Ser/Thr residues<sup><a href="#CR1" class="usa-link" aria-describedby="CR1">1</a>–<a href="#CR3" class="usa-link" aria-describedby="CR3">3</a></sup>. The mammalian DYRK subfamily is composed of six members: DYRK1A, DYRK1B, DYRK2, DYRK3, DYRK4, and DYRK5. Each member possesses a conserved kinase domain and an adjacent N-terminal DYRK homology box. Variations in their N- and C-terminal regions confer each with unique cellular localizations, substrate specificities and functions<sup><a href="#CR1" class="usa-link" aria-describedby="CR1">1</a>,<a href="#CR2" class="usa-link" aria-describedby="CR2">2</a></sup>. DYRK-mediated phosphorylation has been shown to mediate their functions by modulating the activation, localization and turnover of its substrates<sup><a href="#CR2" class="usa-link" aria-describedby="CR2">2</a></sup><sup>,</sup><sup><a href="#CR4" class="usa-link" aria-describedby="CR4">4</a></sup>. Although their kinase activity is essential for most functions, DYRKs also use their non-catalytic domains as scaffolds for signaling complexes<sup><a href="#CR3" class="usa-link" aria-describedby="CR3">3</a>,<a href="#CR4" class="usa-link" aria-describedby="CR4">4</a></sup>.</p>
<p id="Par5">DYRKs are pleiotropic kinases with crucial roles in a wide range of cellular processes and signaling pathways essential for development and homeostasis<sup><a href="#CR2" class="usa-link" aria-describedby="CR2">2</a></sup><sup>,</sup><sup><a href="#CR4" class="usa-link" aria-describedby="CR4">4</a>–<a href="#CR6" class="usa-link" aria-describedby="CR6">6</a></sup>. Their aberrant regulation and expression have been implicated in several human pathologies including cancer, neurological diseases and virus infection. Consequently, DYRKs have received significant attention as therapeutic targets for these diseases, leading to the development of DYRK inhibitors such as GSK-626616<sup><a href="#CR2" class="usa-link" aria-describedby="CR2">2</a></sup><sup>,</sup><sup><a href="#CR7" class="usa-link" aria-describedby="CR7">7</a>–<a href="#CR9" class="usa-link" aria-describedby="CR9">9</a></sup>. Achieving specific and effective therapeutic targeting of DYRKs requires comprehensive understanding of each member of this kinase family. Originally, DYRKs were discovered for their functions in cell growth and differentiation in model organisms such as yeast, Dictyostelium and Drosophila melanogaster<sup><a href="#CR1" class="usa-link" aria-describedby="CR1">1</a></sup><sup>,</sup><sup><a href="#CR4" class="usa-link" aria-describedby="CR4">4</a></sup><sup>,</sup><sup><a href="#CR10" class="usa-link" aria-describedby="CR10">10</a></sup>. Among the mammalian DYRK kinase family, DYRK1A is the most extensively studied member, initially gained attention due to its association with Trisomy 21<sup><a href="#CR11" class="usa-link" aria-describedby="CR11">11</a></sup>. Subsequent studies have revealed its functions in diverse cellular and developmental processes, including neuronal development, synaptic transmission, chromatin remodeling, transcriptional control, alternative mRNA splicing and circadian timekeeping<sup><a href="#CR8" class="usa-link" aria-describedby="CR8">8</a>,<a href="#CR9" class="usa-link" aria-describedby="CR9">9</a>,<a href="#CR11" class="usa-link" aria-describedby="CR11">11</a></sup>.</p>
<p id="Par6">More recently, DYRK3 has emerged as a family member involved in essential cellular processes and disease. It has been implicated in maintaining the homeostasis of membrane-less organelles and the secretory pathway by regulating the phase transition behavior of their components<sup><a href="#CR12" class="usa-link" aria-describedby="CR12">12</a>–<a href="#CR15" class="usa-link" aria-describedby="CR15">15</a></sup>. Its kinase activity is required for mitotic progression and fidelity by facilitating the mitotic disassembly of various membrane-less organelles, such as centriolar satellites and stress granules<sup><a href="#CR14" class="usa-link" aria-describedby="CR14">14</a></sup>. Moreover, DYRK3 couples stress granule dynamics and to mTORC1 signaling and regulates secretory trafficking by modulating the biophysical properties of endoplasmic reticulum exit sites<sup><a href="#CR13" class="usa-link" aria-describedby="CR13">13</a>,<a href="#CR15" class="usa-link" aria-describedby="CR15">15</a></sup>. Finally, DYRK3 promotes efficient cell motility by regulating the stability of lamellipodia, which has implications for the link between DYRKs and tumor cell migration<sup><a href="#CR16" class="usa-link" aria-describedby="CR16">16</a></sup>.</p>
<p id="Par7">Several DYRK kinases have been reported to regulate cellular signaling by controlling the formation of the primary cilium, a microtubule-based cellular projection that transduces key developmental pathways such as Hedgehog and Wnt<sup><a href="#CR17" class="usa-link" aria-describedby="CR17">17</a></sup>. The assembly of a signaling-competent primary cilium is a highly coordinated, multistep process involving numerous protein complexes, signaling pathways and cellular structures<sup><a href="#CR18" class="usa-link" aria-describedby="CR18">18</a></sup>. Both in vivo and in vitro studies identified DYRK2 as a critical regulator of cilium assembly and ciliary signaling<sup><a href="#CR19" class="usa-link" aria-describedby="CR19">19</a>,<a href="#CR20" class="usa-link" aria-describedby="CR20">20</a></sup>. DYRK2 loss in mice suppresses Sonic Hedgehog (Shh) signaling, leading to skeletal abnormalities, congenital malformations in multiple organs, and perinatal death due to respiratory failure, highlighting its essential role in embryogenesis and development<sup><a href="#CR19" class="usa-link" aria-describedby="CR19">19</a>,<a href="#CR21" class="usa-link" aria-describedby="CR21">21</a></sup>. At the cellular level, DYRK2 deletion leads to abnormal ciliary morphology, defective trafficking of Hedgehog pathway components, and down-regulation of cilium disassembly genes including AURKA. Notably, DYRK1A and DYRK2 modulate Hedgehog-signaling pathway in opposing ways, while oncogenic RAS inhibits this pathway in part by activating DYRK1B<sup><a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>–<a href="#CR24" class="usa-link" aria-describedby="CR24">24</a></sup>. While these studies highlight DYRKs as regulatory kinases of the cilium, the functions of all family members, such as DYRK3, and their potential cooperation in cilium formation and function remain largely unexplored.</p>
<p id="Par8">In this study, we generated the in vivo proximity interactome of DYRK3, which revealed DYRK3’s proximity to a wide range of cellular processes, including key components of primary cilium biogenesis. Using a combination of functional assays, live-cell imaging and ultrastructure expansion microscopy (U-ExM), we showed that the kinase activity of DYRKs is required for regulating primary cilium length, stability and morphology. Notably, we found that multiple DYRK family members, particularly DYRK2 and DYRK3, cooperate during cilium length regulation. Collectively, our results identify primary cilium length and stability as a critical process regulated by DYRK kinases and targeted by their pharmacological inhibition.</p></section><section id="Sec2"><h2 class="pmc_sec_title">Results</h2>
<section id="Sec3"><h3 class="pmc_sec_title">Identification and network analysis of the DYRK3 proximity interactome</h3>
<p id="Par9">To identify the proximity interactome of DYRK3 in mammalian cells, we applied the BioID approach in human embryonic kidney (HEK293T) cells, which are widely used in proximity proteomics studies and thus make benchmarking easier. Inhibition of DYRK3 kinase activity enriched its known interactions and altered its interaction profile<sup><a href="#CR14" class="usa-link" aria-describedby="CR14">14</a></sup>. Therefore, we performed proximity mapping using both the wild-type (WT) and kinase-dead (KD) versions of DYRK3 for a comprehensive analysis of its interactome. For these experiments, we transiently transfected the cells with FLAG-BirA* (hereafter referred to as BirA*) fusions of DYRK3-WT and DYRK3-KD and treated them with 50 μM of biotin for 18 hours (h). Immunoblotting of cell lysates for streptavidin, FLAG and DYRK3 confirmed the overexpression of the fusion proteins relative to endogenous DYRK3, as well as successful streptavidin pulldown of biotinylated proteins, including DYRK3 as the bait protein (Fig. <a href="#MOESM1" class="usa-link">S1A</a>, <a href="#MOESM1" class="usa-link">S1B</a>). BirA*-fusions of both DYRK3-WT and DYRK3-KD localized to the centrosome and cytosol, and stimulated biotinylation at these sites (Fig. <a href="#Fig1" class="usa-link">1A</a>, Fig. <a href="#MOESM1" class="usa-link">S1C</a>). The kinase activity of DYRK3 was shown to induce the disassembly of centriolar satellites, which localize around centrosomes in epithelial cells<sup><a href="#CR14" class="usa-link" aria-describedby="CR14">14</a></sup>. In agreement, we found a decrease in pericentrosomal PCM1 levels in cells expressing DYRK3-WT compared to those expressing DYRK3-KD (Fig. <a href="#Fig1" class="usa-link">1B</a>). These results indicate that BirA*-DYRK3, although overexpressed, is functionally active.</p>
<figure class="fig xbox font-sm" id="Fig1"><h4 class="obj_head">Fig. 1. Identification of DYRK3-WT and KD proximity interactome.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371027_42003_2025_8373_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/b6f8/12371027/95bdd0aaf227/42003_2025_8373_Fig1_HTML.jpg" loading="lazy" id="d33e489" height="998" width="733" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>A</strong>,<strong> B</strong> Effects of DYRK3-WT and KD over-expression on the disassembly of centriolar satellites. <strong>A</strong> HEK293T cells were transfected with FLAG-BirA*-DYRK3-WT or FLAG-BirA*-DYRK3-KD and treated with 50 μM biotin for 18 h. Cells were then fixed and stained for centriolar satellites with anti-PCM1, biotinylated proteins with fluorescent streptavidin and centrosome with anti-gamma-tubulin. DNA was visualized with DAPI. Scale bar, 10 μm. Yellow dashed boxes indicate the zoomed-in centrosome. Scale bar for insets, 5 μm. <strong>B</strong> Quantification of percentage of centriolar satellite disassemlby for (<strong>A</strong>). Data represents mean ± SD of three independent experiments. <em>n</em> &gt; 100 cells per experiment. DYRK3-WT 78.66% ± 2.31; DYRK3-KD 22.08% ± 3.35. Student’s t-test. **<em>P</em> &lt; 0.01. <strong>C</strong> Quantification of PCM1 intensity at the pericentrosomal region for (<strong>A</strong>). Data represents mean ± SD of three independent experiments. <em>n</em> &gt; 70 cells per experiment. DYRK3-WT = 1, DYRK3-KD = 2.46 ± 0.26. Student’s t-test. ***<em>P</em> &lt; 0.01. <strong>C</strong>–<strong>E</strong> Comparative analysis of high-confidence interactors for DYRK3-WT and KD with published studies. This includes comparisons of <strong>C</strong> DYRK3-WT with DYRK3-KD proximity interactomes generated in this study, (<strong>D</strong>) DYRK3-WT and KD proximity interactomes with affinity purification-mass spectrometry (AP-MS) physical proteomes of DYRK3 in wild-type cells and cells treated with GSK-626616<sup><a href="#CR14" class="usa-link" aria-describedby="CR14">14</a></sup>, and <strong>E</strong> the proximity interactome of DYRK3 generated by BioID in this study and TurboID in ref. <sup><a href="#CR13" class="usa-link" aria-describedby="CR13">13</a></sup> and the AP-MS physical interactome<sup><a href="#CR14" class="usa-link" aria-describedby="CR14">14</a></sup>. <strong>F</strong> GO-enrichment analysis of the DYRK3-WT and KD proximity interactors based on their cellular compartments. The x-axis represents the log-transformed <em>p</em>-value (Fisher’s exact test) of GO terms. <strong>G</strong> Enrichment map of DYRK3 proximity interactors linked to the primary cilium. The DYRK3 proximity interactors were grouped using DAVID functional annotation tool and literature mining. The interconnectedness among the proteins of each network was determined by the STRING database and the map is visualized using CytoScape. The edge thickness indicates the strength of the interactions. <strong>H</strong> Validation of DYRK3 centriolar satellite proximity interactors by FLAG pulldown. HEK293T cells were transiently transfected with FLAG, FLAG-DYRK3-WT or FLAG-DYRK3-KD. 24 h after transfection, cells were lysed, and proteins were precipitated by FLAG affinity beads. The initial sample and immunoprecipitated proteins were run on a gel and immunoblotted with fluorescent streptavidin and antibodies against FLAG, KIF7, PCM1, CEP131 and CCDC138.</p></figcaption></figure><p id="Par10">We next performed large scale pulldowns followed by label-free quantitative proteomics to generate the proximity maps for DYRK3-WT and DYRK3-KD. Cells expressing BirA* were used as a control. To define the high-confidence proximity interactors of DYRK3-WT and DYRK3-KD, we filtered out low-confidence interactors using two different analysis methods and thresholds as described in the methods section (Supplementary Data <a href="#MOESM5" class="usa-link">1–3</a>). First, we performed Normalized Spectral Abundance Factor (NSAF) analysis and included proteins with a log<sub>2</sub> NSAF value greater than 1<sup><a href="#CR25" class="usa-link" aria-describedby="CR25">25</a>,<a href="#CR26" class="usa-link" aria-describedby="CR26">26</a></sup>. Next, we removed common mass spectrometry contaminants (accounting for &gt;30% of the contaminants) using The Contaminant Repository for Affinity Purification – Mass Spectrometry data (CRAPome)<sup><a href="#CR27" class="usa-link" aria-describedby="CR27">27</a></sup>. These combined thresholds identified 111 and 127 proteins as high confidence interactors of DYRK3-WT and DYRK3-KD, respectively (Supplementary Data <a href="#MOESM5" class="usa-link">1</a>, <a href="#MOESM6" class="usa-link">2</a>). The comparison between proximity interactors of DYRK3-WT and DYRK3-KD revealed 57 common proteins, corresponding to a 31% overlap within their combined proteomes (Fig. <a href="#Fig1" class="usa-link">1C</a>). The proteomes of DYRK3 in wild-type cells and cells treated with the DYRK kinase inhibitor GSK-626616 were previously identified by affinity purification of GFP-DYRK3 combined with SILAC-based quantitative proteomics (AP-MS)<sup><a href="#CR14" class="usa-link" aria-describedby="CR14">14</a></sup>. Notably, these proteomes shared 206 proteins, representing a 40% overlap within their combined proteomes (Fig. <a href="#Fig1" class="usa-link">1D</a>). Findings from both proximity-mapping and AP-MS studies suggest that inactivation of DYRK3 kinase activity, either through a point mutation or pharmacological inhibition, significantly alters its proteomic profile (Supplementary Data <a href="#MOESM7" class="usa-link">3</a>). In addition to AP-MS data, we compared the proximity interactors of DYRK3-WT and KD from our study with those previously identified using TurboID proximity labeling method<sup><a href="#CR13" class="usa-link" aria-describedby="CR13">13</a></sup>. TurboID has a shorter labeling time compared to BioID, which enables probing temporal snapshots of protein interactions, thereby generating different interactomes<sup><a href="#CR28" class="usa-link" aria-describedby="CR28">28</a>,<a href="#CR29" class="usa-link" aria-describedby="CR29">29</a></sup>. This comparison revealed that ~20% of the DYRK3-WT proximity interactors were shared, with 22 proteins overlapping between the BioID and TurboID methods, and 23 between BioID and AP-MS (Fig. <a href="#Fig1" class="usa-link">1E</a>). The shared interactors across all three approaches included SEC23B, SEC16A, SEC24B, SEC24A, which regulate the secretion of proteins from ER and Golgi network in a DYRK3-dependent manner<sup><a href="#CR13" class="usa-link" aria-describedby="CR13">13</a></sup>. Comparison of the DYRK3 proximity interactome with that of DYRK2 revealed that the two kinases share about 10% of their proximity interactors (Mehnert et al.<sup><a href="#CR30" class="usa-link" aria-describedby="CR30">30</a></sup>,) (Fig. <a href="#MOESM1" class="usa-link">S1F</a>). Among these shared interactors are centriolar satellite proteins (PCM1, CCDC138, OFD1, SDCCAG3), nucleoporins (NUP107, NUP160), and SEC16A. These findings suggest that most of the interactors identified in our study had not been reported before.</p>
<p id="Par11">Next, we organized the high confidence DYRK3-WT and DYRK3-KD proximity interactors ranked by their fold change into an interaction network by combining STRING databased and ClusterONE plug-in on Cytoscape. The resulting networks identified a diverse array of proteins across 14 major functional clusters: 8 for wild-type DYRK3 and 6 for kinase-dead DYRK3 (<em>p</em> &lt; 0.05) (Fig. <a href="#MOESM1" class="usa-link">S2A</a>). For DYRK3-WT, these clusters included spindle assembly, Wnt signaling pathway, ribosome biogenesis, RNA splicing, nuclear transport, COPII-coated vesicle budding, epidermis development, phosphate biosynthetic processes. For DYRK3-KD, the clusters were ribosomal biogenesis, P-body assembly, cilium organization, COPII-coated cargo loading, DNA damage signaling and Toll-like receptor signaling for DYRK3-KD (Fig. <a href="#MOESM1" class="usa-link">S2B</a>). Additionally, we performed Gene Ontology (GO) analysis of the DYRK3-WT and DYRK3-KD proximity interactors based on “cellular compartment” and “biological processes” (Fig. <a href="#Fig1" class="usa-link">1F</a>, Fig. <a href="#MOESM1" class="usa-link">S1G</a>). Consistent with the previously described DYRK3 functions and compartments and thus underscoring the robustness of our approach, DYRK3-WT and DYRK3-KD proximity interactors showed significant (<em>p</em> &lt; 0.05) enrichment for GO terms related to the secretory pathway such as “cargo loading into COPII-coated vesicles”, “protein import to nucleus”, “ER to Golgi vesicle-mediated transport” as well as mRNA processing, splicing, and metabolism (Fig. <a href="#MOESM1" class="usa-link">S1G</a>). Additionally, there was significant enrichment across various compartments such as the COPII vesicle coat, cilium, centrosome, P and PML bodies, endoplasmic reticulum exit site, and focal adhesion (Fig. <a href="#MOESM1" class="usa-link">S1G</a>). Of note, a recent study defined DYRK3’s role in cell migration by influencing the stability of protrusions and localization of focal adhesions in migrating cells<sup><a href="#CR16" class="usa-link" aria-describedby="CR16">16</a></sup>. A sub-interaction network of proteins involved in primary cilium biogenesis and functions, along with the identification of ‘primary cilium’ as a significant GO term, suggests functional and regulatory connections to DYRK3 (Fig. <a href="#Fig1" class="usa-link">1G</a>). This is consistent with the reported functions of other members of the DYRK family, such as DYRK1 and DYRK2, during primary cilium morphology and signaling<sup><a href="#CR4" class="usa-link" aria-describedby="CR4">4</a>,<a href="#CR19" class="usa-link" aria-describedby="CR19">19</a></sup>.</p>
<p id="Par12">To validate the DYRK3 proximity interactome and its association with centriolar satellites and regulators of the primary cilium, we performed both FLAG and streptavidin pulldowns in cells expressing either FLAG-BirA* (control) and FLAG-BirA*-DYRK3. We then immunoblotted for new DYRK3 interactors predicted by the proximity map (Fig. <a href="#Fig1" class="usa-link">1H</a>, Fig. <a href="#MOESM1" class="usa-link">S1D</a>, <a href="#MOESM1" class="usa-link">S1E</a>). Specifically, we focused on proteins linked to centriolar satellites and primary cilium, as identified in previous interaction and loss-of-function studies, including KIF7, PCM1, CEP131 and CCDC138<sup><a href="#CR31" class="usa-link" aria-describedby="CR31">31</a>–<a href="#CR33" class="usa-link" aria-describedby="CR33">33</a></sup>. As a negative control, we used gamma-tubulin, which was not identified in the DYRK3 proximity interactome. FLAG and streptavidin pulldowns from cells expressing FLAG-BirA*-DYRK3-WT and FLAG-BirA*-DYRK3-KD confirmed physical and proximity interactions of DYRK3 with KIF7, PCM1, CEP131 and CCDC138, but not with gamma-tubulin (Fig. <a href="#Fig1" class="usa-link">1H</a>, Fig. <a href="#MOESM1" class="usa-link">S1E</a>. These results validate the DYRK3’s interactions with regulators of the primary cilium, further supporting its potential roles at the primary cilium.</p></section><section id="Sec4"><h3 class="pmc_sec_title">DYRK3 is required for cilium length regulation</h3>
<p id="Par13">Given its interactions with known ciliary regulators, we investigated whether DYRK3 is required for primary cilium biogenesis. To this end, we performed siRNA and shRNA-mediated loss-of-function experiments using previously validated target sequences<sup><a href="#CR14" class="usa-link" aria-describedby="CR14">14</a>,<a href="#CR34" class="usa-link" aria-describedby="CR34">34</a></sup>. Immunoblotting revealed partial DYRK3 depletion, showing a 40% reduction with siDYRK3 and 67% reduction with shDYRK3 (Fig. <a href="#MOESM1" class="usa-link">S3A</a>, <a href="#MOESM1" class="usa-link">S3B</a>). For cilium assembly experiments, DYRK3-depleted confluent RPE1 cultures were serum-starved for 24 h to induce cilium formation. We then quantified ciliation efficiency and cilium length by staining for acetylated tubulin, a marker of the ciliary axoneme. DYRK3 depletion by siRNA or shRNA both led to a significant increase in cilium length (1.3-fold) without affecting ciliation efficiency (Fig. <a href="#Fig2" class="usa-link">2A–C</a>, Fig. <a href="#MOESM1" class="usa-link">S3C</a>, <a href="#MOESM1" class="usa-link">S3D</a>). Notably, despite differences in depletion levels between the two methods, the relative increase in cilium length was similar.</p>
<figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2. DYRK3 and DYRK2 cooperate to regulate cilium length.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371027_42003_2025_8373_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/b6f8/12371027/dc6805b7f7c8/42003_2025_8373_Fig2_HTML.jpg" loading="lazy" id="d33e783" height="995" width="717" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>A</strong>–<strong>C</strong> DYRK3 depletion leads to cilium elongation. <strong>A</strong> Representative images show the effects of control and DYRK3 siRNA treatments on cilia length. RPE1 cells were transfected with control and DYRK3 siRNAs. After 48 h of depletion, cells were serum starved for 24 h, fixed and stained for anti-acetylated tubulin and anti-gamma-tubulin. Scale bar, 10 μm. Quantification of <strong>B</strong> ciliation percentage and <strong>C</strong> cilium length based on experiment shown in (<strong>A</strong>). Data represents mean ± SD of three independent experiments. <em>n</em> &gt; 83 cells per experiment. Cilium length: siControl = 3.54 μm ± 0.12, siDYRK3 = 4.68 μm ± 0.46. Ciliation percentage: siControl = 57.72% ± 5.18, siDYRK3 = 52.91 ± 1.73. Statistical significance was determined using Student’s t-test based on the mean values of the three experiments. *<em>P</em> &lt; 0.05, ns: non-significant. <strong>D</strong>,<strong> E</strong> Depletion of DYRK3 changes cilium distal and proximal segment ratio. <strong>D</strong> Representative images show the changes in the proximal and distal segments of the cilium during cilium assembly upon DYRK3 depletion. RPE1 cells were transfected with control and DYRK3 siRNAs. After 48 h of depletion, cells were serum starved for 24 h, fixed and stained for anti-acetylated tubulin and anti-gamma-tubulin. Scale bar, 5 μm. <strong>E</strong> Quantification of the distal segment to proximal segment ratio and proximal segment length based on the experiment shown in (<strong>D</strong>). Data represents mean ± SD of three independent experiments. <em>n</em> &gt; 63 cells per experiment. Distal segment/acetylated tubulin ratio: DMSO = 0.39 ± 0.01, GSK-626616 = 0.50 ± 0.04. Proximal segment length: DMSO = 2.12 μm ± 0.13, GSK-626616 = 2.14 μm ± 0.17. Statistical significance was determined using Student’s t-test based on the mean values of the three experiments. *<em>P</em> &lt; 0.05. <strong>F</strong>,<strong> G</strong> DYRK3 and DYRK2 cooperate to regulate cilium length. <strong>F</strong> Representative images of RPE1 cells transfected with siControl, siDYRK2, siDYRK3 and siDYRK2/siDYRK3 for 72 h. 24 h after serum starvation, cells were fixed and stained for anti-acetylated tubulin and anti-gamma-tubulin. Scale bar, 5 μm. <strong>G</strong> Quantification of cilium length based on the experiment shown in (<strong>H</strong>). Data represents mean ± SD of three independent experiments. <em>n</em> = 100 cells per experiment. siControl: 3.66 μm ± 0.17; siDYRK2: 6.83 μm ± 0.51; siDYRK3: 4.88 μm ± 0.13; siDYRK2/siDYRK3: 7.55 μm ± 0.60. <strong>H</strong>,<strong> I</strong> DYRK2 and DYRK3 rescue the cilium length phenotypes observed in DYRK3 depletion. <strong>H</strong> Representative images of RPE1 cells, RPE1 cells stably expressing mNG-mDYRK3 or mNG-DYRK2 transfected with either siControl or siDYRK3. 24 h after serum starvation, cells were fixed and stained for anti-acetylated tubulin and anti-gamma-tubulin. Scale bar, 5 μm. <strong>I</strong> Quantification of cilium length based on the experiment shown in (<strong>J</strong>). Data represents mean ± SD of three independent experiments. <em>n</em> &gt; 62 cells per experiment. RPE1 siControl: 3.56 μm ± 0.41, RPE1 siDYRK3: 5.04 μm ± 0.44; RPE1::mNG-DYRK2 siControl: 3.67 μm ± 0.18, ; RPE1::mNG-DYRK2 siDYRK3: 3.60 μm ± 0.16; RPE1::mNG-mDYRK3 siControl: 3.50 μm ± 0.18, RPE1::mNG-mDYRK3 siDYRK3: 3.31 μm ± 0.07. <strong>J</strong>,<strong> K</strong> DYRK3 partially rescues cilium elongation caused by DYRK2 depletion. <strong>J</strong> Representative images of RPE1 and RPE1::mNG-mDYRK3 cells transfected with siControl and siDYRK2. Cells are treated with indicated siRNAs for 72 h and then fixed and stained with acetylated tubulin (Acet-tub) and gamma-tubulin. Scale bar, 5 μm. <strong>K</strong> Quantification of cilium length based on the experiment shown in (<strong>J</strong>). Data represents mean ± SD of three independent experiments. <em>n</em> &gt; 86 cells per experiment. RPE1 siControl = 4.07 μm ± 0.37, RPE1 siDYRK2 = 9.05 μm ± 0.78, RPE1::mNG-mDYRK3 siControl = 3.52 μm ± 0.21, RPE1::mNG-mDYRK3 siDYRK2 = 6.82 μm ± 0.25. Statistical significance was determined using Student’s t-test based on the mean values of the three experiments. **<em>P</em> &lt; 0.01, ***<em>P</em> &lt; 0.001, ****<em>P</em> &lt; 0.0001, ns: non-significant.</p></figcaption></figure><p id="Par14">The ciliary axoneme in RPE1 cells is sub-divided into proximal (PS) and distal (DS) segments, characterized by regions rich and poor in polyglutamylation, respectively<sup><a href="#CR35" class="usa-link" aria-describedby="CR35">35</a></sup>. Importantly, hyper elongation of the DS leads to ciliary instability and defective Hedgehog pathway activation. Therefore, we investigated whether DYRK3 depletion-induced elongation of the axoneme alters the ratio of DS to PS. To this end, we stained control and DYRK3-depleted cells using markers for axonemal microtubules: acetylated and polyglutamylated-tubulin (Fig. <a href="#Fig2" class="usa-link">2D</a>, Fig. <a href="#MOESM1" class="usa-link">S3C</a>). We measured the lengths of these regions and defined the polyglutamylated tubulin-rich region as the PS (Fig. <a href="#Fig2" class="usa-link">2D, E</a>, Fig. <a href="#MOESM1" class="usa-link">S3C–F</a>). The length of the DS was calculated as the difference between the total cilium length (acetylated-tubulin-stained region) and the PS. While the length of DS increased, the length of PS remained unaltered in DYRK3-depleted cells (Fig. <a href="#Fig2" class="usa-link">2E</a>, Fig. <a href="#MOESM1" class="usa-link">S3E</a>, <a href="#MOESM1" class="usa-link">S3F</a>). Importantly, while the DS accounted for approximately 0.39 of the cilium length in control cells, DYRK3 depletion increased the DS length to about 0.50 of the total cilium length (Fig. <a href="#Fig2" class="usa-link">2E</a>). These results show that DYRK3 is required for the regulation of DS elongation and suggest its potential function in ciliary stability and signaling.</p></section><section id="Sec5"><h3 class="pmc_sec_title">DYRK3 and DYRK2 cooperate in regulating cilium length</h3>
<p id="Par15">Several DYRK kinases, particularly DYRK1A and DYRK2, has been previously described for functions at the primary cilium<sup><a href="#CR2" class="usa-link" aria-describedby="CR2">2</a></sup><sup>,</sup><sup><a href="#CR4" class="usa-link" aria-describedby="CR4">4</a></sup><sup>,</sup><sup><a href="#CR19" class="usa-link" aria-describedby="CR19">19</a></sup>. We next investigated whether DYRK3 cooperates with or functions redundantly with these DYRKs in regulating cilium length. To address this, we performed co-depletion and phenotypic rescue experiments with DYRK2, the best characterized DYRK kinase for its ciliary functions. First, we depleted DYRK2 and DYRK3 individually or in combination, followed by 24 h serum starvation to induce cilium assembly. DYRK2 depletion led to a 1.86-fold increase in cilium length compared to control cells (Fig. <a href="#Fig2" class="usa-link">2F, G</a>). In comparison, DYRK3 depletion caused a more modest 1.33-fold increase. Co-depleting DYRK2 and DYRK3 caused a further increase in cilium length, 2.06-fold compared to control, which was greater than the increase seen with depleting either protein alone (Fig. <a href="#Fig2" class="usa-link">2G</a>).</p>
<p id="Par16">To assess the extent of functional redundancy between DYRK2 and DYRK3, we performed phenotypic rescue experiments using RPE1 cells stably expressing siRNA-resistant mNeonGreen-tagged mouse DYRK3 (hereafter mNG-mDYRK3) and mNeonGreen-DYRK2 (hereafter mNG-DYRK2). Expression of both fusion proteins was confirmed using immunofluorescence and immunoblotting (Fig. <a href="#MOESM1" class="usa-link">S3G</a>, <a href="#MOESM1" class="usa-link">S3H</a>, <a href="#MOESM1" class="usa-link">S3I</a>). Both mDYRK3 and DYRK2 localized to the centrosome and centriolar satellites in these cells (Fig. <a href="#MOESM1" class="usa-link">S3H</a>, <a href="#MOESM1" class="usa-link">S3I</a>). Moreover, overexpression of mNG-DYRK2 disrupted pericentrosomal clustering of centriolar satellites, though to a lesser extent than DYRK3 (Fig. <a href="#MOESM1" class="usa-link">S3J</a>, <a href="#MOESM1" class="usa-link">S3F</a>, Fig. <a href="#Fig1" class="usa-link">1A, C</a>). Stable expression of mNG-mDYRK3 and mNG-DYRK2 did not lead to changes in cilium length as compared to control cells. However, the cilium elongation phenotype resulting from DYRK3 depletion was rescued in both lines (Fig. <a href="#Fig2" class="usa-link">2H, I</a>). In contrast, DYRK2 depletion-induced elongation was only partially rescued by mNG-mDYRK3 expression (Fig. <a href="#Fig2" class="usa-link">2J, K</a>). Collectively, these findings suggest that DYRK2 and DYRK3 cooperate in regulating cilium length, and that other DYRK family members may also contribute to cilium biogenesis.</p></section><section id="Sec6"><h3 class="pmc_sec_title">Kinase activity of DYRKs regulate primary cilium length and morphology during cilium assembly</h3>
<p id="Par17">The cooperative roles of DYRK2 and DYRK3 in regulating cilium length suggest that additional DYRK family members may also contribute to this process. To investigate this possibility in an unbiased manner, we used pharmacological inhibition to assess the impact of DYRK kinase activity on cilium assembly, maintenance and dynamics. Specifically, we used GSK-626616 (hereafter GSK), originally developed as a selective DYRK3 inhibitor but later shown to inhibit multiple DYRKs with similar potency<sup><a href="#CR7" class="usa-link" aria-describedby="CR7">7</a>,<a href="#CR15" class="usa-link" aria-describedby="CR15">15</a></sup>. We first examined the impact of GSK treatment on cilium assembly. Confluent RPE1 cultures were treated with 1 μM GSK for 24 h during serum starvation, followed by staining for acetylated tubulin (Fig. <a href="#Fig3" class="usa-link">3A</a>). GSK-treated cells had about at 25% reduction in ciliation efficiency compared to control cells (Fig. <a href="#Fig3" class="usa-link">3B, C</a>). This supports the involvement of DYRKs other than DYRK3, which alone did not produce this phenotype (Fig. <a href="#Fig2" class="usa-link">2B</a>).</p>
<figure class="fig xbox font-sm" id="Fig3"><h4 class="obj_head">Fig. 3. DYRK kinase activity regulates the length and morphology of cilia during cilium assembly.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371027_42003_2025_8373_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/b6f8/12371027/3b1d9f96f3da/42003_2025_8373_Fig3_HTML.jpg" loading="lazy" id="d33e979" height="814" width="800" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>A</strong> Workflow for primary cilium assembly experiments. Cells were seeded on coverslips and treated with 1 μM DYRK3 inhibitor GSK-626616 in serum starvation medium. After 24 h of incubation, cells were fixed and stained with the indicated antibodies for subsequent analysis. <strong>B</strong> Inhibition of DYRK kinase activity leads to defects in cilium assembly, length and morphology. Representative confocal images show defects in ciliogenesis and cilium length and curvature during cilium assembly. RPE1 cells were treated with serum starvation medium containing DMSO as a control or 1 μM GSK-626616. 24 h after treatment, cells were fixed and stained for the primary cilium using anti-acetylated tubulin (Acet-tub). DNA was visualized with DAPI. Scale bar, 10 μm. Quantification of <strong>C</strong> ciliation percentage, (<strong>D</strong>) cilium length and <strong>E</strong> cilium curvature from the experiment shown in (<strong>B</strong>). Data represents mean ± SD of three independent experiments. <em>n</em> &gt; 75 cells per experiment. Ciliation percentage: DMSO = 85.50% ± 4.10, GSK-626616 = 66.61% ± 2.89. Cilium length: DMSO = 4.36 μm ± 0.12, GSK-626616 = 8.41 μm ± 0.60, cilium curvature: DMSO = 1.05 ± 0.004, GSK-626616 = 1.13 ± 0.019. Statistical significance was determined using Student’s t-test based on the mean values of the three experiments. **<em>P</em> &lt; 0.01, ***<em>P</em> &lt; 0.001. <strong>F</strong> Inhibition of DYRK kinase activity leads to changes in cilium distal and proximal segment ratio. Representative images show the changes in the proximal and distal segments of the cilium during cilium assembly. RPE1 cells were treated with serum starvation medium containing DMSO as a control or 1 μM GSK-626616. 24 h after treatment, cells were fixed and stained for anti-acetylated tubulin (Acet-tub) and proximal segment using anti-glutamylated tubulin (Glut-tub). DNA was visualized with DAPI. Scale bar, 10 μm. <strong>G</strong>,<strong> H</strong> Quantification of the distal segment to proximal segment ratio based on the experiment shown in (<strong>F</strong>). Quantifications include (<strong>G</strong>) proximal segment length and <strong>H</strong> distal segment to acetylated tubulin ratio. The schematic depicts regions of the primary cilium that are rich in polyglutamylation (shown in cyan) and regions that are poor in polyglutamylation (shown in magenta, excluding the cyan areas). Data represents mean ± SD of three independent experiments. <em>n</em> &gt; 66 cells per experiment. Proximal segment length: DMSO = 3.44 μm ± 0.21, GSK-626616 = 6.03 μm ± 0.55, Distal segment/acetylated tubulin ratio: DMSO = 0.19 ± 0.04, GSK-626616 = 0.26 ± 0.02. Statistical significance was determined using Student’s t-test based on the mean values of the three experiments. *<em>P</em> &lt; 0.05, **<em>P</em> &lt; 0.01. <strong>I</strong> Inhibition of DYRK kinase activity leads to changes in cilium morphology. Representative confocal images show defects in cilium morphology during cilium assembly. RPE1 cells were treated with serum starvation medium containing DMSO as a control or 1 μM GSK-626616. 24 h after treatment, cells were processed for U-ExM and stained for anti-Arl13b (yellow) and anti-acetylated tubulin (magenta). The arrows point to defects in elongated cilia. Scale bars shown on the images are not adjusted for the expansion factor (4.2x). Insets display 10× magnifications of the cilia sections highlighted within a yellow dashed box. <strong>J</strong> Quantification of tip bulging observed in (<strong>I</strong>). Cells treated with GSK-626616 during cilium assembly are quantified for the bulging at the cilia tip. Data represents mean ± SD of three independent experiments. <em>n</em> &gt; 100 cells per experiment. DMSO: 5.40% ± 2.36 GSK: 17.12% ± 3.22. Statistical significance was determined using Student’s t-test based on the mean values of the three experiments. **<em>P</em> &lt; 0.01.</p></figcaption></figure><p id="Par18">Cilia in GSK-treated cells were 1.93-fold longer than in control cells, with a greater increase than observed in DYRK3-depleted cells. Additionally, these cilia exhibited a curved morphology, in contrast to the shorter and straight cilia observed in control cells (Fig. <a href="#Fig3" class="usa-link">3B, E</a>). Notably, both the DS and PS lengths increased upon GSK treatment (Fig. <a href="#Fig3" class="usa-link">3F–H</a>). While the DS accounted for ~19% of the cilium length in control cells, GSK treatment increased the DS proportion to about 26% of the total cilium length (Fig. <a href="#Fig3" class="usa-link">3H</a>). These results show that DYRK kinase activity is required for regulating DS elongation and suggest a potential role in ciliary stability and signaling.</p>
<p id="Par19">To elucidate how inhibition of DYRK kinase activity affects cilia morphology, we examined the nanoscale organization of the cilium by performing ultrastructure expansion microscopy (U-ExM) of control and GSK-treated cells stained for Arl13b and acetylated tubulin. We identified several types of morphological defects of the ciliary axoneme and membrane upon DYRK inhibition (Fig.<a href="#Fig3" class="usa-link">3I</a>). First, all axonemes in cells treated with GSK were longer than those in control cells. Second, a subset of cilia in GSK-treated cells had a bulged membrane at their distal ends. The percentage of bulged cilia tip increased when the cells are treated with GSK (Fig. <a href="#Fig2" class="usa-link">2I</a>). In these cells, the acetylated tubulin signal spanned the entire primary cilium length and occupied the bulging ciliary tip. Additionally, we observed that cilia in GSK-treated cells showed lower concentrations of the small GTPase Arl13b compared to control cells (Fig. <a href="#Fig3" class="usa-link">3I</a>). We confirmed this reduction by quantitative immunofluorescence of the Arl13b signal in control and GSK-treated cells stained for Arl13b and acetylated tubulin and imaged using confocal microscopy (Fig. <a href="#MOESM1" class="usa-link">S4A</a>, <a href="#MOESM1" class="usa-link">S4B</a>). In contrast, analysis of acetylated tubulin showed no significant difference between control and GSK-treated cells (Fig. <a href="#MOESM1" class="usa-link">S4A</a>, <a href="#MOESM1" class="usa-link">S4C</a>). These findings indicate that DYRK kinase activity is required for efficient ciliation, proper ciliary length and architecture as well as the ciliary localization of Arl13b during cilium assembly.</p>
<p id="Par20">To test whether DYRK family members other than DYRK2 and DYRK3 contribute to the cilium elongation phenotype observed with GSK treatment, we examined whether GSK further enhances cilium elongation following DYRK2 and DYRK3 depletion. RPE1 cells were depleted of DYRK2 and DYRK3 using specific siRNAs and subsequently treated with 1 µM GSK for 24 h. DYRK2 depletion alone led to a 1.88-fold increase in cilium length, while DYRK3 depletion resulted in a 1.31-fold increase compared to control DMSO-treated cells (Fig. <a href="#MOESM1" class="usa-link">S4D</a>, <a href="#MOESM1" class="usa-link">S4E</a>). Interestingly, GSK treatment induced a further of 1.50-fold and 2.11-fold cilium elongation in both DYRK2- and DYRK3-depleted cells, similar to its effect in control cells. These findings support the involvement of additional DYRK family members in regulating cilium length.</p></section><section id="Sec7"><h3 class="pmc_sec_title">Kinase activity of DYRKs is required for primary cilium maintenance</h3>
<p id="Par21">The primary cilium assembles in quiescent cells and once formed, maintains its length, stability and structure until disassembly is induced by growth stimuli. To study the function of DYRKs in cilium maintenance, we assessed the effects of inhibiting DYRKs’ kinase activity on mature cilia. Confluent RPE1 cells were serum starved for 24 h to induce ciliogenesis, then treated with GSK for an additional 24 h in serum starvation medium, followed by quantification of the percentage of ciliation was quantified (Fig. <a href="#Fig4" class="usa-link">4A</a>). There was no significant difference in the percentage of ciliated cells between control and GSK-treated groups (Fig. <a href="#Fig4" class="usa-link">4B, C</a>). Similar to the assembly experiments, inhibition of DYRK kinase activity led to an increase in cilium length and curvature during maintenance of the cilium (Fig. <a href="#Fig4" class="usa-link">4D, E</a>). Notably, the average cilium length was 3.47-fold longer when cells were treated with GSK during the maintenance phase compared to the assembly phase, indicating that DYRK inhibition has a stronger effect at this stage. A similar greater increase was also observed in the percentage of cilium length occupied by the DS. While DS accounted for ~25% of the cilium length in control cells, GSK treatment increased the DS length to about 47% of the total cilium length (Fig. <a href="#Fig4" class="usa-link">4F–H</a>). Additionally, U-ExM analysis of cilia in maintenance experiments showed that cilia in GSK-treated cells exhibited morphological defects in the ciliary axoneme and membrane, including elongation and bulging of the ciliary axonemes and membranes at their distal ends (Fig. <a href="#Fig4" class="usa-link">4I</a>). Similar to the assembly experiments, cilia in GSK-treated cells showed lower concentrations of the small GTPase Arl13b and similar concentrations of acetylated tubulin compared to control cells (Fig. <a href="#Fig4" class="usa-link">4I</a>, Fig. <a href="#MOESM1" class="usa-link">S4F–H</a>). Collectively, these findings indicate that DYRK kinase activity is required for maintaining the primary cilium with proper length, architecture and ciliary protein content.</p>
<figure class="fig xbox font-sm" id="Fig4"><h4 class="obj_head">Fig. 4. DYRK kinase activity regulates the length and morphology of cilia during cilium maintenance.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371027_42003_2025_8373_Fig4_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/b6f8/12371027/f64c4a9d878d/42003_2025_8373_Fig4_HTML.jpg" loading="lazy" id="d33e1138" height="825" width="800" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>A</strong> Workflow for primary cilium maintenance experiments. Cells were seeded on coverslips, serum starved for 24 h and then treated with 1 μM GSK-626616 in serum starvation medium. Following treatment, cells were fixed and stained with the indicated antibodies for subsequent analysis. <strong>B</strong> Inhibition of DYRK kinase activity leads to defects in cilium length and morphology. Representative confocal images show defects in cilium length and curvature during cilium maintenance. RPE1 cells were serum starved for 24 h and treated with serum starvation medium containing DMSO as a control or 1 μM GSK-626616. 24 h after treatment, cells were fixed and stained for the primary cilium using anti-acetylated tubulin (Acet-tub). DNA was visualized with DAPI. Scale bar, 10 μm. Quantification of <strong>C</strong> ciliation percentage, (<strong>D</strong>) cilium length and <strong>E</strong> cilium curvature from the experiment shown in (<strong>B</strong>). Data represents mean ± SD of three independent experiments. <em>n</em> &gt; 100 cells per experiment. Ciliation percentage: DMSO = 89.99% ± 1.01, GSK-626616 = 85.78% ± 2.52. Cilium length: DMSO = 3.63 μm ± 1.00, GSK-626616 = 12.58 μm ± 3.49. Cilium curvature: DMSO = 1.08 ± 0.001, GSK-626616 = 1.26 ± 0.030. Statistical significance was determined using Student’s t-test based on the mean values of the three experiments. ***<em>P</em> &lt; 0.001, ****<em>P</em> &lt; 0.0001, ns: non-significant. <strong>F</strong> Inhibition of DYRK kinase activity leads to changes in cilium distal and proximal segment ratio. Representative images show the changes in the proximal and distal segments of the cilium during cilium maintenance. RPE1 cells were serum starved for 24 h and treated with serum starvation medium containing DMSO as a control or 1 μM GSK-626616. 24 h after treatment, cells were fixed and stained for the distal segment using anti-acetylated tubulin (Acet-tub) and proximal segment using anti-glutamylated tubulin (Glut-tub). DNA was visualized with DAPI. Scale bar, 10 μm. <strong>G</strong>,<strong> H</strong> Quantification of the distal segment to proximal segment ratio based on the experiment shown in (<strong>F</strong>). Quantifications include (<strong>G</strong>) proximal segment length and <strong>H</strong> distal segment to acetylated tubulin ratio. The schematic depicts regions of the primary cilium that are rich in polyglutamylation (shown in cyan) and regions that are poor in polyglutamylation (shown in magenta, excluding the cyan areas). Data represents mean ± SD of three independent experiments. <em>n</em> &gt; 45 cells per experiment. Proximal segment length: DMSO = 2.72 μm ± 0.43, GSK-626616 = 6.7 μm ± 0.39. Distal segment/acetylated tubulin ratio: DMSO = 0.25 ± 0.10, GSK-626616 = 0.47 ± 0.06. Statistical significance was determined using Student’s t-test based on the mean values of the three experiments. ***<em>P</em> &lt; 0.001, *<em>P</em> &lt; 0.05. <strong>I</strong> Inhibition of DYRK kinase activity leads to changes in cilium morphology. Representative confocal images show defects in cilium morphology during cilium assembly. RPE1 cells were serum starved for 24 h and treated with serum starvation medium containing DMSO as a control or 1 μM GSK-626616. 24 h after treatment, cells were processed for U-ExM and stained for anti-Arl13b (yellow) and anti-acetylated tubulin (magenta). The arrows point to defects in elongated cilia. Scale bars shown on the images are not adjusted for the expansion factor (4.2x). Insets display 10× magnifications of the cilia sections highlighted within a yellow dashed box. <strong>J</strong> Quantification of tip bulging observed in (<strong>I</strong>). Cells treated with GSK-626616 during cilium maintenance are quantified for the bulging at the cilia tip. Data represents mean ± SD of three independent experiments. <em>n</em> &gt; 100 cells per experiment. DMSO: 4.37% ± 1.98 GSK-626616: 14.88% ± 2.79. Statistical significance was determined using Student’s t-test based on the mean values of the three experiments. *<em>P</em> &lt; 0.05.</p></figcaption></figure><p id="Par22">To gain insight into how DYRKs regulate cilium length and architecture, we next investigated the role of DYRK kinases in the maintenance kinetics and stability of the primary cilium. Ciliated cells treated with GSK formed longer cilia with a wide range of lengths, suggesting possible ciliary instability (Figs. <a href="#Fig3" class="usa-link">3D</a>, <a href="#Fig3" class="usa-link">4D</a>). To test this, we compared the dynamics, stability, and length of steady-state cilia between control and GSK-treated cells. We performed live imaging of ciliated RPE1 cells that stably express mNeonGreen fusion of CSPP1 (mNG–CSPP1), a microtubule-associated protein that marks the ciliary axoneme (Movies <a href="#MOESM1" class="usa-link">S1</a>-<a href="#MOESM1" class="usa-link">3</a>). After 24 h of serum starvation, we imaged the cells every 5 min for 14-h post-GSK treatment using full confocal stacks and quantified the cilium length from these movies. In agreement with fixed cell quantifications, DYRK inhibition resulted in longer cilia compared to those in control cells (Fig. <a href="#Fig5" class="usa-link">5B, C</a>). By 12 h, the average cilium length was about 5.6 µm in control cells compared to about 11.1 µm in GSK-treated cells (Fig. <a href="#Fig5" class="usa-link">5C</a>). Importantly, while the cilium length remained relatively constant in control cells, we observed significant variations in length among GSK-treated cells, supporting defects in cilium stability (Fig. <a href="#Fig5" class="usa-link">5A, B</a>). Strikingly, we observed a group of cells in which the entire cilium was lost, a process termed whole-cilium shedding, followed by subsequent cilium regrowth (Fig. <a href="#Fig5" class="usa-link">5A</a>). The incidence of complete cilium shedding increased from 11.52% in control cells to 45.41% in GSK-treated cells, suggesting that cilia become unstable upon DYRK inhibition (Fig. <a href="#Fig5" class="usa-link">5D</a>). These findings highlight the important role of DYRK kinase activity in maintaining the steady-state stability of cilia.</p>
<figure class="fig xbox font-sm" id="Fig5"><h4 class="obj_head">Fig. 5. Inhibiton of DYRK kinase activity causes instability of steady-state cilia.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371027_42003_2025_8373_Fig5_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/b6f8/12371027/69159e555ebc/42003_2025_8373_Fig5_HTML.jpg" loading="lazy" id="d33e1210" height="899" width="800" alt="Fig. 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>A</strong> Inhibition of DYRK kinase activity leads to fluctuations in steady-state cilium length and increased whole-cilium shedding, characterized by the immediate loss of the steady-state cilium. RPE1::mNeonGreen(mNG)-CSPP1 cells were cultured in FluoroDish, serum-starved for 24 h, and then imaged using time-lapse confocal microscopy immediately after the addition of serum starvation medium containing either DMSO as a control or 1 μM GSK-626616. Images were acquired every 5 min over 14 h. Representative still images from the movies show one control cilium maintaining its length throughout the imaging period and one GSK-626616-treated cilium undergoing whole-cilium shedding followed by regrowth. RPE1::mNeonGreen (mNG)-CSPP1 cells were grown in Fluorodish, serum starved for 24 h and imaged using time lapse confocal microscopy immediately after addition of serum starvation medium containing DMSO as a control or 1 μM GSK-626616. Images were acquired every 5 min for 14 h. Representative still images from movies show one control cilium that maintains its length during the time of imaging and one GSK-treated cilium that undergoes whole-cilium shedding followed by cilium growth during the time of imaging. Still images are inverted to better emphasize cilia. Ciliary dynamics are depicted in graphs showing cilium length on the y-axis and time in minutes on the x-axis. Scale bar: 5 μm. <strong>B</strong>,<strong> C</strong> Quantification of changes in steady-state cilium length based on experiments shown in (<strong>A</strong>). <strong>B</strong> shows quantifications of individual cilium lengths on the y-axis against time in minutes on the x-axis for control and 1 μM GSK-626616-treated RPE1 cells. <strong>C</strong> Ciliary kinetics are shown as length curves, measured from mNG-CSPP1 fluorescence in three independent experiments, <em>n</em> &gt; 13 cilia analyzed per experiment. <strong>D</strong> Quantification of intact steady-state cilia and steady-state cilia that underwent whole cilium shedding in control and 1 μM GSK-626616-treated RPE1 cells. Data represents mean ± SD of three independent experiments. <em>n</em> &gt; 25 cells per experiment. Intact cilium: DMSO = 88.48% ± 4.21, GSK-626616 = 54.59% ± 4.80. Whole cilium shedding: DMSO = 11.52% ± 4.21, GSK-626616 = 45.41% ± 4.80. Statistical significance was determined using Student’s t-test based on the mean values of the three experiments. **<em>P</em> &lt; 0.01.</p></figcaption></figure></section><section id="Sec8"><h3 class="pmc_sec_title">Kinase activity of DYRKs controls cilium length by regulating distinct modules involved in cilium length regulation</h3>
<p id="Par23">We hypothesized that the kinase activity of DYRKs might regulate cilium elongation and stability by either promoting the recruitment or retention of factors, or by suppressing or activating regulators involved in these processes. To test this and investigate how DYRK kinase activity regulates cilium length, we used two different approaches. The first approach involved a comparative analysis of centrosomal and ciliary levels of proteins required for cilium length regulation between control and GSK-treated cells. The second approach included depleting or pharmacologically inhibiting regulators of cilium length in GSK-treated cells to determine whether this would restore or exacerbate the cilium elongation phenotype. We identified the regulators of cilium length assessed in these approaches by considering the new interactions revealed by the DYRK3 proximity map, DYRK3’s known roles in the homeostasis of membrane-less organelles, and our knowledge of cilium length regulation. Specifically, we examined whether DYRK kinase activity influences cilium length by regulating or working together with the IFT machinery, centriolar satellites, ciliary kinesins and actin cytoskeleton<sup><a href="#CR36" class="usa-link" aria-describedby="CR36">36</a>–<a href="#CR38" class="usa-link" aria-describedby="CR38">38</a></sup>.</p>
<p id="Par24">The elongation of the ciliary axoneme requires the IFT machinery, which facilitates bidirectional transport of proteins such as tubulin dimers along the axoneme<sup><a href="#CR39" class="usa-link" aria-describedby="CR39">39</a>,<a href="#CR40" class="usa-link" aria-describedby="CR40">40</a></sup>. Therefore, we analyzed the effect of DYRK inhibition on the localization of both the anterograde IFT-B and retrograde IFT-A machinery to the cilia. Using quantitative immunofluorescence, we measured the ciliary levels and concentration of the IFT-B component IFT88 in control and GSK-treated cells. In assembly experiments, GSK-treated cells showed a significant increase in total IFT88 levels and concentration at the cilia compared to control cells (Fig. <a href="#Fig6" class="usa-link">6A, B</a>). In maintenance experiments, IFT88 concentration decreased relative to control cells, even though its levels in the cilia increased (Fig. <a href="#MOESM1" class="usa-link">S5A</a>, <a href="#MOESM1" class="usa-link">S5B</a>). The variation in IFT88 concentration between assembly and maintenance experiments after GSK treatment may be due to a larger increase in cilium length during the maintenance compared to the assembly phases. Similar to GSK treatment, DYRK3 depletion significantly increased both the intensity and concentration of IFT88 at the axoneme, suggesting that DYRK3 regulates IFT (Fig. <a href="#MOESM1" class="usa-link">S5C</a>, <a href="#MOESM1" class="usa-link">S5D</a>). We then determined the effect of the GSK treatment on retrograde transport on the axoneme by staining cells for IFT140, an IFT-A component. GSK-treated cells exhibited a predominant accumulation of IFT140 at the ciliary tip, with a two-fold increase compared to control cells during both cilium assembly and maintenance (Fig. <a href="#Fig6" class="usa-link">6C, D</a>, Fig. <a href="#MOESM1" class="usa-link">S5E</a>, <a href="#MOESM1" class="usa-link">S5F</a>). These results suggest that DYRKs may regulate cilium length by modulating ciliary trafficking through both the IFT-A and IFT-B complexes.</p>
<figure class="fig xbox font-sm" id="Fig6"><h4 class="obj_head">Fig. 6. DYRK kinase activity regulates cilium length by modulating ciliary content, the actin and microtubule cytoskeletons and centriolar satellites.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371027_42003_2025_8373_Fig6_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/b6f8/12371027/8e9d8f54d0d3/42003_2025_8373_Fig6_HTML.jpg" loading="lazy" id="d33e1440" height="994" width="720" alt="Fig. 6"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>A</strong>, <strong>B</strong> Inhibition of DYRK kinase activity leads to changes in ciliary levels and concentration of IFT88. <strong>A</strong> Representative images show the changes ciliary levels and concentration of IFT88. RPE1 cells were treated with serum starvation medium containing DMSO as a control or 1 μM GSK-626616. 24 h after treatment, cells were fixed and stained for anti-acetylated tubulin (Acet-tub) and anti-IFT88. Scale bar, 5 μm. <strong>B</strong> Quantification of ciliary IFT88 levels and the concentration during cilium assembly based on the experiment shown in (A). Data represents mean ± SD of three independent experiments. <em>n</em> &gt; 50 cells per experiment. Normalized ciliary IFT88 levels: DMSO = 1, GSK-626616 = 1.78 ± 0.11. Ciliary IFT88 concentration: DMSO = 1, GSK-626616 = 1.10 ± 0.15. Statistical significance was determined using Student’s t-test based on the mean values of the three experiments. **<em>P</em> &lt; 0.01, ns: non-significant. <strong>C</strong>, <strong>D</strong> Inhibition of DYRK kinase activity leads accumulation of IFT140 at the ciliary tip. <strong>C</strong> Representative images show the accumulation of IFT140 at the ciliary tip during cilium assembly. RPE1 cells were treated with serum starvation medium containing DMSO as a control or 1 μM GSK-626616. 24 h after treatment, cells were fixed and stained for anti-acetylated tubulin (Acet-tub) and anti-IFT140. Scale bar, 2 μm. <strong>D</strong> Quantification of ciliary tip IFT140 levels during cilium assembly based on the experiment shown in (<strong>C</strong>). Data represents mean ± SD of three independent experiments. <em>n</em> &gt; 43 cells per experiment. Normalized IFT140 ciliary tip levels: DMSO = 1, GSK-626616 = 1.94 ± 0.49. Statistical significance was determined using Student’s t-test based on the mean values of the three experiments. *<em>P</em> &lt; 0.05. <strong>E</strong>, <strong>F</strong> Depletion of KIF7 did not further elongate cilia in cells inhibited for the kinase activity of DYRKs. <strong>E</strong> Representative images show the effects of KIF7 depletion on cilium length in control and GSK-626616-treated cells. RPE1 cells were transfected with control and KIF7 siRNAs. Following 24 h of serum starvation, cells were treated with either DMSO as a control or 1 μM GSK-6266160 for 6 h in serum stimulation medium. Cells were then fixed and stained for anti-acetylated tubulin and anti-gamma-tubulin. Scale bar, 5 μm. <strong>F</strong> Quantification of cilium length based on the experiment shown in (<strong>E</strong>). Data represents mean ± SD of three independent experiments. <em>n</em> &gt; 70 cells per experiment. siControl+DMSO = 3.40 μm ± 0.09, siKIF7+DMSO = 5.61 μm ± 0.72,; siControl+GSK-626616 = 7.62 μm ± 1.2, siKIF7+GSK-626616 = 6.43 μm ± 1.07,. Statistical significance was determined using Student’s t-test based on the mean values of the three experiments. **<em>P</em> &lt; 0.01, ****<em>P</em> &lt; 0.0001, ns: non-significant. <strong>G</strong>, <strong>H</strong> CytoD treatment did not further elongate cilia in the cells inhibited for the kinase activity of DYRKs. <strong>G</strong> Representative images show the effect of Cytochalasin D (CytoD) treatment on cilium length in control and GSK-626616-treated cells. RPE1 cells were serum starved for 24 h and treated with serum starvation medium containing DMSO as a control or 1 μM GSK-626616. During the last 3 h of GSK-626616 treatment, cells were treated with 500 nM Cytochalasin D (CytoD), then fixed and stained for acetylated tubulin (Acet-tub) and gamma-tubulin. Scale bar, 5 μm. <strong>H</strong> Quantification of cilium length elongation based on the experiment shown in (<strong>G</strong>). Data represents mean ± SD of three independent experiments. <em>n</em> &gt; 70 cells per experiment. DMSO = 3.92 μm ± 0.15, GSK-626616 = 11.69 μm ± 1.77, CytoD: 5.48 μm ± 0.41, GSK-626616+CytoD: 11.22 μm ± 2.02. Statistical significance was determined using Student’s t-test based on the mean values of the three experiments. **<em>P</em> &lt; 0.01, ns: non-significant. <strong>I</strong>, <strong>J</strong> Depletion of PCM1 reduced the cilia length in cells inhibited for the kinase activity of DYRKs. <strong>I</strong> Representative images show the effect of PCM1 depletion on cilium length in control and GSK-626616-treated cells. RPE1 cells were transfected with control and PCM1 siRNA. Following 24 h of serum starvation, cells were treated with either DMSO as a control or 1 μM GSK-6266160 for 24 h in serum stimulation medium. Cells were then fixed and stained for anti-acetylated tubulin and anti-gamma-tubulin. Scale bar, 5 μm. <strong>J</strong> Quantification of cilium length based on the experiment shown in (<strong>I</strong>). Data represents mean ± SD of three independent experiments. <em>n</em> &gt; 64 cells per experiment. siControl+DMSO = 4.44 μm ± 0.74, siPCM1+DMSO = = 3.21 μm ± 0.14, siControl+GSK-626616 = 13.60 μm ± 1.28, siPCM1+GSK-626616 = = 11.56 μm ± 2.32. Statistical significance was determined using Student’s t-test based on the mean values of the three experiments. *<em>P</em> &lt; 0.05, ns: non-significant. <strong>K</strong>, <strong>L</strong> Inhibition of DYRK kinase activity disrupts pericentrosomal localization and integrity of centriolar satellites. <strong>K</strong> Representative images show the effect of GSK-626616 treatment on the pericentrosomal levels of the centriolar satellite marker PCM1 during cilium assembly. RPE1 cells were treated with serum starvation medium containing DMSO as a control or 1 μM GSK-626616. 24 h after treatment, cells were fixed and stained for anti-PCM1, anti-acetylated tubulin, and anti-gamma-tubulin. Scale bar, 5 μm. <strong>L</strong> Quantification of pericentrosomal PCM1 levels based on the experiment shown in (<strong>K</strong>). Data represents mean ± SD of three independent experiments. <em>n</em> &gt; 88 cells per experiment. DMSO = 1, GSK-626616 = 0.40 ± 0.06. ****<em>P</em> &lt; 0.0001. <strong>M</strong>, <strong>N</strong> Inhibition of DYRK kinase activity disrupts pericentrosomal localization and integrity of centriolar satellites during cilium maintenance. <strong>M</strong> Representative images show the effect of GSK-626616 treatment on the pericentrosomal levels of the centriolar satellite marker PCM1 during cilium maintenance. RPE1 cells were serum starved for 24 h and treated with serum starvation medium containing DMSO as a control or 1 μM GSK-626616. 24 h after treatment, cells were fixed and stained for anti-PCM1, anti-acetylated tubulin, and anti-gamma-tubulin. Scale bar, 5 μm. <strong>N</strong> Quantification of pericentrosomal PCM1 levels based on the experiment shown in (<strong>M</strong>). Data represents mean ± SD of three independent experiments. <em>n</em> &gt; 100 cells per experiment. Normalized PCM1 levels at the basal body: DMSO = 1, GSK-626616 = 0.15 ± 0.03. Statistical significance was determined using Student’s t-test based on the mean values of the three experiments. ****<em>P</em> &lt; 0.0001.</p></figcaption></figure><p id="Par25">Similar to GSK inhibition, depletion of the non-motile kinesin KIF7 was shown to elongate cilia by increasing the distal-to-proximal segment ratio<sup><a href="#CR35" class="usa-link" aria-describedby="CR35">35</a></sup>. Moreover, KIF7 was shown to repress cilia elongation by modulating tubulin polymerization dynamics at the cilium tip<sup><a href="#CR41" class="usa-link" aria-describedby="CR41">41</a>–<a href="#CR43" class="usa-link" aria-describedby="CR43">43</a></sup>. To explore a potential link between DYRKs and KIF7 in cilium length regulation, we first tested their interaction using co-immunoprecipitation experiments. KIF7 co-precipitated with FLAG-DYRK3-WT, and this interaction was not affected by GSK treatment (Fig. <a href="#MOESM1" class="usa-link">S5G</a>). We then examined the effect of KIF7 depletion on cilium length in cells treated with either DMSO control or GSK. KIF7 was efficiently depleted in both control and GSK-treated RPE1 cells, as confirmed by immunoblotting (Fig. <a href="#MOESM1" class="usa-link">S5H</a>). While KIF7 depletion led to cilium elongation in control cells, it did not have additive effect in GSK-treated cells, suggesting that DYRKs act upstream of KIF7 in the regulation of ciliary microtubules during cilium length control (Fig. <a href="#Fig6" class="usa-link">6E, F</a>).</p>
<p id="Par26">In addition to the microtubule cytoskeleton, actin cytoskeleton plays important roles during primary cilium formation and length. Depolymerization of actin filaments was shown to promote ciliary growth<sup><a href="#CR44" class="usa-link" aria-describedby="CR44">44</a>,<a href="#CR45" class="usa-link" aria-describedby="CR45">45</a></sup>. Consistent with this, treatment of RPE1 cells with the actin polymerization inhibitor cytochalasin D (CytoD) led to a significant increase in cilium length (Fig. S<a href="#MOESM1" class="usa-link">6B</a>, Fig. <a href="#Fig6" class="usa-link">6G, H</a>). However, CytoD treatment did not further elongate cilia in GSK-inhibited cells, suggesting that DYRKs might function upstream of actin cytoskeleton (Fig. <a href="#Fig6" class="usa-link">6G, H</a>).</p>
<p id="Par27">GSK treatment was shown to inhibit the mitotic disassembly of various membrane-less organelles including centriolar satellites<sup><a href="#CR14" class="usa-link" aria-describedby="CR14">14</a></sup>. Centriolar satellites regulate primary cilium formation and function by controlling the centrosomal and ciliary levels of proteins involved in these processes<sup><a href="#CR33" class="usa-link" aria-describedby="CR33">33</a>,<a href="#CR46" class="usa-link" aria-describedby="CR46">46</a>–<a href="#CR48" class="usa-link" aria-describedby="CR48">48</a></sup>. Given their ciliary roles and known links to DYRK3, we investigated the functional relationship between DYRKs and centriolar satellites by testing how PCM1 depletion affects GSK-induced cilium elongation. PCM1 was efficiently depleted in RPE1 cells, as confirmed by immunoblotting (Fig. <a href="#MOESM1" class="usa-link">S5I</a>). PCM1 depletion significantly shortened cilia in control cells and caused a slight, statistically non-significant decrease in cilium length in GSK-treated cells (Fig. <a href="#Fig6" class="usa-link">6I, J</a>). Notably, centriolar satellites marked by PCM1 staining were dispersed throughout the cytoplasm in GSK-treated cells during both cilium assembly and maintenance, and PCM1 cellular levels were reduced relative to controls (Fig. <a href="#Fig6" class="usa-link">6K–N</a>, Fig. <a href="#MOESM1" class="usa-link">S5J</a>). Together, these results suggest that DYRKs might regulate cilium length in part through modulating centriolar satellite distribution and abundance.</p></section></section><section id="Sec9"><h2 class="pmc_sec_title">Discussion</h2>
<p id="Par28">In this study, we mapped the proximity interactome of DYRK3, which confirmed multiple known interactors and revealed new associations. Using both wild-type and kinase-dead DYRK3, we demonstrated that its kinase activity influences the interaction landscape, consistent with previous studies<sup><a href="#CR14" class="usa-link" aria-describedby="CR14">14</a></sup>. Cross-referencing our data with previous AP-MS and TurboID studies of DYRK2 and DYRK3 revealed only partial overlap. This likely reflects differences in labeling duration, bait expression, and methodological sensitivity for DYRK3, as well as distinct functional roles or mechanisms of action of DYRK2 and DYRK3<sup><a href="#CR13" class="usa-link" aria-describedby="CR13">13</a>,<a href="#CR14" class="usa-link" aria-describedby="CR14">14</a>,<a href="#CR20" class="usa-link" aria-describedby="CR20">20</a>,<a href="#CR30" class="usa-link" aria-describedby="CR30">30</a></sup>. Notably, DYRK3 proximity map revealed a sub-network of ciliary and centriolar satellite proteins, and we validated several of these interactions such as KIF7 and PCM1 by immunoprecipitation, suggesting a previously unrecognized role for DYRK3 in primary cilium regulation.</p>
<p id="Par29">RNAi-mediated DYRK3 depletion led to cilium elongation and increased ciliary IFT88 levels, confirming its roles in regulation of cilium length and composition. Co-depletion of DYRK3 and DYRK2, a well-characterized regulator of cilium length, led to an even greater increase in cilium length, with DYRK2 playing a more prominent role in this process<sup><a href="#CR19" class="usa-link" aria-describedby="CR19">19</a>–<a href="#CR21" class="usa-link" aria-describedby="CR21">21</a></sup>. This supports cooperation between DYRK2 and DYRK3 in regulating cilium length. Notably, rescue experiments showed the two kinases can partially compensate for one another, suggesting they cooperate through both shared and distinct targets. Finally, pharmacological inhibition of DYRK kinases with GSK further enhanced cilium elongation in DYRK2- or DYRK3-depleted cells and lead to a greater cilium elongation compared to DYRK3 depletion alone, indicating that additional DYRK family members also contribute to cilium length regulation. Future studies are required to delineate the specific contributions of these kinases to cilium length control and elucidate their in vivo functions using knockout mouse models.</p>
<p id="Par30">Pharmacological inhibition of DYRK kinase activity with GSK during cilium assembly and maintenance specifically increased the distal-to-proximal segment ratio, which has been shown to be critical for cilium stability, kinetics and signaling<sup><a href="#CR35" class="usa-link" aria-describedby="CR35">35</a></sup>. In agreement, steady-state cilia in GSK-treated cells revealed increased fluctuations in cilia length and more frequent whole-cilia shedding, often followed by cilia regrowth. These cilia exhibited morphological defects including elongation or curving of cilia and bulging of the ciliary membranes at their distal ends. While these results highlight the role of DYRK kinase activity in maintaining cilium stability, the underlying mechanisms remain unclear. One possibility is that distal segment elongation creates fragile axonemal regions that are more prone to ectocytosis, potentially accounting for the observed cilium instability. Notably, we did not find changes in ciliary acetylated tubulin levels following GSK treatment, suggesting that DYRKs do not promote cilium stability through tubulin acetylation. Instead, they may act via alternative tubulin post-translational modifications or by phosphorylating proteins involved in regulating ciliary stability.</p>
<p id="Par31">To further elucidate how DYRKs regulate cilium length, we examined whether GSK-induced cilium elongation results from altered localization or activity of known cilium regulators. Specifically, we examined DYRKs’ functional relationship to negative regulators such as the actin cytoskeleton and the ciliary kinesin KIF7, as well as positive regulators like the IFT machinery and centriolar satellites. Among these, only depletion of the centriolar satellite scaffold PCM1 modestly suppressed the GSK-induced elongation phenotype, while perturbations of other factors did not affect cilium length. Moreover, GSK treatment impaired ciliary trafficking via both IFT-A and IFT-B complexes, potentially contributing to cilium elongation by affecting the transport of ciliary cargoes into or out of the cilium. Collectively, these findings position DYRKs as central regulators of cilium length, acting through multiple pathways including centriolar satellite dynamics and IFT-mediated trafficking.</p>
<p id="Par32">Among the ciliary regulators tested, we identified interactions between DYRK3 and both KIF7 and PCM1 using pulldown experiments. KIF7 is a well-characterized negative regulator of cilium length that controls microtubule plus-end dynamics at the ciliary tip<sup><a href="#CR41" class="usa-link" aria-describedby="CR41">41</a>,<a href="#CR42" class="usa-link" aria-describedby="CR42">42</a>,<a href="#CR49" class="usa-link" aria-describedby="CR49">49</a>,<a href="#CR50" class="usa-link" aria-describedby="CR50">50</a></sup>. The lack of further cilium elongation upon KIF7 depletion in GSK-treated cells suggests that DYRKs act upstream of KIF7, possibly by regulating its cytoplasmic or ciliary pools and modulating its localization or activation at the ciliary tip. Given the role of centriolar satellites in storing and trafficking centrosome and ciliary proteins, it is plausible that DYRK3-KIF7 interaction occurs at these structures<sup><a href="#CR48" class="usa-link" aria-describedby="CR48">48</a>,<a href="#CR51" class="usa-link" aria-describedby="CR51">51</a>,<a href="#CR52" class="usa-link" aria-describedby="CR52">52</a></sup>. Further supporting a connection between DYRKs and centriolar satellites, DYRK3 was shown to promote centriolar satellite disassembly during mitosis<sup><a href="#CR14" class="usa-link" aria-describedby="CR14">14</a></sup>. In contrast, our data reveal that GSK treatment disrupts pericentrosomal centriolar satellite clustering and induces their disassembly in quiescent cells, in part by decreasing the PCM1’s cellular levels. These findings suggest that DYRK3 regulates centriolar satellites through distinct mechanisms depending on the cell cycle stage. Future studies should elucidate how DYRKs regulate PCM1 abundance and centriolar satellite dynamics across different contexts.</p>
<p id="Par33">DYRK kinases have been studied as drug targets due to their links to vital cellular and organismal processes and diseases. GSK was originally characterized as a potent DYRK3 inhibitor with potential for anemia therapy<sup><a href="#CR7" class="usa-link" aria-describedby="CR7">7</a></sup>. Since then, it has been widely used to study DYRK3’s roles in the regulation of membrane-less organelles, mTORC signaling and the secretory pathway<sup><a href="#CR13" class="usa-link" aria-describedby="CR13">13</a>–<a href="#CR15" class="usa-link" aria-describedby="CR15">15</a></sup>. However, a recent in vitro kinome-wide specificity profiling across 451 kinases demonstrated that GSK inhibits multiple DYRK family members, not just DYRK3<sup><a href="#CR15" class="usa-link" aria-describedby="CR15">15</a></sup>. This broader target profile underscores the need for caution when interpreting studies that rely solely on this inhibitor and emphasizes the importance of developing more selective DYRK inhibitors. In this study, we explicitly attribute GSK-induced ciliary phenotypes to inhibition of DYRK kinases more broadly, rather than exclusively to DYRK3. Importantly, we showed that GSK treatment affects multiple aspects of cilium biology, providing new context for its application in future studies of DYRK function.</p>
<p id="Par34">Collectively, our findings establish DYRKs as key regulators of cilium length, stability and morphology during assembly and maintenance. Because many ciliopathies involve either abnormally short or excessively long cilia that compromise signaling, our results provide new insights into the pathogenesis of these diseases and may inform the development of targated therapeutic strategies<sup><a href="#CR53" class="usa-link" aria-describedby="CR53">53</a>–<a href="#CR57" class="usa-link" aria-describedby="CR57">57</a></sup></p></section><section id="Sec10"><h2 class="pmc_sec_title">Methods</h2>
<section id="Sec11"><h3 class="pmc_sec_title">Cell culture, transfection and treatments</h3>
<p id="Par35">RPE1 cells were grown in DMEM/F12 50/50 (Pan Biotech, Cat. # P04-41250) supplemented with 10% fetal bovine serum (FBS, Life Technologies, Ref. # 10270-106, Lot # 42Q5283K)) and 1% penicillin-streptomycin (P/S). HEK293T cells were grown in DMEM (Pan Biotech, Cat. # P04-03590) supplemented with 10% FBS and 1% P/S. All cells were cultured at 37 °C and 5% CO<sub>2</sub>. RPE1::mNG-CSPP1 cell lines were described previously<sup><a href="#CR58" class="usa-link" aria-describedby="CR58">58</a></sup>. All cell lines were tested for mycoplasma by MycoAlert Mycoplasma Detection Kit (Lonza). HEK293T cells were transfected with the plasmids using 1 μg/μl polyethylenimine, MW 25 kDa (PEI, Sigma-Aldrich, St. Louis, MO). Briefly, the plasmids were diluted in Opti-MEM (Invitrogen) and incubated with PEI for 45 min at room temperature. Cells were incubated for 48 h with the transfection mix, and the expression of fusion proteins in transiently expressing cells were confirmed by immunofluorescence and immunoblotting.</p>
<p id="Par36">For cilium assembly and maintenance experiments, cells were grown on coverslips until they reached 100% confluency. For cilium assembly, serum starvation medium (DMEM F12 supplemented with 1% P/S without FBS) was added along with 1 μM GSK-626616 (Tocris Bioscience, Cat#6638) and incubated for 24 h. For cilium maintenance, cells were first treated with serum starvation medium for 24 h. Subsequently, the ciliated cells were treated with 1 μM GSK-626616 for an additional 24 h. Cells were fixed and stained with the indicated antibodies. For experiments involving DYRK3 inhibition combined with KIF7 depletion, the treatment duration with 1 µM GSK-626616 was shortened to 6 h.</p>
<p id="Par37">For actin depolymerization experiments, cells were treated with 500 nM Cytochalasin D (CytoD) for 3 h. In co-treatment experiments with GSK-626616 and CytoD, cells were initially cultured in serum starvation medium for 24 h, followed by the addition of 1 µM GSK-626616 in the same medium for an additional 24 h. CytoD was introduced into the medium during the final 3 h of the GSK-626616 treatment. Cells were then fixed and stained with the specified antibodies.</p></section><section id="Sec12"><h3 class="pmc_sec_title">Plasmids and siRNA transfections</h3>
<p id="Par38">pENTR_D_TOPO_WT (wild type) DYRK3 and pENTR_D_TOPO_KD (kinase dead) DYRK3 plasmids were kindly provided by the Pelkmans Laboratory at the University of Zurich. The pDEST-FlagBirA*-DYRK3-WT and pDEST-FlagBirA*-DYRK3-KD constructs were generated through Gateway recombination, using the the respective DYRK3 constructs and pDEST- pcDNA5.1-FRT/TO-BirA*-Flag plasmid, which was provided by Anne-Claude Gingras at the University of Toronto, Canada.</p>
<p id="Par39">Cells were seeded onto coverslips at 70% confluency and transfected with 50 nM of siRNAs in two sequential transfections using Lipofectamine RNAiMAX (Life Technologies) in OPTIMEM (Life Technologies) according to the manufacturer’s instructions. DYRK3 was depleted using an siRNA with the sequence (5′-CGGAUUUUGGAGCAUCUUAtt-3′), which was previously validated<sup><a href="#CR14" class="usa-link" aria-describedby="CR14">14</a></sup>. For KIF7 depletion, two siRNAs (5′-UGCAGGAGCUCGAGCGGAAGUGCA-3′) and 5′-GCAGAUUGCCUUCUCGGAAUGGAGA-3′) were pooled together (Integrated DNA Technologies). PCM1 was depleted using an siRNA with the sequence (5′-UCAGCUUCGUGAUUCUCAG-3′), which was previously validated<sup><a href="#CR25" class="usa-link" aria-describedby="CR25">25</a></sup>. For DYRK2 depletion, a previously validated siRNA (5′-GCC<a href="https://www.sciencedirect.com/topics/pharmacology-toxicology-and-pharmaceutical-science/abnormal-grain-growth" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">AGG</a>UAUGGCAUGCCCAUUGAU-3′; Integrated DNA Technologies) was used<sup><a href="#CR59" class="usa-link" aria-describedby="CR59">59</a></sup>. A non-targeting siRNA (5′-AUUGUUUACAUAACCGGACAUAAUC-3′; Integrated DNA Technologies) served as a control for RNAi experiments.</p></section><section id="Sec13"><h3 class="pmc_sec_title">Biotin-streptavidin affinity purification</h3>
<p id="Par40">For the BioID experiments, HEK293T cells were grown in complete medium on 5 × 15 cm plates until they reached 90% confluency. They were then transfected with FLAG-BirA*-, FLAG-BirA*-DYRK3-WT or FLAG-BirA*-DYRK3-KD expression constructs. 24 h post-transfection, cells were treated with 50 μM biotin for 18 h. After biotin treatment, cells were lysed in lysis buffer (20 mM HEPES, pH 7.8, 5 mM K-acetate, 0.5 mM MgCl<sub>2</sub>, 0.5 mM DTT, protease inhibitors) and sonicated. An equal volume of 4 °C 50 mM Tris (pH 7.4) was added to the extracts, and the insoluble material was pelleted by centrifugation. The soluble fraction from the whole cell lysates was then incubated with Streptavidin agarose beads (Thermo Scientific). Beads were collected and washed twice in wash buffer 1 (2% SDS in dH2O), once with wash buffer 2 (0.2% deoxycholate, 1% Triton X-100, 500 mM NaCI, 1 mM EDTA, and 50 mM Hepes, pH 7.5), once with wash buffer 3 (250 mM LiCI, 0.5% NP-40, 0.5% deoxycholate, 1% Triton X-100, 500 mM NaCI, 1 mM EDTA and 10 mM Tris, pH 8.1), and twice with wash buffer 4 (50 mM Tris, pH 7.4, and 50 mM NaCI). 10% of the sample was reserved for Western blot analysis, and 90% of the sample to be analyzed by mass spectrometry was washed twice in 50 mM NH<sub>4</sub>HCO<sub>3</sub>. Confirmation of DYRK3 proximity interactors was performed by immunoblotting using FLAG antibodies to detect the fusion protein and streptavidin to identify biotinylated proteins.</p></section><section id="Sec14"><h3 class="pmc_sec_title">FLAG affinity pulldown</h3>
<p id="Par41">HEK293T cells were transfected with FLAG-DYRK3-WT and FLAG-DYRK3-KD plasmids. FLAG used as a negative control. 48 h after transfection, cells were washed and lysed with FLAG pulldown buffer (50 mM HEPES, pH 8, 100 mM KCl, 2 mM EDTA, 10% glycerol, 0.1% NP40 freshly supplemented with protease inhibitors, 10 μg/ml Leupeptin, Pepstatin, and Chymostatin, 1 mM PMSF). Lysates were centrifuged at 13,000 rpm for 10 min at 4 °C and supernatants were transferred to a tube. 50 μl from each sample was saved as input. The rest of the supernatant was immunoprecipitated with FLAG M2 affinity gel (Sigma, A2220) for 90 min at 4 °C. After washing 3× with FLAG pulldown lysis buffer, samples were resuspended in SDS containing sample buffer. The samples were immunoblotted with the indicated antibodies.</p></section><section id="Sec15"><h3 class="pmc_sec_title">Immunofluorescence, microscopy and image analysis</h3>
<p id="Par42">For immunofluorescence experiments, cells were grown on coverslips, washed with PBS, and fixed by either 4% PFA diluted in cytoskeleton buffer (100 mM NaCl (Sigma-Aldrich, S9888), 300 mM sucrose (Sigma-Aldrich, S0389), 3 mM MgCl<sub>2</sub> (Sigma-Aldrich, M2670), and 10 mM PIPES (Sigma-Aldrich, P6757)) for 10 min at room temperature or ice-cold methanol at - 20 °C for 10 min. After washing three times with PBS, cells were blocked with 3% BSA (Capricorn Scientific) in PBS plus 0.1% Triton X-100 and incubated with primary antibodies in blocking solution for 1 h at room temperature. Cells were washed three times with PBS and incubated with secondary antibodies at 1:2000 dilution for 1 h and DAPI (Thermo Fisher Scientific cat#D1306) at 1:5000 for 5 min at room temperature. Following three washes with PBS, cells were mounted using Mowiol mounting medium containing N-propyl gallate (Sigma-Aldrich). Anti-PCM1 antibody was generated and used for immunofluorescence as previously described<sup><a href="#CR26" class="usa-link" aria-describedby="CR26">26</a></sup>. Other antibodies used for immunofluorescence in this study were mouse anti-tubulin (GTU-88; Sigma-Aldrich-T6557) at 1:2000, mouse anti-acetylated tubulin (Santa Cruz Biotechnology, sc-23950) at 1:10,000, mouse anti-FLAG IgG1 (Proteintech, 66008-3-Ig) at 1:1000, rabbit anti-Arl13b (Proteintech, 17711-1-AP) at 1:500, mouse anti-glutamylated tubulin (Adipogen, AG-20B-0020 clone GT335) at 1:1000 and rabbit anti-IFT88 (Proteintech, 12780-1-AP) 1:1000. Following primary antibody incubation and 3× PBS wash, coverslips were incubated with Alexa Flour 488-, 568-, or 633-coupled (Life Technologies), and they were used at 1:2000. Biotinylated proteins were detected with streptavidin coupled to Alexa Fluor 488 or 594 (Thermo Fisher). DNA was stained with 40, 6-diamidino-2- phenylindole (DAPI; 1 lg/ml). Samples were mounted using Mowiol mounting medium containing N-propyl gallate (Sigma-Aldrich).</p>
<p id="Par43">For the assessment of protein localization and level quantifications, images were acquired with Leica DMi8 fluorescent microscope with a stack size of 5 µm and step size of 0.25 µm in 1024 × 1024 format using an HC PL APO CS2 63×1.4 NA oil objective. Higher resolution images were taken by using an HC PL APO CS2 63×1.4 NA oil objective with Leica SP8 confocal microscope 1024 × 1024 pixel format, with pixel size ranging from 90 to 45 nm. Time-lapse live imaging was performed with Leica SP8 confocal microscope equipped with an incubation chamber using HC PL APO CS2 63 × 1.4 NA oil objective. For imaging primary cilium dynamics during DYRK3 inhibition in cilium maintenance, cells were incubated with DMEM/F12 containing 0.5% FBS for 24 h, followed by the addition of 1 µM GSK-616626. Imaging was performed with a frequency of 5 min per frame, with a 1 μm step size and a 10 μm stack size, in a 1024 × 1024 pixel format for 14 h. All data acquisition was done in a blinded manner.</p>
<p id="Par44">The percentage of ciliated cells was determined by counting the total number of cells and the number of cells with primary cilia, as identified through DAPI staining of nuclei and either Arl13b or acetylated tubulin staining of the cilia. Cilium curvature analysis was performed by dividing the total cilium length, quantified by acetylated tubulin, by the shortest distance from the basal body to the cilium tip. For quantitative immunofluorescence of ciliary and centrosomal protein levels, z-stacks of cells were acquired using identical gain and exposure settings, which were determined by the fluorescence signal in control cells. These z-stacks were used to create maximum-intensity projections using ImageJ (National Institutes of Health, Bethesda, MD). Ciliary regions were identified by the Arl13b and/or acetylated tubulin signals. Centrosome regions were defined by centrosomal marker staining for each cell, and the total pixel intensity of a circular 3 μm² area centered on the centrosome in each cell was measured using ImageJ and defined as the centrosomal intensity. Background intensity was subtracted from the centrosomal and ciliary fluorescence intensities. This subtraction was performed by quantifying fluorescence intensity in a region of equal dimensions in the area adjacent to the primary cilium or centrosome. Ciliary protein concentration was calculated by dividing the total ciliary fluorescence signal of the protein of interest to the cilium area, which was quantified using Arl13B or acetylated tubulin staining. Ciliary protein concentrations and centrosomal protein levels were normalized relative to the control group’s mean ( = 1). Statistical analysis was performed by normalizing these values to their mean.</p></section><section id="Sec16"><h3 class="pmc_sec_title">Cell lysis and immunoblotting</h3>
<p id="Par45">Cells were lysed in 50 mM Tris (pH 7.6), 150 mM NaCI, 1% Triton X-100, and protease inhibitors for 30 min at 4 <sup>o</sup>C followed by centrifugation at 15,000 <em>g</em> for 15 min. The protein concentration of the resulting supernatants was determined with the Bradford solution (Bio-Rad Laboratories, CA, USA). For immunoblotting, equal quantities of cell extracts were resolved on SDS–PAGE gels, transferred onto nitrocellulose membranes, blocked with TBST in 5% milk for 1 h at room temperature. Blots were incubated with primary antibodies diluted in 5% BSA in TBST overnight at 4 °C, washed with TBST three times for 5 min, and blotted with secondary antibodies for 1 h at room temperature. After washing blots with TBST three times for 5 min, they were visualized with the LI-COR Odyssey Infrared Imaging System and software at 169 mm (LI-COR Biosciences).Primary antibodies used for Western blotting were rabbit anti-PCM1 (Proteintech-198561-1-AP) at 1:1000, rabbit anti-FLAG (Cell signaling,#2368) at 1:1000, mouse anti-vinculin (Santa Cruz Biotechnology, sc-55465), rabbit anti-CEP131 (Bethyl Laboratories) at 1:1000, rabbit anti-CCDC138 (Sigma Aldrich, HPA049899) at 1:1000, mouse anti-gamma tubulin (Sigma Aldrich, T6557) at 1:1000, mouse anti-acetylated tubulin (Santa Cruz Biotechnology, sc-23950) at 1:1000, mouse anti-DYRK3 (Santa Cruz Biotechnology, sc-390532) at 1:1000 and rabbit anti-KIF7 (gift from Kathlyn V. Anderson) at 1:1000<sup><a href="#CR41" class="usa-link" aria-describedby="CR41">41</a></sup>. Secondary antibodies used for Western blotting were IRDye 680-, IRDye 800-coupled and streptavidin (Invitrogen, 521378) and were used 1:10000 (LI-COR Biosciences).</p>
<p id="Par46">Quantification of band intensities from the immunoblots were performed using ImageJ. A square was drawn around the bands for PCM1 and vinculin under different treatments, and both the band and background intensities were measured. The background intensity was subtracted from the band intensity. Intensities of the PCM1 bands were normalized to their respective vinculin band intensities and presented as a percentage.</p></section><section id="Sec17"><h3 class="pmc_sec_title">U-ExM and image analysis</h3>
<p id="Par47">U-ExM was performed as previously described<sup><a href="#CR60" class="usa-link" aria-describedby="CR60">60</a></sup>. Briefly, RPE1 cells were grown on 12 mm coverslips in a 24-well plate. Coverslips were incubated in 1.4% formaldehyde/2% acrylamide solution in 1X PBS for 5 h at 37 °C. Cells are embedded into a gel prepared with Monomer Solution (for one gel, 25 μl of sodium acrylamide, stock solution at 38% [wt/wt] diluted with nuclease-free water, 12.5 μl of acrylamide, 2.5 μl of bisacrylamide, and 5 μl of 10X PBS) supplemented with tetramethylethylenediamine and ammonium persulfate (final concentration of 0.5%) for 1 h at 37 °C. Denaturation was performed at 95 °C for 90 min and gels were stained with primary antibodies for 3 h at 37 °C. Gels were washed three times for 10 min at RT with 1X PBS with 0.1% Triton-X (PBS-T). Secondary antibody incubation is carried out for 2 h 30 min at 37 °C and gels are washed with three times for 10 min washes in PBS-T at RT. Gels were expanded in 100 ml distilled water three times before imaging. The diameter of the gels is measured with a ruler and the expansion factor is calculated by dividing the diameter to 12 mm. Gels were cropped into pieces and mounted to 24-mm coverslips coated with Poly-D-lysine.</p>
<p id="Par48">U-ExM samples were imaged using a Leica SP8 confocal microscope equipped with an HC PL APO CS2 63 × 1.4 NA oil objective and 0.30 μm z stacks, and the images were denoised using the Huygens software. The primary antibodies used in these experiments are rabbit anti-Arl13b (1:500, Proteintech, 17711-1-AP), mouse anti-acetylated tubulin (1:1000, Santa Cruz Biotechnology, sc-23950) and mouse anti-glutamylated tubulin (1:500, Adipogen, AG-20B-0020 clone GT335). The secondary antibodies used in these experiments are goat anti-rabbit IgG 488 (A21141; Life Technologies), and goat anti-mouse IgG2b 568 (A21144; Invitrogen) at 1:1000, and goat anti-mouse IgG1 633 (1:1000, A21126; Invitrogen).</p></section><section id="Sec18"><h3 class="pmc_sec_title">Mass spectrometry analysis</h3>
<p id="Par49">Proteins bound to beads were reduced by 20-min incubation with 5 mM TCEP (tris(2-carboxyethyl) phosphine) and alkylated in the dark by treatment with 10 mM Iodoacetamide for 20 additional minutes. The proteins were subsequently digested by adding Sequencing Grade Modified Trypsin (Promega, Madison, WI, USA) and placing the reaction mixture in a Thermomixer (Eppendorf, Westbury, NY) and incubating overnight at 37 °C at 750 rpm. The next day, the sample was acidified with formic acid to a final concentration of 5% and spun at 22,000 g for 30 min. The supernatant was carefully transferred to a separate microfuge tube so as not to disturb the bead pellet and pressure-loaded into a biphasic trap column. MS analysis of the samples was performed using MudPIT technology [S7]. Capillary columns were prepared in-house from particle slurries in methanol. An analytical column was generated by pulling a 100 lm ID/360 lm OD capillary (Polymicro Technologies, Inc., Phoenix, AZ) to 3 lm ID tip. The pulled column was packed with reverse-phase particles (Aqua C18, 3 lm dia., 90 A pores, Phenomenex, Torrance, CA) until 15 cm long. A biphasic trapping column was prepared by creating a Kasil frit at one end of an undeactivated 250 lm ID/360 lm OD capillary (Agilent Technologies, Inc., Santa Clara, CA), which was then successively packed with 2.5 cm strong cation exchange particles (Partisphere SCX, 5 lm dia., 100 A pores, Phenomenex, Torrance, CA) and 2.5 cm reverse-phase particles (Aqua C18, 5 lm dia., 90 A pores, Phenomenex, Torrance, CA). The trapping column was equilibrated using buffer A (5% acetonitrile/0.1% formic acid) prior to sample loading. After sample loading and prior to MS analysis, the resin-bound peptides were desalted with buffer A by letting it flow through the trap column. The trap and analytical columns were assembled using a zero-dead volume union (Upchurch Scientific, Oak Harbor, WA). LCMS/MS analysis was performed on LTQ Orbitrap or LTQ Orbitrap Velos (Thermo Scientific, San Jose, CA, USA) interfaced at the front end with a quaternary HP 1100 series HPLC pump (Agilent Technology, Santa Clara, CA, USA) using an in-house built electrospray stage. Electrospray was performed directly from the analytical column by applying the ESI voltage at a tee (150 lm ID, Upchurch Scientific) directly downstream of a 1:1000 split flow used to reduce the flow rate to 250 nl/min through the columns. A fully automated 6-step MudPIT run was performed on each sample using a three mobile phase system consisting of buffer A (5% acetonitrile/0.1% formic acid), buffer B (80% acetonitrile/0.1% formic acid), and buffer C (500 mM ammonium acetate/5% acetonitrile/0.1% formic acid). The first step was 60 min reverse-phase run, whereas five subsequent steps were of 120 min duration with different concentration of buffer C run for 4 min at the beginning of each of the gradient. In LTQ Orbitrap Velos, peptides were analyzed using a Top-20 data-dependent acquisition method in which fragmentation spectra are acquired for the top 20 peptide ions above a predetermined signal threshold. As peptides were eluted from the microcapillary column, they were electrosprayed directly into the mass spectrometer with the application of a distal 2.4 kV spray voltage. For each cycle, fullscan MS spectra (m/z range 300–1600) were acquired in the Orbitrap with the resolution set to a value of 60,000 at m/z 400 and an automatic gain control (AGC) target of 1 × 106 ions and the maximal injection time of 250 ms. For MS/MS scans, the target value was 10,000 ions with injection time of 25 ms. Once analyzed, the selected peptide ions were dynamically excluded from further analysis for 120 s to allow for the selection of lower-abundance ions for subsequent fragmentation and detection using the setting for repeat count = 1, repeat duration = 30 ms, and exclusion list size = 500. Charge state filtering, where ions with singly or unassigned charge states were rejected from fragmentation, was enabled. The minimum MS signal for triggering MS/MS was set to 500, and an activation time of 10 ms was used. All tandem mass spectra were collected using normalized collision energy of 35%, an isolation window of 2 Th. In LTQ Orbitrap, peptides were analyzed using a Top-10 data dependent acquisition method. For protein identification, we used Integrated Proteomics Pipeline (IP2, San Diego, CA) software, a web-based proteomics data analysis platform that supports both cloud and cluster computing, developed by Integrated Proteomics Applications, Inc. (<a href="http://www.integratedproteomics.com/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://www.integratedproteomics.com/</a>). Tandem mass spectra were extracted from the Xcalibur data system format (.raw) into MS2 format using RawXtract1.9.9.2. The MS/MS spectra were searched with the ProLuCID algorithm against the EBI human IPI database (version 3.71, release date March 24, 2010) that was concatenated to a decoy database in which the sequence for each entry in the original database was reversed. The database also had sequence for two proteins, E. coli BirA-R118G appended to it. The search parameters include 50 ppm peptide precursor mass tolerance and 0.6 Da for the fragment mass tolerance acquired in the ion trap.</p>
<p id="Par50">The initial wide precursor mass tolerance in the database search was subjected to post-search filtering and eventually constrained to 20 ppm. Carbamidomethylating on cysteine was defined as fixed modification, and phosphorylation on STY was included as variable modification in the search criteria. The search space also included all fully and semi-tryptic peptide candidates of length of at least six amino acids. Maximum number of internal miscleavages was kept unlimited, thereby allowing all cleavage points for consideration. ProLuCID outputs were assembled and filtered using the DTASelect2.0 program that groups related spectra by protein and removes those that do not pass basic data-quality criteria [S8]. DTASelect2.0 combines XCorr and DCN measurements using a quadratic discriminant function to compute a confidence score to achieve a user specified false discovery rate (less than 1% in this analysis).</p></section><section id="Sec19"><h3 class="pmc_sec_title">Mass spectrometry data analysis for BioID experiments</h3>
<p id="Par51">For identification of high-confidence proximity interaction maps for DYRK3 WT and DYRK3 KD, data from two biological replicates and two technical replicates were used along with control two replicates of FlagBirA* control data. For mass spectrometry analysis, NSAF values were generated for each protein by dividing each peptide spectrum match (PSM) value by the total PSM count in that dataset. The fold change was calculated by dividing the NSAF values of DYRK3 WT and DYRK3 KD interactors by their NSAF values in control datasets. The NSAF values equal to 0 in the control condition were replaced with the smallest NSAF values represented in the dataset. Proteins with log<sub>2</sub>(NSAF) value &gt; 1 were accounted. Furthermore, the remaining proteins were submitted to CRAPome (<a href="https://reprint-apms.org" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://reprint-apms.org</a>), which is a contaminant repository for mass spectrometry data collected from affinity purification experiments and a list with contaminancy percentage (%) was calculated<sup><a href="#CR27" class="usa-link" aria-describedby="CR27">27</a></sup>. Proteins with contaminancy percentage &gt;30% were considered as a contaminant and removed. The interaction maps were drawn with 110 and 126 interactors for DYRK3 WT and DYRK3 KD, respectively. The GO terms for the proximity interactors were determined by using the Database for Annotation, Visualization, and Integrated Discovery (DAVID)<sup><a href="#CR61" class="usa-link" aria-describedby="CR61">61</a></sup>. For Fig. <a href="#Fig1" class="usa-link">1</a>, the proteins that cluster as primary cilium biogenesis were determined using DAVID and published data. For Fig. <a href="#Fig1" class="usa-link">1</a> and Fig. <a href="#MOESM1" class="usa-link">S2</a>, the interaction maps were drawn using STRING database. The functional clusters and GO categories for these clusters are determined with the Clustering with Overlapping Neighborhood Expansion (ClusterONE) plug-in of Cytoscape and BinGO plugins (<em>P</em> &lt; 0.05). GO terms were determined by using DAVID. The network output file was visualized using Cytoscape 3.7.2<sup><a href="#CR62" class="usa-link" aria-describedby="CR62">62</a></sup>.</p></section><section id="Sec20"><h3 class="pmc_sec_title">Statistical analysis</h3>
<p id="Par52">Statistical results, average, and standard deviation values were computed and plotted by using Prism (GraphPad, La Jolla, CA). Student’s t-test and one-way ANOVA test were applied to compare the statistical significance of the measurements. Following key is followed for asterisk placeholders for <em>P</em>-values in the figures: *<em>P</em> &lt; 0.05, **<em>P</em> &lt; 0.01, ***<em>P</em> &lt; 0.001, ****<em>P</em> &lt; 0.0001.</p></section><section id="Sec21"><h3 class="pmc_sec_title">Reporting summary</h3>
<p id="Par53">Further information on research design is available in the <a href="#MOESM11" class="usa-link">Nature Portfolio Reporting Summary</a> linked to this article.</p></section></section><section id="Sec22"><h2 class="pmc_sec_title">Supplementary information</h2>
<section class="sm xbox font-sm" id="MOESM1"><div class="media p"><div class="caption">
<a href="/articles/instance/12371027/bin/42003_2025_8373_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information</a><sup> (66.8MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM2"><div class="media p"><div class="caption">
<a href="/articles/instance/12371027/bin/42003_2025_8373_MOESM2_ESM.avi" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Movie 1</a><sup> (2.1MB, avi) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM3"><div class="media p"><div class="caption">
<a href="/articles/instance/12371027/bin/42003_2025_8373_MOESM3_ESM.avi" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Movie 2</a><sup> (2.9MB, avi) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM4"><div class="media p"><div class="caption">
<a href="/articles/instance/12371027/bin/42003_2025_8373_MOESM4_ESM.avi" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Movie 3</a><sup> (4.4MB, avi) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM5"><div class="media p"><div class="caption">
<a href="/articles/instance/12371027/bin/42003_2025_8373_MOESM5_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 1</a><sup> (360.5KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM6"><div class="media p"><div class="caption">
<a href="/articles/instance/12371027/bin/42003_2025_8373_MOESM6_ESM.xls" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 2</a><sup> (1,016KB, xls) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM7"><div class="media p"><div class="caption">
<a href="/articles/instance/12371027/bin/42003_2025_8373_MOESM7_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 3</a><sup> (23.3KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM8"><div class="media p"><div class="caption">
<a href="/articles/instance/12371027/bin/42003_2025_8373_MOESM8_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 5</a><sup> (119.6KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM9"><div class="media p"><div class="caption">
<a href="/articles/instance/12371027/bin/42003_2025_8373_MOESM9_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 4</a><sup> (221.4KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM10"><div class="media p"><div class="caption">
<a href="/articles/instance/12371027/bin/42003_2025_8373_MOESM10_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">42003_2025_8373_MOESM10_ESM.pdf</a><sup> (30.2KB, pdf) </sup><p>Description of Additional Supplementary Files</p>
</div></div></section><section class="sm xbox font-sm" id="MOESM11"><div class="media p"><div class="caption">
<a href="/articles/instance/12371027/bin/42003_2025_8373_MOESM11_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Reporting Summary</a><sup> (1.6MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM12"><div class="media p"><div class="caption">
<a href="/articles/instance/12371027/bin/42003_2025_8373_MOESM12_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Transparent Peer Review file</a><sup> (607.2KB, pdf) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>We acknowledge Jovana Deretic, Efe Begar, Irem Sultan Dilbaz, Başak Turan and Ezgi Odabaşı for their insightful feedback on this work. We acknowledge Koç University Proteomics Facility, Büşra Akarlar, and Nurhan Özlü for mass spectrometry analysis. This project received funding from The Scientific and Technological Research Council of Turkey (TUBITAK) ARDEB 124Z381 and BIDEB 123C357 to MDA and European Union’s Horizon Europe research and innovation program under the European Research Council Starting grant agreement “ERC, SatelliteHomeostasis, 101078097” to ENF. This work was also supported by EMBO Young Investigator Award and Istanbul Development Agency YEP-57 Grant to ENF.</p></section><section id="notes1"><h2 class="pmc_sec_title">Author contributions</h2>
<p>M.D. Arslanhan conducted all experiments except BioID and analyzed all data. E. Topçu generated the proteomics datasets and M.D. Arslanhan performed their analysis. E.N. Firat-Karalar supervised the project. E.N. Firat-Karalar and M.D. Arslanhan conceptualized the study and wrote the manuscript. E.N. Firat-Karalar and M.D. Arslanhan obtained funding. All authors contributed to manuscript editing and discussion.</p></section><section id="notes2"><h2 class="pmc_sec_title">Peer review</h2>
<section id="FPar1"><h3 class="pmc_sec_title">Peer review information</h3>
<p id="Par54"><em>Communications Biology</em> thanks the anonymous reviewers for their contribution to the peer review of this work. Primary Handling Editors: Tiago Dantas and Manuel Breuer. A peer review file is available.</p></section></section><section id="notes3"><h2 class="pmc_sec_title">Data availability</h2>
<p>All mass spectrometry raw data generated in this study have been deposited to the ProteomeXchange Consortium via the PRIDE partner repository with the dataset identifier PXD064683. The source mass spectrometry data for Fig. <a href="#Fig1" class="usa-link">1</a> and Supplementary Figs. <a href="#MOESM1" class="usa-link">S1</a> and <a href="#MOESM1" class="usa-link">S2</a> are available as Supplementary Data <a href="#MOESM5" class="usa-link">1</a> and <a href="#MOESM6" class="usa-link">2</a>. The numerical source data are available as Supplementary Data <a href="#MOESM9" class="usa-link">4</a>-<a href="#MOESM8" class="usa-link">5</a>.</p></section><section id="FPar2"><h2 class="pmc_sec_title">Competing interests</h2>
<p id="Par55">The authors declare no competing interests.</p></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="fn1"><p><strong>Publisher’s note</strong> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></div></div></section><section id="sec23"><h2 class="pmc_sec_title">Supplementary information</h2>
<p>The online version contains supplementary material available at 10.1038/s42003-025-08373-5.</p></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Aranda, S., Laguna, A. &amp; de la Luna, S. DYRK family of protein kinases: evolutionary relationships, biochemical properties, and functional roles. <em>FASEB J.</em><strong>25</strong>, 449–462 (2011).
</cite> [<a href="https://doi.org/10.1096/fj.10-165837" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21048044/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Aranda,%20S.,%20Laguna,%20A.%20&amp;%20de%20la%20Luna,%20S.%20DYRK%20family%20of%20protein%20kinases:%20evolutionary%20relationships,%20biochemical%20properties,%20and%20functional%20roles.%20FASEB%20J.25,%20449%E2%80%93462%20(2011)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>Boni, J., Rubio-Perez, C., Lopez-Bigas, N., Fillat, C. &amp; de la Luna, S. The DYRK family of kinases in cancer: molecular functions and therapeutic opportunities. <em>Cancers</em><strong>12</strong>10.3390/cancers12082106 (2020).</cite> [<a href="https://doi.org/10.3390/cancers12082106" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7465136/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32751160/" class="usa-link">PubMed</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>Soppa, U. &amp; Becker, W. DYRK protein kinases. <em>Curr. Biol.</em><strong>25</strong>, R488–R489 (2015).
</cite> [<a href="https://doi.org/10.1016/j.cub.2015.02.067" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26079075/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Soppa,%20U.%20&amp;%20Becker,%20W.%20DYRK%20protein%20kinases.%20Curr.%20Biol.25,%20R488%E2%80%93R489%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Yoshida, S. &amp; Yoshida, K. New insights into the roles for DYRK family in mammalian development and congenital diseases. <em>Genes Dis.</em><strong>10</strong>, 758–770 (2023).
</cite> [<a href="https://doi.org/10.1016/j.gendis.2021.12.004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10308075/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37396550/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yoshida,%20S.%20&amp;%20Yoshida,%20K.%20New%20insights%20into%20the%20roles%20for%20DYRK%20family%20in%20mammalian%20development%20and%20congenital%20diseases.%20Genes%20Dis.10,%20758%E2%80%93770%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Becker, W. Emerging role of DYRK family protein kinases as regulators of protein stability in cell cycle control. <em>Cell Cycle</em><strong>11</strong>, 3389–3394 (2012).
</cite> [<a href="https://doi.org/10.4161/cc.21404" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3466549/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22918246/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Becker,%20W.%20Emerging%20role%20of%20DYRK%20family%20protein%20kinases%20as%20regulators%20of%20protein%20stability%20in%20cell%20cycle%20control.%20Cell%20Cycle11,%203389%E2%80%933394%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Correa-Saez, A. et al. Updating dual-specificity tyrosine-phosphorylation-regulated kinase 2 (DYRK2): molecular basis, functions and role in diseases. <em>Cell Mol. Life Sci.</em><strong>77</strong>, 4747–4763 (2020).
</cite> [<a href="https://doi.org/10.1007/s00018-020-03556-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7658070/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32462403/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Correa-Saez,%20A.%20et%20al.%20Updating%20dual-specificity%20tyrosine-phosphorylation-regulated%20kinase%202%20(DYRK2):%20molecular%20basis,%20functions%20and%20role%20in%20diseases.%20Cell%20Mol.%20Life%20Sci.77,%204747%E2%80%934763%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Erickson-Miller, C. L. et al. GSK626616: A DYRK3 inhibitor as a potential new therapy for the treatment of anemia. <em>Blood</em><strong>110</strong>, 156a–157a (2007).</cite> [<a href="https://scholar.google.com/scholar_lookup?Erickson-Miller,%20C.%20L.%20et%20al.%20GSK626616:%20A%20DYRK3%20inhibitor%20as%20a%20potential%20new%20therapy%20for%20the%20treatment%20of%20anemia.%20Blood110,%20156a%E2%80%93157a%20(2007)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>Liu, T. et al. DYRK1A inhibitors for disease therapy: Current status and perspectives. <em>Eur. J. Med. Chem.</em><strong>229</strong>10.1016/j.ejmech.2021.114062 (2022).</cite> [<a href="https://doi.org/10.1016/j.ejmech.2021.114062" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34954592/" class="usa-link">PubMed</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>Ogawa, Y. et al. Development of a novel selective inhibitor of the Down syndrome-related kinase Dyrk1A. <em>Nat. Commun.</em><strong>1</strong>10.1038/ncomms1090 (2010).</cite> [<a href="https://doi.org/10.1038/ncomms1090" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20981014/" class="usa-link">PubMed</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>Becker, W. &amp; Joost, H. G. Structural and functional characteristics of Dyrk, a novel subfamily of protein kinases with dual specificity. <em>Prog. Nucleic Acid Res.</em><strong>62</strong>, 1–17 (1999).</cite> [<a href="https://doi.org/10.1016/s0079-6603(08)60503-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9932450/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Becker,%20W.%20&amp;%20Joost,%20H.%20G.%20Structural%20and%20functional%20characteristics%20of%20Dyrk,%20a%20novel%20subfamily%20of%20protein%20kinases%20with%20dual%20specificity.%20Prog.%20Nucleic%20Acid%20Res.62,%201%E2%80%9317%20(1999)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Laham, A. J., Saber-Ayad, M. &amp; El-Awady, R. DYRK1A: a down syndrome-related dual protein kinase with a versatile role in tumorigenesis. <em>Cell. Mol. Life Sci.</em><strong>78</strong>, 603–619 (2021).
</cite> [<a href="https://doi.org/10.1007/s00018-020-03626-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11071757/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32870330/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Laham,%20A.%20J.,%20Saber-Ayad,%20M.%20&amp;%20El-Awady,%20R.%20DYRK1A:%20a%20down%20syndrome-related%20dual%20protein%20kinase%20with%20a%20versatile%20role%20in%20tumorigenesis.%20Cell.%20Mol.%20Life%20Sci.78,%20603%E2%80%93619%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Berchtold, D., Battich, N. &amp; Pelkmans, L. A systems-level study reveals regulators of membrane-less organelles in human cells. <em>Mol. Cell</em><strong>72</strong>, 1035–1049.e1035 (2018).
</cite> [<a href="https://doi.org/10.1016/j.molcel.2018.10.036" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30503769/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Berchtold,%20D.,%20Battich,%20N.%20&amp;%20Pelkmans,%20L.%20A%20systems-level%20study%20reveals%20regulators%20of%20membrane-less%20organelles%20in%20human%20cells.%20Mol.%20Cell72,%201035%E2%80%931049.e1035%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>Gallo, R., Rai, A. K., McIntyre, A. B. R., Meyer, K. &amp; Pelkmans, L. DYRK3 enables secretory trafficking by maintaining the liquid-like state of ER exit sites. <em>Dev. Cell</em><strong>58</strong>, 1880–1897.e1811 (2023).
</cite> [<a href="https://doi.org/10.1016/j.devcel.2023.08.005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37643612/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gallo,%20R.,%20Rai,%20A.%20K.,%20McIntyre,%20A.%20B.%20R.,%20Meyer,%20K.%20&amp;%20Pelkmans,%20L.%20DYRK3%20enables%20secretory%20trafficking%20by%20maintaining%20the%20liquid-like%20state%20of%20ER%20exit%20sites.%20Dev.%20Cell58,%201880%E2%80%931897.e1811%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>Rai, A. K., Chen, J. X., Selbach, M. &amp; Pelkmans, L. Kinase-controlled phase transition of membraneless organelles in mitosis. <em>Nature</em><strong>559</strong>, 211–216 (2018).
</cite> [<a href="https://doi.org/10.1038/s41586-018-0279-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29973724/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rai,%20A.%20K.,%20Chen,%20J.%20X.,%20Selbach,%20M.%20&amp;%20Pelkmans,%20L.%20Kinase-controlled%20phase%20transition%20of%20membraneless%20organelles%20in%20mitosis.%20Nature559,%20211%E2%80%93216%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>Wippich, F. et al. Dual specificity kinase DYRK3 couples stress granule condensation/dissolution to mTORC1 signaling. <em>Cell</em><strong>152</strong>, 791–805 (2013).
</cite> [<a href="https://doi.org/10.1016/j.cell.2013.01.033" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23415227/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wippich,%20F.%20et%20al.%20Dual%20specificity%20kinase%20DYRK3%20couples%20stress%20granule%20condensation/dissolution%20to%20mTORC1%20signaling.%20Cell152,%20791%E2%80%93805%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Ramella, M. et al. Dual specificity kinase DYRK3 regulates cell migration by influencing the stability of protrusions. <em>iScience</em><strong>27</strong>, 109440 (2024).
</cite> [<a href="https://doi.org/10.1016/j.isci.2024.109440" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10952033/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38510137/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ramella,%20M.%20et%20al.%20Dual%20specificity%20kinase%20DYRK3%20regulates%20cell%20migration%20by%20influencing%20the%20stability%20of%20protrusions.%20iScience27,%20109440%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>Nachury, M. V. &amp; Mick, D. U. Establishing and regulating the composition of cilia for signal transduction. <em>Nat. Rev. Mol. Cell Biol</em>. 10.1038/s41580-019-0116-4 (2019).</cite> [<a href="https://doi.org/10.1038/s41580-019-0116-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6738346/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30948801/" class="usa-link">PubMed</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Breslow, D. K. &amp; Holland, A. J. Mechanism and regulation of centriole and cilium biogenesis. <em>Annu Rev. Biochem.</em><strong>88</strong>, 691–724 (2019).
</cite> [<a href="https://doi.org/10.1146/annurev-biochem-013118-111153" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6588485/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30601682/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Breslow,%20D.%20K.%20&amp;%20Holland,%20A.%20J.%20Mechanism%20and%20regulation%20of%20centriole%20and%20cilium%20biogenesis.%20Annu%20Rev.%20Biochem.88,%20691%E2%80%93724%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Yoshida, S. et al. The novel ciliogenesis regulator DYRK2 governs Hedgehog signaling during mouse embryogenesis. <em>Elife</em><strong>9</strong>. 10.7554/eLife.57381 (2020).</cite> [<a href="https://doi.org/10.7554/eLife.57381" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7410489/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32758357/" class="usa-link">PubMed</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>Yoshida, S. et al. Positive regulation of Hedgehog signaling via phosphorylation of GLI2/GLI3 by DYRK2 kinase. <em>Proc. Natl. Acad. Sci. USA</em><strong>121</strong>, e2320070121 (2024).
</cite> [<a href="https://doi.org/10.1073/pnas.2320070121" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11252808/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38968120/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yoshida,%20S.%20et%20al.%20Positive%20regulation%20of%20Hedgehog%20signaling%20via%20phosphorylation%20of%20GLI2/GLI3%20by%20DYRK2%20kinase.%20Proc.%20Natl.%20Acad.%20Sci.%20USA121,%20e2320070121%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>Yogosawa, S. et al. Mice lacking DYRK2 exhibit congenital malformations with lung hypoplasia and altered Foxf1 expression gradient. <em>Commun. Biol.</em><strong>4</strong>, 1204 (2021).
</cite> [<a href="https://doi.org/10.1038/s42003-021-02734-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8528819/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34671097/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yogosawa,%20S.%20et%20al.%20Mice%20lacking%20DYRK2%20exhibit%20congenital%20malformations%20with%20lung%20hypoplasia%20and%20altered%20Foxf1%20expression%20gradient.%20Commun.%20Biol.4,%201204%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>Lauth, M. et al. DYRK1B-dependent autocrine-to-paracrine shift of Hedgehog signaling by mutant RAS. <em>Nat. Struct. Mol. Biol.</em><strong>17</strong>, 718–725 (2010).
</cite> [<a href="https://doi.org/10.1038/nsmb.1833" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20512148/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lauth,%20M.%20et%20al.%20DYRK1B-dependent%20autocrine-to-paracrine%20shift%20of%20Hedgehog%20signaling%20by%20mutant%20RAS.%20Nat.%20Struct.%20Mol.%20Biol.17,%20718%E2%80%93725%20(2010)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>Mao, J. et al. Regulation of Gli1 transcriptional activity in the nucleus by Dyrk1. <em>J. Biol. Chem.</em><strong>277</strong>, 35156–35161 (2002).
</cite> [<a href="https://doi.org/10.1074/jbc.M206743200" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12138125/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mao,%20J.%20et%20al.%20Regulation%20of%20Gli1%20transcriptional%20activity%20in%20the%20nucleus%20by%20Dyrk1.%20J.%20Biol.%20Chem.277,%2035156%E2%80%9335161%20(2002)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>Varjosalo, M. et al. Application of active and kinase-deficient kinome collection for identification of kinases regulating hedgehog signaling. <em>Cell</em><strong>133</strong>, 537–548 (2008).
</cite> [<a href="https://doi.org/10.1016/j.cell.2008.02.047" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18455992/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Varjosalo,%20M.%20et%20al.%20Application%20of%20active%20and%20kinase-deficient%20kinome%20collection%20for%20identification%20of%20kinases%20regulating%20hedgehog%20signaling.%20Cell133,%20537%E2%80%93548%20(2008)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>Arslanhan, M. D., Rauniyar, N., Yates, J. R., 3rd &amp; Firat-Karalar, E. N. Aurora Kinase A proximity map reveals centriolar satellites as regulators of its ciliary function. <em>EMBO Rep</em>. e51902. 10.15252/embr.202051902 (2021).</cite> [<a href="https://doi.org/10.15252/embr.202051902" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8339716/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34169630/" class="usa-link">PubMed</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>Firat-Karalar, E. N., Rauniyar, N., Yates, J. R. 3rd &amp; Stearns, T. Proximity interactions among centrosome components identify regulators of centriole duplication. <em>Curr. Biol.</em><strong>24</strong>, 664–670 (2014).
</cite> [<a href="https://doi.org/10.1016/j.cub.2014.01.067" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4004176/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24613305/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Firat-Karalar,%20E.%20N.,%20Rauniyar,%20N.,%20Yates,%20J.%20R.%203rd%20&amp;%20Stearns,%20T.%20Proximity%20interactions%20among%20centrosome%20components%20identify%20regulators%20of%20centriole%20duplication.%20Curr.%20Biol.24,%20664%E2%80%93670%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Mellacheruvu, D. et al. The CRAPome: a contaminant repository for affinity purification-mass spectrometry data. <em>Nat. Methods</em><strong>10</strong>, 730–736 (2013).
</cite> [<a href="https://doi.org/10.1038/nmeth.2557" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3773500/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23921808/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mellacheruvu,%20D.%20et%20al.%20The%20CRAPome:%20a%20contaminant%20repository%20for%20affinity%20purification-mass%20spectrometry%20data.%20Nat.%20Methods10,%20730%E2%80%93736%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Branon, T. C. et al. Efficient proximity labeling in living cells and organisms with TurboID. <em>Nat. Biotechnol.</em><strong>36</strong>, 880–887 (2018).
</cite> [<a href="https://doi.org/10.1038/nbt.4201" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6126969/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30125270/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Branon,%20T.%20C.%20et%20al.%20Efficient%20proximity%20labeling%20in%20living%20cells%20and%20organisms%20with%20TurboID.%20Nat.%20Biotechnol.36,%20880%E2%80%93887%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>May, D. G., Scott, K. L., Campos, A. R. &amp; Roux, K. J. Comparative application of BioID and TurboID for protein-proximity biotinylation. <em>Cells</em><strong>9</strong>10.3390/cells9051070 (2020).</cite> [<a href="https://doi.org/10.3390/cells9051070" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7290721/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32344865/" class="usa-link">PubMed</a>]</li>
<li id="CR30">
<span class="label">30.</span><cite>Mehnert, M. et al. Multi-layered proteomic analyses decode compositional and functional effects of cancer mutations on kinase complexes. <em>Nat. Commun.</em><strong>11</strong>, 3563 (2020).
</cite> [<a href="https://doi.org/10.1038/s41467-020-17387-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7366679/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32678104/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mehnert,%20M.%20et%20al.%20Multi-layered%20proteomic%20analyses%20decode%20compositional%20and%20functional%20effects%20of%20cancer%20mutations%20on%20kinase%20complexes.%20Nat.%20Commun.11,%203563%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>Gupta, G. D. et al. A dynamic protein interaction landscape of the human centrosome-cilium interface. <em>Cell</em><strong>163</strong>, 1484–1499 (2015).
</cite> [<a href="https://doi.org/10.1016/j.cell.2015.10.065" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5089374/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26638075/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gupta,%20G.%20D.%20et%20al.%20A%20dynamic%20protein%20interaction%20landscape%20of%20the%20human%20centrosome-cilium%20interface.%20Cell163,%201484%E2%80%931499%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>Hall, E. A. et al. Acute versus chronic loss of mammalian Azi1/Cep131 results in distinct ciliary phenotypes. <em>PLoS Genet.</em><strong>9</strong>, e1003928 (2013).
</cite> [<a href="https://doi.org/10.1371/journal.pgen.1003928" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3887133/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24415959/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hall,%20E.%20A.%20et%20al.%20Acute%20versus%20chronic%20loss%20of%20mammalian%20Azi1/Cep131%20results%20in%20distinct%20ciliary%20phenotypes.%20PLoS%20Genet.9,%20e1003928%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Odabasi, E., Gul, S., Kavakli, I. H. &amp; Firat-Karalar, E. N. Centriolar satellites are required for efficient ciliogenesis and ciliary content regulation. <em>EMBO Rep.</em>10.15252/embr.201947723 (2019).</cite> [<a href="https://doi.org/10.15252/embr.201947723" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6549029/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31023719/" class="usa-link">PubMed</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>Ma, F. et al. Dual-specificity tyrosine phosphorylation-regulated kinase 3 loss activates purine metabolism and promotes hepatocellular carcinoma progression. <em>Hepatology</em><strong>70</strong>, 1785–1803 (2019).
</cite> [<a href="https://doi.org/10.1002/hep.30703" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31066068/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ma,%20F.%20et%20al.%20Dual-specificity%20tyrosine%20phosphorylation-regulated%20kinase%203%20loss%20activates%20purine%20metabolism%20and%20promotes%20hepatocellular%20carcinoma%20progression.%20Hepatology70,%201785%E2%80%931803%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>Kanamaru, T., Neuner, A., Kurtulmus, B. &amp; Pereira, G. Balancing the length of the distal tip by septins is key for stability and signalling function of primary cilia. <em>EMBO J.</em><strong>41</strong>, e108843 (2022).
</cite> [<a href="https://doi.org/10.15252/embj.2021108843" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8724769/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34981518/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kanamaru,%20T.,%20Neuner,%20A.,%20Kurtulmus,%20B.%20&amp;%20Pereira,%20G.%20Balancing%20the%20length%20of%20the%20distal%20tip%20by%20septins%20is%20key%20for%20stability%20and%20signalling%20function%20of%20primary%20cilia.%20EMBO%20J.41,%20e108843%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR36">
<span class="label">36.</span><cite>Sharma, A., Olieric, N. &amp; Steinmetz, M. O. Centriole length control. <em>Curr. Opin. Struct. Biol.</em><strong>66</strong>, 89–95 (2021).
</cite> [<a href="https://doi.org/10.1016/j.sbi.2020.10.011" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33220554/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sharma,%20A.,%20Olieric,%20N.%20&amp;%20Steinmetz,%20M.%20O.%20Centriole%20length%20control.%20Curr.%20Opin.%20Struct.%20Biol.66,%2089%E2%80%9395%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR37">
<span class="label">37.</span><cite>Reilly, M. L. &amp; Benmerah, A. Ciliary kinesins beyond IFT: Cilium length, disassembly, cargo transport and signalling. <em>Biol. Cell</em><strong>111</strong>, 79–94 (2019).
</cite> [<a href="https://doi.org/10.1111/boc.201800074" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30720881/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Reilly,%20M.%20L.%20&amp;%20Benmerah,%20A.%20Ciliary%20kinesins%20beyond%20IFT:%20Cilium%20length,%20disassembly,%20cargo%20transport%20and%20signalling.%20Biol.%20Cell111,%2079%E2%80%9394%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR38">
<span class="label">38.</span><cite>Keeling, J., Tsiokas, L. &amp; Maskey, D. Cellular Mechanisms of Ciliary Length Control. <em>Cells</em><strong>5</strong>10.3390/cells5010006 (2016).</cite> [<a href="https://doi.org/10.3390/cells5010006" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4810091/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26840332/" class="usa-link">PubMed</a>]</li>
<li id="CR39">
<span class="label">39.</span><cite>Bhogaraju, S. et al. Molecular basis of tubulin transport within the cilium by IFT74 and IFT81. <em>Science</em><strong>341</strong>, 1009–1012 (2013).
</cite> [<a href="https://doi.org/10.1126/science.1240985" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4359902/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23990561/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bhogaraju,%20S.%20et%20al.%20Molecular%20basis%20of%20tubulin%20transport%20within%20the%20cilium%20by%20IFT74%20and%20IFT81.%20Science341,%201009%E2%80%931012%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR40">
<span class="label">40.</span><cite>Broekhuis, J. R., Leong, W. Y. &amp; Jansen, G. Regulation of cilium length and intraflagellar transport. <em>Int Rev. Cell Mol. Biol.</em><strong>303</strong>, 101–138 (2013).
</cite> [<a href="https://doi.org/10.1016/B978-0-12-407697-6.00003-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23445809/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Broekhuis,%20J.%20R.,%20Leong,%20W.%20Y.%20&amp;%20Jansen,%20G.%20Regulation%20of%20cilium%20length%20and%20intraflagellar%20transport.%20Int%20Rev.%20Cell%20Mol.%20Biol.303,%20101%E2%80%93138%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR41">
<span class="label">41.</span><cite>He, M. et al. The kinesin-4 protein Kif7 regulates mammalian Hedgehog signalling by organizing the cilium tip compartment. <em>Nat. Cell Biol.</em><strong>16</strong>, 663–672 (2014).
</cite> [<a href="https://doi.org/10.1038/ncb2988" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4085576/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24952464/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?He,%20M.%20et%20al.%20The%20kinesin-4%20protein%20Kif7%20regulates%20mammalian%20Hedgehog%20signalling%20by%20organizing%20the%20cilium%20tip%20compartment.%20Nat.%20Cell%20Biol.16,%20663%E2%80%93672%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR42">
<span class="label">42.</span><cite>Jiang, S. et al. Interplay between the Kinesin and Tubulin Mechanochemical Cycles Underlies Microtubule Tip Tracking by the Non-motile Ciliary Kinesin Kif7. <em>Dev. Cell</em><strong>49</strong>, 711–730.e718 (2019).
</cite> [<a href="https://doi.org/10.1016/j.devcel.2019.04.001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6550315/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31031197/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jiang,%20S.%20et%20al.%20Interplay%20between%20the%20Kinesin%20and%20Tubulin%20Mechanochemical%20Cycles%20Underlies%20Microtubule%20Tip%20Tracking%20by%20the%20Non-motile%20Ciliary%20Kinesin%20Kif7.%20Dev.%20Cell49,%20711%E2%80%93730.e718%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR43">
<span class="label">43.</span><cite>Yue, Y., Engelke, M. F., Blasius, T. L. &amp; Verhey, K. J. Hedgehog-induced ciliary trafficking of kinesin-4 motor KIF7 requires intraflagellar transport but not KIF7’s microtubule binding. <em>Mol. Biol. Cell</em><strong>33</strong>, br1 (2022).
</cite> [<a href="https://doi.org/10.1091/mbc.E21-04-0215" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8886809/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34705483/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yue,%20Y.,%20Engelke,%20M.%20F.,%20Blasius,%20T.%20L.%20&amp;%20Verhey,%20K.%20J.%20Hedgehog-induced%20ciliary%20trafficking%20of%20kinesin-4%20motor%20KIF7%20requires%20intraflagellar%20transport%20but%20not%20KIF7%E2%80%99s%20microtubule%20binding.%20Mol.%20Biol.%20Cell33,%20br1%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR44">
<span class="label">44.</span><cite>Kim, J. et al. Functional genomic screen for modulators of ciliogenesis and cilium length. <em>Nature</em><strong>464</strong>, 1048–1051 (2010).
</cite> [<a href="https://doi.org/10.1038/nature08895" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2929961/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20393563/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kim,%20J.%20et%20al.%20Functional%20genomic%20screen%20for%20modulators%20of%20ciliogenesis%20and%20cilium%20length.%20Nature464,%201048%E2%80%931051%20(2010)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR45">
<span class="label">45.</span><cite>Sharma, N., Kosan, Z. A., Stallworth, J. E., Berbari, N. F. &amp; Yoder, B. K. Soluble levels of cytosolic tubulin regulate ciliary length control. <em>Mol. Biol. Cell</em><strong>22</strong>, 806–816 (2011).
</cite> [<a href="https://doi.org/10.1091/mbc.E10-03-0269" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3057705/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21270438/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sharma,%20N.,%20Kosan,%20Z.%20A.,%20Stallworth,%20J.%20E.,%20Berbari,%20N.%20F.%20&amp;%20Yoder,%20B.%20K.%20Soluble%20levels%20of%20cytosolic%20tubulin%20regulate%20ciliary%20length%20control.%20Mol.%20Biol.%20Cell22,%20806%E2%80%93816%20(2011)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR46">
<span class="label">46.</span><cite>Tischer, J., Carden, S. &amp; Gergely, F. Accessorizing the centrosome: new insights into centriolar appendages and satellites. <em>Curr. Opin. Struct. Biol.</em><strong>66</strong>, 148–155 (2021).
</cite> [<a href="https://doi.org/10.1016/j.sbi.2020.10.021" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33279729/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tischer,%20J.,%20Carden,%20S.%20&amp;%20Gergely,%20F.%20Accessorizing%20the%20centrosome:%20new%20insights%20into%20centriolar%20appendages%20and%20satellites.%20Curr.%20Opin.%20Struct.%20Biol.66,%20148%E2%80%93155%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR47">
<span class="label">47.</span><cite>Aydin, O. Z., Taflan, S. O., Gurkaslar, C. &amp; Firat-Karalar, E. N. Acute inhibition of centriolar satellite function and positioning reveals their functions at the primary cilium. <em>PLoS Biol.</em><strong>18</strong>, e3000679 (2020).
</cite> [<a href="https://doi.org/10.1371/journal.pbio.3000679" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7326281/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32555591/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Aydin,%20O.%20Z.,%20Taflan,%20S.%20O.,%20Gurkaslar,%20C.%20&amp;%20Firat-Karalar,%20E.%20N.%20Acute%20inhibition%20of%20centriolar%20satellite%20function%20and%20positioning%20reveals%20their%20functions%20at%20the%20primary%20cilium.%20PLoS%20Biol.18,%20e3000679%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR48">
<span class="label">48.</span><cite>Odabasi, E., Batman, U. &amp; Firat-Karalar, E. N. Unraveling the mysteries of centriolar satellites: time to rewrite the textbooks about the centrosome/cilium complex. <em>Mol. Biol. Cell</em><strong>31</strong>, 866–872 (2020).
</cite> [<a href="https://doi.org/10.1091/mbc.E19-07-0402" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7185976/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32286929/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Odabasi,%20E.,%20Batman,%20U.%20&amp;%20Firat-Karalar,%20E.%20N.%20Unraveling%20the%20mysteries%20of%20centriolar%20satellites:%20time%20to%20rewrite%20the%20textbooks%20about%20the%20centrosome/cilium%20complex.%20Mol.%20Biol.%20Cell31,%20866%E2%80%93872%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR49">
<span class="label">49.</span><cite>Liem, K. F. Jr., He, M., Ocbina, P. J. &amp; Anderson, K. V. Mouse Kif7/Costal2 is a cilia-associated protein that regulates Sonic hedgehog signaling. <em>Proc. Natl. Acad. Sci. USA</em><strong>106</strong>, 13377–13382 (2009).
</cite> [<a href="https://doi.org/10.1073/pnas.0906944106" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2726420/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19666503/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Liem,%20K.%20F.%20Jr.,%20He,%20M.,%20Ocbina,%20P.%20J.%20&amp;%20Anderson,%20K.%20V.%20Mouse%20Kif7/Costal2%20is%20a%20cilia-associated%20protein%20that%20regulates%20Sonic%20hedgehog%20signaling.%20Proc.%20Natl.%20Acad.%20Sci.%20USA106,%2013377%E2%80%9313382%20(2009)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR50">
<span class="label">50.</span><cite>Pedersen, L. B. &amp; Akhmanova, A. Kif7 keeps cilia tips in shape. <em>Nat. Cell Biol.</em><strong>16</strong>, 623–625 (2014).
</cite> [<a href="https://doi.org/10.1038/ncb2997" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24981634/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Pedersen,%20L.%20B.%20&amp;%20Akhmanova,%20A.%20Kif7%20keeps%20cilia%20tips%20in%20shape.%20Nat.%20Cell%20Biol.16,%20623%E2%80%93625%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR51">
<span class="label">51.</span><cite>Begar, E., Seyrek, E. &amp; Firat-Karalar, E. N. Navigating centriolar satellites: the role of PCM1 in cellular and organismal processes. <em>FEBS J</em>. 10.1111/febs.17194 (2024).</cite> [<a href="https://doi.org/10.1111/febs.17194" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11839937/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38825736/" class="usa-link">PubMed</a>]</li>
<li id="CR52">
<span class="label">52.</span><cite>Prosser, S. L. &amp; Pelletier, L. Centriolar satellite biogenesis and function in vertebrate cells. <em>J. Cell Sci.</em><strong>133</strong>10.1242/jcs.239566 (2020).</cite> [<a href="https://doi.org/10.1242/jcs.239566" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31896603/" class="usa-link">PubMed</a>]</li>
<li id="CR53">
<span class="label">53.</span><cite>Guen, V. J. et al. STAR syndrome-associated CDK10/Cyclin M regulates actin network architecture and ciliogenesis. <em>Cell Cycle</em><strong>15</strong>, 678–688 (2016).
</cite> [<a href="https://doi.org/10.1080/15384101.2016.1147632" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4846108/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27104747/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Guen,%20V.%20J.%20et%20al.%20STAR%20syndrome-associated%20CDK10/Cyclin%20M%20regulates%20actin%20network%20architecture%20and%20ciliogenesis.%20Cell%20Cycle15,%20678%E2%80%93688%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR54">
<span class="label">54.</span><cite>Jacoby, M. et al. INPP5E mutations cause primary cilium signaling defects, ciliary instability and ciliopathies in human and mouse. <em>Nat. Genet</em><strong>41</strong>, 1027–1031 (2009).
</cite> [<a href="https://doi.org/10.1038/ng.427" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19668215/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jacoby,%20M.%20et%20al.%20INPP5E%20mutations%20cause%20primary%20cilium%20signaling%20defects,%20ciliary%20instability%20and%20ciliopathies%20in%20human%20and%20mouse.%20Nat.%20Genet41,%201027%E2%80%931031%20(2009)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR55">
<span class="label">55.</span><cite>May-Simera, H. L. et al. Primary cilium-mediated retinal pigment epithelium maturation is disrupted in ciliopathy patient cells. <em>Cell Rep.</em><strong>22</strong>, 189–205 (2018).
</cite> [<a href="https://doi.org/10.1016/j.celrep.2017.12.038" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6166245/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29298421/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?May-Simera,%20H.%20L.%20et%20al.%20Primary%20cilium-mediated%20retinal%20pigment%20epithelium%20maturation%20is%20disrupted%20in%20ciliopathy%20patient%20cells.%20Cell%20Rep.22,%20189%E2%80%93205%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR56">
<span class="label">56.</span><cite>Ramsbottom, S. A. et al. Targeted exon skipping of a CEP290 mutation rescues Joubert syndrome phenotypes in vitro and in a murine model. <em>Proc. Natl. Acad. Sci. USA</em><strong>115</strong>, 12489–12494 (2018).
</cite> [<a href="https://doi.org/10.1073/pnas.1809432115" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6298104/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30446612/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ramsbottom,%20S.%20A.%20et%20al.%20Targeted%20exon%20skipping%20of%20a%20CEP290%20mutation%20rescues%20Joubert%20syndrome%20phenotypes%20in%20vitro%20and%20in%20a%20murine%20model.%20Proc.%20Natl.%20Acad.%20Sci.%20USA115,%2012489%E2%80%9312494%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR57">
<span class="label">57.</span><cite>Reiter, J. F. &amp; Leroux, M. R. Genes and molecular pathways underpinning ciliopathies. <em>Nat. Rev. Mol. Cell Biol.</em><strong>18</strong>, 533–547 (2017).
</cite> [<a href="https://doi.org/10.1038/nrm.2017.60" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5851292/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28698599/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Reiter,%20J.%20F.%20&amp;%20Leroux,%20M.%20R.%20Genes%20and%20molecular%20pathways%20underpinning%20ciliopathies.%20Nat.%20Rev.%20Mol.%20Cell%20Biol.18,%20533%E2%80%93547%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR58">
<span class="label">58.</span><cite>Frikstad, K. M. et al. A CEP104-CSPP1 Complex Is Required for Formation of Primary Cilia Competent in Hedgehog Signaling. <em>Cell Rep.</em><strong>28</strong>, 1907–1922.e1906 (2019).
</cite> [<a href="https://doi.org/10.1016/j.celrep.2019.07.025" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6702141/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31412255/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Frikstad,%20K.%20M.%20et%20al.%20A%20CEP104-CSPP1%20Complex%20Is%20Required%20for%20Formation%20of%20Primary%20Cilia%20Competent%20in%20Hedgehog%20Signaling.%20Cell%20Rep.28,%201907%E2%80%931922.e1906%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR59">
<span class="label">59.</span><cite>Yan, H. et al. Low Expression of DYRK2 (Dual Specificity Tyrosine Phosphorylation Regulated Kinase 2) Correlates with Poor Prognosis in Colorectal Cancer. <em>PLoS ONE</em><strong>11</strong>, e0159954 (2016).
</cite> [<a href="https://doi.org/10.1371/journal.pone.0159954" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4988784/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27532268/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yan,%20H.%20et%20al.%20Low%20Expression%20of%20DYRK2%20(Dual%20Specificity%20Tyrosine%20Phosphorylation%20Regulated%20Kinase%202)%20Correlates%20with%20Poor%20Prognosis%20in%20Colorectal%20Cancer.%20PLoS%20ONE11,%20e0159954%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR60">
<span class="label">60.</span><cite>Arslanhan, M. D. <em>et al</em>. CCDC15 localizes to the centriole inner scaffold and controls centriole length and integrity. <em>J Cell Biol</em><strong>222</strong> (2023). 10.1083/jcb.202305009</cite> [<a href="https://doi.org/10.1083/jcb.202305009" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10630097/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37934472/" class="usa-link">PubMed</a>]</li>
<li id="CR61">
<span class="label">61.</span><cite>Sherman, B. T. et al. DAVID: a web server for functional enrichment analysis and functional annotation of gene lists (2021 update). <em>Nucleic Acids Res.</em><strong>50</strong>, W216–W221 (2022).
</cite> [<a href="https://doi.org/10.1093/nar/gkac194" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9252805/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35325185/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sherman,%20B.%20T.%20et%20al.%20DAVID:%20a%20web%20server%20for%20functional%20enrichment%20analysis%20and%20functional%20annotation%20of%20gene%20lists%20(2021%20update).%20Nucleic%20Acids%20Res.50,%20W216%E2%80%93W221%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR62">
<span class="label">62.</span><cite>Nepusz, T., Yu, H. &amp; Paccanaro, A. Detecting overlapping protein complexes in protein-protein interaction networks. <em>Nat. Methods</em><strong>9</strong>, 471–472 (2012).
</cite> [<a href="https://doi.org/10.1038/nmeth.1938" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3543700/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22426491/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Nepusz,%20T.,%20Yu,%20H.%20&amp;%20Paccanaro,%20A.%20Detecting%20overlapping%20protein%20complexes%20in%20protein-protein%20interaction%20networks.%20Nat.%20Methods9,%20471%E2%80%93472%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12371027/bin/42003_2025_8373_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information</a><sup> (66.8MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12371027/bin/42003_2025_8373_MOESM2_ESM.avi" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Movie 1</a><sup> (2.1MB, avi) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12371027/bin/42003_2025_8373_MOESM3_ESM.avi" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Movie 2</a><sup> (2.9MB, avi) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material4_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12371027/bin/42003_2025_8373_MOESM4_ESM.avi" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Movie 3</a><sup> (4.4MB, avi) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material5_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12371027/bin/42003_2025_8373_MOESM5_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 1</a><sup> (360.5KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material6_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12371027/bin/42003_2025_8373_MOESM6_ESM.xls" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 2</a><sup> (1,016KB, xls) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material7_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12371027/bin/42003_2025_8373_MOESM7_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 3</a><sup> (23.3KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material8_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12371027/bin/42003_2025_8373_MOESM8_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 5</a><sup> (119.6KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material9_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12371027/bin/42003_2025_8373_MOESM9_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 4</a><sup> (221.4KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material10_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12371027/bin/42003_2025_8373_MOESM10_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">42003_2025_8373_MOESM10_ESM.pdf</a><sup> (30.2KB, pdf) </sup><p>Description of Additional Supplementary Files</p>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material11_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12371027/bin/42003_2025_8373_MOESM11_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Reporting Summary</a><sup> (1.6MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material12_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12371027/bin/42003_2025_8373_MOESM12_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Transparent Peer Review file</a><sup> (607.2KB, pdf) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>All mass spectrometry raw data generated in this study have been deposited to the ProteomeXchange Consortium via the PRIDE partner repository with the dataset identifier PXD064683. The source mass spectrometry data for Fig. <a href="#Fig1" class="usa-link">1</a> and Supplementary Figs. <a href="#MOESM1" class="usa-link">S1</a> and <a href="#MOESM1" class="usa-link">S2</a> are available as Supplementary Data <a href="#MOESM5" class="usa-link">1</a> and <a href="#MOESM6" class="usa-link">2</a>. The numerical source data are available as Supplementary Data <a href="#MOESM9" class="usa-link">4</a>-<a href="#MOESM8" class="usa-link">5</a>.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Communications Biology are provided here courtesy of <strong>Nature Publishing Group</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1038/s42003-025-08373-5"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/42003_2025_Article_8373.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (14.8 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12371027/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12371027/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12371027%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371027/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12371027/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12371027/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40841441/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12371027/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40841441/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12371027/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12371027/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="kjFMjojJQNTyPXy7UZjjmhIbkyuQDEXIKaedH19mmZWpl51gZ7Mp1lh9dYjFakoe">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
