
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Characterizing the role of early life factors in machine learning-based multimorbidity risk prediction - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="152552628AF438330552620022111D39.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="pdig">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12360575/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="PLOS Digital Health">
<meta name="citation_title" content="Characterizing the role of early life factors in machine learning-based multimorbidity risk prediction">
<meta name="citation_author" content="Vien Ngoc Dang">
<meta name="citation_author_institution" content="Departament de Matemàtiques i Informàtica, Facultat de Matemàtiques i Informàtica, Universitat de Barcelona, Barcelona, Spain">
<meta name="citation_author" content="Charlotte Cecil">
<meta name="citation_author_institution" content="Department of Child and Adolescent Psychiatry/Psychology, Erasmus MC, University Medical Center Rotterdam, Rotterdam, The Netherlands">
<meta name="citation_author_institution" content="Department of Epidemiology, Erasmus MC, Rotterdam, University Medical Center Rotterdam, Rotterdam, The Netherlands">
<meta name="citation_author_institution" content="Molecular Epidemiology, Department of Biomedical Data Sciences, Leiden University Medical Center, Leiden, The Netherlands">
<meta name="citation_author" content="Carmine M Pariante">
<meta name="citation_author_institution" content="Institute of Psychiatry, Psychology and Neuroscience, Department of Psychological Medicine, Maurice Wohl Clinical Neuroscience Institute, King’s College London, London, United Kingdom">
<meta name="citation_author_institution" content="National Institute for Health Research (NIHR) Mental Health Biomedical Research Centre at South London and Maudsley NHS Foundation Trust, London, United Kingdom">
<meta name="citation_author" content="Jerónimo Hernández-González">
<meta name="citation_author_institution" content="Departament d’Informàtica, Matemàtica Aplicada i Estadística, Universitat de Girona, Girona, Spain">
<meta name="citation_author" content="Karim Lekadir">
<meta name="citation_author_institution" content="Departament de Matemàtiques i Informàtica, Facultat de Matemàtiques i Informàtica, Universitat de Barcelona, Barcelona, Spain">
<meta name="citation_author_institution" content="Institució Catalana de Recerca i Estudis Avançats (ICREA), Barcelona, Spain">
<meta name="citation_publication_date" content="2025 Aug 18">
<meta name="citation_volume" content="4">
<meta name="citation_issue" content="8">
<meta name="citation_firstpage" content="e0000982">
<meta name="citation_doi" content="10.1371/journal.pdig.0000982">
<meta name="citation_pmid" content="40825013">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12360575/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12360575/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12360575/pdf/pdig.0000982.pdf">
<meta name="description" content="Recent evidence suggests that psycho-cardio-metabolic (PCM) multimorbidity finds its origins in exposure to early-life factors (ELFs), making the exploration of this association crucial for understanding and effective management of these complex ...">
<meta name="og:title" content="Characterizing the role of early life factors in machine learning-based multimorbidity risk prediction">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Recent evidence suggests that psycho-cardio-metabolic (PCM) multimorbidity finds its origins in exposure to early-life factors (ELFs), making the exploration of this association crucial for understanding and effective management of these complex ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12360575/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12360575">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1371/journal.pdig.0000982"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/pdig.0000982.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12360575%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12360575/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12360575/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12360575/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-pdig.png" alt="PLOS Digital Health logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to PLOS Digital Health" title="Link to PLOS Digital Health" shape="default" href="https://doi.org/10.1371/journal.pdig.0000982" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">PLOS Digit Health</button></div>. 2025 Aug 18;4(8):e0000982. doi: <a href="https://doi.org/10.1371/journal.pdig.0000982" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1371/journal.pdig.0000982</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22PLOS%20Digit%20Health%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22PLOS%20Digit%20Health%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22PLOS%20Digit%20Health%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22PLOS%20Digit%20Health%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Characterizing the role of early life factors in machine learning-based multimorbidity risk prediction</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Dang%20VN%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Vien Ngoc Dang</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Vien Ngoc Dang</span></h3>
<div class="p">
<sup>1</sup>Departament de Matemàtiques i Informàtica, Facultat de Matemàtiques i Informàtica, Universitat de Barcelona, Barcelona, Spain</div>
<div>Conceptualization, Data curation, Formal analysis, Investigation, Methodology, Validation, Visualization, Writing – original draft, Writing – review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Dang%20VN%22%5BAuthor%5D" class="usa-link"><span class="name western">Vien Ngoc Dang</span></a>
</div>
</div>
<sup>1,</sup><sup>*</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Cecil%20C%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Charlotte Cecil</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Charlotte Cecil</span></h3>
<div class="p">
<sup>2</sup>Department of Child and Adolescent Psychiatry/Psychology, Erasmus MC, University Medical Center Rotterdam, Rotterdam, The Netherlands</div>
<div class="p">
<sup>3</sup>Department of Epidemiology, Erasmus MC, Rotterdam, University Medical Center Rotterdam, Rotterdam, The Netherlands</div>
<div class="p">
<sup>4</sup>Molecular Epidemiology, Department of Biomedical Data Sciences, Leiden University Medical Center, Leiden, The Netherlands</div>
<div>Writing – review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Cecil%20C%22%5BAuthor%5D" class="usa-link"><span class="name western">Charlotte Cecil</span></a>
</div>
</div>
<sup>2,</sup><sup>3,</sup><sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Pariante%20CM%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Carmine M Pariante</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Carmine M Pariante</span></h3>
<div class="p">
<sup>5</sup>Institute of Psychiatry, Psychology and Neuroscience, Department of Psychological Medicine, Maurice Wohl Clinical Neuroscience Institute, King’s College London, London, United Kingdom</div>
<div class="p">
<sup>6</sup>National Institute for Health Research (NIHR) Mental Health Biomedical Research Centre at South London and Maudsley NHS Foundation Trust, London, United Kingdom</div>
<div>Writing – review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Pariante%20CM%22%5BAuthor%5D" class="usa-link"><span class="name western">Carmine M Pariante</span></a>
</div>
</div>
<sup>5,</sup><sup>6</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hern%C3%A1ndez-Gonz%C3%A1lez%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Jerónimo Hernández-González</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Jerónimo Hernández-González</span></h3>
<div class="p">
<sup>7</sup>Departament d’Informàtica, Matemàtica Aplicada i Estadística, Universitat de Girona, Girona, Spain</div>
<div>Conceptualization, Methodology, Supervision, Writing – original draft, Writing – review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hern%C3%A1ndez-Gonz%C3%A1lez%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Jerónimo Hernández-González</span></a>
</div>
</div>
<sup>7</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lekadir%20K%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Karim Lekadir</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Karim Lekadir</span></h3>
<div class="p">
<sup>1</sup>Departament de Matemàtiques i Informàtica, Facultat de Matemàtiques i Informàtica, Universitat de Barcelona, Barcelona, Spain</div>
<div class="p">
<sup>8</sup>Institució Catalana de Recerca i Estudis Avançats (ICREA), Barcelona, Spain</div>
<div>Funding acquisition, Resources, Supervision</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lekadir%20K%22%5BAuthor%5D" class="usa-link"><span class="name western">Karim Lekadir</span></a>
</div>
</div>
<sup>1,</sup><sup>8</sup>
</div>
<div class="cg p">Editor: <span class="name western">Gloria Hyunjung Kwak</span><sup>9</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="aff001">
<sup>1</sup>Departament de Matemàtiques i Informàtica, Facultat de Matemàtiques i Informàtica, Universitat de Barcelona, Barcelona, Spain</div>
<div id="aff002">
<sup>2</sup>Department of Child and Adolescent Psychiatry/Psychology, Erasmus MC, University Medical Center Rotterdam, Rotterdam, The Netherlands</div>
<div id="aff003">
<sup>3</sup>Department of Epidemiology, Erasmus MC, Rotterdam, University Medical Center Rotterdam, Rotterdam, The Netherlands</div>
<div id="aff004">
<sup>4</sup>Molecular Epidemiology, Department of Biomedical Data Sciences, Leiden University Medical Center, Leiden, The Netherlands</div>
<div id="aff005">
<sup>5</sup>Institute of Psychiatry, Psychology and Neuroscience, Department of Psychological Medicine, Maurice Wohl Clinical Neuroscience Institute, King’s College London, London, United Kingdom</div>
<div id="aff006">
<sup>6</sup>National Institute for Health Research (NIHR) Mental Health Biomedical Research Centre at South London and Maudsley NHS Foundation Trust, London, United Kingdom</div>
<div id="aff007">
<sup>7</sup>Departament d’Informàtica, Matemàtica Aplicada i Estadística, Universitat de Girona, Girona, Spain</div>
<div id="aff008">
<sup>8</sup>Institució Catalana de Recerca i Estudis Avançats (ICREA), Barcelona, Spain</div>
<div id="edit1">
<sup>9</sup>Emory University, UNITED STATES OF AMERICA</div>
<div class="author-notes p">
<div class="fn" id="cor001">
<sup>✉</sup><p class="display-inline">* E-mail: <span>dangn@ub.edu</span></p>
</div>
<div class="fn" id="coi001"><p>In the past three years, Professor Carmine Pariante has conducted research on inflammation and mental health in collaboration with Compass Pathways (UK) and the Wellcome Trust strategy award to the Neuroimmunology of Mood Disorders and Alzheimer’s Disease (NIMA) Consortium (2015–2023; 104025/Z/14/Z), which was also funded by Janssen, GlaxoSmithKline, Lundbeck, and Pfizer. He is also funded by the NIHR Biomedical Research Centre at the South London and Maudsley NHS Trust and King’s College London. The rest of the authors declare that no competing interests exist.</p></div>
</div>
<h4 class="font-secondary">Roles</h4>
<div class="p">
<strong class="contrib"><span class="name western">Vien Ngoc Dang</span></strong>: <span class="role">Conceptualization, Data curation, Formal analysis, Investigation, Methodology, Validation, Visualization, Writing – original draft, Writing – review &amp; editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">Charlotte Cecil</span></strong>: <span class="role">Writing – review &amp; editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">Carmine M Pariante</span></strong>: <span class="role">Writing – review &amp; editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">Jerónimo Hernández-González</span></strong>: <span class="role">Conceptualization, Methodology, Supervision, Writing – original draft, Writing – review &amp; editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">Karim Lekadir</span></strong>: <span class="role">Funding acquisition, Resources, Supervision</span>
</div>
<div class="p">
<strong class="contrib"><span class="name western">Gloria Hyunjung Kwak</span></strong>: <span class="role">Editor</span>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 Feb 27; Accepted 2025 Jul 7; Collection date 2025 Aug.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© 2025 Dang et al</div>
<p>This is an open access article distributed under the terms of the <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Creative Commons Attribution License</a>, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12360575  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40825013/" class="usa-link">40825013</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>Abstract</h2>
<p>Recent evidence suggests that psycho-cardio-metabolic (PCM) multimorbidity finds its origins in exposure to early-life factors (ELFs), making the exploration of this association crucial for understanding and effective management of these complex health issues. Moreover, risk prediction models for cardiovascular diseases (CVD) and diabetes, as recommended by current clinical guidelines, typically demonstrate sub-optimal performance in clinically relevant sub-populations where these ELFs may play a substantial role. Our methodological approach investigates the contribution of ELFs to machine-learning-based risk prediction models for comorbid populations, incorporating a wide set of early-life and proximal variables, with a special focus on prenatal and postnatal ELFs. To address the complexity of integrating diverse early-life and proximal factors, we leverage models capable of handling high-dimensional, heterogeneous data sources to enhance prediction accuracy in complex clinical populations. The long-term predictive ability of ELFs, along with their influence on model decisions, is assessed with the learned models, and global and local model-agnostic interpretative techniques allow us to elucidate some interactions leading to multimorbidity. The data for this study is derived from the UK Biobank, showcasing both the strengths and limitations inherent in utilizing a single, large-scale database for such research. Our results show enhanced predictive performance for CVD (AUC-ROC: +7.9%, Acc: +14.7%, Cohen’s d: 1.5) among individuals with concurrent mental health issues (depression or anxiety) and diabetes. Similarly, we demonstrate improved diabetes risk prediction (AUC-ROC: +12.3%, Acc: +13.5%, Cohen’s d: 2.5) in those with concurrent mental health conditions and CVD. The inspection of these models, which integrate a large set of ELFs and other predictors (including the 7-core Framingham and UKDiabetes variables), provides key information that could lead to a more profound understanding of psycho-cardio-metabolic multimorbidity. Our findings highlight the utility of incorporating life-course factors into risk models. Integrating a diverse range of physiological, psychological, and ELFs becomes particularly pertinent in the context of multimorbidity.</p></section><section class="abstract" id="abstract2"><h2>Author summary</h2>
<p>Many people experience multiple chronic diseases at the same time, such as heart disease, diabetes, and mental health conditions, which makes their health harder to predict and manage. This challenge contributes to health disparities, as individuals with multiple conditions often receive less accurate risk assessments and have limited access to tailored care. Research suggests that experiences early in life, from prenatal development to childhood, can shape long-term health, but these factors are rarely included in disease prediction models. In this study, we used machine learning to understand how ELFs contribute to predicting the risk of heart disease and diabetes, especially in people with mental health conditions. Our results show that including ELFs makes risk predictions more accurate, particularly for people affected by multiple diseases. We also visualize the influence of these factors in health risk prediction, helping to better understand the connection between early-life experiences and later disease. This research highlights the importance of considering life-course factors in disease prevention, paving the way for more personalized and fair approaches to healthcare.</p></section><section id="sec001"><h2 class="pmc_sec_title">Introduction</h2>
<p>The World Health Organisation classifies cardiovascular diseases (CVD) and diabetes among the foremost non-communicable diseases worldwide [<a href="#pdig.0000982.ref001" class="usa-link" aria-describedby="pdig.0000982.ref001">1</a>]. While CVD is the leading cause of global mortality [<a href="#pdig.0000982.ref002" class="usa-link" aria-describedby="pdig.0000982.ref002">2</a>], the prevalence of diabetes is rapidly increasing, imposing a significant burden on healthcare systems [<a href="#pdig.0000982.ref003" class="usa-link" aria-describedby="pdig.0000982.ref003">3</a>]. Apart from their distinct complexities, these conditions frequently intersect with mental health disorders, such as depression and anxiety [<a href="#pdig.0000982.ref004" class="usa-link" aria-describedby="pdig.0000982.ref004">4</a>], and also co-occur with one another [<a href="#pdig.0000982.ref005" class="usa-link" aria-describedby="pdig.0000982.ref005">5</a>], suggesting shared environmental and genetic factors that contribute to multimorbidity patterns [<a href="#pdig.0000982.ref006" class="usa-link" aria-describedby="pdig.0000982.ref006">6</a>]. However, the mechanisms leading to psycho-cardio-metabolic (PCM) multimorbidity, characterized by the simultaneous presence of two or more chronic conditions, are not well understood. Epidemiological studies have shown an association between early-life factors (ELFs), spanning prenatal, postnatal, and childhood periods, and the development of PCM conditions [<a href="#pdig.0000982.ref007" class="usa-link" aria-describedby="pdig.0000982.ref007">7</a>]. Prenatal factors, such as maternal smoking or challenges like poor fetal growth, as well as postnatal and childhood experiences like abuse or neglect, substantially influence long-term health outcomes, setting the stage for multiple chronic conditions [<a href="#pdig.0000982.ref008" class="usa-link" aria-describedby="pdig.0000982.ref008">8</a>]. These ELFs have been associated with an elevated risk of developing depression [<a href="#pdig.0000982.ref009" class="usa-link" aria-describedby="pdig.0000982.ref009">9</a>], anxiety [<a href="#pdig.0000982.ref010" class="usa-link" aria-describedby="pdig.0000982.ref010">10</a>], CVD [<a href="#pdig.0000982.ref011" class="usa-link" aria-describedby="pdig.0000982.ref011">11</a>], and diabetes [<a href="#pdig.0000982.ref012" class="usa-link" aria-describedby="pdig.0000982.ref012">12</a>] later in life, contributing individually to the risk profile of these disorders and underscoring the importance of considering these factors in understanding and addressing the patterns of multimorbidity.</p>
<p>The intricate nature of PCM multimorbidity, characterized by complex interrelationships among various health conditions and risk factors, poses substantial challenges in accurate disease prediction, particularly in the context of comorbid conditions. This brings us to another pivotal concern: the efficacy of risk prediction models currently recommended by clinical guidelines, which typically demonstrate sub-optimal performance, especially in clinically relevant sub-populations [<a href="#pdig.0000982.ref013" class="usa-link" aria-describedby="pdig.0000982.ref013">13</a>]. For example, in predicting CVD, the Framingham Risk Score, as recommended by the 2010 American College of Cardiology/American Heart Association (ACC/AHA) guidelines [<a href="#pdig.0000982.ref014" class="usa-link" aria-describedby="pdig.0000982.ref014">14</a>], has been demonstrated to be less effective in individuals with diabetes [<a href="#pdig.0000982.ref013" class="usa-link" aria-describedby="pdig.0000982.ref013">13</a>]. This limitation is largely attributable to the restrictive assumptions of the model and limited predictors, which frequently presuppose a linear relationship between risk factors and disease outcomes [<a href="#pdig.0000982.ref015" class="usa-link" aria-describedby="pdig.0000982.ref015">15</a>]. Such simplifications may not fully capture the complex and often non-linear dynamics inherent in disease progression. For instance, associations between risk factors such as hypertension on CVD outcomes may become more complex in patients with a history of early childhood adversities. Such adversities could lead to mental disorders and diabetes, potentially intensifying the interplay between hypertension and CVD outcomes disproportionately. This exemplifies a non-linear dynamic, where the combined effect of these conditions on CVD progression is greater than the sum of their individual effects, particularly in patients with such comorbidity profiles. Building on these observations, it is also important to consider the role of ELFs in these prediction models. While research has identified associations between ELFs and the later development of CVD or diabetes, the extent of their contribution to predictive accuracy is unclear. It is important to distinguish that merely identifying statistical associations between variables, even if they are longitudinal and strong, does not necessarily translate into effective predictors for new observations, underscoring a gap in applying these findings to predictive modeling [<a href="#pdig.0000982.ref016" class="usa-link" aria-describedby="pdig.0000982.ref016">16</a>]. In a similar vein, causal inference approaches (e.g., propensity scoring) can offer insights into mechanisms or intervention effects, but they are not designed to optimize predictive performance in complex, comorbid populations. Therefore, the strategic selection and application of these factors are essential for accurate predictions [<a href="#pdig.0000982.ref017" class="usa-link" aria-describedby="pdig.0000982.ref017">17</a>]. Machine learning (ML) presents a transformative opportunity to enhance predictive models for CVD and diabetes. ML approaches, capable of processing vast and complex datasets, can identify not only novel risk factors but also intricate, non-linear interactions among them [<a href="#pdig.0000982.ref013" class="usa-link" aria-describedby="pdig.0000982.ref013">13</a>]. This capability is expected to improve the accuracy of predictive models [<a href="#pdig.0000982.ref013" class="usa-link" aria-describedby="pdig.0000982.ref013">13</a>, <a href="#pdig.0000982.ref018" class="usa-link" aria-describedby="pdig.0000982.ref018">18</a>].</p>
<p>Given the dynamics of PCM multimorbidity, we expect that ML surpasses traditional approaches in accurately predicting CVD and diabetes by integrating ELFs and more proximal risk factors as predictors in ML models. In this paper, we leverage ML to investigate the role of ELFs in predicting CVD and diabetes, particularly within comorbid groups. Specifically, we employ XGBoost (XGB) [<a href="#pdig.0000982.ref019" class="usa-link" aria-describedby="pdig.0000982.ref019">19</a>] models. The ability of this type of model to handle high-dimensional data makes it appropriate for biomedical studies. This choice is supported by findings from a recent systematic review on type 2 diabetes prediction, where tree-based models were shown to achieve superior predictive performance [<a href="#pdig.0000982.ref020" class="usa-link" aria-describedby="pdig.0000982.ref020">20</a>]. Utilizing the UK Biobank (UKB) [<a href="#pdig.0000982.ref021" class="usa-link" aria-describedby="pdig.0000982.ref021">21</a>], a comprehensive national health resource from the UK, our objectives are twofold. First, we aim to systematically compare the long-term predictive ability of ELFs for CVD and diabetes among individuals with and without comorbidities. We define these comorbidities as the co-occurrence of mental disorders (depression or anxiety) with either diabetes, in the context of CVD prediction, or with CVD, when predicting diabetes outcomes. Second, we explore the prediction model drivers of these outcomes, including the potential role of ELFs as key contributors, thereby enhancing our understanding of developing multimorbidity. Through this approach, we aim to provide insights that could enhance the precision of existing risk prediction algorithms, specifically tailored for individuals with comorbid conditions. This research incorporates both proximal and early-life risk factors within a large prospective cohort study, rigorously tracking a wide range of variables for each participant. Recent studies have explored the ability of automated ML frameworks to explore wide arrays of non-traditional risk factors. For example, up to 473 and 109 input variables were considered for CVD [<a href="#pdig.0000982.ref013" class="usa-link" aria-describedby="pdig.0000982.ref013">13</a>] and diabetes [<a href="#pdig.0000982.ref018" class="usa-link" aria-describedby="pdig.0000982.ref018">18</a>] prediction, respectively. However, ELFs were excluded due to missing rates exceeding 50% in patient outcomes. This study conducts a comprehensive investigation into the value of ELFs in predicting CVD and diabetes within comorbid groups, utilizing statistical and ML techniques. Our study shows that these factors not only enhance the precision of our predictive models but also prove to be key drivers of prediction in these complex patient groups.</p></section><section id="sec002"><h2 class="pmc_sec_title">Materials and methods</h2>
<section id="sec003"><h3 class="pmc_sec_title">Study design and participants</h3>
<p><strong><em>Selection of dataset.</em></strong> For this study, we use the UKB for its comprehensive health data, which spans demographics, health status, and disease incidence among a diverse representation of the UK population [<a href="#pdig.0000982.ref021" class="usa-link" aria-describedby="pdig.0000982.ref021">21</a>]. The archive includes fully anonymized records from half a million individuals collected from 2006 to 2019. For our analysis, a subset of approximately 150,000 participants without prior diagnoses of CVD or diabetes for which ELFs were available was selected. This allows us to track the onset of these conditions over a decade. <a href="#pdig.0000982.s002" class="usa-link">S1 Fig</a> illustrates the stratification of these participants, detailing the process for selecting the final study cohorts with CVD and diabetes outcomes.</p>
<p>The UKB data were accessed for research purposes from November 11, 2022, to December 17, 2023. Access to the UKB data for this research is facilitated under the project titled ’Association between Early-Life-Stress and Psycho-Cardio-Metabolic Multi-Morbidity: The EarlyCause H2020 Project’ (application number 65769). The UKB obtained ethical approval from the North West Multi-centre Research Ethics Committee (MREC) and the Community Health Index Advisory Group (CHIAG). We adhere to the ethical approval obtained by the UKB through the signature of MTA (Material Transfer Agreement). All participants in this study provided written informed consent.</p>
<p><strong><em>Selection of variables.</em></strong> The study considers a range of ELFs available from our UKB application, including eight ELF variables (both prenatal and postnatal, including exposure to maternal smoking and childhood maltreatment; details in <a href="#pdig.0000982.s001" class="usa-link">S1 Table</a>), contextualized within a spectrum of demographic and proximal risk factors. Demographic characteristics include age, sex, and ethnicity, while proximal risk factors comprise external exposome data: socioeconomic status (income, qualifications) and physio-metric data (body mass index, height, weight, etc.). They also encompass family history (illnesses of parents or siblings), medical history, lifestyle characteristics (physical activity, diet, sleep habits, smoking, and alcohol use), mental health history (depression, anxiety), and subjective well-being indicators (mood swings, loneliness, etc.). Internal exposome is also available in the form of blood assay results, reflecting the body’s internal biological environment. This includes key biomarkers such as Apolipoprotein A, Apolipoprotein B, C-reactive protein, glycated hemoglobin (HbA1c), and random glucose (RG). Note that while HbA1c and glucose are well-established markers for diabetes [<a href="#pdig.0000982.ref022" class="usa-link" aria-describedby="pdig.0000982.ref022">22</a>], in our study, they are measured at study entry and treated as baseline predictors. Our models are designed to assess future diabetes risk in an initially non-diabetic population, with individuals diagnosed with diabetes at baseline excluded from the cohort. This approach avoids data leakage and aligns with prior diabetes risk prediction research [<a href="#pdig.0000982.ref018" class="usa-link" aria-describedby="pdig.0000982.ref018">18</a>, <a href="#pdig.0000982.ref023" class="usa-link" aria-describedby="pdig.0000982.ref023">23</a>, <a href="#pdig.0000982.ref024" class="usa-link" aria-describedby="pdig.0000982.ref024">24</a>]. All variables had a missingness rate of less than 25% among participants with positive diabetes outcomes, corresponding to a missingness rate of 20% for the entire participant population. To address missing data, we applied a statistical imputation algorithm to recover the missing values. <a href="#pdig.0000982.s001" class="usa-link">S1 Table</a> lists the ELFs along with the other input variables among the 64 considered, grouped by category (e.g., sociodemographics, psychosocial factors, blood assays). These variables were preprocessed into 83 features as a result of one-hot encoding applied to categorical variables with multiple levels.</p>
<p><strong><em>Selection of benchmarks.</em></strong> The performance of our models is compared with that of well-established risk prediction algorithms: the Framingham score for CVD and DiabetesUK for diabetes. The Framingham score, based on seven core risk factors (sex, age, systolic blood pressure, treatment for hypertension, smoking status, history of diabetes, and BMI), is a widely recognized algorithm for assessing CVD risk [<a href="#pdig.0000982.ref025" class="usa-link" aria-describedby="pdig.0000982.ref025">25</a>]. UKDiabetes [<a href="#pdig.0000982.ref026" class="usa-link" aria-describedby="pdig.0000982.ref026">26</a>], which incorporates seven features (sex, age, ethnicity, family history, waist size, BMI, and high blood pressure requiring treatment), was selected for assessing diabetes risk due to its alignment with the demographics of the UKB dataset. Framingham and DiabetesUK models are used in our study as benchmarks to visualize the challenges of risk prediction in the comorbid population. Although their performance could be improved through different learning strategies, we decided to use the available models as model optimization is not the primary focus of our study. Additionally, we assess the performance of ML models learned using only the seven core variables used by Framingham score and Diabetes UK for CVD and diabetes risk prediction, respectively. This enables us to separately investigate the influence of employing ML models and the impact of using a broader variable set on prediction accuracy.</p>
<p><strong><em>Model and validation strategy.</em></strong> In our study, we utilize XGB, an ML algorithm that sets a series of gradient-boosted trees to progressively refine predictions, thereby systematically enhancing model accuracy. Its nonparametric nature contributes to robustness in high-dimensional settings, and the algorithm’s ability to recognize complex variable interactions makes it particularly apt for this study. Models are implemented using Python’s Scikit-learn library [<a href="#pdig.0000982.ref027" class="usa-link" aria-describedby="pdig.0000982.ref027">27</a>], with hyperparameter optimization via the Optuna framework [<a href="#pdig.0000982.ref028" class="usa-link" aria-describedby="pdig.0000982.ref028">28</a>]. Note that our goal is <em>not</em> solely to develop the best ML frameworks for CVD and diabetes risk prediction possible, but rather to leverage ML models’ capacities to reveal latent relationships among variables, particularly to elucidate the association between ELFs and the development of PCM conditions.</p>
<p>To ensure the reported model performance is robust and not unduly influenced by overfitting, we conduct 5-fold stratified cross-validation. Our evaluation metrics include the area under the receiver operating characteristic curve (AUC-ROC) for assessing prediction accuracy, and the Brier score for model calibration. These are separately assessed on non-comorbid and comorbid test groups. It is important to note that a single, unified XGB model is trained to evaluate performance across both the comorbid and non-comorbid subgroups, rather than developing separate models for each subgroup. This approach allows for a consistent evaluation of predictor contributions and comparative performance across the two groups. Additionally, decision curve analysis [<a href="#pdig.0000982.ref029" class="usa-link" aria-describedby="pdig.0000982.ref029">29</a>] is performed to assess the clinical utility of models for the test’s comorbid subgroup. This analysis calculates the clinical ’net benefit’ of prediction models across a range of risk threshold probabilities, <em>t</em>, which represents the minimum likelihood at which intervention is considered. Given a threshold <em>t</em>, the net benefit <em>b</em>(<em>t</em>) is defined by the difference between the proportion of true positives (TP) and the proportion of false positives (FP) adjusted for <em>t</em>:</p>
<table class="disp-formula p" id="pdig.0000982.e001"><tr><td class="formula"><math id="M1" display="block" overflow="linebreak"><mrow><mrow><mi>b</mi><mo stretchy="false">(</mo><mi>t</mi><mo stretchy="false">)</mo><mo>=</mo><mfrac><mrow><mtext>TP</mtext></mrow><mrow><mi>n</mi></mrow></mfrac><mo>−</mo><mfrac><mrow><mtext>FP</mtext></mrow><mrow><mi>n</mi></mrow></mfrac><mrow><mo fence="true" form="prefix" stretchy="true">(</mo><mfrac><mrow><mi>t</mi></mrow><mrow><mn>1</mn><mo>−</mo><mi>t</mi></mrow></mfrac><mo fence="true" form="postfix" stretchy="true">)</mo></mrow></mrow></mrow></math></td></tr></table>
<p>where <em>n</em> represents the number of samples. For the sake of comparison, we also show the decision curves of two simplistic models: one that predicts risk for all patients, and a model that never predicts risk. Models that yield a higher net benefit at clinically relevant decision thresholds are preferred, as they provide a more advantageous balance between correctly identifying cases and avoiding unnecessary interventions for these groups.</p>
<p><strong><em>Outcome.</em></strong> In our study, a CVD event is identified through the assignment of ICD-10 diagnosis codes F01 (vascular dementia), I20-I25 (coronary/ischemic heart diseases), I50 (heart failure events, including acute and chronic systolic heart failures), and I60-I69 (cerebrovascular diseases), or ICD-9 codes 410-414 (ischemic heart disease) and 430-434, 436-438 (cerebrovascular disease), with self-reported data also included. While the diagnosis of vascular dementia can be challenging due to confounding factors, its inclusion follows methodologies from prior large-scale studies to ensure comprehensive coverage of CVD-related conditions [<a href="#pdig.0000982.ref013" class="usa-link" aria-describedby="pdig.0000982.ref013">13</a>]. For diabetes, events are identified by the assignment of ICD-10 codes E10 (type 1 diabetes), E11 (type 2 diabetes), and E13/E14 (unspecified diabetes), or ICD-9 code 250 (Diabetes mellitus), along with self-reported data. Gestational diabetes (ICD-10 O24) and diabetes insipidus (ICD-10 E23.2) were not included as they are distinct conditions with separate etiologies and clinical implications. Additionally, comorbid conditions of depression and anxiety are identified based on ICD-10 codes F32 and F33, and ICD-9 code 311 for depression, and ICD-10 code F41 and ICD-9 code 3000 for anxiety, including self-reported instances of these conditions.</p></section><section id="sec004"><h3 class="pmc_sec_title">Variable contribution analysis</h3>
<p>In this section, we examine the contribution of individual variables to model performance and decision-making, including early-life factors. Specifically, we study their predictive utility for health outcomes and their role in shaping model decisions across subgroups.</p>
<p><strong><em>Group-specific Bayes error estimation.</em></strong> In our analysis, we quantify the performance limits of CVD and diabetes predictive tasks across patient subgroups by approximating the theoretical Bayes optimal classifier, which attains the smallest expected error. This minimal achievable error is also referred to as the Bayes error or irreducible noise, representing the portion of prediction error that cannot be eliminated by any model regardless of the learning algorithm. Estimating group-specific Bayes error provides insights into the inherent difficulty of disease prediction within each subgroup, helping to identify whether certain populations are fundamentally harder to forecast, independent of model choice.</p>
<p>We need to estimate the Bayes error since calculating the real value is unfeasible in general practice. We estimate its upper and lower bounds to narrow it down for each subgroup. The lower bound (Elow) represents the best-case minimal error achievable under ideal conditions, while the upper bound (Eup) reflects a worst-case estimate of the minimal error attainable even by the optimal classifier. The interval drawn with the upper and lower bounds for different subgroups can be compared. Non-overlapping intervals might indicate a different inherent difficulty of prediction for the groups.</p>
<p>We conduct a comparative evaluation on the estimation of Bayes error lower and upper bounds (<em>E</em><sub><em>low</em></sub> and <em>E</em><sub><em>up</em></sub>) for non-comorbid and comorbid groups, following the methodology outlined in [<a href="#pdig.0000982.ref030" class="usa-link" aria-describedby="pdig.0000982.ref030">30</a>], through non-parametric techniques including the Mahalanobis distance [<a href="#pdig.0000982.ref031" class="usa-link" aria-describedby="pdig.0000982.ref031">31</a>], Bhattacharyya distance [<a href="#pdig.0000982.ref032" class="usa-link" aria-describedby="pdig.0000982.ref032">32</a>], and a k-nearest neighbor method [<a href="#pdig.0000982.ref033" class="usa-link" aria-describedby="pdig.0000982.ref033">33</a>] with <em>k</em> = 5 and 5-fold cross-validation. This analysis is applied across two distinct sets: a subset of the seven core variables utilized by benchmarks and the whole set of variables, to investigate the benefits of using additional information to make predictions for specific patient subgroups.</p>
<p><strong><em>Performance model drivers.</em></strong> Performance model drivers are variables that significantly impact the model’s accuracy, serving as key determinants in achieving optimal predictive performance. These are referred to as <em>predictive features</em>. In our study, we focus on evaluating the impact of ELFs and other risk factors on the performance of predictive models for CVD and diabetes. Our goal is to quantify how specific features drive model performance, thereby allowing us to identify robust risk predictors, ultimately leading to enhanced risk prediction performance. To achieve this, we employ a twofold strategy rigorously applied to the test sets to ensure the robustness and generalizability of our findings. We measure (1) the individual predictive ability of each variable and (2) their synergistic predictive power when combined with others.</p>
<p><em>1. Predictive power of individual variables.</em> To evaluate the individual predictive ability of each variable, we fit XGB models for each of the 64 variables separately. Each model is trained with all the training samples, utilizing a single variable as input. Separate evaluations of these models are carried out for the non-comorbid and comorbid test groups using the AUC-ROC. This comparison allows us to identify which variables have a consistent predictive capacity across different health conditions and which are unique or more influential in one group over the other.</p>
<p><em>2. Predictive power of synergistic variables.</em> While assessing the <em>predictive power of individual variables</em> reveals their standalone relevance, we use permutation-based variable importance (PFI) to evaluate their relative influence by measuring the change in the model’s performance upon each variable’s perturbation. Specifically, we calculate the increase in prediction error resulting from the random shuffling of the variable’s values across samples [<a href="#pdig.0000982.ref034" class="usa-link" aria-describedby="pdig.0000982.ref034">34</a>]. If a variable is predictive of the outcome, shuffling will weaken its relationship with the outcome and consequently increase the model’s error. Variables with no strong relationship with the outcome will see little to no change in error after shuffling. Permutation-based feature importance can underestimate the influence of highly correlated features as the information provided by one of them is still available after shuffling from the other one. To anticipate potential underestimation resulting from highly correlated features, Spearman’s rank correlation analysis [<a href="#pdig.0000982.ref035" class="usa-link" aria-describedby="pdig.0000982.ref035">35</a>] is conducted on our training data. Our examination confirms that none of the variables exhibit complete correlation, ensuring the robustness of our analysis. We define <em>PFI</em><sub><em>j</em></sub> for each variable <em>j</em> as follows:</p>
<table class="disp-formula p" id="pdig.0000982.e002"><tr><td class="formula"><math id="M2" display="block" overflow="linebreak"><mrow><mrow><mi>P</mi><mi>F</mi><msub><mi>I</mi><mrow><mi>j</mi></mrow></msub><mo>=</mo><mfrac><mrow><mn>1</mn></mrow><mrow><mi>n</mi></mrow></mfrac><msubsup><mo>∑</mo><mrow><mi>i</mi><mo>=</mo><mn>1</mn></mrow><mrow><mi>n</mi></mrow></msubsup><mi>L</mi><mo stretchy="false">(</mo><msub><mi>y</mi><mi>i</mi></msub><mo>,</mo><mover><mrow><mi>f</mi></mrow><mo stretchy="false">^</mo></mover><mo stretchy="false">(</mo><msub><mi>x</mi><mi>i</mi></msub><mo stretchy="false">)</mo><mo stretchy="false">)</mo><mo>−</mo><mi>L</mi><mo stretchy="false">(</mo><msub><mi>y</mi><mi>i</mi></msub><mo>,</mo><mover><mrow><mi>f</mi></mrow><mo stretchy="false">^</mo></mover><mo stretchy="false">(</mo><msubsup><mi>x</mi><mi>i</mi><mrow><mi>s</mi><mi>h</mi><mo stretchy="false">(</mo><mi>j</mi><mo stretchy="false">)</mo></mrow></msubsup><mo stretchy="false">)</mo><mo stretchy="false">)</mo></mrow></mrow></math></td></tr></table>
<p>where <em>n</em> represents the number of samples in the group being analyzed, which could be either the non-comorbid (NC) or comorbid (C) test group, <em>L</em> is the loss function used to evaluate the model, <em>y</em><sub><em>i</em></sub> is the true outcome for sample <em>x</em><sub><em>i</em></sub>, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pdig.0000982.e003"><math id="M3" display="inline" overflow="linebreak"><mrow><mover><mrow><mi>f</mi></mrow><mo stretchy="false">^</mo></mover><mo stretchy="false">(</mo><msub><mi>x</mi><mi>i</mi></msub><mo stretchy="false">)</mo></mrow></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pdig.0000982.e004"><math id="M4" display="inline" overflow="linebreak"><mrow><mover><mrow><mi>f</mi></mrow><mo stretchy="false">^</mo></mover><mo stretchy="false">(</mo><msubsup><mi>x</mi><mi>i</mi><mrow><mi>s</mi><mi>h</mi><mo stretchy="false">(</mo><mi>j</mi><mo stretchy="false">)</mo></mrow></msubsup><mo stretchy="false">)</mo></mrow></math></span> are the model’s predictions before and after permuting the <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pdig.0000982.e005"><math id="M5" display="inline" overflow="linebreak"><mrow><msup><mtext mathvariant="italic">j</mtext><mrow><mtext mathvariant="italic">th</mtext></mrow></msup></mrow></math></span> variable. A large <em>PFI</em><sub><em>j</em></sub> value indicates that variable <em>j</em> is a significant contributor to the model’s predictive performance.</p>
<p><strong><em>Prediction model drivers.</em></strong> Prediction model drivers are variables that directly influence the model’s predictive decisions, thereby determining the specific outcomes of interest. They are also referred to as decision’s <em>explanatory features</em>. Prediction drivers influence the outcome provided by the model, being it right or wrong, whereas performance drivers boost the right outcome. In our study, we aim to identify and quantify the role of ELFs, alongside other risk factors, in influencing our models’ prediction of CVD and diabetes across distinct sub-populations, specifically within the non-comorbid (NC) and comorbid (C) groups. To achieve this, we employ a twofold strategy:</p>
<p><em>1. SHAP variable importance.</em> We integrate SHapley Additive exPlanations (SHAP) values [<a href="#pdig.0000982.ref036" class="usa-link" aria-describedby="pdig.0000982.ref036">36</a>], a local model-agnostic interpretation method, to quantify the marginal contribution of each variable <em>j</em> in the prediction of model <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pdig.0000982.e006"><math id="M6" display="inline" overflow="linebreak"><mrow><mover><mrow><mi>f</mi></mrow><mo stretchy="false">^</mo></mover></mrow></math></span> for instance <em>x</em>. In this study, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pdig.0000982.e007"><math id="M7" display="inline" overflow="linebreak"><mrow><mover><mrow><mi>f</mi></mrow><mo stretchy="false">^</mo></mover></mrow></math></span> refers to our XGB model. This method helps us understand how individual variables affect specific predictive outcomes, providing insights into their influence on the model’s decision-making process at an individual prediction level. The SHAP value is defined as:</p>
<table class="disp-formula p" id="pdig.0000982.e008"><tr><td class="formula"><math id="M8" display="block" overflow="linebreak"><mrow><mrow><msub><mi>ϕ</mi><mi>j</mi></msub><mo stretchy="false">(</mo><mi>x</mi><mo stretchy="false">)</mo><mo>=</mo><msub><mo>∑</mo><mrow><mi>V</mi><mo>⊆</mo><mi>F</mi><mi>⧵</mi><mo stretchy="false">{</mo><mi>j</mi><mo stretchy="false">}</mo></mrow></msub><mfrac><mrow><mo stretchy="false">|</mo><mi>V</mi><mo stretchy="false">|</mo><mo>!</mo><mo stretchy="false">(</mo><mo stretchy="false">|</mo><mi>F</mi><mo stretchy="false">|</mo><mo>−</mo><mo stretchy="false">|</mo><mi>V</mi><mo stretchy="false">|</mo><mo>−</mo><mn>1</mn><mo stretchy="false">)</mo><mo>!</mo></mrow><mrow><mo stretchy="false">|</mo><mi>F</mi><mo stretchy="false">|</mo><mo>!</mo></mrow></mfrac><mo stretchy="false">[</mo><msub><mover><mrow><mi>f</mi></mrow><mo stretchy="false">^</mo></mover><mrow><mi>V</mi><mo>∪</mo><mo stretchy="false">{</mo><mi>j</mi><mo stretchy="false">}</mo></mrow></msub><mo stretchy="false">(</mo><msub><mi>x</mi><mrow><mi>V</mi><mo>∪</mo><mo stretchy="false">{</mo><mi>j</mi><mo stretchy="false">}</mo></mrow></msub><mo stretchy="false">)</mo><mo>−</mo><msub><mover><mrow><mi>f</mi></mrow><mo stretchy="false">^</mo></mover><mi>V</mi></msub><mo stretchy="false">(</mo><msub><mi>x</mi><mi>V</mi></msub><mo stretchy="false">)</mo><mo stretchy="false">]</mo></mrow></mrow></math></td></tr></table>
<p>where <em>F</em> is the set of all features and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pdig.0000982.e009"><math id="M9" display="inline" overflow="linebreak"><mrow><mo stretchy="false">|</mo><mi>F</mi><mo stretchy="false">|</mo></mrow></math></span> is the number of features. SHAP value is computed for each feature over all possible variable combinations (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pdig.0000982.e010"><math id="M10" display="inline" overflow="linebreak"><mrow><mi>V</mi><mo>⊆</mo><mi>F</mi></mrow></math></span>), effectively capturing the non-linear and interaction effects characteristic of complex biological systems. The local accuracy property of SHAP ensures that the sum of all variable contributions equals the model’s prediction, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pdig.0000982.e011"><math id="M11" display="inline" overflow="linebreak"><mrow><mover><mrow><mi>f</mi></mrow><mo stretchy="false">^</mo></mover><mo stretchy="false">(</mo><mi>x</mi><mo stretchy="false">)</mo><mo>=</mo><mi>E</mi><mo stretchy="false">[</mo><mover><mrow><mi>f</mi></mrow><mo stretchy="false">^</mo></mover><mo stretchy="false">(</mo><mi>x</mi><mo stretchy="false">)</mo><mo stretchy="false">]</mo><mspace width="0.167em"></mspace><mrow><mo>+</mo></mrow><mspace width="0.167em"></mspace><msubsup><mo>∑</mo><mrow><mi>j</mi><mo>=</mo><mn>1</mn></mrow><mrow><mo stretchy="false">|</mo><mi>F</mi><mo stretchy="false">|</mo></mrow></msubsup><msub><mi>ϕ</mi><mi>j</mi></msub><mo stretchy="false">(</mo><mi>x</mi><mo stretchy="false">)</mo></mrow></math></span>, where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pdig.0000982.e012"><math id="M12" display="inline" overflow="linebreak"><mrow><mi>E</mi><mo stretchy="false">[</mo><mover><mrow><mi>f</mi></mrow><mo stretchy="false">^</mo></mover><mo stretchy="false">(</mo><mi>x</mi><mo stretchy="false">)</mo><mo stretchy="false">]</mo></mrow></math></span> is the expected value of the model’s output. This decomposition enables the identification of the most influential variables as those with greater contributions to the prediction show higher SHAP value attributions. In this study, global variable importance is evaluated by calculating the average of the absolute SHAP values across instances. We define the SHAP variable importance <em>I</em><sub><em>j</em></sub> for each variable <em>j</em> as follows:</p>
<table class="disp-formula p" id="pdig.0000982.e013"><tr><td class="formula"><math id="M13" display="block" overflow="linebreak"><mrow><mrow><msub><mi>I</mi><mrow><mi>j</mi></mrow></msub><mo>=</mo><mfrac><mrow><mn>1</mn></mrow><mrow><mi>n</mi></mrow></mfrac><msubsup><mo>∑</mo><mrow><mi>i</mi><mo>=</mo><mn>1</mn></mrow><mrow><mi>n</mi></mrow></msubsup><mo stretchy="false">|</mo><msub><mi>ϕ</mi><mrow><mi>j</mi></mrow></msub><mo stretchy="false">(</mo><msub><mi>x</mi><mrow><mi>i</mi></mrow></msub><mo stretchy="false">)</mo><mo stretchy="false">|</mo></mrow></mrow></math></td></tr></table>
<p>where <em>n</em> represents the number of test samples in the group being analyzed, which could be either the non-comorbid (NC) or comorbid (C) group. Through this dual analysis, we identify features integral to both comorbid and non-comorbid states, as well as those exclusive to comorbid conditions. This approach clarifies the significance of ELFs and other variables in their contribution to comorbid profiles.</p>
<p><em>2. Dual-group risk effect size.</em> While <em>SHAP values</em> provide a detailed, instance-level understanding of variable contributions, the <em>risk effect size</em> offers a complementary perspective on how variables influence risk profiles across sub-groups. Effect size, in statistical terms, refers to the magnitude of the difference between groups in a study (e.g., difference between group means). In our context, risk effect size quantifies the degree to which a variable influences the model’s predicted risk distribution between high-risk and low-risk groups for developing CVD and diabetes. We use Cohen’s D [<a href="#pdig.0000982.ref037" class="usa-link" aria-describedby="pdig.0000982.ref037">37</a>], a statistical metric that quantifies the difference in means between two groups, to measure the variable’s distributional shift between high-risk and low-risk groups, as predicted by the model. The larger the Cohen’s D value, the greater the distributional shift, and thus, the greater the influence of the feature on the model’s risk stratification. We apply this procedure (Algorithm 1) to both non-comorbid and comorbid test groups, separately. Through separate rankings of features based on their Cohen’s D values in each group, we discern variables that significantly impact the risk profile, reflecting their distinct contributions across these diverse health contexts.</p>
<p>
<strong>Algorithm 1: Dual-group risk effect size analyzer.</strong>
</p>
<p>
<kbd><strong>Input:</strong> S, f, <em>t</em> = 0.5 <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pdig.0000982.e014"><math id="M14" display="inline" overflow="linebreak"><mrow><mo>⊳</mo></mrow></math></span> S: set of samples, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pdig.0000982.e015"><math id="M15" display="inline" overflow="linebreak"><mrow><mrow><mi>f</mi></mrow></mrow></math></span>: model, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pdig.0000982.e016"><math id="M16" display="inline" overflow="linebreak"><mrow><mrow><mi>t</mi></mrow></mrow></math></span>:</kbd>
</p>
<p>
<kbd>  threshold to define low/high-risk patients (default value, 0.5)</kbd>
</p>
<p>
<kbd><strong>Output:</strong> Variables sorted by descending <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pdig.0000982.e017"><math id="M17" display="inline" overflow="linebreak"><mrow><msub><mi>𝒟</mi><mrow><mi>j</mi></mrow></msub></mrow></math></span> value</kbd>
</p>
<p>
<kbd>1: Identify the high-risk and low-risk patients by applying</kbd>
</p>
<p>
<kbd>  threshold <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pdig.0000982.e018"><math id="M18" display="inline" overflow="linebreak"><mrow><mrow><mi>t</mi></mrow></mrow></math></span> to <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pdig.0000982.e019"><math id="M19" display="inline" overflow="linebreak"><mrow><mrow><mi>f</mi></mrow></mrow></math></span>’s risk probability predictions.</kbd>
</p>
<p>
<kbd>  <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pdig.0000982.e020"><math id="M20" display="inline" overflow="linebreak"><mrow><msub><mi>G</mi><mrow><mtext>high</mtext></mrow></msub><mo>=</mo><mo stretchy="false">{</mo><mi>x</mi><mo>∈</mo><mtext>S</mtext><mo>∣</mo><mi>f</mi><mo stretchy="false">(</mo><mi>x</mi><mo stretchy="false">)</mo><mo>&gt;</mo><mi>t</mi><mo stretchy="false">}</mo></mrow></math></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pdig.0000982.e021"><math id="M21" display="inline" overflow="linebreak"><mrow><msub><mi>G</mi><mrow><mtext>low</mtext></mrow></msub><mo>=</mo><mo stretchy="false">{</mo><mi>x</mi><mo>∈</mo><mtext>S</mtext><mo>∣</mo><mi>f</mi><mo stretchy="false">(</mo><mi>x</mi><mo stretchy="false">)</mo><mo>≤</mo><mn>1</mn><mo>−</mo><mi>t</mi><mo stretchy="false">}</mo></mrow></math></span></kbd>
</p>
<p>
<kbd>2: For each sample subset and feature <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pdig.0000982.e022"><math id="M22" display="inline" overflow="linebreak"><mrow><mrow><mi>j</mi></mrow></mrow></math></span>, compute Cohen’s D:</kbd>
</p>
<p>
<kbd>  <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pdig.0000982.e023"><math id="M23" display="inline" overflow="linebreak"><mrow><msub><mi>𝒟</mi><mrow><mi>j</mi></mrow></msub><mo>=</mo><mrow><mo fence="true" form="prefix" stretchy="true">|</mo><msub><mover><mrow><mi>X</mi></mrow><mo stretchy="true">¯</mo></mover><mrow><mi>j</mi><mo>,</mo><mtext>high</mtext></mrow></msub><mo>−</mo><msub><mover><mrow><mi>X</mi></mrow><mo stretchy="true">¯</mo></mover><mrow><mi>j</mi><mo>,</mo><mtext>low</mtext></mrow></msub><mo fence="true" form="postfix" stretchy="true">|</mo></mrow><mo>/</mo><msub><mi>s</mi><mrow><mi>j</mi><mo>,</mo><mtext>pooled</mtext></mrow></msub></mrow></math></span></kbd>
</p>
<p>
<kbd>  where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pdig.0000982.e024"><math id="M24" display="inline" overflow="linebreak"><mrow><msub><mover><mrow><mi>X</mi></mrow><mo stretchy="true">¯</mo></mover><mrow><mi>j</mi><mo>,</mo><mtext>high</mtext></mrow></msub><mo>=</mo><mfrac><mrow><mn>1</mn></mrow><mrow><mo stretchy="false">|</mo><msub><mi>G</mi><mrow><mi>h</mi><mi>i</mi><mi>g</mi><mi>h</mi></mrow></msub><mo stretchy="false">|</mo></mrow></mfrac><msub><mo>∑</mo><mrow><mi>x</mi><mo>∈</mo><msub><mi>G</mi><mrow><mi>h</mi><mi>i</mi><mi>g</mi><mi>h</mi></mrow></msub></mrow></msub><msub><mi>x</mi><mi>j</mi></msub></mrow></math></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pdig.0000982.e025"><math id="M25" display="inline" overflow="linebreak"><mrow><msub><mover><mrow><mi>X</mi></mrow><mo stretchy="true">¯</mo></mover><mrow><mi>j</mi><mo>,</mo><mtext>low</mtext></mrow></msub><mo>=</mo><mfrac><mrow><mn>1</mn></mrow><mrow><mo stretchy="false">|</mo><msub><mi>G</mi><mrow><mi>l</mi><mi>o</mi><mi>w</mi></mrow></msub><mo stretchy="false">|</mo></mrow></mfrac><msub><mo>∑</mo><mrow><mi>x</mi><mo>∈</mo><msub><mi>G</mi><mrow><mi>l</mi><mi>o</mi><mi>w</mi></mrow></msub></mrow></msub><msub><mi>x</mi><mi>j</mi></msub></mrow></math></span> and</kbd>
</p>
<p>
<kbd>  <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pdig.0000982.e026"><math id="M26" display="inline" overflow="linebreak"><mrow><msub><mi>s</mi><mrow><mi>j</mi><mo>,</mo><mtext>pooled</mtext></mrow></msub><mo>=</mo><mrow><mo fence="true" form="prefix" stretchy="true">(</mo><mrow><mo fence="true" form="prefix" stretchy="true">(</mo><mo stretchy="false">(</mo><mo stretchy="false">|</mo><msub><mi>G</mi><mrow><mtext>high</mtext></mrow></msub><mo stretchy="false">|</mo><mo>−</mo><mn>1</mn><mo stretchy="false">)</mo><mo>·</mo><msubsup><mi>s</mi><mrow><mi>j</mi><mo>,</mo><mtext>high</mtext></mrow><mn>2</mn></msubsup><mo>+</mo><mo stretchy="false">(</mo><mo stretchy="false">|</mo><msub><mi>G</mi><mrow><mtext>low</mtext></mrow></msub><mo stretchy="false">|</mo><mo>−</mo><mn>1</mn><mo stretchy="false">)</mo><mo>·</mo><msubsup><mi>s</mi><mrow><mi>j</mi><mo>,</mo><mtext>low</mtext></mrow><mn>2</mn></msubsup><mo fence="true" form="postfix" stretchy="true">)</mo></mrow><mo>/</mo><mrow><mo fence="true" form="prefix" stretchy="true">(</mo><mo stretchy="false">|</mo><msub><mi>G</mi><mrow><mtext>high</mtext></mrow></msub><mo stretchy="false">|</mo><mo>+</mo><mo stretchy="false">|</mo><msub><mi>G</mi><mrow><mtext>low</mtext></mrow></msub><mo stretchy="false">|</mo><mo>−</mo><mn>2</mn><mo fence="true" form="postfix" stretchy="true">)</mo></mrow><mo fence="true" form="postfix" stretchy="true">)</mo></mrow><msup><mi> </mi><mrow><mfrac><mrow><mn>1</mn></mrow><mrow><mn>2</mn></mrow></mfrac></mrow></msup></mrow></math></span></kbd>
</p>
<p>
<kbd>  with <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pdig.0000982.e027"><math id="M27" display="inline" overflow="linebreak"><mrow><msubsup><mi>s</mi><mrow><mi>j</mi><mo>,</mo><mtext>high</mtext></mrow><mn>2</mn></msubsup></mrow></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pdig.0000982.e028"><math id="M28" display="inline" overflow="linebreak"><mrow><msubsup><mi>s</mi><mrow><mi>j</mi><mo>,</mo><mtext>low</mtext></mrow><mn>2</mn></msubsup></mrow></math></span> being the variances of feature <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pdig.0000982.e029"><math id="M29" display="inline" overflow="linebreak"><mrow><mrow><mi>j</mi></mrow></mrow></math></span> for <em>G</em><sub><em>high</em></sub></kbd>
</p>
<p>
<kbd>  and <em>G</em><sub><em>low</em></sub>, respectively</kbd>
</p>
<p>
<kbd>3: Sort variables by descending <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pdig.0000982.e030"><math id="M30" display="inline" overflow="linebreak"><mrow><msub><mi>𝒟</mi><mrow><mi>j</mi></mrow></msub></mrow></math></span> value</kbd>
</p>
<p>To enhance the robustness of the estimation of the variables’ predictive power and their contribution to model decisions, 5-fold stratified cross-validation is used. Rather than relying on a single train-test split, our results average empirical evidence across different training-test data splits, using the entire dataset for testing. It is important to understand that SHAP values help interpret the influence of each individual variable into the model’s predictions (e.g., a variable taking a specific value might be associated in the model with a large predicted probability of disease). However, note that this cannot be interpreted as a measurement of how much each variable boosts the model’s predictive performance (e.g., accuracy or AUC). SHAP values have been used in the past to rank the top predictors for diabetes risk in the general population [<a href="#pdig.0000982.ref023" class="usa-link" aria-describedby="pdig.0000982.ref023">23</a>], offering insights into key model contributors. We distinguish between prediction and performance drivers and use the appropriate techniques to find them.</p>
<p><strong><em>Post-hoc analysis based on dominant performance drivers.</em></strong> In addition to the primary analyses, we conducted exploratory, post-hoc evaluations to further characterize the contribution of variables that emerged as dominant performance drivers. In particular, HbA1c is identified as a key performance driver for diabetes prediction, especially within the comorbid population. HbA1c is a well-established clinical marker for diabetes diagnosis. In our study, it is measured at patient entry, prior to disease onset, enabling us to assess its predictive value as a baseline risk factor.</p>
<p>To understand how HbA1c contributes to performance in this group and to assess its interaction with other predictors, we perform a post-hoc stratified analysis based on baseline HbA1c values. The cohort of 149,847 participants was divided into two subgroups: a low-normal subgroup (HbA1c &lt; 5.34%) and a high-normal subgroup (HbA1c <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pdig.0000982.e031"><math id="M31" display="inline" overflow="linebreak"><mrow><mo>≥</mo></mrow></math></span> 5.34%), where 5.34% is the mean HbA1c value in the whole population [<a href="#pdig.0000982.ref022" class="usa-link" aria-describedby="pdig.0000982.ref022">22</a>]. In this analysis, we evaluate model performance across different predictor sets. These include: (1) DiabetesUK, (2) XGB (7 core variables), (3) XGB (HbA1c only), (4) XGB (all variables except HbA1c), and (5) XGB (all variables). Each model is evaluated using 5-fold cross-validation, and AUC-ROC metrics are calculated for the HbA1c-defined subgroups.</p>
<p>To summarize the analytical approach and goals addressed in this study, we provide a workflow diagram (<a href="#pdig.0000982.g001" class="usa-link">Fig 1</a>). This figure outlines the main analytical components: subgroup performance analysis, characterization of performance and prediction model drivers, and post-hoc exploratory analyses. All implementation details, including code and library versions, are available at <a href="https://github.com/ngoc-vien-dang/ELF" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/ngoc-vien-dang/ELF</a>.</p>
<figure class="fig xbox font-sm" id="pdig.0000982.g001"><h4 class="obj_head">Fig 1. Overview of the analytical framework characterizing the contribution of early-life and proximal factors in ML-based risk prediction of psycho-cardio-metabolic multimorbidity.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12360575_pdig.0000982.g001.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c40c/12360575/c0dd1c502ae1/pdig.0000982.g001.jpg" loading="lazy" height="298" width="748" alt="Fig 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/pdig.0000982.g001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></figure></section></section><section id="sec005"><h2 class="pmc_sec_title">Results</h2>
<section id="sec006"><h3 class="pmc_sec_title">Predictive performance assessment</h3>
<section id="sec007"><h4 class="pmc_sec_title">Comparative performance of prediction models.</h4>
<p>The predictive performance of the different models, validated separately on non-comorbid and comorbid populations, is illustrated in <a href="#pdig.0000982.t001" class="usa-link">Table 1</a>. The baseline models, the Framingham score, DiabetesUK, and XGB (7 core variables), show reduced accuracy in the comorbid population, while the XGB (all variables) models exhibit enhanced performance for this population. In the non-comorbid group, the XGB (all variables) models also outperform the baselines in terms of AUC-ROC. The Brier score, shown in <a href="#pdig.0000982.t001" class="usa-link">Table 1</a> too, aligns with these results, indicating a consistently better performance of the XGB (all variables) models across both populations.</p>
<section class="tw xbox font-sm" id="pdig.0000982.t001"><h5 class="obj_head">Table 1. Comparison of models’ predictive performance for CVD and Diabetes with and without comorbid conditions.</h5>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<colgroup span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
</colgroup>
<thead>
<tr>
<th align="left" rowspan="1" colspan="1">Outcome</th>
<th align="left" rowspan="1" colspan="1">Model</th>
<th align="left" colspan="4" rowspan="1">Non-Comorbid</th>
<th align="left" colspan="4" rowspan="1">Comorbid</th>
</tr>
<tr>
<th align="left" rowspan="1" colspan="1"></th>
<th align="left" rowspan="1" colspan="1"></th>
<th align="left" colspan="2" rowspan="1">AUC-ROC</th>
<th align="left" colspan="2" rowspan="1">Brier score</th>
<th align="left" colspan="2" rowspan="1">AUC-ROC</th>
<th align="left" colspan="2" rowspan="1">Brier score</th>
</tr>
<tr>
<th align="left" rowspan="1" colspan="1"></th>
<th align="left" rowspan="1" colspan="1"></th>
<th align="left" rowspan="1" colspan="1">Value (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pdig.0000982.e032"><math id="M32" display="inline" overflow="linebreak"><mrow><mo>↑</mo></mrow></math></span>)</th>
<th align="left" rowspan="1" colspan="1">Change</th>
<th align="left" rowspan="1" colspan="1">Value (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pdig.0000982.e033"><math id="M33" display="inline" overflow="linebreak"><mrow><mo>↓</mo></mrow></math></span>)</th>
<th align="left" rowspan="1" colspan="1">Change</th>
<th align="left" rowspan="1" colspan="1">Value (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pdig.0000982.e034"><math id="M34" display="inline" overflow="linebreak"><mrow><mo>↑</mo></mrow></math></span>)</th>
<th align="left" rowspan="1" colspan="1">Change</th>
<th align="left" rowspan="1" colspan="1">Value (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pdig.0000982.e035"><math id="M35" display="inline" overflow="linebreak"><mrow><mo>↓</mo></mrow></math></span>)</th>
<th align="left" rowspan="1" colspan="1">Change</th>
</tr>
</thead>
<tbody>
<tr>
<td align="left" rowspan="1" colspan="1">CVD</td>
<td align="left" rowspan="1" colspan="1">Framingham score</td>
<td align="left" rowspan="1" colspan="1">0.70 ± 0.00</td>
<td align="left" rowspan="1" colspan="1">Baseline</td>
<td align="left" rowspan="1" colspan="1">0.08 ± 0.00</td>
<td align="left" rowspan="1" colspan="1">Baseline</td>
<td align="left" rowspan="1" colspan="1">0.55 ± 0.03</td>
<td align="left" rowspan="1" colspan="1">Baseline</td>
<td align="left" rowspan="1" colspan="1">0.22 ± 0.01</td>
<td align="left" rowspan="1" colspan="1">Baseline</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">XGB (7 core variables)</td>
<td align="left" rowspan="1" colspan="1">0.72 ± 0.00</td>
<td align="left" rowspan="1" colspan="1">+1.73%</td>
<td align="left" rowspan="1" colspan="1">0.07 ± 0.00</td>
<td align="left" rowspan="1" colspan="1">-1.48%</td>
<td align="left" rowspan="1" colspan="1">0.60 ± 0.02</td>
<td align="left" rowspan="1" colspan="1">+4.83%</td>
<td align="left" rowspan="1" colspan="1">0.14 ± 0.01</td>
<td align="left" rowspan="1" colspan="1">-7.36%</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">XGB (all variables)</td>
<td align="left" rowspan="1" colspan="1">0.73 ± 0.00</td>
<td align="left" rowspan="1" colspan="1">+2.8%</td>
<td align="left" rowspan="1" colspan="1">0.06 ± 0.00</td>
<td align="left" rowspan="1" colspan="1">-1.86%</td>
<td align="left" rowspan="1" colspan="1">0.63 ± 0.02</td>
<td align="left" rowspan="1" colspan="1">+7.9%</td>
<td align="left" rowspan="1" colspan="1">0.14 ± 0.01</td>
<td align="left" rowspan="1" colspan="1">-7.34%</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Diabetes</td>
<td align="left" rowspan="1" colspan="1">DiabetesUK</td>
<td align="left" rowspan="1" colspan="1">0.78 ± 0.01</td>
<td align="left" rowspan="1" colspan="1">Baseline</td>
<td align="left" rowspan="1" colspan="1">0.09 ± 0.00</td>
<td align="left" rowspan="1" colspan="1">Baseline</td>
<td align="left" rowspan="1" colspan="1">0.70 ± 0.01</td>
<td align="left" rowspan="1" colspan="1">Baseline</td>
<td align="left" rowspan="1" colspan="1">0.18 ± 0.01</td>
<td align="left" rowspan="1" colspan="1">Baseline</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">XGB (7 core variables)</td>
<td align="left" rowspan="1" colspan="1">0.79 ± 0.00</td>
<td align="left" rowspan="1" colspan="1">+0.92%</td>
<td align="left" rowspan="1" colspan="1">0.03 ± 0.01</td>
<td align="left" rowspan="1" colspan="1">-6.39%</td>
<td align="left" rowspan="1" colspan="1">0.73 ± 0.01</td>
<td align="left" rowspan="1" colspan="1">+3.07%</td>
<td align="left" rowspan="1" colspan="1">0.10 ± 0.01</td>
<td align="left" rowspan="1" colspan="1">-8.01%</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">XGB (all variables)</td>
<td align="left" rowspan="1" colspan="1">0.87 ± 0.00</td>
<td align="left" rowspan="1" colspan="1">+9.07%</td>
<td align="left" rowspan="1" colspan="1">0.03 ± 0.00</td>
<td align="left" rowspan="1" colspan="1">-6.57%</td>
<td align="left" rowspan="1" colspan="1">0.82 ± 0.03</td>
<td align="left" rowspan="1" colspan="1">+12.3%</td>
<td align="left" rowspan="1" colspan="1">0.09 ± 0.02</td>
<td align="left" rowspan="1" colspan="1">-8.68%</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/pdig.0000982.t001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><p>Note that the XGB (7 core variables) models still achieve an improvement over the Framingham score and DiabetesUK indicating that model selection can boost the performance of a model based on the 7 core features. For CVD prediction, the XGB (7 core variables) model improves the AUC-ROC by +1.73% and +4.83% over the Framingham score in the non-comorbid and comorbid groups, respectively. Similarly, for diabetes prediction, it improves the AUC-ROC by +0.92% and +3.07% over DiabetesUK in the non-comorbid and comorbid groups, respectively.</p></section><section id="sec008"><h4 class="pmc_sec_title">Comparative decision curve analysis of prediction models.</h4>
<p>Further augmenting our understanding, decision curve analysis in <a href="#pdig.0000982.g002" class="usa-link">Fig 2</a> compares the net benefit of our XGB models with that of the benchmark models (an XGB model with the 7 core variables, Framingham model for CVD, and DiabetesUK model for diabetes) in the comorbid population. The plots also include the behavior of two simplistic predictions: a dummy model that always predicts that all patients will develop the condition (’all’), and another one that always predicts that none patient will (’none’). The performance of benchmark models, closely aligned with the ’all’ strategy, suggests that they may overestimate risk, potentially leading to a high false positive rate in the comorbid population. The analysis shows that XGB (all variables) models provide greater net benefit than all three benchmark models across a broad range of risk thresholds for both CVD and diabetes prediction.</p>
<figure class="fig xbox font-sm" id="pdig.0000982.g002"><h5 class="obj_head">Fig 2. Decision curve analysis for comorbid groups.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12360575_pdig.0000982.g002.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c40c/12360575/f4d21649dd95/pdig.0000982.g002.jpg" loading="lazy" height="298" width="748" alt="Fig 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/pdig.0000982.g002/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Net benefit is provided for different models in (A) CVD prediction and (B) diabetes prediction. The different models include the corresponding benchmarks (blue) –Framingham Score for CVD in (A) and DiabetesUK in (B)–, an XGB model with all variables (green), an XGB model with the corresponding 7 core variables (cyan), and baseline ‘all’ (purple) and ‘none’ (orange) strategies</p></figcaption></figure><p>Specifically in CVD, the XGB (all variables) model begins to deliver a greater net benefit than both the ’all’ strategy and the Framingham score when the decision threshold exceeds 0.2, while the XGB (7 core variables) model follows closely. For diabetes, the XGB (all variables) model outperforms both the ’all’ strategy and DiabetesUK at a decision threshold above 0.1, and the XGB (7 core variables) model shows a similar trend. Additionally, the XGB (all variables) models for both CVD and diabetes prediction surpass the net benefit of the ’none’ strategy across all thresholds, confirming their effectiveness in predicting outcomes for comorbid groups. Next, we analyze the group-specific Bayes error estimates to contextualize the results from <a href="#pdig.0000982.t001" class="usa-link">Table 1</a>.</p></section></section><section id="sec009"><h3 class="pmc_sec_title">Group-specific Bayes errors</h3>
<p>Group-specific Bayes error estimates presented in <a href="#pdig.0000982.t002" class="usa-link">Table 2</a> for both CVD and diabetes prediction tasks across comorbid and non-comorbid populations indicate consistently higher noise estimates for comorbid groups than non-comorbid groups, utilizing two distinct subsets: 7 core variables and all variables. Particularly, nearest neighbor estimates provide non-overlapping intervals between non-comorbid and comorbid groups, and the one for comorbid subgroup is consistently higher. This suggests that the prediction task is more difficult among individuals from this population. Additionally, noise estimates for CVD are consistently higher relative to those for diabetes. These findings align with the performance differences of models shown in <a href="#pdig.0000982.t001" class="usa-link">Table 1</a>, which shows higher predictive performance for diabetes tasks than for CVD across both subgroups. The Bayes error estimates with the three techniques show similar trends. Estimations based on Bhatttacharyya’s method considering all the variables are not provided due to singular matrix errors during their computation.</p>
<section class="tw xbox font-sm" id="pdig.0000982.t002"><h4 class="obj_head">Table 2. Bayes error’s lower and upper bounds estimation (<em>E</em><sub><em>low</em></sub> and <em>E</em><sub><em>up</em></sub>) of comorbid and non-comorbid groups for CVD and diabetes prediction. Mahalanobis distance estimates provide only upper bounds; Some Bhattacharyya’s estimates are not available due to singular matrix errors.</h4>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<colgroup span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
</colgroup>
<thead>
<tr>
<th align="left" rowspan="1" colspan="1">Outcome</th>
<th align="left" rowspan="1" colspan="1">Method</th>
<th align="left" rowspan="1" colspan="1">Var. set</th>
<th align="left" colspan="2" rowspan="1">Non-Comorbid</th>
<th align="left" colspan="2" rowspan="1">Comorbid</th>
</tr>
<tr>
<th align="left" rowspan="1" colspan="1"></th>
<th align="left" rowspan="1" colspan="1"></th>
<th align="left" rowspan="1" colspan="1"></th>
<th align="left" rowspan="1" colspan="1">
<em>E</em><sub><em>low</em></sub> (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pdig.0000982.e036"><math id="M36" display="inline" overflow="linebreak"><mrow><mo>↓</mo></mrow></math></span>)</th>
<th align="left" rowspan="1" colspan="1">
<em>E</em><sub><em>up</em></sub> (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pdig.0000982.e037"><math id="M37" display="inline" overflow="linebreak"><mrow><mo>↓</mo></mrow></math></span>)</th>
<th align="left" rowspan="1" colspan="1">
<em>E</em><sub><em>low</em></sub> (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pdig.0000982.e038"><math id="M38" display="inline" overflow="linebreak"><mrow><mo>↓</mo></mrow></math></span>)</th>
<th align="left" rowspan="1" colspan="1">
<em>E</em><sub><em>up</em></sub> (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pdig.0000982.e039"><math id="M39" display="inline" overflow="linebreak"><mrow><mo>↓</mo></mrow></math></span>)</th>
</tr>
</thead>
<tbody>
<tr>
<td align="left" rowspan="1" colspan="1">CVD</td>
<td align="left" rowspan="1" colspan="1">Mahalanobis</td>
<td align="left" rowspan="1" colspan="1">7 core vars.</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">0.122571</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">0.282146</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">All vars.</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">
<span class="text-underline">0.121617</span>
</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">0.238344</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">Bhattacharyya</td>
<td align="left" rowspan="1" colspan="1">7 core vars.</td>
<td align="left" rowspan="1" colspan="1">0.053802</td>
<td align="left" rowspan="1" colspan="1">0.225627</td>
<td align="left" rowspan="1" colspan="1">0.152879</td>
<td align="left" rowspan="1" colspan="1">0.359871</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">All vars.</td>
<td align="left" rowspan="1" colspan="1">
<span class="text-underline">0.025346</span>
</td>
<td align="left" rowspan="1" colspan="1">
<span class="text-underline">0.157174</span>
</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">–</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">Nearest Neighbors</td>
<td align="left" rowspan="1" colspan="1">7 core vars.</td>
<td align="left" rowspan="1" colspan="1">0.038402</td>
<td align="left" rowspan="1" colspan="1">0.073854</td>
<td align="left" rowspan="1" colspan="1">0.110000</td>
<td align="left" rowspan="1" colspan="1">0.194059</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">All vars.</td>
<td align="left" rowspan="1" colspan="1">
<span class="text-underline">0.037158</span>
</td>
<td align="left" rowspan="1" colspan="1">
<span class="text-underline">0.071600</span>
</td>
<td align="left" rowspan="1" colspan="1">0.105107</td>
<td align="left" rowspan="1" colspan="1">0.188119</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Diabetes</td>
<td align="left" rowspan="1" colspan="1">Mahalanobis</td>
<td align="left" rowspan="1" colspan="1">7 core vars.</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">0.047010</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">0.194003</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">All vars.</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">
<span class="text-underline">0.043798</span>
</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">0.154115</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">Bhattacharyya</td>
<td align="left" rowspan="1" colspan="1">7 core vars.</td>
<td align="left" rowspan="1" colspan="1">0.015062</td>
<td align="left" rowspan="1" colspan="1">0.121801</td>
<td align="left" rowspan="1" colspan="1">0.076367</td>
<td align="left" rowspan="1" colspan="1">0.265584</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">All vars.</td>
<td align="left" rowspan="1" colspan="1">
<span class="text-underline">0.001459</span>
</td>
<td align="left" rowspan="1" colspan="1">
<span class="text-underline">0.038166</span>
</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">–</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">Nearest Neighbors</td>
<td align="left" rowspan="1" colspan="1">7 core vars.</td>
<td align="left" rowspan="1" colspan="1">0.013230</td>
<td align="left" rowspan="1" colspan="1">0.026110</td>
<td align="left" rowspan="1" colspan="1">0.070000</td>
<td align="left" rowspan="1" colspan="1">0.127923</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">All vars.</td>
<td align="left" rowspan="1" colspan="1">
<span class="text-underline">0.012713</span>
</td>
<td align="left" rowspan="1" colspan="1">
<span class="text-underline">0.025102</span>
</td>
<td align="left" rowspan="1" colspan="1">0.063133</td>
<td align="left" rowspan="1" colspan="1">0.118294</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/pdig.0000982.t002/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><p>The demonstrated efficacy of the XGB (all variables) models lays the groundwork for employing post-hoc approaches, previously outlined, for variable ranking.</p></section><section id="sec010"><h3 class="pmc_sec_title">Key drivers of model performance</h3>
<section id="sec011"><h4 class="pmc_sec_title">Predictive power of individual variables in comorbid and non-comorbid groups.</h4>
<p>The analysis of the <em>predictive power of individual variables</em> is summarized in a scatter plot (<a href="#pdig.0000982.g003" class="usa-link">Fig 3</a>) comparing the individual predictive ability of the variables for comorbid versus non-comorbid groups, separately for CVD and diabetes. To improve visual interpretability and enable consistent comparison, the horizontal and vertical ranges are delimited to a lower bound based on the minimum value observed among ELF variables and an upper bound at the 99th percentile of the distribution of values from both conditions. Each point represents a variable and it is located according to the performance of a model learned exclusively with this predictive variable on the C and NC groups (x and y axis, resp.). The diagonal line represents single-variable models that show the same performance on both subgroups. For both conditions, variables tend to have a higher predictive ability in the non-comorbid group, as most points concentrate above the line of similar effect. In CVD prediction, <em>blood assays</em> and <em>sociodemographic</em> factors exhibit the highest predictive ability for both non-comorbid and comorbid groups but are more predictive for non-comorbid groups. In diabetes prediction, <em>blood assays</em>, along with <em>physical measures and activities</em>, display a similar pattern, demonstrating stronger predictive ability in non-comorbid groups. The complete distribution of AUC-ROC values for all the features is available in <a href="#pdig.0000982.s008" class="usa-link">S4 Fig</a>.</p>
<figure class="fig xbox font-sm" id="pdig.0000982.g003"><h5 class="obj_head">Fig 3. Predictive ability of individual variables for (A) CVD and (B) diabetes prediction.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12360575_pdig.0000982.g003.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c40c/12360575/4d92dd53a434/pdig.0000982.g003.jpg" loading="lazy" height="373" width="750" alt="Fig 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/pdig.0000982.g003/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Each plot compares AUC-ROC performance in the comorbid group (x-axis) vs performance in the non-comorbid (y-axis) group. Each point represents an XGB model learned with a single feature. Colors differentiate models learned with a variable of different type. The star marker indicates models learned with a variable used by the benchmarks. The dotted grey line indicates equal performance in both groups. Deviation from this line allows for identifying variables that have a stronger predictive impact on one group over the other.</p></figcaption></figure><p>The distribution of ELFs varies, with some positioned above or near the line of similar effect, reflecting a mixed predictive influence on both comorbid and non-comorbid groups. However, a significant number of ELFs, notably <em>breastfed as a baby</em> (AUC-ROC: 0.54 for comorbid, 0.51 for non-comorbid) for CVD and <em>felt loved as a child</em> (AUC-ROC: 0.55 for comorbid, 0.52 for non-comorbid) for diabetes are located below the line, suggesting their particular relevance and contribution to the predictive models for comorbid conditions. These ELFs, along with other variables positioned below the line, could explain the enhanced performance of the XGB models for these groups as detailed in <a href="#pdig.0000982.t001" class="usa-link">Table 1</a>. Note that the AUC-ROC values of these ELFs slightly exceed 0.5, suggesting that their individual utility in the long-term prediction of CVD and diabetes, while present, is limited. However, their higher relevance in the comorbid group suggests they provide added value for risk stratification within this population. Additionally, any factor with modest individual AUC-ROC values can still be useful for ML models when combined with other features, especially in complex populations like comorbid groups.</p></section><section id="sec012"><h4 class="pmc_sec_title">PFI analysis of predictive variables in comorbid and non-comorbid groups.</h4>
<p>The <em>PFI</em> analysis underscores the predictive ability of age for CVD and HbA1c for diabetes. <a href="#pdig.0000982.t003" class="usa-link">Table 3</a> shows results for the comorbid group, whereas details on the non-comorbid group can be found in <a href="#pdig.0000982.s003" class="usa-link">S2 Table</a> and <a href="#pdig.0000982.s005" class="usa-link">S3 Table</a>. These findings align with the results of previous works [<a href="#pdig.0000982.ref013" class="usa-link" aria-describedby="pdig.0000982.ref013">13</a>, <a href="#pdig.0000982.ref018" class="usa-link" aria-describedby="pdig.0000982.ref018">18</a>]. Our analysis reveals that while there is a notable overlap in the most relevant predictors of CVD and diabetes in both non-comorbid and comorbid groups, with 60% of the top 30 variables being common to both populations, there are specific variables that distinctly influence the comorbid group. Traditional risk factors, such as <em>smoking status</em> in the case of CVD and <em>waist size</em> for diabetes, maintain their predictive validity in the non-comorbid group; however, their predictive power diminishes in the comorbid group for both conditions. Furthermore, <em>self-reported overall health rating</em>, which is not usually considered in existing risk prediction models, and clinical measures such as <em>metabolic factors</em>, show more predictive power for individuals with comorbid conditions. This variation aligns with the findings in <a href="#pdig.0000982.t001" class="usa-link">Table 1</a>, where the XGB models learned with all the variables demonstrate superior performance over benchmark models, especially for the comorbid group.</p>
<section class="tw xbox font-sm" id="pdig.0000982.t003"><h5 class="obj_head">Table 3. Permutation-based variable ranking for predicting CVD and diabetes in comorbid groups.</h5>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<colgroup span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
</colgroup>
<thead><tr>
<th align="left" rowspan="1" colspan="1">Variable (CVD prediction)</th>
<th align="left" rowspan="1" colspan="1">Score</th>
<th align="left" rowspan="1" colspan="1">Variable (Diabetes prediction)</th>
<th align="left" rowspan="1" colspan="1">Score</th>
</tr></thead>
<tbody>
<tr>
<td align="left" rowspan="1" colspan="1">Age<sup>*†</sup>
</td>
<td align="left" rowspan="1" colspan="1">0.056253</td>
<td align="left" rowspan="1" colspan="1">HbA1c<sup>†</sup>
</td>
<td align="left" rowspan="1" colspan="1">0.119222</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">OverallHealthRating<sup>†</sup>
</td>
<td align="left" rowspan="1" colspan="1">0.016420</td>
<td align="left" rowspan="1" colspan="1">WaistCircumference<sup>*†</sup>
</td>
<td align="left" rowspan="1" colspan="1">0.023598</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Glucose<sup>†</sup>
</td>
<td align="left" rowspan="1" colspan="1">0.007265</td>
<td align="left" rowspan="1" colspan="1">OverallHealthRating<sup>†</sup>
</td>
<td align="left" rowspan="1" colspan="1">0.010247</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">CurrentEmploymentStatus_Retired</td>
<td align="left" rowspan="1" colspan="1">0.007114</td>
<td align="left" rowspan="1" colspan="1">Triglycerides<sup>†</sup>
</td>
<td align="left" rowspan="1" colspan="1">0.006574</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">WaistCircumference<sup>†</sup>
</td>
<td align="left" rowspan="1" colspan="1">0.006805</td>
<td align="left" rowspan="1" colspan="1">HDLCholesterol<sup>†</sup>
</td>
<td align="left" rowspan="1" colspan="1">0.005897</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">HbA1c<sup>†</sup>
</td>
<td align="left" rowspan="1" colspan="1">0.006483</td>
<td align="left" rowspan="1" colspan="1">Glucose<sup>†</sup>
</td>
<td align="left" rowspan="1" colspan="1">0.003622</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">CurrentEmploymentStatus_Paid/SelfEmployed<sup>†</sup>
</td>
<td align="left" rowspan="1" colspan="1">0.003173</td>
<td align="left" rowspan="1" colspan="1">Hypertension<sup>*†</sup>
</td>
<td align="left" rowspan="1" colspan="1">0.003426</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">LDLDirect<sup>†</sup>
</td>
<td align="left" rowspan="1" colspan="1">0.003124</td>
<td align="left" rowspan="1" colspan="1">Sex<sup>†</sup>
</td>
<td align="left" rowspan="1" colspan="1">0.003144</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">IllnessesOfMother_CVD<sup>†</sup>
</td>
<td align="left" rowspan="1" colspan="1">0.002986</td>
<td align="left" rowspan="1" colspan="1">BodyFatPercentage<sup>†</sup>
</td>
<td align="left" rowspan="1" colspan="1">0.002629</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">SmokingStatus<sup>*†</sup>
</td>
<td align="left" rowspan="1" colspan="1">0.002780</td>
<td align="left" rowspan="1" colspan="1">Age<sup>*†</sup>
</td>
<td align="left" rowspan="1" colspan="1">0.001551</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">DiastolicBloodPressure<sup>†</sup>
</td>
<td align="left" rowspan="1" colspan="1">0.002697</td>
<td align="left" rowspan="1" colspan="1">CurrentEmploymentStatus_Sick/Disabled</td>
<td align="left" rowspan="1" colspan="1">0.001472</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Depression</td>
<td align="left" rowspan="1" colspan="1">0.002218</td>
<td align="left" rowspan="1" colspan="1">Qualifications_UnivDegree<sup>†</sup>
</td>
<td align="left" rowspan="1" colspan="1">0.001422</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">CurrentEmploymentStatus_Sick/Disabled<sup>†</sup>
</td>
<td align="left" rowspan="1" colspan="1">0.002155</td>
<td align="left" rowspan="1" colspan="1">BMI<sup>*†</sup>
</td>
<td align="left" rowspan="1" colspan="1">0.001389</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">PorkIntake</td>
<td align="left" rowspan="1" colspan="1">0.001957</td>
<td align="left" rowspan="1" colspan="1">
<span class="text-underline">SomeoneToTakeToDoctorWhenNeededAsAChild</span>
</td>
<td align="left" rowspan="1" colspan="1">0.000909</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">CReactiveProtein<sup>†</sup>
</td>
<td align="left" rowspan="1" colspan="1">0.002057</td>
<td align="left" rowspan="1" colspan="1">CerealIntake</td>
<td align="left" rowspan="1" colspan="1">0.000909</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<span class="text-underline">PhysicallyAbusedByFamilyAsAChild</span>
</td>
<td align="left" rowspan="1" colspan="1">0.002031</td>
<td align="left" rowspan="1" colspan="1">
<span class="text-underline">FeltHatedByFamilyMemberAsAChild</span>
</td>
<td align="left" rowspan="1" colspan="1">0.000865</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Cholesterol<sup>†</sup>
</td>
<td align="left" rowspan="1" colspan="1">0.001815</td>
<td align="left" rowspan="1" colspan="1">SleepDuration</td>
<td align="left" rowspan="1" colspan="1">0.000858</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">ApolipoproteinA<sup>†</sup>
</td>
<td align="left" rowspan="1" colspan="1">0.001792</td>
<td align="left" rowspan="1" colspan="1">AvgHouseholdIncome<sup>†</sup>
</td>
<td align="left" rowspan="1" colspan="1">0.000729</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">BodyFatPercentage<sup>†</sup>
</td>
<td align="left" rowspan="1" colspan="1">0.001593</td>
<td align="left" rowspan="1" colspan="1">RawVegetableIntake</td>
<td align="left" rowspan="1" colspan="1">0.000705</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<span class="text-underline">MaternalSmokingAroundBirth</span>
</td>
<td align="left" rowspan="1" colspan="1">0.001591</td>
<td align="left" rowspan="1" colspan="1">ProcessedMeatIntake<sup>†</sup>
</td>
<td align="left" rowspan="1" colspan="1">0.000691</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">SaltAddedToFood</td>
<td align="left" rowspan="1" colspan="1">0.001431</td>
<td align="left" rowspan="1" colspan="1">Depression</td>
<td align="left" rowspan="1" colspan="1">0.000648</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">SleepDuration</td>
<td align="left" rowspan="1" colspan="1">0.001137</td>
<td align="left" rowspan="1" colspan="1">Qualifications_NoneAbove</td>
<td align="left" rowspan="1" colspan="1">0.000625</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">AlcoholIntakeFrequency</td>
<td align="left" rowspan="1" colspan="1">0.000868</td>
<td align="left" rowspan="1" colspan="1">IllnessesOfMother_Diabetes<sup>*†</sup>
</td>
<td align="left" rowspan="1" colspan="1">0.000591</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">WaterIntake</td>
<td align="left" rowspan="1" colspan="1">0.000839</td>
<td align="left" rowspan="1" colspan="1">IllnessesOfSiblings_Diabetes<sup>*†</sup>
</td>
<td align="left" rowspan="1" colspan="1">0.000532</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<span class="text-underline">FeltHatedByFamilyMemberAsAChild</span>
<sup>†</sup>
</td>
<td align="left" rowspan="1" colspan="1">0.000759</td>
<td align="left" rowspan="1" colspan="1">CReactiveProtein<sup>†</sup>
</td>
<td align="left" rowspan="1" colspan="1">0.000442</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<span class="text-underline">BreastfedAsABaby</span>
</td>
<td align="left" rowspan="1" colspan="1">0.000708</td>
<td align="left" rowspan="1" colspan="1">IllnessesOfFather_CVD</td>
<td align="left" rowspan="1" colspan="1">0.000434</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Qualifications_UnivDegree<sup>†</sup>
</td>
<td align="left" rowspan="1" colspan="1">0.000623</td>
<td align="left" rowspan="1" colspan="1">
<span class="text-underline">FeltLovedAsAChild</span>
</td>
<td align="left" rowspan="1" colspan="1">0.000416</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">HDLCholesterol<sup>†</sup>
</td>
<td align="left" rowspan="1" colspan="1">0.000580</td>
<td align="left" rowspan="1" colspan="1">AlcoholIntakeFrequency<sup>†</sup>
</td>
<td align="left" rowspan="1" colspan="1">0.000394</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">MoodSwings</td>
<td align="left" rowspan="1" colspan="1">0.000530</td>
<td align="left" rowspan="1" colspan="1">PoultryIntake</td>
<td align="left" rowspan="1" colspan="1">0.000387</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Qualifications_ProfQual(Nurse/Teach)</td>
<td align="left" rowspan="1" colspan="1">0.000517</td>
<td align="left" rowspan="1" colspan="1">IllnessesOfMother_CVD</td>
<td align="left" rowspan="1" colspan="1">0.000347</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/pdig.0000982.t003/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="t003fn001"><p>* Variables used by the benchmarks.<sup>†</sup> Shared variables between comorbid and non-comorbid groups. Underlined variables indicate ELFs.</p></div></div></section><p>As shown in <a href="#pdig.0000982.s003" class="usa-link">S2 Table</a> and <a href="#pdig.0000982.s005" class="usa-link">S3 Table</a>, the analysis indicates an elevated prevalence and ranking of ELFs in the comorbid group for both CVD and diabetes, indicating their interdependent contribution to the model’s overall predictive performance in patients with multiple health issues. Specifically, in CVD prediction, 4 ELFs rank in the top 30 for comorbid groups among 83 features, versus 2 in non-comorbid groups (with lower placement), and one overlapping ELF between groups. For diabetes, 3 ELFs make the top 30 in comorbid groups, against a single, lower-ranked ELF in non-comorbid groups. This finding is consistent with the detailed predictive ability of individual variables analysis presented in <a href="#pdig.0000982.g003" class="usa-link">Fig 3</a>, confirming the significant role of ELFs in comorbid scenarios.</p></section></section><section id="sec013"><h3 class="pmc_sec_title">Key drivers of model prediction</h3>
<p>In this section, we analyze the variables central to predicting the risk of CVD and diabetes, focusing on the contrasting profiles between non-comorbid and comorbid groups. This analysis provides detailed explanations of what led to a patient’s predicted risk, as determined by ML-based models.</p>
<section id="sec014"><h4 class="pmc_sec_title">SHAP value analysis of predictive variables in comorbid and non-comorbid groups.</h4>
<p><strong><em>CVD prediction.</em></strong> For those with comorbid conditions, as shown in <a href="#pdig.0000982.g004" class="usa-link">Fig 4</a>, <em>age</em>, <em>sex</em>, and the presence of <em>diabetes</em> are the most significant variables for predicting the risk of CVD, indicating a higher risk profile in these patients. Additionally, metabolic health measures such as <em>waist circumference</em> and <em>LDL cholesterol</em> play significant roles, underscoring that the learning method has assigned them a large influence in the model predictions, which is more notorious in the case of comorbidities. Notably, psychosocial experiences, captured by variables reflecting <em>childhood abuse</em>, <em>feelings of isolation</em>, and <em>depressive symptoms</em>, are identified as powerful predictors. This would support the integration of psychological health considerations into a comprehensive CVD risk assessment, especially for patients managing coexisting conditions. In the non-comorbid cohort, our <em>SHAP value</em> analysis for CVD prediction, detailed in <a href="#pdig.0000982.s010" class="usa-link">S5 Fig</a>, underscores the impact of personal lifestyle factors, including <em>alcohol intake frequency</em> and <em>smoking status</em>, as well as <em>social engagement</em>, <em>overall health ratings</em>, and <em>body fat percentage</em>, as significant drivers of risk prediction. Traditional risk factors such as <em>age</em> and <em>blood pressure</em> retain their expected prominence. These results emphasize the importance of lifestyle choices in developing CVD risk for individuals without additional health conditions.</p>
<figure class="fig xbox font-sm" id="pdig.0000982.g004"><h5 class="obj_head">Fig 4. Averaged SHAP scores for the top 30 influential variables in the prediction models for CVD and diabetes within comorbid patient groups.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12360575_pdig.0000982.g004.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c40c/12360575/45f983c27fd5/pdig.0000982.g004.jpg" loading="lazy" height="360" width="748" alt="Fig 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/pdig.0000982.g004/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Core variables used by the benchmark models are marked with an asterisk (*). Coloured bars highlight ELFs.</p></figcaption></figure><p><strong><em>Diabetes prediction.</em></strong> In the context of diabetes prediction, the analysis presented in <a href="#pdig.0000982.g004" class="usa-link">Fig 4</a> for the comorbid group underscores an amplified significance of metabolic factors, with <em>waist circumference</em> and <em>HbA1c</em> demonstrating higher SHAP scores. These findings align with established clinical understanding, demonstrating that the model has accurately captured key risk factors for comorbid patients in a clinically relevant manner. This trend points to the significant role of metabolic dysregulation in diabetes risk in individuals with multiple health challenges. Additionally, the presence of CVD is identified as a key predictor. The SHAP scores also highlight the critical role of psychosocial factors, particularly <em>depression</em>, indicating the evident interplay between mental and physical health in diabetes management for comorbid patients. Furthermore, ELFs like <em>having someone to take to the doctor when needed as a child</em> and <em>being breastfed as a baby</em> are found to have enduring impacts on health outcomes. These findings advocate for a diabetes risk model that integrates a diverse range of physiological, psychological, and ELFs, particularly pertinent in the context of comorbidity.</p>
<p>For individuals without comorbidities, <em>glucose levels</em> unsurprisingly dominate as a key factor, followed by traditional clinical markers such as <em>HbA1c</em> and <em>waist circumference</em>, reflecting their critical roles in diabetes risk prediction. Beyond these, lifestyle habits including <em>smoking status</em> and dietary patterns, particularly the consumption of <em>cooked vegetables</em> and <em>fresh fruit</em>, are identified as influential, which underscores the importance of day-to-day health behaviors in diabetes risk prediction in the comorbid population. Note that the referred dietary factors included in this study reflect adult dietary behaviors, rather than early-life dietary exposures.</p>
<p>Poultry intake also ranked as a prominent risk predictor for diabetes, placing 10th in our SHAP analysis. This result aligns with recent evidence from a multi-regional study, which observed a positive association between poultry intake and diabetes risk, particularly in European populations, including the UK [<a href="#pdig.0000982.ref038" class="usa-link" aria-describedby="pdig.0000982.ref038">38</a>]. However, the association was less consistent in regions such as South Asia and the Eastern Mediterranean. Given poultry’s classification as a healthier alternative to red meat, these results suggest the need for further investigation into its role in diabetes risk prediction. Notably, ELFs such as feeling loved during childhood, although less dominant, still appear in the analysis, suggesting that the learning technique is uncovering a subtle yet present impact on long-term health outcomes. However, it is crucial to acknowledge that these findings, particularly from self-reported ELFs like childhood maltreatment, may be influenced by reporting biases, especially in individuals with mental health issues who may be more likely to negatively appraise their life histories, potentially influencing associations between these factors and health outcomes. For a detailed analysis of the non-comorbid cohort, see <a href="#pdig.0000982.s011" class="usa-link">S6 Fig</a>.</p></section><section id="sec015"><h4 class="pmc_sec_title">Dual-group risk effect size analysis of predictive variables.</h4>
<p>In the assessment of explanatory variables using the <em>dual-group risk effect size analyzer</em>, <a href="#pdig.0000982.g005" class="usa-link">Fig 5</a> shows a substantial overlap of approximately 60% in the top-30 variables between non-comorbid and comorbid groups for both CVD and diabetes. <em>Age</em> and <em>HbA1c</em> are consistently the top-ranking variables for these groups in CVD and diabetes, respectively (see <a href="#pdig.0000982.s007" class="usa-link">S4 Table</a> and <a href="#pdig.0000982.s009" class="usa-link">S5 Table</a> for a list of variables ordered by their effect size). In the comorbid group, ELFs as well as psychosocial factors such as <em>mood swings</em> and <em>irritability</em>, which are generally underrepresented in traditional risk models, display a significant effect size for both CVD and diabetes prediction. This may be explained by the possibility that these indicators reflect underlying depression severity, influencing the likelihood of developing comorbid conditions like CVD and diabetes. This indicates that our models rely on emotional well-being and formative childhood experiences for disease risk prediction among these individuals. The <em>dual-group risk effect size</em> analysis thus complements the <em>SHAP value</em> analysis by providing additional insights into how these variables, especially ELFs, distinctly influence risk stratification in comorbid scenarios.</p>
<figure class="fig xbox font-sm" id="pdig.0000982.g005"><h5 class="obj_head">Fig 5. Differential top features by effect size: comorbid vs. non-comorbid.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12360575_pdig.0000982.g005.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c40c/12360575/66cbf16c5cd2/pdig.0000982.g005.jpg" loading="lazy" height="546" width="748" alt="Fig 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/pdig.0000982.g005/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>The top 30 features are identified by their effect size for (A) CVD and (B) Diabetes predictive models. All features are indicated by blue dots, and red dots specifically highlight ELFs. Variables marked with an asterisk (*) are used in benchmarks.</p></figcaption></figure><p>Expanding upon these findings, we observe that the association between ELFs and PCM multimorbidity can be modified by lifestyle factors. This association seems to be positively moderated by health-promoting behaviors such as increased intake of <em>fresh fruits</em>, and both <em>raw and cooked vegetables</em>, alongside engagement in <em>leisure social activities</em> and consistent <em>physical activity</em>. Conversely, the association is exacerbated by negative lifestyle habits, including <em>smoking</em> and <em>alcohol consumption</em>, along with psychological factors like <em>mood swings</em> and <em>feelings of loneliness</em>. A causal analysis, which is beyond the scope of this work, would be required to find causal relationships between these behaviors and the respective risk outcomes.</p></section></section><section id="sec016"><h3 class="pmc_sec_title">HbA1c’s role in diabetes prediction for the comorbid cohort</h3>
<p>As outlined in Section Materials and methods, a post-hoc stratified analysis was performed to evaluate the role of HbA1c in diabetes prediction within the comorbid population, following its identification as a dominant performance driver (<a href="#pdig.0000982.t003" class="usa-link">Table 3</a>). As shown in <a href="#pdig.0000982.t004" class="usa-link">Table 4</a>, the predictive performance of XGB models for the comorbid cohort varies across HbA1c-defined subgroups. In the low-normal HbA1c group, XGB models trained on 7 core variables or non-laboratory variables achieve high predictive accuracy, demonstrating the model’s ability to predict diabetes risk without relying on HbA1c. By contrast, the XGB model trained only on HbA1c shows limited predictive value for this group. In the high-normal HbA1c group, HbA1c plays a stronger role, with its inclusion improving the model’s performance compared to using all predictors except HbA1c. The XGB model trained only with HbA1c also outperforms the benchmarks (XGB trained on 7 core variables and DiabetesUK). However, HbA1c is not the sole performance driver in this group. The XGB model trained only on HbA1c achieves lower predictive performance than the model trained on all predictors. Additionally, the model trained on all predictors except HbA1c demonstrates competitive performance. These results underscore HbA1c’s critical role as a performance driver, particularly in the high-normal group, while highlighting the importance of integrating diverse predictors to achieve robust performance across the whole population of comorbid patients.</p>
<section class="tw xbox font-sm" id="pdig.0000982.t004"><h4 class="obj_head">Table 4. Comparison of models’ predictive performance (AUC-ROC) for diabetes risk in HbA1c-defined subgroups within the comorbid group.</h4>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<colgroup span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
</colgroup>
<thead><tr>
<th align="left" rowspan="1" colspan="1">Method</th>
<th align="left" rowspan="1" colspan="1">HbA1c &lt; 5.34%</th>
<th align="left" rowspan="1" colspan="1">HbA1c <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pdig.0000982.e077"><math id="M77" display="inline" overflow="linebreak"><mrow><mo>≥</mo></mrow></math></span> 5.34%</th>
</tr></thead>
<tbody>
<tr>
<td align="left" rowspan="1" colspan="1">DiabetesUK</td>
<td align="left" rowspan="1" colspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pdig.0000982.e078">
<math id="M78" display="inline" overflow="linebreak"><mrow><mn>0.83</mn><mi>±</mi><mn>0.05</mn></mrow></math>
</span>
</td>
<td align="left" rowspan="1" colspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pdig.0000982.e079">
<math id="M79" display="inline" overflow="linebreak"><mrow><mn>0.66</mn><mi>±</mi><mn>0.02</mn></mrow></math>
</span>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">XGB (7 core variables)</td>
<td align="left" rowspan="1" colspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pdig.0000982.e080">
<math id="M80" display="inline" overflow="linebreak"><mrow><mn>0.88</mn><mi>±</mi><mn>0.06</mn></mrow></math>
</span>
</td>
<td align="left" rowspan="1" colspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pdig.0000982.e081">
<math id="M81" display="inline" overflow="linebreak"><mrow><mn>0.70</mn><mi>±</mi><mn>0.02</mn></mrow></math>
</span>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">XGB (HbA1c)</td>
<td align="left" rowspan="1" colspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pdig.0000982.e082">
<math id="M82" display="inline" overflow="linebreak"><mrow><mn>0.51</mn><mi>±</mi><mn>0.10</mn></mrow></math>
</span>
</td>
<td align="left" rowspan="1" colspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pdig.0000982.e083">
<math id="M83" display="inline" overflow="linebreak"><mrow><mn>0.75</mn><mi>±</mi><mn>0.04</mn></mrow></math>
</span>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">XGB (all variables - HbA1c)</td>
<td align="left" rowspan="1" colspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pdig.0000982.e084">
<math id="M84" display="inline" overflow="linebreak"><mrow><mn>0.87</mn><mi>±</mi><mn>0.04</mn></mrow></math>
</span>
</td>
<td align="left" rowspan="1" colspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pdig.0000982.e085">
<math id="M85" display="inline" overflow="linebreak"><mrow><mn>0.74</mn><mi>±</mi><mn>0.02</mn></mrow></math>
</span>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">XGB (all variables)</td>
<td align="left" rowspan="1" colspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pdig.0000982.e086">
<math id="M86" display="inline" overflow="linebreak"><mrow><mn>0.87</mn><mi>±</mi><mn>0.05</mn></mrow></math>
</span>
</td>
<td align="left" rowspan="1" colspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pdig.0000982.e087">
<math id="M87" display="inline" overflow="linebreak"><mrow><mn>0.81</mn><mi>±</mi><mn>0.04</mn></mrow></math>
</span>
</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/pdig.0000982.t004/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><p><a href="#pdig.0000982.g006" class="usa-link">Fig 6</a> displays the feature importances as performance drivers in the XGB model with all the features across HbA1c-defined subgroups. In the high-normal group, HbA1c ranks as the top feature by FPI score, while in the low-normal group, it ranks second, with waist circumference as the leading contributor. The relative differences in variable scores suggest that the model relies on many predictors other than HbA1c to maintain performance across both subgroups.</p>
<figure class="fig xbox font-sm" id="pdig.0000982.g006"><h4 class="obj_head">Fig 6. Top 30 features for XGB (all variables) by FPI score in HbA1c-defined cohorts within the comorbid group.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12360575_pdig.0000982.g006.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c40c/12360575/6bebd478a0f7/pdig.0000982.g006.jpg" loading="lazy" height="360" width="748" alt="Fig 6"></a></p>
<div class="p text-right font-secondary"><a href="figure/pdig.0000982.g006/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Core variables used by the benchmark models are marked with an asterisk (*). Green bars highlight ELFs; orange bars indicate HbA1c. Note that PFI scores are log-transformed to enhance the visibility of smaller values while maintaining relative differences.</p></figcaption></figure><p>The role of HbA1c as a prediction driver is slightly different, as shown in <a href="#pdig.0000982.g007" class="usa-link">Fig 7</a>. In the high-normal HbA1c group, HbA1c ranks as the top feature, with the largest effect size among all predictors, reflecting its prominent role in shaping individual-level predictions. In the low-normal HbA1c group, HbA1c ranks sixth, with predictors such as waist circumference, BMI, and overall health rating ranking higher. Notably, laboratory-based features are key prediction drivers in the high-normal group, while ELFs consistently feature among the top predictors in both subgroups, reflecting their long-term impact on diabetes risk. These results emphasize the importance of using clinical laboratory markers and early-life information to support risk stratification and preventive strategies in the comorbid group.</p>
<figure class="fig xbox font-sm" id="pdig.0000982.g007"><h4 class="obj_head">Fig 7. Top 30 features for XGB (all variables) by risk effect size in HbA1c-defined cohorts within the comorbid group.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12360575_pdig.0000982.g007.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c40c/12360575/137aaaf8f5de/pdig.0000982.g007.jpg" loading="lazy" height="361" width="750" alt="Fig 7"></a></p>
<div class="p text-right font-secondary"><a href="figure/pdig.0000982.g007/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Core variables used by the benchmark models are marked with an asterisk (*). Green bars highlight ELFs; orange bars indicate HbA1c.</p></figcaption></figure></section><section id="sec017"><h3 class="pmc_sec_title">ELF contribution for diabetes prediction with smaller predictor sets</h3>
<p>Diabetes prediction can achieve high accuracy with a minimal number of predictors, as few as four biomarkers, illustrating a balance between prediction accuracy and the cost of acquiring additional variables in clinical practice [<a href="#pdig.0000982.ref039" class="usa-link" aria-describedby="pdig.0000982.ref039">39</a>–<a href="#pdig.0000982.ref041" class="usa-link" aria-describedby="pdig.0000982.ref041">41</a>]. In this line, we designed an experiment aimed at identifying a compact set of predictors that preserves high predictive performance and evaluates ELF contributions within this set. We start with the seven core UKDiabetes’ features plus all the ELFs. Incrementally, we add other features to the model based on their PFI scores in the high-normal HbA1c group. This strategy allowed us to assess how the inclusion of each variable influenced the model’s predictive performance.</p>
<p>We observe that the AUC-ROC improved rapidly with the first 20 variables, which include the 7 core variables and ELFs, but plateaued after the inclusion of approximately 25 predictors. Beyond this point, additional features provided only marginal gains. This result suggests that a focused subset of around 25 predictors achieves an optimal balance between predictive utility and simplicity, leading to further evaluation of ELF contributions in this model. This 25-feature model achieves an AUC-ROC of 0.90 ± 0.06 (95% CI) for the non-comorbid group and 0.82 ± 0.03 (95% CI) for the comorbid group. These results are slightly better than those achieved by the XGB (all variables) model in <a href="#pdig.0000982.t001" class="usa-link">Table 1</a>, demonstrating its ability to balance predictive accuracy and simplicity effectively.</p>
<p>We also examine ELF contributions in the absence of lab-based predictors. This analysis aims to assess the relative importance of ELFs when costly laboratory features are excluded, relying only on easy-to-collect variables. Thus, we learned an XGB model only with non-laboratory variables and it achieved an AUC-ROC of 0.89 ± 0.07 (95% CI) for the non-comorbid group and 0.73 ± 0.03 (95% CI) for the comorbid group.</p>
<p>ELF contributions consistently increase in both the top 25-variable model and the non-laboratory variable model relative to the all-variable model, as summarized in <a href="#pdig.0000982.t005" class="usa-link">Table 5</a>. Notably, SHAP analysis shows that ELF contributions rise from 6.97% in the all-variable model to 16.70% in the top 25-variable model, representing an absolute gain of 9.74 percentage points (a 139.77% relative increase). The non-laboratory variable model also exhibits higher ELF contributions, reinforcing their role as key predictors in the comorbid population when laboratory biomarkers are not available.</p>
<section class="tw xbox font-sm" id="pdig.0000982.t005"><h4 class="obj_head">Table 5. ELF accumulated contributions, measured by PFI, SHAP, and effect size, for the comorbid group in XGB models learned with all variables, non-laboratory variables, and the top 25 variables. Normalized values ensure comparability across models. Gain refers to the absolute and relative increases in ELF contributions compared to the all-variable baseline model. Absolute gain is expressed as the percentage point difference, while relative gain (in parentheses) is calculated as the percentage increase relative to the baseline value.</h4>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<colgroup span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
</colgroup>
<thead>
<tr>
<th align="left" rowspan="1" colspan="1">Model</th>
<th align="left" colspan="2" rowspan="1">PFI</th>
<th align="left" colspan="2" rowspan="1">SHAP</th>
<th align="left" colspan="2" rowspan="1">Effect size</th>
</tr>
<tr>
<th align="left" rowspan="1" colspan="1"></th>
<th align="left" rowspan="1" colspan="1">Value (%)</th>
<th align="left" rowspan="1" colspan="1">Gain</th>
<th align="left" rowspan="1" colspan="1">Value (%)</th>
<th align="left" rowspan="1" colspan="1">Gain</th>
<th align="left" rowspan="1" colspan="1">Value (%)</th>
<th align="left" rowspan="1" colspan="1">Gain</th>
</tr>
</thead>
<tbody>
<tr>
<td align="left" rowspan="1" colspan="1">XGB (all vars)</td>
<td align="left" rowspan="1" colspan="1">0.45</td>
<td align="left" rowspan="1" colspan="1">Baseline</td>
<td align="left" rowspan="1" colspan="1">6.97</td>
<td align="left" rowspan="1" colspan="1">Baseline</td>
<td align="left" rowspan="1" colspan="1">5.11</td>
<td align="left" rowspan="1" colspan="1">Baseline</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">XGB (Top 25 variables)</td>
<td align="left" rowspan="1" colspan="1">2.91</td>
<td align="left" rowspan="1" colspan="1">+2.91 (547.58)</td>
<td align="left" rowspan="1" colspan="1">16.70</td>
<td align="left" rowspan="1" colspan="1">+9.74 (139.77)</td>
<td align="left" rowspan="1" colspan="1">9.13</td>
<td align="left" rowspan="1" colspan="1">+9.74 (139.77)</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">XGB (non-lab. variables)</td>
<td align="left" rowspan="1" colspan="1">2.24</td>
<td align="left" rowspan="1" colspan="1">+1.79 (397.73)</td>
<td align="left" rowspan="1" colspan="1">10.29</td>
<td align="left" rowspan="1" colspan="1">+3.33 (47.74)</td>
<td align="left" rowspan="1" colspan="1">5.48</td>
<td align="left" rowspan="1" colspan="1">+0.37 (7.19)</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/pdig.0000982.t005/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section></section></section><section id="sec018"><h2 class="pmc_sec_title">Discussion</h2>
<section id="sec019"><h3 class="pmc_sec_title">Key findings</h3>
<p>In a comprehensive analysis based on the UKB, our first key finding is that the ML model, trained with data from 150,000 participants and 64 variables, demonstrates improvement in predicting the 10-year risk of CVD and diabetes compared to traditional risk assessment tools, specifically the Framingham score and the DiabetesUK model, respectively. Our analysis shows that traditional risk models, while demonstrating utility in the general population, are limited when applied to individuals with multiple health conditions. Being constrained by linear assumptions, they may not fully capture the complexities inherent in such populations. Conversely, ML models uncover these complexities by learning the variable interactions and maintaining robustness in high-dimensional settings. They consistently reach closer to the estimated error bounds, indicating their superior predictive capability. Not only the use of ML models, considering a broader set of features (both benchmarks use only 7 core features) is proven to be beneficial for predictive performance. Our analysis also highlights that while some of these 64 variables may not be strongly predictive in isolation, their consideration in a multivariate ML model offers improved predictions for comorbid populations, which are frequently underserved by existing prediction guidelines. Take <em>triglycerides</em> as an example, which is ranked 4th in the permutation-based variable ranking for comorbid groups in diabetes prediction (see <a href="#pdig.0000982.t003" class="usa-link">Table 3</a>), whereas its individual AUC-ROC score is only 0.48. Similarly, while HbA1c is a strong standalone predictor, its predictive capacity is further enhanced when combined with other features, demonstrating that a combination of features is essential to achieving the best-performing model. Notably, a model with 25 predictors achieves slightly better performance than the all-variable model. In this reduced set, ELFs exhibit greater relative importance, underscoring their contribution to risk prediction for comorbid populations. Secondly, leveraging global and local model-agnostic interpretation approaches on our ML models, which incorporate ELFs and proximal risk factors, such as physical measures and activity, diet, and substance use, is instrumental in discovering potentially novel risk predictors for CVD and diabetes. This approach enlarges the set of variables traditionally considered relevant for predicting CVD and diabetes outcomes, thereby advancing our understanding of multimorbidity by revealing complex interactions among these variables, including both predictive and explanatory features. ELFs appear as key drivers of both performance and prediction when combined with proximal factors in comorbid groups, underscoring their role in improving the accuracy of CVD and diabetes risk predictions and understanding complex disease interactions. For instance, being <em>breastfed as a baby</em> for CVD, and <em>having someone to take to the doctor when needed as a child</em> for diabetes demonstrates information value in comorbid scenarios. While acknowledging the information value of these factors, it is crucial to recognize that breastfeeding is influenced by broader socioeconomic, educational, and cultural contexts [<a href="#pdig.0000982.ref042" class="usa-link" aria-describedby="pdig.0000982.ref042">42</a>], and having accessible medical care in childhood is a potential marker of physical neglect [<a href="#pdig.0000982.ref043" class="usa-link" aria-describedby="pdig.0000982.ref043">43</a>]. These elements highlight the complexity of isolating direct impacts on health outcomes, emphasizing the need for careful interpretation when such factors appear significant in predictive models for specific individuals. Our results support the claim that ELFs contribute to risk prediction in comorbid groups. While our study provides an initial understanding of their role in multimorbid populations, their contributions may vary within broader populations and, thus, further research is needed to comprehend how these findings generalize to more general populations. Additionally, this study has identified the crucial role of metabolic health measures, including both non-laboratory and laboratory variables, as well as psychological factors, in enhancing the accuracy of predicting CVD and diabetes for comorbid groups.</p></section><section id="sec020"><h3 class="pmc_sec_title">Clinical implications</h3>
<p>Our findings suggest that ML models may offer an improvement over traditional risk assessment tools such as the Framingham score and DiabetesUK, a gain that is particularly relevant for patients with comorbid conditions. The ML models appear to capture comorbid-specific risk factors, including influences from early life, that are not usually considered by existing algorithms. This indicates the potential for utilizing ML models in clinical settings for more personalized risk assessments since such models are robust in handling high-dimensional data, which facilitates the incorporation of life-course factors into risk models. The use of <em>post-hoc</em> interpretative methods on these models aids in making complex data more understandable, which could support clinicians in making informed decisions. However, further validation and research are necessary to fully grasp the implications of these findings for clinical practice, especially in the management and prevention of CVD and diabetes among patients with comorbidities. It is important to note that the 64-variable model, as used in this study, is not intended for direct clinical use. Rather, it serves as a tool to explore the role of ELFs as both performance drivers and prediction drivers in risk models for comorbid patients. Insights from this analysis aim to raise awareness of the importance of ELFs in risk prediction, encouraging the integration of key predictors, including ELFs, into future clinical models.</p></section><section id="sec021"><h3 class="pmc_sec_title">Limitations</h3>
<p>The main limitation of our study is that the exclusive use of the UKB affects the generalizability of our findings. With over 96% of participants being white and the age range of participants limited to 40–70 years at the time of assessment, the ethnic and age diversity is limited. Additionally, most of the non-laboratory variables in the UKB, such as health ratings, are obtained through an automated touchscreen questionnaire, a practice that may introduce self-reporting biases, potentially affecting the performance of the models. Furthermore, our study’s scope for investigating the contribution of early life factors is limited by the inclusion of only eight of these variables. Recent evidence [<a href="#pdig.0000982.ref006" class="usa-link" aria-describedby="pdig.0000982.ref006">6</a>, <a href="#pdig.0000982.ref044" class="usa-link" aria-describedby="pdig.0000982.ref044">44</a>] suggests that a wider variety of ELFs should be considered, such as perinatal characteristics like birth weight and prematurity, along with parental and neighborhood characteristics, within the context of more proximal risk factors.</p>
<p>Another limitation of our study is the performance of the models for predicting individuals with comorbid conditions at baseline. Addressing this issue could involve exploring alternative ML model families that might uncover different types of relationships within the data, potentially enhancing performance for this group. Crucially, observed differences in the smallest expected error (Bayes error) between comorbid and non-comorbid groups indicate that no model can achieve zero discrimination in performance without additional information. Therefore, integrating a broader spectrum of input variables may further enhance predictive accuracy for individuals with comorbid conditions. We encourage further research aimed at identifying more informative features and fostering greater transparency around the predictors used in these models, which may ultimately improve prediction performance for comorbid groups. We also note that our models identify statistical associations between variables rather than causal pathways. Therefore, utilizing <em>causal</em> machine learning would be instrumental in exploring potential causal links between ELFs and multimorbidity outcomes in more diverse population-based samples.</p></section></section><section id="sec022"><h2 class="pmc_sec_title">Supporting information</h2>
<section class="sm xbox font-sm" id="pdig.0000982.s001"><div class="caption p">
<span>S1 Table. Summary of all UKB input variables used in the study.</span><p>(PDF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12360575/bin/pdig.0000982.s001.pdf" data-ga-action="click_feat_suppl" class="usa-link">pdig.0000982.s001.pdf</a><sup> (76.7KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="pdig.0000982.s002"><div class="caption p">
<span>S1 Fig. Flow diagram of participant stratification based on early-life factors (ELFs), detailing CVD and diabetes outcomes in non-comorbid and comorbid subgroups.</span><p>(PDF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12360575/bin/pdig.0000982.s002.pdf" data-ga-action="click_feat_suppl" class="usa-link">pdig.0000982.s002.pdf</a><sup> (64KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="pdig.0000982.s003"><div class="caption p">
<span>S2 Table. Permutation-based variable ranking for predicting CVD in non-comorbid and comorbid groups.</span><p>(PDF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12360575/bin/pdig.0000982.s003.pdf" data-ga-action="click_feat_suppl" class="usa-link">pdig.0000982.s003.pdf</a><sup> (79.4KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="pdig.0000982.s004"><div class="caption p">
<span>S2 Fig. Comprehensive overview of participant profiles for the final study cohort with CVD outcomes: (a) Demographic and socioeconomic distribution, illustrating the population sample’s diversity.</span><p>(b) Proportion of early-life factors, underlining their frequency and potential impact on later CVD health outcomes.</p>
<p>(PDF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12360575/bin/pdig.0000982.s004.pdf" data-ga-action="click_feat_suppl" class="usa-link">pdig.0000982.s004.pdf</a><sup> (75.6KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="pdig.0000982.s005"><div class="caption p">
<span>S3 Table. Permutation-based variable ranking for predicting diabetes in non-comorbid and comorbid groups.</span><p>(PDF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12360575/bin/pdig.0000982.s005.pdf" data-ga-action="click_feat_suppl" class="usa-link">pdig.0000982.s005.pdf</a><sup> (67.3KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="pdig.0000982.s006"><div class="caption p">
<span>S3 Fig. Comprehensive overview of participant profiles for the final study cohort with diabetes outcomes: (a) Demographic and socioeconomic distribution, illustrating the population sample’s diversity.</span><p>(b) Proportion of early-life factors, underlining their frequency and potential impact on later diabetes health outcomes.</p>
<p>(PDF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12360575/bin/pdig.0000982.s006.pdf" data-ga-action="click_feat_suppl" class="usa-link">pdig.0000982.s006.pdf</a><sup> (75.5KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="pdig.0000982.s007"><div class="caption p">
<span>S4 Table. Differential top features by effect size: comorbid vs. non-comorbid.</span><p>The top 30 features are ranked by their effect size for CVD.</p>
<p>(PDF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12360575/bin/pdig.0000982.s007.pdf" data-ga-action="click_feat_suppl" class="usa-link">pdig.0000982.s007.pdf</a><sup> (88KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="pdig.0000982.s008"><div class="caption p">
<span>S4 Fig. Predictive ability of individual variables for CVD and diabetes prediction in comorbid and non-comorbid groups.</span><p>Complements <a href="#pdig.0000982.g003" class="usa-link">Fig 3</a> by showing the full distribution of AUC-ROC values for all features.</p>
<p>(PDF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12360575/bin/pdig.0000982.s008.pdf" data-ga-action="click_feat_suppl" class="usa-link">pdig.0000982.s008.pdf</a><sup> (98.9KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="pdig.0000982.s009"><div class="caption p">
<span>S5 Table. Differential top features by effect size: comorbid vs. non-comorbid.</span><p>The top 30 features are ranked by their effect size for diabetes.</p>
<p>(PDF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12360575/bin/pdig.0000982.s009.pdf" data-ga-action="click_feat_suppl" class="usa-link">pdig.0000982.s009.pdf</a><sup> (87.4KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="pdig.0000982.s010"><div class="caption p">
<span>S5 Fig. Shap scores of the top 30 influential variables in CVD prediction model for non-comorbid and comorbid groups.</span><p>Variables marked with an asterisk ( ) are those currently employed in current risk assessment models.</p>
<p>(PDF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12360575/bin/pdig.0000982.s010.pdf" data-ga-action="click_feat_suppl" class="usa-link">pdig.0000982.s010.pdf</a><sup> (48.5KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="pdig.0000982.s011"><div class="caption p">
<span>S6 Fig. Shap scores of the top 30 influential variables in diabetes prediction model for non-comorbid and comorbid groups.</span><p>Variables marked with an asterisk ( ) are those currently employed in current risk assessment models.</p>
<p>(PDF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12360575/bin/pdig.0000982.s011.pdf" data-ga-action="click_feat_suppl" class="usa-link">pdig.0000982.s011.pdf</a><sup> (48.3KB, pdf) </sup>
</div></div></section></section><section id="notes1"><h2 class="pmc_sec_title">Data Availability</h2>
<p>Data from this study are from the UK Biobank (UKB), available upon approved application. Researchers can register and request access at <a href="https://www.ukbiobank.ac.uk/enable-your-research/register" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ukbiobank.ac.uk/enable-your-research/register</a>. The results for UKB in this study will be returned to the UK Biobank within 6 months since publication, as required.</p></section><section id="funding-statement1" lang="en"><h2 class="pmc_sec_title">Funding Statement</h2>
<p>This study was supported by funding from the European Commission under Grant Agreement No. 101137146, STAGE. The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.</p></section><section id="ref-list1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="ref-list1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="pdig.0000982.ref001">
<span class="label">1.</span><cite>World Health Organization. Noncommunicable diseases. 2024. [cited 2024 May 2]. 2024. <a href="https://www.who.int/health-topics/noncommunicable-diseases" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.who.int/health-topics/noncommunicable-diseases</a></cite>
</li>
<li id="pdig.0000982.ref002">
<span class="label">2.</span><cite>World Health Organization. Cardiovascular diseases. 2024. [cited 2024 May 2]. <a href="https://www.who.int/health-topics/cardiovascular-diseases" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.who.int/health-topics/cardiovascular-diseases</a></cite>
</li>
<li id="pdig.0000982.ref003">
<span class="label">3.</span><cite>GBD 2021 Diabetes Collaborators. Global, regional, and national burden of diabetes from 1990 to 2021, with projections of prevalence to 2050: a systematic analysis for the Global Burden of Disease Study 2021. Lancet.
2023;402(10397):203–34. doi: 10.1016/S0140-6736(23)01301-6

</cite> [<a href="https://doi.org/10.1016/S0140-6736(23)01301-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10364581/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37356446/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Lancet.&amp;title=Global,%20regional,%20and%20national%20burden%20of%20diabetes%20from%201990%20to%202021,%20with%20projections%20of%20prevalence%20to%202050:%20a%20systematic%20analysis%20for%20the%20Global%20Burden%20of%20Disease%20Study%202021&amp;volume=402&amp;issue=10397&amp;publication_year=2023&amp;pages=203-34&amp;pmid=37356446&amp;doi=10.1016/S0140-6736(23)01301-6&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pdig.0000982.ref004">
<span class="label">4.</span><cite>Stein DJ, Benjet C, Gureje O, Lund C, Scott KM, Poznyak V, et al. Integrating mental health with other non-communicable diseases. BMJ.
2019;364:l295. doi: 10.1136/bmj.l295

</cite> [<a href="https://doi.org/10.1136/bmj.l295" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6348425/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30692081/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMJ.&amp;title=Integrating%20mental%20health%20with%20other%20non-communicable%20diseases&amp;author=DJ%20Stein&amp;author=C%20Benjet&amp;author=O%20Gureje&amp;author=C%20Lund&amp;author=KM%20Scott&amp;volume=364&amp;publication_year=2019&amp;pmid=30692081&amp;doi=10.1136/bmj.l295&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pdig.0000982.ref005">
<span class="label">5.</span><cite>Einarson TR, Acs A, Ludwig C, Panton UH. Prevalence of cardiovascular disease in type 2 diabetes: a systematic literature review of scientific evidence from across the world in 2007 -2017. Cardiovasc Diabetol.
2018;17(1):83. doi: 10.1186/s12933-018-0728-6

</cite> [<a href="https://doi.org/10.1186/s12933-018-0728-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5994068/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29884191/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cardiovasc%20Diabetol.&amp;title=Prevalence%20of%20cardiovascular%20disease%20in%20type%202%20diabetes:%20a%20systematic%20literature%20review%20of%20scientific%20evidence%20from%20across%20the%20world%20in%202007%20-2017&amp;author=TR%20Einarson&amp;author=A%20Acs&amp;author=C%20Ludwig&amp;author=UH%20Panton&amp;volume=17&amp;issue=1&amp;publication_year=2018&amp;pages=83&amp;pmid=29884191&amp;doi=10.1186/s12933-018-0728-6&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pdig.0000982.ref006">
<span class="label">6.</span><cite>Mariani N, Borsini A, Cecil CAM, Felix JF, Sebert S, Cattaneo A, et al. Identifying causative mechanisms linking early-life stress to psycho-cardio-metabolic multi-morbidity: the earlycause project. PLoS One.
2021;16(1):e0245475. doi: 10.1371/journal.pone.0245475

</cite> [<a href="https://doi.org/10.1371/journal.pone.0245475" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7819604/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33476328/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20One.&amp;title=Identifying%20causative%20mechanisms%20linking%20early-life%20stress%20to%20psycho-cardio-metabolic%20multi-morbidity:%20the%20earlycause%20project&amp;author=N%20Mariani&amp;author=A%20Borsini&amp;author=CAM%20Cecil&amp;author=JF%20Felix&amp;author=S%20Sebert&amp;volume=16&amp;issue=1&amp;publication_year=2021&amp;pmid=33476328&amp;doi=10.1371/journal.pone.0245475&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pdig.0000982.ref007">
<span class="label">7.</span><cite>Carr CP, Martins CMS, Stingel AM, Lemgruber VB, Juruena MF. The role of early life stress in adult psychiatric disorders: a systematic review according to childhood trauma subtypes. J Nerv Ment Dis.
2013;201(12):1007–20. doi: 10.1097/NMD.0000000000000049

</cite> [<a href="https://doi.org/10.1097/NMD.0000000000000049" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24284634/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Nerv%20Ment%20Dis.&amp;title=The%20role%20of%20early%20life%20stress%20in%20adult%20psychiatric%20disorders:%20a%20systematic%20review%20according%20to%20childhood%20trauma%20subtypes&amp;author=CP%20Carr&amp;author=CMS%20Martins&amp;author=AM%20Stingel&amp;author=VB%20Lemgruber&amp;author=MF%20Juruena&amp;volume=201&amp;issue=12&amp;publication_year=2013&amp;pages=1007-20&amp;pmid=24284634&amp;doi=10.1097/NMD.0000000000000049&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pdig.0000982.ref008">
<span class="label">8.</span><cite>Leeb RT. Child maltreatment surveillance: uniform definitions for public health and recommended data elements. Centers for Disease Control and Prevention, National Center for Injury Prevention and Control; 2008. <a href="https://stacks.cdc.gov/view/cdc/11493" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://stacks.cdc.gov/view/cdc/11493</a></cite>
</li>
<li id="pdig.0000982.ref009">
<span class="label">9.</span><cite>Heim C, Binder EB. Current research trends in early life stress and depression: review of human studies on sensitive periods, gene-environment interactions, and epigenetics. Exp Neurol.
2012;233(1):102–11. doi: 10.1016/j.expneurol.2011.10.032

</cite> [<a href="https://doi.org/10.1016/j.expneurol.2011.10.032" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22101006/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Exp%20Neurol.&amp;title=Current%20research%20trends%20in%20early%20life%20stress%20and%20depression:%20review%20of%20human%20studies%20on%20sensitive%20periods,%20gene-environment%20interactions,%20and%20epigenetics&amp;author=C%20Heim&amp;author=EB%20Binder&amp;volume=233&amp;issue=1&amp;publication_year=2012&amp;pages=102-11&amp;pmid=22101006&amp;doi=10.1016/j.expneurol.2011.10.032&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pdig.0000982.ref010">
<span class="label">10.</span><cite>Lähdepuro A, Savolainen K, Lahti-Pulkkinen M, Eriksson JG, Lahti J, Tuovinen S, et al. The impact of early life stress on anxiety symptoms in late adulthood. Sci Rep.
2019;9(1):4395. doi: 10.1038/s41598-019-40698-0

</cite> [<a href="https://doi.org/10.1038/s41598-019-40698-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6416302/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30867476/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sci%20Rep.&amp;title=The%20impact%20of%20early%20life%20stress%20on%20anxiety%20symptoms%20in%20late%20adulthood&amp;author=A%20L%C3%A4hdepuro&amp;author=K%20Savolainen&amp;author=M%20Lahti-Pulkkinen&amp;author=JG%20Eriksson&amp;author=J%20Lahti&amp;volume=9&amp;issue=1&amp;publication_year=2019&amp;pages=4395&amp;pmid=30867476&amp;doi=10.1038/s41598-019-40698-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pdig.0000982.ref011">
<span class="label">11.</span><cite>Murphy MO, Cohn DM, Loria AS. Developmental origins of cardiovascular disease: Impact of early life stress in humans and rodents. Neurosci Biobehav Rev.
2017;74(Pt B):453–65. doi: 10.1016/j.neubiorev.2016.07.018

</cite> [<a href="https://doi.org/10.1016/j.neubiorev.2016.07.018" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5250589/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27450581/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Neurosci%20Biobehav%20Rev.&amp;title=Developmental%20origins%20of%20cardiovascular%20disease:%20Impact%20of%20early%20life%20stress%20in%20humans%20and%20rodents&amp;author=MO%20Murphy&amp;author=DM%20Cohn&amp;author=AS%20Loria&amp;volume=74&amp;publication_year=2017&amp;pages=453-65&amp;pmid=27450581&amp;doi=10.1016/j.neubiorev.2016.07.018&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pdig.0000982.ref012">
<span class="label">12.</span><cite>Jiang X, Ma H, Wang Y, Liu Y. Early life factors and type 2 diabetes mellitus. J Diabetes Res.
2013;2013:485082. doi: 10.1155/2013/485082

</cite> [<a href="https://doi.org/10.1155/2013/485082" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3876901/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24455747/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Diabetes%20Res.&amp;title=Early%20life%20factors%20and%20type%202%20diabetes%20mellitus&amp;author=X%20Jiang&amp;author=H%20Ma&amp;author=Y%20Wang&amp;author=Y%20Liu&amp;volume=2013&amp;publication_year=2013&amp;pages=485082&amp;pmid=24455747&amp;doi=10.1155/2013/485082&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pdig.0000982.ref013">
<span class="label">13.</span><cite>Alaa AM, Bolton T, Di Angelantonio E, Rudd JHF, van der Schaar M. Cardiovascular disease risk prediction using automated machine learning: a prospective study of 423,604 UK Biobank participants. PLoS One.
2019;14(5):e0213653. doi: 10.1371/journal.pone.0213653

</cite> [<a href="https://doi.org/10.1371/journal.pone.0213653" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6519796/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31091238/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20One.&amp;title=Cardiovascular%20disease%20risk%20prediction%20using%20automated%20machine%20learning:%20a%20prospective%20study%20of%20423,604%20UK%20Biobank%20participants&amp;author=AM%20Alaa&amp;author=T%20Bolton&amp;author=E%20Di%20Angelantonio&amp;author=JHF%20Rudd&amp;author=M%20van%20der%20Schaar&amp;volume=14&amp;issue=5&amp;publication_year=2019&amp;pmid=31091238&amp;doi=10.1371/journal.pone.0213653&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pdig.0000982.ref014">
<span class="label">14.</span><cite>Greenland P, Alpert JS, Beller GA, Benjamin EJ, Budoff MJ, Fayad ZA, et al. 2010 ACCF/AHA guideline for assessment of cardiovascular risk in asymptomatic adults: a report of the American College of Cardiology Foundation/American Heart Association Task Force on Practice Guidelines. J Am Coll Cardiol.
2010;56(25):e50-103. doi: 10.1016/j.jacc.2010.09.001

</cite> [<a href="https://doi.org/10.1016/j.jacc.2010.09.001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21144964/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Am%20Coll%20Cardiol.&amp;title=2010%20ACCF/AHA%20guideline%20for%20assessment%20of%20cardiovascular%20risk%20in%20asymptomatic%20adults:%20a%20report%20of%20the%20American%20College%20of%20Cardiology%20Foundation/American%20Heart%20Association%20Task%20Force%20on%20Practice%20Guidelines&amp;author=P%20Greenland&amp;author=JS%20Alpert&amp;author=GA%20Beller&amp;author=EJ%20Benjamin&amp;author=MJ%20Budoff&amp;volume=56&amp;issue=25&amp;publication_year=2010&amp;pmid=21144964&amp;doi=10.1016/j.jacc.2010.09.001&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pdig.0000982.ref015">
<span class="label">15.</span><cite>Siontis GCM, Tzoulaki I, Siontis KC, Ioannidis JPA. Comparisons of established risk prediction models for cardiovascular disease: systematic review. BMJ.
2012;344:e3318. doi: 10.1136/bmj.e3318

</cite> [<a href="https://doi.org/10.1136/bmj.e3318" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22628003/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMJ.&amp;title=Comparisons%20of%20established%20risk%20prediction%20models%20for%20cardiovascular%20disease:%20systematic%20review&amp;author=GCM%20Siontis&amp;author=I%20Tzoulaki&amp;author=KC%20Siontis&amp;author=JPA%20Ioannidis&amp;volume=344&amp;publication_year=2012&amp;pmid=22628003&amp;doi=10.1136/bmj.e3318&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pdig.0000982.ref016">
<span class="label">16.</span><cite>Bzdok D, Varoquaux G, Steyerberg EW. Prediction, not association, paves the road to precision medicine. JAMA Psychiatry.
2021;78(2):127–8. doi: 10.1001/jamapsychiatry.2020.2549

</cite> [<a href="https://doi.org/10.1001/jamapsychiatry.2020.2549" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32804995/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=JAMA%20Psychiatry.&amp;title=Prediction,%20not%20association,%20paves%20the%20road%20to%20precision%20medicine&amp;author=D%20Bzdok&amp;author=G%20Varoquaux&amp;author=EW%20Steyerberg&amp;volume=78&amp;issue=2&amp;publication_year=2021&amp;pages=127-8&amp;pmid=32804995&amp;doi=10.1001/jamapsychiatry.2020.2549&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pdig.0000982.ref017">
<span class="label">17.</span><cite>Shmueli G. To explain or to predict?
Statist Sci.
2010;25(3). doi: 10.1214/10-sts330</cite> [<a href="https://doi.org/10.1214/10-sts330" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Statist%20Sci.&amp;title=To%20explain%20or%20to%20predict?&amp;author=G%20Shmueli&amp;volume=25&amp;issue=3&amp;publication_year=2010&amp;doi=10.1214/10-sts330&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pdig.0000982.ref018">
<span class="label">18.</span><cite>Imrie F, Cebere B, McKinney EF, van der Schaar M. AutoPrognosis 2.0: democratizing diagnostic and prognostic modeling in healthcare with automated machine learning. PLOS Digit Health.
2023;2(6):e0000276. doi: 10.1371/journal.pdig.0000276

</cite> [<a href="https://doi.org/10.1371/journal.pdig.0000276" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10287005/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37347752/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLOS%20Digit%20Health.&amp;title=AutoPrognosis%202.0:%20democratizing%20diagnostic%20and%20prognostic%20modeling%20in%20healthcare%20with%20automated%20machine%20learning&amp;author=F%20Imrie&amp;author=B%20Cebere&amp;author=EF%20McKinney&amp;author=M%20van%20der%20Schaar&amp;volume=2&amp;issue=6&amp;publication_year=2023&amp;pmid=37347752&amp;doi=10.1371/journal.pdig.0000276&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pdig.0000982.ref019">
<span class="label">19.</span><cite>Chen T, Guestrin C. XGBoost. In: Proceedings of the 22nd ACM SIGKDD International Conference on Knowledge Discovery and Data Mining. 2016. p. 785–94. 10.1145/2939672.2939785</cite> [<a href="https://doi.org/10.1145/2939672.2939785" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>]</li>
<li id="pdig.0000982.ref020">
<span class="label">20.</span><cite>Fregoso-Aparicio L, Noguez J, Montesinos L, García-García JA. Machine learning and deep learning predictive models for type 2 diabetes: a systematic review. Diabetol Metab Syndr.
2021;13(1):148. doi: 10.1186/s13098-021-00767-9

</cite> [<a href="https://doi.org/10.1186/s13098-021-00767-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8686642/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34930452/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Diabetol%20Metab%20Syndr.&amp;title=Machine%20learning%20and%20deep%20learning%20predictive%20models%20for%20type%202%20diabetes:%20a%20systematic%20review&amp;author=L%20Fregoso-Aparicio&amp;author=J%20Noguez&amp;author=L%20Montesinos&amp;author=JA%20Garc%C3%ADa-Garc%C3%ADa&amp;volume=13&amp;issue=1&amp;publication_year=2021&amp;pages=148&amp;pmid=34930452&amp;doi=10.1186/s13098-021-00767-9&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pdig.0000982.ref021">
<span class="label">21.</span><cite>Sudlow C, Gallacher J, Allen N, Beral V, Burton P, Danesh J, et al. UK biobank: an open access resource for identifying the causes of a wide range of complex diseases of middle and old age. PLoS Med.
2015;12(3):e1001779. doi: 10.1371/journal.pmed.1001779

</cite> [<a href="https://doi.org/10.1371/journal.pmed.1001779" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4380465/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25826379/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20Med.&amp;title=UK%20biobank:%20an%20open%20access%20resource%20for%20identifying%20the%20causes%20of%20a%20wide%20range%20of%20complex%20diseases%20of%20middle%20and%20old%20age&amp;author=C%20Sudlow&amp;author=J%20Gallacher&amp;author=N%20Allen&amp;author=V%20Beral&amp;author=P%20Burton&amp;volume=12&amp;issue=3&amp;publication_year=2015&amp;pmid=25826379&amp;doi=10.1371/journal.pmed.1001779&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pdig.0000982.ref022">
<span class="label">22.</span><cite>American Diabetes Association. A1C and diagnosis. 2024. [cited 2024 Dec 8]. <a href="https://diabetes.org/diabetes/a1c/diagnosis" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://diabetes.org/diabetes/a1c/diagnosis</a></cite>
</li>
<li id="pdig.0000982.ref023">
<span class="label">23.</span><cite>Lugner M, Rawshani A, Helleryd E, Eliasson B. Identifying top ten predictors of type 2 diabetes through machine learning analysis of UK Biobank data. Sci Rep.
2024;14(1). doi: 10.1038/s41598-024-52023-5</cite> [<a href="https://doi.org/10.1038/s41598-024-52023-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10808323/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38267466/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sci%20Rep.&amp;title=Identifying%20top%20ten%20predictors%20of%20type%202%20diabetes%20through%20machine%20learning%20analysis%20of%20UK%20Biobank%20data&amp;author=M%20Lugner&amp;author=A%20Rawshani&amp;author=E%20Helleryd&amp;author=B%20Eliasson&amp;volume=14&amp;issue=1&amp;publication_year=2024&amp;pmid=38267466&amp;doi=10.1038/s41598-024-52023-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pdig.0000982.ref024">
<span class="label">24.</span><cite>Hippisley-Cox J, Coupland C. Development and validation of QDiabetes-2018 risk prediction algorithm to estimate future risk of type 2 diabetes: cohort study. BMJ.
2017;359:j5019. doi: 10.1136/bmj.j5019

</cite> [<a href="https://doi.org/10.1136/bmj.j5019" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5694979/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29158232/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMJ.&amp;title=Development%20and%20validation%20of%20QDiabetes-2018%20risk%20prediction%20algorithm%20to%20estimate%20future%20risk%20of%20type%202%20diabetes:%20cohort%20study&amp;author=J%20Hippisley-Cox&amp;author=C%20Coupland&amp;volume=359&amp;publication_year=2017&amp;pmid=29158232&amp;doi=10.1136/bmj.j5019&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pdig.0000982.ref025">
<span class="label">25.</span><cite>National Heart, Lung, and Blood Institute. Framingham Heart Study. 2024. [cited 2024 May 2]. <a href="https://www.framinghamheartstudy.org" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.framinghamheartstudy.org</a></cite>
</li>
<li id="pdig.0000982.ref026">
<span class="label">26.</span><cite>Diabetes UK. Know your risk of type 2 diabetes. 2024. [cited 2024 May 2]. <a href="https://riskscore.diabetes.org.uk/start" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://riskscore.diabetes.org.uk/start</a></cite>
</li>
<li id="pdig.0000982.ref027">
<span class="label">27.</span><cite>Pedregosa F, Varoquaux G, Gramfort A, Michel V, Thirion B, Grisel O. Scikit-learn: machine learning in python. Journal of Machine Learning Research.
2011;12:2825–30.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Machine%20Learning%20Research.&amp;title=Scikit-learn:%20machine%20learning%20in%20python&amp;author=F%20Pedregosa&amp;author=G%20Varoquaux&amp;author=A%20Gramfort&amp;author=V%20Michel&amp;author=B%20Thirion&amp;volume=12&amp;publication_year=2011&amp;pages=2825-30&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pdig.0000982.ref028">
<span class="label">28.</span><cite>Murphy AH. A new vector partition of the probability score. J Appl Meteor.
1973;12(4):595–600. doi: </cite> [<a href="https://doi.org/10.1175/1520-0450(1973)012&amp;#x0003c;0595:anvpot&amp;#x0003e;2.0.co;2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Appl%20Meteor.&amp;title=A%20new%20vector%20partition%20of%20the%20probability%20score&amp;author=AH%20Murphy&amp;volume=12&amp;issue=4&amp;publication_year=1973&amp;pages=595-600&amp;doi=10.1175/1520-0450(1973)012&amp;#x0003c;0595:anvpot&amp;#x0003e;2.0.co;2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pdig.0000982.ref029">
<span class="label">29.</span><cite>Vickers AJ, Elkin EB. Decision curve analysis: a novel method for evaluating prediction models. Med Decis Making.
2006;26(6):565–74. doi: 10.1177/0272989X06295361

</cite> [<a href="https://doi.org/10.1177/0272989X06295361" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2577036/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17099194/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Med%20Decis%20Making.&amp;title=Decision%20curve%20analysis:%20a%20novel%20method%20for%20evaluating%20prediction%20models&amp;author=AJ%20Vickers&amp;author=EB%20Elkin&amp;volume=26&amp;issue=6&amp;publication_year=2006&amp;pages=565-74&amp;pmid=17099194&amp;doi=10.1177/0272989X06295361&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pdig.0000982.ref030">
<span class="label">30.</span><cite>Chen IY, Johansson FD, Sontag D. Why is my classifier discriminatory?. In: Proceedings of the 32nd International Conference on Neural Information Processing Systems. Curran Associates Inc.; 2018. p. 3543–54. <a href="https://dl.acm.org/doi/10.5555/3327144.3327272" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://dl.acm.org/doi/10.5555/3327144.3327272</a></cite>
</li>
<li id="pdig.0000982.ref031">
<span class="label">31.</span><cite>Mahalanobis PC. On the generalized distance in statistics. Sankhya A.
2018;80(Suppl 1):1–7. doi: 10.1007/s13171-019-00164-5</cite> [<a href="https://doi.org/10.1007/s13171-019-00164-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sankhya%20A.&amp;title=On%20the%20generalized%20distance%20in%20statistics&amp;author=PC%20Mahalanobis&amp;volume=80&amp;publication_year=2018&amp;pages=1-7&amp;doi=10.1007/s13171-019-00164-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pdig.0000982.ref032">
<span class="label">32.</span><cite>Bhattacharyya A. On a measure of divergence between two statistical populations defined by their probability distributions. Bull Calcutta Math Soc.
1943;35:99–109.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Bull%20Calcutta%20Math%20Soc.&amp;title=On%20a%20measure%20of%20divergence%20between%20two%20statistical%20populations%20defined%20by%20their%20probability%20distributions&amp;author=A%20Bhattacharyya&amp;volume=35&amp;publication_year=1943&amp;pages=99-109&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pdig.0000982.ref033">
<span class="label">33.</span><cite>Cover T, Hart P. Nearest neighbor pattern classification. IEEE Trans Inform Theory.
1967;13(1):21–7. doi: 10.1109/tit.1967.1053964</cite> [<a href="https://doi.org/10.1109/tit.1967.1053964" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=IEEE%20Trans%20Inform%20Theory.&amp;title=Nearest%20neighbor%20pattern%20classification&amp;author=T%20Cover&amp;author=P%20Hart&amp;volume=13&amp;issue=1&amp;publication_year=1967&amp;pages=21-7&amp;doi=10.1109/tit.1967.1053964&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pdig.0000982.ref034">
<span class="label">34.</span><cite>Breiman L. Random Forests. Machine Learning.
2001;45(1):5–32. doi: 10.1023/a:1010933404324</cite> [<a href="https://doi.org/10.1023/a:1010933404324" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Machine%20Learning.&amp;title=Random%20Forests&amp;author=L%20Breiman&amp;volume=45&amp;issue=1&amp;publication_year=2001&amp;pages=5-32&amp;doi=10.1023/a:1010933404324&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pdig.0000982.ref035">
<span class="label">35.</span><cite>Zar JH. Spearman Rank Correlation. Encyclopedia of Biostatistics. Wiley; 2005. 10.1002/0470011815.b2a15150</cite> [<a href="https://doi.org/10.1002/0470011815.b2a15150" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>]</li>
<li id="pdig.0000982.ref036">
<span class="label">36.</span><cite>Lundberg S, Lee SI. A unified approach to interpreting model predictions. In: Advances in Neural Information Processing Systems. Curran Associates; 2017 . p. 4765–74. <a href="https://dl.acm.org/doi/10.5555/3295222.3295230" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://dl.acm.org/doi/10.5555/3295222.3295230</a></cite>
</li>
<li id="pdig.0000982.ref037">
<span class="label">37.</span><cite>Cohen J. Statistical power analysis for the behavioral sciences. Routledge; 2013. 10.4324/9780203771587</cite> [<a href="https://doi.org/10.4324/9780203771587" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>]</li>
<li id="pdig.0000982.ref038">
<span class="label">38.</span><cite>Li C, Bishop TRP, Imamura F, Sharp SJ, Pearce M, Brage S, et al. Meat consumption and incident type 2 diabetes: an individual-participant federated meta-analysis of 1<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pdig.0000982.e088"><math id="M88" display="inline" overflow="linebreak"><mrow><mo>·</mo></mrow></math></span>97 million adults with 100 000 incident cases from 31 cohorts in 20 countries. Lancet Diabetes Endocrinol.
2024;12(9):619–30. doi: 10.1016/S2213-8587(24)00179-7

</cite> [<a href="https://doi.org/10.1016/S2213-8587(24)00179-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39174161/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Lancet%20Diabetes%20Endocrinol.&amp;title=Meat%20consumption%20and%20incident%20type%202%20diabetes:%20an%20individual-participant%20federated%20meta-analysis%20of%201%C2%B797%20million%20adults%20with%20100%20000%20incident%20cases%20from%2031%20cohorts%20in%2020%20countries&amp;author=C%20Li&amp;author=TRP%20Bishop&amp;author=F%20Imamura&amp;author=SJ%20Sharp&amp;author=M%20Pearce&amp;volume=12&amp;issue=9&amp;publication_year=2024&amp;pages=619-30&amp;pmid=39174161&amp;doi=10.1016/S2213-8587(24)00179-7&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pdig.0000982.ref039">
<span class="label">39.</span><cite>Al Yousef MZ, Yasky AF, Al Shammari R, Ferwana MS. Early prediction of diabetes by applying data mining techniques: a retrospective cohort study. Medicine (Baltimore).
2022;101(29):e29588. doi: 10.1097/MD.0000000000029588

</cite> [<a href="https://doi.org/10.1097/MD.0000000000029588" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9302319/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35866773/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Medicine%20(Baltimore).&amp;title=Early%20prediction%20of%20diabetes%20by%20applying%20data%20mining%20techniques:%20a%20retrospective%20cohort%20study&amp;author=MZ%20Al%20Yousef&amp;author=AF%20Yasky&amp;author=R%20Al%20Shammari&amp;author=MS%20Ferwana&amp;volume=101&amp;issue=29&amp;publication_year=2022&amp;pmid=35866773&amp;doi=10.1097/MD.0000000000029588&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pdig.0000982.ref040">
<span class="label">40.</span><cite>Yousef H, Feng SF, Jelinek HF. Exploratory risk prediction of type II diabetes with isolation forests and novel biomarkers. Sci Rep.
2024;14(1). doi: 10.1038/s41598-024-65044-x</cite> [<a href="https://doi.org/10.1038/s41598-024-65044-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11193708/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38909127/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sci%20Rep.&amp;title=Exploratory%20risk%20prediction%20of%20type%20II%20diabetes%20with%20isolation%20forests%20and%20novel%20biomarkers&amp;author=H%20Yousef&amp;author=SF%20Feng&amp;author=HF%20Jelinek&amp;volume=14&amp;issue=1&amp;publication_year=2024&amp;pmid=38909127&amp;doi=10.1038/s41598-024-65044-x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pdig.0000982.ref041">
<span class="label">41.</span><cite>Iparraguirre-Villanueva O, Espinola-Linares K, Flores Castañeda RO, Cabanillas-Carbonell M. Application of machine learning models for early detection and accurate classification of type 2 diabetes. Diagnostics (Basel).
2023;13(14):2383. doi: 10.3390/diagnostics13142383

</cite> [<a href="https://doi.org/10.3390/diagnostics13142383" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10378239/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37510127/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Diagnostics%20(Basel).&amp;title=Application%20of%20machine%20learning%20models%20for%20early%20detection%20and%20accurate%20classification%20of%20type%202%20diabetes&amp;author=O%20Iparraguirre-Villanueva&amp;author=K%20Espinola-Linares&amp;author=RO%20Flores%20Casta%C3%B1eda&amp;author=M%20Cabanillas-Carbonell&amp;volume=13&amp;issue=14&amp;publication_year=2023&amp;pages=2383&amp;pmid=37510127&amp;doi=10.3390/diagnostics13142383&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pdig.0000982.ref042">
<span class="label">42.</span><cite>Celi AC, Rich-Edwards JW, Richardson MK, Kleinman KP, Gillman MW. Immigration, race/ethnicity, and social and economic factors as predictors of breastfeeding initiation. Arch Pediatr Adolesc Med.
2005;159(3):255–60. doi: 10.1001/archpedi.159.3.255

</cite> [<a href="https://doi.org/10.1001/archpedi.159.3.255" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15753269/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Arch%20Pediatr%20Adolesc%20Med.&amp;title=Immigration,%20race/ethnicity,%20and%20social%20and%20economic%20factors%20as%20predictors%20of%20breastfeeding%20initiation&amp;author=AC%20Celi&amp;author=JW%20Rich-Edwards&amp;author=MK%20Richardson&amp;author=KP%20Kleinman&amp;author=MW%20Gillman&amp;volume=159&amp;issue=3&amp;publication_year=2005&amp;pages=255-60&amp;pmid=15753269&amp;doi=10.1001/archpedi.159.3.255&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pdig.0000982.ref043">
<span class="label">43.</span><cite>Knutson JF, DeGarmo DS, Reid JB. Social disadvantage and neglectful parenting as precursors to the development of antisocial and aggressive child behavior: Testing a theoretical model. Aggressive Behavior.
2004;30(3):187–205. doi: 10.1002/ab.20016</cite> [<a href="https://doi.org/10.1002/ab.20016" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Aggressive%20Behavior.&amp;title=Social%20disadvantage%20and%20neglectful%20parenting%20as%20precursors%20to%20the%20development%20of%20antisocial%20and%20aggressive%20child%20behavior:%20Testing%20a%20theoretical%20model&amp;author=JF%20Knutson&amp;author=DS%20DeGarmo&amp;author=JB%20Reid&amp;volume=30&amp;issue=3&amp;publication_year=2004&amp;pages=187-205&amp;doi=10.1002/ab.20016&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pdig.0000982.ref044">
<span class="label">44.</span><cite>Noll JG, Shalev I. The Biology of Early Life Stress. Springer; 2018. 10.1007/978-3-319-72589-5</cite> [<a href="https://doi.org/10.1007/978-3-319-72589-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>]</li>
</ul></section></section></section><article class="sub-article" id="pdig.0000982.r001"><section class="pmc-layout__citation font-secondary font-xs"><div>PLOS Digit Health. doi: <a href="https://doi.org/10.1371/journal.pdig.0000982.r001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1371/journal.pdig.0000982.r001</a>
</div></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Decision Letter 0</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kwak%20GH%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Gloria Hyunjung Kwak</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Gloria Hyunjung Kwak</span></h3>
<div>Section Editor</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kwak%20GH%22%5BAuthor%5D" class="usa-link"><span class="name western">Gloria Hyunjung Kwak</span></a>
</div>
</div>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a.c" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="clp_a.c" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a.c" class="d-panel p" style="display: none">
<h4 class="font-secondary">Roles</h4>
<div class="p">
<strong class="contrib"><span class="name western">Gloria Hyunjung Kwak</span></strong>: <span class="role">Section Editor</span>
</div>
</div>
<div id="clp_a.c" class="d-panel p" style="display: none">
<div>© 2025 Gloria Hyunjung Kwak</div>
<p>This is an open access article distributed under the terms of the <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Creative Commons Attribution License</a>, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div></div>
</div></section><section class="body sub-article-body"><hr class="headless">
<p>
<em>7 Apr 2025</em>
</p>
<p>PDIG-D-25-00125Characterizing the role of early life factors in machine learning-based multimorbidity risk predictionPLOS Digital Health Dear Dr. Dang, Thank you for submitting your manuscript to PLOS Digital Health. After careful consideration, we feel that it has merit but does not fully meet PLOS Digital Health's publication criteria as it currently stands. Therefore, we invite you to submit a revised version of the manuscript that addresses the points raised during the review process. Please submit your revised manuscript within 60 days Jun 06 2025 11:59PM. If you will need more time than this to complete your revisions, please reply to this message or contact the journal office at <span>digitalhealth@plos.org</span>. When you're ready to submit your revision, log on to <a href="https://www.editorialmanager.com/pdig/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.editorialmanager.com/pdig/</a> and select the 'Submissions Needing Revision' folder to locate your manuscript file. Please include the following items when submitting your revised manuscript:* A rebuttal letter that responds to each point raised by the editor and reviewer(s). You should upload this letter as a separate file labeled '<span class="text-underline">Response to Reviewers</span>'. This file does not need to include responses to any formatting updates and technical items listed in the 'Journal Requirements' section below.* A marked-up copy of your manuscript that highlights changes made to the original version. You should upload this as a separate file labeled '<span class="text-underline">Revised Manuscript with Track Changes</span>'.* An unmarked version of your revised paper without tracked changes. You should upload this as a separate file labeled '<span class="text-underline">Manuscript</span>'. If you would like to make changes to your financial disclosure, competing interests statement, or data availability statement, please make these updates within the submission form at the time of resubmission. Guidelines for resubmitting your figure files are available below the reviewer comments at the end of this letter. We look forward to receiving your revised manuscript. Kind regards, Gloria Hyunjung KwakSection EditorPLOS Digital Health Gloria Hyunjung KwakSection EditorPLOS Digital Health Leo Anthony CeliEditor-in-ChiefPLOS Digital Healthorcid.org/0000-0001-6712-6626 <strong>Journal Requirements:</strong></p>
<p>1. We ask that a manuscript source file is provided at Revision. Please upload your manuscript file as a .doc, .docx, .rtf or .tex. <strong>Additional Editor Comments (if provided):</strong>   [Note: HTML markup is below. Please do not edit.] <strong>Reviewers' Comments:</strong> Reviewer's Responses to Questions</p>
<p>

<strong>Comments to the Author</strong>
</p>
<p>1. Does this manuscript meet PLOS Digital Health’s <a href="https://journals.plos.org/digitalhealth/s/journal-information#loc-criteria-for-publication" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">publication criteria</a>? Is the manuscript technically sound, and do the data support the conclusions? The manuscript must describe methodologically and ethically rigorous research with conclusions that are appropriately drawn based on the data presented.</p>
<p>Reviewer #1: Yes</p>
<p>Reviewer #2: Yes</p>
<p>**********</p>
<p>2. Has the statistical analysis been performed appropriately and rigorously?</p>
<p>Reviewer #1: Yes</p>
<p>Reviewer #2: Yes</p>
<p>**********</p>
<p>3. Have the authors made all data underlying the findings in their manuscript fully available (please refer to the Data Availability Statement at the start of the manuscript PDF file)?</p>
<p>The <a href="https://journals.plos.org/digitalhealth/s/data-availability" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PLOS Data policy</a> requires authors to make all data underlying the findings described in their manuscript fully available without restriction, with rare exception. The data should be provided as part of the manuscript or its supporting information, or deposited to a public repository. For example, in addition to summary statistics, the data points behind means, medians and variance measures should be available. If there are restrictions on publicly sharing data—e.g. participant privacy or use of data from a third party—those must be specified.</p>
<p>Reviewer #1: Yes</p>
<p>Reviewer #2: Yes</p>
<p>**********</p>
<p>4. Is the manuscript presented in an intelligible fashion and written in standard English?</p>
<p>PLOS Digital Health does not copyedit accepted manuscripts, so the language in submitted articles must be clear, correct, and unambiguous. Any typographical or grammatical errors should be corrected at revision, so please note any specific errors here.</p>
<p>Reviewer #1: Yes</p>
<p>Reviewer #2: Yes</p>
<p>**********</p>
<p>5. Review Comments to the Author</p>
<p>Please use the space provided to explain your answers to the questions above. You may also include additional comments for the author, including concerns about dual publication, research ethics, or publication ethics. (Please upload your review as an attachment if it exceeds 20,000 characters)</p>
<p>Reviewer #1: Dear Author,</p>
<p>This study analyzes the contribution of ELFs to machine-learning-based risk prediction models for comorbid populations. Yo also utilized incorporation of life-course factors into risk models. Integrating a diverse range of physiological, psychological, and ELFs. I anm wobdering about integrsation of such a multilayered data to develop an AI moder with a highly heterogenous data??. The abstract is unstructured.</p>
<p>Reviewer #2: This paper uses a series of machine learning models and analyses to</p>
<p>attempt to characterize the role of early life factors in</p>
<p>multimorbidity risk prediction. Specifically, the authors hope to use</p>
<p>an examination the role of these factors in predictive models to</p>
<p>inform an understanding of how they might impact outcomes.</p>
<p>The paper is through and interesting, including a broad range of</p>
<p>analyses that provide substantial understanding. Although there are a</p>
<p>number of points that could use clarification, I did not identify any</p>
<p>major methodological concerns.</p>
<p>I am curious about one important point that was not addressed in the</p>
<p>paper. If the goal was to address the role of early life factors, why</p>
<p>start with machine learning models as the preferred analysis approach?</p>
<p>Causal methods, such as propensity scoring or causal structural</p>
<p>discovery, might yield some important insights, particularly</p>
<p>regarding potential interactions between observed factors. The reasons</p>
<p>for the selected approaches should be given and contrasted with</p>
<p>alternative means of addressing questions of interest.</p>
<p>My other major concern with this paper is that the number of analyses</p>
<p>is large and the volume of data is somewhat overwhelming. Some</p>
<p>additional guidance, perhaps in terms of a workflow figure detailing</p>
<p>the types of analyses and the goals presented might make the paper</p>
<p>easier to read.</p>
<p>A number of minor readability/comprehensibility issues arose during my</p>
<p>reading of the paper:</p>
<p>1. In terms of selection of valuables (line 137-139), it might help to</p>
<p>update Table S1 to indicate which factors are early life factors.</p>
<p>2. Same lines, how are the 64 variables processed into 83 features?</p>
<p>Details should be provided.</p>
<p>3. Value of information - 207-217. Particularly since "value of</p>
<p>information" has a specific meaning in many contexts, this term</p>
<p>should be defined in more detail.</p>
<p>4. Same section - the group-specific Bayes error estimation methods</p>
<p>might not be familiar to all readers and should be described in a bit</p>
<p>more detail.</p>
<p>5. I don't understand what is meant by the comment that "metrics like</p>
<p>SHAP explain how variables contribute to performance but do not</p>
<p>directly quantify their contribution to predictive performance. I</p>
<p>assume that the point here is that SHAP is simply the result of</p>
<p>multiple models that approximate the original model, but I am not quite</p>
<p>sure. This somewhat cryptic statement should be replaced with</p>
<p>additional clarification.</p>
<p>6. The description of "Group-specific Bayes errors" is confusing,</p>
<p>mentioning noise estimates that were not discussed when the</p>
<p>Group-specific errors where introduced earlier. Although it's clear on</p>
<p>a second read that "noise estimates" and "errors" are meant to be</p>
<p>interchangeable, this paragraph would benefit from some clarification.</p>
<p>7. Figure 1 - the caption needs to be edited. "Framming score" is not</p>
<p>correct.</p>
<p>8. Figure 2 - the graphs as presented are a bit hard to read due to</p>
<p>the difference in the range covered in the x- and y- axes. For</p>
<p>example, the CVD graph seems to range from roughly 0.4- 0.6 on the</p>
<p>x-axis and 0.45-0.63 on the y axis. This difference leads the</p>
<p>equivalence (y=x) line to be skewed to the right, thus complicating</p>
<p>the interpretation of which was better. A parallel presentation with</p>
<p>the y=x line being a straight line from lower-left to upper right,</p>
<p>would be much clearer.</p>
<p>9. The discussion of HbA1c's role in diabetes prediction reads more</p>
<p>like methods than results. Perhaps this should be moved to the methods</p>
<p>section.</p>
<p>10. Finally, there are a great many results in this paper that are</p>
<p>hard to interpret. Some sort of tabular summary and/or clearer</p>
<p>delineation of the topics in the discussion would make the paper more</p>
<p>readable and the conclusions more digestible.</p>
<p>**********</p>
<p>6. PLOS authors have the option to publish the peer review history of their article (<a href="https://journals.plos.org/digitalhealth/s/editorial-and-peer-review-process#loc-peer-review-history" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">what does this mean?</a>). If published, this will include your full peer review and any attached files.</p>
<p><strong>Do you want your identity to be public for this peer review?</strong> If you choose “no”, your identity will remain anonymous but your review may still be made public.</p>
<p>For information about this choice, including consent withdrawal, please see our <a href="https://www.plos.org/privacy-policy" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Privacy Policy</a>.</p>
<p>Reviewer #1: No</p>
<p>Reviewer #2: <strong>Yes: </strong>Harry Hochheiser</p>
<p>**********</p>
<p> [NOTE: If reviewer comments were submitted as an attachment file, they will be attached to this email and accessible via the submission site. Please log into your account, locate the manuscript record, and check for the action link "View Attachments". If this link does not appear, there are no attachment files.] <strong>Figure resubmission:</strong> While revising your submission, please upload your figure files to the Preflight Analysis and Conversion Engine (PACE) digital diagnostic tool, <a href="https://pacev2.apexcovantage.com/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://pacev2.apexcovantage.com/</a>. PACE helps ensure that figures meet PLOS requirements. To use PACE, you must first register as a user. Registration is free. Then, login and navigate to the UPLOAD tab, where you will find detailed instructions on how to use the tool. If you encounter any issues or have any questions when using PACE, please email PLOS at <span>figures@plos.org</span>. Please note that Supporting Information files do not need this step. If there are other versions of figure files still present in your submission file inventory at resubmission, please replace them with the PACE-processed versions. <strong>Reproducibility:</strong> To enhance the reproducibility of your results, we recommend that authors of applicable studies deposit laboratory protocols in protocols.io, where a protocol can be assigned its own identifier (DOI) such that it can be cited independently in the future. Additionally, PLOS ONE offers an option to publish peer-reviewed clinical study protocols. Read more information on sharing protocols at <a href="https://plos.org/protocols?utm_medium=editorial-email&amp;utm_source=authorletters&amp;utm_campaign=protocols" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://plos.org/protocols?utm_medium=editorial-email&amp;utm_source=authorletters&amp;utm_campaign=protocols</a></p></section></article><article class="sub-article" id="pdig.0000982.r002"><section class="pmc-layout__citation font-secondary font-xs"><div>PLOS Digit Health. 2025 Aug 18;4(8):e0000982. doi: <a href="https://doi.org/10.1371/journal.pdig.0000982.r002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1371/journal.pdig.0000982.r002</a>
</div></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Author response to Decision Letter 1</h1></hgroup><ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="anp_a.d" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a.d" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="anp_a.d" class="d-panel p" style="display: none"><div class="notes p"><section id="historyfront-stub2" class="history"><p>Collection date 2025 Aug.</p></section></div></div>
<div id="clp_a.d" class="d-panel p" style="display: none"><div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div></div>
</div>
<div></div>
</div></section><section class="body sub-article-body"><hr class="headless">
<p>
<em>9 May 2025</em>
</p>
<section class="sm xbox font-sm" id="pdig.0000982.s012"><div class="caption p">
<span>Attachment</span><p>Submitted filename: <em>Response to Reviewers.pdf</em></p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12360575/bin/pdig.0000982.s012.pdf" data-ga-action="click_feat_suppl" class="usa-link">pdig.0000982.s012.pdf</a><sup> (346.4KB, pdf) </sup>
</div></div></section></section></article><article class="sub-article" id="pdig.0000982.r003"><section class="pmc-layout__citation font-secondary font-xs"><div>PLOS Digit Health. doi: <a href="https://doi.org/10.1371/journal.pdig.0000982.r003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1371/journal.pdig.0000982.r003</a>
</div></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Decision Letter 1</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kwak%20GH%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Gloria Hyunjung Kwak</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Gloria Hyunjung Kwak</span></h3>
<div>Section Editor</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kwak%20GH%22%5BAuthor%5D" class="usa-link"><span class="name western">Gloria Hyunjung Kwak</span></a>
</div>
</div>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a.e" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="clp_a.e" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a.e" class="d-panel p" style="display: none">
<h4 class="font-secondary">Roles</h4>
<div class="p">
<strong class="contrib"><span class="name western">Gloria Hyunjung Kwak</span></strong>: <span class="role">Section Editor</span>
</div>
</div>
<div id="clp_a.e" class="d-panel p" style="display: none">
<div>© 2025 Gloria Hyunjung Kwak</div>
<p>This is an open access article distributed under the terms of the <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Creative Commons Attribution License</a>, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div></div>
</div></section><section class="body sub-article-body"><hr class="headless">
<p>
<em>29 Jun 2025</em>
</p>
<p>PDIG-D-25-00125R1Characterizing the role of early life factors in machine learning-based multimorbidity risk predictionPLOS Digital Health Dear Dr. Dang, Thank you for submitting your manuscript to PLOS Digital Health. After careful consideration, we feel that it has merit but does not fully meet PLOS Digital Health's publication criteria as it currently stands. Therefore, we invite you to submit a revised version of the manuscript that addresses the points raised during the review process. Please submit your revised manuscript within 30 days Jul 29 2025 11:59PM. If you will need more time than this to complete your revisions, please reply to this message or contact the journal office at <span>digitalhealth@plos.org</span>. When you're ready to submit your revision, log on to <a href="https://www.editorialmanager.com/pdig/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.editorialmanager.com/pdig/</a> and select the 'Submissions Needing Revision' folder to locate your manuscript file. Please include the following items when submitting your revised manuscript:* A rebuttal letter that responds to each point raised by the editor and reviewer(s). You should upload this letter as a separate file labeled '<span class="text-underline">Response to Reviewers</span>'. This file does not need to include responses to any formatting updates and technical items listed in the 'Journal Requirements' section below.* A marked-up copy of your manuscript that highlights changes made to the original version. You should upload this as a separate file labeled '<span class="text-underline">Revised Manuscript with Track Changes</span>'.* An unmarked version of your revised paper without tracked changes. You should upload this as a separate file labeled '<span class="text-underline">Manuscript</span>'. If you would like to make changes to your financial disclosure, competing interests statement, or data availability statement, please make these updates within the submission form at the time of resubmission. Guidelines for resubmitting your figure files are available below the reviewer comments at the end of this letter. We look forward to receiving your revised manuscript. Kind regards, Gloria Hyunjung KwakSection EditorPLOS Digital Health Gloria Hyunjung KwakSection EditorPLOS Digital Health Leo Anthony CeliEditor-in-ChiefPLOS Digital Healthorcid.org/0000-0001-6712-6626   [Note: HTML markup is below. Please do not edit.] <strong>Reviewers' Comments:</strong> Reviewer's Responses to Questions</p>
<p>

<strong>Comments to the Author</strong>
</p>
<p>1. If the authors have adequately addressed your comments raised in a previous round of review and you feel that this manuscript is now acceptable for publication, you may indicate that here to bypass the “Comments to the Author” section, enter your conflict of interest statement in the “Confidential to Editor” section, and submit your "Accept" recommendation.</p>
<p>Reviewer #2: (No Response)</p>
<p>Reviewer #3: (No Response)</p>
<p>**********</p>
<p>2. Does this manuscript meet PLOS Digital Health’s <a href="https://journals.plos.org/digitalhealth/s/journal-information#loc-criteria-for-publication" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">publication criteria</a>? Is the manuscript technically sound, and do the data support the conclusions? The manuscript must describe methodologically and ethically rigorous research with conclusions that are appropriately drawn based on the data presented.</p>
<p>Reviewer #2: Yes</p>
<p>Reviewer #3: Yes</p>
<p>**********</p>
<p>3. Has the statistical analysis been performed appropriately and rigorously?</p>
<p>Reviewer #2: Yes</p>
<p>Reviewer #3: I don't know</p>
<p>**********</p>
<p>4. Have the authors made all data underlying the findings in their manuscript fully available (please refer to the Data Availability Statement at the start of the manuscript PDF file)?</p>
<p>The <a href="https://journals.plos.org/digitalhealth/s/data-availability" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PLOS Data policy</a> requires authors to make all data underlying the findings described in their manuscript fully available without restriction, with rare exception. The data should be provided as part of the manuscript or its supporting information, or deposited to a public repository. For example, in addition to summary statistics, the data points behind means, medians and variance measures should be available. If there are restrictions on publicly sharing data—e.g. participant privacy or use of data from a third party—those must be specified.</p>
<p>Reviewer #2: Yes</p>
<p>Reviewer #3: Yes</p>
<p>**********</p>
<p>5. Is the manuscript presented in an intelligible fashion and written in standard English?</p>
<p>PLOS Digital Health does not copyedit accepted manuscripts, so the language in submitted articles must be clear, correct, and unambiguous. Any typographical or grammatical errors should be corrected at revision, so please note any specific errors here.</p>
<p>Reviewer #2: Yes</p>
<p>Reviewer #3: Yes</p>
<p>**********</p>
<p>6. Review Comments to the Author</p>
<p>Please use the space provided to explain your answers to the questions above. You may also include additional comments for the author, including concerns about dual publication, research ethics, or publication ethics. (Please upload your review as an attachment if it exceeds 20,000 characters)</p>
<p>Reviewer #2: This revised paper addresses most of my concerns. Thanks to the</p>
<p>authors for their attention to comments. I have two outstanding</p>
<p>concerns:</p>
<p>1 I appreciate your change in language regarding "Value of</p>
<p>information", but I find the revision to be inadequate. "Value of</p>
<p>information" is an accepted phrase in the literature, involving</p>
<p>structured methods for determining the marginal value of information</p>
<p>in decision-making. See, for example, Heath, et al. 2017 (DOI</p>
<p>10.1177/0272989X176976) for an overview. As this paper does not use</p>
<p>any of those methods, calling this section a VOI analysis is</p>
<p>inappropriate. I suggest rephrasing this section, perhaps labeling</p>
<p>it as "analysis of contributions of predictive factors".</p>
<p>2. I appreciate the response regarding the graphs comparing the ROCs</p>
<p>for comorbid and non-comorbid groups, but I believe the groups are</p>
<p>still misleading. I suggest revising both graphs to have 0.5 as the</p>
<p>lowest point for both the x- and y-axes. This would provide the</p>
<p>desired 45 degree angle while avoiding both the compression that might</p>
<p>be associated with showing the whole range and the difficulties of</p>
<p>comparison created by having the range of values on the axes differ</p>
<p>for the two graphs.</p>
<p>Two minor points</p>
<p>1. Please provide version numbers for libraries mentioned in the</p>
<p>methods section.</p>
<p>2. Consider making the code available as open source.</p>
<p>Reviewer #3: The objective of this study is to study how machine learning can investigate the role of early-life factors in predicting cardiovascular disease and diabetes. This essay is an attempt to contribute to the predictive model literature by showing how to represent and integrate early-life factors in model development and of the importance of doing so. This is challenging due to the complexity of the relevant variables, their nonlinear interactions, and the assumptions a model might make. The hope is to provide insights that enhance the accuracy of existing risk prediction algorithms, specifically tailored for individuals with comorbid conditions.</p>
<p>I found the essay intriguing, very well written, and admirably detailed in its methodological description and findings. Quite frankly, this is a “big” article. I’m not trained in the data science dimensions that inform the study design so I must defer to other reviewers who can comment on their validity. I can say, though, that the effort to study early-life factors seems to me quite valuable, such that even if readers find material to criticize, the issues presented in this paper should be of value to the digital health community. Nevertheless, it would seem to me that the only persons with enough background to critique the methodology of the study would be data science specialists. Consequently, their review of this study is of quintessential importance.</p>
<p>Three drawbacks of the study are already mentioned by the authors. The first, that the model was normed using a United Kingdom database, with over 96% of participants being white. Another problem might be the accuracy of the data being entered, especially as it reports on early life factors. One is especially worried about data that are not only inaccurate but missing. A third factor is simply the plethora of variables programmed into the model. I suppose one can question whether that degree of detail is necessary or really contributes to the model’s predictive accuracy—although that question would obviously be addressed in future test applications of the model.</p>
<p>Ultimately, I found this essay very impressive in virtually every way. Should other reviewers find the methodology and findings acceptable, I would strongly endorse the article’s acceptance and eventual publication.</p>
<p>**********</p>
<p>7. PLOS authors have the option to publish the peer review history of their article (<a href="https://journals.plos.org/digitalhealth/s/editorial-and-peer-review-process#loc-peer-review-history" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">what does this mean?</a>). If published, this will include your full peer review and any attached files.</p>
<p><strong>Do you want your identity to be public for this peer review?</strong> If you choose “no”, your identity will remain anonymous but your review may still be made public.</p>
<p>For information about this choice, including consent withdrawal, please see our <a href="https://www.plos.org/privacy-policy" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Privacy Policy</a>.</p>
<p>Reviewer #2: <strong>Yes: </strong>Harry Hochheiser</p>
<p>Reviewer #3: No</p>
<p>**********</p>
<p> [NOTE: If reviewer comments were submitted as an attachment file, they will be attached to this email and accessible via the submission site. Please log into your account, locate the manuscript record, and check for the action link "View Attachments". If this link does not appear, there are no attachment files.] <strong>Figure resubmission:</strong> While revising your submission, please upload your figure files to the Preflight Analysis and Conversion Engine (PACE) digital diagnostic tool, <a href="https://pacev2.apexcovantage.com/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://pacev2.apexcovantage.com/</a>. PACE helps ensure that figures meet PLOS requirements. To use PACE, you must first register as a user. Registration is free. Then, login and navigate to the UPLOAD tab, where you will find detailed instructions on how to use the tool. If you encounter any issues or have any questions when using PACE, please email PLOS at <span>figures@plos.org</span>. Please note that Supporting Information files do not need this step. If there are other versions of figure files still present in your submission file inventory at resubmission, please replace them with the PACE-processed versions. <strong>Reproducibility:</strong> To enhance the reproducibility of your results, we recommend that authors of applicable studies deposit laboratory protocols in protocols.io, where a protocol can be assigned its own identifier (DOI) such that it can be cited independently in the future. Additionally, PLOS ONE offers an option to publish peer-reviewed clinical study protocols. Read more information on sharing protocols at <a href="https://plos.org/protocols?utm_medium=editorial-email&amp;utm_source=authorletters&amp;utm_campaign=protocols" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://plos.org/protocols?utm_medium=editorial-email&amp;utm_source=authorletters&amp;utm_campaign=protocols</a></p></section></article><article class="sub-article" id="pdig.0000982.r004"><section class="pmc-layout__citation font-secondary font-xs"><div>PLOS Digit Health. 2025 Aug 18;4(8):e0000982. doi: <a href="https://doi.org/10.1371/journal.pdig.0000982.r004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1371/journal.pdig.0000982.r004</a>
</div></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Author response to Decision Letter 2</h1></hgroup><ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="anp_a.f" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a.f" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="anp_a.f" class="d-panel p" style="display: none"><div class="notes p"><section id="historyfront-stub4" class="history"><p>Collection date 2025 Aug.</p></section></div></div>
<div id="clp_a.f" class="d-panel p" style="display: none"><div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div></div>
</div>
<div></div>
</div></section><section class="body sub-article-body"><hr class="headless">
<p>
<em>3 Jul 2025</em>
</p>
<section class="sm xbox font-sm" id="pdig.0000982.s013"><div class="caption p">
<span>Attachment</span><p>Submitted filename: <em>Response_to_Reviewers_auresp_2.pdf</em></p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12360575/bin/pdig.0000982.s013.pdf" data-ga-action="click_feat_suppl" class="usa-link">pdig.0000982.s013.pdf</a><sup> (323.9KB, pdf) </sup>
</div></div></section></section></article><article class="sub-article" id="pdig.0000982.r005"><section class="pmc-layout__citation font-secondary font-xs"><div>PLOS Digit Health. doi: <a href="https://doi.org/10.1371/journal.pdig.0000982.r005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1371/journal.pdig.0000982.r005</a>
</div></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Decision Letter 2</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kwak%20GH%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Gloria Hyunjung Kwak</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Gloria Hyunjung Kwak</span></h3>
<div>Section Editor</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kwak%20GH%22%5BAuthor%5D" class="usa-link"><span class="name western">Gloria Hyunjung Kwak</span></a>
</div>
</div>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a.g" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="clp_a.g" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a.g" class="d-panel p" style="display: none">
<h4 class="font-secondary">Roles</h4>
<div class="p">
<strong class="contrib"><span class="name western">Gloria Hyunjung Kwak</span></strong>: <span class="role">Section Editor</span>
</div>
</div>
<div id="clp_a.g" class="d-panel p" style="display: none">
<div>© 2025 Gloria Hyunjung Kwak</div>
<p>This is an open access article distributed under the terms of the <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Creative Commons Attribution License</a>, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div></div>
</div></section><section class="body sub-article-body"><hr class="headless">
<p>
<em>23 Jul 2025</em>
</p>
<p>Characterizing the role of early life factors in machine learning-based multimorbidity risk prediction</p>
<p>PDIG-D-25-00125R2</p>
<p>Dear Dr. Dang,</p>
<p>We're pleased to inform you that your manuscript has been judged scientifically suitable for publication and will be formally accepted for publication once it meets all outstanding technical requirements.</p>
<p>Within one week, you'll receive an e-mail detailing the required amendments. When these have been addressed, you'll receive a formal acceptance letter and your manuscript will be scheduled for publication.</p>
<p>An invoice for payment will follow shortly after the formal acceptance. To ensure an efficient process, please log into Editorial Manager at <a href="https://www.editorialmanager.com/pdig/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.editorialmanager.com/pdig/</a> click the 'Update My Information' link at the top of the page, and double check that your user information is up-to-date. For billing related questions, please contact billing support at <a href="https://plos.my.site.com/s/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://plos.my.site.com/s/</a>.</p>
<p>If your institution or institutions have a press office, please notify them about your upcoming paper to help maximize its impact. If they'll be preparing press materials, please inform our press team as soon as possible -- no later than 48 hours after receiving the formal acceptance. Your manuscript will remain under strict press embargo until 2 pm Eastern Time on the date of publication. For more information, please contact <span>digitalhealth@plos.org</span>.</p>
<p>Kind regards,</p>
<p>Sarah Mayo</p>
<p>Staff Admin</p>
<p>PLOS Digital Health</p>
<p>Additional Editor Comments (optional):</p>
<p>Reviewers' comments:</p></section></article><article class="sub-article" id="_ad93_"><section class="pmc-layout__citation font-secondary font-xs"><div></div></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Associated Data</h1></hgroup><ul class="d-buttons inline-list"></ul>
<div class="d-panels font-secondary-light"></div>
<div></div>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
</div></section><section class="body sub-article-body"><section id="_adsm93_" lang="en" class="supplementary-materials"><h2 class="pmc_sec_title">Supplementary Materials</h2>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="caption p">
<span>S1 Table. Summary of all UKB input variables used in the study.</span><p>(PDF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12360575/bin/pdig.0000982.s001.pdf" data-ga-action="click_feat_suppl" class="usa-link">pdig.0000982.s001.pdf</a><sup> (76.7KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="caption p">
<span>S1 Fig. Flow diagram of participant stratification based on early-life factors (ELFs), detailing CVD and diabetes outcomes in non-comorbid and comorbid subgroups.</span><p>(PDF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12360575/bin/pdig.0000982.s002.pdf" data-ga-action="click_feat_suppl" class="usa-link">pdig.0000982.s002.pdf</a><sup> (64KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="caption p">
<span>S2 Table. Permutation-based variable ranking for predicting CVD in non-comorbid and comorbid groups.</span><p>(PDF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12360575/bin/pdig.0000982.s003.pdf" data-ga-action="click_feat_suppl" class="usa-link">pdig.0000982.s003.pdf</a><sup> (79.4KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material4_reqid_"><div class="caption p">
<span>S2 Fig. Comprehensive overview of participant profiles for the final study cohort with CVD outcomes: (a) Demographic and socioeconomic distribution, illustrating the population sample’s diversity.</span><p>(b) Proportion of early-life factors, underlining their frequency and potential impact on later CVD health outcomes.</p>
<p>(PDF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12360575/bin/pdig.0000982.s004.pdf" data-ga-action="click_feat_suppl" class="usa-link">pdig.0000982.s004.pdf</a><sup> (75.6KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material5_reqid_"><div class="caption p">
<span>S3 Table. Permutation-based variable ranking for predicting diabetes in non-comorbid and comorbid groups.</span><p>(PDF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12360575/bin/pdig.0000982.s005.pdf" data-ga-action="click_feat_suppl" class="usa-link">pdig.0000982.s005.pdf</a><sup> (67.3KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material6_reqid_"><div class="caption p">
<span>S3 Fig. Comprehensive overview of participant profiles for the final study cohort with diabetes outcomes: (a) Demographic and socioeconomic distribution, illustrating the population sample’s diversity.</span><p>(b) Proportion of early-life factors, underlining their frequency and potential impact on later diabetes health outcomes.</p>
<p>(PDF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12360575/bin/pdig.0000982.s006.pdf" data-ga-action="click_feat_suppl" class="usa-link">pdig.0000982.s006.pdf</a><sup> (75.5KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material7_reqid_"><div class="caption p">
<span>S4 Table. Differential top features by effect size: comorbid vs. non-comorbid.</span><p>The top 30 features are ranked by their effect size for CVD.</p>
<p>(PDF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12360575/bin/pdig.0000982.s007.pdf" data-ga-action="click_feat_suppl" class="usa-link">pdig.0000982.s007.pdf</a><sup> (88KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material8_reqid_"><div class="caption p">
<span>S4 Fig. Predictive ability of individual variables for CVD and diabetes prediction in comorbid and non-comorbid groups.</span><p>Complements <a href="#pdig.0000982.g003" class="usa-link">Fig 3</a> by showing the full distribution of AUC-ROC values for all features.</p>
<p>(PDF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12360575/bin/pdig.0000982.s008.pdf" data-ga-action="click_feat_suppl" class="usa-link">pdig.0000982.s008.pdf</a><sup> (98.9KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material9_reqid_"><div class="caption p">
<span>S5 Table. Differential top features by effect size: comorbid vs. non-comorbid.</span><p>The top 30 features are ranked by their effect size for diabetes.</p>
<p>(PDF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12360575/bin/pdig.0000982.s009.pdf" data-ga-action="click_feat_suppl" class="usa-link">pdig.0000982.s009.pdf</a><sup> (87.4KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material10_reqid_"><div class="caption p">
<span>S5 Fig. Shap scores of the top 30 influential variables in CVD prediction model for non-comorbid and comorbid groups.</span><p>Variables marked with an asterisk ( ) are those currently employed in current risk assessment models.</p>
<p>(PDF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12360575/bin/pdig.0000982.s010.pdf" data-ga-action="click_feat_suppl" class="usa-link">pdig.0000982.s010.pdf</a><sup> (48.5KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material11_reqid_"><div class="caption p">
<span>S6 Fig. Shap scores of the top 30 influential variables in diabetes prediction model for non-comorbid and comorbid groups.</span><p>Variables marked with an asterisk ( ) are those currently employed in current risk assessment models.</p>
<p>(PDF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12360575/bin/pdig.0000982.s011.pdf" data-ga-action="click_feat_suppl" class="usa-link">pdig.0000982.s011.pdf</a><sup> (48.3KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material12_reqid_"><div class="caption p">
<span>Attachment</span><p>Submitted filename: <em>Response to Reviewers.pdf</em></p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12360575/bin/pdig.0000982.s012.pdf" data-ga-action="click_feat_suppl" class="usa-link">pdig.0000982.s012.pdf</a><sup> (346.4KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material13_reqid_"><div class="caption p">
<span>Attachment</span><p>Submitted filename: <em>Response_to_Reviewers_auresp_2.pdf</em></p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12360575/bin/pdig.0000982.s013.pdf" data-ga-action="click_feat_suppl" class="usa-link">pdig.0000982.s013.pdf</a><sup> (323.9KB, pdf) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h2 class="pmc_sec_title">Data Availability Statement</h2>
<p>Data from this study are from the UK Biobank (UKB), available upon approved application. Researchers can register and request access at <a href="https://www.ukbiobank.ac.uk/enable-your-research/register" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ukbiobank.ac.uk/enable-your-research/register</a>. The results for UKB in this study will be returned to the UK Biobank within 6 months since publication, as required.</p></section></section></article><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from PLOS Digital Health are provided here courtesy of <strong>PLOS</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1371/journal.pdig.0000982"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/pdig.0000982.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (5.3 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12360575/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12360575/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12360575%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12360575/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12360575/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12360575/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40825013/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12360575/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40825013/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12360575/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12360575/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="0syd2Ajc6teDphxTnX7nBBVpwPOZPlKhQBr5ho4EUWPKwmm83iYmWsNUixcN45ZV">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
        <script type="module" crossorigin="" src="/static/assets/math-574fdcc6.js"></script>
    
    

    </body>
</html>
