
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Early lineage segregation of primary myotubes from secondary myotubes and adult muscle stem cells - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="152539AC8AF201630539AC000F7A5C39.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="ncomms">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12374003/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Nature Communications">
<meta name="citation_title" content="Early lineage segregation of primary myotubes from secondary myotubes and adult muscle stem cells">
<meta name="citation_author" content="Gauthier Toulouse">
<meta name="citation_author_institution" content="Université Claude Bernard Lyon1, MeLiS Laboratory, CNRS, INSERM, Lyon, France">
<meta name="citation_author" content="William Jarassier">
<meta name="citation_author_institution" content="Université Claude Bernard Lyon1, PGNM Laboratory, CNRS, INSERM, Lyon, France">
<meta name="citation_author" content="Sabrina Jagot">
<meta name="citation_author_institution" content="Université Claude Bernard Lyon1, MeLiS Laboratory, CNRS, INSERM, Lyon, France">
<meta name="citation_author" content="Valérie Morin">
<meta name="citation_author_institution" content="Université Claude Bernard Lyon1, MeLiS Laboratory, CNRS, INSERM, Lyon, France">
<meta name="citation_author" content="Fabien Le Grand">
<meta name="citation_author_institution" content="Université Claude Bernard Lyon1, MeLiS Laboratory, CNRS, INSERM, Lyon, France">
<meta name="citation_author_institution" content="Université Claude Bernard Lyon1, PGNM Laboratory, CNRS, INSERM, Lyon, France">
<meta name="citation_author" content="Christophe Marcelle">
<meta name="citation_author_institution" content="Université Claude Bernard Lyon1, MeLiS Laboratory, CNRS, INSERM, Lyon, France">
<meta name="citation_author_institution" content="Monash University, Australian Regenerative Medicine Institute, Clayton, VIC Australia">
<meta name="citation_publication_date" content="2025 Aug 22">
<meta name="citation_volume" content="16">
<meta name="citation_firstpage" content="7858">
<meta name="citation_doi" content="10.1038/s41467-025-61767-1">
<meta name="citation_pmid" content="40846698">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12374003/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12374003/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12374003/pdf/41467_2025_Article_61767.pdf">
<meta name="description" content="Myogenesis in amniotes occurs in two waves. Primary myotubes express slow myosin (often with fast myosin) and likely act as scaffolds for secondary myotubes, which express only fast myosin. The embryonic origins and relationships of these lineages, ...">
<meta name="og:title" content="Early lineage segregation of primary myotubes from secondary myotubes and adult muscle stem cells">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Myogenesis in amniotes occurs in two waves. Primary myotubes express slow myosin (often with fast myosin) and likely act as scaffolds for secondary myotubes, which express only fast myosin. The embryonic origins and relationships of these lineages, ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12374003/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12374003">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1038/s41467-025-61767-1"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/41467_2025_Article_61767.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12374003%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12374003/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12374003/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12374003/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-ncomms.gif" alt="Nature Communications logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Nature Communications" title="Link to Nature Communications" shape="default" href="http://www.nature.com/ncomms/index.html" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Nat Commun</button></div>. 2025 Aug 22;16:7858. doi: <a href="https://doi.org/10.1038/s41467-025-61767-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1038/s41467-025-61767-1</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Nat%20Commun%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nat%20Commun%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Nat%20Commun%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Nat%20Commun%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Early lineage segregation of primary myotubes from secondary myotubes and adult muscle stem cells</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Toulouse%20G%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Gauthier Toulouse</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Gauthier Toulouse</span></h3>
<div class="p">
<sup>1</sup>Université Claude Bernard Lyon1, MeLiS Laboratory, CNRS, INSERM, Lyon, France </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Toulouse%20G%22%5BAuthor%5D" class="usa-link"><span class="name western">Gauthier Toulouse</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Jarassier%20W%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">William Jarassier</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">William Jarassier</span></h3>
<div class="p">
<sup>2</sup>Université Claude Bernard Lyon1, PGNM Laboratory, CNRS, INSERM, Lyon, France </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Jarassier%20W%22%5BAuthor%5D" class="usa-link"><span class="name western">William Jarassier</span></a>
</div>
</div>
<sup>2,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Jagot%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Sabrina Jagot</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Sabrina Jagot</span></h3>
<div class="p">
<sup>1</sup>Université Claude Bernard Lyon1, MeLiS Laboratory, CNRS, INSERM, Lyon, France </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Jagot%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Sabrina Jagot</span></a>
</div>
</div>
<sup>1,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Morin%20V%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Valérie Morin</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Valérie Morin</span></h3>
<div class="p">
<sup>1</sup>Université Claude Bernard Lyon1, MeLiS Laboratory, CNRS, INSERM, Lyon, France </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Morin%20V%22%5BAuthor%5D" class="usa-link"><span class="name western">Valérie Morin</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Le%20Grand%20F%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Fabien Le Grand</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Fabien Le Grand</span></h3>
<div class="p">
<sup>1</sup>Université Claude Bernard Lyon1, MeLiS Laboratory, CNRS, INSERM, Lyon, France </div>
<div class="p">
<sup>2</sup>Université Claude Bernard Lyon1, PGNM Laboratory, CNRS, INSERM, Lyon, France </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Le%20Grand%20F%22%5BAuthor%5D" class="usa-link"><span class="name western">Fabien Le Grand</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Marcelle%20C%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Christophe Marcelle</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Christophe Marcelle</span></h3>
<div class="p">
<sup>1</sup>Université Claude Bernard Lyon1, MeLiS Laboratory, CNRS, INSERM, Lyon, France </div>
<div class="p">
<sup>3</sup>Monash University, Australian Regenerative Medicine Institute, Clayton, VIC Australia </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Marcelle%20C%22%5BAuthor%5D" class="usa-link"><span class="name western">Christophe Marcelle</span></a>
</div>
</div>
<sup>1,</sup><sup>3,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>Université Claude Bernard Lyon1, MeLiS Laboratory, CNRS, INSERM, Lyon, France </div>
<div id="Aff2">
<sup>2</sup>Université Claude Bernard Lyon1, PGNM Laboratory, CNRS, INSERM, Lyon, France </div>
<div id="Aff3">
<sup>3</sup>Monash University, Australian Regenerative Medicine Institute, Clayton, VIC Australia </div>
<div class="author-notes p">
<div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div>
<div class="fn" id="_eqcntrb93pmc__">
<sup>#</sup><p class="display-inline">Contributed equally.</p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2024 Nov 8; Accepted 2025 Jun 27; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12374003  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40846698/" class="usa-link">40846698</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<p id="Par1">Myogenesis in amniotes occurs in two waves. Primary myotubes express slow myosin (often with fast myosin) and likely act as scaffolds for secondary myotubes, which express only fast myosin. The embryonic origins and relationships of these lineages, and their connection to satellite cells, remain unknown. Here, we combine a TCF-LEF/β-catenin signaling reporter with precise in vivo electroporation in avian embryos to trace limb muscle progenitors from early migration to fetal stages. We identify two distinct progenitor populations that coexist from the onset: reporter-positive cells give rise exclusively to primary myotubes, while reporter-negative cells generate secondary myotubes and satellite cells. We also reveal a previously unrecognized role for TCF-LEF/β-catenin signaling in spatially organizing the primary lineage via Cxcr4-mediated control of myoblast migration. These findings redefine the developmental origin of myogenic lineages, resolve a longstanding question in muscle biology, and provide a molecular framework for investigating how muscle fiber diversity emerges and how distinct lineages contribute to the functional specialization of skeletal muscle.</p>
<section id="kwd-group1" class="kwd-group"><p><strong>Subject terms:</strong> Musculoskeletal development, Cell lineage, Embryology</p></section></section><section class="abstract" id="Abs2"><hr class="headless">
<p id="Par2">This study shows that two distinct progenitor populations build embryonic muscle in birds—one forms primary fibers via Wnt signaling, the other gives rise to secondary fibers and satellite cells—and that this dual origin is conserved in humans.</p></section><section id="Sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p id="Par3">The early steps of skeletal muscle morphogenesis have been extensively studied, leading to the emergence of detailed models of the cellular and molecular mechanisms regulating the first steps of myogenesis<sup><a href="#CR1" class="usa-link" aria-describedby="CR1">1</a>–<a href="#CR8" class="usa-link" aria-describedby="CR8">8</a></sup>. One of the lesser-known steps of myogenesis is the generation of so-called “primary” and “secondary” myotubes. Microscopists studying muscle formation in vertebrate (birds and rodents) embryos half a century ago have identified two distinct waves of myogenesis. The first, taking place during the embryonic phase of development, generates what they have named primary myotubes. The second wave, observed in fetal life, results in the emergence of secondary myotubes. Distinct morphological characteristics separate primary and secondary myotubes<sup><a href="#CR9" class="usa-link" aria-describedby="CR9">9</a>,<a href="#CR10" class="usa-link" aria-describedby="CR10">10</a></sup>. Primary myotubes are initially large and extend from tendon to tendon of embryonic muscles. Secondary myotubes are initially small and tightly organized around primary myotube<sup><a href="#CR9" class="usa-link" aria-describedby="CR9">9</a>,<a href="#CR11" class="usa-link" aria-describedby="CR11">11</a>,<a href="#CR12" class="usa-link" aria-describedby="CR12">12</a></sup>. Such an association led to the widely held but unproven assumption that primary myotubes may serve as an organizing scaffold for the morphogenesis of the secondary lineage<sup><a href="#CR10" class="usa-link" aria-describedby="CR10">10</a></sup>. Primary myotubes form before motoneurons reach the muscle masses, and they develop normally until birth despite denervation or inhibition of neural activity<sup><a href="#CR13" class="usa-link" aria-describedby="CR13">13</a>,<a href="#CR14" class="usa-link" aria-describedby="CR14">14</a></sup>. Secondary myotubes, on the contrary, form after motoneurons have reached the muscle masses and most authors agree that the emergence of secondary myotubes is dependent on innervation<sup><a href="#CR13" class="usa-link" aria-describedby="CR13">13</a>–<a href="#CR18" class="usa-link" aria-describedby="CR18">18</a></sup>. Biochemical properties also distinguish primary and secondary myotubes. In rodents and birds, primary myotubes are characterized by their expression of slow myosin (in variable combination with embryonic/fast myosin), whereas secondary myotubes express exclusively embryonic fast myosin<sup><a href="#CR19" class="usa-link" aria-describedby="CR19">19</a>–<a href="#CR22" class="usa-link" aria-describedby="CR22">22</a></sup>. Crucially, this initial, genetically encoded, fiber type pattern is plastic and modulated throughout life by factors such as innervation, exercise, and hormonal influences<sup><a href="#CR10" class="usa-link" aria-describedby="CR10">10</a></sup>.</p>
<p id="Par4">Studies on the cellular origin of primary and secondary myotubes have identified three successive waves of myoblast populations: embryonic, fetal and adult (or satellite cell) myoblasts<sup><a href="#CR10" class="usa-link" aria-describedby="CR10">10</a></sup>. Embryonic myoblasts are isolated from muscle masses during the first week of development in chicken and up to the second week in mouse; fetal myoblasts thereafter<sup><a href="#CR23" class="usa-link" aria-describedby="CR23">23</a></sup>. The adult/satellite cell population settles under the myofibers’ basal lamina shortly before hatching or birth and serves as a reserve stem cell population for muscle growth and repair after birth and into adulthood. In vitro, embryonic and fetal myoblasts exhibit intrinsic differences in fusion ability, proliferation and response to growth factors<sup><a href="#CR10" class="usa-link" aria-describedby="CR10">10</a>,<a href="#CR24" class="usa-link" aria-describedby="CR24">24</a>–<a href="#CR28" class="usa-link" aria-describedby="CR28">28</a></sup>. Furthermore, transcriptomics analyses of these two populations confirmed that they possess distinct genetic programs<sup><a href="#CR29" class="usa-link" aria-describedby="CR29">29</a>,<a href="#CR30" class="usa-link" aria-describedby="CR30">30</a></sup>.</p>
<p id="Par5">Nonetheless, the lineage connection between these two populations of myoblasts remains a subject of unresolved matter<sup><a href="#CR10" class="usa-link" aria-describedby="CR10">10</a>,<a href="#CR31" class="usa-link" aria-describedby="CR31">31</a></sup>. Limb grafting in chicken embryos suggested that two successive waves of myogenic progenitors, each with distinct characteristics, migrate into the limb bud to form primary and secondary myotubes<sup><a href="#CR32" class="usa-link" aria-describedby="CR32">32</a></sup>. In vitro studies have further complicated the issue, yielding conflicting hypotheses that propose either a single evolving population, transitioning from displaying embryonic to fetal characteristics<sup><a href="#CR28" class="usa-link" aria-describedby="CR28">28</a>,<a href="#CR29" class="usa-link" aria-describedby="CR29">29</a>,<a href="#CR33" class="usa-link" aria-describedby="CR33">33</a></sup> or two distinct populations present in the limb from the onset of its formation<sup><a href="#CR10" class="usa-link" aria-describedby="CR10">10</a>,<a href="#CR25" class="usa-link" aria-describedby="CR25">25</a>,<a href="#CR34" class="usa-link" aria-describedby="CR34">34</a></sup>.</p>
<p id="Par6">Here, we have analyzed the developmental path that muscle progenitors of chicken embryos follow from the time of their migration away from the wing somites into the limb bud mesenchyme until late fetal life, shortly before hatching. Using the in vivo electroporation technique to specifically target the muscle progenitor population and a transcriptional reporter to monitor TCF-LEF/β-catenin dependent signaling activity in real time, we have discovered that TCF-LEF/β-catenin activation distinguishes positive and negative muscle progenitors within the early growing wing bud. Biochemical analyses showed that the reporter’s activity is restricted to early stage (PAX3<sup>+</sup>/PAX7<sup>+</sup>/MYF5<sup>+</sup>/MYOD<sup>-</sup>) of myoblast differentiation and confined to the early phases of embryonic development, while absent thereafter until hatching. We designed a novel, dynamic Tet-on-based tool (named TCF-Trace) to follow the fate of reporter-positive myogenic progenitors. This demonstrated that all reporter-positive myoblasts readily differentiated into primary myotubes, while the reporter-negative myoblasts gave rise to the late-forming, secondary myofibers, and also to satellite cells. Next, we demonstrated that TCF-LEF/β-catenin signaling does not regulate cell proliferation and differentiation but rather plays a crucial role in the spatial distribution of limb primary muscle progenitors. Transcriptomic profiling of the two progenitor types in chicken confirmed the existence of two distinct co-existing lineages. Remarkably, strong similarities between species suggest a conserved developmental program, with Wnt signaling tightly associated with primary myogenesis in both chicken and human. Finally, we show that Wnt signaling likely influences progenitor distribution via transcriptional control of the migration regulator CXCR4.</p>
<p id="Par7">These discoveries not only address a long-standing question in muscle biology but also provide a crucial molecular gateway for understanding how these early developmental processes contribute to the fiber type diversity of muscle fibers.</p></section><section id="Sec2"><h2 class="pmc_sec_title">Results</h2>
<section id="Sec3"><h3 class="pmc_sec_title">TCF-LEF/β-catenin dependent signaling is restricted to early limb muscle development</h3>
<p id="Par8">We recently generated avian transgenic lines that revealed unexpected features of TCF/β-catenin-responding cells and tissues<sup><a href="#CR35" class="usa-link" aria-describedby="CR35">35</a></sup>. Among those tissues, we were intrigued by the presence of cells activating the pathway within the limb muscles masses. We decided to investigate further this observation. All limb muscles originate from a single origin, the lateral part of the somite (named the ventro-lateral lip - VLL)<sup><a href="#CR1" class="usa-link" aria-describedby="CR1">1</a></sup>, which can be easily targeted using the electroporation technique<sup><a href="#CR36" class="usa-link" aria-describedby="CR36">36</a></sup>. This contrasts with trunk muscles that originate from various regions of somites<sup><a href="#CR5" class="usa-link" aria-describedby="CR5">5</a>,<a href="#CR6" class="usa-link" aria-describedby="CR6">6</a></sup>. The VLL therefore constitutes an attractive experimental model to study all steps of muscle formation and patterning (Supplementary Fig. <a href="#MOESM1" class="usa-link">1</a>). To verify that TCF/β-catenin-dependent signaling was active in the myogenic lineage, we have electroporated the transcriptional reporter of TCF/β-catenin-dependent signaling used previously<sup><a href="#CR35" class="usa-link" aria-describedby="CR35">35</a></sup> in the VLL of somites 16–21 of E2.5 chicken embryos<sup><a href="#CR1" class="usa-link" aria-describedby="CR1">1</a>,<a href="#CR4" class="usa-link" aria-describedby="CR4">4</a></sup>, from which limb muscle progenitors originate (Fig. <a href="#Fig1" class="usa-link">1a</a>; Supplementary Fig. <a href="#MOESM1" class="usa-link">1</a>). The transcriptional reporter, named 16TF-VNP, comprises sixteen TCF/LEF binding sites upstream of a destabilized, nuclear Venus fluorescent protein, surrounded by translational enhancers to boost its activity (see “Material and Methods”; Fig. <a href="#Fig1" class="usa-link">1b</a>). This and all electroporated constructs used in this study were cloned into expression vectors containing two Tol2 sequences (T2: transposable elements from medaka fish<sup><a href="#CR37" class="usa-link" aria-describedby="CR37">37</a></sup>) surrounding the entire construct to allow their stable integration in the chicken genome in the presence of transposase, provided by a co-electroporated transposase plasmid (CAGGS-Transposase). Since electroporation leads to the mosaic transfection of the targeted cell population, the reporter was co-electroporated with a ubiquitously-expressed fluorescent marker to identify and analyze electroporated cells individually. We followed the activity of the TCF/β-catenin-responding cells in the myogenic lineage throughout embryonic and fetal development.</p>
<figure class="fig xbox font-sm" id="Fig1"><h4 class="obj_head">Fig. 1. TCF-LEF/β-catenin dependent signaling is restricted to early limb muscle development.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12374003_41467_2025_61767_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dd7d/12374003/5169835fdd20/41467_2025_61767_Fig1_HTML.jpg" loading="lazy" id="d33e462" height="1000" width="699" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Brachial somites were electroporated at E2.5 and embryos were analyzed at indicated timepoints. <strong>b</strong> Representation of the transcriptional reporter (16TF-VNP) used to monitor TCF/LEF/β-catenin dependent signaling. 16 TCF/LEF binding sites (BS) were placed upstream of a minimal promoter (miniP) driving the expression of a nuclear, destabilized Venus fluorescent protein (VNP); three translational enhancers were added (IVS, Syn21 and p10) to boost protein production. <strong>c</strong>–<strong>e</strong> Dorsal view of confocal stacks of brachial somites electroporated with a ubiquitously expressed dTomato and the 16TF-VNP, observed at E3 and E3.5. E3 VLL timepoint represents the epithelial electroporated cells still located in the VLL at E3 while E3 Migrat. represents the mesenchymal, migrating electroporated cells at E3. Somite borders are indicated by dotted lines; <strong>d</strong> is an enlargement of (<strong>c</strong>). <strong>f</strong>–<strong>i</strong> Dorsal views of confocal stacks of limb buds observed between E4.5 and E7.5, electroporated with either a ubiquitous TagBFP (<strong>f</strong>–<strong>h</strong>) or a ubiquitous nuclear dTomato (<strong>i</strong>), together with the 16TF-VNP. Arrowheads in (<strong>i</strong>) indicate the last remaining 16TF-VNP<sup>+</sup> cells present at E7.5. <strong>j</strong> Transversal section of E9.5 limb bud electroporated with a ubiquitous nuclear dTomato together with the 16TF-VNP. <strong>k</strong> Quantification of the percentage of 16TF-VNP<sup>+</sup> cells between E3 and E6.5, each dot represents a limb bud (<em>n</em> = 5, 5, 4, 9, 5 and 6 for each time point, respectively). <strong>l</strong> Schematic representation of 16TF-VNP activity in the myogenic lineage during development. Box plots show the median (center line), 25th and 75th percentiles (box limits), and whiskers extending to the smallest and largest values within 1.5 x IQR. Scale bars: 100 μm (<strong>c</strong>–<strong>f</strong>) or 200 μm (<strong>g</strong><strong>–</strong><strong>j</strong>).</p></figcaption></figure><p id="Par9">At E3, i.e. twelve hours after electroporation, the migration of progenitors emanating from the VLL has started (Fig. <a href="#Fig1" class="usa-link">1c, d</a>). At that stage, two distinct locations of TCF/β-catenin-responding cells were observed: a strong expression in a majority (66%) of electroporated epithelial cells located within the VLL and a weaker expression in a minority (21%) of migrating, electroporated cells exiting from the VLL (Fig. <a href="#Fig1" class="usa-link">1c, k</a>; Supplementary Fig. <a href="#MOESM1" class="usa-link">2a, b</a>). The expression of PAX7 in all (16TF-VNP<sup>+</sup> and 16TF-VNP<sup>-</sup>) electroporated migrating cells confirmed that these are bona fide muscle progenitors (Supplementary Fig. <a href="#MOESM1" class="usa-link">2a–c</a>). Half a day later, at E3.5, as all muscle progenitors have exited the VLL<sup><a href="#CR38" class="usa-link" aria-describedby="CR38">38</a></sup>, 45% of electroporated muscle progenitors were strongly positive for the 16TF-VNP reporter (Fig. <a href="#Fig1" class="usa-link">1e, k</a>). During the following three days of development (E4.5 to E6.5) this proportion and level of expression remained relatively stable, in about 50–55% of electroporated cells (Fig. <a href="#Fig1" class="usa-link">1f–h, k</a>). During this time period, 16TF-VNP<sup>+</sup> cells were distributed among 16TF-VNP<sup>-</sup>cells, with an increasing tendency towards a localization at the distal end of the muscle masses as development proceeded (Fig. <a href="#Fig1" class="usa-link">1f–h</a>). 16TF-VNP<sup>+</sup> cells were evenly distributed along the dorso-ventral axis of muscles and were also present in the ventral muscle mass of the limb bud (Supplementary Fig. <a href="#MOESM1" class="usa-link">2d–f</a>). A sharp decrease in the reporter’s activity was however observed on the next day, at E7.5, as 16TF-VNP<sup>+</sup> cells became sparse and were confined mainly to the distal-most part of the muscle masses (Fig. <a href="#Fig1" class="usa-link">1I</a>, arrowheads).</p>
<p id="Par10">We then performed long-term analyses of the reporter’s activity. Embryos electroporated at E2.5 were analyzed on transversal and longitudinal sections of limbs collected at E9.5, E12.5, E14.5, E16.5 and E18.5. Despite a massive labeling of the muscle masses by the ubiquitously expressed electroporation marker, from E9.5 - E18.5, we did not detect any 16TF-VNP<sup>+</sup> cells at any of the analyzed developmental stages (Fig. <a href="#Fig1" class="usa-link">1j</a>; Supplementary Fig. <a href="#MOESM1" class="usa-link">3</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">4</a>).</p>
<p id="Par11">These data demonstrate that TCF/LEF transcription in the muscle progenitor population is dynamic (see Fig. <a href="#Fig1" class="usa-link">1l</a>). An important feature of the reporter’s activity is that it was observed in about 50% of electroporated muscle progenitors from E3.5 until E6.5. During that time period, muscle progenitors differentiated into many multinucleated muscle fibers, the first visible at around E5/E5.5<sup><a href="#CR36" class="usa-link" aria-describedby="CR36">36</a></sup>. As muscle differentiation progressed and distinct muscle bundles emerged, the reporter activity sharply dropped and was kept silent until hatching.</p></section><section id="Sec4"><h3 class="pmc_sec_title">TCF-LEF/β-catenin positive cells are PAX3<sup>+</sup>/PAX7<sup>+</sup>/MYF5<sup>+</sup>/MYOD<sup>-</sup> muscle progenitors</h3>
<p id="Par12">We then investigated the myogenic differentiation state of TCF-LEF/β-catenin activating cells. During myogenesis, muscle progenitors sequentially express different transcription factors that correspond to different phases of myogenic commitment<sup><a href="#CR8" class="usa-link" aria-describedby="CR8">8</a>,<a href="#CR39" class="usa-link" aria-describedby="CR39">39</a>,<a href="#CR40" class="usa-link" aria-describedby="CR40">40</a></sup>. In the mouse and chicken embryos, the proliferative muscle progenitor population comprises PAX7<sup>+</sup>MYF5<sup>−</sup> slow-dividing and PAX7<sup>+</sup>/MYF5<sup>+</sup> fast-dividing cells<sup><a href="#CR41" class="usa-link" aria-describedby="CR41">41</a></sup>. MYOD expression signals the exit of progenitors from cell cycle, and MYOG expression corresponds to terminally differentiated, pre-fusing muscle cells<sup><a href="#CR42" class="usa-link" aria-describedby="CR42">42</a></sup>.</p>
<p id="Par13">In birds, PAX7 and PAX3 proteins are co-expressed in limb myogenic progenitors, from the moment they exit the VLL and migrate into the limb mesenchyme<sup><a href="#CR43" class="usa-link" aria-describedby="CR43">43</a>,<a href="#CR44" class="usa-link" aria-describedby="CR44">44</a></sup>. In fact, PAX3 and PAX7 co-expression persists in all limb myogenic progenitors throughout development, from E10.5-E16.5, i.e., when progenitors assume satellite cell positions along the myofibers under the basal lamina (Supplementary Fig. <a href="#MOESM1" class="usa-link">5d–g</a>)<sup><a href="#CR6" class="usa-link" aria-describedby="CR6">6</a></sup>. Therefore, all 16TF-VNP<sup>+</sup> and 16TF-VNP<sup>-</sup> progenitors present in the limb at E4.5 co-expressed PAX3 and PAX7 (Supplementary Fig. <a href="#MOESM1" class="usa-link">5a–c</a>).</p>
<p id="Par14">To further characterize the molecular and proliferative signature of TCF-LEF/β-catenin activating cells in the limb bud, we performed immunostainings against PAX7, MYF5, MYOD and EdU. At E4.5, all muscle cells present in avian limb muscle masses are mononucleated and express PAX7<sup><a href="#CR36" class="usa-link" aria-describedby="CR36">36</a></sup>. At this stage, we observed that while MYF5 expression was widespread throughout the entire muscle progenitor population, MYOD expression was restricted to its central region (Fig. <a href="#Fig2" class="usa-link">2b, e</a>). We observed that all 16TF-VNP<sup>+</sup> cells (100%) expressed MYF5, while only 6% expressed MYOD (Fig. <a href="#Fig2" class="usa-link">2b–g</a>). At E6.5, many polynucleated MyHC<sup>+</sup> muscle fibers are present throughout the growing wing and they are tightly intermingled with single-cell progenitors<sup><a href="#CR36" class="usa-link" aria-describedby="CR36">36</a></sup>. Similar to E4.5, we observed at E6.5 that the vast majority of 16TF-VNP<sup>+</sup> cells (93%) co-expressed PAX7 and MYF5, but that most (85%) 16TF-VNP<sup>+</sup> did not express MYOD (Fig. <a href="#Fig2" class="usa-link">2h–k</a>). At E7.5, even though very few progenitors expressed the reporter, all of them were PAX7<sup>+</sup> (Fig. <a href="#Fig2" class="usa-link">2l</a>). Even though we have previously shown that all PAX7<sup>+</sup>/MYF5<sup>+</sup> progenitors are faster-dividing cells than PAX7<sup>+</sup>/MYF5<sup>-</sup> progenitors<sup><a href="#CR41" class="usa-link" aria-describedby="CR41">41</a></sup>, it was possible that the 16TF-VNP<sup>+</sup> and 16TF-VNP<sup>-</sup>subpopulations proliferated at different rates. However, labeling of embryos with EdU showed that both the negative and the positive populations displayed the same rate of proliferation (Supplementary Fig. <a href="#MOESM1" class="usa-link">6a–c</a>).</p>
<figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2. 16TF-VNP<sup>+</sup> cells are early myogenic progenitors.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12374003_41467_2025_61767_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dd7d/12374003/84adc7180434/41467_2025_61767_Fig2_HTML.jpg" loading="lazy" id="d33e811" height="434" width="667" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Brachial somites were electroporated at E2.5 and embryos were analyzed at indicated timepoints. <strong>b</strong>–<strong>e</strong> Dorsal views of confocal stacks of E4.5 limb buds electroporated at E2.5 with an ubiquitous TagBFP, the 16TF-VNP reporter and immunostained for MYF5 (<strong>b</strong>, <strong>c</strong>) or MYOD (<strong>d</strong>, <strong>e</strong>). White arrowheads in (<strong>c</strong>) indicates 16TF-VNP<sup>+</sup>/MYF5<sup>+</sup> cells; yellow arrowheads in (<strong>e</strong>) indicate some of the few 16TF-VNP<sup>+</sup>/MYOD<sup>+</sup> observed. <strong>f</strong>, <strong>g</strong> Quantification of the percentage of 16TF-VNP<sup>+</sup> cells positive for MYF5 (<strong>e</strong>) and MYOD (<strong>f</strong>) each dot represents a limb bud (<em>n</em> = 5 and 4, respectively). <strong>h</strong>, <strong>i</strong> Dorsal view of confocal stacks of E6.5 limb buds electroporated with a ubiquitous TagBFP, the 16TF-VNP reporter, and stained for PAX7 and MYF5 (<strong>h</strong>) or PAX7 and MYOD (<strong>i</strong>). The TagBFP channel is not represented. The white arrowheads indicate 16TF-VNP<sup>+</sup>/PAX7<sup>+</sup>/MYF5<sup>+</sup> cells (<strong>g</strong>) and 16TF-VNP<sup>+</sup>/PAX7<sup>+</sup>/MYOD<sup>-</sup> ones (<strong>i</strong>). The yellow arrowhead in (<strong>i</strong>) indicates a 16TF-VNP<sup>+</sup>/PAX7<sup>+</sup>/MYOD<sup>+</sup> cell. <strong>j</strong>, <strong>k</strong> Quantification of the percentage of 16TF-VNP<sup>+</sup> cells positive for PAX7 and MYF5 (<strong>j</strong>) or PAX7 and MYOD (<strong>k</strong>), each dot represents a limb bud (<em>n</em> = 5 for both staining). <strong>l</strong>, <strong>m</strong> Dorsal view of confocal stacks of E7.5 limb buds electroporated with a ubiquitous nuclear dTomato, the 16TF-VNP reporter, and stained for PAX7. The arrowhead indicates one of the few remaining 16TF-VNP<sup>+</sup> cells that also expresses PAX7 (<em>n</em> = 5 embryos). <strong>m</strong> is an enlargement of (<strong>l</strong>). Box plots show the median (center line), 25th and 75th percentiles (box limits), and whiskers extending to the smallest and largest values within 1.5 x IQR. Scale bars: 50 μm (<strong>h</strong>, <strong>i</strong>, <strong>m</strong>), 100 μm (<strong>b</strong>–<strong>e</strong>), or 200 μm (<strong>l</strong>).</p></figcaption></figure><p id="Par15">Together, these experiments demonstrate that TCF-LEF/β-catenin dependent signaling is strictly restricted to a narrow time window of myogenesis, to a population of proliferating PAX3<sup>+</sup>/PAX7<sup>+</sup>/MYF5<sup>+</sup> progenitors. As soon as these progenitors progress further along the myogenic path, initiating MYOD expression and exiting cell cycle, TCF-LEF/β-catenin dependent signaling is turned off.</p></section><section id="Sec5"><h3 class="pmc_sec_title">TCF-Trace, a dynamic lineage tracing system to follow the fate of TCF-LEF/β-catenin<sup>+</sup> myogenic precursors</h3>
<p id="Par16">The 16TF-VNP reporter we engineered provides a snapshot of TCF-LEF/β-catenin dependent signaling at the time of analysis. It was, however, possible that limb muscle progenitors fluctuate between a 16TF-VNP<sup>+</sup> and a 16TF-VNP<sup>-</sup> state. To test this, we designed a reporter construct where the destabilized nuclear mVenus fluorescent protein was placed in tandem with a stable nuclear mCherry (half-life: about 18 h<sup><a href="#CR45" class="usa-link" aria-describedby="CR45">45</a></sup>). This technique has been used in <em>Drosophilia</em><sup><a href="#CR46" class="usa-link" aria-describedby="CR46">46</a></sup> to test whether the activity of a promoter is fluctuating between an ON/OFF state. If the activity of the reporter was fluctuating, there would be more cells labeled with the stable mCherry than the destabilized mVenus. On the contrary, if cells constantly respond to the signal, only double-positive cells should be observed. The construct was electroporated in the VLL at E2.5 and examined in the limb at E4.5. We observed a near-perfect (98%) correlation of mVenus and mCherry stainings, indicating that, within the time frame of the experiment, the 16TF-VNP<sup>+</sup> progenitors maintain the reporter’s activity and do not fluctuate between a positive and negative state (Supplementary Fig. <a href="#MOESM1" class="usa-link">7a-c</a>).</p>
<p id="Par17">It therefore became important to investigate the long-term fate of myogenic progenitors that activate TCF-LEF/β-catenin dependent signaling. To address this, we developed a lineage tracing system using both Tet-On and Cre-Lox technologies (Fig. <a href="#Fig3" class="usa-link">3a</a>). This system aims to permanently label cells with dTomato fluorescence when they simultaneously experience TCF-LEF/β-catenin signaling and are exposed to doxycycline (Fig. <a href="#Fig3" class="usa-link">3b</a>). The Tet-on technology<sup><a href="#CR47" class="usa-link" aria-describedby="CR47">47</a>,<a href="#CR48" class="usa-link" aria-describedby="CR48">48</a></sup> is a well-established system of drug-induced gene activation, which displays low background and high induction rates, in particular when using the “rtTA/TRE 3 G” system<sup><a href="#CR49" class="usa-link" aria-describedby="CR49">49</a>,<a href="#CR50" class="usa-link" aria-describedby="CR50">50</a></sup>. Combining this to the CRE-mediated excision of “Stop/PolyA” sequences placed upstream of a fluorescent protein should deliver a very sensitive system to permanently label myogenic progenitors. However, a significant drawback of the Tet-on system is that the rtTA protein is very stable<sup><a href="#CR51" class="usa-link" aria-describedby="CR51">51</a></sup>, making it unsuitable for dynamic studies. We hypothesized that the rtTA protein could be destabilized, a process that should significantly enhance its utility for dynamic studies but that comes at the cost of significantly weaker protein expression<sup><a href="#CR46" class="usa-link" aria-describedby="CR46">46</a></sup>. We first engineered a rtTA construct where a PEST proteolytic signal was inserted at its N terminus (PEST-rtTA)<sup><a href="#CR52" class="usa-link" aria-describedby="CR52">52</a></sup>.</p>
<figure class="fig xbox font-sm" id="Fig3"><h4 class="obj_head">Fig. 3. TCF-Trace, a tool to follow the fate of cells experiencing TCF-LEF/β-catenin dependent signaling.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12374003_41467_2025_61767_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dd7d/12374003/152bbc4ade1a/41467_2025_61767_Fig3_HTML.jpg" loading="lazy" id="d33e972" height="957" width="800" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong>, <strong>b</strong> Schematics of the constructs tested. The three constructs comprise 16 TCF/LEF binding sites upstream of a minimal promoter driving the expression of a rtTA (1) or a rtTA fused with a PEST sequence at its N-terminal part (2) or a rtTA fused with a PEST sequence at its N-terminal part, flanked by translational enhancers (3). All constructs were co-electroporated with a plasmid containing a rtTA-dependent (Tetracycline-Response Element, TRE) CRE recombinase and another plasmid containing a CRE-inducible nuclear dTomato. <strong>b</strong> Upon a response to TCF/LEF signaling in presence of doxycycline cells are permanently labeled with dTomato. <strong>c</strong> Embryos were electroporated in the neural tube at E2.5, induced with doxycycline and analyzed one day later. <strong>d</strong>–<strong>i</strong> Dorsal view of confocal stacks of E3.5 neural tube (NT) electroporated with a ubiquitous H2B-TagBFP and the rtTA plasmid (<strong>d</strong>, <strong>e</strong>), the destabilized rtTA (<strong>f</strong>, <strong>g</strong>) or the destabilized and boosted rtTA (<strong>h</strong>, <strong>i</strong>). The dotted lines delineate the electroporated, right side of the neural tube. <strong>j</strong> Quantification of (<strong>d</strong>–<strong>i</strong>), each dot represents an embryo (<em>n</em> = 4, 4, 4, 5, 4 and 4 for each condition, respectively). Statistical comparison was performed using the non-parametric Kruskal-Wallis test for multiple comparisons, followed by Dunn-Bonferroni post-hoc test. * = <em>p</em> &lt; 0.05. <strong>k</strong>, <strong>m</strong> Dorsal view of confocal stacks of E4.5 limb buds electroporated at E2.5 with an ubiquitous mVenus, a ubiquitous rtTA, the CRE and dTomato plasmids, doxycycline was added at E2.5 and E3.5. <strong>m</strong> Quantification of the percentage of dTomato cells within the mVenus<sup>+</sup> population<sup>,</sup> each dot represents a limb bud (<em>n</em> = 6). Box plots show the median (center line), 25th and 75th percentiles (box limits), and whiskers extending to the smallest and largest values within 1.5 x IQR. Scale bars: 50 μm (<strong>d</strong>–<strong>i</strong>, <strong>l</strong>) or 100 μm (<strong>l</strong>).</p></figcaption></figure><p id="Par18">We used the electroporation technique to test this constructs in the dorsal region of the neural tube (Fig. <a href="#Fig3" class="usa-link">3c</a>), known to respond to TCF/LEF signaling<sup><a href="#CR35" class="usa-link" aria-describedby="CR35">35</a></sup>. In the control experiment, we tested a native (not destabilized) rtTA construct, and we observed that the addition of doxycycline to electroporated embryos led to a strong response, with many visible dTomato<sup>+</sup> neural cells (Fig. <a href="#Fig3" class="usa-link">3a, d, e</a>). Expectedly, fusing PEST sequences to the rtTA construct led to a strong decrease in the efficiency of the response, with only few visible dTomato<sup>+</sup> neural cells (Fig. <a href="#Fig3" class="usa-link">3a, f, g</a>). To address the reduced expression levels, we incorporated to the rtTA/PEST construct translation enhancer sequences (IVS/Syn21/p10; see above). The addition of translational enhancers to the rtTA-PEST sequence restored a labeling efficiency that was comparable to that observed with the original rtTA construct (Fig. <a href="#Fig3" class="usa-link">3a, h, i</a>). This suggests that the two-step strategy (destabilization/translation enhancement) generated a sensitive Tet-on system that dynamically responds to doxycycline exposure when TCF-LEF/β-catenin dependent signaling is active.</p>
<p id="Par19">We tested the efficiency of this tracing system. Tracing TCF-LEF/β-catenin-responding cells involves a succession of molecular steps (doxycycline-triggered activation of Cre expression, excision of Stop sequences, and expression of the lineage tracing fluorescent protein, (Fig. <a href="#Fig3" class="usa-link">3b</a>). To evaluate the efficiency of the system to permanently label all cells experiencing TCF-LEF/β-catenin dependent signaling, we substituted the 16TF promoter with a CAGGS ubiquitous promoter. This should theoretically lead to the activation of the tracing fluorescent protein in all electroporated cells upon doxycycline addition. The VLL of brachial somites were co-electroporated with this plasmid mix, and embryos were exposed to a doxycycline solution for two consecutive days and then analyzed one day later at E4.5 (Fig. <a href="#Fig3" class="usa-link">3j</a>). We observed that 98% of the electroporated cells, labeled by the expression of the constitutive mVenus protein, also expressed dTomato (Fig. <a href="#Fig3" class="usa-link">3k–m</a>). This near-perfect correlation between the expression of mVenus and dTomato suggests that, despite significant destabilization of the rtTA protein and the many molecular steps required to activate the reporter, the doxycycline-mediated induction of dTomato fluorescence by the Tet-on/CRE system we designed is highly efficient. Furthermore, this experiment indicates that despite using multiple independent plasmids (five, including the transposase construct), all electroporated cells appear to have simultaneously incorporated them. This efficient and dynamic tool, that we named <em><span class="text-underline">TCF-Trace</span></em>, is the first molecular tool aimed at following the fate of cells experiencing temporary bursts of TCF-LEF/β-catenin-dependent signaling. We therefore proceeded to investigate the fate of myogenic precursor cells labeled by TCF-Trace.</p></section><section id="Sec6"><h3 class="pmc_sec_title">Two distinct progenitor populations co-exist in early limb myogenesis</h3>
<p id="Par20">The TCF-Trace system displays accurate temporal labeling of targeted cells without significant lag from previous cell history. This temporal precision is crucial in our experimental design, as we electroporate VLL cells that display a high TCF-LEF/β-catenin-dependent activity at E2.5 (Fig. <a href="#Fig1" class="usa-link">1c, d</a>), an activity, likely linked to their epithelial state<sup><a href="#CR53" class="usa-link" aria-describedby="CR53">53</a>–<a href="#CR55" class="usa-link" aria-describedby="CR55">55</a></sup>, that we do not intend to trace. We therefore initiated the lineage fate of PAX7<sup>+</sup>/MYF5<sup>+</sup> progenitors present in the limb from embryonic day 4.5 (E4.5).</p>
<p id="Par21">To do this, we electroporated the VLL of E2.5 brachial somites with a combination of the three plasmids described above, together with a ubiquitously expressed mVenus as an electroporation marker and the transposase plasmid. Subsequently, at E4.5, 5.5, and 6.5, doxycycline was added to developing embryos, aiming to label all progenitors activating TCF-LEF/β-catenin-dependent signaling during that time window. We analyzed the embryos at E7.5, at a time when the activity of the 16TF-VNP reporter is almost extinguished (Fig. <a href="#Fig1" class="usa-link">1l</a>, Fig. <a href="#Fig4" class="usa-link">4a</a>). We tested for expression of PAX7, mVenus, and dTomato proteins (Fig. <a href="#Fig4" class="usa-link">4b–d</a>). We observed numerous electroporated (green) myofibers containing dTomato<sup>+</sup> nuclei (red), indicating a massive contribution of the TCF-LEF/β-catenin<sup>+</sup> progenitors to myotube formation (Fig. <a href="#Fig4" class="usa-link">4b</a>).</p>
<figure class="fig xbox font-sm" id="Fig4"><h4 class="obj_head">Fig. 4. Lineage analysis of TCF-Trace<sup>+</sup> and TCF-Trace<sup>-</sup> populations.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12374003_41467_2025_61767_Fig4_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dd7d/12374003/e6a04ffaddd8/41467_2025_61767_Fig4_HTML.jpg" loading="lazy" id="d33e1202" height="999" width="747" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> TCF-Trace was induced with doxycycline during the time period (E4.5-E6.5) in which muscle progenitors respond to TCF/LEF signaling. Brachial somites of E2.5 embryos were electroporated, TCF-Trace was induced 2–4 days later, and embryos were analyzed at various stages from E7.5 to E16.5. <strong>b</strong>, <strong>c</strong> Dorsal view of confocal stacks of E7.5 limb bud electroporated with a ubiquitous mVenus, the TCF-Trace lineage tool driving the expression of a nuclear dTomato and stained for PAX7. <strong>c</strong> is an enlargement of (<strong>b</strong>). Arrowheads in (<strong>c</strong>) indicate electroporated TCF-Trace<sup>-</sup> muscle progenitors <strong>d</strong> Quantification of the percentage of TCF-Trace<sup>-</sup> or TCF-Trace<sup>+</sup> cells in the PAX7<sup>+</sup> electroporated population at E7.5 in the proximal and the distal part of the muscle mass, each dot represent a limb bud. <strong>e</strong>, <strong>f</strong> Transverse sections of E9.5 (<strong>e</strong>) and E12.5 (<strong>f</strong>) limb buds electroporated with a myofiber-specific dTomato and the TCF-Trace lineage tool driving the expression of eGFP. Arrowheads in (<strong>f</strong>) indicate electroporated TCF-Trace<sup>-</sup> myofibers. <strong>g</strong> Representative example of a TCF-Trace<sup>+</sup> myotube surrounded by a smaller TCF-Trace<sup>-</sup> myotube (white arrowhead). <strong>h</strong> Quantification of the percentage of TCF-Trace<sup>-</sup> and TCF-Trace<sup>+</sup> myotubes in the MLC<sup>+</sup> electroporated population, each dot represents a section, <em>n</em> = 7 limbs for each condition, each dot represents a section, <em>n</em> = 28, 24, and 24 for E9.5, E12.5, and E16.5, respectively. Statistical comparison was performed using the non-parametric, Kruskal-Wallis test for multiple comparisons, followed by Dunn-Bonferroni post-hoc test. NS: <em>p</em> &gt; 0.05, ****: <em>p</em> &lt; 0.0001. <strong>i</strong> Transverse sections of E16.5 limb buds electroporated with a myofiber-specific dTomato, the TCF-Trace lineage tool driving the expression of eGFP, and immunostained for MYH7. Arrowheads indicate electroporated TCF-Trace<sup>+</sup> myofibers expressing MYH7. <strong>j</strong> Quantification of the percentage of TCF-Trace<sup>-</sup> or TCF-Trace<sup>+</sup> myotubes in the MYH7<sup>+</sup> electroporated myotubes, each dot represents a section, <em>n</em> = 6 limbs each dot represents a section (<em>n</em> = 52 sections). <strong>k</strong> Transverse sections of E16.5 limb buds electroporated with a nuclear Achilles, the TCF-Trace lineage tool driving the expression of a nuclear dTomato and stained for PAX7. Arrowheads indicate electroporated TCF-Trace<sup>-</sup> muscle progenitors. <strong>l</strong> Quantification of the percentage of TCF-Trace<sup>-</sup> or TCF-Trace<sup>+</sup> cells in the PAX7<sup>+</sup> electroporated population, each dot represents a section, <em>n</em> = 7 limbs, each dot represents a section (<em>n</em> = 25 sections). Box plots show the median (center line), 25th and 75th percentiles (box limits), and whiskers extending to the smallest and largest values within 1.5xIQR. Scale bars: 10 μm (<strong>g</strong>, <strong>i</strong>, <strong>k</strong>), 20 μm (<strong>e</strong>, <strong>f</strong>), 50 μm (<strong>c</strong>), or 200 μm (<strong>b</strong>).</p></figcaption></figure><p id="Par22">Strikingly, this analysis unveiled an additional finding: a significant proportion (65–72%) of PAX7<sup>+</sup>, electroporated progenitors were not labeled by TCF-Trace. This was observed both in the proximal and distal region of the muscle masses (arrows in Fig. <a href="#Fig4" class="usa-link">4c</a>, <a href="#Fig4" class="usa-link">d</a>). As we demonstrated the high sensitivity of the tracing system we designed (Fig. <a href="#Fig3" class="usa-link">3j–m</a>), it is unlikely that the absence of label is due to a failure to detect and trace all TCF-LEF/β-catenin<sup>+</sup> cells. Instead, it suggests the presence of a population of progenitors in the developing limb that never activate TCF-LEF/β-catenin signaling, co-existing with the TCF-Trace<sup>+</sup> progenitor population. This intriguing observation triggered further investigation into the long-term fate of these distinct populations.</p></section><section id="Sec7"><h3 class="pmc_sec_title">TCF-LEF/β-catenin<sup>+</sup> myogenic precursors differentiate into primary myotubes</h3>
<p id="Par23">To streamline myotube analyses, we engineered a second version of TCF-Trace, where a cytoplasmic forms of the fluorescent protein eGFP was used. This enabled straightforward identification of the entire myotube diameters on cross sections. The electroporation tracer was a cytoplasmic form of dTomato driven by the Myosin Light Chain (MLC<sup><a href="#CR56" class="usa-link" aria-describedby="CR56">56</a></sup>) promoter, known to be expressed in myofibers. On cross sections at E9.5, we observed that nearly all (93%) electroporated myofibers (in red) were positive for eGFP (in green; Fig. <a href="#Fig4" class="usa-link">4e, h</a>). At this stage in chicken embryos, only primary myotubes are present in muscle masses<sup><a href="#CR10" class="usa-link" aria-describedby="CR10">10</a></sup>. Based on this temporal criterion, we therefore hypothesized that the TCF-LEF/β-catenin<sup>+</sup> population represents the precursors of primary myotubes.</p>
<p id="Par24">To confirm this, we proceeded with an immunostaining approach. It is widely accepted that all slow myosin-expressing myofibers are primary myotubes<sup><a href="#CR19" class="usa-link" aria-describedby="CR19">19</a>–<a href="#CR22" class="usa-link" aria-describedby="CR22">22</a></sup>. As reports on slow myosin expression in chicken are scattered over many publications and developmental stages, we performed a comprehensive survey of slow myosin expression in the chicken forelimb from embryonic to fetal stages of development (Supplementary Fig. <a href="#MOESM1" class="usa-link">8</a>), and observed that its expression significantly progresses along development, starting to be visible at E9.5 in a few of the muscle masses present at that stage (Supplementary Fig. <a href="#MOESM1" class="usa-link">7c</a>), and steadily intensifying to encompass all muscles at E18.5 (Supplementary Fig. <a href="#MOESM1" class="usa-link">7d–h</a>). From E12.5 until E18.5, slow (primary) myofibers are progressively surrounded by small, fast (presumably secondary) myofibers (Supplementary Fig. <a href="#MOESM1" class="usa-link">8i–p</a>). To investigate the slow vs fast signature of the TCF-LEF/β-catenin-derived myofibers during embryonic and fetal development, we performed a lineage tracing experiment similar to the one above but analyzed the embryos at stage E16.5 (Fig. <a href="#Fig4" class="usa-link">4i–j</a>). We observed that nearly all (98%) slow myosin-positive myotubes expressing the electroporation marker derived from the TCF-LEF/β-catenin<sup>+</sup>-derived lineage (Fig. <a href="#Fig4" class="usa-link">4i, j</a>). Therefore, based on temporal and biochemical criteria, these experiments demonstrate that TCF-LEF/β-catenin<sup>+</sup> progenitors present in the growing limb bud constitute the cellular origin of primary myotubes.</p></section><section id="Sec8"><h3 class="pmc_sec_title">TCF-LEF/β-catenin<sup>-</sup> myogenic precursors differentiate into secondary myotubes and satellite cells</h3>
<p id="Par25">We then wondered what is the origin of secondary myotubes. Secondary myotubes have been shown to appear after E9 in chicken, often surrounding primary myotubes (Supplementary Fig. <a href="#MOESM1" class="usa-link">8i,j</a>)<sup><a href="#CR10" class="usa-link" aria-describedby="CR10">10</a>,<a href="#CR57" class="usa-link" aria-describedby="CR57">57</a></sup>. We followed the fate of TCF-Trace-positive and -negative progenitors, using the same protocol as in the previous experiment set, but left the embryos to develop until E12.5. In contrast to the analyses done at E9.5, we found that 21% of electroporated (red) myotubes were not labeled by the TCF-Trace system (green Fig. <a href="#Fig4" class="usa-link">4f–h</a>). Often, we observed that dTomato-only myotubes were smaller than eGFP-positive myotubes and that they were located at their periphery, which are typical characteristics of secondary myotubes (Fig. <a href="#Fig4" class="usa-link">4g</a>, arrowheads). Therefore, based on temporal and morphological criteria, these experiments strongly suggest that TCF-LEF/β-catenin<sup>-</sup> progenitors present in the growing limb bud constitute the cellular origin of secondary myotubes. Interestingly, in a similar analysis done at E16.5, we observed a significant decrease in the proportion of TCF-Trace-negative myotubes, paralleled by an increase in the proportion of TCF-Trace positive myotubes (Fig. <a href="#Fig4" class="usa-link">4h</a>). It is possible that the shift in the proportions of TCF-Trace<sup>+</sup> and TCF-Trace<sup>-</sup> myotubes from E12.5 until E16.5 is due to a mixing between both lineages, either through myoblast to myofibers fusion or through fusion between myofibers.</p>
<p id="Par26">Finally, we determined whether satellite cells originate from one, the other, or both progenitor populations. VLL cells of E2.5 brachial somites were electroporated with a combination of the three plasmids described above, together with a ubiquitously expressed H2B-Achilles as an electroporation marker. At E4.5, E5.5, and E6.5, doxycycline was added to developing embryos. In birds and rodents, satellite cells are the only PAX7<sup>+</sup> cells present in muscle masses in late fetal stages, just before hatching/birth<sup><a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>,<a href="#CR58" class="usa-link" aria-describedby="CR58">58</a>–<a href="#CR61" class="usa-link" aria-describedby="CR61">61</a></sup>. We explored whether PAX7<sup>+</sup> cells present at E16.5 were labelled by the tracing system. We found that the vast majority (87%) of electroporated satellite cells present in limb muscle masses were TCF-Trace-negative, while a minority (13%) were positive (Fig. <a href="#Fig4" class="usa-link">4k, l</a>). This finding suggests that most satellite cells present in muscles at hatching derive from the TCF-LEF/β-catenin<sup>-</sup> progenitor population present in early limb buds, with a minor contribution originating from the TCF-LEF/β-catenin<sup>+</sup> lineage.</p></section><section id="Sec9"><h3 class="pmc_sec_title">TCF-LEF/β-catenin signaling regulates the spatial distribution of primary progenitors</h3>
<p id="Par27">With the TCF-LEF/β-catenin reporter revealing pivotal connections between the embryonic origins of primary and secondary myotube lineages and adult muscle stem cells, it naturally prompted us to investigate how this signaling pathway influences these critical developmental processes. Canonical, β-catenin-dependent Wnt signaling has been proposed to fulfill various functions during embryonic myogenesis, synergizing or antagonizing with other signaling pathways, such as Sonic Hedgehog, Bone Morphogenic Proteins, or NOTCH<sup><a href="#CR40" class="usa-link" aria-describedby="CR40">40</a>,<a href="#CR62" class="usa-link" aria-describedby="CR62">62</a>–<a href="#CR65" class="usa-link" aria-describedby="CR65">65</a></sup>. In the limb, conflicting data suggest that β-catenin-dependent Wnt signaling activates, inhibits or is dispensable for embryonic myogenesis<sup><a href="#CR66" class="usa-link" aria-describedby="CR66">66</a>–<a href="#CR69" class="usa-link" aria-describedby="CR69">69</a></sup>. The expression of a large variety of Wnt signaling molecules (e.g. Wnt secreted signals, Wnt receptors and co-receptors, Wnt antagonists) during limb development<sup><a href="#CR70" class="usa-link" aria-describedby="CR70">70</a>–<a href="#CR72" class="usa-link" aria-describedby="CR72">72</a></sup> and different experimental procedures may explain those contradictory results. Furthermore, our discovery of the existence of two co-existing populations of limb muscle progenitors, distinguished by their response to TCF-LEF/β-catenin-dependent signaling, necessitates a reevaluation of the role of Wnt signaling in limb myogenesis. The precise spatio-temporal control provided by in vivo electroporation offers a powerful alternative to classical approaches, potentially clarifying the discrepancies observed in previous studies.</p>
<p id="Par28">To address the role of Wnt β-catenin-dependent signaling during limb myogenesis, we used a dominant-negative form of Lef1 (DN Lef1), known to efficiently inhibit Wnt β-catenin-dependent signaling<sup><a href="#CR73" class="usa-link" aria-describedby="CR73">73</a></sup>. This construct was electroporated at E2.5 in the lateral portion of brachial somites to target the limb muscle progenitor population. Its effect was evaluated at E4.5, at a time when the 16TF-VNP reporter is active in 50% of the progenitor population (Fig. <a href="#Fig1" class="usa-link">1k, l</a>). We examined myogenic progenitor proliferation and differentiation under this experimental setting. No differences were observed in the progenitor proliferation, as determined by phospho-histone H3 or EdU labeling (Supplementary Fig. <a href="#MOESM1" class="usa-link">9a–f</a>). Similarly, the early myogenic differentiation program remained unaltered, since the expression of the differentiation marker MYF5 was not altered (Supplementary Fig. <a href="#MOESM1" class="usa-link">9g–i</a>). To evaluate the role of TCF/LEF signaling in late myogenic commitment, we designed an inducible form of DN-Lef1 by placing its coding sequence, along with that of a nuclear dTomato, under the control of a bidirectional TRE promoter (pBI). We co-electroporated this construct with a ubiquitous rtTA-expressing plasmid and added doxycycline at E4.5, once myoblasts have migrated into the limb bud but have not yet expressed the terminal differentiation marker <em>MyoG</em>. Two days after doxycycline induction (E6.5), the percentage of MYOG<sup>+</sup> cells was similar to that of controls (Supplementary Fig. <a href="#MOESM1" class="usa-link">9j–m</a>).</p>
<p id="Par29">Interestingly, the spatial distribution of progenitors electroporated with the ubiquitous DN Lef1 revealed a notable effect. At E4.5, a subset of these cells exhibited a tendency to remain in a proximal location, closer to the somites from which they originated. In contrast, those that migrated distally showed an increased dispersion away from the limb axis. Quantitative analysis confirmed a significant difference in distribution between the progenitors expressing DN Lef1 and the controls (Fig. <a href="#Fig5" class="usa-link">5a–d</a>). To rule out the possibility that this phenotype was due to the inhibition of TCF-LEF/β-catenin signaling in the VLL, where it likely maintains its epithelialization state<sup><a href="#CR53" class="usa-link" aria-describedby="CR53">53</a>–<a href="#CR55" class="usa-link" aria-describedby="CR55">55</a></sup>, we repeated the experiment with the inducible form of DNLEF1 construct by activating its expression at E4.5 and examined embryos at E6.5. The results were consistent with the previous experiment (Supplementary Fig. <a href="#MOESM1" class="usa-link">9n, o</a>), reinforcing the role of TCF-LEF/β-catenin signaling in the control of muscle progenitor distribution.</p>
<figure class="fig xbox font-sm" id="Fig5"><h4 class="obj_head">Fig. 5. TCF-LEF/β-catenin controls the spatial distribution of primary myotube progenitors via <em>Cxcr4.</em>
</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12374003_41467_2025_61767_Fig5_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dd7d/12374003/d4d50e1c5fd8/41467_2025_61767_Fig5_HTML.jpg" loading="lazy" id="d33e1512" height="1004" width="775" alt="Fig. 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong>, <strong>b</strong> Dorsal view of a confocal stack of limbs electroporated with a ubiquitously expressed nuclear dTomato alone (<strong>a</strong>) or together with DN Lef1 (<strong>b</strong>). The asterisk in (<strong>b</strong>) indicates the clump of cell that remains located proximally. <strong>c</strong>, <strong>d</strong> Quantifications of the repartition of muscle progenitors in the limb, each dot represents an embryo (<em>n</em> = 9 and 8 for (<strong>c</strong>) and <em>n</em> = 10 and 12 for (<strong>d</strong>)). Statistical comparison was performed using the two-sided non-parametric Wilcoxon test. *** = <em>p</em> &lt; 0.001. <strong>e</strong> Procedure followed to perform scRNAseq on electroporated limb muscle progenitors. <strong>f</strong> Six distinct clusters were identified within the total population of myoblasts. <strong>g</strong>, <strong>h</strong> 16TF-VNP<sup>-</sup> (red) and 16TF-VNP<sup>+</sup> (yellow) distribute in all, but cluster 5. <strong>i</strong> List of m<strong>i</strong>gration-associated genes differentially expressed in the 16TF-VNP<sup>+</sup> and 16TF<sup>-</sup>VNP<sup>-</sup> populations of myoblasts. <strong>j</strong>, <strong>k</strong> HCR FISH to detect <em>Cxcr4</em> expression in control limb myoblasts (<strong>j</strong>) or after DN Lef1 over-expression (<strong>k</strong>). <strong>l</strong> Quantification of (<strong>j</strong>, <strong>k</strong>), 4 limb buds were analyzed and each dot represent a cell (<em>n</em> = 75 cells in total). Statistical comparison was performed using the two-sided non-parametric Wilcoxon test. *** = <em>p</em> &lt; 0.001. Box plots show the median (center line), 25th and 75th percentiles (box limits), and whiskers extending to the smallest and largest values within 1.5xIQR. Scale bars: 100 μm (<strong>a</strong>, <strong>b</strong>).</p></figcaption></figure><p id="Par30">While the many Wnt ligands expressed during limb formation in amniote embryos could potentially regulate the migration and/or the distribution of the primary myotube progenitors, an important question was whether Wnt signaling controls this process directly, acting as a trophic factor, or indirectly, through transcriptional regulation of another molecule. To test this, we injected <em>Wnt1</em>-expressing cells into early limb buds at E3.5 and analyzed the distribution of (PAX7<sup>+</sup>) muscle progenitor cells one day after (Supplementary Fig. <a href="#MOESM1" class="usa-link">10a–d</a>). These manipulations did not lead to detectable defects in the normal distribution of muscle progenitors in experimental embryos (Supplementary Fig. <a href="#MOESM1" class="usa-link">10e–j</a>), suggesting that, while Wnt signaling regulates progenitor migration, Wnt ligands do not act as positional cues in this process.</p></section><section id="Sec10"><h3 class="pmc_sec_title">A common developmental program underlies chicken and human limb myogenesis</h3>
<p id="Par31">A multitude of genes and signaling pathways are known to regulate muscle progenitor migration during embryogenesis<sup><a href="#CR74" class="usa-link" aria-describedby="CR74">74</a>–<a href="#CR77" class="usa-link" aria-describedby="CR77">77</a></sup>, making a candidate gene approach to test targets of TCF-LEF/β-catenin signaling in primary myotube progenitors a tedious endeavor. Therefore, we adopted a transcriptomic approach, performed on limb muscle progenitors co-electroporated with a ubiquitously-expressed nuclear dTomato and the 16TF-VNP reporter, followed by cell sorting based on red fluorescence (Supplementary Fig. <a href="#MOESM1" class="usa-link">11</a>). This enriched population of myoblasts was then subjected to single-cell RNA sequencing (scRNAseq) analysis (Fig. <a href="#Fig5" class="usa-link">5e</a>).</p>
<p id="Par32">Using unsupervised graph-based clustering combined with FFT-accelerated interpolation-based t-SNE (FIt-SNE), we identified several distinct clusters of cells (Fig. <a href="#Fig5" class="usa-link">5e</a>). We employed negative selection to exclude non-muscle contaminating cells, identified by specific markers (e.g., <em>Csf1r</em> for macrophages; <em>Flt1</em> for endothelial cells, see Supplementary Fig. <a href="#MOESM1" class="usa-link">12a</a>), while retaining cells expressing <em>Pax7</em>, a known marker of muscle progenitors. Within <em>Pax7</em><sup>+</sup> cells, we identified six distinct clusters that, based on the expression of known factors (Supplementary Fig. <a href="#MOESM1" class="usa-link">12b, c</a>, Supplementary Data <a href="#MOESM3" class="usa-link">1</a>), represent stages of proliferation and differentiation of muscle progenitors (Fig. <a href="#Fig5" class="usa-link">5f</a>). By mapping for the presence of transcripts from the two electroporated transgenes within these clusters, individual cells could be grouped into two distinct populations: 16TF-VNP<sup>-</sup> (corresponding to secondary myotube and satellite cell progenitors) and 16TF-VNP<sup>+</sup> (encompassing progenitors of the primary myotube lineage; red and yellow dots, respectively, Fig. <a href="#Fig5" class="usa-link">5g, h</a>). The 16TF-VNP<sup>+</sup> cells were absent in Cluster 5, which comprises progenitors in the late stages of myogenic differentiation (<em>e.g</em>., <em>MyoD</em><sup>+</sup>, <em>MyoG</em><sup>+</sup>, <em>Tmem8c/Mymk</em><sup>+</sup>; Supplementary Fig. <a href="#MOESM1" class="usa-link">12c</a>), consistent with earlier data (Fig. <a href="#Fig2" class="usa-link">2</a>). However, cells from the two populations were found in all other clusters, suggesting that at that stage of development, their transcriptional signatures are largely similar.</p>
<p id="Par33">Despite this high similarity, we identified 35 genes differentially expressed in the 16TF-VNP<sup>+</sup> population compared to the 16TF-VNP<sup>-</sup> cells (Supplementary Fig. <a href="#MOESM1" class="usa-link">12d</a>, Supplementary Data <a href="#MOESM4" class="usa-link">2</a>). This analysis revealed that i) known members and targets of Wnt canonical signaling were expressed at a higher level in the 16TF-VNP<sup>+</sup> cell population, while secreted inhibitors of the pathway were more prevalent in the 16TF-VNP<sup>-</sup> cell population (Supplementary Fig. <a href="#MOESM1" class="usa-link">12d, e</a>); ii) factors known to regulate cell migration in various contexts were overexpressed in the 16TF-VNP<sup>+</sup> population (e.g. <em>Cxcr4</em> during myogenesis<sup><a href="#CR78" class="usa-link" aria-describedby="CR78">78</a></sup>; <em>Epha7</em> and <em>Nectin3</em> in the hematopoietic system<sup><a href="#CR79" class="usa-link" aria-describedby="CR79">79</a>,<a href="#CR80" class="usa-link" aria-describedby="CR80">80</a></sup>; <em>Epha7</em> in the nervous system<sup><a href="#CR81" class="usa-link" aria-describedby="CR81">81</a>,<a href="#CR82" class="usa-link" aria-describedby="CR82">82</a></sup>; <em>Cadm1</em>, <em>Wasf3</em>, <em>Sh3pxd2a, Mylk, Brca1, Prrx1 and Erc1</em> in cancer<sup><a href="#CR83" class="usa-link" aria-describedby="CR83">83</a>–<a href="#CR89" class="usa-link" aria-describedby="CR89">89</a></sup>; Fig. <a href="#Fig5" class="usa-link">5i</a>). To further analyze our findings, we compared our dataset with recently published single-cell RNA sequencing data from human embryonic limbs<sup><a href="#CR90" class="usa-link" aria-describedby="CR90">90</a></sup>. Mapping myogenic cells between 5 to 9 weeks of development, this study had identified two distinct myogenic trajectories, termed first (or “embryonic”) and second (or “fetal”) myogenesis and a transient intermediate state linking the two (Supplementary Fig. <a href="#MOESM1" class="usa-link">13a-d</a>). Strikingly, several genes enriched in the 16TF-VNP<sup>+</sup> population (<em>CADM1</em>, <em>NECTIN3</em>, <em>CXCR4</em>, and <em>LEF1</em>) were also highly expressed during human embryonic myogenesis (Supplementary Fig. <a href="#MOESM1" class="usa-link">13e</a>). For each human myogenic cell, we calculated a score based on the level of expression of the 35 most highly expressed genes in the 16TF-VNP<sup>+</sup> population in chicken (see methods for a detailed explanation). Cells with the highest score preferentially mapped to the first “embryonic” and transitory state of myogenesis of the human dataset (Supplementary Fig. <a href="#MOESM1" class="usa-link">13f</a>). Intrigued, we asked whether Wnt-TCF/LEF signaling might also characterize embryonic myogenesis in human. To test this, we calculated a score for each human myogenic cell, based on genes known to be Wnt targets during muscle regeneration<sup><a href="#CR91" class="usa-link" aria-describedby="CR91">91</a>,<a href="#CR92" class="usa-link" aria-describedby="CR92">92</a></sup>. This analysis revealed a strong correlation between the embryonic trajectory and high Wnt target gene expression (Supplementary Fig. <a href="#MOESM1" class="usa-link">13g</a>), suggesting a conserved association between Wnt signaling and primary myogenesis in both chicken and human.</p>
<p id="Par34">Together, the strong transcriptomic similarities between TCF/LEF-responsive, primary myotube lineage progenitor cells in chicken and the ‘embryonic’ myogenic population identified in human limbs, combined with their shared association with active Wnt signaling, support the idea that these populations are homologous, pointing to a conserved developmental program between birds and mammals.</p></section><section id="Sec11"><h3 class="pmc_sec_title">CXCR4 acts downstream of TCF-LEF/β-catenin to regulate the spatial distribution of primary progenitors</h3>
<p id="Par35">In the transcriptomic analyses described above, the chemokine receptor <em>Cxcr4</em> emerged as a particularly intriguing candidate. In mouse and chicken embryos, <em>Cxcr4</em>, together with its cognate ligand <em>Sdf1</em>, regulates muscle progenitor migration. <em>Cxcr4</em> is expressed in a subset of limb muscle progenitors, while <em>Sdf1</em>, expressed in the limb mesenchyme, acts as a trophic factor to guide the migration of <em>Cxcr4</em>-positive progenitors. Interestingly, as was shown for TCF-LEF/β-catenin-positive cells (Fig. <a href="#Fig2" class="usa-link">2b–g</a>), <em>Cxcr4</em> and <em>MyoD</em> expressions are mutually exclusive<sup><a href="#CR78" class="usa-link" aria-describedby="CR78">78</a></sup>.</p>
<p id="Par36">These intriguing parallels suggested that <em>Cxcr4</em> might be a target of TCF-LEF/β-catenin-dependent signaling in early limb progenitors of the primary myotube lineage. To test this hypothesis, we electroporated the DN Lef1 construct at E2.5 in the VLL of brachial somites. Two days later, the limb buds were analyzed by HCR FISH<sup><a href="#CR93" class="usa-link" aria-describedby="CR93">93</a></sup> to detect the expression of <em>Cxcr4</em>. We observed that the expression of <em>Cxcr4</em> was strongly downregulated in electroporated PAX7<sup>+</sup> myoblasts compared to the PAX7<sup>+</sup> WT myoblasts (Fig. <a href="#Fig5" class="usa-link">5j–l</a>). Altogether, these results strongly suggest that TCF-LEF/β-catenin-dependent signaling in the progenitors of the primary myotube lineage is essential to regulate their distribution in the growing limb bud, partially or entirely through the transcriptional regulation of the <em>Cxcr4</em> receptor. Further studies will elucidate whether Wnt signaling regulates the other migration regulators we identified, which could have broader implications for the spatial organization and function of primary myotubes.</p></section></section><section id="Sec12"><h2 class="pmc_sec_title">Discussion</h2>
<p id="Par37">The discoveries outlined in this study address two enduring puzzles that have persisted within the field of myogenesis for four decades.</p>
<p id="Par38">i) Firstly, it resolves the longstanding question of the developmental origin of primary and secondary myotubes. It unequivocally demonstrates that these myotubes arise from two distinct myogenic progenitor populations that coexist early in the limb and are distinguished by their TCF-LEF/β-catenin-dependent Wnt signaling activity. While a recent transcriptomic analysis of the early developing human limb identified two co-existing myogenic lineages, their long-term developmental fates had not been explored<sup><a href="#CR90" class="usa-link" aria-describedby="CR90">90</a></sup>. The striking similarities we identified between human and chicken transcriptomic signatures, combined with our long-term lineage analyses, strongly suggest that the ontogenesis of primary and secondary myogenesis is governed by conserved molecular and cellular mechanisms across all amniotes.</p>
<p id="Par39">ii) Secondly, our finding clarifies a controversy surrounding the lineage association between embryonic and fetal myoblasts. The observation that myogenic precursors isolated from embryonic and fetal limbs exclusively generated primary or secondary myofibers, respectively, had led to the conclusion that myogenic progenitors consist of a homogeneous precursor population that evolves as development progresses<sup><a href="#CR26" class="usa-link" aria-describedby="CR26">26</a>,<a href="#CR28" class="usa-link" aria-describedby="CR28">28</a>,<a href="#CR29" class="usa-link" aria-describedby="CR29">29</a></sup>. Contradictory observations showed that embryonic and fetal myoblasts are two distinct precursor populations, sequentially migrating into the limb mesenchyme<sup><a href="#CR32" class="usa-link" aria-describedby="CR32">32</a></sup>, or present from the onset of limb formation<sup><a href="#CR10" class="usa-link" aria-describedby="CR10">10</a>,<a href="#CR34" class="usa-link" aria-describedby="CR34">34</a></sup>. Our study resolves this controversy by demonstrating that the myogenic progenitors present during limb development comprise a mixed population of precursors for primary and secondary myotubes. A surprising outcome of our findings is the identification of a shared origin for limb secondary myotubes and satellite cells. Although it is long established that all limb muscles, including its satellite cell component, originate from the VLL<sup><a href="#CR94" class="usa-link" aria-describedby="CR94">94</a>,<a href="#CR95" class="usa-link" aria-describedby="CR95">95</a></sup>, the developmental path that VLL cells follow to generate satellite cells, once they have migrated into the limb mesenchyme, has not been investigated.</p>
<p id="Par40">These findings prompt a re-evaluation of muscle development in both trunk and limb. Trunk muscle formation has been thoroughly studied and is understood to occur in two distinct stages: i) an initial stage, where epithelial cells from the edge of the dermomyotome closest to the neural tube contribute to the formation of the primary myotome, composed of mononucleated myocytes<sup><a href="#CR5" class="usa-link" aria-describedby="CR5">5</a>,<a href="#CR7" class="usa-link" aria-describedby="CR7">7</a>,<a href="#CR96" class="usa-link" aria-describedby="CR96">96</a></sup>. This primary myotome then transitions into a polynucleated structure through fusion with cells from other dermomyotome borders<sup><a href="#CR36" class="usa-link" aria-describedby="CR36">36</a></sup>; ii) a second stage, where satellite cell progenitors emerge from the central dermomyotome and migrate into the primary myotome. These progenitors initially fuse with each other to form additional polynucleated fibers, with a subset set aside to become satellite cells<sup><a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>,<a href="#CR58" class="usa-link" aria-describedby="CR58">58</a>,<a href="#CR59" class="usa-link" aria-describedby="CR59">59</a></sup>.</p>
<p id="Par41">The similarities between these processes in the trunk and our observations in the limb suggest a compelling hypothesis: despite the apparent differences in their origins—trunk muscles forming within the structured somite and limb muscles arising from migrating progenitors—both may follow a similar developmental pattern involving two distinct progenitor populations with different fates: i) the primary myotome and cells from the dermomyotome borders in the trunk may be analogous to the primary myotube lineage we identified in the limb; ii) cells from the central dermomyotome could correspond to the TCF-Trace-negative progenitors in the limb, along with their derivatives, the satellite cells. Further analyses will be necessary to determine whether the biochemical and cellular characteristics of these myogenic progenitor populations support a unified model of trunk and limb muscle formation.</p>
<p id="Par42">As was observed in the trunk<sup><a href="#CR36" class="usa-link" aria-describedby="CR36">36</a></sup>, it is fascinating that limb muscle progenitors carefully choose their fusion partners in an environment where all progenitors and myofibers are tightly intertwined, such that TCF-Trace<sup>+</sup> progenitors only fuse to myofibers derived from their own lineage, while TCF-Trace<sup>-</sup> progenitors, at least initially, fuse to myofibers from their own lineage. The cellular and molecular underpinning of such intricate fusion pattern remains to be uncovered.</p>
<p id="Par43">Lastly, the use of the TCF-LEF/β-catenin-dependent signaling reporter was pivotal in revealing the embryonic origins of two distinct populations of myogenic progenitors and their derivatives. This prompted a deeper investigation into the role of this pathway in limb myogenesis, leading to the discovery of its involvement in the spatial distribution of primary myotube progenitors. The identification of <em>Cxcr4</em>, a known regulator of limb progenitor migration, as a transcriptional target of TCF-LEF/β-catenin-dependent signaling underscores Wnt canonical signaling’s role in migration, likely acting indirectly through <em>Cxcr4</em>. Interestingly, <em>Cxcr4</em> loss of function was shown to result in myoblast apoptosis<sup><a href="#CR78" class="usa-link" aria-describedby="CR78">78</a></sup>, suggesting that while Wnt signaling may not directly influence cell survival or proliferation, it could act upstream of genes like <em>Cxcr4</em> that do so. Finally, the identification of additional transcriptional targets associated with cell migration across various contexts suggests that <em>Cxcr4</em> may only partially control myoblast migration in the limb.</p>
<p id="Par44">This finding adds yet another layer to the diverse functions of TCF-LEF/β-catenin-dependent signaling during myogenesis, complementing the extensive body of research on Wnt signaling in this process (see refs. <sup><a href="#CR62" class="usa-link" aria-describedby="CR62">62</a>,<a href="#CR63" class="usa-link" aria-describedby="CR63">63</a>,<a href="#CR97" class="usa-link" aria-describedby="CR97">97</a></sup> for reviews). While many studies have suggested an essential role for Wnt signaling in the cell decision-making to initiate myogenesis in somites, the intricate morphogenetic processes that accompany this cell fate decision—such as the epithelial-mesenchymal transition of dermomyotomal cells, their migration into the primary myotome, and their orientation and growth along the antero-posterior axis—have made it challenging to pinpoint the specific roles of canonical Wnt signaling. This complexity might explain the contradictory findings in the field<sup><a href="#CR3" class="usa-link" aria-describedby="CR3">3</a>,<a href="#CR64" class="usa-link" aria-describedby="CR64">64</a>,<a href="#CR65" class="usa-link" aria-describedby="CR65">65</a>,<a href="#CR98" class="usa-link" aria-describedby="CR98">98</a>–<a href="#CR100" class="usa-link" aria-describedby="CR100">100</a></sup>.</p>
<p id="Par45">Divergent data also exist regarding Wnt signaling’s role in limb myogenesis and adult muscle regeneration, with studies suggesting that Wnts can promote, inhibit, or be dispensable for myogenesis<sup><a href="#CR67" class="usa-link" aria-describedby="CR67">67</a>,<a href="#CR101" class="usa-link" aria-describedby="CR101">101</a>,<a href="#CR102" class="usa-link" aria-describedby="CR102">102</a></sup>. Given the extensive literature on Wnts’ potential roles in myogenesis and the numerous Wnts expressed during limb development<sup><a href="#CR71" class="usa-link" aria-describedby="CR71">71</a>,<a href="#CR103" class="usa-link" aria-describedby="CR103">103</a>,<a href="#CR104" class="usa-link" aria-describedby="CR104">104</a></sup>, it was surprising to observe that the 16TF-VNP reporter activity was restricted to a relatively short time window of limb myogenesis. It was completely absent in one lineage and silent during the majority of late myogenesis, including in satellite cells. These observations rule out the possibility that Wnts, via canonical, TCF-LEF/β-catenin-dependent signaling, play a role in late myogenesis, tissue patterning, or the emergence of the satellite cell population.</p>
<p id="Par46">Our results align with previous studies in mice and chicken, that indicated that Wnt signaling is dispensable for <em>Myf5</em> and <em>MyoD</em> expression and myofiber formation in early myogenesis<sup><a href="#CR53" class="usa-link" aria-describedby="CR53">53</a>,<a href="#CR69" class="usa-link" aria-describedby="CR69">69</a></sup>. Hutcheson et al. also suggested that there exists a PAX7+ lineage that contributes to late myotube formation. However, this study did not analyze the significance of those findings the context of the primary and secondary myotube lineages.</p>
<p id="Par47">The discovery of a molecular signature distinguishing primary from secondary myotube lineages is poised to revitalize a field that has been largely dormant for years due to the absence of specific molecular markers for these lineages. This breakthrough opens new avenues for investigation, enabling comprehensive molecular characterizations of both lineages. This will allow addressing critical questions about the developmental pathways their progenitors follow to generate functional muscles in the embryo and their roles during development, as well as during regeneration and repair in the adult. The 16TF-VNP reporter and the TCF-Trace system, combined with advanced “omics” technologies, will be instrumental in deciphering the molecular pathways involved in some of the most enigmatic events of late myogenesis.</p></section><section id="Sec13"><h2 class="pmc_sec_title">Methods</h2>
<section id="Sec14"><h3 class="pmc_sec_title">In ovo chicken embryos electroporation</h3>
<p id="Par48">Fertilized chicken eggs were obtained from a local breeder. To target wing muscle progenitors, the ventro-lateral lip (referred to as the VLL) of brachial somites in E2.5 chicken embryos was electroporated as described previously<sup><a href="#CR36" class="usa-link" aria-describedby="CR36">36</a></sup>. Briefly, chicken embryos were incubated at 37.5 °C until stage HH16 (52 h or E2.5). The plasmid solution was injected into the brachial somites (somites 16–21) using a glass capillary. Three pulses of 50 V, 10 ms in duration, and spaced by 10 ms, were applied directly to the embryo using tungsten and platinum electrodes. Eggs were then sealed and placed back in the incubator until the desired stage was reached. Cells emanating from the VLL and migrating into the wing bud mesenchyme constitute a mixed population of very early migrating endothelial progenitors, followed by myogenic progenitors<sup><a href="#CR38" class="usa-link" aria-describedby="CR38">38</a>,<a href="#CR94" class="usa-link" aria-describedby="CR94">94</a>,<a href="#CR105" class="usa-link" aria-describedby="CR105">105</a>–<a href="#CR107" class="usa-link" aria-describedby="CR107">107</a></sup>. This migration occurs over a short window of time, beginning soon after somite formation and lasting 15–20 h<sup><a href="#CR94" class="usa-link" aria-describedby="CR94">94</a>,<a href="#CR105" class="usa-link" aria-describedby="CR105">105</a></sup>. We developed an electroporation protocol that primarily targets myogenic progenitors (Supplementary Fig. <a href="#MOESM1" class="usa-link">2a</a>).</p></section><section id="Sec15"><h3 class="pmc_sec_title">Expression constructs</h3>
<p id="Par49">All constructs used in this study contained two Tol2 sequences (T2: transposable elements from medaka fish<sup><a href="#CR37" class="usa-link" aria-describedby="CR37">37</a></sup>) surrounding the entire construct to allow their stable integration in the chicken genome in the presence of transposase, provided by a co-electroporated transposase plasmid (CAGGS-Transposase). This plasmid does not contain Tol2 sequences and is thus gradually diluted along with cell division.</p>
<p id="Par50">Depending on the experimental or immunostaining strategies, several plasmids were used to visualize electroporated cells. These plasmids contained the coding sequences for various fluorescent proteins with different spectral characteristics and/or directed to different cellular compartments. The T2-CAGGS:BFP and T2-CAGGS:mVenus plasmids were used to label the cytoplasm of all electroporated cells with blue and green fluorescent proteins, respectively. The T2-CAGGS:H2B-BFP, T2-CAGGS:H2B-Achilles and T2-CAGGS:dTomatoNLS were used to labelled the nuclei of all electroporated cells with blue, green and red fluorescent proteins, respectively. H2B- and NLS-fused fluorescent proteins were used to easily detect single cells.</p>
<p id="Par51">Wnt binding to its receptor can trigger three downstream pathways. The best characterized (and referred to as “canonical”) Wnt cellular response results in the inhibition of the β-catenin destruction complex, ultimately leading to the translocation of β-catenin into the nucleus where it partners with members of the TCF/LEF family of transcription factors to activate various Wnt target genes. To monitor this canonical transcriptional response, efficient transcription-based reporter systems that combine the DNA binding sites of TCF/LEF upstream of a minimal promoter and a reporter gene have been designed, the first of which named TOPFlash<sup><a href="#CR108" class="usa-link" aria-describedby="CR108">108</a></sup>. Since the first description of the TOPFlash reporter, many derivatives have been generated that combined multiple TCF/LEF binding sites driving a variety of reporter genes. The high stability of the most commonly used reporters (b-galactosidase, GFP, RFP…) leads to their considerable accumulation in cells, activating the pathway, thus greatly facilitating their detection. However, two major drawbacks are: i) an important lag-time between the activation of the pathway and the accumulation of sufficient amounts of the reporter to be detected, and ii) conversely, the detection of signals in tissues where the activity of the pathway may have already ceased. Most of these constructs are therefore unsuitable to detect rapid spatiotemporal changes in TCF-LEF/β-catenin dependent signaling. We have used here a recently developed reporter of Wnt/β-catenin dependent signaling activity, named 16TF-VNP<sup><a href="#CR35" class="usa-link" aria-describedby="CR35">35</a></sup>, where the combination of 16 TCF/LEF-binding sites<sup><a href="#CR108" class="usa-link" aria-describedby="CR108">108</a></sup>, translational enhancers<sup><a href="#CR46" class="usa-link" aria-describedby="CR46">46</a>,<a href="#CR109" class="usa-link" aria-describedby="CR109">109</a></sup>, and a destabilized (half-life, 1.8 h<sup><a href="#CR110" class="usa-link" aria-describedby="CR110">110</a></sup>), fast maturing mVenus fluorescent protein<sup><a href="#CR111" class="usa-link" aria-describedby="CR111">111</a></sup> generated the most sensitive to date, yet dynamic, molecular tool to detect TCF/β-catenin-responding cells and tissues in developing embryos, with no detectable background<sup><a href="#CR35" class="usa-link" aria-describedby="CR35">35</a>,<a href="#CR112" class="usa-link" aria-describedby="CR112">112</a></sup>.</p>
<p id="Par52">For short-term pseudo-lineage of TCF/LEF responding cells, we inserted a P2A sequence downstream of and in-frame with the 16TF-VNP reporter, followed by a stable form of nuclear mCherry (half-life, about 18 h). This results in the production of two fluorescent proteins with different half-life. This strategy was described previously in <em>Drosophilia</em><sup><a href="#CR46" class="usa-link" aria-describedby="CR46">46</a></sup> to generate a dual-color Trans Timer construct that provides spatio-temporal information on signaling pathway activities.</p>
<p id="Par53">Even stable fluorescent proteins, such as GFP or RFP, are not suitable for long-term studies. For this, we have engineered a tri-partite lineage tracing system based on the Tet-On and the Cre-Lox technologies. A plasmid containing the destabilized rtTA transactivator gene under the control of the 16TF promoter was mixed with a plasmid encompassing a TRE (Tetracycline Response Element) promoter driving the expression of a CRE recombinase (T2-TRE:CRE) and a plasmid coding for a CRE-inducible nuclear dTomato (T2-CAGGS:LoxP-STOP-LoxP-dTomatoNLS) or a cytoplasmic eGFP (T2-CAGGS:LoxP-STOP-LoxP-eGFP).</p>
<p id="Par54">For blocking the transcriptional activity of the TCF/LEF signaling we used a dominant negative form of the LEF1 protein, consisting of a truncated form of LEF1 that binds the DNA but not to β-catenin, thereby blocking intrinsic LEF1 activity as a dominant negative mutant<sup><a href="#CR113" class="usa-link" aria-describedby="CR113">113</a></sup>. This mutant was cloned downstream of the strong and ubiquitous CAGGS promoter (T2-CAGGS:DNLef1) this construction was tested in ref. <sup><a href="#CR35" class="usa-link" aria-describedby="CR35">35</a></sup>. For testing specifically the latter effect on <em>MyoG</em> expression, we cloned this mutant into a bi-directional tetracycline response element that regulates the expression of both a nuclear dTomato and the DNLef1 (T2-pBI:DNLef1/dTomatoNLS). Upon co-electroporation with a ubiquitous rtTA plasmid (T2-CAGGS:rtTA) and addition of doxycycline, this plasmid allows the temporal control of DNLEF1 expression.</p>
<p id="Par55">As the electroporation technique results in the mosaic expression of transgenes in targeted cells, a ubiquitously expressed electroporation marker was added to identify and analyze all electroporated cells.</p>
<p id="Par56">To label grafted cells, we used a plasmid coding for a membrane-bound dTomato (T2-CAGGS:mbdTomato). To induce the expression of Wnt1 in the grafted cells, we transfected them with a plasmid containing the coding sequence of the mouse Wnt1 gene in which a Myc-tag was inserted, under the control of the CMV promoter. This Myc-tagged <em>Wnt1</em> construct was shown to induce a duplication of axis in Xenopus embryo as “canonical” TCF/LEF activating WNT proteins do<sup><a href="#CR114" class="usa-link" aria-describedby="CR114">114</a></sup>.</p></section><section id="Sec16"><h3 class="pmc_sec_title">Wholemount immunohistochemistry</h3>
<p id="Par57">E3 to E7.5 electroporated limb buds were imaged in whole mount as described<sup><a href="#CR36" class="usa-link" aria-describedby="CR36">36</a></sup>. Briefly, after dissection, samples were fixed for 1 h in 4% formaldehyde/PBS at room temperature (RT), briefly rinsed with PBS, pre-incubated in washing buffer (0.2% BSA, 0.1% Triton X-100, 0.2% SDS in PBS) for 1 h at RT, and incubated overnight at 4 °C with the primary antibodies. They were washed at least 5 times during the following day and incubated with secondary antibodies overnight at 4 °C. As fixation significantly decreases the brightness of fluorescent proteins (particularly GFP), we often used primary antibodies against fluorescent proteins to facilitate their detection. The following antibodies were used: rabbit polyclonal antibody (IgG) against dTomato (ab62341, Abcam, 1/1000); chicken polyclonal antibody (IgY) against eGFP, mVenus and Achilles (A10262, Invitrogen, 1/1000); mouse monoclonal antibody (IgG2a) against eGFP, mVenus and Achilles (A11120, Invitrogen, 1/1000); rabbit polyclonal antibody (IgG) against TagBFP (AB233, Evrogen, 1/500); mouse monoclonal antibody (IgG1) against PAX7 (<a href="https://www.ncbi.nlm.nih.gov/nuccore/AB528428" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">AB528428</a>, DSHB, 1/10); mouse monoclonal antibody (IgG2b) against Myosin Heavy Chain (MF20, AB2147781, DSHB, 1/10) and mouse monoclonal antibody (IgA) against the slow myosin MYH7B (S58, <a href="https://www.ncbi.nlm.nih.gov/nuccore/AB528377" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">AB528377</a>, DSHB, 1/10). These primary antibodies were used in combination with species-matched secondary antibodies conjugated with Alexa fluorochromes (488 nm, 555 nm, or 647 nm) from ThermoFisher or SouthernBiotech at a concentration of 1/500. After staining, samples were incubated in 50% glycerol/PBS for 1 h at RT and in 80% glycerol/PBS for several hours before analysis.</p></section><section id="Sec17"><h3 class="pmc_sec_title">Cryosectioning</h3>
<p id="Par58">E9.5 to E18.5 limb buds were fixed for whole-mount imaging as described in the whole-mount imaging section, washed in PBS for 1 hour, and then incubated in 7.5% sucrose/PBS and 15% sucrose/PBS at room temperature (RT) for 30 min to several hours, depending on their size. Next, they were immersed in a pre-melted solution of 15% sucrose and 7.5% bovine skin gelatin at 42 °C with agitation overnight before being placed into cryosectioning molds. The samples were then immersed in dry ice-cold 100% ethanol and stored at −80 °C until sectioning. They were cryosectioned using a Leica cryostat at a thickness of 18 µm. Sections were stored at −80 °C until analysis. Immuno-histochemistry was performed as described for whole-mount imaging, using the same set of antibodies, with the exception that secondary antibodies were incubated for only 2 h at RT. Nuclei were detected using DAPI. The slides were mounted in Fluoromount <sup>TM</sup>. Transversal and longitudinal sections were performed for the analyses of the 16xTF-VNP reporter at E9.5, while only transversal sections of the extensor muscle of the zeugopod were done at E12.5, E14.5, E16.5 and E18.5. Analysis of the TCF-Trace experiments after E9.5 was performed with transversal sections targeting the extensor muscle of the zeugopod.</p></section><section id="Sec18"><h3 class="pmc_sec_title">Doxycycline induction</h3>
<p id="Par59">Stock solution of doxycycline at a concentration of 20 mg/ml in ddH2O was prepared in advance and stored at −20 °C. A solution at 3.5 μg/ml was prepared by diluting the stock solution into sterile Ringer’s solution on the day of the injection; 300 μl of the solution was added per embryo.</p></section><section id="Sec19"><h3 class="pmc_sec_title">EdU incorporation</h3>
<p id="Par60">50 μl of a 10 mM of EdU solution was added directly onto the embryo that was placed back in the incubator for 1 h. Embryos were then dissected, fixed and immunostained as described above. Once immunostained, samples were pre-incubated in 250 μl of PBS with 1 μl of Alexa fluorophore for 1 h at RT. Separately 150 μl of PBS was mixed with 100 μl of ascorbic acid at 0.5 M and 2 μl of a 1 M CuSO<sub>4</sub> solution and added to the pre-incubated samples. Embryos were incubated overnight at 4 °C with agitation, washed at least five times the following day and cleared into glycerol as described.</p></section><section id="Sec20"><h3 class="pmc_sec_title">
<em>Cxcr4</em> fluorescent in situ hybridization</h3>
<p id="Par61">The <em>Cxcr4</em> probe solution is composed of 21 small probes directly designed by Molecular Instrument and already tested in ref. <sup><a href="#CR115" class="usa-link" aria-describedby="CR115">115</a></sup>. For a detailed protocol of wholemount HCR on chicken embryos see ref. <sup><a href="#CR115" class="usa-link" aria-describedby="CR115">115</a></sup>. Electroporated limb buds were collected two days after electroporation, fixed in 4% paraformaldehyde for 1 h at RT. Embryos were washed twice in PBS and dehydrated in growing concentration of MetOH/PBS (25%/50%/75%/100%). Embryos were stored at least one night at −20 °C before being rehydrated in decreasing concentration of MetOH/PBS (75%/50%/25%/PBS) and post fixed for 20 min in 4% PFA for 20 min at RT. Limb buds were then washed twice in PBS for 5 min, then once in a 1:1 solution of PBS/SSCT and finally once in SSCT. The embryos were pre-hybridized in 500 μL of hybridization buffer for 30 min at 37 °C (HCR™ Buffers, Molecular Instruments). The pre-hybridization solution was replaced by the solution with the probes (2 pmol in 500 μL hybridization buffer) and incubated overnight at 37 °C with shaking. After removal of the probe solution, samples were washed 4 times in 1 mL of washing buffer (HCR™ Buffers, Molecular Instruments) for 15 min at 37 °C with shaking. Two 5 min washes in SSCT solution were made at RT. Next, the embryos were incubated in 500 μL of amplification buffer for 5 min at RT (HCR™ Buffers, Molecular Instruments). The hairpins h1 and h2 (30 pmol (10 μL for 500 μL of buffer)) were heated separately at 95 °C for 90 s, left at RT for 30 min minimum in the dark before being added in 500 μL of amplification buffer. The amplifier in the amplification buffer was coupled with Alexa 647 for fluorescent detection. The pre-amplification solution was replaced by the solution containing the amplifiers and incubated overnight in the dark at RT with gentle agitation. Amplifier solution was removed, and embryos were washed in SSCT at RT (2 × 5 min, 2 × 30 min, 1 × 5 min). Embryos were then stained as described above for the dTomato and PAX7, cleared in glycerol and imaged at the confocal.</p></section><section id="Sec21"><h3 class="pmc_sec_title">Imaging, quantifications and statistical analyses</h3>
<p id="Par62">For whole-mount and cryosectioned samples, imaging was performed using a Leica SP5 confocal microscope with a resolution of 1024×1024 pixels, utilizing either a 20x or 40x objective with a Z-step size of 2 µm. Images were analyzed using Fiji with the Cell Counter plugin. Plot and statistical analyses were done using R, with ggplot2 and ggsignif packages. For experiments that were analyzed in wholemount, each dot represents an embryo. For cryosections, each dot represents a section, and the total number of embryos is indicated in the figure legend. For the <em>Cxcr4</em> experiment, each dot represents a cell, and the total number of embryos is indicated in the figure legend. For each condition, a minimum of 4 embryos were analyzed and for each embryo or section, a minimum of 250 cells were counted, leading to a total number of 1000 to 10,000 cells per condition. Experiments with two different conditions were compared using Wilcoxon-Mann-Whitney test and experiment with three different conditions with Kruskal-Wallis test associated with Dunn-Bonferroni post-hoc test. NS represents a <em>p</em>-value &gt; 0.05, * a <em>p</em>-value &lt; 0.01, *** a <em>p</em>-value &lt; 0.001, and **** a <em>p</em>-value &lt; 0.0001.</p></section><section id="Sec22"><h3 class="pmc_sec_title">Purification of electroporated single cell</h3>
<p id="Par63">E4.5 electroporated chicken embryos with a ubiquitous dTomato and the 16TF-VNP reporter were screen under a fluorescent binocular, and electroporated limb buds were quickly dissected and incubated with 500 μl of pre-warmed Dispase (1.5 mg/ml in DMEM/10 mM Hepes), pipette up and down 10 times and incubated 15 min at 37 °C. The sample was homogenized every 5 min then 500 μl of pre-warmed Trypsin (0.05% in DMEM) was added to the tube, homogenized and incubated 3 min at 37 °C. Samples were then transferred into a 15 ml falcon tube, and the reaction was stop with 10 ml of Hanks buffer (for 100 ml: 10 ml of HBSS 10X, 250 mg of BSA, 1 ml of Hepes 1 M, in sterile ddH<sub>2</sub>O), homogenized and centrifugated 10 min at 500 g. The pellet was re-suspended in 4 ml of Hanks buffer and filtered with a pre-humidified 40 μm sterile filter and re-centrifugated 10 min at 500 g. The final pellet was re-suspended into 250 μl of Hanks buffer and added to 250 μl of Hanks buffer in a pre-humidified FACS tube. For sorting, we added DAPI (1/1000) in the final Hanks buffer solution and prepare a sample containing non-electroporated tissues and non-electroporated tissues stained with DAPI to calibrate the sorting. Cells were then sorting according to the dTomato fluorescence and collected into Hank’s buffer. An example of the fluorescence-activated cell sorting (FACS) gating strategy is provided in Supplementary Fig. <a href="#MOESM1" class="usa-link">12</a>. For the single-cell RNA-seq experiment, a total of 6 electroporated limb buds were pooled together in the same tube.</p></section><section id="Sec23"><h3 class="pmc_sec_title">Single cell RNA-seq analyses of chicken embryo</h3>
<p id="Par64">To profile cell-type composition after electroporation, we performed single cell RNA-sequencing (scRNAseq) on chicken embryo limb bud electroporated with the CAGGS:dTomatoNLS and the 16TF:VNP plasmid (see section above). Data demultiplexed in fastq format was checked with fastqc and fastq_screen to control quality sequencing. Sequencing reads were processed with STAR<sup><a href="#CR116" class="usa-link" aria-describedby="CR116">116</a></sup> (v.2.7.9a) to align and quantify data. Genome and transcriptome used is GRCg6a.105 from Ensembl database &amp; custom references gtf and fasta files corresponding to the plasmids used. We merged both datasets in one. The downstream treatment for analysing separately and integratively the data used Seurat package (v5.1.0)<sup><a href="#CR117" class="usa-link" aria-describedby="CR117">117</a></sup> after filtering was <em>SCTransform</em> with <em>glmGamPoi</em> method<sup><a href="#CR118" class="usa-link" aria-describedby="CR118">118</a></sup> and a regression of percentage of mitochrondrial, PCA on SCT assay. We determined Doublet barcode with <em>scDblFinder</em><sup><a href="#CR119" class="usa-link" aria-describedby="CR119">119</a></sup> R packages (v1.16.0) on cluster-based approach and <em>dbr</em> = <em>0.1</em>. We selected any features where mean (count <sub>sample</sub>)<sub>feature</sub> &gt; 0.1. (s)KNN graph build on pca reduction matrix with a dimension depending of sample (we selected the last elbow point where cumulative sums of percentage Standard Deviation &gt; 90% and the n + 1 dimension not exceed &lt;1% supplementary), we used Louvain Clustering with <em>igraph</em> method. We used also FFT-accelerated Interpolation-based t-SNE<sup><a href="#CR120" class="usa-link" aria-describedby="CR120">120</a>,<a href="#CR121" class="usa-link" aria-describedby="CR121">121</a></sup> (FItSNE) with these parameters: a modification of initialization (first and second PCA dimensions are divided by standard deviation of first PCA dimension multiplied by 0.0001), <em>perplexity_list</em> used is a list starting 30 to nCells<sub>integrated dataset</sub>/100) and <em>learning rate = nCells</em><sub><em>integrated dataset</em></sub>
<em>/ 12</em>, all PCA dimensions selected from the last step and 1000 iterations to run. Biomarkers are identified after <em>PrepSCTFindMarkers</em> on SCT assay and performed with <em>FindAllMarkers</em> with MAST test<sup><a href="#CR122" class="usa-link" aria-describedby="CR122">122</a></sup>. We filtered filtered biomarkers to conserve only features |pct.1<sub>feature/cluster</sub> – pct.2<sub>feature/cluster</sub> | &gt; 0.2 and p-adj &lt;0.05. We selected a top5 on each cluster found and sign of log2FC. In total, 2108 cells were sequenced and used for further analysis. We identified the presence of absence of the dTomato and the mVenus transcripts and identified 432 cells where dTomato only was expressed ≥ 15 (~19%) &amp; 55 cells where mVenus was expressed ≥ 15 both (~2.6%). Each cell covered a value range of nCount RNA between 805 to 20000, nFeature RNA between 601 to 4462 and %mitochondrial ≤ 0.7%. Muscle progenitors dataset was isolated after identification of clusters based on genes expression and the same protocol of downstream refinement is used (SCT + PCA + dynamic selection of dimension PCA’s). P-values for differential expression between clusters were calculated using the MAST method in Seurat’s FindAllMarkers package, which models single-cell expression and accounts for detection rate; values were adjusted using the Benjamini–Hochberg correction. Differential expression between 16TF-VNP<sup>+</sup> and 16TF-VNP<sup>-</sup> cells was assessed using the MAST method in Seurat, which fits a hurdle model combining logistic regression for detection and linear modeling for expression levels. These p-values reflect the significance of expression differences while accounting for cellular detection rate. The scRNAseq data presented in this manuscript have been deposited in NCBI’s Gene Expression Omnibus under the accession number <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE283478" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE283478</a>.</p></section><section id="Sec24"><h3 class="pmc_sec_title">Analysis of <em>DN Lef1</em> phenotypes</h3>
<p id="Par65">Images of E4.5 control and DN Lef1 limb bud were acquires in wholemount on the confocal as described above. 3D acquisitions were transformed into 2D images with a maximal projection in Fiji. For the percentage of dispersion, an oblong shape was drawn around the muscle mass to measure its total area and then a threshold was set to measure the area of all the nuclei using the “Analyze particles” function. The percentage of dispersion was calculated as follow:</p>
<table class="disp-formula p" id="Equ1"><tr><td class="formula"><math id="d33e2234" display="block"><mi>P</mi><mi>e</mi><mi>r</mi><mi>c</mi><mi>e</mi><mi>n</mi><mi>t</mi><mi>a</mi><mi>g</mi><mi>e</mi><mspace width="0.16em"></mspace><mi>o</mi><mi>f</mi><mspace width="0.16em"></mspace><mi>D</mi><mi>i</mi><mi>s</mi><mi>p</mi><mi>e</mi><mi>r</mi><mi>s</mi><mi>i</mi><mi>o</mi><mi>n</mi><mo>=</mo><mfrac><mrow><mrow><mo>(</mo><mrow><mi>T</mi><mi>o</mi><mi>t</mi><mi>a</mi><mi>l</mi><mspace width="0.16em"></mspace><mi>A</mi><mi>r</mi><mi>e</mi><mi>a</mi><mo>−</mo><mi>S</mi><mi>u</mi><mi>m</mi><mspace width="0.16em"></mspace><mi>o</mi><mi>f</mi><mspace width="0.16em"></mspace><mi>P</mi><mi>a</mi><mi>r</mi><mi>t</mi><mi>i</mi><mi>c</mi><mi>l</mi><mi>e</mi><mi>s</mi><mspace width="0.16em"></mspace><mi>A</mi><mi>r</mi><mi>e</mi><mi>a</mi></mrow><mo>)</mo></mrow></mrow><mrow><mi>T</mi><mi>o</mi><mi>t</mi><mi>a</mi><mi>l</mi><mspace width="0.16em"></mspace><mi>A</mi><mi>r</mi><mi>e</mi><mi>a</mi></mrow></mfrac><mo>×</mo><mn>100</mn></math></td></tr></table>
<p id="Par66">For the center of mass, a line was traced in the middle of the muscle mass, along the proximo-distal axis of the limb and the center of mass was measured using the “Measure” function. The center of mass is defined as the brightness-weighted average of the x and y coordinates of all pixels in the image or selection. Therefore, if the repartition of the fluorescence is homogeneous along the proximo-distal axis the center of mass should be around the middle of the region of interest. The center of mass was normalized by the total length of the muscle mass. A value around 50 represents a homogeneous repartition.</p>
<table class="disp-formula p" id="Equ2"><tr><td class="formula"><math id="d33e2323" display="block"><mi>R</mi><mi>a</mi><mi>t</mi><mi>i</mi><mi>o</mi><mo>=</mo><mfrac><mrow><mi>Y</mi><mi>M</mi><mspace width="0.16em"></mspace><mi>o</mi><mi>f</mi><mspace width="0.16em"></mspace><mi>C</mi><mi>e</mi><mi>n</mi><mi>t</mi><mi>e</mi><mi>r</mi><mspace width="0.16em"></mspace><mi>o</mi><mi>f</mi><mspace width="0.16em"></mspace><mi>M</mi><mi>a</mi><mi>s</mi><mi>s</mi></mrow><mrow><mi>T</mi><mi>o</mi><mi>t</mi><mi>a</mi><mi>l</mi><mspace width="0.16em"></mspace><mi>L</mi><mi>e</mi><mi>n</mi><mi>g</mi><mi>t</mi><mi>h</mi><mspace width="0.16em"></mspace><mi>o</mi><mi>f</mi><mspace width="0.16em"></mspace><mi>t</mi><mi>h</mi><mi>e</mi><mspace width="0.16em"></mspace><mi>M</mi><mi>u</mi><mi>s</mi><mi>c</mi><mi>l</mi><mi>e</mi><mspace width="0.16em"></mspace><mi>M</mi><mi>a</mi><mi>s</mi><mi>s</mi></mrow></mfrac><mo>×</mo><mn>100</mn></math></td></tr></table></section><section id="Sec25"><h3 class="pmc_sec_title">In ovo cell grafting</h3>
<p id="Par67">7 million HEK-293 cells (#CRL-1573, ATCC) were plated in 10 cm culture-plates with DMEM / 1% Penicillin-Streptomycin / 10% FBS. The following day, 7.5 μg of the CAGGS:mbdTomato plasmid in combination or not with 7.5 μg of the CMV:Wnt1-Myc plasmid were mixed with 1.5 ml of Optimem and 150 μl of Polyethylenimine (PEI, #408727, Sigma-Aldrich) and incubated 30 min at room temperature. The plasmid mix was then added dropwise on antibiotic-free HEK-293. A total of two plates for each condition were transfected. The next day, cells were detached by the addition of 3 ml of 0.25% trypsin in DMEM. The reaction was blocked with DMEM / 1% Penicillin-Streptomycin/10% FBS and cells were harvested in a 15 ml Falcon tube, centrifuged, washed with 1 ml of DMEM/1% Penicillin-Streptomycin/10% FBS and centrifuged in 1.5 ml Eppendorf tubes. Supernatant was removed to obtain a thick preparation of cells. Cells were then loaded into a glass capillary (30-0047, Harvard Apparatus) with a Hamilton Microliter™ syringe #805 and injected using a PicoSpritzer injector into E3.5 chicken embryo limb buds. Wholemount embryos were immunostained and cleared as described above before being imaged with a fluorescent binocular. For the detection of the MYC-tag, embryos were cryosectioned, immunostained, and the MYC-tag was revealed using an anti MYC primary antibody (ab32, Abcam, IgG1, mouse, 1/200), in combination with a Goat anti-Mouse IgG biotinylated (1/200, 31800, ThermoFischer) and a Streptavidin coupled with Alexa 488 (S32354, ThermoFischer). The solidity and the circularity of the muscle mass were evaluated using the “Measure” function in Fiji.</p></section><section id="Sec26"><h3 class="pmc_sec_title">Analysis of the human embryonic limb atlas</h3>
<p id="Par68">Processed data of myogenesis cells from human embryonic limb atlas was downloaded (<a href="https://limb-dev.cellgeni.sanger.ac.uk/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://limb-dev.cellgeni.sanger.ac.uk/</a>) and analyzed in python (v3.10.16) with scanpy (v1.10.4), numpy (v2.0.2), matplotlib (v3.10.0) and igraph (v0.11.8) libraries. Development trajectories were reproduced following the pipeline describe by the authors<sup><a href="#CR90" class="usa-link" aria-describedby="CR90">90</a></sup>. Briefly, we performed nonlinear dimensionality reduction using diffusion maps (scanpy tl.diffmap with 15 components), and the neighborhood graph (scanpy pp.neighbors) was realized based on the 15 components of diffusion maps. We used PAGA<sup><a href="#CR123" class="usa-link" aria-describedby="CR123">123</a></sup> with force-directed graph from scanpy library to reproduce the abstracted graph of partitions and visualize the development trajectories. From the differential analysis between 16TF-VPN<sup>-</sup> and 16TF-VPN<sup>+</sup> cells we selected genes with a <em>p</em>-value &lt; 0.05, a Log Fold Change &gt; 2 and pct.2 &lt; 0.6 to filter up regulated genes in 16TF-VNP<sup>+</sup> cells and not expressed in more than 60% in 16TF-VNP<sup>-</sup> cells. The Wnt-TCF/LEF signature was composed of genes regulated by TCF/LEF-dependent Wnt signaling in adult muscle stem cells, previously described (Rudolf et al.,<sup><a href="#CR92" class="usa-link" aria-describedby="CR92">92</a></sup> Parisi et al.,<sup><a href="#CR91" class="usa-link" aria-describedby="CR91">91</a></sup>). We then visualized target genes expression in human sample and were able to score each cell in function of the expression level of each signature gene sets (scanpy.tl.score_genes).</p></section><section id="Sec27"><h3 class="pmc_sec_title">Ethics</h3>
<p id="Par69">All experiments conducted on chicken embryos were done in accordance with French and European legislation, which does not require specific ethical approval for work on avian embryos (Article R214-88, Code rural et de la pêche maritime; Directive 2010/63/EU).</p></section><section id="Sec28"><h3 class="pmc_sec_title">Reporting summary</h3>
<p id="Par70">Further information on research design is available in the <a href="#MOESM5" class="usa-link">Nature Portfolio Reporting Summary</a> linked to this article.</p></section></section><section id="Sec29"><h2 class="pmc_sec_title">Supplementary information</h2>
<section class="sm xbox font-sm" id="MOESM1"><div class="media p"><div class="caption">
<a href="/articles/instance/12374003/bin/41467_2025_61767_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information</a><sup> (665MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM2"><div class="media p"><div class="caption">
<a href="/articles/instance/12374003/bin/41467_2025_61767_MOESM2_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link">41467_2025_61767_MOESM2_ESM.docx</a><sup> (14KB, docx) </sup><p>Description of Additional Supplementary Information</p>
</div></div></section><section class="sm xbox font-sm" id="MOESM3"><div class="media p"><div class="caption">
<a href="/articles/instance/12374003/bin/41467_2025_61767_MOESM3_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 1</a><sup> (928.9KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM4"><div class="media p"><div class="caption">
<a href="/articles/instance/12374003/bin/41467_2025_61767_MOESM4_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 2</a><sup> (530.7KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM5"><div class="media p"><div class="caption">
<a href="/articles/instance/12374003/bin/41467_2025_61767_MOESM5_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Reporting Summary</a><sup> (102.1KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM6"><div class="media p"><div class="caption">
<a href="/articles/instance/12374003/bin/41467_2025_61767_MOESM6_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Transparent Peer Review file</a><sup> (3.8MB, pdf) </sup>
</div></div></section></section><section id="Sec30"><h2 class="pmc_sec_title">Source data</h2>
<section class="sm xbox font-sm" id="MOESM7"><div class="media p"><div class="caption">
<a href="/articles/instance/12374003/bin/41467_2025_61767_MOESM7_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Source Data</a><sup> (35.3KB, xlsx) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>We thank Drs Frank Stockdale, Frédéric Relaix, and Jean-Louis Bessereau for critical reading of the manuscript and insightful discussions on this study. We thank the Centre d’Imagerie Quantitative Lyon-Est (CIQLE) for imaging support. We thank Cyril Dégletagne from the Plateforme de Génomique des Cancers for support with single-cell RNA-seq. This research has been supported by a grant from the Agence Nationale de la Recherche (ANR) and a grant from AFM-Telethon (MyoNeurALP). GT has received a PhD scholarship from the Ecole Normale Supérieure de Lyon (CDSN).</p></section><section id="notes1"><h2 class="pmc_sec_title">Author contributions</h2>
<p>Conceptualization: G.T., C.M., F.L.G. Methodology: G.T., C.M. Formal analysis: W.J., F.L.G., S.J. Investigation: G.T., V.M. Funding acquisition: C.M., F.L.G. Supervision: C.M., F.L.G. Validation: G.T. Writing—original draft: G.T., C.M. Writing—review and editing: C.M., G.T.</p></section><section id="notes2"><h2 class="pmc_sec_title">Peer review</h2>
<section id="FPar1"><h3 class="pmc_sec_title">Peer review information</h3>
<p id="Par71"><em>Nature Communications</em> thanks Sam Mathew and the other anonymous reviewer(s) for their contribution to the peer review of this work. A peer review file is available.</p></section></section><section id="notes3"><h2 class="pmc_sec_title">Data availability</h2>
<p>scRNA-sequencing data sets for chicken embryos are available at the Gene Expression Omnibys (GEO) under the accession number <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE283478" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE283478</a>. <a href="#Sec30" class="usa-link">Source data</a> are provided with this paper.</p></section><section id="notes4"><h2 class="pmc_sec_title">Code availability</h2>
<p>Code is available upon request.</p></section><section id="FPar2"><h2 class="pmc_sec_title">Competing interests</h2>
<p id="Par72">The authors declare no competing interests</p></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm">
<div class="fn p" id="fn1"><p><strong>Publisher’s note</strong> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></div>
<div class="fn p" id="fn2"><p>These authors contributed equally: William Jarassier, Sabrina Jagot.</p></div>
</div></section><section id="_ci93_" lang="en" class="contrib-info"><h2 class="pmc_sec_title">Contributor Information</h2>
<p>Fabien Le Grand, Email: fabien.le-grand@univ-lyon1.fr.</p>
<p>Christophe Marcelle, Email: christophe.marcelle@univ-lyon1.fr.</p></section><section id="sec31"><h2 class="pmc_sec_title">Supplementary information</h2>
<p>The online version contains supplementary material available at 10.1038/s41467-025-61767-1.</p></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Christ, B. &amp; Ordahl, C. Early stages of chick somite development. <em>Anat. Embryol.</em><strong>191</strong>, 381–396 (1995).</cite> [<a href="https://doi.org/10.1007/BF00304424" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/7625610/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Christ,%20B.%20&amp;%20Ordahl,%20C.%20Early%20stages%20of%20chick%20somite%20development.%20Anat.%20Embryol.191,%20381%E2%80%93396%20(1995)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>Bryson-Richardson, R. J. &amp; Currie, P. D. The genetics of vertebrate myogenesis. <em>Nat. Rev. Genet. 2008 98</em><strong>9</strong>, 632–646 (2008).</cite> [<a href="https://doi.org/10.1038/nrg2369" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18636072/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bryson-Richardson,%20R.%20J.%20&amp;%20Currie,%20P.%20D.%20The%20genetics%20of%20vertebrate%20myogenesis.%20Nat.%20Rev.%20Genet.%202008%20989,%20632%E2%80%93646%20(2008)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>Hirst, C. E. C. E. &amp; Marcelle, C. <em>The avian embryo as a model system for skeletal myogenesis</em>. <em>Results and Problems in Cell Differentiation</em><strong>56</strong>, 99–122 (Springer, 2015).</cite> [<a href="https://doi.org/10.1007/978-3-662-44608-9_5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25344668/" class="usa-link">PubMed</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Scaal, M. &amp; Marcelle, C. Chick muscle development. <em>Int. J. Dev. Biol.</em><strong>62</strong>, 127–136 (2018).
</cite> [<a href="https://doi.org/10.1387/ijdb.170312cm" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29616720/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Scaal,%20M.%20&amp;%20Marcelle,%20C.%20Chick%20muscle%20development.%20Int.%20J.%20Dev.%20Biol.62,%20127%E2%80%93136%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Gros, J., Scaal, M. &amp; Marcelle, C. A two-Step mechanism for myotome formation in chick. <em>Dev. Cell</em><strong>6</strong>, 875-882 (2004).</cite> [<a href="https://doi.org/10.1016/j.devcel.2004.05.006" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15177035/" class="usa-link">PubMed</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Gros, J., Manceau, M., Thomé, V. &amp; Marcelle, C. A common somitic origin for embryonic muscle progenitors and satellite cells. <em>Nature</em><strong>435</strong>, 954–958 (2005).
</cite> [<a href="https://doi.org/10.1038/nature03572" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15843802/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gros,%20J.,%20Manceau,%20M.,%20Thom%C3%A9,%20V.%20&amp;%20Marcelle,%20C.%20A%20common%20somitic%20origin%20for%20embryonic%20muscle%20progenitors%20and%20satellite%20cells.%20Nature435,%20954%E2%80%93958%20(2005)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Kalcheim, C., Cinnamon, Y. &amp; Kahane, N. Myotome formation: a multistage process. <em>Cell Tissue Res.</em><strong>296</strong>, 161–173 (1999).
</cite> [<a href="https://doi.org/10.1007/s004410051277" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10199976/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kalcheim,%20C.,%20Cinnamon,%20Y.%20&amp;%20Kahane,%20N.%20Myotome%20formation:%20a%20multistage%20process.%20Cell%20Tissue%20Res.296,%20161%E2%80%93173%20(1999)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>Chal, J. &amp; Pourquié, O. Making muscle: skeletal myogenesis in vivo and in vitro. <em>Development</em><strong>144</strong>, 2104–2122 (2017).
</cite> [<a href="https://doi.org/10.1242/dev.151035" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28634270/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chal,%20J.%20&amp;%20Pourqui%C3%A9,%20O.%20Making%20muscle:%20skeletal%20myogenesis%20in%20vivo%20and%20in%20vitro.%20Development144,%202104%E2%80%932122%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>Ontell, M. &amp; Kozeka, K. The organogenesis of murine striated muscle: a cytoarchitectural study. <em>Am. J. Anat.</em><strong>171</strong>, 133–148 (1984).
</cite> [<a href="https://doi.org/10.1002/aja.1001710202" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/6496372/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ontell,%20M.%20&amp;%20Kozeka,%20K.%20The%20organogenesis%20of%20murine%20striated%20muscle:%20a%20cytoarchitectural%20study.%20Am.%20J.%20Anat.171,%20133%E2%80%93148%20(1984)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>Stockdale, F. E. Myogenic cell lineages. <em>Dev. Biol.</em><strong>154</strong>, 284–298 (1992).
</cite> [<a href="https://doi.org/10.1016/0012-1606(92)90068-r" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/1426639/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Stockdale,%20F.%20E.%20Myogenic%20cell%20lineages.%20Dev.%20Biol.154,%20284%E2%80%93298%20(1992)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Kelly, A. M. &amp; Zacks, S. I. The histogenesis of rat intercostal muscle. <em>J. Cell Biol.</em><strong>42</strong>, 135–153 (1969).
</cite> [<a href="https://doi.org/10.1083/jcb.42.1.135" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2107573/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/5786979/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kelly,%20A.%20M.%20&amp;%20Zacks,%20S.%20I.%20The%20histogenesis%20of%20rat%20intercostal%20muscle.%20J.%20Cell%20Biol.42,%20135%E2%80%93153%20(1969)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Duxson, M. J., Usson, Y. &amp; Harris, A. J. The origin of secondary myotubes in mammalian skeletal muscles: ultrastructural studies. <em>Development</em><strong>107</strong>, 743–750 (1989).
</cite> [<a href="https://doi.org/10.1242/dev.107.4.743" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/2483685/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Duxson,%20M.%20J.,%20Usson,%20Y.%20&amp;%20Harris,%20A.%20J.%20The%20origin%20of%20secondary%20myotubes%20in%20mammalian%20skeletal%20muscles:%20ultrastructural%20studies.%20Development107,%20743%E2%80%93750%20(1989)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>Crow, M. T. &amp; Stockdale, F. E. Myosin expression and specialization among the earliest muscle fibers of the developing avian limb. <em>Dev. Biol.</em><strong>113</strong>, 238–254 (1986).
</cite> [<a href="https://doi.org/10.1016/0012-1606(86)90126-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/3943663/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Crow,%20M.%20T.%20&amp;%20Stockdale,%20F.%20E.%20Myosin%20expression%20and%20specialization%20among%20the%20earliest%20muscle%20fibers%20of%20the%20developing%20avian%20limb.%20Dev.%20Biol.113,%20238%E2%80%93254%20(1986)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>Lance-Jones, C. &amp; Landmesser, L. Pathway selection by chick lumbosacral motoneurons during normal development. <em>Proc. R. Soc. Lond. Ser. B. Biol. Sci.</em><strong>214</strong>, 1–18 (1981).
</cite> [<a href="https://doi.org/10.1098/rspb.1981.0079" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/6121327/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lance-Jones,%20C.%20&amp;%20Landmesser,%20L.%20Pathway%20selection%20by%20chick%20lumbosacral%20motoneurons%20during%20normal%20development.%20Proc.%20R.%20Soc.%20Lond.%20Ser.%20B.%20Biol.%20Sci.214,%201%E2%80%9318%20(1981)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>Richard, A. F. et al. Genesis of muscle fiber-type diversity during mouse embryogenesis relies on Six1 and Six4 gene expression. <em>Dev. Biol.</em><strong>359</strong>, 303–320 (2011).
</cite> [<a href="https://doi.org/10.1016/j.ydbio.2011.08.010" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21884692/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Richard,%20A.%20F.%20et%20al.%20Genesis%20of%20muscle%20fiber-type%20diversity%20during%20mouse%20embryogenesis%20relies%20on%20Six1%20and%20Six4%20gene%20expression.%20Dev.%20Biol.359,%20303%E2%80%93320%20(2011)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Harris, A. J. Embryonic growth and innervation of rat skeletal muscles I. Neural regulation of muscle fibre numbers. <em>Philos. Trans. R. Soc. Lond. B Biol. Sci.</em><strong>293</strong>, 257–277 (1981).
</cite> [<a href="https://doi.org/10.1098/rstb.1981.0076" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/6116248/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Harris,%20A.%20J.%20Embryonic%20growth%20and%20innervation%20of%20rat%20skeletal%20muscles%20I.%20Neural%20regulation%20of%20muscle%20fibre%20numbers.%20Philos.%20Trans.%20R.%20Soc.%20Lond.%20B%20Biol.%20Sci.293,%20257%E2%80%93277%20(1981)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>McLennan, I. S. The development of the pattern of innervation in chicken hindlimb muscles: Evidence for specification of nerve-muscle connections. <em>Dev. Biol.</em><strong>97</strong>, 229–238 (1983).
</cite> [<a href="https://doi.org/10.1016/0012-1606(83)90080-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/6188640/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?McLennan,%20I.%20S.%20The%20development%20of%20the%20pattern%20of%20innervation%20in%20chicken%20hindlimb%20muscles:%20Evidence%20for%20specification%20of%20nerve-muscle%20connections.%20Dev.%20Biol.97,%20229%E2%80%93238%20(1983)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>McLennan, I. S. Neural dependence and independence of myotube production in chicken hindlimb muscles. <em>Dev. Biol.</em><strong>98</strong>, 287–294 (1983).
</cite> [<a href="https://doi.org/10.1016/0012-1606(83)90359-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/6873458/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?McLennan,%20I.%20S.%20Neural%20dependence%20and%20independence%20of%20myotube%20production%20in%20chicken%20hindlimb%20muscles.%20Dev.%20Biol.98,%20287%E2%80%93294%20(1983)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Condon, K., Silberstein, L., Blau, H. M. &amp; Thompson, W. J. Development of muscle fiber types in the prenatal rat hindlimb. <em>Dev. Biol.</em><strong>138</strong>, 256–274 (1990).
</cite> [<a href="https://doi.org/10.1016/0012-1606(90)90196-p" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/2108065/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Condon,%20K.,%20Silberstein,%20L.,%20Blau,%20H.%20M.%20&amp;%20Thompson,%20W.%20J.%20Development%20of%20muscle%20fiber%20types%20in%20the%20prenatal%20rat%20hindlimb.%20Dev.%20Biol.138,%20256%E2%80%93274%20(1990)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>Kelly, A. M. &amp; Rubinstein, N. A. Why are fetal muscles slow?. <em>Nature</em><strong>288</strong>, 266–269 (1980).
</cite> [<a href="https://doi.org/10.1038/288266a0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/7001253/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kelly,%20A.%20M.%20&amp;%20Rubinstein,%20N.%20A.%20Why%20are%20fetal%20muscles%20slow?.%20Nature288,%20266%E2%80%93269%20(1980)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>Narusawa, M. et al. Slow myosin in developing rat skeletal muscle. <em>J. Cell Biol.</em><strong>104</strong>, 447–459 (1987).
</cite> [<a href="https://doi.org/10.1083/jcb.104.3.447" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2114541/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/3546335/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Narusawa,%20M.%20et%20al.%20Slow%20myosin%20in%20developing%20rat%20skeletal%20muscle.%20J.%20Cell%20Biol.104,%20447%E2%80%93459%20(1987)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>Schiaffino, S., Rossi, A. C., Smerdu, V., Leinwand, L. A. &amp; Reggiani, C. Developmental myosins: expression patterns and functional significance. <em>Skelet. Muscle</em><strong>5</strong>, 22 (2015).</cite> [<a href="https://doi.org/10.1186/s13395-015-0046-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4502549/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26180627/" class="usa-link">PubMed</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>Hauschka, S. D. Clonal analysis of vertebrate myogenesis. 3. Developmental changes in the muscle-colony-forming cells of the human fetal limb. <em>Dev. Biol.</em><strong>37</strong>, 345–368 (1974).
</cite> [<a href="https://doi.org/10.1016/0012-1606(74)90154-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/4826281/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hauschka,%20S.%20D.%20Clonal%20analysis%20of%20vertebrate%20myogenesis.%203.%20Developmental%20changes%20in%20the%20muscle-colony-forming%20cells%20of%20the%20human%20fetal%20limb.%20Dev.%20Biol.37,%20345%E2%80%93368%20(1974)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>Miller, J. B. &amp; Stockdale, F. E. Developmental origins of skeletal muscle fibers: clonal analysis of myogenic cell lineages based on expression of fast and slow myosin heavy chains. <em>Proc. Natl. Acad. Sci. USA</em><strong>83</strong>, 3860–3864 (1986).
</cite> [<a href="https://doi.org/10.1073/pnas.83.11.3860" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC323624/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/3520558/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Miller,%20J.%20B.%20&amp;%20Stockdale,%20F.%20E.%20Developmental%20origins%20of%20skeletal%20muscle%20fibers:%20clonal%20analysis%20of%20myogenic%20cell%20lineages%20based%20on%20expression%20of%20fast%20and%20slow%20myosin%20heavy%20chains.%20Proc.%20Natl.%20Acad.%20Sci.%20USA83,%203860%E2%80%933864%20(1986)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>Miller, J. B. &amp; Stockdale, F. E. Developmental regulation of the multiple myogenic cell lineages of the avian embryo. <em>J. Cell Biol.</em><strong>103</strong>, 2197–2208 (1986).
</cite> [<a href="https://doi.org/10.1083/jcb.103.6.2197" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2114613/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/3782296/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Miller,%20J.%20B.%20&amp;%20Stockdale,%20F.%20E.%20Developmental%20regulation%20of%20the%20multiple%20myogenic%20cell%20lineages%20of%20the%20avian%20embryo.%20J.%20Cell%20Biol.103,%202197%E2%80%932208%20(1986)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>Mouly, V., Toutant, M. &amp; Fiszman, M. Y. Chick and quail limb bud myoblasts, isolated at different times during muscle development, express stage-specific phenotypes when differentiated in culture. <em>Cell Differ</em>. <strong>20</strong>, 17–25 (1987).</cite> [<a href="https://doi.org/10.1016/0045-6039(87)90461-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/2434244/" class="usa-link">PubMed</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Tajbakhsh, S. Skeletal muscle stem and progenitor cells: reconciling genetics and lineage. <em>Exp. Cell Res.</em><strong>306</strong>, 364–372 (2005).
</cite> [<a href="https://doi.org/10.1016/j.yexcr.2005.03.033" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15882864/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tajbakhsh,%20S.%20Skeletal%20muscle%20stem%20and%20progenitor%20cells:%20reconciling%20genetics%20and%20lineage.%20Exp.%20Cell%20Res.306,%20364%E2%80%93372%20(2005)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Biressi, S., Molinaro, M. &amp; Cossu, G. Cellular heterogeneity during vertebrate skeletal muscle development. <em>Dev. Biol.</em><strong>308</strong>, 281–293 (2007).
</cite> [<a href="https://doi.org/10.1016/j.ydbio.2007.06.006" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17612520/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Biressi,%20S.,%20Molinaro,%20M.%20&amp;%20Cossu,%20G.%20Cellular%20heterogeneity%20during%20vertebrate%20skeletal%20muscle%20development.%20Dev.%20Biol.308,%20281%E2%80%93293%20(2007)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>Biressi, S. et al. Intrinsic phenotypic diversity of embryonic and fetal myoblasts is revealed by genome-wide gene expression analysis on purified cells. <em>Dev. Biol.</em><strong>304</strong>, 633–651 (2007).
</cite> [<a href="https://doi.org/10.1016/j.ydbio.2007.01.016" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17292343/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Biressi,%20S.%20et%20al.%20Intrinsic%20phenotypic%20diversity%20of%20embryonic%20and%20fetal%20myoblasts%20is%20revealed%20by%20genome-wide%20gene%20expression%20analysis%20on%20purified%20cells.%20Dev.%20Biol.304,%20633%E2%80%93651%20(2007)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR30">
<span class="label">30.</span><cite>Messina, G. et al. Nfix regulates fetal-specific transcription in developing skeletal muscle. <em>Cell</em><strong>140</strong>, 554–566 (2010).
</cite> [<a href="https://doi.org/10.1016/j.cell.2010.01.027" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20178747/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Messina,%20G.%20et%20al.%20Nfix%20regulates%20fetal-specific%20transcription%20in%20developing%20skeletal%20muscle.%20Cell140,%20554%E2%80%93566%20(2010)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>Murphy, M. &amp; Kardon, G. Origin of vertebrate limb muscle: the role of progenitor and myoblast populations. <em>Curr. Top. Dev. Biol.</em><strong>96</strong>, 1–32 (2011).
</cite> [<a href="https://doi.org/10.1016/B978-0-12-385940-2.00001-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5573708/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21621065/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Murphy,%20M.%20&amp;%20Kardon,%20G.%20Origin%20of%20vertebrate%20limb%20muscle:%20the%20role%20of%20progenitor%20and%20myoblast%20populations.%20Curr.%20Top.%20Dev.%20Biol.96,%201%E2%80%9332%20(2011)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>Seed, J. &amp; Hauschka, S. D. Temporal separation of the migration of distinct myogenic precursor populations into the developing chick wing bud. <em>Dev. Biol.</em><strong>106</strong>, 389–393 (1984).
</cite> [<a href="https://doi.org/10.1016/0012-1606(84)90237-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/6500180/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Seed,%20J.%20&amp;%20Hauschka,%20S.%20D.%20Temporal%20separation%20of%20the%20migration%20of%20distinct%20myogenic%20precursor%20populations%20into%20the%20developing%20chick%20wing%20bud.%20Dev.%20Biol.106,%20389%E2%80%93393%20(1984)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Cossu, G. &amp; Biressi, S. Satellite cells, myoblasts and other occasional myogenic progenitors: possible origin, phenotypic features and role in muscle regeneration. <em>Semin. Cell Dev. Biol.</em><strong>16</strong>, 623–631 (2005).
</cite> [<a href="https://doi.org/10.1016/j.semcdb.2005.07.003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16118057/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cossu,%20G.%20&amp;%20Biressi,%20S.%20Satellite%20cells,%20myoblasts%20and%20other%20occasional%20myogenic%20progenitors:%20possible%20origin,%20phenotypic%20features%20and%20role%20in%20muscle%20regeneration.%20Semin.%20Cell%20Dev.%20Biol.16,%20623%E2%80%93631%20(2005)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>Stockdale, F. E. &amp; Miller, J. B. The cellular basis of myosin heavy chain isoform expression during development of avian skeletal muscles. <em>Dev. Biol.</em><strong>123</strong>, 1–9 (1987).
</cite> [<a href="https://doi.org/10.1016/0012-1606(87)90420-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/3305110/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Stockdale,%20F.%20E.%20&amp;%20Miller,%20J.%20B.%20The%20cellular%20basis%20of%20myosin%20heavy%20chain%20isoform%20expression%20during%20development%20of%20avian%20skeletal%20muscles.%20Dev.%20Biol.123,%201%E2%80%939%20(1987)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>Barzilai-Tutsch, H. et al. TopFlash transgenic quail reveals dynamic TCF/β-catenin signaling during avian embryonic development. <em>bioRxiv</em> 2021.06.11.448089 10.1101/2021.06.11.448089 (2021).</cite>
</li>
<li id="CR36">
<span class="label">36.</span><cite>Sieiro-Mosti, D., De La Celle, M., Pelé, M. &amp; Marcelle, C. A dynamic analysis of muscle fusion in the chick embryo. <em>Development</em>. <strong>141</strong>, 3605-3611 (2014).</cite> [<a href="https://doi.org/10.1242/dev.114546" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25183875/" class="usa-link">PubMed</a>]</li>
<li id="CR37">
<span class="label">37.</span><cite>Koga, A., Suzuki, M., Inagaki, H., Bessho, Y. &amp; Hori, H. Transposable element in fish. <em>Nat 1996 3836595</em><strong>383</strong>, 30–30 (1996).</cite> [<a href="https://doi.org/10.1038/383030a0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/8779712/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Koga,%20A.,%20Suzuki,%20M.,%20Inagaki,%20H.,%20Bessho,%20Y.%20&amp;%20Hori,%20H.%20Transposable%20element%20in%20fish.%20Nat%201996%203836595383,%2030%E2%80%9330%20(1996)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR38">
<span class="label">38.</span><cite>Yvernogeau, L., Auda-Boucher, G. &amp; Fontaine-Perus, J. Limb bud colonization by somite-derived angioblasts is a crucial step for myoblast emigration. <em>Development</em>. <strong>139</strong>, 277–287 (2012).</cite> [<a href="https://doi.org/10.1242/dev.067678" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22129828/" class="usa-link">PubMed</a>]</li>
<li id="CR39">
<span class="label">39.</span><cite>Buckingham, M. &amp; Vincent, S. D. Distinct and dynamic myogenic populations in the vertebrate embryo. <em>Curr. Opin. Genet. Dev.</em><strong>19</strong>, 444–453 (2009).
</cite> [<a href="https://doi.org/10.1016/j.gde.2009.08.001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19762225/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Buckingham,%20M.%20&amp;%20Vincent,%20S.%20D.%20Distinct%20and%20dynamic%20myogenic%20populations%20in%20the%20vertebrate%20embryo.%20Curr.%20Opin.%20Genet.%20Dev.19,%20444%E2%80%93453%20(2009)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR40">
<span class="label">40.</span><cite>Bryson-Richardson, R. J. &amp; Currie, P. D. <em>The genetics of vertebrate myogenesis</em>. <em>Nature Reviews Genetics</em><strong>9</strong>, 632–646 (Nature Publishing Group, 2008).</cite> [<a href="https://doi.org/10.1038/nrg2369" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18636072/" class="usa-link">PubMed</a>]</li>
<li id="CR41">
<span class="label">41.</span><cite>Picard, C. A. &amp; Marcelle, C. Two distinct muscle progenitor populations coexist throughout amniote development. <em>Dev. Biol</em>. <strong>373</strong>, 141-148 (2013).</cite> [<a href="https://doi.org/10.1016/j.ydbio.2012.10.018" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23085413/" class="usa-link">PubMed</a>]</li>
<li id="CR42">
<span class="label">42.</span><cite>Manceau, M. et al. Myostatin promotes the terminal differentiation of embryonic muscle progenitors. <em>Genes Dev</em>. <strong>22</strong>, 668-681 (2008).</cite> [<a href="https://doi.org/10.1101/gad.454408" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2259035/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18316481/" class="usa-link">PubMed</a>]</li>
<li id="CR43">
<span class="label">43.</span><cite>Venters, S. J. et al. Precocious terminal differentiation of premigratory limb muscle precursor cells requires positive signalling. <em>Dev. Dyn</em>. <strong>229</strong>, (2004).</cite> [<a href="https://doi.org/10.1002/dvdy.20016" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/14991714/" class="usa-link">PubMed</a>]</li>
<li id="CR44">
<span class="label">44.</span><cite>Kawakami, A., Kimura-Kawakami, M., Nomura, T. &amp; Fujisawa, H. Distributions of PAX6 and PAX7 proteins suggest their involvement in both early and late phases of chick brain development. <em>Mech. Dev.</em><strong>66</strong>, 119–130 (1997).
</cite> [<a href="https://doi.org/10.1016/s0925-4773(97)00097-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9376315/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kawakami,%20A.,%20Kimura-Kawakami,%20M.,%20Nomura,%20T.%20&amp;%20Fujisawa,%20H.%20Distributions%20of%20PAX6%20and%20PAX7%20proteins%20suggest%20their%20involvement%20in%20both%20early%20and%20late%20phases%20of%20chick%20brain%20development.%20Mech.%20Dev.66,%20119%E2%80%93130%20(1997)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR45">
<span class="label">45.</span><cite>Heng, Y. C. &amp; Foo, J. L. Development of destabilized mCherry fluorescent proteins for applications in the model yeast Saccharomyces cerevisiae. <em>Biotechnol. Notes</em><strong>3</strong>, 108–112 (2022).
</cite> [<a href="https://doi.org/10.1016/j.biotno.2022.12.001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11446383/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39416457/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Heng,%20Y.%20C.%20&amp;%20Foo,%20J.%20L.%20Development%20of%20destabilized%20mCherry%20fluorescent%20proteins%20for%20applications%20in%20the%20model%20yeast%20Saccharomyces%20cerevisiae.%20Biotechnol.%20Notes3,%20108%E2%80%93112%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR46">
<span class="label">46.</span><cite>He, L., Binari, R., Huang, J., Falo-Sanjuan, J. &amp; Perrimon, N. In vivo study of gene expression with an enhanced dual-color fluorescent transcriptional timer. <em>Elife</em>10.7554/eLife.46181 (2019).</cite> [<a href="https://doi.org/10.7554/eLife.46181" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6660218/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31140975/" class="usa-link">PubMed</a>]</li>
<li id="CR47">
<span class="label">47.</span><cite>Gossen, M. &amp; Bujard, H. Tight control of gene expression in mammalian cells by tetracycline-responsive promoters. <em>Proc. Natl. Acad. Sci. USA</em><strong>89</strong>, 5547–5551 (1992).
</cite> [<a href="https://doi.org/10.1073/pnas.89.12.5547" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC49329/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/1319065/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gossen,%20M.%20&amp;%20Bujard,%20H.%20Tight%20control%20of%20gene%20expression%20in%20mammalian%20cells%20by%20tetracycline-responsive%20promoters.%20Proc.%20Natl.%20Acad.%20Sci.%20USA89,%205547%E2%80%935551%20(1992)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR48">
<span class="label">48.</span><cite>Sprengel, R. &amp; Hasan, M. T. Tetracycline-controlled genetic switches. <em>Handb. Exp. Pharmacol.</em><strong>178</strong>, 49–72 (2007).</cite> [<a href="https://doi.org/10.1007/978-3-540-35109-2_3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17203651/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sprengel,%20R.%20&amp;%20Hasan,%20M.%20T.%20Tetracycline-controlled%20genetic%20switches.%20Handb.%20Exp.%20Pharmacol.178,%2049%E2%80%9372%20(2007)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR49">
<span class="label">49.</span><cite>Alaei, S. et al. An improved reprogrammable mouse model harbouring the reverse tetracycline-controlled transcriptional transactivator 3. <em>Stem Cell Res.</em><strong>17</strong>, 49–53 (2016).
</cite> [<a href="https://doi.org/10.1016/j.scr.2016.05.008" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27558603/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Alaei,%20S.%20et%20al.%20An%20improved%20reprogrammable%20mouse%20model%20harbouring%20the%20reverse%20tetracycline-controlled%20transcriptional%20transactivator%203.%20Stem%20Cell%20Res.17,%2049%E2%80%9353%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR50">
<span class="label">50.</span><cite>Das, A. T. et al. Viral evolution as a tool to improve the tetracycline-regulated gene expression system. <em>J. Biol. Chem.</em><strong>279</strong>, 18776–18782 (2004).
</cite> [<a href="https://doi.org/10.1074/jbc.M313895200" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/14761948/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Das,%20A.%20T.%20et%20al.%20Viral%20evolution%20as%20a%20tool%20to%20improve%20the%20tetracycline-regulated%20gene%20expression%20system.%20J.%20Biol.%20Chem.279,%2018776%E2%80%9318782%20(2004)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR51">
<span class="label">51.</span><cite>Chassin, H. et al. A modular degron library for synthetic circuits in mammalian cells. <em>Nat. Commun.</em><strong>10</strong>, 2013 (2019).
</cite> [<a href="https://doi.org/10.1038/s41467-019-09974-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6494899/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31043592/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chassin,%20H.%20et%20al.%20A%20modular%20degron%20library%20for%20synthetic%20circuits%20in%20mammalian%20cells.%20Nat.%20Commun.10,%202013%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR52">
<span class="label">52.</span><cite>Rechsteiner, M. &amp; Rogers, S. W. PEST sequences and regulation by proteolysis. <em>Trends Biochem. Sci.</em><strong>21</strong>, 267–271 (1996).
</cite> [<a href="https://pubmed.ncbi.nlm.nih.gov/8755249/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rechsteiner,%20M.%20&amp;%20Rogers,%20S.%20W.%20PEST%20sequences%20and%20regulation%20by%20proteolysis.%20Trends%20Biochem.%20Sci.21,%20267%E2%80%93271%20(1996)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR53">
<span class="label">53.</span><cite>Hutcheson, D. A., Zhao, J., Merrell, A., Haldar, M. &amp; Kardon, G. Embryonic and fetal limb myogenic cells are derived from developmentally distinct progenitors and have different requirements for β-catenin. <em>Genes Dev.</em><strong>23</strong>, 997–1013 (2009).
</cite> [<a href="https://doi.org/10.1101/gad.1769009" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2675868/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19346403/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hutcheson,%20D.%20A.,%20Zhao,%20J.,%20Merrell,%20A.,%20Haldar,%20M.%20&amp;%20Kardon,%20G.%20Embryonic%20and%20fetal%20limb%20myogenic%20cells%20are%20derived%20from%20developmentally%20distinct%20progenitors%20and%20have%20different%20requirements%20for%20%CE%B2-catenin.%20Genes%20Dev.23,%20997%E2%80%931013%20(2009)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR54">
<span class="label">54.</span><cite>Linker, C. et al. β-catenin-dependent Wnt signalling controls the epithelial organisation of somites through the activation of paraxis. <em>Development</em><strong>132</strong>, 3895-3905 (2005).</cite> [<a href="https://doi.org/10.1242/dev.01961" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16100089/" class="usa-link">PubMed</a>]</li>
<li id="CR55">
<span class="label">55.</span><cite>Geetha-Loganathan, P., Nimmagadda, S., Huang, R., Christ, B. &amp; Scaal, M. Regulation of ectodermal Wnt6 expression by the neural tube is transduced by dermomyotomal Wnt11: a mechanism of dermomyotomal lip sustainment. <em>Development</em><strong>133</strong>, 2897–2904 (2006).
</cite> [<a href="https://doi.org/10.1242/dev.02464" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16818447/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Geetha-Loganathan,%20P.,%20Nimmagadda,%20S.,%20Huang,%20R.,%20Christ,%20B.%20&amp;%20Scaal,%20M.%20Regulation%20of%20ectodermal%20Wnt6%20expression%20by%20the%20neural%20tube%20is%20transduced%20by%20dermomyotomal%20Wnt11:%20a%20mechanism%20of%20dermomyotomal%20lip%20sustainment.%20Development133,%202897%E2%80%932904%20(2006)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR56">
<span class="label">56.</span><cite>Donoghue, M., Ernst, H., Wentworth, B., Nadal-Ginard, B. &amp; Rosenthal, N. A muscle-specific enhancer is located at the 3’ end of the myosin light-chain 1/3 gene locus. <em>Genes Dev</em>. 10.1101/gad.2.12b.1779 (1988).</cite> [<a href="https://doi.org/10.1101/gad.2.12b.1779" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/3240859/" class="usa-link">PubMed</a>]</li>
<li id="CR57">
<span class="label">57.</span><cite>Fredette, B. J. &amp; Landmesser, L. T. A reevaluation of the role of innervation in primary and secondary myogenesis in developing chick muscle. <em>Dev. Biol.</em><strong>143</strong>, 19–35 (1991).
</cite> [<a href="https://doi.org/10.1016/0012-1606(91)90051-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/1824627/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Fredette,%20B.%20J.%20&amp;%20Landmesser,%20L.%20T.%20A%20reevaluation%20of%20the%20role%20of%20innervation%20in%20primary%20and%20secondary%20myogenesis%20in%20developing%20chick%20muscle.%20Dev.%20Biol.143,%2019%E2%80%9335%20(1991)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR58">
<span class="label">58.</span><cite>Relaix, F., Rocancourt, D., Mansouri, A. &amp; Buckingham, M. A Pax3/Pax7-dependent population of skeletal muscle progenitor cells. <em>Nature</em>10.1038/nature03594 (2005).</cite> [<a href="https://doi.org/10.1038/nature03594" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15843801/" class="usa-link">PubMed</a>]</li>
<li id="CR59">
<span class="label">59.</span><cite>Kassar-Duchossoy, L. et al. Pax3/Pax7 mark a novel population of primitive myogenic cells during development. <em>Genes Dev</em>. 10.1101/gad.345505 (2005).</cite> [<a href="https://doi.org/10.1101/gad.345505" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1151658/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15964993/" class="usa-link">PubMed</a>]</li>
<li id="CR60">
<span class="label">60.</span><cite>Feldman, J. L. &amp; Stockdale, F. E. Temporal appearance of satellite cells during myogenesis. <em>Dev. Biol.</em><strong>153</strong>, 217–226 (1992).
</cite> [<a href="https://doi.org/10.1016/0012-1606(92)90107-r" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/1397679/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Feldman,%20J.%20L.%20&amp;%20Stockdale,%20F.%20E.%20Temporal%20appearance%20of%20satellite%20cells%20during%20myogenesis.%20Dev.%20Biol.153,%20217%E2%80%93226%20(1992)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR61">
<span class="label">61.</span><cite>Hartley, R. S., Bandman, E. &amp; Yablonka-Reuveni, Z. Skeletal muscle satellite cells appear during late chicken embryogenesis. <em>Dev. Biol.</em><strong>153</strong>, 206–216 (1992).
</cite> [<a href="https://doi.org/10.1016/0012-1606(92)90106-q" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4075331/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/1397678/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hartley,%20R.%20S.,%20Bandman,%20E.%20&amp;%20Yablonka-Reuveni,%20Z.%20Skeletal%20muscle%20satellite%20cells%20appear%20during%20late%20chicken%20embryogenesis.%20Dev.%20Biol.153,%20206%E2%80%93216%20(1992)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR62">
<span class="label">62.</span><cite>von Maltzahn, J., Chang, N. C., Bentzinger, C. F. &amp; Rudnicki, M. A. Wnt signaling in myogenesis. <em>Trends Cell Biol.</em><strong>22</strong>, 602–609 (2012).
</cite> [<a href="https://doi.org/10.1016/j.tcb.2012.07.008" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3479319/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22944199/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?von%20Maltzahn,%20J.,%20Chang,%20N.%20C.,%20Bentzinger,%20C.%20F.%20&amp;%20Rudnicki,%20M.%20A.%20Wnt%20signaling%20in%20myogenesis.%20Trends%20Cell%20Biol.22,%20602%E2%80%93609%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR63">
<span class="label">63.</span><cite>Girardi, F. &amp; Le Grand, F. Wnt signaling in skeletal muscle development and regeneration. <em>Prog. Mol. Biol. Transl. Sci.</em><strong>153</strong>, 157–179 (2018).
</cite> [<a href="https://doi.org/10.1016/bs.pmbts.2017.11.026" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29389515/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Girardi,%20F.%20&amp;%20Le%20Grand,%20F.%20Wnt%20signaling%20in%20skeletal%20muscle%20development%20and%20regeneration.%20Prog.%20Mol.%20Biol.%20Transl.%20Sci.153,%20157%E2%80%93179%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR64">
<span class="label">64.</span><cite>Rios, A. C., Serralbo, O., Salgado, D. &amp; Marcelle, C. Neural crest regulates myogenesis through the transient activation of NOTCH. <em>Nature</em><strong>473</strong>, 532–535 (2011).
</cite> [<a href="https://doi.org/10.1038/nature09970" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21572437/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rios,%20A.%20C.,%20Serralbo,%20O.,%20Salgado,%20D.%20&amp;%20Marcelle,%20C.%20Neural%20crest%20regulates%20myogenesis%20through%20the%20transient%20activation%20of%20NOTCH.%20Nature473,%20532%E2%80%93535%20(2011)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR65">
<span class="label">65.</span><cite>Sieiro, D., Rios, A. C., Hirst, C. E. &amp; Marcelle, C. Cytoplasmic NOTCH and membrane-derived β-catenin link cell fate choice to epithelial-mesenchymal transition during myogenesis. <em>Elife</em><strong>5</strong>, e14847 (2016).</cite> [<a href="https://doi.org/10.7554/eLife.14847" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4917337/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27218451/" class="usa-link">PubMed</a>]</li>
<li id="CR66">
<span class="label">66.</span><cite>Geetha-Loganathan, P. et al. Ectodermal Wnt-6 promotes Myf5-dependent avian limb myogenesis. <em>Dev. Biol.</em><strong>288</strong>, 221–233 (2005).
</cite> [<a href="https://doi.org/10.1016/j.ydbio.2005.09.035" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16271265/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Geetha-Loganathan,%20P.%20et%20al.%20Ectodermal%20Wnt-6%20promotes%20Myf5-dependent%20avian%20limb%20myogenesis.%20Dev.%20Biol.288,%20221%E2%80%93233%20(2005)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR67">
<span class="label">67.</span><cite>Anakwe, K. et al. Wnt signalling regulates myogenic differentiation in the developing avian wing. <em>Development</em><strong>130</strong>, 3503–3514 (2003).
</cite> [<a href="https://doi.org/10.1242/dev.00538" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12810597/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Anakwe,%20K.%20et%20al.%20Wnt%20signalling%20regulates%20myogenic%20differentiation%20in%20the%20developing%20avian%20wing.%20Development130,%203503%E2%80%933514%20(2003)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR68">
<span class="label">68.</span><cite>Miller, K. A. et al. A highly conserved Wnt-dependent TCF4 binding site within the proximal enhancer of the anti-myogenic Msx1 gene supports expression within Pax3-expressing limb bud muscle precursor cells. <em>Dev. Biol.</em><strong>311</strong>, 665–678 (2007).
</cite> [<a href="https://doi.org/10.1016/j.ydbio.2007.07.022" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17727834/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Miller,%20K.%20A.%20et%20al.%20A%20highly%20conserved%20Wnt-dependent%20TCF4%20binding%20site%20within%20the%20proximal%20enhancer%20of%20the%20anti-myogenic%20Msx1%20gene%20supports%20expression%20within%20Pax3-expressing%20limb%20bud%20muscle%20precursor%20cells.%20Dev.%20Biol.311,%20665%E2%80%93678%20(2007)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR69">
<span class="label">69.</span><cite>Abu-Elmagd, M. et al. Wnt/Lef1 signaling acts via Pitx2 to regulate somite myogenesis. <em>Dev. Biol.</em><strong>337</strong>, 211–219 (2010).
</cite> [<a href="https://doi.org/10.1016/j.ydbio.2009.10.023" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19850024/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Abu-Elmagd,%20M.%20et%20al.%20Wnt/Lef1%20signaling%20acts%20via%20Pitx2%20to%20regulate%20somite%20myogenesis.%20Dev.%20Biol.337,%20211%E2%80%93219%20(2010)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR70">
<span class="label">70.</span><cite>Witte, F., Dokas, J., Neuendorf, F., Mundlos, S. &amp; Stricker, S. Comprehensive expression analysis of all Wnt genes and their major secreted antagonists during mouse limb development and cartilage differentiation. <em>Gene Expr. Patterns</em><strong>9</strong>, 215–223 (2009).
</cite> [<a href="https://doi.org/10.1016/j.gep.2008.12.009" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19185060/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Witte,%20F.,%20Dokas,%20J.,%20Neuendorf,%20F.,%20Mundlos,%20S.%20&amp;%20Stricker,%20S.%20Comprehensive%20expression%20analysis%20of%20all%20Wnt%20genes%20and%20their%20major%20secreted%20antagonists%20during%20mouse%20limb%20development%20and%20cartilage%20differentiation.%20Gene%20Expr.%20Patterns9,%20215%E2%80%93223%20(2009)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR71">
<span class="label">71.</span><cite>Church, V. L. &amp; Francis-West, P. Wnt signalling during limb development. The Wnt Gene Family. <em>Int. J. Dev. Biol.</em><strong>46</strong>, 927–936 (2002).
</cite> [<a href="https://pubmed.ncbi.nlm.nih.gov/12455630/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Church,%20V.%20L.%20&amp;%20Francis-West,%20P.%20Wnt%20signalling%20during%20limb%20development.%20The%20Wnt%20Gene%20Family.%20Int.%20J.%20Dev.%20Biol.46,%20927%E2%80%93936%20(2002)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR72">
<span class="label">72.</span><cite>Loganathan, P. G., Nimmagadda, S., Huang, R., Scaal, M. &amp; Christ, B. Comparative analysis of the expression patterns of Wnts during chick limb development. <em>Histochem. Cell Biol.</em><strong>123</strong>, 195–201 (2005).
</cite> [<a href="https://doi.org/10.1007/s00418-005-0756-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15778857/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Loganathan,%20P.%20G.,%20Nimmagadda,%20S.,%20Huang,%20R.,%20Scaal,%20M.%20&amp;%20Christ,%20B.%20Comparative%20analysis%20of%20the%20expression%20patterns%20of%20Wnts%20during%20chick%20limb%20development.%20Histochem.%20Cell%20Biol.123,%20195%E2%80%93201%20(2005)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR73">
<span class="label">73.</span><cite>Molenaar, M. et al. XTcf-3 transcription factor mediates beta-catenin-induced axis formation in Xenopus embryos. <em>Cell</em><strong>86</strong>, 391–399 (1996).
</cite> [<a href="https://doi.org/10.1016/s0092-8674(00)80112-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/8756721/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Molenaar,%20M.%20et%20al.%20XTcf-3%20transcription%20factor%20mediates%20beta-catenin-induced%20axis%20formation%20in%20Xenopus%20embryos.%20Cell86,%20391%E2%80%93399%20(1996)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR74">
<span class="label">74.</span><cite>Christ, B. &amp; Brand-Saberi, B. Limb muscle development. <em>Int. J. Dev. Biol.</em><strong>46</strong>, 905–914 (2002).
</cite> [<a href="https://pubmed.ncbi.nlm.nih.gov/12455628/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Christ,%20B.%20&amp;%20Brand-Saberi,%20B.%20Limb%20muscle%20development.%20Int.%20J.%20Dev.%20Biol.46,%20905%E2%80%93914%20(2002)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR75">
<span class="label">75.</span><cite>Birchmeier, C. &amp; Brohmann, H. Genes that control the development of migrating muscle precursor cells. <em>Curr. Opin. Cell Biol.</em><strong>12</strong>, 725–730 (2000).
</cite> [<a href="https://doi.org/10.1016/s0955-0674(00)00159-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11063939/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Birchmeier,%20C.%20&amp;%20Brohmann,%20H.%20Genes%20that%20control%20the%20development%20of%20migrating%20muscle%20precursor%20cells.%20Curr.%20Opin.%20Cell%20Biol.12,%20725%E2%80%93730%20(2000)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR76">
<span class="label">76.</span><cite>Buckingham, M. et al. The formation of skeletal muscle: from somite to limb. <em>J. Anat.</em><strong>202</strong>, 59–68 (2003).
</cite> [<a href="https://doi.org/10.1046/j.1469-7580.2003.00139.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1571050/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12587921/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Buckingham,%20M.%20et%20al.%20The%20formation%20of%20skeletal%20muscle:%20from%20somite%20to%20limb.%20J.%20Anat.202,%2059%E2%80%9368%20(2003)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR77">
<span class="label">77.</span><cite>Vasyutina, E. &amp; Birchmeier, C. The development of migrating muscle precursor cells. <em>Anat. Embryol</em>. <strong>211</strong>, 37–41 (2006).</cite> [<a href="https://doi.org/10.1007/s00429-006-0118-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16977478/" class="usa-link">PubMed</a>]</li>
<li id="CR78">
<span class="label">78.</span><cite>Vasyutina, E. et al. CXCR4 and Gab1 cooperate to control the development of migrating muscle progenitor cells. <em>Genes Dev.</em><strong>19</strong>, 2187–2198 (2005).
</cite> [<a href="https://doi.org/10.1101/gad.346205" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1221889/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16166380/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Vasyutina,%20E.%20et%20al.%20CXCR4%20and%20Gab1%20cooperate%20to%20control%20the%20development%20of%20migrating%20muscle%20progenitor%20cells.%20Genes%20Dev.19,%202187%E2%80%932198%20(2005)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR79">
<span class="label">79.</span><cite>Nguyen, T. M., Arthur, A., Zannettino, A. C. W. &amp; Gronthos, S. EphA5 and EphA7 forward signaling enhances human hematopoietic stem and progenitor cell maintenance, migration, and adhesion via Rac1 activation. <em>Exp. Hematol.</em><strong>48</strong>, 72–78 (2017).
</cite> [<a href="https://doi.org/10.1016/j.exphem.2016.12.001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27988259/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Nguyen,%20T.%20M.,%20Arthur,%20A.,%20Zannettino,%20A.%20C.%20W.%20&amp;%20Gronthos,%20S.%20EphA5%20and%20EphA7%20forward%20signaling%20enhances%20human%20hematopoietic%20stem%20and%20progenitor%20cell%20maintenance,%20migration,%20and%20adhesion%20via%20Rac1%20activation.%20Exp.%20Hematol.48,%2072%E2%80%9378%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR80">
<span class="label">80.</span><cite>Devilard, E., Xerri, L., Dubreuil, P., Lopez, M. &amp; Reymond, N. Nectin-3 (CD113) interacts with nectin-2 (CD112) to promote lymphocyte transendothelial migration. <em>PLoS One</em><strong>8</strong>, e77424 (2013).
</cite> [<a href="https://doi.org/10.1371/journal.pone.0077424" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3792040/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24116228/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Devilard,%20E.,%20Xerri,%20L.,%20Dubreuil,%20P.,%20Lopez,%20M.%20&amp;%20Reymond,%20N.%20Nectin-3%20(CD113)%20interacts%20with%20nectin-2%20(CD112)%20to%20promote%20lymphocyte%20transendothelial%20migration.%20PLoS%20One8,%20e77424%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR81">
<span class="label">81.</span><cite>Iguchi, T. et al. Mutually repulsive EphA7–EfnA5 organize region-to-region corticopontine projection by inhibiting collateral extension. <em>J. Neurosci.</em><strong>41</strong>, 4795–4808 (2021).
</cite> [<a href="https://doi.org/10.1523/JNEUROSCI.0367-20.2021" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8260171/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33906900/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Iguchi,%20T.%20et%20al.%20Mutually%20repulsive%20EphA7%E2%80%93EfnA5%20organize%20region-to-region%20corticopontine%20projection%20by%20inhibiting%20collateral%20extension.%20J.%20Neurosci.41,%204795%E2%80%934808%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR82">
<span class="label">82.</span><cite>Gleeson, J. G., Peter T, L., Flanagan, L. A. &amp; Walsh, C. A. Doublecortin is a microtubule-associated protein and is expressed widely by migrating neurons. <em>Neuron</em><strong>23</strong>, 257–271 (1999).
</cite> [<a href="https://doi.org/10.1016/s0896-6273(00)80778-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10399933/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gleeson,%20J.%20G.,%20Peter%20T,%20L.,%20Flanagan,%20L.%20A.%20&amp;%20Walsh,%20C.%20A.%20Doublecortin%20is%20a%20microtubule-associated%20protein%20and%20is%20expressed%20widely%20by%20migrating%20neurons.%20Neuron23,%20257%E2%80%93271%20(1999)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR83">
<span class="label">83.</span><cite>Yuan, J. et al. CADM1 promotes adhesion to vascular endothelial cells and transendothelial migration in cultured GIST cells. <em>Oncol. Lett</em>. <strong>23</strong>, 86 (2022).</cite> [<a href="https://doi.org/10.3892/ol.2022.13206" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8805184/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35126728/" class="usa-link">PubMed</a>]</li>
<li id="CR84">
<span class="label">84.</span><cite>Teng, Y., Ghoshal, P., Ngoka, L., Mei, Y. &amp; Cowell, J. K. Critical role of the WASF3 gene in JAK2/STAT3 regulation of cancer cell motility. <em>Carcinogenesis</em><strong>34</strong>, 1994 (2013).
</cite> [<a href="https://doi.org/10.1093/carcin/bgt167" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3765048/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23677069/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Teng,%20Y.,%20Ghoshal,%20P.,%20Ngoka,%20L.,%20Mei,%20Y.%20&amp;%20Cowell,%20J.%20K.%20Critical%20role%20of%20the%20WASF3%20gene%20in%20JAK2/STAT3%20regulation%20of%20cancer%20cell%20motility.%20Carcinogenesis34,%201994%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR85">
<span class="label">85.</span><cite>Coene, E. D. et al. A novel role for BRCA1 in regulating breast cancer cell spreading and motility. <em>J. Cell Biol.</em><strong>192</strong>, 497–512 (2011).
</cite> [<a href="https://doi.org/10.1083/jcb.201004136" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3101087/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21282464/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Coene,%20E.%20D.%20et%20al.%20A%20novel%20role%20for%20BRCA1%20in%20regulating%20breast%20cancer%20cell%20spreading%20and%20motility.%20J.%20Cell%20Biol.192,%20497%E2%80%93512%20(2011)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR86">
<span class="label">86.</span><cite>Guo, J. et al. PRRX1 promotes epithelial–mesenchymal transition through the Wnt/β-catenin pathway in gastric cancer. <em>Med. Oncol.</em><strong>32</strong>, 1–12 (2015).</cite> [<a href="https://doi.org/10.1007/s12032-014-0393-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25428393/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Guo,%20J.%20et%20al.%20PRRX1%20promotes%20epithelial%E2%80%93mesenchymal%20transition%20through%20the%20Wnt/%CE%B2-catenin%20pathway%20in%20gastric%20cancer.%20Med.%20Oncol.32,%201%E2%80%9312%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR87">
<span class="label">87.</span><cite>Sala, K. et al. The ERC1 scaffold protein implicated in cell motility drives the assembly of a liquid phase. <em>Sci. Rep.</em><strong>9</strong>, 1–14 (2019).
</cite> [<a href="https://doi.org/10.1038/s41598-019-49630-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6753080/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31537859/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sala,%20K.%20et%20al.%20The%20ERC1%20scaffold%20protein%20implicated%20in%20cell%20motility%20drives%20the%20assembly%20of%20a%20liquid%20phase.%20Sci.%20Rep.9,%201%E2%80%9314%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR88">
<span class="label">88.</span><cite>Abram, C. L. et al. The adaptor protein fish associates with members of the ADAMs family and localizes to podosomes of Src-transformed cells. <em>J. Biol. Chem.</em><strong>278</strong>, 16844–16851 (2003).
</cite> [<a href="https://doi.org/10.1074/jbc.M300267200" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12615925/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Abram,%20C.%20L.%20et%20al.%20The%20adaptor%20protein%20fish%20associates%20with%20members%20of%20the%20ADAMs%20family%20and%20localizes%20to%20podosomes%20of%20Src-transformed%20cells.%20J.%20Biol.%20Chem.278,%2016844%E2%80%9316851%20(2003)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR89">
<span class="label">89.</span><cite>Lin, C. Y. et al. Starvation-inactivated MTOR triggers cell migration via a ULK1-SH3PXD2A/TKS5-MMP14 pathway in ovarian carcinoma. <em>Autophagy</em><strong>19</strong>, 3151–3168 (2023).
</cite> [<a href="https://doi.org/10.1080/15548627.2023.2239633" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10621272/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37505094/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lin,%20C.%20Y.%20et%20al.%20Starvation-inactivated%20MTOR%20triggers%20cell%20migration%20via%20a%20ULK1-SH3PXD2A/TKS5-MMP14%20pathway%20in%20ovarian%20carcinoma.%20Autophagy19,%203151%E2%80%933168%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR90">
<span class="label">90.</span><cite>Zhang, B. et al. A human embryonic limb cell atlas resolved in space and time. <em>Nature</em><strong>635</strong>, 668-678 (2023).</cite> [<a href="https://doi.org/10.1038/s41586-023-06806-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7616500/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38057666/" class="usa-link">PubMed</a>]</li>
<li id="CR91">
<span class="label">91.</span><cite>Parisi, A. et al. APC is required for muscle stem cell proliferation and skeletal muscle tissue repair. <em>J. Cell Biol</em>. 10.1083/jcb.201501053 (2015).</cite> [<a href="https://doi.org/10.1083/jcb.201501053" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4555822/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26304725/" class="usa-link">PubMed</a>]</li>
<li id="CR92">
<span class="label">92.</span><cite>Rudolf, A. et al. β-catenin activation in muscle progenitor cells regulates tissue repair. <em>Cell Rep</em>. 10.1016/j.celrep.2016.04.022 (2016).</cite> [<a href="https://doi.org/10.1016/j.celrep.2016.04.022" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27134174/" class="usa-link">PubMed</a>]</li>
<li id="CR93">
<span class="label">93.</span><cite>Bi, S., Yue, S. &amp; Zhang, S. Hybridization chain reaction: a versatile molecular tool for biosensing, bioimaging, and biomedicine. <em>Chem. Soc. Rev.</em><strong>46</strong>, 4281–4298 (2017).
</cite> [<a href="https://doi.org/10.1039/c7cs00055c" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28573275/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bi,%20S.,%20Yue,%20S.%20&amp;%20Zhang,%20S.%20Hybridization%20chain%20reaction:%20a%20versatile%20molecular%20tool%20for%20biosensing,%20bioimaging,%20and%20biomedicine.%20Chem.%20Soc.%20Rev.46,%204281%E2%80%934298%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR94">
<span class="label">94.</span><cite>Christ, B., Jacob, H. J. &amp; Jacob, M. Experimental analysis of the origin of the wing musculature in avian embryos. <em>Anat. Embryol.</em><strong>150</strong>, 171–186 (1977).</cite> [<a href="https://doi.org/10.1007/BF00316649" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/857700/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Christ,%20B.,%20Jacob,%20H.%20J.%20&amp;%20Jacob,%20M.%20Experimental%20analysis%20of%20the%20origin%20of%20the%20wing%20musculature%20in%20avian%20embryos.%20Anat.%20Embryol.150,%20171%E2%80%93186%20(1977)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR95">
<span class="label">95.</span><cite>Chevallier, A., Kieny, M. &amp; Mauger, A. Limb-somite relationship: origin of the limb musculature. <em>Development</em><strong>41</strong>, 245–258 (1977).</cite> [<a href="https://pubmed.ncbi.nlm.nih.gov/591873/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chevallier,%20A.,%20Kieny,%20M.%20&amp;%20Mauger,%20A.%20Limb-somite%20relationship:%20origin%20of%20the%20limb%20musculature.%20Development41,%20245%E2%80%93258%20(1977)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR96">
<span class="label">96.</span><cite>Denetclaw, W. F. &amp; Ordahl, C. P. The growth of the dermomyotome and formation of early myotome lineages in thoracolumbar somites of chicken embryos. <em>Development</em><strong>127</strong>, 893–905 (2000).
</cite> [<a href="https://doi.org/10.1242/dev.127.4.893" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10648247/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Denetclaw,%20W.%20F.%20&amp;%20Ordahl,%20C.%20P.%20The%20growth%20of%20the%20dermomyotome%20and%20formation%20of%20early%20myotome%20lineages%20in%20thoracolumbar%20somites%20of%20chicken%20embryos.%20Development127,%20893%E2%80%93905%20(2000)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR97">
<span class="label">97.</span><cite>Cossu, G. &amp; Borello, U. Wnt signaling and the activation of myogenesis in mammals. <em>EMBO J.</em><strong>18</strong>, 6867–6872 (1999).
</cite> [<a href="https://doi.org/10.1093/emboj/18.24.6867" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1171749/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10601008/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cossu,%20G.%20&amp;%20Borello,%20U.%20Wnt%20signaling%20and%20the%20activation%20of%20myogenesis%20in%20mammals.%20EMBO%20J.18,%206867%E2%80%936872%20(1999)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR98">
<span class="label">98.</span><cite>Ikeya, M. &amp; Takada, S. Wnt signaling from the dorsal neural tube is required for the formation of the medial dermomyotome. <em>Development</em><strong>125</strong>, 4969–4976 (1998).
</cite> [<a href="https://doi.org/10.1242/dev.125.24.4969" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9811581/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ikeya,%20M.%20&amp;%20Takada,%20S.%20Wnt%20signaling%20from%20the%20dorsal%20neural%20tube%20is%20required%20for%20the%20formation%20of%20the%20medial%20dermomyotome.%20Development125,%204969%E2%80%934976%20(1998)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR99">
<span class="label">99.</span><cite>Münsterberg, A. E., Kitajewski, J., Bumcrot, D. A., McMahon, A. P. &amp; Lassar, A. B. Combinatorial signaling by Sonic hedgehog and Wnt family members induces myogenic bHLH gene expression in the somite. <em>Genes Dev.</em><strong>9</strong>, 2911–2922 (1995).
</cite> [<a href="https://doi.org/10.1101/gad.9.23.2911" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/7498788/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?M%C3%BCnsterberg,%20A.%20E.,%20Kitajewski,%20J.,%20Bumcrot,%20D.%20A.,%20McMahon,%20A.%20P.%20&amp;%20Lassar,%20A.%20B.%20Combinatorial%20signaling%20by%20Sonic%20hedgehog%20and%20Wnt%20family%20members%20induces%20myogenic%20bHLH%20gene%20expression%20in%20the%20somite.%20Genes%20Dev.9,%202911%E2%80%932922%20(1995)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR100">
<span class="label">100.</span><cite>Marcelle, C., Stark, M. R. &amp; Bronner-Fraser, M. Coordinate actions of BMPs, Wnts, Shh and noggin mediate patterning of the dorsal somite. <em>Development</em><strong>124</strong>, 3955-3963 (1997).</cite> [<a href="https://doi.org/10.1242/dev.124.20.3955" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9374393/" class="usa-link">PubMed</a>]</li>
<li id="CR101">
<span class="label">101.</span><cite>Polesskaya, A., Seale, P. &amp; Rudnicki, M. A. Wnt signaling induces the myogenic specification of resident CD45+ adult stem cells during muscle regeneration. <em>Cell</em><strong>113</strong>, 841–852 (2003).
</cite> [<a href="https://doi.org/10.1016/s0092-8674(03)00437-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12837243/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Polesskaya,%20A.,%20Seale,%20P.%20&amp;%20Rudnicki,%20M.%20A.%20Wnt%20signaling%20induces%20the%20myogenic%20specification%20of%20resident%20CD45+%20adult%20stem%20cells%20during%20muscle%20regeneration.%20Cell113,%20841%E2%80%93852%20(2003)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR102">
<span class="label">102.</span><cite>Murphy, M. M. et al. Transiently active Wnt/β-catenin signaling is not required but must be silenced for stem cell function during muscle regeneration. <em>Stem Cell Rep.</em><strong>3</strong>, 475–488 (2014).</cite> [<a href="https://doi.org/10.1016/j.stemcr.2014.06.019" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4266007/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25241745/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Murphy,%20M.%20M.%20et%20al.%20Transiently%20active%20Wnt/%CE%B2-catenin%20signaling%20is%20not%20required%20but%20must%20be%20silenced%20for%20stem%20cell%20function%20during%20muscle%20regeneration.%20Stem%20Cell%20Rep.3,%20475%E2%80%93488%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR103">
<span class="label">103.</span><cite>Yang, Y. Wnts and wing: Wnt signaling in vertebrate limb development and musculoskeletal morphogenesis. <em>Birth Defects Res. Part C Embryo Today Rev.</em><strong>69</strong>, 305–317 (2003).</cite> [<a href="https://doi.org/10.1002/bdrc.10026" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/14745971/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yang,%20Y.%20Wnts%20and%20wing:%20Wnt%20signaling%20in%20vertebrate%20limb%20development%20and%20musculoskeletal%20morphogenesis.%20Birth%20Defects%20Res.%20Part%20C%20Embryo%20Today%20Rev.69,%20305%E2%80%93317%20(2003)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR104">
<span class="label">104.</span><cite>Geetha-Loganathan, P., Nimmagadda, S. &amp; Scaal, M. Wnt signaling in limb organogenesis. <em>Organogenesis</em><strong>4</strong>, 109–115 (2008).
</cite> [<a href="https://doi.org/10.4161/org.4.2.5857" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2634256/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19279722/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Geetha-Loganathan,%20P.,%20Nimmagadda,%20S.%20&amp;%20Scaal,%20M.%20Wnt%20signaling%20in%20limb%20organogenesis.%20Organogenesis4,%20109%E2%80%93115%20(2008)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR105">
<span class="label">105.</span><cite>Chevallier, A. Etude de la migration des cellules somitiques dans le mésoderme somatopleural de l’ébauche de l’aile. <em>Wilhelm. Roux’s. Arch. Dev. Biol.</em><strong>184</strong>, 57–73 (1978).</cite> [<a href="https://doi.org/10.1007/BF00848669" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28305013/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chevallier,%20A.%20Etude%20de%20la%20migration%20des%20cellules%20somitiques%20dans%20le%20m%C3%A9soderme%20somatopleural%20de%20l%E2%80%99%C3%A9bauche%20de%20l%E2%80%99aile.%20Wilhelm.%20Roux%E2%80%99s.%20Arch.%20Dev.%20Biol.184,%2057%E2%80%9373%20(1978)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR106">
<span class="label">106.</span><cite>Eichmann, A., Marcelle, C., Bréant, C. &amp; Le Douarin, N. M. Two molecules related to the VEGF receptor are expressed in early endothelial cells during avian embryonic development. <em>Mech. Dev</em>. <strong>42</strong>, 33-48 (1993).</cite> [<a href="https://doi.org/10.1016/0925-4773(93)90096-g" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/8396413/" class="usa-link">PubMed</a>]</li>
<li id="CR107">
<span class="label">107.</span><cite>Wilting, J., Eichmann, A. &amp; Christ, B. Expression of the avian VEGF receptor homologues Quek1 and Quek2 in blood-vascular and lymphatic endothelial and non-endothelial cells during quail embryonic development. <em>Cell Tissue Res.</em><strong>288</strong>, 207–223 (1997).
</cite> [<a href="https://doi.org/10.1007/s004410050807" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9082957/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wilting,%20J.,%20Eichmann,%20A.%20&amp;%20Christ,%20B.%20Expression%20of%20the%20avian%20VEGF%20receptor%20homologues%20Quek1%20and%20Quek2%20in%20blood-vascular%20and%20lymphatic%20endothelial%20and%20non-endothelial%20cells%20during%20quail%20embryonic%20development.%20Cell%20Tissue%20Res.288,%20207%E2%80%93223%20(1997)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR108">
<span class="label">108.</span><cite>Korinek, V. et al. Constitutive transcriptional activation by a β-catenin-Tcf complex in APC(-/-) colon carcinoma. <em>Science</em>. 10.1126/science.275.5307.1784 (1997).</cite> [<a href="https://doi.org/10.1126/science.275.5307.1784" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9065401/" class="usa-link">PubMed</a>]</li>
<li id="CR109">
<span class="label">109.</span><cite>Pfeiffer, B. D., Truman, J. W. &amp; Rubin, G. M. Using translational enhancers to increase transgene expression in Drosophila. <em>Proc. Natl. Acad. Sci. USA</em><strong>109</strong>, 6626–6631 (2012).
</cite> [<a href="https://doi.org/10.1073/pnas.1204520109" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3340069/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22493255/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Pfeiffer,%20B.%20D.,%20Truman,%20J.%20W.%20&amp;%20Rubin,%20G.%20M.%20Using%20translational%20enhancers%20to%20increase%20transgene%20expression%20in%20Drosophila.%20Proc.%20Natl.%20Acad.%20Sci.%20USA109,%206626%E2%80%936631%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR110">
<span class="label">110.</span><cite>Abranches, E., Bekman, E. &amp; Henrique, D. Generation and characterization of a novel mouse embryonic stem cell line with a dynamic reporter of Nanog expression. <em>PLoS One</em><strong>8</strong>, e59928 (2013).</cite> [<a href="https://doi.org/10.1371/journal.pone.0059928" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3602340/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23527287/" class="usa-link">PubMed</a>]</li>
<li id="CR111">
<span class="label">111.</span><cite>Nagai, T. et al. A variant of yellow fluorescent protein with fast and efficient maturation for cell-biological applications. <em>Nat. Biotechnol</em>. 10.1038/nbt0102-87 (2002).</cite> [<a href="https://doi.org/10.1038/nbt0102-87" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11753368/" class="usa-link">PubMed</a>]</li>
<li id="CR112">
<span class="label">112.</span><cite>Rios, A. C. A. C., Denans, N. &amp; Marcelle, C. Real-time observation of Wnt β-catenin signaling in the chick embryo. <em>Dev. Dyn.</em><strong>239</strong>, 346–353 (2010).
</cite> [<a href="https://doi.org/10.1002/dvdy.22174" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20014451/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rios,%20A.%20C.%20A.%20C.,%20Denans,%20N.%20&amp;%20Marcelle,%20C.%20Real-time%20observation%20of%20Wnt%20%CE%B2-catenin%20signaling%20in%20the%20chick%20embryo.%20Dev.%20Dyn.239,%20346%E2%80%93353%20(2010)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR113">
<span class="label">113.</span><cite>Kengaku, M. et al. Distinct WNT pathways regulating AER formation and dorsoventral polarity in the chick limb bud. <em>Sci. (80-).</em><strong>280</strong>, 1274–1277 (1998).</cite> [<a href="https://doi.org/10.1126/science.280.5367.1274" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9596583/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kengaku,%20M.%20et%20al.%20Distinct%20WNT%20pathways%20regulating%20AER%20formation%20and%20dorsoventral%20polarity%20in%20the%20chick%20limb%20bud.%20Sci.%20(80-).280,%201274%E2%80%931277%20(1998)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR114">
<span class="label">114.</span><cite>McMahon, A. P. &amp; Moon, R. T. Ectopic expression of the proto-oncogene int-1 in Xenopus embryos leads to duplication of the embryonic axis. <em>Cell</em><strong>58</strong>, 1075–1084 (1989).
</cite> [<a href="https://doi.org/10.1016/0092-8674(89)90506-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/2673541/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?McMahon,%20A.%20P.%20&amp;%20Moon,%20R.%20T.%20Ectopic%20expression%20of%20the%20proto-oncogene%20int-1%20in%20Xenopus%20embryos%20leads%20to%20duplication%20of%20the%20embryonic%20axis.%20Cell58,%201075%E2%80%931084%20(1989)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR115">
<span class="label">115.</span><cite>André, M., Dinvaut, S., Castellani, V. &amp; Falk, J. 3D exploration of gene expression in chicken embryos through combined RNA fluorescence in situ hybridization, immunofluorescence, and clearing. <em>BMC Biol.</em><strong>22</strong>, 1–16 (2024).
</cite> [<a href="https://doi.org/10.1186/s12915-024-01922-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11149291/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38831263/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Andr%C3%A9,%20M.,%20Dinvaut,%20S.,%20Castellani,%20V.%20&amp;%20Falk,%20J.%203D%20exploration%20of%20gene%20expression%20in%20chicken%20embryos%20through%20combined%20RNA%20fluorescence%20in%20situ%20hybridization,%20immunofluorescence,%20and%20clearing.%20BMC%20Biol.22,%201%E2%80%9316%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR116">
<span class="label">116.</span><cite>Dobin, A. et al. STAR: ultrafast universal RNA-seq aligner. <em>Bioinformatics</em><strong>29</strong>, 15–21 (2013).
</cite> [<a href="https://doi.org/10.1093/bioinformatics/bts635" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3530905/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23104886/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Dobin,%20A.%20et%20al.%20STAR:%20ultrafast%20universal%20RNA-seq%20aligner.%20Bioinformatics29,%2015%E2%80%9321%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR117">
<span class="label">117.</span><cite>Hao, Y. et al. Dictionary learning for integrative, multimodal and scalable single-cell analysis. <em>Nat. Biotechnol.</em><strong>42</strong>, 293–304 (2023).
</cite> [<a href="https://doi.org/10.1038/s41587-023-01767-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10928517/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37231261/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hao,%20Y.%20et%20al.%20Dictionary%20learning%20for%20integrative,%20multimodal%20and%20scalable%20single-cell%20analysis.%20Nat.%20Biotechnol.42,%20293%E2%80%93304%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR118">
<span class="label">118.</span><cite>Ahlmann-Eltze, C. &amp; Huber, W. glmGamPoi: fitting Gamma-Poisson generalized linear models on single cell count data. <em>Bioinformatics</em><strong>36</strong>, 5701–5702 (2021).
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btaa1009" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8023675/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33295604/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ahlmann-Eltze,%20C.%20&amp;%20Huber,%20W.%20glmGamPoi:%20fitting%20Gamma-Poisson%20generalized%20linear%20models%20on%20single%20cell%20count%20data.%20Bioinformatics36,%205701%E2%80%935702%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR119">
<span class="label">119.</span><cite>Germain, P. L., Robinson, M. D., Lun, A., Garcia Meixide, C. &amp; Macnair, W. Doublet identification in single-cell sequencing data using <em>scDblFinder</em>. <em>F1000Research</em><strong>10</strong>, 979 (2022).</cite> [<a href="https://doi.org/10.12688/f1000research.73600.1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9204188/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35814628/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Germain,%20P.%20L.,%20Robinson,%20M.%20D.,%20Lun,%20A.,%20Garcia%20Meixide,%20C.%20&amp;%20Macnair,%20W.%20Doublet%20identification%20in%20single-cell%20sequencing%20data%20using%20scDblFinder.%20F1000Research10,%20979%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR120">
<span class="label">120.</span><cite>Van Der Maaten, L., Courville, A., Fergus, R. &amp; Manning, C. Accelerating t-SNE using Tree-Based Algorithms. <em>J. Mach. Learn. Res.</em><strong>15</strong>, 3221–3245 (2014).</cite> [<a href="https://scholar.google.com/scholar_lookup?Van%20Der%20Maaten,%20L.,%20Courville,%20A.,%20Fergus,%20R.%20&amp;%20Manning,%20C.%20Accelerating%20t-SNE%20using%20Tree-Based%20Algorithms.%20J.%20Mach.%20Learn.%20Res.15,%203221%E2%80%933245%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR121">
<span class="label">121.</span><cite>Linderman, G. C., Rachh, M., Hoskins, J. G., Steinerberger, S. &amp; Kluger, Y. Fast interpolation-based t-SNE for improved visualization of single-cell RNA-seq data. <em>Nat. Methods</em><strong>16</strong>, 243–245 (2019).
</cite> [<a href="https://doi.org/10.1038/s41592-018-0308-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6402590/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30742040/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Linderman,%20G.%20C.,%20Rachh,%20M.,%20Hoskins,%20J.%20G.,%20Steinerberger,%20S.%20&amp;%20Kluger,%20Y.%20Fast%20interpolation-based%20t-SNE%20for%20improved%20visualization%20of%20single-cell%20RNA-seq%20data.%20Nat.%20Methods16,%20243%E2%80%93245%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR122">
<span class="label">122.</span><cite>Finak, G. et al. MAST: A flexible statistical framework for assessing transcriptional changes and characterizing heterogeneity in single-cell RNA sequencing data. <em>Genome Biol.</em><strong>16</strong>, 1–13 (2015).
</cite> [<a href="https://doi.org/10.1186/s13059-015-0844-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4676162/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26653891/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Finak,%20G.%20et%20al.%20MAST:%20A%20flexible%20statistical%20framework%20for%20assessing%20transcriptional%20changes%20and%20characterizing%20heterogeneity%20in%20single-cell%20RNA%20sequencing%20data.%20Genome%20Biol.16,%201%E2%80%9313%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR123">
<span class="label">123.</span><cite>Wolf, F. A. et al. PAGA: graph abstraction reconciles clustering with trajectory inference through a topology preserving map of single cells. <em>Genome Biol.</em><strong>20</strong>, 1–9 (2019).
</cite> [<a href="https://doi.org/10.1186/s13059-019-1663-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6425583/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30890159/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wolf,%20F.%20A.%20et%20al.%20PAGA:%20graph%20abstraction%20reconciles%20clustering%20with%20trajectory%20inference%20through%20a%20topology%20preserving%20map%20of%20single%20cells.%20Genome%20Biol.20,%201%E2%80%939%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12374003/bin/41467_2025_61767_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information</a><sup> (665MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12374003/bin/41467_2025_61767_MOESM2_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link">41467_2025_61767_MOESM2_ESM.docx</a><sup> (14KB, docx) </sup><p>Description of Additional Supplementary Information</p>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12374003/bin/41467_2025_61767_MOESM3_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 1</a><sup> (928.9KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material4_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12374003/bin/41467_2025_61767_MOESM4_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 2</a><sup> (530.7KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material5_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12374003/bin/41467_2025_61767_MOESM5_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Reporting Summary</a><sup> (102.1KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material6_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12374003/bin/41467_2025_61767_MOESM6_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Transparent Peer Review file</a><sup> (3.8MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material7_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12374003/bin/41467_2025_61767_MOESM7_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Source Data</a><sup> (35.3KB, xlsx) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>scRNA-sequencing data sets for chicken embryos are available at the Gene Expression Omnibys (GEO) under the accession number <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE283478" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE283478</a>. <a href="#Sec30" class="usa-link">Source data</a> are provided with this paper.</p>
<p>Code is available upon request.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Nature Communications are provided here courtesy of <strong>Nature Publishing Group</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1038/s41467-025-61767-1"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/41467_2025_Article_61767.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (10.5 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12374003/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12374003/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12374003%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12374003/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12374003/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12374003/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40846698/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12374003/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40846698/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12374003/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12374003/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="f16Kf3EPDeIwlOEx5Fni9fMYuXCVTmOomsvh3dFbr0d82IOjXxq9V5SsmFLh3csj">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
        <script type="module" crossorigin="" src="/static/assets/math-574fdcc6.js"></script>
    
    

    </body>
</html>
