
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            NADPH oxidase 1/4 dual inhibition impairs transforming growth factor-beta protumorigenic effects in cholangiocarcinoma cancer-associated fibroblasts - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="1525531B8AF43EB305531B002260E2D8.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="sigtrans">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12358586/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Signal Transduction and Targeted Therapy">
<meta name="citation_title" content="NADPH oxidase 1/4 dual inhibition impairs transforming growth factor-beta protumorigenic effects in cholangiocarcinoma cancer-associated fibroblasts">
<meta name="citation_author" content="Josep Amengual">
<meta name="citation_author_institution" content="TGF-β and Cancer Group, Oncobell Program, Bellvitge Biomedical Research Institute (IDIBELL), Barcelona, Spain">
<meta name="citation_author_institution" content="CIBEREHD, National Biomedical Research Institute on Liver and Gastrointestinal Diseases, Instituto de Salud Carlos III, Madrid, Spain">
<meta name="citation_author" content="Ester Gonzalez-Sanchez">
<meta name="citation_author_institution" content="TGF-β and Cancer Group, Oncobell Program, Bellvitge Biomedical Research Institute (IDIBELL), Barcelona, Spain">
<meta name="citation_author_institution" content="CIBEREHD, National Biomedical Research Institute on Liver and Gastrointestinal Diseases, Instituto de Salud Carlos III, Madrid, Spain">
<meta name="citation_author_institution" content="Department of Physiology and Pharmacology, University of Salamanca, 37007 Salamanca, Spain">
<meta name="citation_author_institution" content="HepatoBiliary Tumours Lab, Centro de Investigación del Cáncer and Instituto de Biología Molecular y Celular del Cáncer, CSIC-Universidad de Salamanca, Salamanca, 37007 Spain">
<meta name="citation_author" content="Mariana Yáñez-Bartolome">
<meta name="citation_author_institution" content="Upper Gastrointestinal and Endocrine Tumour Unit, Vall d’Hebron Institute of Oncology (VHIO), Barcelona, Spain">
<meta name="citation_author" content="Laura Sererols-Viñas">
<meta name="citation_author_institution" content="Tumor Microenvironment Plasticity and Heterogeneity Group, Instituto de Investigaciones Biomédicas August Pi i Sunyer (IDIBAPS), Barcelona, Spain">
<meta name="citation_author" content="Aashreya Ravichandra">
<meta name="citation_author_institution" content="TUM School of Medicine and Health, Department of Clinical Medicine – Clinical Department for Internal Medicine II, University Medical Center, Technical University of Munich, Munich, Germany">
<meta name="citation_author" content="Celia Guiton">
<meta name="citation_author_institution" content="TGF-β and Cancer Group, Oncobell Program, Bellvitge Biomedical Research Institute (IDIBELL), Barcelona, Spain">
<meta name="citation_author" content="Noel P Fuste">
<meta name="citation_author_institution" content="TGF-β and Cancer Group, Oncobell Program, Bellvitge Biomedical Research Institute (IDIBELL), Barcelona, Spain">
<meta name="citation_author" content="Ania Alay">
<meta name="citation_author_institution" content="Unit of Bioinformatics for Precision Oncology, Catalan Institute of Oncology (ICO), L’Hospitalet de Llobregat, Barcelona, Spain">
<meta name="citation_author_institution" content="Preclinical and Experimental Research in Thoracic Tumors (PReTT), Oncobell Program, IDIBELL, L’Hospitalet de Llobregat, Barcelona, Spain">
<meta name="citation_author" content="Sara Hijazo-Pechero">
<meta name="citation_author_institution" content="Preclinical and Experimental Research in Thoracic Tumors (PReTT), Oncobell Program, IDIBELL, L’Hospitalet de Llobregat, Barcelona, Spain">
<meta name="citation_author" content="Beatriz Martín-Mur">
<meta name="citation_author_institution" content="Centro Nacional de Análisis Genómico (CNAG), C/Baldiri Reixac 4, 08028 Barcelona, Spain">
<meta name="citation_author_institution" content="University of Barcelona (UB), 08034 Barcelona, Spain">
<meta name="citation_author" content="Marta Gut">
<meta name="citation_author_institution" content="Centro Nacional de Análisis Genómico (CNAG), C/Baldiri Reixac 4, 08028 Barcelona, Spain">
<meta name="citation_author_institution" content="University of Barcelona (UB), 08034 Barcelona, Spain">
<meta name="citation_author" content="Anna Esteve-Codina">
<meta name="citation_author_institution" content="Centro Nacional de Análisis Genómico (CNAG), C/Baldiri Reixac 4, 08028 Barcelona, Spain">
<meta name="citation_author_institution" content="University of Barcelona (UB), 08034 Barcelona, Spain">
<meta name="citation_author" content="Ana Cantos-Cortes">
<meta name="citation_author_institution" content="TGF-β and Cancer Group, Oncobell Program, Bellvitge Biomedical Research Institute (IDIBELL), Barcelona, Spain">
<meta name="citation_author_institution" content="CIBEREHD, National Biomedical Research Institute on Liver and Gastrointestinal Diseases, Instituto de Salud Carlos III, Madrid, Spain">
<meta name="citation_author" content="Rut Espinosa-Sotelo">
<meta name="citation_author_institution" content="TGF-β and Cancer Group, Oncobell Program, Bellvitge Biomedical Research Institute (IDIBELL), Barcelona, Spain">
<meta name="citation_author_institution" content="CIBEREHD, National Biomedical Research Institute on Liver and Gastrointestinal Diseases, Instituto de Salud Carlos III, Madrid, Spain">
<meta name="citation_author" content="Emilio Ramos">
<meta name="citation_author_institution" content="CIBEREHD, National Biomedical Research Institute on Liver and Gastrointestinal Diseases, Instituto de Salud Carlos III, Madrid, Spain">
<meta name="citation_author_institution" content="Faculty of Medicine and Health Sciences, University of Barcelona, L’Hospitalet de Llobregat, Barcelona, Spain">
<meta name="citation_author_institution" content="Department of Surgery, Liver Transplant Unit, University Hospital of Bellvitge, Barcelona, Spain">
<meta name="citation_author" content="Teresa Serrano">
<meta name="citation_author_institution" content="CIBEREHD, National Biomedical Research Institute on Liver and Gastrointestinal Diseases, Instituto de Salud Carlos III, Madrid, Spain">
<meta name="citation_author_institution" content="Faculty of Medicine and Health Sciences, University of Barcelona, L’Hospitalet de Llobregat, Barcelona, Spain">
<meta name="citation_author_institution" content="Pathological Anatomy Service, University Hospital of Bellvitge, Barcelona, Spain">
<meta name="citation_author" content="Mariona Calvo">
<meta name="citation_author_institution" content="TGF-β and Cancer Group, Oncobell Program, Bellvitge Biomedical Research Institute (IDIBELL), Barcelona, Spain">
<meta name="citation_author_institution" content="Oncología Médica, Institut Català d’Oncologia (ICO-IDIBELL), L’Hospitalet del Llobregat, Barcelona, Spain">
<meta name="citation_author" content="Berta Laquente">
<meta name="citation_author_institution" content="Oncología Médica, Institut Català d’Oncologia (ICO-IDIBELL), L’Hospitalet del Llobregat, Barcelona, Spain">
<meta name="citation_author" content="Joana Ferrer">
<meta name="citation_author_institution" content="CIBEREHD, National Biomedical Research Institute on Liver and Gastrointestinal Diseases, Instituto de Salud Carlos III, Madrid, Spain">
<meta name="citation_author_institution" content="Barcelona Clinic Liver Cancer (BCLC) group. Institut d’Investigacions Biomèdiques August Pi I Sunyer (IDIBAPS). Surgery Department, Liver Oncology Unit, Hospital Clinic de Barcelona, Barcelona, Spain Barcelona University, Barcelona, Spain">
<meta name="citation_author" content="Gabriel Pons">
<meta name="citation_author_institution" content="Physiological Sciences Department, University of Barcelona, Barcelona, Spain">
<meta name="citation_author" content="Andrés Mendez-Lucas">
<meta name="citation_author_institution" content="Physiological Sciences Department, University of Barcelona, Barcelona, Spain">
<meta name="citation_author_institution" content="Nutrition, metabolism and gene therapy group. Diabetes and Metabolism Program. IDIBELL, L’Hospitalet de Llobregat, Barcelona, Spain">
<meta name="citation_author" content="Steven Dooley">
<meta name="citation_author_institution" content="Department of Medicine II, Section Molecular Hepatology, Medical Faculty Mannheim, Heidelberg University, Mannheim, Germany">
<meta name="citation_author" content="Sumera I Ilyas">
<meta name="citation_author_institution" content="Division of Gastroenterology and Hepatology, Mayo Clinic, Rochester, Minnesota USA">
<meta name="citation_author" content="Marie Vallette">
<meta name="citation_author_institution" content="UMRS_938, Sorbonne Université, Inserm, Centre de Recherche Saint-Antoine (CRSA), Paris, France">
<meta name="citation_author" content="Lynda Aoudjehane">
<meta name="citation_author_institution" content="UMRS_938, Sorbonne Université, Inserm, Centre de Recherche Saint-Antoine (CRSA), Paris, France">
<meta name="citation_author_institution" content="ICAN BioCell-Human Liver Biology core, IHU- Foundation for innovation in Cardiometabolism and Nutrition (IHU-ICAN), F-75013 Paris, France">
<meta name="citation_author_institution" content="Sorbonne Université, INSERM, UMRS 1166/IHU-ICAN, F-75013 Paris, France">
<meta name="citation_author" content="Marie Lequoy">
<meta name="citation_author_institution" content="UMRS_938, Sorbonne Université, Inserm, Centre de Recherche Saint-Antoine (CRSA), Paris, France">
<meta name="citation_author_institution" content="AP-HP Sorbonne Université, Hôpital Universitaire Saint Antoine, Service d´Hépatologie, F-75012 Paris, France">
<meta name="citation_author" content="Laura Fouassier">
<meta name="citation_author_institution" content="UMRS_938, Sorbonne Université, Inserm, Centre de Recherche Saint-Antoine (CRSA), Paris, France">
<meta name="citation_author" content="Cédric Coulouarn">
<meta name="citation_author_institution" content="Inserm, Univ Rennes, OSS (Oncogenesis, Stress, Signaling) UMR_S 1242, Centre de Lutte contre le Cancer Eugène Marquis, F-35042 Rennes, France">
<meta name="citation_author" content="Silvia Affò">
<meta name="citation_author_institution" content="Tumor Microenvironment Plasticity and Heterogeneity Group, Instituto de Investigaciones Biomédicas August Pi i Sunyer (IDIBAPS), Barcelona, Spain">
<meta name="citation_author" content="Alexander Scheiter">
<meta name="citation_author_institution" content="Institute of Pathology, University of Regensburg, Regensburg, Germany">
<meta name="citation_author" content="Diego F Calvisi">
<meta name="citation_author_institution" content="Institute of Pathology, University of Regensburg, Regensburg, Germany">
<meta name="citation_author" content="Tian V Tian">
<meta name="citation_author_institution" content="Upper Gastrointestinal and Endocrine Tumour Unit, Vall d’Hebron Institute of Oncology (VHIO), Barcelona, Spain">
<meta name="citation_author" content="Isabel Fabregat">
<meta name="citation_author_institution" content="TGF-β and Cancer Group, Oncobell Program, Bellvitge Biomedical Research Institute (IDIBELL), Barcelona, Spain">
<meta name="citation_author_institution" content="CIBEREHD, National Biomedical Research Institute on Liver and Gastrointestinal Diseases, Instituto de Salud Carlos III, Madrid, Spain">
<meta name="citation_author" content="Javier Vaquero">
<meta name="citation_author_institution" content="TGF-β and Cancer Group, Oncobell Program, Bellvitge Biomedical Research Institute (IDIBELL), Barcelona, Spain">
<meta name="citation_author_institution" content="CIBEREHD, National Biomedical Research Institute on Liver and Gastrointestinal Diseases, Instituto de Salud Carlos III, Madrid, Spain">
<meta name="citation_author_institution" content="HepatoBiliary Tumours Lab, Centro de Investigación del Cáncer and Instituto de Biología Molecular y Celular del Cáncer, CSIC-Universidad de Salamanca, Salamanca, 37007 Spain">
<meta name="citation_publication_date" content="2025 Aug 18">
<meta name="citation_volume" content="10">
<meta name="citation_firstpage" content="257">
<meta name="citation_doi" content="10.1038/s41392-025-02347-z">
<meta name="citation_pmid" content="40820091">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12358586/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12358586/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12358586/pdf/41392_2025_Article_2347.pdf">
<meta name="description" content="Transforming growth factor beta (TGF-β) signalling has become an attractive therapeutic target due to its pro-tumorigenic actions on epithelial cells and its immunosuppressive effects in the tumour microenvironment. In intrahepatic ...">
<meta name="og:title" content="NADPH oxidase 1/4 dual inhibition impairs transforming growth factor-beta protumorigenic effects in cholangiocarcinoma cancer-associated fibroblasts">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Transforming growth factor beta (TGF-β) signalling has become an attractive therapeutic target due to its pro-tumorigenic actions on epithelial cells and its immunosuppressive effects in the tumour microenvironment. In intrahepatic ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12358586/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12358586">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1038/s41392-025-02347-z"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/41392_2025_Article_2347.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12358586%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12358586/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12358586/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12358586/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-sigtrans.jpg" alt="Signal Transduction and Targeted Therapy logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Signal Transduction and Targeted Therapy" title="Link to Signal Transduction and Targeted Therapy" shape="default" href="https://www.nature.com/sigtrans/" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Signal Transduct Target Ther</button></div>. 2025 Aug 18;10:257. doi: <a href="https://doi.org/10.1038/s41392-025-02347-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1038/s41392-025-02347-z</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Signal%20Transduct%20Target%20Ther%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Signal%20Transduct%20Target%20Ther%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Signal%20Transduct%20Target%20Ther%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Signal%20Transduct%20Target%20Ther%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>NADPH oxidase 1/4 dual inhibition impairs transforming growth factor-beta protumorigenic effects in cholangiocarcinoma cancer-associated fibroblasts</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Amengual%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Josep Amengual</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Josep Amengual</span></h3>
<div class="p">
<sup>1</sup>TGF-β and Cancer Group, Oncobell Program, Bellvitge Biomedical Research Institute (IDIBELL), Barcelona, Spain </div>
<div class="p">
<sup>2</sup>CIBEREHD, National Biomedical Research Institute on Liver and Gastrointestinal Diseases, Instituto de Salud Carlos III, Madrid, Spain </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Amengual%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Josep Amengual</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gonzalez-Sanchez%20E%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Ester Gonzalez-Sanchez</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Ester Gonzalez-Sanchez</span></h3>
<div class="p">
<sup>1</sup>TGF-β and Cancer Group, Oncobell Program, Bellvitge Biomedical Research Institute (IDIBELL), Barcelona, Spain </div>
<div class="p">
<sup>2</sup>CIBEREHD, National Biomedical Research Institute on Liver and Gastrointestinal Diseases, Instituto de Salud Carlos III, Madrid, Spain </div>
<div class="p">
<sup>3</sup>Department of Physiology and Pharmacology, University of Salamanca, 37007 Salamanca, Spain </div>
<div class="p">
<sup>4</sup>HepatoBiliary Tumours Lab, Centro de Investigación del Cáncer and Instituto de Biología Molecular y Celular del Cáncer, CSIC-Universidad de Salamanca, Salamanca, 37007 Spain </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gonzalez-Sanchez%20E%22%5BAuthor%5D" class="usa-link"><span class="name western">Ester Gonzalez-Sanchez</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>3,</sup><sup>4,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Y%C3%A1%C3%B1ez-Bartolome%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Mariana Yáñez-Bartolome</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Mariana Yáñez-Bartolome</span></h3>
<div class="p">
<sup>5</sup>Upper Gastrointestinal and Endocrine Tumour Unit, Vall d’Hebron Institute of Oncology (VHIO), Barcelona, Spain </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Y%C3%A1%C3%B1ez-Bartolome%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Mariana Yáñez-Bartolome</span></a>
</div>
</div>
<sup>5</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sererols-Vi%C3%B1as%20L%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Laura Sererols-Viñas</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Laura Sererols-Viñas</span></h3>
<div class="p">
<sup>6</sup>Tumor Microenvironment Plasticity and Heterogeneity Group, Instituto de Investigaciones Biomédicas August Pi i Sunyer (IDIBAPS), Barcelona, Spain </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sererols-Vi%C3%B1as%20L%22%5BAuthor%5D" class="usa-link"><span class="name western">Laura Sererols-Viñas</span></a>
</div>
</div>
<sup>6</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ravichandra%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Aashreya Ravichandra</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Aashreya Ravichandra</span></h3>
<div class="p">
<sup>7</sup>TUM School of Medicine and Health, Department of Clinical Medicine – Clinical Department for Internal Medicine II, University Medical Center, Technical University of Munich, Munich, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ravichandra%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Aashreya Ravichandra</span></a>
</div>
</div>
<sup>7</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Guiton%20C%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Celia Guiton</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Celia Guiton</span></h3>
<div class="p">
<sup>1</sup>TGF-β and Cancer Group, Oncobell Program, Bellvitge Biomedical Research Institute (IDIBELL), Barcelona, Spain </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Guiton%20C%22%5BAuthor%5D" class="usa-link"><span class="name western">Celia Guiton</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Fuste%20NP%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Noel P Fuste</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Noel P Fuste</span></h3>
<div class="p">
<sup>1</sup>TGF-β and Cancer Group, Oncobell Program, Bellvitge Biomedical Research Institute (IDIBELL), Barcelona, Spain </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Fuste%20NP%22%5BAuthor%5D" class="usa-link"><span class="name western">Noel P Fuste</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Alay%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Ania Alay</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Ania Alay</span></h3>
<div class="p">
<sup>8</sup>Unit of Bioinformatics for Precision Oncology, Catalan Institute of Oncology (ICO), L’Hospitalet de Llobregat, Barcelona, Spain </div>
<div class="p">
<sup>9</sup>Preclinical and Experimental Research in Thoracic Tumors (PReTT), Oncobell Program, IDIBELL, L’Hospitalet de Llobregat, Barcelona, Spain </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Alay%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Ania Alay</span></a>
</div>
</div>
<sup>8,</sup><sup>9</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hijazo-Pechero%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id9"><span class="name western">Sara Hijazo-Pechero</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">Sara Hijazo-Pechero</span></h3>
<div class="p">
<sup>9</sup>Preclinical and Experimental Research in Thoracic Tumors (PReTT), Oncobell Program, IDIBELL, L’Hospitalet de Llobregat, Barcelona, Spain </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hijazo-Pechero%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Sara Hijazo-Pechero</span></a>
</div>
</div>
<sup>9</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mart%C3%ADn-Mur%20B%22%5BAuthor%5D" class="usa-link" aria-describedby="id10"><span class="name western">Beatriz Martín-Mur</span></a><div hidden="hidden" id="id10">
<h3><span class="name western">Beatriz Martín-Mur</span></h3>
<div class="p">
<sup>10</sup>Centro Nacional de Análisis Genómico (CNAG), C/Baldiri Reixac 4, 08028 Barcelona, Spain </div>
<div class="p">
<sup>11</sup>University of Barcelona (UB), 08034 Barcelona, Spain </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mart%C3%ADn-Mur%20B%22%5BAuthor%5D" class="usa-link"><span class="name western">Beatriz Martín-Mur</span></a>
</div>
</div>
<sup>10,</sup><sup>11</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gut%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id11"><span class="name western">Marta Gut</span></a><div hidden="hidden" id="id11">
<h3><span class="name western">Marta Gut</span></h3>
<div class="p">
<sup>10</sup>Centro Nacional de Análisis Genómico (CNAG), C/Baldiri Reixac 4, 08028 Barcelona, Spain </div>
<div class="p">
<sup>11</sup>University of Barcelona (UB), 08034 Barcelona, Spain </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gut%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Marta Gut</span></a>
</div>
</div>
<sup>10,</sup><sup>11</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Esteve-Codina%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id12"><span class="name western">Anna Esteve-Codina</span></a><div hidden="hidden" id="id12">
<h3><span class="name western">Anna Esteve-Codina</span></h3>
<div class="p">
<sup>10</sup>Centro Nacional de Análisis Genómico (CNAG), C/Baldiri Reixac 4, 08028 Barcelona, Spain </div>
<div class="p">
<sup>11</sup>University of Barcelona (UB), 08034 Barcelona, Spain </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Esteve-Codina%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Anna Esteve-Codina</span></a>
</div>
</div>
<sup>10,</sup><sup>11</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Cantos-Cortes%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id13"><span class="name western">Ana Cantos-Cortes</span></a><div hidden="hidden" id="id13">
<h3><span class="name western">Ana Cantos-Cortes</span></h3>
<div class="p">
<sup>1</sup>TGF-β and Cancer Group, Oncobell Program, Bellvitge Biomedical Research Institute (IDIBELL), Barcelona, Spain </div>
<div class="p">
<sup>2</sup>CIBEREHD, National Biomedical Research Institute on Liver and Gastrointestinal Diseases, Instituto de Salud Carlos III, Madrid, Spain </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Cantos-Cortes%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Ana Cantos-Cortes</span></a>
</div>
</div>
<sup>1,</sup><sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Espinosa-Sotelo%20R%22%5BAuthor%5D" class="usa-link" aria-describedby="id14"><span class="name western">Rut Espinosa-Sotelo</span></a><div hidden="hidden" id="id14">
<h3><span class="name western">Rut Espinosa-Sotelo</span></h3>
<div class="p">
<sup>1</sup>TGF-β and Cancer Group, Oncobell Program, Bellvitge Biomedical Research Institute (IDIBELL), Barcelona, Spain </div>
<div class="p">
<sup>2</sup>CIBEREHD, National Biomedical Research Institute on Liver and Gastrointestinal Diseases, Instituto de Salud Carlos III, Madrid, Spain </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Espinosa-Sotelo%20R%22%5BAuthor%5D" class="usa-link"><span class="name western">Rut Espinosa-Sotelo</span></a>
</div>
</div>
<sup>1,</sup><sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ramos%20E%22%5BAuthor%5D" class="usa-link" aria-describedby="id15"><span class="name western">Emilio Ramos</span></a><div hidden="hidden" id="id15">
<h3><span class="name western">Emilio Ramos</span></h3>
<div class="p">
<sup>2</sup>CIBEREHD, National Biomedical Research Institute on Liver and Gastrointestinal Diseases, Instituto de Salud Carlos III, Madrid, Spain </div>
<div class="p">
<sup>12</sup>Faculty of Medicine and Health Sciences, University of Barcelona, L’Hospitalet de Llobregat, Barcelona, Spain </div>
<div class="p">
<sup>13</sup>Department of Surgery, Liver Transplant Unit, University Hospital of Bellvitge, Barcelona, Spain </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ramos%20E%22%5BAuthor%5D" class="usa-link"><span class="name western">Emilio Ramos</span></a>
</div>
</div>
<sup>2,</sup><sup>12,</sup><sup>13</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Serrano%20T%22%5BAuthor%5D" class="usa-link" aria-describedby="id16"><span class="name western">Teresa Serrano</span></a><div hidden="hidden" id="id16">
<h3><span class="name western">Teresa Serrano</span></h3>
<div class="p">
<sup>2</sup>CIBEREHD, National Biomedical Research Institute on Liver and Gastrointestinal Diseases, Instituto de Salud Carlos III, Madrid, Spain </div>
<div class="p">
<sup>12</sup>Faculty of Medicine and Health Sciences, University of Barcelona, L’Hospitalet de Llobregat, Barcelona, Spain </div>
<div class="p">
<sup>14</sup>Pathological Anatomy Service, University Hospital of Bellvitge, Barcelona, Spain </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Serrano%20T%22%5BAuthor%5D" class="usa-link"><span class="name western">Teresa Serrano</span></a>
</div>
</div>
<sup>2,</sup><sup>12,</sup><sup>14</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Calvo%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id17"><span class="name western">Mariona Calvo</span></a><div hidden="hidden" id="id17">
<h3><span class="name western">Mariona Calvo</span></h3>
<div class="p">
<sup>1</sup>TGF-β and Cancer Group, Oncobell Program, Bellvitge Biomedical Research Institute (IDIBELL), Barcelona, Spain </div>
<div class="p">
<sup>15</sup>Oncología Médica, Institut Català d’Oncologia (ICO-IDIBELL), L’Hospitalet del Llobregat, Barcelona, Spain </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Calvo%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Mariona Calvo</span></a>
</div>
</div>
<sup>1,</sup><sup>15</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Laquente%20B%22%5BAuthor%5D" class="usa-link" aria-describedby="id18"><span class="name western">Berta Laquente</span></a><div hidden="hidden" id="id18">
<h3><span class="name western">Berta Laquente</span></h3>
<div class="p">
<sup>15</sup>Oncología Médica, Institut Català d’Oncologia (ICO-IDIBELL), L’Hospitalet del Llobregat, Barcelona, Spain </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Laquente%20B%22%5BAuthor%5D" class="usa-link"><span class="name western">Berta Laquente</span></a>
</div>
</div>
<sup>15</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ferrer%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id19"><span class="name western">Joana Ferrer</span></a><div hidden="hidden" id="id19">
<h3><span class="name western">Joana Ferrer</span></h3>
<div class="p">
<sup>2</sup>CIBEREHD, National Biomedical Research Institute on Liver and Gastrointestinal Diseases, Instituto de Salud Carlos III, Madrid, Spain </div>
<div class="p">
<sup>16</sup>Barcelona Clinic Liver Cancer (BCLC) group. Institut d’Investigacions Biomèdiques August Pi I Sunyer (IDIBAPS). Surgery Department, Liver Oncology Unit, Hospital Clinic de Barcelona, Barcelona, Spain Barcelona University, Barcelona, Spain </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ferrer%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Joana Ferrer</span></a>
</div>
</div>
<sup>2,</sup><sup>16</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Pons%20G%22%5BAuthor%5D" class="usa-link" aria-describedby="id20"><span class="name western">Gabriel Pons</span></a><div hidden="hidden" id="id20">
<h3><span class="name western">Gabriel Pons</span></h3>
<div class="p">
<sup>17</sup>Physiological Sciences Department, University of Barcelona, Barcelona, Spain </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Pons%20G%22%5BAuthor%5D" class="usa-link"><span class="name western">Gabriel Pons</span></a>
</div>
</div>
<sup>17</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mendez-Lucas%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id21"><span class="name western">Andrés Mendez-Lucas</span></a><div hidden="hidden" id="id21">
<h3><span class="name western">Andrés Mendez-Lucas</span></h3>
<div class="p">
<sup>17</sup>Physiological Sciences Department, University of Barcelona, Barcelona, Spain </div>
<div class="p">
<sup>18</sup>Nutrition, metabolism and gene therapy group. Diabetes and Metabolism Program. IDIBELL, L’Hospitalet de Llobregat, Barcelona, Spain </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mendez-Lucas%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Andrés Mendez-Lucas</span></a>
</div>
</div>
<sup>17,</sup><sup>18</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Dooley%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id22"><span class="name western">Steven Dooley</span></a><div hidden="hidden" id="id22">
<h3><span class="name western">Steven Dooley</span></h3>
<div class="p">
<sup>19</sup>Department of Medicine II, Section Molecular Hepatology, Medical Faculty Mannheim, Heidelberg University, Mannheim, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Dooley%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Steven Dooley</span></a>
</div>
</div>
<sup>19</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ilyas%20SI%22%5BAuthor%5D" class="usa-link" aria-describedby="id23"><span class="name western">Sumera I Ilyas</span></a><div hidden="hidden" id="id23">
<h3><span class="name western">Sumera I Ilyas</span></h3>
<div class="p">
<sup>20</sup>Division of Gastroenterology and Hepatology, Mayo Clinic, Rochester, Minnesota USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ilyas%20SI%22%5BAuthor%5D" class="usa-link"><span class="name western">Sumera I Ilyas</span></a>
</div>
</div>
<sup>20</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Vallette%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id24"><span class="name western">Marie Vallette</span></a><div hidden="hidden" id="id24">
<h3><span class="name western">Marie Vallette</span></h3>
<div class="p">
<sup>21</sup>UMRS_938, Sorbonne Université, Inserm, Centre de Recherche Saint-Antoine (CRSA), Paris, France </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Vallette%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Marie Vallette</span></a>
</div>
</div>
<sup>21</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Aoudjehane%20L%22%5BAuthor%5D" class="usa-link" aria-describedby="id25"><span class="name western">Lynda Aoudjehane</span></a><div hidden="hidden" id="id25">
<h3><span class="name western">Lynda Aoudjehane</span></h3>
<div class="p">
<sup>21</sup>UMRS_938, Sorbonne Université, Inserm, Centre de Recherche Saint-Antoine (CRSA), Paris, France </div>
<div class="p">
<sup>22</sup>ICAN BioCell-Human Liver Biology core, IHU- Foundation for innovation in Cardiometabolism and Nutrition (IHU-ICAN), F-75013 Paris, France </div>
<div class="p">
<sup>23</sup>Sorbonne Université, INSERM, UMRS 1166/IHU-ICAN, F-75013 Paris, France </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Aoudjehane%20L%22%5BAuthor%5D" class="usa-link"><span class="name western">Lynda Aoudjehane</span></a>
</div>
</div>
<sup>21,</sup><sup>22,</sup><sup>23</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lequoy%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id26"><span class="name western">Marie Lequoy</span></a><div hidden="hidden" id="id26">
<h3><span class="name western">Marie Lequoy</span></h3>
<div class="p">
<sup>21</sup>UMRS_938, Sorbonne Université, Inserm, Centre de Recherche Saint-Antoine (CRSA), Paris, France </div>
<div class="p">
<sup>24</sup>AP-HP Sorbonne Université, Hôpital Universitaire Saint Antoine, Service d´Hépatologie, F-75012 Paris, France </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lequoy%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Marie Lequoy</span></a>
</div>
</div>
<sup>21,</sup><sup>24</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Fouassier%20L%22%5BAuthor%5D" class="usa-link" aria-describedby="id27"><span class="name western">Laura Fouassier</span></a><div hidden="hidden" id="id27">
<h3><span class="name western">Laura Fouassier</span></h3>
<div class="p">
<sup>21</sup>UMRS_938, Sorbonne Université, Inserm, Centre de Recherche Saint-Antoine (CRSA), Paris, France </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Fouassier%20L%22%5BAuthor%5D" class="usa-link"><span class="name western">Laura Fouassier</span></a>
</div>
</div>
<sup>21</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Coulouarn%20C%22%5BAuthor%5D" class="usa-link" aria-describedby="id28"><span class="name western">Cédric Coulouarn</span></a><div hidden="hidden" id="id28">
<h3><span class="name western">Cédric Coulouarn</span></h3>
<div class="p">
<sup>25</sup>Inserm, Univ Rennes, OSS (Oncogenesis, Stress, Signaling) UMR_S 1242, Centre de Lutte contre le Cancer Eugène Marquis, F-35042 Rennes, France </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Coulouarn%20C%22%5BAuthor%5D" class="usa-link"><span class="name western">Cédric Coulouarn</span></a>
</div>
</div>
<sup>25</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Aff%C3%B2%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id29"><span class="name western">Silvia Affò</span></a><div hidden="hidden" id="id29">
<h3><span class="name western">Silvia Affò</span></h3>
<div class="p">
<sup>6</sup>Tumor Microenvironment Plasticity and Heterogeneity Group, Instituto de Investigaciones Biomédicas August Pi i Sunyer (IDIBAPS), Barcelona, Spain </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Aff%C3%B2%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Silvia Affò</span></a>
</div>
</div>
<sup>6</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Scheiter%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id30"><span class="name western">Alexander Scheiter</span></a><div hidden="hidden" id="id30">
<h3><span class="name western">Alexander Scheiter</span></h3>
<div class="p">
<sup>26</sup>Institute of Pathology, University of Regensburg, Regensburg, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Scheiter%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Alexander Scheiter</span></a>
</div>
</div>
<sup>26</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Calvisi%20DF%22%5BAuthor%5D" class="usa-link" aria-describedby="id31"><span class="name western">Diego F Calvisi</span></a><div hidden="hidden" id="id31">
<h3><span class="name western">Diego F Calvisi</span></h3>
<div class="p">
<sup>26</sup>Institute of Pathology, University of Regensburg, Regensburg, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Calvisi%20DF%22%5BAuthor%5D" class="usa-link"><span class="name western">Diego F Calvisi</span></a>
</div>
</div>
<sup>26</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tian%20TV%22%5BAuthor%5D" class="usa-link" aria-describedby="id32"><span class="name western">Tian V Tian</span></a><div hidden="hidden" id="id32">
<h3><span class="name western">Tian V Tian</span></h3>
<div class="p">
<sup>5</sup>Upper Gastrointestinal and Endocrine Tumour Unit, Vall d’Hebron Institute of Oncology (VHIO), Barcelona, Spain </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tian%20TV%22%5BAuthor%5D" class="usa-link"><span class="name western">Tian V Tian</span></a>
</div>
</div>
<sup>5</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Fabregat%20I%22%5BAuthor%5D" class="usa-link" aria-describedby="id33"><span class="name western">Isabel Fabregat</span></a><div hidden="hidden" id="id33">
<h3><span class="name western">Isabel Fabregat</span></h3>
<div class="p">
<sup>1</sup>TGF-β and Cancer Group, Oncobell Program, Bellvitge Biomedical Research Institute (IDIBELL), Barcelona, Spain </div>
<div class="p">
<sup>2</sup>CIBEREHD, National Biomedical Research Institute on Liver and Gastrointestinal Diseases, Instituto de Salud Carlos III, Madrid, Spain </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Fabregat%20I%22%5BAuthor%5D" class="usa-link"><span class="name western">Isabel Fabregat</span></a>
</div>
</div>
<sup>1,</sup><sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Vaquero%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id34"><span class="name western">Javier Vaquero</span></a><div hidden="hidden" id="id34">
<h3><span class="name western">Javier Vaquero</span></h3>
<div class="p">
<sup>1</sup>TGF-β and Cancer Group, Oncobell Program, Bellvitge Biomedical Research Institute (IDIBELL), Barcelona, Spain </div>
<div class="p">
<sup>2</sup>CIBEREHD, National Biomedical Research Institute on Liver and Gastrointestinal Diseases, Instituto de Salud Carlos III, Madrid, Spain </div>
<div class="p">
<sup>4</sup>HepatoBiliary Tumours Lab, Centro de Investigación del Cáncer and Instituto de Biología Molecular y Celular del Cáncer, CSIC-Universidad de Salamanca, Salamanca, 37007 Spain </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Vaquero%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Javier Vaquero</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>4,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>TGF-β and Cancer Group, Oncobell Program, Bellvitge Biomedical Research Institute (IDIBELL), Barcelona, Spain </div>
<div id="Aff2">
<sup>2</sup>CIBEREHD, National Biomedical Research Institute on Liver and Gastrointestinal Diseases, Instituto de Salud Carlos III, Madrid, Spain </div>
<div id="Aff3">
<sup>3</sup>Department of Physiology and Pharmacology, University of Salamanca, 37007 Salamanca, Spain </div>
<div id="Aff4">
<sup>4</sup>HepatoBiliary Tumours Lab, Centro de Investigación del Cáncer and Instituto de Biología Molecular y Celular del Cáncer, CSIC-Universidad de Salamanca, Salamanca, 37007 Spain </div>
<div id="Aff5">
<sup>5</sup>Upper Gastrointestinal and Endocrine Tumour Unit, Vall d’Hebron Institute of Oncology (VHIO), Barcelona, Spain </div>
<div id="Aff6">
<sup>6</sup>Tumor Microenvironment Plasticity and Heterogeneity Group, Instituto de Investigaciones Biomédicas August Pi i Sunyer (IDIBAPS), Barcelona, Spain </div>
<div id="Aff7">
<sup>7</sup>TUM School of Medicine and Health, Department of Clinical Medicine – Clinical Department for Internal Medicine II, University Medical Center, Technical University of Munich, Munich, Germany </div>
<div id="Aff8">
<sup>8</sup>Unit of Bioinformatics for Precision Oncology, Catalan Institute of Oncology (ICO), L’Hospitalet de Llobregat, Barcelona, Spain </div>
<div id="Aff9">
<sup>9</sup>Preclinical and Experimental Research in Thoracic Tumors (PReTT), Oncobell Program, IDIBELL, L’Hospitalet de Llobregat, Barcelona, Spain </div>
<div id="Aff10">
<sup>10</sup>Centro Nacional de Análisis Genómico (CNAG), C/Baldiri Reixac 4, 08028 Barcelona, Spain </div>
<div id="Aff11">
<sup>11</sup>University of Barcelona (UB), 08034 Barcelona, Spain </div>
<div id="Aff12">
<sup>12</sup>Faculty of Medicine and Health Sciences, University of Barcelona, L’Hospitalet de Llobregat, Barcelona, Spain </div>
<div id="Aff13">
<sup>13</sup>Department of Surgery, Liver Transplant Unit, University Hospital of Bellvitge, Barcelona, Spain </div>
<div id="Aff14">
<sup>14</sup>Pathological Anatomy Service, University Hospital of Bellvitge, Barcelona, Spain </div>
<div id="Aff15">
<sup>15</sup>Oncología Médica, Institut Català d’Oncologia (ICO-IDIBELL), L’Hospitalet del Llobregat, Barcelona, Spain </div>
<div id="Aff16">
<sup>16</sup>Barcelona Clinic Liver Cancer (BCLC) group. Institut d’Investigacions Biomèdiques August Pi I Sunyer (IDIBAPS). Surgery Department, Liver Oncology Unit, Hospital Clinic de Barcelona, Barcelona, Spain Barcelona University, Barcelona, Spain </div>
<div id="Aff17">
<sup>17</sup>Physiological Sciences Department, University of Barcelona, Barcelona, Spain </div>
<div id="Aff18">
<sup>18</sup>Nutrition, metabolism and gene therapy group. Diabetes and Metabolism Program. IDIBELL, L’Hospitalet de Llobregat, Barcelona, Spain </div>
<div id="Aff19">
<sup>19</sup>Department of Medicine II, Section Molecular Hepatology, Medical Faculty Mannheim, Heidelberg University, Mannheim, Germany </div>
<div id="Aff20">
<sup>20</sup>Division of Gastroenterology and Hepatology, Mayo Clinic, Rochester, Minnesota USA </div>
<div id="Aff21">
<sup>21</sup>UMRS_938, Sorbonne Université, Inserm, Centre de Recherche Saint-Antoine (CRSA), Paris, France </div>
<div id="Aff22">
<sup>22</sup>ICAN BioCell-Human Liver Biology core, IHU- Foundation for innovation in Cardiometabolism and Nutrition (IHU-ICAN), F-75013 Paris, France </div>
<div id="Aff23">
<sup>23</sup>Sorbonne Université, INSERM, UMRS 1166/IHU-ICAN, F-75013 Paris, France </div>
<div id="Aff24">
<sup>24</sup>AP-HP Sorbonne Université, Hôpital Universitaire Saint Antoine, Service d´Hépatologie, F-75012 Paris, France </div>
<div id="Aff25">
<sup>25</sup>Inserm, Univ Rennes, OSS (Oncogenesis, Stress, Signaling) UMR_S 1242, Centre de Lutte contre le Cancer Eugène Marquis, F-35042 Rennes, France </div>
<div id="Aff26">
<sup>26</sup>Institute of Pathology, University of Regensburg, Regensburg, Germany </div>
<div class="author-notes p">
<div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div>
<div class="fn" id="_eqcntrb93pmc__">
<sup>#</sup><p class="display-inline">Contributed equally.</p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2024 Sep 2; Revised 2025 Jun 13; Accepted 2025 Jul 10; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12358586  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40820091/" class="usa-link">40820091</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<p id="Par1">Transforming growth factor beta (TGF-β) signalling has become an attractive therapeutic target due to its pro-tumorigenic actions on epithelial cells and its immunosuppressive effects in the tumour microenvironment. In intrahepatic cholangiocarcinoma (iCCA), a highly aggressive malignancy of the biliary tract with poor prognosis, the latest clinical trials using TGF-β inhibitors have failed indicating that the specific actions carried out by TGF-β in iCCA are yet not well delineated. Here, we show that TGF-β signalling is highly active in iCCA and exerts a prominent suppressor effect on tumour cell lines and organoids established from iCCA metastases biopsies, that relies on a functional canonical SMAD2/3/4 signalling. Thus, TGF-β inhibitors promote, instead of inhibiting, tumour cell growth. In this context, a promising strategy is to target intracellular proteins downstream the TGF-β receptors accounting only for TGF-β pro-tumorigenic actions. NADPH oxidase 4 (NOX4), a downstream mediator of the TGF-β signalling pathway, is strictly expressed in cancer-associated fibroblasts (CAF) of iCCA and acts in concert with NOX1 to regulate CAF functions. Use of a dual NOX4/NOX1 inhibitor impaired CAF actions and reduced tumour growth in vitro and in two different in vivo iCCA experimental models. Collectively, our findings reveal an actionable way to specifically target TGF-β pro-tumorigenic actions in CAF from iCCA without undesirable side effects on tumour cells, suggesting a potentially bright future for dual NOX4/NOX1 inhibitors in the clinics, alone or in combination with other therapies.</p>
<section id="kwd-group1" class="kwd-group"><p><strong>Subject terms:</strong> Cancer therapy, Cancer microenvironment, Target identification</p></section></section><section id="Sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p id="Par2">Intrahepatic cholangiocarcinoma (iCCA) is a very aggressive malignancy of the biliary tree with a poor prognosis, mainly due to late diagnosis and resistance to therapies.<sup><a href="#CR1" class="usa-link" aria-describedby="CR1">1</a>,<a href="#CR2" class="usa-link" aria-describedby="CR2">2</a></sup> iCCA is characterised by a prominent tumour microenvironment (TME) enriched with cancer-associated fibroblasts (CAF), which produce abundant collagen and elastic fibres that are deposited in the extracellular matrix (ECM). These components contribute to tumour progression and chemoresistance.<sup><a href="#CR1" class="usa-link" aria-describedby="CR1">1</a>,<a href="#CR2" class="usa-link" aria-describedby="CR2">2</a></sup> For the majority of iCCA patients who are not eligible for surgery, the combination of chemotherapy (gemcitabine plus cisplatin) with immunotherapy (durvalumab or pembrolizumab) was recently approved as the new standard of care.<sup><a href="#CR3" class="usa-link" aria-describedby="CR3">3</a>–<a href="#CR5" class="usa-link" aria-describedby="CR5">5</a></sup> Although the use of immunotherapy represents the first improvement in patient care in many years, this combination still shows limited efficacy in a significant number of patients. Thus, finding new ways to boost the immune response in these unresponsive patients has become essential. In this context, Transforming Growth Factor-beta (TGF-β), which displays potent immunosuppressive activity, has emerged as an attractive target, and the use of TGF-β inhibitors has gained interest. Nevertheless, the latest phase II/III clinical trial (<a href="https://clinicaltrials.gov/ct2/show/NCT04066491" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">NCT04066491</a>) using a TGF-β inhibitor in combination with chemotherapy failed to meet expectations and has been discontinued,<sup><a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>,<a href="#CR7" class="usa-link" aria-describedby="CR7">7</a></sup> indicating that there are unknown mechanisms compromising the use of TGF-β as a therapeutical target in CCA. Importantly, despite the use of TGF-β inhibitors in CCA clinical trials, there is no evidence supporting their efficacy in preclinical models of CCA.</p>
<p id="Par3">TGF-β activates a complex signalling pathway that regulates different cellular processes, including proliferation, migration, and cell death.<sup><a href="#CR8" class="usa-link" aria-describedby="CR8">8</a></sup> Thus, TGF-β dysregulation plays an important role in cancer development and progression, but its use as a therapeutic target remains challenging due to its dual ability to both inhibit and promote carcinogenesis. In the liver, as in many other organs, TGF-β acts as a tumour suppressor in the early stages of tumour development, while in advanced stages it exerts tumour-promoting actions.<sup><a href="#CR8" class="usa-link" aria-describedby="CR8">8</a></sup> In addition, TGF-β helps to generate an advantageous TME by activating CAF, inducing angiogenesis and promoting immune suppression and evasion.<sup><a href="#CR8" class="usa-link" aria-describedby="CR8">8</a></sup> Despite being thoroughly investigated in some cancers such as hepatocellular carcinoma (HCC)<sup><a href="#CR9" class="usa-link" aria-describedby="CR9">9</a></sup> and pancreatic cancer,<sup><a href="#CR10" class="usa-link" aria-describedby="CR10">10</a></sup> these processes are still understudied in CCA.<sup><a href="#CR11" class="usa-link" aria-describedby="CR11">11</a></sup></p>
<p id="Par4">Therefore, the most efficient strategies to successfully target the TGF-β signalling pathway would be those that allow for the preservation of the suppressor effects of TGF-β while impairing its pro-tumorigenic actions. In this context, a promising strategy is to target intracellular proteins downstream of the TGF-β receptors that account solely for TGF-β pro-tumorigenic functions. Among these candidates, NOX4, a member of the NADPH oxidase family, is a downstream mediator of TGF-β signalling in both tumour cells and the TME,<sup><a href="#CR12" class="usa-link" aria-describedby="CR12">12</a></sup> and is a key source of reactive oxygen species (ROS). We have previously shown that NOX4 acts as a negative regulator of hepatocarcinogenesis in tumour cells,<sup><a href="#CR13" class="usa-link" aria-describedby="CR13">13</a></sup> while other studies have demonstrated that NOX4 may be involved in tumour progression and chemoresistance in other tumour types.<sup><a href="#CR14" class="usa-link" aria-describedby="CR14">14</a></sup> Despite its controversial role in tumour cells, most reports support the central and specific role of NOX4 in regulating CAF activation in multiple cancers.<sup><a href="#CR15" class="usa-link" aria-describedby="CR15">15</a></sup> However, recent reports also suggest that NOX4 is dispensable for myofibroblast differentiation.<sup><a href="#CR16" class="usa-link" aria-describedby="CR16">16</a></sup> Interestingly, the functions and therapeutic potential of NOX4 in tumour cells and the TME from iCCA still remain unknown.</p>
<p id="Par5">Here, we aimed to better understand the role and therapeutic potential of the TGF-β-NOX4 axis in iCCA. Using comprehensive in vitro and in vivo models, we found that TGF-β signalling activation occurs widely across species in iCCA. Our analysis reveals that, regardless of the tumour stage, TGF-β exerts a potent suppressor effect on iCCA tumour cells that is dependent on a functional SMAD2/3/4 complex and that inhibition of TGF-β receptors boosts iCCA progression by promoting tumour cell growth. We also demonstrate that NOX4 is upregulated in iCCA and specifically expressed in CAF. However, NOX4 depletion is insufficient to affect iCCA progression due to a compensatory mechanism driven by NOX1. Accordingly, NOX4/NOX1 dual inhibition impairs CAF functions in vitro and reduces tumour burden in preclinical in vivo models of iCCA.</p></section><section id="Sec2"><h2 class="pmc_sec_title">Results</h2>
<section id="Sec3"><h3 class="pmc_sec_title">TGF-β signalling is activated in both human and murine CCA samples</h3>
<p id="Par6">We first analysed the status of the TGF-β signalling pathway in CCA patients in comparison with different murine models of iCCA, because the information about TGF-β in the later species is scarce in the literature. We used state-of-the-art mouse iCCA models,<sup><a href="#CR17" class="usa-link" aria-describedby="CR17">17</a></sup> including the syngeneic orthotopic SB1 model<sup><a href="#CR18" class="usa-link" aria-describedby="CR18">18</a></sup> and the hydrodynamic tail vein injection (HTVI) AKT-YAP, and the AKT-NICD models.<sup><a href="#CR19" class="usa-link" aria-describedby="CR19">19</a></sup> Notably, these mouse models develop iCCA that, like human iCCA, show increased expression of <em>Krt19</em>, indicative of malignant cholangiocyte proliferation, and <em>Acta2</em> and <em>Col1a1</em>, indicative of a well-formed desmoplastic stroma with a high content of CAF (Supplementary Fig. <a href="#MOESM1" class="usa-link">1a, b</a>). We therefore compared them with data from iCCA patients from our own cohort of patients (IDIBELL cohort) and two public databases (TCGA and TIGER-LC cohorts). Our analyses revealed a significant increase in mRNA expression of <em>TGFB1-2</em> ligands and <em>TGFBR1</em> in iCCA compared to non-tumoral tissue, both in human cohorts (IDIBELL, TCGA and TIGER-LC) and in the iCCA animal models (Fig. <a href="#Fig1" class="usa-link">1a, b</a>). The expression of <em>TGFB3</em> and <em>TGFBR2</em> was also increased in the animal models and in the TCGA cohort, while the <em>TGFBR3</em> expression remained stable or decreased, depending on the model (Fig. <a href="#Fig1" class="usa-link">1a, b</a>). The increase in TGF-β ligands and receptors suggested activation of the TGF-β signalling in iCCA, that was confirmed by increased phosphorylation of SMAD2 in both human and murine iCCA tissues, compared to non-tumoral liver (Fig. <a href="#Fig1" class="usa-link">1c</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">2a</a>). The activation of TGF-β signalling led to increased downstream gene expression, as indicated by the correlation of <em>SMAD7</em> expression—a typical SMAD-positively regulated gene—with <em>TGFB1</em> in human iCCA (Supplementary Fig. <a href="#MOESM1" class="usa-link">2b</a>) and the increased <em>Smad7</em> mRNA expression in tumours from the mouse iCCA models (Supplementary Fig. <a href="#MOESM1" class="usa-link">2c</a>). Importantly, high expression of TGF-β1 has been correlated with poor survival in iCCA patients.<sup><a href="#CR20" class="usa-link" aria-describedby="CR20">20</a></sup> Accordingly, we found that high expression of <em>TGFBR1</em> and <em>TGFBR2</em> receptors also correlates with poor survival in a cohort of 246 iCCA patients (<a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE244807" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE244807</a>) (Fig. <a href="#Fig1" class="usa-link">1d</a>). Moreover, since genetic mutations in core components of the TGF-β signalling pathway, such as <em>SMAD4</em>,<sup><a href="#CR21" class="usa-link" aria-describedby="CR21">21</a></sup> are very frequent in some tumours, we decided to rule out this mechanism of escape from TGF-β suppressor effects in iCCA. Analysis of the mutational status of the ligands (<em>TGFB1-3</em>), receptors (<em>TGFBR1-3</em>) and SMADs (<em>SMAD2-4</em>) in 1121 iCCA patients revealed a low mutation frequency in these genes, with driver mutations present in only 6.6% of patients (Fig. <a href="#Fig1" class="usa-link">1e</a>). Altogether, these data demonstrate that TGF-β signalling is active and functional across species in iCCA and impacts iCCA patient survival.</p>
<figure class="fig xbox font-sm" id="Fig1"><h4 class="obj_head">Fig. 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12358586_41392_2025_2347_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f9cc/12358586/839803277ee7/41392_2025_2347_Fig1_HTML.jpg" loading="lazy" id="d33e902" height="999" width="711" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>TGF-β signalling is activated and functional in human and mouse intrahepatic cholangiocarcinoma (iCCA) tumours. <strong>a</strong>, <strong>b</strong> TGF-β ligands (<em>TGFB</em>) and receptors (<em>TGFBR</em>) mRNA expression in CCA tumours compared to surrounding liver tissue from the IDIBELL (<em>n</em> = 17), the TCGA (<em>n</em> = 36) and TIGER-LC (<em>n</em> = 91) cohorts of patients (<strong>a</strong>) and from the SB1 syngeneic orthotopic (<em>n</em> = 9) and the AKT-YAP (<em>n</em> = 7) and AKT-NICD hydrodynamic tail vein injection iCCA tumour models (<strong>b</strong>). <strong>c</strong> Representative immunohistochemical images showing phophoSMAD2 expression, used as a readout of the activation of the TGF-β signalling pathway, in samples from iCCA patients, and SB1, AKT-YAP and AKT-NICD iCCA animal models. Scale: 100 µm. <strong>d</strong> Kaplan-Meier curve of 5-year overall survival of patients from <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE244807" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE244807</a> cohort (<em>n</em> = 246) stratified based on <em>TGFBR1 and TGFBR2</em> expression (Cox Proportional Hazards model). <strong>e</strong> Mutational status of different members of the TGF-β canonical signalling pathway in iCCA patients. mRNA expression levels were determined by RT-qPCR in samples from the IDIBELL cohort of patients and the animal models. Gene expression data from public data bases was obtained using RNAseq (TCGA cohort) or microarrays (TIGER cohort). Values are expressed as means ± SEM. *<em>p</em> &lt; 0.05; **<em>p</em> &lt; 0.01, ***<em>p</em> &lt; 0.001, ****<em>p</em> &lt; 0.0001 as compared to NT. NT non-tumoral, T tumour</p></figcaption></figure></section><section id="Sec4"><h3 class="pmc_sec_title">TGF-β inhibition promotes CCA tumour cell growth in vitro</h3>
<p id="Par7">The dual effect of TGF-β in tumour cells has been extensively studied in different cancers, including HCC. TGF-β is able to repress proliferation and induce apoptosis in HCC cells that resemble earlier stages of tumour development, while cells resembling advanced stages are able to escape from TGF-β suppressor functions by different mechanisms and exploit its pro-tumorigenic properties, becoming more migratory and invasive.<sup><a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>–<a href="#CR24" class="usa-link" aria-describedby="CR24">24</a></sup> Since these processes remain largely understudied in iCCA, we performed in vitro studies to evaluate the effects of TGF-β1 on the viability and morphology of seven human iCCA cell lines (Fig. <a href="#Fig2" class="usa-link">2a, b</a>). Five cell lines (KKU-213A, KKU-213C, HuCCT1, HuH28 and RBE) showed a significant reduction in cell viability below 50% and changes in morphology that could account for an epithelial-mesenchymal transition process, while two cell lines (CCLP1 and SG231) did not show changes in these parameters (Fig. <a href="#Fig2" class="usa-link">2a, b</a>). We then analysed the activation of canonical and non-canonical intracellular signalling pathways by determining the phosphorylation levels of SMAD2 and SMAD3, and STAT3, AKT and ERK, respectively. This analysis revealed activation of non-canonical signalling (STAT3, AKT or ERK) pathways in all cell lines (Fig. <a href="#Fig2" class="usa-link">2c</a>). Interestingly, only the five cell lines with reduced viability upon TGF-β1 treatment showed increased phosphorylation of canonical SMAD2/SMAD3, whereas the two cell lines that did not exhibit reduced viability did not display increased phosphorylation of either SMAD2 or SMAD3 (Fig. <a href="#Fig2" class="usa-link">2c</a>). To explain the lack of TGF-β1 effects we analysed the expression of the main actors involved in the TGF-β canonical signalling pathway. All seven cell lines showed normal and detectable mRNA levels of the three TGF-β ligands and three TGF-β receptors (Supplementary Fig. <a href="#MOESM1" class="usa-link">3</a>). However, further analysis of SMAD protein expression showed that CCLP1 cells lacked SMAD3 expression, and SG231 cells expressed very high levels of SMAD7 (Fig. <a href="#Fig2" class="usa-link">2d</a>), which acts as a negative regulator of TGF-β signalling,<sup><a href="#CR25" class="usa-link" aria-describedby="CR25">25</a></sup> explaining why these two cell lines do not respond to TGF-β. In summary (Fig. <a href="#Fig2" class="usa-link">2e</a>), TGF-β1 exerts a strong suppressive effect on iCCA cell lines that is dependent on a functional canonical SMAD signalling, which represents an important difference with HCC, where cells can maintain an active canonical TGF-β signalling and still not respond to TGF-β suppressor effects. Further analysis of HuCCT1 and RBE cells, which respond to TGF-β1 in terms of viability suppression (Fig. <a href="#Fig2" class="usa-link">2a</a>) and SMAD-dependent gene regulation (Supplementary Fig. <a href="#MOESM1" class="usa-link">4</a>), indicated that the suppressive effect of TGF-β1 occurs via reduced cell proliferation, as indicated by reduced Ki67 staining (Supplementary Fig. <a href="#MOESM1" class="usa-link">5a</a>). We also identified a reduction in cyclin D1 expression, suggesting a possible mechanism (Supplementary Fig. <a href="#MOESM1" class="usa-link">5b</a>). Finally, we observed an induction of apoptosis, indicated by increased cleavage of PARP (cPARP) (Supplementary Fig. <a href="#MOESM1" class="usa-link">5c</a>).</p>
<figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12358586_41392_2025_2347_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f9cc/12358586/dfefa0f558e8/41392_2025_2347_Fig2_HTML.jpg" loading="lazy" id="d33e991" height="856" width="790" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>TGF-β1 effects on intrahepatic cholangiocarcinoma (iCCA) cell lines. <strong>a</strong> Cell viability of iCCA cell lines after exposure to TGF-β1 (2 ng/ml) for 72 h, as determined by cell counting. <strong>b</strong> Representative images showing the morphology of the iCCA cell lines at the time of cell viability determination. Scale: 250 µm. <strong>c</strong> Representative images of Western blot analysis of phosphorylation and total expression of SMAD2, SMAD3, STAT3, AKT and ERK1/2 in iCCA cell lines exposed to TGF-β1 (2 ng/ml) for the indicated times. <strong>d</strong> Representative images of Western blot analysis showing the basal protein expression of SMAD2, SMAD3, SMAD4 and SMAD7 in iCCA cell lines. <strong>e</strong> Summary of responses in a panel of 7 iCCA cell lines to TGF-β1 exposure based on the results displayed in panels <strong>a</strong>-<strong>d</strong>. Values are expressed as means ± SEM from at least 3 cultures. **<em>p</em> &lt; 0.01, ***<em>p</em> &lt; 0.001 as compared to the vehicle</p></figcaption></figure><p id="Par8">The strong suppressor effect induced by TGF-β1 on iCCA cell lines led us to hypothesize that the use of TGF-β inhibitors, such as galunisertib (a selective TGF-β receptor type I kinase inhibitor), would produce the opposite effect. Indeed, galunisertib inhibited the phosphorylation of SMAD2 induced by TGF-β1 treatment in HuCCT1 and RBE, and also reduced the basal levels of SMAD2 phosphorylation (Fig. <a href="#Fig3" class="usa-link">3a</a>), indicating that it is able to impair the autocrine signalling of the TGF-β produced by iCCA cells. We therefore conducted 2D and 3D in vitro functional studies to validate this point. Colony and spheroid formation assays demonstrated that TGF-β1 strongly reduced and galunisertib significantly increased the number of colonies and the size of the spheres formed by HuCCT1 and RBE cells (Fig. <a href="#Fig3" class="usa-link">3b, c</a>). The combination of TGF-β1 and galunisertib showed similar effects than galunisertib alone indicating that this dose of galunisertib is enough to inhibit the autocrine TGF-β signalling in iCCA cell lines, as well as the exocrine TGF-β1 added to the cultures. Galunisertib is able to inhibit numerous other kinases at this dose.<sup><a href="#CR26" class="usa-link" aria-describedby="CR26">26</a></sup> However, all these off targets are involved in increasing proliferation/survival of the cells and, thus, its inhibition would not explain the increase in the number of colonies and in the size of spheres (Fig. <a href="#Fig3" class="usa-link">3b, c</a>), indicating that the effects observed are mediated by TGF-β signalling inhibition.</p>
<figure class="fig xbox font-sm" id="Fig3"><h4 class="obj_head">Fig. 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12358586_41392_2025_2347_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f9cc/12358586/f5670ff2c3d7/41392_2025_2347_Fig3_HTML.jpg" loading="lazy" id="d33e1048" height="999" width="787" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>TGF-β receptor inhibition promotes intrahepatic cholangiocarcinoma (iCCA) tumour cell growth by inhibiting SMAD canonical signalling. <strong>a</strong> Western blot images showing SMAD2 phosphorylation in HuCCT1 and RBE cells treated with TGF-β1 (2 ng/ml), TGF-β1 receptor I inhibitor galunisertib (10 µM) or both, for 1 h. <strong>b</strong> Number of colonies of HuCCT1 and RBE cells after 7-10 days of treatment with TGF-β1 (2 ng/ml), galunisertib (10 µM) or both. Representative images of colonies are shown <strong>c</strong> Size of spheres from HuCCT1 and RBE cells after 4 days of treatment with TGF-β1, galunisertib (10 µM) or both. Representative images of spheres at the time of size analysis are shown. Scale: 100 µm. <strong>d, e</strong> RNA-seq analyses conducted in HuCCT1 and RBE cells treated with TGF-β1 (2 ng/ml) or TGF-β1 receptor I inhibitor galunisertib (10 µM). Heatmap (<strong>d</strong>) showing changes in the expression of different genes in the iCCA cell lines in response to TGF-β1 or galunisertib. Dot plot (<strong>e</strong>) showing differences in enrichment for key events related with “ECM (extracellular matrix) and migration” and “Cell cycle and proliferation” in iCCA cells. <strong>f</strong> Dose-response curves of TGF-β1 and galunisertib for PDXO153 organoids derived from CCA_PDXs (PDXO). Cell viability was determined using a Cell Titer-Glo assay 4 days after the treatment initiation. Representative images of Western blot analysis of phosphoSMAD3 and total SMAD3 in PDX153 organoids after treatment with TGF-β1 (2 ng/ml) or galunisertib (10 µM) for 4 days are shown. <strong>g</strong> Representative immunohistochemical images showing human panCK (labelling human iCCA tumour cells), murine vimentin (labelling stromal murine cells) and Ki67 (as a marker of cell proliferation) in PDXO153 organoids after the same treatment than in <strong>f</strong>. Scale: 200 µm. Values are expressed as means ± SEM from at least 3 cultures. **<em>p</em> &lt; 0.01, ***<em>p</em> &lt; 0.001, as compared to the vehicle. V vehicle, T TGF-β1, G galunisertib</p></figcaption></figure><p id="Par9">To better understand the actions of TGF-β signalling in iCCA, we compared the transcriptomic changes induced in iCCA cells after exposing them to TGF-β1 or galunisertib alone by RNA-seq analysis, showing that the top genes regulated were modulated in an opposite way by TGF-β1 and galunisertib, as expected (Fig. <a href="#Fig3" class="usa-link">3d</a>). Gene set enrichment analysis (GSEA) further validated the effect of these compounds, showing upregulation and inhibition of signalling by TGF-β-receptor complex and SMADs in cells treated with TGF-β1 and galunisertib, respectively (Supplementary Fig. <a href="#MOESM1" class="usa-link">6</a>-<a href="#MOESM1" class="usa-link">7</a>). Notably, <em>SMAD7</em> and <em>SERPINE1</em> were upregulated by TGF-β1 and downregulated by galunisertib (Fig. <a href="#Fig3" class="usa-link">3d</a>). It was previously described, that the pro-tumorigenic effects of TGF-β are related to the increase of the migration capabilities of the cell and the modification of the ECM to render them more invasive,<sup><a href="#CR8" class="usa-link" aria-describedby="CR8">8</a>,<a href="#CR27" class="usa-link" aria-describedby="CR27">27</a></sup> while its suppressor effects are related to the regulation of cell cycle and proliferation.<sup><a href="#CR8" class="usa-link" aria-describedby="CR8">8</a></sup> In line with this, our GSEA analysis revealed that TGF-β1 upregulated pathways related to ECM modulation and migration (i.e. interactions with surface integrins), while it downregulated pathways involved in DNA synthesis and cell cycle progression (Fig. <a href="#Fig3" class="usa-link">3e</a> and Supplementary Figs. <a href="#MOESM1" class="usa-link">8</a>–<a href="#MOESM1" class="usa-link">10</a>), consistent with TGF-β suppressor effects. Intriguingly, the effects of galunisertib on ECM and migration were much less prominent, but its effects were concentrated in upregulating pathways related to cell cycle progression and DNA synthesis (Fig. <a href="#Fig3" class="usa-link">3e</a> and Supplementary Figs. <a href="#MOESM1" class="usa-link">8</a>–<a href="#MOESM1" class="usa-link">10</a>). When analysing specific genes, we observed upregulation by TGF-β and downregulation by galunisertib of typical TGF-β target genes related to its suppressor activity (<em>BMF</em>, <em>CDKN1A</em> and <em>GADD45B</em>). Additionally, we also identified other genes with the same profile (Fig. <a href="#Fig3" class="usa-link">3d</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">11a</a>) that are not TGF-β known targets but would be of interest to be explored in the future as there are reports showing a potential suppressor role in other tumours (<em>CCDC80</em><sup><a href="#CR28" class="usa-link" aria-describedby="CR28">28</a></sup>, <em>FRMD6</em><sup><a href="#CR29" class="usa-link" aria-describedby="CR29">29</a></sup> and <em>VDR</em><sup><a href="#CR30" class="usa-link" aria-describedby="CR30">30</a></sup>). Interestingly, the expression of some of these genes (<em>BMF</em>, <em>CDKN1A</em>, <em>GADD45B</em> and <em>CCDC80</em>) changed accordingly in the spheroid model (Supplementary Fig. <a href="#MOESM1" class="usa-link">11b</a>).</p>
<p id="Par10">We wanted to further validate the effect of TGF-β1 and galunisertib in organoids derived from PDXs (PDXO) established from iCCA metastases biopsies, which better recapitulate tumour complexity. Three of the four organoid models tested showed a strong decrease in viability by TGF-β1 and a significant increase by galunisertib (Fig. <a href="#Fig3" class="usa-link">3f</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">12</a>). In line with our previous observations, the only organoid showing no changes in viability lacked a functional canonical TGF-β signalling with no SMAD3 phosphorylation, corroborating that TGF-β signalling inhibition promotes tumour cell growth only when SMAD canonical signalling is properly functional. To further validate the suppressor role of TGF-β signalling in the organoids, we conducted co-immunohistochemical studies of panCK and Ki67 to specifically label tumour proliferating cells. As shown in Fig. <a href="#Fig3" class="usa-link">3g</a>, TGF-β reduced both organoid size and tumour cell proliferation, while galunisertib increased both of them. Lack of murine vimentin staining confirmed the absence of murine cells and the purity of our human organoids.</p>
<p id="Par11">Altogether, our results show that TGF-β impairs cell growth in all iCCA cell lines with intact TGF-β signalling, and not in those with impaired signalling. Therefore, TGF-β role over the tumour cells is prominently suppressor in iCCA and, thus, the use of inhibitors targeting the TGF-β receptor may present counterproductive effects by inducing tumour cell proliferation.</p></section><section id="Sec5"><h3 class="pmc_sec_title">NOX4 is overexpressed in iCCA and it is restricted to CAF</h3>
<p id="Par12">Facing the above-described evidence, we decided to follow a different strategy and investigate downstream mediators of the TGF-β signalling pathway that would allow us to target exclusively TGF-β pro-tumorigenic actions. Among the potential candidates, we focused our attention on NOX4 because it has been implicated in mediating TGF-β pro-tumorigenic effects on the TME, more specifically in CAF activation in several tumours.<sup><a href="#CR15" class="usa-link" aria-describedby="CR15">15</a></sup> Furthermore, NOX4 inhibitors exist, are commercially available and are being tested at the clinical level in different settings. This potential, together with the lack of information about the role and therapeutic potential of NOX4 in iCCA tumour cells and TME, makes NOX4 the ideal candidate.</p>
<p id="Par13">Analysis of the human datasets showed increased <em>NOX4</em> expression in human iCCA <em>versus</em> surrounding non-tumorous liver (Fig. <a href="#Fig4" class="usa-link">4a</a>). Interestingly, analysis of scRNAseq data demonstrated that <em>NOX4</em> expression was restricted primarily to CAF (Fig. <a href="#Fig4" class="usa-link">4b</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">13</a>). Accordingly, the expression of <em>NOX4</em> correlated with the expression of CAF markers in iCCA patients (Supplementary Fig. <a href="#MOESM1" class="usa-link">14</a>). Further evaluation of scRNAseq data allowed us to define the four major CAF subpopulations (myCAF, iCAF, vCAF and apCAF) and to determine that <em>NOX4</em> was mainly expressed in myCAF and, to a lesser extent, in apCAF (Fig. <a href="#Fig4" class="usa-link">4b</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">15</a>). To confirm the expression of NOX4 in CAF from iCCA we carried out immunohistochemical analysis and staining for different markers of tumour cells and CAF. For the staining of NOX4 we used an antibody previously manufactured and validated for IHC.<sup><a href="#CR31" class="usa-link" aria-describedby="CR31">31</a></sup> Indeed, our analysis revealed that NOX4 shows an intense and specific localization in stromal areas of iCCA where α-SMA and Picro Sirius red signals are also strong, indicating NOX4 expression in CAF, while no expression was found in CCA tumour cells (Fig. <a href="#Fig4" class="usa-link">4c</a>). Furthermore, analysis of data from laser-microdissected iCCA stromal samples revealed an increase in NOX4 expression in tumour stroma <em>versus</em> non-tumoral stroma (Fig. <a href="#Fig4" class="usa-link">4d</a>) and a correlation of <em>NOX4</em> expression with that of typical markers of CAF (<em>ACTA2</em> and <em>COL1A1</em>) (Fig. <a href="#Fig4" class="usa-link">4e</a>), suggesting that <em>NOX4</em> expression may play a role in the activation of CAF during carcinogenesis. Indeed, <em>NOX4</em> expression was absent in iCCA cell lines, compared to non-tumoral liver or HCC cell lines (Supplementary Fig. <a href="#MOESM1" class="usa-link">16</a>), while it was present in LX2-HSC and hTERT-HSC cell lines, primary HSC-GFP and CAF from iCCA, correlating with the degree of activation as indicated by the levels of α-SMA (Fig. <a href="#Fig4" class="usa-link">4f</a>). Exposure of these fibroblastic cells to TGF-β1 to stimulate their activation showed an increased expression of <em>NOX4</em> compared to the vehicle, concomitant with an increased expression of α-SMA (<em>ACTA2</em>) and Collagen 1 (<em>COL1A1</em>) (Fig. <a href="#Fig4" class="usa-link">4g–j</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">17a, b</a>). Interestingly, in HSC cells the mRNA expression of <em>NOX4</em> increased at earlier times (from 3 h to 72 h) (Fig. <a href="#Fig4" class="usa-link">4g</a>), compared to that of <em>ACTA2</em> and <em>COL1A1</em> (starting from 24 h) (Supplementary Fig. <a href="#MOESM1" class="usa-link">17</a> a-b), suggesting that NOX4 may be indeed a downstream mediator of TGF-β induced fibroblast transdifferentiation, as it has been described in other tumours.<sup><a href="#CR15" class="usa-link" aria-describedby="CR15">15</a></sup></p>
<figure class="fig xbox font-sm" id="Fig4"><h4 class="obj_head">Fig. 4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12358586_41392_2025_2347_Fig4_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f9cc/12358586/bec254500c2e/41392_2025_2347_Fig4_HTML.jpg" loading="lazy" id="d33e1383" height="1000" width="779" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>NOX4 is overexpressed in intrahepatic cholangiocarcinoma (iCCA) and specifically expressed in cancer-associated fibroblasts (CAF). <strong>a</strong>
<em>NOX4</em> mRNA expression in CCA tumours compared to surrounding liver tissue (NTL) from the IDIBELL (<em>n</em> = 17), the TCGA (<em>n</em> = 36) and TIGER-LC (<em>n</em> = 91) cohorts of patients. <strong>b</strong> Dot plot showing <em>NOX4</em> expression in cell types and cancer-associated fibroblasts (CAF) subtypes (apCAF antigen-presenting CAF, iCAF inflammatory CAF, myCAF myofibroblast CAF vCAF vascular CAF) from scRNAseq data set <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE201425" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE201425</a> from CCA biopsies. <strong>c</strong> Representative immunohistochemistry images from human iCCA showing tumour architecture (hematoxylin-eosin, H&amp;E), tumour cells (CK19), extracellular matrix (Picro Sirius Red) produced by CAF (α-SMA), and NOX4 localization. Scale: 200 µm (100 µm in magnification pictures). <strong>d</strong>
<em>NOX4</em> mRNA expression in non-tumoral (NT) or tumoral (T) stromal samples from the Rennes microdissection cohort. <strong>e</strong> Correlation between <em>ACTA2</em> and <em>COL1A1</em> with <em>NOX4</em> expression in microdissected stroma from 10 iCCA samples. <strong>f</strong>
<em>NOX4</em> mRNA and α-SMA protein basal expression in LX2-HSC, hTERT-HSC, HSC-GFP cells and CAF, as determined by RT-QPCR and Western blot, respectively. <strong>g</strong>, <strong>h</strong>
<em>NOX4</em> mRNA expression in LX2-HSC, hTERT-HSC and HSC-GFP cells (<strong>g</strong>) and CAF (<strong>h</strong>) after exposure to TGF-β1 (2 ng/ml) for the indicated times. Values are expressed as fold change <em>versus</em> vehicle (3 h time point in <strong>g</strong>) and represented as means ± SEM from at least 3 cultures. <strong>i, j</strong> Representative images of Western blot analysis of Collagen 1 (COL1), α-SMA and NOX4 protein expression in LX2-HSC, hTERT-HSC and HSC-GFP cells (<strong>i</strong>) and CAF (<strong>j</strong>) after exposure to TGF-β1 (2 ng/ml) for the indicated times. mRNA expression levels were determined by RT-qPCR in samples from the IDIBELL cohort of patients. Gene expression data from public data bases was obtained using RNAseq (TCGA cohort) or microarrays (TIGER cohort). *<em>p</em> &lt; 0.05, **<em>p</em> &lt; 0.01, ***<em>p</em> &lt; 0.001, ****<em>p</em> &lt; 0.0001; as compared to NTL or the vehicle. Correlations were determined using Spearman’s correlation analysis</p></figcaption></figure></section><section id="Sec6"><h3 class="pmc_sec_title">Dual NOX4 and NOX1 depletion is necessary to impair human liver fibroblast transdifferentiation</h3>
<p id="Par14">To evaluate the potential impact of targeting NOX4 in iCCA we performed a syngeneic orthotopic model by implanting the SB1 cells in WT and NOX4<sup>-/-</sup> mice (Supplementary Fig. <a href="#MOESM1" class="usa-link">18</a>). Surprisingly, despite the absence of NOX4 in stromal cells, tumours from WT and NOX4<sup>-/-</sup> mice were identical in size and in the expression levels of <em>Krt19</em>, α-Sma (<em>Acta2</em>) and collagen 1 (<em>Col1a1</em>) (Fig. <a href="#Fig5" class="usa-link">5a, b</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">19a</a>). To determine the role of NOX4 in the transdifferentiation of liver fibroblasts and to explore the potential reasons for the absence of differences observed in NOX4<sup>-/-</sup> mice in vivo, we opted to model the depletion of NOX4 in vitro using the CRISPR-Cas9 strategy. Reduced <em>NOX4</em> mRNA and protein expression in hTERT-HSC cells was observed (Fig. <a href="#Fig5" class="usa-link">5c, d</a>), but once again, both CRISPR-Control and CRISPR-NOX4 cells showed similar responses to TGF-β1 with an increase in the expression of both α-SMA (<em>ACTA2</em>) and Collagen 1 (<em>COL1A1</em>) (Fig. <a href="#Fig5" class="usa-link">5c, d</a>). Interestingly, TGF-β1 was able to increase ROS levels in CRISPR-Control cells, while basal ROS levels were already elevated in CRISPR-NOX4 cells and did not change after exposure to TGF-β1 (Fig. <a href="#Fig5" class="usa-link">5e</a>). This result led us to think that there could be a compensatory mechanism increasing ROS levels enough to allow TGF-β1-induced fibroblast transdifferentiation. To ascertain if another NADPH oxidase was involved, we treated CRISPR-NOX4 cells with diphenyleneiodonium chloride (DPI), a specific NADPH oxidase inhibitor widely used as a reference compound in NOX research. In these cells, DPI abolished TGF-β1 induced increase of α-SMA and Collagen 1 (Fig. <a href="#Fig5" class="usa-link">5f</a>) and reduced ROS levels, indicating that another NADPH oxidase was involved. Further analysis of the expression of other NADPH oxidases and the factors forming their regulatory complexes showed a strong increase in the expression of p22phox (<em>CYBA</em>) and a stable and detectable expression of <em>NOX1</em> (Fig. <a href="#Fig5" class="usa-link">5g</a>), while no other NOX was detectable and the rest of regulatory components did not suffer a change in expression that could account for the increased ROS levels (Supplementary Fig. <a href="#MOESM1" class="usa-link">19b</a>). Notably, NOX1 expression was also detectable at the mRNA and protein level in human CAF from CCA (Supplementary Fig. <a href="#MOESM1" class="usa-link">19c</a>). Since NOX4 is constitutively active and it is regulated by modulating its expression, while NOX1 expression is mostly stable but its activity is regulated by other partners from its complex, we hypothesized that the increase in p22phox resulted from NOX4 knock-down could be able to further increase the activity of NOX1. To investigate this possibility, we attempted to generate double NOX4/NOX1 CRISPR cells, but all our attempts were unsuccessful, probably because total depletion of both NOX4 and NOX1 is too deleterious for the fibroblasts. Thus, we proceeded to perform transient downregulation of <em>NOX1</em> by siRNA transfection. Indeed, siRNA downregulation of <em>NOX1</em> in CRISPR-NOX4 cells clearly downregulated NOX1 protein levels, abolished TGF-β1-induced increase of α-SMA and Collagen 1, and reduced ROS levels (Fig. <a href="#Fig5" class="usa-link">5h</a>), validating this compensatory mechanism. To further elucidate the specific contribution of NOX4 and/or NOX1 to TGF-β-induced fibroblast transdifferentiation, we performed additional siRNA studies in WT cells. Notably, only the combined downregulation of <em>NOX4</em> and <em>NOX1</em> was able to impair TGF-β1-induced increase in α-SMA and Collagen 1 in both hTERT-HSC and LX2-HSC cells (Fig. <a href="#Fig5" class="usa-link">5i</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">20a–c</a>). Immunohistochemical studies with a validated NOX1 antibody<sup><a href="#CR32" class="usa-link" aria-describedby="CR32">32</a></sup> in serial sections from human iCCA confirmed NOX4 and NOX1 protein expression in the same stromal cells (Fig. <a href="#Fig5" class="usa-link">5j</a>). Interestingly, our findings correlate with patient survival, since only patients with low expression of both <em>NOX4</em> and <em>NOX1</em> show better survival than those with high expression of one or both <em>NOXs</em> (cohort <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE244807" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE244807</a>) (Fig. <a href="#Fig5" class="usa-link">5k</a>), indicating the important role played by the duet NOX4/NOX1 in iCCA pathobiology.</p>
<figure class="fig xbox font-sm" id="Fig5"><h4 class="obj_head">Fig. 5.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12358586_41392_2025_2347_Fig5_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f9cc/12358586/9be36e3e873f/41392_2025_2347_Fig5_HTML.jpg" loading="lazy" id="d33e1617" height="984" width="790" alt="Fig. 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>NOX4 and NOX1 act in concert to allow TGF-β-induced fibroblast transdifferentiation. Analyses were conducted on the syngeneic SB1 orthotopic iCCA tumours in WT and NOX4<sup>-/-</sup> mice (<em>n</em> = 6), established as illustrated in Supplementary Fig. <a href="#MOESM1" class="usa-link">18</a>. <strong>a</strong> Representative macroscopic images of the tumours at term and tissue sections stained with hematoxylin-eosin (H&amp;E), to illustrate the size of the tumours, α-SMA to visualize cancer-associated fibroblasts (CAF) and Picro Sirius Red to visualize collagen deposition. Scale (H&amp;E): 2.5 mm. Scale (α-SMA and Picro Sirius Red): 250 µm. <strong>b</strong> α-SMA and Picro Sirius Red positive area was quantified using ImageJ. <strong>c</strong>, <strong>d</strong> mRNA (<strong>c</strong>) and protein (<strong>d</strong>) expression of <em>NOX4</em>, <em>ACTA2</em> (α-SMA) and <em>COL1A1</em> (Collagen 1, COL1) after TGF-β1 treatment for 48 h in the hTERT-HSC CRISPR-Control and CRISPR-NOX4 cells, determined by RT-qPCR and Western blot. <strong>e</strong> H<sub>2</sub>O<sub>2</sub> production was measured by Amplex red in CRISPR-Control and CRISPR-NOX4 cells after TGF-β1 treatment for 24 h and expressed as the amount of H<sub>2</sub>O<sub>2</sub> normalized by mg of protein per hour. <strong>f</strong> Protein expression of α-SMA and COL1A1 and H<sub>2</sub>O<sub>2</sub> production (as determined in <strong>d</strong>, <strong>e</strong>) in CRISPR-NOX4 cells treated with TGF-β1 in absence or presence of the NADPH oxidase inhibitor DPI (2.5 µM). <strong>g</strong> mRNA and protein expression of <em>CYBA</em> (p22phox) and NOX1 after TGF-β1 treatment for 48 h in the CRISPR-NOX4 and CRISPR-control cells, determined by RT-QPCR and Western blot respectively. <strong>h</strong> Protein expression of NOX1, α-SMA and COL1A1 and H<sub>2</sub>O<sub>2</sub> production (as determined in <strong>d</strong>, <strong>e</strong>) in CRISPR-NOX4 cells treated with TGF-β1 and transfected with siRNA against <em>NOX1</em> or the scrambled version (siControl). <strong>i</strong> Protein expression of α-SMA, COL1, NOX4 and NOX1 in hTERT-HSC WT cells transfected with siRNA against <em>NOX4</em>, <em>NOX1</em> or both, and treated with TGF-β1. <strong>j</strong> Immunohistochemical analysis of NOX4 and NOX1 localization in serial tissue sections from human iCCA samples (IDIBELL cohort) (representative images). Scale: 200 µm. <strong>K</strong> Kaplan-Meier curve of 5-year overall survival of patients from <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE244807" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE244807</a> cohort (<em>n</em> = 246) stratified based on <em>NOX4</em> and <em>NOX1</em> expression (Cox Proportional Hazards model). mRNA expression levels are expressed as fold change versus vehicle. Values are expressed as means ± SEM from at least 3 cultures. *<em>p</em> &lt; 0.05; **<em>p</em> &lt; 0.01</p></figcaption></figure></section><section id="Sec7"><h3 class="pmc_sec_title">Dual NOX4/NOX1 inhibition with setanaxib impairs HSC and CAF functions and reduces iCCA growth in vitro</h3>
<p id="Par15">The necessity of impairing the activity of both NOXs to blunt fibroblast transdifferentiation led us to evaluate the activity of a new inhibitor. Despite DPI specificity in molecular experimental settings, its therapeutic development has not advanced. Thus, as a potential therapeutic tool, we decided to use setanaxib, a dual NOX4/NOX1 inhibitor that has been successfully tested in clinical trials of cholangiopathies (<a href="https://clinicaltrials.gov/ct2/show/NCT03226067" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">NCT03226067</a>).<sup><a href="#CR33" class="usa-link" aria-describedby="CR33">33</a></sup> For this purpose, we used LX2-HSC, hTERT-HSC and HSC-GFP that show different degrees of basal activation and we further stimulated them with TGF-β1 in the absence or presence of setanaxib and galunisertib, the latter as positive control for TGF-β1 signalling inhibition. Both setanaxib and galunisertib were able to impair TGF-β1-induced increment in the expression of both α-SMA and Collagen 1 (Fig. <a href="#Fig6" class="usa-link">6a</a>). Interestingly, both drugs showed the ability to inhibit the expression of these proteins in the absence of TGF-β1, especially that of Collagen 1 (Fig. <a href="#Fig6" class="usa-link">6a</a>). Next, we decided to evaluate the transcriptomic effects of both setanaxib and galunisertib in HSC by RNAseq (Supplementary Fig. <a href="#MOESM1" class="usa-link">21a</a>). Interestingly, RNAseq analysis showed that setanaxib was able to regulate a higher number of genes than galunisertib (1390 vs 325) (Supplementary Fig. <a href="#MOESM1" class="usa-link">21b, c</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">22</a>-<a href="#MOESM1" class="usa-link">23</a>). GSEA analysis showed that setanaxib was able to inhibit many signalling pathways involved in myofibroblasts activation, as well as ECM organization, while galunisertib mostly regulated pathways related to ECM metabolism and interaction with integrins (Fig. <a href="#Fig6" class="usa-link">6b</a> and Supplementary Figs. <a href="#MOESM1" class="usa-link">24</a>-<a href="#MOESM1" class="usa-link">26</a>). Interestingly, when we analysed the relative enrichment in the myCAF signature in the HSC cell lines treated with the inhibitors, we observed that both setanaxib and galunisertib were able to reduce it, compared with untreated cells, especially in the case of hTERT-HSC and HSC-GFP cells, indicating the potential of these inhibitors to inhibit the myCAF phenotype (Supplementary Fig. <a href="#MOESM1" class="usa-link">27</a>). Since both galunisertib and setanaxib could inhibit fibroblast functions, we decided to test their impact on CCA growth using mixed spheroids formed by malignant cells and HSC-GFP. To follow the dynamics of the experiment we labelled the tumour cells with mCherry and compared it with the GFP of HSC cells. Setanaxib had no impact on the size of spheroids formed by tumour cells alone or in combination with TMNK1 endothelial cells or THP-1 macrophages (Supplementary Fig. <a href="#MOESM1" class="usa-link">28</a>) but significantly reduced the growth of mixed spheroids (Fig. <a href="#Fig6" class="usa-link">6c</a>), demonstrating the specificity of setanaxib on liver myofibroblasts, the only NOX4/NOX1 expressing cells. However, galunisertib increased the size of mixed spheroids (Fig. <a href="#Fig6" class="usa-link">6c</a>), indicating that, despite being able to inhibit fibroblast functions, galunisertib effects on promoting tumour cell growth are predominant. Interestingly, green fluorescence from HSC-GFP decreased after treatment with setanaxib, which may be due to the inhibition of NOX1 previously described function on fibroblast proliferation.<sup><a href="#CR34" class="usa-link" aria-describedby="CR34">34</a></sup> The inactivation of fibroblasts together with reduced proliferation may impact the crosstalk of tumour cells and fibroblast since both spheroid size and red fluorescence from tumour cells are reduced after setanaxib treatment (Fig. <a href="#Fig6" class="usa-link">6c</a>). Next, we validated the efficacy of setanaxib on CAF from iCCA patients. Indeed, setanaxib was able to reduce the protein levels of CAF markers both in absence and presence of TGF-β1, especially that of Collagen 1 in three different sets of CAF (Fig. <a href="#Fig6" class="usa-link">6d</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">29</a>). Finally, setanaxib was also able to reduce tumour growth in mixed spheroids formed by iCCA cells and CAF, while it did not have any effect on spheroids formed solely by tumour cells (Fig. <a href="#Fig6" class="usa-link">6e</a>). In summary, these data suggest that setanaxib may be a promising strategy to target tumour growth by targeting CAF functions.</p>
<figure class="fig xbox font-sm" id="Fig6"><h4 class="obj_head">Fig. 6.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12358586_41392_2025_2347_Fig6_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f9cc/12358586/0f27c4b5804f/41392_2025_2347_Fig6_HTML.jpg" loading="lazy" id="d33e1716" height="1000" width="785" alt="Fig. 6"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Dual NOX4/NOX1 inhibition impairs TGF-β-induced fibroblast transdifferentiation and cancer-associated fibroblasts (CAF) functions. <strong>a</strong> Representative images of Western blot analysis of Collagen 1 (COL1) and α-SMA in hepatic stellate cell (HSC) lines treated with TGF-β1 or the vehicle in absence or presence of setanaxib (left) or galunisertib (right) for 48 hours. <strong>b</strong> Dot plot showing differences in enrichment for key events related with signalling pathways involved in myofibroblast transdifferentiation in HSC lines treated with setanaxib or galunisertib. <strong>c</strong> Size of mixed spheres from HuCCT1 or RBE cells tagged with mCherry in combination with HSC-GFP after 4 days of treatment with setanaxib or galunisertib. Representative phase contrast images of spheres at the time of size analysis are shown. Also, fluorescence microscopy images of mCherry and GFP intensity are shown. Scale: 100 µm. <strong>d</strong> Representative images of Western blot analysis of COL1 and α-SMA in CAF treated with TGF-β1 (2 ng/ml) or the vehicle in absence or presence of setanaxib (40 µM) for 48 hours. <strong>e</strong> Size of spheres from HuCCT1 (marked in green with FITC) alone or in combination with CAF (marked in red with TRITC) after 1, 4 or 7 days of treatment with setanaxib or the vehicle. Representative phase contrast and images of spheres at the time of size analysis are shown. Also, fluorescence microscopy images of FITC and TRITC are shown. Scale: 50 µm. Values are expressed as means ± SEM from at least 3 cultures. *<em>p</em> &lt; 0.05, **<em>p</em> &lt; 0.01; ***<em>p</em> &lt; 0.001; as compared with the vehicle</p></figcaption></figure></section><section id="Sec8"><h3 class="pmc_sec_title">Dual NOX4/NOX1 inhibition with setanaxib but not TGF-β receptor inhibition with galunisertib reduces iCCA tumour burden in vivo</h3>
<p id="Par16">Based on our in vitro studies, we hypothesised that using TGF-β inhibitors as therapy would be counterproductive due to the opposite effects of galunisertib on tumour cells (increase in tumour cell growth) versus TME cells (inhibition of CAF functions), while the use of NOX4/NOX1 inhibitors, such as setanaxib, would reduce tumour growth by specifically targeting TGF-β pro-tumorigenic actions in both newly recruited HSC and CAF. Thus, as a final step, we decided to compare the effects of galunisertib and setanaxib in vivo. For this purpose we followed two strategies: 1/we conducted an experiment in a PDX model to confirm the deleterious effect of galunisertib in a more humanized model and to evaluate the potential of setanaxib in inhibiting fibroblasts from origins other than the liver; 2/ we carried out an experiment in the AKT-YAP model to evaluate the effects of both drugs in a state-of-the-art iCCA murine model, that shows a prominent desmoplastic stroma with strong collagen deposition and is performed in immunocompetent mice.</p>
<p id="Par17">For the PDX model, we chose PDX153, from which PDXO153, used in our in vitro assays (Fig. <a href="#Fig3" class="usa-link">3f</a>), was derived. After implantation, we waited until the tumours reached a mean size of 150 mm<sup>3</sup> and then we started the treatments that continued for three weeks (Supplementary Fig. <a href="#MOESM1" class="usa-link">30a</a>). As shown in Fig. <a href="#Fig7" class="usa-link">7a</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">30b</a>, galunisertib increased the growth of the tumours compared with the vehicle, while setanaxib strongly suppressed tumour growth. Importantly, both drugs were safe and did not produce any significant changes in the weight of the animals (Fig. <a href="#Fig7" class="usa-link">7b</a>). To further evaluate the effects of galunisertib and setanaxib on the tumours we performed immunohistochemical stainings of different markers. First, we evaluated the growth and survival of cells inside the tumour by staining of Ki67 and cleaved caspase 3. In agreement with the tumour growth, setanaxib was able to reduce cell proliferation and increase apoptosis, while galunisertib only decreased apoptosis, but had no effect on cell proliferation (Fig. <a href="#Fig7" class="usa-link">7c</a>). Then, we evaluated the changes in the fibrotic stroma of the tumours by analysing α-SMA and Picro Sirius Red staining. As expected, setanaxib strongly reduced the amount of CAF and the collagen deposition compared with the vehicle, while galunisertib did not show significant changes (Fig. <a href="#Fig7" class="usa-link">7d</a>).</p>
<figure class="fig xbox font-sm" id="Fig7"><h4 class="obj_head">Fig. 7.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12358586_41392_2025_2347_Fig7_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f9cc/12358586/e25fa8e62be4/41392_2025_2347_Fig7_HTML.jpg" loading="lazy" id="d33e1784" height="775" width="790" alt="Fig. 7"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig7/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Therapeutic effect of dual NOX4/NOX1 inhibition with setanaxib on tumour progression in an intrahepatic cholangiocarcinoma (iCCA) PDX model. Tumour fragments from a previously characterized PDX (PDX153) were subcutaneously reimplanted into 6-week-old female NOD.CB-17-Prkdc scid/Rj. Once tumours reached an approximate volume of 150 mm<sup>3</sup>, mice started to be treated with galunisertib (150 mg/kg) or setanaxib (60 mg/kg). <strong>a</strong> Tumour growth was evaluated over 21 days in mice receiving vehicle, setanaxib or galunisertib (<em>n</em> = 6-7 animals per group). <strong>b</strong> Animals body weight was monitored through the experimental procedure time span to rule out potential toxic effects of any of the drugs. <strong>c, d</strong> Representative images of hematoxylin-eosin (H&amp;E) and immunohistochemistry for Ki67 and cleaved-Caspase 3 (cCap3) (<strong>c</strong>). Immunohistochemistry for α-SMA and Picro Sirius Red (<strong>d</strong>). Scale: 100 µm. Quantification of Ki67, cCasp3 and Picro Red Sirius positive areas are shown below. Values are expressed as means ± SEM. *<em>p</em> &lt; 0.05; **<em>p</em> &lt; 0.01; ***<em>p</em> &lt; 0.001; as compared with untreated mice</p></figcaption></figure><p id="Par18">After the promising results from the PDX model we proceeded to perform a similar experiment in the AKT-YAP iCCA model. In this case, we induced tumorigenesis and four weeks later, once the tumours were established, we treated the animals for two additional weeks before sacrifice (Supplementary Fig. <a href="#MOESM1" class="usa-link">31a</a>). Despite not being sacrificed at endpoint, when livers reached their peak in weight in this animal model,<sup><a href="#CR35" class="usa-link" aria-describedby="CR35">35</a></sup> liver to body weight ratio in the setanaxib group showed a reduction compared to vehicle and galunisertib (Supplementary Fig. <a href="#MOESM1" class="usa-link">31b</a>). Indeed, analysis of the livers showed a reduced number of tumours in the setanaxib group. In contrast, galunisertib had the opposite effect and increased the number of tumour nodules, corroborating our in vitro results (Fig. <a href="#Fig8" class="usa-link">8a, b</a>). Histopathological analysis of the tumours showed that vehicle and galunisertib-treated livers contained mostly undifferentiated iCCA with mainly trabecular growth within a desmoplastic stroma. Tumours showed expansive rather than infiltrative growth. On the contrary, setanaxib-treated livers showed smaller tumour nodules with pycnotic nuclei and the appearance of lower-grade lesions (Fig. <a href="#Fig8" class="usa-link">8c</a>). Accordingly, quantification of IHC showed a reduced percentage of Ki67 positive cells and increased cleaved caspase 3 staining in setanaxib group, while the opposite effects were observed in galunisertib-treated tumours (Fig. <a href="#Fig8" class="usa-link">8c</a>). Further staining of the TME showed similar disposition of CAF within the tumours in the three groups, although the staining was less intense in the setanaxib group. Accordingly, the setanaxib group showed reduced Picro Sirius Red staining compared to vehicle and galunisertib groups, indicating less collagen deposition by CAF (Fig. <a href="#Fig8" class="usa-link">8d</a>). Therefore, altogether, these results corroborated our previous in vitro observations. Moreover, it is known that ECM deposition by CAF, including different collagens, increases tumour stiffness and forms a barrier that impedes T cell access into the interior of the tumours, negatively impacting the efficacy of current immunotherapies.<sup><a href="#CR36" class="usa-link" aria-describedby="CR36">36</a></sup> In this sense, analysis of CD4-positive T cells showed that these cells are localised in the surroundings of iCCA tumours in this animal model. Interestingly, galunisertib increased the recruitment of CD4 T cells, but they remained excluded in the outer ring of the tumours, while setanaxib treatment increased the intratumoural presence of CD4 T cells (Fig. <a href="#Fig8" class="usa-link">8e</a>). Furthermore, treatment with setanaxib also reduced the high expression of the immune checkpoint (ICP) PD-L1 in the tumour margins (Fig. <a href="#Fig8" class="usa-link">8e</a>). No changes in other immune subpopulations or ICP analysed in this study were observed (Supplementary Fig. <a href="#MOESM1" class="usa-link">31c</a>). In an attempt to simulate the potential use of setanaxib in iCCA patients we stratified the patients from the <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE244807" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE244807</a> cohort based on the enrichment in the gene signature obtained in the RNAseq analysis from Supplementary Fig. <a href="#MOESM1" class="usa-link">22</a>, corresponding with the genes commonly altered in the three HSC cell lines after exposure to setanaxib. Interestingly, patients enriched in setanaxib signature show a prolonged survival (Fig. <a href="#Fig8" class="usa-link">8f</a>), reduced <em>CD274</em>/PD-L1 expression levels (Fig. <a href="#Fig8" class="usa-link">8g</a>) and an increase in activated CD4 infiltration (Fig. <a href="#Fig8" class="usa-link">8h</a>), accordingly with our results in mice treated with setanaxib (Fig. <a href="#Fig8" class="usa-link">8e</a>).</p>
<figure class="fig xbox font-sm" id="Fig8"><h4 class="obj_head">Fig. 8.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12358586_41392_2025_2347_Fig8_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f9cc/12358586/429e8619c5b4/41392_2025_2347_Fig8_HTML.jpg" loading="lazy" id="d33e1915" height="960" width="790" alt="Fig. 8"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig8/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Dual NOX4/NOX1 inhibition with setanaxib reduces tumour burden in vivo in the AKT/YAP mouse intrahepatic cholangiocarcinoma (iCCA) model. iCCA tumours were induced using the HTVI AKT-YAP model and treated with galunisertib (150 mg/kg) or setanaxib (60 mg/kg) as illustrated in Supplementary Fig. <a href="#MOESM1" class="usa-link">31a</a>. <strong>a</strong> Representative macroscopic images of livers and Cytokeratin-19 (CK19) immunohistochemistry (to mark CCA cells) from animals treated with galunisertib or the vehicle. Scale: 1 mm. <strong>b</strong> Quantification of number of nodules/area and maximal tumour size in livers from <strong>a</strong>. <strong>c</strong>–<strong>e</strong> Representative images (left) of hematoxylin-eosin (H&amp;E) and IHCs for Ki67 and cleaved-Caspase 3 (cCap3) (<strong>c</strong>), IHC for α-SMA and Picro Sirius Red (<strong>d</strong>) and IHCs for PD-L1 and CD4 (<strong>e</strong>) in liver sections from mice treated with vehicle, galunisertib and setanaxib. Scale: 100 µm. Quantification of Red Sirius and Ki67 positive areas and cCasp3, PD-L1 and CD4 positive cells are shown below. Magnifications of CD4 intratumoural staining are shown. Arrows indicate CD4 positive cells. Values are expressed as means ± SEM from 7 animals. *<em>p</em> &lt; 0.05; **<em>p</em> &lt; 0.01; ***<em>p</em> &lt; 0.001; ****<em>p</em> &lt; 0.0001; as compared with untreated mice. <strong>f</strong> Kaplan-Meier curve of 5-year overall survival of patients from <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE244807" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE244807</a> cohort (<em>n</em> = 246) stratified based on enrichment of the setanaxib gene signature showed in Supplementary Fig. <a href="#MOESM1" class="usa-link">22</a> (Cox Proportional Hazards model). <strong>g</strong>
<em>CD274</em>/PDL1 expression in patients from <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE244807" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE244807</a> cohort stratified as in <strong>f</strong>. <strong>h</strong> Activated CD4 infiltration in patients from <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE244807" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE244807</a> cohort stratified as in <strong>f</strong></p></figcaption></figure><p id="Par19">Altogether, our results show that setanaxib allows targeting TGF-β pro-tumorigenic actions mediated by CAF, reducing tumour growth and modulating the TME, and represents a new therapeutic approach against iCCA.</p></section></section><section id="Sec9"><h2 class="pmc_sec_title">Discussion</h2>
<p id="Par20">TGF-β signalling plays a central role in carcinogenesis by modulating key events both in tumour cells and the TME.<sup><a href="#CR8" class="usa-link" aria-describedby="CR8">8</a></sup> Therefore, to exploit the potential of TGF-β as a therapeutic target in iCCA, it is essential to understand the specific actions that TGF-β promotes in the different cells conforming the tumour. It has been widely reported that TGF-β can suppress or promote tumour progression depending on the stage of the disease. In general, TGF-β suppresses tumour growth at early stages, while acts as a tumour-promoting agent at later stages by inducing invasion and metastasis.<sup><a href="#CR8" class="usa-link" aria-describedby="CR8">8</a></sup> To benefit from the tumour-promoting functions of TGF-β, tumour cells must decouple the TGF-β signalling from apoptosis while maintaining a functional TGF-β pathway.<sup><a href="#CR8" class="usa-link" aria-describedby="CR8">8</a></sup> Our findings reveal important differences in the effects of TGF-β signalling activation in iCCA tumour cells compared to other cancers. In HCC, the most frequent liver cancer, tumour cells resembling later stages of tumour development can bypass TGF-β-induced apoptosis and cell cycle arrest and exploit TGF-β pro-tumorigenic functions while maintaining an active canonical TGF-β signalling.<sup><a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>–<a href="#CR24" class="usa-link" aria-describedby="CR24">24</a></sup> Our in vitro data shows that only iCCA cells with impaired SMAD2/3/4 signalling are able to escape TGF-β suppressor effects. On the contrary, cells with active SMAD2/3/4 undergo apoptosis and cell cycle arrest regardless of the stage of the tumour cell. We further validated this observation in organoids derived from metastatic iCCA samples, corresponding to the latest stage of tumour development. In other cancers, such as pancreatic and skin carcinomas, premalignant epithelial progenitors carrying mutations in <em>KRAS</em> or <em>HRAS</em> undergo apoptosis in response to TGF-β, and to escape this apoptosis, RAS-mutant cells acquire alterations that inactivate the TGF-β pathway or decouple TGF-β signalling from apoptosis.<sup><a href="#CR37" class="usa-link" aria-describedby="CR37">37</a>,<a href="#CR38" class="usa-link" aria-describedby="CR38">38</a></sup> This characteristic also differs in iCCA. Indeed, our in vitro studies analysing proliferation arrest and apoptosis were performed in two cell lines (HuCCT1 and RBE) that carry mutations in <em>KRAS</em>.<sup><a href="#CR39" class="usa-link" aria-describedby="CR39">39</a></sup></p>
<p id="Par21">The recent approval of immunotherapy for the treatment of iCCA patients marked the first significant change in the standard of care for the majority of iCCA patients in decades. However, suboptimal responses in a substantial percentage of patients have directed strategies towards combining immunotherapy with inhibitors of other major signalling pathways exerting immunosuppressive actions. In this context, TGF-β appeared to be an ideal candidate, but the latest clinical trial using TGF-β inhibitors was discontinued because it did not meet its primary end-point of improving overall survival.<sup><a href="#CR6" class="usa-link" aria-describedby="CR6">6</a></sup> The findings from our study may have important translational implications in this sense. Indeed, we demonstrated that TGF-β signalling inhibition using TGF-β receptor inhibitors promotes iCCA tumour cell growth. Even in mixed spheroids, the pro-proliferative effect of TGF-β inhibitors on iCCA cells prevails over the inhibitory effect on CAF, promoting spheroid growth. This, together with the low mutation rate in the members of the TGF-β canonical signalling pathway observed in iCCA patients, may counteract the positive effects of TGF-β inhibitors in boosting immunotherapies. Other tumours carrying a high percentage of mutations in key components of the canonical signalling, such as pancreatic cancer<sup><a href="#CR40" class="usa-link" aria-describedby="CR40">40</a></sup> or extrahepatic CCA,<sup><a href="#CR41" class="usa-link" aria-describedby="CR41">41</a></sup> in which <em>SMAD4</em> is one of the most mutated genes, could benefit from the use of these inhibitors. Nevertheless, further preclinical studies combining TGF-β inhibitors with immunotherapies in immunocompetent iCCA models must be conducted to clarify this issue.</p>
<p id="Par22">To bypass the undesired effects of TGF-β inhibitors on tumour cells, we decided to switch strategies and search for downstream mediators specific to TGF-β pro-tumorigenic actions, focusing our attention on NOX4, which is highly increased in iCCA compared to normal tissue. The role of NOX4 in the tumour cell has been controversial, being defined as pro-tumorigenic in some tumours,<sup><a href="#CR14" class="usa-link" aria-describedby="CR14">14</a></sup> and suppressor in others, including HCC.<sup><a href="#CR13" class="usa-link" aria-describedby="CR13">13</a></sup> Interestingly, our data revealed that NOX4 expression is restricted to CAF from the TME in iCCA, which allows its therapeutical inhibition without undesired side effects on tumour cells. However, while the role of NOX4 in favouring TGF-β-induced fibroblast transdifferentiation has been undisputed over the years in different pathologies, including several cancers,<sup><a href="#CR15" class="usa-link" aria-describedby="CR15">15</a>,<a href="#CR42" class="usa-link" aria-describedby="CR42">42</a>–<a href="#CR44" class="usa-link" aria-describedby="CR44">44</a></sup> a very recent study indicated that NOX4 is dispensable for skin myofibroblast differentiation and wound healing.<sup><a href="#CR16" class="usa-link" aria-describedby="CR16">16</a></sup> In that report, NOX4<sup>-/-</sup> mice did not display differences compared to WT mice in terms of wound healing, similar to the lack of impact of NOX4 invalidation on iCCA tumours in the SB1 syngeneic model found in our study. In agreement with this, primary fibroblasts isolated from mice or patients with NOX4 mutations were able to be activated after exposure to TGF-β, again in a similar way to our CRISPR-NOX4 hTERT cells. However, that report did not provide a mechanism by which fibroblasts can respond to TGF-β in the absence of NOX4, whereas we show here that NOX4 and NOX1 act in concert to allow TGF-β-induced transdifferentiation, at least in iCCA, and that dual inhibition is necessary to impair the aforementioned transdifferentiation. Thus, setanaxib could specifically target CAF functions and reduce iCCA growth both in vitro and in vivo. In this regard, our study presents some vulnerabilities. For instance, our inability to successfully produce a double NOX4/NOX1 CRISPR derived from hTERT-HSC prevented us from further validating this mechanism in a stable KO model. Whether the impossibility to generate this double CRISPR is due to technical reasons or to the lethality of depleting the expression of NOX4 and NOX1 simultaneously, is a question that remains unanswered. Another open question is why some studies found that depletion of NOX4 expression alone, either by shRNA techniques in vitro or in NOX4<sup>-/-</sup> mice, was able to impact fibroblast activation in other tumour types, such as prostate,<sup><a href="#CR45" class="usa-link" aria-describedby="CR45">45</a></sup> oesophageal<sup><a href="#CR15" class="usa-link" aria-describedby="CR15">15</a></sup> or breast cancers.<sup><a href="#CR46" class="usa-link" aria-describedby="CR46">46</a></sup> Perhaps the levels and/or function of NOX1 or the components of its regulatory complex (i.e. p22phox) are different in these organs and insufficient to trigger a compensatory mechanism after NOX4 depletion. Therefore, further investigation of these compensatory mechanisms is needed to untangle their true complexity.</p>
<p id="Par23">Importantly, setanaxib has recently reached the clinical stage. In a clinical trial conducted in patients with primary biliary cholangitis (PBC) (<a href="https://clinicaltrials.gov/ct2/show/NCT03226067" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">NCT03226067</a>), setanaxib showed evidence of potential anti-cholestatic and anti-fibrotic effects,<sup><a href="#CR33" class="usa-link" aria-describedby="CR33">33</a></sup> which led to further investigation of setanaxib activity in a new clinical trial in PBC patients with elevated liver stiffness (<a href="https://clinicaltrials.gov/ct2/show/NCT05014672" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">NCT05014672</a>). Furthermore, following the promising data demonstrating that setanaxib could overcome CD8 T exclusion in tumours from preclinical murine models,<sup><a href="#CR42" class="usa-link" aria-describedby="CR42">42</a></sup> setanaxib is also being tested in combination with immunotherapy in a Phase II clinical trial in patients with recurrent or metastatic squamous cell carcinoma of the head and neck (<a href="https://clinicaltrials.gov/ct2/show/NCT05323656" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">NCT05323656</a>). Interestingly, preliminary transcriptomic data from this trial showed that the two top pathways altered by setanaxib were the “Idiopathic Pulmonary Fibrosis Signalling Pathway” and the “Hepatic Fibrosis/Hepatic Stellate Cell Activation Pathway”, supporting the potential of setanaxib as an anti-fibrotic agent also in cancer. Indeed, we described here how setanaxib was able to reduce iCCA tumour growth as well as intratumoural fibrosis, correlating with the fact that NOX4 is mainly concentrated in myCAF, which produce ECM. Intriguingly, a proportion of NOX4 expression is found in apCAF, which have recently been involved in immunosuppression in pancreatic cancer.<sup><a href="#CR47" class="usa-link" aria-describedby="CR47">47</a></sup> In this sense, we also observed a reduction of PD-L1 expression concomitant with lower α-SMA staining at the border of iCCA tumours in the setanaxib group compared to the vehicle and galunisertib groups. A recent report described that PD-L1 is expressed in HSC and is inducible by TGF-β.<sup><a href="#CR48" class="usa-link" aria-describedby="CR48">48</a></sup> Furthermore, PD-L1 was able to control HSC activation by regulating TGF-β signalling, which impacted the ability of HSC to communicate with iCCA tumour cells via paracrine signalling, reducing tumour growth.<sup><a href="#CR48" class="usa-link" aria-describedby="CR48">48</a></sup> This suggests that the reduction of PD-L1 expression by setanaxib may impact tumour growth independently of its role in the immune system regulation. Nevertheless, we also observed increased CD4 T cell infiltration in the setanaxib group, which is probably facilitated by the reduction of intratumoural fibrosis, as previously shown.<sup><a href="#CR36" class="usa-link" aria-describedby="CR36">36</a></sup> Importantly, another recent study correlated intratumoural CD4 T cell infiltration with better survival in iCCA patients.<sup><a href="#CR49" class="usa-link" aria-describedby="CR49">49</a></sup> In this regard, our simulation of iCCA stratification based on the setanaxib signature obtained in vitro in HSC rendered similar results: reduced PD-L1 expression, increased CD4 infiltration and an improved survival in iCCA patients. Nevertheless, further studies are needed to better characterise the importance of this infiltration and the potential impact of the different subpopulations of CD4<sup>+</sup> T cells, which have been shown to play diverse roles and exert different effects on patient survival.<sup><a href="#CR50" class="usa-link" aria-describedby="CR50">50</a></sup> Indeed, in light of our results and the fact that immunotherapy is now the current standard of care for iCCA patients,<sup><a href="#CR3" class="usa-link" aria-describedby="CR3">3</a>–<a href="#CR5" class="usa-link" aria-describedby="CR5">5</a></sup> the next logical step in our investigations is to continue investigating the effects of setanaxib in combination with ICP inhibitors. However, given the reduction we observed in PD-L1 levels after treatment with setanaxib in the AKT-YAP model, anti-PD-L1 or anti-PD1 therapies may not be the best options, and a complementary analysis of the expression of other ICP in our model may be necessary to select the optimal therapy. In addition, it is important to consider that in combination with ICP, all iCCA patients still receive the chemotherapeutics gemcitabine and cisplatin, which may need to be added to the mixture in further experiments. Such an experiment, coupled with next-generation techniques of spatial transcriptomics could provide a more profound understanding of the dynamics of the TME under the effects of setanaxib and current iCCA therapies. If results are positive, this may lead to new clinical trials of setanaxib in iCCA patients. Altogether, the present evidence suggests a potentially bright future for dual NOX4/NOX1 inhibitors in clinics settings.</p>
<p id="Par24">In summary, our study provides compelling evidence of the suppressive function of TGF-β signalling on iCCA tumour cells, challenging the use of TGF-β receptor inhibitors. However, we have also demonstrated that inhibition of NOX4/NOX1, downstream effectors of TGF-β signalling, specifically impairs CAF functions and reduces CCA growth by modulating the TME (Fig. <a href="#Fig9" class="usa-link">9</a>). Further evaluation of NOX4/NOX1 inhibitors in combination with current therapies is needed to exploit the full potential of these molecules. Our study also suggests that a deeper understanding of TGF-β actions in both tumour cells and the TME across different cancer types is imperative to identify new therapeutic targets downstream of TGF-β receptors that allow the specific inhibition of TGF-β pro-tumorigenic effects.</p>
<figure class="fig xbox font-sm" id="Fig9"><h3 class="obj_head">Fig. 9.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12358586_41392_2025_2347_Fig9_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f9cc/12358586/0dbd2cede5c2/41392_2025_2347_Fig9_HTML.jpg" loading="lazy" id="d33e2074" height="235" width="790" alt="Fig. 9"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig9/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Impact of different strategies for TGF-β signalling inhibition on intrahepatic cholangiocarcinoma (iCCA) growth. In iCCA, TGF-β exerts suppressor effects on the tumoral cells but promotes cancer-associated fibroblasts (CAF) activation, contributing to the normal tumour growth (left panel). Inhibition of TGF-β signalling at the receptors level blunts TGF-β actions in both cell types, overall resulting in a promotion of tumour growth (middle panel). Inhibiting TGF-β signalling downstream using NOX4/NOX1 dual inhibitors blocks TGF-β mediated CAF activation while maintaining its suppressor effects on the tumour cells, therefore impairing tumour growth (right panel)</p></figcaption></figure></section><section id="Sec10"><h2 class="pmc_sec_title">Materials and methods</h2>
<p id="Par25">Further details on the experimental methods are available in the Supplementary Materials section online.</p>
<section id="Sec11"><h3 class="pmc_sec_title">Ethic approval statements</h3>
<p id="Par26">All experiments complied with the EU Directive 2010/63/UE for animal experiments and the corresponding institution’s guidelines. Procedures for the generation and treatment of the SB1 syngeneic and AKT-YAP and AKT-NICD hydrodynamic tail vein injection animal models were approved by the Ethical Committee for animal experimentation of IDIBELL and by the General Direction of Environment and Biodiversity, Government of Catalonia (#6292 and #4589). Procedures for the generation and treatment of PDX animal models were approved by the Ethical Committee for the Use of Experimental Animals at the Vall d’Hebron Institute of Research (VHIR) (number 40/21) and by the Catalan Government (number 11650).</p>
<p id="Par27">The human tissues used in this study were collected with the required informed consent in written from each patient and the approval of the Institutional Review Board (Comite Etico de Investigacion Clínica-CEIC, University Hospital of Bellvitge) (Ref. PR350/21). Patients’ written consent form and the study protocol conformed to the ethical guidelines of the 1975 Declaration of Helsinki. CAF isolated in Barcelona were from resections from Hospital Clinic de Barcelona, under the HCB/2021/0732 ethical approval from the Hospital Clinic ethics committee. In the case of CAF isolated in Paris, the Committee for Personal Protection of Nord Ouest I (CPP) gave a favourable opinion on 11 April 2024 (National number: 2019-A01679-48 / SI number: 19.01424.061615-MS03.1/Internal ref: APHP190473). Participants signed an informed consent during the inclusion visit.</p></section><section id="Sec12"><h3 class="pmc_sec_title">Reagents</h3>
<p id="Par28">TGF-β1 (#<a href="https://www.ncbi.nlm.nih.gov/nuccore/T70039" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">T70039</a>, Sigma-Aldrich, Saint Louis, MI, USA) was used at 2 ng/ml. Galunisertib (LY2157299) (S2230, Selleckchem, Houston, TX, USA) was used at 10 μM. Setanaxib (GKT137831) (A104916, AmBeed, Arlington Heights, IL, USA) was used at 40 μM. DPI (D2926 Sigma-Aldrich) was used at 2.5 μM.</p></section><section id="Sec13"><h3 class="pmc_sec_title">Cell culture</h3>
<p id="Par29">HuCCT1, SG231, CCLP1, and HuH28 cells, derived from human iCCA and TMNK1 derived from liver endothelial cells, were kindly provided by Dr. Laura Fouassier (CRSA, Paris, France). KKU-213A, KKU-213C and RBE cells, derived from human iCCA, were kindly provided by Dr. Gianluigi Giannelli (IRCCS “Saverio de Bellis”, Castellana Grotte, Italy). SB1, derived from murine iCCA, were kindly provided by Dr. Gregory J Gores and Dr. Sumera Ilyas (Mayo Clinic, Rochester, USA). LX2-HSC and hTERT-HSC cells derived from human hepatic stellate cells (HSC) were kindly provided by Dr. Lynda Aoudjehane (ICAN, Paris, France). HSC-GFP cells, human primary HSC transfected with GFP, were kindly provided by Dr. David G Mollevi (IDIBELL, Barcelona, Spain). All culture media and supplements were obtained from Gibco, Thermo Fisher Scientific (Waltham, MA, USA). THP-1 cells were purchased from the American Type Culture Collection (ATCC). HuCCT1, CCLP1, SG231, KKU-213A and KKU-213C cells were cultured in DMEM 1 g/L glucose (21885-025), 10 mmol/L HEPES (15630-056). HuH28 cells were cultured in MEM (41090-028). RBE and THP-1 cells were cultured in RPMI (61870-036) supplemented with HCO<sub>3</sub><sup>-</sup> (25080-060) 0.75%. SB1, LX2-HSC, hTERT-HSC and TMNK-1 cells were cultured in DMEM 4.5 g/l glucose (31966-021). HSC-GFP were cultured in DMEM/F-12 (10565018). All media were supplemented with antibiotics (100 UI/mL penicillin and 100 μg/mL streptomycin (15140-122)), antimycotic (0.25 mg/mL amphotericin B (15290018), and 10% FBS (10270-106). Cell lines were routinely screened for the presence of mycoplasma and authenticated for polymorphic markers to prevent cross-contamination. The medium was changed every three days and cells were maintained in a humidified atmosphere at 37 °C containing 5% CO<sub>2</sub>.</p></section><section id="Sec14"><h3 class="pmc_sec_title">NOX4 loss-of-function CRISPR-CAS9 cell model</h3>
<p id="Par30">To generate a pool of hTERT-HSC cells lacking NOX4 protein, we used the CRISPR-Cas9 system. Short-guide RNAs (gRNA) were designed to target the gene and then cloned into the pSpCas9(BB)-2A-puro vector (supplied by Addgene, Watertown, MA, USA), which encodes an RNA polymerase III promotor for the transcription of the guide, the Cas9 endonuclease, and a gene providing resistance to puromycin. hTERT-HSC cell lines were transfected for two hours with Lipofectamine® 2000 Reagent (Thermo Fisher Scientific). An empty vector without gRNA was used as the negative control. Puromycin was added for 48 h at 2 μg/mL for selection. The selected cells were then checked for mRNA and protein knockdown by RT-QPCR and immunoblot, respectively.</p></section><section id="Sec15"><h3 class="pmc_sec_title">RNA silencing</h3>
<p id="Par31"><em>NOX4</em> and <em>NOX1</em> expression was silenced by using the siRNAs listed in Supplementary Table <a href="#MOESM1" class="usa-link">1</a>. Transient transfections were performed with 100 nM siRNA using DharmaFECT 4 (Thermo Fisher Scientific). Cells were seeded in 12-well or 6-well plates and 24 h later they were transfected with the corresponding siRNA. 24 h after transfection cell culture media was changed to remove transfection complexes. Then, the appropriate protocol for each experiment was followed.</p></section><section id="Sec16"><h3 class="pmc_sec_title">Colony formation assay</h3>
<p id="Par32">Cells were plated in 6-well plates at a density of 500 cells/well and then they were cultured for 7-10 days. Colonies were fixed with 4% paraformaldehyde, stained with 1% crystal violet and counted. Only colonies with more than 50 cells were considered.</p></section><section id="Sec17"><h3 class="pmc_sec_title">Sphere formation assay</h3>
<p id="Par33">Spheroids were created using the hanging drop method. For all the experiments, cells were suspended, at a concentration of 300.000 cells/ml for tumour cells alone or 300.000 + 300.000 for mixed spheroids containing tumour cells and HSC-GFP in a proportion 1:1. Cells were suspended in medium with 0.24% of methylcellulose (M7027, Sigma-Aldrich) in the absence or presence of TGF-β1, galunisertib or setanaxib. Twenty-five µl drops were pipetted onto the lid of 100 mm dishes, that were inverted over dishes containing 5 ml of cell culture medium to prevent drying. After 4 days of incubation at 37 °C and 10% CO<sub>2</sub>, spheroids were transferred by pipetting onto a low-attachment 6-well culture plate (3471, Corning). Images of 10–20 spheroids per experiment were acquired with a NIKON Eclipse Ti2 microscope and spheroids size was measured with ImageJ software (National Institute of Health, USA).</p></section><section id="Sec18"><h3 class="pmc_sec_title">Experiments with CAF</h3>
<section id="Sec19"><h4 class="pmc_sec_title">CAF isolation</h4>
<p id="Par34">Human primary CAF were isolated by outgrowth from tumoral tissue obtained from iCCA resections. CAFs were maintained in DMEM Glutamax 10%FBS 1%PS and were used between passages 3 and 10.</p></section><section id="Sec20"><h4 class="pmc_sec_title">2D CAF experiment</h4>
<p id="Par35">Primary CAF derived from three different patients were used for this experiment. 50.000 cells were plated in a 12-well plate and were maintained in DMEM Glutamax 10%FBS 1% Penicillin/Streptomycin. When cells reached 70% confluence, they were starved using DMEM Glutamax 1%FBS 1% Penicillin/Streptomycin overnight and were treated with vehicle (DMSO), TGF-β1, setanaxib or the combination of both. 48 h later, cells were collected in lysis buffer (RIPA Buffer 1X (Sigma-Aldrich), Protease Inhibitor (Roche) and Phosphatase inhibitor (Roche)), for protein extraction.</p></section><section id="Sec21"><h4 class="pmc_sec_title">3D spheroids experiment</h4>
<p id="Par36">Spheroids were generated using human primary CAF (<em>n</em> = 1) and HuCCT1 cells using a 1:3 ratio. Briefly, CAF were stained using the Wheat Germ Agglutinin (WGA) Alexa Fluor 594 conjugate (Invitrogen), and HuCCT1 were stained with the Vybrant CFDA Cell Tracer (Invitrogen). A total of 20.000 cells were seeded in each well of a Corning Elplasia 96-well Round Bottom Ultra-Low Attachment plate, in DMEM Glutamax, 10%FBS and 1% Penicillin/Streptomycin, together with the correspondent treatment: vehicle (DMSO) or setanaxib. Plates were centrifuged at 1000 rpm for 1 minute and were placed in a cell culture incubator at 37 °C and 5% CO<sub>2</sub>. Pictures of the spheroids were acquired using a Nikon Eclipse Ti-S Inverted fluorescent microscopy on days 2, 4 and 7, and the area of the spheroids was measured using Image J software.</p></section></section><section id="Sec22"><h3 class="pmc_sec_title">CCA organoid ex vivo three-dimensional cultures</h3>
<p id="Par37">CCA patient-derived tumour cells were isolated from CCA_PDX through a combination of mechanic disruption and enzymatic disaggregation as previously described.<sup><a href="#CR51" class="usa-link" aria-describedby="CR51">51</a></sup> Briefly, PDX tumours with a volume of less than 500 mm<sup>3</sup> were freshly collected in DMEM/F12/HEPES (L0093-500, Biowest, Nuaillé, France) after surgical resection, minced using sterile scalpels, and dissociated for a maximum of 90 minutes in DMEM/F12/HEPES supplemented with 0.3 mg/mL collagenase (C9891, Sigma-Aldrich), 0.1 mg/mL hyaluronidase (H3506, Sigma-Aldrich), 2% BSA (VWRC0332, VWR, Radnor, PA, USA), 5 mg/mL insulin (I1882, Sigma-Aldrich), and 50 mg/mL gentamycin (15750-037, Gibco, Thermo Fisher Scientific). After centrifugation, pellets were further dissociated using 0.05% trypsin (HYCLSH30236.02, VWR), 5 mg/mL Dispase (7923, STEMCELL Technologies, Vancouver, Canada) and 1 mg/mL DNase (D4263, Sigma-Aldrich). Red blood cells were eliminated by washing the cell pellet with 1x Red Blood Cell Lysis Buffer solution (00-4333-57, eBioscience, San Diego, CA, USA).</p>
<p id="Par38">For organoid generation, mouse stromal cell depletion was conducted using a commercial kit (Miltenyi Biotec, 130-104-694). The eluent containing human epithelial tumor cells fraction was resuspended in Matrigel Basement Membrane Matrix (Matrigel; #354277, Corning Corning, NY, USA) and seeded into pre-heated 6-well culture plates (140675, Thermo Fisher Scientific). As a standard, drops of 25 μL of the mixture were seeded into culture plates and incubated for 30 min at 37 °C without medium to allow the matrix solidification. Organoids were cultured in advanced DMEM/F12 medium supplemented with 1% penicillin/streptomycin, 10 mM HEPES, 1% Glutamax, 30% Wnt conditioned medium (from L1Wnt3A cell line), 15% RSPO1 conditioned medium (from 293tRSPO1mCh cell line) and 7.5% Noggin conditioned medium (from 293tnogginmCh cell line). The basal organoid medium was further supplemented with 1 x B27 supplement (17504044, Gibco, Thermo Fisher Scientific), 1x N2 supplement (17502048, Gibco, Thermo Fisher Scientific), 50 ng/ml human EGF (E9644, Sigma-Aldrich), 10 nM Gastrin I (Sigma G9145), 125 µg/ml Normocin (ant-nr-05, Invivogen, Toulouse, France), 2.5 µg/ml Plasmocin (ant-mpt-1, Invivogen). Four organoids were used in this study: PDXO85 (derived from liver metastasis), PDXO133 (derived from liver metastasis), PDXO153 (derived from bone metastasis) and PDXO162 (derived from peritoneum metastasis).</p>
<p id="Par39">For conducting efficacy drug tests, organoids were dissociated to generate a single-cell suspension and embedded in Matrigel (CLS356234) in order to seed at a density of 2 × 10<sup>4</sup> cells per well in pre-heated Corning 96-Well White Polystyrene Microplates (CLS3610, Corning). Following organoid generation (24-48 h later) cells were treated with either vehicle (DMSO) or the corresponding drug and cultured at 37 °C in 5% of CO<sub>2</sub> for 4 days. Medium and treatments were refreshed every 2–3 days. Cell viability was measured by CellTiter-Glo assay. Matrigel was melted by direct on-plate incubation with 50 μL PBS with1 mmol/L EDTA for 1 hour at 4 °C protected from light. Cell viability was quantified using CellTiter-Glo Luminescent Cell Viability Assay (Promega; G7570), according to the manufacturer’s instructions. Luminescence was measured with infinite M2000 Pro (Tecan) and i-control 1.11 software.</p></section><section id="Sec23"><h3 class="pmc_sec_title">Gene expression profiling</h3>
<p id="Par40">Gene expression datasets established from laser microdissected<sup><a href="#CR52" class="usa-link" aria-describedby="CR52">52</a></sup> CCA tumours were previously described. Freshly frozen tumour samples were obtained through the French liver cancer biobanks network – INCa (BB-0033-00085).</p></section><section id="Sec24"><h3 class="pmc_sec_title">Comparison of mRNA expression from public databases</h3>
<section id="Sec25"><h4 class="pmc_sec_title">TCGA cohort</h4>
<p id="Par41">The mRNA expression levels of <em>KRT19</em>, <em>ACTA2</em>, <em>COL1A1</em>, <em>TGFB1</em>, <em>TGFB2</em>, <em>TGFB3</em>, <em>TGFBR1</em>, <em>TGFBR2, TGFBR3 and NOX4</em> in non-tumoral tissue and CCA tumours from the repository of The Cancer Genome Atlas (TCGA) were analysed through the GEPIA database (<a href="http://gepia.cancer-pku.cn/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://gepia.cancer-pku.cn/</a>)<sup><a href="#CR53" class="usa-link" aria-describedby="CR53">53</a></sup> and were represented as means ± SEM. The mRNA expression levels of <em>TGFB1 versus SMAD7</em> and <em>NOX4</em>
<em>versus ACTA2, COL1A1</em>, <em>LOXL1</em> and <em>LOXL2</em> in CCA tumours from the repository of TCGA were analysed through the cBioPortal database (<a href="https://www.cbioportal.org/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.cbioportal.org/</a>) and represented as Spearman correlation coefficients (r) and <em>p</em>-values to assess the significance.<sup><a href="#CR54" class="usa-link" aria-describedby="CR54">54</a>,<a href="#CR55" class="usa-link" aria-describedby="CR55">55</a></sup></p></section><section id="Sec26"><h4 class="pmc_sec_title">TIGER-LC cohort</h4>
<p id="Par42">Raw microarray data was downloaded from Gene Expression Omnibus (accession ID: <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE76297" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE76297</a>) (TIGER-LC cohort) and processed using robust multiarray method implemented in the affy package version 1.56 available through the Bioconductor software project (<a href="https://bioconductor.org/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://bioconductor.org</a>) using R v4.0.4.<sup><a href="#CR56" class="usa-link" aria-describedby="CR56">56</a></sup> Probe-set to gene mapping was done using the array annotation file available at Gemma database (<a href="https://gemma.msl.ubc.ca/arrays/showArrayDesign.html?id=785" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://gemma.msl.ubc.ca/arrays/showArrayDesign.html?id=785</a>), selecting the most expressed probe as representative of gene expression when multiple mapping probes occurred to avoid duplicated genes. Comparison of median gene expression of <em>TGFB1</em>, <em>TGFB2</em>, <em>TGFB3</em>, <em>TGFBR1</em>, <em>TGFBR2</em>, <em>TGFBR3</em> and <em>NOX4</em> between tumour and non-tumour samples was done with Mann–Whitney U test and Spearman correlation analysis was performed to assess the correlation of gene expression between <em>TGFB1</em> and <em>SMAD7</em>.</p></section><section id="Sec27"><h4 class="pmc_sec_title">Single-cell RNA-seq</h4>
<p id="Par43">The single-cell RNA-seq dataset was processed, analysed and visualized using the Trailmaker® (<a href="https://scp.biomage.net/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://scp.biomage.net/</a>) hosted by Biomage (<a href="https://biomage.net/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://biomage.net/</a>). First, pre-filtered count matrices from Shi et al. (<a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE201425" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE201425</a>) were uploaded to Trailmaker®. Briefly, to pre-process the data, barcodes with low unique molecular identifiers (UMIs) and or high mitochondrial reads were filtered out. Next, to exclude outliers, a robust linear model was fitted to all samples. Lastly, barcodes with high doublet scores were excluded from the analysis, resulting in high quality barcodes that were used in subsequent integration steps. Briefly, to integrate the barcodes, data were log-normalised and the top 2000 highly variable genes were selected. Next, principal component analysis (PCA) was performed, and the top 30 principal components, explaining 89.67% of the total variance, were used to perform sample batch correction with the integrated Harmony R package. Louvain method of clustering was employed. To reduce dimensionality, data was visualised using t-distributed stochastic neighbour embedding (t-SNE). To identify cluster-specific marker genes, the marker genes of each cluster were compared to all other clusters and appropriately annotated. The CAF subset identified from this analysis was further examined via the sample subset feature. Filtering steps were disabled (due to pre-filtered data) and the subset was analysed via the same integration steps outlined above. Signatures of upregulated genes in CAF subtypes were identified (adj. <em>p</em>-value &lt; 0.05, expression of the gene in at least 1/3 of the cells, and average log2FC &gt; 1) in the scRNA-seq data using Seurat. These gene-expression signatures were then deconvoluted in the bulk RNA-seq experiments using Gene Set Variation Analysis (GSVA),<sup><a href="#CR57" class="usa-link" aria-describedby="CR57">57</a></sup> which provides the relative enrichment of a gene-expression signature for each sample, ranging from -1 to 1, where higher values are associated with more relative enrichment in a specific sample compared to the others.</p></section><section id="Sec28"><h4 class="pmc_sec_title">Survival analyses</h4>
<p id="Par44">Survival analyses were conducted in the <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE244807" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE244807</a> dataset, which had available gene expression and overall survival data for 246 patients with intrahepatic cholangiocarcinoma (iCCA).<sup><a href="#CR58" class="usa-link" aria-describedby="CR58">58</a></sup> Kaplan-Meier curves were plotted, and hazard ratios and 95% confidence intervals were obtained from a Cox Proportional Hazards model adjusted for sex and age clinical covariates. Gene set variation analysis (GSVA) was used to assess the relative activation of the setanaxib treatment signature in the samples.<sup><a href="#CR57" class="usa-link" aria-describedby="CR57">57</a></sup> Median GSVA scores, in the case of signatures, or median gene expression value, in the case of individual genes (i.e. NOX1, NOX4), were used to stratify patients in low or high group, respectively. All analyses were performed using R 4.0.4.</p></section><section id="Sec29"><h4 class="pmc_sec_title">Active CD4 T cells infiltration evaluation</h4>
<p id="Par45">Active CD4 T cell signatures were obtained from a previous study.<sup><a href="#CR59" class="usa-link" aria-describedby="CR59">59</a></sup> GSVA was used to assess the relative infiltration of CD4 T cells in the samples.<sup><a href="#CR57" class="usa-link" aria-describedby="CR57">57</a></sup> Differences in median CD4 T cell infiltration scores between setanaxib signature low and setanaxib signature high groups were tested using a Mann- Whitney U test. <em>P</em>-values were corrected for multiple testing with False Discovery Rate (FDR).</p></section></section><section id="Sec30"><h3 class="pmc_sec_title">RNA-seq analysis</h3>
<section id="Sec31"><h4 class="pmc_sec_title">Short-Read RNA Sequencing</h4>
<p id="Par46">Total RNA quantification was performed using the Qubit® RNA BR Assay kit (Thermo Fisher Scientific), and the RNA integrity was assessed using the RNA 6000 Nano Bioanalyzer 2100 Assay (Agilent Technologies, Santa Clara, CA, USA). To prepare the RNASeq libraries, the KAPA Stranded mRNA-Seq Illumina Platforms Kit (Roche, Basel, Switzeland) was used, following the manufacturer’s recommendations with 500 ng of total RNA as the input material. Library quality was assessed on an Agilent 2100 Bioanalyzer using the DNA 7500 assay. The libraries were sequenced on the NovaSeq 6000 (Illumina, San Diego, CA, USA) with a read length of 2 × 151 bp, in accordance with the manufacturer’s protocol for dual indexing. Image analysis, base calling, and quality scoring of the run were executed using the manufacturer’s Real Time Analysis (RTA 3.4.4) software.</p></section><section id="Sec32"><h4 class="pmc_sec_title">RNA-seq data processing and analysis</h4>
<p id="Par47">RNA-seq reads were trimmed using Trim Galore (<a href="https://github.com/FelixKrueger/TrimGalore" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/FelixKrueger/TrimGalore</a>), setting the length parameter at 35 and the stringency at 10. Trimmed reads were mapped against the human reference genome (GRCh38) using STAR aligner version 2.7.8a<sup><a href="#CR60" class="usa-link" aria-describedby="CR60">60</a></sup> with ENCODE parameters, and annotated genes were quantified with RSEM version 1.3.0<sup><a href="#CR61" class="usa-link" aria-describedby="CR61">61</a></sup> using GENCODE v42 as reference. Differential expression analysis was performed with limma v3.4.2 R package, using TMM normalisation. The ‘voom’ function<sup><a href="#CR62" class="usa-link" aria-describedby="CR62">62</a></sup> was used to estimate the mean-variance relationship and to compute observation-level weights. The linear model was fitted for each cell line with the corresponding voom-transformed counts, and contrasts were extracted. Genes were considered differentially expressed (DEG) with a <em>p</em>-value adjusted below 0.05 and subsets of DEG were represented in heatmaps with the pheatmap R package, using voom-transformed counts scaled by row. A gene set enrichment analysis (GSEA) was performed on each list of genes pre-ranked by the limma moderated t-statistic, with the fgsea v1.12 R package,<sup><a href="#CR63" class="usa-link" aria-describedby="CR63">63</a></sup> against the Reactome database.</p></section></section><section id="Sec33"><h3 class="pmc_sec_title">RNA and reverse transcription-PCR</h3>
<p id="Par48">E.Z.N.A.® Total RNA Kit II (Omega bio-tek, Norcross, GA, USA) was used for total RNA isolation following the manufacturer’s instructions. Culture plates were washed with PBS and cells were detached with Trypsin and pelleted by centrifugation. Pellets were dissolved in RLT lysis buffer containing 10 μL/mL of β-mercaptoethanol. 1 μg of total RNA isolated from each sample was reverse-transcribed with random primers for complementary DNA synthesis, using a High Capacity RNA to cDNA Master Mix Kit (Applied Biosystems, Foster City, CA, USA), according to the manufacturer’s protocol. For the real-time qPCR, expression levels were determined in duplicate in a LightCycler® 480 Real-time PCR system, using the LightCycler® 480 SYBR Green I Master (Roche Diagnostics GmbH, Mannheim, Germany). Primer sequences are provided in Supplementary Tables <a href="#MOESM1" class="usa-link">2</a> and <a href="#MOESM1" class="usa-link">3</a>. Gene expression was normalised to <em>GAPDH</em> mRNA content for human genes and to <em>Rpl32</em> mRNA content for mouse genes and was expressed relatively to the control condition of each experiment. The relative expression of each target gene was determined from replicate samples using the formula 2<sup>-ΔΔCt</sup>.</p></section><section id="Sec34"><h3 class="pmc_sec_title">Immunoblot analysis</h3>
<p id="Par49">For obtaining whole-cell lysates for immunoblotting, cell cultures were lysed in RIPA buffer (Sigma-Aldrich) supplemented with 1 mmol/L orthovanadate and a cocktail of protease inhibitors. Proteins were quantified using a Pierce BCA kit (Thermo Fisher Scientific). Western blot analysis was performed as previously described.<sup><a href="#CR64" class="usa-link" aria-describedby="CR64">64</a></sup> Primary antibodies are provided in Supplementary Table <a href="#MOESM1" class="usa-link">4</a>.</p>
<section id="Sec35"><h4 class="pmc_sec_title">Amplex red</h4>
<p id="Par50">Extracellular hydrogen peroxide levels were detected using an Amplex® Red assay kit (A36006, Invitrogen, Thermo Fisher Scientific). Cells were seeded on 12-well plates in DMEM with 10% FBS. The next day, cells were deprived of serum for 24 h in DMEM without phenol red (21063-029) at 0.5% FBS and cultured with the corresponding treatment for 24 h. Then, the media was refreshed containing Amplex® Red reagent (10 μM) and horseradish peroxidase (0.1 U/mL). After 2 h incubation at 37 °C, fluorescence was measured in duplicate in a BMG FLUOstar OPTIMA fluorescence plate reader (BMG Labtech, Ortenberg, Germany) with excitation and emission at 530 nm and 590 nm, respectively. H<sub>2</sub>O<sub>2</sub> concentration was extrapolated using a standard curve and values were normalised to cell viability by crystal violet staining (61135, Sigma-Aldrich).</p></section></section><section id="Sec36"><h3 class="pmc_sec_title">Immunohistochemical analysis</h3>
<section id="Sec37"><h4 class="pmc_sec_title">Tumour samples</h4>
<p id="Par51">Sections of 5 μm in thickness were used for subsequent assays. For CK19 IHC staining, antigen retrieval was achieved in TE buffer (pH 8.0), while sodium citrate buffer (pH 6.0) was used for all other targets. Slides were submerged in retrieval buffer and heated in a microwave on a high level for 10 minutes. After cooling down, slices were blocked with goat serum and the Avidin-Biotin blocking kit (Vector Laboratories, Burlingame, CA). Specimens were incubated with the designated primary antibody at 4 °C overnight. Detailed information on the antibodies used is listed in Supplementary Table <a href="#MOESM1" class="usa-link">4</a>. Subsequently, 3% hydrogen peroxide was applied for 10 min to quench endogenous peroxidase activity. Slices were incubated in the biotin-conjugated secondary antibody (Life Technology, Waltham, MA) at a 1:500 dilution for 30 min at room temperature. The Vectastain Elite ABC Kit (Vector Laboratories) and DAB substrates (Dako North America, Carpinteria, CA) were applied to visualise the immunostainings. Naphthol-AS-D-Chloracetatesterase (CLAE) histochemistry was performed according to standard diagnostic practice.</p>
<p id="Par52">For analysis of collagen deposition, 5 μm-cut paraffin-embedded tissue sections were dewaxed and rehydrated in an Autostainer XL (Leica Biosystems, Deer Park, IL, USA) and stained with Picro Sirius Red (Picric acid 197378; Direct-Red 80 365548, Sigma-Aldrich).</p>
<p id="Par53">Stained sections were scanned on a virtual slide scanner Pannoramic 1000 Flash RX® (Sysmex Europe SE, Norderstedt, Germany) and images were analysed blindly by a pathologist, AS, and two investigators EG-S and JV. Quantification of positive cells or positive-stained areas was performed using Slideviewer and ImageJ analysis software (National Institutes of Health, Bethesda, MD, USA) respectively. Quantitative analyses were conducted in at least 10 tumours per mouse. In the case of intratumoural CD4-positive cells, only the inner area of the tumour was considered.</p></section><section id="Sec38"><h4 class="pmc_sec_title">PDXO samples</h4>
<p id="Par54">Organoid samples were analysed by Next Generation IHC (NGI) using the tumor-Panel to assess. Briefly, Organoids were harvested from culture and processed for paraffin embedding. The organoid suspension was centrifuged at 300 x g for 5 minutes at 4 °C. The pellet was washed twice with ice-cold PBS and fixed in 4% paraformaldehyde for 60 minutes at 4 °C. Post-fixation, the sample was centrifuged (300 x g, 5 minutes, 4 °C) and washed once with PBS. For agarose embedding, the organoid pellet was resuspended in 100 μL of the 2% agarose solution. The agarose was allowed to solidify completely at 4 °C for approximately 30 minutes. The agarose block was dehydrated and embedded in FFPE. Pan-cytokeratin (pan-CK) was included as a marker for iCCA tumor cells, Vimentin (Vim) for the detection of murine stromal cells and Ki67 staining was included as a marker of cell proliferation. The NGI protocol consists of iterative cycles of staining/destaining on the same tissue section and uses the combination of Ventana Discovery Ultra (Roche Diagnostics), Nanozoomer slide scanner (Hamamatsu) and Visiopharm image analysis software. Briefly, an alcohol soluble chromogen (DISCOVERY AEC KIT (#760-258, Roche-Ventana)) was used to allow the destaining of the samples. After each automated IHC, samples were mounted in aqueous medium and digitalized (cycle 1). Subsequently, the section was destained in alcohol and submitted to the staining cycle including all the markers. Positive controls were included in each slide. Antibodies and protocols used in this study are listed in Supplementary Table <a href="#MOESM1" class="usa-link">4</a>.</p></section></section><section id="Sec39"><h3 class="pmc_sec_title">CCA murine models</h3>
<section id="Sec40"><h4 class="pmc_sec_title">Syngeneic, orthotopic mouse model of CCA</h4>
<p id="Par55">NOX4<sup>−/−</sup> (B6.129-Nox4tm1Kkr/J) mice, generated in Dr. Krause’s Laboratory,<sup><a href="#CR65" class="usa-link" aria-describedby="CR65">65</a></sup> were obtained from Jackson Laboratories (together with C57BL/6 J mice, the appropriated controls, as suggested by the provider) and housed at IDIBELL (Barcelona, Spain). Murine SB1 iCCA cells derived from an oncogene-driven murine model of iCCA were maintained in the culture medium, as previously described.<sup><a href="#CR18" class="usa-link" aria-describedby="CR18">18</a></sup> Mice were anesthetised using 1.5% to 3% isoflurane (IsoFlo 250 ml, Zoetis). Under deep anaesthesia, the abdominal cavity was opened by a 1 cm incision below the xiphoid process. A sterile cotton-tipped applicator was used to expose the superolateral aspect of the medial lobe of the liver. Using a 27-gauge needle, 40 μL of standard media containing 0.5 × 10<sup>6</sup> SB1 cells were injected into the lateral aspect of the medial lobe. A cotton-tipped applicator was held over the injection site to prevent cell leakage and blood loss. Subsequently, the abdominal wall and skin were closed in separate layers with absorbable chromic 3-0 gut suture material. Four weeks following SB1 cell implantation, mice were sacrificed, and tumour and adjacent liver were collected for the different analyses.</p></section><section id="Sec41"><h4 class="pmc_sec_title">Hydrodynamic Tail Vein Injection model of CCA</h4>
<p id="Par56">WT male FVB/N mice were purchased from Charles River (Charles River Laboratories Les Oncins, France). The hydrodynamic injection was performed as described previously.<sup><a href="#CR19" class="usa-link" aria-describedby="CR19">19</a></sup> In brief, 20 μg pT3-EF1α-HA-myr-AKT and 20 μg pT3-EF1α-YapS127A (AKT-YAP model) or 20 μg pT3-EF1α-HA-myr-AKT and 20 μg pT3-EF1α-NICD1 (AKT-NICD model), with 2 μg pCMV/sleeping beauty transposase (SB) were diluted in 2 mL of saline (0.9% NaCl), filtered through 0.22-μm filter, and injected into the lateral tail vein of 6- to 8-week-old mice in 5 to 7 seconds. Four weeks after the injection, when the tumours were developed, 150 mg/kg of galunisertib, 60 mg/kg of setanaxib or the vehicle (1% (w/v) carboxymethylcellulose (#C9481 Sigma-Aldrich, 0.25% (v/v) Tween80 (#59924 Sigma-Aldrich), and 0.05% (v/v) Antifoam (#59920 C Sigma-Aldrich,) in purified water) were given to the mice daily during two weeks prior to sacrifice for the obtention of tissue samples.</p></section><section id="Sec42"><h4 class="pmc_sec_title">PDX generation and treatment</h4>
<p id="Par57">Human PDX153 was generated by subcutaneous implantation of a metastatic liver biopsy from a patient diagnosed with intrahepatic CCA. PDX153 tumour pieces (~1-2mm<sup>3</sup>) were subcutaneously implanted into the right and left flanks of 6- to 8-week-old female NOD.CB-17-Prkdc scid/Rj mice (Janvier Labs, RRID:MGI:3760616). Animals were housed in air-filtered flow cages with a 12:12 light/dark cycle, and food and water were provided ad libitum.</p>
<p id="Par58">Upon xenograft growth (50-150 mm<sup>3</sup>), PDX153-bearing mice were randomized into 3 groups for treatment with vehicle, setanaxib 60 mg/kg, or galunisertib 150 mg/kg daily by oral gavage for 22 days. All drugs were dissolved in vehicle [1% (w/v) carboxymethylcellulose (#C9481 Sigma-Aldrich, 0.25% (v/v) Tween80 (#59924 Sigma-Aldrich), and 0.05% (v/v) Antifoam (#59920 C Sigma-Aldrich]. Tumour growth was measured twice per week with a calliper; investigators were blinded to treatment effect. Mice weights were recorded two times per week. Tumour volumes were calculated using the formula: V = (length x width<sup>2</sup>)/2. Mice were euthanized by CO<sub>2</sub> inhalation when tumours reached 1-1.5 cm<sup>3</sup> or severe weight loss occurred, according to institutional guidelines. PDX tumours were harvested at the end of the experiment.</p>
<p id="Par59">Animals were hosted under 12 h light/dark cycles with free access to food and water.</p></section></section><section id="Sec43"><h3 class="pmc_sec_title">CCA specimens</h3>
<p id="Par60">Samples from tumour tissues and surrounding non-tumoral liver were obtained from patients during surgical procedures (transplantation or resection) at the Bellvitge University Hospital (HUB) (Supplementary Table <a href="#MOESM1" class="usa-link">5</a>).</p></section><section id="Sec44"><h3 class="pmc_sec_title">Statistical analysis</h3>
<p id="Par61">Results were analysed using the GraphPad Prism 5.0 statistical software (GraphPad Software, Boston, MA, USA). Data are shown as means ± standard error of the mean (SEM). For comparisons between two groups, parametric Student t tests or nonparametric Mann–Whitney tests were used.</p></section></section><section id="Sec45"><h2 class="pmc_sec_title">Supplementary information</h2>
<section class="sm xbox font-sm" id="MOESM1"><div class="media p"><div class="caption">
<a href="/articles/instance/12358586/bin/41392_2025_2347_MOESM1_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material</a><sup> (20MB, docx) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>This work was supported by (i) Spanish Association for Cancer Research (AECC), Spain, grant #PRYGN211279FABR, to I.F.; (ii) Agencia Estatal de Investigación (AEI), Ministry of Science and Innovation (MICIN), Spain, cofounded by FEDER funds/Development Fund—a way to build Europe, grant numbers #RTI2018-094079-B-100, #RTC2019-007125-1, to I.F.; #PID2019-108651RJ-I00, PID2022-141984OB-I00 and RYC2021-034121-I, to J.V. J.V. was funded by AEI, MICIN, through the Retos Investigación grant number PID2019-108651RJ-I00/DOI 10.13039/501100011033 and “Ramon y Cajal” program RYC2021-034121-I. The CIBEREHD, National Biomedical Research Institute on Liver and Gastrointestinal Diseases, is funded by the Instituto de Salud Carlos III, Spain. We thank the CERCA Programme/Generalitat de Catalunya for institutional support. Institutional support to CNAG was provided by the Spanish Ministry of Science and Innovation through the Instituto de Salud Carlos III, and by the Generalitat de Catalunya through the Departament de Salut and the Departament de Recerca i Universitats. C.C. is supported by Inserm, Université de Rennes, Ligue Contre le Cancer (R23037NN), INCa (EU TRANSCAN23-002-2023-129, INCa_18688, R21251ND), ITMO Cancer of AVIESAN within the framework of the 2021-2030 Cancer Control Strategy, on funds administered by Inserm (C18007NS, C20013NS, C20014NS), the French Ministry of Health and the French National Cancer Institute (PRT-K20-136). T.V.T. was supported by grants from the Spanish Ministry of Science, Innovation and University (PID2019-108008RJ-I00, PID2023-146827OB-I00), Fundació La Marató TV3 (201931-31), and “Ramon y Cajal” program (RYC2020-029098-I). L.F. received financial support from the “Ligue contre le cancer” (RS24/75-62) and from ITMO Cancer of Aviesan on funds administered by Inserm (C21044DS/ ASC21044DSA). S.A. is supported by PID2021-124694OA-I00, MCIN/AEI/ 10.13039/501100011033 and FEDER Una manera de hacer Europa, The European Union grant agreement 101077312* and Ramon y Cajal program RYC2022-036321-I. * Views and opinions expressed are however those of the author(s) only and do not necessarily reflect those of the European Union. Neither the European Union nor the granting authority can be held responsible for them.” The authors are extremely thankful to Dr. James Doroshow and his team for the wonderful antibodies and tools that allowed NOX4 and NOX1 detection by IHC. We also acknowledge the Scientific and Technical Services of IDIBELL, in particular the Animal Facility led by Rosa Bonavia and the Histology Platform led by Lola Mulero with Jose Llamas and Mireia Torres as specialists. We also thank the common services from CIC-IBMCC, in particular the Microscopy Unit coordinated by Ana Isabel Garcia and the Molecular Pathology service led by Maria del Carmen Garcia Macias. We thank Profs Dominique Wendum and Magali Svrcek from the Pathology Department and Prof François Paye from Digestive Surgical Surgery Departments of Saint- Antoine Hospital, Paris, France. ML and LF acknowledge Assistance Publique – Hôpitaux de Paris (Direction de la Recherche Clinique et de l’Innovation), France. The authors want to acknowledge Ignacio Negro, Isabel Vaquero and Dr Aranzazu Sanchez and Dr Blanca Herrera, for their unvaluable help with the logistics of the manuscript revision. This article is based upon work from the COST Action CA22125 Precision medicine in biliary tract cancer (Precision-BTC-Network) supported by COST (European Cooperation in Science and Technology: <a href="http://www.cost.eu" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">www.cost.eu</a>), in collaboration with the European Network for the Study of Cholangiocarcinoma (ENS-CCA: <a href="http://www.enscca.org/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://www.enscca.org/</a>).</p></section><section id="notes1"><h2 class="pmc_sec_title">Author contributions</h2>
<p>J.V. conceived the project. J.V. and I.F. provided funding and supervision. J.V., I.F., E.G.S. and J.A. designed the experiments. J.V., E.G.S., J.A., M.Y.B., L.S.V., A.R., C.G., S.H.P., B.M.M., A.C.C., R.E.S. M.V., C.C. and A.S. acquired the data. N.P.F., A.A., M.G., A.E.C., E.R., T.S., M.C., B.L., J.F., G.P., A.M.L., S.D., S.I.I., L.A., M.L., L.F., S.A., D.C. and T.V.T. provided essential reagents, samples and tools. J.V., I.F., E.G.S. and J.A. analysed and interpreted the data. All authors critically revised the article. All authors have read and approved the article.</p></section><section id="notes2"><h2 class="pmc_sec_title">Data availability</h2>
<p>RNA-seq datasets generated in this study are available at Gene Expression Omnibus (GEO) under the accession number <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE274024" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE274024</a>. The rest of the data produced in this study can be found in the article and its supplementary data files. The publicly available data used in this study, which were generated by other researchers, were sourced from TCGA-CHOL and GEO databases under the accession numbers <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE76297" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE76297</a>, <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE244807" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE244807</a> and <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE201425" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE201425</a>.</p></section><section id="FPar1"><h2 class="pmc_sec_title">Competing interests</h2>
<p id="Par62">T.V.T. received research grants from AstraZeneca, Lilly Oncology, Servier, Alentis, and Incyte, which are irrelevant to the current study. Personal fee from AstraZeneca and Incyte. Remaining authors have no conflicts of interest to declare related to this manuscript.</p></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm">
<div class="fn p" id="fn1"><p><strong>Publisher’s note</strong> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></div>
<div class="fn p" id="fn2"><p>These authors contributed equally: Josep Amengual, Ester Gonzalez-Sanchez</p></div>
<div class="fn p" id="fn3"><p>These authors jointly supervised this work: Isabel Fabregat, Javier Vaquero</p></div>
</div></section><section id="sec46"><h2 class="pmc_sec_title">Supplementary information</h2>
<p>The online version contains supplementary material available at 10.1038/s41392-025-02347-z.</p></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Banales, J. M. et al. Expert consensus document: Cholangiocarcinoma: current knowledge and future perspectives consensus statement from the European Network for the Study of Cholangiocarcinoma (ENS-CCA). <em>Nat. Rev. Gastroenterol. Hepatol.</em><strong>13</strong>, 261–280 (2016).
</cite> [<a href="https://doi.org/10.1038/nrgastro.2016.51" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27095655/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Banales,%20J.%20M.%20et%20al.%20Expert%20consensus%20document:%20Cholangiocarcinoma:%20current%20knowledge%20and%20future%20perspectives%20consensus%20statement%20from%20the%20European%20Network%20for%20the%20Study%20of%20Cholangiocarcinoma%20(ENS-CCA).%20Nat.%20Rev.%20Gastroenterol.%20Hepatol.13,%20261%E2%80%93280%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>Banales, J. M. et al. Cholangiocarcinoma 2020: the next horizon in mechanisms and management. <em>Nat. Rev. Gastroenterol. Hepatol.</em><strong>17</strong>, 557–588 (2020).
</cite> [<a href="https://doi.org/10.1038/s41575-020-0310-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7447603/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32606456/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Banales,%20J.%20M.%20et%20al.%20Cholangiocarcinoma%202020:%20the%20next%20horizon%20in%20mechanisms%20and%20management.%20Nat.%20Rev.%20Gastroenterol.%20Hepatol.17,%20557%E2%80%93588%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>Oh, D. Y. et al. Plain language summary of the TOPAZ-1 study: durvalumab and chemotherapy for advanced biliary tract cancer. <em>Future Oncol.</em><strong>19</strong>, 2277–2289 (2023).
</cite> [<a href="https://doi.org/10.2217/fon-2023-0468" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37746835/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Oh,%20D.%20Y.%20et%20al.%20Plain%20language%20summary%20of%20the%20TOPAZ-1%20study:%20durvalumab%20and%20chemotherapy%20for%20advanced%20biliary%20tract%20cancer.%20Future%20Oncol.19,%202277%E2%80%932289%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Rimini, M. et al. Durvalumab plus gemcitabine and cisplatin in advanced biliary tract cancer: An early exploratory analysis of real-world data. <em>Liver Int</em><strong>43</strong>, 1803–1812 (2023).
</cite> [<a href="https://doi.org/10.1111/liv.15641" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37452505/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rimini,%20M.%20et%20al.%20Durvalumab%20plus%20gemcitabine%20and%20cisplatin%20in%20advanced%20biliary%20tract%20cancer:%20An%20early%20exploratory%20analysis%20of%20real-world%20data.%20Liver%20Int43,%201803%E2%80%931812%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Kelley, R. K. et al. Pembrolizumab in combination with gemcitabine and cisplatin compared with gemcitabine and cisplatin alone for patients with advanced biliary tract cancer (KEYNOTE-966): a randomised, double-blind, placebo-controlled, phase 3 trial. <em>Lancet</em><strong>401</strong>, 1853–1865 (2023).
</cite> [<a href="https://doi.org/10.1016/S0140-6736(23)00727-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37075781/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kelley,%20R.%20K.%20et%20al.%20Pembrolizumab%20in%20combination%20with%20gemcitabine%20and%20cisplatin%20compared%20with%20gemcitabine%20and%20cisplatin%20alone%20for%20patients%20with%20advanced%20biliary%20tract%20cancer%20(KEYNOTE-966):%20a%20randomised,%20double-blind,%20placebo-controlled,%20phase%203%20trial.%20Lancet401,%201853%E2%80%931865%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Skouteris, N. et al. Immune checkpoint inhibitors and combinations with other agents in cholangiocarcinoma. <em>Immunotherapy</em><strong>15</strong>, 487–502 (2023).
</cite> [<a href="https://doi.org/10.2217/imt-2022-0225" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36876442/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Skouteris,%20N.%20et%20al.%20Immune%20checkpoint%20inhibitors%20and%20combinations%20with%20other%20agents%20in%20cholangiocarcinoma.%20Immunotherapy15,%20487%E2%80%93502%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Oh, D. Y. et al. Bintrafusp alfa and chemotherapy as first-line treatment in biliary tract cancer: A randomized phase 2/3 trial. <em>Hepatology</em><strong>8</strong>, 823–836 (2025).</cite> [<a href="https://doi.org/10.1097/HEP.0000000000000965" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11825481/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38875119/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Oh,%20D.%20Y.%20et%20al.%20Bintrafusp%20alfa%20and%20chemotherapy%20as%20first-line%20treatment%20in%20biliary%20tract%20cancer:%20A%20randomized%20phase%202/3%20trial.%20Hepatology8,%20823%E2%80%93836%20(2025)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>Massague, J. &amp; Sheppard, D. TGF-beta signaling in health and disease. <em>Cell</em><strong>186</strong>, 4007–4037 (2023).
</cite> [<a href="https://doi.org/10.1016/j.cell.2023.07.036" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10772989/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37714133/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Massague,%20J.%20&amp;%20Sheppard,%20D.%20TGF-beta%20signaling%20in%20health%20and%20disease.%20Cell186,%204007%E2%80%934037%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>Gonzalez-Sanchez, E. et al. The TGF-beta Pathway: A Pharmacological Target in Hepatocellular Carcinoma?. <em>Cancers (Basel)</em><strong>13</strong>, 3248 (2021).
</cite> [<a href="https://doi.org/10.3390/cancers13133248" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8268320/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34209646/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gonzalez-Sanchez,%20E.%20et%20al.%20The%20TGF-beta%20Pathway:%20A%20Pharmacological%20Target%20in%20Hepatocellular%20Carcinoma?.%20Cancers%20(Basel)13,%203248%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>Tindall, R. R., Bailey-Lundberg, J. M., Cao, Y. &amp; Ko, T. C. The TGF-beta superfamily as potential therapeutic targets in pancreatic cancer. <em>Front Oncol.</em><strong>14</strong>, 1362247 (2024).
</cite> [<a href="https://doi.org/10.3389/fonc.2024.1362247" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10944957/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38500662/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tindall,%20R.%20R.,%20Bailey-Lundberg,%20J.%20M.,%20Cao,%20Y.%20&amp;%20Ko,%20T.%20C.%20The%20TGF-beta%20superfamily%20as%20potential%20therapeutic%20targets%20in%20pancreatic%20cancer.%20Front%20Oncol.14,%201362247%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Papoutsoglou, P., Louis, C. &amp; Coulouarn, C. Transforming Growth Factor-Beta (TGFbeta) Signaling Pathway in Cholangiocarcinoma. <em>Cells</em><strong>8</strong>, 960 (2019).
</cite> [<a href="https://doi.org/10.3390/cells8090960" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6770250/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31450767/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Papoutsoglou,%20P.,%20Louis,%20C.%20&amp;%20Coulouarn,%20C.%20Transforming%20Growth%20Factor-Beta%20(TGFbeta)%20Signaling%20Pathway%20in%20Cholangiocarcinoma.%20Cells8,%20960%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Gong, S., Wang, S. &amp; Shao, M. NADPH Oxidase 4: A Potential Therapeutic Target of Malignancy. <em>Front Cell Dev. Biol.</em><strong>10</strong>, 884412 (2022).
</cite> [<a href="https://doi.org/10.3389/fcell.2022.884412" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9130727/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35646942/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gong,%20S.,%20Wang,%20S.%20&amp;%20Shao,%20M.%20NADPH%20Oxidase%204:%20A%20Potential%20Therapeutic%20Target%20of%20Malignancy.%20Front%20Cell%20Dev.%20Biol.10,%20884412%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>Crosas-Molist, E. et al. The NADPH oxidase NOX4 inhibits hepatocyte proliferation and liver cancer progression. <em>Free Radic. Biol. Med</em>. <strong>69</strong>, 338–347 (2014).
</cite> [<a href="https://doi.org/10.1016/j.freeradbiomed.2014.01.040" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24509161/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Crosas-Molist,%20E.%20et%20al.%20The%20NADPH%20oxidase%20NOX4%20inhibits%20hepatocyte%20proliferation%20and%20liver%20cancer%20progression.%20Free%20Radic.%20Biol.%20Med.%2069,%20338%E2%80%93347%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>Shanmugasundaram, K. et al. NOX4 functions as a mitochondrial energetic sensor coupling cancer metabolic reprogramming to drug resistance. <em>Nat. Commun.</em><strong>8</strong>, 997 (2017).
</cite> [<a href="https://doi.org/10.1038/s41467-017-01106-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5648812/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29051480/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Shanmugasundaram,%20K.%20et%20al.%20NOX4%20functions%20as%20a%20mitochondrial%20energetic%20sensor%20coupling%20cancer%20metabolic%20reprogramming%20to%20drug%20resistance.%20Nat.%20Commun.8,%20997%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>Hanley, C. J. et al. Targeting the Myofibroblastic Cancer-Associated Fibroblast Phenotype Through Inhibition of NOX4. <em>J. Natl Cancer Inst.</em><strong>110</strong>, 109–120 (2018).
</cite> [<a href="https://doi.org/10.1093/jnci/djx121" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5903651/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28922779/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hanley,%20C.%20J.%20et%20al.%20Targeting%20the%20Myofibroblastic%20Cancer-Associated%20Fibroblast%20Phenotype%20Through%20Inhibition%20of%20NOX4.%20J.%20Natl%20Cancer%20Inst.110,%20109%E2%80%93120%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Siedlar, A. M. et al. NADPH oxidase 4 is dispensable for skin myofibroblast differentiation and wound healing. <em>Redox Biol.</em><strong>60</strong>, 102609 (2023).
</cite> [<a href="https://doi.org/10.1016/j.redox.2023.102609" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9950659/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36708644/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Siedlar,%20A.%20M.%20et%20al.%20NADPH%20oxidase%204%20is%20dispensable%20for%20skin%20myofibroblast%20differentiation%20and%20wound%20healing.%20Redox%20Biol.60,%20102609%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>Calvisi, D. F. et al. Criteria for preclinical models of cholangiocarcinoma: scientific and medical relevance. <em>Nat. Rev. Gastroenterol. Hepatol.</em><strong>20</strong>, 462–480 (2023).
</cite> [<a href="https://doi.org/10.1038/s41575-022-00739-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36755084/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Calvisi,%20D.%20F.%20et%20al.%20Criteria%20for%20preclinical%20models%20of%20cholangiocarcinoma:%20scientific%20and%20medical%20relevance.%20Nat.%20Rev.%20Gastroenterol.%20Hepatol.20,%20462%E2%80%93480%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Ilyas, S. I. et al. YAP-associated chromosomal instability and cholangiocarcinoma in mice. <em>Oncotarget</em><strong>9</strong>, 5892–5905 (2018).
</cite> [<a href="https://doi.org/10.18632/oncotarget.23638" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5814182/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29464042/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ilyas,%20S.%20I.%20et%20al.%20YAP-associated%20chromosomal%20instability%20and%20cholangiocarcinoma%20in%20mice.%20Oncotarget9,%205892%E2%80%935905%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Chen, X. &amp; Calvisi, D. F. Hydrodynamic transfection for generation of novel mouse models for liver cancer research. <em>Am. J. Pathol.</em><strong>184</strong>, 912–923 (2014).
</cite> [<a href="https://doi.org/10.1016/j.ajpath.2013.12.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3969989/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24480331/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chen,%20X.%20&amp;%20Calvisi,%20D.%20F.%20Hydrodynamic%20transfection%20for%20generation%20of%20novel%20mouse%20models%20for%20liver%20cancer%20research.%20Am.%20J.%20Pathol.184,%20912%E2%80%93923%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>Chen, Y. et al. TGF-beta1 expression is associated with invasion and metastasis of intrahepatic cholangiocarcinoma. <em>Biol. Res</em>. <strong>48</strong>, 26 (2015).
</cite> [<a href="https://doi.org/10.1186/s40659-015-0016-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4513632/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25993985/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chen,%20Y.%20et%20al.%20TGF-beta1%20expression%20is%20associated%20with%20invasion%20and%20metastasis%20of%20intrahepatic%20cholangiocarcinoma.%20Biol.%20Res.%2048,%2026%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>Miyaki, M. &amp; Kuroki, T. Role of Smad4 (DPC4) inactivation in human cancer. <em>Biochem Biophys. Res Commun.</em><strong>306</strong>, 799–804 (2003).
</cite> [<a href="https://doi.org/10.1016/s0006-291x(03)01066-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12821112/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Miyaki,%20M.%20&amp;%20Kuroki,%20T.%20Role%20of%20Smad4%20(DPC4)%20inactivation%20in%20human%20cancer.%20Biochem%20Biophys.%20Res%20Commun.306,%20799%E2%80%93804%20(2003)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>Coulouarn, C., Factor, V. M. &amp; Thorgeirsson, S. S. Transforming growth factor-beta gene expression signature in mouse hepatocytes predicts clinical outcome in human cancer. <em>Hepatology</em><strong>47</strong>, 2059–2067 (2008).
</cite> [<a href="https://doi.org/10.1002/hep.22283" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2762280/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18506891/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Coulouarn,%20C.,%20Factor,%20V.%20M.%20&amp;%20Thorgeirsson,%20S.%20S.%20Transforming%20growth%20factor-beta%20gene%20expression%20signature%20in%20mouse%20hepatocytes%20predicts%20clinical%20outcome%20in%20human%20cancer.%20Hepatology47,%202059%E2%80%932067%20(2008)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>Malfettone, A. et al. Transforming growth factor-beta-induced plasticity causes a migratory stemness phenotype in hepatocellular carcinoma. <em>Cancer Lett.</em><strong>392</strong>, 39–50 (2017).
</cite> [<a href="https://doi.org/10.1016/j.canlet.2017.01.037" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28161507/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Malfettone,%20A.%20et%20al.%20Transforming%20growth%20factor-beta-induced%20plasticity%20causes%20a%20migratory%20stemness%20phenotype%20in%20hepatocellular%20carcinoma.%20Cancer%20Lett.392,%2039%E2%80%9350%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>Caballero-Diaz, D. et al. Clathrin switches transforming growth factor-beta role to pro-tumorigenic in liver cancer. <em>J. Hepatol.</em><strong>72</strong>, 125–134 (2020).
</cite> [<a href="https://doi.org/10.1016/j.jhep.2019.09.012" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31562907/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Caballero-Diaz,%20D.%20et%20al.%20Clathrin%20switches%20transforming%20growth%20factor-beta%20role%20to%20pro-tumorigenic%20in%20liver%20cancer.%20J.%20Hepatol.72,%20125%E2%80%93134%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>Yan, X. &amp; Chen, Y. G. Smad7: not only a regulator, but also a cross-talk mediator of TGF-beta signalling. <em>Biochem J.</em><strong>434</strong>, 1–10 (2011).
</cite> [<a href="https://doi.org/10.1042/BJ20101827" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21269274/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yan,%20X.%20&amp;%20Chen,%20Y.%20G.%20Smad7:%20not%20only%20a%20regulator,%20but%20also%20a%20cross-talk%20mediator%20of%20TGF-beta%20signalling.%20Biochem%20J.434,%201%E2%80%9310%20(2011)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>Yingling, J. M. et al. Preclinical assessment of galunisertib (LY2157299 monohydrate), a first-in-class transforming growth factor-beta receptor type I inhibitor. <em>Oncotarget</em><strong>9</strong>, 6659–6677 (2018).
</cite> [<a href="https://doi.org/10.18632/oncotarget.23795" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5805504/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29467918/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yingling,%20J.%20M.%20et%20al.%20Preclinical%20assessment%20of%20galunisertib%20(LY2157299%20monohydrate),%20a%20first-in-class%20transforming%20growth%20factor-beta%20receptor%20type%20I%20inhibitor.%20Oncotarget9,%206659%E2%80%936677%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Bertran, E. et al. Overactivation of the TGF-beta pathway confers a mesenchymal-like phenotype and CXCR4-dependent migratory properties to liver tumor cells. <em>Hepatology</em><strong>58</strong>, 2032–2044 (2013).
</cite> [<a href="https://doi.org/10.1002/hep.26597" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23813475/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bertran,%20E.%20et%20al.%20Overactivation%20of%20the%20TGF-beta%20pathway%20confers%20a%20mesenchymal-like%20phenotype%20and%20CXCR4-dependent%20migratory%20properties%20to%20liver%20tumor%20cells.%20Hepatology58,%202032%E2%80%932044%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Ferraro, A. et al. Tumor suppressor role of the CL2/DRO1/CCDC80 gene in thyroid carcinogenesis. <em>J. Clin. Endocrinol. Metab.</em><strong>98</strong>, 2834–2843 (2013).
</cite> [<a href="https://doi.org/10.1210/jc.2012-2926" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23666966/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ferraro,%20A.%20et%20al.%20Tumor%20suppressor%20role%20of%20the%20CL2/DRO1/CCDC80%20gene%20in%20thyroid%20carcinogenesis.%20J.%20Clin.%20Endocrinol.%20Metab.98,%202834%E2%80%932843%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>Haldrup, J. et al. FRMD6 has tumor suppressor functions in prostate cancer. <em>Oncogene</em><strong>40</strong>, 763–776 (2021).
</cite> [<a href="https://doi.org/10.1038/s41388-020-01548-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33249427/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Haldrup,%20J.%20et%20al.%20FRMD6%20has%20tumor%20suppressor%20functions%20in%20prostate%20cancer.%20Oncogene40,%20763%E2%80%93776%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR30">
<span class="label">30.</span><cite>Seubwai, W., Wongkham, C., Puapairoj, A., Khuntikeo, N. &amp; Wongkham, S. Overexpression of vitamin D receptor indicates a good prognosis for cholangiocarcinoma: implications for therapeutics. <em>Cancer</em><strong>109</strong>, 2497–2505 (2007).
</cite> [<a href="https://doi.org/10.1002/cncr.22716" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17487855/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Seubwai,%20W.,%20Wongkham,%20C.,%20Puapairoj,%20A.,%20Khuntikeo,%20N.%20&amp;%20Wongkham,%20S.%20Overexpression%20of%20vitamin%20D%20receptor%20indicates%20a%20good%20prognosis%20for%20cholangiocarcinoma:%20implications%20for%20therapeutics.%20Cancer109,%202497%E2%80%932505%20(2007)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>Meitzler, J. L. et al. Decoding NADPH oxidase 4 expression in human tumors. <em>Redox Biol.</em><strong>13</strong>, 182–195 (2017).
</cite> [<a href="https://doi.org/10.1016/j.redox.2017.05.016" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5458090/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28578276/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Meitzler,%20J.%20L.%20et%20al.%20Decoding%20NADPH%20oxidase%204%20expression%20in%20human%20tumors.%20Redox%20Biol.13,%20182%E2%80%93195%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>Lu, J. et al. NADPH oxidase 1 is highly expressed in human large and small bowel cancers. <em>PLoS One</em><strong>15</strong>, e0233208 (2020).
</cite> [<a href="https://doi.org/10.1371/journal.pone.0233208" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7237001/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32428030/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lu,%20J.%20et%20al.%20NADPH%20oxidase%201%20is%20highly%20expressed%20in%20human%20large%20and%20small%20bowel%20cancers.%20PLoS%20One15,%20e0233208%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Invernizzi, P. et al. Setanaxib, a first-in-class selective NADPH oxidase 1/4 inhibitor for primary biliary cholangitis: A randomized, placebo-controlled, phase 2 trial. <em>Liver Int</em><strong>43</strong>, 1507–1522 (2023).
</cite> [<a href="https://doi.org/10.1111/liv.15596" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37183520/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Invernizzi,%20P.%20et%20al.%20Setanaxib,%20a%20first-in-class%20selective%20NADPH%20oxidase%201/4%20inhibitor%20for%20primary%20biliary%20cholangitis:%20A%20randomized,%20placebo-controlled,%20phase%202%20trial.%20Liver%20Int43,%201507%E2%80%931522%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>Cui, W. et al. NOX1/nicotinamide adenine dinucleotide phosphate, reduced form (NADPH) oxidase promotes proliferation of stellate cells and aggravates liver fibrosis induced by bile duct ligation. <em>Hepatology</em><strong>54</strong>, 949–958 (2011).
</cite> [<a href="https://doi.org/10.1002/hep.24465" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21618578/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cui,%20W.%20et%20al.%20NOX1/nicotinamide%20adenine%20dinucleotide%20phosphate,%20reduced%20form%20(NADPH)%20oxidase%20promotes%20proliferation%20of%20stellate%20cells%20and%20aggravates%20liver%20fibrosis%20induced%20by%20bile%20duct%20ligation.%20Hepatology54,%20949%E2%80%93958%20(2011)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>Song, X. et al. Focal adhesion kinase (FAK) promotes cholangiocarcinoma development and progression via YAP activation. <em>J. Hepatol.</em><strong>75</strong>, 888–899 (2021).
</cite> [<a href="https://doi.org/10.1016/j.jhep.2021.05.018" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8453055/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34052254/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Song,%20X.%20et%20al.%20Focal%20adhesion%20kinase%20(FAK)%20promotes%20cholangiocarcinoma%20development%20and%20progression%20via%20YAP%20activation.%20J.%20Hepatol.75,%20888%E2%80%93899%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR36">
<span class="label">36.</span><cite>Nicolas-Boluda, A. et al. Tumor stiffening reversion through collagen crosslinking inhibition improves T cell migration and anti-PD-1 treatment. <em>Elife</em><strong>10</strong>, e58688 (2021).
</cite> [<a href="https://doi.org/10.7554/eLife.58688" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8203293/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34106045/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Nicolas-Boluda,%20A.%20et%20al.%20Tumor%20stiffening%20reversion%20through%20collagen%20crosslinking%20inhibition%20improves%20T%20cell%20migration%20and%20anti-PD-1%20treatment.%20Elife10,%20e58688%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR37">
<span class="label">37.</span><cite>David, C. J. et al. TGF-beta Tumor Suppression through a Lethal EMT. <em>Cell</em><strong>164</strong>, 1015–1030 (2016).
</cite> [<a href="https://doi.org/10.1016/j.cell.2016.01.009" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4801341/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26898331/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?David,%20C.%20J.%20et%20al.%20TGF-beta%20Tumor%20Suppression%20through%20a%20Lethal%20EMT.%20Cell164,%201015%E2%80%931030%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR38">
<span class="label">38.</span><cite>Guasch, G. et al. Loss of TGFbeta signaling destabilizes homeostasis and promotes squamous cell carcinomas in stratified epithelia. <em>Cancer Cell</em><strong>12</strong>, 313–327 (2007).
</cite> [<a href="https://doi.org/10.1016/j.ccr.2007.08.020" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2424201/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17936557/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Guasch,%20G.%20et%20al.%20Loss%20of%20TGFbeta%20signaling%20destabilizes%20homeostasis%20and%20promotes%20squamous%20cell%20carcinomas%20in%20stratified%20epithelia.%20Cancer%20Cell12,%20313%E2%80%93327%20(2007)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR39">
<span class="label">39.</span><cite>Dong, M. et al. Efficacy of MEK inhibition in a K-Ras-driven cholangiocarcinoma preclinical model. <em>Cell Death Dis.</em><strong>9</strong>, 31 (2018).
</cite> [<a href="https://doi.org/10.1038/s41419-017-0183-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5833851/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29348467/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Dong,%20M.%20et%20al.%20Efficacy%20of%20MEK%20inhibition%20in%20a%20K-Ras-driven%20cholangiocarcinoma%20preclinical%20model.%20Cell%20Death%20Dis.9,%2031%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR40">
<span class="label">40.</span><cite>Bailey, P. et al. Genomic analyses identify molecular subtypes of pancreatic cancer. <em>Nature</em><strong>531</strong>, 47–52 (2016).
</cite> [<a href="https://doi.org/10.1038/nature16965" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26909576/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bailey,%20P.%20et%20al.%20Genomic%20analyses%20identify%20molecular%20subtypes%20of%20pancreatic%20cancer.%20Nature531,%2047%E2%80%9352%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR41">
<span class="label">41.</span><cite>Valle, J. W., Lamarca, A., Goyal, L., Barriuso, J. &amp; Zhu, A. X. New Horizons for Precision Medicine in Biliary Tract Cancers. <em>Cancer Discov.</em><strong>7</strong>, 943–962 (2017).
</cite> [<a href="https://doi.org/10.1158/2159-8290.CD-17-0245" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5586506/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28818953/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Valle,%20J.%20W.,%20Lamarca,%20A.,%20Goyal,%20L.,%20Barriuso,%20J.%20&amp;%20Zhu,%20A.%20X.%20New%20Horizons%20for%20Precision%20Medicine%20in%20Biliary%20Tract%20Cancers.%20Cancer%20Discov.7,%20943%E2%80%93962%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR42">
<span class="label">42.</span><cite>Ford, K. et al. NOX4 Inhibition Potentiates Immunotherapy by Overcoming Cancer-Associated Fibroblast-Mediated CD8 T-cell Exclusion from Tumors. <em>Cancer Res</em>. <strong>80</strong>, 1846–1860 (2020).
</cite> [<a href="https://doi.org/10.1158/0008-5472.CAN-19-3158" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7611230/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32122909/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ford,%20K.%20et%20al.%20NOX4%20Inhibition%20Potentiates%20Immunotherapy%20by%20Overcoming%20Cancer-Associated%20Fibroblast-Mediated%20CD8%20T-cell%20Exclusion%20from%20Tumors.%20Cancer%20Res.%2080,%201846%E2%80%931860%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR43">
<span class="label">43.</span><cite>Crestani, B., Besnard, V. &amp; Boczkowski, J. Signalling pathways from NADPH oxidase-4 to idiopathic pulmonary fibrosis. <em>Int J. Biochem Cell Biol.</em><strong>43</strong>, 1086–1089 (2011).
</cite> [<a href="https://doi.org/10.1016/j.biocel.2011.04.003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21513813/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Crestani,%20B.,%20Besnard,%20V.%20&amp;%20Boczkowski,%20J.%20Signalling%20pathways%20from%20NADPH%20oxidase-4%20to%20idiopathic%20pulmonary%20fibrosis.%20Int%20J.%20Biochem%20Cell%20Biol.43,%201086%E2%80%931089%20(2011)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR44">
<span class="label">44.</span><cite>Barnes, J. L. &amp; Gorin, Y. Myofibroblast differentiation during fibrosis: role of NAD(P)H oxidases. <em>Kidney Int</em><strong>79</strong>, 944–956 (2011).
</cite> [<a href="https://doi.org/10.1038/ki.2010.516" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3675765/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21307839/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Barnes,%20J.%20L.%20&amp;%20Gorin,%20Y.%20Myofibroblast%20differentiation%20during%20fibrosis:%20role%20of%20NAD(P)H%20oxidases.%20Kidney%20Int79,%20944%E2%80%93956%20(2011)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR45">
<span class="label">45.</span><cite>Sampson, N. et al. Inhibition of Nox4-dependent ROS signaling attenuates prostate fibroblast activation and abrogates stromal-mediated protumorigenic interactions. <em>Int J. Cancer</em><strong>143</strong>, 383–395 (2018).
</cite> [<a href="https://doi.org/10.1002/ijc.31316" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6067067/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29441570/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sampson,%20N.%20et%20al.%20Inhibition%20of%20Nox4-dependent%20ROS%20signaling%20attenuates%20prostate%20fibroblast%20activation%20and%20abrogates%20stromal-mediated%20protumorigenic%20interactions.%20Int%20J.%20Cancer143,%20383%E2%80%93395%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR46">
<span class="label">46.</span><cite>Mir, S. et al. Upregulation of Nox4 induces a pro-survival Nrf2 response in cancer-associated fibroblasts that promotes tumorigenesis and metastasis, in part via Birc5 induction. <em>Breast Cancer Res</em>. <strong>24</strong>, 48 (2022).
</cite> [<a href="https://doi.org/10.1186/s13058-022-01548-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9281082/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35836253/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mir,%20S.%20et%20al.%20Upregulation%20of%20Nox4%20induces%20a%20pro-survival%20Nrf2%20response%20in%20cancer-associated%20fibroblasts%20that%20promotes%20tumorigenesis%20and%20metastasis,%20in%20part%20via%20Birc5%20induction.%20Breast%20Cancer%20Res.%2024,%2048%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR47">
<span class="label">47.</span><cite>Huang, H. et al. Mesothelial cell-derived antigen-presenting cancer-associated fibroblasts induce expansion of regulatory T cells in pancreatic cancer. <em>Cancer Cell</em><strong>40</strong>, 656–673 (2022).
</cite> [<a href="https://doi.org/10.1016/j.ccell.2022.04.011" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9197998/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35523176/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Huang,%20H.%20et%20al.%20Mesothelial%20cell-derived%20antigen-presenting%20cancer-associated%20fibroblasts%20induce%20expansion%20of%20regulatory%20T%20cells%20in%20pancreatic%20cancer.%20Cancer%20Cell40,%20656%E2%80%93673%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR48">
<span class="label">48.</span><cite>Sun, L. et al. PD-L1 promotes myofibroblastic activation of hepatic stellate cells by distinct mechanisms selective for TGF-beta receptor I versus II. <em>Cell Rep.</em><strong>38</strong>, 110349 (2022).
</cite> [<a href="https://doi.org/10.1016/j.celrep.2022.110349" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8903892/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35139382/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sun,%20L.%20et%20al.%20PD-L1%20promotes%20myofibroblastic%20activation%20of%20hepatic%20stellate%20cells%20by%20distinct%20mechanisms%20selective%20for%20TGF-beta%20receptor%20I%20versus%20II.%20Cell%20Rep.38,%20110349%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR49">
<span class="label">49.</span><cite>Carapeto, F. et al. The immunogenomic landscape of resected intrahepatic cholangiocarcinoma. <em>Hepatology</em><strong>75</strong>, 297–308 (2022).
</cite> [<a href="https://doi.org/10.1002/hep.32150" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8766948/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34510503/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Carapeto,%20F.%20et%20al.%20The%20immunogenomic%20landscape%20of%20resected%20intrahepatic%20cholangiocarcinoma.%20Hepatology75,%20297%E2%80%93308%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR50">
<span class="label">50.</span><cite>Xia, T. et al. Immune cell atlas of cholangiocarcinomas reveals distinct tumor microenvironments and associated prognoses. <em>J. Hematol. Oncol.</em><strong>15</strong>, 37 (2022).
</cite> [<a href="https://doi.org/10.1186/s13045-022-01253-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8962046/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35346322/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Xia,%20T.%20et%20al.%20Immune%20cell%20atlas%20of%20cholangiocarcinomas%20reveals%20distinct%20tumor%20microenvironments%20and%20associated%20prognoses.%20J.%20Hematol.%20Oncol.15,%2037%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR51">
<span class="label">51.</span><cite>Serra-Camprubi, Q. et al. Human Metastatic Cholangiocarcinoma Patient-Derived Xenografts and Tumoroids for Preclinical Drug Evaluation. <em>Clin. Cancer Res</em>. <strong>29</strong>, 432–445 (2023).
</cite> [<a href="https://doi.org/10.1158/1078-0432.CCR-22-2551" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9873249/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36374558/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Serra-Camprubi,%20Q.%20et%20al.%20Human%20Metastatic%20Cholangiocarcinoma%20Patient-Derived%20Xenografts%20and%20Tumoroids%20for%20Preclinical%20Drug%20Evaluation.%20Clin.%20Cancer%20Res.%2029,%20432%E2%80%93445%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR52">
<span class="label">52.</span><cite>Sulpice, L. et al. Molecular profiling of stroma identifies osteopontin as an independent predictor of poor prognosis in intrahepatic cholangiocarcinoma. <em>Hepatology</em><strong>58</strong>, 1992–2000 (2013).
</cite> [<a href="https://doi.org/10.1002/hep.26577" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23775819/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sulpice,%20L.%20et%20al.%20Molecular%20profiling%20of%20stroma%20identifies%20osteopontin%20as%20an%20independent%20predictor%20of%20poor%20prognosis%20in%20intrahepatic%20cholangiocarcinoma.%20Hepatology58,%201992%E2%80%932000%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR53">
<span class="label">53.</span><cite>Tang, Z. et al. GEPIA: a web server for cancer and normal gene expression profiling and interactive analyses. <em>Nucleic Acids Res</em><strong>45</strong>, W98–W102 (2017).
</cite> [<a href="https://doi.org/10.1093/nar/gkx247" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5570223/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28407145/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tang,%20Z.%20et%20al.%20GEPIA:%20a%20web%20server%20for%20cancer%20and%20normal%20gene%20expression%20profiling%20and%20interactive%20analyses.%20Nucleic%20Acids%20Res45,%20W98%E2%80%93W102%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR54">
<span class="label">54.</span><cite>Cerami, E. et al. The cBio cancer genomics portal: an open platform for exploring multidimensional cancer genomics data. <em>Cancer Discov.</em><strong>2</strong>, 401–404 (2012).
</cite> [<a href="https://doi.org/10.1158/2159-8290.CD-12-0095" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3956037/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22588877/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cerami,%20E.%20et%20al.%20The%20cBio%20cancer%20genomics%20portal:%20an%20open%20platform%20for%20exploring%20multidimensional%20cancer%20genomics%20data.%20Cancer%20Discov.2,%20401%E2%80%93404%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR55">
<span class="label">55.</span><cite>Gao, J. et al. Integrative analysis of complex cancer genomics and clinical profiles using the cBioPortal. <em>Sci. Signal</em><strong>6</strong>, pl1 (2013).
</cite> [<a href="https://doi.org/10.1126/scisignal.2004088" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4160307/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23550210/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gao,%20J.%20et%20al.%20Integrative%20analysis%20of%20complex%20cancer%20genomics%20and%20clinical%20profiles%20using%20the%20cBioPortal.%20Sci.%20Signal6,%20pl1%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR56">
<span class="label">56.</span><cite>Irizarry, R. A. et al. Exploration, normalization, and summaries of high density oligonucleotide array probe level data. <em>Biostatistics</em><strong>4</strong>, 249–264 (2003).
</cite> [<a href="https://doi.org/10.1093/biostatistics/4.2.249" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12925520/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Irizarry,%20R.%20A.%20et%20al.%20Exploration,%20normalization,%20and%20summaries%20of%20high%20density%20oligonucleotide%20array%20probe%20level%20data.%20Biostatistics4,%20249%E2%80%93264%20(2003)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR57">
<span class="label">57.</span><cite>Hanzelmann, S., Castelo, R. &amp; Guinney, J. GSVA: gene set variation analysis for microarray and RNA-seq data. <em>BMC Bioinforma.</em><strong>14</strong>, 7 (2013).</cite> [<a href="https://doi.org/10.1186/1471-2105-14-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3618321/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23323831/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hanzelmann,%20S.,%20Castelo,%20R.%20&amp;%20Guinney,%20J.%20GSVA:%20gene%20set%20variation%20analysis%20for%20microarray%20and%20RNA-seq%20data.%20BMC%20Bioinforma.14,%207%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR58">
<span class="label">58.</span><cite>Beaufrère A, L. T. et al. Self-supervised learning to predict intrahepatic cholangiocarcinoma transcriptomic classes on routine histology. bioRxiv; 2024. p. 2024.01.15.575652.</cite>
</li>
<li id="CR59">
<span class="label">59.</span><cite>Charoentong, P. et al. Pan-cancer Immunogenomic Analyses Reveal Genotype-Immunophenotype Relationships and Predictors of Response to Checkpoint Blockade. <em>Cell Rep.</em><strong>18</strong>, 248–262 (2017).
</cite> [<a href="https://doi.org/10.1016/j.celrep.2016.12.019" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28052254/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Charoentong,%20P.%20et%20al.%20Pan-cancer%20Immunogenomic%20Analyses%20Reveal%20Genotype-Immunophenotype%20Relationships%20and%20Predictors%20of%20Response%20to%20Checkpoint%20Blockade.%20Cell%20Rep.18,%20248%E2%80%93262%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR60">
<span class="label">60.</span><cite>Dobin, A. et al. STAR: ultrafast universal RNA-seq aligner. <em>Bioinformatics</em><strong>29</strong>, 15–21 (2013).
</cite> [<a href="https://doi.org/10.1093/bioinformatics/bts635" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3530905/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23104886/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Dobin,%20A.%20et%20al.%20STAR:%20ultrafast%20universal%20RNA-seq%20aligner.%20Bioinformatics29,%2015%E2%80%9321%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR61">
<span class="label">61.</span><cite>Li, B. &amp; Dewey, C. N. RSEM: accurate transcript quantification from RNA-Seq data with or without a reference genome. <em>BMC Bioinforma.</em><strong>12</strong>, 323 (2011).</cite> [<a href="https://doi.org/10.1186/1471-2105-12-323" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3163565/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21816040/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li,%20B.%20&amp;%20Dewey,%20C.%20N.%20RSEM:%20accurate%20transcript%20quantification%20from%20RNA-Seq%20data%20with%20or%20without%20a%20reference%20genome.%20BMC%20Bioinforma.12,%20323%20(2011)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR62">
<span class="label">62.</span><cite>Law, C. W., Chen, Y., Shi, W. &amp; Smyth, G. K. voom: Precision weights unlock linear model analysis tools for RNA-seq read counts. <em>Genome Biol.</em><strong>15</strong>, R29 (2014).
</cite> [<a href="https://doi.org/10.1186/gb-2014-15-2-r29" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4053721/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24485249/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Law,%20C.%20W.,%20Chen,%20Y.,%20Shi,%20W.%20&amp;%20Smyth,%20G.%20K.%20voom:%20Precision%20weights%20unlock%20linear%20model%20analysis%20tools%20for%20RNA-seq%20read%20counts.%20Genome%20Biol.15,%20R29%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR63">
<span class="label">63.</span><cite>Korotkevich, G. e. a. F. g. s. e. a. b., 060012, 10.1101/060012 (2021).</cite>
</li>
<li id="CR64">
<span class="label">64.</span><cite>Vaquero, J. et al. The IGF2/IR/IGF1R Pathway in Tumor Cells and Myofibroblasts Mediates Resistance to EGFR Inhibition in Cholangiocarcinoma. <em>Clin. Cancer Res</em><strong>24</strong>, 4282–4296 (2018).
</cite> [<a href="https://doi.org/10.1158/1078-0432.CCR-17-3725" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29716918/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Vaquero,%20J.%20et%20al.%20The%20IGF2/IR/IGF1R%20Pathway%20in%20Tumor%20Cells%20and%20Myofibroblasts%20Mediates%20Resistance%20to%20EGFR%20Inhibition%20in%20Cholangiocarcinoma.%20Clin.%20Cancer%20Res24,%204282%E2%80%934296%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR65">
<span class="label">65.</span><cite>Carnesecchi, S. et al. A key role for NOX4 in epithelial cell death during development of lung fibrosis. <em>Antioxid. Redox Signal</em><strong>15</strong>, 607–619 (2011).
</cite> [<a href="https://doi.org/10.1089/ars.2010.3829" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3163392/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21391892/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Carnesecchi,%20S.%20et%20al.%20A%20key%20role%20for%20NOX4%20in%20epithelial%20cell%20death%20during%20development%20of%20lung%20fibrosis.%20Antioxid.%20Redox%20Signal15,%20607%E2%80%93619%20(2011)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12358586/bin/41392_2025_2347_MOESM1_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material</a><sup> (20MB, docx) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>RNA-seq datasets generated in this study are available at Gene Expression Omnibus (GEO) under the accession number <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE274024" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE274024</a>. The rest of the data produced in this study can be found in the article and its supplementary data files. The publicly available data used in this study, which were generated by other researchers, were sourced from TCGA-CHOL and GEO databases under the accession numbers <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE76297" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE76297</a>, <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE244807" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE244807</a> and <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE201425" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE201425</a>.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Signal Transduction and Targeted Therapy are provided here courtesy of <strong>Nature Publishing Group</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1038/s41392-025-02347-z"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/41392_2025_Article_2347.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (11.5 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12358586/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12358586/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12358586%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12358586/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12358586/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12358586/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40820091/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12358586/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40820091/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12358586/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12358586/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="MbJeM58KGyIDLxffQBrS8TmkucdaRmjR9x0HbNKfJc1d6fganWkpmHXYvT1cczCR">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
