
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Toward a Beautiful Amalgam: The Necessity of Heterogeneity in RNA Science and Research Culture - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="D2E4A1F48AF1E15305A1F40054C539CD.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="acsbmcau">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371483/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="ACS Bio &amp; Med Chem Au">
<meta name="citation_title" content="Toward a Beautiful Amalgam: The Necessity of Heterogeneity in RNA Science and Research Culture">
<meta name="citation_author" content="Sudeshi M Abedeera">
<meta name="citation_author_institution" content="Department of Biochemistry and Biophysics, Perelman School of Medicine, University of Pennsylvania, Philadelphia, Pennsylvania 19104, United States">
<meta name="citation_author" content="Mary Donnelly">
<meta name="citation_author_institution" content="Department of Biochemistry and Biophysics, Perelman School of Medicine, University of Pennsylvania, Philadelphia, Pennsylvania 19104, United States">
<meta name="citation_author" content="James Hagerty">
<meta name="citation_author_institution" content="Department of Biochemistry and Biophysics, Perelman School of Medicine, University of Pennsylvania, Philadelphia, Pennsylvania 19104, United States">
<meta name="citation_author_institution" content="Howard Hughes Medical Institute, Chevy Chase, Maryland 20815, United States">
<meta name="citation_author" content="Shaila Kolli">
<meta name="citation_author_institution" content="Department of Biochemistry and Biophysics, Perelman School of Medicine, University of Pennsylvania, Philadelphia, Pennsylvania 19104, United States">
<meta name="citation_author" content="Srinivasa R Penumutchu">
<meta name="citation_author_institution" content="Department of Biochemistry and Biophysics, Perelman School of Medicine, University of Pennsylvania, Philadelphia, Pennsylvania 19104, United States">
<meta name="citation_author_institution" content="Howard Hughes Medical Institute, Chevy Chase, Maryland 20815, United States">
<meta name="citation_author" content="Louis G Smith">
<meta name="citation_author_institution" content="Department of Biochemistry and Biophysics, Perelman School of Medicine, University of Pennsylvania, Philadelphia, Pennsylvania 19104, United States">
<meta name="citation_author" content="Blanton S Tolbert">
<meta name="citation_author_institution" content="Department of Biochemistry and Biophysics, Perelman School of Medicine, University of Pennsylvania, Philadelphia, Pennsylvania 19104, United States">
<meta name="citation_author_institution" content="Howard Hughes Medical Institute, Chevy Chase, Maryland 20815, United States">
<meta name="citation_publication_date" content="2025 Jul 12">
<meta name="citation_volume" content="5">
<meta name="citation_issue" content="4">
<meta name="citation_firstpage" content="519">
<meta name="citation_doi" content="10.1021/acsbiomedchemau.5c00114">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371483/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371483/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371483/pdf/bg5c00114.pdf">
<meta name="description" content="RNA biology exemplifies functional heterogeneitydistinct RNA classes are expressed in tissue- and development-specific contexts, adopt dynamic conformational ensembles, and form intricate, context-dependent interactions with proteins and other ...">
<meta name="og:title" content="Toward a Beautiful Amalgam: The Necessity of Heterogeneity in RNA Science and Research Culture">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="RNA biology exemplifies functional heterogeneitydistinct RNA classes are expressed in tissue- and development-specific contexts, adopt dynamic conformational ensembles, and form intricate, context-dependent interactions with proteins and other ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371483/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12371483">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1021/acsbiomedchemau.5c00114"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/bg5c00114.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12371483%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12371483/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12371483/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371483/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-acsbmcau.png" alt="ACS Bio &amp; Med Chem Au logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to ACS Bio &amp; Med Chem Au" title="Link to ACS Bio &amp; Med Chem Au" shape="default" href="https://pubs.acs.org/journal/abmcb8" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">ACS Bio Med Chem Au</button></div>. 2025 Jul 12;5(4):519–530. doi: <a href="https://doi.org/10.1021/acsbiomedchemau.5c00114" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1021/acsbiomedchemau.5c00114</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22ACS%20Bio%20Med%20Chem%20Au%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22ACS%20Bio%20Med%20Chem%20Au%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22ACS%20Bio%20Med%20Chem%20Au%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22ACS%20Bio%20Med%20Chem%20Au%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Toward a Beautiful Amalgam: The Necessity of Heterogeneity
in RNA Science and Research Culture</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Abedeera%20SM%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Sudeshi M Abedeera</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Sudeshi M Abedeera</span></h3>
<div class="p">
<sup>†</sup>
Department
of Biochemistry and Biophysics, Perelman School of Medicine, University of Pennsylvania, Philadelphia, Pennsylvania
19104, United States
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Abedeera%20SM%22%5BAuthor%5D" class="usa-link"><span class="name western">Sudeshi M Abedeera</span></a>
</div>
</div>
<sup>†</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Donnelly%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Mary Donnelly</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Mary Donnelly</span></h3>
<div class="p">
<sup>†</sup>
Department
of Biochemistry and Biophysics, Perelman School of Medicine, University of Pennsylvania, Philadelphia, Pennsylvania
19104, United States
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Donnelly%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Mary Donnelly</span></a>
</div>
</div>
<sup>†</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hagerty%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">James Hagerty</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">James Hagerty</span></h3>
<div class="p">
<sup>†</sup>
Department
of Biochemistry and Biophysics, Perelman School of Medicine, University of Pennsylvania, Philadelphia, Pennsylvania
19104, United States
</div>
<div class="p">
<sup>‡</sup>
Howard
Hughes Medical Institute, Chevy
Chase, Maryland
20815, United States
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hagerty%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">James Hagerty</span></a>
</div>
</div>
<sup>†,</sup><sup>‡</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kolli%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Shaila Kolli</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Shaila Kolli</span></h3>
<div class="p">
<sup>†</sup>
Department
of Biochemistry and Biophysics, Perelman School of Medicine, University of Pennsylvania, Philadelphia, Pennsylvania
19104, United States
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kolli%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Shaila Kolli</span></a>
</div>
</div>
<sup>†</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Penumutchu%20SR%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Srinivasa R Penumutchu</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Srinivasa R Penumutchu</span></h3>
<div class="p">
<sup>†</sup>
Department
of Biochemistry and Biophysics, Perelman School of Medicine, University of Pennsylvania, Philadelphia, Pennsylvania
19104, United States
</div>
<div class="p">
<sup>‡</sup>
Howard
Hughes Medical Institute, Chevy
Chase, Maryland
20815, United States
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Penumutchu%20SR%22%5BAuthor%5D" class="usa-link"><span class="name western">Srinivasa R Penumutchu</span></a>
</div>
</div>
<sup>†,</sup><sup>‡</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Smith%20LG%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Louis G Smith</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Louis G Smith</span></h3>
<div class="p">
<sup>†</sup>
Department
of Biochemistry and Biophysics, Perelman School of Medicine, University of Pennsylvania, Philadelphia, Pennsylvania
19104, United States
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Smith%20LG%22%5BAuthor%5D" class="usa-link"><span class="name western">Louis G Smith</span></a>
</div>
</div>
<sup>†</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tolbert%20BS%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Blanton S Tolbert</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Blanton S Tolbert</span></h3>
<div class="p">
<sup>†</sup>
Department
of Biochemistry and Biophysics, Perelman School of Medicine, University of Pennsylvania, Philadelphia, Pennsylvania
19104, United States
</div>
<div class="p">
<sup>‡</sup>
Howard
Hughes Medical Institute, Chevy
Chase, Maryland
20815, United States
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tolbert%20BS%22%5BAuthor%5D" class="usa-link"><span class="name western">Blanton S Tolbert</span></a>
</div>
</div>
<sup>†,</sup><sup>‡,</sup><sup>*</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="aff1">
<sup>†</sup>
Department
of Biochemistry and Biophysics, Perelman School of Medicine, University of Pennsylvania, Philadelphia, Pennsylvania
19104, United States
</div>
<div id="aff2">
<sup>‡</sup>
Howard
Hughes Medical Institute, Chevy
Chase, Maryland
20815, United States
</div>
<div class="author-notes p"><div class="fn" id="cor1">
<sup>*</sup><p class="display-inline">
Email: <span>blanton.tolbert@pennmedicine.upenn.edu</span>.</p>
</div></div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 May 5; Accepted 2025 Jun 30; Revised 2025 Jun 30; Collection date 2025 Aug 20.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© 2025 The Authors. Published by American Chemical Society</div>
<p>This article is licensed under CC-BY-NC-ND 4.0</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12371483</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>Abstract</h2>
<p>RNA biology exemplifies
functional heterogeneitydistinct
RNA classes are expressed in tissue- and development-specific contexts,
adopt dynamic conformational ensembles, and form intricate, context-dependent
interactions with proteins and other molecules to regulate gene expression.
These features make RNA a powerful metaphor for reimagining scientific
culture. Just as RNA achieves biological complexity through versatility,
feedback loops, and communication, research environments thrive when
they support dynamic interactions, structural adaptability, and the
intentional inclusion of divergent perspectives and experiences. However,
unlike RNA, research culture is shaped by human behavior, institutional
norms, and systemic barriersforces that can suppress innovation
and limit who contributes to scientific discovery. Scientific excellence
demands the integration of wide-ranging perspectives to challenge
paradigms and push boundaries. Yet entrenched structures often reward
conformity and marginalize creativity born from difference. By embracing
the principles inherent to RNA biologycontextual responsiveness,
structural plasticity, and cooperativitywe can transform scientific
culture into one that is more inclusive, welcoming, and adaptable.
This perspective argues that the biological elegance of RNA offers
more than molecular insight; it provides a conceptual framework for
building research environments that harness the full spectrum of talent
in our richly heterogeneous society, ultimately accelerating scientific
progress and broadening its societal impact.</p>
<section id="kwd-group1" class="kwd-group"><p><strong>Keywords:</strong> heterogeneity, RNA, versatility, science
culture, feedback loops, communication</p></section></section><section class="abstract" id="abstract2"><hr class="headless">
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/bef3/12371483/b963b171e38a/bg5c00114_0008.jpg" loading="lazy" id="agr1" height="369" width="598" alt="graphic file with name bg5c00114_0008.jpg"></p></section><section class="abstract" id="abstract3"><hr class="headless">
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/bef3/12371483/b963b171e38a/bg5c00114_0006.jpg" loading="lazy" id="tgr1" height="369" width="598" alt="graphic file with name bg5c00114_0006.jpg"></p></section><section id="sec1"><h2 class="pmc_sec_title">1. Introduction</h2>
<p>The interactions that
take place among researchers, as well as
the influence of their various talents, perspectives, values, and
motivations, lie at the heart of science culture. Centuries of inquiry
and discovery have shown that science thrives as a collective enterpriseone
that welcomes a multitude of viewpoints and encourages the development,
critique, and refinement of a wide range of theories. Historians and philosophers who have examined the scientific
process tend to share the belief that scientific progress is more
likely to occur when research communities comprise members with a
variety of skill sets, different areas of expertise, and a diverse
range of professional and personal experiences.<em>
,
</em> Heterogeneous
scientific communities can deploy a wider range of experiences to
overcome barriers to progress, including unchallenged assumptions,
underappreciated areas of study, and the exclusion of qualified participants.
Scientists cannot predict when or from what avenues of investigation
innovative ideas will emerge. To give just one example from the realm
of RNA science: studies designed to investigate bacterial immunity
inadvertently led to the identification of restriction enzymes and
CRISPR-Cas systems-two tools that have revolutionized the fields of
molecular biology and genetics.<em>
−

</em>
</p>
<p>Homogeneous organizational
models, in contrast, tend to stifle
creative potential and diminish the pool of potential scientists. These models resist accommodating varied learning
styles or perspectives and can undervalue the contributions of those
with less expertise. The effects of homogeneity, whether in a biological
or organizational context, can lead to the enrichment of undesirable
traits.<em>
,
</em> In biology, such traits may result in susceptibility
to pathology or in an inability to adapt and flourish. Reliance on
the use of monocultures for banana cultivation in the early 20th century,
for example, had dire consequences, including the near-complete loss
of a variety from a single fungal infection, a problem that persists
to this day, threatening the existence of the Cavendish variety.<em>
,
</em> Likewise, homogeneous scientific teams are at greater risk of rigidity
in thinking, ranging from a fear of departing from the status quo
to dogmatism.
</p>
<p>To build a robust,
adaptable, and responsive team, we must bring
together a wide range of viewpoints and skills. To create a team with
this range, we must include members from different disciplines, training
levels, technical backgrounds, and cultural contexts. Equally important
is creating the space and time needed to synthesize these diverse
perspectives, allowing the most resilient and high-quality outcomes
to emerge.<em>
,−





</em> When scientific challenges are tackled, we must embrace iteration,
maintain healthy skepticism, and rigorously scrutinize new findings.
Just as vital is establishing a common language to foster effective
communication and sustain momentum throughout the research process.
Ultimately, heterogeneous scientific environments not only amplify
the return on research investments but also accelerate discoveries
that advance the greater good of humanity.</p>
<p>Our lab’s
exploration of RNA biochemistry and biophysics
has provided us with case studies that illustrate the value of heterogeneity
and in turn has provided us with useful metaphors that we believe
can benefit science culture.<em>
−





</em> The lessons we have learned about the versatility of RNA interactions,
the feedback loops they employ in fine-tuning biological processes,
and the communication strategies they use to link diverse components
of protein-RNA systems are analogous to the advantages of heterogeneity
in science culture.</p>
<p>In the following sections, we will demonstrate
in parallel how
heterogeneous RNA systems rely on a core set of principles to optimize
biological outcomes and how these same principles can be applied to
research environments to enhance the overall scientific experience,
through repeating headings in the RNA Systems and Research Culture
sections. Here, we provide examples that are well-known to the scientific
community as well as those from our own area of RNA research to clearly
demonstrate the lessons from RNA research that can be adapted to build
a better and thriving research culture. Notably, this perspective
is not meant to serve as a comprehensive review of RNA biology but
rather to illustrate how the principles underlying RNA function can
serve as a metaphor for how the scientific community can optimize
the research enterprise.</p></section><section id="sec2"><h2 class="pmc_sec_title">2. Heterogeneous RNA Systems</h2>
<section id="sec2.1"><h3 class="pmc_sec_title">2.1. Versatility and Diversity Lead to Optimization</h3>
<p>According
to the central dogma of molecular biology, which is now
understood to be an incomplete model, genetic information was initially
believed to flow only in a single direction, from DNA to RNA to protein. Although this model correctly identified RNA
as a central regulator of life processes, it also oversimplified RNA’s
role, reducing it to that of a passive messenger that merely copies
and transmits genetic instructions. Likewise, before RNA components
were formally identified and classified, some noncoding regions of
RNA were thought to have no purpose. The
RNA revolution in contemporary research has subsequently revealed
that RNA is far more than just a decoder and that both coding and
noncoding regions actively participate in a wide array of biological
processes.<em>
,
</em> We are now familiar with an astounding
diversity of RNA biotypes and RNA regulatory mechanisms that together
play a role in nearly all aspects of cellular metabolism and adaptation.
RNA’s inherent diversity and versatility allow for optimization
across systems and in turn enhanced organismal fitness, while defects
in RNA regulation are often associated with disease.<em>
,
</em>
</p>
<p>The diversity and versatility of RNA molecules are reflected
in their multifunctional nature and the structural changes that facilitate
them. While the coding role of RNA facilitates the production of proteins
from DNA, noncoding functions of diverse types of RNA regulate various
aspects of cellular homeostasis including protein production.<em>
,
</em>
<a href="#fig1" class="usa-link">Figure 
</a> is a generalized
representation of several different origins of RNA heterogeneity and
a subset of their respective outcomes.</p>
<figure class="fig xbox font-sm" id="fig1"><h4 class="obj_head">1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371483_bg5c00114_0001.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/bef3/12371483/f11b7ef9cbcd/bg5c00114_0001.jpg" loading="lazy" id="gr1" height="675" width="695" alt="1"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Representations of the
RNA heterogeneity. RNA heterogeneity helps
to regulate various essential biological processes, such as cellular
differentiation, tissue/organ development, and response to environmental
stress. This figure illustrates several well-known sources of RNA
heterogeneity: (A) alternative splicing of pre-mRNA into different
mature mRNAs gives rise to different protein products; (B) RNA modifications
during stress conditions alter its stability, translation, or localization
in order to adapt to a stressor; (C) specific regulatory elements,
such as internal ribosome entry sites (IRES), within mRNAs that code
for stress response proteins, help cells to survive during environmental
stress conditions that interfere with the translation of the majority
of proteins. (D) Different levels of gene expression aid in cellular
differentiation and are responsible for their unique functions, (E)
different RNA conformations allow the binding of specific sets of
proteins, depending on cellular needs, and (F) RNA mutations can inhibit
existing interactions with proteins/metabolites or facilitate new
interactions with proteins/metabolites, to provide a survival advantage.
mRNA: mRNA, pre-mRNA: premature mRNA, IRES: internal ribosome entry
site.</p></figcaption></figure><p>At the molecular level, RNA adopts
dynamic conformations, forming
ensembles that govern interactions with proteins and other molecules.<em>
,,
</em> Chemical modifications on RNA
bases or its ribose sugar add yet another layer of complexity by altering
these structural ensembles, thereby modulating interactions with other
biological ligands.<em>
,
</em> RNA, in association with proteins,
also organizes itself into membraneless, phase-separated condensates,
creating microenvironments for specific biochemical reactions.<em>
−

</em> Adding to its intrigue, RNA exhibits cell- and tissue-specific expression
patterns to produce the proteins that are needed to carry out cell-
and tissue-specific functions.<em>
−



</em> Processes such as alternative splicing, post-transcriptional modifications,
changes in gene expression, and conformational dynamics between different
secondary and tertiary structures play critical roles in cellular
differentiation during development, immune responses, and adaptation
to environmental stress.<em>
,,−

</em>
</p>
<section id="sec2.1.1"><h4 class="pmc_sec_title">2.1.1. Example 1: RNA Modifications
in Response
to Environmental Stress Leads to Optimization</h4>
<p>RNA modifications
that occur in response to environmental stress illustrate one way
that RNA heterogeneity leads to optimization. In response to various
stress stimuli, modified RNA mediates the assembly of translation
factors, mRNAs, RNA-binding proteins (RBPs), and other proteins to
form membraneless cellular compartments called stress granules (SGs).<em>
,
</em> SGs play roles in the regulation of translation, cell signaling,
mRNA storage, and stabilization during stress (<a href="#fig2" class="usa-link">Figure 
</a>A). SGs quickly assemble upon exposure to
stress and then rapidly disperse after the stress subsides. SGs further contribute to optimization on an
organismal level by playing a role in innate immune responses to viral
infections. Recent studies of SG dynamics have suggested that the
regulated assembly of SGs, and in turn optimization, is absent in
several pathophysiological conditions, including viral infections,
cancer, and neurodegenerative diseases.
</p>
<figure class="fig xbox font-sm" id="fig2"><h5 class="obj_head">2.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371483_bg5c00114_0002.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/bef3/12371483/bde4fd524935/bg5c00114_0002.jpg" loading="lazy" id="gr2" height="485" width="687" alt="2"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Versatility
and diversity of RNA led to optimization. (A) RNA modifications
in response to environmental stress leads to optimization. This figure
illustrates an example where RNA gets m<sup>6</sup>A-modified in response
to environmental stress, which in turn mediates the assembly of translation
factors, mRNAs, RNA-binding proteins (RBPs), and other cellular proteins
to form membraneless cellular compartments called stress granules
(SGs). Assembly and disassembly of SGs are highly dynamic, rendering
diverse roles to optimize cellular functions during environmental
stress. (B) Mutations in viral RNA leads to resistance against antiviral
drugs. This figure illustrates an example where Enterovirus A71 evolves
to possess a pair of nucleotide mutations in stem-loop II (SLII) within
its 5′-UTR to gain resistance to an antiviral small molecule
drug (DMA-135). The mutated SLII retains binding affinity for the
positive regulatory protein hnRNPA1; however, DMA-135 forms a ternary
complex with wild-type SLII and the negative regulatory protein AUF1,
thereby preventing productive interactions with hnRNPA1. In contrast,
AUF1 cannot bind mutated SLII due to the loss of its binding epitope.</p></figcaption></figure></section><section id="sec2.1.2"><h4 class="pmc_sec_title">2.1.2. Example 2: Mutations
in Viral RNA Leads
to Resistance against Antiviral Drugs</h4>
<p>Viruses often exploit
the versatility of RNA structures and interactions to drive replication
and pathogenicity. The IRES located at
the 5′-end of Enterovirus A71 (EV-A71) RNA adopts a conserved
structure with 6 stem-loops that are unique to each other in their
conformations. These structures coordinate the recruitment of a set
of cellular proteins, which in turn regulate viral protein synthesis
and genome replication (<a href="#fig2" class="usa-link">Figure 
</a>B). Our research in collaboration
with other groups has discovered a small molecule (SM) that binds
to and perturbs the conformation of one of these stem-loops, thereby
inhibiting EV-A71 replication. This inhibition
was achieved because the SM induced a change in the stem-loop structure,
which selectively enhanced its binding affinity for a negative regulator
protein relative to that of the positive regulator protein of IRES-driven
translation. To understand the selective pressure that the SM induces,
we passaged the virus in culture iteratively. The virus developed
resistance to the SM through a pair of nucleotide mutations adjacent
to the SM binding site. These escape
mutations represent a fundamental biological strategy by which RNA
virusesincluding EV-A71leverage RNA heterogeneity
to enable efficient replication and long-term persistence (<a href="#fig2" class="usa-link">Figure 
</a>B).</p></section></section><section id="sec2.2"><h3 class="pmc_sec_title">2.2. Feedback Loops Lead to Optimization</h3>
<p>When the cost of
inaccuracy would prove to be intolerable, marked
heterogeneity of biological systemsoften in the form of feedback
loopsexists to optimize critical processes. Thus, many RNA-dependent systems employ feedback loops
that enable the up- or down-regulation of various cellular responses
to ensure the smooth functioning of an organism.</p>
<section id="sec2.2.1"><h4 class="pmc_sec_title">2.2.1. Example
1: Heterogeneity of miRNAs Regulates
Cellular Processes and Enables Adaptation via Feedback Loops</h4>
<p>The fine-tuning of gene expression through positive and negative
feedback loops employed by miRNAs illustrates one way in which RNA
heterogeneity contributes to the optimization of fundamental biological
processes. MicroRNAs (miRNAs) are a class of short RNAs (19–24
nucleotides) that are well-known to bind messenger RNA (mRNA) and
either initiate the degradation of mRNA or otherwise prevent downstream
translation, thereby reducing the level of production of a given protein
product. However, some miRNAs bind to specific regions of RNA to increase
the abundance of certain proteins. This
class of RNAs exhibits significant heterogeneity in both their nucleotide
sequences and their capabilities. Many hundreds of miRNAs have been
discovered so far, and it is estimated that together they are responsible
for regulating the proteins encoded by approximately 30% to 80% of
human genes.
</p>
<p>The feedback loops
used by miRNAs enable adaptation to changing conditions and enact
suitable responses to various stimuli. The miRNA-mediated feedback loops are involved not only in sensing
metabolites and regulating metabolic pathways but also in stress signaling
pathways, where miRNA acts as a signal modulator.<em>
,
</em> Disruption of these feedback loops is known to result in various
pathological conditions, including cancer, diabetes, cardiovascular,
neurodegenerative, and autoimmune disorders.
</p>
<p>Consider EV-A71: to date, several miRNAs (miR-296-5p, miR-23b,
and miR-16–5p) are implicated in the suppression of viral replication.
Out of them, miR-16–5p was demonstrated to be involved in two
feedback loop mechanisms that inhibit EV-A71 replication (<a href="#fig3" class="usa-link">Figure 
</a>A). Upon infection,
EV-A71 induces the level of miR-16-5p itself as well as the activation
of caspase that facilitates the processing of pri-miR-16-5p. The resultant
higher levels of miR-16-5p in turn activate caspase-3, leading to
cell apoptosis. Hence, miR-16-5p acts as a positive feedback regulator
in EV-A71-induced cell apoptosis. In addition, increased levels of
miR-16-5p suppress the expression of cell cycle regulator proteins
cyclin E1 and cyclin D1. This inhibits cell division, resulting in
the suppression of EV-A71 replication. In the latter process, miR-16-5p
acts as a negative feedback regulator in EV-A71 replication. Hence,
the host cell uses its elevated levels of miR-16-5p upon EV-A71 infection
in two feedback loops, against the same virus to inhibit its replication
and to abort the production and release of new viral particles. This
is a great example of how our cells utilize feedback loops to gain
protection against viral infections, thereby maintaining health/homeostasis.
</p>
<figure class="fig xbox font-sm" id="fig3"><h5 class="obj_head">3.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371483_bg5c00114_0003.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/bef3/12371483/3f79846c7a90/bg5c00114_0003.jpg" loading="lazy" id="gr3" height="512" width="742" alt="3"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Heterogeneous RNAs are involved in feedback loops that
lead to
optimization. (A) Heterogeneity of miRNAs regulates cellular processes
and enables adaptation via feedback loops. This figure illustrates
an example where increased levels of miRNA-16-5p upon EV-A71 infection
are involved in positive and negative feedback loops in order to provide
immunity against EV-A71 via inhibition of viral replication. (B) RNA
conformational heterogeneity regulates the translation of some genes
in bacteria via riboswitches. This figure illustrates the function
of the cobalamin (B<sub>12</sub>) riboswitch at the 5′-end
of the mRNA (<em>btuB</em>) encoding the cobalamin transport
protein. Sufficient levels of cobalamin in the cytoplasm results in
the binding of cobalamin to the riboswitch (translation ON), changing
its confirmation to have a kissing-loop (KL) interaction, thereby
sequestering theribosome-binding site (RBS). As a result, translation
of mRNA is turned OFF, preventing excess intake of Cobalamin.</p></figcaption></figure></section><section id="sec2.2.2"><h4 class="pmc_sec_title">2.2.2. Example 2: RNA Conformational
Heterogeneity
Regulates Translation of Some Messages via Riboswitches</h4>
<p>Another
example of RNA feedback loops is riboswitches. These folding elements
usually occur in the mRNAs of prokaryotes, eukaryotes, and archaea
and exhibit RNA folding dynamics that are related to the binding of
an aptamer to a ligand.<em>
−

</em> When the ligand is abundant,
the equilibrium is pushed toward the binding competent fold, which
usually overlaps with some part of the mRNA’s expression platform
(or, in the eukaryotic case, the pre-mRNA’s splicing platform).
This can either sequester or expose the elements of the expression
platform needed for translation initiation, which provides positive
or negative regulation of the gene product, respectively. Many riboswitch
aptamers bind ligands that are produced by the gene encoded in the
message that the riboswitch regulates, tightly integrating the feedback
loop in one message. A classic explicit example of this would be an
“off” or negative riboswitch that sequesters the ribosome
binding site upon ligand binding by having it take part in a kissing
loop interaction (<a href="#fig3" class="usa-link">Figure 
</a>B). For example, the cobalamin (B12) riboswitch works via
this mechanism. This tunes the genes
that synthesize cobalamin, which the riboswitch appears upstream of,
such that the level of cobalamin in the bacterial cell is kept relatively
constant and the factors that synthesize it are not wastefully overproduced.</p></section><section id="sec2.2.3"><h4 class="pmc_sec_title">2.2.3. Example 3: RNA Conformational Heterogeneity
Permits Concentration Window Modulation of Cellular Signals</h4>
<p>Post-transcriptional modulation of gene products can also occur via
5′ UTR conformational changes, exposing binding sites for regulatory
proteins. A concrete example of this comes from the interactions carbon
storage regulatory protein A (CsrA) has with one of its many targets,
the uxuB mRNA. The gene product is a <span class="font-variant-small-caps">d</span>-mannonate oxidoreductase
that uses NAD as a cofactor and allows bacteria to grow on fructuronate.<em>
,
</em> CsrA conventionally binds to a short RNA motif that matches the
Shine-Dalgarno sequence but can bind to many alternative locations
on RNAs. RNAs can occupy heterogeneous conformations that can then
expose or sequester these CsrA binding sites.
</p>
<p>In the case of uxuB, a CsrA binding site in the 5′
UTR and another within the first 100 nt of the coding sequence result
in a tuned interplay that serves as a band-pass filter on the amount
of gene product. At low concentrations
of CsrA, the ribosome binding site (RBS) is sequestered in the secondary
structure. As the protein binds to higher-affinity epitopes, cooperatively
causing the 5′ UTR to refold, exposing the RBS, and resulting
in more translation of uxuB. Because the RBS is a lower-affinity binding
site for CsrA, at higher concentrations of CsrA it is occluded, inhibiting
translation of uxuB. Through cooperative RNA–protein interactions,
the level of uxuB production is restricted to a range of CsrA concentrations
bracketed from above and below. This level setting prevents both over
and underproduction of this metabolic gene used during fructonalose-based
exponential phase growth in .</p></section></section><section id="sec2.3"><h3 class="pmc_sec_title">2.3. Communication Leads to Optimization</h3>
<p>Part of the strength of RNA heterogeneity can be traced to its role
in an interconnected, collaborative system, where it interacts with
proteins and other macromolecules and can mobilize information across
a range of compartments. Although a cell cannot possibly know what
new problems it is likely to encounter, through channels of communication
that link diverse components, cells can stay broadly prepared, coordinate,
and communicate information to solve unexpected problems when they
arise. Such communication involves a network of diverse RNA molecules
whose communication among themselves and with other RNA, proteins,
and metabolites leads to the optimization of an RNA-dependent system.
The ability of different RNAs to fold into unique conformations that
allow the binding of specific protein(s) drives RNA–protein
communication, while the ability to form different types of base pairing
drives RNA–RNA communication.</p>
<section id="sec2.3.1"><h4 class="pmc_sec_title">2.3.1. Example
1: Communication among Heterogeneous
RNAs Facilitates Optimized Protein Synthesis</h4>
<p>Protein synthesis,
which takes place in the ribosome, provides an illustration of one
way in which communication among heterogeneous RNAs within cells leads
to the optimization of a process. Ribosome contains four distinct
rRNAs (rRNAs), three in the large subunit, and one in the small subunit.
Each rRNA adopts unique structural features spanning the RNA that
permit precise interactions with ribosomal proteins to assemble the
functional eukaryotic ribosome (<a href="#fig4" class="usa-link">Figure 
</a>A).
</p>
<figure class="fig xbox font-sm" id="fig4"><h5 class="obj_head">4.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371483_bg5c00114_0004.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/bef3/12371483/14265a98add6/bg5c00114_0004.jpg" loading="lazy" id="gr4" height="473" width="684" alt="4"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Communication mediated
by heterogeneous RNAs leads to optimization.
(A) Communication among heterogeneous RNAs facilitates optimized protein
synthesis. This figure illustrates the heterogeneous rRNAs (rRNAs)
that adopt unique structural features and interact precisely with
ribosomal proteins (r-proteins) to assemble the functional eukaryotic
ribosome. The mRNAs are heterogeneous and communicate the unique order
of amino acids for proteins by copying the information encoded in
DNA. This process of protein synthesis is possible only because of
the vast heterogeneity among tRNAs, which have unique anticodon sequences
and can attach to only one specific amino acid. (B) Communication
mediated by heterogeneous viral RNA structures facilitates viral replication.
This figure illustrates how the heterogeneity within the six stem-loop
structures (SLI-VI) in the EV-A71 5′-UTR results in differential
regulation of viral RNA replication and translation via recruiting
unique sets of host proteins onto the cloverleaf structure and internal
ribosome entry site (IRES), respectively.</p></figcaption></figure><p>Ribosomes are responsible for attaching amino acids in the order
specified by mRNA to form proteins. All proteins are formed by different
combinations of 20 amino acids. The mRNAs themselves are heterogeneous
and communicate the unique order of amino acids for proteins by copying
the information encoded in DNA. This process of protein synthesis
is possible only because of the vast heterogeneity among tRNAs, which
can attach to only one specific amino acid. The communication between
the 3-nucleotide sequence (codon) in mRNA and the corresponding complementary
sequence (anticodon) in a specific tRNA molecule results in mRNA–tRNA
binding, followed by the attachment of the specific amino acid that
tRNA brought to the site (<a href="#fig4" class="usa-link">Figure 
</a>A). Most importantly, a tRNA attached to methionine
recognizes and binds to the start codon on mRNA, thereby marking the
accurate starting point for the ribosome for protein synthesis. Hence, the diversity of tRNA not only facilitates
the synthesis of a protein with precise sequence but also allows the
synthesis of any protein that is needed by the cell. Aberrant modifications
or splicing of tRNA can result in miscommunication, which in turn
results in the misincorporation of amino acids and the production
of inaccurate protein products. Perturbation in these critical forms
of communication has been implicated in many diseases, including cancer,
diabetes, and neurological and neurodevelopmental disorders.<em>
,
</em>
</p></section><section id="sec2.3.2"><h4 class="pmc_sec_title">2.3.2. Example 2: Communication Mediated by Heterogeneous
Virus- and Host-Derived RNAs Facilitate Viral Replication</h4>
<p>EV-A71 replication depends on coordinated interactions between various
RNA elements within its genome and host cellular RNAs to support viral
protein synthesis and RNA replication. Although the 5′-UTR
is a single RNA segment, it adopts six distinct stem-loop structures
(SLI-SLVI), each playing critical roles in these processes. The SLI
structure specifically facilitates viral RNA replication by recruiting
essential protein components. In contrast, stem-loop SLII-SLVI collectively
form the internal ribosome entry site (IRES), with each individual
structure interacting with a unique set of host proteins to differentially
regulate IRES-mediated (cap-independent) translation (<a href="#fig4" class="usa-link">Figure 
</a>B). Some of the recruited host
proteins enhance translation, while others inhibit it, enabling fine-tuned
regulation of viral protein synthesis. Additionally, both virus- and host-derived noncoding RNAs regulate
various stages of viral replication through multiple mechanisms. These
include binding to viral RNA to modulate protein synthesis, interacting
with host proteins to suppress antiviral defense pathways, and targeting
host genes or mRNAs to either promote or suppress the expression of
proteins that respectively support or restrict viral replication.<em>
−

</em>
</p>
<p>In both examples, exquisite communication of different RNAs
with proteins and RNA via the language of unique binding conformations
and base pairing is what allows for precise information transfer across
the layers in both viral and cellular biology. Thus, RNA is written
in the language of biology and will be translated to coordinate and/or
communicate among different components across biological systems.</p></section></section></section><section id="sec3"><h2 class="pmc_sec_title">3. Heterogeneous Research Culture</h2>
<section id="sec3.1"><h3 class="pmc_sec_title">3.1. Versatility and Diversity Lead to Optimization</h3>
<p>In much
the same way that the versatility and diversity of RNA
optimizes the response to a given condition, such as environmental
stress, a scientific team comprising members with diverse skills and
perspectives enhances the versatility of the team and increases the
likelihood of solving a particular problem.<em>
,,−




</em> Even opinions and perspectives that initially may seem irrelevant
sometimes lead to breakthroughs and more creative problem-solving.</p>
<p>In conventional research culture, for example, observations or
other input by members with less experience might be politely dismissednot
because their contributions lack scientific merit, but simply because
such members may lack extensive expertise in RNA biology. We’ve
found that such dismissals can represent missed opportunities to embrace
fresh perspectives and foster innovation. Inclusive scientific environments
foster divergent and creative thinking by cultivating constructive
cultures that value and support the contributions of each stakeholder,
regardless of hierarchy or experience.<em>
,−




</em> Optimized teams are grounded in a shared awareness that the scientific
community functions best as an interconnected ecosystem in which curiosity
and creativity can thrive.</p>
<section id="sec3.1.1"><h4 class="pmc_sec_title">3.1.1. Example 1: Accepting
the Perspectives of
Nonscientists without Judgment Leads to Versatility and Successful
Research Outcomes</h4>
<p>Versatility in scientific endeavors also
sometimes involves openness to acquiring valuable missing information
from an unexpected source outside the team. A member of our group
experienced this while preparing to perform a surgical transplant
on snails during his PhD research. The goal of the downstream experiments
was to develop a novel strategy for germline integration of exogenous
DNA in parasitic flatworms that had no stable genetic tools available.
The first step in this process was developing a surgical protocol
to infect snails via a transplant of cultured flatworm tissue. Such
surgery involves making a 2 mm opening in a brittle snail shell. After
several unsuccessful attempts using small-diameter drills in a hand-held
rotary power tool, he shared the frustration of this experiment with
his father, a trained machinist without academic training or experience
in biology. However, the scientist’s father was able to quickly
propose a solutionthe use of a bit capable of grinding slowly,
rather than cutting, which would preserve the integrity of the brittle
snail shell. This new tool advanced his team’s research and
reinforced the importance of remaining open to new perspectives and
contributions from individuals outside the academic scientific environment.</p>
<p>Great scientific discoveries often come when different disciplines
intersect, and some of the best insights arise when researchers step
outside their own field. Just as RNA structures and modifications
shift and adapt within different cellular environments to often change
their binding partners, science progresses when we embrace heterogeneous
perspectives, different ways of thinking, forging of new relationships,
and problem-solving approaches from various backgrounds.<em>
,,,,
</em>
</p></section></section><section id="sec3.2"><h3 class="pmc_sec_title">3.2. Feedback
Loops Lead to Optimization</h3>
<p>Much in the same way that RNA
heterogeneity provides mechanisms for
optimization by means of feedback loops, science culture should employ
mechanisms of self-regulation and self-correction to continually tune
its performance to its context. When obstacles or problems are encountered
during scientific investigations, scientists must select “among
a universe of potential adjustments” that can be made. They must determine which “among a universe
of threads that can be tightened or loosened” will be most
likely to “sustain the fabric” of an experiment or “reweave
it”.<em>
,,
</em> A heterogeneous team can be brought together to offer new insight,
increase the speed of correction, and foster greater team cohesion
that allows for more feedback to provided.<em>
,,,,
</em>
</p>
<section id="sec3.2.1"><h4 class="pmc_sec_title">3.2.1. Example 1: Feedback Loops
Involving a Heterogeneous
Team Improves Troubleshooting</h4>
<p>The troubleshooting process
during scientific research, in many ways, resembles a feedback loop
process. In traditional feedback loops, an irregularity in the signal
that leads to suboptimal functioning triggers corrective processes
within the system that adjust to improve performance in a cyclical
and continuous fashion. The product of the cycle becomes the input
back into the cycle until a steady state is reached. Although mediated
by human input, troubleshooting a scientific procedure is similar
to a feedback loop in that it is a cyclical process of identifying
problems in a suboptimally functioning system, and the product of
one step (correcting one variable) becomes part of the input to the
next step by changing how the procedure is conducted. If the suspected
error is changed but does not fix the problem, that change is incorporated
into the next round of testing until a “steady state”
is reached, i.e., the procedure works, and the “input”
of testing ideas can stop. Such troubleshooting also leverages heterogeneity
by incorporating ideas regarding adjustments from multiple sources
and from consultation with colleagues who have a variety of backgrounds,
experiences, and perspectives.<em>
,,,,
</em> When scientists work together and initiate a positive feedback loop,
it can create optimally functioning and thriving scientific communities
and accelerate the progress of scientific discovery. Implementing
positive feedback loops in daily interactions contributes to the creation
of productive research teams.</p></section><section id="sec3.2.2"><h4 class="pmc_sec_title">3.2.2. Example
2: Heterogeneous Research Teams
Foster Cultures That Create Positive Feedback Loops That Improve the
Experience of Science and Scientific Outputs</h4>
<p>Research environments
that value divergent views and foster spaces for constructive feedback
are more likely to produce research outputs that are rigorously vetted
internally, ultimately achieving a higher standard of excellence. Such a group is more likely to generate innovative
research questions that can lead to greater scientific discoveries.<em>
,−




</em> Unique experiences and ideas will provide multiple perspectives
to investigate a given research question. Exposure to divergent perspectives
among the group will help to eliminate potential biases in research
design or data interpretation, thereby increasing the accuracy and
reliability of the research findings. The impact of scientific discovery
becomes greater when it is relevant, well-communicated, and accessible
to a wider range of populations/communities. This attribute in scientific
research is more likely to be achieved by a group of scientists with
many different backgrounds.</p>
<p>In an optimally functioning laboratory,
peer review is not confined to the final stages but occurs throughout
the lifespan of a project, serving as an iterative feedback mechanism
that continually refines the quality and interpretation of the science.
To enable this, every member of the research team must feel safe to
contribute their perspectives, information must be shared transparently,
research silos must be dismantled, and hierarchical barriers based
on experience should be flattened. When these conditions are met,
emerging scientific concepts are sharpened and strengthened through
internal iteration before being disseminated to the broader scientific
community. The positive feedback loop existing within a heterogeneous
group improves the research outcomes in an iterative way during each
step of the process (generation of a research question, experimental
design, data analysis, and outreach). Hence, functional diversity
in research groups has the potential to create positive feedback loops
that gradually enhance scientific discoveries to better serve humanity.
</p>
<p>Over the years, our research team has
included individuals from
every stage of their academic journey. High school students have worked
alongside undergraduates, graduate students, postdoctoral fellows,
and senior scientists on major scientific projects. Nontraditional
students and returning community college scholars have also actively
contributed to the advancement of our research. This heterogeneity
has not only enriched our scientific efforts but also fostered organic,
continuous feedback loops. The diversity of experiences and perspectives
has strengthened the quality of our work, deepened our collective
understanding, and ultimately led to more rigorous and innovative
science. Moreover, many of these individuals have persisted in scientific
careers, in part due to a positive feedback culture that cultivated
confidence and reinforced the development of a scientific identity.</p></section></section><section id="sec3.3"><h3 class="pmc_sec_title">3.3. Communication Leads to Optimization</h3>
<p>One of the challenges in scientific research, especially in biology
and RNA biochemistry, is that such work often requires an in-depth
study of individual components within a biological system. While a
division of labor among members of a team is an essential part of
the process of scientific discovery, it can be easy for individual
team members to lose sight of larger project goals. It is essential
to link all the individual components of any project by means of a
form of communication that optimizes the process of scientific inquiry.<em>
,
</em>
</p>
<section id="sec3.3.1"><h4 class="pmc_sec_title">3.3.1. Example 1: Success of a Research Project
is Based on Proper Communication among the Diverse Members of the
Team As Well As to the Scientific Community</h4>
<p>The path toward
the preparation of a scientific manuscript is a common example of
a project involving the work of diverse members of our team, whose
efforts must be linked by an avenue of communication. The process
of curating data from individual experiments and compiling them to
draft a manuscript requires the compilation of several smaller contributions
into a larger cohesive whole. This journey for us often begins with
a team meeting, during which we discuss the goal of the paper we hope
to produce and determine the roles that individual authors will play
in collecting data and relaying information to other team members.
We then share updates, raw data, and early analyses of the data asynchronously
using messaging and project management tools.</p>
<p>Because our team
comprises biochemists, NMR experts, crystallographers, biologists,
and other members with diverse training and research experience, it
is necessary for us to write clearly, to define and explain as we
go, rather than presuming comprehension. It has also been critical
that we learn enough about what our lab mates are doing to understand
each othera process of meeting someone partway across the
difference in scientific experience that can exist between team members
in heterogeneous groups. This helps us draw better conclusions from
our findings because it brings the varying expertise and viewpoints
of the group members into play. Subsequently, we compile all contributions
and begin to draft a manuscript. This final stage of the process requires
several authors to synthesize the team’s findings and create
compelling language to appropriately articulate our findings and their
importance in the field of RNA research.</p></section><section id="sec3.3.2"><h4 class="pmc_sec_title">3.3.2. Example
2: Communication among Diverse Members
of a Team Drives Innovation and Better Research Outputs</h4>
<p>Inadvertently,
we recently gained interesting insight into our communication dynamics
after a series of meetings that featured discussions focused on the
topic of “allostery”, a mechanism of biochemical regulation
by which long-distance communication can occur within molecules. For
example, a single mutation of an amino acid in an RNA-binding protein
can significantly affect RNA–protein interactions, even though
the mutation and RNA-binding sites are located far apart from each
other. Our group has recently experienced an influx of new staff scientists
and postdocs with different areas of expertise who were just beginning
to establish methods of communicating with one another. Our Principal
Investigator broached the topic of “allostery” in RNA-binding
proteins, which led to one of our senior scientists developing a new
modeling analysis to characterize communication networks within RNA–protein
complexes and propose likely routes of communication from distant
mutations. The senior scientist’s work helped us not only to
start a new research project but also to better understand that “allostery”
is cooperation and communication across many small parts, including
parts that may not seem related to the function of interest. These
discussions helped us gain insight into our own dynamics, and it became
clear that we all have similar effects on each other, and we can gain
insight by communicating and seeking input even though we are focused
on a different aspect of a project and bring divergent perspectives
as part of our interpretation.</p></section></section></section><section id="sec4"><h2 class="pmc_sec_title">4. Conclusions
and Outlook</h2>
<p>Science historians have noted that by the early
twentieth century,
there was a significant shift in focus from individual scientists
(luminaries such as Galileo and Newton) to the collective achievements
of communities of scientists.<em>
,,
</em> Scientific facts with a high degree of accuracy were believed to
emerge most efficiently by means of evolutionary processes: incremental
transformations in thinking by scientists with diverse backgrounds
and viewpoints. This view of scientific
progress continues today, as scientists share data, evaluate evidence,
respond to criticism, and adjust their opinions within a larger scientific
community and frameworks provided by peer-reviewed journals, scientific
conferences, and educational programs.
</p>
<p>Just as the strength of RNA-mediated biological systems can
be
traced to the heterogeneity of RNA, the strength of scientific communities
and their findings frequently can be traced to the diversity of their
membersa form of heterogeneity that helps to optimize the
processes of scientific investigation in several important ways, including
those that we have outlined here (<a href="#fig5" class="usa-link">Figure 
</a>). Just as the versatility and diversity
of RNA molecules allow for multifunctionality, resilience, and enhanced
fitness, fostering versatility and diversity within scientific culture
enhances the discovery process by increasing flexibility, adaptability,
and resourcefulness of a scientific team. Just as RNA-mediated feedback
loops assist in upregulating or downregulating various processes in
response to stimuli and in turn help to optimize the functioning of
an organism, feedback loops in scientific research provide a system
for locating and correcting errors, as well as allowing for the iterative
development of ideas and the optimization of procedures. RNA’s
role as an accurate carrier of genetic information and therefore a
source of efficient communication throughout the cell is also instructive
for research teams. In the scientific community, researchers must
routinely link the efforts of separate subgroups devoted to specific
tasks by establishing methods for successfully communicating data,
procedures, and findings to optimize the process of scientific inquiry
and increase the chances of success.</p>
<figure class="fig xbox font-sm" id="fig5"><h3 class="obj_head">5.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371483_bg5c00114_0005.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/bef3/12371483/0bfd15c10584/bg5c00114_0005.jpg" loading="lazy" id="gr5" height="587" width="748" alt="5"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Heterogeneity within a system leads to
optimization. The heterogeneity
of RNA optimizes and increases the fitness of RNA-mediated biological
systems. Similarly, heterogeneity strengthens the scientific community
and optimizes the process of scientific investigation.</p></figcaption></figure><p>Homogeneous scientific communities, whose backgrounds and
assumptions
are more likely to be shared by all members, are more vulnerable to
the weaknesses that may be inherent in shared subjectivity.<em>
,
</em> The scientific conclusions of such communities are less likely to
benefit from the “transformative interrogation” characteristic
of heterogeneous communities.<em>
,
</em> In contrast, a heterogeneous
group of scientists is likely to minimize shared subjectivity and
correct individual biases and assumptions. Interactions among members of heterogeneous teams tend to be self-correcting
and therefore tend to optimize the function of the group.<em>
,
</em> In general, conclusions are more likely to be accurate when an expert
consensus emerges in a scientific community that is diverse and characterized
by ample opportunities for peer review and openness and responsiveness
to criticism. The divergent thinking and
multifunctionality of heterogeneous teams also enable them to solve
complex problems more efficiently and effectively.<em>
,,
</em>
</p>
<p>The strengths that characterize heterogeneous
scientific communities
have become increasingly important in recent years, as scientific
conclusions regarding a wide range of subjectsfrom climate
change to vaccine safetyhave attracted heightened skepticism.
Public distrust of the sciences often can be traced to suspicions
of undue influence or conflicts of interestas when scientific
conclusions emerge from a single, homogeneous entity that provides
funding and/or is focused on the promotion of its own interests. Heterogeneous scientific communities that can
demonstrate, with integrity and transparency, the incorporation of
diverse skillsets and backgrounds are more likely to encourage public
trust by arriving at accurate, objective, and unbiased conclusions.</p>
<p>We opened this perspective with the following evocative question:
Can our research cultures evolve toward greater innovation by embracing
the very principles of biological heterogeneity that we passionately
explore in our scientific investigations? We believe that the answer
is a resounding yes.</p>
<p>Since founding our research group in 2009,
we have intentionally
cultivated a team whose members represent a wide range of backgrounds,
career stages, intellectual perspectives, and technical expertise,
all united in the pursuit of uncovering the fundamental principles
of RNA biology. Our progress has been driven by an environment where
every voice is heard and ideas are sharpened through open dialogue,
constructive feedback, critical thinking, and the empowerment of divergent
viewpoints. We actively draw on the unique insights of each team member
to foster a sense of belonging, strengthen team dynamics, and propel
our science forward.</p>
<p>Equally essential to our human diversity
is our scientific approach:
we deliberately integrate complementary and orthogonal methodologies
to investigate RNA mechanisms with depth and rigor. These conditions
have created more than a productive laboratory: they have nurtured
a culture of collaboration, mutual growth, shared understanding, and
intellectual curiosity. They remind us that heterogeneity, in both
people and approaches, is not merely compatible with scientific excellence;
it is fundamental to achieving it.</p>
<p>Scientific advancement is,
at its core, a human endeavor. As such,
our ideological beliefs, cultural norms, and adherence to the status
quo inevitably shape the enterprise, determining not only how science
is conducted but also who benefits from its discoveries. It is time
for us as scientists to embody the very principles of biological heterogeneity
that we so passionately explore in our research. By embracing greater
heterogeneityof thought, background, experience, and approachwe
can transform the culture of science and cultivate a more vibrant,
inclusive, and innovative scientific community. In doing so, we not
only expand the boundaries of discovery but also enhance our ability
to address the most pressing challenges of our time and better serve
the needs of humanity.</p></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgments</h2>
<p>The authors would like to thank Susan Worley, an
independent science writer, for her assistance in the preparation
of this manuscript, as well as the other members of the Tolbert Lab:
Yuchen He, Barrington Henry, Agata Jacewicz, Arjun Narasimhan, and
Amarachi Onwuka. All figures in this paper, the graphical abstract
and <a href="#fig1" class="usa-link">Figures 
</a>–, were created by Sudeshi Abedeera using BioRender
(<a href="https://BioRender.com" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://BioRender.com</a>)</p></section><section id="notes1"><p>The authors
declare no competing financial interest.</p></section><section id="spn1"><p>Published as part of <em>ACS Bio &amp; Med Chem Au</em>
<a href="https://pubs.acs.org/curated-content?journal=abmcb8%26ref=feature" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">special issue</a> “Juneteenth 2025”.</p></section><section id="ref-list1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="ref-list1_sec2"><ol class="ref-list font-sm">
<li id="ref1">
<cite>

Oreskes, N.

Why Trust Science?, Princeton University Press, 2019.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=Why%20Trust%20Science?&amp;publication_year=2019&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref2">
<cite>

Kuhn, T. S.

The Structure of Scientific
Revolutions, University of Chicago Press, 1962.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=The%20Structure%20of%20Scientific%20Revolutions&amp;publication_year=1962&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref3">
<cite>
National
Research Council (US) Committee on Assessing Behavioral and Social
Science Research on Aging
A Strategy for Assessing
Science: Behavioral and Social Research on Aging, 
Feller, I.
; 
Stern, P.
C.
, Eds.; National
Academies Press (US): Washinton (DC), 2007; DOI: 10.17226/11788.</cite> [<a href="https://doi.org/10.17226/11788" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20669474/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?title=A%20Strategy%20for%20Assessing%20Science:%20Behavioral%20and%20Social%20Research%20on%20Aging&amp;publication_year=2007&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref4"><cite>
The Nobel Prize. Nobel Prize in
Physiology or Medicine
1978 Press release, <a href="https://www.nobelprize.org/prizes/medicine/1978/press-release" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.nobelprize.org/prizes/medicine/1978/press-release</a>. Accessed 21 April 2025.</cite></li>
<li id="ref5">
<cite>Bolotin A., Quinquis B., Sorokin A., Ehrlich S. D.. Clustered regularly
interspaced short palindrome repeats (CRISPRs) have spacers of extrachromosomal
origin. Microbiology. 2005;151(8):2551–2561. doi: 10.1099/mic.0.28048-0.</cite> [<a href="https://doi.org/10.1099/mic.0.28048-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16079334/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Microbiology&amp;title=Clustered%20regularly%20interspaced%20short%20palindrome%20repeats%20(CRISPRs)%20have%20spacers%20of%20extrachromosomal%20origin&amp;author=A.%20Bolotin&amp;author=B.%20Quinquis&amp;author=A.%20Sorokin&amp;author=S.%20D.%20Ehrlich&amp;volume=151&amp;issue=8&amp;publication_year=2005&amp;pages=2551-2561&amp;pmid=16079334&amp;doi=10.1099/mic.0.28048-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref6">
<cite>Barrangou R.. The roles
of CRISPR–Cas systems in adaptive immunity and beyond. Curr. Opin. Immunol. 2015;32:36–41. doi: 10.1016/j.coi.2014.12.008.</cite> [<a href="https://doi.org/10.1016/j.coi.2014.12.008" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25574773/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Curr.%20Opin.%20Immunol.&amp;title=The%20roles%20of%20CRISPR%E2%80%93Cas%20systems%20in%20adaptive%20immunity%20and%20beyond&amp;author=R.%20Barrangou&amp;volume=32&amp;publication_year=2015&amp;pages=36-41&amp;pmid=25574773&amp;doi=10.1016/j.coi.2014.12.008&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref7">
<cite>

Jackson, S. E.
; 
Joshi, A.
. 
Work Team Diversity. In APA Handbook of Industrial and Organizational Psychology. 
Zedeck, S.
, Ed.; American Psychological Association, pp. 651–686; 2011.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=APA%20Handbook%20of%20Industrial%20and%20Organizational%20Psychology&amp;publication_year=2011&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref8">
<cite>Xu H., Liu M., Bu Y., Sun S., Zhang Y., Zhang C., Acuna D. E., Gray S., Meyer E., Ding Y.. The impact
of heterogeneous shared leadership in scientific teams. Inf. Process. Manage. 2024;61(1):103542. doi: 10.1016/j.ipm.2023.103542.</cite> [<a href="https://doi.org/10.1016/j.ipm.2023.103542" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Inf.%20Process.%20Manage.&amp;title=The%20impact%20of%20heterogeneous%20shared%20leadership%20in%20scientific%20teams&amp;author=H.%20Xu&amp;author=M.%20Liu&amp;author=Y.%20Bu&amp;author=S.%20Sun&amp;author=Y.%20Zhang&amp;volume=61&amp;issue=1&amp;publication_year=2024&amp;pages=103542&amp;doi=10.1016/j.ipm.2023.103542&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref9">
<cite>AlShebli B. K., Rahwan T., Woon W. L.. The preeminence
of ethnic diversity
in scientific collaboration. Nat. Commun. 2018;9(1):5163. doi: 10.1038/s41467-018-07634-8.</cite> [<a href="https://doi.org/10.1038/s41467-018-07634-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6279741/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30514841/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Commun.&amp;title=The%20preeminence%20of%20ethnic%20diversity%20in%20scientific%20collaboration&amp;author=B.%20K.%20AlShebli&amp;author=T.%20Rahwan&amp;author=W.%20L.%20Woon&amp;volume=9&amp;issue=1&amp;publication_year=2018&amp;pages=5163&amp;pmid=30514841&amp;doi=10.1038/s41467-018-07634-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref10">
<cite>Ploetz R. C.. Panama
Disease: An Old Nemesis Rears Its Ugly Head: Part 1. The Beginnings
of the Banana Export Trades. Plant Health Prog. 2005;6(1):18. doi: 10.1094/PHP-2005-1221-01-RV.</cite> [<a href="https://doi.org/10.1094/PHP-2005-1221-01-RV" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Health%20Prog.&amp;title=Panama%20Disease:%20An%20Old%20Nemesis%20Rears%20Its%20Ugly%20Head:%20Part%201.%20The%20Beginnings%20of%20the%20Banana%20Export%20Trades&amp;author=R.%20C.%20Ploetz&amp;volume=6&amp;issue=1&amp;publication_year=2005&amp;pages=18&amp;doi=10.1094/PHP-2005-1221-01-RV&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref11">
<cite>Ploetz R.. Panama Disease:
An Old Nemesis Rears its Ugly Head – Part 2. The Cavendish
Era and Beyond. Plant Health Prog. 2006;7:36. doi: 10.1094/PHP-2006-0308-01-RV.</cite> [<a href="https://doi.org/10.1094/PHP-2006-0308-01-RV" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Health%20Prog.&amp;title=Panama%20Disease:%20An%20Old%20Nemesis%20Rears%20its%20Ugly%20Head%20%E2%80%93%20Part%202.%20The%20Cavendish%20Era%20and%20Beyond&amp;author=R.%20Ploetz&amp;volume=7&amp;publication_year=2006&amp;pages=36&amp;doi=10.1094/PHP-2006-0308-01-RV&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref12">
<cite>Phillips K. W., Liljenquist K. A., Neale M. A.. Is the Pain Worth the Gain? The Advantages
and Liabilities of Agreeing With Socially Distinct Newcomers. Pers. Soc. Psychol. Bull. 2009;35(3):336–350. doi: 10.1177/0146167208328062.</cite> [<a href="https://doi.org/10.1177/0146167208328062" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19098258/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Pers.%20Soc.%20Psychol.%20Bull.&amp;title=Is%20the%20Pain%20Worth%20the%20Gain?%20The%20Advantages%20and%20Liabilities%20of%20Agreeing%20With%20Socially%20Distinct%20Newcomers&amp;author=K.%20W.%20Phillips&amp;author=K.%20A.%20Liljenquist&amp;author=M.%20A.%20Neale&amp;volume=35&amp;issue=3&amp;publication_year=2009&amp;pages=336-350&amp;pmid=19098258&amp;doi=10.1177/0146167208328062&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref13">
<cite>Nathan M., Lee N.. Cultural Diversity,
Innovation, and Entrepreneurship: Firm-level
Evidence from London. Econ. Geogr. 2013;89(4):367–394. doi: 10.1111/ecge.12016.</cite> [<a href="https://doi.org/10.1111/ecge.12016" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Econ.%20Geogr.&amp;title=Cultural%20Diversity,%20Innovation,%20and%20Entrepreneurship:%20Firm-level%20Evidence%20from%20London&amp;author=M.%20Nathan&amp;author=N.%20Lee&amp;volume=89&amp;issue=4&amp;publication_year=2013&amp;pages=367-394&amp;doi=10.1111/ecge.12016&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref14">
<cite>Hall K. L., Vogel A. L., Huang G. C., Serrano K. J., Rice E. L., Tsakraklides S. P., Fiore S. M.. The science of team science: A review
of the empirical evidence and research gaps on collaboration in science. Am. Psychol. 2018;73(4):532–548. doi: 10.1037/amp0000319.</cite> [<a href="https://doi.org/10.1037/amp0000319" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29792466/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Am.%20Psychol.&amp;title=The%20science%20of%20team%20science:%20A%20review%20of%20the%20empirical%20evidence%20and%20research%20gaps%20on%20collaboration%20in%20science&amp;author=K.%20L.%20Hall&amp;author=A.%20L.%20Vogel&amp;author=G.%20C.%20Huang&amp;author=K.%20J.%20Serrano&amp;author=E.%20L.%20Rice&amp;volume=73&amp;issue=4&amp;publication_year=2018&amp;pages=532-548&amp;pmid=29792466&amp;doi=10.1037/amp0000319&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref15">
<cite>Albaharna M., D’Auria G., DeSmet A., Fifer A., Meijknecht L., Rubenstein K.. Go, teams: When teams get healthier, the whole organization
benefits. McKinsey Q. 2024;60:1–12.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=McKinsey%20Q.&amp;title=Go,%20teams:%20When%20teams%20get%20healthier,%20the%20whole%20organization%20benefits&amp;author=M.%20Albaharna&amp;author=G.%20D%E2%80%99Auria&amp;author=A.%20DeSmet&amp;author=A.%20Fifer&amp;author=L.%20Meijknecht&amp;volume=60&amp;publication_year=2024&amp;pages=1-12&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref16">
<cite>Yang Y., Tian T. Y., Woodruff T. K., Jones B. F., Uzzi B.. Gender-diverse
teams produce more novel and higher-impact scientific ideas. Proc. Int. Acad. Sci. 2022;119(36):e2200841119.  doi: 10.1073/pnas.2200841119.</cite> [<a href="https://doi.org/10.1073/pnas.2200841119" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9456721/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36037387/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proc.%20Int.%20Acad.%20Sci.&amp;title=Gender-diverse%20teams%20produce%20more%20novel%20and%20higher-impact%20scientific%20ideas&amp;author=Y.%20Yang&amp;author=T.%20Y.%20Tian&amp;author=T.%20K.%20Woodruff&amp;author=B.%20F.%20Jones&amp;author=B.%20Uzzi&amp;volume=119&amp;issue=36&amp;publication_year=2022&amp;pages=e2200841119&amp;pmid=36037387&amp;doi=10.1073/pnas.2200841119&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref17">
<cite>Stahl G. K., Maznevski M. L.. Unraveling
the effects of cultural diversity in teams:
A retrospective of research on multicultural work groups and an agenda
for future research. J. Int. Bus. Stud. 2021;52(1):4–22. doi: 10.1057/s41267-020-00389-9.</cite> [<a href="https://doi.org/10.1057/s41267-020-00389-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7812115/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33487775/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Int.%20Bus.%20Stud.&amp;title=Unraveling%20the%20effects%20of%20cultural%20diversity%20in%20teams:%20A%20retrospective%20of%20research%20on%20multicultural%20work%20groups%20and%20an%20agenda%20for%20future%20research&amp;author=G.%20K.%20Stahl&amp;author=M.%20L.%20Maznevski&amp;volume=52&amp;issue=1&amp;publication_year=2021&amp;pages=4-22&amp;pmid=33487775&amp;doi=10.1057/s41267-020-00389-9&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref18">
<cite>

Stengers, I.

Another Science is Possible:
A Manifesto for Slow Science, Polity Press, 2018.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=Another%20Science%20is%20Possible:%20A%20Manifesto%20for%20Slow%20Science&amp;publication_year=2018&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref19">
<cite>Abedeera S.
M., Davila-Calderon J., Haddad C., Henry B., King J., Penumutchu S., Tolbert B. S.. The Repurposing of Cellular Proteins
during Enterovirus A71 Infection. Viruses. 2024;16(1):75. doi: 10.3390/v16010075.</cite> [<a href="https://doi.org/10.3390/v16010075" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10821071/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38257775/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Viruses&amp;title=The%20Repurposing%20of%20Cellular%20Proteins%20during%20Enterovirus%20A71%20Infection&amp;author=S.%20M.%20Abedeera&amp;author=J.%20Davila-Calderon&amp;author=C.%20Haddad&amp;author=B.%20Henry&amp;author=J.%20King&amp;volume=16&amp;issue=1&amp;publication_year=2024&amp;pages=75&amp;pmid=38257775&amp;doi=10.3390/v16010075&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref20">
<cite>Davila-Calderon J., Patwardhan N. N., Chiu L. Y., Sugarman A., Cai Z., Penutmutchu S. R., Li M. L., Brewer G., Hargrove A. E., Tolbert B. S.. IRES-targeting
small molecule inhibits enterovirus
71 replication via allosteric stabilization of a ternary complex. Nat. Commun. 2020;11(1):4775. doi: 10.1038/s41467-020-18594-3.</cite> [<a href="https://doi.org/10.1038/s41467-020-18594-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7508794/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32963221/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Commun.&amp;title=IRES-targeting%20small%20molecule%20inhibits%20enterovirus%2071%20replication%20via%20allosteric%20stabilization%20of%20a%20ternary%20complex&amp;author=J.%20Davila-Calderon&amp;author=N.%20N.%20Patwardhan&amp;author=L.%20Y.%20Chiu&amp;author=A.%20Sugarman&amp;author=Z.%20Cai&amp;volume=11&amp;issue=1&amp;publication_year=2020&amp;pages=4775&amp;pmid=32963221&amp;doi=10.1038/s41467-020-18594-3&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref21">
<cite>Davila-Calderon J., Li M. L., Penumutchu S. R., Haddad C., Malcolm L., King J., Hargrove A. E., Brewer G., Tolbert B. S.. Enterovirus
evolution reveals the mechanism of an RNA-targeted antiviral and determinants
of viral replication. Sci. Adv. 2024;10(7):eadg3060.  doi: 10.1126/sciadv.adg3060.</cite> [<a href="https://doi.org/10.1126/sciadv.adg3060" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10871541/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38363831/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sci.%20Adv.&amp;title=Enterovirus%20evolution%20reveals%20the%20mechanism%20of%20an%20RNA-targeted%20antiviral%20and%20determinants%20of%20viral%20replication&amp;author=J.%20Davila-Calderon&amp;author=M.%20L.%20Li&amp;author=S.%20R.%20Penumutchu&amp;author=C.%20Haddad&amp;author=L.%20Malcolm&amp;volume=10&amp;issue=7&amp;publication_year=2024&amp;pages=eadg3060&amp;pmid=38363831&amp;doi=10.1126/sciadv.adg3060&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref22">
<cite>Jain N., Lin H.-C., Morgan C. E., Harris M. E., Tolbert B. S.. Rules of
RNA specificity of hnRNP A1 revealed by global and quantitative analysis
of its affinity distribution. Proc. Int. Acad.
Sci. 2017;114(9):2206–2211. doi: 10.1073/pnas.1616371114.</cite> [<a href="https://doi.org/10.1073/pnas.1616371114" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5338530/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28193894/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proc.%20Int.%20Acad.%20Sci.&amp;title=Rules%20of%20RNA%20specificity%20of%20hnRNP%20A1%20revealed%20by%20global%20and%20quantitative%20analysis%20of%20its%20affinity%20distribution&amp;author=N.%20Jain&amp;author=H.-C.%20Lin&amp;author=C.%20E.%20Morgan&amp;author=M.%20E.%20Harris&amp;author=B.%20S.%20Tolbert&amp;volume=114&amp;issue=9&amp;publication_year=2017&amp;pages=2206-2211&amp;pmid=28193894&amp;doi=10.1073/pnas.1616371114&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref23">
<cite>Levengood J. D., Potoyan D., Penumutchu S., Kumar A., Zhou Q., Wang Y., Hansen A. L., Kutluay S., Roche J., Tolbert B. S.. Thermodynamic coupling of the tandem RRM domains of
hnRNP A1 underlie its pleiotropic RNA binding functions. Sci. Adv. 2024;10(28):eadk6580.  doi: 10.1126/sciadv.adk6580.</cite> [<a href="https://doi.org/10.1126/sciadv.adk6580" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11235170/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38985864/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sci.%20Adv.&amp;title=Thermodynamic%20coupling%20of%20the%20tandem%20RRM%20domains%20of%20hnRNP%20A1%20underlie%20its%20pleiotropic%20RNA%20binding%20functions&amp;author=J.%20D.%20Levengood&amp;author=D.%20Potoyan&amp;author=S.%20Penumutchu&amp;author=A.%20Kumar&amp;author=Q.%20Zhou&amp;volume=10&amp;issue=28&amp;publication_year=2024&amp;pages=eadk6580&amp;pmid=38985864&amp;doi=10.1126/sciadv.adk6580&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref24">
<cite>Levengood J. D., Tolbert B. S.. Idiosyncrasies of
hnRNP A1-RNA recognition: Can binding
mode influence function. Semin. Cell Dev. Biol. 2019;86:150–161. doi: 10.1016/j.semcdb.2018.04.001.</cite> [<a href="https://doi.org/10.1016/j.semcdb.2018.04.001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6177329/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29625167/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Semin.%20Cell%20Dev.%20Biol.&amp;title=Idiosyncrasies%20of%20hnRNP%20A1-RNA%20recognition:%20Can%20binding%20mode%20influence%20function&amp;author=J.%20D.%20Levengood&amp;author=B.%20S.%20Tolbert&amp;volume=86&amp;publication_year=2019&amp;pages=150-161&amp;pmid=29625167&amp;doi=10.1016/j.semcdb.2018.04.001&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref25">
<cite>Tolbert M., Morgan C. E., Pollum M., Crespo-Hernández C. E., Li M.-L., Brewer G., Tolbert B. S.. HnRNP A1 Alters
the Structure of a Conserved Enterovirus IRES Domain to Stimulate
Viral Translation. J. Mol. Biol. 2017;429(19):2841–2858. doi: 10.1016/j.jmb.2017.06.007.</cite> [<a href="https://doi.org/10.1016/j.jmb.2017.06.007" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5610934/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28625847/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Mol.%20Biol.&amp;title=HnRNP%20A1%20Alters%20the%20Structure%20of%20a%20Conserved%20Enterovirus%20IRES%20Domain%20to%20Stimulate%20Viral%20Translation&amp;author=M.%20Tolbert&amp;author=C.%20E.%20Morgan&amp;author=M.%20Pollum&amp;author=C.%20E.%20Crespo-Hern%C3%A1ndez&amp;author=M.-L.%20Li&amp;volume=429&amp;issue=19&amp;publication_year=2017&amp;pages=2841-2858&amp;pmid=28625847&amp;doi=10.1016/j.jmb.2017.06.007&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref26">
<cite>Crick F.. Central Dogma
of Molecular Biology. Nature. 1970;227(5258):561–563. doi: 10.1038/227561a0.</cite> [<a href="https://doi.org/10.1038/227561a0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/4913914/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Central%20Dogma%20of%20Molecular%20Biology&amp;author=F.%20Crick&amp;volume=227&amp;issue=5258&amp;publication_year=1970&amp;pages=561-563&amp;pmid=4913914&amp;doi=10.1038/227561a0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref27">
<cite>Palazzo A. F., Lee E. S.. Non-coding RNA:
What is functional and what is junk? Front.
Genet. 2015;6:2. doi: 10.3389/fgene.2015.00002.</cite> [<a href="https://doi.org/10.3389/fgene.2015.00002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4306305/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25674102/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Front.%20Genet.&amp;title=Non-coding%20RNA:%20What%20is%20functional%20and%20what%20is%20junk?&amp;author=A.%20F.%20Palazzo&amp;author=E.%20S.%20Lee&amp;volume=6&amp;publication_year=2015&amp;pages=2&amp;pmid=25674102&amp;doi=10.3389/fgene.2015.00002&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref28">
<cite>Cech T. R.. The RNA
worlds in context. Cold Spring Harbor Perspect.
Biol. 2012;4(7):a006742. doi: 10.1101/cshperspect.a006742.</cite> [<a href="https://doi.org/10.1101/cshperspect.a006742" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3385955/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21441585/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cold%20Spring%20Harbor%20Perspect.%20Biol.&amp;title=The%20RNA%20worlds%20in%20context&amp;author=T.%20R.%20Cech&amp;volume=4&amp;issue=7&amp;publication_year=2012&amp;pages=a006742&amp;pmid=21441585&amp;doi=10.1101/cshperspect.a006742&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref29">
<cite>Haseltine W. A., Patarca R.. The RNA Revolution in the Central Molecular Biology
Dogma Evolution. Int. J. Mol. Sci. 2024;25(23):12695. doi: 10.3390/ijms252312695.</cite> [<a href="https://doi.org/10.3390/ijms252312695" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11641533/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39684407/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Int.%20J.%20Mol.%20Sci.&amp;title=The%20RNA%20Revolution%20in%20the%20Central%20Molecular%20Biology%20Dogma%20Evolution&amp;author=W.%20A.%20Haseltine&amp;author=R.%20Patarca&amp;volume=25&amp;issue=23&amp;publication_year=2024&amp;pages=12695&amp;pmid=39684407&amp;doi=10.3390/ijms252312695&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref30">
<cite>Licatalosi D. D., Darnell R. B.. RNA processing and
its regulation: Global insights
into biological networks. Nat. Rev. Genet. 2010;11(1):75–87. doi: 10.1038/nrg2673.</cite> [<a href="https://doi.org/10.1038/nrg2673" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3229837/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20019688/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Rev.%20Genet.&amp;title=RNA%20processing%20and%20its%20regulation:%20Global%20insights%20into%20biological%20networks&amp;author=D.%20D.%20Licatalosi&amp;author=R.%20B.%20Darnell&amp;volume=11&amp;issue=1&amp;publication_year=2010&amp;pages=75-87&amp;pmid=20019688&amp;doi=10.1038/nrg2673&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref31">
<cite>Cooper T.
A., Wan L., Dreyfuss G.. RNA and Disease. Cell. 2009;136(4):777–793. doi: 10.1016/j.cell.2009.02.011.</cite> [<a href="https://doi.org/10.1016/j.cell.2009.02.011" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2866189/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19239895/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=RNA%20and%20Disease&amp;author=T.%20A.%20Cooper&amp;author=L.%20Wan&amp;author=G.%20Dreyfuss&amp;volume=136&amp;issue=4&amp;publication_year=2009&amp;pages=777-793&amp;pmid=19239895&amp;doi=10.1016/j.cell.2009.02.011&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref32">
<cite>Bose R., Saleem I., Mustoe A. M.. Causes, functions, and therapeutic
possibilities of RNA secondary structure ensembles and alternative
states. Cell Chem. Biol. 2024;31(1):17–35. doi: 10.1016/j.chembiol.2023.12.010.</cite> [<a href="https://doi.org/10.1016/j.chembiol.2023.12.010" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10842484/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38199037/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Chem.%20Biol.&amp;title=Causes,%20functions,%20and%20therapeutic%20possibilities%20of%20RNA%20secondary%20structure%20ensembles%20and%20alternative%20states&amp;author=R.%20Bose&amp;author=I.%20Saleem&amp;author=A.%20M.%20Mustoe&amp;volume=31&amp;issue=1&amp;publication_year=2024&amp;pages=17-35&amp;pmid=38199037&amp;doi=10.1016/j.chembiol.2023.12.010&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref33">
<cite>Ganser L. R., Kelly M. L., Herschlag D., Al-Hashimi H. M.. The roles
of structural dynamics in the cellular functions of RNAs. Nat. Rev. Mol. Cell Biol. 2019;20(8):474–489. doi: 10.1038/s41580-019-0136-0.</cite> [<a href="https://doi.org/10.1038/s41580-019-0136-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7656661/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31182864/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Rev.%20Mol.%20Cell%20Biol.&amp;title=The%20roles%20of%20structural%20dynamics%20in%20the%20cellular%20functions%20of%20RNAs&amp;author=L.%20R.%20Ganser&amp;author=M.%20L.%20Kelly&amp;author=D.%20Herschlag&amp;author=H.%20M.%20Al-Hashimi&amp;volume=20&amp;issue=8&amp;publication_year=2019&amp;pages=474-489&amp;pmid=31182864&amp;doi=10.1038/s41580-019-0136-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref34">
<cite>Das G., Harikrishna S., Gore K. R.. Investigating the Effect of Chemical
Modifications on the Ribose Sugar Conformation, Watson–Crick
Base Pairing, and Intrastrand Stacking Interactions: A Theoretical
Approach. J. Phys. Chem. B. 2024;128(35):8313–8331. doi: 10.1021/acs.jpcb.4c02557.</cite> [<a href="https://doi.org/10.1021/acs.jpcb.4c02557" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39172066/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Phys.%20Chem.%20B&amp;title=Investigating%20the%20Effect%20of%20Chemical%20Modifications%20on%20the%20Ribose%20Sugar%20Conformation,%20Watson%E2%80%93Crick%20Base%20Pairing,%20and%20Intrastrand%20Stacking%20Interactions:%20A%20Theoretical%20Approach&amp;author=G.%20Das&amp;author=S.%20Harikrishna&amp;author=K.%20R.%20Gore&amp;volume=128&amp;issue=35&amp;publication_year=2024&amp;pages=8313-8331&amp;pmid=39172066&amp;doi=10.1021/acs.jpcb.4c02557&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref35">
<cite>Ontiveros R. J., Stoute J., Liu K. F.. The chemical
diversity of RNA modifications. Biochem. J. 2019;476(8):1227–1245. doi: 10.1042/BCJ20180445.</cite> [<a href="https://doi.org/10.1042/BCJ20180445" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9073955/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31028151/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biochem.%20J.&amp;title=The%20chemical%20diversity%20of%20RNA%20modifications&amp;author=R.%20J.%20Ontiveros&amp;author=J.%20Stoute&amp;author=K.%20F.%20Liu&amp;volume=476&amp;issue=8&amp;publication_year=2019&amp;pages=1227-1245&amp;pmid=31028151&amp;doi=10.1042/BCJ20180445&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref36">
<cite>Gomes E., Shorter J.. The molecular language of membraneless organelles. J. Biol. Chem. 2019;294(18):7115–7127. doi: 10.1074/jbc.TM118.001192.</cite> [<a href="https://doi.org/10.1074/jbc.TM118.001192" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6509512/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30045872/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Biol.%20Chem.&amp;title=The%20molecular%20language%20of%20membraneless%20organelles&amp;author=E.%20Gomes&amp;author=J.%20Shorter&amp;volume=294&amp;issue=18&amp;publication_year=2019&amp;pages=7115-7127&amp;pmid=30045872&amp;doi=10.1074/jbc.TM118.001192&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref37">
<cite>Rhine K., Vidaurre V., Myong S.. RNA Droplets. Annu. Rev. Biophys. 2020;49(1):247–265. doi: 10.1146/annurev-biophys-052118-115508.</cite> [<a href="https://doi.org/10.1146/annurev-biophys-052118-115508" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7695521/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32040349/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Annu.%20Rev.%20Biophys.&amp;title=RNA%20Droplets&amp;author=K.%20Rhine&amp;author=V.%20Vidaurre&amp;author=S.%20Myong&amp;volume=49&amp;issue=1&amp;publication_year=2020&amp;pages=247-265&amp;pmid=32040349&amp;doi=10.1146/annurev-biophys-052118-115508&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref38">
<cite>Ladouceur A.-M., Parmar B. S., Biedzinski S., Wall J., Tope S. G., Cohn D., Kim A., Soubry N., Reyes-Lamothe R., Weber S. C.. Clusters of bacterial
RNA polymerase are biomolecular
condensates that assemble through liquid–liquid phase separation. Proc. Int. Acad. Sci. 2020;117(31):18540–18549. doi: 10.1073/pnas.2005019117.</cite> [<a href="https://doi.org/10.1073/pnas.2005019117" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7414142/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32675239/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proc.%20Int.%20Acad.%20Sci.&amp;title=Clusters%20of%20bacterial%20RNA%20polymerase%20are%20biomolecular%20condensates%20that%20assemble%20through%20liquid%E2%80%93liquid%20phase%20separation&amp;author=A.-M.%20Ladouceur&amp;author=B.%20S.%20Parmar&amp;author=S.%20Biedzinski&amp;author=J.%20Wall&amp;author=S.%20G.%20Tope&amp;volume=117&amp;issue=31&amp;publication_year=2020&amp;pages=18540-18549&amp;pmid=32675239&amp;doi=10.1073/pnas.2005019117&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref39">
<cite>Dittmar K. A., Goodenbour J. M., Pan T.. Tissue-specific differences in human
transfer RNA expression. PLoS Genet. 2006;2(12):e221.  doi: 10.1371/journal.pgen.0020221.</cite> [<a href="https://doi.org/10.1371/journal.pgen.0020221" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1713254/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17194224/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20Genet.&amp;title=Tissue-specific%20differences%20in%20human%20transfer%20RNA%20expression&amp;author=K.%20A.%20Dittmar&amp;author=J.%20M.%20Goodenbour&amp;author=T.%20Pan&amp;volume=2&amp;issue=12&amp;publication_year=2006&amp;pages=e221&amp;pmid=17194224&amp;doi=10.1371/journal.pgen.0020221&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref40">
<cite>Wilkinson E., Cui Y.-H., He Y.-Y.. Roles of RNA Modifications in Diverse
Cellular Functions. Front. Cell Dev. Biol. 2022;10:10. doi: 10.3389/fcell.2022.828683.</cite> [<a href="https://doi.org/10.3389/fcell.2022.828683" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8957929/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35350378/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Front.%20Cell%20Dev.%20Biol.&amp;title=Roles%20of%20RNA%20Modifications%20in%20Diverse%20Cellular%20Functions&amp;author=E.%20Wilkinson&amp;author=Y.-H.%20Cui&amp;author=Y.-Y.%20He&amp;volume=10&amp;publication_year=2022&amp;pages=10&amp;pmid=35350378&amp;doi=10.3389/fcell.2022.828683&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref41">
<cite>Vasic R., Gao Y., Liu C., Halene S.. The Role of RNA Epigenetic Modification
in Normal and Malignant Hematopoiesis. Curr.
Stem Cell Rep. 2020;6(4):144–155. doi: 10.1007/s40778-020-00178-y.</cite> [<a href="https://doi.org/10.1007/s40778-020-00178-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7992056/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33777659/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Curr.%20Stem%20Cell%20Rep.&amp;title=The%20Role%20of%20RNA%20Epigenetic%20Modification%20in%20Normal%20and%20Malignant%20Hematopoiesis&amp;author=R.%20Vasic&amp;author=Y.%20Gao&amp;author=C.%20Liu&amp;author=S.%20Halene&amp;volume=6&amp;issue=4&amp;publication_year=2020&amp;pages=144-155&amp;pmid=33777659&amp;doi=10.1007/s40778-020-00178-y&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref42">
<cite>Misir S., Wu N., Yang B. B.. Specific expression
and functions of circular RNAs. Cell Death Differ. 2022;29(3):481–491. doi: 10.1038/s41418-022-00948-7.</cite> [<a href="https://doi.org/10.1038/s41418-022-00948-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8901656/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35169296/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Death%20Differ.&amp;title=Specific%20expression%20and%20functions%20of%20circular%20RNAs&amp;author=S.%20Misir&amp;author=N.%20Wu&amp;author=B.%20B.%20Yang&amp;volume=29&amp;issue=3&amp;publication_year=2022&amp;pages=481-491&amp;pmid=35169296&amp;doi=10.1038/s41418-022-00948-7&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref43">
<cite>Tack D. C., Su Z., Yu Y., Bevilacqua P. C., Assmann S. M.. Tissue-specific
changes in the RNA structurome mediate salinity response in Arabidopsis. RNA. 2020;26(4):492–511. doi: 10.1261/rna.072850.119.</cite> [<a href="https://doi.org/10.1261/rna.072850.119" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7075263/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31937672/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=RNA&amp;title=Tissue-specific%20changes%20in%20the%20RNA%20structurome%20mediate%20salinity%20response%20in%20Arabidopsis&amp;author=D.%20C.%20Tack&amp;author=Z.%20Su&amp;author=Y.%20Yu&amp;author=P.%20C.%20Bevilacqua&amp;author=S.%20M.%20Assmann&amp;volume=26&amp;issue=4&amp;publication_year=2020&amp;pages=492-511&amp;pmid=31937672&amp;doi=10.1261/rna.072850.119&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref44">
<cite>Tao Y., Zhang Q., Wang H., Yang X., Mu H.. Alternative
splicing and related RNA binding proteins in human health and disease. Signal Transduction Targeted Ther. 2024;9(1):26. doi: 10.1038/s41392-024-01734-2.</cite> [<a href="https://doi.org/10.1038/s41392-024-01734-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10835012/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38302461/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Signal%20Transduction%20Targeted%20Ther.&amp;title=Alternative%20splicing%20and%20related%20RNA%20binding%20proteins%20in%20human%20health%20and%20disease&amp;author=Y.%20Tao&amp;author=Q.%20Zhang&amp;author=H.%20Wang&amp;author=X.%20Yang&amp;author=H.%20Mu&amp;volume=9&amp;issue=1&amp;publication_year=2024&amp;pages=26&amp;pmid=38302461&amp;doi=10.1038/s41392-024-01734-2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref45">
<cite>Cui L., Ma R., Cai J., Guo C., Chen Z., Yao L., Wang Y., Fan R., Wang X., Shi Y.. RNA modifications:
Importance in immune cell biology and related diseases. Signal Transduction Targeted Ther. 2022;7(1):334. doi: 10.1038/s41392-022-01175-9.</cite> [<a href="https://doi.org/10.1038/s41392-022-01175-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9499983/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36138023/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Signal%20Transduction%20Targeted%20Ther.&amp;title=RNA%20modifications:%20Importance%20in%20immune%20cell%20biology%20and%20related%20diseases&amp;author=L.%20Cui&amp;author=R.%20Ma&amp;author=J.%20Cai&amp;author=C.%20Guo&amp;author=Z.%20Chen&amp;volume=7&amp;issue=1&amp;publication_year=2022&amp;pages=334&amp;pmid=36138023&amp;doi=10.1038/s41392-022-01175-9&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref46">
<cite>Wilkinson E., Cui Y. H., He Y. Y.. Context-Dependent
Roles of RNA Modifications
in Stress Responses and Diseases. Int. J. Mol.
Sci. 2021;22(4):1949. doi: 10.3390/ijms22041949.</cite> [<a href="https://doi.org/10.3390/ijms22041949" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7920320/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33669361/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Int.%20J.%20Mol.%20Sci.&amp;title=Context-Dependent%20Roles%20of%20RNA%20Modifications%20in%20Stress%20Responses%20and%20Diseases&amp;author=E.%20Wilkinson&amp;author=Y.%20H.%20Cui&amp;author=Y.%20Y.%20He&amp;volume=22&amp;issue=4&amp;publication_year=2021&amp;pages=1949&amp;pmid=33669361&amp;doi=10.3390/ijms22041949&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref47">
<cite>Fu Y., Zhuang X.. m6A-binding YTHDF proteins
promote stress granule formation. Nat. Chem.
Biol. 2020;16(9):955–963. doi: 10.1038/s41589-020-0524-y.</cite> [<a href="https://doi.org/10.1038/s41589-020-0524-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7442727/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32451507/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Chem.%20Biol.&amp;title=m6A-binding%20YTHDF%20proteins%20promote%20stress%20granule%20formation&amp;author=Y.%20Fu&amp;author=X.%20Zhuang&amp;volume=16&amp;issue=9&amp;publication_year=2020&amp;pages=955-963&amp;pmid=32451507&amp;doi=10.1038/s41589-020-0524-y&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref48">
<cite>Ries R. J., Zaccara S., Klein P., Olarerin-George A., Namkoong S., Pickering B. F., Patil D. P., Kwak H., Lee J. H., Jaffrey S. R.. m6A enhances the phase separation
potential of mRNA. Nature. 2019;571(7765):424–428. doi: 10.1038/s41586-019-1374-1.</cite> [<a href="https://doi.org/10.1038/s41586-019-1374-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6662915/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31292544/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=m6A%20enhances%20the%20phase%20separation%20potential%20of%20mRNA&amp;author=R.%20J.%20Ries&amp;author=S.%20Zaccara&amp;author=P.%20Klein&amp;author=A.%20Olarerin-George&amp;author=S.%20Namkoong&amp;volume=571&amp;issue=7765&amp;publication_year=2019&amp;pages=424-428&amp;pmid=31292544&amp;doi=10.1038/s41586-019-1374-1&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref49">
<cite>Wang J., Gan Y., Cao J., Dong X., Ouyang W.. Pathophysiology of
stress granules: An emerging link to diseases (Review) Int. J. Mol. Med. 2022;49(4):44. doi: 10.3892/ijmm.2022.5099.</cite> [<a href="https://doi.org/10.3892/ijmm.2022.5099" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8846937/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35137915/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Int.%20J.%20Mol.%20Med.&amp;title=Pathophysiology%20of%20stress%20granules:%20An%20emerging%20link%20to%20diseases%20(Review)&amp;author=J.%20Wang&amp;author=Y.%20Gan&amp;author=J.%20Cao&amp;author=X.%20Dong&amp;author=W.%20Ouyang&amp;volume=49&amp;issue=4&amp;publication_year=2022&amp;pages=44&amp;pmid=35137915&amp;doi=10.3892/ijmm.2022.5099&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref50">
<cite>van
Breugel M., Bongers F., Norden N., Meave J., Amissah L., Chanthorn W., Chazdon R., Craven D., Farrior C., Hall J. S.. et al. Feedback loops drive
ecological succession; towards a unified conceptual framework. Biol. Rev. 2024;99(3):928–949. doi: 10.1111/brv.13051.</cite> [<a href="https://doi.org/10.1111/brv.13051" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38226776/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biol.%20Rev.&amp;title=Feedback%20loops%20drive%20ecological%20succession;%20towards%20a%20unified%20conceptual%20framework&amp;author=M.%20van%20Breugel&amp;author=F.%20Bongers&amp;author=N.%20Norden&amp;author=J.%20Meave&amp;author=L.%20Amissah&amp;volume=99&amp;issue=3&amp;publication_year=2024&amp;pages=928-949&amp;pmid=38226776&amp;doi=10.1111/brv.13051&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref51">
<cite>Zheng C., Zheng Z., Sun J., Zhang Y., Wei C., Ke X., Liu Y., Deng L., Wang H.. MiR-16–5p mediates
a positive feedback loop in EV71-induced apoptosis and suppresses
virus replication. Sci. Rep. 2017;7(1):16422. doi: 10.1038/s41598-017-16616-7.</cite> [<a href="https://doi.org/10.1038/s41598-017-16616-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5703983/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29180670/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sci.%20Rep.&amp;title=MiR-16%E2%80%935p%20mediates%20a%20positive%20feedback%20loop%20in%20EV71-induced%20apoptosis%20and%20suppresses%20virus%20replication&amp;author=C.%20Zheng&amp;author=Z.%20Zheng&amp;author=J.%20Sun&amp;author=Y.%20Zhang&amp;author=C.%20Wei&amp;volume=7&amp;issue=1&amp;publication_year=2017&amp;pages=16422&amp;pmid=29180670&amp;doi=10.1038/s41598-017-16616-7&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref52">
<cite>Almeida M. I., Reis R. M., Calin G. A.. MicroRNA
history: Discovery, recent
applications, and next frontiers. Mutat. Res. 2011;717(1):1–8. doi: 10.1016/j.mrfmmm.2011.03.009.</cite> [<a href="https://doi.org/10.1016/j.mrfmmm.2011.03.009" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21458467/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mutat.%20Res.&amp;title=MicroRNA%20history:%20Discovery,%20recent%20applications,%20and%20next%20frontiers&amp;author=M.%20I.%20Almeida&amp;author=R.%20M.%20Reis&amp;author=G.%20A.%20Calin&amp;volume=717&amp;issue=1&amp;publication_year=2011&amp;pages=1-8&amp;pmid=21458467&amp;doi=10.1016/j.mrfmmm.2011.03.009&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref53">
<cite>O’Brien J., Hayder H., Zayed Y., Peng C.. Overview of MicroRNA
Biogenesis, Mechanisms of Actions, and Circulation. Front. Endocrinol. 2018;9:402. doi: 10.3389/fendo.2018.00402.</cite> [<a href="https://doi.org/10.3389/fendo.2018.00402" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6085463/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30123182/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Front.%20Endocrinol.&amp;title=Overview%20of%20MicroRNA%20Biogenesis,%20Mechanisms%20of%20Actions,%20and%20Circulation&amp;author=J.%20O%E2%80%99Brien&amp;author=H.%20Hayder&amp;author=Y.%20Zayed&amp;author=C.%20Peng&amp;volume=9&amp;publication_year=2018&amp;pages=402&amp;pmid=30123182&amp;doi=10.3389/fendo.2018.00402&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref54">
<cite>Rottiers V., Obad S., Petri A., McGarrah R., Lindholm M. W., Black J. C., Sinha S., Goody R. J., Lawrence M. S., deLemos A. S.. et al. Pharmacological inhibition of a microRNA family
in nonhuman primates by a seed-targeting 8-mer antimiR. Sci. Transl. Med. 2013;5(212):212ra162. doi: 10.1126/scitranslmed.3006840.</cite> [<a href="https://doi.org/10.1126/scitranslmed.3006840" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4033576/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24259050/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sci.%20Transl.%20Med.&amp;title=Pharmacological%20inhibition%20of%20a%20microRNA%20family%20in%20nonhuman%20primates%20by%20a%20seed-targeting%208-mer%20antimiR&amp;author=V.%20Rottiers&amp;author=S.%20Obad&amp;author=A.%20Petri&amp;author=R.%20McGarrah&amp;author=M.%20W.%20Lindholm&amp;volume=5&amp;issue=212&amp;publication_year=2013&amp;pages=212ra162&amp;pmid=24259050&amp;doi=10.1126/scitranslmed.3006840&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref55">
<cite>Mendell J. T., Olson E. N.. MicroRNAs in stress signaling and
human disease. Cell. 2012;148(6):1172–1187. doi: 10.1016/j.cell.2012.02.005.</cite> [<a href="https://doi.org/10.1016/j.cell.2012.02.005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3308137/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22424228/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=MicroRNAs%20in%20stress%20signaling%20and%20human%20disease&amp;author=J.%20T.%20Mendell&amp;author=E.%20N.%20Olson&amp;volume=148&amp;issue=6&amp;publication_year=2012&amp;pages=1172-1187&amp;pmid=22424228&amp;doi=10.1016/j.cell.2012.02.005&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref56">
<cite>Ha T. Y.. The Role
of MicroRNAs in Regulatory T Cells and in the Immune Response. Immune Netw. 2011;11(1):11–41. doi: 10.4110/in.2011.11.1.11.</cite> [<a href="https://doi.org/10.4110/in.2011.11.1.11" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3072673/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21494372/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Immune%20Netw.&amp;title=The%20Role%20of%20MicroRNAs%20in%20Regulatory%20T%20Cells%20and%20in%20the%20Immune%20Response&amp;author=T.%20Y.%20Ha&amp;volume=11&amp;issue=1&amp;publication_year=2011&amp;pages=11-41&amp;pmid=21494372&amp;doi=10.4110/in.2011.11.1.11&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref57">
<cite>Breaker R. R.. Riboswitches
and Translation Control. Cold Spring Harbor
Perspect. Biol. 2018;10(11):a032797. doi: 10.1101/cshperspect.a032797.</cite> [<a href="https://doi.org/10.1101/cshperspect.a032797" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6211393/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29844057/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cold%20Spring%20Harbor%20Perspect.%20Biol.&amp;title=Riboswitches%20and%20Translation%20Control&amp;author=R.%20R.%20Breaker&amp;volume=10&amp;issue=11&amp;publication_year=2018&amp;pages=a032797&amp;pmid=29844057&amp;doi=10.1101/cshperspect.a032797&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref58">
<cite>Serganov A., Nudler E.. A Decade of Riboswitches. Cell. 2013;152(1):17–24. doi: 10.1016/j.cell.2012.12.024.</cite> [<a href="https://doi.org/10.1016/j.cell.2012.12.024" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4215550/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23332744/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=A%20Decade%20of%20Riboswitches&amp;author=A.%20Serganov&amp;author=E.%20Nudler&amp;volume=152&amp;issue=1&amp;publication_year=2013&amp;pages=17-24&amp;pmid=23332744&amp;doi=10.1016/j.cell.2012.12.024&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref59">
<cite>Speed M. C., Burkhart B. W., Picking J. W., Santangelo T. J.. An Archaeal
Fluoride-Responsive Riboswitch Provides an Inducible Expression System
for Hyperthermophiles. Appl. Environ. Microbiol. 2018;84(7):e02306-17.  doi: 10.1128/AEM.02306-17.</cite> [<a href="https://doi.org/10.1128/AEM.02306-17" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5861836/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29352088/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Appl.%20Environ.%20Microbiol.&amp;title=An%20Archaeal%20Fluoride-Responsive%20Riboswitch%20Provides%20an%20Inducible%20Expression%20System%20for%20Hyperthermophiles&amp;author=M.%20C.%20Speed&amp;author=B.%20W.%20Burkhart&amp;author=J.%20W.%20Picking&amp;author=T.%20J.%20Santangelo&amp;volume=84&amp;issue=7&amp;publication_year=2018&amp;pages=e02306-17&amp;pmid=29352088&amp;doi=10.1128/AEM.02306-17&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref60">
<cite>Johnson J. E. Jr, Reyes F. E., Polaski J. T., Batey R. T.. B12 cofactors
directly stabilize an mRNA regulatory switch. Nature. 2012;492(7427):133–137. doi: 10.1038/nature11607.</cite> [<a href="https://doi.org/10.1038/nature11607" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3518761/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23064232/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=B12%20cofactors%20directly%20stabilize%20an%20mRNA%20regulatory%20switch&amp;author=J.%20E.%20Johnson&amp;author=F.%20E.%20Reyes&amp;author=J.%20T.%20Polaski&amp;author=R.%20T.%20Batey&amp;volume=492&amp;issue=7427&amp;publication_year=2012&amp;pages=133-137&amp;pmid=23064232&amp;doi=10.1038/nature11607&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref61">
<cite>Romeo T., Vakulskas C. A., Babitzke P.. Post-transcriptional regulation on
a global scale: Form and function of Csr/Rsm systems. Environ. Microbiol. 2013;15(2):313–324. doi: 10.1111/j.1462-2920.2012.02794.x.</cite> [<a href="https://doi.org/10.1111/j.1462-2920.2012.02794.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3443267/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22672726/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Environ.%20Microbiol.&amp;title=Post-transcriptional%20regulation%20on%20a%20global%20scale:%20Form%20and%20function%20of%20Csr/Rsm%20systems&amp;author=T.%20Romeo&amp;author=C.%20A.%20Vakulskas&amp;author=P.%20Babitzke&amp;volume=15&amp;issue=2&amp;publication_year=2013&amp;pages=313-324&amp;pmid=22672726&amp;doi=10.1111/j.1462-2920.2012.02794.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref62">
<cite>Hugouvieux-Cotte-Pattat N., Robert-Baudouy J.. Regulation
of Expression of the uxu Operon and of the
uxuR Regulatory Gene in Escherichia coli K12. Microbiology. 1983;129(11):3345–3353. doi: 10.1099/00221287-129-11-3345.</cite> [<a href="https://doi.org/10.1099/00221287-129-11-3345" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/6420508/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Microbiology&amp;title=Regulation%20of%20Expression%20of%20the%20uxu%20Operon%20and%20of%20the%20uxuR%20Regulatory%20Gene%20in%20Escherichia%20coli%20K12&amp;author=N.%20Hugouvieux-Cotte-Pattat&amp;author=J.%20Robert-Baudouy&amp;volume=129&amp;issue=11&amp;publication_year=1983&amp;pages=3345-3353&amp;pmid=6420508&amp;doi=10.1099/00221287-129-11-3345&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref63">
<cite>Leistra A. N., Gelderman G., Sowa S. W., Moon-Walker A., Salis H. M., Contreras L. M.. A Canonical
Biophysical Model of
the CsrA Global Regulator Suggests Flexible Regulator-Target Interactions. Sci. Rep. 2018;8(1):9892. doi: 10.1038/s41598-018-27474-2.</cite> [<a href="https://doi.org/10.1038/s41598-018-27474-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6028588/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29967470/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sci.%20Rep.&amp;title=A%20Canonical%20Biophysical%20Model%20of%20the%20CsrA%20Global%20Regulator%20Suggests%20Flexible%20Regulator-Target%20Interactions&amp;author=A.%20N.%20Leistra&amp;author=G.%20Gelderman&amp;author=S.%20W.%20Sowa&amp;author=A.%20Moon-Walker&amp;author=H.%20M.%20Salis&amp;volume=8&amp;issue=1&amp;publication_year=2018&amp;pages=9892&amp;pmid=29967470&amp;doi=10.1038/s41598-018-27474-2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref64">
<cite>Rojano-Nisimura A. M., Simmons T. R., Lukasiewicz A. J., Buchser R., Ruzek J. S., Avila J. L., Contreras L. M.. Concentration-Dependent
CsrA Regulation
of the uxuB Transcript Leads to Development of a Post-Transcriptional
Bandpass Filter. ACS Synth. Biol. 2025;14(4):1084–1098. doi: 10.1021/acssynbio.4c00668.</cite> [<a href="https://doi.org/10.1021/acssynbio.4c00668" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40202123/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=ACS%20Synth.%20Biol.&amp;title=Concentration-Dependent%20CsrA%20Regulation%20of%20the%20uxuB%20Transcript%20Leads%20to%20Development%20of%20a%20Post-Transcriptional%20Bandpass%20Filter&amp;author=A.%20M.%20Rojano-Nisimura&amp;author=T.%20R.%20Simmons&amp;author=A.%20J.%20Lukasiewicz&amp;author=R.%20Buchser&amp;author=J.%20S.%20Ruzek&amp;volume=14&amp;issue=4&amp;publication_year=2025&amp;pages=1084-1098&amp;pmid=40202123&amp;doi=10.1021/acssynbio.4c00668&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref65">
<cite>Fromont-Racine M., Senger B., Saveanu C., Fasiolo F.. Ribosome assembly in
eukaryotes. Gene. 2003;313:17–42. doi: 10.1016/S0378-1119(03)00629-2.</cite> [<a href="https://doi.org/10.1016/S0378-1119(03)00629-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12957375/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Gene&amp;title=Ribosome%20assembly%20in%20eukaryotes&amp;author=M.%20Fromont-Racine&amp;author=B.%20Senger&amp;author=C.%20Saveanu&amp;author=F.%20Fasiolo&amp;volume=313&amp;publication_year=2003&amp;pages=17-42&amp;pmid=12957375&amp;doi=10.1016/S0378-1119(03)00629-2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref66">
<cite>

Cooper, G. M.

The Cell: A Molecular
Approach, 2nd ed.; Sinauer
Associates, 2000.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=The%20Cell:%20A%20Molecular%20Approach&amp;publication_year=2000&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref67">
<cite>Orellana E. A., Siegal E., Gregory R. I.. tRNA dysregulation and disease. Nat. Rev. Genet. 2022;23(11):651–664. doi: 10.1038/s41576-022-00501-9.</cite> [<a href="https://doi.org/10.1038/s41576-022-00501-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11170316/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35681060/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Rev.%20Genet.&amp;title=tRNA%20dysregulation%20and%20disease&amp;author=E.%20A.%20Orellana&amp;author=E.%20Siegal&amp;author=R.%20I.%20Gregory&amp;volume=23&amp;issue=11&amp;publication_year=2022&amp;pages=651-664&amp;pmid=35681060&amp;doi=10.1038/s41576-022-00501-9&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref68">
<cite>Abbott J. A., Francklyn C. S., Robey-Bond S. M.. Transfer RNA and human disease. Front. Genet. 2014;5:158. doi: 10.3389/fgene.2014.00158.</cite> [<a href="https://doi.org/10.3389/fgene.2014.00158" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4042891/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24917879/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Front.%20Genet.&amp;title=Transfer%20RNA%20and%20human%20disease&amp;author=J.%20A.%20Abbott&amp;author=C.%20S.%20Francklyn&amp;author=S.%20M.%20Robey-Bond&amp;volume=5&amp;publication_year=2014&amp;pages=158&amp;pmid=24917879&amp;doi=10.3389/fgene.2014.00158&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref69">
<cite>Tycowski K. T., Guo Y. E., Lee N., Moss W. N., Vallery T. K., Xie M., Steitz J. A.. Viral noncoding
RNAs: More surprises. Genes Dev. 2015;29(6):567–584. doi: 10.1101/gad.259077.115.</cite> [<a href="https://doi.org/10.1101/gad.259077.115" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4378190/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25792595/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genes%20Dev.&amp;title=Viral%20noncoding%20RNAs:%20More%20surprises&amp;author=K.%20T.%20Tycowski&amp;author=Y.%20E.%20Guo&amp;author=N.%20Lee&amp;author=W.%20N.%20Moss&amp;author=T.%20K.%20Vallery&amp;volume=29&amp;issue=6&amp;publication_year=2015&amp;pages=567-584&amp;pmid=25792595&amp;doi=10.1101/gad.259077.115&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref70">
<cite>Cao Y., Wu J., Hu Y., Chai Y., Song J., Duan J., Zhang S., Xu X.. Virus-induced lncRNA-BTX allows viral
replication by regulating intracellular translocation of DHX9 and
ILF3 to induce innate escape. Cell Rep. 2023;42(10):113262. doi: 10.1016/j.celrep.2023.113262.</cite> [<a href="https://doi.org/10.1016/j.celrep.2023.113262" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37864796/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Rep.&amp;title=Virus-induced%20lncRNA-BTX%20allows%20viral%20replication%20by%20regulating%20intracellular%20translocation%20of%20DHX9%20and%20ILF3%20to%20induce%20innate%20escape&amp;author=Y.%20Cao&amp;author=J.%20Wu&amp;author=Y.%20Hu&amp;author=Y.%20Chai&amp;author=J.%20Song&amp;volume=42&amp;issue=10&amp;publication_year=2023&amp;pages=113262&amp;pmid=37864796&amp;doi=10.1016/j.celrep.2023.113262&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref71">
<cite>Gallo A., Bulati M., Miceli V., Amodio N., Conaldi P. G.. Non-Coding
RNAs: Strategy for Viruses’ Offensive. Non-Coding RNA. 2020;6(3):38. doi: 10.3390/ncrna6030038.</cite> [<a href="https://doi.org/10.3390/ncrna6030038" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7549346/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32927786/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Non-Coding%20RNA&amp;title=Non-Coding%20RNAs:%20Strategy%20for%20Viruses%E2%80%99%20Offensive&amp;author=A.%20Gallo&amp;author=M.%20Bulati&amp;author=V.%20Miceli&amp;author=N.%20Amodio&amp;author=P.%20G.%20Conaldi&amp;volume=6&amp;issue=3&amp;publication_year=2020&amp;pages=38&amp;pmid=32927786&amp;doi=10.3390/ncrna6030038&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref72">
<cite>

Zammito, J. H.

A Nice Derangement
of Epistemes: Post-positivism in the Study of Science from Quine to
Latour, University of Chicago Press, 2004.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=A%20Nice%20Derangement%20of%20Epistemes:%20Post-positivism%20in%20the%20Study%20of%20Science%20from%20Quine%20to%20Latour&amp;publication_year=2004&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref73">
<cite>Quine W. V.
O.. Two
Dogmas of Empiricism. Philos. Rev. 1951;60(1):20–43. doi: 10.2307/2266637.</cite> [<a href="https://doi.org/10.2307/2266637" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Philos.%20Rev.&amp;title=Two%20Dogmas%20of%20Empiricism&amp;author=W.%20V.%20O.%20Quine&amp;volume=60&amp;issue=1&amp;publication_year=1951&amp;pages=20-43&amp;doi=10.2307/2266637&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref74">
<cite>Hong L., Page S. E.. Groups of diverse
problem solvers can outperform groups
of high-ability problem solvers. Proc. Int.
Acad. Sci. 2004;101(46):16385–16389. doi: 10.1073/pnas.0403723101.</cite> [<a href="https://doi.org/10.1073/pnas.0403723101" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC528939/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15534225/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proc.%20Int.%20Acad.%20Sci.&amp;title=Groups%20of%20diverse%20problem%20solvers%20can%20outperform%20groups%20of%20high-ability%20problem%20solvers&amp;author=L.%20Hong&amp;author=S.%20E.%20Page&amp;volume=101&amp;issue=46&amp;publication_year=2004&amp;pages=16385-16389&amp;pmid=15534225&amp;doi=10.1073/pnas.0403723101&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref75">
<cite>

Fleck, L.

Genesis and Development
of a Scientific Fact, 
Bradley, F.
; 
Trenn, T. J.
, Eds.; University of Chicago Press: Basel, 1981.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=Genesis%20and%20Development%20of%20a%20Scientific%20Fact&amp;publication_year=1981&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref76">
<cite>

Longino, H. E.

Science as Social
Knowledge: Values and Objectivity in Scientific Inquiry, Princeton University Press, 1990.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=Science%20as%20Social%20Knowledge:%20Values%20and%20Objectivity%20in%20Scientific%20Inquiry&amp;publication_year=1990&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref77">
<cite>Swartz T. H., Palermo A.-G. S., Masur S. K., Aberg J. A.. The Science
and
Value of Diversity: Closing the Gaps in Our Understanding of Inclusion
and Diversity. J. Infect. Dis. 2019;220(220 Suppl 2):SS33–S41. doi: 10.1093/infdis/jiz174.</cite> [<a href="https://doi.org/10.1093/infdis/jiz174" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6701939/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31430380/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Infect.%20Dis.&amp;title=The%20Science%20and%20Value%20of%20Diversity:%20Closing%20the%20Gaps%20in%20Our%20Understanding%20of%20Inclusion%20and%20Diversity&amp;author=T.%20H.%20Swartz&amp;author=A.-G.%20S.%20Palermo&amp;author=S.%20K.%20Masur&amp;author=J.%20A.%20Aberg&amp;volume=220&amp;issue=220%20Suppl%202&amp;publication_year=2019&amp;pages=SS33-S41&amp;pmid=31430380&amp;doi=10.1093/infdis/jiz174&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref78">
<cite>Tawfik D. S.. Messy biology
and the origins of evolutionary innovations. Nat. Chem. Biol. 2010;6(10):692–696. doi: 10.1038/nchembio.441.</cite> [<a href="https://doi.org/10.1038/nchembio.441" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20852602/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Chem.%20Biol.&amp;title=Messy%20biology%20and%20the%20origins%20of%20evolutionary%20innovations&amp;author=D.%20S.%20Tawfik&amp;volume=6&amp;issue=10&amp;publication_year=2010&amp;pages=692-696&amp;pmid=20852602&amp;doi=10.1038/nchembio.441&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ol></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from ACS Bio &amp; Med Chem Au are provided here courtesy of <strong>American Chemical Society</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1021/acsbiomedchemau.5c00114"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/bg5c00114.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (4.5 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12371483/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12371483/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12371483%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371483/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12371483/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12371483/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12371483/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12371483/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12371483/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="Rc92Bf46VuBPjbfGeukCw72D42IEzWLzkie8qWJTaplM5ZhNDtTh8jF0tyesqub9">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
