
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            The Escherichia coli replication initiator DnaA is titrated on the chromosome - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="D2E4A6508AF24AC305A650002B8F5266.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="ncomms">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371093/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Nature Communications">
<meta name="citation_title" content="The Escherichia coli replication initiator DnaA is titrated on the chromosome">
<meta name="citation_author" content="Lorenzo Olivi">
<meta name="citation_author_institution" content="Laboratory of Microbiology, Wageningen University &amp; Research, Wageningen, Netherlands">
<meta name="citation_author" content="Stephan Köstlbacher">
<meta name="citation_author_institution" content="Laboratory of Microbiology, Wageningen University &amp; Research, Wageningen, Netherlands">
<meta name="citation_author" content="Christina Ludwig">
<meta name="citation_author_institution" content="Bavarian Center for Biomolecular Mass Spectrometry (BayBioMS), School of Life Sciences, Technical University of Munich, Munich, Germany">
<meta name="citation_author" content="Mees Langendoen">
<meta name="citation_author_institution" content="Laboratory of Microbiology, Wageningen University &amp; Research, Wageningen, Netherlands">
<meta name="citation_author" content="Nico J Claassens">
<meta name="citation_author_institution" content="Laboratory of Microbiology, Wageningen University &amp; Research, Wageningen, Netherlands">
<meta name="citation_author" content="Thijs J G Ettema">
<meta name="citation_author_institution" content="Laboratory of Microbiology, Wageningen University &amp; Research, Wageningen, Netherlands">
<meta name="citation_author" content="John van der Oost">
<meta name="citation_author_institution" content="Laboratory of Microbiology, Wageningen University &amp; Research, Wageningen, Netherlands">
<meta name="citation_author" content="Pieter Rein ten Wolde">
<meta name="citation_author_institution" content="Biochemical Networks Group, Institute AMOLF, Amsterdam, Netherlands">
<meta name="citation_author" content="Johannes Hohlbein">
<meta name="citation_author_institution" content="Laboratory of Biophysics, Wageningen University &amp; Research, Wageningen, Netherlands">
<meta name="citation_author_institution" content="Microspectroscopy Research Facility, Wageningen University &amp; Research, Wageningen, Netherlands">
<meta name="citation_author" content="Raymond H J Staals">
<meta name="citation_author_institution" content="Laboratory of Microbiology, Wageningen University &amp; Research, Wageningen, Netherlands">
<meta name="citation_publication_date" content="2025 Aug 21">
<meta name="citation_volume" content="16">
<meta name="citation_firstpage" content="7813">
<meta name="citation_doi" content="10.1038/s41467-025-63147-1">
<meta name="citation_pmid" content="40841366">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371093/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371093/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371093/pdf/41467_2025_Article_63147.pdf">
<meta name="description" content="DNA replication initiation is orchestrated in bacteria by the replication initiator DnaA. Two models for regulation of DnaA activity in Escherichia coli have been proposed: the switch between an active and inactive form, and the titration of DnaA on ...">
<meta name="og:title" content="The Escherichia coli replication initiator DnaA is titrated on the chromosome">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="DNA replication initiation is orchestrated in bacteria by the replication initiator DnaA. Two models for regulation of DnaA activity in Escherichia coli have been proposed: the switch between an active and inactive form, and the titration of DnaA on ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371093/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12371093">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1038/s41467-025-63147-1"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/41467_2025_Article_63147.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12371093%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12371093/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12371093/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371093/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-ncomms.gif" alt="Nature Communications logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Nature Communications" title="Link to Nature Communications" shape="default" href="http://www.nature.com/ncomms/index.html" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Nat Commun</button></div>. 2025 Aug 21;16:7813. doi: <a href="https://doi.org/10.1038/s41467-025-63147-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1038/s41467-025-63147-1</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Nat%20Commun%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nat%20Commun%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Nat%20Commun%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Nat%20Commun%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>The <em>Escherichia coli</em> replication initiator DnaA is titrated on the chromosome</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Olivi%20L%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Lorenzo Olivi</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Lorenzo Olivi</span></h3>
<div class="p">
<sup>1</sup>Laboratory of Microbiology, Wageningen University &amp; Research, Wageningen, Netherlands </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Olivi%20L%22%5BAuthor%5D" class="usa-link"><span class="name western">Lorenzo Olivi</span></a>
</div>
</div>
<sup>1,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22K%C3%B6stlbacher%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Stephan Köstlbacher</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Stephan Köstlbacher</span></h3>
<div class="p">
<sup>1</sup>Laboratory of Microbiology, Wageningen University &amp; Research, Wageningen, Netherlands </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22K%C3%B6stlbacher%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Stephan Köstlbacher</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ludwig%20C%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Christina Ludwig</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Christina Ludwig</span></h3>
<div class="p">
<sup>2</sup>Bavarian Center for Biomolecular Mass Spectrometry (BayBioMS), School of Life Sciences, Technical University of Munich, Munich, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ludwig%20C%22%5BAuthor%5D" class="usa-link"><span class="name western">Christina Ludwig</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Langendoen%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Mees Langendoen</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Mees Langendoen</span></h3>
<div class="p">
<sup>1</sup>Laboratory of Microbiology, Wageningen University &amp; Research, Wageningen, Netherlands </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Langendoen%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Mees Langendoen</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Claassens%20NJ%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Nico J Claassens</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Nico J Claassens</span></h3>
<div class="p">
<sup>1</sup>Laboratory of Microbiology, Wageningen University &amp; Research, Wageningen, Netherlands </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Claassens%20NJ%22%5BAuthor%5D" class="usa-link"><span class="name western">Nico J Claassens</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ettema%20TJG%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Thijs J G Ettema</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Thijs J G Ettema</span></h3>
<div class="p">
<sup>1</sup>Laboratory of Microbiology, Wageningen University &amp; Research, Wageningen, Netherlands </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ettema%20TJG%22%5BAuthor%5D" class="usa-link"><span class="name western">Thijs J G Ettema</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22van%20der%20Oost%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">John van der Oost</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">John van der Oost</span></h3>
<div class="p">
<sup>1</sup>Laboratory of Microbiology, Wageningen University &amp; Research, Wageningen, Netherlands </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22van%20der%20Oost%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">John van der Oost</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22ten%20Wolde%20PR%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Pieter Rein ten Wolde</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Pieter Rein ten Wolde</span></h3>
<div class="p">
<sup>3</sup>Biochemical Networks Group, Institute AMOLF, Amsterdam, Netherlands </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22ten%20Wolde%20PR%22%5BAuthor%5D" class="usa-link"><span class="name western">Pieter Rein ten Wolde</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hohlbein%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id9"><span class="name western">Johannes Hohlbein</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">Johannes Hohlbein</span></h3>
<div class="p">
<sup>4</sup>Laboratory of Biophysics, Wageningen University &amp; Research, Wageningen, Netherlands </div>
<div class="p">
<sup>5</sup>Microspectroscopy Research Facility, Wageningen University &amp; Research, Wageningen, Netherlands </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hohlbein%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Johannes Hohlbein</span></a>
</div>
</div>
<sup>4,</sup><sup>5,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Staals%20RHJ%22%5BAuthor%5D" class="usa-link" aria-describedby="id10"><span class="name western">Raymond H J Staals</span></a><div hidden="hidden" id="id10">
<h3><span class="name western">Raymond H J Staals</span></h3>
<div class="p">
<sup>1</sup>Laboratory of Microbiology, Wageningen University &amp; Research, Wageningen, Netherlands </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Staals%20RHJ%22%5BAuthor%5D" class="usa-link"><span class="name western">Raymond H J Staals</span></a>
</div>
</div>
<sup>1,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>Laboratory of Microbiology, Wageningen University &amp; Research, Wageningen, Netherlands </div>
<div id="Aff2">
<sup>2</sup>Bavarian Center for Biomolecular Mass Spectrometry (BayBioMS), School of Life Sciences, Technical University of Munich, Munich, Germany </div>
<div id="Aff3">
<sup>3</sup>Biochemical Networks Group, Institute AMOLF, Amsterdam, Netherlands </div>
<div id="Aff4">
<sup>4</sup>Laboratory of Biophysics, Wageningen University &amp; Research, Wageningen, Netherlands </div>
<div id="Aff5">
<sup>5</sup>Microspectroscopy Research Facility, Wageningen University &amp; Research, Wageningen, Netherlands </div>
<div class="author-notes p"><div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div></div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2024 Dec 6; Accepted 2025 Aug 11; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12371093  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40841366/" class="usa-link">40841366</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<p id="Par1">DNA replication initiation is orchestrated in bacteria by the replication initiator DnaA. Two models for regulation of DnaA activity in <em>Escherichia coli</em> have been proposed: the switch between an active and inactive form, and the titration of DnaA on the chromosome. Although proposed decades ago, experimental evidence of a titration-based control mechanism is still lacking. Here, we first identified a conserved high-density region of binding motifs near the origin of replication, an advantageous trait for titration of DnaA. We then investigated the mobility of DnaA by visualising single proteins inside single cells of wild-type and deletion mutants <em>E. coli</em> strains, while monitoring cellular size and DNA content. Our results indicate that the chromosome of <em>E. coli</em> controls the free amount of DnaA in a growth rate-dependent fashion. Moreover, they address long-standing questions on the relevance of DnaA titration in stabilising DNA replication by preventing re-initiation events during slow growth.</p>
<section id="kwd-group1" class="kwd-group"><p><strong>Subject terms:</strong> Origin firing, Bacteria, DNA replication, DNA-binding proteins</p></section></section><section class="abstract" id="Abs2"><hr class="headless">
<p id="Par2">A DNA-binding protein, DnaA, regulates the initiation of chromosomal replication in bacteria. Here, Olivi et al. show that the bacterial chromosome sequesters DnaA in a growth rate-dependent manner, suggesting a role for titration in the coordination of DNA replication.</p></section><section id="Sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p id="Par3">The correct coordination of DNA replication with cellular growth is crucial for any living organism<sup><a href="#CR1" class="usa-link" aria-describedby="CR1">1</a>–<a href="#CR3" class="usa-link" aria-describedby="CR3">3</a></sup>. Remarkably, the orchestration of these processes allows <em>Escherichia coli</em> to divide faster than the time it takes to replicate its chromosome<sup><a href="#CR4" class="usa-link" aria-describedby="CR4">4</a>–<a href="#CR7" class="usa-link" aria-describedby="CR7">7</a></sup>. This surprising ability was first explained in 1968 by a model stating that <em>E. coli</em> initiates one or more rounds of replication before the initial replicative event is completed<sup><a href="#CR4" class="usa-link" aria-describedby="CR4">4</a></sup>. Subsequently, an extension of the model introduced the concept of volume of initiation, a specific cellular size per origin of replication<sup><a href="#CR8" class="usa-link" aria-describedby="CR8">8</a></sup>. Since then, the coupling of DNA replication and cellular growth has been experimentally confirmed<sup><a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>,<a href="#CR7" class="usa-link" aria-describedby="CR7">7</a></sup> and the volume of initiation has been identified as a tightly controlled parameters in the <em>E. coli</em> cell cycle<sup><a href="#CR7" class="usa-link" aria-describedby="CR7">7</a>,<a href="#CR9" class="usa-link" aria-describedby="CR9">9</a>,<a href="#CR10" class="usa-link" aria-describedby="CR10">10</a></sup>.</p>
<p id="Par4">At the molecular level, <em>E. coli</em> initiates DNA replication by unwinding the origin of replication (<em>oriC</em>) through the binding of the replication initiator protein DnaA<sup><a href="#CR11" class="usa-link" aria-describedby="CR11">11</a></sup>. The regulation and biochemistry of DnaA have been extensively studied<sup><a href="#CR11" class="usa-link" aria-describedby="CR11">11</a>,<a href="#CR12" class="usa-link" aria-describedby="CR12">12</a></sup>, yet how <em>E. coli</em> achieves its characteristic DNA replication behaviour is still a matter of debate<sup><a href="#CR1" class="usa-link" aria-describedby="CR1">1</a>,<a href="#CR11" class="usa-link" aria-describedby="CR11">11</a>,<a href="#CR13" class="usa-link" aria-describedby="CR13">13</a>–<a href="#CR15" class="usa-link" aria-describedby="CR15">15</a></sup>. At the expression level, the <em>dnaA</em> gene is negatively autoregulated<sup><a href="#CR16" class="usa-link" aria-describedby="CR16">16</a></sup> and its near-balanced expression has been reported to lead to stable intracellular concentrations of DnaA across the cell cycle<sup><a href="#CR17" class="usa-link" aria-describedby="CR17">17</a></sup>. At the same time, the population average concentration of DnaA increases at faster growth rates<sup><a href="#CR10" class="usa-link" aria-describedby="CR10">10</a></sup>. The DnaA protein binds 9 bp-long DNA sequences, termed DnaA boxes. Both high-affinity and low-affinity boxes exist, with the first class matching a consensus sequence (TTWTNCACA)<sup><a href="#CR18" class="usa-link" aria-describedby="CR18">18</a></sup> and the latter carrying multiple variations<sup><a href="#CR19" class="usa-link" aria-describedby="CR19">19</a>,<a href="#CR20" class="usa-link" aria-describedby="CR20">20</a></sup>. Two models have been proposed to clarify how the DnaA-mediated unwinding of <em>oriC</em> is regulated. First, the initiator titration model<sup><a href="#CR21" class="usa-link" aria-describedby="CR21">21</a>,<a href="#CR22" class="usa-link" aria-describedby="CR22">22</a></sup> dictates that DnaA is sequestered by hundreds of high-affinity DnaA boxes present on the chromosome of <em>E. coli</em><sup><a href="#CR14" class="usa-link" aria-describedby="CR14">14</a></sup> (Fig. <a href="#Fig1" class="usa-link">1</a>, top). Only after enough DnaA is accumulated to saturate these boxes, DnaA can bind the low-affinity sites present at <em>oriC</em><sup><a href="#CR19" class="usa-link" aria-describedby="CR19">19</a>,<a href="#CR23" class="usa-link" aria-describedby="CR23">23</a>,<a href="#CR24" class="usa-link" aria-describedby="CR24">24</a></sup> and unwind it. With structural and biochemical characterisation of DnaA advancing, a second model emerged. This switch model (Fig. <a href="#Fig1" class="usa-link">1</a>, bottom) is based on the finding that DnaA switches between an ATP-bound state, competent in origin unwinding, and an inactive, ADP-bound state<sup><a href="#CR11" class="usa-link" aria-describedby="CR11">11</a></sup>. The interconversion between states is regulated by several mechanisms. Newly synthesised DnaA proteins predominantly bind to ATP due to its higher abundance in the cell compared to ADP<sup><a href="#CR25" class="usa-link" aria-describedby="CR25">25</a></sup>. Further, non-coding sequences called DnaA-reactivating sequences 1 and 2 (<em>DARS1</em> and <em>DARS2</em>) are involved in recruiting DnaA to nine particular DnaA boxes, where ADP is replaced with ATP<sup><a href="#CR26" class="usa-link" aria-describedby="CR26">26</a></sup>. At the same time, a similar re-activating mechanism has been proposed for interactions with acid phospholipids in vitro<sup><a href="#CR27" class="usa-link" aria-describedby="CR27">27</a></sup>, although this mechanism still lacks confirmation in vivo. Conversely, the non-coding sequence <em>datA</em> recruits DnaA with five other DnaA boxes, where <em>datA-</em>mediated DnaA-ATP hydrolysis (DDAH) occurs<sup><a href="#CR28" class="usa-link" aria-describedby="CR28">28</a></sup>. Finally, the Hda protein associates with the β-clamp of DNA polymerase III during replication to stimulate hydrolysis of DnaA-bound ATP in a process termed regulatory inactivation of DnaA (RIDA)<sup><a href="#CR29" class="usa-link" aria-describedby="CR29">29</a></sup>. Both DnaA-ATP and DnaA-ADP have been shown to be competent in binding high-affinity boxes, whereas only DnaA-ATP can bind to low-affinity boxes<sup><a href="#CR30" class="usa-link" aria-describedby="CR30">30</a></sup>, such as the ones present on <em>oriC</em><sup><a href="#CR19" class="usa-link" aria-describedby="CR19">19</a></sup>. Therefore, the switch model dictates that it is the accumulation of specifically the DnaA-ATP form, competent in unwinding of <em>oriC</em>, that determines the moment of initiation of DNA replication.</p>
<figure class="fig xbox font-sm" id="Fig1"><h3 class="obj_head">Fig. 1. The two models attempting to explain the control of DNA replication in <em>E. coli</em>.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/370b/12371093/36a9389a72cc/41467_2025_63147_Fig1_HTML.jpg" loading="lazy" id="d33e523" height="462" width="666" alt="Fig. 1"></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>DnaA negatively auto-regulates its own expression (left). Newly synthesised DnaA is bound by ATP. In the initiator titration model (top), the initiator protein is sequestered by high-affinity DnaA boxes throughout the chromosome. Only when all boxes are saturated, DnaA can bind to low-affinity boxes present on <em>oriC</em>, initiating DNA replication. In the switch model (bottom), DnaA is present in either an ATP-bound, active form, or an ADP-bound, inactive form. Several mechanisms are involved in the switch between the two forms. Once DnaA-ATP accumulates over a specific threshold, it can bind and unwind <em>oriC</em>.</p></figcaption></figure><p id="Par5">Since the characterisation of DnaA-ADP/ATP interconversion, the switch model has largely replaced the initiator titration model in explaining the replicative behaviour of <em>E. coli</em>. Yet, <em>E. coli</em> strains carrying deletions of the switch control components <em>datA</em>, <em>DARS1, DARS2</em> and <em>hda</em>, either individually or combined, exhibit defects in initiation, but are still viable<sup><a href="#CR17" class="usa-link" aria-describedby="CR17">17</a>,<a href="#CR31" class="usa-link" aria-describedby="CR31">31</a>,<a href="#CR32" class="usa-link" aria-describedby="CR32">32</a></sup>. The viability of the quadruple mutant was attributed to the intrinsic ATPase ability of DnaA, while postulating that initiator titration could provide an additional stabilising effect<sup><a href="#CR17" class="usa-link" aria-describedby="CR17">17</a></sup>. Moreover, recent single-cell analysis of the oscillation of DnaA-ATP levels in <em>E. coli</em> showed that reaching a peak level of DnaA-ATP is a necessary but not sufficient condition to initiate DNA replication<sup><a href="#CR33" class="usa-link" aria-describedby="CR33">33</a></sup>. In light of these results, either the accumulation of DnaA-ATP or its binding to the low-affinity binding sites on <em>oriC</em> could be delayed by the titration of the initiator on the <em>E. coli</em> chromosome. In agreement with this, previous in silico work suggested that concerted DnaA interconversion and its titration on the chromosome can enhance the stability of <em>E. coli</em> cell cycles in all growth conditions<sup><a href="#CR34" class="usa-link" aria-describedby="CR34">34</a>,<a href="#CR35" class="usa-link" aria-describedby="CR35">35</a></sup>, with a larger effect at slower growth rates<sup><a href="#CR36" class="usa-link" aria-describedby="CR36">36</a></sup>.</p>
<p id="Par6">The titration of DnaA on the <em>E. coli</em> chromosome was hypothesised more than 40 years ago<sup><a href="#CR21" class="usa-link" aria-describedby="CR21">21</a></sup> and has been considered as present in many recent modelling<sup><a href="#CR34" class="usa-link" aria-describedby="CR34">34</a>–<a href="#CR36" class="usa-link" aria-describedby="CR36">36</a></sup> and experimental studies<sup><a href="#CR17" class="usa-link" aria-describedby="CR17">17</a>,<a href="#CR32" class="usa-link" aria-describedby="CR32">32</a>,<a href="#CR33" class="usa-link" aria-describedby="CR33">33</a></sup>. Yet, there is still no experimental evidence that <em>E. coli</em> can control the free concentration of DnaA via titration. To address this gap in knowledge, we first analysed the configuration of <em>E. coli</em> genome and showed that DnaA boxes are distributed favourably for titration. Specifically, DnaA boxes accumulate towards the origin of replication, a trait conserved within many <em>E. coli</em> strains as well as in <em>Salmonella enterica</em>. Revealing the presence of DnaA titration experimentally requires assessing the binding state of this protein in live cells. Previous biochemical assays already probed the interactions of DnaA with DNA<sup><a href="#CR18" class="usa-link" aria-describedby="CR18">18</a>,<a href="#CR37" class="usa-link" aria-describedby="CR37">37</a>,<a href="#CR38" class="usa-link" aria-describedby="CR38">38</a></sup>, but they all required cell lysis. Additionally, earlier in vivo studies introduced titration sites via additional copies of <em>oriC</em><sup><a href="#CR39" class="usa-link" aria-describedby="CR39">39</a></sup> or <em>datA</em><sup><a href="#CR40" class="usa-link" aria-describedby="CR40">40</a>–<a href="#CR42" class="usa-link" aria-describedby="CR42">42</a></sup> to obtain insights on the effect of titration on DNA replication control. Yet, these studies were performed only under fast growth conditions and before <em>datA</em> had been identified as a DnaA-inactivating locus. Further, the additional titration sites were provided on plasmids, inevitably decoupling DnaA titration from DNA replication. Here, we overcome these limitations by directly probing interactions between DnaA and DNA in live <em>E. coli</em> cells via single-particle tracking photoactivatable localisation microscopy (sptPALM)<sup><a href="#CR43" class="usa-link" aria-describedby="CR43">43</a></sup>. We generated fusions of the native DnaA with the photoactivatable fluorescent protein PAmCherry2.1<sup><a href="#CR44" class="usa-link" aria-describedby="CR44">44</a></sup> in the wild-type <em>E. coli</em> MG1655 and in mutant strains in which <em>datA, DARS1</em> and <em>DARS2</em> were deleted either individually or together<sup><a href="#CR45" class="usa-link" aria-describedby="CR45">45</a></sup>. Building on our previously established experimental framework<sup><a href="#CR44" class="usa-link" aria-describedby="CR44">44</a></sup> (Supplementary Fig. <a href="#MOESM1" class="usa-link">1</a>), we grew all strains in constant optical density settings and monitored relevant cellular parameters (i.e., cell area and DNA content). By then performing sptPALM, we calculated the overall bound fraction of DnaA, as well as the mobility of the DnaA tracks generated throughout the cell cycle in different growth conditions. In this way, we provide experimental evidence that the <em>E. coli</em> chromosome is indeed capable of titrating DnaA, keeping its free fraction low in a growth rate-dependent fashion. Furthermore, our analysis suggests that DnaA titration increases in conditions in which more DnaA-ATP is present and decreases when the cell lacks DnaA reactivating power.</p></section><section id="Sec2"><h2 class="pmc_sec_title">Results</h2>
<section id="Sec3"><h3 class="pmc_sec_title">DnaA boxes are preferentially accumulated near the origin of replication</h3>
<p id="Par7">A genomic configuration in which DnaA boxes are distributed preferentially towards <em>oriC</em> allows for a large fraction of sites to be replicated soon after the start of DNA replication and is thus ideal for optimal titration of DnaA. The hundreds of binding motifs present on the chromosome of <em>E. coli</em> are believed to be homogeneously distributed<sup><a href="#CR12" class="usa-link" aria-describedby="CR12">12</a>,<a href="#CR18" class="usa-link" aria-describedby="CR18">18</a></sup>, yet this assumption has not been tested. We thus compared the number of DnaA boxes per position (radius of circular plot) in windows of 10 centisomes on the <em>E. coli</em> chromosome with the number expected when sampling from a uniform distribution (Fig. <a href="#Fig2" class="usa-link">2A</a>). We used a list of motifs that includes both the consensus sequence of DnaA box (TTWTNCACA)<sup><a href="#CR18" class="usa-link" aria-describedby="CR18">18</a></sup>, as well as other motifs functional in <em>datA, DARS1</em> and <em>DARS2</em> (HHMTHCWVH)<sup><a href="#CR45" class="usa-link" aria-describedby="CR45">45</a></sup>. We identified an extended region of significant overrepresentation of DnaA boxes between centisomes 98 and 5, specifically revealing a preferred enrichment of DnaA boxes around <em>oriC</em>. By evaluating the number of exact matches to permutated DnaA box motifs on the chromosome (Fig. <a href="#Fig2" class="usa-link">2A</a>), we further rejected the hypothesis that gradients in genome composition (e.g., GC-content skew) caused the observed enrichment. The same conclusions were reached when using only the consensus sequence (Supplementary Fig. <a href="#MOESM1" class="usa-link">2A</a>) or when counting DnaA boxes in windows of either 2 or 6 centisomes (Supplementary Fig. <a href="#MOESM1" class="usa-link">2B</a>). Additionally, we tried excluding <em>oriC</em> to test whether its high density of DnaA boxes could bias our analysis. Even without considering the boxes present at the origin, the enrichment in the adjacent region persisted (Supplementary Fig. <a href="#MOESM1" class="usa-link">2C</a>).</p>
<figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2. Density peaks of DnaA boxes across the chromosome of <em>E. coli</em> strains.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371093_41467_2025_63147_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/370b/12371093/7ba6815ff485/41467_2025_63147_Fig2_HTML.jpg" loading="lazy" id="d33e818" height="460" width="798" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>A</strong> Circular representation of the <em>E. coli</em> MG1655 chromosome, depicting DnaA box numbers at distinct positions. The actual numbers of DnaA boxes on the <em>E. coli</em> MG1655 chromosome is depicted as the white line and is compared to expected numbers (5 ≤ <em>p</em> ≤ 95) based on 1000 samples from a uniform distribution (light grey bands), and based on sequence matches to 1000 permutations of DnaA box sequences (dark grey band). Regions highlighted in yellow and orange indicate significant enrichment relative to the uniform null model and the sequence-permuted null model, respectively (one-sided permutation test, pseudo <em>p</em> &lt; 0.05, unadjusted for multiple comparisons). The radius of the circle indicates the number of boxes per centisomes, counted within a 10 centisomes window. <strong>B</strong> Circular representation of DnaA box density across multiple <em>E. coli</em> genomes and one singular <em>Salmonella enterica</em> chromosome, expressed as Z-scores. The density of <em>E. coli</em> MG1655 is depicted as the white line, while other 55 <em>E. coli</em> strains are in grey and <em>S. enterica</em>, considered as a distant relative species, is depicted in dark grey. For <em>E. coli</em> MG1655 and <em>S. enterica</em>, the global maximum is also plotted. The radius of the circle indicates the number of standard deviations away from the mean (Z-score) per centisome, counted within a 10 centisomes window. See Supplementary Fig. <a href="#MOESM1" class="usa-link">3</a> for density plots of each individual <em>E. coli</em> strain. In all panels, the outer ring represents the relative distance from the origin of replication, expressed in centisomes and with the 0 at <em>oriC</em>. In all panels, we used a TTWTNCACA motif for the consensus DnaA box and a HHMTHCWVH motif to include the boxes present among <em>datA</em>, <em>DARS1</em> and <em>DARS2</em> in the analysis. Source data are provided as a Source Data file.</p></figcaption></figure><p id="Par8">The accumulation of DnaA boxes towards <em>oriC</em> was not a characteristic unique to <em>E. coli</em> MG1655. We plotted the density of DnaA boxes across the chromosomes of 56 <em>E. coli</em> strains and obtained a Z-score (indicated from the radius of the circular plot) (Fig. <a href="#Fig2" class="usa-link">2B</a>). As for our previous analysis, we included both consensus motifs and other types of boxes found in <em>datA, DARS1</em> and <em>DARS2</em>, for an average of 902 ± 77 boxes per <em>E. coli</em> genome (see Table <a href="#MOESM1" class="usa-link">S4</a> in Supplementary Information for strain-specific numbers). We found that most <em>E. coli</em> chromosomes (<em>n</em> = 42, 74% of all analysed strains) have their global density maximum of DnaA boxes within 5 centisomes of the <em>oriC</em>. Further expanding this analysis, we observed that 93% of the genomes had either a local or a global maximum within 10 centisomes from <em>oriC</em> (see also Supplementary Fig. <a href="#MOESM1" class="usa-link">3</a>), with this region having a median DnaA box density two standard deviations above the chromosomal mean (Z-score median = 2.1, interquartile range 1.8–2.2). Interestingly, a similar distribution pattern is also observed in the distant relative <em>Salmonella enterica</em> (Fig. <a href="#Fig2" class="usa-link">2B</a>), suggesting an evolutionary conservation of this genomic arrangement beyond <em>E. coli</em>.</p>
<p id="Par9">Our analysis highlights an accumulation of DnaA boxes near the origin of replication. The significant overrepresentation of box sequences compared to the genome composition further suggests positive selection for DnaA boxes on the chromosome (Supplementary Fig. <a href="#MOESM1" class="usa-link">2D</a>). Finally, the preferential accumulation of these motifs on the lagging strand of DNA replication (Supplementary Fig. <a href="#MOESM1" class="usa-link">2E</a>) hints at cooperation between initiator titration and the switch-related mechanism of RIDA<sup><a href="#CR29" class="usa-link" aria-describedby="CR29">29</a></sup>, which functions through DNA polymerase III β-clamps<sup><a href="#CR46" class="usa-link" aria-describedby="CR46">46</a></sup> left on the lagging strands of replication<sup><a href="#CR47" class="usa-link" aria-describedby="CR47">47</a></sup>. Notably, the abundance of DnaA boxes near <em>oriC</em> means that a considerable fraction of these motifs is replicated in the early stages of the DNA replication elongation step. As a result, both the total number of DnaA boxes and their concentration can quickly increase after initiation, a feature that could help initiator titration during fast growth<sup><a href="#CR36" class="usa-link" aria-describedby="CR36">36</a></sup>.</p></section><section id="Sec4"><h3 class="pmc_sec_title">Growth rate-dependent titration of DnaA on the <em>E. coli</em> chromosome</h3>
<p id="Par10">After observing that DnaA boxes are preferentially arranged close to <em>oriC</em>, we moved on to seek experimental confirmation of the ability of the <em>E. coli</em> chromosome to titrate DnaA. To this end, we generated several fusions of DnaA in domain II<sup><a href="#CR48" class="usa-link" aria-describedby="CR48">48</a>,<a href="#CR49" class="usa-link" aria-describedby="CR49">49</a></sup> with photoactivatable or photoswitchable fluorescent proteins. The fusions were created by replacing amino acids 87-106<sup><a href="#CR48" class="usa-link" aria-describedby="CR48">48</a>,<a href="#CR49" class="usa-link" aria-describedby="CR49">49</a></sup> with the different fluorescent proteins (see Methods for details on the construction process). We ultimately selected PAmCherry2.1<sup><a href="#CR44" class="usa-link" aria-describedby="CR44">44</a></sup> for its low impact on DNA content (Supplementary Fig. <a href="#MOESM1" class="usa-link">4B</a>). Follow up characterisation confirmed that the <em>dnaA-PAmCherry2.1</em> mutation did not altered the cell area (Supplementary Fig. <a href="#MOESM1" class="usa-link">4C</a>) nor the DnaA expression levels compared to the wild-type background (Supplementary Fig. <a href="#MOESM1" class="usa-link">4D</a>). We additionally generated a plasmid carrying a copy of the <em>dnaAp</em> promoter driving transcription of the LacZ β-galactosidase. Previous studies have used the expression of LacZ as a proxy for the ratio of DnaA-ATP/DnaA-ADP in <em>E. coli</em> cells<sup><a href="#CR17" class="usa-link" aria-describedby="CR17">17</a>,<a href="#CR50" class="usa-link" aria-describedby="CR50">50</a></sup>, based on the fact that DnaA-ATP is more efficient in repressing its own promoter<sup><a href="#CR12" class="usa-link" aria-describedby="CR12">12</a>,<a href="#CR16" class="usa-link" aria-describedby="CR16">16</a></sup>. Once again, we observed no difference between wild-type <em>E. coli</em> and our <em>E. coli</em> MG1655 <em>dnaA-PAmCherry2.1</em> in all growth conditions (Supplementary Fig. <a href="#MOESM1" class="usa-link">4E</a>). Building on our previously established experimental framework<sup><a href="#CR44" class="usa-link" aria-describedby="CR44">44</a></sup> (Supplementary Fig. <a href="#MOESM1" class="usa-link">1</a>), we grew the obtained <em>E. coli</em> MG1655 <em>dnaA-PAmCherry2.1</em> strain (Fig. <a href="#Fig3" class="usa-link">3A</a>) at a constant optical density to maintain cells exponentially growing in either a slow (doubling time 187 ± 32 min), intermediate (doubling time 62 ± 9 min) or fast growth regime (doubling time 31 ± 3 min). We then performed sptPALM (Supplementary Movie <a href="#MOESM3" class="usa-link">1</a>) and generated diffusional histograms of DnaA in different conditions. We analysed the distributions with Monte-Carlo diffusion distribution analysis (MC-DDA) and obtained kinetic rates governing transitions between a free and a DNA-bound state of DnaA (Fig. <a href="#Fig3" class="usa-link">3B</a>). In this way, we observed a decreasing trend of the association rate <em>k</em><sub>Free</sub>→<sub>Bound</sub> from slow to fast growth regime, whereas the dissociation rate <em>k</em><sub>Bound</sub>→<sub>Free</sub> remained constant (Fig. <a href="#Fig3" class="usa-link">3C</a>, top). We then used the kinetic rates to estimate the average DnaA bound fraction as <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq1"><math id="d33e1022"><mfrac><mrow><msub><mrow><mi>k</mi></mrow><mrow><mi mathvariant="normal">Free</mi><mo>→</mo><mi mathvariant="normal">Bound</mi></mrow></msub></mrow><mrow><msub><mrow><mi>k</mi></mrow><mrow><mi mathvariant="normal">Free</mi><mo>→</mo><mi mathvariant="normal">Bound</mi></mrow></msub><mspace width="0.25em"></mspace><mo>+</mo><mspace width="0.25em"></mspace><msub><mrow><mi>k</mi></mrow><mrow><mi mathvariant="normal">Bound</mi><mo>→</mo><mi mathvariant="normal">Free</mi></mrow></msub></mrow></mfrac></math></span>. The overall bound fraction of DnaA decreased from 75 ± 2% in the slow growth regime, to 67 ± 3% in the intermediate and 55 ± 7% in the fast growth regime (Fig. <a href="#Fig3" class="usa-link">3C</a>, bottom). The DNA-bound fraction in fast growth is at the lower end of a previously reported range of 55–89% for other DNA-binding proteins in <em>E. coli</em> grown in similar growth conditions<sup><a href="#CR44" class="usa-link" aria-describedby="CR44">44</a>,<a href="#CR51" class="usa-link" aria-describedby="CR51">51</a></sup>. The extracted <em>k</em><sub>Bound</sub>→<sub>Free</sub> rates suggest short-lived interactions of DnaA with the chromosome (average <em>t</em><sub>Bound</sub> = 16 ± 2 ms), consistent with previous reports<sup><a href="#CR52" class="usa-link" aria-describedby="CR52">52</a></sup>. The binding of DnaA to DNA has been reported to be temperature-dependent in vitro, with weaker binding occurring at lower temperature<sup><a href="#CR53" class="usa-link" aria-describedby="CR53">53</a></sup>. Therefore, our single-particle experiments performed at room temperature could have led to an underestimation of the bound fractions. However, this temperature-dependent effect was constant throughout all our experiments and does not invalidate the observed growth rate-dependent behaviour. We further note that, together with changing the carbon source, intermediate growth was achieved by lowering the culture temperature to 30 °C (see Methods). Whereas changes in culture temperature affect growth rate, these changes have been reported to have a smaller impact on cellular mass<sup><a href="#CR10" class="usa-link" aria-describedby="CR10">10</a>,<a href="#CR54" class="usa-link" aria-describedby="CR54">54</a></sup>. Still, <em>E. coli</em> cells exhibited the characteristic trend of larger cell size with increasing growth rate<sup><a href="#CR7" class="usa-link" aria-describedby="CR7">7</a>,<a href="#CR10" class="usa-link" aria-describedby="CR10">10</a></sup>, with the measured areas progressively increasing from slow to fast growth conditions (Fig. <a href="#Fig4" class="usa-link">4B</a>).</p>
<figure class="fig xbox font-sm" id="Fig3"><h4 class="obj_head">Fig. 3. SptPALM of DnaA in live <em>E. coli</em> MG1655 cells.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371093_41467_2025_63147_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/370b/12371093/0e1cd5229b28/41467_2025_63147_Fig3_HTML.jpg" loading="lazy" id="d33e1160" height="513" width="667" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>A</strong> Photoactivatable fluorescent protein PAmCherry2.1 was fused to DnaA in its native chromosomal locus by replacing amino acids 87-106 in Domain II of DnaA. The domains of DnaA are indicated with Roman numerals. Domain II and IIIb are depicted in light grey, yet are not specifically indicated for visualisation purposes due to their small size. On the bottom, the codon numbers of each domain of the wild-type DnaA are indicated, not accounting for the PAmCherry2.1 insertion. <strong>B</strong> Distributions of diffusion coefficients of DnaA in live <em>E. coli</em> cells grown in either slow, intermediate or fast growth regimes. The number of cells and tracks per histogram is shown in the upper left corner. Histograms are fitted (black line) with a theoretical description of 250,000 particles moving between a free and a bound state. On the right, kinetic rates describing the fitting are provided. <strong>C</strong> Trends of the kinetic rates <em>k</em><sub>Free</sub>→<sub>Bound</sub> and <em>k</em><sub>Bound</sub>→<sub>Free</sub> and average bound fraction of DnaA in different growth regimes. Data are presented as mean values ± SD, derived mathematically from the fitting of diffusional distribution or from kinetic rates in (<strong>B</strong>). <em>P</em>-values are calculated by a two-sided T test. <strong>D</strong> Trends of DnaA mobility throughout the cell cycle. The average diffusion coefficient of each cell is plotted against the area of the cell, with the black line indicating the rolling median of the population. The grey ribbons represent the interquartile range. The magenta vertical line indicates the area at which cells initiate DNA replication. See Supplementary Fig. <a href="#MOESM1" class="usa-link">4G</a> for the area distribution of the cells. Source data are provided as a Source Data file.</p></figcaption></figure><figure class="fig xbox font-sm" id="Fig4"><h4 class="obj_head">Fig. 4. The effect of the Δ<em>DARS1</em>, Δ<em>DARS2</em>, Δ<em>datA</em> triple deletion on the binding of DnaA and the DNA replication of <em>E. coli</em>.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371093_41467_2025_63147_Fig4_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/370b/12371093/0afd25eb95b7/41467_2025_63147_Fig4_HTML.jpg" loading="lazy" id="d33e1285" height="415" width="667" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>A</strong> The <em>dnaA::PAmCherry2.1</em> mutation was introduced in the chromosomal locus of <em>dnaA</em> in the <em>E. coli</em> Δ3D strain. <strong>B</strong> Measurements of DNA content (top) (see Supplementary Fig. <a href="#MOESM1" class="usa-link">5A</a> for a complete list of replicates) and cellular area (bottom) of either the wild-type or the mutant strain for cells grown in slow (<em>n</em><sub>WT</sub> = 1409 cells, <em>n</em><sub>Δ3D</sub> = 1056 cells), intermediate (<em>n</em><sub>WT</sub> = 790 cells, <em>n</em><sub>Δ3D</sub> = 469 cells) or fast growth regime (<em>n</em><sub>WT</sub> = 693 cells, <em>n</em><sub>Δ3D</sub> = 480 cells). For the bar graph, data are presented as mean values ± SD, <em>n</em> = 3 biologically independent replicates. <em>P</em>-values are calculated by a two-sided T test. Box plots indicate median (middle line), 25th and 75th percentile (box) and 5th and 95th percentile (whiskers) as well as outliers (single points). <strong>C</strong> The diffusion coefficient histograms of DnaA obtained from sptPALM of <em>E. coli</em> Δ3D cells growing either in slow, intermediate or fast growth regime. Histograms are fitted (black line) with a theoretical description of 250,000 particles moving between a free and a bound state. The number of cells and tracks per histogram is shown in the upper left corner. On the right, the kinetic rates describing the fitting are provided. <strong>D</strong> Average bound fraction of DnaA in each growth regime for either the wild-type <em>E. coli</em> MG1655 or the triple deletion mutant <em>E. coli</em> Δ3D strain. Data are presented as mean values ± SD, derived mathematically from kinetic rates in (<strong>C</strong>). <em>P</em>-values are calculated by a two-sided T test. <strong>E</strong> Trends of DnaA mobility throughout the cell cycle either in slow (<em>n</em><sub>WT</sub> = 958 cells, <em>n</em><sub>Δ3D</sub> = 664 cells), intermediate (<em>n</em><sub>WT</sub> = 688 cells, <em>n</em><sub>Δ3D</sub> = 659 cells) or fast growth regime (<em>n</em><sub>WT</sub> = 697 cells, <em>n</em><sub>Δ3D</sub> = 705 cells). The average diffusion coefficient of each cell is plotted against the area of the cell, with the black line indicating the rolling median of the population. The grey ribbons represent the interquartile range, whereas the dotted magenta lines represent the wild-type references. See Supplementary Fig. <a href="#MOESM1" class="usa-link">5F</a> for the area distribution of the cells. Source data are provided as a Source Data file.</p></figcaption></figure><p id="Par11">We then calculated an average of the diffusion coefficients for each cell and compared it with the respective size of each cell (Fig. <a href="#Fig3" class="usa-link">3D</a>). In this way, we provided a proxy for the change in mobility of DnaA throughout the cell cycle in each growth condition. This area-dependent analysis further extended our previous observations on the average bound fraction of DnaA to the mobility of this protein in different phases of the cell cycle (Fig. <a href="#Fig3" class="usa-link">3D</a>). Cells in slow growth regime exhibited a low average diffusion coefficient (~0.5 μm<sup>2</sup>/s) throughout the cell cycle, whereas in intermediate and fast growth regime DnaA had an overall higher mobility (around 0.7–0.8 μm<sup>2</sup>/s).</p>
<p id="Par12">Our sptPALM assay revealed that the chromosome can effectively sequester DnaA in a growth rate-dependent fashion, as illustrated by the average DnaA bound fraction and the protein mobility through the cell cycle. This capacity is the highest in slow growth regime and progressively diminishes until a minimum during fast growth. The observed reduction in the bound fraction with increasing growth rate (Fig. <a href="#Fig3" class="usa-link">3C</a>) is likely the result of a faster accumulation of cellular volume. To maintain a constant intracellular concentration<sup><a href="#CR17" class="usa-link" aria-describedby="CR17">17</a></sup>, the synthesis rate of DnaA must increase with growth rate. Since DnaA concentrations are higher in <em>E. coli</em> cells rapidly dividing<sup><a href="#CR10" class="usa-link" aria-describedby="CR10">10</a></sup>, the increase in DnaA expression is also more pronounced at higher growth rates. In contrast, the synthesis of new titration sites per origin is set by the speed of DNA replication and is independent of growth rate<sup><a href="#CR7" class="usa-link" aria-describedby="CR7">7</a></sup>. As a result, the synthesis rate of DnaA increases faster with growth rate than the synthesis rate of titration sites, leading to a reduction in the effective titration power of the chromosome. These observations are consistent with previous mathematical modelling of a titration control mechanism<sup><a href="#CR36" class="usa-link" aria-describedby="CR36">36</a></sup>.</p></section><section id="Sec5"><h3 class="pmc_sec_title">
<em>DARS1, DARS2</em> and <em>datA</em> account for a large fraction of the chromosome titration power</h3>
<p id="Par13">The capacity of <em>E. coli</em> chromosome to sequester DnaA hints at initiator titration being a possible mechanism for controlling the activity of DnaA. The genome of this bacterium has hundreds of DnaA boxes that can recruit DnaA, peculiarly arranged to have a higher density towards the origin (Fig. <a href="#Fig2" class="usa-link">2</a>). Whereas the potential function of most of these boxes is still unknown, six of them are at the <em>oriC</em>, while 26 more are located among the switch control loci <em>DARS1, DARS2</em> and <em>datA</em><sup><a href="#CR45" class="usa-link" aria-describedby="CR45">45</a></sup>. To investigate the contribution of these boxes to the state of DnaA, we introduced the <em>dnaA-PAmCherry2.1</em> mutation in the triple deletion strain <em>E. coli</em> Δ3D (Δ<em>datA</em>, Δ<em>DARS1</em>, Δ<em>DARS2</em>)<sup><a href="#CR45" class="usa-link" aria-describedby="CR45">45</a></sup> (Fig. <a href="#Fig4" class="usa-link">4A</a>). We then characterised its phenotype at a population and a single-molecule level.</p>
<p id="Par14">We first measured the DNA content (see also Supplementary Fig. <a href="#MOESM1" class="usa-link">5A</a>) and cell area of <em>E. coli</em> Δ3D in the different growth regimes (Fig. <a href="#Fig4" class="usa-link">4B</a>). We observed a reduction in DNA content and an increase in cellular area in fast and intermediate growth regimes. The anti-correlation between the change in average cell size and the change in average number of origins upon deletion of loci mediating DnaA interconversion is consistent with previous reports<sup><a href="#CR17" class="usa-link" aria-describedby="CR17">17</a>,<a href="#CR45" class="usa-link" aria-describedby="CR45">45</a></sup> and with the idea that cells initiate a new round of replication after reaching a well-defined size<sup><a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>,<a href="#CR7" class="usa-link" aria-describedby="CR7">7</a>,<a href="#CR55" class="usa-link" aria-describedby="CR55">55</a></sup>. This size is then larger in <em>E. coli</em> Δ3D than in <em>E. coli</em> MG1655 (Fig. <a href="#Fig4" class="usa-link">4B</a>) due to the slight decrease in DnaA-ATP levels (Supplementary Fig. <a href="#MOESM1" class="usa-link">5B</a>) caused by the triple mutation. Interestingly, this anti-correlation was lost when <em>E. coli</em> Δ3D was cultured in slow growth regime. The median cellular area increased from 1.5 µm<sup>2</sup> in the wild-type to 2.2 µm<sup>2</sup> in <em>E. coli</em> Δ3D, pointing at delayed initiation. The <em>dnaAp</em> promoter was also more active in <em>E. coli</em> Δ3D than in the wild-type (Supplementary Fig. <a href="#MOESM1" class="usa-link">5B</a>), suggesting lower levels of DnaA-ATP as observed during intermediate and fast growth. The change in activity was milder compared to the other growth condition, likely due to a smaller contribution of RIDA and <em>DARS2</em> during slow growth. However, DNA content analysis revealed cells with a higher number of origins than the wild-type. More specifically, during slow growth <em>E. coli</em> Δ3D seems to cycle between 2 and 4 copies of the origin, rather than 1 to 2 as in the wild-type (Supplementary Fig. <a href="#MOESM1" class="usa-link">5A</a>). The triple mutant grew similarly to the wild-type strain in slow and fast growth regime and slower in intermediate growth regime (Supplementary Fig. <a href="#MOESM1" class="usa-link">5C</a>). From the number of cells with specific numbers of origins, we also obtained an asynchrony index and observed that this value was higher in <em>E. coli</em> Δ3D compared to the wild-type strain in all growth conditions (Supplementary Fig. <a href="#MOESM1" class="usa-link">5D</a>).</p>
<p id="Par15">We then performed sptPALM on <em>E. coli</em> Δ3D <em>dnaA-PAmCherry2.1</em> for cells grown in either slow, intermediate or fast growth regime (Fig. <a href="#Fig4" class="usa-link">4C</a>). The absence of <em>datA, DARS1</em> and <em>DARS2</em> impacted the interaction of DnaA with the chromosome only during slow growth. In this condition, the bound fraction decreased from 75 ± 2% in the wild-type to 58 ± 2% in <em>E. coli</em> Δ3D (Fig. <a href="#Fig4" class="usa-link">4D</a>). The average mobility of DnaA increased during the first part of the cell cycle, before stabilising at around 1 μm<sup>2</sup>/s (Fig. <a href="#Fig4" class="usa-link">4E</a>). On the other hand, kinetic rates, bound fraction and DnaA mobility in <em>E. coli</em> Δ3D remained similar to the wild-type in intermediate and fast growth regimes (Fig. <a href="#Fig4" class="usa-link">4D, E</a>). The near-wild-type levels of DnaA in <em>E. coli</em> Δ3D (Supplementary Fig. <a href="#MOESM1" class="usa-link">5E</a>) indicate that the decrease in DnaA bound fraction and increase in its mobility are likely due to a loss of important titration sites, rather than to an accumulation of extra DnaA proteins.</p>
<p id="Par16">Altogether, the deletion of <em>datA, DARS1</em> and <em>DARS2</em> led to different phenotypes in the different growth conditions. During slow growth, we observed a decreased bound fraction and increased mobility of DnaA, despite the higher DNA content indicating that more DnaA boxes are present compared to the wild-type condition. These observations lead us to believe that the contribution of the three switch control loci on titration is substantial under these conditions. More interestingly, the higher mobility of DnaA in this condition was also correlated with an increase in both the average number of origins and cell size. The increase in both average cell size and DNA content can be reconciled by noting that the activity of the <em>dnaAp</em> promoter increased during slow growth (Supplementary Fig. <a href="#MOESM1" class="usa-link">5B</a>). This increase points at a lower abundance of DnaA-ATP in <em>E. coli</em> Δ3D compared to the wild-type, leading to delayed initiations and thus higher values of cell area. Still, the higher DNA content is in apparent contradiction with the lower abundance of DnaA-ATP and could be explained by the presence of frequent re-initiation events. As such, these results indicate that regulation of the DNA-bound state of DnaA could be important for the control of replication during slow growth.</p></section><section id="Sec6"><h3 class="pmc_sec_title">The DnaA-ATP/DnaA-ADP ratio affects DnaA binding to the chromosome</h3>
<p id="Par17">We next set out to understand whether the increased availability of free DnaA had an impact on the increase of DNA content in <em>E. coli</em> Δ3D through reduced initiator titration or whether the phenotype could be solely explained through the removal of switch components. To this end, we introduced single deletions of either <em>datA, DARS1</em> or <em>DARS2</em> and further investigated the behaviour of <em>E. coli</em> during slow growth.</p>
<p id="Par18">As before, we performed sptPALM and derived kinetic rates through fitting (Fig. <a href="#Fig5" class="usa-link">5A</a>). The changes in bound fraction observed in the single deletion mutants did not add up to yield the major decrease observed in <em>E. coli</em> Δ3D (Fig. <a href="#Fig5" class="usa-link">5B</a>). The deletion of <em>datA</em> caused an increase in the average DnaA bound fraction (from 75 ± 2% to 82 ± 3%), whereas removing <em>DARS1</em> caused a reduction to around 69%. The Δ<em>DARS2</em> mutation had no significant impact on DnaA bound fraction. By measuring the DNA content (Fig. <a href="#Fig5" class="usa-link">5C</a>; see also Supplementary Fig. <a href="#MOESM1" class="usa-link">6A</a>) and cell size of the strains (Fig. <a href="#Fig5" class="usa-link">5D</a>) we observed the expected anti-correlation in the <em>datA</em> and <em>DARS1</em> single deletion strains. The absence of <em>datA</em> led to an increase in the average number of origins, smaller cells and reduced activity of the <em>dnaAp</em> promoter (Supplementary Fig. <a href="#MOESM1" class="usa-link">6B</a>). All these observations are consistent with the idea that <em>datA</em> inactivates DnaA and its deletion lowers the initiation volume<sup><a href="#CR36" class="usa-link" aria-describedby="CR36">36</a></sup>. Notably, the average number of origins of the Δ<em>datA</em> population was lower than the one of <em>E. coli</em> Δ3D (2.1 versus 2.5). At the same time, deletion of <em>DARS1</em> led to larger cells with a lower number of origins than <em>E. coli</em> MG1655 and more active <em>dnaAp</em> promoter (Supplementary Fig. <a href="#MOESM1" class="usa-link">6B</a>), in line with the reactivating role of this locus. Removing <em>DARS1</em> thus leads to an increase in initiation volume<sup><a href="#CR36" class="usa-link" aria-describedby="CR36">36</a></sup>. The deletion of <em>DARS2</em> seemed not to impact the average number of origins of <em>E. coli</em> (1.77 ± 0.004 compared to 1.78 ± 0.016) nor the activity of the <em>dnaAp</em> promoter when compared to the wild-type (Supplementary Fig. <a href="#MOESM1" class="usa-link">6B</a>). Such results are in line with the hypothesis that <em>DARS2</em> is inactive during slow growth<sup><a href="#CR36" class="usa-link" aria-describedby="CR36">36</a></sup> due to the lack of the protein Fis<sup><a href="#CR56" class="usa-link" aria-describedby="CR56">56</a>,<a href="#CR57" class="usa-link" aria-describedby="CR57">57</a></sup>. Yet, the Δ<em>DARS2</em> mutation led to a slight increase in the average cell size. All single deletion strains maintained near-wild-type levels of DnaA expression (Supplementary Fig. <a href="#MOESM1" class="usa-link">6C</a>). Moreover, they showed near-wild-type values of asynchrony index (Supplementary Fig. <a href="#MOESM1" class="usa-link">6D</a>) and mostly cycled between one and two origins (Supplementary Fig. <a href="#MOESM1" class="usa-link">6A</a>), pointing at a low frequency of re-initiation events.</p>
<figure class="fig xbox font-sm" id="Fig5"><h4 class="obj_head">Fig. 5. Population- and single-molecule-level phenotypic characterization of single <em>datA</em>, <em>DARS1</em> or <em>DARS2</em> deletions in slow growth regime.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371093_41467_2025_63147_Fig5_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/370b/12371093/cc2a32622bcf/41467_2025_63147_Fig5_HTML.jpg" loading="lazy" id="d33e1736" height="516" width="798" alt="Fig. 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>A</strong> Diffusion coefficient histograms of DnaA obtained from sptPALM with cells growing in slow growth regime in the absence of either <em>datA</em>, <em>DARS1</em> or <em>DARS2</em>. Histograms are fitted (black line) with a theoretical description of 250,000 particles moving between a free and a bound state. The number of cells and tracks per histogram is shown in the upper left corner. On top, the kinetic rates describing the fitting are provided for each growth regime. <strong>B</strong> Average bound fraction of DnaA, as estimated from the kinetic rates, for wild-type, triple deletion and the three single deletion <em>E. coli</em> mutants. Data are presented as mean values ± SD, derived mathematically from kinetic rates in (<strong>A</strong>). <em>P</em>-values are calculated by a two-sided T test. <strong>C</strong>, <strong>D</strong> Comparison of the average number of origins of replication (<strong>C</strong>) and the cell area of the population (<em>n</em><sub>Δ<em>datA</em></sub> = 3016 cells, <em>n</em><sub>Δ<em>DARS1</em></sub> = 760 cells, <em>n</em><sub>Δ<em>DARS2</em></sub> = 1300 cells) (<strong>D</strong>). See Supplementary Fig. <a href="#MOESM1" class="usa-link">6A</a> for a complete list of replicates of DNA content measurements via flow cytometry. For bar graphs, data are presented as mean values ± SD, <em>n</em> = 3 biologically independent replicates. Box plots indicate median (middle line), 25th and 75th percentile (box) and 5th and 95th percentile (whiskers) as well as outliers (single points). <em>P</em>-values are calculated by a two-sided T test. Source data are provided as a Source Data file.</p></figcaption></figure><p id="Par19">Our analysis of single deletion mutants highlighted a correlation between the DnaA-ATP/DnaA-ADP ratio and the bound fraction of DnaA. Specifically, mutations that led to a higher fraction of DnaA-ATP (Δ<em>datA</em>) caused more DnaA to be titrated on the chromosome, while the opposite was observed in the Δ<em>DARS1</em> mutant strain. Such changes can be explained by the fact that DnaA-ATP better interacts with low-affinity DnaA boxes<sup><a href="#CR19" class="usa-link" aria-describedby="CR19">19</a>,<a href="#CR20" class="usa-link" aria-describedby="CR20">20</a>,<a href="#CR30" class="usa-link" aria-describedby="CR30">30</a></sup> than DnaA-ADP and can form dimers or oligomers<sup><a href="#CR58" class="usa-link" aria-describedby="CR58">58</a></sup>. Differences in DNA content and removal of specific loci also change the average number of DnaA boxes in the cell. On a phenotypical level, deletions of <em>datA</em> and <em>DARS1</em> behaved as predicted by a switch-only model, exhibiting the characteristic anti-correlation between average cell size and average DNA content. These observations are consistent with a model in which replication is initiated once per cell cycle at a well-defined volume, whose threshold value is predominantly set by the balance of activation and deactivation<sup><a href="#CR35" class="usa-link" aria-describedby="CR35">35</a>,<a href="#CR36" class="usa-link" aria-describedby="CR36">36</a></sup>. On the other hand, the severely re-initiating phenotype of <em>E. coli</em> Δ3D could not be explained solely by perturbation of factors involved in the switch mechanism. Rather, it revealed a complex behaviour that needs to be explained through a combination of DnaA interconversion and its titration on the <em>E. coli</em> chromosome.</p></section><section id="Sec7"><h3 class="pmc_sec_title">Changes in DnaA ATP/ADP ratio or titration do not impact its function as transcription factor</h3>
<p id="Par20">Next to initiating DNA replication, DnaA is also a transcription factor involved in the regulation of several genes<sup><a href="#CR59" class="usa-link" aria-describedby="CR59">59</a>,<a href="#CR60" class="usa-link" aria-describedby="CR60">60</a></sup>. Previous reports showed how the activity of transcription factors can be modulated by introducing additional binding sites functioning as titration spots<sup><a href="#CR61" class="usa-link" aria-describedby="CR61">61</a></sup>. Additionally, negative autoregulation is a characteristic shared between DnaA and a variety of other <em>E. coli</em> transcription factors<sup><a href="#CR62" class="usa-link" aria-describedby="CR62">62</a></sup>. As such, we investigated whether the titration of DnaA on the <em>E. coli</em> chromosome could function as a control mechanism for its activity as a transcriptional regulator. DnaA has been reported to activate the transcription of the <em>nrdA, polA, glpD</em> and <em>fliC</em> genes<sup><a href="#CR59" class="usa-link" aria-describedby="CR59">59</a>,<a href="#CR63" class="usa-link" aria-describedby="CR63">63</a>–<a href="#CR65" class="usa-link" aria-describedby="CR65">65</a></sup>, while it acts as a repressor of the <em>mioC, uvrB, aldA, guaB, proS, rpoH, iraD</em> and <em>dnaA</em> genes<sup><a href="#CR59" class="usa-link" aria-describedby="CR59">59</a>,<a href="#CR66" class="usa-link" aria-describedby="CR66">66</a>–<a href="#CR72" class="usa-link" aria-describedby="CR72">72</a></sup>. Recently, chromatin immunoprecipitation experiments performed during fast growth provided evidence that the <em>purH, rne</em> and <em>nrdD</em> genes are also regulated by DnaA<sup><a href="#CR73" class="usa-link" aria-describedby="CR73">73</a></sup>. Notably, only four of these genes (<em>mioC</em>, <em>polA, purH</em> and <em>dnaA</em>) fall within the chromosomal region with a high density of DnaA boxes (Fig. <a href="#Fig6" class="usa-link">6A</a>). In total, the DnaA boxes on the promoter region of these genes account for 14 of the 72 motifs found around <em>oriC</em>; the total increases to 22 when considering <em>oriC</em>. Thus, we exclude the possibility that the previously observed enrichment in boxes towards the origin can originate uniquely from the clustering of genes that are part of DnaA regulon in this region.</p>
<figure class="fig xbox font-sm" id="Fig6"><h4 class="obj_head">Fig. 6. The effect of Δ<em>datA</em>, Δ<em>DARS1</em> and Δ<em>DARS2</em> deletion on the proteome of <em>E. coli</em>.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371093_41467_2025_63147_Fig6_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/370b/12371093/aa523b410587/41467_2025_63147_Fig6_HTML.jpg" loading="lazy" id="d33e1979" height="527" width="800" alt="Fig. 6"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>A</strong> Graphical representation of the genomic position of the DnaA regulon (black lines), of the <em>dnaA</em> gene (magenta line) and of known DnaA interaction loci (grey boxes). The number between brackets indicate the number of DnaA boxes on each gene. The white and blue line represents the count of DnaA boxes on the chromosome of <em>E. coli</em> MG1655, while the yellow area represents the region with significant DnaA boxes enrichment around <em>oriC</em>. Distances from the origin are expressed in centisomes, starting at <em>oriC</em>. <strong>B</strong>, <strong>C</strong> Global changes in the proteomes of (<strong>B</strong>) <em>E. coli</em> Δ3D in either slow (green), intermediate (blue) or fast (purple) growth regime or (<strong>C</strong>) <em>E. coli</em> Δ<em>datA</em>, <em>E. coli</em> Δ<em>DARS1</em> and <em>E. coli</em> Δ<em>DARS2</em> during slow growth. In all cases, the proteomes are compared to <em>E. coli</em> MG1655 <em>dnaA-PAmCherry2.1</em> in the same growth condition. Log<sub>10</sub> of fold changes in protein intensity are plotted against significance (-log<sub>10</sub>(false discovery rate)), with cutoffs placed at 2.5 fold-change in expression and a false discovery rate of &lt;0.05. Proteins outside the cutoffs are coloured according to the growth regime, while other proteins are in grey. Members of the DnaA regulon are circled in black, whereas DnaA is circled in magenta. <strong>D</strong> Principal component analysis showing all biological replicates for <em>E. coli</em> MG1655 <em>dnaA-PAmCherry2.1</em> (WT), <em>E. coli</em> Δ3D, <em>E. coli</em> Δ<em>datA</em>, <em>E. coli</em> Δ<em>DARS1</em> and <em>E. coli</em> Δ<em>DARS2</em> in all relevant growth conditions. Source data are provided as a Source Data file. The complete set of protein intensity signals is available in PRIDE.</p></figcaption></figure><p id="Par21">Nevertheless, we examined the proteomes of wild-type <em>E. coli</em> and <em>E. coli</em> Δ3D in the different growth conditions to obtain insights into how the triple deletion altered the global proteome (Fig. <a href="#Fig6" class="usa-link">6B</a>). We argued that, in the case in which titration plays a role specifically in the transcription factor activity of DnaA, the decrease in DnaA bound fraction in <em>E. coli</em> Δ3D during slow growth would generate unique changes in expression of the DnaA regulon. Despite large changes in the global proteome, we noticed that the majority of the DnaA regulon remained at wild-type levels upon deletion of <em>datA, DARS1</em> and <em>DARS2</em> in all growth regimes. The only exceptions were the flagellar protein FliC and the glycerol 3-phosphate dehydrogenase GlpD, which are overexpressed in all the tested growth conditions. Both the <em>fliC</em> and <em>glpD</em> genes are known to be positively regulated by DnaA. As such, it is surprising that mutations that lower the DnaA-ATP fraction, and thus its capacity to oligomerise on DNA<sup><a href="#CR58" class="usa-link" aria-describedby="CR58">58</a></sup>, would result in their overexpression. Yet, the notion that these genes are part of the DnaA regulon, at least during fast growth, has been recently called into question<sup><a href="#CR73" class="usa-link" aria-describedby="CR73">73</a></sup>. We also detected an enrichment in the expression of the anaerobic ribonucleoside-triphosphate reductase NrdD during intermediate and fast growth. Next, we analysed the proteomes of the single deletion mutants cultured in slow growth regime (Fig. <a href="#Fig6" class="usa-link">6C</a>). Single deletions of <em>datA</em> or <em>DARS1</em>, which led to marked changes in DnaA bound fraction (Fig. <a href="#Fig5" class="usa-link">5B</a>), DNA content (Fig. <a href="#Fig5" class="usa-link">5C</a>) and <em>dnaAp</em> activity (Supplementary Fig. <a href="#MOESM1" class="usa-link">6B</a>) did not substantially alter the global proteome of <em>E. coli</em> (Fig. <a href="#Fig6" class="usa-link">6C</a>). Conversely, the deletion of <em>DARS2</em> generated a large change in <em>E. coli</em> expression profile, including the overexpression of FliC, GlpD and NrdD. After performing principal component analysis, we further noticed that <em>E. coli</em> Δ<em>DARS2</em> and <em>E. coli</em> Δ3D clustered closely together (Fig. <a href="#Fig6" class="usa-link">6D</a>), meaning that both strains exhibit similar changes in protein expression. This observation was surprising, considering that the deletion of <em>DARS2</em> during slow growth did not alter DnaA bound fraction (Fig. <a href="#Fig5" class="usa-link">5B</a>), DNA content (Fig. <a href="#Fig5" class="usa-link">5C</a>) or <em>dnaAp</em> activity (Supplementary Fig. <a href="#MOESM1" class="usa-link">6B</a>). Yet, it serves as an explanation of the observed change in <em>E. coli</em> cell size upon its deletion (Fig. <a href="#Fig5" class="usa-link">5D</a>).</p>
<p id="Par22">Overall, mutations altering the DnaA-ATP fraction of DnaA (i.e., <em>dnaAp</em> activity) and/or its titration on <em>E. coli</em> chromosome (i.e., DnaA bound fraction) had no effect on the expression of DnaA regulon. Rather, our proteomics analysis hints at a yet unclear role of <em>DARS2</em> in the control of gene expression in <em>E. coli</em>. Whereas further investigating this role is beyond the scope of this study, we can conclude that switch and titration mechanisms seem to have little-to-no impact on the transcriptional regulator activity of DnaA during slow growth. Nevertheless, the binding motifs present on the promoter regions of the DnaA-regulated genes could play a role both in their transcription and in stabilising DNA replication via titration of DnaA.</p></section></section><section id="Sec8"><h2 class="pmc_sec_title">Discussion</h2>
<p id="Par23">In this work we present a systematic characterisation of DnaA behaviour in <em>E. coli</em> based on direct visualisation of DnaA. We confirm previous reports of transient interactions of DnaA with DNA<sup><a href="#CR52" class="usa-link" aria-describedby="CR52">52</a></sup> with an improved time resolution (10 ms against the previous 41 ms), while further enlarging the range of tested growth conditions and genetic backgrounds and implementing a cell size-dependent analysis. We provide a genomic basis and long-awaited experimental evidence of the titration capacity of the <em>E. coli</em> chromosome proposed more than 40 years ago<sup><a href="#CR22" class="usa-link" aria-describedby="CR22">22</a></sup>. Finally, we show that the state of the nucleotide bound to DnaA affects its binding on the chromosome of <em>E. coli</em> and that altering the titration power of the chromosome can impact the control of DNA replication during slow growth. On the other hand, these changes did not impact the activity of DnaA as transcription factor.</p>
<p id="Par24">While assessing the state of DnaA in the wild-type <em>E. coli</em> MG1655 (Fig. <a href="#Fig3" class="usa-link">3</a>), we observed that the association rate <em>k</em><sub>Free</sub>→<sub>Bound</sub> constantly decreased from slow to fast growth regime, while the dissociation rate <em>k</em><sub>Bound</sub>→<sub>Free</sub> remained largely constant. As a result, the bound fraction of DnaA was highest during the slow growth regime and reached a minimum in the fast growth regime (Fig. <a href="#Fig3" class="usa-link">3C</a>). This phenomenon is the result of two previously described <em>E. coli</em> characteristics: (i) near-balanced biosynthesis<sup><a href="#CR17" class="usa-link" aria-describedby="CR17">17</a>,<a href="#CR74" class="usa-link" aria-describedby="CR74">74</a></sup>, leading to constant intracellular DnaA concentration during the cell cycle, and (ii) the approximately constant speed of DNA replication elongation<sup><a href="#CR7" class="usa-link" aria-describedby="CR7">7</a></sup>. Maintaining constant DnaA concentrations requires increasing the expression of the <em>dnaA</em> gene<sup><a href="#CR10" class="usa-link" aria-describedby="CR10">10</a>,<a href="#CR74" class="usa-link" aria-describedby="CR74">74</a></sup> in conditions in which the cellular volume increases faster, i.e. during higher growth rates<sup><a href="#CR6" class="usa-link" aria-describedby="CR6">6</a></sup>. On the other hand, the rate at which new DnaA boxes are replicated per origin is independent of the growth rate, due to the nearly constant speed of DNA replication<sup><a href="#CR7" class="usa-link" aria-describedby="CR7">7</a></sup>. Therefore, in increasingly faster growth conditions, the rate of new DnaA synthesis progressively exceeds the rate by which DnaA boxes accumulate<sup><a href="#CR36" class="usa-link" aria-describedby="CR36">36</a></sup>. Moreover, the total concentration of DnaA is higher during faster growth<sup><a href="#CR10" class="usa-link" aria-describedby="CR10">10</a></sup>. Taking these features into consideration, our observed decreasing trend for the association rate indicates an excess of total DnaA proteins over the number of available titration sites at the steady-state. As a result, the chromosome ability to maintain a low concentration of free DnaA is impacted at higher growth rates. Still, the bound fraction accounted for more than 50% of visible DnaA proteins in all experimental conditions (Figs. <a href="#Fig3" class="usa-link">3</a>C, <a href="#Fig4" class="usa-link">4</a>D, <a href="#Fig5" class="usa-link">5B</a>). Whereas the number and rate of synthesis of DnaA boxes per origin is constant across growth rates, the same does not hold for the total number of DnaA boxes per cell. First, the DNA content of <em>E. coli</em> cells increases with increasing growth rates (Fig. <a href="#Fig4" class="usa-link">4A</a>). Second, cells eventually enter a regime of overlapping replication forks, in which the synthesis of new DNA proceeds through parallel replication rounds. The resulting increase in the total number of DnaA boxes per cell and in the rate at which these boxes are synthesised during progressively faster growth likely has an impact on the bound fraction of DnaA. Moreover, we hypothesise that the binding of DnaA to boxes scattered throughout the chromosome is also responsible for the overall high bound fractions, with pairs of closely spaced boxes<sup><a href="#CR73" class="usa-link" aria-describedby="CR73">73</a></sup> and specific loci (e.g. <em>mioC</em><sup><a href="#CR75" class="usa-link" aria-describedby="CR75">75</a></sup> and <em>oriC</em><sup><a href="#CR76" class="usa-link" aria-describedby="CR76">76</a>–<a href="#CR78" class="usa-link" aria-describedby="CR78">78</a></sup>) mediating more stable forms of binding. The preferential accumulation of boxes towards <em>oriC</em> (Fig. <a href="#Fig2" class="usa-link">2B</a>) then means that new DnaA boxes can rapidly be synthesised after DNA replication initiation, enhancing the titration power of the chromosome at all growth rates. This genomic configuration and the trends of association rates and DnaA bound fractions in different growth regimes (Fig. <a href="#Fig3" class="usa-link">3C</a>) are in accordance with previous modelling efforts<sup><a href="#CR34" class="usa-link" aria-describedby="CR34">34</a>,<a href="#CR36" class="usa-link" aria-describedby="CR36">36</a></sup> predicting the existence of a titration-based control mechanism.</p>
<p id="Par25">The original initiator titration model<sup><a href="#CR22" class="usa-link" aria-describedby="CR22">22</a></sup> has recently been expanded to include the interconversion of DnaA<sup><a href="#CR17" class="usa-link" aria-describedby="CR17">17</a>,<a href="#CR35" class="usa-link" aria-describedby="CR35">35</a>,<a href="#CR36" class="usa-link" aria-describedby="CR36">36</a></sup>. Our experimental evidence supports these updated models, while providing new insights to complement them. In these models, the titration of both forms of DnaA on the chromosome<sup><a href="#CR12" class="usa-link" aria-describedby="CR12">12</a>,<a href="#CR37" class="usa-link" aria-describedby="CR37">37</a>,<a href="#CR79" class="usa-link" aria-describedby="CR79">79</a></sup> keeps the free DnaA concentration relatively low throughout the cell cycle, enhancing the synchrony of replication firing and preventing re-initiation events<sup><a href="#CR34" class="usa-link" aria-describedby="CR34">34</a></sup>. Due to the growth rate-dependent decrease in the DnaA bound fraction (Fig. <a href="#Fig3" class="usa-link">3</a>), this mechanism is predicted to be prevalent during slow growth<sup><a href="#CR34" class="usa-link" aria-describedby="CR34">34</a></sup>. In line with this prediction, we report low mobility of DnaA across all observed cell size values in wild-type <em>E. coli</em> during slow growth (Fig. <a href="#Fig3" class="usa-link">3C</a>) and we observed no re-initiation events (Supplementary Fig. <a href="#MOESM1" class="usa-link">5A</a>). After all DnaA boxes are saturated, specifically DnaA-ATP binds to the low-affinity motifs on <em>oriC</em><sup><a href="#CR19" class="usa-link" aria-describedby="CR19">19</a>,<a href="#CR23" class="usa-link" aria-describedby="CR23">23</a>,<a href="#CR24" class="usa-link" aria-describedby="CR24">24</a></sup>, thereby initiating replication. In the 10 min following replication initiation, SeqA binds to <em>oriC</em> and <em>dnaAp</em> to prevent subsequent firing of the same origin and transcription of new DnaA, respectively<sup><a href="#CR80" class="usa-link" aria-describedby="CR80">80</a></sup>. The repression of <em>dnaA</em> transcription enacted by SeqA could act in concert with the synthesis of new DnaA boxes via replication to temporarily enhance titration and reduce overall DnaA activity. This process would be favoured by the particular chromosomal configuration reported here (Fig. <a href="#Fig2" class="usa-link">2</a>), leading to a rapid accumulation of new boxes. At the same time, RIDA and <em>datA</em> inactivate DnaA by promoting ATP hydrolysis, effectively preventing new rounds of initiation. Titration can also play a role in enhancing the action of RIDA. In RIDA, the protein Hda is loaded onto DNA polymerase β-clamps that are left on the lagging strand of DNA replication after the synthesis of Okazaki fragments<sup><a href="#CR47" class="usa-link" aria-describedby="CR47">47</a></sup>. From there, Hda inactivates nearby DnaA-ATP by stimulating the hydrolysis of ATP<sup><a href="#CR81" class="usa-link" aria-describedby="CR81">81</a></sup>. The striking preference for accumulating DnaA boxes on the lagging strand of each replichore (Supplementary Fig. <a href="#MOESM1" class="usa-link">2E</a>) likely increases interactions with the chromosome in proximity of Hda proteins, ensuring efficient inactivation of DnaA.</p>
<p id="Par26">Initiator titration models often consider that both forms of DnaA are titrated equally by the DnaA boxes on the chromosome<sup><a href="#CR17" class="usa-link" aria-describedby="CR17">17</a></sup>. Instead, our analysis hints at changes in the DnaA-ATP/DnaA-ADP ratio influencing the titration of DnaA on the chromosome of <em>E. coli</em>, at least during slow growth. Such effects could then have relevance in the control of DNA replication. The Δ<em>datA</em> mutation led to both an increase in DnaA-ATP/DnaA-ADP ratio (Supplementary Fig. <a href="#MOESM1" class="usa-link">6B</a>) and in the bound fraction of DnaA (Fig. <a href="#Fig5" class="usa-link">5</a>). The increase in DnaA titration could affect replication initiation by limiting the binding of DnaA-ATP to <em>oriC</em> and thus re-initiation events (Fig. <a href="#Fig7" class="usa-link">7</a>, top), consistent with the low fraction of cells with more than two origins we observed (Supplementary Fig. <a href="#MOESM1" class="usa-link">6A</a>). Conversely, the deletion of <em>DARS1</em> resulted in a lower abundance of DnaA-ATP (Supplementary Fig. <a href="#MOESM1" class="usa-link">6B</a>) and caused a decrease in the bound fraction of DnaA (Fig. <a href="#Fig5" class="usa-link">5</a>). In this sense, when the cell lacks a major DnaA reactivating component, a decrease in titration could lead to higher amounts of free DnaA-ATP and mitigate delays in replication initiation (Fig. <a href="#Fig7" class="usa-link">7</a>, middle), thus providing robustness. A role of titration in complementing interconversion of DnaA is also consistent with the fact that reaching peak levels of DnaA-ATP is a necessary but not sufficient condition to initiate replication<sup><a href="#CR33" class="usa-link" aria-describedby="CR33">33</a></sup>.</p>
<figure class="fig xbox font-sm" id="Fig7"><h3 class="obj_head">Fig. 7. The proposed role of titration in controlling re-initiation events.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371093_41467_2025_63147_Fig7_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/370b/12371093/f8bad7e047c9/41467_2025_63147_Fig7_HTML.jpg" loading="lazy" id="d33e2401" height="546" width="800" alt="Fig. 7"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig7/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>First, DnaA-ATP is accumulated to levels allowing DNA replication initiation (left). In <em>E. coli</em> Δ<em>datA</em> (top) the level of DnaA-ATP is higher, leading to early initiations and overall smaller cells. The increased titration can remove DnaA-ATP from the cytosol and minimise re-initiation events. In <em>E. coli</em> Δ<em>DARS1</em> (middle) the level of DnaA-ATP is lower than wild-type. Therefore, the cell needs more time to accumulate enough DnaA-ATP, leading to delayed initiation and larger cells. The decrease in titration can help mitigate this delay, by leading to higher levels of free DnaA-ATP competent in origin binding. In <em>E. coli</em> Δ3D (bottom) the level of DnaA-ATP is lower, leading again to longer times needed to accumulate DnaA-ATP, delayed initiation and larger cells. At the same time, titration is severely hampered. Therefore, when the threshold of DnaA-ATP is met, active DnaA remains free in the cytosol and can bind once again to <em>oriC</em>, leading to frequent re-initiations.</p></figcaption></figure><p id="Par27">The peculiar phenotype of <em>E. coli</em> Δ3D during slow growth, where both DNA content and cell size increase (Fig. <a href="#Fig4" class="usa-link">4</a>), can also be explained by considering the combined action of titration and switch mechanisms. In this mutant, the deletions of <em>datA</em> and <em>DARS1</em> leave the cell in a state of DnaA-ATP deficiency, as indicated by the increased activity of the <em>dnaAp</em> promoter in our bulk assay (Supplementary Fig. <a href="#MOESM1" class="usa-link">6B</a>). This effect also leads to a matching decrease in titration (see Δ<em>DARS1</em> mutant), which is further impacted by the loss of <em>DARS1, DARS2</em> and especially <em>datA</em><sup><a href="#CR40" class="usa-link" aria-describedby="CR40">40</a>,<a href="#CR82" class="usa-link" aria-describedby="CR82">82</a></sup> as titration loci. Due to the lack of reactivating loci, the only mechanism left for the cell to accumulate sufficient DnaA-ATP is through synthesis of new DnaA<sup><a href="#CR25" class="usa-link" aria-describedby="CR25">25</a></sup>. The accumulation of new DnaA-ATP then leads to the increase in DnaA mobility that we observed at the beginning of <em>E. coli</em> Δ3D cell cycle (Fig. <a href="#Fig4" class="usa-link">4D</a>). After sufficient DnaA-ATP is accumulated and replication is initiated, new DnaA boxes are rapidly synthesised, thereby stabilising DnaA mobility (Fig. <a href="#Fig4" class="usa-link">4C</a>) and lowering its activity. At this point, <em>E. coli</em> is mostly relying on the action of RIDA to inactivate DnaA, together with its intrinsic ATPase activity<sup><a href="#CR17" class="usa-link" aria-describedby="CR17">17</a></sup>. In this scenario, most DnaA is still freely diffusing rather than bound on the chromosome (Fig. <a href="#Fig4" class="usa-link">4D</a>) and modest accumulation of more DnaA-ATP could then stimulate cells to initiate new rounds of replication after the release of SeqA from <em>dnaAp</em> and <em>oriC</em><sup><a href="#CR17" class="usa-link" aria-describedby="CR17">17</a></sup> (Fig. <a href="#Fig7" class="usa-link">7</a>, bottom). Such frequent re-initiations would then explain the observed increase in DNA content in <em>E. coli</em> Δ3D during slow growth. It is possible that the decreased rate of DnaA-ATP hydrolysis caused by the absence of <em>datA</em> is sufficient to explain the re-initiating phenotype of the triple deletion mutant. It should further be noted that the <em>dnaAp</em> activity assay employed here relies on bulk measurements that could mask transient peaks in the active form of DnaA in key moments of the cell cycle. However, if high DnaA-ATP levels would be the sole determinant of re-initiation frequency, <em>E. coli</em> Δ<em>datA</em> would show the highest DNA content out of all the strains examined during slow growth. In <em>E. coli</em> Δ<em>datA</em>, the effect of the absence of the inactivating locus is further aggravated by the presence of the reactivating <em>DARS1</em> locus, leading to the highest DnaA-ATP levels observed (Supplementary Fig. <a href="#MOESM1" class="usa-link">6B</a>). Yet, <em>E. coli</em> Δ<em>datA</em> showed a lower number of origins than <em>E. coli</em> Δ3D (Fig. <a href="#Fig5" class="usa-link">5C</a>) and near-wild-type values of asynchrony index (Supplementary Fig. <a href="#MOESM1" class="usa-link">6D</a>). We believe that this surprising difference can also serve as indirect evidence for the role of DnaA titration on the coordination of DNA replication.</p>
<p id="Par28">Overall, our findings provide important experimental support of previous models in which initiator titration stabilises replication cycles<sup><a href="#CR32" class="usa-link" aria-describedby="CR32">32</a>,<a href="#CR36" class="usa-link" aria-describedby="CR36">36</a></sup> and suppresses intrinsic noise in DNA replication initiation. A titration-based control mechanism was hypothesised to be an ancient way of controlling DNA replication in slow-growing ancestors of <em>E. coli</em><sup><a href="#CR32" class="usa-link" aria-describedby="CR32">32</a>,<a href="#CR35" class="usa-link" aria-describedby="CR35">35</a></sup>. Interestingly, the peculiar chromosomal configuration that we found in <em>E. coli</em> MG1655, with a high representation of DnaA boxes arranged close to the chromosomal <em>oriC</em>, was present in most of the analysed strains of this species. The conservation of this arrangement perhaps hints at initiator titration being a primordial form of DNA replication control. In the future, specifically altering the titration of DnaA by removing binding motifs outside of <em>datA</em>, <em>DARS1</em> and <em>DARS2</em> could shine further light on the role of this control system also in setting the volume of initiation. Extending previous attempts of chromatin immunoprecipitation of <em>E. coli</em> DnaA<sup><a href="#CR73" class="usa-link" aria-describedby="CR73">73</a></sup> to different growth rates and genetic backgrounds could also provide a better understanding of the usage of each box. Finally, implementing microfluidics-based cultivation strategies<sup><a href="#CR83" class="usa-link" aria-describedby="CR83">83</a></sup> could then allow to probe variations in DnaA abundance<sup><a href="#CR17" class="usa-link" aria-describedby="CR17">17</a></sup> and the DnaA-ATP/DnaA-ADP ratio<sup><a href="#CR33" class="usa-link" aria-describedby="CR33">33</a></sup>, as well as to connect changes in DnaA mobility with key moments of the <em>E. coli</em> cell cycle. In any case, our experimental pipeline may serve as a solid basis for any future research on this elusive form of control, for which direct visualisation of the initiator protein DnaA is posed to be essential.</p></section><section id="Sec9"><h2 class="pmc_sec_title">Methods</h2>
<section id="Sec10"><h3 class="pmc_sec_title">Computational identification of chromosomal DnaA box densities</h3>
<p id="Par29">Nucleotide fasta files of several <em>E. coli</em> strains genomes<sup><a href="#CR45" class="usa-link" aria-describedby="CR45">45</a></sup>, as well as the reference genome of <em>Salmonella enterica</em> were downloaded from RefSeq<sup><a href="#CR84" class="usa-link" aria-describedby="CR84">84</a></sup>. Identifiers <a href="https://www.ncbi.nlm.nih.gov/nuccore/NC_017644.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">NC_017644.1</a>, <a href="https://www.ncbi.nlm.nih.gov/nuccore/NC_017663.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">NC_017663.1</a>, and <a href="https://www.ncbi.nlm.nih.gov/nuccore/NC_020518.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">NC_020518.1</a> were removed from RefSeq and therefore excluded from analysis. A custom-made Python script (matchBox.py) was developed to scan circular chromosomes and identify exact matches to the DnaA box motifs, either on the forward or reverse strand. The consensus DnaA box sequence (TTWTNCACA)<sup><a href="#CR18" class="usa-link" aria-describedby="CR18">18</a></sup> and other sequences reported as active in DnaA binding across <em>datA, DARS1</em> and <em>DARS2</em> (HHMTHCWVH)<sup><a href="#CR45" class="usa-link" aria-describedby="CR45">45</a></sup> were included in the list of potential matches. We accounted for the circular nature of <em>E. coli</em> and <em>S. enterica</em> chromosomes by allowing detection of motifs near the origin and terminus. Matches were identified and assigned a sequence ID, position, orientation and motif. A complete list of the matches for each genome is available in the Source Data file. The distribution of DnaA box motifs was then analysed and visualised with R v4.4.1<sup><a href="#CR85" class="usa-link" aria-describedby="CR85">85</a></sup>. The relative positions of DnaA box motifs within each chromosome was calculated by scaling to centisomes, i.e. dividing the total length of each genome in 100 equal intervals<sup><a href="#CR86" class="usa-link" aria-describedby="CR86">86</a></sup>, starting from the position of <em>oriC</em>, determined via the DoriC database<sup><a href="#CR87" class="usa-link" aria-describedby="CR87">87</a></sup>. The null hypothesis of DnaA boxes being homogenously distributed was generated by repeatedly (<em>n</em> = 1000) simulating DnaA box positions over the chromosome from a uniform distribution in R v4.4.1<sup><a href="#CR85" class="usa-link" aria-describedby="CR85">85</a></sup> with the ‘runif’ function. We calculated the number of boxes per position in a sliding window approach by moving over the scaled chromosome in either 2, 6 or 10 centisome windows, taking 1 centisome steps. Pseudo <em>p</em>-values were obtained based on the rank of the real box number per position compared to the simulated values from a uniform distribution with a significance cutoff of <em>p</em>-value &lt; 0.05. To additionally test whether the non-uniform distribution of DnaA boxes derived from the genome compositional background (i.e., GC-content), we created 1000 permutations of the used DnaA box sequences using a custom-made Python script (shuffleBoxes.py). We searched the chromosome for identical matches to the permutated boxes and calculated the number of motifs per position in a sliding window approach as for the uniform distribution test. Pseudo <em>p</em>-values were calculated and used as before to obtain significance cutoffs.</p>
<p id="Par30">Circular statistics was employed to estimate the density of DnaA boxes along the genomes, using the circular v0.5-0 package<sup><a href="#CR88" class="usa-link" aria-describedby="CR88">88</a></sup>, and the von Mises kernel to smooth the density of motifs. The density estimates were subsequently scaled to Z-score to indicate the number of standard deviations the value is away from the mean density of boxes across the chromosome (Z-score = 0). Extreme density values were extracted with the zoo v1.8-12 package<sup><a href="#CR89" class="usa-link" aria-describedby="CR89">89</a></sup> ‘rollapply’ function, with the largest Z-score assigned as the global maximum for that genome. Data was then visualised with the ggplot2 v3.5.1 package<sup><a href="#CR90" class="usa-link" aria-describedby="CR90">90</a></sup>.</p></section><section id="Sec11"><h3 class="pmc_sec_title">Bacterial strains, plasmids and growth conditions</h3>
<p id="Par31">A list of bacterial strains used in this study is provided in Supplementary Information (Table <a href="#MOESM1" class="usa-link">S1</a>). <em>Escherichia coli</em> DH5α (NEB) was used for general plasmid propagation and standard molecular techniques. The wild-type <em>Escherichia coli</em> MG1655 and mutant strains lacking the genomic loci <em>datA, DARS1</em> and <em>DARS2</em> either individually or in combination<sup><a href="#CR45" class="usa-link" aria-describedby="CR45">45</a></sup> were used to investigate DnaA interactions with the bacterial chromosome.</p>
<p id="Par32">All <em>E. coli</em> strains were routinely grown at 30 °C or 37 °C and 200 rpm in LB liquid medium (10 g/L tryptone, 10 g/L NaCl, 5 g/L yeast extract) for strain propagation. M9 liquid medium (5 g/L KH<sub>2</sub>PO<sub>4</sub>, 0.5 g/L NaCl, 6.78 g/L Na<sub>2</sub>HPO<sub>4</sub>, 1 g/L NH<sub>4</sub>Cl, 4.98 mg/L FeCl<sub>3</sub>, 0.84 mg/L ZnCl<sub>2</sub>, 0.13 mg/L CuCl<sub>2</sub> · 2H<sub>2</sub>O, 0.1 mg/L CoCl<sub>2</sub> · 6H<sub>2</sub>O, 0.1 mg/L H<sub>3</sub>BO<sub>3</sub>, 0.016 mg/L MnCl · 4H<sub>2</sub>O, 0.5 g/L MgSO<sub>4</sub> · 7H<sub>2</sub>O, 11 mg/L CaCl<sub>2</sub>) was used during constant density cultivations or single-particle tracking. Growth conditions to achieve different growth regimes were: M9 supplemented with 4 g/L glucose and 1x RPMI 1640 amino acids at 37 °C for fast growth regime; M9 supplemented with 4 g/L succinate and 1x RPMI 1640 amino acids at 30 °C for intermediate growth regime; and M9 supplemented with 4 g/L acetate at 37 °C for slow growth regime. Media were eventually supplemented with agar (15 g/L) to obtain solid media and with antibiotics for plasmid propagation (50 mg/L kanamycin, 100 mg/L spectinomycin, 100 mg/L ampicillin, 25 mg/L chloramphenicol).</p>
<p id="Par33">Phosphate-buffered saline solution (PBS) (8 g/L NaCl, 0.2 g/L KCl, 1.42 g/L Na<sub>2</sub>HPO<sub>4</sub>, 0.24 g/L KH<sub>2</sub>PO<sub>4</sub>) was used to wash cells during sample preparation.</p></section><section id="Sec12"><h3 class="pmc_sec_title">Plasmid construction</h3>
<p id="Par34">A list of all constructs used in this study is provided in Supplementary Information (Table <a href="#MOESM1" class="usa-link">S2</a>). The pCas and pTarget system<sup><a href="#CR91" class="usa-link" aria-describedby="CR91">91</a></sup> was used to obtain genomic mutants of <em>E. coli</em> MG1655. The pTarget_dnaA-PAFP plasmid series were constructed from pTarget<sup><a href="#CR91" class="usa-link" aria-describedby="CR91">91</a></sup>. The pTarget_dnaA-PAFP plasmid series expressed a single-guide RNA targeting the chromosomal <em>dnaA</em> gene of <em>E. coli</em> in its domain II and a repair template flanked by 50 bp-long homology arms. The repair template consisted of the gene encoding one of four photoactivatable or photoconvertible fluorescent proteins (PAFP), replacing bases 259-312 of the native <em>dnaA</em> gene<sup><a href="#CR48" class="usa-link" aria-describedby="CR48">48</a>,<a href="#CR49" class="usa-link" aria-describedby="CR49">49</a></sup>. The tested PAFP were Dronpa2<sup><a href="#CR92" class="usa-link" aria-describedby="CR92">92</a></sup> (BG25452), mEos4b<sup><a href="#CR93" class="usa-link" aria-describedby="CR93">93</a></sup> (BG25454), mMaple3<sup><a href="#CR94" class="usa-link" aria-describedby="CR94">94</a></sup> (BG25455) and PAmCherry2.1, a M10L<sup><a href="#CR95" class="usa-link" aria-describedby="CR95">95</a></sup> mutant of PAmCherry2<sup><a href="#CR96" class="usa-link" aria-describedby="CR96">96</a></sup>. Sequences were codon-optimised using the IDT Codon Optimisation Tool (IDT) and DNA fragments coding for Dronpa2, mEos4b and mMaple3 were chemically synthesised (IDT). The sequences of the chemically synthesised fragments are provided in Supplementary Information (Table <a href="#MOESM1" class="usa-link">S3</a>). The sequence of the codon-optimised PAmCherry2.1 was retrieved from the pLbdCas12a-PAmCherry2.1_scrambled plasmid<sup><a href="#CR44" class="usa-link" aria-describedby="CR44">44</a></sup>.</p>
<p id="Par35">The pDnaA211-PAmCherry2.1 plasmid was based on a pBeloBAC11 plasmid<sup><a href="#CR97" class="usa-link" aria-describedby="CR97">97</a></sup> modified to express proteins of interest via the TetR-repressible promoter PLtetO-1 and using a bicistronic design<sup><a href="#CR98" class="usa-link" aria-describedby="CR98">98</a></sup>. The plasmid contained the mutant allele <em>dnaA211</em>, encoding a M426T mutation reported to abolish binding to DNA<sup><a href="#CR99" class="usa-link" aria-describedby="CR99">99</a></sup>. The mutation was introduced via site-directed mutagenesis using primers BG36136 and BG36137. The gene encoding PAmCherry2.1 was used to replace bases 259-312 of the DnaA211 gene.</p>
<p id="Par36">The pdnaAp-LacZ plasmid was based on the design of the pTACDNAA plasmid<sup><a href="#CR17" class="usa-link" aria-describedby="CR17">17</a>,<a href="#CR50" class="usa-link" aria-describedby="CR50">50</a></sup>, using a pSC101 plasmid containing the <em>lacZ</em> gene under the control of <em>dnaAp</em>, the native <em>dnaA</em> promoter. Both <em>dnaAp</em> and <em>lacZ</em> were amplified from the genome of <em>E. coli</em> MG1655, using primers BG35613 and BG35614 for <em>dnaAp</em> and BG36203 and BG36204 for <em>lacZ</em>.</p>
<p id="Par37">For cloning purposes, DNA fragments were amplified by PCR using Q5® High-Fidelity DNA Polymerase (NEB) following manufacturer instructions. Specific oligonucleotides were designed and synthesised (IDT) to introduce proper overhangs for assembly. Assembly was performed using NEBuilder® HiFi DNA Assembly Master Mix (NEB).</p></section><section id="Sec13"><h3 class="pmc_sec_title">Electro-competent cells preparation and transformation procedure</h3>
<p id="Par38">Pre-cultures of <em>E. coli</em> were grown overnight in LB liquid medium at 37 °C and 200 rpm. Cells were made electrocompetent by re-inoculating and culturing them at 37 °C and 200 rpm in LB. Strains carrying either the pCas9<sup><a href="#CR91" class="usa-link" aria-describedby="CR91">91</a></sup>, the pCas9_ampR or the pSIJ8<sup><a href="#CR100" class="usa-link" aria-describedby="CR100">100</a></sup> plasmid were instead cultured at 30 °C and 200 rpm in LB, supplemented with L-arabinose (final concentration 10 mM) to induce expression of the λ-Red system. Upon reaching OD<sub>600nm</sub> of 0.4, the cells were cooled down to 4 °C, washed one time with 1 culture volume of ice-cold Milli-Q water and two times with 0.5 culture volumes of an ice-cold 10% V/V glycerol solution in Milli-Q water. Cells were then suspended in ice-cold 10% glycerol to a final volume of 400 µL for each 100 mL of initial culture volume and dispensed in 40 μL aliquots. All washing steps were performed at 4 °C, by centrifugation for 10 min at 3000 x<em>g</em>.</p>
<p id="Par39">Electroporation was performed in ice-cold 2 mm electroporation cuvettes at 2500 V, 200 Ω and 25μF. Immediately after electroporation, cells were recovered in LB medium. Recovery was performed either at 30 °C, 750 rpm for 2 h for cells harbouring either the pCas9, the pCas9_ampR or the pSIJ8 plasmid or at 37 °C, 750 rpm for 1 h in all other instances. Following recovery, cells were spread on LB solid medium supplemented with appropriate antibiotics for plasmid maintenance and incubated overnight at the appropriate temperature.</p></section><section id="Sec14"><h3 class="pmc_sec_title">Creation of the <em>dnaA-PAFP</em> fusion mutants</h3>
<p id="Par40">The different photoactivatable or photoconvertible proteins were knocked in domain II of DnaA, replacing base pairs 259-312 of the native <em>dnaA</em> gene<sup><a href="#CR48" class="usa-link" aria-describedby="CR48">48</a>,<a href="#CR49" class="usa-link" aria-describedby="CR49">49</a></sup>. To this end, a previously reported procedure was followed, consisting of a two-plasmid system<sup><a href="#CR91" class="usa-link" aria-describedby="CR91">91</a></sup>. Briefly, electrocompetent cells of wild-type <em>E. coli</em> MG1655 harbouring the pCas plasmid were transformed with one of the four pTarget_dnaA-PAFP plasmids. Correct insertion of the PAFP was confirmed first by PCR using primers BG21364 and BG21365 and then by Sanger sequencing of the locus. The same approach was used for the generation of the <em>dnaA-PAFP</em> fusion mutant in the <em>E. coli</em> Δ3D background. In this case, the pCas_ampR was used instead of the original pCas plasmid and only the pTarget_dnaA-PAmCherry2.1 plasmid was used to introduce the mutation. After insertion of <em>PAmCherry2.1</em> gene in <em>E. coli</em> MG1655 and <em>E. coli</em> Δ3D, whole genome sequencing was performed to confirm that the modified locus was the only source of DnaA.</p>
<p id="Par41">In the case of the single deletion mutants, electrocompetent cells of <em>E. coli</em> MG1655 <em>dnaA-PAmCherry2.1</em> harbouring the pSIJ8 plasmid<sup><a href="#CR100" class="usa-link" aria-describedby="CR100">100</a></sup> were transformed with a linear fragment replacing either <em>DARS1</em> or <em>DARS2</em> with a chloramphenicol resistance cassette or <em>datA</em> with a kanamycin resistance cassette. The linear fragments were obtained by amplifying the genomic locus of <em>E. coli</em> mutant strains carrying either the Δ<em>datA::kanR</em> (primers BG23801 and BG23802), Δ<em>DARS1::cat</em> (primers BG23803 and BG23804) or Δ<em>DARS2::cat</em> allele<sup><a href="#CR45" class="usa-link" aria-describedby="CR45">45</a></sup> (primers BG23805 and BG23806). Transformant colonies were first selected in the appropriate resistance and the locus of interest was then further confirmed by PCR. The sequences of the oligonucleotides are provided in Supplementary Information (Table <a href="#MOESM1" class="usa-link">S3</a>).</p></section><section id="Sec15"><h3 class="pmc_sec_title">Turbidostat cultivation of cell cultures</h3>
<p id="Par42"><em>E. coli</em> strains were grown overnight in 5 mL of LB medium at 37 °C and 180 rpm. The densely-grown culture was then used to start a turbidostat cultivation using the Chi.Bio platform<sup><a href="#CR101" class="usa-link" aria-describedby="CR101">101</a></sup>. Experiments were started according to manufacturer instructions for fluidic lines, electrical connections and user interface. Cultures were started at an OD<sub>600nm</sub> of 0.05 in M9 medium. Both the M9 supplementation and the temperature were selected according to the specific growth regime to obtain. Default settings were employed for stirring and frequency of optical density measurements. A 650 nm laser diode was used to monitor the optical density of the culture, to minimise accidental photoactivation of PAmCherry2.1. The target optical density was set to 0.4 AU (corresponding to an OD<sub>600nm</sub> of ~0.2) and the media reservoir was filled with M9 supplemented as the cultivation chambers. Supplemented M9 medium was pumped in the vessel from the reservoir to maintain a constant optical density. Cells were balanced in exponential growth at the target optical density for at least ten generations before 10 mL of culture was harvested for imaging. The remainder was subjected to replication run-out for DNA content measurements.</p>
<p id="Par43">In a turbidostat, the dilution rate (<em>D</em>) is equal to the growth rate (<em>µ</em>). For each cultivation, the total volume (<em>V</em><sub>SteadyState</sub>) and time (<em>t</em><sub>SteadyState</sub>) spent at the steady-state optical density were used to calculate the flow rate <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq2"><math id="d33e2997"><mi>F</mi><mo>=</mo><mfrac><mrow><msub><mrow><mi>V</mi></mrow><mrow><mi mathvariant="normal">SteadyState</mi></mrow></msub></mrow><mrow><msub><mrow><mi>t</mi></mrow><mrow><mi mathvariant="normal">SteadyState</mi></mrow></msub></mrow></mfrac></math></span>. <em>F</em> was then used together with the cultivation volume (<em>V</em><sub>Culture</sub> = 20 mL) to calculate dilution rate and thus growth rate, following <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq3"><math id="d33e3029"><mi>μ</mi><mo>=</mo><mi>D</mi><mo>=</mo><mfrac><mrow><mi>F</mi></mrow><mrow><msub><mrow><mi>V</mi></mrow><mrow><mi mathvariant="normal">Culture</mi></mrow></msub></mrow></mfrac></math></span>.</p></section><section id="Sec16"><h3 class="pmc_sec_title">Sample preparation and single-particle tracking photoactivatable localisation microscopy</h3>
<p id="Par44">To perform sptPALM of DnaA in live <em>E. coli</em>, 10 mL of cells were collected in a 50 mL Falcon tube from turbidostat cultures growing in one of the three different growth regimes (slow, intermediate, fast). Cells were then washed three times in PBS before removing the supernatant and resuspending the pellet in 50 μL of PBS. 1–2 μL of the final cell suspension were immobilised on M9 agarose pads between two heat-treated glass coverslips (#1.5H, 170 μm thickness).</p>
<p id="Par45">All sptPALM experiments were performed at room temperature using the miCube open microscopy framework<sup><a href="#CR102" class="usa-link" aria-describedby="CR102">102</a></sup>. Briefly, the microscope mounted an Omicron laser engine, a Nikon TIRF objective (100x, oil immersion, 1.49 NA, HP-SR) and an Andor Zyla 4.2 PLUS camera running at a 10 Hz for brightfield imaging acquisition and 100 Hz for sptPALM. For each imaging experiment, 300 frames were acquired at 100 ms intervals with brightfield illumination using a commercial LED light (INREDA, IKEA, Sweden). For sptPALM, individual videos of 30,000 frames were acquired at 10 ms intervals. Multiple videos were collected for each field of view, until exhaustion of fluorophores. The Single Molecule Imaging Laser Engine (SMILE) software was used to control the lasers (<a href="https://hohlbeinlab.github.io/miCube/LaserTrack_Arduino.html" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://hohlbeinlab.github.io/miCube/LaserTrack_Arduino.html</a>). A 561 nm laser with ~0.12 W/cm<sup>2</sup> power output was used for HiLo-to-TIRF illumination with 4 ms stroboscopic illumination in the middle of 10 frames. A 405 nm laser was used to activate the PAmCherry2.1 fluorophores. The 405 nm laser was initially provided with low-power (μW/cm<sup>2</sup> range) and with a 0.5 ms stroboscopic illumination at the beginning of 10 ms frames<sup><a href="#CR103" class="usa-link" aria-describedby="CR103">103</a></sup>. Both the power and the stroboscopic illumination were progressively increased throughout the imaging until exhaustion of fluorophores. Raw data was acquired using Micro-Manager<sup><a href="#CR104" class="usa-link" aria-describedby="CR104">104</a></sup>. During acquisition, a 2 × 2 binning was used, yielding an effective pixel size of 119 × 119 nm. The excitation field of ~30 × 30 µm was restricted to regions of interest of 256 × 256 pixels or smaller during imaging. The first 500 frames of each video were discarded, to prevent attempted localisation of overlapping fluorophores and to pre-bleach fluorescent contaminants in and around the cells.</p></section><section id="Sec17"><h3 class="pmc_sec_title">Cells segmentation and cell area and volume estimation</h3>
<p id="Par46">Cell segmentation was performed as previously described<sup><a href="#CR44" class="usa-link" aria-describedby="CR44">44</a></sup> using ImageJ<sup><a href="#CR105" class="usa-link" aria-describedby="CR105">105</a></sup>/Fiji<sup><a href="#CR106" class="usa-link" aria-describedby="CR106">106</a></sup>. Briefly, watershed-based segmentation<sup><a href="#CR107" class="usa-link" aria-describedby="CR107">107</a></sup> (<a href="http://imagej.net/Interactive_Watershed" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://imagej.net/Interactive_Watershed</a>) was performed to obtain pixel-accurate cell outlines from the brightfield images. From there, a value for the area in pixels of each identified cell was obtained and the camera pixel size (119 × 119 nm) was used to generate the corresponding area measurements in µm<sup>2</sup>. The brightfield images, together with their segmented counterparts are available on Zenodo (10.5281/zenodo.13930507). A list of obtained cellular area is available in the Source Data file.</p>
<p id="Par47">When cell volume was needed to estimate volume of initiation, previously reported<sup><a href="#CR108" class="usa-link" aria-describedby="CR108">108</a></sup> aspect ratios (<em>AR</em>) of <em>E. coli</em> cells cultured in growth conditions similar to the ones in this study were used (Slow: <em>μ</em> = 0.2 h<sup>−</sup><sup>1</sup> and <em>AR</em> = 3.43; Intermediate: <em>μ</em> = 0.7 h<sup>−1</sup> and <em>AR</em> = 4.14; Fast: <em>μ</em> = 1.4 h<sup>−1</sup> and AR = 4.3). <em>E. coli</em> cells were considered as cylinders with a hemisphere at each pole and a relationship between area of <em>E. coli</em> (<em>A</em>) and radius (<em>r</em>) of the sphere and cylinder was obtained as <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq4"><math id="d33e3160"><mi>r</mi><mo>=</mo><msqrt><mrow><mfrac><mrow><mi>A</mi></mrow><mrow><mi mathvariant="normal">π</mi><mo>+</mo><mn>4</mn><mo>⋅</mo><mi>A</mi><mi>R</mi><mo>−</mo><mn>4</mn></mrow></mfrac></mrow></msqrt></math></span>. From here, volume values were obtained as <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq5"><math id="d33e3185"><mi>V</mi><mo>=</mo><mrow><mo>(</mo><mrow><mn>2</mn><mo>⋅</mo><mi>A</mi><mi>R</mi><mo>−</mo><mn>0.7</mn></mrow><mo>)</mo></mrow><mo>⋅</mo><mi mathvariant="normal">π</mi><mo>⋅</mo><msup><mrow><msqrt><mrow><mfrac><mrow><mi>A</mi></mrow><mrow><mi mathvariant="normal">π</mi><mo>+</mo><mn>4</mn><mo>⋅</mo><mi>A</mi><mi>R</mi><mo>−</mo><mn>4</mn></mrow></mfrac></mrow></msqrt></mrow><mrow><mspace width="0.25em"></mspace><mn>3</mn></mrow></msup></math></span>. The volume of initiation (<em>V</em> *) was then obtained using the mathematical relationship <em>V</em> * = <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq6"><math id="d33e3236"><mfrac><mrow><mfenced close="⟩" open="⟨"><mrow><mi>V</mi></mrow></mfenced></mrow><mrow><mi mathvariant="normal">ln</mi><mn>2</mn><mspace width="0.25em"></mspace><msub><mrow><mo>⋅</mo><mspace width="0.25em"></mspace><mi>n</mi></mrow><mrow><mi>o</mi><mi>r</mi><mi>i</mi></mrow></msub></mrow></mfrac></math></span> between average cellular volume <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq7"><math id="d33e3263"><mfenced close=")" open="("><mrow><mfenced close="⟩" open="⟨"><mrow><mi>V</mi></mrow></mfenced></mrow></mfenced></math></span> and the average number of origins (<em>n</em><sub><em>oriC</em></sub>). <em>V</em> * values were then converted back to area values using the relationship previously mentioned between the two values.</p></section><section id="Sec18"><h3 class="pmc_sec_title">Localisation of fluorophores and tracking of single particles</h3>
<p id="Par48">Single-molecule localisation was performed via the ImageJ/Fiji plugin ThunderSTORM<sup><a href="#CR109" class="usa-link" aria-describedby="CR109">109</a></sup>, with added maximum likelihood estimation-based single-molecule localisation algorithms. First, a 50-frame temporal median filter<sup><a href="#CR110" class="usa-link" aria-describedby="CR110">110</a></sup> (<a href="https://github.com/HohlbeinLab/FTM2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/HohlbeinLab/FTM2</a>) was applied to the sptPALM movies to correct background intensities<sup><a href="#CR111" class="usa-link" aria-describedby="CR111">111</a></sup>. Image filtering was performed through a difference-of-Gaussians filter (Sigma1 = 2 px, Sigma2 = 8 px). The approximate localisation of molecules was determined via a local maximum with peak intensity threshold of <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq8"><math id="d33e3306"><mi mathvariant="normal">std</mi><mrow><mo>(</mo><mrow><mi>W</mi><mi>a</mi><mi>v</mi><mi>e</mi><mo>.</mo><mi>F</mi><mn>1</mn></mrow><mo>)</mo></mrow><mo>⋅</mo><mn>1.2</mn></math></span> and 8-neighbourhood connectivity. Sub-pixel localisation was performed through Gaussian-based maximum likelihood estimation, with a fit radius of 4 pixels (Sigma = 1.5 px). A custom-written, MATLAB-based pipeline was used to process and analyse the imaging data. A complete list of localisations for each replicate and condition, together with the custom-written MATLAB pipeline is available on Zenodo (10.5281/zenodo.13930507).</p>
<p id="Par49">Different output files from ThunderSTORM were combined when multiple videos had to be recorded for the same field of view. Localisations were then assigned a cell ID if they fell inside a cell and were discarded if not. Single, valid localisations were linked into tracks according to spatial and temporal distances. The tracking procedure was performed as previously reported<sup><a href="#CR102" class="usa-link" aria-describedby="CR102">102</a></sup> and yielded the number of tracks observed in each single cell and an overall apparent diffusion coefficient distribution. For each track featuring at least three localisations, the apparent diffusion coefficient <em>D</em>* was obtained by calculating the mean square displacement between the first <em>n</em> steps and taking the average of that, where <em>n</em> is the number of localisations minus one. The diffusion coefficients of all tracks were then collected into 85 logarithmic-divided bins from <em>D*</em> = 0.04 μm<sup>2</sup>/s to <em>D*</em> = 10 μm<sup>2</sup>/s.</p></section><section id="Sec19"><h3 class="pmc_sec_title">Monte-Carlo diffusion distribution analysis of DnaA</h3>
<p id="Par50">To interpret the diffusional distributions obtained for DnaA-PAmCherry2.1 fusion mutants, a set number of proteins was simulated moving between a bound and a free state in a two linear state model (50,000 for the fit, 250,000 for the visualisation), using Monte-Carlo diffusion distribution analysis (MC-DDA)<sup><a href="#CR44" class="usa-link" aria-describedby="CR44">44</a></sup>. The distributions were then fitted with a general Levenberg-Marquardt procedure in MATLAB, yielding kinetic rates with a 95% confidence interval. A single value for localisation uncertainty was used (<em>σ</em> = 0.035 μm, or <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq9"><math id="d33e3371"><msubsup><mrow><mi>D</mi></mrow><mrow><mi mathvariant="normal">Immobile</mi></mrow><mrow><mo>*</mo></mrow></msubsup></math></span> = 0.12 μm<sup>2</sup>/s). DnaA was assigned a <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq10"><math id="d33e3389"><msubsup><mrow><mi>D</mi></mrow><mrow><mi mathvariant="normal">Free</mi></mrow><mrow><mo>*</mo></mrow></msubsup></math></span> diffusion constant value for its free state and a <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq11"><math id="d33e3406"><msubsup><mrow><mi>D</mi></mrow><mrow><mi mathvariant="normal">Bound</mi></mrow><mrow><mo>*</mo></mrow></msubsup></math></span> diffusion constant value for its bound state (see below for an explanation of the values). Additionally, initial rates <em>k</em><sub>i</sub> governing the transitions between the free and bound state (<em>k</em><sub>Free</sub>→ <sub>Bound</sub>, <em>k</em><sub>Bound</sub>→<sub>Free</sub>) were assigned to the proteins. The proteins were randomly placed inside a cell, simulated as a cylinder (length 2 μm and radius 0.5 μm) with two hemispheres at its poles (radius 0.5 μm). Each protein is randomly put in one of the two different states, based on the probability set by their kinetic rates <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq12"><math id="d33e3439"><mfenced close=")" open="("><mrow><msub><mrow><mi>p</mi></mrow><mrow><mi mathvariant="normal">Free</mi></mrow></msub><mo>=</mo><mfrac><mrow><msub><mrow><mi>k</mi></mrow><mrow><mi mathvariant="normal">Bound</mi><mo>→</mo><mi mathvariant="normal">Free</mi></mrow></msub></mrow><mrow><msub><mrow><mi>k</mi></mrow><mrow><mi mathvariant="normal">Bound</mi><mo>→</mo><mi mathvariant="normal">Free</mi></mrow></msub><mspace width="0.25em"></mspace><mo>+</mo><mspace width="0.25em"></mspace><msub><mrow><mi>k</mi></mrow><mrow><mi mathvariant="normal">Free</mi><mo>→</mo><mi mathvariant="normal">Bound</mi></mrow></msub></mrow></mfrac><mo>,</mo><msub><mrow><mi>p</mi></mrow><mrow><mi mathvariant="normal">Bound</mi></mrow></msub><mo>=</mo><mfrac><mrow><msub><mrow><mi>k</mi></mrow><mrow><mi mathvariant="normal">Free</mi><mo>→</mo><mi mathvariant="normal">Bound</mi></mrow></msub></mrow><mrow><msub><mrow><mi>k</mi></mrow><mrow><mi mathvariant="normal">Bound</mi><mo>→</mo><mi mathvariant="normal">Free</mi></mrow></msub><mspace width="0.25em"></mspace><mo>+</mo><mspace width="0.25em"></mspace><msub><mrow><mi>k</mi></mrow><mrow><mi mathvariant="normal">Free</mi><mo>→</mo><mi mathvariant="normal">Bound</mi></mrow></msub></mrow></mfrac></mrow></mfenced></math></span>. From there, the proteins are given a time before they are changed to a different state, defined as state-change time <em>t</em><sub>Change</sub> and calculated as <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq13"><math id="d33e3534"><msub><mrow><mi>t</mi></mrow><mrow><mi mathvariant="normal">Change</mi></mrow></msub><mo>=</mo><mfrac><mrow><mi>log</mi><mrow><mo>(</mo><mrow><mi>r</mi><mi>a</mi><mi>n</mi><mi>d</mi></mrow><mo>)</mo></mrow></mrow><mrow><mo>−</mo><mi>k</mi></mrow></mfrac></math></span>, where <em>rand</em> is an evenly distributed random number and <em>k</em> is the kinetic rate governing the transition between the current and the next state. The movement of each protein is simulated with over-sampling with regards to the frame-time (<em>t</em><sub>Frame</sub> = 10 ms, <em>t</em><sub>Step</sub> = 0.1 ms) and their state is recorded every <em>t</em><sub>Frame</sub> interval. In their free state, each DnaA protein moves for a distance (<em>s</em>) equal to a randomly sampled normal distribution, centred around <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq14"><math id="d33e3586"><msubsup><mrow><mi>D</mi></mrow><mrow><mi mathvariant="normal">Free</mi></mrow><mrow><mo>*</mo></mrow></msubsup></math></span> and with <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq15"><math id="d33e3602"><mi>s</mi><mo>=</mo><msqrt><mrow><mn>2</mn><mo>⋅</mo><msubsup><mrow><mi>D</mi></mrow><mrow><mi mathvariant="normal">Free</mi></mrow><mrow><mo>*</mo></mrow></msubsup><mo>⋅</mo><msub><mrow><mi>t</mi></mrow><mrow><mi mathvariant="normal">step</mi></mrow></msub></mrow></msqrt></math></span>. At every step, the <em>t</em><sub>Change</sub> is subtracted with the <em>t</em><sub>Step</sub>. If the value becomes ≤ 0, the protein switches its state and new diffusion coefficient and <em>t</em><sub>Change</sub> are assigned. Every 10 ms, the current location of the proteins is convoluted with a random localisation error, taken from a randomly sampled normal distribution with a localisation uncertainty of σ = 0.035 μm. Each simulated protein had a pre-determined number of localisations, leading to simulated tracks ranging from 1 to 8 steps. The number of tracks of each length follows an exponential decay with a mean track length of three steps, following previous experimental observations of PAmCherry2<sup><a href="#CR112" class="usa-link" aria-describedby="CR112">112</a></sup>. The <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq16"><math id="d33e3648"><msubsup><mrow><mi>D</mi></mrow><mrow><mi mathvariant="normal">Free</mi></mrow><mrow><mo>*</mo></mrow></msubsup></math></span> value assigned to DnaA was 2.7 μm<sup>2</sup>/s, based on its hydrodynamic radius and accounting for cytoplasmic retardation due to crowding and viscosity ( ~20x)<sup><a href="#CR113" class="usa-link" aria-describedby="CR113">113</a></sup>. The value of the DnaA hydrodynamic radius ( ~2.48 nm) was obtained using the in-silico tool HullRad<sup><a href="#CR114" class="usa-link" aria-describedby="CR114">114</a></sup>, providing as input a predicted crystal structure of the DnaA-PAmCherry2.1 fusion mutant, generated through AlphaFold<sup><a href="#CR115" class="usa-link" aria-describedby="CR115">115</a></sup>/ColabFold<sup><a href="#CR116" class="usa-link" aria-describedby="CR116">116</a></sup>. This diffusion value is similar to previous estimates of diffusion coefficients of proteins moving within the bacterial cytoplasm<sup><a href="#CR117" class="usa-link" aria-describedby="CR117">117</a>–<a href="#CR119" class="usa-link" aria-describedby="CR119">119</a></sup>. We further confirmed this value by using the pDnaA211-PAmCherry2.1 plasmid to express DnaA211<sup><a href="#CR99" class="usa-link" aria-describedby="CR99">99</a></sup>, a DnaA mutant impaired in DNA binding and thus solely moving in a free state. We then analysed the diffusional distributions of the protein as described above, simulating a static species (<em>k</em><sub>Free</sub>→<sub>Bound</sub> = <em>k</em><sub>Bound</sub>→<sub>Free</sub> = 0 s<sup>−1</sup>) and estimating diffusion coefficients that would best fit the experimental data. In this way, we obtained a diffusion coefficient of 2.6 ± 0.2 μm<sup>2</sup>/s across all growth regimes (Supplementary Fig. <a href="#MOESM1" class="usa-link">4F</a>). In our set-up, proteins bound to DNA appear as immobile<sup><a href="#CR44" class="usa-link" aria-describedby="CR44">44</a>,<a href="#CR102" class="usa-link" aria-describedby="CR102">102</a></sup>; therefore, DnaA was considered as immobile in its DNA-bound state, with a diffusion coefficient solely determined by the localisation uncertainty (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq17"><math id="d33e3722"><msubsup><mrow><mi>D</mi></mrow><mrow><mi mathvariant="normal">Bound</mi></mrow><mrow><mo>*</mo></mrow></msubsup></math></span> = <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq18"><math id="d33e3738"><msubsup><mrow><mi>D</mi></mrow><mrow><mi mathvariant="normal">Immobile</mi></mrow><mrow><mo>*</mo></mrow></msubsup></math></span> = 0.12 µm<sup>2</sup>/s). The bound fraction of DnaA was then calculated using the obtained kinetic rates as <em>p</em><sub>Bound</sub> = <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq19"><math id="d33e3761"><mfrac><mrow><msub><mrow><mi>k</mi></mrow><mrow><mi mathvariant="normal">Free</mi><mo>→</mo><mi mathvariant="normal">bound</mi></mrow></msub></mrow><mrow><msub><mrow><mi>k</mi></mrow><mrow><mi mathvariant="normal">Bound</mi><mo>→</mo><mi mathvariant="normal">free</mi></mrow></msub><mo>+</mo><mspace width="0.25em"></mspace><msub><mrow><mi>k</mi></mrow><mrow><mi mathvariant="normal">Free</mi><mo>→</mo><mi mathvariant="normal">bound</mi></mrow></msub></mrow></mfrac></math></span>. The result was then multiplied by 100 to display it as a percentage.</p></section><section id="Sec20"><h3 class="pmc_sec_title">Analysis of DnaA behaviour throughout the cell cycle</h3>
<p id="Par51">To study the difference in behaviour of DnaA across the cell cycle, we assigned to each cell a single value of average diffusion coefficient. This value was obtained by summing the diffusion coefficient of tracks a single cell contained and then dividing it by the number of tracks. We used the area to probe the different phases of the cell cycle, by plotting the obtained average diffusion coefficient over the area of the cell. We then calculated and displayed rolling medians, and 25 and 75% quantiles in R v4.4.1 with the zoo v1.8-12 package<sup><a href="#CR89" class="usa-link" aria-describedby="CR89">89</a></sup>, using a rolling window size of ‘k = 75’. Distributions of cell areas were also obtained.</p></section><section id="Sec21"><h3 class="pmc_sec_title">Flow cytometry estimation of number of origins</h3>
<p id="Par52">To obtain the average number of origins present in an <em>E. coli</em> population, both reference cells (wild-type in slow growth regime, 1 to 2 copies) and experimental cells (mutant strains in different growth regimes) samples were prepared<sup><a href="#CR7" class="usa-link" aria-describedby="CR7">7</a></sup>. In both cases, turbidostat cultures of <em>E. coli</em> were subjected to run-out replication. Rifampicin (150 μg/mL final concentration) and cephalexin (15 μg/mL final concentration) were added, the turbidostat fermentation was stopped and cells were left incubating with constant stirring and temperature for additional 6-6.5 h. Then, 2 mL of the final culture were first washed in the same volume of PBS, subsequently added to 18 mL of ice-cold 70% V/V ethanol and stored at 4 °C for at least 12 h<sup><a href="#CR120" class="usa-link" aria-describedby="CR120">120</a></sup>. Reference cells were obtained from turbidostat cultures of wild-type <em>E. coli</em> MG1655 in slow growth regime, where cells typically cycle between 1 and 2 copies of the chromosome<sup><a href="#CR6" class="usa-link" aria-describedby="CR6">6</a></sup>. Experimental cells were obtained for all strains harbouring the <em>dnaA-PAmCherry2.1</em> gene in different growth regimes.</p>
<p id="Par53">Both reference and experimental cells samples were washed once with PBS and then resuspended in 1 mL of PBS in an Eppendorf tube. The membrane of the reference cells was then stained with MitoTracker<sup>TM</sup> Deep Red FM (Invitrogen<sup>TM</sup>) to a final concentration of 1 μg/mL and left incubating for 1–1.5 h at room temperature, in the dark, with constant shaking. Reference cells were then washed two times with PBS and finally resuspended in 1 mL of PBS. For absolute DNA content measurement, 25 μL of reference cells were mixed with 75 μL of experimental cells and the cell mixture was stained with 20 μL of 100x Quant-iT<sup>TM</sup> PicoGreen<sup>TM</sup> dsDNA reagent (Invitrogen<sup>TM</sup>) in 25% DMSO. The cells were incubated together with the nucleic acid dye for 30–60 min at room temperature in the dark. Finally, 200 μL of PBS was added to the samples.</p>
<p id="Par54">Analysis of DNA content was performed with an Attune™ NxT Flow Cytometer (Invitrogen<sup>TM</sup>) and the Attune™ proprietary software. For each sample, forward and side scatter measurements were obtained, together with emission filtered with a 530/30 nm transmission filter for DNA content measurement (BL1-H channel, voltage = 260) and emission filtered with a 695/40 nm transmission filter for MitoTracker<sup>TM</sup> measurement (YL3-H channel, voltage = 450). Reference cells were separated from experimental cells by gating the far-red emission. At least 10,000 single cells events were collected for the experimental cells. The DNA content of both the reference cells and the experimental cells populations were represented as a histogram versus fluorescence on the green channel. The intensity of the highest peak for the population of reference cells, representing two copies of the chromosomes after run-out replication, was used to quantify absolute number of origins. The intensity maximum of each peak was identified and the number of cells in a 500 A.U. range centred around the peak was used to calculate the average number of origins, asynchrony index and relative abundance of cells with one origin of each sample.</p></section><section id="Sec22"><h3 class="pmc_sec_title">Proteomic sample generation</h3>
<p id="Par55">To perform proteomics, 10 mL of turbidostat cultures of <em>E. coli</em> strains in different growth conditions were harvested. Cells were then washed twice in PBS before being pelleted and bacteria were lysed with 100% trifluoracetic acid (TFA) according to the SPEED protocol<sup><a href="#CR121" class="usa-link" aria-describedby="CR121">121</a></sup> with slight adaptations<sup><a href="#CR122" class="usa-link" aria-describedby="CR122">122</a></sup>. Briefly, 30 μL of 100% TFA was added to the pellet, incubated for 5 min at 55 °C and neutralised with 270 μL of 2 M Tris (pH was not adjusted). Lysed samples were quantified via Bradford assay. 50 μg total protein amount per sample were reduced and alkylated (9 mM tris(2-carboxyethyl)phosphine (TCEP) and 40 mM chloroacetamide (CAA)). Protein digestion was performed with a Trypsin (Roche)-to-protein ratio of 1:50 overnight at 37 °C. Samples were desalted by solid-phase extraction (C18) and stored after lyophilisation at −80 °C.</p></section><section id="Sec23"><h3 class="pmc_sec_title">Proteomic data acquisition</h3>
<p id="Par56">Peptides were analysed on a Vanquish Neo UHPLC (micro-flow configuration; Thermo Fisher Scientific) coupled to an Orbitrap Exploris 480 mass spectrometer (Thermo Fisher Scientific). Around 25 μg peptides were applied directly onto a commercially available Acclaim PepMap 100 C18 column (2 μm particle size, 1 mm ID x 150 mm, 100 Å pore size; Thermo Fisher Scientific) and separated using a 60 min linear gradient ranging from 3% to 28% solvent B (0.1% FA, 3% DMSO in ACN) in solvent A (0.1% FA, 3% DMSO in HPLC grade water) at a flow rate of 50 μL/min. The mass spectrometer was operated in data-independent acquisition (DIA) mode. MS1 full scans (360–1300 m/z) were acquired with a resolution of 120,000, a normalized AGC target value of 100% and a custom maximum injection time. The cycle time was set to 3 seconds. MS2 scans (200–1800 m/z) were acquired over 40 DIA segments with widths adjusted to the precursor density (see Supplementary Table <a href="#MOESM1" class="usa-link">8</a> in Wu et al.<sup><a href="#CR123" class="usa-link" aria-describedby="CR123">123</a></sup>). The scan resolution in the Orbitrap was set to 15,000 with a normalized AGC target value of 100% and a custom maximum injection time. The HCD collision energy was set to 30%.</p></section><section id="Sec24"><h3 class="pmc_sec_title">Proteomic data analysis</h3>
<p id="Par57">The mass spectrometric raw files were analyzed with DIA-NN<sup><a href="#CR124" class="usa-link" aria-describedby="CR124">124</a></sup> (version 1.8) using the Uniprot reference fasta file for <em>E. coli</em> K-12 (UP000000625, download 04.06.2023, 4403 protein entries) appended with common contaminants. FASTA digest for library-free search and deep learning-based spectra was enabled. N-term M excision, C carbamidomethylation, Ox(M), and Ac(N-term) were activated. The peptide length range was set from 7 to 30, precursor charge ranged from 2 to 6, precursor m/z ranged from 360 to 1300 and fragment ion m/z ranged from 200 to 1800. Usage of isotopologues and match-between runs (MBR) were enabled. Precursor FDR was set to 1%. All other parameters were kept as default. The DIA-NN report table “report.pg_matrix” was imported into the data visualisation platform omicsViewer<sup><a href="#CR125" class="usa-link" aria-describedby="CR125">125</a></sup>, where proteins were filtered for being quantified in all three replicate measurements of at least one experimental condition. Finally, remaining missing values in the data matrix were imputed by a protein-specific constant value, which was defined as the lowest detected value over all samples divided by two. Additionally, a maximal imputed intensity value was defined as 15% quantile of the protein distribution from the complete dataset.</p></section><section id="Sec25"><h3 class="pmc_sec_title">LacZ β-galactosidase assay</h3>
<p id="Par58">To use the activity of the LacZ β-galactosidase as a proxy for DnaA-ATP/DnaA-ADP ratio, 10 mL of turbidostat cultures of <em>E. coli</em> strains carrying the pdnaAp-LacZ plasmid in different growth conditions were harvested. Cells were permeabilised using B-PER™ Bacterial Protein Extraction Reagent (ThermoFisher Scientific™) following manufacturer instructions. Permeabilised cells were then diluted 10 times before addition of <em>o-</em>nitrophenyl-β-D-galactopyranoside to a final concentration of 0.7 mg/mL. The β-galactosidase activity was determined as previously described<sup><a href="#CR126" class="usa-link" aria-describedby="CR126">126</a></sup>. Briefly, absorbance at 420 nm was measured using a Synergy H1 microplate reader (BioTek<sup>TM</sup>) and then normalised for the average signal obtained from the wild-type strain in the same growth condition.</p></section><section id="Sec26"><h3 class="pmc_sec_title">Reporting summary</h3>
<p id="Par59">Further information on research design is available in the <a href="#MOESM4" class="usa-link">Nature Portfolio Reporting Summary</a> linked to this article.</p></section></section><section id="Sec27"><h2 class="pmc_sec_title">Supplementary information</h2>
<section class="sm xbox font-sm" id="MOESM1"><div class="media p"><div class="caption">
<a href="/articles/instance/12371093/bin/41467_2025_63147_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information</a><sup> (1.8MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM2"><div class="media p"><div class="caption">
<a href="/articles/instance/12371093/bin/41467_2025_63147_MOESM2_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">41467_2025_63147_MOESM2_ESM.pdf</a><sup> (39.8KB, pdf) </sup><p>Description of Additional Supplementary Files</p>
</div></div></section><section class="sm xbox font-sm" id="MOESM3"><div class="media p"><div class="caption">
<a href="/articles/instance/12371093/bin/41467_2025_63147_MOESM3_ESM.mov" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Movie 1</a><sup> (27.8MB, mov) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM4"><div class="media p"><div class="caption">
<a href="/articles/instance/12371093/bin/41467_2025_63147_MOESM4_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Reporting Summary</a><sup> (88.9KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM5"><div class="media p"><div class="caption">
<a href="/articles/instance/12371093/bin/41467_2025_63147_MOESM5_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Transparent Peer Review file</a><sup> (708.9KB, pdf) </sup>
</div></div></section></section><section id="Sec28"><h2 class="pmc_sec_title">Source data</h2>
<section class="sm xbox font-sm" id="MOESM6"><div class="media p"><div class="caption">
<a href="/articles/instance/12371093/bin/41467_2025_63147_MOESM6_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Source Data</a><sup> (2.4MB, xlsx) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>We want to thank Dr. Mareike Berger for the inspiring discussions at every stage of the study. We would like to thank prof. Anders Løbner-Olesen (University of Copenhagen) for kindly providing us with the <em>E. coli</em> strains carrying deletions for <em>datA, DARS1</em> and <em>DARS2</em>, alone or in combination. We would like to thank Franziska Hackbarth for excellent technical assistance and maintenance of mass spectrometers. L.O. thanks Ricardo Villegas-Warren for his help in generating the predicted crystal structure of DnaA-PAmCherry2.1, Tim Althuis for his guidance in preparing the samples for the proteomics analysis and Dr. Charlotte Koster for her guidance in performing the LacZ assays. L.O., N.J.C., J.vd.O and P.R.t.W. acknowledge financial support from The Netherlands Organization of Scientific Research (NWO/OCW) Gravitation program Building a Synthetic Cell (BaSyC) (024.003.019). R.H.J.S. is supported by a VIDI grant (VI.Vidi.203.074) from NWO. T.J.G.E. is supported by a European Research Council Consolidator Grant (817834), a VICI grant from the Netherlands Organization of Scientific Research (VI.C.192.016) and a Volkswagen Foundation ‘Life’ grant (96725). The Exploris 480 mass spectrometer was funded in part by the German Research Foundation (INST 95/1435-1 FUGG).</p></section><section id="notes1"><h2 class="pmc_sec_title">Author contributions</h2>
<p>L.O. designed the research. L.O., C.L. and M.L. performed experiments. S.K. generated in silico data. S.K., J.H. and C.L. wrote the scripts used to generate and analyse data. L.O., S.K. and C.L. analysed the data. L.O. and S.K. drafted the manuscript and produced figures. L.O., J.H., R.H.J.S. supervised the work. L.O., S.K., C.L., M.L., N.J.C., T.J.G.E., J.v.d.O., P.R.t.W., J.H., R.H.J.S. contributed to editing the manuscript.</p></section><section id="notes2"><h2 class="pmc_sec_title">Peer review</h2>
<section id="FPar1"><h3 class="pmc_sec_title">Peer review information</h3>
<p id="Par60"><em>Nature Communications</em> thanks Tsutomu Katayama and the other, anonymous, reviewers for their contribution to the peer review of this work. A peer review file is available.</p></section></section><section id="notes3"><h2 class="pmc_sec_title">Data availability</h2>
<p>All the localisation data and related brightfield images used during sptPALM in this study have been deposited on Zenodo (10.5281/zenodo.13939193). A list of the matches to the DnaA box sequences on the genome of all analysed <em>E. coli</em> strains and all cellular and single-molecule parameters generated in this study are provided in the Source Data file. The mass spectrometric raw files, as well as the DIA-NN output files have been deposited to the ProteomeXchange Consortium via the PRIDE partner repository and can be accessed using the identifier <a href="https://proteomecentral.proteomexchange.org/cgi/GetDataset?ID=PXD064841" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PXD064841</a>. All the mutant <em>E. coli</em> strains carrying the <em>dnaA-PAmCherry2.1</em> locus are available upon request. <a href="#Sec28" class="usa-link">Source data</a> are provided with this paper.</p></section><section id="notes4"><h2 class="pmc_sec_title">Code availability</h2>
<p>Scripts used in this study to analyse position of DnaA boxes across genomes and to generate single-particle tracking data have been deposited on Zenodo (10.5281/zenodo.13939193). The scripts used to analyse the disposition of DnaA boxes on the genomes of <em>E. coli</em> and <em>S. enterica</em> are also available on GitHub (<a href="https://github.com/stephkoest/Ecoli_titration" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/stephkoest/Ecoli_titration</a>).</p></section><section id="FPar2"><h2 class="pmc_sec_title">Competing interests</h2>
<p id="Par61">The authors declare no competing interests.</p></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="fn1"><p><strong>Publisher’s note</strong> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></div></div></section><section id="_ci93_" lang="en" class="contrib-info"><h2 class="pmc_sec_title">Contributor Information</h2>
<p>Lorenzo Olivi, Email: lorenzo.olivi@wur.nl, Email: lorenzo.olivi.1994@gmail.com.</p>
<p>Johannes Hohlbein, Email: johannes.hohlbein@wur.nl.</p>
<p>Raymond H. J. Staals, Email: raymond.staals@wur.nl</p></section><section id="sec29"><h2 class="pmc_sec_title">Supplementary information</h2>
<p>The online version contains supplementary material available at 10.1038/s41467-025-63147-1.</p></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Willis, L. &amp; Huang, K. C. Sizing up the bacterial cell cycle. <em>Nat. Rev. Microbiol.</em><strong>15</strong>, 606–620 (2017).
</cite> [<a href="https://doi.org/10.1038/nrmicro.2017.79" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28804128/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Willis,%20L.%20&amp;%20Huang,%20K.%20C.%20Sizing%20up%20the%20bacterial%20cell%20cycle.%20Nat.%20Rev.%20Microbiol.15,%20606%E2%80%93620%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>Reyes-Lamothe, R. &amp; Sherratt, D. J. The bacterial cell cycle, chromosome inheritance and cell growth. <em>Nat. Rev. Microbiol.</em><strong>17</strong>, 467–478 (2019).
</cite> [<a href="https://doi.org/10.1038/s41579-019-0212-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31164753/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Reyes-Lamothe,%20R.%20&amp;%20Sherratt,%20D.%20J.%20The%20bacterial%20cell%20cycle,%20chromosome%20inheritance%20and%20cell%20growth.%20Nat.%20Rev.%20Microbiol.17,%20467%E2%80%93478%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>Olivi, L. et al. Towards a synthetic cell cycle. <em>Nat. Commun.</em><strong>12</strong>, 1–11 (2021).
</cite> [<a href="https://doi.org/10.1038/s41467-021-24772-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8313558/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34312383/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Olivi,%20L.%20et%20al.%20Towards%20a%20synthetic%20cell%20cycle.%20Nat.%20Commun.12,%201%E2%80%9311%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Helmstetter, C. E. &amp; Cooper, S. DNA synthesis during the division cycle of rapidly growing <em>Escherichia coli</em> B/r. <em>J. Mol. Biol.</em><strong>31</strong>, 507–518 (1968).
</cite> [<a href="https://doi.org/10.1016/0022-2836(68)90424-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/4866336/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Helmstetter,%20C.%20E.%20&amp;%20Cooper,%20S.%20DNA%20synthesis%20during%20the%20division%20cycle%20of%20rapidly%20growing%20Escherichia%20coli%20B/r.%20J.%20Mol.%20Biol.31,%20507%E2%80%93518%20(1968)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Cooper, S. &amp; Helmstetter, C. E. Chromosome replication and the division cycle of <em>Escherichia</em>. <em>J. Mol. Biol.</em><strong>31</strong>, 519–540 (1968).
</cite> [<a href="https://doi.org/10.1016/0022-2836(68)90425-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/4866337/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cooper,%20S.%20&amp;%20Helmstetter,%20C.%20E.%20Chromosome%20replication%20and%20the%20division%20cycle%20of%20Escherichia.%20J.%20Mol.%20Biol.31,%20519%E2%80%93540%20(1968)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Wallden, M., Fange, D., Lundius, E. G., Baltekin, Ö &amp; Elf, J. The Synchronization of Replication and Division Cycles in Individual <em>E. coli</em>. <em>Cells Cell</em><strong>166</strong>, 729–739 (2016).
</cite> [<a href="https://doi.org/10.1016/j.cell.2016.06.052" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27471967/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wallden,%20M.,%20Fange,%20D.,%20Lundius,%20E.%20G.,%20Baltekin,%20%C3%96%20&amp;%20Elf,%20J.%20The%20Synchronization%20of%20Replication%20and%20Division%20Cycles%20in%20Individual%20E.%20coli.%20Cells%20Cell166,%20729%E2%80%93739%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Si, F. et al. Invariance of Initiation Mass and Predictability of Cell Size in <em>Escherichia coli</em>. <em>Curr. Biol.</em><strong>27</strong>, 1278–1287 (2017).
</cite> [<a href="https://doi.org/10.1016/j.cub.2017.03.022" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5474944/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28416114/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Si,%20F.%20et%20al.%20Invariance%20of%20Initiation%20Mass%20and%20Predictability%20of%20Cell%20Size%20in%20Escherichia%20coli.%20Curr.%20Biol.27,%201278%E2%80%931287%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>Donachie, W. D. Relationship between Cell Size and Time of Initiation of DNA Replication. <em>Nature</em><strong>219</strong>, 1077–1079 (1968).
</cite> [<a href="https://doi.org/10.1038/2191077a0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/4876941/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Donachie,%20W.%20D.%20Relationship%20between%20Cell%20Size%20and%20Time%20of%20Initiation%20of%20DNA%20Replication.%20Nature219,%201077%E2%80%931079%20(1968)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>Sture Wold, K. S., Steen, H. B., Stokke, T. &amp; Boye, E. The initiation mass for DNA replication in <em>Escherichia coli</em> K-12 is dependent on growth rate. <em>EMBO J.</em><strong>13</strong>, 2097–2102 (1994).
</cite> [<a href="https://doi.org/10.1002/j.1460-2075.1994.tb06485.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC395061/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/8187762/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sture%20Wold,%20K.%20S.,%20Steen,%20H.%20B.,%20Stokke,%20T.%20&amp;%20Boye,%20E.%20The%20initiation%20mass%20for%20DNA%20replication%20in%20Escherichia%20coli%20K-12%20is%20dependent%20on%20growth%20rate.%20EMBO%20J.13,%202097%E2%80%932102%20(1994)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>Zheng, H. et al. General quantitative relations linking cell growth and the cell cycle in <em>Escherichia coli</em>. <em>Nat. Microbiol. 2020 5:8</em><strong>5</strong>, 995–1001 (2020).</cite> [<a href="https://doi.org/10.1038/s41564-020-0717-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8059327/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32424336/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zheng,%20H.%20et%20al.%20General%20quantitative%20relations%20linking%20cell%20growth%20and%20the%20cell%20cycle%20in%20Escherichia%20coli.%20Nat.%20Microbiol.%202020%205:85,%20995%E2%80%931001%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Katayama, T., Kasho, K. &amp; Kawakami, H. The DnaA Cycle in Escherichia coli: Activation, Function and Inactivation of the Initiator Protein. <em>Front. Microbiol.</em><strong>8</strong>, (2017).</cite> [<a href="https://doi.org/10.3389/fmicb.2017.02496" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5742627/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29312202/" class="usa-link">PubMed</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Hansen, F. G. &amp; Atlung, T. The DnaA Tale. <em>Front. Microbiol.</em><strong>9</strong>, 334195 (2018).</cite> [<a href="https://doi.org/10.3389/fmicb.2018.00319" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5835720/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29541066/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hansen,%20F.%20G.%20&amp;%20Atlung,%20T.%20The%20DnaA%20Tale.%20Front.%20Microbiol.9,%20334195%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>Skarstad, K. &amp; Katayama, T. Regulating DNA Replication in Bacteria. <em>Cold Spring Harb. Perspect. Biol.</em><strong>5</strong>, 1–17 (2013).</cite> [<a href="https://doi.org/10.1101/cshperspect.a012922" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3683904/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23471435/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Skarstad,%20K.%20&amp;%20Katayama,%20T.%20Regulating%20DNA%20Replication%20in%20Bacteria.%20Cold%20Spring%20Harb.%20Perspect.%20Biol.5,%201%E2%80%9317%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>Riber, L., Frimodt-Møller, J., Charbon, G. &amp; Løbner-Olesen, A. Multiple DNA binding proteins contribute to timing of chromosome replication in <em>E. coli</em>. <em>Front. Mol. Biosci.</em><strong>3</strong>, 211230 (2016).</cite> [<a href="https://doi.org/10.3389/fmolb.2016.00029" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4924351/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27446932/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Riber,%20L.,%20Frimodt-M%C3%B8ller,%20J.,%20Charbon,%20G.%20&amp;%20L%C3%B8bner-Olesen,%20A.%20Multiple%20DNA%20binding%20proteins%20contribute%20to%20timing%20of%20chromosome%20replication%20in%20E.%20coli.%20Front.%20Mol.%20Biosci.3,%20211230%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>Dewachter, L., Verstraeten, N., Fauvart, M. &amp; Michiels, J. An integrative view of cell cycle control in <em>Escherichia coli</em>. <em>FEMS Microbiol. Rev.</em><strong>42</strong>, 116–136 (2018).
</cite> [<a href="https://doi.org/10.1093/femsre/fuy005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29365084/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Dewachter,%20L.,%20Verstraeten,%20N.,%20Fauvart,%20M.%20&amp;%20Michiels,%20J.%20An%20integrative%20view%20of%20cell%20cycle%20control%20in%20Escherichia%20coli.%20FEMS%20Microbiol.%20Rev.42,%20116%E2%80%93136%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Speck, C., Weigel, C. &amp; Messer, W. ATP– and ADP–DnaA protein, a molecular switch in gene regulation. <em>EMBO J.</em><strong>18</strong>, 6169–6176 (1999).
</cite> [<a href="https://doi.org/10.1093/emboj/18.21.6169" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1171680/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10545126/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Speck,%20C.,%20Weigel,%20C.%20&amp;%20Messer,%20W.%20ATP%E2%80%93%20and%20ADP%E2%80%93DnaA%20protein,%20a%20molecular%20switch%20in%20gene%20regulation.%20EMBO%20J.18,%206169%E2%80%936176%20(1999)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>Boesen, T. O. et al. Dispensability of extrinsic DnaA regulators in Escherichia coli cell-cycle control. <em>Proc. Natl Acad. Sci.</em><strong>121</strong>, (2024).</cite> [<a href="https://doi.org/10.1073/pnas.2322772121" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11331064/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40014855/" class="usa-link">PubMed</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Roth, A. &amp; Messer, W. High-affinity binding sites for the initiator protein DnaA on the chromosome of <em>Escherichia coli</em>. <em>Mol. Microbiol.</em><strong>28</strong>, 395–401 (1998).
</cite> [<a href="https://doi.org/10.1046/j.1365-2958.1998.00813.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9622363/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Roth,%20A.%20&amp;%20Messer,%20W.%20High-affinity%20binding%20sites%20for%20the%20initiator%20protein%20DnaA%20on%20the%20chromosome%20of%20Escherichia%20coli.%20Mol.%20Microbiol.28,%20395%E2%80%93401%20(1998)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Rozgaja, T. A. et al. Two oppositely oriented arrays of low-affinity recognition sites in <em>oriC</em> guide progressive binding of DnaA during <em>Escherichiacoli</em> pre-RC assembly. <em>Mol. Microbiol.</em><strong>82</strong>, 475–488 (2011).
</cite> [<a href="https://doi.org/10.1111/j.1365-2958.2011.07827.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3192301/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21895796/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rozgaja,%20T.%20A.%20et%20al.%20Two%20oppositely%20oriented%20arrays%20of%20low-affinity%20recognition%20sites%20in%20oriC%20guide%20progressive%20binding%20of%20DnaA%20during%20Escherichiacoli%20pre-RC%20assembly.%20Mol.%20Microbiol.82,%20475%E2%80%93488%20(2011)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>Noguchi, Y., Sakiyama, Y., Kawakami, H. &amp; Katayama, T. The Arg Fingers of Key DnaA Protomers Are Oriented Inward within the Replication Origin <em>oriC</em> and Stimulate DnaA Subcomplexes in the Initiation Complex. <em>J. Biol. Chem.</em><strong>290</strong>, 20295–20312 (2015).
</cite> [<a href="https://doi.org/10.1074/jbc.M115.662601" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4536437/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26126826/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Noguchi,%20Y.,%20Sakiyama,%20Y.,%20Kawakami,%20H.%20&amp;%20Katayama,%20T.%20The%20Arg%20Fingers%20of%20Key%20DnaA%20Protomers%20Are%20Oriented%20Inward%20within%20the%20Replication%20Origin%20oriC%20and%20Stimulate%20DnaA%20Subcomplexes%20in%20the%20Initiation%20Complex.%20J.%20Biol.%20Chem.290,%2020295%E2%80%9320312%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>Sompayrac, L. et al. Autorepressor Model for Control of DNA Replication. <em>Nat. N. Biol.</em><strong>241</strong>, 133–135 (1973).</cite> [<a href="https://doi.org/10.1038/newbio241133a0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/4573387/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sompayrac,%20L.%20et%20al.%20Autorepressor%20Model%20for%20Control%20of%20DNA%20Replication.%20Nat.%20N.%20Biol.241,%20133%E2%80%93135%20(1973)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>Hansen, F. G., Christensen, B. B. &amp; Atlung, T. The initiator titration model: computer simulation of chromosome and minichromosome control. <em>Res. Microbiol.</em><strong>142</strong>, 161–167 (1991).
</cite> [<a href="https://doi.org/10.1016/0923-2508(91)90025-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/1925015/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hansen,%20F.%20G.,%20Christensen,%20B.%20B.%20&amp;%20Atlung,%20T.%20The%20initiator%20titration%20model:%20computer%20simulation%20of%20chromosome%20and%20minichromosome%20control.%20Res.%20Microbiol.142,%20161%E2%80%93167%20(1991)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>McGarry, K. C., Ryan, V. T., Grimwade, J. E. &amp; Leonard, A. C. Two discriminatory binding sites in the <em>Escherichia coli</em> replication origin are required for DNA strand opening by initiator DnaA-ATP. <em>Proc. Natl Acad. Sci. USA</em><strong>101</strong>, 2811–2816 (2004).
</cite> [<a href="https://doi.org/10.1073/pnas.0400340101" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC365702/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/14978287/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?McGarry,%20K.%20C.,%20Ryan,%20V.%20T.,%20Grimwade,%20J.%20E.%20&amp;%20Leonard,%20A.%20C.%20Two%20discriminatory%20binding%20sites%20in%20the%20Escherichia%20coli%20replication%20origin%20are%20required%20for%20DNA%20strand%20opening%20by%20initiator%20DnaA-ATP.%20Proc.%20Natl%20Acad.%20Sci.%20USA101,%202811%E2%80%932816%20(2004)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>Ozaki, S. et al. A common mechanism for the ATP-DnaA-dependent formation of open complexes at the replication origin. <em>J. Biol. Chem.</em><strong>283</strong>, 8351–8362 (2008).
</cite> [<a href="https://doi.org/10.1074/jbc.M708684200" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18216012/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ozaki,%20S.%20et%20al.%20A%20common%20mechanism%20for%20the%20ATP-DnaA-dependent%20formation%20of%20open%20complexes%20at%20the%20replication%20origin.%20J.%20Biol.%20Chem.283,%208351%E2%80%938362%20(2008)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>Katayama, T., Ozaki, S., Keyamura, K. &amp; Fujimitsu, K. Regulation of the replication cycle: conserved and diverse regulatory systems for DnaA and <em>oriC</em>. <em>Nat. Rev. Microbiol.</em><strong>8</strong>, 163–170 (2010).
</cite> [<a href="https://doi.org/10.1038/nrmicro2314" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20157337/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Katayama,%20T.,%20Ozaki,%20S.,%20Keyamura,%20K.%20&amp;%20Fujimitsu,%20K.%20Regulation%20of%20the%20replication%20cycle:%20conserved%20and%20diverse%20regulatory%20systems%20for%20DnaA%20and%20oriC.%20Nat.%20Rev.%20Microbiol.8,%20163%E2%80%93170%20(2010)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>Fujimitsu, K., Senriuchi, T. &amp; Katayama, T. Specific genomic sequences of <em>E. coli</em> promote replicational initiation by directly reactivating ADP-DnaA. <em>Genes Dev.</em><strong>23</strong>, 1221–1233 (2009).
</cite> [<a href="https://doi.org/10.1101/gad.1775809" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2685538/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19401329/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Fujimitsu,%20K.,%20Senriuchi,%20T.%20&amp;%20Katayama,%20T.%20Specific%20genomic%20sequences%20of%20E.%20coli%20promote%20replicational%20initiation%20by%20directly%20reactivating%20ADP-DnaA.%20Genes%20Dev.23,%201221%E2%80%931233%20(2009)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Sekimizu, K. &amp; Kornberg, A. Cardiolipin activation of dnaA protein, the initiation protein of replication in <em>Escherichia coli</em>. <em>J. Biol. Chem.</em><strong>263</strong>, 7131–7135 (1988).
</cite> [<a href="https://pubmed.ncbi.nlm.nih.gov/2835364/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sekimizu,%20K.%20&amp;%20Kornberg,%20A.%20Cardiolipin%20activation%20of%20dnaA%20protein,%20the%20initiation%20protein%20of%20replication%20in%20Escherichia%20coli.%20J.%20Biol.%20Chem.263,%207131%E2%80%937135%20(1988)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Kasho, K. &amp; Katayama, T. DnaA binding locus <em>datA</em> promotes DnaA-ATP hydrolysis to enable cell cycle-coordinated replication initiation. <em>Proc. Natl Acad. Sci.</em><strong>110</strong>, 936–941 (2013).
</cite> [<a href="https://doi.org/10.1073/pnas.1212070110" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3549119/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23277577/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kasho,%20K.%20&amp;%20Katayama,%20T.%20DnaA%20binding%20locus%20datA%20promotes%20DnaA-ATP%20hydrolysis%20to%20enable%20cell%20cycle-coordinated%20replication%20initiation.%20Proc.%20Natl%20Acad.%20Sci.110,%20936%E2%80%93941%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>Kato, J. I. &amp; Katayama, T. Hda, a novel DnaA-related protein, regulates the replication cycle in <em>Escherichia coli</em>. <em>EMBO J.</em><strong>20</strong>, 4253–4262 (2001).
</cite> [<a href="https://doi.org/10.1093/emboj/20.15.4253" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC149159/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11483528/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kato,%20J.%20I.%20&amp;%20Katayama,%20T.%20Hda,%20a%20novel%20DnaA-related%20protein,%20regulates%20the%20replication%20cycle%20in%20Escherichia%20coli.%20EMBO%20J.20,%204253%E2%80%934262%20(2001)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR30">
<span class="label">30.</span><cite>Miyoshi, K., Tatsumoto, Y., Ozaki, S. &amp; Katayama, T. Negative feedback for <em>DARS2</em> –Fis complex by ATP–DnaA supports the cell cycle-coordinated regulation for chromosome replication. <em>Nucleic Acids Res.</em><strong>49</strong>, 12820–12835 (2021).
</cite> [<a href="https://doi.org/10.1093/nar/gkab1171" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8682772/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34871419/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Miyoshi,%20K.,%20Tatsumoto,%20Y.,%20Ozaki,%20S.%20&amp;%20Katayama,%20T.%20Negative%20feedback%20for%20DARS2%20%E2%80%93Fis%20complex%20by%20ATP%E2%80%93DnaA%20supports%20the%20cell%20cycle-coordinated%20regulation%20for%20chromosome%20replication.%20Nucleic%20Acids%20Res.49,%2012820%E2%80%9312835%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>Frimodt-Møller, J., Charbon, G., Krogfelt, K. A. &amp; Løbner-Olesen, A. DNA Replication Control Is Linked to Genomic Positioning of Control Regions in <em>Escherichia coli</em>. <em>PLoS Genet.</em><strong>12</strong>, 1–27 (2016).</cite> [<a href="https://doi.org/10.1371/journal.pgen.1006286" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5010248/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27589233/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Frimodt-M%C3%B8ller,%20J.,%20Charbon,%20G.,%20Krogfelt,%20K.%20A.%20&amp;%20L%C3%B8bner-Olesen,%20A.%20DNA%20Replication%20Control%20Is%20Linked%20to%20Genomic%20Positioning%20of%20Control%20Regions%20in%20Escherichia%20coli.%20PLoS%20Genet.12,%201%E2%80%9327%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>Knöppel, A., Broström, O., Gras, K., Elf, J. &amp; Fange, D. Regulatory elements coordinating initiation of chromosome replication to the <em>Escherichia coli</em> cell cycle. <em>Proc. Natl Acad. Sci.</em><strong>120</strong>, e2213795120 (2023).
</cite> [<a href="https://doi.org/10.1073/pnas.2213795120" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10235992/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37220276/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kn%C3%B6ppel,%20A.,%20Brostr%C3%B6m,%20O.,%20Gras,%20K.,%20Elf,%20J.%20&amp;%20Fange,%20D.%20Regulatory%20elements%20coordinating%20initiation%20of%20chromosome%20replication%20to%20the%20Escherichia%20coli%20cell%20cycle.%20Proc.%20Natl%20Acad.%20Sci.120,%20e2213795120%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Iuliani, I., Mbemba, G., Lagomarsino, M. C. &amp; Sclavi, B. Direct single-cell observation of a key Escherichia coli cell-cycle oscillator. <em>Sci. Adv.</em><strong>10</strong>, (2024).</cite> [<a href="https://doi.org/10.1126/sciadv.ado5398" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC466948/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39018394/" class="usa-link">PubMed</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>Berger, M. &amp; ten Wolde, P. R. Synchronous Replication Initiation of Multiple Origins. <em>PRX Life</em><strong>1</strong>, 013007 (2023).</cite> [<a href="https://scholar.google.com/scholar_lookup?Berger,%20M.%20&amp;%20ten%20Wolde,%20P.%20R.%20Synchronous%20Replication%20Initiation%20of%20Multiple%20Origins.%20PRX%20Life1,%20013007%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>Fu, H., Xiao, F. &amp; Jun, S. Bacterial Replication Initiation as Precision Control by Protein Counting. <em>PRX Life</em><strong>1</strong>, 013011 (2023).
</cite> [<a href="https://doi.org/10.1103/prxlife.1.013011" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10977104/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38550259/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Fu,%20H.,%20Xiao,%20F.%20&amp;%20Jun,%20S.%20Bacterial%20Replication%20Initiation%20as%20Precision%20Control%20by%20Protein%20Counting.%20PRX%20Life1,%20013011%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR36">
<span class="label">36.</span><cite>Berger, M. &amp; ten Wolde, P. R. Robust replication initiation from coupled homeostatic mechanisms. <em>Nat. Commun. 2022 13:1</em><strong>13</strong>, 1–13 (2022).</cite> [<a href="https://doi.org/10.1038/s41467-022-33886-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9640692/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36344507/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Berger,%20M.%20&amp;%20ten%20Wolde,%20P.%20R.%20Robust%20replication%20initiation%20from%20coupled%20homeostatic%20mechanisms.%20Nat.%20Commun.%202022%2013:113,%201%E2%80%9313%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR37">
<span class="label">37.</span><cite>Schaper, S. &amp; Messer, W. Interaction of the initiator protein DnaA of <em>Escherichia coli</em> with its DNA target. <em>J. Biol. Chem.</em><strong>270</strong>, 17622–17626 (1995).
</cite> [<a href="https://doi.org/10.1074/jbc.270.29.17622" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/7615570/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Schaper,%20S.%20&amp;%20Messer,%20W.%20Interaction%20of%20the%20initiator%20protein%20DnaA%20of%20Escherichia%20coli%20with%20its%20DNA%20target.%20J.%20Biol.%20Chem.270,%2017622%E2%80%9317626%20(1995)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR38">
<span class="label">38.</span><cite>Newman, G. &amp; Crooke, E. DnaA, the Initiator of <em>Escherichia coli</em> Chromosomal Replication, Is Located at the Cell Membrane. <em>J. Bacteriol.</em><strong>182</strong>, 2604–2610 (2000).
</cite> [<a href="https://doi.org/10.1128/jb.182.9.2604-2610.2000" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC111327/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10762265/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Newman,%20G.%20&amp;%20Crooke,%20E.%20DnaA,%20the%20Initiator%20of%20Escherichia%20coli%20Chromosomal%20Replication,%20Is%20Located%20at%20the%20Cell%20Membrane.%20J.%20Bacteriol.182,%202604%E2%80%932610%20(2000)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR39">
<span class="label">39.</span><cite>Hansen, F. G., Koefoed, S., Sørensen, L. &amp; Atlung, T. Titration of DnaA protein by <em>oriC</em> DnaA-boxes increases dnaA gene expression in <em>Escherichia coli</em>. <em>EMBO J.</em><strong>6</strong>, 255–258 (1987).
</cite> [<a href="https://doi.org/10.1002/j.1460-2075.1987.tb04747.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC553385/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/3034578/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hansen,%20F.%20G.,%20Koefoed,%20S.,%20S%C3%B8rensen,%20L.%20&amp;%20Atlung,%20T.%20Titration%20of%20DnaA%20protein%20by%20oriC%20DnaA-boxes%20increases%20dnaA%20gene%20expression%20in%20Escherichia%20coli.%20EMBO%20J.6,%20255%E2%80%93258%20(1987)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR40">
<span class="label">40.</span><cite>Kitagawa, R., Ozaki, T., Moriya, S. &amp; Ogawa, T. Negative control of replication initiation by a novel chromosomal locus exhibiting exceptional affinity for <em>Escherichia coli</em> DnaA protein. <em>Genes Dev.</em><strong>12</strong>, 3032–3043 (1998).
</cite> [<a href="https://doi.org/10.1101/gad.12.19.3032" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC317192/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9765205/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kitagawa,%20R.,%20Ozaki,%20T.,%20Moriya,%20S.%20&amp;%20Ogawa,%20T.%20Negative%20control%20of%20replication%20initiation%20by%20a%20novel%20chromosomal%20locus%20exhibiting%20exceptional%20affinity%20for%20Escherichia%20coli%20DnaA%20protein.%20Genes%20Dev.12,%203032%E2%80%933043%20(1998)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR41">
<span class="label">41.</span><cite>Boye, E., Skarstad, K. &amp; Løbner-Olesen, A. Regulation of chromosomal replication by DnaA protein availability in <em>Escherichia coli</em>: effects of the <em>datA</em> region. <em>Biochim Biophys. Acta</em><strong>1521</strong>, 73–80 (2001).
</cite> [<a href="https://doi.org/10.1016/s0167-4781(01)00292-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11690638/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Boye,%20E.,%20Skarstad,%20K.%20&amp;%20L%C3%B8bner-Olesen,%20A.%20Regulation%20of%20chromosomal%20replication%20by%20DnaA%20protein%20availability%20in%20Escherichia%20coli:%20effects%20of%20the%20datA%20region.%20Biochim%20Biophys.%20Acta1521,%2073%E2%80%9380%20(2001)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR42">
<span class="label">42.</span><cite>Morigen, Løbner-Olesen, A. &amp; Skarstad, K. Titration of the <em>Escherichia coli</em> DnaA protein to excess <em>datA</em> sites causes destabilization of replication forks, delayed replication initiation and delayed cell division. <em>Mol. Microbiol</em><strong>50</strong>, 349–362 (2003).
</cite> [<a href="https://doi.org/10.1046/j.1365-2958.2003.03695.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/14507385/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Morigen,%20L%C3%B8bner-Olesen,%20A.%20&amp;%20Skarstad,%20K.%20Titration%20of%20the%20Escherichia%20coli%20DnaA%20protein%20to%20excess%20datA%20sites%20causes%20destabilization%20of%20replication%20forks,%20delayed%20replication%20initiation%20and%20delayed%20cell%20division.%20Mol.%20Microbiol50,%20349%E2%80%93362%20(2003)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR43">
<span class="label">43.</span><cite>Shen, H. et al. Single Particle Tracking: From Theory to Biophysical Applications. <em>Chem. Rev.</em><strong>117</strong>, 7331–7376 (2017).
</cite> [<a href="https://doi.org/10.1021/acs.chemrev.6b00815" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28520419/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Shen,%20H.%20et%20al.%20Single%20Particle%20Tracking:%20From%20Theory%20to%20Biophysical%20Applications.%20Chem.%20Rev.117,%207331%E2%80%937376%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR44">
<span class="label">44.</span><cite>Olivi, L. et al. Live-cell imaging reveals the trade-off between target search flexibility and efficiency for Cas9 and Cas12a. <em>Nucleic Acids Res.</em><strong>1</strong>, 13–14 (2024).</cite> [<a href="https://doi.org/10.1093/nar/gkae283" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11109954/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38647045/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Olivi,%20L.%20et%20al.%20Live-cell%20imaging%20reveals%20the%20trade-off%20between%20target%20search%20flexibility%20and%20efficiency%20for%20Cas9%20and%20Cas12a.%20Nucleic%20Acids%20Res.1,%2013%E2%80%9314%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR45">
<span class="label">45.</span><cite>Frimodt-Møller, J., Charbon, G., Krogfelt, K. A. &amp; Løbner-Olesen, A. Control regions for chromosome replication are conserved with respect to sequence and location among <em>Escherichia coli</em> strains. <em>Front. Microbiol.</em><strong>6</strong>, (2015).</cite> [<a href="https://doi.org/10.3389/fmicb.2015.01011" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4585315/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26441936/" class="usa-link">PubMed</a>]</li>
<li id="CR46">
<span class="label">46.</span><cite>Kawakami, H., Su’etsugu, M. &amp; Katayama, T. An isolated Hda–clamp complex is functional in the regulatory inactivation of DnaA and DNA replication. <em>J. Struct. Biol.</em><strong>156</strong>, 220–229 (2006).
</cite> [<a href="https://doi.org/10.1016/j.jsb.2006.02.007" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16603382/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kawakami,%20H.,%20Su%E2%80%99etsugu,%20M.%20&amp;%20Katayama,%20T.%20An%20isolated%20Hda%E2%80%93clamp%20complex%20is%20functional%20in%20the%20regulatory%20inactivation%20of%20DnaA%20and%20DNA%20replication.%20J.%20Struct.%20Biol.156,%20220%E2%80%93229%20(2006)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR47">
<span class="label">47.</span><cite>Moolman, M. C. et al. Slow unloading leads to DNA-bound β2-sliding clamp accumulation in live <em>Escherichia coli</em> cells. <em>Nat. Commun.</em><strong>5</strong>, 5820 (2014).
</cite> [<a href="https://doi.org/10.1038/ncomms6820" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4284645/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25520215/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Moolman,%20M.%20C.%20et%20al.%20Slow%20unloading%20leads%20to%20DNA-bound%20%CE%B22-sliding%20clamp%20accumulation%20in%20live%20Escherichia%20coli%20cells.%20Nat.%20Commun.5,%205820%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR48">
<span class="label">48.</span><cite>Nozaki, S. &amp; Ogawa, T. Determination of the minimum domain II size of <em>Escherichia coli</em> DnaA protein essential for cell viability. <em>Microbiology</em><strong>154</strong>, 3379–3384 (2008).
</cite> [<a href="https://doi.org/10.1099/mic.0.2008/019745-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18957591/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Nozaki,%20S.%20&amp;%20Ogawa,%20T.%20Determination%20of%20the%20minimum%20domain%20II%20size%20of%20Escherichia%20coli%20DnaA%20protein%20essential%20for%20cell%20viability.%20Microbiology154,%203379%E2%80%933384%20(2008)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR49">
<span class="label">49.</span><cite>Nozaki, S., Niki, H. &amp; Ogawa, T. Replication initiator DnaA of <em>Escherichia coli</em> changes its assembly form on the replication origin during the cell cycle. <em>J. Bacteriol.</em><strong>191</strong>, 4807–4814 (2009).
</cite> [<a href="https://doi.org/10.1128/JB.00435-09" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2715721/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19502409/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Nozaki,%20S.,%20Niki,%20H.%20&amp;%20Ogawa,%20T.%20Replication%20initiator%20DnaA%20of%20Escherichia%20coli%20changes%20its%20assembly%20form%20on%20the%20replication%20origin%20during%20the%20cell%20cycle.%20J.%20Bacteriol.191,%204807%E2%80%934814%20(2009)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR50">
<span class="label">50.</span><cite>Charbon, G. et al. Suppressors of DnaA(ATP) imposed overinitiation in <em>Escherichia coli</em>. <em>Mol. Microbiol</em><strong>79</strong>, 914–928 (2011).
</cite> [<a href="https://doi.org/10.1111/j.1365-2958.2010.07493.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21299647/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Charbon,%20G.%20et%20al.%20Suppressors%20of%20DnaA(ATP)%20imposed%20overinitiation%20in%20Escherichia%20coli.%20Mol.%20Microbiol79,%20914%E2%80%93928%20(2011)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR51">
<span class="label">51.</span><cite>Stracy, M. et al. Transient non-specific DNA binding dominates the target search of bacterial DNA-binding proteins. <em>Mol. cell</em><strong>81</strong>, 1499–1514.e6 (2021).
</cite> [<a href="https://doi.org/10.1016/j.molcel.2021.01.039" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8022225/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33621478/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Stracy,%20M.%20et%20al.%20Transient%20non-specific%20DNA%20binding%20dominates%20the%20target%20search%20of%20bacterial%20DNA-binding%20proteins.%20Mol.%20cell81,%201499%E2%80%931514.e6%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR52">
<span class="label">52.</span><cite>Schenk, K. et al. Rapid turnover of DnaA at replication origin regions contributes to initiation control of DNA replication. <em>PLOS Genet.</em><strong>13</strong>, e1006561 (2017).
</cite> [<a href="https://doi.org/10.1371/journal.pgen.1006561" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5319796/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28166228/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Schenk,%20K.%20et%20al.%20Rapid%20turnover%20of%20DnaA%20at%20replication%20origin%20regions%20contributes%20to%20initiation%20control%20of%20DNA%20replication.%20PLOS%20Genet.13,%20e1006561%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR53">
<span class="label">53.</span><cite>Saggioro, C., Olliver, A. &amp; Sclavi, B. Temperature-dependence of the DnaA-DNA interaction and its effect on the autoregulation of <em>dnaA</em> expression. <em>Biochem J.</em><strong>449</strong>, 333–341 (2013).
</cite> [<a href="https://doi.org/10.1042/BJ20120876" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23092251/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Saggioro,%20C.,%20Olliver,%20A.%20&amp;%20Sclavi,%20B.%20Temperature-dependence%20of%20the%20DnaA-DNA%20interaction%20and%20its%20effect%20on%20the%20autoregulation%20of%20dnaA%20expression.%20Biochem%20J.449,%20333%E2%80%93341%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR54">
<span class="label">54.</span><cite>Schaechter, M., MaalØe, O. &amp; Kjeldgaard, N. O. Dependency on Medium and Temperature of Cell Size and Chemical Composition during Balanced Growth of <em>Salmonella typhimurium</em>. <em>Microbiology</em><strong>19</strong>, 592–606 (1958).</cite> [<a href="https://doi.org/10.1099/00221287-19-3-592" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/13611202/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Schaechter,%20M.,%20Maal%C3%98e,%20O.%20&amp;%20Kjeldgaard,%20N.%20O.%20Dependency%20on%20Medium%20and%20Temperature%20of%20Cell%20Size%20and%20Chemical%20Composition%20during%20Balanced%20Growth%20of%20Salmonella%20typhimurium.%20Microbiology19,%20592%E2%80%93606%20(1958)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR55">
<span class="label">55.</span><cite>Si, F. et al. Mechanistic Origin of Cell-Size Control and Homeostasis in Bacteria. <em>Curr. Biol.</em><strong>29</strong>, 1760–1770.e7 (2019).
</cite> [<a href="https://doi.org/10.1016/j.cub.2019.04.062" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6548602/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31104932/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Si,%20F.%20et%20al.%20Mechanistic%20Origin%20of%20Cell-Size%20Control%20and%20Homeostasis%20in%20Bacteria.%20Curr.%20Biol.29,%201760%E2%80%931770.e7%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR56">
<span class="label">56.</span><cite>Flåtten, I. &amp; Skarstad, K. The Fis Protein Has a Stimulating Role in Initiation of Replication in <em>Escherichia coli</em> In Vivo. <em>PLOS ONE</em><strong>8</strong>, e83562 (2013).
</cite> [<a href="https://doi.org/10.1371/journal.pone.0083562" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3865182/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24358293/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Fl%C3%A5tten,%20I.%20&amp;%20Skarstad,%20K.%20The%20Fis%20Protein%20Has%20a%20Stimulating%20Role%20in%20Initiation%20of%20Replication%20in%20Escherichia%20coli%20In%20Vivo.%20PLOS%20ONE8,%20e83562%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR57">
<span class="label">57.</span><cite>Kasho, K., Fujimitsu, K., Matoba, T., Oshima, T. &amp; Katayama, T. Timely binding of IHF and Fis to DARS2 regulates ATP–DnaA production and replication initiation. <em>Nucleic Acids Res.</em><strong>42</strong>, 13134–13149 (2014).
</cite> [<a href="https://doi.org/10.1093/nar/gku1051" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4245941/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25378325/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kasho,%20K.,%20Fujimitsu,%20K.,%20Matoba,%20T.,%20Oshima,%20T.%20&amp;%20Katayama,%20T.%20Timely%20binding%20of%20IHF%20and%20Fis%20to%20DARS2%20regulates%20ATP%E2%80%93DnaA%20production%20and%20replication%20initiation.%20Nucleic%20Acids%20Res.42,%2013134%E2%80%9313149%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR58">
<span class="label">58.</span><cite>Erzberger, J. P., Mott, M. L. &amp; Berger, J. M. Structural basis for ATP-dependent DnaA assembly and replication-origin remodeling. <em>Nat. Struct. Mol. Biol.</em><strong>13</strong>, 676–683 (2006).
</cite> [<a href="https://doi.org/10.1038/nsmb1115" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16829961/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Erzberger,%20J.%20P.,%20Mott,%20M.%20L.%20&amp;%20Berger,%20J.%20M.%20Structural%20basis%20for%20ATP-dependent%20DnaA%20assembly%20and%20replication-origin%20remodeling.%20Nat.%20Struct.%20Mol.%20Biol.13,%20676%E2%80%93683%20(2006)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR59">
<span class="label">59.</span><cite>Messer, W. &amp; Weigel, C. DnaA initiator-also a transcription factor. <em>Mol. Microbiol</em><strong>24</strong>, 1–6 (1997).
</cite> [<a href="https://doi.org/10.1046/j.1365-2958.1997.3171678.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9140960/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Messer,%20W.%20&amp;%20Weigel,%20C.%20DnaA%20initiator-also%20a%20transcription%20factor.%20Mol.%20Microbiol24,%201%E2%80%936%20(1997)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR60">
<span class="label">60.</span><cite>Menikpurage, I. P., Woo, K. &amp; Mera, P. E. Transcriptional Activity of the Bacterial Replication Initiator DnaA. <em>Front Microbiol</em><strong>12</strong>, 662317 (2021).
</cite> [<a href="https://doi.org/10.3389/fmicb.2021.662317" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8203912/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34140937/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Menikpurage,%20I.%20P.,%20Woo,%20K.%20&amp;%20Mera,%20P.%20E.%20Transcriptional%20Activity%20of%20the%20Bacterial%20Replication%20Initiator%20DnaA.%20Front%20Microbiol12,%20662317%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR61">
<span class="label">61.</span><cite>Gao, R. et al. A balancing act in transcription regulation by response regulators: titration of transcription factor activity by decoy DNA binding sites. <em>Nucleic Acids Res.</em><strong>49</strong>, 11537–11549 (2021).
</cite> [<a href="https://doi.org/10.1093/nar/gkab935" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8599769/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34669947/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gao,%20R.%20et%20al.%20A%20balancing%20act%20in%20transcription%20regulation%20by%20response%20regulators:%20titration%20of%20transcription%20factor%20activity%20by%20decoy%20DNA%20binding%20sites.%20Nucleic%20Acids%20Res.49,%2011537%E2%80%9311549%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR62">
<span class="label">62.</span><cite>Rosenfeld, N., Elowitz, M. B. &amp; Alon, U. Negative Autoregulation Speeds the Response Times of Transcription Networks. <em>J. Mol. Biol.</em><strong>323</strong>, 785–793 (2002).
</cite> [<a href="https://doi.org/10.1016/s0022-2836(02)00994-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12417193/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rosenfeld,%20N.,%20Elowitz,%20M.%20B.%20&amp;%20Alon,%20U.%20Negative%20Autoregulation%20Speeds%20the%20Response%20Times%20of%20Transcription%20Networks.%20J.%20Mol.%20Biol.323,%20785%E2%80%93793%20(2002)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR63">
<span class="label">63.</span><cite>Mizushima, T., Tomura, A., Shinpuku, T., Miki, T. &amp; Sekimizu, K. Loss of flagellation in <em>dnaA</em> mutants of <em>Escherichia coli</em>. <em>J. Bacteriol.</em><strong>176</strong>, 5544–5546 (1994).
</cite> [<a href="https://doi.org/10.1128/jb.176.17.5544-5546.1994" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC196746/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/8071236/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mizushima,%20T.,%20Tomura,%20A.,%20Shinpuku,%20T.,%20Miki,%20T.%20&amp;%20Sekimizu,%20K.%20Loss%20of%20flagellation%20in%20dnaA%20mutants%20of%20Escherichia%20coli.%20J.%20Bacteriol.176,%205544%E2%80%935546%20(1994)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR64">
<span class="label">64.</span><cite>Augustin, L. B., Jacobson, B. A. &amp; Fuchs, J. A. <em>Escherichia coli</em> Fis and DnaA proteins bind specifically to the <em>nrd</em> promoter region and affect expression of an <em>nrd-lac</em> fusion. <em>J. Bacteriol.</em><strong>176</strong>, 378–387 (1994).
</cite> [<a href="https://doi.org/10.1128/jb.176.2.378-387.1994" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC205060/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/8288532/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Augustin,%20L.%20B.,%20Jacobson,%20B.%20A.%20&amp;%20Fuchs,%20J.%20A.%20Escherichia%20coli%20Fis%20and%20DnaA%20proteins%20bind%20specifically%20to%20the%20nrd%20promoter%20region%20and%20affect%20expression%20of%20an%20nrd-lac%20fusion.%20J.%20Bacteriol.176,%20378%E2%80%93387%20(1994)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR65">
<span class="label">65.</span><cite>Quiñones, A., Wandt, G., Kleinstäuber, S. &amp; Messer, W. DnaA protein stimulates <em>polA</em> gene expression in <em>Escherichia coli</em>. <em>Mol. Microbiol</em><strong>23</strong>, 1193–1202 (1997).
</cite> [<a href="https://doi.org/10.1046/j.1365-2958.1997.2961658.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9106210/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Qui%C3%B1ones,%20A.,%20Wandt,%20G.,%20Kleinst%C3%A4uber,%20S.%20&amp;%20Messer,%20W.%20DnaA%20protein%20stimulates%20polA%20gene%20expression%20in%20Escherichia%20coli.%20Mol.%20Microbiol23,%201193%E2%80%931202%20(1997)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR66">
<span class="label">66.</span><cite>Løbner-Olesen, A., Atlung, T. &amp; Rasmussen, K. V. Stability and replication control of <em>Escherichia coli</em> minichromosomes. <em>J. Bacteriol.</em><strong>169</strong>, 2835–2842 (1987).
</cite> [<a href="https://doi.org/10.1128/jb.169.6.2835-2842.1987" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC212196/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/3294807/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?L%C3%B8bner-Olesen,%20A.,%20Atlung,%20T.%20&amp;%20Rasmussen,%20K.%20V.%20Stability%20and%20replication%20control%20of%20Escherichia%20coli%20minichromosomes.%20J.%20Bacteriol.169,%202835%E2%80%932842%20(1987)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR67">
<span class="label">67.</span><cite>Wang, Q. P. &amp; Kaguni, J. M. dnaA protein regulates transcriptions of the <em>rpoH</em> gene of <em>Escherichia coli</em>. <em>J. Biol. Chem.</em><strong>264</strong>, 7338–7344 (1989).
</cite> [<a href="https://pubmed.ncbi.nlm.nih.gov/2540187/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang,%20Q.%20P.%20&amp;%20Kaguni,%20J.%20M.%20dnaA%20protein%20regulates%20transcriptions%20of%20the%20rpoH%20gene%20of%20Escherichia%20coli.%20J.%20Biol.%20Chem.264,%207338%E2%80%937344%20(1989)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR68">
<span class="label">68.</span><cite>Zhou, Z. &amp; Syvanen, M. Identification and sequence of the <em>drpA</em> gene from <em>Escherichia coli</em>. <em>J. Bacteriol.</em><strong>172</strong>, 281–286 (1990).
</cite> [<a href="https://doi.org/10.1128/jb.172.1.281-286.1990" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC208429/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/1688424/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhou,%20Z.%20&amp;%20Syvanen,%20M.%20Identification%20and%20sequence%20of%20the%20drpA%20gene%20from%20Escherichia%20coli.%20J.%20Bacteriol.172,%20281%E2%80%93286%20(1990)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR69">
<span class="label">69.</span><cite>Tesfa-Selase, F. &amp; Drabble, W. T. Regulation of the <em>gua</em> operon of <em>Escherichia coli</em> by the DnaA protein. <em>Mol. Gen. Genet</em><strong>231</strong>, 256–264 (1992).
</cite> [<a href="https://doi.org/10.1007/BF00279799" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/1736096/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tesfa-Selase,%20F.%20&amp;%20Drabble,%20W.%20T.%20Regulation%20of%20the%20gua%20operon%20of%20Escherichia%20coli%20by%20the%20DnaA%20protein.%20Mol.%20Gen.%20Genet231,%20256%E2%80%93264%20(1992)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR70">
<span class="label">70.</span><cite>Ozaki, T., Kumaki, Y., Kitagawa, R. &amp; Ogawa, T. Anomalous DnaA protein binding to the regulatory region of the <em>Escherichia coli aldA</em> gene. <em>Microbiol. (Read.)</em><strong>147</strong>, 153–159 (2001).</cite> [<a href="https://doi.org/10.1099/00221287-147-1-153" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11160809/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ozaki,%20T.,%20Kumaki,%20Y.,%20Kitagawa,%20R.%20&amp;%20Ogawa,%20T.%20Anomalous%20DnaA%20protein%20binding%20to%20the%20regulatory%20region%20of%20the%20Escherichia%20coli%20aldA%20gene.%20Microbiol.%20(Read.)147,%20153%E2%80%93159%20(2001)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR71">
<span class="label">71.</span><cite>Wurihan et al. DnaA and LexA Proteins Regulate Transcription of the <em>uvrB</em> Gene in <em>Escherichia coli</em>: The Role of DnaA in the Control of the SOS Regulon. <em>Front Microbiol</em><strong>9</strong>, 1212 (2018).
</cite> [<a href="https://doi.org/10.3389/fmicb.2018.01212" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6015884/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29967594/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wurihan%20et%20al.%20DnaA%20and%20LexA%20Proteins%20Regulate%20Transcription%20of%20the%20uvrB%20Gene%20in%20Escherichia%20coli:%20The%20Role%20of%20DnaA%20in%20the%20Control%20of%20the%20SOS%20Regulon.%20Front%20Microbiol9,%201212%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR72">
<span class="label">72.</span><cite>Sass, T. H., Ferrazzoli, A. E. &amp; Lovett, S. T. DnaA and SspA regulation of the <em>iraD</em> gene of <em>Escherichia coli</em>: an alternative DNA damage response independent of LexA/RecA. <em>Genetics</em><strong>221</strong>, iyac062 (2022).
</cite> [<a href="https://doi.org/10.1093/genetics/iyac062" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9157160/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35445706/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sass,%20T.%20H.,%20Ferrazzoli,%20A.%20E.%20&amp;%20Lovett,%20S.%20T.%20DnaA%20and%20SspA%20regulation%20of%20the%20iraD%20gene%20of%20Escherichia%20coli:%20an%20alternative%20DNA%20damage%20response%20independent%20of%20LexA/RecA.%20Genetics221,%20iyac062%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR73">
<span class="label">73.</span><cite>Stringer, A. M., Fitzgerald, D. M. &amp; Wade, J. T. Mapping the <em>Escherichia coli</em> DnaA-binding landscape reveals a preference for binding pairs of closely spaced DNA sites. <em>Microbiology</em><strong>170</strong>, (2024).</cite> [<a href="https://doi.org/10.1099/mic.0.001474" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11317965/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39012340/" class="usa-link">PubMed</a>]</li>
<li id="CR74">
<span class="label">74.</span><cite>Lin, J. &amp; Amir, A. Homeostasis of protein and mRNA concentrations in growing cells. <em>Nat. Commun. 2018 9:1</em><strong>9</strong>, 1–11 (2018).</cite> [<a href="https://doi.org/10.1038/s41467-018-06714-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6206055/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30374016/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lin,%20J.%20&amp;%20Amir,%20A.%20Homeostasis%20of%20protein%20and%20mRNA%20concentrations%20in%20growing%20cells.%20Nat.%20Commun.%202018%209:19,%201%E2%80%9311%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR75">
<span class="label">75.</span><cite>Hansen, F. G., Christensen, B. B., Nielsen, C. B. &amp; Atlung, T. Insights into the quality of DnaA boxes and their cooperativity. <em>J. Mol. Biol.</em><strong>355</strong>, 85–95 (2006).
</cite> [<a href="https://doi.org/10.1016/j.jmb.2005.10.036" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16298387/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hansen,%20F.%20G.,%20Christensen,%20B.%20B.,%20Nielsen,%20C.%20B.%20&amp;%20Atlung,%20T.%20Insights%20into%20the%20quality%20of%20DnaA%20boxes%20and%20their%20cooperativity.%20J.%20Mol.%20Biol.355,%2085%E2%80%9395%20(2006)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR76">
<span class="label">76.</span><cite>Cassler, M. R., Grimwade, J. E. &amp; Leonard, A. C. Cell cycle-specific changes in nucleoprotein complexes at a chromosomal replication origin. <em>EMBO J.</em><strong>14</strong>, 5833–5841 (1995).
</cite> [<a href="https://doi.org/10.1002/j.1460-2075.1995.tb00271.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC394701/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/8846776/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cassler,%20M.%20R.,%20Grimwade,%20J.%20E.%20&amp;%20Leonard,%20A.%20C.%20Cell%20cycle-specific%20changes%20in%20nucleoprotein%20complexes%20at%20a%20chromosomal%20replication%20origin.%20EMBO%20J.14,%205833%E2%80%935841%20(1995)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR77">
<span class="label">77.</span><cite>Kawakamii, H., Keyamura, K. &amp; Katayama, T. Formation of an ATP-DnaA-specific initiation complex requires DnaA arginine 285, a conserved motif in the AAA+ protein family. <em>J. Biol. Chem.</em><strong>280</strong>, 27420–27430 (2005).
</cite> [<a href="https://doi.org/10.1074/jbc.M502764200" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15901724/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kawakamii,%20H.,%20Keyamura,%20K.%20&amp;%20Katayama,%20T.%20Formation%20of%20an%20ATP-DnaA-specific%20initiation%20complex%20requires%20DnaA%20arginine%20285,%20a%20conserved%20motif%20in%20the%20AAA+%20protein%20family.%20J.%20Biol.%20Chem.280,%2027420%E2%80%9327430%20(2005)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR78">
<span class="label">78.</span><cite>Stepankiw, N., Kaidow, A., Boye, E. &amp; Bates, D. The right half of the <em>Escherichia coli</em> replication origin is not essential for viability, but facilitates multi-forked replication. <em>Mol. Microbiol.</em><strong>74</strong>, 467–479 (2009).
</cite> [<a href="https://doi.org/10.1111/j.1365-2958.2009.06877.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2967437/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19737351/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Stepankiw,%20N.,%20Kaidow,%20A.,%20Boye,%20E.%20&amp;%20Bates,%20D.%20The%20right%20half%20of%20the%20Escherichia%20coli%20replication%20origin%20is%20not%20essential%20for%20viability,%20but%20facilitates%20multi-forked%20replication.%20Mol.%20Microbiol.74,%20467%E2%80%93479%20(2009)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR79">
<span class="label">79.</span><cite>Fuller, R. S., Funnell, B. E. &amp; Kornberg, A. The dnaA protein complex with the <em>E. coli</em> chromosomal replication origin (<em>oriC</em>) and other DNA sites. <em>Cell</em><strong>38</strong>, 889–900 (1984).
</cite> [<a href="https://doi.org/10.1016/0092-8674(84)90284-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/6091903/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Fuller,%20R.%20S.,%20Funnell,%20B.%20E.%20&amp;%20Kornberg,%20A.%20The%20dnaA%20protein%20complex%20with%20the%20E.%20coli%20chromosomal%20replication%20origin%20(oriC)%20and%20other%20DNA%20sites.%20Cell38,%20889%E2%80%93900%20(1984)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR80">
<span class="label">80.</span><cite>Waldminghaus, T. &amp; Skarstad, K. The <em>Escherichia coli</em> SeqA protein. <em>Plasmid</em><strong>61</strong>, 141–150 (2009).
</cite> [<a href="https://doi.org/10.1016/j.plasmid.2009.02.004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19254745/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Waldminghaus,%20T.%20&amp;%20Skarstad,%20K.%20The%20Escherichia%20coli%20SeqA%20protein.%20Plasmid61,%20141%E2%80%93150%20(2009)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR81">
<span class="label">81.</span><cite>Riber, L. et al. Hda-mediated inactivation of the DnaA protein and <em>dnaA</em> gene autoregulation act in concert to ensure homeostatic maintenance of the <em>Escherichia coli</em> chromosome. <em>Genes Dev.</em><strong>20</strong>, 2121–2134 (2006).
</cite> [<a href="https://doi.org/10.1101/gad.379506" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1536062/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16882985/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Riber,%20L.%20et%20al.%20Hda-mediated%20inactivation%20of%20the%20DnaA%20protein%20and%20dnaA%20gene%20autoregulation%20act%20in%20concert%20to%20ensure%20homeostatic%20maintenance%20of%20the%20Escherichia%20coli%20chromosome.%20Genes%20Dev.20,%202121%E2%80%932134%20(2006)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR82">
<span class="label">82.</span><cite>Nozaki, S., Yamada, Y. &amp; Ogawa, T. Initiator titration complex formed at <em>datA</em> with the aid of IHF regulates replication timing in <em>Escherichia coli</em>. <em>Genes Cells</em><strong>14</strong>, 329–341 (2009).
</cite> [<a href="https://doi.org/10.1111/j.1365-2443.2008.01269.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19170757/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Nozaki,%20S.,%20Yamada,%20Y.%20&amp;%20Ogawa,%20T.%20Initiator%20titration%20complex%20formed%20at%20datA%20with%20the%20aid%20of%20IHF%20regulates%20replication%20timing%20in%20Escherichia%20coli.%20Genes%20Cells14,%20329%E2%80%93341%20(2009)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR83">
<span class="label">83.</span><cite>Wang, P. et al. Robust Growth of <em>Escherichia coli</em>. <em>Curr. Biol.</em><strong>20</strong>, 1099–1103 (2010).
</cite> [<a href="https://doi.org/10.1016/j.cub.2010.04.045" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2902570/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20537537/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang,%20P.%20et%20al.%20Robust%20Growth%20of%20Escherichia%20coli.%20Curr.%20Biol.20,%201099%E2%80%931103%20(2010)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR84">
<span class="label">84.</span><cite>Agarwala, R. et al. Database resources of the National Center for Biotechnology Information. <em>Nucleic Acids Res.</em><strong>46</strong>, D8–D13 (2018).
</cite> [<a href="https://doi.org/10.1093/nar/gkx1095" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5753372/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29140470/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Agarwala,%20R.%20et%20al.%20Database%20resources%20of%20the%20National%20Center%20for%20Biotechnology%20Information.%20Nucleic%20Acids%20Res.46,%20D8%E2%80%93D13%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR85">
<span class="label">85.</span><cite>R Core Team. R Core Team 2023 R: A language and environment for statistical computing. R foundation for statistical computing. <a href="https://www.R-project.org/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.R-project.org/</a>. <em>R Foundation for Statistical Computing</em> 2021 (2023).</cite>
</li>
<li id="CR86">
<span class="label">86.</span><cite>Blattner, F. R. et al. The Complete Genome Sequence of <em>Escherichia coli</em> K-12. <em>Science</em><strong>277</strong>, 1453–1462 (1997).
</cite> [<a href="https://doi.org/10.1126/science.277.5331.1453" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9278503/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Blattner,%20F.%20R.%20et%20al.%20The%20Complete%20Genome%20Sequence%20of%20Escherichia%20coli%20K-12.%20Science277,%201453%E2%80%931462%20(1997)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR87">
<span class="label">87.</span><cite>Dong, M. J., Luo, H. &amp; Gao, F. DoriC 12.0: an updated database of replication origins in both complete and draft prokaryotic genomes. <em>Nucleic Acids Res.</em><strong>51</strong>, D117–D120 (2023).
</cite> [<a href="https://doi.org/10.1093/nar/gkac964" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9825612/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36305822/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Dong,%20M.%20J.,%20Luo,%20H.%20&amp;%20Gao,%20F.%20DoriC%2012.0:%20an%20updated%20database%20of%20replication%20origins%20in%20both%20complete%20and%20draft%20prokaryotic%20genomes.%20Nucleic%20Acids%20Res.51,%20D117%E2%80%93D120%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR88">
<span class="label">88.</span><cite>Agostinelli, C. &amp; Lund, U. R package ‘circular’: Circular Statistics (version 0.5-0). (2024).</cite>
</li>
<li id="CR89">
<span class="label">89.</span><cite>Zeileis, A. &amp; Grothendieck, G. zoo: S3 Infrastructure for Regular and Irregular Time Series. <em>J. Stat. Softw.</em><strong>14</strong>, 1–27 (2005).</cite> [<a href="https://scholar.google.com/scholar_lookup?Zeileis,%20A.%20&amp;%20Grothendieck,%20G.%20zoo:%20S3%20Infrastructure%20for%20Regular%20and%20Irregular%20Time%20Series.%20J.%20Stat.%20Softw.14,%201%E2%80%9327%20(2005)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR90">
<span class="label">90.</span><cite>Wickham, H. <em>Ggplot2: Elegant Graphics for Data Analysis</em>. (Springer International Publishing, Cham, 2016). 10.1007/978-3-319-24277-4.</cite>
</li>
<li id="CR91">
<span class="label">91.</span><cite>Jiang, Y. et al. Multigene editing in the <em>Escherichia coli</em> genome via the CRISPR-Cas9 system. <em>Appl. Environ. Microbiol.</em><strong>81</strong>, 2506–2514 (2015).
</cite> [<a href="https://doi.org/10.1128/AEM.04023-14" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4357945/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25636838/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jiang,%20Y.%20et%20al.%20Multigene%20editing%20in%20the%20Escherichia%20coli%20genome%20via%20the%20CRISPR-Cas9%20system.%20Appl.%20Environ.%20Microbiol.81,%202506%E2%80%932514%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR92">
<span class="label">92.</span><cite>Ando, R., Flors, C., Mizuno, H., Hofkens, J. &amp; Miyawaki, A. Highlighted Generation of Fluorescence Signals Using Simultaneous Two-Color Irradiation on Dronpa Mutants. <em>Biophysical J.</em><strong>92</strong>, L97–L99 (2007).</cite> [<a href="https://doi.org/10.1529/biophysj.107.105882" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1877776/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17384059/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ando,%20R.,%20Flors,%20C.,%20Mizuno,%20H.,%20Hofkens,%20J.%20&amp;%20Miyawaki,%20A.%20Highlighted%20Generation%20of%20Fluorescence%20Signals%20Using%20Simultaneous%20Two-Color%20Irradiation%20on%20Dronpa%20Mutants.%20Biophysical%20J.92,%20L97%E2%80%93L99%20(2007)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR93">
<span class="label">93.</span><cite>De Zitter, E. et al. Mechanistic investigation of mEos4b reveals a strategy to reduce track interruptions in sptPALM. <em>Nat. Methods</em><strong>16</strong>, 707–710 (2019).
</cite> [<a href="https://doi.org/10.1038/s41592-019-0462-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31285624/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?De%20Zitter,%20E.%20et%20al.%20Mechanistic%20investigation%20of%20mEos4b%20reveals%20a%20strategy%20to%20reduce%20track%20interruptions%20in%20sptPALM.%20Nat.%20Methods16,%20707%E2%80%93710%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR94">
<span class="label">94.</span><cite>Wang, S., Moffitt, J. R., Dempsey, G. T., Xie, X. S. &amp; Zhuang, X. Characterization and development of photoactivatable fluorescent proteins for single-molecule-based superresolution imaging. <em>Proc. Natl Acad. Sci. USA</em><strong>111</strong>, 8452–8457 (2014).
</cite> [<a href="https://doi.org/10.1073/pnas.1406593111" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4060684/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24912163/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang,%20S.,%20Moffitt,%20J.%20R.,%20Dempsey,%20G.%20T.,%20Xie,%20X.%20S.%20&amp;%20Zhuang,%20X.%20Characterization%20and%20development%20of%20photoactivatable%20fluorescent%20proteins%20for%20single-molecule-based%20superresolution%20imaging.%20Proc.%20Natl%20Acad.%20Sci.%20USA111,%208452%E2%80%938457%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR95">
<span class="label">95.</span><cite>Fages-Lartaud, M., Tietze, L., Elie, F., Lale, R. &amp; Hohmann-Marriott, M. F. mCherry contains a fluorescent protein isoform that interferes with its reporter function. <em>Front. Bioeng. Biotechnol.</em><strong>10</strong>, 892138 (2022).
</cite> [<a href="https://doi.org/10.3389/fbioe.2022.892138" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9395592/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36017355/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Fages-Lartaud,%20M.,%20Tietze,%20L.,%20Elie,%20F.,%20Lale,%20R.%20&amp;%20Hohmann-Marriott,%20M.%20F.%20mCherry%20contains%20a%20fluorescent%20protein%20isoform%20that%20interferes%20with%20its%20reporter%20function.%20Front.%20Bioeng.%20Biotechnol.10,%20892138%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR96">
<span class="label">96.</span><cite>Subach, F. V. et al. Photoactivatable mCherry for high-resolution two-color fluorescence microscopy. <em>Nat. Methods</em><strong>6</strong>, 153–159 (2009).
</cite> [<a href="https://doi.org/10.1038/nmeth.1298" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2901231/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19169259/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Subach,%20F.%20V.%20et%20al.%20Photoactivatable%20mCherry%20for%20high-resolution%20two-color%20fluorescence%20microscopy.%20Nat.%20Methods6,%20153%E2%80%93159%20(2009)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR97">
<span class="label">97.</span><cite>Wang, K., Boysen, C., Shizuya, H., Simon, M. I. &amp; Hood, L. Complete nucleotide sequence of two generations of a bacterial artificial chromosome cloning vector. <em>BioTechniques</em><strong>23</strong>, 992–994 (1997).
</cite> [<a href="https://doi.org/10.2144/97236bm04" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9421622/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang,%20K.,%20Boysen,%20C.,%20Shizuya,%20H.,%20Simon,%20M.%20I.%20&amp;%20Hood,%20L.%20Complete%20nucleotide%20sequence%20of%20two%20generations%20of%20a%20bacterial%20artificial%20chromosome%20cloning%20vector.%20BioTechniques23,%20992%E2%80%93994%20(1997)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR98">
<span class="label">98.</span><cite>Claassens, N. J. et al. Bicistronic Design-Based Continuous and High-Level Membrane Protein Production in <em>Escherichia coli</em>. <em>ACS Synth. Biol.</em><strong>8</strong>, 1685–1690 (2019).
</cite> [<a href="https://doi.org/10.1021/acssynbio.9b00101" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6646956/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31264406/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Claassens,%20N.%20J.%20et%20al.%20Bicistronic%20Design-Based%20Continuous%20and%20High-Level%20Membrane%20Protein%20Production%20in%20Escherichia%20coli.%20ACS%20Synth.%20Biol.8,%201685%E2%80%931690%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR99">
<span class="label">99.</span><cite>Hansen, F. G., Koefoed, S. &amp; Atlung, T. Cloning and nucleotide sequence determination of twelve mutant <em>dnaA</em> genes of <em>Escherichia coli</em>. <em>Molec. Gen. Genet.</em><strong>234</strong>, 14–21 (1992).
</cite> [<a href="https://doi.org/10.1007/BF00272340" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/1495477/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hansen,%20F.%20G.,%20Koefoed,%20S.%20&amp;%20Atlung,%20T.%20Cloning%20and%20nucleotide%20sequence%20determination%20of%20twelve%20mutant%20dnaA%20genes%20of%20Escherichia%20coli.%20Molec.%20Gen.%20Genet.234,%2014%E2%80%9321%20(1992)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR100">
<span class="label">100.</span><cite>Jensen, S. I., Lennen, R. M., Herrgård, M. J. &amp; Nielsen, A. T. Seven gene deletions in seven days: Fast generation of Escherichia coli strains tolerant to acetate and osmotic stress. <em>Sci. Rep.</em><strong>5</strong>, (2015).</cite> [<a href="https://doi.org/10.1038/srep17874" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4672327/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26643270/" class="usa-link">PubMed</a>]</li>
<li id="CR101">
<span class="label">101.</span><cite>Steel, H., Habgood, R., Kelly, C. &amp; Papachristodoulou, A. In situ characterisation and manipulation of biological systems with Chi.Bio. <em>PLOS Biol.</em><strong>18</strong>, e3000794 (2020).
</cite> [<a href="https://doi.org/10.1371/journal.pbio.3000794" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7419009/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32730242/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Steel,%20H.,%20Habgood,%20R.,%20Kelly,%20C.%20&amp;%20Papachristodoulou,%20A.%20In%20situ%20characterisation%20and%20manipulation%20of%20biological%20systems%20with%20Chi.Bio.%20PLOS%20Biol.18,%20e3000794%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR102">
<span class="label">102.</span><cite>Martens, K. J. A. et al. Visualisation of dCas9 target search in vivo using an open-microscopy framework. <em>Nat. Commun.</em><strong>10</strong>, 1–11 (2019).
</cite> [<a href="https://doi.org/10.1038/s41467-019-11514-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6685946/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31391532/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Martens,%20K.%20J.%20A.%20et%20al.%20Visualisation%20of%20dCas9%20target%20search%20in%20vivo%20using%20an%20open-microscopy%20framework.%20Nat.%20Commun.10,%201%E2%80%9311%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR103">
<span class="label">103.</span><cite>Farooq, S. &amp; Hohlbein, J. Camera-based single-molecule FRET detection with improved time resolution. <em>Phys. Chem. Chem. Phys.</em><strong>17</strong>, 27862–27872 (2015).
</cite> [<a href="https://doi.org/10.1039/c5cp04137f" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26439729/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Farooq,%20S.%20&amp;%20Hohlbein,%20J.%20Camera-based%20single-molecule%20FRET%20detection%20with%20improved%20time%20resolution.%20Phys.%20Chem.%20Chem.%20Phys.17,%2027862%E2%80%9327872%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR104">
<span class="label">104.</span><cite>Edelstein, A. D. et al. Advanced methods of microscope control using μManager software. <em>J. Biol. Methods</em><strong>1</strong>, e10 (2014).
</cite> [<a href="https://doi.org/10.14440/jbm.2014.36" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4297649/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25606571/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Edelstein,%20A.%20D.%20et%20al.%20Advanced%20methods%20of%20microscope%20control%20using%20%CE%BCManager%20software.%20J.%20Biol.%20Methods1,%20e10%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR105">
<span class="label">105.</span><cite>Schneider, C. A., Rasband, W. S. &amp; Eliceiri, K. W. NIH Image to ImageJ: 25 years of image analysis. <em>Nat. Methods 2012 9:7</em><strong>9</strong>, 671–675 (2012).</cite> [<a href="https://doi.org/10.1038/nmeth.2089" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5554542/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22930834/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Schneider,%20C.%20A.,%20Rasband,%20W.%20S.%20&amp;%20Eliceiri,%20K.%20W.%20NIH%20Image%20to%20ImageJ:%2025%20years%20of%20image%20analysis.%20Nat.%20Methods%202012%209:79,%20671%E2%80%93675%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR106">
<span class="label">106.</span><cite>Schindelin, J. et al. Fiji: an open-source platform for biological-image analysis. <em>Nat. Methods 2012 9:7</em><strong>9</strong>, 676–682 (2012).</cite> [<a href="https://doi.org/10.1038/nmeth.2019" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3855844/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22743772/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Schindelin,%20J.%20et%20al.%20Fiji:%20an%20open-source%20platform%20for%20biological-image%20analysis.%20Nat.%20Methods%202012%209:79,%20676%E2%80%93682%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR107">
<span class="label">107.</span><cite>Vincent, L., Vincent, L. &amp; Soille, P. Watersheds in Digital Spaces: An Efficient Algorithm Based on Immersion Simulations. <em>IEEE Trans. Pattern Anal. Mach. Intell.</em><strong>13</strong>, 583–598 (1991).</cite> [<a href="https://scholar.google.com/scholar_lookup?Vincent,%20L.,%20Vincent,%20L.%20&amp;%20Soille,%20P.%20Watersheds%20in%20Digital%20Spaces:%20An%20Efficient%20Algorithm%20Based%20on%20Immersion%20Simulations.%20IEEE%20Trans.%20Pattern%20Anal.%20Mach.%20Intell.13,%20583%E2%80%93598%20(1991)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR108">
<span class="label">108.</span><cite>Govers, S. K., Campos, M., Tyagi, B., Laloux, G. &amp; Jacobs-Wagner, C. Apparent simplicity and emergent robustness in the control of the <em>Escherichia coli</em> cell cycle. <em>Cell Syst.</em><strong>15</strong>, 19–36.e5 (2024).
</cite> [<a href="https://doi.org/10.1016/j.cels.2023.12.001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38157847/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Govers,%20S.%20K.,%20Campos,%20M.,%20Tyagi,%20B.,%20Laloux,%20G.%20&amp;%20Jacobs-Wagner,%20C.%20Apparent%20simplicity%20and%20emergent%20robustness%20in%20the%20control%20of%20the%20Escherichia%20coli%20cell%20cycle.%20Cell%20Syst.15,%2019%E2%80%9336.e5%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR109">
<span class="label">109.</span><cite>Ovesný, M., Křížek, P., Borkovec, J., Švindrych, Z. &amp; Hagen, G. M. ThunderSTORM: a comprehensive ImageJ plug-in for PALM and STORM data analysis and super-resolution imaging. <em>Bioinformatics</em><strong>30</strong>, 2389–2390 (2014).
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btu202" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4207427/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24771516/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ovesn%C3%BD,%20M.,%20K%C5%99%C3%AD%C5%BEek,%20P.,%20Borkovec,%20J.,%20%C5%A0vindrych,%20Z.%20&amp;%20Hagen,%20G.%20M.%20ThunderSTORM:%20a%20comprehensive%20ImageJ%20plug-in%20for%20PALM%20and%20STORM%20data%20analysis%20and%20super-resolution%20imaging.%20Bioinformatics30,%202389%E2%80%932390%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR110">
<span class="label">110.</span><cite>Jabermoradi, A., Yang, S., Gobes, M. I., Van Duynhoven, J. P. M. &amp; Hohlbein, J. Enabling single-molecule localization microscopy in turbid food emulsions. <em>Philosoph. Transac. Ser. A, Math., Phys, Engineering Sci.</em><strong>380</strong>, (2022).</cite> [<a href="https://doi.org/10.1098/rsta.2020.0164" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35152755/" class="usa-link">PubMed</a>]</li>
<li id="CR111">
<span class="label">111.</span><cite>Hoogendoorn, E. et al. The fidelity of stochastic single-molecule super-resolution reconstructions critically depends upon robust background estimation. <em>Sci. Rep. 2014 4:1</em><strong>4</strong>, 1–10 (2014).</cite> [<a href="https://doi.org/10.1038/srep03854" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3900998/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24458236/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hoogendoorn,%20E.%20et%20al.%20The%20fidelity%20of%20stochastic%20single-molecule%20super-resolution%20reconstructions%20critically%20depends%20upon%20robust%20background%20estimation.%20Sci.%20Rep.%202014%204:14,%201%E2%80%9310%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR112">
<span class="label">112.</span><cite>Van Beljouw, S. P. B. et al. Evaluating single-particle tracking by photo-activation localization microscopy (sptPALM) in Lactococcus lactis. <em>Phys. Biol.</em><strong>16</strong>, 035001 (2019).
</cite> [<a href="https://doi.org/10.1088/1478-3975/ab0162" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30673632/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Van%20Beljouw,%20S.%20P.%20B.%20et%20al.%20Evaluating%20single-particle%20tracking%20by%20photo-activation%20localization%20microscopy%20(sptPALM)%20in%20Lactococcus%20lactis.%20Phys.%20Biol.16,%20035001%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR113">
<span class="label">113.</span><cite>Trovato, F. &amp; Tozzini, V. Diffusion within the cytoplasm: A mesoscale model of interacting macromolecules. <em>Biophysical J.</em><strong>107</strong>, 2579–2591 (2014).</cite> [<a href="https://doi.org/10.1016/j.bpj.2014.09.043" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4255235/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25468337/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Trovato,%20F.%20&amp;%20Tozzini,%20V.%20Diffusion%20within%20the%20cytoplasm:%20A%20mesoscale%20model%20of%20interacting%20macromolecules.%20Biophysical%20J.107,%202579%E2%80%932591%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR114">
<span class="label">114.</span><cite>Fleming, P. J. &amp; Fleming, K. G. HullRad: Fast Calculations of Folded and Disordered Protein and Nucleic Acid Hydrodynamic Properties. <em>Biophysical J.</em><strong>114</strong>, 856–869 (2018).</cite> [<a href="https://doi.org/10.1016/j.bpj.2018.01.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5984988/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29490246/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Fleming,%20P.%20J.%20&amp;%20Fleming,%20K.%20G.%20HullRad:%20Fast%20Calculations%20of%20Folded%20and%20Disordered%20Protein%20and%20Nucleic%20Acid%20Hydrodynamic%20Properties.%20Biophysical%20J.114,%20856%E2%80%93869%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR115">
<span class="label">115.</span><cite>Jumper, J. et al. Highly accurate protein structure prediction with AlphaFold. <em>Nat. 2021 596:7873</em><strong>596</strong>, 583–589 (2021).</cite> [<a href="https://doi.org/10.1038/s41586-021-03819-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8371605/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34265844/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jumper,%20J.%20et%20al.%20Highly%20accurate%20protein%20structure%20prediction%20with%20AlphaFold.%20Nat.%202021%20596:7873596,%20583%E2%80%93589%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR116">
<span class="label">116.</span><cite>Mirdita, M. et al. ColabFold: making protein folding accessible to all. <em>Nat. Methods 2022 19:6</em><strong>19</strong>, 679–682 (2022).</cite> [<a href="https://doi.org/10.1038/s41592-022-01488-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9184281/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35637307/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mirdita,%20M.%20et%20al.%20ColabFold:%20making%20protein%20folding%20accessible%20to%20all.%20Nat.%20Methods%202022%2019:619,%20679%E2%80%93682%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR117">
<span class="label">117.</span><cite>Elf, J., Li, G. W. &amp; Xie, X. S. Probing transcription factor dynamics at the single-molecule level in a living cell. <em>Sci. (N.Y.)</em><strong>316</strong>, 1191–1194 (2007).</cite> [<a href="https://doi.org/10.1126/science.1141967" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2853898/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17525339/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Elf,%20J.,%20Li,%20G.%20W.%20&amp;%20Xie,%20X.%20S.%20Probing%20transcription%20factor%20dynamics%20at%20the%20single-molecule%20level%20in%20a%20living%20cell.%20Sci.%20(N.Y.)316,%201191%E2%80%931194%20(2007)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR118">
<span class="label">118.</span><cite>Persson, F., Lindén, M., Unoson, C. &amp; Elf, J. Extracting intracellular diffusive states and transition rates from single-molecule tracking data. <em>Nat. Methods 2013 10:3</em><strong>10</strong>, 265–269 (2013).</cite> [<a href="https://doi.org/10.1038/nmeth.2367" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23396281/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Persson,%20F.,%20Lind%C3%A9n,%20M.,%20Unoson,%20C.%20&amp;%20Elf,%20J.%20Extracting%20intracellular%20diffusive%20states%20and%20transition%20rates%20from%20single-molecule%20tracking%20data.%20Nat.%20Methods%202013%2010:310,%20265%E2%80%93269%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR119">
<span class="label">119.</span><cite>Kapanidis, A. N., Uphoff, S. &amp; Stracy, M. Understanding Protein Mobility in Bacteria by Tracking Single Molecules. <em>J. Mol. Biol.</em><strong>430</strong>, 4443–4455 (2018).
</cite> [<a href="https://doi.org/10.1016/j.jmb.2018.05.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6198114/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29753778/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kapanidis,%20A.%20N.,%20Uphoff,%20S.%20&amp;%20Stracy,%20M.%20Understanding%20Protein%20Mobility%20in%20Bacteria%20by%20Tracking%20Single%20Molecules.%20J.%20Mol.%20Biol.430,%204443%E2%80%934455%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR120">
<span class="label">120.</span><cite>Ferullo, D. J., Cooper, D. L., Moore, H. R. &amp; Lovett, S. T. Cell cycle synchronization of <em>Escherichia coli</em> using the stringent response, with fluorescence labeling assays for DNA content and replication. <em>Methods</em><strong>48</strong>, 8–13 (2009).
</cite> [<a href="https://doi.org/10.1016/j.ymeth.2009.02.010" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2746677/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19245839/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ferullo,%20D.%20J.,%20Cooper,%20D.%20L.,%20Moore,%20H.%20R.%20&amp;%20Lovett,%20S.%20T.%20Cell%20cycle%20synchronization%20of%20Escherichia%20coli%20using%20the%20stringent%20response,%20with%20fluorescence%20labeling%20assays%20for%20DNA%20content%20and%20replication.%20Methods48,%208%E2%80%9313%20(2009)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR121">
<span class="label">121.</span><cite>Doellinger, J., Schneider, A., Hoeller, M. &amp; Lasch, P. Sample Preparation by Easy Extraction and Digestion (SPEED) - A Universal, Rapid, and Detergent-free Protocol for Proteomics Based on Acid Extraction. <em>Mol. Cell Proteom.</em><strong>19</strong>, 209–222 (2020).</cite> [<a href="https://doi.org/10.1074/mcp.TIR119.001616" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6944244/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31754045/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Doellinger,%20J.,%20Schneider,%20A.,%20Hoeller,%20M.%20&amp;%20Lasch,%20P.%20Sample%20Preparation%20by%20Easy%20Extraction%20and%20Digestion%20(SPEED)%20-%20A%20Universal,%20Rapid,%20and%20Detergent-free%20Protocol%20for%20Proteomics%20Based%20on%20Acid%20Extraction.%20Mol.%20Cell%20Proteom.19,%20209%E2%80%93222%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR122">
<span class="label">122.</span><cite>Abele, M. et al. Unified Workflow for the Rapid and In-Depth Characterization of Bacterial Proteomes. <em>Mol. Cell Proteom.</em><strong>22</strong>, 100612 (2023).</cite> [<a href="https://doi.org/10.1016/j.mcpro.2023.100612" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10407251/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37391045/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Abele,%20M.%20et%20al.%20Unified%20Workflow%20for%20the%20Rapid%20and%20In-Depth%20Characterization%20of%20Bacterial%20Proteomes.%20Mol.%20Cell%20Proteom.22,%20100612%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR123">
<span class="label">123.</span><cite>Wu, C. et al. Enzyme expression kinetics by <em>Escherichia coli</em> during transition from rich to minimal media depends on proteome reserves. <em>Nat. Microbiol</em><strong>8</strong>, 347–359 (2023).
</cite> [<a href="https://doi.org/10.1038/s41564-022-01310-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9994330/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36737588/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wu,%20C.%20et%20al.%20Enzyme%20expression%20kinetics%20by%20Escherichia%20coli%20during%20transition%20from%20rich%20to%20minimal%20media%20depends%20on%20proteome%20reserves.%20Nat.%20Microbiol8,%20347%E2%80%93359%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR124">
<span class="label">124.</span><cite>Demichev, V., Messner, C. B., Vernardis, S. I., Lilley, K. S. &amp; Ralser, M. DIA-NN: neural networks and interference correction enable deep proteome coverage in high throughput. <em>Nat. Methods</em><strong>17</strong>, 41–44 (2020).
</cite> [<a href="https://doi.org/10.1038/s41592-019-0638-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6949130/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31768060/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Demichev,%20V.,%20Messner,%20C.%20B.,%20Vernardis,%20S.%20I.,%20Lilley,%20K.%20S.%20&amp;%20Ralser,%20M.%20DIA-NN:%20neural%20networks%20and%20interference%20correction%20enable%20deep%20proteome%20coverage%20in%20high%20throughput.%20Nat.%20Methods17,%2041%E2%80%9344%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR125">
<span class="label">125.</span><cite>Meng, C. Fast analyzing, exploring and sharing quantitative omics data using omicsViewer. 2022.03.10.483845 Preprint at 10.1101/2022.03.10.483845 (2022).</cite>
</li>
<li id="CR126">
<span class="label">126.</span><cite><em>Experiments in molecular genetics</em>. (Yi Hsien Pub. Co, Taipei, 1984).</cite>
</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12371093/bin/41467_2025_63147_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information</a><sup> (1.8MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12371093/bin/41467_2025_63147_MOESM2_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">41467_2025_63147_MOESM2_ESM.pdf</a><sup> (39.8KB, pdf) </sup><p>Description of Additional Supplementary Files</p>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12371093/bin/41467_2025_63147_MOESM3_ESM.mov" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Movie 1</a><sup> (27.8MB, mov) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material4_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12371093/bin/41467_2025_63147_MOESM4_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Reporting Summary</a><sup> (88.9KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material5_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12371093/bin/41467_2025_63147_MOESM5_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Transparent Peer Review file</a><sup> (708.9KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material6_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12371093/bin/41467_2025_63147_MOESM6_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Source Data</a><sup> (2.4MB, xlsx) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>All the localisation data and related brightfield images used during sptPALM in this study have been deposited on Zenodo (10.5281/zenodo.13939193). A list of the matches to the DnaA box sequences on the genome of all analysed <em>E. coli</em> strains and all cellular and single-molecule parameters generated in this study are provided in the Source Data file. The mass spectrometric raw files, as well as the DIA-NN output files have been deposited to the ProteomeXchange Consortium via the PRIDE partner repository and can be accessed using the identifier <a href="https://proteomecentral.proteomexchange.org/cgi/GetDataset?ID=PXD064841" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PXD064841</a>. All the mutant <em>E. coli</em> strains carrying the <em>dnaA-PAmCherry2.1</em> locus are available upon request. <a href="#Sec28" class="usa-link">Source data</a> are provided with this paper.</p>
<p>Scripts used in this study to analyse position of DnaA boxes across genomes and to generate single-particle tracking data have been deposited on Zenodo (10.5281/zenodo.13939193). The scripts used to analyse the disposition of DnaA boxes on the genomes of <em>E. coli</em> and <em>S. enterica</em> are also available on GitHub (<a href="https://github.com/stephkoest/Ecoli_titration" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/stephkoest/Ecoli_titration</a>).</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Nature Communications are provided here courtesy of <strong>Nature Publishing Group</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1038/s41467-025-63147-1"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/41467_2025_Article_63147.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (3.4 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12371093/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12371093/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12371093%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371093/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12371093/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12371093/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40841366/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12371093/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40841366/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12371093/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12371093/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="fG6UyhEqtvMnUiyLTXLQU24k1KMwdboMNGyCKYSUDVrubfJ3khu3e7Q8dkcTvFDr">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
        <script type="module" crossorigin="" src="/static/assets/math-574fdcc6.js"></script>
    
    

    </body>
</html>
