
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            A telomere-to-telomere gap-free genome assembly of the protandrous hermaphrodite Asian seabass (Lates calcarifer) - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="AE53312E8AF25D6305312E001592143A.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="sdata">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371012/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Scientific Data">
<meta name="citation_title" content="A telomere-to-telomere gap-free genome assembly of the protandrous hermaphrodite Asian seabass (Lates calcarifer)">
<meta name="citation_author" content="Xinhui Zhang">
<meta name="citation_author_institution" content="Laboratory of Aquatic Genomics, College of Life Sciences and Oceanography, Shenzhen University, Shenzhen, 518057 China">
<meta name="citation_author_institution" content="Shenzhen Key Lab of Marine Genomics, Guangdong Provincial Key Lab of Molecular Breeding in Marine Economic Animals, BGI Academy of Marine Sciences, Shenzhen, 518081 China">
<meta name="citation_author" content="Jieming Chen">
<meta name="citation_author_institution" content="Shenzhen Key Lab of Marine Genomics, Guangdong Provincial Key Lab of Molecular Breeding in Marine Economic Animals, BGI Academy of Marine Sciences, Shenzhen, 518081 China">
<meta name="citation_author" content="Wenchuan Zhou">
<meta name="citation_author_institution" content="Shenzhen Ocean Development and Promotion Center, Shenzhen, 518000 China">
<meta name="citation_author" content="Jiufu Wen">
<meta name="citation_author_institution" content="Key Laboratory of South China Sea Fishery Resources Exploitation and Utilization, Ministry of Agriculture and Rural Affairs, South China Sea Fisheries Research Institute, Chinese Academy of Fishery Sciences, Guangzhou, 510300 China">
<meta name="citation_author_institution" content="Hainan Engineering Research Center for Deep-Sea Aquaculture and Processing, Sanya, 572018 China">
<meta name="citation_author" content="Qiong Shi">
<meta name="citation_author_institution" content="Laboratory of Aquatic Genomics, College of Life Sciences and Oceanography, Shenzhen University, Shenzhen, 518057 China">
<meta name="citation_author_institution" content="Shenzhen Key Lab of Marine Genomics, Guangdong Provincial Key Lab of Molecular Breeding in Marine Economic Animals, BGI Academy of Marine Sciences, Shenzhen, 518081 China">
<meta name="citation_publication_date" content="2025 Aug 21">
<meta name="citation_volume" content="12">
<meta name="citation_firstpage" content="1457">
<meta name="citation_doi" content="10.1038/s41597-025-05735-w">
<meta name="citation_pmid" content="40841804">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371012/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371012/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371012/pdf/41597_2025_Article_5735.pdf">
<meta name="description" content="As a protandrous hermaphroditic fish species with natural sex change from male to female, Asian seabass (Lates calcarifer) represents an attractive model for studying sequential hermaphroditism. In this study, we constructed the first ...">
<meta name="og:title" content="A telomere-to-telomere gap-free genome assembly of the protandrous hermaphrodite Asian seabass (Lates calcarifer)">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="As a protandrous hermaphroditic fish species with natural sex change from male to female, Asian seabass (Lates calcarifer) represents an attractive model for studying sequential hermaphroditism. In this study, we constructed the first ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371012/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12371012">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1038/s41597-025-05735-w"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/41597_2025_Article_5735.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12371012%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12371012/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12371012/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371012/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-sdata.jpg" alt="Scientific Data logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Scientific Data" title="Link to Scientific Data" shape="default" href="http://www.nature.com/sdata/" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Sci Data</button></div>. 2025 Aug 21;12:1457. doi: <a href="https://doi.org/10.1038/s41597-025-05735-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1038/s41597-025-05735-w</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Sci%20Data%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sci%20Data%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Sci%20Data%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Sci%20Data%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>A telomere-to-telomere gap-free genome assembly of the protandrous hermaphrodite Asian seabass (<em>Lates calcarifer</em>)</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20X%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Xinhui Zhang</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Xinhui Zhang</span></h3>
<div class="p">
<sup>1</sup>Laboratory of Aquatic Genomics, College of Life Sciences and Oceanography, Shenzhen University, Shenzhen, 518057 China </div>
<div class="p">
<sup>2</sup>Shenzhen Key Lab of Marine Genomics, Guangdong Provincial Key Lab of Molecular Breeding in Marine Economic Animals, BGI Academy of Marine Sciences, Shenzhen, 518081 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20X%22%5BAuthor%5D" class="usa-link"><span class="name western">Xinhui Zhang</span></a>
</div>
</div>
<sup>1,</sup><sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chen%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Jieming Chen</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Jieming Chen</span></h3>
<div class="p">
<sup>2</sup>Shenzhen Key Lab of Marine Genomics, Guangdong Provincial Key Lab of Molecular Breeding in Marine Economic Animals, BGI Academy of Marine Sciences, Shenzhen, 518081 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chen%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Jieming Chen</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhou%20W%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Wenchuan Zhou</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Wenchuan Zhou</span></h3>
<div class="p">
<sup>3</sup>Shenzhen Ocean Development and Promotion Center, Shenzhen, 518000 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhou%20W%22%5BAuthor%5D" class="usa-link"><span class="name western">Wenchuan Zhou</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wen%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Jiufu Wen</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Jiufu Wen</span></h3>
<div class="p">
<sup>4</sup>Key Laboratory of South China Sea Fishery Resources Exploitation and Utilization, Ministry of Agriculture and Rural Affairs, South China Sea Fisheries Research Institute, Chinese Academy of Fishery Sciences, Guangzhou, 510300 China </div>
<div class="p">
<sup>5</sup>Hainan Engineering Research Center for Deep-Sea Aquaculture and Processing, Sanya, 572018 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wen%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Jiufu Wen</span></a>
</div>
</div>
<sup>4,</sup><sup>5,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Shi%20Q%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Qiong Shi</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Qiong Shi</span></h3>
<div class="p">
<sup>1</sup>Laboratory of Aquatic Genomics, College of Life Sciences and Oceanography, Shenzhen University, Shenzhen, 518057 China </div>
<div class="p">
<sup>2</sup>Shenzhen Key Lab of Marine Genomics, Guangdong Provincial Key Lab of Molecular Breeding in Marine Economic Animals, BGI Academy of Marine Sciences, Shenzhen, 518081 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Shi%20Q%22%5BAuthor%5D" class="usa-link"><span class="name western">Qiong Shi</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>Laboratory of Aquatic Genomics, College of Life Sciences and Oceanography, Shenzhen University, Shenzhen, 518057 China </div>
<div id="Aff2">
<sup>2</sup>Shenzhen Key Lab of Marine Genomics, Guangdong Provincial Key Lab of Molecular Breeding in Marine Economic Animals, BGI Academy of Marine Sciences, Shenzhen, 518081 China </div>
<div id="Aff3">
<sup>3</sup>Shenzhen Ocean Development and Promotion Center, Shenzhen, 518000 China </div>
<div id="Aff4">
<sup>4</sup>Key Laboratory of South China Sea Fishery Resources Exploitation and Utilization, Ministry of Agriculture and Rural Affairs, South China Sea Fisheries Research Institute, Chinese Academy of Fishery Sciences, Guangzhou, 510300 China </div>
<div id="Aff5">
<sup>5</sup>Hainan Engineering Research Center for Deep-Sea Aquaculture and Processing, Sanya, 572018 China </div>
<div class="author-notes p"><div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div></div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 Apr 10; Accepted 2025 Jul 31; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by-nc-nd/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12371012  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40841804/" class="usa-link">40841804</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<p id="Par1">As a protandrous hermaphroditic fish species with natural sex change from male to female, Asian seabass (<em>Lates calcarifer</em>) represents an attractive model for studying sequential hermaphroditism. In this study, we constructed the first telomere-to-telomere (T2T) gap-free genome assembly of Asian seabass, by integration of MGI short-read, PacBio HiFi long-read, ONT ultra-long and Hi-C sequencing technologies. The haplotypic 614.19 Mb genome sequences were successfully anchored onto 24 chromosomes, demonstrating exceptional contiguity with a contig N50 of 26.57 Mb. Comprehensive annotation revealed precise localization of telomeric repeats and centromeric regions across various chromosomes. Good results from Merqury (QV: 57.8), CRAQ (99.45%) and BUSCO (100%) indicate a high level of accuracy for the assembled genome. ONT ultra-long and PacBio HiFi sequencing data were aligned with the assembly using minimap2, resulting in a mapping rate over 98%. Repetitive elements accounted for 18.18% (111.64 Mb) of the entire genome, and a total of 25,093 protein-coding genes were annotated. This high-quality T2T genome assembly provides a valuable genetic resource for in-depth comparative genomics, population genetics, molecular breeding, and functional studies of this economically important marine species. This reference assembly also facilitates investigations into the detailed molecular mechanisms underlying its unique reproductive strategy of the protandrous hermaphrodite Asian seabass.</p>
<section id="kwd-group1" class="kwd-group"><p><strong>Subject terms:</strong> Genome, Evolutionary genetics</p></section></section><section id="Sec1"><h2 class="pmc_sec_title">Background &amp; Summary</h2>
<p id="Par2">Sex determination is a genetic or epigenetic process that initiates and regulates the developmental trajectory of sexual differentiation, whereas sex differentiation encompasses the cascade of morphological and physiological events through which a bi-potential gonad progressively develops into either a testis or an ovary, culminating in the establishment of species-specific secondary sexual characteristics<sup><a href="#CR1" class="usa-link" aria-describedby="CR1">1</a></sup>. Compared with those highly conserved sex determination systems in various mammals and birds, fishes exhibit remarkable diversity in sex determination patterns. They present more diversified sex determination modes than higher vertebrates, such as genetic sex determination (GSD), environmental sex determination (ESD), and the coexistence of both<sup><a href="#CR2" class="usa-link" aria-describedby="CR2">2</a>,<a href="#CR3" class="usa-link" aria-describedby="CR3">3</a></sup>. Notably, among diverse environmental cues, temperature emerges as the most influential exogenous factor to modulate sexual development in fishes. Numerous species across different taxa have been documented to own thermally sensitive sex determination, where incubation temperature during critical developmental windows can override genotypic sex determinants. Good examples include European seabass (<em>Dicentrarchus labrax</em>)<sup><a href="#CR4" class="usa-link" aria-describedby="CR4">4</a></sup>, tilapia (<em>Nile tilapia</em> and <em>Oreochromis niloticus</em>)<sup><a href="#CR5" class="usa-link" aria-describedby="CR5">5</a></sup>, and Atlantic halibut (<em>Hippoglossus hippoglossus</em>)<sup><a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>,<a href="#CR7" class="usa-link" aria-describedby="CR7">7</a></sup>. These fishes exhibit interesting characteristics of temperature-dependent sex determination, and their sex ratios can change significantly with variations in environmental temperature during their hatching period.</p>
<p id="Par3">In addition to gonochorism (separate sexes), fishes also exhibit hermaphroditism as an important reproductive strategy. Approximately 2% of teleost fishes are hermaphroditic, distributed across 27 families within 7 orders<sup><a href="#CR8" class="usa-link" aria-describedby="CR8">8</a></sup>. Sex change is a biological process in which an organism transitions from its original sex to another through specific physiological mechanisms. Organisms capable of naturally undergoing sex change are referred to hermaphrodites, which are typically categorized into protandrous (male-to-female) and protogynous (female-to-male)<sup><a href="#CR9" class="usa-link" aria-describedby="CR9">9</a></sup>. Common examples in these fishes include groupers, black seabream, clownfish, and ricefield eel<sup><a href="#CR10" class="usa-link" aria-describedby="CR10">10</a>–<a href="#CR13" class="usa-link" aria-describedby="CR13">13</a></sup>.</p>
<p id="Par4">Asian seabass holds substantial cultural and economic values throughout the tropical Indo-West Pacific region, serving as both a key fishery resource and a commercially important aquaculture species<sup><a href="#CR14" class="usa-link" aria-describedby="CR14">14</a></sup>. As a protandrous hermaphroditic fish<sup><a href="#CR15" class="usa-link" aria-describedby="CR15">15</a></sup>, it usually first develops into a male at 3–4 years of age, and then approximately 90% of individuals undergoes natural sex change to female by age 6<sup><a href="#CR16" class="usa-link" aria-describedby="CR16">16</a></sup>. Despite its remarkable reproductive strategy, the genetic mechanisms underlying sex change in Asian seabass remain poorly understood, as is the case for most hermaphroditic species. Genomic resources, including DNA markers, high-resolution linkage maps, transcriptomes, reference genome sequences along with their comprehensive annotations, play a pivotal role in supporting aquaculture. These valuable genetic resources provide a solid foundation for diverse applications, enabling comprehensive genetic investigations to support development of sophisticated artificial breeding strategies. Ultimately, they contribute to the sustainable expansion and increased productivity of international aquaculture industry<sup><a href="#CR14" class="usa-link" aria-describedby="CR14">14</a></sup>. Given the economic value of Asian seabass and its remarkable natural sex change, construction of its high-quality genome assembly is absolutely essential.</p>
<p id="Par5">In this study, we combined MGI short-read, PacBio HiFi long-read, ONT (Oxford Nanopore Technologies) ultra-long, and Hi-C sequencing data to generate a high-fidelity T2T genome assembly of Asian seabass. This assembly was rigorously assessed for quality, and its key genomic features were systematically characterized. In fact, this gap-free and complete reference assembly represents a substantial improvement over any previous assembly of this species<sup><a href="#CR17" class="usa-link" aria-describedby="CR17">17</a></sup>. It will not only facilitate population genetic research and evolutionary study, but also provide an important genetic resource for molecular breeding and investigating molecular mechanisms of sex change in this economically important fish.</p></section><section id="Sec2"><h2 class="pmc_sec_title">Methods</h2>
<section id="Sec3"><h3 class="pmc_sec_title">Sample collection</h3>
<p id="Par6">A male Asian seabass (Fig. <a href="#Fig1" class="usa-link">1A</a>) was collected from a local aquaculture facility of the South China Sea Fisheries Research Institute under Chinese Academy of Fishery Sciences, which is located in Guangzhou City, Guangdong Province, China. Muscle tissue was sampled for whole-genome sequencing, including MGI short read, PacBio HiFi long read, ONT (Oxford Nanopore Technologies) Ultra-long and Hi-C sequencing technologies. Additionally, seven distinct tissues (such as gill, brain, liver, muscle, eye, testis, and skin) were collected for transcriptome sequencing (Table <a href="#Tab1" class="usa-link">1</a>). Upon dissection into small fragments, the tissue samples were washed with ice-cold PBS (pH 7.4) to eliminate blood residues and contaminants. After removing outside liquid by blotting, these samples were rapidly frozen in liquid nitrogen and subsequently maintained at −80 °C before use. For transcriptome sequencing, frozen specimens were shipped in dry ice containers to the sequencing company (BGI, Shenzhen, Guangdong, China).</p>
<figure class="fig xbox font-sm" id="Fig1"><h4 class="obj_head">Fig. 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371012_41597_2025_5735_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/fd2c/12371012/ae96fbd2458f/41597_2025_5735_Fig1_HTML.jpg" loading="lazy" id="d33e319" height="999" width="740" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Asian seabass and its whole-genome sequence distribution. (<strong>A</strong>) A morphological image of the sequenced Asian seabass. (<strong>B</strong>) A k-mer (21-mer) distribution curve for estimation of the genome size.</p></figcaption></figure><section class="tw xbox font-sm" id="Tab1"><h4 class="obj_head">Table 1.</h4>
<div class="caption p"><p>Sequencing data of the Asian seabass genome and transcriptomes.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th colspan="1" rowspan="1">Type</th>
<th colspan="1" rowspan="1">Library type</th>
<th colspan="1" rowspan="1">Raw data (Gb)</th>
<th colspan="1" rowspan="1">Clean data (Gb)</th>
<th colspan="1" rowspan="1">Read N50/ length (bp)</th>
<th colspan="1" rowspan="1">Coverage of the genome (×)</th>
</tr></thead>
<tbody>
<tr>
<td rowspan="4" colspan="1"><strong>DNA</strong></td>
<td colspan="1" rowspan="1">MGI</td>
<td colspan="1" rowspan="1">37.41</td>
<td colspan="1" rowspan="1">33.69</td>
<td colspan="1" rowspan="1">150</td>
<td colspan="1" rowspan="1">54</td>
</tr>
<tr>
<td colspan="1" rowspan="1">PacBio HiFi</td>
<td colspan="1" rowspan="1">/</td>
<td colspan="1" rowspan="1">90.47</td>
<td colspan="1" rowspan="1">18,366*</td>
<td colspan="1" rowspan="1">141</td>
</tr>
<tr>
<td colspan="1" rowspan="1">ONT Ultra-long</td>
<td colspan="1" rowspan="1">/</td>
<td colspan="1" rowspan="1">61.34</td>
<td colspan="1" rowspan="1">71,701*</td>
<td colspan="1" rowspan="1">95</td>
</tr>
<tr>
<td colspan="1" rowspan="1">Hi-C</td>
<td colspan="1" rowspan="1">102.32</td>
<td colspan="1" rowspan="1">93.8</td>
<td colspan="1" rowspan="1">150</td>
<td colspan="1" rowspan="1">133.33</td>
</tr>
<tr>
<td rowspan="7" colspan="1"><strong>RNA</strong></td>
<td colspan="1" rowspan="1">Eye</td>
<td colspan="1" rowspan="1">6.104</td>
<td colspan="1" rowspan="1">5.236</td>
<td colspan="1" rowspan="1">150</td>
<td colspan="1" rowspan="1">/</td>
</tr>
<tr>
<td colspan="1" rowspan="1">Muscle</td>
<td colspan="1" rowspan="1">6.207</td>
<td colspan="1" rowspan="1">5.625</td>
<td colspan="1" rowspan="1">150</td>
<td colspan="1" rowspan="1">/</td>
</tr>
<tr>
<td colspan="1" rowspan="1">Skin</td>
<td colspan="1" rowspan="1">6.332</td>
<td colspan="1" rowspan="1">5.769</td>
<td colspan="1" rowspan="1">150</td>
<td colspan="1" rowspan="1">/</td>
</tr>
<tr>
<td colspan="1" rowspan="1">Liver</td>
<td colspan="1" rowspan="1">7.007</td>
<td colspan="1" rowspan="1">6.377</td>
<td colspan="1" rowspan="1">150</td>
<td colspan="1" rowspan="1">/</td>
</tr>
<tr>
<td colspan="1" rowspan="1">Gill</td>
<td colspan="1" rowspan="1">6.868</td>
<td colspan="1" rowspan="1">6.263</td>
<td colspan="1" rowspan="1">150</td>
<td colspan="1" rowspan="1">/</td>
</tr>
<tr>
<td colspan="1" rowspan="1">Brain</td>
<td colspan="1" rowspan="1">6.335</td>
<td colspan="1" rowspan="1">5.783</td>
<td colspan="1" rowspan="1">150</td>
<td colspan="1" rowspan="1">/</td>
</tr>
<tr>
<td colspan="1" rowspan="1">Testis</td>
<td colspan="1" rowspan="1">9.195</td>
<td colspan="1" rowspan="1">8.395</td>
<td colspan="1" rowspan="1">150</td>
<td colspan="1" rowspan="1">/</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="_fn_p9"><p>*For the PacBio HiFi and ONT Utra-long sequencing, this number is N50 of reads; for others, it denotes read length.</p></div></div></section></section><section id="Sec4"><h3 class="pmc_sec_title">DNA extraction and genome sequencing</h3>
<p id="Par7">Genomic DNA (gDNA) was extracted from muscle tissue using a QIAamp DNA Mini Kit (Qiagen, Valencia, CA, USA) following the manufacturer’s protocols<sup><a href="#CR18" class="usa-link" aria-describedby="CR18">18</a></sup>. Fragment size, purity, and quantification of the extracted gDNA were assessed via 0.75% agarose gel electrophoresis, an Agilent 2100 Bioanalyzer (Agilent Technologies, Palo Alto, CA, USA) and a Qubit Fluorometer (Thermo Fisher Scientific, Waltham, MA, USA), respectively.</p>
<p id="Par8">For the MGI short-read sequencing, gDNA was randomly fragmented using a MGIEasy Universal DNA Library Preparation Kit (MGI, Shenzhen, China) to construct a library with an insert-size of 350 bp. Sequencing was performed on a DNBSEQ-T7 platform (MGI), generating 37.4 Gb of raw 150-bp paired-end reads, and then filtered by fastp v0.12.6<sup><a href="#CR19" class="usa-link" aria-describedby="CR19">19</a></sup> (parameter: -n 0 -f 5 -F 5 -t 5 -T 5) to remove adaptor sequences and low-quality reads. Finally, a total of 33.69 Gb of clean reads (Table <a href="#Tab1" class="usa-link">1</a>) were obtained for further data error correction and genome-size estimation.</p>
<p id="Par9">For the PacBio HiFi sequencing, approximately 10 μg of high-quality gDNA was applied to construct a SMRTbell library following the manufacturer’s standard protocol (SMRTbell Express Template Prep Kit 2.0; Pacific Biosciences, Menlo Park, CA, USA), which was then sequenced on a PacBio Sequel II System using the circular consensus sequencing (CCS) technology. A total of 90.47 Gb of HiFi reads with a N50 of 18,366 bp were obtained (Table <a href="#Tab1" class="usa-link">1</a>) using the CCS v6.0.0<sup><a href="#CR20" class="usa-link" aria-describedby="CR20">20</a></sup> (Circular Consensus Sequencing) software with the optimized parameter “-min-passes 3”.</p>
<p id="Par10">Two ultra-long read libraries were constructed using Oxford Nanopore Technologies (ONT) protocols, which were sequenced on a PromethION platform (Oxford Nanopore Technologies Co., Littlemore, Oxford, UK). Raw reads were initially processed to eliminate those with a quality value (QV) lower than 7 using the NanoFilt v2.8.0<sup><a href="#CR21" class="usa-link" aria-describedby="CR21">21</a></sup> software. Finally, a total of 1.54 million clean reads were retained, accumulating a substantial base count of 61.32 Gb. The average read length was 39.69 kb, with an N50 length of 71.17 kb (Table <a href="#Tab1" class="usa-link">1</a>).</p>
<p id="Par11">For the high-throughput chromosome conformation capture (Hi-C) sequencing, one Hi-C library was generated using a GrandOmics Hi-C kit (GrandOmics, Wuhan, Hubei, China) following the manufacturer’s protocol. In brief, gDNA was first cross-linked using a 4% formaldehyde solution to stabilize chromatin structures. Subsequently, the DNA was digested with the restriction enzyme <em>MboI</em> to introduce specific cleavage sites. Those resulting DNA fragments were then labeled with biotin-14-dCTP, allowing for incorporation of a detectable marker. The labeled DNA fragments were ligated using T4 DNA ligase to facilitate subsequent enrichment steps. Following ligation, the DNA was further digested to yield fragments in the size range of 200 to 600 bp. The library was sequenced on a DNBSEQ-T7 platform (MGI, Shenzhen, China) using a 150-bp paired-end model. The Hi-C sequencing technology generated 102.32 Gb of raw data. Subsequently, fastp v0.12.6<sup><a href="#CR19" class="usa-link" aria-describedby="CR19">19</a></sup> was applied to filter adaptor sequences and low-quality reads. Finally, 93.8 Gb of Hi-C clean data were retained (Table <a href="#Tab1" class="usa-link">1</a>) for chromosome assembly.</p></section><section id="Sec5"><h3 class="pmc_sec_title">RNA extraction and transcriptome sequencing (RNA-seq)</h3>
<p id="Par12">Total RNA was extracted from seven tissues separately according to a standard Trizol protocol (Invitrogen, Frederick, MD, USA), followed by purification with a Qiagen RNeasy Mini Kit (Qiagen, Germantown, MD, USA). RNA concentration and integrity were measured using a NanoDrop 8000 Spectrophotometer (Thermo Fisher Scientific, Waltham, MA, USA) and an Agilent 2100 Bioanalyzer (Agilent Technologies, Santa Clara, CA, USA), respectively. Only those RNA samples with OD260/280 ≥ 1.8 and RNA integrity ≥ 7.0 were selected for transcriptome sequencing. RNA was used for construction of a cDNA library followed the manufacture’s guideline, which was then sequenced on a HiSeq X Ten platform (Illumina, San Diego, CA, USA). A total of 48.07 Gb of transcriptome raw data were generated (Table <a href="#Tab1" class="usa-link">1</a>), which aided in annotation of protein-coding genes and prediction of gene structures.</p></section><section id="Sec6"><h3 class="pmc_sec_title">Genome-size estimation and construction of a T2T genome assembly</h3>
<p id="Par13">To estimate the genome size of Asian seabass, we employed jellyfish (v2.2.10)<sup><a href="#CR22" class="usa-link" aria-describedby="CR22">22</a></sup> to perform k-mer counting with k = 21, and the parameters were set as ‘-m 21 -s 10 G -C’. Subsequently, a generated histogram was utilized as an input file for GenomeScope v2.0<sup><a href="#CR23" class="usa-link" aria-describedby="CR23">23</a></sup> to estimate genetic characteristics. This approach provided a sequence-derived estimate of the Asian seabass genome characteristics prior to assembly. Our analysis results show that the genome size of Asian seabass is approximately 576.74 Mb, with an estimated heterozygosity of about 0.46% (Fig. <a href="#Fig1" class="usa-link">1B</a>) and repetitive sequences accounting for 32.79 Mb (5.69%).</p>
<p id="Par14">Primary contigs were initially generated by assembling PacBio HiFi and ONT data using Hifiasm v0.19.8<sup><a href="#CR24" class="usa-link" aria-describedby="CR24">24</a></sup> with default parameters. Then, purge_dups v1.2.5<sup><a href="#CR25" class="usa-link" aria-describedby="CR25">25</a></sup> was employed to remove haplotypic and heterozygous duplications from the <em>de novo</em> assembly, yielding a final assembly with a total length of 614.08 Mb.</p>
<p id="Par15">Using the preliminary assembly as the reference, Hi-C clean reads were utilized to construct chromosomes for Asian seabass. First, the Hi-C reads were mapped to the assembled contigs using bowtie2 v2.2.5 (–very-sensitive -L 20–score-min L, −0.6, −0.2–end-to-end)<sup><a href="#CR26" class="usa-link" aria-describedby="CR26">26</a></sup>. Subsequently, the HiC-Pro v2.8.1<sup><a href="#CR27" class="usa-link" aria-describedby="CR27">27</a></sup> pipeline was applied to detect ligation products, retaining only valid paired reads for downstream analysis. Based on these valid reads, the primary assembly was clustered, ordered, and oriented into chromosomes using the Juicer v1.5<sup><a href="#CR28" class="usa-link" aria-describedby="CR28">28</a></sup> and 3D-DNA v3.0<sup><a href="#CR29" class="usa-link" aria-describedby="CR29">29</a></sup> software with parameters -m haploid -r 2 -c 24. Juicebox v1.11.08<sup><a href="#CR30" class="usa-link" aria-describedby="CR30">30</a></sup> was employed to visualize before manually adjusting the candidate assemblies.</p>
<p id="Par16">To fill the remaining gaps, those corrected ultra-long ONT reads were applied to generate a gap-free genome assembly using TGS-GapCloser v1.2.1<sup><a href="#CR31" class="usa-link" aria-describedby="CR31">31</a></sup> with optimized parameter “–min_match 1000–min_nread 3” and LR_Gapcloser v1.0<sup><a href="#CR32" class="usa-link" aria-describedby="CR32">32</a></sup> with the parameter “-t 35 -m 1000000 -v 500”. The final genome assembly spans 614.19 Mb, and it is anchored onto 24 chromosomes (Fig. <a href="#Fig2" class="usa-link">2</a>), among them the longest and the shortest are 31.85 Mb and 14.85 Mb, respectively (Table <a href="#Tab2" class="usa-link">2</a>).</p>
<figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371012_41597_2025_5735_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/fd2c/12371012/77a4e33c3636/41597_2025_5735_Fig2_HTML.jpg" loading="lazy" id="d33e611" height="361" width="749" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>The first T2T genome assembly of Asian seabass. (<strong>A</strong>) Genome-wide chromatin interactions at a 500-kb resolution. Color blocks represent corresponding interactions, with various strengths from yellow (low) to red (high). (<strong>B</strong>) A Circos plot of the main genome features. From outside to inside include the 24 chromosomes, gene density, GC content, repetitive sequences density, and a colinear relationship among chromosomes of the Asian seabass genome assembly. Note that the density calculation window is set as 100 kb.</p></figcaption></figure><section class="tw xbox font-sm" id="Tab2"><h4 class="obj_head">Table 2.</h4>
<div class="caption p"><p>Comparison of the available genome assemblies for Asian seabass.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th colspan="1" rowspan="1">Category</th>
<th colspan="1" rowspan="1">This study</th>
<th colspan="1" rowspan="1">
<em>L. calcarifer</em> (ASB-BC8)<sup><a href="#CR17" class="usa-link" aria-describedby="CR17">17</a></sup>
</th>
</tr></thead>
<tbody>
<tr>
<td colspan="1" rowspan="1"><strong>Genome survey (Mb)</strong></td>
<td colspan="1" rowspan="1">576.74</td>
<td colspan="1" rowspan="1">593–648</td>
</tr>
<tr>
<td colspan="1" rowspan="1"><strong>Genome length (bp)</strong></td>
<td colspan="1" rowspan="1">614,195,649</td>
<td colspan="1" rowspan="1">668,464,831</td>
</tr>
<tr>
<td colspan="1" rowspan="1"><strong>Longest scaffold (bp)</strong></td>
<td colspan="1" rowspan="1">31,852,513</td>
<td colspan="1" rowspan="1">30,776,907</td>
</tr>
<tr>
<td colspan="1" rowspan="1"><strong>Number of scaffolds</strong></td>
<td colspan="1" rowspan="1">24</td>
<td colspan="1" rowspan="1">3,807</td>
</tr>
<tr>
<td colspan="1" rowspan="1"><strong>Contig N50 (bp)</strong></td>
<td colspan="1" rowspan="1">26,575,253</td>
<td colspan="1" rowspan="1">1,066,117</td>
</tr>
<tr>
<td colspan="1" rowspan="1"><strong>Scaffold N50 (bp)</strong></td>
<td colspan="1" rowspan="1">26,575,253</td>
<td colspan="1" rowspan="1">25,848,596</td>
</tr>
<tr>
<td colspan="1" rowspan="1"><strong>GC content</strong></td>
<td colspan="1" rowspan="1">40.7%</td>
<td colspan="1" rowspan="1">40.8%</td>
</tr>
<tr>
<td colspan="1" rowspan="1"><strong>BUSCO</strong></td>
<td colspan="1" rowspan="1">100% (S:99.94%; D:0.16%)</td>
<td colspan="1" rowspan="1">99.7% (S:96%; D:3.7%)</td>
</tr>
<tr>
<td colspan="1" rowspan="1"><strong>Number of chromosomes</strong></td>
<td colspan="1" rowspan="1">24</td>
<td colspan="1" rowspan="1">24</td>
</tr>
<tr>
<td colspan="1" rowspan="1"><strong>Chromosome length (bp)</strong></td>
<td colspan="1" rowspan="1">614,195,649</td>
<td colspan="1" rowspan="1">586,924,032</td>
</tr>
<tr>
<td colspan="1" rowspan="1"><strong>Repetitive sequence</strong></td>
<td colspan="1" rowspan="1">18.18%</td>
<td colspan="1" rowspan="1">/</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="_fn_p22"><p>Abbreviations: S, single copy complete genes; D, duplicated complete genes.</p></div></div></section></section><section id="Sec7"><h3 class="pmc_sec_title">Identification of the centromere and telomere sequences</h3>
<p id="Par17">Telomeres were identified by searching for the target sequence (CCCTAA/TTAGGG) at both ends of each chromosome using Telomere-to-Telomere Toolkit quarTeT v1.1.1<sup><a href="#CR33" class="usa-link" aria-describedby="CR33">33</a></sup>. Centromeres, as specialized DNA sequences connecting sister chromatids, exhibit complex structures in most animals and plants with highly repetitive satellite DNA and scattered retrotransposon sequences. In this study, after identifying repeat sequences according to TRF v4.0.4<sup><a href="#CR34" class="usa-link" aria-describedby="CR34">34</a></sup> and RepeatMasker v4.0.6<sup><a href="#CR35" class="usa-link" aria-describedby="CR35">35</a></sup> and obtaining a TE annotation file, quarTeT v1.1.1<sup><a href="#CR33" class="usa-link" aria-describedby="CR33">33</a></sup> was applied to identify centromeres, and the candidate interval range of every centromere was predicted. Ultimately, we determined that the Asian seabass genome contains a complete set of 24 centromeres and 48 telomeres (Table <a href="#Tab3" class="usa-link">3</a>; Fig. <a href="#Fig3" class="usa-link">3</a>).</p>
<section class="tw xbox font-sm" id="Tab3"><h4 class="obj_head">Table 3.</h4>
<div class="caption p"><p>Telomere and centromere positions in the assembled genome.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead>
<tr>
<th rowspan="2" colspan="1">Chr</th>
<th rowspan="2" colspan="1">Contig</th>
<th rowspan="2" colspan="1">Length (bp)</th>
<th rowspan="2" colspan="1">Gap</th>
<th colspan="4" rowspan="1">Telomere (Te)</th>
<th colspan="2" rowspan="1">Centromere (Ce)</th>
</tr>
<tr>
<th colspan="1" rowspan="1">Upstream Start</th>
<th colspan="1" rowspan="1">Upstream End</th>
<th colspan="1" rowspan="1">Downstream Start</th>
<th colspan="1" rowspan="1">Downstream End</th>
<th colspan="1" rowspan="1">Start</th>
<th colspan="1" rowspan="1">End</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="1" rowspan="1">Chr01</td>
<td colspan="1" rowspan="1">1</td>
<td colspan="1" rowspan="1">31,852,513</td>
<td colspan="1" rowspan="1">0</td>
<td colspan="1" rowspan="1">251</td>
<td colspan="1" rowspan="1">3,343</td>
<td colspan="1" rowspan="1">31,848,788</td>
<td colspan="1" rowspan="1">31,852,328</td>
<td colspan="1" rowspan="1">1,505,150</td>
<td colspan="1" rowspan="1">1,784,594</td>
</tr>
<tr>
<td colspan="1" rowspan="1">Chr02</td>
<td colspan="1" rowspan="1">1</td>
<td colspan="1" rowspan="1">31,638,724</td>
<td colspan="1" rowspan="1">0</td>
<td colspan="1" rowspan="1">29</td>
<td colspan="1" rowspan="1">1,,056</td>
<td colspan="1" rowspan="1">31,638,090</td>
<td colspan="1" rowspan="1">31,638,401</td>
<td colspan="1" rowspan="1">4,132,572</td>
<td colspan="1" rowspan="1">4,235,565</td>
</tr>
<tr>
<td colspan="1" rowspan="1">Chr03</td>
<td colspan="1" rowspan="1">1</td>
<td colspan="1" rowspan="1">29,918,513</td>
<td colspan="1" rowspan="1">0</td>
<td colspan="1" rowspan="1">64</td>
<td colspan="1" rowspan="1">6134</td>
<td colspan="1" rowspan="1">29,917,458</td>
<td colspan="1" rowspan="1">29,918,485</td>
<td colspan="1" rowspan="1">27,923,600</td>
<td colspan="1" rowspan="1">28,634,341</td>
</tr>
<tr>
<td colspan="1" rowspan="1">Chr04</td>
<td colspan="1" rowspan="1">1</td>
<td colspan="1" rowspan="1">29,558,833</td>
<td colspan="1" rowspan="1">0</td>
<td colspan="1" rowspan="1">493</td>
<td colspan="1" rowspan="1">3,991</td>
<td colspan="1" rowspan="1">29,553,555</td>
<td colspan="1" rowspan="1">29,558,758</td>
<td colspan="1" rowspan="1">16,763,413</td>
<td colspan="1" rowspan="1">16,977,890</td>
</tr>
<tr>
<td colspan="1" rowspan="1">Chr05</td>
<td colspan="1" rowspan="1">1</td>
<td colspan="1" rowspan="1">29,570,087</td>
<td colspan="1" rowspan="1">0</td>
<td colspan="1" rowspan="1">38</td>
<td colspan="1" rowspan="1">7,538</td>
<td colspan="1" rowspan="1">29,569,431</td>
<td colspan="1" rowspan="1">29,570,087</td>
<td colspan="1" rowspan="1">28,554,835</td>
<td colspan="1" rowspan="1">29,493,014</td>
</tr>
<tr>
<td colspan="1" rowspan="1">Chr06</td>
<td colspan="1" rowspan="1">1</td>
<td colspan="1" rowspan="1">29,199,514</td>
<td colspan="1" rowspan="1">0</td>
<td colspan="1" rowspan="1">63</td>
<td colspan="1" rowspan="1">5,031</td>
<td colspan="1" rowspan="1">29,198,324</td>
<td colspan="1" rowspan="1">29,199,475</td>
<td colspan="1" rowspan="1">9,609</td>
<td colspan="1" rowspan="1">779,212</td>
</tr>
<tr>
<td colspan="1" rowspan="1">Chr07</td>
<td colspan="1" rowspan="1">1</td>
<td colspan="1" rowspan="1">29,179,243</td>
<td colspan="1" rowspan="1">0</td>
<td colspan="1" rowspan="1">5</td>
<td colspan="1" rowspan="1">3,891</td>
<td colspan="1" rowspan="1">29,179,155</td>
<td colspan="1" rowspan="1">29,179,208</td>
<td colspan="1" rowspan="1">28,437,506</td>
<td colspan="1" rowspan="1">28,968,124</td>
</tr>
<tr>
<td colspan="1" rowspan="1">Chr08</td>
<td colspan="1" rowspan="1">1</td>
<td colspan="1" rowspan="1">27,751,246</td>
<td colspan="1" rowspan="1">0</td>
<td colspan="1" rowspan="1">7</td>
<td colspan="1" rowspan="1">6,788</td>
<td colspan="1" rowspan="1">27,747,489</td>
<td colspan="1" rowspan="1">27,751,129</td>
<td colspan="1" rowspan="1">1,198,505</td>
<td colspan="1" rowspan="1">1,281,772</td>
</tr>
<tr>
<td colspan="1" rowspan="1">Chr09</td>
<td colspan="1" rowspan="1">1</td>
<td colspan="1" rowspan="1">27,635,561</td>
<td colspan="1" rowspan="1">0</td>
<td colspan="1" rowspan="1">3</td>
<td colspan="1" rowspan="1">5,138</td>
<td colspan="1" rowspan="1">27,632,115</td>
<td colspan="1" rowspan="1">27,635,202</td>
<td colspan="1" rowspan="1">24,779,176</td>
<td colspan="1" rowspan="1">24,919,240</td>
</tr>
<tr>
<td colspan="1" rowspan="1">Chr10</td>
<td colspan="1" rowspan="1">1</td>
<td colspan="1" rowspan="1">26,717,877</td>
<td colspan="1" rowspan="1">0</td>
<td colspan="1" rowspan="1">44</td>
<td colspan="1" rowspan="1">3,433</td>
<td colspan="1" rowspan="1">26,713,583</td>
<td colspan="1" rowspan="1">26,717,829</td>
<td colspan="1" rowspan="1">11,069,449</td>
<td colspan="1" rowspan="1">11,101,393</td>
</tr>
<tr>
<td colspan="1" rowspan="1">Chr11</td>
<td colspan="1" rowspan="1">1</td>
<td colspan="1" rowspan="1">26,575,253</td>
<td colspan="1" rowspan="1">0</td>
<td colspan="1" rowspan="1">102</td>
<td colspan="1" rowspan="1">4,914</td>
<td colspan="1" rowspan="1">26,568,413</td>
<td colspan="1" rowspan="1">26,575,161</td>
<td colspan="1" rowspan="1">23,814,253</td>
<td colspan="1" rowspan="1">24,022,695</td>
</tr>
<tr>
<td colspan="1" rowspan="1">Chr12</td>
<td colspan="1" rowspan="1">1</td>
<td colspan="1" rowspan="1">26,190,281</td>
<td colspan="1" rowspan="1">0</td>
<td colspan="1" rowspan="1">3</td>
<td colspan="1" rowspan="1">4,913</td>
<td colspan="1" rowspan="1">26,185,410</td>
<td colspan="1" rowspan="1">26,190,261</td>
<td colspan="1" rowspan="1">1,972,765</td>
<td colspan="1" rowspan="1">2,010,536</td>
</tr>
<tr>
<td colspan="1" rowspan="1">Chr13</td>
<td colspan="1" rowspan="1">1</td>
<td colspan="1" rowspan="1">25,913,521</td>
<td colspan="1" rowspan="1">0</td>
<td colspan="1" rowspan="1">2</td>
<td colspan="1" rowspan="1">2,370</td>
<td colspan="1" rowspan="1">25,908,749</td>
<td colspan="1" rowspan="1">25,913,302</td>
<td colspan="1" rowspan="1">23,680,651</td>
<td colspan="1" rowspan="1">23,817,342</td>
</tr>
<tr>
<td colspan="1" rowspan="1">Chr14</td>
<td colspan="1" rowspan="1">1</td>
<td colspan="1" rowspan="1">25,614,823</td>
<td colspan="1" rowspan="1">0</td>
<td colspan="1" rowspan="1">6</td>
<td colspan="1" rowspan="1">4,500</td>
<td colspan="1" rowspan="1">25,609,695</td>
<td colspan="1" rowspan="1">25,614,798</td>
<td colspan="1" rowspan="1">130,448</td>
<td colspan="1" rowspan="1">540,036</td>
</tr>
<tr>
<td colspan="1" rowspan="1">Chr15</td>
<td colspan="1" rowspan="1">1</td>
<td colspan="1" rowspan="1">25,420,547</td>
<td colspan="1" rowspan="1">0</td>
<td colspan="1" rowspan="1">26</td>
<td colspan="1" rowspan="1">3,571</td>
<td colspan="1" rowspan="1">25,420,129</td>
<td colspan="1" rowspan="1">25,420,291</td>
<td colspan="1" rowspan="1">80,051</td>
<td colspan="1" rowspan="1">483,348</td>
</tr>
<tr>
<td colspan="1" rowspan="1">Chr16</td>
<td colspan="1" rowspan="1">1</td>
<td colspan="1" rowspan="1">25,111,693</td>
<td colspan="1" rowspan="1">0</td>
<td colspan="1" rowspan="1">3</td>
<td colspan="1" rowspan="1">4,038</td>
<td colspan="1" rowspan="1">25,065,727</td>
<td colspan="1" rowspan="1">25,111,549</td>
<td colspan="1" rowspan="1">1,479,578</td>
<td colspan="1" rowspan="1">1,691,299</td>
</tr>
<tr>
<td colspan="1" rowspan="1">Chr17</td>
<td colspan="1" rowspan="1">1</td>
<td colspan="1" rowspan="1">23,846,329</td>
<td colspan="1" rowspan="1">0</td>
<td colspan="1" rowspan="1">304</td>
<td colspan="1" rowspan="1">4,318</td>
<td colspan="1" rowspan="1">23,841,449</td>
<td colspan="1" rowspan="1">23,846,329</td>
<td colspan="1" rowspan="1">20,554,022</td>
<td colspan="1" rowspan="1">20,584,804</td>
</tr>
<tr>
<td colspan="1" rowspan="1">Chr18</td>
<td colspan="1" rowspan="1">1</td>
<td colspan="1" rowspan="1">23,429,025</td>
<td colspan="1" rowspan="1">0</td>
<td colspan="1" rowspan="1">380</td>
<td colspan="1" rowspan="1">4,462</td>
<td colspan="1" rowspan="1">23,428,592</td>
<td colspan="1" rowspan="1">23,428,950</td>
<td colspan="1" rowspan="1">118,946</td>
<td colspan="1" rowspan="1">609,424</td>
</tr>
<tr>
<td colspan="1" rowspan="1">Chr19</td>
<td colspan="1" rowspan="1">1</td>
<td colspan="1" rowspan="1">22,557,243</td>
<td colspan="1" rowspan="1">0</td>
<td colspan="1" rowspan="1">558</td>
<td colspan="1" rowspan="1">3,151</td>
<td colspan="1" rowspan="1">22,550,858</td>
<td colspan="1" rowspan="1">22,557,068</td>
<td colspan="1" rowspan="1">1,711,822</td>
<td colspan="1" rowspan="1">1,903,333</td>
</tr>
<tr>
<td colspan="1" rowspan="1">Chr20</td>
<td colspan="1" rowspan="1">1</td>
<td colspan="1" rowspan="1">21,388,025</td>
<td colspan="1" rowspan="1">0</td>
<td colspan="1" rowspan="1">110</td>
<td colspan="1" rowspan="1">5,748</td>
<td colspan="1" rowspan="1">21,330,373</td>
<td colspan="1" rowspan="1">21,388,021</td>
<td colspan="1" rowspan="1">20,427,089</td>
<td colspan="1" rowspan="1">21,220,047</td>
</tr>
<tr>
<td colspan="1" rowspan="1">Chr21</td>
<td colspan="1" rowspan="1">1</td>
<td colspan="1" rowspan="1">21,383,011</td>
<td colspan="1" rowspan="1">0</td>
<td colspan="1" rowspan="1">29</td>
<td colspan="1" rowspan="1">5,745</td>
<td colspan="1" rowspan="1">21,378,941</td>
<td colspan="1" rowspan="1">21,382,968</td>
<td colspan="1" rowspan="1">20,010,214</td>
<td colspan="1" rowspan="1">20,126,177</td>
</tr>
<tr>
<td colspan="1" rowspan="1">Chr22</td>
<td colspan="1" rowspan="1">1</td>
<td colspan="1" rowspan="1">19,598,755</td>
<td colspan="1" rowspan="1">0</td>
<td colspan="1" rowspan="1">4</td>
<td colspan="1" rowspan="1">3,098</td>
<td colspan="1" rowspan="1">19,593,794</td>
<td colspan="1" rowspan="1">19,598,751</td>
<td colspan="1" rowspan="1">194,577</td>
<td colspan="1" rowspan="1">544,417</td>
</tr>
<tr>
<td colspan="1" rowspan="1">Chr23</td>
<td colspan="1" rowspan="1">1</td>
<td colspan="1" rowspan="1">19,288,435</td>
<td colspan="1" rowspan="1">0</td>
<td colspan="1" rowspan="1">148</td>
<td colspan="1" rowspan="1">3,373</td>
<td colspan="1" rowspan="1">19,285,242</td>
<td colspan="1" rowspan="1">19,288,431</td>
<td colspan="1" rowspan="1">17,836,318</td>
<td colspan="1" rowspan="1">17,972,547</td>
</tr>
<tr>
<td colspan="1" rowspan="1">Chr24</td>
<td colspan="1" rowspan="1">1</td>
<td colspan="1" rowspan="1">14,856,597</td>
<td colspan="1" rowspan="1">0</td>
<td colspan="1" rowspan="1">456</td>
<td colspan="1" rowspan="1">28,012</td>
<td colspan="1" rowspan="1">14,852,448</td>
<td colspan="1" rowspan="1">14,856,551</td>
<td colspan="1" rowspan="1">405,579</td>
<td colspan="1" rowspan="1">524,259</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><figure class="fig xbox font-sm" id="Fig3"><h4 class="obj_head">Fig. 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371012_41597_2025_5735_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/fd2c/12371012/fa7dc4e76ecd/41597_2025_5735_Fig3_HTML.jpg" loading="lazy" id="d33e1321" height="712" width="760" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Genome-wide localization of repetitive elements (REs), telomeres and centromeres. The triangles at both ends of each chromosome represent the telomere regions, and the gully area within each chromosome stands for the centromere region.</p></figcaption></figure></section><section id="Sec8"><h3 class="pmc_sec_title">Annotation of repeat elements</h3>
<p id="Par18">For prediction of repetitive elements (REs), tandem repeats were first annotated using TRF v4.0.4<sup><a href="#CR34" class="usa-link" aria-describedby="CR34">34</a></sup> and GMATA v2.2<sup><a href="#CR36" class="usa-link" aria-describedby="CR36">36</a></sup>. TRF was employed to identify simple sequence repeats (SSRs), whereas GMATA was used to recognize all tandem REs across the entire genome.</p>
<p id="Par19">Transposable elements (TEs) in the assembled genome were predicted using a combination of homology-based and <em>de novo</em> methods. For the homology approach, TEs were identified using RepeatMasker v4.0.6 and RepeatProteinMask v4.0.6<sup><a href="#CR35" class="usa-link" aria-describedby="CR35">35</a></sup>. For the <em>de novo</em> approach, RepeatModeler v1.0.8<sup><a href="#CR37" class="usa-link" aria-describedby="CR37">37</a></sup> and LTR_FINDER v1.0.6<sup><a href="#CR38" class="usa-link" aria-describedby="CR38">38</a></sup> were employed to generate a <em>de novo</em> repeat library, and RepeatMasker was applied to annotate REs against this repeat library. The annotation results of all repetitive sequences were merged into a comprehensive dataset. This comprehensive annotation revealed 111.64 Mb of repetitive sequences, which account for 18.18% of the assembled Asian seabass genome (Fig. <a href="#Fig3" class="usa-link">3</a>). The most abundant repetitive element was DNA transposons at 9.00% (55.26 Mb), followed by long interspersed nuclear elements (LINEs) at 2.89% (17.76 Mb) and long terminal repeats (LTRs) at 2.46% (15.07 Mb) (see Table <a href="#Tab4" class="usa-link">4</a>).</p>
<section class="tw xbox font-sm" id="Tab4"><h4 class="obj_head">Table 4.</h4>
<div class="caption p"><p>Classification of repetitive sequences in Asian seabass genome.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th colspan="3" rowspan="1">Type</th>
<th colspan="1" rowspan="1">Length (bp)</th>
<th colspan="1" rowspan="1">Count</th>
<th colspan="1" rowspan="1">% of Genome</th>
</tr></thead>
<tbody>
<tr>
<td rowspan="5" colspan="1">Dispersed repeats</td>
<td colspan="2" rowspan="1">DNA transposons</td>
<td colspan="1" rowspan="1">55,263,108</td>
<td colspan="1" rowspan="1">477,149</td>
<td colspan="1" rowspan="1">9.00</td>
</tr>
<tr>
<td rowspan="3" colspan="1">Retroelements</td>
<td colspan="1" rowspan="1">LINE</td>
<td colspan="1" rowspan="1">17,763,761</td>
<td colspan="1" rowspan="1">124,264</td>
<td colspan="1" rowspan="1">2.89</td>
</tr>
<tr>
<td colspan="1" rowspan="1">LTR</td>
<td colspan="1" rowspan="1">15,078,735</td>
<td colspan="1" rowspan="1">141,007</td>
<td colspan="1" rowspan="1">2.46</td>
</tr>
<tr>
<td colspan="1" rowspan="1">SINE</td>
<td colspan="1" rowspan="1">2,414,481</td>
<td colspan="1" rowspan="1">20,229</td>
<td colspan="1" rowspan="1">0.39</td>
</tr>
<tr>
<td colspan="2" rowspan="1">Unclassified</td>
<td colspan="1" rowspan="1">3,985,113</td>
<td colspan="1" rowspan="1">26,905</td>
<td colspan="1" rowspan="1">0.65</td>
</tr>
<tr>
<td rowspan="2" colspan="1">Tandem Repeats</td>
<td colspan="2" rowspan="1">Simple repeats</td>
<td colspan="1" rowspan="1">1,766,456</td>
<td colspan="1" rowspan="1">149,486</td>
<td colspan="1" rowspan="1">0.29</td>
</tr>
<tr>
<td colspan="2" rowspan="1">Satellites</td>
<td colspan="1" rowspan="1">3,174,653</td>
<td colspan="1" rowspan="1">50,007</td>
<td colspan="1" rowspan="1">0.52</td>
</tr>
<tr>
<td colspan="3" rowspan="1">Unknown</td>
<td colspan="1" rowspan="1">12,194,948</td>
<td colspan="1" rowspan="1">95,844</td>
<td colspan="1" rowspan="1">1.98</td>
</tr>
<tr>
<td colspan="3" rowspan="1">Total</td>
<td colspan="1" rowspan="1">111,641,255</td>
<td colspan="1" rowspan="1">10,848,891</td>
<td colspan="1" rowspan="1">18.18</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section></section><section id="Sec9"><h3 class="pmc_sec_title">Prediction and functional annotation of protein-coding genes</h3>
<p id="Par20">Repetitive regions of the assembled genome were masked prior to prediction of genes and their structures. Protein-coding genes was annotated by combination of three methods, including <em>de novo</em>, homology and RNA-seq-based annotations. First, AUGUSTUS v3.2.1<sup><a href="#CR39" class="usa-link" aria-describedby="CR39">39</a></sup> and GlimmerHMM v3.0.4<sup><a href="#CR40" class="usa-link" aria-describedby="CR40">40</a></sup> were employed to perform the <em>ab inito</em> gene structure prediction. Second, GeMoMa v1.6.4<sup><a href="#CR41" class="usa-link" aria-describedby="CR41">41</a></sup> was applied for the homology-based prediction. We aligned homology proteins from five representative fish species, including <em>Epinephelus fuscoguttatus</em> (brown-marbled grouper, GCA_011397635.1), <em>Epinephelus moara</em> (kelp grouper, GCA_006386435.1), <em>Lates japonicus</em> (Japanese lates, GCA_033238685.1), <em>Perca flavescens</em> (yellow Perch, GCA_004354835.1) and <em>Sebastes umbrosus</em> (Honeycomb rockfish, GCA_015220745.1) downloaded from the NCBI. Third, the RNA-seq data from seven tissues were assembled into contigs using Trinity v2.5.1<sup><a href="#CR42" class="usa-link" aria-describedby="CR42">42</a></sup>, and then gene structures were identified using PASA v2.3.3<sup><a href="#CR43" class="usa-link" aria-describedby="CR43">43</a></sup>. Finally, gene sets were integrated by the Evidence Modeler (EVM) pipeline v1.0<sup><a href="#CR44" class="usa-link" aria-describedby="CR44">44</a></sup>.</p>
<p id="Par21">A total of 25,093 protein-coding genes were annotated, with an average gene length of 13.81 kb and an average coding sequence (CDS) length of 1,721.49 bp (Table <a href="#Tab5" class="usa-link">5</a>). Protein-coding genes were evaluated using BUSCO with the actinopterygii_odb10 database as the reference. More than 98.8% of complete BUSCOs were identified within the predicted protein-coding genes.</p>
<section class="tw xbox font-sm" id="Tab5"><h4 class="obj_head">Table 5.</h4>
<div class="caption p"><p>Summary of the predicted gene structures using three methods.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead>
<tr>
<th rowspan="2" colspan="1">Method</th>
<th rowspan="2" colspan="1">Software/Species</th>
<th rowspan="2" colspan="1">Number</th>
<th colspan="4" rowspan="1">Average length (bp)</th>
<th rowspan="2" colspan="1">Average exon per gene</th>
</tr>
<tr>
<th colspan="1" rowspan="1">gene</th>
<th colspan="1" rowspan="1">CDS</th>
<th colspan="1" rowspan="1">exon</th>
<th colspan="1" rowspan="1">intron</th>
</tr>
</thead>
<tbody>
<tr>
<td rowspan="2" colspan="1"><strong><em>De novo</em></strong></td>
<td colspan="1" rowspan="1">Augustus</td>
<td colspan="1" rowspan="1">24,459</td>
<td colspan="1" rowspan="1">14,516.87</td>
<td colspan="1" rowspan="1">1,741.71</td>
<td colspan="1" rowspan="1">159.82</td>
<td colspan="1" rowspan="1">1,290.65</td>
<td colspan="1" rowspan="1">10.9</td>
</tr>
<tr>
<td colspan="1" rowspan="1">Glimmer</td>
<td colspan="1" rowspan="1">39,727</td>
<td colspan="1" rowspan="1">14,042.91</td>
<td colspan="1" rowspan="1">1,020.96</td>
<td colspan="1" rowspan="1">161.29</td>
<td colspan="1" rowspan="1">2,443.21</td>
<td colspan="1" rowspan="1">6.33</td>
</tr>
<tr>
<td rowspan="5" colspan="1"><strong>Homolog</strong></td>
<td colspan="1" rowspan="1"><em>E. moara</em></td>
<td colspan="1" rowspan="1">50,987</td>
<td colspan="1" rowspan="1">23,273.19</td>
<td colspan="1" rowspan="1">1,658.37</td>
<td colspan="1" rowspan="1">180.14</td>
<td colspan="1" rowspan="1">2,634.1</td>
<td colspan="1" rowspan="1">9.21</td>
</tr>
<tr>
<td colspan="1" rowspan="1"><em>L. japonicus</em></td>
<td colspan="1" rowspan="1">51,010</td>
<td colspan="1" rowspan="1">19,380.37</td>
<td colspan="1" rowspan="1">1,656.83</td>
<td colspan="1" rowspan="1">179.59</td>
<td colspan="1" rowspan="1">2,154.73</td>
<td colspan="1" rowspan="1">9.23</td>
</tr>
<tr>
<td colspan="1" rowspan="1"><em>E. fuscoguttatus</em></td>
<td colspan="1" rowspan="1">52,748</td>
<td colspan="1" rowspan="1">23,530.9</td>
<td colspan="1" rowspan="1">1,656.82</td>
<td colspan="1" rowspan="1">180.53</td>
<td colspan="1" rowspan="1">2,674.84</td>
<td colspan="1" rowspan="1">9.18</td>
</tr>
<tr>
<td colspan="1" rowspan="1"><em>P. flavescens</em></td>
<td colspan="1" rowspan="1">53,847</td>
<td colspan="1" rowspan="1">24,587.81</td>
<td colspan="1" rowspan="1">1,638.19</td>
<td colspan="1" rowspan="1">181.46</td>
<td colspan="1" rowspan="1">2,858.73</td>
<td colspan="1" rowspan="1">9.03</td>
</tr>
<tr>
<td colspan="1" rowspan="1"><em>S. umbrosus</em></td>
<td colspan="1" rowspan="1">53,366</td>
<td colspan="1" rowspan="1">24,022.61</td>
<td colspan="1" rowspan="1">1,673.48</td>
<td colspan="1" rowspan="1">185.4</td>
<td colspan="1" rowspan="1">2,784.43</td>
<td colspan="1" rowspan="1">9.03</td>
</tr>
<tr>
<td colspan="1" rowspan="1"><strong>RNA-seq</strong></td>
<td colspan="1" rowspan="1">PASA</td>
<td colspan="1" rowspan="1">21,217</td>
<td colspan="1" rowspan="1">16,814.96</td>
<td colspan="1" rowspan="1">3,699.28</td>
<td colspan="1" rowspan="1">302.27</td>
<td colspan="1" rowspan="1">1,167.04</td>
<td colspan="1" rowspan="1">12.24</td>
</tr>
<tr>
<td colspan="1" rowspan="1"><strong>Integrated</strong></td>
<td colspan="1" rowspan="1">EVM</td>
<td colspan="1" rowspan="1">25,093</td>
<td colspan="1" rowspan="1">13,819.54</td>
<td colspan="1" rowspan="1">1,721.49</td>
<td colspan="1" rowspan="1">168.93</td>
<td colspan="1" rowspan="1">1,316.4</td>
<td colspan="1" rowspan="1">10.19</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><p id="Par22">Functional annotation of the protein-coding genes was performed using Blastp v2.2.26<sup><a href="#CR45" class="usa-link" aria-describedby="CR45">45</a></sup>, which aligned deduced protein sequences against five public databases including NCBI Non-Redundant Protein Sequence (NR), SwissProt<sup><a href="#CR46" class="usa-link" aria-describedby="CR46">46</a></sup>, Gene Ontology (GO)<sup><a href="#CR47" class="usa-link" aria-describedby="CR47">47</a></sup>, Kyoto Encyclopedia of Genes and Genomes (KEGG)<sup><a href="#CR48" class="usa-link" aria-describedby="CR48">48</a></sup> and EuKaryotic Orthologous Groups (KOG)<sup><a href="#CR49" class="usa-link" aria-describedby="CR49">49</a></sup>, with an E-value cutoff of &lt;1e−5. Ultimately, 23,711 protein-coding genes (94.49% of the total predicted genes) were functionally annotated, with at least one hit for each gene in the searched databases (Table <a href="#Tab6" class="usa-link">6</a>).</p>
<section class="tw xbox font-sm" id="Tab6"><h4 class="obj_head">Table 6.</h4>
<div class="caption p"><p>Functional annotation of predicted protein-coding genes.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th colspan="1" rowspan="1">Database</th>
<th colspan="1" rowspan="1">Number</th>
<th colspan="1" rowspan="1">Percentage (%)</th>
</tr></thead>
<tbody>
<tr>
<td colspan="1" rowspan="1"><strong>Total</strong></td>
<td colspan="1" rowspan="1">25,093</td>
<td colspan="1" rowspan="1">100</td>
</tr>
<tr>
<td colspan="1" rowspan="1"><strong>NR</strong></td>
<td colspan="1" rowspan="1">23,699</td>
<td colspan="1" rowspan="1">94.44</td>
</tr>
<tr>
<td colspan="1" rowspan="1"><strong>Swissprot</strong></td>
<td colspan="1" rowspan="1">21,269</td>
<td colspan="1" rowspan="1">84.76</td>
</tr>
<tr>
<td colspan="1" rowspan="1"><strong>KEGG</strong></td>
<td colspan="1" rowspan="1">16,838</td>
<td colspan="1" rowspan="1">67.10</td>
</tr>
<tr>
<td colspan="1" rowspan="1"><strong>GO</strong></td>
<td colspan="1" rowspan="1">16,260</td>
<td colspan="1" rowspan="1">64.80</td>
</tr>
<tr>
<td colspan="1" rowspan="1"><strong>KOG</strong></td>
<td colspan="1" rowspan="1">15,510</td>
<td colspan="1" rowspan="1">61.81</td>
</tr>
<tr>
<td colspan="1" rowspan="1"><strong>Overall</strong></td>
<td colspan="1" rowspan="1">23,711</td>
<td colspan="1" rowspan="1">94.49</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="_fn_p34"><p>Overall represents the total number of annotated genes with at least one hit from the five searched databases.</p></div></div></section></section></section><section id="Sec10"><h2 class="pmc_sec_title">Data Records</h2>
<p id="Par23">Files of the MGI, PacBio, ONT, Hi-C and transcriptome sequencing, and the assembled genome for Asian seabass were deposited at NCBI under the accession number PRJNA1245135. Raw reads are available in the Sequence Reads Archive (SRA) with the accession numbers SRR32997291 to SRR32997305<sup><a href="#CR50" class="usa-link" aria-describedby="CR50">50</a></sup>. The genome assembly, predicted coding sequences and function annotation files of Asian seabass were stored in Figshare (No: m9.figshare.28735226)<sup><a href="#CR51" class="usa-link" aria-describedby="CR51">51</a></sup>. The genome assembly has also been deposited at the NCBl/GenBank under the accession number of GCA_051027255.1<sup><a href="#CR52" class="usa-link" aria-describedby="CR52">52</a></sup>.</p></section><section id="Sec11"><h2 class="pmc_sec_title">Technical Validation</h2>
<p id="Par24">To evaluate the quality of our genome assembly, we employed four approaches. First, BUSCO v5.2.2<sup><a href="#CR53" class="usa-link" aria-describedby="CR53">53</a></sup> was employed to examine completeness. A total of 100% (single copy complete genes (S): 99.84%, duplicated complete genes (D): 0.16%) of complete BUSCOs in the actinopterygii_odb10 database were identified. Second, Merqury v1.328<sup><a href="#CR54" class="usa-link" aria-describedby="CR54">54</a></sup> was applied to estimate the base-level accuracy and completeness on the basis of k-mer counts (generated from Illumina and PacBio HiFi reads), resulting in a QV of 40.59 and 57.80 respectively. Third, Clipping information for Revealing Assembly Quality (CRAQ, v1.09)<sup><a href="#CR55" class="usa-link" aria-describedby="CR55">55</a></sup> was used to assess the accuracy of our genome assembly based on PacBio HiFi and Illumina reads, resulting in a R-AQI (assembly quality indicator) of 98.42 and a S-AQI of 99.45. Fourth, we mapped the sequencing data to the assembled genome using bwa v0.7.17<sup><a href="#CR56" class="usa-link" aria-describedby="CR56">56</a></sup> and minimap2 v2.26<sup><a href="#CR57" class="usa-link" aria-describedby="CR57">57</a></sup>, which showed mapping rates of 99.46% for the MGI data, 99.99% for the PacBio data, and 98.43% for the ONT data. These results collectively support high quality of the Asian seabass genome assembly. The BUSCO completeness value was calculated to be 98.8% for the predicted protein-coding genes of Asian seabass (Table <a href="#Tab7" class="usa-link">7</a>). To further evaluate the quality of these predicted protein-coding genes, we aligned the transcriptome data to the assembled genome using STAR v 2.7.11b<sup><a href="#CR58" class="usa-link" aria-describedby="CR58">58</a></sup>, and then calculated the exonic coverage rate with bedtools v2.29.2<sup><a href="#CR59" class="usa-link" aria-describedby="CR59">59</a></sup>. We observed that 94.71% of the exonic regions had been covered with sequencing reads, indicating high annotation accuracy (see Table <a href="#Tab7" class="usa-link">7</a>).</p>
<section class="tw xbox font-sm" id="Tab7"><h3 class="obj_head">Table 7.</h3>
<div class="caption p"><p>Assessment metrics of the genome assembly and annotation.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th colspan="1" rowspan="1">Type</th>
<th colspan="2" rowspan="1">Evaluation Methods</th>
<th colspan="1" rowspan="1">Results</th>
</tr></thead>
<tbody>
<tr>
<td rowspan="8" colspan="1"><strong>Genome accuracy and completeness</strong></td>
<td colspan="2" rowspan="1">Mapping short reads rate</td>
<td colspan="1" rowspan="1">99.46%</td>
</tr>
<tr>
<td colspan="2" rowspan="1">Mapping HiFi reads rate</td>
<td colspan="1" rowspan="1">99.99%</td>
</tr>
<tr>
<td colspan="2" rowspan="1">Mapping ONT reads rate</td>
<td colspan="1" rowspan="1">98.43%</td>
</tr>
<tr>
<td rowspan="2" colspan="1">QV</td>
<td colspan="1" rowspan="1">Short reads</td>
<td colspan="1" rowspan="1">40.59</td>
</tr>
<tr>
<td colspan="1" rowspan="1">HiFi reads</td>
<td colspan="1" rowspan="1">57.80</td>
</tr>
<tr>
<td rowspan="2" colspan="1">CRAQ</td>
<td colspan="1" rowspan="1">R-AQI</td>
<td colspan="1" rowspan="1">98.42%</td>
</tr>
<tr>
<td colspan="1" rowspan="1">S-AQI</td>
<td colspan="1" rowspan="1">99.45%</td>
</tr>
<tr>
<td colspan="2" rowspan="1">BUSCO</td>
<td colspan="1" rowspan="1">100%</td>
</tr>
<tr>
<td rowspan="6" colspan="1"><strong>Annotation quality</strong></td>
<td colspan="2" rowspan="1">Complete BUSCOs</td>
<td colspan="1" rowspan="1">98.8% (3,599)</td>
</tr>
<tr>
<td colspan="2" rowspan="1">Complete and single-copy BUSCOs (S)</td>
<td colspan="1" rowspan="1">98.2% (3,576)</td>
</tr>
<tr>
<td colspan="2" rowspan="1">Complete and duplicated BUSCOs (D)</td>
<td colspan="1" rowspan="1">0.6% (23)</td>
</tr>
<tr>
<td colspan="2" rowspan="1">Fragmented BUSCOs (F)</td>
<td colspan="1" rowspan="1">0 (0)</td>
</tr>
<tr>
<td colspan="2" rowspan="1">Missing BUSCOs (M)</td>
<td colspan="1" rowspan="1">1.2% (41)</td>
</tr>
<tr>
<td colspan="2" rowspan="1">RNA-seq coverage ratio of the exonic regions</td>
<td colspan="1" rowspan="1">94.71%</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab7/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>This work was supported by Shenzhen Natural Science Foundation (no. JCYJ20241202124511016) and National Key Research and Development Program of China (no. 2022YFE0139700).</p></section><section id="notes1"><h2 class="pmc_sec_title">Author contributions</h2>
<p>Q.S. conceived and designed the study. X.Z., J.W. and J.C. collected the samples. X.Z., J.C. and J.W. performed data analysis. J.W. and W.Z. conducted experiments for species identification. X.Z. and J.W. wrote the manuscript. Q.S. revised the manuscript. All authors read and approved the final manuscript for publication.</p></section><section id="notes2"><h2 class="pmc_sec_title">Code availability</h2>
<p>The versions and parameters of bioinformatics tools applied in this study have been described in the Method section. If no parameter is provided, the default is set. No custom code was used.</p></section><section id="FPar1"><h2 class="pmc_sec_title">Competing interests</h2>
<p id="Par25">The authors declare no competing interests.</p></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="fn1"><p><strong>Publisher’s note</strong> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></div></div></section><section id="_ci93_" lang="en" class="contrib-info"><h2 class="pmc_sec_title">Contributor Information</h2>
<p>Jiufu Wen, Email: nhswjf@163.com.</p>
<p>Qiong Shi, Email: shiqiong@szu.edu.cn, Email: shiqiong@genomics.cn.</p></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Gamble, T. <em>et al</em>. Sex determination. <em>Current Biology</em><strong>22</strong>(8), 257–262 (2012).</cite> [<a href="https://doi.org/10.1016/j.cub.2012.02.054" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11003734/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22537624/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gamble,%20T.%20et%20al.%20Sex%20determination.%20Current%20Biology22(8),%20257%E2%80%93262%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>Penman, D. J. <em>et al</em>. Fish gonadogenesis. Part I: genetic and environmental mechanisms of sex determination. <em>Reviews in Fisheries Science</em><strong>16</strong>(sup1), 16–34 (2008).</cite> [<a href="https://scholar.google.com/scholar_lookup?Penman,%20D.%20J.%20et%20al.%20Fish%20gonadogenesis.%20Part%20I:%20genetic%20and%20environmental%20mechanisms%20of%20sex%20determination.%20Reviews%20in%20Fisheries%20Science16(sup1),%2016%E2%80%9334%20(2008)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>Devlin, R. H. <em>et al</em>. Sex determination and sex differentiation in fish: an overview of genetic, physiological, and environmental influences. <em>Aquaculture</em><strong>208</strong>(3-4), 191–364 (2002).</cite> [<a href="https://scholar.google.com/scholar_lookup?Devlin,%20R.%20H.%20et%20al.%20Sex%20determination%20and%20sex%20differentiation%20in%20fish:%20an%20overview%20of%20genetic,%20physiological,%20and%20environmental%20influences.%20Aquaculture208(3-4),%20191%E2%80%93364%20(2002)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Piferrer, F. <em>et al</em>. Genetic, endocrine, and environmental components of sex determination and differentiation in the European sea bass (<em>Dicentrarchus labrax L</em>.). <em>General and comparative endocrinology</em><strong>142</strong>(1-2), 102–110 (2005).
</cite> [<a href="https://doi.org/10.1016/j.ygcen.2005.02.011" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15862554/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Piferrer,%20F.%20et%20al.%20Genetic,%20endocrine,%20and%20environmental%20components%20of%20sex%20determination%20and%20differentiation%20in%20the%20European%20sea%20bass%20(Dicentrarchus%20labrax%20L.).%20General%20and%20comparative%20endocrinology142(1-2),%20102%E2%80%93110%20(2005)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Baroiller, J. F. <em>et al</em>. Tilapia sex determination: where temperature and genetics meet. <em>Comparative Biochemistry and Physiology Part A: Molecular &amp; Integrative Physiology</em><strong>153</strong>(1), 30–38 (2009).</cite> [<a href="https://doi.org/10.1016/j.cbpa.2008.11.018" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19101647/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Baroiller,%20J.%20F.%20et%20al.%20Tilapia%20sex%20determination:%20where%20temperature%20and%20genetics%20meet.%20Comparative%20Biochemistry%20and%20Physiology%20Part%20A:%20Molecular%20&amp;%20Integrative%20Physiology153(1),%2030%E2%80%9338%20(2009)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Palaiokostas, C. <em>et al</em>. Mapping the sex determination locus in the Atlantic halibut (<em>Hippoglossus hippoglossus</em>) using RAD sequencing. <em>BMC genomics</em><strong>14</strong>, 1–12 (2013).
</cite> [<a href="https://doi.org/10.1186/1471-2164-14-566" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3765698/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23957753/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Palaiokostas,%20C.%20et%20al.%20Mapping%20the%20sex%20determination%20locus%20in%20the%20Atlantic%20halibut%20(Hippoglossus%20hippoglossus)%20using%20RAD%20sequencing.%20BMC%20genomics14,%201%E2%80%9312%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Hughes, V. <em>et al</em>. Effect of rearing temperature on sex ratio in juvenile Atlantic halibut, <em>Hippoglossus hippoglossus</em>. <em>Environmental biology of fishes</em><strong>81</strong>, 415–419 (2008).</cite> [<a href="https://scholar.google.com/scholar_lookup?Hughes,%20V.%20et%20al.%20Effect%20of%20rearing%20temperature%20on%20sex%20ratio%20in%20juvenile%20Atlantic%20halibut,%20Hippoglossus%20hippoglossus.%20Environmental%20biology%20of%20fishes81,%20415%E2%80%93419%20(2008)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>Avise, J. C. <em>et al</em>. Evolutionary perspectives on hermaphroditism in fishes. <em>Sexual Development</em><strong>3</strong>(2-3), 152–163 (2009).
</cite> [<a href="https://doi.org/10.1159/000223079" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19684459/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Avise,%20J.%20C.%20et%20al.%20Evolutionary%20perspectives%20on%20hermaphroditism%20in%20fishes.%20Sexual%20Development3(2-3),%20152%E2%80%93163%20(2009)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>Kuwamura, T. <em>et al</em>. Sex change of primary males in a diandric labrid <em>Halichoeres trimaculatus</em>: coexistence of protandry and protogyny within a species. <em>Journal of Fish Biology</em><strong>70</strong>(6), 1898–1906 (2007).</cite> [<a href="https://scholar.google.com/scholar_lookup?Kuwamura,%20T.%20et%20al.%20Sex%20change%20of%20primary%20males%20in%20a%20diandric%20labrid%20Halichoeres%20trimaculatus:%20coexistence%20of%20protandry%20and%20protogyny%20within%20a%20species.%20Journal%20of%20Fish%20Biology70(6),%201898%E2%80%931906%20(2007)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>Li, S. <em>et al</em>. Mechanisms of sex differentiation and sex reversal in hermaphrodite fish as revealed by the <em>Epinephelus coioides</em> genome. <em>Molecular Ecology Resources</em><strong>23</strong>(4), 920–932 (2023).
</cite> [<a href="https://doi.org/10.1111/1755-0998.13753" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36631404/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li,%20S.%20et%20al.%20Mechanisms%20of%20sex%20differentiation%20and%20sex%20reversal%20in%20hermaphrodite%20fish%20as%20revealed%20by%20the%20Epinephelus%20coioides%20genome.%20Molecular%20Ecology%20Resources23(4),%20920%E2%80%93932%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Zhang, K. <em>et al</em>. A telomere-to-telomere genome assembly of the protandrous hermaphrodite blackhead seabream, <em>Acanthopagrus schlegelii</em>. <em>Scientific Data</em><strong>12</strong>(1), 350 (2025).
</cite> [<a href="https://doi.org/10.1038/s41597-025-04602-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11868651/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40016269/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhang,%20K.%20et%20al.%20A%20telomere-to-telomere%20genome%20assembly%20of%20the%20protandrous%20hermaphrodite%20blackhead%20seabream,%20Acanthopagrus%20schlegelii.%20Scientific%20Data12(1),%20350%20(2025)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Casas, L. <em>et al</em>. Sex change in clownfish: molecular insights from transcriptome analysis. <em>Scientific Reports</em><strong>6</strong>(1), 35461 (2016).
</cite> [<a href="https://doi.org/10.1038/srep35461" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5066260/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27748421/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Casas,%20L.%20et%20al.%20Sex%20change%20in%20clownfish:%20molecular%20insights%20from%20transcriptome%20analysis.%20Scientific%20Reports6(1),%2035461%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>Cheng, H. <em>et al</em>. The rice field eel as a model system for vertebrate sexual development. <em>Cytogenetic and Genome Research</em><strong>101</strong>(3-4), 274–277 (2003).
</cite> [<a href="https://doi.org/10.1159/000074348" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/14684994/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cheng,%20H.%20et%20al.%20The%20rice%20field%20eel%20as%20a%20model%20system%20for%20vertebrate%20sexual%20development.%20Cytogenetic%20and%20Genome%20Research101(3-4),%20274%E2%80%93277%20(2003)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>Yue, G. H. <em>et al</em>. Genomic resources and their applications in aquaculture of Asian seabass (<em>Lates calcarifer</em>). <em>Reviews in Aquaculture</em><strong>15</strong>(2), 853–871 (2023).</cite> [<a href="https://scholar.google.com/scholar_lookup?Yue,%20G.%20H.%20et%20al.%20Genomic%20resources%20and%20their%20applications%20in%20aquaculture%20of%20Asian%20seabass%20(Lates%20calcarifer).%20Reviews%20in%20Aquaculture15(2),%20853%E2%80%93871%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>Athauda, S. <em>et al</em>. Effect of rearing water temperature on protandrous sex inversion in cultured Asian Seabass (<em>Lates calcarifer</em>). <em>General and Comparative Endocrinology</em><strong>175</strong>(3), 416–423 (2012).
</cite> [<a href="https://doi.org/10.1016/j.ygcen.2011.11.040" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22155035/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Athauda,%20S.%20et%20al.%20Effect%20of%20rearing%20water%20temperature%20on%20protandrous%20sex%20inversion%20in%20cultured%20Asian%20Seabass%20(Lates%20calcarifer).%20General%20and%20Comparative%20Endocrinology175(3),%20416%E2%80%93423%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Jerry, D. R. Biology and culture of Asian seabass <em>Lates calcarifer</em>. <em>CRC Press</em> (2013).</cite>
</li>
<li id="CR17">
<span class="label">17.</span><cite>Vij, S. <em>et al</em>. Chromosomal-level assembly of the Asian seabass genome using long sequence reads and multi-layered scaffolding. <em>PLoS Genetics</em><strong>12</strong>(4), e1005954 (2016).
</cite> [<a href="https://doi.org/10.1371/journal.pgen.1005954" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4833346/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27082250/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Vij,%20S.%20et%20al.%20Chromosomal-level%20assembly%20of%20the%20Asian%20seabass%20genome%20using%20long%20sequence%20reads%20and%20multi-layered%20scaffolding.%20PLoS%20Genetics12(4),%20e1005954%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Mei, L. <em>et al</em>. Evaluation of QIAamp® DNA Stool Mini Kit for ecological studies of gut microbiota. <em>Journal of Microbiological Methods</em><strong>54</strong>(1), 13–20 (2003).
</cite> [<a href="https://doi.org/10.1016/s0167-7012(02)00260-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12732417/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mei,%20L.%20et%20al.%20Evaluation%20of%20QIAamp%C2%AE%20DNA%20Stool%20Mini%20Kit%20for%20ecological%20studies%20of%20gut%20microbiota.%20Journal%20of%20Microbiological%20Methods54(1),%2013%E2%80%9320%20(2003)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Chen S. <em>et al</em>. fastp: an ultra-fast all-in-one FASTQ preprocessor. <em>Bioinformatics</em><strong>34</strong>(17), i884–i890.</cite> [<a href="https://doi.org/10.1093/bioinformatics/bty560" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6129281/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30423086/" class="usa-link">PubMed</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>Rhoads, A. <em>et al</em>. PacBio Sequencing and Its Applications. <em>Genomics Proteomics &amp; Bioinformatics</em><strong>13</strong>, 278–289 (2015).</cite> [<a href="https://doi.org/10.1016/j.gpb.2015.08.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4678779/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26542840/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rhoads,%20A.%20et%20al.%20PacBio%20Sequencing%20and%20Its%20Applications.%20Genomics%20Proteomics%20&amp;%20Bioinformatics13,%20278%E2%80%93289%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>De Coster, W. <em>et al</em>. NanoPack: visualizing and processing long-read sequencing data. <em>Bioinformatics</em><strong>34</strong>(15), 2666–2669 (2018).
</cite> [<a href="https://doi.org/10.1093/bioinformatics/bty149" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6061794/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29547981/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?De%20Coster,%20W.%20et%20al.%20NanoPack:%20visualizing%20and%20processing%20long-read%20sequencing%20data.%20Bioinformatics34(15),%202666%E2%80%932669%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>Marçais, G. <em>et al</em>. A fast, lock-free approach for efficient parallel counting of occurrences of k-mers. <em>Bioinformatics</em><strong>27</strong>(6), 764–770 (2011).
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btr011" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3051319/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21217122/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mar%C3%A7ais,%20G.%20et%20al.%20A%20fast,%20lock-free%20approach%20for%20efficient%20parallel%20counting%20of%20occurrences%20of%20k-mers.%20Bioinformatics27(6),%20764%E2%80%93770%20(2011)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>Vurture, G. W. <em>et al</em>. GenomeScope: fast reference-free genome profiling from short reads. <em>Bioinformatics</em><strong>33</strong>(14), 2202–2204 (2017).
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btx153" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5870704/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28369201/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Vurture,%20G.%20W.%20et%20al.%20GenomeScope:%20fast%20reference-free%20genome%20profiling%20from%20short%20reads.%20Bioinformatics33(14),%202202%E2%80%932204%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>Cheng, H. <em>et al</em>. Haplotype-resolved de novo assembly using phased assembly graphs with hifiasm. <em>Nature Methods</em><strong>18</strong>, 170–175 (2021).
</cite> [<a href="https://doi.org/10.1038/s41592-020-01056-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7961889/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33526886/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cheng,%20H.%20et%20al.%20Haplotype-resolved%20de%20novo%20assembly%20using%20phased%20assembly%20graphs%20with%20hifiasm.%20Nature%20Methods18,%20170%E2%80%93175%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>Roach, M. J. <em>et al</em>. Purge Haplotigs: allelic contig reassignment for third-gen diploid genome assemblies. <em>BMC Bioinformatics</em><strong>19</strong>, 1–10 (2018).
</cite> [<a href="https://doi.org/10.1186/s12859-018-2485-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6267036/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30497373/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Roach,%20M.%20J.%20et%20al.%20Purge%20Haplotigs:%20allelic%20contig%20reassignment%20for%20third-gen%20diploid%20genome%20assemblies.%20BMC%20Bioinformatics19,%201%E2%80%9310%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>Langmead, B. <em>et al</em>. Fast gapped-read alignment with Bowtie 2. <em>Nature Methods</em><strong>9</strong>(4), 357–359 (2012).
</cite> [<a href="https://doi.org/10.1038/nmeth.1923" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3322381/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22388286/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Langmead,%20B.%20et%20al.%20Fast%20gapped-read%20alignment%20with%20Bowtie%202.%20Nature%20Methods9(4),%20357%E2%80%93359%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Dekker, J. <em>et al</em>. HiC-Pro: an optimized and flexible pipeline for Hi-C data processing. <em>Genome Biology</em><strong>16</strong>, 259 (2015).
</cite> [<a href="https://doi.org/10.1186/s13059-015-0831-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4665391/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26619908/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Dekker,%20J.%20et%20al.%20HiC-Pro:%20an%20optimized%20and%20flexible%20pipeline%20for%20Hi-C%20data%20processing.%20Genome%20Biology16,%20259%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Durand, N. C. <em>et al</em>. Juicer provides a one-click system for analyzing loop-resolution Hi-C experiments. <em>Cell Systems</em><strong>3</strong>(1), 95–98 (2016).
</cite> [<a href="https://doi.org/10.1016/j.cels.2016.07.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5846465/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27467249/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Durand,%20N.%20C.%20et%20al.%20Juicer%20provides%20a%20one-click%20system%20for%20analyzing%20loop-resolution%20Hi-C%20experiments.%20Cell%20Systems3(1),%2095%E2%80%9398%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>Dudchenko, O. <em>et al</em>. De novo assembly of the <em>Aedes aegypti</em> genome using Hi-C yields chromosome-length scaffolds. <em>Science</em><strong>356</strong>, 92–95 (2017).
</cite> [<a href="https://doi.org/10.1126/science.aal3327" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5635820/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28336562/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Dudchenko,%20O.%20et%20al.%20De%20novo%20assembly%20of%20the%20Aedes%20aegypti%20genome%20using%20Hi-C%20yields%20chromosome-length%20scaffolds.%20Science356,%2092%E2%80%9395%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR30">
<span class="label">30.</span><cite>Durand, N. C. <em>et al</em>. Juicebox provides a visualization system for Hi-C contact maps with unlimited zoom. <em>Cell Systems</em><strong>3</strong>, 99–101 (2016).
</cite> [<a href="https://doi.org/10.1016/j.cels.2015.07.012" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5596920/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27467250/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Durand,%20N.%20C.%20et%20al.%20Juicebox%20provides%20a%20visualization%20system%20for%20Hi-C%20contact%20maps%20with%20unlimited%20zoom.%20Cell%20Systems3,%2099%E2%80%93101%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>Xu, M. <em>et al</em>. TGS-GapCloser: a fast and accurate gap closer for large genomes with low coverage of error-prone long reads. <em>GigaScience</em><strong>9</strong>(9), giaa094 (2020).
</cite> [<a href="https://doi.org/10.1093/gigascience/giaa094" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7476103/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32893860/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Xu,%20M.%20et%20al.%20TGS-GapCloser:%20a%20fast%20and%20accurate%20gap%20closer%20for%20large%20genomes%20with%20low%20coverage%20of%20error-prone%20long%20reads.%20GigaScience9(9),%20giaa094%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>Xu, G. C. <em>et al</em>. LR_Gapcloser: a tiling path-based gap closer that uses long reads to complete genome assembly. <em>Gigascience</em><strong>8</strong>(1), giy157 (2019).
</cite> [<a href="https://doi.org/10.1093/gigascience/giy157" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6324547/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30576505/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Xu,%20G.%20C.%20et%20al.%20LR_Gapcloser:%20a%20tiling%20path-based%20gap%20closer%20that%20uses%20long%20reads%20to%20complete%20genome%20assembly.%20Gigascience8(1),%20giy157%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Lin, Y. <em>et al</em>. quarTeT: a telomere-to-telomere toolkit for gap-free genome assembly and centromeric repeat identification. <em>Horticulture Research</em><strong>10</strong>(8), uhad127 (2023).
</cite> [<a href="https://doi.org/10.1093/hr/uhad127" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10407605/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37560017/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lin,%20Y.%20et%20al.%20quarTeT:%20a%20telomere-to-telomere%20toolkit%20for%20gap-free%20genome%20assembly%20and%20centromeric%20repeat%20identification.%20Horticulture%20Research10(8),%20uhad127%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>Benson, G. Tandem repeats finder: a program to analyze DNA sequences. <em>Nucleic Acids Research</em><strong>27</strong>(2), 573–580 (1999).
</cite> [<a href="https://doi.org/10.1093/nar/27.2.573" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC148217/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9862982/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Benson,%20G.%20Tandem%20repeats%20finder:%20a%20program%20to%20analyze%20DNA%20sequences.%20Nucleic%20Acids%20Research27(2),%20573%E2%80%93580%20(1999)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>Tarailo-Graovac, M. <em>et al</em>. Using RepeatMasker to identify repetitive elements in genomic sequences. <em>Current Protocols in Bioinformatics</em> Chapter 4, 4–10 (2009).</cite> [<a href="https://doi.org/10.1002/0471250953.bi0410s25" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19274634/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tarailo-Graovac,%20M.%20et%20al.%20Using%20RepeatMasker%20to%20identify%20repetitive%20elements%20in%20genomic%20sequences.%20Current%20Protocols%20in%20Bioinformatics%20Chapter%204,%204%E2%80%9310%20(2009)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR36">
<span class="label">36.</span><cite>Wang, X. &amp; Wang, L. GMATA: an integrated software package for genome-scale SSR mining, marker development and viewing. <em>Frontiers in Plant Science</em><strong>7</strong>, 1350 (2016).
</cite> [<a href="https://doi.org/10.3389/fpls.2016.01350" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5020087/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27679641/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang,%20X.%20&amp;%20Wang,%20L.%20GMATA:%20an%20integrated%20software%20package%20for%20genome-scale%20SSR%20mining,%20marker%20development%20and%20viewing.%20Frontiers%20in%20Plant%20Science7,%201350%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR37">
<span class="label">37.</span><cite>Flynn, J. M. <em>et al</em>. RepeatModeler2 for automated genomic discovery of transposable element families. <em>Proceedings of the National Academy of Science of the United States of America</em><strong>117</strong>, 9451–9457 (2020).</cite> [<a href="https://doi.org/10.1073/pnas.1921046117" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7196820/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32300014/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Flynn,%20J.%20M.%20et%20al.%20RepeatModeler2%20for%20automated%20genomic%20discovery%20of%20transposable%20element%20families.%20Proceedings%20of%20the%20National%20Academy%20of%20Science%20of%20the%20United%20States%20of%20America117,%209451%E2%80%939457%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR38">
<span class="label">38.</span><cite>Xu, Z. <em>et al</em>. LTR_FINDER: an efficient tool for the prediction of full-length LTR retrotransposons. <em>Nucleic Acids Research</em><strong>35</strong>, W265–268 (2007).
</cite> [<a href="https://doi.org/10.1093/nar/gkm286" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1933203/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17485477/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Xu,%20Z.%20et%20al.%20LTR_FINDER:%20an%20efficient%20tool%20for%20the%20prediction%20of%20full-length%20LTR%20retrotransposons.%20Nucleic%20Acids%20Research35,%20W265%E2%80%93268%20(2007)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR39">
<span class="label">39.</span><cite>Stanke, M. <em>et al</em>. AUGUSTUS: ab initio prediction of alternative transcripts. <em>Nucleic Acids Research</em><strong>34</strong>, W435–439 (2006).
</cite> [<a href="https://doi.org/10.1093/nar/gkl200" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1538822/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16845043/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Stanke,%20M.%20et%20al.%20AUGUSTUS:%20ab%20initio%20prediction%20of%20alternative%20transcripts.%20Nucleic%20Acids%20Research34,%20W435%E2%80%93439%20(2006)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR40">
<span class="label">40.</span><cite>Majoros, W. H. <em>et al</em>. TigrScan and GlimmerHMM: two open source ab initio eukaryotic gene-finders. <em>Bioinformatics</em><strong>20</strong>(16), 2878–2879 (2004).
</cite> [<a href="https://doi.org/10.1093/bioinformatics/bth315" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15145805/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Majoros,%20W.%20H.%20et%20al.%20TigrScan%20and%20GlimmerHMM:%20two%20open%20source%20ab%20initio%20eukaryotic%20gene-finders.%20Bioinformatics20(16),%202878%E2%80%932879%20(2004)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR41">
<span class="label">41.</span><cite>Keilwagen, J. <em>et al</em>. GeMoMa: homology-based gene prediction utilizing intron position conservation and RNA-seq data. <em>Methods in Molecular Biology</em><strong>1962</strong>, 161–177 (2019).
</cite> [<a href="https://doi.org/10.1007/978-1-4939-9173-0_9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31020559/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Keilwagen,%20J.%20et%20al.%20GeMoMa:%20homology-based%20gene%20prediction%20utilizing%20intron%20position%20conservation%20and%20RNA-seq%20data.%20Methods%20in%20Molecular%20Biology1962,%20161%E2%80%93177%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR42">
<span class="label">42.</span><cite>Haas, B. J. <em>et al</em>. De novo transcript sequence reconstruction from RNA-seq using the Trinity platform for reference generation and analysis. <em>Nat. Protoc.</em><strong>8</strong>, 1494–1512 (2013).
</cite> [<a href="https://doi.org/10.1038/nprot.2013.084" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3875132/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23845962/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Haas,%20B.%20J.%20et%20al.%20De%20novo%20transcript%20sequence%20reconstruction%20from%20RNA-seq%20using%20the%20Trinity%20platform%20for%20reference%20generation%20and%20analysis.%20Nat.%20Protoc.8,%201494%E2%80%931512%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR43">
<span class="label">43.</span><cite>Haas, B. J. <em>et al</em>. Improving the Arabidopsis genome annotation using maximal transcript alignment assemblies. <em>Nucleic Acids Research</em><strong>31</strong>(19), 5654–5666 (2003).
</cite> [<a href="https://doi.org/10.1093/nar/gkg770" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC206470/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/14500829/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Haas,%20B.%20J.%20et%20al.%20Improving%20the%20Arabidopsis%20genome%20annotation%20using%20maximal%20transcript%20alignment%20assemblies.%20Nucleic%20Acids%20Research31(19),%205654%E2%80%935666%20(2003)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR44">
<span class="label">44.</span><cite>Haas, B. J. <em>et al</em>. Automated eukaryotic gene structure annotation using EVidenceModeler and the Program to Assemble Spliced Alignments. <em>Genome Biology</em><strong>9</strong>, R7 (2008).
</cite> [<a href="https://doi.org/10.1186/gb-2008-9-1-r7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2395244/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18190707/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Haas,%20B.%20J.%20et%20al.%20Automated%20eukaryotic%20gene%20structure%20annotation%20using%20EVidenceModeler%20and%20the%20Program%20to%20Assemble%20Spliced%20Alignments.%20Genome%20Biology9,%20R7%20(2008)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR45">
<span class="label">45.</span><cite>Altschul, S. F. <em>et al</em>. Basic local alignment search tool. <em>Journal of Molecular Biology</em><strong>215</strong>(3), 403–410 (1990).
</cite> [<a href="https://doi.org/10.1016/S0022-2836(05)80360-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/2231712/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Altschul,%20S.%20F.%20et%20al.%20Basic%20local%20alignment%20search%20tool.%20Journal%20of%20Molecular%20Biology215(3),%20403%E2%80%93410%20(1990)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR46">
<span class="label">46.</span><cite>Bairoch, A. <em>et al</em>. The SWISS-PROT protein sequence database and its supplement TrEMBL in 2000. <em>Nucleic Acids Research</em><strong>28</strong>(1), 45–48 (2000).
</cite> [<a href="https://doi.org/10.1093/nar/28.1.45" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC102476/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10592178/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bairoch,%20A.%20et%20al.%20The%20SWISS-PROT%20protein%20sequence%20database%20and%20its%20supplement%20TrEMBL%20in%202000.%20Nucleic%20Acids%20Research28(1),%2045%E2%80%9348%20(2000)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR47">
<span class="label">47.</span><cite>Ashburner, M. <em>et al</em>. Gene ontology: tool for the unification of biology. <em>Nature genetics</em><strong>25</strong>(1), 25–29 (2000).
</cite> [<a href="https://doi.org/10.1038/75556" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3037419/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10802651/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ashburner,%20M.%20et%20al.%20Gene%20ontology:%20tool%20for%20the%20unification%20of%20biology.%20Nature%20genetics25(1),%2025%E2%80%9329%20(2000)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR48">
<span class="label">48.</span><cite>Kanehisa, M. <em>et al</em>. KEGG as a reference resource for gene and protein annotation. <em>Nucleic acids research</em><strong>44</strong>(D1), D457–D462 (2016).
</cite> [<a href="https://doi.org/10.1093/nar/gkv1070" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4702792/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26476454/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kanehisa,%20M.%20et%20al.%20KEGG%20as%20a%20reference%20resource%20for%20gene%20and%20protein%20annotation.%20Nucleic%20acids%20research44(D1),%20D457%E2%80%93D462%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR49">
<span class="label">49.</span><cite>Korf, I. Gene finding in novel genomes. <em>BMC Bioinformatics</em><strong>5</strong>, 59 (2004).
</cite> [<a href="https://doi.org/10.1186/1471-2105-5-59" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC421630/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15144565/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Korf,%20I.%20Gene%20finding%20in%20novel%20genomes.%20BMC%20Bioinformatics5,%2059%20(2004)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR50">
<span class="label">50.</span><cite><em>NCBI Sequence Read Archive.</em><a href="https://identifiers.org/ncbi/insdc.sra:SRP576768" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://identifiers.org/ncbi/insdc.sra:SRP576768</a> (2025).</cite>
</li>
<li id="CR51">
<span class="label">51.</span><cite>Zhang, X. Genome assembly, predicted coding sequences and functional annotation files of <em>L. calcarifer</em>. <em>Figshare.</em>10.6084/m9.figshare.28735226 (2025).</cite>
</li>
<li id="CR52">
<span class="label">52.</span><cite><em>NCBI GenBank</em><a href="https://identifiers.org/ncbi/insdc.gca:GCA_051027255.1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://identifiers.org/ncbi/insdc.gca:GCA_051027255.1</a> (2025).</cite>
</li>
<li id="CR53">
<span class="label">53.</span><cite>Simao, F. A. <em>et al</em>. BUSCO: assessing genome assembly and annotation completeness with single-copy orthologs. <em>Bioinformatics</em><strong>31</strong>, 3210–3212 (2015).
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btv351" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26059717/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Simao,%20F.%20A.%20et%20al.%20BUSCO:%20assessing%20genome%20assembly%20and%20annotation%20completeness%20with%20single-copy%20orthologs.%20Bioinformatics31,%203210%E2%80%933212%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR54">
<span class="label">54.</span><cite>Rhie, A. <em>et al</em>. Merqury: reference-free quality, completeness, and phasing assessment for genome assemblies. <em>Genome Biology</em><strong>21</strong>(1), 245 (2020).
</cite> [<a href="https://doi.org/10.1186/s13059-020-02134-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7488777/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32928274/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rhie,%20A.%20et%20al.%20Merqury:%20reference-free%20quality,%20completeness,%20and%20phasing%20assessment%20for%20genome%20assemblies.%20Genome%20Biology21(1),%20245%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR55">
<span class="label">55.</span><cite>Li, K. <em>et al</em>. Identification of errors in draft genome assemblies at single-nucleotide resolution for quality assessment and improvement. <em>Nature Communications</em><strong>14</strong>(1), 6556 (2023).
</cite> [<a href="https://doi.org/10.1038/s41467-023-42336-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10582259/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37848433/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li,%20K.%20et%20al.%20Identification%20of%20errors%20in%20draft%20genome%20assemblies%20at%20single-nucleotide%20resolution%20for%20quality%20assessment%20and%20improvement.%20Nature%20Communications14(1),%206556%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR56">
<span class="label">56.</span><cite>Li, H. <em>et al</em>. Fast and accurate short read alignment with Burrows–Wheeler transform. <em>Bioinformatics</em><strong>25</strong>(14), 1754–1760 (2009).
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btp324" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2705234/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19451168/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li,%20H.%20et%20al.%20Fast%20and%20accurate%20short%20read%20alignment%20with%20Burrows%E2%80%93Wheeler%20transform.%20Bioinformatics25(14),%201754%E2%80%931760%20(2009)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR57">
<span class="label">57.</span><cite>Li, H. Minimap2: pairwise alignment for nucleotide sequences. <em>Bioinformatics</em><strong>34</strong>(18), 3094–3100 (2018).
</cite> [<a href="https://doi.org/10.1093/bioinformatics/bty191" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6137996/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29750242/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li,%20H.%20Minimap2:%20pairwise%20alignment%20for%20nucleotide%20sequences.%20Bioinformatics34(18),%203094%E2%80%933100%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR58">
<span class="label">58.</span><cite>Dobin, A. <em>et al</em>. STAR: ultrafast universal RNA-seq aligner. <em>Bioinformatics</em><strong>29</strong>(1), 15–21 (2013).
</cite> [<a href="https://doi.org/10.1093/bioinformatics/bts635" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3530905/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23104886/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Dobin,%20A.%20et%20al.%20STAR:%20ultrafast%20universal%20RNA-seq%20aligner.%20Bioinformatics29(1),%2015%E2%80%9321%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR59">
<span class="label">59.</span><cite>Quinlan, A. R. <em>et al</em>. BEDTools: a flexible suite of utilities for comparing genomic features. <em>Bioinformatics</em><strong>26</strong>(6), 841–842 (2010).
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btq033" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2832824/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20110278/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Quinlan,%20A.%20R.%20et%20al.%20BEDTools:%20a%20flexible%20suite%20of%20utilities%20for%20comparing%20genomic%20features.%20Bioinformatics26(6),%20841%E2%80%93842%20(2010)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adc93_" lang="en" class="data-citations"><h3 class="pmc_sec_title">Data Citations</h3>
<ol class="ref-list font-sm">
<li id="_dbx_ref_citation-alternatives48"><cite><em>NCBI Sequence Read Archive.</em><a href="https://identifiers.org/ncbi/insdc.sra:SRP576768" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://identifiers.org/ncbi/insdc.sra:SRP576768</a> (2025).</cite></li>
<li id="_dbx_ref_citation-alternatives49"><cite>Zhang, X. Genome assembly, predicted coding sequences and functional annotation files of <em>L. calcarifer</em>. <em>Figshare.</em>10.6084/m9.figshare.28735226 (2025).</cite></li>
<li id="_dbx_ref_citation-alternatives50"><cite><em>NCBI GenBank</em><a href="https://identifiers.org/ncbi/insdc.gca:GCA_051027255.1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://identifiers.org/ncbi/insdc.gca:GCA_051027255.1</a> (2025).</cite></li>
</ol></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>The versions and parameters of bioinformatics tools applied in this study have been described in the Method section. If no parameter is provided, the default is set. No custom code was used.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Scientific Data are provided here courtesy of <strong>Nature Publishing Group</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1038/s41597-025-05735-w"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/41597_2025_Article_5735.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (4.8 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12371012/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12371012/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12371012%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371012/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12371012/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12371012/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40841804/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12371012/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40841804/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12371012/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12371012/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="QgO4N20m65YAEt6nqrFoI4CODTV7U4m5bVWhnzRW6c4NJBj76ms7zXvrnick6qaf">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
