
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Structural disorder and distinctive motifs in the C-terminal region of the MADS-domain transcription factors are conserved across diverse taxa - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="152538178AF1D9A3053817002CA27029.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="plosone">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373214/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="PLOS One">
<meta name="citation_title" content="Structural disorder and distinctive motifs in the C-terminal region of the MADS-domain transcription factors are conserved across diverse taxa">
<meta name="citation_author" content="Erandi Ramírez-Aguirre">
<meta name="citation_author_institution" content="Laboratorio de Genética Molecular, Desarrollo y Evolución de Plantas, Depto. de Ecología Funcional, Instituto de Ecología, Universidad Nacional Autónoma de México, Ciudad de México, México">
<meta name="citation_author" content="Teresa Beatriz Nava-Ramírez">
<meta name="citation_author_institution" content="Departamento de Biología Molecular de Plantas, Instituto de Biotecnología, Universidad Nacional Autónoma de México, Cuernavaca, México">
<meta name="citation_author" content="Alejandra A Covarrubias">
<meta name="citation_author_institution" content="Departamento de Biología Molecular de Plantas, Instituto de Biotecnología, Universidad Nacional Autónoma de México, Cuernavaca, México">
<meta name="citation_author" content="Adriana Garay-Arroyo">
<meta name="citation_author_institution" content="Laboratorio de Genética Molecular, Desarrollo y Evolución de Plantas, Depto. de Ecología Funcional, Instituto de Ecología, Universidad Nacional Autónoma de México, Ciudad de México, México">
<meta name="citation_publication_date" content="2025 Aug 22">
<meta name="citation_volume" content="20">
<meta name="citation_issue" content="8">
<meta name="citation_firstpage" content="e0330098">
<meta name="citation_doi" content="10.1371/journal.pone.0330098">
<meta name="citation_pmid" content="40844982">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373214/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373214/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373214/pdf/pone.0330098.pdf">
<meta name="description" content="Plants have acquired the ability to adapt and respond to varying environmental conditions through modifications in their developmental programs. This adaptability relies on the plant’s capacity to sense environmental cues and respond via diverse ...">
<meta name="og:title" content="Structural disorder and distinctive motifs in the C-terminal region of the MADS-domain transcription factors are conserved across diverse taxa">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Plants have acquired the ability to adapt and respond to varying environmental conditions through modifications in their developmental programs. This adaptability relies on the plant’s capacity to sense environmental cues and respond via diverse ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373214/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12373214">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1371/journal.pone.0330098"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/pone.0330098.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12373214%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12373214/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12373214/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373214/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-plosone.png" alt="PLOS One logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to PLOS One" title="Link to PLOS One" shape="default" href="https://doi.org/10.1371/journal.pone.0330098" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">PLoS One</button></div>. 2025 Aug 22;20(8):e0330098. doi: <a href="https://doi.org/10.1371/journal.pone.0330098" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1371/journal.pone.0330098</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22PLoS%20One%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22PLoS%20One%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22PLoS%20One%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22PLoS%20One%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Structural disorder and distinctive motifs in the C-terminal region of the MADS-domain transcription factors are conserved across diverse taxa</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ram%C3%ADrez-Aguirre%20E%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Erandi Ramírez-Aguirre</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Erandi Ramírez-Aguirre</span></h3>
<div class="p">
<sup>1</sup>Laboratorio de Genética Molecular, Desarrollo y Evolución de Plantas, Depto. de Ecología Funcional, Instituto de Ecología, Universidad Nacional Autónoma de México, Ciudad de México, México</div>
<div>Data curation, Formal analysis, Investigation, Methodology, Visualization, Writing – original draft, Writing – review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ram%C3%ADrez-Aguirre%20E%22%5BAuthor%5D" class="usa-link"><span class="name western">Erandi Ramírez-Aguirre</span></a>
</div>
</div>
<sup>1,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nava-Ram%C3%ADrez%20TB%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Teresa Beatriz Nava-Ramírez</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Teresa Beatriz Nava-Ramírez</span></h3>
<div class="p">
<sup>2</sup>Departamento de Biología Molecular de Plantas, Instituto de Biotecnología, Universidad Nacional Autónoma de México, Cuernavaca, México</div>
<div>Formal analysis, Investigation, Methodology, Visualization, Writing – original draft, Writing – review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nava-Ram%C3%ADrez%20TB%22%5BAuthor%5D" class="usa-link"><span class="name western">Teresa Beatriz Nava-Ramírez</span></a>
</div>
</div>
<sup>2,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Covarrubias%20AA%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Alejandra A Covarrubias</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Alejandra A Covarrubias</span></h3>
<div class="p">
<sup>2</sup>Departamento de Biología Molecular de Plantas, Instituto de Biotecnología, Universidad Nacional Autónoma de México, Cuernavaca, México</div>
<div>Conceptualization, Formal analysis, Investigation, Supervision, Writing – original draft, Writing – review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Covarrubias%20AA%22%5BAuthor%5D" class="usa-link"><span class="name western">Alejandra A Covarrubias</span></a>
</div>
</div>
<sup>2,</sup><sup>*</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Garay-Arroyo%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Adriana Garay-Arroyo</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Adriana Garay-Arroyo</span></h3>
<div class="p">
<sup>1</sup>Laboratorio de Genética Molecular, Desarrollo y Evolución de Plantas, Depto. de Ecología Funcional, Instituto de Ecología, Universidad Nacional Autónoma de México, Ciudad de México, México</div>
<div>Conceptualization, Funding acquisition, Investigation, Supervision, Writing – original draft, Writing – review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Garay-Arroyo%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Adriana Garay-Arroyo</span></a>
</div>
</div>
<sup>1,</sup><sup>*</sup>
</div>
<div class="cg p">Editor: <span class="name western">Ajit Prakash</span><sup>3</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="aff001">
<sup>1</sup>Laboratorio de Genética Molecular, Desarrollo y Evolución de Plantas, Depto. de Ecología Funcional, Instituto de Ecología, Universidad Nacional Autónoma de México, Ciudad de México, México</div>
<div id="aff002">
<sup>2</sup>Departamento de Biología Molecular de Plantas, Instituto de Biotecnología, Universidad Nacional Autónoma de México, Cuernavaca, México</div>
<div id="edit1">
<sup>3</sup>UNC CH: The University of North Carolina at Chapel Hill, UNITED STATES OF AMERICA</div>
<div class="author-notes p">
<div class="fn" id="cor001">
<sup>✉</sup><p class="display-inline">* E-mail: <span>agaray@iecologia.unam.mx</span> (AG-A); <span>alejandra.covarrubias@ibt.unam.mx</span> (AAC)</p>
</div>
<div class="fn" id="coi001"><p><strong>Competing Interests: </strong>The authors have declared that no competing interests exist.</p></div>
<div class="fn" id="fn001"><p>☯ These authors have equal participation in the research and are the first authors.</p></div>
<div class="fn" id="_eqcntrb93pmc__">
<sup>#</sup><p class="display-inline">Contributed equally.</p>
</div>
</div>
<h4 class="font-secondary">Roles</h4>
<div class="p">
<strong class="contrib"><span class="name western">Erandi Ramírez-Aguirre</span></strong>: <span class="role">Data curation, Formal analysis, Investigation, Methodology, Visualization, Writing – original draft, Writing – review &amp; editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">Teresa Beatriz Nava-Ramírez</span></strong>: <span class="role">Formal analysis, Investigation, Methodology, Visualization, Writing – original draft, Writing – review &amp; editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">Alejandra A Covarrubias</span></strong>: <span class="role">Conceptualization, Formal analysis, Investigation, Supervision, Writing – original draft, Writing – review &amp; editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">Adriana Garay-Arroyo</span></strong>: <span class="role">Conceptualization, Funding acquisition, Investigation, Supervision, Writing – original draft, Writing – review &amp; editing</span>
</div>
<div class="p">
<strong class="contrib"><span class="name western">Ajit Prakash</span></strong>: <span class="role">Editor</span>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 Feb 25; Accepted 2025 Jul 26; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© 2025 Ramírez-Aguirre et al</div>
<p>This is an open access article distributed under the terms of the <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Creative Commons Attribution License</a>, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12373214  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40844982/" class="usa-link">40844982</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>Abstract</h2>
<p>Plants have acquired the ability to adapt and respond to varying environmental conditions through modifications in their developmental programs. This adaptability relies on the plant’s capacity to sense environmental cues and respond via diverse signal transduction pathways and transcriptional regulation. Transcription factors are central in these processes, orchestrating specific gene expression in both developmental and stress responses. In <em>Arabidopsis thaliana</em>, 91% of transcription factors contain large intrinsically disordered regions (IDRs). The structural flexibility in these regions is critical in protein-protein interactions and contributes to functional versatility across different cell types. MADS-domain transcription factors constitute an eukaryotic protein family involved in a diversity of developmental processes and stress responses. Using bioinformatic tools, we found that most Arabidopsis MADS-domain proteins contain IDRs (≥30 residues) in their C-terminal region, with a higher proportion of global disorder in Type II compared to Type I MADS-domain proteins. Remarkably orthologous proteins from non-plant species in the Eukarya domain (<em>Drosophila melanogaster</em>, <em>Saccharomyces cerevisiae,</em> and <em>Homo sapiens</em>) also present disordered C-terminal regions, containing longer IDRs than those found in Arabidopsis<em>,</em> or other analyzed plant species. Additionally, conserved motifs were identified within the C-terminal IDRs of Arabidopsis Type I and Type II MADS-domain proteins, suggesting interactions with co-regulatory partners. We also identified putative activation domains in the C-terminal region of Type I and Type II MADS-domain proteins. The involvement of IDRs in selecting co-regulators is further supported by the identification of Molecular Recognition Features (MoRFs) in Type II MADS-domain proteins. The conserved structural disorder in the C-terminal region of MADS-domain proteins, which includes specific motifs, across diverse domains of life provides valuable insights into their structural properties and mechanisms of action as transcriptional regulators.</p></section><section id="sec001"><h2 class="pmc_sec_title">Introduction</h2>
<p>Plants have developed a wide range of mechanisms to adapt to changing environmental conditions. Their ability to sense various signals and respond accordingly through diverse and complex signaling pathways to accomplish accurate gene regulation, allows them to exhibit phenotypic plasticity. This plasticity enables plants to optimize their growth and development in response to factors such as light, temperature, water availability, nutrient levels, and biotic stresses. Remarkably different species of plants share plastic traits associated with their specific habitats (e.g., [<a href="#pone.0330098.ref001" class="usa-link" aria-describedby="pone.0330098.ref001">1</a>]), highlighting the adaptive role of plastic responses and underlying the functional role of the genes involved and their regulation [<a href="#pone.0330098.ref002" class="usa-link" aria-describedby="pone.0330098.ref002">2</a>].</p>
<p>Transcription factors (TFs) are central regulators in all organisms, playing key roles in diverse biological processes, such as growth, differentiation, hormone signaling, stress responses, and immune defense. By integrating signals from different pathways, TFs orchestrate the complex regulatory networks that underlie plant development and adaptation, enabling plants to respond dynamically to changing environmental conditions [<a href="#pone.0330098.ref003" class="usa-link" aria-describedby="pone.0330098.ref003">3</a>–<a href="#pone.0330098.ref009" class="usa-link" aria-describedby="pone.0330098.ref009">9</a>].</p>
<p>MADS-domain TFs belong to a eukaryotic protein family that participates in several developmental processes and stress responses [<a href="#pone.0330098.ref010" class="usa-link" aria-describedby="pone.0330098.ref010">10</a>,<a href="#pone.0330098.ref011" class="usa-link" aria-describedby="pone.0330098.ref011">11</a>]. The name MADS-box is derived from the four founding members of this family: <em>MINICHROMOSOME MAINTENANCE 1</em> (<em>MCM1</em>) in yeast, <em>AGAMOUS</em> (<em>AG</em>) in <em>Arabidopsis thaliana</em> (from now on Arabidopsis), <em>DEFICIENS</em> (<em>DEF</em>) in <em>Antirrhinum majus</em> (snapdragon), and <em>SERUM RESPONSE FACTOR</em> (<em>SRF</em>) in humans [<a href="#pone.0330098.ref012" class="usa-link" aria-describedby="pone.0330098.ref012">12</a>]. The MADS (M) domain is highly conserved and present in all members of the family and is typically located in the N-terminal region of the protein. However, there are exceptions to this pattern, such as in SRF in humans and MCM1 in yeast, where the MADS domain is found in different positions within the protein [<a href="#pone.0330098.ref013" class="usa-link" aria-describedby="pone.0330098.ref013">13</a>,<a href="#pone.0330098.ref014" class="usa-link" aria-describedby="pone.0330098.ref014">14</a>]. While some organisms have just a few MADS-box genes, such as <em>Saccharomyces cerevisiae</em> (yeast) with four, <em>Drosophila melanogaster</em> with two, and <em>Homo sapiens</em> with five [<a href="#pone.0330098.ref015" class="usa-link" aria-describedby="pone.0330098.ref015">15</a>–<a href="#pone.0330098.ref018" class="usa-link" aria-describedby="pone.0330098.ref018">18</a>], plants exhibit a considerably higher abundance of these genes (52–167 in 21 angiosperms) [<a href="#pone.0330098.ref019" class="usa-link" aria-describedby="pone.0330098.ref019">19</a>–<a href="#pone.0330098.ref022" class="usa-link" aria-describedby="pone.0330098.ref022">22</a>]. MADS-domain proteins are classified into two main types: Type I (Serum Response Factor (SRF)-like), and Type II (Myocyte Enhancer Factor (MEF)-like) based on their sequence characteristics, genomic organization, and functional roles [<a href="#pone.0330098.ref023" class="usa-link" aria-describedby="pone.0330098.ref023">23</a>,<a href="#pone.0330098.ref024" class="usa-link" aria-describedby="pone.0330098.ref024">24</a>]. In yeast, MADS-domain proteins are involved in key processes such as arginine metabolism, osmotic stress response, and mating type regulation. In <em>Drosophila</em>, MEF2 genes participate in muscle differentiation, while in humans the different MEF2 genes also contribute to heart and neural development, and the response to various diseases [<a href="#pone.0330098.ref013" class="usa-link" aria-describedby="pone.0330098.ref013">13</a>,<a href="#pone.0330098.ref016" class="usa-link" aria-describedby="pone.0330098.ref016">16</a>,<a href="#pone.0330098.ref025" class="usa-link" aria-describedby="pone.0330098.ref025">25</a>–<a href="#pone.0330098.ref027" class="usa-link" aria-describedby="pone.0330098.ref027">27</a>]. In plants, MADS-domain proteins play several roles in development, encompassing the transition to flowering, flower organ development, fruit ripening, root development, and vegetative phase change, among others [<a href="#pone.0330098.ref024" class="usa-link" aria-describedby="pone.0330098.ref024">24</a>]. In Arabidopsis, Type I MADS-domain proteins have been primarily implicated in female gametogenesis and seed development, while Type II MADS-domain proteins play central roles in controlling floral organ identity and flower development in angiosperms and are involved in almost all Arabidopsis developmental processes [<a href="#pone.0330098.ref024" class="usa-link" aria-describedby="pone.0330098.ref024">24</a>,<a href="#pone.0330098.ref028" class="usa-link" aria-describedby="pone.0330098.ref028">28</a>]. Type I MADS-domain proteins typically consist of three domains: M, I (intervening), and C (C-terminal domain), whereas Type II proteins possess four domains, forming the acronym MIKC [<a href="#pone.0330098.ref029" class="usa-link" aria-describedby="pone.0330098.ref029">29</a>–<a href="#pone.0330098.ref033" class="usa-link" aria-describedby="pone.0330098.ref033">33</a>]. In both cases, the “M” domain is responsible for DNA binding but also participates in protein-protein interactions. MADS-domain proteins bind to a specific DNA sequence known as the “CArG” box located in the regulatory regions of target genes. The “I” domain is located between the “M” and the “K” domains. While it is less conserved than the “M” domain, it also participates in DNA-binding and dimerization specificity [<a href="#pone.0330098.ref033" class="usa-link" aria-describedby="pone.0330098.ref033">33</a>,<a href="#pone.0330098.ref034" class="usa-link" aria-describedby="pone.0330098.ref034">34</a>]. The “K” domain derives its name from its structural resemblance to proteins known as keratins. This domain is involved in protein-protein interactions and contributes to the formation of higher-order complexes. Finally, the “C” domain is located at the C-terminal of the protein and exhibits variations in length and sequence among different MADS-domain proteins. For some MADS-domain proteins, it has been shown that this domain participates in transactivation and in the formation of higher-order complexes [<a href="#pone.0330098.ref035" class="usa-link" aria-describedby="pone.0330098.ref035">35</a>].</p>
<p>A phylogenetic analysis of 107 Arabidopsis MADS-box sequences revealed that these genes can be grouped into two main lineages (Type I and Type II) or five subfamilies: Mα, Mβ, and Mγ, and MIKC<sup>c</sup> and MIKC*. The Mα, Mβ, and Mγ, subfamilies belong to Type I MADS-domain proteins whereas the MIKCc and MIKC* subfamilies are classified as Type II [<a href="#pone.0330098.ref024" class="usa-link" aria-describedby="pone.0330098.ref024">24</a>,<a href="#pone.0330098.ref036" class="usa-link" aria-describedby="pone.0330098.ref036">36</a>]. The genomic distribution of MIKC genes, along with evidence from genome history, suggests that these genes existed before the Arabidopsis genome polyploidization event and are distinct from the Mα, Mβ, and Mγ subfamilies [<a href="#pone.0330098.ref023" class="usa-link" aria-describedby="pone.0330098.ref023">23</a>,<a href="#pone.0330098.ref036" class="usa-link" aria-describedby="pone.0330098.ref036">36</a>].</p>
<p>In Arabidopsis, 91% of TFs are characterized by large intrinsically disordered regions (IDRs), crucial for diverse cellular functions [<a href="#pone.0330098.ref037" class="usa-link" aria-describedby="pone.0330098.ref037">37</a>]. IDRs are segments of proteins that lack a fixed or stable three-dimensional structure under physiological conditions. Their unique physicochemical properties arise from the specific nature of their amino acid sequences and the characteristics of the individual amino acid residues within these sequences. Intrinsically disordered proteins (IDPs) and/or IDRs often lack a significant number of hydrophobic residues typically associated with folded protein domains whereas they are enriched in amino acids that allow their flexibility [<a href="#pone.0330098.ref038" class="usa-link" aria-describedby="pone.0330098.ref038">38</a>]. The IDP amino acid sequences enable a structural dynamic that depends on the physicochemical properties of their environment and the interactions with other molecules [<a href="#pone.0330098.ref039" class="usa-link" aria-describedby="pone.0330098.ref039">39</a>,<a href="#pone.0330098.ref040" class="usa-link" aria-describedby="pone.0330098.ref040">40</a>]. This flexibility plays a pivotal role in mediating IDP multiple protein-protein interactions and in their participation in different developmental processes, such as cell cycle, transcriptional control, and responses to different stress conditions [<a href="#pone.0330098.ref041" class="usa-link" aria-describedby="pone.0330098.ref041">41</a>,<a href="#pone.0330098.ref042" class="usa-link" aria-describedby="pone.0330098.ref042">42</a>]. However, only a limited number of TF families have been analyzed, highlighting the presence of IDRs and their association with their role in their respective functions [<a href="#pone.0330098.ref043" class="usa-link" aria-describedby="pone.0330098.ref043">43</a>–<a href="#pone.0330098.ref047" class="usa-link" aria-describedby="pone.0330098.ref047">47</a>].</p>
<p>Interestingly, IDRs often contain specific sequences known as Molecular Recognition Features (MoRFs), these short transiently folding sequences play a key role in determining site-partner specificity [<a href="#pone.0330098.ref048" class="usa-link" aria-describedby="pone.0330098.ref048">48</a>,<a href="#pone.0330098.ref049" class="usa-link" aria-describedby="pone.0330098.ref049">49</a>]. MoRFs show distinctive physicochemical characteristics that may aid in protein interaction through the hydrophobicity of their amino acid composition (rich in proline and methionine) and the hydrophilic nature of IDRs [<a href="#pone.0330098.ref050" class="usa-link" aria-describedby="pone.0330098.ref050">50</a>,<a href="#pone.0330098.ref051" class="usa-link" aria-describedby="pone.0330098.ref051">51</a>]. The charge of individual amino acids and their electrostatic interactions affect the conformational structure of the protein, which in turn affects its binding specificity and stability. This has led to the hypothesis that MoRFs are key participants in the multi-partner binding capacity of hub proteins [<a href="#pone.0330098.ref052" class="usa-link" aria-describedby="pone.0330098.ref052">52</a>–<a href="#pone.0330098.ref054" class="usa-link" aria-describedby="pone.0330098.ref054">54</a>]. The dynamic interconnectivity of IDPs/IDRs has also been associated with their ability to aggregate through liquid-liquid phase separation (LLPS), a process that plays a critical role in protein and RNA organization [<a href="#pone.0330098.ref055" class="usa-link" aria-describedby="pone.0330098.ref055">55</a>]. LLPS is now recognized as a fundamental organizational and regulatory principle across all organisms. It enables the concentration of specific proteins and nucleic acids into biomolecular condensates, i.e., membrane-less organelles that regulate cellular activity by localizing particular proteins, accelerating enzymatic reactions, and favoring selective interactions. This dynamic regulation supports the precise control of a diversity of processes involved in growth, development, and responses to environmental changes and pathogens [<a href="#pone.0330098.ref056" class="usa-link" aria-describedby="pone.0330098.ref056">56</a>].</p>
<p>In this work, we used bioinformatic tools and publicly available datasets to investigate the presence of IDRs in MADS-domain proteins from plants and other taxa. We found that Arabidopsis Type I and Type II MADS-domain TFs present a high level of disorder, ranging from 20 to 80%, with longer IDRs in their C-terminal domain. This characteristic was found to be conserved in orthologs of these proteins from other plant species as well as from <em>Drosophila melanogaster</em>, <em>Saccharomyces cerevisiae,</em> and <em>Homo sapiens</em>. Of note, within the C-terminal region of proteins in both the SOC1 and the FLC clades [<a href="#pone.0330098.ref023" class="usa-link" aria-describedby="pone.0330098.ref023">23</a>,<a href="#pone.0330098.ref024" class="usa-link" aria-describedby="pone.0330098.ref024">24</a>], we found two different motifs common to all members of the SOC1 and FLC clades, suggesting functional restriction and phylogenetic conservation within these groups. Interestingly, the SOC1 motif is conserved across SOC1 orthologs from different plant species, and in other Arabidopsis Type II MADS-domain proteins. Many Arabidopsis MADS-domain proteins also contain MoRFs that overlap with or are located just a few amino acids away from the SOC1 and the FLC motifs in the last residues of the C-terminal region. We mapped putative activation domains (ADs) across all clades of Type I and Type II MADS-domain proteins on their C-terminal region. Interestingly, Type I showed a higher proportion of ADs compared to Type II MADS-domain proteins. Finally, our <em>in silico</em> analysis predicts that some MADS-domain proteins may form condensates, potentially leading to different conformational arrangements and expanding their functional roles. The information in this work adds valuable insights to better understand the MADS TFs molecular mechanisms involved in the control of plant growth, morphogenesis, and responses to environmental changes.</p></section><section id="sec002"><h2 class="pmc_sec_title">Materials and methods</h2>
<section id="sec003"><h3 class="pmc_sec_title">Complete sequence retrieval and domain description</h3>
<p>Individual protein sequences of <em>Arabidopsis thaliana</em> were retrieved from the TAIR database [<a href="#pone.0330098.ref057" class="usa-link" aria-describedby="pone.0330098.ref057">57</a>] except for STK (AGL11) which was retrieved from UniProt (<a href="https://www.ncbi.nlm.nih.gov/protein/Q38836" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Q38836</a>). Protein domains were identified using the UniProt database [<a href="#pone.0330098.ref058" class="usa-link" aria-describedby="pone.0330098.ref058">58</a>]. The I domain was defined based on the <em>AtSOC1</em> protein obtained from the supplementary information in Lai et al. (2021) [<a href="#pone.0330098.ref033" class="usa-link" aria-describedby="pone.0330098.ref033">33</a>] for both Type I and Type II MADS-domain proteins. In this study, the C-terminal region was defined as the sequence located downstream of the I (for both Type I MADS-domain proteins) or downstream of the K domain (for Type II MADS-domain proteins). Retrieval of protein sequences from other plants and non-plant organisms was obtained from GenBank, the Rice Annotation Project DataBase, and UniProt [<a href="#pone.0330098.ref058" class="usa-link" aria-describedby="pone.0330098.ref058">58</a>–<a href="#pone.0330098.ref060" class="usa-link" aria-describedby="pone.0330098.ref060">60</a>] (<a href="#pone.0330098.s001" class="usa-link">S1 Table</a>). A total of 143 sequences were collected, including 100 from Arabidopsis, nine from rice, and at least one homolog from basal plants and other angiosperms (<a href="#pone.0330098.s001" class="usa-link">S1 Table</a>). The selection of the Type I MADS-domain proteins was based on Bemer et al. (2010) [<a href="#pone.0330098.ref028" class="usa-link" aria-describedby="pone.0330098.ref028">28</a>], while the selection of rice sequences covering both Types of MADS-domain protein orthologs was based on Arora et al. (2007) [<a href="#pone.0330098.ref061" class="usa-link" aria-describedby="pone.0330098.ref061">61</a>].</p></section><section id="sec004"><h3 class="pmc_sec_title">Sequence alignment and phylogenetic analysis</h3>
<p>After retrieving the individual protein sequences, they were aligned with the MAFFT algorithm at the MAFFT web server [<a href="#pone.0330098.ref062" class="usa-link" aria-describedby="pone.0330098.ref062">62</a>,<a href="#pone.0330098.ref063" class="usa-link" aria-describedby="pone.0330098.ref063">63</a>]. The parameters used were L-INS-i and the mafft-homolog function with UniRef. After the alignment of the 143 sequences, we visually inspected for ambiguous misalignments with the AliView program [<a href="#pone.0330098.ref064" class="usa-link" aria-describedby="pone.0330098.ref064">64</a>]. Using this alignment, we determined the position of the I domain for Type I and Type II MADS-domain proteins. The M and the K domains for all MADS-domain proteins were retrieved from Uniprot. This approach allowed us to accurately map the C-terminal region along with its corresponding IDRs. By assigning approximately 190 amino acid residues up to the K-domain, we found a better alignment of conserved regions among the selected protein sequences. The alignment with the MIK domains was used to recover a phylogeny by applying a Maximum Likelihood approach with the RAxML algorithm, following a JTT substitution model, at the CIPRES website [<a href="#pone.0330098.ref065" class="usa-link" aria-describedby="pone.0330098.ref065">65</a>–<a href="#pone.0330098.ref068" class="usa-link" aria-describedby="pone.0330098.ref068">68</a>].</p></section><section id="sec005"><h3 class="pmc_sec_title">Motif and Molecular Recognition Features (MoRFs) identification</h3>
<p>The MEME algorithm [<a href="#pone.0330098.ref069" class="usa-link" aria-describedby="pone.0330098.ref069">69</a>,<a href="#pone.0330098.ref070" class="usa-link" aria-describedby="pone.0330098.ref070">70</a>] was used to identify motifs within the whole protein sequence or the C-terminal regions. Molecular Recognition Features (MoRFs) are small IDRs potentially involved in the initial events of molecular recognition during protein-protein interactions. These regions undergo disorder-to-order transitions upon binding. MoRFs were identified in the complete protein sequences using the fMoRFpred tool that uses the physicochemical properties of amino acid residues to fit a Support Vector Machine (SVM) model for predicting the presence of MoRFs within IDRs [<a href="#pone.0330098.ref049" class="usa-link" aria-describedby="pone.0330098.ref049">49</a>,<a href="#pone.0330098.ref071" class="usa-link" aria-describedby="pone.0330098.ref071">71</a>]. To map the predicted MoRFs, their locations were aligned across the protein sequences alongside the MEME motifs.</p></section><section id="sec006"><h3 class="pmc_sec_title">Mapping of activation domains within the C-terminal region of MADS-domain proteins</h3>
<p>Potential activation domains (ADs) in MADS-domain proteins were retrieved directly from Morffy et al. (2024) [<a href="#pone.0330098.ref072" class="usa-link" aria-describedby="pone.0330098.ref072">72</a>] (<a href="#pone.0330098.s002" class="usa-link">S2 Table</a>). In this study, the authors experimentally identified activation domains in various plant TFs using a comprehensive library. This library consisted of overlapping 40 amino acid fragments, spanning the entire set of plant TFs, with a step size of 10 amino acids, resulting in a total of 68,441 fragments. These fragments were screened in yeast to assess their transcriptional activation capacity. Based on this experiment and subsequent normalization, a (Plant Activation Domain Identification) PADI score was assigned to each fragment. Among the fragments showing transcriptional activation activity, some corresponded to MADS-domain TFs. The PADI scores included in our manuscript are those reported here, and the localization of activation domains (AD) within MADS-domain TFs was inferred from the results obtained in Morffy et al. (2024) [<a href="#pone.0330098.ref072" class="usa-link" aria-describedby="pone.0330098.ref072">72</a>], where the authors applied a neural network-based algorithm, known as transcriptional activation domain activity (TADA) network. Their work integrated multiple layers of analysis, including the construction of a feature matrix and the use of methods to assess the impact of both individual input features and border local and global interactions predicted by TADA. Additionally, these results were further analyzed using deep learning to identify key properties relevant to the prediction of ADs. These authors also applied a Shapley additive explanations (SHAP) analysis to capture non-linear and linear correlations, thereby uncovering complex patterns. This was followed by additional deep-learning steps that culminated in the development of a tool capable of predicting potential ADs. Using the Morffy et al. (2024) [<a href="#pone.0330098.ref072" class="usa-link" aria-describedby="pone.0330098.ref072">72</a>] dataset, we filtered the information for MADS-domain proteins to localize the “NOT-AD” and the “AD” fragments within their C-terminal IDR.</p></section><section id="sec007"><h3 class="pmc_sec_title">IDR prediction and structural disorder score of the complete proteins</h3>
<p>The majority of Arabidopsis IDRs were retrieved from the Alphafold section of the MobiDb database [<a href="#pone.0330098.ref073" class="usa-link" aria-describedby="pone.0330098.ref073">73</a>] (accessed in November 2023). The IDR prediction was conducted with the AlphaFold Colab Notebook with default values or directly in the AlphaFold web server [<a href="#pone.0330098.ref074" class="usa-link" aria-describedby="pone.0330098.ref074">74</a>] (accessed in January 2024). The resulting structures were visually inspected with the <em>pbd.file</em> coupled with the Predicted Aligned Error (PAE) of the proteins in the Chimera X molecular visualizer [<a href="#pone.0330098.ref075" class="usa-link" aria-describedby="pone.0330098.ref075">75</a>]. We also checked the IDRs for each prediction in JalView and accounted for the low Temperature Factor regions, which correspond to the low pLDDT values [<a href="#pone.0330098.ref076" class="usa-link" aria-describedby="pone.0330098.ref076">76</a>]. Disordered regions predicted by AlphaFold are those with a pLDDT &lt; 50, usually seen as ribbon-like structures [<a href="#pone.0330098.ref077" class="usa-link" aria-describedby="pone.0330098.ref077">77</a>, <a href="#pone.0330098.ref078" class="usa-link" aria-describedby="pone.0330098.ref078">78</a>]. The structural disorder analysis was assessed using the RIDAO platform [<a href="#pone.0330098.ref079" class="usa-link" aria-describedby="pone.0330098.ref079">79</a>], which includes various intrinsic disorder predictors [<a href="#pone.0330098.ref079" class="usa-link" aria-describedby="pone.0330098.ref079">79</a>,<a href="#pone.0330098.ref080" class="usa-link" aria-describedby="pone.0330098.ref080">80</a>]. Protein sequences in FASTA format were used as input for the analysis, the amino acid sequence of the C-terminal region was manually extracted from the original full-length protein sequences and formatted in FASTA. The RIDAO platform provides two key metrics for each protein: The Average Disorder Score (ADS) and the Percent of Predicted Intrinsic Disorder Residues (PPIDR). The ADS indicate the overall propensity of a protein to be intrinsically disordered, allowing a comprehensive analysis of structural disorder. The PPIDR represents the proportion of amino acids predicted to be disordered, considering those with significant intrinsic disorder score (&gt; 0.5), relative to the total number of residues in the protein [<a href="#pone.0330098.ref081" class="usa-link" aria-describedby="pone.0330098.ref081">81</a>].</p></section><section id="sec008"><h3 class="pmc_sec_title">Physicochemical properties of disordered proteins</h3>
<p>Parameters related to the amino acid residue charge (NCPR: net charge per residue, and FCR: fraction of charge residues) and their distribution (patterning parameter kappa) [<a href="#pone.0330098.ref052" class="usa-link" aria-describedby="pone.0330098.ref052">52</a>] across the C-terminal regions of MADS-domain proteins were calculated using tools available in the CIDER web server [<a href="#pone.0330098.ref082" class="usa-link" aria-describedby="pone.0330098.ref082">82</a>,<a href="#pone.0330098.ref083" class="usa-link" aria-describedby="pone.0330098.ref083">83</a>].</p></section><section id="sec009"><h3 class="pmc_sec_title">Post-translational modifications (phosphorylation)</h3>
<p>Phosphorylation in serine, threonine, and tyrosine residues within the complete protein sequences was predicted using the NetPhos algorithm [<a href="#pone.0330098.ref084" class="usa-link" aria-describedby="pone.0330098.ref084">84</a>–<a href="#pone.0330098.ref086" class="usa-link" aria-describedby="pone.0330098.ref086">86</a>]. Additionally, experimentally validated data were obtained from the ATHENA at [<a href="#pone.0330098.ref087" class="usa-link" aria-describedby="pone.0330098.ref087">87</a>] (accessed in February 2024) and the EPSD databases (accessed in May 2025) [<a href="#pone.0330098.ref088" class="usa-link" aria-describedby="pone.0330098.ref088">88</a>].</p></section><section id="sec010"><h3 class="pmc_sec_title">Condensate formation tendency</h3>
<p>Condensate propensity was calculated for the complete MADS-domain protein and their C-terminal domain region sequences using the FuzDrop algorithm [<a href="#pone.0330098.ref089" class="usa-link" aria-describedby="pone.0330098.ref089">89</a>,<a href="#pone.0330098.ref090" class="usa-link" aria-describedby="pone.0330098.ref090">90</a>]. Protein sequences in FASTA format were used as input for the analysis, the amino acid sequence of the C-terminal region was manually extracted from the original full-length protein sequences and formatted in FASTA.</p></section><section id="sec011"><h3 class="pmc_sec_title">Data analyses and image rendering</h3>
<p>We evaluated differences in disorder ADS and PPIDRS among MADS-domain proteins, with a Wilcoxon test under a permutation approach with the coin R package [<a href="#pone.0330098.ref091" class="usa-link" aria-describedby="pone.0330098.ref091">91</a>]. Also, we compared the raw number of phosphorylation sites among Arabidopsis MADS-domain protein types with a Chi-squared test. All the analyses and graphics were performed in the R program [<a href="#pone.0330098.ref092" class="usa-link" aria-describedby="pone.0330098.ref092">92</a>]. Protein diagrams were made with the drawProteins library from the Bioconductor repository, and the final rendering was done with the GIMP program [<a href="#pone.0330098.ref093" class="usa-link" aria-describedby="pone.0330098.ref093">93</a>] (license GPLv3. Version 3.0.4, Free Software). Statistical graphs were made with the tidyverse libraries and with ggplot2 side-by-side working libraries (<a href="#pone.0330098.s003" class="usa-link">S3 Table</a>). Other R packages used here are cited in <a href="#pone.0330098.s003" class="usa-link">S3 Table</a>.</p></section></section><section id="sec012"><h2 class="pmc_sec_title">Results</h2>
<section id="sec013"><h3 class="pmc_sec_title">The C-terminal domain of Arabidopsis MADS-domain transcription factor family presents structural disorder propensity</h3>
<p>The function of MADS-domain proteins in development is conserved across diverse plant species and other taxa. Given the diversity of cell types and conditions in which these transcription factors regulate gene expression, the structural disorder appears to be an advantageous property, enabling efficient and versatile functionality. In this study, we evaluated the occurrence of structural disorder in 58 Type I and 42 Type II MADS-domain proteins from Arabidopsis to gain insight into their protein structure and its relationship to their function. To evaluate the degree of disorder, the Average Disorder Score (ADS) and the Percent of Predicted Intrinsic Disorder Residues (PPIDR) were calculated in the RIDAO platform, for both the complete proteins and their C-terminal regions. For Type I MADS-domain proteins, both the mean ADS and mean PPIDR of the complete proteins and their C-terminal regions are similar (<a href="#pone.0330098.g001" class="usa-link">Fig 1A</a> and <a href="#pone.0330098.g001" class="usa-link">1C</a>, left panel). In contrast, for Type II MADS-domain proteins, both the mean ADS and mean PPIDR are larger for the C-terminal region than for the complete proteins (<a href="#pone.0330098.g001" class="usa-link">Fig 1A</a> and <a href="#pone.0330098.g001" class="usa-link">1C</a>, right panel). Also, the relation between the protein length and ADS or PPIDR values revealed a broader length distribution (100–450 amino acid residues) for Type I MADS-domain proteins and only a minor inverse correlation between protein length and global disorder (<a href="#pone.0330098.g001" class="usa-link">Fig 1B</a> and <a href="#pone.0330098.g001" class="usa-link">1D</a>, left panel). In contrast, Type II proteins exhibited a narrower length range (200–300 amino acid residues) but followed a positive trend between length and disorder (<a href="#pone.0330098.g001" class="usa-link">Fig 1B</a> and <a href="#pone.0330098.g001" class="usa-link">1D</a>, right panel). Moreover, Type II MADS-domain proteins present a significantly higher disorder (ADS and PPIDR) compared to Type I proteins (permutation Wilcoxon test, [ADS] Z = −3.1497, p = 0.0016; [PPIDR] Z = −3.897 p = 1x10-<sup>4</sup>, <a href="#pone.0330098.g001" class="usa-link">Fig 1</a>). AlphaFold-based analysis of the structural disorder distribution revealed that this property is predominantly located in the C-terminal regions for both types: in Type I this region follows the I-domain, while in Type II proteins it is found beyond the K-domain (<a href="#pone.0330098.g002" class="usa-link">Fig 2A</a>, <a href="#pone.0330098.s004" class="usa-link">S4 Table</a>). Notably, the longest IDRs in both types were consistently located within the C-terminal region, and the IDRs at the C-terminal region in Type I MADS-domain proteins were longer than in the Type II MADS-domain proteins (permutation Wilcoxon test, Z = 4.1552, p-value &lt; 1e-04p, <a href="#pone.0330098.g002" class="usa-link">Fig 2</a>, <a href="#pone.0330098.s004" class="usa-link">S4 Table</a>). The number of IDRs containing at least 30 amino acid residues was similar between the two groups (<a href="#pone.0330098.s004" class="usa-link">S4 Table</a>, Type I total count = 51, Type II total count = 42, χ<sup>2</sup> = 0.87, df = 1, p = 0.351). Nevertheless, in nine Type I MADS-domain proteins (AGAMOUS-LIKE91 [AGL91], AGL29, AGL58, AGL64, AGL87, AGL101, AGL59, AGL102, AGL85) we only detected IDRs shorter than 30 amino acid residues, whereas all Type II MADS-domain proteins contain at least one IDR with ≥30 amino acids. Interestingly, most MADS-domain proteins present an IDR at the beginning of the N-terminal, in the first residues of the M-domain.</p>
<figure class="fig xbox font-sm" id="pone.0330098.g001"><h4 class="obj_head">Fig 1. The Average Disorder Score (ADS) and Predicted Percentage of Intrinsically Disordered Residues (PPIDR) values in Type I and Type II MADS-domain proteins.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373214_pone.0330098.g001.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/7d0c/12373214/e258a90af0a6/pone.0330098.g001.jpg" loading="lazy" height="1265" width="684" alt="Fig 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/pone.0330098.g001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>(A)</strong> Distribution of ADS in MADS-domain TFs. <strong>(B)</strong> Scatterplots showing the association between protein length and ADS. <strong>(C)</strong> Distribution of PPIDR values in MADS-domain TFs. <strong>(D)</strong> Scatterplots showing the association between protein length and PPIDR. Red dots indicate the mean of each value, and black dots indicate the raw values of either ADS or PPIDR per protein. Statistical analysis using the Wilcoxon test shows a significant difference between Type I and Type II proteins (p = 0.0002).</p></figcaption></figure><figure class="fig xbox font-sm" id="pone.0330098.g002"><h4 class="obj_head">Fig 2. MIKC domains, disordered regions, and phosphorylation sites for MADS-domain proteins.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373214_pone.0330098.g002.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/7d0c/12373214/dfe9a75e2e36/pone.0330098.g002.jpg" loading="lazy" height="612" width="723" alt="Fig 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/pone.0330098.g002/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>(A)</strong> Schematic representation of Type I and Type II MADS-domain proteins indicating their distinctive domains: MADS (M = yellow), Intervening (I = green), Keratin-like (K = blue), and Intrinsically Disordered Regions (IDR = red). Numbers below indicate the amino acid positions. Dots above the domains indicate putative phosphorylation sites. The columns to the right of each protein diagram indicate their corresponding values for PPIDR and ADS. Proteins are arranged in descending order according to their PPIDR. <strong>(B)</strong> Distribution of predicted phosphorylation sites in the N-terminal and C-terminal regions of Type I and Type II MADS-domain proteins. Whiskers indicate ±1.5 times the interquartile range (IQR) according to the Tukey method; the middle line denotes the median.</p></figcaption></figure><p>The flexibility and dimensions of IDRs depend on their amino acid composition, where the charge content and hydrophobicity are key factors. Because the composition of most IDPs includes positive and negative charges, some of their characteristics can be described by the fraction of charged residues (FCR) and net charge per residue (NCPR). Nevertheless, the best descriptors for IDP conformational properties are the FCR and the distribution of oppositely charged residues, defined by a patterning parameter named kappa (k) [<a href="#pone.0330098.ref052" class="usa-link" aria-describedby="pone.0330098.ref052">52</a>]. On average, polypeptides with low kappa-values (closer to 0) are predicted to adopt more extended conformations, while sequences with higher kappa-values (closer to 1) are expected to form more compact, hairpin-like structures [<a href="#pone.0330098.ref052" class="usa-link" aria-describedby="pone.0330098.ref052">52</a>]. The kappa-values obtained for the C-terminal of MADS-domain proteins showed that these regions in Type I and Type II proteins are prone to adopting extended conformations, likely due to the counterbalance of the interchain electrostatic interactions resulting from the more even distribution of oppositely charged residues. This result aligns with a mean NCPR close to zero and a mean FCR value at the boundary between weak and strong polyampholytes obtained for these IDRs (<a href="#pone.0330098.s007" class="usa-link">S1 Fig</a>). Although this conformational analysis was applied to the IDRs in isolation from the rest of the protein and under certain conditions, which may influence their overall conformational properties, these correlations further reinforce the potential impact of these physicochemical properties on their structural organization and dimensions.</p></section><section id="sec014"><h3 class="pmc_sec_title">The C-terminal region of Arabidopsis MADS-domain proteins has a high propensity for phosphorylation</h3>
<p>As phosphorylation is closely associated with protein disorder [<a href="#pone.0330098.ref094" class="usa-link" aria-describedby="pone.0330098.ref094">94</a>], we predicted the phosphorylation propensity of Type I and Type II MADS-domain proteins using the NetPhos algorithm. When comparing the C-terminal sequences of both MADS-domain protein types, we found that the Type II C-terminal contains a similar number of phosphorylation sites compared to those of the Type I C-terminal region (number of phosphorylation sites/ C-terminal length, permutation Wilcoxon test, Z = −1.8163, p = 0.069) (<a href="#pone.0330098.g002" class="usa-link">Fig 2B</a>). In contrast, Type I MADS-domain proteins exhibited more abundance of predicted phosphorylation sites in regions outside of their C-terminal region (<a href="#pone.0330098.g002" class="usa-link">Fig 2B</a>).</p>
<p>To further investigate the significance of the phosphorylation sites between the N-terminal and the C-terminal regions of MADS-domain proteins, we examined experimentally validated phosphorylation sites in Arabidopsis using two actively curated databases, ATHENA and EPSD. Compared with the large number of predicted sites from NetPhos, experimentally confirmed sites are less represented in Arabidopsis MADS-domain proteins. For Type I MADS-domain proteins, we found 48 experimentally verified sites in the N-terminal region and 17 sites in the C-terminal region. For Type II MADS-domain proteins, there were 18 sites in the N-terminal and 13 in the C-terminal regions (<a href="#pone.0330098.s005" class="usa-link">S5 Table</a>).</p></section><section id="sec015"><h3 class="pmc_sec_title">The disordered C-terminal is a conserved feature of MADS-domain proteins across diverse taxa</h3>
<p>Type I and Type II Arabidopsis MADS-domain proteins contain two or three domains, respectively, which we grouped into the N-terminal region (<a href="#pone.0330098.g002" class="usa-link">Fig 2A</a>). As both types have a large, disordered C-terminal region (<a href="#pone.0330098.g002" class="usa-link">Fig 2A</a>), we investigated whether this characteristic is more broadly conserved across organisms within the Eukarya domain. As a proof of concept, we only examined MADS-domain proteins from three of the four kingdoms within the Eukarya domain: Plantae, Fungi, and Animalia [<a href="#pone.0330098.ref095" class="usa-link" aria-describedby="pone.0330098.ref095">95</a>]. We selected sequences from <em>Homo sapiens</em> [<a href="#pone.0330098.ref005" class="usa-link" aria-describedby="pone.0330098.ref005">5</a>]<em>, Drosophila melanogaster</em> [<a href="#pone.0330098.ref002" class="usa-link" aria-describedby="pone.0330098.ref002">2</a>], and <em>Saccharomyces cerevisiae</em> [<a href="#pone.0330098.ref004" class="usa-link" aria-describedby="pone.0330098.ref004">4</a>] and used Arabidopsis MADS-domain protein sequences as a reference due to their extensive characterization. These sequences were compared with those of various phyla within Plantae including Chlorophytes, Charophytes, Gymnospermae, and Angiospermae. We extended this comparison to model species within Animalia (Chordata and Arthropoda) and Fungi (Ascomycota) (<a href="#pone.0330098.g003" class="usa-link">Fig 3</a> and <a href="#pone.0330098.s008" class="usa-link">S2 Fig</a>). This analysis showed that all the selected MADS-domain proteins of non-plant species contain a disordered C-terminal region and that at least for these proteins, these regions are considerably longer than in most plant MADS-domain proteins (<a href="#pone.0330098.g003" class="usa-link">Fig 3</a> and <a href="#pone.0330098.s008" class="usa-link">S2 Fig</a>). Moreover, regardless of the taxonomic group to which a species belongs, the C-terminal region consistently presents a higher level of disorder compared to the full-length protein, either with ADS or PPIDR (<a href="#pone.0330098.s009" class="usa-link">S3 Fig</a>).</p>
<figure class="fig xbox font-sm" id="pone.0330098.g003"><h4 class="obj_head">Fig 3. Intrinsically disordered regions (IDRs) in selected MADS-domain proteins from plant and non-plant species.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373214_pone.0330098.g003.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/7d0c/12373214/454773852ddc/pone.0330098.g003.jpg" loading="lazy" height="386" width="700" alt="Fig 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/pone.0330098.g003/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>The phylogenetic tree presented here is a simplified and rescaled version of the original presented in Supplementary Fig 2, with branch lengths adjusted for clarity. Deeper nodes within Type I MADS-domain protein clades exhibit low phylogenetic support, and their relationships should be interpreted with caution. In contrast, the more derived nodes, particularly among Type II proteins, show stronger phylogenetic support, consistent with previously published phylogenies. PPIDR and ADS values for each protein are shown in the columns to the right of the diagram.</p></figcaption></figure></section><section id="sec016"><h3 class="pmc_sec_title">The disordered C-terminal regions of Arabidopsis MADS-domain proteins contain conserved motifs and MoRFs</h3>
<p>To further characterize the C-terminal region of the MADS-domain proteins, we looked for conserved motifs within this region and identified three distinct motifs both in Type I and in Type II MADS-domain proteins (<a href="#pone.0330098.g004" class="usa-link">Fig 4</a>). Among the Arabidopsis MADS-domain proteins, there is a conserved pattern of motif distribution, with motifs shared in a subfamily-specific manner. Within the β subfamily of Type I MADS-domain proteins [<a href="#pone.0330098.ref024" class="usa-link" aria-describedby="pone.0330098.ref024">24</a>], 10 out of 19 proteins (<a href="#pone.0330098.g004" class="usa-link">Fig 4</a>) contain both motif 1 and motif 2, whereas seven proteins from both the γ and β subfamilies possess only motif 2 (<a href="#pone.0330098.g004" class="usa-link">Fig 4</a>). In the γ subfamily [<a href="#pone.0330098.ref024" class="usa-link" aria-describedby="pone.0330098.ref024">24</a>], 10 out of 17 proteins have motif 3 (<a href="#pone.0330098.g004" class="usa-link">Fig 4</a>). Our analysis shows that none of the α subfamily of Type I MADS-domain proteins contain any distinctive motif.</p>
<figure class="fig xbox font-sm" id="pone.0330098.g004"><h4 class="obj_head">Fig 4. Motifs identified within the IDRs of the C-terminal regions of MADS-domain proteins.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373214_pone.0330098.g004.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/7d0c/12373214/75949a3484fe/pone.0330098.g004.jpg" loading="lazy" height="684" width="751" alt="Fig 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/pone.0330098.g004/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Motifs were detected using the MEME algorithm [<a href="#pone.0330098.ref121" class="usa-link" aria-describedby="pone.0330098.ref121">121</a>] based on the C-terminal sequences of Type I and <strong>(</strong>left panel<strong>)</strong> Type II MADS-domain proteins (right panel). Type I MADS-domain proteins do not contain molecular recognition features (MoRFs; purple rectangles) near or within the identified motifs. In contrast, in Type II MADS-domain proteins, MoRFs (purple rectangles) overlap with the predicted motifs, supporting their potential functional relevance. MorRFs were predicted using the fMoRF algorithm [<a href="#pone.0330098.ref123" class="usa-link" aria-describedby="pone.0330098.ref123">123</a>]. Consensus sequences of the identified motifs shown in panels (lower panel)<strong>.</strong></p></figcaption></figure><p>Among Type II MADS-domain proteins, three motifs are shared by members of different clades. In the SOC1 clade (SOC1, XAL2, AGL42, AGL19, AGL71 and AGL72), all members contain motif 4, which is not only shared by the SOC1 clade members but also by other proteins outside this clade, including SEP3, AGL24, SEP2, AGL15, and SVP (<a href="#pone.0330098.g004" class="usa-link">Fig 4</a>, right panel). Within the FLC clade, all members shared motif 5, suggesting functional constraints and convergent acquisition (<a href="#pone.0330098.g004" class="usa-link">Fig 4</a>). Motif 6 is exclusive to CAL and AP1, which are partially redundant in floral meristem determination and belong to the AP1 clade [<a href="#pone.0330098.ref096" class="usa-link" aria-describedby="pone.0330098.ref096">96</a>]. Additionally, we selected various SOC1 orthologs and members of the FLC clade to further characterize their conserved motifs. For SOC1 orthologs, the consensus motif is predicted to span 19 residues, with a core of 10 highly conserved residues across angiosperms, except in <em>Populus trichocarpa</em> (<a href="#pone.0330098.s010" class="usa-link">S4A Fig</a>). The FLC consensus motif is 21 amino acids long and contains at least eight conserved residues , primarily located in the latter portion of the sequence [<a href="#pone.0330098.ref024" class="usa-link" aria-describedby="pone.0330098.ref024">24</a>] (<a href="#pone.0330098.s010" class="usa-link">S4B Fig</a>).</p>
<p>We also evaluated the Molecular Recognition Features (MoRFs) across the full-length MADS-domain proteins to determine whether the ubiquitous IDRs in the C-terminal coincide with any MoRFs. Since MoRFs are known to mediate molecular recognition and are proposed to facilitate specific interactions among proteins, we hypothesized that the C-terminal would contain at least one predicted MoRF. Indeed, we found that several MADS-domain proteins, regardless of their species of origin, have a MoRF within the last 10 amino acids of their C-terminal region (<a href="#pone.0330098.s011" class="usa-link">S5 Fig</a>). Moreover, several MADS-domain proteins show 1–4 amino acid MoRFs at the beginning of their N-terminal region. Intriguingly, one of these proteins is AG, which also has an IDR of at least 30 amino acids in the N-terminal region. Interestingly, the MoRFs in SOC1, AGL24, AGL42 and SVP proteins were found associated with the SOC1 motif (<a href="#pone.0330098.g004" class="usa-link">Fig 4</a>). Similarly, the FLC motif also shows predicted MoRFs within the last 5–7 amino acids, located towards the end of the motif in all the proteins of the FLC clade. In contrast, Type I MADS-domain proteins do not present any MoRFs coinciding with identified motifs (<a href="#pone.0330098.g004" class="usa-link">Fig 4</a>).</p></section><section id="sec017"><h3 class="pmc_sec_title">The disordered C-terminal regions of MADS-domain proteins contain potential activation domains</h3>
<p>Activation domains (AD) in TFs play a central role in the function of these proteins, as they constitute the recruiting sites of coactivator complexes to activate transcription [<a href="#pone.0330098.ref097" class="usa-link" aria-describedby="pone.0330098.ref097">97</a>]. A connection between IDRs and ADs has been established in several transcription factor families, highlighting the importance of structural flexibility in recognizing diverse molecular targets according to the cellular conditions [<a href="#pone.0330098.ref098" class="usa-link" aria-describedby="pone.0330098.ref098">98</a>,<a href="#pone.0330098.ref099" class="usa-link" aria-describedby="pone.0330098.ref099">99</a>]. The highly conserved presence of a C-terminal IDR in Type I and Type II MADS-domain proteins prompted us to search for potential ADs in this region. Using the plant activation domain identification (PADI) score developed by Morffy et al. (2024) [<a href="#pone.0330098.ref072" class="usa-link" aria-describedby="pone.0330098.ref072">72</a>] to analyze these C-terminal regions, we identified high-scoring regions in 34 (58.3%) of Type I MADS-domain proteins, indicating the presence of a high proportion of potential ADs, compared to the 9 (22.5%) of Type II MADS-domain proteins with a high PADI score (<a href="#pone.0330098.g005" class="usa-link">Fig 5</a> and <a href="#pone.0330098.s002" class="usa-link">S2 Table</a>).</p>
<figure class="fig xbox font-sm" id="pone.0330098.g005"><h4 class="obj_head">Fig 5. Potential activation domains in the C-terminal region of MADS-domain proteins.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/7d0c/12373214/801c933da4df/pone.0330098.g005.jpg" loading="lazy" height="1723" width="722" alt="Fig 5"></p>
<div class="p text-right font-secondary"><a href="figure/pone.0330098.g005/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>The Predictive Activation Domain Index (PADI), or scaled activation score, predicts the likelihood of putative activation domains (ADs), with fragments with a PADI score ≥1 classified as potential ADs [<a href="#pone.0330098.ref064" class="usa-link" aria-describedby="pone.0330098.ref064">64</a>]. In the graphs, dots indicate the positions of 40-amino-acid fragments predicted to containADs, while black stars mark experimentally validated ADs. Numbers along the horizontal lines show regions within the protein that are enriched in putative ADs. Colored boxes represent the defined domains in Type I and Type II MADS-domain proteins: MADS domain (yellow), I domain (green), K domain (blue), and C-terminal region (red).</p></figcaption></figure><p>Although not all MADS-domain proteins exhibited potential ADs, numerous regions with a high PADI score were found in some of them. To look for possible AD distribution patterns, we graphed the localization of the high PADI scoring regions across the C-terminal region of the MADS-domain proteins. Given the large amount of Type I proteins, we grouped them by subfamilies (Mα, Mβ, and Mγ) as described for Type I MADS-domain proteins to improve clarity in the analysis [<a href="#pone.0330098.ref024" class="usa-link" aria-describedby="pone.0330098.ref024">24</a>,<a href="#pone.0330098.ref036" class="usa-link" aria-describedby="pone.0330098.ref036">36</a>]. The C-terminal region of Type I Mα MADS-domain proteins showed a higher abundance of potential ADs (208–341). In contrast, the highest abundance of putative ADs in Type I Mβ was found from 111 to 324 amino acid residues. For the Type I Mγ subfamily, the highest AD abundance was found in two distinct sections (161–278 and 281–339) of their central region. In the case of Type II MADS-domain proteins, potential ADs were more evenly distributed across the region between 171–255 amino acid residues (<a href="#pone.0330098.g005" class="usa-link">Fig 5</a> and <a href="#pone.0330098.s002" class="usa-link">S2 Table</a>).</p>
<p>Additionally, for Type I MADS-domain proteins, we found that putative ADs are associated with motifs previously identified in one member of the Mβ subfamily (AGL81), and one member of the Mγ subfamily (PHE1) [<a href="#pone.0330098.ref036" class="usa-link" aria-describedby="pone.0330098.ref036">36</a>]. Similarly, for Type II MADS-domain proteins, some ADs are associated with different motifs identified in one member of the SOC clade (AGL19), in two members of the SQUA clade (CAL, AP1), and two members of the SEP clade (SEP2 and SEP3) [<a href="#pone.0330098.ref024" class="usa-link" aria-describedby="pone.0330098.ref024">24</a>]. The overlap of specific motifs within particular phylogenetic groups of MADS-domain proteins supports the functional significance of these potential ADs.</p></section><section id="sec018"><h3 class="pmc_sec_title">MADS-domain proteins show a propensity for liquid-liquid phase separation</h3>
<p>Some proteins can form compartments within cells where certain proteins, RNA, and metabolites concentrate to orchestrate specific cellular processes. These compartments are generated via Liquid-Liquid Phase Separation (LLPS), a process in which certain molecules concentrate to form a new liquid phase distinct from the surroundings. Some proteins are considered droplet drivers according to their likelihood of forming a droplet state via pLLPS. In this state, protein interactions can occur in different binding configurations, making IDPs common components of these structures. To explore the propensity of MADS-domain proteins to spontaneously undergo LLPS and form condensed cellular states, we used the FuzDrop server [<a href="#pone.0330098.ref089" class="usa-link" aria-describedby="pone.0330098.ref089">89</a>]. The FuzDrop algorithm defines a probability threshold value (p<sub>LLPS</sub> ≥ 0.60) to identify proteins capable of phase separation and driving droplet formation. This analysis showed that among Type I MADS-domain proteins and their C-terminal domain, the LLPS propensity is widely distributed between proteins with low or high ADS or PPIDR, finding 21 proteins with LLPS propensity (AGL103, AGL93, AGL89, AGL53, AGL74, AGL48, AGL77, AGL102, AGL60, AGL64, AGL23, AGL56, AGL92, AGL98, AGL75, AGL52, AGL76, AGL45, AGL81, AGL43, AGL29), and three more when analyzed only the C-terminal region (AGL91, AGL99, AGL86), except six of the first proteins (AGL60, AGL64, AGL98, AGL52, AGL76, AGL81) (<a href="#pone.0330098.g006" class="usa-link">Fig 6</a> and <a href="#pone.0330098.s004" class="usa-link">S4 Table</a>). Regarding Type II MADS-domain proteins, only four proteins showed LLPS propensity (AGL104, AGL79, AGL66, GOA), whose ADS are between 0.4 and 0.6 (<a href="#pone.0330098.s004" class="usa-link">S4 Table</a>). This number increased to 17 more when only the C-terminal region was analyzed (including SEP1, SEP4, AGL13, AGL15, AGL18, FCL, MAF1, AP1, AG, AGL19, AGL67, AGL24, SHP2, MAF5, AGL17, AGL72, MAF4) (<a href="#pone.0330098.g006" class="usa-link">Fig 6</a> and <a href="#pone.0330098.s004" class="usa-link">S4 Table</a>).</p>
<figure class="fig xbox font-sm" id="pone.0330098.g006"><h4 class="obj_head">Fig 6. Liquid-Liquid Phase Separation (LLPS) propensity in Type I (right panel) and Type II (left panel) MADS-domain proteins.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373214_pone.0330098.g006.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/7d0c/12373214/95e90dc0961b/pone.0330098.g006.jpg" loading="lazy" height="1014" width="709" alt="Fig 6"></a></p>
<div class="p text-right font-secondary"><a href="figure/pone.0330098.g006/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>The LLPS probability index (pLLPS) was calculated using FuzPred, while the ADS and PPIDR were derived from RIDAO predictions. The scatterplots illustrate the relationship between the pLLPS index and ADS <strong>(A)</strong> or PPIDR <strong>(B)</strong> for the full-length proteins (dark diamonds) and the C-terminal (light grey diamonds), for Type I and Type II MADS-domain proteins.</p></figcaption></figure></section></section><section id="sec019"><h2 class="pmc_sec_title">Discussion</h2>
<p>The Arabidopsis MADS-domain proteins participate in nearly all developmental processes and are also involved in many different stress responses [<a href="#pone.0330098.ref011" class="usa-link" aria-describedby="pone.0330098.ref011">11</a>,<a href="#pone.0330098.ref029" class="usa-link" aria-describedby="pone.0330098.ref029">29</a>,<a href="#pone.0330098.ref100" class="usa-link" aria-describedby="pone.0330098.ref100">100</a>]. These proteins are divided into two groups based on their phylogenetic relationship and protein domains: Type I with three domains (M, I, and C), and Type II with four domains (M, I, K, and C) [<a href="#pone.0330098.ref024" class="usa-link" aria-describedby="pone.0330098.ref024">24</a>,<a href="#pone.0330098.ref035" class="usa-link" aria-describedby="pone.0330098.ref035">35</a>,<a href="#pone.0330098.ref101" class="usa-link" aria-describedby="pone.0330098.ref101">101</a>,<a href="#pone.0330098.ref102" class="usa-link" aria-describedby="pone.0330098.ref102">102</a>].</p>
<p>In this work, we demonstrated through <em>in silico</em> analyses that the C-terminal regions of 100 Arabidopsis MADS-domain proteins listed in UniProt are enriched with IDRs (≥30 residues), with no significant differences in the number of IDRs between Type I and Type II proteins. Although we haven’t been able to find information regarding the functional characterization of these regions in Type I proteins, several examples underscore the significance of the C-terminal domain in the function of Type II MADS-domain proteins. In particular, specific phenotypes and altered protein-protein interactions in Type II MADS-domain proteins have been associated with point mutations or deletions within IDRs in the C-terminal region, highlighting their functional importance (<a href="#pone.0330098.g007" class="usa-link">Fig 7</a> and <a href="#pone.0330098.s006" class="usa-link">S6 Table</a>). For instance, in Arabidopsis, <em>Raphanus sativus, Nicotiana sylvestris</em> and <em>N. tabacum</em>, the C-terminal regions of <em>APETALA 1</em> (<em>AP1</em>) and its orthologs have been shown to mediate transcriptional activation in yeast and mammalian cells [<a href="#pone.0330098.ref101" class="usa-link" aria-describedby="pone.0330098.ref101">101</a>]. Additionally, three <em>AP1</em> loss-of-function mutants (<em>ap1–4</em>, <em>ap1–6</em> and <em>ap1–8</em>), all with mutations in the C-terminal region, exhibit different phenotypes [<a href="#pone.0330098.ref103" class="usa-link" aria-describedby="pone.0330098.ref103">103</a>]. Similarly, the C-terminal domains of GLOBOSA (GLO) and DEFICIENS (DEF) in <em>Antirrinhum majus</em> are critical for the interaction between GLO and DEF and between DEF and SQUAMOSA (SQUA) [<a href="#pone.0330098.ref103" class="usa-link" aria-describedby="pone.0330098.ref103">103</a>]. Furthermore, the C-terminal domain plays a central role in mediating interactions between MADS-domain proteins and non-MADS-domain proteins. For instance, the co-repressors SEUSS (SEU) and LEUNIG (LUG) interact with AP1 or SEP3 through their C-terminal domains [<a href="#pone.0330098.ref104" class="usa-link" aria-describedby="pone.0330098.ref104">104</a>]. In Arabidopsis, the K and C-terminal domains of AGAMOUS (AG) are also indispensable for DNA binding [<a href="#pone.0330098.ref105" class="usa-link" aria-describedby="pone.0330098.ref105">105</a>,<a href="#pone.0330098.ref106" class="usa-link" aria-describedby="pone.0330098.ref106">106</a>]. Interestingly, a small IDR in the N-terminal of AG, located before the MADS-domain (<a href="#pone.0330098.g001" class="usa-link">Fig 1</a>), is essential for its function, as constructs lacking this IDR exhibit a phenotype like an <em>AP2</em> mutant. Moreover, overexpression of AG without its C-terminal domain results in a phenotype like that of the <em>AG</em> loss-of-function mutant (<em>ag</em>), indicating that the C-terminal domain participates in AG functions [<a href="#pone.0330098.ref106" class="usa-link" aria-describedby="pone.0330098.ref106">106</a>]. For SEP3, the interaction between helices in the N-terminal domain and those in the C-terminal domain of different partner proteins creates a hydrophobic interface that facilitates dimerization [<a href="#pone.0330098.ref107" class="usa-link" aria-describedby="pone.0330098.ref107">107</a>]. This supports the hypothesis that the C-terminal domain participates in the stabilization of protein complexes. The dimerization of TFs has an important role in regulating heterodimerization, enabling dynamic temporal responses to changes in protein concentrations, among other functions [<a href="#pone.0330098.ref108" class="usa-link" aria-describedby="pone.0330098.ref108">108</a>]. Given that most MADS-domain proteins function as homo or heterodimers [<a href="#pone.0330098.ref107" class="usa-link" aria-describedby="pone.0330098.ref107">107</a>], the role of IDRs in their C-terminal region becomes of particular interest.</p>
<figure class="fig xbox font-sm" id="pone.0330098.g007"><h3 class="obj_head">Fig 7. Proposed model for the role of intrinsically disordered regions (IDRs) in the C-terminal region of MADS-domain transcription factors.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373214_pone.0330098.g007.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/7d0c/12373214/ef5df2afcb98/pone.0330098.g007.jpg" loading="lazy" height="556" width="763" alt="Fig 7"></a></p>
<div class="p text-right font-secondary"><a href="figure/pone.0330098.g007/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Top left panel – Schematic representation of Type I and Type II MADS proteins. Different functional domains are color-coded, with the C-terminal region illustrated as a red ribbon. In the C-terminal IDR are Activation Domains (ADs), Molecular Recognition Features (MoRFs) or motifs, and phosphorylation sites (see color-codes to the left of the diagrams). These features are associated with promoting protein–protein interactions both between MADS-domain proteins and with other regulatory partners. Some MADS-domain proteins display a high propensity for liquid–liquid phase separation, suggesting their involvement in the formation of biomolecular condensates. Such transcription factor condensates have been reported to enhance target gene expression and, in some cases, to recruit components of the transcriptional machinery (e.g., RNA Pol II). In some instances, deletion of the C-terminal region of MADS-domain proteins disrupts their protein–protein interactions, potentially impairing dimer formation and resulting in reduced activation of target gene activation.</p></figcaption></figure><p>The analysis conducted in this study also revealed that structural disorder in the C-terminal region of MADS-domain proteins is widely conserved across diverse taxa, including Drosophila, yeast, and human MADS-domain proteins. In humans, there are four MADS-domain proteins (MEF2A-D) mainly involved in neural development, muscle formation, heart development, and carcinogenesis. Of note, like some Arabidopsis MADS-domain proteins, the C-terminal domain of human MEF2B and MEF2D is required for transactivation [<a href="#pone.0330098.ref109" class="usa-link" aria-describedby="pone.0330098.ref109">109</a>,<a href="#pone.0330098.ref110" class="usa-link" aria-describedby="pone.0330098.ref110">110</a>]. Furthermore, phosphorylation at specific sites within the C-terminal domain of MEF2D has been shown to inhibit its transcriptional activity [<a href="#pone.0330098.ref111" class="usa-link" aria-describedby="pone.0330098.ref111">111</a>]. Consistent with these observations, our analysis also found a high frequency of predicted phosphorylation sites in the C-terminal domain of both Type I and Type II Arabidopsis MADS-domain proteins. This finding also aligns with previous observations showing a higher phosphorylation propensity in IDRs compared to ordered regions across entire proteomes [<a href="#pone.0330098.ref094" class="usa-link" aria-describedby="pone.0330098.ref094">94</a>]. It underscores the functional relevance of these post-translational modifications and highlights the prevalence of multisite phosphorylation within disorder regions. Although some of the sites predicted from the NetPhos may not be functional and/or are still waiting for experimentally testing, the relatively balanced distribution of experimentally tested sites across both regions in Type II suggests functional relevance in both the N-terminal and C-terminal regions. Furthermore, the higher representation of sites in the N-terminal regions (M, I, and K domains) may partly reflect a historical research focus on the DNA-binding function of the M domain. On the contrary, the C-terminal region is more variable and has not been throughly studied and might play an important but underappreciated role in MADS-domain protein function. The presence of multiple phosphorylation sites in IDRs has been associated with their role in modulating gradual cellular responses and mediating protein-protein interactions, emphasizing the regulatory role of phosphorylation within disordered domains [<a href="#pone.0330098.ref112" class="usa-link" aria-describedby="pone.0330098.ref112">112</a>–<a href="#pone.0330098.ref115" class="usa-link" aria-describedby="pone.0330098.ref115">115</a>]. Further investigation into the function of the predicted phosphorylation sites within the C-terminal disordered region of MADS-domain proteins will enhance our understanding of the mechanisms by which these TFs regulate gene expression.</p>
<p>By searching for conserved motifs within the disordered C-terminal region of MADS-domain proteins, we identified three distinct and specific motifs in Type I and three for Type II MADS-domain proteins. Interestingly, motif 3 identified in Type I MADS-domain proteins corresponds to a conserved region previously reported by De Bodt et al. (2003) [<a href="#pone.0330098.ref116" class="usa-link" aria-describedby="pone.0330098.ref116">116</a>]. Furthermore, some of these motifs were found in all the members of different clades, suggesting that they are associated with particular functions mediated by specific interactions and shared between the related proteins. This is the case of two motifs found in Type I MADS-domain proteins that are shared among some MADS-domain proteins of Mγ and Mβ clades [<a href="#pone.0330098.ref024" class="usa-link" aria-describedby="pone.0330098.ref024">24</a>]. Motif 4 and 5, identified in Type II MADS-domain proteins (<a href="#pone.0330098.g004" class="usa-link">Fig 4B</a>), is conserved across all SOC1-clade and FLC-clade proteins, respectively, highlighting its potential functional significance. Conducting motif-swapping experiments among SOC1 and FLC clade members and MADS-domain proteins that naturally lack this motif would provide valuable insights into its functional significance. The presence of clade-specific motifs in MADS-domain proteins may be attributed to the highly conserved interaction networks within different plant MADS-domain protein clades [<a href="#pone.0330098.ref117" class="usa-link" aria-describedby="pone.0330098.ref117">117</a>]. Interactions among MADS-domain proteins are restricted by their highly conserved domains in the complete proteins [<a href="#pone.0330098.ref117" class="usa-link" aria-describedby="pone.0330098.ref117">117</a>]. Conserved motifs within the variable C-terminal region identified in this study might also be involved in facilitating and stabilizing the interactions between MADS-domain proteins.</p>
<p>Several examples in plants demonstrate that transcriptional activation depends on the recruitment of coactivators. In MADS-domain proteins, tetramer formation increases the DNA regions available for transcriptional binding, enhancing their regulatory capacity [<a href="#pone.0330098.ref107" class="usa-link" aria-describedby="pone.0330098.ref107">107</a>,<a href="#pone.0330098.ref118" class="usa-link" aria-describedby="pone.0330098.ref118">118</a>]. Using the data from activation domains (ADs) obtained by Morffy et al. (2024) [<a href="#pone.0330098.ref072" class="usa-link" aria-describedby="pone.0330098.ref072">72</a>], we showed that a significant proportion of the potential ADs for MADS-domain TFs are in their C-terminal IDR. Moreover, we found that some of the identified ADs overlap with a conserved motif in the MADS-domain TFs of the SOC1 clade. This is particularly evident in the C-terminal of Type II MADS-domain proteins. The conserved motif in the C-terminal IDR of SEP homologs across different angiosperms [<a href="#pone.0330098.ref119" class="usa-link" aria-describedby="pone.0330098.ref119">119</a>], which corresponds to motif 4 in the SOC TFs, coincides with one of the ADs identified by Morffy et al. (2024) [<a href="#pone.0330098.ref072" class="usa-link" aria-describedby="pone.0330098.ref072">72</a>]. This overlap strongly suggests that this motif has a functional significance. We made similar observations for AP1 TFs, where ADs, characterized by the presence of acidic, proline-rich, and glutamine-rich subdomains, have been experimentally identified in their C-terminal IDRs [<a href="#pone.0330098.ref101" class="usa-link" aria-describedby="pone.0330098.ref101">101</a>].</p>
<p>Interestingly, Type I MADS-domain proteins exhibit more putative ADs than Type II and this could reflect specific roles for Type I MADS-domain proteins not only in the female gametophyte development but also in seed development [<a href="#pone.0330098.ref028" class="usa-link" aria-describedby="pone.0330098.ref028">28</a>,<a href="#pone.0330098.ref120" class="usa-link" aria-describedby="pone.0330098.ref120">120</a>–<a href="#pone.0330098.ref124" class="usa-link" aria-describedby="pone.0330098.ref124">124</a>]. Unfortunately, we were unable to find any study specifically analyzing the functional relevance of the C-terminal domains or any other regions of these proteins. This outcome adds further interest to the findings presented here and encourages future research into the role of the potential ADs in mediating the association between MADS-domain proteins and their co-regulators.</p>
<p>Finally, our analysis revealed that some MADS-domain proteins have a propensity to undergo liquid-liquid phase separation (LLPS). Nevertheless, even among those with high disorder scores, particularly within Type II MADS-domain proteins, not all appear capable of undergoing LLPS. This discrepancy may be influenced by the fact that our analysis was based solely on protein primary structures, without accounting for possible posttranslational modifications such as phosphorylation. As we show, MADS-domain TFs could be phosphorylated at multiple sites. These posttranslational modifications, whether occurring at one or several sites, could be implicated in the promotion of LLPS, with the extent of phase separation likely dependent on specific cellular conditions. Additionally, the LLPS propensity obtained in this analysis is in agreement with findings showing that the capacity to drive LLPS is not determined solely by the disordered nature of the sequence. Instead, it depends on specific sequence features, such as the distribution and patterning of aromatic and charged residues. These sequence-encoded patterns are essential for enabling the multivalent interactions required for the formation of biomolecular condensates [<a href="#pone.0330098.ref125" class="usa-link" aria-describedby="pone.0330098.ref125">125</a>]. When only the C-terminal region of MADS-domain proteins is analyzed for LLPS propensity, the effect of intrinsic disorder shifts the scores upward. A greater number of proteins present LLPS scores above 0.6 compared to analyses of the full-length proteins, supporting the contribution of IDRs to protein condensation propensity. Furthermore, some studies have suggested that ADs can drive TF phase separation, leading to the formation of transcriptional condensates associated with chromatin [<a href="#pone.0330098.ref126" class="usa-link" aria-describedby="pone.0330098.ref126">126</a>–<a href="#pone.0330098.ref128" class="usa-link" aria-describedby="pone.0330098.ref128">128</a>]. However, recent findings indicate that this mechanism is not universal, as the recruitment of activators can also occur independently of phase separation. Important events that enhance transcriptional activation include the multivalent interactions mediated by the ADs, which increase the residence time of TFs on chromatin and thereby promote the recruitment of coactivators [<a href="#pone.0330098.ref097" class="usa-link" aria-describedby="pone.0330098.ref097">97</a>].</p>
<p>This study highlights common characteristics shared by MADS-domain proteins, not only in plants but also across organisms from other domains of life. Notably, the presence of a disordered C-terminal region in Type I and Type II MADS-domain TFs stands out. The functional significance of this region is strongly supported by the identification of several potential ADs and phosphorylation sites. Furthermore, we identified not only putative protein-protein interaction sites within this region but also conserved motifs specific to evolutionary related MADS-domain proteins, further supporting their role in the transcriptional regulatory function of these TFs.</p>
<p>The remarkable conservation of the structurally disordered C-terminal region in MADS-domain TFs suggests specific biological functions for this region. Some of these may be associated with the presence of conserved motifs and/or phosphorylation sites. However, these elements correspond to short segments within a broader region that, based on primary sequence alignments, do not appear to be under strong evolutionary constraint. To date, the persistence of IDRs across the proteomes of all analyzed organisms remains an open question. Although some evolutionary approaches have attempted to identify conserved molecular features (i.e., NCPR, kappa, FCR, etc.) in the amino acid sequences of IDRs [<a href="#pone.0330098.ref129" class="usa-link" aria-describedby="pone.0330098.ref129">129</a>], the findings suggest that, while certain molecular features are linked to known functions in yeast IDRs and may reflect a mechanism of IDR evolution, this pattern does not appear to extend to multicellular organisms like <em>Drosophila</em> [<a href="#pone.0330098.ref130" class="usa-link" aria-describedby="pone.0330098.ref130">130</a>]. These observations show that although IDRs may follow unique patterns of amino acid substitutions, intrinsic disorder itself is subjected to dynamic evolutionary processes, shaped by more complex evolutionary constraints across evolving properties of different domains of life.</p>
<p>Considering the well-established role of MADS-domain TFs in regulating diverse developmental processes and stress responses, the conservation of a structurally disordered C-terminal region across all family members, along with the presence conserved motifs, potential activation domains and phosphorylation sites, suggest a shared regulatory mechanism. Based on these findings, we proposed a mechanistic model describing the functional role of specific structural elements within the C-terminal domain of both Type I and Type II MADS-domain TFs. The structural flexibility provided by the IDRs in the C-terminal domain, combined with the presence of MoRFs, ADs, and a high phosphorylation propensity, points to a regulatory role in modulating the MADS-domain TF activity within transcriptional complexes (<a href="#pone.0330098.g007" class="usa-link">Fig 7</a>, <a href="#pone.0330098.s006" class="usa-link">S6 Table</a>). In this model, these IDRs confer both structural flexibility and modularity, facilitating the formation of dynamic protein complexes. This may occur either through a high propensity for liquid-liquid phase separation (LLPS) or by promoting transient interactions with other proteins, both of which support the formation of transcriptional condensates (<a href="#pone.0330098.g007" class="usa-link">Fig 7</a>). In the present study, we identified MoRFs and ADs within the C-terminal IDRs, where MoRFs likely mediate specific, transient interactions with regulatory proteins or other MADS-domain TFs in concert with ADs, these elements may modulate the assembly and stability of transcriptional complexes. Previous reports suggest that the formation of condensates enriched in transcription factors enables fine-tuned regulation of transcriptional activity, particularly in response to physiological or developmental cues [<a href="#pone.0330098.ref131" class="usa-link" aria-describedby="pone.0330098.ref131">131</a>,<a href="#pone.0330098.ref132" class="usa-link" aria-describedby="pone.0330098.ref132">132</a>]. Within such condensates, a variety of proteins can interact with TF IDRs through their ADs and LLPS-driven mechanisms further supporting the functional significance of the C-terminal IDRs in MADS-domain TFs.</p>
<p>Overall, this study provides valuable insights for a deeper understanding of the relationship between protein structure and function for MADS-domain TFs. We believe this information will encourage and support further experimental studies by researchers working on MADS-domain proteins in diverse biological systems, especially to investigate the functional relevance of these conserved regions.</p></section><section id="sec020"><h2 class="pmc_sec_title">Supporting information</h2>
<section class="sm xbox font-sm" id="pone.0330098.s001"><div class="caption p">
<span>S1 Table. MADS-domain proteins of Arabidopsis, other plant species, and non-plant species used in this study.</span><p>(DOCX)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12373214/bin/pone.0330098.s001.docx" data-ga-action="click_feat_suppl" class="usa-link">pone.0330098.s001.docx</a><sup> (36KB, docx) </sup>
</div></div></section><section class="sm xbox font-sm" id="pone.0330098.s002"><div class="caption p">
<span>S2 Table. MADS-domain proteins analyzed by Morffy et al. (2024) [<a href="#pone.0330098.ref064" class="usa-link" aria-describedby="pone.0330098.ref064">64</a>].</span><p>AD = activation domain. AD, Maybe and Not AD definitions are given based on the PADI (plant activation domain identification). PADI score ≥1 and mean disorder &gt;0.5 are defined as “AD”; PADI score ≥1 and mean disorder de ≤ 0.5 are defined as “Maybe”; PADI &lt;1 are defined as “Not AD”.</p>
<p>(DOCX)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12373214/bin/pone.0330098.s002.docx" data-ga-action="click_feat_suppl" class="usa-link">pone.0330098.s002.docx</a><sup> (25.3KB, docx) </sup>
</div></div></section><section class="sm xbox font-sm" id="pone.0330098.s003"><div class="caption p">
<span>S3 Table. R packages used in this research.</span><p>These packages are available from CRAN (<a href="https://CRAN.R-project.org/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://CRAN.R-project.org/</a>) or Bioconductor (Huber et al., 2015).</p>
<p>(DOCX)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12373214/bin/pone.0330098.s003.docx" data-ga-action="click_feat_suppl" class="usa-link">pone.0330098.s003.docx</a><sup> (18.8KB, docx) </sup>
</div></div></section><section class="sm xbox font-sm" id="pone.0330098.s004"><div class="caption p">
<span>S4 Table. Arabidopsis MADS-domain proteins, disordered regions and phosphorylation sites.</span><p>(XLSX)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12373214/bin/pone.0330098.s004.xlsx" data-ga-action="click_feat_suppl" class="usa-link">pone.0330098.s004.xlsx</a><sup> (24.3KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="pone.0330098.s005"><div class="caption p">
<span>S5 Table. Experimentally tested phosphorylation sites obtained from two different databases Athena (<a href="https://athena.proteomics.wzw.tum.de/master_arabidopsisshiny/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://athena.proteomics.wzw.tum.de/master_arabidopsisshiny/</a>), and EPSD (<a href="https://epsd.biocuckoo.cn/Browse.php" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://epsd.biocuckoo.cn/Browse.php</a>).</span><p>Sites were mapped on domains, according to UniProt and Liu et al., 2021 definition. See main text for a detailed description.</p>
<p>(DOCX)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12373214/bin/pone.0330098.s005.docx" data-ga-action="click_feat_suppl" class="usa-link">pone.0330098.s005.docx</a><sup> (139.7KB, docx) </sup>
</div></div></section><section class="sm xbox font-sm" id="pone.0330098.s006"><div class="caption p">
<span>S6 Table. Experimental evidence supporting the functional relevance of the C-terminal region in MADS-domain proteins across different species.</span><p>(DOCX)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12373214/bin/pone.0330098.s006.docx" data-ga-action="click_feat_suppl" class="usa-link">pone.0330098.s006.docx</a><sup> (140.1KB, docx) </sup>
</div></div></section><section class="sm xbox font-sm" id="pone.0330098.s007"><div class="caption p">
<span>S1 Fig. Physicochemical properties of Type I and Type II MADS-domain proteins.</span><p><strong>(A)</strong> Net charge per residue (NCPR), <strong>(B)</strong> Charge distribution (Kappa), and <strong>(C)</strong> Fraction of Charged Residues (FCR). Whiskers indicate ±1.5*IQR based on Tukey test. The middle line represents the median.</p>
<p>(TIF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12373214/bin/pone.0330098.s007.tif" data-ga-action="click_feat_suppl" class="usa-link">pone.0330098.s007.tif</a><sup> (75.5KB, tif) </sup>
</div></div></section><section class="sm xbox font-sm" id="pone.0330098.s008"><div class="caption p">
<span>S2 Fig. A complete Maximum Likelihood (ML) phylogeny of Arabidopsis MADS-domain proteins including MADS-domain proteins of other plant species and non-plant species.</span><p><em>Oryza sativa japonica</em> (<em>Os</em>)<em>, Solanum dulcamara</em> (<em>SD</em>)<em>, Mangifera indica</em> (<em>MI</em>)<em>, Populus trichocarpa</em> (<em>PT</em>)<em>, Amborella trichopoda</em> (<em>AMB</em>)<em>, Pinus radiata</em> (<em>Prad</em>)<em>, Selaginella mollendorffii</em> (<em>SM</em>)<em>, Chara braunii</em> (<em>CB</em>)<em>, Chlorella dessiccata</em> (<em>CD</em>)<em>, Saccharomyces cerevisiae</em> (<em>SC</em>)<em>, Drosophila melanogaster</em> (<em>DM</em>)<em>, and Homo sapiens</em> (<em>HS</em>). Yellow-shaded branches cover Type I MADS grouped with SRF-like and MEF-like MADS-domain proteins. Purple-shaded branches cover most Type II MADS-domain proteins. Numbers adjacent to nodes represent bootstrap support. Orange dots at particular nodes indicate the putative ancestral motif for that specific clade.</p>
<p>(TIF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12373214/bin/pone.0330098.s008.tif" data-ga-action="click_feat_suppl" class="usa-link">pone.0330098.s008.tif</a><sup> (3.2MB, tif) </sup>
</div></div></section><section class="sm xbox font-sm" id="pone.0330098.s009"><div class="caption p">
<span>S3 Fig. Structural disorder in the C-terminal region and the full-length proteins of MADS-domain transcription factors from different organisms.</span><p>Boxplots showing the ADS and PPIDR values for the C-terminal region and the full-length proteins from: <strong>(A)</strong> Plant species analysed in this study, excluding Arabidopsis. <strong>(B)</strong>
<em>Saccharomyces cerevisiae</em>, <strong>(C)</strong>
<em>Homo sapiens</em>, and <strong>(D)</strong>
<em>Drosophila melanogaster</em>. Whiskers indicate ±1.5*IQR according to the Tukey test. The middle line represents the median.</p>
<p>(TIF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12373214/bin/pone.0330098.s009.tif" data-ga-action="click_feat_suppl" class="usa-link">pone.0330098.s009.tif</a><sup> (372.2KB, tif) </sup>
</div></div></section><section class="sm xbox font-sm" id="pone.0330098.s010"><div class="caption p">
<span>S4 Fig. Distinctive motifs in MADS-domain proteins of SOC1 (A) and FLC (B) clades.</span><p>The phylogenetic tree was derived from the complete MADS-domain protein tree shown in Supplementary Fig 2. To enhance the visualization of phylogenetic relationships among the proteins, branch lengths were rescaled and truncated. The conserved amino acid residues of the SOC1-motif and FLC-motif are highlighted in bold within the consensus motif.</p>
<p>(TIF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12373214/bin/pone.0330098.s010.tif" data-ga-action="click_feat_suppl" class="usa-link">pone.0330098.s010.tif</a><sup> (217.5KB, tif) </sup>
</div></div></section><section class="sm xbox font-sm" id="pone.0330098.s011"><div class="caption p">
<span>S5 Fig. Identified MoRFs in MADS-domain proteins.</span><p>MADS-domain protein sequences from Arabidopsis, other plants, and non-plant organisms are shown, with predicted MoRFs highlighted in yellow Putative MoRFs were identified using the fmoRFpred algorithm [<a href="#pone.0330098.ref123" class="usa-link" aria-describedby="pone.0330098.ref123">123</a>], based on the analysis of full-length protein sequences.</p>
<p>(TIF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12373214/bin/pone.0330098.s011.tif" data-ga-action="click_feat_suppl" class="usa-link">pone.0330098.s011.tif</a><sup> (1.1MB, tif) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgments</h2>
<p>The first authors wish to thank Consejo Nacional de Humanidades, Ciencias y Tecnología (CONAHCyT) for the postdoctoral scholarships granted (E.R.A., CVU number 413896; T.N.R., CVU 501149).</p></section><section id="notes1"><h2 class="pmc_sec_title">Data Availability</h2>
<p>All relevant data are within the paper and its <a href="#sec020" class="usa-link">Supporting information</a> files.</p></section><section id="funding-statement1" lang="en"><h2 class="pmc_sec_title">Funding Statement</h2>
<p>This work was partially financed by the projects PAPIIT No. IN213524 from Universidad Nacional Autónoma de México (UNAM), and project No. CBF2023- 2024-1002 from the Consejo Nacional de Humanidades, Ciencias y Tecnologías (CONAHCyT) granted to A.G.A.; and projects No. CF-2019/252952 and CF2023-I-503 from the Consejo Nacional de Humanidades, Ciencias y Tecnologías (CONAHCyT) granted to A.A.C.</p></section><section id="ref-list1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="ref-list1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="pone.0330098.ref001">
<span class="label">1.</span><cite>Wells CL, Pigliucci M. Adaptive phenotypic plasticity: the case of heterophylly in aquatic plants. Perspect Plant Ecol Evol Syst. 2000;3:1–18.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Perspect%20Plant%20Ecol%20Evol%20Syst&amp;title=Adaptive%20phenotypic%20plasticity:%20the%20case%20of%20heterophylly%20in%20aquatic%20plants&amp;author=CL%20Wells&amp;author=M%20Pigliucci&amp;volume=3&amp;publication_year=2000&amp;pages=1-18&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330098.ref002">
<span class="label">2.</span><cite>Sultan SE. Phenotypic plasticity for plant development, function and life history. Trends Plant Sci. 2000;5(12):537–42. doi: 10.1016/s1360-1385(00)01797-0

</cite> [<a href="https://doi.org/10.1016/s1360-1385(00)01797-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11120476/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Trends%20Plant%20Sci&amp;title=Phenotypic%20plasticity%20for%20plant%20development,%20function%20and%20life%20history&amp;author=SE%20Sultan&amp;volume=5&amp;issue=12&amp;publication_year=2000&amp;pages=537-42&amp;pmid=11120476&amp;doi=10.1016/s1360-1385(00)01797-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330098.ref003">
<span class="label">3.</span><cite>Spitz F, Furlong EEM. Transcription factors: from enhancer binding to developmental control. Nat Rev Genet. 2012;13(9):613–26. doi: 10.1038/nrg3207

</cite> [<a href="https://doi.org/10.1038/nrg3207" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22868264/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Genet&amp;title=Transcription%20factors:%20from%20enhancer%20binding%20to%20developmental%20control&amp;author=F%20Spitz&amp;author=EEM%20Furlong&amp;volume=13&amp;issue=9&amp;publication_year=2012&amp;pages=613-26&amp;pmid=22868264&amp;doi=10.1038/nrg3207&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330098.ref004">
<span class="label">4.</span><cite>Schwechheimer C, Bevan M. The regulation of transcription factor activity in plants. Trends Plant Sci. 1998;3:378–83.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Trends%20Plant%20Sci&amp;title=The%20regulation%20of%20transcription%20factor%20activity%20in%20plants&amp;author=C%20Schwechheimer&amp;author=M%20Bevan&amp;volume=3&amp;publication_year=1998&amp;pages=378-83&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330098.ref005">
<span class="label">5.</span><cite>Mosa KA, Ismail A, Helmy M. Plant Stress Tolerance. Springer International Publishing. 2017.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=Plant%20Stress%20Tolerance&amp;author=KA%20Mosa&amp;author=A%20Ismail&amp;author=M%20Helmy&amp;publication_year=2017&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330098.ref006">
<span class="label">6.</span><cite>Hoang XLT, Nhi DNH, Thu NBA, Thao NP, Tran LSP. Transcription factors and their roles in signal transduction in plants under abiotic stresses. Curr Genomics. 2017;18:483–97.
</cite> [<a href="https://doi.org/10.2174/1389202918666170227150057" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5684650/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29204078/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Curr%20Genomics&amp;title=Transcription%20factors%20and%20their%20roles%20in%20signal%20transduction%20in%20plants%20under%20abiotic%20stresses&amp;author=XLT%20Hoang&amp;author=DNH%20Nhi&amp;author=NBA%20Thu&amp;author=NP%20Thao&amp;author=LSP%20Tran&amp;volume=18&amp;publication_year=2017&amp;pages=483-97&amp;pmid=29204078&amp;doi=10.2174/1389202918666170227150057&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330098.ref007">
<span class="label">7.</span><cite>Kaufmann K, Muiño JM, Jauregui R, Airoldi CA, Smaczniak C, Krajewski P, et al. Target genes of the MADS transcription factor SEPALLATA3: integration of developmental and hormonal pathways in the Arabidopsis flower. PLoS Biol. 2009;7(4):e1000090. doi: 10.1371/journal.pbio.1000090

</cite> [<a href="https://doi.org/10.1371/journal.pbio.1000090" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2671559/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19385720/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20Biol&amp;title=Target%20genes%20of%20the%20MADS%20transcription%20factor%20SEPALLATA3:%20integration%20of%20developmental%20and%20hormonal%20pathways%20in%20the%20Arabidopsis%20flower&amp;author=K%20Kaufmann&amp;author=JM%20Mui%C3%B1o&amp;author=R%20Jauregui&amp;author=CA%20Airoldi&amp;author=C%20Smaczniak&amp;volume=7&amp;issue=4&amp;publication_year=2009&amp;pmid=19385720&amp;doi=10.1371/journal.pbio.1000090&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330098.ref008">
<span class="label">8.</span><cite>Yu L-H, Miao Z-Q, Qi G-F, Wu J, Cai X-T, Mao J-L, et al. MADS-box transcription factor AGL21 regulates lateral root development and responds to multiple external and physiological signals. Mol Plant. 2014;7(11):1653–69. doi: 10.1093/mp/ssu088

</cite> [<a href="https://doi.org/10.1093/mp/ssu088" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4228986/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25122697/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol%20Plant&amp;title=MADS-box%20transcription%20factor%20AGL21%20regulates%20lateral%20root%20development%20and%20responds%20to%20multiple%20external%20and%20physiological%20signals&amp;author=L-H%20Yu&amp;author=Z-Q%20Miao&amp;author=G-F%20Qi&amp;author=J%20Wu&amp;author=X-T%20Cai&amp;volume=7&amp;issue=11&amp;publication_year=2014&amp;pages=1653-69&amp;pmid=25122697&amp;doi=10.1093/mp/ssu088&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330098.ref009">
<span class="label">9.</span><cite>Babu MM, Luscombe NM, Aravind L, Gerstein M, Teichmann SA. Structure and evolution of transcriptional regulatory networks. Curr Opin Struct Biol. 2004;14(3):283–91. doi: 10.1016/j.sbi.2004.05.004

</cite> [<a href="https://doi.org/10.1016/j.sbi.2004.05.004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15193307/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Curr%20Opin%20Struct%20Biol&amp;title=Structure%20and%20evolution%20of%20transcriptional%20regulatory%20networks&amp;author=MM%20Babu&amp;author=NM%20Luscombe&amp;author=L%20Aravind&amp;author=M%20Gerstein&amp;author=SA%20Teichmann&amp;volume=14&amp;issue=3&amp;publication_year=2004&amp;pages=283-91&amp;pmid=15193307&amp;doi=10.1016/j.sbi.2004.05.004&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330098.ref010">
<span class="label">10.</span><cite>Melzer R, Theissen G. MADS and more: transcription factors that shape the plant. Methods Mol Biol. 2011;754:3–18. doi: 10.1007/978-1-61779-154-3_1

</cite> [<a href="https://doi.org/10.1007/978-1-61779-154-3_1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21720944/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Methods%20Mol%20Biol&amp;title=MADS%20and%20more:%20transcription%20factors%20that%20shape%20the%20plant&amp;author=R%20Melzer&amp;author=G%20Theissen&amp;volume=754&amp;publication_year=2011&amp;pages=3-18&amp;pmid=21720944&amp;doi=10.1007/978-1-61779-154-3_1&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330098.ref011">
<span class="label">11.</span><cite>Castelán-Muñoz N, Herrera J, Cajero-Sánchez W, Arrizubieta M, Trejo C, García-Ponce B. MADS-Box genes are key components of genetic regulatory networks involved in abiotic stress and plastic developmental responses in plants. Front Plant Sci. 2019;10:853.
</cite> [<a href="https://doi.org/10.3389/fpls.2019.00853" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6636334/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31354752/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Front%20Plant%20Sci&amp;title=MADS-Box%20genes%20are%20key%20components%20of%20genetic%20regulatory%20networks%20involved%20in%20abiotic%20stress%20and%20plastic%20developmental%20responses%20in%20plants&amp;author=N%20Castel%C3%A1n-Mu%C3%B1oz&amp;author=J%20Herrera&amp;author=W%20Cajero-S%C3%A1nchez&amp;author=M%20Arrizubieta&amp;author=C%20Trejo&amp;volume=10&amp;publication_year=2019&amp;pages=853&amp;pmid=31354752&amp;doi=10.3389/fpls.2019.00853&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330098.ref012">
<span class="label">12.</span><cite>Schwarz-Sommer Z, Huijser P, Nacken W, Saedler H, Sommer H. Genetic Control of Flower Development by Homeotic Genes in Antirrhinum majus. Science. 1990;250(4983):931–6. doi: 10.1126/science.250.4983.931

</cite> [<a href="https://doi.org/10.1126/science.250.4983.931" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17746916/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Genetic%20Control%20of%20Flower%20Development%20by%20Homeotic%20Genes%20in%20Antirrhinum%20majus&amp;author=Z%20Schwarz-Sommer&amp;author=P%20Huijser&amp;author=W%20Nacken&amp;author=H%20Saedler&amp;author=H%20Sommer&amp;volume=250&amp;issue=4983&amp;publication_year=1990&amp;pages=931-6&amp;pmid=17746916&amp;doi=10.1126/science.250.4983.931&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330098.ref013">
<span class="label">13.</span><cite>Wu W, de Folter S, Shen X, Zhang W, Tao S. Vertebrate paralogous MEF2 genes: origin, conservation, and evolution. PLoS One. 2011;6(3):e17334. doi: 10.1371/journal.pone.0017334

</cite> [<a href="https://doi.org/10.1371/journal.pone.0017334" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3048864/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21394201/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20One&amp;title=Vertebrate%20paralogous%20MEF2%20genes:%20origin,%20conservation,%20and%20evolution&amp;author=W%20Wu&amp;author=S%20de%20Folter&amp;author=X%20Shen&amp;author=W%20Zhang&amp;author=S%20Tao&amp;volume=6&amp;issue=3&amp;publication_year=2011&amp;pmid=21394201&amp;doi=10.1371/journal.pone.0017334&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330098.ref014">
<span class="label">14.</span><cite>Acton TB, Zhong H, Vershon AK. DNA-binding specificity of Mcm1: operator mutations that alter DNA-bending and transcriptional activities by a MADS box protein. Mol Cell Biol. 1997;17(4):1881–9. doi: 10.1128/MCB.17.4.1881

</cite> [<a href="https://doi.org/10.1128/MCB.17.4.1881" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC232035/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9121436/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol%20Cell%20Biol&amp;title=DNA-binding%20specificity%20of%20Mcm1:%20operator%20mutations%20that%20alter%20DNA-bending%20and%20transcriptional%20activities%20by%20a%20MADS%20box%20protein&amp;author=TB%20Acton&amp;author=H%20Zhong&amp;author=AK%20Vershon&amp;volume=17&amp;issue=4&amp;publication_year=1997&amp;pages=1881-9&amp;pmid=9121436&amp;doi=10.1128/MCB.17.4.1881&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330098.ref015">
<span class="label">15.</span><cite>Dichoso D, Brodigan T, Chwoe KY, Lee JS, Llacer R, Park M, et al. The MADS-Box factor CeMEF2 is not essential for Caenorhabditis elegans myogenesis and development. Dev Biol. 2000;223(2):431–40. doi: 10.1006/dbio.2000.9758

</cite> [<a href="https://doi.org/10.1006/dbio.2000.9758" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10882527/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Dev%20Biol&amp;title=The%20MADS-Box%20factor%20CeMEF2%20is%20not%20essential%20for%20Caenorhabditis%20elegans%20myogenesis%20and%20development&amp;author=D%20Dichoso&amp;author=T%20Brodigan&amp;author=KY%20Chwoe&amp;author=JS%20Lee&amp;author=R%20Llacer&amp;volume=223&amp;issue=2&amp;publication_year=2000&amp;pages=431-40&amp;pmid=10882527&amp;doi=10.1006/dbio.2000.9758&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330098.ref016">
<span class="label">16.</span><cite>Lilly B, Galewsky S, Firulli AB, Schulz RA, Olson EN. D-MEF2: a MADS-box transcription factor expressed in differentiating mesoderm and muscle cell lineages during Drosophila embryogenesis. Proc Natl Acad Sci. 1994;91:5662–6.
</cite> [<a href="https://doi.org/10.1073/pnas.91.12.5662" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC44056/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/8202544/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci&amp;title=D-MEF2:%20a%20MADS-box%20transcription%20factor%20expressed%20in%20differentiating%20mesoderm%20and%20muscle%20cell%20lineages%20during%20Drosophila%20embryogenesis&amp;author=B%20Lilly&amp;author=S%20Galewsky&amp;author=AB%20Firulli&amp;author=RA%20Schulz&amp;author=EN%20Olson&amp;volume=91&amp;publication_year=1994&amp;pages=5662-6&amp;pmid=8202544&amp;doi=10.1073/pnas.91.12.5662&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330098.ref017">
<span class="label">17.</span><cite>Pollock R, Treisman R. Human SRF-related proteins: DNA-binding properties and potential regulatory targets. Genes Dev. 1991;5(12A):2327–41. doi: 10.1101/gad.5.12a.2327

</cite> [<a href="https://doi.org/10.1101/gad.5.12a.2327" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/1748287/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genes%20Dev&amp;title=Human%20SRF-related%20proteins:%20DNA-binding%20properties%20and%20potential%20regulatory%20targets&amp;author=R%20Pollock&amp;author=R%20Treisman&amp;volume=5&amp;publication_year=1991&amp;pages=2327-41&amp;pmid=1748287&amp;doi=10.1101/gad.5.12a.2327&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330098.ref018">
<span class="label">18.</span><cite>Alvarez-Buylla ER, Liljegren SJ, Pelaz S, Gold SE, Burgeff C, Ditta GS, et al. MADS-box gene evolution beyond flowers: expression in pollen, endosperm, guard cells, roots and trichomes. Plant J. 2000;24(4):457–66. doi: 10.1046/j.1365-313x.2000.00891.x

</cite> [<a href="https://doi.org/10.1046/j.1365-313x.2000.00891.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11115127/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20J&amp;title=MADS-box%20gene%20evolution%20beyond%20flowers:%20expression%20in%20pollen,%20endosperm,%20guard%20cells,%20roots%20and%20trichomes&amp;author=ER%20Alvarez-Buylla&amp;author=SJ%20Liljegren&amp;author=S%20Pelaz&amp;author=SE%20Gold&amp;author=C%20Burgeff&amp;volume=24&amp;issue=4&amp;publication_year=2000&amp;pages=457-66&amp;pmid=11115127&amp;doi=10.1046/j.1365-313x.2000.00891.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330098.ref019">
<span class="label">19.</span><cite>Becker A, Theissen G. The major clades of MADS-box genes and their role in the development and evolution of flowering plants. Mol Phylogenet Evol. 2003;29(3):464–89. doi: 10.1016/s1055-7903(03)00207-0

</cite> [<a href="https://doi.org/10.1016/s1055-7903(03)00207-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/14615187/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol%20Phylogenet%20Evol&amp;title=The%20major%20clades%20of%20MADS-box%20genes%20and%20their%20role%20in%20the%20development%20and%20evolution%20of%20flowering%20plants&amp;author=A%20Becker&amp;author=G%20Theissen&amp;volume=29&amp;issue=3&amp;publication_year=2003&amp;pages=464-89&amp;pmid=14615187&amp;doi=10.1016/s1055-7903(03)00207-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330098.ref020">
<span class="label">20.</span><cite>Meng D, Cao Y, Chen T, Abdullah M, Jin Q, Fan H, et al. Evolution and functional divergence of MADS-box genes in Pyrus. Sci Rep. 2019;9(1):1266. doi: 10.1038/s41598-018-37897-6

</cite> [<a href="https://doi.org/10.1038/s41598-018-37897-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6362034/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30718750/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sci%20Rep&amp;title=Evolution%20and%20functional%20divergence%20of%20MADS-box%20genes%20in%20Pyrus&amp;author=D%20Meng&amp;author=Y%20Cao&amp;author=T%20Chen&amp;author=M%20Abdullah&amp;author=Q%20Jin&amp;volume=9&amp;issue=1&amp;publication_year=2019&amp;pages=1266&amp;pmid=30718750&amp;doi=10.1038/s41598-018-37897-6&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330098.ref021">
<span class="label">21.</span><cite>Shore P, Sharrocks AD. The MADS-box family of transcription factors. Eur J Biochem. 1995;229(1):1–13. doi: 10.1111/j.1432-1033.1995.tb20430.x

</cite> [<a href="https://doi.org/10.1111/j.1432-1033.1995.tb20430.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/7744019/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Biochem&amp;title=The%20MADS-box%20family%20of%20transcription%20factors&amp;author=P%20Shore&amp;author=AD%20Sharrocks&amp;volume=229&amp;issue=1&amp;publication_year=1995&amp;pages=1-13&amp;pmid=7744019&amp;doi=10.1111/j.1432-1033.1995.tb20430.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330098.ref022">
<span class="label">22.</span><cite>Qiu Y, Li Z, Walther D, Köhler C. Updated Phylogeny and Protein Structure Predictions Revise the Hypothesis on the Origin of MADS-box Transcription Factors in Land Plants. Mol Biol Evol. 2023;40(9):msad194. doi: 10.1093/molbev/msad194

</cite> [<a href="https://doi.org/10.1093/molbev/msad194" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10484287/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37652031/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol%20Biol%20Evol&amp;title=Updated%20Phylogeny%20and%20Protein%20Structure%20Predictions%20Revise%20the%20Hypothesis%20on%20the%20Origin%20of%20MADS-box%20Transcription%20Factors%20in%20Land%20Plants&amp;author=Y%20Qiu&amp;author=Z%20Li&amp;author=D%20Walther&amp;author=C%20K%C3%B6hler&amp;volume=40&amp;issue=9&amp;publication_year=2023&amp;pmid=37652031&amp;doi=10.1093/molbev/msad194&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330098.ref023">
<span class="label">23.</span><cite>Alvarez-Buylla ER, Pelaz S, Liljegren SJ, Gold SE, Burgeff C, Ditta GS, et al. An ancestral MADS-box gene duplication occurred before the divergence of plants and animals. Proc Natl Acad Sci U S A. 2000;97(10):5328–33. doi: 10.1073/pnas.97.10.5328

</cite> [<a href="https://doi.org/10.1073/pnas.97.10.5328" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC25828/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10805792/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&amp;title=An%20ancestral%20MADS-box%20gene%20duplication%20occurred%20before%20the%20divergence%20of%20plants%20and%20animals&amp;author=ER%20Alvarez-Buylla&amp;author=S%20Pelaz&amp;author=SJ%20Liljegren&amp;author=SE%20Gold&amp;author=C%20Burgeff&amp;volume=97&amp;issue=10&amp;publication_year=2000&amp;pages=5328-33&amp;pmid=10805792&amp;doi=10.1073/pnas.97.10.5328&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330098.ref024">
<span class="label">24.</span><cite>Smaczniak C, Immink RGH, Angenent GC, Kaufmann K. Developmental and evolutionary diversity of plant MADS-domain factors: insights from recent studies. Development. 2012;139(17):3081–98. doi: 10.1242/dev.074674

</cite> [<a href="https://doi.org/10.1242/dev.074674" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22872082/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Development&amp;title=Developmental%20and%20evolutionary%20diversity%20of%20plant%20MADS-domain%20factors:%20insights%20from%20recent%20studies&amp;author=C%20Smaczniak&amp;author=RGH%20Immink&amp;author=GC%20Angenent&amp;author=K%20Kaufmann&amp;volume=139&amp;issue=17&amp;publication_year=2012&amp;pages=3081-98&amp;pmid=22872082&amp;doi=10.1242/dev.074674&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330098.ref025">
<span class="label">25.</span><cite>Chen X, Gao B, Ponnusamy M, Lin Z, Liu J. MEF2 signaling and human diseases. Oncotarget. 2017;8(67):112152–65. doi: 10.18632/oncotarget.22899

</cite> [<a href="https://doi.org/10.18632/oncotarget.22899" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5762387/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29340119/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Oncotarget&amp;title=MEF2%20signaling%20and%20human%20diseases&amp;author=X%20Chen&amp;author=B%20Gao&amp;author=M%20Ponnusamy&amp;author=Z%20Lin&amp;author=J%20Liu&amp;volume=8&amp;issue=67&amp;publication_year=2017&amp;pages=112152-65&amp;pmid=29340119&amp;doi=10.18632/oncotarget.22899&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330098.ref026">
<span class="label">26.</span><cite>Mead J, Bruning AR, Gill MK, Steiner AM, Acton TB, Vershon AK. Interactions of the Mcm1 MADS box protein with cofactors that regulate mating in yeast. Mol Cell Biol. 2002;22(13):4607–21. doi: 10.1128/MCB.22.13.4607-4621.2002

</cite> [<a href="https://doi.org/10.1128/MCB.22.13.4607-4621.2002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC133914/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12052870/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol%20Cell%20Biol&amp;title=Interactions%20of%20the%20Mcm1%20MADS%20box%20protein%20with%20cofactors%20that%20regulate%20mating%20in%20yeast&amp;author=J%20Mead&amp;author=AR%20Bruning&amp;author=MK%20Gill&amp;author=AM%20Steiner&amp;author=TB%20Acton&amp;volume=22&amp;issue=13&amp;publication_year=2002&amp;pages=4607-21&amp;pmid=12052870&amp;doi=10.1128/MCB.22.13.4607-4621.2002&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330098.ref027">
<span class="label">27.</span><cite>Messenguy F, Dubois E. Role of MADS box proteins and their cofactors in combinatorial control of gene expression and cell development. Gene. 2003;316:1–21. doi: 10.1016/s0378-1119(03)00747-9

</cite> [<a href="https://doi.org/10.1016/s0378-1119(03)00747-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/14563547/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Gene&amp;title=Role%20of%20MADS%20box%20proteins%20and%20their%20cofactors%20in%20combinatorial%20control%20of%20gene%20expression%20and%20cell%20development&amp;author=F%20Messenguy&amp;author=E%20Dubois&amp;volume=316&amp;publication_year=2003&amp;pages=1-21&amp;pmid=14563547&amp;doi=10.1016/s0378-1119(03)00747-9&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330098.ref028">
<span class="label">28.</span><cite>Bemer M, Heijmans K, Airoldi C, Davies B, Angenent GC. An atlas of type I MADS-box gene expression during female gametophyte and seed development in Arabidopsis. Plant Physiol. 2010;154:287–300.
</cite> [<a href="https://doi.org/10.1104/pp.110.160770" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2938147/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20631316/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Physiol&amp;title=An%20atlas%20of%20type%20I%20MADS-box%20gene%20expression%20during%20female%20gametophyte%20and%20seed%20development%20in%20Arabidopsis&amp;author=M%20Bemer&amp;author=K%20Heijmans&amp;author=C%20Airoldi&amp;author=B%20Davies&amp;author=GC%20Angenent&amp;volume=154&amp;publication_year=2010&amp;pages=287-300&amp;pmid=20631316&amp;doi=10.1104/pp.110.160770&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330098.ref029">
<span class="label">29.</span><cite>Theißen G, Kim JT, Saedler H. Classification and phylogeny of the MADS-box multigene family suggest defined roles of MADS-box gene subfamilies in the morphological evolution of eukaryotes. J Mol Evol. 1996;43:484–516.
</cite> [<a href="https://doi.org/10.1007/BF02337521" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/8875863/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Mol%20Evol&amp;title=Classification%20and%20phylogeny%20of%20the%20MADS-box%20multigene%20family%20suggest%20defined%20roles%20of%20MADS-box%20gene%20subfamilies%20in%20the%20morphological%20evolution%20of%20eukaryotes&amp;author=G%20Thei%C3%9Fen&amp;author=JT%20Kim&amp;author=H%20Saedler&amp;volume=43&amp;publication_year=1996&amp;pages=484-516&amp;pmid=8875863&amp;doi=10.1007/BF02337521&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330098.ref030">
<span class="label">30.</span><cite>Krizek BA, Meyerowitz EM. Mapping the protein regions responsible for the functional specificities of the Arabidopsis MADS domain organ-identity proteins. Proc Natl Acad Sci U S A. 1996;93(9):4063–70. doi: 10.1073/pnas.93.9.4063

</cite> [<a href="https://doi.org/10.1073/pnas.93.9.4063" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC39487/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/8633017/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&amp;title=Mapping%20the%20protein%20regions%20responsible%20for%20the%20functional%20specificities%20of%20the%20Arabidopsis%20MADS%20domain%20organ-identity%20proteins&amp;author=BA%20Krizek&amp;author=EM%20Meyerowitz&amp;volume=93&amp;issue=9&amp;publication_year=1996&amp;pages=4063-70&amp;pmid=8633017&amp;doi=10.1073/pnas.93.9.4063&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330098.ref031">
<span class="label">31.</span><cite>Riechmann JL, Krizek BA, Meyerowitz EM. Dimerization specificity of Arabidopsis MADS domain homeotic proteins APETALA1, APETALA3, PISTILLATA, and AGAMOUS. Proc Natl Acad Sci U S A. 1996;93(10):4793–8. doi: 10.1073/pnas.93.10.4793

</cite> [<a href="https://doi.org/10.1073/pnas.93.10.4793" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC39358/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/8643482/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&amp;title=Dimerization%20specificity%20of%20Arabidopsis%20MADS%20domain%20homeotic%20proteins%20APETALA1,%20APETALA3,%20PISTILLATA,%20and%20AGAMOUS&amp;author=JL%20Riechmann&amp;author=BA%20Krizek&amp;author=EM%20Meyerowitz&amp;volume=93&amp;issue=10&amp;publication_year=1996&amp;pages=4793-8&amp;pmid=8643482&amp;doi=10.1073/pnas.93.10.4793&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330098.ref032">
<span class="label">32.</span><cite>Yang Y, Jack T. Defining subdomains of the K domain important for protein-protein interactions of plant MADS proteins. Plant Mol Biol. 2004;55(1):45–59. doi: 10.1007/s11103-004-0416-7

</cite> [<a href="https://doi.org/10.1007/s11103-004-0416-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15604664/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Mol%20Biol&amp;title=Defining%20subdomains%20of%20the%20K%20domain%20important%20for%20protein-protein%20interactions%20of%20plant%20MADS%20proteins&amp;author=Y%20Yang&amp;author=T%20Jack&amp;volume=55&amp;issue=1&amp;publication_year=2004&amp;pages=45-59&amp;pmid=15604664&amp;doi=10.1007/s11103-004-0416-7&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330098.ref033">
<span class="label">33.</span><cite>Lai X, Vega-Léon R, Hugouvieux V, Blanc-Mathieu R, van der Wal F, Lucas J, et al. The intervening domain is required for DNA-binding and functional identity of plant MADS transcription factors. Nat Commun. 2021;12:4760.
</cite> [<a href="https://doi.org/10.1038/s41467-021-24978-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8346517/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34362909/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Commun&amp;title=The%20intervening%20domain%20is%20required%20for%20DNA-binding%20and%20functional%20identity%20of%20plant%20MADS%20transcription%20factors&amp;author=X%20Lai&amp;author=R%20Vega-L%C3%A9on&amp;author=V%20Hugouvieux&amp;author=R%20Blanc-Mathieu&amp;author=F%20van%20der%20Wal&amp;volume=12&amp;publication_year=2021&amp;pages=4760&amp;pmid=34362909&amp;doi=10.1038/s41467-021-24978-w&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330098.ref034">
<span class="label">34.</span><cite>Fan HY, Hu Y, Tudor M, Ma H. Specific interactions between the K domains of AG and AGLs, members of the MADS domain family of DNA binding proteins. Plant J. 1997;12(5):999–1010. doi: 10.1046/j.1365-313x.1997.12050999.x

</cite> [<a href="https://doi.org/10.1046/j.1365-313x.1997.12050999.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9418042/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20J&amp;title=Specific%20interactions%20between%20the%20K%20domains%20of%20AG%20and%20AGLs,%20members%20of%20the%20MADS%20domain%20family%20of%20DNA%20binding%20proteins&amp;author=HY%20Fan&amp;author=Y%20Hu&amp;author=M%20Tudor&amp;author=H%20Ma&amp;volume=12&amp;issue=5&amp;publication_year=1997&amp;pages=999-1010&amp;pmid=9418042&amp;doi=10.1046/j.1365-313x.1997.12050999.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330098.ref035">
<span class="label">35.</span><cite>Riechmann JL, Meyerowitz EM. MADS domain proteins in plant development. Biol Chem. 1997;378(10):1079–101. 
</cite> [<a href="https://pubmed.ncbi.nlm.nih.gov/9372178/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biol%20Chem&amp;title=MADS%20domain%20proteins%20in%20plant%20development&amp;author=JL%20Riechmann&amp;author=EM%20Meyerowitz&amp;volume=378&amp;issue=10&amp;publication_year=1997&amp;pages=1079-101&amp;pmid=9372178&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330098.ref036">
<span class="label">36.</span><cite>Parenicová L, de Folter S, Kieffer M, Horner DS, Favalli C, Busscher J, et al. Molecular and phylogenetic analyses of the complete MADS-box transcription factor family in Arabidopsis: new openings to the MADS world. Plant Cell. 2003;15(7):1538–51. doi: 10.1105/tpc.011544

</cite> [<a href="https://doi.org/10.1105/tpc.011544" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC165399/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12837945/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Cell&amp;title=Molecular%20and%20phylogenetic%20analyses%20of%20the%20complete%20MADS-box%20transcription%20factor%20family%20in%20Arabidopsis:%20new%20openings%20to%20the%20MADS%20world&amp;author=L%20Parenicov%C3%A1&amp;author=S%20de%20Folter&amp;author=M%20Kieffer&amp;author=DS%20Horner&amp;author=C%20Favalli&amp;volume=15&amp;issue=7&amp;publication_year=2003&amp;pages=1538-51&amp;pmid=12837945&amp;doi=10.1105/tpc.011544&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330098.ref037">
<span class="label">37.</span><cite>Salladini E, Jørgensen MLM, Theisen FF, Skriver K. Intrinsic Disorder in Plant Transcription Factor Systems: Functional Implications. Int J Mol Sci. 2020;21(24):9755. doi: 10.3390/ijms21249755

</cite> [<a href="https://doi.org/10.3390/ijms21249755" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7767404/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33371315/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Int%20J%20Mol%20Sci&amp;title=Intrinsic%20Disorder%20in%20Plant%20Transcription%20Factor%20Systems:%20Functional%20Implications&amp;author=E%20Salladini&amp;author=MLM%20J%C3%B8rgensen&amp;author=FF%20Theisen&amp;author=K%20Skriver&amp;volume=21&amp;issue=24&amp;publication_year=2020&amp;pages=9755&amp;pmid=33371315&amp;doi=10.3390/ijms21249755&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330098.ref038">
<span class="label">38.</span><cite>Dunker AK, Brown CJ, Lawson JD, Iakoucheva LM, Obradović Z. Intrinsic disorder and protein function. Biochemistry. 2002;41(21):6573–82. doi: 10.1021/bi012159+

</cite> [<a href="https://doi.org/10.1021/bi012159+" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12022860/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biochemistry&amp;title=Intrinsic%20disorder%20and%20protein%20function&amp;author=AK%20Dunker&amp;author=CJ%20Brown&amp;author=JD%20Lawson&amp;author=LM%20Iakoucheva&amp;author=Z%20Obradovi%C4%87&amp;volume=41&amp;issue=21&amp;publication_year=2002&amp;pages=6573-82&amp;pmid=12022860&amp;doi=10.1021/bi012159+&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330098.ref039">
<span class="label">39.</span><cite>Dunker AK, Lawson JD, Brown CJ, Williams RM, Romero P, Oh JS, et al. Intrinsically disordered protein. J Mol Graph Model. 2001;19(1):26–59. doi: 10.1016/s1093-3263(00)00138-8

</cite> [<a href="https://doi.org/10.1016/s1093-3263(00)00138-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11381529/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Mol%20Graph%20Model&amp;title=Intrinsically%20disordered%20protein&amp;author=AK%20Dunker&amp;author=JD%20Lawson&amp;author=CJ%20Brown&amp;author=RM%20Williams&amp;author=P%20Romero&amp;volume=19&amp;issue=1&amp;publication_year=2001&amp;pages=26-59&amp;pmid=11381529&amp;doi=10.1016/s1093-3263(00)00138-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330098.ref040">
<span class="label">40.</span><cite>Radivojac P, Obradovic Z, Smith DK, Zhu G, Vucetic S, Brown CJ. Protein flexibility and intrinsic disorder. Protein Sci. 2004;13:71–80.
</cite> [<a href="https://doi.org/10.1110/ps.03128904" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2286519/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/14691223/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Protein%20Sci&amp;title=Protein%20flexibility%20and%20intrinsic%20disorder&amp;author=P%20Radivojac&amp;author=Z%20Obradovic&amp;author=DK%20Smith&amp;author=G%20Zhu&amp;author=S%20Vucetic&amp;volume=13&amp;publication_year=2004&amp;pages=71-80&amp;pmid=14691223&amp;doi=10.1110/ps.03128904&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330098.ref041">
<span class="label">41.</span><cite>Dyson HJ, Wright PE. Intrinsically unstructured proteins and their functions. Nat Rev Mol Cell Biol. 2005;6(3):197–208. doi: 10.1038/nrm1589

</cite> [<a href="https://doi.org/10.1038/nrm1589" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15738986/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Mol%20Cell%20Biol&amp;title=Intrinsically%20unstructured%20proteins%20and%20their%20functions&amp;author=HJ%20Dyson&amp;author=PE%20Wright&amp;volume=6&amp;issue=3&amp;publication_year=2005&amp;pages=197-208&amp;pmid=15738986&amp;doi=10.1038/nrm1589&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330098.ref042">
<span class="label">42.</span><cite>Tompa P. Intrinsically disordered proteins: A 10-year recap. Trends in Biochem Sci. 2012;37:509–16.
</cite> [<a href="https://doi.org/10.1016/j.tibs.2012.08.004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22989858/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Trends%20in%20Biochem%20Sci&amp;title=Intrinsically%20disordered%20proteins:%20A%2010-year%20recap&amp;author=P%20Tompa&amp;volume=37&amp;publication_year=2012&amp;pages=509-16&amp;pmid=22989858&amp;doi=10.1016/j.tibs.2012.08.004&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330098.ref043">
<span class="label">43.</span><cite>O’Shea C, Kryger M, Stender EGP, Kragelund BB, Willemoës M, Skriver K. Protein intrinsic disorder in Arabidopsis NAC transcription factors: transcriptional activation by ANAC013 and ANAC046 and their interactions with RCD1. Biochem J. 2015;465(2):281–94. doi: 10.1042/BJ20141045

</cite> [<a href="https://doi.org/10.1042/BJ20141045" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25348421/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biochem%20J&amp;title=Protein%20intrinsic%20disorder%20in%20Arabidopsis%20NAC%20transcription%20factors:%20transcriptional%20activation%20by%20ANAC013%20and%20ANAC046%20and%20their%20interactions%20with%20RCD1&amp;author=C%20O%E2%80%99Shea&amp;author=M%20Kryger&amp;author=EGP%20Stender&amp;author=BB%20Kragelund&amp;author=M%20Willemo%C3%ABs&amp;volume=465&amp;issue=2&amp;publication_year=2015&amp;pages=281-94&amp;pmid=25348421&amp;doi=10.1042/BJ20141045&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330098.ref044">
<span class="label">44.</span><cite>Stender EG, O’Shea C, Skriver K. Subgroup-specific intrinsic disorder profiles of arabidopsis NAC transcription factors: Identification of functional hotspots. Plant Signal Behav. 2015;10:e1010967.</cite> [<a href="https://doi.org/10.1080/15592324.2015.1010967" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4622513/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26107850/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Signal%20Behav&amp;title=Subgroup-specific%20intrinsic%20disorder%20profiles%20of%20arabidopsis%20NAC%20transcription%20factors:%20Identification%20of%20functional%20hotspots&amp;author=EG%20Stender&amp;author=C%20O%E2%80%99Shea&amp;author=K%20Skriver&amp;volume=10&amp;publication_year=2015&amp;pmid=26107850&amp;doi=10.1080/15592324.2015.1010967&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330098.ref045">
<span class="label">45.</span><cite>Sun X, Jones WT, Rikkerink EHA. GRAS proteins: the versatile roles of intrinsically disordered proteins in plant signalling. Biochem J. 2012;442(1):1–12. doi: 10.1042/BJ20111766

</cite> [<a href="https://doi.org/10.1042/BJ20111766" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22280012/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biochem%20J&amp;title=GRAS%20proteins:%20the%20versatile%20roles%20of%20intrinsically%20disordered%20proteins%20in%20plant%20signalling&amp;author=X%20Sun&amp;author=WT%20Jones&amp;author=EHA%20Rikkerink&amp;volume=442&amp;issue=1&amp;publication_year=2012&amp;pages=1-12&amp;pmid=22280012&amp;doi=10.1042/BJ20111766&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330098.ref046">
<span class="label">46.</span><cite>Valsecchi I, Guittard-Crilat E, Maldiney R, Habricot Y, Lignon S, Lebrun R, et al. The intrinsically disordered C-terminal region of Arabidopsis thaliana TCP8 transcription factor acts both as a transactivation and self-assembly domain. Mol Biosyst. 2013;9(9):2282–95. doi: 10.1039/c3mb70128j

</cite> [<a href="https://doi.org/10.1039/c3mb70128j" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23760157/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol%20Biosyst&amp;title=The%20intrinsically%20disordered%20C-terminal%20region%20of%20Arabidopsis%20thaliana%20TCP8%20transcription%20factor%20acts%20both%20as%20a%20transactivation%20and%20self-assembly%20domain&amp;author=I%20Valsecchi&amp;author=E%20Guittard-Crilat&amp;author=R%20Maldiney&amp;author=Y%20Habricot&amp;author=S%20Lignon&amp;volume=9&amp;issue=9&amp;publication_year=2013&amp;pages=2282-95&amp;pmid=23760157&amp;doi=10.1039/c3mb70128j&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330098.ref047">
<span class="label">47.</span><cite>Tarczewska A, Greb-Markiewicz B. The Significance of the Intrinsically Disordered Regions for the Functions of the bHLH Transcription Factors. Int J Mol Sci. 2019;20(21):5306. doi: 10.3390/ijms20215306

</cite> [<a href="https://doi.org/10.3390/ijms20215306" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6862971/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31653121/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Int%20J%20Mol%20Sci&amp;title=The%20Significance%20of%20the%20Intrinsically%20Disordered%20Regions%20for%20the%20Functions%20of%20the%20bHLH%20Transcription%20Factors&amp;author=A%20Tarczewska&amp;author=B%20Greb-Markiewicz&amp;volume=20&amp;issue=21&amp;publication_year=2019&amp;pages=5306&amp;pmid=31653121&amp;doi=10.3390/ijms20215306&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330098.ref048">
<span class="label">48.</span><cite>Oldfield CJ, Cheng Y, Cortese MS, Romero P, Uversky VN, Dunker AK. Coupled folding and binding with alpha-helix-forming molecular recognition elements. Biochemistry. 2005;44(37):12454–70. doi: 10.1021/bi050736e

</cite> [<a href="https://doi.org/10.1021/bi050736e" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16156658/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biochemistry&amp;title=Coupled%20folding%20and%20binding%20with%20alpha-helix-forming%20molecular%20recognition%20elements&amp;author=CJ%20Oldfield&amp;author=Y%20Cheng&amp;author=MS%20Cortese&amp;author=P%20Romero&amp;author=VN%20Uversky&amp;volume=44&amp;issue=37&amp;publication_year=2005&amp;pages=12454-70&amp;pmid=16156658&amp;doi=10.1021/bi050736e&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330098.ref049">
<span class="label">49.</span><cite>Yan J, Dunker AK, Uversky VN, Kurgan L. Molecular recognition features (MoRFs) in three domains of life. Mol Biosyst. 2016;12(3):697–710. doi: 10.1039/c5mb00640f

</cite> [<a href="https://doi.org/10.1039/c5mb00640f" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26651072/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol%20Biosyst&amp;title=Molecular%20recognition%20features%20(MoRFs)%20in%20three%20domains%20of%20life&amp;author=J%20Yan&amp;author=AK%20Dunker&amp;author=VN%20Uversky&amp;author=L%20Kurgan&amp;volume=12&amp;issue=3&amp;publication_year=2016&amp;pages=697-710&amp;pmid=26651072&amp;doi=10.1039/c5mb00640f&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330098.ref050">
<span class="label">50.</span><cite>Mohan A, Oldfield CJ, Radivojac P, Vacic V, Cortese MS, Dunker AK, et al. Analysis of molecular recognition features (MoRFs). J Mol Biol. 2006;362(5):1043–59. doi: 10.1016/j.jmb.2006.07.087

</cite> [<a href="https://doi.org/10.1016/j.jmb.2006.07.087" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16935303/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Mol%20Biol&amp;title=Analysis%20of%20molecular%20recognition%20features%20(MoRFs)&amp;author=A%20Mohan&amp;author=CJ%20Oldfield&amp;author=P%20Radivojac&amp;author=V%20Vacic&amp;author=MS%20Cortese&amp;volume=362&amp;issue=5&amp;publication_year=2006&amp;pages=1043-59&amp;pmid=16935303&amp;doi=10.1016/j.jmb.2006.07.087&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330098.ref051">
<span class="label">51.</span><cite>Vacic V, Oldfield CJ, Mohan A, Radivojac P, Cortese MS, Uversky VN, et al. Characterization of molecular recognition features, MoRFs, and their binding partners. J Proteome Res. 2007;6(6):2351–66. doi: 10.1021/pr0701411

</cite> [<a href="https://doi.org/10.1021/pr0701411" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2570643/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17488107/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Proteome%20Res&amp;title=Characterization%20of%20molecular%20recognition%20features,%20MoRFs,%20and%20their%20binding%20partners&amp;author=V%20Vacic&amp;author=CJ%20Oldfield&amp;author=A%20Mohan&amp;author=P%20Radivojac&amp;author=MS%20Cortese&amp;volume=6&amp;issue=6&amp;publication_year=2007&amp;pages=2351-66&amp;pmid=17488107&amp;doi=10.1021/pr0701411&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330098.ref052">
<span class="label">52.</span><cite>Das RK, Pappu RV. Conformations of intrinsically disordered proteins are influenced by linear sequence distributions of oppositely charged residues. Proc Natl Acad Sci U S A. 2013;110(33):13392–7. doi: 10.1073/pnas.1304749110

</cite> [<a href="https://doi.org/10.1073/pnas.1304749110" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3746876/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23901099/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&amp;title=Conformations%20of%20intrinsically%20disordered%20proteins%20are%20influenced%20by%20linear%20sequence%20distributions%20of%20oppositely%20charged%20residues&amp;author=RK%20Das&amp;author=RV%20Pappu&amp;volume=110&amp;issue=33&amp;publication_year=2013&amp;pages=13392-7&amp;pmid=23901099&amp;doi=10.1073/pnas.1304749110&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330098.ref053">
<span class="label">53.</span><cite>Bhattarai A, Emerson IA. Dynamic conformational flexibility and molecular interactions of intrinsically disordered proteins. J Biosci. 2020;45:29. doi: 10.1007/s12038-020-0010-4

</cite> [<a href="https://doi.org/10.1007/s12038-020-0010-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32020911/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Biosci&amp;title=Dynamic%20conformational%20flexibility%20and%20molecular%20interactions%20of%20intrinsically%20disordered%20proteins&amp;author=A%20Bhattarai&amp;author=IA%20Emerson&amp;volume=45&amp;publication_year=2020&amp;pages=29&amp;pmid=32020911&amp;doi=10.1007/s12038-020-0010-4&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330098.ref054">
<span class="label">54.</span><cite>Dunker AK, Cortese MS, Romero P, Iakoucheva LM, Uversky VN. Flexible nets. FEBS Journal. 2005;272:5129–48.
</cite> [<a href="https://doi.org/10.1111/j.1742-4658.2005.04948.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16218947/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=FEBS%20Journal&amp;title=Flexible%20nets&amp;author=AK%20Dunker&amp;author=MS%20Cortese&amp;author=P%20Romero&amp;author=LM%20Iakoucheva&amp;author=VN%20Uversky&amp;volume=272&amp;publication_year=2005&amp;pages=5129-48&amp;pmid=16218947&amp;doi=10.1111/j.1742-4658.2005.04948.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330098.ref055">
<span class="label">55.</span><cite>Alberti S. The wisdom of crowds: regulating cell function through condensed states of living matter. J Cell Sci. 2017;130(17):2789–96. doi: 10.1242/jcs.200295

</cite> [<a href="https://doi.org/10.1242/jcs.200295" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28808090/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Cell%20Sci&amp;title=The%20wisdom%20of%20crowds:%20regulating%20cell%20function%20through%20condensed%20states%20of%20living%20matter&amp;author=S%20Alberti&amp;volume=130&amp;issue=17&amp;publication_year=2017&amp;pages=2789-96&amp;pmid=28808090&amp;doi=10.1242/jcs.200295&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330098.ref056">
<span class="label">56.</span><cite>Alberti S, Hyman AA. Biomolecular condensates at the nexus of cellular stress, protein aggregation disease and ageing. Nat Rev Mol Cell Biol. 2021;22(3):196–213. doi: 10.1038/s41580-020-00326-6

</cite> [<a href="https://doi.org/10.1038/s41580-020-00326-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33510441/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Mol%20Cell%20Biol&amp;title=Biomolecular%20condensates%20at%20the%20nexus%20of%20cellular%20stress,%20protein%20aggregation%20disease%20and%20ageing&amp;author=S%20Alberti&amp;author=AA%20Hyman&amp;volume=22&amp;issue=3&amp;publication_year=2021&amp;pages=196-213&amp;pmid=33510441&amp;doi=10.1038/s41580-020-00326-6&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330098.ref057">
<span class="label">57.</span><cite>TAIR - Home. Available from: <a href="https://www.arabidopsis.org/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.arabidopsis.org/</a>.</cite>
</li>
<li id="pone.0330098.ref058">
<span class="label">58.</span><cite>UniProt. Available from: <a href="https://www.uniprot.org/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.uniprot.org/</a>.</cite>
</li>
<li id="pone.0330098.ref059">
<span class="label">59.</span><cite>National Center for Biotechnology Information. National Center for Biotechnology Information. Available from: <a href="https://www.ncbi.nlm.nih.gov/" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/</a>.</cite>
</li>
<li id="pone.0330098.ref060">
<span class="label">60.</span><cite>RAP-DB | HOME. Available from: <a href="https://rapdb.dna.affrc.go.jp/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://rapdb.dna.affrc.go.jp/</a>.</cite>
</li>
<li id="pone.0330098.ref061">
<span class="label">61.</span><cite>Arora R, Agarwal P, Ray S, Singh AK, Singh VP, Tyagi AK, et al. MADS-box gene family in rice: genome-wide identification, organization and expression profiling during reproductive development and stress. BMC Genomics. 2007;8:242. doi: 10.1186/1471-2164-8-242

</cite> [<a href="https://doi.org/10.1186/1471-2164-8-242" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1947970/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17640358/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Genomics&amp;title=MADS-box%20gene%20family%20in%20rice:%20genome-wide%20identification,%20organization%20and%20expression%20profiling%20during%20reproductive%20development%20and%20stress&amp;author=R%20Arora&amp;author=P%20Agarwal&amp;author=S%20Ray&amp;author=AK%20Singh&amp;author=VP%20Singh&amp;volume=8&amp;publication_year=2007&amp;pages=242&amp;pmid=17640358&amp;doi=10.1186/1471-2164-8-242&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330098.ref062">
<span class="label">62.</span><cite>MAFFT alignment and NJ/ UPGMA phylogeny. Available from: <a href="https://mafft.cbrc.jp/alignment/server/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://mafft.cbrc.jp/alignment/server/</a>.</cite>
</li>
<li id="pone.0330098.ref063">
<span class="label">63.</span><cite>Katoh K, Rozewicki J, Yamada KD. MAFFT online service: multiple sequence alignment, interactive sequence choice and visualization. Brief Bioinform. 2019;20(4):1160–6. doi: 10.1093/bib/bbx108

</cite> [<a href="https://doi.org/10.1093/bib/bbx108" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6781576/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28968734/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Brief%20Bioinform&amp;title=MAFFT%20online%20service:%20multiple%20sequence%20alignment,%20interactive%20sequence%20choice%20and%20visualization&amp;author=K%20Katoh&amp;author=J%20Rozewicki&amp;author=KD%20Yamada&amp;volume=20&amp;issue=4&amp;publication_year=2019&amp;pages=1160-6&amp;pmid=28968734&amp;doi=10.1093/bib/bbx108&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330098.ref064">
<span class="label">64.</span><cite>Larsson A. AliView: a fast and lightweight alignment viewer and editor for large datasets. Bioinformatics. 2014;30(22):3276–8. doi: 10.1093/bioinformatics/btu531

</cite> [<a href="https://doi.org/10.1093/bioinformatics/btu531" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4221126/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25095880/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=AliView:%20a%20fast%20and%20lightweight%20alignment%20viewer%20and%20editor%20for%20large%20datasets&amp;author=A%20Larsson&amp;volume=30&amp;issue=22&amp;publication_year=2014&amp;pages=3276-8&amp;pmid=25095880&amp;doi=10.1093/bioinformatics/btu531&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330098.ref065">
<span class="label">65.</span><cite>Jones DT, Taylor WR, Thornton JM. The rapid generation of mutation data matrices from protein sequences. Comput Appl Biosci. 1992;8(3):275–82. doi: 10.1093/bioinformatics/8.3.275

</cite> [<a href="https://doi.org/10.1093/bioinformatics/8.3.275" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/1633570/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Comput%20Appl%20Biosci&amp;title=The%20rapid%20generation%20of%20mutation%20data%20matrices%20from%20protein%20sequences&amp;author=DT%20Jones&amp;author=WR%20Taylor&amp;author=JM%20Thornton&amp;volume=8&amp;issue=3&amp;publication_year=1992&amp;pages=275-82&amp;pmid=1633570&amp;doi=10.1093/bioinformatics/8.3.275&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330098.ref066">
<span class="label">66.</span><cite>Stamatakis A. RAxML version 8: a tool for phylogenetic analysis and post-analysis of large phylogenies. Bioinformatics. 2014;30(9):1312–3. doi: 10.1093/bioinformatics/btu033

</cite> [<a href="https://doi.org/10.1093/bioinformatics/btu033" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3998144/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24451623/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=RAxML%20version%208:%20a%20tool%20for%20phylogenetic%20analysis%20and%20post-analysis%20of%20large%20phylogenies&amp;author=A%20Stamatakis&amp;volume=30&amp;issue=9&amp;publication_year=2014&amp;pages=1312-3&amp;pmid=24451623&amp;doi=10.1093/bioinformatics/btu033&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330098.ref067">
<span class="label">67.</span><cite>Miller MA, Pfeiffer W, Schwartz T. Creating the CIPRES Science Gateway for inference of large phylogenetic trees. In: 2010 Gateway Computing Environments Workshop, GCE. 2010.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=2010%20Gateway%20Computing%20Environments%20Workshop,%20GCE&amp;author=MA%20Miller&amp;author=W%20Pfeiffer&amp;author=T%20Schwartz&amp;publication_year=2010&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330098.ref068">
<span class="label">68.</span><cite>Portal | CIPRES. Available from: <a href="https://www.phylo.org/index.php/site" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.phylo.org/index.php/site</a>.</cite>
</li>
<li id="pone.0330098.ref069">
<span class="label">69.</span><cite>Meme - Submission Form. Available from: <a href="https://meme-suite.org/meme/tools/meme" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://meme-suite.org/meme/tools/meme</a>.</cite>
</li>
<li id="pone.0330098.ref070">
<span class="label">70.</span><cite>Bailey TL, Johnson J, Grant CE, Noble WS. The MEME Suite. Nucleic Acids Research. 2015;43:W39-49.</cite> [<a href="https://doi.org/10.1093/nar/gkv416" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4489269/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25953851/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Research&amp;title=The%20MEME%20Suite&amp;author=TL%20Bailey&amp;author=J%20Johnson&amp;author=CE%20Grant&amp;author=WS%20Noble&amp;volume=43&amp;publication_year=2015&amp;pmid=25953851&amp;doi=10.1093/nar/gkv416&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330098.ref071">
<span class="label">71.</span><cite>fMoRFpred - fast Molecular Recognition Feature predictor. Available from: <a href="http://biomine.cs.vcu.edu/servers/fMoRFpred/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://biomine.cs.vcu.edu/servers/fMoRFpred/</a>.</cite>
</li>
<li id="pone.0330098.ref072">
<span class="label">72.</span><cite>Morffy N, Van den Broeck L, Miller C, Emenecker RJ, Bryant JA Jr, Lee TM, et al. Identification of plant transcriptional activation domains. Nature. 2024;632(8023):166–73. doi: 10.1038/s41586-024-07707-3

</cite> [<a href="https://doi.org/10.1038/s41586-024-07707-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11589624/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39020176/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Identification%20of%20plant%20transcriptional%20activation%20domains&amp;author=N%20Morffy&amp;author=L%20Van%20den%20Broeck&amp;author=C%20Miller&amp;author=RJ%20Emenecker&amp;author=JA%20Bryant&amp;volume=632&amp;issue=8023&amp;publication_year=2024&amp;pages=166-73&amp;pmid=39020176&amp;doi=10.1038/s41586-024-07707-3&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330098.ref073">
<span class="label">73.</span><cite>Piovesan D, Del Conte A, Clementel D, Monzon AM, Bevilacqua M, Aspromonte MC, et al. MobiDB: 10 years of intrinsically disordered proteins. Nucleic Acids Res. 2023;51:D438-44.</cite> [<a href="https://doi.org/10.1093/nar/gkac1065" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9825420/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36416266/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=MobiDB:%2010%20years%20of%20intrinsically%20disordered%20proteins&amp;author=D%20Piovesan&amp;author=A%20Del%20Conte&amp;author=D%20Clementel&amp;author=AM%20Monzon&amp;author=M%20Bevilacqua&amp;volume=51&amp;publication_year=2023&amp;pmid=36416266&amp;doi=10.1093/nar/gkac1065&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330098.ref074">
<span class="label">74.</span><cite>AlphaFold. Google Colab. Available from: <a href="https://colab.research.google.com/github/deepmind/alphafold/blob/main/notebooks/AlphaFold.ipynb" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://colab.research.google.com/github/deepmind/alphafold/blob/main/notebooks/AlphaFold.ipynb</a>.</cite> [<a href="https://scholar.google.com/scholar_lookup?AlphaFold.%20Google%20Colab.%20Available%20from:%20https://colab.research.google.com/github/deepmind/alphafold/blob/main/notebooks/AlphaFold.ipynb." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330098.ref075">
<span class="label">75.</span><cite>Meng EC, Goddard TD, Pettersen EF, Couch GS, Pearson ZJ, Morris JH, et al. UCSF ChimeraX: Tools for structure building and analysis. Protein Sci. 2023;32(11):e4792. doi: 10.1002/pro.4792

</cite> [<a href="https://doi.org/10.1002/pro.4792" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10588335/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37774136/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Protein%20Sci&amp;title=UCSF%20ChimeraX:%20Tools%20for%20structure%20building%20and%20analysis&amp;author=EC%20Meng&amp;author=TD%20Goddard&amp;author=EF%20Pettersen&amp;author=GS%20Couch&amp;author=ZJ%20Pearson&amp;volume=32&amp;issue=11&amp;publication_year=2023&amp;pmid=37774136&amp;doi=10.1002/pro.4792&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330098.ref076">
<span class="label">76.</span><cite>Annotation from 3D structure data. Available from: <a href="https://www.jalview.org/help/html/features/xsspannotation.html" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.jalview.org/help/html/features/xsspannotation.html</a>.</cite>
</li>
<li id="pone.0330098.ref077">
<span class="label">77.</span><cite>Jumper J, Evans R, Pritzel A, Green T, Figurnov M, Ronneberger O, et al. Highly accurate protein structure prediction with AlphaFold. Nature. 2021;596(7873):583–9. doi: 10.1038/s41586-021-03819-2

</cite> [<a href="https://doi.org/10.1038/s41586-021-03819-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8371605/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34265844/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Highly%20accurate%20protein%20structure%20prediction%20with%20AlphaFold&amp;author=J%20Jumper&amp;author=R%20Evans&amp;author=A%20Pritzel&amp;author=T%20Green&amp;author=M%20Figurnov&amp;volume=596&amp;issue=7873&amp;publication_year=2021&amp;pages=583-9&amp;pmid=34265844&amp;doi=10.1038/s41586-021-03819-2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330098.ref078">
<span class="label">78.</span><cite>Wilson CJ, Choy WY, Karttunen M. AlphaFold2: A role for disordered protein/region prediction?
Int J Mol Sci. 2022;23:4591.
</cite> [<a href="https://doi.org/10.3390/ijms23094591" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9104326/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35562983/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Int%20J%20Mol%20Sci&amp;title=AlphaFold2:%20A%20role%20for%20disordered%20protein/region%20prediction?&amp;author=CJ%20Wilson&amp;author=WY%20Choy&amp;author=M%20Karttunen&amp;volume=23&amp;publication_year=2022&amp;pages=4591&amp;pmid=35562983&amp;doi=10.3390/ijms23094591&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330098.ref079">
<span class="label">79.</span><cite>RIDAO: Rapid Intrinsic Disorder Analysis Online. Available from: <a href="https://ridao.app/users/sign_in" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://ridao.app/users/sign_in</a>.</cite>
</li>
<li id="pone.0330098.ref080">
<span class="label">80.</span><cite>Dayhoff GW 2nd, Uversky VN. Rapid prediction and analysis of protein intrinsic disorder. Protein Sci. 2022;31(12):e4496. doi: 10.1002/pro.4496

</cite> [<a href="https://doi.org/10.1002/pro.4496" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9679974/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36334049/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Protein%20Sci&amp;title=Rapid%20prediction%20and%20analysis%20of%20protein%20intrinsic%20disorder&amp;author=2nd%20Dayhoff%20GW&amp;author=VN%20Uversky&amp;volume=31&amp;issue=12&amp;publication_year=2022&amp;pmid=36334049&amp;doi=10.1002/pro.4496&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330098.ref081">
<span class="label">81.</span><cite>Shukla S, Lastorka SS, Uversky VN. Intrinsic Disorder and Phase Separation Coordinate Exocytosis, Motility, and Chromatin Remodeling in the Human Acrosomal Proteome. Proteomes. 2025;13(2):16. doi: 10.3390/proteomes13020016

</cite> [<a href="https://doi.org/10.3390/proteomes13020016" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC12101322/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40407495/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proteomes&amp;title=Intrinsic%20Disorder%20and%20Phase%20Separation%20Coordinate%20Exocytosis,%20Motility,%20and%20Chromatin%20Remodeling%20in%20the%20Human%20Acrosomal%20Proteome&amp;author=S%20Shukla&amp;author=SS%20Lastorka&amp;author=VN%20Uversky&amp;volume=13&amp;issue=2&amp;publication_year=2025&amp;pages=16&amp;pmid=40407495&amp;doi=10.3390/proteomes13020016&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330098.ref082">
<span class="label">82.</span><cite>Pappu Lab. Available from: <a href="https://pappulab.wustl.edu/CIDERinfo.html" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://pappulab.wustl.edu/CIDERinfo.html</a>.</cite>
</li>
<li id="pone.0330098.ref083">
<span class="label">83.</span><cite>Holehouse AS, Das RK, Ahad JN, Richardson MOG, Pappu RV. CIDER: Resources to Analyze Sequence-Ensemble Relationships of Intrinsically Disordered Proteins. Biophys J. 2017;112(1):16–21. doi: 10.1016/j.bpj.2016.11.3200

</cite> [<a href="https://doi.org/10.1016/j.bpj.2016.11.3200" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5232785/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28076807/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biophys%20J&amp;title=CIDER:%20Resources%20to%20Analyze%20Sequence-Ensemble%20Relationships%20of%20Intrinsically%20Disordered%20Proteins&amp;author=AS%20Holehouse&amp;author=RK%20Das&amp;author=JN%20Ahad&amp;author=MOG%20Richardson&amp;author=RV%20Pappu&amp;volume=112&amp;issue=1&amp;publication_year=2017&amp;pages=16-21&amp;pmid=28076807&amp;doi=10.1016/j.bpj.2016.11.3200&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330098.ref084">
<span class="label">84.</span><cite>NetPhos 3.1 - DTU Health Tech - Bioinformatic Services. Available from: <a href="https://services.healthtech.dtu.dk/services/NetPhos-3.1/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://services.healthtech.dtu.dk/services/NetPhos-3.1/</a>.</cite>
</li>
<li id="pone.0330098.ref085">
<span class="label">85.</span><cite>Blom N, Gammeltoft S, Brunak S. Sequence and structure-based prediction of eukaryotic protein phosphorylation sites. J Mol Biol. 1999;294:1351–62.
</cite> [<a href="https://doi.org/10.1006/jmbi.1999.3310" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10600390/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Mol%20Biol&amp;title=Sequence%20and%20structure-based%20prediction%20of%20eukaryotic%20protein%20phosphorylation%20sites&amp;author=N%20Blom&amp;author=S%20Gammeltoft&amp;author=S%20Brunak&amp;volume=294&amp;publication_year=1999&amp;pages=1351-62&amp;pmid=10600390&amp;doi=10.1006/jmbi.1999.3310&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330098.ref086">
<span class="label">86.</span><cite>Blom N, Sicheritz-Pontén T, Gupta R, Gammeltoft S, Brunak S. Prediction of post-translational glycosylation and phosphorylation of proteins from the amino acid sequence. Proteomics. 2004;4(6):1633–49. doi: 10.1002/pmic.200300771

</cite> [<a href="https://doi.org/10.1002/pmic.200300771" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15174133/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proteomics&amp;title=Prediction%20of%20post-translational%20glycosylation%20and%20phosphorylation%20of%20proteins%20from%20the%20amino%20acid%20sequence&amp;author=N%20Blom&amp;author=T%20Sicheritz-Pont%C3%A9n&amp;author=R%20Gupta&amp;author=S%20Gammeltoft&amp;author=S%20Brunak&amp;volume=4&amp;issue=6&amp;publication_year=2004&amp;pages=1633-49&amp;pmid=15174133&amp;doi=10.1002/pmic.200300771&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330098.ref087">
<span class="label">87.</span><cite>ATHENA. Available from: <a href="https://athena.proteomics.wzw.tum.de/master_arabidopsisshiny/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://athena.proteomics.wzw.tum.de/master_arabidopsisshiny/</a>.</cite>
</li>
<li id="pone.0330098.ref088">
<span class="label">88.</span><cite>Lin S, Wang C, Zhou J, Shi Y, Ruan C, Tu Y, et al. EPSD: a well-annotated data resource of protein phosphorylation sites in eukaryotes. Brief Bioinform. 2021;22(1):298–307. doi: 10.1093/bib/bbz169

</cite> [<a href="https://doi.org/10.1093/bib/bbz169" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32008039/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Brief%20Bioinform&amp;title=EPSD:%20a%20well-annotated%20data%20resource%20of%20protein%20phosphorylation%20sites%20in%20eukaryotes&amp;author=S%20Lin&amp;author=C%20Wang&amp;author=J%20Zhou&amp;author=Y%20Shi&amp;author=C%20Ruan&amp;volume=22&amp;issue=1&amp;publication_year=2021&amp;pages=298-307&amp;pmid=32008039&amp;doi=10.1093/bib/bbz169&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330098.ref089">
<span class="label">89.</span><cite>Hatos A, Tosatto SCE, Vendruscolo M, Fuxreiter M. FuzDrop on AlphaFold: visualizing the sequence-dependent propensity of liquid-liquid phase separation and aggregation of proteins. Nucleic Acids Res. 2022;50(W1):W337–44. doi: 10.1093/nar/gkac386

</cite> [<a href="https://doi.org/10.1093/nar/gkac386" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9252777/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35610022/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=FuzDrop%20on%20AlphaFold:%20visualizing%20the%20sequence-dependent%20propensity%20of%20liquid-liquid%20phase%20separation%20and%20aggregation%20of%20proteins&amp;author=A%20Hatos&amp;author=SCE%20Tosatto&amp;author=M%20Vendruscolo&amp;author=M%20Fuxreiter&amp;volume=50&amp;publication_year=2022&amp;pmid=35610022&amp;doi=10.1093/nar/gkac386&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330098.ref090">
<span class="label">90.</span><cite>FuzDrop. Available from: <a href="https://fuzdrop.bio.unipd.it/predictor" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://fuzdrop.bio.unipd.it/predictor</a></cite>
</li>
<li id="pone.0330098.ref091">
<span class="label">91.</span><cite>Hothorn T, Van De Wiel MA, Hornik K, Zeileis A. Implementing a class of permutation tests: The coin package. J Stat Softw. 2008;28:1–23.27774042
</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Stat%20Softw&amp;title=Implementing%20a%20class%20of%20permutation%20tests:%20The%20coin%20package&amp;author=T%20Hothorn&amp;author=MA%20Van%20De%20Wiel&amp;author=K%20Hornik&amp;author=A%20Zeileis&amp;volume=28&amp;publication_year=2008&amp;pages=1-23&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330098.ref092">
<span class="label">92.</span><cite>R Core Team. R: A Language and Environment for Statistical Computing. R Foundation for Statistical Computing. 2024.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=R:%20A%20Language%20and%20Environment%20for%20Statistical%20Computing&amp;publication_year=2024&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330098.ref093">
<span class="label">93.</span><cite>GIMP Development Team. GNU Image Manipulation Program (GIMP), Version 3.0.4. Community. 2025.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=GNU%20Image%20Manipulation%20Program%20(GIMP),%20Version%203.0.4&amp;publication_year=2025&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330098.ref094">
<span class="label">94.</span><cite>Iakoucheva LM, Radivojac P, Brown CJ, O’Connor TR, Sikes JG, Obradovic Z, et al. The importance of intrinsic disorder for protein phosphorylation. Nucleic Acids Res. 2004;32(3):1037–49. doi: 10.1093/nar/gkh253

</cite> [<a href="https://doi.org/10.1093/nar/gkh253" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC373391/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/14960716/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=The%20importance%20of%20intrinsic%20disorder%20for%20protein%20phosphorylation&amp;author=LM%20Iakoucheva&amp;author=P%20Radivojac&amp;author=CJ%20Brown&amp;author=TR%20O%E2%80%99Connor&amp;author=JG%20Sikes&amp;volume=32&amp;issue=3&amp;publication_year=2004&amp;pages=1037-49&amp;pmid=14960716&amp;doi=10.1093/nar/gkh253&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330098.ref095">
<span class="label">95.</span><cite>Margulis L, Chapman MJ. Kingdoms and Domains: An Illustrated Guide to the Phyla of Life on Earth, Fourth Edition. Elsevier. 2009.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=Kingdoms%20and%20Domains:%20An%20Illustrated%20Guide%20to%20the%20Phyla%20of%20Life%20on%20Earth,%20Fourth%20Edition&amp;author=L%20Margulis&amp;author=MJ%20Chapman&amp;publication_year=2009&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330098.ref096">
<span class="label">96.</span><cite>Alvarez-Buylla ER, García-Ponce B, Garay-Arroyo A. Unique and redundant functional domains of APETALA1 and CAULIFLOWER, two recently duplicated Arabidopsis thaliana floral MADS-box genes. J Exp Bot. 2006;57(12):3099–107. doi: 10.1093/jxb/erl081

</cite> [<a href="https://doi.org/10.1093/jxb/erl081" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16893974/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Exp%20Bot&amp;title=Unique%20and%20redundant%20functional%20domains%20of%20APETALA1%20and%20CAULIFLOWER,%20two%20recently%20duplicated%20Arabidopsis%20thaliana%20floral%20MADS-box%20genes&amp;author=ER%20Alvarez-Buylla&amp;author=B%20Garc%C3%ADa-Ponce&amp;author=A%20Garay-Arroyo&amp;volume=57&amp;issue=12&amp;publication_year=2006&amp;pages=3099-107&amp;pmid=16893974&amp;doi=10.1093/jxb/erl081&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330098.ref097">
<span class="label">97.</span><cite>Trojanowski J, Frank L, Rademacher A, Mücke N, Grigaitis P, Rippe K. Transcription activation is enhanced by multivalent interactions independent of phase separation. Mol Cell. 2022;82(10):1878–1893.e10. doi: 10.1016/j.molcel.2022.04.017

</cite> [<a href="https://doi.org/10.1016/j.molcel.2022.04.017" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35537448/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol%20Cell&amp;title=Transcription%20activation%20is%20enhanced%20by%20multivalent%20interactions%20independent%20of%20phase%20separation&amp;author=J%20Trojanowski&amp;author=L%20Frank&amp;author=A%20Rademacher&amp;author=N%20M%C3%BCcke&amp;author=P%20Grigaitis&amp;volume=82&amp;issue=10&amp;publication_year=2022&amp;pages=1878-1893.e10&amp;pmid=35537448&amp;doi=10.1016/j.molcel.2022.04.017&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330098.ref098">
<span class="label">98.</span><cite>Liu J, Perumal NB, Oldfield CJ, Su EW, Uversky VN, Dunker AK. Intrinsic disorder in transcription factors. Biochemistry. 2006;45(22):6873–88. doi: 10.1021/bi0602718

</cite> [<a href="https://doi.org/10.1021/bi0602718" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2538555/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16734424/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biochemistry&amp;title=Intrinsic%20disorder%20in%20transcription%20factors&amp;author=J%20Liu&amp;author=NB%20Perumal&amp;author=CJ%20Oldfield&amp;author=EW%20Su&amp;author=VN%20Uversky&amp;volume=45&amp;issue=22&amp;publication_year=2006&amp;pages=6873-88&amp;pmid=16734424&amp;doi=10.1021/bi0602718&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330098.ref099">
<span class="label">99.</span><cite>Dyson HJ, Wright PE. Role of Intrinsic Protein Disorder in the Function and Interactions of the Transcriptional Coactivators CREB-binding Protein (CBP) and p300. J Biol Chem. 2016;291(13):6714–22. doi: 10.1074/jbc.R115.692020

</cite> [<a href="https://doi.org/10.1074/jbc.R115.692020" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4807259/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26851278/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&amp;title=Role%20of%20Intrinsic%20Protein%20Disorder%20in%20the%20Function%20and%20Interactions%20of%20the%20Transcriptional%20Coactivators%20CREB-binding%20Protein%20(CBP)%20and%20p300&amp;author=HJ%20Dyson&amp;author=PE%20Wright&amp;volume=291&amp;issue=13&amp;publication_year=2016&amp;pages=6714-22&amp;pmid=26851278&amp;doi=10.1074/jbc.R115.692020&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330098.ref100">
<span class="label">100.</span><cite>Vandenbussche M, Theissen G, Van de Peer Y, Gerats T. Structural diversification and neo-functionalization during floral MADS-box gene evolution by C-terminal frameshift mutations. Nucleic Acids Res. 2003;31(15):4401–9. doi: 10.1093/nar/gkg642

</cite> [<a href="https://doi.org/10.1093/nar/gkg642" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC169922/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12888499/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=Structural%20diversification%20and%20neo-functionalization%20during%20floral%20MADS-box%20gene%20evolution%20by%20C-terminal%20frameshift%20mutations&amp;author=M%20Vandenbussche&amp;author=G%20Theissen&amp;author=Y%20Van%20de%20Peer&amp;author=T%20Gerats&amp;volume=31&amp;issue=15&amp;publication_year=2003&amp;pages=4401-9&amp;pmid=12888499&amp;doi=10.1093/nar/gkg642&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330098.ref101">
<span class="label">101.</span><cite>Cho S, Jang S, Chae S, Chung KM, Moon YH, An G, et al. Analysis of the C-terminal region of Arabidopsis thaliana APETALA1 as a transcription activation domain. Plant Mol Biol. 1999;40(3):419–29. doi: 10.1023/a:1006273127067

</cite> [<a href="https://doi.org/10.1023/a:1006273127067" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10437826/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Mol%20Biol&amp;title=Analysis%20of%20the%20C-terminal%20region%20of%20Arabidopsis%20thaliana%20APETALA1%20as%20a%20transcription%20activation%20domain&amp;author=S%20Cho&amp;author=S%20Jang&amp;author=S%20Chae&amp;author=KM%20Chung&amp;author=YH%20Moon&amp;volume=40&amp;issue=3&amp;publication_year=1999&amp;pages=419-29&amp;pmid=10437826&amp;doi=10.1023/a:1006273127067&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330098.ref102">
<span class="label">102.</span><cite>Kaufmann K, Melzer R, Theissen G. MIKC-type MADS-domain proteins: structural modularity, protein interactions and network evolution in land plants. Gene. 2005;347(2):183–98. doi: 10.1016/j.gene.2004.12.014

</cite> [<a href="https://doi.org/10.1016/j.gene.2004.12.014" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15777618/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Gene&amp;title=MIKC-type%20MADS-domain%20proteins:%20structural%20modularity,%20protein%20interactions%20and%20network%20evolution%20in%20land%20plants&amp;author=K%20Kaufmann&amp;author=R%20Melzer&amp;author=G%20Theissen&amp;volume=347&amp;issue=2&amp;publication_year=2005&amp;pages=183-98&amp;pmid=15777618&amp;doi=10.1016/j.gene.2004.12.014&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330098.ref103">
<span class="label">103.</span><cite>Egea-Cortines M, Saedler H, Sommer H. Ternary complex formation between the MADS-box proteins SQUAMOSA, DEFICIENS and GLOBOSA is involved in the control of floral architecture in Antirrhinum majus. EMBO J. 1999;18(19):5370–9. doi: 10.1093/emboj/18.19.5370

</cite> [<a href="https://doi.org/10.1093/emboj/18.19.5370" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1171606/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10508169/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=EMBO%20J&amp;title=Ternary%20complex%20formation%20between%20the%20MADS-box%20proteins%20SQUAMOSA,%20DEFICIENS%20and%20GLOBOSA%20is%20involved%20in%20the%20control%20of%20floral%20architecture%20in%20Antirrhinum%20majus&amp;author=M%20Egea-Cortines&amp;author=H%20Saedler&amp;author=H%20Sommer&amp;volume=18&amp;issue=19&amp;publication_year=1999&amp;pages=5370-9&amp;pmid=10508169&amp;doi=10.1093/emboj/18.19.5370&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330098.ref104">
<span class="label">104.</span><cite>Sridhar VV, Surendrarao A, Liu Z. APETALA1 and SEPALLATA3 interact with SEUSS to mediate transcription repression during flower development. Development. 2006;133(16):3159–66. doi: 10.1242/dev.02498

</cite> [<a href="https://doi.org/10.1242/dev.02498" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16854969/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Development&amp;title=APETALA1%20and%20SEPALLATA3%20interact%20with%20SEUSS%20to%20mediate%20transcription%20repression%20during%20flower%20development&amp;author=VV%20Sridhar&amp;author=A%20Surendrarao&amp;author=Z%20Liu&amp;volume=133&amp;issue=16&amp;publication_year=2006&amp;pages=3159-66&amp;pmid=16854969&amp;doi=10.1242/dev.02498&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330098.ref105">
<span class="label">105.</span><cite>Shiraishi H, Okada K, Shimura Y. Nucleotide sequences recognized by the AGAMOUS MADS domain of Arabidopsis thaliana in vitro. Plant J. 1993;4(2):385–98. doi: 10.1046/j.1365-313x.1993.04020385.x

</cite> [<a href="https://doi.org/10.1046/j.1365-313x.1993.04020385.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/8106084/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20J&amp;title=Nucleotide%20sequences%20recognized%20by%20the%20AGAMOUS%20MADS%20domain%20of%20Arabidopsis%20thaliana%20in%20vitro&amp;author=H%20Shiraishi&amp;author=K%20Okada&amp;author=Y%20Shimura&amp;volume=4&amp;issue=2&amp;publication_year=1993&amp;pages=385-98&amp;pmid=8106084&amp;doi=10.1046/j.1365-313x.1993.04020385.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330098.ref106">
<span class="label">106.</span><cite>Mizukami Y, Ma H. Determination of Arabidopsis floral meristem identity by AGAMOUS. Plant Cell. 1997;9(3):393–408. doi: 10.1105/tpc.9.3.393

</cite> [<a href="https://doi.org/10.1105/tpc.9.3.393" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC156926/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9090883/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Cell&amp;title=Determination%20of%20Arabidopsis%20floral%20meristem%20identity%20by%20AGAMOUS&amp;author=Y%20Mizukami&amp;author=H%20Ma&amp;volume=9&amp;issue=3&amp;publication_year=1997&amp;pages=393-408&amp;pmid=9090883&amp;doi=10.1105/tpc.9.3.393&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330098.ref107">
<span class="label">107.</span><cite>Puranik S, Acajjaoui S, Conn S, Costa L, Conn V, Vial A, et al. Structural basis for the oligomerization of the MADS domain transcription factor SEPALLATA3 in Arabidopsis. Plant Cell. 2014;26(9):3603–15. doi: 10.1105/tpc.114.127910

</cite> [<a href="https://doi.org/10.1105/tpc.114.127910" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4213154/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25228343/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Cell&amp;title=Structural%20basis%20for%20the%20oligomerization%20of%20the%20MADS%20domain%20transcription%20factor%20SEPALLATA3%20in%20Arabidopsis&amp;author=S%20Puranik&amp;author=S%20Acajjaoui&amp;author=S%20Conn&amp;author=L%20Costa&amp;author=V%20Conn&amp;volume=26&amp;issue=9&amp;publication_year=2014&amp;pages=3603-15&amp;pmid=25228343&amp;doi=10.1105/tpc.114.127910&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330098.ref108">
<span class="label">108.</span><cite>Amoutzias GD, Robertson DL, Van de Peer Y, Oliver SG. Choose your partners: dimerization in eukaryotic transcription factors. Trends Biochem Sci. 2008;33(5):220–9. doi: 10.1016/j.tibs.2008.02.002

</cite> [<a href="https://doi.org/10.1016/j.tibs.2008.02.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18406148/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Trends%20Biochem%20Sci&amp;title=Choose%20your%20partners:%20dimerization%20in%20eukaryotic%20transcription%20factors&amp;author=GD%20Amoutzias&amp;author=DL%20Robertson&amp;author=Y%20Van%20de%20Peer&amp;author=SG%20Oliver&amp;volume=33&amp;issue=5&amp;publication_year=2008&amp;pages=220-9&amp;pmid=18406148&amp;doi=10.1016/j.tibs.2008.02.002&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330098.ref109">
<span class="label">109.</span><cite>Martin JF, Miano JM, Hustad CM, Copeland NG, Jenkins NA, Olson EN. A Mef2 gene that generates a muscle-specific isoform via alternative mRNA splicing. Mol Cell Biol. 1994;14(3):1647–56. doi: 10.1128/mcb.14.3.1647-1656.1994

</cite> [<a href="https://doi.org/10.1128/mcb.14.3.1647-1656.1994" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC358523/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/8114702/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol%20Cell%20Biol&amp;title=A%20Mef2%20gene%20that%20generates%20a%20muscle-specific%20isoform%20via%20alternative%20mRNA%20splicing&amp;author=JF%20Martin&amp;author=JM%20Miano&amp;author=CM%20Hustad&amp;author=NG%20Copeland&amp;author=NA%20Jenkins&amp;volume=14&amp;issue=3&amp;publication_year=1994&amp;pages=1647-56&amp;pmid=8114702&amp;doi=10.1128/mcb.14.3.1647-1656.1994&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330098.ref110">
<span class="label">110.</span><cite>Molkentin JD, Li L, Olson EN. Phosphorylation of the MADS-Box transcription factor MEF2C enhances its DNA binding activity. J Biol Chem. 1996;271(29):17199–204. doi: 10.1074/jbc.271.29.17199

</cite> [<a href="https://doi.org/10.1074/jbc.271.29.17199" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/8663403/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&amp;title=Phosphorylation%20of%20the%20MADS-Box%20transcription%20factor%20MEF2C%20enhances%20its%20DNA%20binding%20activity&amp;author=JD%20Molkentin&amp;author=L%20Li&amp;author=EN%20Olson&amp;volume=271&amp;issue=29&amp;publication_year=1996&amp;pages=17199-204&amp;pmid=8663403&amp;doi=10.1074/jbc.271.29.17199&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330098.ref111">
<span class="label">111.</span><cite>Wang X, She H, Mao Z. Phosphorylation of neuronal survival factor MEF2D by glycogen synthase kinase 3beta in neuronal apoptosis. J Biol Chem. 2009;284(47):32619–26. doi: 10.1074/jbc.M109.067785

</cite> [<a href="https://doi.org/10.1074/jbc.M109.067785" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2781676/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19801631/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&amp;title=Phosphorylation%20of%20neuronal%20survival%20factor%20MEF2D%20by%20glycogen%20synthase%20kinase%203beta%20in%20neuronal%20apoptosis&amp;author=X%20Wang&amp;author=H%20She&amp;author=Z%20Mao&amp;volume=284&amp;issue=47&amp;publication_year=2009&amp;pages=32619-26&amp;pmid=19801631&amp;doi=10.1074/jbc.M109.067785&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330098.ref112">
<span class="label">112.</span><cite>Darling AL, Uversky VN. Intrinsic disorder and posttranslational modifications: The darker side of the biological dark matter. Front Genet. 2018;4:9.</cite> [<a href="https://doi.org/10.3389/fgene.2018.00158" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5945825/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29780404/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Front%20Genet&amp;title=Intrinsic%20disorder%20and%20posttranslational%20modifications:%20The%20darker%20side%20of%20the%20biological%20dark%20matter&amp;author=AL%20Darling&amp;author=VN%20Uversky&amp;volume=4&amp;publication_year=2018&amp;pages=9&amp;pmid=29780404&amp;doi=10.3389/fgene.2018.00158&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330098.ref113">
<span class="label">113.</span><cite>Jin F, Grater F. How multisite phosphorylation impacts the conformations of intrinsically disordered proteins. PLoS Comput Biol. 2021;17:e1008939.</cite> [<a href="https://doi.org/10.1371/journal.pcbi.1008939" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8148376/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33945530/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20Comput%20Biol&amp;title=How%20multisite%20phosphorylation%20impacts%20the%20conformations%20of%20intrinsically%20disordered%20proteins&amp;author=F%20Jin&amp;author=F%20Grater&amp;volume=17&amp;publication_year=2021&amp;pmid=33945530&amp;doi=10.1371/journal.pcbi.1008939&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330098.ref114">
<span class="label">114.</span><cite>Nosella ML, Forman-Kay JD. Phosphorylation-dependent regulation of messenger RNA transcription, processing and translation within biomolecular condensates. Curr Opin Cell Biol. 2021;69:30–40. doi: 10.1016/j.ceb.2020.12.007

</cite> [<a href="https://doi.org/10.1016/j.ceb.2020.12.007" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33450720/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Curr%20Opin%20Cell%20Biol&amp;title=Phosphorylation-dependent%20regulation%20of%20messenger%20RNA%20transcription,%20processing%20and%20translation%20within%20biomolecular%20condensates&amp;author=ML%20Nosella&amp;author=JD%20Forman-Kay&amp;volume=69&amp;publication_year=2021&amp;pages=30-40&amp;pmid=33450720&amp;doi=10.1016/j.ceb.2020.12.007&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330098.ref115">
<span class="label">115.</span><cite>Shnitkind S, Martinez-Yamout MA, Dyson HJ, Wright PE. Structural Basis for Graded Inhibition of CREB:DNA Interactions by Multisite Phosphorylation. Biochemistry. 2018;57(51):6964–72. doi: 10.1021/acs.biochem.8b01092

</cite> [<a href="https://doi.org/10.1021/acs.biochem.8b01092" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6474821/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30507144/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biochemistry&amp;title=Structural%20Basis%20for%20Graded%20Inhibition%20of%20CREB:DNA%20Interactions%20by%20Multisite%20Phosphorylation&amp;author=S%20Shnitkind&amp;author=MA%20Martinez-Yamout&amp;author=HJ%20Dyson&amp;author=PE%20Wright&amp;volume=57&amp;issue=51&amp;publication_year=2018&amp;pages=6964-72&amp;pmid=30507144&amp;doi=10.1021/acs.biochem.8b01092&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330098.ref116">
<span class="label">116.</span><cite>De Bodt S, Raes J, Van de Peer Y, Theissen G. And then there were many: MADS goes genomic. Trends Plant Sci. 2003;8(10):475–83. doi: 10.1016/j.tplants.2003.09.006

</cite> [<a href="https://doi.org/10.1016/j.tplants.2003.09.006" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/14557044/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Trends%20Plant%20Sci&amp;title=And%20then%20there%20were%20many:%20MADS%20goes%20genomic&amp;author=S%20De%20Bodt&amp;author=J%20Raes&amp;author=Y%20Van%20de%20Peer&amp;author=G%20Theissen&amp;volume=8&amp;issue=10&amp;publication_year=2003&amp;pages=475-83&amp;pmid=14557044&amp;doi=10.1016/j.tplants.2003.09.006&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330098.ref117">
<span class="label">117.</span><cite>Veron AS, Kaufmann K, Bornberg-Bauer E. Evidence of interaction network evolution by whole-genome duplications: a case study in MADS-box proteins. Mol Biol Evol. 2007;24(3):670–8. doi: 10.1093/molbev/msl197

</cite> [<a href="https://doi.org/10.1093/molbev/msl197" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17175526/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol%20Biol%20Evol&amp;title=Evidence%20of%20interaction%20network%20evolution%20by%20whole-genome%20duplications:%20a%20case%20study%20in%20MADS-box%20proteins&amp;author=AS%20Veron&amp;author=K%20Kaufmann&amp;author=E%20Bornberg-Bauer&amp;volume=24&amp;issue=3&amp;publication_year=2007&amp;pages=670-8&amp;pmid=17175526&amp;doi=10.1093/molbev/msl197&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330098.ref118">
<span class="label">118.</span><cite>Strader L, Weijers D, Wagner D. Plant transcription factors - being in the right place with the right company. Curr Opin Plant Biol. 2022;65:102136. doi: 10.1016/j.pbi.2021.102136

</cite> [<a href="https://doi.org/10.1016/j.pbi.2021.102136" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8844091/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34856504/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Curr%20Opin%20Plant%20Biol&amp;title=Plant%20transcription%20factors%20-%20being%20in%20the%20right%20place%20with%20the%20right%20company&amp;author=L%20Strader&amp;author=D%20Weijers&amp;author=D%20Wagner&amp;volume=65&amp;publication_year=2022&amp;pages=102136&amp;pmid=34856504&amp;doi=10.1016/j.pbi.2021.102136&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330098.ref119">
<span class="label">119.</span><cite>Zahn LM, Leebens-Mack JH, Arrington JM, Hu Y, Landherr LL, DePamphilis CW. Conservation and divergence in the AGAMOUS subfamily of MADS-box genes: evidence of independent sub- and neofunctionalization events. Evol Dev. 2006;8:30–45.
</cite> [<a href="https://doi.org/10.1111/j.1525-142X.2006.05073.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16409381/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Evol%20Dev&amp;title=Conservation%20and%20divergence%20in%20the%20AGAMOUS%20subfamily%20of%20MADS-box%20genes:%20evidence%20of%20independent%20sub-%20and%20neofunctionalization%20events&amp;author=LM%20Zahn&amp;author=JH%20Leebens-Mack&amp;author=JM%20Arrington&amp;author=Y%20Hu&amp;author=LL%20Landherr&amp;volume=8&amp;publication_year=2006&amp;pages=30-45&amp;pmid=16409381&amp;doi=10.1111/j.1525-142X.2006.05073.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330098.ref120">
<span class="label">120.</span><cite>Bemer M, Wolters-Arts M, Grossniklaus U, Angenent GC. The MADS domain protein DIANA acts together with AGAMOUS-LIKE80 to specify the central cell in Arabidopsis ovules. Plant Cell. 2008;20(8):2088–101. doi: 10.1105/tpc.108.058958

</cite> [<a href="https://doi.org/10.1105/tpc.108.058958" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2553608/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18713950/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Cell&amp;title=The%20MADS%20domain%20protein%20DIANA%20acts%20together%20with%20AGAMOUS-LIKE80%20to%20specify%20the%20central%20cell%20in%20Arabidopsis%20ovules&amp;author=M%20Bemer&amp;author=M%20Wolters-Arts&amp;author=U%20Grossniklaus&amp;author=GC%20Angenent&amp;volume=20&amp;issue=8&amp;publication_year=2008&amp;pages=2088-101&amp;pmid=18713950&amp;doi=10.1105/tpc.108.058958&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330098.ref121">
<span class="label">121.</span><cite>Colombo M, Masiero S, Vanzulli S, Lardelli P, Kater MM, Colombo L. AGL23, a type I MADS-box gene that controls female gametophyte and embryo development in Arabidopsis. Plant J. 2008;54(6):1037–48. doi: 10.1111/j.1365-313X.2008.03485.x

</cite> [<a href="https://doi.org/10.1111/j.1365-313X.2008.03485.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18346189/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20J&amp;title=AGL23,%20a%20type%20I%20MADS-box%20gene%20that%20controls%20female%20gametophyte%20and%20embryo%20development%20in%20Arabidopsis&amp;author=M%20Colombo&amp;author=S%20Masiero&amp;author=S%20Vanzulli&amp;author=P%20Lardelli&amp;author=MM%20Kater&amp;volume=54&amp;issue=6&amp;publication_year=2008&amp;pages=1037-48&amp;pmid=18346189&amp;doi=10.1111/j.1365-313X.2008.03485.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330098.ref122">
<span class="label">122.</span><cite>Kang IH, Steffen JG, Portereiko MF, Lloyd A, Drews GN. The AGL62 MADS domain protein regulates cellularization during endosperm development in Arabidopsis. Plant Cell. 2008;20:635–47.
</cite> [<a href="https://doi.org/10.1105/tpc.107.055137" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2329934/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18334668/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Cell&amp;title=The%20AGL62%20MADS%20domain%20protein%20regulates%20cellularization%20during%20endosperm%20development%20in%20Arabidopsis&amp;author=IH%20Kang&amp;author=JG%20Steffen&amp;author=MF%20Portereiko&amp;author=A%20Lloyd&amp;author=GN%20Drews&amp;volume=20&amp;publication_year=2008&amp;pages=635-47&amp;pmid=18334668&amp;doi=10.1105/tpc.107.055137&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330098.ref123">
<span class="label">123.</span><cite>Köhler C, Hennig L, Spillane C, Pien S, Gruissem W, Grossniklaus U. The Polycomb-group protein MEDEA regulates seed development by controlling expression of the MADS-box gene PHERES1. Genes Dev. 2003;17:1540–53.
</cite> [<a href="https://doi.org/10.1101/gad.257403" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC196083/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12815071/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genes%20Dev&amp;title=The%20Polycomb-group%20protein%20MEDEA%20regulates%20seed%20development%20by%20controlling%20expression%20of%20the%20MADS-box%20gene%20PHERES1&amp;author=C%20K%C3%B6hler&amp;author=L%20Hennig&amp;author=C%20Spillane&amp;author=S%20Pien&amp;author=W%20Gruissem&amp;volume=17&amp;publication_year=2003&amp;pages=1540-53&amp;pmid=12815071&amp;doi=10.1101/gad.257403&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330098.ref124">
<span class="label">124.</span><cite>Zhang WJ, Zhou Y, Zhang Y, Su YH, Xu T. Protein phosphorylation: A molecular switch in plant signaling. Cell Rep. 2023;42(7):112729. doi: 10.1016/j.celrep.2023.112729

</cite> [<a href="https://doi.org/10.1016/j.celrep.2023.112729" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37405922/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Rep&amp;title=Protein%20phosphorylation:%20A%20molecular%20switch%20in%20plant%20signaling&amp;author=WJ%20Zhang&amp;author=Y%20Zhou&amp;author=Y%20Zhang&amp;author=YH%20Su&amp;author=T%20Xu&amp;volume=42&amp;issue=7&amp;publication_year=2023&amp;pages=112729&amp;pmid=37405922&amp;doi=10.1016/j.celrep.2023.112729&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330098.ref125">
<span class="label">125.</span><cite>Borcherds W, Bremer A, Borgia MB, Mittag T. How do intrinsically disordered protein regions encode a driving force for liquid-liquid phase separation?
Curr Opin Struct Biol. 2021;67:41–50. doi: 10.1016/j.sbi.2020.09.004

</cite> [<a href="https://doi.org/10.1016/j.sbi.2020.09.004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8044266/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33069007/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Curr%20Opin%20Struct%20Biol&amp;title=How%20do%20intrinsically%20disordered%20protein%20regions%20encode%20a%20driving%20force%20for%20liquid-liquid%20phase%20separation?&amp;author=W%20Borcherds&amp;author=A%20Bremer&amp;author=MB%20Borgia&amp;author=T%20Mittag&amp;volume=67&amp;publication_year=2021&amp;pages=41-50&amp;pmid=33069007&amp;doi=10.1016/j.sbi.2020.09.004&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330098.ref126">
<span class="label">126.</span><cite>Pei G, Lyons H, Li P, Sabari BR. Transcription regulation by biomolecular condensates. Nat Rev Mol Cell Biol. 2025;26(3):213–36. doi: 10.1038/s41580-024-00789-x

</cite> [<a href="https://doi.org/10.1038/s41580-024-00789-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC12186837/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39516712/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Mol%20Cell%20Biol&amp;title=Transcription%20regulation%20by%20biomolecular%20condensates&amp;author=G%20Pei&amp;author=H%20Lyons&amp;author=P%20Li&amp;author=BR%20Sabari&amp;volume=26&amp;issue=3&amp;publication_year=2025&amp;pages=213-36&amp;pmid=39516712&amp;doi=10.1038/s41580-024-00789-x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330098.ref127">
<span class="label">127.</span><cite>Sabari BR. Biomolecular Condensates and Gene Activation in Development and Disease. Dev Cell. 2020;55(1):84–96. doi: 10.1016/j.devcel.2020.09.005

</cite> [<a href="https://doi.org/10.1016/j.devcel.2020.09.005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33049213/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Dev%20Cell&amp;title=Biomolecular%20Condensates%20and%20Gene%20Activation%20in%20Development%20and%20Disease&amp;author=BR%20Sabari&amp;volume=55&amp;issue=1&amp;publication_year=2020&amp;pages=84-96&amp;pmid=33049213&amp;doi=10.1016/j.devcel.2020.09.005&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330098.ref128">
<span class="label">128.</span><cite>Wei M-T, Chang Y-C, Shimobayashi SF, Shin Y, Strom AR, Brangwynne CP. Nucleated transcriptional condensates amplify gene expression. Nat Cell Biol. 2020;22(10):1187–96. doi: 10.1038/s41556-020-00578-6

</cite> [<a href="https://doi.org/10.1038/s41556-020-00578-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32929202/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Cell%20Biol&amp;title=Nucleated%20transcriptional%20condensates%20amplify%20gene%20expression&amp;author=M-T%20Wei&amp;author=Y-C%20Chang&amp;author=SF%20Shimobayashi&amp;author=Y%20Shin&amp;author=AR%20Strom&amp;volume=22&amp;issue=10&amp;publication_year=2020&amp;pages=1187-96&amp;pmid=32929202&amp;doi=10.1038/s41556-020-00578-6&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330098.ref129">
<span class="label">129.</span><cite>Zarin T, Strome B, Nguyen Ba AN, Alberti S, Forman-Kay JD, Moses AM. Proteome-wide signatures of function in highly diverged intrinsically disordered regions. Elife. 2019;8:e46883. doi: 10.7554/eLife.46883

</cite> [<a href="https://doi.org/10.7554/eLife.46883" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6634968/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31264965/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Elife&amp;title=Proteome-wide%20signatures%20of%20function%20in%20highly%20diverged%20intrinsically%20disordered%20regions&amp;author=T%20Zarin&amp;author=B%20Strome&amp;author=AN%20Nguyen%20Ba&amp;author=S%20Alberti&amp;author=JD%20Forman-Kay&amp;volume=8&amp;publication_year=2019&amp;pmid=31264965&amp;doi=10.7554/eLife.46883&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330098.ref130">
<span class="label">130.</span><cite>Singleton MD, Eisen MB. Evolutionary analyses of intrinsically disordered regions reveal widespread signals of conservation. PLoS Comput Biol. 2024;20(4):e1012028. doi: 10.1371/journal.pcbi.1012028

</cite> [<a href="https://doi.org/10.1371/journal.pcbi.1012028" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11075841/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38662765/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20Comput%20Biol&amp;title=Evolutionary%20analyses%20of%20intrinsically%20disordered%20regions%20reveal%20widespread%20signals%20of%20conservation&amp;author=MD%20Singleton&amp;author=MB%20Eisen&amp;volume=20&amp;issue=4&amp;publication_year=2024&amp;pmid=38662765&amp;doi=10.1371/journal.pcbi.1012028&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330098.ref131">
<span class="label">131.</span><cite>Mann R, Notani D. Transcription factor condensates and signaling driven transcription. Nucleus. 2023;14:2205758.
</cite> [<a href="https://doi.org/10.1080/19491034.2023.2205758" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10155639/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37129580/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleus&amp;title=Transcription%20factor%20condensates%20and%20signaling%20driven%20transcription&amp;author=R%20Mann&amp;author=D%20Notani&amp;volume=14&amp;publication_year=2023&amp;pages=2205758&amp;pmid=37129580&amp;doi=10.1080/19491034.2023.2205758&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330098.ref132">
<span class="label">132.</span><cite>Boija A, Klein IA, Sabari BR, Dall’Agnese A, Coffey EL, Zamudio AV, et al. Transcription Factors Activate Genes through the Phase-Separation Capacity of Their Activation Domains. Cell. 2018;175(7):1842-1855.e16. doi: 10.1016/j.cell.2018.10.042

</cite> [<a href="https://doi.org/10.1016/j.cell.2018.10.042" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6295254/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30449618/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Transcription%20Factors%20Activate%20Genes%20through%20the%20Phase-Separation%20Capacity%20of%20Their%20Activation%20Domains&amp;author=A%20Boija&amp;author=IA%20Klein&amp;author=BR%20Sabari&amp;author=A%20Dall%E2%80%99Agnese&amp;author=EL%20Coffey&amp;volume=175&amp;issue=7&amp;publication_year=2018&amp;pmid=30449618&amp;doi=10.1016/j.cell.2018.10.042&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section></section><article class="sub-article" id="pone.0330098.r001"><section class="pmc-layout__citation font-secondary font-xs"><div>PLoS One. 2025 Aug 22;20(8):e0330098. doi: <a href="https://doi.org/10.1371/journal.pone.0330098.r001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1371/journal.pone.0330098.r001</a>
</div></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Author response to Decision Letter 0</h1></hgroup><ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="anp_a.c" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a.c" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="anp_a.c" class="d-panel p" style="display: none"><div class="notes p"><section id="historyfront-stub1" class="history"><p>Collection date 2025.</p></section></div></div>
<div id="clp_a.c" class="d-panel p" style="display: none"><div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div></div>
</div>
<div></div>
</div></section><section class="body sub-article-body"><hr class="headless">
<section class="bt xbox font-sm" id="pone.0330098.box001"><section id="sec0122"><h2 class="pmc_sec_title">Transfer Alert</h2>
<p>This paper was transferred from another journal. As a result, its full editorial history (including decision letters, peer reviews and author responses) may not be present.</p></section></section><p>
<em>25 Feb 2025</em>
</p></section></article><article class="sub-article" id="pone.0330098.r002"><section class="pmc-layout__citation font-secondary font-xs"><div>PLoS One. doi: <a href="https://doi.org/10.1371/journal.pone.0330098.r002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1371/journal.pone.0330098.r002</a>
</div></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Decision Letter 0</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Prakash%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Ajit Prakash</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Ajit Prakash</span></h3>
<div>Academic Editor</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Prakash%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Ajit Prakash</span></a>
</div>
</div>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a.d" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="clp_a.d" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a.d" class="d-panel p" style="display: none">
<h4 class="font-secondary">Roles</h4>
<div class="p">
<strong class="contrib"><span class="name western">Ajit Prakash</span></strong>: <span class="role">Academic Editor</span>
</div>
</div>
<div id="clp_a.d" class="d-panel p" style="display: none">
<div>© 2025 Ajit Prakash</div>
<p>This is an open access article distributed under the terms of the <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Creative Commons Attribution License</a> , which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div></div>
</div></section><section class="body sub-article-body"><hr class="headless">
<p>
<em>30 Apr 2025</em>
</p>
<p>Dear Dr. Garay Arroyo,</p>
<p>Thank you for submitting your manuscript to PLOS ONE. After careful consideration, we feel that it has merit but does not fully meet PLOS ONE’s publication criteria as it currently stands. Therefore, we invite you to submit a revised version of the manuscript that addresses the points raised during the review process.</p>
<p>Please submit your revised manuscript by Jun 14 2025 11:59PM. If you will need more time than this to complete your revisions, please reply to this message or contact the journal office at <span>plosone@plos.org</span> . When you're ready to submit your revision, log on to <a href="https://www.editorialmanager.com/pone/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.editorialmanager.com/pone/</a> and select the 'Submissions Needing Revision' folder to locate your manuscript file.</p>
<ul class="list" style="list-style-type:disc">
<li><p>A rebuttal letter that responds to each point raised by the academic editor and reviewer(s). You should upload this letter as a separate file labeled 'Response to Reviewers'.</p></li>
<li><p>A marked-up copy of your manuscript that highlights changes made to the original version. You should upload this as a separate file labeled 'Revised Manuscript with Track Changes'.</p></li>
<li><p>An unmarked version of your revised paper without tracked changes. You should upload this as a separate file labeled 'Manuscript'.</p></li>
</ul>
<p>If you would like to make changes to your financial disclosure, please include your updated statement in your cover letter. Guidelines for resubmitting your figure files are available below the reviewer comments at the end of this letter.</p>
<p>If applicable, we recommend that you deposit your laboratory protocols in protocols.io to enhance the reproducibility of your results. Protocols.io assigns your protocol its own identifier (DOI) so that it can be cited independently in the future. For instructions see: <a href="https://journals.plos.org/plosone/s/submission-guidelines#loc-laboratory-protocols" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://journals.plos.org/plosone/s/submission-guidelines#loc-laboratory-protocols</a> . Additionally, PLOS ONE offers an option for publishing peer-reviewed Lab Protocol articles, which describe protocols hosted on protocols.io. Read more information on sharing protocols at <a href="https://plos.org/protocols?utm_medium=editorial-email&amp;utm_source=authorletters&amp;utm_campaign=protocols" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://plos.org/protocols?utm_medium=editorial-email&amp;utm_source=authorletters&amp;utm_campaign=protocols</a> .</p>
<p>We look forward to receiving your revised manuscript.</p>
<p>Kind regards,</p>
<p>Ajit Prakash, PhD</p>
<p>Academic Editor</p>
<p>PLOS ONE</p>
<p>Journal Requirements:</p>
<p>When submitting your revision, we need you to address these additional requirements.</p>
<p>1. Please ensure that your manuscript meets PLOS ONE's style requirements, including those for file naming. The PLOS ONE style templates can be found at</p>
<p><a href="https://journals.plos.org/plosone/s/file?id=wjVg/PLOSOne_formatting_sample_main_body.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://journals.plos.org/plosone/s/file?id=wjVg/PLOSOne_formatting_sample_main_body.pdf</a> and</p>
<p>
<a href="https://journals.plos.org/plosone/s/file?id=ba62/PLOSOne_formatting_sample_title_authors_affiliations.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://journals.plos.org/plosone/s/file?id=ba62/PLOSOne_formatting_sample_title_authors_affiliations.pdf</a>
</p>
<p>2. Thank you for stating in your Funding Statement: [This work was partially financed by the projects PAPIIT No. IN213524 from Universidad Nacional Autónoma de México (UNAM), and project No. CBF2023-2024-1002 from the Consejo Nacional de Humanidades, Ciencias y Tecnologías</p>
<p>(CONAHCyT) granted to A.G.A.; and project No. CF2023-I-503 from the Consejo</p>
<p>Nacional de Humanidades, Ciencias y Tecnologías (CONAHCyT) granted to A.A.C.].</p>
<p>Please provide an amended statement that declares *all* the funding or sources of support (whether external or internal to your organization) received during this study, as detailed online in our guide for authors at <a href="http://journals.plos.org/plosone/s/submit-now.%C2%A0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://journals.plos.org/plosone/s/submit-now. </a> Please also include the statement “There was no additional external funding received for this study.” in your updated Funding Statement.</p>
<p>Please include your amended Funding Statement within your cover letter. We will change the online submission form on your behalf.</p>
<p>3. Please update your submission to use the PLOS LaTeX template. The template and more information on our requirements for LaTeX submissions can be found at <a href="http://journals.plos.org/plosone/s/latex" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://journals.plos.org/plosone/s/latex</a></p>
<p>4. Please upload a new copy of Figure S5 as the detail is not clear. Please follow the link for more information: <a href="https://blogs.plos.org/plos/2019/06/looking-good-tips-for-creating-your-plos-figures-graphics/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://blogs.plos.org/plos/2019/06/looking-good-tips-for-creating-your-plos-figures-graphics/</a>"" <a href="https://blogs.plos.org/plos/2019/06/looking-good-tips-for-creating-your-plos-figures-graphics/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://blogs.plos.org/plos/2019/06/looking-good-tips-for-creating-your-plos-figures-graphics/</a></p>
<p>5. Thank you for stating the following financial disclosure: [This work was partially financed by the projects PAPIIT No. IN213524 from Universidad Nacional Autónoma de México (UNAM), and project No. CBF2023-2024-1002 from the Consejo Nacional de Humanidades, Ciencias y Tecnologías</p>
<p>(CONAHCyT) granted to A.G.A.; and project No. CF2023-I-503 from the Consejo</p>
<p>Nacional de Humanidades, Ciencias y Tecnologías (CONAHCyT) granted to A.A.C.].</p>
<p>Please state what role the funders took in the study.  If the funders had no role, please state: ""The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.""</p>
<p>If this statement is not correct you must amend it as needed.</p>
<p>Please include this amended Role of Funder statement in your cover letter; we will change the online submission form on your behalf.</p>
<p>Additional Editor Comments:</p>
<p>Please address the concerns raised by reviewers and provide point to point response.</p>
<p>[Note: HTML markup is below. Please do not edit.]</p>
<p>Reviewers' comments:</p>
<p>Reviewer's Responses to Questions</p>
<p>
<strong>Comments to the Author</strong>
</p>
<p>1. Is the manuscript technically sound, and do the data support the conclusions?</p>
<p>Reviewer #1: Yes</p>
<p>Reviewer #2: Yes</p>
<p>**********</p>
<p>2. Has the statistical analysis been performed appropriately and rigorously? --&gt;?&gt;</p>
<p>Reviewer #1: Yes</p>
<p>Reviewer #2: No</p>
<p>**********</p>
<p>3. Have the authors made all data underlying the findings in their manuscript fully available??&gt;</p>
<p>The <a href="http://www.plosone.org/static/policies.action#sharing" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PLOS Data policy</a></p>
<p>Reviewer #1: Yes</p>
<p>Reviewer #2: Yes</p>
<p>**********</p>
<p>4. Is the manuscript presented in an intelligible fashion and written in standard English??&gt;</p>
<p>Reviewer #1: No</p>
<p>Reviewer #2: Yes</p>
<p>**********</p>
<p>Reviewer #1: The authors have explored the role of intrinsically disordered regions (IDRs) in transcription factors, focusing on MADS-domain proteins in Arabidopsis thaliana. IDRs contribute to protein-protein interactions and functional flexibility. In this study via bioinformatic analysis revealed that most Arabidopsis MADS-domain proteins have IDRs in their C-terminal region, with Type II proteins exhibiting more disorder than Type I. Similar IDRs were found in orthologous proteins from non-plant species (Drosophila, Saccharomyces cerevisiae, and Homo sapiens), often with longer disordered regions. Conserved motifs and putative activation domains were identified in Arabidopsis MADS-domain proteins, suggesting interactions with co-regulatory partners. These findings highlight the evolutionary conservation of structural disorder in MADS-domain proteins and its role in transcriptional regulation.</p>
<p>Comments:</p>
<p>I think one of the main concerns of the current submission is more editorial. The manuscript was heavily disorganized, especially with figure legends. It was oddly placed and poorly labeled.</p>
<p>Figure 1) In the box plot showing MDP for Type 1 MAD domain, there is no significant difference between complete protein and C-terminal region like shown in Type 2. The authors have not commented on how that impacts the conclusion on C-terminal IDPs. Does that effect the difference in the protein length and RIDAO Map score.</p>
<p>Figure 2) The pattern of phosphorylation sites in Type 1 and Type 2 different. Phosphorylation can be a key event in stabilizing proteins, and based on the data on Type 2 have higher propensity of phosphorylation in c-terminal and with a small incremental effect. This does not alone support the conclusion that c -terminal of MADS protein have higher phosphorylation site, there should be more analysis done. Comparing non-MADS IDPs can be useful.</p>
<p>Figure 5) PADI score is dependent on sequence composition, supporting this analysis with different PADI threshold and performance matrix with sensitivity, specificity, ppv, npv with same data can increase the confidence on this analysis.</p>
<p>Figure 6) IDPs drive condensate formation and Type 2 have more MDP index and correlation in figure 6 how Type 1 have more LLPS forming propensity than Type 2. The authors have addressed the discrepancy in discussion. It would add more value if there is more similar analysis on known IDPs and LLP forming index. Additionally what if they only input C-terminal region , will that change the LLPS forming Index ?</p>
<p>The discussion is really long, it will help the manuscript if the authors try to keep it more concise in general.</p>
<p>Reviewer #2: This manuscript presents a bioinformatic analysis of intrinsically disordered regions in MADS-domain transcription factors across diverse taxa. The authors demonstrate that C-terminal disorder is consistently present in both plant and non-plant MADS-domain proteins, with Type II proteins showing higher overall disorder than Type I proteins. They identify conserved motifs within these disordered regions that appear clade-specific, along with overlapping Molecular Recognition Features particularly in Type II proteins. The study further characterizes potential activation domains and liquid-liquid phase separation propensities, finding more activation domains in Type I proteins. While the conservation of these structural features across evolutionary distant taxa strongly suggests functional importance in transcriptional regulation, I have some concerns about the exclusively computational nature of the analysis. The study would benefit from experimental validation of key predictions, clearer distinction between correlation and causality in evolutionary conservation, more robust statistical analysis, and a more integrated model explaining how these various structural features collectively contribute to MADS-domain protein function.</p>
<p>Concern(s)/ Limitations and comments:</p>
<p>- My core concern is about the experimental validation. The study relies exclusively on bioinformatic predictions without any experimental validation of the identified IDRs, MoRFs, or activation domains. While computational approaches provide valuable insights, experimental confirmation is needed to significantly strengthen the findings. I would suggest authors include at least some experimental validation for key findings, particularly for the identified motifs and their potential functional roles. Techniques such as circular dichroism (CD) spectroscopy could confirm the disordered nature of the C-terminal regions, while yeast two-hybrid or co-immunoprecipitation assays could validate protein-protein interactions mediated by the identified MoRFs. Moreover, directed mutagenesis of conserved motifs followed by functional assays would help establish their biological significance, as demonstrated by Cho et al. (1999) for the C-terminal region of APETALA1.</p>
<p>- While the conservation of structural disorder across taxa is interesting, the manuscript doesn't establish whether this conservation reflects functional constraints or simply represents a structural consequence of sequence evolution. Further discussion on this point would strengthen the paper. As noted by Pancsa and Tompa (2012), intrinsic disorder can be maintained through both purifying selection and neutral evolution. The authors could address this by analyzing the rates of synonymous versus non-synonymous substitutions in the disordered regions compared to ordered domains, which would provide evidence for selective pressure maintaining these IDRs. Additionally, examining the conservation of physicochemical properties rather than just sequence could help distinguish between functional conservation and structural coincidence.</p>
<p>- Although the manuscript speculates on the functional significance of the identified motifs and structural features, it doesn't provide concrete evidence for their roles. I would suggest exploring how these findings connect to known functional differences between Type I and Type II MADS-domain proteins. For instance, the predominance of activation domains in Type I proteins could be related to their specific roles in gametogenesis, while the different patterns of MoRFs in Type II proteins might explain their more diverse developmental functions. Supporting this connection with existing phenotypic data from mutant studies would significantly strengthen the biological relevance of the identified structural features. The recent work by Morffy et al. (2024) on plant transcriptional activation domains could provide a framework for such functional integration.</p>
<p>- While the paper identifies various structural features (IDRs, MoRFs, ADs), it doesn't fully integrate these findings into a cohesive model of how these features might work together to influence protein function. I believe the authors should develop a unified mechanistic model showing how the identified structural elements (IDRs, MoRFs, and ADs) work together to regulate MADS-domain protein function in transcriptional complexes, similar to the framework proposed by Wright and Dyson (2015) for IDR-mediated assembly of transcriptional machinery.</p>
<p>- I believe the comparison between plant and non-plant MADS-domain proteins needs to be expanded to better understand the evolutionary conservation and divergence of these structural features. The current analysis shows that non-plant MADS-domain proteins have longer IDRs, but doesn't explore whether this reflects different functional constraints or adaptations. I suggest including a more detailed phylogenetic analysis that traces the evolution of specific structural features across lineages, similar to the approach taken by Davey et al. (2012) for short linear motifs. This could reveal whether certain structural properties emerged independently in different lineages or were present in the common ancestor of all MADS-domain proteins.</p>
<p>- I believe the statistical analyses need strengthening, especially when comparing Type I and Type II proteins. While Wilcoxon tests are used appropriately, the authors should implement corrections for multiple testing and address how the unequal sample sizes (58 Type I vs. 42 Type II proteins) might affect statistical power. I recommend incorporating bootstrapping or permutation tests to increase confidence in the reported differences, following approaches described by Efron and Tibshirani (1994) for biological data analysis.</p>
<p>**********</p>
<p><a href="https://journals.plos.org/plosone/s/editorial-and-peer-review-process#loc-peer-review-history" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">what does this mean?</a> ). If published, this will include your full peer review and any attached files.</p>
<p>If you choose “no”, your identity will remain anonymous but your review may still be made public.</p>
<p><strong>Do you want your identity to be public for this peer review?</strong> For information about this choice, including consent withdrawal, please see our <a href="https://www.plos.org/privacy-policy" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Privacy Policy</a></p>
<p>Reviewer #1: No</p>
<p>Reviewer #2: <strong>Yes: </strong> Mohammad Ashhar Iqbal Khan</p>
<p>**********</p>
<p>[NOTE: If reviewer comments were submitted as an attachment file, they will be attached to this email and accessible via the submission site. Please log into your account, locate the manuscript record, and check for the action link "View Attachments". If this link does not appear, there are no attachment files.]</p>
<p>While revising your submission, please upload your figure files to the Preflight Analysis and Conversion Engine (PACE) digital diagnostic tool, <a href="https://pacev2.apexcovantage.com/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://pacev2.apexcovantage.com/</a> . PACE helps ensure that figures meet PLOS requirements. To use PACE, you must first register as a user. Registration is free. Then, login and navigate to the UPLOAD tab, where you will find detailed instructions on how to use the tool. If you encounter any issues or have any questions when using PACE, please email PLOS at <span>figures@plos.org</span> . Please note that Supporting Information files do not need this step.</p>
<section class="sm xbox font-sm" id="pone.0330098.s012"><div class="caption p">
<span>Attachment</span><p>Submitted filename: <em>Comments.pdf</em></p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12373214/bin/pone.0330098.s012.pdf" data-ga-action="click_feat_suppl" class="usa-link">pone.0330098.s012.pdf</a><sup> (189.6KB, pdf) </sup>
</div></div></section></section></article><article class="sub-article" id="pone.0330098.r003"><section class="pmc-layout__citation font-secondary font-xs"><div>PLoS One. 2025 Aug 22;20(8):e0330098. doi: <a href="https://doi.org/10.1371/journal.pone.0330098.r003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1371/journal.pone.0330098.r003</a>
</div></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Author response to Decision Letter 1</h1></hgroup><ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="anp_a.e" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a.e" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="anp_a.e" class="d-panel p" style="display: none"><div class="notes p"><section id="historyfront-stub3" class="history"><p>Collection date 2025.</p></section></div></div>
<div id="clp_a.e" class="d-panel p" style="display: none"><div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div></div>
</div>
<div></div>
</div></section><section class="body sub-article-body"><hr class="headless">
<p>
<em>4 Jul 2025</em>
</p>
<p>We thank the reviewers for their valuable comments and feedback, which have undoubtedly helped us improve the manuscript. Thanks to your comments and in the interest of greater clarity, we have decided to include both disorder scores retrieved from the RIDAO platform in the intrinsic disorder analysis: the Average Disorder Score (ADS) and the Percentage of Predicted Disordered Residues (PPDR). In the originally submitted version of the manuscript, we only included the PPDR (MDP), which corresponds to PPDR or Per-MDP in the RIDAO platform. According to RIDAO, PPDR represents the proportion of amino acids with a disorder above 0.5, indicating significant intrinsic disorder, whereas ADS reflects the overall disorder propensity of the protein. As expected, both parameters show a positive correlation. The inclusion of both metrics reinforces the evidence for the intrinsic disorder propensity obtained in the analyzed MADS domain TFs.</p>
<p>Reviewer #1: The authors have explored the role of intrinsically disordered regions (IDRs) in transcription factors, focusing on MADS-domain proteins in Arabidopsis thaliana. IDRs contribute to protein-protein interactions and functional flexibility. In this study via bioinformatic analysis revealed that most Arabidopsis MADS-domain proteins have IDRs in their C-terminal region, with Type II proteins exhibiting more disorder than Type I. Similar IDRs were found in orthologous proteins from non-plant species (Drosophila, Saccharomyces cerevisiae, and Homo sapiens), often with longer disordered regions. Conserved motifs and putative activation domains were identified in Arabidopsis MADS-domain proteins, suggesting interactions with co-regulatory partners. These findings highlight the evolutionary conservation of structural disorder in MADS-domain proteins and its role in transcriptional regulation.</p>
<p>Comments:</p>
<p>- I think one of the main concerns of the current submission is more editorial. The manuscript was heavily disorganized, especially with figure legends. It was oddly placed and poorly labeled.</p>
<p>R: Thank you for this observation. The Figure legends were included in the manuscript according to the journal´s instructions. However, as requested by the reviewer, we have included them after the references and have improved their writing to enhance readability. Additionally, we have thoroughly revised the manuscript to improve organization.</p>
<p>Figure 1) In the box plot showing MDP for Type 1 MAD domain, there is no significant difference between complete protein and C-terminal region like shown in Type 2. The authors have not commented on how that impacts the conclusion on C-terminal IDPs. Does that effect the difference in the protein length and RIDAO Map score.</p>
<p>R: The lack of differences in the disorder index between the C-terminal domain and the complete Type I MADS-domain proteins reflects the relative proportion of the C-terminal domain within the complete protein. Type I MADS-domain proteins are generally longer than Type II, and their C-terminal regions represent a significantly larger proportion of their total sequence. Therefore, the disorder scores of the C-terminal domain contribute substantially to the overall disorder scores of the entire protein.</p>
<p>-Figure 2) The pattern of phosphorylation sites in Type 1 and Type 2 different. Phosphorylation can be a key event in stabilizing proteins, and based on the data on Type 2 have higher propensity of phosphorylation in c-terminal and with a small incremental effect. This does not alone support the conclusion that c -terminal of MADS protein have higher phosphorylation site, there should be more analysis done. Comparing non-MADS IDPs can be useful.</p>
<p>R: We agree with the reviewer’s comment. Analysis of the phosphorylation propensity in a protein sequence does not assure that the predicted sites will be phosphorylated. However, there are several reports supporting that intrinsically disordered regions contain a larger proportion of potential phosphorylation sites than those regions with an ordered and stable structure (Iakoucheva et al. 2004 DOI: 10.1093/nar/gkh253; Gao and Xu, 2012 PMID: 22174266; Marijn and Ott, 2012 doi.org/10.3389/fpls.2012.00086; Koike et al., 2019 doi.org/10.1002/pro.3789; Usher et al., 2024 <a href="https://doi.org/10.1016/j.bpj.2024.10.021;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://doi.org/10.1016/j.bpj.2024.10.021;</a> Holehouse and Kragelund, 2024 doi.org/10.1038/s41580-023-00673-0). This finding makes sense given the higher exposition of potential phosphorylation sites in more flexible regions. Additionally, it has been found that the amino acid composition, sequence complexity, hydrophobicity, charge, and other sequence features surrounding confirmed phosphorylation sites closely resemble those characteristics of IDRs (Iakoucheva et al. 2004, DOI: 10.1093/nar/gkh253 and by Koike et al., 2019, DOI:10.1002/pro.3789).</p>
<p>The analysis suggested by the reviewer with a more extensive set of proteins has been done by Iakoucheva et al. (2004; DOI: 10.1093/nar/gkh253) using different proteomes. These authors have used complete proteomes, finding that proteins containing IDRs show a higher phosphorylation propensity than ordered proteins. We integrated this information into the manuscript (see Discusion, page 19, lines 523 – 542)</p>
<p>Because most of the proposed sites have not been experimentally validated, we have not stated that this is the case. Most of the validated phospho-sites are in the MADS-domain proteins' conserved motifs, and these experiments have been conducted under specific developmental stages and/or growth conditions, leaving aside alternative or additional phosphorylations under different experimental conditions. We think that the trend of these analyses is because, in general, the MADS’ IDRs have not received enough attention.</p>
<p>This is more evident when we look at the number of validated phosphorylation sites in MADS-domain Type II TFs, the best-characterized of this family. Type II MADS TFs show a similar number of phospho-sites in both the N- and C-terminal regions. This finding contrasts with the data on MADS-domain Type I TFs, where the validated phospho-sites are more abundant in the N-terminal region (see new Supplementary Table S5). Therefore, we think that the data in our manuscript provides an invitation to perform more methodical studies on MADS IDRs to find out the functional relevance of these regions, which are highly represented in this protein family.</p>
<p>Additionally, we have added a new paragraph in the phosphorylation results (Page 14, lines 343-351) that compares the experimentally validated phosphorylation patterns between Type I and Type II MADS-domain proteins (see new Supplementary Table S5).</p>
<p>We have carefully revised the manuscript, ensuring that the analysis we present in this manuscript refers to propensities, based on validated bioinformatic tools.</p>
<p>Figure 5) PADI score is dependent on sequence composition, supporting this analysis with different PADI threshold and performance matrix with sensitivity, specificity, ppv, npv with same data can increase the confidence on this analysis.</p>
<p>R: As mentioned in the manuscript, the data on plant activation domain identification (PADI) were directly obtained from Morffy et al. (2024; DOI: 10.1038/s41586-024-07707-3). In that study, the authors experimentally identified activation domains in various plant TFs using a comprehensive library. This library consisted of overlapping 40 amino acid fragments, spanning the entire set of plant TFs, with a step size of 10 amino acids, resulting in a total of 68,441 fragments. These fragments were screened in yeast to assess their transcriptional activation capacity. Based on this experiment and subsequent normalization, a PADI score was assigned to each fragment. Among the fragments showing transcriptional activation activity, some corresponded to MADS-domain TFs. The PADI scores included in our manuscript are those reported in this study. The localization of activation domains (AD) within MADS-domain TFs was inferred from the results obtained in Morffy et al. (2024), where the authors applied a neural network-based algorithm, known as transcriptional activation domain activity (TADA) network. Their work integrated multiple layers of analysis, including the construction of a feature matrix and the use of methods to assess the impact of both individual input features and border local and global interactions predicted by TADA. Additionally, these results were further analyzed using deep learning to identify key properties relevant to the prediction of ADs. A SHAP analysis was applied to capture non-linear and linear correlations, thereby uncovering complex patterns. This was followed by additional deep-learning steps that culminated in the development of a tool capable of predicting potential ADs. We have included part of this information in the Materials and Methods section. Page 8-9 lines 195-218.</p>
<p>As the reviewer may notice, modifying each of the many steps involved in calculating the propensity of specific sequences to function as ADs would be a highly complex task. We consider that such efforts are beyond the scope of the present manuscript.</p>
<p>- Figure 6) IDPs drive condensate formation and Type 2 have more MDP index and correlation in figure 6 how Type 1 have more LLPS forming propensity than Type 2. The authors have addressed the discrepancy in discussion. It would add more value if there is more similar analysis on known IDPs and LLP forming index.</p>
<p>R: Three different computational approaches were employed to evaluate the LLPS propensity index: FuzPred, PSPredict, and PSHunter. The table containing the values obtained for the full-length proteins and their corresponding C-terminal regions is included as part of the response to the reviewers (pLLPs software comparison in a reviewer's specific file). Values indicating a high propensity for liquid–liquid phase separation, according to the criteria specified by each prediction tool, are highlighted in yellow.</p>
<p>Among these, the analysis performed with FuzPred revealed a higher LLPS propensity in comparison to the other two predictors, although all three identified certain proteins with the potential to form biomolecular condensates.</p>
<p>The three tools are designed to predict the likelihood of a protein sequence undergoing liquid-liquid phase separation, particularly in the context of intrinsically disordered regions, but PSPredict and PSHunter exhibit some limitations. Specifically, these methods do not explicitly account for conformational dynamics or contextual interaction features, and they have reduced resolution in capturing the local contributions of specific sequence segments (Chu et al., 2022, doi.org/10.1186/s12859-022-04599-w; Sun et al., 2024, /doi.org/10.1038/s41467-024-46901-9).</p>
<p>In contrast, FuzPred incorporates the prediction of conformational fuzziness, referring to the propensity of a disordered region to retain dynamic and structurally adaptable behavior even upon molecular interaction. This approach explicitly considers both multivalency and structural flexibility as key drivers of LLPs, thereby allowing for the identification of functionally relevant interaction-prone regions that may not be detected by more generalized predictors. For this reason, FuzPred was selected as the primary tool for assessing LLPS propensity in our study (Hatos et al., 2023; doi: 10.1093/nar/gkad214).</p>
<p>Additionally, what if they only input C-terminal region, will that change the LLPS forming Index?</p>
<p>R: To address the reviewer’s question, which may also be shared by readers, we have included in the revised Figure 6 the data corresponding to the C-terminal regions of the MADS domain TFs. As shown in the updated figure, and as expected, the LLPS propensity indices for these regions are generally higher than those observed for the full-length proteins. This supports the notion that the C-terminal regions contribute significantly to LLPS propensity, and that the rest of the protein sequences may also influence these predictions. Consequently, the LLPS propensity of the full-length proteins does not always correlate with that of their C-terminal regions.</p>
<p>As noted in our answer to the previous comment, most LLPS predictors focus on the properties of IDRs, as these regions are key facilitators of protein-protein interactions, some of which drive LLPS through homotypic interactions among disordered segments. Moreover, IDRs can mediate interactions with other proteins, further promoting condensate formation (Xu et al., 2024; doi: 10.1073/pnas.2407633121).</p>
<p>However, the capacity to drive LLPS is not determined solely by the disordered nature of the sequence. Instead, it depends on specific sequence features, such as the distribution and patterning of aromatic and charged residues. These sequence-encoded patterns are essential for enabling the multivalent interactions required for the formation of biomolecular condensates (Borcherds et al., 2021; doi: 10.1016/j.sbi.2020.09.004).</p>
<p>We have added a comment on this point in the Discussion section of the manuscript (Page 21-22 lines 586-612).</p>
<p>The discussion is really long; it will help the manuscript if the authors try to keep it more concise in general.</p>
<p>R: As requested by the reviewer, we have made an effort to reduce the length of the Discussion section. However, this has proven challenging due to the simultaneous request to include additional information.</p>
<p>Reviewer #2: This manuscript presents a bioinformatic analysis of intrinsically disordered regions in MADS-domain transcription factors across diverse taxa. The authors demonstrate that C-terminal disorder is consistently present in both plant and non-plant MADS-domain proteins, with Type II proteins showing higher overall disorder than Type I proteins. They identify conserved motifs within these disordered regions that appear clade-specific, along with overlapping Molecular Recognition Features, particularly in Type II proteins. The study further characterizes potential activation domains and liquid-liquid phase separation propensities, finding more activation domains in Type I proteins. While the conservation of these structural features across evolutionary distant taxa strongly suggests functional importance in transcriptional regulation, I have some concerns about the exclusively computational nature of the analysis. The study would benefit from experimental validation of key predictions, a clearer distinction between correlation and causality in evolutionary conservation, more robust statistical analysis, and a more integrated model explaining how these various structural features collectively contribute to MADS-domain protein function.</p>
<p>Concern(s)/ Limitations and comments:</p>
<p>- My core concern is about the experimental validation. The study relies exclusively on bioinformatic predictions without any experimental validation of the identified IDRs, MoRFs, or activation domains. While computational approaches provide valuable insights, experimental confirmation is needed to significantly strengthen the findings. I would suggest authors include at least some experimental validation for key findings, particularly for the identified motifs and their potential functional roles. Techniques such as circular dichroism (CD) spectroscopy could confirm the disordered nature of the C-terminal regions, while yeast two-hybrid or co-immunoprecipitation assays could validate protein-protein interactions mediated by the identified MoRFs. Moreover, directed mutagenesis of conserved motifs followed by functional assays would help establish their biological significance, as demonstrated by Cho et al. (1999) for the C-terminal region of APETALA1.</p>
<p>R: We agree with the reviewer’s comment that experimental confirmation would add valuable support to our findings. However, such validation would require a substantial investment of time and resources, which will significantly delay the dissemination of the information presented in this manuscript to the scientific community interested in this important group of TFs. Moreover, this is particularly important because the present study provides, for the first time, evidence that the presence of IDRs in the C-terminal region of MADS-domain TFs is a conserved feature. Furthermore, our results suggest that this structural feature is not exclusive to</p>
<section class="sm xbox font-sm" id="pone.0330098.s014"><div class="caption p">
<span>Attachment</span><p>Submitted filename: <em>PLOS ONE answers to reviewers.docx</em></p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12373214/bin/pone.0330098.s014.docx" data-ga-action="click_feat_suppl" class="usa-link">pone.0330098.s014.docx</a><sup> (2.6MB, docx) </sup>
</div></div></section></section></article><article class="sub-article" id="pone.0330098.r004"><section class="pmc-layout__citation font-secondary font-xs"><div>PLoS One. doi: <a href="https://doi.org/10.1371/journal.pone.0330098.r004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1371/journal.pone.0330098.r004</a>
</div></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Decision Letter 1</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Prakash%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Ajit Prakash</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Ajit Prakash</span></h3>
<div>Academic Editor</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Prakash%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Ajit Prakash</span></a>
</div>
</div>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a.f" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="clp_a.f" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a.f" class="d-panel p" style="display: none">
<h4 class="font-secondary">Roles</h4>
<div class="p">
<strong class="contrib"><span class="name western">Ajit Prakash</span></strong>: <span class="role">Academic Editor</span>
</div>
</div>
<div id="clp_a.f" class="d-panel p" style="display: none">
<div>© 2025 Ajit Prakash</div>
<p>This is an open access article distributed under the terms of the <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Creative Commons Attribution License</a> , which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div></div>
</div></section><section class="body sub-article-body"><hr class="headless">
<p>
<em>27 Jul 2025</em>
</p>
<p>Structural disorder and distinctive motifs in the C-terminal region of the MADS-domain transcription factors are conserved across diverse taxa</p>
<p>PONE-D-25-10320R1</p>
<p>Dear Dr. Garay Arroyo,</p>
<p>We’re pleased to inform you that your manuscript has been judged scientifically suitable for publication and will be formally accepted for publication once it meets all outstanding technical requirements.</p>
<p>Within one week, you’ll receive an e-mail detailing the required amendments. When these have been addressed, you’ll receive a formal acceptance letter and your manuscript will be scheduled for publication.</p>
<p>An invoice will be generated when your article is formally accepted. Please note, if your institution has a publishing partnership with PLOS and your article meets the relevant criteria, all or part of your publication costs will be covered. Please make sure your user information is up-to-date by logging into Editorial Manager at <a href="https://www.editorialmanager.com/pone/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Editorial Manager®</a>  and clicking the ‘Update My Information' link at the top of the page. For questions related to billing, please contact <a href="https://plos.my.site.com/s/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">billing support</a> .</p>
<p>If your institution or institutions have a press office, please notify them about your upcoming paper to help maximize its impact. If they’ll be preparing press materials, please inform our press team as soon as possible -- no later than 48 hours after receiving the formal acceptance. Your manuscript will remain under strict press embargo until 2 pm Eastern Time on the date of publication. For more information, please contact onepress@plos.org.</p>
<p>Kind regards,</p>
<p>Ajit Prakash, PhD</p>
<p>Academic Editor</p>
<p>PLOS ONE</p>
<p>Additional Editor Comments (optional):</p>
<p>Reviewers' comments:</p>
<p>Reviewer's Responses to Questions</p>
<p>
<strong>Comments to the Author</strong>
</p>
<p>Reviewer #1: All comments have been addressed</p>
<p>Reviewer #2: All comments have been addressed</p>
<p>**********</p>
<p>2. Is the manuscript technically sound, and do the data support the conclusions??&gt;</p>
<p>Reviewer #1: Yes</p>
<p>Reviewer #2: Yes</p>
<p>**********</p>
<p>3. Has the statistical analysis been performed appropriately and rigorously? --&gt;?&gt;</p>
<p>Reviewer #1: I Don't Know</p>
<p>Reviewer #2: Yes</p>
<p>**********</p>
<p>4. Have the authors made all data underlying the findings in their manuscript fully available??&gt;</p>
<p>The <a href="http://www.plosone.org/static/policies.action#sharing" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PLOS Data policy</a></p>
<p>Reviewer #1: Yes</p>
<p>Reviewer #2: Yes</p>
<p>**********</p>
<p>5. Is the manuscript presented in an intelligible fashion and written in standard English??&gt;</p>
<p>Reviewer #1: Yes</p>
<p>Reviewer #2: Yes</p>
<p>**********</p>
<p>Reviewer #1: The authors have addressed all the comments and the authors have also provided evidence from other studies to support the questions they could have not addressed.</p>
<p>Reviewer #2: (No Response)</p>
<p>**********</p>
<p><a href="https://journals.plos.org/plosone/s/editorial-and-peer-review-process#loc-peer-review-history" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">what does this mean?</a> ). If published, this will include your full peer review and any attached files.</p>
<p>If you choose “no”, your identity will remain anonymous but your review may still be made public.</p>
<p><strong>Do you want your identity to be public for this peer review?</strong> For information about this choice, including consent withdrawal, please see our <a href="https://www.plos.org/privacy-policy" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Privacy Policy</a></p>
<p>Reviewer #1: No</p>
<p>Reviewer #2: <strong>Yes: </strong> Mohammad Ashhar Iqbal Khan</p>
<p>**********</p></section></article><article class="sub-article" id="pone.0330098.r005"><section class="pmc-layout__citation font-secondary font-xs"><div>PLoS One. doi: <a href="https://doi.org/10.1371/journal.pone.0330098.r005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1371/journal.pone.0330098.r005</a>
</div></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Acceptance letter</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Prakash%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Ajit Prakash</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Ajit Prakash</span></h3>
<div>Academic Editor</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Prakash%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Ajit Prakash</span></a>
</div>
</div>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a.g" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="clp_a.g" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a.g" class="d-panel p" style="display: none">
<h4 class="font-secondary">Roles</h4>
<div class="p">
<strong class="contrib"><span class="name western">Ajit Prakash</span></strong>: <span class="role">Academic Editor</span>
</div>
</div>
<div id="clp_a.g" class="d-panel p" style="display: none">
<div>© 2025 Ajit Prakash</div>
<p>This is an open access article distributed under the terms of the <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Creative Commons Attribution License</a> , which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div></div>
</div></section><section class="body sub-article-body"><hr class="headless">
<p>PONE-D-25-10320R1</p>
<p>PLOS ONE</p>
<p>Dear Dr. Garay-Arroyo,</p>
<p>I'm pleased to inform you that your manuscript has been deemed suitable for publication in PLOS ONE. Congratulations! Your manuscript is now being handed over to our production team.</p>
<p>At this stage, our production department will prepare your paper for publication. This includes ensuring the following:</p>
<p>* All references, tables, and figures are properly cited</p>
<p>* All relevant supporting information is included in the manuscript submission,</p>
<p>* There are no issues that prevent the paper from being properly typeset</p>
<p>You will receive further instructions from the production team, including instructions on how to review your proof when it is ready. Please keep in mind that we are working through a large volume of accepted articles, so please give us a few days to review your paper and let you know the next and final steps.</p>
<p>Lastly, if your institution or institutions have a press office, please let them know about your upcoming paper now to help maximize its impact. If they'll be preparing press materials, please inform our press team within the next 48 hours. Your manuscript will remain under strict press embargo until 2 pm Eastern Time on the date of publication. For more information, please contact onepress@plos.org.</p>
<p>You will receive an invoice from PLOS for your publication fee after your manuscript has reached the completed accept phase. If you receive an email requesting payment before acceptance or for any other service, this may be a phishing scheme. Learn how to identify phishing emails and protect your accounts at <a href="https://explore.plos.org/phishing" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://explore.plos.org/phishing</a>.</p>
<p>If we can help with anything else, please email us at customercare@plos.org.</p>
<p>Thank you for submitting your work to PLOS ONE and supporting open access.</p>
<p>Kind regards,</p>
<p>PLOS ONE Editorial Office Staff</p>
<p>on behalf of</p>
<p>Dr. Ajit Prakash</p>
<p>Academic Editor</p>
<p>PLOS ONE</p></section></article><article class="sub-article" id="_ad93_"><section class="pmc-layout__citation font-secondary font-xs"><div></div></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Associated Data</h1></hgroup><ul class="d-buttons inline-list"></ul>
<div class="d-panels font-secondary-light"></div>
<div></div>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
</div></section><section class="body sub-article-body"><section id="_adsm93_" lang="en" class="supplementary-materials"><h2 class="pmc_sec_title">Supplementary Materials</h2>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="caption p">
<span>S1 Table. MADS-domain proteins of Arabidopsis, other plant species, and non-plant species used in this study.</span><p>(DOCX)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12373214/bin/pone.0330098.s001.docx" data-ga-action="click_feat_suppl" class="usa-link">pone.0330098.s001.docx</a><sup> (36KB, docx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="caption p">
<span>S2 Table. MADS-domain proteins analyzed by Morffy et al. (2024) [<a href="#pone.0330098.ref064" class="usa-link" aria-describedby="pone.0330098.ref064">64</a>].</span><p>AD = activation domain. AD, Maybe and Not AD definitions are given based on the PADI (plant activation domain identification). PADI score ≥1 and mean disorder &gt;0.5 are defined as “AD”; PADI score ≥1 and mean disorder de ≤ 0.5 are defined as “Maybe”; PADI &lt;1 are defined as “Not AD”.</p>
<p>(DOCX)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12373214/bin/pone.0330098.s002.docx" data-ga-action="click_feat_suppl" class="usa-link">pone.0330098.s002.docx</a><sup> (25.3KB, docx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="caption p">
<span>S3 Table. R packages used in this research.</span><p>These packages are available from CRAN (<a href="https://CRAN.R-project.org/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://CRAN.R-project.org/</a>) or Bioconductor (Huber et al., 2015).</p>
<p>(DOCX)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12373214/bin/pone.0330098.s003.docx" data-ga-action="click_feat_suppl" class="usa-link">pone.0330098.s003.docx</a><sup> (18.8KB, docx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material4_reqid_"><div class="caption p">
<span>S4 Table. Arabidopsis MADS-domain proteins, disordered regions and phosphorylation sites.</span><p>(XLSX)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12373214/bin/pone.0330098.s004.xlsx" data-ga-action="click_feat_suppl" class="usa-link">pone.0330098.s004.xlsx</a><sup> (24.3KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material5_reqid_"><div class="caption p">
<span>S5 Table. Experimentally tested phosphorylation sites obtained from two different databases Athena (<a href="https://athena.proteomics.wzw.tum.de/master_arabidopsisshiny/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://athena.proteomics.wzw.tum.de/master_arabidopsisshiny/</a>), and EPSD (<a href="https://epsd.biocuckoo.cn/Browse.php" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://epsd.biocuckoo.cn/Browse.php</a>).</span><p>Sites were mapped on domains, according to UniProt and Liu et al., 2021 definition. See main text for a detailed description.</p>
<p>(DOCX)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12373214/bin/pone.0330098.s005.docx" data-ga-action="click_feat_suppl" class="usa-link">pone.0330098.s005.docx</a><sup> (139.7KB, docx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material6_reqid_"><div class="caption p">
<span>S6 Table. Experimental evidence supporting the functional relevance of the C-terminal region in MADS-domain proteins across different species.</span><p>(DOCX)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12373214/bin/pone.0330098.s006.docx" data-ga-action="click_feat_suppl" class="usa-link">pone.0330098.s006.docx</a><sup> (140.1KB, docx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material7_reqid_"><div class="caption p">
<span>S1 Fig. Physicochemical properties of Type I and Type II MADS-domain proteins.</span><p><strong>(A)</strong> Net charge per residue (NCPR), <strong>(B)</strong> Charge distribution (Kappa), and <strong>(C)</strong> Fraction of Charged Residues (FCR). Whiskers indicate ±1.5*IQR based on Tukey test. The middle line represents the median.</p>
<p>(TIF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12373214/bin/pone.0330098.s007.tif" data-ga-action="click_feat_suppl" class="usa-link">pone.0330098.s007.tif</a><sup> (75.5KB, tif) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material8_reqid_"><div class="caption p">
<span>S2 Fig. A complete Maximum Likelihood (ML) phylogeny of Arabidopsis MADS-domain proteins including MADS-domain proteins of other plant species and non-plant species.</span><p><em>Oryza sativa japonica</em> (<em>Os</em>)<em>, Solanum dulcamara</em> (<em>SD</em>)<em>, Mangifera indica</em> (<em>MI</em>)<em>, Populus trichocarpa</em> (<em>PT</em>)<em>, Amborella trichopoda</em> (<em>AMB</em>)<em>, Pinus radiata</em> (<em>Prad</em>)<em>, Selaginella mollendorffii</em> (<em>SM</em>)<em>, Chara braunii</em> (<em>CB</em>)<em>, Chlorella dessiccata</em> (<em>CD</em>)<em>, Saccharomyces cerevisiae</em> (<em>SC</em>)<em>, Drosophila melanogaster</em> (<em>DM</em>)<em>, and Homo sapiens</em> (<em>HS</em>). Yellow-shaded branches cover Type I MADS grouped with SRF-like and MEF-like MADS-domain proteins. Purple-shaded branches cover most Type II MADS-domain proteins. Numbers adjacent to nodes represent bootstrap support. Orange dots at particular nodes indicate the putative ancestral motif for that specific clade.</p>
<p>(TIF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12373214/bin/pone.0330098.s008.tif" data-ga-action="click_feat_suppl" class="usa-link">pone.0330098.s008.tif</a><sup> (3.2MB, tif) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material9_reqid_"><div class="caption p">
<span>S3 Fig. Structural disorder in the C-terminal region and the full-length proteins of MADS-domain transcription factors from different organisms.</span><p>Boxplots showing the ADS and PPIDR values for the C-terminal region and the full-length proteins from: <strong>(A)</strong> Plant species analysed in this study, excluding Arabidopsis. <strong>(B)</strong>
<em>Saccharomyces cerevisiae</em>, <strong>(C)</strong>
<em>Homo sapiens</em>, and <strong>(D)</strong>
<em>Drosophila melanogaster</em>. Whiskers indicate ±1.5*IQR according to the Tukey test. The middle line represents the median.</p>
<p>(TIF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12373214/bin/pone.0330098.s009.tif" data-ga-action="click_feat_suppl" class="usa-link">pone.0330098.s009.tif</a><sup> (372.2KB, tif) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material10_reqid_"><div class="caption p">
<span>S4 Fig. Distinctive motifs in MADS-domain proteins of SOC1 (A) and FLC (B) clades.</span><p>The phylogenetic tree was derived from the complete MADS-domain protein tree shown in Supplementary Fig 2. To enhance the visualization of phylogenetic relationships among the proteins, branch lengths were rescaled and truncated. The conserved amino acid residues of the SOC1-motif and FLC-motif are highlighted in bold within the consensus motif.</p>
<p>(TIF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12373214/bin/pone.0330098.s010.tif" data-ga-action="click_feat_suppl" class="usa-link">pone.0330098.s010.tif</a><sup> (217.5KB, tif) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material11_reqid_"><div class="caption p">
<span>S5 Fig. Identified MoRFs in MADS-domain proteins.</span><p>MADS-domain protein sequences from Arabidopsis, other plants, and non-plant organisms are shown, with predicted MoRFs highlighted in yellow Putative MoRFs were identified using the fmoRFpred algorithm [<a href="#pone.0330098.ref123" class="usa-link" aria-describedby="pone.0330098.ref123">123</a>], based on the analysis of full-length protein sequences.</p>
<p>(TIF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12373214/bin/pone.0330098.s011.tif" data-ga-action="click_feat_suppl" class="usa-link">pone.0330098.s011.tif</a><sup> (1.1MB, tif) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material12_reqid_"><div class="caption p">
<span>Attachment</span><p>Submitted filename: <em>Comments.pdf</em></p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12373214/bin/pone.0330098.s012.pdf" data-ga-action="click_feat_suppl" class="usa-link">pone.0330098.s012.pdf</a><sup> (189.6KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material13_reqid_"><div class="caption p">
<span>Attachment</span><p>Submitted filename: <em>PLOS ONE answers to reviewers.docx</em></p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12373214/bin/pone.0330098.s014.docx" data-ga-action="click_feat_suppl" class="usa-link">pone.0330098.s014.docx</a><sup> (2.6MB, docx) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h2 class="pmc_sec_title">Data Availability Statement</h2>
<p>All relevant data are within the paper and its <a href="#sec020" class="usa-link">Supporting information</a> files.</p></section></section></article><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from PLOS One are provided here courtesy of <strong>PLOS</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1371/journal.pone.0330098"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/pone.0330098.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (2.1 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12373214/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12373214/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12373214%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373214/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12373214/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12373214/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40844982/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12373214/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40844982/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12373214/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12373214/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="VkxmmW4UXDtAgpwDrZyVOL9I6DvabrSM0skxAu3XPacztLhOdGa2eVXXmKhwJHTb">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
