
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            A chickpea MAGIC population to dissect the genetics of complex traits - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="D2E4852D8AEF24E305852D002BFD5A01.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="blackwellopen">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12375853/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="The Plant Genome">
<meta name="citation_title" content="A chickpea MAGIC population to dissect the genetics of complex traits">
<meta name="citation_author" content="Oluwaseun J Akinlade">
<meta name="citation_author_institution" content="Queensland Alliance for Agriculture and Food Innovation, The University of Queensland, Brisbane, Queensland, Australia">
<meta name="citation_author_institution" content="State Agricultural Biotechnology Centre, Centre for Crop and Food Innovation, Food Futures Institute, Murdoch University, Murdoch, Western Australia, Australia">
<meta name="citation_author" content="Hannah Robinson">
<meta name="citation_author_institution" content="Queensland Alliance for Agriculture and Food Innovation, The University of Queensland, Brisbane, Queensland, Australia">
<meta name="citation_author" content="Yichen Kang">
<meta name="citation_author_institution" content="Queensland Alliance for Agriculture and Food Innovation, The University of Queensland, Brisbane, Queensland, Australia">
<meta name="citation_author" content="Mahendar Thudi">
<meta name="citation_author_institution" content="College of Agriculture, Family Sciences and Technology, Fort Valley State University, Fort Valley, Georgia, USA">
<meta name="citation_author_institution" content="International Crops Research Institute for the Semi‐Arid Tropics (ICRISAT), Patancheru, Telangana, India">
<meta name="citation_author" content="Srinivasan Samineni">
<meta name="citation_author_institution" content="International Crops Research Institute for the Semi‐Arid Tropics (ICRISAT), Patancheru, Telangana, India">
<meta name="citation_author_institution" content="Crop Diversification and Genetics, International Center for Biosaline Agriculture (ICBA), Dubai, UAE">
<meta name="citation_author" content="Pooran Gaur">
<meta name="citation_author_institution" content="International Crops Research Institute for the Semi‐Arid Tropics (ICRISAT), Patancheru, Telangana, India">
<meta name="citation_author" content="Millicent R Smith">
<meta name="citation_author_institution" content="School of Agriculture and Food Sustainability, Faculty of Science, The University of Queensland, Gatton, Queensland, Australia">
<meta name="citation_author" content="Kai P Voss‐Fels">
<meta name="citation_author_institution" content="Department of Grapevine Breeding, Hochschule Geisenheim University, Geisenheim, Germany">
<meta name="citation_author" content="Roy Costilla">
<meta name="citation_author_institution" content="Cawthron Institute, Nelson, New Zealand">
<meta name="citation_author" content="Rajeev K Varshney">
<meta name="citation_author_institution" content="State Agricultural Biotechnology Centre, Centre for Crop and Food Innovation, Food Futures Institute, Murdoch University, Murdoch, Western Australia, Australia">
<meta name="citation_author" content="Eric Dinglasan">
<meta name="citation_author_institution" content="Queensland Alliance for Agriculture and Food Innovation, The University of Queensland, Brisbane, Queensland, Australia">
<meta name="citation_author" content="Lee T Hickey">
<meta name="citation_author_institution" content="Queensland Alliance for Agriculture and Food Innovation, The University of Queensland, Brisbane, Queensland, Australia">
<meta name="citation_publication_date" content="2025 Aug 24">
<meta name="citation_volume" content="18">
<meta name="citation_issue" content="3">
<meta name="citation_firstpage" content="e70096">
<meta name="citation_doi" content="10.1002/tpg2.70096">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12375853/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12375853/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12375853/pdf/TPG2-18-e70096.pdf">
<meta name="description" content="Multiparent populations are now widespread in crop genetic studies as they capture more genetic diversity and offer high statistical power for detecting quantitative trait loci (QTLs). To confirm the suitability of using a recently developed ...">
<meta name="og:title" content="A chickpea MAGIC population to dissect the genetics of complex traits">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Multiparent populations are now widespread in crop genetic studies as they capture more genetic diversity and offer high statistical power for detecting quantitative trait loci (QTLs). To confirm the suitability of using a recently developed ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12375853/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12375853">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1002/tpg2.70096"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/TPG2-18-e70096.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12375853%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12375853/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12375853/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12375853/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-blackwellopen.png" alt="Wiley Open Access Collection logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Wiley Open Access Collection" title="Link to Wiley Open Access Collection" shape="default" href="https://doi.org/10.1002/tpg2.70096" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Plant Genome</button></div>. 2025 Aug 24;18(3):e70096. doi: <a href="https://doi.org/10.1002/tpg2.70096" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1002/tpg2.70096</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Plant%20Genome%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Plant%20Genome%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Plant%20Genome%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Plant%20Genome%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>A chickpea MAGIC population to dissect the genetics of complex traits</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Akinlade%20OJ%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Oluwaseun J Akinlade</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Oluwaseun J Akinlade</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Queensland Alliance for Agriculture and Food Innovation, The University of Queensland, Brisbane, Queensland, Australia</div>
<div class="p">
<sup>
<sup>8</sup>
</sup>State Agricultural Biotechnology Centre, Centre for Crop and Food Innovation, Food Futures Institute, Murdoch University, Murdoch, Western Australia, Australia</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Akinlade%20OJ%22%5BAuthor%5D" class="usa-link"><span class="name western">Oluwaseun J Akinlade</span></a>
</div>
</div>
<sup>1,</sup><sup>8</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Robinson%20H%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Hannah Robinson</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Hannah Robinson</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Queensland Alliance for Agriculture and Food Innovation, The University of Queensland, Brisbane, Queensland, Australia</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Robinson%20H%22%5BAuthor%5D" class="usa-link"><span class="name western">Hannah Robinson</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kang%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Yichen Kang</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Yichen Kang</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Queensland Alliance for Agriculture and Food Innovation, The University of Queensland, Brisbane, Queensland, Australia</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kang%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yichen Kang</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Thudi%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Mahendar Thudi</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Mahendar Thudi</span></h3>
<div class="p">
<sup>
<sup>2</sup>
</sup>College of Agriculture, Family Sciences and Technology, Fort Valley State University, Fort Valley, Georgia, USA</div>
<div class="p">
<sup>
<sup>3</sup>
</sup>International Crops Research Institute for the Semi‐Arid Tropics (ICRISAT), Patancheru, Telangana, India</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Thudi%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Mahendar Thudi</span></a>
</div>
</div>
<sup>2,</sup><sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Samineni%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Srinivasan Samineni</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Srinivasan Samineni</span></h3>
<div class="p">
<sup>
<sup>3</sup>
</sup>International Crops Research Institute for the Semi‐Arid Tropics (ICRISAT), Patancheru, Telangana, India</div>
<div class="p">
<sup>
<sup>4</sup>
</sup>Crop Diversification and Genetics, International Center for Biosaline Agriculture (ICBA), Dubai, UAE</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Samineni%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Srinivasan Samineni</span></a>
</div>
</div>
<sup>3,</sup><sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gaur%20P%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Pooran Gaur</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Pooran Gaur</span></h3>
<div class="p">
<sup>
<sup>3</sup>
</sup>International Crops Research Institute for the Semi‐Arid Tropics (ICRISAT), Patancheru, Telangana, India</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gaur%20P%22%5BAuthor%5D" class="usa-link"><span class="name western">Pooran Gaur</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Smith%20MR%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Millicent R Smith</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Millicent R Smith</span></h3>
<div class="p">
<sup>
<sup>5</sup>
</sup>School of Agriculture and Food Sustainability, Faculty of Science, The University of Queensland, Gatton, Queensland, Australia</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Smith%20MR%22%5BAuthor%5D" class="usa-link"><span class="name western">Millicent R Smith</span></a>
</div>
</div>
<sup>5</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Voss%E2%80%90Fels%20KP%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Kai P Voss‐Fels</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Kai P Voss‐Fels</span></h3>
<div class="p">
<sup>
<sup>6</sup>
</sup>Department of Grapevine Breeding, Hochschule Geisenheim University, Geisenheim, Germany</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Voss%E2%80%90Fels%20KP%22%5BAuthor%5D" class="usa-link"><span class="name western">Kai P Voss‐Fels</span></a>
</div>
</div>
<sup>6</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Costilla%20R%22%5BAuthor%5D" class="usa-link" aria-describedby="id9"><span class="name western">Roy Costilla</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">Roy Costilla</span></h3>
<div class="p">
<sup>
<sup>7</sup>
</sup>Cawthron Institute, Nelson, New Zealand</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Costilla%20R%22%5BAuthor%5D" class="usa-link"><span class="name western">Roy Costilla</span></a>
</div>
</div>
<sup>7</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Varshney%20RK%22%5BAuthor%5D" class="usa-link" aria-describedby="id10"><span class="name western">Rajeev K Varshney</span></a><div hidden="hidden" id="id10">
<h3><span class="name western">Rajeev K Varshney</span></h3>
<div class="p">
<sup>
<sup>8</sup>
</sup>State Agricultural Biotechnology Centre, Centre for Crop and Food Innovation, Food Futures Institute, Murdoch University, Murdoch, Western Australia, Australia</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Varshney%20RK%22%5BAuthor%5D" class="usa-link"><span class="name western">Rajeev K Varshney</span></a>
</div>
</div>
<sup>8</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Dinglasan%20E%22%5BAuthor%5D" class="usa-link" aria-describedby="id11"><span class="name western">Eric Dinglasan</span></a><div hidden="hidden" id="id11">
<h3><span class="name western">Eric Dinglasan</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Queensland Alliance for Agriculture and Food Innovation, The University of Queensland, Brisbane, Queensland, Australia</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Dinglasan%20E%22%5BAuthor%5D" class="usa-link"><span class="name western">Eric Dinglasan</span></a>
</div>
</div>
<sup>1,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hickey%20LT%22%5BAuthor%5D" class="usa-link" aria-describedby="id12"><span class="name western">Lee T Hickey</span></a><div hidden="hidden" id="id12">
<h3><span class="name western">Lee T Hickey</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Queensland Alliance for Agriculture and Food Innovation, The University of Queensland, Brisbane, Queensland, Australia</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hickey%20LT%22%5BAuthor%5D" class="usa-link"><span class="name western">Lee T Hickey</span></a>
</div>
</div>
<sup>1,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="tpg270096-aff-0001">
<sup>
<sup>1</sup>
</sup>Queensland Alliance for Agriculture and Food Innovation, The University of Queensland, Brisbane, Queensland, Australia</div>
<div id="tpg270096-aff-0002">
<sup>
<sup>2</sup>
</sup>College of Agriculture, Family Sciences and Technology, Fort Valley State University, Fort Valley, Georgia, USA</div>
<div id="tpg270096-aff-0003">
<sup>
<sup>3</sup>
</sup>International Crops Research Institute for the Semi‐Arid Tropics (ICRISAT), Patancheru, Telangana, India</div>
<div id="tpg270096-aff-0004">
<sup>
<sup>4</sup>
</sup>Crop Diversification and Genetics, International Center for Biosaline Agriculture (ICBA), Dubai, UAE</div>
<div id="tpg270096-aff-0005">
<sup>
<sup>5</sup>
</sup>School of Agriculture and Food Sustainability, Faculty of Science, The University of Queensland, Gatton, Queensland, Australia</div>
<div id="tpg270096-aff-0006">
<sup>
<sup>6</sup>
</sup>Department of Grapevine Breeding, Hochschule Geisenheim University, Geisenheim, Germany</div>
<div id="tpg270096-aff-0007">
<sup>
<sup>7</sup>
</sup>Cawthron Institute, Nelson, New Zealand</div>
<div id="tpg270096-aff-0008">
<sup>
<sup>8</sup>
</sup>State Agricultural Biotechnology Centre, Centre for Crop and Food Innovation, Food Futures Institute, Murdoch University, Murdoch, Western Australia, Australia</div>
<div class="author-notes p">
<div class="fn" id="correspondenceTo">
<sup>*</sup><p class="display-inline">

<strong>Correspondence</strong>
, Eric Dinglasan and Lee T. Hickey, The University of Queensland, Queensland Alliance for Agriculture and Food Innovation, Brisbane, Queensland, Australia. Email: <span>e.dinglasan@uq.edu.au</span> and Email: <span>l.hickey@uq.edu.au</span>

</p>
</div>
<div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Revised 2025 Jul 3; Received 2025 Jan 8; Accepted 2025 Jul 5; Issue date 2025 Sep.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© 2025 The Author(s). <em>The Plant Genome</em> published by Wiley Periodicals LLC on behalf of Crop Science Society of America.</div>
<p>This is an open access article under the terms of the <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by/4.0/</a> License, which permits use, distribution and reproduction in any medium, provided the original work is properly cited.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12375853</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>Abstract</h2>
<p>Multiparent populations are now widespread in crop genetic studies as they capture more genetic diversity and offer high statistical power for detecting quantitative trait loci (QTLs). To confirm the suitability of using a recently developed chickpea (<em>Cicer arietinum</em> L.) multi‐parent advanced generation intercross (MAGIC) population for genetic studies, we characterized the diversity of the eight founder lines and explored the linkage disequilibrium decay, marker coverage, segregation distortion, allelic variation, and structure of the population. The MAGIC population was genotyped using whole‐genome sequencing; following marker curation, a total of 4255 high‐quality polymorphic single nucleotide polymorphism markers were used for genomic analyses. To demonstrate the effectiveness of the MAGIC population to dissect the genetics of key agronomic traits (days to 50% flowering and plant height), we employed both a genome‐wide mapping approach using fixed and random model circulating probability unification and a haplotype‐based mapping using the local genomic estimated breeding value approach. Our analyses confirmed the role of genomic regions previously reported in the literature and identified several new QTLs for days to 50% flowering and plant height. We also showed the potential for trait improvement through stacking the top 10 haploblocks to develop early flowering chickpea and selection of desirable haplotypes on chromosome 4 to improve plant height. Our results demonstrate the chickpea MAGIC population is a valuable resource for researchers and pre‐breeders to study the genetic architecture of complex traits and allelic variation to accelerate crop improvement in chickpea.</p></section><section class="abstract" id="abstract2"><h2>Core Ideas</h2>
<ul id="tpg270096-list-10001" class="list" style="list-style-type:disc">
<li><p>Eight‐parent chickpea MAGIC population (1135 lines) offers intermediate level of diversity for genetic studies and breeding.</p></li>
<li><p>Genome‐wide association study identified seven novel quantitative trait loci (QTLs) for flowering time and plant height with a strong enrichment of stress‐responsive, developmental, and regulatory genes.</p></li>
<li><p>Haplotype mapping revealed key blocks, with stacking strategies for optimizing flowering and plant height.</p></li>
<li><p>Rapid linkage disequilibrium decay (∼1 Mbp) in the multi‐parent advanced generation intercross (MAGIC) population supports mapping resolution suitable for MAGIC populations for mapping complex quantitative traits.</p></li>
<li><p>Co‐located QTLs for flowering and height suggest pleiotropy, enabling simultaneous trait improvement.</p></li>
</ul></section><section class="abstract" id="abstract3"><h2>Plain Language Summary</h2>
<p>Chickpea is a vital global crop, providing nutrition and supporting sustainable agriculture. To improve its resilience and productivity, understanding the genetic factors influencing complex traits is essential. We used a novel multi‐parent advanced generation intercross (MAGIC) population to study chickpea genetics. Our research identified several regions in the chickpea genome influencing flowering time and plant height, including seven previously unknown regions. By analyzing groups of markers (haplotypes), we demonstrated how combining optimal genetic variants can produce chickpeas that flower earlier or grow taller, depending on breeding goals. These findings provide new tools for breeders to develop chickpea varieties capable of withstanding environmental challenges and increasing yields. Furthermore, the MAGIC population is a valuable resource for exploring other complex traits, contributing to efforts to ensure global food security.</p></section><hr class="headless">
<h3>Abbreviations</h3>
<dl class="def-list" id="tpg270096-dl-0001">
<dt>DTF</dt>
<dd><p>days to 50% flowering; GEBV, genomic estimated breeding values;</p></dd>
<dt>GWAS</dt>
<dd><p>genome‐wide association study</p></dd>
<dt>LD</dt>
<dd><p>linkage disequilibrium</p></dd>
<dt>MAGIC</dt>
<dd><p>multi‐parent advanced generation intercross</p></dd>
<dt>PHT</dt>
<dd><p>plant height</p></dd>
<dt>QTL</dt>
<dd><p>quantitative trait locus</p></dd>
<dt>RIL</dt>
<dd><p>recombinant inbred line</p></dd>
<dt>SNP</dt>
<dd><p>single nucleotide polymorphism</p></dd>
<dt>SSD</dt>
<dd><p>single seed descent</p></dd>
</dl>
<section id="tpg270096-sec-0010"><h2 class="pmc_sec_title">1. INTRODUCTION</h2>
<p>Chickpea (<em>Cicer arietinum</em> L.) is an important legume that is rich in nutrients and is a source of proteins, amino acids, and essential vitamins (Bar‐El Dadon et al., <a href="#tpg270096-bib-0005" class="usa-link" aria-describedby="tpg270096-bib-0005">2017</a>; Paul et al., <a href="#tpg270096-bib-0033" class="usa-link" aria-describedby="tpg270096-bib-0033">2018</a>). It is the second most consumed and third most‐produced legume globally (Upadhyaya et al., <a href="#tpg270096-bib-0053" class="usa-link" aria-describedby="tpg270096-bib-0053">2016</a>; Varshney et al., <a href="#tpg270096-bib-0050" class="usa-link" aria-describedby="tpg270096-bib-0050">2013</a>). Chickpea is a self‐pollinating, diploid (2<em>n</em> = 2<em>x</em> = 16) (van der Maesen, <a href="#tpg270096-bib-0054" class="usa-link" aria-describedby="tpg270096-bib-0054">1987</a>) with a genome size of ∼738 Mb. Chickpea has two ecotypes: Desi, which has a small seed size with a thick seed coat, compared to Kabuli, which has a larger seed size with a smooth seed coat. Despite the growing importance of chickpea globally for food systems, current and future production is threatened by climate change (Akinlade et al., <a href="#tpg270096-bib-0002" class="usa-link" aria-describedby="tpg270096-bib-0002">2022</a>). Consequently, there is an urgent need to improve the understanding of the genetic architecture of adaptive traits that can be utilized to develop more resilient, productive chickpea varieties.</p>
<p>Genetic gain in crop improvement programs relies heavily on genetic diversity (H. Li, Rasheed et al., <a href="#tpg270096-bib-0028" class="usa-link" aria-describedby="tpg270096-bib-0028">2018</a>; Y. Li, Ruperao et al., <a href="#tpg270096-bib-0030" class="usa-link" aria-describedby="tpg270096-bib-0030">2018</a>). The availability of reference genome sequences for chickpea has enhanced our understanding of the genetic architecture of key traits (Varshney et al., <a href="#tpg270096-bib-0050" class="usa-link" aria-describedby="tpg270096-bib-0050">2013</a>, <a href="#tpg270096-bib-0051" class="usa-link" aria-describedby="tpg270096-bib-0051">2019</a>). Allelic variation for traits is important for crop improvement and has been reported in many studies (Jha et al., <a href="#tpg270096-bib-0022" class="usa-link" aria-describedby="tpg270096-bib-0022">2021</a>; Sannemann et al., <a href="#tpg270096-bib-0042" class="usa-link" aria-describedby="tpg270096-bib-0042">2018</a>). Identifying genes and mapping specific QTLs influencing phenotypes or traits of interest requires an experimental population with broader genetic variability for the target traits (Ladejobi et al., <a href="#tpg270096-bib-0025" class="usa-link" aria-describedby="tpg270096-bib-0025">2016</a>). Studies to date have focused primarily on bi‐parental mapping populations and diversity panels for QTL discovery (Kale et al., <a href="#tpg270096-bib-0023" class="usa-link" aria-describedby="tpg270096-bib-0023">2015</a>; Nguyen et al., <a href="#tpg270096-bib-0032" class="usa-link" aria-describedby="tpg270096-bib-0032">2022</a>; Paul et al., <a href="#tpg270096-bib-0033" class="usa-link" aria-describedby="tpg270096-bib-0033">2018</a>; Pushpavalli et al., <a href="#tpg270096-bib-0035" class="usa-link" aria-describedby="tpg270096-bib-0035">2015</a>; Sivasakthi et al., <a href="#tpg270096-bib-0044" class="usa-link" aria-describedby="tpg270096-bib-0044">2018</a>; Thudi et al., <a href="#tpg270096-bib-0047" class="usa-link" aria-describedby="tpg270096-bib-0047">2014</a>).</p>
<p>Multi‐parental mapping populations, such as multi‐parent advanced generation intercross (MAGIC) and nested association mapping, enable the evaluation of multiple alleles and greater recombination events to improve precision in QTL mapping (Ladejobi et al., <a href="#tpg270096-bib-0025" class="usa-link" aria-describedby="tpg270096-bib-0025">2016</a>; Scott et al., <a href="#tpg270096-bib-0043" class="usa-link" aria-describedby="tpg270096-bib-0043">2020</a>). The concept of the MAGIC population is a modification and extension of advanced intercross lines proposed by Darvasi and Soller (<a href="#tpg270096-bib-0055" class="usa-link" aria-describedby="tpg270096-bib-0055">1995</a>). In comparison to diversity panels, MAGIC populations can be advantageous as allelic variation within a set of founder lines can be reshuffled to improve mapping accuracy and balanced allele frequency can enhance statistical power (Ren et al., <a href="#tpg270096-bib-0038" class="usa-link" aria-describedby="tpg270096-bib-0038">2020</a>). In practice, the intercrossing of founder lines breaks down the population structure via abundant recombination events (Han et al., <a href="#tpg270096-bib-0013" class="usa-link" aria-describedby="tpg270096-bib-0013">2020</a>) and increases linkage disequilibrium (LD) decay (Rockman &amp; Kruglyak, <a href="#tpg270096-bib-0039" class="usa-link" aria-describedby="tpg270096-bib-0039">2008</a>).</p>
<p>MAGIC population development follows a straightforward design depending on the number of founder lines in the design (Stadlmeier et al., <a href="#tpg270096-bib-0045" class="usa-link" aria-describedby="tpg270096-bib-0045">2018</a>), and populations have previously been developed in many crops including wheat (Huang et al., <a href="#tpg270096-bib-0018" class="usa-link" aria-describedby="tpg270096-bib-0018">2012</a>), rice (Bandillo et al., <a href="#tpg270096-bib-0004" class="usa-link" aria-describedby="tpg270096-bib-0004">2013</a>), maize (Holland, <a href="#tpg270096-bib-0016" class="usa-link" aria-describedby="tpg270096-bib-0016">2015</a>), barley (Hemshrot et al., <a href="#tpg270096-bib-0015" class="usa-link" aria-describedby="tpg270096-bib-0015">2019</a>), cowpea (Ravelombola et al., <a href="#tpg270096-bib-0037" class="usa-link" aria-describedby="tpg270096-bib-0037">2021</a>), and soybean (Hashemi et al., <a href="#tpg270096-bib-0014" class="usa-link" aria-describedby="tpg270096-bib-0014">2022</a>) and successfully used to gain new insight into important traits.</p>
<p>In this study, we aimed to explore the genetic architecture of a recently developed chickpea MAGIC population that contains 1135 lines derived from eight founders (Samineni et al., <a href="#tpg270096-bib-0041" class="usa-link" aria-describedby="tpg270096-bib-0041">2021</a>; Thudi et al., <a href="#tpg270096-bib-0046" class="usa-link" aria-describedby="tpg270096-bib-0046">2024</a>). The objectives were (1) to investigate the genetic architecture of the chickpea MAGIC population including population structure, LD, and linkage decay, and (2) to evaluate the utility of the chickpea MAGIC population for complex trait dissection using a case study for two agronomic traits: days to 50% flowering (DTF) and plant height (PHT).</p></section><section id="tpg270096-sec-0020"><h2 class="pmc_sec_title">2. MATERIALS AND METHODS</h2>
<section id="tpg270096-sec-0030"><h3 class="pmc_sec_title">2.1. MAGIC population development</h3>
<p>The population was developed over a 4‐year period (2009–2013) by the International Crops Research Institute of the Semi‐Arid Tropics, in partnership with the Indian Agricultural Research Institute (Samineni et al., <a href="#tpg270096-bib-0041" class="usa-link" aria-describedby="tpg270096-bib-0041">2021</a>). Briefly, the eight founders were intercrossed using a stepwise approach. First, 28 two‐way crosses were performed, which were then used to generate 14 four‐way crosses, and finally used to generate seven eight‐way crosses. Following crossing, eight selfing generations were then performed using single seed descent (SSD) to generate inbred lines, and seed produced in the final generation was bulked to generate 1135 MAGIC lines examined in this study. A graphical representation of the crossing design and population development is presented in Figure <a href="#tpg270096-supinfo-0001" class="usa-link">S1</a>.</p>
<section class="bt xbox font-sm" id="boxed-text1"><section id="tpg270096-sec-0040"><h4 class="pmc_sec_title">Core Ideas</h4>
<ul id="tpg270096-list-0001" class="list" style="list-style-type:disc">
<li><p>Eight‐parent chickpea MAGIC population (1135 lines) offers intermediate level of diversity for genetic studies and breeding.</p></li>
<li><p>Genome‐wide association study identified seven novel quantitative trait loci (QTLs) for flowering time and plant height with a strong enrichment of stress‐responsive, developmental, and regulatory genes.</p></li>
<li><p>Haplotype mapping revealed key blocks, with stacking strategies for optimizing flowering and plant height.</p></li>
<li><p>Rapid linkage disequilibrium decay (∼1 Mbp) in the multi‐parent advanced generation intercross (MAGIC) population supports mapping resolution suitable for MAGIC populations for mapping complex quantitative traits.</p></li>
<li><p>Co‐located QTLs for flowering and height suggest pleiotropy, enabling simultaneous trait improvement.</p></li>
</ul></section></section></section><section id="tpg270096-sec-0050"><h3 class="pmc_sec_title">2.2. Genome sequencing and single nucleotide polymorphism calling</h3>
<p>This population was genotyped using whole‐genome sequencing method. Leaf tissue from 15‐day‐old seedlings was used for high‐throughput mini‐DNA extraction (Cuc et al., <a href="#tpg270096-bib-0056" class="usa-link" aria-describedby="tpg270096-bib-0056">2008</a>). As described by Varshney et al. (<a href="#tpg270096-bib-0057" class="usa-link" aria-describedby="tpg270096-bib-0057">2017</a>), 500 bp insert sizes were constructed for all samples for the whole genome sequencing of the MAGIC population (Thudi et al., <a href="#tpg270096-bib-0046" class="usa-link" aria-describedby="tpg270096-bib-0046">2024</a>). Illumina HiSeq2500 sequenced each library's paired‐end (PE) readings, and line data were trimmed and purified. Reads containing greater than 5% bases or polyA structures, as well as low‐quality reads, adapter contamination, and small insert size reads were filtered out. Additionally, read endpoint base quality was filtered out. Thereafter, the data were aligned on the Crop Development Centre Frontier genome (Varshney et al., <a href="#tpg270096-bib-0050" class="usa-link" aria-describedby="tpg270096-bib-0050">2013</a>) using BWA (Burrows‐Wheeler aligner; H. Li et al., <a href="#tpg270096-bib-0027" class="usa-link" aria-describedby="tpg270096-bib-0027">2009</a>). Post‐alignment processing was performed using Samtools, including sorting, marking duplicates, and indexing of BAM (binary alignment map) files (H. Li et al. <a href="#tpg270096-bib-0027" class="usa-link" aria-describedby="tpg270096-bib-0027">2009</a>) and variant discovery and genotype calling were conducted through the Genome Analysis ToolKit (GATK v2.). The variant calling pipeline is referred as the “GATK Best Practices” workflow (DePristo et al., <a href="#tpg270096-bib-0009" class="usa-link" aria-describedby="tpg270096-bib-0009">2011</a>). On average, the founders were sequenced to a depth of 48.74x, with individual depths ranging from 43.61x to 51.58x. For the MAGIC recombinant inbred lines (RILs), the average sequencing depth across the MAGIC lines was 6.97x, with a range from 4.45x to 14.56x.</p></section><section id="tpg270096-sec-0060"><h3 class="pmc_sec_title">2.3. Genotypic data quality control and imputation</h3>
<p>VCF (variant call format) files generated for each chromosome, containing genotypic information from the MAGIC lines and founders, were merged using BCFtools (Danecek et al., <a href="#tpg270096-bib-0008" class="usa-link" aria-describedby="tpg270096-bib-0008">2021</a>), resulting in a total of 494,325 single nucleotide polymorphisms (SNPs). The following quality control thresholds were used to remove SNP markers: minor allele frequency (MAF) ≤ 0.01, genomic read depth &lt; 5, missing rate per SNP &gt; 50%, and missing rate per sample &gt; 50%. Monomorphic SNPs were removed from the dataset resulting in 29,391 SNPs. We used EAGLE v2.4.1 (Loh et al., <a href="#tpg270096-bib-0031" class="usa-link" aria-describedby="tpg270096-bib-0031">2016</a>) for phasing and Minimac3 v2.0.1 (Howie et al., <a href="#tpg270096-bib-0017" class="usa-link" aria-describedby="tpg270096-bib-0017">2012</a>) to impute missing genotypes. The sequence data of the eight founders were used as the reference to impute the genotypes of the 1135 MAGIC lines (target sample). The accuracy of the imputation was calculated using Minimac's <em>R</em>
<sup>2</sup> statistic (<em>R</em>
<sup>2</sup>). Only SNPs with <em>R</em>
<sup>2 </sup>&gt; 0.95 were kept for downstream analysis. Finally, to reduce marker redundancy, highly correlated markers (&gt;0.99) were removed resulting in 4255 high‐quality SNP markers for the subsequent analysis.</p></section><section id="tpg270096-sec-0070"><h3 class="pmc_sec_title">2.4. Investigating diversity of founders</h3>
<p>The eight founders used for the development of this MAGIC population were a combination of released cultivars and elite breeding lines preferred across Africa and Asia (Table <a href="#tpg270096-supinfo-0001" class="usa-link">S1</a>).</p>
<p>To investigate the genetic diversity represented by the founders, the eight genotypes (founders) were co‐analyzed with a diversity panel comprising 91 desi genotypes collected across the world from 1970 to 2002. A common set of 2286 polymorphic SNPs across all 99 genotypes was used for analysis. First, Roger's genetic distance was calculated using the R package “SelectionTools” version 22.1 (<a href="http://population-genetics.uni-giessen.de/software/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://population‐genetics.uni‐giessen.de/software/</a>). Then, based on the distance matrix, <em>k</em>‐means clustering was performed, whereby the optimum <em>k</em> number was determined using the R package “NbClust” version 3 (Charrad et al., <a href="#tpg270096-bib-0058" class="usa-link" aria-describedby="tpg270096-bib-0058">2014</a>). We constructed a dendrogram of relatedness using complete clustering and heatmap to show where the eight founders sit within the diversity panel.</p></section><section id="tpg270096-sec-0080"><h3 class="pmc_sec_title">2.5. Population structure and genetic analysis among the founders and MAGIC lines</h3>
<p>To analyze the population structure, the genotypic data of the 1135 MAGIC lines and the eight founder lines were merged using PLINK (Purcell et al., <a href="#tpg270096-bib-0034" class="usa-link" aria-describedby="tpg270096-bib-0034">2007</a>). The merged genotypic data, comprised of 4255 high‐quality SNP markers, were used to calculate Roger's genetic distance. Based on the distance matrix, a classical multidimensional scaling was performed. To visualize the population structure, the first four principal components (PCs) were plotted. The MAGIC population was grouped based on the seven pedigrees defined by crossing order.</p>
<p>The genetic structure was explored by estimating the LD decay for the entire MAGIC population. The pairwise coefficient of correlations (<em>r</em>
<sup>2</sup>) between adjacent SNP markers was calculated. The decay of LD in relation to the SNP position was determined by fitting the locally estimated scatterplot smoothing curves (Wickham &amp; Sievert, <a href="#tpg270096-bib-0059" class="usa-link" aria-describedby="tpg270096-bib-0059">2016</a>). SNP markers greater than 50 Mbp position were considered unlinked and eliminated from the analysis (Roncallo et al., <a href="#tpg270096-bib-0040" class="usa-link" aria-describedby="tpg270096-bib-0040">2021</a>). Distributions of minor allele frequency and observed heterozygosity were also calculated (Figure <a href="#tpg270096-supinfo-0001" class="usa-link">S2</a>). All analyses were conducted in R using SelectionTools package and visualized using gglplot.</p></section><section id="tpg270096-sec-0090"><h3 class="pmc_sec_title">2.6. Phenotyping days to 50% flowering and plant height</h3>
<p>To evaluate the potential of the MAGIC population to map QTL and dissect the genetics of complex traits, we focused on two key agronomic traits: DTF and PHT (cm). The population was evaluated in the field in 2013 at the International Crops Research Institute for the Semi‐Arid Tropics. DTF is noted visually when 50% of the plants within the plot are flowering, and PHT (cm) is measured with ruler at maturity. The experimental design was augmented with parents as checks with eight replicates for estimation of error variance, as described by Samineni et al. (<a href="#tpg270096-bib-0041" class="usa-link" aria-describedby="tpg270096-bib-0041">2021</a>). The MAGIC lines were not replicated in this augmented design. Prior to the genome‐wide association mapping analysis, phenotypes were adjusted for block effect, and error variance was estimated using founder lines as checks in the experimental design. We used the ASReml‐R package (Butler et al., <a href="#tpg270096-bib-0007" class="usa-link" aria-describedby="tpg270096-bib-0007">2009</a>) in R to fit the following mixed model for the 2 years (R Core Team, <a href="#tpg270096-bib-0060" class="usa-link" aria-describedby="tpg270096-bib-0060">2021</a>):</p>
<table class="disp-formula p" id="tpg270096-disp-0001"><tr><td class="formula"><math id="jats-math-1" display="block"><mrow><mrow><mi mathvariant="bold-italic">y</mi><mo linebreak="badbreak">=</mo><mi>X</mi><mi mathvariant="normal">β</mi><mo linebreak="goodbreak">+</mo><mi>p</mi><mi mathvariant="bold-italic">w</mi><mo linebreak="goodbreak">+</mo><mi>z</mi><mi mathvariant="bold-italic">g</mi><mo linebreak="goodbreak">+</mo><mo>∈</mo><mo>,</mo></mrow></mrow></math></td></tr></table>
<p>where <strong>
<em>y</em>
</strong> represents a matrix of phenotypic values for each trait per year, <em>β</em> represents a vector of block effect, <strong>
<em>w</em>
</strong> represents a vector of the parents of the MAGIC lines, <strong>g</strong> represents the vector of MAGIC lines, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-2" display="inline"><mrow><mo>∈</mo></mrow></math></span> is error variance. For the base model, we fitted the parents (checks) as fixed effect, and the MAGIC lines as random. We removed outliers after running the base model and tested for significance of both the block and row effects in a fixed and random forms using the Wald test statistics and loglikelihood ratio, respectively. Thereafter, the final model was the same as the base model having confirmed that block and row effects were not significant in fixed and random effects. The adjusted phenotypic data were used for the downstream genome‐wide association analysis.</p></section><section id="tpg270096-sec-0100"><h3 class="pmc_sec_title">2.7. Genome‐wide association studies</h3>
<p>Genome‐wide association studies (GWASs) used 4255 SNP markers and phenotypes for DTF and PHT. GWAS was conducted in R using genomic association and prediction integrated version 3 (Wang &amp; Zhang, <a href="#tpg270096-bib-0052" class="usa-link" aria-describedby="tpg270096-bib-0052">2021</a>) following the statistical computations of the fixed and random model circulating probability unification as described by Liu et al. (<a href="#tpg270096-bib-0061" class="usa-link" aria-describedby="tpg270096-bib-0061">2016</a>). Marker‐trait associations for DTF and PHT were subjected to multiple testing corrections to identify significant QTL (Gao et al., <a href="#tpg270096-bib-0011" class="usa-link" aria-describedby="tpg270096-bib-0011">2008</a>). We used quantile‐quantile plots that represented expected and observed probability of obtaining associations of markers with respective traits −log10(<em>p</em>) values and a cut‐off threshold of −log10(<em>p</em>) ≥ 4.0 to reduce the type II error rate. Significant SNP markers physically linked in LD (<em>r</em>
<sup>2</sup> ≥ 0.5) were considered as a single QTL. Significant QTLs were compared to known QTLs in the literature.</p></section><section id="tpg270096-sec-0110"><h3 class="pmc_sec_title">2.8. Haplotype‐based mapping</h3>
<section id="tpg270096-sec-0120"><h4 class="pmc_sec_title">2.8.1. Construction of LD blocks</h4>
<p>Genome‐wide LD block construction was performed using an algorithm in the R package “SelectionTools”. First, <em>r</em>
<sup>2</sup> values across SNP markers were calculated throughout each chromosome. An LD threshold of <em>r</em>
<sup>2</sup> = 0.5 was applied to group SNP markers. A genome scan was performed where flanking markers within the threshold were grouped together as an LD block. To accommodate for incorrectly positioned markers or biased LD estimations, each block had a marker tolerance parameter of <em>t</em> = 3. This meant that even if a flanking marker did not match the LD grouping requirement, the block was still extended if at least <em>t</em> neighboring flanking markers did. If the <em>r</em>
<sup>2</sup> value of a pair of markers was larger than the threshold value of 0.5, the pair was identified as a new LD block. Flanking markers on either side of the LD block were considered in the following phase and included in the same block if their pairwise <em>r</em>
<sup>2</sup> values with the LD block's corresponding outside markers were also above the threshold value. The block was deemed complete if there were more than t flanking markers with a lower LD than <em>r</em>
<sup>2</sup> = 0.5. This process was repeated until all the markers had been assigned to their appropriate blocks.</p></section></section><section id="tpg270096-sec-0130"><h3 class="pmc_sec_title">2.9. Estimation of local GEBV and block variance</h3>
<p>The flowchart of the procedure is according to Voss‐Fels et al. (2019). First, we used the ridge‐regression best linear unbiased prediction model to estimate all marker effects simultaneously. Thereafter, we summed up the predicted allelic effects of each realized haplotype for all genome‐wide LD blocks, also referred as “local” genomic estimated breeding value (local GEBV). Finally, the variances of each LD block were estimated on all the local GEBVs for all the haplotypes on each LD block (Voss‐Fels et al., <a href="#tpg270096-bib-0062" class="usa-link" aria-describedby="tpg270096-bib-0062">2019</a>). LD blocks with high variances were considered containing the QTLs associated with DTF and PHT (Brunner et al., <a href="#tpg270096-bib-0006" class="usa-link" aria-describedby="tpg270096-bib-0006">2024</a>).</p></section><section id="tpg270096-sec-0140"><h3 class="pmc_sec_title">2.10. Co‐location analysis of QTL</h3>
<p>The genomic regions detected in this study were positioned with the already reported QTL using the physical position and displayed using MapChart version 2.3 (Voorrips, <a href="#tpg270096-bib-0063" class="usa-link" aria-describedby="tpg270096-bib-0063">2002</a>). We investigated the co‐location of the genomic regions associated with vigor and other agronomic traits already reported.</p>
<section id="tpg270096-sec-0150"><h4 class="pmc_sec_title">2.10.1. Haplotype analyses and haplotype block stacking</h4>
<p>In order to generate an estimate of the genealogies between common allelic variants for the most significant haploblock in our MAGIC population for DTF and PHT, we performed haplotype analysis. Haplotypes that were in strong LD were used to generate haplotype networks by employing the TCS (Templeton, Crandall, and Sing) phylogenetic network approach that was implemented in the software PopART (Leigh et al., <a href="#tpg270096-bib-0064" class="usa-link" aria-describedby="tpg270096-bib-0064">2015</a>). The means of lines carrying haplotype variants were then fitted to a linear model and tested for statistical significance in R.</p>
<p>The impacts of stacking haplotypes at the 10 blocks with the highest variance were analyzed for both traits. For the “desirable” haplotypes, we considered the haplotypes with negative effects for DTF, as early‐maturing chickpea plant could be advantageous, particularly when grown in a terminal drought‐prone environment. While for the PHT, we considered the positive effects haplotypes (associated with taller PHT) of the block with highest variance as shown to be advantageous. Taller plants height has a higher yield potential and enhanced efficiency for mechanical harvesting.</p></section></section></section><section id="tpg270096-sec-0160"><h2 class="pmc_sec_title">3. RESULTS</h2>
<section id="tpg270096-sec-0170"><h3 class="pmc_sec_title">3.1. Marker distribution, founder diversity, and genetic structure of the MAGIC population</h3>
<p>Following the removal of highly correlated markers, 4255 SNPs remained. Markers varied and were unequally distributed across and within the eight chromosomes (Figure <a href="#tpg270096-fig-0001" class="usa-link">1</a>). Based on the physical map, chromosome 3 had the lowest coverage of markers (six markers/Mbp), whereas chromosome 7 showed the highest coverage (17 markers/Mbp) (Figure <a href="#tpg270096-fig-0001" class="usa-link">1</a>). Across all chromosomes, there were some regions with low marker coverage indicating areas of genotyping bias. Co‐analysis of the MAGIC founders with the diversity panel provided insights into the genetic variation represented in the eight founders. Complete linkage hierarchical clustering subdivided the MAGIC founders and the diversity panel into two major clusters and further subdivided them into four distinct groups according to <em>k</em>‐means clustering with the optimum <em>k</em> number (Figure <a href="#tpg270096-fig-0002" class="usa-link">2</a>). Clusters 1 (<em>n </em>= 41) and 4 (<em>n </em>= 23) consisted of majority of the diversity panels, while Clusters 2 (<em>n</em> = 13) and 3 (<em>n</em> = 21) comprised of mixture of diversity panels and the MAGIC founder lines. Founders ICC 4958 and JG 11 are genetically more similar (Cluster 2) and differentiated with ICCV 97105, JG 16, JD 130, JAKI 9218, ICCV 10, and ICCV 0108 (Cluster 3) (Figure <a href="#tpg270096-fig-0002" class="usa-link">2</a>). There were notable conserved chromosomal regions within the eight founders and high allelic variation as evident on chromosomes (Ca) 3, 5, and 6 (Figure <a href="#tpg270096-supinfo-0001" class="usa-link">S3</a>).</p>
<figure class="fig xbox font-sm" id="tpg270096-fig-0001"><h4 class="obj_head">FIGURE 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/a6f8/12375853/5bfb070e061e/TPG2-18-e70096-g002.jpg" loading="lazy" id="jats-graphic-1" height="367" width="700" alt="FIGURE 1"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270096-fig-0001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Single nucleotide polymorphism (SNP) density and distribution in the multi‐parent advanced generation intercross (MAGIC) population across the eight chromosomes of chickpea. The number of SNP markers is estimated and visualized in 1 Mb window size for each of the chromosomes. The number of SNPs per Mb is color‐coded with red, which indicates high SNP density, while green indicates low SNP density.</p></figcaption></figure><figure class="fig xbox font-sm" id="tpg270096-fig-0002"><h4 class="obj_head">FIGURE 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/a6f8/12375853/28d28d556082/TPG2-18-e70096-g004.jpg" loading="lazy" id="jats-graphic-3" height="511" width="560" alt="FIGURE 2"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270096-fig-0002/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Genetic relationships among diversity panel (<em>n = </em>91) and the eight founder lines of the multi‐parent advanced generation intercross (MAGIC) population. Complete linkage hierarchical analysis and <em>k</em>‐means clustering with <em>k = </em>4 optimum number of clusters were identified: Cluster 1 (<em>n </em>= 41, purple), Cluster 2 (<em>n </em>= 13, pink), Cluster 3 (<em>n </em>= 22, green), and Cluster 4 (<em>n </em>= 22, orange). The eight founder lines are indicated on the right‐hand side of the heatmap.</p></figcaption></figure><p>Intercrossing of the eight founders followed by selfing generations successfully reshuffled the genome and resulted in a MAGIC population with minimal population structure (Figure <a href="#tpg270096-fig-0003" class="usa-link">3</a>), and clustering of lines did not generally correspond with crossing order (Figure <a href="#tpg270096-supinfo-0001" class="usa-link">S4</a>). Analysis of the whole genome SNP data for the entire MAGIC population revealed that PCs1–4, explained 12.89%, 9.97%, 7.85%, and 7.29% variance, respectively. The cumulative variation explained by the first two PCs accounted for 22.86% of the genetic variability in the population. The genetic distance showed limited population structure relative to the founder lines (Figure <a href="#tpg270096-fig-0003" class="usa-link">3</a>). LD within the MAGIC population showed a faster decay across the genome with a threshold of <em>r<sup>2</sup>
</em> = 0.2 at a distance of ∼1 Mbp (Figure <a href="#tpg270096-fig-0004" class="usa-link">4</a>). Overall, the mean MAF and observed heterozygosity were 0.290 and 0.021, respectively (Table <a href="#tpg270096-supinfo-0001" class="usa-link">S3</a>; Figure <a href="#tpg270096-supinfo-0001" class="usa-link">S2</a>).</p>
<figure class="fig xbox font-sm" id="tpg270096-fig-0003"><h4 class="obj_head">FIGURE 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/a6f8/12375853/2c11acd2f20a/TPG2-18-e70096-g003.jpg" loading="lazy" id="jats-graphic-5" height="512" width="700" alt="FIGURE 3"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270096-fig-0003/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Population structure of the multi‐parent advanced generation intercross (MAGIC) population and the eight founder lines. Principal component analysis based on the pairwise (Roger's genetic distances) calculated using 4255 high‐quality polymorphic single nucleotide polymorphism (SNP) markers for 1135 MAGIC lines and 8 founder lines. The variation explained by principal components (PCs) was PC1 = 12.89%, PC2 = 9.97%, PC3 = 7.85%, and PC4 = 7.29%. Each founder is a distinct color, and the MAGIC lines are displayed in black.</p></figcaption></figure><figure class="fig xbox font-sm" id="tpg270096-fig-0004"><h4 class="obj_head">FIGURE 4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/a6f8/12375853/17805777eaea/TPG2-18-e70096-g007.jpg" loading="lazy" id="jats-graphic-7" height="311" width="400" alt="FIGURE 4"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270096-fig-0004/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Genome‐wide linkage disequilibrium (LD) decay plot of the entire multi‐parent advanced generation intercross (MAGIC) population using 4255 high‐quality single nucleotide polymorphism (SNP) markers. Pairwise LD values (<em>r</em>
<sup>2</sup>) are plotted against physical SNP position. The red inner trend line is the locally estimated scatterplot smoothing (LOESS) curve. LD decay is shown as the intersection of the curves at the threshold of <em>r<sup>2</sup>
</em> = 0.2 at the distance of ∼1 Mbp.</p></figcaption></figure></section><section id="tpg270096-sec-0180"><h3 class="pmc_sec_title">3.2. Phenotypic variation for plant height (PHT) and days to flowering (DTF) in the MAGIC population</h3>
<p>Evaluation of MAGIC lines in the field across two seasons revealed transgressive segregation for both PHT and DTF, as some of the MAGIC lines had higher or lower phenotypic values compared to the founders (Figure <a href="#tpg270096-fig-0005" class="usa-link">5</a>). For example, PHT for founders ranged from 32.70 (ICC 4958) to 39.01 cm (ICCV 97105), whereas MAGIC lines ranged from 24.63 to 43.42 cm (Figure <a href="#tpg270096-fig-0005" class="usa-link">5</a>), indicating that particular MAGIC lines are shorter or taller than the eight founders. For DTF, the founders reached 50% flowering between 48.67 days after sowing (JG 11) and 53.37 days after sowing (ICCV 97105), whereas MAGIC lines ranged from 44.83 to 58.71 days (Figure <a href="#tpg270096-fig-0005" class="usa-link">5</a>), demonstrating that progeny showed a wider range in flowering behavior than the founders. The overall correlation between DTF and PHT among the population was <em>r<sup>2</sup>
</em> = 0.11 (<em>p</em> &lt; 0.01).</p>
<figure class="fig xbox font-sm" id="tpg270096-fig-0005"><h4 class="obj_head">FIGURE 5.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/a6f8/12375853/db0e05d2ab6f/TPG2-18-e70096-g008.jpg" loading="lazy" id="jats-graphic-9" height="350" width="400" alt="FIGURE 5"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270096-fig-0005/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Phenotypic variation across multi‐parent advanced generation intercross (MAGIC) population grown under field conditions. (A) Density plot of days to 50% flowering showing the positions of the founders (each indicated in a distinct color). (B) Density plot of the MAGIC population for plant height, the distribution showing the positions of the founders (each indicated in a distinct color).</p></figcaption></figure></section><section id="tpg270096-sec-0190"><h3 class="pmc_sec_title">3.3. Genomic regions associated with plant height and flowering</h3>
<p>A total of 19 unique SNPs (−log10(<em>p</em>‐value) = 4.00) were associated with both PHT and DTF (Figure <a href="#tpg270096-fig-0006" class="usa-link">6</a>; Figure <a href="#tpg270096-supinfo-0001" class="usa-link">S5</a>). Markers associated with PHT were detected on chromosomes 4, 5, 6, and 8, whereas markers associated with DTF were detected on chromosomes 2, 3, 4, 5, 6, and 8. The genome‐wide LD was employed as a criterion to identify whether two closely positioned associated markers were representing the same QTL region. SNP markers with LD ≥ 0.1 are linked (Table <a href="#tpg270096-supinfo-0001" class="usa-link">S2</a>). A total of 12 QTL regions were identified based on this threshold (Table <a href="#tpg270096-tbl-0001" class="usa-link">1</a>).</p>
<figure class="fig xbox font-sm" id="tpg270096-fig-0006"><h4 class="obj_head">FIGURE 6.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/a6f8/12375853/07491975fd72/TPG2-18-e70096-g006.jpg" loading="lazy" id="jats-graphic-11" height="301" width="600" alt="FIGURE 6"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270096-fig-0006/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Manhattan plot illustrating results from genome‐wide association mapping in the multi‐parent advanced generation intercross (MAGIC) population for days to 50% flowering (green) and plant height (blue) using 1135 MAGIC lines and 4255 high‐quality single nucleotide polymorphism (SNP) markers. Each dot on the Manhattan plot represents a single SNP marker. The horizontal line represents the threshold for significant marker‐trait associations (−log<sub>10</sub>(<em>p</em>) = 4.00), which are displayed as a larger sized dot.</p></figcaption></figure><section class="tw xbox font-sm" id="tpg270096-tbl-0001"><h4 class="obj_head">TABLE 1.</h4>
<div class="caption p"><p>Summary of the quantitative trait locus (QTL) for days to 50% flowering (Q‐DTF) and plant height (Q‐PH) in the chickpea multi‐parent advanced generation intercross (MAGIC) population.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<thead><tr style="border-bottom:solid 1px #000000">
<th align="left" rowspan="1" colspan="1">
<em>Q</em>/<em>N</em>
</th>
<th align="left" rowspan="1" colspan="1">Chr</th>
<th align="left" rowspan="1" colspan="1">QTL</th>
<th align="left" rowspan="1" colspan="1">Marker.Position</th>
<th align="left" rowspan="1" colspan="1">DTF −log10(<em>p</em>)</th>
<th align="left" rowspan="1" colspan="1">PH −log10(<em>p</em>)</th>
<th align="left" rowspan="1" colspan="1">QTL</th>
<th align="left" rowspan="1" colspan="1">References</th>
</tr></thead>
<tbody>
<tr>
<td align="left" rowspan="1" colspan="1">1</td>
<td align="left" rowspan="1" colspan="1">2</td>
<td align="left" rowspan="1" colspan="1">
<em>Q‐DTF1</em>
</td>
<td align="left" rowspan="1" colspan="1">M2.32058207</td>
<td align="left" rowspan="1" colspan="1">6.93</td>
<td align="left" rowspan="1" colspan="1">NA</td>
<td align="left" rowspan="1" colspan="1">Potentially novel</td>
<td align="left" rowspan="1" colspan="1"></td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">2</td>
<td align="left" rowspan="1" colspan="1">3</td>
<td align="left" rowspan="1" colspan="1">
<em>Q‐DTF2</em>
</td>
<td align="left" rowspan="1" colspan="1">M3.16743467</td>
<td align="left" rowspan="1" colspan="1">5.43</td>
<td align="left" rowspan="1" colspan="1">NA</td>
<td align="left" rowspan="1" colspan="1">
<em>qtlDtf‐3.1</em>
</td>
<td align="left" rowspan="1" colspan="1">Abdi et al. (<a href="#tpg270096-bib-0001" class="usa-link" aria-describedby="tpg270096-bib-0001">2016</a>)</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">3</td>
<td align="left" rowspan="1" colspan="1">3</td>
<td align="left" rowspan="1" colspan="1">
<em>Q‐DTF3</em>
</td>
<td align="left" rowspan="1" colspan="1">M3.23724626</td>
<td align="left" rowspan="1" colspan="1">4.41</td>
<td align="left" rowspan="1" colspan="1">NA</td>
<td align="left" rowspan="1" colspan="1">Potentially novel</td>
<td align="left" rowspan="1" colspan="1"></td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">4</td>
<td align="left" rowspan="1" colspan="1">4</td>
<td align="left" rowspan="1" colspan="1">
<em>Q‐DTF4</em>
</td>
<td align="left" rowspan="1" colspan="1">M4.1487970</td>
<td align="left" rowspan="1" colspan="1">4.03</td>
<td align="left" rowspan="1" colspan="1">NA</td>
<td align="left" rowspan="1" colspan="1">
<em>qtlDtf‐4.1</em>
</td>
<td align="left" rowspan="1" colspan="1">Abdi et al. (<a href="#tpg270096-bib-0001" class="usa-link" aria-describedby="tpg270096-bib-0001">2016</a>)</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">5</td>
<td align="left" rowspan="1" colspan="1">4</td>
<td align="left" rowspan="1" colspan="1">
<em>Q‐DTF5</em>
</td>
<td align="left" rowspan="1" colspan="1">M4.12926479</td>
<td align="left" rowspan="1" colspan="1">7.37</td>
<td align="left" rowspan="1" colspan="1">NA</td>
<td align="left" rowspan="1" colspan="1">
<em>qtlDtf‐4.2</em>
</td>
<td align="left" rowspan="1" colspan="1">Abdi et al. (<a href="#tpg270096-bib-0001" class="usa-link" aria-describedby="tpg270096-bib-0001">2016</a>)</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">4</td>
<td align="left" rowspan="1" colspan="1">
<em>Q‐PH1</em>
</td>
<td align="left" rowspan="1" colspan="1">M4.13421126</td>
<td align="left" rowspan="1" colspan="1">NA</td>
<td align="left" rowspan="1" colspan="1">31.77</td>
<td align="left" rowspan="1" colspan="1">
<em>qPH4.1/qtlDtf‐4.3</em>, <em>Ca4_Vqtl</em>
</td>
<td align="left" rowspan="1" colspan="1">Kujur et al. (<a href="#tpg270096-bib-0024" class="usa-link" aria-describedby="tpg270096-bib-0024">2016</a>)</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">6</td>
<td align="left" rowspan="1" colspan="1">4</td>
<td align="left" rowspan="1" colspan="1">
<em>Q‐DTF6</em>
</td>
<td align="left" rowspan="1" colspan="1">M4.17785478</td>
<td align="left" rowspan="1" colspan="1">4.12</td>
<td align="left" rowspan="1" colspan="1">NA</td>
<td align="left" rowspan="1" colspan="1">
<em>qtlDtf‐4.4</em>
</td>
<td align="left" rowspan="1" colspan="1">Kujur et al. (<a href="#tpg270096-bib-0024" class="usa-link" aria-describedby="tpg270096-bib-0024">2016</a>)</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">7</td>
<td align="left" rowspan="1" colspan="1">5</td>
<td align="left" rowspan="1" colspan="1">
<em>Q‐PH2</em>
</td>
<td align="left" rowspan="1" colspan="1">M5.32027655</td>
<td align="left" rowspan="1" colspan="1">NA</td>
<td align="left" rowspan="1" colspan="1">4.77</td>
<td align="left" rowspan="1" colspan="1">Potentially novel</td>
<td align="left" rowspan="1" colspan="1"></td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">8</td>
<td align="left" rowspan="1" colspan="1">5</td>
<td align="left" rowspan="1" colspan="1">
<em>Q‐DTF7</em>
</td>
<td align="left" rowspan="1" colspan="1">M5.32448226</td>
<td align="left" rowspan="1" colspan="1">6.34</td>
<td align="left" rowspan="1" colspan="1">NA</td>
<td align="left" rowspan="1" colspan="1">
<em>qtlDtf‐5.1</em>
</td>
<td align="left" rowspan="1" colspan="1">Kujur et al. (<a href="#tpg270096-bib-0024" class="usa-link" aria-describedby="tpg270096-bib-0024">2016</a>)</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">5</td>
<td align="left" rowspan="1" colspan="1">
<em>Q‐DTF8</em>
</td>
<td align="left" rowspan="1" colspan="1">M5.37601329</td>
<td align="left" rowspan="1" colspan="1">5.41</td>
<td align="left" rowspan="1" colspan="1">NA</td>
<td align="left" rowspan="1" colspan="1">
<em>qtlDtf‐5.2</em>
</td>
<td align="left" rowspan="1" colspan="1">Kujur et al. (<a href="#tpg270096-bib-0024" class="usa-link" aria-describedby="tpg270096-bib-0024">2016</a>)</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">5</td>
<td align="left" rowspan="1" colspan="1">
<em>Q‐PH3</em>
</td>
<td align="left" rowspan="1" colspan="1">M5.38102595</td>
<td align="left" rowspan="1" colspan="1">NA</td>
<td align="left" rowspan="1" colspan="1">4.07</td>
<td align="left" rowspan="1" colspan="1">Potentially novel</td>
<td align="left" rowspan="1" colspan="1"></td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">9</td>
<td align="left" rowspan="1" colspan="1">6</td>
<td align="left" rowspan="1" colspan="1">
<em>Q‐DTF9</em>
</td>
<td align="left" rowspan="1" colspan="1">M6.7564965</td>
<td align="left" rowspan="1" colspan="1">6.5</td>
<td align="left" rowspan="1" colspan="1">NA</td>
<td align="left" rowspan="1" colspan="1">Potentially novel</td>
<td align="left" rowspan="1" colspan="1"></td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">6</td>
<td align="left" rowspan="1" colspan="1">
<em>Q‐PH4</em>
</td>
<td align="left" rowspan="1" colspan="1">M6.9942982</td>
<td align="left" rowspan="1" colspan="1">NA</td>
<td align="left" rowspan="1" colspan="1">4.04</td>
<td align="left" rowspan="1" colspan="1">
<em>QR3pht01</em>
</td>
<td align="left" rowspan="1" colspan="1">Gupta et al. (<a href="#tpg270096-bib-0012" class="usa-link" aria-describedby="tpg270096-bib-0012">2015</a>); Varshney et al. (<a href="#tpg270096-bib-0065" class="usa-link" aria-describedby="tpg270096-bib-0065">2014</a>)</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">10</td>
<td align="left" rowspan="1" colspan="1">6</td>
<td align="left" rowspan="1" colspan="1">
<em>Q‐DTF10</em>
</td>
<td align="left" rowspan="1" colspan="1">M6.19084539</td>
<td align="left" rowspan="1" colspan="1">5.73</td>
<td align="left" rowspan="1" colspan="1">NA</td>
<td align="left" rowspan="1" colspan="1">Potentially novel</td>
<td align="left" rowspan="1" colspan="1"></td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">11</td>
<td align="left" rowspan="1" colspan="1">8</td>
<td align="left" rowspan="1" colspan="1">
<em>Q‐PH5</em>
</td>
<td align="left" rowspan="1" colspan="1">M8.673893</td>
<td align="left" rowspan="1" colspan="1">NA</td>
<td align="left" rowspan="1" colspan="1">4.3</td>
<td align="left" rowspan="1" colspan="1">
<em>qPH8.1</em>
</td>
<td align="left" rowspan="1" colspan="1">Kujur et al. (2016)</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">12</td>
<td align="left" rowspan="1" colspan="1">8</td>
<td align="left" rowspan="1" colspan="1">
<em>Q‐DTF11</em>
</td>
<td align="left" rowspan="1" colspan="1">M8.2762755</td>
<td align="left" rowspan="1" colspan="1">4.95</td>
<td align="left" rowspan="1" colspan="1">NA</td>
<td align="left" rowspan="1" colspan="1">
<em>qtlDtf‐8.1</em>
</td>
<td align="left" rowspan="1" colspan="1">Jaganathan et al. (<a href="#tpg270096-bib-0021" class="usa-link" aria-describedby="tpg270096-bib-0021">2015</a>); Varshney et al. (<a href="#tpg270096-bib-0065" class="usa-link" aria-describedby="tpg270096-bib-0065">2014</a>)</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">8</td>
<td align="left" rowspan="1" colspan="1">
<em>Q‐DTF12/Q‐PH6</em>
</td>
<td align="left" rowspan="1" colspan="1">M8.3011407</td>
<td align="left" rowspan="1" colspan="1">29.96</td>
<td align="left" rowspan="1" colspan="1">4.33</td>
<td align="left" rowspan="1" colspan="1">
<em>qtlDtf‐8.2</em>
</td>
<td align="left" rowspan="1" colspan="1">Jaganathan et al. (<a href="#tpg270096-bib-0021" class="usa-link" aria-describedby="tpg270096-bib-0021">2015</a>); Varshney et al. (<a href="#tpg270096-bib-0065" class="usa-link" aria-describedby="tpg270096-bib-0065">2014</a>)</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">8</td>
<td align="left" rowspan="1" colspan="1">
<em>Q‐PH7</em>
</td>
<td align="left" rowspan="1" colspan="1">M8.4764769</td>
<td align="left" rowspan="1" colspan="1">NA</td>
<td align="left" rowspan="1" colspan="1">7.96</td>
<td align="left" rowspan="1" colspan="1">
<em>qPH8.1</em>
</td>
<td align="left" rowspan="1" colspan="1">Jaganathan et al. (<a href="#tpg270096-bib-0021" class="usa-link" aria-describedby="tpg270096-bib-0021">2015</a>); Kujur et al. (<a href="#tpg270096-bib-0024" class="usa-link" aria-describedby="tpg270096-bib-0024">2016</a>); Varshney et al. (<a href="#tpg270096-bib-0065" class="usa-link" aria-describedby="tpg270096-bib-0065">2014</a>)</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">8</td>
<td align="left" rowspan="1" colspan="1">
<em>Q‐DTF13</em>
</td>
<td align="left" rowspan="1" colspan="1">M8.5166666</td>
<td align="left" rowspan="1" colspan="1">11.15</td>
<td align="left" rowspan="1" colspan="1">NA</td>
<td align="left" rowspan="1" colspan="1">Potentially novel</td>
<td align="left" rowspan="1" colspan="1"> </td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/tpg270096-tbl-0001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="tpg270096-tbl1-note-0002"><p>Abbreviations: LD, linkage disequilibrium. NA, not applicable.</p></div></div></section><p>A total of six unique QTLs were associated with DTF, and two unique QTLs associated with PHT. The most significant QTL for PHT (<em>Q‐PH1</em>) is closely linked with a QTL for DTF (<em>Q‐DTH5</em>). Interestingly, the most significant QTL for DTF (<em>Q‐PH12</em>) was also detected associated with a QTL for PHT (<em>Q‐PH6</em>) by a unique marker (M8.3011407). This corresponds to a genomic region previously identified and associated with DTF, and our results also identified this region to be associated with PHT. Notably, seven potentially novel QTLs for both PHT (<em>n </em>= 2) and DTF (<em>n </em>= 5) were identified using the MAGIC population. Overall, a total of four common closely linked QTLs were identified modulating both PHT and DTF.</p></section><section id="tpg270096-sec-0200"><h3 class="pmc_sec_title">3.4. Haplotype and effects of stacking of importance region for DTF and PHT</h3>
<p>We also performed haplotype‐based mapping for DTF and PHT using the local genomic estimated breeding value approach (Table <a href="#tpg270096-supinfo-0001" class="usa-link">S4</a>). We identified the top 10 haplotype blocks for both DTF and PHT based on the highest variances. LD block <em>b000646</em> on chromosome 8 showed the highest variance for DTF (Figure <a href="#tpg270096-fig-0007" class="usa-link">7A</a>), and <em>b000250</em> on chromosome 4 ranked highest for PHT (Figure <a href="#tpg270096-fig-0007" class="usa-link">7C</a>).</p>
<figure class="fig xbox font-sm" id="tpg270096-fig-0007"><h4 class="obj_head">FIGURE 7.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/a6f8/12375853/7c45c7a14f28/TPG2-18-e70096-g005.jpg" loading="lazy" id="jats-graphic-13" height="411" width="546" alt="FIGURE 7"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270096-fig-0007/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Haplotype‐based mapping of days to 50% flowering (DTF) and plant height (PHT) haplotype effects and variances. (A and C) Manhattan plot of haploblocks showing variances on the eight chickpea chromosomes, and (B and D) corresponding haplotype effects (local GEBV [genomic estimated breeding values]) in high variance blocks for DTF and PHT, respectively. For visualization, haploblock variance is scaled to clearly illustrate the relationship of haplotype effects and haploblock variance.</p></figcaption></figure><p>To explore the distribution of haplotype effects for DTF and PHT, Figure <a href="#tpg270096-fig-0007" class="usa-link">7B</a> (DTF) and <a href="#tpg270096-fig-0007" class="usa-link">D</a> (PHT) shows their effects plotted against a scaled (0–1) variance for each block. Interestingly, both negative and positive effect haplotypes were evident for both traits (Figure <a href="#tpg270096-fig-0007" class="usa-link">7B,D</a>). For DTF, negative effect haplotypes were associated with earlier flowering, while for PHT, positive effect haplotypes were associated with taller PHT.</p>
<p>Our in silico haplotype stacking analysis revealed a linear trend between the number of desirable haplotype blocks and DTF (Figure <a href="#tpg270096-fig-0008" class="usa-link">8</a>). Meanwhile, because the visualization of the block variance showed a distinct block with highest variance on chromosome 4 (Figure <a href="#tpg270096-fig-0007" class="usa-link">7D</a>), haplotype stacking analysis that was performed using this highest variance PHT block revealed a similar trend (Figure <a href="#tpg270096-fig-0009" class="usa-link">9</a>).</p>
<figure class="fig xbox font-sm" id="tpg270096-fig-0008"><h4 class="obj_head">FIGURE 8.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/a6f8/12375853/31e746f2b4bf/TPG2-18-e70096-g001.jpg" loading="lazy" id="jats-graphic-15" height="334" width="600" alt="FIGURE 8"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270096-fig-0008/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>The effect of stacking top 10 haplotype blocks with negative effects (favorable haplotype blocks) on days to 50% flowering (DTF). Horizontal line within each boxplot represents the median value.</p></figcaption></figure><figure class="fig xbox font-sm" id="tpg270096-fig-0009"><h4 class="obj_head">FIGURE 9.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/a6f8/12375853/8c4d05739a36/TPG2-18-e70096-g009.jpg" loading="lazy" id="jats-graphic-17" height="750" width="600" alt="FIGURE 9"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270096-fig-0009/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Haplotype (hap) analysis of <em>b000250</em>, which showed the highest variance for plant height. (A) Haplotype network of the 11 haplotype variants segregating in the multi‐parent advanced generation intercross (MAGIC) population. (B) Boxplot illustrating the plant height (PHT) phenotypes of lines carrying the haplotype variants. Middle horizontal line represents mean plant height.</p></figcaption></figure><p>The block was on chromosome 4 and ranged from 11.84 to 14.18 Mbp. Haplotype analysis of the block revealed a total of 11 haplotype variants (hapPHT1–11) segregating in the MAGIC population (Figure <a href="#tpg270096-fig-0009" class="usa-link">9</a>). Of these, four showed high frequencies (greater than 70 individuals in the whole population), while four showed medium frequencies (more than 10 individuals), and three haplotypes were deemed rare (1 in the whole population). To understand the PHT response among haplotypes, we compared the haplotypes using the phenotypic dataset for their cumulative effects. HapPHT1 is most common haplotype with 338 counts in the population (∼32%), but hapPHT7, which was only present in 1.1% of the population, displayed the highest mean for PHT. The haplotypes hapPHT1, hapPHT2, and hapPHT3 performed similarly, and hapPHT4, haPHT5, and haPHT6 also performed similarly for PHT.</p></section></section><section id="tpg270096-sec-0210"><h2 class="pmc_sec_title">4. DISCUSSION</h2>
<p>In this study, we analyzed the genetic architecture of a recently developed chickpea MAGIC population derived from eight genetically diverse founder lines and explored its utility for dissecting the genetics of traits to support crop improvement programs. Through the analysis of a panel of diverse chickpea accessions collected from 14 countries across the globe, we revealed the high degree of genetic diversity captured by MAGIC founder lines. As expected, the resulting population showed little to no population structure related to founder effect or crossing order and a high rate of LD decay, which are common features of MAGIC populations developed in other crop species. GWAS identified previously reported and novel QTL controlling both flowering and PHT, including four genomic regions that were associated with both traits. Haplotype analysis revealed new alleles combination that could potentially be used to breed improved chickpea varieties with earlier flowering to avoid terminal drought or taller PHT, which could improve efficiency of mechanized harvesting.</p>
<section id="tpg270096-sec-0220"><h3 class="pmc_sec_title">4.1. Founders exhibit the genetic diversity essential for MAGIC population development</h3>
<p>Genetic diversity is an essential ingredient to support genetic gain in crop improvement programs. MAGIC populations derived from diverse founders provide useful resources to explore and reshuffle genetic diversity that could be integrated into a breeding program. The chickpea MAGIC population was generated from eight diverse Asian and African elite chickpea varieties. To explore the genetic variation represented by the eight founders, we studied a diverse panel of 91 chickpea genotypes. Cluster analysis using whole‐genome SNP markers determined that the MAGIC founders captured a substantial proportion of diversity present in the 91 desi varieties, with founders falling into two of the four clusters (Figure <a href="#tpg270096-fig-0002" class="usa-link">2</a>). Therefore, the degree of diversity captured in this MAGIC population makes it a valuable resource for pre‐breeders and plant scientists to study important traits for chickpea improvement compared to using biparental populations. This is because biparental populations capture allelic variation from only two parental lines, whereas the MAGIC population captures and reshuffles the genetic diversity of eight founders, thereby representing a broader and more diverse genetic base. Genome‐wide allele assessment of the founder lines revealed key chromosome regions that were polymorphic and conserved among select founders (Figure <a href="#tpg270096-supinfo-0001" class="usa-link">S3</a>); however, as the crossing strategy recombined parental alleles from eight founders, the MAGIC progeny showed polymorphic SNP variation across most of the genome. Overall, genomic characterization of the founders showed they represent a degree of genetic diversity present in chickpea germplasm sourced from Asia and Africa.</p></section><section id="tpg270096-sec-0230"><h3 class="pmc_sec_title">4.2. A MAGIC resource with low population structure and rapid LD decay</h3>
<p>It is critical to consider population structure, LD decay, and genetic relatedness as key features of a mapping population that can determine the success and outcome of genotype–phenotype association studies (Ladejobi et al., <a href="#tpg270096-bib-0025" class="usa-link" aria-describedby="tpg270096-bib-0025">2016</a>; Scott et al., <a href="#tpg270096-bib-0043" class="usa-link" aria-describedby="tpg270096-bib-0043">2020</a>). The intercrossing strategy employed to generate an eight‐way MAGIC population aims to recombine the genome of diverse founders in the resulting progeny. As a result, low population structure was observed. For example, only 12.89% and 9.97% of variation in the population was explained by the first two PCs (Figure <a href="#tpg270096-fig-0003" class="usa-link">3</a>). Despite the fact that founder lines were crossed in different orders to create the MAGIC progeny, no clear clustering pattern was associated with crossing order (Figure <a href="#tpg270096-supinfo-0001" class="usa-link">S4</a>). Similar results have been reported for MAGIC populations in other crops including common bean (Diaz et al., <a href="#tpg270096-bib-0010" class="usa-link" aria-describedby="tpg270096-bib-0010">2020</a>), cowpea (Huynh et al., <a href="#tpg270096-bib-0019" class="usa-link" aria-describedby="tpg270096-bib-0019">2018</a>), and rice (Bandillo et al., <a href="#tpg270096-bib-0004" class="usa-link" aria-describedby="tpg270096-bib-0004">2013</a>). This indicates that MAGIC progeny are truly genetic mosaics of the founders used and highlights the effectiveness of the approach across species to help dissect the genetic architecture of the trait of interest.</p>
<p>A key driver of LD decay during population development is recombination, where higher recombination rate leads to higher LD decay. However, several other factors can also influence LD decay including mutation rate, relatedness (kinship), selection, genetic diversity, and population structure (Bastien et al., <a href="#tpg270096-bib-0066" class="usa-link" aria-describedby="tpg270096-bib-0066">2014</a>). In this chickpea MAGIC population, LD decayed rapidly and reached a threshold of <em>r</em>
<sup>2</sup> = 0.2 at a distance of ∼1 Mbp. This is faster LD decay when compared to decay in chickpea biparental populations (Y. Li et al., <a href="#tpg270096-bib-0029" class="usa-link" aria-describedby="tpg270096-bib-0029">2017</a>). The study by Thudi et al. (<a href="#tpg270096-bib-0046" class="usa-link" aria-describedby="tpg270096-bib-0046">2024</a>) analyzed the same MAGIC population, but used a different set of markers, and reported a similar threshold of ∼1.2 Mbp (Thudi et al., 2024). While we consider this to be a rapid LD decay for chickpea, previous studies have reported LD decay as fast as 0.7 Mbp using a panel of advanced breeding lines and cultivars (Raman et al., <a href="#tpg270096-bib-0036" class="usa-link" aria-describedby="tpg270096-bib-0036">2022</a>). Without having access to a common set of markers, we hypothesize that this is likely due to a different level of genetic diversity. Certainly, rapid LD decay is a common feature of MAGIC populations, such as those created for common bean (Diaz et al., <a href="#tpg270096-bib-0010" class="usa-link" aria-describedby="tpg270096-bib-0010">2020</a>) and cowpea (Huynh et al., <a href="#tpg270096-bib-0019" class="usa-link" aria-describedby="tpg270096-bib-0019">2018</a>), as the successive rounds of crossing followed by multiple generations of SSD assists in recombining linkage blocks, thereby reducing LD (Islam et al., <a href="#tpg270096-bib-0020" class="usa-link" aria-describedby="tpg270096-bib-0020">2016</a>). The fast LD decay is useful to support high‐resolution QTL mapping to reduce the size of the chromosomal interval and reduce the number of candidate genes underlying the QTL, which is important for subsequent QTL introgression, haplotype stacking, or gene discovery studies (Alqudah et al., <a href="#tpg270096-bib-0003" class="usa-link" aria-describedby="tpg270096-bib-0003">2020</a>).</p>
<p>While bi‐parental chickpea populations have been widely adopted in the past to successfully map QTL, this MAGIC population offers advantages and opportunities for future studies. For instance, the allelic variation segregating in the population, and LD decay rate, presents opportunities to map important regions for variety development with multiple alleles at each locus. The large population size of 1135 RILs will also improve the statistical power to detect more and rare QTL, even alleles that are contributed by a single founder, which still occur at a frequency of ∼12.5% in the population.</p></section><section id="tpg270096-sec-0240"><h3 class="pmc_sec_title">4.3. QTL discovery using the MAGIC population reveals key genomic regions associated with DTF and PHT</h3>
<p>In this chickpea MAGIC population, we observed wide phenotypic variation for DTF. Notably, some lines displayed transgressive segregation with earlier and later flowering behavior in comparison to the founders (Figure <a href="#tpg270096-fig-0008" class="usa-link">8</a>). This phenotypic variation enabled us to map 13 QTLs for DTF across chromosomes 2, 3, 4, 5, 6, and 8. Our analyses reconfirmed flowering as a complex trait, as evidenced by the many loci across the genome, each with relatively small effects. QTL on chromosomes 2 (Q‐DTF2), 4 (Q‐DTF4; Q‐DTF5/Q‐PH1), 5 (Q‐DTF7; Q‐DTF8), and 8 (Q‐DTF11; Q‐DTF12/Q‐PH6; Q‐DTF13) were positioned in close proximity to flowering QTLs reported in previous studies (Abdi et al., <a href="#tpg270096-bib-0001" class="usa-link" aria-describedby="tpg270096-bib-0001">2016</a>; Jaganathan et al., <a href="#tpg270096-bib-0021" class="usa-link" aria-describedby="tpg270096-bib-0021">2015</a>; Kujur et al., <a href="#tpg270096-bib-0024" class="usa-link" aria-describedby="tpg270096-bib-0024">2016</a>; Varshney, Mir et al., <a href="#tpg270096-bib-0048" class="usa-link" aria-describedby="tpg270096-bib-0048">2014</a>). Notably, five QTLs were deemed new, including those positioned on chromosomes 2 (Q‐DTF1), 3 (Q‐DTF1), 6 (Q‐DTF9; Q‐DTF10), and 8 (Q‐DTF13).</p>
<p>Transgressive segregation for PHT was also observed in the MAGIC population. A total of seven QTLs were mapped on chromosomes 4 (Q‐PH1), 5 (Q‐PH2 and Q‐PH3), 6 (Q‐PH4), and 8 (Q‐PH5, Q‐PHT6, and Q‐PH13). Two of the seven QTLs (i.e., Q‐PH1 and Q‐PH6) co‐located with regions detected in previous studies (Gupta et al., <a href="#tpg270096-bib-0012" class="usa-link" aria-describedby="tpg270096-bib-0012">2015</a>; Kujur et al., <a href="#tpg270096-bib-0024" class="usa-link" aria-describedby="tpg270096-bib-0024">2016</a>; Varshney, Mohan et al., <a href="#tpg270096-bib-0049" class="usa-link" aria-describedby="tpg270096-bib-0049">2014</a>). Two QTLs on chromosome 5 (Q‐PH2 and Q‐PH3) were deemed new QTLs for PHT. This highlights the utility of the MAGIC population to discover potentially new loci that could be useful for crop improvement.</p>
<p>Interestingly, we observed a positive correlation between DTF and PHT, suggesting overlapping genetic controls. Our GWAS identified four genomic regions associated with both traits on chromosomes 4 (Q‐DFT5/Q‐PH1), 5 (Q‐DFT7/Q‐DTF8/Q‐PH3), 6 (Q‐DFT9/Q‐PH4), and 8 (Q‐DTF11/Q‐DTF12/Q‐PH6/Q‐PH7/Q‐DTF13) (Table <a href="#tpg270096-tbl-0001" class="usa-link">1</a>). Notably, the same genomic region on chromosome 4 (Q‐DFT5/Q‐PH1) at 12.65–13.06 Mbp co‐located with a QTL reported for vigor (Ca4_Vqtl1) by Nguyen et al. (<a href="#tpg270096-bib-0032" class="usa-link" aria-describedby="tpg270096-bib-0032">2022</a>). This suggests the region could be involved in modulating all three traits: vigor, PHT, and flowering time. To confirm this, the MAGIC population could be phenotyped for vigor to test the allele effects on all three traits. Besides, functional categorization analysis of the genes co‐located within ±50 kb of the potential new QTL revealed a strong enrichment of stress‐responsive, developmental, and regulatory genes. Overall, the GWAS results highlight the power of the MAGIC population to dissect the genetic regions underlying complex traits, such as DTF and PHT. The distribution of the detected QTL across chickpea chromosomes indicates that these traits have complex genetic architectures and are both quantitative in nature. Superior MAGIC progeny identified in this study may be useful donor material for inclusion in breeding programs targeting improvement of flowering and PHT, or other important traits of interest.</p></section><section id="tpg270096-sec-0250"><h3 class="pmc_sec_title">4.4. Haplotype and stacking analyses reveal key allelic combinations for early DTF and taller PHT</h3>
<p>Analysis of haplotypes that include multiple markers in high LD can be used to account for the fact that multiple marker alleles are inherited as a unit and are less likely to be affected by recombination (Bevan et al., <a href="#tpg270096-bib-0067" class="usa-link" aria-describedby="tpg270096-bib-0067">2017</a>; Hao et al., <a href="#tpg270096-bib-0068" class="usa-link" aria-describedby="tpg270096-bib-0068">2012</a>). In our study, the highest variance haploblock for DTF was <em>b000646</em> on chromosome 8 and <em>b000250</em> on chromosome 4 for PHT. Haplotype networks showed genealogies among different haplotypes and graphically display their relatedness. Most importantly, it is constructed and used to reveal the allelic variants that arise from different recombination or chromosomal mutations (Glaszmann et al., <a href="#tpg270096-bib-0069" class="usa-link" aria-describedby="tpg270096-bib-0069">2010</a>). The LD block <em>b000250</em> for PHT comprises of 74 makers. The haplotype with the largest effect on PHT at <em>b000250</em> was hapPHT7 (Figure <a href="#tpg270096-fig-0009" class="usa-link">9</a>), which only occurred at a frequency of 1.1% in the population (<em>n</em> = 11). The rare haplotype was contributed by founder ICCV 10, yet many other haplotype variants that were more common in the population showed similar mean phenotypes for PHT, for instance, hapPHT4, hapPHT6, and hapPHT5. These desirable haplotypes could assist the selection and breeding of taller chickpea varieties that may be better suited for mechanical harvesting.</p>
<p>Through the in silico stacking analyses, we also demonstrated the potential for combining multiple desirable haplotypes to modulate time to flowering. Genomics‐assisted breeding approaches could be employed using speed breeding methods to generate stacked lines faster (Varshney et al., <a href="#tpg270096-bib-0070" class="usa-link" aria-describedby="tpg270096-bib-0070">2021</a>), which could lead to development of early flowering chickpea, which is a desirable trait in many production enviroments to avoid drought stress and resulting yield losses in chickpea.</p></section></section><section id="tpg270096-sec-0260"><h2 class="pmc_sec_title">5. CONCLUSION</h2>
<p>This study conducted a genetic characterization of a newly developed MAGIC population derived from eight founders. Our analyses demonstrate the founders, while originally selected for performance in a range of production environments, also represent a high degree of genetic diversity based on co‐analysis of germplasm from Asia and Africa. A large population was developed (1135 lines), and, as anticipated, showed features in common with other MAGIC populations, such as low population structure, high rate of LD decay, and high allelic variation. We highlighted the power of the MAGIC population for trait mapping by studying the genetics of DTF and PHT. Analyses confirmed the role of genomic regions previously reported in the literature and identified several new QTL for both traits. Interestingly, a QTL associated with both DTF and PHT on chromosome 4 (Q‐DTF5/Q‐PH1) co‐located with a QTL reported for vigor by Kujur et al. (<a href="#tpg270096-bib-0024" class="usa-link" aria-describedby="tpg270096-bib-0024">2016</a>), suggesting the region may harbor a locus with pleiotropic effects, or alternatively multiple linked loci, that chickpea breeders could potentially exploit to improve all three traits simultaneously. Haplotype and stacking analyses showed the potential for trait improvement through stacking the top 10 haploblocks to develop early flowering chickpea, and selection of desirable haplotypes on chromosome 4 to improve PHT. In summary, the chickpea MAGIC population provides a useful genetic resource for genetics research and pre‐breeding programs for a range of traits important for crop improvement.</p></section><section id="tpg270096-sec-0270"><h2 class="pmc_sec_title">AUTHOR CONTRIBUTIONS</h2>
<p>
<strong>Oluwaseun J. Akinlade</strong>: Conceptualization; data curation; formal analysis; methodology; writing—original draft. <strong>Hannah Robinson</strong>: Writing—review and editing. <strong>Yichen Kang</strong>: Writing—review and editing. <strong>Mahendar Thudi</strong>: Resources; writing—review and editing. <strong>Srinivasan Samineni</strong>: Resources; writing—review and editing. <strong>Pooran Gaur</strong>: Resources; writing—review and editing. <strong>Millicent R. Smith</strong>: Writing—review and editing. <strong>Kai P. Voss‐Fels</strong>: Writing—review and editing. <strong>Roy Costilla</strong>: Writing—review and editing. <strong>Rajeev K. Varshney</strong>: Resources; supervision; writing—review and editing. <strong>Eric Dinglasan</strong>: Conceptualization; supervision; writing—review and editing. <strong>Lee T. Hickey</strong>: Conceptualization; supervision; writing—review and editing.</p></section><section id="tpg270096-sec-0290"><h2 class="pmc_sec_title">CONFLICT OF INTEREST STATEMENT</h2>
<p>The authors declare no conflicts of interest.</p></section><section id="sec29"><h2 class="pmc_sec_title">Supporting information</h2>
<section class="sm xbox font-sm" id="tpg270096-supinfo-0001"></section><section class="sm xbox font-sm" id="tpg270096-supitem-0001"><div class="caption p">
<p>Supplementary Table 1 List and code of parental lines used for the development of the chickpea MAGIC population, including their description and country of cultivation.</p>
<p>Supplementary Table 2. Linkage disequilibrium (LD) among the pairs of markers significantly associated with DTF and PHT to determine those that are linked in order to categorize them into QTL. SNP markers with LD ≥ 0.1 are linked</p>
<p>Supplementary Table 3 Summary statistics of distributions of minor allele frequencies (MAF) and observed heterozygosity in genotypic data of population.</p>
<p>Supplementary Table 4 Summary of the QTL for days to 50% flowering (<em>Q‐DTF</em>) and plant height (<em>Q‐PH</em>) in the chickpea MAGIC population. Linked markers in LD ≥ 0.1 are highlighted in bold. With the haplo‐blocks co‐located in the same regions.</p>
<p>Supplementary Figure 1. Population structure of the MAGIC population and their founder parents. Principal component analysis based on the pairwise genomic relationship matrix (Roger's distances) calculated from 4255 high‐quality polymorphic SNP markers in 1135 MAGIC lines representing seven crossing designs and eight founder lines. The variation explained by principal components were PC1 = 12.89% and PC2 = 9.97%. The colored triangles represent the parents while the colored circles represent the cross‐design (pedigree).</p>
<p>Supplementary Figure 2 (A) Allele frequency spectra (histogram distributions of minor allele frequency [MAF]). (B) Distribution histograms of observed heterozygosity across the MAGIC population.</p>
<p>Supplementary Figure 3. Genome‐wide profile of the eight MAGIC founder lines. SNP alleles are represented by colors: adenine (A) is indicated in yellow, cytosine (C) is represented in blue, guanine (G) is represented in red and thymine (T) is represented in green.</p>
<p>Supplementary Figure 4. Population structure of the MAGIC population and their founder parents. Principal component analysis based on the pairwise genomic relationship matrix (Roger's distances) calculated from 4255 high‐quality polymorphic SNP markers in 1135 MAGIC lines representing seven crossing designs and eight founder lines. The variation explained by principal components were PC1 = 12.89% and PC2 = 9.97%. The colored triangles represent the parents while the colored circles represent the cross‐design (pedigree).</p>
<p>Supplementary Figure 5. Genome‐wide association mapping for plant height (PHT) and days to 50% flowering (DTF) for 1135 MAGIC lines using 4255 high‐quality SNP markers. <em>QQ</em> plots for plant height in blue and days to 50% flowering in green are displayed.</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12375853/bin/TPG2-18-e70096-s001.pdf" data-ga-action="click_feat_suppl" class="usa-link">TPG2-18-e70096-s001.pdf</a><sup> (702KB, pdf) </sup>
</div></div></section></section><section id="tpg270096-sec-0280" class="ack"><h2 class="pmc_sec_title">ACKNOWLEDGMENTS</h2>
<p>We are grateful for the development of the MAGIC population by the researchers at International Crops Research Institute for the Semi‐Arid Tropics (ICRISAT).</p>
<p>Open access publishing facilitated by The University of Queensland, as part of the Wiley ‐ The University of Queensland agreement via the Council of Australian University Librarians.</p></section><section id="notes1"><p>


Akinlade, O. J.
, 
Robinson, H.
, 
Kang, Y.
, 
Thudi, M.
, 
Samineni, S.
, 
Gaur, P.
, 
Smith, M. R.
, 
Voss‐Fels, K. P.
, 
Costilla, R.
, 
Varshney, R. K.
, 
Dinglasan, E.
, &amp; 
Hickey, L. T.
 (2025). A chickpea MAGIC population to dissect the genetics of complex traits. The Plant Genome, 18, e70096. 10.1002/tpg2.70096

</p>
<section id="fn-group1" class="fn-group"><div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="tpg270096-note-0001"><p>Assigned to Associate Editor Mukesh Jain.</p></div></div></section></section><section id="_ci93_" lang="en" class="contrib-info"><h2 class="pmc_sec_title">Contributor Information</h2>
<p>Eric Dinglasan, Email: e.dinglasan@uq.edu.au.</p>
<p>Lee T. Hickey, Email: l.hickey@uq.edu.au.</p></section><section id="tpg270096-sec-0310"><h2 class="pmc_sec_title">DATA AVAILABILITY STATEMENT</h2>
<p>All data supporting the findings of this study are available within the paper and within its <a href="#tpg270096-supinfo-0001" class="usa-link">Supporting Information</a>. Genotypic and phenotypic datasets are available from the corresponding author upon request.</p></section><section id="tpg270096-bibl-0001" class="ref-list"><h2 class="pmc_sec_title">REFERENCES</h2>
<section id="tpg270096-bibl-0001_sec2"><ol class="ref-list font-sm">
<li id="tpg270096-bib-0001">
<cite>

Abdi, K.
, 
Deokar, A.
, 
Banniza, S.
, 
Warkentin, T.
, &amp; 
Tar'an, B.
 (2016). QTL mapping of early flowering and resistance to Ascochyta blight in chickpea (<em>Cicer arietinum</em> L.). Genome, 59, 413–425. 10.1139/gen-2016-0036

</cite> [<a href="https://doi.org/10.1139/gen-2016-0036" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27244453/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genome&amp;title=QTL%20mapping%20of%20early%20flowering%20and%20resistance%20to%20Ascochyta%20blight%20in%20chickpea%20(Cicer%20arietinum%20L.)&amp;author=K.%20Abdi&amp;author=A.%20Deokar&amp;author=S.%20Banniza&amp;author=T.%20Warkentin&amp;author=B.%20Tar'an&amp;volume=59&amp;publication_year=2016&amp;pages=413-425&amp;pmid=27244453&amp;doi=10.1139/gen-2016-0036&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270096-bib-0002">
<cite>

Akinlade, O. J.
, 
Voss‐Fels, K.
, 
Costilla, R.
, 
Kholova, J.
, 
Choudhary, S.
, 
Varshney, R. K.
, 
Hickey, L. T.
, &amp; 
Smith, M. R.
 (2022). Designing chickpea for a hotter drier world. Euphytica, 218(7), Article 100. 10.1007/s10681-022-03048-2</cite> [<a href="https://doi.org/10.1007/s10681-022-03048-" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Euphytica&amp;title=Designing%20chickpea%20for%20a%20hotter%20drier%20world&amp;author=O.%20J.%20Akinlade&amp;author=K.%20Voss%E2%80%90Fels&amp;author=R.%20Costilla&amp;author=J.%20Kholova&amp;author=S.%20Choudhary&amp;volume=218&amp;issue=7&amp;publication_year=2022&amp;pages=100&amp;doi=10.1007/s10681-022-03048-&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270096-bib-0003">
<cite>

Alqudah, A. M.
, 
Sallam, A.
, 
Stephen Baenziger, P.
, &amp; 
Börner, A.
 (2020). GWAS: Fast‐forwarding gene identification and characterization in temperate cereals: Lessons from barley—A review. Journal of Advanced Research, 22, 119–135. 10.1016/j.jare.2019.10.013

</cite> [<a href="https://doi.org/10.1016/j.jare.2019.10.013" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6961222/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31956447/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Advanced%20Research&amp;title=GWAS:%20Fast%E2%80%90forwarding%20gene%20identification%20and%20characterization%20in%20temperate%20cereals:%20Lessons%20from%20barley%E2%80%94A%20review&amp;author=A.%20M.%20Alqudah&amp;author=A.%20Sallam&amp;author=P.%20Stephen%20Baenziger&amp;author=A.%20B%C3%B6rner&amp;volume=22&amp;publication_year=2020&amp;pages=119-135&amp;pmid=31956447&amp;doi=10.1016/j.jare.2019.10.013&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270096-bib-0004">
<cite>

Bandillo, N.
, 
Raghavan, C.
, 
Muyco, P. A.
, 
Sevilla, M. A. L.
, 
Lobina, I. T.
, 
Dilla‐Ermita, C. J.
, 
Tung, C.‐W.
, 
Mccouch, S.
, 
Thomson, M.
, 
Mauleon, R.
, 
Singh, R. K.
, 
Gregorio, G.
, 
Redoña, E.
, &amp; 
Leung, H.
 (2013). Multi‐parent advanced generation inter‐cross (MAGIC) populations in rice: Progress and potential for genetics research and breeding. Rice, 6(1), Article 11. 10.1186/1939-8433-6-11

</cite> [<a href="https://doi.org/10.1186/1939-8433-6-11" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4883706/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24280183/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Rice&amp;title=Multi%E2%80%90parent%20advanced%20generation%20inter%E2%80%90cross%20(MAGIC)%20populations%20in%20rice:%20Progress%20and%20potential%20for%20genetics%20research%20and%20breeding&amp;author=N.%20Bandillo&amp;author=C.%20Raghavan&amp;author=P.%20A.%20Muyco&amp;author=M.%20A.%20L.%20Sevilla&amp;author=I.%20T.%20Lobina&amp;volume=6&amp;issue=1&amp;publication_year=2013&amp;pages=11&amp;pmid=24280183&amp;doi=10.1186/1939-8433-6-11&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270096-bib-0005">
<cite>

Bar‐El Dadon, S.
, 
Abbo, S.
, &amp; 
Reifen, R.
 (2017). Leveraging traditional crops for better nutrition and health—The case of chickpea. Trends in Food Science &amp; Technology, 64, 39–47. 10.1016/j.tifs.2017.04.002
</cite> [<a href="https://doi.org/10.1016/j.tifs.2017.04.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Trends%20in%20Food%20Science%20&amp;%20Technology&amp;title=Leveraging%20traditional%20crops%20for%20better%20nutrition%20and%20health%E2%80%94The%20case%20of%20chickpea&amp;author=S.%20Bar%E2%80%90El%20Dadon&amp;author=S.%20Abbo&amp;author=R.%20Reifen&amp;volume=64&amp;publication_year=2017&amp;pages=39-47&amp;doi=10.1016/j.tifs.2017.04.002&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270096-bib-0066">
<cite>

Bastien, M.
, 
Sonah, H.
, &amp; 
Belzile, F.
 (2014). Genome Wide Association Mapping of <em>Sclerotinia sclerotiorum</em> Resistance in Soybean with a Genotyping‐by‐Sequencing Approach. The The Plant Genome, 7, plantgenome2013.10.0030. 10.3835/plantgenome2013.10.0030
</cite> [<a href="https://doi.org/10.3835/plantgenome2013.10.0030" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=The%20The%20Plant%20Genome&amp;title=Genome%20Wide%20Association%20Mapping%20of%20Sclerotinia%20sclerotiorum%20Resistance%20in%20Soybean%20with%20a%20Genotyping%E2%80%90by%E2%80%90Sequencing%20Approach&amp;author=M.%20Bastien&amp;author=H.%20Sonah&amp;author=F.%20Belzile&amp;volume=7&amp;publication_year=2014&amp;doi=10.3835/plantgenome2013.10.0030&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270096-bib-0067">
<cite>

Bevan, M. W.
, 
Uauy, C.
, 
Wulff, B. B.
, 
Zhou, J.
, 
Krasileva, K.
, &amp; 
Clark, M. D.
 (2017). Genomic innovation for crop improvement. Nature, 543(7645), 346–354. 10.1038/nature22011

</cite> [<a href="https://doi.org/10.1038/nature22011" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28300107/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Genomic%20innovation%20for%20crop%20improvement&amp;author=M.%20W.%20Bevan&amp;author=C.%20Uauy&amp;author=B.%20B.%20Wulff&amp;author=J.%20Zhou&amp;author=K.%20Krasileva&amp;volume=543&amp;issue=7645&amp;publication_year=2017&amp;pages=346-354&amp;pmid=28300107&amp;doi=10.1038/nature22011&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270096-bib-0006">
<cite>

Brunner, S. M.
, 
Dinglasan, E.
, 
Baraibar, S.
, 
Alahmad, S.
, 
Katsikis, C.
, 
van der Meer, S.
, 
Godoy, J.
, 
Moody, D.
, 
Smith, M.
, 
Hickey, L.
, &amp; 
Robinson, H.
 (2024). Characterizing stay‐green in barley across diverse environments: Unveiling novel haplotypes. Theoretical and Applied Genetics [Theoretische Und Angewandte Genetik], 137(6), Article 120. 10.1007/s00122-024-04612-1

</cite> [<a href="https://doi.org/10.1007/s00122-024-04612-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11074220/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38709310/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Theoretical%20and%20Applied%20Genetics%20%5BTheoretische%20Und%20Angewandte%20Genetik%5D&amp;title=Characterizing%20stay%E2%80%90green%20in%20barley%20across%20diverse%20environments:%20Unveiling%20novel%20haplotypes&amp;author=S.%20M.%20Brunner&amp;author=E.%20Dinglasan&amp;author=S.%20Baraibar&amp;author=S.%20Alahmad&amp;author=C.%20Katsikis&amp;volume=137&amp;issue=6&amp;publication_year=2024&amp;pages=120&amp;pmid=38709310&amp;doi=10.1007/s00122-024-04612-1&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270096-bib-0007">
<cite>

Butler, D.
, 
Cullis, B. R.
, 
Gilmour, A.
, &amp; 
Gogel, B. J.
 (2009). 
ASReml‐R reference manual
 (Technical Report). Queensland Department of Primary Industries. <a href="https://asreml.kb.vsni.co.uk/wp-content/uploads/sites/3/ASReml-R-3-Reference-Manual.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://asreml.kb.vsni.co.uk/wp-content/uploads/sites/3/ASReml-R-3-Reference-Manual.pdf</a>
</cite> [<a href="https://scholar.google.com/scholar_lookup?Butler,%20D.%20,%20Cullis,%20B.%20R.%20,%20Gilmour,%20A.%20,%20&amp;%20Gogel,%20B.%20J.%20(2009).%20ASReml%E2%80%90R%20reference%20manual%20(Technical%20Report).%20Queensland%20Department%20of%20Primary%20Industries.%20https://asreml.kb.vsni.co.uk/wp-content/uploads/sites/3/ASReml-R-3-Reference-Manual.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270096-bib-0058">
<cite>

Charrad, M.
, 
Ghazzali, N.
, 
Boiteau, V.
, &amp; 
Niknafs, A.
 (2014). NbClust: An R package for determining the relevant number of clusters in a data set. Journal of Statistical Software, 61(6), 1–36. 10.18637/jss.v061.i06
</cite> [<a href="https://doi.org/10.18637/jss.v061.i06" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Statistical%20Software&amp;title=NbClust:%20An%20R%20package%20for%20determining%20the%20relevant%20number%20of%20clusters%20in%20a%20data%20set&amp;author=M.%20Charrad&amp;author=N.%20Ghazzali&amp;author=V.%20Boiteau&amp;author=A.%20Niknafs&amp;volume=61&amp;issue=6&amp;publication_year=2014&amp;pages=1-36&amp;doi=10.18637/jss.v061.i06&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270096-bib-0056">
<cite>

Cuc, L. M.
, 
Mace, E. S.
, 
Crouch, J. H.
, 
Quang, V. D.
, 
Long, T. D.
, &amp; 
Varshney, R. K.
 (2008). Isolation and characterization of novel microsatellite markers and their application for diversity assessment in cultivated groundnut (<em>Arachis hypogaea</em>). BMC plant biology, 8(1), 55. 10.1186/1471-2229-8-55

</cite> [<a href="https://doi.org/10.1186/1471-2229-8-55" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2416452/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18482440/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20plant%20biology&amp;title=Isolation%20and%20characterization%20of%20novel%20microsatellite%20markers%20and%20their%20application%20for%20diversity%20assessment%20in%20cultivated%20groundnut%20(Arachis%20hypogaea)&amp;author=L.%20M.%20Cuc&amp;author=E.%20S.%20Mace&amp;author=J.%20H.%20Crouch&amp;author=V.%20D.%20Quang&amp;author=T.%20D.%20Long&amp;volume=8&amp;issue=1&amp;publication_year=2008&amp;pages=55&amp;pmid=18482440&amp;doi=10.1186/1471-2229-8-55&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270096-bib-0008">
<cite>

Danecek, P.
, 
Bonfield, J. K.
, 
Liddle, J.
, 
Marshall, J.
, 
Ohan, V.
, 
Pollard, M. O.
, 
Whitwham, A.
, 
Keane, T.
, 
Mccarthy, S. A.
, 
Davies, R. M.
, &amp; 
Li, H.
 (2021). Twelve years of SAMtools and BCFtools. GigaScience, 10(2), giab008. 10.1093/gigascience/giab008

</cite> [<a href="https://doi.org/10.1093/gigascience/giab008" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7931819/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33590861/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=GigaScience&amp;title=Twelve%20years%20of%20SAMtools%20and%20BCFtools&amp;author=P.%20Danecek&amp;author=J.%20K.%20Bonfield&amp;author=J.%20Liddle&amp;author=J.%20Marshall&amp;author=V.%20Ohan&amp;volume=10&amp;issue=2&amp;publication_year=2021&amp;pages=giab008&amp;pmid=33590861&amp;doi=10.1093/gigascience/giab008&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270096-bib-0055">
<cite>

Darvasi, A.
, &amp; 
Soller, M.
 (1995). Advanced intercross lines, an experimental population for fine genetic mapping. Genetics, 141(3), 1199–1207. 10.1093/genetics/141.3.1199

</cite> [<a href="https://doi.org/10.1093/genetics/141.3.1199" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1206841/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/8582624/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genetics&amp;title=Advanced%20intercross%20lines,%20an%20experimental%20population%20for%20fine%20genetic%20mapping&amp;author=A.%20Darvasi&amp;author=M.%20Soller&amp;volume=141&amp;issue=3&amp;publication_year=1995&amp;pages=1199-1207&amp;pmid=8582624&amp;doi=10.1093/genetics/141.3.1199&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270096-bib-0009">
<cite>

Depristo, M. A.
, 
Banks, E.
, 
Poplin, R.
, 
Garimella, K. V.
, 
Maguire, J. R.
, 
Hartl, C.
, 
Philippakis, A. A.
, 
Del Angel, G.
, 
Rivas, M. A.
, 
Hanna, M.
, 
Mckenna, A.
, 
Fennell, T. J.
, 
Kernytsky, A. M.
, 
Sivachenko, A. Y.
, 
Cibulskis, K.
, 
Gabriel, S. B.
, 
Altshuler, D.
, &amp; 
Daly, M. J.
 (2011). A framework for variation discovery and genotyping using next‐generation DNA sequencing data. Nature Genetics, 43, 491–498. 10.1038/ng.806

</cite> [<a href="https://doi.org/10.1038/ng.806" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3083463/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21478889/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Genetics&amp;title=A%20framework%20for%20variation%20discovery%20and%20genotyping%20using%20next%E2%80%90generation%20DNA%20sequencing%20data&amp;author=M.%20A.%20Depristo&amp;author=E.%20Banks&amp;author=R.%20Poplin&amp;author=K.%20V.%20Garimella&amp;author=J.%20R.%20Maguire&amp;volume=43&amp;publication_year=2011&amp;pages=491-498&amp;pmid=21478889&amp;doi=10.1038/ng.806&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270096-bib-0010">
<cite>

Diaz, S.
, 
Ariza‐Suarez, D.
, 
Izquierdo, P.
, 
Lobaton, J. D.
, 
De La Hoz, J. F.
, 
Acevedo, F.
, 
Duitama, J.
, 
Guerrero, A. F.
, 
Cajiao, C.
, 
Mayor, V.
, 
Beebe, S. E.
, &amp; 
Raatz, B.
 (2020). Genetic mapping for agronomic traits in a MAGIC population of common bean (<em>Phaseolus vulgaris</em> L.) under drought conditions. BMC Genomics, 21(1), Article 799. 10.1186/s12864-020-07213-6

</cite> [<a href="https://doi.org/10.1186/s12864-020-07213-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7670608/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33198642/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Genomics&amp;title=Genetic%20mapping%20for%20agronomic%20traits%20in%20a%20MAGIC%20population%20of%20common%20bean%20(Phaseolus%20vulgaris%20L.)%20under%20drought%20conditions&amp;author=S.%20Diaz&amp;author=D.%20Ariza%E2%80%90Suarez&amp;author=P.%20Izquierdo&amp;author=J.%20D.%20Lobaton&amp;author=J.%20F.%20De%20La%20Hoz&amp;volume=21&amp;issue=1&amp;publication_year=2020&amp;pages=799&amp;pmid=33198642&amp;doi=10.1186/s12864-020-07213-6&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270096-bib-0011">
<cite>

Gao, X.
, 
Starmer, J.
, &amp; 
Martin, E. R.
 (2008). A multiple testing correction method for genetic association studies using correlated single nucleotide polymorphisms. Genetic Epidemiology, 32(4), 361–369. 10.1002/gepi.20310

</cite> [<a href="https://doi.org/10.1002/gepi.20310" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18271029/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genetic%20Epidemiology&amp;title=A%20multiple%20testing%20correction%20method%20for%20genetic%20association%20studies%20using%20correlated%20single%20nucleotide%20polymorphisms&amp;author=X.%20Gao&amp;author=J.%20Starmer&amp;author=E.%20R.%20Martin&amp;volume=32&amp;issue=4&amp;publication_year=2008&amp;pages=361-369&amp;pmid=18271029&amp;doi=10.1002/gepi.20310&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270096-bib-0069">
<cite>

Glaszmann, J. C.
, 
Kilian, B.
, 
Upadhyaya, H. D.
, &amp; 
Varshney, R. K.
 (2010). Accessing genetic diversity for crop improvement. Current opinion in plant biology, 13(2), 167–173. 10.1016/j.pbi.2010.01.004

</cite> [<a href="https://doi.org/10.1016/j.pbi.2010.01.004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20167531/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Current%20opinion%20in%20plant%20biology&amp;title=Accessing%20genetic%20diversity%20for%20crop%20improvement&amp;author=J.%20C.%20Glaszmann&amp;author=B.%20Kilian&amp;author=H.%20D.%20Upadhyaya&amp;author=R.%20K.%20Varshney&amp;volume=13&amp;issue=2&amp;publication_year=2010&amp;pages=167-173&amp;pmid=20167531&amp;doi=10.1016/j.pbi.2010.01.004&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270096-bib-0012">
<cite>

Gupta, S.
, 
Kumar, T.
, 
Verma, S.
, 
Bharadwaj, C.
, &amp; 
Bhatia, S.
 (2015). Development of gene‐based markers for use in construction of the chickpea (<em>Cicer arietinum</em> L.) genetic linkage map and identification of QTLs associated with seed weight and plant height. Molecular Biology Reports, 42(11), 1571–1580. 10.1007/s11033-015-3925-3

</cite> [<a href="https://doi.org/10.1007/s11033-015-3925-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26446030/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Biology%20Reports&amp;title=Development%20of%20gene%E2%80%90based%20markers%20for%20use%20in%20construction%20of%20the%20chickpea%20(Cicer%20arietinum%20L.)%20genetic%20linkage%20map%20and%20identification%20of%20QTLs%20associated%20with%20seed%20weight%20and%20plant%20height&amp;author=S.%20Gupta&amp;author=T.%20Kumar&amp;author=S.%20Verma&amp;author=C.%20Bharadwaj&amp;author=S.%20Bhatia&amp;volume=42&amp;issue=11&amp;publication_year=2015&amp;pages=1571-1580&amp;pmid=26446030&amp;doi=10.1007/s11033-015-3925-3&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270096-bib-0013">
<cite>

Han, Z.
, 
Hu, G.
, 
Liu, H.
, 
Liang, F.
, 
Yang, L.
, 
Zhao, H.
, 
Zhang, Q.
, 
Li, Z.
, 
Zhang, Q.
, &amp; 
Xing, Y.
 (2020). Bin‐based genome‐wide association analyses improve power and resolution in QTL mapping and identify favorable alleles from multiple parents in a four‐way MAGIC rice population. Theoretical and Applied Genetics [Theoretische Und Angewandte Genetik], 133(1), 59–71. 10.1007/s00122-019-03440-y

</cite> [<a href="https://doi.org/10.1007/s00122-019-03440-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31549182/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Theoretical%20and%20Applied%20Genetics%20%5BTheoretische%20Und%20Angewandte%20Genetik%5D&amp;title=Bin%E2%80%90based%20genome%E2%80%90wide%20association%20analyses%20improve%20power%20and%20resolution%20in%20QTL%20mapping%20and%20identify%20favorable%20alleles%20from%20multiple%20parents%20in%20a%20four%E2%80%90way%20MAGIC%20rice%20population&amp;author=Z.%20Han&amp;author=G.%20Hu&amp;author=H.%20Liu&amp;author=F.%20Liang&amp;author=L.%20Yang&amp;volume=133&amp;issue=1&amp;publication_year=2020&amp;pages=59-71&amp;pmid=31549182&amp;doi=10.1007/s00122-019-03440-y&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270096-bib-0068">
<cite>

Hao, C.
, 
Wang, Y.
, 
Hou, J.
, 
Feuillet, C.
, 
Balfourier, F.
, &amp; 
Zhang, X.
 (2012). Association mapping and haplotype analysis of a 3.1‐Mb genomic region involved in Fusarium head blight resistance on wheat chromosome 3BS. 10.1371/journal.pone.0046444
</cite> [<a href="https://doi.org/10.1371/journal.pone.0046444" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3465345/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23071572/" class="usa-link">PubMed</a>]</li>
<li id="tpg270096-bib-0014">
<cite>

Hashemi, S. M.
, 
Perry, G.
, 
Rajcan, I.
, &amp; 
Eskandari, M.
 (2022). SoyMAGIC: An unprecedented platform for genetic studies and breeding activities in soybean. Frontiers in Plant Science, 13, 945471. 10.3389/fpls.2022.945471

</cite> [<a href="https://doi.org/10.3389/fpls.2022.945471" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9301248/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35874009/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Plant%20Science&amp;title=SoyMAGIC:%20An%20unprecedented%20platform%20for%20genetic%20studies%20and%20breeding%20activities%20in%20soybean&amp;author=S.%20M.%20Hashemi&amp;author=G.%20Perry&amp;author=I.%20Rajcan&amp;author=M.%20Eskandari&amp;volume=13&amp;publication_year=2022&amp;pages=945471&amp;pmid=35874009&amp;doi=10.3389/fpls.2022.945471&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270096-bib-0015">
<cite>

Hemshrot, A.
, 
Poets, A. M.
, 
Tyagi, P.
, 
Lei, L.
, 
Carter, C. K.
, 
Hirsch, C. N.
, 
Li, L.
, 
Brown‐Guedira, G.
, 
Morrell, P. L.
, 
Muehlbauer, G. J.
, &amp; 
Smith, K. P.
 (2019). Development of a multiparent population for genetic mapping and allele discovery in six‐row barley. Genetics, 213(2), 595–613. 10.1534/genetics.119.302046

</cite> [<a href="https://doi.org/10.1534/genetics.119.302046" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6781892/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31358533/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genetics&amp;title=Development%20of%20a%20multiparent%20population%20for%20genetic%20mapping%20and%20allele%20discovery%20in%20six%E2%80%90row%20barley&amp;author=A.%20Hemshrot&amp;author=A.%20M.%20Poets&amp;author=P.%20Tyagi&amp;author=L.%20Lei&amp;author=C.%20K.%20Carter&amp;volume=213&amp;issue=2&amp;publication_year=2019&amp;pages=595-613&amp;pmid=31358533&amp;doi=10.1534/genetics.119.302046&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270096-bib-0016">
<cite>

Holland, J. B.
 (2015). MAGIC maize: A new resource for plant genetics. Genome Biology, 16, Article 163. 10.1186/s13059-015-0713-2

</cite> [<a href="https://doi.org/10.1186/s13059-015-0713-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4566361/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26357905/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genome%20Biology&amp;title=MAGIC%20maize:%20A%20new%20resource%20for%20plant%20genetics&amp;author=J.%20B.%20Holland&amp;volume=16&amp;publication_year=2015&amp;pages=163&amp;pmid=26357905&amp;doi=10.1186/s13059-015-0713-2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270096-bib-0017">
<cite>

Howie, B.
, 
Fuchsberger, C.
, 
Stephens, M.
, 
Marchini, J.
, &amp; 
Abecasis, G. R.
 (2012). Fast and accurate genotype imputation in genome‐wide association studies through pre‐phasing. Nature Genetics, 44(8), 955–959. 10.1038/ng.2354

</cite> [<a href="https://doi.org/10.1038/ng.2354" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3696580/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22820512/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Genetics&amp;title=Fast%20and%20accurate%20genotype%20imputation%20in%20genome%E2%80%90wide%20association%20studies%20through%20pre%E2%80%90phasing&amp;author=B.%20Howie&amp;author=C.%20Fuchsberger&amp;author=M.%20Stephens&amp;author=J.%20Marchini&amp;author=G.%20R.%20Abecasis&amp;volume=44&amp;issue=8&amp;publication_year=2012&amp;pages=955-959&amp;pmid=22820512&amp;doi=10.1038/ng.2354&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270096-bib-0018">
<cite>

Huang, B. E.
, 
George, A. W.
, 
Forrest, K. L.
, 
Kilian, A.
, 
Hayden, M. J.
, 
Morell, M. K.
, &amp; 
Cavanagh, C. R.
 (2012). A multiparent advanced generation inter‐cross population for genetic analysis in wheat. Plant Biotechnology Journal, 10(7), 826–839. 10.1111/j.1467-7652.2012.00702.x

</cite> [<a href="https://doi.org/10.1111/j.1467-7652.2012.00702.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22594629/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Biotechnology%20Journal&amp;title=A%20multiparent%20advanced%20generation%20inter%E2%80%90cross%20population%20for%20genetic%20analysis%20in%20wheat&amp;author=B.%20E.%20Huang&amp;author=A.%20W.%20George&amp;author=K.%20L.%20Forrest&amp;author=A.%20Kilian&amp;author=M.%20J.%20Hayden&amp;volume=10&amp;issue=7&amp;publication_year=2012&amp;pages=826-839&amp;pmid=22594629&amp;doi=10.1111/j.1467-7652.2012.00702.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270096-bib-0019">
<cite>

Huynh, B.‐L.
, 
Ehlers, J. D.
, 
Huang, B. E.
, 
Muñoz‐Amatriaín, M.
, 
Lonardi, S.
, 
Santos, J. R. P.
, 
Ndeve, A.
, 
Batieno, B. J.
, 
Boukar, O.
, 
Cisse, N.
, 
Drabo, I.
, 
Fatokun, C.
, 
Kusi, F.
, 
Agyare, R. Y.
, 
Guo, Y.‐N.
, 
Herniter, I.
, 
Lo, S.
, 
Wanamaker, S. I.
, 
Xu, S.
, … 
Roberts, P. A.
 (2018). A multi‐parent advanced generation inter‐cross (MAGIC) population for genetic analysis and improvement of cowpea (<em>Vigna unguiculata</em> L. Walp.). Plant Journal, 93(6), 1129–1142. 10.1111/tpj.13827
</cite> [<a href="https://doi.org/10.1111/tpj.13827" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29356213/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Journal&amp;title=A%20multi%E2%80%90parent%20advanced%20generation%20inter%E2%80%90cross%20(MAGIC)%20population%20for%20genetic%20analysis%20and%20improvement%20of%20cowpea%20(Vigna%20unguiculata%20L.%20Walp.)&amp;author=B.%E2%80%90L.%20Huynh&amp;author=J.%20D.%20Ehlers&amp;author=B.%20E.%20Huang&amp;author=M.%20Mu%C3%B1oz%E2%80%90Amatria%C3%ADn&amp;author=S.%20Lonardi&amp;volume=93&amp;issue=6&amp;publication_year=2018&amp;pages=1129-1142&amp;pmid=29356213&amp;doi=10.1111/tpj.13827&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270096-bib-0020">
<cite>

Islam, M. S.
, 
Thyssen, G. N.
, 
Jenkins, J. N.
, 
Zeng, L.
, 
Delhom, C. D.
, 
Mccarty, J. C.
, 
Deng, D. D.
, 
Hinchliffe, D. J.
, 
Jones, D. C.
, &amp; 
Fang, D. D.
 (2016). A MAGIC population‐based genome‐wide association study reveals functional association of <em>GhRBB1_A07</em> gene with superior fiber quality in cotton. BMC Genomics, 17(1), Article 903. 10.1186/s12864-016-3249-2

</cite> [<a href="https://doi.org/10.1186/s12864-016-3249-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5103610/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27829353/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Genomics&amp;title=A%20MAGIC%20population%E2%80%90based%20genome%E2%80%90wide%20association%20study%20reveals%20functional%20association%20of%20GhRBB1_A07%20gene%20with%20superior%20fiber%20quality%20in%20cotton&amp;author=M.%20S.%20Islam&amp;author=G.%20N.%20Thyssen&amp;author=J.%20N.%20Jenkins&amp;author=L.%20Zeng&amp;author=C.%20D.%20Delhom&amp;volume=17&amp;issue=1&amp;publication_year=2016&amp;pages=903&amp;pmid=27829353&amp;doi=10.1186/s12864-016-3249-2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270096-bib-0021">
<cite>

Jaganathan, D.
, 
Thudi, M.
, 
Kale, S.
, 
Azam, S.
, 
Roorkiwal, M.
, 
Gaur, P. M.
, 
Kishor, P. B. K.
, 
Nguyen, H.
, 
Sutton, T.
, &amp; 
Varshney, R. K.
 (2015). Genotyping‐by‐sequencing based intra‐specific genetic map refines a “QTL‐hotspot” region for drought tolerance in chickpea. Molecular Genetics and Genomics, 290(2), 559–571. 10.1007/s00438-014-0932-3

</cite> [<a href="https://doi.org/10.1007/s00438-014-0932-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4361754/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25344290/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Genetics%20and%20Genomics&amp;title=Genotyping%E2%80%90by%E2%80%90sequencing%20based%20intra%E2%80%90specific%20genetic%20map%20refines%20a%20%E2%80%9CQTL%E2%80%90hotspot%E2%80%9D%20region%20for%20drought%20tolerance%20in%20chickpea&amp;author=D.%20Jaganathan&amp;author=M.%20Thudi&amp;author=S.%20Kale&amp;author=S.%20Azam&amp;author=M.%20Roorkiwal&amp;volume=290&amp;issue=2&amp;publication_year=2015&amp;pages=559-571&amp;pmid=25344290&amp;doi=10.1007/s00438-014-0932-3&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270096-bib-0022">
<cite>

Jha, U. C.
, 
Nayyar, H.
, 
Palakurthi, R.
, 
Jha, R.
, 
Valluri, V.
, 
Bajaj, P.
, 
Chitikineni, A.
, 
Singh, N. P.
, 
Varshney, R. K.
, &amp; 
Thudi, M.
 (2021). Major QTLs and potential candidate genes for heat stress tolerance identified in chickpea (<em>Cicer arietinum</em> L.). Frontiers in Plant Science, 12, Article 655103. 10.3389/fpls.2021.655103

</cite> [<a href="https://doi.org/10.3389/fpls.2021.655103" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8350164/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34381469/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Plant%20Science&amp;title=Major%20QTLs%20and%20potential%20candidate%20genes%20for%20heat%20stress%20tolerance%20identified%20in%20chickpea%20(Cicer%20arietinum%20L.)&amp;author=U.%20C.%20Jha&amp;author=H.%20Nayyar&amp;author=R.%20Palakurthi&amp;author=R.%20Jha&amp;author=V.%20Valluri&amp;volume=12&amp;publication_year=2021&amp;pages=655103&amp;pmid=34381469&amp;doi=10.3389/fpls.2021.655103&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270096-bib-0023">
<cite>

Kale, S. M.
, 
Jaganathan, D.
, 
Ruperao, P.
, 
Chen, C.
, 
Punna, R.
, 
Kudapa, H.
, 
Thudi, M.
, 
Roorkiwal, M.
, 
Katta, M. A.
, 
Doddamani, D.
, 
Garg, V.
, 
Kishor, P. B. K.
, 
Gaur, P. M.
, 
Nguyen, H. T.
, 
Batley, J.
, 
Edwards, D.
, 
Sutton, T.
, &amp; 
Varshney, R. K.
 (2015). Prioritization of candidate genes in “QTL‐hotspot” region for drought tolerance in chickpea (<em>Cicer arietinum</em> L.). Scientific Reports, 5, Article 15296. 10.1038/srep15296

</cite> [<a href="https://doi.org/10.1038/srep15296" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4609953/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26478518/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Scientific%20Reports&amp;title=Prioritization%20of%20candidate%20genes%20in%20%E2%80%9CQTL%E2%80%90hotspot%E2%80%9D%20region%20for%20drought%20tolerance%20in%20chickpea%20(Cicer%20arietinum%20L.)&amp;author=S.%20M.%20Kale&amp;author=D.%20Jaganathan&amp;author=P.%20Ruperao&amp;author=C.%20Chen&amp;author=R.%20Punna&amp;volume=5&amp;publication_year=2015&amp;pages=15296&amp;pmid=26478518&amp;doi=10.1038/srep15296&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270096-bib-0024">
<cite>

Kujur, A.
, 
Upadhyaya, H. D.
, 
Bajaj, D.
, 
Gowda, C. L. L.
, 
Sharma, S.
, 
Tyagi, A. K.
, &amp; 
Parida, S. K.
 (2016). Identification of candidate genes and natural allelic variants for QTLs governing plant height in chickpea. Scientific Reports, 6, Article 27968. 10.1038/srep27968

</cite> [<a href="https://doi.org/10.1038/srep27968" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4913251/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27319304/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Scientific%20Reports&amp;title=Identification%20of%20candidate%20genes%20and%20natural%20allelic%20variants%20for%20QTLs%20governing%20plant%20height%20in%20chickpea&amp;author=A.%20Kujur&amp;author=H.%20D.%20Upadhyaya&amp;author=D.%20Bajaj&amp;author=C.%20L.%20L.%20Gowda&amp;author=S.%20Sharma&amp;volume=6&amp;publication_year=2016&amp;pages=27968&amp;pmid=27319304&amp;doi=10.1038/srep27968&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270096-bib-0025">
<cite>

Ladejobi, O.
, 
Elderfield, J.
, 
Gardner, K. A.
, 
Gaynor, R. C.
, 
Hickey, J.
, 
Hibberd, J. M.
, 
Mackay, I. J.
, &amp; 
Bentley, A. R.
 (2016). Maximizing the potential of multi‐parental crop populations. Applied &amp; Translational Genomics, 11, 9–17. 10.1016/j.atg.2016.10.002

</cite> [<a href="https://doi.org/10.1016/j.atg.2016.10.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5167364/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28018845/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Applied%20&amp;%20Translational%20Genomics&amp;title=Maximizing%20the%20potential%20of%20multi%E2%80%90parental%20crop%20populations&amp;author=O.%20Ladejobi&amp;author=J.%20Elderfield&amp;author=K.%20A.%20Gardner&amp;author=R.%20C.%20Gaynor&amp;author=J.%20Hickey&amp;volume=11&amp;publication_year=2016&amp;pages=9-17&amp;pmid=28018845&amp;doi=10.1016/j.atg.2016.10.002&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270096-bib-0064">
<cite>

Leigh, J. W.
, 
Bryant, D.
, &amp; 
Nakagawa, S.
 (2015). POPART: Full‐feature software for haplotype network construction. Methods in Ecology &amp; Evolution, 6(9), 1110–1116. 10.1111/2041-210X.12410
</cite> [<a href="https://doi.org/10.1111/2041-210X.12410" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Methods%20in%20Ecology%20&amp;%20Evolution&amp;title=POPART:%20Full%E2%80%90feature%20software%20for%20haplotype%20network%20construction&amp;author=J.%20W.%20Leigh&amp;author=D.%20Bryant&amp;author=S.%20Nakagawa&amp;volume=6&amp;issue=9&amp;publication_year=2015&amp;pages=1110-1116&amp;doi=10.1111/2041-210X.12410&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270096-bib-0027">
<cite>

Li, H.
, 
Handsaker, B.
, 
Wysoker, A.
, 
Fennell, T.
, 
Ruan, J.
, 
Homer, N.
, 
Marth, G.
, 
Abecasis, G.
, 
Durbin, R.
, &amp; 1000 Genome Project Data Processing Subgroup
. (2009). The sequence alignment/map (SAM) format and SAMtools. Bioinformatics, 25, 2078–2079. 10.1093/bioinformatics/btp352

</cite> [<a href="https://doi.org/10.1093/bioinformatics/btp352" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2723002/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19505943/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=The%20sequence%20alignment/map%20(SAM)%20format%20and%20SAMtools&amp;author=H.%20Li&amp;author=B.%20Handsaker&amp;author=A.%20Wysoker&amp;author=T.%20Fennell&amp;author=J.%20Ruan&amp;volume=25&amp;publication_year=2009&amp;pages=2078-2079&amp;pmid=19505943&amp;doi=10.1093/bioinformatics/btp352&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270096-bib-0028">
<cite>

Li, H.
, 
Rasheed, A.
, 
Hickey, L. T.
, &amp; 
He, Z.
 (2018). Fast‐forwarding genetic gain. Trends in Plant Science, 23(3), 184–186. 10.1016/j.tplants.2018.01.007

</cite> [<a href="https://doi.org/10.1016/j.tplants.2018.01.007" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29426713/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Trends%20in%20Plant%20Science&amp;title=Fast%E2%80%90forwarding%20genetic%20gain&amp;author=H.%20Li&amp;author=A.%20Rasheed&amp;author=L.%20T.%20Hickey&amp;author=Z.%20He&amp;volume=23&amp;issue=3&amp;publication_year=2018&amp;pages=184-186&amp;pmid=29426713&amp;doi=10.1016/j.tplants.2018.01.007&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270096-bib-0029">
<cite>

Li, Y.
, 
Ruperao, P.
, 
Batley, J.
, 
Edwards, D.
, 
Davidson, J.
, 
Hobson, K.
, &amp; 
Sutton, T.
 (2017). Genome analysis identified novel candidate genes for ascochyta blight resistance in chickpea using whole genome re‐sequencing data. Frontiers in Plant Science, 8, Article 359. 10.3389/fpls.2017.00359

</cite> [<a href="https://doi.org/10.3389/fpls.2017.00359" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5355423/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28367154/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Plant%20Science&amp;title=Genome%20analysis%20identified%20novel%20candidate%20genes%20for%20ascochyta%20blight%20resistance%20in%20chickpea%20using%20whole%20genome%20re%E2%80%90sequencing%20data&amp;author=Y.%20Li&amp;author=P.%20Ruperao&amp;author=J.%20Batley&amp;author=D.%20Edwards&amp;author=J.%20Davidson&amp;volume=8&amp;publication_year=2017&amp;pages=359&amp;pmid=28367154&amp;doi=10.3389/fpls.2017.00359&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270096-bib-0030">
<cite>

Li, Y.
, 
Ruperao, P.
, 
Batley, J.
, 
Edwards, D.
, 
Khan, T.
, 
Colmer, T. D.
, 
Pang, J.
, 
Siddique, K. H. M.
, &amp; 
Sutton, T.
 (2018). Investigating drought tolerance in chickpea using genome‐wide association mapping and genomic selection based on whole‐genome resequencing data. Frontiers in Plant Science, 9, Article 190. 10.3389/fpls.2018.00190

</cite> [<a href="https://doi.org/10.3389/fpls.2018.00190" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5825913/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29515606/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Plant%20Science&amp;title=Investigating%20drought%20tolerance%20in%20chickpea%20using%20genome%E2%80%90wide%20association%20mapping%20and%20genomic%20selection%20based%20on%20whole%E2%80%90genome%20resequencing%20data&amp;author=Y.%20Li&amp;author=P.%20Ruperao&amp;author=J.%20Batley&amp;author=D.%20Edwards&amp;author=T.%20Khan&amp;volume=9&amp;publication_year=2018&amp;pages=190&amp;pmid=29515606&amp;doi=10.3389/fpls.2018.00190&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270096-bib-0061">
<cite>

Liu, X.
, 
Huang, M.
, 
Fan, B.
, 
Buckler, E. S.
, &amp; 
Zhang, Z.
 (2016). Iterative usage of fixed and random effect models for powerful and efficient genome‐wide association studies. PLoS genetics, 12(2), e1005767. 10.1371/journal.pgen.1005767

</cite> [<a href="https://doi.org/10.1371/journal.pgen.1005767" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4734661/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26828793/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20genetics&amp;title=Iterative%20usage%20of%20fixed%20and%20random%20effect%20models%20for%20powerful%20and%20efficient%20genome%E2%80%90wide%20association%20studies&amp;author=X.%20Liu&amp;author=M.%20Huang&amp;author=B.%20Fan&amp;author=E.%20S.%20Buckler&amp;author=Z.%20Zhang&amp;volume=12&amp;issue=2&amp;publication_year=2016&amp;pages=e1005767&amp;pmid=26828793&amp;doi=10.1371/journal.pgen.1005767&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270096-bib-0031">
<cite>

Loh, P.‐R.
, 
Danecek, P.
, 
Palamara, P. F.
, 
Fuchsberger, C.
, 
Reshef, Y. A.
, 
K Finucane, H.
, 
Schoenherr, S.
, 
Forer, L.
, 
Mccarthy, S.
, 
Abecasis, G. R.
, 
Durbin, R.
, &amp; 
Price, A. L.
 (2016). Reference‐based phasing using the Haplotype Reference Consortium panel. Nature Genetics, 48(11), 1443–1448. 10.1038/ng.3679

</cite> [<a href="https://doi.org/10.1038/ng.3679" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5096458/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27694958/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Genetics&amp;title=Reference%E2%80%90based%20phasing%20using%20the%20Haplotype%20Reference%20Consortium%20panel&amp;author=P.%E2%80%90R.%20Loh&amp;author=P.%20Danecek&amp;author=P.%20F.%20Palamara&amp;author=C.%20Fuchsberger&amp;author=Y.%20A.%20Reshef&amp;volume=48&amp;issue=11&amp;publication_year=2016&amp;pages=1443-1448&amp;pmid=27694958&amp;doi=10.1038/ng.3679&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270096-bib-0032">
<cite>

Nguyen, D. T.
, 
Hayes, J. E.
, 
Atieno, J.
, 
Li, Y.
, 
Baumann, U.
, 
Pattison, A.
, 
Bramley, H.
, 
Hobson, K.
, 
Roorkiwal, M.
, 
Varshney, R. K.
, 
Colmer, T. D.
, &amp; 
Sutton, T.
 (2022). The genetics of vigour‐related traits in chickpea (<em>Cicer arietinum</em> L.): Insights from genomic data. Theoretical and Applied Genetics, 135(1), 107–124. 10.1007/s00122-021-03954-4

</cite> [<a href="https://doi.org/10.1007/s00122-021-03954-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34643761/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Theoretical%20and%20Applied%20Genetics&amp;title=The%20genetics%20of%20vigour%E2%80%90related%20traits%20in%20chickpea%20(Cicer%20arietinum%20L.):%20Insights%20from%20genomic%20data&amp;author=D.%20T.%20Nguyen&amp;author=J.%20E.%20Hayes&amp;author=J.%20Atieno&amp;author=Y.%20Li&amp;author=U.%20Baumann&amp;volume=135&amp;issue=1&amp;publication_year=2022&amp;pages=107-124&amp;pmid=34643761&amp;doi=10.1007/s00122-021-03954-4&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270096-bib-0033">
<cite>

Paul, P. J.
, 
Samineni, S.
, 
Thudi, M.
, 
Sajja, S. B.
, 
Rathore, A.
, 
Das, R. R.
, 
Khan, A. W.
, 
Chaturvedi, S. K.
, 
Lavanya, G. R.
, 
Varshney, R. K.
, &amp; 
Gaur, P. M.
 (2018). Molecular mapping of QTLs for heat tolerance in chickpea. International Journal of Molecular Sciences, 19(8), 2166. 10.3390/ijms19082166

</cite> [<a href="https://doi.org/10.3390/ijms19082166" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6121679/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30044369/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=International%20Journal%20of%20Molecular%20Sciences&amp;title=Molecular%20mapping%20of%20QTLs%20for%20heat%20tolerance%20in%20chickpea&amp;author=P.%20J.%20Paul&amp;author=S.%20Samineni&amp;author=M.%20Thudi&amp;author=S.%20B.%20Sajja&amp;author=A.%20Rathore&amp;volume=19&amp;issue=8&amp;publication_year=2018&amp;pages=2166&amp;pmid=30044369&amp;doi=10.3390/ijms19082166&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270096-bib-0034">
<cite>

Purcell, S.
, 
Neale, B.
, 
Todd‐Brown, K.
, 
Thomas, L.
, 
Ferreira, M. A. R.
, 
Bender, D.
, 
Maller, J.
, 
Sklar, P.
, 
de Bakker, P. I. W.
, 
Daly, M. J.
, &amp; 
Sham, P. C
 (2007). PLINK: A tool set for whole‐genome association and population‐based linkage analyses. American Journal of Human Genetics, 81(3), 559–575. 10.1086/519795

</cite> [<a href="https://doi.org/10.1086/519795" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1950838/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17701901/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=American%20Journal%20of%20Human%20Genetics&amp;title=PLINK:%20A%20tool%20set%20for%20whole%E2%80%90genome%20association%20and%20population%E2%80%90based%20linkage%20analyses&amp;author=S.%20Purcell&amp;author=B.%20Neale&amp;author=K.%20Todd%E2%80%90Brown&amp;author=L.%20Thomas&amp;author=M.%20A.%20R.%20Ferreira&amp;volume=81&amp;issue=3&amp;publication_year=2007&amp;pages=559-575&amp;pmid=17701901&amp;doi=10.1086/519795&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270096-bib-0035">
<cite>

Pushpavalli, R.
, 
Krishnamurthy, L.
, 
Thudi, M.
, 
Gaur, P. M.
, 
Rao, M. V.
, 
Siddique, K. H.
, 
Colmer, T. D.
, 
Turner, N. C.
, 
Varshney, R. K.
, &amp; 
Vadez, V.
 (2015). Two key genomic regions harbour QTLs for salinity tolerance in ICCV 2 × JG 11 derived chickpea (<em>Cicer arietinum</em> L.) recombinant inbred lines. BMC Plant Biology, 15, Article 124. 10.1186/s12870-015-0491-8

</cite> [<a href="https://doi.org/10.1186/s12870-015-0491-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4440540/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25994494/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Plant%20Biology&amp;title=Two%20key%20genomic%20regions%20harbour%20QTLs%20for%20salinity%20tolerance%20in%20ICCV%202%E2%80%89%C3%97%E2%80%89JG%2011%20derived%20chickpea%20(Cicer%20arietinum%20L.)%20recombinant%20inbred%20lines&amp;author=R.%20Pushpavalli&amp;author=L.%20Krishnamurthy&amp;author=M.%20Thudi&amp;author=P.%20M.%20Gaur&amp;author=M.%20V.%20Rao&amp;volume=15&amp;publication_year=2015&amp;pages=124&amp;pmid=25994494&amp;doi=10.1186/s12870-015-0491-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270096-bib-0060">
<cite>
R Core Team
. (2021). 
R: A language and environment for statistical computing
. R Foundation for Statistical Computing. <a href="https://www.R-project.org/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.R‐project.org/</a>
</cite> [<a href="https://scholar.google.com/scholar_lookup?R%20Core%20Team%20.%20(2021).%20R:%20A%20language%20and%20environment%20for%20statistical%20computing%20.%20R%20Foundation%20for%20Statistical%20Computing.%20https://www.R%E2%80%90project.org/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270096-bib-0036">
<cite>

Raman, R.
, 
Warren, A.
, 
Krysinska‐Kaczmarek, M.
, 
Rohan, M.
, 
Sharma, N.
, 
Dron, N.
, 
Davidson, J.
, 
Moore, K.
, &amp; 
Hobson, K.
 (2022). Genome‐wide association analyses track genomic regions for resistance to <em>Ascochyta rabiei</em> in Australian chickpea breeding germplasm. Frontiers in Plant Science, 18(13), Article 877266. 10.3389/fpls.2022.877266
</cite> [<a href="https://doi.org/10.3389/fpls.2022.877266" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9159299/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35665159/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Plant%20Science&amp;title=Genome%E2%80%90wide%20association%20analyses%20track%20genomic%20regions%20for%20resistance%20to%20Ascochyta%20rabiei%20in%20Australian%20chickpea%20breeding%20germplasm&amp;author=R.%20Raman&amp;author=A.%20Warren&amp;author=M.%20Krysinska%E2%80%90Kaczmarek&amp;author=M.%20Rohan&amp;author=N.%20Sharma&amp;volume=18&amp;issue=13&amp;publication_year=2022&amp;pages=877266&amp;pmid=35665159&amp;doi=10.3389/fpls.2022.877266&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270096-bib-0037">
<cite>

Ravelombola, W.
, 
Shi, A.
, &amp; 
Huynh, B.‐L.
 (2021). Loci discovery, network‐guided approach, and genomic prediction for drought tolerance index in a multi‐parent advanced generation intercross (MAGIC) cowpea population. Horticulture Research, 8(1), 24. 10.1038/s41438-021-00462-w

</cite> [<a href="https://doi.org/10.1038/s41438-021-00462-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7848001/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33518704/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Horticulture%20Research&amp;title=Loci%20discovery,%20network%E2%80%90guided%20approach,%20and%20genomic%20prediction%20for%20drought%20tolerance%20index%20in%20a%20multi%E2%80%90parent%20advanced%20generation%20intercross%20(MAGIC)%20cowpea%20population&amp;author=W.%20Ravelombola&amp;author=A.%20Shi&amp;author=B.%E2%80%90L.%20Huynh&amp;volume=8&amp;issue=1&amp;publication_year=2021&amp;pages=24&amp;pmid=33518704&amp;doi=10.1038/s41438-021-00462-w&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270096-bib-0038">
<cite>

Ren, W.
, 
Gong, X.
, 
Li, K.
, 
Zhang, H.
, 
Chen, F.
, &amp; 
Pan, Q.
 (2020). Recombination pattern characterization via simulation using different maize populations. International Journal of Molecular Sciences, 21(6), 2222. 10.3390/ijms21062222

</cite> [<a href="https://doi.org/10.3390/ijms21062222" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7139635/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32210156/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=International%20Journal%20of%20Molecular%20Sciences&amp;title=Recombination%20pattern%20characterization%20via%20simulation%20using%20different%20maize%20populations&amp;author=W.%20Ren&amp;author=X.%20Gong&amp;author=K.%20Li&amp;author=H.%20Zhang&amp;author=F.%20Chen&amp;volume=21&amp;issue=6&amp;publication_year=2020&amp;pages=2222&amp;pmid=32210156&amp;doi=10.3390/ijms21062222&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270096-bib-0039">
<cite>

Rockman, M. V.
, &amp; 
Kruglyak, L.
 (2008). Breeding designs for recombinant inbred advanced intercross lines. Genetics, 179(2), 1069–1078. 10.1534/genetics.107.083873

</cite> [<a href="https://doi.org/10.1534/genetics.107.083873" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2429860/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18505881/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genetics&amp;title=Breeding%20designs%20for%20recombinant%20inbred%20advanced%20intercross%20lines&amp;author=M.%20V.%20Rockman&amp;author=L.%20Kruglyak&amp;volume=179&amp;issue=2&amp;publication_year=2008&amp;pages=1069-1078&amp;pmid=18505881&amp;doi=10.1534/genetics.107.083873&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270096-bib-0040">
<cite>

Roncallo, P. F.
, 
Larsen, A. O.
, 
Achilli, A. L.
, 
Pierre, C. S.
, 
Gallo, C. A.
, 
Dreisigacker, S.
, &amp; 
Echenique, V.
 (2021). Linkage disequilibrium patterns, population structure and diversity analysis in a worldwide durum wheat collection including Argentinian genotypes. BMC Genomics, 22(1), Article 233. 10.1186/s12864-021-07519-z

</cite> [<a href="https://doi.org/10.1186/s12864-021-07519-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8022437/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33820546/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Genomics&amp;title=Linkage%20disequilibrium%20patterns,%20population%20structure%20and%20diversity%20analysis%20in%20a%20worldwide%20durum%20wheat%20collection%20including%20Argentinian%20genotypes&amp;author=P.%20F.%20Roncallo&amp;author=A.%20O.%20Larsen&amp;author=A.%20L.%20Achilli&amp;author=C.%20S.%20Pierre&amp;author=C.%20A.%20Gallo&amp;volume=22&amp;issue=1&amp;publication_year=2021&amp;pages=233&amp;pmid=33820546&amp;doi=10.1186/s12864-021-07519-z&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270096-bib-0041">
<cite>

Samineni, S.
, 
Sajja, S. B.
, 
Mondal, B.
, 
Chand, U.
, 
Thudi, M.
, 
Varshney, R. K.
, &amp; 
Gaur, P. M.
 (2021). MAGIC lines in chickpea: Development and exploitation of genetic diversity. Euphytica, 217(7), Article 137. 10.1007/s10681-021-02874-0
</cite> [<a href="https://doi.org/10.1007/s10681-021-02874-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Euphytica&amp;title=MAGIC%20lines%20in%20chickpea:%20Development%20and%20exploitation%20of%20genetic%20diversity&amp;author=S.%20Samineni&amp;author=S.%20B.%20Sajja&amp;author=B.%20Mondal&amp;author=U.%20Chand&amp;author=M.%20Thudi&amp;volume=217&amp;issue=7&amp;publication_year=2021&amp;pages=137&amp;doi=10.1007/s10681-021-02874-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270096-bib-0042">
<cite>

Sannemann, W.
, 
Lisker, A.
, 
Maurer, A.
, 
Léon, J.
, 
Kazman, E.
, 
Cöster, H.
, 
Holzapfel, J.
, 
Kempf, H.
, 
Korzun, V.
, 
Ebmeyer, E.
, &amp; 
Pillen, K.
 (2018). Adaptive selection of founder segments and epistatic control of plant height in the MAGIC winter wheat population WM‐800. BMC Genomics, 19(1), Article 559. 10.1186/s12864-018-4915-3

</cite> [<a href="https://doi.org/10.1186/s12864-018-4915-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6069784/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30064354/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Genomics&amp;title=Adaptive%20selection%20of%20founder%20segments%20and%20epistatic%20control%20of%20plant%20height%20in%20the%20MAGIC%20winter%20wheat%20population%20WM%E2%80%90800&amp;author=W.%20Sannemann&amp;author=A.%20Lisker&amp;author=A.%20Maurer&amp;author=J.%20L%C3%A9on&amp;author=E.%20Kazman&amp;volume=19&amp;issue=1&amp;publication_year=2018&amp;pages=559&amp;pmid=30064354&amp;doi=10.1186/s12864-018-4915-3&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270096-bib-0043">
<cite>

Scott, M. F.
, 
Ladejobi, O.
, 
Amer, S.
, 
Bentley, A. R.
, 
Biernaskie, J.
, 
Boden, S. A.
, 
Clark, M.
, 
Dell'Acqua, M.
, 
Dixon, L. E.
, 
Filippi, C. V.
, 
Fradgley, N.
, 
Gardner, K. A.
, 
Mackay, I. J.
, 
O'Sullivan, D.
, 
Percival‐Alwyn, L.
, 
Roorkiwal, M.
, 
Singh, R. K.
, 
Thudi, M.
, 
Varshney, R. K.
, … 
Mott, R.
 (2020). Multi‐parent populations in crops: A toolbox integrating genomics and genetic mapping with breeding. Heredity, 125(6), 396–416. 10.1038/s41437-020-0336-6

</cite> [<a href="https://doi.org/10.1038/s41437-020-0336-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7784848/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32616877/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Heredity&amp;title=Multi%E2%80%90parent%20populations%20in%20crops:%20A%20toolbox%20integrating%20genomics%20and%20genetic%20mapping%20with%20breeding&amp;author=M.%20F.%20Scott&amp;author=O.%20Ladejobi&amp;author=S.%20Amer&amp;author=A.%20R.%20Bentley&amp;author=J.%20Biernaskie&amp;volume=125&amp;issue=6&amp;publication_year=2020&amp;pages=396-416&amp;pmid=32616877&amp;doi=10.1038/s41437-020-0336-6&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270096-bib-0044">
<cite>

Sivasakthi, K.
, 
Thudi, M.
, 
Tharanya, M.
, 
Kale, S. M.
, 
Kholová, J.
, 
Halime, M. H.
, 
Jaganathan, D.
, 
Baddam, R.
, 
Thirunalasundari, T.
, 
Gaur, P. M.
, 
Varshney, R. K.
, &amp; 
Vadez, V.
 (2018). Plant vigour QTLs co‐map with an earlier reported QTL hotspot for drought tolerance while water saving QTLs map in other regions of the chickpea genome. BMC Plant Biology, 18(1), Article 29. 10.1186/s12870-018-1245-1

</cite> [<a href="https://doi.org/10.1186/s12870-018-1245-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5801699/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29409451/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Plant%20Biology&amp;title=Plant%20vigour%20QTLs%20co%E2%80%90map%20with%20an%20earlier%20reported%20QTL%20hotspot%20for%20drought%20tolerance%20while%20water%20saving%20QTLs%20map%20in%20other%20regions%20of%20the%20chickpea%20genome&amp;author=K.%20Sivasakthi&amp;author=M.%20Thudi&amp;author=M.%20Tharanya&amp;author=S.%20M.%20Kale&amp;author=J.%20Kholov%C3%A1&amp;volume=18&amp;issue=1&amp;publication_year=2018&amp;pages=29&amp;pmid=29409451&amp;doi=10.1186/s12870-018-1245-1&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270096-bib-0045">
<cite>

Stadlmeier, M.
, 
Hartl, L.
, &amp; 
Mohler, V.
 (2018). Usefulness of a multiparent advanced generation intercross population with a greatly reduced mating design for genetic studies in winter wheat. Frontiers in Plant Science, 9, Article 1825. 10.3389/fpls.2018.01825

</cite> [<a href="https://doi.org/10.3389/fpls.2018.01825" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6291512/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30574161/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Plant%20Science&amp;title=Usefulness%20of%20a%20multiparent%20advanced%20generation%20intercross%20population%20with%20a%20greatly%20reduced%20mating%20design%20for%20genetic%20studies%20in%20winter%20wheat&amp;author=M.%20Stadlmeier&amp;author=L.%20Hartl&amp;author=V.%20Mohler&amp;volume=9&amp;publication_year=2018&amp;pages=1825&amp;pmid=30574161&amp;doi=10.3389/fpls.2018.01825&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270096-bib-0046">
<cite>

Thudi, M.
, 
Samineni, S.
, 
Li, W.
, 
Boer, M. P.
, 
Roorkiwal, M.
, 
Yang, Z.
, 
Ladejobi, F.
, 
Zheng, C.
, 
Chitikineni, A.
, 
Nayak, S.
, 
He, Z.
, 
Valluri, V.
, 
Bajaj, P.
, 
Khan, A. W.
, 
Gaur, P. M.
, 
van Eeuwijk, F.
, 
Mott, R.
, 
Xin, L.
, &amp; 
Varshney, R. K.
 (2024). Whole genome resequencing and phenotyping of MAGIC population for high resolution mapping of drought tolerance in chickpea. The Plant Genome, 17(1), e20333. 10.1002/tpg2.20333

</cite> [<a href="https://doi.org/10.1002/tpg2.20333" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37122200/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=The%20Plant%20Genome&amp;title=Whole%20genome%20resequencing%20and%20phenotyping%20of%20MAGIC%20population%20for%20high%20resolution%20mapping%20of%20drought%20tolerance%20in%20chickpea&amp;author=M.%20Thudi&amp;author=S.%20Samineni&amp;author=W.%20Li&amp;author=M.%20P.%20Boer&amp;author=M.%20Roorkiwal&amp;volume=17&amp;issue=1&amp;publication_year=2024&amp;pages=e20333&amp;pmid=37122200&amp;doi=10.1002/tpg2.20333&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270096-bib-0047">
<cite>

Thudi, M.
, 
Upadhyaya, H. D.
, 
Rathore, A.
, 
Gaur, P. M.
, 
Krishnamurthy, L.
, 
Roorkiwal, M.
, 
Nayak, S. N.
, 
Chaturvedi, S. K.
, 
Basu, P. S.
, 
Gangarao, N. V. P. R.
, 
Fikre, A.
, 
Kimurto, P.
, 
Sharma, P. C.
, 
Sheshashayee, M. S.
, 
Tobita, S.
, 
Kashiwagi, J.
, 
Ito, O.
, 
Killian, A.
, &amp; 
Varshney, R. K.
 (2014). Genetic dissection of drought and heat tolerance in chickpea through genome‐wide and candidate gene‐based association mapping approaches. PLoS ONE, 9(5), e96758. 10.1371/journal.pone.0096758

</cite> [<a href="https://doi.org/10.1371/journal.pone.0096758" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4011848/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24801366/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20ONE&amp;title=Genetic%20dissection%20of%20drought%20and%20heat%20tolerance%20in%20chickpea%20through%20genome%E2%80%90wide%20and%20candidate%20gene%E2%80%90based%20association%20mapping%20approaches&amp;author=M.%20Thudi&amp;author=H.%20D.%20Upadhyaya&amp;author=A.%20Rathore&amp;author=P.%20M.%20Gaur&amp;author=L.%20Krishnamurthy&amp;volume=9&amp;issue=5&amp;publication_year=2014&amp;pages=e96758&amp;pmid=24801366&amp;doi=10.1371/journal.pone.0096758&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270096-bib-0053">
<cite>

Upadhyaya, H. D.
, 
Bajaj, D.
, 
Narnoliya, L.
, 
Das, S.
, 
Kumar, V.
, 
Gowda, C. L.
, 
Sharma, S.
, 
Tyagi, A. K.
, &amp; 
Parida, S. K.
 (2016). Genome‐Wide Scans for Delineation of Candidate Genes Regulating Seed‐Protein Content in Chickpea. Frontiers in plant science, 7, 302. 10.3389/fpls.2016.00302

</cite> [<a href="https://doi.org/10.3389/fpls.2016.00302" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4803732/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27047499/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20plant%20science&amp;title=Genome%E2%80%90Wide%20Scans%20for%20Delineation%20of%20Candidate%20Genes%20Regulating%20Seed%E2%80%90Protein%20Content%20in%20Chickpea&amp;author=H.%20D.%20Upadhyaya&amp;author=D.%20Bajaj&amp;author=L.%20Narnoliya&amp;author=S.%20Das&amp;author=V.%20Kumar&amp;volume=7&amp;publication_year=2016&amp;pages=302&amp;pmid=27047499&amp;doi=10.3389/fpls.2016.00302&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270096-bib-0054">
<cite>

Van der Maesen, L. J. G.
 (1987). Origin, history and taxonomy of chickpea. In Saxena M. C., &amp; Singh K. B. (Eds.), The chickpea (pp. 11–34). CAB International.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=The%20chickpea&amp;author=L.%20J.%20G.%20Van%20der%20Maesen&amp;publication_year=1987&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270096-bib-0048">
<cite>

Varshney, R. K.
, 
Mir, R. R.
, 
Bhatia, S.
, 
Thudi, M.
, 
Hu, Y.
, 
Azam, S.
, 
Zhang, Y.
, 
Jaganathan, D.
, 
You, F. M.
, 
Gao, J.
, 
Riera‐Lizarazu, O.
, &amp; 
Luo, M.‐C.
 (2014). Integrated physical, genetic and genome map of chickpea (<em>Cicer arietinum</em> L.). Functional &amp; Integrative Genomics, 14(1), 59–73. 10.1007/s10142-014-0363-6

</cite> [<a href="https://doi.org/10.1007/s10142-014-0363-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4273598/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24610029/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Functional%20&amp;%20Integrative%20Genomics&amp;title=Integrated%20physical,%20genetic%20and%20genome%20map%20of%20chickpea%20(Cicer%20arietinum%20L.)&amp;author=R.%20K.%20Varshney&amp;author=R.%20R.%20Mir&amp;author=S.%20Bhatia&amp;author=M.%20Thudi&amp;author=Y.%20Hu&amp;volume=14&amp;issue=1&amp;publication_year=2014&amp;pages=59-73&amp;pmid=24610029&amp;doi=10.1007/s10142-014-0363-6&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270096-bib-0049">
<cite>

Varshney, R. K.
, 
Mohan, S. M.
, 
Gaur, P. M.
, 
Chamarthi, S. K.
, 
Singh, V. K.
, 
Srinivasan, S.
, 
Swapna, N.
, 
Sharma, M.
, 
Pande, S.
, 
Singh, S.
, &amp; 
Kaur, L.
 (2014). Marker‐assisted backcrossing to introgress resistance to Fusarium wilt race 1 and Ascochyta blight in C 214, an elite cultivar of chickpea. The Plant Genome, 7(1), plantgenome2013.2010.0035. 10.3835/plantgenome2013.10.0035
</cite> [<a href="https://doi.org/10.3835/plantgenome2013.10.0035" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=The%20Plant%20Genome&amp;title=Marker%E2%80%90assisted%20backcrossing%20to%20introgress%20resistance%20to%20Fusarium%20wilt%20race%201%20and%20Ascochyta%20blight%20in%20C%20214,%20an%20elite%20cultivar%20of%20chickpea&amp;author=R.%20K.%20Varshney&amp;author=S.%20M.%20Mohan&amp;author=P.%20M.%20Gaur&amp;author=S.%20K.%20Chamarthi&amp;author=V.%20K.%20Singh&amp;volume=7&amp;issue=1&amp;publication_year=2014&amp;doi=10.3835/plantgenome2013.10.0035&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270096-bib-0050">
<cite>

Varshney, R. K.
, 
Song, C.
, 
Saxena, R. K.
, 
Azam, S.
, 
Yu, S.
, 
Sharpe, A. G.
, 
Cannon, S.
, 
Baek, J.
, 
Rosen, B. D.
, 
Tar'an, B.
, 
Millan, T.
, 
Zhang, X.
, 
Ramsay, L. D.
, 
Iwata, A.
, 
Wang, Y.
, 
Nelson, W.
, 
Farmer, A. D.
, 
Gaur, P. M.
, 
Soderlund, C.
, … 
Cook, D. R.
 (2013). Draft genome sequence of chickpea (<em>Cicer arietinum</em>) provides a resource for trait improvement. Nature Biotechnology, 31(3), 240–246. 10.1038/nbt.2491
</cite> [<a href="https://doi.org/10.1038/nbt.2491" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23354103/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Biotechnology&amp;title=Draft%20genome%20sequence%20of%20chickpea%20(Cicer%20arietinum)%20provides%20a%20resource%20for%20trait%20improvement&amp;author=R.%20K.%20Varshney&amp;author=C.%20Song&amp;author=R.%20K.%20Saxena&amp;author=S.%20Azam&amp;author=S.%20Yu&amp;volume=31&amp;issue=3&amp;publication_year=2013&amp;pages=240-246&amp;pmid=23354103&amp;doi=10.1038/nbt.2491&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270096-bib-0051">
<cite>

Varshney, R. K.
, 
Thudi, M.
, 
Roorkiwal, M.
, 
He, W.
, 
Upadhyaya, H. D.
, 
Yang, W.
, 
Bajaj, P.
, 
Cubry, P.
, 
Rathore, A.
, 
Jian, J.
, 
Doddamani, D.
, 
Khan, A. W.
, 
Garg, V.
, 
Chitikineni, A.
, 
Xu, D.
, 
Gaur, P. M.
, 
Singh, N. P.
, 
Chaturvedi, S. K.
, 
Nadigatla, G. V. P. R.
, … 
Liu, X.
 (2019). Resequencing of 429 chickpea accessions from 45 countries provides insights into genome diversity, domestication and agronomic traits. Nature Genetics, 51(5), 857–864. 10.1038/s41588-019-0401-3

</cite> [<a href="https://doi.org/10.1038/s41588-019-0401-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31036963/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Genetics&amp;title=Resequencing%20of%20429%20chickpea%20accessions%20from%2045%20countries%20provides%20insights%20into%20genome%20diversity,%20domestication%20and%20agronomic%20traits&amp;author=R.%20K.%20Varshney&amp;author=M.%20Thudi&amp;author=M.%20Roorkiwal&amp;author=W.%20He&amp;author=H.%20D.%20Upadhyaya&amp;volume=51&amp;issue=5&amp;publication_year=2019&amp;pages=857-864&amp;pmid=31036963&amp;doi=10.1038/s41588-019-0401-3&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270096-bib-0057">
<cite>

Varshney, R. K.
, 
Shi, C.
, 
Thudi, M.
, 
Mariac, C.
, 
Wallace, J.
, 
Qi, P.
, 
Zhang, H.
, 
Zhao, Y.
, 
Wang, X.
, 
Rathore, A.
, 
Srivastava, R. K.
, 
Chitikineni, A.
, 
Fan, G.
, 
Bajaj, P.
, 
Punnuri, S.
, 
Gupta, S. K.
, 
Wang, H.
, 
Jiang, Y.
, 
Couderc, M.
, … 
Xu, X.
 (2017). Pearl millet genome sequence provides a resource to improve agronomic traits in arid environments. Nature biotechnology, 35(10), 969–976. 10.1038/nbt.3943
</cite> [<a href="https://doi.org/10.1038/nbt.3943" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6871012/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28922347/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20biotechnology&amp;title=Pearl%20millet%20genome%20sequence%20provides%20a%20resource%20to%20improve%20agronomic%20traits%20in%20arid%20environments&amp;author=R.%20K.%20Varshney&amp;author=C.%20Shi&amp;author=M.%20Thudi&amp;author=C.%20Mariac&amp;author=J.%20Wallace&amp;volume=35&amp;issue=10&amp;publication_year=2017&amp;pages=969-976&amp;pmid=28922347&amp;doi=10.1038/nbt.3943&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270096-bib-0065">
<cite>

Varshney, R. K.
, 
Thudi, M.
, 
Nayak, S. N.
, 
Gaur, P. M.
, 
Kashiwagi, J.
, 
Krishnamurthy, L.
, 
Jaganathan, D.
, 
Koppolu, J.
, 
Bohra, A.
, 
Tripathi, S.
, 
Rathore, A.
, 
Jukanti, A. K.
, 
Jayalakshmi, V.
, 
Vemula, A.
, 
Singh, S. J.
, 
Yasin, M.
, 
Sheshshayee, M. S.
, &amp; 
Viswanatha, K. P.
 (2014). Genetic dissection of drought tolerance in chickpea (<em>Cicer arietinum</em> L.). Theoretical and Applied Genetics, 127(2), 445–462. 10.1007/s00122-013-2230-6

</cite> [<a href="https://doi.org/10.1007/s00122-013-2230-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3910274/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24326458/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Theoretical%20and%20Applied%20Genetics&amp;title=Genetic%20dissection%20of%20drought%20tolerance%20in%20chickpea%20(Cicer%20arietinum%20L.)&amp;author=R.%20K.%20Varshney&amp;author=M.%20Thudi&amp;author=S.%20N.%20Nayak&amp;author=P.%20M.%20Gaur&amp;author=J.%20Kashiwagi&amp;volume=127&amp;issue=2&amp;publication_year=2014&amp;pages=445-462&amp;pmid=24326458&amp;doi=10.1007/s00122-013-2230-6&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270096-bib-0070">
<cite>

Varshney, R. K.
, 
Roorkiwal, M.
, 
Sun, S.
, 
Bajaj, P.
, 
Chitikineni, A.
, 
Thudi, M.
, 
Singh, N. P.
, 
Du, X.
, 
Upadhyaya, H. D.
, 
Khan, A. W.
, 
Wang, Y.
, 
Garg, V.
, 
Fan, G.
, 
Cowling, W. A.
, 
Crossa, J.
, 
Gentzbittel, L.
, 
Voss‐Fels, K. P.
, 
Valluri, V. K.
, 
Sinha, P.
, … 
Liu, X.
 (2021). A chickpea genetic variation map based on the sequencing of 3,366 genomes. Nature, 599(7886), 622–627. 10.1038/s41586-021-04066-1

</cite> [<a href="https://doi.org/10.1038/s41586-021-04066-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8612933/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34759320/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=A%20chickpea%20genetic%20variation%20map%20based%20on%20the%20sequencing%20of%203,366%20genomes&amp;author=R.%20K.%20Varshney&amp;author=M.%20Roorkiwal&amp;author=S.%20Sun&amp;author=P.%20Bajaj&amp;author=A.%20Chitikineni&amp;volume=599&amp;issue=7886&amp;publication_year=2021&amp;pages=622-627&amp;pmid=34759320&amp;doi=10.1038/s41586-021-04066-1&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270096-bib-0063">
<cite>

Voorrips, R.
 (2002). MapChart: Software for the graphical presentation of linkage maps and QTLs. Journal of heredity, 93(1), 77–78. 10.1093/jhered/93.1.77

</cite> [<a href="https://doi.org/10.1093/jhered/93.1.77" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12011185/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20heredity&amp;title=MapChart:%20Software%20for%20the%20graphical%20presentation%20of%20linkage%20maps%20and%20QTLs&amp;author=R.%20Voorrips&amp;volume=93&amp;issue=1&amp;publication_year=2002&amp;pages=77-78&amp;pmid=12011185&amp;doi=10.1093/jhered/93.1.77&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270096-bib-0062">
<cite>

Voss‐Fels, K. P.
, 
Stahl, A.
, 
Wittkop, B.
, 
Lichthardt, C.
, 
Nagler, S.
, 
Rose, T.
, 
Chen, T.‐W.
, 
Zetzsche, H.
, 
Seddig, S.
, 
Majid Baig, M.
, 
Ballvora, A.
, 
Frisch, M.
, 
Ross, E.
, 
Hayes, B. J.
, 
Hayden, M. J.
, 
Ordon, F.
, 
Leon, J.
, 
Kage, H.
, 
Friedt, W.
, … 
Snowdon, R. J.
 (2019). Breeding improves wheat productivity under contrasting agrochemical input levels. Nature Plants, 5(7), 706–714. 10.1038/s41477-019-0445-5

</cite> [<a href="https://doi.org/10.1038/s41477-019-0445-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31209285/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Plants&amp;title=Breeding%20improves%20wheat%20productivity%20under%20contrasting%20agrochemical%20input%20levels&amp;author=K.%20P.%20Voss%E2%80%90Fels&amp;author=A.%20Stahl&amp;author=B.%20Wittkop&amp;author=C.%20Lichthardt&amp;author=S.%20Nagler&amp;volume=5&amp;issue=7&amp;publication_year=2019&amp;pages=706-714&amp;pmid=31209285&amp;doi=10.1038/s41477-019-0445-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270096-bib-0052">
<cite>

Wang, J.
, &amp; 
Zhang, Z.
 (2021). GAPIT version 3: Boosting power and accuracy for genomic association and prediction. Genomics, Proteomics &amp; Bioinformatics, 19, 629–640. 10.1016/j.gpb.2021.08.005
</cite> [<a href="https://doi.org/10.1016/j.gpb.2021.08.005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9121400/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34492338/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genomics,%20Proteomics%20&amp;%20Bioinformatics&amp;title=GAPIT%20version%203:%20Boosting%20power%20and%20accuracy%20for%20genomic%20association%20and%20prediction&amp;author=J.%20Wang&amp;author=Z.%20Zhang&amp;volume=19&amp;publication_year=2021&amp;pages=629-640&amp;pmid=34492338&amp;doi=10.1016/j.gpb.2021.08.005&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270096-bib-0059">
<cite>

Wickham, H.
, &amp; 
Sievert, C.
 (2016). ggplot2: Elegant graphics for data analysis (2nd ed.). Springer.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=ggplot2:%20Elegant%20graphics%20for%20data%20analysis&amp;author=H.%20Wickham&amp;author=C.%20Sievert&amp;publication_year=2016&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ol></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="caption p">
<p>Supplementary Table 1 List and code of parental lines used for the development of the chickpea MAGIC population, including their description and country of cultivation.</p>
<p>Supplementary Table 2. Linkage disequilibrium (LD) among the pairs of markers significantly associated with DTF and PHT to determine those that are linked in order to categorize them into QTL. SNP markers with LD ≥ 0.1 are linked</p>
<p>Supplementary Table 3 Summary statistics of distributions of minor allele frequencies (MAF) and observed heterozygosity in genotypic data of population.</p>
<p>Supplementary Table 4 Summary of the QTL for days to 50% flowering (<em>Q‐DTF</em>) and plant height (<em>Q‐PH</em>) in the chickpea MAGIC population. Linked markers in LD ≥ 0.1 are highlighted in bold. With the haplo‐blocks co‐located in the same regions.</p>
<p>Supplementary Figure 1. Population structure of the MAGIC population and their founder parents. Principal component analysis based on the pairwise genomic relationship matrix (Roger's distances) calculated from 4255 high‐quality polymorphic SNP markers in 1135 MAGIC lines representing seven crossing designs and eight founder lines. The variation explained by principal components were PC1 = 12.89% and PC2 = 9.97%. The colored triangles represent the parents while the colored circles represent the cross‐design (pedigree).</p>
<p>Supplementary Figure 2 (A) Allele frequency spectra (histogram distributions of minor allele frequency [MAF]). (B) Distribution histograms of observed heterozygosity across the MAGIC population.</p>
<p>Supplementary Figure 3. Genome‐wide profile of the eight MAGIC founder lines. SNP alleles are represented by colors: adenine (A) is indicated in yellow, cytosine (C) is represented in blue, guanine (G) is represented in red and thymine (T) is represented in green.</p>
<p>Supplementary Figure 4. Population structure of the MAGIC population and their founder parents. Principal component analysis based on the pairwise genomic relationship matrix (Roger's distances) calculated from 4255 high‐quality polymorphic SNP markers in 1135 MAGIC lines representing seven crossing designs and eight founder lines. The variation explained by principal components were PC1 = 12.89% and PC2 = 9.97%. The colored triangles represent the parents while the colored circles represent the cross‐design (pedigree).</p>
<p>Supplementary Figure 5. Genome‐wide association mapping for plant height (PHT) and days to 50% flowering (DTF) for 1135 MAGIC lines using 4255 high‐quality SNP markers. <em>QQ</em> plots for plant height in blue and days to 50% flowering in green are displayed.</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12375853/bin/TPG2-18-e70096-s001.pdf" data-ga-action="click_feat_suppl" class="usa-link">TPG2-18-e70096-s001.pdf</a><sup> (702KB, pdf) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>All data supporting the findings of this study are available within the paper and within its <a href="#tpg270096-supinfo-0001" class="usa-link">Supporting Information</a>. Genotypic and phenotypic datasets are available from the corresponding author upon request.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from The Plant Genome are provided here courtesy of <strong>Wiley</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1002/tpg2.70096"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/TPG2-18-e70096.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (1.8 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12375853/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12375853/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12375853%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12375853/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12375853/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12375853/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12375853/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12375853/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12375853/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="QC0Zna1UanPvxP0JT2ElZRX26UMgTpahFPZHj7O4daUei12TVo7CBWpXGFln2Gcg">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
        <script type="module" crossorigin="" src="/static/assets/math-574fdcc6.js"></script>
    
    

    </body>
</html>
