
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            FBXW7 regulates MYRF levels to control myelin capacity and homeostasis in the adult central nervous system - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="D2E4A0918AF1C95305A091005B0685D5.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="ncomms">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370921/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Nature Communications">
<meta name="citation_title" content="FBXW7 regulates MYRF levels to control myelin capacity and homeostasis in the adult central nervous system">
<meta name="citation_author" content="Hannah Y Collins">
<meta name="citation_author_institution" content="Jungers Center for Neurosciences Research, Department of Neurology, Oregon Health &amp; Science University, Portland, OR USA">
<meta name="citation_author_institution" content="Vollum Institute, Oregon Health &amp; Science University, Portland, OR USA">
<meta name="citation_author" content="Ryan A Doan">
<meta name="citation_author_institution" content="Vollum Institute, Oregon Health &amp; Science University, Portland, OR USA">
<meta name="citation_author" content="Jiaxing Li">
<meta name="citation_author_institution" content="Vollum Institute, Oregon Health &amp; Science University, Portland, OR USA">
<meta name="citation_author_institution" content="Present Address: Department of Neuroscience, Medical University of South Carolina, Charleston, SC USA">
<meta name="citation_author" content="Jason E Early">
<meta name="citation_author_institution" content="Centre for Discovery Brain Sciences, MS Society Edinburgh Centre for MS Research, University of Edinburgh, Edinburgh, UK">
<meta name="citation_author" content="Megan E Madden">
<meta name="citation_author_institution" content="Centre for Discovery Brain Sciences, MS Society Edinburgh Centre for MS Research, University of Edinburgh, Edinburgh, UK">
<meta name="citation_author" content="Tyrell Simkins">
<meta name="citation_author_institution" content="Jungers Center for Neurosciences Research, Department of Neurology, Oregon Health &amp; Science University, Portland, OR USA">
<meta name="citation_author_institution" content="Vollum Institute, Oregon Health &amp; Science University, Portland, OR USA">
<meta name="citation_author_institution" content="Present Address: Medical Director/Physician Scientist at Cytokinetics, South San Francisco, California, USA">
<meta name="citation_author" content="David A Lyons">
<meta name="citation_author_institution" content="Centre for Discovery Brain Sciences, MS Society Edinburgh Centre for MS Research, University of Edinburgh, Edinburgh, UK">
<meta name="citation_author" content="Kelly R Monk">
<meta name="citation_author_institution" content="Vollum Institute, Oregon Health &amp; Science University, Portland, OR USA">
<meta name="citation_author" content="Ben Emery">
<meta name="citation_author_institution" content="Jungers Center for Neurosciences Research, Department of Neurology, Oregon Health &amp; Science University, Portland, OR USA">
<meta name="citation_publication_date" content="2025 Aug 21">
<meta name="citation_volume" content="16">
<meta name="citation_firstpage" content="7822">
<meta name="citation_doi" content="10.1038/s41467-025-62715-9">
<meta name="citation_pmid" content="40841354">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370921/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370921/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370921/pdf/41467_2025_Article_62715.pdf">
<meta name="description" content="Myelin, along with the oligodendrocytes (OLs) that produce it, is essential for proper central nervous system (CNS) function in vertebrates. Although the accurate targeting of myelin to axons and its maintenance are critical for CNS performance, the ...">
<meta name="og:title" content="FBXW7 regulates MYRF levels to control myelin capacity and homeostasis in the adult central nervous system">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Myelin, along with the oligodendrocytes (OLs) that produce it, is essential for proper central nervous system (CNS) function in vertebrates. Although the accurate targeting of myelin to axons and its maintenance are critical for CNS performance, the ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370921/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12370921">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1038/s41467-025-62715-9"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/41467_2025_Article_62715.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12370921%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12370921/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12370921/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370921/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-ncomms.gif" alt="Nature Communications logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Nature Communications" title="Link to Nature Communications" shape="default" href="http://www.nature.com/ncomms/index.html" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Nat Commun</button></div>. 2025 Aug 21;16:7822. doi: <a href="https://doi.org/10.1038/s41467-025-62715-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1038/s41467-025-62715-9</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Nat%20Commun%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nat%20Commun%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Nat%20Commun%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Nat%20Commun%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>FBXW7 regulates MYRF levels to control myelin capacity and homeostasis in the adult central nervous system</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Collins%20HY%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Hannah Y Collins</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Hannah Y Collins</span></h3>
<div class="p">
<sup>1</sup>Jungers Center for Neurosciences Research, Department of Neurology, Oregon Health &amp; Science University, Portland, OR USA </div>
<div class="p">
<sup>2</sup>Vollum Institute, Oregon Health &amp; Science University, Portland, OR USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Collins%20HY%22%5BAuthor%5D" class="usa-link"><span class="name western">Hannah Y Collins</span></a>
</div>
</div>
<sup>1,</sup><sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Doan%20RA%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Ryan A Doan</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Ryan A Doan</span></h3>
<div class="p">
<sup>2</sup>Vollum Institute, Oregon Health &amp; Science University, Portland, OR USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Doan%20RA%22%5BAuthor%5D" class="usa-link"><span class="name western">Ryan A Doan</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Li%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Jiaxing Li</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Jiaxing Li</span></h3>
<div class="p">
<sup>2</sup>Vollum Institute, Oregon Health &amp; Science University, Portland, OR USA </div>
<div class="p">
<sup>4</sup>Present Address: Department of Neuroscience, Medical University of South Carolina, Charleston, SC USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Li%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Jiaxing Li</span></a>
</div>
</div>
<sup>2,</sup><sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Early%20JE%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Jason E Early</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Jason E Early</span></h3>
<div class="p">
<sup>3</sup>Centre for Discovery Brain Sciences, MS Society Edinburgh Centre for MS Research, University of Edinburgh, Edinburgh, UK </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Early%20JE%22%5BAuthor%5D" class="usa-link"><span class="name western">Jason E Early</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Madden%20ME%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Megan E Madden</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Megan E Madden</span></h3>
<div class="p">
<sup>3</sup>Centre for Discovery Brain Sciences, MS Society Edinburgh Centre for MS Research, University of Edinburgh, Edinburgh, UK </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Madden%20ME%22%5BAuthor%5D" class="usa-link"><span class="name western">Megan E Madden</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Simkins%20T%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Tyrell Simkins</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Tyrell Simkins</span></h3>
<div class="p">
<sup>1</sup>Jungers Center for Neurosciences Research, Department of Neurology, Oregon Health &amp; Science University, Portland, OR USA </div>
<div class="p">
<sup>2</sup>Vollum Institute, Oregon Health &amp; Science University, Portland, OR USA </div>
<div class="p">
<sup>5</sup>Present Address: Medical Director/Physician Scientist at Cytokinetics, South San Francisco, California, USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Simkins%20T%22%5BAuthor%5D" class="usa-link"><span class="name western">Tyrell Simkins</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>5</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lyons%20DA%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">David A Lyons</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">David A Lyons</span></h3>
<div class="p">
<sup>3</sup>Centre for Discovery Brain Sciences, MS Society Edinburgh Centre for MS Research, University of Edinburgh, Edinburgh, UK </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lyons%20DA%22%5BAuthor%5D" class="usa-link"><span class="name western">David A Lyons</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Monk%20KR%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Kelly R Monk</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Kelly R Monk</span></h3>
<div class="p">
<sup>2</sup>Vollum Institute, Oregon Health &amp; Science University, Portland, OR USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Monk%20KR%22%5BAuthor%5D" class="usa-link"><span class="name western">Kelly R Monk</span></a>
</div>
</div>
<sup>2,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Emery%20B%22%5BAuthor%5D" class="usa-link" aria-describedby="id9"><span class="name western">Ben Emery</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">Ben Emery</span></h3>
<div class="p">
<sup>1</sup>Jungers Center for Neurosciences Research, Department of Neurology, Oregon Health &amp; Science University, Portland, OR USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Emery%20B%22%5BAuthor%5D" class="usa-link"><span class="name western">Ben Emery</span></a>
</div>
</div>
<sup>1,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>Jungers Center for Neurosciences Research, Department of Neurology, Oregon Health &amp; Science University, Portland, OR USA </div>
<div id="Aff2">
<sup>2</sup>Vollum Institute, Oregon Health &amp; Science University, Portland, OR USA </div>
<div id="Aff3">
<sup>3</sup>Centre for Discovery Brain Sciences, MS Society Edinburgh Centre for MS Research, University of Edinburgh, Edinburgh, UK </div>
<div id="Aff4">
<sup>4</sup>Present Address: Department of Neuroscience, Medical University of South Carolina, Charleston, SC USA </div>
<div id="Aff5">
<sup>5</sup>Present Address: Medical Director/Physician Scientist at Cytokinetics, South San Francisco, California, USA </div>
<div class="author-notes p"><div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div></div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2024 Oct 14; Accepted 2025 Jul 23; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by-nc-nd/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12370921  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40841354/" class="usa-link">40841354</a>
</div>
<div class="ra xbox p" role="complementary" aria-label="Related or updated information about this article"><div>
<strong>Previous version available:</strong> This article is based on a previously available preprint posted on bioRxiv on October 17, 2024: "<a href="/articles/PMC11507870/" class="usa-link">FBXW7 regulates MYRF levels to control myelin capacity and homeostasis in the adult CNS</a>".</div></div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<p id="Par1">Myelin, along with the oligodendrocytes (OLs) that produce it, is essential for proper central nervous system (CNS) function in vertebrates. Although the accurate targeting of myelin to axons and its maintenance are critical for CNS performance, the molecular pathways that regulate these processes remain poorly understood. Through a combination of zebrafish genetics, mouse models, and primary OL cultures, we find that FBXW7, a recognition subunit of an E3 ubiquitin ligase complex, is a regulator of adult myelination in the CNS. Loss of <em>Fbxw7</em> in myelinating OLs results in increased myelin sheath lengths with no change in myelin thickness. As the animals age, they develop progressive abnormalities including myelin outfolds, disrupted paranodal organization, and ectopic ensheathment of neuronal cell bodies with myelin. Through biochemical studies we find that FBXW7 directly binds and degrades the N-terminus of Myelin Regulatory Factor (N-MYRF), to control the balance between OL myelin growth and homeostasis.</p>
<section id="kwd-group1" class="kwd-group"><p><strong>Subject terms:</strong> Oligodendrocyte, Myelin biology and repair, Cellular neuroscience</p></section></section><section class="abstract" id="Abs2"><hr class="headless">
<p id="Par2">The homeostasis of myelin in the central nervous system is highly regulated, but the underlying mechanisms are unclear. Here, the authors show that the FBXW7 protein has a role in constraining myelin growth in the developing and adult central nervous system, preventing the accumulation of myelin abnormalities. FBXW7 acts in part through targeting the MYRF transcription factor for degradation.</p></section><section id="Sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p id="Par3">In the vertebrate central nervous system (CNS), myelin is produced by specialized glial cells called oligodendrocytes (OLs). OLs wrap segments of axons, creating a multi-layered sheath that speeds the transmission of nerve impulses and provides critical support to axons<sup><a href="#CR1" class="usa-link" aria-describedby="CR1">1</a>–<a href="#CR4" class="usa-link" aria-describedby="CR4">4</a></sup>. The formation and targeting of myelin is influenced by external cues such as axonal caliber and neuronal activity, but is also tightly controlled by cell-intrinsic programs<sup><a href="#CR5" class="usa-link" aria-describedby="CR5">5</a>–<a href="#CR12" class="usa-link" aria-describedby="CR12">12</a></sup>. As OL precursor cells (OPCs) differentiate from a dynamic and proliferative state into a relatively stable post-mitotic myelinating OL, they reorganize and expand their cytoskeleton, cytoplasm, and membrane<sup><a href="#CR13" class="usa-link" aria-describedby="CR13">13</a>–<a href="#CR15" class="usa-link" aria-describedby="CR15">15</a></sup>, requiring significant transcriptional changes<sup><a href="#CR16" class="usa-link" aria-describedby="CR16">16</a>–<a href="#CR19" class="usa-link" aria-describedby="CR19">19</a></sup>. Although OLs are long-lived, surviving up to years in mice and decades in human white matter tracts, the myelin constituents themselves turn over comparatively rapidly, with a half-life of months<sup><a href="#CR20" class="usa-link" aria-describedby="CR20">20</a>–<a href="#CR23" class="usa-link" aria-describedby="CR23">23</a></sup>. Individual myelin sheaths can also be remodeled throughout life<sup><a href="#CR12" class="usa-link" aria-describedby="CR12">12</a>,<a href="#CR24" class="usa-link" aria-describedby="CR24">24</a>,<a href="#CR25" class="usa-link" aria-describedby="CR25">25</a></sup>. Therefore, understanding the molecular pathways and mechanisms that balance myelin growth and homeostasis is crucial for understanding myelin’s role in health, aging, and disease.</p>
<p id="Par4">Previous work from our lab and others has identified F-box and WD repeat domain-containing protein 7 (FBXW7) as a key negative regulator of developmental myelination by both Schwann cells in the peripheral nervous system (PNS) of mice and OLs in the CNS of zebrafish<sup><a href="#CR26" class="usa-link" aria-describedby="CR26">26</a>–<a href="#CR29" class="usa-link" aria-describedby="CR29">29</a></sup>. Within the PNS, FBXW7 regulates Schwann cell numbers, along with their myelin thickness<sup><a href="#CR28" class="usa-link" aria-describedby="CR28">28</a></sup>. Surprisingly, loss of <em>Fbxw7</em> also results in a breakdown of the normal 1:1 relationship between myelinating Schwann cells and axons, with individual <em>Fbxw7</em> conditional knockout Schwann cells aberrantly myelinating multiple axons<sup><a href="#CR28" class="usa-link" aria-describedby="CR28">28</a></sup>. Within the CNS of zebrafish, <em>fbxw7</em> regulates neural stem cell fate through Notch signaling, biasing the cells towards an OPC fate and increasing the pool of OL lineage cells in the spinal cord<sup><a href="#CR26" class="usa-link" aria-describedby="CR26">26</a></sup>. At later stages, loss of <em>fbxw7</em> leads to increased myelin sheath length, attributed to dysregulation of mTOR signaling<sup><a href="#CR27" class="usa-link" aria-describedby="CR27">27</a></sup>. <em>Fbxw7</em> encodes the F-box domain-containing recognition subunit of a SKP1-Cullin-Fbox (SCF) E3 ubiquitin ligase complex. It mediates its biological effects through targeting specific proteins for proteasomal degradation, thus controlling their total levels in the cell<sup><a href="#CR26" class="usa-link" aria-describedby="CR26">26</a>,<a href="#CR27" class="usa-link" aria-describedby="CR27">27</a>,<a href="#CR30" class="usa-link" aria-describedby="CR30">30</a>–<a href="#CR32" class="usa-link" aria-describedby="CR32">32</a></sup>. FBXW7 substrates are highly variable between cell types and have not yet been investigated in myelinating cells in an unbiased manner.</p>
<p id="Par5">To interrogate the role of FBXW7 in the regulation of CNS myelination, we used a combination of zebrafish, primary mammalian OL cultures, and conditional knockout mouse models. We found that inactivation of <em>fbxw7</em> in developing zebrafish resulted in enhanced OL maturation in the spinal cord. Conditional ablation of <em>Fbxw7</em> in mature OLs in the adult mouse CNS increased myelin sheath lengths but also resulted in progressive myelin abnormalities including outfolds, disrupted paranodal organization, and ectopic ensheathment of neuronal cell bodies with myelin. We found that <em>Fbxw7</em>-deficient OLs had no changes in mTOR protein levels in primary mammalian OL cultures, suggesting the myelin phenotypes were not a consequence of dysregulated mTOR signaling. Previous work in hepatocarcinoma cells identified the pro-myelination transcription factor Myelin Regulatory Factor (MYRF) as a target of FBXW7<sup><a href="#CR33" class="usa-link" aria-describedby="CR33">33</a></sup>. We demonstrate that the N-terminus of MYRF is a direct target of FBXW7 in OLs both in vitro and in vivo, with levels of N-MYRF protein and many of its transcriptional targets substantially increased in <em>Fbxw7</em>-deficient OLs. We also found that <em>Myrf</em> haploinsufficiency is sufficient to ameliorate both the increase in OL numbers in <em>fbxw7</em> knockout zebrafish, as well as myelin outfolds and ectopic myelination of granule cell in the cerebellum in our conditional knock-out mouse models. Taken together, our findings demonstrate that FBXW7 is an evolutionarily conserved negative regulator of OL myelination and that its negative regulation of MYRF in the adult CNS is required for long-term myelin homeostatic maintenance.</p></section><section id="Sec2"><h2 class="pmc_sec_title">Results</h2>
<section id="Sec3"><h3 class="pmc_sec_title">Fbxw7 regulates OPC specification and OL myelination in the zebrafish spinal cord</h3>
<p id="Par6">In zebrafish, global <em>fbxw7</em> mutations cause hypermyelination in both the PNS and CNS<sup><a href="#CR26" class="usa-link" aria-describedby="CR26">26</a>–<a href="#CR28" class="usa-link" aria-describedby="CR28">28</a></sup>. Previously published zebrafish <em>fbxw7</em> mutant alleles are late mutations within the region of the gene encoding the WD40 substrate recognition domain of the FBXW7 protein; therefore, existing mutants may not be complete loss-of-function alleles<sup><a href="#CR26" class="usa-link" aria-describedby="CR26">26</a>,<a href="#CR29" class="usa-link" aria-describedby="CR29">29</a></sup>. We used CRISPR-Cas9-mediated genome editing to create a new mutation, <em>fbxw7</em><sup><em>vo86</em></sup>. This mutation introduces a frameshift and early stop codon in exon 5, which encodes the F-Box domain that allows FBXW7 to interact with its E3 complex (Supplementary Fig. <a href="#MOESM1" class="usa-link">1a, b</a>). Through in situ hybridization, live imaging, and qPCR, we found that the <em>fbxw7</em><sup><em>vo86</em></sup> mutation phenocopies the previously described N-ethyl-N-nitrosourea (ENU)-generated <em>fbxw</em><sup><em>stl64</em></sup> mutants, including an increase in <em>myelin basic protein</em> (<em>mbp</em>) driven GFP in the dorsal spinal cord of <em>Tg(mbp:EGFP-caax)</em> animals, and an increase in total <em>mbp</em> RNA by qRT-PCR (Fig. <a href="#Fig1" class="usa-link">1a, b</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">1c–e</a>). Since <em>fbxw7</em><sup><em>vo86</em></sup> phenocopies the original ENU-generated mutation, we concluded that both of these mutations represent full loss-of-function alleles. As the <em>fbxw7</em><sup><em>vo86</em></sup> mutation disrupts the F-Box domain, the same region targeted in the Fbxw7<sup>fl/fl</sup> mouse line<sup><a href="#CR34" class="usa-link" aria-describedby="CR34">34</a></sup> also used in our studies (described below), this allele was used for all subsequent analyses.</p>
<figure class="fig xbox font-sm" id="Fig1"><h4 class="obj_head">Fig. 1. Fbxw7 regulates OPC specification and OL myelination in the zebrafish spinal cord.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370921_41467_2025_62715_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/23a7/12370921/d61b479bb540/41467_2025_62715_Fig1_HTML.jpg" loading="lazy" id="d33e644" height="696" width="683" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Spinal cords of <em>fbxw7</em><sup><em>vo86</em></sup> and WT controls in (top) <em>Tg(mbp:eGFP-caax)</em>, (middle) <em>Tg</em>(<em>olig2:dsRED)</em>, and (bottom) <em>Tg(mbp:nls-eGFP)</em> transgenic backgrounds at 3 and 5 days post-fertilization (dpf; dotted-line delineates dorsal and ventral spinal cord tracts). <strong>b</strong>
<em>mbp</em> RNA levels evaluated by qRT-PCR in 5 dpf <em>fbxw7</em><sup><em>vo86</em></sup> and WT control whole larvae. Mean ± SEM, N = 3 (larvae). Statistical significance determined by unpaired, two-tailed Student’s t test. <strong>c</strong> Quantification of <em>olig2:dsRED</em><sup><em>+</em></sup> cells/body segment in the dorsal spinal cord at 2, 3, and 5 dpf. Mean ± SEM, N = 7 (larvae). Statistical significance determined by two-way ANOVA. <strong>d</strong> Quantification of <em>mbp:nls-eGFP</em><sup><em>+</em></sup> OLs in the dorsal spinal cord/body segment at 3 dpf. Mean ± SEM, N = 7 (larvae). Statistical significance determined by unpaired, two-tailed Student’s t test. <strong>e</strong>
<em>mbp-nls:eGFP</em><sup><em>+</em></sup> OLs from Fig. 1d were divided by the average number of <em>olig2:dsRED</em><sup><em>+</em></sup> cells/body segments in dorsal spinal cord at 3 dpf from Fig. 1c. Mean ± SEM, statistical significance determined by unpaired, two-tailed Student’s t test. <strong>f</strong> Spinal cords of <em>fbxw7</em><sup><em>vo86</em></sup> and WT control zebrafish in <em>Tg(mbp:eGFP-caax)</em> and <em>Tg(mbp:nls-eGFP)</em> transgenic backgrounds live-imaged at 3 dpf. Solid arrowheads denote OL somas, hollow arrowheads denote neuronal cell body wrapping. <strong>g</strong> Representative image of an <em>fbxw7</em><sup><em>vo86</em></sup> larva in <em>Tg(mbp:eGFP-caax)</em> (white) and <em>Tg(nbt:dsRED)</em> (magenta) backgrounds showing neuronal cell bodies in the spinal cord wrapped in <em>mbp</em><sup><em>+</em></sup> membrane. <strong>h</strong> Quantification of wrapped neuronal cell bodies in the spinal cord in both <em>fbxw7</em><sup><em>vo86</em></sup> and WT controls at 3 dpf. Mean ± SEM, N = 5 (larvae). Statistical significance determined by unpaired, two-tailed Student’s t test. <strong>i</strong> Representative images of individual labeled OLs from cell-type specific CRISPR-Cas9 knock-down of <em>fbxw7</em> in <em>sox10</em> expressing cells in the zebrafish spinal cord at 6 dpf. The UAS-nlscas9-P2A-TagRFPT-caax construct allows visualization of Cas9-expressing cells. <strong>j</strong> Quantification of average sheath length and <strong>k</strong> number per OL in controls and with <em>fbxw7</em> knockdown. Mean ± SEM, Control N = 47 (cells from 40 animals), sgRNA<sup><em>fbxw7</em></sup> N = 36 (cells from 29 animals). Statistical significance determined by unpaired, two-tailed Student’s t test. Created in BioRender. Emery, B. (2024) BioRender.com/o45h010.</p></figcaption></figure><p id="Par7">In prior work, global disruption of <em>fbxw7</em> in zebrafish led to enhanced OPC specification through disinhibition of Notch1 signaling, observed as an increase in <em>mbp</em> expression and numbers of <em>olig2:dsRED</em>-expressing cells<sup><a href="#CR26" class="usa-link" aria-describedby="CR26">26</a></sup>. Consistent with this, <em>fbxw7</em><sup><em>vo86</em></sup> mutants present with a significant increase in <em>mbp:EGFP-caax</em> expression and <em>olig2:dsRED</em><sup><em>+</em></sup> cell numbers in the developing dorsal spinal cord relative to wild-type controls at 2-, 3-, and 5-days post-fertilization (dpf) (Fig. <a href="#Fig1" class="usa-link">1a–c</a>). To examine later stages of the OL lineage, we crossed <em>fbxw7</em><sup><em>vo86</em></sup> mutants into a transgenic <em>Tg(mbp-nls:eGFP)</em> line to label mature OL nuclei. Relative to wild-type clutchmates, the density of <em>mbp-nls:eGFP</em>-expressing cells was significantly increased in <em>fbxw7</em><sup><em>vo86</em></sup> mutants, even when normalized to the increased numbers of OL lineage cells (<em>olig2:dsRED</em><sup><em>+</em></sup>) at 3 dpf (Fig. <a href="#Fig1" class="usa-link">1a, d, e</a>). Additionally, we found that <em>fbxw7</em><sup><em>vo86</em></sup> mutant zebrafish displayed a significant number of neuronal cell bodies in the spinal cord wrapped in <em>mbp</em><sup>+</sup> membrane in the stable <em>Tg(mbp-nls:EGFP)</em>; <em>Tg(mbp:EGFP-caax)</em> transgenic background at 3 dpf (Fig. <a href="#Fig1" class="usa-link">1f–h</a>). While this phenotype may be due to the increase in OL numbers in the zebrafish spinal cord as previously reported<sup><a href="#CR35" class="usa-link" aria-describedby="CR35">35</a></sup>, it may also suggest that loss of Fbxw7 broadly disrupts OL myelin membrane organization and targeting. To determine whether Fbxw7 could regulate myelination via an OL intrinsic mechanism, we utilized a cell-specific CRISPR-Cas9-mediated gene disruption system our lab had previously developed<sup><a href="#CR36" class="usa-link" aria-describedby="CR36">36</a></sup>. We found that <em>sox10</em>-driven disruption of <em>fbxw7</em> in sparsely targeted OL lineage cells resulted in a significant increase in myelin sheath lengths at 6 dpf, with no change in the number of sheaths formed per individual OL (Fig. <a href="#Fig1" class="usa-link">1i–k</a>). These data suggest that Fbxw7 regulates OL myelination through a cell-autonomous mechanism in the developing zebrafish spinal cord.</p></section><section id="Sec4"><h3 class="pmc_sec_title">FBXW7 regulates OL myelin sheath length, paranodal organization, and myelin homeostasis in both gray and white matter</h3>
<p id="Par8">Our findings in zebrafish indicated that Fbxw7 regulates key aspects of myelin growth early in development. Whether this role is conserved in mammalian CNS and whether FBXW7 regulates myelination past these early developmental stages has not been explored to date. To address this, we created an inducible <em>Fbxw7</em> knockout (icKO) mouse by crossing <em>Fbxw7</em><sup><em>fl/fl</em></sup> mice<sup><a href="#CR34" class="usa-link" aria-describedby="CR34">34</a></sup> to the <em>Plp1</em>-<em>Cre</em><sup><em>ERT</em></sup> line<sup><a href="#CR37" class="usa-link" aria-describedby="CR37">37</a></sup>, allowing for tamoxifen (TAM) -inducible knockout of <em>Fbxw7</em> in mature OLs. <em>Fbxw7</em><sup><em>fl/fl</em></sup>; <em>Plp1</em>-<em>Cre</em><sup><em>ERT</em>+</sup> mice (<em>Plp1-Cre</em><sup><em>ERT</em></sup>
<em>-Fbxw7</em><sup><em>fl/fl</em></sup>) and their CreERT negative littermate controls (<em>Fbxw7</em><sup><em>fl/fl</em></sup>) were treated with TAM at 8 weeks of age, and tissue was taken 1-, 3-, and 6-months post-TAM for subsequent analyses (Fig. <a href="#Fig2" class="usa-link">2a</a>).</p>
<figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2. FBXW7 regulates OL internode length, myelin homeostasis, and paranode organization in mouse gray and white matter.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370921_41467_2025_62715_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/23a7/12370921/9b3e69fab120/41467_2025_62715_Fig2_HTML.jpg" loading="lazy" id="d33e946" height="1007" width="779" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Schematic of <em>Fbxw7</em><sup><em>fl/f</em>l</sup> and <em>Plp1-Cre</em><sup><em>ERT</em></sup><em>-Fbxw7</em><sup><em>fl/fl</em></sup> mouse lines and experimental pipeline. <strong>b</strong> Representative images of myelin internodes in <em>Fbxw7</em><sup><em>fl/f</em>l</sup> and <em>Plp1-Cre</em><sup><em>ERT</em></sup><em>-Fbxw7</em><sup><em>fl/fl</em></sup> 1-month post-tamoxifen (TAM) in mouse layer I of the primary somatosensory cortex (pSS), stained with MBP (magenta), CNP1 (green), and CASPR (cyan). Arrowheads show CASPR<sup>+</sup> boundaries of internodes. <strong>c</strong> Quantification of sheath length from layer I pSS. Colored bolded dots represent average of ROI from individual biological replicates (N = 4 mice per genotype, average of 5 ROIs/mouse), small hollow dots represent values from individual sheaths (<em>Fbxw7</em><sup><em>fl/f</em>l</sup> N = 197, <em>Plp1-Cre</em><sup><em>ERT</em></sup><em>-Fbxw7</em><sup><em>fl/fl</em></sup> N = 210). Statistical significance determined by unpaired, two-tailed Student’s t-test on ROI averages ± SEM. <strong>d</strong> Higher magnification images of myelin outfolds (arrowheads) in pSS cortex 6 months post-TAM. <strong>e</strong> Quantification of myelin outfolds in the pSS (N = 3 mice, average of 5 ROIs/mouse). Data shown as average ± SEM. Statistical significance determined by two-way ANOVA with Tukey’s multiple comparisons test. <strong>f</strong> Representative images of CASPR and ANKG stained longitudinal optic nerve sections from <em>Fbxw7</em><sup><em>fl/fl</em></sup> and <em>Plp1-Cre</em><sup><em>ERT</em></sup><em>-Fbxw7</em><sup><em>fl/fl</em></sup> mice at 6-months post-TAM. <strong>g</strong> Quantification of CASPR staining intensity as a function of distance from ANKG<sup>+</sup> node. Thick lines represent mean for each genotype with &gt; 5 individual heminodes quantified, thin lines represent individual heminode intensity histograms. Statistical significance calculated by nonparametric two-tailed Kolmogorov-Smirnov test (<em>P</em> &lt; 0.0001). <em>Fbxw7</em><sup><em>fl/f</em>l</sup> total nodes= 27, <em>Plp1-Cre</em><sup><em>ERT</em></sup><em>-Fbxw7</em><sup><em>fl/fl</em></sup> total nodes=32, averaged across 3 mice/genotype. Created in BioRender. Emery, B. (2024) BioRender.com/d36g650.</p></figcaption></figure><p id="Par9">To determine if loss of <em>Fbxw7</em> in mature OLs regulated myelin sheath maintenance, we sectioned cortical flat-mounts of layer I and performed immunofluorescence (IF) for (MBP; compact myelin), 2’,3’-cyclic nucleotide 3’ phosphodiesterase (CNP1; non-compact myelin), and contactin-associated protein (CASPR; paranodes). At 1-month post-TAM, <em>Plp1-Cre</em><sup><em>ERT</em></sup><em>-Fbxw7</em><sup><em>fl/fl</em></sup> OLs showed a significant increase in myelin sheath length in the primary somatosensory cortex (pSS) (Fig. <a href="#Fig2" class="usa-link">2b, c</a>), indicating that FBXW7 regulates myelin capacity in mature OLs. Additionally, we observed an increase in the number of MBP/CNP1<sup>+</sup> focal hyperintensities that appeared to be myelin outfolds, which increased significantly as the animals aged (Fig. <a href="#Fig2" class="usa-link">2d, e</a>). To investigate FBXW7’s role in white matter, we performed IF on <em>Plp1-Cre</em><sup><em>ERT</em></sup><em>-Fbxw7</em><sup><em>fl/fl</em></sup> optic nerves at 6-months post-TAM. We found a pronounced breakdown of nodal organization with a significant (p &lt; 0.0001) broadening in the distribution of <em>Plp1-Cre</em><sup><em>ERT</em></sup><em>-Fbxw7</em><sup><em>fl/fl</em></sup> CASPR intensity at each heminode compared to controls (Fig. <a href="#Fig2" class="usa-link">2f, g</a>). While <em>Plp1-Cre</em><sup><em>ERT</em></sup><em>-Fbxw7</em><sup><em>fl/fl</em></sup> animals had no change in weight or general health (Supplementary Fig. <a href="#MOESM1" class="usa-link">2a, b</a>), we observed a significant reduction of OL numbers in the upper cortex (layers I-VI) at 6 months post-TAM in <em>Plp1-Cre</em><sup><em>ERT</em></sup><em>-Fbxw7</em><sup><em>fl/fl</em></sup> animals relative to their age-matched control littermates (Supplementary Fig. <a href="#MOESM1" class="usa-link">2c</a>). This reduction was only observed above layer IV, with no change in OL numbers in deeper cortical layers or the corpus callosum (Supplementary Fig. <a href="#MOESM1" class="usa-link">2d, e</a>). Additionally, we found no change in OPC numbers at any of these timepoints or regions (Supplementary Fig. <a href="#MOESM1" class="usa-link">2f–h</a>), nor changes in astrocyte reactivity (Supplementary Fig. <a href="#MOESM1" class="usa-link">2i</a>). Collectively, these data suggest that FBXW7 functions in many aspects of OL biology, from early modulation of sheath lengths to long-term maintenance of paranode organization and myelin homeostasis in mammalian OLs.</p>
<p id="Par10">Given that both gray and white matter tracts in <em>Plp1-Cre</em><sup><em>ERT</em></sup><em>-Fbxw7</em><sup><em>fl/fl</em></sup> animals showed evidence of disrupted nodal organization and outfolds at 6-months post-TAM, we next wanted to assess the ultrastructure of the myelin. We therefore performed transmission electron microscopy (TEM) on optic nerves from <em>Plp1-Cre</em><sup><em>ERT</em></sup><em>-Fbxw7</em><sup><em>fl/fl</em></sup> and <em>Fbxw7</em><sup><em>fl/fl</em></sup> littermate controls 6 months post-TAM. Consistent with our observations in layer I of the cortex, we also found a significant increase in the number of myelin outfolds in <em>Plp1-Cre</em><sup><em>ERT</em></sup><em>-Fbxw7</em><sup><em>fl/fl</em></sup> optic nerves (Fig. <a href="#Fig3" class="usa-link">3a, b</a>). While control animals did have outfolds at low frequencies, as expected at 8 months of age<sup><a href="#CR25" class="usa-link" aria-describedby="CR25">25</a></sup>, the number and average length of outfolds in the <em>Plp1-Cre</em><sup><em>ERT</em></sup><em>-Fbxw7</em><sup><em>fl/fl</em></sup> was significantly higher (Fig. <a href="#Fig3" class="usa-link">3b, c</a>). Although myelin ultrastructure was disrupted in the optic nerve, we found no change in the proportion of axons myelinated or their corresponding g-ratios when severe outfolds were excluded from analyses (Fig. <a href="#Fig3" class="usa-link">3d, e</a>). Along with outfolds, we also observed other myelin abnormalities throughout the optic nerve including myelin whorls, cellular accumulations, and double myelin sheaths (sheaths enveloped by an overlying sheath) (Fig. <a href="#Fig3" class="usa-link">3f–h</a>). This double myelination was also observed by IF in layer I of the pSS cortex, where we found CASPR<sup>+</sup> paranodes under MBP<sup>+</sup> myelin sheaths in <em>Plp1-Cre</em><sup><em>ERT</em></sup><em>-Fbxw7</em><sup><em>fl/fl</em></sup> mice, suggestive of double myelinated axons (Fig. <a href="#Fig3" class="usa-link">3i</a>). The outfolds in the white matter tracts of the optic nerve and corpus callosum were so severe they were also visible at the light level by IF in <em>Plp1-Cre</em><sup><em>ERT</em></sup><em>-Fbxw7</em><sup><em>fl/fl</em></sup> animals as hyperintense MBP+ puncta (Supplementary Fig. <a href="#MOESM1" class="usa-link">3a–c</a>). Additionally, we also observed myelin sheath blebbing and outfolds in <em>fbxw7</em><sup><em>vo86</em></sup> mutant zebrafish when injected with a sox10:eGFP-caax plasmid to label individual OLs at 5 dpf (Supplementary Fig. <a href="#MOESM1" class="usa-link">3d</a>). Taken together, these data show that FBXW7 is a conserved regulator of myelin sheath length, which was independent of myelin sheath thickness, as well as long-term maintenance of myelin homeostasis and nodal organization.</p>
<figure class="fig xbox font-sm" id="Fig3"><h4 class="obj_head">Fig. 3. Loss of <em>Fbxw7</em> in OLs results in severe myelin outfolds in the optic nerve.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370921_41467_2025_62715_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/23a7/12370921/aaafe1da3646/41467_2025_62715_Fig3_HTML.jpg" loading="lazy" id="d33e1281" height="1017" width="800" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Representative image from transmission electron microscopy (TEM) micrographs of optic nerves from <em>Fbxw7</em><sup><em>fl/fl</em></sup> and <em>Plp1-Cre</em><sup><em>ERT</em></sup><em>-Fbxw7</em><sup><em>fl/fl</em></sup> mice at 6-months post-TAM. Outfolds highlighted by arrow. <strong>b–e</strong> Quantification of number of outfolds (<strong>b</strong>), severity of outfolds (<strong>c</strong>), number of myelinated axons (<strong>d</strong>), and g-ratios (<strong>e</strong>). <em>Fbxw7</em><sup><em>fl/fl</em></sup> (N = 4 mice), <em>Plp1-Cre</em><sup><em>ERT</em></sup><em>-Fbxw7</em><sup><em>fl/fl</em></sup> (N = 3 mice). <strong>f</strong> Representative images of different myelin abnormalities in <em>Plp1-Cre</em><sup><em>ERT</em></sup><em>-Fbxw7</em><sup><em>fl/fl</em></sup> optic nerves at 6-months post-TAM. <strong>g</strong>, <strong>h</strong> Quantification of OL myelin whorls in <em>Fbxw7</em><sup><em>fl/fl</em></sup> (N = 4 mice), <em>Plp1-Cre</em><sup><em>ERT</em></sup><em>-Fbxw7</em><sup><em>fl/fl</em></sup> (N = 3 mice) and double myelinated axons in <em>Fbxw7</em><sup><em>fl/fl</em></sup> (N = 3 mice), <em>Plp1-Cre</em><sup><em>ERT</em></sup><em>-Fbxw7</em><sup><em>fl/fl</em></sup> (N = 3 mice). All data displayed as mean ± SEM. Statistical significance determined by unpaired, two-tailed Student’s t test on animal averages. <strong>i</strong> Example image from pSS cortex of a CASPR<sup>+</sup>, CNP1<sup>+</sup> paranode under MBP<sup>+</sup> myelin sheath. Created in BioRender. Emery, B. (2024) BioRender.com/k49b495.</p></figcaption></figure></section><section id="Sec5"><h3 class="pmc_sec_title">Loss of <em>Fbxw7</em> in mature OLs results in ectopic ensheathment of neuronal cell bodies with myelin</h3>
<p id="Par11">Myelination within the CNS is highly targeted, with populations of axons displaying preferential degrees of myelination<sup><a href="#CR25" class="usa-link" aria-describedby="CR25">25</a>,<a href="#CR38" class="usa-link" aria-describedby="CR38">38</a>,<a href="#CR39" class="usa-link" aria-describedby="CR39">39</a></sup> and OLs typically do not myelinate structures such as blood vessels or neuronal cell bodies<sup><a href="#CR40" class="usa-link" aria-describedby="CR40">40</a>–<a href="#CR42" class="usa-link" aria-describedby="CR42">42</a></sup>. There are also precise regional borders of myelination, as occurs in the cerebellum, which has distinct myelinated (granular layer) and non-myelinated (molecular layer) layers<sup><a href="#CR43" class="usa-link" aria-describedby="CR43">43</a></sup>. Exactly how this selective process is regulated is not well understood. We found that loss of <em>Fbxw7</em> in mature OLs did not alter the tight delineation of myelination between the granular and molecular layers of the cerebellum, with the molecular layer remaining unmyelinated (Fig. <a href="#Fig4" class="usa-link">4a</a>, Supplementary Fig. <a href="#MOESM1" class="usa-link">4a</a>). Unexpectedly, however, <em>Plp1-Cre</em><sup><em>ERT</em></sup><em>-Fbxw7</em><sup><em>fl/fl</em></sup> animals displayed a significant number of granule cell bodies within the granular layer wrapped in MBP<sup>+</sup> membrane starting at 3-months post-TAM, which drastically increased at 6-months post-TAM (Fig. <a href="#Fig4" class="usa-link">4a–d</a>, and movie <a href="#MOESM3" class="usa-link">1</a>). This mistargeting of myelin appeared to be selective to the cerebellar granule cell population; we did not observe any neuronal cell bodies wrapped in myelin in the cortex, nor did we find other structures, such as blood vessels, wrapped in MBP<sup>+</sup> membrane in any regions of the CNS analyzed. Additionally, we did not observe a change in the number of OLs in our <em>Plp1-Cre</em><sup><em>ERT</em></sup><em>-Fbxw7</em><sup><em>fl/fl</em></sup> cerebellums compared to controls (Fig. <a href="#Fig4" class="usa-link">4e</a>). It is important to note that Bergman glia in the cerebellum express <em>Plp1</em> and, therefore, may have undergone <em>Fbxw7</em> recombination in our <em>Plp1-Cre</em><sup><em>ERT</em></sup><em>-Fbxw7</em><sup><em>fl/fl</em></sup> animals<sup><a href="#CR44" class="usa-link" aria-describedby="CR44">44</a></sup>. While we cannot exclude <em>Fbxw7</em> KO Bergman glia as a contributing factor in our granule cell body ensheathment, we did not observe any obvious change in the number or morphology of the Purkinje cells, which are supported by Bergman glia, at 6 months post-TAM in the cerebellum (Supplementary Fig. <a href="#MOESM1" class="usa-link">4a</a>). Additionally, we found no change in GFAP expression (expressed by Bergman glia and astrocytes in the cerebellum) or reactivity of microglia (Supplementary Fig. <a href="#MOESM1" class="usa-link">4b</a>). To assess whether the myelin ensheathment was due to myelination of the cell bodies, or severe myelin outfolds that wrapped around the small somas of the granule cells, EM was performed of the cerebellum of the 6-month post-TAM animals. High-resolution IF imaging of MBP and NeuN (Fig. <a href="#Fig4" class="usa-link">4f</a>) combined with TEM (Fig. <a href="#Fig4" class="usa-link">4g</a>), showed that the observed ensheathment of the granule cells with myelin was likely due to excessive myelin outfolds in <em>Plp1-Cre</em><sup><em>ERT</em></sup><em>-Fbxw7</em><sup><em>fl/fl</em></sup> animals. Since this ensheathment was not observed in any other area of the CNS analyzed, it is likely due to the high density and small size of the granule cell nuclei in the cerebellum. In addition to myelin outfolds, large cellular accumulations were also observed in putative Purkinje neuron axons in the <em>Plp1-Cre</em><sup><em>ERT</em></sup><em>-Fbxw7</em><sup><em>fl/fl</em></sup> animals (Fig. <a href="#Fig4" class="usa-link">4h</a>). These accumulations have been previously shown to occur in Purkinje axons after Purkinje cell-type specific deletions of cathepsin D (CTSD), which resulted in a significant increase in stressed and apoptotic Purkinje cells<sup><a href="#CR45" class="usa-link" aria-describedby="CR45">45</a></sup>. Taken together we have shown that loss of <em>Fbxw7</em> in myelinating OLs of the cerebellum resulted in large outfolds that encircled and enwrapped the cell bodies of the granule cells, while also causing axon pathology.</p>
<figure class="fig xbox font-sm" id="Fig4"><h4 class="obj_head">Fig. 4. Loss of <em>Fbxw7</em> results in ectopic ensheathment of neuronal cell bodies in the cerebellum.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370921_41467_2025_62715_Fig4_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/23a7/12370921/b63369a1237f/41467_2025_62715_Fig4_HTML.jpg" loading="lazy" id="d33e1547" height="832" width="685" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Representative images of anti-MBP stained cerebellums from <em>Fbxw7</em><sup><em>fl/fl</em></sup> and <em>Plp1-Cre</em><sup><em>ERT</em></sup><em>-Fbxw7</em><sup><em>fl/fl</em></sup> animals at 6-months post-TAM. Dotted line denotes the border of the granule layer. <strong>b</strong> Higher magnification image of MBP staining in the granular layer of a <em>Plp1-Cre</em><sup><em>ERT</em></sup><em>-Fbxw7</em><sup><em>fl/fl</em></sup> animal showing cupped myelin structures, which surround NeuN+ nuclei (shown at higher magnification in <strong>c</strong>). <strong>d</strong> Quantification of NeuN<sup>+</sup> cells wrapped in MBP positive membrane in the granular layer of the cerebellum of <em>Fbxw7</em><sup><em>fl/fl</em></sup> (1 m: N = 4, 3 m: N = 3, 6 m: N = 4) and <em>Plp1-Cre</em><sup><em>ERT</em></sup><em>-Fbxw7</em><sup><em>fl/fl</em></sup> mice (1 m: N = 4, 3 m: N = 3, 6 m: N = 4) at 1-, 3-, and 6-months post-TAM. <strong>e</strong> Quantification of CC1<sup>+</sup> OLs in the white matter and granular layer of <em>Fbxw7</em><sup><em>fl/fl</em></sup> (1 m: N = 4, 3 m: N = 3, 6 m: N = 4) and <em>Plp1-Cre</em><sup><em>ERT</em></sup><em>-Fbxw7</em><sup><em>fl/fl</em></sup> mice (1 m: N = 4, 3 m: N = 3, 6 m: N = 4) at 1-, 3-, and 6-months post-TAM. Data shown as average ± SEM. Statistical significance determined by two-way ANOVA. <strong>f</strong> Image of MBP<sup>+</sup> myelin outfold (white) around a granule cell (magenta) by IF at 6-months post-TAM reconstructed in Imaris. a and b dotted lines represent orientation of EM images in g. <strong>g</strong> Representative images of myelin outfolds around granule cell nuclei in the granule layer of the cerebellum at 6-months post-TAM. This was observed in all 4 mutant mice animals analyzed. <strong>h</strong> Images of severe cellular accumulations in the axons of putative Purkinje cells in <em>Plp1-Cre</em><sup><em>ERT</em></sup><em>-Fbxw7</em><sup><em>fl/fl</em></sup> mice at 6-months post-TAM. This was observed in all mutant animals analyzed (N = 4). Created in BioRender. Emery, B. (2024) BioRender.com/o45h010/k49b495.</p></figcaption></figure></section><section id="Sec6"><h3 class="pmc_sec_title">FBXW7 binds and degrades the N-terminus of MYRF</h3>
<p id="Par12">FBXW7 is a recognition subunit of the SKP1-Cullin-Fbox (SCF) E3 ubiquitin ligase complex. Its role is to recognize protein substrates following their phosphorylation at a phosphodegron motif, bringing them into the complex for ubiquitin tagging and subsequent proteasomal degradation<sup><a href="#CR30" class="usa-link" aria-describedby="CR30">30</a>,<a href="#CR31" class="usa-link" aria-describedby="CR31">31</a>,<a href="#CR34" class="usa-link" aria-describedby="CR34">34</a>,<a href="#CR46" class="usa-link" aria-describedby="CR46">46</a></sup>. This raised the question of which FBXW7 substrates are dysregulated in OLs after deletion of <em>Fbxw7</em> to result in accelerated and ectopic myelin formation. As a preliminary analysis, we selected a set of known FBXW7 substrates including mTOR, JNK, and cJun that are also known to regulate myelination<sup><a href="#CR27" class="usa-link" aria-describedby="CR27">27</a>,<a href="#CR28" class="usa-link" aria-describedby="CR28">28</a>,<a href="#CR30" class="usa-link" aria-describedby="CR30">30</a>,<a href="#CR47" class="usa-link" aria-describedby="CR47">47</a></sup> and screened them by western blot in si<em>Fbxw7-</em>treated rat primary OL cultures. Surprisingly, we found no detectable changes in protein levels of mTOR, p-mTOR<sup>ser2448</sup>, JNK, or cJun (Supplementary Fig. <a href="#MOESM1" class="usa-link">5a</a>). To screen for FBXW7 substrates in OLs in an unbiased manner, we designed a dominant-negative FLAG-tagged version of FBXW7 missing its F-Box domain, driven under the CMV promoter (Fig. <a href="#Fig5" class="usa-link">5a</a>). The F-Box domain of FBXW7 is required for its interaction with the SCF-E3 complex, allowing FBXW7 to disengage from its substrates<sup><a href="#CR33" class="usa-link" aria-describedby="CR33">33</a>,<a href="#CR46" class="usa-link" aria-describedby="CR46">46</a></sup>. Deletion of the F-Box domain while leaving the substrate recognition domains (WD40 repeats) intact results in a buildup of the 3xFLAG-FBXW7<sup>ΔF-Box</sup> protein bound to its substrates, allowing for effective protein-protein pulldown<sup><a href="#CR33" class="usa-link" aria-describedby="CR33">33</a></sup>. We electroporated primary rat OPCs with CMV-3xFLAG-FBXW7<sup>∆F-Box</sup> or pMax-GFP controls and differentiated them for 3 days (at which point approximately 60–70% of the Olig2<sup>+</sup> cells are MBP<sup>+</sup>). We then lysed cells, performed co-immunoprecipitation (co-IP) with an anti-FLAG antibody and assessed the eluted proteins using unbiased Liquid Chromatography-Mass Spectrometry (LC-MS) (Fig. <a href="#Fig5" class="usa-link">5b</a>). FBXW7 and MYCBP2, a known E3-independent negative regulator of FBXW7<sup><a href="#CR48" class="usa-link" aria-describedby="CR48">48</a></sup>, were the most highly enriched proteins by LC-MS, validating the effectiveness of our pulldown. We also found that FBXW7<sup>ΔF-Box</sup> bound MAP1B (a microtubule-associated protein), RAE1 (an RNA export protein), MYRF (a pro-myelination transcription factor), and MYO1D (an unconventional myosin). Of these targets, MYRF seemed the best-placed to mediate the phenotypes seen following loss of <em>Fbxw7</em>, and so it became our subsequent focus.</p>
<figure class="fig xbox font-sm" id="Fig5"><h4 class="obj_head">Fig. 5. FBXW7 binds and degrades the N-terminal MYRF.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370921_41467_2025_62715_Fig5_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/23a7/12370921/fb7e4fca081f/41467_2025_62715_Fig5_HTML.jpg" loading="lazy" id="d33e1703" height="992" width="782" alt="Fig. 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Schematic showing workflow of primary rat OPC isolation, expansion, and electroporation with either a dominant-negative Fbxw7 construct (CMV-3xFLAG-Fbxw7<sup>ΔFBox</sup>), pooled siRNAs against <em>Fbxw7</em> or non-targeting controls (siControl). <strong>b</strong> LC-MS peptide counts for significant proteins enriched by anti-FLAG pull-down in 3xFlag-Fbxw7<sup>ΔFBox</sup> electroporated cells normalized to GFP electroporated controls. Mean ± SEM, (N = 4 independent cell isolations and IPs). Statistical significance determined by multiple unpaired, two-tailed Student’s t test. <strong>c</strong> IP-western blot for MYRF following pull down of FBXW7<sup>∆FBox</sup> from cultured rat OLs at 72 h differentiation. Representative blot shown from 3 independent experiments. <strong>d</strong> Western blot analysis of the N-MYRF cleavage product in si<em>Fbxw7</em> or siCont electroporated cells at 24 or 48 h differentiation. Western blots were repeated 7 times with 5 biological replicates. <strong>e</strong> Western blot of MYRF in siRNA electroporated OLs treated with cycloheximide (CHX) for 4 h. Relative intensity of the N-MYRF band quantified in <strong>f</strong>. Mean ± SEM, N = 4 independent cell isolations. Statistical significance determined by unpaired, two-tailed Student’s t test. <strong>g</strong> Representative images of MBP and MAG expression in cultured OLs differentiated for 48 or 72 h after electroporation with siControl or si<em>Fbxw7</em>. <strong>h</strong>, <strong>i</strong> MBP+ and MAG+ OLs normalized to total OLIG2+ cells. Mean ± SEM, N = 6 independent cell isolations with 2 technical replicates (coverslips) per isolation. Statistical significance determined by two-way ANOVA. <strong>j</strong> qRT-PCR for myelin genes on siRNA treated cells. Mean ± SEM, N = 3 (independent cell isolations) with 2 technical replicates (qRT-PCR). Statistical significance determined by multiple unpaired, two-tailed Student’s t test. <strong>k</strong> Quantification of EdU incorporation in primary rat OPCs following siControl or si<em>Fbxw7</em> electroporation. Mean ± SEM, N = 3 independent cell isolations with 4 technical replicates (coverslips). Statistical significance determined by unpaired, two-tailed Student’s t test. <strong>l</strong>–<strong>n</strong> Proteins with +/- &gt; 1.2 fold change were sorted by gene ontology (GO) terms “lipid metabolism,” “myelin,” and “adhesion”. N = 4 independent cell isolations with 2 technical replicates. <strong>o</strong> Table of the top 10 enriched proteins by TMT-LS/MS in si<em>Fbxw7</em> electroporated OLs relative to siControl electroporated OLs at 3 days differentiation. Proteins with 1 or more MYRF ChIP-Seq peaks within 50 kb of the transcription start site of their corresponding gene identified based on previously published data<sup><a href="#CR50" class="usa-link" aria-describedby="CR50">50</a></sup>. Created in BioRender. Emery, B. (2024) BioRender.com/w60g354.</p></figcaption></figure><p id="Par13">MYRF is initially produced as an endoplasmic reticulum (ER)-bound transmembrane protein. It undergoes a self-cleavage event allowing the N-terminal cleavage product (N-MYRF) to translocate to the nucleus, where it works with SOX10 at the enhancers of many essential myelin genes to promote their transcription<sup><a href="#CR49" class="usa-link" aria-describedby="CR49">49</a>–<a href="#CR52" class="usa-link" aria-describedby="CR52">52</a></sup>. MYRF levels are tightly controlled within the OL lineage, and its expression is essential for OL differentiation as well as the production and maintenance of compact myelin<sup><a href="#CR17" class="usa-link" aria-describedby="CR17">17</a>,<a href="#CR53" class="usa-link" aria-describedby="CR53">53</a>–<a href="#CR55" class="usa-link" aria-describedby="CR55">55</a></sup>. Notably, MYRF has recently been reported as an FBXW7 target in hepatocarcinoma cells, suggesting the interaction may be broadly conserved across cell types<sup><a href="#CR33" class="usa-link" aria-describedby="CR33">33</a></sup>. To confirm the MYRF interaction with FBXW7 in primary OLs, we performed western blots on 3xFLAG-FBXW7<sup>∆F-Box</sup> co-IPs. Pull-down with anti-FLAG strongly enriched for endogenous N-MYRF, but not full-length MYRF (Fig. <a href="#Fig5" class="usa-link">5c</a>). To determine the effects of this interaction on MYRF levels, we electroporated primary rat OPCs with si<em>Fbxw7</em> and siControl, differentiated them for 24–48 h, and blotted for endogenous N-MYRF. Knockdown of <em>Fbxw7</em> led to a substantial increase in the levels of N-MYRF, consistent with FBXW7’s role in proteasomal degradation<sup><a href="#CR56" class="usa-link" aria-describedby="CR56">56</a></sup> (Fig. <a href="#Fig5" class="usa-link">5d</a>). To confirm that the increased N-MYRF levels were due to decreased degradation in the absence of FBXW7, we treated siControl and si<em>Fbxw7</em> OL cultures with cycloheximide (CHX) to inhibit protein translation. We found that after 4 h of CHX treatment, control cells had degraded the majority of both full-length MYRF and N-MYRF. In contrast, <em>Fbxw7</em> knockdown cells showed little reduction in N-MYRF levels, but near complete loss of the full-length protein (Fig. <a href="#Fig5" class="usa-link">5e, f</a>). Together, these findings strongly supported a role for FBXW7 in N-MYRF degradation.</p>
<p id="Par14">When analyzing the N-MYRF blots we noticed two distinct molecular weights of N-MYRF separated by ~2 kDa, with the higher molecular weight band becoming more prevalent with <em>Fbxw7</em> knockdown (Fig. <a href="#Fig5" class="usa-link">5d, e</a>). Nakayama and colleagues found that phosphorylation of N-MYRF at serine 138 and 142 by GSK3β was required for FBXW7 to interact with N-MYRF<sup><a href="#CR33" class="usa-link" aria-describedby="CR33">33</a></sup>. Additional analyses of N-MYRF at serine 138 and 142 by sequence analyses predicted CDK5 as an additional potential mediator of N-MYRF phosphorylation. To determine whether build-up of a phosphorylated form of N-MYRF accounted for the observed change in molecular weight, control and si<em>Fbxw7</em> OL lysates were treated with a lambda phosphatase, and molecular weights were evaluated by western blot (Supplementary Fig. <a href="#MOESM1" class="usa-link">5b</a>). We found that treatment with phosphatase resulted in a near total loss of the larger molecular weight N-MYRF in both control and si<em>Fbxw7</em> OLs, consistent with phosphorylated N-MYRF constituting the majority of the increased N-MYRF in our <em>Fbxw7</em> knockdown OLs. To determine if GSK3β or CDK5 were the kinase responsible for phosphorylating the phospodegron motif in N-MYRF to induce its interaction with FBXW7, we electroporated OPC cultures with pooled siRNAs against <em>Gsk3b, Cdk5</em>, or both, and differentiated them for 3 days. We found no change in the level of N-MYRF or corresponding myelin proteins (Supplementary Fig. <a href="#MOESM1" class="usa-link">5c</a>) in any of the knockdown conditions. While the GSK3β antibody showed a non-specific band around 50 kD, its predicted molecular weight band showed effective protein knockdown. Additionally, knockdown efficacy for siRNA against <em>Fbxw7</em>, <em>Gsk3b</em>, and <em>Cdk5</em> was &gt;60% as determined by qRT-PCR (Supplementary Fig. <a href="#MOESM1" class="usa-link">5d</a>). Together, these data suggest that although N-MYRF is phosphorylated in OLs and that this phosphorylated form is the target of FBXW7, this can occur through a GSK3β- and CDK5-independent mechanism.</p>
<p id="Par15">Given MYRF’s well-defined role in OL differentiation and myelination we next wanted to investigate the functional consequences of elevated MYRF levels in <em>Fbxw7</em> knockdown OLs. si<em>Fbxw7</em>-electroporated OLs differentiated for 48–72 h showed a significant increase in the proportion of MBP<sup>+</sup> and myelin-associated glycoprotein (MAG)<sup>+</sup> cells compared to controls (Fig. <a href="#Fig5" class="usa-link">5g–i</a>). In addition to an increased proportion of cells expressing myelin proteins, cultures also showed significant increases in <em>Mbp</em>, <em>Mag</em>, and <em>Plp1</em> mRNA as assessed by qRT-PCR (Fig. <a href="#Fig5" class="usa-link">5j</a>). To determine if loss of <em>Fbxw7</em> in OPCs was affected proliferation, si<em>Fbxw7</em> and control OPCs were kept in proliferation media for 2 days after siRNA electroporation to allow for effective knockdown, then pulsed with 5-ethynyl-2’-deoxyuridine (EdU) for 6 h. Within that time, approximately 40% of OPCs had undergone a round of division, with no significant change seen in EdU incorporation between si<em>Fbxw7</em> or siControl treated OPCs (Fig. <a href="#Fig5" class="usa-link">5k</a>), indicating that loss of <em>Fbxw7</em> in OPCs was not sufficient to induce <em>Myrf</em> expression and the transition to a post-mitotic OL. These results indicate that once OPCs begin to differentiate and express <em>Myrf</em>, FBXW7 serves to regulate N-MYRF protein levels to control the balance and timing of OL myelination.</p>
<p id="Par16">To further understand the consequences of loss of <em>Fbxw7</em> on the OL proteome, we performed LC-MS on lysates from si<em>Fbxw7</em> and siControl electroporated rat OLs at 3 days of differentiation. Cell lysates were labeled with tandem mass tags (TMT), pooled, and run through LC-MS. Over 2700 proteins were sequenced with an R<sup>2</sup> value of 0.99-1 within treatment groups. Select enriched proteins with validated antibodies were confirmed by western blot (Supplementary Fig. <a href="#MOESM1" class="usa-link">5e</a>). We found that <em>Fbxw7</em> knockdown in primary OLs resulted in significant changes in 253 proteins with a false discovery rate (FDR) &lt; 0.01 and 426 proteins with an FDR &lt; 0.05. Within the 253 proteins with an FDR &lt; 0.01, 158 proteins showed increased levels with <em>Fbxw7</em> knockdown and 95 showed reduced levels (Data <a href="#MOESM4" class="usa-link">S1</a>). Proteins with significant changes and a fold change greater than ±1.2 were sorted by Gene Ontology functions “lipid metabolism,” “myelin,” and “adhesion” (Fig. 5l–n) to provide a list of proteins with potential roles in FBXW7-dependent myelination.</p>
<p id="Par17">During OL differentiation, N-MYRF directly binds the enhancer regions of genes underpinning myelination, with enrichment of N-MYRF chromatin-immunoprecipitation (ChIP) peaks seen within 50 kb of the transcription start sites of genes induced during OL differentiation<sup><a href="#CR50" class="usa-link" aria-describedby="CR50">50</a></sup>. Notably, when the proteins were ranked by fold change following <em>Fbxw7</em> knockdown, 7 of the top 10 upregulated proteins and 73% of all proteins with fold change &gt;1.5 had a predicted N-MYRF binding motif within 50 kb of the transcriptional start site of their corresponding gene (Fig. <a href="#Fig5" class="usa-link">5o</a>). When we assessed RNA from corresponding samples by qPCR, we found that all the top 10 enriched proteins showed significant increases in their transcript levels with <em>Fbxw7</em> knockdown, suggesting that many of the changes in protein abundance following loss of <em>Fbxw7</em> were secondary to elevated MYRF transcriptional activity (Supplementary Fig. <a href="#MOESM1" class="usa-link">5f</a>). We observed that <em>Myrf</em> RNA was significantly down regulated in the si<em>Fbxw7</em> treated cells compared to controls over time, suggesting an additional mechanism(s) by which OLs regulate N-MYRF levels (Supplementary Fig. <a href="#MOESM1" class="usa-link">5g</a>).</p></section><section id="Sec7"><h3 class="pmc_sec_title">Loss of <em>Fbxw7</em> in OLs increases nuclear MYRF levels in vivo</h3>
<p id="Par18">We next sought to determine whether the myelin changes seen in <em>Plp1-Cre</em><sup><em>ERT</em></sup><em>-Fbxw7</em><sup><em>fl/fl</em></sup> mice may be mediated by elevated MYRF levels. Since the antibody we used to detect MYRF recognizes an epitope within the N-terminal 100 amino acids of MYRF, it recognizes both the full-length form, which is bound to the ER, and the N-terminal cleavage product, which is translocated to the nucleus<sup><a href="#CR50" class="usa-link" aria-describedby="CR50">50</a>,<a href="#CR51" class="usa-link" aria-describedby="CR51">51</a></sup>. To determine the abundance of each, we use the cellular localization of the cytoplasmic (full-length) or nuclear (N-terminal) for quantification of MYRF levels in <em>Plp1-Cre</em><sup><em>ERT</em></sup><em>-Fbxw7</em><sup><em>fl/fl</em></sup> and control optic nerve OLs. In <em>Plp1-Cre</em><sup><em>ERT</em></sup><em>-Fbxw7</em><sup><em>fl/fl</em></sup> animals, we found a significant increase in the levels of nuclear-MYRF relative to their control littermates at 1- and 3-months post-<em>Fbxw7</em> deletion (Fig. <a href="#Fig6" class="usa-link">6a, b, d</a>). Subsequently, the levels of nuclear MYRF in <em>Plp1-Cre</em><sup><em>ERT</em></sup><em>-Fbxw7</em><sup><em>fl/fl</em></sup> optic nerves returned to control levels by 6 months post-TAM. This was associated with a significant reduction in the ratio of cytoplasmic to total (nuclear + cytoplasmic) MYRF levels (Fig. <a href="#Fig6" class="usa-link">6e</a>). Hypothesizing that this reduced ratio of full-length MYRF to nuclear MYRF may represent a homeostatic response by the OL to control MYRF protein levels in the absence of FBXW7 by down-regulating <em>Myrf</em> transcription, we first performed qRT-PCR for <em>Myrf</em> transcripts in the si<em>Fbxw7</em> electroporated rat OLs in vitro, finding a significant reduction relative to siControl electroporated cells from 5 days post-knock down (Supplementary Fig. <a href="#MOESM1" class="usa-link">5g</a>). To determine whether <em>Myrf</em> mRNA was also reduced in vivo, we isolated from optic nerves from 1- and 12-months post-TAM animals. At 1-month post-TAM there was a 62.7% reduction in <em>Myrf</em> transcript levels compared to controls. This reduction persisted at 12-months post-TAM, which showed a 61.5% reduction in <em>Myrf</em> levels compared to age-matched controls (Fig. <a href="#Fig6" class="usa-link">6f</a>).</p>
<figure class="fig xbox font-sm" id="Fig6"><h4 class="obj_head">Fig. 6. FBXW7 regulates OL MYRF levels in vivo.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370921_41467_2025_62715_Fig6_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/23a7/12370921/c0de915bdf8b/41467_2025_62715_Fig6_HTML.jpg" loading="lazy" id="d33e2173" height="897" width="791" alt="Fig. 6"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Representative images of optic nerves from <em>Fbxw7</em><sup><em>fl/fl</em></sup> and <em>Plp1-Cre</em><sup><em>ERT</em></sup><em>-Fbxw7</em><sup><em>fl/fl</em></sup> mice at 1-, 3-, and 6-months post-TAM stained for MYRF. <strong>b</strong> High-resolution images showing cytoplasmic localization of MYRF in the optic nerve OLs of 6-months post-TAM animals. Arrowheads indicate cytoplasmic localization of MYRF (likely uncleaved precursor) in the <em>Fbxw7</em><sup><em>fl/fl</em></sup> control. <strong>c</strong> Representative images of cerebellums from <em>Fbxw7</em><sup><em>fl/fl</em></sup> and <em>Plp1-Cre</em><sup><em>ERT</em></sup><em>-Fbxw7</em><sup><em>fl/fl</em></sup> mice at 6-months post-TAM stained for MYRF. <strong>d</strong> Quantification of nuclear MYRF intensity in <em>Plp1-Cre</em><sup><em>ERT</em></sup><em>-Fbxw7</em><sup><em>fl/fl</em></sup> (1 m: N = 3, 3 m: N = 3, 6 m: N = 3) optic nerves normalized to controls (1 m: N = 3, 3 m: N = 3, 6 m: N = 4) at 1-, 3-, and 6-months post-TAM. Data shown as mean ± SEM. Statistical significance determined by two-way ANOVA with Tukey’s multiple comparisons test. <strong>e</strong> Ratio of cytoplasmic relative to total (cytoplasmic and nuclear) MYRF at 6-months post-TAM in <em>Fbxw7</em><sup><em>fl/fl</em></sup> (N = 4) and <em>Plp1-Cre</em><sup><em>ERT</em></sup><em>-Fbxw7</em><sup><em>fl/fl</em></sup> mice (N = 3). Data shown as mean ± SEM, statistical significance determined by unpaired, two-tailed Student’s t test. <strong>f</strong> qRT-PCR quantification of <em>Myrf</em> RNA fold change in the optic nerve from of <em>Fbxw7</em><sup><em>fl/fl</em></sup> (1 m: N = 3, 12 m: N = 3) and <em>Plp1-Cre</em><sup><em>ERT</em></sup><em>-Fbxw7</em><sup><em>fl/fl</em></sup> mice (1 m: N = 3, 12 m: N = 6) at 1- and 12- months post-TAM. Data shown as mean ± SEM. Statistical significance determined by two-way ANOVA with Šídák’s multiple comparisons test. <strong>g</strong> Quantification of nuclear MYRF intensity in <em>Plp1-Cre</em><sup><em>ERT</em></sup><em>-Fbxw7</em><sup><em>fl/fl</em></sup> cerebellums normalized to controls at 1-, 3-, and 6-months post-TAM. For both genotypes, 1 m (N = 4), 3 m (N = 4), 6 m (N = 4). Data shown as mean ± SEM. Statistical significance determined by two-way ANOVA with Tukey’s multiple comparisons test. <strong>h</strong> Quantification of NeuN<sup>+</sup> cells wrapped in MBP positive membrane in the granule cell layer of the cerebellum in 6- and 12-month post-TAMed <em>Plp1-Cre</em><sup><em>ERT</em></sup><em>-Fbxw7</em><sup><em>fl/fl</em></sup> (1 m: N = 4, 12 m: N = 3) and control animals (1 m: N = 4, 12 m: N = 3). Data shown as mean ± SEM. Statistical significance determined by two-way ANOVA with Šídák’s multiple comparisons test. <strong>i</strong> qRT-PCR quantification of <em>Myrf</em> RNA fold change in the cerebellum from <em>Fbxw7</em><sup><em>fl/fl</em></sup> (1 m: N = 4, 12 m: N = 5) and <em>Plp1-Cre</em><sup><em>ERT</em></sup><em>-Fbxw7</em><sup><em>fl/fl</em></sup> (1 m: N = 3, 12 m: N = 5) mice at 1- and 12- months post-TAM. Data shown as mean ± SEM. Statistical significance determined by two-way ANOVA with Tukey’s multiple comparisons test. Created in BioRender. Emery, B. (2024) BioRender.com/o45h010/k49b495.</p></figcaption></figure><p id="Par19">To determine if the increase in MYRF protein levels were seen throughout the CNS, we next evaluated MYRF levels in the cerebellum. We found significantly increased MYRF levels at 3- and 6-months post-TAM (Fig. <a href="#Fig6" class="usa-link">6c, g</a>). Since MYRF levels remained elevated at 6-months post-TAM in the cerebellum, we wondered if this would lead to a continued accumulation of myelin ensheathed granule cells in our aged 12-month post-TAMed animals. Indeed, we found that there was a 4-fold increase in the number of ensheathed granule cell bodies between 6- and 12-months post-TAM in <em>Plp1-Cre</em><sup><em>ERT</em></sup><em>-Fbxw7</em><sup><em>fl/fl</em></sup> animals (Fig. <a href="#Fig6" class="usa-link">6h</a>). As in the optic nerve, there was also a significant decrease in <em>Myrf</em> transcript levels in the cerebellum of <em>Plp1-Cre</em><sup><em>ERT</em></sup><em>-Fbxw7</em><sup><em>fl/fl</em></sup> animals at both 1- and 12-months post-TAM compared to age matched controls. Taken together these data demonstrate that FBXW7 negatively regulates MYRF protein levels within OLs across the life span of the animal, and that loss of FBXW7, and corresponding increases in MYRF protein levels, triggers an additional mechanism(s) that downregulates the transcription of <em>Myrf</em> in an attempt to balance the levels of this important transcription factor.</p></section><section id="Sec8"><h3 class="pmc_sec_title">
<em>Myrf</em> haploinsufficiency rescues myelin phenotype in zebrafish and mouse models</h3>
<p id="Par20">The elevation of N-MYRF protein levels in <em>Plp1-Cre</em><sup><em>ERT</em></sup><em>-Fbxw7</em><sup><em>fl/fl</em></sup> animals, combined with the significant downregulation of <em>Myrf</em> RNA in vivo and in vitro, strongly suggested that elevated N-MYRF levels may underlie the aberrant myelination seen following loss of <em>Fbxw7</em>. Since <em>Fbxw7</em> loss of function OLs appear to utilize an additional mechanism(s) to down regulate <em>Myrf</em> RNA, we wondered if crossing our <em>Plp1-Cre</em><sup><em>ERT</em></sup><em>-Fbxw7</em><sup><em>fl/fl</em></sup> mutant animals into a <em>Myrf</em> heterozygous background would reduce the burden of N-MYRF protein levels and rescue <em>Fbxw7</em> KO myelin phenotypes. To address this we crossed our <em>Plp1-Cre</em><sup><em>ERT</em></sup>-<em>Fbxw7</em><sup><em>fl/fl</em></sup> animals into the established <em>Myrf</em><sup><em>fl/fl</em></sup> line<sup><a href="#CR17" class="usa-link" aria-describedby="CR17">17</a></sup> to create <em>Plp1-Cre</em><sup><em>ERT</em></sup><em>-Fbxw7</em><sup><em>fl/fl</em></sup><em>; Myrf</em><sup><em>fl/wt</em></sup> mice line along with their CreERT negative littermate controls (<em>Fbxw7</em><sup><em>fl/fl</em></sup><em>; Myrf</em><sup><em>fl/wt</em></sup>). TAM was administered at 8 weeks of age, and tissue was harvested at 1-, 3-, and 6-months post-TAM for subsequent analyses.</p>
<p id="Par21">At 3-months post-TAM, when we observed the largest increase in MYRF protein levels in <em>Fbxw7</em> icKO optic nerves, we observed a significant decrease in MYRF intensity in <em>Plp1-Cre</em><sup><em>ERT</em></sup><em>-Fbxw7</em><sup><em>fl/fl</em></sup><em>; Myrf</em><sup><em>fl/wt</em></sup> animals compared to icKO animals (Fig. <a href="#Fig7" class="usa-link">7a, e</a>). We next evaluated Myrf levels in the cerebellum. Again, we found that <em>Plp1-Cre</em><sup><em>ERT</em></sup><em>-Fbxw7</em><sup><em>fl/fl</em></sup><em>;Myrf</em><sup><em>fl/wt</em></sup> animals had a significant decrease in MYRF staining intensity at 6-month post-TAM (Fig. <a href="#Fig7" class="usa-link">7b, f</a>). To determine if an additional floxed allele in our <em>Plp1</em>-<em>CreERT</em> animals reduced <em>Fbxw7</em> recombination, we performed qRT-PCR on the optic nerves in our <em>Plp1-Cre</em><sup><em>ERT</em></sup><em>-Fbxw7</em><sup><em>fl/fl</em></sup><em>;Myrf</em><sup><em>fl/wt</em></sup> 1-month post-TAM animals. We found no significant change in <em>Fbxw7</em> expression between <em>Fbxw7</em> icKO and <em>Plp1-Cre</em><sup><em>ERT</em></sup><em>-Fbxw7</em><sup><em>fl/fl</em></sup><em>;Myrf</em><sup><em>fl/wt</em></sup> animals at 1-month post-TAM (Supplementary Fig. <a href="#MOESM1" class="usa-link">6a</a>). We next wanted to assess how <em>Myrf</em> haploinsufficiency and normalization of MYRF protein levels would influence the myelin phenotypes observed in <em>Fbxw7</em> icKO mice. We found a significant decrease in granule cell bodies ensheathed by myelin outfolds in <em>Plp1-Cre</em><sup><em>ERT</em></sup><em>-Fbxw7</em><sup><em>fl/fl</em></sup><em>;Myrf</em><sup><em>fl/wt</em></sup> animals compared to <em>Fbxw7</em> icKO animals in the cerebellum at 6-months post-TAM (Fig.<a href="#Fig7" class="usa-link">7b, h</a>). Additionally, deletion of one allele of <em>Myrf</em> was sufficient to prevent the increase in myelin outfolds present in layer I of the pSS in <em>Fbxw7</em> icKO animals at 6-months post-TAM by IF (Fig. <a href="#Fig7" class="usa-link">7c, g</a>).</p>
<figure class="fig xbox font-sm" id="Fig7"><h4 class="obj_head">Fig. 7. <em>Myrf</em> haploinsufficiency suppresses myelin phenotypes in <em>Fbxw7</em> mutant models.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370921_41467_2025_62715_Fig7_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/23a7/12370921/73ac1dea5b76/41467_2025_62715_Fig7_HTML.jpg" loading="lazy" id="d33e2819" height="1017" width="788" alt="Fig. 7"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig7/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Representative images of optic nerves from <em>Fbxw7</em><sup><em>fl/fl</em></sup>, <em>Plp1-Cre</em><sup><em>ERT</em></sup><em>-Fbxw7</em><sup><em>fl/fl</em></sup>, and <em>Plp1-Cre</em><sup><em>ERT</em></sup><em>-Fbxw7</em><sup><em>fl/fl</em></sup><em>; Myrf</em>
<sup><em>fl/wt</em></sup> mice at 3-months post-TAM stained for MYRF. <strong>b</strong> Representative images of cerebellums from <em>Fbxw7</em><sup><em>fl/fl</em></sup>, <em>Plp1-Cre</em><sup><em>ERT</em></sup><em>-Fbxw7</em><sup><em>fl/fl</em></sup>, and <em>Plp1-Cre</em><sup><em>ERT</em></sup><em>-Fbxw7</em><sup><em>fl/fl</em></sup><em>; Myrf</em>
<sup><em>fl/wt</em></sup> mice at 6-months post-TAM stained for MYRF, MBP, and NeuN. <strong>c</strong> Representative images of myelin internodes in layer I of the pSS in <em>Fbxw7</em><sup><em>fl/fl</em></sup>, <em>Plp1-Cre</em><sup><em>ERT</em></sup><em>-Fbxw7</em><sup><em>fl/fl</em></sup>, and <em>Plp1-Cre</em><sup><em>ERT</em></sup><em>-Fbxw7</em><sup><em>fl/fl</em></sup><em>; Myrf</em>
<sup><em>fl/wt</em></sup> mice at 6-months post-TAM stained with MBP (magenta), CNP1 (green), and CASPR (cyan). <strong>d</strong> Representative images of the spinal cords of <em>fbxw7</em><sup><em>vo86/vo86</em></sup> and <em>fbxw7</em><sup><em>vo86/vo86</em></sup><em>; myrf</em><sup><em>ue70/</em>WT</sup> zebrafish on <em>Tg(mbp:eGFP-caax)</em> and <em>Tg(mbp:nls-eGFP)</em> transgenic backgrounds at 3 dpf. <strong>e</strong> Quantification of nuclear MYRF intensity in <em>Fbxw7</em><sup><em>fl/fl</em></sup> (N = 4), <em>Plp1-Cre</em><sup><em>ERT</em></sup><em>-Fbxw7</em><sup><em>fl/fl</em></sup> (N = 3), and <em>Plp1-Cre</em><sup><em>ERT</em></sup><em>-Fbxw7</em><sup><em>fl/fl</em></sup><em>; Myrf</em>
<sup><em>fl/wt</em></sup> (N = 3), normalized to controls at 3-months post-TAM in the optic nerve. Data shown as mean ± SEM. Statistical significance determined by two-way ANOVA with Tukey’s multiple comparisons test. <strong>f</strong> Quantification of nuclear MYRF intensity in <em>Fbxw7</em><sup><em>fl/fl</em></sup> (N = 4), <em>Plp1-Cre</em><sup><em>ERT</em></sup><em>-Fbxw7</em><sup><em>fl/fl</em></sup> (N = 3), and <em>Plp1-Cre</em><sup><em>ERT</em></sup><em>-Fbxw7</em><sup><em>fl/fl</em></sup><em>; Myrf</em>
<sup><em>fl/wt</em></sup> (N = 4), normalized to controls at 6- months post-TAM in the cerebellum. For both e and f, small dots represent individual cell intensities, larger dots represent animal means ± SEM with Tukey’s multiple comparisons test. Statistical significance determined by one-way ANOVA on animal means. <strong>g</strong> Quantification of myelin outfolds in layer I of the pSS in <em>Fbxw7</em><sup><em>fl/fl</em></sup> (N = 6), <em>Plp1-Cre</em><sup><em>ERT</em></sup><em>-Fbxw7</em><sup><em>fl/fl</em></sup> (N = 6), and <em>Plp1-Cre</em><sup><em>ERT</em></sup><em>-Fbxw7</em><sup><em>fl/fl</em></sup><em>; Myrf</em><sup><em>fl/wt</em></sup> (N = 4) mice at 6-months post-TAM. Data shown as mean ± SEM, each datapoint represents mean from an individual animal. Statistical significance determined by two-way ANOVA, with Tukey’s multiple comparisons test. <strong>h</strong> Quantification of NeuN<sup>+</sup> cells wrapped in MBP<sup>+</sup> membrane in the granule layer of the cerebellum of <em>Fbxw7</em><sup><em>fl/fl</em></sup> and <em>Plp1-Cre</em><sup><em>ERT</em></sup><em>-Fbxw7</em><sup><em>fl/fl</em></sup> mice at 3- and 6- months post-TAM. <em>Fbxw7</em><sup><em>fl/fl</em></sup> (3 m: N = 7, 6 m: N = 5), <em>Plp1-Cre</em><sup><em>ERT</em></sup><em>-Fbxw7</em><sup><em>fl/fl</em></sup> (3 m: N = 6, 6 m: N = 6), and <em>Plp1-Cre</em><sup><em>ERT</em></sup><em>-Fbxw7</em><sup><em>fl/fl</em></sup><em>; Myrf</em><sup><em>fl/wt</em></sup> (3 m: N = 2, 6 m: N = 4). Data shown as mean ± SEM. Statistical significance determined by two-way ANOVA with Tukey’s multiple comparisons test. <strong>i</strong> Quantification of <em>mbp:nls</em><sup><em>+</em></sup> OLs in the zebrafish dorsal spinal cord of each genotype. WT (N = 5), <em>myrf</em><sup><em>ue70/</em>WT</sup> (N = 9), <em>fbxw7</em><sup><em>vo86</em>/WT</sup> (N = 6), <em>fbxw7</em><sup><em>vo86/vo86</em></sup> (N = 6), <em>fbxw7</em><sup><em>vo86/vo86</em></sup><em>; myrf</em><sup><em>ue70/</em>WT</sup> (N = 4). Mean ± SEM, statistical significance determined by two-way ANOVA with Tukey’s multiple comparisons test.</p></figcaption></figure><p id="Par22">To test if the epistatic relationship between <em>myrf</em> and <em>fbxw7</em> is evolutionarily conserved in zebrafish OLs, we crossed the <em>fbxw7</em><sup><em>vo86/vo86</em></sup> line to a <em>myrf</em> mutant line (<em>myrf</em><sup><em>ue70</em>/WT</sup>)<sup><a href="#CR57" class="usa-link" aria-describedby="CR57">57</a></sup> to determine if reducing <em>myrf</em> levels could suppress <em>fbxw7</em> mutant phenotypes in vivo. At 3 dpf, we observed a significant reduction in the number of <em>mbp:nls-EGFP</em><sup><em>+</em></sup> OLs in the spinal cord of <em>fbxw7</em><sup><em>vo86/vo86</em></sup><em>, myrf</em><sup><em>ue70</em>/WT</sup> animals compared to <em>fbxw7</em><sup><em>vo86/vo86</em></sup> mutants alone (Fig. <a href="#Fig6" class="usa-link">6d, i</a>) as well as neuronal cell body myelin ensheathment (Supplementary Fig. <a href="#MOESM1" class="usa-link">6b</a>). These data, along with our work in primary OL cultures and conditional KO mouse models show that FBXW7 is an evolutionarily conserved negative regulator of N-MYRF in OLs required for many facets of OL biology, from OL sheath length, paranodal organization, to long-term homeostatic maintenance of myelin.</p></section></section><section id="Sec9"><h2 class="pmc_sec_title">Discussion</h2>
<p id="Par23">Here we present evidence that once cells are committed to the OL lineage, FBXW7 regulates OL myelin capacity, organization, and homeostasis, in part through the negative regulation of the N-MYRF transcription factor. With the temporal control of inducible conditional knock-out mice we found that loss of <em>Fbxw7</em> in myelinating OLs resulted in increased myelin sheath length, severe myelin outfolds, disorganized paranodes, and surprisingly, wrapping of neuronal cell bodies in the cerebellum by severe outfolds. These phenotypes are particularly striking since <em>Fbxw7</em> was targeted in mature OLs, indicating that inactivation of FBXW7 reinitiates aspects of myelin growth in the adult CNS. While myelin sheath plasticity has been reported in the context of axonal activity<sup><a href="#CR7" class="usa-link" aria-describedby="CR7">7</a>,<a href="#CR36" class="usa-link" aria-describedby="CR36">36</a>,<a href="#CR58" class="usa-link" aria-describedby="CR58">58</a></sup>, the underlying mechanisms that regulate these changes have not been fully characterized. Because loss of <em>Fbxw7</em> results in changes to many adhesion and cytoskeletal proteins, it is possible that FBXW7-mediated negative regulation of OL proteins may represent one of these underlying mechanisms of plasticity, potentially through its regulation of N-MYRF and its transcriptional targets.</p>
<p id="Par24">Ensheathment of neuronal cell bodies occurs in zebrafish when OL numbers exceed their normal balance to receptive axons<sup><a href="#CR35" class="usa-link" aria-describedby="CR35">35</a></sup>. It is entirely possible that the increase in ensheathed neuronal cell bodies we observe in the <em>fbxw7</em><sup><em>vo86</em></sup> zebrafish is a consequence of increased OPC specification and OL numbers<sup><a href="#CR26" class="usa-link" aria-describedby="CR26">26</a></sup>. In contrast, myelin outfolds that ensheathed the cerebellar granule cell bodies of <em>Plp1-Cre</em><sup><em>ERT</em></sup><em>-Fbxw7</em><sup><em>fl/fl</em></sup> mice were not accompanied by an increase in the density of OLs, so are unlikely to be mediated by a mismatch between the myelinating cells and their targets. We did not observe myelin outfolds wrapping cell bodies or other structures in the cerebral cortex of <em>Plp1-Cre</em><sup><em>ERT</em></sup><em>-Fbxw7</em><sup><em>fl/fl</em></sup> animals, although outfolds were present. Why cerebellar granule cell bodies were the only observed neurons wrapped in myelin outfolds in the <em>Fbxw7</em> conditional knockouts is puzzling. Perhaps the density and/or size of cerebellar granule cells make them especially susceptible to ensheathment by redundant myelin outfolds. Indeed, ensheathment of granule cell bodies has also been observed in the normal development of the toad<sup><a href="#CR59" class="usa-link" aria-describedby="CR59">59</a></sup> and squirrel monkey cerebellum<sup><a href="#CR60" class="usa-link" aria-describedby="CR60">60</a></sup>, following N-WASP inactivation within the OL lineage cells<sup><a href="#CR61" class="usa-link" aria-describedby="CR61">61</a></sup>, and with over-expression of the extracellular domain of Cadm4 adhesion protein in OLs<sup><a href="#CR62" class="usa-link" aria-describedby="CR62">62</a></sup>. While aberrant cell body myelination has also been observed in the spinal cord of global KO of myelin-guidance protein JAM2, this was believed to be driven by neuronal loss of JAM2. Since the granule layer of the cerebellum is the primary region where cell body myelination is observed when OLs are targeted, it suggests that the unique architecture of this region is highly prone to ectopic myelin placement. Additionally, our findings that <em>Fbxw7</em> loss of function myelin causes putative Purkinje axon pathology also leads to the interesting question of how inappropriate myelin may affect the health and function of the underlying neurons and circuitry.</p>
<p id="Par25">In Schwann cells, FBXW7 regulates myelin sheath thickness, with no obvious change to sheath length<sup><a href="#CR28" class="usa-link" aria-describedby="CR28">28</a></sup>. In OLs these roles seem to be reversed, with FBXW7 regulating myelin sheath length but not thickness. Although this may be due to inherent differences in the biology of these two cell types, it may also be due to the differences in the tools used to evaluate its function. In our prior Schwann cell work, <em>Fbxw7</em> was constitutively deleted in development using Desert hedgehog Cre (<em>Dhh-Cre</em>), which is expressed in Schwann cell precursors as early as E12.5<sup><a href="#CR28" class="usa-link" aria-describedby="CR28">28</a>,<a href="#CR63" class="usa-link" aria-describedby="CR63">63</a></sup>. In contrast, here we used a TAM-inducible system to delete <em>Fbxw7</em> from myelinating OLs in 8-week-old <em>Plp1-CreERT</em> mice. Whether the effects of <em>Fbxw7</em> on myelination would change depending on the timing of OL deletion remains unclear but is an exciting proposition for future work.</p>
<p id="Par26">Our studies and others highlight the complicated role FBXW7 plays in myelinating cell biology across species and cell types. <em>Fbxw7</em> is widely expressed in most, if not all cell types in the CNS, and its biological functions depend on the available substrates within each cell<sup><a href="#CR46" class="usa-link" aria-describedby="CR46">46</a>,<a href="#CR56" class="usa-link" aria-describedby="CR56">56</a>,<a href="#CR64" class="usa-link" aria-describedby="CR64">64</a></sup>. Therefore, FBXW7 is likely to have distinct targets and diverging roles at different stages of the OL lifespan. For instance, FBXW7 negatively regulates NOTCH levels in neural precursor cells (NPCs) to control OPC specification in zebrafish<sup><a href="#CR26" class="usa-link" aria-describedby="CR26">26</a></sup>. The same group also found that in later stages of the OL lineage, FBXW7 negatively regulates mTOR to control myelination in the spinal cord of zebrafish. Likewise, we previously showed that in the mammalian PNS, FBXW7 regulates early Schwann cell numbers, axonal ensheathment, and myelin thickness in an mTOR-dependent manner<sup><a href="#CR28" class="usa-link" aria-describedby="CR28">28</a></sup>. In contrast, here we present evidence that in mammalian OLs, mTOR was not a direct target of FBXW7, with knockdown of <em>Fbxw7</em> resulting in no detectable change in mTOR protein abundance, phosphorylation, or downstream signaling. Using an unbiased pull-down and LC-MS approach, we instead identified several direct FBXW7-interacting proteins in primary OLs: MYCBP2, MAP1B, RAE1, and MYRF. Of these, MYCBP2 was a previously identified FBXW7 interactor that likely inhibits its activity independent of the E3 complex<sup><a href="#CR48" class="usa-link" aria-describedby="CR48">48</a></sup>. Although we focus here on MYRF, MAP1B and RAE1 also represent intriguing FBXW7 targets within myelinating cells for future studies.</p>
<p id="Par27">As a critical regulator of OL differentiation and myelination, MYRF levels appear to be tightly regulated within the OL lineage. Not only is MYRF subject to tight transcriptional regulation by SOX10 and ZFP24<sup><a href="#CR65" class="usa-link" aria-describedby="CR65">65</a></sup>, but its mRNA is subject to negative regulation by miR-145-5p in OPCs, presumably to discourage premature differentiation<sup><a href="#CR66" class="usa-link" aria-describedby="CR66">66</a></sup>. While we have shown that loss of FBXW7 results in an increase in the abundance of the N-terminal MYRF cleavage product, this increase in protein levels seems to have engaged a compensatory mechanism resulting in a decrease in <em>Myrf</em> transcript levels, presumably in an attempt to return MYRF to normal levels. Our MS data from <em>Fbxw7</em> knock-down cells showed that ZFP24 was significantly downregulated compared to controls, presenting a possible link to the change in <em>Myrf</em> transcription<sup><a href="#CR65" class="usa-link" aria-describedby="CR65">65</a></sup> and an unexplored mechanism that OLs may use to regulate <em>Myrf</em> mRNA levels and myelin homeostasis. Additionally, while we showed <em>Myrf</em> mRNA was significantly downregulated in both the ON and cerebellum at later timepoints, it’s interesting to note that this transcriptional downregulation was only sufficient to reduce nuclear MYRF protein levels in the ON but not the cerebellum. This raises an intriguing question: does the underlying physiology of the myelinated circuit affect OLs requirement and expression of MYRF? While we have shown that FBXW7 binds N-MYRF to control its levels both in vitro and in vivo, this does not preclude the possibility that additional mechanisms are involved in MYRF protein stability, cleavage, and function.</p>
<p id="Par28">The full-length MYRF protein trimerizes and self-cleaves to enable release of the active transcription factor<sup><a href="#CR50" class="usa-link" aria-describedby="CR50">50</a>,<a href="#CR51" class="usa-link" aria-describedby="CR51">51</a>,<a href="#CR67" class="usa-link" aria-describedby="CR67">67</a></sup>. Although this process is negatively regulated by TMEM98, the protein product of one of MYRF’s own target genes<sup><a href="#CR67" class="usa-link" aria-describedby="CR67">67</a>,<a href="#CR68" class="usa-link" aria-describedby="CR68">68</a></sup>, the degree to which TMEM98 negatively regulates the production of the N-MYRF transcription factor at endogenous levels remains unclear. FBXW7 negatively regulates N-MYRF levels in primary OLs, corroborating the finding that FBXW7 targets N-MYRF in mHepa cells<sup><a href="#CR33" class="usa-link" aria-describedby="CR33">33</a></sup> and suggesting a conserved regulatory mechanism. Indeed, the fact that myelin and OL abnormalities could be suppressed in <em>Plp1-Cre</em><sup><em>ERT</em></sup><em>-Fbxw7</em><sup><em>fl/fl</em></sup> mice and <em>fbxw7</em><sup><em>vo86</em></sup> fish by <em>Myrf</em> haploinsufficiency highlights N-MYRF as a central FBXW7 target within the OL lineage. In contrast to mHepa cells, however, in primary OLs, the FBXW7 interaction does not seem to depend on the phosphorylation of the phosphodegron motif by GSK3β or CDK5. The intracellular pathways and mediating kinases that initiate N-MYRF turnover by FBXW7 will be important to determine in future work.</p>
<p id="Par29">Notably, the dysregulated proteins seen in <em>Fbxw7</em> knockdown OLs included many cell adhesion and cell surface proteins. Previous research has highlighted the connection between adhesion molecules and proper myelination. For instance, when contactin-associated protein (<em>Caspr</em>), contactin-1 (<em>Cntn1</em>), neurofascin (<em>Nfasc155</em>), and myelin-associated glycoprotein (<em>Mag</em>) were globally deleted in both the zebrafish and mouse CNS, similar myelin phenotypes were observed as in <em>Plp1-Cre</em><sup><em>ERT</em></sup><em>-Fbxw7</em><sup><em>fl/fl</em></sup> animals<sup><a href="#CR69" class="usa-link" aria-describedby="CR69">69</a></sup>. Specifically, loss of these adhesion proteins resulted in outfolds, double myelinated axons, and disorganized paranodal loops as well as neuronal cell body ensheathment in the zebrafish spinal cord. It is exciting that within our model, where <em>Fbxw7</em> deletion is restricted to myelinating OLs, we see such similar phenotypes when compared to global disruption of adhesion proteins in the CNS.</p>
<p id="Par30">In summary, we have shown that FBXW7 is an evolutionarily conserved regulator of OL myelin capacity and homeostasis. We found that FBXW7 regulates myelination by controlling sheath elongation independent of myelin wraps and is required for long-term maintenance of myelin integrity and paranodal organization within the adult CNS, in part, through its negative regulation of N-MYRF.</p></section><section id="Sec10"><h2 class="pmc_sec_title">Methods</h2>
<section id="Sec11"><h3 class="pmc_sec_title">Zebrafish husbandry</h3>
<p id="Par31">All zebrafish experiments were performed in compliance with the institutional ethical regulations for animal testing and research at Oregon Health &amp; Science University (OHSU, protocol TR02_IP00001148). <em>fbxw7</em><sup><em>stl64</em></sup>, <em>fbxw7</em><sup><em>vo86</em></sup>, and <em>myrf</em><sup><em>ue70</em></sup> zebrafish were maintained as heterozygotes. Experimental larvae were generated by incrosses to yield wild-type, heterozygous, and homozygous zebrafish. To create <em>fbxw7</em><sup><em>vo86/vo86</em></sup><em>; myrf</em><sup><em>ue70</em>/WT</sup> animals, <em>fbxw7</em><sup><em>vo86/</em>WT</sup>; <em>myrf</em><sup><em>ue70</em>/WT</sup> zebrafish were outcrossed to <em>fbxw7</em><sup><em>vo86/</em>WT</sup>. Zebrafish larvae are fed a diet of rotifers and dry food (Gemma 75) from 5 days post-fertilization (dpf) until 21 dpf. From 21 dpf until 3 months, fish are fed using rotifers and dry food (Gemma 150). Adult fish are maintained and fed with brine shrimp and dry food (Gemma 300). For larval zebrafish studies, sex cannot be considered as a biological variable as sex has not yet been determined.</p>
<section class="tw xbox font-sm" id="Taba"><div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups"><tbody>
<tr><td align="left" colspan="1" rowspan="1">Zebrafish transgenic lines</td></tr>
<tr><td align="left" colspan="1" rowspan="1">
<em>Tg(mbp:eGFP-caax)</em><sup><a href="#CR70" class="usa-link" aria-describedby="CR70">70</a></sup>
</td></tr>
<tr><td align="left" colspan="1" rowspan="1">
<em>Tg</em>(<em>olig2:dsRED)</em><sup><a href="#CR71" class="usa-link" aria-describedby="CR71">71</a></sup>
</td></tr>
<tr><td align="left" colspan="1" rowspan="1">
<em>Tg(mbp-nls:eGFP)</em><sup><a href="#CR72" class="usa-link" aria-describedby="CR72">72</a></sup>
</td></tr>
<tr><td align="left" colspan="1" rowspan="1">
<em>Tg(nbt:dsRED)</em><sup><a href="#CR73" class="usa-link" aria-describedby="CR73">73</a></sup>
</td></tr>
</tbody></table></div>
<div class="p text-right font-secondary"><a href="table/Taba/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section></section><section id="Sec12"><h3 class="pmc_sec_title">Generation of <em>fbxw7</em><sup><em>vo86</em></sup> zebrafish mutants</h3>
<p id="Par32">CRISPR/Cas9-mediated genome editing was used to generate genetic mutants in zebrafish. The CHOPCHOP web tool<sup><a href="#CR74" class="usa-link" aria-describedby="CR74">74</a></sup> was used to select target sites, and individual sgRNAs were synthesized using the MEGAshortscript T7 Transcription kit (Thermo Fisher). The sgRNA 5-GATGTAATCCGTCGTCTCTGTGG-3 was mixed with Cas9 Nuclease (Integrated DNA Technologies) to a final concentration of 50 ng/mL sgRNA and 1 mg/mL of Cas9 protein and injected into one-celled zygotes at a volume of 1-2 nL. Progeny of injected F0 generation animals were screened for the presence of inherited indels resulting in frameshifts or truncations by PCR, and these F1 progenies were used to establish stable mutant lines. Genotyping for both larval and adult zebrafish was performed by digesting tissue in Tris buffer with Proteinase-K overnight at 55 °C. PCR was performed with GoTaq DNA Polymerase (Promega, M300A). For <em>fbxw7</em><sup><em>vo86</em></sup> genotyping, PCR with (F 5-AAAATAGGGGCTTGCTCTGG-3, R 5-AAGTCCAGTTAAATTGAGAAGCC-3) was used to amply a 530 bp region around indels. PCR products were digested with 10U of BsmBI-v2 (NEB, R0580) at 55 °C overnight and resolved on a 2% agarose gel. The uncleaved <em>fbxw7</em><sup><em>vo86</em></sup> product size was 530 bp, and WT cleaved product sizes were 294 and 236 bp.</p></section><section id="Sec13"><h3 class="pmc_sec_title">Zebrafish mosaic cell labeling</h3>
<p id="Par33">For mosaic labeling of oligodendrocytes (OLs), <em>fbxw7</em><sup><em>vo86/</em>WT</sup> zebrafish were incrossed, and fertilized one-cell zygotes were injected with 1–2 nl of a solution containing 10 ng of sox10:EGFP-caax plasmid, 25 ng of Tol2 transposase mRNA, 0.02% phenol red and 0.2 M KCl. Embryos were genotyped after imaging as described above.</p></section><section id="Sec14"><h3 class="pmc_sec_title">Cell-type specific CRISPR-Cas9 gene disruption in zebrafish</h3>
<p id="Par34">For cell-type specific CRISPR-Cas9-mediated gene disruption we utilized methods as previously described<sup><a href="#CR36" class="usa-link" aria-describedby="CR36">36</a></sup>. Briefly, sgRNAs targeting <em>fbxw7</em> exon 5 (5-GATGTAATCCGTCGTCTCTGTGG-3) and exon 7 (5-GCTGCCTGAAGCAGATCCTTTGG-3) were cloned into 10xUAS:myrmScarlet-p2A-Cas9, U6:sgRNA1;U6:sgRNA2 backbones and injected into <em>Tg(sox10:Kalta4)</em><sup><a href="#CR75" class="usa-link" aria-describedby="CR75">75</a></sup> fertilized embryos at the one-cell stage. Empty backbones were used as controls. At desired timepoints, fish were anesthetized with 600 μM tricaine (TRS5, Pentair), screened for fluorescence, embedded laterally in 1.5% low-melting-point agarose (A9414, Sigma), and imaged with a 20x dipping objective on a ZEISS LSM 980 with Airyscan 2. Sheaths were analyzed using ImageJ<sup><a href="#CR36" class="usa-link" aria-describedby="CR36">36</a></sup>.</p></section><section id="Sec15"><h3 class="pmc_sec_title">Mouse husbandry and tamoxifen (TAM) administration</h3>
<p id="Par35">All mice were housed in OHSU animal facilities, maintained in a pathogen-free temperature and humidity-controlled environment on a 12-h light/dark cycle. All procedures were approved by the OHSU Institutional Animal Care and Use Committee (protocols TR01_IP00001328 and TR02_IP00001148). Fbxw7<sup>fl/fl</sup> mice<sup><a href="#CR34" class="usa-link" aria-describedby="CR34">34</a></sup> were previously purchased from Jackson Laboratories (B6;129-<em>Fbxw7</em><sup><em>tm1Iaai</em></sup>/J, JAX: 017563) and crossed to Plp1-Cre<sup>ERT</sup> mice (B6.Cg-Tg[Plp1-cre/ERT]3Pop/J, JAX: 005975). <em>Myrf</em><sup><em>fl/fl</em></sup> mice<sup><a href="#CR17" class="usa-link" aria-describedby="CR17">17</a></sup>, maintained on a C57BL/6 N background, were crossed into the <em>Plp1-CreERT</em>-<em>Fbxw7</em><sup><em>fl/fl</em></sup> animals to create <em>Plp1-Cre</em><sup><em>ERT</em></sup><em>-Fbxw7</em><sup><em>fl/fl</em></sup><em>;Myrf</em><sup><em>fl/wt</em></sup> animals. Cre<sup>ERT</sup> negative littermates served as controls. Genotypes were determined by PCR analysis using established primers for each line and were revalidated at experimental endpoints. Genotyping primers below. All experiments were conducted in both male and female mice. For TAM injection, 8-week-old mice were dosed with 100 mg/kg tamoxifen (Sigma T5648, dissolved at 20 mg/ml in corn oil) for five consecutive days via intraperitoneal injection.</p>
<section class="tw xbox font-sm" id="Tabb"><div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups"><tbody>
<tr>
<td align="left" colspan="1" rowspan="1">Gene</td>
<td align="left" colspan="1" rowspan="1">5-Forward primer</td>
<td align="left" colspan="1" rowspan="1">Reverse primer-3</td>
<td align="left" colspan="1" rowspan="1">Size</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>Fbxw7</em><sup><em>fl/fl</em></sup>
</td>
<td align="left" colspan="1" rowspan="1">GGCTTAGCATATCAGCTATGG</td>
<td align="left" colspan="1" rowspan="1">ATAGTAATCCTCCTGCCTTGGC</td>
<td align="left" colspan="1" rowspan="1">497 bp</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>Fbxw7</em><sup><em>wt/wt</em></sup>
</td>
<td align="left" colspan="1" rowspan="1">GGCTTAGCATATCAGCTATGG</td>
<td align="left" colspan="1" rowspan="1">ATAGTAATCCTCCTGCCTTGGC</td>
<td align="left" colspan="1" rowspan="1">315 bp</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>Plp1-Cre</em><sup><em>ERT</em></sup>
</td>
<td align="left" colspan="1" rowspan="1">ATACCGGAGATCATGCAAGC</td>
<td align="left" colspan="1" rowspan="1">GGCCAGGCTGTTCTTCTTAG</td>
<td align="left" colspan="1" rowspan="1">200 bp</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Cre control</td>
<td align="left" colspan="1" rowspan="1">CTAGGCCACAGAATTGAAAGATCT</td>
<td align="left" colspan="1" rowspan="1">GTAGGTGGAAATTCTAGCATCATCC</td>
<td align="left" colspan="1" rowspan="1">324 bp</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>Myrf</em><sup><em>wt/wt</em></sup>
</td>
<td align="left" colspan="1" rowspan="1">GGGAGGGGGCTTCAAGGAGTGT</td>
<td align="left" colspan="1" rowspan="1">CCCCCAGCATGCCGATGTACAC</td>
<td align="left" colspan="1" rowspan="1">460 bp</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>Myrf</em><sup><em>fl/fl</em></sup>
</td>
<td align="left" colspan="1" rowspan="1">GGGAGGGGGCTTCAAGGAGTGT</td>
<td align="left" colspan="1" rowspan="1">CCTTTCGCCAGGGGGATCTTG</td>
<td align="left" colspan="1" rowspan="1">668 bp</td>
</tr>
</tbody></table></div>
<div class="p text-right font-secondary"><a href="table/Tabb/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section></section><section id="Sec16"><h3 class="pmc_sec_title">Tissue processing</h3>
<p id="Par36">Mice were terminally anaesthetized with ketamine (400 mg/kg) and xylazine (60 mg/kg) before being transcardially perfused with 20 mL of phosphate buffered saline (PBS) and 40 mL of 4% paraformaldehyde (19210, Electron Microscopy Sciences) in PBS. For immunofluorescence (IF), tissues were post-fixed in 4% paraformaldehyde in PBS (2 h for optic nerves, overnight for brains) at 4 °C and cryopreserved in 30% sucrose for at least 72 h at 4 °C. Cortical flat mounts were processed as previously described<sup><a href="#CR39" class="usa-link" aria-describedby="CR39">39</a></sup>. Cryopreserved tissue was embedded in OCT (4583, Sakura), frozen on dry ice, and stored at −80 °C until sectioning on a cryostat (Leica CM3050-S). Cryosections (12 μm thickness for brain, 16 µm for optic nerve) were mounted on Superfrost Plus slides (1255015, Fisher Scientific) and stored at −80 °C. Tissue for transmission electron microscopy was post-fixed in 2% paraformaldehyde (15710, Electron Microscopy Sciences) with 2% glutaraldehyde (16310, Electron Microscopy Sciences).</p></section><section id="Sec17"><h3 class="pmc_sec_title">Immunofluorescence</h3>
<p id="Par37">Slides stored in −80 °C were air dried for at least 2 h before being rehydrated in 1x PBS. For MBP staining, tissue was delipidated by treating slides with ascending and descending ethanol solutions (50%, 75%, 85%, 95%, 100%) before being washed 3x in 1x PBS. Slides were blocked for 1 h at room temperature with 10% fetal calf serum (SH30910.03, Cytiva) with 0.2% Triton X-100 (10789704001, Sigma). Primary antibodies were applied overnight in 1x PBS, 5% fetal calf serum and 0.2% Triton X-100 in a sealed humidified container at room temperature. The following primary antibodies were used: chicken anti-MBP (1:500; MBP, Aves), mouse anti-CC1 monoclonal (1:500; OP80, Millipore), goat anti-PDGFRα (1:500; AF1062, R&amp;D Systems), rabbit anti-Iba1 (1:1000; 019-19741, Wako), rabbit anti-GFAP (1:1000; Z0334, Dako), rabbit anti-MYRF (1:500; A16355, ABclonal), mouse anti-Calbindin1 (1:500; C9848, Sigma). Following incubation with primary antibodies, slides were washed 3x in 1 × 0.2% Triton X-100 PBS before appropriate Alexa Fluor 488, 555 or 647 secondary antibodies (1:1,000; Invitrogen) were applied for one h at room temperature. Slides were then again washed 3x with 1 × 0.2% Triton X-100 PBS, washed in Milli-Q H<sub>2</sub>O, air dried, then coverslipped with Fluoromount G (0100-01, Southern Biotech). Images were acquired with in 2 weeks.</p>
<p id="Par38">Primary rat OLs were cultured on glass coverslips, fixed for 8 min in 4% PFA in PBS and stained in 24-well plates as described above. We used the following primary antibodies: chicken anti-MBP (1:500; MBP, Aves), rabbit anti-OLIG2 (1:500; AB9610, Millipore), and mouse anti-MAG (1:500; AB1567, Millipore). Coverslips were mounted with ProLong Diamond (<a href="https://www.ncbi.nlm.nih.gov/protein/P36965" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">P36965</a>, Thermo Fisher) on Superfrost Plus slides.</p>
<p id="Par39">Cortical flatmounts were sectioned at 40 μm and stored in PBS with 0.02% NaN<sub>3</sub> (sodium azide) at 4 °C for 1 week–2 months. For staining, tissue was blocked in 10% fetal calf serum with 0.2% Triton X-100 for 2 h with agitation at room temperature. Flatmounts were then incubated with primary antibodies at room temperature with agitation in 1x PBS 0.2% Triton X-100 for 4 days. We used the following primary antibodies: chicken anti-MBP (1:200; MBP, Aves), rabbit anti-CASPR (1:500; 34151-001, Abcam), and mouse anti-CNPase1 (1:500; MAB326, Millipore). Tissue was washed 3 × 20 min 1 × 0.2% Triton X-100 PBS. Alexa Fluor 488, 555 and 647 secondary antibodies (1:1,000; Invitrogen) were applied for two days at 4 °C protected from light. Tissue was washed 3 × 20 min 1 × 0.2% Triton X-100 PBS, slide mounted, rinsed in water, air dried, and coverslipped with Fluoromount G (0100-01, Southern Biotech). For quantification of sheath length from layer I pSS, 280 μm × 280 μm × 30 μm images were taken of the pSS and 2 random ROIs were generated. All sheaths that passed through the ROIs were measured using ImageJ NeuroTracer in 3D.</p>
<p id="Par40">Immunofluorescence from tissue was acquired on a ZEISS LSM 980 with Airyscan 2. Immunofluorescence from cultured OLs were imaged on Zeiss ApoTome2 at 20x. All cell counts and fluorescence intensities were quantified using ImageJ.</p></section><section id="Sec18"><h3 class="pmc_sec_title">Isolation and electroporation of primary rat OPCs</h3>
<p id="Par41">Rat OPCs were isolated from P6-8 Sprague Dawley rat pups as previously described<sup><a href="#CR76" class="usa-link" aria-describedby="CR76">76</a></sup>. OPCs from each animal were expanded in 3 × 175 cm<sup>2</sup> flasks for 3–4 days in the presence of 10 ng/mL platelet derived growth factor-aa (PDGFAA, Peprotech 100-13 A). Cells were harvested fresh for each round of experiments and used at the time of first passage. OPCs were electroporated with Amaxa Basic Nucleofector Kit for Primary Mammalian Glial Cells (VPI-1006, Lonza) with 20 nM siRNAs/5 million OPCs or 4 μg of plasmid/5 million OPCs. siRNA pools for rat <em>Fbxw7</em> (L-115782-00-0005, Horizon), <em>Gsk3b</em> (L-080108-02-0005, Horizon), <em>Cdk5</em> (L-094624-02-0005, Horizon), or non-targeting controls (D-001810-10-05, Horizon) were used. Primary rat OLs were plated at 20k/coverslip for staining, 250k cells/well of a 6-well plate for RNA, and 1 million cells/60 × 15 mm plate for protein isolation.</p></section><section id="Sec19"><h3 class="pmc_sec_title">EdU incorporation and cycloheximide (CHX) treatment</h3>
<p id="Par42">Primary rat OPCs were expanded and electroporated with siRNAs and replated into proliferation media containing PDGFAA for 48 h. Cells were pulsed with 10 μM 5-ethynyl-2’-deoxyuridine (EdU) for 6 hours. Cells were fixed with 4% PFA in PBS for 8 min at room temperature. Cells were stained with Click-iT EdU Cell Proliferation Kit with Alexa Fluor 647 dye (<a href="https://www.ncbi.nlm.nih.gov/nuccore/C10340" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">C10340</a>, Thermo Fisher). siRNA electroporated OLs were differentiated for 3 days and treated with cycloheximide (CHX; 239763-M, Sigma) at 100 μg/mL for 4 hours to stop protein translation. Cell lysates were processed for Western blot analyses as described below.</p></section><section id="Sec20"><h3 class="pmc_sec_title">Transmission electron microscopy (TEM)</h3>
<p id="Par43">Following post-fixation in 2% paraformaldehyde (15710, Electron Microscopy Sciences) with 2% glutaraldehyde (16310, Electron Microscopy Sciences), optic nerves and cerebellums were stored in a buffer of 1.5% paraformaldehyde, 1.5% glutaraldehyde, 50 mM sucrose, 22.5 mM CaCl<sub>2</sub> 2H<sub>2</sub>O in 0.1 M cacodylate buffer for at least seven days. Tissue was then infiltrated with 2% osmium tetroxide (19190, Electron Microscopy Sciences) using a Biowave Pro+ microwave (Ted Pella)<sup><a href="#CR54" class="usa-link" aria-describedby="CR54">54</a></sup> before dehydration in acetone and embedding in Embed 812 (14120, Electron Microscopy Sciences). 0.4 µm sections were cut on an ultramicrotome and stained with 1% Toluidine Blue (T161-25, Fisher Scientific) with 2% sodium borate (21130, Electron Microscopy Sciences). 60 nm sections were mounted on copper grids (FCF100-Cu-50, Electron Microscopy Sciences) and counterstained with UranyLess for 5 min followed by 3% lead citrate (22409, 22410, Electron Microscopy Sciences) for 5 min. Grids were imaged at 4800x on an FEI Tecnai T12 transmission electron microscope with a 16 Mpx camera (Advanced Microscopy Techniques Corp). For g-ratio analysis, 5–8 images per animal were used. Outer myelin and axon diameters for g-ratio analyses were manually traced using ImageJ.</p></section><section id="Sec21"><h3 class="pmc_sec_title">Cloning of dominant-negative 3xFLAG- FBXW7<sup>∆F-Box</sup>
</h3>
<p id="Par44"><em>Fbxw7</em> coding sequence missing the F-Box domain with upstream 3x FLAG tags (3xFLAG- Fbxw7<sup>∆F-Box</sup>) was purchased as a double stranded gBlocks gene fragment from Integrated DNA Technologies (IDT) with KpnI + HindIII restriction enzyme overhangs. pCMV-SPORT6 backbone and 3xFLAG- FBXW7<sup>∆F-Box</sup> inserts were digested with KpnI + HindIII restriction enzymes, gel purified, and ligated with T4 DNA ligase (EL0011, Thermo Fisher). Constructs were transformed into DH5α one-shot competent cells (12297016, Thermo Fisher) and purified with a PureLink HiPure plasmid maxiprep kit (K0491, ThermoFisher). pmaxGFP vector (Lonza) was used as a control for co-IP experiments.</p></section><section id="Sec22"><h3 class="pmc_sec_title">Immunoprecipitation of 3xFLAG- FBXW7<sup>∆F-Box</sup> in cultured OLs</h3>
<p id="Par45">4 μg of CMV-3xFLAG-FBXW7<sup>∆F-Box</sup> and pmaxGFP were electroporated into rat OPCs. Cells were differentiated for 3 days and then lysed with cell lysis buffer (20 mM Tris pH7.5, 150 mM NaCl, 1% Triton X-100, 1 mM EDTA, 1 mM EGTA) with cOmplete Mini Protease Inhibitor Cocktail (11836153001, Millipore) for 30 min at 4 °C with rotation. Lysates were spun at 4 °C for 10 min at 9800 × <em>g</em>. 5% of lysates were frozen for input controls. 4 μg of mouse anti-FLAG M2 antibody (F3165, Millipore) was added to lysates and rotated at 4 °C for 2 h. 40 μL Dynabeads Protein G (10003D, Thermo Fisher) was added to lysates and rotated for 1 hour at 4°C. Beads were collected with a magnetic rack and washed 5x with cell lysis buffer with rotation at 4 °C for 5 mins. Proteins were released from Dynabeads Protein G beads for LC-MS by boiling in 1% SDS. Proteins for Western blots were boiled with 1x Laemmli buffer.</p></section><section id="Sec23"><h3 class="pmc_sec_title">Western blots</h3>
<p id="Par46">For western blots on cultured OLs, plates were washed 3x with cold DPBS and lysed with RIPA buffer (50 mM Tris-HCL pH 8.0, 150 mM NaCl, 1% NP-40, 0.5% Sodium deoxycholate, 0.1% SDS, 1 mM EDTA, 0.5 mM EGTA) with complete protease inhibitors (11836153001, Roche), and phosphatase inhibitors (04906837001, Roche) before being spun at 13,000 × <em>g</em> at 4 °C. Protein lysate was removed and frozen at −80 °C. Lysates were boiled at 98 °C in 1x Laemmli buffer for 5 min and run on Bis-Tris-gel (NP0335BOX, Invitrogen). To transfer proteins to PVDF membranes (IPVH00010, Thermo Scientific), transfer cassettes were assembled (A25977, Thermo Fisher Scientific) and filled with transfer buffer (NP0006-01, Thermo Scientific) containing 10% methanol and transferred at 20 V for 1 hour. Following transfer, blots were rinsed in 1x TBS with 0.1% Tween-20 (TBST) before blocking in 1x TBST with 5% milk powder for one hour at room temperature. Blots were probed with antibodies against MYRF (16355, ABClonal), mTOR (2983, Cell Signaling), Phospho-mTOR (Ser2448; 5536, Cell Signaling), cJun (9261 L, Cell Systems), TMEM98 (14731-1-AP, Proteintech), MYO1D (ab70204, Abcam), GSK3β (ab32391, abcam), DPYSL5 (CRMP5; ab36203, Abcam). All antibodies for western blots were used at a concentration of 1:1000. Blots were incubated in primary antibodies diluted in 2.5% BSA (BP9706-100, Fisher Scientific) with 1% NaN<sub>3</sub> in TBST overnight at 4 °C. After overnight incubation, blots were washed in 3x TBST and incubated with appropriate HRP-conjugated secondary (Goat anti-rat 7077, Cell Signaling; Goat anti-mouse 7076, Cell Signaling, Goat anti-rabbit 7074, Cell Signaling) at 1:5000 for two hours with 2.5% milk powder in TBST. Immunoreactivity was visualized using chemiluminescence (34080, Thermo Fisher Scientific) and imaged on a Syngene GBox iChemiXT. Blots were then re-probed with β-actin-HRP (1:5000; A3854, Sigma). Densitometric analyses were performed in ImageJ by quantifying the intensity of bands relative to the ACTB loading control and then normalized to background.</p></section><section id="Sec24"><h3 class="pmc_sec_title">Mass spectrometry and analysis</h3>
<section id="Sec25"><h4 class="pmc_sec_title">Bulk mass spectrometry</h4>
<p id="Par47">OLs electroporated with pooled siRNAs targeting <em>Fbxw7</em> or non-targeting controls were differentiated for 72 hours and lysed with eFASP buffer (4% SDS, 0.2% DCA, 100 mM TEAB), frozen at −80 °C, and submitted to the OHSU proteomic core. Samples were then sonicated using Bioruptor Pico (30 s on 30 s off, 10 cycles), heated to 90 °C for 10 mins, cooled, centrifuged, and protein concentration was determined by BCA. 55 μg of protein/sample were digested with eFASP and measured by peptide assay. 18 μg peptides/sample were labeled with TMT 11-plex, normalized, and pooled. Pooled TMT samples were then run on an 18 fraction 2dRPRP LC/MS on Orbitrap Fusion. Data was then analyzed with COMET/PAWS pipeline in edgeR.</p></section><section id="Sec26"><h4 class="pmc_sec_title">Immunoprecipitation mass spectrometry</h4>
<p id="Par48">OL samples from IP were prepared for proteomics analysis. Samples were dried on a SpeedVac and resuspended in 5% SDS, 50 mM TEAB pH 8. Samples were reduced with dithiothreitol and alkylated with iodoacetamide. Proteins were bound to S-trap columns (Protifi, Inc.) where they were washed before being digested with trypsin. A Quantitative Colorimetric Peptide Assay (23275, Pierce) was performed. Peptides were analyzed by LC-MS on a Dionex Ultimate HPLC to an Orbitrap Q-Exactive mass spectrometer (Thermo Fisher Scientific, Waltham, MA).</p></section></section><section id="Sec27"><h3 class="pmc_sec_title">qRT-PCR</h3>
<p id="Par49">RNA was isolated from primary rat OPC/OLs with the RNeasy Mini Kit (74104, QIAGEN) and stored at −80 °C. RNA from optic nerves and cerebellums were isolated by homogenizing tissue in 700 μl of TRIzol (15596026, Thermo Fisher). 140 μl of chloroform (J67241.K4, Thermo Fisher) was added to lysate and shaken for 30 seconds and left at room temperature for 5 min. Reactions were then centrifuged for 15 min at 12,000 × <em>g</em> at 4 °C. The upper, colorless, aqueous phase containing RNA was then purified further with RNeasy UCP Micro (optic nerves) and Mini (cerebellums) Kits (73934, 74104, QIAGEN) and stored at −80 °C. cDNA was generated with SuperScript III First-Strand Synthesis (18080400, Thermo Fisher) and stored at −20 °C. qPCR was performed with PowerUP SYBR Green (A25742, Thermo Fisher) on a QuantStudio 6 Flex Real-Time PCR System (4485691, Thermo Fisher). qRT-PCR primers were designed on the Integrated DNA Technologies (IDT) PrimerQuest program.</p>
<section class="tw xbox font-sm" id="Tabc"><div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups"><tbody>
<tr>
<td align="left" colspan="1" rowspan="1">qRT-PCR</td>
<td align="left" colspan="1" rowspan="1">Primers</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Gene</td>
<td align="left" colspan="1" rowspan="1">Forward</td>
<td align="left" colspan="1" rowspan="1">Reverse</td>
<td align="left" colspan="1" rowspan="1">Species</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"><em>Myrf</em></td>
<td align="left" colspan="1" rowspan="1">CTTCCTTCTGGTCATCTCCTTG</td>
<td align="left" colspan="1" rowspan="1">GAGAGTGAAGGTGGGCAAATA</td>
<td align="left" colspan="1" rowspan="1">Mouse</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"><em>Actb</em></td>
<td align="left" colspan="1" rowspan="1">GAGGTATCCTGACCCTGAAGTA</td>
<td align="left" colspan="1" rowspan="1">CACACGCAGCTCATTGTAGA</td>
<td align="left" colspan="1" rowspan="1">Mouse</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"><em>Mbp</em></td>
<td align="left" colspan="1" rowspan="1">AGTCGCAGAGGACCCAAGAT</td>
<td align="left" colspan="1" rowspan="1">ACAGGCCTCTCCCCTTTC</td>
<td align="left" colspan="1" rowspan="1">Rat</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"><em>Plp1</em></td>
<td align="left" colspan="1" rowspan="1">CACTTACAGCAGGTGATTAGAGG</td>
<td align="left" colspan="1" rowspan="1">AAACAAGAGATAAACAACTGGGA</td>
<td align="left" colspan="1" rowspan="1">Rat</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"><em>Mog</em></td>
<td align="left" colspan="1" rowspan="1">GTCTATCGGCGAGGGAAAGGT</td>
<td align="left" colspan="1" rowspan="1">CACGGCGGCTTCTTCTTGGT</td>
<td align="left" colspan="1" rowspan="1">Rat</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"><em>Myrf</em></td>
<td align="left" colspan="1" rowspan="1">CAGCAGTGGGAAAGGGAATAA</td>
<td align="left" colspan="1" rowspan="1">AACTCAGCTCCCGATAGAGAT</td>
<td align="left" colspan="1" rowspan="1">Rat</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"><em>Tmem98</em></td>
<td align="left" colspan="1" rowspan="1">CAGTGTAACACCACGTCTACC</td>
<td align="left" colspan="1" rowspan="1">CTCGATTCTTAGAGGGCAACTC</td>
<td align="left" colspan="1" rowspan="1">Rat</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"><em>Itgb3</em></td>
<td align="left" colspan="1" rowspan="1">GCTGTCCTGTATGTGGTAGAAG</td>
<td align="left" colspan="1" rowspan="1">CAGAGTAGCAAGGCCAATGA</td>
<td align="left" colspan="1" rowspan="1">Rat</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"><em>Dpysl5</em></td>
<td align="left" colspan="1" rowspan="1">TCCATACCCACACCCTCATA</td>
<td align="left" colspan="1" rowspan="1">ACTCTCTCCTACTCTCCCTTTC</td>
<td align="left" colspan="1" rowspan="1">Rat</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"><em>Myo1d</em></td>
<td align="left" colspan="1" rowspan="1">ACTTCAAGCGCCTCATGTATAA</td>
<td align="left" colspan="1" rowspan="1">CCTCTTGGTCACCTCTGTAATG</td>
<td align="left" colspan="1" rowspan="1">Rat</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"><em>Fam107b</em></td>
<td align="left" colspan="1" rowspan="1">GAAGCACGAGAGCCGATTATAG</td>
<td align="left" colspan="1" rowspan="1">TCAAGGGCAAGCCATCTTAC</td>
<td align="left" colspan="1" rowspan="1">Rat</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"><em>Vsnl1</em></td>
<td align="left" colspan="1" rowspan="1">TGTCTGGCCCACATACAATAC</td>
<td align="left" colspan="1" rowspan="1">CGAGAGGTTACAAATGAGGTAAGA</td>
<td align="left" colspan="1" rowspan="1">Rat</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"><em>Pea15</em></td>
<td align="left" colspan="1" rowspan="1">ATGAAGACACAGGAGAGAGAGA</td>
<td align="left" colspan="1" rowspan="1">GTGAGTGTATTAGGGCAGGTTAG</td>
<td align="left" colspan="1" rowspan="1">Rat</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"><em>Tppp</em></td>
<td align="left" colspan="1" rowspan="1">ACTGGGCAGAACTCAGAATG</td>
<td align="left" colspan="1" rowspan="1">CAATGAATCACGGCCCAAAC</td>
<td align="left" colspan="1" rowspan="1">Rat</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"><em>Rpl13A</em></td>
<td align="left" colspan="1" rowspan="1">CTCATGATGACTGCAGCAAACC</td>
<td align="left" colspan="1" rowspan="1">GGATCCCTCCAC CCTATGACA</td>
<td align="left" colspan="1" rowspan="1">Rat</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"><em>mbp</em></td>
<td align="left" colspan="1" rowspan="1">AATCAGCAGGTTCTTCGGAGGAGA</td>
<td align="left" colspan="1" rowspan="1">AAGAAATGCACGACAGGGTTGACG</td>
<td align="left" colspan="1" rowspan="1">Zebrafish</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"><em>actb</em></td>
<td align="left" colspan="1" rowspan="1">CGAGCTGTCTTCCCATCCA</td>
<td align="left" colspan="1" rowspan="1">TCACCAACGTAGCTGTCTTTCTG</td>
<td align="left" colspan="1" rowspan="1">Zebrafish</td>
</tr>
</tbody></table></div>
<div class="p text-right font-secondary"><a href="table/Tabc/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section></section><section id="Sec28"><h3 class="pmc_sec_title">Quantification and statistical analysis</h3>
<p id="Par50">Myelin outfolds in the corpus callosum were quantified in ImageJ by image thresholding and particle analyses. MYRF IF in the optic nerve and cerebellum was quantified in ImageJ by overlaying a DAPI mask on MYRF images followed by particle analyses for mean gray values. CC1<sup>+</sup> cell in the cortex and corpus callosum were quantified manually. MBP<sup>+</sup> wrapped NeuN<sup>+</sup> cell bodies were quantified manually in the cerebellum. Western blots were quantified by densitometry in ImageJ. All bands were normalized to blot background and bands from proteins of interest were normalized to loading controls. Statistical analyses were conducted with Prism 10 (Graphpad). In all cases the figure legend indicates the statistical test used and <em>P</em> values are presented in figures. Sample size is stated in figure legends. Animals were assigned to group based on genotype by random selection and analyses were conducted blinded to genotype.</p>
<p id="Par51">Mass spectrometry data was processed using the PAW pipeline<sup><a href="#CR77" class="usa-link" aria-describedby="CR77">77</a></sup> with the Comet search engine (version 2016.03)<sup><a href="#CR78" class="usa-link" aria-describedby="CR78">78</a></sup> searching versus the Rattus Norvegicus Uniprot database (UP000002494, Taxon ID:10116, one protein sequence per gene). The mass spectrometry proteomics data have been deposited to the ProteomeXchange Consortium via the PRIDE partner repository with the dataset identifier PXD063570<sup><a href="#CR79" class="usa-link" aria-describedby="CR79">79</a>–<a href="#CR81" class="usa-link" aria-describedby="CR81">81</a></sup>.</p></section><section id="Sec29"><h3 class="pmc_sec_title">Availability of materials</h3>
<p id="Par52">Fish lines and reagents generated in this study including plasmids will be made available on request to the corresponding authors (monk@ohsu.edu and emeryb@ohsu.edu).</p></section><section id="Sec30"><h3 class="pmc_sec_title">Reporting summary</h3>
<p id="Par53">Further information on research design is available in the <a href="#MOESM5" class="usa-link">Nature Portfolio Reporting Summary</a> linked to this article.</p></section></section><section id="Sec31"><h2 class="pmc_sec_title">Supplementary information</h2>
<section class="sm xbox font-sm" id="MOESM1"><div class="media p"><div class="caption">
<a href="/articles/instance/12370921/bin/41467_2025_62715_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information</a><sup> (7.8MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM2"><div class="media p"><div class="caption">
<a href="/articles/instance/12370921/bin/41467_2025_62715_MOESM2_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">41467_2025_62715_MOESM2_ESM.pdf</a><sup> (24.6KB, pdf) </sup><p>Description of Additional Supplementary Files</p>
</div></div></section><section class="sm xbox font-sm" id="MOESM3"><div class="media p"><div class="caption">
<a href="/articles/instance/12370921/bin/41467_2025_62715_MOESM3_ESM.avi" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Movie 1</a><sup> (37.3MB, avi) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM4"><div class="media p"><div class="caption">
<a href="/articles/instance/12370921/bin/41467_2025_62715_MOESM4_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Dataset 1</a><sup> (149.6KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM5"><div class="media p"><div class="caption">
<a href="/articles/instance/12370921/bin/41467_2025_62715_MOESM5_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Reporting Summary</a><sup> (149.4KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM6"><div class="media p"><div class="caption">
<a href="/articles/instance/12370921/bin/41467_2025_62715_MOESM6_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Transparent Peer Review file</a><sup> (222.1KB, pdf) </sup>
</div></div></section></section><section id="Sec32"><h2 class="pmc_sec_title">Source data</h2>
<section class="sm xbox font-sm" id="MOESM7"><div class="media p"><div class="caption">
<a href="/articles/instance/12370921/bin/41467_2025_62715_MOESM7_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Source Data</a><sup> (96MB, xlsx) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>We would like to thank current and past members of the Emery and Monks labs, particularly Suhail Akram, Emma Brennan, Austin Forbes, Tia Perry, and Adriana Reyes for excellent animal care. We would especially like to thank Dr. Breanne Harty of the Monk lab for her work on first characterizing FBXW7 in the zebrafish CNS and the mouse PNS. We would also like to thank Dr. Ronald Waclaw at Cincinnati Children’s Hospital for alerting us to the anti-MYRF antibody. We would like to thank Dr. Ashok Reddy of the OHSU Proteomic Core for his comments and suggestions while designing our proteomic experiments. Imaging work was supported by the OHSU Advanced Light Microscopy Core (ALMC, RRID: SCR_009960). The research reported in this publication used computational infrastructure of OHSU's Advanced Computing Center (ACC, RRID: SCR_009959) supported by NIH S10OD034224. Mass spectrometric analysis was performed by the OHSU Proteomics Shared Resource (RRID: SCR_009991) with partial support from NIH core grants P30EY010572, P30CA069533, and S10OD012246. This project was supported by the National Multiple Sclerosis Society (RG-1901-33272) and the National Institute of Neurological Disorders and Stroke (R01NS138188) to B.E. and K.R.M. and the Herbert R. &amp; Jeanne C. Mayer Foundation and the National Institutes of Health - National Institute of Neurological Disorders and Stroke (F31NS122433) to H.Y.C. B.E. was supported by an endowment from the Warren family.</p></section><section id="notes1"><h2 class="pmc_sec_title">Author contributions</h2>
<p>H.Y.C., T.S., K.R.M., and B.E. conceived of the project. H.Y.C designed, performed, and analyzed all experiments with the following exceptions: J.L generated the <em>vo86</em> zebrafish line (H.Y.C performed all validation experiments and analyses). J.L. performed and analyzed the <em>fbxw7</em> cell-specific CRISPR-Cas9-mediated gene disruption in Fig. <a href="#Fig1" class="usa-link">1f–h</a>. R.A.D. processed and imaged optic nerve and cerebellum TEMs in Figs. <a href="#Fig3" class="usa-link">3</a> and <a href="#Fig4" class="usa-link">4</a>. D.A.L., J.E.E. and M.E.M. generated the <em>myrf</em> <sup><em>ue70</em></sup> zebrafish line in Fig. <a href="#Fig7" class="usa-link">7d, I</a>, and Supplementary Fig. <a href="#MOESM1" class="usa-link">6b</a>, which D.A.L. provided prior to its publication. H.Y.C., K.R.M., and B.E. wrote the manuscript. All authors provided feedback on the manuscript and approved the submitted version.</p></section><section id="notes2"><h2 class="pmc_sec_title">Peer review</h2>
<section id="FPar1"><h3 class="pmc_sec_title">Peer review information</h3>
<p id="Par54"><em>Nature Communications</em> thanks the anonymous reviewer(s) for their contribution to the peer review of this work. A peer review file is available.</p></section></section><section id="notes3"><h2 class="pmc_sec_title">Data availability</h2>
<p>Source data are provided with this paper and mass spectrometry data is available via ProteomeXchange with identifier PXD063570: (<a href="https://www.ebi.ac.uk/pride/archive/projects/PXD063570" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ebi.ac.uk/pride/archive/projects/PXD063570</a>). <a href="#Sec32" class="usa-link">Source data</a> are provided with this paper.</p></section><section id="FPar2"><h2 class="pmc_sec_title">Competing interests</h2>
<p id="Par55">The authors declare no competing interests.</p></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="fn1"><p><strong>Publisher’s note</strong> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></div></div></section><section id="_ci93_" lang="en" class="contrib-info"><h2 class="pmc_sec_title">Contributor Information</h2>
<p>Kelly R. Monk, Email: monk@ohsu.edu</p>
<p>Ben Emery, Email: emeryb@ohsu.edu.</p></section><section id="sec33"><h2 class="pmc_sec_title">Supplementary information</h2>
<p>The online version contains supplementary material available at 10.1038/s41467-025-62715-9.</p></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Waxman, S. G. &amp; Bennett, M. V. L. Relative conduction velocities of small myelinated and non-myelinated fibres in the central nervous system. <em>Nat. N. Biol.</em><strong>238</strong>, 217–219 (1972).</cite> [<a href="https://doi.org/10.1038/newbio238217a0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/4506206/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Waxman,%20S.%20G.%20&amp;%20Bennett,%20M.%20V.%20L.%20Relative%20conduction%20velocities%20of%20small%20myelinated%20and%20non-myelinated%20fibres%20in%20the%20central%20nervous%20system.%20Nat.%20N.%20Biol.238,%20217%E2%80%93219%20(1972)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>Ritchie, J. M. et al. <em>Physiological Basis of Conduction in Myelinated Nerve Fibers</em>. in <em>Myelin</em> (ed. Morell, P.) 117–145 (Springer US, Boston, MA, 1984).</cite>
</li>
<li id="CR3">
<span class="label">3.</span><cite>Nave, K.-A. &amp; Trapp, B. D. Axon-glial signaling and the glial support of axon function. <em>Annu. Rev. Neurosci.</em><strong>31</strong>, 535–561 (2008).
</cite> [<a href="https://doi.org/10.1146/annurev.neuro.30.051606.094309" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18558866/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Nave,%20K.-A.%20&amp;%20Trapp,%20B.%20D.%20Axon-glial%20signaling%20and%20the%20glial%20support%20of%20axon%20function.%20Annu.%20Rev.%20Neurosci.31,%20535%E2%80%93561%20(2008)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Huxley, A. F. &amp; Stämpfli, R. Direct determination of membrane resting potential and action potential in single myelinated nerve fibres. <em>J. Physiol.</em><strong>112</strong>, 476–495 (1951).
</cite> [<a href="https://doi.org/10.1113/jphysiol.1951.sp004545" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1393015/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/14825228/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Huxley,%20A.%20F.%20&amp;%20St%C3%A4mpfli,%20R.%20Direct%20determination%20of%20membrane%20resting%20potential%20and%20action%20potential%20in%20single%20myelinated%20nerve%20fibres.%20J.%20Physiol.112,%20476%E2%80%93495%20(1951)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Raff, M. C., Miller, R. H. &amp; Noble, M. A glial progenitor cell that develops in vitro into an astrocyte or an oligodendrocyte depending on culture medium. <em>Nature</em><strong>303</strong>, 390–396 (1983).
</cite> [<a href="https://doi.org/10.1038/303390a0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/6304520/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Raff,%20M.%20C.,%20Miller,%20R.%20H.%20&amp;%20Noble,%20M.%20A%20glial%20progenitor%20cell%20that%20develops%20in%20vitro%20into%20an%20astrocyte%20or%20an%20oligodendrocyte%20depending%20on%20culture%20medium.%20Nature303,%20390%E2%80%93396%20(1983)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Lee, S. et al. A culture system to study oligodendrocyte myelination-processes using engineered nanofibers. <em>Nat. Methods</em><strong>9</strong>, 917–922 (2012).
</cite> [<a href="https://doi.org/10.1038/nmeth.2105" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3433633/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22796663/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lee,%20S.%20et%20al.%20A%20culture%20system%20to%20study%20oligodendrocyte%20myelination-processes%20using%20engineered%20nanofibers.%20Nat.%20Methods9,%20917%E2%80%93922%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Gibson, E. M. et al. Neuronal activity promotes oligodendrogenesis and adaptive myelination in the mammalian brain. <em>Science</em><strong>344</strong>, 1252304 (2014).
</cite> [<a href="https://doi.org/10.1126/science.1252304" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4096908/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24727982/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gibson,%20E.%20M.%20et%20al.%20Neuronal%20activity%20promotes%20oligodendrogenesis%20and%20adaptive%20myelination%20in%20the%20mammalian%20brain.%20Science344,%201252304%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>Mitew, S. et al. Pharmacogenetic stimulation of neuronal activity increases myelination in an axon-specific manner. <em>Nat. Commun.</em><strong>9</strong>, 306 (2018).
</cite> [<a href="https://doi.org/10.1038/s41467-017-02719-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5778130/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29358753/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mitew,%20S.%20et%20al.%20Pharmacogenetic%20stimulation%20of%20neuronal%20activity%20increases%20myelination%20in%20an%20axon-specific%20manner.%20Nat.%20Commun.9,%20306%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>Czopka, T., ffrench-Constant, C. &amp; Lyons, D. A. Individual oligodendrocytes have only a few hours in which to generate new myelin sheaths in vivo. <em>Dev. Cell</em><strong>25</strong>, 599–609 (2013).
</cite> [<a href="https://doi.org/10.1016/j.devcel.2013.05.013" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4013507/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23806617/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Czopka,%20T.,%20ffrench-Constant,%20C.%20&amp;%20Lyons,%20D.%20A.%20Individual%20oligodendrocytes%20have%20only%20a%20few%20hours%20in%20which%20to%20generate%20new%20myelin%20sheaths%20in%20vivo.%20Dev.%20Cell25,%20599%E2%80%93609%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>Zhou, Q., Choi, G. &amp; Anderson, D. J. The bHLH transcription factor Olig2 Promotes oligodendrocyte differentiation in collaboration with Nkx2.2. <em>Neuron</em><strong>31</strong>, 791–807 (2001).
</cite> [<a href="https://doi.org/10.1016/s0896-6273(01)00414-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11567617/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhou,%20Q.,%20Choi,%20G.%20&amp;%20Anderson,%20D.%20J.%20The%20bHLH%20transcription%20factor%20Olig2%20Promotes%20oligodendrocyte%20differentiation%20in%20collaboration%20with%20Nkx2.2.%20Neuron31,%20791%E2%80%93807%20(2001)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Hughes, E. G., Kang, S. H., Fukaya, M. &amp; Bergles, D. E. Oligodendrocyte progenitors balance growth with self-repulsion to achieve homeostasis in the adult brain. <em>Nat. Neurosci.</em><strong>16</strong>, 668–676 (2013).
</cite> [<a href="https://doi.org/10.1038/nn.3390" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3807738/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23624515/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hughes,%20E.%20G.,%20Kang,%20S.%20H.,%20Fukaya,%20M.%20&amp;%20Bergles,%20D.%20E.%20Oligodendrocyte%20progenitors%20balance%20growth%20with%20self-repulsion%20to%20achieve%20homeostasis%20in%20the%20adult%20brain.%20Nat.%20Neurosci.16,%20668%E2%80%93676%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Hughes, E. G., Orthmann-Murphy, J. L., Langseth, A. J. &amp; Bergles, D. E. Myelin remodeling through experience-dependent oligodendrogenesis in the adult somatosensory cortex. <em>Nat. Neurosci.</em><strong>21</strong>, 696–706 (2018).
</cite> [<a href="https://doi.org/10.1038/s41593-018-0121-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5920726/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29556025/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hughes,%20E.%20G.,%20Orthmann-Murphy,%20J.%20L.,%20Langseth,%20A.%20J.%20&amp;%20Bergles,%20D.%20E.%20Myelin%20remodeling%20through%20experience-dependent%20oligodendrogenesis%20in%20the%20adult%20somatosensory%20cortex.%20Nat.%20Neurosci.21,%20696%E2%80%93706%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>Saher, G. et al. High cholesterol level is essential for myelin membrane growth. <em>Nat. Neurosci.</em><strong>8</strong>, 468–475 (2005).
</cite> [<a href="https://doi.org/10.1038/nn1426" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15793579/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Saher,%20G.%20et%20al.%20High%20cholesterol%20level%20is%20essential%20for%20myelin%20membrane%20growth.%20Nat.%20Neurosci.8,%20468%E2%80%93475%20(2005)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>Nawaz, S. et al. Actin filament turnover drives leading edge growth during myelin sheath formation in the central nervous system. <em>Dev. Cell</em><strong>34</strong>, 139–151 (2015).
</cite> [<a href="https://doi.org/10.1016/j.devcel.2015.05.013" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4736019/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26166299/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Nawaz,%20S.%20et%20al.%20Actin%20filament%20turnover%20drives%20leading%20edge%20growth%20during%20myelin%20sheath%20formation%20in%20the%20central%20nervous%20system.%20Dev.%20Cell34,%20139%E2%80%93151%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>Zuchero, J. B. et al. CNS myelin wrapping is driven by actin disassembly. <em>Dev. Cell</em><strong>34</strong>, 152–167 (2015).
</cite> [<a href="https://doi.org/10.1016/j.devcel.2015.06.011" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4519368/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26166300/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zuchero,%20J.%20B.%20et%20al.%20CNS%20myelin%20wrapping%20is%20driven%20by%20actin%20disassembly.%20Dev.%20Cell34,%20152%E2%80%93167%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Cahoy, J. D. et al. A transcriptome database for astrocytes, neurons, and oligodendrocytes: a new resource for understanding brain development and function. <em>J. Neurosci. J. Soc.</em><strong>28</strong>, 264–278 (2008).</cite> [<a href="https://doi.org/10.1523/JNEUROSCI.4178-07.2008" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6671143/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18171944/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cahoy,%20J.%20D.%20et%20al.%20A%20transcriptome%20database%20for%20astrocytes,%20neurons,%20and%20oligodendrocytes:%20a%20new%20resource%20for%20understanding%20brain%20development%20and%20function.%20J.%20Neurosci.%20J.%20Soc.28,%20264%E2%80%93278%20(2008)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>Emery, B. et al. Myelin gene regulatory factor is a critical transcriptional regulator required for CNS myelination. <em>Cell</em><strong>138</strong>, 172–185 (2009).
</cite> [<a href="https://doi.org/10.1016/j.cell.2009.04.031" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2757090/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19596243/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Emery,%20B.%20et%20al.%20Myelin%20gene%20regulatory%20factor%20is%20a%20critical%20transcriptional%20regulator%20required%20for%20CNS%20myelination.%20Cell138,%20172%E2%80%93185%20(2009)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Elbaz, B. &amp; Popko, B. Molecular control of oligodendrocyte development. <em>Trends Neurosci.</em><strong>42</strong>, 263–277 (2019).
</cite> [<a href="https://doi.org/10.1016/j.tins.2019.01.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7397568/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30770136/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Elbaz,%20B.%20&amp;%20Popko,%20B.%20Molecular%20control%20of%20oligodendrocyte%20development.%20Trends%20Neurosci.42,%20263%E2%80%93277%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Marques, S. et al. Transcriptional convergence of oligodendrocyte lineage progenitors during development. <em>Dev. Cell</em><strong>46</strong>, 504–517.e7 (2018).
</cite> [<a href="https://doi.org/10.1016/j.devcel.2018.07.005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6104814/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30078729/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Marques,%20S.%20et%20al.%20Transcriptional%20convergence%20of%20oligodendrocyte%20lineage%20progenitors%20during%20development.%20Dev.%20Cell46,%20504%E2%80%93517.e7%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>Maggie, S.Y. et al. Dynamics of oligodendrocyte generation and myelination in the human brain. <em>Cell</em><strong>159</strong>, 766–74 (2014).</cite> [<a href="https://doi.org/10.1016/j.cell.2014.10.011" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25417154/" class="usa-link">PubMed</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>Sabri, M. I., Bone, A. H. &amp; Davison, A. N. Turnover of myelin and other structural proteins in the developing rat brain. <em>Biochem. J.</em><strong>142</strong>, 499–507 (1974).
</cite> [<a href="https://doi.org/10.1042/bj1420499" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1168314/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/4142927/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sabri,%20M.%20I.,%20Bone,%20A.%20H.%20&amp;%20Davison,%20A.%20N.%20Turnover%20of%20myelin%20and%20other%20structural%20proteins%20in%20the%20developing%20rat%20brain.%20Biochem.%20J.142,%20499%E2%80%93507%20(1974)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>Saher, G. &amp; Simons, M. <em>Cholesterol and Myelin Biogenesis</em>. in <em>Cholesterol Binding and Cholesterol Transport Proteins: Structure and Function in Health and Disease</em> (ed. Harris, J. R.) 489–508 (Springer Netherlands, Dordrecht, 2010).</cite>
</li>
<li id="CR23">
<span class="label">23.</span><cite>Tripathi, R. B. et al. Remarkable stability of myelinating oligodendrocytes in mice. <em>Cell Rep.</em><strong>21</strong>, 316–323 (2017).
</cite> [<a href="https://doi.org/10.1016/j.celrep.2017.09.050" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5643547/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29020619/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tripathi,%20R.%20B.%20et%20al.%20Remarkable%20stability%20of%20myelinating%20oligodendrocytes%20in%20mice.%20Cell%20Rep.21,%20316%E2%80%93323%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>Auer, F., Vagionitis, S. &amp; Czopka, T. Evidence for myelin sheath remodeling in the CNS revealed by in vivo imaging. <em>Curr. Biol.</em><strong>28</strong>, 549–559.e3 (2018).
</cite> [<a href="https://doi.org/10.1016/j.cub.2018.01.017" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29429620/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Auer,%20F.,%20Vagionitis,%20S.%20&amp;%20Czopka,%20T.%20Evidence%20for%20myelin%20sheath%20remodeling%20in%20the%20CNS%20revealed%20by%20in%20vivo%20imaging.%20Curr.%20Biol.28,%20549%E2%80%93559.e3%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>Hill, R. A., Li, A. M. &amp; Grutzendler, J. Lifelong cortical myelin plasticity and age-related degeneration in the live mammalian brain. <em>Nat. Neurosci.</em><strong>21</strong>, 683–695 (2018).
</cite> [<a href="https://doi.org/10.1038/s41593-018-0120-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5920745/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29556031/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hill,%20R.%20A.,%20Li,%20A.%20M.%20&amp;%20Grutzendler,%20J.%20Lifelong%20cortical%20myelin%20plasticity%20and%20age-related%20degeneration%20in%20the%20live%20mammalian%20brain.%20Nat.%20Neurosci.21,%20683%E2%80%93695%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>Snyder, J. L., Kearns, C. A. &amp; Appel, B. Fbxw7 regulates Notch to control specification of neural precursors for oligodendrocyte fate. <em>Neural Dev.</em><strong>7</strong>, 15 (2012).
</cite> [<a href="https://doi.org/10.1186/1749-8104-7-15" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3404928/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22554084/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Snyder,%20J.%20L.,%20Kearns,%20C.%20A.%20&amp;%20Appel,%20B.%20Fbxw7%20regulates%20Notch%20to%20control%20specification%20of%20neural%20precursors%20for%20oligodendrocyte%20fate.%20Neural%20Dev.7,%2015%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Kearns, C. A., Ravanelli, A. M., Cooper, K. &amp; Appel, B. Fbxw7 limits myelination by inhibiting mTOR signaling. <em>J. Neurosci.</em><strong>35</strong>, 14861–14871 (2015).
</cite> [<a href="https://doi.org/10.1523/JNEUROSCI.4968-14.2015" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4635133/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26538655/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kearns,%20C.%20A.,%20Ravanelli,%20A.%20M.,%20Cooper,%20K.%20&amp;%20Appel,%20B.%20Fbxw7%20limits%20myelination%20by%20inhibiting%20mTOR%20signaling.%20J.%20Neurosci.35,%2014861%E2%80%9314871%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Harty, B. L. et al. Myelinating Schwann cells ensheath multiple axons in the absence of E3 ligase component Fbxw7. <em>Nat. Commun.</em><strong>10</strong>, 2976 (2019).
</cite> [<a href="https://doi.org/10.1038/s41467-019-10881-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6611888/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31278268/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Harty,%20B.%20L.%20et%20al.%20Myelinating%20Schwann%20cells%20ensheath%20multiple%20axons%20in%20the%20absence%20of%20E3%20ligase%20component%20Fbxw7.%20Nat.%20Commun.10,%202976%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>Sanchez, N. E. et al. Whole genome sequencing-based mapping and candidate identification of mutations from fixed zebrafish tissue. <em>G3(Bethesda)</em><strong>7</strong>, 3415–3425 (2017).
</cite> [<a href="https://doi.org/10.1534/g3.117.300212" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5633390/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28855284/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sanchez,%20N.%20E.%20et%20al.%20Whole%20genome%20sequencing-based%20mapping%20and%20candidate%20identification%20of%20mutations%20from%20fixed%20zebrafish%20tissue.%20G3(Bethesda)7,%203415%E2%80%933425%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR30">
<span class="label">30.</span><cite>Nateri, A. S., Riera-Sans, L., Costa, C. D. &amp; Behrens, A. The ubiquitin ligase SCFFbw7 antagonizes apoptotic JNK signaling. <em>Science</em><strong>303</strong>, 1374–1378 (2004).
</cite> [<a href="https://doi.org/10.1126/science.1092880" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/14739463/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Nateri,%20A.%20S.,%20Riera-Sans,%20L.,%20Costa,%20C.%20D.%20&amp;%20Behrens,%20A.%20The%20ubiquitin%20ligase%20SCFFbw7%20antagonizes%20apoptotic%20JNK%20signaling.%20Science303,%201374%E2%80%931378%20(2004)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>Ye, X. et al. Recognition of Phosphodegron Motifs in Human Cyclin E by the SCFFbw7 Ubiquitin Ligase*. <em>J. Biol. Chem.</em><strong>279</strong>, 50110–50119 (2004).
</cite> [<a href="https://doi.org/10.1074/jbc.M409226200" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15364936/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ye,%20X.%20et%20al.%20Recognition%20of%20Phosphodegron%20Motifs%20in%20Human%20Cyclin%20E%20by%20the%20SCFFbw7%20Ubiquitin%20Ligase*.%20J.%20Biol.%20Chem.279,%2050110%E2%80%9350119%20(2004)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>Yada, M. et al. Phosphorylation-dependent degradation of c-Myc is mediated by the F-box protein Fbw7. <em>EMBO J.</em><strong>23</strong>, 2116–2125 (2004).
</cite> [<a href="https://doi.org/10.1038/sj.emboj.7600217" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC424394/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15103331/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yada,%20M.%20et%20al.%20Phosphorylation-dependent%20degradation%20of%20c-Myc%20is%20mediated%20by%20the%20F-box%20protein%20Fbw7.%20EMBO%20J.23,%202116%E2%80%932125%20(2004)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Nakayama, S., Yumimoto, K., Kawamura, A. &amp; Nakayama, K. I. Degradation of the endoplasmic reticulum-anchored transcription factor MyRF by the ubiquitin ligase SCFFbxw7 in a manner dependent on the kinase GSK-3. <em>J. Biol. Chem.</em><strong>293</strong>, 5705–5714 (2018).
</cite> [<a href="https://doi.org/10.1074/jbc.RA117.000741" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5900774/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29472293/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Nakayama,%20S.,%20Yumimoto,%20K.,%20Kawamura,%20A.%20&amp;%20Nakayama,%20K.%20I.%20Degradation%20of%20the%20endoplasmic%20reticulum-anchored%20transcription%20factor%20MyRF%20by%20the%20ubiquitin%20ligase%20SCFFbxw7%20in%20a%20manner%20dependent%20on%20the%20kinase%20GSK-3.%20J.%20Biol.%20Chem.293,%205705%E2%80%935714%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>Thompson, B. J. et al. Control of hematopoietic stem cell quiescence by the E3 ubiquitin ligase Fbw7. <em>J. Exp. Med.</em><strong>205</strong>, 1395–1408 (2008).
</cite> [<a href="https://doi.org/10.1084/jem.20080277" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2413036/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18474632/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Thompson,%20B.%20J.%20et%20al.%20Control%20of%20hematopoietic%20stem%20cell%20quiescence%20by%20the%20E3%20ubiquitin%20ligase%20Fbw7.%20J.%20Exp.%20Med.205,%201395%E2%80%931408%20(2008)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>Almeida, R. G. et al. Myelination of neuronal cell bodies when myelin supply exceeds axonal demand. <em>Curr. Biol.</em><strong>28</strong>, 1296–1305.e5 (2018).
</cite> [<a href="https://doi.org/10.1016/j.cub.2018.02.068" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5912901/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29628374/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Almeida,%20R.%20G.%20et%20al.%20Myelination%20of%20neuronal%20cell%20bodies%20when%20myelin%20supply%20exceeds%20axonal%20demand.%20Curr.%20Biol.28,%201296%E2%80%931305.e5%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR36">
<span class="label">36.</span><cite>Li, J., Miramontes, T. G., Czopka, T. &amp; Monk, K. R. Synaptic input and Ca2+ activity in zebrafish oligodendrocyte precursor cells contribute to myelin sheath formation. <em>Nat. Neurosci.</em><strong>27</strong>, 219–231 (2024).
</cite> [<a href="https://doi.org/10.1038/s41593-023-01553-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38216650/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li,%20J.,%20Miramontes,%20T.%20G.,%20Czopka,%20T.%20&amp;%20Monk,%20K.%20R.%20Synaptic%20input%20and%20Ca2+%20activity%20in%20zebrafish%20oligodendrocyte%20precursor%20cells%20contribute%20to%20myelin%20sheath%20formation.%20Nat.%20Neurosci.27,%20219%E2%80%93231%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR37">
<span class="label">37.</span><cite>Doerflinger, N. H., Macklin, W. B. &amp; Popko, B. Inducible site-specific recombination in myelinating cells. <em>Genes. N. Y. N. 2000</em><strong>35</strong>, 63–72 (2003).</cite> [<a href="https://doi.org/10.1002/gene.10154" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12481300/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Doerflinger,%20N.%20H.,%20Macklin,%20W.%20B.%20&amp;%20Popko,%20B.%20Inducible%20site-specific%20recombination%20in%20myelinating%20cells.%20Genes.%20N.%20Y.%20N.%20200035,%2063%E2%80%9372%20(2003)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR38">
<span class="label">38.</span><cite>Orthmann-Murphy, J. et al. Remyelination alters the pattern of myelin in the cerebral cortex. <em>eLife</em><strong>9</strong>, e56621 (2020).
</cite> [<a href="https://doi.org/10.7554/eLife.56621" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7292648/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32459173/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Orthmann-Murphy,%20J.%20et%20al.%20Remyelination%20alters%20the%20pattern%20of%20myelin%20in%20the%20cerebral%20cortex.%20eLife9,%20e56621%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR39">
<span class="label">39.</span><cite>Call, C. L. &amp; Bergles, D. E. Cortical neurons exhibit diverse myelination patterns that scale between mouse brain regions and regenerate after demyelination. <em>Nat. Commun.</em><strong>12</strong>, 4767 (2021).
</cite> [<a href="https://doi.org/10.1038/s41467-021-25035-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8346564/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34362912/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Call,%20C.%20L.%20&amp;%20Bergles,%20D.%20E.%20Cortical%20neurons%20exhibit%20diverse%20myelination%20patterns%20that%20scale%20between%20mouse%20brain%20regions%20and%20regenerate%20after%20demyelination.%20Nat.%20Commun.12,%204767%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR40">
<span class="label">40.</span><cite>Almeida, R. G. The Rules of attraction in central nervous system myelination. <em>Front. Cell. Neurosci</em>. <strong>12</strong>, 367 (2018).</cite> [<a href="https://doi.org/10.3389/fncel.2018.00367" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6196289/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30374292/" class="usa-link">PubMed</a>]</li>
<li id="CR41">
<span class="label">41.</span><cite>Klingseisen, A. et al. Oligodendrocyte neurofascin independently regulates both myelin targeting and sheath growth in the CNS. <em>Dev. Cell</em><strong>51</strong>, 730–744.e6 (2019).
</cite> [<a href="https://doi.org/10.1016/j.devcel.2019.10.016" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6912162/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31761670/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Klingseisen,%20A.%20et%20al.%20Oligodendrocyte%20neurofascin%20independently%20regulates%20both%20myelin%20targeting%20and%20sheath%20growth%20in%20the%20CNS.%20Dev.%20Cell51,%20730%E2%80%93744.e6%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR42">
<span class="label">42.</span><cite>Redmond, S. A. et al. Somatodendritic expression of JAM2 inhibits oligodendrocyte myelination. <em>Neuron</em><strong>91</strong>, 824–836 (2016).
</cite> [<a href="https://doi.org/10.1016/j.neuron.2016.07.021" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4990461/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27499083/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Redmond,%20S.%20A.%20et%20al.%20Somatodendritic%20expression%20of%20JAM2%20inhibits%20oligodendrocyte%20myelination.%20Neuron91,%20824%E2%80%93836%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR43">
<span class="label">43.</span><cite>Barron, T., Saifetiarova, J., Bhat, M. A. &amp; Kim, J. H. Myelination of purkinje axons is critical for resilient synaptic transmission in the deep cerebellar nucleus. <em>Sci. Rep.</em><strong>8</strong>, 1022 (2018).
</cite> [<a href="https://doi.org/10.1038/s41598-018-19314-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5773691/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29348594/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Barron,%20T.,%20Saifetiarova,%20J.,%20Bhat,%20M.%20A.%20&amp;%20Kim,%20J.%20H.%20Myelination%20of%20purkinje%20axons%20is%20critical%20for%20resilient%20synaptic%20transmission%20in%20the%20deep%20cerebellar%20nucleus.%20Sci.%20Rep.8,%201022%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR44">
<span class="label">44.</span><cite>Chung, S.-H., Guo, F., Jiang, P., Pleasure, D. E. &amp; Deng, W. Olig2/Plp-positive progenitor cells give rise to Bergmann glia in the cerebellum. <em>Cell Death Dis.</em><strong>4</strong>, e546 (2013).
</cite> [<a href="https://doi.org/10.1038/cddis.2013.74" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3615735/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23492777/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chung,%20S.-H.,%20Guo,%20F.,%20Jiang,%20P.,%20Pleasure,%20D.%20E.%20&amp;%20Deng,%20W.%20Olig2/Plp-positive%20progenitor%20cells%20give%20rise%20to%20Bergmann%20glia%20in%20the%20cerebellum.%20Cell%20Death%20Dis.4,%20e546%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR45">
<span class="label">45.</span><cite>Koike, M. et al. Purkinje cells are more vulnerable to the specific depletion of cathepsin D than to that of Atg7. <em>Am. J. Pathol.</em><strong>187</strong>, 1586–1600 (2017).
</cite> [<a href="https://doi.org/10.1016/j.ajpath.2017.02.020" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28502476/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Koike,%20M.%20et%20al.%20Purkinje%20cells%20are%20more%20vulnerable%20to%20the%20specific%20depletion%20of%20cathepsin%20D%20than%20to%20that%20of%20Atg7.%20Am.%20J.%20Pathol.187,%201586%E2%80%931600%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR46">
<span class="label">46.</span><cite>Winston, J. T., Koepp, D. M., Zhu, C., Elledge, S. J. &amp; Harper, J. W. A family of mammalian F-box proteins. <em>Curr. Biol.</em><strong>9</strong>, 1180–S3 (1999).
</cite> [<a href="https://doi.org/10.1016/S0960-9822(00)80021-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10531037/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Winston,%20J.%20T.,%20Koepp,%20D.%20M.,%20Zhu,%20C.,%20Elledge,%20S.%20J.%20&amp;%20Harper,%20J.%20W.%20A%20family%20of%20mammalian%20F-box%20proteins.%20Curr.%20Biol.9,%201180%E2%80%93S3%20(1999)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR47">
<span class="label">47.</span><cite>Zhang, J. et al. Rack1 protects N-terminal phosphorylated c-Jun from Fbw7-mediated degradation. <em>Oncogene</em><strong>31</strong>, 1835–1844 (2012).
</cite> [<a href="https://doi.org/10.1038/onc.2011.369" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21860413/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhang,%20J.%20et%20al.%20Rack1%20protects%20N-terminal%20phosphorylated%20c-Jun%20from%20Fbw7-mediated%20degradation.%20Oncogene31,%201835%E2%80%931844%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR48">
<span class="label">48.</span><cite>Richter, K. T., Kschonsak, Y. T., Vodicska, B. &amp; Hoffmann, I. FBXO45-MYCBP2 regulates mitotic cell fate by targeting FBXW7 for degradation. <em>Cell Death Differ.</em><strong>27</strong>, 758–772 (2020).
</cite> [<a href="https://doi.org/10.1038/s41418-019-0385-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7206008/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31285543/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Richter,%20K.%20T.,%20Kschonsak,%20Y.%20T.,%20Vodicska,%20B.%20&amp;%20Hoffmann,%20I.%20FBXO45-MYCBP2%20regulates%20mitotic%20cell%20fate%20by%20targeting%20FBXW7%20for%20degradation.%20Cell%20Death%20Differ.27,%20758%E2%80%93772%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR49">
<span class="label">49.</span><cite>Hornig, J. et al. The transcription factors Sox10 and Myrf define an essential regulatory network module in differentiating oligodendrocytes. <em>PLOS Genet</em><strong>9</strong>, e1003907 (2013).
</cite> [<a href="https://doi.org/10.1371/journal.pgen.1003907" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3814293/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24204311/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hornig,%20J.%20et%20al.%20The%20transcription%20factors%20Sox10%20and%20Myrf%20define%20an%20essential%20regulatory%20network%20module%20in%20differentiating%20oligodendrocytes.%20PLOS%20Genet9,%20e1003907%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR50">
<span class="label">50.</span><cite>Bujalka, H. et al. MYRF is a membrane-associated transcription factor that autoproteolytically cleaves to directly activate myelin genes. <em>PLOS Biol.</em><strong>11</strong>, e1001625 (2013).
</cite> [<a href="https://doi.org/10.1371/journal.pbio.1001625" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3742440/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23966833/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bujalka,%20H.%20et%20al.%20MYRF%20is%20a%20membrane-associated%20transcription%20factor%20that%20autoproteolytically%20cleaves%20to%20directly%20activate%20myelin%20genes.%20PLOS%20Biol.11,%20e1001625%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR51">
<span class="label">51.</span><cite>Li, Z., Park, Y. &amp; Marcotte, E. M. A bacteriophage tailspike domain promotes self-cleavage of a human membrane-bound transcription factor, the myelin regulatory factor MYRF. <em>PLOS Biol.</em><strong>11</strong>, e1001624 (2013).
</cite> [<a href="https://doi.org/10.1371/journal.pbio.1001624" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3742443/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23966832/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li,%20Z.,%20Park,%20Y.%20&amp;%20Marcotte,%20E.%20M.%20A%20bacteriophage%20tailspike%20domain%20promotes%20self-cleavage%20of%20a%20human%20membrane-bound%20transcription%20factor,%20the%20myelin%20regulatory%20factor%20MYRF.%20PLOS%20Biol.11,%20e1001624%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR52">
<span class="label">52.</span><cite>Aprato, J. et al. Myrf guides target gene selection of transcription factor Sox10 during oligodendroglial development. <em>Nucleic Acids Res</em><strong>48</strong>, 1254–1270 (2020).
</cite> [<a href="https://doi.org/10.1093/nar/gkz1158" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7026603/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31828317/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Aprato,%20J.%20et%20al.%20Myrf%20guides%20target%20gene%20selection%20of%20transcription%20factor%20Sox10%20during%20oligodendroglial%20development.%20Nucleic%20Acids%20Res48,%201254%E2%80%931270%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR53">
<span class="label">53.</span><cite>Hartley, M. D. et al. Myelin repair stimulated by CNS-selective thyroid hormone action. <em>JCI Insight</em><strong>4</strong>, e126329 (2019).</cite> [<a href="https://doi.org/10.1172/jci.insight.126329" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6538346/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30996143/" class="usa-link">PubMed</a>]</li>
<li id="CR54">
<span class="label">54.</span><cite>Duncan, G. J. et al. Remyelination protects neurons from DLK-mediated neurodegeneration. <em>Nat. Commun.</em><strong>15</strong>, 9148 (2024).
</cite> [<a href="https://doi.org/10.1038/s41467-024-53429-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11500002/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39443516/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Duncan,%20G.%20J.%20et%20al.%20Remyelination%20protects%20neurons%20from%20DLK-mediated%20neurodegeneration.%20Nat.%20Commun.15,%209148%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR55">
<span class="label">55.</span><cite>Koenning, M. et al. Myelin gene regulatory factor is required for maintenance of myelin and mature oligodendrocyte identity in the adult CNS. <em>J. Neurosci.</em><strong>32</strong>, 12528–12542 (2012).
</cite> [<a href="https://doi.org/10.1523/JNEUROSCI.1069-12.2012" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3752083/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22956843/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Koenning,%20M.%20et%20al.%20Myelin%20gene%20regulatory%20factor%20is%20required%20for%20maintenance%20of%20myelin%20and%20mature%20oligodendrocyte%20identity%20in%20the%20adult%20CNS.%20J.%20Neurosci.32,%2012528%E2%80%9312542%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR56">
<span class="label">56.</span><cite>Jin, J., Ang, X. L., Shirogane, T. &amp; Wade Harper, J. <em>Identification of Substrates For F-box Proteins</em>. in <em>Methods in Enzymology</em>. <strong>399</strong>, 287–309 (Academic Press, 2005).</cite> [<a href="https://doi.org/10.1016/S0076-6879(05)99020-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16338364/" class="usa-link">PubMed</a>]</li>
<li id="CR57">
<span class="label">57.</span><cite>Madden, M. E. et al. CNS hypomyelination disrupts axonal conduction and behavior in larval zebrafish. <em>J. Soc. Neurosci.</em><strong>41</strong>, 9099–9111 (2021).</cite> [<a href="https://doi.org/10.1523/JNEUROSCI.0842-21.2021" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8570833/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34544838/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Madden,%20M.%20E.%20et%20al.%20CNS%20hypomyelination%20disrupts%20axonal%20conduction%20and%20behavior%20in%20larval%20zebrafish.%20J.%20Soc.%20Neurosci.41,%209099%E2%80%939111%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR58">
<span class="label">58.</span><cite>Hines, J. H., Ravanelli, A. M., Schwindt, R., Scott, E. K. &amp; Appel, B. Neuronal activity biases axon selection for myelination in vivo. <em>Nat. Neurosci.</em><strong>18</strong>, 683–689 (2015).
</cite> [<a href="https://doi.org/10.1038/nn.3992" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4414883/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25849987/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hines,%20J.%20H.,%20Ravanelli,%20A.%20M.,%20Schwindt,%20R.,%20Scott,%20E.%20K.%20&amp;%20Appel,%20B.%20Neuronal%20activity%20biases%20axon%20selection%20for%20myelination%20in%20vivo.%20Nat.%20Neurosci.18,%20683%E2%80%93689%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR59">
<span class="label">59.</span><cite>Rosenbluth, J. Redundant myelin sheaths and other ultrastructural features of the toad cerebellum. <em>J. Cell Biol.</em><strong>28</strong>, 73–93 (1966).
</cite> [<a href="https://doi.org/10.1083/jcb.28.1.73" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2106890/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/5901501/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rosenbluth,%20J.%20Redundant%20myelin%20sheaths%20and%20other%20ultrastructural%20features%20of%20the%20toad%20cerebellum.%20J.%20Cell%20Biol.28,%2073%E2%80%9393%20(1966)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR60">
<span class="label">60.</span><cite>Cooper, M. H. &amp; Beal, J. A. Myelinated granule cell bodies in the cerebellum of the monkey (Saimiri sciureus). <em>Anat. Rec.</em><strong>187</strong>, 249–255 (1977).
</cite> [<a href="https://doi.org/10.1002/ar.1091870210" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/403832/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cooper,%20M.%20H.%20&amp;%20Beal,%20J.%20A.%20Myelinated%20granule%20cell%20bodies%20in%20the%20cerebellum%20of%20the%20monkey%20(Saimiri%20sciureus).%20Anat.%20Rec.187,%20249%E2%80%93255%20(1977)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR61">
<span class="label">61.</span><cite>Katanov, C. et al. N-wasp regulates oligodendrocyte myelination. <em>J. Neurosci.</em><strong>40</strong>, 6103–6111 (2020).
</cite> [<a href="https://doi.org/10.1523/JNEUROSCI.0912-20.2020" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7406274/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32601246/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Katanov,%20C.%20et%20al.%20N-wasp%20regulates%20oligodendrocyte%20myelination.%20J.%20Neurosci.40,%206103%E2%80%936111%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR62">
<span class="label">62.</span><cite>Elazar, N. et al. Axoglial adhesion by Cadm4 regulates CNS myelination. <em>Neuron</em><strong>101</strong>, 224–231.e5 (2019).
</cite> [<a href="https://doi.org/10.1016/j.neuron.2018.11.032" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6371057/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30551998/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Elazar,%20N.%20et%20al.%20Axoglial%20adhesion%20by%20Cadm4%20regulates%20CNS%20myelination.%20Neuron101,%20224%E2%80%93231.e5%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR63">
<span class="label">63.</span><cite>Jaegle, M. et al. The POU proteins Brn-2 and Oct-6 share important functions in Schwann cell development. <em>Genes Dev.</em><strong>17</strong>, 1380–1391 (2003).
</cite> [<a href="https://doi.org/10.1101/gad.258203" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC196070/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12782656/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jaegle,%20M.%20et%20al.%20The%20POU%20proteins%20Brn-2%20and%20Oct-6%20share%20important%20functions%20in%20Schwann%20cell%20development.%20Genes%20Dev.17,%201380%E2%80%931391%20(2003)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR64">
<span class="label">64.</span><cite>Kipreos, E. T. &amp; Pagano, M. The F-box protein family. <em>Genome Biol.</em><strong>1</strong>, 3002 (2000).</cite> [<a href="https://doi.org/10.1186/gb-2000-1-5-reviews3002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC138887/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11178263/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kipreos,%20E.%20T.%20&amp;%20Pagano,%20M.%20The%20F-box%20protein%20family.%20Genome%20Biol.1,%203002%20(2000)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR65">
<span class="label">65.</span><cite>Elbaz, B. et al. Phosphorylation state of ZFP24 controls oligodendrocyte differentiation. <em>Cell Rep.</em><strong>23</strong>, 2254–2263 (2018).
</cite> [<a href="https://doi.org/10.1016/j.celrep.2018.04.089" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6002757/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29791837/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Elbaz,%20B.%20et%20al.%20Phosphorylation%20state%20of%20ZFP24%20controls%20oligodendrocyte%20differentiation.%20Cell%20Rep.23,%202254%E2%80%932263%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR66">
<span class="label">66.</span><cite>Kornfeld, S. F., Cummings, S. E., Fathi, S., Bonin, S. R. &amp; Kothary, R. MiRNA-145-5p prevents differentiation of oligodendrocyte progenitor cells by regulating expression of myelin gene regulatory factor. <em>J. Cell. Physiol.</em><strong>236</strong>, 997–1012 (2021).
</cite> [<a href="https://doi.org/10.1002/jcp.29910" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32602617/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kornfeld,%20S.%20F.,%20Cummings,%20S.%20E.,%20Fathi,%20S.,%20Bonin,%20S.%20R.%20&amp;%20Kothary,%20R.%20MiRNA-145-5p%20prevents%20differentiation%20of%20oligodendrocyte%20progenitor%20cells%20by%20regulating%20expression%20of%20myelin%20gene%20regulatory%20factor.%20J.%20Cell.%20Physiol.236,%20997%E2%80%931012%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR67">
<span class="label">67.</span><cite>Garnai, S. J. et al. Variants in myelin regulatory factor (MYRF) cause autosomal dominant and syndromic nanophthalmos in humans and retinal degeneration in mice. <em>PLOS Genet</em><strong>15</strong>, e1008130 (2019).
</cite> [<a href="https://doi.org/10.1371/journal.pgen.1008130" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6527243/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31048900/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Garnai,%20S.%20J.%20et%20al.%20Variants%20in%20myelin%20regulatory%20factor%20(MYRF)%20cause%20autosomal%20dominant%20and%20syndromic%20nanophthalmos%20in%20humans%20and%20retinal%20degeneration%20in%20mice.%20PLOS%20Genet15,%20e1008130%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR68">
<span class="label">68.</span><cite>Huang, H. et al. Interactive repression of MYRF self-cleavage and activity in oligodendrocyte differentiation by TMEM98 protein. <em>J. Neurosci.</em><strong>38</strong>, 9829–9839 (2018).
</cite> [<a href="https://doi.org/10.1523/JNEUROSCI.0154-18.2018" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6596239/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30249802/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Huang,%20H.%20et%20al.%20Interactive%20repression%20of%20MYRF%20self-cleavage%20and%20activity%20in%20oligodendrocyte%20differentiation%20by%20TMEM98%20protein.%20J.%20Neurosci.38,%209829%E2%80%939839%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR69">
<span class="label">69.</span><cite>Djannatian, M. et al. Two adhesive systems cooperatively regulate axon ensheathment and myelin growth in the CNS. <em>Nat. Commun.</em><strong>10</strong>, 4794 (2019).
</cite> [<a href="https://doi.org/10.1038/s41467-019-12789-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6805957/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31641127/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Djannatian,%20M.%20et%20al.%20Two%20adhesive%20systems%20cooperatively%20regulate%20axon%20ensheathment%20and%20myelin%20growth%20in%20the%20CNS.%20Nat.%20Commun.10,%204794%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR70">
<span class="label">70.</span><cite>Almeida, R. G., Czopka, T., ffrench-Constant, C. &amp; Lyons, D. A. Individual axons regulate the myelinating potential of single oligodendrocytes in vivo. <em>Development</em><strong>138</strong>, 4443–4450 (2011).
</cite> [<a href="https://doi.org/10.1242/dev.071001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3177314/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21880787/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Almeida,%20R.%20G.,%20Czopka,%20T.,%20ffrench-Constant,%20C.%20&amp;%20Lyons,%20D.%20A.%20Individual%20axons%20regulate%20the%20myelinating%20potential%20of%20single%20oligodendrocytes%20in%20vivo.%20Development138,%204443%E2%80%934450%20(2011)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR71">
<span class="label">71.</span><cite>Kim, H. et al. Notch-regulated perineurium development from zebrafish spinal cord. <em>Neurosci. Lett.</em><strong>448</strong>, 240–244 (2008).
</cite> [<a href="https://doi.org/10.1016/j.neulet.2008.10.072" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18977412/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kim,%20H.%20et%20al.%20Notch-regulated%20perineurium%20development%20from%20zebrafish%20spinal%20cord.%20Neurosci.%20Lett.448,%20240%E2%80%93244%20(2008)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR72">
<span class="label">72.</span><cite>Karttunen, M. J., Czopka, T., Goedhart, M., Early, J. J. &amp; Lyons, D. A. Regeneration of myelin sheaths of normal length and thickness in the zebrafish CNS correlates with growth of axons in caliber. <em>PLOS ONE</em><strong>12</strong>, e0178058 (2017).
</cite> [<a href="https://doi.org/10.1371/journal.pone.0178058" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5444792/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28542521/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Karttunen,%20M.%20J.,%20Czopka,%20T.,%20Goedhart,%20M.,%20Early,%20J.%20J.%20&amp;%20Lyons,%20D.%20A.%20Regeneration%20of%20myelin%20sheaths%20of%20normal%20length%20and%20thickness%20in%20the%20zebrafish%20CNS%20correlates%20with%20growth%20of%20axons%20in%20caliber.%20PLOS%20ONE12,%20e0178058%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR73">
<span class="label">73.</span><cite>Peri, F. &amp; Nüsslein-Volhard, C. Live imaging of neuronal degradation by microglia reveals a role for v0-ATPase a1 in phagosomal fusion in vivo. <em>Cell</em><strong>133</strong>, 916–927 (2008).
</cite> [<a href="https://doi.org/10.1016/j.cell.2008.04.037" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18510934/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Peri,%20F.%20&amp;%20N%C3%BCsslein-Volhard,%20C.%20Live%20imaging%20of%20neuronal%20degradation%20by%20microglia%20reveals%20a%20role%20for%20v0-ATPase%20a1%20in%20phagosomal%20fusion%20in%20vivo.%20Cell133,%20916%E2%80%93927%20(2008)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR74">
<span class="label">74.</span><cite>Labun, K. et al. CHOPCHOP v3: expanding the CRISPR web toolbox beyond genome editing. <em>Nucleic Acids Res</em><strong>47</strong>, W171–W174 (2019).
</cite> [<a href="https://doi.org/10.1093/nar/gkz365" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6602426/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31106371/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Labun,%20K.%20et%20al.%20CHOPCHOP%20v3:%20expanding%20the%20CRISPR%20web%20toolbox%20beyond%20genome%20editing.%20Nucleic%20Acids%20Res47,%20W171%E2%80%93W174%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR75">
<span class="label">75.</span><cite>Almeida, R. G. &amp; Lyons, D. A. Intersectional gene expression in zebrafish using the split KalTA4 system. <em>Zebrafish</em><strong>12</strong>, 377–386 (2015).
</cite> [<a href="https://doi.org/10.1089/zeb.2015.1086" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4677521/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26485616/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Almeida,%20R.%20G.%20&amp;%20Lyons,%20D.%20A.%20Intersectional%20gene%20expression%20in%20zebrafish%20using%20the%20split%20KalTA4%20system.%20Zebrafish12,%20377%E2%80%93386%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR76">
<span class="label">76.</span><cite>Dugas, J. C. &amp; Emery, B. Purification of oligodendrocyte precursor cells from rat cortices by immunopanning. <em>Cold Spring Harb. Protoc.</em><strong>2013</strong>, pdb.prot070862 (2013).</cite> [<a href="https://doi.org/10.1101/pdb.prot070862" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23906908/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Dugas,%20J.%20C.%20&amp;%20Emery,%20B.%20Purification%20of%20oligodendrocyte%20precursor%20cells%20from%20rat%20cortices%20by%20immunopanning.%20Cold%20Spring%20Harb.%20Protoc.2013,%20pdb.prot070862%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR77">
<span class="label">77.</span><cite>Wilmarth, P. A., Riviere, M. A. &amp; David, L. L. Techniques for accurate protein identification in shotgun proteomic studies of human, mouse, bovine, and chicken lenses. <em>J. Ocul. Biol. Dis. Info</em><strong>2</strong>, 223–234 (2009).</cite> [<a href="https://doi.org/10.1007/s12177-009-9042-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2816815/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20157357/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wilmarth,%20P.%20A.,%20Riviere,%20M.%20A.%20&amp;%20David,%20L.%20L.%20Techniques%20for%20accurate%20protein%20identification%20in%20shotgun%20proteomic%20studies%20of%20human,%20mouse,%20bovine,%20and%20chicken%20lenses.%20J.%20Ocul.%20Biol.%20Dis.%20Info2,%20223%E2%80%93234%20(2009)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR78">
<span class="label">78.</span><cite>Eng, J. K., Jahan, T. A. &amp; Hoopmann, M. R. Comet: an open-source MS/MS sequence database search tool. <em>Proteomics</em><strong>13</strong>, 22–24 (2013).
</cite> [<a href="https://doi.org/10.1002/pmic.201200439" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23148064/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Eng,%20J.%20K.,%20Jahan,%20T.%20A.%20&amp;%20Hoopmann,%20M.%20R.%20Comet:%20an%20open-source%20MS/MS%20sequence%20database%20search%20tool.%20Proteomics13,%2022%E2%80%9324%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR79">
<span class="label">79.</span><cite>Perez-Riverol, Y. et al. The PRIDE database at 20 years: 2025 update. <em>Nucleic Acids Res</em><strong>53</strong>, D543–D553 (2024).</cite> [<a href="https://doi.org/10.1093/nar/gkae1011" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11701690/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39494541/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Perez-Riverol,%20Y.%20et%20al.%20The%20PRIDE%20database%20at%2020%20years:%202025%20update.%20Nucleic%20Acids%20Res53,%20D543%E2%80%93D553%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR80">
<span class="label">80.</span><cite>Deutsch, E. W. et al. The ProteomeXchange consortium at 10 years: 2023 update. <em>Nucleic Acids Res</em><strong>51</strong>, D1539–D1548 (2023).
</cite> [<a href="https://doi.org/10.1093/nar/gkac1040" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9825490/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36370099/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Deutsch,%20E.%20W.%20et%20al.%20The%20ProteomeXchange%20consortium%20at%2010%20years:%202023%20update.%20Nucleic%20Acids%20Res51,%20D1539%E2%80%93D1548%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR81">
<span class="label">81.</span><cite>Perez-Riverol, Y. et al. PRIDE inspector toolsuite: moving toward a universal visualization tool for proteomics data standard formats and quality assessment of ProteomeXchange datasets. <em>Mol. Cell. Proteom. MCP</em><strong>15</strong>, 305–317 (2016).
</cite> [<a href="https://doi.org/10.1074/mcp.O115.050229" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4762524/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26545397/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Perez-Riverol,%20Y.%20et%20al.%20PRIDE%20inspector%20toolsuite:%20moving%20toward%20a%20universal%20visualization%20tool%20for%20proteomics%20data%20standard%20formats%20and%20quality%20assessment%20of%20ProteomeXchange%20datasets.%20Mol.%20Cell.%20Proteom.%20MCP15,%20305%E2%80%93317%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12370921/bin/41467_2025_62715_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information</a><sup> (7.8MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12370921/bin/41467_2025_62715_MOESM2_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">41467_2025_62715_MOESM2_ESM.pdf</a><sup> (24.6KB, pdf) </sup><p>Description of Additional Supplementary Files</p>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12370921/bin/41467_2025_62715_MOESM3_ESM.avi" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Movie 1</a><sup> (37.3MB, avi) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material4_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12370921/bin/41467_2025_62715_MOESM4_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Dataset 1</a><sup> (149.6KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material5_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12370921/bin/41467_2025_62715_MOESM5_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Reporting Summary</a><sup> (149.4KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material6_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12370921/bin/41467_2025_62715_MOESM6_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Transparent Peer Review file</a><sup> (222.1KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material7_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12370921/bin/41467_2025_62715_MOESM7_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Source Data</a><sup> (96MB, xlsx) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>Source data are provided with this paper and mass spectrometry data is available via ProteomeXchange with identifier PXD063570: (<a href="https://www.ebi.ac.uk/pride/archive/projects/PXD063570" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ebi.ac.uk/pride/archive/projects/PXD063570</a>). <a href="#Sec32" class="usa-link">Source data</a> are provided with this paper.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Nature Communications are provided here courtesy of <strong>Nature Publishing Group</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1038/s41467-025-62715-9"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/41467_2025_Article_62715.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (8.9 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12370921/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12370921/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12370921%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370921/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12370921/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12370921/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40841354/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12370921/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40841354/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12370921/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12370921/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="Oc8IerN5cwfLKX5dVOqlqVRkuF7k5SqcnYX1ffAhlXs3tyCyTc4oa7Ie0CQAYX9r">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
