
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Isotopologue Ratios Identify 13C‐Depleted Biomarkers in Environmental Samples Impacted by Methane Turnover - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="D2E481798AEEC7F305817900048F1A7E.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="blackwellopen">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12322946/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Rapid Communications in Mass Spectrometry">
<meta name="citation_title" content="Isotopologue Ratios Identify 13C‐Depleted Biomarkers in Environmental Samples Impacted by Methane Turnover">
<meta name="citation_author" content="Janina Groninga">
<meta name="citation_author_institution" content="MARUM – Center for Marine Environmental Sciences, and Faculty of Geosciences, University of Bremen, Bremen, Germany">
<meta name="citation_author" content="Julius Lipp">
<meta name="citation_author_institution" content="MARUM – Center for Marine Environmental Sciences, and Faculty of Geosciences, University of Bremen, Bremen, Germany">
<meta name="citation_author" content="Min Song">
<meta name="citation_author_institution" content="MARUM – Center for Marine Environmental Sciences, and Faculty of Geosciences, University of Bremen, Bremen, Germany">
<meta name="citation_author_institution" content="Department of Earth Sciences, University of Toronto, Toronto, Ontario, Canada">
<meta name="citation_author" content="Kai‐Uwe Hinrichs">
<meta name="citation_author_institution" content="MARUM – Center for Marine Environmental Sciences, and Faculty of Geosciences, University of Bremen, Bremen, Germany">
<meta name="citation_publication_date" content="2025 Aug 5">
<meta name="citation_volume" content="39">
<meta name="citation_issue" content="22">
<meta name="citation_firstpage" content="e10118">
<meta name="citation_doi" content="10.1002/rcm.10118">
<meta name="citation_pmid" content="40762245">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12322946/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12322946/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12322946/pdf/RCM-39-e10118.pdf">
<meta name="description" content="The stable carbon isotopic composition (δ13C) of individual lipids is of great value in studying carbon cycling. Among those, microbial lipids in sediments impacted by high methane turnover stand out due to their uniquely depleted isotopic ...">
<meta name="og:title" content="Isotopologue Ratios Identify 13C‐Depleted Biomarkers in Environmental Samples Impacted by Methane Turnover">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="The stable carbon isotopic composition (δ13C) of individual lipids is of great value in studying carbon cycling. Among those, microbial lipids in sediments impacted by high methane turnover stand out due to their uniquely depleted isotopic ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12322946/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12322946">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1002/rcm.10118"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/RCM-39-e10118.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12322946%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12322946/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12322946/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12322946/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-blackwellopen.png" alt="Wiley Open Access Collection logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Wiley Open Access Collection" title="Link to Wiley Open Access Collection" shape="default" href="https://doi.org/10.1002/rcm.10118" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Rapid Commun Mass Spectrom</button></div>. 2025 Aug 5;39(22):e10118. doi: <a href="https://doi.org/10.1002/rcm.10118" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1002/rcm.10118</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Rapid%20Commun%20Mass%20Spectrom%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Rapid%20Commun%20Mass%20Spectrom%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Rapid%20Commun%20Mass%20Spectrom%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Rapid%20Commun%20Mass%20Spectrom%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Isotopologue Ratios Identify <sup>13</sup>C‐Depleted Biomarkers in Environmental Samples Impacted by Methane Turnover</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Groninga%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Janina Groninga</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Janina Groninga</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>MARUM – Center for Marine Environmental Sciences, and Faculty of Geosciences, University of Bremen, Bremen, Germany</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Groninga%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Janina Groninga</span></a>
</div>
</div>
<sup>1,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lipp%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Julius Lipp</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Julius Lipp</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>MARUM – Center for Marine Environmental Sciences, and Faculty of Geosciences, University of Bremen, Bremen, Germany</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lipp%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Julius Lipp</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Song%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Min Song</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Min Song</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>MARUM – Center for Marine Environmental Sciences, and Faculty of Geosciences, University of Bremen, Bremen, Germany</div>
<div class="p">
<sup>
<sup>2</sup>
</sup>Department of Earth Sciences, University of Toronto, Toronto, Ontario, Canada</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Song%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Min Song</span></a>
</div>
</div>
<sup>1,</sup><sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hinrichs%20KU%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Kai‐Uwe Hinrichs</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Kai‐Uwe Hinrichs</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>MARUM – Center for Marine Environmental Sciences, and Faculty of Geosciences, University of Bremen, Bremen, Germany</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hinrichs%20KU%22%5BAuthor%5D" class="usa-link"><span class="name western">Kai‐Uwe Hinrichs</span></a>
</div>
</div>
<sup>1</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="rcm10118-aff-0001">
<sup>
<sup>1</sup>
</sup>MARUM – Center for Marine Environmental Sciences, and Faculty of Geosciences, University of Bremen, Bremen, Germany</div>
<div id="rcm10118-aff-0002">
<sup>
<sup>2</sup>
</sup>Department of Earth Sciences, University of Toronto, Toronto, Ontario, Canada</div>
<div class="author-notes p">
<div class="fn" id="correspondenceTo">
<sup>*</sup><p class="display-inline">

<strong>Correspondence:</strong>
 
Janina Groninga (<span>jgroninga@marum.de</span>)
</p>
</div>
<div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Revised 2025 May 30; Received 2024 Dec 13; Accepted 2025 Jul 28; Issue date 2025 Nov 30.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© 2025 The Author(s). <em>Rapid Communications in Mass Spectrometry</em> published by John Wiley &amp; Sons Ltd.</div>
<p>This is an open access article under the terms of the <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by/4.0/</a> License, which permits use, distribution and reproduction in any medium, provided the original work is properly cited.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12322946  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40762245/" class="usa-link">40762245</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>ABSTRACT</h2>
<section id="rcm10118-sec-0001"><h3 class="pmc_sec_title">Rationale</h3>
<p>The stable carbon isotopic composition (δ<sup>13</sup>C) of individual lipids is of great value in studying carbon cycling. Among those, microbial lipids in sediments impacted by high methane turnover stand out due to their uniquely depleted isotopic fingerprint. However, gas chromatography/isotope ratio mass spectrometry (GC/irMS) is limited to volatile compounds, whereas intact polar lipids require extensive preprocessing, which results in the loss of chemotaxonomic information. Expanding compound‐specific isotopic information to intact polar lipids would enhance insights into the microbial turnover of methane.</p></section><section id="rcm10118-sec-0002"><h3 class="pmc_sec_title">Methods</h3>
<p>We performed ultra‐high‐performance liquid chromatography/electrospray ionization/high‐resolution mass spectrometry (UHPLC/ESI/HRMS) to analyze standards of archaeol and lipid extracts from a diverse set of sediment samples of a hydrothermal methane seep system. Using the ratio of the M1 isotopologue over the monoisotopic isotopologue M0, we calculated the δ<sup>13</sup>C values of archaeol and various polar, non‐GC‐amenable lipids. The δ<sup>13</sup>C values of archaeol obtained via ratios were compared to those measured via GC/irMS.</p></section><section id="rcm10118-sec-0003"><h3 class="pmc_sec_title">Results</h3>
<p>δ<sup>13</sup>C values of archaeol determined in natural samples via GC/irMS and the UHPLC/HRMS approach were strongly correlated (<em>R</em>
<sup>2</sup> = 0.94; <em>N</em> = 76–82) across a wide range of δ<sup>13</sup>C values (GC‐irMS = −119‰ to −34‰). Biomarkers associated with methane turnover consistently yielded δ<sup>13</sup>C values below −60‰, whereas the δ<sup>13</sup>C values of compounds presumably associated with the photosynthesis‐based food web remained above −45‰. UHPLC/HRMS measurements of archaeol standard further indicated that δ<sup>13</sup>C values can be reliably determined across an M0 signal‐intensity range of approximately one order of magnitude.</p></section><section id="rcm10118-sec-0004"><h3 class="pmc_sec_title">Conclusions</h3>
<p>Our results highlight that the M1/M0 ratio from UHPLC/HRMS measurements can be utilized to evaluate the carbon isotopic fingerprint of non‐GC‐amenable lipids and to reliably detect lipid biomarkers putatively associated with microbial methane turnover carrying extremely depleted isotopic signatures. This paves the way for a comprehensive exploration of intact lipids associated with microbial methane turnover in environmental samples.</p></section><section id="Ace_Pre_6" class="kwd-group"><p><strong>Keywords:</strong> compound‐specific isotope analysis (CSIA), high‐resolution‐MS, methane turnover, microbial lipids, stable carbon isotopes, UHPLC/MS</p></section></section><section id="rcm10118-sec-0005"><h2 class="pmc_sec_title">1. Introduction</h2>
<p>The stable carbon isotopic composition (δ<sup>13</sup>C) of microbial lipids varies considerably due to the kinetic isotope effects of the carbon fixation pathway and the δ<sup>13</sup>C value of the assimilated carbon sources [<a href="#rcm10118-bib-0001" class="usa-link" aria-describedby="rcm10118-bib-0001">1</a>, <a href="#rcm10118-bib-0002" class="usa-link" aria-describedby="rcm10118-bib-0002">2</a>, <a href="#rcm10118-bib-0003" class="usa-link" aria-describedby="rcm10118-bib-0003">3</a>, <a href="#rcm10118-bib-0004" class="usa-link" aria-describedby="rcm10118-bib-0004">4</a>]. Particularly distinctive isotopic signatures can be observed in microbial lipids extracted from sediments with strong microbial turnover of methane. These lipids are usually characterized by extreme<sup>13</sup>C‐depletion with δ<sup>13</sup>C values ranging from −60‰ to −140‰ [<a href="#rcm10118-bib-0005" class="usa-link" aria-describedby="rcm10118-bib-0005">5</a>, <a href="#rcm10118-bib-0006" class="usa-link" aria-describedby="rcm10118-bib-0006">6</a>, <a href="#rcm10118-bib-0007" class="usa-link" aria-describedby="rcm10118-bib-0007">7</a>, <a href="#rcm10118-bib-0008" class="usa-link" aria-describedby="rcm10118-bib-0008">8</a>, <a href="#rcm10118-bib-0009" class="usa-link" aria-describedby="rcm10118-bib-0009">9</a>, <a href="#rcm10118-bib-0010" class="usa-link" aria-describedby="rcm10118-bib-0010">10</a>, <a href="#rcm10118-bib-0011" class="usa-link" aria-describedby="rcm10118-bib-0011">11</a>]. Such negative values in these environmental settings may originate from direct incorporation of methane‐derived carbon for biosynthesis by methane‐oxidizing archaea, as suggested in initial studies that observed such strong<sup>13</sup>C‐depletions [<a href="#rcm10118-bib-0005" class="usa-link" aria-describedby="rcm10118-bib-0005">5</a>, <a href="#rcm10118-bib-0012" class="usa-link" aria-describedby="rcm10118-bib-0012">12</a>]. Other possibilities are the incorporation of <sup>13</sup>C‐depleted methane‐derived CO<sub>2</sub> or organic intermediates by the bacterial partners [<a href="#rcm10118-bib-0012" class="usa-link" aria-describedby="rcm10118-bib-0012">12</a>], from chemoorganoautotrophic uptake of methane‐derived CO<sub>2</sub> by anaerobic methane‐oxidizing archaea [<a href="#rcm10118-bib-0013" class="usa-link" aria-describedby="rcm10118-bib-0013">13</a>, <a href="#rcm10118-bib-0014" class="usa-link" aria-describedby="rcm10118-bib-0014">14</a>], from heterotrophic consumption of relatively fresh biomass from anaerobic methane‐oxidizing sedimentary communities [<a href="#rcm10118-bib-0015" class="usa-link" aria-describedby="rcm10118-bib-0015">15</a>], or from co‐occurring autotrophic methanogens that incorporate strongly <sup>13</sup>C‐depleted CO<sub>2</sub>, which results from the high methane‐oxidizing activity in close proximity [<a href="#rcm10118-bib-0016" class="usa-link" aria-describedby="rcm10118-bib-0016">16</a>]. An additional mechanism resulting in strong <sup>13</sup>C‐depletion of archaeal lipids is lipid biosynthesis by certain methylotrophic methanogens [<a href="#rcm10118-bib-0017" class="usa-link" aria-describedby="rcm10118-bib-0017">17</a>, <a href="#rcm10118-bib-0018" class="usa-link" aria-describedby="rcm10118-bib-0018">18</a>], although the presence of such products has not been confirmed in natural settings. The wide range of isotopic values of typically −60‰ to −140‰ accordingly results from high variability of δ<sup>13</sup>C values of methane, depending on its source and overprint by oxidative processes [<a href="#rcm10118-bib-0019" class="usa-link" aria-describedby="rcm10118-bib-0019">19</a>], mixed sources of the lipids from microbes with different carbon metabolisms, the rates of methane oxidation, which determine the degree of<sup>13</sup>C‐depletion in the porewater's inorganic carbon pool, and possibly other physiological factors. No matter what the exact mechanism is, all the abovementioned pathways leading to the accumulation of strongly <sup>13</sup>C‐depleted lipids are associated with high rates of methane turnover, as typically found in seep settings, and clearly distinguish compounds associated with these processes from compounds associated with photosynthesis‐based food webs. These distinct isotopic fingerprints make compound‐specific isotope analysis (CSIA) an invaluable tool for investigating microbial methane turnover in modern and paleoenvironments.</p>
<p>A routine method for CSIA is gas chromatography/isotope ratio mass spectrometry (GC/irMS) [<a href="#rcm10118-bib-0020" class="usa-link" aria-describedby="rcm10118-bib-0020">20</a>, <a href="#rcm10118-bib-0021" class="usa-link" aria-describedby="rcm10118-bib-0021">21</a>]. Although it is a powerful tool in environmental sciences, it is not without limitations. GC/irMS is mainly restricted to GC‐amenable lipids, often requiring chemical derivatization, purification, and/or cleavage reactions, constraining the available lipid pool for δ<sup>13</sup>C determination. More complex and unmodified biomolecules with higher chemotaxonomic specificity, such as intact polar lipids (IPLs), fall outside the scope of GC/irMS. Recently, other methods facilitating isotope analysis of intact molecules have been established in the form of spooling wire microcombustion (SWiM)‐irMS [<a href="#rcm10118-bib-0022" class="usa-link" aria-describedby="rcm10118-bib-0022">22</a>]. However, to obtain accurate results, SWiM‐irMS necessitates intense sample purification, which can be challenging for complex environmental samples.</p>
<p>During the last decade, a promising alternative technique, which shifted focus onto the isotopologue pattern derived from ultra‐high‐performance liquid chromatography high‐resolution mass spectrometry (UHPLC/HRMS), emerged in protein‐based stable isotope labeling (SIP) studies [<a href="#rcm10118-bib-0023" class="usa-link" aria-describedby="rcm10118-bib-0023">23</a>, <a href="#rcm10118-bib-0024" class="usa-link" aria-describedby="rcm10118-bib-0024">24</a>, <a href="#rcm10118-bib-0025" class="usa-link" aria-describedby="rcm10118-bib-0025">25</a>] and has recently been adopted in lipidomics [<a href="#rcm10118-bib-0026" class="usa-link" aria-describedby="rcm10118-bib-0026">26</a>]. By comparing the theoretical isotope pattern of a molecule with an observed pattern from stable isotope amended experiments (e.g., <sup>13</sup>C, <sup>2</sup>H, and <sup>15</sup>N), it is possible to trace and quantify the incorporation of labeled substrates into peptides or lipids [<a href="#rcm10118-bib-0023" class="usa-link" aria-describedby="rcm10118-bib-0023">23</a>, <a href="#rcm10118-bib-0024" class="usa-link" aria-describedby="rcm10118-bib-0024">24</a>, <a href="#rcm10118-bib-0025" class="usa-link" aria-describedby="rcm10118-bib-0025">25</a>, <a href="#rcm10118-bib-0026" class="usa-link" aria-describedby="rcm10118-bib-0026">26</a>]. Nevertheless, these studies exclusively focus on stable isotope‐labeled biomolecules, and the application of naturally occurring isotope patterns in environmental samples has not been fully explored because variations in natural isotope compositions are substantially smaller than those resulting from SIP experiments. However, high microbial turnover of methane in sedimentary environments is associated with a unique isotopic fingerprint that may leave a distinctive and quantifiable imprint on the isotopologues.</p>
<p>Recent studies have already demonstrated that high‐resolution mass spectrometry (HRMS), such as Orbitrap‐MS, operating at resolving powers &gt; 100 000, can successfully determine the natural stable carbon isotopic composition of small organic molecules [<a href="#rcm10118-bib-0027" class="usa-link" aria-describedby="rcm10118-bib-0027">27</a>] such as amino acid [<a href="#rcm10118-bib-0028" class="usa-link" aria-describedby="rcm10118-bib-0028">28</a>] and acetate [<a href="#rcm10118-bib-0029" class="usa-link" aria-describedby="rcm10118-bib-0029">29</a>]. However, these HRMS‐based techniques remain limited to relatively simple analytes and are not yet applicable to more complex, intact molecules in environmental mixtures. To address this limitation, we propose an isotopologue‐based method that employs UHPLC coupled to electrospray ionization quadrupole time‐of‐flight (ESI‐QToF) mass spectrometry for CSIA of complex, chemotaxonomically valuable intact lipids. We explore to what extent δ<sup>13</sup>C values based on the ratio of the monoisotopic peak (M0; composed of the most abundant isotopes) and its first isotopologue (M1; one heavy isotope substituent) can differentiate <sup>13</sup>C‐depleted isotopic signatures from comparatively <sup>13</sup>C‐enriched compounds and lay out the conditions that must be fulfilled for reliable δ<sup>13</sup>C assessment. As the generally lower resolution of QToF‐MS does not allow a separation of isobaric isotopologues (<sup>13</sup>C vs. <sup>2</sup>H), the approach focuses on large isotopic differences and offers a high‐throughput solution that balances precision, analytical feasibility, and biogeochemical insight.</p></section><section id="rcm10118-sec-0006"><h2 class="pmc_sec_title">2. Experimental</h2>
<section id="rcm10118-sec-0007"><h3 class="pmc_sec_title">2.1. Sample Preparation</h3>
<p>The sediment cores for this study were collected during the R/V Atlantis cruises AT42‐05 (Core ID: 4992‐14 and 5000‐9; November 15–29, 2018) and AT37‐06 (December 9–27, 2016) to the southern Guaymas Basin spreading center (Core ID: 4869‐3, 4870‐28, 4871‐28) and at the off‐axis Central Seep site on the northwestern Guaymas flanks (Core ID: 4867‐3) using the ALVIN submersible. The total lipid extracts (TLE) for UHPLC/HRMS and GC/irMS analysis were obtained following the modified Bligh and Dyer protocol [<a href="#rcm10118-bib-0030" class="usa-link" aria-describedby="rcm10118-bib-0030">30</a>] by successive extraction with solvent mixtures of dichloromethane:methanol:buffer (DCM:MeOH:buffer, 1:2:0.8, v:v:v), with the buffer consisting of a solution of monopotassium phosphate (8.7 g L<sup>−1</sup>, pH 7.4) for the first two extraction steps and trichloroacetic acid (50 g L<sup>−1</sup>, pH 2) for the final two extraction steps. The extracts were combined, washed with deionized Milli‐Q water, dried under a stream of N<sub>2</sub>, and stored at −20°C until analysis. An aliquot of the TLE was further processed for GC/irMS measurement by separating the <em>n</em>‐hexane‐soluble fraction into three fractions with increasing polarity (F1 + F2: hydrocarbons + ketones, F3: alcohols, F4: fatty acids) using an aminopropyl solid‐phase extraction (SPE) cartridge [<a href="#rcm10118-bib-0012" class="usa-link" aria-describedby="rcm10118-bib-0012">12</a>]. The fractions were eluted with 4 mL of <em>n</em>‐hexane and 6 mL of <em>n</em>‐hexane/DCM (3:1) for F1 and F2, respectively, 7 mL of DCM/acetone (9:1) for F3, and 8 mL of 2% formic acid in DCM for the final F4 fraction. The alcohol fraction containing archaeol was converted to trimethylsilyl (TMS) derivatives by treatment with <em>N</em>,<em>O</em>‐bis(trimethylsilyl)fluoroacetamide (BSTFA; Merck, Germany) and pyridine (Carl Roth, Germany) at 70°C for 1 h. Additional archaeol standards (archaeol<sub>std</sub>; 4ME 16:0 Diether DG; Avanti Polar Lipids, USA) were analyzed alongside the natural samples as references.</p></section><section id="rcm10118-sec-0008"><h3 class="pmc_sec_title">2.2. GC/irMS and EA/irMS</h3>
<p>For stable carbon isotope analysis of archaeol, TMS‐derivatized aliquots of the alcohol fractions were injected into a Trace GC Ultra (ThermoFinnigan, Germany) equipped with a Restek Rxi‐5ms column (30 m × 250 μm × 0.25 μm, Restek, Bad Homburg, Germany) and coupled to a Delta V Plus irMS via a GC IsoLink and ConFlo IV interface (Thermo Fisher Scientific GmbH, Bremen, Germany). Injection was performed in splitless mode at 300°C, with a temperature program starting at 60°C for 1 min, increasing to 150°C at 10°C min<sup>−1</sup>, then ramping up to 320°C at a rate of 4°C min<sup>−1</sup>, followed by a final hold time of 17.5 min. The helium carrier gas was set to a constant flow of 1.0 mL min<sup>−1</sup>, and the oxidation oven was set to 940°C. The δ<sup>13</sup>C values were calibrated with CO<sub>2</sub> reference gas (δ<sup>13</sup>C = −33.4‰) at the beginning and end of each run and later corrected for the three additional carbon atoms introduced by the derivatizing reagent (δ<sup>13</sup>C TMS = −30.1‰ ± 0.2‰). An analytical precision of &lt; ~1‰ was determined via replicate injections of hydrocarbon standard mixtures. Additionally, a δ<sup>13</sup>C value of −33.3‰ ± 0.1‰ for the archaeol<sub>std</sub> was determined via triplicate measurements using a ThermoFinnigan Flash 2000 Elemental Analyzer connected to a Delta V Plus isotope ratio mass spectrometer (Thermo Fisher Scientific GmbH, Bremen, Germany).</p>
<p>All data processing was conducted using the Isodat 3.0 software (Thermo Fisher Scientific GmbH, Bremen, Germany), with isotopic values being expressed in delta notation (δ) in parts per thousand (‰) as δ<sup>13</sup>C relative to the international Vienna Pee Dee Belemnite (VPDB) standard.</p></section><section id="rcm10118-sec-0009"><h3 class="pmc_sec_title">2.3. Reversed‐Phase UHPLC/ESI/MS</h3>
<p>The TLEs from the sediment samples, as well as the archaeol<sub>std</sub>, were analyzed using reversed‐phase UHPLC/ESI/MS, with chromatographic separation performed on a Dionex Ultimate 3000RS UHPLC system using the separation methods of Wörmer et al. [<a href="#rcm10118-bib-0031" class="usa-link" aria-describedby="rcm10118-bib-0031">31</a>] (i.e., short RP) and Zhu et al. [<a href="#rcm10118-bib-0032" class="usa-link" aria-describedby="rcm10118-bib-0032">32</a>] (i.e., long RP).</p>
<p>Analyte separation, according to Wörmer et al. [<a href="#rcm10118-bib-0031" class="usa-link" aria-describedby="rcm10118-bib-0031">31</a>], was performed on an Acquity UPLC BEH C18 column (1.7 μm; 2.1 × 150 mm, Waters Corporation, Eschborn, Germany). The mobile phase consisted of eluent A (methanol:water, v:v, 85:15, 0.04% formic acid, 0.1% NH<sub>4</sub>OH) and eluent B (propan‐2‐ol:methanol, v:v, 50:50, 0.04% formic acid, 0.1% NH<sub>4</sub>OH), with the following gradient: 0% B for 0 min, increasing to 15% at 2 min, then to 85% by 20 min and 100% by 28 min. The flow rate was maintained at 0.4 mL min<sup>−1</sup>, and the column was held at 65°C.</p>
<p>Following Zhu et al. [<a href="#rcm10118-bib-0032" class="usa-link" aria-describedby="rcm10118-bib-0032">32</a>], measurements were performed on an ACE 3 C18 column (3 μm; 2.1 × 150 mm, VWR International GmbH, Darmstadt, Germany) with a mobile phase consisting of eluent A (methanol, 0.04% formic acid, 0.1% NH<sub>4</sub>OH) and eluent B (propan‐2‐ol, v:v, 50:50, 0.04% formic acid, 0.1% NH<sub>4</sub>OH). The gradient started with 0% B for 0 min, increasing to 5% by 6 min, followed by an increase to 50% and 95% by 30 and 60 min, respectively, and a final increase to 100% by 75 min. The flow rate was set to 0.2 mL min<sup>−1</sup>, and the column temperature was maintained at 45°C.</p>
<p>The RP/UHPLC system was coupled to a Bruker maXis plus ultra‐high‐resolution quadrupole time‐of‐flight mass spectrometer (UHR‐qToF, Bruker Daltonics, Bremen, Germany) equipped with an ESI interface operated in positive ion mode with a dry gas (N<sub>2</sub>) flow of 4 L min<sup>−1</sup> and a drying gas temperature of 200°C. Spectra were acquired from 50 <em>m/z</em> to 2000 <em>m/z</em> with focus mode off, resulting in a mass resolution of 27 000 at <em>m/z</em> 1222. Every analysis was mass‐calibrated using loop injections of a calibration standard mixture (Agilent TuneMix) and further corrected using a lock mass calibrant (992.0098 Da), achieving a general mass accuracy lower than 2 ppm for all targeted compounds. All lipids mentioned were identified based on accurate mass, retention time, and characteristic fragmentation patterns in MS/MS. Extracted ion chromatograms with an isolation width of ±5 to ±10 mDa were created for the M0 and M1 peaks of all targeted lipids and integrated using DataAnalysis 5.0 (Bruker Daltonics, Bremen, Germany).</p></section><section id="rcm10118-sec-0010"><h3 class="pmc_sec_title">2.4. δ<sup>13</sup>C Calculation Based on M0/M1</h3>
<p>Our approach for CSIA relies on the ratio of the M1 isotopologue and M0 peaks, calculated from the manually integrated area of both peaks in extracted ion chromatograms exported from the DataAnalysis software. The mass resolution of the QTOF‐MS is not high enough to separate isobaric isotopes that produce the same nominal M1 mass (e.g., separate <sup>13</sup>C from <sup>2</sup>H or <sup>15</sup>N), and as a result, the M1 peak area contains isotopic contributions of several elements. To calculate the δ<sup>13</sup>C values from the M1/M0 ratio, we first corrected the observed M1/M0 ratio <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-1" display="inline" overflow="linebreak"><mrow><mrow><mfenced close=")" open="("><mrow><mfrac><mrow><mi>M</mi><mn>1</mn></mrow><mrow><mi>M</mi><mn>0</mn></mrow></mfrac><mspace width="0.25em"></mspace><mi mathvariant="italic">obs</mi></mrow></mfenced></mrow></mrow></math></span> by subtracting the isotopic contributions from deuterium (<sup>2</sup>H), oxygen (<sup>17</sup>O), nitrogen (<sup>15</sup>N), and other elements, such as sulfur (<sup>33</sup>S), if present, in the respective molecule, and then dividing the result by the number of carbon atoms (nC), as shown in Equation (<a href="#rcm10118-disp-0001" class="usa-link">1</a>). The contribution of each element is determined by the number of atoms (nH, nO, and nN) present in the target compound, including atoms introduced during adduct formation, and their natural fractional isotopic abundances (<sup>17</sup>O = 0.00038, <sup>15</sup>N = 0.00364) [<a href="#rcm10118-bib-0033" class="usa-link" aria-describedby="rcm10118-bib-0033">33</a>]. To ensure a more accurate <sup>13</sup>C determination, we also accounted for the isotopic contribution of <sup>2</sup>H by adjusting the M1/M0 based on an estimated δ<sup>2</sup>H value relative to the Vienna Standard Mean Ocean Water (VSMOW = 0.00015576) for target compounds.</p>
<table class="disp-formula p" id="rcm10118-disp-0001"><tr>
<td class="formula"><math id="jats-math-2" display="block" overflow="linebreak"><mrow><mrow><mfrac><mrow><mi>M</mi><mn>1</mn></mrow><mrow><mi>M</mi><mn>0</mn></mrow></mfrac><mtext mathvariant="italic">corr</mtext><mo linebreak="goodbreak">=</mo><mfrac><mrow><mfrac><mrow><mi>M</mi><mn>1</mn></mrow><mrow><mi>M</mi><mn>0</mn></mrow></mfrac><mi mathvariant="italic">obs</mi><mo>−</mo><mfenced close=")" open="("><mrow><mfenced close=")" open="("><mrow><mfenced close=")" open="("><mrow><mfrac><mrow><msup><mi>δ</mi><mn>2</mn></msup><mi>H</mi></mrow><mn>1000</mn></mfrac><mo linebreak="goodbreak">+</mo><mn>1</mn></mrow></mfenced><mo>*</mo><mn>0.00015576</mn></mrow></mfenced><mo>*</mo><mi mathvariant="italic">nH</mi></mrow></mfenced><mo>−</mo><mfenced close=")" open="("><mrow><mi mathvariant="italic">nO</mi><mo>*</mo><mn>0.00038</mn></mrow></mfenced><mo>−</mo><mfenced close=")" open="("><mrow><mi mathvariant="italic">nN</mi><mo>*</mo><mn>0.00364</mn></mrow></mfenced></mrow><mi mathvariant="italic">nC</mi></mfrac></mrow></mrow></math></td>
<td class="label">(1)</td>
</tr></table>
<p>The δ<sup>2</sup>H values of microbial lipids can vary significantly depending on environmental factors and the central metabolic pathway of their producer [<a href="#rcm10118-bib-0034" class="usa-link" aria-describedby="rcm10118-bib-0034">34</a>, <a href="#rcm10118-bib-0035" class="usa-link" aria-describedby="rcm10118-bib-0035">35</a>, <a href="#rcm10118-bib-0036" class="usa-link" aria-describedby="rcm10118-bib-0036">36</a>, <a href="#rcm10118-bib-0037" class="usa-link" aria-describedby="rcm10118-bib-0037">37</a>, <a href="#rcm10118-bib-0038" class="usa-link" aria-describedby="rcm10118-bib-0038">38</a>, <a href="#rcm10118-bib-0039" class="usa-link" aria-describedby="rcm10118-bib-0039">39</a>, <a href="#rcm10118-bib-0040" class="usa-link" aria-describedby="rcm10118-bib-0040">40</a>, <a href="#rcm10118-bib-0041" class="usa-link" aria-describedby="rcm10118-bib-0041">41</a>]. Thus, we conducted a comprehensive assessment of δ<sup>2</sup>H variability in environmental lipids to select approximate estimated δ<sup>2</sup>H values that minimize the uncertainty introduced via δ<sup>2</sup>H variability. We compiled δ<sup>2</sup>H values for a wide range of lipid classes, including archaeol, biphytanes, even and odd short‐chained fatty acids, methyl‐branched fatty acids, sterols, hopanols, and phytol and its derivatives, reported from diverse environmental matrices such as marine sediments, water columns, hot springs, and lakes [<a href="#rcm10118-bib-0034" class="usa-link" aria-describedby="rcm10118-bib-0034">34</a>, <a href="#rcm10118-bib-0035" class="usa-link" aria-describedby="rcm10118-bib-0035">35</a>, <a href="#rcm10118-bib-0039" class="usa-link" aria-describedby="rcm10118-bib-0039">39</a>, <a href="#rcm10118-bib-0041" class="usa-link" aria-describedby="rcm10118-bib-0041">41</a>, <a href="#rcm10118-bib-0042" class="usa-link" aria-describedby="rcm10118-bib-0042">42</a>, <a href="#rcm10118-bib-0043" class="usa-link" aria-describedby="rcm10118-bib-0043">43</a>, <a href="#rcm10118-bib-0044" class="usa-link" aria-describedby="rcm10118-bib-0044">44</a>, <a href="#rcm10118-bib-0045" class="usa-link" aria-describedby="rcm10118-bib-0045">45</a>, <a href="#rcm10118-bib-0046" class="usa-link" aria-describedby="rcm10118-bib-0046">46</a>, <a href="#rcm10118-bib-0047" class="usa-link" aria-describedby="rcm10118-bib-0047">47</a>, <a href="#rcm10118-bib-0048" class="usa-link" aria-describedby="rcm10118-bib-0048">48</a>] (Figure <a href="#rcm10118-supitem-0001" class="usa-link">S1</a>). Lipids were categorized into their dominant biological origin (archaeal, bacterial, or eukaryotic) based on chemotaxonomy, and for each group, we determined a δ<sup>2</sup>H range and midpoint value. For archaeal lipids (e.g., archaeol, hydroxyarchaeol, and biphytanes), the δ<sup>2</sup>H values ranged from −349‰ to −118‰, with a midpoint of −234‰ and a corresponding uncertainty of ±116‰. Although δ<sup>2</sup>H data derived from laboratory cultures such as halophilic archaea [<a href="#rcm10118-bib-0036" class="usa-link" aria-describedby="rcm10118-bib-0036">36</a>], ammonia‐oxidizing archaea [<a href="#rcm10118-bib-0049" class="usa-link" aria-describedby="rcm10118-bib-0049">49</a>], <em>Sulfolobus</em> spp. [<a href="#rcm10118-bib-0035" class="usa-link" aria-describedby="rcm10118-bib-0035">35</a>], and the methanogen 
<em>Methanosarcina barkeri</em>
 [<a href="#rcm10118-bib-0050" class="usa-link" aria-describedby="rcm10118-bib-0050">50</a>] were not included in Figure <a href="#rcm10118-supitem-0001" class="usa-link">S1</a>, the observed range of lipid–water δ<sup>2</sup>H fractionation (approximately −300‰ to −150‰) is consistent with values reported from environmental matrices. Accordingly, a midpoint value of −234‰ for all δ<sup>13</sup>C calculations involving archaeal lipids has been utilized in this study. For lipids with a putative bacterial or eukaryotic origin, the respective values can be adapted according to Figure <a href="#rcm10118-supitem-0001" class="usa-link">S1</a>. In that regard, we note that pure culture experiments by Zhang et al. [<a href="#rcm10118-bib-0040" class="usa-link" aria-describedby="rcm10118-bib-0040">40</a>] and Osburn et al. [<a href="#rcm10118-bib-0051" class="usa-link" aria-describedby="rcm10118-bib-0051">51</a>] have shown that heterotrophic bacteria may produce lipids with significantly more <sup>2</sup>H‐enriched signatures (&gt; 50‰). However, we propose that δ<sup>2</sup>H values derived from environmental matrices better reflect the natural range of isotopic variability in complex mixtures. We encourage targeted adjustments regarding the estimated δ<sup>2</sup>H value if compound‐specific data for lipids in a given environment is available.</p>
<p>As demonstrated in Table <a href="#rcm10118-tbl-0001" class="usa-link">1</a>, even a large shift in δ<sup>2</sup>H (i.e., ±116‰) results in a relatively minor shift of approximately ±4‰ in the corresponding δ<sup>13</sup>C value. This uncertainty remains minor compared to the substantial isotopic differences (&gt; ~15‰) observed between lipids associated with methane‐derived carbon and those unrelated to methane turnover. Thus, it does not compromise our approach, which concentrates on identifying extremely <sup>13</sup>C‐depleted compounds, such as those related to microbial methane oxidation [<a href="#rcm10118-bib-0005" class="usa-link" aria-describedby="rcm10118-bib-0005">5</a>, <a href="#rcm10118-bib-0006" class="usa-link" aria-describedby="rcm10118-bib-0006">6</a>, <a href="#rcm10118-bib-0007" class="usa-link" aria-describedby="rcm10118-bib-0007">7</a>, <a href="#rcm10118-bib-0008" class="usa-link" aria-describedby="rcm10118-bib-0008">8</a>, <a href="#rcm10118-bib-0009" class="usa-link" aria-describedby="rcm10118-bib-0009">9</a>, <a href="#rcm10118-bib-0010" class="usa-link" aria-describedby="rcm10118-bib-0010">10</a>, <a href="#rcm10118-bib-0011" class="usa-link" aria-describedby="rcm10118-bib-0011">11</a>], rather than permille precision. These uncertainties must be acknowledged when reporting δ<sup>13</sup>C values derived from isotopologue ratios, as detailed in Section <a href="#rcm10118-sec-0013" class="usa-link">3.2</a> and Table <a href="#rcm10118-tbl-0002" class="usa-link">2</a>.</p>
<section class="tw xbox font-sm" id="rcm10118-tbl-0001"><h4 class="obj_head">TABLE 1.</h4>
<div class="caption p"><p>Three examples showing the influence of different δ<sup>2</sup>H values on δ<sup>13</sup>C determination using Equations (<a href="#rcm10118-disp-0001" class="usa-link">1</a>) and (<a href="#rcm10118-disp-0002" class="usa-link">2</a>), where the M1/M0 ratio for archaeol<sub>std</sub> (C<sub>43</sub>H<sub>88</sub>O<sub>3</sub> + NH<sub>4</sub>
<sup>+</sup>) was adjusted using an assumed δ<sup>2</sup>H value of −118‰, −234‰, and −349‰ (cf. Figure <a href="#rcm10118-supitem-0001" class="usa-link">S1</a>) relative to VSMOW to account for potential variation in hydrogen isotopic composition of the lipid.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col align="left" span="1">
<col align="center" span="1">
<col align="center" span="1">
<col align="center" span="1">
<col align="center" span="1">
<col align="center" span="1">
<thead valign="bottom">
<tr style="border-bottom:solid 1px #000000">
<th colspan="3" style="border-bottom:solid 1px #000000" align="left" valign="bottom" rowspan="1">C<sub>43</sub>H<sub>88</sub>O<sub>3</sub> + NH<sub>4</sub>
<sup>+</sup>
</th>
<th align="center" colspan="2" style="border-bottom:solid 1px #000000" valign="bottom" rowspan="1">Influence on calculated δ<sup>13</sup>C values</th>
<th align="center" rowspan="3" style="border-bottom:solid 1px #000000" valign="bottom" colspan="1">δ<sup>13</sup>C ± δ<sup>2</sup>H‐introduced uncertainty [‰]</th>
</tr>
<tr style="border-bottom:solid 1px #000000">
<th style="border-bottom:solid 1px #000000" align="left" valign="bottom" rowspan="1" colspan="1">M0 area</th>
<th align="center" style="border-bottom:solid 1px #000000" valign="bottom" rowspan="1" colspan="1">M1 area</th>
<th align="center" rowspan="2" style="border-bottom:solid 1px #000000" valign="bottom" colspan="1">M1/M0</th>
<th align="center" style="border-bottom:solid 1px #000000" valign="bottom" rowspan="1" colspan="1">δ<sup>2</sup>H</th>
<th align="center" style="border-bottom:solid 1px #000000" valign="bottom" rowspan="1" colspan="1">δ<sup>13</sup>C</th>
</tr>
<tr style="border-bottom:solid 1px #000000">
<th align="left" valign="bottom" rowspan="1" colspan="1">[arb.u.]</th>
<th align="center" valign="bottom" rowspan="1" colspan="1">[arb.u.]</th>
<th align="center" valign="bottom" rowspan="1" colspan="1">[‰]</th>
<th align="center" valign="bottom" rowspan="1" colspan="1">[‰]</th>
</tr>
</thead>
<tbody valign="top">
<tr>
<td rowspan="3" align="left" valign="top" colspan="1">4 308 658</td>
<td align="center" rowspan="3" valign="top" colspan="1">2 074 659</td>
<td align="center" rowspan="3" valign="top" colspan="1">0.482</td>
<td align="center" valign="top" rowspan="1" colspan="1">−118</td>
<td align="center" valign="top" rowspan="1" colspan="1">−35</td>
<td align="center" rowspan="3" valign="top" colspan="1">−31 ± 4</td>
</tr>
<tr>
<td align="center" valign="top" rowspan="1" colspan="1">−234</td>
<td align="center" valign="top" rowspan="1" colspan="1">−31</td>
</tr>
<tr>
<td align="center" valign="top" rowspan="1" colspan="1">−349</td>
<td align="center" valign="top" rowspan="1" colspan="1">−28</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/rcm10118-tbl-0001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><section class="tw xbox font-sm" id="rcm10118-tbl-0002"><h4 class="obj_head">TABLE 2.</h4>
<div class="caption p"><p>Replicate isotopologue‐based δ<sup>13</sup>C values of diglycosidic archaeol (2G‐AR), monoglycosidic hydroxyarchaeol (1G‐OH‐AR), and triacylglycerol (TAG)‐C48:0 calculated from UHPLC/HRMS data following the analytical method of Wörmer et al. [<a href="#rcm10118-bib-0031" class="usa-link" aria-describedby="rcm10118-bib-0031">31</a>]. Measurements were conducted from the surface sediments of the Cathedral Hill site (core 5000‐9, 0–2 cmbsf). δ<sup>13</sup>C values were calculated assuming three different δ<sup>2</sup>H values based on the estimated δ<sup>2</sup>H variability for archaeal (−234‰ ± 116‰; 2G‐AR, 1G‐OH‐AR) and eukaryotic lipids (250‰ ± 230‰, TAG‐C48:0) on the calculated δ<sup>13</sup>C value (cf. Figure <a href="#rcm10118-supitem-0001" class="usa-link">S1</a>). Standard deviations (SD) of replicate measurements and total propagated uncertainties were calculated using the root‐sum‐square method, combining replicate variability and δ<sup>2</sup>H‐related uncertainty.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col align="left" span="1">
<col align="center" span="1">
<col align="center" span="1">
<col align="center" span="1">
<col align="center" span="1">
<col align="center" span="1">
<col align="center" span="1">
<col align="center" span="1">
<col align="center" span="1">
<col align="center" span="1">
<col align="center" span="1">
<col align="center" span="1">
<thead valign="bottom">
<tr style="border-bottom:solid 1px #000000">
<th rowspan="2" style="border-bottom:solid 1px #000000" align="left" valign="bottom" colspan="1"></th>
<th align="center" rowspan="2" style="border-bottom:solid 1px #000000" valign="bottom" colspan="1">Repl.</th>
<th align="center" rowspan="2" style="border-bottom:solid 1px #000000" valign="bottom" colspan="1">M1/M0</th>
<th align="center" colspan="3" style="border-bottom:solid 1px #000000" valign="bottom" rowspan="1">δ<sup>13</sup>C in ‰</th>
<th align="center" rowspan="2" style="border-bottom:solid 1px #000000" valign="bottom" colspan="1">δ<sup>2</sup>H‐related shift [‰]</th>
<th align="center" rowspan="2" style="border-bottom:solid 1px #000000" valign="bottom" colspan="1">SD [‰] (Repl.)</th>
<th align="center" rowspan="2" style="border-bottom:solid 1px #000000" valign="bottom" colspan="1">SD [‰] (propagated)</th>
<th align="center" rowspan="2" style="border-bottom:solid 1px #000000" valign="bottom" colspan="1">Avg. δ<sup>13</sup>C [‰] δ<sup>2</sup>H = Mid.</th>
<th align="center" rowspan="2" style="border-bottom:solid 1px #000000" valign="bottom" colspan="1">Reported value</th>
<th align="center" style="border-bottom:solid 1px #000000" valign="bottom" rowspan="1" colspan="1"></th>
</tr>
<tr style="border-bottom:solid 1px #000000">
<th align="center" valign="bottom" rowspan="1" colspan="1">δ<sup>2</sup>H = Max.<a href="#rcm10118-note-0002" class="usa-link">
<sup>a</sup>
</a>
</th>
<th align="center" valign="bottom" rowspan="1" colspan="1">δ<sup>2</sup>H = Mid.<a href="#rcm10118-note-0002" class="usa-link">
<sup>a</sup>
</a>
</th>
<th align="center" valign="bottom" rowspan="1" colspan="1">δ<sup>2</sup>H = Min.<a href="#rcm10118-note-0002" class="usa-link">
<sup>a</sup>
</a>
</th>
<th align="center" valign="bottom" rowspan="1" colspan="1"></th>
</tr>
</thead>
<tbody valign="top">
<tr>
<td rowspan="4" align="left" valign="top" colspan="1">2G‐AR</td>
<td align="center" valign="top" rowspan="1" colspan="1">#1</td>
<td align="center" valign="top" rowspan="1" colspan="1">57.17</td>
<td align="center" valign="top" rowspan="1" colspan="1">−109</td>
<td align="center" valign="top" rowspan="1" colspan="1">
<strong>−106</strong>
</td>
<td align="center" valign="top" rowspan="1" colspan="1">−103</td>
<td align="center" rowspan="4" valign="top" colspan="1">±3</td>
<td align="center" rowspan="4" valign="top" colspan="1">±5</td>
<td align="center" rowspan="4" valign="top" colspan="1">±6</td>
<td align="center" rowspan="4" valign="top" colspan="1">−105</td>
<td align="center" rowspan="4" valign="top" colspan="1">−105‰ ± 6‰</td>
<td align="center" valign="top" rowspan="1" colspan="1"></td>
</tr>
<tr>
<td align="center" valign="top" rowspan="1" colspan="1">#2</td>
<td align="center" valign="top" rowspan="1" colspan="1">56.78</td>
<td align="center" valign="top" rowspan="1" colspan="1">−116</td>
<td align="center" valign="top" rowspan="1" colspan="1">
<strong>−112</strong>
</td>
<td align="center" valign="top" rowspan="1" colspan="1">−109</td>
<td align="center" valign="top" rowspan="1" colspan="1"></td>
</tr>
<tr>
<td align="center" valign="top" rowspan="1" colspan="1">#3</td>
<td align="center" valign="top" rowspan="1" colspan="1">57.59</td>
<td align="center" valign="top" rowspan="1" colspan="1">−102</td>
<td align="center" valign="top" rowspan="1" colspan="1">
<strong>−99</strong>
</td>
<td align="center" valign="top" rowspan="1" colspan="1">−96</td>
<td align="center" valign="top" rowspan="1" colspan="1"></td>
</tr>
<tr>
<td align="center" valign="top" rowspan="1" colspan="1">#4</td>
<td align="center" valign="top" rowspan="1" colspan="1">57.41</td>
<td align="center" valign="top" rowspan="1" colspan="1">−105</td>
<td align="center" valign="top" rowspan="1" colspan="1">
<strong>−102</strong>
</td>
<td align="center" valign="top" rowspan="1" colspan="1">−99</td>
<td align="center" valign="top" rowspan="1" colspan="1"></td>
</tr>
<tr>
<td rowspan="4" align="left" valign="top" colspan="1">1G‐OH‐AR</td>
<td align="center" valign="top" rowspan="1" colspan="1">#1</td>
<td align="center" valign="top" rowspan="1" colspan="1">52.19</td>
<td align="center" valign="top" rowspan="1" colspan="1">−86</td>
<td align="center" valign="top" rowspan="1" colspan="1">
<strong>−82</strong>
</td>
<td align="center" valign="top" rowspan="1" colspan="1">−79</td>
<td align="center" rowspan="4" valign="top" colspan="1">±3</td>
<td align="center" rowspan="4" valign="top" colspan="1">±5</td>
<td align="center" rowspan="4" valign="top" colspan="1">±6</td>
<td align="center" rowspan="4" valign="top" colspan="1">−86</td>
<td align="center" rowspan="4" valign="top" colspan="1">−86‰ ± 6‰</td>
<td align="center" valign="top" rowspan="1" colspan="1"></td>
</tr>
<tr>
<td align="center" valign="top" rowspan="1" colspan="1">#2</td>
<td align="center" valign="top" rowspan="1" colspan="1">51.68</td>
<td align="center" valign="top" rowspan="1" colspan="1">−95</td>
<td align="center" valign="top" rowspan="1" colspan="1">
<strong>−92</strong>
</td>
<td align="center" valign="top" rowspan="1" colspan="1">−88</td>
<td align="center" valign="top" rowspan="1" colspan="1"></td>
</tr>
<tr>
<td align="center" valign="top" rowspan="1" colspan="1">#3</td>
<td align="center" valign="top" rowspan="1" colspan="1">51.83</td>
<td align="center" valign="top" rowspan="1" colspan="1">−92</td>
<td align="center" valign="top" rowspan="1" colspan="1">
<strong>−89</strong>
</td>
<td align="center" valign="top" rowspan="1" colspan="1">−86</td>
<td align="center" valign="top" rowspan="1" colspan="1"></td>
</tr>
<tr>
<td align="center" valign="top" rowspan="1" colspan="1">#4</td>
<td align="center" valign="top" rowspan="1" colspan="1">52.29</td>
<td align="center" valign="top" rowspan="1" colspan="1">−84</td>
<td align="center" valign="top" rowspan="1" colspan="1">
<strong>−81</strong>
</td>
<td align="center" valign="top" rowspan="1" colspan="1">−77</td>
<td align="center" valign="top" rowspan="1" colspan="1"></td>
</tr>
<tr>
<td rowspan="4" align="left" valign="top" colspan="1">TAG‐C<sub>48:0</sub>
</td>
<td align="center" valign="top" rowspan="1" colspan="1">#1</td>
<td align="center" valign="top" rowspan="1" colspan="1">57.05</td>
<td align="center" valign="top" rowspan="1" colspan="1">−37</td>
<td align="center" valign="top" rowspan="1" colspan="1">
<strong>−31</strong>
</td>
<td align="center" valign="top" rowspan="1" colspan="1">−24</td>
<td align="center" rowspan="4" valign="top" colspan="1">±6</td>
<td align="center" rowspan="4" valign="top" colspan="1">±3</td>
<td align="center" rowspan="4" valign="top" colspan="1">±7</td>
<td align="center" rowspan="4" valign="top" colspan="1">−26</td>
<td align="center" rowspan="4" valign="top" colspan="1">−28‰ ± 7‰</td>
<td align="center" valign="top" rowspan="1" colspan="1"></td>
</tr>
<tr>
<td align="center" valign="top" rowspan="1" colspan="1">#2</td>
<td align="center" valign="top" rowspan="1" colspan="1">57.29</td>
<td align="center" valign="top" rowspan="1" colspan="1">−33</td>
<td align="center" valign="top" rowspan="1" colspan="1">
<strong>−27</strong>
</td>
<td align="center" valign="top" rowspan="1" colspan="1">−20</td>
<td align="center" valign="top" rowspan="1" colspan="1"></td>
</tr>
<tr>
<td align="center" valign="top" rowspan="1" colspan="1">#3</td>
<td align="center" valign="top" rowspan="1" colspan="1">57.33</td>
<td align="center" valign="top" rowspan="1" colspan="1">−32</td>
<td align="center" valign="top" rowspan="1" colspan="1">
<strong>−26</strong>
</td>
<td align="center" valign="top" rowspan="1" colspan="1">−19</td>
<td align="center" valign="top" rowspan="1" colspan="1"></td>
</tr>
<tr>
<td align="center" valign="top" rowspan="1" colspan="1">#4</td>
<td align="center" valign="top" rowspan="1" colspan="1">57.52</td>
<td align="center" valign="top" rowspan="1" colspan="1">−29</td>
<td align="center" valign="top" rowspan="1" colspan="1">
<strong>−23</strong>
</td>
<td align="center" valign="top" rowspan="1" colspan="1">−16</td>
<td align="center" valign="top" rowspan="1" colspan="1"></td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/rcm10118-tbl-0002/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="rcm10118-note-0002">
<sup>
<sup>a</sup>
</sup><p class="display-inline">2G‐AR and 1G‐OH‐AR (both NH<sub>4</sub>
<sup>+</sup>): δ<sup>2</sup>H: minimum: −118‰; midpoint: −234‰; maximum: −349‰ (archaeal). TAG‐C<sub>48:0</sub> (NH<sub>4</sub>
<sup>+</sup>): δ<sup>2</sup>H: Minimum: −20‰; midpoint: −250‰; maximum: −481‰ (eukaryotic) (cf. Figure <a href="#rcm10118-fig-0001" class="usa-link">1</a>).</p>
</div></div></section><p>Isotopic variations of nitrogen, whether introduced via adduct formation or naturally occurring in a target compound, are exceptionally small, and even exaggerated δ<sup>15</sup>N values (relative to atmospheric N<sub>2</sub>) have a negligible effect. The range of δ<sup>15</sup>N values observed in bulk biomass [<a href="#rcm10118-bib-0052" class="usa-link" aria-describedby="rcm10118-bib-0052">52</a>, <a href="#rcm10118-bib-0053" class="usa-link" aria-describedby="rcm10118-bib-0053">53</a>], amino acids [<a href="#rcm10118-bib-0052" class="usa-link" aria-describedby="rcm10118-bib-0052">52</a>, <a href="#rcm10118-bib-0054" class="usa-link" aria-describedby="rcm10118-bib-0054">54</a>, <a href="#rcm10118-bib-0055" class="usa-link" aria-describedby="rcm10118-bib-0055">55</a>, <a href="#rcm10118-bib-0056" class="usa-link" aria-describedby="rcm10118-bib-0056">56</a>], and nitrogen‐containing lipids [<a href="#rcm10118-bib-0052" class="usa-link" aria-describedby="rcm10118-bib-0052">52</a>, <a href="#rcm10118-bib-0057" class="usa-link" aria-describedby="rcm10118-bib-0057">57</a>] generally remains between +30‰, representing an extremely enriched endmember, and a minimum of −30‰. NH<sub>4</sub>
<sup>+</sup> adducts, introduced via the addition of industrial NH<sub>4</sub>OH during UHPLC analysis, are expected to exhibit δ<sup>15</sup>N values close to 0‰ [<a href="#rcm10118-bib-0058" class="usa-link" aria-describedby="rcm10118-bib-0058">58</a>], reflecting the isotopic composition of air‐derived N<sub>2</sub> from Haber–Bosch synthesis. Even when simulating a shift from +30‰ to −30‰ in nitrogen‐containing lipids, the effect on the corresponding δ<sup>13</sup>C value is limited to the first decimal (Table <a href="#rcm10118-supitem-0001" class="usa-link">S1</a>). With a natural abundance ten times lower than that of <sup>15</sup>N, <sup>17</sup>O has an even smaller effect on δ<sup>13</sup>C values [<a href="#rcm10118-bib-0033" class="usa-link" aria-describedby="rcm10118-bib-0033">33</a>].</p>
<p>Considering these factors, the δ<sup>13</sup>C value expressed in ‰ can be calculated according to Equation (<a href="#rcm10118-disp-0002" class="usa-link">2</a>) with 0.0111802 corresponding to the<sup>13</sup>C/<sup>12</sup>C ratio of the VPDB standard.</p>
<table class="disp-formula p" id="rcm10118-disp-0002"><tr>
<td class="formula"><math id="jats-math-3" display="block" overflow="linebreak"><mrow><mrow><mi mathvariant="normal">δ</mi><mmultiscripts><mi>C</mi><mprescripts></mprescripts><mn>0</mn><mn>13</mn></mmultiscripts><mo linebreak="goodbreak">=</mo><mfenced close=")" open="("><mrow><mfrac><mrow><mfrac><mrow><mi>M</mi><mn>1</mn></mrow><mrow><mi>M</mi><mn>0</mn></mrow></mfrac><mtext mathvariant="italic">corr</mtext></mrow><mn>0.0111802</mn></mfrac><mo linebreak="goodbreak">−</mo><mn>1</mn></mrow></mfenced><mo>*</mo><mn>1000</mn></mrow></mrow></math></td>
<td class="label">(2)</td>
</tr></table></section></section><section id="rcm10118-sec-0011"><h2 class="pmc_sec_title">3. Results and Discussion</h2>
<section id="rcm10118-sec-0012"><h3 class="pmc_sec_title">3.1. Criteria for Reliable δ<sup>13</sup>C Determination in UHPLC/HRMS</h3>
<p>To assess the signal stability of the M1/M0 ratio in UHPLC/HRMS, we analyzed a total of 178 LC/HRMS measurements following Wörmer et al. [<a href="#rcm10118-bib-0031" class="usa-link" aria-describedby="rcm10118-bib-0031">31</a>] and 105 measurements according to Zhu et al. [<a href="#rcm10118-bib-0032" class="usa-link" aria-describedby="rcm10118-bib-0032">32</a>], with on‐column archaeol<sub>std</sub> injections ranging from 0.1 to 30 ng. Archaeol forms multiple adducts; thus, we separately calculated the δ<sup>13</sup>C value for the protonated form [M + H]<sup>+</sup> and its ammoniated form [M + NH₄]<sup>+</sup>.</p>
<p>Figure <a href="#rcm10118-fig-0001" class="usa-link">1</a> illustrates the relationship between the M0 peak area expressed in arbitrary units (arb.u.) and the calculated δ<sup>13</sup>C values for the archaeol<sub>std</sub> based on Equations (<a href="#rcm10118-disp-0001" class="usa-link">1</a>) and (<a href="#rcm10118-disp-0002" class="usa-link">2</a>), with the top panel showing data acquired from UHPLC/HRMS measurements following Wörmer et al. [<a href="#rcm10118-bib-0031" class="usa-link" aria-describedby="rcm10118-bib-0031">31</a>]. A key observation is the rapid decline in δ<sup>13</sup>C values, reaching as low as −260‰ when the M0 peak area falls below ~2 × 10<sup>6</sup> arb.u., indicated by the magenta line. This clearly shows that the M1/M0 ratio becomes unreliable below this threshold. Based on the injection of archaeol<sub>std</sub>, on‐column amounts of 10 ng consistently yield peak areas necessary for robust δ<sup>13</sup>C calculation (Figure <a href="#rcm10118-supitem-0001" class="usa-link">S2</a>). It is worth noting that signal intensity generally depends on instrument performance, ionization efficiency, adduct formation, and the response factors of different lipid species. Therefore, peak areas rather than absolute quantities of the analytes provide a more robust and consistent parameter for defining thresholds.</p>
<figure class="fig xbox font-sm" id="rcm10118-fig-0001"><h4 class="obj_head">FIGURE 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c915/12322946/bff2ceba3ef4/RCM-39-e10118-g005.jpg" loading="lazy" id="jats-graphic-1" height="549" width="709" alt="FIGURE 1"></p>
<div class="p text-right font-secondary"><a href="figure/rcm10118-fig-0001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Relationship between the monoisotopic peak area (M0) in arbitrary units (arb.u.) and the calculated δ<sup>13</sup>C values based on Equations (<a href="#rcm10118-disp-0001" class="usa-link">1</a>) and (<a href="#rcm10118-disp-0002" class="usa-link">2</a>) for the archaeol standard shown. Results from the two most abundant adducts are shown, that is, archaeol + H<sup>+</sup> (left panels) and archaeol + NH₄<sup>+</sup> (right panels). The measurements were conducted in reversed phase‐UHPLC/ESI/qToF/MS following Wörmer et al. [<a href="#rcm10118-bib-0031" class="usa-link" aria-describedby="rcm10118-bib-0031">31</a>] (top panels) and Zhu et al. [<a href="#rcm10118-bib-0032" class="usa-link" aria-describedby="rcm10118-bib-0032">32</a>] (bottom panels), with injection amounts ranging from 0.1 to 30 ng on‐column. The area shaded in blue indicates occurrence of oversaturated M0 peaks.</p></figcaption></figure><p>Once the threshold is surpassed, the δ<sup>13</sup>C values become stable and align closely with the expected δ<sup>13</sup>C value of −33.3‰ determined via EA/irMS. The high signal intensity range, on the other hand, is characterized by abnormally high δ<sup>13</sup>C values in the upper range of the peak area. These outliers result from detector saturation, which causes an overrepresentation of the M1 isotopologue relative to M0, leading to artificially heavy apparent δ<sup>13</sup>C values.</p>
<p>A similar observation can be made when examining δ<sup>13</sup>C values obtained from UHPLC/HRMS measurements following Zhu et al. [<a href="#rcm10118-bib-0032" class="usa-link" aria-describedby="rcm10118-bib-0032">32</a>], as shown in the bottom panel of Figure <a href="#rcm10118-fig-0001" class="usa-link">1</a>. However, this method involves a longer chromatographic separation with a total analysis time of 90 min and lower scan rates (1 Hz) for detected compounds, resulting in generally larger peak areas. Consequently, the M0 peak area threshold for UHPLC/HRMS is higher. Based on Figure <a href="#rcm10118-fig-0001" class="usa-link">1</a>, we define an approximate M0 peak area threshold of 5 × 10<sup>6</sup> arb.u. Once the M0 peak area threshold is surpassed, the δ<sup>13</sup>C values consistently remain above −45‰ and align with the expected δ<sup>13</sup>C value of −33.3‰ as well, and no data point falls within the range that can be considered an isotopic signature characteristic for lipids related to methane oxidation.</p>
<p>The relationship between δ<sup>13</sup>C stability and the M0 peak area for [M + NH₄]<sup>+</sup> and [M + H]<sup>+</sup> is comparable (Figure <a href="#rcm10118-fig-0001" class="usa-link">1</a>). However, a small but significant offset can be observed as the [M + H]<sup>+</sup> adduct systematically yields slightly lower δ<sup>13</sup>C values than the expected value of −33.3‰. It is conceivable that this phenomenon results from isotopic fractionation during ionization in the ESI source, but a mechanistic explanation lies beyond the scope of this study. The isotopic composition of the adduct itself can be excluded as a cause, as its impact, even when considering extreme δ<sup>15</sup>N values, has minimal influence on the δ<sup>13</sup>C (Table <a href="#rcm10118-supitem-0001" class="usa-link">S1</a>).</p>
<p>For reliable δ<sup>13</sup>C determination, the M0 peak area must meet the following criteria (note that the absolute intensity values will differ on other instruments, and these ranges will have to be determined individually): M0 peak area above 2 × 10<sup>6</sup> arb.u. for chromatographic separation following Wörmer et al. [<a href="#rcm10118-bib-0031" class="usa-link" aria-describedby="rcm10118-bib-0031">31</a>] and 5 × 10<sup>6</sup> arb.u. following Zhu et al. [<a href="#rcm10118-bib-0032" class="usa-link" aria-describedby="rcm10118-bib-0032">32</a>] without signs of detector oversaturation. Furthermore, peaks showing interference from coeluting compounds must be excluded when applying the proposed method.</p>
<p>Additionally, we studied the effect of the variability in manual peak integration on the resulting δ<sup>13</sup>C values by comparing five different peak integration approaches, each of which differed in the way the peak boundaries were defined (Figure <a href="#rcm10118-fig-0002" class="usa-link">2</a>). Overall, the results indicate that differences in peak integration only have a small effect on the calculated δ<sup>13</sup>C values of the target compounds, especially when the objective is the identification of extremely <sup>13</sup>C‐depleted compounds.</p>
<figure class="fig xbox font-sm" id="rcm10118-fig-0002"><h4 class="obj_head">FIGURE 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c915/12322946/ea545f818ecb/RCM-39-e10118-g001.jpg" loading="lazy" id="jats-graphic-3" height="361" width="709" alt="FIGURE 2"></p>
<div class="p text-right font-secondary"><a href="figure/rcm10118-fig-0002/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>The impact of variations during manual peak integration on the δ<sup>13</sup>C calculation for an archaeol standard (top panel) and<sup>13</sup>C‐depleted archaeol from an AOM‐influenced sediment sample from the Aceto Balsamico area of Guaymas Basin (Core 4992‐14, 4–6 cmbsf) (bottom panel), both detected as [M + NH₄]<sup>+</sup> adducts. Five integration approaches are shown, ranging from broader peak limits (left) to more narrowly focused on the central peak (right). Corresponding M1/M0 ratios and δ<sup>13</sup>C values in ‰ (Equations (<a href="#rcm10118-disp-0001" class="usa-link">1</a>) and (<a href="#rcm10118-disp-0002" class="usa-link">2</a>)) are displayed below each peak, with minimal variation in δ<sup>13</sup>C values across all different peak integration approaches.</p></figcaption></figure></section><section id="rcm10118-sec-0013"><h3 class="pmc_sec_title">3.2. δ<sup>13</sup>C Determination via GC/irMS vs. UHPLC/ESI/HRMS</h3>
<p>We compared the δ<sup>13</sup>C values of archaeol derived from traditional GC/irMS and those calculated from the M1/M0 ratios acquired from UHPLC/HRMS in two sediment cores collected from the hot Cathedral Hill (5000‐9) and temperate Aceto Balsamico core (4992‐14) at the Guaymas Basin (Figure <a href="#rcm10118-fig-0003" class="usa-link">3</a>). Both sites are well documented for the presence of anaerobic oxidation of methane (AOM) and anaerobic methane‐oxidizing archaea (ANME) [<a href="#rcm10118-bib-0059" class="usa-link" aria-describedby="rcm10118-bib-0059">59</a>, <a href="#rcm10118-bib-0060" class="usa-link" aria-describedby="rcm10118-bib-0060">60</a>, <a href="#rcm10118-bib-0061" class="usa-link" aria-describedby="rcm10118-bib-0061">61</a>, <a href="#rcm10118-bib-0062" class="usa-link" aria-describedby="rcm10118-bib-0062">62</a>, <a href="#rcm10118-bib-0063" class="usa-link" aria-describedby="rcm10118-bib-0063">63</a>, <a href="#rcm10118-bib-0064" class="usa-link" aria-describedby="rcm10118-bib-0064">64</a>] and we have detected high abundances of archaeol in samples from both cores. Given the established link between AOM and substantial <sup>13</sup>C‐depletion in the corresponding lipid biomarkers [<a href="#rcm10118-bib-0005" class="usa-link" aria-describedby="rcm10118-bib-0005">5</a>, <a href="#rcm10118-bib-0006" class="usa-link" aria-describedby="rcm10118-bib-0006">6</a>, <a href="#rcm10118-bib-0007" class="usa-link" aria-describedby="rcm10118-bib-0007">7</a>, <a href="#rcm10118-bib-0008" class="usa-link" aria-describedby="rcm10118-bib-0008">8</a>, <a href="#rcm10118-bib-0009" class="usa-link" aria-describedby="rcm10118-bib-0009">9</a>, <a href="#rcm10118-bib-0010" class="usa-link" aria-describedby="rcm10118-bib-0010">10</a>, <a href="#rcm10118-bib-0011" class="usa-link" aria-describedby="rcm10118-bib-0011">11</a>], these samples provide a robust signal for comparing δ<sup>13</sup>C determinations across different analytical techniques.</p>
<figure class="fig xbox font-sm" id="rcm10118-fig-0003"><h4 class="obj_head">FIGURE 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c915/12322946/07709308f05a/RCM-39-e10118-g002.jpg" loading="lazy" id="jats-graphic-5" height="711" width="669" alt="FIGURE 3"></p>
<div class="p text-right font-secondary"><a href="figure/rcm10118-fig-0003/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Comparison of δ<sup>13</sup>C values [‰] obtained from GC/irMS (blue diamonds) and reversed‐phase UHPLC/ESI/HRMS following Equations (<a href="#rcm10118-disp-0001" class="usa-link">1</a>) and (<a href="#rcm10118-disp-0002" class="usa-link">2</a>) for archaeol + NH₄<sup>+</sup> and archaeol + H<sup>+</sup> (orange circles) across two sediment cores from the Guaymas Basin. The upper panels show data from the Cathedral Hill core (5000‐9), whereas the lower panels represent samples from the Aceto Balsamico core (4992‐14). To generate δ<sup>13</sup>C values from the UHPLC/HRMS data, we calculated the M1/M0 ratio using the peak area of the respective peaks and applied Equations (<a href="#rcm10118-disp-0001" class="usa-link">1</a>) and (<a href="#rcm10118-disp-0002" class="usa-link">2</a>) using a fixed δ<sup>2</sup>H value of −234‰ (cf. Figure <a href="#rcm10118-supitem-0001" class="usa-link">S1</a>). Error bars reflect the propagated error, combining δ<sup>2</sup>H‐introduced uncertainty (±116‰) and the analytical standard deviation across multiple independent UHPLC/HRMS measurements, with <em>n</em> = X indicating the number of runs, fulfilling the following criteria: Only peaks with an integrated M0 area above 2 × 10<sup>6</sup> arb.u. and 5 × 10<sup>6</sup> arb.u. were included, depending on the applied LC separation methods after Wörmer et al. [<a href="#rcm10118-bib-0031" class="usa-link" aria-describedby="rcm10118-bib-0031">31</a>] and Zhu et al. [<a href="#rcm10118-bib-0032" class="usa-link" aria-describedby="rcm10118-bib-0032">32</a>], respectively (cf. Figure <a href="#rcm10118-fig-0001" class="usa-link">1</a>). Peaks showing signs of coelution, or oversaturation were excluded. A paired, non‐parametric Wilcoxon signed‐rank test indicated a statistically significant difference between δ<sup>13</sup>C values (average of replicates) derived from both adducts (<em>n</em> = 20, V = 0, <em>p</em> = 1.91 × 10<sup>−6</sup>) with a directional trend towards more <sup>13</sup>C‐depleted values found in the H<sup>+</sup> adduct of archaeol.</p></figcaption></figure><p>The δ<sup>13</sup>C values for the two most abundant adducts ([M + NH₄]<sup>+</sup> and [M + H]<sup>+</sup>) were calculated independently. GC/irMS and HRMS‐derived δ<sup>13</sup>C profiles exhibit the same depth‐related trends in both sediment cores, particularly in core 5000‐9 (Figure <a href="#rcm10118-fig-0003" class="usa-link">3</a> and Figure <a href="#rcm10118-supitem-0001" class="usa-link">S3</a>). Here, both isotope determination techniques reveal a pronounced downcore shift towards a <sup>13</sup>C enrichment, highlighting that our isotopologue‐based approach captures isotopic variations. The pronounced <sup>13</sup>C‐depletion of archaeol (&lt; 70‰) in the surface sediment in both cores is consistent with the production of these compounds by ANME, and high abundances of ANME have indeed been reported in adjacent sediment cores at Cathedral Hill (5000‐11; 5000‐20) and at Aceto Balsamico (4992‐22; 4992‐23) in the form of Amplicon Sequence Variants (ASVs) [<a href="#rcm10118-bib-0062" class="usa-link" aria-describedby="rcm10118-bib-0062">62</a>]. Whereas δ<sup>13</sup>C‐methane has not been analyzed in those cores, δ<sup>13</sup>C‐methane values found in Guaymas Basin sediment range from −30‰ to 80‰, with more enriched values being found in strongly hydrothermally influenced settings [<a href="#rcm10118-bib-0065" class="usa-link" aria-describedby="rcm10118-bib-0065">65</a>, <a href="#rcm10118-bib-0066" class="usa-link" aria-describedby="rcm10118-bib-0066">66</a>]. This is roughly in line with the observed offset between methane and AOM‐related lipids [<a href="#rcm10118-bib-0012" class="usa-link" aria-describedby="rcm10118-bib-0012">12</a>, <a href="#rcm10118-bib-0067" class="usa-link" aria-describedby="rcm10118-bib-0067">67</a>]. At Cathedral Hill, the downcore increase in δ<sup>13</sup>C values of archaeol up to −35‰ points towards a decreased influence of ANME and a more dominant presence of other archaea, assimilating less <sup>13</sup>C‐depleted carbon sources. For instance, Ramírez et al. [<a href="#rcm10118-bib-0062" class="usa-link" aria-describedby="rcm10118-bib-0062">62</a>] reported a heightened presence of Bathyarchaeotal ASVs in the deeper layers of the Cathedral Hill sediment, which may contribute to a more <sup>13</sup>C‐enriched archaeol pool. Standard deviations in the range of ±1‰ and ±9‰, based on two to four independent UHPLC/HRMS runs, indicate good reproducibility of the δ<sup>13</sup>C values.</p>
<p>Nonetheless, a consistent offset can be observed with the UHPLC/HRMS‐based determination exhibiting slightly lower isotopic values compared to the conventional GC/irMS measurements (Figure <a href="#rcm10118-fig-0003" class="usa-link">3</a>). This offset can be observed for both adducts, although the [M + H]<sup>+</sup> adduct consistently shows a more pronounced deviation of on average 11‰ towards more depleted δ<sup>13</sup>C values, in analogy to the trend observed in Figure <a href="#rcm10118-fig-0001" class="usa-link">1</a>. Peak intensity differences can be excluded as the cause, as both adducts were roughly equally abundant and exceeded the required M0 peak area threshold. These adduct‐specific effects should be further investigated in future studies in order to establish a comprehensive mechanistic understanding and possibly a foundation for correcting these effects. Although these effects do not perturb the differentiation between strongly<sup>13</sup>C‐depleted lipids characteristic of methane oxidation from those derived from other metabolic pathways, they must be acknowledged.</p>
<p>We further reinforced these findings by incorporating results from an additional sample set. Collectively, the dataset captures a wide range of δ<sup>13</sup>C values for archaeol in four additional sediment cores across hydrothermal vent and methane seep sites at the Guaymas Basin [<a href="#rcm10118-bib-0068" class="usa-link" aria-describedby="rcm10118-bib-0068">68</a>], spanning δ<sup>13</sup>C values from −130‰ to −34‰. As outlined in Song et al. [<a href="#rcm10118-bib-0065" class="usa-link" aria-describedby="rcm10118-bib-0065">65</a>] and Teske et al. [<a href="#rcm10118-bib-0066" class="usa-link" aria-describedby="rcm10118-bib-0066">66</a>], all sites are influenced by AOM to varying degrees. The most depleted δ<sup>13</sup>C values (&lt; −110‰) were observed at the cold seep site “Octopus Mound” (Core 4867‐3) (Figure <a href="#rcm10118-fig-0004" class="usa-link">4</a>), consistent with admixtures of biogenic methane (−55‰ to −75‰) at this location [<a href="#rcm10118-bib-0065" class="usa-link" aria-describedby="rcm10118-bib-0065">65</a>, <a href="#rcm10118-bib-0066" class="usa-link" aria-describedby="rcm10118-bib-0066">66</a>]. δ<sup>13</sup>C values acquired from the traditional GC/irMS and UHPLC/HRMS, with both adducts analyzed separately, are strongly correlated across a wide range of δ<sup>13</sup>C values (Figure <a href="#rcm10118-fig-0004" class="usa-link">4</a>). Consistent with the observation in the downcore profiles (Figure <a href="#rcm10118-fig-0003" class="usa-link">3</a>), the negative isotopic offset is more pronounced for the protonated molecular ions ([M + H]<sup>+</sup>) compared to their ammoniated counterpart ([M + NH₄]<sup>+</sup>).</p>
<figure class="fig xbox font-sm" id="rcm10118-fig-0004"><h4 class="obj_head">FIGURE 4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c915/12322946/89d7c3725631/RCM-39-e10118-g003.jpg" loading="lazy" id="jats-graphic-7" height="323" width="709" alt="FIGURE 4"></p>
<div class="p text-right font-secondary"><a href="figure/rcm10118-fig-0004/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Correlation between δ<sup>13</sup>C values measured by GC/irMS and UHPLC/HRMS for archaeol as [M + NH₄]<sup>+</sup> (left) and [M + H]<sup>+</sup> (right) adducts. Squares represent UHPLC/HRMS measurements after Zhu et al. [<a href="#rcm10118-bib-0032" class="usa-link" aria-describedby="rcm10118-bib-0032">32</a>], whereas circles indicate data obtained following Wörmer et al. [<a href="#rcm10118-bib-0031" class="usa-link" aria-describedby="rcm10118-bib-0031">31</a>]. The linear regression with the corresponding equations and <em>R</em>
<sup>2</sup> values, as well as the line of equality, is shown. Yellow symbols highlight δ<sup>13</sup>C values from an archaeol standard, green symbols represent measurements conducted before the year 2021, and orange symbols indicate repeating measurements from 2024. Only peaks with an integrated M0 area greater than 2 × 10<sup>6</sup> arb.u. (LC method after Wörmer et al. [<a href="#rcm10118-bib-0031" class="usa-link" aria-describedby="rcm10118-bib-0031">31</a>]) and 5 × 10<sup>6</sup> arb.u. (LC method after Zhu et al. [<a href="#rcm10118-bib-0032" class="usa-link" aria-describedby="rcm10118-bib-0032">32</a>]) were considered (see Figure <a href="#rcm10118-fig-0001" class="usa-link">1</a>). Peaks that showed signs of oversaturation or coelution were excluded. The total number of data points is <em>N</em> = 76 for [M + NH₄]<sup>+</sup> and <em>N</em> = 82 for [M + H]<sup>+</sup>.</p></figcaption></figure><p>Given the strong correlation with well‐established GC/irMS‐derived <strong>δ</strong>
<sup>
<strong>13</strong>
</sup>
<strong>C</strong> values and the matching depth trends illustrated in Figure <a href="#rcm10118-fig-0003" class="usa-link">3</a>, we propose that the M1/M0 ratio derived from UHPLC/HRMS analysis represents a promising tool for the identification of strongly<sup>13</sup>C‐depleted lipids and other metabolites in environmental samples typically associated with the assimilation of methane‐derived carbon, provided that the criteria outlined in 3.1 are met.</p>
<p>In Table <a href="#rcm10118-tbl-0002" class="usa-link">2</a>, we provide a general guideline for reporting the analytical and δ<sup>2</sup>H‐introduced uncertainty of derived δ<sup>13</sup>C values. We performed replicate δ<sup>13</sup>C calculations for three selected lipids from sediment core 5000‐9 at 0–2 cm below seafloor (cmbsf). The δ<sup>13</sup>C values were calculated based on the range and midpoint δ<sup>2</sup>H values emphasized in Section <a href="#rcm10118-sec-0010" class="usa-link">2.4</a> and Figure <a href="#rcm10118-supitem-0001" class="usa-link">S1</a>. The total propagated uncertainty was determined using the root‐sum‐square (RSS) method, combining the standard deviation (SD) across the replicates and the δ<sup>2</sup>H‐based uncertainty. The reported δ<sup>13</sup>C values should be accompanied by a total propagated SD, accounting for both replicate variability and δ<sup>2</sup>H‐related uncertainty.</p></section><section id="rcm10118-sec-0014"><h3 class="pmc_sec_title">3.3. δ<sup>13</sup>C Profiling of AOM and Non‐AOM‐Related Biomarkers in Hydrothermal Sediment</h3>
<p>In the following, we demonstrate the potential of our proposed method as a high‐throughput approach for CSIA of a diverse pool of complex and intact compounds in raw lipid extracts. We selected multiple intact lipids extracted from the surface (0–20 cmbsf) of the Guaymas Basin sediment cores 5000‐9 (Cathedral Hill) and 4992‐14 (Aceto Balsamico) that are not amenable to GC‐based CSIA and calculated their δ<sup>13</sup>C values following Equations (<a href="#rcm10118-disp-0001" class="usa-link">1</a>) and (<a href="#rcm10118-disp-0002" class="usa-link">2</a>) (Figure <a href="#rcm10118-fig-0005" class="usa-link">5</a>). For each compound, the δ<sup>2</sup>H value used for correction was selected based on the predominant origin of the lipid (cf. Figure <a href="#rcm10118-supitem-0001" class="usa-link">S1</a>). First, we selected nine compounds where a connection to AOM can be confidently excluded on the basis of their chemotaxonomic classification. These include hydroxypheophytin <em>a</em>, pyropheophytin <em>a</em>, and pyropheophorbide <em>a</em> 27:2 steryl ester, all related to chlorophyll <em>a</em> derived from photosynthesizing organisms [<a href="#rcm10118-bib-0069" class="usa-link" aria-describedby="rcm10118-bib-0069">69</a>, <a href="#rcm10118-bib-0070" class="usa-link" aria-describedby="rcm10118-bib-0070">70</a>, <a href="#rcm10118-bib-0071" class="usa-link" aria-describedby="rcm10118-bib-0071">71</a>, <a href="#rcm10118-bib-0072" class="usa-link" aria-describedby="rcm10118-bib-0072">72</a>]; a ceramide (Cer 18:0, O2/16:0,O) with various potential bacterial and eukaryotic sources, including fungi [<a href="#rcm10118-bib-0073" class="usa-link" aria-describedby="rcm10118-bib-0073">73</a>, <a href="#rcm10118-bib-0074" class="usa-link" aria-describedby="rcm10118-bib-0074">74</a>, <a href="#rcm10118-bib-0075" class="usa-link" aria-describedby="rcm10118-bib-0075">75</a>, <a href="#rcm10118-bib-0076" class="usa-link" aria-describedby="rcm10118-bib-0076">76</a>]; triacylglycerols (TAG)‐C<sub>48:0</sub> and C<sub>46:0</sub> as storage lipids [<a href="#rcm10118-bib-0077" class="usa-link" aria-describedby="rcm10118-bib-0077">77</a>], likely originating from various microbial sources in the water column; and PC‐DAG‐C<sub>34:1</sub> and PC‐DAG‐C<sub>32:1</sub>, commonly found in bacterial and eukaryotic phytoplankton and nitrifying and heterotrophic bacteria [<a href="#rcm10118-bib-0078" class="usa-link" aria-describedby="rcm10118-bib-0078">78</a>, <a href="#rcm10118-bib-0079" class="usa-link" aria-describedby="rcm10118-bib-0079">79</a>, <a href="#rcm10118-bib-0080" class="usa-link" aria-describedby="rcm10118-bib-0080">80</a>]. The electron carrier ubiquinone‐8:8 (UQ‐8:8) is primarily associated with diverse <em>Proteobacteria</em> [<a href="#rcm10118-bib-0081" class="usa-link" aria-describedby="rcm10118-bib-0081">81</a>, <a href="#rcm10118-bib-0082" class="usa-link" aria-describedby="rcm10118-bib-0082">82</a>, <a href="#rcm10118-bib-0083" class="usa-link" aria-describedby="rcm10118-bib-0083">83</a>], including some aerobic methanotrophic bacteria [<a href="#rcm10118-bib-0084" class="usa-link" aria-describedby="rcm10118-bib-0084">84</a>]. However, any contribution of UQ‐8:8 from aerobic methanotrophs is negligible due to the rapid oxygen depletion in the sediment [<a href="#rcm10118-bib-0062" class="usa-link" aria-describedby="rcm10118-bib-0062">62</a>, <a href="#rcm10118-bib-0085" class="usa-link" aria-describedby="rcm10118-bib-0085">85</a>]. Although our approach is not suited for source assignment that relies on smaller isotopic differences (~ &lt; 15‰), the observed δ<sup>13</sup>C values for these lipids, ranging from −11‰ to −44‰ (Figure <a href="#rcm10118-fig-0005" class="usa-link">5</a>), are broadly consistent with their chemotaxonomic classification as predominantly photosynthetic primary producers and/or secondary consumers utilizing photosynthetically derived organic matter [<a href="#rcm10118-bib-0086" class="usa-link" aria-describedby="rcm10118-bib-0086">86</a>, <a href="#rcm10118-bib-0087" class="usa-link" aria-describedby="rcm10118-bib-0087">87</a>, <a href="#rcm10118-bib-0088" class="usa-link" aria-describedby="rcm10118-bib-0088">88</a>, <a href="#rcm10118-bib-0089" class="usa-link" aria-describedby="rcm10118-bib-0089">89</a>].</p>
<figure class="fig xbox font-sm" id="rcm10118-fig-0005"><h4 class="obj_head">FIGURE 5.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c915/12322946/104ce005f02e/RCM-39-e10118-g004.jpg" loading="lazy" id="jats-graphic-9" height="497" width="709" alt="FIGURE 5"></p>
<div class="p text-right font-secondary"><a href="figure/rcm10118-fig-0005/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Horizontal strip plots of δ<sup>13</sup>C values in ‰, determined from M1/M0 ratios and following Equations (<a href="#rcm10118-disp-0001" class="usa-link">1</a>) and (<a href="#rcm10118-disp-0002" class="usa-link">2</a>), for various lipids, detected in sediment core 5000‐9 (Cathedral Hill, Guaymas Basin; circles) and 4992‐14 (Aceto Balsamico, Guaymas Basin; squares), including diglycosidic archaeol (2G‐Archaeol), phosphatidylglycerol archaeol (PG‐Archaeol), phosphatidylinositol archaeol (PI‐Archaeol), phosphatidylglycerol hydroxyarchaeol (PG‐OH‐Archaeol), monoglycosidic hydroxyarchaeol (1G‐OH‐Archaeol), phosphatidylserine hydroxyarchaeol (PS‐OH‐Archaeol); phosphatidylethanolamine acyl ether glycerol (PE‐AEG‐C<sub>33:2</sub>), acyletherglycerol (AEG‐C<sub>35:0</sub>); ubiquinone‐8:8 (UQ‐8:8), phosphatidylcholine diacylglycerol C<sub>32:1</sub> (PC‐DAG‐C<sub>32:1</sub>), phosphatidylcholine diacylglycerol C<sub>32:1</sub> (PC‐DAG‐C<sub>32:1</sub>), triacylglycerol‐C<sub>46:0</sub> (TAG‐C<sub>46:0</sub>), triacylglycerol‐C<sub>48:0</sub> (TAG‐C<sub>48:0</sub>), ceramide (Cer) 18;02/16;0, pyropheophorbide <em>a</em> 27:2 sterylester, pyropheophytin <em>a</em> and hydroxypheophytin <em>a</em>. Metabolites are categorized, based on whether a direct or indirect connection to methane turnover has been reported (bottom), versus those with no conceivable association to methane biogeochemistry (top). The most abundant adduct for each compound was used for δ<sup>13</sup>C calculation. Datapoints represent individual UHPLC/HRMS measurements following Wörmer et al. [<a href="#rcm10118-bib-0031" class="usa-link" aria-describedby="rcm10118-bib-0031">31</a>] for sediment samples along depth profiles (0–20 cm below seafloor at 2 cm intervals) that met the following conditions: M0 Peak area &gt; 2 × 10<sup>6</sup> arb.u. (see Figure <a href="#rcm10118-fig-0001" class="usa-link">1</a>), no coeluting peaks and no signs of detector oversaturation. The shaded region indicates the range of methanotrophic δ<sup>13</sup>C values reported in previous studies in environmental samples [<a href="#rcm10118-bib-0005" class="usa-link" aria-describedby="rcm10118-bib-0005">5</a>, <a href="#rcm10118-bib-0006" class="usa-link" aria-describedby="rcm10118-bib-0006">6</a>, <a href="#rcm10118-bib-0007" class="usa-link" aria-describedby="rcm10118-bib-0007">7</a>, <a href="#rcm10118-bib-0008" class="usa-link" aria-describedby="rcm10118-bib-0008">8</a>, <a href="#rcm10118-bib-0009" class="usa-link" aria-describedby="rcm10118-bib-0009">9</a>, <a href="#rcm10118-bib-0010" class="usa-link" aria-describedby="rcm10118-bib-0010">10</a>, <a href="#rcm10118-bib-0011" class="usa-link" aria-describedby="rcm10118-bib-0011">11</a>], with the gray shaded region highlighting the range (−65‰ to −50‰), where source attribution must be nuanced due to the higher uncertainty of this approach.</p></figcaption></figure><p>To contrast these comparably high δ<sup>13</sup>C values, we selected six IPLs widely recognized as potential biomarkers for ANME, specifically of the ANME‐2 cluster [<a href="#rcm10118-bib-0090" class="usa-link" aria-describedby="rcm10118-bib-0090">90</a>, <a href="#rcm10118-bib-0091" class="usa-link" aria-describedby="rcm10118-bib-0091">91</a>], in methane‐rich environments. These include diglycosidic archaeol (2G‐archaeol), phosphatidylglycerol archaeol (PG‐archaeol), phosphatidylinositol archaeol (PI‐archaeol), phosphatidylglycerol hydroxyarchaeol (PG‐OH‐archaeol), monoglycosidic hydroxyarchaeol (1G‐OH‐archaeol), and phosphatidylserine hydroxyarchaeol (PS‐OH‐archaeol). Given the well‐documented presence of ANME in the surficial sediments of the Cathedral Hill and Aceto Balsamico area [<a href="#rcm10118-bib-0062" class="usa-link" aria-describedby="rcm10118-bib-0062">62</a>], an isotopically light signature of these lipids is expected. Indeed, all potential ANME‐related lipids are substantially <sup>13</sup>C‐depleted with δ<sup>13</sup>C values ranging from −65‰ down to −114‰. These values fall within the δ<sup>13</sup>C range of archaeal lipids commonly found in sediments with high methane turnover (approximately −60‰ to −140‰) [<a href="#rcm10118-bib-0005" class="usa-link" aria-describedby="rcm10118-bib-0005">5</a>, <a href="#rcm10118-bib-0006" class="usa-link" aria-describedby="rcm10118-bib-0006">6</a>, <a href="#rcm10118-bib-0007" class="usa-link" aria-describedby="rcm10118-bib-0007">7</a>, <a href="#rcm10118-bib-0008" class="usa-link" aria-describedby="rcm10118-bib-0008">8</a>, <a href="#rcm10118-bib-0009" class="usa-link" aria-describedby="rcm10118-bib-0009">9</a>, <a href="#rcm10118-bib-0010" class="usa-link" aria-describedby="rcm10118-bib-0010">10</a>, <a href="#rcm10118-bib-0011" class="usa-link" aria-describedby="rcm10118-bib-0011">11</a>] (Figure <a href="#rcm10118-fig-0005" class="usa-link">5</a>). The chemotaxonomically less specific archaeol, which could be regarded as a degradation product from various intact archaeols, shows a larger range of δ<sup>13</sup>C values, indicating increased admixtures of other archaeal sources in some samples (cf. Figure <a href="#rcm10118-fig-0003" class="usa-link">3</a>).</p>
<p>Aside from the <sup>13</sup>C‐depleted archaeal lipids, several bacterial ether lipids also showed considerable<sup>13</sup>C‐depletion. These include phosphatidylethanolamine acyletherglycerol C<sub>33:2</sub> (PE‐AEG‐C<sub>33:2</sub>) and acyletherglycerol C<sub>35:0</sub> (AEG‐C<sub>35:0</sub>), whose δ<sup>13</sup>C values remain below −60‰ (Figure <a href="#rcm10118-fig-0005" class="usa-link">5</a>). In various marine environments affected by AOM processes, PE‐AEG lipids are often found to be associated with mesophilic or thermophilic sulfate‐reducing bacteria (SRB) that are considered to be syntrophic partner bacteria in AOM [<a href="#rcm10118-bib-0090" class="usa-link" aria-describedby="rcm10118-bib-0090">90</a>, <a href="#rcm10118-bib-0092" class="usa-link" aria-describedby="rcm10118-bib-0092">92</a>, <a href="#rcm10118-bib-0093" class="usa-link" aria-describedby="rcm10118-bib-0093">93</a>, <a href="#rcm10118-bib-0094" class="usa-link" aria-describedby="rcm10118-bib-0094">94</a>]. In hydrothermal sediments of the Guaymas Basin, SRB commonly occur in association with sulfate‐dependent AOM [<a href="#rcm10118-bib-0061" class="usa-link" aria-describedby="rcm10118-bib-0061">61</a>, <a href="#rcm10118-bib-0062" class="usa-link" aria-describedby="rcm10118-bib-0062">62</a>, <a href="#rcm10118-bib-0063" class="usa-link" aria-describedby="rcm10118-bib-0063">63</a>]. The observed <sup>13</sup>C‐depletion in PE‐AEG‐C<sub>33:2</sub> and PE‐AEG‐C<sub>35:0</sub> in our samples indicates that the indirect incorporation of methane‐derived carbon contributes to the <sup>13</sup>C‐depleted nature of the corresponding lipid biomarkers as observed elsewhere [<a href="#rcm10118-bib-0016" class="usa-link" aria-describedby="rcm10118-bib-0016">16</a>].</p>
<p>The clear separation of isotopically light AOM‐associated core and intact lipids from comparatively <sup>13</sup>C‐enriched compounds with no connection to methane biogeochemistry presented in Figure <a href="#rcm10118-fig-0005" class="usa-link">5</a> underscores the utility of our proposed method for δ<sup>13</sup>C analysis of intact molecules in complex samples, which intends to identify the distinctly <sup>13</sup>C‐depleted molecular signals indicative of the assimilation of methane‐derived carbon. We emphasize that our proposed approach is not designed to achieve per‐mille level precision and accuracy, and the analytical uncertainty, the uncertainty introduced by the approximated δ<sup>2</sup>H value, and potential adduct‐dependent offset (e.g., H<sup>+</sup> and NH₄<sup>+</sup>) must be acknowledged as detailed in Section <a href="#rcm10118-sec-0013" class="usa-link">3.2</a>. In particular, when δ<sup>13</sup>C values fall in a transitional range of ~−50‰ to −65‰, a straightforward distinction of lipids from methane‐oxidizing and methanogenic archaea may be difficult [<a href="#rcm10118-bib-0016" class="usa-link" aria-describedby="rcm10118-bib-0016">16</a>].</p>
<p>There is broad potential for applying the isotopologue‐based approach for δ<sup>13</sup>C analysis in lipidomics for targeting complex and intact lipids not amenable to traditional GC‐based CSIA. Particularly interesting target compounds include glycerol dialkyl glycerol tetraethers (GDGTs), which are frequently associated with ANME‐1 in methane‐rich environments [<a href="#rcm10118-bib-0090" class="usa-link" aria-describedby="rcm10118-bib-0090">90</a>, <a href="#rcm10118-bib-0091" class="usa-link" aria-describedby="rcm10118-bib-0091">91</a>]. Although the GDGT signals in the current dataset did not meet the required criteria (peak areas below the set threshold and minor coelution of ring‐bearing GDGTs) for robust δ<sup>13</sup>C calculation, our approach is applicable for δ<sup>13</sup>C analysis of GDGT‐based lipids in other datasets where the required criteria are met. Beyond the identification of isotopic signatures related to methane turnover, our approach may also be applied to experiments involving <sup>13</sup>C‐labeled substrates. The <sup>13</sup>C enrichment typically observed in lipids during labeling experiments far exceeds the natural isotopic variability, making our approach highly suitable for tracing compound‐specific<sup>13</sup>C‐label incorporation.</p></section></section><section id="rcm10118-sec-0015"><h2 class="pmc_sec_title">4. Conclusions</h2>
<p>We demonstrated that the M1/M0 ratios acquired from UHPLC/HRMS analysis of sedimentary lipid extracts capture natural isotopic variations related to extreme <sup>13</sup>C‐depletion in lipids associated with microbial methane oxidation and can be utilized to calculate approximate δ<sup>13</sup>C values according to Equations (<a href="#rcm10118-disp-0001" class="usa-link">1</a>) and (<a href="#rcm10118-disp-0002" class="usa-link">2</a>). To produce reliable δ<sup>13</sup>C values, it is essential to determine the optimal intensity range for a given MS system and LC separation protocol in use while ensuring that coeluting peaks are avoided. When these criteria are met, this technique offers a promising approach for determining the stable carbon isotopic compositions of a wide range of microbial lipids that were previously inaccessible via GC/irMS. We emphasize that this approach is currently focused exclusively on identifying large isotopic deviations characteristic of lipids related to high methane turnover and is not optimized for more fine‐scale isotopic fingerprinting. Other future applications include stable isotope probing experiments targeting lipids, where the accuracy and precision achieved here are typically fully satisfactory for the quantification of label uptake. In conclusion, our proposed approach enables the analysis of a wide spectrum of nonvolatile, chemotaxonomically specific biomolecules with the aim of detecting microbial methane turnover in complex environmental samples.</p></section><section id="rcm10118-sec-0020"><h2 class="pmc_sec_title">Author Contributions</h2>
<p>
<strong>Janina Groninga:</strong> conceptualization, writing – original draft, investigation, visualization, writing – review and editing, formal analysis. <strong>Julius Lipp:</strong> conceptualization, writing – review and editing, methodology, resources. <strong>Min Song:</strong> writing – review and editing, resources. <strong>Kai‐Uwe Hinrichs:</strong> conceptualization, supervision, resources, writing – review and editing.</p></section><section id="rcm10118-sec-0017"><h2 class="pmc_sec_title">Conflicts of Interest</h2>
<p>The authors declare no conflicts of interest.</p></section><section id="sec18"><h2 class="pmc_sec_title">Supporting information</h2>
<section class="sm xbox font-sm" id="rcm10118-supitem-0001"><div class="caption p">
<p>
<strong>Table S1:</strong> Three examples showing the influence of different δ<sup>15</sup>N values on δ<sup>13</sup>C determination based on Equations (<a href="#rcm10118-disp-0001" class="usa-link">1</a>) and (<a href="#rcm10118-disp-0002" class="usa-link">2</a>), assuming a fixed δ<sup>2</sup>H value of −234‰. M1/M0 ratio for ammoniated archaeolstd (C43H88O3 + NH4<sup>+</sup>) was adjusted using an assumed δ<sup>15</sup>N value of −30‰, 0‰, and 30‰ relative to atmospheric N2 to account for potential variation in the nitrogen isotopic composition.</p>
<p>
<strong>Figure S1:</strong> Variability of reported δ<sup>2</sup>H values (‰ vs. VSMOW) in lipids derived from environmental matrices, categorized by the inferred biological origin. Lipids were grouped into archaeal, bacterial, and eukaryotic based on chemotaxonomy. <strong>Archaeal lipids:</strong> Archaeol, hydroxyarchaeol, biphytanes. <strong>Bacterial lipids:</strong> Methyl‐branched fatty acids, odd‐chain fatty acids (C<sub>15</sub>, C<sub>17</sub>, C<sub>19</sub>), cyclopropyl fatty acids, and hopanols. <strong>Eukaryotic lipids:</strong> Even‐chain short‐chain fatty acids (C<sub>14</sub>, C<sub>16</sub>, C<sub>18</sub>), polyunsaturated fatty acids, sterols, phytol, and phytol derivatives. Minimum, maximum, and midpoint δ<sup>2</sup>H values are shown for each group. δ<sup>2</sup>H data were collected from the following studies: Sessions et al. [34]; Li et al. [43]; Jones et al. [42]; Naraoka et al. [46]; Kaneko et al. [35]; Osburn et al. [45]; Wegener et al. [48]; Dawson et al. [47]; Kellerman et al. [44]; Heinzelmann et al. [39, 41].</p>
<p>
<strong>Figure S2:</strong> Relationship between the on‐column of archaeol standard injections (in ng) and δ13C value. The δ<sup>13</sup>C values were calculated for the [M + NH₄]⁺ adduct and all measurements were conducted using reversed phase‐UHPLC/ESI/qToF/MS following the method of Wörmer et al. [31]. The dashed line indicates the δ<sup>13</sup>C value of the archaeol standard determined by EA‐IRMS (−33.3‰).</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12322946/bin/RCM-39-e10118-s001.pdf" data-ga-action="click_feat_suppl" class="usa-link">RCM-39-e10118-s001.pdf</a><sup> (540.3KB, pdf) </sup>
</div></div></section></section><section id="rcm10118-sec-0016" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>We thank Jenny Altun and Julia Cordes for their assistance with instrumental analyses (GC/irMS and UHPLC/HRMS). We also extend our gratitude to the captain, crew, and shipboard scientists aboard cruises AT42‐05 and AT37‐06 for their efforts in sample collection. This research was funded by the Deutsche Forschungsgemeinschaft (DFG) under Germany's Excellence Strategy, Grant No. EXC‐2077‐390741603. Open Access funding enabled and organized by Projekt DEAL.</p></section><section id="notes1"><p>

Groninga J., Lipp J., Song M., and Hinrichs K.-U., “Isotopologue Ratios Identify <sup>13</sup>C‐Depleted Biomarkers in Environmental Samples Impacted by Methane Turnover,” Rapid Communications in Mass Spectrometry
39, no. 22 (2025): e10118, 10.1002/rcm.10118.
</p>
<section id="rcm10118-ntgp-0001" class="fn-group"><div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="rcm10118-note-0001"><p>
<strong>Funding:</strong> This work was supported by Deutsche Forschungsgemeinschaft (EXC‐2077‐390741603).</p></div></div></section></section><section id="rcm10118-sec-0019"><h2 class="pmc_sec_title">Data Availability Statement</h2>
<p>The data that support the findings of this study are available from the corresponding author upon request. Georeferenced data can be accessed via the PANGAEA data repository.</p></section><section id="rcm10118-bibl-0001" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="rcm10118-bibl-0001_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="rcm10118-bib-0001">
<span class="label">1.</span><cite>
Abraham W. R., Hesse C., and Pelz O., “Ratios of Carbon Isotopes in Microbial Lipids as an Indicator of Substrate Usage,” Applied and Environmental Microbiology
64, no. 11 (1998): 4202–4209, 10.1128/AEM.64.11.4202-4209.1998.
</cite> [<a href="https://doi.org/10.1128/AEM.64.11.4202-4209.1998" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC106628/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9797266/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Applied%20and%20Environmental%20Microbiology&amp;title=Ratios%20of%20Carbon%20Isotopes%20in%20Microbial%20Lipids%20as%20an%20Indicator%20of%20Substrate%20Usage&amp;author=W.%20R.%20Abraham&amp;author=C.%20Hesse&amp;author=O.%20Pelz&amp;volume=64&amp;issue=11&amp;publication_year=1998&amp;pages=4202-4209&amp;pmid=9797266&amp;doi=10.1128/AEM.64.11.4202-4209.1998&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10118-bib-0002">
<span class="label">2.</span><cite>
Pancost R. D. and Sinninghe Damsté J. S., “Carbon Isotopic Compositions of Prokaryotic Lipids as Tracers of Carbon Cycling in Diverse Settings,” Chemical Geology
195, no. 1–4 (2003): 29–58, 10.1016/S0009-2541(02)00387-X.</cite> [<a href="https://doi.org/10.1016/S0009-2541(02)00387-X" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Chemical%20Geology&amp;title=Carbon%20Isotopic%20Compositions%20of%20Prokaryotic%20Lipids%20as%20Tracers%20of%20Carbon%20Cycling%20in%20Diverse%20Settings&amp;author=R.%20D.%20Pancost&amp;author=J.%20S.%20Sinninghe%20Damst%C3%A9&amp;volume=195&amp;issue=1%E2%80%934&amp;publication_year=2003&amp;pages=29-58&amp;doi=10.1016/S0009-2541(02)00387-X&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10118-bib-0003">
<span class="label">3.</span><cite>
Hayes J. M., “Fractionation of Carbon and Hydrogen Isotopes in Biosynthetic Processes,” Reviews in Mineralogy and Geochemistry
43 (2001): 225–278.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Reviews%20in%20Mineralogy%20and%20Geochemistry&amp;title=Fractionation%20of%20Carbon%20and%20Hydrogen%20Isotopes%20in%20Biosynthetic%20Processes&amp;author=J.%20M.%20Hayes&amp;volume=43&amp;publication_year=2001&amp;pages=225-278&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10118-bib-0004">
<span class="label">4.</span><cite>
Londry K. L., Jahnke L. L., and Des Marais D. J., “Stable Carbon Isotope Ratios of Lipid Biomarkers of Sulfate‐Reducing Bacteria,” Applied and Environmental Microbiology
70, no. 2 (2004): 745–751, 10.1128/AEM.70.2.745-751.2004.
</cite> [<a href="https://doi.org/10.1128/AEM.70.2.745-751.2004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC348830/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/14766550/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Applied%20and%20Environmental%20Microbiology&amp;title=Stable%20Carbon%20Isotope%20Ratios%20of%20Lipid%20Biomarkers%20of%20Sulfate%E2%80%90Reducing%20Bacteria&amp;author=K.%20L.%20Londry&amp;author=L.%20L.%20Jahnke&amp;author=D.%20J.%20Des%20Marais&amp;volume=70&amp;issue=2&amp;publication_year=2004&amp;pages=745-751&amp;pmid=14766550&amp;doi=10.1128/AEM.70.2.745-751.2004&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10118-bib-0005">
<span class="label">5.</span><cite>
Hinrichs K. U., Hayes J. M., Sylva S. P., Brewer P. G., and DeLong E. F., “Methane‐Consuming Archaebacteria in Marine Sediments,” Nature
398, no. 6730 (1999): 802–805, 10.1038/19751.
</cite> [<a href="https://doi.org/10.1038/19751" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10235261/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Methane%E2%80%90Consuming%20Archaebacteria%20in%20Marine%20Sediments&amp;author=K.%20U.%20Hinrichs&amp;author=J.%20M.%20Hayes&amp;author=S.%20P.%20Sylva&amp;author=P.%20G.%20Brewer&amp;author=E.%20F.%20DeLong&amp;volume=398&amp;issue=6730&amp;publication_year=1999&amp;pages=802-805&amp;pmid=10235261&amp;doi=10.1038/19751&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10118-bib-0006">
<span class="label">6.</span><cite>
Pancost R. D., Sinninghe Damsté J. S., De Lint S., Van Der Maarel M. J. E. C., Gottschal J. C., and The Medinaut Shipboard Scientific Party
, “Biomarker Evidence for Widespread Anaerobic Methane Oxidation in Mediterranean Sediments by a Consortium of Methanogenic Archaea and Bacteria,” Applied and Environmental Microbiology
66, no. 3 (2000): 1126–1132, 10.1128/AEM.66.3.1126-1132.2000.
</cite> [<a href="https://doi.org/10.1128/AEM.66.3.1126-1132.2000" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC91952/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10698781/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Applied%20and%20Environmental%20Microbiology&amp;title=Biomarker%20Evidence%20for%20Widespread%20Anaerobic%20Methane%20Oxidation%20in%20Mediterranean%20Sediments%20by%20a%20Consortium%20of%20Methanogenic%20Archaea%20and%20Bacteria&amp;author=R.%20D.%20Pancost&amp;author=J.%20S.%20Sinninghe%20Damst%C3%A9&amp;author=S.%20De%20Lint&amp;author=M.%20J.%20E.%20C.%20Van%20Der%20Maarel&amp;author=J.%20C.%20Gottschal&amp;volume=66&amp;issue=3&amp;publication_year=2000&amp;pages=1126-1132&amp;pmid=10698781&amp;doi=10.1128/AEM.66.3.1126-1132.2000&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10118-bib-0007">
<span class="label">7.</span><cite>
Blumenberg M., Seifert R., Reitner J., Pape T., and Michaelis W., “Membrane Lipid Patterns Typify Distinct Anaerobic Methanotrophic Consortia,” Proceedings of the National Academy of Sciences
101, no. 30 (2004): 11111–11116, 10.1073/pnas.0401188101.</cite> [<a href="https://doi.org/10.1073/pnas.0401188101" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC503748/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15258285/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proceedings%20of%20the%20National%20Academy%20of%20Sciences&amp;title=Membrane%20Lipid%20Patterns%20Typify%20Distinct%20Anaerobic%20Methanotrophic%20Consortia&amp;author=M.%20Blumenberg&amp;author=R.%20Seifert&amp;author=J.%20Reitner&amp;author=T.%20Pape&amp;author=W.%20Michaelis&amp;volume=101&amp;issue=30&amp;publication_year=2004&amp;pages=11111-11116&amp;pmid=15258285&amp;doi=10.1073/pnas.0401188101&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10118-bib-0008">
<span class="label">8.</span><cite>
Elvert M., Hopmans E. C., Treude T., Boetius A., and Suess E., “Spatial Variations of Methanotrophic Consortia at Cold Methane Seeps: Implications From a High‐Resolution Molecular and Isotopic Approach,” Geobiology
3, no. 3 (2005): 195–209, 10.1111/j.1472-4669.2005.00051.x.</cite> [<a href="https://doi.org/10.1111/j.1472-4669.2005.00051.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Geobiology&amp;title=Spatial%20Variations%20of%20Methanotrophic%20Consortia%20at%20Cold%20Methane%20Seeps:%20Implications%20From%20a%20High%E2%80%90Resolution%20Molecular%20and%20Isotopic%20Approach&amp;author=M.%20Elvert&amp;author=E.%20C.%20Hopmans&amp;author=T.%20Treude&amp;author=A.%20Boetius&amp;author=E.%20Suess&amp;volume=3&amp;issue=3&amp;publication_year=2005&amp;pages=195-209&amp;doi=10.1111/j.1472-4669.2005.00051.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10118-bib-0009">
<span class="label">9.</span><cite>
Niemann H., Elvert M., Hovland M., et al., “Methane Emission and Consumption at a North Sea Gas Seep (Tommeliten Area),” Biogeosciences
2, no. 4 (2005): 335–351, 10.5194/bg-2-335-2005.</cite> [<a href="https://doi.org/10.5194/bg-2-335-2005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biogeosciences&amp;title=Methane%20Emission%20and%20Consumption%20at%20a%20North%20Sea%20Gas%20Seep%20(Tommeliten%20Area)&amp;author=H.%20Niemann&amp;author=M.%20Elvert&amp;author=M.%20Hovland&amp;volume=2&amp;issue=4&amp;publication_year=2005&amp;pages=335-351&amp;doi=10.5194/bg-2-335-2005&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10118-bib-0010">
<span class="label">10.</span><cite>
Niemann H. and Elvert M., “Diagnostic Lipid Biomarker and Stable Carbon Isotope Signatures of Microbial Communities Mediating the Anaerobic Oxidation of Methane With Sulphate,” Organic Geochemistry
39, no. 12 (2008): 1668–1677, 10.1016/j.orggeochem.2007.11.003.</cite> [<a href="https://doi.org/10.1016/j.orggeochem.2007.11.003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Organic%20Geochemistry&amp;title=Diagnostic%20Lipid%20Biomarker%20and%20Stable%20Carbon%20Isotope%20Signatures%20of%20Microbial%20Communities%20Mediating%20the%20Anaerobic%20Oxidation%20of%20Methane%20With%20Sulphate&amp;author=H.%20Niemann&amp;author=M.%20Elvert&amp;volume=39&amp;issue=12&amp;publication_year=2008&amp;pages=1668-1677&amp;doi=10.1016/j.orggeochem.2007.11.003&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10118-bib-0011">
<span class="label">11.</span><cite>
Haas A., Peckmann J., Elvert M., Sahling H., and Bohrmann G., “Patterns of Carbonate Authigenesis at the Kouilou Pockmarks on the Congo Deep‐Sea Fan,” Marine Geology
268, no. 1–4 (2010): 129–136, 10.1016/j.margeo.2009.10.027.</cite> [<a href="https://doi.org/10.1016/j.margeo.2009.10.027" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Marine%20Geology&amp;title=Patterns%20of%20Carbonate%20Authigenesis%20at%20the%20Kouilou%20Pockmarks%20on%20the%20Congo%20Deep%E2%80%90Sea%20Fan&amp;author=A.%20Haas&amp;author=J.%20Peckmann&amp;author=M.%20Elvert&amp;author=H.%20Sahling&amp;author=G.%20Bohrmann&amp;volume=268&amp;issue=1%E2%80%934&amp;publication_year=2010&amp;pages=129-136&amp;doi=10.1016/j.margeo.2009.10.027&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10118-bib-0012">
<span class="label">12.</span><cite>
Hinrichs K. U., Summons R. E., Orphan V., Sylva S. P., and Hayes J. M., “Molecular and Isotopic Analysis of Anaerobic Methane‐Oxidizing Communities in Marine Sediments,” Organic Geochemistry
31, no. 12 (2000): 1685–1701, 10.1016/S0146-6380(00)00106-6.</cite> [<a href="https://doi.org/10.1016/S0146-6380(00)00106-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Organic%20Geochemistry&amp;title=Molecular%20and%20Isotopic%20Analysis%20of%20Anaerobic%20Methane%E2%80%90Oxidizing%20Communities%20in%20Marine%20Sediments&amp;author=K.%20U.%20Hinrichs&amp;author=R.%20E.%20Summons&amp;author=V.%20Orphan&amp;author=S.%20P.%20Sylva&amp;author=J.%20M.%20Hayes&amp;volume=31&amp;issue=12&amp;publication_year=2000&amp;pages=1685-1701&amp;doi=10.1016/S0146-6380(00)00106-6&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10118-bib-0013">
<span class="label">13.</span><cite>
Kellermann M. Y., Wegener G., Elvert M., et al., “Autotrophy as a Predominant Mode of Carbon Fixation in Anaerobic Methane‐Oxidizing Microbial Communities,” Proceedings of the National Academy of Sciences
109, no. 47 (2012): 19321–19326, 10.1073/pnas.1208795109.</cite> [<a href="https://doi.org/10.1073/pnas.1208795109" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3511159/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23129626/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proceedings%20of%20the%20National%20Academy%20of%20Sciences&amp;title=Autotrophy%20as%20a%20Predominant%20Mode%20of%20Carbon%20Fixation%20in%20Anaerobic%20Methane%E2%80%90Oxidizing%20Microbial%20Communities&amp;author=M.%20Y.%20Kellermann&amp;author=G.%20Wegener&amp;author=M.%20Elvert&amp;volume=109&amp;issue=47&amp;publication_year=2012&amp;pages=19321-19326&amp;pmid=23129626&amp;doi=10.1073/pnas.1208795109&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10118-bib-0014">
<span class="label">14.</span><cite>
Wegener G., Niemann H., Elvert M., Hinrichs K., and Boetius A., “Assimilation of Methane and Inorganic Carbon by Microbial Communities Mediating the Anaerobic Oxidation of Methane,” Environmental Microbiology
10, no. 9 (2008): 2287–2298, 10.1111/j.1462-2920.2008.01653.x.
</cite> [<a href="https://doi.org/10.1111/j.1462-2920.2008.01653.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18498367/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Environmental%20Microbiology&amp;title=Assimilation%20of%20Methane%20and%20Inorganic%20Carbon%20by%20Microbial%20Communities%20Mediating%20the%20Anaerobic%20Oxidation%20of%20Methane&amp;author=G.%20Wegener&amp;author=H.%20Niemann&amp;author=M.%20Elvert&amp;author=K.%20Hinrichs&amp;author=A.%20Boetius&amp;volume=10&amp;issue=9&amp;publication_year=2008&amp;pages=2287-2298&amp;pmid=18498367&amp;doi=10.1111/j.1462-2920.2008.01653.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10118-bib-0015">
<span class="label">15.</span><cite>
Zhu Q. Z., Wegener G., Hinrichs K. U., and Elvert M., “Activity of Ancillary Heterotrophic Community Members in Anaerobic Methane‐Oxidizing Cultures,” Frontiers in Microbiology
13 (2022): 912299, 10.3389/fmicb.2022.912299.
</cite> [<a href="https://doi.org/10.3389/fmicb.2022.912299" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9201399/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35722308/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Microbiology&amp;title=Activity%20of%20Ancillary%20Heterotrophic%20Community%20Members%20in%20Anaerobic%20Methane%E2%80%90Oxidizing%20Cultures&amp;author=Q.%20Z.%20Zhu&amp;author=G.%20Wegener&amp;author=K.%20U.%20Hinrichs&amp;author=M.%20Elvert&amp;volume=13&amp;publication_year=2022&amp;pages=912299&amp;pmid=35722308&amp;doi=10.3389/fmicb.2022.912299&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10118-bib-0016">
<span class="label">16.</span><cite>
Schubotz F., Lipp J. S., Elvert M., and Hinrichs K. U., “Stable Carbon Isotopic Compositions of Intact Polar Lipids Reveal Complex Carbon Flow Patterns Among Hydrocarbon Degrading Microbial Communities at the Chapopote Asphalt Volcano,” Geochimica et Cosmochimica Acta
75, no. 16 (2011): 4399–4415, 10.1016/j.gca.2011.05.018.</cite> [<a href="https://doi.org/10.1016/j.gca.2011.05.018" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Geochimica%20et%20Cosmochimica%20Acta&amp;title=Stable%20Carbon%20Isotopic%20Compositions%20of%20Intact%20Polar%20Lipids%20Reveal%20Complex%20Carbon%20Flow%20Patterns%20Among%20Hydrocarbon%20Degrading%20Microbial%20Communities%20at%20the%20Chapopote%20Asphalt%20Volcano&amp;author=F.%20Schubotz&amp;author=J.%20S.%20Lipp&amp;author=M.%20Elvert&amp;author=K.%20U.%20Hinrichs&amp;volume=75&amp;issue=16&amp;publication_year=2011&amp;pages=4399-4415&amp;doi=10.1016/j.gca.2011.05.018&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10118-bib-0017">
<span class="label">17.</span><cite>
Summons R. E., Franzmann P. D., and Nichols P. D., “Carbon Isotopic Fractionation Associated With Methylotrophic Methanogenesis,” Organic Geochemistry
28, no. 7–8 (1998): 465–475, 10.1016/S0146-6380(98)00011-4.</cite> [<a href="https://doi.org/10.1016/S0146-6380(98)00011-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Organic%20Geochemistry&amp;title=Carbon%20Isotopic%20Fractionation%20Associated%20With%20Methylotrophic%20Methanogenesis&amp;author=R.%20E.%20Summons&amp;author=P.%20D.%20Franzmann&amp;author=P.%20D.%20Nichols&amp;volume=28&amp;issue=7%E2%80%938&amp;publication_year=1998&amp;pages=465-475&amp;doi=10.1016/S0146-6380(98)00011-4&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10118-bib-0018">
<span class="label">18.</span><cite>
Londry K. L., Dawson K. G., Grover H. D., Summons R. E., and Bradley A. S., “Stable Carbon Isotope Fractionation Between Substrates and Products of <em>Methanosarcina barkeri</em>
,” Organic Geochemistry
39, no. 5 (2008): 608–621, 10.1016/j.orggeochem.2008.03.002.</cite> [<a href="https://doi.org/10.1016/j.orggeochem.2008.03.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Organic%20Geochemistry&amp;title=Stable%20Carbon%20Isotope%20Fractionation%20Between%20Substrates%20and%20Products%20of%20Methanosarcina%20barkeri&amp;author=K.%20L.%20Londry&amp;author=K.%20G.%20Dawson&amp;author=H.%20D.%20Grover&amp;author=R.%20E.%20Summons&amp;author=A.%20S.%20Bradley&amp;volume=39&amp;issue=5&amp;publication_year=2008&amp;pages=608-621&amp;doi=10.1016/j.orggeochem.2008.03.002&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10118-bib-0019">
<span class="label">19.</span><cite>
Whiticar M. J., “Carbon and Hydrogen Isotope Systematics of Bacterial Formation and Oxidation of Methane,” Chemical Geology
161, no. 1–3 (1999): 291–314, 10.1016/S0009-2541(99)00092-3.</cite> [<a href="https://doi.org/10.1016/S0009-2541(99)00092-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Chemical%20Geology&amp;title=Carbon%20and%20Hydrogen%20Isotope%20Systematics%20of%20Bacterial%20Formation%20and%20Oxidation%20of%20Methane&amp;author=M.%20J.%20Whiticar&amp;volume=161&amp;issue=1%E2%80%933&amp;publication_year=1999&amp;pages=291-314&amp;doi=10.1016/S0009-2541(99)00092-3&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10118-bib-0020">
<span class="label">20.</span><cite>
Matthews D. E. and Hayes J. M., “Isotope‐Ratio‐Monitoring Gas Chromatography‐Mass Spectrometry,” Analytical Chemistry
50, no. 11 (1978): 1465–1473.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Analytical%20Chemistry&amp;title=Isotope%E2%80%90Ratio%E2%80%90Monitoring%20Gas%20Chromatography%E2%80%90Mass%20Spectrometry&amp;author=D.%20E.%20Matthews&amp;author=J.%20M.%20Hayes&amp;volume=50&amp;issue=11&amp;publication_year=1978&amp;pages=1465-1473&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10118-bib-0021">
<span class="label">21.</span><cite>
Sessions A. L., “Isotope‐Ratio Detection for Gas Chromatography,” Journal of Separation Science
29, no. 12 (2006): 1946–1961, 10.1002/jssc.200600002.
</cite> [<a href="https://doi.org/10.1002/jssc.200600002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16970194/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Separation%20Science&amp;title=Isotope%E2%80%90Ratio%20Detection%20for%20Gas%20Chromatography&amp;author=A.%20L.%20Sessions&amp;volume=29&amp;issue=12&amp;publication_year=2006&amp;pages=1946-1961&amp;pmid=16970194&amp;doi=10.1002/jssc.200600002&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10118-bib-0022">
<span class="label">22.</span><cite>
Pearson A., Hurley S. J., Walter S. R. S., Kusch S., Lichtin S., and Zhang Y. G., “Stable Carbon Isotope Ratios of Intact GDGTs Indicate Heterogeneous Sources to Marine Sediments,” Geochimica et Cosmochimica Acta
181 (2016): 18–35, 10.1016/j.gca.2016.02.034.</cite> [<a href="https://doi.org/10.1016/j.gca.2016.02.034" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Geochimica%20et%20Cosmochimica%20Acta&amp;title=Stable%20Carbon%20Isotope%20Ratios%20of%20Intact%20GDGTs%20Indicate%20Heterogeneous%20Sources%20to%20Marine%20Sediments&amp;author=A.%20Pearson&amp;author=S.%20J.%20Hurley&amp;author=S.%20R.%20S.%20Walter&amp;author=S.%20Kusch&amp;author=S.%20Lichtin&amp;volume=181&amp;publication_year=2016&amp;pages=18-35&amp;doi=10.1016/j.gca.2016.02.034&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10118-bib-0023">
<span class="label">23.</span><cite>
Taubert M., Jehmlich N., Vogt C., et al., “Time Resolved Protein‐Based Stable Isotope Probing (Protein‐SIP) Analysis Allows Quantification of Induced Proteins in Substrate Shift Experiments,” Proteomics
11, no. 11 (2011): 2265–2274, 10.1002/pmic.201000788.
</cite> [<a href="https://doi.org/10.1002/pmic.201000788" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21598395/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proteomics&amp;title=Time%20Resolved%20Protein%E2%80%90Based%20Stable%20Isotope%20Probing%20(Protein%E2%80%90SIP)%20Analysis%20Allows%20Quantification%20of%20Induced%20Proteins%20in%20Substrate%20Shift%20Experiments&amp;author=M.%20Taubert&amp;author=N.%20Jehmlich&amp;author=C.%20Vogt&amp;volume=11&amp;issue=11&amp;publication_year=2011&amp;pages=2265-2274&amp;pmid=21598395&amp;doi=10.1002/pmic.201000788&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10118-bib-0024">
<span class="label">24.</span><cite>
Fischer C. R., Bowen B. P., Pan C., Northen T. R., and Banfield J. F., “Stable‐Isotope Probing Reveals That Hydrogen Isotope Fractionation in Proteins and Lipids in a Microbial Community Are Different and Species‐Specific,” ACS Chemical Biology
8, no. 8 (2013): 1755–1763, 10.1021/cb400210q.
</cite> [<a href="https://doi.org/10.1021/cb400210q" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23713674/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=ACS%20Chemical%20Biology&amp;title=Stable%E2%80%90Isotope%20Probing%20Reveals%20That%20Hydrogen%20Isotope%20Fractionation%20in%20Proteins%20and%20Lipids%20in%20a%20Microbial%20Community%20Are%20Different%20and%20Species%E2%80%90Specific&amp;author=C.%20R.%20Fischer&amp;author=B.%20P.%20Bowen&amp;author=C.%20Pan&amp;author=T.%20R.%20Northen&amp;author=J.%20F.%20Banfield&amp;volume=8&amp;issue=8&amp;publication_year=2013&amp;pages=1755-1763&amp;pmid=23713674&amp;doi=10.1021/cb400210q&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10118-bib-0025">
<span class="label">25.</span><cite>
Sachsenberg T., Herbst F. A., Taubert M., et al., “MetaProSIP: Automated Inference of Stable Isotope Incorporation Rates in Proteins for Functional Metaproteomics,” Journal of Proteome Research
14, no. 2 (2015): 619–627, 10.1021/pr500245w.
</cite> [<a href="https://doi.org/10.1021/pr500245w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25412983/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Proteome%20Research&amp;title=MetaProSIP:%20Automated%20Inference%20of%20Stable%20Isotope%20Incorporation%20Rates%20in%20Proteins%20for%20Functional%20Metaproteomics&amp;author=T.%20Sachsenberg&amp;author=F.%20A.%20Herbst&amp;author=M.%20Taubert&amp;volume=14&amp;issue=2&amp;publication_year=2015&amp;pages=619-627&amp;pmid=25412983&amp;doi=10.1021/pr500245w&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10118-bib-0026">
<span class="label">26.</span><cite>
Neubauer C., Sessions A. L., Booth I. R., et al., “Towards Measuring Growth Rates of Pathogens During Infections by D<sub>2</sub>O‐Labeling Lipidomics,” Rapid Communications in Mass Spectrometry
32, no. 24 (2018): 2129–2140, 10.1002/rcm.8288.
</cite> [<a href="https://doi.org/10.1002/rcm.8288" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30252972/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Rapid%20Communications%20in%20Mass%20Spectrometry&amp;title=Towards%20Measuring%20Growth%20Rates%20of%20Pathogens%20During%20Infections%20by%20D2O%E2%80%90Labeling%20Lipidomics&amp;author=C.%20Neubauer&amp;author=A.%20L.%20Sessions&amp;author=I.%20R.%20Booth&amp;volume=32&amp;issue=24&amp;publication_year=2018&amp;pages=2129-2140&amp;pmid=30252972&amp;doi=10.1002/rcm.8288&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10118-bib-0027">
<span class="label">27.</span><cite>
Eiler J., Cesar J., Chimiak L., et al., “Analysis of Molecular Isotopic Structures at High Precision and Accuracy by Orbitrap Mass Spectrometry,” International Journal of Mass Spectrometry
422 (2017): 126–142, 10.1016/j.ijms.2017.10.002.</cite> [<a href="https://doi.org/10.1016/j.ijms.2017.10.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=International%20Journal%20of%20Mass%20Spectrometry&amp;title=Analysis%20of%20Molecular%20Isotopic%20Structures%20at%20High%20Precision%20and%20Accuracy%20by%20Orbitrap%20Mass%20Spectrometry&amp;author=J.%20Eiler&amp;author=J.%20Cesar&amp;author=L.%20Chimiak&amp;volume=422&amp;publication_year=2017&amp;pages=126-142&amp;doi=10.1016/j.ijms.2017.10.002&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10118-bib-0028">
<span class="label">28.</span><cite>
Neubauer C., Sweredoski M. J., Moradian A., Newman D. K., Robins R. J., and Eiler J. M., “Scanning the Isotopic Structure of Molecules by Tandem Mass Spectrometry,” International Journal of Mass Spectrometry
434 (2018): 276–286, 10.1016/j.ijms.2018.08.001.</cite> [<a href="https://doi.org/10.1016/j.ijms.2018.08.001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=International%20Journal%20of%20Mass%20Spectrometry&amp;title=Scanning%20the%20Isotopic%20Structure%20of%20Molecules%20by%20Tandem%20Mass%20Spectrometry&amp;author=C.%20Neubauer&amp;author=M.%20J.%20Sweredoski&amp;author=A.%20Moradian&amp;author=D.%20K.%20Newman&amp;author=R.%20J.%20Robins&amp;volume=434&amp;publication_year=2018&amp;pages=276-286&amp;doi=10.1016/j.ijms.2018.08.001&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10118-bib-0029">
<span class="label">29.</span><cite>
Mueller E. P., Sessions A. L., Sauer P. E., Weiss G. M., and Eiler J. M., “Simultaneous, High‐Precision Measurements of δ<sup>2</sup>H and δ<sup>13</sup>C in Nanomole Quantities of Acetate Using Electrospray Ionization‐Quadrupole‐Orbitrap Mass Spectrometry,” Analytical Chemistry
94, no. 2 (2022): 1092–1100, 10.1021/acs.analchem.1c04141.
</cite> [<a href="https://doi.org/10.1021/acs.analchem.1c04141" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34967622/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Analytical%20Chemistry&amp;title=Simultaneous,%20High%E2%80%90Precision%20Measurements%20of%20%CE%B42H%20and%20%CE%B413C%20in%20Nanomole%20Quantities%20of%20Acetate%20Using%20Electrospray%20Ionization%E2%80%90Quadrupole%E2%80%90Orbitrap%20Mass%20Spectrometry&amp;author=E.%20P.%20Mueller&amp;author=A.%20L.%20Sessions&amp;author=P.%20E.%20Sauer&amp;author=G.%20M.%20Weiss&amp;author=J.%20M.%20Eiler&amp;volume=94&amp;issue=2&amp;publication_year=2022&amp;pages=1092-1100&amp;pmid=34967622&amp;doi=10.1021/acs.analchem.1c04141&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10118-bib-0030">
<span class="label">30.</span><cite>
Sturt H. F., Summons R. E., Smith K., Elvert M., and Hinrichs K., “Intact Polar Membrane Lipids in Prokaryotes and Sediments Deciphered by High‐Performance Liquid Chromatography/Electrospray Ionization Multistage Mass Spectrometry—New Biomarkers for Biogeochemistry and Microbial Ecology,” Rapid Communications in Mass Spectrometry
18, no. 6 (2004): 617–628, 10.1002/rcm.1378.
</cite> [<a href="https://doi.org/10.1002/rcm.1378" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15052572/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Rapid%20Communications%20in%20Mass%20Spectrometry&amp;title=Intact%20Polar%20Membrane%20Lipids%20in%20Prokaryotes%20and%20Sediments%20Deciphered%20by%20High%E2%80%90Performance%20Liquid%20Chromatography/Electrospray%20Ionization%20Multistage%20Mass%20Spectrometry%E2%80%94New%20Biomarkers%20for%20Biogeochemistry%20and%20Microbial%20Ecology&amp;author=H.%20F.%20Sturt&amp;author=R.%20E.%20Summons&amp;author=K.%20Smith&amp;author=M.%20Elvert&amp;author=K.%20Hinrichs&amp;volume=18&amp;issue=6&amp;publication_year=2004&amp;pages=617-628&amp;pmid=15052572&amp;doi=10.1002/rcm.1378&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10118-bib-0031">
<span class="label">31.</span><cite>
Wörmer L., Lipp J. S., Schröder J. M., and Hinrichs K. U., “Application of Two New LC–ESI–MS Methods for Improved Detection of Intact Polar Lipids (IPLs) in Environmental Samples,” Organic Geochemistry
59 (2013): 10–21, 10.1016/j.orggeochem.2013.03.004.</cite> [<a href="https://doi.org/10.1016/j.orggeochem.2013.03.004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Organic%20Geochemistry&amp;title=Application%20of%20Two%20New%20LC%E2%80%93ESI%E2%80%93MS%20Methods%20for%20Improved%20Detection%20of%20Intact%20Polar%20Lipids%20(IPLs)%20in%20Environmental%20Samples&amp;author=L.%20W%C3%B6rmer&amp;author=J.%20S.%20Lipp&amp;author=J.%20M.%20Schr%C3%B6der&amp;author=K.%20U.%20Hinrichs&amp;volume=59&amp;publication_year=2013&amp;pages=10-21&amp;doi=10.1016/j.orggeochem.2013.03.004&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10118-bib-0032">
<span class="label">32.</span><cite>
Zhu C., Lipp J. S., Wörmer L., Becker K. W., Schröder J., and Hinrichs K. U., “Comprehensive Glycerol Ether Lipid Fingerprints Through a Novel Reversed Phase Liquid Chromatography–Mass Spectrometry Protocol,” Organic Geochemistry
65 (2013): 53–62, 10.1016/j.orggeochem.2013.09.012.</cite> [<a href="https://doi.org/10.1016/j.orggeochem.2013.09.012" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Organic%20Geochemistry&amp;title=Comprehensive%20Glycerol%20Ether%20Lipid%20Fingerprints%20Through%20a%20Novel%20Reversed%20Phase%20Liquid%20Chromatography%E2%80%93Mass%20Spectrometry%20Protocol&amp;author=C.%20Zhu&amp;author=J.%20S.%20Lipp&amp;author=L.%20W%C3%B6rmer&amp;author=K.%20W.%20Becker&amp;author=J.%20Schr%C3%B6der&amp;volume=65&amp;publication_year=2013&amp;pages=53-62&amp;doi=10.1016/j.orggeochem.2013.09.012&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10118-bib-0033">
<span class="label">33.</span><cite>
Coursey J. S., Schwab D. J., Tsai J. J., and Dragoset R. A., “Atomic Weights and Isotopic Compositions (Version 4.1),” Natl Inst Stand Technol Gaithersburg MD, (2015), <a href="https://www.nist.gov/pml/atomic-weights-and-isotopic-compositions-relative-atomic-masses" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.nist.gov/pml/atomic‐weights‐and‐isotopic‐compositions‐relative‐atomic‐masses</a>.</cite>
</li>
<li id="rcm10118-bib-0034">
<span class="label">34.</span><cite>
Sessions A. L., Burgoyne T. W., Schimmelmann A., and Hayes J. M., “Fractionation of Hydrogen Isotopes in Lipid Biosynthesis,” Organic Geochemistry
30, no. 9 (1999): 1193–1200, 10.1016/S0146-6380(99)00094-7.</cite> [<a href="https://doi.org/10.1016/S0146-6380(99)00094-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Organic%20Geochemistry&amp;title=Fractionation%20of%20Hydrogen%20Isotopes%20in%20Lipid%20Biosynthesis&amp;author=A.%20L.%20Sessions&amp;author=T.%20W.%20Burgoyne&amp;author=A.%20Schimmelmann&amp;author=J.%20M.%20Hayes&amp;volume=30&amp;issue=9&amp;publication_year=1999&amp;pages=1193-1200&amp;doi=10.1016/S0146-6380(99)00094-7&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10118-bib-0035">
<span class="label">35.</span><cite>
Kaneko M., Kitajima F., and Naraoka H., “Stable Hydrogen Isotope Measurement of Archaeal Ether‐Bound Hydrocarbons,” Organic Geochemistry
42, no. 2 (2011): 166–172, 10.1016/j.orggeochem.2010.11.002.</cite> [<a href="https://doi.org/10.1016/j.orggeochem.2010.11.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Organic%20Geochemistry&amp;title=Stable%20Hydrogen%20Isotope%20Measurement%20of%20Archaeal%20Ether%E2%80%90Bound%20Hydrocarbons&amp;author=M.%20Kaneko&amp;author=F.%20Kitajima&amp;author=H.%20Naraoka&amp;volume=42&amp;issue=2&amp;publication_year=2011&amp;pages=166-172&amp;doi=10.1016/j.orggeochem.2010.11.002&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10118-bib-0036">
<span class="label">36.</span><cite>
Dirghangi S. S. and Pagani M., “Hydrogen Isotope Fractionation During Lipid Biosynthesis by <em>Haloarcula marismortui</em>
,” Geochimica et Cosmochimica Acta
119 (2013): 381–390, 10.1016/j.gca.2013.05.023.</cite> [<a href="https://doi.org/10.1016/j.gca.2013.05.023" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Geochimica%20et%20Cosmochimica%20Acta&amp;title=Hydrogen%20Isotope%20Fractionation%20During%20Lipid%20Biosynthesis%20by%20Haloarcula%20marismortui&amp;author=S.%20S.%20Dirghangi&amp;author=M.%20Pagani&amp;volume=119&amp;publication_year=2013&amp;pages=381-390&amp;doi=10.1016/j.gca.2013.05.023&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10118-bib-0037">
<span class="label">37.</span><cite>
Huguet A., Meador T. B., Laggoun‐Défarge F., et al., “Production Rates of Bacterial Tetraether Lipids and Fatty Acids in Peatland Under Varying Oxygen Concentrations,” Geochimica et Cosmochimica Acta
203 (2017): 103–116, 10.1016/j.gca.2017.01.012.</cite> [<a href="https://doi.org/10.1016/j.gca.2017.01.012" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Geochimica%20et%20Cosmochimica%20Acta&amp;title=Production%20Rates%20of%20Bacterial%20Tetraether%20Lipids%20and%20Fatty%20Acids%20in%20Peatland%20Under%20Varying%20Oxygen%20Concentrations&amp;author=A.%20Huguet&amp;author=T.%20B.%20Meador&amp;author=F.%20Laggoun%E2%80%90D%C3%A9farge&amp;volume=203&amp;publication_year=2017&amp;pages=103-116&amp;doi=10.1016/j.gca.2017.01.012&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10118-bib-0038">
<span class="label">38.</span><cite>
Rozo L., Lazzari L., Schefuß E., Mulitza S., and Carreira R., “Molecular Distribution and Isotopic Composition (δD and δ<sup>13</sup>C) of Lipids Dataset for a Sediment Core (GeoB16202‐2) From the North‐Eastern Brazilian Margin,” Data in Brief
33 (2020): 106508, 10.1016/j.dib.2020.106508.
</cite> [<a href="https://doi.org/10.1016/j.dib.2020.106508" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7689042/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33294508/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Data%20in%20Brief&amp;title=Molecular%20Distribution%20and%20Isotopic%20Composition%20(%CE%B4D%20and%20%CE%B413C)%20of%20Lipids%20Dataset%20for%20a%20Sediment%20Core%20(GeoB16202%E2%80%902)%20From%20the%20North%E2%80%90Eastern%20Brazilian%20Margin&amp;author=L.%20Rozo&amp;author=L.%20Lazzari&amp;author=E.%20Schefu%C3%9F&amp;author=S.%20Mulitza&amp;author=R.%20Carreira&amp;volume=33&amp;publication_year=2020&amp;pages=106508&amp;pmid=33294508&amp;doi=10.1016/j.dib.2020.106508&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10118-bib-0039">
<span class="label">39.</span><cite>
Heinzelmann S. M., Bale N. J., Villanueva L., et al., “Seasonal Changes in the D/H Ratio of Fatty Acids of Pelagic Microorganisms in the Coastal North Sea,” Biogeosciences
13, no. 19 (2016): 5527–5539, 10.5194/bg-13-5527-2016.</cite> [<a href="https://doi.org/10.5194/bg-13-5527-2016" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biogeosciences&amp;title=Seasonal%20Changes%20in%20the%20D/H%20Ratio%20of%20Fatty%20Acids%20of%20Pelagic%20Microorganisms%20in%20the%20Coastal%20North%20Sea&amp;author=S.%20M.%20Heinzelmann&amp;author=N.%20J.%20Bale&amp;author=L.%20Villanueva&amp;volume=13&amp;issue=19&amp;publication_year=2016&amp;pages=5527-5539&amp;doi=10.5194/bg-13-5527-2016&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10118-bib-0040">
<span class="label">40.</span><cite>
Zhang X., Gillespie A. L., and Sessions A. L., “Large D/H Variations in Bacterial Lipids Reflect Central Metabolic Pathways,” Proceedings of the National Academy of Sciences
106, no. 31 (2009): 12580–12586, 10.1073/pnas.0903030106.</cite> [<a href="https://doi.org/10.1073/pnas.0903030106" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2722351/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19617564/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proceedings%20of%20the%20National%20Academy%20of%20Sciences&amp;title=Large%20D/H%20Variations%20in%20Bacterial%20Lipids%20Reflect%20Central%20Metabolic%20Pathways&amp;author=X.%20Zhang&amp;author=A.%20L.%20Gillespie&amp;author=A.%20L.%20Sessions&amp;volume=106&amp;issue=31&amp;publication_year=2009&amp;pages=12580-12586&amp;pmid=19617564&amp;doi=10.1073/pnas.0903030106&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10118-bib-0041">
<span class="label">41.</span><cite>
Heinzelmann S. M., Villanueva L., Lipsewers Y. A., et al., “Assessing the Metabolism of Sedimentary Microbial Communities Using the Hydrogen Isotopic Composition of Fatty Acids,” Organic Geochemistry
124 (2018): 123–132, 10.1016/j.orggeochem.2018.07.011.</cite> [<a href="https://doi.org/10.1016/j.orggeochem.2018.07.011" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Organic%20Geochemistry&amp;title=Assessing%20the%20Metabolism%20of%20Sedimentary%20Microbial%20Communities%20Using%20the%20Hydrogen%20Isotopic%20Composition%20of%20Fatty%20Acids&amp;author=S.%20M.%20Heinzelmann&amp;author=L.%20Villanueva&amp;author=Y.%20A.%20Lipsewers&amp;volume=124&amp;publication_year=2018&amp;pages=123-132&amp;doi=10.1016/j.orggeochem.2018.07.011&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10118-bib-0042">
<span class="label">42.</span><cite>
Jones A. A., Sessions A. L., Campbell B. J., Li C., and Valentine D. L., “D/H Ratios of Fatty Acids From Marine Particulate Organic Matter in the California Borderland Basins,” Organic Geochemistry
39, no. 5 (2008): 485–500, 10.1016/j.orggeochem.2007.11.001.</cite> [<a href="https://doi.org/10.1016/j.orggeochem.2007.11.001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Organic%20Geochemistry&amp;title=D/H%20Ratios%20of%20Fatty%20Acids%20From%20Marine%20Particulate%20Organic%20Matter%20in%20the%20California%20Borderland%20Basins&amp;author=A.%20A.%20Jones&amp;author=A.%20L.%20Sessions&amp;author=B.%20J.%20Campbell&amp;author=C.%20Li&amp;author=D.%20L.%20Valentine&amp;volume=39&amp;issue=5&amp;publication_year=2008&amp;pages=485-500&amp;doi=10.1016/j.orggeochem.2007.11.001&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10118-bib-0043">
<span class="label">43.</span><cite>
Li C., Sessions A. L., Kinnaman F. S., and Valentine D. L., “Hydrogen‐Isotopic Variability in Lipids From Santa Barbara Basin Sediments,” Geochimica et Cosmochimica Acta
73, no. 16 (2009): 4803–4823, 10.1016/j.gca.2009.05.056.</cite> [<a href="https://doi.org/10.1016/j.gca.2009.05.056" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Geochimica%20et%20Cosmochimica%20Acta&amp;title=Hydrogen%E2%80%90Isotopic%20Variability%20in%20Lipids%20From%20Santa%20Barbara%20Basin%20Sediments&amp;author=C.%20Li&amp;author=A.%20L.%20Sessions&amp;author=F.%20S.%20Kinnaman&amp;author=D.%20L.%20Valentine&amp;volume=73&amp;issue=16&amp;publication_year=2009&amp;pages=4803-4823&amp;doi=10.1016/j.gca.2009.05.056&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10118-bib-0044">
<span class="label">44.</span><cite>
Kellermann M. Y., Yoshinaga M. Y., Wegener G., Krukenberg V., and Hinrichs K. U., “Tracing the Production and Fate of Individual Archaeal Intact Polar Lipids Using Stable Isotope Probing,” Organic Geochemistry
95 (2016): 13–20, 10.1016/j.orggeochem.2016.02.004.</cite> [<a href="https://doi.org/10.1016/j.orggeochem.2016.02.004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Organic%20Geochemistry&amp;title=Tracing%20the%20Production%20and%20Fate%20of%20Individual%20Archaeal%20Intact%20Polar%20Lipids%20Using%20Stable%20Isotope%20Probing&amp;author=M.%20Y.%20Kellermann&amp;author=M.%20Y.%20Yoshinaga&amp;author=G.%20Wegener&amp;author=V.%20Krukenberg&amp;author=K.%20U.%20Hinrichs&amp;volume=95&amp;publication_year=2016&amp;pages=13-20&amp;doi=10.1016/j.orggeochem.2016.02.004&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10118-bib-0045">
<span class="label">45.</span><cite>
Osburn M. R., Sessions A. L., Pepe‐Ranney C., and Spear J. R., “Hydrogen‐Isotopic Variability in Fatty Acids From Yellowstone National Park Hot Spring Microbial Communities,” Geochimica et Cosmochimica Acta
75, no. 17 (2011): 4830–4845, 10.1016/j.gca.2011.05.038.</cite> [<a href="https://doi.org/10.1016/j.gca.2011.05.038" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Geochimica%20et%20Cosmochimica%20Acta&amp;title=Hydrogen%E2%80%90Isotopic%20Variability%20in%20Fatty%20Acids%20From%20Yellowstone%20National%20Park%20Hot%20Spring%20Microbial%20Communities&amp;author=M.%20R.%20Osburn&amp;author=A.%20L.%20Sessions&amp;author=C.%20Pepe%E2%80%90Ranney&amp;author=J.%20R.%20Spear&amp;volume=75&amp;issue=17&amp;publication_year=2011&amp;pages=4830-4845&amp;doi=10.1016/j.gca.2011.05.038&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10118-bib-0046">
<span class="label">46.</span><cite>
Naraoka H., Uehara T., Hanada S., and Kakegawa T., “δ13C–δD Distribution of Lipid Biomarkers in a Bacterial Mat From a Hot Spring in Miyagi Prefecture, NE Japan,” Organic Geochemistry
41, no. 4 (2010): 398–403, 10.1016/j.orggeochem.2009.11.008.</cite> [<a href="https://doi.org/10.1016/j.orggeochem.2009.11.008" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Organic%20Geochemistry&amp;title=%CE%B413C%E2%80%93%CE%B4D%20Distribution%20of%20Lipid%20Biomarkers%20in%20a%20Bacterial%20Mat%20From%20a%20Hot%20Spring%20in%20Miyagi%20Prefecture,%20NE%20Japan&amp;author=H.%20Naraoka&amp;author=T.%20Uehara&amp;author=S.%20Hanada&amp;author=T.%20Kakegawa&amp;volume=41&amp;issue=4&amp;publication_year=2010&amp;pages=398-403&amp;doi=10.1016/j.orggeochem.2009.11.008&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10118-bib-0047">
<span class="label">47.</span><cite>
Dawson K. S., Osburn M. R., Sessions A. L., and Orphan V. J., “Metabolic Associations With Archaea Drive Shifts in Hydrogen Isotope Fractionation in Sulfate‐Reducing Bacterial Lipids in Cocultures and Methane Seeps,” Geobiology
13, no. 5 (2015): 462–477, 10.1111/gbi.12140.
</cite> [<a href="https://doi.org/10.1111/gbi.12140" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25923659/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Geobiology&amp;title=Metabolic%20Associations%20With%20Archaea%20Drive%20Shifts%20in%20Hydrogen%20Isotope%20Fractionation%20in%20Sulfate%E2%80%90Reducing%20Bacterial%20Lipids%20in%20Cocultures%20and%20Methane%20Seeps&amp;author=K.%20S.%20Dawson&amp;author=M.%20R.%20Osburn&amp;author=A.%20L.%20Sessions&amp;author=V.%20J.%20Orphan&amp;volume=13&amp;issue=5&amp;publication_year=2015&amp;pages=462-477&amp;pmid=25923659&amp;doi=10.1111/gbi.12140&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10118-bib-0048">
<span class="label">48.</span><cite>
Wegener G., Bausch M., Holler T., et al., “Assessing Sub‐Seafloor Microbial Activity by Combined Stable Isotope Probing With Deuterated Water and <sup>13</sup>C‐Bicarbonate,” Environmental Microbiology
14, no. 6 (2012): 1517–1527, 10.1111/j.1462-2920.2012.02739.x.
</cite> [<a href="https://doi.org/10.1111/j.1462-2920.2012.02739.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22498240/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Environmental%20Microbiology&amp;title=Assessing%20Sub%E2%80%90Seafloor%20Microbial%20Activity%20by%20Combined%20Stable%20Isotope%20Probing%20With%20Deuterated%20Water%20and%2013C%E2%80%90Bicarbonate&amp;author=G.%20Wegener&amp;author=M.%20Bausch&amp;author=T.%20Holler&amp;volume=14&amp;issue=6&amp;publication_year=2012&amp;pages=1517-1527&amp;pmid=22498240&amp;doi=10.1111/j.1462-2920.2012.02739.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10118-bib-0049">
<span class="label">49.</span><cite>
Leavitt W. D., Kopf S. H., Weber Y., et al., “Controls on the Hydrogen Isotope Composition of Tetraether Lipids in an Autotrophic Ammonia‐Oxidizing Marine Archaeon,” Geochimica et Cosmochimica Acta
352 (2023): 194–210, 10.1016/j.gca.2023.04.033.</cite> [<a href="https://doi.org/10.1016/j.gca.2023.04.033" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Geochimica%20et%20Cosmochimica%20Acta&amp;title=Controls%20on%20the%20Hydrogen%20Isotope%20Composition%20of%20Tetraether%20Lipids%20in%20an%20Autotrophic%20Ammonia%E2%80%90Oxidizing%20Marine%20Archaeon&amp;author=W.%20D.%20Leavitt&amp;author=S.%20H.%20Kopf&amp;author=Y.%20Weber&amp;volume=352&amp;publication_year=2023&amp;pages=194-210&amp;doi=10.1016/j.gca.2023.04.033&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10118-bib-0050">
<span class="label">50.</span><cite>
Wu W., Meador T. B., Könneke M., Elvert M., Wegener G., and Hinrichs K. U., “Substrate‐Dependent Incorporation of Carbon and Hydrogen for Lipid Biosynthesis by <em>Methanosarcina barkeri</em>
,” Environmental Microbiology Reports
12, no. 5 (2020): 555–567, 10.1111/1758-2229.12876.
</cite> [<a href="https://doi.org/10.1111/1758-2229.12876" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32783290/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Environmental%20Microbiology%20Reports&amp;title=Substrate%E2%80%90Dependent%20Incorporation%20of%20Carbon%20and%20Hydrogen%20for%20Lipid%20Biosynthesis%20by%20Methanosarcina%20barkeri&amp;author=W.%20Wu&amp;author=T.%20B.%20Meador&amp;author=M.%20K%C3%B6nneke&amp;author=M.%20Elvert&amp;author=G.%20Wegener&amp;volume=12&amp;issue=5&amp;publication_year=2020&amp;pages=555-567&amp;pmid=32783290&amp;doi=10.1111/1758-2229.12876&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10118-bib-0051">
<span class="label">51.</span><cite>
Osburn M. R., Dawson K. S., Fogel M. L., and Sessions A. L., “Fractionation of Hydrogen Isotopes by Sulfate‐ and Nitrate‐Reducing Bacteria,” Frontiers in Microbiology
7 (2016): 7, 10.3389/fmicb.2016.01166.
</cite> [<a href="https://doi.org/10.3389/fmicb.2016.01166" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4969296/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27531993/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Microbiology&amp;title=Fractionation%20of%20Hydrogen%20Isotopes%20by%20Sulfate%E2%80%90%20and%20Nitrate%E2%80%90Reducing%20Bacteria&amp;author=M.%20R.%20Osburn&amp;author=K.%20S.%20Dawson&amp;author=M.%20L.%20Fogel&amp;author=A.%20L.%20Sessions&amp;volume=7&amp;publication_year=2016&amp;pages=7&amp;pmid=27531993&amp;doi=10.3389/fmicb.2016.01166&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10118-bib-0052">
<span class="label">52.</span><cite>
Svensson E., Schouten S., Hopmans E. C., Middelburg J. J., and Sinninghe Damsté J. S., “Factors Controlling the Stable Nitrogen Isotopic Composition (δ15N) of Lipids in Marine Animals. Savage C, ed,” PLoS ONE
11, no. 1 (2016): e0146321, 10.1371/journal.pone.0146321.
</cite> [<a href="https://doi.org/10.1371/journal.pone.0146321" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4701503/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26731720/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20ONE&amp;title=Factors%20Controlling%20the%20Stable%20Nitrogen%20Isotopic%20Composition%20(%CE%B415N)%20of%20Lipids%20in%20Marine%20Animals.%20Savage%20C,%20ed&amp;author=E.%20Svensson&amp;author=S.%20Schouten&amp;author=E.%20C.%20Hopmans&amp;author=J.%20J.%20Middelburg&amp;author=J.%20S.%20Sinninghe%20Damst%C3%A9&amp;volume=11&amp;issue=1&amp;publication_year=2016&amp;pages=e0146321&amp;pmid=26731720&amp;doi=10.1371/journal.pone.0146321&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10118-bib-0053">
<span class="label">53.</span><cite>
Fogel M. L., Wooller M. J., Cheeseman J., et al., “Unusually Negative Nitrogen Isotopic Compositions (δ<sup>15</sup>N) of Mangroves and Lichens in an Oligotrophic, Microbially‐Influenced Ecosystem,” Biogeosciences
5, no. 6 (2008): 1693–1704, 10.5194/bg-5-1693-2008.</cite> [<a href="https://doi.org/10.5194/bg-5-1693-2008" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biogeosciences&amp;title=Unusually%20Negative%20Nitrogen%20Isotopic%20Compositions%20(%CE%B415N)%20of%20Mangroves%20and%20Lichens%20in%20an%20Oligotrophic,%20Microbially%E2%80%90Influenced%20Ecosystem&amp;author=M.%20L.%20Fogel&amp;author=M.%20J.%20Wooller&amp;author=J.%20Cheeseman&amp;volume=5&amp;issue=6&amp;publication_year=2008&amp;pages=1693-1704&amp;doi=10.5194/bg-5-1693-2008&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10118-bib-0054">
<span class="label">54.</span><cite>
Chikaraishi Y., Ogawa N. O., Kashiyama Y., et al., “Determination of Aquatic Food‐Web Structure Based on Compound‐Specific Nitrogen Isotopic Composition of Amino Acids,” Limnology and Oceanography: Methods
7, no. 11 (2009): 740–750, 10.4319/lom.2009.7.740.</cite> [<a href="https://doi.org/10.4319/lom.2009.7.740" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Limnology%20and%20Oceanography:%20Methods&amp;title=Determination%20of%20Aquatic%20Food%E2%80%90Web%20Structure%20Based%20on%20Compound%E2%80%90Specific%20Nitrogen%20Isotopic%20Composition%20of%20Amino%20Acids&amp;author=Y.%20Chikaraishi&amp;author=N.%20O.%20Ogawa&amp;author=Y.%20Kashiyama&amp;volume=7&amp;issue=11&amp;publication_year=2009&amp;pages=740-750&amp;doi=10.4319/lom.2009.7.740&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10118-bib-0055">
<span class="label">55.</span><cite>
Chikaraishi Y., Kashiyama Y., Ogawa N., Kitazato H., and Ohkouchi N., “Metabolic Control of Nitrogen Isotope Composition of Amino Acids in Macroalgae and Gastropods: Implications for Aquatic Food Web Studies,” Marine Ecology Progress Series
342 (2007): 85–90, 10.3354/meps342085.</cite> [<a href="https://doi.org/10.3354/meps342085" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Marine%20Ecology%20Progress%20Series&amp;title=Metabolic%20Control%20of%20Nitrogen%20Isotope%20Composition%20of%20Amino%20Acids%20in%20Macroalgae%20and%20Gastropods:%20Implications%20for%20Aquatic%20Food%20Web%20Studies&amp;author=Y.%20Chikaraishi&amp;author=Y.%20Kashiyama&amp;author=N.%20Ogawa&amp;author=H.%20Kitazato&amp;author=N.%20Ohkouchi&amp;volume=342&amp;publication_year=2007&amp;pages=85-90&amp;doi=10.3354/meps342085&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10118-bib-0056">
<span class="label">56.</span><cite>
Calleja M. L., Batista F., Peacock M., Kudela R., and McCarthy M. D., “Changes in Compound Specific δ<sup>15</sup>N Amino Acid Signatures and d/l Ratios in Marine Dissolved Organic Matter Induced by Heterotrophic Bacterial Reworking,” Marine Chemistry
149 (2013): 32–44, 10.1016/j.marchem.2012.12.001.</cite> [<a href="https://doi.org/10.1016/j.marchem.2012.12.001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Marine%20Chemistry&amp;title=Changes%20in%20Compound%20Specific%20%CE%B415N%20Amino%20Acid%20Signatures%20and%20d/l%20Ratios%20in%20Marine%20Dissolved%20Organic%20Matter%20Induced%20by%20Heterotrophic%20Bacterial%20Reworking&amp;author=M.%20L.%20Calleja&amp;author=F.%20Batista&amp;author=M.%20Peacock&amp;author=R.%20Kudela&amp;author=M.%20D.%20McCarthy&amp;volume=149&amp;publication_year=2013&amp;pages=32-44&amp;doi=10.1016/j.marchem.2012.12.001&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10118-bib-0057">
<span class="label">57.</span><cite>
Svensson E., Schouten S., Stam A., Middelburg J. J., and Sinninghe Damsté J. S., “Compound‐Specific Stable Isotope Analysis of Nitrogen‐Containing Intact Polar Lipids: Compound‐Specific δ<sup>15</sup>N Determination of Nitrogen‐Containing Lipids,” Rapid Communications in Mass Spectrometry
29, no. 23 (2015): 2263–2271, 10.1002/rcm.7393.
</cite> [<a href="https://doi.org/10.1002/rcm.7393" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26522319/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Rapid%20Communications%20in%20Mass%20Spectrometry&amp;title=Compound%E2%80%90Specific%20Stable%20Isotope%20Analysis%20of%20Nitrogen%E2%80%90Containing%20Intact%20Polar%20Lipids:%20Compound%E2%80%90Specific%20%CE%B415N%20Determination%20of%20Nitrogen%E2%80%90Containing%20Lipids&amp;author=E.%20Svensson&amp;author=S.%20Schouten&amp;author=A.%20Stam&amp;author=J.%20J.%20Middelburg&amp;author=J.%20S.%20Sinninghe%20Damst%C3%A9&amp;volume=29&amp;issue=23&amp;publication_year=2015&amp;pages=2263-2271&amp;pmid=26522319&amp;doi=10.1002/rcm.7393&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10118-bib-0058">
<span class="label">58.</span><cite>
Owens N. J. P., “Natural Variations in 15N in the Marine Environment,” in Advances in Marine Biology, vol. 24 (Elsevier, 1988): 389–451, 10.1016/S0065-2881(08)60077-2.</cite> [<a href="https://doi.org/10.1016/S0065-2881(08)60077-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?title=Advances%20in%20Marine%20Biology&amp;author=N.%20J.%20P.%20Owens&amp;publication_year=1988&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10118-bib-0059">
<span class="label">59.</span><cite>
Teske A., Hinrichs K. U., Edgcomb V., et al., “Microbial Diversity of Hydrothermal Sediments in the Guaymas Basin: Evidence for Anaerobic Methanotrophic Communities,” Applied and Environmental Microbiology
68, no. 4 (2002): 1994–2007, 10.1128/AEM.68.4.1994-2007.2002.
</cite> [<a href="https://doi.org/10.1128/AEM.68.4.1994-2007.2002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC123873/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11916723/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Applied%20and%20Environmental%20Microbiology&amp;title=Microbial%20Diversity%20of%20Hydrothermal%20Sediments%20in%20the%20Guaymas%20Basin:%20Evidence%20for%20Anaerobic%20Methanotrophic%20Communities&amp;author=A.%20Teske&amp;author=K.%20U.%20Hinrichs&amp;author=V.%20Edgcomb&amp;volume=68&amp;issue=4&amp;publication_year=2002&amp;pages=1994-2007&amp;pmid=11916723&amp;doi=10.1128/AEM.68.4.1994-2007.2002&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10118-bib-0060">
<span class="label">60.</span><cite>
Biddle J. F., Cardman Z., Mendlovitz H., et al., “Anaerobic Oxidation of Methane at Different Temperature Regimes in Guaymas Basin hydrothermal Sediments,” ISME Journal
6, no. 5 (2012): 1018–1031, 10.1038/ismej.2011.164.
</cite> [<a href="https://doi.org/10.1038/ismej.2011.164" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3329104/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22094346/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=ISME%20Journal&amp;title=Anaerobic%20Oxidation%20of%20Methane%20at%20Different%20Temperature%20Regimes%20in%20Guaymas%20Basin%20hydrothermal%20Sediments&amp;author=J.%20F.%20Biddle&amp;author=Z.%20Cardman&amp;author=H.%20Mendlovitz&amp;volume=6&amp;issue=5&amp;publication_year=2012&amp;pages=1018-1031&amp;pmid=22094346&amp;doi=10.1038/ismej.2011.164&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10118-bib-0061">
<span class="label">61.</span><cite>
Dowell F., Cardman Z., Dasarathy S., et al., “Microbial Communities in Methane‐ and Short Chain Alkane‐Rich Hydrothermal Sediments of Guaymas Basin,” Frontiers in Microbiology
7 (2016): 17, 10.3389/fmicb.2016.00017.
</cite> [<a href="https://doi.org/10.3389/fmicb.2016.00017" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4731509/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26858698/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Microbiology&amp;title=Microbial%20Communities%20in%20Methane%E2%80%90%20and%20Short%20Chain%20Alkane%E2%80%90Rich%20Hydrothermal%20Sediments%20of%20Guaymas%20Basin&amp;author=F.%20Dowell&amp;author=Z.%20Cardman&amp;author=S.%20Dasarathy&amp;volume=7&amp;publication_year=2016&amp;pages=17&amp;pmid=26858698&amp;doi=10.3389/fmicb.2016.00017&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10118-bib-0062">
<span class="label">62.</span><cite>
Ramírez G. A., Mara P., Sehein T., et al., “Environmental Factors Shaping Bacterial, Archaeal and Fungal Community Structure in Hydrothermal Sediments of Guaymas Basin, Gulf of California,” PLoS ONE
16, no. 9 (2021): e0256321, 10.1371/journal.pone.0256321.
</cite> [<a href="https://doi.org/10.1371/journal.pone.0256321" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8425543/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34495995/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20ONE&amp;title=Environmental%20Factors%20Shaping%20Bacterial,%20Archaeal%20and%20Fungal%20Community%20Structure%20in%20Hydrothermal%20Sediments%20of%20Guaymas%20Basin,%20Gulf%20of%20California&amp;author=G.%20A.%20Ram%C3%ADrez&amp;author=P.%20Mara&amp;author=T.%20Sehein&amp;volume=16&amp;issue=9&amp;publication_year=2021&amp;pages=e0256321&amp;pmid=34495995&amp;doi=10.1371/journal.pone.0256321&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10118-bib-0063">
<span class="label">63.</span><cite>
Engelen B., Nguyen T., Heyerhoff B., et al., “Microbial Communities of Hydrothermal Guaymas Basin Surficial Sediment Profiled at 2 Millimeter‐Scale‐Resolution,” Frontiers in Microbiology
12 (2021): 710881, 10.3389/fmicb.2021.710881.
</cite> [<a href="https://doi.org/10.3389/fmicb.2021.710881" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8322767/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34335545/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Microbiology&amp;title=Microbial%20Communities%20of%20Hydrothermal%20Guaymas%20Basin%20Surficial%20Sediment%20Profiled%20at%202%20Millimeter%E2%80%90Scale%E2%80%90Resolution&amp;author=B.%20Engelen&amp;author=T.%20Nguyen&amp;author=B.%20Heyerhoff&amp;volume=12&amp;publication_year=2021&amp;pages=710881&amp;pmid=34335545&amp;doi=10.3389/fmicb.2021.710881&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10118-bib-0064">
<span class="label">64.</span><cite>
Teske A., “Guaymas Basin, a Hydrothermal Hydrocarbon Seep Ecosystem,” in Marine Hydrocarbon Seeps, eds. Teske A. and Carvalho V. (Springer Oceanography. Springer International Publishing, 2020): 43–68, 10.1007/978-3-030-34827-4_3.</cite> [<a href="https://doi.org/10.1007/978-3-030-34827-4_3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?title=Marine%20Hydrocarbon%20Seeps&amp;author=A.%20Teske&amp;publication_year=2020&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10118-bib-0065">
<span class="label">65.</span><cite>
Song M., Schubotz F., Kellermann M. Y., et al., “Formation of Ethane and Propane via Abiotic Reductive Conversion of Acetic Acid in Hydrothermal Sediments,” Proceedings of the National Academy of Sciences
118, no. 47 (2021): e2005219118, 10.1073/pnas.2005219118.</cite> [<a href="https://doi.org/10.1073/pnas.2005219118" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8617489/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34782456/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proceedings%20of%20the%20National%20Academy%20of%20Sciences&amp;title=Formation%20of%20Ethane%20and%20Propane%20via%20Abiotic%20Reductive%20Conversion%20of%20Acetic%20Acid%20in%20Hydrothermal%20Sediments&amp;author=M.%20Song&amp;author=F.%20Schubotz&amp;author=M.%20Y.%20Kellermann&amp;volume=118&amp;issue=47&amp;publication_year=2021&amp;pages=e2005219118&amp;pmid=34782456&amp;doi=10.1073/pnas.2005219118&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10118-bib-0066">
<span class="label">66.</span><cite>
Teske A., Wegener G., Chanton J. P., et al., “Microbial Communities Under Distinct Thermal and Geochemical Regimes in Axial and Off‐Axis Sediments of Guaymas Basin,” Frontiers in Microbiology
12 (2021): 633649, 10.3389/fmicb.2021.633649.
</cite> [<a href="https://doi.org/10.3389/fmicb.2021.633649" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7906980/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33643265/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Microbiology&amp;title=Microbial%20Communities%20Under%20Distinct%20Thermal%20and%20Geochemical%20Regimes%20in%20Axial%20and%20Off%E2%80%90Axis%20Sediments%20of%20Guaymas%20Basin&amp;author=A.%20Teske&amp;author=G.%20Wegener&amp;author=J.%20P.%20Chanton&amp;volume=12&amp;publication_year=2021&amp;pages=633649&amp;pmid=33643265&amp;doi=10.3389/fmicb.2021.633649&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10118-bib-0067">
<span class="label">67.</span><cite>
Hinrichs K. U. and Boetius A., “The Anaerobic Oxidation of Methane: New Insights in Microbial Ecology and Biogeochemistry,” in Ocean Margin Systems, eds. Wefer G., Billett D., Hebbeln D., Jørgensen B. B., Schlüter M., and Van Weering T. C. E. (Springer Berlin Heidelberg, 2002): 457–477, 10.1007/978-3-662-05127-6_28.</cite> [<a href="https://doi.org/10.1007/978-3-662-05127-6_28" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?title=Ocean%20Margin%20Systems&amp;author=K.%20U.%20Hinrichs&amp;author=A.%20Boetius&amp;publication_year=2002&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10118-bib-0068">
<span class="label">68.</span><cite>
Song M., Gaseous Hydrocarbon Cycling and Lipid Biogeochemistry in Cold and Hot Seep Sediments (Universität Bremen, 2020), 10.26092/ELIB/215.</cite> [<a href="https://doi.org/10.26092/ELIB/215" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?title=Gaseous%20Hydrocarbon%20Cycling%20and%20Lipid%20Biogeochemistry%20in%20Cold%20and%20Hot%20Seep%20Sediments&amp;author=M.%20Song&amp;publication_year=2020&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10118-bib-0069">
<span class="label">69.</span><cite>
Bale N. J., Airs R. L., Martin P., Lampitt R. S., and Llewellyn C. A., “Chlorophyll‐<em>a</em> Transformations Associated With Sinking Diatoms During Termination of a North Atlantic Spring Bloom,” Marine Chemistry
172 (2015): 23–33, 10.1016/j.marchem.2015.03.005.</cite> [<a href="https://doi.org/10.1016/j.marchem.2015.03.005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Marine%20Chemistry&amp;title=Chlorophyll%E2%80%90a%20Transformations%20Associated%20With%20Sinking%20Diatoms%20During%20Termination%20of%20a%20North%20Atlantic%20Spring%20Bloom&amp;author=N.%20J.%20Bale&amp;author=R.%20L.%20Airs&amp;author=P.%20Martin&amp;author=R.%20S.%20Lampitt&amp;author=C.%20A.%20Llewellyn&amp;volume=172&amp;publication_year=2015&amp;pages=23-33&amp;doi=10.1016/j.marchem.2015.03.005&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10118-bib-0070">
<span class="label">70.</span><cite>
Bale N. J., Airs R. L., and Llewellyn C. A., “Type I and Type II Chlorophyll‐<em>a</em> Transformation Products Associated With Algal Senescence,” Organic Geochemistry
42, no. 5 (2011): 451–464, 10.1016/j.orggeochem.2011.03.016.</cite> [<a href="https://doi.org/10.1016/j.orggeochem.2011.03.016" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Organic%20Geochemistry&amp;title=Type%20I%20and%20Type%20II%20Chlorophyll%E2%80%90a%20Transformation%20Products%20Associated%20With%20Algal%20Senescence&amp;author=N.%20J.%20Bale&amp;author=R.%20L.%20Airs&amp;author=C.%20A.%20Llewellyn&amp;volume=42&amp;issue=5&amp;publication_year=2011&amp;pages=451-464&amp;doi=10.1016/j.orggeochem.2011.03.016&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10118-bib-0071">
<span class="label">71.</span><cite>
Szymczak‐Żyła M., Kowalewska G., and Louda J. W., “Chlorophyll‐<em>a</em> and Derivatives in Recent Sediments as Indicators of Productivity and Depositional Conditions,” Marine Chemistry
125, no. 1 (2011): 39–48, 10.1016/j.marchem.2011.02.002.</cite> [<a href="https://doi.org/10.1016/j.marchem.2011.02.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Marine%20Chemistry&amp;title=Chlorophyll%E2%80%90a%20and%20Derivatives%20in%20Recent%20Sediments%20as%20Indicators%20of%20Productivity%20and%20Depositional%20Conditions&amp;author=M.%20Szymczak%E2%80%90%C5%BBy%C5%82a&amp;author=G.%20Kowalewska&amp;author=J.%20W.%20Louda&amp;volume=125&amp;issue=1&amp;publication_year=2011&amp;pages=39-48&amp;doi=10.1016/j.marchem.2011.02.002&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10118-bib-0072">
<span class="label">72.</span><cite>
Szymczak‐Żyla M., Kowalewska G., and Louda J. W., “The Influence of Microorganisms on Chlorophyll a Degradation in the Marine Environment,” Limnology and Oceanography
53, no. 2 (2008): 851–862, 10.4319/lo.2008.53.2.0851.</cite> [<a href="https://doi.org/10.4319/lo.2008.53.2.0851" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Limnology%20and%20Oceanography&amp;title=The%20Influence%20of%20Microorganisms%20on%20Chlorophyll%20a%20Degradation%20in%20the%20Marine%20Environment&amp;author=M.%20Szymczak%E2%80%90%C5%BByla&amp;author=G.%20Kowalewska&amp;author=J.%20W.%20Louda&amp;volume=53&amp;issue=2&amp;publication_year=2008&amp;pages=851-862&amp;doi=10.4319/lo.2008.53.2.0851&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10118-bib-0073">
<span class="label">73.</span><cite>
Mamode Cassim A., Grison M., Ito Y., Simon‐Plas F., Mongrand S., and Boutté Y., “Sphingolipids in Plants: A Guidebook on Their Function in Membrane Architecture, Cellular Processes, and Environmental or Developmental Responses,” FEBS Letters
594, no. 22 (2020): 3719–3738, 10.1002/1873-3468.13987.
</cite> [<a href="https://doi.org/10.1002/1873-3468.13987" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33151562/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=FEBS%20Letters&amp;title=Sphingolipids%20in%20Plants:%20A%20Guidebook%20on%20Their%20Function%20in%20Membrane%20Architecture,%20Cellular%20Processes,%20and%20Environmental%20or%20Developmental%20Responses&amp;author=A.%20Mamode%20Cassim&amp;author=M.%20Grison&amp;author=Y.%20Ito&amp;author=F.%20Simon%E2%80%90Plas&amp;author=S.%20Mongrand&amp;volume=594&amp;issue=22&amp;publication_year=2020&amp;pages=3719-3738&amp;pmid=33151562&amp;doi=10.1002/1873-3468.13987&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10118-bib-0074">
<span class="label">74.</span><cite>
Harrison P. J., Dunn T. M., and Campopiano D. J., “Sphingolipid Biosynthesis in Man and Microbes,” Natural Product Reports
35, no. 9 (2018): 921–954, 10.1039/C8NP00019K.
</cite> [<a href="https://doi.org/10.1039/C8NP00019K" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6148460/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29863195/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Natural%20Product%20Reports&amp;title=Sphingolipid%20Biosynthesis%20in%20Man%20and%20Microbes&amp;author=P.%20J.%20Harrison&amp;author=T.%20M.%20Dunn&amp;author=D.%20J.%20Campopiano&amp;volume=35&amp;issue=9&amp;publication_year=2018&amp;pages=921-954&amp;pmid=29863195&amp;doi=10.1039/C8NP00019K&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10118-bib-0075">
<span class="label">75.</span><cite>
Kawahara K. and Yokota A., “Chemical Characterization of Ceramides Isolated From a Gliding Marine Bacterium 
<em>Aureispira marina</em>

,” Bioscience, Biotechnology, and Biochemistry
86, no. 8 (2022): 1136–1143, 10.1093/bbb/zbac086.
</cite> [<a href="https://doi.org/10.1093/bbb/zbac086" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35648458/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioscience,%20Biotechnology,%20and%20Biochemistry&amp;title=Chemical%20Characterization%20of%20Ceramides%20Isolated%20From%20a%20Gliding%20Marine%20Bacterium%20Aureispira%20marina&amp;author=K.%20Kawahara&amp;author=A.%20Yokota&amp;volume=86&amp;issue=8&amp;publication_year=2022&amp;pages=1136-1143&amp;pmid=35648458&amp;doi=10.1093/bbb/zbac086&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10118-bib-0076">
<span class="label">76.</span><cite>
Ding S., Von Meijenfeldt F. A. B., Bale N. J., Sinninghe Damsté J. S., and Villanueva L., “Production of Structurally Diverse Sphingolipids by Anaerobic Marine Bacteria in the Euxinic Black Sea Water Column,” ISME Journal
18, no. 1 (2024): wrae153, 10.1093/ismejo/wrae153.
</cite> [<a href="https://doi.org/10.1093/ismejo/wrae153" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11334938/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39113610/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=ISME%20Journal&amp;title=Production%20of%20Structurally%20Diverse%20Sphingolipids%20by%20Anaerobic%20Marine%20Bacteria%20in%20the%20Euxinic%20Black%20Sea%20Water%20Column&amp;author=S.%20Ding&amp;author=F.%20A.%20B.%20Von%20Meijenfeldt&amp;author=N.%20J.%20Bale&amp;author=J.%20S.%20Sinninghe%20Damst%C3%A9&amp;author=L.%20Villanueva&amp;volume=18&amp;issue=1&amp;publication_year=2024&amp;pages=wrae153&amp;pmid=39113610&amp;doi=10.1093/ismejo/wrae153&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10118-bib-0077">
<span class="label">77.</span><cite>
Alvarez H. and Steinbüchel A., “Triacylglycerols in Prokaryotic Microorganisms,” Applied Microbiology and Biotechnology
60 (2002): 367–376, 10.1007/s00253-002-1135-0.
</cite> [<a href="https://doi.org/10.1007/s00253-002-1135-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12466875/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Applied%20Microbiology%20and%20Biotechnology&amp;title=Triacylglycerols%20in%20Prokaryotic%20Microorganisms&amp;author=H.%20Alvarez&amp;author=A.%20Steinb%C3%BCchel&amp;volume=60&amp;publication_year=2002&amp;pages=367-376&amp;pmid=12466875&amp;doi=10.1007/s00253-002-1135-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10118-bib-0078">
<span class="label">78.</span><cite>
Sohlenkamp C., López‐Lara I. M., and Geiger O., “Biosynthesis of Phosphatidylcholine in Bacteria,” Progress in Lipid Research
42, no. 2 (2003): 115–162, 10.1016/S0163-7827(02)00050-4.
</cite> [<a href="https://doi.org/10.1016/S0163-7827(02)00050-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12547654/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Progress%20in%20Lipid%20Research&amp;title=Biosynthesis%20of%20Phosphatidylcholine%20in%20Bacteria&amp;author=C.%20Sohlenkamp&amp;author=I.%20M.%20L%C3%B3pez%E2%80%90Lara&amp;author=O.%20Geiger&amp;volume=42&amp;issue=2&amp;publication_year=2003&amp;pages=115-162&amp;pmid=12547654&amp;doi=10.1016/S0163-7827(02)00050-4&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10118-bib-0079">
<span class="label">79.</span><cite>
Schubotz F., Wakeham S. G., Lipp J. S., Fredricks H. F., and Hinrichs K., “Detection of Microbial Biomass by Intact Polar Membrane Lipid Analysis in the Water Column and Surface sediments of the Black Sea,” Environmental Microbiology
11, no. 10 (2009): 2720–2734, 10.1111/j.1462-2920.2009.01999.x.
</cite> [<a href="https://doi.org/10.1111/j.1462-2920.2009.01999.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19624710/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Environmental%20Microbiology&amp;title=Detection%20of%20Microbial%20Biomass%20by%20Intact%20Polar%20Membrane%20Lipid%20Analysis%20in%20the%20Water%20Column%20and%20Surface%20sediments%20of%20the%20Black%20Sea&amp;author=F.%20Schubotz&amp;author=S.%20G.%20Wakeham&amp;author=J.%20S.%20Lipp&amp;author=H.%20F.%20Fredricks&amp;author=K.%20Hinrichs&amp;volume=11&amp;issue=10&amp;publication_year=2009&amp;pages=2720-2734&amp;pmid=19624710&amp;doi=10.1111/j.1462-2920.2009.01999.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10118-bib-0080">
<span class="label">80.</span><cite>
Van Mooy B. A. S. and Fredricks H. F., “Bacterial and Eukaryotic Intact Polar Lipids in the Eastern Subtropical South Pacific: Water‐Column Distribution, Planktonic Sources, and Fatty Acid Composition,” Geochimica et Cosmochimica Acta
74, no. 22 (2010): 6499–6516, 10.1016/j.gca.2010.08.026.</cite> [<a href="https://doi.org/10.1016/j.gca.2010.08.026" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Geochimica%20et%20Cosmochimica%20Acta&amp;title=Bacterial%20and%20Eukaryotic%20Intact%20Polar%20Lipids%20in%20the%20Eastern%20Subtropical%20South%20Pacific:%20Water%E2%80%90Column%20Distribution,%20Planktonic%20Sources,%20and%20Fatty%20Acid%20Composition&amp;author=B.%20A.%20S.%20Van%20Mooy&amp;author=H.%20F.%20Fredricks&amp;volume=74&amp;issue=22&amp;publication_year=2010&amp;pages=6499-6516&amp;doi=10.1016/j.gca.2010.08.026&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10118-bib-0081">
<span class="label">81.</span><cite>
Nowicka B. and Kruk J., “Occurrence, Biosynthesis and Function of Isoprenoid Quinones,” Biochimica et Biophysica Acta BBA ‐ Bioenergetic
1797, no. 9 (2010): 1587–1605, 10.1016/j.bbabio.2010.06.007.</cite> [<a href="https://doi.org/10.1016/j.bbabio.2010.06.007" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20599680/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biochimica%20et%20Biophysica%20Acta%20BBA%20%E2%80%90%20Bioenergetic&amp;title=Occurrence,%20Biosynthesis%20and%20Function%20of%20Isoprenoid%20Quinones&amp;author=B.%20Nowicka&amp;author=J.%20Kruk&amp;volume=1797&amp;issue=9&amp;publication_year=2010&amp;pages=1587-1605&amp;pmid=20599680&amp;doi=10.1016/j.bbabio.2010.06.007&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10118-bib-0082">
<span class="label">82.</span><cite>
Becker K. W., Elling F. J., Schröder J. M., et al., “Isoprenoid Quinones Resolve the Stratification of Redox Processes in a Biogeochemical Continuum From the Photic Zone to Deep Anoxic Sediments of the Black Sea,” Applied and Environmental Microbiology
84, no. 10 (2018): e02736‐17, 10.1128/AEM.02736-17.
</cite> [<a href="https://doi.org/10.1128/AEM.02736-17" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5930376/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29523543/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Applied%20and%20Environmental%20Microbiology&amp;title=Isoprenoid%20Quinones%20Resolve%20the%20Stratification%20of%20Redox%20Processes%20in%20a%20Biogeochemical%20Continuum%20From%20the%20Photic%20Zone%20to%20Deep%20Anoxic%20Sediments%20of%20the%20Black%20Sea&amp;author=K.%20W.%20Becker&amp;author=F.%20J.%20Elling&amp;author=J.%20M.%20Schr%C3%B6der&amp;volume=84&amp;issue=10&amp;publication_year=2018&amp;pages=e02736%E2%80%9017&amp;pmid=29523543&amp;doi=10.1128/AEM.02736-17&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10118-bib-0083">
<span class="label">83.</span><cite>
Elling F. J., Pierrel F., Chobert S. C., et al., “A Novel Quinone Biosynthetic Pathway Illuminates the Evolution of Aerobic Metabolism,” Proceedings of the National Academy of Sciences
122, no. 8 (2025): e2421994122, 10.1073/pnas.2421994122.</cite> [<a href="https://doi.org/10.1073/pnas.2421994122" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11874023/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39977315/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proceedings%20of%20the%20National%20Academy%20of%20Sciences&amp;title=A%20Novel%20Quinone%20Biosynthetic%20Pathway%20Illuminates%20the%20Evolution%20of%20Aerobic%20Metabolism&amp;author=F.%20J.%20Elling&amp;author=F.%20Pierrel&amp;author=S.%20C.%20Chobert&amp;volume=122&amp;issue=8&amp;publication_year=2025&amp;pages=e2421994122&amp;pmid=39977315&amp;doi=10.1073/pnas.2421994122&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10118-bib-0084">
<span class="label">84.</span><cite>
Collins M. D. and Green P. N., “Isolation and Characterization of a Novel Coenzyme Q From Some Methane‐Oxidizing Bacteria,” Biochemical and Biophysical Research Communications
133, no. 3 (1985): 1125–1131, 10.1016/0006-291X(85)91253-7.
</cite> [<a href="https://doi.org/10.1016/0006-291X(85)91253-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/3936502/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biochemical%20and%20Biophysical%20Research%20Communications&amp;title=Isolation%20and%20Characterization%20of%20a%20Novel%20Coenzyme%20Q%20From%20Some%20Methane%E2%80%90Oxidizing%20Bacteria&amp;author=M.%20D.%20Collins&amp;author=P.%20N.%20Green&amp;volume=133&amp;issue=3&amp;publication_year=1985&amp;pages=1125-1131&amp;pmid=3936502&amp;doi=10.1016/0006-291X(85)91253-7&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10118-bib-0085">
<span class="label">85.</span><cite>
Teske A., De Beer D., McKay L. J., et al., “The Guaymas Basin Hiking Guide to Hydrothermal Mounds, Chimneys, and Microbial Mats: Complex Seafloor Expressions of Subsurface Hydrothermal Circulation,” Frontiers in Microbiology
7 (2016): 75, 10.3389/fmicb.2016.00075.
</cite> [<a href="https://doi.org/10.3389/fmicb.2016.00075" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4757712/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26925032/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Microbiology&amp;title=The%20Guaymas%20Basin%20Hiking%20Guide%20to%20Hydrothermal%20Mounds,%20Chimneys,%20and%20Microbial%20Mats:%20Complex%20Seafloor%20Expressions%20of%20Subsurface%20Hydrothermal%20Circulation&amp;author=A.%20Teske&amp;author=D.%20De%20Beer&amp;author=L.%20J.%20McKay&amp;volume=7&amp;publication_year=2016&amp;pages=75&amp;pmid=26925032&amp;doi=10.3389/fmicb.2016.00075&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10118-bib-0086">
<span class="label">86.</span><cite>
Taipale S. J., Peltomaa E., Hiltunen M., et al., “Inferring Phytoplankton, Terrestrial Plant and Bacteria Bulk δ<sup>13</sup>C Values From Compound Specific Analyses of Lipids and Fatty Acids,” PLoS ONE
10, no. 7 (2015): e0133974, 10.1371/journal.pone.0133974.
</cite> [<a href="https://doi.org/10.1371/journal.pone.0133974" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4514774/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26208114/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20ONE&amp;title=Inferring%20Phytoplankton,%20Terrestrial%20Plant%20and%20Bacteria%20Bulk%20%CE%B413C%20Values%20From%20Compound%20Specific%20Analyses%20of%20Lipids%20and%20Fatty%20Acids&amp;author=S.%20J.%20Taipale&amp;author=E.%20Peltomaa&amp;author=M.%20Hiltunen&amp;volume=10&amp;issue=7&amp;publication_year=2015&amp;pages=e0133974&amp;pmid=26208114&amp;doi=10.1371/journal.pone.0133974&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10118-bib-0087">
<span class="label">87.</span><cite>
Henderson L. C., Wittmers F., Carlson C. A., Worden A. Z., and Close H. G., “Variable Carbon Isotope Fractionation of Photosynthetic Communities Over Depth in an Open‐Ocean Euphotic Zone,” Proceedings of the National Academy of Sciences
121, no. 10 (2024): e2304613121, 10.1073/pnas.2304613121.</cite> [<a href="https://doi.org/10.1073/pnas.2304613121" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10927520/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38408243/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proceedings%20of%20the%20National%20Academy%20of%20Sciences&amp;title=Variable%20Carbon%20Isotope%20Fractionation%20of%20Photosynthetic%20Communities%20Over%20Depth%20in%20an%20Open%E2%80%90Ocean%20Euphotic%20Zone&amp;author=L.%20C.%20Henderson&amp;author=F.%20Wittmers&amp;author=C.%20A.%20Carlson&amp;author=A.%20Z.%20Worden&amp;author=H.%20G.%20Close&amp;volume=121&amp;issue=10&amp;publication_year=2024&amp;pages=e2304613121&amp;pmid=38408243&amp;doi=10.1073/pnas.2304613121&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10118-bib-0088">
<span class="label">88.</span><cite>
Close H. G., Wakeham S. G., and Pearson A., “Lipid and <sup>13</sup>C signatures of Submicron and Suspended Particulate Organic Matter in the Eastern Tropical North Pacific: Implications for the Contribution of Bacteria,” Deep Sea Research Part A. Oceanographic Research Papers
85 (2014): 15–34, 10.1016/j.dsr.2013.11.005.</cite> [<a href="https://doi.org/10.1016/j.dsr.2013.11.005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Deep%20Sea%20Research%20Part%20A.%20Oceanographic%20Research%20Papers&amp;title=Lipid%20and%2013C%20signatures%20of%20Submicron%20and%20Suspended%20Particulate%20Organic%20Matter%20in%20the%20Eastern%20Tropical%20North%20Pacific:%20Implications%20for%20the%20Contribution%20of%20Bacteria&amp;author=H.%20G.%20Close&amp;author=S.%20G.%20Wakeham&amp;author=A.%20Pearson&amp;volume=85&amp;publication_year=2014&amp;pages=15-34&amp;doi=10.1016/j.dsr.2013.11.005&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10118-bib-0089">
<span class="label">89.</span><cite>
Freeman K. H., Hayes J. M., Trendel J. M., and Albrecht P., “Evidence From Carbon Isotope Measurements for Diverse Origins of Sedimentary Hydrocarbons,” Nature
343, no. 6255 (1990): 254–256, 10.1038/343254a0.
</cite> [<a href="https://doi.org/10.1038/343254a0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11536462/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Evidence%20From%20Carbon%20Isotope%20Measurements%20for%20Diverse%20Origins%20of%20Sedimentary%20Hydrocarbons&amp;author=K.%20H.%20Freeman&amp;author=J.%20M.%20Hayes&amp;author=J.%20M.%20Trendel&amp;author=P.%20Albrecht&amp;volume=343&amp;issue=6255&amp;publication_year=1990&amp;pages=254-256&amp;pmid=11536462&amp;doi=10.1038/343254a0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10118-bib-0090">
<span class="label">90.</span><cite>
Rossel P. E., Lipp J. S., Fredricks H. F., et al., “Intact Polar Lipids of Anaerobic Methanotrophic Archaea and Associated Bacteria,” Organic Geochemistry
39, no. 8 (2008): 992–999, 10.1016/j.orggeochem.2008.02.021.</cite> [<a href="https://doi.org/10.1016/j.orggeochem.2008.02.021" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Organic%20Geochemistry&amp;title=Intact%20Polar%20Lipids%20of%20Anaerobic%20Methanotrophic%20Archaea%20and%20Associated%20Bacteria&amp;author=P.%20E.%20Rossel&amp;author=J.%20S.%20Lipp&amp;author=H.%20F.%20Fredricks&amp;volume=39&amp;issue=8&amp;publication_year=2008&amp;pages=992-999&amp;doi=10.1016/j.orggeochem.2008.02.021&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10118-bib-0091">
<span class="label">91.</span><cite>
Rossel P. E., Elvert M., Ramette A., Boetius A., and Hinrichs K. U., “Factors Controlling the Distribution of Anaerobic Methanotrophic Communities in Marine Environments: Evidence From Intact Polar Membrane Lipids,” Geochimica et Cosmochimica Acta
75, no. 1 (2011): 164–184, 10.1016/j.gca.2010.09.031.</cite> [<a href="https://doi.org/10.1016/j.gca.2010.09.031" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Geochimica%20et%20Cosmochimica%20Acta&amp;title=Factors%20Controlling%20the%20Distribution%20of%20Anaerobic%20Methanotrophic%20Communities%20in%20Marine%20Environments:%20Evidence%20From%20Intact%20Polar%20Membrane%20Lipids&amp;author=P.%20E.%20Rossel&amp;author=M.%20Elvert&amp;author=A.%20Ramette&amp;author=A.%20Boetius&amp;author=K.%20U.%20Hinrichs&amp;volume=75&amp;issue=1&amp;publication_year=2011&amp;pages=164-184&amp;doi=10.1016/j.gca.2010.09.031&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10118-bib-0092">
<span class="label">92.</span><cite>
Schubotz F., Lipp J. S., Elvert M., et al., “Petroleum Degradation and Associated Microbial Signatures at the Chapopote Asphalt Volcano, Southern Gulf of Mexico,” Geochimica et Cosmochimica Acta
75, no. 16 (2011): 4377–4398, 10.1016/j.gca.2011.05.025.</cite> [<a href="https://doi.org/10.1016/j.gca.2011.05.025" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Geochimica%20et%20Cosmochimica%20Acta&amp;title=Petroleum%20Degradation%20and%20Associated%20Microbial%20Signatures%20at%20the%20Chapopote%20Asphalt%20Volcano,%20Southern%20Gulf%20of%20Mexico&amp;author=F.%20Schubotz&amp;author=J.%20S.%20Lipp&amp;author=M.%20Elvert&amp;volume=75&amp;issue=16&amp;publication_year=2011&amp;pages=4377-4398&amp;doi=10.1016/j.gca.2011.05.025&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10118-bib-0093">
<span class="label">93.</span><cite>
Schreiber L., Holler T., Knittel K., Meyerdierks A., and Amann R., “Identification of the Dominant Sulfate‐Reducing Bacterial Partner of Anaerobic Methanotrophs of the ANME‐2 Clade,” Environmental Microbiology
12, no. 8 (2010): 2327–2340, 10.1111/j.1462-2920.2010.02275.x.
</cite> [<a href="https://doi.org/10.1111/j.1462-2920.2010.02275.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21966923/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Environmental%20Microbiology&amp;title=Identification%20of%20the%20Dominant%20Sulfate%E2%80%90Reducing%20Bacterial%20Partner%20of%20Anaerobic%20Methanotrophs%20of%20the%20ANME%E2%80%902%20Clade&amp;author=L.%20Schreiber&amp;author=T.%20Holler&amp;author=K.%20Knittel&amp;author=A.%20Meyerdierks&amp;author=R.%20Amann&amp;volume=12&amp;issue=8&amp;publication_year=2010&amp;pages=2327-2340&amp;pmid=21966923&amp;doi=10.1111/j.1462-2920.2010.02275.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10118-bib-0094">
<span class="label">94.</span><cite>
Orphan V. J., House C. H., Hinrichs K. U., McKeegan K. D., and DeLong E. F., “Multiple Archaeal Groups Mediate Methane Oxidation in Anoxic Cold Seep Sediments,” Proceedings of the National Academy of Sciences
99, no. 11 (2002): 7663–7668, 10.1073/pnas.072210299.</cite> [<a href="https://doi.org/10.1073/pnas.072210299" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC124316/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12032340/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proceedings%20of%20the%20National%20Academy%20of%20Sciences&amp;title=Multiple%20Archaeal%20Groups%20Mediate%20Methane%20Oxidation%20in%20Anoxic%20Cold%20Seep%20Sediments&amp;author=V.%20J.%20Orphan&amp;author=C.%20H.%20House&amp;author=K.%20U.%20Hinrichs&amp;author=K.%20D.%20McKeegan&amp;author=E.%20F.%20DeLong&amp;volume=99&amp;issue=11&amp;publication_year=2002&amp;pages=7663-7668&amp;pmid=12032340&amp;doi=10.1073/pnas.072210299&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="caption p">
<p>
<strong>Table S1:</strong> Three examples showing the influence of different δ<sup>15</sup>N values on δ<sup>13</sup>C determination based on Equations (<a href="#rcm10118-disp-0001" class="usa-link">1</a>) and (<a href="#rcm10118-disp-0002" class="usa-link">2</a>), assuming a fixed δ<sup>2</sup>H value of −234‰. M1/M0 ratio for ammoniated archaeolstd (C43H88O3 + NH4<sup>+</sup>) was adjusted using an assumed δ<sup>15</sup>N value of −30‰, 0‰, and 30‰ relative to atmospheric N2 to account for potential variation in the nitrogen isotopic composition.</p>
<p>
<strong>Figure S1:</strong> Variability of reported δ<sup>2</sup>H values (‰ vs. VSMOW) in lipids derived from environmental matrices, categorized by the inferred biological origin. Lipids were grouped into archaeal, bacterial, and eukaryotic based on chemotaxonomy. <strong>Archaeal lipids:</strong> Archaeol, hydroxyarchaeol, biphytanes. <strong>Bacterial lipids:</strong> Methyl‐branched fatty acids, odd‐chain fatty acids (C<sub>15</sub>, C<sub>17</sub>, C<sub>19</sub>), cyclopropyl fatty acids, and hopanols. <strong>Eukaryotic lipids:</strong> Even‐chain short‐chain fatty acids (C<sub>14</sub>, C<sub>16</sub>, C<sub>18</sub>), polyunsaturated fatty acids, sterols, phytol, and phytol derivatives. Minimum, maximum, and midpoint δ<sup>2</sup>H values are shown for each group. δ<sup>2</sup>H data were collected from the following studies: Sessions et al. [34]; Li et al. [43]; Jones et al. [42]; Naraoka et al. [46]; Kaneko et al. [35]; Osburn et al. [45]; Wegener et al. [48]; Dawson et al. [47]; Kellerman et al. [44]; Heinzelmann et al. [39, 41].</p>
<p>
<strong>Figure S2:</strong> Relationship between the on‐column of archaeol standard injections (in ng) and δ13C value. The δ<sup>13</sup>C values were calculated for the [M + NH₄]⁺ adduct and all measurements were conducted using reversed phase‐UHPLC/ESI/qToF/MS following the method of Wörmer et al. [31]. The dashed line indicates the δ<sup>13</sup>C value of the archaeol standard determined by EA‐IRMS (−33.3‰).</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12322946/bin/RCM-39-e10118-s001.pdf" data-ga-action="click_feat_suppl" class="usa-link">RCM-39-e10118-s001.pdf</a><sup> (540.3KB, pdf) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>The data that support the findings of this study are available from the corresponding author upon request. Georeferenced data can be accessed via the PANGAEA data repository.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Rapid Communications in Mass Spectrometry are provided here courtesy of <strong>Wiley</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1002/rcm.10118"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/RCM-39-e10118.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (4.3 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12322946/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12322946/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12322946%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12322946/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12322946/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12322946/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40762245/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12322946/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40762245/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12322946/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12322946/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="AGSa2hVJTgx3giOClt3HgUVAiQY9iOX3Q5XpTVSn8dcTgv8F0wdXmOWvrJarS4Ba">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
        <script type="module" crossorigin="" src="/static/assets/math-574fdcc6.js"></script>
    
    

    </body>
</html>
