
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Improving genomic prediction for plant disease using environmental covariates - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="D2E4AECC8AF310A305AECC00184B2727.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="plantmeth">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366029/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Plant Methods">
<meta name="citation_title" content="Improving genomic prediction for plant disease using environmental covariates">
<meta name="citation_author" content="Charlotte Brault">
<meta name="citation_author_institution" content="Department of Agronomy and Plant Genetics, University of Minnesota, St. Paul, MN 55108 USA">
<meta name="citation_author" content="Emily J Conley">
<meta name="citation_author_institution" content="Department of Agronomy and Plant Genetics, University of Minnesota, St. Paul, MN 55108 USA">
<meta name="citation_author" content="Andrew C Read">
<meta name="citation_author_institution" content="USDA-ARS, Plant Science Research Unit, St. Paul, MN USA">
<meta name="citation_author" content="Andrew J Green">
<meta name="citation_author_institution" content="Department of Plant Sciences, North Dakota State University, Fargo, ND USA">
<meta name="citation_author" content="Karl D Glover">
<meta name="citation_author_institution" content="Agronomy, Horticulture, and Plant Science Department, South Dakota State University, Brookings, SD USA">
<meta name="citation_author" content="Jason P Cook">
<meta name="citation_author_institution" content="Plant Sciences and Plant Pathology Department, Montana State University, Bozeman, MT 59717 USA">
<meta name="citation_author" content="Harsimardeep S Gill">
<meta name="citation_author_institution" content="Department of Agronomy and Plant Genetics, University of Minnesota, St. Paul, MN 55108 USA">
<meta name="citation_author" content="Jason D Fiedler">
<meta name="citation_author_institution" content="USDA-ARS Cereal Crops Improvement Research Unit, Edward T. Schafer Agricultural Research Center, Fargo, ND USA">
<meta name="citation_author" content="James A Anderson">
<meta name="citation_author_institution" content="Department of Agronomy and Plant Genetics, University of Minnesota, St. Paul, MN 55108 USA">
<meta name="citation_publication_date" content="2025 Aug 20">
<meta name="citation_volume" content="21">
<meta name="citation_firstpage" content="114">
<meta name="citation_doi" content="10.1186/s13007-025-01418-0">
<meta name="citation_pmid" content="40836294">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366029/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366029/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366029/pdf/13007_2025_Article_1418.pdf">
<meta name="description" content="Fusarium Head Blight (FHB) is a destructive fungal disease affecting wheat and barley, leading to significant yield losses and reduced grain quality. Susceptibility to FHB is influenced by genetic factors, environmental conditions, and ...">
<meta name="og:title" content="Improving genomic prediction for plant disease using environmental covariates">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Fusarium Head Blight (FHB) is a destructive fungal disease affecting wheat and barley, leading to significant yield losses and reduced grain quality. Susceptibility to FHB is influenced by genetic factors, environmental conditions, and ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366029/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12366029">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1186/s13007-025-01418-0"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/13007_2025_Article_1418.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12366029%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12366029/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12366029/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366029/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-plantmeth.png" alt="Plant Methods logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Plant Methods" title="Link to Plant Methods" shape="default" href="https://plantmethods.biomedcentral.com/" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Plant Methods</button></div>. 2025 Aug 20;21:114. doi: <a href="https://doi.org/10.1186/s13007-025-01418-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1186/s13007-025-01418-0</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Plant%20Methods%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Plant%20Methods%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Plant%20Methods%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Plant%20Methods%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Improving genomic prediction for plant disease using environmental covariates</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Brault%20C%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Charlotte Brault</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Charlotte Brault</span></h3>
<div class="p">
<sup>1</sup>Department of Agronomy and Plant Genetics, University of Minnesota, St. Paul, MN 55108 USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Brault%20C%22%5BAuthor%5D" class="usa-link"><span class="name western">Charlotte Brault</span></a>
</div>
</div>
<sup>1,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Conley%20EJ%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Emily J Conley</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Emily J Conley</span></h3>
<div class="p">
<sup>1</sup>Department of Agronomy and Plant Genetics, University of Minnesota, St. Paul, MN 55108 USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Conley%20EJ%22%5BAuthor%5D" class="usa-link"><span class="name western">Emily J Conley</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Read%20AC%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Andrew C Read</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Andrew C Read</span></h3>
<div class="p">
<sup>2</sup>USDA-ARS, Plant Science Research Unit, St. Paul, MN USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Read%20AC%22%5BAuthor%5D" class="usa-link"><span class="name western">Andrew C Read</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Green%20AJ%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Andrew J Green</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Andrew J Green</span></h3>
<div class="p">
<sup>3</sup>Department of Plant Sciences, North Dakota State University, Fargo, ND USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Green%20AJ%22%5BAuthor%5D" class="usa-link"><span class="name western">Andrew J Green</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Glover%20KD%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Karl D Glover</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Karl D Glover</span></h3>
<div class="p">
<sup>4</sup>Agronomy, Horticulture, and Plant Science Department, South Dakota State University, Brookings, SD USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Glover%20KD%22%5BAuthor%5D" class="usa-link"><span class="name western">Karl D Glover</span></a>
</div>
</div>
<sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Cook%20JP%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Jason P Cook</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Jason P Cook</span></h3>
<div class="p">
<sup>5</sup>Plant Sciences and Plant Pathology Department, Montana State University, Bozeman, MT 59717 USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Cook%20JP%22%5BAuthor%5D" class="usa-link"><span class="name western">Jason P Cook</span></a>
</div>
</div>
<sup>5</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gill%20HS%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Harsimardeep S Gill</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Harsimardeep S Gill</span></h3>
<div class="p">
<sup>1</sup>Department of Agronomy and Plant Genetics, University of Minnesota, St. Paul, MN 55108 USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gill%20HS%22%5BAuthor%5D" class="usa-link"><span class="name western">Harsimardeep S Gill</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Fiedler%20JD%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Jason D Fiedler</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Jason D Fiedler</span></h3>
<div class="p">
<sup>6</sup>USDA-ARS Cereal Crops Improvement Research Unit, Edward T. Schafer Agricultural Research Center, Fargo, ND USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Fiedler%20JD%22%5BAuthor%5D" class="usa-link"><span class="name western">Jason D Fiedler</span></a>
</div>
</div>
<sup>6,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Anderson%20JA%22%5BAuthor%5D" class="usa-link" aria-describedby="id9"><span class="name western">James A Anderson</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">James A Anderson</span></h3>
<div class="p">
<sup>1</sup>Department of Agronomy and Plant Genetics, University of Minnesota, St. Paul, MN 55108 USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Anderson%20JA%22%5BAuthor%5D" class="usa-link"><span class="name western">James A Anderson</span></a>
</div>
</div>
<sup>1</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>Department of Agronomy and Plant Genetics, University of Minnesota, St. Paul, MN 55108 USA </div>
<div id="Aff2">
<sup>2</sup>USDA-ARS, Plant Science Research Unit, St. Paul, MN USA </div>
<div id="Aff3">
<sup>3</sup>Department of Plant Sciences, North Dakota State University, Fargo, ND USA </div>
<div id="Aff4">
<sup>4</sup>Agronomy, Horticulture, and Plant Science Department, South Dakota State University, Brookings, SD USA </div>
<div id="Aff5">
<sup>5</sup>Plant Sciences and Plant Pathology Department, Montana State University, Bozeman, MT 59717 USA </div>
<div id="Aff6">
<sup>6</sup>USDA-ARS Cereal Crops Improvement Research Unit, Edward T. Schafer Agricultural Research Center, Fargo, ND USA </div>
<div class="author-notes p"><div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div></div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 Apr 1; Accepted 2025 Jul 5; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© This is a U.S. Government work and not under copyright protection in the US; foreign copyright protection may apply 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12366029  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40836294/" class="usa-link">40836294</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<section id="sec1"><h3 class="pmc_sec_title">Background</h3>
<p id="Par1">Fusarium Head Blight (FHB) is a destructive fungal disease affecting wheat and barley, leading to significant yield losses and reduced grain quality. Susceptibility to FHB is influenced by genetic factors, environmental conditions, and genotype-by-environment interactions (GxE), making it challenging to predict disease resistance across diverse environments. This study investigates GxE in a long-term spring wheat multi-environment uniform nursery trial focusing on the evaluation of resistant lines in northern US breeding programs.</p></section><section id="sec2"><h3 class="pmc_sec_title">Results</h3>
<p id="Par2">Traditionally, GxE has been analyzed as a reaction norm over an environment index. Here, we computed the environment index as a linear combination of environmental covariables specific to each environment, and we derived an environment relationship matrix. Three methods were compared, all aimed at predicting untested genotypes in untested environments: the widely used Finlay-Wilkinson regression (FW), the joint-genomic regression analysis (JGRA) method, and mixed models incorporating an environmental covariates matrix. These were benchmarked against a baseline genomic selection model (GS) without environmental covariates. Predictive abilities were assessed within and across environments. The results revealed that the JGRA marker effect method was more accurate than GS in within- and across-environment predictions, although the differences were small. The predictive ability slightly decreased when the target environment was less related to the training environments. Mixed models performed similarly to JGRA within-environment, but JGRA outperformed the other methods for across-environment predictions. Additionally, JGRA identified significant genetic markers associated with baseline FHB resistance and environmental sensitivity. Furthermore, location-specific genomic estimated breeding values were predicted, providing insights into genotype stability across varying locations.</p></section><section id="sec3"><h3 class="pmc_sec_title">Conclusion</h3>
<p id="Par3">These findings highlight the value of incorporating environmental covariates to increase predictive ability and improve the selection of resistant genotypes for diverse, untested environments. By leveraging this approach, breeders can effectively exploit GxE interactions to improve disease management at no additional cost.</p></section><section id="sec4"><h3 class="pmc_sec_title">Supplementary Information</h3>
<p>The online version contains supplementary material available at 10.1186/s13007-025-01418-0.</p></section><section id="kwd-group1" lang="en" class="kwd-group"><p><strong>Keywords:</strong> Genotype-by-environment interaction, Plant breeding, Disease resistance, Genomic prediction, Wheat</p></section></section><section id="Sec1"><h2 class="pmc_sec_title">Background</h2>
<p id="Par4">Food staple production is facing multiple challenges: an increasing world population, unpredictable weather events, and the need to decrease pesticide use. To address these challenges, the selection of adapted varieties is critical. In the northern United States Great Plains, hard red spring wheat (<em>Triticum aestivum</em> L.) is the 3rd largest crop, after corn and soybean [<a href="#CR59" class="usa-link" aria-describedby="CR59">59</a>]. A very threatening disease in this region is Fusarium head blight (FHB), a fungal disease caused primarily by <em>Fusarium graminearum</em> [<a href="#CR44" class="usa-link" aria-describedby="CR44">44</a>]. It develops mostly during anthesis and causes bleached or shriveled kernels, as well as deoxynivalenol (DON) mycotoxin accumulation in the grains. High DON concentrations are harmful to humans and animals, making grain unmarketable. Quantitative trait loci (QTLs) of moderate to small effect have been identified and introgressed from Chinese wheat, especially from ‘SUMAI 3’ [<a href="#CR2" class="usa-link" aria-describedby="CR2">2</a>]. Further studies verified several QTL such as <em>Fhb1</em>, <em>Fhb3</em>, and <em>Fhb5</em> [<a href="#CR2" class="usa-link" aria-describedby="CR2">2</a>, <a href="#CR8" class="usa-link" aria-describedby="CR8">8</a>, <a href="#CR62" class="usa-link" aria-describedby="CR62">62</a>]. Nevertheless, FHB resistance remains a complex trait, with many small-effect loci involved [<a href="#CR40" class="usa-link" aria-describedby="CR40">40</a>, <a href="#CR50" class="usa-link" aria-describedby="CR50">50</a>, <a href="#CR61" class="usa-link" aria-describedby="CR61">61</a>]. The climate and the environment are known to strongly influence FHB susceptibility and DON accumulation in the grain [<a href="#CR40" class="usa-link" aria-describedby="CR40">40</a>, <a href="#CR51" class="usa-link" aria-describedby="CR51">51</a>, <a href="#CR52" class="usa-link" aria-describedby="CR52">52</a>].</p>
<p id="Par5">Since 1995, resistance to FHB has been evaluated in the uniform regional scab nursery (URSN). Approximately 30 varieties are tested annually in inoculated trials at four to six locations. Across 30 years, 817 lines have been tested in 11 locations, encompassing Minnesota (MN), North Dakota (ND), South Dakota (SD) states, and for some years, Manitoba or Saskatchewan provinces (MB and SK, respectively) in Canada. The cultivars mostly originate from public breeding programs from the University of Minnesota (UMN), North Dakota State University (NDSU), South Dakota State University (SDSU), and Montana State University (MSU), with some contributions from private companies for some years. This multi-environment trial (MET) allows for a good characterization of FHB resistance across environments.</p>
<p id="Par6">In the last two decades, genomic selection [<a href="#CR4" class="usa-link" aria-describedby="CR4">4</a>, <a href="#CR39" class="usa-link" aria-describedby="CR39">39</a>] has become increasingly popular in plant and animal breeding [<a href="#CR14" class="usa-link" aria-describedby="CR14">14</a>]. It consists of using genomic information to train a genomic selection model with phenotypic data so that the prediction of test lines can be done using genomic data only [<a href="#CR39" class="usa-link" aria-describedby="CR39">39</a>]. The practical implementation of genomic prediction (GP) in breeding programs requires some genetic relatedness between the training set and the test set, good marker coverage, and a sufficient training set size [<a href="#CR1" class="usa-link" aria-describedby="CR1">1</a>, <a href="#CR21" class="usa-link" aria-describedby="CR21">21</a>, <a href="#CR23" class="usa-link" aria-describedby="CR23">23</a>, <a href="#CR25" class="usa-link" aria-describedby="CR25">25</a>]. With enough resources, GP has proven to be efficient in improving genetic gain in many crops, including wheat [<a href="#CR3" class="usa-link" aria-describedby="CR3">3</a>, <a href="#CR13" class="usa-link" aria-describedby="CR13">13</a>, <a href="#CR14" class="usa-link" aria-describedby="CR14">14</a>, <a href="#CR36" class="usa-link" aria-describedby="CR36">36</a>, <a href="#CR63" class="usa-link" aria-describedby="CR63">63</a>].</p>
<p id="Par7">However, GP is usually based on the entry-mean value across environments, or predictions are made within environments, thus ignoring the environmental effect and the interaction between the genotype and the environment [<a href="#CR9" class="usa-link" aria-describedby="CR9">9</a>].</p>
<p id="Par8">In quantitative genetics theory, phenotypic variation is determined by genetics, environment, and genotype-by-environment (GxE) variances [<a href="#CR18" class="usa-link" aria-describedby="CR18">18</a>]. When the GxE variance is high, the ranking of genotypes changes across environments, which impacts the accuracy of breeding decisions. Indeed, allelic effects might not be consistent across environments [<a href="#CR42" class="usa-link" aria-describedby="CR42">42</a>, <a href="#CR56" class="usa-link" aria-describedby="CR56">56</a>]. Thus, including GxE in the breeding pipeline can increase the accuracy of GP [<a href="#CR9" class="usa-link" aria-describedby="CR9">9</a>] and enable the selection of genotypes adapted to specific environmental conditions to be targeted [<a href="#CR28" class="usa-link" aria-describedby="CR28">28</a>].</p>
<p id="Par9">One of the earliest methods of characterizing GxE variation involves regressing the phenotypic value against the mean phenotypic value for each environment and genotype as a reaction norm [<a href="#CR16" class="usa-link" aria-describedby="CR16">16</a>, <a href="#CR34" class="usa-link" aria-describedby="CR34">34</a>, <a href="#CR65" class="usa-link" aria-describedby="CR65">65</a>]. This method is now known as the Finlay-Wilkinson (FW) regression [<a href="#CR19" class="usa-link" aria-describedby="CR19">19</a>]. The regressor then contains an estimate of the environment’s suitability, and the genotype slope provides an estimation of its stability. This method has recently been adapted to incorporate environment covariates (ECs), making it possible to predict performance in an unknown environment [<a href="#CR46" class="usa-link" aria-describedby="CR46">46</a>, <a href="#CR58" class="usa-link" aria-describedby="CR58">58</a>].</p>
<p id="Par10">More recently, ECs have become more accessible for a wide range of locations and years and have been integrated into GP models using kernels [<a href="#CR10" class="usa-link" aria-describedby="CR10">10</a>–<a href="#CR12" class="usa-link" aria-describedby="CR12">12</a>, <a href="#CR24" class="usa-link" aria-describedby="CR24">24</a>, <a href="#CR28" class="usa-link" aria-describedby="CR28">28</a>, <a href="#CR41" class="usa-link" aria-describedby="CR41">41</a>, <a href="#CR47" class="usa-link" aria-describedby="CR47">47</a>]. In some cases, the ECs were used to compute an environmental relationship matrix (ERM) [<a href="#CR10" class="usa-link" aria-describedby="CR10">10</a>, <a href="#CR28" class="usa-link" aria-describedby="CR28">28</a>, <a href="#CR43" class="usa-link" aria-describedby="CR43">43</a>] in the same way as the genomic relationship matrix (GRM) to capture the covariance between untested environments. However, this method implies that all ECs have the same impact on the phenotypes. Lopez-Cruz et al. [<a href="#CR35" class="usa-link" aria-describedby="CR35">35</a>] regressed each EC against the phenotype in a mixed model to decompose the GxE into each environmental component. That requires a multiple testing procedure with many parameters to be estimated. To overcome that, Li et al. [<a href="#CR31" class="usa-link" aria-describedby="CR31">31</a>–<a href="#CR33" class="usa-link" aria-describedby="CR33">33</a>] searched for the most appropriate EC and associated time window through the critical environment regressor with informed search (CERIS) joint genomic regression analysis (JGRA) This method allowed them to characterize more closely the allelic effect of known genes on the flowering time trait [<a href="#CR32" class="usa-link" aria-describedby="CR32">32</a>]. Factor analytics is another method for overcoming the dimension problem of the prediction when considering the phenotypes in multiple environments as multiple traits [<a href="#CR26" class="usa-link" aria-describedby="CR26">26</a>, <a href="#CR49" class="usa-link" aria-describedby="CR49">49</a>].</p>
<p id="Par11">Using an environment index is a simple, univariate method to integrate GxE, however, the definition of a single variable integrating the environment effect is challenging. A synthetic variable from ECs can also be computed via various methods [<a href="#CR46" class="usa-link" aria-describedby="CR46">46</a>, <a href="#CR48" class="usa-link" aria-describedby="CR48">48</a>, <a href="#CR57" class="usa-link" aria-describedby="CR57">57</a>].</p>
<p id="Par12">In recent years, multiple methods have been developed to predict GxE, but comprehensive comparisons between those methods are lacking. This study aims to provide a thorough comparison of GxE methods for predicting untested environments and untested genotypes using a multiyear wheat dataset applied to FHB-related trait prediction. Specifically, we tested whether including GxE in prediction models increases the predictive ability and which model is the most appropriate for this inclusion. In addition to the prediction, the significance of the markers for explaining the baseline resistance or sensitivity to the environment was characterized. Finally, the ranking of genotypes in each location was estimated. To the best of our knowledge, this is the first study to include ECs to predict GxE for disease traits, despite some evidence that disease-related traits are affected by GxE [<a href="#CR55" class="usa-link" aria-describedby="CR55">55</a>].</p></section><section id="Sec2"><h2 class="pmc_sec_title">Method</h2>
<section id="Sec3"><h3 class="pmc_sec_title">Experimental design</h3>
<p id="Par13">The experimental design of the URSN population was described previously [<a href="#CR5" class="usa-link" aria-describedby="CR5">5</a>]. A randomized complete block design was used at each location with three blocks. After each year, the data were aggregated at the genotype level, accounting for field heterogeneity or outliers if necessary. Each year, three to six locations were used, spanning Minnesota, North Dakota, South Dakota, and Manitoba (Canada) for some years (Fig. <a href="#Fig1" class="usa-link">1</a>A, B).</p>
<figure class="fig xbox font-sm" id="Fig1"><h4 class="obj_head">Fig. 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12366029_13007_2025_1418_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/21f2/12366029/720ff98c5044/13007_2025_1418_Fig1_HTML.jpg" loading="lazy" id="MO1" height="789" width="797" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Description of the experimental design. <strong>A</strong> Experimental design for the selected lines, for disease index (DIS) and visual scabby kernel (VSK) traits. Each tile’s color represents the number of genotypes tested in that environment. <strong>B</strong> Map of the uniform regional scab nursery (URSN) locations. The size of the point indicates the number of times a location has been used in the 1995–2024 period. <strong>C</strong> Distribution of aggregated phenotypic data per location, colored by year of trial. The cross corresponds to the mean in each location, while the boxplot displays the quartiles. For both plots, the x-axis represents the locations of the uniform regional scab nursery (URSN) trials; the number of observations is indicated below each boxplot</p></figcaption></figure><p id="Par14">FHB phenotyping was performed for two traits: visual scabby kernel (VSK, expressed in percentage, crop ontology code CO_321:0001155) and disease index (DIS, expressed in percentage, crop ontology code CO_321:0501030), which is the multiplication between incidence and severity. The locations were inoculated with infected grain spawn, except for St. Paul (MN), for which the inoculation was done with inoculum spraying, and Brookings (SD), for which both methods were used. In the protocols, plots were inoculated around the anthesis date and misted to ensure good disease propagation.</p>
<p id="Par15">The check ‘WHEATON’ was consistently used over the 30 years of experiments. Other checks have been used for many years, ensuring good connectivity.</p>
<p id="Par16">From the initial dataset, individuals without genotypic data or tested in less than four environments were filtered out, and environments with fewer than 10 individuals were removed. The remaining dataset comprised 2,024 and 1,947 observations with 280 and 283 lines for 107 and 109 environments for VSK and DIS, respectively (Fig. <a href="#Fig1" class="usa-link">1</a>A). Most of the time, genotypes were tested at all the locations for a given year.</p>
<p id="Par17">Since no genotyping information was needed, phenotypic data from all available environments with more than 10 genotypes (n = 121 for DIS and n = 125 for VSK) were used for the determination of the environment index.</p>
<p id="Par18">An analysis of variance was conducted after a first subset of the data to retain the genotypes tested for more than one year, leading to 83 genotypes remaining. A mixed effect model was fitted, such as: <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq1"><math id="d33e526"><mrow><msub><mi>y</mi><mrow><mi mathvariant="italic">ijk</mi></mrow></msub><mo>=</mo><mi>μ</mi><mo>+</mo><msub><mi>g</mi><mi>i</mi></msub><mo>+</mo><msub><mi>L</mi><mi>j</mi></msub><mo>+</mo><msub><mi>Y</mi><mi>k</mi></msub><mo>+</mo><msub><mrow><mo stretchy="false">(</mo><mi>g</mi><mo>:</mo><mi>L</mi><mo stretchy="false">)</mo></mrow><mrow><mi mathvariant="italic">ij</mi></mrow></msub><mo>+</mo><msub><mrow><mo stretchy="false">(</mo><mi>g</mi><mo>:</mo><mi>Y</mi><mo stretchy="false">)</mo></mrow><mrow><mi mathvariant="italic">ik</mi></mrow></msub><mo>+</mo><msub><mrow><mo stretchy="false">(</mo><mi>L</mi><mo>:</mo><mi>Y</mi><mo stretchy="false">)</mo></mrow><mrow><mi mathvariant="italic">jk</mi></mrow></msub><mo>+</mo><msub><mi>ϵ</mi><mrow><mi mathvariant="italic">ijk</mi></mrow></msub></mrow></math></span>, with <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq2"><math id="d33e600"><msub><mi>g</mi><mi>i</mi></msub></math></span> the main genetic effect (83 levels), <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq3"><math id="d33e611"><msub><mi>L</mi><mi>j</mi></msub></math></span> the location effect (11 levels), <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq4"><math id="d33e622"><msub><mi>Y</mi><mi>k</mi></msub></math></span> the year effect (30 levels), and their corresponding interaction (:). All effects were fitted as random with independent and identically distributed variances.</p></section><section id="Sec4"><h3 class="pmc_sec_title">Genotypic data analysis</h3>
<p id="Par19">The lines were genotyped with the USDA-3K genotypic array, comprising 3421 single-nucleotide polymorphisms (SNPs), mapped on the IWGSC Chinese Spring reference genome v2.1 [<a href="#CR67" class="usa-link" aria-describedby="CR67">67</a>]. Markers and individuals with more than 25% missing data were excluded. Beagle software was used to impute the remaining missing data with default parameters [<a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>, <a href="#CR7" class="usa-link" aria-describedby="CR7">7</a>]. A minor allele frequency cutoff of 5% was used, leading to 2301 SNPs remaining for 467 individuals. The genomic matrix was scaled, and the additive genomic relationship matrix (GRM) was computed using the rrBLUP R package [<a href="#CR17" class="usa-link" aria-describedby="CR17">17</a>] following VanRaden [<a href="#CR60" class="usa-link" aria-describedby="CR60">60</a>] method as <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq5"><math id="d33e653"><mrow><mi>G</mi><mo>=</mo><mi>M</mi><msup><mrow><mi>M</mi></mrow><mo>′</mo></msup><mo stretchy="false">/</mo><mrow><mo stretchy="false">(</mo><mn>2</mn><msubsup><mo>∑</mo><mrow><mi>i</mi><mo>=</mo><mn>1</mn></mrow><mi>m</mi></msubsup><msub><mi>p</mi><mi>i</mi></msub><mrow><mo stretchy="false">(</mo><mn>1</mn><mo>-</mo><msub><mi>p</mi><mi>i</mi></msub><mo stretchy="false">)</mo></mrow><mo stretchy="false">)</mo></mrow></mrow></math></span>, with <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq6"><math id="d33e697"><mrow><msub><mi>M</mi><mrow><mi mathvariant="italic">ij</mi></mrow></msub><mo>=</mo><msub><mi>X</mi><mrow><mi mathvariant="italic">ij</mi></mrow></msub><mo>+</mo><mn>1</mn><mo>-</mo><mn>2</mn><msub><mi>p</mi><mi>i</mi></msub></mrow></math></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq7"><math id="d33e724"><msub><mi>X</mi><mrow><mi mathvariant="italic">ij</mi></mrow></msub></math></span> being the genotype information for the line <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq8"><math id="d33e737"><mi>j</mi></math></span> and the marker <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq9"><math id="d33e746"><mi>i</mi></math></span>, coded as 0, 1, or 2 for the number of alleles in common with the reference genome, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq10"><math id="d33e755"><msub><mi>p</mi><mi>i</mi></msub></math></span> the major allele frequency, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq11"><math id="d33e767"><mi>m</mi></math></span> the number of markers.</p></section><section id="Sec5"><h3 class="pmc_sec_title">Environmental data analysis</h3>
<p id="Par20">The environmental data was extracted and combined from two sources: NASA (National Aeronautics and Space Administration) and NOAA (National Oceanic and Atmospheric Administration) weather stations (see Table S1 for the identification of the stations used). The following variables were used from NOAA stations: daily rainfall precipitation, minimum, maximum, and average daily temperatures. Furthermore, the R package <em>EnvRType</em>, through <em>nasapower</em>, was applied to search for the remaining ECs and to impute missing values based on the geographical coordinates of the field trials [<a href="#CR10" class="usa-link" aria-describedby="CR10">10</a>, <a href="#CR53" class="usa-link" aria-describedby="CR53">53</a>, <a href="#CR54" class="usa-link" aria-describedby="CR54">54</a>]. Other variables from the <em>EnvRtype</em> include computed variables related to plant growth and ecophysiology, such as potential evapotranspiration or the slope of the saturation vapor pressure curve, for example. The <em>soilType</em> R package was used to get 10 soil variables [<a href="#CR20" class="usa-link" aria-describedby="CR20">20</a>] based on the World Soil Database (WoSIS). A full description of the ECs is available in Table <a href="#Tab1" class="usa-link">1</a>.
</p>
<section class="tw xbox font-sm" id="Tab1"><h4 class="obj_head">Table 1.</h4>
<div class="caption p"><p>Description of environmental covariates (ECs) obtained with <em>nasapower</em> and NOAA weather stations</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Variable</th>
<th align="left" colspan="1" rowspan="1">Description</th>
<th align="left" colspan="1" rowspan="1">Unit</th>
<th align="left" colspan="1" rowspan="1">Location variable</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">ALLSKY_SFC_LW_DWN</td>
<td align="left" colspan="1" rowspan="1">All sky insolation incidence on a horizontal surface</td>
<td align="left" colspan="1" rowspan="1">MJ.m<sup>-2</sup>.d<sup>-1</sup>
</td>
<td align="left" colspan="1" rowspan="1">FALSE</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">ALLSKY_SFC_PAR_TOT</td>
<td align="left" colspan="1" rowspan="1">All Sky Surface Photosynthetically Active Radiation (PAR) Total</td>
<td align="left" colspan="1" rowspan="1">W.m<sup>-2</sup>
</td>
<td align="left" colspan="1" rowspan="1">FALSE</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">ALLSKY_SFC_SW_DNI</td>
<td align="left" colspan="1" rowspan="1">All Sky Surface Shortwave Downward Direct Normal Irradiance</td>
<td align="left" colspan="1" rowspan="1">W.m<sup>-2</sup>
</td>
<td align="left" colspan="1" rowspan="1">FALSE</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">ALLSKY_SFC_SW_DWN</td>
<td align="left" colspan="1" rowspan="1">Thermal infrared longwave radiative flux</td>
<td align="left" colspan="1" rowspan="1">MJ.m<sup>-2</sup>.d<sup>-1</sup>
</td>
<td align="left" colspan="1" rowspan="1">FALSE</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">ALLSKY_SFC_UVA</td>
<td align="left" colspan="1" rowspan="1">All Sky Surface Ultraviolet A (315 nm-400 nm) Irradiance</td>
<td align="left" colspan="1" rowspan="1">W.m<sup>-2</sup>
</td>
<td align="left" colspan="1" rowspan="1">FALSE</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">ALLSKY_SFC_UVB</td>
<td align="left" colspan="1" rowspan="1">All Sky Surface Ultraviolet B (280 nm-315 nm) Irradiance</td>
<td align="left" colspan="1" rowspan="1">W.m<sup>-2</sup>
</td>
<td align="left" colspan="1" rowspan="1">FALSE</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">ETP</td>
<td align="left" colspan="1" rowspan="1">Potential evapotranspiration</td>
<td align="left" colspan="1" rowspan="1">mm.d<sup>-1</sup>
</td>
<td align="left" colspan="1" rowspan="1">FALSE</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">EVPTRNS</td>
<td align="left" colspan="1" rowspan="1">Evapotranspiration energy flux at the surface of the earth</td>
<td align="left" colspan="1" rowspan="1">MJ.m<sup>-2</sup>.d<sup>-1</sup>
</td>
<td align="left" colspan="1" rowspan="1">FALSE</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">FROST_DAYS</td>
<td align="left" colspan="1" rowspan="1">If it was a frost day (temperature less than 0C or 32F)</td>
<td align="left" colspan="1" rowspan="1">-</td>
<td align="left" colspan="1" rowspan="1">FALSE</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">FRUE</td>
<td align="left" colspan="1" rowspan="1">Effect of temperature on radiation-use-efficiency</td>
<td align="left" colspan="1" rowspan="1">-</td>
<td align="left" colspan="1" rowspan="1">FALSE</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">GWETROOT</td>
<td align="left" colspan="1" rowspan="1">Root Zone Soil Wetness (layer from 0 to 100 cm), ranging from 0 to 1 (water-free to saturated soil)</td>
<td align="left" colspan="1" rowspan="1">-</td>
<td align="left" colspan="1" rowspan="1">FALSE</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">GWETTOP</td>
<td align="left" colspan="1" rowspan="1">Surface Soil Wetness (layer from 0 to 5 cm), ranging from 0 to 1 (water-free to saturated soil)</td>
<td align="left" colspan="1" rowspan="1">-</td>
<td align="left" colspan="1" rowspan="1">FALSE</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">LAT</td>
<td align="left" colspan="1" rowspan="1">Latitude</td>
<td align="left" colspan="1" rowspan="1">°</td>
<td align="left" colspan="1" rowspan="1">TRUE</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">LON</td>
<td align="left" colspan="1" rowspan="1">Longitude</td>
<td align="left" colspan="1" rowspan="1">°</td>
<td align="left" colspan="1" rowspan="1">TRUE</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">N</td>
<td align="left" colspan="1" rowspan="1">Photoperiod computed from latitude and Julian day</td>
<td align="left" colspan="1" rowspan="1">h</td>
<td align="left" colspan="1" rowspan="1">TRUE</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">n</td>
<td align="left" colspan="1" rowspan="1">Number of sunny hours in the day for a clear sky (no clouds)</td>
<td align="left" colspan="1" rowspan="1">h</td>
<td align="left" colspan="1" rowspan="1">FALSE</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">PAR_TEMP</td>
<td align="left" colspan="1" rowspan="1">Computed ratio between ALLSKY_SFC_PAR_TOT and T2M</td>
<td align="left" colspan="1" rowspan="1">-</td>
<td align="left" colspan="1" rowspan="1">FALSE</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">P-ETP</td>
<td align="left" colspan="1" rowspan="1">Computed difference between preciptation and evapotranspiration</td>
<td align="left" colspan="1" rowspan="1">mm.d<sup>-1</sup>
</td>
<td align="left" colspan="1" rowspan="1">FALSE</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">PRECTOT</td>
<td align="left" colspan="1" rowspan="1">Rainfall precipitation</td>
<td align="left" colspan="1" rowspan="1">mm.d<sup>-1</sup>
</td>
<td align="left" colspan="1" rowspan="1">FALSE</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">QV2M</td>
<td align="left" colspan="1" rowspan="1">Specific humidity at 2 m</td>
<td align="left" colspan="1" rowspan="1">%</td>
<td align="left" colspan="1" rowspan="1">FALSE</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">RH2M</td>
<td align="left" colspan="1" rowspan="1">Relative air humidity at 2 m</td>
<td align="left" colspan="1" rowspan="1">%</td>
<td align="left" colspan="1" rowspan="1">FALSE</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">RTA</td>
<td align="left" colspan="1" rowspan="1">Global solar radiation based on latitude and Julian day</td>
<td align="left" colspan="1" rowspan="1">W.m<sup>-2</sup>
</td>
<td align="left" colspan="1" rowspan="1">FALSE</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">SNOW</td>
<td align="left" colspan="1" rowspan="1">Snow fall</td>
<td align="left" colspan="1" rowspan="1">mm</td>
<td align="left" colspan="1" rowspan="1">FALSE</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">SNWD</td>
<td align="left" colspan="1" rowspan="1">Snow depth</td>
<td align="left" colspan="1" rowspan="1">mm</td>
<td align="left" colspan="1" rowspan="1">FALSE</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">SPV</td>
<td align="left" colspan="1" rowspan="1">Slope of saturation vapor pressure curve</td>
<td align="left" colspan="1" rowspan="1">kPA.C°.d<sup>-1</sup>
</td>
<td align="left" colspan="1" rowspan="1">FALSE</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">T2M</td>
<td align="left" colspan="1" rowspan="1">Temperature at 2 m</td>
<td align="left" colspan="1" rowspan="1">C°.d<sup>-1</sup>
</td>
<td align="left" colspan="1" rowspan="1">FALSE</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">T2M_RANGE</td>
<td align="left" colspan="1" rowspan="1">Temperature range</td>
<td align="left" colspan="1" rowspan="1">C°.d<sup>-1</sup>
</td>
<td align="left" colspan="1" rowspan="1">FALSE</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">T2MDEW</td>
<td align="left" colspan="1" rowspan="1">Dew-point temperature at 2 m</td>
<td align="left" colspan="1" rowspan="1">C°.d<sup>-1</sup>
</td>
<td align="left" colspan="1" rowspan="1">FALSE</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">T2M_MAX</td>
<td align="left" colspan="1" rowspan="1">Maximum air temperature at 2 m</td>
<td align="left" colspan="1" rowspan="1">C°.d<sup>-1</sup>
</td>
<td align="left" colspan="1" rowspan="1">FALSE</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">T2M_MIN</td>
<td align="left" colspan="1" rowspan="1">Minimum air temperature at 2 m</td>
<td align="left" colspan="1" rowspan="1">C°.d<sup>-1</sup>
</td>
<td align="left" colspan="1" rowspan="1">FALSE</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">TH1</td>
<td align="left" colspan="1" rowspan="1">Temperature-Humidity Index by NRC 1971</td>
<td align="left" colspan="1" rowspan="1">C°.d<sup>-1</sup>
</td>
<td align="left" colspan="1" rowspan="1">FALSE</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">TH2</td>
<td align="left" colspan="1" rowspan="1">Temperature-Humidity Index by Yousef 1985</td>
<td align="left" colspan="1" rowspan="1">C°.d<sup>-1</sup>
</td>
<td align="left" colspan="1" rowspan="1">FALSE</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">VPD</td>
<td align="left" colspan="1" rowspan="1">Vapor pressure deficit</td>
<td align="left" colspan="1" rowspan="1">kPA.d<sup>-1</sup>
</td>
<td align="left" colspan="1" rowspan="1">FALSE</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">WS2M</td>
<td align="left" colspan="1" rowspan="1">Wind speed at 10 m</td>
<td align="left" colspan="1" rowspan="1">m.s<sup>-1</sup>
</td>
<td align="left" colspan="1" rowspan="1">FALSE</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">CLAY</td>
<td align="left" colspan="1" rowspan="1">Clay total</td>
<td align="left" colspan="1" rowspan="1">g/100 g</td>
<td align="left" colspan="1" rowspan="1">TRUE</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">CECPH7</td>
<td align="left" colspan="1" rowspan="1">Cation exchange capacity at pH7</td>
<td align="left" colspan="1" rowspan="1">Cmol(c)/kg</td>
<td align="left" colspan="1" rowspan="1">TRUE</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">ECEC</td>
<td align="left" colspan="1" rowspan="1">Effective cation exchange capacity</td>
<td align="left" colspan="1" rowspan="1">Cmol(c)/kg</td>
<td align="left" colspan="1" rowspan="1">TRUE</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">NITKJD</td>
<td align="left" colspan="1" rowspan="1">Total nitrogen</td>
<td align="left" colspan="1" rowspan="1">g/kg</td>
<td align="left" colspan="1" rowspan="1">TRUE</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">ORGC</td>
<td align="left" colspan="1" rowspan="1">Organic carbon</td>
<td align="left" colspan="1" rowspan="1">g/kg</td>
<td align="left" colspan="1" rowspan="1">TRUE</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">SILT</td>
<td align="left" colspan="1" rowspan="1">Silt total</td>
<td align="left" colspan="1" rowspan="1">g/100 g</td>
<td align="left" colspan="1" rowspan="1">TRUE</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">TCEQ</td>
<td align="left" colspan="1" rowspan="1">Calcium carbonate capacity</td>
<td align="left" colspan="1" rowspan="1">Cmol /kg</td>
<td align="left" colspan="1" rowspan="1">TRUE</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">WG0006</td>
<td align="left" colspan="1" rowspan="1">Water retention gravimetric 6 kPa</td>
<td align="left" colspan="1" rowspan="1">g/100 g</td>
<td align="left" colspan="1" rowspan="1">TRUE</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">WV0006</td>
<td align="left" colspan="1" rowspan="1">Water retention volumetric 6 kPa</td>
<td align="left" colspan="1" rowspan="1">Cm3/100cm3</td>
<td align="left" colspan="1" rowspan="1">TRUE</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">PHPTOT</td>
<td align="left" colspan="1" rowspan="1">Total phosphorus</td>
<td align="left" colspan="1" rowspan="1">mg/kg</td>
<td align="left" colspan="1" rowspan="1">TRUE</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">SAND</td>
<td align="left" colspan="1" rowspan="1">Total sand</td>
<td align="left" colspan="1" rowspan="1">g/100 g</td>
<td align="left" colspan="1" rowspan="1">TRUE</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">TOTC</td>
<td align="left" colspan="1" rowspan="1">Total carbon</td>
<td align="left" colspan="1" rowspan="1">g/100 g</td>
<td align="left" colspan="1" rowspan="1">TRUE</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><p id="Par21">For each phenotyping environment, the daily environmental data were subset to dates between planting and harvesting. Finally, as in Costa-Neto et al. [<a href="#CR10" class="usa-link" aria-describedby="CR10">10</a>], we aggregated ECs across five wheat stages based on growing degree days, following Zadok’s scale [<a href="#CR66" class="usa-link" aria-describedby="CR66">66</a>].</p>
<p id="Par22">Wheat stages and their corresponding cumulative growing Celsius degree-days since sowing (GDD):</p>
<ul class="list" style="list-style-type:disc">
<li><p id="Par23">Stage 0–3 (V03) sowing to double-ridge: 400 GDD</p></li>
<li><p id="Par24">Stage 3–5 (V35) double-ridge to terminal spikelet: 685 GDD</p></li>
<li><p id="Par25">Stage 5–6 (V67) terminal spikelet to heading: 993 GDD</p></li>
<li><p id="Par26">Stage 6–7 (V67) heading to anthesis: 1075 GDD</p></li>
<li><p id="Par27">Stage 7–9 (V79) anthesis to maturity: 1825 GDD.</p></li>
</ul>
<p id="Par28">These values were obtained from the <a href="https://ipad.fas.usda.gov/cropexplorer/description.aspx?legendid=314" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://ipad.fas.usda.gov/cropexplorer/description.aspx?legendid=314</a> website (last accessed on 2024-03-17) and corrected for the heading date based on field observations.</p>
<p id="Par29">After aggregation, the environmental data (W-matrix) comprised k = 143 ECs (26 weather variables divided into five stages, 10 soil ECs, and three location-specific ECs: latitude, longitude, and daylight hours) for 330 environments (for the 11 locations and each of the 30 years). Each column of the W-matrix was centered and scaled.</p>
<p id="Par30">From the W-matrix, the environmental relationship matrix was computed following [<a href="#CR10" class="usa-link" aria-describedby="CR10">10</a>] as <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq12"><math id="d33e1338"><mrow><mi mathvariant="normal">Ω</mi><mo>=</mo><mfrac><mrow><mi>W</mi><msup><mi>W</mi><mo>′</mo></msup></mrow><mrow><mi>t</mi><mi>r</mi><mo stretchy="false">(</mo><mi>W</mi><mo stretchy="false">)</mo><mo stretchy="false">/</mo><mi>k</mi></mrow></mfrac></mrow></math></span>, where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq13"><math id="d33e1367"><mrow><mi>t</mi><mi>r</mi><mo stretchy="false">(</mo><mi>W</mi><mo stretchy="false">)</mo></mrow></math></span> is the trace of the W matrix and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq14"><math id="d33e1383"><mi>k</mi></math></span> represents the number of ECs. The environments were characterized by their ECs through the ERM and Principal Component Analysis (PCA).</p></section><section id="Sec6"><h3 class="pmc_sec_title">Determination of the environment index</h3>
<p id="Par31">The environment index was computed as a linear regression between the ECs, with the relative weights predicted by partial least square regression (PLSR). This method is expected to account for the correlation between the ECs. The explanatory variable was the mean phenotype in each of the 121 or 125 environments, and the dependent variables were the ECs at different stages. Thirteen different sets of ECs were compared: fitting variables from the five wheat stages separately; all wheat stages, including or excluding location-specific variables (i.e., the 10 soil variables and latitude, longitude, and number of daylight hours), and soil variables only. Using a leave-one-environment-out cross-validation scheme for each trait, the PLSR method was applied for each combination of variables (the number of components was selected by an inner cross-validation using the Q2 parameter) and the Pearson correlation coefficient between predicted and observed mean phenotype was calculated (Fig. <a href="#Fig2" class="usa-link">2</a>).</p>
<figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12366029_13007_2025_1418_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/21f2/12366029/e3a2b632c4e3/13007_2025_1418_Fig2_HTML.jpg" loading="lazy" id="MO2" height="380" width="780" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Determination of the environment index. Correlation between mean phenotype and predicted environment index for each environment, using all or a selection of environmental covariates (ECs). <strong>A</strong> for visual scabby kernel (VSK) trait, <strong>B</strong> for disease index (DIS) trait. The ECs specific to each location, encompassing soil variables, latitude, longitude, and daylight hours, were removed in one of the methods (green circle). Description of stages: V03: sowing to double-ridge, V35: double-ridge to terminal spikelet, V56: terminal spikelet to heading, V67: heading to anthesis, V79: anthesis to maturity</p></figcaption></figure><p id="Par32">Some specific weather variables such as temperature, precipitation, and relative humidity are known to strongly affect FHB, especially around anthesis [<a href="#CR29" class="usa-link" aria-describedby="CR29">29</a>, <a href="#CR64" class="usa-link" aria-describedby="CR64">64</a>]. To test that hypothesis, we computed the sum of those three variables averaged over ± 14 days around anthesis, determined as 993 GDD after sowing (De Wolf, personal communication).</p></section><section id="Sec7"><h3 class="pmc_sec_title">GxE methods</h3>
<section id="Sec8"><h4 class="pmc_sec_title">FW method</h4>
<p id="Par33">The R package FW [<a href="#CR30" class="usa-link" aria-describedby="CR30">30</a>] was used to compute the random regression of the phenotype against the mean environmental phenotype. First, intercept <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq15"><math id="d33e1430"><mover accent="true"><mi>μ</mi><mo stretchy="true">^</mo></mover></math></span> and slope <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq16"><math id="d33e1442"><mover accent="true"><mi>β</mi><mo stretchy="true">^</mo></mover></math></span> parameters were estimated for each genotype using the FW method, using a Bayesian regression. The phenotypic data used here were phenotypic values obtained in each environment and beforehand corrected for spatial heterogeneity, subset for training set genotypes and training set environments. The intercept and slope parameters were subsequently predicted for the testing set genotypes by genomic prediction, using the ridge regression BLUP (RR-BLUP) method, using genomic markers. The final phenotypic value for the test set was determined via the PLSR environment index. This method was compared with JGRA as it corresponds to the most popular method, adapted here for prediction in an unknown environment.</p></section><section id="Sec9"><h4 class="pmc_sec_title">JGRA</h4>
<ul class="list" style="list-style-type:disc"><li><p id="Par35"><strong>JGRA RN (reaction norm)</strong></p></li></ul>
<p id="Par36">In JGRA RN, for each line, a linear regression model is fitted for each genotype such as <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq17"><math id="d33e1465"><mrow><msub><mrow><mi mathvariant="bold-italic">y</mi></mrow><mrow><mi mathvariant="bold-italic">j</mi></mrow></msub><mo>=</mo><msub><mi>μ</mi><mi>j</mi></msub><mo>+</mo><mi>δ</mi><msub><mrow><mi mathvariant="bold-italic">β</mi></mrow><mrow><mi mathvariant="bold-italic">j</mi></mrow></msub><mo>+</mo><msub><mrow><mi mathvariant="bold-italic">ε</mi></mrow><mi>j</mi></msub></mrow></math></span>, with <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq18"><math id="d33e1500"><msub><mi>y</mi><mi>j</mi></msub></math></span> the (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq19"><math id="d33e1511"><mrow><mi>p</mi><mo>×</mo><mn>1</mn><mo stretchy="false">)</mo></mrow></math></span> vector of the phenotypic value for the genotype <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq20"><math id="d33e1525"><mi>j</mi></math></span> tested in several environments, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq21"><math id="d33e1534"><msub><mi>μ</mi><mi>j</mi></msub></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq22"><math id="d33e1546"><msub><mi>β</mi><mi>j</mi></msub></math></span> are the random genotype-specific intercept and the slope, respectively, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq23"><math id="d33e1557"><mi>δ</mi></math></span> is the vector of environment index for each environment (dimension <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq24"><math id="d33e1566"><mrow><mi>p</mi><mo>×</mo><mn>1</mn></mrow></math></span>), and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq25"><math id="d33e1578"><msub><mrow><mi mathvariant="bold-italic">ε</mi></mrow><mi>j</mi></msub></math></span> the residuals. We assume that <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq26"><math id="d33e1591"><msub><mi>μ</mi><mi>j</mi></msub></math></span>
<sub>j</sub><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq27"><math id="d33e1604"><msub><mi>β</mi><mi>j</mi></msub></math></span>
<sub>j</sub>, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq28"><math id="d33e1617"><msub><mrow><mi mathvariant="bold-italic">ε</mi></mrow><mi>j</mi></msub></math></span> are normally distributed, with a mean of 0 and with <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq29"><math id="d33e1630"><mrow><msub><mrow><msup><mrow><mi>σ</mi></mrow><mn>2</mn></msup></mrow><mi>μ</mi></msub><msub><mrow><msup><mrow><mi>σ</mi></mrow><mn>2</mn></msup></mrow><mi>β</mi></msub><msub><mrow><msup><mrow><mi>σ</mi></mrow><mn>2</mn></msup></mrow><mi>ϵ</mi></msub></mrow></math></span> variances. The normality of the residuals of that linear regression was assessed with a Shapiro–Wilk test, with 92% of the prediction combinations having a p-value superior to 0.05. The intercept and slope parameters were combined for all genotypes in the training set.</p>
<p id="Par37">Then, intercept and slope parameters were predicted for the validation set by GP using RR-BLUP method and genomic markers. Finally, genotypic values were predicted as <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq30"><math id="d33e1662"><mrow><msub><mover accent="true"><mrow><mi mathvariant="bold-italic">y</mi></mrow><mo stretchy="true">^</mo></mover><mrow><mrow><mi mathvariant="bold-italic">i</mi></mrow><mo>,</mo><mrow><mi mathvariant="bold-italic">j</mi></mrow></mrow></msub><mo>=</mo><mover accent="true"><msub><mi>μ</mi><mi>j</mi></msub><mo stretchy="true">^</mo></mover><mo>+</mo><msub><mi>δ</mi><mi>i</mi></msub><mover accent="true"><msub><mrow><mi mathvariant="bold-italic">β</mi></mrow><mrow><mi mathvariant="bold-italic">j</mi></mrow></msub><mo stretchy="true">^</mo></mover></mrow></math></span> for each genotype in each environment.</p>
<ul class="list" style="list-style-type:disc"><li><p id="Par38"><strong>JGRA Marker Effect</strong></p></li></ul>
<p id="Par39">In the JGRA Marker Effect method, a GP model is fitted within each environment, such as <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq31"><math id="d33e1714"><mrow><msub><mrow><mi mathvariant="bold-italic">y</mi></mrow><mrow><mi mathvariant="bold-italic">i</mi></mrow></msub><mo>=</mo><mi>Z</mi><msub><mrow><mi mathvariant="bold-italic">u</mi></mrow><mrow><mi mathvariant="bold-italic">i</mi></mrow></msub><mo>+</mo><msub><mrow><mi mathvariant="bold-italic">ε</mi></mrow><mrow><mi mathvariant="bold-italic">i</mi></mrow></msub></mrow></math></span>, with <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq32"><math id="d33e1747"><msub><mrow><mi mathvariant="bold-italic">y</mi></mrow><mrow><mi mathvariant="bold-italic">i</mi></mrow></msub></math></span> the vector of phenotype for genotypes tested in the environment <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq33"><math id="d33e1762"><mi>i</mi></math></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq34"><math id="d33e1771"><mi>Z</mi></math></span> is the marker matrix, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq35"><math id="d33e1780"><mi>u</mi></math></span> is the vector marker effects, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq36"><math id="d33e1790"><mrow><mrow><mi mathvariant="bold-italic">u</mi></mrow><mo>∼</mo><mi>N</mi><mfenced close=")" open="("><mn>0</mn><mo>,</mo><mi>I</mi><msubsup><mi>σ</mi><mrow><mi>u</mi></mrow><mn>2</mn></msubsup></mfenced></mrow></math></span>, with <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq37"><math id="d33e1813"><msubsup><mi>σ</mi><mrow><mi>u</mi></mrow><mn>2</mn></msubsup></math></span> the genetic variance, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq38"><math id="d33e1826"><msub><mrow><mi mathvariant="bold-italic">ε</mi></mrow><mrow><mi mathvariant="bold-italic">i</mi></mrow></msub></math></span> and the residuals in that environment <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq39"><math id="d33e1841"><mi>i</mi></math></span>. The normality of the residuals has been checked, for each scenario, trait and testing environment, 76% and 77% these combinations had residuals with a Shapiro–Wilk test p-value superior to 0.05 for DIS and VSK traits, respectively. Marker effects from all environments were combined. Then, for each marker, their effects were regressed against the environment index so that the marker effects could be predicted for a new environment based on the environment index. Then, the predicted phenotypic value for an unknown genotype and environment becomes: <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq40"><math id="d33e1850"><mrow><mover accent="true"><msub><mrow><mi mathvariant="bold-italic">y</mi></mrow><mrow><mi mathvariant="bold-italic">j</mi></mrow></msub><mo stretchy="true">^</mo></mover><mo>=</mo><mi>α</mi><mo>+</mo><mi>Z</mi><mover accent="true"><msub><mrow><mi mathvariant="bold-italic">u</mi></mrow><mrow><mi mathvariant="bold-italic">j</mi></mrow></msub><mo stretchy="true">^</mo></mover></mrow></math></span>, with <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq41"><math id="d33e1883"><mover accent="true"><msub><mrow><mi mathvariant="bold-italic">y</mi></mrow><mrow><mi mathvariant="bold-italic">j</mi></mrow></msub><mo stretchy="true">^</mo></mover></math></span> the predicted phenotypic value for the testing set genotypes in testing environment <em>j</em>, α the overall intercept, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq42"><math id="d33e1905"><mover accent="true"><msub><mrow><mi mathvariant="bold-italic">u</mi></mrow><mrow><mi mathvariant="bold-italic">j</mi></mrow></msub><mo stretchy="true">^</mo></mover></math></span> the predicted marker effect for the environment <em>j</em>.</p>
<p id="Par40">For comparison, genomic selection using the RR-BLUP method was also implemented, using only genomic data and without any ECs, with the same training and testing set partition as for the JGRA method.</p></section><section id="Sec10"><h4 class="pmc_sec_title">Bayesian mixed models</h4>
<p id="Par41">Following Jarquín et al. [<a href="#CR28" class="usa-link" aria-describedby="CR28">28</a>], we compared different prediction models, including a combination of genotype (G), environment (E), and genotype-by-environment (GxE) effects.</p>
<ul class="list" style="list-style-type:disc"><li><p id="Par42"><strong>Model G</strong></p></li></ul>
<p id="Par43">In that model, the genotype effect was included in the model, such as: <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq43"><math id="d33e1942"><mrow><mrow><mi mathvariant="bold-italic">y</mi></mrow><mo>=</mo><mn>1</mn><mi>μ</mi><mo>+</mo><mi>X</mi><mrow><mi mathvariant="bold-italic">β</mi></mrow><mo>+</mo><mrow><mi mathvariant="bold-italic">ε</mi></mrow></mrow></math></span>, with <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq44"><math id="d33e1966"><mrow><mi mathvariant="bold-italic">y</mi></mrow></math></span> the vector of phenotypes in several environments of dimension <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq45"><math id="d33e1977"><mrow><mi>o</mi><mo>×</mo><mn>1</mn></mrow></math></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq46"><math id="d33e1989"><mi>o</mi></math></span> being the number of observations, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq47"><math id="d33e1998"><mi>μ</mi></math></span> the overall mean, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq48"><math id="d33e2008"><mrow><mn>1</mn></mrow></math></span> a vector of 1’s of dimension <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq49"><math id="d33e2018"><mrow><mi>o</mi><mo>×</mo><mn>1</mn></mrow></math></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq50"><math id="d33e2030"><mrow><mi>X</mi><mo>=</mo><msub><mi>Z</mi><mi>G</mi></msub><mo>×</mo><mi>M</mi></mrow></math></span>, with <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq51"><math id="d33e2046"><msub><mi>Z</mi><mi>G</mi></msub></math></span> the design matrix that connects the observation to the genotypes of dimension <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq52"><math id="d33e2057"><mrow><mi>o</mi><mo>×</mo><mi>n</mi></mrow></math></span>, with <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq53"><math id="d33e2069"><mi>n</mi></math></span> the number of genotypes, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq54"><math id="d33e2079"><mi>M</mi></math></span> the scaled SNP matrix of dimension <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq55"><math id="d33e2088"><mrow><mi>n</mi><mo>×</mo><mi>m</mi></mrow></math></span>, with <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq56"><math id="d33e2100"><mi>m</mi></math></span> the number of SNP markers, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq57"><math id="d33e2109"><mrow><mi mathvariant="bold-italic">β</mi></mrow></math></span> the unknown vector of random marker effects, of dimension <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq58"><math id="d33e2120"><mrow><mi>p</mi><mo>×</mo><mn>1</mn></mrow></math></span>, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq59"><math id="d33e2132"><mrow><mi mathvariant="bold-italic">ε</mi></mrow></math></span> the vector of residuals, assumed independent, identically distributed, and following a normal distribution with a variance of <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq60"><math id="d33e2144"><msubsup><mi>σ</mi><mrow><mi>ε</mi></mrow><mn>2</mn></msubsup></math></span>. The effects were predicted using 10,000 burn-ins and 200,000 iterations, with the BGLR R package [<a href="#CR45" class="usa-link" aria-describedby="CR45">45</a>].</p>
<ul class="list" style="list-style-type:disc"><li><p id="Par44"><strong>Model G + E</strong></p></li></ul>
<p id="Par45">In that model, the environment effect was added, such as: <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq61"><math id="d33e2167"><mrow><mrow><mi mathvariant="bold-italic">y</mi></mrow><mo>=</mo><mn>1</mn><mi>μ</mi><mo>+</mo><mi>X</mi><mrow><mi mathvariant="bold-italic">β</mi></mrow><mo>+</mo><mi>S</mi><mrow><mi mathvariant="bold-italic">γ</mi></mrow><mo>+</mo><mrow><mi mathvariant="bold-italic">ε</mi></mrow></mrow></math></span>. <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq62"><math id="d33e2197"><mrow><mtext>S</mtext><mo>=</mo><msub><mi>Z</mi><mi>E</mi></msub><mo>×</mo><mtext>W</mtext></mrow></math></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq63"><math id="d33e2213"><msub><mi>Z</mi><mi>E</mi></msub></math></span> is the design matrix that connects the observations to the environments of dimension <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq64"><math id="d33e2224"><mrow><mi>o</mi><mo>×</mo><mi>t</mi></mrow></math></span> with <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq65"><math id="d33e2236"><mi>t</mi></math></span> the number of environments, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq66"><math id="d33e2246"><mi>W</mi></math></span> is the matrix of ECs in each environment (dimension <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq67"><math id="d33e2255"><mrow><mi>t</mi><mo>×</mo><mi>k</mi></mrow></math></span>), and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq68"><math id="d33e2267"><mrow><mi mathvariant="bold-italic">γ</mi></mrow></math></span> is the unknown random environment effects.</p>
<ul class="list" style="list-style-type:disc"><li><p id="Par46"><strong>Model G + GxE</strong></p></li></ul>
<p id="Par47">In that model, the interaction between the genotype and the environment was added, and the effects were predicted with the reproducible kernel Hilbert space (RKHS) method using a linear kernel, following Crossa et al. [<a href="#CR15" class="usa-link" aria-describedby="CR15">15</a>] and with the G effect as previously described. The G + GxE model is defined as: <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq69"><math id="d33e2288"><mrow><mrow><mi mathvariant="bold-italic">y</mi></mrow><mo>=</mo><mn>1</mn><mi>μ</mi><mo>+</mo><mi>X</mi><mrow><mi mathvariant="bold-italic">β</mi></mrow><mo>+</mo><mi>g</mi><mi>w</mi><mo>+</mo><mrow><mi mathvariant="bold-italic">ε</mi></mrow></mrow></math></span>, with <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq70"><math id="d33e2316"><mrow><mi mathvariant="italic">gw</mi></mrow></math></span> being the interaction between the genotype and the environment, with <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq71"><math id="d33e2327"><mrow><mi>g</mi><mi>w</mi><mo>∼</mo><mi>N</mi><mfenced close=")" open="("><mn>0</mn><mo>,</mo><mfenced close="]" open="["><msup><mrow><mo stretchy="false">(</mo><msub><mi>Z</mi><mi>G</mi></msub><mi>G</mi><msub><mrow><msup><mrow><mi>Z</mi></mrow><mo>′</mo></msup></mrow><mi>G</mi></msub><mo stretchy="false">)</mo></mrow><mo>∘</mo></msup><mrow><mo stretchy="false">(</mo><msub><mi>Z</mi><mi>E</mi></msub><mi mathvariant="normal">Ω</mi><msub><mrow><msup><mrow><mi>Z</mi></mrow><mo>′</mo></msup></mrow><mi>E</mi></msub><mo stretchy="false">)</mo></mrow></mfenced><msub><mrow><msup><mrow><mi>σ</mi></mrow><mn>2</mn></msup></mrow><mrow><mi mathvariant="italic">gw</mi></mrow></msub></mfenced></mrow></math></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq72"><math id="d33e2388"><mmultiscripts><mrow></mrow><mrow></mrow><mo>∘</mo></mmultiscripts></math></span> denotes the Hadamard product between the matrices, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq73"><math id="d33e2401"><mi>G</mi></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq74"><math id="d33e2410"><mi mathvariant="normal">Ω</mi></math></span> are the genomic and the environmental relationship matrices, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq75"><math id="d33e2420"><msub><mrow><msup><mrow><mi>σ</mi></mrow><mn>2</mn></msup></mrow><mrow><mi mathvariant="italic">gw</mi></mrow></msub></math></span> is the variance due to GxE.</p>
<ul class="list" style="list-style-type:disc"><li><p id="Par48"><strong>Model G + E + GxE</strong></p></li></ul>
<p id="Par49">In that model, all effects were included, such as <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq76"><math id="d33e2444"><mrow><mrow><mi mathvariant="bold-italic">y</mi></mrow><mo>=</mo><mn>1</mn><mi>μ</mi><mo>+</mo><mi>X</mi><mrow><mi mathvariant="bold-italic">β</mi></mrow><mo>+</mo><mi>S</mi><mrow><mi mathvariant="bold-italic">γ</mi></mrow><mo>+</mo><mi>g</mi><mi>w</mi><mo>+</mo><mrow><mi mathvariant="bold-italic">ε</mi></mrow></mrow></math></span>, with all the effects previously described.</p>
<p id="Par50">The four models (G, G + E, G + GxE, and G + E + GxE) were compared with different methods: Bayesian ridge regression (BRR), Bayesian Lasso (BL), BayesA, BayesB, BayesC, and RKHS for fitting the genotype and the environment effects, while RKHS was constantly used to fit the GxE effect. For the RKHS method, the models were changed to accommodate for incorporating of the kernels corresponding to each of the effects. The BayesB method was retained and used across all four models because it provided the highest predictive ability on average (data not shown). In BayesB, a small proportion SNPs have non-zero effects and their variance follow an inverse chi-square distribution [<a href="#CR39" class="usa-link" aria-describedby="CR39">39</a>].</p></section></section><section id="Sec11"><h3 class="pmc_sec_title">Prediction scenarios</h3>
<p id="Par51">Several prediction scenarios were compared, all for predicting untested (unknown) genotypes by GP. The different scenarios tested varied the relationship between the training and the testing environments. Genotypes were predicted for each environment separately. The data was split into train and test, using the following rules for each of the scenarios:</p>
<ul class="list" style="list-style-type:disc">
<li><p id="Par52">Known location/known year (knLoc.knYr): All other environments are included in the training set.</p></li>
<li><p id="Par53">New location/new year (nLoc.nYr): Environments from the same year or location as the testing environment were excluded.</p></li>
</ul>
<p id="Par54">For all the scenarios, the genotypes to be predicted were excluded from the training set. The determination of ECs for each wheat stage was conducted across all environments before defining the prediction scenario. Thus, the planting date and the dates of wheat stages based on GDD were used even for the ‘nLoc.nYr’ prediction scenario (see below).</p>
<p id="Par55">For the FW method and JGRA RN, the parameters were estimated in the training set and predicted by genomic prediction for the testing set. For the JGRA marker, the estimation of marker effects was done using environments and genotypes from the training set only.</p>
<p id="Par56">The within-environment predictive ability was calculated as the Pearson correlation between the observed and the predicted value over the testing set of genotypes and for each environment. Similarly, the across-environment predictive ability is the Pearson correlation calculated when aggregating predicted values across all testing sets, genotypes, and environments.</p></section><section id="Sec12"><h3 class="pmc_sec_title">Significance of markers from JGRA</h3>
<p id="Par57">For each environment, phenotypic data was scaled to a variance of one and marker effects were estimated using the JGRA marker effect method applied on all genotypes. Then, the marker effects were regressed against the mean phenotype (instead of the environment index determined by the ECs). Finally, the significance of the linear regression of the intercept and slope parameters was estimated. A Manhattan plot was produced from the significance of the intercept and the slope, and the Bonferroni threshold was calculated to account for the multi-testing, such as: <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq77"><math id="d33e2503"><mrow><mi>T</mi><mi>h</mi><mi>r</mi><mi>e</mi><mi>s</mi><mi>h</mi><mi>o</mi><mi>l</mi><mi>d</mi><mo>=</mo><mo>-</mo><msub><mrow><mi mathvariant="italic">log</mi></mrow><mn>10</mn></msub><mrow><mo stretchy="false">(</mo><mfrac><mrow><mn>0.05</mn></mrow><mi>m</mi></mfrac><mo stretchy="false">)</mo></mrow></mrow></math></span>, with <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq78"><math id="d33e2538"><mi>m</mi></math></span> the number of markers.</p>
<p id="Par58">The JGRA reaction norm was also fit using all available genotypes and using the mean phenotype. From the estimation of the intercept and the slope for each genotype, a genome-wide association study (GWAS) was performed using the MM4LMM method [<a href="#CR37" class="usa-link" aria-describedby="CR37">37</a>]. The normality of the residuals was checked with a QQ-plot. The GRM was used to account for the genetic structure. For DIS, 280 individuals were available, and 283 for VSK traits. The same Bonferroni threshold was used for determining significant associations from the Manhattan plot.</p></section></section><section id="Sec13"><h2 class="pmc_sec_title">Results</h2>
<section id="Sec14"><h3 class="pmc_sec_title">Phenotyping description</h3>
<p id="Par59">Phenotypic distributions showed variability among the locations and years, indicating a strong impact of the environment. Among the most tested locations, PRP (Prosper, North Dakota) displayed the highest level of FHB, whereas STP (St. Paul, Minnesota) had the lowest level of FHB (Fig. <a href="#Fig1" class="usa-link">1</a>C). The correlation between DIS and VSK traits was 0.68. The heritability could not be computed since only entry-mean values were available for a given trial. When extracting BLUPs through a mixed model using the same dataset, Brault et al. [<a href="#CR5" class="usa-link" aria-describedby="CR5">5</a>] obtained reliability values of 0.646 and 0.702 for DIS and VSK, respectively.</p>
<p id="Par60">The genetic variance accounted for 41.1% and 42.8% of the total variance for DIS and VSK traits, respectively, with slightly less residual variance for DIS (Figure S1). The environmental effect, encompassing location, year, and location-by-year interaction, summed up to 34% and 30% for DIS and VSK, respectively. The GxE component, encompassing the interaction between genotype with location or year, summed up to 5% and 3% for DIS and VSK, respectively (Figure S1).</p></section><section id="Sec15"><h3 class="pmc_sec_title">Environment description based on environmental covariates</h3>
<section id="Sec16"><h4 class="pmc_sec_title">Description and comparison of the environment indices</h4>
<p id="Par61">The set of ECs with the highest correlation used only location-specific ECs for VSK with R = 0.49 and V35 for DIS with R = 0.38 (Fig. <a href="#Fig2" class="usa-link">2</a>). Adding location variables always increased the accuracy of the model. Interestingly, the location-specific ECs alone had a good correlation for both traits (R = 0.35 for DIS). For the following analysis, we used the best set of ECs for each trait, i.e., V35 for DIS and location-specific ECs alone for VSK.</p>
<p id="Par62">The weights associated with each EC varied among the traits but the sign of the effects was consistent (Figure S2). The ECs affecting the most DIS trait were TOTC ( +), GWETROOT ( +), N (-), and ECEC (-). Lots of location-specific ECs were among the ECs with the highest absolute value. For the VSK trait, the selected set of ECs included only location-specific ECs, with TOTC ( +), SILT ( +), TCEQ (-), and LON (-) being the most significant variables. Signs of the EC weights were indicated with ( +) and (-) for positive and negative, respectively.</p>
<p id="Par63">The two environment indices from the PLSR and the sum of three ECs were compared with the JGRA methods. The results showed that the predictive ability values were mostly similar between the two indices, except for JGRA RN, with a superiority of the environment index from PLSR (Figure S3). Therefore, this environment index was kept for the following analyses.</p></section><section id="Sec17"><h4 class="pmc_sec_title">Description of the ERM matrix</h4>
<p id="Par64">The ERM showed a mild structure among the locations (Figure S4). As expected, some locations geographically close to each other were more related in the ERM, such as FRG (Fargo, ND, USA) and PRP (Prosper, ND, USA), as displayed in Fig. <a href="#Fig1" class="usa-link">1</a>B. Some locations displayed highly differentiated years, such as BRK (Brookings, SD, USA) and CRK (Crookston, MN, USA). Diagonal stripes were visible in the off-diagonal squares (between different locations), indicating higher relatedness of environments from the same year (Figure S4).</p></section></section><section id="Sec18"><h3 class="pmc_sec_title">Prediction methods results</h3>
<section id="Sec19"><h4 class="pmc_sec_title">Finlay-Wilkinson results</h4>
<ul class="list" style="list-style-type:disc"><li><p id="Par66"><strong>Within-environment results</strong></p></li></ul>
<p id="Par67">Predictive ability for the intercept parameter was almost twice as high compared to the predictive ability for the slope parameter with mean values of 0.78 and 0.83 for intercept, and 0.37 and 0.47 for slope, for DIS and VSK, respectively (Figure S5A). The predictive ability for predicting phenotypes was 0.46 and 0.50 for the two traits. There was a little effect of the prediction scenario, with a maximum decrease of predictive ability of 0.03.</p>
<ul class="list" style="list-style-type:disc"><li><p id="Par68"><strong>Across-environment results</strong></p></li></ul>
<p id="Par69">Predictive ability across all environments was smaller than in within-environment, with values close to 0.32 for DIS and VSK. There was a small, expected decrease in predictive ability values over the two scenarios of 0.015 for DIS and 0.04 for VSK (Figure S5B).</p></section><section id="Sec20"><h4 class="pmc_sec_title">JGRA results</h4>
<ul class="list" style="list-style-type:disc"><li><p id="Par71"><strong>Within-environment results</strong></p></li></ul>
<p id="Par72">Predictive ability values within environments were highest for the JGRA marker effect (average of 0.59), and lowest for JGRA RN (average of 0.39), and intermediate for GS (average of 0.56). All the pairwise comparisons between the methods were highly significant (p-value &lt; 0.01) based on a <em>t-test</em>, except for the comparison between GS and JGRA marker effect for VSK trait (Fig. <a href="#Fig3" class="usa-link">3</a>A). Predictive ability was higher for VSK than for DIS (mean predictive ability of 0.59 and 0.43). The range of predictive ability values was very large for all methods, from -0.30 to 0.86 for JGRA RN for the “knLoc.knYr” scenario and DIS trait, and from − 0.14 to 0.98 for JGRA marker effect for the “knLoc.knYr” scenario and DIS trait.</p>
<ul class="list" style="list-style-type:disc"><li><p id="Par73"><strong>Across-environment results</strong></p></li></ul>
<figure class="fig xbox font-sm" id="Fig3"><h5 class="obj_head">Fig. 3.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12366029_13007_2025_1418_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/21f2/12366029/2e69902336ee/13007_2025_1418_Fig3_HTML.jpg" loading="lazy" id="MO3" height="666" width="797" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Comparison of predictive ability for JGRA methods. <strong>A</strong>: Within-environment predictive ability for disease index (DIS) and visual scabby kernel (VSK) traits, three methods and two prediction scenarios. Paired <em>t-tests</em> were done to compare the significance of the difference of predictive ability between the methods (with the two scenarios), “***” indicates a p-value below 0.001, and “NS.” a p-value above 0.1. The cross represents the mean predictive ability, while the points are the individual predictive ability in each environment.<strong> B</strong>: Across-environment predictive ability for the two traits, three methods, and two prediction scenarios</p></figcaption></figure><p id="Par74">The predictive ability across environments was smaller than in within-environment, with a smaller range (Fig. <a href="#Fig3" class="usa-link">3</a>B). Otherwise, similar trends were observed, with the same ranking between the methods, traits, and scenarios. For example, for DIS and the “knLoc.knYr” scenario, predictive ability values were 0.380, 0.330, and 0.181 for JGRA marker effect, GS, and JGRA RN, respectively. Interestingly, there was a smaller difference in predictive ability between the two scenarios for the JGRA marker effect than for GS, indicating that the former did comparatively better in the most challenging scenario.</p></section><section id="Sec21"><h4 class="pmc_sec_title">Mixed model results</h4>
<p id="Par75">Similar trends were observed between the two traits for the mixed model results (Fig. <a href="#Fig4" class="usa-link">4</a>). As expected, there was a decrease in the predictive ability between the two scenarios, but the magnitude of decrease was a bit more pronounced in across-environment prediction (Fig. <a href="#Fig4" class="usa-link">4</a>B), with a decrease up to 0.185 for DIS and G + E + GxE, compared to a maximum decrease of 0.017 in within-environment prediction for the G model and DIS trait.</p>
<figure class="fig xbox font-sm" id="Fig4"><h5 class="obj_head">Fig. 4.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12366029_13007_2025_1418_Fig4_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/21f2/12366029/a6d84f3e655c/13007_2025_1418_Fig4_HTML.jpg" loading="lazy" id="MO4" height="707" width="797" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Comparison of predictive ability for mixed models. Mixed model results for the BayesB method for two traits (DIS: disease index and VSK: visual scabby kernel), two prediction scenarios (known location and known year, new location and new year), and four model types. <strong>A</strong>: within-environment predictive ability, <strong>B</strong>: across-environment predictive ability. Paired <em>t-tests</em> were conducted to compare the significance of the difference of predictive ability between the methods when including or not including GxE in the model, “***” indicates a p-value below 0.001, “**” indicates a p-value below 0.05, and “*” indicates a p-value below 0.1</p></figcaption></figure><p id="Par76">Interestingly, models that included the environment effect (i.e., G + E and G + E + GxE) were superior in within-environment, and G + E was the best among those two, with an average predictive ability of 0.576 and 0.647 for DIS and VSK traits, respectively (Fig. <a href="#Fig4" class="usa-link">4</a>A). In across-environment, the model types that didn’t include the environment effect (G and G + GxE) were the best, especially for the new location and new year scenario (Fig. <a href="#Fig4" class="usa-link">4</a>B).</p></section></section><section id="Sec22"><h3 class="pmc_sec_title">Comparison of methods</h3>
<p id="Par77">For the comparison of methods, the best feature for each method was used, so JGRA marker effect was selected for JGRA. For mixed models, in within-environment, the G + E model was selected, and the G + GxE model was selected in across-environment. Results showed that JGRA and mixed model were the best models in within-environment, with non-significant differences for both traits (Fig. <a href="#Fig5" class="usa-link">5</a>A).</p>
<figure class="fig xbox font-sm" id="Fig5"><h4 class="obj_head">Fig. 5.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12366029_13007_2025_1418_Fig5_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/21f2/12366029/44cb3026a45d/13007_2025_1418_Fig5_HTML.jpg" loading="lazy" id="MO5" height="710" width="797" alt="Fig. 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Comparison of predictive ability for FW, JGRA, and mixed models. Comparison of three prediction models: FW: Finlay-Wilkinson regression, using an environment index, JGRA: joint genomic regression analysis, with the marker effect type of model, and mixed model using the BayesB method, and G + E + GxE model type in within-environment (<strong>A</strong>), and G + GxE model type in across-environment (<strong>B</strong>)</p></figcaption></figure><p id="Par78">The ranking of methods was the same both within- and across-environments, with the lowest predictive ability for FW, followed by mixed models and JGRA (Fig. <a href="#Fig5" class="usa-link">5</a>). In within-environment, FW was significantly lower than the other two methods, although there was no significant difference between JGRA and the mixed model. Predictive ability was little affected by the trait or the prediction scenario (Fig. <a href="#Fig5" class="usa-link">5</a>A). In across-environment, FW had the lowest predictive ability, followed by mixed models and JGRA. Specifically, there was a stronger drop of predictive ability when predicting new location and year in mixed models, than in JGRA (Fig. <a href="#Fig5" class="usa-link">5</a>B). For example, for DIS trait, in across-environment and for new location &amp; new year scenario, predictive ability was of 0.362 and 0.244 for JGRA and mixed models, respectively. For JGRA and mixed models, VSK was slightly better predicted than the DIS trait, although it was the opposite for the FW method.</p></section><section id="Sec23"><h3 class="pmc_sec_title">JGRA marker significance</h3>
<p id="Par79">When refitting the JGRA marker effect method, marker effects were regressed against the mean phenotype. From that, many markers were highly significant for the intercept parameters, with -log<sub>10</sub> p-values up to 19.7 and 12.7 for VSK and DIS, respectively (Fig. <a href="#Fig6" class="usa-link">6</a>).</p>
<figure class="fig xbox font-sm" id="Fig6"><h4 class="obj_head">Fig. 6.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12366029_13007_2025_1418_Fig6_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/21f2/12366029/ea3fcf4013dc/13007_2025_1418_Fig6_HTML.jpg" loading="lazy" id="MO6" height="749" width="797" alt="Fig. 6"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Regression of the marker effects of two traits for the intercept and slope parameters. Regression plot of the most significant markers of intercept (<strong>A</strong>&amp;<strong>C</strong>) and slope (<strong>B</strong>&amp;<strong>D</strong>) parameters for DIS (<strong>A</strong>&amp;<strong>B</strong>) and VSK (<strong>B</strong>&amp;<strong>D</strong>) traits from the JGRA marker effect. The name of the single nucleotide polymorphisms (SNP) with the highest -log<sub>10</sub> p-value were displayed in each subplot. Each dot represents a marker effect estimated in a different environment. The table on the left of each subplot summarizes the linear regression analysis of the marker effect against the environment index (corresponding here to the mean phenotype). The label of the y-axis represents the marker name with the chromosome number and its physical position</p></figcaption></figure><p id="Par80">For the slope parameter, the significance levels were smaller.</p>
<p id="Par81">The most significant SNP for VSK and intercept parameter was located on chromosome 6D, at 2.4 Mbp (megabase pair). The most significant SNP for VSK trait and slope parameter (-log<sub>10</sub> p-value = 6.73) was located on chromosome 6D at position 2.03 Mbp. These markers were close and for both of them, the intercept and the slope were significant. The most significant SNP for DIS and intercept parameter was located on chromosome 5B, at 579 Mbp (-log<sub>10</sub> p-value = 12.7). For the slope, the most significant marker was located on chromosome 4A, at 17.1 Mbp (-log<sub>10</sub> p-value = 4.24) was located on chromosome 6D at position 2.03 Mbp. A significant slope means that the estimated marker effect was more positive (or negative) as the mean phenotype increased (environment with more FHB pressure).</p>
<p id="Par82">One <em>Fhb1</em> QTL (IFA-FM227, located on chromosome 3B, at 13.8 Mbp) included in the genotypic array had a significant effect on VSK and DIS intercepts, with a -log<sub>10</sub> p-value of 9.21 and 6.39, respectively.</p>
<p id="Par83">A regular GWAS was applied on the intercept and slope parameters estimated using the JGRA RN method. The results in Figure S6 showed that the significance levels were much lower, with only one marker reaching the Bonferroni significance threshold for VSK and the intercept parameter for a marker located on chromosome 3D at position 352 Mbp (Figure S6C). The ranges of the p-values were roughly the same for both traits and parameters (Figure S6).</p></section><section id="Sec24"><h3 class="pmc_sec_title">Prediction of best genotypes in each location</h3>
<p id="Par84">The JGRA marker effect was fitted using the whole population, and an environment index composed only of location-specific ECs was used to predict the genomic estimated breeding values (GEBVs) for each location specifically (Fig. <a href="#Fig7" class="usa-link">7</a>). From that, the breeders can select the most adapted genotypes in each location. In Fig. <a href="#Fig7" class="usa-link">7</a>, some genotypes still display a strong variability between the locations. For example, ‘PI382161' was the best genotype for LGD location (Langdon, North Dakota, USA) with the lowest predicted DIS and VSK genotypic value but was not among the 50% best genotypes for PRP location (Prosper, North Dakota, USA) (Fig. <a href="#Fig7" class="usa-link">7</a>). Interestingly, ‘ND2710’ and ‘BACUP’ were among the best genotypes both for having low DIS and VSK and were both among the resistant checks in the URSN population. The ‘ND2710’ genotype was released in 1998 and has in its pedigree both ‘SUMAI 3’, which is a Chinese cultivar as a source of the genetic resistance to FHB, and ‘WHEATON’, which is one of the susceptible checks tested in this nursery.</p>
<figure class="fig xbox font-sm" id="Fig7"><h4 class="obj_head">Fig. 7.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12366029_13007_2025_1418_Fig7_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/21f2/12366029/a91a1bcd6e86/13007_2025_1418_Fig7_HTML.jpg" loading="lazy" id="MO7" height="791" width="797" alt="Fig. 7"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig7/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Genomic estimated breeding values (GEBVs) predicted for all genotyped individuals in each location. An environment index was used from a partial least square regression model based on location-specific environmental covariates. The genotypes that were among the best in a given location were highlighted and the genotype names were indicated on the right part of the figure. <strong>A: disease index trait, B:</strong> visual scabby kernel trait.</p></figcaption></figure></section></section><section id="Sec25"><h2 class="pmc_sec_title">Discussion</h2>
<p id="Par85">This study provided a comprehensive analysis of the gain brought by ECs in the prediction of G and GxE for two FHB traits. The comparison of three methods, two scenarios, and two traits indicated that the JGRA marker effect was the best method overall and was significantly better than GS, which didn’t incorporate ECs. Predictive ability remained quite stable across the traits and prediction scenarios implemented, suggesting that JGRA was a robust method for a wide range of implementation. Many significant marker-trait associations were found from the output of the JGRA marker effect for both intercept and slope parameters. To our knowledge, these results are the first peer-reviewed attempt to predict GxE based on ECs focusing on a plant disease-related trait.</p>
<section id="Sec26"><h3 class="pmc_sec_title">Scenarios implemented</h3>
<p id="Par86">In the GxE literature, prediction scenarios are traditionally referred to as CV1 and CV2 for predicting untested and tested genotypes in a tested environment, respectively [<a href="#CR9" class="usa-link" aria-describedby="CR9">9</a>]. Unlike other publications, CV2 is used as our baseline scenario (“KnLoc.knYr”). The scenario of unknown year and location (“nLoc.nYr”) is more challenging because the environments used in the training set are less related to the validation set. Few publications have implemented this type of prediction scenario [<a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>].</p>
<p id="Par87">Other scenarios could have been compared, such as excluding the location of the testing set environment from the training set or similarly excluding the year. Given the experimental design, these scenarios would yield similar results to the scenarios tested here, because most of the genotypes were tested for only one year. Thus, the base scenario tested “knLoc.knYr” is already excluding almost all observations from the year to be predicted.</p>
<p id="Par88">Another alternative would have been to include observations from the testing environment in the training set, but its applicability was limited in this dataset because of the small number of observations in each environment. Then, subsampling some individuals in the testing set environment to be included in the training set further reduced the size of the testing set, which was on average 18 individuals.</p>
<p id="Par89">Predicting unknown genotypes was the common factor of the prediction scenarios tested, as we are seeking methods applicable to breeding programs.</p></section><section id="Sec27"><h3 class="pmc_sec_title">Incorporation of GxE in the prediction model</h3>
<section id="Sec28"><h4 class="pmc_sec_title">Relative importance of ECs</h4>
<p id="Par90">The set of ECs with the highest predictability of the mean phenotype was not the one around the heading and anthesis stage, i.e., V67 (Fig. <a href="#Fig2" class="usa-link">2</a>). Location-specific environmental covariates alone provided a fair correlation with the mean phenotype for a given environment using PLSR (Fig. <a href="#Fig2" class="usa-link">2</a>). This means that unpredictable yearly variations were proportionally less important to determine the level of FHB in a given environment. Thus, the prediction of the genotypic value based on GxE will not depend on the year. Nevertheless, these results were related to the context of inoculated and misted trials, where high precipitation and humidity were ensured. The total level of FHB in a given environment will surely be more variable from year to year in a regular trial. In that sense, this study differs from most of the other GxE studies, focused on yield. Indeed, yield was expected to have much stronger environmental variation and GxE than disease resistance under semi-controlled trials. This was consistent with the variance component analysis, where a small proportion of the variation was due to year or genotype-by-year interaction (Figure S1).</p>
<p id="Par91">Including only biologically meaningful ECs to constitute the environment index slightly decreased the predictive ability of JGRA RN, mostly for VSK in within-environment and for both traits in across-environment (Figure S3). Besides, the environment index based on the sum of three ECs did not correlate with the mean phenotype (data not shown), yet the predictive ability was close to that when maximizing the predictability of the mean phenotype value through PLSR. Then, the correlation between the environment index and the mean phenotype is not a good indicator of the suitability of a given environment index. Further studies are required to provide more explanation on that.</p></section><section id="Sec29"><h4 class="pmc_sec_title">Types of modelling integrating ECs</h4>
<p id="Par92">When using mixed models, the environment and GxE effects were incorporated using a matrix containing all ECs. Conversely, the environment index provided a single synthetic value to represent the environment quality. This latter strategy seems to be more powerful in predicting the relative performance across environments. Although the ECs could have been weighed when computing the W matrix, based on the weights estimated by PLSR when determining the environment index.</p>
<p id="Par93">Weather data at the heading and anthesis stages can play an important role in the disease spreading. Nevertheless, this effect was minimized by the experimental design, which ensured a rather consistent inoculum for all genotypes, whatever their phenology stage. Besides, neither heading nor anthesis dates were available at the genotype level for most of the environments.</p>
<p id="Par94">A model has been developed to predict the probability of an FHB epidemics, based on weather data around anthesis to help wheat growers determine if a fungicide application is needed [<a href="#CR51" class="usa-link" aria-describedby="CR51">51</a>, <a href="#CR52" class="usa-link" aria-describedby="CR52">52</a>]. In this tool, the user selects the level of resistance to FHB of their genotype, but the model doesn’t explicitly model or predict GxE.</p></section></section><section id="Sec30"><h3 class="pmc_sec_title">Comparison of the methods</h3>
<p id="Par95">In JGRA RN, estimation of intercept and slope is expected to be less accurate because on average, 4.5 observations were available for a given genotype. However, when combining all intercept and slope estimates across all training set genotypes, it constitutes a good training set size for genomic prediction (between 262 to 264 genotypes depending on the scenario and trait). Then, we expect genomic prediction to be accurate, depending on the heritability of the trait.</p>
<p id="Par96">In the JGRA marker effect, the first genomic prediction step was based on a few observations (average number of observations per environment is 18.2 and 18.6 for DIS and VSK, respectively). Thus, the prediction of marker effects is expected to be less accurate. However, the number of points for the regression of marker effects against the environment index was 107 and 109 for DIS and VSK, respectively, so this regression is expected to be more accurate. When comparing the environment indices (Figure S3), it was clear that the choice of the environment index had a much larger impact on JGRA RN than on JGRA marker effect, for which the predictive ability remained stable. For JGRA RN, a regression of the phenotype against the environment index is fitted for each genotype. Then, the parameters of this regression were highly affected by the choice of the environment index because there was a small number of environments in which each genotype was tested. On the other hand, for the JGRA marker effect, the environment index was used for the regression of marker effects, with more than 100 observations in our experimental design; this probably made the JGRA marker effect less sensitive to the choice of the environment index. Then, the JGRA marker effect should be preferred if a large number of environments are available in the dataset.</p>
<p id="Par97">Li et al. [<a href="#CR33" class="usa-link" aria-describedby="CR33">33</a>] observed no difference between the JGRA RN and JGRA marker effect methods, probably because the experimental design was balanced, with all genotypes tested in all environments. This was definitely not the case in our study, which highlights the advantage of each method depending on the experimental design.</p>
<p id="Par98">For the mixed models results, it was observed that including an environment effect in the model increased the predictive ability in within-environment, i.e., when the correlation was calculated separately for each environment (Fig. <a href="#Fig4" class="usa-link">4</a>A). Although it seems counterintuitive, we hypothesize that it is explained by a better fitting of the model in the training set.</p></section><section id="Sec31"><h3 class="pmc_sec_title">Main takeaway for breeding</h3>
<p id="Par99">Overall, adding GxE in the prediction model consistently improved the predictive ability by a small amount, despite the small percentage of variance explained by GxE (Figure S1). This might be explained by the fact that the trials were inoculated and misted, reducing the impact of the environment. This was consistent with less significant marker effects for the slope parameter than the intercept (Fig. <a href="#Fig6" class="usa-link">6</a>). Nevertheless, in the predictions, the ranking of the genotypes varied a lot among the locations (Fig. <a href="#Fig7" class="usa-link">7</a>). Indeed, the proportion of variance for GxE does not quantify the amount of GxE for each genotype.</p>
<p id="Par100">The prediction scenario “nLoc.nYr” included <em>in season</em> weather data and information about wheat stages that would not be known at the onset of a growing season. In that sense, the environment was not truly unknown, which could contribute to overestimating predictive ability. That is why the prediction of the best genotypes in each location was done using only location-specific ECs (Fig. <a href="#Fig7" class="usa-link">7</a>). An alternative is to fit the genomic prediction model with historical weather data, but it is rarely done in the literature [<a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>, <a href="#CR43" class="usa-link" aria-describedby="CR43">43</a>].</p></section><section id="Sec32"><h3 class="pmc_sec_title">Comparison with other studies that tested the usefulness of including GxE</h3>
<p id="Par101">In our study, we showed a significant moderate improvement in predictive ability when including GxE in the prediction model, i.e., the JGRA marker effect was better than GS (Fig. <a href="#Fig3" class="usa-link">3</a>), and mixed models that include GxE were significantly better for both traits (Fig. <a href="#Fig4" class="usa-link">4</a>). Depending on the study and the method used, the same or different conclusions were obtained in other publications. Most of the studies on GxE were focused on traits such as yield and heading date, which are more prone to show GxE than disease resistance. We could not find any study on disease-related traits comparable to what we have conducted here.</p>
<p id="Par102">Several studies observed no or minor improvement in predictive ability when including ECs [<a href="#CR24" class="usa-link" aria-describedby="CR24">24</a>, <a href="#CR43" class="usa-link" aria-describedby="CR43">43</a>, <a href="#CR47" class="usa-link" aria-describedby="CR47">47</a>]. On the contrary, Costa-Neto, Fritsche-Neto, et al. [<a href="#CR11" class="usa-link" aria-describedby="CR11">11</a>, <a href="#CR12" class="usa-link" aria-describedby="CR12">12</a>] and <a href="#CR27" class="usa-link" aria-describedby="CR27">27</a>] observed an increase in predictive ability. Ly et al. [<a href="#CR38" class="usa-link" aria-describedby="CR38">38</a>] computed GxE as a random reaction norm over a few ECs, they observed that the ECs were explaining a significant part of the GxE variance, although the gain in prediction was modest. Piepho and Blancon [<a href="#CR46" class="usa-link" aria-describedby="CR46">46</a>] and Tadese et al. [<a href="#CR57" class="usa-link" aria-describedby="CR57">57</a>] used methodologies comparable to the JGRA RN method. They both found a significant increase in predictive ability when including one or two synthetic covariates.</p>
<p id="Par103">Compared to previous studies, our across-environment predictive ability was much lower. We hypothesize that it might be due to the lower environmental variability for the type of traits we focused on.</p>
<p id="Par104">Li et al. [<a href="#CR32" class="usa-link" aria-describedby="CR32">32</a>] was one of the only studies that incorporated GxE to decipher the genetic architecture of the GxE components of trait variation by computing a GWAS on the intercept and slope parameters from the JGRA RN method. They found several already well-known loci for flowering time trait, however, they didn’t uncover new loci by harnessing the marker effects from JGRA, as we did in Fig. <a href="#Fig6" class="usa-link">6</a>. The p-values are arising from a linear regression between marker effects and the mean phenotype, thus, they are not comparable to a GWAS. We hypothesized that since marker effects were estimated using the RR-BLUP method, all markers had a non-zero effect, potentially inflating the significance level of the intercept parameter.</p>
<p id="Par105">This study represents the first attempt to compare three major types of methods for modelling GxE in plant breeding. For the first time, disease resistance traits were predicted using ECs, in challenging prediction scenarios in which both the genotype and the environment were untested.</p></section></section><section id="Sec33"><h2 class="pmc_sec_title">Supplementary Information</h2>
<section class="sm xbox font-sm" id="MOESM1"><div class="media p"><div class="caption">
<a href="/articles/instance/12366029/bin/13007_2025_1418_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary material 1.</a><sup> (2.1MB, pdf) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>We thank Dave Garvin, Susan Reynolds, Jochum Wiersma, Danielle Fiebelkorn-Wrucke, Sarah Blecha, Nate Stuart, Jeannie Gilbert, Brent McCallum, and Maria Antonia Henriquez for conducting the phenotyping experiments of the URSN population. We thank Mary Osenga and the USDA-ARS North Central Small Grains Genotyping Laboratory in Fargo for performing DNA extraction and genotyping. We thank Erick De Wolf for his input on the computation of the environment index. We thank Aaron Lorenz for his useful feedback.</p></section><section id="notes1"><h2 class="pmc_sec_title">Author contributions</h2>
<p>C.B: Formal analysis, investigation, methodology, visualization, writing original draft and editing. E.C: Data curation, formal analysis, validation, draft review. A.R: Project administration, resources, draft review. A.G: Data curation, validation, draft review. K.G: Data curation, validation, draft review. J.C: Data curation, validation, draft review. H.G: Investigation, methodology, validation, draft review. J.F: Conceptualization, funding acquisition, project administration, supervision, validation, draft review. J.A: Conceptualization, data curation, formal analysis, funding acquisition, project administration, supervision, draft review.</p></section><section id="notes2"><h2 class="pmc_sec_title">Funding</h2>
<p>This research has been funded by the US Wheat and Barley Scab Initiative Project FY22-SP-007 through the USDA-ARS CRIS project 3060-21000-046-000D.</p></section><section id="notes3"><h2 class="pmc_sec_title">Data availability</h2>
<p>Phenotypic data from all URSN trials are available on the T3 database (<a href="https://wheat.triticeaetoolbox.org/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://wheat.triticeaetoolbox.org/</a>). All scripts, intermediate files, result tables, and figures to reproduce the results are available in a public repository: 10.5061/dryad.wstqjq2z0.</p></section><section id="notes4"><h2 class="pmc_sec_title">Declarations</h2>
<section id="FPar1"><h3 class="pmc_sec_title">Ethics approval and consent to participate</h3>
<p id="Par106">Not applicable.</p></section><section id="FPar2"><h3 class="pmc_sec_title">Consent for publication</h3>
<p id="Par107">Not applicable.</p></section><section id="FPar3"><h3 class="pmc_sec_title">Competing interests</h3>
<p id="Par108">The authors declare no competing interests.</p></section></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="fn1">
<p><strong>Publisher's Note</strong></p>
<p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p>
</div></div></section><section id="_ci93_" lang="en" class="contrib-info"><h2 class="pmc_sec_title">Contributor Information</h2>
<p>Charlotte Brault, Email: charlotte.brault@live.com.</p>
<p>Jason D. Fiedler, Email: jason.fiedler@usda.gov</p></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Alemu A, Åstrand J, Montesinos-López OA, y Sánchez JI, Fernández-Gónzalez J, Tadesse W, Vetukuri RR, Carlsson AS, Ceplitis A, Crossa J, Ortiz R, Chawade A. Genomic selection in plant breeding: key factors shaping two decades of progress. Mol Plant. 2024;17(4):552–78. 10.1016/j.molp.2024.03.007.
</cite> [<a href="https://doi.org/10.1016/j.molp.2024.03.007" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38475993/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Alemu%20A,%20%C3%85strand%20J,%20Montesinos-L%C3%B3pez%20OA,%20y%20S%C3%A1nchez%20JI,%20Fern%C3%A1ndez-G%C3%B3nzalez%20J,%20Tadesse%20W,%20Vetukuri%20RR,%20Carlsson%20AS,%20Ceplitis%20A,%20Crossa%20J,%20Ortiz%20R,%20Chawade%20A.%20Genomic%20selection%20in%20plant%20breeding:%20key%20factors%20shaping%20two%20decades%20of%20progress.%20Mol%20Plant.%202024;17(4):552%E2%80%9378.%2010.1016/j.molp.2024.03.007." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>Anderson JA, Stack RW, Liu S, Waldron BL, Fjeld AD, Coyne C, Moreno-Sevilla B, Mitchell Fetch J, Song QJ, Cregan PB, Frohberg RC. DNA markers for fusarium head blight resistance QTLs in two wheat populations. Theor Appl Genet. 2001;102(8):1164–8. 10.1007/s001220000509.</cite> [<a href="https://scholar.google.com/scholar_lookup?Anderson%20JA,%20Stack%20RW,%20Liu%20S,%20Waldron%20BL,%20Fjeld%20AD,%20Coyne%20C,%20Moreno-Sevilla%20B,%20Mitchell%20Fetch%20J,%20Song%20QJ,%20Cregan%20PB,%20Frohberg%20RC.%20DNA%20markers%20for%20fusarium%20head%20blight%20resistance%20QTLs%20in%20two%20wheat%20populations.%20Theor%20Appl%20Genet.%202001;102(8):1164%E2%80%938.%2010.1007/s001220000509." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>Arruda MP, Brown PJ, Lipka AE, Krill AM, Thurber C, Kolb FL. Genomic selection for predicting fusarium head blight resistance in a wheat breeding program. Plant Genome. 2015. 10.3835/plantgenome2015.01.0003.
</cite> [<a href="https://doi.org/10.3835/plantgenome2015.01.0003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33228272/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Arruda%20MP,%20Brown%20PJ,%20Lipka%20AE,%20Krill%20AM,%20Thurber%20C,%20Kolb%20FL.%20Genomic%20selection%20for%20predicting%20fusarium%20head%20blight%20resistance%20in%20a%20wheat%20breeding%20program.%20Plant%20Genome.%202015.%2010.3835/plantgenome2015.01.0003." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Bernardo R. Prediction of maize single-cross performance using RFLPs and Information from related hybrids. Crop Sci. 1994. 10.2135/cropsci1994.0011183X003400010003x.</cite> [<a href="https://scholar.google.com/scholar_lookup?Bernardo%20R.%20Prediction%20of%20maize%20single-cross%20performance%20using%20RFLPs%20and%20Information%20from%20related%20hybrids.%20Crop%20Sci.%201994.%2010.2135/cropsci1994.0011183X003400010003x." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Brault C, Conley EJ, Green AJ, Glover KD, Cook JP, Gill HS, Read AC, Fiedler JD, Anderson JA. Leveraging historical trials to predict fusarium head blight resistance in spring wheat breeding programs. Plant Genom. 2025;18(1): e20559. 10.1002/tpg2.20559.</cite> [<a href="https://doi.org/10.1002/tpg2.20559" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11800066/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39912127/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Brault%20C,%20Conley%20EJ,%20Green%20AJ,%20Glover%20KD,%20Cook%20JP,%20Gill%20HS,%20Read%20AC,%20Fiedler%20JD,%20Anderson%20JA.%20Leveraging%20historical%20trials%20to%20predict%20fusarium%20head%20blight%20resistance%20in%20spring%20wheat%20breeding%20programs.%20Plant%20Genom.%202025;18(1):%20e20559.%2010.1002/tpg2.20559." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Browning BL, Tian X, Zhou Y, Browning SR. Fast two-stage phasing of large-scale sequence data. Am J Human Genet. 2021;108(10):1880–90. 10.1016/j.ajhg.2021.08.005.
</cite> [<a href="https://doi.org/10.1016/j.ajhg.2021.08.005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8551421/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34478634/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Browning%20BL,%20Tian%20X,%20Zhou%20Y,%20Browning%20SR.%20Fast%20two-stage%20phasing%20of%20large-scale%20sequence%20data.%20Am%20J%20Human%20Genet.%202021;108(10):1880%E2%80%9390.%2010.1016/j.ajhg.2021.08.005." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Browning BL, Zhou Y, Browning SR. A one-penny imputed genome from next-generation reference panels. Am J Human Genet. 2018;103(3):338–48. 10.1016/j.ajhg.2018.07.015.
</cite> [<a href="https://doi.org/10.1016/j.ajhg.2018.07.015" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6128308/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30100085/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Browning%20BL,%20Zhou%20Y,%20Browning%20SR.%20A%20one-penny%20imputed%20genome%20from%20next-generation%20reference%20panels.%20Am%20J%20Human%20Genet.%202018;103(3):338%E2%80%9348.%2010.1016/j.ajhg.2018.07.015." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>Buerstmayr H, Steiner B, Hartl L, Griesser M, Angerer N, Lengauer D, Miedaner T, Schneider B, Lemmens M. Molecular mapping of QTLs for fusarium head blight resistance in spring wheat. II. Resistance to fungal penetration and spread. Theor Appl Genet. 2003;107(3):503–8. 10.1007/s00122-003-1272-6.
</cite> [<a href="https://doi.org/10.1007/s00122-003-1272-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12768240/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Buerstmayr%20H,%20Steiner%20B,%20Hartl%20L,%20Griesser%20M,%20Angerer%20N,%20Lengauer%20D,%20Miedaner%20T,%20Schneider%20B,%20Lemmens%20M.%20Molecular%20mapping%20of%20QTLs%20for%20fusarium%20head%20blight%20resistance%20in%20spring%20wheat.%20II.%20Resistance%20to%20fungal%20penetration%20and%20spread.%20Theor%20Appl%20Genet.%202003;107(3):503%E2%80%938.%2010.1007/s00122-003-1272-6." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>Burgueño J, de los Campos G, Weigel K, Crossa J. Genomic prediction of breeding values when modeling genotype × environment interaction using pedigree and dense molecular markers. Crop Sci. 2012;52(2):707–19. 10.2135/cropsci2011.06.0299.</cite> [<a href="https://scholar.google.com/scholar_lookup?Burgue%C3%B1o%20J,%20de%20los%20Campos%20G,%20Weigel%20K,%20Crossa%20J.%20Genomic%20prediction%20of%20breeding%20values%20when%20modeling%20genotype%20%C3%97%20environment%20interaction%20using%20pedigree%20and%20dense%20molecular%20markers.%20Crop%20Sci.%202012;52(2):707%E2%80%9319.%2010.2135/cropsci2011.06.0299." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>Costa-Neto G, Crespo-Herrera L, Fradgley N, Gardner K, Bentley AR, Dreisigacker S, Fritsche-Neto R, Montesinos-López OA, Crossa J. Envirome-wide associations enhance multi-year genome-based prediction of historical wheat breeding data. G3 Genes|Genomes|Genetics. 2023;13(2):313. 10.1093/g3journal/jkac313.</cite> [<a href="https://doi.org/10.1093/g3journal/jkac313" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9911085/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36454213/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Costa-Neto%20G,%20Crespo-Herrera%20L,%20Fradgley%20N,%20Gardner%20K,%20Bentley%20AR,%20Dreisigacker%20S,%20Fritsche-Neto%20R,%20Montesinos-L%C3%B3pez%20OA,%20Crossa%20J.%20Envirome-wide%20associations%20enhance%20multi-year%20genome-based%20prediction%20of%20historical%20wheat%20breeding%20data.%20G3%20Genes%7CGenomes%7CGenetics.%202023;13(2):313.%2010.1093/g3journal/jkac313." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Costa-Neto G, Fritsche-Neto R, Crossa J. nonlinear kernels, dominance, and envirotyping data increase the accuracy of genome-based prediction in multi-environment trials. Heredity. 2021;126(1):92–106. 10.1038/s41437-020-00353-1.
</cite> [<a href="https://doi.org/10.1038/s41437-020-00353-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7852533/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32855544/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Costa-Neto%20G,%20Fritsche-Neto%20R,%20Crossa%20J.%20nonlinear%20kernels,%20dominance,%20and%20envirotyping%20data%20increase%20the%20accuracy%20of%20genome-based%20prediction%20in%20multi-environment%20trials.%20Heredity.%202021;126(1):92%E2%80%93106.%2010.1038/s41437-020-00353-1." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Costa-Neto G, Galli G, Carvalho HF, Crossa J, Fritsche-Neto R. EnvRtype: a software to interplay enviromics and quantitative genomics in agriculture. G3 Genes|Genomes|Genetics. 2021. 10.1093/g3journal/jkab040.
</cite> [<a href="https://doi.org/10.1093/g3journal/jkab040" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8049414/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33835165/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Costa-Neto%20G,%20Galli%20G,%20Carvalho%20HF,%20Crossa%20J,%20Fritsche-Neto%20R.%20EnvRtype:%20a%20software%20to%20interplay%20enviromics%20and%20quantitative%20genomics%20in%20agriculture.%20G3%20Genes%7CGenomes%7CGenetics.%202021.%2010.1093/g3journal/jkab040." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>Crossa J, Pérez P, Hickey J, Burgueño J, Ornella L, Cerón-Rojas J, Zhang X, Dreisigacker S, Babu R, Li Y, Bonnett D, Mathews K. Genomic prediction in CIMMYT maize and wheat breeding programs. Heredity. 2014;112(1):48–60. 10.1038/hdy.2013.16.
</cite> [<a href="https://doi.org/10.1038/hdy.2013.16" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3860161/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23572121/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Crossa%20J,%20P%C3%A9rez%20P,%20Hickey%20J,%20Burgue%C3%B1o%20J,%20Ornella%20L,%20Cer%C3%B3n-Rojas%20J,%20Zhang%20X,%20Dreisigacker%20S,%20Babu%20R,%20Li%20Y,%20Bonnett%20D,%20Mathews%20K.%20Genomic%20prediction%20in%20CIMMYT%20maize%20and%20wheat%20breeding%20programs.%20Heredity.%202014;112(1):48%E2%80%9360.%2010.1038/hdy.2013.16." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>Crossa J, Pérez-Rodríguez P, Cuevas J, Montesinos-López O, Jarquín D, de Los G, Campos JB, González-Camacho JM, Pérez-Elizalde S, Beyene Y, Dreisigacker S, Singh R, Zhang X, Gowda M, Roorkiwal M, Rutkoski J, Varshney RK. Genomic selection in plant breeding: methods, models, and perspectives. Trends Plant Sci. 2017;22(11):961–75. 10.1016/j.tplants.2017.08.011.
</cite> [<a href="https://doi.org/10.1016/j.tplants.2017.08.011" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28965742/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Crossa%20J,%20P%C3%A9rez-Rodr%C3%ADguez%20P,%20Cuevas%20J,%20Montesinos-L%C3%B3pez%20O,%20Jarqu%C3%ADn%20D,%20de%20Los%20G,%20Campos%20JB,%20Gonz%C3%A1lez-Camacho%20JM,%20P%C3%A9rez-Elizalde%20S,%20Beyene%20Y,%20Dreisigacker%20S,%20Singh%20R,%20Zhang%20X,%20Gowda%20M,%20Roorkiwal%20M,%20Rutkoski%20J,%20Varshney%20RK.%20Genomic%20selection%20in%20plant%20breeding:%20methods,%20models,%20and%20perspectives.%20Trends%20Plant%20Sci.%202017;22(11):961%E2%80%9375.%2010.1016/j.tplants.2017.08.011." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>Crossa J, Montesinos-López OA, Pérez-Rodríguez P, Costa-Neto G, Fritsche-Neto R, Ortiz R, Martini JWR, Lillemo M, Montesinos-López A, Jarquin D, Breseghello F, Cuevas J, Rincent R (2022) Genome and Environment Based Prediction Models and Methods of Complex Traits Incorporating Genotype × Environment Interaction. In Ahmadi N, Bartholomé J (Eds.), Complex Trait Prediction: Methods and Protocols (pp. 245–283). Springer US. 10.1007/978-1-0716-2205-6_9</cite>
</li>
<li id="CR16">
<span class="label">16.</span><cite>Eberhart SA, Russell WA. Stability parameters for comparing varieties. Crop Sci. 1966. 10.2135/cropsci1966.0011183X000600010011x.</cite> [<a href="https://scholar.google.com/scholar_lookup?Eberhart%20SA,%20Russell%20WA.%20Stability%20parameters%20for%20comparing%20varieties.%20Crop%20Sci.%201966.%2010.2135/cropsci1966.0011183X000600010011x." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>Endelman JB. Ridge Regression and other kernels for genomic selection with R package rrBLUP. Plant Genome. 2011. 10.3835/plantgenome2011.08.0024.</cite> [<a href="https://scholar.google.com/scholar_lookup?Endelman%20JB.%20Ridge%20Regression%20and%20other%20kernels%20for%20genomic%20selection%20with%20R%20package%20rrBLUP.%20Plant%20Genome.%202011.%2010.3835/plantgenome2011.08.0024." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Falconer DS, Mackay TFC. Introduction to quantitative genetics. 4th ed. Harlow: Pearson, Prentice Hall; 2009.</cite> [<a href="https://scholar.google.com/scholar_lookup?Falconer%20DS,%20Mackay%20TFC.%20Introduction%20to%20quantitative%20genetics.%204th%20ed.%20Harlow:%20Pearson,%20Prentice%20Hall;%202009." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Finlay KW, Wilkinson GN. The analysis of adaptation in a plant-breeding programme. Aust J Agric Res. 1963;14(6):742–54. 10.1071/ar9630742.</cite> [<a href="https://scholar.google.com/scholar_lookup?Finlay%20KW,%20Wilkinson%20GN.%20The%20analysis%20of%20adaptation%20in%20a%20plant-breeding%20programme.%20Aust%20J%20Agric%20Res.%201963;14(6):742%E2%80%9354.%2010.1071/ar9630742." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>Fritsche-Neto R. SoilType: an R package to interplay soil characterization in plant science. Agron J. 2023. 10.1002/agj2.21383.</cite> [<a href="https://scholar.google.com/scholar_lookup?Fritsche-Neto%20R.%20SoilType:%20an%20R%20package%20to%20interplay%20soil%20characterization%20in%20plant%20science.%20Agron%20J.%202023.%2010.1002/agj2.21383." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>Consortium, R2d2, Fugeray-Scarbel A, Bastien C, Dupont-Nivet M, Lemarié S. Why and how to switch to genomic selection: lessons from plant and animal breeding experience. Front Genet. 2021. 10.3389/fgene.2021.629737.</cite> [<a href="https://doi.org/10.3389/fgene.2021.629737" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8301370/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34305998/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Consortium,%20R2d2,%20Fugeray-Scarbel%20A,%20Bastien%20C,%20Dupont-Nivet%20M,%20Lemari%C3%A9%20S.%20Why%20and%20how%20to%20switch%20to%20genomic%20selection:%20lessons%20from%20plant%20and%20animal%20breeding%20experience.%20Front%20Genet.%202021.%2010.3389/fgene.2021.629737." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>Gillberg J, Marttinen P, Mamitsuka H, Kaski S. Modelling G×E with historical weather information improves genomic prediction in new environments. Bioinformatics. 2019;35(20):4045–52. 10.1093/bioinformatics/btz197.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btz197" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6792123/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30977782/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gillberg%20J,%20Marttinen%20P,%20Mamitsuka%20H,%20Kaski%20S.%20Modelling%20G%C3%97E%20with%20historical%20weather%20information%20improves%20genomic%20prediction%20in%20new%20environments.%20Bioinformatics.%202019;35(20):4045%E2%80%9352.%2010.1093/bioinformatics/btz197." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>Habier D, Tetens J, Seefried F-R, Lichtner P, Thaller G. The impact of genetic relationship information on genomic breeding values in German Holstein Cattle. Genet Sel Evol. 2010;42(1):5. 10.1186/1297-9686-42-5.
</cite> [<a href="https://doi.org/10.1186/1297-9686-42-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2838754/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20170500/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Habier%20D,%20Tetens%20J,%20Seefried%20F-R,%20Lichtner%20P,%20Thaller%20G.%20The%20impact%20of%20genetic%20relationship%20information%20on%20genomic%20breeding%20values%20in%20German%20Holstein%20Cattle.%20Genet%20Sel%20Evol.%202010;42(1):5.%2010.1186/1297-9686-42-5." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>He S, Thistlethwaite R, Forrest K, Shi F, Hayden MJ, Trethowan R, Daetwyler HD. Extension of a haplotype-based genomic prediction model to manage multi-environment wheat data using environmental covariates. Theor Appl Genet. 2019;132(11):3143–54. 10.1007/s00122-019-03413-1.
</cite> [<a href="https://doi.org/10.1007/s00122-019-03413-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31435703/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?He%20S,%20Thistlethwaite%20R,%20Forrest%20K,%20Shi%20F,%20Hayden%20MJ,%20Trethowan%20R,%20Daetwyler%20HD.%20Extension%20of%20a%20haplotype-based%20genomic%20prediction%20model%20to%20manage%20multi-environment%20wheat%20data%20using%20environmental%20covariates.%20Theor%20Appl%20Genet.%202019;132(11):3143%E2%80%9354.%2010.1007/s00122-019-03413-1." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>Heffner EL, Sorrells ME, Jannink J-L. Genomic selection for crop improvement. Crop Sci. 2009;49(1):1. 10.2135/cropsci2008.08.0512.</cite> [<a href="https://scholar.google.com/scholar_lookup?Heffner%20EL,%20Sorrells%20ME,%20Jannink%20J-L.%20Genomic%20selection%20for%20crop%20improvement.%20Crop%20Sci.%202009;49(1):1.%2010.2135/cropsci2008.08.0512." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>Hu H, Rincent R, Runcie DE. MegaLMM improves genomic predictions in new environments using environmental covariates. Genetics. 2024;229:1–41.</cite> [<a href="https://doi.org/10.1093/genetics/iyae171" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11708919/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39471330/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hu%20H,%20Rincent%20R,%20Runcie%20DE.%20MegaLMM%20improves%20genomic%20predictions%20in%20new%20environments%20using%20environmental%20covariates.%20Genetics.%202024;229:1%E2%80%9341." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Jarquin D, de Leon N, Romay C, Bohn M, Buckler ES, Ciampitti I, Edwards J, Ertl D, Flint-Garcia S, Gore MA, Graham C, Hirsch CN, Holland JB, Hooker D, Kaeppler SM, Knoll J, Lee EC, Lawrence-Dill CJ, Lynch JP, …Lorenz, A. R. Utility of Climatic Information via Combining Ability Models to Improve Genomic Prediction for Yield Within theGenomes to Fields Maize Project. Frontiers in Genetics. 2021. 10.3389/fgene.2020.592769.
</cite> [<a href="https://doi.org/10.3389/fgene.2020.592769" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7982677/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33763106/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jarquin%20D,%20de%20Leon%20N,%20Romay%20C,%20Bohn%20M,%20Buckler%20ES,%20Ciampitti%20I,%20Edwards%20J,%20Ertl%20D,%20Flint-Garcia%20S,%20Gore%20MA,%20Graham%20C,%20Hirsch%20CN,%20Holland%20JB,%20Hooker%20D,%20Kaeppler%20SM,%20Knoll%20J,%20Lee%20EC,%20Lawrence-Dill%20CJ,%20Lynch%20JP,%20%E2%80%A6Lorenz,%20A.%20R.%20Utility%20of%20Climatic%20Information%20via%20Combining%20Ability%20Models%20to%20Improve%20Genomic%20Prediction%20for%20Yield%20Within%20theGenomes%20to%20Fields%20Maize%20Project.%20Frontiers%20in%20Genetics.%202021.%2010.3389/fgene.2020.592769." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Jarquín D, Crossa J, Lacaze X, Du Cheyron P, Daucourt J, Lorgeou J, Piraux F, Guerreiro L, Pérez P, Calus M, Burgueño J, de los Campos G. A reaction norm model for genomic selection using high-dimensional genomic and environmental data. Theor Appl Genet. 2014;127(3):595–607. 10.1007/s00122-013-2243-1.
</cite> [<a href="https://doi.org/10.1007/s00122-013-2243-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3931944/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24337101/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jarqu%C3%ADn%20D,%20Crossa%20J,%20Lacaze%20X,%20Du%20Cheyron%20P,%20Daucourt%20J,%20Lorgeou%20J,%20Piraux%20F,%20Guerreiro%20L,%20P%C3%A9rez%20P,%20Calus%20M,%20Burgue%C3%B1o%20J,%20de%20los%20Campos%20G.%20A%20reaction%20norm%20model%20for%20genomic%20selection%20using%20high-dimensional%20genomic%20and%20environmental%20data.%20Theor%20Appl%20Genet.%202014;127(3):595%E2%80%93607.%2010.1007/s00122-013-2243-1." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>Kriss AB, Paul PA, Madden LV. Relationship between yearly fluctuations in fusarium head blight intensity and environmental variables: a window-pane analysis. Phytopathology®. 2010;100(8):784–97. 10.1094/PHYTO-100-8-0784.
</cite> [<a href="https://doi.org/10.1094/PHYTO-100-8-0784" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20626282/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kriss%20AB,%20Paul%20PA,%20Madden%20LV.%20Relationship%20between%20yearly%20fluctuations%20in%20fusarium%20head%20blight%20intensity%20and%20environmental%20variables:%20a%20window-pane%20analysis.%20Phytopathology%C2%AE.%202010;100(8):784%E2%80%9397.%2010.1094/PHYTO-100-8-0784." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR30">
<span class="label">30.</span><cite>Lian L, de los Campos G (2015) FW: an R package for finlay–wilkinson regression that incorporates genomic/pedigree information and covariance structures between environments. G3: Genes Gen Genet. 6(3):589–597. 10.1534/g3.115.026328</cite> [<a href="https://doi.org/10.1534/g3.115.026328" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4777122/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26715095/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lian%20L,%20de%20los%20Campos%20G%20(2015)%20FW:%20an%20R%20package%20for%20finlay%E2%80%93wilkinson%20regression%20that%20incorporates%20genomic/pedigree%20information%20and%20covariance%20structures%20between%20environments.%20G3:%20Genes%20Gen%20Genet.%206(3):589%E2%80%93597.%2010.1534/g3.115.026328" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>Li X, Guo T, Bai G, Zhang Z, See D, Marshall J, Garland-Campbell KA, Jianming Yu. Genetics-inspired data-driven approaches explain and predict crop performance fluctuations attributed to changing climatic conditions. Mol Plant. 2022;15(2):203–6. 10.1016/j.molp.2022.01.001.
</cite> [<a href="https://doi.org/10.1016/j.molp.2022.01.001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34999020/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li%20X,%20Guo%20T,%20Bai%20G,%20Zhang%20Z,%20See%20D,%20Marshall%20J,%20Garland-Campbell%20KA,%20Jianming%20Yu.%20Genetics-inspired%20data-driven%20approaches%20explain%20and%20predict%20crop%20performance%20fluctuations%20attributed%20to%20changing%20climatic%20conditions.%20Mol%20Plant.%202022;15(2):203%E2%80%936.%2010.1016/j.molp.2022.01.001." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>Li X, Guo T, Wang J, Bekele WA, Sukumaran S, Vanous AE, McNellie JP, Tibbs-Cortes LE, Lopes MS, Lamkey KR, Westgate ME, McKay JK, Archontoulis SV, Reynolds MP, Tinker NA, Schnable PS, Jianming Yu. An Integrated framework reinstating the environmental dimension for GWAS and genomic selection in crops. Mol Plant. 2021;14(6):874–87. 10.1016/j.molp.2021.03.010.
</cite> [<a href="https://doi.org/10.1016/j.molp.2021.03.010" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33713844/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li%20X,%20Guo%20T,%20Wang%20J,%20Bekele%20WA,%20Sukumaran%20S,%20Vanous%20AE,%20McNellie%20JP,%20Tibbs-Cortes%20LE,%20Lopes%20MS,%20Lamkey%20KR,%20Westgate%20ME,%20McKay%20JK,%20Archontoulis%20SV,%20Reynolds%20MP,%20Tinker%20NA,%20Schnable%20PS,%20Jianming%20Yu.%20An%20Integrated%20framework%20reinstating%20the%20environmental%20dimension%20for%20GWAS%20and%20genomic%20selection%20in%20crops.%20Mol%20Plant.%202021;14(6):874%E2%80%9387.%2010.1016/j.molp.2021.03.010." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Li X, Guo T, Qi Mu, Li X, Jianming Yu. Genomic and environmental determinants and their interplay underlying phenotypic plasticity. Proc Natl Acad Sci. 2018;115(26):6679–84. 10.1073/pnas.1718326115.
</cite> [<a href="https://doi.org/10.1073/pnas.1718326115" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6042117/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29891664/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li%20X,%20Guo%20T,%20Qi%20Mu,%20Li%20X,%20Jianming%20Yu.%20Genomic%20and%20environmental%20determinants%20and%20their%20interplay%20underlying%20phenotypic%20plasticity.%20Proc%20Natl%20Acad%20Sci.%202018;115(26):6679%E2%80%9384.%2010.1073/pnas.1718326115." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>Lin CS, Binns MR, Lefkovitch LP. Stability analysis: where do we stand? Crop Sci. 1986. 10.2135/cropsci1986.0011183X002600050012x.</cite> [<a href="https://scholar.google.com/scholar_lookup?Lin%20CS,%20Binns%20MR,%20Lefkovitch%20LP.%20Stability%20analysis:%20where%20do%20we%20stand?%20Crop%20Sci.%201986.%2010.2135/cropsci1986.0011183X002600050012x." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>Lopez-Cruz M, Aguate FM, Washburn JD, de Leon N, Kaeppler SM, Lima DC, Tan R, Thompson A, De La Bretonne LW, de los Campos G. Leveraging data from the genomes-to-fields initiative to investigate genotype-by-environment interactions in Maize in North America. Nat Commun. 2023;14(1):6904. 10.1038/s41467-023-42687-4.
</cite> [<a href="https://doi.org/10.1038/s41467-023-42687-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10616096/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37903778/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lopez-Cruz%20M,%20Aguate%20FM,%20Washburn%20JD,%20de%20Leon%20N,%20Kaeppler%20SM,%20Lima%20DC,%20Tan%20R,%20Thompson%20A,%20De%20La%20Bretonne%20LW,%20de%20los%20Campos%20G.%20Leveraging%20data%20from%20the%20genomes-to-fields%20initiative%20to%20investigate%20genotype-by-environment%20interactions%20in%20Maize%20in%20North%20America.%20Nat%20Commun.%202023;14(1):6904.%2010.1038/s41467-023-42687-4." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR36">
<span class="label">36.</span><cite>Lorenz AJ. Resource allocation for maximizing prediction accuracy and genetic gain of genomic selection in plant breeding: a simulation experiment. G3 Genes|Genomes|Genetics. 2013;3(3):481–91. 10.1534/g3.112.004911.
</cite> [<a href="https://doi.org/10.1534/g3.112.004911" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3583455/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23450123/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lorenz%20AJ.%20Resource%20allocation%20for%20maximizing%20prediction%20accuracy%20and%20genetic%20gain%20of%20genomic%20selection%20in%20plant%20breeding:%20a%20simulation%20experiment.%20G3%20Genes%7CGenomes%7CGenetics.%202013;3(3):481%E2%80%9391.%2010.1534/g3.112.004911." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR37">
<span class="label">37.</span><cite>Laporte F, Mary-Huard T, GQMS CoreFunctions Team. MM4LMM: inference of linear mixed models through MM algorithm. 2024.</cite>
</li>
<li id="CR38">
<span class="label">38.</span><cite>Ly D, Huet S, Gauffreteau A, Rincent R, Touzy G, Mini A, Jannink J-L, Cormier F, Paux E, Lafarge S, Le Gouis J, Charmet G. Whole-genome prediction of reaction norms to environmental stress in bread wheat (Triticum Aestivum L.) by genomic random regression. Field Crop Res. 2018;216:32–41. 10.1016/j.fcr.2017.08.020.</cite> [<a href="https://scholar.google.com/scholar_lookup?Ly%20D,%20Huet%20S,%20Gauffreteau%20A,%20Rincent%20R,%20Touzy%20G,%20Mini%20A,%20Jannink%20J-L,%20Cormier%20F,%20Paux%20E,%20Lafarge%20S,%20Le%20Gouis%20J,%20Charmet%20G.%20Whole-genome%20prediction%20of%20reaction%20norms%20to%20environmental%20stress%20in%20bread%20wheat%20(Triticum%20Aestivum%20L.)%20by%20genomic%20random%20regression.%20Field%20Crop%20Res.%202018;216:32%E2%80%9341.%2010.1016/j.fcr.2017.08.020." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR39">
<span class="label">39.</span><cite>Meuwissen T, Hayes B, Goddard M. Prediction of total genetic value using genome-wide dense marker maps. Genetics. 2001;157(4):1819–29. 10.1093/genetics/157.4.1819.
</cite> [<a href="https://doi.org/10.1093/genetics/157.4.1819" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1461589/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11290733/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Meuwissen%20T,%20Hayes%20B,%20Goddard%20M.%20Prediction%20of%20total%20genetic%20value%20using%20genome-wide%20dense%20marker%20maps.%20Genetics.%202001;157(4):1819%E2%80%9329.%2010.1093/genetics/157.4.1819." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR40">
<span class="label">40.</span><cite>Miedaner T, Reinbrecht C, Lauber U, Schollenberger M, Geiger HH. Effects of genotype and genotype—environment interaction on deoxynivalenol accumulation and resistance to fusarium head blight in rye, triticale, and wheat. Plant Breed. 2001;120(2):97–105. 10.1046/j.1439-0523.2001.00580.x.</cite> [<a href="https://scholar.google.com/scholar_lookup?Miedaner%20T,%20Reinbrecht%20C,%20Lauber%20U,%20Schollenberger%20M,%20Geiger%20HH.%20Effects%20of%20genotype%20and%20genotype%E2%80%94environment%20interaction%20on%20deoxynivalenol%20accumulation%20and%20resistance%20to%20fusarium%20head%20blight%20in%20rye,%20triticale,%20and%20wheat.%20Plant%20Breed.%202001;120(2):97%E2%80%93105.%2010.1046/j.1439-0523.2001.00580.x." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR41">
<span class="label">41.</span><cite>Millet EJ, Kruijer W, Coupel-Ledru A, Prado SA, Cabrera-Bosquet L, Lacube S, Charcosset A, Welcker C, van Eeuwijk F, Tardieu F. Genomic prediction of maize yield across european environmental conditions. Nat Genet. 2019;51(6):952–6. 10.1038/s41588-019-0414-y.
</cite> [<a href="https://doi.org/10.1038/s41588-019-0414-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31110353/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Millet%20EJ,%20Kruijer%20W,%20Coupel-Ledru%20A,%20Prado%20SA,%20Cabrera-Bosquet%20L,%20Lacube%20S,%20Charcosset%20A,%20Welcker%20C,%20van%20Eeuwijk%20F,%20Tardieu%20F.%20Genomic%20prediction%20of%20maize%20yield%20across%20european%20environmental%20conditions.%20Nat%20Genet.%202019;51(6):952%E2%80%936.%2010.1038/s41588-019-0414-y." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR42">
<span class="label">42.</span><cite>Napier JD, Heckman RW, Juenger TE. Gene-by-environment interactions in plants: molecular mechanisms, environmental drivers, and adaptive plasticity. Plant Cell. 2023;35(1):109–24. 10.1093/plcell/koac322.
</cite> [<a href="https://doi.org/10.1093/plcell/koac322" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9806611/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36342220/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Napier%20JD,%20Heckman%20RW,%20Juenger%20TE.%20Gene-by-environment%20interactions%20in%20plants:%20molecular%20mechanisms,%20environmental%20drivers,%20and%20adaptive%20plasticity.%20Plant%20Cell.%202023;35(1):109%E2%80%9324.%2010.1093/plcell/koac322." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR43">
<span class="label">43.</span><cite>Neyhart JL, Silverstein KAT, Smith KP. Accurate predictions of barley phenotypes using genomewide markers and environmental covariates. Crop Sci. 2022;62(5):1821–33. 10.1002/csc2.20782.</cite> [<a href="https://scholar.google.com/scholar_lookup?Neyhart%20JL,%20Silverstein%20KAT,%20Smith%20KP.%20Accurate%20predictions%20of%20barley%20phenotypes%20using%20genomewide%20markers%20and%20environmental%20covariates.%20Crop%20Sci.%202022;62(5):1821%E2%80%9333.%2010.1002/csc2.20782." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR44">
<span class="label">44.</span><cite>O’Donnell K, Corby Kistler H, Tacke BK, Casper HH. Gene genealogies reveal global phylogeographic structure and reproductive isolation among lineages of fusarium graminearum, the fungus causing wheat scab. Proc Natl Acad Sci. 2000;97(14):7905–10. 10.1073/pnas.130193297.
</cite> [<a href="https://doi.org/10.1073/pnas.130193297" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC16643/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10869425/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?O%E2%80%99Donnell%20K,%20Corby%20Kistler%20H,%20Tacke%20BK,%20Casper%20HH.%20Gene%20genealogies%20reveal%20global%20phylogeographic%20structure%20and%20reproductive%20isolation%20among%20lineages%20of%20fusarium%20graminearum,%20the%20fungus%20causing%20wheat%20scab.%20Proc%20Natl%20Acad%20Sci.%202000;97(14):7905%E2%80%9310.%2010.1073/pnas.130193297." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR45">
<span class="label">45.</span><cite>Pérez P, de los Campos G. Genome-wide regression and prediction with the BGLR statistical package. Genetics. 2014;198(2):483–95. 10.1534/genetics.114.164442.
</cite> [<a href="https://doi.org/10.1534/genetics.114.164442" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4196607/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25009151/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?P%C3%A9rez%20P,%20de%20los%20Campos%20G.%20Genome-wide%20regression%20and%20prediction%20with%20the%20BGLR%20statistical%20package.%20Genetics.%202014;198(2):483%E2%80%9395.%2010.1534/genetics.114.164442." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR46">
<span class="label">46.</span><cite>Piepho H-P, Blancon J. Extending finlay-wilkinson regression with environmental covariates. Plant Breed. 2023;142(5):621–31. 10.1111/pbr.13130.</cite> [<a href="https://scholar.google.com/scholar_lookup?Piepho%20H-P,%20Blancon%20J.%20Extending%20finlay-wilkinson%20regression%20with%20environmental%20covariates.%20Plant%20Breed.%202023;142(5):621%E2%80%9331.%2010.1111/pbr.13130." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR47">
<span class="label">47.</span><cite>Rebollo I, Aguilar I, Pérez F, de Vida F, Molina LG, Rosas JE. Genotype by environment interaction characterization and its modeling with random regression to climatic variables in two rice breeding populations. Crop Sci. 2023;63(4):2220–40. 10.1002/csc2.21029.</cite> [<a href="https://scholar.google.com/scholar_lookup?Rebollo%20I,%20Aguilar%20I,%20P%C3%A9rez%20F,%20de%20Vida%20F,%20Molina%20LG,%20Rosas%20JE.%20Genotype%20by%20environment%20interaction%20characterization%20and%20its%20modeling%20with%20random%20regression%20to%20climatic%20variables%20in%20two%20rice%20breeding%20populations.%20Crop%20Sci.%202023;63(4):2220%E2%80%9340.%2010.1002/csc2.21029." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR48">
<span class="label">48.</span><cite>Resende RT, Xavier A, Pedro IT, Silva MPM, Resende DJ, Marcatti GE. GIS-Based G × E modeling of maize hybrids through enviromic markers engineering. New Phytol. 2025;245(1):102–16. 10.1111/nph.19951.
</cite> [<a href="https://doi.org/10.1111/nph.19951" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11617650/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39014516/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Resende%20RT,%20Xavier%20A,%20Pedro%20IT,%20Silva%20MPM,%20Resende%20DJ,%20Marcatti%20GE.%20GIS-Based%20G%20%C3%97%20E%20modeling%20of%20maize%20hybrids%20through%20enviromic%20markers%20engineering.%20New%20Phytol.%202025;245(1):102%E2%80%9316.%2010.1111/nph.19951." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR49">
<span class="label">49.</span><cite>Runcie DE, Jiayi Qu, Cheng H, Crawford L. MegaLMM: Mega-scale linear mixed models for genomic predictions with thousands of traits. Genome Biol. 2021;22(1):213. 10.1186/s13059-021-02416-w.
</cite> [<a href="https://doi.org/10.1186/s13059-021-02416-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8299638/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34301310/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Runcie%20DE,%20Jiayi%20Qu,%20Cheng%20H,%20Crawford%20L.%20MegaLMM:%20Mega-scale%20linear%20mixed%20models%20for%20genomic%20predictions%20with%20thousands%20of%20traits.%20Genome%20Biol.%202021;22(1):213.%2010.1186/s13059-021-02416-w." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR50">
<span class="label">50.</span><cite>Rutkoski J, Benson J, Jia Yi, Brown-Guedira G, Jannink J-L, Sorrells M. Evaluation of genomic prediction methods for fusarium head blight resistance in wheat. Plant Genom. 2012;5(2):51–61. 10.3835/plantgenome2012.02.0001.</cite> [<a href="https://scholar.google.com/scholar_lookup?Rutkoski%20J,%20Benson%20J,%20Jia%20Yi,%20Brown-Guedira%20G,%20Jannink%20J-L,%20Sorrells%20M.%20Evaluation%20of%20genomic%20prediction%20methods%20for%20fusarium%20head%20blight%20resistance%20in%20wheat.%20Plant%20Genom.%202012;5(2):51%E2%80%9361.%2010.3835/plantgenome2012.02.0001." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR51">
<span class="label">51.</span><cite>Shah DA, De Wolf ED, Paul PA, Madden LV. Functional data analysis of weather variables linked to fusarium head blight epidemics in the United States. Phytopathology®. 2019;109(1):96–110. 10.1094/PHYTO-11-17-0386-R.
</cite> [<a href="https://doi.org/10.1094/PHYTO-11-17-0386-R" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29897307/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Shah%20DA,%20De%20Wolf%20ED,%20Paul%20PA,%20Madden%20LV.%20Functional%20data%20analysis%20of%20weather%20variables%20linked%20to%20fusarium%20head%20blight%20epidemics%20in%20the%20United%20States.%20Phytopathology%C2%AE.%202019;109(1):96%E2%80%93110.%2010.1094/PHYTO-11-17-0386-R." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR52">
<span class="label">52.</span><cite>Shah DA, De Wolf ED, Paul PA, Madden LV. Into the trees: random forests for predicting fusarium head blight epidemics of wheat in the United States. Phytopathology®. 2023;113(8):1483–93. 10.1094/PHYTO-10-22-0380-R.
</cite> [<a href="https://doi.org/10.1094/PHYTO-10-22-0380-R" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36880796/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Shah%20DA,%20De%20Wolf%20ED,%20Paul%20PA,%20Madden%20LV.%20Into%20the%20trees:%20random%20forests%20for%20predicting%20fusarium%20head%20blight%20epidemics%20of%20wheat%20in%20the%20United%20States.%20Phytopathology%C2%AE.%202023;113(8):1483%E2%80%9393.%2010.1094/PHYTO-10-22-0380-R." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR53">
<span class="label">53.</span><cite>Sparks AH. Nasapower: a NASA POWER global meteorology, surface solar energy and climatology data client for R. J Open Sour Softw. 2018;3(30):1035. 10.21105/joss.01035.</cite> [<a href="https://scholar.google.com/scholar_lookup?Sparks%20AH.%20Nasapower:%20a%20NASA%20POWER%20global%20meteorology,%20surface%20solar%20energy%20and%20climatology%20data%20client%20for%20R.%20J%20Open%20Sour%20Softw.%202018;3(30):1035.%2010.21105/joss.01035." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR54">
<span class="label">54.</span><cite>Sparks AH, Chamberlain S, Boyer A, Kavili H, Miguez F, Salmon M, Alderman P, Blagotić A, Daróczi G. Nasapower: NASA POWER API Client. 2024.</cite>
</li>
<li id="CR55">
<span class="label">55.</span><cite>Steiner B, Buerstmayr M, Michel S, Schweiger W, Lemmens M, Buerstmayr H. Breeding strategies and advances in line selection for fusarium head blight resistance in wheat. Trop Plant Pathol. 2017;42(3):165–74. 10.1007/s40858-017-0127-7.</cite> [<a href="https://scholar.google.com/scholar_lookup?Steiner%20B,%20Buerstmayr%20M,%20Michel%20S,%20Schweiger%20W,%20Lemmens%20M,%20Buerstmayr%20H.%20Breeding%20strategies%20and%20advances%20in%20line%20selection%20for%20fusarium%20head%20blight%20resistance%20in%20wheat.%20Trop%20Plant%20Pathol.%202017;42(3):165%E2%80%9374.%2010.1007/s40858-017-0127-7." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR56">
<span class="label">56.</span><cite>Tardieu F. Any trait or trait-related allele can confer drought tolerance: just design the right drought scenario. J Exp Bot. 2012;63(1):25–31. 10.1093/jxb/err269.
</cite> [<a href="https://doi.org/10.1093/jxb/err269" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21963615/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tardieu%20F.%20Any%20trait%20or%20trait-related%20allele%20can%20confer%20drought%20tolerance:%20just%20design%20the%20right%20drought%20scenario.%20J%20Exp%20Bot.%202012;63(1):25%E2%80%9331.%2010.1093/jxb/err269." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR57">
<span class="label">57.</span><cite>Tadese D, Piepho HP, Hartung J. 2023. Accuracy of Prediction from Multi-Environment Trials to New Locations Using Pedigree Information and Environmental Covariates. 2023.</cite> [<a href="https://doi.org/10.1007/s00122-024-04684-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11236881/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38985188/" class="usa-link">PubMed</a>]</li>
<li id="CR58">
<span class="label">58.</span><cite>Tolhurst DJ, Chris Gaynor R, Gardunia B, Hickey JM, Gorjanc G. Genomic Selection using random regressions on known and latent environmental covariates. Theor Appl Genet. 2022;135(10):3393–415. 10.1007/s00122-022-04186-w.
</cite> [<a href="https://doi.org/10.1007/s00122-022-04186-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9519718/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36066596/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tolhurst%20DJ,%20Chris%20Gaynor%20R,%20Gardunia%20B,%20Hickey%20JM,%20Gorjanc%20G.%20Genomic%20Selection%20using%20random%20regressions%20on%20known%20and%20latent%20environmental%20covariates.%20Theor%20Appl%20Genet.%202022;135(10):3393%E2%80%93415.%2010.1007/s00122-022-04186-w." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR59">
<span class="label">59.</span><cite>USDA National Agricultural Statistics Service. 2024. USDA NASS Statistics by Subject. USDA.</cite>
</li>
<li id="CR60">
<span class="label">60.</span><cite>VanRaden PM. Efficient methods to compute genomic predictions. J Dairy Sci. 2008;91(11):4414–23. 10.3168/jds.2007-0980.
</cite> [<a href="https://doi.org/10.3168/jds.2007-0980" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18946147/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?VanRaden%20PM.%20Efficient%20methods%20to%20compute%20genomic%20predictions.%20J%20Dairy%20Sci.%202008;91(11):4414%E2%80%9323.%2010.3168/jds.2007-0980." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR61">
<span class="label">61.</span><cite>Venske E, dos Santos RS, da Rosa Farias D, Rother V, da Maia LC, Pegoraro C, de Oliveira AC. Meta-analysis of the QTLome of fusarium head blight resistance in bread wheat: refining the current puzzle. Front Plant Sci. 2019. 10.3389/fpls.2019.00727.
</cite> [<a href="https://doi.org/10.3389/fpls.2019.00727" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6585393/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31263469/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Venske%20E,%20dos%20Santos%20RS,%20da%20Rosa%20Farias%20D,%20Rother%20V,%20da%20Maia%20LC,%20Pegoraro%20C,%20de%20Oliveira%20AC.%20Meta-analysis%20of%20the%20QTLome%20of%20fusarium%20head%20blight%20resistance%20in%20bread%20wheat:%20refining%20the%20current%20puzzle.%20Front%20Plant%20Sci.%202019.%2010.3389/fpls.2019.00727." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR62">
<span class="label">62.</span><cite>Waldron BL, Moreno-Sevilla B, Anderson JA, Stack RW, Frohberg RC. RFLP mapping of QTL for fusarium head blight resistance in wheat. Crop Sci. 1999. 10.2135/cropsci1999.0011183X003900030032x.</cite> [<a href="https://scholar.google.com/scholar_lookup?Waldron%20BL,%20Moreno-Sevilla%20B,%20Anderson%20JA,%20Stack%20RW,%20Frohberg%20RC.%20RFLP%20mapping%20of%20QTL%20for%20fusarium%20head%20blight%20resistance%20in%20wheat.%20Crop%20Sci.%201999.%2010.2135/cropsci1999.0011183X003900030032x." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR63">
<span class="label">63.</span><cite>Windhausen VS, Atlin GN, Hickey JM, Crossa J, Jannink JL, Sorrells ME, Raman B, Cairns JE, Tarekegne A, Semagn K, Beyene Y, Grudloyma P, Technow F, Riedelsheimer C, Melchinger AE. Effectiveness of genomic prediction of maize hybrid performance in different breeding populations and environments. G3 Genes|Genomes|Genetics. 2012;2(11):1427–36. 10.1534/g3.112.003699.
</cite> [<a href="https://doi.org/10.1534/g3.112.003699" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3484673/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23173094/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Windhausen%20VS,%20Atlin%20GN,%20Hickey%20JM,%20Crossa%20J,%20Jannink%20JL,%20Sorrells%20ME,%20Raman%20B,%20Cairns%20JE,%20Tarekegne%20A,%20Semagn%20K,%20Beyene%20Y,%20Grudloyma%20P,%20Technow%20F,%20Riedelsheimer%20C,%20Melchinger%20AE.%20Effectiveness%20of%20genomic%20prediction%20of%20maize%20hybrid%20performance%20in%20different%20breeding%20populations%20and%20environments.%20G3%20Genes%7CGenomes%7CGenetics.%202012;2(11):1427%E2%80%9336.%2010.1534/g3.112.003699." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR64">
<span class="label">64.</span><cite>De Wolf ED, Madden LV, Lipps PE. Risk assessment models for wheat fusarium head blight epidemics based on within-season weather data. Phytopathology®. 2003;93(4):428–35. 10.1094/PHYTO.2003.93.4.428.
</cite> [<a href="https://doi.org/10.1094/PHYTO.2003.93.4.428" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18944357/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?De%20Wolf%20ED,%20Madden%20LV,%20Lipps%20PE.%20Risk%20assessment%20models%20for%20wheat%20fusarium%20head%20blight%20epidemics%20based%20on%20within-season%20weather%20data.%20Phytopathology%C2%AE.%202003;93(4):428%E2%80%9335.%2010.1094/PHYTO.2003.93.4.428." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR65">
<span class="label">65.</span><cite>Yates F, Cochran WG. The analysis of groups of experiments. J Agric Sci. 1938;28(4):556–80. 10.1017/S0021859600050978.</cite> [<a href="https://scholar.google.com/scholar_lookup?Yates%20F,%20Cochran%20WG.%20The%20analysis%20of%20groups%20of%20experiments.%20J%20Agric%20Sci.%201938;28(4):556%E2%80%9380.%2010.1017/S0021859600050978." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR66">
<span class="label">66.</span><cite>Zadoks JC, Chang TT, Konzak CF. A decimal code for the growth stages of cereals. Weed Res. 1974;14(6):415–21. 10.1111/j.1365-3180.1974.tb01084.x.</cite> [<a href="https://scholar.google.com/scholar_lookup?Zadoks%20JC,%20Chang%20TT,%20Konzak%20CF.%20A%20decimal%20code%20for%20the%20growth%20stages%20of%20cereals.%20Weed%20Res.%201974;14(6):415%E2%80%9321.%2010.1111/j.1365-3180.1974.tb01084.x." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR67">
<span class="label">67.</span><cite>Zhu T, Wang Le, Rimbert H, Rodriguez JC, Deal KR, De Oliveira R, Choulet F, Keeble-Gagnère G, Tibbits J, Rogers J, Eversole K, Appels R, Gu YQ, Mascher M, Dvorak J, Luo M-C. Optical Maps refine the bread wheat Triticum Aestivum Cv. Chinese spring genome assembly. Plant J. 2021;107(1):303–14. 10.1111/tpj.15289.
</cite> [<a href="https://doi.org/10.1111/tpj.15289" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8360199/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33893684/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhu%20T,%20Wang%20Le,%20Rimbert%20H,%20Rodriguez%20JC,%20Deal%20KR,%20De%20Oliveira%20R,%20Choulet%20F,%20Keeble-Gagn%C3%A8re%20G,%20Tibbits%20J,%20Rogers%20J,%20Eversole%20K,%20Appels%20R,%20Gu%20YQ,%20Mascher%20M,%20Dvorak%20J,%20Luo%20M-C.%20Optical%20Maps%20refine%20the%20bread%20wheat%20Triticum%20Aestivum%20Cv.%20Chinese%20spring%20genome%20assembly.%20Plant%20J.%202021;107(1):303%E2%80%9314.%2010.1111/tpj.15289." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12366029/bin/13007_2025_1418_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary material 1.</a><sup> (2.1MB, pdf) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>Phenotypic data from all URSN trials are available on the T3 database (<a href="https://wheat.triticeaetoolbox.org/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://wheat.triticeaetoolbox.org/</a>). All scripts, intermediate files, result tables, and figures to reproduce the results are available in a public repository: 10.5061/dryad.wstqjq2z0.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Plant Methods are provided here courtesy of <strong>BMC</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1186/s13007-025-01418-0"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/13007_2025_Article_1418.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (2.5 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12366029/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12366029/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12366029%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366029/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12366029/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12366029/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40836294/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12366029/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40836294/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12366029/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12366029/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="SbBdyyiKNmRMhD15uH37P2ZSxlW5krbOKAoVfSdtEFffETJ23CjYbz6EpXbD1U4v">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
        <script type="module" crossorigin="" src="/static/assets/math-574fdcc6.js"></script>
    
    

    </body>
</html>
