
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Combining QTL mapping and transcriptomics to identify candidate genes for cold tolerance during the budding and seedling stages in rice - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="D2E4B69D8AF3BEB305B69D00124F63F4.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="bmcgeno">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12363062/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="BMC Genomics">
<meta name="citation_title" content="Combining QTL mapping and transcriptomics to identify candidate genes for cold tolerance during the budding and seedling stages in rice">
<meta name="citation_author" content="Chun Ae Kim">
<meta name="citation_author_institution" content="State Key Laboratory of Crop Gene Resources and Breeding, Institute of Crop Sciences, Chinese Academy of Agricultural Sciences, Beijing, 100081 China">
<meta name="citation_author_institution" content="Molecular Plant Breeding Lab, Branch of Biotechnology, Democratic, State Academy of Sciences, Pyongyang, Democratic People’s Republic of Korea">
<meta name="citation_author" content="Wenqiang Chen">
<meta name="citation_author_institution" content="State Key Laboratory of Crop Gene Resources and Breeding, Institute of Crop Sciences, Chinese Academy of Agricultural Sciences, Beijing, 100081 China">
<meta name="citation_author" content="Shuangbing Zhu">
<meta name="citation_author_institution" content="Shenzhen Branch, Guangdong Laboratory for Lingnan Modern Agriculture, Agricultural Genomics Institute at Shenzhen, Chinese Academy of Agricultural Sciences, Shenzhen, 518120 China">
<meta name="citation_author" content="Guogen Zhang">
<meta name="citation_author_institution" content="State Key Laboratory of Crop Gene Resources and Breeding, Institute of Crop Sciences, Chinese Academy of Agricultural Sciences, Beijing, 100081 China">
<meta name="citation_author" content="Congcong Shen">
<meta name="citation_author_institution" content="Shenzhen Branch, Guangdong Laboratory for Lingnan Modern Agriculture, Agricultural Genomics Institute at Shenzhen, Chinese Academy of Agricultural Sciences, Shenzhen, 518120 China">
<meta name="citation_author" content="Kai Chen">
<meta name="citation_author_institution" content="Shenzhen Branch, Guangdong Laboratory for Lingnan Modern Agriculture, Agricultural Genomics Institute at Shenzhen, Chinese Academy of Agricultural Sciences, Shenzhen, 518120 China">
<meta name="citation_author" content="Xiuqin Zhao">
<meta name="citation_author_institution" content="State Key Laboratory of Crop Gene Resources and Breeding, Institute of Crop Sciences, Chinese Academy of Agricultural Sciences, Beijing, 100081 China">
<meta name="citation_author" content="Tianqing Zheng">
<meta name="citation_author_institution" content="State Key Laboratory of Crop Gene Resources and Breeding, Institute of Crop Sciences, Chinese Academy of Agricultural Sciences, Beijing, 100081 China">
<meta name="citation_author" content="Wensheng Wang">
<meta name="citation_author_institution" content="State Key Laboratory of Crop Gene Resources and Breeding, Institute of Crop Sciences, Chinese Academy of Agricultural Sciences, Beijing, 100081 China">
<meta name="citation_author" content="Jianlong Xu">
<meta name="citation_author_institution" content="State Key Laboratory of Crop Gene Resources and Breeding, Institute of Crop Sciences, Chinese Academy of Agricultural Sciences, Beijing, 100081 China">
<meta name="citation_author_institution" content="Shenzhen Branch, Guangdong Laboratory for Lingnan Modern Agriculture, Agricultural Genomics Institute at Shenzhen, Chinese Academy of Agricultural Sciences, Shenzhen, 518120 China">
<meta name="citation_author_institution" content="National Nanfan Research Institute (Sanya), Chinese Academy of Agricultural Sciences, Sanya, Hainan, 572024 China">
<meta name="citation_publication_date" content="2025 Aug 19">
<meta name="citation_volume" content="26">
<meta name="citation_firstpage" content="756">
<meta name="citation_doi" content="10.1186/s12864-025-11937-8">
<meta name="citation_pmid" content="40830828">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12363062/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12363062/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12363062/pdf/12864_2025_Article_11937.pdf">
<meta name="description" content="Rice, being a thermophilic crop, exhibits high sensitivity to low-temperature stress throughout its growth and development. Consequently, enhancing cold tolerance (CT) has been a paramount objective in rice breeding programs. The budding and ...">
<meta name="og:title" content="Combining QTL mapping and transcriptomics to identify candidate genes for cold tolerance during the budding and seedling stages in rice">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Rice, being a thermophilic crop, exhibits high sensitivity to low-temperature stress throughout its growth and development. Consequently, enhancing cold tolerance (CT) has been a paramount objective in rice breeding programs. The budding and ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12363062/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12363062">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1186/s12864-025-11937-8"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/12864_2025_Article_11937.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12363062%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12363062/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12363062/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12363062/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-bmcgeno.png" alt="BMC Genomics logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to BMC Genomics" title="Link to BMC Genomics" shape="default" href="https://bmcgenomics.biomedcentral.com/" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">BMC Genomics</button></div>. 2025 Aug 19;26:756. doi: <a href="https://doi.org/10.1186/s12864-025-11937-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1186/s12864-025-11937-8</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22BMC%20Genomics%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22BMC%20Genomics%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22BMC%20Genomics%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22BMC%20Genomics%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Combining QTL mapping and transcriptomics to identify candidate genes for cold tolerance during the budding and seedling stages in rice</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kim%20CA%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Chun Ae Kim</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Chun Ae Kim</span></h3>
<div class="p">
<sup>1</sup>State Key Laboratory of Crop Gene Resources and Breeding, Institute of Crop Sciences, Chinese Academy of Agricultural Sciences, Beijing, 100081 China </div>
<div class="p">
<sup>4</sup>Molecular Plant Breeding Lab, Branch of Biotechnology, Democratic, State Academy of Sciences, Pyongyang, Democratic People’s Republic of Korea </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kim%20CA%22%5BAuthor%5D" class="usa-link"><span class="name western">Chun Ae Kim</span></a>
</div>
</div>
<sup>1,</sup><sup>4,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chen%20W%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Wenqiang Chen</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Wenqiang Chen</span></h3>
<div class="p">
<sup>1</sup>State Key Laboratory of Crop Gene Resources and Breeding, Institute of Crop Sciences, Chinese Academy of Agricultural Sciences, Beijing, 100081 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chen%20W%22%5BAuthor%5D" class="usa-link"><span class="name western">Wenqiang Chen</span></a>
</div>
</div>
<sup>1,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhu%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Shuangbing Zhu</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Shuangbing Zhu</span></h3>
<div class="p">
<sup>2</sup>Shenzhen Branch, Guangdong Laboratory for Lingnan Modern Agriculture, Agricultural Genomics Institute at Shenzhen, Chinese Academy of Agricultural Sciences, Shenzhen, 518120 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhu%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Shuangbing Zhu</span></a>
</div>
</div>
<sup>2,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20G%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Guogen Zhang</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Guogen Zhang</span></h3>
<div class="p">
<sup>1</sup>State Key Laboratory of Crop Gene Resources and Breeding, Institute of Crop Sciences, Chinese Academy of Agricultural Sciences, Beijing, 100081 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20G%22%5BAuthor%5D" class="usa-link"><span class="name western">Guogen Zhang</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Shen%20C%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Congcong Shen</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Congcong Shen</span></h3>
<div class="p">
<sup>2</sup>Shenzhen Branch, Guangdong Laboratory for Lingnan Modern Agriculture, Agricultural Genomics Institute at Shenzhen, Chinese Academy of Agricultural Sciences, Shenzhen, 518120 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Shen%20C%22%5BAuthor%5D" class="usa-link"><span class="name western">Congcong Shen</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chen%20K%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Kai Chen</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Kai Chen</span></h3>
<div class="p">
<sup>2</sup>Shenzhen Branch, Guangdong Laboratory for Lingnan Modern Agriculture, Agricultural Genomics Institute at Shenzhen, Chinese Academy of Agricultural Sciences, Shenzhen, 518120 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chen%20K%22%5BAuthor%5D" class="usa-link"><span class="name western">Kai Chen</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhao%20X%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Xiuqin Zhao</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Xiuqin Zhao</span></h3>
<div class="p">
<sup>1</sup>State Key Laboratory of Crop Gene Resources and Breeding, Institute of Crop Sciences, Chinese Academy of Agricultural Sciences, Beijing, 100081 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhao%20X%22%5BAuthor%5D" class="usa-link"><span class="name western">Xiuqin Zhao</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zheng%20T%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Tianqing Zheng</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Tianqing Zheng</span></h3>
<div class="p">
<sup>1</sup>State Key Laboratory of Crop Gene Resources and Breeding, Institute of Crop Sciences, Chinese Academy of Agricultural Sciences, Beijing, 100081 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zheng%20T%22%5BAuthor%5D" class="usa-link"><span class="name western">Tianqing Zheng</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20W%22%5BAuthor%5D" class="usa-link" aria-describedby="id9"><span class="name western">Wensheng Wang</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">Wensheng Wang</span></h3>
<div class="p">
<sup>1</sup>State Key Laboratory of Crop Gene Resources and Breeding, Institute of Crop Sciences, Chinese Academy of Agricultural Sciences, Beijing, 100081 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20W%22%5BAuthor%5D" class="usa-link"><span class="name western">Wensheng Wang</span></a>
</div>
</div>
<sup>1,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Xu%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id10"><span class="name western">Jianlong Xu</span></a><div hidden="hidden" id="id10">
<h3><span class="name western">Jianlong Xu</span></h3>
<div class="p">
<sup>1</sup>State Key Laboratory of Crop Gene Resources and Breeding, Institute of Crop Sciences, Chinese Academy of Agricultural Sciences, Beijing, 100081 China </div>
<div class="p">
<sup>2</sup>Shenzhen Branch, Guangdong Laboratory for Lingnan Modern Agriculture, Agricultural Genomics Institute at Shenzhen, Chinese Academy of Agricultural Sciences, Shenzhen, 518120 China </div>
<div class="p">
<sup>3</sup>National Nanfan Research Institute (Sanya), Chinese Academy of Agricultural Sciences, Sanya, Hainan, 572024 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Xu%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Jianlong Xu</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>3,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>State Key Laboratory of Crop Gene Resources and Breeding, Institute of Crop Sciences, Chinese Academy of Agricultural Sciences, Beijing, 100081 China </div>
<div id="Aff2">
<sup>2</sup>Shenzhen Branch, Guangdong Laboratory for Lingnan Modern Agriculture, Agricultural Genomics Institute at Shenzhen, Chinese Academy of Agricultural Sciences, Shenzhen, 518120 China </div>
<div id="Aff3">
<sup>3</sup>National Nanfan Research Institute (Sanya), Chinese Academy of Agricultural Sciences, Sanya, Hainan, 572024 China </div>
<div id="Aff4">
<sup>4</sup>Molecular Plant Breeding Lab, Branch of Biotechnology, Democratic, State Academy of Sciences, Pyongyang, Democratic People’s Republic of Korea </div>
<div class="author-notes p">
<div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div>
<div class="fn" id="_eqcntrb93pmc__">
<sup>#</sup><p class="display-inline">Contributed equally.</p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 Apr 17; Accepted 2025 Jul 23; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by-nc-nd/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12363062  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40830828/" class="usa-link">40830828</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<section id="sec1"><h3 class="pmc_sec_title">Background</h3>
<p id="Par1">Rice, being a thermophilic crop, exhibits high sensitivity to low-temperature stress throughout its growth and development. Consequently, enhancing cold tolerance (CT) has been a paramount objective in rice breeding programs. The budding and seedling stages are particularly susceptible to low-temperature damage, making it crucial to improve CT during these stages to ensure the stable establishment and development of the rice population.</p></section><section id="sec2"><h3 class="pmc_sec_title">Results</h3>
<p id="Par2">In this study, we exposed the parental lines Nipponbare (NIP) and Searice 86 (SR86), along with their derived 170 doubled-haploid (DH) population lines, to cold treatments during both the budding and seedling stages. Quantitative trait locus (QTL) mapping was performed using statistical indices such as the survival rate at the budding stage (SRBS), severity of damage at the budding stage (SDBS), survival rate at the seedling stage (SRSS), and wilting degree at the seedling stage (WDSS). This analysis identified four QTLs at the budding stage and eight QTLs at the seedling stage. Furthermore, by integrating differentially expressed genes (DEGs) from transcriptomic data with genes located within the QTL regions, we identified 10 candidate genes for the budding stage and 11 candidate genes for the seedling stage. Based on DNA sequence variations between the parental lines, changes in gene expression under cold treatment, and haplotype analyses, the key candidate genes were ultimately determined to be <em>Os02g0250600</em> for the budding stage and <em>Os06g0696600</em> for the seedling stage. Additionally, Gene Ontology (GO) and Kyoto Encyclopedia of Genes and Genomes (KEGG) enrichment analyses of transcriptomic data from both stages revealed significant differences in the regulatory pathways involved in CT between the budding and seedling stages.</p></section><section id="sec3"><h3 class="pmc_sec_title">Conclusion</h3>
<p id="Par3">The results indicate that <em>Os02g0250600</em> is the pivotal gene responsible for CT at the budding stage, with haplotype 4 exhibiting the highest level of CT. Meanwhile, <em>Os06g0696600</em> plays a crucial role in CT at the seedling stage, where haplotypes 2 and 4 have been identified as advantageous. A comprehensive analysis integrating QTL and transcriptome data from both stages revealed distinct differences in CT mechanisms, highlighting stage-specific variations. This study provides valuable theoretical insights and practical references for the cloning of CT genes and the development of cold-tolerant rice varieties during the budding and seedling stages.</p></section><section id="sec4"><h3 class="pmc_sec_title">Supplementary Information</h3>
<p>The online version contains supplementary material available at 10.1186/s12864-025-11937-8.</p></section><section id="kwd-group1" lang="en" class="kwd-group"><p><strong>Keywords:</strong> Quantitative trait locus (QTL), Transcriptomics, Cold tolerance (CT), Candidate genes, Rice</p></section></section><section id="Sec1"><h2 class="pmc_sec_title">Background</h2>
<p id="Par24">Rice (<em>Oryza sativa</em> L.) is one of the most important staple crops globally, feeding approximately half of the world’s population [<a href="#CR1" class="usa-link" aria-describedby="CR1">1</a>]. However, rice growth is highly sensitive to environmental conditions, particularly temperature fluctuations [<a href="#CR2" class="usa-link" aria-describedby="CR2">2</a>]. With the intensification of global climate change, extreme weather events have become more frequent, posing significant challenges to rice cultivation in various regions due to cold stress. This stress not only affects seed germination and seedling growth but can also lead to severe yield reductions or even complete crop failure [<a href="#CR3" class="usa-link" aria-describedby="CR3">3</a>–<a href="#CR5" class="usa-link" aria-describedby="CR5">5</a>]. Therefore, developing cold-tolerant rice varieties has become a crucial goal for ensuring global food security and promoting sustainable agriculture.</p>
<p id="Par25">The research and promotion of cold-tolerant rice have significant practical value worldwide. In temperate regions such as Japan, Korea, northeastern China, California in the United States and parts of Europe (e.g., Italy and Spain), low temperatures in spring often hinder rice growth, affecting early development and final yield [<a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>–<a href="#CR8" class="usa-link" aria-describedby="CR8">8</a>]. In recent years, with the increasing adoption of direct-seeded rice technology, CT during the budding and seedling stages has become a key focus in rice breeding [<a href="#CR9" class="usa-link" aria-describedby="CR9">9</a>]. By selecting and cultivating cold-tolerant varieties, the negative impact of early spring low temperatures on sprout and seedling can be effectively reduced, facilitating the expansion of direct-seeded rice cultivation [<a href="#CR10" class="usa-link" aria-describedby="CR10">10</a>]. Additionally, in high-altitude regions such as the Kunming province and Qinghai-Tibet Plateau in southwestern China, the Andes in South America, and the Himalayan countries including Nepal, Bhutan, and northern India, large diurnal temperature variations and frequent extreme cold weather during spring and autumn severely restrict rice cultivation and yield [<a href="#CR8" class="usa-link" aria-describedby="CR8">8</a>]. However, with population growth and increasing food demand, the need for rice cultivation in these areas has been rising. In recent years, successful trials of cold-tolerant rice varieties have demonstrated that, through genetic improvement and advanced breeding techniques, rice cultivation can be expanded to high-altitude regions, enhancing local food self-sufficiency.</p>
<p id="Par26">The genetics of rice CT is complex and classified as a quantitative trait governed by multiple genes. Studies focusing on CT during the budding and seedling stages have identified several QTLs associated with these traits. Yang et al. [<a href="#CR11" class="usa-link" aria-describedby="CR11">11</a>] conducted high-throughput sequencing analysis on 460 F<sub>2:3</sub> mapping population individuals, identifying a key QTL for CT at the bud-bursting stage, designated <em>qCTBB9</em>. By analyzing changes in gene expression levels under cold stress and performing haplotype analysis, <em>Os09g0444200</em> was pinpointed as the candidate gene for this locus. Similarly, Zhang et al. [<a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>] performed a genome-wide association study (GWAS) on 249 indica rice varieties, detecting 47 significant single-nucleotide polymorphism (SNP) loci associated with severity of damage (SD) and survival rate (SR). Through linkage disequilibrium decay analysis, GO enrichment analysis, RNA-seq data, and gene expression pattern analysis, three potential CT candidate genes (<em>LOC_Os01g55510</em>, <em>LOC_Os01g55350</em>, and <em>LOC_Os01g55560</em>) were identified. Recently, Wu et al. [<a href="#CR12" class="usa-link" aria-describedby="CR12">12</a>] constructed a recombinant inbred line (RIL) population by crossing NIP with WD16343 and identified a major CT QTL, <em>qCTS11</em>, on chromosome 11 through bulked segregant analysis (BSA)-seq, leading to the discovery of two CT genes, <em>COLD11</em> and <em>OsCTS11</em>. Shi et al. [<a href="#CR13" class="usa-link" aria-describedby="CR13">13</a>] investigated the seedling-stage CT traits of 1,992 rice germplasms and conducted GWAS analysis on SR and leaf cold tolerance (SCT), identifying a total of 29 QTLs associated with seedling-stage CT, among which four QTLs (<em>qSR3.1a</em>, <em>qSR4.1a</em>, <em>qSR11.1x</em>, and <em>qSR12.1a</em>) played significant roles. To date, numerous CT genes have been identified and cloned in rice. These include <em>qLTG3</em> [<a href="#CR14" class="usa-link" aria-describedby="CR14">14</a>], <em>OsSAP16</em> [<a href="#CR15" class="usa-link" aria-describedby="CR15">15</a>], <em>OsLTPL159</em> [<a href="#CR16" class="usa-link" aria-describedby="CR16">16</a>], and <em>OsUBC12</em> [<a href="#CR17" class="usa-link" aria-describedby="CR17">17</a>], which confer CT during the germination stage; <em>COLD1</em> [<a href="#CR18" class="usa-link" aria-describedby="CR18">18</a>], <em>bZIP73</em> [<a href="#CR19" class="usa-link" aria-describedby="CR19">19</a>], <em>qPSR10</em> [<a href="#CR20" class="usa-link" aria-describedby="CR20">20</a>], <em>HAN1</em> [<a href="#CR21" class="usa-link" aria-describedby="CR21">21</a>], <em>COG1</em> [<a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>], <em>COLD11</em> [<a href="#CR23" class="usa-link" aria-describedby="CR23">23</a>], <em>SOP10</em> [<a href="#CR24" class="usa-link" aria-describedby="CR24">24</a>], and <em>COG3</em> [<a href="#CR25" class="usa-link" aria-describedby="CR25">25</a>], which regulate CT at the seedling stage; and <em>Ctb1</em> [<a href="#CR26" class="usa-link" aria-describedby="CR26">26</a>], <em>CTB4a</em> [<a href="#CR27" class="usa-link" aria-describedby="CR27">27</a>], <em>LTT1</em> [<a href="#CR28" class="usa-link" aria-describedby="CR28">28</a>], <em>CTB2</em> [<a href="#CR8" class="usa-link" aria-describedby="CR8">8</a>], <em>LEA9</em> [<a href="#CR29" class="usa-link" aria-describedby="CR29">29</a>], <em>CTB5</em> [<a href="#CR30" class="usa-link" aria-describedby="CR30">30</a>], and <em>CTB6</em> [<a href="#CR31" class="usa-link" aria-describedby="CR31">31</a>], which influence CT at the booting stage. Despite these advancements, only a limited number of cloned genes have been effectively utilized in practical breeding programs for enhancing CT. Consequently, the identification and exploitation of new favorable alleles or haplotypes from germplasm resources remain crucial for efficient rice breeding aimed at improving CT.</p>
<p id="Par27">This study employed the cold-tolerant japonica rice variety NIP and the cold-sensitive indica rice variety SR86, along with their DH population, as experimental materials. We evaluated CT-related traits, including SDBS and SRBS at the budding stage, and SRSS and WDSS at the seedling stage under low-temperature stress conditions to identify QTLs associated with CT in rice. Transcriptome analysis was conducted on the parental lines and extreme progenies under both normal and cold stress conditions. The DEGs were then mapped to the QTL regions, leading to the identification of 10 candidate genes at the budding stage and 11 at the seedling stage. Further transcriptome and haplotype analyses of these genes identified <em>Os02g0250600</em> as a key candidate gene for the budding stage and <em>Os06g0696600</em> for the seedling stage. Additionally, GO and KEGG pathway analyses were performed on the DEGs under both conditions, revealing important regulatory pathways involved in rice CT. These findings provide a foundation for elucidating the molecular mechanisms of CT and offer valuable resources for breeding cold-tolerant rice varieties.</p></section><section id="Sec2"><h2 class="pmc_sec_title">Materials and methods</h2>
<section id="Sec3"><h3 class="pmc_sec_title">Plant materials</h3>
<p id="Par28">The experimental materials utilized in this study encompass two parental lines, Searice 86 (SR86) and Nipponbare (NIP), along with a DH population. This DH population was previously developed by our research team through the hybridization of the two parental lines followed by anther culture. SR86, an ancient indica subspecies that is phylogenetically proximate to the divergence point of major rice varieties, has been extensively employed as a donor parent for salt tolerance [<a href="#CR32" class="usa-link" aria-describedby="CR32">32</a>]. The DH population comprises 170 lines. All plant cultivation and cold treatment experiments were conducted at the Institute of Crop Sciences (39.9578°N, 116.3163°E), Chinese Academy of Agricultural Sciences.</p></section><section id="Sec4"><h3 class="pmc_sec_title">Identification of CT at the budding stage</h3>
<p id="Par29">Dormancy was broken by exposing seeds to a 50 °C oven for five consecutive days. Subsequently, the seeds were subjected to sodium hypochlorite treatment, followed by thorough rinsing and soaking to enhance germination. Thirty seeds exhibiting a shoot length of 5 mm were selected and placed in Petri dishes containing 10 mL of distilled water. The dishes were incubated in a growth chamber set at 4 °C under dark conditions for ten days with the reference to Li et al. [<a href="#CR3" class="usa-link" aria-describedby="CR3">3</a>], with three replicates. Following the cold treatment, the Petri dishes containing the seeds were transferred to an incubator maintained at 30 °C with a photoperiod of 14 h light and 10 h darkness for a recovery period of five days. The SDBS was evaluated after three days of recovery, using the grading scale provided in Table <a href="#Tab1" class="usa-link">1.</a>. The SRBS was assessed after five days of recovery and calculated as the percentage of surviving seedlings out of the initial 30.</p>
<section class="tw xbox font-sm" id="Tab1"><h4 class="obj_head">Table 1.</h4>
<div class="caption p"><p>Scoring of CT at the budding and seedling stages</p></div>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12363062_12864_2025_11937_Tab1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/b6c5/12363062/cd6911cac1c9/12864_2025_11937_Tab1_HTML.jpg" loading="lazy" id="MO1" height="276" width="740" alt="graphic file with name 12864_2025_11937_Tab1_HTML.jpg"></a></p>
<div class="p text-right font-secondary"><a href="table/Tab1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section></section><section id="Sec5"><h3 class="pmc_sec_title">Identification of CT at the seedling stage</h3>
<p id="Par30">The dormancy-broken seeds were subjected to surface sterilization using a sodium hypochlorite solution, followed by thorough rinsing and soaking to promote germination. Germinated seeds were then placed in PCR plates with perforated bottoms and transferred to a climate chamber maintained at 30 °C with a photoperiod of 14 h light/10 hours dark for hydroponic cultivation. Each treatment group consisted of 8 seedlings, with three biological replicates. Upon reaching the three-leaf and one-heart stage, the seedlings underwent a 6-day low-temperature stress treatment at 5 °C with the reference to Wu et al. [<a href="#CR12" class="usa-link" aria-describedby="CR12">12</a>], while all other conditions remained constant. Following this, the seedlings were returned to the 30 °C climate chamber under the same photoperiod for a 7-day recovery period. At the conclusion of the recovery period, WDSS and SRSS were assessed. The WDSS grading scale is detailed in Table <a href="#Tab1" class="usa-link">1</a>. The survival rate was calculated as follows: SRSS (%) = (Number of survival plants/Total number of plants) × 100.</p></section><section id="Sec6"><h3 class="pmc_sec_title">QTL Analysis</h3>
<p id="Par31">High-quality SNP markers, characterized by less than 5% missing data and exhibiting polymorphism between the parental lines, were selected from the 40 K SNP dataset for genetic map construction and QTL mapping analysis. Redundant markers were eliminated using the BIN function in ICI Mapping (version 4.2). A genetic linkage map was subsequently constructed based on the remaining SNP markers. QTL mapping for the four CT-related traits was conducted using the BIP function [<a href="#CR33" class="usa-link" aria-describedby="CR33">33</a>], with a LOD threshold of 2.5 set for QTL identification. Regions with LOD values exceeding 2.5 were deemed to contain QTLs, and their phenotypic variation rates and additive effects were calculated. In the DH population, a negative additive effect signifies that the allele originates from the parent “SR86”, whereas a positive additive effect indicates an origin from the parent “NIP”.</p></section><section id="Sec7"><h3 class="pmc_sec_title">Transcriptome sequencing and analysis of the parental varieties and their DH progeny</h3>
<p id="Par32">Based on their responses to cold treatment at the budding and seedling stages, NIP was identified as a cold-tolerant material, whereas SR86 was classified as a cold-sensitive material. According to the performance of CT and QTL mapping results, the following DH lines were selected: DH58, which exhibited CT at both budding and seedling stages; DH253 and DH200, which showed cold sensitivity at these stages. After seed soaking and germination, seeds from the four lines (NIP, SR86, DH58, and DH253) were divided into two groups: one group was grown at room temperature as the control, while the other group was subjected to a 4 °C treatment. Following a 48-hour cold treatment, samples from the control group were collected and designated as BS-NIP-CK, BS-SR86-CK, BS-DH58-CK, and BS-DH253-CK, respectively. Similarly, samples from the treatment group were collected and named BS-NIP-LT, BS-SR86-LT, BS-DH58-LT, and BS-DH253-LT. Consistent protocols were employed for the sampling of four lines (NIP, SR86, DH58, and DH200) at the seedling stage. Upon reaching the three-leaf and one-heart stage, the experimental group was subjected to a 4 °C treatment for 48 h. The above-ground parts of each sample were collected and designated as SS-NIP-LT, SS-SR86-LT, SS-DH58-LT, and SS-DH200-LT. Concurrently, the control group samples were collected at room temperature and named SS-NIP-CK, SS-SR86-CK, SS-DH58-CK, and SS-DH200-CK. Each sample was collected in triplicate for biological replication and immediately frozen in liquid nitrogen. RNA extraction, cDNA library preparation, and sequencing were conducted by Beijing Tsingke Biotech Co., Ltd. Differential expression analysis was performed using DESeq2 software, with a significance threshold set at <em>P</em>-value ≤ 0.05 and a fold change of |log2FC| ≥ 1. Functional enrichment analysis of GO and KEGG pathways was conducted using clusterProfiler software [<a href="#CR34" class="usa-link" aria-describedby="CR34">34</a>, <a href="#CR35" class="usa-link" aria-describedby="CR35">35</a>]. KEGG pathway enrichment was assessed using a <em>P</em>-value &lt; 0.05, while GO terms with a <em>P</em>-value &lt; 0.01 were considered significantly enriched.</p></section><section id="Sec8"><h3 class="pmc_sec_title">The identified key genes underwent haplotype analysis</h3>
<p id="Par33">The data utilized for the analysis of cold-tolerant haplotypes at the budding stage were derived from previously published results by our laboratory [<a href="#CR36" class="usa-link" aria-describedby="CR36">36</a>]. The population, sourced from the 3 K Rice Genome Project [<a href="#CR37" class="usa-link" aria-describedby="CR37">37</a>], comprised 513 accessions: 129 from the Aus subgroup, 35 from the Basmati subgroup, 119 from the GJ subgroup, and 230 from the XI subgroup. Similarly, the CT data at the seedling stage were obtained from our laboratory’s recent study, encompassing 1,992 accessions: 473 from the GJ subgroup, 1,290 from the XI subgroup, 133 from the Aus subgroup, 35 from the Basmati subgroup, and 133 classified as admixtures [<a href="#CR13" class="usa-link" aria-describedby="CR13">13</a>]. Haplotypes with phenotypic counts exceeding 30 were selected to investigate the potential association between the identified key genes and CT in rice.</p></section><section id="Sec9"><h3 class="pmc_sec_title">Quantitative real-time PCR</h3>
<p id="Par34">The RNA extraction materials were consistent with those used for transcriptome sequencing. Total RNA (1 µg) was reverse-transcribed into cDNA using FastKing gDNA Dispelling RT SuperMix kit (Tiangen, China, KR118-02). qRT-PCR was performed using a Light Cycler 480 instrument (Roche, Switzerland, 05015278001) with the Taq Pro Universal SYBR qPCR Master Mix (Vazyme, China, Q712) including three biological replicates. <em>UBQ</em> was used as the internal control, and the relative expression levels of the target genes were quantified using the comparative cycle threshold method [<a href="#CR38" class="usa-link" aria-describedby="CR38">38</a>]. Eight genes were randomly selected from the differentially expressed genes identified in the budding and seedling stage transcriptomes for qRT-PCR validation, respectively. Primers used for qRT-PCR are listed in Table S37.</p></section></section><section id="Sec10"><h2 class="pmc_sec_title">Results</h2>
<section id="Sec11"><h3 class="pmc_sec_title">Phenotypic analysis of the budding and seedling stages under low-temperature conditions</h3>
<p id="Par35">To assess the CT of the parental lines NIP and SR86, we evaluated the survival rates (SRBS and SRSS) and damage degrees (SDBS and WDSS) following cold treatment at both the budding and seedling stages (Table <a href="#Tab2" class="usa-link">2</a>, Fig. S1). The results indicated that the SRBS and SRSS of NIP were significantly higher than those of SR86, while the SDBS and WDSS were significantly lower. These findings suggest substantial genetic differences in CT traits between NIP and SR86 at both developmental stages, rendering the populations derived from them suitable for QTL analysis and candidate gene identification. Phenotypic evaluation of the DH population revealed that SRBS ranged from 0 to 100%, with an average of 55.84%; SDBS ranged from 1.02 to 9.00, with an average of 5.07; SRSS ranged from 0 to 100%, with an average of 30.30%; and WDSS ranged from 1.00 to 9.00, with an average of 6.19 (Table 2; Fig. 1A-D, Table S1, S2). Correlation analysis under cold stress showed that the correlation coefficients between SRBS and SDBS, and between SRSS and WDSS, were both below − 0.95, indicating a strong negative correlation between survival rate and damage degree within the same treatment stage. However, the correlation coefficients between traits across different stages were relatively low, suggesting distinct genetic mechanisms control CT at the budding and seedling stages (Fig. <a href="#Fig1" class="usa-link">1</a>E).</p>
<section class="tw xbox font-sm" id="Tab2"><h4 class="obj_head">Table 2.</h4>
<div class="caption p"><p>Performance of parents and 170 DH lines for cold tolerance-related traits at budding and seedling stages.</p></div>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12363062_12864_2025_11937_Tab2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/b6c5/12363062/fb4f8dc72ed6/12864_2025_11937_Tab2_HTML.jpg" loading="lazy" id="MO2" height="203" width="696" alt="graphic file with name 12864_2025_11937_Tab2_HTML.jpg"></a></p>
<div class="p text-right font-secondary"><a href="table/Tab2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><figure class="fig xbox font-sm" id="Fig1"><h4 class="obj_head">Fig. 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12363062_12864_2025_11937_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/b6c5/12363062/123619b87335/12864_2025_11937_Fig1_HTML.jpg" loading="lazy" id="MO3" height="1215" width="744" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>The distribution and correlation analysis of four cold tolerance-related traits in the DH population. A, The distribution of survival rates of the DH population at the budding stage. B, The distribution of severity of damage of the DH population at the budding stage. C, The distribution of survival rates of the DH population at the seedling stage. D, The distribution of severity of damage of the DH population at the seedling stage. E, The correlation analysis of four cold tolerance-related traits in the DH population. The red line in the scatter plots represents the correlation trend. ** indicate significance at <em>P</em> &lt; 0.01</p></figcaption></figure></section><section id="Sec12"><h3 class="pmc_sec_title">QTLs identified for cold-tolerance related traits</h3>
<p id="Par37">CT QTL analysis was conducted using ICI Mapping 4.2 software, which identified a total of 12 QTLs: 2 SRBS, 2 SDBS, 5 SRSS, and 3 WDSS, distributed across chromosomes 2, 3, 6, 9, 11, and 12. Notably, the QTLs <em>SRBS2-1</em>, <em>SRBS2-2</em>, <em>SRSS9-1</em>, and <em>SRSS11-1</em> are colocalized with <em>SDBS2-1</em>, <em>SDBS2-2</em>, <em>WDSS9-1</em>, and <em>WDSS11-1</em>, respectively (Table <a href="#Tab3" class="usa-link">3</a>). Based on the confidence intervals of each QTL, the number of genes within each region was determined as follows: <em>SRBS2-1</em>/<em>SDBS2-1</em> contains 70 genes (Table S3); <em>SRBS2-2</em>/<em>SDBS2-2</em> contains 41 genes (Table S4); <em>SRSS3-1</em> contains 8 genes (Table S5); <em>WDSS3-1</em> contains 15 genes (Table S6); <em>SRSS6-1</em> contains 95 genes (Table S7); <em>SRSS9-1</em>/<em>WDSS9-1</em> contains 19 genes (Table S8); <em>SRSS11-1</em>/<em>WDSS11-1</em> contains 19 genes (Table S9); and <em>SRSS12-1</em> contains three genes (Table S10).</p>
<section class="tw xbox font-sm" id="Tab3"><h4 class="obj_head">Table 3.</h4>
<div class="caption p"><p>QTL identified for cold tolerance-related traits from the DH population.</p></div>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12363062_12864_2025_11937_Tab3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/b6c5/12363062/fd609158e466/12864_2025_11937_Tab3_HTML.jpg" loading="lazy" id="MO4" height="360" width="775" alt="graphic file with name 12864_2025_11937_Tab3_HTML.jpg"></a></p>
<div class="p text-right font-secondary"><a href="table/Tab3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section></section><section id="Sec13"><h3 class="pmc_sec_title">Identification of candidate genes through transcriptome analysis at the budding stage</h3>
<p id="Par38">The mRNA sequencing of 24 samples collected during the budding stage yielded a total of 38,640,054 to 53,279,300 raw reads. Following stringent quality control and filtering procedures, we obtained 34,978,336 to 49,081,006 clean reads. The proportion of high-quality bases (Q30) across all samples exceeded 94.25%, with overall mapping rates ranging from 93.36 to 97.76% and unique mapping rates varying between 83.72% and 95.49% (Table S11). Principal component analysis (PCA) based on transcriptome data demonstrated clear separation among different varieties and consistent clustering within the same variety, thereby validating the robustness and reliability of our data (Fig. S2A). To verify the accuracy of the RNA-seq data at the budding stage, eight different types of DEGs were randomly selected for qRT-PCR analysis. The results showed that the expression patterns of all genes were consistent with the RNA-seq data (Fig. S3).</p>
<p id="Par39">To explore transcriptomic differences under cold stress at the budding stage, RNA-seq comparisons were made between NIP and SR86, and between DH58 and DH253. A total of 4,600, 6,128, 9,123, and 6,809 DEGs were identified for G1 (BS-NIP-CK vs. BS-SR86-CK), G2 (BS-NIP-LT vs. BS-SR86-LT), G3 (BS-DH58-CK vs. BS-DH253-CK), and G4 (BS-DH58-LT vs. BS-DH253-LT), respectively. Among these, 3,051, 3,553, 4,801, and 3,464 DEGs were down-regulated, while 1,549, 2,575, 4,322, and 3,345 DEGs were up-regulated in G1, G2, G3, and G4, respectively (Fig. <a href="#Fig2" class="usa-link">2</a>A-C). We compared the DEGs with the genes contained in the CT QTLs at the budding stage and found that 76 BS-QTL-candidates did not match any DEGs. This indicates that these genes showed no differential expression between parents or extreme individuals under either normal or low-temperature conditions. Among the BS-QTL-candidates, 9 matched with G3, 4 matched with G2, 2 matched with G4, and 1 matched with G1, suggesting that these genes exhibited differential expression in only one specific condition. Additionally, 4 BS-QTL-candidates matched both G2 and G4, 2 matched both G2 and G3, 2 matched both G1 and G2, and 2 matched both G1 and G4, indicating that these genes were differentially expressed in two conditions. Furthermore, 3 BS-QTL-candidates matched G1, G2, and G4, while 2 matched G1, G3, and G4, showing differential expression in three conditions. Finally, 5 BS-QTL-candidates matched all four conditions, indicating differential expression across all scenarios (Fig. <a href="#Fig2" class="usa-link">2</a>D, Table S3, S4, S12). Since differences in gene expression among materials often lead to phenotypic variation, we used the criterion of differential expression in at least three groups to select candidate genes, ultimately identifying 10 key candidate genes. Specifically, <em>Os02g0282900</em>, <em>Os02g0249600</em>, <em>Os02g0250600</em>, and <em>Os02g0281200</em> were consistently down-regulated across all four groups. <em>Os02g0285800</em> exhibited down-regulation in G1 but up-regulation in G2, G3, and G4. <em>Os02g0245200</em> was down-regulated in G1, G3, and G4. <em>Os02g0287000</em> showed up-regulation in G1, G3, and G4. <em>Os02g0285900</em> and <em>Os02g0244700</em> were down-regulated in G1, G2, and G4. <em>Os02g0288000</em> was up-regulated in G1, G2, and G4 (Table<a href="#Tab4" class="usa-link">4</a> ).</p>
<figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12363062_12864_2025_11937_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/b6c5/12363062/335e05d9143c/12864_2025_11937_Fig2_HTML.jpg" loading="lazy" id="MO6" height="523" width="669" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Analysis of differentially expressed genes in the transcriptomes of parents and two extreme lines at the budding stage, combined with QTL candidate gene association analysis. A, The number of DEGs in the comparison between the parents and two extreme lines. B, Veen diagram of down-regulation genes. C, Veen diagram of up-regulation genes. D, The overlap between the four groups of DEGs and QTL candidate genes. Treatments include the normal control treatment (CK) and the low-temperature treatment (LT)</p></figcaption></figure><section class="tw xbox font-sm" id="Tab4"><h4 class="obj_head">Table 4.</h4>
<div class="caption p"><p>Candidate genes for cold tolerance at the budding stage. Note: G1:BS-NIP-CK vs. BS-SR86-CK; G2:BS-NIP-LT vs. BS-SR86-LT; G3:BS-DH58-CK vs. BS-DH253-CK; G4:BS-DH58-LT vs. BS-DH253-LT.</p></div>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12363062_12864_2025_11937_Tab4_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/b6c5/12363062/e42e6d6e8b01/12864_2025_11937_Tab4_HTML.jpg" loading="lazy" id="MO5" height="364" width="778" alt="graphic file with name 12864_2025_11937_Tab4_HTML.jpg"></a></p>
<div class="p text-right font-secondary"><a href="table/Tab4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section></section><section id="Sec14"><h3 class="pmc_sec_title">Identification of candidate genes through transcriptome analysis at the seedling stage</h3>
<p id="Par41">The mRNA sequencing of 24 samples treated at the seedling stage yielded a total of 42,126,406 to 58,871,060 raw reads. After stringent quality control filtering, we obtained 36,043,890 to 55,439,556 clean reads. The proportion of high-quality bases (Q30) across all samples exceeded 93.43%, with mapping rates ranging from 92.8 to 98.81% and unique mapping rates spanning from 83.2 to 96.05% (Table S13). PCA based on transcriptome data demonstrated clear separation among different varieties and consistent clustering within the same variety, thereby validating the high reliability of our data (Fig. S2B). Similarly, to verify the accuracy of the RNA-seq data at the seedling stage, eight different types of DEGs were randomly selected for qRT-PCR analysis. The results showed that the expression patterns of all genes were consistent with the RNA-seq data (Fig. S4).</p>
<p id="Par42">To elucidate the transcriptomic differences in response to cold stress at the seedling stage between rice accessions with varying levels of CT, we conducted RNA-seq analyses comparing NIP with SR86 and DH58 with DH200. The analysis identified 5,751, 7,076, 5,938, and 10,442 DEGs for G5 (SS-NIP-CK vs. SS-SR86-CK), G6 (SS-NIP-LT vs. SS-SR86-LT), G7 (SS-DH58-CK vs. SS-DH200-CK), and G8 (SS-DH58-LT vs. SS-DH200-LT), respectively. Specifically, 3,633, 3,538, 2,969, and 5,221 DEGs were down-regulated, while 2,118, 3,793, 3,697, and 5,770 DEGs were up-regulated in G5, G6, G7, and G8, respectively (Fig. <a href="#Fig3" class="usa-link">3</a>A-C). We compared the DEGs with the genes contained in the CT QTLs at the seedling stage and found that 102 SS-QTL-candidates did not match any DEGs. This indicates that these genes showed no differential expression between parents or extreme individuals under either normal or low-temperature conditions. Among the SS-QTL-candidates, 12 matched with G4, 6 matched with G2, 5 matched with G1, and 2 matched with G3, suggesting that these genes exhibited differential expression in only one specific condition. Additionally, 8 SS-QTL-candidates matched both G3 and G4, 7 matched both G2 and G4, 4 matched both G1 and G3, and 2 matched both G1 and G3, indicating that these genes were differentially expressed in two conditions. Furthermore, 6 SS-QTL-candidates matched G2, G3 and G4, 2 matched G1, G3 and G4, and 1 matched G1, G2 and G3, showing differential expression in three conditions. Finally, 2 SS-QTL-candidates matched all four conditions, indicating differential expression across all scenarios (Fig. <a href="#Fig3" class="usa-link">3</a>D, Table S5-10, 14). We used the criterion of differential expression in at least three groups to select candidate genes, ultimately identifying 11 key candidate genes. Notably, <em>Os06g0690200</em> was consistently down-regulated across all four groups. Conversely, <em>Os03g0207475</em> and <em>Os06g0693000</em> were consistently up-regulated. <em>Os06g0696600</em> exhibited down-regulation in G5 but up-regulation in G7 and G8. <em>Os06g0693100</em> showed up-regulation in G5, G7, and G8. Additionally, <em>Os06g0691400</em>, <em>Os06g0695500</em>, <em>Os06g0687450</em>, <em>Os09g0375000</em>, <em>Os11g0499600</em>, and <em>Os03g0189100</em> were up-regulated in G6, G7, and G8 (Table <a href="#Tab5" class="usa-link">5</a>).</p>
<figure class="fig xbox font-sm" id="Fig3"><h4 class="obj_head">Fig. 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12363062_12864_2025_11937_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/b6c5/12363062/afb836d336b2/12864_2025_11937_Fig3_HTML.jpg" loading="lazy" id="MO8" height="507" width="669" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Analysis of differentially expressed genes in the transcriptomes of parents and two extreme lines at the seedling stage, combined with QTL candidate gene association analysis. A, The number of DEGs in the comparison between the parents and two extreme lines. B, Veen diagram of down-regulation genes. C, Veen diagram of up-regulation genes. D, The overlap between the four groups of DEGs and QTL candidate genes. Treatments include the normal control treatment (CK) and the low-temperature treatment (LT)</p></figcaption></figure><section class="tw xbox font-sm" id="Tab5"><h4 class="obj_head">Table 5.</h4>
<div class="caption p"><p>Candidate genes for cold tolerance at the seedling stage. Note: G5: SS-NIP-CK vs. SS-SR86-CK; G6: SS-NIP-LT vs. SS-SR86-LT; G7: SS-DH58-CK vs. SS-DH200-CK; G8: SS-DH58-LT vs. SS-DH200-LT.</p></div>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12363062_12864_2025_11937_Tab5_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/b6c5/12363062/295b4e4b293c/12864_2025_11937_Tab5_HTML.jpg" loading="lazy" id="MO7" height="415" width="753" alt="graphic file with name 12864_2025_11937_Tab5_HTML.jpg"></a></p>
<div class="p text-right font-secondary"><a href="table/Tab5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section></section><section id="Sec15"><h3 class="pmc_sec_title">Haplotype analysis of candidate genes for CT at the budding stage</h3>
<p id="Par44">During the initial phase of our study, we identified a total of 10 candidate genes. To determine the key candidate gene, we first compared the DNA sequences of these 10 genes between the two parental lines. The results showed that 9 genes contained nonsynonymous SNPs, and 4 genes had InDels. Subsequently, we analyzed changes in gene expression levels before and after cold treatment. Among these genes, <em>Os02g0250600</em> showed increased expression levels in both parents following cold treatment, indicating its induction by cold stress (Table S35). We also validated the expression of this gene through qRT-PCR, and the results were consistent with the transcriptome data (Fig. S5A). Therefore, we identified <em>Os02g0250600</em> as the most critical candidate gene for CT at the budding stage. This gene encodes a late embryogenesis abundant (LEA) domain-containing protein (Table <a href="#Tab4" class="usa-link">4</a>). Leveraging published data on CT for 3 K germplasms, we conducted a haplotype analysis of the coding sequence (CDS) region of this gene. The analysis revealed 17 SNPs, which were grouped into five distinct haplotypes. Notably, Hap4 demonstrated the lowest SDBS and the highest SRBS, suggesting it is an advantageous haplotype for CT at the budding stage. Specifically, NIP belongs to Hap4, while SR86 is associated with Hap5, a less favorable haplotype. The distribution of haplotypes across subgroups was as follows: Hap1 was predominantly found in the Aus subgroup, Hap2 appeared mainly in the Aus and XI subgroups, Hap3 was predominant in the GJ and XI subgroups, Hap4 was primarily present in the GJ subgroup, and Hap5 was exclusive to the XI subgroup (Fig. <a href="#Fig4" class="usa-link">4</a>A-D).</p>
<figure class="fig xbox font-sm" id="Fig4"><h4 class="obj_head">Fig. 4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12363062_12864_2025_11937_Fig4_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/b6c5/12363062/523dc997c8d2/12864_2025_11937_Fig4_HTML.jpg" loading="lazy" id="MO9" height="800" width="669" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Haplotype analysis of candidate genes. A, CDS region haplotype analysis of <em>Os02g0250600</em>. B-C, SDBS and SRBS distribution across the whole population for the five haplotypes of <em>Os02g0250600</em>. Distinct letters above each box plot signify significant differences among haplotypes, as per Duncan’s multiple range post-hoc test (<em>P</em> &lt; 0.05). (D) The number of subpopulations in the five haplotypes of <em>Os02g0250600</em>. E, CDS region haplotype analysis of <em>Os06g0696600</em>. E-F, WDSS and SRSS distribution across the whole population for the five haplotypes of <em>Os06g0696600</em>. Distinct letters above each box plot signify significant differences among haplotypes, as per Duncan’s multiple range post-hoc test (<em>P</em> &lt; 0.05). (D) The number of subpopulations in the five haplotypes of <em>Os06g0696600</em>. Bas: Basmati, GJ: Geng/japonica and XI: Xian/indica</p></figcaption></figure></section><section id="Sec16"><h3 class="pmc_sec_title">Haplotype analysis of key genes for CT at the seedling stage</h3>
<p id="Par46">At the seedling stage, a total of 11 candidate genes were identified. Upon comparing the DNA sequences of these genes between the two parental lines, it was observed that eight genes exhibited nonsynonymous SNPs, and 2 genes contained InDels. Expression analysis indicated that <em>Os06g0696600</em> demonstrated elevated expression levels in both parents following cold treatment, suggesting its induction by cold stress (Table S36). We also validated the expression of this gene through qRT-PCR, and the results were consistent with the transcriptome data (Fig. S5B). Consequently, <em>Os06g0696600</em> was identified as the most critical candidate gene for CT at the seedling stage. This gene encodes a xyloglucan endo-transglycosylase homolog (Table <a href="#Tab5" class="usa-link">5</a>). Utilizing published population data, haplotype analysis of the CDS region of this gene revealed three SNPs, which were classified into five distinct haplotypes. Comparative analysis of shoot survival with WDSS and shoot survival with recovery rate SRSS across the five haplotypes showed that Hap2 and Hap4 exhibited significantly lower WDSS and higher SRSS compared to Hap1, Hap3, and Hap5, indicating that Hap2 and Hap4 are advantageous haplotypes for CT at the seedling stage. NIP belongs to Hap2, while SR86 is associated with Hap3. The distribution of haplotypes revealed that Hap1, Hap3, and Hap5 were predominantly present in the XI subgroup, whereas Hap2 and Hap4 were primarily found in the GJ subgroup (Fig. <a href="#Fig4" class="usa-link">4</a>E-H).</p></section><section id="Sec17"><h3 class="pmc_sec_title">Key biological processes and pathways governing CT during the budding and seedling stages in rice</h3>
<p id="Par47">To elucidate the key biological processes and pathways influenced by cold stress during the budding and seedling stages in rice, we conducted GO and KEGG enrichment analyses on DEGs from the same materials under normal and low-temperature conditions. The comparisons encompassed BS-NIP-CK vs. BS-NIP-LT, BS-SR86-CK vs. BS-SR86-LT, BS-DH58-CK vs. BS-DH58-LT, BS-DH253-CK vs. BS-DH253-LT, SS-NIP-CK vs. SS-NIP-LT, SS-SR86-CK vs. SS-SR86-LT, SS-DH58-CK vs. SS-DH58-LT, and SS-DH253-CK vs. SS-DH253-LT. These analyses identified 1,001, 1,551, 1,350, 1,643, 675, 1,224, 963, and 900 significantly enriched GO terms, respectively, as well as 18, 15, 12, 8, 17, 17, 14, and 29 significantly enriched KEGG pathways, respectively (Tables S15-18, 20–23, 25–28, 30–33). In the analysis of the four groups during the budding stage, 164 GO terms and two KEGG pathways were found to be completely overlapping, while for the four groups during the seedling stage, 116 GO terms and two KEGG pathways exhibited complete overlap (Fig. <a href="#Fig5" class="usa-link">5</a>A-D, Tables S19, 24, 29, 34). The common GO enrichment at the budding stage was mainly concentrated in the nucleus (GO:0005634) and cytoplasm (GO:0005737), while KEGG enrichment was primarily found in spliceosome (ko03040) and plant hormone signal transduction pathways (ko04075) (Fig. S6). In contrast, the common GO enrichment at the seedling stage was mainly associated with the chloroplast (GO:0009507) and response to abscisic acid (GO:0009737), and the KEGG enrichment was predominantly involved in metabolic pathways (ko01100) and carbon metabolism (ko01200) (Fig. S7). These results suggest that the overlapping biological processes and pathways may constitute core components of the regulatory mechanisms for CT during the budding and seedling stages in rice. When comparing the core components of CT regulatory mechanisms between the budding and seedling stages, only 14 overlapping GO terms and no overlapping KEGG pathways were identified (Fig. <a href="#Fig5" class="usa-link">5</a>E, F), indicating significant differences in the regulatory networks and pathways involved in CT between these two developmental stages.</p>
<figure class="fig xbox font-sm" id="Fig5"><h4 class="obj_head">Fig. 5.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12363062_12864_2025_11937_Fig5_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/b6c5/12363062/cd3ce587192f/12864_2025_11937_Fig5_HTML.jpg" loading="lazy" id="MO10" height="683" width="669" alt="Fig. 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Comparative analysis of GO and KEGG. A, Venn diagram showing the unique and shared GO terms among NIP, SR86, DH58, and DH253 under cold treatment and control conditions during the budding stage. B, Venn diagram showing the unique and shared KEGG pathways among NIP, SR86, DH58, and DH253 under cold treatment and control conditions during the budding stage. C, Venn diagram showing the unique and shared GO terms among NIP, SR86, DH58, and DH200 under cold treatment and control conditions during the seedling stage. D, Venn diagram showing the unique and shared KEGG pathways among NIP, SR86, DH58, and DH200 under cold treatment and control conditions during the seedling stage. E, Venn diagram showing the unique and shared GO terms between the budding and seedling stages. F, Venn diagram showing the unique and shared KEGG pathways between the budding and seedling stages</p></figcaption></figure></section></section><section id="Sec18"><h2 class="pmc_sec_title">Discussion</h2>
<section id="Sec19"><h3 class="pmc_sec_title">Integrating transcriptomics with QTL mapping effectively identifies candidate genes associated with CT in rice</h3>
<p id="Par49">Low temperature exerts a substantial influence on rice production, particularly during the critical budding and seedling stages, resulting in delayed germination and extensive seedling rot [<a href="#CR39" class="usa-link" aria-describedby="CR39">39</a>]. Therefore, identifying and cloning CT genes from resistant varieties for use in breeding programs remains a key objective in rice improvement. The conventional approach to gene mapping involves crossing cold-tolerant with cold-sensitive varieties to develop recombinant inbred lines, double haploid lines, and introgression line populations [<a href="#CR40" class="usa-link" aria-describedby="CR40">40</a>, <a href="#CR41" class="usa-link" aria-describedby="CR41">41</a>]. However, the identified QTLs are frequently located within broad genomic regions, complicating the process of gene cloning. Transcriptomics offers a powerful tool for rapidly identifying DEGs associated with a target trait by comparing control and treatment samples [<a href="#CR42" class="usa-link" aria-describedby="CR42">42</a>]. Therefore, integrating QTL mapping with transcriptomics can markedly enhance the efficiency of gene localization and has emerged as a prevalent strategy. For instance, Yin et al. [<a href="#CR43" class="usa-link" aria-describedby="CR43">43</a>] combined transcriptomic data from parental lines and extreme progeny with QTL mapping results for anaerobic stress in a DH population, ultimately pinpointing key genes <em>Os05g0498700</em> and <em>Os01g0866100</em>, which negatively regulate coleoptile length and improve rice’s tolerance to anaerobic germination. Similarly, Zhou et al. [<a href="#CR44" class="usa-link" aria-describedby="CR44">44</a>] utilized RNA-seq, BSA-seq, and linkage analysis in a BC₅F₂ population and its parents—Xieqingzao B (XB) and 19H19 (derived from an interspecific cross between Dongxiang wild rice (<em>Oryza rufipogon</em> Griff.) and XB)—to identify <em>LOC_Os12g18729</em> as a CT gene at the seedling stage. In this study, we employed a cold-tolerant variety NIP, a cold-sensitive variety SR86, and a DH population of 170 individuals for QTL mapping under cold treatment at both the budding and seedling stages. Four QTLs were respectively identified at the budding stage and eight at the seedling stage. By comparing with previously reported QTLs, we found that the seedling stage CT QTL <em>qCTS3</em> [<a href="#CR45" class="usa-link" aria-describedby="CR45">45</a>] is colocalized with S<em>RSS3-1</em>, the low-temperature vigor of germination QTL <em>qLVG6</em> [<a href="#CR46" class="usa-link" aria-describedby="CR46">46</a>] colocalized with <em>SRSS6-1</em>, the low-temperature germination QTL <em>qLTG-9</em> [<a href="#CR47" class="usa-link" aria-describedby="CR47">47</a>] colocalized with <em>SRSS9-1</em> and <em>WDSS9-1</em>, and the low-temperature germination QTL q<em>LTG-11-1</em> [<a href="#CR48" class="usa-link" aria-describedby="CR48">48</a>] colocalized with <em>SRSS11-1</em> and <em>WDSS11-1</em>. Transcriptomic analysis was conducted on NIP, SR86, and two extreme materials at both stages, comparing DEGs within the QTL intervals. Consequently, 10 candidate genes for CT were identified at the budding stage and 11 at the seedling stage.</p></section><section id="Sec20"><h3 class="pmc_sec_title">Os02g0250600 and Os06g0696600 play crucial roles in the budding and seedling development stages of rice</h3>
<p id="Par50">Analysis of 10 candidate genes for CT at the budding stage revealed that only <em>Os02g0250600</em> exhibited nonsynonymous SNP variations and InDel differences in the coding regions of both parental lines and was upregulated under cold stress. Haplotype analysis further corroborated this finding, identifying Hap4 as the superior haplotype for CT, predominantly distributed within the GJ subgroup. Based on the distribution of nonsynonymous SNP variations and haplotypes, the three SNPs located at positions 8,516,827, 8,517,867, and 8,518,165 are crucial for the differences in CT among different haplotypes. This gene encodes a LEA domain-containing protein, which is specifically expressed and highly abundant in the embryo and endosperm of seeds (<a href="https://rice.uga.edu/cgi-bin/ORF_infopage.cgi?orf=LOC_Os02g15250" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://rice.uga.edu/cgi-bin/ORF_infopage.cgi? orf=LOC_Os02g15250</a>). Given that the budding stage encompasses the development of the embryo root and shoot, the expression of this gene during this phase is also elevated, underscoring its critical role in conferring CT during germination. LEA proteins are not only present during seed development but can be also strongly induced in vegetative tissues under abiotic stresses such as drought, low temperature, and high salinity [<a href="#CR49" class="usa-link" aria-describedby="CR49">49</a>, <a href="#CR50" class="usa-link" aria-describedby="CR50">50</a>]. As key protective factors in plant responses to cold stress, LEA proteins help maintain cellular structure and functional integrity through multiple mechanisms, including molecular chaperone activity, ion chelation, osmotic regulation, and ice crystal control [<a href="#CR51" class="usa-link" aria-describedby="CR51">51</a>]. In rice, several LEA protein-encoding genes have been previously reported to be associated with cold stress, such as <em>OsLEA5</em> [<a href="#CR52" class="usa-link" aria-describedby="CR52">52</a>], <em>wsi18</em> [<a href="#CR53" class="usa-link" aria-describedby="CR53">53</a>], <em>OsLEA9</em> [<a href="#CR29" class="usa-link" aria-describedby="CR29">29</a>], and <em>OsLEA4</em> [<a href="#CR54" class="usa-link" aria-describedby="CR54">54</a>]. Based on these findings, we propose that <em>Os02g0250600</em> is a key regulator of CT at the budding stage in rice.</p>
<p id="Par51">Among the 11 candidate genes associated with CT at the seedling stage, we identified <em>Os06g0696600</em>, which encodes a member of glycosyl hydrolase family 16 (<a href="https://rice.uga.edu/cgi-bin/ORF_infopage.cgi?orf=LOC_Os06g48180" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://rice.uga.edu/cgi-bin/ORF_infopage.cgi?orf=LOC_Os06g48180</a>). Glycosyl hydrolase family 16 enzymes enhance plant CT through three mechanisms: cell wall remodeling, membrane stabilization, and signal transduction [<a href="#CR55" class="usa-link" aria-describedby="CR55">55</a>, <a href="#CR56" class="usa-link" aria-describedby="CR56">56</a>]. In rice, this gene family comprises 32 homologous genes, four of which have been cloned: <em>OsXTH8</em> [<a href="#CR57" class="usa-link" aria-describedby="CR57">57</a>], <em>OsXTH11</em>, <em>OsXTH19</em>, and <em>OsXTH20</em> [<a href="#CR58" class="usa-link" aria-describedby="CR58">58</a>]. According to RNA-seq data from rice (<a href="https://plantrnadb.com/ricerna/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://plantrnadb.com/ricerna/</a>), the expression levels of these genes are significantly upregulated under cold stress conditions, indicating that glycosyl hydrolase family 16 plays a crucial role in conferring CT during the seedling stage. Haplotype analysis of <em>Os06g0696600</em> revealed that Hap2 and Hap4 are superior haplotypes for CT, predominantly observed in the GJ subgroup. Based on the distribution of nonsynonymous SNP variation and haplotypes, the SNP located at positions 29,144,968 is crucial for the difference in CT among different haplotypes.</p>
<p id="Par52">NIP, as a cold-tolerant rice variety, has had several CT genes cloned from its genome. <em>COG3</em> encodes a calmodulin-binding protein that maintains the protease activity of OsFtsH2, thereby regulating CT in rice during the booting and seedling stages [<a href="#CR25" class="usa-link" aria-describedby="CR25">25</a>]. <em>COLD1</em> encodes a G-protein signaling regulator that, upon sensing low temperatures, interacts with the G-protein α subunit RGA, activates Ca<sup>2+</sup> channels, and subsequently enhances G-protein GTPase activity, thereby improving CT [<a href="#CR18" class="usa-link" aria-describedby="CR18">18</a>]. The functions of these two genes do not appear to be part of the established regulatory networks, highlighting the complexity of CT mechanisms in rice. While the relationship between these genes and CT has been preliminarily analyzed, further validation through genetic materials and molecular biology experiments is essential to confirm their specific roles. In the future, research will focus on CRISPR/Cas9-mediated functional analysis of these key genes, investigation of their interactions with hormonal and metabolic pathways, to further elucidate their roles in regulating responses to cold stress.</p></section><section id="Sec21"><h3 class="pmc_sec_title">Differences in CT mechanisms between the budding and seedling stages</h3>
<p id="Par53">The budding and seedling stages represent distinct developmental phases in rice. The budding stage encompasses the period from seed imbibition until the emergence of the first leaf, characterized by low CT. During this phase, low temperatures can inhibit embryo root and shoot development, resulting in reduced germination rates or growth arrest [<a href="#CR42" class="usa-link" aria-describedby="CR42">42</a>]. In contrast, the seedling stage is marked by relatively developed roots and leaves, enabling active absorption of external nutrients and water. Cold stress during this stage primarily slows growth rates rather than causing direct mortality [<a href="#CR45" class="usa-link" aria-describedby="CR45">45</a>]. Rice CT exhibits stage-specific characteristics, with different QTLs identified at various growth stages within the same population [<a href="#CR59" class="usa-link" aria-describedby="CR59">59</a>]. In this study, we identified CT QTLs at both the budding and seedling stages in the same population. Correlation analysis revealed that traits within the same stage were highly correlated, whereas correlations between traits from different stages were relatively low. QTL analysis showed no overlap between the QTL intervals for the two stages, and comparisons of GO and KEGG analyses indicated minimal overlap. These findings are consistent with previous studies [<a href="#CR4" class="usa-link" aria-describedby="CR4">4</a>, <a href="#CR60" class="usa-link" aria-describedby="CR60">60</a>, <a href="#CR61" class="usa-link" aria-describedby="CR61">61</a>], suggesting that the genetic mechanisms underlying CT at the budding and seedling stages are distinct.</p></section><section id="Sec22"><h3 class="pmc_sec_title">Implication in rice breeding for CT during whole growing stage</h3>
<p id="Par54">Rice plants exhibit varying levels of CT at different developmental stages, demonstrating particular sensitivity during germination, seedling, and especially reproductive phases. Cold stress during the germination and seedling stages adversely affects emergence and survival rates, thereby impacting the morphological establishment of the rice population. Conversely, cold stress during the reproductive growth stage reduces seed setting rates, ultimately affecting rice yield. Studies have shown that genetic factors influencing CT at different developmental stages are largely independent in rice [<a href="#CR5" class="usa-link" aria-describedby="CR5">5</a>, <a href="#CR59" class="usa-link" aria-describedby="CR59">59</a>]. To date, a considerable number of CT-related genes have been identified and cloned in rice during the germination, seedling, and booting stages, as previously mentioned. Following functional verification, the two newly identified genes, <em>Os02g0250600</em> and <em>Os06g0696600</em>, can be integrated with other CT genes at the germination, seedling, and booting stages using marker-assisted selection to develop varieties with enhanced CT across all stages. Meanwhile, given the distinct stage-specific nature of CT in rice, it is recommended that breeding practices optimize the selection criteria and strategies for CT traits at each developmental stage separately, in order to achieve coordinated improvement of CT across multiple growth stages through molecular breeding approaches.</p></section></section><section id="Sec23"><h2 class="pmc_sec_title">Conclusions</h2>
<p id="Par55">Based on the evaluation of four traits, it was observed that NIP demonstrated significantly higher CT than SR86 during both the budding and seedling stages. In the derived DH population, four QTLs associated with CT at the budding stage and eight QTLs at the seedling stage were identified. Transcriptome and haplotype analyses ultimately pinpointed <em>Os02g0250600</em> as the key gene responsible for CT at the budding stage and <em>Os06g0696600</em> as the key gene at the seedling stage. Notably, there was no overlap between the QTLs identified at the budding and seedling stages. Furthermore, GO and KEGG enrichment analyses revealed significant differences in CT mechanisms between the two developmental stages, indicating that rice CT exhibits distinct stage-specific characteristics. This study not only identifies two important candidate CT genes for functional analysis and rice CT breeding applications, but also deepens our understanding of CT mechanisms across different rice developmental stages.</p></section><section id="Sec24"><h2 class="pmc_sec_title">Supplementary Information</h2>
<section class="sm xbox font-sm" id="MOESM1"><div class="media p"><div class="caption">
<a href="/articles/instance/12363062/bin/12864_2025_11937_MOESM1_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 1.</a><sup> (1.2MB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM2"><div class="media p"><div class="caption">
<a href="/articles/instance/12363062/bin/12864_2025_11937_MOESM2_ESM.zip" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 2.</a><sup> (2.3MB, zip) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>Not applicable.</p></section><section id="glossary1" class="glossary"><h2 class="pmc_sec_title">Abbreviations</h2>
<dl class="def-list">
<dt>CT</dt>
<dd><p id="Par4">Cold tolerance</p></dd>
<dt>NIP</dt>
<dd><p id="Par5">Nipponbare</p></dd>
<dt>SR86</dt>
<dd><p id="Par6">Searice 86</p></dd>
<dt>DH</dt>
<dd><p id="Par7">Doubled-haploid</p></dd>
<dt>QTLs</dt>
<dd><p id="Par8">Quantitative trait locus</p></dd>
<dt>SRBS</dt>
<dd><p id="Par9">Survival rate at the budding stage</p></dd>
<dt>SDBS</dt>
<dd><p id="Par10">Severity of damage at the budding stage</p></dd>
<dt>SRSS</dt>
<dd><p id="Par11">Survival rate at the seedling stage</p></dd>
<dt>WDSS</dt>
<dd><p id="Par12">Wilting degree at the seedling stage</p></dd>
<dt>DEGs</dt>
<dd><p id="Par13">Differentially expressed genes</p></dd>
<dt>GO</dt>
<dd><p id="Par14">Gene Ontology</p></dd>
<dt>KEGG</dt>
<dd><p id="Par15">Kyoto Encyclopedia of Genes and Genomes</p></dd>
<dt>GWAS</dt>
<dd><p id="Par16">Genome-wide association study</p></dd>
<dt>SNP</dt>
<dd><p id="Par17">Single-nucleotide polymorphism</p></dd>
<dt>SD</dt>
<dd><p id="Par18">Severity of damage</p></dd>
<dt>SR</dt>
<dd><p id="Par19">Survival rate</p></dd>
<dt>BSA</dt>
<dd><p id="Par20">Bulked segregant analysis</p></dd>
<dt>PCA</dt>
<dd><p id="Par21">Principal component analysis</p></dd>
<dt>LEA</dt>
<dd><p id="Par22">Late embryogenesis abundant</p></dd>
<dt>CDS</dt>
<dd><p id="Par23">Coding sequence</p></dd>
</dl></section><section id="notes1"><h2 class="pmc_sec_title">Authors’ contributions</h2>
<p>JL X and WS W conceived and designed the research. CA K, WQ C, SB Z, CC S, K C, and XQ Z performed experiments. CA K, WQ C, and GG Z participated in data analysis. WQ C and JL X wrote the manuscript. JL X and TQ Z corrected the manuscript. The final manuscript was read and approved by all authors.</p></section><section id="notes2"><h2 class="pmc_sec_title">Funding</h2>
<p>The research was funded by Key Areas Research and Development Programs of Guangdong Province (2022B0202060005), Shenzhen Science and Technology Program (Grant No. KCXFZ20211020163808012), and Nanfan Special Project, CAAS (YBXM2315).</p></section><section id="notes3"><h2 class="pmc_sec_title">Data availability</h2>
<p>The RNA-seq raw data of this article was deposited in the NCBI SRA under accession number PRJNA1259743 and PRJNA1260178. All data generated or analysed during this study are included in this published article and its supplementary information files.</p></section><section id="notes4"><h2 class="pmc_sec_title">Declarations</h2>
<section id="FPar1"><h3 class="pmc_sec_title">Ethics approval and consent to participate</h3>
<p id="Par56">Not applicable.</p></section><section id="FPar2"><h3 class="pmc_sec_title">Consent for publication</h3>
<p id="Par57">Not applicable.</p></section><section id="FPar3"><h3 class="pmc_sec_title">Competing interests</h3>
<p id="Par58">The authors declare no competing interests.</p></section></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm">
<div class="fn p" id="fn1">
<p><strong>Publisher’s Note</strong></p>
<p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p>
</div>
<div class="fn p" id="fn2"><p>Chun Ae Kim, Wenqiang Chen and Shuangbing Zhu contributed equally to this work.</p></div>
</div></section><section id="_ci93_" lang="en" class="contrib-info"><h2 class="pmc_sec_title">Contributor Information</h2>
<p>Wensheng Wang, Email: wangwensheng@caas.cn.</p>
<p>Jianlong Xu, Email: xujianlong@caas.cn.</p></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Ghazy MI, El-Naem SA, Hefeina AG, Sallam A, Eltaher S. Genome-wide association study of rice diversity panel reveals new QTLs for tolerance to water deficit under the Egyptian conditions. Rice. 2024;17(1): 29.
</cite> [<a href="https://doi.org/10.1186/s12284-024-00703-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11035518/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38649523/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ghazy%20MI,%20El-Naem%20SA,%20Hefeina%20AG,%20Sallam%20A,%20Eltaher%20S.%20Genome-wide%20association%20study%20of%20rice%20diversity%20panel%20reveals%20new%20QTLs%20for%20tolerance%20to%20water%20deficit%20under%20the%20Egyptian%20conditions.%20Rice.%202024;17(1):%2029." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>Jia Y, Liu H, Qu Z, Wang J, Wang X, Wang Z, et al. Transcriptome sequencing and iTRAQ of different rice cultivars provide insight into molecular mechanisms of cold-tolerance response in Japonica rice. Rice. 2020;13(1):43.
</cite> [<a href="https://doi.org/10.1186/s12284-020-00401-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7310054/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32572635/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jia%20Y,%20Liu%20H,%20Qu%20Z,%20Wang%20J,%20Wang%20X,%20Wang%20Z,%20et%20al.%20Transcriptome%20sequencing%20and%20iTRAQ%20of%20different%20rice%20cultivars%20provide%20insight%20into%20molecular%20mechanisms%20of%20cold-tolerance%20response%20in%20Japonica%20rice.%20Rice.%202020;13(1):43." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>Li N, Miao J, Li Y, Ji F, Yang M, Dai K, et al. Comparative transcriptome analysis and meta-QTLs mapping reveal the regulatory mechanism of cold tolerance in rice at the budding stage. Heliyon. 2024;10(18):e37933.
</cite> [<a href="https://doi.org/10.1016/j.heliyon.2024.e37933" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11425124/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39328527/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li%20N,%20Miao%20J,%20Li%20Y,%20Ji%20F,%20Yang%20M,%20Dai%20K,%20et%20al.%20Comparative%20transcriptome%20analysis%20and%20meta-QTLs%20mapping%20reveal%20the%20regulatory%20mechanism%20of%20cold%20tolerance%20in%20rice%20at%20the%20budding%20stage.%20Heliyon.%202024;10(18):e37933." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Andaya VC, Mackill DJ. Mapping of QTLs associated with cold tolerance during the vegetative stage in rice. J Exp Bot. 2003;54(392):2579–85.
</cite> [<a href="https://doi.org/10.1093/jxb/erg243" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12966040/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Andaya%20VC,%20Mackill%20DJ.%20Mapping%20of%20QTLs%20associated%20with%20cold%20tolerance%20during%20the%20vegetative%20stage%20in%20rice.%20J%20Exp%20Bot.%202003;54(392):2579%E2%80%9385." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Najeeb S, Ali J, Mahender A, Pang YL, Zilhas J, Murugaiyan V, et al. Identification of main-effect quantitative trait loci (QTLs) for low-temperature stress tolerance germination and early seedling vigor-related traits in rice (<em>Oryza sativa</em> L). Mol Breed. 2020;40(1): 10.
</cite> [<a href="https://doi.org/10.1007/s11032-019-1090-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6944268/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31975784/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Najeeb%20S,%20Ali%20J,%20Mahender%20A,%20Pang%20YL,%20Zilhas%20J,%20Murugaiyan%20V,%20et%20al.%20Identification%20of%20main-effect%20quantitative%20trait%20loci%20(QTLs)%20for%20low-temperature%20stress%20tolerance%20germination%20and%20early%20seedling%20vigor-related%20traits%20in%20rice%20(Oryza%20sativa%20L).%20Mol%20Breed.%202020;40(1):%2010." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Zhang M, Ye J, Xu Q, Feng Y, Yuan X, Yu H, et al. Genome-wide association study of cold tolerance of Chinese <em>indica</em> rice varieties at the bud burst stage. Plant Cell Rep. 2018;37(3):529–39.
</cite> [<a href="https://doi.org/10.1007/s00299-017-2247-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29322237/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhang%20M,%20Ye%20J,%20Xu%20Q,%20Feng%20Y,%20Yuan%20X,%20Yu%20H,%20et%20al.%20Genome-wide%20association%20study%20of%20cold%20tolerance%20of%20Chinese%20indica%20rice%20varieties%20at%20the%20bud%20burst%20stage.%20Plant%20Cell%20Rep.%202018;37(3):529%E2%80%9339." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Suh JP, Jeung JU, Lee JI, Choi YH, Yea JD, Virk PS, et al. Identification and analysis of QTLs controlling cold tolerance at the reproductive stage and validation of effective QTLs in cold-tolerant genotypes of rice (<em>Oryza sativa</em> L). Theor Appl Genet. 2010;120(5):985–95.
</cite> [<a href="https://doi.org/10.1007/s00122-009-1226-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20012263/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Suh%20JP,%20Jeung%20JU,%20Lee%20JI,%20Choi%20YH,%20Yea%20JD,%20Virk%20PS,%20et%20al.%20Identification%20and%20analysis%20of%20QTLs%20controlling%20cold%20tolerance%20at%20the%20reproductive%20stage%20and%20validation%20of%20effective%20QTLs%20in%20cold-tolerant%20genotypes%20of%20rice%20(Oryza%20sativa%20L).%20Theor%20Appl%20Genet.%202010;120(5):985%E2%80%9395." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>Li J, Zeng Y, Pan Y, Zhou L, Zhang Z, Guo H, et al. Stepwise selection of natural variations at <em>CTB2</em> and <em>CTB4a</em> improves cold adaptation during domestication of Japonica rice. New Phytol. 2021;231(3):1056–72.
</cite> [<a href="https://doi.org/10.1111/nph.17407" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33892513/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li%20J,%20Zeng%20Y,%20Pan%20Y,%20Zhou%20L,%20Zhang%20Z,%20Guo%20H,%20et%20al.%20Stepwise%20selection%20of%20natural%20variations%20at%20CTB2%20and%20CTB4a%20improves%20cold%20adaptation%20during%20domestication%20of%20Japonica%20rice.%20New%20Phytol.%202021;231(3):1056%E2%80%9372." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>Li X, Dong J, Zhu W, Zhao J, Zhou L. Progress in the study of functional genes related to direct seeding of rice. Mol Breed. 2023;43(6):46.
</cite> [<a href="https://doi.org/10.1007/s11032-023-01388-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10248684/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37309311/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li%20X,%20Dong%20J,%20Zhu%20W,%20Zhao%20J,%20Zhou%20L.%20Progress%20in%20the%20study%20of%20functional%20genes%20related%20to%20direct%20seeding%20of%20rice.%20Mol%20Breed.%202023;43(6):46." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>Wang D, Liu Z, Xiao Y, Liu X, Chen Y, Zhang Z, et al. Association mapping and functional analysis of rice cold tolerance QTLs at the bud burst stage. Rice. 2021;14(1): 98.
</cite> [<a href="https://doi.org/10.1186/s12284-021-00538-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8626552/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34825994/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang%20D,%20Liu%20Z,%20Xiao%20Y,%20Liu%20X,%20Chen%20Y,%20Zhang%20Z,%20et%20al.%20Association%20mapping%20and%20functional%20analysis%20of%20rice%20cold%20tolerance%20QTLs%20at%20the%20bud%20burst%20stage.%20Rice.%202021;14(1):%2098." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Yang L, Lei L, Li P, Wang J, Wang C, Yang F, et al. Identification of candidate genes conferring cold tolerance to rice (<em>Oryza sativa</em> L.) at the bud-bursting stage using bulk segregant analysis sequencing and linkage mapping. Front Plant Sci. 2021;12:647239.
</cite> [<a href="https://doi.org/10.3389/fpls.2021.647239" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8006307/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33790929/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yang%20L,%20Lei%20L,%20Li%20P,%20Wang%20J,%20Wang%20C,%20Yang%20F,%20et%20al.%20Identification%20of%20candidate%20genes%20conferring%20cold%20tolerance%20to%20rice%20(Oryza%20sativa%20L.)%20at%20the%20bud-bursting%20stage%20using%20bulk%20segregant%20analysis%20sequencing%20and%20linkage%20mapping.%20Front%20Plant%20Sci.%202021;12:647239." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Wu B, Fu M, Du J, Wang M, Zhang S, Li S, et al. Identification of the cold-related genes <em>COLD11</em> and <em>OsCTS11</em> via BSA-seq and fine mapping at the rice seedling stage. Rice. 2024;17(1): 72.
</cite> [<a href="https://doi.org/10.1186/s12284-024-00749-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11584825/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39576378/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wu%20B,%20Fu%20M,%20Du%20J,%20Wang%20M,%20Zhang%20S,%20Li%20S,%20et%20al.%20Identification%20of%20the%20cold-related%20genes%20COLD11%20and%20OsCTS11%20via%20BSA-seq%20and%20fine%20mapping%20at%20the%20rice%20seedling%20stage.%20Rice.%202024;17(1):%2072." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>Shi H, Zhang W, Cao H, Zhai L, Song Q, Xu J. Identification of candidate genes for cold tolerance at seedling stage by GWAS in rice (<em>Oryza sativa</em> L). Biology. 2024;13:784.
</cite> [<a href="https://doi.org/10.3390/biology13100784" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11505075/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39452093/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Shi%20H,%20Zhang%20W,%20Cao%20H,%20Zhai%20L,%20Song%20Q,%20Xu%20J.%20Identification%20of%20candidate%20genes%20for%20cold%20tolerance%20at%20seedling%20stage%20by%20GWAS%20in%20rice%20(Oryza%20sativa%20L).%20Biology.%202024;13:784." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>Fujino K, Sekiguchi H, Matsuda Y, Sugimoto K, Ono K, Yano M. Molecular identification of a major quantitative trait locus, <em>qLTG3-1</em>, controlling low-temperature germinability in rice. Proc Natl Acad Sci U S A. 2008;105(34):12623–8.
</cite> [<a href="https://doi.org/10.1073/pnas.0805303105" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2527961/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18719107/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Fujino%20K,%20Sekiguchi%20H,%20Matsuda%20Y,%20Sugimoto%20K,%20Ono%20K,%20Yano%20M.%20Molecular%20identification%20of%20a%20major%20quantitative%20trait%20locus,%20qLTG3-1,%20controlling%20low-temperature%20germinability%20in%20rice.%20Proc%20Natl%20Acad%20Sci%20U%20S%20A.%202008;105(34):12623%E2%80%938." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>Wang X, Zou B, Shao Q, Cui Y, Lu S, Zhang Y, et al. Natural variation reveals that <em>OsSAP16</em> controls low-temperature germination in rice. J Exp Bot. 2018;69(3):413–21.
</cite> [<a href="https://doi.org/10.1093/jxb/erx413" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5853544/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29237030/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang%20X,%20Zou%20B,%20Shao%20Q,%20Cui%20Y,%20Lu%20S,%20Zhang%20Y,%20et%20al.%20Natural%20variation%20reveals%20that%20OsSAP16%20controls%20low-temperature%20germination%20in%20rice.%20J%20Exp%20Bot.%202018;69(3):413%E2%80%9321." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Zhao J, Wang S, Qin J, Sun C, Liu F. The lipid transfer protein OsLTPL159 is involved in cold tolerance at the early seedling stage in rice. Plant Biotechnol J. 2020;18(3):756–69.
</cite> [<a href="https://doi.org/10.1111/pbi.13243" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7004919/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31469486/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhao%20J,%20Wang%20S,%20Qin%20J,%20Sun%20C,%20Liu%20F.%20The%20lipid%20transfer%20protein%20OsLTPL159%20is%20involved%20in%20cold%20tolerance%20at%20the%20early%20seedling%20stage%20in%20rice.%20Plant%20Biotechnol%20J.%202020;18(3):756%E2%80%9369." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>Zhang C, Wang H, Tian X, Lin X, Han Y, Han Z, et al. A transposon insertion in the promoter of <em>OsUBC12</em> enhances cold tolerance during Japonica rice germination. Nat Commun. 2024;15(1):2211.
</cite> [<a href="https://doi.org/10.1038/s41467-024-46420-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10937917/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38480722/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhang%20C,%20Wang%20H,%20Tian%20X,%20Lin%20X,%20Han%20Y,%20Han%20Z,%20et%20al.%20A%20transposon%20insertion%20in%20the%20promoter%20of%20OsUBC12%20enhances%20cold%20tolerance%20during%20Japonica%20rice%20germination.%20Nat%20Commun.%202024;15(1):2211." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Ma Y, Dai X, Xu Y, Luo W, Zheng X, Zeng D, et al. <em>COLD1</em> confers chilling tolerance in rice. Cell. 2015;160(6):1209–21.
</cite> [<a href="https://doi.org/10.1016/j.cell.2015.01.046" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25728666/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ma%20Y,%20Dai%20X,%20Xu%20Y,%20Luo%20W,%20Zheng%20X,%20Zeng%20D,%20et%20al.%20COLD1%20confers%20chilling%20tolerance%20in%20rice.%20Cell.%202015;160(6):1209%E2%80%9321." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Liu C, Ou S, Mao B, Tang J, Wang W, Wang H, et al. Early selection of <em>bZIP73</em> facilitated adaptation of Japonica rice to cold climates. Nat Commun. 2018;9(1):3302.
</cite> [<a href="https://doi.org/10.1038/s41467-018-05753-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6098049/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30120236/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Liu%20C,%20Ou%20S,%20Mao%20B,%20Tang%20J,%20Wang%20W,%20Wang%20H,%20et%20al.%20Early%20selection%20of%20bZIP73%20facilitated%20adaptation%20of%20Japonica%20rice%20to%20cold%20climates.%20Nat%20Commun.%202018;9(1):3302." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>Xiao N, Gao Y, Qian H, Gao Q, Wu Y, Zhang D, et al. Identification of genes related to cold tolerance and a functional allele that confers cold tolerance. Plant Physiol. 2018;177(3):1108–23.
</cite> [<a href="https://doi.org/10.1104/pp.18.00209" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6052991/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29764927/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Xiao%20N,%20Gao%20Y,%20Qian%20H,%20Gao%20Q,%20Wu%20Y,%20Zhang%20D,%20et%20al.%20Identification%20of%20genes%20related%20to%20cold%20tolerance%20and%20a%20functional%20allele%20that%20confers%20cold%20tolerance.%20Plant%20Physiol.%202018;177(3):1108%E2%80%9323." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>Mao D, Xin Y, Tan Y, Hu X, Bai J, Liu ZY, et al. Natural variation in the <em>HAN1</em> gene confers chilling tolerance in rice and allowed adaptation to a temperate climate. Proc Natl Acad Sci U S A. 2019;116(9):3494–501.
</cite> [<a href="https://doi.org/10.1073/pnas.1819769116" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6397538/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30808744/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mao%20D,%20Xin%20Y,%20Tan%20Y,%20Hu%20X,%20Bai%20J,%20Liu%20ZY,%20et%20al.%20Natural%20variation%20in%20the%20HAN1%20gene%20confers%20chilling%20tolerance%20in%20rice%20and%20allowed%20adaptation%20to%20a%20temperate%20climate.%20Proc%20Natl%20Acad%20Sci%20U%20S%20A.%202019;116(9):3494%E2%80%93501." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>Xia C, Liang G, Chong K, Xu Y. The <em>COG1</em>-<em>OsSERL2</em> complex senses cold to trigger signaling network for chilling tolerance in Japonica rice. Nat Commun. 2023;14(1):3104.
</cite> [<a href="https://doi.org/10.1038/s41467-023-38860-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10227007/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37248220/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Xia%20C,%20Liang%20G,%20Chong%20K,%20Xu%20Y.%20The%20COG1-OsSERL2%20complex%20senses%20cold%20to%20trigger%20signaling%20network%20for%20chilling%20tolerance%20in%20Japonica%20rice.%20Nat%20Commun.%202023;14(1):3104." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>Li Z, Wang B, Luo W, Xu Y, Wang J, Xue Z, et al. Natural variation of codon repeats in <em>COLD11</em> endows rice with chilling resilience. Sci Adv. 2023;9(1): eabq5506.
</cite> [<a href="https://doi.org/10.1126/sciadv.abq5506" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9821855/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36608134/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li%20Z,%20Wang%20B,%20Luo%20W,%20Xu%20Y,%20Wang%20J,%20Xue%20Z,%20et%20al.%20Natural%20variation%20of%20codon%20repeats%20in%20COLD11%20endows%20rice%20with%20chilling%20resilience.%20Sci%20Adv.%202023;9(1):%20eabq5506." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>Zu X, Luo L, Wang Z, Gong J, Yang C, Wang Y, et al. A mitochondrial pentatricopeptide repeat protein enhances cold tolerance by modulating mitochondrial superoxide in rice. Nat Commun. 2023;14(1):6789.
</cite> [<a href="https://doi.org/10.1038/s41467-023-42269-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10600133/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37880207/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zu%20X,%20Luo%20L,%20Wang%20Z,%20Gong%20J,%20Yang%20C,%20Wang%20Y,%20et%20al.%20A%20mitochondrial%20pentatricopeptide%20repeat%20protein%20enhances%20cold%20tolerance%20by%20modulating%20mitochondrial%20superoxide%20in%20rice.%20Nat%20Commun.%202023;14(1):6789." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>Liu D, Luo S, Li Z, Liang G, Guo Y, Xu Y, et al.<em>COG3</em>confers the chilling tolerance to mediate <em>OsFtsH2-D1</em> module in rice. New Phytol. 2024;241(5):2143–57.
</cite> [<a href="https://doi.org/10.1111/nph.19514" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38173177/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Liu%20D,%20Luo%20S,%20Li%20Z,%20Liang%20G,%20Guo%20Y,%20Xu%20Y,%20et%20al.COG3confers%20the%20chilling%20tolerance%20to%20mediate%20OsFtsH2-D1%20module%20in%20rice.%20New%20Phytol.%202024;241(5):2143%E2%80%9357." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>Saito K, Hayano-Saito Y, Maruyama-Funatsuki W, Sato Y, Kato A. Physical mapping and putative candidate gene identification of a quantitative trait locus <em>Ctb1</em> for cold tolerance at the booting stage of rice. Theor Appl Genet. 2004;109(3):515–22.
</cite> [<a href="https://doi.org/10.1007/s00122-004-1667-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15114474/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Saito%20K,%20Hayano-Saito%20Y,%20Maruyama-Funatsuki%20W,%20Sato%20Y,%20Kato%20A.%20Physical%20mapping%20and%20putative%20candidate%20gene%20identification%20of%20a%20quantitative%20trait%20locus%20Ctb1%20for%20cold%20tolerance%20at%20the%20booting%20stage%20of%20rice.%20Theor%20Appl%20Genet.%202004;109(3):515%E2%80%9322." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Zhang Z, Li J, Pan Y, Li J, Zhou L, Shi H, et al. Natural variation in <em>CTB4a</em> enhances rice adaptation to cold habitats. Nat Commun. 2017;8: 14788.
</cite> [<a href="https://doi.org/10.1038/ncomms14788" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5376651/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28332574/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhang%20Z,%20Li%20J,%20Pan%20Y,%20Li%20J,%20Zhou%20L,%20Shi%20H,%20et%20al.%20Natural%20variation%20in%20CTB4a%20enhances%20rice%20adaptation%20to%20cold%20habitats.%20Nat%20Commun.%202017;8:%2014788." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Xu Y, Wang R, Wang Y, Zhang L, Yao S. A point mutation in<em>LTT1</em>enhances cold tolerance at the booting stage in rice. Plant Cell Environ. 2020;43(4):992–1007.
</cite> [<a href="https://doi.org/10.1111/pce.13717" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7154693/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31922260/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Xu%20Y,%20Wang%20R,%20Wang%20Y,%20Zhang%20L,%20Yao%20S.%20A%20point%20mutation%20inLTT1enhances%20cold%20tolerance%20at%20the%20booting%20stage%20in%20rice.%20Plant%20Cell%20Environ.%202020;43(4):992%E2%80%931007." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>Lou Q, Guo H, Li J, Han S, Khan NU, Gu Y, et al. Cold-adaptive evolution at the reproductive stage in geng/japonica subspecies reveals the role of <em>OsMAPK3</em> and <em>OsLEA9</em>. Plant J. 2022;111(4):1032–51.
</cite> [<a href="https://doi.org/10.1111/tpj.15870" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35706359/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lou%20Q,%20Guo%20H,%20Li%20J,%20Han%20S,%20Khan%20NU,%20Gu%20Y,%20et%20al.%20Cold-adaptive%20evolution%20at%20the%20reproductive%20stage%20in%20geng/japonica%20subspecies%20reveals%20the%20role%20of%20OsMAPK3%20and%20OsLEA9.%20Plant%20J.%202022;111(4):1032%E2%80%9351." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR30">
<span class="label">30.</span><cite>Guo H, Gao S, Li H, Yang J, Li J, Gu Y, et al. Natural variation of <em>CTB5</em> confers cold adaptation in plateau japonica rice. Nat Commun. 2025;16(1):1032.
</cite> [<a href="https://doi.org/10.1038/s41467-025-56174-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11763261/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39863601/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Guo%20H,%20Gao%20S,%20Li%20H,%20Yang%20J,%20Li%20J,%20Gu%20Y,%20et%20al.%20Natural%20variation%20of%20CTB5%20confers%20cold%20adaptation%20in%20plateau%20japonica%20rice.%20Nat%20Commun.%202025;16(1):1032." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>Gao S, Li J, Zeng Y, Li H, Guo Z, Guo H, et al. <em>CTB6</em> confers cold tolerance at the booting stage by maintaining tapetum development in rice. Adv Sci. 2025;2:e2411357.</cite> [<a href="https://doi.org/10.1002/advs.202411357" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11905004/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39840555/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gao%20S,%20Li%20J,%20Zeng%20Y,%20Li%20H,%20Guo%20Z,%20Guo%20H,%20et%20al.%20CTB6%20confers%20cold%20tolerance%20at%20the%20booting%20stage%20by%20maintaining%20tapetum%20development%20in%20rice.%20Adv%20Sci.%202025;2:e2411357." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>Chen R, Cheng Y, Han S, Van Handel B, Dong L, Li X, et al. Whole genome sequencing and comparative transcriptome analysis of a novel seawater adapted, salt-resistant rice cultivar-sea rice 86. BMC Genomics. 2017;18:655.
</cite> [<a href="https://doi.org/10.1186/s12864-017-4037-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5569538/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28835208/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chen%20R,%20Cheng%20Y,%20Han%20S,%20Van%20Handel%20B,%20Dong%20L,%20Li%20X,%20et%20al.%20Whole%20genome%20sequencing%20and%20comparative%20transcriptome%20analysis%20of%20a%20novel%20seawater%20adapted,%20salt-resistant%20rice%20cultivar-sea%20rice%2086.%20BMC%20Genomics.%202017;18:655." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Cao Z, Guo Y, Yang Q, He Y, Fetouh MI, Warner RM, et al. Genome-wide identification of quantitative trait loci for important plant and flower traits in <em>Petunia</em>using a high-density linkage map and an interspecific recombinant inbred population derived from<em>Petunia integrifolia</em>and<em>P. axillaris</em>. Hortic Res. 2019;6:27.
</cite> [<a href="https://doi.org/10.1038/s41438-018-0091-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6355904/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30729017/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cao%20Z,%20Guo%20Y,%20Yang%20Q,%20He%20Y,%20Fetouh%20MI,%20Warner%20RM,%20et%20al.%20Genome-wide%20identification%20of%20quantitative%20trait%20loci%20for%20important%20plant%20and%20flower%20traits%20in%20Petuniausing%20a%20high-density%20linkage%20map%20and%20an%20interspecific%20recombinant%20inbred%20population%20derived%20fromPetunia%20integrifoliaandP.%20axillaris.%20Hortic%20Res.%202019;6:27." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>Ashburner M, Ball CA, Blake JA, Botstein D, Butler H, Cherry JM, The gene ontology consortium, et al. Gene ontology: tool for the unification of biology. Nat Genet. 2000;25(1):25–9.
</cite> [<a href="https://doi.org/10.1038/75556" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3037419/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10802651/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ashburner%20M,%20Ball%20CA,%20Blake%20JA,%20Botstein%20D,%20Butler%20H,%20Cherry%20JM,%20The%20gene%20ontology%20consortium,%20et%20al.%20Gene%20ontology:%20tool%20for%20the%20unification%20of%20biology.%20Nat%20Genet.%202000;25(1):25%E2%80%939." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>Ogata H, Goto S, Sato K, Fujibuchi W, Bono H, Kanehisa M. KEGG: Kyoto encyclopedia of genes and genomes. Nucleic Acids Res. 1999;27(1):29–34.
</cite> [<a href="https://doi.org/10.1093/nar/27.1.29" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC148090/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9847135/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ogata%20H,%20Goto%20S,%20Sato%20K,%20Fujibuchi%20W,%20Bono%20H,%20Kanehisa%20M.%20KEGG:%20Kyoto%20encyclopedia%20of%20genes%20and%20genomes.%20Nucleic%20Acids%20Res.%201999;27(1):29%E2%80%9334." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR36">
<span class="label">36.</span><cite>Chen W, Li P, Zhang G, Chen K, Li Z, Xu J. Combining a genome-wide association study and gene-based haplotype analysis to identify candidate genes for cold tolerance at the bud burst stage in rice <em>(Oryza sativa</em> L). Agron (Basel). 2023;1(3):413–26.</cite> [<a href="https://scholar.google.com/scholar_lookup?Chen%20W,%20Li%20P,%20Zhang%20G,%20Chen%20K,%20Li%20Z,%20Xu%20J.%20Combining%20a%20genome-wide%20association%20study%20and%20gene-based%20haplotype%20analysis%20to%20identify%20candidate%20genes%20for%20cold%20tolerance%20at%20the%20bud%20burst%20stage%20in%20rice%20(Oryza%20sativa%20L).%20Agron%20(Basel).%202023;1(3):413%E2%80%9326." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR37">
<span class="label">37.</span><cite>Wang W, Mauleon R, Hu Z, Chebotarov D, Tai S, Wu Z, et al. Genomic variation in 3,010 diverse accessions of Asian cultivated rice. Nature. 2018;557(7703):43–9.
</cite> [<a href="https://doi.org/10.1038/s41586-018-0063-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6784863/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29695866/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang%20W,%20Mauleon%20R,%20Hu%20Z,%20Chebotarov%20D,%20Tai%20S,%20Wu%20Z,%20et%20al.%20Genomic%20variation%20in%203,010%20diverse%20accessions%20of%20Asian%20cultivated%20rice.%20Nature.%202018;557(7703):43%E2%80%939." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR38">
<span class="label">38.</span><cite>Livak KJ, Schmittgen TD. Analysis of relative gene expression data using real-time quantitative PCR and the 2(-Delta delta C(T)) method. Methods. 2001;25(4):402–8.
</cite> [<a href="https://doi.org/10.1006/meth.2001.1262" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11846609/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Livak%20KJ,%20Schmittgen%20TD.%20Analysis%20of%20relative%20gene%20expression%20data%20using%20real-time%20quantitative%20PCR%20and%20the%202(-Delta%20delta%20C(T))%20method.%20Methods.%202001;25(4):402%E2%80%938." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR39">
<span class="label">39.</span><cite>Zhang Q, Chen Q, Wang S, Hong Y, Wang Z. Rice and cold stress: methods for its evaluation and summary of cold tolerance-related quantitative trait loci. Rice. 2014;7(1):24.
</cite> [<a href="https://doi.org/10.1186/s12284-014-0024-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4182278/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25279026/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhang%20Q,%20Chen%20Q,%20Wang%20S,%20Hong%20Y,%20Wang%20Z.%20Rice%20and%20cold%20stress:%20methods%20for%20its%20evaluation%20and%20summary%20of%20cold%20tolerance-related%20quantitative%20trait%20loci.%20Rice.%202014;7(1):24." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR40">
<span class="label">40.</span><cite>Guo H, Guo Y, Zeng Y, Zou A, Khan NU, Gu Y, et al. QTL detection and candidate gene identification of <em>qCTB1</em> for cold tolerance in the Yunnan plateau landrace rice. Mol Breed. 2024;44(8): 50.
</cite> [<a href="https://doi.org/10.1007/s11032-024-01488-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11282035/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39070774/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Guo%20H,%20Guo%20Y,%20Zeng%20Y,%20Zou%20A,%20Khan%20NU,%20Gu%20Y,%20et%20al.%20QTL%20detection%20and%20candidate%20gene%20identification%20of%20qCTB1%20for%20cold%20tolerance%20in%20the%20Yunnan%20plateau%20landrace%20rice.%20Mol%20Breed.%202024;44(8):%2050." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR41">
<span class="label">41.</span><cite>Pan YH, Nong BX, Chen L, Yang XH, Xia XZ, Zhang ZQ, et al. Qtl mapping and identification of candidate genes for cold tolerance at the germination stage in wild rice. Genes Genomics. 2023;45(7):867–85.
</cite> [<a href="https://doi.org/10.1007/s13258-023-01394-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37209287/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Pan%20YH,%20Nong%20BX,%20Chen%20L,%20Yang%20XH,%20Xia%20XZ,%20Zhang%20ZQ,%20et%20al.%20Qtl%20mapping%20and%20identification%20of%20candidate%20genes%20for%20cold%20tolerance%20at%20the%20germination%20stage%20in%20wild%20rice.%20Genes%20Genomics.%202023;45(7):867%E2%80%9385." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR42">
<span class="label">42.</span><cite>Yang J, Miao J, Li N, Zhou Z, Dai K, Ji F, et al. Genetic dissection of cold tolerance at the budding stage of rice in an indica-japonica recombination inbred line population. Plant Physiol Biochem. 2023;204: 108086.
</cite> [<a href="https://doi.org/10.1016/j.plaphy.2023.108086" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37890228/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yang%20J,%20Miao%20J,%20Li%20N,%20Zhou%20Z,%20Dai%20K,%20Ji%20F,%20et%20al.%20Genetic%20dissection%20of%20cold%20tolerance%20at%20the%20budding%20stage%20of%20rice%20in%20an%20indica-japonica%20recombination%20inbred%20line%20population.%20Plant%20Physiol%20Biochem.%202023;204:%20108086." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR43">
<span class="label">43.</span><cite>Yin M, Zheng Z, Zhang Y, Wang S, Zuo L, Lei Y, et al. Identification of key genes and pathways for anaerobic germination tolerance in rice using weighted gene co-expression network analysis (WGCNA) in association with quantitative trait locus (QTL) mapping. Rice. 2024;7(1):37.</cite> [<a href="https://doi.org/10.1186/s12284-024-00714-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11143092/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38819744/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yin%20M,%20Zheng%20Z,%20Zhang%20Y,%20Wang%20S,%20Zuo%20L,%20Lei%20Y,%20et%20al.%20Identification%20of%20key%20genes%20and%20pathways%20for%20anaerobic%20germination%20tolerance%20in%20rice%20using%20weighted%20gene%20co-expression%20network%20analysis%20(WGCNA)%20in%20association%20with%20quantitative%20trait%20locus%20(QTL)%20mapping.%20Rice.%202024;7(1):37." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR44">
<span class="label">44.</span><cite>Zhou S, Wu T, Li X, Wang S, Hu B. Identification of candidate genes controlling cold tolerance at the early seedling stage from Dongxiang wild rice by QTL mapping, BSA-Seq and RNA-Seq. BMC Plant Biol. 2024;24(1):649.
</cite> [<a href="https://doi.org/10.1186/s12870-024-05369-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11232298/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38977989/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhou%20S,%20Wu%20T,%20Li%20X,%20Wang%20S,%20Hu%20B.%20Identification%20of%20candidate%20genes%20controlling%20cold%20tolerance%20at%20the%20early%20seedling%20stage%20from%20Dongxiang%20wild%20rice%20by%20QTL%20mapping,%20BSA-Seq%20and%20RNA-Seq.%20BMC%20Plant%20Biol.%202024;24(1):649." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR45">
<span class="label">45.</span><cite>Jo S, Jang SG, Lee SB, Lee JY, Cho JH, Kang JW, et al. Analysis of quantitative trait loci and candidate gene exploration associated with cold tolerance in rice (<em>Oryza sativa</em> L.) during the seedling stage. Front Plant Sci. 2025;15:1508333.
</cite> [<a href="https://doi.org/10.3389/fpls.2024.1508333" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11747135/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39840352/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jo%20S,%20Jang%20SG,%20Lee%20SB,%20Lee%20JY,%20Cho%20JH,%20Kang%20JW,%20et%20al.%20Analysis%20of%20quantitative%20trait%20loci%20and%20candidate%20gene%20exploration%20associated%20with%20cold%20tolerance%20in%20rice%20(Oryza%20sativa%20L.)%20during%20the%20seedling%20stage.%20Front%20Plant%20Sci.%202025;15:1508333." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR46">
<span class="label">46.</span><cite>Han LZ, Zhang YY, Qiao YL, Cao GL, Zhang SY, Jong Hwan K, et al. Genetic and QTL analysis for low-temperature Vigor of germination in rice. Acta Genet Sin. 2006;33(11):998–1006.
</cite> [<a href="https://doi.org/10.1016/S0379-4172(06)60135-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17112971/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Han%20LZ,%20Zhang%20YY,%20Qiao%20YL,%20Cao%20GL,%20Zhang%20SY,%20Jong%20Hwan%20K,%20et%20al.%20Genetic%20and%20QTL%20analysis%20for%20low-temperature%20Vigor%20of%20germination%20in%20rice.%20Acta%20Genet%20Sin.%202006;33(11):998%E2%80%931006." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR47">
<span class="label">47.</span><cite>Li L, Liu X, Xie K, Wang Y, Liu F, Lin Q, et al. <em>qLTG-9</em>, a stable quantitative trait locus for low-temperature germination in rice (<em>Oryza sativa</em> L). Theor Appl Genet. 2013;126(9):2313–22.
</cite> [<a href="https://doi.org/10.1007/s00122-013-2137-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23748708/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li%20L,%20Liu%20X,%20Xie%20K,%20Wang%20Y,%20Liu%20F,%20Lin%20Q,%20et%20al.%20qLTG-9,%20a%20stable%20quantitative%20trait%20locus%20for%20low-temperature%20germination%20in%20rice%20(Oryza%20sativa%20L).%20Theor%20Appl%20Genet.%202013;126(9):2313%E2%80%9322." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR48">
<span class="label">48.</span><cite>Jiang L, Liu S, Hou M, Tang J, Chen L, Zhai H, et al. Analysis of QTLs for seed low temperature germinability and anoxia germinability in rice (<em>Oryza sativa</em> L). Field Crops Res. 2006;98:68–75.</cite> [<a href="https://scholar.google.com/scholar_lookup?Jiang%20L,%20Liu%20S,%20Hou%20M,%20Tang%20J,%20Chen%20L,%20Zhai%20H,%20et%20al.%20Analysis%20of%20QTLs%20for%20seed%20low%20temperature%20germinability%20and%20anoxia%20germinability%20in%20rice%20(Oryza%20sativa%20L).%20Field%20Crops%20Res.%202006;98:68%E2%80%9375." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR49">
<span class="label">49.</span><cite>Hundertmark M, Hincha DK. LEA (late embryogenesis abundant) proteins and their encoding genes in <em>Arabidopsis Thaliana</em>. BMC Genomics. 2008;9:118.
</cite> [<a href="https://doi.org/10.1186/1471-2164-9-118" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2292704/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18318901/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hundertmark%20M,%20Hincha%20DK.%20LEA%20(late%20embryogenesis%20abundant)%20proteins%20and%20their%20encoding%20genes%20in%20Arabidopsis%20Thaliana.%20BMC%20Genomics.%202008;9:118." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR50">
<span class="label">50.</span><cite>Tunnacliffe A, Wise MJ. The continuing conundrum of the LEA proteins. Naturwissenschaften. 2007;94(10):791–812.
</cite> [<a href="https://doi.org/10.1007/s00114-007-0254-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17479232/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tunnacliffe%20A,%20Wise%20MJ.%20The%20continuing%20conundrum%20of%20the%20LEA%20proteins.%20Naturwissenschaften.%202007;94(10):791%E2%80%93812." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR51">
<span class="label">51.</span><cite>Wang W, Vinocur B, Altman A. Plant responses to drought, salinity and extreme temperatures: towards genetic engineering for stress tolerance. Planta. 2003;218(1):1–14.
</cite> [<a href="https://doi.org/10.1007/s00425-003-1105-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/14513379/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang%20W,%20Vinocur%20B,%20Altman%20A.%20Plant%20responses%20to%20drought,%20salinity%20and%20extreme%20temperatures:%20towards%20genetic%20engineering%20for%20stress%20tolerance.%20Planta.%202003;218(1):1%E2%80%9314." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR52">
<span class="label">52.</span><cite>He S, Tan L, Hu Z, Chen G, Wang G, Hu T. Molecular characterization and functional analysis by heterologous expression in <em>E. coli</em> under diverse abiotic stresses for <em>OsLEA5</em>, the atypical hydrophobic LEA protein from <em>Oryza sativa</em> L. Mol Genet Genomics. 2012;287(1):39–54.
</cite> [<a href="https://doi.org/10.1007/s00438-011-0660-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22127413/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?He%20S,%20Tan%20L,%20Hu%20Z,%20Chen%20G,%20Wang%20G,%20Hu%20T.%20Molecular%20characterization%20and%20functional%20analysis%20by%20heterologous%20expression%20in%20E.%20coli%20under%20diverse%20abiotic%20stresses%20for%20OsLEA5,%20the%20atypical%20hydrophobic%20LEA%20protein%20from%20Oryza%20sativa%20L.%20Mol%20Genet%20Genomics.%202012;287(1):39%E2%80%9354." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR53">
<span class="label">53.</span><cite>Takahashi R, Joshee N, Kitagawa Y. Induction of chilling resistance by water stress, and cDNA sequence analysis and expression of water stress-regulated genes in rice. Plant Mol Biol. 1994;26(1):339–52.
</cite> [<a href="https://doi.org/10.1007/BF00039544" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/7948880/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Takahashi%20R,%20Joshee%20N,%20Kitagawa%20Y.%20Induction%20of%20chilling%20resistance%20by%20water%20stress,%20and%20cDNA%20sequence%20analysis%20and%20expression%20of%20water%20stress-regulated%20genes%20in%20rice.%20Plant%20Mol%20Biol.%201994;26(1):339%E2%80%9352." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR54">
<span class="label">54.</span><cite>Hu T, Zeng H, He S, Wu Y, Wang G, Huang X. Molecular analysis of <em>OsLEA4</em> and its contributions to improve <em>E. coli</em> viability. Appl Biochem Biotechnol. 2012;166(1):222–33.
</cite> [<a href="https://doi.org/10.1007/s12010-011-9418-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22057936/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hu%20T,%20Zeng%20H,%20He%20S,%20Wu%20Y,%20Wang%20G,%20Huang%20X.%20Molecular%20analysis%20of%20OsLEA4%20and%20its%20contributions%20to%20improve%20E.%20coli%20viability.%20Appl%20Biochem%20Biotechnol.%202012;166(1):222%E2%80%9333." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR55">
<span class="label">55.</span><cite>Cosgrove D. Growth of the plant cell wall. Nat Rev Mol Cell Biol. 2005. 10.1038/nrm1746.
</cite> [<a href="https://doi.org/10.1038/nrm1746" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16261190/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cosgrove%20D.%20Growth%20of%20the%20plant%20cell%20wall.%20Nat%20Rev%20Mol%20Cell%20Biol.%202005.%2010.1038/nrm1746." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR56">
<span class="label">56.</span><cite>Tenhaken R. Cell wall remodeling under abiotic stress. Front Plant Sci. 2015;5:771.
</cite> [<a href="https://doi.org/10.3389/fpls.2014.00771" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4285730/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25709610/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tenhaken%20R.%20Cell%20wall%20remodeling%20under%20abiotic%20stress.%20Front%20Plant%20Sci.%202015;5:771." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR57">
<span class="label">57.</span><cite>Jan A, Yang G, Nakamura H, Ichikawa H, Kitano H, Matsuoka M, et al. Characterization of a Xyloglucan endotransglucosylase gene that is up-regulated by Gibberellin in rice. Plant Physiol. 2004;136(3):3670–81.
</cite> [<a href="https://doi.org/10.1104/pp.104.052274" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC527165/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15516498/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jan%20A,%20Yang%20G,%20Nakamura%20H,%20Ichikawa%20H,%20Kitano%20H,%20Matsuoka%20M,%20et%20al.%20Characterization%20of%20a%20Xyloglucan%20endotransglucosylase%20gene%20that%20is%20up-regulated%20by%20Gibberellin%20in%20rice.%20Plant%20Physiol.%202004;136(3):3670%E2%80%9381." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR58">
<span class="label">58.</span><cite>Hara Y, Yokoyama R, Osakabe K, Toki S, Nishitani K. Function of xyloglucan endotransglucosylase/hydrolases in rice. Ann Bot. 2014;114(6):1309–18.
</cite> [<a href="https://doi.org/10.1093/aob/mct292" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4195539/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24363334/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hara%20Y,%20Yokoyama%20R,%20Osakabe%20K,%20Toki%20S,%20Nishitani%20K.%20Function%20of%20xyloglucan%20endotransglucosylase/hydrolases%20in%20rice.%20Ann%20Bot.%202014;114(6):1309%E2%80%9318." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR59">
<span class="label">59.</span><cite>Ranawake AL, Nakamura C. Cold tolerance of an inbred line population of rice (<em>Oryza sativa</em> L) at different growth stages. Trop Agricultural Res Ext. 2011;14(2):24–33.</cite>
</li>
<li id="CR60">
<span class="label">60.</span><cite>Ranawake AL, Manangkil OE, Yoshida S, Ishii T, Mori N, Nakamura C. Mapping QTLs for cold tolerance at germination and the early seedling stage in rice (<em>Oryza sativa</em> L). Biotechnol Biotechnol Equip. 2014;28(6):989–98.
</cite> [<a href="https://doi.org/10.1080/13102818.2014.978539" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4684067/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26740779/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ranawake%20AL,%20Manangkil%20OE,%20Yoshida%20S,%20Ishii%20T,%20Mori%20N,%20Nakamura%20C.%20Mapping%20QTLs%20for%20cold%20tolerance%20at%20germination%20and%20the%20early%20seedling%20stage%20in%20rice%20(Oryza%20sativa%20L).%20Biotechnol%20Biotechnol%20Equip.%202014;28(6):989%E2%80%9398." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR61">
<span class="label">61.</span><cite>Zhou L, Zeng Y, Hu G, Pan Y, Yang S, You A, et al. Characterization and identification of cold tolerant near-isogenic lines in rice. Breed Sci. 2012;62(2):196–201.
</cite> [<a href="https://doi.org/10.1270/jsbbs.62.196" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3405969/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23136531/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhou%20L,%20Zeng%20Y,%20Hu%20G,%20Pan%20Y,%20Yang%20S,%20You%20A,%20et%20al.%20Characterization%20and%20identification%20of%20cold%20tolerant%20near-isogenic%20lines%20in%20rice.%20Breed%20Sci.%202012;62(2):196%E2%80%93201." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12363062/bin/12864_2025_11937_MOESM1_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 1.</a><sup> (1.2MB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12363062/bin/12864_2025_11937_MOESM2_ESM.zip" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 2.</a><sup> (2.3MB, zip) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>The RNA-seq raw data of this article was deposited in the NCBI SRA under accession number PRJNA1259743 and PRJNA1260178. All data generated or analysed during this study are included in this published article and its supplementary information files.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from BMC Genomics are provided here courtesy of <strong>BMC</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1186/s12864-025-11937-8"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/12864_2025_Article_11937.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (3.2 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12363062/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12363062/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12363062%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12363062/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12363062/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12363062/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40830828/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12363062/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40830828/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12363062/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12363062/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="IlGbBAnxpCiWiEWYhEFkgLP5BQspy3ovc3ZXdrRJO2mdqFNje9o6g3zXzSTDeRSH">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
