
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Tumor cell-adipocyte gap junctions activate lipolysis and contribute to breast tumorigenesis - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="AE53387F8AF30AF305387F00137CD5AA.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="ncomms">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12368202/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Nature Communications">
<meta name="citation_title" content="Tumor cell-adipocyte gap junctions activate lipolysis and contribute to breast tumorigenesis">
<meta name="citation_author" content="Jeremy Williams">
<meta name="citation_author_institution" content="Department of Cell &amp; Tissue Biology, University of California, San Francisco, San Francisco, CA USA">
<meta name="citation_author_institution" content="Biomedical Sciences Graduate Program, University of California, San Francisco, San Francisco, CA USA">
<meta name="citation_author" content="Roman Camarda">
<meta name="citation_author_institution" content="Department of Cell &amp; Tissue Biology, University of California, San Francisco, San Francisco, CA USA">
<meta name="citation_author_institution" content="Biomedical Sciences Graduate Program, University of California, San Francisco, San Francisco, CA USA">
<meta name="citation_author" content="Serghei Malkov">
<meta name="citation_author_institution" content="Department of Radiology &amp; Biomedical Imaging, University of California, San Francisco, San Francisco, CA USA">
<meta name="citation_author" content="Lisa J Zimmerman">
<meta name="citation_author_institution" content="Department of Biochemistry, Vanderbilt University School of Medicine, Nashville, TN USA">
<meta name="citation_author_institution" content="Jim Ayers Institute for Precancer Detection and Diagnosis, Vanderbilt-Ingram Cancer Center, Nashville, TN USA">
<meta name="citation_author" content="Suzanne Manning">
<meta name="citation_author_institution" content="Department of Pathology, Vanderbilt University School of Medicine, Nashville, TN USA">
<meta name="citation_author" content="Dvir Aran">
<meta name="citation_author_institution" content="Faculty of Biology, Technion, Israel Institute of Technology, Haifa, Israel">
<meta name="citation_author_institution" content="The Taub Faculty of Computer Science, Technion, Israel Institute of Technology, Haifa, Israel">
<meta name="citation_author" content="Andrew Beardsley">
<meta name="citation_author_institution" content="Department of Cell &amp; Tissue Biology, University of California, San Francisco, San Francisco, CA USA">
<meta name="citation_author_institution" content="Department of Medicine, University of California, San Francisco, San Francisco, CA USA">
<meta name="citation_author" content="Daniel Van de Mark">
<meta name="citation_author_institution" content="Department of Cell &amp; Tissue Biology, University of California, San Francisco, San Francisco, CA USA">
<meta name="citation_author" content="Rachel Nakagawa">
<meta name="citation_author_institution" content="Department of Cell &amp; Tissue Biology, University of California, San Francisco, San Francisco, CA USA">
<meta name="citation_author_institution" content="Biomedical Sciences Graduate Program, University of California, San Francisco, San Francisco, CA USA">
<meta name="citation_author" content="Yong Chen">
<meta name="citation_author_institution" content="Department of Cell &amp; Tissue Biology, University of California, San Francisco, San Francisco, CA USA">
<meta name="citation_author_institution" content="Diabetes Center, University of California, San Francisco, San Francisco, CA USA">
<meta name="citation_author_institution" content="Eli and Edythe Broad Center of Regeneration Medicine and Stem Cell Research, University of California, San Francisco, San Francisco, CA USA">
<meta name="citation_author_institution" content="Tongji Medical College, Huazhong University of Science and Technology, Wuhan, China">
<meta name="citation_author" content="Charles Berdan">
<meta name="citation_author_institution" content="Department of Chemistry, University of California, Berkeley, Berkeley, CA USA">
<meta name="citation_author_institution" content="Department of Molecular &amp; Cell Biology, University of California, Berkeley, Berkeley, CA USA">
<meta name="citation_author_institution" content="Department of Nutritional Sciences &amp; Toxicology, University of California, Berkeley, Berkeley, CA USA">
<meta name="citation_author" content="Sharon M Louie">
<meta name="citation_author_institution" content="Department of Chemistry, University of California, Berkeley, Berkeley, CA USA">
<meta name="citation_author_institution" content="Department of Molecular &amp; Cell Biology, University of California, Berkeley, Berkeley, CA USA">
<meta name="citation_author_institution" content="Department of Nutritional Sciences &amp; Toxicology, University of California, Berkeley, Berkeley, CA USA">
<meta name="citation_author" content="Celine Mahieu">
<meta name="citation_author_institution" content="Department of Cell &amp; Tissue Biology, University of California, San Francisco, San Francisco, CA USA">
<meta name="citation_author" content="Daphne Superville">
<meta name="citation_author_institution" content="Department of Cell &amp; Tissue Biology, University of California, San Francisco, San Francisco, CA USA">
<meta name="citation_author_institution" content="Biomedical Sciences Graduate Program, University of California, San Francisco, San Francisco, CA USA">
<meta name="citation_author" content="Juliane Winkler">
<meta name="citation_author_institution" content="Department of Anatomy, University of California, San Francisco, San Francisco, CA USA">
<meta name="citation_author_institution" content="Helen Diller Family Comprehensive Cancer Center, University of California, San Francisco, San Francisco, CA USA">
<meta name="citation_author_institution" content="Center for Cancer Research, Medical University of Vienna, Vienna, Austria">
<meta name="citation_author" content="Elizabeth Willey">
<meta name="citation_author_institution" content="Department of Anatomy, University of California, San Francisco, San Francisco, CA USA">
<meta name="citation_author_institution" content="Helen Diller Family Comprehensive Cancer Center, University of California, San Francisco, San Francisco, CA USA">
<meta name="citation_author" content="Erica J Hutchins">
<meta name="citation_author_institution" content="Department of Cell &amp; Tissue Biology, University of California, San Francisco, San Francisco, CA USA">
<meta name="citation_author_institution" content="Eli and Edythe Broad Center of Regeneration Medicine and Stem Cell Research, University of California, San Francisco, San Francisco, CA USA">
<meta name="citation_author" content="John D Gagnon">
<meta name="citation_author_institution" content="Biomedical Sciences Graduate Program, University of California, San Francisco, San Francisco, CA USA">
<meta name="citation_author_institution" content="Department of Microbiology &amp; Immunology, University of California, San Francisco, San Francisco, CA USA">
<meta name="citation_author_institution" content="Sandler Asthma Basic Research Center, University of California, San Francisco, San Francisco, CA USA">
<meta name="citation_author" content="Seda Kilinc Avsaroglu">
<meta name="citation_author_institution" content="Department of Cell &amp; Tissue Biology, University of California, San Francisco, San Francisco, CA USA">
<meta name="citation_author" content="Kosaku Shinoda">
<meta name="citation_author_institution" content="Department of Cell &amp; Tissue Biology, University of California, San Francisco, San Francisco, CA USA">
<meta name="citation_author_institution" content="Diabetes Center, University of California, San Francisco, San Francisco, CA USA">
<meta name="citation_author_institution" content="Eli and Edythe Broad Center of Regeneration Medicine and Stem Cell Research, University of California, San Francisco, San Francisco, CA USA">
<meta name="citation_author_institution" content="Department of Medicine and Molecular Pharmacology, Albert Einstein College of Medicine, Bronx, NY USA">
<meta name="citation_author" content="Matthew Gruner">
<meta name="citation_author_institution" content="Department of Cell &amp; Tissue Biology, University of California, San Francisco, San Francisco, CA USA">
<meta name="citation_author" content="Hiroshi Nishida">
<meta name="citation_author_institution" content="Division of Endocrinology, Diabetes and Metabolism, Beth Israel Deaconess Medical Center, Harvard Medical School, Boston, MA USA">
<meta name="citation_author_institution" content="Howard Hughes Medical Institute, Chevy Chase, MD USA">
<meta name="citation_author" content="K Mark Ansel">
<meta name="citation_author_institution" content="Department of Microbiology &amp; Immunology, University of California, San Francisco, San Francisco, CA USA">
<meta name="citation_author_institution" content="Sandler Asthma Basic Research Center, University of California, San Francisco, San Francisco, CA USA">
<meta name="citation_author" content="Zena Werb">
<meta name="citation_author_institution" content="Department of Anatomy, University of California, San Francisco, San Francisco, CA USA">
<meta name="citation_author_institution" content="Helen Diller Family Comprehensive Cancer Center, University of California, San Francisco, San Francisco, CA USA">
<meta name="citation_author" content="Daniel K Nomura">
<meta name="citation_author_institution" content="Department of Chemistry, University of California, Berkeley, Berkeley, CA USA">
<meta name="citation_author_institution" content="Department of Molecular &amp; Cell Biology, University of California, Berkeley, Berkeley, CA USA">
<meta name="citation_author_institution" content="Department of Nutritional Sciences &amp; Toxicology, University of California, Berkeley, Berkeley, CA USA">
<meta name="citation_author" content="Shingo Kajimura">
<meta name="citation_author_institution" content="Department of Cell &amp; Tissue Biology, University of California, San Francisco, San Francisco, CA USA">
<meta name="citation_author_institution" content="Diabetes Center, University of California, San Francisco, San Francisco, CA USA">
<meta name="citation_author_institution" content="Eli and Edythe Broad Center of Regeneration Medicine and Stem Cell Research, University of California, San Francisco, San Francisco, CA USA">
<meta name="citation_author_institution" content="Division of Endocrinology, Diabetes and Metabolism, Beth Israel Deaconess Medical Center, Harvard Medical School, Boston, MA USA">
<meta name="citation_author_institution" content="Howard Hughes Medical Institute, Chevy Chase, MD USA">
<meta name="citation_author" content="Atul J Butte">
<meta name="citation_author_institution" content="Faculty of Biology, Technion, Israel Institute of Technology, Haifa, Israel">
<meta name="citation_author" content="Melinda E Sanders">
<meta name="citation_author_institution" content="Department of Pathology, Vanderbilt University School of Medicine, Nashville, TN USA">
<meta name="citation_author" content="Daniel C Liebler">
<meta name="citation_author_institution" content="Department of Biochemistry, Vanderbilt University School of Medicine, Nashville, TN USA">
<meta name="citation_author_institution" content="Jim Ayers Institute for Precancer Detection and Diagnosis, Vanderbilt-Ingram Cancer Center, Nashville, TN USA">
<meta name="citation_author" content="Hope S Rugo">
<meta name="citation_author_institution" content="Department of Medicine, University of California, San Francisco, San Francisco, CA USA">
<meta name="citation_author_institution" content="Department of Medical Oncology &amp; Therapeutics Research, City of Hope Comprehensive Cancer Center, Duarte, CA USA">
<meta name="citation_author" content="Gregor Krings">
<meta name="citation_author_institution" content="Department of Pathology, University of California, San Francisco, San Francisco, CA USA">
<meta name="citation_author" content="John A Shepherd">
<meta name="citation_author_institution" content="Cancer Center, University of Hawaii, Honolulu, HI USA">
<meta name="citation_author" content="Andrei Goga">
<meta name="citation_author_institution" content="Department of Cell &amp; Tissue Biology, University of California, San Francisco, San Francisco, CA USA">
<meta name="citation_author_institution" content="Department of Medicine, University of California, San Francisco, San Francisco, CA USA">
<meta name="citation_author_institution" content="Helen Diller Family Comprehensive Cancer Center, University of California, San Francisco, San Francisco, CA USA">
<meta name="citation_publication_date" content="2025 Aug 20">
<meta name="citation_volume" content="16">
<meta name="citation_firstpage" content="7438">
<meta name="citation_doi" content="10.1038/s41467-025-62486-3">
<meta name="citation_pmid" content="40835606">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12368202/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12368202/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12368202/pdf/41467_2025_Article_62486.pdf">
<meta name="description" content="A pro-tumorigenic role for adipocytes has been identified in breast cancer, and reliance on fatty acid catabolism found in aggressive tumors. The molecular mechanisms by which tumor cells coopt neighboring adipocytes, however, remain incompletely ...">
<meta name="og:title" content="Tumor cell-adipocyte gap junctions activate lipolysis and contribute to breast tumorigenesis">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="A pro-tumorigenic role for adipocytes has been identified in breast cancer, and reliance on fatty acid catabolism found in aggressive tumors. The molecular mechanisms by which tumor cells coopt neighboring adipocytes, however, remain incompletely ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12368202/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Searchâ€¦
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12368202">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1038/s41467-025-62486-3"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/41467_2025_Article_62486.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12368202%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12368202/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12368202/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12368202/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-ncomms.gif" alt="Nature Communications logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Nature Communications" title="Link to Nature Communications" shape="default" href="http://www.nature.com/ncomms/index.html" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Nat Commun</button></div>. 2025 Aug 20;16:7438. doi: <a href="https://doi.org/10.1038/s41467-025-62486-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1038/s41467-025-62486-3</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Nat%20Commun%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nat%20Commun%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Nat%20Commun%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Nat%20Commun%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Tumor cell-adipocyte gap junctions activate lipolysis and contribute to breast tumorigenesis</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Williams%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Jeremy Williams</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Jeremy Williams</span></h3>
<div class="p">
<sup>1</sup>Department of Cell &amp; Tissue Biology, University of California, San Francisco, San Francisco, CA USA </div>
<div class="p">
<sup>2</sup>Biomedical Sciences Graduate Program, University of California, San Francisco, San Francisco, CA USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Williams%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Jeremy Williams</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Camarda%20R%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Roman Camarda</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Roman Camarda</span></h3>
<div class="p">
<sup>1</sup>Department of Cell &amp; Tissue Biology, University of California, San Francisco, San Francisco, CA USA </div>
<div class="p">
<sup>2</sup>Biomedical Sciences Graduate Program, University of California, San Francisco, San Francisco, CA USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Camarda%20R%22%5BAuthor%5D" class="usa-link"><span class="name western">Roman Camarda</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Malkov%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Serghei Malkov</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Serghei Malkov</span></h3>
<div class="p">
<sup>3</sup>Department of Radiology &amp; Biomedical Imaging, University of California, San Francisco, San Francisco, CA USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Malkov%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Serghei Malkov</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zimmerman%20LJ%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Lisa J Zimmerman</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Lisa J Zimmerman</span></h3>
<div class="p">
<sup>4</sup>Department of Biochemistry, Vanderbilt University School of Medicine, Nashville, TN USA </div>
<div class="p">
<sup>5</sup>Jim Ayers Institute for Precancer Detection and Diagnosis, Vanderbilt-Ingram Cancer Center, Nashville, TN USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zimmerman%20LJ%22%5BAuthor%5D" class="usa-link"><span class="name western">Lisa J Zimmerman</span></a>
</div>
</div>
<sup>4,</sup><sup>5</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Manning%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Suzanne Manning</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Suzanne Manning</span></h3>
<div class="p">
<sup>6</sup>Department of Pathology, Vanderbilt University School of Medicine, Nashville, TN USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Manning%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Suzanne Manning</span></a>
</div>
</div>
<sup>6</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Aran%20D%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Dvir Aran</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Dvir Aran</span></h3>
<div class="p">
<sup>7</sup>Faculty of Biology, Technion, Israel Institute of Technology, Haifa, Israel </div>
<div class="p">
<sup>8</sup>The Taub Faculty of Computer Science, Technion, Israel Institute of Technology, Haifa, Israel </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Aran%20D%22%5BAuthor%5D" class="usa-link"><span class="name western">Dvir Aran</span></a>
</div>
</div>
<sup>7,</sup><sup>8</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Beardsley%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Andrew Beardsley</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Andrew Beardsley</span></h3>
<div class="p">
<sup>1</sup>Department of Cell &amp; Tissue Biology, University of California, San Francisco, San Francisco, CA USA </div>
<div class="p">
<sup>9</sup>Department of Medicine, University of California, San Francisco, San Francisco, CA USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Beardsley%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Andrew Beardsley</span></a>
</div>
</div>
<sup>1,</sup><sup>9</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Van%20de%20Mark%20D%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Daniel Van de Mark</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Daniel Van de Mark</span></h3>
<div class="p">
<sup>1</sup>Department of Cell &amp; Tissue Biology, University of California, San Francisco, San Francisco, CA USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Van%20de%20Mark%20D%22%5BAuthor%5D" class="usa-link"><span class="name western">Daniel Van de Mark</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nakagawa%20R%22%5BAuthor%5D" class="usa-link" aria-describedby="id9"><span class="name western">Rachel Nakagawa</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">Rachel Nakagawa</span></h3>
<div class="p">
<sup>1</sup>Department of Cell &amp; Tissue Biology, University of California, San Francisco, San Francisco, CA USA </div>
<div class="p">
<sup>2</sup>Biomedical Sciences Graduate Program, University of California, San Francisco, San Francisco, CA USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nakagawa%20R%22%5BAuthor%5D" class="usa-link"><span class="name western">Rachel Nakagawa</span></a>
</div>
</div>
<sup>1,</sup><sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chen%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id10"><span class="name western">Yong Chen</span></a><div hidden="hidden" id="id10">
<h3><span class="name western">Yong Chen</span></h3>
<div class="p">
<sup>1</sup>Department of Cell &amp; Tissue Biology, University of California, San Francisco, San Francisco, CA USA </div>
<div class="p">
<sup>10</sup>Diabetes Center, University of California, San Francisco, San Francisco, CA USA </div>
<div class="p">
<sup>11</sup>Eli and Edythe Broad Center of Regeneration Medicine and Stem Cell Research, University of California, San Francisco, San Francisco, CA USA </div>
<div class="p">
<sup>12</sup>Tongji Medical College, Huazhong University of Science and Technology, Wuhan, China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chen%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yong Chen</span></a>
</div>
</div>
<sup>1,</sup><sup>10,</sup><sup>11,</sup><sup>12</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Berdan%20C%22%5BAuthor%5D" class="usa-link" aria-describedby="id11"><span class="name western">Charles Berdan</span></a><div hidden="hidden" id="id11">
<h3><span class="name western">Charles Berdan</span></h3>
<div class="p">
<sup>13</sup>Department of Chemistry, University of California, Berkeley, Berkeley, CA USA </div>
<div class="p">
<sup>14</sup>Department of Molecular &amp; Cell Biology, University of California, Berkeley, Berkeley, CA USA </div>
<div class="p">
<sup>15</sup>Department of Nutritional Sciences &amp; Toxicology, University of California, Berkeley, Berkeley, CA USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Berdan%20C%22%5BAuthor%5D" class="usa-link"><span class="name western">Charles Berdan</span></a>
</div>
</div>
<sup>13,</sup><sup>14,</sup><sup>15</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Louie%20SM%22%5BAuthor%5D" class="usa-link" aria-describedby="id12"><span class="name western">Sharon M Louie</span></a><div hidden="hidden" id="id12">
<h3><span class="name western">Sharon M Louie</span></h3>
<div class="p">
<sup>13</sup>Department of Chemistry, University of California, Berkeley, Berkeley, CA USA </div>
<div class="p">
<sup>14</sup>Department of Molecular &amp; Cell Biology, University of California, Berkeley, Berkeley, CA USA </div>
<div class="p">
<sup>15</sup>Department of Nutritional Sciences &amp; Toxicology, University of California, Berkeley, Berkeley, CA USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Louie%20SM%22%5BAuthor%5D" class="usa-link"><span class="name western">Sharon M Louie</span></a>
</div>
</div>
<sup>13,</sup><sup>14,</sup><sup>15</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mahieu%20C%22%5BAuthor%5D" class="usa-link" aria-describedby="id13"><span class="name western">Celine Mahieu</span></a><div hidden="hidden" id="id13">
<h3><span class="name western">Celine Mahieu</span></h3>
<div class="p">
<sup>1</sup>Department of Cell &amp; Tissue Biology, University of California, San Francisco, San Francisco, CA USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mahieu%20C%22%5BAuthor%5D" class="usa-link"><span class="name western">Celine Mahieu</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Superville%20D%22%5BAuthor%5D" class="usa-link" aria-describedby="id14"><span class="name western">Daphne Superville</span></a><div hidden="hidden" id="id14">
<h3><span class="name western">Daphne Superville</span></h3>
<div class="p">
<sup>1</sup>Department of Cell &amp; Tissue Biology, University of California, San Francisco, San Francisco, CA USA </div>
<div class="p">
<sup>2</sup>Biomedical Sciences Graduate Program, University of California, San Francisco, San Francisco, CA USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Superville%20D%22%5BAuthor%5D" class="usa-link"><span class="name western">Daphne Superville</span></a>
</div>
</div>
<sup>1,</sup><sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Winkler%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id15"><span class="name western">Juliane Winkler</span></a><div hidden="hidden" id="id15">
<h3><span class="name western">Juliane Winkler</span></h3>
<div class="p">
<sup>16</sup>Department of Anatomy, University of California, San Francisco, San Francisco, CA USA </div>
<div class="p">
<sup>17</sup>Helen Diller Family Comprehensive Cancer Center, University of California, San Francisco, San Francisco, CA USA </div>
<div class="p">
<sup>18</sup>Center for Cancer Research, Medical University of Vienna, Vienna, Austria </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Winkler%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Juliane Winkler</span></a>
</div>
</div>
<sup>16,</sup><sup>17,</sup><sup>18</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Willey%20E%22%5BAuthor%5D" class="usa-link" aria-describedby="id16"><span class="name western">Elizabeth Willey</span></a><div hidden="hidden" id="id16">
<h3><span class="name western">Elizabeth Willey</span></h3>
<div class="p">
<sup>16</sup>Department of Anatomy, University of California, San Francisco, San Francisco, CA USA </div>
<div class="p">
<sup>17</sup>Helen Diller Family Comprehensive Cancer Center, University of California, San Francisco, San Francisco, CA USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Willey%20E%22%5BAuthor%5D" class="usa-link"><span class="name western">Elizabeth Willey</span></a>
</div>
</div>
<sup>16,</sup><sup>17</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hutchins%20EJ%22%5BAuthor%5D" class="usa-link" aria-describedby="id17"><span class="name western">Erica J Hutchins</span></a><div hidden="hidden" id="id17">
<h3><span class="name western">Erica J Hutchins</span></h3>
<div class="p">
<sup>1</sup>Department of Cell &amp; Tissue Biology, University of California, San Francisco, San Francisco, CA USA </div>
<div class="p">
<sup>11</sup>Eli and Edythe Broad Center of Regeneration Medicine and Stem Cell Research, University of California, San Francisco, San Francisco, CA USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hutchins%20EJ%22%5BAuthor%5D" class="usa-link"><span class="name western">Erica J Hutchins</span></a>
</div>
</div>
<sup>1,</sup><sup>11</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gagnon%20JD%22%5BAuthor%5D" class="usa-link" aria-describedby="id18"><span class="name western">John D Gagnon</span></a><div hidden="hidden" id="id18">
<h3><span class="name western">John D Gagnon</span></h3>
<div class="p">
<sup>2</sup>Biomedical Sciences Graduate Program, University of California, San Francisco, San Francisco, CA USA </div>
<div class="p">
<sup>19</sup>Department of Microbiology &amp; Immunology, University of California, San Francisco, San Francisco, CA USA </div>
<div class="p">
<sup>20</sup>Sandler Asthma Basic Research Center, University of California, San Francisco, San Francisco, CA USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gagnon%20JD%22%5BAuthor%5D" class="usa-link"><span class="name western">John D Gagnon</span></a>
</div>
</div>
<sup>2,</sup><sup>19,</sup><sup>20</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Avsaroglu%20SK%22%5BAuthor%5D" class="usa-link" aria-describedby="id19"><span class="name western">Seda Kilinc Avsaroglu</span></a><div hidden="hidden" id="id19">
<h3><span class="name western">Seda Kilinc Avsaroglu</span></h3>
<div class="p">
<sup>1</sup>Department of Cell &amp; Tissue Biology, University of California, San Francisco, San Francisco, CA USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Avsaroglu%20SK%22%5BAuthor%5D" class="usa-link"><span class="name western">Seda Kilinc Avsaroglu</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Shinoda%20K%22%5BAuthor%5D" class="usa-link" aria-describedby="id20"><span class="name western">Kosaku Shinoda</span></a><div hidden="hidden" id="id20">
<h3><span class="name western">Kosaku Shinoda</span></h3>
<div class="p">
<sup>1</sup>Department of Cell &amp; Tissue Biology, University of California, San Francisco, San Francisco, CA USA </div>
<div class="p">
<sup>10</sup>Diabetes Center, University of California, San Francisco, San Francisco, CA USA </div>
<div class="p">
<sup>11</sup>Eli and Edythe Broad Center of Regeneration Medicine and Stem Cell Research, University of California, San Francisco, San Francisco, CA USA </div>
<div class="p">
<sup>21</sup>Department of Medicine and Molecular Pharmacology, Albert Einstein College of Medicine, Bronx, NY USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Shinoda%20K%22%5BAuthor%5D" class="usa-link"><span class="name western">Kosaku Shinoda</span></a>
</div>
</div>
<sup>1,</sup><sup>10,</sup><sup>11,</sup><sup>21</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gruner%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id21"><span class="name western">Matthew Gruner</span></a><div hidden="hidden" id="id21">
<h3><span class="name western">Matthew Gruner</span></h3>
<div class="p">
<sup>1</sup>Department of Cell &amp; Tissue Biology, University of California, San Francisco, San Francisco, CA USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gruner%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Matthew Gruner</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nishida%20H%22%5BAuthor%5D" class="usa-link" aria-describedby="id22"><span class="name western">Hiroshi Nishida</span></a><div hidden="hidden" id="id22">
<h3><span class="name western">Hiroshi Nishida</span></h3>
<div class="p">
<sup>22</sup>Division of Endocrinology, Diabetes and Metabolism, Beth Israel Deaconess Medical Center, Harvard Medical School, Boston, MA USA </div>
<div class="p">
<sup>23</sup>Howard Hughes Medical Institute, Chevy Chase, MD USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nishida%20H%22%5BAuthor%5D" class="usa-link"><span class="name western">Hiroshi Nishida</span></a>
</div>
</div>
<sup>22,</sup><sup>23</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ansel%20KM%22%5BAuthor%5D" class="usa-link" aria-describedby="id23"><span class="name western">K Mark Ansel</span></a><div hidden="hidden" id="id23">
<h3><span class="name western">K Mark Ansel</span></h3>
<div class="p">
<sup>19</sup>Department of Microbiology &amp; Immunology, University of California, San Francisco, San Francisco, CA USA </div>
<div class="p">
<sup>20</sup>Sandler Asthma Basic Research Center, University of California, San Francisco, San Francisco, CA USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ansel%20KM%22%5BAuthor%5D" class="usa-link"><span class="name western">K Mark Ansel</span></a>
</div>
</div>
<sup>19,</sup><sup>20</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Werb%20Z%22%5BAuthor%5D" class="usa-link" aria-describedby="id24"><span class="name western">Zena Werb</span></a><div hidden="hidden" id="id24">
<h3><span class="name western">Zena Werb</span></h3>
<div class="p">
<sup>16</sup>Department of Anatomy, University of California, San Francisco, San Francisco, CA USA </div>
<div class="p">
<sup>17</sup>Helen Diller Family Comprehensive Cancer Center, University of California, San Francisco, San Francisco, CA USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Werb%20Z%22%5BAuthor%5D" class="usa-link"><span class="name western">Zena Werb</span></a>
</div>
</div>
<sup>16,</sup><sup>17</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nomura%20DK%22%5BAuthor%5D" class="usa-link" aria-describedby="id25"><span class="name western">Daniel K Nomura</span></a><div hidden="hidden" id="id25">
<h3><span class="name western">Daniel K Nomura</span></h3>
<div class="p">
<sup>13</sup>Department of Chemistry, University of California, Berkeley, Berkeley, CA USA </div>
<div class="p">
<sup>14</sup>Department of Molecular &amp; Cell Biology, University of California, Berkeley, Berkeley, CA USA </div>
<div class="p">
<sup>15</sup>Department of Nutritional Sciences &amp; Toxicology, University of California, Berkeley, Berkeley, CA USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nomura%20DK%22%5BAuthor%5D" class="usa-link"><span class="name western">Daniel K Nomura</span></a>
</div>
</div>
<sup>13,</sup><sup>14,</sup><sup>15</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kajimura%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id26"><span class="name western">Shingo Kajimura</span></a><div hidden="hidden" id="id26">
<h3><span class="name western">Shingo Kajimura</span></h3>
<div class="p">
<sup>1</sup>Department of Cell &amp; Tissue Biology, University of California, San Francisco, San Francisco, CA USA </div>
<div class="p">
<sup>10</sup>Diabetes Center, University of California, San Francisco, San Francisco, CA USA </div>
<div class="p">
<sup>11</sup>Eli and Edythe Broad Center of Regeneration Medicine and Stem Cell Research, University of California, San Francisco, San Francisco, CA USA </div>
<div class="p">
<sup>22</sup>Division of Endocrinology, Diabetes and Metabolism, Beth Israel Deaconess Medical Center, Harvard Medical School, Boston, MA USA </div>
<div class="p">
<sup>23</sup>Howard Hughes Medical Institute, Chevy Chase, MD USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kajimura%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Shingo Kajimura</span></a>
</div>
</div>
<sup>1,</sup><sup>10,</sup><sup>11,</sup><sup>22,</sup><sup>23</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Butte%20AJ%22%5BAuthor%5D" class="usa-link" aria-describedby="id27"><span class="name western">Atul J Butte</span></a><div hidden="hidden" id="id27">
<h3><span class="name western">Atul J Butte</span></h3>
<div class="p">
<sup>7</sup>Faculty of Biology, Technion, Israel Institute of Technology, Haifa, Israel </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Butte%20AJ%22%5BAuthor%5D" class="usa-link"><span class="name western">Atul J Butte</span></a>
</div>
</div>
<sup>7</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sanders%20ME%22%5BAuthor%5D" class="usa-link" aria-describedby="id28"><span class="name western">Melinda E Sanders</span></a><div hidden="hidden" id="id28">
<h3><span class="name western">Melinda E Sanders</span></h3>
<div class="p">
<sup>6</sup>Department of Pathology, Vanderbilt University School of Medicine, Nashville, TN USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sanders%20ME%22%5BAuthor%5D" class="usa-link"><span class="name western">Melinda E Sanders</span></a>
</div>
</div>
<sup>6</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Liebler%20DC%22%5BAuthor%5D" class="usa-link" aria-describedby="id29"><span class="name western">Daniel C Liebler</span></a><div hidden="hidden" id="id29">
<h3><span class="name western">Daniel C Liebler</span></h3>
<div class="p">
<sup>4</sup>Department of Biochemistry, Vanderbilt University School of Medicine, Nashville, TN USA </div>
<div class="p">
<sup>5</sup>Jim Ayers Institute for Precancer Detection and Diagnosis, Vanderbilt-Ingram Cancer Center, Nashville, TN USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Liebler%20DC%22%5BAuthor%5D" class="usa-link"><span class="name western">Daniel C Liebler</span></a>
</div>
</div>
<sup>4,</sup><sup>5</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Rugo%20HS%22%5BAuthor%5D" class="usa-link" aria-describedby="id30"><span class="name western">Hope S Rugo</span></a><div hidden="hidden" id="id30">
<h3><span class="name western">Hope S Rugo</span></h3>
<div class="p">
<sup>9</sup>Department of Medicine, University of California, San Francisco, San Francisco, CA USA </div>
<div class="p">
<sup>24</sup>Department of Medical Oncology &amp; Therapeutics Research, City of Hope Comprehensive Cancer Center, Duarte, CA USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Rugo%20HS%22%5BAuthor%5D" class="usa-link"><span class="name western">Hope S Rugo</span></a>
</div>
</div>
<sup>9,</sup><sup>24</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Krings%20G%22%5BAuthor%5D" class="usa-link" aria-describedby="id31"><span class="name western">Gregor Krings</span></a><div hidden="hidden" id="id31">
<h3><span class="name western">Gregor Krings</span></h3>
<div class="p">
<sup>25</sup>Department of Pathology, University of California, San Francisco, San Francisco, CA USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Krings%20G%22%5BAuthor%5D" class="usa-link"><span class="name western">Gregor Krings</span></a>
</div>
</div>
<sup>25</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Shepherd%20JA%22%5BAuthor%5D" class="usa-link" aria-describedby="id32"><span class="name western">John A Shepherd</span></a><div hidden="hidden" id="id32">
<h3><span class="name western">John A Shepherd</span></h3>
<div class="p">
<sup>26</sup>Cancer Center, University of Hawaii, Honolulu, HI USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Shepherd%20JA%22%5BAuthor%5D" class="usa-link"><span class="name western">John A Shepherd</span></a>
</div>
</div>
<sup>26</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Goga%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id33"><span class="name western">Andrei Goga</span></a><div hidden="hidden" id="id33">
<h3><span class="name western">Andrei Goga</span></h3>
<div class="p">
<sup>1</sup>Department of Cell &amp; Tissue Biology, University of California, San Francisco, San Francisco, CA USA </div>
<div class="p">
<sup>9</sup>Department of Medicine, University of California, San Francisco, San Francisco, CA USA </div>
<div class="p">
<sup>17</sup>Helen Diller Family Comprehensive Cancer Center, University of California, San Francisco, San Francisco, CA USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Goga%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Andrei Goga</span></a>
</div>
</div>
<sup>1,</sup><sup>9,</sup><sup>17,</sup><sup>âœ‰</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>Department of Cell &amp; Tissue Biology, University of California, San Francisco, San Francisco, CA USA </div>
<div id="Aff2">
<sup>2</sup>Biomedical Sciences Graduate Program, University of California, San Francisco, San Francisco, CA USA </div>
<div id="Aff3">
<sup>3</sup>Department of Radiology &amp; Biomedical Imaging, University of California, San Francisco, San Francisco, CA USA </div>
<div id="Aff4">
<sup>4</sup>Department of Biochemistry, Vanderbilt University School of Medicine, Nashville, TN USA </div>
<div id="Aff5">
<sup>5</sup>Jim Ayers Institute for Precancer Detection and Diagnosis, Vanderbilt-Ingram Cancer Center, Nashville, TN USA </div>
<div id="Aff6">
<sup>6</sup>Department of Pathology, Vanderbilt University School of Medicine, Nashville, TN USA </div>
<div id="Aff7">
<sup>7</sup>Faculty of Biology, Technion, Israel Institute of Technology, Haifa, Israel </div>
<div id="Aff8">
<sup>8</sup>The Taub Faculty of Computer Science, Technion, Israel Institute of Technology, Haifa, Israel </div>
<div id="Aff9">
<sup>9</sup>Department of Medicine, University of California, San Francisco, San Francisco, CA USA </div>
<div id="Aff10">
<sup>10</sup>Diabetes Center, University of California, San Francisco, San Francisco, CA USA </div>
<div id="Aff11">
<sup>11</sup>Eli and Edythe Broad Center of Regeneration Medicine and Stem Cell Research, University of California, San Francisco, San Francisco, CA USA </div>
<div id="Aff12">
<sup>12</sup>Tongji Medical College, Huazhong University of Science and Technology, Wuhan, China </div>
<div id="Aff13">
<sup>13</sup>Department of Chemistry, University of California, Berkeley, Berkeley, CA USA </div>
<div id="Aff14">
<sup>14</sup>Department of Molecular &amp; Cell Biology, University of California, Berkeley, Berkeley, CA USA </div>
<div id="Aff15">
<sup>15</sup>Department of Nutritional Sciences &amp; Toxicology, University of California, Berkeley, Berkeley, CA USA </div>
<div id="Aff16">
<sup>16</sup>Department of Anatomy, University of California, San Francisco, San Francisco, CA USA </div>
<div id="Aff17">
<sup>17</sup>Helen Diller Family Comprehensive Cancer Center, University of California, San Francisco, San Francisco, CA USA </div>
<div id="Aff18">
<sup>18</sup>Center for Cancer Research, Medical University of Vienna, Vienna, Austria </div>
<div id="Aff19">
<sup>19</sup>Department of Microbiology &amp; Immunology, University of California, San Francisco, San Francisco, CA USA </div>
<div id="Aff20">
<sup>20</sup>Sandler Asthma Basic Research Center, University of California, San Francisco, San Francisco, CA USA </div>
<div id="Aff21">
<sup>21</sup>Department of Medicine and Molecular Pharmacology, Albert Einstein College of Medicine, Bronx, NY USA </div>
<div id="Aff22">
<sup>22</sup>Division of Endocrinology, Diabetes and Metabolism, Beth Israel Deaconess Medical Center, Harvard Medical School, Boston, MA USA </div>
<div id="Aff23">
<sup>23</sup>Howard Hughes Medical Institute, Chevy Chase, MD USA </div>
<div id="Aff24">
<sup>24</sup>Department of Medical Oncology &amp; Therapeutics Research, City of Hope Comprehensive Cancer Center, Duarte, CA USA </div>
<div id="Aff25">
<sup>25</sup>Department of Pathology, University of California, San Francisco, San Francisco, CA USA </div>
<div id="Aff26">
<sup>26</sup>Cancer Center, University of Hawaii, Honolulu, HI USA </div>
<div class="author-notes p">
<div class="fn" id="_fncrsp93pmc__">
<sup>âœ‰</sup><p class="display-inline">Corresponding author.</p>
</div>
<div class="fn" id="_eqcntrb93pmc__">
<sup>#</sup><p class="display-inline">Contributed equally.</p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2024 Oct 26; Accepted 2025 Jul 18; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>Â© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the articleâ€™s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the articleâ€™s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12368202Â Â PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40835606/" class="usa-link">40835606</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<p id="Par1">A pro-tumorigenic role for adipocytes has been identified in breast cancer, and reliance on fatty acid catabolism found in aggressive tumors. The molecular mechanisms by which tumor cells coopt neighboring adipocytes, however, remain incompletely understood. Here, we describe a direct interaction linking tumorigenesis to adjacent adipocytes. We examine breast tumors and their normal adjacent tissue from several patient cohorts, patient-derived xenografts, and mouse models, and find that lipolysis and lipolytic signaling are activated in neighboring adipose tissue. We find that functional gap junctions form between breast cancer cells and adipocytes. As a result, cAMP is transferred from breast cancer cells to adipocytes and activates lipolysis in a gap junction-dependent manner. We find that connexin 31 (<em>GJB3</em>) promotes receptor triple negative breast cancer growth and activation of lipolysis in vivo. Thus, direct tumor cell-adipocyte interaction contributes to tumorigenesis and may serve as a new therapeutic target in breast cancer.</p>
<section id="kwd-group1" class="kwd-group"><p><strong>Subject terms:</strong> Breast cancer, Cancer metabolism</p></section></section><section class="abstract" id="Abs2"><hr class="headless">
<p id="Par2">Breast cancer cells interact with neighbouring adipocytes, but the mechanisms are not fully understood. Here, the authors show that triple-negative breast cancer (TNBC) cells transfer cAMP through gap junctions, activating lipolysis in tumour-associated adipocytes to promote TNBC growth.</p></section><section id="Sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p id="Par3">A variety of cancers, including those of the breast, arise within adipose tissue depots<sup><a href="#CR1" class="usa-link" aria-describedby="CR1">1</a></sup>. Therefore, heterotypic cell-cell interactions exist between adipocytes and invading cancer cells in these organs during tumor development. We and others discovered that triple-negative breast cancers (TNBC, estrogen/progesterone/HER2 receptor-negative) utilize and require fatty acid oxidation (FAO) to fuel bioenergetic metabolism<sup><a href="#CR2" class="usa-link" aria-describedby="CR2">2</a>,<a href="#CR3" class="usa-link" aria-describedby="CR3">3</a></sup>. Epidemiological studies have identified an increased risk for TNBC cancers to form in premenopausal patients with elevated BMI, suggesting that increased adiposity may contribute to tumorigenesis<sup><a href="#CR4" class="usa-link" aria-describedby="CR4">4</a></sup>. Nevertheless, how TNBCs acquire fatty acids for tumor growth remains unclear.</p>
<p id="Par4">Prior work has demonstrated that cancer cells interact indirectly with adipocytes through release of chemokines, cytokines, and other paracrine factors that can remotely activate adipocyte lipolysis. In turn, cancer-associated adipocytes undergoing lipolysis can secrete pro-tumorigenic cytokines, including tumor necrosis factor-Î±<sup><a href="#CR5" class="usa-link" aria-describedby="CR5">5</a>â€“<a href="#CR12" class="usa-link" aria-describedby="CR12">12</a></sup>. Whether direct interactions at the cancer cell-adipocyte interface contribute to lipolysis in the breast tumor niche, however, is not known.</p>
<p id="Par5">Multiple studies found that adipocyte-derived fatty acids can be taken up and oxidized by proximate cancer cells<sup><a href="#CR5" class="usa-link" aria-describedby="CR5">5</a>â€“<a href="#CR11" class="usa-link" aria-describedby="CR11">11</a>,<a href="#CR13" class="usa-link" aria-describedby="CR13">13</a></sup>. These studies, however, have widely modeled the cancer-adipocyte interface in vitro using transwell co-culture methods that cannot recapitulate the direct cell-cell contact observed in vivo<sup><a href="#CR7" class="usa-link" aria-describedby="CR7">7</a>â€“<a href="#CR10" class="usa-link" aria-describedby="CR10">10</a>,<a href="#CR12" class="usa-link" aria-describedby="CR12">12</a>â€“<a href="#CR14" class="usa-link" aria-describedby="CR14">14</a></sup>. Furthermore, clinical evidence for elevated lipolysis in breast tumor-adjacent adipocytes has not been well established. Mammary adipocytes undergo enhanced lipolysis when in close proximity to non-tumor epithelial cells, suggesting that local pro-lipolytic mechanisms exist, but have yet to be identified between tumor cells and adipocytes<sup><a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>,<a href="#CR15" class="usa-link" aria-describedby="CR15">15</a></sup>.</p>
<p id="Par6">In this study, we investigate the breast tumor-adipocyte interface and examine how cell-cell contact contributes to tumorigenesis. We find diminished lipid content and smaller adipocytes adjacent to patient breast tumors and asked whether this indicates a mechanism of adipocyte lipolysis which depends on tumor cell proximity. Through analysis of patient breast tumors and normal adjacent tissue, direct cancer cell-adipocyte co-cultures, and in vivo xenograft models, we present a mechanism of contact-dependent lipolytic signaling transduced from cancer cells to adipocytes by gap junctions containing connexin (Cx) 31 (<em>GJB3</em>). We demonstrate that tumor Cx31 depletion diminishes MYC-high TNBC tumor growth.</p></section><section id="Sec2"><h2 class="pmc_sec_title">Results</h2>
<section id="Sec3"><h3 class="pmc_sec_title">Diminished lipid content in breast tumor-adjacent adipocytes</h3>
<p id="Par7">To determine if lipolysis occurs in normal tissue adjacent to breast tumors (NAT), which includes adipocytes, we employed four independent strategies. First, we employed three-component breast (3CB) composition measurement, a radiographic imaging method derived from dual-energy mammography that allows for quantification of a tissueâ€™s water, lipid, and protein content<sup><a href="#CR16" class="usa-link" aria-describedby="CR16">16</a></sup>. We postulated that, if tumors induce lipolysis in adipocytes, we will observe differences in lipid content between normal adjacent tissue (NAT) nearer to the tumor and NAT farther away. Using 3CB imaging, we assessed the lipid content of breast tumors and the first 6â€‰mm of surrounding NAT, segmented into 2â€‰mm â€œconcentric ringsâ€ from 46 patients with invasive breast cancer (Fig.Â <a href="#Fig1" class="usa-link">1a</a> and Supplementary DataÂ <a href="#MOESM3" class="usa-link">1</a>). As we have previously demonstrated<sup><a href="#CR17" class="usa-link" aria-describedby="CR17">17</a></sup>, we found a significant decrease in lipid content in tumor lesions compared to NAT 0â€“2â€‰mm away (R1) (Fig.Â <a href="#Fig1" class="usa-link">1b</a>). This difference is congruent with breast tumors being epithelial in nature, while adipose tissue is the major constituent of normal breast<sup><a href="#CR15" class="usa-link" aria-describedby="CR15">15</a></sup>. We also found that within NAT there was a significant stepwise decrease in lipid content comparing R3 (4â€“6â€‰mm) to R2 (2â€“4â€‰mm), and R2 to R1 (Fig.Â <a href="#Fig1" class="usa-link">1b</a>). In addition, we asked whether changes in lipid content between R3 and R1 NAT correlate with receptor status or tumor grade (Supplementary DataÂ <a href="#MOESM3" class="usa-link">1</a> and Supplementary TableÂ <a href="#MOESM1" class="usa-link">1</a>). We found that NAT surrounding triple-negative (TN) and grade 2/3 tumors trended towards a greater average decrease in lipid content between R3 and R1 than NAT surrounding receptor-positive (RP) and grade 1 tumors, respectively (Supplementary Fig.Â <a href="#MOESM1" class="usa-link">1a, b</a>). These data suggest that adipocytes near breast tumors have partially depleted lipid stores, and that TN and higher-grade tumors may induce this phenomenon to a greater degree than RP and low-grade tumors. We quantified average adipocyte size in R1 and R3 in the 11 of the 46 patients imaged with 3CB for whom histological sections of treatment-naÃ¯ve tumor and NAT at the time of surgical resection were available (Fig.Â <a href="#Fig1" class="usa-link">1a</a>, Supplementary Fig.Â <a href="#MOESM1" class="usa-link">1c</a>, and Supplementary DataÂ <a href="#MOESM3" class="usa-link">1</a>). Similar to the change in lipid content observed with 3CB, we found a significant decrease in adipocyte size in R1 compared to R3 in all patients analyzed, suggesting adipocytes are smaller when nearer to breast tumors (Fig.Â <a href="#Fig1" class="usa-link">1c</a>). Finally, we correlated the change in lipid content and adipocyte size on an individual patient basis. We found a positive trend (<em>R</em>â€‰=â€‰0.5818, <em>p</em>â€‰=â€‰0.0656) between the change in lipid content and adipocyte area (Fig.Â <a href="#Fig1" class="usa-link">1d</a>). Taken together, these data suggest adipocytes are smaller and have diminished lipid content, two phenotypes that are established indicators of lipolysis<sup><a href="#CR18" class="usa-link" aria-describedby="CR18">18</a></sup>, when adjacent to breast tumors.</p>
<figure class="fig xbox font-sm" id="Fig1"><h4 class="obj_head">Fig. 1. Lipolysis and lipolytic signaling are activated in breast tumor-adjacent adipocytes from breast cancer patients and mouse models of breast cancer.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12368202_41467_2025_62486_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/0818/12368202/3a3fc9bafce8/41467_2025_62486_Fig1_HTML.jpg" loading="lazy" id="d33e1012" height="996" width="744" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Representative lipid content image (left) and hematoxylin and eosin-stained excision specimen (right) from patients with invasive breast cancer. Lesion (L), and NAT 0â€“2â€‰mm (R1), 2â€“4â€‰mm (R2), and 4â€“6â€‰mm (R3) away are indicated. <strong>b</strong> Percent lipid content (lipid content/lipidâ€‰+â€‰waterâ€‰+â€‰protein content) of L, R1, R2, and R3 from patients (<em>n</em>â€‰=â€‰46) with invasive breast cancer (L vs R1 <em>p</em>â€‰&lt;â€‰0.0001, R1 vs R2 <em>p</em>â€‰=â€‰0.0038, R2 vs R3 <em>p</em>â€‰=â€‰0.0451). <strong>c</strong> Adipocyte area in R1 (blue) and R3 (red) from a subset of patients (<em>n</em>â€‰=â€‰11) in (<strong>b</strong>). Black line indicates mean adipocyte area, and patient identifiers are indicated. Each point represents an individual adipocyte. <strong>d</strong> Correlation of change in lipid content in (<strong>b</strong>) and change in average adipocyte area in (<strong>c</strong>) from R3 to R1 for matched patients in (<strong>c</strong>). <strong>e</strong> ssGSEA enrichment scores for cAMP-dependent lipolysis signature in primary breast tumors (<em>n</em>â€‰=â€‰9), NAT 1â€‰cm (<em>n</em>â€‰=â€‰7), 2â€‰cm (<em>n</em>â€‰=â€‰5), 3â€‰cm (<em>n</em>â€‰=â€‰3), and 4â€‰cm (<em>n</em>â€‰=â€‰4), and healthy non-tumor breast tissue (<em>n</em>â€‰=â€‰10). <strong>f</strong> HNF4Î± protein abundance from LC-MS/MS of primary healthy control breast tissue (<em>n</em>â€‰=â€‰42, <em>p</em>â€‰&lt;â€‰0.0001), NAT (<em>n</em>â€‰=â€‰4) and stroma (<em>n</em>â€‰=â€‰36, <em>p</em>â€‰&lt;â€‰0.0001), and of luminal A (<em>n</em>â€‰=â€‰38, <em>p</em>â€‰&lt;â€‰0.0001), luminal B (<em>n</em>â€‰=â€‰6, <em>p</em>â€‰&lt;â€‰0.0001), luminal A/B (<em>n</em>â€‰=â€‰1, <em>p</em>â€‰=â€‰0.0153), HER2-amplified (<em>n</em>â€‰=â€‰9, <em>p</em>â€‰&lt;â€‰0.0001), HER2-amplified/luminal B (<em>n</em>â€‰=â€‰5, <em>p</em>â€‰&lt;â€‰0.0001), and basal (<em>n</em>â€‰=â€‰16, <em>p</em>â€‰&lt;â€‰0.0001) tumors. Each point represents individual sample LCM on which LC-MS/MS was performed; LCM and LC-MS/MS were performed in (<em>n</em>â€‰=â€‰2) technical duplicates on sequential histological slides from each patient, and technical duplicates are displayed. <strong>g</strong> Immunoblot analysis (left) showing expression levels of lipolysis activators HSL and HNF4Î±, and phosphorylated HSL (pHSL S563) in healthy non-tumor mammary gland and NAT and tumor tissues from a panel of PDXs. Quantification (right) of displayed pHSL/HSL ratio, normalized to b-actin levels, for non-tumor (blue), and NAT (red) and indicated tumors. <strong>h</strong> Immunoblot analysis (left) showing expression levels of lipolysis activators HSL and HNF4Î±, and phosphorylated HSL (pHSL S563) in healthy non-tumor mammary gland (<em>n</em>â€‰=â€‰3 mice), mock-transplanted mammary gland (<em>n</em>â€‰=â€‰3 mice), and NAT and tumor tissues from (<em>n</em>â€‰=â€‰3) MTB-TOM allografts. Quantification (right) of displayed pHSL/HSL ratio, normalized to b-actin level for each biological replicate. For (<strong>b</strong> and <strong>e</strong>), solid black lines indicate matched samples from individual patients. For (<strong>f</strong> and <strong>h</strong>) meanâ€‰Â±â€‰s.e.m. is shown. *<em>P</em>â€‰&lt;â€‰0.05, **<em>P</em>â€‰&lt;â€‰0.01, ***<em>P</em>â€‰&lt;â€‰0.001, ****<em>P</em>â€‰&lt;â€‰0.0001; repeated measures one-way ANOVA with multiple comparisons (<strong>b</strong>), two-way ANOVA with multiple comparisons (<strong>c</strong>), Spearman correlation and two-tailed <em>t</em> test (<strong>d</strong>), repeated measures mixed effects model with multiple comparisons (<strong>e</strong>), ordinary one-way ANOVA with multiple comparisons (<strong>f</strong> and <strong>h</strong>). For (<strong>g</strong> and <strong>h</strong>), the samples derive from the same experiment, but different gels for pHSL(S563), HNF4Î±, and Actin, and another for HSL were processed in parallel. Source data are provided as a Source Data file.</p></figcaption></figure></section><section id="Sec4"><h3 class="pmc_sec_title">Lipolysis and lipolytic signaling are activated in NAT</h3>
<p id="Par8">Second, we sought to determine if gene expression changes associated with lipolysis were observed in tumor-adjacent adipocytes. We generated a lipolysis gene expression signature by identifying the 100 genes most upregulated when a differentiated adipocyte cell culture model is stimulated with cAMP, an inducer of lipolytic signaling<sup><a href="#CR19" class="usa-link" aria-describedby="CR19">19</a></sup>. We then used a publicly available gene expression dataset for primary breast tumors as well as matched NAT 1, 2, 3, and 4â€‰cm away, to determine if enrichment of the lipolysis signature occurred in NAT in comparison to non-tumor breast tissue obtained from healthy individuals using single-set gene set enrichment analysis<sup><a href="#CR20" class="usa-link" aria-describedby="CR20">20</a>,<a href="#CR21" class="usa-link" aria-describedby="CR21">21</a></sup>. We found a significant elevation of the cAMP-dependent lipolysis signature in tumor and NAT from all analyzed regions compared to control tissue (Fig.Â <a href="#Fig1" class="usa-link">1e</a>). These data indicate that lipolytic signaling is activated in breast tumor adjacent adipocytes up to 4â€‰cm away from the primary tumor. While adipose tissue is sparsely innervated, a recent study found that adipocytes can propagate pro-lipolytic sympathetic signals via direct transfer of cAMP through adipocyte-adipocyte gap junctions<sup><a href="#CR22" class="usa-link" aria-describedby="CR22">22</a></sup>. We observed elevation of cAMP signaling up to 4â€‰cm away from patient tumors (Fig.Â <a href="#Fig1" class="usa-link">1e</a>), suggesting that tumor-adjacent adipocytes might also disperse a pro-lipolytic stimulus to distant adipocytes via gap junctions.</p>
<p id="Par9">Third, we sought to determine if there are changes to protein abundance in tumor-adjacent NAT indicative of lipolysis activation. We conducted laser capture microdissection (LCM, approximately 10,000 cells per capture) on primary breast tumors from 75 patients, representing all major PAM50 subtypes. For a subset of patients, we also collected matched stroma and/or NAT. As a control, we conducted LCM on non-tumor breast tissue from 42 healthy subjects (Supplementary DataÂ <a href="#MOESM1" class="usa-link">2a</a>). Global proteomic analysis was performed using liquid chromatography-tandem mass spectrometry (LC-MS/MS) (Supplementary DataÂ <a href="#MOESM1" class="usa-link">2b</a>). Notably, one of the most significantly upregulated proteins in NAT, and indeed one of the most NAT-specific proteins, compared to all other tissues examined was hepatocyte nuclear factor 4-Î± (HNF4Î±) (Fig.Â <a href="#Fig1" class="usa-link">1f</a>). As HNF4Î± is an established, essential activator of lipolysis in adipose tissue<sup><a href="#CR23" class="usa-link" aria-describedby="CR23">23</a></sup>, these data indicate lipolysis is robustly activated in breast tumor adjacent adipose tissue.</p>
<p id="Par10">Fourth, we sought to validate the observations made in our clinical datasets using mouse models of breast cancer. Hormone sensitive lipase (HSL) is a critical lipolytic enzyme; its activation by cAMP-dependent protein kinase A (PKA) leads to phosphorylation at serine 563<sup><a href="#CR18" class="usa-link" aria-describedby="CR18">18</a>,<a href="#CR19" class="usa-link" aria-describedby="CR19">19</a></sup>, while prolonged activation results in downregulation of total HSL expression through a negative feedback mechanism<sup><a href="#CR24" class="usa-link" aria-describedby="CR24">24</a>,<a href="#CR25" class="usa-link" aria-describedby="CR25">25</a></sup>. We performed immunoblot analysis to probe for HSL, phospho-HSL (S563), and HNF4Î± in tumor and NAT, as well as corresponding control mammary tissues, from three well-characterized breast cancer patient-derived xenograft (PDX) models (HCI002, HCI009, HCI010) and a transgenic model of MYC-driven TNBC (MTB-TOM)<sup><a href="#CR26" class="usa-link" aria-describedby="CR26">26</a>,<a href="#CR27" class="usa-link" aria-describedby="CR27">27</a></sup>. In all models analyzed, a downregulation of total HSL in NAT compared to control tissue was observed (Fig.Â <a href="#Fig1" class="usa-link">1g, h</a>). Downregulation of total HSL has been observed in individuals with obesity and in an independent analysis of primary breast tumor NAT, and is thought to be the result of a negative feedback loop in adipocytes in response to chronic lipolysis<sup><a href="#CR24" class="usa-link" aria-describedby="CR24">24</a>,<a href="#CR25" class="usa-link" aria-describedby="CR25">25</a></sup>. Additionally, in 3 of the 4 models examined, we found an increase in HNF4Î± protein or in phospho-HSL/total HSL ratio (Fig.Â <a href="#Fig1" class="usa-link">1g, h</a>), both associated with increased lipolysis<sup><a href="#CR18" class="usa-link" aria-describedby="CR18">18</a>,<a href="#CR23" class="usa-link" aria-describedby="CR23">23</a></sup>. Taken together, our concurrent findings in 3 independent clinical datasets and several models of patient-derived xenograft and transgenic mouse breast cancers indicate that lipolysis is activated, to varying degrees, in breast cancer-adjacent adipose tissue. These findings support the conclusion that â€œnormalâ€ tissue adjacent to tumors is, in fact, not normal<sup><a href="#CR28" class="usa-link" aria-describedby="CR28">28</a></sup>; in the context of breast cancer. Tumor-adjacent adipocytes have markers of activated lipolysis with corresponding diminished lipid stores.</p></section><section id="Sec5"><h3 class="pmc_sec_title">TNBC-adjacent lipolysis and gap junctions</h3>
<p id="Par11">We next sought to determine the contribution of cell-cell contact to lipolysis activation in breast tumor-adjacent adipocytes. Gap junctions are cell-cell junctions formed by a family of proteins called connexins, which are known to transport a variety of small molecules (&lt;1â€‰kD), including cAMP<sup><a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>,<a href="#CR29" class="usa-link" aria-describedby="CR29">29</a></sup>. Connexins were long thought to play tumor-suppressive roles in cancer<sup><a href="#CR30" class="usa-link" aria-describedby="CR30">30</a>â€“<a href="#CR32" class="usa-link" aria-describedby="CR32">32</a></sup>, but recent evidence from a variety of tumor types has challenged this notion<sup><a href="#CR29" class="usa-link" aria-describedby="CR29">29</a>,<a href="#CR33" class="usa-link" aria-describedby="CR33">33</a>â€“<a href="#CR35" class="usa-link" aria-describedby="CR35">35</a></sup>. Given that adipocytes are capable of transferring cAMP and activating lipolysis in a homotypic interaction with other adipocytes<sup><a href="#CR22" class="usa-link" aria-describedby="CR22">22</a></sup>, we hypothesized that gap junctions may also form between tumor cells and adipocytes in a heterotypic fashion to activate lipolysis via transfer of cAMP. Using a well-established dye transfer assay<sup><a href="#CR34" class="usa-link" aria-describedby="CR34">34</a></sup>, we first probed for presence of functional gap junctions between breast cancer cells. Because gap junction function in breast tumors has not been clearly defined, we tested whether the TNBC cell line HCC1143 or the more indolent RP cell line T47D could transfer gap junction-dependent dyes to the same tumor cell line. Both lines formed functional gap junctions, but dye transfer between HCC1143 cells was 30-fold increased (Fig.Â <a href="#Fig2" class="usa-link">2a</a>) compared to transfer amongst T47D cells. Thus, we reasoned there may be differences in sensitivity to gap junction inhibition between TN and RP cells. Furthermore, given the upregulation of the MYC oncogene in the majority of TNBC<sup><a href="#CR36" class="usa-link" aria-describedby="CR36">36</a>,<a href="#CR37" class="usa-link" aria-describedby="CR37">37</a></sup>, we asked whether MYC expression affects gap junction dependence. We examined if gap junction inhibition alters cellular ATP as a proxy for cell abundance in a panel of TN and RP human breast cell lines with varying MYC levels<sup><a href="#CR2" class="usa-link" aria-describedby="CR2">2</a></sup>. Intriguingly, TNBC cell lines with high MYC expression<sup><a href="#CR2" class="usa-link" aria-describedby="CR2">2</a></sup>, including HCC1143, were significantly more sensitive to 24â€‰h of treatment with the pan-gap junction inhibitor carbenoxolone (CBX) than the low MYC TNBC or RP cell lines tested (Fig.Â <a href="#Fig2" class="usa-link">2b</a>). In addition, dye transfer to HCC1143 cells was significantly reduced by 30.63% (<em>p</em>â€‰&lt;â€‰0.0001) following treatment with CBX (Fig.Â <a href="#Fig2" class="usa-link">2c</a>). These data suggest that gap junction communication occurs between breast cancer cells, and that a threshold amount of gap junction activity may be required for MYC-high TN cell viability.</p>
<figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2. Breast cancer cells form functional gap junctions and express Cx31.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12368202_41467_2025_62486_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/0818/12368202/431ff9502e7b/41467_2025_62486_Fig2_HTML.jpg" loading="lazy" id="d33e1274" height="807" width="795" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Relative frequency of dye transfer from Calcein AM-loaded cells (donor) to unloaded mCherry-labelled cells (recipient) as determined by FACS (fluorescence-activated cell sorting) analysis (<em>p</em>â€‰=â€‰0.0050). Each point represents a biological replicate. <strong>b</strong> ATP levels in TN high MYC (red), TN low MYC (orange), and RP (blue) cell lines after treatment with 150â€‰Î¼M CBX for 24â€‰h, relative to untreated (control) cells. Each point represents a biological replicate averaging three technical replicates. <strong>c</strong> Relative frequency of dye transfer from Calcein AM-loaded cells (donor) to unloaded mCherry-labeled cells (recipient) treated with 150â€‰Î¼M CBX or vehicle control for 24â€‰h, as determined by FACS analysis (<em>p</em>â€‰&lt;â€‰0.0001). Each point represents a biological replicate. <strong>d</strong> Fold change (log2) in expression of indicated <em>GJA</em> (navy), <em>GJB</em> (maroon), <em>GJC</em> (purple), and <em>GJD</em> (green) connexin genes in TN (<em>n</em>â€‰=â€‰123) versus RP (<em>n</em>â€‰=â€‰648) tumors based on RNA-seq data acquired from TCGA of 771 breast cancer patients. <strong>e</strong> Fraction of cells in patient tumors of RP (blue, <em>nâ€‰</em>=â€‰6) and TNBC (red, <em>n</em>â€‰=â€‰5) subtypes expressing indicated gap junction (GJ) family members, based on sc-RNA-seq of 317 tumor cells. <strong>f</strong> Fold change (log2) in expression of indicated <em>GJA</em> (navy), <em>GJB</em> (maroon), and <em>GJC</em> (purple) connexin genes in tumor (T, <em>n</em>â€‰=â€‰10) versus non-tumor (NT, <em>n</em>â€‰=â€‰3) tissues based on RNA-seq data from MTB-TOM allograft-bearing mice or healthy controls, respectively. For (<strong>a</strong>â€“<strong>c</strong>) meanâ€‰Â±â€‰s.e.m. of three independent biological replicates is shown. **<em>P</em>â€‰&lt;â€‰0.01, ****<em>P</em>â€‰&lt;â€‰0.0001; unpaired two-tailed <em>t</em> test (<strong>a</strong> and <strong>c</strong>); ordinary one-way ANOVA with multiple comparisons (<strong>b</strong>). For (<strong>d</strong> and <strong>f</strong>), all differential expression analysis was done using the â€œlimmaâ€ R package with a 0.05 adjusted <em>P</em> value. Source data are provided as a Source Data file.</p></figcaption></figure></section><section id="Sec6"><h3 class="pmc_sec_title">TNBC tumors feature elevated connexin 31 (<em>GJB3</em>)</h3>
<p id="Par12">To delineate the role of connexins in TN compared to RP breast cancer, we examined the expression of the 21 connexin genes in 771 primary human breast cancers, TN (<em>n</em>â€‰=â€‰123) and RP (<em>n</em>â€‰=â€‰648), using publicly available RNA-seq data from The Cancer Genome Atlas (TCGA). Of the 20 connexins for which data was available, 5/20 were significantly downregulated, and 11/20 were significantly upregulated. These 11 upregulated connexins included 5 of the 7 gap junction B (GJB) family members (Fig.Â <a href="#Fig2" class="usa-link">2d</a>). To probe gap junction expression at the cellular level, we also examined scRNA-seq (<em>n</em>â€‰=â€‰317) of primary patient tumors (<em>n</em>â€‰=â€‰11)<sup><a href="#CR38" class="usa-link" aria-describedby="CR38">38</a></sup>. Expression of GJBs was observed in a greater fraction (47.2% vs. 29.8%) of TN than RP tumor cells, and GJBs were the most frequently expressed gap junction family for TN, but not for RP tumor cells (Fig.Â <a href="#Fig2" class="usa-link">2e</a> and Supplementary Fig.Â <a href="#MOESM1" class="usa-link">2</a>). As an independent approach to examine in vivo expression of connexins in TNBC, we then performed RNA-seq on MTB-TOM tumors and non-tumor control tissue (Supplementary DataÂ <a href="#MOESM5" class="usa-link">3</a>). Of the 10 connexins for which data were available, 2/10 were significantly downregulated, 4/10 were significantly upregulated, and 4/10 were not significantly changed in MTB-TOM tumors versus control non-tumor tissue (Fig.Â <a href="#Fig2" class="usa-link">2f</a>). Connexin 31 (<em>GJB3</em>, Cx31) was the most significantly elevated connexin in both human TN tumors and the MYC-driven TNBC model. Thus, we focused the remainder of our studies on Cx31. Cx31 has been found to be expressed in keratinocytes, the small intestine, and the colon<sup><a href="#CR39" class="usa-link" aria-describedby="CR39">39</a>,<a href="#CR40" class="usa-link" aria-describedby="CR40">40</a></sup>. Although roles for various connexins as oncogenes and/or tumor suppressors have been described<sup><a href="#CR29" class="usa-link" aria-describedby="CR29">29</a>,<a href="#CR33" class="usa-link" aria-describedby="CR33">33</a></sup>, a pro-tumorigenic function of Cx31 has not been previously established.</p></section><section id="Sec7"><h3 class="pmc_sec_title">TNBC and mammary adipocytes express Cx31</h3>
<p id="Par13">Accordingly, we sought to determine if functional Cx31-containing gap junctions form between breast cancer cells and adipocytes. To validate the presence of cancer-adipocyte gap junctions in TNBC, we began by examining primary patient biopsies for expression of Cx31 and of pan-cytokeratin to distinguish tumor cells. We found that both TN tumor cells and adipocytes robustly express Cx31 at the plasma membrane. Further, we found many points of cell-cell contact occurred in vivo between tumors and adipocytes (Fig.Â <a href="#Fig3" class="usa-link">3a</a>). To model the cell-cell contact observed in vivo between breast cancer cells and adipocytes, we developed three independent co-culture models. First, we performed 3-dimensional ex vivo studies by co-culturing breast cancer cells directly within primary patient breast fat (Fig.Â <a href="#Fig3" class="usa-link">3b</a>). We stably transduced HCC1143 (TNBC) and T47D (RP) with an mCherry expression plasmid, then introduced either mCherry-HCC1143 or -T47D cells directly into mammary adipose tissue (PT001) and co-cultured overnight. Tumor cell-adipocyte co-cultures were formalin-fixed, paraffin-embedded, and probed for Cx31 and pan-cytokeratin expression, then imaged using immunofluorescent microscopy. We found that both HCC1143 cells and adipocytes robustly expressed Cx31 at the plasma membrane; HCC1143 formed close cell-cell contacts with primary adipocytes (Fig.Â <a href="#Fig3" class="usa-link">3b</a>, top). In contrast, while T47D cells formed cancer cell-cancer cell contacts, we did not observe close cancer cell-adipocyte contacts (Fig.Â <a href="#Fig3" class="usa-link">3b</a>, bottom). These data suggest that Cx31 can be expressed at both the tumor cell and adipocyte plasma membrane, and that breast cancer cells can form close cell-cell contacts with adipocytes.</p>
<figure class="fig xbox font-sm" id="Fig3"><h4 class="obj_head">Fig. 3. Breast cancer cell-adipocyte gap junctions form, transfer cAMP, and activate lipolytic signaling dependent on Cx31 expression.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12368202_41467_2025_62486_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/0818/12368202/631e9eb401b6/41467_2025_62486_Fig3_HTML.jpg" loading="lazy" id="d33e1476" height="994" width="754" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Staining with Cx31 (green) and pan-cytokeratin (magenta) of primary TNBC patient biopsies. Scale bar, top 100â€‰Î¼m, bottom 25â€‰Î¼m. <strong>b</strong> Staining with Cx31 (green) and pan-cytokeratin (magenta) of primary mammary adipose tissue from a healthy individual (PT001) injected with TN mCherry-HCC1143 cells (top) or RP mCherry-T47D cells (bottom) and co-cultured overnight. White arrowheads indicate staining of Cx31 along point of contact between HCC1143 and adipocyte plasma membranes. Scale bar, 25â€‰Î¼m. <strong>c</strong> Immunoblot analysis showing protein expression levels of Cx31 in vitro in (<strong>a</strong>) panel of clonally derived control <em>GJB3</em><sup>WT</sup> and partial depletion TN lines with one-third and two- thirds loss of <em>GJB3</em> expression. For the Cx31-depleted lines each clone is referred to by level of <em>GJB3</em> expression (e.g., <em>GJB3</em><sup>Med</sup> expresses two-thirds WT level, and <em>GJB3</em><sup>Low</sup> expresses one third <em>GJB3</em><sup>WT</sup> level). Quantification of displayed Cx31 level normalized to b-actin level is indicated. <strong>d</strong> Staining with Cx31 (magenta), pHSL(S563) (yellow), and phalloidin (blue), of healthy patient primary mammary tissue (PT002) injected with GFP-expressing HCC1143 <em>GJB3</em><sup>WT</sup> (top), HCC1143-<em>GJB3</em><sup>Low</sup> (middle), or T47D cells (bottom) and co-cultured overnight. White arrowheads indicate Cx31 staining at GFP cancer cell-adipocyte interface. Scale bar, 20â€‰Î¼m. <strong>e</strong> Dye transfer from indicated HCC1143 control and Cx31-depleted lines to primary mammary adipose tissue of indicated (<em>n</em>â€‰=â€‰3) healthy individuals. <strong>f</strong> cAMP levels in TN high MYC (red), TN low MYC (orange, <em>p</em>â€‰=â€‰0.0487), and RP (blue, <em>p</em>â€‰=â€‰0.487) cell lines after treatment with 150â€‰Î¼M CBX for 24â€‰h, relative to untreated (control) cells. Each point represents a biological replicate averaging three technical replicates. <strong>g</strong> cAMP transfer from indicated HCC1143 control and Cx31 partial expression loss lines to primary mammary adipose tissue of indicated (<em>n</em>â€‰=â€‰3) healthy individuals. <strong>h</strong> Fold change in <em>UCP1</em> (red) and <em>FABP4</em> (blue) expression in differentiated adipocytes after treatment with vehicle (control) or 10â€‰Î¼M forskolin, or co-cultured with indicated Cx31 partial expression loss lines for 24â€‰h. Representative results from experiments done in biological triplicates shown for (<strong>a</strong>â€“<strong>d</strong>). For (<strong>f</strong> and <strong>h</strong>) meanâ€‰Â±â€‰s.e.m. of three independent biological replicates is shown. ^<em>P</em>â€‰&lt;â€‰0.10, *<em>P</em>â€‰&lt;â€‰0.05, **<em>P</em>â€‰&lt;â€‰0.01, ***<em>P</em>â€‰&lt;â€‰0.001; repeated measures one-way ANOVA with multiple comparisons for (<strong>e</strong> and <strong>g</strong>), ordinary one-way ANOVA with multipl<strong>e</strong> comparisons for (<strong>f</strong> and <strong>h</strong>). Source data are provided as a Source Data file.</p></figcaption></figure><p id="Par14">To determine the role of Cx31 in TNBC-adipocyte interactions, we used CRISPR/Cas9 to generate a series of <em>GJB3</em>-depleted TN cell lines (HS578T and HCC1143). In TN MYC-high TN cell line HCC1143, we generated two clones, with ~1/3 and ~2/3 <em>GJB3</em> expression loss (HCC1143 <em>GJB3</em><sup>Med</sup> and <em>GJB3</em><sup>Low</sup>). In TN MYC-low line HS578T, we generated two distinct clones with ~1/3 <em>GJB3</em> expression loss (HS578T <em>GJB3</em><sup>Med-1</sup> and <em>GJB3</em><sup>Med-2</sup>) (Fig.Â <a href="#Fig3" class="usa-link">3c</a>). Despite several attempts, we were unable to generate TN cell lines with complete Cx31 loss, suggesting that a basal level of Cx31 expression is required for TN cancer cell growth.</p></section><section id="Sec8"><h3 class="pmc_sec_title">Cx31 depletion impacts TNBC-adipocyte cell contact</h3>
<p id="Par15">To examine how Cx31 expression impacted cancer cell-adipocyte contact, we performed ex vivo co-cultures with primary patient breast fat using the partially depleted Cx31 cell lines. We stably transduced TN HCC1143 <em>GJB3</em><sup>WT</sup> and <em>GJB3</em><sup>Low</sup> cell lines, as well as RP line T47D, with a GFP expression plasmid, then cultured each line directly within primary mammary adipose tissues from healthy individuals (PT002, PT003). After overnight incubation, co-cultured tissues were formalin-fixed and probed for expression of Cx31 and lipolysis marker pHSL(S563)<sup><a href="#CR18" class="usa-link" aria-describedby="CR18">18</a></sup>. Tissues were then cleared<sup><a href="#CR41" class="usa-link" aria-describedby="CR41">41</a></sup> and imaged via whole mount fluorescence microscopy. We found that HCC1143 <em>GJB3</em><sup>WT</sup> cells formed extended cancer cell-adipocyte contacts, in tight conformation with adjacent adipocytes (Fig.Â <a href="#Fig3" class="usa-link">3d</a> and Supplementary Fig.Â <a href="#MOESM1" class="usa-link">3a</a>, top). GJB3 aggregates forming puncta at the cancer cell-adipocyte interface were observed. In contrast, Cx31-depleted HCC1143 <em>GJB3</em><sup>Low</sup> cells formed tangential contacts with adjacent adipocytes (Fig.Â <a href="#Fig3" class="usa-link">3d</a> and Supplementary Fig.Â <a href="#MOESM1" class="usa-link">3a</a>, middle), which we note closely mimic the tangential cancer cell-adipocyte conformation observed in T47D (RP) co-cultures (Fig.Â <a href="#Fig3" class="usa-link">3d</a> and Supplementary Fig.Â <a href="#MOESM1" class="usa-link">3a</a>, bottom). Both HCC1143 <em>GJB3</em><sup>Low</sup> (TN) and T47D (RP) co-cultures lacked GJB3 puncta at the cancer cell-adipocyte interface. In mock co-cultures, our positive control forskolin, which raises intracellular cAMP levels by activating adenylyl cyclase<sup><a href="#CR19" class="usa-link" aria-describedby="CR19">19</a></sup>, robustly induced pHSL(S563) expression and increased puncta compared to vehicle-treated mammary adipose tissue (Supplementary Fig.Â <a href="#MOESM1" class="usa-link">3bâ€“d</a>). We observed greater pHSL(S563) expression and elevated puncta in adipose tissue co-cultured with HCC1143 <em>GJB3</em><sup>WT</sup> cells than tissues with HCC1143 <em>GJB3</em><sup>Low</sup> or T47D cells (Supplementary Fig.Â <a href="#MOESM1" class="usa-link">3e, f</a>), indicating more cAMP-dependent PKA activity. These results suggest that Cx31 level in breast cancer can mediate cell contact with surrounding adipocytes and alter lipolytic signaling.</p></section><section id="Sec9"><h3 class="pmc_sec_title">TNBC cells form functional GJB3 gap junctions with adipocytes</h3>
<p id="Par16">We next sought to determine if Cx31 expression impacted tumor cell-adipocyte communication using a co-culture model in which HCC1143 <em>GJB3</em><sup>WT</sup>, <em>GJB3</em><sup>Med</sup>, or <em>GJB3</em><sup>Low</sup> cells were seeded in 2D culture and loaded with gap junction-transferable dye. We added primary mammary adipose tissue from three healthy individuals (PT004, PT005, PT006) directly on top of the monolayers to permit direct contact. Tumor cells and adipocytes were co-cultured for 5â€‰h and then assayed for dye transfer from the cancer cells to adipocytes. We found that robust dye transfer occurred from the HCC1143 <em>GJB3</em><sup>WT</sup> cells to mammary adipocytes from all three patients (Fig.Â <a href="#Fig3" class="usa-link">3e</a>). However, depletion of Cx31 expression by 1/3 or 2/3 in the <em>GJB3</em><sup>Med</sup> and <em>GJB3</em><sup>Low</sup> lines, respectively, resulted in a significant decrease in dye transfer compared to <em>GJB3</em><sup>WT</sup> control cells (Fig.Â <a href="#Fig3" class="usa-link">3e</a>). These data suggest that functional gap junctions form between TN breast cancer cells and adipocytes and can be diminished by Cx31 depletion.</p></section><section id="Sec10"><h3 class="pmc_sec_title">Breast cancer cell gap junctions are permeable to cAMP</h3>
<p id="Par17">To determine if breast cancer cell gap junctions are permeable to cAMP, we treated a panel of human TN and RP cell lines with CBX for 24â€‰h to inhibit pan-gap junction function and ascertain if cAMP was retained in the tumor cells. In 5 of 6 lines tested, we found marked increases in the levels of intracellular cAMP concentration in CBX- versus vehicle-treated cells (Fig.Â <a href="#Fig3" class="usa-link">3f</a>). Additionally, significantly higher concentrations of cAMP were observed in high MYC TN cells in comparison to low MYC TN or RP cells (Fig.Â <a href="#Fig3" class="usa-link">3f</a>). The increase in intracellular cAMP following pan-gap junction inhibition in 5 of 6 lines examined suggests that breast cancer cell gap junctions are indeed permeable to cAMP.</p></section><section id="Sec11"><h3 class="pmc_sec_title">GJB3 gap junctions transduce cAMP from TNBC to adipocytes</h3>
<p id="Par18">We next tested whether cAMP is directly transferred from breast cancer cells to adipocytes and if the abundance of Cx31 alters transfer. HCC1143 <em>GJB3</em><sup>WT</sup>, <em>GJB3</em><sup>Med</sup>, or <em>GJB3</em><sup>Low</sup> cells were seeded and loaded with a fluorescent cAMP analogue (fluo-cAMP). These monolayer cultures were then co-cultured in direct contact with primary mammary adipose tissue from three healthy individuals (PT007, PT008, PT009) and incubated for 5â€‰h. Adipocytes were then isolated from the tumor cells and assayed for fluo-cAMP. We found that cAMP transfer occurred from control <em>GJB3</em><sup>WT</sup> cells to adipocytes from all three patients (Fig.Â <a href="#Fig3" class="usa-link">3g</a>). However, as we observed with transfer of gap junction-permeable dye (Fig.Â <a href="#Fig3" class="usa-link">3e</a>), depletion of Cx31 resulted in a significant reduction of cAMP transfer (Fig.Â <a href="#Fig3" class="usa-link">3g</a>). Thus, cAMP is transferred from TN breast cancer cells to adipocytes and is diminished following Cx31 depletion.</p></section><section id="Sec12"><h3 class="pmc_sec_title">TNBC-adjacent adipocytes exhibit activated cAMP signaling</h3>
<p id="Par19">We next sought to determine if downstream cAMP signaling is activated in adipocytes in a gap junction-dependent manner. To determine if cAMP signaling is activated in adipocytes upon cell-cell contact with breast cancer cells, we used a primary mouse preadipocyte model that can be differentiated to adipocytes in vitro<sup><a href="#CR19" class="usa-link" aria-describedby="CR19">19</a>,<a href="#CR42" class="usa-link" aria-describedby="CR42">42</a></sup>. This model is ideal to study downstream signaling during co-culture because changes in adipocyte transcription can be assayed via qRT-PCR using murine-specific primers. Adipocytes were terminally differentiated and then HS578T and HCC1143 <em>GJB3</em> partial depletion cell lines were seeded directly on top of adipocyte cultures. After co-culturing the cells for 24â€‰h, we extracted RNA and assayed for changes in murine-specific (thus adipocyte-specific in this system) expression of <em>UCP1</em>, a known cAMP-responsive gene in adipocytes<sup><a href="#CR19" class="usa-link" aria-describedby="CR19">19</a></sup>, to measure cAMP signaling. We also assayed for mouse <em>FABP4</em> expression as a marker of adipocyte differentiation. Our positive control, forskolin, robustly induced <em>UCP1</em> expression compared to vehicle-treated cells (Fig.Â <a href="#Fig3" class="usa-link">3h</a>). Co-culturing with HCC1143 <em>GJB3</em><sup>WT</sup> and <em>GJB3</em><sup>Med</sup> lines both induced adipocyte UCP1 expression, but <em>UCP1</em> induction was significantly reduced in the <em>GJB3</em><sup>Low</sup> co-cultures (Fig.Â <a href="#Fig3" class="usa-link">3h</a>). In contrast, none of the MYC low HS578T lines, including the <em>GJB3</em><sup>WT</sup> control, were capable of inducing adipocyteÂ <em>UCP1</em> expression (Fig.Â <a href="#Fig3" class="usa-link">3h</a>). All conditions, including forskolin treatment, resulted in reduced <em>FABP4</em> expression (Fig.Â <a href="#Fig3" class="usa-link">3h</a>), suggesting effects on adipocyte differentiation are distinct from those observed on cAMP signaling. Given that Cx31 expression is similar in HS578T <em>GJB3</em><sup>WT</sup> and HCC1143 <em>GJB3</em><sup>Low</sup> cells (Fig.Â <a href="#Fig3" class="usa-link">3c</a>), and that neither activate cAMP signaling (Fig.Â <a href="#Fig3" class="usa-link">3h</a>), it is possible that a Cx31 expression threshold is required for breast cancer cells to activate cAMP signaling in adjacent adipocytes. Although direct transfer of cAMP amongst adipocytes via a homotypic gap junction interaction has been described<sup><a href="#CR22" class="usa-link" aria-describedby="CR22">22</a></sup>, gap junction-dependent activation of adipocyte lipolysis in a heterotypic manner by a tumor cell has not been previously demonstrated.</p></section><section id="Sec13"><h3 class="pmc_sec_title">Cx31 gap junctions promote breast tumorigenesis in vivo</h3>
<p id="Par20">Finally, we sought to determine the contribution Cx31 gap junction expression to tumorigenesis. We found that HS578T <em>GJB3</em><sup>Med-1</sup> and <em>GJB3</em><sup>Med-2</sup>, and HCC1143 <em>GJB3</em><sup>Med</sup> cell lines did not display a difference in proliferation compared to their respective <em>GJB3</em><sup>WT</sup> control lines (Fig.Â <a href="#Fig4" class="usa-link">4a</a>). In contrast, HCC1143 <em>GJB3</em><sup>Low</sup> cells demonstrate a significant reduction in proliferation, while maintaining 93.7% viability relative to Cas9 controls (Fig.Â <a href="#Fig4" class="usa-link">4a</a>). These data suggest that, even in the absence of breast cancer cell-adipocyte interaction, Cx31 promotes breast cancer cell proliferation. To determine the contribution of Cx31 to breast tumorigenesis in vivo, we transplanted each of the HS578T and HCC1143 Cx31 partial depletion lines into the mammary fat pad of immune-compromised NOD-SCID/gamma (NSG) female mice and assayed for time of tumor onset and ethical endpoint (when theÂ tumor reaches 2â€‰cm in any dimension). Remarkably, the HS578T lines, in which partial <em>GJB3</em> knockout had no effect on cell proliferation ex vivo (Fig.Â <a href="#Fig4" class="usa-link">4a</a>), 0/10 mice that received HS578T <em>GJB3</em><sup>Med-1</sup> or <em>GJB3</em><sup>Med-2</sup> xenografts (5 per line) developed tumors within 180 days (Fig.Â <a href="#Fig4" class="usa-link">4b</a>). Among the HCC1143 lines, the <em>GJB3</em><sup>Med</sup> line displayed a significant delay in both tumor onset and time to ethical endpoint, while only 3 of 5 mice transplanted with the <em>GJB3</em><sup><em>Low</em></sup> line developed tumors and none reached ethical endpoint within 180 days (Fig.Â <a href="#Fig4" class="usa-link">4b</a>). We performed an independent xenograft model wherein inducible Cx31 hairpins were transduced into the TN-MYC<sup>High</sup> BT549 human breast cell line and found that Cx31 depletion significantly enhanced tumor-free survival compared to controls (Fig.Â <a href="#Fig4" class="usa-link">4c</a>). Our data indicate that decreasing Cx31 expression is sufficient to impair tumor growth, suggesting that Cx31 gap junctions promote breast tumorigenesis in vivo.</p>
<figure class="fig xbox font-sm" id="Fig4"><h4 class="obj_head">Fig. 4. Cx31 loss impairs breast cancer cell growth in vitro, tumorigenesis, and activation of lipolysis in adjacent adipocytes in vivo.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12368202_41467_2025_62486_Fig4_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/0818/12368202/38c01055c225/41467_2025_62486_Fig4_HTML.jpg" loading="lazy" id="d33e1994" height="988" width="741" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Cell growth of indicated Cx31 partial depletion cell lines in HCC1143 (left) and HS578T (middle) over 72â€‰h (<em>n</em>â€‰=â€‰3 biological replicates), and cell viability at 72â€‰h of indicated lines normalized to WT control (right, <em>n</em>â€‰=â€‰3 biological replicates). <strong>b</strong> Kaplanâ€“Meier analysis of tumor onset (top) and ethical endpoint survival (bottom) of mice bearing indicated Cx31 partial expression loss orthotopic xenografts (<em>n</em>â€‰=â€‰5 per group). <strong>c</strong> Kaplanâ€“Meier analysis of tumor onset in mice bearing indicated orthotopic xenografts with inducible Cx31 (shCx31) or GFP (shGFP) hairpin, with doxycycline (solid line, shGFP <em>n</em>â€‰=â€‰7, shCx31 <em>n</em>â€‰=â€‰5 mice) and without doxycycline (broken line, shGFP <em>n</em>â€‰=â€‰13, shCx31 <em>n</em>â€‰=â€‰5 mice). <strong>d</strong> Immunoblot analysis (left) showing expression levels of HSL and phosphorylated HSL (pHSL S563) in healthy non-tumor mammary gland (gray, <em>n</em>â€‰=â€‰2 mice) and NAT from mice bearing indicated <em>GJB3</em> WT (black), Med (blue) or Low (red) xenografts (<em>n</em>â€‰=â€‰3) or mice that were transplanted, but did not develop a tumor (yellow, <em>n</em>â€‰=â€‰2). Quantification of displayed total HSL(middle) and pHSL/HSL ratio (right), normalized to b-actin levels. Biological replicates from distinct mice are indicated. The samples derive from the same experiment, but different gels for HSL and Actin, and another for pHSL, were processed in parallel. <strong>e</strong> Fold change in cAMP levels in HCC1143 <em>GJB3</em><sup>Med</sup> (<em>n</em>â€‰=â€‰5) xenografts versus HCC1143 <em>GJB3</em><sup>WT</sup> (<em>n</em>â€‰=â€‰4) xenografts (<em>p</em>â€‰=â€‰0.0492). <strong>f</strong> Adipocyte area adjacent to HCC1143 <em>GJB3</em> Med xenografts (pooled from <em>n</em>â€‰=â€‰5 tumors, <em>n</em>â€‰=â€‰517 adipocytes) and HCC1143 <em>GJB3</em> WT xenografts (pooled <em>from</em>
<em>n</em>â€‰=â€‰4, <em>n</em>â€‰=â€‰771 adipocytes) and area in control non-tumor (NT) tissue (pooled <em>n</em>â€‰=â€‰3 mice, <em>n</em>â€‰=â€‰2611 adipocytes). Broken line indicates mean adipocyte area; dotted lines indicate quartiles. Each point represents an individual adipocyte. <strong>g</strong> Kaplanâ€“Meier analysis of tumor onset of mice bearing HCC1143 <em>GJB3</em><sup>WT</sup> (black) or <em>GJB3</em><sup>Med</sup> (blue) orthotopic xenografts (<em>n</em>â€‰=â€‰5 per group) and treated with vehicle (solid line) or with 1â€‰mg/kg <a href="https://www.ncbi.nlm.nih.gov/nuccore/CL316243" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">CL316243</a> (broken line). For (<strong>b</strong> and <strong>c</strong>), ethical endpoint survival indicates the percentage of mice bearing xenografts &lt;2â€‰cm in any dimension. For (<strong>a</strong>, <strong>d</strong>, and <strong>e</strong>) meanâ€‰Â±â€‰s.e.m. is shown. *<em>P</em>â€‰&lt;â€‰0.05, **<em>P</em>â€‰&lt;â€‰0.01, ***<em>P</em>â€‰&lt;â€‰0.001, ****<em>P</em>â€‰&lt;â€‰0.0001; unpaired two-tailed <em>t</em> test (<strong>a</strong>) (left and center) and (<strong>e</strong>), log-rank test (<strong>b</strong>, <strong>c</strong>, and <strong>g</strong>), ordinary one-way ANOVA with multiple comparisons (<strong>a</strong>) (right), (<strong>d</strong> and <strong>f</strong>). Source data are provided as a Source Data file.</p></figcaption></figure></section><section id="Sec14"><h3 class="pmc_sec_title">NAT of GJB3-depleted tumors exhibit decreased lipolysis</h3>
<p id="Par21">We sought to clarify the effects of Cx31 on lipolysis versus other effects on tumor growth. To determine if control and Cx31 partially depleted tumors differentially induced lipolysis, we collected tumor and NAT from HCC1143 <em>GJB3</em><sup>WT</sup>, <em>GJB3</em><sup>Med</sup>, and <em>GJB3</em><sup>Low</sup> tumor-bearing mice, as well as residual mammary glands from the two <em>GJB3</em><sup><em>Low</em></sup> mice that were transplanted but never developed tumors. Using immunoblot analysis, we probed for markers of lipolysis. Notably, a marked reduction in total HSL expression was found in 3 of 3 HCC1143 <em>GJB3</em><sup>WT</sup> NAT samples compared to control tissues (Fig.Â <a href="#Fig4" class="usa-link">4d</a>), consistent with persistent activation of lipolysis leading to HSL downregulation<sup><a href="#CR24" class="usa-link" aria-describedby="CR24">24</a>,<a href="#CR25" class="usa-link" aria-describedby="CR25">25</a></sup>. In contrast, we did not observe a consistent change in HSL expression in any of the other NAT samples analyzed from tumors with partial Cx31 expression loss (Fig.Â <a href="#Fig4" class="usa-link">4d</a>). Interestingly, we found a marked increase in phospho-HSL/HSL ratio in both the HCC1143 <em>GJB3</em><sup>WT</sup> and <em>GJB3</em><sup>Med</sup> NAT samples, but this difference was significantly reduced in HCC1143 <em>GJB3</em><sup>Low</sup> NAT (Fig.Â <a href="#Fig4" class="usa-link">4D</a>). The increase in phospho-HSL/HSL in <em>GJB3</em><sup>Med</sup> NAT may be due to alternative modes of lipolysis activation, such as secreted pro-lipolytic cytokines<sup><a href="#CR5" class="usa-link" aria-describedby="CR5">5</a></sup>, which is congruent with the observed increase in <em>UCP1</em> expression during <em>GJB3</em><sup>Med</sup>-adipocyte co-culture (Fig.Â <a href="#Fig3" class="usa-link">3h</a>). To further interrogate lipolytic signaling in NAT, we probed for cAMP abundance in HCC1143 <em>GJB3</em><sup>WT</sup> and <em>GJB3</em><sup>Med</sup> tumors by mass spectrometry. We found a significant increase in intratumoral cAMP level in HCC1143 <em>GJB3</em><sup>Med</sup> tumors compared to the <em>GJB3</em><sup>WT</sup> control tumors (Fig.Â <a href="#Fig4" class="usa-link">4e</a>), consistent with diminished transfer of cAMP to NAT. We examined <em>GJB3</em><sup>WT</sup> and <em>GJB3</em><sup>Med</sup> tumors and associated NAT, and assayed for differences in adjacent adipocyte size, as an indicator of lipolysis. We found a significant increase in the average size of adipocytes adjacent to <em>GJB3</em><sup><em>Med</em></sup> tumors compared to <em>GJB3</em><sup>WT</sup> control tumors (Fig.Â <a href="#Fig4" class="usa-link">4f</a>), again supporting a decreased induction of lipolysis in NAT from Cx31 partial knockout tumors.</p></section><section id="Sec15"><h3 class="pmc_sec_title">Activation of lipolysis rescues GJB3-depleted tumor growth</h3>
<p id="Par22">Finally, if the delay in HCC1143 <em>GJB3</em><sup>Med</sup> tumor onset (Fig.Â <a href="#Fig4" class="usa-link">4b</a>) was due to an inability to activate lipolysis in adjacent adipocytes, we reasoned that pharmacological activation of lipolysis should rescue this phenotype. Indeed, we found that daily intra-peritoneal injection of <a href="https://www.ncbi.nlm.nih.gov/nuccore/CL316243" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">CL316243</a>, a specific Î²3-receptor agonist known to activate lipolysis in vivo<sup><a href="#CR43" class="usa-link" aria-describedby="CR43">43</a></sup>, rescued the delay in tumor onset observed in HCC1143 <em>GJB3</em><sup>Med</sup> tumors, but did not further promote the growth of HCC1143 <em>GJB3</em><sup>WT</sup> tumors (Fig.Â <a href="#Fig4" class="usa-link">4g</a>). Taken together, these data indicate that cAMP signaling and lipolysis are activated in breast tumor-adjacent adipocytes, and that the abundance of Cx31 expression alters these phenotypes in vivo.</p></section></section><section id="Sec16"><h2 class="pmc_sec_title">Discussion</h2>
<p id="Par23">Here we find that lipolysis is activated in breast cancer-adjacent adipose tissue, and that functional gap junctions form between breast cancer cells and adipocytes. cAMP can be transferred via breast cancer and activate lipolysis in adjacent adipocytes. Higher Cx31 expression is associated with increased cAMP transfer and induction of lipolysis, as well as more aggressive tumor growth. We established a previously unappreciated, functional role for GJB3 gap junctions in activating lipolysis in tumor-adjacent adipose tissue and promoting breast tumor growth in vivo. Breast cancer-associated gap junctions represent a largely unexplored therapeutic target to treat breast tumors. The recent discovery of gap junction formation and pro-tumorigenic signal exchange between brain metastatic carcinoma cells and astrocytes<sup><a href="#CR34" class="usa-link" aria-describedby="CR34">34</a></sup> suggests that gap junction-dependent interactions between tumor and non-tumor cells may be an emerging hallmark of tumorigenesis.</p>
<p id="Par24">In the breast microenvironment, a diversity of pro-tumorigenic interactions occur between cancer cells and adipocytes, including the transfer of lipolytic factors to adipocytes by tumor cells. Most previously established mechanisms of lipolysis induction rely on remote signaling to adipocytes<sup><a href="#CR44" class="usa-link" aria-describedby="CR44">44</a>,<a href="#CR45" class="usa-link" aria-describedby="CR45">45</a></sup>, either through circulatory endocrine factors that activate Î²-adrenergic receptors on the adipocyte membrane, or through paracrine cytokines, chemokines, and growth factors secreted from nearby adipocytes. Models recapitulating tumor-adipocyte interactions often do not consider contact-mediated signaling, and thus the contributions of direct, juxtacrine tumor-adipocyte signaling to tumorigenesis.</p>
<p id="Par25">In this study, we observe direct transfer of lipolytic cAMP signaling and contributions to TNBC tumorigenesis by GJB3 gap junctions, a potential link between dependance on fatty acid oxidation observed in TNBC<sup><a href="#CR2" class="usa-link" aria-describedby="CR2">2</a></sup> and the adipose-rich breast tumor niche. The present data do not, however, exclude transfer of other factors by these GJB3 gap junctions, or transfer of cAMP to adipocytes by other kinds of gap junctions. The present data also do not indicate that intratumor cAMP levels are greater in TN compared to RP tumors, but that elevated Cx31 in TNBC permits increased transfer of cAMP signal from tumor to adipose tumor microenvironment and increased lipolysis in tumor-adjacent adipocytes. Fully isolating the contribution of Cx31 to this phenotype presents a technical challenge. The inability to generate fully Cx31-null patient-derived TNBC cell lines suggests that some degree of expression is required to maintain tumor cell proliferation. The prolonged tumor-free survival and time to ethical endpoint observed for mice bearing <em>GJB3</em>-depleted TN-MYC<sup>Low</sup> xenografts, which in vitro were less sensitive to gap junction inhibition than TN-MYC<sup>High</sup> cell lines, imply an additional tumor-intrinsic role for Cx31 as well. Several <em>GJB</em> family members are differentially increased when comparing transcript levels in patient TN to RP tumors, suggesting multiple gap junction proteins may be relevant to tumor growth in vivo.</p>
<p id="Par26">Further exploration of juxtacrine signaling, in particular GJ-mediated intercellular communication, may be critical in understanding the tropism of metastatic cancers to adipose-rich tissue depots throughout the body. Aggressive ovarian and gastric cancers share tropism for the omentum, a major adipose tissue depot, and also feature pro-metastatic roles for tumor-adjacent fat<sup><a href="#CR7" class="usa-link" aria-describedby="CR7">7</a>,<a href="#CR46" class="usa-link" aria-describedby="CR46">46</a></sup>. Bone marrow adipose tissue provides a dormancy niche for breast cancer metastases<sup><a href="#CR47" class="usa-link" aria-describedby="CR47">47</a></sup>, possibly providing metabolic fuel in addition to signals that support quiescence<sup><a href="#CR48" class="usa-link" aria-describedby="CR48">48</a></sup>. Individuals with obesity often face increased risk: in primary breast, as well as ovarian, and gastric cancers, excess weight is anÂ established risk factor<sup><a href="#CR7" class="usa-link" aria-describedby="CR7">7</a>,<a href="#CR49" class="usa-link" aria-describedby="CR49">49</a></sup>. It is possible that direct tumor-adipocyte signaling contributes to these phenotypes. Given evidence that TNBC both oxidize fatty acids for growth<sup><a href="#CR2" class="usa-link" aria-describedby="CR2">2</a></sup> and directly transfer lipolytic cAMP to the tumor niche, systemic lipolysisâ€”whether naturally occurring or induced, as in the case of GLP-1 agonist-mediated weight loss<sup><a href="#CR50" class="usa-link" aria-describedby="CR50">50</a>,<a href="#CR51" class="usa-link" aria-describedby="CR51">51</a></sup>â€”could supplant juxtracrine lipolytic signaling to increase availability of free fatty acids (FFA). In mice bearing <em>GJB3</em><sup>Med</sup> TNBC xenografts, we used a known lipolysis-inducing agent <a href="https://www.ncbi.nlm.nih.gov/nuccore/CL316243" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">CL316243</a>, which had previously been validated in C57BL/6 male mice to increase circulating FFA. While we did not directly assess its impact on circulating FFA levels in our model, we found <a href="https://www.ncbi.nlm.nih.gov/nuccore/CL316243" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">CL316243</a> to increase tumorigenesis and note that lipolysis-inducing agents could feed tumorigenesis in MYC-driven breast cancers with low GJB3. Additional studies with an anti-lipolytic agent are warranted to assess whether systemic or localized decreases of FFA levels may inhibit growth of TNBC.</p></section><section id="Sec17"><h2 class="pmc_sec_title">Methods</h2>
<section id="Sec18"><h3 class="pmc_sec_title">Ethics declaration</h3>
<p id="Par27">All animal care and rodent experiments comply with the University of California, San Franciscoâ€™s regulations for controlled substance usage, for biological agent usage, and for animal studies (Institutional Animal Care and Use Committee, Protocol AN200579-00I). All human specimen usage was reviewed and approved by the respective institutional review boards and informed consent was received from participants. The prospective diagnostic 3CB imaging clinical study, initially published in Drukker et al.<sup><a href="#CR17" class="usa-link" aria-describedby="CR17">17</a></sup>, was approved by the respective institutional review boards at the University of California, San Francisco (UCSF) and at the Moffitt Cancer Center, Tampa, Florida, and followed Health Insurance Portability and Accountability Act-compliant protocols. All study participants provided written informed consent and received compensation for any imaging conducted in addition to the standard of care they were receiving. Invasive tumor samples from participants (Fig.Â <a href="#Fig1" class="usa-link">1aâ€“d</a>Â andÂ Supplementary Fig. <a href="#MOESM1" class="usa-link">1a-c</a>) were collected as part of this study with patient informed consent, without additional compensation. For normal and invasive tumor samples used for microdissection and mass-spec analysis (Fig.Â <a href="#Fig1" class="usa-link">1f</a>), all patients provided informed consent and were not reimbursed for participation. For invasive breast cancer, IHC staining (Fig.Â <a href="#Fig3" class="usa-link">3a</a>), all patients provided informed consent and were not reimbursed for participation. Normal reduction mammoplasty samples used for human adipocyte-tumor co-culture (Fig.Â <a href="#Fig3" class="usa-link">3b, e, g</a>Â andÂ Supplementary Fig. <a href="#MOESM1" class="usa-link">3a-f</a>) were received from the Cooperative Human Tissue Network (CHTN) with the approval of the IRB of UCSF. Tissues were received as deidentified samples, and all donors provided written informed consent. As the CHTN collects remnant samples the patients were not reimbursed for their participation.</p></section><section id="Sec19"><h3 class="pmc_sec_title">3CB patient population</h3>
<p id="Par28">Five hundred women with suspicious mammography findings (BIRADS 4 or greater) were recruited and imaged before their biopsies using a 3-compartment decomposition dual-energy mammography protocol (3CB). This was multicenter study with two recruitment sites: University of California, San Francisco, CaliforniaÂ and Moffitt Cancer Center, Tampa, Florida. All patients received a biopsy of the suspicious area, and breast biopsies were clinically reviewed by the pathologists. A subset of pathology-proven triple-negative (<em>n</em>â€‰=â€‰6) and receptor-positive (<em>n</em>â€‰=â€‰40) invasive cancers were selected for this study. All women received both cranio-caudal (CC) and mediolateral-oblique (MLO) views. Exclusion criteria for the study were no prior cancer, biopsies, or breast ipsilateral alterations, and no occult findings.</p></section><section id="Sec20"><h3 class="pmc_sec_title">3CB imaging protocol</h3>
<p id="Par29">The 3CB method combines the dual-energy X-ray mammography attenuations and breast thickness map to solve for the three unknowns water, lipid, and protein content<sup><a href="#CR16" class="usa-link" aria-describedby="CR16">16</a></sup>. The Hologic Selenia full-field digital mammography system (Hologic, Inc.) was used to image women with 3CB. Two dual-energy mammograms were acquired on each womanâ€™s affected breast using a single compression. The first exposure was made under conditions of regular clinical screening mammogram. The second mammogram was acquired at a fixed voltage (39â€‰kVp) and mAs for all participants. A high-energy exposure (39â€‰kVp/Rh filter) was made using an additional 3-mm plate of aluminum in the beam to increase the average energy of the high-energy image. We limited the total dose of this procedure to be approximately 110% of the mean-glandular dose of an average screening mammogram. The images were collected under an investigational review board approval to measure breast composition. The breast thickness map was modeled using the SXA phantom<sup><a href="#CR52" class="usa-link" aria-describedby="CR52">52</a></sup>. The thickness validation procedure concluded in a weekly scanning of specially designed quality assurance phantom<sup><a href="#CR53" class="usa-link" aria-describedby="CR53">53</a></sup>. The calibration standards and 3CB algorithms are described in full elsewhere<sup><a href="#CR16" class="usa-link" aria-describedby="CR16">16</a>,<a href="#CR54" class="usa-link" aria-describedby="CR54">54</a></sup>. The region of interests of lesions and three surrounding rings of 2â€‰mm distance outward from lesion boundary were derived for water, lipid, and protein maps. The median lipid measures of regions of interest within lesions, three rings outside of lesions, differences, and ratios between lesions and rings were generated for both CC and MLO mammograms. Average values of generated variables of two views were used.</p></section><section id="Sec21"><h3 class="pmc_sec_title">Histological sectioning, hematoxylin and eosin staining, and adipocyte area quantification</h3>
<p id="Par30">Invasive breast carcinomas were obtained from the Pathology Departments of the University of California, San Francisco (San Francisco, CA) and Moffitt Cancer Center (Tampa, FL). The study population included 39 hormone receptor-positive tumors (32 ER+/PR+/HER2-, 2 ER+/PRâˆ’/HER2-, 4 ER+/PR+/HER2+, and 1 ER+/PRâˆ’/HER2+), 6 triple-negative (ERâˆ’/PRâˆ’/HER2-) tumors, and 1 ERâˆ’/PRâˆ’/HER2+ tumor. Thirty-nine tumors were invasive ductal carcinomas and 7 were invasive lobular carcinomas. Tissue was fixed in 10% formalin and embedded in paraffin, and 4-micron sections were cut for hematoxylin and eosin (H&amp;E) and immunohistochemical ER, PR, and HER2 staining, as well as HER2 fluorescence in situ hybridization (FISH) for a subset of tumors. ER, PR, and HER2 were scored according to ASCO/CAP guidelines<sup><a href="#CR55" class="usa-link" aria-describedby="CR55">55</a>,<a href="#CR56" class="usa-link" aria-describedby="CR56">56</a></sup>. An H&amp;E-stained slide demonstrating tumor and sufficient (at least 0.5â€‰cm) NAT was chosen from each of 11 tumors with available slides and subjected to whole slide scanning at 400Ã— magnification using an Aperio XT scanner (Leica Biopsystems, Buffalo Grove, IL). Images were visualized using ImageScope software (Leica Biosystems). For each tumor, 4 representative images at 50Ã— magnification (at least 50 adipocytes per image) from R1 and R3 were analyzed using Fiji imaging software with the opensource Adiposoft v1.13 plugin<sup><a href="#CR57" class="usa-link" aria-describedby="CR57">57</a></sup>.</p></section><section id="Sec22"><h3 class="pmc_sec_title">cAMP-dependent lipolysis signature</h3>
<p id="Par31">The cAMP-dependent lipolysis gene signature was generated using RNA-seq data of cAMP-treated adipocytes<sup><a href="#CR19" class="usa-link" aria-describedby="CR19">19</a></sup>. Differentially expressed genes were sorted according to their <em>P</em> value and the top 100 upregulated genes were chosen for the signature. This signature was then used to calculated enrichment scores using the single-set gene set enrichment analysis (ssGSEA) method<sup><a href="#CR21" class="usa-link" aria-describedby="CR21">21</a></sup>. â€œcAMP 100 signatureâ€ enrichment scores were calculated for a dataset containing multiple samples from multiple regions surrounding breast tumors<sup><a href="#CR20" class="usa-link" aria-describedby="CR20">20</a></sup>. The dataset includes samples from the tumor itself (<em>n</em>â€‰=â€‰9), and NAT 1â€‰cm (<em>n</em>â€‰=â€‰7), 2â€‰cm (<em>n</em>â€‰=â€‰5), 3â€‰cm (<em>n</em>â€‰=â€‰3), and 4â€‰cm (<em>n</em>â€‰=â€‰4) away from the tumor, in addition to healthy normal samples (<em>n</em>â€‰=â€‰10). The spatial dataset of multiple regions surrounding breast tumors was downloaded from EMBL-EBI ArrayExpress (Accession E-TABM-276). Raw CEL files were downloaded and processed using custom Affymetrix GeneChip Human Genome U133 Plus 2.0 CDF obtained from BrainArray<sup><a href="#CR58" class="usa-link" aria-describedby="CR58">58</a></sup>. The processing and normalization were performed using the Robust Multi-array Average (RMA) procedure on Affymetrix microarray data.</p></section><section id="Sec23"><h3 class="pmc_sec_title">Laser capture microdissection</h3>
<p id="Par32">Breast tumor tissue was sectioned at 6â€‰Âµm in a Leica CM 1850 Cryostat (Leica Microsystems GmbH). The sections were mounted on uncharged glass slides without the use of embedding media and placed immediately in 70% ethanol for 30â€‰s. Subsequent dehydration was achieved using graded alcohols and xylene treatments as follows: 95% ethanol for 1â€‰min, 100% ethanol for 1â€‰min (times 2), xylene for 2â€‰min and second xylene 3â€‰min. Slides were then dried in a laminar flow hood for 5â€‰min prior to microdissection. Then, sections were laser captured microdissected with PixCell II LCM system (Arcturus Engineering). Approximately 5000 shots using the 30 micron infrared laser beam were utilized to obtain approximately 10,000 cells per dissection. All samples were microdissected in duplicate on sequential sections.</p></section><section id="Sec24"><h3 class="pmc_sec_title">SDS-PAGE and In-Gel Digestion</h3>
<p id="Par33">All membranes containing the microdissected cells from breast tumor tissue were removed and placed directly into a 1.5â€‰mL Eppendorf tube. Membranes containing the microdissected cells were suspended in 20â€‰Î¼L of SDS sample buffer, reduced with DTT and heated in a 70â€“80â€‰Â°C water bath for approximately 10â€‰min. The supernatant was then electrophoresed approximately 2â€‰cm into a 10% Bis Tris gel, stained with Colloidal Blue with destaining with water, and the region was excised and subjected to in-gel trypsin digestion using a standard protocol. Briefly, the gel regions were excised and washed with 100â€‰mM ammonium bicarbonate for 15â€‰min. The liquid was discarded and replaced with fresh 100â€‰mM ammonium bicarbonate and the proteins reduced with 5â€‰mM DTT for 20â€‰min at 55â€‰Â°C. After cooling to room temperature, iodoacetamide was added at 10â€‰mM final concentration and samples wereÂ placed in the dark for 20â€‰min at room temperature. The solution was discarded and the gel pieces washed with 50% acetonitrile/50â€‰mM ammonium bicarbonate for 20â€‰min, followed by dehydration with 100% acetonitrile. The liquid was removed and the gel pieces were completely dried, re-swelled with 0.5â€‰Î¼g of modified trypsin (Promega) in 100â€‰mM NH<sub>4</sub>HCO<sub>3,</sub> and digested overnight at 37â€‰Â°C. Peptides were extracted by three changes of 60% acetonitrile/0.1% TFA, and all extracts were combined and dried <em>in vacuo</em>. Samples were reconstituted in 35â€‰Î¼L 0.1% formic acid for LC-MS/MS analysis.</p></section><section id="Sec25"><h3 class="pmc_sec_title">LC-MS/MS analysis, protein identification, and quantitation</h3>
<p id="Par34">Peptide digests were analyzed on a Thermo LTQ Orbitrap Velos ion trap mass spectrometer equipped with an Eksigent NanoLC 2D pump and AS-1 autosampler as described previously<sup><a href="#CR59" class="usa-link" aria-describedby="CR59">59</a></sup>. Peptide sequence identification from MS/MS spectra employed the UniProt human protein sequence database, release 2025_02, and database search with SequestHT with Proteome Discoverer Software version 2.5 (ThermoFisher Scientific). The dataset contained 1,698,073 MS/MS spectra, with 439,489 peptide-spectrum matches (FDR 0.004) corresponding to 14,501 peptide groups FDR 0.01) and 2751 proteins (FDR 0.01). HNF4Î± was identified by peptide-spectrum match to MS/MS spectra of the 2+ and 3+ precursors corresponding to the fully tryptic sequence LLPGAVATIVKPLSAIPQPTITK, which was identified with a q value of 0.000. Label-free quantitation was performed using precursor ion intensity. Peptide groups were normalized to the total peptide amount, with the normalization factor being the sum of the sample and the maximum sum of all files analyzed. The peptide group abundances were summed to determine the protein abundance. Protein abundances were scaled so that the averages of all samples were 100.</p></section><section id="Sec26"><h3 class="pmc_sec_title">Orthotopic xenograft and allograft studies</h3>
<p id="Par35">The human samples used to generate patient-derived xenograft (PDX) tumors, as well as the human non-tumor samples, were previously described<sup><a href="#CR26" class="usa-link" aria-describedby="CR26">26</a></sup>. The generation of the MTB-TOM tumor model has been previously described<sup><a href="#CR27" class="usa-link" aria-describedby="CR27">27</a></sup>. Four-week-old female WT FVB/N mice (Taconic FVB-F) and immunocompromised NOD/SCID-gamma (NSG) mice (Taconic NODSC) were purchased from Taconic Biosciences. Viably frozen MTB-TOM, HCI002, HCI009, and HCI010 tumor samples were transplanted into the 4th mammary fat pad, following clearance of associated lymph node and epithelium, of respective FVB/N (MTB-TOM) and NSG mice under 2% isoflurane. FVB/N mice were administered dietary doxycycline starting one day before transplant surgery (Bio-Serv #S3888). Tumor growth was monitored daily by caliper measurement in two dimensions. When tumors reached 1â€‰cm (MTB-TOM) or 2â€‰cm (PDX) in any dimension mice were euthanizedÂ and tumor and NAT were isolated, and flash-frozen in liquid nitrogen. For the HCC1143 and HS578T control and Cx31 partial expression loss orthotopic xenografts, for the BT549 shRNA knockdown orthotopic xenografts, and for the HCC1143 partial expression loss <a href="https://www.ncbi.nlm.nih.gov/nuccore/CL316243" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">CL316243</a> studies, 5â€‰Ã—â€‰10<sup>5</sup> cells were resuspended 1:1 with Matrigel (Corning) and injected into the cleared mammary fat pads of 4-week-old WT NSG female mice under 2% isoflurane. Tumor incidence and growth were monitored daily via palpation and caliper measurement, respectively. Mice were euthanized after 180 days or after tumors reached 2â€‰cm in any dimension. For HCC1143 <em>GJB3</em><sup><em>WT</em></sup> and <em>GJB3</em><sup><em>Med</em></sup> xenografts, a central slice of tumor and surrounding NAT was fixed in 4% paraformaldehyde and embedded in paraffin for histological sectioning, H&amp;E staining, and adipocyte area quantification, while the remaining tumor and NAT tissues were flash-frozen in liquid nitrogen. For other xenografts, NAT was isolated and flash-frozen in liquid nitrogen. For the <a href="https://www.ncbi.nlm.nih.gov/nuccore/CL316243" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">CL316243</a> experiment, mice were randomized into experimental groups and moved into new cages immediately post-orthotopic xenograft. The following day, drug treatment was initiated, and mice received vehicle or 1â€‰mg/kg <a href="https://www.ncbi.nlm.nih.gov/nuccore/CL316243" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">CL316243</a>, delivered by intraperitoneal injection, daily until tumor incidence was recorded via palpation. For the Cx31 shRNA knockdown experiments, mice were randomized into experimental groups (with or without doxycycline). In the shCx31 or shGFP knockdown groups mice were administered doxycycline dietarily starting one day before transplant surgery (Bio-Serv #S3888), while mice in other groups received standard chow. All mice imported from Taconic Biosciences were given 3â€“7 days to acclimate to our facilities prior to tumor transplantation or cell injection. Mice were moved into new cages immediately following tumor transplantation or cell injection. All mice were maintained at UCSF rodent barrier facilities, on a 12:12â€‰h light:dark cycle, at 68â€“76â€‰F and 30â€“70% humidity. Because the vast majority of breast cancers occur in women, female mice were chosen for all orthotopic xenograft and allograft studies. In accordance with the UCSF Institutional Animal Care &amp; Use Committee (IACUC) guidelines, the maximal tumor size permitted was 2â€‰cm in any dimension, and this limit was not exceeded in any of the orthotopic xenograft or allograft studies. All animal study protocols described in this, and other sections were given ethical approval by the UCSF IACUC.</p></section><section id="Sec27"><h3 class="pmc_sec_title">Immunoblot analysis</h3>
<p id="Par36">Proteins were extracted using RIPA buffer (Thermo) and proteinase (Roche) plus phosphatase (Roche) inhibitor cocktails. Protein extracts were resolved using 4â€“12% SDS-PAGE gels (Life Technologies) and transferred to nitrocellulose membranes (Life Technologies). Membranes were probed with primary antibodies overnight on a 4â€‰Â°C shaker, then incubated with horseradish peroxidase (HRP)-conjugated secondary antibodies, and signals were visualized with ECL (Bio-Rad). Primary antibodies targeting the following proteins were used: Î²-actin (actin) (sc-47778 HRP, Santa Cruz, 1:10,000), pHSL S563 (4139, Cell Signaling, 1:1000), HSL (4107, Cell Signaling, 1:1000), HNF4Î± (ab41898, Abcam, 1:1000), and Cx31 (ab236620, Abcam, 1:1000). Chemiluminescent signals were acquired with the Bio-Rad ChemiDoc XRS+ System equipped with a supersensitive CCD camera. Where indicated, unsaturated band intensities were quantified using Bio-Rad Image Lab software.</p></section><section id="Sec28"><h3 class="pmc_sec_title">Cell culture and virus production</h3>
<p id="Par37">A panel of established TN and RP human breast cancer cell lines, and their culture conditions, have previously been described<sup><a href="#CR60" class="usa-link" aria-describedby="CR60">60</a></sup>. These cell lines HCC1428 (HCC1428â€“CRL-2327), T47D (T-47Dâ€“HTB-133), HCC3153 (CVCL_3377), HS578T (Hs 578Tâ€“HTB-126), BT549 (BT-549â€“HTB-122) and HCC1143 (HCC1143â€“CRL-2321) were derived from the primary breast cancer cells of female patients and were originally obtained from the collection of Dr. Adi Gazdar at UT Southwestern Medical Center, or from ATCC. No cell line used in this paper is listed in the database of commonly misidentified cell lines that is maintained by the International Cell Line Authentication Committee (ICLAC) (<a href="http://iclac.org/databases/cross-contaminations/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://iclac.org/databases/cross-contaminations/</a>). All lines were found to be negative for mycoplasma contamination. Lentiviruses for Cas9 and sgRNAs were produced in 293T cells using standard polyethylenimine (Polysciences Inc.) transfection protocols.</p></section><section id="Sec29"><h3 class="pmc_sec_title">Dye transfer and FACS analysis</h3>
<p id="Par38">For cancer cell-cancer cell transfer, monolayers of indicated lines (donors) were labelled with 1â€‰Î¼M CalceinAM dye (Life Technologies) at 37â€‰Â°C for 40â€‰min. Dye-loaded â€œdonorâ€ cells were washed three times with PBS, and then single-cell suspensions of 1.5â€‰Ã—â€‰10<sup>5</sup> mCherry-labelled cells (recipients) were added for 5â€‰h. For CBX treatment studies, monolayers of indicated lines (recipients) were pre-treated for 24â€‰h with 150â€‰Î¼M CBX or vehicle. Indicated â€œdonorâ€ cells were loaded in suspension with CalceinAM dye (Life Technologies) at 37â€‰Â°C for 40â€‰min, washed three times with PBS, and added onto indicated â€œrecipientâ€ cells for 5â€‰h. Dye transfer was quantified by BD LSRFORTESSA or BD LSR II (BD Biosciences). Gating strategy to identify mCherry-positive, Calcein-positive cell population is described in Supplementary Fig.Â <a href="#MOESM1" class="usa-link">4</a>. For cancer cell-adipocyte transfer, monolayers of indicated control or Cx31 partial knockout lines (donors) were labelled with 1â€‰Î¼M CalceinAM dye at 37â€‰Â°C for 40â€‰min. Dye-loaded cells were washed three times with PBS, and then primary mammary adipose tissues (recipient) were added for 5â€‰h. Primary adipose tissue was isolated from co-culture, washed with PBS, and dye transfer was quantified by measurement of total adipose fluorescence using a Tecan fluorescent plate reader.</p></section><section id="Sec30"><h3 class="pmc_sec_title">Gene expression analysis</h3>
<p id="Par39">TCGA breast-invasive carcinoma dataset was sourced from data generated by TCGA Research Network (<a href="https://www.cancer.gov/ccg/research/genome-sequencing/tcga" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.cancer.gov/ccg/research/genome-sequencing/tcga</a>), made available on the University of California, Santa Cruz (UCSC) Cancer Browser. DGE analysis of TN compared to RP patient tumors was calculated using the â€˜limmaâ€™ R package<sup><a href="#CR61" class="usa-link" aria-describedby="CR61">61</a></sup>. Single-cell RNA-seq data was sourced from data generated by Chung et al.<sup><a href="#CR38" class="usa-link" aria-describedby="CR38">38</a></sup>. For the MTB-TOM RNA-seq dataset, MTB-TOM mice (MMTV-rtTA/TetO-MYC)<sup><a href="#CR27" class="usa-link" aria-describedby="CR27">27</a></sup> were given doxycycline (<em>n</em>â€‰=â€‰10) or standard chow (<em>n</em>â€‰=â€‰3), and spontaneous tumors (<em>n</em>â€‰=â€‰10) at ethical endpoint (1â€‰cm in any direction) as well as mammary glands from naÃ¯ve mice (<em>n</em>â€‰=â€‰3) were flash-frozen in liquid nitrogen. Library preparation and Illumina RNA-seq was performed by Q<sup>2</sup>Solutions (<a href="http://www.q2labsolutions.com" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">www.q2labsolutions.com</a>). DGE analysis of MTB-TOM compared to normal mammary gland was performed using the DESeq2 package<sup><a href="#CR62" class="usa-link" aria-describedby="CR62">62</a></sup>. All RNA was isolated using the RNAeasy kit (Qiagen). See â€œData availabilityâ€ for gene expression analysis datasets.</p></section><section id="Sec31"><h3 class="pmc_sec_title">ATP quantification</h3>
<p id="Par40">To determine the effects of CBX treatment on ATP levels, tumor cells were seeded in 96-well plates at 5000â€“7000 cells per well and cultured in the presence of vehicleÂ  or 150â€‰Î¼M CBX (Sigma) for 24â€‰h, with triplicate samples for each condition. Relative ATP concentrations were determined using the CellTiter-Glo Luminescent Cell Viability Assay (Promega).</p></section><section id="Sec32"><h3 class="pmc_sec_title">Isolation of primary mammary adipose tissue</h3>
<p id="Par41">Reduction mammoplasty samples were obtained from the CHTN. Samples were washed in DPBS supplemented with 1% Penicillin/Streptomycin and 0.1% Gentamicin (all GIBCO). Mammary adipose tissue was separated mechanically from epithelial tissue using a razor blade, and was then cryopreserved in freezing medium (10% DMSO (Sigma) in FBS (X&amp;Y Cell Culture)). Normal.</p></section><section id="Sec33"><h3 class="pmc_sec_title">Immunofluorescence staining and microscopy</h3>
<p id="Par42">For adipose tissue cancer cell co-cultures imaged whole mount, 2â€‰Ã—â€‰10<sup>6</sup> of the indicated GFP-labelled cell line was suspended in 500â€‰Î¼L DMEM/F-12(Gibco 11320033) containing 10% FBS and injected into primary mammary adipose tissue from a healthy individual, then cultured at 37â€‰Â°C for 24â€‰h. For immunofluorescence labeling of co-culture tissues, samples were washed three times in PBS and fixed in 4% paraformaldehyde, permeabilized in 0.5% Triton X-100 for 15â€‰min, and blocked in 10% goat serum in PBS with 0.25â€‰g/L BSA, 0.2% Triton X-100, and 0.41% Tween-20 overnight. Samples were then incubated overnight with primary antibodies (Cx31, WH0002707M1, Sigma, 1:100, and pHSL(S563), 4139, Cell Sig, 1:100), and then overnight with Alexa Fluor-647 or -546 conjugated antibodies. Phalloidin-stained co-cultures were subsequently incubated overnight with Phalloidin-405 (A30104, Invitrogen, 1:200). Finally, using an established protocol for whole mount breast tissue imaging<sup><a href="#CR41" class="usa-link" aria-describedby="CR41">41</a></sup>, co-culture tissues were cleared through overnight incubation at 4â€‰Â°C in a â€œFUnGIâ€ solution of 50% glycerol (vol/vol), 2.5â€‰M fructose, 2.5â€‰M urea, 10.6â€‰mM Tris Base, and 1â€‰mM EDTA. Confocal images were acquired using a Zeiss LSM900 with Airyscan 2 detector. For pHSL(S563) image quantification, fluorescence was measured using Fiji imaging software, and Difference of Gaussians was used for analysis of puncta number and percent area in Fiji. Version 2.10.0. For sectioned adipose tissue co-culture, 1â€‰Ã—â€‰106 of the indicated mCherry-labelled cell line was injected into primary mammary adipose tissue and cultured at 37â€‰Â°C for 18â€‰h. The co-cultures were examined using fluorescent microscopy to identify regions of adipose tissue containing mCherry-positive cancer cells. These regions were isolated and fixed in 4% paraformaldehyde and embedded in paraffin. Primary TNBCs used for immunofluorescence were identified and retrieved from the clinical archives of the UCSF Department of Pathology. All tumors consisted of estrogen receptor (ER)-, progesterone receptor (PR)-, and HER2-negative invasive ductal carcinomas. Breast tissue was fixed in 10% formalin and embedded in paraffin. Tumor blocks with sufficient tumor and adjacent (at least 0.5â€‰cm) normal tissue were selected, and 4â€‰Î¼m sections were cut on plus-charged slides for immunofluorescence. Patients provided written informed consent and did not receive reimbursement. This study was approved by the UCSF institutional review board. For immunofluorescence labeling of sectioned co-cultures and primary TNBC, slides were dewaxed in xylene followed by rehydration in graded ethanol (100, 95, 70%) and deionized H<sub>2</sub>O. Antigen retrieval was performed in 10â€‰mM Tris, 1â€‰mM EDTA, 0.05% Tween 20, pH 9 at 121â€‰Â°C for 4â€‰min. Subsequently, tissue sections were blocked in 1% bovine serum albuminÂ and 2% fetal bovine serum in PBS for 5â€‰min, then incubated with primary antibodies (Cx31, 12880, Proteintech, 1:50 and pan-cytokeratin, sc-81714, Santa Cruz, 1:50) overnight at 4â€‰Â°C. Following several PBS washes, sections were incubated with Alexa Fluor-488 or -568 conjugated antibodies, counterstained with DAPI (Sigma), and mounted using Vectashield (Vector). Epifluorescence images were acquired either by spinning disk microscopy on a customized microscope setup as previously described<sup><a href="#CR63" class="usa-link" aria-describedby="CR63">63</a>â€“<a href="#CR65" class="usa-link" aria-describedby="CR65">65</a></sup> except that the system was upgraded with a next generation scientific CCD camera (cMyo, 293 Photometrics) with 4.5â€‰Î¼m pixels allowing optimal spatial sampling using a Ã…~60 NA 1.49 objective (CFI 294 APO TIRF; Nikon), or at the UCSF Nikon Imaging Center using a Nikon Ti Microscope equipped with an Andor Zyla 5.5 megapixel sCMOS camera and Lumencor Spectra-X 6-channel LED illuminator. Images were collected using a Plan Apo Î» 20Ã—/0.75 lens.</p></section><section id="Sec34"><h3 class="pmc_sec_title">Generation of Cx31 partial expression loss lines</h3>
<p id="Par43">LentiCas9-Blast (Addgene plasmid #52962) and lentiGuide-Puro (Addgene plasmid #52963) were gifts from Feng Zhang. sgRNAs against Cx31 were constructed using the Feng Zhang Lab CRISPR Design Tool (crispr.mit.edu). sgRNAs used were as follows:</p>
<p id="Par44">Cx31 exon 1 sg1: CCAGATGCGCCCGAACGCTGTGG (HS578T <em>GJB3</em><sup><em>Med-1</em></sup> and HCC1143 <em>GJB3</em><sup><em>Med</em></sup>)</p>
<p id="Par45">Cx31 exon 1 sg2: CCGGGTGCTGGTATACGTGGTGG (HS578T <em>GJB3</em><sup><em>Med-2</em></sup> and HCC1143 <em>GJB3</em><sup><em>Low</em></sup>)</p>
<p id="Par46">ShRNAs against Cx31 and GFP control were constructed using Tet-pLKO-Puro (Addgene plasmid #21915). shRNAs used were as follows:</p>
<p id="Par47">shCx31:</p>
<p id="Par48">shCx31_F: ccggAAGCTCATCATTGAGTTCCTCctcgagGAGGAACTCAATGATGAGCTTtttttg</p>
<p id="Par49">shCx31_R: aattcaaaaaAAGCTCATCATTGAGTTCCTCctcgagGAGGAACTCAATGATGAGCTT</p>
<p id="Par50">shGFP<sup><a href="#CR66" class="usa-link" aria-describedby="CR66">66</a></sup>:</p>
<p id="Par51">shGFP_F: CCGGTACAACAGCCACAACGTCTATCTCGACATAGACGTTGTGGCTGTTGTATTTTTG</p>
<p id="Par52">shGFP_R:CAAAAATACAACAGCCACAACGTCTATGTCGAGATAGACGTTGTGGCTGTTGTACCGG</p>
<p id="Par53">Lentiviral transduction was performed in DMEM supplemented with 10% FBS and polybrene 10â€‰Î¼g/mL. For sgRNA transduction, Cas9-expressing cells were enriched by Blasticidin (10â€“15â€‰Î¼g/mL Gemini BioProducts) selection for seven days. Cas9+ cells were subsequently transduced with lentiGuide-Puro (with sgRNAs targeting Cx31), followed by puromycin (1â€‰Î¼g/mL; Gibco) for seven days. Thereafter, clonal selection was performed and clones screened for loss of target gene protein expression by immunoblot analysis. For shRNAs, cells were transduced with Tet-pLKO-Puro (with shRNAs targeting Cx31 or GFP control<sup><a href="#CR66" class="usa-link" aria-describedby="CR66">66</a></sup>) followed by puromycin (2 ug/mL; Gibco) for seven days, after which knockdown of target protein was confirmed by immunoblot analysis.</p></section><section id="Sec35"><h3 class="pmc_sec_title">cAMP quantification</h3>
<p id="Par54">For in vitro studies, tumor cells were seeded in 96-well plates at 5000â€“7000 cells per well and cultured in the presence of vehicle or 150â€‰Î¼M CBX (Sigma) for 24â€‰h, with triplicate samples for each condition. Changes in cAMP concentration were determined using the cAMP-Glo Assay (Promega).</p>
<p id="Par55">For in vivo studies, frozen tissue was homogenized using a TissueLyser in 300â€‰Î¼l of 40:40:20 acetonitrile:methanol:water with the addition of 1â€‰nM (final concentration) of D3-[15â€‰N]serine as an internal extraction standard (Cambridge Isotopes Laboratories Inc, DNLM-6863). 10â€‰Î¼l of cleared supernatant (via centrifugation at 20,000â€‰Ã—â€‰g., 10â€‰min, at 4â€‰Â°C) was used for SRMâ€“LC-MS/MS using a normal-phase Luna NH2 column (Phenomenex). Mobile phases were buffer A (composed of 100% acetonitrile) and buffer B (composed of 95:5 water:acetonitrile). Solvent modifiers were 0.2% ammonium hydroxide with 50â€‰mM ammonium acetate for negative ionization mode. cAMP levels were analyzed using the MassHunter software package (Agilent Technologies) by quantifying the transition from parent precursor mass to product ions.</p></section><section id="Sec36"><h3 class="pmc_sec_title">cAMP transfer</h3>
<p id="Par56">For cancer cell-adipocyte transfer, monolayers of indicated control or Cx31 partial knockout lines (donors) were labelled with 2â€‰Î¼M fluo-cAMP (Biolog Life Science Institute) at 37â€‰Â°C for 30â€‰min. cAMP-loaded cells were washed three times with PBS, and then primary mammary adipose tissues (recipient) were added for 5â€‰h. Primary adipose tissue was isolated from co-culture, washed with PBS, and cAMP transfer was quantified by measurement of total adipose fluorescence using a Tecan fluorescent plate reader.</p></section><section id="Sec37"><h3 class="pmc_sec_title">Preadipocyte differentiation and qRT-PCR</h3>
<p id="Par57">Primary mouse preadipocytes were differentiated as previously described<sup><a href="#CR67" class="usa-link" aria-describedby="CR67">67</a></sup>. Monolayers of differentiated adipocytes were washed with PBS, and then treated with vehicle or 10â€‰Î¼M forskolin (Sigma), or seeded with 1â€‰Ã—â€‰105 of the indicated cancer lines. Total RNA was isolated from co-cultures after 20â€‰h using the RNeasy kit (Qiagen). One Î¼g of total RNA was reverse transcribed using iScript cDNA synthesis kit (Bio-Rad). The relative expression levelsÂ of <em>UCP1</em>, <em>FABP4</em>, and <em>GAPDH</em> were analyzed using a SYBR Green Real-Time PCR kit (Thermo) with an Applied Biosystems QuantStudio 6 Flex Real-Time PCR System thermocycler (Thermo). Variation was determined using the Î”Î”CT method<sup><a href="#CR68" class="usa-link" aria-describedby="CR68">68</a></sup>(<em>48</em>) with <em>GAPDH</em> mRNA levels as an internal control. Mouse-specific primers used were as follows:</p>
<p id="Par58">GAPDH forward: CCAGCTACTCGCGGCTTTA</p>
<p id="Par59">GAPDH reverse: GTTCACACCGACCTTCACCA</p>
<p id="Par60">UCP1 forward: CACCTTCCCGCTGGACACT</p>
<p id="Par61">UCP1 reverse: CCCTAGGACACCTTTATACCTAATGG</p>
<p id="Par62">FABP4 forward: ACACCGAGATTTCCTTCAAACTG</p>
<p id="Par63">FABP4 reverse: CCATCTAGGGTTATGATGCTCTTCA</p></section><section id="Sec38"><h3 class="pmc_sec_title">Proliferation and viability assays</h3>
<p id="Par64">To determine the effects of Cx31 partial knockout on cell proliferation and viability, the indicated cell lines were seeded in 6-well plates at 1.5â€‰Ã—â€‰10<sup>5</sup> cells/well. Cells were harvested at 24, 48, and 72â€‰h. Cell counts and cell viability by trypan blue exclusion were determined using the Countess Automated Cell Counter (Life Technologies) according to the manufacturerâ€™s instructions.</p></section><section id="Sec39"><h3 class="pmc_sec_title">Statistics &amp; reproducibility</h3>
<p id="Par65">Prism software (v 10.4.1) was used to generate and analyze Spearman correlation (Fig.Â <a href="#Fig1" class="usa-link">1d</a>) and the survival plots (Fig.Â <a href="#Fig4" class="usa-link">4b, c, and g</a>). Survival plot <em>P</em>-values were generated using a log-rank test. Correlation <em>P</em> values were generated using ordinary one-way ANOVA with multiple comparisons (Figs.Â <a href="#Fig1" class="usa-link">1f, h</a>, <a href="#Fig2" class="usa-link">2b</a>, <a href="#Fig3" class="usa-link">3f, h</a>, <a href="#Fig4" class="usa-link">4a</a> right, <a href="#Fig4" class="usa-link">4d and f</a>), two-way ANOVA with multiple comparisons (Fig.Â <a href="#Fig1" class="usa-link">1c</a>), repeated measures one-way ANOVA with multiple comparisons (Figs.Â <a href="#Fig1" class="usa-link">1b</a> and <a href="#Fig3" class="usa-link">3e and g</a>), repeated measures mixed effects model with multiple comparisons (Fig.Â <a href="#Fig1" class="usa-link">1e</a>), and unpaired two-tailed <em>t</em> test (Figs.Â <a href="#Fig2" class="usa-link">2a and c</a>, <a href="#Fig4" class="usa-link">4a</a> left and center, <a href="#Fig4" class="usa-link">4e</a>, and Supplementary Fig.Â <a href="#MOESM1" class="usa-link">1a, b</a>). These analyses were performed using PRISM software. DGE analysis of TN compared to RP patient tumors (Fig.Â <a href="#Fig2" class="usa-link">2d</a>) was calculated using the â€œlimmaâ€ R package<sup><a href="#CR61" class="usa-link" aria-describedby="CR61">61</a></sup>. DGE analysis of MTB-TOM compared to normal mammary gland (Fig.Â <a href="#Fig2" class="usa-link">2f</a>) was performed using the DESeq2 package<sup><a href="#CR62" class="usa-link" aria-describedby="CR62">62</a></sup>. Differential expression analyses (Fig.Â <a href="#Fig2" class="usa-link">2d and f</a>) were calculated with a false discovery rate of 0.05. Biological replicates are shown in Figs.Â <a href="#Fig1" class="usa-link">1bâ€“e, g, h</a>, <a href="#Fig2" class="usa-link">2aâ€“c</a>, <a href="#Fig3" class="usa-link">3eâ€“h</a>, <a href="#Fig4" class="usa-link">4a, câ€“f</a>, and Supplementary Fig.Â <a href="#MOESM1" class="usa-link">1a, b</a>. Technical duplicates for each biological replicate are shown in Fig.Â <a href="#Fig1" class="usa-link">1f</a>. Immunofluorescence images shown in Fig.Â <a href="#Fig3" class="usa-link">3a, b and d</a> and Supplementary Fig.Â <a href="#MOESM1" class="usa-link">3a, b and e</a> are representative from experiments repeated in 3 independent biological replicates.</p>
<p id="Par66">No statistical method was used to predetermine sample size. The investigators were not blinded to allocation during in vivo experiments. The investigators were blinded during immunofluorescence analysis. All in vivo studies were randomized as tumors reached predetermined volume, or palpability, on a per-experiment basis as described in <em>Orthotopic xenograft and allograft studies</em> above. No samples from in vivo studies were processed and then excluded from immunoblot, intracellular cAMP or Adiposoft analysis, except where tumor size was insufficient to allow for analysis via multiple methods. No mice that completed the xenograft or allograft studies were excluded from analyses. For all in vitro and in vivo experiments, sample size was not chosen with consideration of adequate power to detect a pre-specified effect size. For in vitro studies, all completed experiments are reported. For in vivo studies, number of indicated mice per individual experiment represents the total number of mice treated and analyzed.</p></section><section id="Sec40"><h3 class="pmc_sec_title">Reporting summary</h3>
<p id="Par67">Further information on research design is available in theÂ <a href="#MOESM6" class="usa-link">Nature Portfolio Reporting Summary</a> linked to this article.</p></section></section><section id="Sec41"><h2 class="pmc_sec_title">Supplementary information</h2>
<section class="sm xbox font-sm" id="MOESM1"><div class="media p"><div class="caption">
<a href="/articles/instance/12368202/bin/41467_2025_62486_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information</a><sup> (12.5MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM2"><div class="media p"><div class="caption">
<a href="/articles/instance/12368202/bin/41467_2025_62486_MOESM2_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">41467_2025_62486_MOESM2_ESM.pdf</a><sup> (44.1KB, pdf) </sup><p>Description of Additional Supplementary Files</p>
</div></div></section><section class="sm xbox font-sm" id="MOESM3"><div class="media p"><div class="caption">
<a href="/articles/instance/12368202/bin/41467_2025_62486_MOESM3_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 1</a><sup> (13.2KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM4"><div class="media p"><div class="caption">
<a href="/articles/instance/12368202/bin/41467_2025_62486_MOESM4_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 2</a><sup> (3.3MB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM5"><div class="media p"><div class="caption">
<a href="/articles/instance/12368202/bin/41467_2025_62486_MOESM5_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 3</a><sup> (2.5MB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM6"><div class="media p"><div class="caption">
<a href="/articles/instance/12368202/bin/41467_2025_62486_MOESM6_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Reporting Summary</a><sup> (128.3KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM7"><div class="media p"><div class="caption">
<a href="/articles/instance/12368202/bin/41467_2025_62486_MOESM7_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Transparent Peer Review file</a><sup> (1.4MB, pdf) </sup>
</div></div></section></section><section id="Sec42"><h2 class="pmc_sec_title">Source data</h2>
<section class="sm xbox font-sm" id="MOESM8"><div class="media p"><div class="caption">
<a href="/articles/instance/12368202/bin/41467_2025_62486_MOESM8_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Source Data</a><sup> (18.9MB, xlsx) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>This work was supported, in part, by the US Department of Defenseâ€“Congressionally Directed Medical Research Programsâ€™ Era of Hope Scholar award W81XWH-12-1-0272 and W81XWH-21-1-0774 (A.G.), the US National Institutes of Health (NIH) grants R01CA17447 and 2R01EB028148 (A.G.) and R01 CA056721 (Z.W.), the Atwater Foundation and Bechtle Family Foundation (A.G.), the Breast Cancer Research Fund and Subramanian Breast Cancer Support Fund (H.R., A.G.), the EMBO postdoctoral fellowship ALTF 159-2017 (Ju.W.), the US K99/R00 NIH Pathway to Independence Award DK110426 (K.S.), the US National Cancer Institute grants R01CA166945 and R01CA257652 (J.A.S.), and the US NIH F99/K00 Predoctoral to Postdoctoral Transition Award F99CA212488 (R.C.) and the US NIH F31 Predoctoral Individual National Research Service Award F31CA243468 (J.W.). Tissue samples were provided by the Cooperative Human Tissue Network (RRID: SCR_004446), a National Cancer Institute supported resource. Other investigators may have received samples from these same tissue specimens. The authors thank A. Welm for guidance in the use of patient-derived xenografts, A. Tward for technical guidance and helpful discussions, K.A. Fontaine for helpful discussions and comments on the manuscript, and S. Samson for a helpful consumer advocate perspective and guidance on our work.</p></section><section id="notes1"><h2 class="pmc_sec_title">Author contributions</h2>
<p>R.C. and A.G. conceived of research. J.W. and R.C. designed and contributed to all in vitro and ex vivo studies, and to in vivo mouse studies, contributed to all data analysis, and wrote the manuscript. Se.M. analyzed 3CB data. L.J.Z. designed LCM and conducted proteomic analyses. Su.M. conducted the LCM. D.A. conducted lipolysis signature enrichment and scRNA-seq analyses. A.B. contributed to immunofluorescent staining and microscopy, and generation of Cx31 partial knockout lines. D.V. characterized Cx31 partial depletion lines, generated Cx31 inducible short hairpin lines, contributed to Cx31 knockdown and <a href="https://www.ncbi.nlm.nih.gov/nuccore/CL316243" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">CL316243</a> mouse studies and provided helpful discussions. S.K.A. contributed to generation of Cx31 inducible short hairpin lines. R.N. contributed to co-culture studies, immunofluorescent staining, tissue clearing, and microscopy, and provided valuable discussion. Y.C. conducted preadipocyte differentiation. C.B. and S.L. conducted mass spectrometry for cAMP. C.M. generated Cas9-expressing cancer cell lines. Ju.W. and E.W. isolated primary mammary adipose tissue. E.J.H. contributed to co-culture microscopy and provided valuable discussion. J.D.G. and D.S. conducted FACS analysis. K.S. provided cAMP-dependent lipolysis signature. M.G. contributed to RNA-seq analysis. H.N. contributed to co-culture studies. K.M.A. supervised FACS analysis. Z.W. supervised primary mammary adipose tissue isolation and provided valuable discussion. D.K.N. supervised mass spectrometry for cAMP and provided valuable discussion. S.K. supervised preadipocyte and cAMP-dependent lipolysis studies and provided valuable discussion. A.J.B. supervised enrichment analysis. M.E.S. and D.C.L. designed and supervised LCM and proteomics. H.R. provided valuable discussion. G.K. designed and conducted histological analyses and provided valuable discussion. J.A.S. conceived of applying 3CB study data, supervised the 3CB study and data analysis, and provided valuable discussion. A.G. supervised the study, and provided valuable discussion and intellectual input. All authors edited the manuscript.</p></section><section id="notes2"><h2 class="pmc_sec_title">Peer review</h2>
<section id="FPar1"><h3 class="pmc_sec_title">Peer review information</h3>
<p id="Par68"><em>Nature Communications</em> thanks Andrew Hoy and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. A peer review file is available.</p></section></section><section id="notes3"><h2 class="pmc_sec_title">Data availability</h2>
<p>The dataset used in Fig.Â <a href="#Fig1" class="usa-link">1e</a> containing multiple samples from multiple regions surrounding breast tumors was previously published<sup><a href="#CR20" class="usa-link" aria-describedby="CR20">20</a></sup> and is available in the ArrayExpress database<sup><a href="#CR69" class="usa-link" aria-describedby="CR69">69</a></sup> under accession code <a href="https://www.ebi.ac.uk/biostudies/arrayexpress/studies/E-TABM-276" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">E-TABM-276</a>. Data used to generate the cAMP 100 signature applied in Fig.Â <a href="#Fig1" class="usa-link">1e</a> was previously published<sup><a href="#CR19" class="usa-link" aria-describedby="CR19">19</a></sup> and are available in the ArrayExpress database<sup><a href="#CR69" class="usa-link" aria-describedby="CR69">69</a></sup> under accession code <a href="https://www.ebi.ac.uk/biostudies/arrayexpress/studies/E-MTAB-2602" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">E-MTAB-2602</a>. Protein MS data containing HNF4Î± spectral counts from healthy patient normal breast, stroma and breast lesion LCM generated for Fig.Â <a href="#Fig1" class="usa-link">1f</a> and Supplementary DataÂ <a href="#MOESM1" class="usa-link">2a, b</a> are available through Proteome Xchange under accession code <a href="https://proteomecentral.proteomexchange.org/cgi/GetDataset?ID=PXD063930" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">MSV000097890</a>. The dataset used in Fig.Â <a href="#Fig2" class="usa-link">2d</a> is available from The Cancer Genome Atlas Program (TCGA) database under accession code phs000178 [<a href="https://www.cancer.gov/ccg/research/genome-sequencing/tcga" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.cancer.gov/ccg/research/genome-sequencing/tcga</a>]. Our annotations of the TCGA dataset used in Fig.Â <a href="#Fig2" class="usa-link">2d</a> are available [<a href="https://bitbucket.org/jeevb/brca" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://bitbucket.org/jeevb/brca</a>]. The single-cell RNA-seq dataset for Fig.Â <a href="#Fig2" class="usa-link">2e</a> and Supplementary Fig.Â <a href="#MOESM1" class="usa-link">2</a> was previously published<sup><a href="#CR38" class="usa-link" aria-describedby="CR38">38</a></sup> and is available in the NCBI Gene Expression Omnibus database under accession code <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE75688" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE75688</a>. The RNA-seq dataset used in Fig.Â <a href="#Fig2" class="usa-link">2f</a> and Supplementary DataÂ <a href="#MOESM5" class="usa-link">3</a> in the MTB-TOM (MMTV-rtTA/TetO-MYC) model comparing spontaneous tumor tissue, from mice on dietary doxycycline, to control non-tumor tissue, from mice on standard chow, was previously published<sup><a href="#CR70" class="usa-link" aria-describedby="CR70">70</a></sup> and is available in the NCBI Gene Expression Omnibus under <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE130921" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE130921</a>.Â <a href="#Sec42" class="usa-link">Source data</a> are provided with this paper.</p></section><section id="FPar2"><h2 class="pmc_sec_title">Competing interests</h2>
<p id="Par69">The authors declare no competing interests.</p></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm">
<div class="fn p" id="fn1"><p><strong>Publisherâ€™s note</strong> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></div>
<div class="fn p" id="fn2"><p>These authors contributed equally: Jeremy Williams, Roman Camarda.</p></div>
</div></section><section id="sec43"><h2 class="pmc_sec_title">Supplementary information</h2>
<p>The online version contains supplementary material available at 10.1038/s41467-025-62486-3.</p></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Goodwin, P. J. &amp; Chlebowski, R. T. Obesity and cancer: insights for clinicians. <em>J. Clin. Oncol.</em><strong>34</strong>, 4197â€“4202 (2016).
</cite> [<a href="https://doi.org/10.1200/JCO.2016.70.5327" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5455321/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27903158/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Goodwin,%20P.%20J.%20&amp;%20Chlebowski,%20R.%20T.%20Obesity%20and%20cancer:%20insights%20for%20clinicians.%20J.%20Clin.%20Oncol.34,%204197%E2%80%934202%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>Camarda, R. et al. Inhibition of fatty acid oxidation as a therapy for MYC-overexpressing triple-negative breast cancer. <em>Nat. Med.</em><strong>22</strong>, 427â€“432 (2016).
</cite> [<a href="https://doi.org/10.1038/nm.4055" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4892846/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26950360/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Camarda,%20R.%20et%20al.%20Inhibition%20of%20fatty%20acid%20oxidation%20as%20a%20therapy%20for%20MYC-overexpressing%20triple-negative%20breast%20cancer.%20Nat.%20Med.22,%20427%E2%80%93432%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>Park, J. H. et al. Fatty acid oxidation-driven Src links mitochondrial energy reprogramming and oncogenic properties in triple-negative breast cancer. <em>Cell Rep.</em><strong>14</strong>, 2154â€“2165 (2016).
</cite> [<a href="https://doi.org/10.1016/j.celrep.2016.02.004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4809061/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26923594/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Park,%20J.%20H.%20et%20al.%20Fatty%20acid%20oxidation-driven%20Src%20links%20mitochondrial%20energy%20reprogramming%20and%20oncogenic%20properties%20in%20triple-negative%20breast%20cancer.%20Cell%20Rep.14,%202154%E2%80%932165%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Barnard, M. E. et al. Body mass index and mammographic density in a multiracial and multiethnic population-based study. <em>Cancer Epidemiol. Biomark. Prev.</em><strong>31</strong>, 1313â€“1323 (2022).</cite> [<a href="https://doi.org/10.1158/1055-9965.EPI-21-1249" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9250611/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35511751/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Barnard,%20M.%20E.%20et%20al.%20Body%20mass%20index%20and%20mammographic%20density%20in%20a%20multiracial%20and%20multiethnic%20population-based%20study.%20Cancer%20Epidemiol.%20Biomark.%20Prev.31,%201313%E2%80%931323%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Ye, H. et al. Leukemic stem cells evade chemotherapy by metabolic adaptation to an adipose tissue niche. <em>Cell Stem Cell</em><strong>19</strong>, 23â€“37 (2016).
</cite> [<a href="https://doi.org/10.1016/j.stem.2016.06.001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4938766/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27374788/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ye,%20H.%20et%20al.%20Leukemic%20stem%20cells%20evade%20chemotherapy%20by%20metabolic%20adaptation%20to%20an%20adipose%20tissue%20niche.%20Cell%20Stem%20Cell19,%2023%E2%80%9337%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Hoy, A. J., Balaban, S. &amp; Saunders, D. N. Adipocyteâ€“tumor cell metabolic crosstalk in breast cancer. <em>Trends Mol. Med.</em><strong>23</strong>, 381â€“392 (2017).
</cite> [<a href="https://doi.org/10.1016/j.molmed.2017.02.009" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28330687/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hoy,%20A.%20J.,%20Balaban,%20S.%20&amp;%20Saunders,%20D.%20N.%20Adipocyte%E2%80%93tumor%20cell%20metabolic%20crosstalk%20in%20breast%20cancer.%20Trends%20Mol.%20Med.23,%20381%E2%80%93392%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Nieman, K. M. et al. Adipocytes promote ovarian cancer metastasis and provide energy for rapid tumor growth. <em>Nat. Med.</em><strong>17</strong>, 1498â€“1503 (2011).
</cite> [<a href="https://doi.org/10.1038/nm.2492" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4157349/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22037646/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Nieman,%20K.%20M.%20et%20al.%20Adipocytes%20promote%20ovarian%20cancer%20metastasis%20and%20provide%20energy%20for%20rapid%20tumor%20growth.%20Nat.%20Med.17,%201498%E2%80%931503%20(2011)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>Balaban, S., Lee, L. S., Schreuder, M. &amp; Hoy, A. J. Obesity and cancer progression: Is there a role of fatty acid metabolism? <em>BioMed. Res. Inter.</em><strong>2015</strong>10.1155/2015/274585 (2015).</cite> [<a href="https://doi.org/10.1155/2015/274585" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4383231/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25866768/" class="usa-link">PubMed</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>Wang, Y. Y. et al. Mammary adipocytes stimulate breast cancer invasion through metabolic remodeling of tumor cells. <em>JCI Insight</em><strong>2</strong>, e87489 (2017).</cite> [<a href="https://doi.org/10.1172/jci.insight.87489" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5313068/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28239646/" class="usa-link">PubMed</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>Wen, Y. A. et al. Adipocytes activate mitochondrial fatty acid oxidation and autophagy to promote tumor growth in colon cancer. <em>Cell Death Dis.</em><strong>8</strong>, e2593 (2017).
</cite> [<a href="https://doi.org/10.1038/cddis.2017.21" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5386470/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28151470/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wen,%20Y.%20A.%20et%20al.%20Adipocytes%20activate%20mitochondrial%20fatty%20acid%20oxidation%20and%20autophagy%20to%20promote%20tumor%20growth%20in%20colon%20cancer.%20Cell%20Death%20Dis.8,%20e2593%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Wang, C., Gao, C., Meng, K., Qiao, H. &amp; Wang, Y. Human adipocytes stimulate invasion of breast cancer MCF-7 cells by secreting IGFBP-2. <em>PLoS ONE</em><strong>10</strong>10.1371/journal.pone.0119348 (2015).</cite> [<a href="https://doi.org/10.1371/journal.pone.0119348" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4352027/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25747684/" class="usa-link">PubMed</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Muller, C. Tumour-surrounding adipocytes are active players in breast cancer progression. <em>Ann. Endocrinol.</em><strong>74</strong>, 108â€“110 (2013).</cite> [<a href="https://doi.org/10.1016/j.ando.2013.02.007" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23566616/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Muller,%20C.%20Tumour-surrounding%20adipocytes%20are%20active%20players%20in%20breast%20cancer%20progression.%20Ann.%20Endocrinol.74,%20108%E2%80%93110%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>Dirat, B. et al. Cancer-associated adipocytes exhibit an activated phenotype and contribute to breast cancer invasion. <em>Cancer Res.</em><strong>71</strong>, 2455â€“2465 (2011).
</cite> [<a href="https://doi.org/10.1158/0008-5472.CAN-10-3323" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21459803/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Dirat,%20B.%20et%20al.%20Cancer-associated%20adipocytes%20exhibit%20an%20activated%20phenotype%20and%20contribute%20to%20breast%20cancer%20invasion.%20Cancer%20Res.71,%202455%E2%80%932465%20(2011)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>Dirat, B. A., Bochet, L., Escourrou, G., Valet, P. &amp; Muller, C. Unraveling the obesity and breast cancer links: a role for cancer-associated adipocytes?. <em>Endocr. Dev.</em><strong>19</strong>, 45â€“52 (2010).
</cite> [<a href="https://doi.org/10.1159/000316896" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20551667/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Dirat,%20B.%20A.,%20Bochet,%20L.,%20Escourrou,%20G.,%20Valet,%20P.%20&amp;%20Muller,%20C.%20Unraveling%20the%20obesity%20and%20breast%20cancer%20links:%20a%20role%20for%20cancer-associated%20adipocytes?.%20Endocr.%20Dev.19,%2045%E2%80%9352%20(2010)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>Hovey, R. C. &amp; Aimo, L. Diverse and active roles for adipocytes during mammary gland growth and function. <em>J. Mammary Gland Biol. Neoplasia</em><strong>15</strong>, 279â€“290 (2010).
</cite> [<a href="https://doi.org/10.1007/s10911-010-9187-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2941079/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20717712/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hovey,%20R.%20C.%20&amp;%20Aimo,%20L.%20Diverse%20and%20active%20roles%20for%20adipocytes%20during%20mammary%20gland%20growth%20and%20function.%20J.%20Mammary%20Gland%20Biol.%20Neoplasia15,%20279%E2%80%93290%20(2010)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Laidevant, A. D., Malkov, S., Flowers, C. I., Kerlikowske, K. &amp; Shepherd, J. A. Compositional breast imaging using a dual-energy mammography protocol. <em>Med Phys.</em><strong>37</strong>, 164â€“174 (2010).
</cite> [<a href="https://doi.org/10.1118/1.3259715" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2801735/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20175478/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Laidevant,%20A.%20D.,%20Malkov,%20S.,%20Flowers,%20C.%20I.,%20Kerlikowske,%20K.%20&amp;%20Shepherd,%20J.%20A.%20Compositional%20breast%20imaging%20using%20a%20dual-energy%20mammography%20protocol.%20Med%20Phys.37,%20164%E2%80%93174%20(2010)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>Drukker, K. et al. Mammographic quantitative image analysis and biologic image composition for breast lesion characterization and classification. <em>Med. Phys.</em><strong>41</strong>, 031915 (2014).
</cite> [<a href="https://doi.org/10.1118/1.4866221" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3978370/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24593733/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Drukker,%20K.%20et%20al.%20Mammographic%20quantitative%20image%20analysis%20and%20biologic%20image%20composition%20for%20breast%20lesion%20characterization%20and%20classification.%20Med.%20Phys.41,%20031915%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Duncan, R. E., Ahmadian, M., Jaworski, K., Sarkadi-Nagy, E. &amp; Sul, H. S. Regulation of lipolysis in adipocytes. <em>Annu. Rev. Nutr.</em><strong>27</strong>, 79â€“101 (2007).
</cite> [<a href="https://doi.org/10.1146/annurev.nutr.27.061406.093734" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2885771/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17313320/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Duncan,%20R.%20E.,%20Ahmadian,%20M.,%20Jaworski,%20K.,%20Sarkadi-Nagy,%20E.%20&amp;%20Sul,%20H.%20S.%20Regulation%20of%20lipolysis%20in%20adipocytes.%20Annu.%20Rev.%20Nutr.27,%2079%E2%80%93101%20(2007)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Shinoda, K. et al. Genetic and functional characterization of clonally derived adult human brown adipocytes. <em>Nat. Med.</em><strong>21</strong>, 389â€“394 (2015).
</cite> [<a href="https://doi.org/10.1038/nm.3819" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4427356/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25774848/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Shinoda,%20K.%20et%20al.%20Genetic%20and%20functional%20characterization%20of%20clonally%20derived%20adult%20human%20brown%20adipocytes.%20Nat.%20Med.21,%20389%E2%80%93394%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>Cheng, A. S. L. et al. Epithelial progeny of estrogen-exposed breast progenitor cells display a cancer-like methylome. <em>Cancer Res.</em><strong>68</strong>, 1786â€“1796 (2008).
</cite> [<a href="https://doi.org/10.1158/0008-5472.CAN-07-5547" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4172329/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18339859/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cheng,%20A.%20S.%20L.%20et%20al.%20Epithelial%20progeny%20of%20estrogen-exposed%20breast%20progenitor%20cells%20display%20a%20cancer-like%20methylome.%20Cancer%20Res.68,%201786%E2%80%931796%20(2008)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>Verhaak, R. G. et al. Prognostically relevant gene signatures of high-grade serous ovarian carcinoma. <em>J. Clin. Invest.</em><strong>123</strong>, 517â€“525 (2013).
</cite> [<a href="https://doi.org/10.1172/JCI65833" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3533304/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23257362/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Verhaak,%20R.%20G.%20et%20al.%20Prognostically%20relevant%20gene%20signatures%20of%20high-grade%20serous%20ovarian%20carcinoma.%20J.%20Clin.%20Invest.123,%20517%E2%80%93525%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>Zhu, Y. et al. Connexin 43 mediates white adipose tissue beiging by facilitating the propagation of sympathetic neuronal signals. <em>Cell Metab.</em><strong>24</strong>, 420â€“433 (2016).
</cite> [<a href="https://doi.org/10.1016/j.cmet.2016.08.005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5024720/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27626200/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhu,%20Y.%20et%20al.%20Connexin%2043%20mediates%20white%20adipose%20tissue%20beiging%20by%20facilitating%20the%20propagation%20of%20sympathetic%20neuronal%20signals.%20Cell%20Metab.24,%20420%E2%80%93433%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>Palanker, L., Tennessen, J. M., Lam, G. &amp; Thummel, C. S. Drosophila HNF4 regulates lipid mobilization and beta-oxidation. <em>Cell Metab.</em><strong>9</strong>, 228â€“239 (2009).
</cite> [<a href="https://doi.org/10.1016/j.cmet.2009.01.009" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2673486/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19254568/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Palanker,%20L.,%20Tennessen,%20J.%20M.,%20Lam,%20G.%20&amp;%20Thummel,%20C.%20S.%20Drosophila%20HNF4%20regulates%20lipid%20mobilization%20and%20beta-oxidation.%20Cell%20Metab.9,%20228%E2%80%93239%20(2009)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>Large, V. et al. Decreased expression and function of adipocyte hormone-sensitive lipase in subcutaneous fat cells of obese subjects. <em>J. Lipid Res.</em><strong>40</strong>, 2059â€“2066 (1999).
</cite> [<a href="https://pubmed.ncbi.nlm.nih.gov/10553009/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Large,%20V.%20et%20al.%20Decreased%20expression%20and%20function%20of%20adipocyte%20hormone-sensitive%20lipase%20in%20subcutaneous%20fat%20cells%20of%20obese%20subjects.%20J.%20Lipid%20Res.40,%202059%E2%80%932066%20(1999)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>Wang, F. et al. Mammary fat of breast cancer: Gene expression profiling and functional characterization. <em>PLoS ONE</em><strong>9</strong>10.1371/journal.pone.0109742 (2014).</cite> [<a href="https://doi.org/10.1371/journal.pone.0109742" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4188628/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25291184/" class="usa-link">PubMed</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>DeRose, Y. S. et al. Tumor grafts derived from women with breast cancer authentically reflect tumor pathology, growth, metastasis and disease outcomes. <em>Nat. Med.</em><strong>17</strong>, 1514â€“1520 (2011).
</cite> [<a href="https://doi.org/10.1038/nm.2454" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3553601/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22019887/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?DeRose,%20Y.%20S.%20et%20al.%20Tumor%20grafts%20derived%20from%20women%20with%20breast%20cancer%20authentically%20reflect%20tumor%20pathology,%20growth,%20metastasis%20and%20disease%20outcomes.%20Nat.%20Med.17,%201514%E2%80%931520%20(2011)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Dâ€™Cruz, C. M. et al. c-MYC induces mammary tumorigenesis by means of a preferred pathway involving spontaneous Kras2 mutations. <em>Nat. Med.</em><strong>7</strong>, 235â€“239 (2001).
</cite> [<a href="https://doi.org/10.1038/84691" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11175856/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?D%E2%80%99Cruz,%20C.%20M.%20et%20al.%20c-MYC%20induces%20mammary%20tumorigenesis%20by%20means%20of%20a%20preferred%20pathway%20involving%20spontaneous%20Kras2%20mutations.%20Nat.%20Med.7,%20235%E2%80%93239%20(2001)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Aran, D. et al. Comprehensive analysis of normal adjacent to tumor transcriptomes. <em>Nat. Commun.</em><strong>8</strong>, 1077 (2017).
</cite> [<a href="https://doi.org/10.1038/s41467-017-01027-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5651823/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29057876/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Aran,%20D.%20et%20al.%20Comprehensive%20analysis%20of%20normal%20adjacent%20to%20tumor%20transcriptomes.%20Nat.%20Commun.8,%201077%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>Aasen, T., Mesnil, M., Naus, C. C., Lampe, P. D. &amp; Laird, D. W. Gap junctions and cancer: communicating for 50 years. <em>Nat. Rev. Cancer</em><strong>16</strong>, 775â€“788 (2016).
</cite> [<a href="https://doi.org/10.1038/nrc.2016.105" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5279857/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27782134/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Aasen,%20T.,%20Mesnil,%20M.,%20Naus,%20C.%20C.,%20Lampe,%20P.%20D.%20&amp;%20Laird,%20D.%20W.%20Gap%20junctions%20and%20cancer:%20communicating%20for%2050%20years.%20Nat.%20Rev.%20Cancer16,%20775%E2%80%93788%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR30">
<span class="label">30.</span><cite>Zhu, D., Caveney, S., Kidder, G. M. &amp; Naus, C. C. Transfection of C6 glioma cells with connexin 43 cDNA: analysis of expression, intercellular coupling, and cell proliferation. <em>Proc. Natl. Acad. Sci.</em><strong>88</strong>, 1883â€“1887 (1991).
</cite> [<a href="https://doi.org/10.1073/pnas.88.5.1883" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC51130/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/1848013/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhu,%20D.,%20Caveney,%20S.,%20Kidder,%20G.%20M.%20&amp;%20Naus,%20C.%20C.%20Transfection%20of%20C6%20glioma%20cells%20with%20connexin%2043%20cDNA:%20analysis%20of%20expression,%20intercellular%20coupling,%20and%20cell%20proliferation.%20Proc.%20Natl.%20Acad.%20Sci.88,%201883%E2%80%931887%20(1991)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>Eghbali, B., Kessler, J. A., Reid, L. M., Roy, C. &amp; Spray, D. C. Involvement of gap junctions in tumorigenesis: transfection of tumor cells with connexin 32 cDNA retards growth in vivo. <em>Proc. Natl. Acad. Sci.</em><strong>88</strong>, 10701â€“10705 (1991).
</cite> [<a href="https://doi.org/10.1073/pnas.88.23.10701" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC52998/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/1660148/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Eghbali,%20B.,%20Kessler,%20J.%20A.,%20Reid,%20L.%20M.,%20Roy,%20C.%20&amp;%20Spray,%20D.%20C.%20Involvement%20of%20gap%20junctions%20in%20tumorigenesis:%20transfection%20of%20tumor%20cells%20with%20connexin%2032%20cDNA%20retards%20growth%20in%20vivo.%20Proc.%20Natl.%20Acad.%20Sci.88,%2010701%E2%80%9310705%20(1991)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>Huang, R.-P. et al. Reversion of the neoplastic phenotype of human glioblastoma cells by connexin 43 (cx43)1. <em>Cancer Res.</em><strong>58</strong>, 5089â€“5096 (1998).
</cite> [<a href="https://pubmed.ncbi.nlm.nih.gov/9823317/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Huang,%20R.-P.%20et%20al.%20Reversion%20of%20the%20neoplastic%20phenotype%20of%20human%20glioblastoma%20cells%20by%20connexin%2043%20(cx43)1.%20Cancer%20Res.58,%205089%E2%80%935096%20(1998)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Gleisner, M. A., Navarrete, M., Hofmann, F., Salazar-Onfray, F. &amp; Tittarelli, A. Mind the gaps in tumor immunity: impact of connexin-mediated intercellular connections. <em>Front Immunol.</em><strong>8</strong>, 1067 (2017).
</cite> [<a href="https://doi.org/10.3389/fimmu.2017.01067" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5585150/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28919895/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gleisner,%20M.%20A.,%20Navarrete,%20M.,%20Hofmann,%20F.,%20Salazar-Onfray,%20F.%20&amp;%20Tittarelli,%20A.%20Mind%20the%20gaps%20in%20tumor%20immunity:%20impact%20of%20connexin-mediated%20intercellular%20connections.%20Front%20Immunol.8,%201067%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>Chen, Q. et al. Carcinomaâ€“astrocyte gap junctions promote brain metastasis by cGAMP transfer. <em>Nature</em><strong>533</strong>, 493â€“498 (2016).
</cite> [<a href="https://doi.org/10.1038/nature18268" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5021195/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27225120/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chen,%20Q.%20et%20al.%20Carcinoma%E2%80%93astrocyte%20gap%20junctions%20promote%20brain%20metastasis%20by%20cGAMP%20transfer.%20Nature533,%20493%E2%80%93498%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>Osswald, M. et al. Brain tumour cells interconnect to a functional and resistant network. <em>Nature</em><strong>528</strong>, 93â€“98 (2015).
</cite> [<a href="https://doi.org/10.1038/nature16071" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26536111/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Osswald,%20M.%20et%20al.%20Brain%20tumour%20cells%20interconnect%20to%20a%20functional%20and%20resistant%20network.%20Nature528,%2093%E2%80%9398%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR36">
<span class="label">36.</span><cite>Horiuchi, D. et al. MYC pathway activation in triple-negative breast cancer is synthetic lethal with CDK inhibition. <em> J. Exp. Med.</em><strong>209</strong>, 679â€“696 (2012).
</cite> [<a href="https://doi.org/10.1084/jem.20111512" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3328367/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22430491/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Horiuchi,%20D.%20et%20al.%20MYC%20pathway%20activation%20in%20triple-negative%20breast%20cancer%20is%20synthetic%20lethal%20with%20CDK%20inhibition.%20J.%20Exp.%20Med.209,%20679%E2%80%93696%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR37">
<span class="label">37.</span><cite>Koboldt, D. C. et al. Comprehensive molecular portraits of human breast tumours. <em>Nature</em><strong>490</strong>, 61â€“70 (2012).
</cite> [<a href="https://doi.org/10.1038/nature11412" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3465532/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23000897/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Koboldt,%20D.%20C.%20et%20al.%20Comprehensive%20molecular%20portraits%20of%20human%20breast%20tumours.%20Nature490,%2061%E2%80%9370%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR38">
<span class="label">38.</span><cite>Chung, W. et al. Single-cell RNA-seq enables comprehensive tumour and immune cell profiling in primary breast cancer. <em>Nat. Commun.</em><strong>8</strong>, 15081 (2017).
</cite> [<a href="https://doi.org/10.1038/ncomms15081" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5424158/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28474673/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chung,%20W.%20et%20al.%20Single-cell%20RNA-seq%20enables%20comprehensive%20tumour%20and%20immune%20cell%20profiling%20in%20primary%20breast%20cancer.%20Nat.%20Commun.8,%2015081%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR39">
<span class="label">39.</span><cite>Wenzel, K., Manthey, D., Willecke, K., Grzeschik, K. H. &amp; Traub, O. Human gap junction protein connexin31: molecular cloning and expression analysis. <em>Biochem. Biophys. Res. Commun.</em><strong>248</strong>, 910â€“915 (1998).
</cite> [<a href="https://doi.org/10.1006/bbrc.1998.9070" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9704026/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wenzel,%20K.,%20Manthey,%20D.,%20Willecke,%20K.,%20Grzeschik,%20K.%20H.%20&amp;%20Traub,%20O.%20Human%20gap%20junction%20protein%20connexin31:%20molecular%20cloning%20and%20expression%20analysis.%20Biochem.%20Biophys.%20Res.%20Commun.248,%20910%E2%80%93915%20(1998)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR40">
<span class="label">40.</span><cite>Willebrords, J., Maes, M., Crespo Yanguas, S. &amp; Vinken, M. Inhibitors of connexin and pannexin channels as potential therapeutics. <em>Pharm. Ther.</em><strong>180</strong>, 144â€“160 (2017).</cite> [<a href="https://doi.org/10.1016/j.pharmthera.2017.07.001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5802387/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28720428/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Willebrords,%20J.,%20Maes,%20M.,%20Crespo%20Yanguas,%20S.%20&amp;%20Vinken,%20M.%20Inhibitors%20of%20connexin%20and%20pannexin%20channels%20as%20potential%20therapeutics.%20Pharm.%20Ther.180,%20144%E2%80%93160%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR41">
<span class="label">41.</span><cite>Rios, A. C. et al. Intraclonal plasticity in mammary tumors revealed through large-scale single-cell resolution 3D imaging. <em>Cancer Cell</em><strong>35</strong>, 618â€“632.e616 (2019).
</cite> [<a href="https://doi.org/10.1016/j.ccell.2019.02.010" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30930118/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rios,%20A.%20C.%20et%20al.%20Intraclonal%20plasticity%20in%20mammary%20tumors%20revealed%20through%20large-scale%20single-cell%20resolution%203D%20imaging.%20Cancer%20Cell35,%20618%E2%80%93632.e616%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR42">
<span class="label">42.</span><cite>Aune, U. L., Ruiz, L. &amp; Kajimura, S. Isolation and differentiation of stromal vascular cells to beige/brite cells. <em>JoVE</em>, e50191 10.3791/50191 (2013).</cite> [<a href="https://doi.org/10.3791/50191" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3641667/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23568137/" class="usa-link">PubMed</a>]</li>
<li id="CR43">
<span class="label">43.</span><cite>Wang, Q. A., Tao, C., Gupta, R. K. &amp; Scherer, P. E. Tracking adipogenesis during white adipose tissue development, expansion and regeneration. <em>Nat. Med</em>. <strong>19</strong>, 1338â€“1344 (2013).
</cite> [<a href="https://doi.org/10.1038/nm.3324" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4075943/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23995282/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang,%20Q.%20A.,%20Tao,%20C.,%20Gupta,%20R.%20K.%20&amp;%20Scherer,%20P.%20E.%20Tracking%20adipogenesis%20during%20white%20adipose%20tissue%20development,%20expansion%20and%20regeneration.%20Nat.%20Med.%2019,%201338%E2%80%931344%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR44">
<span class="label">44.</span><cite>Grabner, G. F., Xie, H., Schweiger, M. &amp; Zechner, R. Lipolysis: cellular mechanisms for lipid mobilization from fat stores. <em>Nat. Metab.</em><strong>3</strong>, 1445â€“1465 (2021).
</cite> [<a href="https://doi.org/10.1038/s42255-021-00493-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34799702/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Grabner,%20G.%20F.,%20Xie,%20H.,%20Schweiger,%20M.%20&amp;%20Zechner,%20R.%20Lipolysis:%20cellular%20mechanisms%20for%20lipid%20mobilization%20from%20fat%20stores.%20Nat.%20Metab.3,%201445%E2%80%931465%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR45">
<span class="label">45.</span><cite>Arner, P. &amp; Langin, D. Lipolysis in lipid turnover, cancer cachexia, and obesity-induced insulin resistance. <em>Trends Endocrinol. Metab.</em><strong>25</strong>, 255â€“262 (2014).
</cite> [<a href="https://doi.org/10.1016/j.tem.2014.03.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24731595/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Arner,%20P.%20&amp;%20Langin,%20D.%20Lipolysis%20in%20lipid%20turnover,%20cancer%20cachexia,%20and%20obesity-induced%20insulin%20resistance.%20Trends%20Endocrinol.%20Metab.25,%20255%E2%80%93262%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR46">
<span class="label">46.</span><cite>Tan, Y. et al. Adipocytes fuel gastric cancer omental metastasis via PITPNC1-mediated fatty acid metabolic reprogramming. <em>Theranostics</em><strong>8</strong>, 5452â€“5468 (2018).
</cite> [<a href="https://doi.org/10.7150/thno.28219" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6276097/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30555557/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tan,%20Y.%20et%20al.%20Adipocytes%20fuel%20gastric%20cancer%20omental%20metastasis%20via%20PITPNC1-mediated%20fatty%20acid%20metabolic%20reprogramming.%20Theranostics8,%205452%E2%80%935468%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR47">
<span class="label">47.</span><cite>Zhang, X. H., Giuliano, M., Trivedi, M. V., Schiff, R. &amp; Osborne, C. K. Metastasis dormancy in estrogen receptor-positive breast cancer. <em>Clin. Cancer Res.</em><strong>19</strong>, 6389â€“6397 (2013).
</cite> [<a href="https://doi.org/10.1158/1078-0432.CCR-13-0838" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3878717/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24298069/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhang,%20X.%20H.,%20Giuliano,%20M.,%20Trivedi,%20M.%20V.,%20Schiff,%20R.%20&amp;%20Osborne,%20C.%20K.%20Metastasis%20dormancy%20in%20estrogen%20receptor-positive%20breast%20cancer.%20Clin.%20Cancer%20Res.19,%206389%E2%80%936397%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR48">
<span class="label">48.</span><cite>Templeton, Z. S. et al. Breast cancer cell colonization of the human bone marrow adipose tissue niche. <em>Neoplasia</em><strong>17</strong>, 849â€“861 (2015).
</cite> [<a href="https://doi.org/10.1016/j.neo.2015.11.005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4688564/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26696367/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Templeton,%20Z.%20S.%20et%20al.%20Breast%20cancer%20cell%20colonization%20of%20the%20human%20bone%20marrow%20adipose%20tissue%20niche.%20Neoplasia17,%20849%E2%80%93861%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR49">
<span class="label">49.</span><cite>Nagle, C. M. et al. Obesity and survival among women with ovarian cancer: results from the Ovarian Cancer Association Consortium. <em>Br. J. Cancer</em><strong>113</strong>, 817â€“826 (2015).
</cite> [<a href="https://doi.org/10.1038/bjc.2015.245" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4559823/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26151456/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Nagle,%20C.%20M.%20et%20al.%20Obesity%20and%20survival%20among%20women%20with%20ovarian%20cancer:%20results%20from%20the%20Ovarian%20Cancer%20Association%20Consortium.%20Br.%20J.%20Cancer113,%20817%E2%80%93826%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR50">
<span class="label">50.</span><cite>Xu, F. et al. GLP-1 receptor agonist promotes brown remodelling in mouse white adipose tissue through SIRT1. <em>Diabetologia</em><strong>59</strong>, 1059â€“1069 (2016).
</cite> [<a href="https://doi.org/10.1007/s00125-016-3896-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26924394/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Xu,%20F.%20et%20al.%20GLP-1%20receptor%20agonist%20promotes%20brown%20remodelling%20in%20mouse%20white%20adipose%20tissue%20through%20SIRT1.%20Diabetologia59,%201059%E2%80%931069%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR51">
<span class="label">51.</span><cite>Sancho, V. et al. Effect of GLP-1 on D-glucose transport, lipolysis and lipogenesis in adipocytes of obese subjects. <em>Int. J. Mol. Med.</em><strong>17</strong>, 1133â€“1137 (2006).
</cite> [<a href="https://pubmed.ncbi.nlm.nih.gov/16685426/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sancho,%20V.%20et%20al.%20Effect%20of%20GLP-1%20on%20D-glucose%20transport,%20lipolysis%20and%20lipogenesis%20in%20adipocytes%20of%20obese%20subjects.%20Int.%20J.%20Mol.%20Med.17,%201133%E2%80%931137%20(2006)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR52">
<span class="label">52.</span><cite>Malkov, S., Wang, J., Kerlikowske, K., Cummings, S. R. &amp; Shepherd, J. A. Single x-ray absorptiometry method for the quantitative mammographic measure of fibroglandular tissue volume. <em>Med. Phys.</em><strong>36</strong>, 5525â€“5536 (2009).
</cite> [<a href="https://doi.org/10.1118/1.3253972" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2789112/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20095265/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Malkov,%20S.,%20Wang,%20J.,%20Kerlikowske,%20K.,%20Cummings,%20S.%20R.%20&amp;%20Shepherd,%20J.%20A.%20Single%20x-ray%20absorptiometry%20method%20for%20the%20quantitative%20mammographic%20measure%20of%20fibroglandular%20tissue%20volume.%20Med.%20Phys.36,%205525%E2%80%935536%20(2009)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR53">
<span class="label">53.</span><cite>Malkov, S., Wang, J., Duewer, F. &amp; Shepherd, J. A. A Calibration Approach for Single-Energy X-ray Absorptiometry Method to Provide Absolute Breast Tissue Composition Accuracy for the Long Term. In <em>Breast Imaging. IWDM 2012. Lecture Notes in Computer Science</em> (eds. Maidment, A. D. A., Bakic, P. R. &amp; Gavenonis, S.) vol 7361 (Springer, Berlin, Heidelberg, 2012).</cite>
</li>
<li id="CR54">
<span class="label">54.</span><cite>Malkov, S. et al. Calibration procedure of three component mammographic breast imaging. <em>International Workshop on Digital Mammography</em>. pp. 211â€“218 (Cham: Springer International Publishing, 2016).</cite>
</li>
<li id="CR55">
<span class="label">55.</span><cite>Hammond, M. E. et al. American Society of Clinical Oncology/College of American Pathologists guideline recommendations for immunohistochemical testing of estrogen and progesterone receptors in breast cancer (unabridged version). <em>Arch. Pathol. Lab Med.</em><strong>134</strong>, e48â€“e72 (2010).
</cite> [<a href="https://doi.org/10.5858/134.7.e48" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20586616/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hammond,%20M.%20E.%20et%20al.%20American%20Society%20of%20Clinical%20Oncology/College%20of%20American%20Pathologists%20guideline%20recommendations%20for%20immunohistochemical%20testing%20of%20estrogen%20and%20progesterone%20receptors%20in%20breast%20cancer%20(unabridged%20version).%20Arch.%20Pathol.%20Lab%20Med.134,%20e48%E2%80%93e72%20(2010)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR56">
<span class="label">56.</span><cite>Wolff, A. C. et al. Recommendations for human epidermal growth factor receptor 2 testing in breast cancer: American Society of Clinical Oncology/College of American Pathologists clinical practice guideline update. <em>J. Clin. Oncol.</em><strong>31</strong>, 3997â€“4013 (2013).
</cite> [<a href="https://doi.org/10.1200/JCO.2013.50.9984" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24101045/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wolff,%20A.%20C.%20et%20al.%20Recommendations%20for%20human%20epidermal%20growth%20factor%20receptor%202%20testing%20in%20breast%20cancer:%20American%20Society%20of%20Clinical%20Oncology/College%20of%20American%20Pathologists%20clinical%20practice%20guideline%20update.%20J.%20Clin.%20Oncol.31,%203997%E2%80%934013%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR57">
<span class="label">57.</span><cite>Galarraga, M. et al. Adiposoft: automated software for the analysis of white adipose tissue cellularity in histological sections. <em>J. Lipid Res.</em><strong>53</strong>, 2791â€“2796 (2012).
</cite> [<a href="https://doi.org/10.1194/jlr.D023788" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3494244/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22993232/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Galarraga,%20M.%20et%20al.%20Adiposoft:%20automated%20software%20for%20the%20analysis%20of%20white%20adipose%20tissue%20cellularity%20in%20histological%20sections.%20J.%20Lipid%20Res.53,%202791%E2%80%932796%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR58">
<span class="label">58.</span><cite>Dai, M. et al. Evolving gene/transcript definitions significantly alter the interpretation of GeneChip data. <em>Nucleic Acids Res.</em><strong>33</strong>, e175 (2005).
</cite> [<a href="https://doi.org/10.1093/nar/gni179" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1283542/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16284200/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Dai,%20M.%20et%20al.%20Evolving%20gene/transcript%20definitions%20significantly%20alter%20the%20interpretation%20of%20GeneChip%20data.%20Nucleic%20Acids%20Res.33,%20e175%20(2005)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR59">
<span class="label">59.</span><cite>Zhang, B. et al. Proteogenomic characterization of human colon and rectal cancer. <em>Nature</em><strong>513</strong>, 382â€“387 (2014).
</cite> [<a href="https://doi.org/10.1038/nature13438" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4249766/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25043054/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhang,%20B.%20et%20al.%20Proteogenomic%20characterization%20of%20human%20colon%20and%20rectal%20cancer.%20Nature513,%20382%E2%80%93387%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR60">
<span class="label">60.</span><cite>Neve, R. M. et al. A collection of breast cancer cell lines for the study of functionally distinct cancer subtypes. <em>Cancer Cell</em><strong>10</strong>, 515â€“527 (2006).
</cite> [<a href="https://doi.org/10.1016/j.ccr.2006.10.008" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2730521/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17157791/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Neve,%20R.%20M.%20et%20al.%20A%20collection%20of%20breast%20cancer%20cell%20lines%20for%20the%20study%20of%20functionally%20distinct%20cancer%20subtypes.%20Cancer%20Cell10,%20515%E2%80%93527%20(2006)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR61">
<span class="label">61.</span><cite>Smyth, G. K. Limma: Linear Models for Microarray Data. In <em>Bioinformatics and Computational Biology Solutions Using R and Bioconductor</em> (eds. Gentleman, R. et al.) 397â€“420 (Springer New York, 2005).</cite>
</li>
<li id="CR62">
<span class="label">62.</span><cite>Love, M. I., Huber, W. &amp; Anders, S. Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2. <em>Genome Biol.</em><strong>15</strong>, 550 (2014).
</cite> [<a href="https://doi.org/10.1186/s13059-014-0550-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4302049/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25516281/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Love,%20M.%20I.,%20Huber,%20W.%20&amp;%20Anders,%20S.%20Moderated%20estimation%20of%20fold%20change%20and%20dispersion%20for%20RNA-seq%20data%20with%20DESeq2.%20Genome%20Biol.15,%20550%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR63">
<span class="label">63.</span><cite>Ettinger, A. &amp; Wittmann, T. Fluorescence live cell imaging. In <em>Methods in Cell Biology</em> Vol. 123 (eds. Waters, J. C. &amp; Wittman, T.) 77â€“94 (Academic Press, 2014).</cite> [<a href="https://doi.org/10.1016/B978-0-12-420138-5.00005-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4198327/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24974023/" class="usa-link">PubMed</a>]</li>
<li id="CR64">
<span class="label">64.</span><cite>Stehbens, S., Pemble, H., Murrow, L. &amp; Wittmann, T. Imaging intracellular protein dynamics by spinning disk confocal microscopy. In <em>Methods in Enzymology</em> (ed. Conn, P. M.) Vol. 504, 293â€“313 (Academic Press, 2012).</cite> [<a href="https://doi.org/10.1016/B978-0-12-391857-4.00015-X" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3495241/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22264541/" class="usa-link">PubMed</a>]</li>
<li id="CR65">
<span class="label">65.</span><cite>Stehbens, S. J. et al. CLASPs link focal-adhesion-associated microtubule capture to localized exocytosis and adhesion site turnover. <em>Nat. Cell Biol.</em><strong>16</strong>, 561â€“573 (2014).
</cite> [<a href="https://doi.org/10.1038/ncb2975" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4108447/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24859005/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Stehbens,%20S.%20J.%20et%20al.%20CLASPs%20link%20focal-adhesion-associated%20microtubule%20capture%20to%20localized%20exocytosis%20and%20adhesion%20site%20turnover.%20Nat.%20Cell%20Biol.16,%20561%E2%80%93573%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR66">
<span class="label">66.</span><cite>Rohrberg, J. et al. MYC dysregulates mitosis, revealing cancer vulnerabilities. <em>Cell Rep.</em><strong>30</strong>, 3368â€“3382.e3367 (2020).
</cite> [<a href="https://doi.org/10.1016/j.celrep.2020.02.041" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7085414/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32160543/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rohrberg,%20J.%20et%20al.%20MYC%20dysregulates%20mitosis,%20revealing%20cancer%20vulnerabilities.%20Cell%20Rep.30,%203368%E2%80%933382.e3367%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR67">
<span class="label">67.</span><cite>Kajimura, S. et al. Initiation of myoblast to brown fat switch by a PRDM16-C/EBP-Î² transcriptional complex. <em>Nature</em><strong>460</strong>, 1154â€“1158 (2009).
</cite> [<a href="https://doi.org/10.1038/nature08262" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2754867/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19641492/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kajimura,%20S.%20et%20al.%20Initiation%20of%20myoblast%20to%20brown%20fat%20switch%20by%20a%20PRDM16-C/EBP-%CE%B2%20transcriptional%20complex.%20Nature460,%201154%E2%80%931158%20(2009)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR68">
<span class="label">68.</span><cite>Livak, K. J. &amp; Schmittgen, T. D. Analysis of relative gene expression data using real-time quantitative PCR and the 2(-Delta Delta C(T)) method. <em>Methods</em><strong>25</strong>, 402â€“408 (2001).
</cite> [<a href="https://doi.org/10.1006/meth.2001.1262" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11846609/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Livak,%20K.%20J.%20&amp;%20Schmittgen,%20T.%20D.%20Analysis%20of%20relative%20gene%20expression%20data%20using%20real-time%20quantitative%20PCR%20and%20the%202(-Delta%20Delta%20C(T))%20method.%20Methods25,%20402%E2%80%93408%20(2001)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR69">
<span class="label">69.</span><cite>Brazma, A. et al. ArrayExpressâ€”a public repository for microarray gene expression data at the EBI. <em>Nucleic Acids Res.</em><strong>31</strong>, 68â€“71 (2003).
</cite> [<a href="https://doi.org/10.1093/nar/gkg091" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC165538/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12519949/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Brazma,%20A.%20et%20al.%20ArrayExpress%E2%80%94a%20public%20repository%20for%20microarray%20gene%20expression%20data%20at%20the%20EBI.%20Nucleic%20Acids%20Res.31,%2068%E2%80%9371%20(2003)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR70">
<span class="label">70.</span><cite>Lee, J. V. et al. Combinatorial immunotherapies overcome MYC-driven immune evasion in triple negative breast cancer. <em>Nat. Commun.</em><strong>13</strong>, 3671 (2022).
</cite> [<a href="https://doi.org/10.1038/s41467-022-31238-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9237085/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35760778/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lee,%20J.%20V.%20et%20al.%20Combinatorial%20immunotherapies%20overcome%20MYC-driven%20immune%20evasion%20in%20triple%20negative%20breast%20cancer.%20Nat.%20Commun.13,%203671%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12368202/bin/41467_2025_62486_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information</a><sup> (12.5MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12368202/bin/41467_2025_62486_MOESM2_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">41467_2025_62486_MOESM2_ESM.pdf</a><sup> (44.1KB, pdf) </sup><p>Description of Additional Supplementary Files</p>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12368202/bin/41467_2025_62486_MOESM3_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 1</a><sup> (13.2KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material4_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12368202/bin/41467_2025_62486_MOESM4_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 2</a><sup> (3.3MB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material5_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12368202/bin/41467_2025_62486_MOESM5_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 3</a><sup> (2.5MB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material6_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12368202/bin/41467_2025_62486_MOESM6_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Reporting Summary</a><sup> (128.3KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material7_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12368202/bin/41467_2025_62486_MOESM7_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Transparent Peer Review file</a><sup> (1.4MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material8_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12368202/bin/41467_2025_62486_MOESM8_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Source Data</a><sup> (18.9MB, xlsx) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>The dataset used in Fig.Â <a href="#Fig1" class="usa-link">1e</a> containing multiple samples from multiple regions surrounding breast tumors was previously published<sup><a href="#CR20" class="usa-link" aria-describedby="CR20">20</a></sup> and is available in the ArrayExpress database<sup><a href="#CR69" class="usa-link" aria-describedby="CR69">69</a></sup> under accession code <a href="https://www.ebi.ac.uk/biostudies/arrayexpress/studies/E-TABM-276" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">E-TABM-276</a>. Data used to generate the cAMP 100 signature applied in Fig.Â <a href="#Fig1" class="usa-link">1e</a> was previously published<sup><a href="#CR19" class="usa-link" aria-describedby="CR19">19</a></sup> and are available in the ArrayExpress database<sup><a href="#CR69" class="usa-link" aria-describedby="CR69">69</a></sup> under accession code <a href="https://www.ebi.ac.uk/biostudies/arrayexpress/studies/E-MTAB-2602" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">E-MTAB-2602</a>. Protein MS data containing HNF4Î± spectral counts from healthy patient normal breast, stroma and breast lesion LCM generated for Fig.Â <a href="#Fig1" class="usa-link">1f</a> and Supplementary DataÂ <a href="#MOESM1" class="usa-link">2a, b</a> are available through Proteome Xchange under accession code <a href="https://proteomecentral.proteomexchange.org/cgi/GetDataset?ID=PXD063930" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">MSV000097890</a>. The dataset used in Fig.Â <a href="#Fig2" class="usa-link">2d</a> is available from The Cancer Genome Atlas Program (TCGA) database under accession code phs000178 [<a href="https://www.cancer.gov/ccg/research/genome-sequencing/tcga" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.cancer.gov/ccg/research/genome-sequencing/tcga</a>]. Our annotations of the TCGA dataset used in Fig.Â <a href="#Fig2" class="usa-link">2d</a> are available [<a href="https://bitbucket.org/jeevb/brca" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://bitbucket.org/jeevb/brca</a>]. The single-cell RNA-seq dataset for Fig.Â <a href="#Fig2" class="usa-link">2e</a> and Supplementary Fig.Â <a href="#MOESM1" class="usa-link">2</a> was previously published<sup><a href="#CR38" class="usa-link" aria-describedby="CR38">38</a></sup> and is available in the NCBI Gene Expression Omnibus database under accession code <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE75688" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE75688</a>. The RNA-seq dataset used in Fig.Â <a href="#Fig2" class="usa-link">2f</a> and Supplementary DataÂ <a href="#MOESM5" class="usa-link">3</a> in the MTB-TOM (MMTV-rtTA/TetO-MYC) model comparing spontaneous tumor tissue, from mice on dietary doxycycline, to control non-tumor tissue, from mice on standard chow, was previously published<sup><a href="#CR70" class="usa-link" aria-describedby="CR70">70</a></sup> and is available in the NCBI Gene Expression Omnibus under <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE130921" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE130921</a>.Â <a href="#Sec42" class="usa-link">Source data</a> are provided with this paper.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Nature Communications are provided here courtesy of <strong>Nature Publishing Group</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1038/s41467-025-62486-3"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/41467_2025_Article_62486.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (2.8Â MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12368202/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12368202/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12368202%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12368202/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12368202/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12368202/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40835606/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12368202/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40835606/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12368202/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12368202/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="vVd8CD9ILGYbxqBBAr70xyBclLrRneFQBi8CT6SHSDTRspyVIOMXHzfNxJ2uEVtu">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
