
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Chromosome-level genome assembly and annotation of Amur Pike (Esox reichertii) - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="152546AB8AF32E130546AB00018A5B39.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="sdata">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12368265/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Scientific Data">
<meta name="citation_title" content="Chromosome-level genome assembly and annotation of Amur Pike (Esox reichertii)">
<meta name="citation_author" content="Yuyong Zhang">
<meta name="citation_author_institution" content="National and Local Joint Engineering Laboratory for Freshwater Fish Breeding, Heilongjiang River Fisheries Research Institute, Chinese Academy of Fishery Sciences, Harbin, 150070 China">
<meta name="citation_author" content="Yongquan Zhang">
<meta name="citation_author_institution" content="National and Local Joint Engineering Laboratory for Freshwater Fish Breeding, Heilongjiang River Fisheries Research Institute, Chinese Academy of Fishery Sciences, Harbin, 150070 China">
<meta name="citation_author" content="Tangbin Huo">
<meta name="citation_author_institution" content="National and Local Joint Engineering Laboratory for Freshwater Fish Breeding, Heilongjiang River Fisheries Research Institute, Chinese Academy of Fishery Sciences, Harbin, 150070 China">
<meta name="citation_author" content="Gerel Qogt">
<meta name="citation_author_institution" content="Administration Bureau of Hulun Lake National Nature Reserve in Inner Mongolia, Hulunbuir, 021000 China">
<meta name="citation_author" content="Ping Zhang">
<meta name="citation_author_institution" content="Administration Bureau of Hulun Lake National Nature Reserve in Inner Mongolia, Hulunbuir, 021000 China">
<meta name="citation_author" content="Huiping Zhang">
<meta name="citation_author_institution" content="Administration Bureau of Hulun Lake National Nature Reserve in Inner Mongolia, Hulunbuir, 021000 China">
<meta name="citation_author" content="Zhiying Jia">
<meta name="citation_author_institution" content="National and Local Joint Engineering Laboratory for Freshwater Fish Breeding, Heilongjiang River Fisheries Research Institute, Chinese Academy of Fishery Sciences, Harbin, 150070 China">
<meta name="citation_publication_date" content="2025 Aug 20">
<meta name="citation_volume" content="12">
<meta name="citation_firstpage" content="1454">
<meta name="citation_doi" content="10.1038/s41597-025-05819-7">
<meta name="citation_pmid" content="40835863">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12368265/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12368265/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12368265/pdf/41597_2025_Article_5819.pdf">
<meta name="description" content="Amur pike (Esox reicherti) is endemic to the Heilongjiang River system in Northeast Asia. Serving as a top predator in its ecosystem, Amur pike carries substantial significance in ecological, economic, and cultural dimensions. Molecular studies have ...">
<meta name="og:title" content="Chromosome-level genome assembly and annotation of Amur Pike (Esox reichertii)">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Amur pike (Esox reicherti) is endemic to the Heilongjiang River system in Northeast Asia. Serving as a top predator in its ecosystem, Amur pike carries substantial significance in ecological, economic, and cultural dimensions. Molecular studies have ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12368265/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12368265">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1038/s41597-025-05819-7"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/41597_2025_Article_5819.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12368265%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12368265/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12368265/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12368265/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-sdata.jpg" alt="Scientific Data logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Scientific Data" title="Link to Scientific Data" shape="default" href="http://www.nature.com/sdata/" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Sci Data</button></div>. 2025 Aug 20;12:1454. doi: <a href="https://doi.org/10.1038/s41597-025-05819-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1038/s41597-025-05819-7</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Sci%20Data%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sci%20Data%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Sci%20Data%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Sci%20Data%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Chromosome-level genome assembly and annotation of Amur Pike (<em>Esox reichertii</em>)</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Yuyong Zhang</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Yuyong Zhang</span></h3>
<div class="p">
<sup>1</sup>National and Local Joint Engineering Laboratory for Freshwater Fish Breeding, Heilongjiang River Fisheries Research Institute, Chinese Academy of Fishery Sciences, Harbin, 150070 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yuyong Zhang</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Yongquan Zhang</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Yongquan Zhang</span></h3>
<div class="p">
<sup>1</sup>National and Local Joint Engineering Laboratory for Freshwater Fish Breeding, Heilongjiang River Fisheries Research Institute, Chinese Academy of Fishery Sciences, Harbin, 150070 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yongquan Zhang</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Huo%20T%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Tangbin Huo</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Tangbin Huo</span></h3>
<div class="p">
<sup>1</sup>National and Local Joint Engineering Laboratory for Freshwater Fish Breeding, Heilongjiang River Fisheries Research Institute, Chinese Academy of Fishery Sciences, Harbin, 150070 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Huo%20T%22%5BAuthor%5D" class="usa-link"><span class="name western">Tangbin Huo</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Qogt%20G%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Gerel Qogt</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Gerel Qogt</span></h3>
<div class="p">
<sup>2</sup>Administration Bureau of Hulun Lake National Nature Reserve in Inner Mongolia, Hulunbuir, 021000 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Qogt%20G%22%5BAuthor%5D" class="usa-link"><span class="name western">Gerel Qogt</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20P%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Ping Zhang</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Ping Zhang</span></h3>
<div class="p">
<sup>2</sup>Administration Bureau of Hulun Lake National Nature Reserve in Inner Mongolia, Hulunbuir, 021000 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20P%22%5BAuthor%5D" class="usa-link"><span class="name western">Ping Zhang</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20H%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Huiping Zhang</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Huiping Zhang</span></h3>
<div class="p">
<sup>2</sup>Administration Bureau of Hulun Lake National Nature Reserve in Inner Mongolia, Hulunbuir, 021000 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20H%22%5BAuthor%5D" class="usa-link"><span class="name western">Huiping Zhang</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Jia%20Z%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Zhiying Jia</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Zhiying Jia</span></h3>
<div class="p">
<sup>1</sup>National and Local Joint Engineering Laboratory for Freshwater Fish Breeding, Heilongjiang River Fisheries Research Institute, Chinese Academy of Fishery Sciences, Harbin, 150070 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Jia%20Z%22%5BAuthor%5D" class="usa-link"><span class="name western">Zhiying Jia</span></a>
</div>
</div>
<sup>1,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>National and Local Joint Engineering Laboratory for Freshwater Fish Breeding, Heilongjiang River Fisheries Research Institute, Chinese Academy of Fishery Sciences, Harbin, 150070 China </div>
<div id="Aff2">
<sup>2</sup>Administration Bureau of Hulun Lake National Nature Reserve in Inner Mongolia, Hulunbuir, 021000 China </div>
<div class="author-notes p"><div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div></div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 Jun 13; Accepted 2025 Aug 13; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by-nc-nd/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12368265  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40835863/" class="usa-link">40835863</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<p id="Par1">Amur pike (<em>Esox reicherti</em>) is endemic to the Heilongjiang River system in Northeast Asia. Serving as a top predator in its ecosystem, Amur pike carries substantial significance in ecological, economic, and cultural dimensions. Molecular studies have firmly established that Esociformes is the sister group of Salmoniformes. Notably, unlike Salmoniformes, Esociformes has not undergone genomic duplication. In this study, we reported a chromosome-level genome assembly of Amur pike, the size of which is 976.47 Mb with a contig N50 of 30.76 Mb. Using Hi-C interaction information, a total of 948.05 Mb (98.51%) of the assembled sequences was successfully anchored and ordered onto 25 pseudochromosomes, ranged from 22.85 Mb to 56.71 Mb. Further analysis identified 368.55 Mb of repetitive sequences and 25,824 protein-coding genes in the genome. Among them, 98.80% (25,515) of the genes were functionally annotated. This high-quality genome could be a valuable resource for future studies of Amur pike. Moreover, it lays a robust groundwork for comparative genomics involving pikes and salmonids.</p>
<section id="kwd-group1" class="kwd-group"><p><strong>Subject terms:</strong> Genome, Ichthyology</p></section></section><section id="Sec1"><h2 class="pmc_sec_title">Background &amp; Summary</h2>
<p id="Par2"><em>Esox</em> is a genus of freshwater fish commonly known as pike or pickerel. It is the only extant genus in the family Esocidae of the order Esociformes<sup><a href="#CR1" class="usa-link" aria-describedby="CR1">1</a></sup>. Pikes are fierce predatory species, regarded as occupying the apex of the ecosystem’s food chain and playing a crucial role in maintaining ecological balance<sup><a href="#CR2" class="usa-link" aria-describedby="CR2">2</a></sup>. With their unique appearance and strong struggling ability, pikes have also become a popular target for human fishing and angling<sup><a href="#CR3" class="usa-link" aria-describedby="CR3">3</a></sup>. Amur pike (<em>Esox reicherti</em>) is endemic to the Heilongjiang River system, covering the northern part of Northeast China<sup><a href="#CR4" class="usa-link" aria-describedby="CR4">4</a>,<a href="#CR5" class="usa-link" aria-describedby="CR5">5</a></sup>. However, owing to factors like overfishing and habitat alterations, its resources have declined significantly<sup><a href="#CR6" class="usa-link" aria-describedby="CR6">6</a></sup>. In China, measures such as fishing moratoriums, habitat restoration, artificial breeding and stock enhancement have been initiated to protect and restore the resources of Amur pike.</p>
<p id="Par3">Molecular evidences strongly supported that the order Esociformes and the order Salmoniformes are sister groups<sup><a href="#CR7" class="usa-link" aria-describedby="CR7">7</a>,<a href="#CR8" class="usa-link" aria-describedby="CR8">8</a></sup>. The later encompasses many fish groups familiar to the public, such as Salmon, trout, char, grayling, whitefish, and so on. Due to their important ecological and economic values, as well as rich morphological and life history diversities, salmonids are one of the most concerned, well studied fish groups<sup><a href="#CR9" class="usa-link" aria-describedby="CR9">9</a></sup>. After the divergence from the order Esociformes, the order Salmoniformes thought to be experienced a whole-genome duplication event, and underwent multiple rounds of rediploidization processes<sup><a href="#CR10" class="usa-link" aria-describedby="CR10">10</a></sup>. During this complex evolutionary process, large-scale rearrangements occurred in the genomes of the order Salmoniformes, leading to obvious differences in the karyotypes among different groups<sup><a href="#CR11" class="usa-link" aria-describedby="CR11">11</a>–<a href="#CR13" class="usa-link" aria-describedby="CR13">13</a></sup>. In contrast, the order Esociformes did not undergo further genome duplication events and have retained their ancestral karyotype. Consequently, they together serve as a model system for assessing the impacts of whole-genome duplication in the vertebrate evolution<sup><a href="#CR14" class="usa-link" aria-describedby="CR14">14</a>,<a href="#CR15" class="usa-link" aria-describedby="CR15">15</a></sup>.</p>
<p id="Par4">Remarkable progress has been made in the genomics of salmonids in the last few decades<sup><a href="#CR10" class="usa-link" aria-describedby="CR10">10</a></sup>. However, the pace of genomic research on the order Esociformes is extremely slow. Within the order Esociformes, a chromosome-level genome is currently available only for the northern pike (<em>Esox lucius</em>). The distribution of northern pike almost encircles the entire Arctic region, spanning the high-latitude areas of Eurasia and extending all the way to North America. Previous studies have indicated that some populations of northern pike follow the ZW sex-determination system, while other populations appear to be more in line with environmental sex determination<sup><a href="#CR16" class="usa-link" aria-describedby="CR16">16</a>,<a href="#CR17" class="usa-link" aria-describedby="CR17">17</a></sup>. Almost all the salmonid species investigated to date are characterized by a male heterogametic sex-determination system<sup><a href="#CR18" class="usa-link" aria-describedby="CR18">18</a></sup>.</p>
<p id="Par5">In this study, we combined the PacBio long-read sequencing, Illumina short-read sequencing, and Hi-C technology to construct a chromosome-level genome of the Amur pike. This high-quality genome provides valuable genetic resources for the protection and utilization of Amur pike, and serves as a crucial resource for understanding the evolutionary impacts of genome duplication and sex chromosome evolution in vertebrates through comparative genomics with salmonids.</p></section><section id="Sec2"><h2 class="pmc_sec_title">Methods</h2>
<section id="Sec3"><h3 class="pmc_sec_title">Sample collection</h3>
<p id="Par6">An adult male Amur pike was collected from the Urxun river in Hulun Lake (48°57′16″N, 117°45′22″E), which is part of the broader Heilongjiang River system in northeastern China. Muscle tissue from this individual was sampled for genome sequencing, while multiple tissues were mixed for transcriptome sequencing. All samples were immediately frozen and subsequently stored in liquid nitrogen until use. All animal procedures adhered to the guidelines for animal care and use at the Heilongjiang River Fisheries Research Institute (HRFRI), Chinese Academy of Fishery Sciences (CAFS). High-quality genomic DNA was extracted from the muscle using the CTAB method for Illumina sequencing. The quality of the extracted DNA was evaluated through agarose gel electrophoresis and with a NanoDrop Spectrophotometer (Thermo Fisher Scientific, USA). The DNA was quantified using a Qubit Fluorometer (Invitrogen, USA).</p></section><section id="Sec4"><h3 class="pmc_sec_title">Genome survey</h3>
<p id="Par7">To estimate the genome size, heterozygosity level, and repeat content of the Amur pike genome, a short-read (350 bp) library was constructed and sequenced on an Illumina Novaseq. 6000 platform (Illumina, CA, USA). Raw data was processed using fastp<sup><a href="#CR19" class="usa-link" aria-describedby="CR19">19</a></sup> software with the parameters “-q 10 -u 50 -y -g -Y 10 -e 20 -l 100 -b 150 -B 150”. After filtering, 68.60 Gb of clean reads were utilized for estimating genome features. The K-mers (K = 21) were counted using Jellyfish<sup><a href="#CR20" class="usa-link" aria-describedby="CR20">20</a></sup> software, and the genome characteristics were calculated with Genomescope<sup><a href="#CR21" class="usa-link" aria-describedby="CR21">21</a></sup> software. The genome size of the Amur pike was estimated to be approximately 861.55 Mb, with a heterozygosity of 0.30% and repeat sequences accounting for 27.27% (Fig. <a href="#Fig1" class="usa-link">1</a>).</p>
<figure class="fig xbox font-sm" id="Fig1"><h4 class="obj_head">Fig. 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/19e5/12368265/7eb566fbffad/41597_2025_5819_Fig1_HTML.jpg" loading="lazy" id="d33e308" height="637" width="632" alt="Fig. 1"></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>The k-mer distribution used to estimate the genome size of Amur pike (<em>Esox reicherti</em>). The distribution was determined based on the Jellyfish analysis using a k-mer size of 21.</p></figcaption></figure><p id="Par8">The difference between the k-mer estimated genome size (861.55 Mb) and the final assembly size (976.47 Mb) can be attributed to the factor that K-mer analysis may underestimate genome size in the presence of highly repetitive sequences, which constitute 37.74% of our assembly.</p></section><section id="Sec5"><h3 class="pmc_sec_title">PacBio Hifi sequencing and assembly</h3>
<p id="Par9">Genomic DNA was fragmented into ~15 kb fragments using the Megaruptor®2. The SMRTbell library was constructed with the SMRTbell Express Template Prep kit 2.0 (Pacific Biosciences, CA, USA), following the manufacturer’s prescribed guidelines. Subsequently, the constructed SMRTbell libraries were sequenced on a PacBio Sequel II system (Pacific Biosciences, CA, USA). As a consequence, a total of 28.9 Gb of PacBio CCS reads were generated for the ensuing genome assembly. The average read length of the subreads sequences was 18,897 bp, and the N50 length was 19,545 bp. High-accuracy CCS data were assembled using hifiasm (v 0.16)<sup><a href="#CR22" class="usa-link" aria-describedby="CR22">22</a></sup> software with the parameters “l = 2, n = 4”. The resulting genome length was approximately 976.47 Mb, consisting of 224 contigs, with a contig N50 of 30.76 Mb.</p></section><section id="Sec6"><h3 class="pmc_sec_title">Hi-C technology help anchor contigs</h3>
<p id="Par10">The Hi-C library was prepared following a previously reported procedure<sup><a href="#CR23" class="usa-link" aria-describedby="CR23">23</a></sup>. The librariy underwent quality control tests using a Qubit 2.0 (Invitrogen, USA), an Agilent 2100 instrument (Agilent Technologies, CA, USA), and q-PCR. Subsequently, 150 bp paired-end (PE) sequencing of the Hi-C library was carried out on the Illumina NovaSeq. 6000 platform. In total, 94.54 Gb of Hi-C read data was obtained. For anchoring contigs, 316,337,296 clean read pairs were generated from the Hi-C library. These reads were mapped to the contig genome using BWA (bwa-0.7.17)<sup><a href="#CR24" class="usa-link" aria-describedby="CR24">24</a></sup> with default parameters, and 270,362,164 read pairs were unique mapped. Among them, 158,990,406 (58.81%) paired reads with mates mapped to different contigs were utilized for Hi-C associated scaffolding. Self-ligation, non-ligation, and other invalid reads were filtered out. Next, we employed the agglomerative hierarchical clustering method in Lachesis<sup><a href="#CR25" class="usa-link" aria-describedby="CR25">25</a></sup> to cluster the contigs. As a result, 155 (69.2%) contigs, with a total length of 962.35 Mb (98.55%), were successfully clustered into 25 groups (Table <a href="#Tab1" class="usa-link">1</a> and Fig. <a href="#Fig2" class="usa-link">2</a>). Lachesis was further used to order and orient these clustered contigs. Eventually, 71 contigs, with a combined length of 948.05 Mb (98.51%), were successfully ordered and oriented. Finally, we obtained a high-quality chromosomal-level assembly for Amur pike, with chromosomal lengths ranging from 22.85 Mb to 56.71 Mb (Table <a href="#Tab1" class="usa-link">1</a> and Fig. <a href="#Fig3" class="usa-link">3</a>).</p>
<section class="tw xbox font-sm" id="Tab1"><h4 class="obj_head">Table 1.</h4>
<div class="caption p"><p>Statistical results of the 25 pseudochromosomes of Amur pike (<em>Esox reicherti</em>) genome.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th colspan="1" rowspan="1">Chromosome</th>
<th colspan="1" rowspan="1">Cluster Num</th>
<th colspan="1" rowspan="1">Cluster Len</th>
<th colspan="1" rowspan="1">Order Num</th>
<th colspan="1" rowspan="1">Order Len</th>
</tr></thead>
<tbody>
<tr>
<td colspan="1" rowspan="1">Chr01</td>
<td colspan="1" rowspan="1">3</td>
<td colspan="1" rowspan="1">45,398,629</td>
<td colspan="1" rowspan="1">1</td>
<td colspan="1" rowspan="1">45,016,340</td>
</tr>
<tr>
<td colspan="1" rowspan="1">Chr02</td>
<td colspan="1" rowspan="1">4</td>
<td colspan="1" rowspan="1">41,779,487</td>
<td colspan="1" rowspan="1">2</td>
<td colspan="1" rowspan="1">41,475,729</td>
</tr>
<tr>
<td colspan="1" rowspan="1">Chr03</td>
<td colspan="1" rowspan="1">2</td>
<td colspan="1" rowspan="1">37,573,492</td>
<td colspan="1" rowspan="1">1</td>
<td colspan="1" rowspan="1">37,504,393</td>
</tr>
<tr>
<td colspan="1" rowspan="1">Chr04</td>
<td colspan="1" rowspan="1">4</td>
<td colspan="1" rowspan="1">37,148,612</td>
<td colspan="1" rowspan="1">3</td>
<td colspan="1" rowspan="1">36,912,036</td>
</tr>
<tr>
<td colspan="1" rowspan="1">Chr05</td>
<td colspan="1" rowspan="1">3</td>
<td colspan="1" rowspan="1">40,452,724</td>
<td colspan="1" rowspan="1">3</td>
<td colspan="1" rowspan="1">40,452,724</td>
</tr>
<tr>
<td colspan="1" rowspan="1">Chr06</td>
<td colspan="1" rowspan="1">5</td>
<td colspan="1" rowspan="1">32,637,212</td>
<td colspan="1" rowspan="1">4</td>
<td colspan="1" rowspan="1">32,282,650</td>
</tr>
<tr>
<td colspan="1" rowspan="1">Chr07</td>
<td colspan="1" rowspan="1">7</td>
<td colspan="1" rowspan="1">51,378,845</td>
<td colspan="1" rowspan="1">2</td>
<td colspan="1" rowspan="1">51,066,231</td>
</tr>
<tr>
<td colspan="1" rowspan="1">Chr08</td>
<td colspan="1" rowspan="1">3</td>
<td colspan="1" rowspan="1">39,693,843</td>
<td colspan="1" rowspan="1">3</td>
<td colspan="1" rowspan="1">39,693,843</td>
</tr>
<tr>
<td colspan="1" rowspan="1">Chr09</td>
<td colspan="1" rowspan="1">5</td>
<td colspan="1" rowspan="1">26,700,867</td>
<td colspan="1" rowspan="1">2</td>
<td colspan="1" rowspan="1">26,534,176</td>
</tr>
<tr>
<td colspan="1" rowspan="1">Chr10</td>
<td colspan="1" rowspan="1">5</td>
<td colspan="1" rowspan="1">31,930,511</td>
<td colspan="1" rowspan="1">3</td>
<td colspan="1" rowspan="1">31,549,898</td>
</tr>
<tr>
<td colspan="1" rowspan="1">Chr11</td>
<td colspan="1" rowspan="1">3</td>
<td colspan="1" rowspan="1">57,065,801</td>
<td colspan="1" rowspan="1">2</td>
<td colspan="1" rowspan="1">56,714,953</td>
</tr>
<tr>
<td colspan="1" rowspan="1">Chr12</td>
<td colspan="1" rowspan="1">2</td>
<td colspan="1" rowspan="1">38,771,796</td>
<td colspan="1" rowspan="1">1</td>
<td colspan="1" rowspan="1">38,734,141</td>
</tr>
<tr>
<td colspan="1" rowspan="1">Chr13</td>
<td colspan="1" rowspan="1">7</td>
<td colspan="1" rowspan="1">42,272,170</td>
<td colspan="1" rowspan="1">4</td>
<td colspan="1" rowspan="1">41,915,782</td>
</tr>
<tr>
<td colspan="1" rowspan="1">Chr14</td>
<td colspan="1" rowspan="1">8</td>
<td colspan="1" rowspan="1">39,942,557</td>
<td colspan="1" rowspan="1">4</td>
<td colspan="1" rowspan="1">39,172,573</td>
</tr>
<tr>
<td colspan="1" rowspan="1">Chr15</td>
<td colspan="1" rowspan="1">8</td>
<td colspan="1" rowspan="1">37,630,798</td>
<td colspan="1" rowspan="1">4</td>
<td colspan="1" rowspan="1">37,204,254</td>
</tr>
<tr>
<td colspan="1" rowspan="1">Chr16</td>
<td colspan="1" rowspan="1">4</td>
<td colspan="1" rowspan="1">35,876,482</td>
<td colspan="1" rowspan="1">4</td>
<td colspan="1" rowspan="1">35,876,482</td>
</tr>
<tr>
<td colspan="1" rowspan="1">Chr17</td>
<td colspan="1" rowspan="1">8</td>
<td colspan="1" rowspan="1">52,326,868</td>
<td colspan="1" rowspan="1">2</td>
<td colspan="1" rowspan="1">51,399,179</td>
</tr>
<tr>
<td colspan="1" rowspan="1">Chr18</td>
<td colspan="1" rowspan="1">3</td>
<td colspan="1" rowspan="1">29,546,056</td>
<td colspan="1" rowspan="1">2</td>
<td colspan="1" rowspan="1">29,423,777</td>
</tr>
<tr>
<td colspan="1" rowspan="1">Chr19</td>
<td colspan="1" rowspan="1">2</td>
<td colspan="1" rowspan="1">48,368,597</td>
<td colspan="1" rowspan="1">1</td>
<td colspan="1" rowspan="1">48,175,596</td>
</tr>
<tr>
<td colspan="1" rowspan="1">Chr20</td>
<td colspan="1" rowspan="1">12</td>
<td colspan="1" rowspan="1">49,806,135</td>
<td colspan="1" rowspan="1">2</td>
<td colspan="1" rowspan="1">48,572,533</td>
</tr>
<tr>
<td colspan="1" rowspan="1">Chr21</td>
<td colspan="1" rowspan="1">32</td>
<td colspan="1" rowspan="1">35,885,350</td>
<td colspan="1" rowspan="1">4</td>
<td colspan="1" rowspan="1">33,022,564</td>
</tr>
<tr>
<td colspan="1" rowspan="1">Chr22</td>
<td colspan="1" rowspan="1">5</td>
<td colspan="1" rowspan="1">26,373,932</td>
<td colspan="1" rowspan="1">3</td>
<td colspan="1" rowspan="1">25,596,888</td>
</tr>
<tr>
<td colspan="1" rowspan="1">Chr23</td>
<td colspan="1" rowspan="1">6</td>
<td colspan="1" rowspan="1">25,511,925</td>
<td colspan="1" rowspan="1">4</td>
<td colspan="1" rowspan="1">25,294,521</td>
</tr>
<tr>
<td colspan="1" rowspan="1">Chr24</td>
<td colspan="1" rowspan="1">12</td>
<td colspan="1" rowspan="1">35,426,624</td>
<td colspan="1" rowspan="1">8</td>
<td colspan="1" rowspan="1">31,613,508</td>
</tr>
<tr>
<td colspan="1" rowspan="1">Chr25</td>
<td colspan="1" rowspan="1">2</td>
<td colspan="1" rowspan="1">22,845,700</td>
<td colspan="1" rowspan="1">2</td>
<td colspan="1" rowspan="1">22,845,700</td>
</tr>
<tr>
<td colspan="1" rowspan="1">Total(Ratio %)</td>
<td colspan="1" rowspan="1">155(69.20)</td>
<td colspan="1" rowspan="1">962,345,013(98.55)</td>
<td colspan="1" rowspan="1">71(45.81)</td>
<td colspan="1" rowspan="1">948,050,471(98.51)</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="_fn_p13"><p>Cluster Num: Number of contigs clustered/assigned to each chromosome; Cluster Len: Total length of sequences clustered to each chromosome; Order Num: Number of contigs that were successfully ordered and oriented within each chromosome; Order Len: Total length of successfully ordered and oriented sequences.</p></div></div></section><figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12368265_41597_2025_5819_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/19e5/12368265/ace8180cbec2/41597_2025_5819_Fig2_HTML.jpg" loading="lazy" id="d33e676" height="762" width="760" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Hi-C intrachromosomal contact map of genome assembly for Amur pike (<em>Esox reicherti</em>). The color bar shows the intensity of interaction from low (white) to high (red).</p></figcaption></figure><figure class="fig xbox font-sm" id="Fig3"><h4 class="obj_head">Fig. 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12368265_41597_2025_5819_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/19e5/12368265/13f62c1179eb/41597_2025_5819_Fig3_HTML.jpg" loading="lazy" id="d33e699" height="652" width="653" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Circos plot of Amur pike (<em>Esox reicherti</em>) genome assembly. The tracks from outside to inside are (<strong>a</strong>) 25 pseudo-chromosomes, (<strong>b</strong>) gene density, (<strong>c</strong>) repeat density, (<strong>d</strong>) GC content and genome collinearity.</p></figcaption></figure></section><section id="Sec7"><h3 class="pmc_sec_title">Annotation of repetitive sequences</h3>
<p id="Par11">Transposon elements (TEs) were identified through a combination of homology-based and de novo approaches. RepeatModeler (<a href="http://www.repeatmasker.org/RepeatModeler/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://www.repeatmasker.org/RepeatModeler/</a>)<sup><a href="#CR26" class="usa-link" aria-describedby="CR26">26</a></sup> was used to detect and classify repetitive sequences in the genome assembly. It can automatically run two de novo repeat-finding programs, namely RECON<sup><a href="#CR27" class="usa-link" aria-describedby="CR27">27</a></sup> and RepeatScout<sup><a href="#CR28" class="usa-link" aria-describedby="CR28">28</a></sup>. Subsequently, full-length long terminal repeat retrotransposons (fl-LTR-RTs) were identified using both LTRharvest (v1.5.9)<sup><a href="#CR29" class="usa-link" aria-describedby="CR29">29</a></sup> and LTR_finder (v2.8)<sup><a href="#CR29" class="usa-link" aria-describedby="CR29">29</a></sup>. Then, LTR_retriever<sup><a href="#CR30" class="usa-link" aria-describedby="CR30">30</a></sup> was employed to generate a high-quality intact fl-LTR-RTs set and a non-redundant LTR library. A non-redundant species-specific TE library was constructed by integrating the de novo TE sequences library mentioned above with the known Dfam (v3.5)<sup><a href="#CR31" class="usa-link" aria-describedby="CR31">31</a></sup> database. For homology search, RepeatMasker (v4.1.2)<sup><a href="#CR32" class="usa-link" aria-describedby="CR32">32</a></sup> was used to identify the known TEs of the Amur pike genome in the Repbase TE library. Tandem repeats were annotated by Tandem Repeats Finder (TRF 409)<sup><a href="#CR33" class="usa-link" aria-describedby="CR33">33</a></sup> and MIcroSAtellite identification tool (MISA v2.1)<sup><a href="#CR34" class="usa-link" aria-describedby="CR34">34</a></sup>. The results showed that the size of the genome’s repetitive sequences was 368.55 Mb, which accounted for 37.74% of the assembled genome. Among these repetitive sequences, transposon elements (TEs) constituted 29.03% of the genome size, while tandem repeat sequences accounted for 8.71% (Table <a href="#Tab2" class="usa-link">2</a>). The detailed results are presented in Table <a href="#Tab2" class="usa-link">2</a>.</p>
<section class="tw xbox font-sm" id="Tab2"><h4 class="obj_head">Table 2.</h4>
<div class="caption p"><p>Classification statistics of repeated sequences in the genome of Amur pike (<em>Esox reicherti</em>).</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th colspan="1" rowspan="1">type</th>
<th colspan="1" rowspan="1">Number</th>
<th colspan="1" rowspan="1">Length</th>
<th colspan="1" rowspan="1">Rate(%)</th>
</tr></thead>
<tbody>
<tr><td colspan="4" rowspan="1">transposable elements</td></tr>
<tr>
<td colspan="1" rowspan="1"> ClassI:Retroelement</td>
<td colspan="1" rowspan="1">367,286</td>
<td colspan="1" rowspan="1">105,908,683</td>
<td colspan="1" rowspan="1">10.85</td>
</tr>
<tr>
<td colspan="1" rowspan="1"> ClassI/DIRS</td>
<td colspan="1" rowspan="1">1,535</td>
<td colspan="1" rowspan="1">466,330</td>
<td colspan="1" rowspan="1">0.05</td>
</tr>
<tr>
<td colspan="1" rowspan="1"> ClassI/LINE</td>
<td colspan="1" rowspan="1">139,913</td>
<td colspan="1" rowspan="1">47,424,552</td>
<td colspan="1" rowspan="1">4.86</td>
</tr>
<tr>
<td colspan="1" rowspan="1"> ClassI/LTR/Copia</td>
<td colspan="1" rowspan="1">130</td>
<td colspan="1" rowspan="1">111,448</td>
<td colspan="1" rowspan="1">0.01</td>
</tr>
<tr>
<td colspan="1" rowspan="1"> ClassI/LTR/ERV</td>
<td colspan="1" rowspan="1">23,485</td>
<td colspan="1" rowspan="1">2,803,292</td>
<td colspan="1" rowspan="1">0.29</td>
</tr>
<tr>
<td colspan="1" rowspan="1"> ClassI/LTR/Gypsy</td>
<td colspan="1" rowspan="1">55,103</td>
<td colspan="1" rowspan="1">22,148,164</td>
<td colspan="1" rowspan="1">2.27</td>
</tr>
<tr>
<td colspan="1" rowspan="1"> ClassI/LTR/Ngaro</td>
<td colspan="1" rowspan="1">3,047</td>
<td colspan="1" rowspan="1">280,342</td>
<td colspan="1" rowspan="1">0.03</td>
</tr>
<tr>
<td colspan="1" rowspan="1"> ClassI/LTR/Pao</td>
<td colspan="1" rowspan="1">1,698</td>
<td colspan="1" rowspan="1">586,351</td>
<td colspan="1" rowspan="1">0.06</td>
</tr>
<tr>
<td colspan="1" rowspan="1"> ClassI/LTR/Unknown</td>
<td colspan="1" rowspan="1">128,001</td>
<td colspan="1" rowspan="1">30,531,488</td>
<td colspan="1" rowspan="1">3.13</td>
</tr>
<tr>
<td colspan="1" rowspan="1"> ClassI/SINE</td>
<td colspan="1" rowspan="1">14,374</td>
<td colspan="1" rowspan="1">1,556,716</td>
<td colspan="1" rowspan="1">0.16</td>
</tr>
<tr>
<td colspan="1" rowspan="1"> ClassII:DNA transposon</td>
<td colspan="1" rowspan="1">622,420</td>
<td colspan="1" rowspan="1">177,437,046</td>
<td colspan="1" rowspan="1">18.17</td>
</tr>
<tr>
<td colspan="1" rowspan="1"> ClassII/Academ</td>
<td colspan="1" rowspan="1">13</td>
<td colspan="1" rowspan="1">859</td>
<td colspan="1" rowspan="1">0</td>
</tr>
<tr>
<td colspan="1" rowspan="1"> ClassII/CACTA</td>
<td colspan="1" rowspan="1">15,017</td>
<td colspan="1" rowspan="1">1,650,774</td>
<td colspan="1" rowspan="1">0.17</td>
</tr>
<tr>
<td colspan="1" rowspan="1"> ClassII/Crypton</td>
<td colspan="1" rowspan="1">583</td>
<td colspan="1" rowspan="1">180,511</td>
<td colspan="1" rowspan="1">0.02</td>
</tr>
<tr>
<td colspan="1" rowspan="1"> ClassII/Dada</td>
<td colspan="1" rowspan="1">888</td>
<td colspan="1" rowspan="1">66,768</td>
<td colspan="1" rowspan="1">0.01</td>
</tr>
<tr>
<td colspan="1" rowspan="1"> ClassII/Ginger</td>
<td colspan="1" rowspan="1">1,365</td>
<td colspan="1" rowspan="1">143,271</td>
<td colspan="1" rowspan="1">0.01</td>
</tr>
<tr>
<td colspan="1" rowspan="1"> ClassII/Helitron</td>
<td colspan="1" rowspan="1">192,328</td>
<td colspan="1" rowspan="1">43,561,209</td>
<td colspan="1" rowspan="1">4.46</td>
</tr>
<tr>
<td colspan="1" rowspan="1"> ClassII/IS3EU</td>
<td colspan="1" rowspan="1">1,614</td>
<td colspan="1" rowspan="1">170,267</td>
<td colspan="1" rowspan="1">0.02</td>
</tr>
<tr>
<td colspan="1" rowspan="1"> ClassII/Kolobok</td>
<td colspan="1" rowspan="1">1,394</td>
<td colspan="1" rowspan="1">156,681</td>
<td colspan="1" rowspan="1">0.02</td>
</tr>
<tr>
<td colspan="1" rowspan="1"> ClassII/Maverick</td>
<td colspan="1" rowspan="1">8,922</td>
<td colspan="1" rowspan="1">4,742,538</td>
<td colspan="1" rowspan="1">0.49</td>
</tr>
<tr>
<td colspan="1" rowspan="1"> ClassII/Merlin</td>
<td colspan="1" rowspan="1">1,059</td>
<td colspan="1" rowspan="1">130,008</td>
<td colspan="1" rowspan="1">0.01</td>
</tr>
<tr>
<td colspan="1" rowspan="1"> ClassII/Mutator</td>
<td colspan="1" rowspan="1">760</td>
<td colspan="1" rowspan="1">62,240</td>
<td colspan="1" rowspan="1">0.01</td>
</tr>
<tr>
<td colspan="1" rowspan="1"> ClassII/P</td>
<td colspan="1" rowspan="1">1,903</td>
<td colspan="1" rowspan="1">523,515</td>
<td colspan="1" rowspan="1">0.05</td>
</tr>
<tr>
<td colspan="1" rowspan="1"> ClassII/PIF-Harbinger</td>
<td colspan="1" rowspan="1">5,850</td>
<td colspan="1" rowspan="1">1,451,413</td>
<td colspan="1" rowspan="1">0.15</td>
</tr>
<tr>
<td colspan="1" rowspan="1"> ClassII/PiggyBac</td>
<td colspan="1" rowspan="1">1,113</td>
<td colspan="1" rowspan="1">204,198</td>
<td colspan="1" rowspan="1">0.02</td>
</tr>
<tr>
<td colspan="1" rowspan="1"> ClassII/Sola</td>
<td colspan="1" rowspan="1">106</td>
<td colspan="1" rowspan="1">18,123</td>
<td colspan="1" rowspan="1">0</td>
</tr>
<tr>
<td colspan="1" rowspan="1"> ClassII/Tc1-Mariner</td>
<td colspan="1" rowspan="1">220,203</td>
<td colspan="1" rowspan="1">93,345,802</td>
<td colspan="1" rowspan="1">9.56</td>
</tr>
<tr>
<td colspan="1" rowspan="1"> ClassII/Unknown</td>
<td colspan="1" rowspan="1">113,860</td>
<td colspan="1" rowspan="1">20,249,011</td>
<td colspan="1" rowspan="1">2.07</td>
</tr>
<tr>
<td colspan="1" rowspan="1"> ClassII/Zator</td>
<td colspan="1" rowspan="1">7</td>
<td colspan="1" rowspan="1">1,091</td>
<td colspan="1" rowspan="1">0</td>
</tr>
<tr>
<td colspan="1" rowspan="1"> ClassII/Zisupton</td>
<td colspan="1" rowspan="1">1,224</td>
<td colspan="1" rowspan="1">72,742</td>
<td colspan="1" rowspan="1">0.01</td>
</tr>
<tr>
<td colspan="1" rowspan="1"> ClassII/hAT</td>
<td colspan="1" rowspan="1">54,211</td>
<td colspan="1" rowspan="1">10,706,025</td>
<td colspan="1" rowspan="1">1.1</td>
</tr>
<tr>
<td colspan="1" rowspan="1"> Unknown</td>
<td colspan="1" rowspan="1">356</td>
<td colspan="1" rowspan="1">118,159</td>
<td colspan="1" rowspan="1">0.01</td>
</tr>
<tr>
<td colspan="1" rowspan="1"> srpRNA</td>
<td colspan="1" rowspan="1">33</td>
<td colspan="1" rowspan="1">9,719</td>
<td colspan="1" rowspan="1">0</td>
</tr>
<tr>
<td colspan="1" rowspan="1"> Total</td>
<td colspan="1" rowspan="1">990095</td>
<td colspan="1" rowspan="1">283473607</td>
<td colspan="1" rowspan="1">29.03</td>
</tr>
<tr><td colspan="4" rowspan="1">tandem repeats</td></tr>
<tr>
<td colspan="1" rowspan="1"> microsatellite(1–9 bp units)</td>
<td colspan="1" rowspan="1">294653</td>
<td colspan="1" rowspan="1">6700080</td>
<td colspan="1" rowspan="1">0.69</td>
</tr>
<tr>
<td colspan="1" rowspan="1"> minisatellite(10–99 bp units)</td>
<td colspan="1" rowspan="1">255708</td>
<td colspan="1" rowspan="1">52303660</td>
<td colspan="1" rowspan="1">5.36</td>
</tr>
<tr>
<td colspan="1" rowspan="1"> satellite(&gt; = 100 bp units)</td>
<td colspan="1" rowspan="1">17978</td>
<td colspan="1" rowspan="1">26071268</td>
<td colspan="1" rowspan="1">2.67</td>
</tr>
<tr>
<td colspan="1" rowspan="1"> Total</td>
<td colspan="1" rowspan="1">568339</td>
<td colspan="1" rowspan="1">85075008</td>
<td colspan="1" rowspan="1">8.71</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section></section><section id="Sec8"><h3 class="pmc_sec_title">Annotation of protein coding gene</h3>
<p id="Par12">Protein-coding genes in the genome assembly of Amur pike were predicted using a comprehensive strategy that combines <em>ab initio</em> prediction, homology-based searches, and transcriptome-assisted annotation. For <em>ab initio</em> prediction, Augustus (v3.2.3)<sup><a href="#CR35" class="usa-link" aria-describedby="CR35">35</a></sup> and SNAP (2013-11-29)<sup><a href="#CR36" class="usa-link" aria-describedby="CR36">36</a></sup> were utilized in our automated gene prediction pipeline. In the homology-based approach, the GeMoMa (v1.7)<sup><a href="#CR37" class="usa-link" aria-describedby="CR37">37</a></sup> software was employed, using reference gene models from five other species (<em>Coregonus clupeaformis</em>, <em>Danio Rerio</em>, <em>Esox lucius</em>, <em>Oncorhynchus mykiss</em>, <em>Salmo salar</em>). For transcriptome-assisted prediction, we conducted RNA-Seq and Iso-Seq on the total RNA extracted from ten tissues, including skin, heart, liver, fins, brain, spleen, kidneys, testis, muscles, and gills. The total RNA was extracted using the TRIzol reagent. Subsequently, RNA-Seq and Iso-Seq were sequenced on the Illumina Novaseq. 6000 platform (Illumina, CA, USA) and the PacBio Sequel II platform (Pacific Biosciences, CA, USA) respectively, following the protocols provided by the manufacturers. A total of 17.79 Gb of RNA-seq data and 64.59 Gb of clean Iso-Seq data were generated, and these data were then used for whole-genome protein-coding gene prediction. The RNA-seq data were mapped to the reference genome using Hisat (v2.1.0)<sup><a href="#CR38" class="usa-link" aria-describedby="CR38">38</a></sup> and assembled by Stringtie (v2.1.4)<sup><a href="#CR39" class="usa-link" aria-describedby="CR39">39</a></sup>. GeneMarkS-T (v5.1)<sup><a href="#CR40" class="usa-link" aria-describedby="CR40">40</a></sup> was used to predict genes based on the assembled transcripts. The PASA (v2.4.1)<sup><a href="#CR41" class="usa-link" aria-describedby="CR41">41</a></sup> software was used to predict genes based on the unigenes and full-length transcripts from the PacBio (ONT) sequencing assembled by Trinity (v2.11)<sup><a href="#CR42" class="usa-link" aria-describedby="CR42">42</a></sup>. Gene models obtained from these different approaches were combined using the EVM software (v1.1.1)<sup><a href="#CR43" class="usa-link" aria-describedby="CR43">43</a></sup> and updated by PASA (v2.4.1)<sup><a href="#CR41" class="usa-link" aria-describedby="CR41">41</a></sup>. Finally, 25,824 protein-coding genes with an average length of 17,504.83 bp were predicted in the assembled Amur pike genome (Table <a href="#Tab3" class="usa-link">3</a>).</p>
<section class="tw xbox font-sm" id="Tab3"><h4 class="obj_head">Table 3.</h4>
<div class="caption p"><p>Summary of the functional gene annotation of Amur pike (<em>Esox reicherti</em>).</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th colspan="1" rowspan="1">Method</th>
<th colspan="1" rowspan="1">Software</th>
<th colspan="1" rowspan="1">Species</th>
<th colspan="1" rowspan="1">Gene number</th>
</tr></thead>
<tbody>
<tr>
<td rowspan="2" colspan="1"><em>Ab initio</em></td>
<td colspan="1" rowspan="1">Augustus</td>
<td colspan="1" rowspan="1">—</td>
<td colspan="1" rowspan="1">34,874</td>
</tr>
<tr>
<td colspan="1" rowspan="1">SNAP</td>
<td colspan="1" rowspan="1">—</td>
<td colspan="1" rowspan="1">67,859</td>
</tr>
<tr>
<td rowspan="5" colspan="1">Homology-based</td>
<td rowspan="5" colspan="1">GeMoMa</td>
<td colspan="1" rowspan="1"><em>Coregonus clupeaformis</em></td>
<td colspan="1" rowspan="1">24,270</td>
</tr>
<tr>
<td colspan="1" rowspan="1"><em>Danio Rerio</em></td>
<td colspan="1" rowspan="1">22,558</td>
</tr>
<tr>
<td colspan="1" rowspan="1"><em>Esox lucius</em></td>
<td colspan="1" rowspan="1">24,788</td>
</tr>
<tr>
<td colspan="1" rowspan="1"><em>Oncorhynchus mykiss</em></td>
<td colspan="1" rowspan="1">23,826</td>
</tr>
<tr>
<td colspan="1" rowspan="1"><em>Salmo salar</em></td>
<td colspan="1" rowspan="1">24,393</td>
</tr>
<tr>
<td rowspan="2" colspan="1">RNAseq</td>
<td colspan="1" rowspan="1">GeneMarkS-T</td>
<td colspan="1" rowspan="1">—</td>
<td colspan="1" rowspan="1">23,351</td>
</tr>
<tr>
<td colspan="1" rowspan="1">PASA</td>
<td colspan="1" rowspan="1">—</td>
<td colspan="1" rowspan="1">14,636</td>
</tr>
<tr>
<td colspan="1" rowspan="1">Integration</td>
<td colspan="1" rowspan="1">EVM</td>
<td colspan="1" rowspan="1">—</td>
<td colspan="1" rowspan="1">25,824</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section></section><section id="Sec9"><h3 class="pmc_sec_title">Functional annotation of protein-coding genes</h3>
<p id="Par13">Gene functions were inferred according to the best match of the alignments to the National Center for Biotechnology Information (NCBI) non-redundant proteins (NR)<sup><a href="#CR44" class="usa-link" aria-describedby="CR44">44</a></sup>, EggNOG<sup><a href="#CR45" class="usa-link" aria-describedby="CR45">45</a></sup>, KOG<sup><a href="#CR46" class="usa-link" aria-describedby="CR46">46</a></sup>, TrEMBL<sup><a href="#CR47" class="usa-link" aria-describedby="CR47">47</a></sup>, Gene Ontology (GO)<sup><a href="#CR48" class="usa-link" aria-describedby="CR48">48</a></sup> and the Kyoto Encyclopedia of Genes and Genomes (KEGG) database<sup><a href="#CR49" class="usa-link" aria-describedby="CR49">49</a></sup> using diamond blastp (diamond v2.0.4.142)<sup><a href="#CR50" class="usa-link" aria-describedby="CR50">50</a></sup> with an E-value threshold of 1E-5. The motifs and domains within gene models were identified by PFAM databases<sup><a href="#CR51" class="usa-link" aria-describedby="CR51">51</a></sup>. In total, 25,515 (about 98.8%) of the predicted protein-coding genes of Amur pike could be functionally annotated with known genes, conserved domains, and Gene Ontology terms (Table <a href="#Tab4" class="usa-link">4</a>).</p>
<section class="tw xbox font-sm" id="Tab4"><h4 class="obj_head">Table 4.</h4>
<div class="caption p"><p>Statistics of functional annotation of protein-coding genes.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th colspan="1" rowspan="1">Anno_Database</th>
<th colspan="1" rowspan="1">Annotated_Number</th>
<th colspan="1" rowspan="1">Annotated_Ratio</th>
</tr></thead>
<tbody>
<tr>
<td colspan="1" rowspan="1">nr_Annotation</td>
<td colspan="1" rowspan="1">25,152</td>
<td colspan="1" rowspan="1">97.4</td>
</tr>
<tr>
<td colspan="1" rowspan="1">eggNOG_Annotation</td>
<td colspan="1" rowspan="1">22,713</td>
<td colspan="1" rowspan="1">87.95</td>
</tr>
<tr>
<td colspan="1" rowspan="1">KOG_Annotation</td>
<td colspan="1" rowspan="1">17,062</td>
<td colspan="1" rowspan="1">66.07</td>
</tr>
<tr>
<td colspan="1" rowspan="1">TrEMBL_Annotation</td>
<td colspan="1" rowspan="1">25,428</td>
<td colspan="1" rowspan="1">98.47</td>
</tr>
<tr>
<td colspan="1" rowspan="1">GO_Annotation</td>
<td colspan="1" rowspan="1">22,704</td>
<td colspan="1" rowspan="1">87.92</td>
</tr>
<tr>
<td colspan="1" rowspan="1">KEGG_Annotation</td>
<td colspan="1" rowspan="1">22,702</td>
<td colspan="1" rowspan="1">87.91</td>
</tr>
<tr>
<td colspan="1" rowspan="1">Pfam_Annotation</td>
<td colspan="1" rowspan="1">23,679</td>
<td colspan="1" rowspan="1">91.69</td>
</tr>
<tr>
<td colspan="1" rowspan="1">All_Annotated</td>
<td colspan="1" rowspan="1">25,515</td>
<td colspan="1" rowspan="1">98.8</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section></section><section id="Sec10"><h3 class="pmc_sec_title">Annotation of non-coding RNA genes</h3>
<p id="Par14">Five types of noncoding RNA (ncRNA) were identified in the genome of Amur pike, including microRNAs (miRNAs), transfer RNAs (tRNAs), ribosomal RNAs (rRNA), small nuclear RNAs (snRNAs) and small nucleolar RNAs (snoRNAs). The tRNAs were predicted using the tRNAscan-SE program (<a href="http://lowelab.ucsc.edu/tRNAscan-SE/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://lowelab.ucsc.edu/tRNAscan-SE/</a>)<sup><a href="#CR52" class="usa-link" aria-describedby="CR52">52</a></sup>. For rRNA identification, we employed barrnap (v0.9)<sup><a href="#CR53" class="usa-link" aria-describedby="CR53">53</a></sup> with default parameters to identify the genes related to rRNA. Other types of ncRNAs were detected by alignment to Rfam database<sup><a href="#CR54" class="usa-link" aria-describedby="CR54">54</a></sup> using infernal (v1.1.3) software<sup><a href="#CR55" class="usa-link" aria-describedby="CR55">55</a></sup>. In total, 76,281 non-coding RNAs (ncRNAs) were annotated, including 50,923 rRNAs, 20,722 tRNAs, 419 miRNAs, 3,644 snRNAs, and 573 snoRNAs. Previous research has demonstrated that the <em>Esox</em> genome exhibits extreme amplification of 5S rRNA genes, a phenomenon considered among the highest reported copy numbers of rRNA genes in the animal kingdom<sup><a href="#CR56" class="usa-link" aria-describedby="CR56">56</a></sup>. The quantification of rRNA genes in the <em>Esox reicherti</em> genome further corroborates this extreme amplification, solidifying its status as an exceptional case of rDNA proliferation in vertebrates.</p></section></section><section id="Sec11"><h2 class="pmc_sec_title">Data Records</h2>
<p id="Par15">All the raw sequencing data utilized in this study have been deposited in the Genome Sequence Archive (GSA) in National Genomics Data Center (NGDC), China National Center for Bioinformation / Beijing Institute of Genomics, Chinese Academy of Sciences, with accession number CRA024841<sup><a href="#CR57" class="usa-link" aria-describedby="CR57">57</a></sup> under BioProject number PRJCA038565<sup><a href="#CR58" class="usa-link" aria-describedby="CR58">58</a></sup>. The genome assembly has been deposited in both NCBI GenBank under accession number <a href="https://www.ncbi.nlm.nih.gov/nuccore/JBPYXS000000000" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">JBPYXS000000000</a><sup><a href="#CR59" class="usa-link" aria-describedby="CR59">59</a></sup> and Genome Warehouse (GWH) of NGDC under accession GWHGDGV00000000.1<sup><a href="#CR60" class="usa-link" aria-describedby="CR60">60</a></sup>. Genome annotation files (gff3, gene, exon, cds, protein files) have been deposited in the Figshare database<sup><a href="#CR61" class="usa-link" aria-describedby="CR61">61</a>–<a href="#CR65" class="usa-link" aria-describedby="CR65">65</a></sup> Table <a href="#Tab4" class="usa-link">4</a>.</p></section><section id="Sec12"><h2 class="pmc_sec_title">Technical Validation</h2>
<section id="Sec13"><h3 class="pmc_sec_title">Assessment of genome quality</h3>
<p id="Par16">We conducted comprehensive quality assessments of the assembled genome through multiple validation approaches. Remapping of Illumina paired-end clean reads and PacBio long reads to the assembled genome using BWA<sup><a href="#CR24" class="usa-link" aria-describedby="CR24">24</a></sup> and Minimap2<sup><a href="#CR66" class="usa-link" aria-describedby="CR66">66</a></sup> yielded mapping rates of 99.32% and 99.75%, respectively, with their high mapping rates and relatively uniform coverage validating the good completeness of the obtained genome (Table <a href="#Tab5" class="usa-link">5</a>). The completeness of the assembled genome sequence was further assessed with Benchmarking Universal Single-Copy Orthologs (BUSCO, v 5.2.2)<sup><a href="#CR67" class="usa-link" aria-describedby="CR67">67</a></sup> based on the actinopterygii_odb10 database<sup><a href="#CR67" class="usa-link" aria-describedby="CR67">67</a></sup>, which identified 3,480 (95.60%) complete orthologs, including 3,429 (94.20%) single-copy orthologs and 51 (1.40%) duplicated orthologs, as well as 23 (0.63%) fragmented orthologs (Table <a href="#Tab6" class="usa-link">6</a>), supporting the high quality of this assembly. Contamination screening using BlobTools v1.1.1<sup><a href="#CR68" class="usa-link" aria-describedby="CR68">68</a></sup> revealed minimal contamination, with 99.54% of assembled sequences (972.0 MB) correctly classified as Chordata, confirming the high purity of our assembly. Sequence accuracy assessment using Merqury v1.3<sup><a href="#CR69" class="usa-link" aria-describedby="CR69">69</a></sup> yielded a Quality Value (QV) score of 45.13, corresponding to an error rate of 3.0657 × 10<sup>−5</sup>, or approximately 1 error per 32,612 base pairs, indicating exceptionally high sequence accuracy. Merqury completeness analysis confirmed that 98.76% of k-mers from the WGS short reads were present in our final assembly, demonstrating excellent representation of the sequenced genome content with minimal sequence loss during the assembly process. Finally, gene annotation quality was assessed using BUSCO v5.2.2 with the actinopterygii database (3,640 conserved core genes), which showed that 98.54% of BUSCO genes were present in our predicted gene set, indicating high completeness of our gene annotation Fig. <a href="#Fig3" class="usa-link">3</a>.</p>
<section class="tw xbox font-sm" id="Tab5"><h4 class="obj_head">Table 5.</h4>
<div class="caption p"><p>Statistics on the re-mapping rate and coverage of next-generation and third-generation sequencing data.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th colspan="1" rowspan="1"></th>
<th colspan="1" rowspan="1">Total reads</th>
<th colspan="1" rowspan="1">Mapped reads</th>
<th colspan="1" rowspan="1">Mapped Rate (%)</th>
<th colspan="1" rowspan="1">Coverage (≥5X)</th>
<th colspan="1" rowspan="1">Coverage (≥10X)</th>
<th colspan="1" rowspan="1">Coverage (≥20X)</th>
</tr></thead>
<tbody>
<tr>
<td colspan="1" rowspan="1">illumina</td>
<td colspan="1" rowspan="1">458,886,040</td>
<td colspan="1" rowspan="1">455,747,584</td>
<td colspan="1" rowspan="1">99.32</td>
<td colspan="1" rowspan="1">99.45</td>
<td colspan="1" rowspan="1">98.89</td>
<td colspan="1" rowspan="1">97.61</td>
</tr>
<tr>
<td colspan="1" rowspan="1">PacBio HiFi</td>
<td colspan="1" rowspan="1">1,529,505</td>
<td colspan="1" rowspan="1">1,525,728</td>
<td colspan="1" rowspan="1">99.75</td>
<td colspan="1" rowspan="1">99.68</td>
<td colspan="1" rowspan="1">98.52</td>
<td colspan="1" rowspan="1">87.02</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><section class="tw xbox font-sm" id="Tab6"><h4 class="obj_head">Table 6.</h4>
<div class="caption p"><p>BUSCO assessment statistics of de novo assembly.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th colspan="1" rowspan="1">Tota</th>
<th colspan="1" rowspan="1">Number</th>
<th colspan="1" rowspan="1">Percentage (%)</th>
</tr></thead>
<tbody>
<tr>
<td colspan="1" rowspan="1">Complete BUSCOs(C)</td>
<td colspan="1" rowspan="1">3480</td>
<td colspan="1" rowspan="1">95.60</td>
</tr>
<tr>
<td colspan="1" rowspan="1">Complete and single-copy BUSCOs(S)</td>
<td colspan="1" rowspan="1">3429</td>
<td colspan="1" rowspan="1">94.20</td>
</tr>
<tr>
<td colspan="1" rowspan="1">Complete and duplicated BUSCOs(D)</td>
<td colspan="1" rowspan="1">51</td>
<td colspan="1" rowspan="1">1.40</td>
</tr>
<tr>
<td colspan="1" rowspan="1">Fragmented BUSCOs(F)</td>
<td colspan="1" rowspan="1">23</td>
<td colspan="1" rowspan="1">0.63</td>
</tr>
<tr>
<td colspan="1" rowspan="1">Missing BUSCOs(M)</td>
<td colspan="1" rowspan="1">137</td>
<td colspan="1" rowspan="1">3.76</td>
</tr>
<tr>
<td colspan="1" rowspan="1">Total Lineage BUSCOs</td>
<td colspan="1" rowspan="1">3,640</td>
<td colspan="1" rowspan="1"></td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section></section><section id="Sec14"><h3 class="pmc_sec_title">Evaluation of gene synteny and collinearity</h3>
<p id="Par17">Collinearity analysis were Performed with the reference genome of northern pike, which could serve as a means to verify the accuracy of genome assembly. Use diamond (v0.9.29.130)<sup><a href="#CR50" class="usa-link" aria-describedby="CR50">50</a></sup> to align the gene sequences of the two species and identify the pairs of similar genes (e &lt; 1e−5, C score &gt; 0.5). Then, JCVI<sup><a href="#CR70" class="usa-link" aria-describedby="CR70">70</a></sup> software was used to filter the C score and draw the collinearity graph. The genome of the Amur pike obtained in this study has a good collinear relationship with that of the northern pike (<a href="https://www.ncbi.nlm.nih.gov/datasets/genome/GCF_011004845.1/" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/datasets/genome/GCF_011004845.1/</a>), which supports the high quality of the assembly and annotation of this genome (Fig. <a href="#Fig4" class="usa-link">4</a>).</p>
<figure class="fig xbox font-sm" id="Fig4"><h4 class="obj_head">Fig. 4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12368265_41597_2025_5819_Fig4_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/19e5/12368265/8b0c095d2765/41597_2025_5819_Fig4_HTML.jpg" loading="lazy" id="d33e1683" height="125" width="760" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Collinearity analysis of genomes for Amur pike (<em>Esox reicherti</em>) and Northern pike (<em>Esox lucius</em>).</p></figcaption></figure></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>This work was funded by the National Key Research and Development Plan (2023YFD2400903); Hulun Lake Basin Ecological Environmental Protection and Governance, Indigenous Fish Protection and Population Restoration Project (HSZCS-G-F-220173); and the Ministry of Agriculture and Rural Affairs of China Financial Special Project (Regularized Investigation on Fishery Resources and Environment in Key Waters of Northwest China).</p></section><section id="notes1"><h2 class="pmc_sec_title">Author contributions</h2>
<p>Zhiying Jia and Yuyong Zhang conceptualized the research project. Yongquan Zhang and Tangbin Huo conducted the investigation. Ping Zhang and Huiping Zhang collected the samples and participated in the investigation. Gerel Qogt provided strategic oversight and leadership throughout the planning and execution of the research activities. Yuyong Zhang drafted the manuscript. Zhiying Jia revised the manuscript. All authors read and approved the final version of the manuscript.</p></section><section id="notes2"><h2 class="pmc_sec_title">Code availability</h2>
<p>The bioinformatic analysis were executed in accordance with the guidelines and procedures outlined in the manuals of the relevant software. The Methods section contains a detailed description of the software and corresponding parameters. No custom code was used in this process.</p></section><section id="FPar1"><h2 class="pmc_sec_title">Competing interests</h2>
<p id="Par18">The authors declare no competing interests.</p></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="fn1"><p><strong>Publisher’s note</strong> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></div></div></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>López, J. A., Chen, W.-J. &amp; Ortí, G. Esociform Phylogeny. <em>Copeia</em><strong>2004</strong>, 449–464 (2004).</cite> [<a href="https://scholar.google.com/scholar_lookup?L%C3%B3pez,%20J.%20A.,%20Chen,%20W.-J.%20&amp;%20Ort%C3%AD,%20G.%20Esociform%20Phylogeny.%20Copeia2004,%20449%E2%80%93464%20(2004)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>Grande, T., Laten, H. &amp; López, J. A. Phylogenetic Relationships of Extant Esocid Species (Teleostei: Salmoniformes) Based on Morphological and Molecular Characters. <em>Copeia</em><strong>2004</strong>, 743–757 (2004).</cite> [<a href="https://scholar.google.com/scholar_lookup?Grande,%20T.,%20Laten,%20H.%20&amp;%20L%C3%B3pez,%20J.%20A.%20Phylogenetic%20Relationships%20of%20Extant%20Esocid%20Species%20(Teleostei:%20Salmoniformes)%20Based%20on%20Morphological%20and%20Molecular%20Characters.%20Copeia2004,%20743%E2%80%93757%20(2004)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>Eschbach, E. <em>et al</em>. Genetic population structure of a top predatory fish (northern pike, <em>Esox lucius</em>) covaries with anthropogenic alteration of freshwater ecosystems. <em>Freshwater Biology</em><strong>66</strong>, 884–901 (2021).</cite> [<a href="https://scholar.google.com/scholar_lookup?Eschbach,%20E.%20et%20al.%20Genetic%20population%20structure%20of%20a%20top%20predatory%20fish%20(northern%20pike,%20Esox%20lucius)%20covaries%20with%20anthropogenic%20alteration%20of%20freshwater%20ecosystems.%20Freshwater%20Biology66,%20884%E2%80%93901%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Skov, C., &amp; Nilsson, P. A. <em>Biology and Ecology of Pike</em> (CRC Press, 2018).</cite>
</li>
<li id="CR5">
<span class="label">5.</span><cite>Skog, A., Vøllestad, L. A., Stenseth, N. C., Kasumyan, A. &amp; Jakobsen, K. S. Circumpolar phylogeography of the northern pike (<em>Esox lucius</em>) and its relationship to the Amur pike (<em>E. reichertii</em>). <em>Front Zool</em><strong>11</strong>, 67 (2014).</cite> [<a href="https://scholar.google.com/scholar_lookup?Skog,%20A.,%20V%C3%B8llestad,%20L.%20A.,%20Stenseth,%20N.%20C.,%20Kasumyan,%20A.%20&amp;%20Jakobsen,%20K.%20S.%20Circumpolar%20phylogeography%20of%20the%20northern%20pike%20(Esox%20lucius)%20and%20its%20relationship%20to%20the%20Amur%20pike%20(E.%20reichertii).%20Front%20Zool11,%2067%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Zhao, Y., Gozlan, R. E. &amp; Zhang, C. Current state of freshwater fisheries in China. <em>Freshwater Fisheries Ecology</em>, 221–230 (2015).</cite>
</li>
<li id="CR7">
<span class="label">7.</span><cite>Ramsden, S. D., Brinkmann, H., Hawryshyn, C. W. &amp; Taylor, J. S. Mitogenomics and the sister of Salmonidae. <em>Trends in Ecology &amp; Evolution</em><strong>18</strong>, 607–610 (2003).</cite> [<a href="https://scholar.google.com/scholar_lookup?Ramsden,%20S.%20D.,%20Brinkmann,%20H.,%20Hawryshyn,%20C.%20W.%20&amp;%20Taylor,%20J.%20S.%20Mitogenomics%20and%20the%20sister%20of%20Salmonidae.%20Trends%20in%20Ecology%20&amp;%20Evolution18,%20607%E2%80%93610%20(2003)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>Campbell, M. A., López, J. A., Sado, T. &amp; Miya, M. Pike and salmon as sister taxa: Detailed intraclade resolution and divergence time estimation of Esociformes + Salmoniformes based on whole mitochondrial genome sequences. <em>Gene</em><strong>530</strong>, 57–65 (2013).
</cite> [<a href="https://doi.org/10.1016/j.gene.2013.07.068" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23954876/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Campbell,%20M.%20A.,%20L%C3%B3pez,%20J.%20A.,%20Sado,%20T.%20&amp;%20Miya,%20M.%20Pike%20and%20salmon%20as%20sister%20taxa:%20Detailed%20intraclade%20resolution%20and%20divergence%20time%20estimation%20of%20Esociformes%20+%20Salmoniformes%20based%20on%20whole%20mitochondrial%20genome%20sequences.%20Gene530,%2057%E2%80%9365%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>Birnie‐Gauvin, K. <em>et al</em>. Life‐history strategies in salmonids: the role of physiology and its consequences. <em>Biological Reviews</em><strong>96</strong>, 2304–2320 (2021).
</cite> [<a href="https://doi.org/10.1111/brv.12753" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34043292/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Birnie%E2%80%90Gauvin,%20K.%20et%20al.%20Life%E2%80%90history%20strategies%20in%20salmonids:%20the%20role%20of%20physiology%20and%20its%20consequences.%20Biological%20Reviews96,%202304%E2%80%932320%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>Dysin, A. P. <em>et al</em>. Salmonidae Genome: Features, Evolutionary and Phylogenetic Characteristics. <em>Genes</em><strong>13</strong>, 2221 (2022).
</cite> [<a href="https://doi.org/10.3390/genes13122221" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9778375/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36553488/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Dysin,%20A.%20P.%20et%20al.%20Salmonidae%20Genome:%20Features,%20Evolutionary%20and%20Phylogenetic%20Characteristics.%20Genes13,%202221%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Gundappa, M. K. <em>et al</em>. Genome-Wide Reconstruction of Rediploidization Following Autopolyploidization across One Hundred Million Years of Salmonid Evolution. <em>Molecular Biology and Evolution</em><strong>39</strong>, msab310 (2022).
</cite> [<a href="https://doi.org/10.1093/molbev/msab310" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8760942/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34718723/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gundappa,%20M.%20K.%20et%20al.%20Genome-Wide%20Reconstruction%20of%20Rediploidization%20Following%20Autopolyploidization%20across%20One%20Hundred%20Million%20Years%20of%20Salmonid%20Evolution.%20Molecular%20Biology%20and%20Evolution39,%20msab310%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Braasch, I. &amp; Postlethwait, J. H. Polyploidy in Fish and the Teleost Genome Duplication. <em>Polyploidy and Genome Evolution</em> Berlin, Heidelberg: Springer Berlin Heidelberg 341–383 (2012).</cite>
</li>
<li id="CR13">
<span class="label">13.</span><cite>Sutherland, B. J. G. <em>et al</em>. Salmonid chromosome evolution as revealed by a novel method for comparing RADseq linkage maps. <em>Genome Biol Evol</em><strong>8</strong>, 3600–3617 (2016).
</cite> [<a href="https://doi.org/10.1093/gbe/evw262" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5381510/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28173098/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sutherland,%20B.%20J.%20G.%20et%20al.%20Salmonid%20chromosome%20evolution%20as%20revealed%20by%20a%20novel%20method%20for%20comparing%20RADseq%20linkage%20maps.%20Genome%20Biol%20Evol8,%203600%E2%80%933617%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>Leong, J. S. <em>et al</em>. Salmo salar and Esox lucius full-length cDNA sequences reveal changes in evolutionary pressures on a post-tetraploidization genome. <em>BMC Genomics</em><strong>11</strong>, 279 (2010).
</cite> [<a href="https://doi.org/10.1186/1471-2164-11-279" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2886063/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20433749/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Leong,%20J.%20S.%20et%20al.%20Salmo%20salar%20and%20Esox%20lucius%20full-length%20cDNA%20sequences%20reveal%20changes%20in%20evolutionary%20pressures%20on%20a%20post-tetraploidization%20genome.%20BMC%20Genomics11,%20279%20(2010)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>Rondeau, E. B. <em>et al</em>. The Genome and Linkage Map of the Northern Pike (Esox lucius): Conserved Synteny Revealed between the Salmonid Sister Group and the Neoteleostei. <em>PLoS ONE</em><strong>9</strong>, e102089 (2014).
</cite> [<a href="https://doi.org/10.1371/journal.pone.0102089" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4113312/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25069045/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rondeau,%20E.%20B.%20et%20al.%20The%20Genome%20and%20Linkage%20Map%20of%20the%20Northern%20Pike%20(Esox%20lucius):%20Conserved%20Synteny%20Revealed%20between%20the%20Salmonid%20Sister%20Group%20and%20the%20Neoteleostei.%20PLoS%20ONE9,%20e102089%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Johnson, H. A. <em>et al</em>. Loss of genetic variation and ancestral sex determination system in North American northern pike characterized by whole-genome resequencing. <em>G3: Genes, Genomes, Genetics</em><strong>14</strong>, jkae183 (2024).
</cite> [<a href="https://doi.org/10.1093/g3journal/jkae183" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11457062/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39115373/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Johnson,%20H.%20A.%20et%20al.%20Loss%20of%20genetic%20variation%20and%20ancestral%20sex%20determination%20system%20in%20North%20American%20northern%20pike%20characterized%20by%20whole-genome%20resequencing.%20G3:%20Genes,%20Genomes,%20Genetics14,%20jkae183%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>Pan, Q. <em>et al</em>. The rise and fall of the ancient northern pike master sex-determining gene. <em>eLife</em><strong>10</strong>, e62858 (2021).
</cite> [<a href="https://doi.org/10.7554/eLife.62858" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7870143/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33506762/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Pan,%20Q.%20et%20al.%20The%20rise%20and%20fall%20of%20the%20ancient%20northern%20pike%20master%20sex-determining%20gene.%20eLife10,%20e62858%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Yano, A. <em>et al</em>. The sexually dimorphic on the Y‐chromosome gene (sdY) is a conserved male‐specific Y‐chromosome sequence in many salmonids. <em>Evolutionary Applications</em><strong>6</strong>, 486–496 (2013).
</cite> [<a href="https://doi.org/10.1111/eva.12032" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3673476/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23745140/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yano,%20A.%20et%20al.%20The%20sexually%20dimorphic%20on%20the%20Y%E2%80%90chromosome%20gene%20(sdY)%20is%20a%20conserved%20male%E2%80%90specific%20Y%E2%80%90chromosome%20sequence%20in%20many%20salmonids.%20Evolutionary%20Applications6,%20486%E2%80%93496%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Chen, S., Zhou, Y., Chen, Y. &amp; Gu, J. fastp: an ultra-fast all-in-one FASTQ preprocessor. <em>Bioinformatics</em><strong>34</strong>, i884–i890 (2018).
</cite> [<a href="https://doi.org/10.1093/bioinformatics/bty560" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6129281/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30423086/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chen,%20S.,%20Zhou,%20Y.,%20Chen,%20Y.%20&amp;%20Gu,%20J.%20fastp:%20an%20ultra-fast%20all-in-one%20FASTQ%20preprocessor.%20Bioinformatics34,%20i884%E2%80%93i890%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>Marçais, G. &amp; Kingsford, C. A fast, lock-free approach for efficient parallel counting of occurrences of k -mers. <em>Bioinformatics</em><strong>27</strong>, 764–770 (2011).
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btr011" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3051319/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21217122/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mar%C3%A7ais,%20G.%20&amp;%20Kingsford,%20C.%20A%20fast,%20lock-free%20approach%20for%20efficient%20parallel%20counting%20of%20occurrences%20of%20k%20-mers.%20Bioinformatics27,%20764%E2%80%93770%20(2011)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>Ranallo-Benavidez, T. R., Jaron, K. S. &amp; Schatz, M. C. GenomeScope 2.0 and Smudgeplot for reference-free profiling of polyploid genomes. <em>Nat Commun</em><strong>11</strong>, 1432 (2020).
</cite> [<a href="https://doi.org/10.1038/s41467-020-14998-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7080791/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32188846/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ranallo-Benavidez,%20T.%20R.,%20Jaron,%20K.%20S.%20&amp;%20Schatz,%20M.%20C.%20GenomeScope%202.0%20and%20Smudgeplot%20for%20reference-free%20profiling%20of%20polyploid%20genomes.%20Nat%20Commun11,%201432%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>Cheng, H., Concepcion, G. T., Feng, X., Zhang, H. &amp; Li, H. Haplotype-resolved de novo assembly using phased assembly graphs with hifiasm. <em>Nat Methods</em><strong>18</strong>, 170–175 (2021).
</cite> [<a href="https://doi.org/10.1038/s41592-020-01056-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7961889/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33526886/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cheng,%20H.,%20Concepcion,%20G.%20T.,%20Feng,%20X.,%20Zhang,%20H.%20&amp;%20Li,%20H.%20Haplotype-resolved%20de%20novo%20assembly%20using%20phased%20assembly%20graphs%20with%20hifiasm.%20Nat%20Methods18,%20170%E2%80%93175%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>Belton, J.-M. <em>et al</em>. Hi–C: A comprehensive technique to capture the conformation of genomes. <em>Methods</em><strong>58</strong>, 268–276 (2012).
</cite> [<a href="https://doi.org/10.1016/j.ymeth.2012.05.001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3874846/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22652625/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Belton,%20J.-M.%20et%20al.%20Hi%E2%80%93C:%20A%20comprehensive%20technique%20to%20capture%20the%20conformation%20of%20genomes.%20Methods58,%20268%E2%80%93276%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>Li, H. Aligning sequence reads, clone sequences and assembly contigs with BWA-MEM. Preprint at 10.48550/ARXIV.1303.3997 (2013).</cite>
</li>
<li id="CR25">
<span class="label">25.</span><cite>Burton, J. N. <em>et al</em>. Chromosome-scale scaffolding of de novo genome assemblies based on chromatin interactions. <em>Nat Biotechnol</em><strong>31</strong>, 1119–1125 (2013).
</cite> [<a href="https://doi.org/10.1038/nbt.2727" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4117202/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24185095/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Burton,%20J.%20N.%20et%20al.%20Chromosome-scale%20scaffolding%20of%20de%20novo%20genome%20assemblies%20based%20on%20chromatin%20interactions.%20Nat%20Biotechnol31,%201119%E2%80%931125%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>Flynn, J. M. <em>et al</em>. RepeatModeler2 for automated genomic discovery of transposable element families. <em>Proc Natl Acad Sci</em><strong>117</strong>, 9451–9457 (2020).
</cite> [<a href="https://doi.org/10.1073/pnas.1921046117" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7196820/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32300014/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Flynn,%20J.%20M.%20et%20al.%20RepeatModeler2%20for%20automated%20genomic%20discovery%20of%20transposable%20element%20families.%20Proc%20Natl%20Acad%20Sci117,%209451%E2%80%939457%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Bao, Z. &amp; Eddy, S. R. Automated De Novo Identification of Repeat Sequence Families in Sequenced Genomes. <em>Genome Res</em><strong>12</strong>, 1269–1276 (2002).
</cite> [<a href="https://doi.org/10.1101/gr.88502" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC186642/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12176934/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bao,%20Z.%20&amp;%20Eddy,%20S.%20R.%20Automated%20De%20Novo%20Identification%20of%20Repeat%20Sequence%20Families%20in%20Sequenced%20Genomes.%20Genome%20Res12,%201269%E2%80%931276%20(2002)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Price, A. L., Jones, N. C. &amp; Pevzner, P. A. 2005. De novo identification of repeat families in large genomes. <em>Bioinformatics</em><strong>21</strong>, i351–i358 (2005).
</cite> [<a href="https://doi.org/10.1093/bioinformatics/bti1018" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15961478/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Price,%20A.%20L.,%20Jones,%20N.%20C.%20&amp;%20Pevzner,%20P.%20A.%202005.%20De%20novo%20identification%20of%20repeat%20families%20in%20large%20genomes.%20Bioinformatics21,%20i351%E2%80%93i358%20(2005)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>Ellinghaus, D., Kurtz, S. &amp; Willhoeft, U. LTRharvest, an efficient and flexible software for de novo detection of LTR retrotransposons. <em>BMC Bioinformatics</em><strong>9</strong>, 18 (2008).
</cite> [<a href="https://doi.org/10.1186/1471-2105-9-18" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2253517/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18194517/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ellinghaus,%20D.,%20Kurtz,%20S.%20&amp;%20Willhoeft,%20U.%20LTRharvest,%20an%20efficient%20and%20flexible%20software%20for%20de%20novo%20detection%20of%20LTR%20retrotransposons.%20BMC%20Bioinformatics9,%2018%20(2008)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR30">
<span class="label">30.</span><cite>Ou, S. &amp; Jiang, N. LTR_retriever: A Highly Accurate and Sensitive Program for Identification of Long Terminal Repeat Retrotransposons. <em>Plant Physiol.</em><strong>176</strong>, 1410–1422 (2018).
</cite> [<a href="https://doi.org/10.1104/pp.17.01310" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5813529/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29233850/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ou,%20S.%20&amp;%20Jiang,%20N.%20LTR_retriever:%20A%20Highly%20Accurate%20and%20Sensitive%20Program%20for%20Identification%20of%20Long%20Terminal%20Repeat%20Retrotransposons.%20Plant%20Physiol.176,%201410%E2%80%931422%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>Wheeler, T. J. <em>et al</em>. Dfam: a database of repetitive DNA based on profile hidden Markov models. <em>Nucleic Acids Res</em><strong>41</strong>, D70–82 (2013).
</cite> [<a href="https://doi.org/10.1093/nar/gks1265" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3531169/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23203985/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wheeler,%20T.%20J.%20et%20al.%20Dfam:%20a%20database%20of%20repetitive%20DNA%20based%20on%20profile%20hidden%20Markov%20models.%20Nucleic%20Acids%20Res41,%20D70%E2%80%9382%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>Tarailo‐Graovac, M. &amp; Chen, N. Using RepeatMasker to Identify Repetitive Elements in Genomic Sequences. <em>CP in Bioinformatics</em><strong>5</strong>, 4–10 (2004).</cite> [<a href="https://doi.org/10.1002/0471250953.bi0410s25" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19274634/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tarailo%E2%80%90Graovac,%20M.%20&amp;%20Chen,%20N.%20Using%20RepeatMasker%20to%20Identify%20Repetitive%20Elements%20in%20Genomic%20Sequences.%20CP%20in%20Bioinformatics5,%204%E2%80%9310%20(2004)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Benson, G. Tandem repeats finder: a program to analyze DNA sequences. <em>Nucleic Acids Research</em><strong>27</strong>, 573–580 (1999).
</cite> [<a href="https://doi.org/10.1093/nar/27.2.573" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC148217/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9862982/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Benson,%20G.%20Tandem%20repeats%20finder:%20a%20program%20to%20analyze%20DNA%20sequences.%20Nucleic%20Acids%20Research27,%20573%E2%80%93580%20(1999)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>Thiel, T., Michalek, W., Varshney, R. &amp; Graner, A. Exploiting EST databases for the development and characterization of gene-derived SSR-markers in barley (Hordeum vulgare L.). <em>Theor Appl Genet</em><strong>106</strong>, 411–422 (2003).
</cite> [<a href="https://doi.org/10.1007/s00122-002-1031-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12589540/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Thiel,%20T.,%20Michalek,%20W.,%20Varshney,%20R.%20&amp;%20Graner,%20A.%20Exploiting%20EST%20databases%20for%20the%20development%20and%20characterization%20of%20gene-derived%20SSR-markers%20in%20barley%20(Hordeum%20vulgare%20L.).%20Theor%20Appl%20Genet106,%20411%E2%80%93422%20(2003)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>Stanke, M., Steinkamp, R., Waack, S. &amp; Morgenstern, B. AUGUSTUS: a web server for gene finding in eukaryotes. <em>Nucleic Acids Res</em><strong>32</strong>, W309–312 (2004).
</cite> [<a href="https://doi.org/10.1093/nar/gkh379" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC441517/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15215400/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Stanke,%20M.,%20Steinkamp,%20R.,%20Waack,%20S.%20&amp;%20Morgenstern,%20B.%20AUGUSTUS:%20a%20web%20server%20for%20gene%20finding%20in%20eukaryotes.%20Nucleic%20Acids%20Res32,%20W309%E2%80%93312%20(2004)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR36">
<span class="label">36.</span><cite>Korf, I. Gene finding in novel genomes. <em>BMC Bioinformatics</em><strong>5</strong>, 59 (2004).
</cite> [<a href="https://doi.org/10.1186/1471-2105-5-59" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC421630/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15144565/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Korf,%20I.%20Gene%20finding%20in%20novel%20genomes.%20BMC%20Bioinformatics5,%2059%20(2004)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR37">
<span class="label">37.</span><cite>Keilwagen, J., Hartung, F. &amp; Grau, J. GeMoMa: Homology-Based Gene Prediction Utilizing Intron Position Conservation and RNA-seq Data. in <em>Gene Prediction</em> (ed. Kollmar, M.) vol. 1962, 161–177 (Springer New York, New York, NY, 2019).</cite> [<a href="https://doi.org/10.1007/978-1-4939-9173-0_9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31020559/" class="usa-link">PubMed</a>]</li>
<li id="CR38">
<span class="label">38.</span><cite>Kim, D., Langmead, B. &amp; Salzberg, S. L. HISAT: a fast spliced aligner with low memory requirements. <em>Nat Methods</em><strong>12</strong>, 357–360 (2015).
</cite> [<a href="https://doi.org/10.1038/nmeth.3317" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4655817/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25751142/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kim,%20D.,%20Langmead,%20B.%20&amp;%20Salzberg,%20S.%20L.%20HISAT:%20a%20fast%20spliced%20aligner%20with%20low%20memory%20requirements.%20Nat%20Methods12,%20357%E2%80%93360%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR39">
<span class="label">39.</span><cite>Pertea, M. <em>et al</em>. StringTie enables improved reconstruction of a transcriptome from RNA-seq reads. <em>Nat Biotechnol</em><strong>33</strong>, 290–295 (2015).
</cite> [<a href="https://doi.org/10.1038/nbt.3122" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4643835/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25690850/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Pertea,%20M.%20et%20al.%20StringTie%20enables%20improved%20reconstruction%20of%20a%20transcriptome%20from%20RNA-seq%20reads.%20Nat%20Biotechnol33,%20290%E2%80%93295%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR40">
<span class="label">40.</span><cite>Price, A. L., Jones, N. C. &amp; Pevzner, P. A. De novo identification of repeat families in large genomes. <em>Bioinformatics</em><strong>21</strong>(Suppl 1), i351–358 (2005).
</cite> [<a href="https://doi.org/10.1093/bioinformatics/bti1018" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15961478/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Price,%20A.%20L.,%20Jones,%20N.%20C.%20&amp;%20Pevzner,%20P.%20A.%20De%20novo%20identification%20of%20repeat%20families%20in%20large%20genomes.%20Bioinformatics21(Suppl%201),%20i351%E2%80%93358%20(2005)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR41">
<span class="label">41.</span><cite>Haas, B. J. Improving the Arabidopsis genome annotation using maximal transcript alignment assemblies. <em>Nucleic Acids Research</em><strong>31</strong>, 5654–5666 (2003).
</cite> [<a href="https://doi.org/10.1093/nar/gkg770" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC206470/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/14500829/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Haas,%20B.%20J.%20Improving%20the%20Arabidopsis%20genome%20annotation%20using%20maximal%20transcript%20alignment%20assemblies.%20Nucleic%20Acids%20Research31,%205654%E2%80%935666%20(2003)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR42">
<span class="label">42.</span><cite>Grabherr, M. G. <em>et al</em>. Full-length transcriptome assembly from RNA-Seq data without a reference genome. <em>Nat Biotechnol</em><strong>29</strong>, 644–652 (2011).
</cite> [<a href="https://doi.org/10.1038/nbt.1883" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3571712/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21572440/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Grabherr,%20M.%20G.%20et%20al.%20Full-length%20transcriptome%20assembly%20from%20RNA-Seq%20data%20without%20a%20reference%20genome.%20Nat%20Biotechnol29,%20644%E2%80%93652%20(2011)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR43">
<span class="label">43.</span><cite>Haas, B. J. <em>et al</em>. Automated eukaryotic gene structure annotation using EVidenceModeler and the Program to Assemble Spliced Alignments. <em>Genome Biol</em><strong>9</strong>, R7 (2008).
</cite> [<a href="https://doi.org/10.1186/gb-2008-9-1-r7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2395244/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18190707/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Haas,%20B.%20J.%20et%20al.%20Automated%20eukaryotic%20gene%20structure%20annotation%20using%20EVidenceModeler%20and%20the%20Program%20to%20Assemble%20Spliced%20Alignments.%20Genome%20Biol9,%20R7%20(2008)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR44">
<span class="label">44.</span><cite>Pruitt, K. D. NCBI Reference Sequence (RefSeq): a curated non-redundant sequence database of genomes, transcripts and proteins. <em>Nucleic Acids Research</em><strong>33</strong>, D501–D504 (2004).</cite> [<a href="https://doi.org/10.1093/nar/gki025" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC539979/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15608248/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Pruitt,%20K.%20D.%20NCBI%20Reference%20Sequence%20(RefSeq):%20a%20curated%20non-redundant%20sequence%20database%20of%20genomes,%20transcripts%20and%20proteins.%20Nucleic%20Acids%20Research33,%20D501%E2%80%93D504%20(2004)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR45">
<span class="label">45.</span><cite>Huerta-Cepas, J. <em>et al</em>. eggNOG 5.0: a hierarchical, functionally and phylogenetically annotated orthology resource based on 5090 organisms and 2502 viruses. <em>Nucleic Acids Research</em><strong>47</strong>, D309–D314 (2019).
</cite> [<a href="https://doi.org/10.1093/nar/gky1085" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6324079/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30418610/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Huerta-Cepas,%20J.%20et%20al.%20eggNOG%205.0:%20a%20hierarchical,%20functionally%20and%20phylogenetically%20annotated%20orthology%20resource%20based%20on%205090%20organisms%20and%202502%20viruses.%20Nucleic%20Acids%20Research47,%20D309%E2%80%93D314%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR46">
<span class="label">46.</span><cite>Tatusov, R. L. <em>et al</em>. The COG database: an updated version includes eukaryotes. <em>BMC Bioinformatics</em><strong>4</strong>, 41 (2003).
</cite> [<a href="https://doi.org/10.1186/1471-2105-4-41" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC222959/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12969510/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tatusov,%20R.%20L.%20et%20al.%20The%20COG%20database:%20an%20updated%20version%20includes%20eukaryotes.%20BMC%20Bioinformatics4,%2041%20(2003)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR47">
<span class="label">47.</span><cite>Boeckmann, B. <em>et al</em>. The SWISS-PROT protein knowledgebase and its supplement TrEMBL in 2003. <em>Nucleic Acids Research</em><strong>31</strong>, 365–370 (2003).
</cite> [<a href="https://doi.org/10.1093/nar/gkg095" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC165542/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12520024/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Boeckmann,%20B.%20et%20al.%20The%20SWISS-PROT%20protein%20knowledgebase%20and%20its%20supplement%20TrEMBL%20in%202003.%20Nucleic%20Acids%20Research31,%20365%E2%80%93370%20(2003)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR48">
<span class="label">48.</span><cite>Gene Ontology Consortium The Gene Ontology (GO) database and informatics resource. <em>Nucleic Acids Research</em><strong>32</strong>, 258D–261 (2004).</cite> [<a href="https://doi.org/10.1093/nar/gkh036" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC308770/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/14681407/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gene%20Ontology%20Consortium%20The%20Gene%20Ontology%20(GO)%20database%20and%20informatics%20resource.%20Nucleic%20Acids%20Research32,%20258D%E2%80%93261%20(2004)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR49">
<span class="label">49.</span><cite>Kanehisa, M., Goto, S., Sato, Y., Furumichi, M. &amp; Tanabe, M. KEGG for integration and interpretation of large-scale molecular data sets. <em>Nucleic Acids Research</em><strong>40</strong>, D109–D114 (2012).
</cite> [<a href="https://doi.org/10.1093/nar/gkr988" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3245020/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22080510/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kanehisa,%20M.,%20Goto,%20S.,%20Sato,%20Y.,%20Furumichi,%20M.%20&amp;%20Tanabe,%20M.%20KEGG%20for%20integration%20and%20interpretation%20of%20large-scale%20molecular%20data%20sets.%20Nucleic%20Acids%20Research40,%20D109%E2%80%93D114%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR50">
<span class="label">50.</span><cite>Buchfink, B., Xie, C. &amp; Huson, D. H. Fast and sensitive protein alignment using DIAMOND. <em>Nature Methods</em><strong>12</strong>, 59–60 (2015).
</cite> [<a href="https://doi.org/10.1038/nmeth.3176" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25402007/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Buchfink,%20B.,%20Xie,%20C.%20&amp;%20Huson,%20D.%20H.%20Fast%20and%20sensitive%20protein%20alignment%20using%20DIAMOND.%20Nature%20Methods12,%2059%E2%80%9360%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR51">
<span class="label">51.</span><cite>Finn, R. D. <em>et al</em>. The Pfam protein families database. <em>Nucleic Acids Res</em><strong>36</strong>, D281–D288 (2008).
</cite> [<a href="https://doi.org/10.1093/nar/gkm960" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2238907/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18039703/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Finn,%20R.%20D.%20et%20al.%20The%20Pfam%20protein%20families%20database.%20Nucleic%20Acids%20Res36,%20D281%E2%80%93D288%20(2008)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR52">
<span class="label">52.</span><cite>Lowe, T. M. &amp; Eddy, S. R. tRNAscan-SE: A Program for Improved Detection of Transfer RNA Genes in Genomic Sequence. <em>Nucleic Acids Research</em><strong>25</strong>, 955–964 (1997).
</cite> [<a href="https://doi.org/10.1093/nar/25.5.955" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC146525/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9023104/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lowe,%20T.%20M.%20&amp;%20Eddy,%20S.%20R.%20tRNAscan-SE:%20A%20Program%20for%20Improved%20Detection%20of%20Transfer%20RNA%20Genes%20in%20Genomic%20Sequence.%20Nucleic%20Acids%20Research25,%20955%E2%80%93964%20(1997)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR53">
<span class="label">53.</span><cite>Loman, T. A Novel Method for Predicting Ribosomal RNA Genes in Prokaryotic Genomes, <a href="http://lup.lub.lu.se/student-papers/record/8914064" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://lup.lub.lu.se/student-papers/record/8914064</a> (2017).</cite>
</li>
<li id="CR54">
<span class="label">54.</span><cite>Kalvari, I. <em>et al</em>. Rfam 14: expanded coverage of metagenomic, viral and microRNA families. <em>Nucleic Acids Research</em><strong>49</strong>, D192–D200 (2021).
</cite> [<a href="https://doi.org/10.1093/nar/gkaa1047" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7779021/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33211869/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kalvari,%20I.%20et%20al.%20Rfam%2014:%20expanded%20coverage%20of%20metagenomic,%20viral%20and%20microRNA%20families.%20Nucleic%20Acids%20Research49,%20D192%E2%80%93D200%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR55">
<span class="label">55.</span><cite>Nawrocki, E. P. &amp; Eddy, S. R. Infernal 1.1: 100-fold faster RNA homology searches. <em>Bioinformatics</em><strong>29</strong>, 2933–2935 (2013).
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btt509" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3810854/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24008419/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Nawrocki,%20E.%20P.%20&amp;%20Eddy,%20S.%20R.%20Infernal%201.1:%20100-fold%20faster%20RNA%20homology%20searches.%20Bioinformatics29,%202933%E2%80%932935%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR56">
<span class="label">56.</span><cite>Symonová, R. <em>et al</em>. Higher-order organisation of extremely amplified, potentially functional and massively methylated 5S rDNA in European pikes (Esox sp.). <em>BMC Genomics</em><strong>18</strong>, 391 (2017).
</cite> [<a href="https://doi.org/10.1186/s12864-017-3774-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5437419/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28521734/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Symonov%C3%A1,%20R.%20et%20al.%20Higher-order%20organisation%20of%20extremely%20amplified,%20potentially%20functional%20and%20massively%20methylated%205S%20rDNA%20in%20European%20pikes%20(Esox%20sp.).%20BMC%20Genomics18,%20391%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR57">
<span class="label">57.</span><cite><em>NGDC Genome Sequence Archive</em><a href="https://ngdc.cncb.ac.cn/gsa/browse/CRA024841" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://ngdc.cncb.ac.cn/gsa/browse/CRA024841</a> (2025).</cite>
</li>
<li id="CR58">
<span class="label">58.</span><cite><em>NGDC Genome Sequence Archive</em><a href="https://ngdc.cncb.ac.cn/bioproject/browse/PRJCA038565" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://ngdc.cncb.ac.cn/bioproject/browse/PRJCA038565</a> (2025).</cite>
</li>
<li id="CR59">
<span class="label">59.</span><cite><em>NCBI GenBank</em><a href="https://identifiers.org/ncbi/insdc:JBPYXS000000000.1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://identifiers.org/ncbi/insdc:JBPYXS000000000.1</a> (2025).</cite>
</li>
<li id="CR60">
<span class="label">60.</span><cite><em>NGDC Genome Warehouse</em><a href="https://ngdc.cncb.ac.cn/gwh/Assembly/93573/show" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://ngdc.cncb.ac.cn/gwh/Assembly/93573/show</a> (2025).</cite>
</li>
<li id="CR61">
<span class="label">61.</span><cite>Zhang, Y. Chr_genome_final_gene.gff3. <em>figshare. Dataset.</em>10.6084/m9.figshare.29663588.v1 (2025).</cite>
</li>
<li id="CR62">
<span class="label">62.</span><cite>Zhang, Y. Chr_genome_final_gene.gff3.gene. <em>figshare. Dataset.</em>10.6084/m9.figshare.29663807.v1 (2025).</cite>
</li>
<li id="CR63">
<span class="label">63.</span><cite>Zhang, Y. Chr_genome_final_gene.gff3.exon. <em>figshare. Dataset.</em>10.6084/m9.figshare.29663870.v1 (2025).</cite>
</li>
<li id="CR64">
<span class="label">64.</span><cite>Zhang, Y. Chr_genome_final_gene.gff3.cds. <em>figshare. Dataset.</em>10.6084/m9.figshare.29663804.v1 (2025).</cite>
</li>
<li id="CR65">
<span class="label">65.</span><cite>Zhang, Y. Chr_genome_final_gene.gff3.pep. <em>figshare. Dataset.</em>10.6084/m9.figshare.29663858.v1 (2025).</cite>
</li>
<li id="CR66">
<span class="label">66.</span><cite>Li, H. Minimap2: pairwise alignment for nucleotide sequences. <em>Bioinformatics</em><strong>34</strong>, 3094–3100 (2018).
</cite> [<a href="https://doi.org/10.1093/bioinformatics/bty191" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6137996/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29750242/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li,%20H.%20Minimap2:%20pairwise%20alignment%20for%20nucleotide%20sequences.%20Bioinformatics34,%203094%E2%80%933100%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR67">
<span class="label">67.</span><cite>Simão, F. A., Waterhouse, R. M., Ioannidis, P., Kriventseva, E. V. &amp; Zdobnov, E. M. BUSCO: assessing genome assembly and annotation completeness with single-copy orthologs. <em>Bioinformatics</em><strong>31</strong>, 3210–3212 (2015).
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btv351" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26059717/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sim%C3%A3o,%20F.%20A.,%20Waterhouse,%20R.%20M.,%20Ioannidis,%20P.,%20Kriventseva,%20E.%20V.%20&amp;%20Zdobnov,%20E.%20M.%20BUSCO:%20assessing%20genome%20assembly%20and%20annotation%20completeness%20with%20single-copy%20orthologs.%20Bioinformatics31,%203210%E2%80%933212%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR68">
<span class="label">68.</span><cite>Laetsch, D. R. &amp; Blaxter, M. L. BlobTools: Interrogation of genome assemblies. <em>F1000Research</em><strong>6</strong>, 1287 (2017).</cite> [<a href="https://scholar.google.com/scholar_lookup?Laetsch,%20D.%20R.%20&amp;%20Blaxter,%20M.%20L.%20BlobTools:%20Interrogation%20of%20genome%20assemblies.%20F1000Research6,%201287%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR69">
<span class="label">69.</span><cite>Rhie, A. <em>et al</em>. Merqury: reference-free quality, completeness, and phasing assessment for genome assemblies. <em>Genome Biol</em><strong>21</strong>, 245 (2020).
</cite> [<a href="https://doi.org/10.1186/s13059-020-02134-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7488777/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32928274/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rhie,%20A.%20et%20al.%20Merqury:%20reference-free%20quality,%20completeness,%20and%20phasing%20assessment%20for%20genome%20assemblies.%20Genome%20Biol21,%20245%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR70">
<span class="label">70.</span><cite>Tang, H. <em>et al</em>. JCVI: A versatile toolkit for comparative genomics analysis. <em>iMeta</em><strong>3</strong>, e211 (2024).
</cite> [<a href="https://doi.org/10.1002/imt2.211" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11316928/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39135687/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tang,%20H.%20et%20al.%20JCVI:%20A%20versatile%20toolkit%20for%20comparative%20genomics%20analysis.%20iMeta3,%20e211%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adc93_" lang="en" class="data-citations"><h3 class="pmc_sec_title">Data Citations</h3>
<ol class="ref-list font-sm">
<li id="_dbx_ref_citation-alternatives51"><cite><em>NGDC Genome Sequence Archive</em><a href="https://ngdc.cncb.ac.cn/gsa/browse/CRA024841" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://ngdc.cncb.ac.cn/gsa/browse/CRA024841</a> (2025).</cite></li>
<li id="_dbx_ref_citation-alternatives52"><cite><em>NGDC Genome Sequence Archive</em><a href="https://ngdc.cncb.ac.cn/bioproject/browse/PRJCA038565" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://ngdc.cncb.ac.cn/bioproject/browse/PRJCA038565</a> (2025).</cite></li>
<li id="_dbx_ref_citation-alternatives53"><cite><em>NCBI GenBank</em><a href="https://identifiers.org/ncbi/insdc:JBPYXS000000000.1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://identifiers.org/ncbi/insdc:JBPYXS000000000.1</a> (2025).</cite></li>
<li id="_dbx_ref_citation-alternatives54"><cite><em>NGDC Genome Warehouse</em><a href="https://ngdc.cncb.ac.cn/gwh/Assembly/93573/show" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://ngdc.cncb.ac.cn/gwh/Assembly/93573/show</a> (2025).</cite></li>
<li id="_dbx_ref_citation-alternatives55"><cite>Zhang, Y. Chr_genome_final_gene.gff3. <em>figshare. Dataset.</em>10.6084/m9.figshare.29663588.v1 (2025).</cite></li>
<li id="_dbx_ref_citation-alternatives56"><cite>Zhang, Y. Chr_genome_final_gene.gff3.gene. <em>figshare. Dataset.</em>10.6084/m9.figshare.29663807.v1 (2025).</cite></li>
<li id="_dbx_ref_citation-alternatives57"><cite>Zhang, Y. Chr_genome_final_gene.gff3.exon. <em>figshare. Dataset.</em>10.6084/m9.figshare.29663870.v1 (2025).</cite></li>
<li id="_dbx_ref_citation-alternatives58"><cite>Zhang, Y. Chr_genome_final_gene.gff3.cds. <em>figshare. Dataset.</em>10.6084/m9.figshare.29663804.v1 (2025).</cite></li>
<li id="_dbx_ref_citation-alternatives59"><cite>Zhang, Y. Chr_genome_final_gene.gff3.pep. <em>figshare. Dataset.</em>10.6084/m9.figshare.29663858.v1 (2025).</cite></li>
</ol></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>The bioinformatic analysis were executed in accordance with the guidelines and procedures outlined in the manuals of the relevant software. The Methods section contains a detailed description of the software and corresponding parameters. No custom code was used in this process.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Scientific Data are provided here courtesy of <strong>Nature Publishing Group</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1038/s41597-025-05819-7"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/41597_2025_Article_5819.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (2.5 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12368265/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12368265/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12368265%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12368265/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12368265/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12368265/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40835863/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12368265/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40835863/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12368265/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12368265/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="kjLNcJ1CL6a52G1OVv26EcT65QOKsuLnDPdtC5Bl76W25iaXxSgw70kYS6ZHR9yq">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
