
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Action mechanism of a novel agrichemical quinofumelin against Fusarium graminearum - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="AE5338128AF305930538120044B82A81.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="elife">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12367298/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="eLife">
<meta name="citation_title" content="Action mechanism of a novel agrichemical quinofumelin against Fusarium graminearum">
<meta name="citation_author" content="Qian Xiu">
<meta name="citation_author_institution" content="State Key Laboratory of Agricultural and Forestry Biosecurity, College of Plant Protection, Nanjing Agricultural University, Nanjing, China">
<meta name="citation_author" content="Xiaoru Yin">
<meta name="citation_author_institution" content="State Key Laboratory of Agricultural and Forestry Biosecurity, College of Plant Protection, Nanjing Agricultural University, Nanjing, China">
<meta name="citation_author" content="Yuanyuan Chen">
<meta name="citation_author_institution" content="College of Science, Nanjing Agricultural University, Nanjing, China">
<meta name="citation_author" content="Ziyang Zhang">
<meta name="citation_author_institution" content="State Key Laboratory of Agricultural and Forestry Biosecurity, College of Plant Protection, Nanjing Agricultural University, Nanjing, China">
<meta name="citation_author" content="Yushuai Mao">
<meta name="citation_author_institution" content="State Key Laboratory of Agricultural and Forestry Biosecurity, College of Plant Protection, Nanjing Agricultural University, Nanjing, China">
<meta name="citation_author" content="Tianshi Wang">
<meta name="citation_author_institution" content="College of Science, Nanjing Agricultural University, Nanjing, China">
<meta name="citation_author" content="Jie Zhang">
<meta name="citation_author_institution" content="State Key Laboratory of Agricultural and Forestry Biosecurity, College of Plant Protection, Nanjing Agricultural University, Nanjing, China">
<meta name="citation_author" content="Mingguo Zhou">
<meta name="citation_author_institution" content="State Key Laboratory of Agricultural and Forestry Biosecurity, College of Plant Protection, Nanjing Agricultural University, Nanjing, China">
<meta name="citation_author" content="Yabing Duan">
<meta name="citation_author_institution" content="State Key Laboratory of Agricultural and Forestry Biosecurity, College of Plant Protection, Nanjing Agricultural University, Nanjing, China">
<meta name="citation_publication_date" content="2025 Aug 20">
<meta name="citation_volume" content="14">
<meta name="citation_firstpage" content="RP105892">
<meta name="citation_doi" content="10.7554/eLife.105892">
<meta name="citation_pmid" content="40832893">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12367298/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12367298/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12367298/pdf/elife-105892.pdf">
<meta name="description" content="Modern fungicides have made significant contributions to crop disease management, but the development of resistant fungal strains has caused their failure in disease control. Therefore, developing fungicides with novel action mechanisms is the most ...">
<meta name="og:title" content="Action mechanism of a novel agrichemical quinofumelin against Fusarium graminearum">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Modern fungicides have made significant contributions to crop disease management, but the development of resistant fungal strains has caused their failure in disease control. Therefore, developing fungicides with novel action mechanisms is the most ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12367298/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12367298">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.7554/eLife.105892"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/elife-105892.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12367298%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12367298/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12367298/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12367298/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-elife.png" alt="eLife logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to eLife" title="Link to eLife" shape="default" href="https://doi.org/10.7554/eLife.105892" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">eLife</button></div>. 2025 Aug 20;14:RP105892. doi: <a href="https://doi.org/10.7554/eLife.105892" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.7554/eLife.105892</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22eLife%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22eLife%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22eLife%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22eLife%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Action mechanism of a novel agrichemical quinofumelin against <em>Fusarium graminearum</em>
</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Xiu%20Q%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Qian Xiu</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Qian Xiu</span></h3>
<div class="p">
<sup>1</sup>State Key Laboratory of Agricultural and Forestry Biosecurity, College of Plant Protection, Nanjing Agricultural University, Nanjing, China</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Xiu%20Q%22%5BAuthor%5D" class="usa-link"><span class="name western">Qian Xiu</span></a>
</div>
</div>
<sup>1,</sup><sup>†</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yin%20X%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Xiaoru Yin</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Xiaoru Yin</span></h3>
<div class="p">
<sup>1</sup>State Key Laboratory of Agricultural and Forestry Biosecurity, College of Plant Protection, Nanjing Agricultural University, Nanjing, China</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yin%20X%22%5BAuthor%5D" class="usa-link"><span class="name western">Xiaoru Yin</span></a>
</div>
</div>
<sup>1,</sup><sup>†</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chen%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Yuanyuan Chen</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Yuanyuan Chen</span></h3>
<div class="p">
<sup>2</sup>College of Science, Nanjing Agricultural University, Nanjing, China</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chen%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yuanyuan Chen</span></a>
</div>
</div>
<sup>2,</sup><sup>†</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20Z%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Ziyang Zhang</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Ziyang Zhang</span></h3>
<div class="p">
<sup>1</sup>State Key Laboratory of Agricultural and Forestry Biosecurity, College of Plant Protection, Nanjing Agricultural University, Nanjing, China</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20Z%22%5BAuthor%5D" class="usa-link"><span class="name western">Ziyang Zhang</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mao%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Yushuai Mao</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Yushuai Mao</span></h3>
<div class="p">
<sup>1</sup>State Key Laboratory of Agricultural and Forestry Biosecurity, College of Plant Protection, Nanjing Agricultural University, Nanjing, China</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mao%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yushuai Mao</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20T%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Tianshi Wang</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Tianshi Wang</span></h3>
<div class="p">
<sup>2</sup>College of Science, Nanjing Agricultural University, Nanjing, China</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20T%22%5BAuthor%5D" class="usa-link"><span class="name western">Tianshi Wang</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Jie Zhang</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Jie Zhang</span></h3>
<div class="p">
<sup>1</sup>State Key Laboratory of Agricultural and Forestry Biosecurity, College of Plant Protection, Nanjing Agricultural University, Nanjing, China</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Jie Zhang</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhou%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Mingguo Zhou</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Mingguo Zhou</span></h3>
<div class="p">
<sup>1</sup>State Key Laboratory of Agricultural and Forestry Biosecurity, College of Plant Protection, Nanjing Agricultural University, Nanjing, China</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhou%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Mingguo Zhou</span></a>
</div>
</div>
<sup>1,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Duan%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id9"><span class="name western">Yabing Duan</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">Yabing Duan</span></h3>
<div class="p">
<sup>1</sup>State Key Laboratory of Agricultural and Forestry Biosecurity, College of Plant Protection, Nanjing Agricultural University, Nanjing, China</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Duan%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yabing Duan</span></a>
</div>
</div>
<sup>1,</sup><sup>✉</sup>
</div>
<div class="cg p">Editors: <span class="name western">Gustavo Henrique Goldman</span><sup>3</sup>, <span class="name western">Volker Dötsch</span><sup>4</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="aff1">
<sup>1</sup>State Key Laboratory of Agricultural and Forestry Biosecurity, College of Plant Protection, Nanjing Agricultural University, Nanjing, China</div>
<div id="aff2">
<sup>2</sup>College of Science, Nanjing Agricultural University, Nanjing, China</div>
<div id="aff3">
<sup>3</sup>Universidade de Sao Paulo, São Paulo, Brazil</div>
<div id="aff4">
<sup>4</sup>Goethe University Frankfurt, Frankfurt am Main, Germany</div>
<div class="author-notes p">
<div class="fn" id="equal-contrib1">
<sup>†</sup><p class="display-inline">These authors contributed equally to this work.</p>
</div>
<div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div>
</div>
<h4 class="font-secondary">Roles</h4>
<div class="p">
<strong class="contrib"><span class="name western">Gustavo Henrique Goldman</span></strong>: <span class="role">Reviewing Editor</span>
</div>
<div>
<strong class="contrib"><span class="name western">Volker Dötsch</span></strong>: <span class="role">Senior Editor</span>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© 2025, Xiu, Yin, Chen et al</div>
<p>This article is distributed under the terms of the <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Creative Commons Attribution License</a>, which permits unrestricted use and redistribution provided that the original author and source are credited.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12367298  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40832893/" class="usa-link">40832893</a>
</div>
<div class="ra xbox p" role="complementary" aria-label="Related or updated information about this article">
<div>
<strong>Previous version available:</strong> This article is based on a previously available preprint with doi: <a href="https://doi.org/10.1101/2025.01.13.632717" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://doi.org/10.1101/2025.01.13.632717</a>.</div>
<div>
<strong>Previous version available:</strong> This article is based on a previously available preprint with doi: <a href="https://doi.org/10.7554/eLife.105892.1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://doi.org/10.7554/eLife.105892.1</a>.</div>
<div>
<strong>Previous version available:</strong> This article is based on a previously available preprint with doi: <a href="https://doi.org/10.7554/eLife.105892.2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://doi.org/10.7554/eLife.105892.2</a>.</div>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>Abstract</h2>
<p>Modern fungicides have made significant contributions to crop disease management, but the development of resistant fungal strains has caused their failure in disease control. Therefore, developing fungicides with novel action mechanisms is the most effective measure to manage resistance. Quinofumelin, a novel quinoline fungicide, exhibits exceptional antifungal activity against phytopathogens. However, there is currently no available information on its mechanism of action. Here, we used transcriptome and metabolome analysis to observe a co-enrichment pattern of differentially expressed genes (DEGs) and differentially accumulated metabolites (DAMs) within pyrimidine biosynthesis pathway (PBP), identifying down-regulation of dihydroorotate dehydrogenase (DHODH). Exogenous uridine monophosphate (UMP), uridine, or uracil (metabolites in PBP) successfully restored quinofumelin-induced inhibition of mycelial growth in <em>Fusarium graminearum</em> and <em>Fusarium asiaticum</em>. Additionally, the deletion of <em>FgDHODHII</em> was determined to be lethal; however, mycelial growth of ΔFgDHODHII mutants could be restored by adding UMP, uridine, or uracil. These findings indicate that the deficiencies in <em>FgDHODHII</em> are functionally equivalent to complete inhibition of its activity by quinofumelin. Finally, molecular docking, surface plasmon resonance (SPR), and microscale thermophoresis (MST) results strongly support the precise interaction between quinofumelin and FgDHODHII. Collectively, these findings provide compelling evidence for the involvement of de novo uracil biosynthesis as a mechanism of action for quinofumelin while identifying FgDHODHII as its specific target.</p>
<section id="kwd-group2" class="kwd-group"><p><strong>Research organism:</strong> Other</p></section></section><section id="s1"><h2 class="pmc_sec_title">Introduction</h2>
<p>Agrochemical fungicides have a history spanning over 3000 years in the control of plant diseases, and have undergone a development process from traditional protective fungicides with multiple action sites to modern selective fungicides with single action site (<a href="#bib19" class="usa-link" aria-describedby="bib19">Lamberth, 2022</a>). The introduction of the first fully systemic seed treatment agent (carboxin, a succinate dehydrogenase inhibitor) in 1966 and the launch of the first broad-spectrum foliar systemic fungicide (benomyl, a tubulin polymerization inhibitor) in 1968 represent significant milestones in the advancement of modern selective fungicides with single action site (<a href="#bib18" class="usa-link" aria-describedby="bib18">Lamberth et al., 2021</a>; <a href="#bib24" class="usa-link" aria-describedby="bib24">Morton and Staub, 2008</a>). Owing to their exceptional selectivity, potent activity, specific target site, and low toxicity, modern selective fungicides have gained widespread acceptance and witnessed an increasing market share year after year. However, like every coin has two sides, these types of fungicides encounter substantial challenges related to serious resistance issues when deployed in agricultural fields (<a href="#bib20" class="usa-link" aria-describedby="bib20">Lucas et al., 2015</a>). Numerous reports have been published on plant pathogens developing resistance to modern selective fungicides, which often result in failures or reductions in the effectiveness of chemical control against plant diseases and lead to decreased crop yields and quality (<a href="https://www.frac.info" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.frac.info</a>; <a href="#bib10" class="usa-link" aria-describedby="bib10">Duan et al., 2019</a>; <a href="#bib16" class="usa-link" aria-describedby="bib16">King et al., 2021</a>; <a href="#bib22" class="usa-link" aria-describedby="bib22">Mao et al., 2024</a>). Therefore, it is crucial to expedite the development of new-generation selective fungicides with innovative modes of action. Based on a single fungicide target, numerous modern selective fungicides can be developed (<a href="#bib33" class="usa-link" aria-describedby="bib33">Umetsu and Shirai, 2020</a>). For instance, 24 commercialized available fungicides have been created so far based on the respiratory chain complex II as the target. Similarly, another 20 commercialized fungicides have been developed targeting the respiratory chain complex III. Additionally, there are already 37 commercialized fungicides that specifically target the fungicide target C14-demethylase in ergosterol biosynthesis (<a href="https://www.frac.info" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.frac.info</a>). However, identifying specific targets that efficiently control plant pathogens while ensuring safety for non-target organisms remains a significant challenge. The rate of discovery for new targets lags far behind current demands.</p>
<p>According to the latest information released by the Fungicide Resistance Action Committee (FRAC), over 30 fungicide selective targets have been identified and acknowledged in the field of fungicide biological research spanning more than 130 years. Currently, these targets have been identified based on the successful development of fungicides and are all related proteins that are essential for pathogen growth or pathogenicity. Extensive identification and functional analysis of essential genes for growth or pathogenicity in plant pathogens have been achieved with rapid advancements in molecular biology techniques; however, there is no precedent for successfully developing commercialized fungicides targeting genes involved in growth or pathogenicity. The creation of highly active modern selective fungicides currently relies on the structural characteristics of existing targets and is accomplished through technologies such as AI drug discovery and design (AIDD) and computer-aided drug design (CADD) (<a href="#bib39" class="usa-link" aria-describedby="bib39">Yang et al., 2019</a>). The biological functions of drug targets are highly conserved, although variations may occur at critical amino acid sites during genetic evolution. Therefore, it is crucial to explore drug targets and analyze their structural characteristics in order to develop novel fungicides with enhanced activity and selectivity. Recently, the Fungicide Biology group from Nanjing Agricultural University proposed a new concept for targeting pesticides based on the principles of targeted drugs in the medicine field (<a href="#bib42" class="usa-link" aria-describedby="bib42">Zhong et al., 2021</a>). The objective is to create specialized targeted pesticides that exhibit higher activity, greater selectivity, stronger specialization, lower toxicity, and environmental friendliness by leveraging the structural characteristics of pesticide targets. This approach aims to provide essential technical support for scientifically controlling crop diseases.</p>
<p>Quinofumelin (CAS 861647-84-9) is a novel quinoline fungicide developed by Mitsui Chemicals Co., Ltd. Japan (<a href="#bib14" class="usa-link" aria-describedby="bib14">Ito et al., 2023</a>). Due to their significant bioactivity, quinoline compounds and their derivatives have found extensive applications in medicine and pesticides (<a href="#bib32" class="usa-link" aria-describedby="bib32">Tao et al., 2021</a>; <a href="#bib37" class="usa-link" aria-describedby="bib37">Wheeler et al., 2003</a>). In our previous study, quinofumelin not only exhibited excellent antifungal activity against the mycelial growth and spore germination of <em>F. graminearum</em>, but also inhibited the biosynthesis of deoxynivalenol (DON) (<a href="#bib38" class="usa-link" aria-describedby="bib38">Xiu et al., 2021</a>). In previous studies, quinofumelin was identified as a specific inhibitor of dihydroorotate dehydrogenase class II (DHODHII) in <em>Pyricularia oryzae,</em> suppressing fungal growth by blocking the conversion of dihydroorotate to orotate (<a href="#bib13" class="usa-link" aria-describedby="bib13">Higashimura et al., 2022</a>). Gene disruption of <em>DHODHII</em> (<em>PoPYR4</em>) further confirmed its critical role in fungal pathogenicity and validated it as the action target site of quinofumelin. However, previous investigations did not report any detailed methodologies for target identification. Additionally, there was a lack of information regarding the precise interaction between quinofumelin and DHODHII. In this study, we used transcriptomic and metabolomic analyses to investigate the co-enrichment patterns of DEGs and (DAMs). This approach enabled the identification of potential metabolic pathways were identified. Ultimately, the pyrimidine biosynthesis pathway was identified as the target. Therefore, this study unveils that the mechanism of action of quinofumelin involves de novo synthesis of uracil, thereby identifying DHODH as its specific target. As a selective fungicidal target, DHODH offers a valuable resource for the design and development of novel fungicides.</p></section><section id="s2"><h2 class="pmc_sec_title">Results</h2>
<section id="s2-1"><h3 class="pmc_sec_title">Transcriptomic analysis of <em>F. graminearum</em> as affected by quinofumelin</h3>
<p>RNA-seq experiments have emerged as the standard approach for quantifying and comparing gene expression levels across a wide range of species and conditions, providing critical insights into cellular phenomena (<a href="#bib27" class="usa-link" aria-describedby="bib27">Pertea et al., 2016</a>). To investigate the impact of quinofumelin on gene expression levels in <em>F. graminearum</em>, we performed an RNA-seq experiment. For quality assessment of the RNA-seq data, we conducted a comprehensive quality control analysis. In the principal component analysis, PC1 and PC2 accounted for 45.59% and 16.98% of the total variance, respectively. The distinct separation observed between the groups indicates significant differences in gene expression between the controls and experimental samples, which could be used for subsequent investigations (<a href="#fig1s1" class="usa-link">Figure 1—figure supplement 1</a>). The clustering heat map effectively categorized the controls and experimental samples into distinct clusters based on their gene expression profiles, demonstrating robust biological replication with minimal intra-group variability and ensuring the reliability of the data. Moreover, notable inter-group dissimilarities highlighted variations among different treatment samples (<a href="#fig1" class="usa-link">Figure 1C</a>).</p>
<figure class="fig xbox font-sm" id="fig1"><h4 class="obj_head">Figure 1. Analysis of transcriptome sequencing data in <em>F</em>.<em>graminearum</em> PH-1 treated with quinofumelin.</h4>
<figcaption><p>(<strong>A</strong>) Bar chart illustrating the comparison of differentially expressed genes between EC<sub>90</sub> treatments and the control. (<strong>B</strong>) Volcano plot depicting up-regulated differentially expressed genes (DEGs) as red scattered dots and down-regulated DEGs as blue scattered dots. (<strong>C</strong>) Cluster heat maps displaying gene relative expression values using RNA-seq data scale-standardized values. (<strong>D</strong>) Bubble map presenting the results of Gene Ontology (GO) enrichment analysis for DEGs. (<strong>E</strong>) Bubble map showcasing the findings of Kyoto Encyclopedia of Genes and Genomes (KEGG) enrichment analysis for DEGs.</p></figcaption><p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12367298_elife-105892-fig1.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8be2/12367298/85d5a49bee38/elife-105892-fig1.jpg" loading="lazy" height="852" width="735" alt="Figure 1."></a></p>
<div class="p"><figure class="fig xbox font-sm" id="fig1s1"><h5 class="obj_head">Figure 1—figure supplement 1. Principal component analysis plots between the experimental (EC90: 1 μg/mL quinofumelin) and control (CK) samples.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12367298_elife-105892-fig1-figsupp1.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8be2/12367298/dc097e0d55a2/elife-105892-fig1-figsupp1.jpg" loading="lazy" height="446" width="735" alt="Figure 1—figure supplement 1."></a></p></figure></div>
<div class="p"><figure class="fig xbox font-sm" id="fig1s2"><h5 class="obj_head">Figure 1—figure supplement 2. Cluster analysis of the metabolite group.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12367298_elife-105892-fig1-figsupp2.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8be2/12367298/f69e2c031760/elife-105892-fig1-figsupp2.jpg" loading="lazy" height="526" width="735" alt="Figure 1—figure supplement 2."></a></p>
<figcaption><div class="p">(<strong>A</strong>) 3D principal component analysis diagram of EC<sub>90</sub>, CK, and QC samples in positive ion mode. (<strong>B</strong>) 3D principal component analysis diagram of EC<sub>90</sub>, CK, and QC samples in negative ion mode. (<strong>C</strong>) Heat map of inter-sample correlation analysis in positive ion mode. (<strong>D</strong>) Heat map of inter-sample correlation analysis in negative ion mode.</div></figcaption></figure></div>
<div class="p text-right font-secondary"><a href="figure/fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></figure><p>The differential analysis unveiled a total of 234 significantly DEGs between the control and experimental samples. Among these findings, there were identified as up-regulated for a count of 97 genes while down-regulation was observed in case of another set comprising around 137 genes (<a href="#fig1" class="usa-link">Figure 1A</a>). A graphical representation was employed to depict the statistical distribution pattern exhibited by these DEGs (<a href="#fig1" class="usa-link">Figure 1B</a>). To gain deeper insights into gene functionalities, Gene Ontology (GO) and Kyoto Encyclopedia of Genes and Genomes (KEGG) enrichment analyses were performed on these DEGs. The results showed that they are notably enriched in seventeen distinct GO terms spanning across two major categories: eleven molecular functions along with six processes (<a href="#fig1" class="usa-link">Figure 1D</a>). Furthermore, the enrichment analysis highlighted significant associations with various functional attributes, including transition metal ion binding, oxidoreductase activity, modified amino acid binding, amide binding, monooxygenase activity, and heme binding. Moreover, the enriched biological processes are primarily involved in transcriptional regulation driven by nucleic-enabled mechanisms, as well as monocarboxylic acid metabolism and RNA biosynthesis. Additionally, the KEGG enrichment analysis identified specific pathways associated with thiamine metabolism, tryptophan metabolism, nitrogen metabolism, amino acid sugar and nucleotide sugar metabolism, pantothenic acid and CoA biosynthesis, as well as nucleotide sugar production compound synthesis (<a href="#fig1" class="usa-link">Figure 1E</a>). In the GO and KEGG enrichment analyses, the detailed counts of down- and up-regulated DEGs for each metabolic pathway are provided in <a href="#supp1" class="usa-link">Supplementary file 1</a>.</p></section><section id="s2-2"><h3 class="pmc_sec_title">Metabolomic analysis of <em>F. graminearum</em> as affected by quinofumelin</h3>
<p>According to the findings from the KEGG enrichment analysis of DEGs derived from transcriptomic data, these DEGs are associated with metabolism pathways as above described. To specifically investigate the metabolic pathways involved in the action mechanism of quinofumelin, we performed further metabolomic experiments. In metabolomics principal component analysis, the contribution rate of PC1 was determined to be 76.6% in positive ion mode and 51.2% in negative ion mode, respectively. Moreover, the quality control (QC) samples exhibited highly concentrated results, indicating excellent instrument stability and remarkable data repeatability. Furthermore, a distinct separation between groups was observed, particularly prominent in the positive ion mode, indicating significant variations in metabolites among different groups that warrant further investigation (<a href="#fig1s2" class="usa-link">Figure 1—figure supplement 2A, B</a>). The findings from Pearson correlation analysis were consistent with those obtained from principal component analysis (<a href="#fig1s2" class="usa-link">Figure 1—figure supplement 2C, D</a>). Strong correlations were identified within samples belonging to the same group, while weak correlations were observed between different groups.</p>
<p>Metabolomics analysis revealed a total of 713 differential metabolites, with 14 up-regulated and 699 down-regulated in the positive ion mode, as well as 230 differential metabolites, including 103 up-regulated and 127 down-regulated in the negative ion mode (<a href="#fig1s2" class="usa-link">Figure 1—figure supplement 2A</a>; <a href="#fig2s1" class="usa-link">Figure 2—figure supplement 1A</a>). Based on the classification of the identified 943 different metabolites, it was observed that amino acids and their derivatives accounted for the highest proportion (17%), followed by organic acids and their derivatives (11%), benzene and substituted derivatives (11%), aldehydes, ketones, esters (11%), and heterocyclic compounds (10%) (<a href="#fig2" class="usa-link">Figure 2c</a>). The volcano plot visually represents the distribution of statistical differences in metabolite abundance between the two groups (<a href="#fig1s2" class="usa-link">Figure 1—figure supplement 2B</a>; <a href="#fig2s1" class="usa-link">Figure 2—figure supplement 1B</a>).</p>
<figure class="fig xbox font-sm" id="fig2"><h4 class="obj_head">Figure 2. Difference analysis and enrichment analysis of metabolome data in <em>F</em>. <em>graminearum</em> PH-1 treated with quinofumelin.</h4>
<figcaption><p>(<strong>A</strong>) Bar chart illustrating the number of differential accumulation metabolites (DAMs) quantity in experimental groups (EC<sub>90</sub>) and control groups (CK) in positive ion mode. (<strong>B</strong>) Volcanic map of all metabolites in positive ion mode. The red scatter points represent up-regulated DAMs, while the blue scatter points represent down-regulated DAMs. (<strong>C</strong>) Classification and proportion of DAMs. (<strong>D</strong>) Bubble diagram of differential accumulation metabolite Kyoto Encyclopedia of Genes and Genomes (KEGG) enrichment pathway. (<strong>E</strong>) Pyrimidine de novo biosynthesis pathway in <em>Fusarium graminearum</em>. The left box plots represent the relative content of eight metabolite samples, while the right box represents the relative content of six gene samples. The color gradient from navy blue to firebrick red indicates a progression from low to high content.</p></figcaption><p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12367298_elife-105892-fig2.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8be2/12367298/4f5b0edfeda8/elife-105892-fig2.jpg" loading="lazy" height="615" width="735" alt="Figure 2."></a></p>
<div class="p"><figure class="fig xbox font-sm" id="fig2s1"><h5 class="obj_head">Figure 2—figure supplement 1. Difference analysis of metabolic group data.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12367298_elife-105892-fig2-figsupp1.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8be2/12367298/fc9b32860522/elife-105892-fig2-figsupp1.jpg" loading="lazy" height="528" width="732" alt="Figure 2—figure supplement 1."></a></p>
<figcaption><div class="p">(<strong>A</strong>) A bar chart illustrating the number of differentially accumulated metabolite (DAM) quantity in experimental group (EC<sub>90</sub>) and control group (CK) in the negative ion mode. (<strong>B</strong>) Volcanic map of all metabolites in the negative ion mode. The red scatter points represent the up-regulated DAMs, and the blue scatter points represent the down-regulated DAMs. (<strong>C</strong>) Pie diagram of differential metabolite enrichment pathway classification.</div></figcaption></figure></div>
<div class="p"><figure class="fig xbox font-sm" id="fig2s2"><h5 class="obj_head">Figure 2—figure supplement 2. DHODHII gene expression in <em>F</em>. <em>graminearum</em> is affected by quinofumelin.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12367298_elife-105892-fig2-figsupp2.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8be2/12367298/f6f67045b39c/elife-105892-fig2-figsupp2.jpg" loading="lazy" height="609" width="735" alt="Figure 2—figure supplement 2."></a></p>
<figcaption><div class="p">After the strain PH-1 was cultured in Yeast extract peptone dextrose (YEPD) medium for 36 hr, the EC<sub>50</sub> (0.035 μg/mL) and EC<sub>90</sub> (1 μg/mL) of quinofumelin were added, respectively, followed by an additional 12 hr incubation period. The expression level of the DHODHII gene was subsequently quantified using RT-qPCR. Data represent the mean values and standard errors derived from three independent replicates. Letters above the columns indicate statistically significant differences among treatments (p&lt;0.05, ANOVA, LSD).</div></figcaption></figure></div>
<div class="p text-right font-secondary"><a href="figure/fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></figure><p>KEGG pathways enriched by differential metabolites primarily include arachidonic acid metabolism, pyrimidine metabolism, alanine, aspartate, and glutamate metabolism, purine metabolism, citrate cycle, arginine and proline metabolism, fructose and mannose metabolism, nitrogen metabolism, and metabolic pathways (<a href="#fig2" class="usa-link">Figure 2D</a>). The classification of enrichment pathways for differential metabolites revealed that the majority of these metabolites were associated with nucleotide metabolism (25%), amino acid metabolism (25%), carbohydrate metabolism (25%), and energy metabolism (12.5%) (<a href="#fig2s1" class="usa-link">Figure 2—figure supplement 1C</a>).</p></section><section id="s2-3"><h3 class="pmc_sec_title">Conjoint analysis of transcriptome and metabolome integration</h3>
<p>Through transcriptome and metabolome data analysis, we observed a co-enrichment of differentially expressed genes and differentially accumulated metabolites in the pyrimidine metabolism pathway. In the expression calorigrams of metabolites and genes depicting the de novo biosynthesis pathway of pyrimidine, we identified down-regulation of dihydroorotate dehydrogenase (DHODH) in the experimental groups. Through in vitro RT-qPCR verification, it was found that after treating with quinofumelin at EC<sub>50</sub> (0.035 μg/mL) and EC<sub>90</sub> (1 μg/mL) concentrations, the expression of the DHODH gene was, respectively, decreased by 11.91% and 33.77% (p＜0.05) (<a href="#fig2s2" class="usa-link">Figure 2—figure supplement 2</a>). This down-regulation hinders the conversion of dihydroorotate to orotate, thereby inhibiting the biosynthesis of downstream metabolites orotidine-5P and uracil, leading to their decreased abundance (<a href="#fig2" class="usa-link">Figure 2E</a>). Thus, we conclude that the action mechanism of quinofumelin may involve inhibition of the de novo biosynthesis pathway of pyrimidine.</p></section><section id="s2-4"><h3 class="pmc_sec_title">Identiﬁcation of DHODHII in <em>F. graminearum</em>
</h3>
<p>The <em>FgDHODHII</em> gene (FGSG_09678) was identified through the integrated analysis of transcriptome and metabolome data. The gene sequence for <em>FgDHODHII</em> was retrieved from the National Center for Biotechnology Information (NCBI) GenBank database. <em>FgDHODHII</em> is a gene consisting of 1419 bp without introns and is predicted to encode a protein consisting of 472 amino acids. Orthologs of <em>FgDHODHII</em> were obtained using the BLAST algorithm. The phylogenetic tree was constructed based on the amino acid sequences of DHODHII homologous proteins with Mega X using the neighbor-joining method. Motif pattern information was generated using the MEME suite (<a href="#bib2" class="usa-link" aria-describedby="bib2">Bailey et al., 2009</a>), while functional domain information and protein coordinates were obtained through CD-search (<a href="#bib23" class="usa-link" aria-describedby="bib23">Marchler-Bauer et al., 2009</a>). TBtools software was used for visualizing the tree representation (<a href="#bib7" class="usa-link" aria-describedby="bib7">Chen et al., 2020</a>). All DHODHII proteins possess the dihydroorotate dehydrogenase domain (<a href="#fig3" class="usa-link">Figure 3</a>), indicating high conservation among fungi. Additionally, the amino acid sequence of the FgDHODHII exhibited 551% similarity to that of DHODHII from <em>Pyricularia oryzae</em>, as previously reported (<a href="#bib13" class="usa-link" aria-describedby="bib13">Higashimura et al., 2022</a>).</p>
<figure class="fig xbox font-sm" id="fig3"><h4 class="obj_head">Figure 3. Phylogenetic tree of DHODHII proteins.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12367298_elife-105892-fig3.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8be2/12367298/adddbf8bf298/elife-105892-fig3.jpg" loading="lazy" height="469" width="735" alt="Figure 3."></a></p>
<div class="p text-right font-secondary"><a href="figure/fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>The phylogenetic tree was constructed based on the amino acid sequences of DHODHII homologous proteins with Mega X using the neighbor-joining method. The bootstrap values from 1000 replications are indicated on the branches. Motif pattern information was generated using the MEME suite12, while functional domain information and protein coordinates were obtained through CD-search. The final phylogenetic tree was visualized using TBtools. FgDHODHII was highlighted in red. The amino acid sequence of <em>DHODHII</em> from <em>Alternaria alternata</em> (AA0117_g542), <em>Aspergillus nidulans</em> (ANIA_05909), <em>Aspergillus oryzae</em> (AO1008_05965), <em>Blumeria graminis</em> (BLGH_00861), <em>Botrytis cinerea</em> (Bcin15g04150), <em>Botrytis elliptica</em> (BELL_0162g00150), <em>Colletotrichum asianum</em> (GQ607_007053), <em>Colletotrichum fioriniae</em> (CFIO01_03831), <em>Colletotrichum higginsianum</em> (CH063_10143), <em>Corynespora cassiicola</em> (BS50DRAFT_576390), <em>Erysiphe necator</em> (EV44_g2092), <em>Erysiphe pulchra</em> (EPUL_004094), <em>Fusarium euwallaceae</em> (BHE90_002825), <em>Fusarium floridanum</em> (CEP51_012005), <em>Fusarium fujikuroi</em> (FFMR_05040), <em>Fusarium graminearum</em> (FGSG_09678), <em>Fusarium nygamai</em> (FNYG_10990), <em>Fusarium odoratissimum</em> (FOIG_08270), <em>Fusarium oxysporum</em> (FOXG_08390), <em>Fusarium pseudograminearum</em> (FPSE_04283), <em>Fusarium solani</em> (NechaG15121), <em>Fusarium verticillioides</em> (FVEG_06288), <em>Magnaporthe oryzae</em> (MGG_08814), <em>Neurospora crassa</em> (NCU06532), <em>Oidium neolycopersici</em> (OnM2_089032), <em>Pestalotiopsis fici</em> (PFICI_04984), <em>Podospora anserina</em> (PODANS_6_7920), <em>Puccinia graminis</em> (PGTG_16739), <em>Puccinia triticina</em> (PTTG_02070), <em>Rhizoctonia solani</em> (RSOLAG22IIIB_08327), <em>Schizosaccharomyces pombe</em> (SPAC57A10.12c), <em>Ustilaginoidea virens</em> (UVI_02052880), <em>Verticillium alfalfae</em> (VDBG_01322), <em>Verticillium dahliae</em> (VDAG_07935), and <em>Zymoseptoria tritici</em> (Mycgr3G72548) were accessed in EnsemblFungi database.</p></figcaption></figure></section><section id="s2-5"><h3 class="pmc_sec_title">Recovery test of mycelial growth suppressed by quinofumelin in <em>Fusarium genus</em>
</h3>
<p>The mycelial growth of <em>F. graminearum</em> PH-1 was significantly inhibited on CZA plates containing quinofumelin at concentrations of 1 μg/mL (<a href="#fig4" class="usa-link">Figure 4</a>). To investigate the potential recovery of mycelial growth by exogenous dihydroorotate, uridine monophosphate (UMP), uridine, or uracil, CZA plates supplemented with quinofumelin were treated with these compounds. Despite the addition of exogenous dihydroorotate at 50 μg/mL failing to restore mycelial growth, the addition of exogenous UMP, uridine, or uracil at 50 μg/mL successfully recovered quinofumelin-induced inhibition of mycelial growth. Similar results were observed in other strains of <em>F. graminearum</em> or <em>F. asiaticum</em> (<a href="#fig4" class="usa-link">Figure 4</a>; <a href="#fig4s1" class="usa-link">Figure 4—figure supplement 1</a>). Thus, it can be concluded that quinofumelin effectively inhibits the pyrimidine biosynthesis pathways.</p>
<figure class="fig xbox font-sm" id="fig4"><h4 class="obj_head">Figure 4. Recovery test of mycelial growth suppressed by quinofumelin.</h4>
<figcaption><p>All strains were incubated on Czapek Solution Agar (CZA) plates at 25°C for 3 days. <em>F. graminearum</em> strains PH-1, 512E-8, SX2117, XY-4, and R10. The left image shows the colony morphology, while the right image is a bar chart of colony diameters.</p></figcaption><p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12367298_elife-105892-fig4.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8be2/12367298/9766e63674b5/elife-105892-fig4.jpg" loading="lazy" height="840" width="735" alt="Figure 4."></a></p>
<div class="p"><figure class="fig xbox font-sm" id="fig4s1"><h5 class="obj_head">Figure 4—figure supplement 1. Recovery test of mycelial growth suppressed by quinofumelin.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12367298_elife-105892-fig4-figsupp1.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8be2/12367298/e0d7ea0a73bc/elife-105892-fig4-figsupp1.jpg" loading="lazy" height="843" width="734" alt="Figure 4—figure supplement 1."></a></p>
<figcaption><div class="p">All strains were incubated on Czapek Solution Agar (CZA) plates at 25°C for 3 days. <em>F. asiaticum</em> strains: 2021, HAC2105, 512 F-10, 512 F-2, and 512 F-6. The left image shows the colony morphology, while the right image is a bar chart of colony diameters.</div></figcaption></figure></div>
<div class="p text-right font-secondary"><a href="figure/fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></figure></section><section id="s2-6"><h3 class="pmc_sec_title">The absence of <em>FgDHODHII</em> is lethal for mycelial growth in <em>F. graminearum</em>
</h3>
<p>To further investigate the biological functions of <em>FgDHODHII</em> in <em>F. graminearum</em>, we employed a homologous recombination strategy to generate target gene deletion mutants. Through PCR analysis, two putative <em>FgDHODHII</em> deletion mutants were successfully identified. The ΔFgDHODHII mutants exhibited distinct phenotypic differences compared to the wild-type progenitor PH-1 and the respective complemented strain (<a href="#fig5" class="usa-link">Figure 5A</a>). These mutants displayed an inability to grow on CZA plates; however, their mycelial growth was restored when supplemented with 50 μg/mL of exogenous UMP, uridine, or uracil, while no recovery was observed with 50 μg/mL of exogenous dihydroorotate (<a href="#fig5" class="usa-link">Figure 5B</a>).</p>
<figure class="fig xbox font-sm" id="fig5"><h4 class="obj_head">Figure 5. Mycelial growth of FgDHODHII deletion mutants on Czapek Solution Agar (CZA) plates.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12367298_elife-105892-fig5.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8be2/12367298/efa496878694/elife-105892-fig5.jpg" loading="lazy" height="788" width="735" alt="Figure 5."></a></p>
<div class="p text-right font-secondary"><a href="figure/fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>(<strong>A</strong>) Mycelial growth of the FgDHODHII deletion mutants and the parental strain was observed in the presence or absence of 50 μg/mL uridine. (<strong>B</strong>) Recovery of mycelial growth of the FgDHODHII deletion mutants was observed in the presence of 50 μg/mL dihydroorotate, uridine monophosphate (UMP), uridine, or uracil. All strains were incubated at 25°C for 3 days.</p></figcaption></figure></section><section id="s2-7"><h3 class="pmc_sec_title">Molecular docking</h3>
<p>To facilitate the molecular docking process between FgDHODHII and quinofumelin, ten distinct binding models were generated using AutoDockTools-1.5.7 software, each accompanied by its respective binding energies calculation results. The most favorable conformational arrangement was determined based on its relative ranking in terms of the computed binding energy values and subsequently visualized through PyMOL 2.5.4 software (<a href="#fig6" class="usa-link">Figure 6A</a>). Notably, the obtained docking results revealed a robust interaction between FgDHODHII protein and quinofumelin as evidenced by a calculated negative free energy value of –6.8 kcal/mol.</p>
<figure class="fig xbox font-sm" id="fig6"><h4 class="obj_head">Figure 6. Validation and analysis of quinofumelin’s binding affinity to FgDHODHII.</h4>
<figcaption><p>(<strong>A</strong>) Three-dimensional model of the wild-type FgDHODHII are depicted, with an enlarged view highlighting the binding site for quinofumelin on the corresponding FgDHODHII models. (<strong>B</strong>) The surface plasmon resonance (SPR) response (RU) analysis and fitting results demonstrate the interaction between quinofumelin and FgDHODHII. (<strong>C</strong>) Microscale thermophoresis (MST) analysis to detect the binding of quinofumelin and FgDHODHII.</p></figcaption><p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12367298_elife-105892-fig6.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8be2/12367298/e3377f55d71f/elife-105892-fig6.jpg" loading="lazy" height="908" width="735" alt="Figure 6."></a></p>
<div class="p"><figure class="fig xbox font-sm" id="fig6s1"><h5 class="obj_head">Figure 6—figure supplement 1. SDS-PAGE electrophoresis of the fusion protein.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12367298_elife-105892-fig6-figsupp1.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8be2/12367298/942e53089c7f/elife-105892-fig6-figsupp1.jpg" loading="lazy" height="1012" width="712" alt="Figure 6—figure supplement 1."></a></p>
<figcaption><div class="p">(<strong>A</strong>) The purification effect of pCold-9 ×His-MBP-TEV-DHODHII. (<strong>B</strong>) The purification effect of pET-28a (+)-DHODHII. (<strong>C</strong>) The purification of pET-28a (+) -DHODHII<sup>124-467</sup>. The lane M is a protein marker; in a and c, lanes 1–8 were supernatant, precipitate, flow through solution, and eluent containing 5, 20, 50, 100, and 250 mM imidazole, respectively. In b, lanes 1–9 were the whole fungus, supernatant, precipitate, flow-through solution, and eluent containing 5, 20, 50, 100, and 250 mM imidazole.</div>
<div class="p">
<section class="sm xbox font-sm" id="fig6s1sdata1"><div class="caption p"><span>Figure 6—figure supplement 1—source data 1. TIFF file containing original western blots for <a href="#fig6s1" class="usa-link">Figure 6—figure supplement 1A</a>, indicating the relevant bands and treatments.</span></div>
<div class="media p" id="d67e882"><div class="caption">
<a href="/articles/instance/12367298/bin/elife-105892-fig6-figsupp1-data1.zip" data-ga-action="click_feat_suppl" class="usa-link">elife-105892-fig6-figsupp1-data1.zip</a><sup> (2.2MB, zip) </sup>
</div></div></section>
</div>
<div class="p">
<section class="sm xbox font-sm" id="fig6s1sdata2"><div class="caption p"><span>Figure 6—figure supplement 1—source data 2. Original files for SDS-PAGE analysis displayed in <a href="#fig6s1" class="usa-link">Figure 6—figure supplement 1A</a>.</span></div>
<div class="media p" id="d67e894"><div class="caption">
<a href="/articles/instance/12367298/bin/elife-105892-fig6-figsupp1-data2.zip" data-ga-action="click_feat_suppl" class="usa-link">elife-105892-fig6-figsupp1-data2.zip</a><sup> (347.3KB, zip) </sup>
</div></div></section>
</div>
<div class="p">
<section class="sm xbox font-sm" id="fig6s1sdata3"><div class="caption p"><span>Figure 6—figure supplement 1—source data 3. TIFF file containing original western blots for <a href="#fig6s1" class="usa-link">Figure 6—figure supplement 1B</a>, indicating the relevant bands and treatments.</span></div>
<div class="media p" id="d67e906"><div class="caption">
<a href="/articles/instance/12367298/bin/elife-105892-fig6-figsupp1-data3.zip" data-ga-action="click_feat_suppl" class="usa-link">elife-105892-fig6-figsupp1-data3.zip</a><sup> (1.5MB, zip) </sup>
</div></div></section>
</div>
<div class="p">
<section class="sm xbox font-sm" id="fig6s1sdata4"><div class="caption p"><span>Figure 6—figure supplement 1—source data 4. Original files for SDS-PAGE analysis displayed in <a href="#fig6s1" class="usa-link">Figure 6—figure supplement 1B</a>.</span></div>
<div class="media p" id="d67e918"><div class="caption">
<a href="/articles/instance/12367298/bin/elife-105892-fig6-figsupp1-data4.zip" data-ga-action="click_feat_suppl" class="usa-link">elife-105892-fig6-figsupp1-data4.zip</a><sup> (646.8KB, zip) </sup>
</div></div></section>
</div>
<div class="p">
<section class="sm xbox font-sm" id="fig6s1sdata5"><div class="caption p"><span>Figure 6—figure supplement 1—source data 5. TIFF file containing original western blots for <a href="#fig6s1" class="usa-link">Figure 6—figure supplement 1C</a>, indicating the relevant bands and treatments.</span></div>
<div class="media p" id="d67e930"><div class="caption">
<a href="/articles/instance/12367298/bin/elife-105892-fig6-figsupp1-data5.zip" data-ga-action="click_feat_suppl" class="usa-link">elife-105892-fig6-figsupp1-data5.zip</a><sup> (1.9MB, zip) </sup>
</div></div></section>
</div>
<div class="p">
<section class="sm xbox font-sm" id="fig6s1sdata6"><div class="caption p"><span>Figure 6—figure supplement 1—source data 6. Original files for SDS-PAGE analysis displayed in <a href="#fig6s1" class="usa-link">Figure 6—figure supplement 1C</a>.</span></div>
<div class="media p" id="d67e942"><div class="caption">
<a href="/articles/instance/12367298/bin/elife-105892-fig6-figsupp1-data6.zip" data-ga-action="click_feat_suppl" class="usa-link">elife-105892-fig6-figsupp1-data6.zip</a><sup> (5.3MB, zip) </sup>
</div></div></section>
</div></figcaption></figure></div>
<div class="p text-right font-secondary"><a href="figure/fig6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></figure></section><section id="s2-8"><h3 class="pmc_sec_title">The precise interaction between FgDHODHII and quinofumelin was verified by surface plasmon resonance analysis</h3>
<p>Surface plasmon resonance (SPR) was utilized to evaluate the interaction affinity between quinofumelin and FgDHODHII in this study. The experiment was conducted using a BIAcore T200 (Cytiva) instrument for biomolecular interaction analysis. FgDHODHII was immobilized as the ligand on the chip surface, while quinofumelin continuously flowed through as the analyte solution. Dilute quinofumelin to different concentration gradients (62.5, 32.15, 15.63, 7.81, 3.91 μM), measure the binding energy, and then fit the binding equilibrium dissociation constant (K<sub>d</sub>) value (<a href="#fig6" class="usa-link">Figure 6B</a>). The SPR results revealed that quinofumelin exhibited fast-binding and fast-dissociation modes when interacting with the target protein FgDHODHII. The calculated K<sub>D</sub> value was determined as 6.606 × 10<sup>–6</sup> M, indicating specific binding of quinofumelin to FgDHODHII.</p></section><section id="s2-9"><h3 class="pmc_sec_title">The interaction between FgDHODHII and quinofumelin was confirmed by microscale thermophoresis</h3>
<p>In this study, microscale thermophoresis (MST) was further used to evaluate the interaction affinity between FgDHODHII and quinofumelin. The strength of the binding can be inferred from the Kd value. The experiment was conducted using MO.Control v1.6.1 and data analysis was performed using MO. Affinity Analysis v2.3. By introducing a fluorescent label GFP to the FgDHODHII protein while maintaining its concentration constant, non-fluorescent labeled small molecules were diluted into 16 concentration gradients. Subsequently, the protein solution and small molecule diluent were mixed at a ratio of 19:1 before being loaded into the capillary for further analysis in the machine. The MST assay demonstrated that FgDHODHII exhibited specific binding to quinofumelin in vitro, with a dissociation constant (Kd) of 0.471±0.268 μM, whereas no binding was observed with phenamacril which lacks cross-resistance with quinofumelin (<a href="#fig6" class="usa-link">Figure 6C</a>).</p></section></section><section id="s3"><h2 class="pmc_sec_title">Discussion</h2>
<p>In over 130 years of research on fungicide biology, more than 30 novel targets for selective fungicides have been identified. Based on these newly discovered targets, over 200 modern selective fungicides have been developed. Consequently, the identification of novel targets for fungicides poses a significantly greater challenge compared to the development of new fungicides. Currently, modern selective fungicides with single action sites are widely applied in agricultural practices for crop disease management. However, these specific types of fungicides encounter substantial obstacles associated with resistance issues when deployed in agricultural fields. Numerous studies have extensively documented cases of plant pathogens developing resistance to modern selective fungicides, resulting in failures or reductions in their efficacy against plant diseases and subsequently leading to decreased crop yields and quality (<a href="#bib10" class="usa-link" aria-describedby="bib10">Duan et al., 2019</a>; <a href="#bib16" class="usa-link" aria-describedby="bib16">King et al., 2021</a>; <a href="#bib22" class="usa-link" aria-describedby="bib22">Mao et al., 2024</a>). For instance, <em>Botrytis cinerea</em>, a significant plant pathogenic fungus, has emerged as a multi-resistant population that is resistant to all currently used modern selective fungicides (<a href="#bib30" class="usa-link" aria-describedby="bib30">Shao et al., 2021</a>; <a href="#bib36" class="usa-link" aria-describedby="bib36">Weber and Petridis, 2023</a>). This poses a formidable challenge for the control of crop gray mold disease caused by <em>B. cinerea</em>. Therefore, there is an urgent need for the development of new fungicides with novel mechanisms of action to effectively manage multiple resistance.</p>
<p>The novel quinoline, quinofumelin, developed by Mitsui Chemicals Co., Ltd. Japan, has demonstrated remarkable antifungal activity against plant pathogens such as <em>F. graminearum</em> and <em>Sclerotinia sclerotiorum</em> (<a href="#bib32" class="usa-link" aria-describedby="bib32">Tao et al., 2021</a>; <a href="#bib38" class="usa-link" aria-describedby="bib38">Xiu et al., 2021</a>). Quinofumelin has been identified as an agrochemical fungicide targeting DHODHII in <em>Pyricularia oryzae</em> (<a href="#bib13" class="usa-link" aria-describedby="bib13">Higashimura et al., 2022</a>). Similarly, ipflufenoquin, another agrochemical fungicide with an analogous mechanism of action to quinofumelin, has been reported to target DHODH in <em>Aspergillus fumigatus</em> (<a href="#bib34" class="usa-link" aria-describedby="bib34">van Rhijn et al., 2024</a>). To elucidate the mode of action of quinofumelin against <em>F. graminearum</em>, we conducted a comprehensive investigation using integrated transcriptomic and metabolomic analyses. The transcriptome data revealed that DEGs were involved in ‘tryptophan metabolism’ and ‘amino sugar and nucleotide sugar metabolism.’ Furthermore, the metabolome data identified DAMs enriched in ‘nucleotide metabolism’ and ‘amino acid metabolism.’ Interestingly, we observed a co-enrichment of DEGs and DAMs in the pyrimidine metabolism pathway. Specifically, we identified a down-regulation of dihydroorotate dehydrogenase (DHODH) in the experimental groups, which impeded the conversion of dihydroorotate to orotate. Consequently, this inhibition led to decreased biosynthesis of downstream metabolites orotidine-5P and uracil. Therefore, we conducted an experiment using <em>F. graminearum</em> standard strain PH-1 to quantify mycelial growth by supplementing exogenous uracil, which is the end product of the uracil biosynthesis pathway. This was performed in the presence of complete inhibition of mycelial growth caused by quinofumelin. Surprisingly, the addition of exogenous uracil restored the inhibited mycelial growth induced by quinofumelin (<a href="#fig4" class="usa-link">Figure 4</a>). Meanwhile, similar results were successfully verified with four other <em>F. graminearum</em> strains and five <em>F. asiaticum</em> strains (<a href="#fig4" class="usa-link">Figure 4</a>, <a href="#fig4s1" class="usa-link">Figure 4—figure supplement 1</a>). Based on our depicted pyrimidine biosynthesis pathway (<a href="#fig2" class="usa-link">Figure 2E</a>), we also supplemented other intermediate products, such as dihydroorotate, orotate, UMP, and uridine, externally. Except for dihydroorotate and orotate, supplementation of exogenous UMP and uridine effectively restored mycelial growth inhibited by quinofumelin as well. These findings strongly support that quinofumelin acts through modulation of the uracil biosynthesis pathway. Based on these findings and in conjunction with previous studies (<a href="#bib13" class="usa-link" aria-describedby="bib13">Higashimura et al., 2022</a>; <a href="#bib34" class="usa-link" aria-describedby="bib34">van Rhijn et al., 2024</a>), these agrochemical fungicides are categorized under FRAC group 52. This classification targets dihydroorotate dehydrogenase (DHODH) during the de novo pyrimidine biosynthesis pathway, as outlined by the Fungicide Resistance Action Committee (<a href="https://www.frac.info" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.frac.info</a>). As a key enzyme in the de novo pyrimidine biosynthesis pathway, DHODH has previously been investigated as a potential target for inhibiting infectious organisms and as an antitumor agent in mammals (<a href="#bib6" class="usa-link" aria-describedby="bib6">Chen et al., 1992</a>; <a href="#bib21" class="usa-link" aria-describedby="bib21">Mao et al., 2021</a>). Inhibiting DHODH can effectively impede the synthesis of DNA, RNA, and glycoproteins, thereby regulating aberrant cell metabolism and proliferation to ultimately achieve therapeutic objectives (<a href="#bib4" class="usa-link" aria-describedby="bib4">Boukalova et al., 2020</a>). Within mammalian cells, DHODH is closely associated with the mitochondrial membrane and plays a pivotal role in the respiratory chain through its interaction with ubiquinone (<a href="#bib5" class="usa-link" aria-describedby="bib5">Chen and Jones, 1976</a>). Here, using amino acid sequences from <em>Homo sapiens</em> DHODHII as a reference query, we have identified FgDHODHII (FGSG_09678), a gene comprising of 1419 bp without any introns, which encodes 472 amino acids in <em>F. graminearum</em>. FgDHODHII possesses a typical DHOD-2 like domain (NCBI: cd04738) and belongs to class 2 DHODs. Class 2 DHODHs are membrane-bound enzymes, whereas class 1 enzymes are located in the cytosol (<a href="#bib25" class="usa-link" aria-describedby="bib25">Nørager et al., 2002</a>). The DHODH enzyme catalyzes a unique redox reaction in the de novo pyrimidine biosynthesis pathway (<a href="#bib9" class="usa-link" aria-describedby="bib9">Dorasamy et al., 2017</a>). The DHOD-2-like domain can bind orotate and flavin mononucleotide (FMN), facilitating the oxidation of dihydroorotate to orotate and the reduction of the FMN cofactor (<a href="#bib1" class="usa-link" aria-describedby="bib1">Alves et al., 2015</a>). Based on the aforementioned findings, we have concluded that DHODHII functions as the target of action for quinofumelin against <em>F. graminearum</em>. Consequently, we performed a protoplast transformation assay to delete FgDHODHII in <em>F. graminearum</em> and further validate the precise target of action for quinofumelin. Typically, most fungicide targets are either lethal genes or essential pathogenic genes. Therefore, it is plausible that the inactivation of FgDHODHII may be lethal. In our genetic transformation experiment, exogenous uridine was supplemented to successfully obtain deletion mutants of the FgDHODHII gene. Fortunately, two ΔFgDHODHII mutants were successfully obtained. These mutants exhibited an inability to grow on PDA or CZA medium; however, mycelial growth was fully restored upon addition of uridine to the medium (<a href="#fig5" class="usa-link">Figure 5A</a>). Additionally, exogenous supplementation of UMP or uracil effectively restored mycelial growth. However, the supplementation of dihydroorotate to the medium did not alleviate the growth defect observed in ΔFgDHODHII mutants (<a href="#fig5" class="usa-link">Figure 5B</a>). Furthermore, mycelial growth in two ΔFgDHODHII mutants was restored by exogenously adding UMP, uracil or uridine under quinofumelin treatments. These findings indicate that the defects of FgDHODHII are functionally equivalent to complete inhibition of its activity by quinofumelin and further confirm that the mechanism of action for quinofumelin involves de novo synthesis of uracil while identifying FgDHODHII as its specific target (<a href="#fig7" class="usa-link">Figure 7</a>).</p>
<figure class="fig xbox font-sm" id="fig7"><h3 class="obj_head">Figure 7. A model of the pyrimidine pathway that quinofumelin attacks.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12367298_elife-105892-fig7.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8be2/12367298/16a386091213/elife-105892-fig7.jpg" loading="lazy" height="442" width="735" alt="Figure 7."></a></p>
<div class="p text-right font-secondary"><a href="figure/fig7/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></figure><p>Molecular docking is a computational procedure employed to predict the optimal binding orientation of a ligand with its macromolecular target (receptor), thereby forming a stable complex. In concise terms, docking is a molecular modeling technique employed to predict the interaction between a protein (enzyme) and small molecules (ligands). In this study, we conducted molecular docking analysis between quinofumelin and the target protein FgDHODHII. The docking results unveiled a robust affinity between FgDHODHII and quinofumelin. Furthermore, these findings provided additional evidence that the strong affinity of quinofumelin with FgDHODHII disrupts its activity and function.</p>
<p>SPR technique is employed for real-time measurement of molecular interactions (<a href="#bib26" class="usa-link" aria-describedby="bib26">Olaru et al., 2015</a>). The SPR sensor map characterizes intermolecular interaction rates primarily by conveying the kinetic rate constant, while it characterizes the binding level between molecules mainly by conveying the affinity constant (<a href="#bib12" class="usa-link" aria-describedby="bib12">Guo, 2012</a>; <a href="#bib28" class="usa-link" aria-describedby="bib28">Piliarik et al., 2009</a>). In this study, we used SPR technology to measure the binding energy and kinetics between FgDHODHII and quinofumelin. The binding kinetics of FgDHODHII and quinofumelin demonstrated fast-binding and fast-dissociation modes. Soluble FgDHODHII exhibited an ideal affinity (Kd) of 6.606 × 10<sup>–6</sup> M with quinofumelin, falling within the suitable range of 10<sup>–3</sup> M to 10<sup>–6</sup> M for biomacromolecule-small molecule interactions. Furthermore, the binding affinity between FgDHODHII and quinofumelin was confirmed by a microscale thermophoresis (MST) assay. Therefore, these data strongly support the precise interaction between quinofumelin and its target protein FgDHODHII. This newly discovered selective target for antimicrobial agents offers a valuable resource for the design and development of targeted pesticides.</p>
<p>Unlike many other dehydrogenases that require NAD or NADP as an electron acceptor, DHODH transfers electrons to ubiquinone, which then transfers electrons into respiratory complex III and eventually to oxygen (<a href="#bib35" class="usa-link" aria-describedby="bib35">Wang and Hekimi, 2016</a>). Thus, the production of orotate in pyrimidine biosynthesis is coupled with the electron transport system and energy production, exerting additional control over nucleotide synthesis and ensuring its coupling to cellular energy metabolism. In the pyrimidine biosynthesis pathway, DHODHII regulates the conversion from dihydroorotate to orotate. Our findings demonstrated that DHODHII is a target of action for quinofumelin. A previous study reported that exogenous orotate but not exogenous dihydroorotate recovered mycelial growth inhibited by quinofumelin, indicating that quinofumelin inhibits the formation of orotate in the pyrimidine biosynthesis pathway (<a href="#bib13" class="usa-link" aria-describedby="bib13">Higashimura et al., 2022</a>). However, we observed that the supplementation of orotate to the medium did not alleviate the growth defect of ΔFgDHODHII mutants. Considering the disparity in mycelial growth inhibition between <em>F. graminearum</em> and other fungi, our data suggest that a nonclassical two-site ping-pong mechanism is the most suitable model for DHODH as previously reported (<a href="#bib6" class="usa-link" aria-describedby="bib6">Chen et al., 1992</a>). In the classical ping-pong mechanism, DHODH interacts with dihydroorotate and ubiquinol in an identical conformation, while orotate and ubiquinone combine with an alternative form (<a href="#bib8" class="usa-link" aria-describedby="bib8">Cleland, 1963</a>). However, similar to brequinar sodium inhibiting the activity of DHODH isolated from mammalian sources, FgDHODHII may follow a nonclassical, two-site ping-pong mechanism. In this non-classical ping-pong mechanism, the two substrates may not share chemical resemblance and, therefore, do not bind to a single binding site but likely bind to two adjacent sites that facilitate reactant transfer. The precise nature of the interaction between quinofumelin and FgDHODHII may require further elucidation studies.</p></section><section id="s4"><h2 class="pmc_sec_title">Materials and methods</h2>
<section id="s4-1"><h3 class="pmc_sec_title">Strains, culture conditions, and fungicides</h3>
<p><em>F. graminearum</em> sensitive strain PH-1 and 9 wild-type strains (<em>F. graminearum</em> and <em>F. asiaticum</em>), and the PH-1 deletion mutants ΔFgDHODHII were maintained in the Fungicide Biology Laboratory, Nanjing Agricultural University.</p>
<p>The media used in this study included potato dextrose agar (PDA) medium, Yeast extract peptone dextrose (YEPD) medium, Czapek Solution Agar (CZA) medium, and LB medium.</p>
<p>Chemical-grade quinofumelin (Mitsui Agricultural Chemical Company of Japan) was prepared with dimethyl sulfoxide at a concentration of 1 × 10<sup>4</sup> μg/mL and stored at 4°C.</p></section><section id="s4-2"><h3 class="pmc_sec_title">Transcriptomic assay of mycelial growth in <em>F. graminearum</em> as affected by quinofumelin</h3>
<p>A total of 20 mL of YEPD medium containing 1 mL of conidia suspension (1 × 10<sup>5</sup> conidia/mL) was incubated with shaking (175 rpm/min) at 25°C. After 24 hr, the medium was added with quinofumelin (EC<sub>90</sub>) at a concentration of 1 μg/mL, while an equal amount of dimethyl sulfoxide was added as the control (CK). The incubation continued for another 48 hr, followed by ﬁltration and collection of hyphae. Subsequently, Genedenovo Biotechnology Co., Ltd. (Guangzhou, China) performed sequencing on the Illumina platform. Quality control measures were implemented to remove linker sequences and low-quality bases from the original reads in order to obtain clean data readings. The filtered data underwent analysis for base composition and mass distribution to visualize data quality. Based on the gene expression data, we used R (<a href="http://www.r-project.org/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://www.r-project.org/</a>) for conducting principal component analysis, while the ‘pheatmap’ package was used to generate the clustering heatmap. DESeq2 was applied to analyze group differences based on quantitative expression results. The screening threshold of p-value &lt;0.05 and FC ≥1.5 or FC ≤0.67 was used to identify unigenes with significant differential expression between the two groups using a t-test. Finally, functional enrichment analysis of GO and KEGG pathways was performed on these differentially expressed unigenes.</p></section><section id="s4-3"><h3 class="pmc_sec_title">Metabonomic assay of mycelial growth in <em>F. graminearum</em> as affected by quinofumelin</h3>
<p>Transcriptome and metabolome samples were obtained from the same treatment group. The widely targeted metabolome analysis was conducted by Metware Biotechnology Co., Ltd. (Wuhan, China). The frozen sample stored at –80°C was thawed on ice. A 400 μL solution (Methanol: Water = 7:3, v/v) containing an internal standard was added to a 20 mg sample and vortexed for 3 min. Subsequently, the sample was sonicated in an ice bath for 10 min and vortexed for 1 min before being placed at –20°C for 30 min. After centrifugation at 12,000 rpm for 10 min (4°C). The sediment was removed, followed by another centrifugation of the supernatant at 12,000 rpm for 3 min (4°C). A volume of 200 μL aliquots of supernatant was transferred for LC-MS analysis. All samples were acquired using the LC-MS system according to machine orders. The original data file acquired by LC-MS was converted into mzML format using ProteoWizard software. Peak extraction, peak alignment, and retention time correction were performed by XCMS program. The identiﬁed metabolites were analyzed using MBEole2.0 (csbg.cnb.csic.es/mbrole2/index.php) and KEGG (<a href="http://www.kegg.jp" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://www.kegg.jp</a>). PLS-DA analysis was used to identify the differentially accumulated metabolites (DAMs), with a screening threshold of p-value &lt;0.05 and FC ≥1.5 or FC ≤0.67. Each sample was replicated six times for biological consistency.</p></section><section id="s4-4"><h3 class="pmc_sec_title">Recovery assay of quinofumelin-induced mycelial growth inhibition in <em>F. graminearum</em>
</h3>
<p><em>F. graminearum</em> sensitive strain PH-1 and 9 wild-type strains (<em>F. graminearum</em> and <em>F. asiaticum</em>), and the PH-1 deletion mutants ΔFgDHODHII, were preincubated on a PDA plate to form mycelial colonies. Mycelial plugs (5 mm in diameter) from the edge of a 3-day-old colony were removed and inoculated onto a CZA plate containing quinofumelin (1 μg/mL) and dihydroorotate, UMP, uridine, or uracil at 50 μg/mL, followed by measurement of colony diameters. Three replicates were conducted for each concentration.</p></section><section id="s4-5"><h3 class="pmc_sec_title">Construction of deletion mutant vectors</h3>
<p>The deletion mutant of <em>FgDHODHII</em> was generated through protoplast transformation, as previously described with some modifications in the transformation procedures (<a href="#bib41" class="usa-link" aria-describedby="bib41">Zhang et al., 2016</a>). For the transformation process, washed protoplasts were suspended in 160 μL sorbitol-tris-calcium chloride (STC) solution and 40 μL sorbitol-peg-tris-calcium chloride (SPTC) solution (40% polyethylene glycol 6000 in STC), along with 2 μg of the aforementioned DNA fragments and 10 μL of heparin (5 mg/mL in STC) to form protoplast suspensions. Subsequently, these mixtures were incubated on ice for 30 min, followed by mixing with 1 mL of SPTC and further incubation at room temperature for 20 min. The resulting suspensions were gently mixed into regeneration medium (RM) (100 mL) containing 50 μg/mL uridine and poured into plates. After incubating for 24 hr at 25°C, the plates were overlaid with selective regeneration medium (SRM) containing 50 μg/mL uridine and continuously incubated for an additional 4 days. The putative transformants could be observed growing on SRM. All primers used are listed in <a href="#table1" class="usa-link">Table 1</a>.</p>
<section class="tw xbox font-sm" id="table1"><h4 class="obj_head">Table 1. Primers used in the study.</h4>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" valign="bottom" rowspan="1" colspan="1">Primer</th>
<th align="left" valign="bottom" rowspan="1" colspan="1">Sequence (5’–3’)</th>
<th align="left" valign="bottom" rowspan="1" colspan="1">Use</th>
</tr></thead>
<tbody>
<tr>
<td align="left" valign="bottom" rowspan="1" colspan="1">FgDHODHII-UF</td>
<td align="left" valign="bottom" rowspan="1" colspan="1">
<em>GGCAGTGAAAGTCTTGTTCAAG</em>
</td>
<td align="left" valign="bottom" rowspan="2" colspan="1">PCR primers to amplify FgDHODHII upstream fragment</td>
</tr>
<tr>
<td align="left" valign="bottom" rowspan="1" colspan="1">FgDHODHII-UR</td>
<td align="left" valign="bottom" rowspan="1" colspan="1">
<em>GCTCCTTCAATATCATCTTCTGTG</em>
<br>
<em>GC</em>
<em>TGGGAAGTAATGCTGGG</em>
</td>
</tr>
<tr>
<td align="left" valign="bottom" rowspan="1" colspan="1">FgDHODHII-DF</td>
<td align="left" valign="bottom" rowspan="1" colspan="1">
<em>GAGACAATACCGGAAGGAAC</em>
<br>
<em>AC</em>
<em>ATGCTATAGCAAGCAACA</em>
</td>
<td align="left" valign="bottom" rowspan="2" colspan="1">PCR primers to amplify FgDHODHII downstream fragment</td>
</tr>
<tr>
<td align="left" valign="bottom" rowspan="1" colspan="1">FgDHODHII-DR</td>
<td align="left" valign="bottom" rowspan="1" colspan="1">
<em>AAACTTAACAAATCATCTTTGCC</em>
</td>
</tr>
<tr>
<td align="left" valign="bottom" rowspan="1" colspan="1">SS-F</td>
<td align="left" valign="bottom" rowspan="1" colspan="1">
<em>ACAGAAGATGATATTGAAGGAGC</em>
</td>
<td align="left" valign="bottom" rowspan="2" colspan="1">PCR primers to amplify Hph-Hsv-tk fragment</td>
</tr>
<tr>
<td align="left" valign="bottom" rowspan="1" colspan="1">SS-R</td>
<td align="left" valign="bottom" rowspan="1" colspan="1">
<em>GAGACAATACCGGAAGGAA</em>
</td>
</tr>
<tr>
<td align="left" valign="bottom" rowspan="1" colspan="1">FgDHODHII-YZ-F</td>
<td align="left" valign="bottom" rowspan="1" colspan="1">
<em>TTTGCCCGATCTCTTGGTCT</em>
</td>
<td align="left" valign="bottom" rowspan="2" colspan="1">PCR primers to verify FgDHODHII</td>
</tr>
<tr>
<td align="left" valign="bottom" rowspan="1" colspan="1">FgDHODHII-YZ-R</td>
<td align="left" valign="bottom" rowspan="1" colspan="1">
<em>AACCACCAGTCTCCATGAGG</em>
</td>
</tr>
<tr>
<td align="left" valign="bottom" rowspan="1" colspan="1">FgDHODHII-GFP-F</td>
<td align="left" valign="bottom" rowspan="1" colspan="1">
<em>CTCATCACCATCACCATCAC</em>
<br>
<em>A</em>
<em>T</em>
<em>GTCTGCCGCTCTCCTACG</em>
</td>
<td align="left" valign="bottom" rowspan="2" colspan="1">Constructed FgDHODHII-GFP fusion vector</td>
</tr>
<tr>
<td align="left" valign="bottom" rowspan="1" colspan="1">FgDHODHII-GFP-R</td>
<td align="left" valign="bottom" rowspan="1" colspan="1">
<em>AGCTCCTCGCCCTTGCTCAC</em>
<br>
<em>CC</em>
<em>TCGAATTCTTCACACCAC</em>
</td>
</tr>
<tr>
<td align="left" valign="bottom" rowspan="1" colspan="1">T7</td>
<td align="left" valign="bottom" rowspan="1" colspan="1">
<em>TAATACGACTCACTATAGG</em>
</td>
<td align="left" valign="bottom" rowspan="2" colspan="1">Amplification of FgDHODHII fragments for sequencing</td>
</tr>
<tr>
<td align="left" valign="bottom" rowspan="1" colspan="1">T7t</td>
<td align="left" valign="bottom" rowspan="1" colspan="1">
<em>CCGCTAGTTATTGCTCAGC</em>
</td>
</tr>
<tr>
<td align="left" valign="bottom" rowspan="1" colspan="1">pET-28a-F</td>
<td align="left" valign="bottom" rowspan="1" colspan="1">
<em>CCTGGTGCCGCGCGGCAGCCATATGCT</em>
<br>
<em>GGCTGTCAATGTTTTTGGAACAG</em>
</td>
<td align="left" valign="bottom" rowspan="2" colspan="1">PCR primers to construct pET28a-FgDHODHII expression vector</td>
</tr>
<tr>
<td align="left" valign="bottom" rowspan="1" colspan="1">pET-28a-R</td>
<td align="left" valign="bottom" rowspan="1" colspan="1">
<em>AAGCTTGTCGACGGAGCTCGAATTCCT</em>
<br>
<em>ATTTTAACTCCTGCTTGATCCTG</em>
</td>
</tr>
<tr>
<td align="left" valign="bottom" rowspan="1" colspan="1">pCold-F</td>
<td align="left" valign="bottom" rowspan="1" colspan="1">
<em>GTAAAGCACGCCATATCGC</em>
</td>
<td align="left" valign="bottom" rowspan="2" colspan="1">PCR primers to construct pCold-FgDHODHII expression vector</td>
</tr>
<tr>
<td align="left" valign="bottom" rowspan="1" colspan="1">pCold-R</td>
<td align="left" valign="bottom" rowspan="1" colspan="1">
<em>CCAAATGGCAGGGATCTTAG</em>
</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/table1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section></section><section id="s4-6"><h3 class="pmc_sec_title">Molecular docking</h3>
<p>The binding affinity of quinofumelin with FgDHODHII protein was assessed through molecular docking using AutoDockTools-1.5.7 software. The three-dimensional structure of quinofumelin (PubChem CID: 23160856) used for docking was obtained from PubChem database, while the three-dimensional structural model of <em>F. graminearum</em> DHODHII protein predicted by AlphaFold (<a href="#bib15" class="usa-link" aria-describedby="bib15">Jumper et al., 2021</a>) was obtained from the UniProt database (<a href="#bib3" class="usa-link" aria-describedby="bib3">Bateman et al., 2019</a>). In the docking studies, binding energy was employed as a metric to evaluate the strength of protein-ligand interaction, where lower values indicated higher affinity between proteins and ligands. The resulting docking outcomes were visualized using PyMOL software version 2.5.4.</p></section><section id="s4-7"><h3 class="pmc_sec_title">SPR verification of specific interactions between FgDHODHII and quinofumelin</h3>
<p>The cDNA sequence of FgDHODHII was analyzed and codon optimization was performed for the <em>E. coli</em> expression system (<a href="#bib17" class="usa-link" aria-describedby="bib17">Kyriakopoulos and Kontoravdi, 2013</a>; <a href="#bib40" class="usa-link" aria-describedby="bib40">Yang et al., 2021</a>). Subsequently, it was ligated into the pCold-9×his MBP plasmid and the pET-28a plasmid at the NdeI and EcoRI sites within the multiple cloning sites to construct recombinant plasmids: pCold-9×his-MBP-TEV-DHODHII (<a href="#bib11" class="usa-link" aria-describedby="bib11">Fang et al., 2018</a>), pET-28a (+)-DHODHII and pET-28a (+)-DHODHII<sup>124-467</sup> (with removal of insoluble fragments) (<a href="#fig6s1" class="usa-link">Figure 6—figure supplement 1</a>).</p>
<p>Using primers T7 and T7t for sequencing resulted in the identification of the correct transformants. Plasmid DNA was extracted using the E.Z.N.A Plasmid Extraction DNA Mini Kit I protocol (Omega, China). For protein overexpression, the extracted plasmids were separately transformed into <em>E. coli</em> BL21 strain (<a href="#bib43" class="usa-link" aria-describedby="bib43">Zhu et al., 2014</a>). A single clone obtained from each transformation was used for protein overexpression and purification. Upon reaching an OD600 of 1.0, 0.6 mM isopropyl-β-D-thiogalactopyranoside (IPTG) was added to induce protein overexpression. Following induction at 16°C for 9 hr, cells were harvested, lysed, and purified using a Ni-NTA column (QIAGEN).</p>
<p>The interaction between FgDHODHII and quinofumelin was evaluated using a Biacore T200 instrument (GE Healthcare). The CM7 sensor chip was employed for immobilizing 50 μg/mL of purified FgDHODHII protein via amino coupling method at a flow rate of 10 μL/min, utilizing 1×PBS-P+containing 5% DMSO as running buffer. The desired coupling amount of FgDHODHII protein on the chip surface was targeted to be within the range of 12,000–24,000 RU. To investigate binding affinity, quinofumelin concentrations were diluted from 3.90625 to 62.5 μM in a constant flow rate running buffer containing 5% DMSO. Following sample injection, a switching buffer was introduced to allow spontaneous dissociation of quinofumelin from FgDHODHII on the chip surface, and this dissociation process was continuously monitored in real time by measuring response values. The affinity value (Kd) was determined through kinetic analysis or steady-state affinity method.</p></section><section id="s4-8"><h3 class="pmc_sec_title">Identification of specific interactions between FgDHODHII and quinofumelin using MST</h3>
<p>To construct the FgDHODHII-GFP cassette, the FgDHODHII fragment containing the native promoter and ORF (without the stop codon) was amplified. Subsequently, the resulting PCR product was co-transformed with XhoI-digested pYF11 into <em>E. coli</em> DH5α as previously reported (<a href="#bib31" class="usa-link" aria-describedby="bib31">Tang et al., 2020</a>). The fusion vector was then used to transform PH-1 to obtain the FgDHODHII-GFP strain. The GFP fusion protein was extracted from mycelia. Two Monolith NT.115 standard capillaries were employed for repeated absorption of a specific protein concentration, and placed in positions 1 and 2 of the sample tray in the MonoTemper Monolith NT.115 instrument. Sixteen PCR tubes were prepared, with 10 μL DMSO added to PCR tubes 2–16 and 20 μL of 5000 μM quinofumelin (or phenamacril) dissolved in DMSO added to PCR tube 1. Then, 10 μL of quinofumelin from PCR tube 1 was transferred and thoroughly mixed with PCR tube 2 using a pipette. These steps were repeated for PCR tubes 3–16. Following mixing with PCR tube 16 (discarding the remaining 10 μL), each PCR tube was thoroughly mixed with 190 μL DHODHII-GFP protein, and incubated at room temperature for 30 min. Monolith NT.115 standard capillaries were inserted into each PCR tube to allow solution entry into the capillary. The capillaries 1–16 were subsequently placed in the sample tray of the device and the program was initiated. All measurements were performed using 80% (Auto-detect) Nano-RED. MonoTemper Monolith NT.115 software was used to evaluate the binding affinity between the target protein and quinofumelin using the Kd values and the signal-to-noise ratio, and the results were combined and analyzed using Mo. Affinity Analysis v2.3 software (<a href="#bib29" class="usa-link" aria-describedby="bib29">Rotem et al., 2016</a>).</p></section><section id="s4-9"><h3 class="pmc_sec_title">Statistical analysis</h3>
<p>The data in the study were analyzed using analysis of variance (ANOVA) with the SPSS 14.0 software (SPSS Inc, Chicago, IL, USA). When ANOVA yielded a significant result (p=0.05), means were compared using Fisher’s protected least significant difference (PLSD).</p></section></section><section id="ack" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>This work was supported by the National Key Research and Development Program of China (2022YFD1400100), and the National Natural Science Foundation of China (32072448).</p></section><section id="funding-statement1" lang="en"><h2 class="pmc_sec_title">Funding Statement</h2>
<p>The funders had no role in study design, data collection and interpretation, or the decision to submit the work for publication.</p></section><section id="_ci93_" lang="en" class="contrib-info"><h2 class="pmc_sec_title">Contributor Information</h2>
<p>Mingguo Zhou, Email: mgzhou@njau.edu.cn.</p>
<p>Yabing Duan, Email: dyb@njau.edu.cn.</p>
<p>Gustavo Henrique Goldman, Universidade de Sao Paulo, São Paulo, Brazil.</p>
<p>Volker Dötsch, Goethe University Frankfurt, Frankfurt am Main, Germany.</p></section><section id="sec23"><h2 class="pmc_sec_title">Funding Information</h2>
<p>This paper was supported by the following grants:</p>
<ul class="list" style="list-style-type:disc">
<li><p>
National Key Research and Development Program of China

2022YFD1400100 to Yabing Duan.</p></li>
<li><p>
National Natural Science Foundation of China

32072448 to Mingguo Zhou.</p></li>
</ul></section><section id="s5"><h2 class="pmc_sec_title">Additional information</h2>
<section id="fn-group1" class="fn-group"><h3 class="pmc_sec_title">Competing interests</h3>
<div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="conf1"><p>No competing interests declared.</p></div></div></section><section id="fn-group2" class="fn-group"><h3 class="pmc_sec_title">Author contributions</h3>
<div class="fn-group p font-secondary-light font-sm">
<div class="fn p" id="con1"><p>Data curation, Software, Investigation, Methodology, Writing - original draft.</p></div>
<div class="fn p" id="con2"><p>Data curation, Software, Investigation, Methodology.</p></div>
<div class="fn p" id="con3"><p>Data curation, Software, Investigation.</p></div>
<div class="fn p" id="con4"><p>Data curation, Investigation, Writing – review and editing.</p></div>
<div class="fn p" id="con5"><p>Data curation, Methodology.</p></div>
<div class="fn p" id="con6"><p>Software.</p></div>
<div class="fn p" id="con7"><p>Supervision, Methodology.</p></div>
<div class="fn p" id="con8"><p>Conceptualization, Supervision, Funding acquisition.</p></div>
<div class="fn p" id="con9"><p>Conceptualization, Supervision, Funding acquisition, Project administration, Writing – review and editing.</p></div>
</div></section></section><section id="s6"><h2 class="pmc_sec_title">Additional files</h2>
<section class="sm xbox font-sm" id="mdar"><div class="caption p"><span>MDAR checklist</span></div>
<div class="media p" id="d67e1547"><div class="caption">
<a href="/articles/instance/12367298/bin/elife-105892-mdarchecklist1.docx" data-ga-action="click_feat_suppl" class="usa-link">elife-105892-mdarchecklist1.docx</a><sup> (87.7KB, docx) </sup>
</div></div></section><section class="sm xbox font-sm" id="supp1"><div class="caption p"><span>Supplementary file 1. Supplementary tables for this article.</span></div>
<div class="media p" id="d67e1552"><div class="caption">
<a href="/articles/instance/12367298/bin/elife-105892-supp1.docx" data-ga-action="click_feat_suppl" class="usa-link">elife-105892-supp1.docx</a><sup> (18.8KB, docx) </sup>
</div></div></section></section><section id="s7"><h2 class="pmc_sec_title">Data availability</h2>
<p>All datasets generated during this study have been deposited in the Dryad Digital Repository and are publicly available under the <a href="https://doi.org/10.5061/dryad.n5tb2rc7r" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://doi.org/10.5061/dryad.n5tb2rc7r</a>.</p>
<p>The following dataset was generated:</p>
<p>
Xiu Q, Yin XR, Chen YY, Zhang ZY, Mao YS, Wang TS, Zhang J, Zhou MG, Duan YB. 2025. Action mechanism of a novel agrichemical quinofumelin against Fusarium graminearum. Dryad Digital Repository. 
</p></section><section id="ref-list1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="ref-list1_sec2"><ol class="ref-list font-sm">
<li id="bib1">
<cite>Alves CN, Silva JRA, Roitberg AE. Insights into the mechanism of oxidation of dihydroorotate to orotate catalysed by human class 2 dihydroorotate dehydrogenase: a QM/MM free energy study. Physical Chemistry Chemical Physics. 2015;17:17790–17796. doi: 10.1039/c5cp02016f.</cite> [<a href="https://doi.org/10.1039/c5cp02016f" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26087682/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Physical%20Chemistry%20Chemical%20Physics&amp;title=Insights%20into%20the%20mechanism%20of%20oxidation%20of%20dihydroorotate%20to%20orotate%20catalysed%20by%20human%20class%202%20dihydroorotate%20dehydrogenase:%20a%20QM/MM%20free%20energy%20study&amp;author=CN%20Alves&amp;author=JRA%20Silva&amp;author=AE%20Roitberg&amp;volume=17&amp;publication_year=2015&amp;pages=17790-17796&amp;pmid=26087682&amp;doi=10.1039/c5cp02016f&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib2">
<cite>Bailey TL, Boden M, Buske FA, Frith M, Grant CE, Clementi L, Ren J, Li WW, Noble WS. MEME SUITE: tools for motif discovery and searching. Nucleic Acids Research. 2009;37:W202–W208. doi: 10.1093/nar/gkp335.</cite> [<a href="https://doi.org/10.1093/nar/gkp335" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2703892/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19458158/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Research&amp;title=MEME%20SUITE:%20tools%20for%20motif%20discovery%20and%20searching&amp;author=TL%20Bailey&amp;author=M%20Boden&amp;author=FA%20Buske&amp;author=M%20Frith&amp;author=CE%20Grant&amp;volume=37&amp;publication_year=2009&amp;pages=W202-W208&amp;pmid=19458158&amp;doi=10.1093/nar/gkp335&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib3">
<cite>Bateman A, Martin MJ, Orchard S, Magrane M, Alpi E, Bely B, Bingley M, Britto R, Bursteinas B, Busiello G, Bye-A-Jee H, Silva A, Giorgi M, Dogan T, Castro LG, Garmiri P, Georghiou G, Gonzales D, Gonzales L, Hatton-Ellis E, Ignatchenko A, Ishtiaq R, Jokinen P, Joshi V, Jyothi D, Lopez R, Luo J, Lussi Y, MacDougall A, Madeira F, Mahmoudy M, Menchi M, Nightingale A, Onwubiko J, Palka B, Pichler K, Pundir S, Qi GY, Raj S, Renaux A, Lopez MR, Saidi R, Sawford T, Shypitsyna A, Speretta E, Turner E, Tyagi N, Vasudev P, Volynkin V, Wardell T, Warner K, Watkins X, Zaru R, Zellner H, Bridge A, Xenarios I, Poux S, Redaschi N, Aimo L, Argoud-Puy G, Auchincloss A, Axelsen K, Bansal P, Baratin D, Blatter MC, Bolleman J, Boutet E, Breuza L, Casals-Casas C, Castro E, Coudert E, Cuche B, Doche M, Dornevil D, Estreicher A, Famiglietti L, Feuermann M, Gasteiger E, Gehant S, Gerritsen V, Gos A, Gruaz N, Hinz U, Hulo C, Hyka-Nouspikel N, Jungo F, Keller G, Kerhornou A, Lara V. UniProt: a worldwide hub of protein knowledge. Nucleic Acids Research. 2019;47:D506–D515. doi: 10.1093/nar/gky1049.</cite> [<a href="https://doi.org/10.1093/nar/gky1049" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6323992/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30395287/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Research&amp;title=UniProt:%20a%20worldwide%20hub%20of%20protein%20knowledge&amp;author=A%20Bateman&amp;author=MJ%20Martin&amp;author=S%20Orchard&amp;author=M%20Magrane&amp;author=E%20Alpi&amp;volume=47&amp;publication_year=2019&amp;pages=D506-D515&amp;pmid=30395287&amp;doi=10.1093/nar/gky1049&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib4">
<cite>Boukalova S, Hubackova S, Milosevic M, Ezrova Z, Neuzil J, Rohlena J. Dihydroorotate dehydrogenase in oxidative phosphorylation and cancer. Biochimica et Biophysica Acta. Molecular Basis of Disease. 2020;1866:165759.  doi: 10.1016/j.bbadis.2020.165759.</cite> [<a href="https://doi.org/10.1016/j.bbadis.2020.165759" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32151633/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biochimica%20et%20Biophysica%20Acta.%20Molecular%20Basis%20of%20Disease&amp;title=Dihydroorotate%20dehydrogenase%20in%20oxidative%20phosphorylation%20and%20cancer&amp;author=S%20Boukalova&amp;author=S%20Hubackova&amp;author=M%20Milosevic&amp;author=Z%20Ezrova&amp;author=J%20Neuzil&amp;volume=1866&amp;publication_year=2020&amp;pages=165759&amp;pmid=32151633&amp;doi=10.1016/j.bbadis.2020.165759&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib5">
<cite>Chen JJ, Jones ME. The cellular location of dihydroorotate dehydrogenase: relation to de novo biosynthesis of pyrimidines. Archives of Biochemistry and Biophysics. 1976;176:82–90. doi: 10.1016/0003-9861(76)90143-0.</cite> [<a href="https://doi.org/10.1016/0003-9861(76)90143-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/184741/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Archives%20of%20Biochemistry%20and%20Biophysics&amp;title=The%20cellular%20location%20of%20dihydroorotate%20dehydrogenase:%20relation%20to%20de%20novo%20biosynthesis%20of%20pyrimidines&amp;author=JJ%20Chen&amp;author=ME%20Jones&amp;volume=176&amp;publication_year=1976&amp;pages=82-90&amp;pmid=184741&amp;doi=10.1016/0003-9861(76)90143-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib6">
<cite>Chen SF, Perrella FW, Behrens DL, Papp LM. Inhibition of dihydroorotate dehydrogenase activity by brequinar sodium. Cancer Research. 1992;52:3521–3527.</cite> [<a href="https://pubmed.ncbi.nlm.nih.gov/1617622/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cancer%20Research&amp;title=Inhibition%20of%20dihydroorotate%20dehydrogenase%20activity%20by%20brequinar%20sodium&amp;author=SF%20Chen&amp;author=FW%20Perrella&amp;author=DL%20Behrens&amp;author=LM%20Papp&amp;volume=52&amp;publication_year=1992&amp;pages=3521-3527&amp;pmid=1617622&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib7">
<cite>Chen C, Chen H, Zhang Y, Thomas HR, Frank MH, He Y, Xia R. TBtools: An integrative toolkit developed for interactive analyses of big biological data. Molecular Plant. 2020;13:1194–1202. doi: 10.1016/j.molp.2020.06.009.</cite> [<a href="https://doi.org/10.1016/j.molp.2020.06.009" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32585190/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Plant&amp;title=TBtools:%20An%20integrative%20toolkit%20developed%20for%20interactive%20analyses%20of%20big%20biological%20data&amp;author=C%20Chen&amp;author=H%20Chen&amp;author=Y%20Zhang&amp;author=HR%20Thomas&amp;author=MH%20Frank&amp;volume=13&amp;publication_year=2020&amp;pages=1194-1202&amp;pmid=32585190&amp;doi=10.1016/j.molp.2020.06.009&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib8">
<cite>Cleland WW. The kinetics of enzyme-catalyzed reactions with two or more substrates or products. III. Prediction of initial velocity and inhibition patterns by inspection. Biochimica et Biophysica Acta. 1963;67:188–196. doi: 10.1016/0006-3002(63)91816-x.</cite> [<a href="https://doi.org/10.1016/0006-3002(63)91816-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/14021669/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biochimica%20et%20Biophysica%20Acta&amp;title=The%20kinetics%20of%20enzyme-catalyzed%20reactions%20with%20two%20or%20more%20substrates%20or%20products.%20III.%20Prediction%20of%20initial%20velocity%20and%20inhibition%20patterns%20by%20inspection&amp;author=WW%20Cleland&amp;volume=67&amp;publication_year=1963&amp;pages=188-196&amp;pmid=14021669&amp;doi=10.1016/0006-3002(63)91816-x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib9">
<cite>Dorasamy MS, Choudhary B, Nellore K, Subramanya H, Wong PF. Dihydroorotate dehydrogenase inhibitors target c-Myc and arrest melanoma, myeloma and lymphoma cells at S-phase. Journal of Cancer. 2017;8:3086–3098. doi: 10.7150/jca.14835.</cite> [<a href="https://doi.org/10.7150/jca.14835" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5604460/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28928900/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Cancer&amp;title=Dihydroorotate%20dehydrogenase%20inhibitors%20target%20c-Myc%20and%20arrest%20melanoma,%20myeloma%20and%20lymphoma%20cells%20at%20S-phase&amp;author=MS%20Dorasamy&amp;author=B%20Choudhary&amp;author=K%20Nellore&amp;author=H%20Subramanya&amp;author=PF%20Wong&amp;volume=8&amp;publication_year=2017&amp;pages=3086-3098&amp;pmid=28928900&amp;doi=10.7150/jca.14835&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib10">
<cite>Duan Y, Xin W, Lu F, Li T, Li M, Wu J, Wang J, Zhou M. Benzimidazole- and QoI-resistance in Corynespora cassiicola populations from greenhouse-cultivated cucumber: An emerging problem in China. Pesticide Biochemistry and Physiology. 2019;153:95–105. doi: 10.1016/j.pestbp.2018.11.006.</cite> [<a href="https://doi.org/10.1016/j.pestbp.2018.11.006" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30744902/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Pesticide%20Biochemistry%20and%20Physiology&amp;title=Benzimidazole-%20and%20QoI-resistance%20in%20Corynespora%20cassiicola%20populations%20from%20greenhouse-cultivated%20cucumber:%20An%20emerging%20problem%20in%20China&amp;author=Y%20Duan&amp;author=W%20Xin&amp;author=F%20Lu&amp;author=T%20Li&amp;author=M%20Li&amp;volume=153&amp;publication_year=2019&amp;pages=95-105&amp;pmid=30744902&amp;doi=10.1016/j.pestbp.2018.11.006&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib11">
<cite>Fang K, Li Y, Yin X, Samad A, Jin T. High-level prokaryotic expression and purification of death domain superfamily with MBP tag. Clinical Laboratory. 2018;64:467–475. doi: 10.7754/Clin.Lab.2017.170922.</cite> [<a href="https://doi.org/10.7754/Clin.Lab.2017.170922" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29739070/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Clinical%20Laboratory&amp;title=High-level%20prokaryotic%20expression%20and%20purification%20of%20death%20domain%20superfamily%20with%20MBP%20tag&amp;author=K%20Fang&amp;author=Y%20Li&amp;author=X%20Yin&amp;author=A%20Samad&amp;author=T%20Jin&amp;volume=64&amp;publication_year=2018&amp;pages=467-475&amp;pmid=29739070&amp;doi=10.7754/Clin.Lab.2017.170922&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib12">
<cite>Guo X. Surface plasmon resonance based biosensor technique: a review. Journal of Biophotonics. 2012;5:483–501. doi: 10.1002/jbio.201200015.</cite> [<a href="https://doi.org/10.1002/jbio.201200015" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22467335/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Biophotonics&amp;title=Surface%20plasmon%20resonance%20based%20biosensor%20technique:%20a%20review&amp;author=X%20Guo&amp;volume=5&amp;publication_year=2012&amp;pages=483-501&amp;pmid=22467335&amp;doi=10.1002/jbio.201200015&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib13">
<cite>Higashimura N, Hamada A, Ohara T, Sakurai S, Ito H, Banba S. The target site of the novel fungicide quinofumelin, Pyricularia oryzae class II dihydroorotate dehydrogenase. Journal of Pesticide Science. 2022;47:190–196. doi: 10.1584/jpestics.D22-027.</cite> [<a href="https://doi.org/10.1584/jpestics.D22-027" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9716045/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36514691/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Pesticide%20Science&amp;title=The%20target%20site%20of%20the%20novel%20fungicide%20quinofumelin,%20Pyricularia%20oryzae%20class%20II%20dihydroorotate%20dehydrogenase&amp;author=N%20Higashimura&amp;author=A%20Hamada&amp;author=T%20Ohara&amp;author=S%20Sakurai&amp;author=H%20Ito&amp;volume=47&amp;publication_year=2022&amp;pages=190-196&amp;pmid=36514691&amp;doi=10.1584/jpestics.D22-027&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib14">
<cite>Ito H, Takada T, Morimoto M, Komai H, Kajino F, Ohara T, Tamagawa Y, Tsuda M, Banba S. Design and biological activity of a novel fungicide, quinofumelin. Journal of Pesticide Science. 2023;48:22–27. doi: 10.1584/jpestics.D22-042.</cite> [<a href="https://doi.org/10.1584/jpestics.D22-042" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9978248/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36874637/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Pesticide%20Science&amp;title=Design%20and%20biological%20activity%20of%20a%20novel%20fungicide,%20quinofumelin&amp;author=H%20Ito&amp;author=T%20Takada&amp;author=M%20Morimoto&amp;author=H%20Komai&amp;author=F%20Kajino&amp;volume=48&amp;publication_year=2023&amp;pages=22-27&amp;pmid=36874637&amp;doi=10.1584/jpestics.D22-042&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib15">
<cite>Jumper J, Evans R, Pritzel A, Green T, Figurnov M, Ronneberger O, Tunyasuvunakool K, Bates R, Žídek A, Potapenko A, Bridgland A, Meyer C, Kohl SAA, Ballard AJ, Cowie A, Romera-Paredes B, Nikolov S, Jain R, Adler J, Back T, Petersen S, Reiman D, Clancy E, Zielinski M, Steinegger M, Pacholska M, Berghammer T, Bodenstein S, Silver D, Vinyals O, Senior AW, Kavukcuoglu K, Kohli P, Hassabis D. Highly accurate protein structure prediction with AlphaFold. Nature. 2021;596:583–589. doi: 10.1038/s41586-021-03819-2.</cite> [<a href="https://doi.org/10.1038/s41586-021-03819-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8371605/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34265844/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Highly%20accurate%20protein%20structure%20prediction%20with%20AlphaFold&amp;author=J%20Jumper&amp;author=R%20Evans&amp;author=A%20Pritzel&amp;author=T%20Green&amp;author=M%20Figurnov&amp;volume=596&amp;publication_year=2021&amp;pages=583-589&amp;pmid=34265844&amp;doi=10.1038/s41586-021-03819-2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib16">
<cite>King KM, Bucur DE, Ritchie F, Hawkins NJ, Kaczmarek AM, Duan Y, Kildea S, West JS, Fraaije BA. Fungicide resistance status and chemical control options for the brassica pathogen Pyrenopeziza brassicae. Plant Pathology. 2021;70:2086–2103. doi: 10.1111/ppa.13441.</cite> [<a href="https://doi.org/10.1111/ppa.13441" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Pathology&amp;title=Fungicide%20resistance%20status%20and%20chemical%20control%20options%20for%20the%20brassica%20pathogen%20Pyrenopeziza%20brassicae&amp;author=KM%20King&amp;author=DE%20Bucur&amp;author=F%20Ritchie&amp;author=NJ%20Hawkins&amp;author=AM%20Kaczmarek&amp;volume=70&amp;publication_year=2021&amp;pages=2086-2103&amp;doi=10.1111/ppa.13441&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib17">
<cite>Kyriakopoulos S, Kontoravdi C. Analysis of the landscape of biologically-derived pharmaceuticals in Europe: dominant production systems, molecule types on the rise and approval trends. European Journal of Pharmaceutical Sciences. 2013;48:428–441. doi: 10.1016/j.ejps.2012.11.016.</cite> [<a href="https://doi.org/10.1016/j.ejps.2012.11.016" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23262060/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=European%20Journal%20of%20Pharmaceutical%20Sciences&amp;title=Analysis%20of%20the%20landscape%20of%20biologically-derived%20pharmaceuticals%20in%20Europe:%20dominant%20production%20systems,%20molecule%20types%20on%20the%20rise%20and%20approval%20trends&amp;author=S%20Kyriakopoulos&amp;author=C%20Kontoravdi&amp;volume=48&amp;publication_year=2013&amp;pages=428-441&amp;pmid=23262060&amp;doi=10.1016/j.ejps.2012.11.016&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib18">
<cite>Lamberth C, Rendine S, Sulzer-Mosse S.  In: Recent Highlights in the Discovery and Optimization of Crop Protection Products. Maienfisch P, Mangelinckx S, editors. Adv Vet Emd-Acad Pr; 2021. Chapter 4 - agrochemical disease control: the story so far; pp. 65–85.</cite> [<a href="https://doi.org/10.1016/B978-0-12-821035-2.00047-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?title=Recent%20Highlights%20in%20the%20Discovery%20and%20Optimization%20of%20Crop%20Protection%20Products&amp;author=C%20Lamberth&amp;author=S%20Rendine&amp;author=S%20Sulzer-Mosse&amp;publication_year=2021&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib19">
<cite>Lamberth C. Latest research trends in agrochemical fungicides: any learnings for pharmaceutical antifungals? ACS Medicinal Chemistry Letters. 2022;13:895–903. doi: 10.1021/acsmedchemlett.2c00113.</cite> [<a href="https://doi.org/10.1021/acsmedchemlett.2c00113" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9190031/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35707143/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=ACS%20Medicinal%20Chemistry%20Letters&amp;title=Latest%20research%20trends%20in%20agrochemical%20fungicides:%20any%20learnings%20for%20pharmaceutical%20antifungals?&amp;author=C%20Lamberth&amp;volume=13&amp;publication_year=2022&amp;pages=895-903&amp;pmid=35707143&amp;doi=10.1021/acsmedchemlett.2c00113&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib20">
<cite>Lucas JA, Hawkins NJ, Fraaije BA. The evolution of fungicide resistance. Advances in Applied Microbiology. 2015;90:29–92. doi: 10.1016/bs.aambs.2014.09.001.</cite> [<a href="https://doi.org/10.1016/bs.aambs.2014.09.001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25596029/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Advances%20in%20Applied%20Microbiology&amp;title=The%20evolution%20of%20fungicide%20resistance&amp;author=JA%20Lucas&amp;author=NJ%20Hawkins&amp;author=BA%20Fraaije&amp;volume=90&amp;publication_year=2015&amp;pages=29-92&amp;pmid=25596029&amp;doi=10.1016/bs.aambs.2014.09.001&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib21">
<cite>Mao C, Liu X, Zhang Y, Lei G, Yan Y, Lee H, Koppula P, Wu S, Zhuang L, Fang B, Poyurovsky MV, Olszewski K, Gan B. DHODH-mediated ferroptosis defence is a targetable vulnerability in cancer. Nature. 2021;593:586–590. doi: 10.1038/s41586-021-03539-7.</cite> [<a href="https://doi.org/10.1038/s41586-021-03539-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8895686/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33981038/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=DHODH-mediated%20ferroptosis%20defence%20is%20a%20targetable%20vulnerability%20in%20cancer&amp;author=C%20Mao&amp;author=X%20Liu&amp;author=Y%20Zhang&amp;author=G%20Lei&amp;author=Y%20Yan&amp;volume=593&amp;publication_year=2021&amp;pages=586-590&amp;pmid=33981038&amp;doi=10.1038/s41586-021-03539-7&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib22">
<cite>Mao Y, Meng X, Zhang Z, Zhang L, Feng Y, Wang T, Zhang J, Cai Y, Wang J, Sheng G, Shen Y, Zhou M, Duan Y. Genetic variation of drug target CYP51 conferring resistance to ergosterol biosynthesis inhibitors in Botrytis cinerea, causing lily gray mould. Industrial Crops and Products. 2024;208:117797.  doi: 10.1016/j.indcrop.2023.117797.</cite> [<a href="https://doi.org/10.1016/j.indcrop.2023.117797" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Industrial%20Crops%20and%20Products&amp;title=Genetic%20variation%20of%20drug%20target%20CYP51%20conferring%20resistance%20to%20ergosterol%20biosynthesis%20inhibitors%20in%20Botrytis%20cinerea,%20causing%20lily%20gray%20mould&amp;author=Y%20Mao&amp;author=X%20Meng&amp;author=Z%20Zhang&amp;author=L%20Zhang&amp;author=Y%20Feng&amp;volume=208&amp;publication_year=2024&amp;pages=117797&amp;doi=10.1016/j.indcrop.2023.117797&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib23">
<cite>Marchler-Bauer A, Anderson JB, Chitsaz F, Derbyshire MK, DeWeese-Scott C, Fong JH, Geer LY, Geer RC, Gonzales NR, Gwadz M, He S, Hurwitz DI, Jackson JD, Ke Z, Lanczycki CJ, Liebert CA, Liu C, Lu F, Lu S, Marchler GH, Mullokandov M, Song JS, Tasneem A, Thanki N, Yamashita RA, Zhang D, Zhang N, Bryant SH. CDD: specific functional annotation with the Conserved Domain Database. Nucleic Acids Research. 2009;37:D205–D210. doi: 10.1093/nar/gkn845.</cite> [<a href="https://doi.org/10.1093/nar/gkn845" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2686570/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18984618/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Research&amp;title=CDD:%20specific%20functional%20annotation%20with%20the%20Conserved%20Domain%20Database&amp;author=A%20Marchler-Bauer&amp;author=JB%20Anderson&amp;author=F%20Chitsaz&amp;author=MK%20Derbyshire&amp;author=C%20DeWeese-Scott&amp;volume=37&amp;publication_year=2009&amp;pages=D205-D210&amp;pmid=18984618&amp;doi=10.1093/nar/gkn845&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib24">
<cite>Morton V, Staub T. A short history of fungicides. APSnet Feature Articles. 2008;1:20080308.  doi: 10.1094/APSnetFeature-2008-0308.</cite> [<a href="https://doi.org/10.1094/APSnetFeature-2008-0308" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=APSnet%20Feature%20Articles&amp;title=A%20short%20history%20of%20fungicides&amp;author=V%20Morton&amp;author=T%20Staub&amp;volume=1&amp;publication_year=2008&amp;pages=20080308&amp;doi=10.1094/APSnetFeature-2008-0308&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib25">
<cite>Nørager S, Jensen KF, Björnberg O, Larsen S. E. coli dihydroorotate dehydrogenase reveals structural and functional distinctions between different classes of dihydroorotate dehydrogenases. Structure. 2002;10:1211–1223. doi: 10.1016/s0969-2126(02)00831-6.</cite> [<a href="https://doi.org/10.1016/s0969-2126(02)00831-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12220493/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Structure&amp;title=E.%20coli%20dihydroorotate%20dehydrogenase%20reveals%20structural%20and%20functional%20distinctions%20between%20different%20classes%20of%20dihydroorotate%20dehydrogenases&amp;author=S%20N%C3%B8rager&amp;author=KF%20Jensen&amp;author=O%20Bj%C3%B6rnberg&amp;author=S%20Larsen&amp;volume=10&amp;publication_year=2002&amp;pages=1211-1223&amp;pmid=12220493&amp;doi=10.1016/s0969-2126(02)00831-6&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib26">
<cite>Olaru A, Bala C, Jaffrezic-Renault N, Aboul-Enein HY. Surface plasmon resonance (SPR) biosensors in pharmaceutical analysis. Critical Reviews in Analytical Chemistry. 2015;45:97–105. doi: 10.1080/10408347.2014.881250.</cite> [<a href="https://doi.org/10.1080/10408347.2014.881250" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25558771/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Critical%20Reviews%20in%20Analytical%20Chemistry&amp;title=Surface%20plasmon%20resonance%20(SPR)%20biosensors%20in%20pharmaceutical%20analysis&amp;author=A%20Olaru&amp;author=C%20Bala&amp;author=N%20Jaffrezic-Renault&amp;author=HY%20Aboul-Enein&amp;volume=45&amp;publication_year=2015&amp;pages=97-105&amp;pmid=25558771&amp;doi=10.1080/10408347.2014.881250&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib27">
<cite>Pertea M, Kim D, Pertea GM, Leek JT, Salzberg SL. Transcript-level expression analysis of RNA-seq experiments with HISAT, StringTie and Ballgown. Nature Protocols. 2016;11:1650–1667. doi: 10.1038/nprot.2016.095.</cite> [<a href="https://doi.org/10.1038/nprot.2016.095" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5032908/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27560171/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Protocols&amp;title=Transcript-level%20expression%20analysis%20of%20RNA-seq%20experiments%20with%20HISAT,%20StringTie%20and%20Ballgown&amp;author=M%20Pertea&amp;author=D%20Kim&amp;author=GM%20Pertea&amp;author=JT%20Leek&amp;author=SL%20Salzberg&amp;volume=11&amp;publication_year=2016&amp;pages=1650-1667&amp;pmid=27560171&amp;doi=10.1038/nprot.2016.095&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib28">
<cite>Piliarik M, Vaisocherová H, Homola J.  In: Biosensors and Biodetection. Rasooly A, Herold KE, editors. Humana Press; 2009. Surface plasmon resonance biosensing; pp. 65–88.</cite> [<a href="https://doi.org/10.1007/978-1-60327-567-5_5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19151937/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?title=Biosensors%20and%20Biodetection&amp;author=M%20Piliarik&amp;author=H%20Vaisocherov%C3%A1&amp;author=J%20Homola&amp;publication_year=2009&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib29">
<cite>Rotem O, Nesper J, Borovok I, Gorovits R, Kolot M, Pasternak Z, Shin I, Glatter T, Pietrokovski S, Jenal U, Jurkevitch E. An extended cyclic Di-GMP network in the predatory bacterium bdellovibrio bacteriovorus. Journal of Bacteriology. 2016;198:127–137. doi: 10.1128/JB.00422-15.</cite> [<a href="https://doi.org/10.1128/JB.00422-15" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4686195/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26324450/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Bacteriology&amp;title=An%20extended%20cyclic%20Di-GMP%20network%20in%20the%20predatory%20bacterium%20bdellovibrio%20bacteriovorus&amp;author=O%20Rotem&amp;author=J%20Nesper&amp;author=I%20Borovok&amp;author=R%20Gorovits&amp;author=M%20Kolot&amp;volume=198&amp;publication_year=2016&amp;pages=127-137&amp;pmid=26324450&amp;doi=10.1128/JB.00422-15&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib30">
<cite>Shao W, Zhao Y, Ma Z. Advances in understanding fungicide resistance in Botrytis cinerea in China. Phytopathology. 2021;111:455–463. doi: 10.1094/PHYTO-07-20-0313-IA.</cite> [<a href="https://doi.org/10.1094/PHYTO-07-20-0313-IA" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33174825/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Phytopathology&amp;title=Advances%20in%20understanding%20fungicide%20resistance%20in%20Botrytis%20cinerea%20in%20China&amp;author=W%20Shao&amp;author=Y%20Zhao&amp;author=Z%20Ma&amp;volume=111&amp;publication_year=2021&amp;pages=455-463&amp;pmid=33174825&amp;doi=10.1094/PHYTO-07-20-0313-IA&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib31">
<cite>Tang G, Chen A, Dawood DH, Liang J, Chen Y, Ma Z. Capping proteins regulate fungal development, DON-toxisome formation and virulence in Fusarium graminearum. Molecular Plant Pathology. 2020;21:173–187. doi: 10.1111/mpp.12887.</cite> [<a href="https://doi.org/10.1111/mpp.12887" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6988429/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31693278/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Plant%20Pathology&amp;title=Capping%20proteins%20regulate%20fungal%20development,%20DON-toxisome%20formation%20and%20virulence%20in%20Fusarium%20graminearum&amp;author=G%20Tang&amp;author=A%20Chen&amp;author=DH%20Dawood&amp;author=J%20Liang&amp;author=Y%20Chen&amp;volume=21&amp;publication_year=2020&amp;pages=173-187&amp;pmid=31693278&amp;doi=10.1111/mpp.12887&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib32">
<cite>Tao X, Zhao H, Xu H, Li Z, Wang J, Song X, Zhou M, Duan Y. Antifungal activity and biological characteristics of the novel fungicide quinofumelin against Sclerotinia sclerotiorum. Plant Disease. 2021;105:2567–2574. doi: 10.1094/PDIS-08-20-1821-RE.</cite> [<a href="https://doi.org/10.1094/PDIS-08-20-1821-RE" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33404275/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Disease&amp;title=Antifungal%20activity%20and%20biological%20characteristics%20of%20the%20novel%20fungicide%20quinofumelin%20against%20Sclerotinia%20sclerotiorum&amp;author=X%20Tao&amp;author=H%20Zhao&amp;author=H%20Xu&amp;author=Z%20Li&amp;author=J%20Wang&amp;volume=105&amp;publication_year=2021&amp;pages=2567-2574&amp;pmid=33404275&amp;doi=10.1094/PDIS-08-20-1821-RE&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib33">
<cite>Umetsu N, Shirai Y. Development of novel pesticides in the 21st century. Journal of Pesticide Science. 2020;45:54–74. doi: 10.1584/jpestics.D20-201.</cite> [<a href="https://doi.org/10.1584/jpestics.D20-201" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7581488/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33132734/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Pesticide%20Science&amp;title=Development%20of%20novel%20pesticides%20in%20the%2021st%20century&amp;author=N%20Umetsu&amp;author=Y%20Shirai&amp;volume=45&amp;publication_year=2020&amp;pages=54-74&amp;pmid=33132734&amp;doi=10.1584/jpestics.D20-201&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib34">
<cite>van Rhijn N, Storer ISR, Birch M, Oliver JD, Bottery MJ, Bromley MJ. Aspergillus fumigatus strains that evolve resistance to the agrochemical fungicide ipflufenoquin in vitro are also resistant to olorofim. Nature Microbiology. 2024;9:29–34. doi: 10.1038/s41564-023-01542-4.</cite> [<a href="https://doi.org/10.1038/s41564-023-01542-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10769868/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38151646/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Microbiology&amp;title=Aspergillus%20fumigatus%20strains%20that%20evolve%20resistance%20to%20the%20agrochemical%20fungicide%20ipflufenoquin%20in%20vitro%20are%20also%20resistant%20to%20olorofim&amp;author=N%20van%20Rhijn&amp;author=ISR%20Storer&amp;author=M%20Birch&amp;author=JD%20Oliver&amp;author=MJ%20Bottery&amp;volume=9&amp;publication_year=2024&amp;pages=29-34&amp;pmid=38151646&amp;doi=10.1038/s41564-023-01542-4&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib35">
<cite>Wang Y, Hekimi S. Understanding Ubiquinone. Trends in Cell Biology. 2016;26:367–378. doi: 10.1016/j.tcb.2015.12.007.</cite> [<a href="https://doi.org/10.1016/j.tcb.2015.12.007" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26827090/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Trends%20in%20Cell%20Biology&amp;title=Understanding%20Ubiquinone&amp;author=Y%20Wang&amp;author=S%20Hekimi&amp;volume=26&amp;publication_year=2016&amp;pages=367-378&amp;pmid=26827090&amp;doi=10.1016/j.tcb.2015.12.007&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib36">
<cite>Weber RWS, Petridis A. Fungicide resistance in Botrytis spp. and regional strategies for its management in Northern European strawberry production. BioTech. 2023;12:12040064.  doi: 10.3390/biotech12040064.</cite> [<a href="https://doi.org/10.3390/biotech12040064" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10660734/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37987481/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BioTech&amp;title=Fungicide%20resistance%20in%20Botrytis%20spp.%20and%20regional%20strategies%20for%20its%20management%20in%20Northern%20European%20strawberry%20production&amp;author=RWS%20Weber&amp;author=A%20Petridis&amp;volume=12&amp;publication_year=2023&amp;pages=12040064&amp;pmid=37987481&amp;doi=10.3390/biotech12040064&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib37">
<cite>Wheeler IE, Hollomon DW, Gustafson G, Mitchell JC, Longhurst C, Zhang Z, Gurr SJ. Quinoxyfen perturbs signal transduction in barley powdery mildew (Blumeria graminis f.sp. hordei) Molecular Plant Pathology. 2003;4:177–186. doi: 10.1046/j.1364-3703.2003.00165.x.</cite> [<a href="https://doi.org/10.1046/j.1364-3703.2003.00165.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20569377/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Plant%20Pathology&amp;title=Quinoxyfen%20perturbs%20signal%20transduction%20in%20barley%20powdery%20mildew%20(Blumeria%20graminis%20f.sp.%20hordei)&amp;author=IE%20Wheeler&amp;author=DW%20Hollomon&amp;author=G%20Gustafson&amp;author=JC%20Mitchell&amp;author=C%20Longhurst&amp;volume=4&amp;publication_year=2003&amp;pages=177-186&amp;pmid=20569377&amp;doi=10.1046/j.1364-3703.2003.00165.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib38">
<cite>Xiu Q, Bi L, Xu H, Li T, Zhou Z, Li Z, Wang J, Duan Y, Zhou M. Antifungal activity of quinofumelin against Fusarium graminearum and its inhibitory effect on DON biosynthesis. Toxins. 2021;13:348.  doi: 10.3390/toxins13050348.</cite> [<a href="https://doi.org/10.3390/toxins13050348" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8151098/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34066154/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Toxins&amp;title=Antifungal%20activity%20of%20quinofumelin%20against%20Fusarium%20graminearum%20and%20its%20inhibitory%20effect%20on%20DON%20biosynthesis&amp;author=Q%20Xiu&amp;author=L%20Bi&amp;author=H%20Xu&amp;author=T%20Li&amp;author=Z%20Zhou&amp;volume=13&amp;publication_year=2021&amp;pages=348&amp;pmid=34066154&amp;doi=10.3390/toxins13050348&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib39">
<cite>Yang X, Wang YF, Byrne R, Schneider G, Yang SY. Concepts of Artificial intelligence for computer-assisted drug discovery. Chemical Reviews. 2019;119:10520–10594. doi: 10.1021/acs.chemrev.8b00728.</cite> [<a href="https://doi.org/10.1021/acs.chemrev.8b00728" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31294972/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Chemical%20Reviews&amp;title=Concepts%20of%20Artificial%20intelligence%20for%20computer-assisted%20drug%20discovery&amp;author=X%20Yang&amp;author=YF%20Wang&amp;author=R%20Byrne&amp;author=G%20Schneider&amp;author=SY%20Yang&amp;volume=119&amp;publication_year=2019&amp;pages=10520-10594&amp;pmid=31294972&amp;doi=10.1021/acs.chemrev.8b00728&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib40">
<cite>Yang D, Prabowo CPS, Eun H, Park SY, Cho IJ, Jiao S, Lee SY. Escherichia coli as a platform microbial host for systems metabolic engineering. Essays in Biochemistry. 2021;65:225–246. doi: 10.1042/EBC20200172.</cite> [<a href="https://doi.org/10.1042/EBC20200172" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33956149/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Essays%20in%20Biochemistry&amp;title=Escherichia%20coli%20as%20a%20platform%20microbial%20host%20for%20systems%20metabolic%20engineering&amp;author=D%20Yang&amp;author=CPS%20Prabowo&amp;author=H%20Eun&amp;author=SY%20Park&amp;author=IJ%20Cho&amp;volume=65&amp;publication_year=2021&amp;pages=225-246&amp;pmid=33956149&amp;doi=10.1042/EBC20200172&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib41">
<cite>Zhang L, Li B, Zhang Y, Jia X, Zhou M. Hexokinase plays a critical role in deoxynivalenol (DON) production and fungal development in Fusarium graminearum. Molecular Plant Pathology. 2016;17:16–28. doi: 10.1111/mpp.12258.</cite> [<a href="https://doi.org/10.1111/mpp.12258" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6638496/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25808544/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Plant%20Pathology&amp;title=Hexokinase%20plays%20a%20critical%20role%20in%20deoxynivalenol%20(DON)%20production%20and%20fungal%20development%20in%20Fusarium%20graminearum&amp;author=L%20Zhang&amp;author=B%20Li&amp;author=Y%20Zhang&amp;author=X%20Jia&amp;author=M%20Zhou&amp;volume=17&amp;publication_year=2016&amp;pages=16-28&amp;pmid=25808544&amp;doi=10.1111/mpp.12258&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib42">
<cite>Zhong L, Li YS, Xiong L, Wang WJ, Wu M, Yuan T, Yang W, Tian CY, Miao Z, Wang TQ, Yang SY. Small molecules in targeted cancer therapy: advances, challenges, and future perspectives. Signal Transduction and Targeted Therapy. 2021;6:201.  doi: 10.1038/s41392-021-00572-w.</cite> [<a href="https://doi.org/10.1038/s41392-021-00572-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8165101/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34054126/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Signal%20Transduction%20and%20Targeted%20Therapy&amp;title=Small%20molecules%20in%20targeted%20cancer%20therapy:%20advances,%20challenges,%20and%20future%20perspectives&amp;author=L%20Zhong&amp;author=YS%20Li&amp;author=L%20Xiong&amp;author=WJ%20Wang&amp;author=M%20Wu&amp;volume=6&amp;publication_year=2021&amp;pages=201&amp;pmid=34054126&amp;doi=10.1038/s41392-021-00572-w&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib43">
<cite>Zhu L, Zhang X, Chang L, Wang A, Feng P, Han L. Molecular cloning, prokaryotic expression and promoter analysis of squalene synthase gene from Schizochytrium Limacinum. Applied Biochemistry and Microbiology. 2014;50:411–419. doi: 10.1134/S0003683814040140.</cite> [<a href="https://doi.org/10.1134/S0003683814040140" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Applied%20Biochemistry%20and%20Microbiology&amp;title=Molecular%20cloning,%20prokaryotic%20expression%20and%20promoter%20analysis%20of%20squalene%20synthase%20gene%20from%20Schizochytrium%20Limacinum&amp;author=L%20Zhu&amp;author=X%20Zhang&amp;author=L%20Chang&amp;author=A%20Wang&amp;author=P%20Feng&amp;volume=50&amp;publication_year=2014&amp;pages=411-419&amp;doi=10.1134/S0003683814040140&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ol></section></section></section><article class="sub-article" id="sa0"><section class="pmc-layout__citation font-secondary font-xs"><div>eLife. doi: <a href="https://doi.org/10.7554/eLife.105892.3.sa0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.7554/eLife.105892.3.sa0</a>
</div></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>eLife Assessment</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Goldman%20GH%22%5BAuthor%5D" class="usa-link" aria-describedby="id10"><span class="name western">Gustavo Henrique Goldman</span></a><div hidden="hidden" id="id10">
<h3><span class="name western">Gustavo Henrique Goldman</span></h3>
<div class="p">
<sup>1</sup>Universidade de Sao Paulo, Brazil</div>
<div>Reviewing Editor</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Goldman%20GH%22%5BAuthor%5D" class="usa-link"><span class="name western">Gustavo Henrique Goldman</span></a>
</div>
</div>
<sup>1</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a.c" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a.c" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a.c" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a.c" class="d-panel p" style="display: none">
<div class="p" id="aff5">
<sup>1</sup>Universidade de Sao Paulo, Brazil</div>
<h4 class="font-secondary">Roles</h4>
<div class="p">
<strong class="contrib"><span class="name western">Gustavo Henrique Goldman</span></strong>: <span class="role">Reviewing Editor</span>
</div>
</div>
<div id="anp_a.c" class="d-panel p" style="display: none"><div class="notes p">
<section id="kwd-group3" class="kwd-group"><p><strong>Keywords:</strong> Convincing</p></section><section id="kwd-group4" class="kwd-group"><p><strong>Keywords:</strong> Valuable</p></section>
</div></div>
<div id="clp_a.c" class="d-panel p" style="display: none"><div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div></div>
</div>
<div></div>
</div></section><section class="body sub-article-body"><hr class="headless">
<p>In this <strong>valuable</strong> study, the authors show the physiological response and molecular pathway mediating the effect of quinofumelin, a developed fungicide with an unknown mechanism. The authors present <strong>convincing</strong> data suggesting the involvement of the uridine/uracil biosynthesis pathway, by combining in vivo microbiology characterization as well as in vitro biochemical binding results.</p></section></article><article class="sub-article" id="sa1"><section class="pmc-layout__citation font-secondary font-xs"><div>eLife. doi: <a href="https://doi.org/10.7554/eLife.105892.3.sa1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.7554/eLife.105892.3.sa1</a>
</div></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Reviewer #2 (Public review):</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Anonymous%22%5BAuthor%5D" class="usa-link" aria-describedby="id11"><span class="name given-only">Anonymous</span></a><div hidden="hidden" id="id11">
<h3><span class="name given-only">Anonymous</span></h3>
<div>Reviewer</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Anonymous%22%5BAuthor%5D" class="usa-link"><span class="name given-only">Anonymous</span></a>
</div>
</div>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a.d" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="clp_a.d" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a.d" class="d-panel p" style="display: none">
<h4 class="font-secondary">Roles</h4>
<div class="p">
<strong class="contrib"><span class="name given-only">Anonymous</span></strong>: <span class="role">Reviewer</span>
</div>
</div>
<div id="clp_a.d" class="d-panel p" style="display: none"><div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div></div>
</div>
<div></div>
</div></section><section class="body sub-article-body"><hr class="headless">
<p>Summary:</p>
<p>In the current study, the authors aim to identify the mode of action/molecular mechanism of characterized a fungicide, quinofumelin, and its biological impact on transcriptomics and metabolomics in Fusarium graminearum and other Fusarium species. Two sets of data were generated between quinofumelin and no treatment group, and differentially abundant transcripts and metabolites were identified, suggesting a potential role of pyrimidine biosynthesis. Upon studying the genetic mutants of the uridine/uracil biosynthesis pathway with quinofumelin treatment and metabolite supplementation, combining in vitro biochemical assay of quinofumelin and F.graminearum dihydroorotate dehydrogenase protein, the authors identified that quinofumelin inhibits the dihydroorotate dehydrogenase and blocks downstream metabolite biosynthesis, limiting fungal metabolism and growth.</p>
<p>Strengths:</p>
<p>Omics datasets were leveraged to understand the physiological impact of quinofumelin, showing the intracellular impact of the fungicide. The characterization of FgDHODHII deletion strains with supplemented metabolites clearly showed the impact of the enzyme on fungal growth. Corroborating in vitro and in vivo data revealed the direct interaction of quinofumelin with Fusarium protein target.</p>
<p>Potential Impact:</p>
<p>Understanding this new mechanism could facilitate rational design or screen for molecules targeting the same pathway, or improve binding affinity and inhibitor potency. Confirming the target of quinofumelin may also help understand its resistance mechanism, and further development of other inhibitory molecules against the target.</p></section></article><article class="sub-article" id="sa2"><section class="pmc-layout__citation font-secondary font-xs"><div>eLife. doi: <a href="https://doi.org/10.7554/eLife.105892.3.sa2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.7554/eLife.105892.3.sa2</a>
</div></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Reviewer #3 (Public review):</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Anonymous%22%5BAuthor%5D" class="usa-link" aria-describedby="id12"><span class="name given-only">Anonymous</span></a><div hidden="hidden" id="id12">
<h3><span class="name given-only">Anonymous</span></h3>
<div>Reviewer</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Anonymous%22%5BAuthor%5D" class="usa-link"><span class="name given-only">Anonymous</span></a>
</div>
</div>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a.e" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="clp_a.e" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a.e" class="d-panel p" style="display: none">
<h4 class="font-secondary">Roles</h4>
<div class="p">
<strong class="contrib"><span class="name given-only">Anonymous</span></strong>: <span class="role">Reviewer</span>
</div>
</div>
<div id="clp_a.e" class="d-panel p" style="display: none"><div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div></div>
</div>
<div></div>
</div></section><section class="body sub-article-body"><hr class="headless">
<p>Summary:</p>
<p>The manuscript shows the mechanism of action of quinofumelin, a novel fungicide, against the fungus Fusarium graminearum. Through omics analysis, phenotypic analysis and in silico approaches, the role of quinofumelin in targeting DHODH is uncovered.</p>
<p>Strengths:</p>
<p>The phenotypic analysis and mutant generation are nice data and add to the role of metabolites in bypassing pyrimidine biosynthesis.</p>
<p>Weaknesses:</p>
<p>The role of DHODH in this class of fungicides has been known and this data does not add any further significance to the field.</p></section></article><article class="sub-article" id="sa3"><section class="pmc-layout__citation font-secondary font-xs"><div>eLife. 2025 Aug 20;14:RP105892. doi: <a href="https://doi.org/10.7554/eLife.105892.3.sa3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.7554/eLife.105892.3.sa3</a>
</div></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Author response</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Xiu%20Q%22%5BAuthor%5D" class="usa-link" aria-describedby="id13"><span class="name western">Qian Xiu</span></a><div hidden="hidden" id="id13">
<h3><span class="name western">Qian Xiu</span></h3>
<div class="p">
<sup>1</sup>Nanjing Agricultural University, Nanjing, China</div>
<div>Author</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Xiu%20Q%22%5BAuthor%5D" class="usa-link"><span class="name western">Qian Xiu</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yin%20X%22%5BAuthor%5D" class="usa-link" aria-describedby="id14"><span class="name western">Xiaoru Yin</span></a><div hidden="hidden" id="id14">
<h3><span class="name western">Xiaoru Yin</span></h3>
<div class="p">
<sup>2</sup>Nanjing Agricultural University, Nanjing, China</div>
<div>Author</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yin%20X%22%5BAuthor%5D" class="usa-link"><span class="name western">Xiaoru Yin</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chen%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id15"><span class="name western">Yuanyuan Chen</span></a><div hidden="hidden" id="id15">
<h3><span class="name western">Yuanyuan Chen</span></h3>
<div class="p">
<sup>3</sup>Nanjing Agricultural University, Nanjing, China</div>
<div>Author</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chen%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yuanyuan Chen</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20Z%22%5BAuthor%5D" class="usa-link" aria-describedby="id16"><span class="name western">Ziyang Zhang</span></a><div hidden="hidden" id="id16">
<h3><span class="name western">Ziyang Zhang</span></h3>
<div class="p">
<sup>4</sup>Nanjing Agricultural University, Nanjing, China</div>
<div>Author</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20Z%22%5BAuthor%5D" class="usa-link"><span class="name western">Ziyang Zhang</span></a>
</div>
</div>
<sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mao%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id17"><span class="name western">Yushuai Mao</span></a><div hidden="hidden" id="id17">
<h3><span class="name western">Yushuai Mao</span></h3>
<div class="p">
<sup>5</sup>Nanjing Agricultural University, Nanjing, China</div>
<div>Author</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mao%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yushuai Mao</span></a>
</div>
</div>
<sup>5</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20T%22%5BAuthor%5D" class="usa-link" aria-describedby="id18"><span class="name western">Tianshi Wang</span></a><div hidden="hidden" id="id18">
<h3><span class="name western">Tianshi Wang</span></h3>
<div class="p">
<sup>6</sup>Nanjing Agricultural University, Nanjing, China</div>
<div>Author</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20T%22%5BAuthor%5D" class="usa-link"><span class="name western">Tianshi Wang</span></a>
</div>
</div>
<sup>6</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id19"><span class="name western">Jie Zhang</span></a><div hidden="hidden" id="id19">
<h3><span class="name western">Jie Zhang</span></h3>
<div class="p">
<sup>7</sup>Nanjing Agricultural University, Nanjing, China</div>
<div>Author</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Jie Zhang</span></a>
</div>
</div>
<sup>7</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhou%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id20"><span class="name western">Mingguo Zhou</span></a><div hidden="hidden" id="id20">
<h3><span class="name western">Mingguo Zhou</span></h3>
<div class="p">
<sup>8</sup>College of Plant Protection, Nanjing Agricultural University, Nanjing, China</div>
<div>Author</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhou%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Mingguo Zhou</span></a>
</div>
</div>
<sup>8</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Duan%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id21"><span class="name western">Yabing Duan</span></a><div hidden="hidden" id="id21">
<h3><span class="name western">Yabing Duan</span></h3>
<div class="p">
<sup>9</sup>Nanjing Agricultural University, Nanjing, China</div>
<div>Author</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Duan%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yabing Duan</span></a>
</div>
</div>
<sup>9</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a.f" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a.f" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a.f" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a.f" class="d-panel p" style="display: none">
<div class="p" id="aff6">
<sup>1</sup>Nanjing Agricultural University, Nanjing, China</div>
<div id="aff7">
<sup>2</sup>Nanjing Agricultural University, Nanjing, China</div>
<div id="aff8">
<sup>3</sup>Nanjing Agricultural University, Nanjing, China</div>
<div id="aff9">
<sup>4</sup>Nanjing Agricultural University, Nanjing, China</div>
<div id="aff10">
<sup>5</sup>Nanjing Agricultural University, Nanjing, China</div>
<div id="aff11">
<sup>6</sup>Nanjing Agricultural University, Nanjing, China</div>
<div id="aff12">
<sup>7</sup>Nanjing Agricultural University, Nanjing, China</div>
<div id="aff13">
<sup>8</sup>College of Plant Protection, Nanjing Agricultural University, Nanjing, China</div>
<div id="aff14">
<sup>9</sup>Nanjing Agricultural University, Nanjing, China</div>
<h4 class="font-secondary">Roles</h4>
<div class="p">
<strong class="contrib"><span class="name western">Qian Xiu</span></strong>: <span class="role">Author</span>
</div>
<div>
<strong class="contrib"><span class="name western">Xiaoru Yin</span></strong>: <span class="role">Author</span>
</div>
<div>
<strong class="contrib"><span class="name western">Yuanyuan Chen</span></strong>: <span class="role">Author</span>
</div>
<div>
<strong class="contrib"><span class="name western">Ziyang Zhang</span></strong>: <span class="role">Author</span>
</div>
<div>
<strong class="contrib"><span class="name western">Yushuai Mao</span></strong>: <span class="role">Author</span>
</div>
<div>
<strong class="contrib"><span class="name western">Tianshi Wang</span></strong>: <span class="role">Author</span>
</div>
<div>
<strong class="contrib"><span class="name western">Jie Zhang</span></strong>: <span class="role">Author</span>
</div>
<div>
<strong class="contrib"><span class="name western">Mingguo Zhou</span></strong>: <span class="role">Author</span>
</div>
<div>
<strong class="contrib"><span class="name western">Yabing Duan</span></strong>: <span class="role">Author</span>
</div>
</div>
<div id="anp_a.f" class="d-panel p" style="display: none"><div class="notes p"><section id="historyfront-stub4" class="history"><p>Collection date 2025.</p></section></div></div>
<div id="clp_a.f" class="d-panel p" style="display: none"><div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div></div>
</div>
<div></div>
</div></section><section class="body sub-article-body"><hr class="headless">
<p>The following is the authors’ response to the original reviews.</p>
<blockquote class="text-italic">
<p>
<strong>Reviewer #1 (Public review):</strong>
</p>
<p>Summary:</p>
<p>Phytophathogens including fungal pathogens such as <em>F. graminearum</em> remain a major threat to agriculture and food security. Several agriculturally relevant fungicides including the potent Quinofumelin have been discovered to date, yet the mechanisms of their action and specific targets within the cell remain unclear. This paper sets out to contribute to addressing these outstanding questions.</p>
</blockquote>
<p>We appreciate the reviewer's accurate summary of our manuscript.</p>
<blockquote class="text-italic">
<p>Strengths:</p>
<p>The paper is generally well-written and provides convincing data to support their claims for the impact of Quinofumelin on fungal growth, the target of the drug, and the potential mechanism. Critically the authors identify an important pyrimidine pathway dihydroorotate dehydrogenase (DHODH) gene FgDHODHII in the pathway or mechanism of the drug from the prominent plant pathogen <em>F. graminearum</em>, confirming it as the target for Quinofumelin. The evidence is supported by transcriptomic, metabolomic as well as MST, SPR, molecular docking/structural biology analyses.</p>
</blockquote>
<p>We appreciate the reviewer's recognition of the strengths of our manuscript.</p>
<blockquote class="text-italic">
<p>Weaknesses:</p>
<p>Whilst the study adds to our knowledge about this drug, it is, however, worth stating that previous reports (although in different organisms) by Higashimura et al., 2022 <a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC9716045/" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://pmc.ncbi.nlm.nih.gov/articles/PMC9716045/</a> had already identified DHODH as the target for Quinofumelin and hence this knowledge is not new and hence the authors may want to tone down the claim that they discovered this mechanism and also give sufficient credit to the previous authors work at the start of the write-up in the introduction section rather than in passing as they did with reference 25? other specific recommendations to improve the text are provided in the recommendations for authors section below.</p>
</blockquote>
<p>We appreciate the reviewer's suggestion. In the revised manuscript, we have incorporated the reference in the introduction section and expanded the discussion of previous work on quinofumelin by Higashimura et al., 2022 in the discussion section to more effectively contextualize their contributions. Moreover, we have made revisions and provided responses in accordance with the recommendations.</p>
<blockquote class="text-italic">
<p>
<strong>Reviewer #2 (Public review):</strong>
</p>
<p>Summary:</p>
<p>In the current study, the authors aim to identify the mode of action/molecular mechanism of characterized a fungicide, quinofumelin, and its biological impact on transcriptomics and metabolomics in Fusarium graminearum and other Fusarium species. Two sets of data were generated between quinofumelin and no treatment group, and differentially abundant transcripts and metabolites were identified. The authors further focused on uridine/uracil biosynthesis pathway, considering the significant up- and down-regulation observed in final metabolites and some of the genes in the pathways. Using a deletion mutant of one of the genes and in vitro biochemical assays, the authors concluded that quinofumelin binds to the dihydroorotate dehydrogenase.</p>
</blockquote>
<p>We appreciate the reviewer's accurate summary of our manuscript.</p>
<blockquote class="text-italic">
<p>Strengths:</p>
<p>Omics datasets were leveraged to understand the physiological impact of quinofumelin, showing the intracellular impact of the fungicide. The characterization of FgDHODHII deletion strains with supplemented metabolites clearly showed the impact of the enzyme on fungal growth.</p>
</blockquote>
<p>We appreciate the reviewer's recognition of the strengths of our manuscript.</p>
<blockquote class="text-italic">
<p>Weaknesses:</p>
<p>Some interpretation of results is not accurate and some experiments lack controls. The comparison between quinofumelin-treated deletion strains, in the presence of different metabolites didn't suggest the fungicide is FgDHODHII specific. A wild type is required in this experiment.</p>
<p>Potential Impact: Confirming the target of quinofumelin may help understand its resistance mehchanism, and further development of other inhibitory molecules against the target.</p>
<p>The manuscript would benefit more in explaining the study rationale if more background on previous characterization of this fungicide on Fusarium is given.</p>
</blockquote>
<p>We appreciate the reviewer's suggestion. Under no treatment with quinofumelin, mycelial growth remains normal and does not require restoration. In the presence of quinofumelin treatment, the supplementation of downstream metabolites in the de novo pyrimidine biosynthesis pathway can restore mycelial growth that is inhibited by quinofumelin. The wild-type control group is illustrated in Figure 4. Figure 5b depicts the phenotypes of the deletion mutants. With respect to the relationship among quinofumelin, FgDHODHII, and other metabolites, quinofumelin specifically targets the key enzyme FgDHODHII in the de novo pyrimidine biosynthesis pathway, disrupting the conversion of dihydroorotate to orotate, which consequently inhibits the synthesis downstream metabolites including uracil. In our previous study, quinofumelin not only exhibited excellent antifungal activity against the mycelial growth and spore germination of <em>F. graminearum</em>, but also inhibited the biosynthesis of deoxynivalenol (DON). We have added this part to the introduction section.</p>
<blockquote class="text-italic">
<p>
<strong>Reviewer #3 (Public review):</strong>
</p>
<p>Summary:</p>
<p>The manuscript shows the mechanism of action of quinofumelin, a novel fungicide, against the fungus Fusarium graminearum. Through omics analysis, phenotypic analysis, and in silico approaches, the role of quinofumelin in targeting DHODH is uncovered.</p>
</blockquote>
<p>We appreciate the reviewer's accurate summary of our manuscript.</p>
<blockquote class="text-italic">
<p>Strengths:</p>
<p>The phenotypic analysis and mutant generation are nice data and add to the role of metabolites in bypassing pyrimidine biosynthesis.</p>
</blockquote>
<p>We appreciate the reviewer's recognition of the strengths of our manuscript.</p>
<blockquote class="text-italic">
<p>Weaknesses:</p>
<p>The role of DHODH in this class of fungicides has been known and this data does not add any further significance to the field. The work of Higashimura et al is not appreciated well enough as they already showed the role of quinofumelin upon DHODH II.</p>
<p>There is no mention of the other fungicide within this class ipflufenoquin, as there is ample data on this molecule.</p>
</blockquote>
<p>We appreciate the reviewer's suggestion. We sincerely appreciate the reviewer's insightful comment regarding the work of Higashimura et al. We agree that their investigation into the role of quinofumelin in DHODH II inhibition provides critical foundational insights for this field. In the revised manuscript, we have incorporated the reference in the introduction section and expanded the discussion of their work in the discussion section to more effectively contextualize their contributions. The information regarding action mechanism of ipflufenoquin against filamentous fungi was added in discussion section.</p>
<blockquote class="text-italic">
<p>
<strong>Reviewer #1 (Recommendations for the authors):</strong>
</p>
<p>(1) Given that the DHODH gene had been identified as a target earlier, could the authors perform blast experiments with this gene instead and let us know the percentage similarity between the FgDHODHII gene and the Pyricularia oryzae class II DHODH gene in the report by Higashimura et al., 2022.</p>
</blockquote>
<p>BLAST experiment revealed that the percentage similarity between the FgDHODHII gene and the class II DHODH gene of <em>P. oryzae</em> was 55.41%. We have added the description ‘Additionally, the amino acid sequence of the FgDHODHII exhibits 55.41% similarity to that of DHODHII from <em>Pyricularia oryzae</em>, as previously reported (Higashimura et al., 2022)’ in section Results.</p>
<blockquote class="text-italic">
<p>(2) Abstract:</p>
<p>The authors started abbreviating new terms e.g. DEG, DMP, etc but then all of a sudden stopped and introduced UMP with no full meaning of the abbreviation. Please give the full meaning of all abbreviations in the text, UMP, STC, RM, etc.</p>
</blockquote>
<p>We have provided the full meaning for all abbreviations as requested.</p>
<blockquote class="text-italic">
<p>(3) Introduction section:</p>
<p>The introduction talks very little about the work of other groups on quinofumelin. Perhaps add this information in and reference them including the work of Higashimura et al., 2022 which has done quite significant work on this topic but is not even mentioned in the background</p>
</blockquote>
<p>We have added the work of other groups on quinofumelin in section introduction.</p>
<blockquote class="text-italic">
<p>(4) General statements:</p>
<p>Please show a model of the pyrimidine pathway that quinofumelin attacks to make it easier for the reader to understand the context. They could just copy this from KEGG</p>
</blockquote>
<p>We have added the model (Fig. 7).</p>
<blockquote class="text-italic">
<p>(5) Line 186:</p>
<p>The authors did a great job of demonstrating interactions with the Quinofumelin and went to lengths to perform MST, SPR, molecular docking, and structural biology analyses yet in the end provide no details about the specific amino acid residues involved in the interaction. I would suggest that site-directed mutagenesis studies be performed on FgDHODHII to identify specific amino acid residues that interact with Quinofumelin and show that their disruption weakens Quinofumelin interaction with FgDHODHII.</p>
</blockquote>
<p>Thank you for this insightful suggestion. We fully agree with the importance of elucidating the interaction mechanism. At present, we are conducting site-directed mutagenesis studies based on interaction sites from docking results and the mutation sites of FgDHODHII from the resistant mutants; however, due to the limitations in the accuracy of existing predictive models, this work remains ongoing. Additionally, we are undertaking co-crystallization experiments of FgDHODHII with quinofumelin to directly and precisely reveal their interaction pattern</p>
<blockquote class="text-italic">
<p>(6) Line 76:</p>
<p>What is the reference or evidence for the statement 'In addition, quinofumelin exhibits no cross-resistance to currently extensively used fungicides, indicating its unique action target against phytopathogenic fungi.</p>
</blockquote>
<p>If two fungicides share the same mechanism of action, they will exhibit cross resistance. Previous studies have demonstrated that quinofumelin retains effective antifungal activity against fungal strains resistant to commercial fungicides, indicating that quinofumelin does not exhibit cross-resistance with other commercially available fungicides and possesses a novel mechanism of action. Additionally, we have added the relevant inference.</p>
<blockquote class="text-italic">
<p>(7) Line 80-82:</p>
<p>Again, considering the work of previous authors, this target is not newly discovered. Please consider toning down this statement 'This newly discovered selective target for antimicrobial agents provides a valuable resource for the design and development of targeted pesticides.'</p>
</blockquote>
<p>We have rewritten the description of this sentence.</p>
<blockquote class="text-italic"><p>(8) Line 138: If the authors have identified DHODH in experimental groups (I assume in F. graminearum), what was the exact locus tag or gene name in F. graminearum, and why not just continue with this gene you identified or what is the point of doing a blast again to find the gene if the DHODH gene if it already came up in your transcriptomic or metabolic studies? This unfortunately doesn't make sense but could be explained better.</p></blockquote>
<p>The information of <em>FgDHODHII</em> (gene ID: FGSG_09678) has been added. We have revised this part.</p>
<blockquote class="text-italic">
<p>
<strong>Reviewer #2 (Recommendations for the authors):</strong>
</p>
<p>(1) Line 40:</p>
<p>Please add a reference.</p>
</blockquote>
<p>We have added the reference</p>
<blockquote class="text-italic">
<p>(2) Line 47:</p>
<p>Please add a reference.</p>
</blockquote>
<p>We have added the reference.</p>
<blockquote class="text-italic">
<p>(3) Line 50:</p>
<p>The lack of target diversity in existing fungicides doesn't necessarily serve as a reason for discovering new targets being more challenging than identifying new fungicides within existing categories, please consider adjusting the argument here. Instead, the authors can consider reasons for the lack of new targets in the field.</p>
</blockquote>
<p>We have revised the description.</p>
<blockquote class="text-italic">
<p>(4) Line 63:</p>
<p>Please cite your source with the new technology.</p>
</blockquote>
<p>We have added the reference.</p>
<blockquote class="text-italic">
<p>(5) Line 68:</p>
<p>What are you referring to for "targeted medicine", do you have a reference?</p>
</blockquote>
<p>We have revised the description and the reference.</p>
<blockquote class="text-italic">
<p>(6) Line 74:</p>
<p>One of the papers referred to "quinoxyfen", what are the similarities and differences between the two? Please elaborate for the readership.</p>
</blockquote>
<p>Quinoxyfen, similar to quinofumelin, contains a quinoline ring structure. It inhibits mycelial growth by disrupting the MAP kinase signaling pathway in fungi (https://www.frac.info). In addition, quinoxyfen still exhibits excellent antifungal activity against the quinofumelin-resistant mutants (the findings from our group), indicating that action mechanism for quinofumelin and quinoxyfen differ.</p>
<blockquote class="text-italic">
<p>(7) Line 84:</p>
<p>Please introduce why RNA-Seq was designed in the study first. What were the groups compared? How was the experiment set up? Without this background, it is hard to know why and how you did the experiment.</p>
</blockquote>
<p>According to your suggestions, we have added the description in Section Results. In addition, the experimental process was described in Section Materials and methods as follows: A total of 20 mL of YEPD medium containing 1 mL of conidia suspension (1×105 conidia/mL) was incubated with shaking (175 rpm/min) at 25°C. After 24 h, the medium was added with quinofumelin at a concentration of 1 μg/mL, while an equal amount of dimethyl sulfoxide was added as the control (CK). The incubation continued for another 48 h, followed by ﬁltration and collection of hyphae. Carry out quantitative expression of genes, and then analyze the differences between groups based on the results of DESeq2 for quantitative expression.</p>
<blockquote class="text-italic">
<p>(8) Figures:</p>
<p>The figure labeling is missing (Figures 1,2,3 etc). Please re-order your figure to match the text</p>
</blockquote>
<p>The figures have been inserted.</p>
<blockquote class="text-italic">
<p>(9) Line. 97:</p>
<p>"Volcano plot" is a common plot to visualize DEGs, you can directly refer to the name.</p>
</blockquote>
<p>We have revised the description.</p>
<blockquote class="text-italic">
<p>(10) Figure 1d, 1e:</p>
<p>Can you separate down- and up-regulated genes here? Does the count refer to gene number?</p>
</blockquote>
<p>The expression information for down- and up-regulated genes is presented in Figure 1a and 1b. However, these bubble plots do not distinguish down- and up-regulated genes. Instead, they only display the significant enrichment of differentially expressed genes in specific metabolic pathways. To more clearly represent the data, we have added the detailed counts of down- and up-regulated genes for each metabolic pathway in Supplementary Table S1 and S2. Here, the term "count" refers to differentially expressed genes that fall within a certain pathway.</p>
<blockquote class="text-italic">
<p>(11) Line 111:</p>
<p>Again, no reasoning or description of why and how the experiment was done here.</p>
</blockquote>
<p>Based on the results of KEGG enrichment analysis, DEMs are associated with pathways such as thiamine metabolism, tryptophan metabolism, nitrogen metabolism, amino acid sugar and nucleotide sugar metabolism, pantothenic acid and CoA biosynthesis, and nucleotide sugar production compounds synthesis. To specifically investigate the metabolic pathways involved action mechanism of quinofumelin, we performed further metabolomic experiments. Therefore, we have added this description according the reviewer’s suggestions.</p>
<blockquote class="text-italic">
<p>(12) Figure 2a:</p>
<p>It seems many more metabolites were reduced than increased. Is this expected? Due to the antifungal activity of this compound, how sick is the fungus upon treatment? A physiological study on F. graminearum (in a dose-dependent manner) should be done prior to the omics study. Why do you think there's a stark difference between positive and negative modes in terms of number of metabolites down- and up-regulated?</p>
</blockquote>
<p>Quinofumelin demonstrates exceptional antifungal activity against <em>Fusarium graminearum</em>. The results indicate that the number of reduced metabolites significantly exceeds the number of increased metabolites upon quinofumelin treatment. Mycelial growth is markedly inhibited under quinofumelin exposure. Prior to conducting omics studies, we performed a series of physiological and biochemical experiments (refer to Qian Xiu's dissertation <a href="https://paper.njau.edu.cn/openfile?dbid=72&amp;objid=50_49_57_56_49_49&amp;flag=free" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://paper.njau.edu.cn/openfile?dbid=72&amp;objid=50_49_57_56_49_49&amp;flag=free</a>). Upon quinofumelin treatment, the number of down-regulated metabolites notably surpasses that of up-regulated metabolites compared to the control group. Based on the findings from the down-regulated metabolites, we conducted experiments by exogenously supplementing these metabolites under quinofumelin treatment to investigate whether mycelial growth could be restored. The results revealed that only the exogenous addition of uracil can restore mycelial growth impaired by quinofumelin.</p>
<p>Quinofumelin exhibits an excellent antifungal activity against <em>F. graminearum</em>. At a concentration of 1 μg/mL, quinofumelin inhibits mycelial growth by up to 90%. This inhibitory effect indicates that life activities of <em>F. graminearum</em> are significantly disrupted by quinofumelin. Consequently, there is a marked difference in down- and up-regulated metabolites between quinofumelin-treated group and untreated control group. The detailed results were presented in Figures 1 and 2.</p>
<blockquote class="text-italic">
<p>(13) Figure 2e:</p>
<p>This is a good analysis. To help represent the data more clearly, the authors can consider representing the expression using fold change with a p-value for each gene.</p>
</blockquote>
<p>To more clearly represent the data, we have incorporated the information on significant differences in metabolites in the de novo pyrimidine biosynthesis pathway, as affected by quinofumelin, in accordance with the reviewer’s suggestions.</p>
<blockquote class="text-italic">
<p>(14) Line 142:</p>
<p>Please indicate fold change and p-value for statistical significance. Did you validate this by RT-qPCR?</p>
</blockquote>
<p>We validated the expression level of the DHODH gene under quinofumelin treatment using RT-qPCR. The results indicated that, upon treatment with the EC50 and EC90 concentrations of quinofumelin, the expression of the DHODH gene was significantly reduced by 11.91% and 33.77%, respectively (P&lt;0.05). The corresponding results have been shown in Figure S4.</p>
<blockquote class="text-italic">
<p>(15) Line 145:</p>
<p>It looks like uracil is the only metabolite differentially abundant in the samples - how did you conclude this whole pathway was impacted by the treatment?</p>
</blockquote>
<p>The experiments involving the exogenous supplementation of uracil revealed that the addition of uracil could restore mycelial growth inhibited by quinofumelin. Consequently, we infer that quinofumelin disrupts the de novo pyrimidine biosynthesis pathway. In addition, as uracil is the end product of the de novo pyrimidine biosynthesis pathway, the disruption of this pathway results in a reduction in uracil levels.</p>
<blockquote class="text-italic">
<p>(16) Figure 3:</p>
<p>What sequence was used as the root of the tree? Why were the species chosen? Since the BLAST query was <em>Homo sapiens</em> sequence, would it be good to use that as the root?</p>
</blockquote>
<p>FgDHODHII sequence was used as the root of the tree. These selected fungal species represent significant plant-pathogenic fungi in agriculture production. According to your suggestion, we have removed the BLAST query of <em>Homo sapiens</em> in Figure 3.</p>
<blockquote class="text-italic">
<p>(17) Figure 4:</p>
<p>How were the concentrations used to test chosen?</p>
</blockquote>
<p>Prior to this experiment, we carried out concentration-dependent exogenous supplementation experiments. The results indicated that 50 μg/mL of uracil can fully restore mycelial growth inhibited by quinofumelin. Consequently, we chose 50 μg/mL as the testing concentration.</p>
<blockquote class="text-italic">
<p>(18) Line 164:</p>
<p>Why do you hypothesize supplementing dihydroorotate would restore resistance? The metabolite seemed accumulated in the treatment condition, whereas downstream metabolites were comparable or even depleted. The DHODH gene expression was suppressed. Would accumulation of dihydroorotate be associated with growth inhibition by quinofumelin? Please include the hypothesis and rationale for the experimental setup.</p>
</blockquote>
<p>DHODH regulates the conversion of dihydroorotate to orotate in the de novo pyrimidine biosynthesis pathway. The inhibition of DHODH by quinofumelin results in the accumulation of dihydroorotate and the depletion of the downstream metabolites, including UMP, uridine and uracil. Consequently, downstream metabolites were considered as positive controls, while upstream metabolite dihydroorotate served as a negative control. This design further demonstrates DHODH as action target of quinofumelin against <em>F. graminearum</em>. In addition, the accumulation of dihydroorotate is not associated with growth inhibition by quinofumelin; however, but the depletion of downstream metabolites in the de novo pyrimidine biosynthesis pathway is closely associated with growth inhibition by quinofumelin.</p>
<blockquote class="text-italic">
<p>(19) Line 168:</p>
<p>I'm not sure if this conclusion is valid from your results in Figure 4 showing which metabolites restore growth.</p>
</blockquote>
<p>o minimize the potential influence of strain-specific effects, five strains were tested in the experiments shown in Figure 4. For each strain, the first row (first column) corresponds to control condition, while second row (first column) represents treatment with 1 μg/mL of quinofumelin, which completely inhibits mycelial growth. The second row (second column) for each strain represents the supplementation with 50 μg/mL of dihydroorotate fails to restore mycelial growth inhibited by quinofumelin. In contrast, the second row (third column, fourth column, fifth colomns) for each strain demonstrated that the supplementation of 50 μg/mL of UMP, uridine and uracil, respectively, can effectively restore mycelial growth inhibited by quinofumelin.</p>
<blockquote class="text-italic">
<p>(20) Figure 5a:</p>
<p>The fact you saw growth of the deletion mutant means it's not lethal. However, the growth was severely inhibited.</p>
</blockquote>
<p>Our experimental results indicate that the growth of the deletion mutant is lethal. The mycelial growth observed originates from mycelial plugs that were not exposed to quinofumelin, rather than from the plates amended with quinofumelin.</p>
<blockquote class="text-italic">
<p>(21) Figure 5b:</p>
<p>Would you expect different restoration of growth in the presence of quinofumelin vs. no treatment? The wild type control is missing here. Any conclusions about the relationship between quinofumelin, FgDHODHII, and other metabolites in the pathway?</p>
</blockquote>
<p>Under no treatment with quinofumelin, mycelial growth remains normal and does not require restoration. In the presence of quinofumelin treatment, the supplementation of downstream metabolites in the de novo pyrimidine biosynthesis pathway can restore mycelial growth that is inhibited by quinofumelin. The wild-type control group is illustrated in Figure 4. Figure 5b depicts the phenotypes of the deletion mutants. With respect to the relationship among quinofumelin, FgDHODHII, and other metabolites, quinofumelin specifically targets the key enzyme FgDHODHII in the de novo pyrimidine biosynthesis pathway, disrupting the conversion of dihydroorotate to orotate, which consequently inhibits the synthesis downstream metabolites including uracil.</p>
<blockquote class="text-italic">
<p>(22) Figure 6b:</p>
<p>Lacking positive and negative controls (known binder and non-binder). What does the Kd (in comparison to other interactions) indicate in terms of binding strength?</p>
</blockquote>
<p>We tested the antifungal activities of publicly reported DHODH inhibitors (such as leflunomide and teriflunomide) against <em>F. graminearum</em>. The results showed that these inhibitors exhibited no significant inhibitory effects against the strain PH-1. Therefore, we lacked an effective chemical for use as a positive control in subsequent experiments. Biacore experiments offers detailed insights into molecular interactions between quinofumelin and DHODHII. As shown in Figure 6b, the left panel illustrates the time-dependent kinetic curve of quinofumelin binding to DHODHII. Within the first 60 s after quinofumelin was introduced onto the DHODHII surface, it bound to the immobilized DHODHII on the chip surface, with the response value increasing proportionally to the quinofumelin concentration. Following cessation of the injection at 60 s, quinofumelin spontaneously dissociated from the DHODHII surface, leading to a corresponding decrease in the response value. The data fitting curve presented on the right panel indicates that the affinity constant KD of quinofumelin for DHODHII is 6.606×10-6 M, which falls within the typical range of KD values (10-3 ~ 10-6 M) for protein-small molecule interaction patterns. A lower KD value indicates a stronger affinity; thus, quinofumelin exhibits strong binding affinity towards DHODHII.</p>
<blockquote class="text-italic">
<p>
<strong>Reviewer #3 (Recommendations for the authors):</strong>
</p>
<p>The authors should add information about the other molecule within this class, ipflufenoquin, and what is known about it. There are already published data on its mode of action on DHODH and the role of pyrimidine biosynthesis.</p>
</blockquote>
<p>We have added the information regarding action mechanism of ipflufenoquin against filamentous fungi in discussion section.</p>
<blockquote class="text-italic"><p>The work of Higashimura et al is not appreciated well enough as they already showed the role of quinofumelin upon DHODH II.</p></blockquote>
<p>We sincerely appreciate the reviewer's insightful comment regarding the work of Higashimura et al. We agree that their investigation into the role of quinofumelin in DHODH II inhibition provides critical foundational insights for this field. In the revised manuscript, we have incorporated the reference in the introduction section and expanded the discussion of their work in the discussion section to more effectively contextualize their contributions.</p>
<blockquote class="text-italic"><p>It is unclear how the protein model was established and this should be included. What species is the molecule from and how was it obtained? How are they different from Fusarium?</p></blockquote>
<p>The three-dimensional structural model of <em>F. graminearum</em> DHODHII protein, as predicted by AlphaFold, was obtained from the UniProt database. Additionally, a detailed description along with appropriate citations has been incorporated in the ‘Manuscript’ file.</p></section></article><article class="sub-article" id="_ad93_"><section class="pmc-layout__citation font-secondary font-xs"><div></div></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Associated Data</h1></hgroup><ul class="d-buttons inline-list"></ul>
<div class="d-panels font-secondary-light"></div>
<div></div>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
</div></section><section class="body sub-article-body"><section id="_adc93_" lang="en" class="data-citations"><h2 class="pmc_sec_title">Data Citations</h2>
<ol class="ref-list font-sm"><li id="_dbx_ref_dataset1">
<cite>Xiu Q, Yin XR, Chen YY, Zhang ZY, Mao YS, Wang TS, Zhang J, Zhou MG, Duan YB. 2025. Action mechanism of a novel agrichemical quinofumelin against Fusarium graminearum. Dryad Digital Repository. </cite> [<a href="https://doi.org/10.5061/dryad.n5tb2rc7r" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC12367298/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40832893/" class="usa-link">PubMed</a>]</li></ol></section><section id="_adsm93_" lang="en" class="supplementary-materials"><h2 class="pmc_sec_title">Supplementary Materials</h2>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="caption p"><span>Figure 6—figure supplement 1—source data 1. TIFF file containing original western blots for <a href="#fig6s1" class="usa-link">Figure 6—figure supplement 1A</a>, indicating the relevant bands and treatments.</span></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12367298/bin/elife-105892-fig6-figsupp1-data1.zip" data-ga-action="click_feat_suppl" class="usa-link">elife-105892-fig6-figsupp1-data1.zip</a><sup> (2.2MB, zip) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="caption p"><span>Figure 6—figure supplement 1—source data 2. Original files for SDS-PAGE analysis displayed in <a href="#fig6s1" class="usa-link">Figure 6—figure supplement 1A</a>.</span></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12367298/bin/elife-105892-fig6-figsupp1-data2.zip" data-ga-action="click_feat_suppl" class="usa-link">elife-105892-fig6-figsupp1-data2.zip</a><sup> (347.3KB, zip) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="caption p"><span>Figure 6—figure supplement 1—source data 3. TIFF file containing original western blots for <a href="#fig6s1" class="usa-link">Figure 6—figure supplement 1B</a>, indicating the relevant bands and treatments.</span></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12367298/bin/elife-105892-fig6-figsupp1-data3.zip" data-ga-action="click_feat_suppl" class="usa-link">elife-105892-fig6-figsupp1-data3.zip</a><sup> (1.5MB, zip) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material4_reqid_"><div class="caption p"><span>Figure 6—figure supplement 1—source data 4. Original files for SDS-PAGE analysis displayed in <a href="#fig6s1" class="usa-link">Figure 6—figure supplement 1B</a>.</span></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12367298/bin/elife-105892-fig6-figsupp1-data4.zip" data-ga-action="click_feat_suppl" class="usa-link">elife-105892-fig6-figsupp1-data4.zip</a><sup> (646.8KB, zip) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material5_reqid_"><div class="caption p"><span>Figure 6—figure supplement 1—source data 5. TIFF file containing original western blots for <a href="#fig6s1" class="usa-link">Figure 6—figure supplement 1C</a>, indicating the relevant bands and treatments.</span></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12367298/bin/elife-105892-fig6-figsupp1-data5.zip" data-ga-action="click_feat_suppl" class="usa-link">elife-105892-fig6-figsupp1-data5.zip</a><sup> (1.9MB, zip) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material6_reqid_"><div class="caption p"><span>Figure 6—figure supplement 1—source data 6. Original files for SDS-PAGE analysis displayed in <a href="#fig6s1" class="usa-link">Figure 6—figure supplement 1C</a>.</span></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12367298/bin/elife-105892-fig6-figsupp1-data6.zip" data-ga-action="click_feat_suppl" class="usa-link">elife-105892-fig6-figsupp1-data6.zip</a><sup> (5.3MB, zip) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material7_reqid_"><div class="caption p"><span>MDAR checklist</span></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12367298/bin/elife-105892-mdarchecklist1.docx" data-ga-action="click_feat_suppl" class="usa-link">elife-105892-mdarchecklist1.docx</a><sup> (87.7KB, docx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material8_reqid_"><div class="caption p"><span>Supplementary file 1. Supplementary tables for this article.</span></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12367298/bin/elife-105892-supp1.docx" data-ga-action="click_feat_suppl" class="usa-link">elife-105892-supp1.docx</a><sup> (18.8KB, docx) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h2 class="pmc_sec_title">Data Availability Statement</h2>
<p>All datasets generated during this study have been deposited in the Dryad Digital Repository and are publicly available under the <a href="https://doi.org/10.5061/dryad.n5tb2rc7r" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://doi.org/10.5061/dryad.n5tb2rc7r</a>.</p>
<p>The following dataset was generated:</p>
<p>
Xiu Q, Yin XR, Chen YY, Zhang ZY, Mao YS, Wang TS, Zhang J, Zhou MG, Duan YB. 2025. Action mechanism of a novel agrichemical quinofumelin against Fusarium graminearum. Dryad Digital Repository. 
</p></section></section></article><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from eLife are provided here courtesy of <strong>eLife Sciences Publications, Ltd</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.7554/eLife.105892"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/elife-105892.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (7.9 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12367298/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12367298/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12367298%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12367298/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12367298/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12367298/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40832893/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12367298/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40832893/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12367298/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12367298/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="meuhfsfJutDWr7oh0YpLCdgLs3A22qu2ZK3Otm8Z3yzPBXxkmnhnWaMvKEeGJ1K2">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
