
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Anoikis-related genes predicts prognosis and therapeutic response in renal cell carcinoma - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="152550B48AF423030550B4001DED4477.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="annmed">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366518/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Annals of Medicine">
<meta name="citation_title" content="Anoikis-related genes predicts prognosis and therapeutic response in renal cell carcinoma">
<meta name="citation_author" content="Lizhi Zhou">
<meta name="citation_author_institution" content="Jiangxi Provincial Key Laboratory of Urinary System Diseases, Department of Urology, The First Affiliated Hospital, Jiangxi Medical College, Nanchang University, Nanchang, Jiangxi, P.R. China">
<meta name="citation_author_institution" content="Postdoctoral Innovation Practice Base, The First Affiliated Hospital, Jiangxi Medical College, Nanchang University, Nanchang, P.R. China">
<meta name="citation_author" content="Hengcheng Lu">
<meta name="citation_author_institution" content="Cardiovascular Research Institute of Jiangxi Province, Jiangxi Provincial People’s Hospital, The First Affiliated Hospital of Nanchang Medical College, Nanchang, Jiangxi Provincial, P.R. China">
<meta name="citation_author" content="Bin Fu">
<meta name="citation_author_institution" content="Jiangxi Provincial Key Laboratory of Urinary System Diseases, Department of Urology, The First Affiliated Hospital, Jiangxi Medical College, Nanchang University, Nanchang, Jiangxi, P.R. China">
<meta name="citation_author" content="Jianhan Fu">
<meta name="citation_author_institution" content="Jiangxi Provincial Key Laboratory of Urinary System Diseases, Department of Urology, The First Affiliated Hospital, Jiangxi Medical College, Nanchang University, Nanchang, Jiangxi, P.R. China">
<meta name="citation_author_institution" content="Department of Urology, Sun Yat-sen Memorial Hospital, Sun Yat-sen University, Guangzhou, P.R. China">
<meta name="citation_publication_date" content="2025 Aug 19">
<meta name="citation_volume" content="57">
<meta name="citation_issue" content="1">
<meta name="citation_firstpage" content="2548042">
<meta name="citation_doi" content="10.1080/07853890.2025.2548042">
<meta name="citation_pmid" content="40830065">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366518/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366518/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366518/pdf/IANN_57_2548042.pdf">
<meta name="description" content="Metastasis represents the primary cause of cancer-related mortality, with a high incidence observed in renal cell carcinoma (RCC). Anoikis, a specialized form of apoptosis, plays a crucial role in preventing displaced cells from adhering to new ...">
<meta name="og:title" content="Anoikis-related genes predicts prognosis and therapeutic response in renal cell carcinoma">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Metastasis represents the primary cause of cancer-related mortality, with a high incidence observed in renal cell carcinoma (RCC). Anoikis, a specialized form of apoptosis, plays a crucial role in preventing displaced cells from adhering to new ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366518/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12366518">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1080/07853890.2025.2548042"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/IANN_57_2548042.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12366518%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12366518/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12366518/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366518/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-annmed.gif" alt="Annals of Medicine logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Annals of Medicine" title="Link to Annals of Medicine" shape="default" href="https://www.tandfonline.com/toc/iann20/current" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Ann Med</button></div>. 2025 Aug 19;57(1):2548042. doi: <a href="https://doi.org/10.1080/07853890.2025.2548042" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1080/07853890.2025.2548042</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Ann%20Med%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ann%20Med%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Ann%20Med%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Ann%20Med%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Anoikis-related genes predicts prognosis and therapeutic response in renal cell carcinoma</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhou%20L%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Lizhi Zhou</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Lizhi Zhou</span></h3>
<div class="p">
<sup>a</sup>Jiangxi Provincial Key Laboratory of Urinary System Diseases, Department of Urology, The First Affiliated Hospital, Jiangxi Medical College, Nanchang University, Nanchang, Jiangxi, P.R. China</div>
<div class="p">
<sup>b</sup>Postdoctoral Innovation Practice Base, The First Affiliated Hospital, Jiangxi Medical College, Nanchang University, Nanchang, P.R. China</div>
<div>Conceptualization, Data curation, Writing – original draft</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhou%20L%22%5BAuthor%5D" class="usa-link"><span class="name western">Lizhi Zhou</span></a>
</div>
</div>
<sup>a,</sup><sup>b,</sup><sup>*</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lu%20H%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Hengcheng Lu</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Hengcheng Lu</span></h3>
<div class="p">
<sup>c</sup>Cardiovascular Research Institute of Jiangxi Province, Jiangxi Provincial People’s Hospital, The First Affiliated Hospital of Nanchang Medical College, Nanchang, Jiangxi Provincial, P.R. China</div>
<div>Conceptualization, Methodology, Writing – original draft</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lu%20H%22%5BAuthor%5D" class="usa-link"><span class="name western">Hengcheng Lu</span></a>
</div>
</div>
<sup>c,</sup><sup>*</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Fu%20B%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Bin Fu</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Bin Fu</span></h3>
<div class="p">
<sup>a</sup>Jiangxi Provincial Key Laboratory of Urinary System Diseases, Department of Urology, The First Affiliated Hospital, Jiangxi Medical College, Nanchang University, Nanchang, Jiangxi, P.R. China</div>
<div>Funding acquisition, Supervision, Validation, Writing – review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Fu%20B%22%5BAuthor%5D" class="usa-link"><span class="name western">Bin Fu</span></a>
</div>
</div>
<sup>a,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Fu%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Jianhan Fu</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Jianhan Fu</span></h3>
<div class="p">
<sup>a</sup>Jiangxi Provincial Key Laboratory of Urinary System Diseases, Department of Urology, The First Affiliated Hospital, Jiangxi Medical College, Nanchang University, Nanchang, Jiangxi, P.R. China</div>
<div class="p">
<sup>d</sup>Department of Urology, Sun Yat-sen Memorial Hospital, Sun Yat-sen University, Guangzhou, P.R. China</div>
<div>Investigation, Methodology, Supervision, Writing – review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Fu%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Jianhan Fu</span></a>
</div>
</div>
<sup>a,</sup><sup>d,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="AF0001">
<sup>a</sup>Jiangxi Provincial Key Laboratory of Urinary System Diseases, Department of Urology, The First Affiliated Hospital, Jiangxi Medical College, Nanchang University, Nanchang, Jiangxi, P.R. China</div>
<div id="AF0002">
<sup>b</sup>Postdoctoral Innovation Practice Base, The First Affiliated Hospital, Jiangxi Medical College, Nanchang University, Nanchang, P.R. China</div>
<div id="AF0003">
<sup>c</sup>Cardiovascular Research Institute of Jiangxi Province, Jiangxi Provincial People’s Hospital, The First Affiliated Hospital of Nanchang Medical College, Nanchang, Jiangxi Provincial, P.R. China</div>
<div id="AF0004">
<sup>d</sup>Department of Urology, Sun Yat-sen Memorial Hospital, Sun Yat-sen University, Guangzhou, P.R. China</div>
<div class="author-notes p">
<div class="fn" id="AUFN1">
<sup>*</sup><p class="display-inline">Lizhi Zhou and Hengcheng Lu contributed equally to this work.</p>
</div>
<div class="fn" id="AUFN2"><p>Supplemental data for this article can be accessed online at <a href="https://doi.org/10.1080/07853890.2025.2548042" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://doi.org/10.1080/07853890.2025.2548042</a>.</p></div>
<div class="fn" id="AN0001">
<sup>✉</sup><p class="display-inline">CONTACT Jianhan Fu <span>415226431@qq.com</span>
Department of Urology, Sun Yat-sen Memorial Hospital, Sun Yat-sen University, Guangzhou,  510120, P.R. China</p>
</div>
<div class="fn" id="AN0002">
<sup>✉</sup><p class="display-inline">Bin Fu <span>urodoc@ncu.edu.cn</span>
Jiangxi Provincial Key Laboratory of Urinary System Diseases, Department of Urology, The First Affiliated Hospital, Jiangxi Medical College, Nanchang University, Nanchang, Jiangxi, P.R. China.</p>
</div>
</div>
<h4 class="font-secondary">Roles</h4>
<div class="p">
<strong class="contrib"><span class="name western">Lizhi Zhou</span></strong>: <span class="role">Conceptualization, Data curation, Writing – original draft</span>
</div>
<div>
<strong class="contrib"><span class="name western">Hengcheng Lu</span></strong>: <span class="role">Conceptualization, Methodology, Writing – original draft</span>
</div>
<div>
<strong class="contrib"><span class="name western">Bin Fu</span></strong>: <span class="role">Funding acquisition, Supervision, Validation, Writing – review &amp; editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">Jianhan Fu</span></strong>: <span class="role">Investigation, Methodology, Supervision, Writing – review &amp; editing</span>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2024 Oct 28; Accepted 2025 May 27; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© 2025 The Author(s). Published by Informa UK Limited, trading as Taylor &amp; Francis Group</div>
<p>This is an Open Access article distributed under the terms of the Creative Commons Attribution License (<a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited. The terms on which this article has been published allow the posting of the Accepted Manuscript in a repository by the author(s) or with their consent.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12366518  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40830065/" class="usa-link">40830065</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>Abstract</h2>
<section id="sec1"><h3 class="pmc_sec_title">Background</h3>
<p>Metastasis represents the primary cause of cancer-related mortality, with a high incidence observed in renal cell carcinoma (RCC). Anoikis, a specialized form of apoptosis, plays a crucial role in preventing displaced cells from adhering to new extracellular matrices (ECM), thus inhibiting their aberrant growth. Notably, cancer cells, especially metastatic ones, exhibit resistance to anoikis. However, the exact mechanisms of anoikis resistance in RCC are not well understood.</p></section><section id="sec2"><h3 class="pmc_sec_title">Methods</h3>
<p>This study integrates bioinformatics, single-cell RNA sequencing and experimental validation to investigate the role of anoikis-related genes (ARG) in RCC, with a focus on MMP9. RNA-seq data from 518 RCC patients and 71 healthy controls (TCGA-KIRC) and external validation cohorts (E-MTAB-1980, <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE22541" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE22541</a>) were analyzed to construct an ARG-based prognostic model. Single-cell RNA sequencing (scRNA-seq, <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE159115" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE159115</a>) was used to assess tumour heterogeneity, while <em>in vitro</em> experiments in RCC cell lines validated MMP9’s impact on anoikis resistance, migration and invasion.</p></section><section id="sec3"><h3 class="pmc_sec_title">Results</h3>
<p>We collected all RNA-seq and single-cell RNA-seq (scRNA-seq) data from multiple online databases and utilized these datasets to develop a novel ARG-based prognostic model called ARGs. Using Cox regression and machine learning, our model achieved a 5-year area under curve (AUC) of 0.79, surpassing existing models in predictive performance. Enrichment analysis revealed distinct immune and metabolic landscapes between ARGs high- and low-risk groups. At the single-cell level, tumour cells were categorized based on ARG expression, revealing heterogeneous anoikis resistance mechanisms. MMP9 was identified as a key prognostic gene (HR = 1.5, <em>p</em> = 0.016) associated with anoikis resistance and RCC metastasis. Functional assays confirmed that MMP9 knockdown increased anoikis by 59% and significantly reduced wound-healing migration by about 30% and transwell invasion by 50%, reinforcing its role in RCC progression.</p></section><section id="sec4"><h3 class="pmc_sec_title">Conclusions</h3>
<p>Targeting anoikis-related genes, particularly MMP9, enhances anoikis sensitivity and reduces RCC invasiveness, offering a potential therapeutic strategy to mitigate metastasis and improve clinical outcomes.</p></section><section id="kwd-group1" class="kwd-group"><p><strong>Keywords:</strong> Anoikis, renal cell carcinoma, metastasis, MMP9</p></section></section><section id="S0001"><h2 class="pmc_sec_title">1. Backgrounds</h2>
<p>Renal cell carcinoma represents one of the most common malignant tumours in the urinary system and leading to around 180,000 deaths per year globally [<a href="#CIT0001" class="usa-link" aria-describedby="CIT0001">1</a>]. Among its various subtypes, clear cell renal cell carcinoma (ccRCC) is the most aggressive, accounting for 70%-80% of all RCC cases [<a href="#CIT0002" class="usa-link" aria-describedby="CIT0002">2</a>]. While early-stage RCC can be effectively treated through surgical resection or ablation techniques, metastatic RCC remains largely incurable, with a 5-year survival rate of only 12%, compared to 75% for RCC overall [<a href="#CIT0003" class="usa-link" aria-describedby="CIT0003">3–5</a>]. These statistics underscore the critical need to explore the mechanisms underlying RCC invasion and metastasis, which are crucial for improving patient outcomes and guiding treatment strategies. For localized RCC, treatment options include partial or radical nephrectomy, as well as minimally invasive techniques such as cryoablation and radiofrequency ablation [<a href="#CIT0006" class="usa-link" aria-describedby="CIT0006">6</a>]. However, once RCC becomes metastatic, treatment options become limited, primarily relying on systemic therapies such as immune checkpoint inhibitors and tyrosine kinase inhibitors (TKIs) [<a href="#CIT0007" class="usa-link" aria-describedby="CIT0007">7</a>]. Although these treatments have improved patient survival, many patients ultimately develop resistance, and the prognosis remains poor, which highlights the necessity for novel therapeutic strategies.</p>
<p>One critical feature of cancer metastasis is the ability of tumour cells to survive and proliferate outside their native microenvironment. Normally, cellular adhesion and interactions with the extracellular matrix (ECM) are essential for cell survival [<a href="#CIT0008" class="usa-link" aria-describedby="CIT0008">8</a>]. Anoikis, a specialized form of programmed cell death, occurs when cells lose their connection to the ECM [<a href="#CIT0008" class="usa-link" aria-describedby="CIT0008">8</a>,<a href="#CIT0009" class="usa-link" aria-describedby="CIT0009">9</a>]. Typically, anoikis efficiently eliminates ectopic cells, preventing their growth on inappropriate tissues and organs [<a href="#CIT0010" class="usa-link" aria-describedby="CIT0010">10</a>]. During the process of anoikis, multiple signalling pathways are triggered, such as those mediated by integrins, growth factor receptors and apoptosis-related mechanisms. These pathways collectively drive apoptosis, which is characterized by DNA fragmentation, mitochondrial dysfunction, cytoskeletal reorganization and the activation of caspases [<a href="#CIT0011" class="usa-link" aria-describedby="CIT0011">11</a>,<a href="#CIT0012" class="usa-link" aria-describedby="CIT0012">12</a>]. Nevertheless, cancer cells can evade anoikis, allowing them to survive, disseminate from the primary site to distant organs and establish metastatic foci [<a href="#CIT0009" class="usa-link" aria-describedby="CIT0009">9</a>,<a href="#CIT0013" class="usa-link" aria-describedby="CIT0013">13</a>]. Metastasis is the primary cause of death in RCC patients, with anoikis resistance being a crucial characteristic of metastatic cancer [<a href="#CIT0014" class="usa-link" aria-describedby="CIT0014">14</a>,<a href="#CIT0015" class="usa-link" aria-describedby="CIT0015">15</a>]. Several key pathways might contribute to anoikis resistance, including the PI3K/Akt pathway, which suppresses pro-apoptotic proteins such as BIM and BAX to prevent caspase activation [<a href="#CIT0016" class="usa-link" aria-describedby="CIT0016">16</a>]. Similarly, the MAPK/ERK pathway promotes cytoskeletal reorganization and enhances integrin-mediated survival signalling, allowing detached cells to evade apoptosis [<a href="#CIT0017" class="usa-link" aria-describedby="CIT0017">17</a>]. Additionally, focal adhesion kinase (FAK) signalling plays a central role by maintaining integrin engagement and activating downstream pro-survival pathways [<a href="#CIT0018" class="usa-link" aria-describedby="CIT0018">18</a>]. However, the mechanisms underlying anoikis resistance in RCC remain poorly understood. With the rapid advancement of bioinformatics technologies, it is now possible to systematically analyze sequencing data to identify prognostic anoikis-related genes, evaluate their potential to predict immunotherapy and TKI responses, and explore their role in RCC heterogeneity at the single-cell level.</p>
<p>Given the significance of anoikis resistance in metastatic RCC progression, targeting and modulating this process may offer a novel therapeutic strategy. Previous studies have indicated that the matrix metalloproteinase (MMP) family, which is overexpressed in nearly all cancer types [<a href="#CIT0019" class="usa-link" aria-describedby="CIT0019">19</a>,<a href="#CIT0020" class="usa-link" aria-describedby="CIT0020">20</a>], including RCC, represents a promising therapeutic target [<a href="#CIT0021" class="usa-link" aria-describedby="CIT0021">21</a>]. MMP9, ranking among the top five anoikis-related genes in differentially expressed genes in metastatic ccRCC has been implicated in cancer invasion, metastasis and upregulation across various tumour types [<a href="#CIT0022" class="usa-link" aria-describedby="CIT0022">22–24</a>]. Several cancers, such as hepatocellular carcinoma, prostate cancer, and head and neck squamous cell carcinomas, have exhibited altered apoptosis resistance and invasion through the regulation of MMP9-related signalling pathways [<a href="#CIT0025" class="usa-link" aria-describedby="CIT0025">25–27</a>].</p>
<p>While MMP9 contributes to maintaining the malignant phenotype of RCC progression, its role in anoikis resistance remains largely unexplored. By degrading ECM components, releasing pro-survival growth factors and activating key pathways such as PI3K/Akt and FAK, MMP9 may create a microenvironment that facilitates metastatic progression [<a href="#CIT0028" class="usa-link" aria-describedby="CIT0028">28</a>]. However, the precise mechanisms underlying its involvement in anoikis sensitivity in RCC still needs further investigation. This study proposes a direction for anoikis-targeted therapy based on anoikis-related genes and experimentally validates MMP9’s involvement in anoikis and its effects on renal cancer cell lines 786 O and Achn. Consequently, our findings offer novel insights into RCC-targeted therapy, suggesting potential innovative strategies for clinical intervention.</p></section><section id="S0002"><h2 class="pmc_sec_title">2. Results</h2>
<section id="S0002-S2001"><h3 class="pmc_sec_title">2.1. Construction of ARGs and the nomogram</h3>
<p>We aimed to identify ARG associated with ccRCC. In TCGA-KIRC cohort, we took the intersection of DEGs between tumour and normal samples (A total of 9870 upregulated genes and 2906 downregulated genes were identified, <a href="#F0001" class="usa-link">Figure 1a</a>), anoikis-related genes (410 genes from GeneCards database) and prognostic genes (14,415 genes with prognosis), resulting in 71 genes for further screening (<a href="#F0001" class="usa-link">Figure 1b</a>). The LASSO, SVM-RFE and RF algorithm were conducted with these genes separately. In the LASSO cox regression, 25 genes were selected based on the optimal value of λ (<a href="https://doi.org/10.1080/07853890.2025.2548042" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Supplementary Figure 1A, 1B</a>). The SVM-RFE algorithm predicted the most accurate results with 41 genes (<a href="https://doi.org/10.1080/07853890.2025.2548042" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Supplementary Figure 1C</a>). For the RF algorithm, we shortlisted the top 50 genes with significant importance (<a href="https://doi.org/10.1080/07853890.2025.2548042" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Supplementary Figure 1D</a>). By taking the intersection of the results of the three machine-learning algorithms, we identified 18 robust signature genes (IRF6, CEACAM4, EDAR, SHC1, HMOX1, PTK6, ADCY10, BAG1, MNX1, KL, PLG, LTB4R2, EDA2R, BID, PLK1, HMCN1, PIK3CG and PLAUR, <a href="#F0001" class="usa-link">Figure 1c</a>). We constructed coxph models for each patient (518 from TCGA-KIRC; 101 from E-MTAB-1980; 74 from <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE22541" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE22541</a>) using these 18 signature genes to calculate ARG signature (ARGs). Univariate cox regression was applied to analyze these signature genes, and all but two genes (HMOX1 and PIK3CG) were found to be potential prognostic biomarkers (<a href="https://doi.org/10.1080/07853890.2025.2548042" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Supplementary Figure 1E</a>). We further performed univariate (HR = 1.23, 95% CI = 1.19–1.28, <em>p</em> &lt; 0.001) and multivariate (HR = 1.16, 95% CI = 1.11–1.22, <em>p</em> &lt; 0.001) cox regression on ARGs with clinical features. The results indicated that ARGs is an independent prognostic factor significantly associated with worse overall survival (OS) time in ccRCC (<a href="#F0001" class="usa-link">Figure 1d</a>, <a href="#F0001" class="usa-link">Figure 1e</a>). A nomogram was constructed with ARGs, grade, T stage and age to enhance the clinical application of ARGs (<a href="#F0001" class="usa-link">Figure 1f</a>). The calibration curves of 1-, 3- and 5-year demonstrated that the nomogram had a relatively good predictive performance (<a href="#F0001" class="usa-link">Figure 1g</a>).</p>
<figure class="fig xbox font-sm" id="F0001"><h4 class="obj_head">Figure 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12366518_IANN_A_2548042_F0001_C.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dd12/12366518/17a59be2064f/IANN_A_2548042_F0001_C.jpg" loading="lazy" height="1089" width="750" alt="Figure 1."></a></p>
<div class="p text-right font-secondary"><a href="figure/F0001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Construction of ARGs and the nomogram. (a) Volcano plot of the DEGs between tumour and normal samples (TCGA-KIRC); (b) Venn plot shows the intersection genes for further screen; (c) Venn plot shows the machine-learning result; (d) Univariate cox regression on ARGs; (e) Multivariate cox regression on ARGs; (f) Nomogram constructed with ARGs and other clinical features; (g) The calibration curves of nomogram.</p></figcaption></figure></section><section id="S0002-S2002"><h3 class="pmc_sec_title">2.2. The prognostic value of ARGs</h3>
<p>To evaluate and validate the prognostic value of ARGs, we utilized three datasets: TCGA-KIRC as the training cohort, E-MATB-1980 and <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE22541" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE22541</a> as the validation cohorts. For each patient, the formula was as follows: <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="ilm0001"><math id="MATH0001" display="inline"><mtext mathvariant="italic">risk score</mtext><mo>=</mo><mrow><mrow><msubsup><mo stretchy="false">∑</mo><mrow><mi>i</mi><mo>=</mo><mn>1</mn></mrow><mrow><mi>n</mi></mrow></msubsup></mrow><mrow><mo stretchy="false">(</mo><mrow><msub><mrow><mtext mathvariant="italic">Coef</mtext></mrow><mrow><mi>i</mi></mrow></msub></mrow><mi>∗</mi><mrow><msub><mrow><mi>x</mi></mrow><mrow><mi>i</mi></mrow></msub></mrow><mo stretchy="false">)</mo></mrow></mrow><mo>,</mo><mi>i</mi></math></span> represents each selected signature gene and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="ilm0002"><math id="MATH0002" display="inline"><mrow><msub><mrow><mi>x</mi></mrow><mrow><mi>i</mi></mrow></msub></mrow></math></span> is the relative expression. Patients were divided into ARGs High- and Low-risk group according to the median value of <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="ilm0003"><math id="MATH0003" display="inline"><mrow><mtext mathvariant="italic">risk score</mtext></mrow></math></span>.We initially analyzed the distribution of several clinical features between the two groups of patients in the TCGA-KIRC cohort and found that patients in the ARGs-High group had more advanced grades, T-stages and M-stages (<a href="https://doi.org/10.1080/07853890.2025.2548042" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Supplementary Figure 2</a>). <a href="#F0002" class="usa-link">Figure 2a</a> illustrated that the ARGs-High group had worse OS in the TCGA-KIRC and E-MATB-1980 cohorts, and worse DFS in the <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE22541" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE22541</a> cohort. In these datasets, patients with higher ARGs had a higher case fatality rate compared to patients in the ARGs-Low group. Moreover, the heatmaps depicted the relative expression of signature genes (<a href="#F0002" class="usa-link">Figure 2b</a>). Time-ROC curves demonstrated that ARGs had excellent predictive performance: AUC = 0.80 for 1-year, 0.75 for 3-year, 0.79 for 5-year survival rate in TCGA-KIRC, AUC = 0.73 for 1-year, 0.79 for 3-year, 0.81 for 5-year survival rate in E-MATB-1980, AUC = 0.79 for 1-year, 0.84 for 3-year, 0.75 for 5-year survival rate in <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE22541" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE22541</a> (<a href="#F0002" class="usa-link">Figure 2c</a>). Furthermore, we compared the AUC of ARGs in TCGA-KIRC with other recently published cell death-related signatures, and ARGs had the highest AUC of 1- and 5-year survival (<a href="#F0002" class="usa-link">Figure 2d</a>).</p>
<figure class="fig xbox font-sm" id="F0002"><h4 class="obj_head">Figure 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12366518_IANN_A_2548042_F0002_C.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dd12/12366518/65e65c7d461c/IANN_A_2548042_F0002_C.jpg" loading="lazy" height="1203" width="750" alt="Figure 2."></a></p>
<div class="p text-right font-secondary"><a href="figure/F0002/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>The prognostic value of ARGs. (a) K-M plots of two groups in three datasets; (b) The distribution of patients and heatmap of DEGs between two groups; (c) Time-ROC curve of ARGs; (d) The AUC of ARGs compared to other published signatures; (e) ROC curves show the ARGs’ efficacy in discriminating primary and metastases ccRCC.</p></figcaption></figure><p>In three datasets that included patients with metastatic ccRCC (<a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE22541" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE22541</a>, <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE85258" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE85258</a> and <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE113204" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE113204</a>), we demonstrated that these 18 ARGs genes had excellent efficacy in discriminating primary ccRCC from metastases with AUC = 0.991, 0.952 and 1.000, respectively (<a href="#F0002" class="usa-link">Figure 2e</a>).</p></section><section id="S0002-S2003"><h3 class="pmc_sec_title">2.3. ARGs predict survival and response to immunotherapy and TKIs</h3>
<p>To evaluate the effectiveness of ARGs as a tool to predict immunotherapy and TKIs sensitivity in ccRCC patients, we analyzed five datasets that included ccRCC patients treated with immunotherapy or TKIs: CheckMate-009 (nivolumab), CheckMate-010 (nivolumab), CheckMate-025 (nivolumab and everolimus), E-MTAB-3218 (nivolumab) and E-MTAB-3267 (sunitinib). Patients were stratified into ARGs-High and ARGs-Low groups as previously described. Strikingly, we found that ARGs still accurately predicted OS or PFS time for patients who received immunotherapy or TKIs (<a href="#F0003" class="usa-link">Figure 3a</a>). In four of the five datasets, except CheckMate-025, the proportion of patients who responded to immunotherapy/TKIs (CR/PR) was lower in the ARGs-High group than in the ARGs-Low group (<a href="#F0003" class="usa-link">Figure 3b</a>). In addition, we identified significant differential expression of many immunotherapeutic targets and targeted-therapeutic targets between the two groups (<a href="#F0003" class="usa-link">Figure 3c, 3d</a>). This result partly explained the distinct response to immunotherapy/TKIs between the ARGs-High and ARGs-Low groups of ccRCC patients.</p>
<figure class="fig xbox font-sm" id="F0003"><h4 class="obj_head">Figure 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12366518_IANN_A_2548042_F0003_C.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dd12/12366518/1f75111a5473/IANN_A_2548042_F0003_C.jpg" loading="lazy" height="478" width="750" alt="Figure 3."></a></p>
<div class="p text-right font-secondary"><a href="figure/F0003/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>ARGs predict survival and response to immunotherapy and TKIs. (a) K-M plots of two groups in immunotherapy and TKIs treated cohorts; (b) The CR/PR ratio between two groups; (c) Boxplot of immunotherapeutic targets mRNA expression between two groups; (d) Boxplot of targeted-therapeutic targets mRNA expression between two groups.</p></figcaption></figure></section><section id="S0002-S2004"><h3 class="pmc_sec_title">2.4. The potential mechanisms and pathways of ARGs</h3>
<p>Considering the exceptional prognostic value of ARGs, the underlying mechanism needs to be investigated. We first analyzed the DEGs (<a href="https://doi.org/10.1080/07853890.2025.2548042" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Supplementary Figure 3</a>) between the ARGs-High group and the ARGs-Low group by GO/KEGG databases (<a href="#F0004" class="usa-link">Figure 4a, 4b</a>). Several biological functions related to tumour growth and anoikis were enriched, including plasma membrane, extracellular matrix and various signalling pathways. In addition, we observed enrichment of several immune-related and metabolic pathways, such as humoral immune response, cytokine-cytokine receptor interaction and retinol metabolism. To compare the activity of these pathways between the two groups, we conducted GSVA analyses in hallmark and metabolic pathways, which revealed differential activation of numerous pathways in these two groups (<a href="#F0004" class="usa-link">Figure 4c, 4D</a>). Furthermore, GSEA confirmed that the ARGs-High group exhibited greater activation of valine, leucine and isoleucine degradation, fatty acid degradation, citrate cycle, arginine biosynthesis, glycolysis and oxidative phosphorylation, compared to the ARGs-Low group (<a href="#F0004" class="usa-link">Figure 4e</a>). These results suggest that patients with different ARGs may exhibit varying tumour immune microenvironment (TIME) profiles, activation of distinct tumour hallmark pathways and metabolic reprogramming, which ultimately impact prognosis.</p>
<figure class="fig xbox font-sm" id="F0004"><h4 class="obj_head">Figure 4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12366518_IANN_A_2548042_F0004_C.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dd12/12366518/82233faadbd1/IANN_A_2548042_F0004_C.jpg" loading="lazy" height="576" width="750" alt="Figure 4."></a></p>
<div class="p text-right font-secondary"><a href="figure/F0004/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>The potential mechanisms and pathways of ARGs. (a) Top GO enrichment; (b) Top KEGG pathway enrichment; (c) GSVA analysis of hallmark pathways; (d) GSVA analysis of metabolic pathways; (e) GSEA analysis of several pathways.</p></figcaption></figure></section><section id="S0002-S2005"><h3 class="pmc_sec_title">2.5. ARGs associates with tumour immune microenvironment</h3>
<p>Further elucidation is necessary to understand the distinct TIME between the two groups and the relationship between ARGs and immune status in ccRCC. Utilizing the CIBERSORT algorithm, we observed a significant reduction in CD4 T memory resting cells, macrophages M1, monocytes, NK resting cells and mast resting cells in the ARGs-High group (<a href="#F0005" class="usa-link">Figure 5a</a>). Furthermore, several immune cells exhibited a positive correlation with ARGs, including macrophages M0, CD4 T memory activated cells, T follicular helper cells and Tregs (<a href="#F0005" class="usa-link">Figure 5b</a>). NK cells, macrophages and monocytes play crucial roles in anti-tumour immunity [<a href="#CIT0029" class="usa-link" aria-describedby="CIT0029">29</a>], while Tregs represent an immunosuppressed state [<a href="#CIT0030" class="usa-link" aria-describedby="CIT0030">30</a>], which may explain the low response rate to immunotherapy in the ARGs-High group. However, the immune score calculated using the ESTIMATE algorithm indicated a higher total immune content in the ARGs-High group (<a href="#F0005" class="usa-link">Figure 5c</a>), and there was no significant correlation between tumour purity and ARGs (<a href="#F0005" class="usa-link">Figure 5d</a>). Therefore, we concluded that ARGs are associated with TIME, but do not significantly affect the proportion of tumour cells.</p>
<figure class="fig xbox font-sm" id="F0005"><h4 class="obj_head">Figure 5.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12366518_IANN_A_2548042_F0005_C.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dd12/12366518/68e0a334423d/IANN_A_2548042_F0005_C.jpg" loading="lazy" height="437" width="750" alt="Figure 5."></a></p>
<div class="p text-right font-secondary"><a href="figure/F0005/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>ARGs associates with tumour immune microenvironment. (a) The proportion of immune cells between two groups calculated by CIBERSORT; (b) The correlation between immune cells and ARGs; (c) The immune score calculated by ESTIMATE; (d) The correlation between tumour purity and ARGs.</p></figcaption></figure></section><section id="S0002-S2006"><h3 class="pmc_sec_title">2.6. Somatic mutations atlas of the ARGs high- and low-risk groups</h3>
<p>Patients in different ARGs subgroups may exhibit different mutation atlases. We summarized the mutation profiles of the two groups in <a href="https://doi.org/10.1080/07853890.2025.2548042" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Supplementary Figure 4</a>, and the top 30 mutated genes in both groups were exhibited in <a href="#F0006" class="usa-link">Figure 6a</a>, with little difference observed. The top 3 most frequently mutated genes in both groups were VHL, PBRM1 and TTN, and the lollipop plots indicated different mutation sites of VHL for the two groups (<a href="#F0006" class="usa-link">Figure 6b</a>). Intriguingly, a higher likelihood of VHL and PBRM1 co-mutations in the ARGs-High group was observed, while this was not the case in the ARGs-Low group (<a href="#F0006" class="usa-link">Figure 6c</a>). Further studies are required to investigate this observation. Notably, the tumour mutational burden (TMB) was statistically higher in the ARGs-High group than in the ARGs-Low group (<a href="#F0006" class="usa-link">Figure 6d</a>).</p>
<figure class="fig xbox font-sm" id="F0006"><h4 class="obj_head">Figure 6.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12366518_IANN_A_2548042_F0006_C.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dd12/12366518/b27be02113ff/IANN_A_2548042_F0006_C.jpg" loading="lazy" height="1108" width="750" alt="Figure 6."></a></p>
<div class="p text-right font-secondary"><a href="figure/F0006/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Somatic mutations atlas of the ARGs high- and low-risk groups and candidate drugs. (a) The top mutated genes in two groups; (b) Lolipop plots show the mutation sites of genes; (c) The co-mutations observed by heatmap; (d) Boxplots shows TMB; (e) Estimation of drug sensitivity according to CTRP database; (f) Top 5 candidate compounds of ARGs-high group.</p></figcaption></figure></section><section id="S0002-S2007"><h3 class="pmc_sec_title">2.7. Candidate drug and sensitivity prediction</h3>
<p>Axitinib and Sorafenib are both clinically used in the treatment of ccRCC patients. Predictions made using the CTRP database revealed that patients in the ARGs-High group were more sensitive to these two drugs, potentially guiding clinical use (<a href="#F0006" class="usa-link">Figure 6e</a>). Additionally, we predicted potential drugs targeting ARGs <em>via</em> the CMap_XSum algorithm, and the top 5 candidate compounds were imatinib, STOCK1N.35696, AH.6809, MK.886 and NU.1025 (<a href="#F0006" class="usa-link">Figure 6f</a>).</p></section><section id="S0002-S2008"><h3 class="pmc_sec_title">2.8. Cell annotations and scoring of scRNA-seq data</h3>
<p>We integrated the scRNA-seq data of seven ccRCC samples after quality control (<a href="https://doi.org/10.1080/07853890.2025.2548042" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Supplementary Figure 5A, 5B</a>). The elbowplot indicated that the finest component numbers were 19 (<a href="https://doi.org/10.1080/07853890.2025.2548042" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Supplementary Figure 5C</a>). A total of 20,817 cells were clustered into 16 clusters (resolution = 0.5) and the Uniform Manifold Approximation and Projection (UMAP) plot showed that the batch effect had been removed (<a href="https://doi.org/10.1080/07853890.2025.2548042" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Supplementary Figure 5D</a>). The cell type-specific markers we used for cell annotation were as follows: T cell (CD3D); NK cell (KLRD1 and NKG7); plasma cell (IGKC and JCHAIN); Mast cell (KIT and TPSB2); tumour cell (CA9, NDUFA4L2 and SLC17A3); endothelium (PECAM1, PTPRB and KDR); mesangial cell (ACTA2 and PDGFRB) [<a href="#CIT0031" class="usa-link" aria-describedby="CIT0031">31</a>] (<a href="https://doi.org/10.1080/07853890.2025.2548042" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Supplementary Figure 5E, 5F, 5G</a>). Finally, we identified seven distinct cell types: 9402 tumour cells, 4105 dendritic cells, 3655 endothelium mesangial cells, 949 T cells, 332 NK cells and 186 plasma cells (<a href="#F0007" class="usa-link">Figure 7a</a>). We next extracted the tumour cells for further analysis. Similar to the previous procedures, after removed a doublet, a total of 9401 singlet tumour cells were clustered into 10 clusters with a resolution of 0.2 (<a href="#F0007" class="usa-link">Figure 7b</a>). We used the ‘AddModuleScore’ function to calculate ARGscore for each tumour cells and divided them into two groups according to the median value of ARGscore (<a href="#F0007" class="usa-link">Figure 7c</a>).</p>
<figure class="fig xbox font-sm" id="F0007"><h4 class="obj_head">Figure 7.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12366518_IANN_A_2548042_F0007_C.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dd12/12366518/51a8957b6881/IANN_A_2548042_F0007_C.jpg" loading="lazy" height="592" width="750" alt="Figure 7."></a></p>
<div class="p text-right font-secondary"><a href="figure/F0007/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Heterogeneity between the two groups of tumour cells at single-cell level. (a) tSNE plot shows all cells in <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE159115" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE159115</a>; (b) tSNE plot shows tumour cells with different Seurat clusters; (c) ARGscore calculated by AddModuleScore; (d) GO/KEGG enrichment between two subgroups of tumour cells; (e) Cancer hallmark pathways between two subgroups; (f) Metabolic pathways between two subgroups.</p></figcaption></figure></section><section id="S0002-S2009"><h3 class="pmc_sec_title">2.9. Heterogeneity between the two groups of tumour cells</h3>
<p>Tumour cells in the two groups may have different biological behaviours and hallmarks, which are related to their malignancy. We conducted GO/KEGG enrichment analysis on the DEGs between the two subgroups of tumour cells. The result shown that many pathways associated with tumour growth and metastasis were enriched, such as regulation of apoptotic signalling pathway, response to hypoxia and cell adhesion molecules (<a href="#F0007" class="usa-link">Figure 7d</a>). The boxplot of ssGSEA compared the relative activity of some interested cancer hallmark pathways (for example, hypoxia, epithelial-mesenchymal transition (EMT) and Angiogenesis) between the two subgroups (<a href="#F0007" class="usa-link">Figure 7e</a>). The ssGSEA score of apoptosis, hypoxia, EMT, inflammatory response, ROS pathway and angiogenesis were higher in the ARGscore_High group. We also found that many metabolic pathways were upregulated in the ARGscore_High group (<a href="#F0007" class="usa-link">Figure 7f</a>). These results suggested that there is heterogeneity among these two groups of tumour cells.</p></section><section id="S0002-S2010"><h3 class="pmc_sec_title">2.10. Pseudotime analysis indicates that anoikis resistance may occur as the tumour cell growth</h3>
<p>In order to depict the dynamic change of ARGscore during tumour cells development, Monocle 2 was employed to portrait a developmental trajectory. <a href="#F0008" class="usa-link">Figure 8a</a> demonstrated that ARGscore decreased as the tumour cell growth. Considering that tumour cells in the ARGscore_High group have higher apoptotic activity (<a href="#F0007" class="usa-link">Figure 7f</a>), we hypothesis that during tumour development, anoikis resistance occurs in tumour cells and therefore ARGscore decreased, which represents apoptosis pathway’s activity. In addition, the dynamic change in the expression of signature genes during the development of tumour cells was shown in <a href="#F0008" class="usa-link">Figure 8b</a>. PLAUR, BID, HMOX1, SHC1 and ADCY10 expression increases with the development trajectory while EDA2R and BAG1 expression decreases.</p>
<figure class="fig xbox font-sm" id="F0008"><h4 class="obj_head">Figure 8.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12366518_IANN_A_2548042_F0008_C.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dd12/12366518/3c990975325d/IANN_A_2548042_F0008_C.jpg" loading="lazy" height="641" width="750" alt="Figure 8."></a></p>
<div class="p text-right font-secondary"><a href="figure/F0008/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Pseudotime analysis and cell-cell communications. (a) The ARGscores increase with pseudotime; (b) The dynamic change of signature genes during the development of tumour cells; (c) Ligands-receptors-targets regulatory network between ARGs_High tumour cells and T cells; (d) Ligands-receptors-targets regulatory network between ARGs_Low tumour cells and T cells; (e) Scenic predicts the top transcription factors in two subgroups of tumour cells.</p></figcaption></figure></section><section id="S0002-S2011"><h3 class="pmc_sec_title">2.11. Ligands-receptors-targets regulatory network and transcription factor analysis in tumour cells</h3>
<p>Using NicheNet, we constructed ligands-receptors-targets regulatory networks between T cells and two groups of tumour cells separately. Between ARGs_High tumour cells and T cells, ITGB2-ICAM1 and TGFB1-TGFBR1 were ligand-receptor pairs with strong regulatory potential (<a href="#F0008" class="usa-link">Figure 8c</a>), while EDN1, JUNB, SOCS3, VIM and COL1A2 were top genes target to TGFB1 (<a href="https://doi.org/10.1080/07853890.2025.2548042" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Supplementary Figure 6A</a>). Between ARGs_Low tumour cells and T cells, ITGB1-CD46 and ITGB1-ENG were ligand-receptor pairs with strong regulatory potential (<a href="#F0008" class="usa-link">Figure 8d</a>), while COL1A2, CCL3, SERPINE1, FN1 and CDKN1A were top genes target to TGFB1 (<a href="https://doi.org/10.1080/07853890.2025.2548042" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Supplementary Figure 6B</a>). SCENIC predicted the top 4 transcription factors that differed expressed most between the two subgroups of tumour cells: SAP30, UQCRB, ZBTB33 and ZNF165 (<a href="#F0008" class="usa-link">Figure 8e</a>).</p></section><section id="S0002-S2012"><h3 class="pmc_sec_title">2.12. Establishment of anoikis model and determination of target gene</h3>
<p>In this study, we investigated the invasion and metastasis of renal clear cell carcinoma using an anoikis model. Following the protocol outlined by Valentinis et al. [<a href="#CIT0032" class="usa-link" aria-describedby="CIT0032">32</a>], we employed poly-HEMA-coated Petri dishes to maintain cells in suspension, thereby inducing anoikis, an anchorage-dependent form of programmed cell death. Cells in the suspension group formed clusters and were unable to adhere to the dish, as opposed to the control group (<a href="#F0009" class="usa-link">Figure 9b</a>). Interestingly, after 72 h in suspension, cells began to reattach, albeit at a slow rate, when transferred to normal Petri dishes for 24 h (<a href="#F0009" class="usa-link">Figure 9b</a>), suggesting partial reactivation. To determine the optimal time for establishing the anoikis model, 786 O cells were treated with poly-HEMA for 0, 12, 24, 48 and 72 h and analyzed by flow cytometry. Our findings demonstrated a gradual increase in anoikis rates as treatment duration increased, with significant damage observed at 72 h (<a href="#F0009" class="usa-link">Figure 9g, 9h</a>). Based on the moderate damage observed at 48 h, we selected this time point for all subsequent experiments.</p>
<figure class="fig xbox font-sm" id="F0009"><h4 class="obj_head">Figure 9.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12366518_IANN_A_2548042_F0009_C.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dd12/12366518/374e610692df/IANN_A_2548042_F0009_C.jpg" loading="lazy" height="1074" width="750" alt="Figure 9."></a></p>
<div class="p text-right font-secondary"><a href="figure/F0009/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>MMP9 was involved in the anoikis resistance of RCC and knockdown of MMP9 significantly attenuated migration by accelerating anoikis in 786 O cells. (a) Morphological representation of anoikis models established by poly-HEMA, scale bars: 50 μm; (b &amp; c) Flowcytometry analysis showed that the percentage of apoptosis increased with the duration of poly-HEMA treatment; (d) qRT-PCR analysis showed significant overexpression of MMP9 in 786 O cell line compared to HK-2 cell line; (e &amp; f) Western blot studies showed that MMP9 was downregulated in poly-HEMA-treated 786 O cells; (g &amp; h) Western blot studies confirmed lentivirus-mediated knockdown of MMP9; (i &amp; j) Wound-healing assay indicated knockdown of MMP9 attenuated migration capacity of 786 O cells, scale bars: 100 μm; (k &amp; l). Transwell assay indicated knockdown of MMP9 alleviated invasion capacity of 786 O cells; (m – p) Flow cytometry analysis and western blot studies demonstrated that knockdown of MMP9 significantly aggravated poly-HEMA induced anoikis. *<em>p</em> &lt; 0.05, **<em>p</em> &lt; 0.01, ***<em>p</em> &lt; 0.001, ****<em>p</em> &lt; 0.0001.</p></figcaption></figure><p>Although the ARGs prognostic model we previously constructed has shown good prognostic predictive performance, this does not mean that all the genes within the model possess strong functions, such as the majority of pseudogenes [<a href="#CIT0033" class="usa-link" aria-describedby="CIT0033">33</a>]. In order to explore potential clinical translational targets of anoikis in ccRCC, it is necessary to re-screen the genes that might have robust functions from scratch. Anoikis is related to tumour metastasis. In order to further screen the key anoikis genes in the process of ccRCC metastasis, we intersected anoikis-related genes, DEGs from TCGA-KIRC, and DEGs from <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE22541" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE22541</a> (ccRCC with metastasis) and finally obtained five candidate genes: MMP9, BIRC3, MUC1, HMCN1 and ANGPTL4 (<a href="https://doi.org/10.1080/07853890.2025.2548042" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Supplementary Figure 7A</a>). Of these, only MMP9 was associated with worse OS and RFS (<a href="https://doi.org/10.1080/07853890.2025.2548042" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Supplementary Figure 7B</a>, C). MMP9 mRNA expression was elevated in ccRCC patient samples and was higher in metastatic ccRCC than non-metastatic (<a href="https://doi.org/10.1080/07853890.2025.2548042" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Supplementary Figure 7D</a>, E). Futhermore, MMP9 mRNA expression was significantly elevated in ARGs_high group (<a href="https://doi.org/10.1080/07853890.2025.2548042" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Supplementary Figure 7F</a>). RT-qPCR analysis validated that MMP9 expression was significantly upregulated in 786 O cells (human renal clear cell carcinoma) compared to HK-2 cells (human renal cortical proximal convoluted tubule cell) (<a href="#F0009" class="usa-link">Figure 9a</a>). Additionally, western blot analysis confirmed that MMP9 was markedly downregulated in 786 O cells treated with poly-HEMA for 48 h (<a href="#F0009" class="usa-link">Figure 9c, 9e</a>), suggesting an important role for MMP9 in anoikis of renal clear cell carcinoma.</p></section><section id="S0002-S2013"><h3 class="pmc_sec_title">2.13. MMP9 knockdown significantly inhibits 786 O cell migration by exacerbating anoikis</h3>
<p>To further investigate the relationship between MMP9 and anoikis, we employed a lentivirus to suppress MMP9 expression in 786 O cells. Western blot analysis validated the reduction in MMP9 expression (<a href="#F0009" class="usa-link">Figure 9g, 9h</a>). The wound-healing assay demonstrated that MMP9 knockdown significantly reduced the migration rate of 786 O cells for about 30% compared to the shNC group (<a href="#F0009" class="usa-link">Figure 9i, 9j</a>). These findings were further corroborated by transwell assay results, which confirmed that MMP9 knockdown revealed a marked decrease in the invasive capacity of 786 O cells for about 50% (<a href="#F0009" class="usa-link">Figure 9k, 9l</a>). Together, these experiments indicate that MMP9 is involved in the invasion and metastasis of ccRCC and that downregulation of MMP9 expression attenuates this process. Moreover, cleaved caspase 3, an apoptosis-related marker, was upregulated in MMP9-knockdown 786 O cells treated with poly-HEMA, as shown by western blot analysis (<a href="#F0009" class="usa-link">Figure 9m, 9n</a>). Similarly, flow cytometry data confirmed that MMP9 downregulation substantially increased the apoptosis rate of 786 O cells for about 59% following poly-HEMA treatment (<a href="#F0009" class="usa-link">Figure 9o, 9p</a>).</p></section><section id="S0002-S2014"><h3 class="pmc_sec_title">2.14. Similar results were observed in another renal carcinoma cell line Achn</h3>
<p>While the efficacy of regulating anoikis to reduce invasion and metastasis of ccRCC was demonstrated in 786 O cell lines, solid tumours and cell lines are known for their heterogeneity. To more comprehensively assess the potential value of regulating anoikis in RCC, we employed another renal carcinoma cell line, Achn, to validate our findings, and observed similar results. Achn cells formed clusters in poly-HEMA-coated Petri dishes but reattached and partially regained their original morphology when transferred to normal Petri dishes (<a href="#F0010" class="usa-link">Figure 10a</a>), consistent with our observations in 786 O cells. Moreover, MMP9 expression was also downregulated in poly-HEMA-treated Achn cells (<a href="#F0010" class="usa-link">Figure 10b, 10c</a>). To further investigate the underlying mechanism, we established an MMP9-knockdown Achn cell line using lentivirus, as previously performed, and confirmed the knockdown by western blot analysis (<a href="#F0010" class="usa-link">Figure 10d, 10e</a>). We then conducted a wound-healing assay in both the shMMP9 and shNC groups. Despite adjusting the observation time points to 6, 18 and 24 h due to the differing growth rates of Achn and 786 O cells, the results remained consistent (<a href="#F0010" class="usa-link">Figure 10f, 10g</a>). The shMMP9 group demonstrated significantly slower wound healing compared to the shNC group. Furthermore, transwell assays performed in the Achn cell line mirrored the findings observed in 786 O cells, with MMP9 knockdown markedly reducing cellular invasion (<a href="#F0010" class="usa-link">Figure 10h, 10i</a>). These results collectively suggest that MMP9 downregulation effectively mitigates invasion and metastasis across multiple RCC cell lines. Western blot analysis of cleaved caspase 3 also indicated that decreased MMP9 notably augmented apoptosis following poly-HEMA treatment (<a href="#F0010" class="usa-link">Figure 10j, 10k</a>). Altogether, our data reveal that downregulating MMP9 expression in poly-HEMA-treated 786 O and Achn cells substantially accelerates the anoikis process, consequently inhibiting their migration and metastasis potential.</p>
<figure class="fig xbox font-sm" id="F0010"><h4 class="obj_head">Figure 10.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12366518_IANN_A_2548042_F0010_C.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dd12/12366518/5f6d139be33d/IANN_A_2548042_F0010_C.jpg" loading="lazy" height="651" width="750" alt="Figure 10."></a></p>
<div class="p text-right font-secondary"><a href="figure/F0010/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>MMP9 knockdown attenuated migration by exacerbating anoikis in Achn cells as well. (a) Morphological representation of Achn anoikis models established by poly-HEMA, scale bars: 50 μm; (b &amp; c) Western blot studies showed that MMP9 was downregulated in poly-HEMA-treated Achn cells; (d &amp; e) Western blot studies confirmed lentivirus-mediated knockdown of MMP9; (f &amp; g) Wound-healing assay indicated knockdown of MMP9 attenuated migration of Achn cells, scale bars: 100 μm; (h &amp; i). Transwell assay alleviated invasion capacity of Achn cells; (j &amp; k). Western blot studies demonstrated that knockdown of MMP9 significantly aggravated poly-HEMA induced anoikis in Achn cells. *<em>p</em> &lt; 0.05, **<em>p</em> &lt; 0.01, ***<em>p</em> &lt; 0.001.</p></figcaption></figure></section></section><section id="S0003"><h2 class="pmc_sec_title">3. Discussions</h2>
<p>In this study, we constructed a novel ARGs prognostic model that effectively stratifies ccRCC patients into two ARG-based risk groups using various machine-learning algorithms. To ensure the robustness of our prognostic model, we applied LASSO, SVM-RFE and Random Forest algorithms, which are widely used for high-dimensional data analysis. These methods complement each other: LASSO effectively selects key variables by imposing regularization, SVM-RFE refines feature selection in complex data structures, and Random Forest provides stable feature ranking. By integrating these approaches, we aimed to enhance the reliability of our selected gene signature.</p>
<p>Our analysis identified 18 dependable ARGs that play key roles in predicting prognosis and therapeutic responses in ccRCC. These genes highlight the intricate balance between cell survival, adhesion and extracellular remodelling, which are fundamental processes in tumour progression. Notably, genes such as SHC1 and PTK6 regulate essential signal transduction pathways, including the MAPK and PI3K cascades, which are critical for both tumour cell proliferation and resistance to programmed cell death—mechanisms that have been strongly implicated in therapeutic resistance in ccRCC [<a href="#CIT0034" class="usa-link" aria-describedby="CIT0034">34</a>,<a href="#CIT0035" class="usa-link" aria-describedby="CIT0035">35</a>]. Meanwhile, regulators of apoptosis like BID and BAG1 may promote anoikis resistance, facilitating metastatic progression, a phenomenon previously described in RCC and other aggressive cancers [<a href="#CIT0035" class="usa-link" aria-describedby="CIT0035">35</a>,<a href="#CIT0036" class="usa-link" aria-describedby="CIT0036">36</a>]. Additionally, genes involved in extracellular matrix remodelling and adhesion, such as PLAUR, PLG and CEACAM4, may enhance tumour invasion by modifying the tumour microenvironment, a process extensively reported in invasive malignancies [<a href="#CIT0037" class="usa-link" aria-describedby="CIT0037">37–39</a>]. Interestingly, IRF6, EDAR and EDA2R, which regulate epithelial differentiation, may further contribute to ccRCC progression by influencing cellular plasticity [<a href="#CIT0040" class="usa-link" aria-describedby="CIT0040">40</a>,<a href="#CIT0041" class="usa-link" aria-describedby="CIT0041">41</a>]. Other genes such as KL, MNX1, HMCN1 and ADCY10 suggest that anoikis resistance is not restricted to survival pathways but also extends to metabolic and proliferative mechanisms that drive both tumour aggressiveness and treatment resistance [<a href="#CIT0042" class="usa-link" aria-describedby="CIT0042">42–45</a>]. At the single-cell level, our analyses revealed distinct ccRCC subpopulations with varying levels of ARG expression, underscoring tumour heterogeneity. Developmental trajectory analysis further suggests that anoikis resistance may emerge as tumours evolve, potentially driving increased metastatic capacity. In addition, we identified a ligand-receptor-target regulatory network and key transcription factors involved in anoikis resistance, providing mechanistic insights that may guide future therapeutic interventions.</p>
<p>Building on these molecular insights, we evaluated the prognostic significance of the ARGs across multiple independent datasets. Our results indicate that this ARG-based risk model is a stable independent prognostic factor for ccRCC, with significant associations with OS in both univariate (HR = 1.23, 95% CI: 1.19–1.28) and multivariate (HR = 1.16, 95% CI: 1.11–1.22) analyses. These HR values reinforce the clinical relevance of ARG expression, as higher risk scores consistently correlated with poorer patient outcomes. Notably, compared with previously published ARG-based prognostic models for RCC, which reported HR values ranging from 1.017 to 1.215 [<a href="#CIT0046" class="usa-link" aria-describedby="CIT0046">46–49</a>], our model falls within a similar or slightly superior range, further supporting its reliability. In addition, our ARGs achieved a 5-year AUC of 0.79, notably outperforming previously reported models (AUC range: 0.671–0.736) [<a href="#CIT0046" class="usa-link" aria-describedby="CIT0046">46–52</a>]. This enhanced predictive accuracy underscores the clinical utility of our model in risk stratification and treatment decision-making. Moreover, our comprehensive validation across three independent cohorts (TCGA-KIRC, <em>n</em> = 518; E-MTAB-1980, <em>n</em> = 101; <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE22541" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE22541</a>, <em>n</em> = 74) ensures greater generalizability compared to previous models, which often relied on smaller or single-cohort datasets [<a href="#CIT0046" class="usa-link" aria-describedby="CIT0046">46–52</a>]. Despite minor variations in HR values across datasets, the overall trend remained consistent, demonstrating that high-risk ARG expression is robustly associated with poor survival. Such multi-cohort validation reinforces the clinical relevance and translational potential of our findings, supporting the integration of ARGs into prognostic models for personalized RCC management.</p>
<p>The reliable performance of our ARGs across diverse datasets has direct implications for clinical management strategies in RCC. Given that various molecular signatures related to tumour biology, including those involved in tumour immune and tumour microenvironment interactions, have been demonstrated to predict differential treatment responses in RCC [<a href="#CIT0053" class="usa-link" aria-describedby="CIT0053">53–55</a>], our ARGs represents a promising predictive tool for personalized therapy selection. Particularly, this signature may help stratify patients who would benefit most from immune checkpoint inhibitors or TKIs, both standard treatments for advanced RCC with variable response rates [<a href="#CIT0007" class="usa-link" aria-describedby="CIT0007">7</a>]. The stable and reproducible nature of our ARGs allows clinicians to potentially select optimal therapeutic combinations based on individual risk profiles derived from our model. Furthermore, the identification of specific ARGs involved in treatment resistance mechanisms provides promising avenues for developing combination therapies that could integrate conventional treatments with novel agents targeting ARG-driven pathways, potentially overcoming resistance mechanisms and improving patient outcomes.</p>
<p>While our primary prognostic model is based on the 18 ARGs associated with overall survival, our metastatic subgroup analysis identified MMP9 as a critical molecular bridge linking anoikis resistance to the metastatic cascade in ccRCC. MMP9 functions as an important mediator of processes essential for tumour development, including ECM remodelling, EMT, cell migration, angiogenesis and immune response modulation [<a href="#CIT0024" class="usa-link" aria-describedby="CIT0024">24</a>]. Particularly relevant to anoikis resistance, MMP9 regulates EMT through the PI3K/AKT/NF-κB signalling pathway—a critical connection since EMT has been established as not only essential for tumour invasion but also a key feature in anoikis resistance development [<a href="#CIT0056" class="usa-link" aria-describedby="CIT0056">56–58</a>]. Additionally, as a primary enzyme in ECM degradation, MMP9 works in concert with molecules such as SMAD2/3 and TIM3 [<a href="#CIT0059" class="usa-link" aria-describedby="CIT0059">59–61</a>], further reinforcing its role in anoikis regulation. Our experimental findings support these mechanistic connections: MMP9 expression was significantly elevated in the RCC cell line 786 O compared to normal kidney HK-2 cells (<a href="#F0009" class="usa-link">Figure 9a</a>), suggesting its importance in RCC pathogenesis. Moreover, in our poly-HEMA-induced extracellular matrix deprivation model, which simulates conditions promoting anoikis, MMP9 expression was significantly reduced (<a href="#F0009" class="usa-link">Figure 9c and 9e</a>), providing potential evidence of its involvement in anoikis-related pathways in RCC.</p>
<p>Anoikis resistance represents a fundamental mechanism in tumour metastasis, enabling cancer cells to survive despite detachment from the extracellular matrix. Previous research has identified several factors regulating anoikis sensitivity in RCC, complementing our findings on ARGs and MMP9. Shinichi and Meredith demonstrated that quinazoline-based drugs induce anoikis in RCC by disrupting survival signals in focal adhesions [<a href="#CIT0021" class="usa-link" aria-describedby="CIT0021">21</a>,<a href="#CIT0062" class="usa-link" aria-describedby="CIT0062">62</a>], while other studies showed that knockdown of HMGA1 and TIM-3 enhances anoikis occurrence [<a href="#CIT0063" class="usa-link" aria-describedby="CIT0063">63</a>], and silencing the tumour suppressor DLK1 increases anoikis resistance. These findings align with our observations regarding the complex interplay between adhesion-related ARGs and metastatic potential. Furthermore, recent experimental approaches such as Lu Wang’s ECM deprivation system effectively reversed RCC’s anoikis resistance and enhanced sensitivity to doxorubicin chemotherapy [<a href="#CIT0064" class="usa-link" aria-describedby="CIT0064">64</a>], while Peng Zhang’s team found that silencing TrkB improved sorafenib efficacy [<a href="#CIT0065" class="usa-link" aria-describedby="CIT0065">65</a>]. These therapeutic strategies highlight an important implication of our ARG analysis: targeting anoikis resistance pathways may not only directly inhibit metastasis but also synergize with existing treatments, creating new therapeutic opportunities for RCC. However, despite these advances, comprehensive models reflecting the prognostic impact of anoikis-related genes have been limited. Recent anoikis-based RCC risk prediction models by Zhuo Chen and Jin Wang represent progress in this direction, though they still require robust experimental validation [<a href="#CIT0049" class="usa-link" aria-describedby="CIT0049">49</a>,<a href="#CIT0066" class="usa-link" aria-describedby="CIT0066">66</a>]—a gap our current ARGs and MMP9’s experimental findings help address.</p>
<p>In this study, we employed flow cytometry to assess apoptosis rates and conducted wound-healing/transwell assays to evaluate the invasive and migratory capabilities of RCC cells. Our results indicated that following MMP9 gene knockdown, the proportion of anoikis cells increased in both the 786 O and Achn cell lines, while their invasive and metastatic capacities decreased. This suggests that MMP9 functions as an inhibitor of anoikis in RCC (<a href="#F0009" class="usa-link">Figures 9</a> and <a href="#F0010" class="usa-link">10</a>). Notably, this study is the first to demonstrate MMP9’s regulatory role in anoikis within the context of RCC. While previous research has established MMP9’s involvement in various mechanisms of tumour invasion and metastasis, it is essential to recognize that these capabilities do not necessarily encompass or equate to anoikis resistance. Therefore, further experiments are warranted to elucidate the specific molecular pathways through which MMP9 regulates anoikis in RCC.</p>
<p>Despite the insights gained from our research, several limitations should be acknowledged. First, while our study leverages bioinformatics analysis alongside clinical specimens, potential biases in data selection could influence the robustness of our findings. Sample size limitations may restrict the generalizability of our results, as larger cohorts are needed to validate the prognostic significance of MMP9 and other ARGs across diverse patient populations. Additionally, confounding factors, such as variations in tumour microenvironments and patient treatment histories, may affect the interpretation of our results. From a translational perspective, MMP9 inhibitors or siRNA-based therapies offer promising avenues for enhancing anoikis sensitivity in RCC. Novel drug delivery systems, such as nanoparticles, could facilitate targeted delivery of these therapeutics, potentially maximizing efficacy while minimizing systemic side effects.</p></section><section id="S0004"><h2 class="pmc_sec_title">4. Conclusions</h2>
<p>In conclusion, this study presents a novel approach to understanding RCC through the identification and analysis of ARGs, with a particular focus on MMP9. By combining bioinformatics and experimental validation, we provide compelling evidence that MMP9 plays a critical role in regulating Anoikis resistance, a key factor in RCC progression and metastasis. Our findings suggest that targeting MMP9 could offer a promising therapeutic strategy for overcoming Anoikis resistance, potentially improving the efficacy of existing treatments such as immune checkpoint inhibitors and TKIs. Furthermore, the identification of ARGs in this study lays the foundation for developing personalized treatment strategies that not only predict prognosis but also help tailor therapies based on specific molecular profiles. As we continue to explore the relationship between ARGs and RCC, this research may contribute to advancing more effective, targeted therapies for RCC patients, ultimately improving clinical outcomes and quality of life. By addressing Anoikis resistance, we may open new avenues for cancer treatment, not only for RCC but also for other metastatic cancers.</p></section><section id="S0005"><h2 class="pmc_sec_title">5. Methods</h2>
<section id="S0005-S2001"><h3 class="pmc_sec_title">5.1. Data acquisition</h3>
<p><a href="#t0001" class="usa-link">Table 1</a> provides a brief summary of the datasets used in this manuscript. The data for this study were collected in October 2023.</p>
<section class="tw xbox font-sm" id="t0001"><h4 class="obj_head">Table 1.</h4>
<div class="caption p"><p>Summary of datasets used in current study.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<colgroup span="1">
<col width="80pt" span="1">
<col width="80pt" span="1">
<col width="80pt" span="1">
</colgroup>
<thead><tr>
<th align="left" colspan="1" rowspan="1">Datasets</th>
<th align="center" colspan="1" rowspan="1">Brief description (<em>n</em> = enrolled)</th>
<th align="center" colspan="1" rowspan="1">Type</th>
</tr></thead>
<tbody valign="top">
<tr>
<td align="left" colspan="1" rowspan="1">TCGA<sup>a</sup>-KIRC<sup>b</sup>
</td>
<td align="left" colspan="1" rowspan="1">ccRCC<sup>c</sup> patients (<em>n</em> = 518) and healthy controls (<em>n</em> = 71)</td>
<td align="left" colspan="1" rowspan="1">RNA-seq<sup>d</sup>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">E-MTAB-1980</td>
<td align="left" colspan="1" rowspan="1">ccRCC patients (<em>n</em> = 101)</td>
<td align="left" colspan="1" rowspan="1">RNA-seq</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">E-MTAB-3218</td>
<td align="left" colspan="1" rowspan="1">ccRCC patients (<em>n</em> = 114) treated by nivolumab</td>
<td align="left" colspan="1" rowspan="1">RNA-seq</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">E-MTAB-3267</td>
<td align="left" colspan="1" rowspan="1">ccRCC patients (<em>n</em> = 59) treated by sunitinib</td>
<td align="left" colspan="1" rowspan="1">RNA-seq</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">CHECKMATE-009</td>
<td align="left" colspan="1" rowspan="1">ccRCC patients (<em>n</em> = 16) treated by nivolumab</td>
<td align="left" colspan="1" rowspan="1">RNA-seq</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">CHECKMATE-010</td>
<td align="left" colspan="1" rowspan="1">ccRCC patients (<em>n</em> = 45) treated by nivolumab</td>
<td align="left" colspan="1" rowspan="1">RNA-seq</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">CHECKMATE-025</td>
<td align="left" colspan="1" rowspan="1">ccRCC patients (<em>n</em> = 250) treated by nivolumab and everolimus</td>
<td align="left" colspan="1" rowspan="1">RNA-seq</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE22541" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE22541</a></td>
<td align="left" colspan="1" rowspan="1">Primary and pulmonary metastasis of ccRCC (<em>n</em> = 68)</td>
<td align="left" colspan="1" rowspan="1">RNA-seq</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE85258" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE85258</a></td>
<td align="left" colspan="1" rowspan="1">Primary and pulmonary metastasis of ccRCC (<em>n</em> = 29)</td>
<td align="left" colspan="1" rowspan="1">RNA-seq</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE113204" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE113204</a></td>
<td align="left" colspan="1" rowspan="1">Primary, pulmonary metastasis and tumour thrombus of ccRCC (<em>n</em> = 14)</td>
<td align="left" colspan="1" rowspan="1">RNA-seq</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE159115" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE159115</a></td>
<td align="left" colspan="1" rowspan="1">ccRCC patients (<em>n</em> = 7)</td>
<td align="left" colspan="1" rowspan="1">scRNA-seq<sup>e</sup>
</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/t0001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p">
<div class="fn" id="TF1"><p><sup>a</sup>TCGA: The Cancer Genome Atlas.</p></div>
<div class="fn" id="TF2"><p><sup>b</sup>KIRC: Kidney Renal Clear Cell Carcinoma.</p></div>
<div class="fn" id="TF3"><p><sup>c</sup>ccRCC: Clear cell renal cell carcinoma.</p></div>
<div class="fn" id="TF4"><p><sup>d</sup>RNA-seq: RNA sequencing.</p></div>
<div class="fn" id="TF5"><p><sup>e</sup>scRNA-seq: Single-cell RNA sequencing.</p></div>
</div></section><p>We downloaded the RNA-seq counts data and corresponding survival data of The Cancer Genome Atlas of kidney renal clear cell carcinoma (TCGA-KIRC) cohort <em>via</em> UCSC Xena (<a href="http://xena.ucsc.edu/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://xena.ucsc.edu/</a>). Genes expressed in less than half of the patients were excluded, and patients with a survival time of less than 30 days were removed. A total of 518 ccRCC patients, 71 healthy controls and 25,503 genes were enrolled. The somatic mutation data of TCGA-KIRC cohort was obtained from TCGA database by R package ‘TCGAmutations’. The transcription profiling data and survival data of E-MTAB-1980 [<a href="#CIT0067" class="usa-link" aria-describedby="CIT0067">67</a>], E-MTAB-3218 [<a href="#CIT0068" class="usa-link" aria-describedby="CIT0068">68</a>] and E-MTAB-3267 [<a href="#CIT0069" class="usa-link" aria-describedby="CIT0069">69</a>] were download from the ArrayExpress database (<a href="https://www.ebi.ac.uk/biostudies/arrayexpress" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ebi.ac.uk/biostudies/arrayexpress</a>). The RNA-seq data of <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE22541" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE22541</a> [<a href="#CIT0070" class="usa-link" aria-describedby="CIT0070">70</a>], <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE85258" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE85258</a> [<a href="#CIT0071" class="usa-link" aria-describedby="CIT0071">71</a>], <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE113204" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE113204</a> [<a href="#CIT0072" class="usa-link" aria-describedby="CIT0072">72</a>] and the scRNA-seq data of <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE159115" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE159115</a> [<a href="#CIT0073" class="usa-link" aria-describedby="CIT0073">73</a>] were download from the Gene Expression Omnibus (GEO) database (<a href="https://www.ncbi.nlm.nih.gov/geo/" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/geo/</a>). The transcription profiling and survival data of ccRCC patients treated by PD-1 blocker or TKIs were obtained from published randomized trial (CheckMate-009/010/025) [<a href="#CIT0055" class="usa-link" aria-describedby="CIT0055">55</a>].</p>
<p>We collected 410 anoikis-related genes from the GeneCards databases (<a href="https://www.genecards.org/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.genecards.org/</a>) with the Relevance Scores &gt; 0.5 (<a href="https://doi.org/10.1080/07853890.2025.2548042" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Supplementary Table S1</a>).</p></section><section id="S0005-S2002"><h3 class="pmc_sec_title">5.2. Differential expression analysis and machine learning</h3>
<p>R package ‘DESeq2’[<a href="#CIT0074" class="usa-link" aria-describedby="CIT0074">74</a>] was employed to screened the differentially expressed genes (DEGs) between the control and ccRCC in the TCGA-KIRC cohort according to the following criteria: adjusted <em>p</em>-value &lt; 0.05 and | Log<sub>2</sub>(fold-change) | &gt;1. For each gene, we divided the patients into two groups according to the median expression value and performed log-rank test. Genes with <em>p</em> &lt; 0.05 were defined as prognostic genes. The intersection of anoikis-related genes, DEGs and prognostic genes were further analyzed by machine learning.</p>
<p>We used three machine-learning algorithms to obtain the best subset of candidate genes. The least absolute shrinkage and selection operator (LASSO) [<a href="#CIT0075" class="usa-link" aria-describedby="CIT0075">75</a>] algorithm was performed by R package ‘glmnet’ (alpha = 1). The minimum penalty parameter (λ) was selected by tenfold cross-validation. Support vector machine-recursive feature elimination (SVM-RFE) [<a href="#CIT0076" class="usa-link" aria-describedby="CIT0076">76</a>] was used to identify candidate genes with Structural Risk Minimization Principle by R package ‘e1071’ (cost = 1, kernel= ‘radial’). The random forest (RF) [<a href="#CIT0077" class="usa-link" aria-describedby="CIT0077">77</a>] algorithm was performed by R package ‘randomForest’(ntree = 500) and chosen the top 50 genes with great importance. Genes used to establish the Anoikis-Related Genes signature (ARGs) were selected by taking the intersection of the result of three algorithms.</p></section><section id="S0005-S2003"><h3 class="pmc_sec_title">5.3. ARGs establishment and construction of a nomogram</h3>
<p>In each dataset, ARGs was established by cox proportional-hazards model with selected signature genes. For each patient, the formula was as follows: <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="ilm0004"><math id="MATH0004" display="inline"><mtext mathvariant="italic">risk score</mtext><mo>=</mo><mrow><mrow><msubsup><mo stretchy="false">∑</mo><mrow><mi>i</mi><mo>=</mo><mn>1</mn></mrow><mrow><mi>n</mi></mrow></msubsup></mrow><mrow><mo stretchy="false">(</mo><mrow><msub><mrow><mtext mathvariant="italic">Coef</mtext></mrow><mrow><mi>i</mi></mrow></msub></mrow><mi>∗</mi><mrow><msub><mrow><mi>x</mi></mrow><mrow><mi>i</mi></mrow></msub></mrow><mo stretchy="false">)</mo></mrow></mrow><mo>,</mo><mi>i</mi></math></span> represents each selected signature gene and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="ilm0005"><math id="MATH0005" display="inline"><mrow><msub><mrow><mi>x</mi></mrow><mrow><mi>i</mi></mrow></msub></mrow></math></span> is the relative expression. Patients were divided into ARGs High- and Low-risk group according to the median value of risk score. patients with a risk score above the median were labelled ‘High-risk’ and those below were ‘Low-risk’.</p>
<p>In TCGA-KIRC cohort, we collected the clinical data of ccRCC patients (age, grade, T stage and laterality). Together with ARGs, univariate and multivariate cox regression analysis was performed on these classical clinical features by R package ‘ezcox’ (<a href="https://github.com/ShixiangWang/ezcox" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/ShixiangWang/ezcox</a>) [<a href="#CIT0078" class="usa-link" aria-describedby="CIT0078">78</a>]. To facilitate clinical application by urologist, a prognostic nomogram was constructed by R package ‘regplot’. The 1-, 3- and 5-year calibration curves were shown to observe nomogram performance by bootstrapping (1000 times).</p></section><section id="S0005-S2004"><h3 class="pmc_sec_title">5.4. Survival analysis</h3>
<p>In datasets with survival information, R packages ‘survival’ and ‘survminer’ were used to compare the OS or DFS between the two groups of ccRCC patients by log-rank test and Kaplan-Meier curves were generated. To assess the sensitivity and specificity of ARGs in predicting survival, receiver operating characteristic (ROC) curves were depicted AUC was calculated by R package ‘TimeROC’.</p>
<p>Given that anoikis plays and crucial role in suppressing tumour metastasis, we tested whether these selected signature genes could discriminate primary tumour from metastases and evaluated their performance by ROC curves in three datasets which including primary ccRCC and mccRCC patients: <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE22541" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE22541</a>, <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE85258" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE85258</a> and <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE113204" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE113204</a>.</p>
<p>Additionally, in TCGA-KIRC cohort, we compared the 1-, 3- and 5-year AUC of our signature to other recently published cell death related gene signatures: a cupoptosis-related gene signature by Ye et al. [<a href="#CIT0079" class="usa-link" aria-describedby="CIT0079">79</a>], a pyroptosis-related gene signature by Xu et al. [<a href="#CIT0080" class="usa-link" aria-describedby="CIT0080">80</a>], a apoptosis-related gene signature by Huang et al. [<a href="#CIT0081" class="usa-link" aria-describedby="CIT0081">81</a>], a necroptosis-related gene signature by Sun et al. [<a href="#CIT0082" class="usa-link" aria-describedby="CIT0082">82</a>], a autophagy-related gene signature by Chong et al. [<a href="#CIT0083" class="usa-link" aria-describedby="CIT0083">83</a>], a ferroptosis-related gene signature by Zhou et al. [<a href="#CIT0084" class="usa-link" aria-describedby="CIT0084">84</a>].</p></section><section id="S0005-S2005"><h3 class="pmc_sec_title">5.5. Enrichment analysis</h3>
<p>In TCGA-KIRC cohort, we firstly performed differentially expressed genes analysis between the High- and Low-ARGs groups to investigate underlying pathways of ARGs. DEGs were input in Gene Ontology (GO) and Kyoto Encyclopaedia of Genes and Genomes (KEGG) databases using R package ‘clusterProfiler’[<a href="#CIT0085" class="usa-link" aria-describedby="CIT0085">85</a>] to perform Over Representation Analysis. The Gene Set Variation Analysis (GSVA) scores of 50 hallmark pathways and metabolism pathways were calculated and compared between the two groups by R package ‘GSVA’. The pathway information files were obtained from the MSigDB database [<a href="#CIT0086" class="usa-link" aria-describedby="CIT0086">86</a>,<a href="#CIT0087" class="usa-link" aria-describedby="CIT0087">87</a>] and R package ‘scMetabolism’[<a href="#CIT0088" class="usa-link" aria-describedby="CIT0088">88</a>]. Gene Set Enrichment Analysis (GSEA) was performed by R package ‘GSEABase’ and visualized by R package ‘GseaVis’ (<a href="https://github.com/junjunlab/GseaVis" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/junjunlab/GseaVis</a>). Pathways with FDR &lt; 0.25, <em>p</em> &lt; 0.05 and |NES| &gt; 1 were considered enrich significantly.</p></section><section id="S0005-S2006"><h3 class="pmc_sec_title">5.6. Depicting tumour immune microenvironment</h3>
<p>To compare the estimated proportion of immune cells in tumour immune microenvironment (TIME) between the two groups of ccRCC in TCGA-KIRC cohort, we utilized the CIBERSORT algorithm [<a href="#CIT0089" class="usa-link" aria-describedby="CIT0089">89</a>]. In addition, the ESTIMATE algorithm [<a href="#CIT0090" class="usa-link" aria-describedby="CIT0090">90</a>] was used to infer tumour purity and calculate StromalScore, ImmuneScore and ESTIMATEScore. The relationship between ARGs and these TIME features was explored by Spearman’s correlation analysis.</p></section><section id="S0005-S2007"><h3 class="pmc_sec_title">5.7. Prediction of potential therapeutic targets and drug sensitivity</h3>
<p>Since many drugs may be or have been clinically used to ccRCC patients, it is important to determine if there is a distinct sensitivity of these drugs in the High- and Low-ARGs groups. We downloaded the data on the sensitivity of many compounds to multiple cancer cell lines from the Cancer Therapeutic Response Portal (CTRP) database [<a href="#CIT0091" class="usa-link" aria-describedby="CIT0091">91–93</a>] and used R package ‘oncoPredict’[<a href="#CIT0094" class="usa-link" aria-describedby="CIT0094">94</a>] to predict the sensitivity of patients in TCGA-KIRC cohort to certain drugs. 50% inhibiting concentration (IC50) was used to represent the drug sensitivity.</p>
<p>ARGs may be a potential therapeutic target. The data gained from CMap database was analyzed by eXtreme Sum algorithm [<a href="#CIT0095" class="usa-link" aria-describedby="CIT0095">95</a>] to search for drugs that might target the DEGs between the two groups.</p></section><section id="S0005-S2008"><h3 class="pmc_sec_title">5.8. Processing scRNA-seq data</h3>
<p><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE159115" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE159115</a> dataset includes the scRNA-seq data (10X Genomics) from seven ccRCC patients and we converted the row count matrix into a Seurat object by R package ‘Seurat’ (version 3.0.0) [<a href="#CIT0096" class="usa-link" aria-describedby="CIT0096">96</a>]. Cells with fewer than 200 unique molecular identifier (UMI) counts or with more than 20% mitochondrial-related UMI counts were removed. R package ‘DoubletFinder’[<a href="#CIT0097" class="usa-link" aria-describedby="CIT0097">97</a>] was employed to identify and remove potential doublets with a DoubletRate of 7.6% for 10,000 cells. After quality control, we removed the potential batch effect by R package ‘harmony’[<a href="#CIT0098" class="usa-link" aria-describedby="CIT0098">98</a>] and obtained a new integrated data matrix. The matrix was log normalized, scaled and centred by ‘NormalizeData’ and ‘ScaleData’ function. In order to reduce the dimensionality, we conducted principal component analysis (PCA) and find the finest component numbers by ‘Elbowplot’ function. We identified the main cell clusters with a resolution of 0.5 and visualized them by ‘FindNeighbors’, ‘FindClusters’, ‘RunTSNE’ and ‘RunUMAP’ function. A list of cell markers from a published manuscript [<a href="#CIT0031" class="usa-link" aria-describedby="CIT0031">31</a>] was used to annotate all cells. ‘AddModuleScore’ function was used to calculate module scores of 410 anoikis-related genes (ARGscore) for each tumour cell, and divided all tumour cells into two groups based on the median value of ARGscore. Because the lack of interest in ribosome pathways, we used ‘FindMarkers’ function to find DEGs between the two groups of cells after removing ribosome genes (|Log2(fold-change) | &gt; 0.5 and <em>p</em>-adjust &lt; 0.05). GO/KEGG analysis, single sample gene set enrichment analysis (ssGSEA) by R package ‘GSVA’ and metabolic pathway analysis by R package ‘scMetabolism’[<a href="#CIT0088" class="usa-link" aria-describedby="CIT0088">88</a>] were performed between the two groups of tumour cells.</p></section><section id="S0005-S2009"><h3 class="pmc_sec_title">5.9. Pseudotime analysis</h3>
<p>To investigate whether anoikis resistance occurs during ccRCC tumour cells growth, we placed all tumour cells on a trajectory calculated by Monocle2 [<a href="#CIT0099" class="usa-link" aria-describedby="CIT0099">99–101</a>] and analyzed the dynamic changes in their ARGscore. ‘plot_pesudotime_heatmap’ function revealed distinct expression patterns of key genes.</p></section><section id="S0005-S2010"><h3 class="pmc_sec_title">5.10. Cell-cell interaction analysis</h3>
<p>To depict the ligands-receptors-transcriptional regulators-targets networks of the two groups of tumour cells, we employed R package ‘nichenetr’[<a href="#CIT0102" class="usa-link" aria-describedby="CIT0102">102</a>]. Considering that T cells are one of the pivotal factors in current immunotherapy [<a href="#CIT0103" class="usa-link" aria-describedby="CIT0103">103</a>], we focused on the potential regulatory network between tumour cells and T cells.</p></section><section id="S0005-S2011"><h3 class="pmc_sec_title">5.11. Transcription factors analysis</h3>
<p>In order to identify preferentially expressed regulons between the two groups of tumour cells, pySCENIC [<a href="#CIT0104" class="usa-link" aria-describedby="CIT0104">104</a>] was performed based on python. ‘calcRSS’ function was used to calculate the regulon specificity score of each group and ΔFoldChange represents the expression difference.</p></section><section id="S0005-S2012"><h3 class="pmc_sec_title">5.12. Cell culture</h3>
<p>Our experiments were conducted using HK-2 cells (RRID: CVCL_0302), 786 O cells (RRID: CVCL_1051) and Achn cells (RRID: CVCL_1067). All cell lines were obtained from Abiowell Biotechnology Co., Ltd., authenticated by STR profiling within the past 3 years, and tested for mycoplasma contamination. The 786 O cells were cultured in RPMI-1640 medium (supplemented with 10% FBS and 1% antibiotics), MEM medium for Achn cells, and low glucose DMEM for HK-2 cells. All cells were maintained at 37 °C in a 5% CO2 atmosphere. Lentivirus Lv-shMMP9 was employed to knockdown MMP9 expression in 786 O and Achn cells, with Lv-shNC serving as a control. Stably transfected cell lines were selected using puromycin (3 μg/ml, 48 h).</p>
<p>An anoikis model was established by culturing cells on poly-HEMA (Sigma)-coated Petri dishes for 12–72 h, as previously described [<a href="#CIT0032" class="usa-link" aria-describedby="CIT0032">32</a>]. Briefly, poly-HEMA was dissolved in 95% ethanol at 12 mg/ml and applied to Petri dishes at 0.95 μl/mm<sup>2</sup>. Coated dishes were washed twice with PBS before use.</p></section><section id="S0005-S2013"><h3 class="pmc_sec_title">5.13. RT-qPCR analysis</h3>
<p>The relative mRNA levels of MMP9 were assessed by RT-qPCR in HK-2 and 786 O cell lines. mRNA was extracted using RNA Trizol (Invitrogen) and reverse-transcribed into cDNA with the PrimeScrip RT reagent kit (Takara). The ChamQ Universal SYBR<sup>®</sup> qPCR Master Mix (Vazyme) was employed to prepare the RT-qPCR mixture. The mixture was then detected and analyzed on a Roche LightCycler<sup>®</sup> 96, with β-actin serving as an internal control. Primers used in this study are listed in <a href="https://doi.org/10.1080/07853890.2025.2548042" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Supplementary Table S2</a>.</p></section><section id="S0005-S2014"><h3 class="pmc_sec_title">5.14. Western blot assay</h3>
<p>Standard protocols were followed for sample preparation, with cells lysed using RIPA buffer and protein concentrations determined <em>via</em> BCA assay. Proteins were separated on SDS-PAGE gels and transferred onto PVDF membranes. The membranes were then blocked with 5% BSA for 1 h, incubated with primary antibodies overnight at 4 °C, and secondary antibodies for 1 h at room temperature. Immunoblotting bands were visualized using an SDS-PAGE chemiluminescence detector. The antibodies used in this study include: MMP9 (Abmart, #<a href="https://www.ncbi.nlm.nih.gov/nuccore/T55205" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">T55205</a>, 1:1000), β-Actin (Proteintech, #66009-1-Ig, 1:5000), GAPDH (Proteintech, #60004-1-Ig, 1:5000), Cleaved Caspase 3 (CST, #9664S, 1:1000), anti-Mouse secondary antibody (Proteintech, Cat: #SA00001-1, 1:5000) and anti-Rabbit secondary antibody (Proteintech, Cat: #SA00001-2, 1:5000).</p></section><section id="S0005-S2015"><h3 class="pmc_sec_title">5.15. Preparation of plasmids</h3>
<p>The pLKO.1 plasmid, obtained from Miaolingbio, was used as shNC. The shMMP9 plasmid was constructed based on the backbone of the pLKO.1 plasmid, with the shRNA sequence for MMP9 designed by the GPP Web Portal (portals.broadinstitute.org). The primers used in this study are listed in <a href="https://doi.org/10.1080/07853890.2025.2548042" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Supplementary Table S2</a>.</p></section><section id="S0005-S2016"><h3 class="pmc_sec_title">5.16. Flow cytometric analysis</h3>
<p>After treatment, cells were digested by EDTA-free trypsin and washed twice with PBS. The cells were then resuspended in binding buffer and stained with Annexin V and DAPI for 10 min to detect apoptosis. All samples were analyzed using a CyAn bench-top analyzer (Beckman Coulter).</p></section><section id="S0005-S2017"><h3 class="pmc_sec_title">5.17. Wound-healing assay</h3>
<p>Cells were seeded in 6-well plates and multiple parallel scratches were made using a 1 ml pipette tip when cell confluence reached 90% to 100%. Serum-free medium was then added to continue culturing the cells. Images were captured using an inverted microscope, and wound closure was analyzed using Fiji software (a distribution of ImageJ2) [<a href="#CIT0105" class="usa-link" aria-describedby="CIT0105">105</a>].</p></section><section id="S0005-S2018"><h3 class="pmc_sec_title">5.18. Transwell invasion assay</h3>
<p>The invasive capacity of RCC cells was assessed using Transwell chambers (8-μm pore size; Corning, USA) pre-coated with Matrigel (Corning, USA) to simulate the extracellular matrix. Cells were serum-starved for 12 h, resuspended in serum-free medium, and seeded into the upper chamber at a density of 30,000 cells per well. The lower chamber was filled with complete medium containing 10% FBS as a chemoattractant. After 48 h of incubation at 37 °C, non-invading cells on the upper membrane surface were carefully removed with a cotton swab. Invaded cells on the lower surface of the membrane were fixed with 4% paraformaldehyde, stained with 0.1% crystal violet, and imaged under an inverted microscope. The number of invaded cells was quantified using Fiji software [<a href="#CIT0105" class="usa-link" aria-describedby="CIT0105">105</a>].</p></section><section id="S0005-S2019"><h3 class="pmc_sec_title">5.19. Statistical analysis</h3>
<p>Statistical analyses and data visualization were conducted using R software (version 4.1.0) and Python (version 3.9.7), while experimental data were analyzed using Prism 9 (version 9.4.1) and Fiji (version 2.9.0). Measurement data are presented as mean ± SD, and Student’s t-test or one-way ANOVA was employed for appropriate data comparisons. A <em>p</em>-value &lt; 0.05 was considered statistically significant. *, **, *** and **** denote <em>p</em> &lt; 0.05, <em>p</em> &lt; 0.01, <em>p</em> &lt; 0.001 and <em>p</em> &lt; 0.0001, respectively.</p></section></section><section id="sec43"><h2 class="pmc_sec_title">Supplementary Material</h2>
<section class="sm xbox font-sm" id="SM0550"><div class="caption p"><span>Supplemental Material</span></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12366518/bin/IANN_A_2548042_SM0550.docx" data-ga-action="click_feat_suppl" class="usa-link">IANN_A_2548042_SM0550.docx</a><sup> (14.4KB, docx) </sup>
</div></div></section><section class="sm xbox font-sm" id="SM4305"><div class="caption p"><span>Supplementary Table S2.docx</span></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12366518/bin/IANN_A_2548042_SM4305.docx" data-ga-action="click_feat_suppl" class="usa-link">IANN_A_2548042_SM4305.docx</a><sup> (12.9KB, docx) </sup>
</div></div></section><section class="sm xbox font-sm" id="SM4304"><div class="caption p"><span>Supplementary Figure 2.tif</span></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12366518/bin/IANN_A_2548042_SM4304.tif" data-ga-action="click_feat_suppl" class="usa-link">IANN_A_2548042_SM4304.tif</a><sup> (1.1MB, tif) </sup>
</div></div></section><section class="sm xbox font-sm" id="SM4303"><div class="caption p"><span>Supplementary Table S1.xlsx</span></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12366518/bin/IANN_A_2548042_SM4303.xlsx" data-ga-action="click_feat_suppl" class="usa-link">IANN_A_2548042_SM4303.xlsx</a><sup> (37.2KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="SM4302"><div class="caption p"><span>Supplementary Figure 4.tif</span></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12366518/bin/IANN_A_2548042_SM4302.tif" data-ga-action="click_feat_suppl" class="usa-link">IANN_A_2548042_SM4302.tif</a><sup> (1.6MB, tif) </sup>
</div></div></section><section class="sm xbox font-sm" id="SM4301"><div class="caption p"><span>Supplementary Figure 6.tif</span></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12366518/bin/IANN_A_2548042_SM4301.tif" data-ga-action="click_feat_suppl" class="usa-link">IANN_A_2548042_SM4301.tif</a><sup> (3.5MB, tif) </sup>
</div></div></section><section class="sm xbox font-sm" id="SM4300"><div class="caption p"><span>Supplementary Figure 1.tif</span></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12366518/bin/IANN_A_2548042_SM4300.tif" data-ga-action="click_feat_suppl" class="usa-link">IANN_A_2548042_SM4300.tif</a><sup> (1.7MB, tif) </sup>
</div></div></section><section class="sm xbox font-sm" id="SM4299"><div class="caption p"><span>Supplementary Figure 5.tif</span></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12366518/bin/IANN_A_2548042_SM4299.tif" data-ga-action="click_feat_suppl" class="usa-link">IANN_A_2548042_SM4299.tif</a><sup> (11.2MB, tif) </sup>
</div></div></section><section class="sm xbox font-sm" id="SM4298"><div class="caption p"><span>Supplementary Figure 7.tif</span></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12366518/bin/IANN_A_2548042_SM4298.tif" data-ga-action="click_feat_suppl" class="usa-link">IANN_A_2548042_SM4298.tif</a><sup> (6.8MB, tif) </sup>
</div></div></section><section class="sm xbox font-sm" id="SM4296"><div class="caption p"><span>Supplementary Figure 3.tif</span></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12366518/bin/IANN_A_2548042_SM4296.tif" data-ga-action="click_feat_suppl" class="usa-link">IANN_A_2548042_SM4296.tif</a><sup> (2.1MB, tif) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgments</h2>
<p>Not applicable.</p></section><section id="funding-statement1" lang="en"><h2 class="pmc_sec_title">Funding Statement</h2>
<p>This study is supported by Foundation of Jiangxi Provincial Key Laboratory of Urinary System Diseases (2024SSY06111), the Jiangxi Provincial ‘Double Thousand Plan’ Fund Project (jxsq2019201027), Key Project of Natural Science Foundation of Jiangxi Province (20212ACB206013), Youth Project of Natural Science Foundation of Jiangxi Province (20212BAB216037) and National Natural Science Foundation of China (82203365).</p></section><section id="S0007"><h2 class="pmc_sec_title">Ethics approval and consent to participate</h2>
<p>Our study was based on publicly available open-source databases and did not involve any direct experiments with human or animal subjects. According to the regulations set forth by the National Health Commission, the Ministry of Education, the Ministry of Science and Technology, and the State Administration of Traditional Chinese Medicine in the ‘Measures for the Ethical Review of Life Sciences and Medical Research Involving Humans’ (2023), Article 32, studies utilizing publicly available data or involving the observation and collection of publicly accessible data without intervention or disruption are eligible for exemption from ethical review. Therefore, ethical approval was waived by the Ethics Committee of the First Affiliated Hospital of Nanchang University. The study adheres to <em>the Declaration of Helsinki</em>.</p></section><section id="S0008"><h2 class="pmc_sec_title">Disclosure statement</h2>
<p>No potential conflict of interest was reported by the authors.</p></section><section id="S0010"><h2 class="pmc_sec_title">Data availability statement</h2>
<p>All RNA-seq data analyzed in our study are available in UCSC Xena (<a href="http://xena.ucsc.edu/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://xena.ucsc.edu/</a>), the Gene Expression Omnibus (<a href="https://www.ncbi.nlm.nih.gov/geo/" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/geo/</a>) and the ArrayExpress database (<a href="https://www.ebi.ac.uk/biostudies/arrayexpress" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ebi.ac.uk/biostudies/arrayexpress</a>). Other data and code are available upon reasonable request from the corresponding author, Bin Fu or Jianhan Fu.</p></section><section id="ref-list1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="ref-list1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CIT0001">
<span class="label">1.</span><cite>Sung H, Ferlay J, Siegel RL, et al. 
Global Cancer Statistics 2020: GLOBOCAN estimates of incidence and mortality worldwide for 36 cancers in 185 countries. CA Cancer J Clin. 2021;71(3):209–249. doi: 10.3322/caac.21660.
</cite> [<a href="https://doi.org/10.3322/caac.21660" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33538338/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=CA%20Cancer%20J%20Clin&amp;title=Global%20Cancer%20Statistics%202020:%20GLOBOCAN%20estimates%20of%20incidence%20and%20mortality%20worldwide%20for%2036%20cancers%20in%20185%20countries&amp;volume=71&amp;issue=3&amp;publication_year=2021&amp;pages=209-249&amp;pmid=33538338&amp;doi=10.3322/caac.21660&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CIT0002">
<span class="label">2.</span><cite>Linehan WM.
Genetic basis of kidney cancer: role of genomics for the development of disease-based therapeutics. Genome Res. 2012;22(11):2089–2100. doi: 10.1101/gr.131110.111.
</cite> [<a href="https://doi.org/10.1101/gr.131110.111" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3483538/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23038766/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genome%20Res&amp;title=Genetic%20basis%20of%20kidney%20cancer:%20role%20of%20genomics%20for%20the%20development%20of%20disease-based%20therapeutics&amp;volume=22&amp;issue=11&amp;publication_year=2012&amp;pages=2089-2100&amp;pmid=23038766&amp;doi=10.1101/gr.131110.111&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CIT0003">
<span class="label">3.</span><cite>Pal S, Gong J, Mhatre SK, et al. 
Real-world treatment patterns and adverse events in metastatic renal cell carcinoma from a large US claims database. BMC Cancer. 2019;19(1):548. doi: 10.1186/s12885-019-5716-z.
</cite> [<a href="https://doi.org/10.1186/s12885-019-5716-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6555983/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31174493/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Cancer&amp;title=Real-world%20treatment%20patterns%20and%20adverse%20events%20in%20metastatic%20renal%20cell%20carcinoma%20from%20a%20large%20US%20claims%20database&amp;volume=19&amp;issue=1&amp;publication_year=2019&amp;pages=548&amp;pmid=31174493&amp;doi=10.1186/s12885-019-5716-z&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CIT0004">
<span class="label">4.</span><cite>Padala SA, Barsouk A, Thandra KC, et al. 
Epidemiology of renal cell carcinoma. World J Oncol. 2020;11(3):79–87. doi: 10.14740/wjon1279.
</cite> [<a href="https://doi.org/10.14740/wjon1279" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7239575/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32494314/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=World%20J%20Oncol&amp;title=Epidemiology%20of%20renal%20cell%20carcinoma&amp;volume=11&amp;issue=3&amp;publication_year=2020&amp;pages=79-87&amp;pmid=32494314&amp;doi=10.14740/wjon1279&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CIT0005">
<span class="label">5.</span><cite>Sellner F, Compérat E, Klimpfinger M.. 
Genetic and epigenetic characteristics in isolated pancreatic metastases of clear-cell renal cell carcinoma. Int J Mol Sci. 2023;24(22):16292. doi: 10.3390/ijms242216292.
</cite> [<a href="https://doi.org/10.3390/ijms242216292" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10671160/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38003482/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Int%20J%20Mol%20Sci&amp;title=Genetic%20and%20epigenetic%20characteristics%20in%20isolated%20pancreatic%20metastases%20of%20clear-cell%20renal%20cell%20carcinoma&amp;volume=24&amp;issue=22&amp;publication_year=2023&amp;pages=16292&amp;pmid=38003482&amp;doi=10.3390/ijms242216292&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CIT0006">
<span class="label">6.</span><cite>Qin Q, Tachibana I, Margulis V, et al. 
A review of neoadjuvant therapy for localized and locally advanced renal cell carcinoma. Cancers (Basel). 2025;17(2):312. doi: 10.3390/cancers17020312.
</cite> [<a href="https://doi.org/10.3390/cancers17020312" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11763366/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39858094/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cancers%20(Basel)&amp;title=A%20review%20of%20neoadjuvant%20therapy%20for%20localized%20and%20locally%20advanced%20renal%20cell%20carcinoma&amp;volume=17&amp;issue=2&amp;publication_year=2025&amp;pages=312&amp;pmid=39858094&amp;doi=10.3390/cancers17020312&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CIT0007">
<span class="label">7.</span><cite>Petrelli F, Vavassori I, Rossitto M, et al. 
Management of metastatic renal cell carcinoma following first-line immune checkpoint therapy failure: a systematic review. Cancers (Basel). 2024;16(14):2598. doi: 10.3390/cancers16142598.
</cite> [<a href="https://doi.org/10.3390/cancers16142598" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11274494/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39061236/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cancers%20(Basel)&amp;title=Management%20of%20metastatic%20renal%20cell%20carcinoma%20following%20first-line%20immune%20checkpoint%20therapy%20failure:%20a%20systematic%20review&amp;volume=16&amp;issue=14&amp;publication_year=2024&amp;pages=2598&amp;pmid=39061236&amp;doi=10.3390/cancers16142598&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CIT0008">
<span class="label">8.</span><cite>Frantz C, Stewart KM, Weaver VM.. 
The extracellular matrix at a glance. J Cell Sci. 2010;123(Pt 24):4195–4200. doi: 10.1242/jcs.023820.
</cite> [<a href="https://doi.org/10.1242/jcs.023820" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2995612/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21123617/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Cell%20Sci&amp;title=The%20extracellular%20matrix%20at%20a%20glance&amp;volume=123&amp;issue=Pt%2024&amp;publication_year=2010&amp;pages=4195-4200&amp;pmid=21123617&amp;doi=10.1242/jcs.023820&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CIT0009">
<span class="label">9.</span><cite>Kim Y-N, Koo KH, Sung JY, et al. 
Anoikis resistance: an essential prerequisite for tumor metastasis. Int J Cell Biol. 2012;2012:306879. doi: 10.1155/2012/306879.
</cite> [<a href="https://doi.org/10.1155/2012/306879" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3296207/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22505926/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Int%20J%20Cell%20Biol&amp;title=Anoikis%20resistance:%20an%20essential%20prerequisite%20for%20tumor%20metastasis&amp;volume=2012&amp;publication_year=2012&amp;pages=306879&amp;pmid=22505926&amp;doi=10.1155/2012/306879&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CIT0010">
<span class="label">10.</span><cite>Guadamillas MC, Cerezo A, Del Pozo MA.. 
Overcoming anoikis–pathways to anchorage-independent growth in cancer. J Cell Sci. 2011;124(Pt 19):3189–3197. doi: 10.1242/jcs.072165.
</cite> [<a href="https://doi.org/10.1242/jcs.072165" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21940791/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Cell%20Sci&amp;title=Overcoming%20anoikis%E2%80%93pathways%20to%20anchorage-independent%20growth%20in%20cancer&amp;volume=124&amp;issue=Pt%2019&amp;publication_year=2011&amp;pages=3189-3197&amp;pmid=21940791&amp;doi=10.1242/jcs.072165&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CIT0011">
<span class="label">11.</span><cite>He C, He J.. 
Metabolic reprogramming and signaling adaptations in anoikis resistance: mechanisms and therapeutic targets. Mol Cell Biochem. 2025;480(6):3315–3342. doi: 10.1007/s11010-024-05199-3.
</cite> [<a href="https://doi.org/10.1007/s11010-024-05199-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39821582/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol%20Cell%20Biochem&amp;title=Metabolic%20reprogramming%20and%20signaling%20adaptations%20in%20anoikis%20resistance:%20mechanisms%20and%20therapeutic%20targets&amp;volume=480&amp;issue=6&amp;publication_year=2025&amp;pages=3315-3342&amp;pmid=39821582&amp;doi=10.1007/s11010-024-05199-3&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CIT0012">
<span class="label">12.</span><cite>Han Y-H, Wang Y, Lee S-J, et al. 
Regulation of anoikis by extrinsic death receptor pathways. Cell Commun Signal. 2023;21(1):227. doi: 10.1186/s12964-023-01247-5.
</cite> [<a href="https://doi.org/10.1186/s12964-023-01247-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10478316/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37667281/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Commun%20Signal&amp;title=Regulation%20of%20anoikis%20by%20extrinsic%20death%20receptor%20pathways&amp;volume=21&amp;issue=1&amp;publication_year=2023&amp;pages=227&amp;pmid=37667281&amp;doi=10.1186/s12964-023-01247-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CIT0013">
<span class="label">13.</span><cite>Douma S, Van Laar T, Zevenhoven J, et al. 
Suppression of anoikis and induction of metastasis by the neurotrophic receptor TrkB. Nature. 2004;430(7003):1034–1039. doi: 10.1038/nature02765.
</cite> [<a href="https://doi.org/10.1038/nature02765" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15329723/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Suppression%20of%20anoikis%20and%20induction%20of%20metastasis%20by%20the%20neurotrophic%20receptor%20TrkB&amp;volume=430&amp;issue=7003&amp;publication_year=2004&amp;pages=1034-1039&amp;pmid=15329723&amp;doi=10.1038/nature02765&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CIT0014">
<span class="label">14.</span><cite>Tiwari N, Gheldof A, Tatari M, et al. 
EMT as the ultimate survival mechanism of cancer cells. Semin Cancer Biol. 2012;22(3):194–207. doi: 10.1016/j.semcancer.2012.02.013.
</cite> [<a href="https://doi.org/10.1016/j.semcancer.2012.02.013" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22406545/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Semin%20Cancer%20Biol&amp;title=EMT%20as%20the%20ultimate%20survival%20mechanism%20of%20cancer%20cells&amp;volume=22&amp;issue=3&amp;publication_year=2012&amp;pages=194-207&amp;pmid=22406545&amp;doi=10.1016/j.semcancer.2012.02.013&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CIT0015">
<span class="label">15.</span><cite>Khan SU, Fatima K, Malik F.. 
Understanding the cell survival mechanism of anoikis-resistant cancer cells during different steps of metastasis. Clin Exp Metastasis. 2022;39(5):715–726. doi: 10.1007/s10585-022-10172-9.
</cite> [<a href="https://doi.org/10.1007/s10585-022-10172-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35829806/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Clin%20Exp%20Metastasis&amp;title=Understanding%20the%20cell%20survival%20mechanism%20of%20anoikis-resistant%20cancer%20cells%20during%20different%20steps%20of%20metastasis&amp;volume=39&amp;issue=5&amp;publication_year=2022&amp;pages=715-726&amp;pmid=35829806&amp;doi=10.1007/s10585-022-10172-9&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CIT0016">
<span class="label">16.</span><cite>de Sousa Mesquita AP, de Araújo Lopes S, Pernambuco Filho PCA, et al. 
Acquisition of anoikis resistance promotes alterations in the Ras/ERK and PI3K/Akt signaling pathways and matrix remodeling in endothelial cells. Apoptosis. 2017;22(9):1116–1137. doi: 10.1007/s10495-017-1392-0.
</cite> [<a href="https://doi.org/10.1007/s10495-017-1392-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28653224/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Apoptosis&amp;title=Acquisition%20of%20anoikis%20resistance%20promotes%20alterations%20in%20the%20Ras/ERK%20and%20PI3K/Akt%20signaling%20pathways%20and%20matrix%20remodeling%20in%20endothelial%20cells&amp;volume=22&amp;issue=9&amp;publication_year=2017&amp;pages=1116-1137&amp;pmid=28653224&amp;doi=10.1007/s10495-017-1392-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CIT0017">
<span class="label">17.</span><cite>Girnius N, Davis RJ.. 
Anoikis mediated by stress-activated mapk signaling pathways. In: Frisch SM, editor. Anoikis: how the extracellular matrix regulates life-or-death decisions. Cham: Springer International Publishing; 2021. p. 161–172.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=Anoikis:%20how%20the%20extracellular%20matrix%20regulates%20life-or-death%20decisions&amp;publication_year=2021&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CIT0018">
<span class="label">18.</span><cite>Tan X, Yan Y, Song B, et al. 
Focal adhesion kinase: from biological functions to therapeutic strategies. Exp Hematol Oncol. 2023;12(1):83. doi: 10.1186/s40164-023-00446-7.
</cite> [<a href="https://doi.org/10.1186/s40164-023-00446-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10519103/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37749625/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Exp%20Hematol%20Oncol&amp;title=Focal%20adhesion%20kinase:%20from%20biological%20functions%20to%20therapeutic%20strategies&amp;volume=12&amp;issue=1&amp;publication_year=2023&amp;pages=83&amp;pmid=37749625&amp;doi=10.1186/s40164-023-00446-7&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CIT0019">
<span class="label">19.</span><cite>Zang M, Zhang B, Zhang Y, et al. 
CEACAM6 promotes gastric cancer invasion and metastasis by inducing epithelial-mesenchymal transition via PI3K/AKT signaling pathway. PLoS One. 2014;9(11):e112908. doi: 10.1371/journal.pone.0112908.
</cite> [<a href="https://doi.org/10.1371/journal.pone.0112908" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4232574/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25398131/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20One&amp;title=CEACAM6%20promotes%20gastric%20cancer%20invasion%20and%20metastasis%20by%20inducing%20epithelial-mesenchymal%20transition%20via%20PI3K/AKT%20signaling%20pathway&amp;volume=9&amp;issue=11&amp;publication_year=2014&amp;pages=e112908&amp;pmid=25398131&amp;doi=10.1371/journal.pone.0112908&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CIT0020">
<span class="label">20.</span><cite>Kalali D.
The role of the matrix metalloproteinase-9 gene in tumor development and metastasis: a narrative review. Glob Med Genet. 2023;10(2):48–53. doi: 10.1055/s-0043-1768166.
</cite> [<a href="https://doi.org/10.1055/s-0043-1768166" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10110361/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37077369/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Glob%20Med%20Genet&amp;title=The%20role%20of%20the%20matrix%20metalloproteinase-9%20gene%20in%20tumor%20development%20and%20metastasis:%20a%20narrative%20review&amp;volume=10&amp;issue=2&amp;publication_year=2023&amp;pages=48-53&amp;pmid=37077369&amp;doi=10.1055/s-0043-1768166&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CIT0021">
<span class="label">21.</span><cite>Egeblad M, Werb Z.. 
New functions for the matrix metalloproteinases in cancer progression. Nat Rev Cancer. 2002;2(3):161–174. doi: 10.1038/nrc745.
</cite> [<a href="https://doi.org/10.1038/nrc745" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11990853/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Cancer&amp;title=New%20functions%20for%20the%20matrix%20metalloproteinases%20in%20cancer%20progression&amp;volume=2&amp;issue=3&amp;publication_year=2002&amp;pages=161-174&amp;pmid=11990853&amp;doi=10.1038/nrc745&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CIT0022">
<span class="label">22.</span><cite>Shon S-K, Kim A, Kim JY, et al. 
Bone morphogenetic protein-4 induced by NDRG2 expression inhibits MMP-9 activity in breast cancer cells. Biochem Biophys Res Commun. 2009;385(2):198–203. doi: 10.1016/j.bbrc.2009.05.038.
</cite> [<a href="https://doi.org/10.1016/j.bbrc.2009.05.038" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19450561/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biochem%20Biophys%20Res%20Commun&amp;title=Bone%20morphogenetic%20protein-4%20induced%20by%20NDRG2%20expression%20inhibits%20MMP-9%20activity%20in%20breast%20cancer%20cells&amp;volume=385&amp;issue=2&amp;publication_year=2009&amp;pages=198-203&amp;pmid=19450561&amp;doi=10.1016/j.bbrc.2009.05.038&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CIT0023">
<span class="label">23.</span><cite>Mondal S, Adhikari N, Banerjee S, et al. 
Matrix metalloproteinase-9 (MMP-9) and its inhibitors in cancer: a minireview. Eur J Med Chem. 2020;194:112260. doi: 10.1016/j.ejmech.2020.112260.
</cite> [<a href="https://doi.org/10.1016/j.ejmech.2020.112260" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32224379/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Med%20Chem&amp;title=Matrix%20metalloproteinase-9%20(MMP-9)%20and%20its%20inhibitors%20in%20cancer:%20a%20minireview&amp;volume=194&amp;publication_year=2020&amp;pages=112260&amp;pmid=32224379&amp;doi=10.1016/j.ejmech.2020.112260&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CIT0024">
<span class="label">24.</span><cite>Augoff K, Hryniewicz-Jankowska A, Tabola R, et al. 
MMP9: a tough target for targeted therapy for cancer. Cancers (Basel). 2022;14(7):1847.), doi: 10.3390/cancers14071847.
</cite> [<a href="https://doi.org/10.3390/cancers14071847" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8998077/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35406619/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cancers%20(Basel)&amp;title=MMP9:%20a%20tough%20target%20for%20targeted%20therapy%20for%20cancer&amp;volume=14&amp;issue=7&amp;publication_year=2022&amp;pages=1847&amp;pmid=35406619&amp;doi=10.3390/cancers14071847&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CIT0025">
<span class="label">25.</span><cite>Shen C-J, Chan S-H, Lee C-T, et al. 
Oleic acid-induced ANGPTL4 enhances head and neck squamous cell carcinoma anoikis resistance and metastasis via up-regulation of fibronectin. Cancer Lett. 2017;386:110–122. doi: 10.1016/j.canlet.2016.11.012.
</cite> [<a href="https://doi.org/10.1016/j.canlet.2016.11.012" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27865799/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cancer%20Lett&amp;title=Oleic%20acid-induced%20ANGPTL4%20enhances%20head%20and%20neck%20squamous%20cell%20carcinoma%20anoikis%20resistance%20and%20metastasis%20via%20up-regulation%20of%20fibronectin&amp;volume=386&amp;publication_year=2017&amp;pages=110-122&amp;pmid=27865799&amp;doi=10.1016/j.canlet.2016.11.012&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CIT0026">
<span class="label">26.</span><cite>Moro L, Arbini AA, Yao JL, et al. 
Mitochondrial DNA depletion in prostate epithelial cells promotes anoikis resistance and invasion through activation of PI3K/Akt2. Cell Death Differ. 2009;16(4):571–583. doi: 10.1038/cdd.2008.178.
</cite> [<a href="https://doi.org/10.1038/cdd.2008.178" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19079138/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Death%20Differ&amp;title=Mitochondrial%20DNA%20depletion%20in%20prostate%20epithelial%20cells%20promotes%20anoikis%20resistance%20and%20invasion%20through%20activation%20of%20PI3K/Akt2&amp;volume=16&amp;issue=4&amp;publication_year=2009&amp;pages=571-583&amp;pmid=19079138&amp;doi=10.1038/cdd.2008.178&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CIT0027">
<span class="label">27.</span><cite>Ma W-L, Hsu C-L, Yeh C-C, et al. 
Hepatic androgen receptor suppresses hepatocellular carcinoma metastasis through modulation of cell migration and anoikis. Hepatology. 2012;56(1):176–185. doi: 10.1002/hep.25644.
</cite> [<a href="https://doi.org/10.1002/hep.25644" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3673306/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22318717/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Hepatology&amp;title=Hepatic%20androgen%20receptor%20suppresses%20hepatocellular%20carcinoma%20metastasis%20through%20modulation%20of%20cell%20migration%20and%20anoikis&amp;volume=56&amp;issue=1&amp;publication_year=2012&amp;pages=176-185&amp;pmid=22318717&amp;doi=10.1002/hep.25644&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CIT0028">
<span class="label">28.</span><cite>Li W, Liu Z, Zhao C, et al. 
Binding of MMP-9-degraded fibronectin to beta6 integrin promotes invasion via the FAK-Src-related Erk1/2 and PI3K/Akt/Smad-1/5/8 pathways in breast cancer. Oncol Rep. 2015;34(3):1345–1352. doi: 10.3892/or.2015.4103.
</cite> [<a href="https://doi.org/10.3892/or.2015.4103" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26134759/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Oncol%20Rep&amp;title=Binding%20of%20MMP-9-degraded%20fibronectin%20to%20beta6%20integrin%20promotes%20invasion%20via%20the%20FAK-Src-related%20Erk1/2%20and%20PI3K/Akt/Smad-1/5/8%20pathways%20in%20breast%20cancer&amp;volume=34&amp;issue=3&amp;publication_year=2015&amp;pages=1345-1352&amp;pmid=26134759&amp;doi=10.3892/or.2015.4103&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CIT0029">
<span class="label">29.</span><cite>Wolf NK, Kissiov DU, Raulet DH.. 
Roles of natural killer cells in immunity to cancer, and applications to immunotherapy. Nat Rev Immunol. 2023;23(2):90–105. doi: 10.1038/s41577-022-00732-1.
</cite> [<a href="https://doi.org/10.1038/s41577-022-00732-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35637393/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Immunol&amp;title=Roles%20of%20natural%20killer%20cells%20in%20immunity%20to%20cancer,%20and%20applications%20to%20immunotherapy&amp;volume=23&amp;issue=2&amp;publication_year=2023&amp;pages=90-105&amp;pmid=35637393&amp;doi=10.1038/s41577-022-00732-1&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CIT0030">
<span class="label">30.</span><cite>Shan F, Somasundaram A, Bruno TC, et al. 
Therapeutic targeting of regulatory T cells in cancer. Trends Cancer. 2022;8(11):944–961. doi: 10.1016/j.trecan.2022.06.008.
</cite> [<a href="https://doi.org/10.1016/j.trecan.2022.06.008" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9588644/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35853825/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Trends%20Cancer&amp;title=Therapeutic%20targeting%20of%20regulatory%20T%20cells%20in%20cancer&amp;volume=8&amp;issue=11&amp;publication_year=2022&amp;pages=944-961&amp;pmid=35853825&amp;doi=10.1016/j.trecan.2022.06.008&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CIT0031">
<span class="label">31.</span><cite>Young MD, Mitchell TJ, Vieira Braga FA, et al. 
Single-cell transcriptomes from human kidneys reveal the cellular identity of renal tumors. Science. 2018;361(6402):594–599. doi: 10.1126/science.aat1699.
</cite> [<a href="https://doi.org/10.1126/science.aat1699" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6104812/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30093597/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Single-cell%20transcriptomes%20from%20human%20kidneys%20reveal%20the%20cellular%20identity%20of%20renal%20tumors&amp;volume=361&amp;issue=6402&amp;publication_year=2018&amp;pages=594-599&amp;pmid=30093597&amp;doi=10.1126/science.aat1699&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CIT0032">
<span class="label">32.</span><cite>Valentinis B, Reiss K, Baserga R.. 
Insulin-like growth factor-I-mediated survival from anoikis: role of cell aggregation and focal adhesion kinase. J Cell Physiol. 1998;176(3):648–657. doi: .
</cite> [<a href="https://doi.org/10.1002/(SICI)1097-4652(199809)176:3&amp;#x0003c;648::AID-JCP22&amp;#x0003e;3.3.CO;2-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9699518/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Cell%20Physiol&amp;title=Insulin-like%20growth%20factor-I-mediated%20survival%20from%20anoikis:%20role%20of%20cell%20aggregation%20and%20focal%20adhesion%20kinase&amp;volume=176&amp;issue=3&amp;publication_year=1998&amp;pages=648-657&amp;pmid=9699518&amp;doi=10.1002/(SICI)1097-4652(199809)176:3&amp;#x0003c;648::AID-JCP22&amp;#x0003e;3.3.CO;2-6&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CIT0033">
<span class="label">33.</span><cite>Liu B, Liu J, Liu K, et al. 
A prognostic signature of five pseudogenes for predicting lower-grade gliomas. Biomed Pharmacother. 2019;117:109116. doi: 10.1016/j.biopha.2019.109116.
</cite> [<a href="https://doi.org/10.1016/j.biopha.2019.109116" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31247469/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biomed%20Pharmacother&amp;title=A%20prognostic%20signature%20of%20five%20pseudogenes%20for%20predicting%20lower-grade%20gliomas&amp;volume=117&amp;publication_year=2019&amp;pages=109116&amp;pmid=31247469&amp;doi=10.1016/j.biopha.2019.109116&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CIT0034">
<span class="label">34.</span><cite>McFarland KN, Wilkes SR, Koss SE, et al. 
Neural-specific inactivation of ShcA results in increased embryonic neural progenitor apoptosis and microencephaly. J Neurosci. 2006;26(30):7885–7897. doi: 10.1523/JNEUROSCI.3524-05.2006.
</cite> [<a href="https://doi.org/10.1523/JNEUROSCI.3524-05.2006" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6674223/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16870734/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Neurosci&amp;title=Neural-specific%20inactivation%20of%20ShcA%20results%20in%20increased%20embryonic%20neural%20progenitor%20apoptosis%20and%20microencephaly&amp;volume=26&amp;issue=30&amp;publication_year=2006&amp;pages=7885-7897&amp;pmid=16870734&amp;doi=10.1523/JNEUROSCI.3524-05.2006&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CIT0035">
<span class="label">35.</span><cite>Ito K, Park SH, Katsyv I, et al. 
PTK6 regulates growth and survival of endocrine therapy-resistant ER+ breast cancer cells. NPJ Breast Cancer. 2017;3(1):45. doi: 10.1038/s41523-017-0047-1.
</cite> [<a href="https://doi.org/10.1038/s41523-017-0047-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5694002/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29167821/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=NPJ%20Breast%20Cancer&amp;title=PTK6%20regulates%20growth%20and%20survival%20of%20endocrine%20therapy-resistant%20ER+%20breast%20cancer%20cells&amp;volume=3&amp;issue=1&amp;publication_year=2017&amp;pages=45&amp;pmid=29167821&amp;doi=10.1038/s41523-017-0047-1&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CIT0036">
<span class="label">36.</span><cite>Yawata A, Adachi M, Okuda H, et al. 
Prolonged cell survival enhances peritoneal dissemination of gastric cancer cells. Oncogene. 1998;16(20):2681–2686. doi: 10.1038/sj.onc.1201792.
</cite> [<a href="https://doi.org/10.1038/sj.onc.1201792" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9632144/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Oncogene&amp;title=Prolonged%20cell%20survival%20enhances%20peritoneal%20dissemination%20of%20gastric%20cancer%20cells&amp;volume=16&amp;issue=20&amp;publication_year=1998&amp;pages=2681-2686&amp;pmid=9632144&amp;doi=10.1038/sj.onc.1201792&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CIT0037">
<span class="label">37.</span><cite>Zhang C, Chen S, Wang W, et al. 
YY1 regulates the proliferation and invasion of triple-negative breast cancer via activating PLAUR. Funct Integr Genomics. 2023;23(3):269. doi: 10.1007/s10142-023-01193-y.
</cite> [<a href="https://doi.org/10.1007/s10142-023-01193-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37552345/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Funct%20Integr%20Genomics&amp;title=YY1%20regulates%20the%20proliferation%20and%20invasion%20of%20triple-negative%20breast%20cancer%20via%20activating%20PLAUR&amp;volume=23&amp;issue=3&amp;publication_year=2023&amp;pages=269&amp;pmid=37552345&amp;doi=10.1007/s10142-023-01193-y&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CIT0038">
<span class="label">38.</span><cite>Fernández-Ginés R, Encinar JA, Escoll M, et al. 
Specific targeting of the NRF2/beta-TrCP axis promotes beneficial effects in NASH. Redox Biol. 2024;69:103027. doi: 10.1016/j.redox.2024.103027.
</cite> [<a href="https://doi.org/10.1016/j.redox.2024.103027" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10808969/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38184999/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Redox%20Biol&amp;title=Specific%20targeting%20of%20the%20NRF2/beta-TrCP%20axis%20promotes%20beneficial%20effects%20in%20NASH&amp;volume=69&amp;publication_year=2024&amp;pages=103027&amp;pmid=38184999&amp;doi=10.1016/j.redox.2024.103027&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CIT0039">
<span class="label">39.</span><cite>Czepczynska-Krezel H, Krop-Watorek A. [Human carcinoembryonic antigen family proteins, structure and function]. Postepy Hig Med Dosw (Online). 2012;66:521–533.
</cite> [<a href="https://doi.org/10.5604/17322693.1004113" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22922152/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Postepy%20Hig%20Med%20Dosw%20(Online)&amp;title=Human%20carcinoembryonic%20antigen%20family%20proteins,%20structure%20and%20function%5D&amp;volume=66&amp;publication_year=2012&amp;pages=521-533&amp;pmid=22922152&amp;doi=10.5604/17322693.1004113&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CIT0040">
<span class="label">40.</span><cite>Richardson RJ, Dixon J, Jiang R, et al. 
Integration of IRF6 and Jagged2 signalling is essential for controlling palatal adhesion and fusion competence. Hum Mol Genet. 2009;18(14):2632–2642. doi: 10.1093/hmg/ddp201.
</cite> [<a href="https://doi.org/10.1093/hmg/ddp201" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2701335/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19439425/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Hum%20Mol%20Genet&amp;title=Integration%20of%20IRF6%20and%20Jagged2%20signalling%20is%20essential%20for%20controlling%20palatal%20adhesion%20and%20fusion%20competence&amp;volume=18&amp;issue=14&amp;publication_year=2009&amp;pages=2632-2642&amp;pmid=19439425&amp;doi=10.1093/hmg/ddp201&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CIT0041">
<span class="label">41.</span><cite>Podzus J, Kowalczyk-Quintas C, Schuepbach-Mallepell S, et al. 
Ectodysplasin A in biological fluids and diagnosis of ectodermal dysplasia. J Dent Res. 2017;96(2):217–224. doi: 10.1177/0022034516673562.
</cite> [<a href="https://doi.org/10.1177/0022034516673562" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28106506/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Dent%20Res&amp;title=Ectodysplasin%20A%20in%20biological%20fluids%20and%20diagnosis%20of%20ectodermal%20dysplasia&amp;volume=96&amp;issue=2&amp;publication_year=2017&amp;pages=217-224&amp;pmid=28106506&amp;doi=10.1177/0022034516673562&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CIT0042">
<span class="label">42.</span><cite>Zhu B, Wu Y, Luo J, et al. 
MNX1 promotes malignant progression of cervical cancer via repressing the transcription of p21(cip1). Front Oncol. 2020;10:1307. doi: 10.3389/fonc.2020.01307.
</cite> [<a href="https://doi.org/10.3389/fonc.2020.01307" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7431913/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32850410/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Front%20Oncol&amp;title=MNX1%20promotes%20malignant%20progression%20of%20cervical%20cancer%20via%20repressing%20the%20transcription%20of%20p21(cip1)&amp;volume=10&amp;publication_year=2020&amp;pages=1307&amp;pmid=32850410&amp;doi=10.3389/fonc.2020.01307&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CIT0043">
<span class="label">43.</span><cite>Woranush W, Moskopp ML, Noll T, et al. 
Quantifying and mathematical modelling of the influence of soluble adenylate cyclase on cell cycle in human endothelial cells with Bayesian inference. J Cell Mol Med. 2022;26(23):5887–5900. doi: 10.1111/jcmm.17611.
</cite> [<a href="https://doi.org/10.1111/jcmm.17611" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9716222/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36372953/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Cell%20Mol%20Med&amp;title=Quantifying%20and%20mathematical%20modelling%20of%20the%20influence%20of%20soluble%20adenylate%20cyclase%20on%20cell%20cycle%20in%20human%20endothelial%20cells%20with%20Bayesian%20inference&amp;volume=26&amp;issue=23&amp;publication_year=2022&amp;pages=5887-5900&amp;pmid=36372953&amp;doi=10.1111/jcmm.17611&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CIT0044">
<span class="label">44.</span><cite>Shahbazi B, Mafakher L, Arab SS, et al. 
Kallistatin as an inhibitory protein against colorectal cancer cells through binding to LRP6. J Biomol Struct Dyn. 2024;42(2):918–934. doi: 10.1080/07391102.2023.2196704.
</cite> [<a href="https://doi.org/10.1080/07391102.2023.2196704" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37114408/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Biomol%20Struct%20Dyn&amp;title=Kallistatin%20as%20an%20inhibitory%20protein%20against%20colorectal%20cancer%20cells%20through%20binding%20to%20LRP6&amp;volume=42&amp;issue=2&amp;publication_year=2024&amp;pages=918-934&amp;pmid=37114408&amp;doi=10.1080/07391102.2023.2196704&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CIT0045">
<span class="label">45.</span><cite>Liu CL, Pan HW, Torng PL, et al. 
SRPX and HMCN1 regulate cancer‑associated fibroblasts to promote the invasiveness of ovarian carcinoma. Oncol Rep. 2019;42(6):2706–2715. doi: 10.3892/or.2019.7379.
</cite> [<a href="https://doi.org/10.3892/or.2019.7379" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31638245/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Oncol%20Rep&amp;title=SRPX%20and%20HMCN1%20regulate%20cancer%E2%80%91associated%20fibroblasts%20to%20promote%20the%20invasiveness%20of%20ovarian%20carcinoma&amp;volume=42&amp;issue=6&amp;publication_year=2019&amp;pages=2706-2715&amp;pmid=31638245&amp;doi=10.3892/or.2019.7379&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CIT0046">
<span class="label">46.</span><cite>Wen X, Hou J, Qi T, et al. 
Anoikis resistance regulates immune infiltration and drug sensitivity in clear-cell renal cell carcinoma: insights from multi omics, single cell analysis and in vitro experiment. Front Immunol. 2024;15:1427475. doi: 10.3389/fimmu.2024.1427475.
</cite> [<a href="https://doi.org/10.3389/fimmu.2024.1427475" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11215044/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38953023/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Front%20Immunol&amp;title=Anoikis%20resistance%20regulates%20immune%20infiltration%20and%20drug%20sensitivity%20in%20clear-cell%20renal%20cell%20carcinoma:%20insights%20from%20multi%20omics,%20single%20cell%20analysis%20and%20in%20vitro%20experiment&amp;volume=15&amp;publication_year=2024&amp;pages=1427475&amp;pmid=38953023&amp;doi=10.3389/fimmu.2024.1427475&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CIT0047">
<span class="label">47.</span><cite>Jiang Y, Wang Y, Wang Z, et al. 
Anoikis-related genes signature development for clear cell renal cell carcinoma prognosis and tumor microenvironment. Sci Rep. 2023;13(1):18909. doi: 10.1038/s41598-023-46398-0.
</cite> [<a href="https://doi.org/10.1038/s41598-023-46398-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10622575/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37919386/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sci%20Rep&amp;title=Anoikis-related%20genes%20signature%20development%20for%20clear%20cell%20renal%20cell%20carcinoma%20prognosis%20and%20tumor%20microenvironment&amp;volume=13&amp;issue=1&amp;publication_year=2023&amp;pages=18909&amp;pmid=37919386&amp;doi=10.1038/s41598-023-46398-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CIT0048">
<span class="label">48.</span><cite>He Z, Gu Y, Yang H, et al. 
Identification and verification of a novel anoikis-related gene signature with prognostic significance in clear cell renal cell carcinoma. J Cancer Res Clin Oncol. 2023;149(13):11661–11678. doi: 10.1007/s00432-023-05012-6.
</cite> [<a href="https://doi.org/10.1007/s00432-023-05012-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11796694/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37402968/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Cancer%20Res%20Clin%20Oncol&amp;title=Identification%20and%20verification%20of%20a%20novel%20anoikis-related%20gene%20signature%20with%20prognostic%20significance%20in%20clear%20cell%20renal%20cell%20carcinoma&amp;volume=149&amp;issue=13&amp;publication_year=2023&amp;pages=11661-11678&amp;pmid=37402968&amp;doi=10.1007/s00432-023-05012-6&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CIT0049">
<span class="label">49.</span><cite>Chen Z, Liu X, Zhu Z, et al. 
A novel anoikis-related prognostic signature associated with prognosis and immune infiltration landscape in clear cell renal cell carcinoma. Front Genet. 2022;13:1039465. doi: 10.3389/fgene.2022.1039465.
</cite> [<a href="https://doi.org/10.3389/fgene.2022.1039465" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9627172/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36338978/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Front%20Genet&amp;title=A%20novel%20anoikis-related%20prognostic%20signature%20associated%20with%20prognosis%20and%20immune%20infiltration%20landscape%20in%20clear%20cell%20renal%20cell%20carcinoma&amp;volume=13&amp;publication_year=2022&amp;pages=1039465&amp;pmid=36338978&amp;doi=10.3389/fgene.2022.1039465&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CIT0050">
<span class="label">50.</span><cite>Wu Q, Sun Y, Qin X, et al. 
Development and validation of a novel anoikis-related gene signature in clear cell renal cell carcinoma. Front Oncol. 2023;13:1211103. doi: 10.3389/fonc.2023.1211103.
</cite> [<a href="https://doi.org/10.3389/fonc.2023.1211103" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10641395/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37965453/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Front%20Oncol&amp;title=Development%20and%20validation%20of%20a%20novel%20anoikis-related%20gene%20signature%20in%20clear%20cell%20renal%20cell%20carcinoma&amp;volume=13&amp;publication_year=2023&amp;pages=1211103&amp;pmid=37965453&amp;doi=10.3389/fonc.2023.1211103&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CIT0051">
<span class="label">51.</span><cite>Liu Y, Shi Z, Zheng J, et al. 
Establishment and validation of a novel anoikis-related prognostic signature of clear cell renal cell carcinoma. Front Immunol. 2023;14:1171883. doi: 10.3389/fimmu.2023.1171883.
</cite> [<a href="https://doi.org/10.3389/fimmu.2023.1171883" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10086373/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37056778/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Front%20Immunol&amp;title=Establishment%20and%20validation%20of%20a%20novel%20anoikis-related%20prognostic%20signature%20of%20clear%20cell%20renal%20cell%20carcinoma&amp;volume=14&amp;publication_year=2023&amp;pages=1171883&amp;pmid=37056778&amp;doi=10.3389/fimmu.2023.1171883&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CIT0052">
<span class="label">52.</span><cite>Li J, Cao Q, Tong M.. 
Deciphering anoikis resistance and identifying prognostic biomarkers in clear cell renal cell carcinoma epithelial cells. Sci Rep. 2024;14(1):12044. doi: 10.1038/s41598-024-62978-0.
</cite> [<a href="https://doi.org/10.1038/s41598-024-62978-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11130322/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38802480/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sci%20Rep&amp;title=Deciphering%20anoikis%20resistance%20and%20identifying%20prognostic%20biomarkers%20in%20clear%20cell%20renal%20cell%20carcinoma%20epithelial%20cells&amp;volume=14&amp;issue=1&amp;publication_year=2024&amp;pages=12044&amp;pmid=38802480&amp;doi=10.1038/s41598-024-62978-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CIT0053">
<span class="label">53.</span><cite>Motzer RJ, Banchereau R, Hamidi H, et al. 
Molecular subsets in renal cancer determine outcome to checkpoint and angiogenesis blockade. Cancer Cell. 2020;38(6):803–817 e4. doi: 10.1016/j.ccell.2020.10.011.
</cite> [<a href="https://doi.org/10.1016/j.ccell.2020.10.011" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8436590/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33157048/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cancer%20Cell&amp;title=Molecular%20subsets%20in%20renal%20cancer%20determine%20outcome%20to%20checkpoint%20and%20angiogenesis%20blockade&amp;volume=38&amp;issue=6&amp;publication_year=2020&amp;pages=803-817%20e4&amp;pmid=33157048&amp;doi=10.1016/j.ccell.2020.10.011&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CIT0054">
<span class="label">54.</span><cite>Miao D, Margolis CA, Gao W, et al. 
Genomic correlates of response to immune checkpoint therapies in clear cell renal cell carcinoma. Science. 2018;359(6377):801–806. doi: 10.1126/science.aan5951.
</cite> [<a href="https://doi.org/10.1126/science.aan5951" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6035749/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29301960/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Genomic%20correlates%20of%20response%20to%20immune%20checkpoint%20therapies%20in%20clear%20cell%20renal%20cell%20carcinoma&amp;volume=359&amp;issue=6377&amp;publication_year=2018&amp;pages=801-806&amp;pmid=29301960&amp;doi=10.1126/science.aan5951&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CIT0055">
<span class="label">55.</span><cite>Braun DA, Hou Y, Bakouny Z, et al. 
Interplay of somatic alterations and immune infiltration modulates response to PD-1 blockade in advanced clear cell renal cell carcinoma. Nat Med. 2020;26(6):909–918. doi: 10.1038/s41591-020-0839-y.
</cite> [<a href="https://doi.org/10.1038/s41591-020-0839-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7499153/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32472114/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Med&amp;title=Interplay%20of%20somatic%20alterations%20and%20immune%20infiltration%20modulates%20response%20to%20PD-1%20blockade%20in%20advanced%20clear%20cell%20renal%20cell%20carcinoma&amp;volume=26&amp;issue=6&amp;publication_year=2020&amp;pages=909-918&amp;pmid=32472114&amp;doi=10.1038/s41591-020-0839-y&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CIT0056">
<span class="label">56.</span><cite>Zhu X, Wang K, Chen Y.. 
Ophiopogonin D suppresses TGF-beta1-mediated metastatic behavior of MDA-MB-231 breast carcinoma cells via regulating ITGB1/FAK/Src/AKT/beta-catenin/MMP-9 signaling axis. Toxicol In Vitro. 2020;69:104973. doi: 10.1016/j.tiv.2020.104973.
</cite> [<a href="https://doi.org/10.1016/j.tiv.2020.104973" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32818624/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Toxicol%20In%20Vitro&amp;title=Ophiopogonin%20D%20suppresses%20TGF-beta1-mediated%20metastatic%20behavior%20of%20MDA-MB-231%20breast%20carcinoma%20cells%20via%20regulating%20ITGB1/FAK/Src/AKT/beta-catenin/MMP-9%20signaling%20axis&amp;volume=69&amp;publication_year=2020&amp;pages=104973&amp;pmid=32818624&amp;doi=10.1016/j.tiv.2020.104973&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CIT0057">
<span class="label">57.</span><cite>Sun Q, Yang Z, Li P, et al. 
A novel miRNA identified in GRSF1 complex drives the metastasis via the PIK3R3/AKT/NF-kappaB and TIMP3/MMP9 pathways in cervical cancer cells. Cell Death Dis. 2019;10(9):636. doi: 10.1038/s41419-019-1841-5.
</cite> [<a href="https://doi.org/10.1038/s41419-019-1841-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6717739/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31474757/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Death%20Dis&amp;title=A%20novel%20miRNA%20identified%20in%20GRSF1%20complex%20drives%20the%20metastasis%20via%20the%20PIK3R3/AKT/NF-kappaB%20and%20TIMP3/MMP9%20pathways%20in%20cervical%20cancer%20cells&amp;volume=10&amp;issue=9&amp;publication_year=2019&amp;pages=636&amp;pmid=31474757&amp;doi=10.1038/s41419-019-1841-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="/articles/PMC12304183/" class="text-red">Retracted</a>]</li>
<li id="CIT0058">
<span class="label">58.</span><cite>Cao Z, Livas T, Kyprianou N.. 
Anoikis and EMT: lethal “liaisons” during cancer progression. Crit Rev Oncog. 2016;21(3-4):155–168. doi: 10.1615/CritRevOncog.2016016955.
</cite> [<a href="https://doi.org/10.1615/CritRevOncog.2016016955" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5451151/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27915969/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Crit%20Rev%20Oncog&amp;title=Anoikis%20and%20EMT:%20lethal%20%E2%80%9Cliaisons%E2%80%9D%20during%20cancer%20progression&amp;volume=21&amp;issue=3-4&amp;publication_year=2016&amp;pages=155-168&amp;pmid=27915969&amp;doi=10.1615/CritRevOncog.2016016955&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CIT0059">
<span class="label">59.</span><cite>Tan C, Song M, Stamer WD, et al. 
miR-21-5p: a viable therapeutic strategy for regulating intraocular pressure. Exp Eye Res. 2020;200:108197. doi: 10.1016/j.exer.2020.108197.
</cite> [<a href="https://doi.org/10.1016/j.exer.2020.108197" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32871166/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Exp%20Eye%20Res&amp;title=miR-21-5p:%20a%20viable%20therapeutic%20strategy%20for%20regulating%20intraocular%20pressure&amp;volume=200&amp;publication_year=2020&amp;pages=108197&amp;pmid=32871166&amp;doi=10.1016/j.exer.2020.108197&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CIT0060">
<span class="label">60.</span><cite>Lim W-C, Choi JW, Song N-E, et al. 
Polysaccharide isolated from persimmon leaves (Diospyros kaki Thunb.) suppresses TGF-beta1-induced epithelial-to-mesenchymal transition in A549 cells. Int J Biol Macromol. 2020;164:3835–3845. doi: 10.1016/j.ijbiomac.2020.08.155.
</cite> [<a href="https://doi.org/10.1016/j.ijbiomac.2020.08.155" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32835798/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Int%20J%20Biol%20Macromol&amp;title=Polysaccharide%20isolated%20from%20persimmon%20leaves%20(Diospyros%20kaki%20Thunb.)%20suppresses%20TGF-beta1-induced%20epithelial-to-mesenchymal%20transition%20in%20A549%20cells&amp;volume=164&amp;publication_year=2020&amp;pages=3835-3845&amp;pmid=32835798&amp;doi=10.1016/j.ijbiomac.2020.08.155&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CIT0061">
<span class="label">61.</span><cite>Di-Poi N, et al. 
Antiapoptotic role of PPARbeta in keratinocytes via transcriptional control of the Akt1 signaling pathway. Mol Cell. 2002;10(4):721–733.
</cite> [<a href="https://doi.org/10.1016/s1097-2765(02)00646-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12419217/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol%20Cell&amp;title=Antiapoptotic%20role%20of%20PPARbeta%20in%20keratinocytes%20via%20transcriptional%20control%20of%20the%20Akt1%20signaling%20pathway&amp;volume=10&amp;issue=4&amp;publication_year=2002&amp;pages=721-733&amp;pmid=12419217&amp;doi=10.1016/s1097-2765(02)00646-9&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CIT0062">
<span class="label">62.</span><cite>Sakamoto S, Schwarze S, Kyprianou N.. 
Anoikis disruption of focal adhesion-Akt signaling impairs renal cell carcinoma. Eur Urol. 2011;59(5):734–744. doi: 10.1016/j.eururo.2010.12.038.
</cite> [<a href="https://doi.org/10.1016/j.eururo.2010.12.038" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5458734/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21269758/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Eur%20Urol&amp;title=Anoikis%20disruption%20of%20focal%20adhesion-Akt%20signaling%20impairs%20renal%20cell%20carcinoma&amp;volume=59&amp;issue=5&amp;publication_year=2011&amp;pages=734-744&amp;pmid=21269758&amp;doi=10.1016/j.eururo.2010.12.038&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CIT0063">
<span class="label">63.</span><cite>Takaha N, Sowa Y, Takeuchi I, et al. 
Expression and role of HMGA1 in renal cell carcinoma. J Urol. 2012;187(6):2215–2222. doi: 10.1016/j.juro.2012.01.069.
</cite> [<a href="https://doi.org/10.1016/j.juro.2012.01.069" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22503056/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Urol&amp;title=Expression%20and%20role%20of%20HMGA1%20in%20renal%20cell%20carcinoma&amp;volume=187&amp;issue=6&amp;publication_year=2012&amp;pages=2215-2222&amp;pmid=22503056&amp;doi=10.1016/j.juro.2012.01.069&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CIT0064">
<span class="label">64.</span><cite>Wang L, Li C, Wang J, et al. 
Transformable ECM deprivation system effectively suppresses renal cell carcinoma by reversing anoikis resistance and increasing chemotherapy sensitivity. Adv Mater. 2022;34(43):e2203518. doi: 10.1002/adma.202203518.
</cite> [<a href="https://doi.org/10.1002/adma.202203518" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36004775/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Adv%20Mater&amp;title=Transformable%20ECM%20deprivation%20system%20effectively%20suppresses%20renal%20cell%20carcinoma%20by%20reversing%20anoikis%20resistance%20and%20increasing%20chemotherapy%20sensitivity&amp;volume=34&amp;issue=43&amp;publication_year=2022&amp;pages=e2203518&amp;pmid=36004775&amp;doi=10.1002/adma.202203518&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CIT0065">
<span class="label">65.</span><cite>Zhang P, Xing Z, Li X, et al. 
Tyrosine receptor kinase B silencing inhibits anoikis‑resistance and improves anticancer efficiency of sorafenib in human renal cancer cells. Int J Oncol. 2016;48(4):1417–1425. doi: 10.3892/ijo.2016.3356.
</cite> [<a href="https://doi.org/10.3892/ijo.2016.3356" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26820170/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Int%20J%20Oncol&amp;title=Tyrosine%20receptor%20kinase%20B%20silencing%20inhibits%20anoikis%E2%80%91resistance%20and%20improves%20anticancer%20efficiency%20of%20sorafenib%20in%20human%20renal%20cancer%20cells&amp;volume=48&amp;issue=4&amp;publication_year=2016&amp;pages=1417-1425&amp;pmid=26820170&amp;doi=10.3892/ijo.2016.3356&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CIT0066">
<span class="label">66.</span><cite>Wang J, Qi X, Wang Q, et al. 
The role and therapeutic significance of the anoikis pathway in renal clear cell carcinoma. Front Oncol. 2022;12:1009984. doi: 10.3389/fonc.2022.1009984.
</cite> [<a href="https://doi.org/10.3389/fonc.2022.1009984" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9557223/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36249029/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Front%20Oncol&amp;title=The%20role%20and%20therapeutic%20significance%20of%20the%20anoikis%20pathway%20in%20renal%20clear%20cell%20carcinoma&amp;volume=12&amp;publication_year=2022&amp;pages=1009984&amp;pmid=36249029&amp;doi=10.3389/fonc.2022.1009984&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CIT0067">
<span class="label">67.</span><cite>Sato Y, Yoshizato T, Shiraishi Y, et al. 
Integrated molecular analysis of clear-cell renal cell carcinoma. Nat Genet. 2013;45(8):860–867. doi: 10.1038/ng.2699.
</cite> [<a href="https://doi.org/10.1038/ng.2699" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23797736/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&amp;title=Integrated%20molecular%20analysis%20of%20clear-cell%20renal%20cell%20carcinoma&amp;volume=45&amp;issue=8&amp;publication_year=2013&amp;pages=860-867&amp;pmid=23797736&amp;doi=10.1038/ng.2699&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CIT0068">
<span class="label">68.</span><cite>Choueiri TK, Fishman MN, Escudier B, et al. 
Immunomodulatory activity of nivolumab in metastatic renal cell carcinoma. Clin Cancer Res. 2016;22(22):5461–5471. doi: 10.1158/1078-0432.CCR-15-2839.
</cite> [<a href="https://doi.org/10.1158/1078-0432.CCR-15-2839" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5106340/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27169994/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Clin%20Cancer%20Res&amp;title=Immunomodulatory%20activity%20of%20nivolumab%20in%20metastatic%20renal%20cell%20carcinoma&amp;volume=22&amp;issue=22&amp;publication_year=2016&amp;pages=5461-5471&amp;pmid=27169994&amp;doi=10.1158/1078-0432.CCR-15-2839&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CIT0069">
<span class="label">69.</span><cite>Beuselinck B, Job S, Becht E, et al. 
Molecular subtypes of clear cell renal cell carcinoma are associated with sunitinib response in the metastatic setting. Clin Cancer Res. 2015;21(6):1329–1339. doi: 10.1158/1078-0432.CCR-14-1128.
</cite> [<a href="https://doi.org/10.1158/1078-0432.CCR-14-1128" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25583177/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Clin%20Cancer%20Res&amp;title=Molecular%20subtypes%20of%20clear%20cell%20renal%20cell%20carcinoma%20are%20associated%20with%20sunitinib%20response%20in%20the%20metastatic%20setting&amp;volume=21&amp;issue=6&amp;publication_year=2015&amp;pages=1329-1339&amp;pmid=25583177&amp;doi=10.1158/1078-0432.CCR-14-1128&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CIT0070">
<span class="label">70.</span><cite>Wuttig D, Zastrow S, Füssel S, et al. 
CD31, EDNRB and TSPAN7 are promising prognostic markers in clear-cell renal cell carcinoma revealed by genome-wide expression analyses of primary tumors and metastases. Int J Cancer. 2012;131(5):E693–704. doi: 10.1002/ijc.27419.
</cite> [<a href="https://doi.org/10.1002/ijc.27419" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22213152/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Int%20J%20Cancer&amp;title=CD31,%20EDNRB%20and%20TSPAN7%20are%20promising%20prognostic%20markers%20in%20clear-cell%20renal%20cell%20carcinoma%20revealed%20by%20genome-wide%20expression%20analyses%20of%20primary%20tumors%20and%20metastases&amp;volume=131&amp;issue=5&amp;publication_year=2012&amp;pages=E693-704&amp;pmid=22213152&amp;doi=10.1002/ijc.27419&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CIT0071">
<span class="label">71.</span><cite>Ho TH, Serie DJ, Parasramka M, et al. 
Differential gene expression profiling of matched primary renal cell carcinoma and metastases reveals upregulation of extracellular matrix genes. Ann Oncol. 2017;28(3):604–610. doi: 10.1093/annonc/mdw652.
</cite> [<a href="https://doi.org/10.1093/annonc/mdw652" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5834119/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27993815/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Ann%20Oncol&amp;title=Differential%20gene%20expression%20profiling%20of%20matched%20primary%20renal%20cell%20carcinoma%20and%20metastases%20reveals%20upregulation%20of%20extracellular%20matrix%20genes&amp;volume=28&amp;issue=3&amp;publication_year=2017&amp;pages=604-610&amp;pmid=27993815&amp;doi=10.1093/annonc/mdw652&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CIT0072">
<span class="label">72.</span><cite>Dagher J, Brunot A, Evrard B, et al. 
Multiple metastatic clones assessed by an integrative multiomics strategy in clear cell renal carcinoma: a case study. J Clin Pathol. 2022;75(6):426–430. doi: 10.1136/jclinpath-2020-207326.
</cite> [<a href="https://doi.org/10.1136/jclinpath-2020-207326" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33766955/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Pathol&amp;title=Multiple%20metastatic%20clones%20assessed%20by%20an%20integrative%20multiomics%20strategy%20in%20clear%20cell%20renal%20carcinoma:%20a%20case%20study&amp;volume=75&amp;issue=6&amp;publication_year=2022&amp;pages=426-430&amp;pmid=33766955&amp;doi=10.1136/jclinpath-2020-207326&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CIT0073">
<span class="label">73.</span><cite>Zhang Y, Narayanan SP, Mannan R, et al. 
Single-cell analyses of renal cell cancers reveal insights into tumor microenvironment, cell of origin, and therapy response. Proc Natl Acad Sci U S A. 2021;118(24):e2103240118. doi: 10.1073/pnas.2103240118.
</cite> [<a href="https://doi.org/10.1073/pnas.2103240118" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8214680/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34099557/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&amp;title=Single-cell%20analyses%20of%20renal%20cell%20cancers%20reveal%20insights%20into%20tumor%20microenvironment,%20cell%20of%20origin,%20and%20therapy%20response&amp;volume=118&amp;issue=24&amp;publication_year=2021&amp;pages=e2103240118&amp;pmid=34099557&amp;doi=10.1073/pnas.2103240118&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CIT0074">
<span class="label">74.</span><cite>Love MI, Huber W, Anders S.. 
Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2. Genome Biol. 2014;15(12):550. doi: 10.1186/s13059-014-0550-8.
</cite> [<a href="https://doi.org/10.1186/s13059-014-0550-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4302049/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25516281/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genome%20Biol&amp;title=Moderated%20estimation%20of%20fold%20change%20and%20dispersion%20for%20RNA-seq%20data%20with%20DESeq2&amp;volume=15&amp;issue=12&amp;publication_year=2014&amp;pages=550&amp;pmid=25516281&amp;doi=10.1186/s13059-014-0550-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CIT0075">
<span class="label">75.</span><cite>Friedman J, Hastie T, Tibshirani R.. 
Regularization paths for generalized linear models via coordinate descent. J Stat Softw. 2010;33(1):1–22. doi: 10.18637/jss.v033.i01.
</cite> [<a href="https://doi.org/10.18637/jss.v033.i01" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2929880/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20808728/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Stat%20Softw&amp;title=Regularization%20paths%20for%20generalized%20linear%20models%20via%20coordinate%20descent&amp;volume=33&amp;issue=1&amp;publication_year=2010&amp;pages=1-22&amp;pmid=20808728&amp;doi=10.18637/jss.v033.i01&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CIT0076">
<span class="label">76.</span><cite>Guo P, Luo Y, Mai G, et al. 
Gene expression profile based classification models of psoriasis. Genomics. 2014;103(1):48–55. doi: 10.1016/j.ygeno.2013.11.001.
</cite> [<a href="https://doi.org/10.1016/j.ygeno.2013.11.001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24239985/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genomics&amp;title=Gene%20expression%20profile%20based%20classification%20models%20of%20psoriasis&amp;volume=103&amp;issue=1&amp;publication_year=2014&amp;pages=48-55&amp;pmid=24239985&amp;doi=10.1016/j.ygeno.2013.11.001&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CIT0077">
<span class="label">77.</span><cite>Sandri M, Zuccolotto P.. 
Variable selection using random forests. Berlin, Heidelberg: Springer Berlin Heidelberg; 2006.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=Variable%20selection%20using%20random%20forests&amp;publication_year=2006&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CIT0078">
<span class="label">78.</span><cite>Wang S, Liu X, Li J, et al. 
ezcox: An R/CRAN package for cox model batch processing and visualization. arXiv. 2021. doi: 10.48550/arXiv.2110.14232</cite> [<a href="https://doi.org/10.48550/arXiv.2110.14232" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang%20S,%20Liu%20X,%20Li%20J,%20et%20al.%20ezcox:%20An%20R/CRAN%20package%20for%20cox%20model%20batch%20processing%20and%20visualization.%20arXiv.%202021.%20doi:%2010.48550/arXiv.2110.14232" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CIT0079">
<span class="label">79.</span><cite>Mei W, Liu X, Jia X, et al. 
A cuproptosis-related gene model for predicting the prognosis of clear cell renal cell carcinoma. Front Genet. 2022;13:905518. doi: 10.3389/fgene.2022.905518.
</cite> [<a href="https://doi.org/10.3389/fgene.2022.905518" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9450221/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36092880/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Front%20Genet&amp;title=A%20cuproptosis-related%20gene%20model%20for%20predicting%20the%20prognosis%20of%20clear%20cell%20renal%20cell%20carcinoma&amp;volume=13&amp;publication_year=2022&amp;pages=905518&amp;pmid=36092880&amp;doi=10.3389/fgene.2022.905518&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CIT0080">
<span class="label">80.</span><cite>Pan X-Q, Huang W, Jin L-W, et al. 
A novel pyroptosis-related prognostic signature for risk stratification and clinical prognosis in clear cell renal cell carcinoma. Dis Markers. 2022. 2022;2022:8093837.
</cite> [<a href="https://doi.org/10.1155/2022/8093837" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8927973/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35308143/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Dis%20Markers&amp;title=A%20novel%20pyroptosis-related%20prognostic%20signature%20for%20risk%20stratification%20and%20clinical%20prognosis%20in%20clear%20cell%20renal%20cell%20carcinoma&amp;volume=2022&amp;publication_year=2022&amp;pages=8093837&amp;pmid=35308143&amp;doi=10.1155/2022/8093837&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CIT0081">
<span class="label">81.</span><cite>Zhong W, Zhang F, Huang C, et al. 
Identification of an apoptosis-related prognostic gene signature and molecular subtypes of clear cell renal cell carcinoma (ccRCC). J Cancer. 2021;12(11):3265–3276. doi: 10.7150/jca.51812.
</cite> [<a href="https://doi.org/10.7150/jca.51812" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8100811/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33976736/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Cancer&amp;title=Identification%20of%20an%20apoptosis-related%20prognostic%20gene%20signature%20and%20molecular%20subtypes%20of%20clear%20cell%20renal%20cell%20carcinoma%20(ccRCC)&amp;volume=12&amp;issue=11&amp;publication_year=2021&amp;pages=3265-3276&amp;pmid=33976736&amp;doi=10.7150/jca.51812&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CIT0082">
<span class="label">82.</span><cite>Bao J-H, Li J-B, Lin H-S, et al. 
Deciphering a novel necroptosis-related miRNA signature for predicting the prognosis of clear cell renal carcinoma. Anal Cell Pathol (Amst). 2022;2022:2721005. doi: 10.1155/2022/2721005.
</cite> [<a href="https://doi.org/10.1155/2022/2721005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9061065/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35509814/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Anal%20Cell%20Pathol%20(Amst)&amp;title=Deciphering%20a%20novel%20necroptosis-related%20miRNA%20signature%20for%20predicting%20the%20prognosis%20of%20clear%20cell%20renal%20carcinoma&amp;volume=2022&amp;publication_year=2022&amp;pages=2721005&amp;pmid=35509814&amp;doi=10.1155/2022/2721005&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CIT0083">
<span class="label">83.</span><cite>He M, Li M, Guan Y, et al. 
A new prognostic risk score: based on the analysis of autophagy-related genes and renal cell carcinoma. Front Genet. 2021;12:820154. doi: 10.3389/fgene.2021.820154.
</cite> [<a href="https://doi.org/10.3389/fgene.2021.820154" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8884161/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35237298/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Front%20Genet&amp;title=A%20new%20prognostic%20risk%20score:%20based%20on%20the%20analysis%20of%20autophagy-related%20genes%20and%20renal%20cell%20carcinoma&amp;volume=12&amp;publication_year=2021&amp;pages=820154&amp;pmid=35237298&amp;doi=10.3389/fgene.2021.820154&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CIT0084">
<span class="label">84.</span><cite>Zhu Z, Zhang C, Qian J, et al. 
Construction and validation of a ferroptosis-related long noncoding RNA signature in clear cell renal cell carcinoma. Cancer Cell Int. 2022;22(1):283. doi: 10.1186/s12935-022-02700-0.
</cite> [<a href="https://doi.org/10.1186/s12935-022-02700-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9476564/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36104748/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cancer%20Cell%20Int&amp;title=Construction%20and%20validation%20of%20a%20ferroptosis-related%20long%20noncoding%20RNA%20signature%20in%20clear%20cell%20renal%20cell%20carcinoma&amp;volume=22&amp;issue=1&amp;publication_year=2022&amp;pages=283&amp;pmid=36104748&amp;doi=10.1186/s12935-022-02700-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CIT0085">
<span class="label">85.</span><cite>Yu G, Wang L-G, Han Y, et al. 
clusterProfiler: an R package for comparing biological themes among gene clusters. OMICS. 2012;16(5):284–287. doi: 10.1089/omi.2011.0118.
</cite> [<a href="https://doi.org/10.1089/omi.2011.0118" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3339379/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22455463/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=OMICS&amp;title=clusterProfiler:%20an%20R%20package%20for%20comparing%20biological%20themes%20among%20gene%20clusters&amp;volume=16&amp;issue=5&amp;publication_year=2012&amp;pages=284-287&amp;pmid=22455463&amp;doi=10.1089/omi.2011.0118&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CIT0086">
<span class="label">86.</span><cite>Subramanian A, Tamayo P, Mootha VK, et al. 
Gene set enrichment analysis: a knowledge-based approach for interpreting genome-wide expression profiles. Proc Natl Acad Sci U S A. 2005;102(43):15545–15550. doi: 10.1073/pnas.0506580102.
</cite> [<a href="https://doi.org/10.1073/pnas.0506580102" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1239896/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16199517/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&amp;title=Gene%20set%20enrichment%20analysis:%20a%20knowledge-based%20approach%20for%20interpreting%20genome-wide%20expression%20profiles&amp;volume=102&amp;issue=43&amp;publication_year=2005&amp;pages=15545-15550&amp;pmid=16199517&amp;doi=10.1073/pnas.0506580102&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CIT0087">
<span class="label">87.</span><cite>Liberzon A, Birger C, Thorvaldsdóttir H, et al. 
The molecular signatures database (MSigDB) hallmark gene set collection. Cell Syst. 2015;1(6):417–425. doi: 10.1016/j.cels.2015.12.004.
</cite> [<a href="https://doi.org/10.1016/j.cels.2015.12.004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4707969/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26771021/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Syst&amp;title=The%20molecular%20signatures%20database%20(MSigDB)%20hallmark%20gene%20set%20collection&amp;volume=1&amp;issue=6&amp;publication_year=2015&amp;pages=417-425&amp;pmid=26771021&amp;doi=10.1016/j.cels.2015.12.004&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CIT0088">
<span class="label">88.</span><cite>Wu Y, Yang S, Ma J, et al. 
Spatiotemporal immune landscape of colorectal cancer liver metastasis at single-cell level. Cancer Discov. 2022;12(1):134–153. doi: 10.1158/2159-8290.CD-21-0316.
</cite> [<a href="https://doi.org/10.1158/2159-8290.CD-21-0316" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34417225/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cancer%20Discov&amp;title=Spatiotemporal%20immune%20landscape%20of%20colorectal%20cancer%20liver%20metastasis%20at%20single-cell%20level&amp;volume=12&amp;issue=1&amp;publication_year=2022&amp;pages=134-153&amp;pmid=34417225&amp;doi=10.1158/2159-8290.CD-21-0316&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CIT0089">
<span class="label">89.</span><cite>Newman AM, Liu CL, Green MR, et al. 
Robust enumeration of cell subsets from tissue expression profiles. Nat Methods. 2015;12(5):453–457. doi: 10.1038/nmeth.3337.
</cite> [<a href="https://doi.org/10.1038/nmeth.3337" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4739640/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25822800/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Methods&amp;title=Robust%20enumeration%20of%20cell%20subsets%20from%20tissue%20expression%20profiles&amp;volume=12&amp;issue=5&amp;publication_year=2015&amp;pages=453-457&amp;pmid=25822800&amp;doi=10.1038/nmeth.3337&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CIT0090">
<span class="label">90.</span><cite>Yoshihara K, Shahmoradgoli M, Martínez E, et al. 
Inferring tumour purity and stromal and immune cell admixture from expression data. Nat Commun. 2013;4(1):2612. doi: 10.1038/ncomms3612.
</cite> [<a href="https://doi.org/10.1038/ncomms3612" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3826632/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24113773/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Commun&amp;title=Inferring%20tumour%20purity%20and%20stromal%20and%20immune%20cell%20admixture%20from%20expression%20data&amp;volume=4&amp;issue=1&amp;publication_year=2013&amp;pages=2612&amp;pmid=24113773&amp;doi=10.1038/ncomms3612&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CIT0091">
<span class="label">91.</span><cite>Seashore-Ludlow B, Rees MG, Cheah JH, et al. 
Harnessing connectivity in a large-scale small-molecule sensitivity dataset. Cancer Discov. 2015;5(11):1210–1223. doi: 10.1158/2159-8290.CD-15-0235.
</cite> [<a href="https://doi.org/10.1158/2159-8290.CD-15-0235" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4631646/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26482930/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cancer%20Discov&amp;title=Harnessing%20connectivity%20in%20a%20large-scale%20small-molecule%20sensitivity%20dataset&amp;volume=5&amp;issue=11&amp;publication_year=2015&amp;pages=1210-1223&amp;pmid=26482930&amp;doi=10.1158/2159-8290.CD-15-0235&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CIT0092">
<span class="label">92.</span><cite>Rees MG, Seashore-Ludlow B, Cheah JH, et al. 
Correlating chemical sensitivity and basal gene expression reveals mechanism of action. Nat Chem Biol. 2016;12(2):109–116. doi: 10.1038/nchembio.1986.
</cite> [<a href="https://doi.org/10.1038/nchembio.1986" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4718762/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26656090/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Chem%20Biol&amp;title=Correlating%20chemical%20sensitivity%20and%20basal%20gene%20expression%20reveals%20mechanism%20of%20action&amp;volume=12&amp;issue=2&amp;publication_year=2016&amp;pages=109-116&amp;pmid=26656090&amp;doi=10.1038/nchembio.1986&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CIT0093">
<span class="label">93.</span><cite>Basu A, Bodycombe NE, Cheah JH, et al. 
An interactive resource to identify cancer genetic and lineage dependencies targeted by small molecules. Cell. 2013;154(5):1151–1161. doi: 10.1016/j.cell.2013.08.003.
</cite> [<a href="https://doi.org/10.1016/j.cell.2013.08.003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3954635/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23993102/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=An%20interactive%20resource%20to%20identify%20cancer%20genetic%20and%20lineage%20dependencies%20targeted%20by%20small%20molecules&amp;volume=154&amp;issue=5&amp;publication_year=2013&amp;pages=1151-1161&amp;pmid=23993102&amp;doi=10.1016/j.cell.2013.08.003&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CIT0094">
<span class="label">94.</span><cite>Maeser D, Gruener RF, Huang RS.. 
oncoPredict: an R package for predicting in vivo or cancer patient drug response and biomarkers from cell line screening data. Brief Bioinform. 2021;22(6):bbab260.
</cite> [<a href="https://doi.org/10.1093/bib/bbab260" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8574972/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34260682/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Brief%20Bioinform&amp;title=oncoPredict:%20an%20R%20package%20for%20predicting%20in%20vivo%20or%20cancer%20patient%20drug%20response%20and%20biomarkers%20from%20cell%20line%20screening%20data&amp;volume=22&amp;issue=6&amp;publication_year=2021&amp;pages=bbab260&amp;pmid=34260682&amp;doi=10.1093/bib/bbab260&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CIT0095">
<span class="label">95.</span><cite>Yang C, et al. 
A survey of optimal strategy for signature-based drug repositioning and an application to liver cancer. Elife. 2022;11:e71880.
</cite> [<a href="https://doi.org/10.7554/eLife.71880" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8893721/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35191375/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Elife&amp;title=A%20survey%20of%20optimal%20strategy%20for%20signature-based%20drug%20repositioning%20and%20an%20application%20to%20liver%20cancer&amp;volume=11&amp;publication_year=2022&amp;pages=e71880&amp;pmid=35191375&amp;doi=10.7554/eLife.71880&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CIT0096">
<span class="label">96.</span><cite>Stuart T, Butler A, Hoffman P, et al. 
Comprehensive integration of single-cell data. Cell. 2019;177(7):1888.e21–1902.e21. doi: 10.1016/j.cell.2019.05.031.
</cite> [<a href="https://doi.org/10.1016/j.cell.2019.05.031" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6687398/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31178118/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Comprehensive%20integration%20of%20single-cell%20data&amp;volume=177&amp;issue=7&amp;publication_year=2019&amp;pages=1888.e21-1902.e21&amp;pmid=31178118&amp;doi=10.1016/j.cell.2019.05.031&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CIT0097">
<span class="label">97.</span><cite>Xi NM, Li JJ.. 
Benchmarking computational doublet-detection methods for single-cell RNA sequencing data. Cell Syst. 2021;12(2):176–194 e6. doi: 10.1016/j.cels.2020.11.008.
</cite> [<a href="https://doi.org/10.1016/j.cels.2020.11.008" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7897250/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33338399/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Syst&amp;title=Benchmarking%20computational%20doublet-detection%20methods%20for%20single-cell%20RNA%20sequencing%20data&amp;volume=12&amp;issue=2&amp;publication_year=2021&amp;pages=176-194%20e6&amp;pmid=33338399&amp;doi=10.1016/j.cels.2020.11.008&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CIT0098">
<span class="label">98.</span><cite>Korsunsky I, Millard N, Fan J, et al. 
Fast, sensitive and accurate integration of single-cell data with Harmony. Nat Methods. 2019;16(12):1289–1296. doi: 10.1038/s41592-019-0619-0.
</cite> [<a href="https://doi.org/10.1038/s41592-019-0619-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6884693/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31740819/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Methods&amp;title=Fast,%20sensitive%20and%20accurate%20integration%20of%20single-cell%20data%20with%20Harmony&amp;volume=16&amp;issue=12&amp;publication_year=2019&amp;pages=1289-1296&amp;pmid=31740819&amp;doi=10.1038/s41592-019-0619-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CIT0099">
<span class="label">99.</span><cite>Trapnell C, Cacchiarelli D, Grimsby J, et al. 
The dynamics and regulators of cell fate decisions are revealed by pseudotemporal ordering of single cells. Nat Biotechnol. 2014;32(4):381–386. doi: 10.1038/nbt.2859.
</cite> [<a href="https://doi.org/10.1038/nbt.2859" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4122333/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24658644/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Biotechnol&amp;title=The%20dynamics%20and%20regulators%20of%20cell%20fate%20decisions%20are%20revealed%20by%20pseudotemporal%20ordering%20of%20single%20cells&amp;volume=32&amp;issue=4&amp;publication_year=2014&amp;pages=381-386&amp;pmid=24658644&amp;doi=10.1038/nbt.2859&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CIT0100">
<span class="label">100.</span><cite>Qiu X, Mao Q, Tang Y, et al. 
Reversed graph embedding resolves complex single-cell trajectories. Nat Methods. 2017;14(10):979–982. doi: 10.1038/nmeth.4402.
</cite> [<a href="https://doi.org/10.1038/nmeth.4402" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5764547/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28825705/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Methods&amp;title=Reversed%20graph%20embedding%20resolves%20complex%20single-cell%20trajectories&amp;volume=14&amp;issue=10&amp;publication_year=2017&amp;pages=979-982&amp;pmid=28825705&amp;doi=10.1038/nmeth.4402&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CIT0101">
<span class="label">101.</span><cite>Qiu X, Hill A, Packer J, et al. 
Single-cell mRNA quantification and differential analysis with census. Nat Methods. 2017;14(3):309–315. doi: 10.1038/nmeth.4150.
</cite> [<a href="https://doi.org/10.1038/nmeth.4150" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5330805/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28114287/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Methods&amp;title=Single-cell%20mRNA%20quantification%20and%20differential%20analysis%20with%20census&amp;volume=14&amp;issue=3&amp;publication_year=2017&amp;pages=309-315&amp;pmid=28114287&amp;doi=10.1038/nmeth.4150&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CIT0102">
<span class="label">102.</span><cite>Browaeys R, Saelens W, Saeys Y.. 
NicheNet: modeling intercellular communication by linking ligands to target genes. Nat Methods. 2020;17(2):159–162. doi: 10.1038/s41592-019-0667-5.
</cite> [<a href="https://doi.org/10.1038/s41592-019-0667-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31819264/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Methods&amp;title=NicheNet:%20modeling%20intercellular%20communication%20by%20linking%20ligands%20to%20target%20genes&amp;volume=17&amp;issue=2&amp;publication_year=2020&amp;pages=159-162&amp;pmid=31819264&amp;doi=10.1038/s41592-019-0667-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CIT0103">
<span class="label">103.</span><cite>Chow A, Perica K, Klebanoff CA, et al. 
Clinical implications of T cell exhaustion for cancer immunotherapy. Nat Rev Clin Oncol. 2022;19(12):775–790. doi: 10.1038/s41571-022-00689-z.
</cite> [<a href="https://doi.org/10.1038/s41571-022-00689-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10984554/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36216928/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Clin%20Oncol&amp;title=Clinical%20implications%20of%20T%20cell%20exhaustion%20for%20cancer%20immunotherapy&amp;volume=19&amp;issue=12&amp;publication_year=2022&amp;pages=775-790&amp;pmid=36216928&amp;doi=10.1038/s41571-022-00689-z&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CIT0104">
<span class="label">104.</span><cite>Kumar N, Mishra B, Athar M, et al. 
Inference of gene regulatory network from single-cell transcriptomic data using pySCENIC. Methods Mol Biol. 2021;2328:171–182. doi: 10.1007/978-1-0716-1534-8_10.
</cite> [<a href="https://doi.org/10.1007/978-1-0716-1534-8_10" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34251625/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Methods%20Mol%20Biol&amp;title=Inference%20of%20gene%20regulatory%20network%20from%20single-cell%20transcriptomic%20data%20using%20pySCENIC&amp;volume=2328&amp;publication_year=2021&amp;pages=171-182&amp;pmid=34251625&amp;doi=10.1007/978-1-0716-1534-8_10&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CIT0105">
<span class="label">105.</span><cite>Schindelin J, Arganda-Carreras I, Frise E, et al. 
Fiji: an open-source platform for biological-image analysis. Nat Methods. 2012;9(7):676–682. doi: 10.1038/nmeth.2019.
</cite> [<a href="https://doi.org/10.1038/nmeth.2019" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3855844/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22743772/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Methods&amp;title=Fiji:%20an%20open-source%20platform%20for%20biological-image%20analysis&amp;volume=9&amp;issue=7&amp;publication_year=2012&amp;pages=676-682&amp;pmid=22743772&amp;doi=10.1038/nmeth.2019&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="caption p"><span>Supplemental Material</span></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12366518/bin/IANN_A_2548042_SM0550.docx" data-ga-action="click_feat_suppl" class="usa-link">IANN_A_2548042_SM0550.docx</a><sup> (14.4KB, docx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="caption p"><span>Supplementary Table S2.docx</span></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12366518/bin/IANN_A_2548042_SM4305.docx" data-ga-action="click_feat_suppl" class="usa-link">IANN_A_2548042_SM4305.docx</a><sup> (12.9KB, docx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="caption p"><span>Supplementary Figure 2.tif</span></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12366518/bin/IANN_A_2548042_SM4304.tif" data-ga-action="click_feat_suppl" class="usa-link">IANN_A_2548042_SM4304.tif</a><sup> (1.1MB, tif) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material4_reqid_"><div class="caption p"><span>Supplementary Table S1.xlsx</span></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12366518/bin/IANN_A_2548042_SM4303.xlsx" data-ga-action="click_feat_suppl" class="usa-link">IANN_A_2548042_SM4303.xlsx</a><sup> (37.2KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material5_reqid_"><div class="caption p"><span>Supplementary Figure 4.tif</span></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12366518/bin/IANN_A_2548042_SM4302.tif" data-ga-action="click_feat_suppl" class="usa-link">IANN_A_2548042_SM4302.tif</a><sup> (1.6MB, tif) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material6_reqid_"><div class="caption p"><span>Supplementary Figure 6.tif</span></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12366518/bin/IANN_A_2548042_SM4301.tif" data-ga-action="click_feat_suppl" class="usa-link">IANN_A_2548042_SM4301.tif</a><sup> (3.5MB, tif) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material7_reqid_"><div class="caption p"><span>Supplementary Figure 1.tif</span></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12366518/bin/IANN_A_2548042_SM4300.tif" data-ga-action="click_feat_suppl" class="usa-link">IANN_A_2548042_SM4300.tif</a><sup> (1.7MB, tif) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material8_reqid_"><div class="caption p"><span>Supplementary Figure 5.tif</span></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12366518/bin/IANN_A_2548042_SM4299.tif" data-ga-action="click_feat_suppl" class="usa-link">IANN_A_2548042_SM4299.tif</a><sup> (11.2MB, tif) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material9_reqid_"><div class="caption p"><span>Supplementary Figure 7.tif</span></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12366518/bin/IANN_A_2548042_SM4298.tif" data-ga-action="click_feat_suppl" class="usa-link">IANN_A_2548042_SM4298.tif</a><sup> (6.8MB, tif) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material10_reqid_"><div class="caption p"><span>Supplementary Figure 3.tif</span></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12366518/bin/IANN_A_2548042_SM4296.tif" data-ga-action="click_feat_suppl" class="usa-link">IANN_A_2548042_SM4296.tif</a><sup> (2.1MB, tif) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>All RNA-seq data analyzed in our study are available in UCSC Xena (<a href="http://xena.ucsc.edu/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://xena.ucsc.edu/</a>), the Gene Expression Omnibus (<a href="https://www.ncbi.nlm.nih.gov/geo/" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/geo/</a>) and the ArrayExpress database (<a href="https://www.ebi.ac.uk/biostudies/arrayexpress" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ebi.ac.uk/biostudies/arrayexpress</a>). Other data and code are available upon reasonable request from the corresponding author, Bin Fu or Jianhan Fu.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Annals of Medicine are provided here courtesy of <strong>Taylor & Francis</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1080/07853890.2025.2548042"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/IANN_57_2548042.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (9.2 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12366518/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12366518/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12366518%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366518/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12366518/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12366518/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40830065/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12366518/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40830065/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12366518/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12366518/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="tir84jDMz6wwslpXOVnPKCaYDB2PqJvDtQXZu8tdlApfcHZW2xTkh6K3U2wI12Ix">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
        <script type="module" crossorigin="" src="/static/assets/math-574fdcc6.js"></script>
    
    

    </body>
</html>
