
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Incorporating information of causal variants in genomic prediction using GBLUP or machine learning models in a simulated livestock population - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="D2E4B50F8AF3A54305B50F003432622C.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="jabsci">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12362903/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Journal of Animal Science and Biotechnology">
<meta name="citation_title" content="Incorporating information of causal variants in genomic prediction using GBLUP or machine learning models in a simulated livestock population">
<meta name="citation_author" content="Jifan Yang">
<meta name="citation_author_institution" content="Animal Breeding and Genomics, Wageningen University &amp; Research, Wageningen, 6700 AH The Netherlands">
<meta name="citation_author" content="Mario P L Calus">
<meta name="citation_author_institution" content="Animal Breeding and Genomics, Wageningen University &amp; Research, Wageningen, 6700 AH The Netherlands">
<meta name="citation_author" content="Yvonne C J Wientjes">
<meta name="citation_author_institution" content="Animal Breeding and Genomics, Wageningen University &amp; Research, Wageningen, 6700 AH The Netherlands">
<meta name="citation_author" content="Theo H E Meuwissen">
<meta name="citation_author_institution" content="Faculty of Life Sciences, Norwegian University of Life Sciences, Ås, 1432 Norway">
<meta name="citation_author" content="Pascal Duenk">
<meta name="citation_author_institution" content="Animal Breeding and Genomics, Wageningen University &amp; Research, Wageningen, 6700 AH The Netherlands">
<meta name="citation_publication_date" content="2025 Aug 19">
<meta name="citation_volume" content="16">
<meta name="citation_firstpage" content="118">
<meta name="citation_doi" content="10.1186/s40104-025-01250-5">
<meta name="citation_pmid" content="40826369">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12362903/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12362903/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12362903/pdf/40104_2025_Article_1250.pdf">
<meta name="description" content="Genomic prediction has revolutionized animal breeding, with GBLUP being the most widely used prediction model. In theory, the accuracy of genomic prediction could be improved by incorporating information from QTL. This strategy could be especially ...">
<meta name="og:title" content="Incorporating information of causal variants in genomic prediction using GBLUP or machine learning models in a simulated livestock population">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Genomic prediction has revolutionized animal breeding, with GBLUP being the most widely used prediction model. In theory, the accuracy of genomic prediction could be improved by incorporating information from QTL. This strategy could be especially ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12362903/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12362903">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1186/s40104-025-01250-5"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/40104_2025_Article_1250.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12362903%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12362903/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12362903/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12362903/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-jabsci.png" alt="Journal of Animal Science and Biotechnology logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Journal of Animal Science and Biotechnology" title="Link to Journal of Animal Science and Biotechnology" shape="default" href="https://jasbsci.biomedcentral.com/" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">J Anim Sci Biotechnol</button></div>. 2025 Aug 19;16:118. doi: <a href="https://doi.org/10.1186/s40104-025-01250-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1186/s40104-025-01250-5</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22J%20Anim%20Sci%20Biotechnol%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22J%20Anim%20Sci%20Biotechnol%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22J%20Anim%20Sci%20Biotechnol%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22J%20Anim%20Sci%20Biotechnol%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Incorporating information of causal variants in genomic prediction using GBLUP or machine learning models in a simulated livestock population</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yang%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Jifan Yang</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Jifan Yang</span></h3>
<div class="p">
<sup>1</sup>Animal Breeding and Genomics, Wageningen University &amp; Research, Wageningen, 6700 AH The Netherlands </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yang%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Jifan Yang</span></a>
</div>
</div>
<sup>1,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Calus%20MPL%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Mario P L Calus</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Mario P L Calus</span></h3>
<div class="p">
<sup>1</sup>Animal Breeding and Genomics, Wageningen University &amp; Research, Wageningen, 6700 AH The Netherlands </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Calus%20MPL%22%5BAuthor%5D" class="usa-link"><span class="name western">Mario P L Calus</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wientjes%20YCJ%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Yvonne C J Wientjes</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Yvonne C J Wientjes</span></h3>
<div class="p">
<sup>1</sup>Animal Breeding and Genomics, Wageningen University &amp; Research, Wageningen, 6700 AH The Netherlands </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wientjes%20YCJ%22%5BAuthor%5D" class="usa-link"><span class="name western">Yvonne C J Wientjes</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Meuwissen%20THE%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Theo H E Meuwissen</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Theo H E Meuwissen</span></h3>
<div class="p">
<sup>2</sup>Faculty of Life Sciences, Norwegian University of Life Sciences, Ås, 1432 Norway </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Meuwissen%20THE%22%5BAuthor%5D" class="usa-link"><span class="name western">Theo H E Meuwissen</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Duenk%20P%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Pascal Duenk</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Pascal Duenk</span></h3>
<div class="p">
<sup>1</sup>Animal Breeding and Genomics, Wageningen University &amp; Research, Wageningen, 6700 AH The Netherlands </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Duenk%20P%22%5BAuthor%5D" class="usa-link"><span class="name western">Pascal Duenk</span></a>
</div>
</div>
<sup>1</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>Animal Breeding and Genomics, Wageningen University &amp; Research, Wageningen, 6700 AH The Netherlands </div>
<div id="Aff2">
<sup>2</sup>Faculty of Life Sciences, Norwegian University of Life Sciences, Ås, 1432 Norway </div>
<div class="author-notes p"><div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div></div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 Apr 3; Accepted 2025 Jun 29; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by/4.0/</a>. The Creative Commons Public Domain Dedication waiver (<a href="https://creativecommons.org/publicdomain/zero/1.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/publicdomain/zero/1.0/</a>) applies to the data made available in this article, unless otherwise stated in a credit line to the data.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12362903  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40826369/" class="usa-link">40826369</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<section id="sec1"><h3 class="pmc_sec_title">Background</h3>
<p id="Par1">Genomic prediction has revolutionized animal breeding, with GBLUP being the most widely used prediction model. In theory, the accuracy of genomic prediction could be improved by incorporating information from QTL. This strategy could be especially beneficial for machine learning models that are able to distinguish informative from uninformative features. The objective of this study was to assess the benefit of incorporating QTL genotypes in GBLUP and machine learning models. This study simulated a selected livestock population where QTL and their effects were known. We used four genomic prediction models, GBLUP, (weighted) 2GBLUP, random forest (RF), and support vector regression (SVR) to predict breeding values of young animals, and considered different scenarios that varied in the proportion of genetic variance explained by the included QTL.</p></section><section id="sec2"><h3 class="pmc_sec_title">Results</h3>
<p id="Par2">2GBLUP resulted in the highest accuracy. Its accuracy increased when the included QTL explained up to 80% of the genetic variance, after which the accuracy dropped. With a weighted 2GBLUP model, the accuracy always increased when more QTL were included. Prediction accuracy of GBLUP was consistently higher than SVR, and the accuracy for both models slightly increased with more QTL information included. The RF model resulted in the lowest prediction accuracy, and did not improve by including QTL information.</p></section><section id="sec3"><h3 class="pmc_sec_title">Conclusions</h3>
<p id="Par3">Our results show that incorporating QTL information in GBLUP and SVR can improve prediction accuracy, but the extent of improvement varies across models. RF had a much lower prediction accuracy than the other models and did not show improvements when QTL information was added. Two possible reasons for this result are that the data structure in our data does not allow RF to fully realize its potential and that RF is not designed well for this particular prediction problem. Our study highlighted the importance of selecting appropriate models for genomic prediction and underscored the potential limitations of machine learning models when applied to genomic prediction in livestock.</p></section><section id="sec4"><h3 class="pmc_sec_title">Supplementary Information</h3>
<p>The online version contains supplementary material available at 10.1186/s40104-025-01250-5.</p></section><section id="kwd-group1" lang="en" class="kwd-group"><p><strong>Keywords:</strong> GBLUP, Genomic prediction, Machine learning, QTL, Random forest, Support vector regression</p></section></section><section id="Sec1"><h2 class="pmc_sec_title">Background</h2>
<p id="Par16">The extensive use of genomic prediction (GP) has significantly increased the genetic improvement in the animal breeding industry [<a href="#CR1" class="usa-link" aria-describedby="CR1">1</a>, <a href="#CR2" class="usa-link" aria-describedby="CR2">2</a>]. In GP, causal variants, i.e., quantitative trait loci (QTL) that influence the trait of interest, are assumed to be in linkage disequilibrium (LD) with at least one single nucleotide polymorphism (SNP) marker. When enough markers are used, the markers are expected to explain together the genetic variance for the trait and, therefore, can be used to predict breeding values for the selection candidates without knowing the actual causal variants [<a href="#CR3" class="usa-link" aria-describedby="CR3">3</a>].</p>
<p id="Par17">Common models used to predict genomic breeding values are GBLUP and ssGBLUP which assume that all markers explain the same amount of genetic variance [<a href="#CR2" class="usa-link" aria-describedby="CR2">2</a>, <a href="#CR4" class="usa-link" aria-describedby="CR4">4</a>, <a href="#CR5" class="usa-link" aria-describedby="CR5">5</a>]. This assumption can be relaxed by using a method named weighted GBLUP which can put more weight on markers that are known to explain a larger proportion of the genetic variance [<a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>]. The assumption of equal variance for all markers can also be relaxed by using Bayesian models. These models require parameters that determine the distribution of marker effects and these parameters can be assumed known or can be estimated from the data [<a href="#CR3" class="usa-link" aria-describedby="CR3">3</a>]. For example, BayesCπ assumes that only a small proportion of markers have an effect and estimates the proportion of markers with no effect (i.e., <em>π</em>) in the model [<a href="#CR7" class="usa-link" aria-describedby="CR7">7</a>].</p>
<p id="Par18">Another option for GP is using machine learning (ML) methods, which do not make assumptions about the distribution of marker effects and the underlying parameters. Examples of ML models applied to genomic prediction are Bayesian neural network [<a href="#CR8" class="usa-link" aria-describedby="CR8">8</a>, <a href="#CR9" class="usa-link" aria-describedby="CR9">9</a>], reproducing kernel Hilbert spaces regression [<a href="#CR10" class="usa-link" aria-describedby="CR10">10</a>, <a href="#CR11" class="usa-link" aria-describedby="CR11">11</a>], support vector regression (SVR) with non-linear kernels [<a href="#CR12" class="usa-link" aria-describedby="CR12">12</a>], and decision tree based algorithms such as random forest (RF) [<a href="#CR13" class="usa-link" aria-describedby="CR13">13</a>, <a href="#CR14" class="usa-link" aria-describedby="CR14">14</a>]. In some of these studies, machine learning models yielded higher prediction accuracy than GBLUP [<a href="#CR8" class="usa-link" aria-describedby="CR8">8</a>, <a href="#CR15" class="usa-link" aria-describedby="CR15">15</a>, <a href="#CR16" class="usa-link" aria-describedby="CR16">16</a>], while others did not find benefits of ML over GBLUP [<a href="#CR17" class="usa-link" aria-describedby="CR17">17</a>–<a href="#CR20" class="usa-link" aria-describedby="CR20">20</a>].</p>
<p id="Par19">Studies aiming to improve accuracy by increasing the number of markers have generally failed. In cattle, there was generally no improvement in accuracy when using more than 100k markers compared to 50k markers, which may be because only relatively few of all whole genome sequence (WGS) variants add useful additional information, while all other added variants may only dilute the signals captured by the marker loci [<a href="#CR21" class="usa-link" aria-describedby="CR21">21</a>–<a href="#CR23" class="usa-link" aria-describedby="CR23">23</a>]. To reduce the impact of added noise when using more markers, feature selection was introduced in some Bayesian models. So far, applications of such models to imputed WGS data did not yield improvements in accuracy compared to using a high density marker panel, which may be explained by the structure of the training dataset [<a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>, <a href="#CR24" class="usa-link" aria-describedby="CR24">24</a>]. Alternatively, ML models can do feature selection by assigning different weights to genetic variants. Two studies reported that a smaller number of markers resulted in a higher prediction accuracy [<a href="#CR19" class="usa-link" aria-describedby="CR19">19</a>, <a href="#CR20" class="usa-link" aria-describedby="CR20">20</a>]. Because the inclusion of an excessive number of markers can negatively impact ML performance due to the risk of overfitting. However, as far as we know, it is not yet known whether the inclusion of informative genetic variants such as information on causal loci in ML models can improve the prediction accuracy.</p>
<p id="Par20">As an alternative to feature selection, some studies have attempted to take a two-step approach. First, a pre-selection of markers was made based on results from genome-wide association studies (GWAS). Second, those pre-selected markers were included as a separate genetic effect in a GBLUP model. Most of these studies have reported that this strategy improved the prediction accuracy [<a href="#CR25" class="usa-link" aria-describedby="CR25">25</a>–<a href="#CR30" class="usa-link" aria-describedby="CR30">30</a>]. The reason for this improvement is that the separation of pre-selected markers prevents the dilution of the genetic signals among the markers that are in weak LD with the causal variants. These studies with the two step approach only focussed on a limited number of top-SNPs. However, due to recent developments in quantitative genetics and molecular genomics, combined with advances in cost-effective sequencing technologies, we can expect that more and more causal loci will be identified [<a href="#CR31" class="usa-link" aria-describedby="CR31">31</a>]. However, it is still unclear whether it is optimal to include all preselected markers, or only the ones that explain most of the genetic variance.</p>
<p id="Par21">Machine learning models differ from GBLUP because they are not specifically designed to predict breeding values. Instead, ML models predict the provided input, such as phenotypes. This fundamental difference might lead to ML models being less accurate than GBLUP for predicting breeding values. However, both approaches can simply be viewed as prediction models that use phenotypes as input, and genotypes as features. Furthermore, some simulation studies have shown that ML models can outperform GBLUP in terms of the correlation between predicted values and breeding value [<a href="#CR8" class="usa-link" aria-describedby="CR8">8</a>, <a href="#CR32" class="usa-link" aria-describedby="CR32">32</a>]. It is therefore interesting to compare the accuracy between ML models and GBLUP, eventhough they aim to predict different things.</p>
<p id="Par22">The objective of this study was to assess the benefit of incorporating QTL information in GBLUP and machine learning models for genomic prediction accuracy, and to find the proportion of genetic variance explained by QTL included in the model at which the highest prediction accuracy is achieved. Our approach was based on simulations of a livestock population in which all the QTL genotypes and effects were known. We used GBLUP, random forest and support vector regression for analysis. The two machine learning methods are representative of two categories of methods, namely ensemble methods (RF) and kernel methods (SVR). In addition, we investigated the reason why there is a difference in prediction accuracy between different models.</p></section><section id="Sec2"><h2 class="pmc_sec_title">Methods</h2>
<section id="Sec3"><h3 class="pmc_sec_title">Simulation</h3>
<p id="Par23">The data was simulated with QMSim [<a href="#CR33" class="usa-link" aria-describedby="CR33">33</a>], using the parameter file in Additional file <a href="#MOESM1" class="usa-link">1</a>. The simulation process tried to mimic a historic dairy cattle population. It is good to note that our aim was not to completely resemble a dairy cattle population, but to simulate a population under selection with a comparable genome structure as livestock species to test the different genomic prediction models. In the first generation of the historic population, there were 2,800 individuals with equal number of both sexes. The population size gradually decreased to 2,000 in generation 2,250, and further gradually decreased to 200 in generation 2,500 to generate linkage disequilibrium. After that, the population size linearly increased to 10,000 in generation 2,505. After the historic population, 20 generations (numbered as 1–20) of 5,000 animals with an equal sex ratio and litter size of 1 were simulated, using 20 males and 5,000 females as parents in each generation. In each new generation, all male parents and half of the female parents were replaced. For both males and females, new parents were selected using pedigree-based estimated breeding values for a polygenic selection trait with an accuracy of 0.8.</p>
<p id="Par24">We simulated 29 pairs of autosomes that each had a size of 100 cM. On each autosome, 350 quantitative trait loci (QTL) and 2,000 SNP markers were randomly placed. Together, the QTL explained all the genetic variance (<em>h</em><sup>2</sup> = 0.25) of the trait under selection, and the QTL effects were sampled from a gamma distribution with a shape parameter of 0.42. True breeding values (TBV) were calculated by multiplying the genotypes (coded as allele counts: 0, 1, 2) with the effects of the counted QTL allele, and phenotypes were calculated as the sum of TBV and a random environmental effect.</p>
<p id="Par25">The final dataset contained the genotypes of animals from generation 0 to 20, and phenotypes of the selected trait from generation 0 to 15, which were available for both sexes. For generation 11–15, 2,500 animals in each generation were randomly chosen to be included in the training dataset. For generation 16, 500 animals were randomly chosen to be included in the test set. Therefore, there are 12,500 animals in the training dataset and 500 animals in the test dataset. We simulated 20 replicates in total.</p>
<p id="Par26">After the simulation, the distribution of allele frequencies of the segregating SNPs and QTL were U-shaped (Fig. <a href="#Fig1" class="usa-link">1</a>). The average total number of segregating SNPs among 20 replicates was 55,043.2 ± 82.9 and there were on average 8,169.8 ± 44.0 segregating QTL in generation 0. We sorted the QTL based on the amount of variance that they explained. So, with increasing number of QTL, the genetic variance explained by QTL increased non-linearly (Fig. <a href="#Fig2" class="usa-link">2</a>). On average, the 8.6 ± 1.4 most important QTL together explained 5% of the genetic variance, and the 3,789.5 ± 78.6 most important QTL explained 99% of the genetic variance, while the 1,392.0 ± 48.1 least important QTL explained only 1% of the genetic variance. 2,988.4 ± 116.9 QTL cumulatively explained less than 0.0005% of the genetic variance, and we excluded these QTL.</p>
<figure class="fig xbox font-sm" id="Fig1"><h4 class="obj_head">Fig. 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12362903_40104_2025_1250_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/cdd4/12362903/0eb03083e2ae/40104_2025_1250_Fig1_HTML.jpg" loading="lazy" id="MO1" height="310" width="778" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Allele frequency distribution of segregating SNPs (<strong>a</strong>) and QTL (<strong>b</strong>) in one replicate. Note that SNPs/QTL that were fixed for a single allele (i.e., with allele frequency of 0 or 1) are not represented in this graph</p></figcaption></figure><figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/cdd4/12362903/333241133882/40104_2025_1250_Fig2_HTML.jpg" loading="lazy" id="MO2" height="591" width="786" alt="Fig. 2"></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>The cumulative proportion of genetic variance explained by QTL versus the number of QTL averaged across replicates. The QTL are ordered from QTL that explain the most genetic variance to QTL that explain the least genetic variance</p></figcaption></figure></section><section id="Sec4"><h3 class="pmc_sec_title">Estimating breeding values</h3>
<section id="Sec5"><h4 class="pmc_sec_title">GBLUP</h4>
<p id="Par27">The first model used to estimate breeding values was GBLUP. With the GBLUP model, we included one random genomic effect without separating QTL from SNP markers. The used model was</p>
<table class="disp-formula p" id="Equa"><tr><td class="formula"><math id="d33e453" display="block"><mrow><mrow><mi mathvariant="bold-italic">y</mi></mrow><mo>=</mo><mn mathvariant="bold">1</mn><mi>μ</mi><mo>+</mo><mrow><mi mathvariant="bold-italic">Z</mi></mrow><mrow><mi mathvariant="bold-italic">u</mi></mrow><mo>+</mo><mrow><mi mathvariant="bold-italic">e</mi></mrow><mo>,</mo></mrow></math></td></tr></table>
<p>where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq1"><math id="d33e481"><mrow><mi mathvariant="bold-italic">y</mi></mrow></math></span> is the vector of phenotypes, <em>µ</em> is the overall mean, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq2"><math id="d33e495"><mn mathvariant="bold">1</mn></math></span> is a vector of ones, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq3"><math id="d33e505"><mrow><mi mathvariant="bold-italic">u</mi></mrow></math></span> is the vector of genomic breeding values of SNP and QTL following a normal distribution <em>N</em>(<strong>0</strong>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq4"><math id="d33e523"><mrow><mrow><mi mathvariant="bold-italic">G</mi></mrow><msubsup><mi>σ</mi><mrow><mi>u</mi></mrow><mn>2</mn></msubsup></mrow></math></span>), <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq5"><math id="d33e540"><mrow><mi mathvariant="bold-italic">Z</mi></mrow></math></span> is an incidence matrix that links <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq6"><math id="d33e551"><mrow><mi mathvariant="bold-italic">u</mi></mrow></math></span> to <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq7"><math id="d33e562"><mrow><mi mathvariant="bold-italic">y</mi></mrow></math></span>, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq8"><math id="d33e574"><mrow><mi mathvariant="bold-italic">e</mi></mrow></math></span> is the vector of random errors, following a normal distribution of <em>N</em>(<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq9"><math id="d33e588"><mrow><mn>0</mn></mrow></math></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq10"><math id="d33e598"><mrow><mrow><mi mathvariant="bold-italic">I</mi></mrow><msubsup><mi>σ</mi><mrow><mi>e</mi></mrow><mn>2</mn></msubsup></mrow></math></span>), where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq11"><math id="d33e615"><msubsup><mi>σ</mi><mrow><mi>e</mi></mrow><mn>2</mn></msubsup></math></span> is the residual variance. <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq12"><math id="d33e628"><mrow><mi mathvariant="bold-italic">G</mi></mrow></math></span> is the genomic relationship matrix which was constructed using genotypes of all SNP and QTL markers, and method (1) of VanRaden [<a href="#CR34" class="usa-link" aria-describedby="CR34">34</a>]:</p>
<table class="disp-formula p" id="Equb"><tr><td class="formula"><math id="d33e643" display="block"><mrow><mrow><mi mathvariant="bold-italic">G</mi></mrow><mo>=</mo><mfrac><msup><mrow><mi mathvariant="bold-italic">WW</mi></mrow><mo>′</mo></msup><mrow><mn>2</mn><msub><mo>∑</mo><mi>j</mi></msub><msub><mi>p</mi><mi>j</mi></msub><mrow><mo stretchy="false">(</mo><mn>1</mn><mo>-</mo><msub><mi>p</mi><mi>j</mi></msub><mo stretchy="false">)</mo></mrow></mrow></mfrac><mrow></mrow><mo>,</mo></mrow></math></td></tr></table>
<p>where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq13"><math id="d33e682"><mrow><mi mathvariant="bold-italic">W</mi></mrow></math></span> includes the centered allele counts of the genotypes, with elements <em>w</em><sub><em>ij</em></sub> = <em>x</em><sub><em>ij</em></sub><em> − 2p</em><sub><em>j</em></sub>, where <em>x</em><sub><em>ij</em></sub> is an element of matrix <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq14"><math id="d33e717"><mrow><mi mathvariant="bold-italic">X</mi></mrow></math></span> containing the genotype for individual <em>i</em> at locus <em>j</em> coded as 0, 1, 2; and <em>p</em><sub><em>j</em></sub> is the frequency of the allele for which the homozygous genotype at locus<em> j</em> is coded as 2.</p></section><section id="Sec6"><h4 class="pmc_sec_title">2GBLUP</h4>
<p id="Par28">In the 2GBLUP model, we separated the pre-selected QTL and SNP markers by including two random genomic effects in the model allowing for different proportions of the genetic variance being explained by QTL and SNPs:</p>
<table class="disp-formula p" id="Equc"><tr><td class="formula"><math id="d33e748" display="block"><mrow><mrow><mi mathvariant="bold-italic">y</mi></mrow><mo>=</mo><mn mathvariant="bold">1</mn><mi>μ</mi><mo>+</mo><mrow><mi mathvariant="bold-italic">Z</mi></mrow><msub><mrow><mi mathvariant="bold-italic">u</mi></mrow><mn>1</mn></msub><mo>+</mo><mrow><mi mathvariant="bold-italic">Z</mi></mrow><msub><mrow><mi mathvariant="bold-italic">u</mi></mrow><mn>2</mn></msub><mo>+</mo><mrow><mi mathvariant="bold-italic">e</mi></mrow><mo>,</mo></mrow></math></td></tr></table>
<p>where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq15"><math id="d33e788"><mrow><mi mathvariant="bold-italic">y</mi></mrow></math></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq16"><math id="d33e799"><mn mathvariant="bold">1</mn></math></span>, <em>µ</em> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq17"><math id="d33e812"><mrow><mi mathvariant="bold-italic">e</mi></mrow></math></span> are the same as for GBLUP, while <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq18"><math id="d33e824"><msub><mrow><mi mathvariant="bold-italic">u</mi></mrow><mn>1</mn></msub></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq19"><math id="d33e837"><msub><mrow><mi mathvariant="bold-italic">u</mi></mrow><mn>2</mn></msub></math></span> are vectors of genomic breeding values based on QTL and SNP markers, respectively, which follow a normal distribution with <em>N</em>(<strong>0</strong>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq20"><math id="d33e856"><mrow><msub><mrow><mi mathvariant="bold-italic">G</mi></mrow><mn>1</mn></msub><msubsup><mi>σ</mi><mrow><msub><mi>u</mi><mn>1</mn></msub></mrow><mn>2</mn></msubsup></mrow></math></span>) and <em>N</em>(<strong>0</strong>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq21"><math id="d33e884"><mrow><msub><mrow><mi mathvariant="bold-italic">G</mi></mrow><mn>2</mn></msub><msubsup><mi>σ</mi><mrow><msub><mi>u</mi><mn>2</mn></msub></mrow><mn>2</mn></msubsup></mrow></math></span>), where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq22"><math id="d33e905"><msubsup><mi>σ</mi><mrow><msub><mi>u</mi><mn>1</mn></msub></mrow><mn>2</mn></msubsup></math></span> + <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq23"><math id="d33e920"><msubsup><mi>σ</mi><mrow><msub><mi>u</mi><mn>2</mn></msub></mrow><mn>2</mn></msubsup></math></span> equal the additive genetic variance (i.e., <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq24"><math id="d33e935"><msubsup><mi>σ</mi><mrow><mi>u</mi></mrow><mn>2</mn></msubsup></math></span> in GBLUP). <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq25"><math id="d33e948"><mrow><mi mathvariant="bold-italic">Z</mi></mrow></math></span> is an incidence matrix that links <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq26"><math id="d33e960"><msub><mrow><mi mathvariant="bold-italic">u</mi></mrow><mn>1</mn></msub></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq27"><math id="d33e973"><msub><mrow><mi mathvariant="bold-italic">u</mi></mrow><mn>2</mn></msub></math></span> to <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq28"><math id="d33e986"><mrow><mi mathvariant="bold-italic">y</mi></mrow></math></span>. <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq29"><math id="d33e997"><msub><mrow><mi mathvariant="bold-italic">G</mi></mrow><mn>2</mn></msub></math></span> was constructed using SNP genotypes, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq30"><math id="d33e1010"><msub><mrow><mi mathvariant="bold-italic">G</mi></mrow><mn>1</mn></msub></math></span> was constructed using QTL genotypes, using the same method as for <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq31"><math id="d33e1023"><mrow><mi mathvariant="bold-italic">G</mi></mrow></math></span>.</p>
<p id="Par29">The underlying assumption in the 2GBLUP model when constructing <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq32"><math id="d33e1036"><msub><mrow><mi mathvariant="bold-italic">G</mi></mrow><mn>1</mn></msub></math></span> was that all QTL explain the same amount of genetic variance, which is not true (Fig. <a href="#Fig2" class="usa-link">2</a>). To investigate the impact of this assumption, we included a model weighted 2GBLUP (w2GBLUP), where the <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq33"><math id="d33e1052"><msub><mrow><mi mathvariant="bold-italic">G</mi></mrow><mn>2</mn></msub></math></span> matrix was computed as before while QTL variances were used as weights to construct a weighted <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq34"><math id="d33e1065"><msub><mrow><mi mathvariant="bold-italic">G</mi></mrow><mn>1</mn></msub></math></span> [<a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>]:</p>
<table class="disp-formula p" id="Equd"><tr><td class="formula"><math id="d33e1082" display="block"><mrow><msub><mrow><mi mathvariant="bold-italic">G</mi></mrow><mn>1</mn></msub><mo>=</mo><mfrac><msup><mrow><mi mathvariant="bold-italic">WDW</mi></mrow><mo>′</mo></msup><mrow><mn>2</mn><msub><mo>∑</mo><mi>j</mi></msub><msub><mi>p</mi><mi>j</mi></msub><mrow><mo stretchy="false">(</mo><mn>1</mn><mo>-</mo><msub><mi>p</mi><mi>j</mi></msub><mo stretchy="false">)</mo></mrow></mrow></mfrac><mrow></mrow><mo>,</mo></mrow></math></td></tr></table>
<p>where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq35"><math id="d33e1123"><mrow><mi mathvariant="bold-italic">D</mi></mrow></math></span> is a diagonal matrix with weights, being the proportion of explained variance by the QTL. The weights for locus <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq36"><math id="d33e1134"><mi>j</mi></math></span> was calculated as <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq37"><math id="d33e1143"><mrow><mn>2</mn><msub><mi>p</mi><mi>j</mi></msub><mrow><mo stretchy="false">(</mo><mn>1</mn><mo>-</mo><msub><mi>p</mi><mi>j</mi></msub><mo stretchy="false">)</mo></mrow><msubsup><mi>α</mi><mrow><mi>j</mi></mrow><mn>2</mn></msubsup></mrow></math></span>. Note that replacing <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq38"><math id="d33e1171"><mrow><mi mathvariant="bold-italic">D</mi></mrow></math></span> by an identity matrix <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq39"><math id="d33e1182"><mrow><mi mathvariant="bold-italic">I</mi></mrow></math></span> yields the matrix used for unweighted 2GBLUP.</p></section><section id="Sec7"><h4 class="pmc_sec_title">Random forest</h4>
<p id="Par30">Random forest (RF) is an ML model that builds a collection of decision trees, whereby each decision tree is slightly different from the other trees [<a href="#CR35" class="usa-link" aria-describedby="CR35">35</a>]. RF uses the average of all trees using different sets of features to determine the predictions in the test set. The RF model [<a href="#CR35" class="usa-link" aria-describedby="CR35">35</a>] that was used can be written as:</p>
<table class="disp-formula p" id="Eque"><tr><td class="formula"><math id="d33e1203" display="block"><mrow><mrow><mi mathvariant="bold-italic">y</mi></mrow><mo>=</mo><munderover><mo>∑</mo><mrow><mi>t</mi><mo>=</mo><mn>1</mn></mrow><mi>T</mi></munderover><msub><mrow><mi mathvariant="bold-italic">h</mi></mrow><mi>t</mi></msub><mfenced close=")" open="("><mrow><mi mathvariant="bold-italic">y</mi></mrow><mo>;</mo><msub><mrow><mi mathvariant="bold-italic">W</mi></mrow><mrow><mi>m</mi><mo>,</mo><mi>t</mi></mrow></msub></mfenced><mrow></mrow><mo>,</mo></mrow></math></td></tr></table>
<p>in which <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq40"><math id="d33e1243"><mrow><mi mathvariant="bold-italic">y</mi></mrow></math></span> is the vector of phenotypes, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq41"><math id="d33e1254"><mrow><msub><mrow><mi mathvariant="bold-italic">h</mi></mrow><mi>t</mi></msub><mfenced close=")" open="("><mrow><mi mathvariant="bold-italic">y</mi></mrow><mo>;</mo><msub><mrow><mi mathvariant="bold-italic">W</mi></mrow><mrow><mi>m</mi><mo>,</mo><mi>t</mi></mrow></msub></mfenced></mrow></math></span> is a specific individual regression for tree <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq42"><math id="d33e1282"><mrow><mi>t</mi><mo>∈</mo><mo stretchy="false">(</mo><mn>1</mn><mo>,</mo><mi>T</mi><mo stretchy="false">)</mo></mrow></math></span>, where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq43"><math id="d33e1300"><mi>T</mi></math></span> is the number of decision trees in the forest, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq44"><math id="d33e1309"><mrow><mi mathvariant="bold-italic">W</mi></mrow></math></span> contain the features (SNPs or SNPs combined with QTL), and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq45"><math id="d33e1320"><mi>m</mi></math></span> is the set of selected features for <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq46"><math id="d33e1329"><mi>t</mi></math></span>. The prediction of each tree is obtained by passing down the features in the tree, and the corresponding estimated value at the terminal node is the predicted value. We tuned four hyperparameters, namely n_estimators (number of trees in the forest), max_depth (maximum depth of the tree), min_samples_leaf (minimum number of samples required to be at a leaf node) and max_features (number of features to consider when looking for the best split) (see Additional file <a href="#MOESM2" class="usa-link">2</a>: Table S1 and Additional file <a href="#MOESM3" class="usa-link">3</a>), and hyperparameters used in the analysis were shown in Table <a href="#Tab1" class="usa-link">1</a>.</p>
<section class="tw xbox font-sm" id="Tab1"><h5 class="obj_head">Table 1.</h5>
<div class="caption p"><p>Hyperparameters<sup>1</sup> used in the machine learning methods</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1"><strong>Method</strong></th>
<th align="left" colspan="1" rowspan="1"><strong>Hyperparameters used</strong></th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">RF</td>
<td align="left" colspan="1" rowspan="1">n_estimators = 1,500, max_features = 2,500</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">SVR</td>
<td align="left" colspan="1" rowspan="1">kernel = rbf</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="_fn_p24"><p><sup>1</sup>Other parameters that are not mentioned in the table were using default settings (see Additional file <a href="#MOESM2" class="usa-link">2</a>)</p></div></div></section></section><section id="Sec8"><h4 class="pmc_sec_title">Support vector regression</h4>
<p id="Par32">Support vector regression (SVR) uses a linear or nonlinear kernel function to map the input space (the SNP and QTL genotypes in our case) to a higher dimensional feature space, and performs modelling and prediction on the feature space. We tested multiple kernels and tuned two hyperparameters, kernel coefficient gamma and regularization parameter <em>C</em> (see Additional file <a href="#MOESM2" class="usa-link">2</a>: Table S2 and Additional file <a href="#MOESM4" class="usa-link">4</a>), and decided to use the radial basis function (rbf), and <em>C</em> = 1 (default) in our analysis (Table <a href="#Tab1" class="usa-link">1</a>).</p></section></section><section id="Sec9"><h3 class="pmc_sec_title">Scenarios</h3>
<p id="Par33">QTL were sorted in descending order based on the amount of variance they explained, which was calculated as <em>2pqa</em><sup><em>2</em></sup>, where <em>a</em> represents the simulated additive genetic effect of the QTL. In the first scenario (QTL0), only SNPs and no QTL were included in the model which means that (weighted) 2GBLUP reduces to GBLUP. In the other scenarios, an increasing number of QTL were included in the data. For those scenarios, we use “QTL” followed by a value to refer to a scenario in which the QTL together explained a percentage of variance equal to that value. For example, QTL80 means that the included QTL together explained 80% of the genetic variance. We included scenarios QTL5, QTL10, QTL15, QTL20, QTL40, QTL60, QTL70, QTL80, QTL85, QTL90, QTL95, QTL99 and QTL100. For GBLUP, RF and SVR, all QTL and SNP were given an equal weight. However, for 2GBLUP, all QTL were given an equal weight, while all SNPs were given another equal weight. In weighted 2GBLUP, relative weights (i.e., QTL variance) were provided as well.</p>
<p id="Par34">For each scenario and each model, accuracy and dispersion bias of predictions were evaluated in the test set. Accuracy was computed as the correlation coefficient between GEBV and TBV, and dispersion bias was computed as the regression coefficient of TBV on GEBV.</p></section><section id="Sec10"><h3 class="pmc_sec_title">Computational implementation</h3>
<p id="Par35">For GBLUP and 2GBLUP, calc_grm [<a href="#CR36" class="usa-link" aria-describedby="CR36">36</a>] was used to calculate the genomic relationship matrices. Thereafter, MTG2 [<a href="#CR37" class="usa-link" aria-describedby="CR37">37</a>] was used to estimate variance components and GEBV for GBLUP, 2GBLUP and w2GBLUP. For the machine learning models, the Scikit-learn package [<a href="#CR38" class="usa-link" aria-describedby="CR38">38</a>] was used.</p></section></section><section id="Sec11"><h2 class="pmc_sec_title">Results</h2>
<p id="Par36">When there was no QTL included (i.e., QTL0), the prediction accuracy was 0.76 for GBLUP (2GBLUP), 0.72 for SVR, and 0.50 for RF. For 2GBLUP, the accuracy gradually increased when more QTL were included, and peaked at about 0.83 when the included QTL explained around 80% of the genetic variance (Fig. <a href="#Fig3" class="usa-link">3</a>). When even more QTL were included, the accuracy dropped and reached a value of 0.80 when all QTL were included. However, at this point, the accuracies were still higher than for GBLUP (0.80 versus 0.76), which indicated that 2GBLUP obtained a higher accuracy than GBLUP for all the scenarios which included QTL. Generally, the accuracy of GBLUP was about 0.04 higher than SVR across scenarios. When more QTL were included in the data, the prediction accuracy of SVR gradually increased, although this increase was very small. Most importantly, the RF model resulted in a much lower accuracy (0.49–0.50) compared to the other models, and the prediction accuracy did not improve by including QTL in the data.</p>
<figure class="fig xbox font-sm" id="Fig3"><h3 class="obj_head">Fig. 3.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/cdd4/12362903/725b87955525/40104_2025_1250_Fig3_HTML.jpg" loading="lazy" id="MO3" height="584" width="787" alt="Fig. 3"></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>The accuracy of genomic prediction in generation 16 using different genomic prediction methods for different amounts of genetic variance explained by QTL</p></figcaption></figure><p id="Par37">By weighting QTL in 2GBLUP (w2GBLUP), the prediction accuracy always increased when more QTL were included (Fig. <a href="#Fig4" class="usa-link">4</a>). For the scenarios with QTL explaining up to 60% of the genetic variance, the accuracy patterns of 2GBLUP and w2GBLUP were very similar. From QTL60 to QTL100, w2GBLUP had a higher prediction accuracy than 2GBLUP, and the difference in accuracy between w2GBLUP and 2GBLUP gradually increased.</p>
<figure class="fig xbox font-sm" id="Fig4"><h3 class="obj_head">Fig. 4.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/cdd4/12362903/d290b3860c56/40104_2025_1250_Fig4_HTML.jpg" loading="lazy" id="MO4" height="589" width="787" alt="Fig. 4"></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>The accuracy of genomic prediction in generation 16 using 2GBLUP and weighted 2GBLUP for different amounts of genetic variance explained by QTL</p></figcaption></figure><p id="Par38">There were nearly no differences in dispersion bias across QTL scenarios, but there were large differences in bias between models (Figs. <a href="#Fig5" class="usa-link">5</a> and <a href="#Fig6" class="usa-link">6</a>). The dispersion bias of GBLUP and (w)2GBLUP were similar (~0.90), while the bias was stronger for SVR (~0.80), and RF (~1.29). While no model yielded unbiased GEBV, GBLUP and 2GBLUP had the least dispersion bias.</p>
<figure class="fig xbox font-sm" id="Fig5"><h3 class="obj_head">Fig. 5.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/cdd4/12362903/6f8d54f0e01a/40104_2025_1250_Fig5_HTML.jpg" loading="lazy" id="MO5" height="591" width="786" alt="Fig. 5"></p>
<div class="p text-right font-secondary"><a href="figure/Fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>The dispersion bias of genomic prediction in generation 16 using different genomic prediction methods for different amounts of genetic variance explained by QTL</p></figcaption></figure><figure class="fig xbox font-sm" id="Fig6"><h3 class="obj_head">Fig. 6.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/cdd4/12362903/aa344a234afe/40104_2025_1250_Fig6_HTML.jpg" loading="lazy" id="MO6" height="592" width="787" alt="Fig. 6"></p>
<div class="p text-right font-secondary"><a href="figure/Fig6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>The dispersion bias of genomic prediction in generation 16 using 2GBLUP and weighted 2GBLUP for different amounts of genetic variance explained by QTL</p></figcaption></figure></section><section id="Sec12"><h2 class="pmc_sec_title">Discussion</h2>
<p id="Par39">We aimed to assess the benefit of incorporating QTL information in GBLUP and machine learning models for genomic prediction accuracy, and to find the proportion of genetic variance explained by the QTL included in the model at which the highest prediction accuracy is achieved. Aiming to improve accuracy, we also introduced 2GBLUP, which is a model that separates previously identified QTL from SNP genotypes by including two random genomic effects. Although knowing all QTL is currently optimistic and even unrealistic, it is expected that an increasing number of QTL will be detected in the future. We found that the prediction accuracy of GBLUP, 2GBLUP and SVR can be improved by incorporating QTL genotypes in the dataset. However, the accuracy of GBLUP and SVR increased by less than 0.01 when the included QTL explained all genetic variance, while the accuracy of 2GBLUP increased by 0.07 when the included QTL explained 80% of the genetic variance and increased by 0.04 when the included QTL explained all genetic variance. The prediction accuracy of RF was much lower than other models, and was unaffected by inclusion of QTL genotypes.</p>
<p id="Par40">It is important to note that the results of this study are based on phenotypes that were simulated from additive (linear) genetic effects only. We chose this approach because breeding values are by definition linearly related to genotypes. If dominance and epistatic effects were present, the results might have been different, potentially favouring the flexibility of ML models. Evaluating the performance of ML models predicting breeding values or phenotypes with non-additive genetic effects will be the focus of our future study.</p>
<section id="Sec13"><h3 class="pmc_sec_title">Prediction accuracy in 2GBLUP</h3>
<p id="Par41">In this simulation study, we knew all the QTL genotypes and their effects, which made it possible to use 2GBLUP. In practice, however, we do not know the QTL, so an alternative approach is required. This approach can start with GWAS which identifies SNPs that are in high LD with the QTL and will therefore be significantly associated to the trait of interest, i.e., top-SNPs. These top-SNPs can be used in a (weighted) 2GBLUP model in a separate variance component to increase prediction accuracy [<a href="#CR28" class="usa-link" aria-describedby="CR28">28</a>]. For weighted 2GBLUP, the weights can be calculated using the allele frequencies and the estimated SNP-effects, which can be obtained from GWAS directly, or by methods that result in an estimate of the SNP effects such as SNP-BLUP [<a href="#CR39" class="usa-link" aria-describedby="CR39">39</a>] or Bayesian methods [<a href="#CR3" class="usa-link" aria-describedby="CR3">3</a>]. It is also possible to get SNP weights directly; Keel et al. [<a href="#CR40" class="usa-link" aria-describedby="CR40">40</a>] used gene expression information from RNA-seq to obtain SNP weights, and Meuwissen et al. [<a href="#CR41" class="usa-link" aria-describedby="CR41">41</a>] proposed GWABLUP, in which SNP weights can be derived based on the GWAS results.</p>
<p id="Par42">Our results showed that the prediction accuracy of GBLUP and 2GBLUP increased when QTL information was included, which was in line with results of earlier research. For example, Brøndum et al. [<a href="#CR25" class="usa-link" aria-describedby="CR25">25</a>] identified 1,623 QTL from imputed sequence data using GWAS, and their results showed that including the QTL genotypes improved prediction accuracy, especially when using the 2GBLUP model. Similarly, Raymond et al. [<a href="#CR28" class="usa-link" aria-describedby="CR28">28</a>] included 133 top-SNPs that were identified by a large meta-GWAS. These top-SNPs explained about 10% of the phenotypic variance, and were included in a separate variance component for multi-breed genomic prediction. Their results showed that GBLUP with two genetic effects (where one effect was based on top-SNPs) performed better than GBLUP with a single genetic effect. Altogether, we conclude that including QTL or top-SNPs as a separate genetic effect in the GBLUP model can result in a higher prediction accuracy compared to when QTL/top-SNPs and SNPs are combined in a single effect.</p>
<p id="Par43">In practice, we expect using only the top-SNPs detected by GWAS in a 2GBLUP model would still lead to higher prediction accuracy compared to regular GBLUP. However, using top-SNPs genotypes will result in a lower accuracy than using QTL genotypes, because the top-SNPs do not fully capture the underlying QTL effects due to imperfect LD.</p>
<p id="Par44">One of our objectives was to find the optimal proportion of genetic variance explained by QTL to be included in the model to reach the highest accuracy. For GBLUP and SVR, the highest accuracy was achieved when all QTL were included. While for 2GBLUP, the highest accuracy was achieved when the included QTL accumulatively explained 80% of the variance. The prediction accuracy of 2GBLUP decreased when including additional QTL. This decrease may be caused by the assumption of the model that the effects of all variants that are used to build the relationship matrix have a common variance [<a href="#CR2" class="usa-link" aria-describedby="CR2">2</a>, <a href="#CR5" class="usa-link" aria-describedby="CR5">5</a>]. However, this assumption was severely violated because nearly 25% of the bottom-ranked QTL jointly explained only 1% of the genetic variance. When more small effect QTL were included, the large effect QTL got more shrinkage, which impaired the prediction accuracy. To overcome this, we tested a 2GBLUP model where QTL variances were used as weights to build the QTL relationship matrix (w2GBLUP). When the QTL cumulatively explained more than 80% of the genetic variance, the accuracies of w2GBLUP further increased (Fig. <a href="#Fig4" class="usa-link">4</a>). We therefore conclude that using a weighted G matrix can avoid a decrease of accuracy when including progressively more QTL that explained a smaller part of the genetic variance.</p>
<p id="Par45">It should be noted that the proportion of genetic variance explained by QTL that results in the highest prediction accuracy may depend on the distribution of QTL effect, number of QTL, selection/replacement design and other factors. When these factors change, the optimal fraction of QTL to include in the model may no longer be at 80% explained genetic variance for 2GBLUP. Furthermore, prediction accuracy can be affected by the heritability and reference population size. The impact of these factors on the accuracy deserves a proper investigation and this will be studied in our next study.</p></section><section id="Sec14"><h3 class="pmc_sec_title">Reasons for low prediction accuracy in RF</h3>
<p id="Par46">The prediction accuracy of RF was about 0.5 for all scenarios, while other models obtained an accuracy above 0.7. This is in contrast to other studies based on real datasets where RF had a comparable prediction accuracy and mean squared error to GBLUP for phenotype prediction [<a href="#CR16" class="usa-link" aria-describedby="CR16">16</a>, <a href="#CR17" class="usa-link" aria-describedby="CR17">17</a>, <a href="#CR32" class="usa-link" aria-describedby="CR32">32</a>]. It should also be noted that these studies used the correlation between the predicted values and phenotypes as accuracy, while we used the correlation between the predicted values and true breeding values. However, in our study, the correlation between the predicted values and the phenotype of RF was also significantly lower than for the other models (results not shown).</p>
<p id="Par47">There are two main reasons for the low accuracy of RF. The first possible reason is that the data structure in this study does not allow RF to fully realize its potential. Machine learning models may perform better than conventional methods when non-additive effects are present, because of their flexibility to deal with complex interactions. In this study, however, we only simulated additive genetic effects, which may limit the benefit of RF. The results of Wang et al. [<a href="#CR16" class="usa-link" aria-describedby="CR16">16</a>] indeed showed that RF had a higher prediction accuracy than GBLUP for litter size in pigs, which is a trait believed to be controlled by substantial dominance and epistasis effects [<a href="#CR42" class="usa-link" aria-describedby="CR42">42</a>].</p>
<p id="Par48">The second possible reason for the low accuracy of RF is that RF is not designed well for this particular prediction problem. Here we provide three different perspectives to illustrate this. First, RF may not be able to handle a large number of features, which are the SNP and QTL genotypes in our case. Other studies have reported that the performance of RF tends to decline when the number of features becomes very large [<a href="#CR43" class="usa-link" aria-describedby="CR43">43</a>, <a href="#CR44" class="usa-link" aria-describedby="CR44">44</a>]. However, it is not quite clear whether it is the total number of features, or the proportion of features that are informative that determines the accuracy of RF. We therefore also tested the influence of number of features (SNPs and QTL), and the ratio of informative (QTL) over non-informative (SNPs) features on the accuracy (see Additional file <a href="#MOESM5" class="usa-link">5</a>: Fig. S1). We simulated a dataset with 20 to 8,179 causal variants (QTL) with a heritability of 1. There are three scenarios: (a) QTL only, (b) QTL combined with SNPs, where the number of SNPs was twice the number of QTL, and (c) QTL combined with a fixed number (10,000) of SNPs. The results showed that the accuracy of RF depended on the total number of informative features, and did not depend on the ratio of number of QTL over number of SNPs. Those results suggest that the accuracy of RF depends on the number of informative QTL underlying the trait, which implies that RF is not suitable for predicting complex traits.</p>
<p id="Par49">Second, RF might be prone to overfitting. We tested for overfitting by looking at the correlation coefficient between phenotype and predicted value in the training dataset for scenario QTL80. The correlation coefficient for 2GBLUP, RF and SVR were 0.661, 0.986 and 0.805, respectively. In theory, the predicted values should only capture the true breeding value and not any variation in the simulated environmental effect, which results in a maximum expected value of this correlation equal to the square root of the heritability, which in our case is 0.5. Given that all models show a higher correlation than 0.5, we conclude that 2GBLUP, RF and SVR all suffered from overfitting. For RF, the overfitting was extreme, because the correlation coefficient was nearly 1. Other scenarios (with different numbers of QTL included) resulted in a similar outcome (results not shown). Normally, overfitting can be overcome by tuning the hyperparameters, but that strategy was not effective in our study (see Additional file <a href="#MOESM3" class="usa-link">3</a>). Another option to solve overfitting may be to increase the size of training dataset, such that the number of features does not exceed the number of data points. In this study, the number of features (loci) was four times higher as the number of data points (animals), which was much higher than in other studies [<a href="#CR45" class="usa-link" aria-describedby="CR45">45</a>, <a href="#CR46" class="usa-link" aria-describedby="CR46">46</a>], and may therefore be an important reason for overfitting.</p>
<p id="Par50">Third, RF generated incorrect feature importance. We got feature importance from RF directly, and the variance of SNPs from backsolving GBLUP. The backsolved SNP variances were scaled, so that their sum was equal to 1. Then we plotted the scaled SNP variance of GBLUP and the feature importance of RF against the scaled true feature variance. The results showed that the feature variance of GBLUP were generally in line with the true variance, while RF overestimated the importance of few features and underestimated the importance of the majority of features (see Additional file <a href="#MOESM5" class="usa-link">5</a>: Fig. S2). We find it difficult to explain the difference between overfitting and incorrect feature importance, but maybe it helps with an analogy to cars: The car does not drive (bad performance of RF) because the motor is not running (i.e., feature importance is incorrect), but the real cause may be that there is no gas in the tank (overfitting).</p>
<p id="Par51">It is worth mentioning that the performance of random forest may be improved by combining it with regularization methods or by performing feature selection [<a href="#CR47" class="usa-link" aria-describedby="CR47">47</a>, <a href="#CR48" class="usa-link" aria-describedby="CR48">48</a>]. However, these strategies are typically dataset-specific and could reduce the generalizability of our study, which is why we were not explored here.</p></section><section id="Sec15"><h3 class="pmc_sec_title">Prediction accuracy in SVR</h3>
<p id="Par52">Our results showed that the prediction accuracy of SVR was consistently lower than that of GBLUP, which is in line with one empirical study [<a href="#CR17" class="usa-link" aria-describedby="CR17">17</a>], but in contrasts with the findings of several other studies [<a href="#CR16" class="usa-link" aria-describedby="CR16">16</a>, <a href="#CR49" class="usa-link" aria-describedby="CR49">49</a>, <a href="#CR50" class="usa-link" aria-describedby="CR50">50</a>]. There may be two possible reasons for this discrepancy. Firstly, the nature of the dataset may restrict the potential of SVR. While the fully additive genetic signals under this data structure match with the assumptions of GBLUP, this is not necessarily the case for SVR, whose potential benefits of capturing non-linear genetic effects cannot be utilized here. In the presence of non-additive genetic effects, SVR may have benefit over GBLUP [<a href="#CR16" class="usa-link" aria-describedby="CR16">16</a>, <a href="#CR49" class="usa-link" aria-describedby="CR49">49</a>, <a href="#CR50" class="usa-link" aria-describedby="CR50">50</a>]. Secondly, SVR is prone to overfitting. When many markers (QTL and SNPs) are included, it seems that GBLUP effectively integrates all marker information through the genomic relationship matrix, making it more robust to noise. Whereas SVR may not distinguish signal from noise efficiently.</p></section><section id="Sec16"><h3 class="pmc_sec_title">Trend in accuracy over generations</h3>
<p id="Par53">It is well known that the accuracy of genomic prediction decays over generations when reference populations are not updated [<a href="#CR3" class="usa-link" aria-describedby="CR3">3</a>, <a href="#CR51" class="usa-link" aria-describedby="CR51">51</a>–<a href="#CR53" class="usa-link" aria-describedby="CR53">53</a>], which is caused by less strong relationships between selection candidates and reference animals when they are more generations apart [<a href="#CR53" class="usa-link" aria-describedby="CR53">53</a>–<a href="#CR55" class="usa-link" aria-describedby="CR55">55</a>]. Another explanation for this is that the LD between SNPs and QTL decays over generations [<a href="#CR56" class="usa-link" aria-describedby="CR56">56</a>]. The focus of this study was to include QTL genotypes as features in genomic prediction, which alleviates the necessity of LD between the SNPs and QTL. Therefore, genomic prediction accuracy may persist better across generations when QTL are included. To test this hypothesis, we compared the accuracy of predictions across generations 16 to 20, while still using the original training dataset (generation 11–15) and randomly selecting 500 animals per generation from generations 17 to 20 to add to the test set.</p>
<p id="Par54">The results showed that when the included QTL explained 5% of genetic variance (scenario QTL5), the rate of decline over generations was similar for GBLUP, 2GBLUP, SVR and RF (see Additional file <a href="#MOESM6" class="usa-link">6</a>: Fig. S1a). However, when QTL that explained 80% of genetic variance were included (scenario QTL80), the accuracy of 2GBLUP decreased less compared to GBLUP (see Additional file <a href="#MOESM6" class="usa-link">6</a>: Fig. S1b). Therefore, the drop in accuracy across generations was similar for GBLUP, RF and SVR for the QTL5 and QTL80 scenarios, while for 2GBLUP the drop was lower with QTL80. We found that the persistency in accuracy across generations was not improved by including QTL information in GBLUP, SVR and RF. Depending on the proportion of variance explained by the included QTL, the accuracy of 2GBLUP was more persistent than GBLUP, which indicates that 2GBLUP is able to mitigate to some extent the impact of relationship or LD decay on the prediction accuracy.</p></section></section><section id="Sec17"><h2 class="pmc_sec_title">Conclusions</h2>
<p id="Par55">This study investigated the impact of incorporating QTL information into genomic prediction models, comparing GBLUP models with the ML models RF and SVR. Our findings show that incorporating QTL genotypes in GBLUP and SVR can improve prediction accuracy. The extent of improvement varies across models, i.e., 2GBLUP benefits most from incorporating QTL while the incorporation of QTL resulted a limited increase in prediction accuracy of SVR and GBLUP. Moreover, GBLUP consistently resulted in a higher prediction accuracy than SVR across all scenarios. These findings suggest that if more QTL information becomes available, the selection of an appropriate model is crucial for optimizing prediction performance. In contrast to (w)2GBLUP, RF did not show improvements when QTL information was included, and had a much lower accuracy than the other models. Two possible reasons for this result are that the data structure in this study does not allow RF to fully realize its potential, and that RF is not designed well for this particular prediction problem. These results highlight the importance of selecting appropriate models for genomic prediction and underscore the potential limitations of machine learning models when applied to genomic prediction in livestock.</p></section><section id="Sec18"><h2 class="pmc_sec_title">Supplementary Information</h2>
<section class="sm xbox font-sm" id="MOESM1"><div class="media p"><div class="caption">
<a href="/articles/instance/12362903/bin/40104_2025_1250_MOESM1_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link">40104_2025_1250_MOESM1_ESM.docx</a><sup> (15.9KB, docx) </sup><p>Additional file 1. QMSim input file. This file contains the QMSim input file of simulated data.</p>
</div></div></section><section class="sm xbox font-sm" id="MOESM2"><div class="media p"><div class="caption">
<a href="/articles/instance/12362903/bin/40104_2025_1250_MOESM2_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link">40104_2025_1250_MOESM2_ESM.docx</a><sup> (14.7KB, docx) </sup><p>Additional file 2. Parameters that machine learning methods required. This file contains the lists of parameters that used in random forest and support vector regression.</p>
</div></div></section><section class="sm xbox font-sm" id="MOESM3"><div class="media p"><div class="caption">
<a href="/articles/instance/12362903/bin/40104_2025_1250_MOESM3_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link">40104_2025_1250_MOESM3_ESM.docx</a><sup> (297.4KB, docx) </sup><p>Additional file 3. Parameter tuning process of random forest. This file contains the detail of parameter tuning process of random forest.</p>
</div></div></section><section class="sm xbox font-sm" id="MOESM4"><div class="media p"><div class="caption">
<a href="/articles/instance/12362903/bin/40104_2025_1250_MOESM4_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link">40104_2025_1250_MOESM4_ESM.docx</a><sup> (105.7KB, docx) </sup><p>Additional file 4. Parameter tuning process of support vector regression. This file contains the detail of parameter tuning process of support vector regression.</p>
</div></div></section><section class="sm xbox font-sm" id="MOESM5"><div class="media p"><div class="caption">
<a href="/articles/instance/12362903/bin/40104_2025_1250_MOESM5_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link">40104_2025_1250_MOESM5_ESM.docx</a><sup> (191KB, docx) </sup><p>Additional file 5: Fig. S1. Prediction accuracy of RF and GBLUP using simulated dataset with different number of informative QTL. a) QTL only, b) QTL combined with SNPs that close to the QTL, and the number of SNPs was twice the number of QTL and c) QTL combined with fixed numberof SNPs. Fig. S2. Real QTL variance against variance or importance obtained from a) GBLUP and b) RF.</p>
</div></div></section><section class="sm xbox font-sm" id="MOESM6"><div class="media p"><div class="caption">
<a href="/articles/instance/12362903/bin/40104_2025_1250_MOESM6_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link">40104_2025_1250_MOESM6_ESM.docx</a><sup> (72.6KB, docx) </sup><p>Additional file 6: Fig. S1. The accuracy of genomic prediction using different genomic prediction methods for a) 5% of genetic variance explained by QTL and b) 80% of genetic variance explained by QTL across generations.</p>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>Not applicable.</p></section><section id="glossary1" class="glossary"><h2 class="pmc_sec_title">Abbreviations</h2>
<dl class="def-list">
<dt>GEBV</dt>
<dd><p id="Par4">Genomic estimated breeding values</p></dd>
<dt>GP</dt>
<dd><p id="Par5">Genomic prediction</p></dd>
<dt>GWAS</dt>
<dd><p id="Par6">Genome-wide association studies</p></dd>
<dt>LD</dt>
<dd><p id="Par7">Linkage disequilibrium</p></dd>
<dt>ML</dt>
<dd><p id="Par8">Machine learning</p></dd>
<dt>QTL</dt>
<dd><p id="Par9">Quantitative trait loci</p></dd>
<dt>RF</dt>
<dd><p id="Par10">Random forest</p></dd>
<dt>SNP</dt>
<dd><p id="Par11">Single nucleotide polymorphism</p></dd>
<dt>SVR</dt>
<dd><p id="Par12">Support vector regression</p></dd>
<dt>TBV</dt>
<dd><p id="Par13">True breeding values</p></dd>
<dt>w2GBLUP</dt>
<dd><p id="Par14">Weighted 2GBLUP</p></dd>
<dt>WGS</dt>
<dd><p id="Par15">Whole genome sequence</p></dd>
</dl></section><section id="notes1"><h2 class="pmc_sec_title">Authors’ contributions</h2>
<p>PD, YCJW, THEM and MPLC participated in the design of the study. JY performed the simulations and statistical analyses and wrote the first draft of the paper. JY, PD, YCJW, THEM and MPLC were involved in the interpretation of the results. All authors read and approved the final manuscript.</p></section><section id="notes2"><h2 class="pmc_sec_title">Funding</h2>
<p>The authors thank to the financial support from China Scholarship Council (CSC, File No. 202007720040), which has sponsored Jifan Yang’s PhD study at Wageningen University &amp; Research. The use of the HPC cluster has been made possible by CAT-AgroFood (Shared Research Facilities Wageningen UR).</p></section><section id="notes3"><h2 class="pmc_sec_title">Data availability</h2>
<p>QMSim input file used to generate the data during this study is Additional file <a href="#MOESM1" class="usa-link">1</a>. Seeds for QMSim simulation and codes for analysis can be found via <a href="https://git.wageningenur.nl/yang110/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://git.wageningenur.nl/yang110/</a>.</p></section><section id="notes4"><h2 class="pmc_sec_title">Declarations</h2>
<section id="FPar1"><h3 class="pmc_sec_title">Ethics approval and consent to participate</h3>
<p id="Par56">Not applicable.</p></section><section id="FPar2"><h3 class="pmc_sec_title">Consent for publication</h3>
<p id="Par57">Not applicable.</p></section><section id="FPar3"><h3 class="pmc_sec_title">Competing interests</h3>
<p id="Par58">The authors declare that they have no competing interests.</p></section></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Hayes BJ, Bowman PJ, Chamberlain AJ, Goddard ME. Invited review: Genomic selection in dairy cattle: progress and challenges. J Dairy Sci. 2009;92:433–43.
</cite> [<a href="https://doi.org/10.3168/jds.2008-1646" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19164653/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hayes%20BJ,%20Bowman%20PJ,%20Chamberlain%20AJ,%20Goddard%20ME.%20Invited%20review:%20Genomic%20selection%20in%20dairy%20cattle:%20progress%20and%20challenges.%20J%20Dairy%20Sci.%202009;92:433%E2%80%9343." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>Van Raden PM, Van Tassell CP, Wiggans GR, Sonstegard TS, Schnabel RD, Taylor JF, et al. Invited review: reliability of genomic predictions for North American Holstein bulls. J Dairy Sci. 2009;92:16–24.
</cite> [<a href="https://doi.org/10.3168/jds.2008-1514" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19109259/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Van%20Raden%20PM,%20Van%20Tassell%20CP,%20Wiggans%20GR,%20Sonstegard%20TS,%20Schnabel%20RD,%20Taylor%20JF,%20et%20al.%20Invited%20review:%20reliability%20of%20genomic%20predictions%20for%20North%20American%20Holstein%20bulls.%20J%20Dairy%20Sci.%202009;92:16%E2%80%9324." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>Meuwissen THE, Hayes BJ, Goddard ME. Prediction of total genetic value using genome-wide dense marker maps. Genetics. 2001;157:1819–29.
</cite> [<a href="https://doi.org/10.1093/genetics/157.4.1819" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1461589/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11290733/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Meuwissen%20THE,%20Hayes%20BJ,%20Goddard%20ME.%20Prediction%20of%20total%20genetic%20value%20using%20genome-wide%20dense%20marker%20maps.%20Genetics.%202001;157:1819%E2%80%9329." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Christensen OF, Lund MS. Genomic prediction when some animals are not genotyped. Genet Sel Evol. 2010;42:2.
</cite> [<a href="https://doi.org/10.1186/1297-9686-42-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2834608/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20105297/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Christensen%20OF,%20Lund%20MS.%20Genomic%20prediction%20when%20some%20animals%20are%20not%20genotyped.%20Genet%20Sel%20Evol.%202010;42:2." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Legarra A, Aguilar I, Misztal I. A relationship matrix including full pedigree and genomic information. J Dairy Sci. 2009;92:4656–63.
</cite> [<a href="https://doi.org/10.3168/jds.2009-2061" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19700729/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Legarra%20A,%20Aguilar%20I,%20Misztal%20I.%20A%20relationship%20matrix%20including%20full%20pedigree%20and%20genomic%20information.%20J%20Dairy%20Sci.%202009;92:4656%E2%80%9363." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Wang H, Misztal I, Aguilar I, Legarra A, Muir WM. Genome-wide association mapping including phenotypes from relatives without genotypes. Genet Res Camb. 2012;94:73–83.
</cite> [<a href="https://doi.org/10.1017/S0016672312000274" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22624567/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang%20H,%20Misztal%20I,%20Aguilar%20I,%20Legarra%20A,%20Muir%20WM.%20Genome-wide%20association%20mapping%20including%20phenotypes%20from%20relatives%20without%20genotypes.%20Genet%20Res%20Camb.%202012;94:73%E2%80%9383." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Habier D, Fernando RL, Kizilkaya K, Garrick DJ. Extension of the bayesian alphabet for genomic selection. BMC Bioinformatics. 2011;12:186.
</cite> [<a href="https://doi.org/10.1186/1471-2105-12-186" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3144464/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21605355/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Habier%20D,%20Fernando%20RL,%20Kizilkaya%20K,%20Garrick%20DJ.%20Extension%20of%20the%20bayesian%20alphabet%20for%20genomic%20selection.%20BMC%20Bioinformatics.%202011;12:186." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>van Bergen GHH, Duenk P, Albers CA, Bijma P, Calus MPL, Wientjes YCJ, et al. Bayesian neural networks with variable selection for prediction of genotypic values. Genet Sel Evol. 2020;52:26.
</cite> [<a href="https://doi.org/10.1186/s12711-020-00544-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7227313/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32414320/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?van%20Bergen%20GHH,%20Duenk%20P,%20Albers%20CA,%20Bijma%20P,%20Calus%20MPL,%20Wientjes%20YCJ,%20et%20al.%20Bayesian%20neural%20networks%20with%20variable%20selection%20for%20prediction%20of%20genotypic%20values.%20Genet%20Sel%20Evol.%202020;52:26." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>Zhao T, Fernando R, Cheng H. Interpretable artificial neural networks incorporating Bayesian alphabet models for genome-wide prediction and association studies. G3 (Bethesda). 2021;11:10.</cite> [<a href="https://doi.org/10.1093/g3journal/jkab228" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8496266/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34499126/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhao%20T,%20Fernando%20R,%20Cheng%20H.%20Interpretable%20artificial%20neural%20networks%20incorporating%20Bayesian%20alphabet%20models%20for%20genome-wide%20prediction%20and%20association%20studies.%20G3%20(Bethesda).%202021;11:10." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>De los Campos G, Gianola D, Rosa GJ, Weigel KA, Crossa J. Semi-parametric genomic-enabled prediction of genetic values using reproducing kernel Hilbert spaces methods. Genet Res (Camb). 2010;92:295–308.
</cite> [<a href="https://doi.org/10.1017/S0016672310000285" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20943010/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?De%20los%20Campos%20G,%20Gianola%20D,%20Rosa%20GJ,%20Weigel%20KA,%20Crossa%20J.%20Semi-parametric%20genomic-enabled%20prediction%20of%20genetic%20values%20using%20reproducing%20kernel%20Hilbert%20spaces%20methods.%20Genet%20Res%20(Camb).%202010;92:295%E2%80%93308." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Gianola D, Fernando RL, Stella A. Genomic-assisted prediction of genetic value with semiparametric procedures. Genetics. 2006;173:1761–76.
</cite> [<a href="https://doi.org/10.1534/genetics.105.049510" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1526664/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16648593/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gianola%20D,%20Fernando%20RL,%20Stella%20A.%20Genomic-assisted%20prediction%20of%20genetic%20value%20with%20semiparametric%20procedures.%20Genetics.%202006;173:1761%E2%80%9376." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Long N, Gianola D, Rosa GJ, Weigel KA. Application of support vector regression to genome-assisted prediction of quantitative traits. Theor Appl Genet. 2011;123:1065–74.
</cite> [<a href="https://doi.org/10.1007/s00122-011-1648-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21739137/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Long%20N,%20Gianola%20D,%20Rosa%20GJ,%20Weigel%20KA.%20Application%20of%20support%20vector%20regression%20to%20genome-assisted%20prediction%20of%20quantitative%20traits.%20Theor%20Appl%20Genet.%202011;123:1065%E2%80%9374." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>Gonzalez-Recio O, Forni S. Genome-wide prediction of discrete traits using Bayesian regressions and machine learning. Genet Sel Evol. 2011;43: 7.
</cite> [<a href="https://doi.org/10.1186/1297-9686-43-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3400433/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21329522/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gonzalez-Recio%20O,%20Forni%20S.%20Genome-wide%20prediction%20of%20discrete%20traits%20using%20Bayesian%20regressions%20and%20machine%20learning.%20Genet%20Sel%20Evol.%202011;43:%207." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>Spindel J, Begum H, Akdemir D, Virk P, Collard B, Redona E, et al. Genomic selection and association mapping in rice (<em>Oryza sativa</em>): effect of trait genetic architecture, training population composition, marker number and statistical model on accuracy of rice genomic selection in elite, tropical rice breeding lines. PLoS Genet. 2015;11: e1004982.
</cite> [<a href="https://doi.org/10.1371/journal.pgen.1004982" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4334555/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25689273/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Spindel%20J,%20Begum%20H,%20Akdemir%20D,%20Virk%20P,%20Collard%20B,%20Redona%20E,%20et%20al.%20Genomic%20selection%20and%20association%20mapping%20in%20rice%20(Oryza%20sativa):%20effect%20of%20trait%20genetic%20architecture,%20training%20population%20composition,%20marker%20number%20and%20statistical%20model%20on%20accuracy%20of%20rice%20genomic%20selection%20in%20elite,%20tropical%20rice%20breeding%20lines.%20PLoS%20Genet.%202015;11:%20e1004982." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>Waldmann P. Approximate Bayesian neural networks in genomic prediction. Genet Sel Evol. 2018;50:70.
</cite> [<a href="https://doi.org/10.1186/s12711-018-0439-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6303864/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30577737/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Waldmann%20P.%20Approximate%20Bayesian%20neural%20networks%20in%20genomic%20prediction.%20Genet%20Sel%20Evol.%202018;50:70." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Wang X, Shi S, Wang G, Luo W, Wei X, Qiu A, et al. Using machine learning to improve the accuracy of genomic prediction of reproduction traits in pigs. J Anim Sci Biotechnol. 2022;13: 60.
</cite> [<a href="https://doi.org/10.1186/s40104-022-00708-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9112588/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35578371/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang%20X,%20Shi%20S,%20Wang%20G,%20Luo%20W,%20Wei%20X,%20Qiu%20A,%20et%20al.%20Using%20machine%20learning%20to%20improve%20the%20accuracy%20of%20genomic%20prediction%20of%20reproduction%20traits%20in%20pigs.%20J%20Anim%20Sci%20Biotechnol.%202022;13:%2060." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>Azodi CB, Bolger E, McCarren A, Roantree M, de Los Campos G, Shiu SH. Benchmarking parametric and machine learning models for genomic prediction of complex traits. G3 (Bethesda). 2019;9:3691–702.
</cite> [<a href="https://doi.org/10.1534/g3.119.400498" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6829122/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31533955/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Azodi%20CB,%20Bolger%20E,%20McCarren%20A,%20Roantree%20M,%20de%20Los%20Campos%20G,%20Shiu%20SH.%20Benchmarking%20parametric%20and%20machine%20learning%20models%20for%20genomic%20prediction%20of%20complex%20traits.%20G3%20(Bethesda).%202019;9:3691%E2%80%93702." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Perez BC, Bink M, Svenson KL, Churchill GA, Calus MPL. Prediction performance of linear models and gradient boosting machine on complex phenotypes in outbred mice. G3 Genes|Genomes|Genetics. 2022;12: 4.</cite> [<a href="https://doi.org/10.1093/g3journal/jkac039" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8982369/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35166767/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Perez%20BC,%20Bink%20M,%20Svenson%20KL,%20Churchill%20GA,%20Calus%20MPL.%20Prediction%20performance%20of%20linear%20models%20and%20gradient%20boosting%20machine%20on%20complex%20phenotypes%20in%20outbred%20mice.%20G3%20Genes%7CGenomes%7CGenetics.%202022;12:%204." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Bellot P, de Los Campos G, Perez-Enciso M. Can deep learning improve genomic prediction of complex human traits? Genetics. 2018;210:809–19.
</cite> [<a href="https://doi.org/10.1534/genetics.118.301298" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6218236/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30171033/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bellot%20P,%20de%20Los%20Campos%20G,%20Perez-Enciso%20M.%20Can%20deep%20learning%20improve%20genomic%20prediction%20of%20complex%20human%20traits?%20Genetics.%202018;210:809%E2%80%9319." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>Okut H, Wu XL, Rosa GJ, Bauck S, Woodward BW, Schnabel RD, et al. Predicting expected progeny difference for marbling score in Angus cattle using artificial neural networks and Bayesian regression models. Genet Sel Evol. 2013;45:34.
</cite> [<a href="https://doi.org/10.1186/1297-9686-45-34" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3851253/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24024641/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Okut%20H,%20Wu%20XL,%20Rosa%20GJ,%20Bauck%20S,%20Woodward%20BW,%20Schnabel%20RD,%20et%20al.%20Predicting%20expected%20progeny%20difference%20for%20marbling%20score%20in%20Angus%20cattle%20using%20artificial%20neural%20networks%20and%20Bayesian%20regression%20models.%20Genet%20Sel%20Evol.%202013;45:34." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>Calus MPL, Bouwman AC, Schrooten C, Veerkamp RF. Efficient genomic prediction based on whole-genome sequence data using split-and-merge Bayesian variable selection. Genet Sel Evol. 2016;48:49.
</cite> [<a href="https://doi.org/10.1186/s12711-016-0225-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4926307/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27357580/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Calus%20MPL,%20Bouwman%20AC,%20Schrooten%20C,%20Veerkamp%20RF.%20Efficient%20genomic%20prediction%20based%20on%20whole-genome%20sequence%20data%20using%20split-and-merge%20Bayesian%20variable%20selection.%20Genet%20Sel%20Evol.%202016;48:49." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>van Binsbergen R, Calus MPL, Bink MC, van Eeuwijk FA, Schrooten C, Veerkamp RF. Genomic prediction using imputed whole-genome sequence data in Holstein Friesian cattle. Genet Sel Evol. 2015;47:71.
</cite> [<a href="https://doi.org/10.1186/s12711-015-0149-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4574568/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26381777/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?van%20Binsbergen%20R,%20Calus%20MPL,%20Bink%20MC,%20van%20Eeuwijk%20FA,%20Schrooten%20C,%20Veerkamp%20RF.%20Genomic%20prediction%20using%20imputed%20whole-genome%20sequence%20data%20in%20Holstein%20Friesian%20cattle.%20Genet%20Sel%20Evol.%202015;47:71." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>van den Berg I, Bowman PJ, MacLeod IM, Hayes BJ, Wang T, Bolormaa S, et al. Multi-breed genomic prediction using Bayes R with sequence data and dropping variants with a small effect. Genet Sel Evol. 2017;49:70.
</cite> [<a href="https://doi.org/10.1186/s12711-017-0347-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5609075/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28934948/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?van%20den%20Berg%20I,%20Bowman%20PJ,%20MacLeod%20IM,%20Hayes%20BJ,%20Wang%20T,%20Bolormaa%20S,%20et%20al.%20Multi-breed%20genomic%20prediction%20using%20Bayes%20R%20with%20sequence%20data%20and%20dropping%20variants%20with%20a%20small%20effect.%20Genet%20Sel%20Evol.%202017;49:70." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>Heidaritabar M, Calus MPL, Megens HJ, Vereijken A, Groenen MA, Bastiaansen JW. Accuracy of genomic prediction using imputed whole-genome sequence data in white layers. J Anim Breed Genet. 2016;133:167–79.
</cite> [<a href="https://doi.org/10.1111/jbg.12199" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26776363/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Heidaritabar%20M,%20Calus%20MPL,%20Megens%20HJ,%20Vereijken%20A,%20Groenen%20MA,%20Bastiaansen%20JW.%20Accuracy%20of%20genomic%20prediction%20using%20imputed%20whole-genome%20sequence%20data%20in%20white%20layers.%20J%20Anim%20Breed%20Genet.%202016;133:167%E2%80%9379." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>Brøndum RF, Su G, Janss L, Sahana G, Guldbrandtsen B, Boichard D, et al. Quantitative trait loci markers derived from whole genome sequence data increases the reliability of genomic prediction. J Dairy Sci. 2015;98:4107–16.
</cite> [<a href="https://doi.org/10.3168/jds.2014-9005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25892697/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Br%C3%B8ndum%20RF,%20Su%20G,%20Janss%20L,%20Sahana%20G,%20Guldbrandtsen%20B,%20Boichard%20D,%20et%20al.%20Quantitative%20trait%20loci%20markers%20derived%20from%20whole%20genome%20sequence%20data%20increases%20the%20reliability%20of%20genomic%20prediction.%20J%20Dairy%20Sci.%202015;98:4107%E2%80%9316." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>Fragomeni BO, Lourenco DAL, Legarra A, VanRaden PM, Misztal I. Alternative SNP weighting for single-step genomic best linear unbiased predictor evaluation of stature in US Holsteins in the presence of selected sequence variants. J Dairy Sci. 2019;102:10012–9.
</cite> [<a href="https://doi.org/10.3168/jds.2019-16262" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31495612/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Fragomeni%20BO,%20Lourenco%20DAL,%20Legarra%20A,%20VanRaden%20PM,%20Misztal%20I.%20Alternative%20SNP%20weighting%20for%20single-step%20genomic%20best%20linear%20unbiased%20predictor%20evaluation%20of%20stature%20in%20US%20Holsteins%20in%20the%20presence%20of%20selected%20sequence%20variants.%20J%20Dairy%20Sci.%202019;102:10012%E2%80%939." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Moghaddar N, Khansefid M, van der Werf JHJ, Bolormaa S, Duijvesteijn N, Clark SA, et al. Genomic prediction based on selected variants from imputed whole-genome sequence data in Australian sheep populations. Genet Sel Evol. 2019;51:72.
</cite> [<a href="https://doi.org/10.1186/s12711-019-0514-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6896509/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31805849/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Moghaddar%20N,%20Khansefid%20M,%20van%20der%20Werf%20JHJ,%20Bolormaa%20S,%20Duijvesteijn%20N,%20Clark%20SA,%20et%20al.%20Genomic%20prediction%20based%20on%20selected%20variants%20from%20imputed%20whole-genome%20sequence%20data%20in%20Australian%20sheep%20populations.%20Genet%20Sel%20Evol.%202019;51:72." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Raymond B, Bouwman AC, Wientjes YCJ, Schrooten C, Houwing-Duistermaat J, Veerkamp RF. Genomic prediction for numerically small breeds, using models with pre-selected and differentially weighted markers. Genet Sel Evol. 2018;50:49.
</cite> [<a href="https://doi.org/10.1186/s12711-018-0419-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6186145/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30314431/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Raymond%20B,%20Bouwman%20AC,%20Wientjes%20YCJ,%20Schrooten%20C,%20Houwing-Duistermaat%20J,%20Veerkamp%20RF.%20Genomic%20prediction%20for%20numerically%20small%20breeds,%20using%20models%20with%20pre-selected%20and%20differentially%20weighted%20markers.%20Genet%20Sel%20Evol.%202018;50:49." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>van den Berg I, Boichard D, Lund MS. Sequence variants selected from a multi-breed GWAS can improve the reliability of genomic predictions in dairy cattle. Genet Sel Evol. 2016;48:83.
</cite> [<a href="https://doi.org/10.1186/s12711-016-0259-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5095991/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27809758/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?van%20den%20Berg%20I,%20Boichard%20D,%20Lund%20MS.%20Sequence%20variants%20selected%20from%20a%20multi-breed%20GWAS%20can%20improve%20the%20reliability%20of%20genomic%20predictions%20in%20dairy%20cattle.%20Genet%20Sel%20Evol.%202016;48:83." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR30">
<span class="label">30.</span><cite>VanRaden PM, Tooker ME, O’Connell JR, Cole JB, Bickhart DM. Selecting sequence variants to improve genomic predictions for dairy cattle. Genet Sel Evol. 2017;49:32.
</cite> [<a href="https://doi.org/10.1186/s12711-017-0307-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5339980/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28270096/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?VanRaden%20PM,%20Tooker%20ME,%20O%E2%80%99Connell%20JR,%20Cole%20JB,%20Bickhart%20DM.%20Selecting%20sequence%20variants%20to%20improve%20genomic%20predictions%20for%20dairy%20cattle.%20Genet%20Sel%20Evol.%202017;49:32." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>Hu ZL, Park CA, Reecy JM. Bringing the animal QTLdb and CorrDB into the future: meeting new challenges and providing updated services. Nucleic Acids Res. 2022;50:D956-61.
</cite> [<a href="https://doi.org/10.1093/nar/gkab1116" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8728226/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34850103/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hu%20ZL,%20Park%20CA,%20Reecy%20JM.%20Bringing%20the%20animal%20QTLdb%20and%20CorrDB%20into%20the%20future:%20meeting%20new%20challenges%20and%20providing%20updated%20services.%20Nucleic%20Acids%20Res.%202022;50:D956-61." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>Abdollahi-Arpanahi R, Gianola D, Penagaricano F. Deep learning versus parametric and ensemble methods for genomic prediction of complex phenotypes. Genet Sel Evol. 2020;52: 12.
</cite> [<a href="https://doi.org/10.1186/s12711-020-00531-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7038529/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32093611/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Abdollahi-Arpanahi%20R,%20Gianola%20D,%20Penagaricano%20F.%20Deep%20learning%20versus%20parametric%20and%20ensemble%20methods%20for%20genomic%20prediction%20of%20complex%20phenotypes.%20Genet%20Sel%20Evol.%202020;52:%2012." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Sargolzaei M, Schenkel FS. QMSim: a large-scale genome simulator for livestock. Bioinformatics. 2009;25:680–1.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btp045" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19176551/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sargolzaei%20M,%20Schenkel%20FS.%20QMSim:%20a%20large-scale%20genome%20simulator%20for%20livestock.%20Bioinformatics.%202009;25:680%E2%80%931." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>Van Raden PM. Efficient methods to compute genomic predictions. J Dairy Sci. 2008;91:4414–23.
</cite> [<a href="https://doi.org/10.3168/jds.2007-0980" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18946147/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Van%20Raden%20PM.%20Efficient%20methods%20to%20compute%20genomic%20predictions.%20J%20Dairy%20Sci.%202008;91:4414%E2%80%9323." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>Breiman L. Random forests. Mach Learn. 2001;45:5–32.</cite> [<a href="https://scholar.google.com/scholar_lookup?Breiman%20L.%20Random%20forests.%20Mach%20Learn.%202001;45:5%E2%80%9332." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR36">
<span class="label">36.</span><cite>Calus MPL, Vandenplas J. Calc_grm—a program to compute pedigree, genomic, and combined relationship matrices. Wageningen: Wageningen University &amp; Research; 2016.</cite>
</li>
<li id="CR37">
<span class="label">37.</span><cite>Lee SH, van der Werf JHJ. MTG2: an efficient algorithm for multivariate linear mixed model analysis based on genomic information. Bioinformatics. 2016;32:1420–2.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btw012" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4848406/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26755623/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lee%20SH,%20van%20der%20Werf%20JHJ.%20MTG2:%20an%20efficient%20algorithm%20for%20multivariate%20linear%20mixed%20model%20analysis%20based%20on%20genomic%20information.%20Bioinformatics.%202016;32:1420%E2%80%932." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR38">
<span class="label">38.</span><cite>Pedregosa F, Varoquaux G, Gramfort A, Michel V, Thirion B, Grisel O, et al. Scikit-learn: machine learning in Python. J Mach Learn Res. 2011;12:2825–30.</cite> [<a href="https://scholar.google.com/scholar_lookup?Pedregosa%20F,%20Varoquaux%20G,%20Gramfort%20A,%20Michel%20V,%20Thirion%20B,%20Grisel%20O,%20et%20al.%20Scikit-learn:%20machine%20learning%20in%20Python.%20J%20Mach%20Learn%20Res.%202011;12:2825%E2%80%9330." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR39">
<span class="label">39.</span><cite>Stranden I, Garrick DJ. Technical note: derivation of equivalent computing algorithms for genomic predictions and reliabilities of animal merit. J Dairy Sci. 2009;92:2971–5.
</cite> [<a href="https://doi.org/10.3168/jds.2008-1929" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19448030/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Stranden%20I,%20Garrick%20DJ.%20Technical%20note:%20derivation%20of%20equivalent%20computing%20algorithms%20for%20genomic%20predictions%20and%20reliabilities%20of%20animal%20merit.%20J%20Dairy%20Sci.%202009;92:2971%E2%80%935." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR40">
<span class="label">40.</span><cite>Keel BN, Snelling WM, Lindholm-Perry AK, Oliver WT, Kuehn LA, Rohrer GA. Using SNP weights derived from gene expression modules to improve GWAS power for feed efficiency in pigs. Front Genet. 2019;10: 1339.
</cite> [<a href="https://doi.org/10.3389/fgene.2019.01339" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6985563/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32038708/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Keel%20BN,%20Snelling%20WM,%20Lindholm-Perry%20AK,%20Oliver%20WT,%20Kuehn%20LA,%20Rohrer%20GA.%20Using%20SNP%20weights%20derived%20from%20gene%20expression%20modules%20to%20improve%20GWAS%20power%20for%20feed%20efficiency%20in%20pigs.%20Front%20Genet.%202019;10:%201339." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR41">
<span class="label">41.</span><cite>Meuwissen T, Eikje LS, Gjuvsland AB. GWABLUP: genome-wide association assisted best linear unbiased prediction of genetic values. Genet Sel Evol. 2024;56:17.
</cite> [<a href="https://doi.org/10.1186/s12711-024-00881-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11234632/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38429665/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Meuwissen%20T,%20Eikje%20LS,%20Gjuvsland%20AB.%20GWABLUP:%20genome-wide%20association%20assisted%20best%20linear%20unbiased%20prediction%20of%20genetic%20values.%20Genet%20Sel%20Evol.%202024;56:17." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR42">
<span class="label">42.</span><cite>Vitezica ZG, Reverter A, Herring W, Legarra A. Dominance and epistatic genetic variances for litter size in pigs using genomic models. Genet Sel Evol. 2018;50:71.
</cite> [<a href="https://doi.org/10.1186/s12711-018-0437-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6303964/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30577727/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Vitezica%20ZG,%20Reverter%20A,%20Herring%20W,%20Legarra%20A.%20Dominance%20and%20epistatic%20genetic%20variances%20for%20litter%20size%20in%20pigs%20using%20genomic%20models.%20Genet%20Sel%20Evol.%202018;50:71." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR43">
<span class="label">43.</span><cite>Amaratunga D, Cabrera J, Lee YS. Enriched random forests. Bioinformatics. 2008;24:2010–4.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btn356" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18650208/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Amaratunga%20D,%20Cabrera%20J,%20Lee%20YS.%20Enriched%20random%20forests.%20Bioinformatics.%202008;24:2010%E2%80%934." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR44">
<span class="label">44.</span><cite>Liu Y, Zhao H. Variable importance-weighted random forests. Quant Biol. 2017;5:338–51.
</cite> [<a href="/articles/PMC6051549/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30034909/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Liu%20Y,%20Zhao%20H.%20Variable%20importance-weighted%20random%20forests.%20Quant%20Biol.%202017;5:338%E2%80%9351." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR45">
<span class="label">45.</span><cite>Farooq M, van Dijk ADJ, Nijveen H, Mansoor S, de Ridder D. Genomic prediction in plants: opportunities for ensemble machine learning based approaches. F1000Res. 2022;11:802.
</cite> [<a href="https://doi.org/10.12688/f1000research.122437.1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10080209/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37035464/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Farooq%20M,%20van%20Dijk%20ADJ,%20Nijveen%20H,%20Mansoor%20S,%20de%20Ridder%20D.%20Genomic%20prediction%20in%20plants:%20opportunities%20for%20ensemble%20machine%20learning%20based%20approaches.%20F1000Res.%202022;11:802." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR46">
<span class="label">46.</span><cite>Wang DR, Guadagno CR, Mao XW, Mackay DS, Pleban JR, Baker RL, et al. A framework for genomics-informed ecophysiological modeling in plants. J Exp Bot. 2019;70:2561–74.
</cite> [<a href="https://doi.org/10.1093/jxb/erz090" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6487588/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30825375/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang%20DR,%20Guadagno%20CR,%20Mao%20XW,%20Mackay%20DS,%20Pleban%20JR,%20Baker%20RL,%20et%20al.%20A%20framework%20for%20genomics-informed%20ecophysiological%20modeling%20in%20plants.%20J%20Exp%20Bot.%202019;70:2561%E2%80%9374." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR47">
<span class="label">47.</span><cite>Okser S, Pahikkala T, Airola A, Salakoski T, Ripatti S, Aittokallio T. Regularized machine learning in the genetic prediction of complex traits. PLoS Genet. 2014;10: e1004754.
</cite> [<a href="https://doi.org/10.1371/journal.pgen.1004754" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4230844/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25393026/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Okser%20S,%20Pahikkala%20T,%20Airola%20A,%20Salakoski%20T,%20Ripatti%20S,%20Aittokallio%20T.%20Regularized%20machine%20learning%20in%20the%20genetic%20prediction%20of%20complex%20traits.%20PLoS%20Genet.%202014;10:%20e1004754." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR48">
<span class="label">48.</span><cite>Chen X, Ishwaran H. Random forests for genomic data analysis. Genomics. 2012;99:323–9.
</cite> [<a href="https://doi.org/10.1016/j.ygeno.2012.04.003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3387489/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22546560/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chen%20X,%20Ishwaran%20H.%20Random%20forests%20for%20genomic%20data%20analysis.%20Genomics.%202012;99:323%E2%80%939." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR49">
<span class="label">49.</span><cite>Tusell L, Bergsma R, Gilbert H, Gianola D, Piles M. Machine learning prediction of crossbred pig feed efficiency and growth rate from single nucleotide polymorphisms. Front Genet. 2020;11: 567818.
</cite> [<a href="https://doi.org/10.3389/fgene.2020.567818" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7775539/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33391339/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tusell%20L,%20Bergsma%20R,%20Gilbert%20H,%20Gianola%20D,%20Piles%20M.%20Machine%20learning%20prediction%20of%20crossbred%20pig%20feed%20efficiency%20and%20growth%20rate%20from%20single%20nucleotide%20polymorphisms.%20Front%20Genet.%202020;11:%20567818." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR50">
<span class="label">50.</span><cite>Liang M, Chang T, An B, Duan X, Du L, Wang X, et al. A stacking ensemble learning framework for genomic prediction. Front Genet. 2021;12: 600040.
</cite> [<a href="https://doi.org/10.3389/fgene.2021.600040" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7969712/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33747037/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Liang%20M,%20Chang%20T,%20An%20B,%20Duan%20X,%20Du%20L,%20Wang%20X,%20et%20al.%20A%20stacking%20ensemble%20learning%20framework%20for%20genomic%20prediction.%20Front%20Genet.%202021;12:%20600040." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR51">
<span class="label">51.</span><cite>Calus MPL. Genomic breeding value prediction: methods and procedures. Animal. 2010;4:157–64.
</cite> [<a href="https://doi.org/10.1017/S1751731109991352" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22443868/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Calus%20MPL.%20Genomic%20breeding%20value%20prediction:%20methods%20and%20procedures.%20Animal.%202010;4:157%E2%80%9364." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR52">
<span class="label">52.</span><cite>Hidalgo J, Lourenco D, Tsuruta S, Masuda Y, Breen V, Hawken R, et al. Investigating the persistence of accuracy of genomic predictions over time in broilers. J Anim Sci. 2021;99:skab239.
</cite> [<a href="https://doi.org/10.1093/jas/skab239" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8420680/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34378776/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hidalgo%20J,%20Lourenco%20D,%20Tsuruta%20S,%20Masuda%20Y,%20Breen%20V,%20Hawken%20R,%20et%20al.%20Investigating%20the%20persistence%20of%20accuracy%20of%20genomic%20predictions%20over%20time%20in%20broilers.%20J%20Anim%20Sci.%202021;99:skab239." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR53">
<span class="label">53.</span><cite>Wolc A, Arango J, Settar P, Fulton JE, O’Sullivan NP, Preisinger R, et al. Persistence of accuracy of genomic estimated breeding values over generations in layer chickens. Genet Sel Evol. 2011;43:23.
</cite> [<a href="https://doi.org/10.1186/1297-9686-43-23" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3144444/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21693035/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wolc%20A,%20Arango%20J,%20Settar%20P,%20Fulton%20JE,%20O%E2%80%99Sullivan%20NP,%20Preisinger%20R,%20et%20al.%20Persistence%20of%20accuracy%20of%20genomic%20estimated%20breeding%20values%20over%20generations%20in%20layer%20chickens.%20Genet%20Sel%20Evol.%202011;43:23." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR54">
<span class="label">54.</span><cite>Habier D, Fernando RL, Dekkers JC. The impact of genetic relationship information on genome-assisted breeding values. Genetics. 2007;177:2389–97.
</cite> [<a href="https://doi.org/10.1534/genetics.107.081190" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2219482/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18073436/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Habier%20D,%20Fernando%20RL,%20Dekkers%20JC.%20The%20impact%20of%20genetic%20relationship%20information%20on%20genome-assisted%20breeding%20values.%20Genetics.%202007;177:2389%E2%80%9397." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR55">
<span class="label">55.</span><cite>Pszczola M, Strabel T, Mulder HA, Calus MPL. Reliability of direct genomic values for animals with different relationships within and to the reference population. J Dairy Sci. 2012;95:389–400.
</cite> [<a href="https://doi.org/10.3168/jds.2011-4338" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22192218/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Pszczola%20M,%20Strabel%20T,%20Mulder%20HA,%20Calus%20MPL.%20Reliability%20of%20direct%20genomic%20values%20for%20animals%20with%20different%20relationships%20within%20and%20to%20the%20reference%20population.%20J%20Dairy%20Sci.%202012;95:389%E2%80%93400." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR56">
<span class="label">56.</span><cite>Pszczola M, Calus MPL. Updating the reference population to achieve constant genomic prediction reliability across generations. Animal. 2016;10:1018–24.
</cite> [<a href="https://doi.org/10.1017/S1751731115002785" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26711815/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Pszczola%20M,%20Calus%20MPL.%20Updating%20the%20reference%20population%20to%20achieve%20constant%20genomic%20prediction%20reliability%20across%20generations.%20Animal.%202016;10:1018%E2%80%9324." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12362903/bin/40104_2025_1250_MOESM1_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link">40104_2025_1250_MOESM1_ESM.docx</a><sup> (15.9KB, docx) </sup><p>Additional file 1. QMSim input file. This file contains the QMSim input file of simulated data.</p>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12362903/bin/40104_2025_1250_MOESM2_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link">40104_2025_1250_MOESM2_ESM.docx</a><sup> (14.7KB, docx) </sup><p>Additional file 2. Parameters that machine learning methods required. This file contains the lists of parameters that used in random forest and support vector regression.</p>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12362903/bin/40104_2025_1250_MOESM3_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link">40104_2025_1250_MOESM3_ESM.docx</a><sup> (297.4KB, docx) </sup><p>Additional file 3. Parameter tuning process of random forest. This file contains the detail of parameter tuning process of random forest.</p>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material4_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12362903/bin/40104_2025_1250_MOESM4_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link">40104_2025_1250_MOESM4_ESM.docx</a><sup> (105.7KB, docx) </sup><p>Additional file 4. Parameter tuning process of support vector regression. This file contains the detail of parameter tuning process of support vector regression.</p>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material5_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12362903/bin/40104_2025_1250_MOESM5_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link">40104_2025_1250_MOESM5_ESM.docx</a><sup> (191KB, docx) </sup><p>Additional file 5: Fig. S1. Prediction accuracy of RF and GBLUP using simulated dataset with different number of informative QTL. a) QTL only, b) QTL combined with SNPs that close to the QTL, and the number of SNPs was twice the number of QTL and c) QTL combined with fixed numberof SNPs. Fig. S2. Real QTL variance against variance or importance obtained from a) GBLUP and b) RF.</p>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material6_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12362903/bin/40104_2025_1250_MOESM6_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link">40104_2025_1250_MOESM6_ESM.docx</a><sup> (72.6KB, docx) </sup><p>Additional file 6: Fig. S1. The accuracy of genomic prediction using different genomic prediction methods for a) 5% of genetic variance explained by QTL and b) 80% of genetic variance explained by QTL across generations.</p>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>QMSim input file used to generate the data during this study is Additional file <a href="#MOESM1" class="usa-link">1</a>. Seeds for QMSim simulation and codes for analysis can be found via <a href="https://git.wageningenur.nl/yang110/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://git.wageningenur.nl/yang110/</a>.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Journal of Animal Science and Biotechnology are provided here courtesy of <strong>BMC</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1186/s40104-025-01250-5"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/40104_2025_Article_1250.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (2.0 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12362903/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12362903/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12362903%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12362903/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12362903/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12362903/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40826369/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12362903/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40826369/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12362903/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12362903/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="3jVKXPhfsJyogfsy6RolJaOKrTmjaxeOj3MmVhGRUEyvDYv5ahvG06l7nZ1m8uWy">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
        <script type="module" crossorigin="" src="/static/assets/math-574fdcc6.js"></script>
    
    

    </body>
</html>
