
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Flavonoid pathway intermediates implicate UVR8 in functions beyond canonical UV-B signaling - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="152543358AF2E33305433500256A252D.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="ncomms">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371036/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Nature Communications">
<meta name="citation_title" content="Flavonoid pathway intermediates implicate UVR8 in functions beyond canonical UV-B signaling">
<meta name="citation_author" content="Nan Jiang">
<meta name="citation_author_institution" content="Department of Biochemistry &amp; Molecular Biology, Michigan State University, East Lansing, MI USA">
<meta name="citation_author_institution" content="Department of Molecular Biosciences and Bioengineering, University of Hawai’i at Mānoa, Honolulu, HI USA">
<meta name="citation_author" content="Tatiana García Navarrete">
<meta name="citation_author_institution" content="Department of Biochemistry &amp; Molecular Biology, Michigan State University, East Lansing, MI USA">
<meta name="citation_author" content="Yun Sun Lee">
<meta name="citation_author_institution" content="Department of Biochemistry &amp; Molecular Biology, Michigan State University, East Lansing, MI USA">
<meta name="citation_author_institution" content="Crop Biotechnology Institute, Institutes of Green-bio Science and Technology, Seoul National University, Pyeongchang, Republic of Korea">
<meta name="citation_author_institution" content="Department of International Agricultural Technology, Seoul National University, Pyeongchang, Republic of Korea">
<meta name="citation_author" content="Emily G Pawlowski">
<meta name="citation_author_institution" content="Department of Biochemistry &amp; Molecular Biology, Michigan State University, East Lansing, MI USA">
<meta name="citation_author" content="Sunyoung Park">
<meta name="citation_author_institution" content="Department of Molecular Biosciences and Bioengineering, University of Hawai’i at Mānoa, Honolulu, HI USA">
<meta name="citation_author" content="Erich Grotewold">
<meta name="citation_author_institution" content="Department of Biochemistry &amp; Molecular Biology, Michigan State University, East Lansing, MI USA">
<meta name="citation_author_institution" content="Department of Plant Biology, Michigan State University, East Lansing, MI USA">
<meta name="citation_publication_date" content="2025 Aug 21">
<meta name="citation_volume" content="16">
<meta name="citation_firstpage" content="7810">
<meta name="citation_doi" content="10.1038/s41467-025-63010-3">
<meta name="citation_pmid" content="40841532">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371036/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371036/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371036/pdf/41467_2025_Article_63010.pdf">
<meta name="description" content="Plants integrate light signals to regulate development and produce protective compounds such as flavonoids under excessive light stress. While these metabolites protect against light damage, their roles in modulating photoreceptor responses remain ...">
<meta name="og:title" content="Flavonoid pathway intermediates implicate UVR8 in functions beyond canonical UV-B signaling">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Plants integrate light signals to regulate development and produce protective compounds such as flavonoids under excessive light stress. While these metabolites protect against light damage, their roles in modulating photoreceptor responses remain ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371036/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12371036">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1038/s41467-025-63010-3"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/41467_2025_Article_63010.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12371036%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12371036/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12371036/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371036/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-ncomms.gif" alt="Nature Communications logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Nature Communications" title="Link to Nature Communications" shape="default" href="http://www.nature.com/ncomms/index.html" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Nat Commun</button></div>. 2025 Aug 21;16:7810. doi: <a href="https://doi.org/10.1038/s41467-025-63010-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1038/s41467-025-63010-3</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Nat%20Commun%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nat%20Commun%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Nat%20Commun%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Nat%20Commun%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Flavonoid pathway intermediates implicate UVR8 in functions beyond canonical UV-B signaling</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Jiang%20N%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Nan Jiang</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Nan Jiang</span></h3>
<div class="p">
<sup>1</sup>Department of Biochemistry &amp; Molecular Biology, Michigan State University, East Lansing, MI USA </div>
<div class="p">
<sup>2</sup>Department of Molecular Biosciences and Bioengineering, University of Hawai’i at Mānoa, Honolulu, HI USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Jiang%20N%22%5BAuthor%5D" class="usa-link"><span class="name western">Nan Jiang</span></a>
</div>
</div>
<sup>1,</sup><sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Garc%C3%ADa%20Navarrete%20T%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Tatiana García Navarrete</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Tatiana García Navarrete</span></h3>
<div class="p">
<sup>1</sup>Department of Biochemistry &amp; Molecular Biology, Michigan State University, East Lansing, MI USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Garc%C3%ADa%20Navarrete%20T%22%5BAuthor%5D" class="usa-link"><span class="name western">Tatiana García Navarrete</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lee%20YS%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Yun Sun Lee</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Yun Sun Lee</span></h3>
<div class="p">
<sup>1</sup>Department of Biochemistry &amp; Molecular Biology, Michigan State University, East Lansing, MI USA </div>
<div class="p">
<sup>3</sup>Crop Biotechnology Institute, Institutes of Green-bio Science and Technology, Seoul National University, Pyeongchang, Republic of Korea </div>
<div class="p">
<sup>4</sup>Department of International Agricultural Technology, Seoul National University, Pyeongchang, Republic of Korea </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lee%20YS%22%5BAuthor%5D" class="usa-link"><span class="name western">Yun Sun Lee</span></a>
</div>
</div>
<sup>1,</sup><sup>3,</sup><sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Pawlowski%20EG%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Emily G Pawlowski</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Emily G Pawlowski</span></h3>
<div class="p">
<sup>1</sup>Department of Biochemistry &amp; Molecular Biology, Michigan State University, East Lansing, MI USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Pawlowski%20EG%22%5BAuthor%5D" class="usa-link"><span class="name western">Emily G Pawlowski</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Park%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Sunyoung Park</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Sunyoung Park</span></h3>
<div class="p">
<sup>2</sup>Department of Molecular Biosciences and Bioengineering, University of Hawai’i at Mānoa, Honolulu, HI USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Park%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Sunyoung Park</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Grotewold%20E%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Erich Grotewold</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Erich Grotewold</span></h3>
<div class="p">
<sup>1</sup>Department of Biochemistry &amp; Molecular Biology, Michigan State University, East Lansing, MI USA </div>
<div class="p">
<sup>5</sup>Department of Plant Biology, Michigan State University, East Lansing, MI USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Grotewold%20E%22%5BAuthor%5D" class="usa-link"><span class="name western">Erich Grotewold</span></a>
</div>
</div>
<sup>1,</sup><sup>5,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>Department of Biochemistry &amp; Molecular Biology, Michigan State University, East Lansing, MI USA </div>
<div id="Aff2">
<sup>2</sup>Department of Molecular Biosciences and Bioengineering, University of Hawai’i at Mānoa, Honolulu, HI USA </div>
<div id="Aff3">
<sup>3</sup>Crop Biotechnology Institute, Institutes of Green-bio Science and Technology, Seoul National University, Pyeongchang, Republic of Korea </div>
<div id="Aff4">
<sup>4</sup>Department of International Agricultural Technology, Seoul National University, Pyeongchang, Republic of Korea </div>
<div id="Aff5">
<sup>5</sup>Department of Plant Biology, Michigan State University, East Lansing, MI USA </div>
<div class="author-notes p"><div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div></div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2024 Jul 10; Accepted 2025 Aug 6; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by-nc-nd/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12371036  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40841532/" class="usa-link">40841532</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<p id="Par1">Plants integrate light signals to regulate development and produce protective compounds such as flavonoids under excessive light stress. While these metabolites protect against light damage, their roles in modulating photoreceptor responses remain unclear. Here, we show that naringenin chalcone (NGC), a flavonoid precursor that accumulates in the <em>Arabidopsis chalcone isomerase</em> mutant <em>tt5</em> but which is also present in wild type plants under solar light, promotes UVR8 monomer accumulation beyond its canonical UV-B response. A genetic suppressor screen reveals that loss of <em>UVR8</em> restores normal growth of <em>tt5</em> mutants grown under high light, irrespective of the presence of UV-B. Biochemical and transcriptomic analyses show that NGC binds to monomeric UVR8, stabilizing its active form and triggering gene expression changes, even in the absence of UV-B. This work identifies a mechanism for stabilizing activated UVR8 and reveals crosstalk between light-induced metabolic intermediates and photoreceptor-mediated developmental regulation.</p>
<section id="kwd-group1" class="kwd-group"><p><strong>Subject terms:</strong> Secondary metabolism, Plant genetics, Plant development, Plant molecular biology</p></section></section><section class="abstract" id="Abs2"><hr class="headless">
<p id="Par2">Whether excessive light activated photo-protective compounds accumulation functions as sunscreen or participating in modulating photoreceptor responses in plants is still unclear. Here, the authors provide evidences to support the role of flavonoid pathway intermediates in integration of light signals into plant development.</p></section><section id="Sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p id="Par3">Plants obtain their energy from sunlight through photosynthesis, and photosynthetically active radiation is normally considered in the 400–700 nm range<sup><a href="#CR1" class="usa-link" aria-describedby="CR1">1</a></sup>. Plants sense light composition through a cadre of photoreceptors, which ultimately integrate light information with other environmental signals (such as temperature) into precise developmental and metabolic responses<sup><a href="#CR2" class="usa-link" aria-describedby="CR2">2</a>–<a href="#CR5" class="usa-link" aria-describedby="CR5">5</a></sup>. Plant photoreceptors include light-oxygen voltage sensors, phytochromes, flavin-binding blue-light sensors, and cryptochromes. Each photoreceptor absorbs a portion of the solar radiation spectrum that reaches Earth (280–750 nm), often through associated chromophores that serve as photon absorption sites<sup><a href="#CR3" class="usa-link" aria-describedby="CR3">3</a></sup>. For example, phytochromes perceive red/far red (R/FR) light (600–750 nm); cryptochromes, phototropins and flavin-binding F-box proteins perceive blue/ultraviolet-A (UV-A) light (320–500 nm); and UV RESISTANCE LOCUS8 (UVR8) perceives UV-A (315–400 nm) and UV-B light (280–315 nm)<sup><a href="#CR3" class="usa-link" aria-describedby="CR3">3</a>,<a href="#CR5" class="usa-link" aria-describedby="CR5">5</a></sup>. The photon contribution of UV light is about 10% of the photosynthetically active radiation, with 95% or more corresponding to UV-A, although geographical location, time of day, cloud and canopy coverage significantly affect the relative influence of UV-B<sup><a href="#CR6" class="usa-link" aria-describedby="CR6">6</a></sup>.</p>
<p id="Par4">UV-B, a minor yet unavoidable component of solar light, can have detrimental effects on plants, including DNA damage primarily through the formation of cyclobutane-pyrimidine dimers, inhibition of the photosynthetic machinery inhibition (<em>e.g</em>., degradation of photosystem II components), and the induction of reactive oxygen species<sup><a href="#CR2" class="usa-link" aria-describedby="CR2">2</a></sup>, which oxidize lipids and other cellular components<sup><a href="#CR7" class="usa-link" aria-describedby="CR7">7</a>–<a href="#CR9" class="usa-link" aria-describedby="CR9">9</a></sup>. The mechanisms evolved by plants to minimize the deleterious UV-B effects include the photo repair of UV-induced pyrimidine dimers through a process known as photoreactivation<sup><a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>,<a href="#CR10" class="usa-link" aria-describedby="CR10">10</a></sup>; the induction of antioxidant enzymes and compounds<sup><a href="#CR11" class="usa-link" aria-describedby="CR11">11</a></sup>; and the accumulation of UV-absorbing compounds including hydroxycinnamate esters and flavonoids, phenolic compounds that accumulate primarily in epidermal cells in response to UV-B radiation and absorb a significant fraction of the UV light, minimizing damage to deeper tissue layers<sup><a href="#CR7" class="usa-link" aria-describedby="CR7">7</a>,<a href="#CR9" class="usa-link" aria-describedby="CR9">9</a>,<a href="#CR12" class="usa-link" aria-describedby="CR12">12</a></sup>. Indeed, <em>Arabidopsis</em> mutants lacking sinapate esters because of a mutation in the ferulic acid hydroxylase (<em>fah1</em>) gene, or flavonoids resulting from loss of CHALCONE ISOMERASE (CHI, encoded by <em>TT5</em>, <em>TRANSPARENT TESTA5</em>) are significantly more sensitive to UV-B radiation<sup><a href="#CR13" class="usa-link" aria-describedby="CR13">13</a></sup>. CHI is responsible for the conversion of naringenin chalcone (NGC) produced by CHALCONE SYNTHASE (CHS) encoded by <em>TT4</em> in <em>Arabidopsis</em>, to the flavanone naringenin, a central intermediate in the biosynthesis of flavonols and anthocyanin pigments<sup><a href="#CR14" class="usa-link" aria-describedby="CR14">14</a></sup>. <em>Arabidopsis tt5</em> mutants grown in conditions that mimic solar light showed growth defects that were partially ameliorated by filtering UV-B light<sup><a href="#CR15" class="usa-link" aria-describedby="CR15">15</a></sup>. The sensitivity of <em>tt5</em> to UV-B was used for a screen of mutants that displayed enhanced UV-B sensitivity, resulting in the identification of UVR8<sup><a href="#CR16" class="usa-link" aria-describedby="CR16">16</a></sup>. UVR8 not only controls UV-B stimulated protective responses, including flavonoid biosynthesis<sup><a href="#CR17" class="usa-link" aria-describedby="CR17">17</a></sup>, but it functions as a UV-B photoreceptor participating in a number of photomorphogenic processes<sup><a href="#CR18" class="usa-link" aria-describedby="CR18">18</a>,<a href="#CR19" class="usa-link" aria-describedby="CR19">19</a></sup>. Under normal growth conditions, UVR8 is majorly present as an inactive dimer in the cytoplasm<sup><a href="#CR20" class="usa-link" aria-describedby="CR20">20</a></sup>. Upon UV-B absorption, UVR8 rapidly monomerizes and translocates to the nucleus, where it interacts with several proteins, including CONSTITUTIVELY PHOTOMORPHOGENIC1 (COP1), a component of the E3 ligase complex, inhibiting the COP1-mediated degradation of the ELONGATED HYPOCOTYL5 (HY5) transcription factor, in part responsible for the induction of various phenolic compounds and anthocyanins<sup><a href="#CR18" class="usa-link" aria-describedby="CR18">18</a>,<a href="#CR21" class="usa-link" aria-describedby="CR21">21</a>–<a href="#CR24" class="usa-link" aria-describedby="CR24">24</a></sup>. Two related WD40 domain proteins, REPRESSOR OF UV-B PHOTOMORPHOGENESIS1 (RUP1) and RUP2, negatively and redundantly regulate UVR8 activity by promoting reversion to the dimer<sup><a href="#CR20" class="usa-link" aria-describedby="CR20">20</a>,<a href="#CR25" class="usa-link" aria-describedby="CR25">25</a></sup>. Disruption of this UVR8 photo-equilibrium, for example, by strong overexpression of a constitutive monomeric version of UVR8, results in strong photomorphogenic phenotypes, even in the dark<sup><a href="#CR26" class="usa-link" aria-describedby="CR26">26</a></sup>. Thus, maintaining the appropriate monomeric/dimeric UVR8 balance is central for proper plant development.</p>
<p id="Par5">Here, we describe a mechanism by which this balance is disrupted by flavonoid pathway intermediates that accumulate in <em>tt5</em> mutants when exposed to moderate high light (HL, ~ 180 μmol cm<sup>−2 </sup>s<sup>−1</sup>) in a growth chamber, resulting in a plethora of developmental defects. Our studies are prompted by the finding that <em>tt5</em> mutants grown in a chamber lacking FR light exhibit stunted growth and several phenotypes, besides the absence of anthocyanin pigments. Under these conditions, wild-type Col-0 plants strongly induce anthocyanin accumulation, and we determine that the <em>tt5</em> developmental phenotypes are recapitulated in HL but disappear when we cross <em>tt4</em> into the <em>tt5</em> mutant, indicating that NGC or its derivatives are responsible. Transcriptome analyses identify many genes specifically affected by HL in <em>tt5</em>, most significantly genes involved in the biosynthesis and response to salicylic acid (SA). High levels of SA and SA glucoside are confirmed by liquid chromatography coupled with tandem mass spectrometry (LC-MS/MS). To identify molecular factors impacted by NGC, we screen for genetic suppressors that restore the developmental phenotypes of <em>tt5</em> mutants under HL conditions. We identify and characterize five suppressors, two corresponding to <em>tt4</em>, and three to loss-of-function alleles of <em>UVR8</em>. The development defects of <em>tt5</em> in HL continue to be present even when UV-B light is eliminated, suggesting a function of UVR8 in plant development that extends beyond the effects promoted by UV-B. We show that <em>tt5</em> plants grown under HL have significantly increased levels of monomeric UVR8 protein, and that the stabilization of the monomer is recapitulated in vitro using pure NGC or methanol extracts of <em>tt5</em> plants grown in HL. Using purified recombinant UVR8, nano-differential scanning fluorimetry (nanoDSF) and microscale thermophoresis (MST), we show that NGC, but not other flavonoids, binds to UVR8 monomers, but not to dimers. Gene expression analyses by RNA-sequencing (RNA-seq) comparing mutant and wild-type <em>Arabidopsis</em> plants identify UV-B independent <em>UVR8</em>-controlled genes that overlap with those induced by HL in <em>tt5</em>, indicating that NGC (or derivatives) modulate an UVR8 activity that is unrelated to UV-B. Though nearly undetectable under artificial growth conditions, substantial levels of NGC derivatives are observed in <em>Arabidopsis</em> wild-type plants exposed to sunlight, highlighting the potential role of flavonoid pathway intermediates in modulating UVR8 activity-likely as part of light signal integration.</p></section><section id="Sec2"><h2 class="pmc_sec_title">Results</h2>
<section id="Sec3"><h3 class="pmc_sec_title">Developmental defects of <em>Arabidopsis tt5</em> mutants in high light</h3>
<p id="Par6">Serendipitously, we grew wild-type (WT, Col-0) and <em>tt5</em> mutant plants in two different growth chambers under similar low light intensities (70–80 μmol cm<sup>−2 </sup>s<sup>−1</sup>). After approximately five weeks of growth, we noticed that, while in one chamber, <em>tt5</em> mutants were indistinguishable from WT (Fig. <a href="#Fig1" class="usa-link">1a</a>), in the other, <em>tt5</em> mutant plants (two independent alleles, <em>tt5-2</em> and <em>tt5-4</em>) were stunted and showed a variety of abnormal phenotypes (Fig. <a href="#Fig1" class="usa-link">1b</a> and Supplementary Table <a href="#MOESM1" class="usa-link">1</a>). The abnormal growth phenotypes were not observed in <em>tt4</em> (<em>tt4-11</em>) mutant plants lacking the CHS activity that produces NGC, the substrate of TT5/CHI (Supplementary Fig. <a href="#MOESM1" class="usa-link">1</a>). When we analyzed the light spectra of both chambers, we determined that the second chamber had less red light and almost no far-red light (Fig. <a href="#Fig1" class="usa-link">1c, d</a>, compare 600–780 nm wavelength profiles). This resulted from the absence of incandescent bulbs (see “Methods”), highlighting the importance of checking light spectra and intensity before conducting experiments. We call the first chamber the “High R + FR chamber” (Fig. <a href="#Fig1" class="usa-link">1a, c</a>), and the second as “Low R + FR chamber” (Fig. <a href="#Fig1" class="usa-link">1b, d</a>). Our results show that the R/FR imbalance present in the second chamber (Fig. <a href="#Fig1" class="usa-link">1b</a>) efficiently promotes anthocyanin accumulation in WT plants. Both R and FR lights were shown to induce anthocyanin accumulation in <em>Arabidopsis</em><sup><a href="#CR27" class="usa-link" aria-describedby="CR27">27</a></sup>; the inhibitory effect of FR on the anthocyanin-inductive effect of R light was not anticipated. Alterations in the R/FR ratio are also known to cause several growth and developmental phenotypes in <em>Arabidopsis</em><sup><a href="#CR28" class="usa-link" aria-describedby="CR28">28</a></sup>; the fact that we only see the effects in <em>tt5-2</em> and <em>tt5-4</em> indicates that this is not a general phytochrome response, but rather that something is unique to these mutants.</p>
<figure class="fig xbox font-sm" id="Fig1"><h4 class="obj_head">Fig. 1. <em>Arabidopsis</em><em>tt5</em> mutants exhibit morphological phenotypes.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371036_41467_2025_63010_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9400/12371036/afe6a4749c9e/41467_2025_63010_Fig1_HTML.jpg" loading="lazy" id="d33e613" height="658" width="683" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Wild type (Col-0), <em>tt4-11</em>, <em>tt5-2</em>, and <em>tt5-4</em> plants were grown for 5 weeks in one chamber. <strong>b</strong> Images of plants with the same genotypes as (<strong>a</strong>), recorded at 5 weeks under another chamber. <em>tt5</em> mutants demonstrate development defects. Scale bar: 1 cm. <strong>c</strong>, <strong>d</strong> Light spectra of the growth conditions in (<strong>a</strong>) and (<strong>b</strong>), respectively. PFD-near UV near-UV (380–400 nm); PFD-B blue wavelength (400–500 nm); PFD-G green wavelength (500–600 nm); PFD-R red wavelength (600–700 nm); PFD-FR far-red wavelength (700–780 nm); PFD photon flux density; R red wavelength; FR far-red wavelength; PPFD photosynthetic photon flux density. Source data are provided as a Source Data file.</p></figcaption></figure><p id="Par7">Because WT plants in the Low R + FR chamber accumulated significant amounts of anthocyanins (Fig. <a href="#Fig1" class="usa-link">1b</a>), we reasoned that high light conditions, known to induce anthocyanin accumulation<sup><a href="#CR22" class="usa-link" aria-describedby="CR22">22</a></sup>, could phenocopy the observed effect of low R + FR light. When we grew plants in full-spectrum moderate-high light (HL, ~ 180 μmol cm<sup>−2</sup> s<sup>−1</sup>, Supplementary Fig. <a href="#MOESM1" class="usa-link">2</a>), we found that <em>tt5-2</em> plants displayed identical phenotypes (Supplementary Fig. <a href="#MOESM1" class="usa-link">3</a>) as observed in the Low R + FR chamber. Expression in <em>tt5-2</em> of the <em>TT5</em> coding region fused to mCherry, driven by the native <em>TT5</em> promoter<sup><a href="#CR29" class="usa-link" aria-describedby="CR29">29</a></sup>, fully complemented the developmental phenotypes of <em>tt5-2</em> in HL (Supplementary Fig. <a href="#MOESM1" class="usa-link">3</a>), indicating that the observed phenotypes are due to <em>tt5</em> and not another mutation. Moreover, these phenotypes were not observed in <em>tt4-11</em> plants, and they were suppressed by crossing <em>tt4-11</em> into <em>tt5-2</em> (Supplementary Fig. <a href="#MOESM1" class="usa-link">3</a>). These results demonstrate that the phenotypes observed for <em>tt5</em> under HL, which included early senescence, reduced inflorescence height and internode length, altered phyllotaxis of flowers and siliques, and reduced pedicel length (Supplementary Fig. <a href="#MOESM1" class="usa-link">4</a> and Supplementary Table <a href="#MOESM1" class="usa-link">1</a>), result from the accumulation of a CHS (TT4) product that cannot be further metabolized in the <em>tt5</em> mutants. To further test this, we expressed the <em>Antirrhinum majus</em> chalcone 4’-<em>O</em>-glucosyltransferase (4’CGT, an activity normally not present in <em>Arabidopsis</em>, Supplementary Fig. <a href="#MOESM1" class="usa-link">1</a>), which uses NGC as substrate<sup><a href="#CR30" class="usa-link" aria-describedby="CR30">30</a></sup>, in <em>tt5-2</em> plants under the constitutive <em>CaMV 35S</em> promoter (p35S::Am4’CGT) and found that the developmental phenotypes characteristic of <em>tt5</em> in HL disappeared (Supplementary Fig. <a href="#MOESM1" class="usa-link">5</a>). We conclude that NGC or a derivative that accumulates in <em>tt5</em> mutants is responsible for the developmental phenotypes observed in <em>tt5</em> under HL.</p></section><section id="Sec4"><h3 class="pmc_sec_title">HL induces major transcriptome changes in <em>tt5</em>
</h3>
<p id="Par8">To determine the gene expression changes potentially responsible for the developmental phenotypes of <em>tt5</em> under HL, we conducted RNA-seq on 3- and 4-week-old Col-0, <em>tt4-11</em>, <em>tt5-2</em>, and <em>tt4-11 tt5-2</em> plants under both moderate high light (HL, ~ 180 μmol cm<sup>−2 </sup>s<sup>−1</sup>) and normal light (NL, ~ 80 μmol cm<sup>−2 </sup>s<sup>−1</sup>). Plants grown three weeks in HL conditions show none of the developmental defects associated with <em>tt5</em> (Supplementary Fig. <a href="#MOESM1" class="usa-link">6a</a>) which start becoming evident four weeks in HL conditions (Supplementary Fig. <a href="#MOESM1" class="usa-link">6b</a>). We first conducted differentially expressed gene (DEG) analyses between each genotype grown in HL versus NL for three (Fig. <a href="#Fig2" class="usa-link">2a</a>) and four weeks (Fig. <a href="#Fig2" class="usa-link">2b</a>). The majority of the DEGs were shared between the three genotypes, yet this is more pronounced at three weeks (2287 + 2206 = 4493 genes; Fig. <a href="#Fig2" class="usa-link">2a</a>) with about half that number (516 + 1397 = 1913; Fig. <a href="#Fig2" class="usa-link">2b</a>) at four weeks.</p>
<figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2. <em>Arabidopsis</em> plants were subjected to RNA-seq analysis and to salicylic acid accumulation measurements under both high light (HL) and normal light (NL) conditions.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371036_41467_2025_63010_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9400/12371036/c004303e5d55/41467_2025_63010_Fig2_HTML.jpg" loading="lazy" id="d33e847" height="979" width="760" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Differential expressed genes (DEGs) and gene ontology (GO) enrichment analysis were conducted on RNA-seq results. The Venn diagrams illustrate shared and unique DEGs among Col-0 HL vs Col-0 NL, <em>tt4-11</em> HL vs <em>tt4-11</em> NL, and <em>tt5-2</em> HL vs <em>tt5-2</em> NL for 3 (<strong>a</strong>) or 4 weeks (<strong>b</strong>). We conducted GO enrichment analysis of the DEGs unique to <em>tt5-2</em>. The <em>x</em>-axis indicates the fold enrichment for each biological process GO term. The top ten most significantly enriched GO terms are displayed. Red bars represent GO-term enrichment of up-regulated genes, while blue bars represent GO-term enrichment of down-regulated genes. <strong>c</strong> DEGs and GO enrichment analysis. Comparisons were made between unique DEGs in <em>tt5-2</em> at 3 and 4 weeks. <strong>d</strong> Salicylic acid accumulation. Quantification of salicylic acid (SA) was performed using targeted LC-MS/MS. The amount of SA (left panel, nmol g<sup>−1</sup> dry weight) was assessed under normal light (gray bars) and high light (red bars) conditions. Experiments were conducted in biological triplicate, with each replicate corresponding to compounds extracted from four plants of the same genetic background. Error bars represent the standard deviation of the average, and <em>p</em>-values were calculated using a two-sided <em>t-</em>test. <em>P</em>-values between Col-0 NL vs Col-0 HL, <em>tt4-11</em> NL vs <em>tt4-11</em> HL, <em>tt5-2</em> NL vs <em>tt5-2</em> HL, and <em>tt4-11 tt5-2</em> NL vs <em>tt4-11 tt5-2</em> HL are 0.115502, &lt;0.00001, 0.000021, and 0.000541, respectively. Source data are provided as a Source Data file.</p></figcaption></figure><p id="Par9">Unique sets of DEGs characterized the response of Col-0, <em>tt4-11</em> and <em>tt5-2</em> to HL (Fig. <a href="#Fig2" class="usa-link">2a, b</a> and Supplementary Data <a href="#MOESM4" class="usa-link">1</a> and <a href="#MOESM5" class="usa-link">2</a>). One group of genes that was notably induced solely in <em>tt5-2</em> under HL conditions both at three weeks (DEG3, Fig. <a href="#Fig2" class="usa-link">2a</a>) and four weeks (DEG7, Fig. <a href="#Fig2" class="usa-link">2b</a>) with a notable overlap (Fig. <a href="#Fig2" class="usa-link">2c</a> and Supplementary Data <a href="#MOESM6" class="usa-link">3</a>) between both time points corresponded to those related to the immune response, and particularly the response to salicylic acid (SA)<sup><a href="#CR31" class="usa-link" aria-describedby="CR31">31</a></sup>. These DEGs were differentially expressed in <em>tt5-2</em> by HL compared to DEGs in the <em>tt4-11 tt5-2</em> double mutant (Supplementary Figs. <a href="#MOESM1" class="usa-link">7a, b</a> and Supplementary Data <a href="#MOESM7" class="usa-link">4</a> and <a href="#MOESM8" class="usa-link">5</a>), suggesting that they are induced by a pathway metabolic product, and not by the absence of the CHI enzyme or reduced flux through the flavonol/anthocyanin pathway. To determine whether 3-week-old <em>tt5-2</em> plants under HL conditions had increased amounts of SA and SA glucoside (SAG), we quantified levels by conducting targeted LC-MS/MS on extracts obtained from the samples used for RNA-seq. Our results show that HL had no significant effect in the accumulation of SA or SAG in WT (<em>P</em> &gt; 0.01, two-sided <em>t-</em>test), but significantly reduced SA and SAG accumulation in <em>tt4-11</em> (<em>P</em> &lt; 0.01, two-sided <em>t-</em>test; Fig. <a href="#Fig2" class="usa-link">2d</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">7c</a>). Consistent with the RNA-seq results, the levels of both SA and SAG were significantly increased (<em>P</em> &lt; 0.01, two-sided <em>t-</em>test) in <em>tt5-2</em>. SA and SAG levels in the <em>tt4-11 tt5-2</em> double mutant were comparable to those in <em>tt4-11</em>, providing evidence that this increase in SA and SAG is a consequence of the increased accumulation of NGC or its derivative (Fig. <a href="#Fig2" class="usa-link">2d</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">7c</a>). Overaccumulation of SA to levels comparable to those present in <em>tt5</em> under HL were shown to induce cell death and dwarfism<sup><a href="#CR32" class="usa-link" aria-describedby="CR32">32</a>–<a href="#CR34" class="usa-link" aria-describedby="CR34">34</a></sup>, like the phenotypes observed in the <em>tt5-2</em> mutants grown under HL (Supplementary Fig. <a href="#MOESM1" class="usa-link">3</a>). Thus, at this point we could not rule out the possibility that the growth and developmental phenotypes observed in <em>tt5-2</em> under HL could result from the high SA levels particular to this mutant.</p></section><section id="Sec5"><h3 class="pmc_sec_title">
<em>UVR8</em> mutations suppress the <em>tt5</em> HL phenotypes</h3>
<p id="Par10">We reasoned that NGC or its derivative interacts with one or more cellular components, resulting in the observed developmental phenotypes. To identify such cellular components, we embarked on identifying suppressors that can prevent the early leaf senescence phenotype of <em>tt5-2</em> in HL. We mutagenized <em>tt5-2</em> seeds with ethyl methyl sulfate (EMS) and screened ~ 10,000 M<sub>2</sub> plants for the absence of early leaf senescence by growing plants in HL conditions for five weeks (Supplementary Fig. <a href="#MOESM1" class="usa-link">8</a>). We identified 15 plants that phenotypically looked like Col-0 yet lacked anthocyanin pigmentation. We predicted that some suppressors would abolish the accumulation of NGC. To establish this, we extracted phenolic compounds from control (Col-0, <em>tt4-11</em>, and <em>tt5-2</em>) and M<sub>2</sub> suppressor lines grown for four weeks in HL, acid-hydrolyzed the extracts to simplify interpretation since NGC and most its derivatives are converted to naringenin after hydrolysis<sup><a href="#CR35" class="usa-link" aria-describedby="CR35">35</a>–<a href="#CR37" class="usa-link" aria-describedby="CR37">37</a></sup>, and analyzed them by untargeted LC-MS/MS using a commercial standard of naringenin as reference (Supplementary Fig. <a href="#MOESM1" class="usa-link">9a, b</a>). As expected, <em>tt5-2</em> accumulates significant levels of the <em>m/z</em> 273.08 peak characteristic of naringenin in hydrolyzed extracts (derived from the conversion of NGC to naringenin during the hydrolysis), which is absent in <em>tt4-11</em> and very low in Col-0 (Supplementary Fig. <a href="#MOESM1" class="usa-link">9c</a>). From the 15 suppressor lines, ten lines showed no naringenin accumulation after hydrolysis. From the remaining five lines accumulating naringenin after hydrolysis, three (S1, S20 and S32) showed naringenin levels comparable to <em>tt5-2</em>, while two (S17 and S33) showed naringenin levels higher than Col-0, but lower than <em>tt5-2</em> (Supplementary Fig. <a href="#MOESM1" class="usa-link">9c</a>). We back-crossed these five suppressor lines to <em>tt5-2</em> to perform bulked segregant analysis by whole genome re-sequencing (Supplementary Fig. <a href="#MOESM1" class="usa-link">8</a>) and determined that the causative mutations were in two distinct chromosomal regions (Supplementary Fig. <a href="#MOESM1" class="usa-link">10</a>). S17 and S33 showed missense mutations in At5g13930 (<em>TT4</em>) (Supplementary Fig. <a href="#MOESM1" class="usa-link">11</a>), consistent with these two lines showing reduced naringenin levels after hydrolysis (Supplementary Fig. <a href="#MOESM1" class="usa-link">9c</a>) and suggesting that the low levels of NGC or its derivative present in these two suppressor lines is insufficient to cause the developmental defects observed in <em>tt5-2</em> under HL. The other three suppressor lines (S1, S20 and S32) contained mutations in At5g63860 (UVR8, UV RESISTANCE LOCUS8), with the S1 and S20 mutations predicted to cause splicing defects at the 3’ end of introns 4 and 2 respectively, in which the normal splicing acceptor sites were mutated, and S32 harboring replacement of R55 with a stop codon (Fig. <a href="#Fig3" class="usa-link">3a</a>). Thus, we named the <em>uvr8</em> alleles <em>uvr8-31</em> (S1), <em>uvr8-32</em> (S20), and <em>uvr8-</em>33 (S32). These results demonstrate that <em>uvr8</em> mutations can abolish the developmental phenotypes of <em>tt5</em> mutants grown in HL (Fig. <a href="#Fig3" class="usa-link">3b</a>) and suggest that UVR8 is a potential target (direct or indirect) of NGC or its derivatives.</p>
<figure class="fig xbox font-sm" id="Fig3"><h4 class="obj_head">Fig. 3. Identification of mutations in the UV-B receptor UVR8 as <em>tt5-2</em> suppressors.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371036_41467_2025_63010_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9400/12371036/b9d9051f7f12/41467_2025_63010_Fig3_HTML.jpg" loading="lazy" id="d33e1124" height="842" width="750" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Structure of the <em>UVR8</em> gene indicating the position and nature of the EMS-induced mutations for the S1, S20 and S32 suppressors. The S1 and S20 mutations cause splicing defects at the 3’ end of introns 4 and 2, respectively. The S32 suppressor harbors a replacement of R55 with a stop codon. <strong>b</strong> Pictures of <em>tt5-2</em> plants and the three <em>tt5</em> suppressors lines, <em>tt5</em> S1 (<em>tt5-2 uvr8-31</em>), <em>tt5</em> S20 (<em>tt5-2 uvr8-32</em>), and <em>tt5</em> S32 (<em>tt5-2 uvr8-33</em>), under high light conditions for 5 weeks. The adaxial and abaxial surfaces of the rosette leaves are observed. Scale bar: 1 cm.</p></figcaption></figure><p id="Par11">UVR8 is primarily recognized as an UV-A and UV-B receptor<sup><a href="#CR5" class="usa-link" aria-describedby="CR5">5</a></sup>, hence we asked whether UV-A/B is necessary for this activity of UVR8. We determined that our initial HL conditions (Supplementary Fig. <a href="#MOESM1" class="usa-link">2</a>) contained low but significant quantities of UV-A/B (280–400 nm; ~ 50 μW/cm<sup>−2</sup>). Thus, we grew Col-0, <em>tt4-11</em>, <em>tt5-2</em>, and <em>tt4-11 tt5-2</em> plants for five weeks in identical HL conditions with and without a filter that absorbs all UV-A/B radiation (Supplementary Fig. <a href="#MOESM1" class="usa-link">12</a>). No difference in the phenotypes of the plants with and without UV-A/B was observed (Supplementary Fig. <a href="#MOESM1" class="usa-link">12</a>) indicating that the participation of UVR8 in the effects of HL on <em>tt5</em> are independent of UV-A/B.</p>
<p id="Par12">Given that <em>uvr8</em> mutations suppress the early senescence and growth phenotype of <em>tt5</em> plants grown in HL, we asked whether the elevated SA and SAG levels remained high in the suppressor lines, or whether they returned to normal levels. Thus, we analyzed SA and SAG by LC-MS/MS and determined that the suppressor lines grown in HL had SA and SAG levels comparable to those of <em>tt5</em> plants (Supplementary Fig. <a href="#MOESM1" class="usa-link">13</a>). These results indicate that the developmental phenotypes <em>tt5</em> plants in HL conditions are not a direct consequence of the enhanced accumulation of SA or SAG but rather depend on <em>UVR8</em> activity.</p></section><section id="Sec6"><h3 class="pmc_sec_title">Increased monomeric UVR8 in <em>tt5</em> plants</h3>
<p id="Par13">To explore the connections between UVR8 function, NGC or its derivative accumulation in <em>tt5</em> grown in HL, and the <em>tt5</em> phenotypes, we collected leaf samples from Col-0, <em>tt4-11</em>, <em>tt5-2</em>, <em>tt4-11 tt5-2</em>, <em>tt5-2 uvr8-31</em>, <em>tt5-2 uvr8-32</em>, and <em>tt5-2 uvr8-33</em> grown for 3, 4, or 5 weeks in HL and NL without UV-A/B (Supplementary Fig. <a href="#MOESM1" class="usa-link">14</a>) for UVR8 analysis. UVR8 monomeric and dimeric forms were analyzed using a UVR8 antibody (α-UVR8) that recognizes the UVR8 dimer only if the polyacrylamide gel is treated with UV-B prior to transfer to the nylon membrane<sup><a href="#CR18" class="usa-link" aria-describedby="CR18">18</a></sup>. This commercial UVR8 antibody recognizes the C-terminal region of the protein (see Methods). Consistent with the molecular analysis of the <em>uvr8-31</em>, <em>uvr8-32</em>, and <em>uvr8-33</em> mutants (Fig. <a href="#Fig4" class="usa-link">4a</a>), no protein was detected in <em>Arabidopsis</em> plants harboring these alleles (lanes 5–7, 12–14 and 19–21, Fig. <a href="#Fig4" class="usa-link">4a</a>). We found that the amount of total UVR8 protein did not change between Col-0, <em>tt4-11</em>, <em>tt5-2</em>, <em>tt4-11 tt5-2</em>, regardless of whether plants were grown for 3, 4, or 5 weeks in HL (Fig. <a href="#Fig4" class="usa-link">4a</a>, denaturing). When the proteins were separated in non-denaturing conditions and the polyacrylamide gel was treated with UV-B to expose the epitope in the dimer (Non-denaturing, Gel: + UV-B, Fig. <a href="#Fig4" class="usa-link">4a</a>), the antibody detected the UVR8 dimer (indicated by D in Fig. <a href="#Fig4" class="usa-link">4a</a>), one band likely non-specific as it was present also in the mutants (indicated with open arrow, Fig. <a href="#Fig4" class="usa-link">4a</a>), and two bands of increased mobility (indicated by M<sub>1</sub> and M<sub>2</sub>, Fig. <a href="#Fig4" class="usa-link">4a</a>), specifically present in <em>tt5-2</em> plants after 4 weeks of HL exposure (line 10, Fig. <a href="#Fig4" class="usa-link">4a</a>), and with intensities that further increased at 5 weeks of HL (lines 17, Fig. <a href="#Fig4" class="usa-link">4a</a>). Based on our results and the literature, M<sub>1</sub> and M<sub>2</sub> correspond to monomeric UVR8<sup><a href="#CR18" class="usa-link" aria-describedby="CR18">18</a>,<a href="#CR25" class="usa-link" aria-describedby="CR25">25</a>,<a href="#CR38" class="usa-link" aria-describedby="CR38">38</a></sup> which can adopt different conformations<sup><a href="#CR39" class="usa-link" aria-describedby="CR39">39</a></sup>. The UVR8 monomer was barely detectable in WT plants grown under HL (lines 8 and 15, Fig. <a href="#Fig4" class="usa-link">4a</a>; see Supplementary Fig. <a href="#MOESM1" class="usa-link">15a</a> for longer exposure), and it was undetectable in WT or <em>tt5-2</em> plants grown under NL (Fig. <a href="#Fig4" class="usa-link">4a</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">15a</a>). These results suggest that NGC or its derivative promotes UVR8 monomer formation by destabilizing the dimer, stabilizing the monomer, preventing it from reforming dimers, or both.</p>
<figure class="fig xbox font-sm" id="Fig4"><h4 class="obj_head">Fig. 4. Distribution of dimeric and monomeric UVR8s.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371036_41467_2025_63010_Fig4_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9400/12371036/da6c250302b8/41467_2025_63010_Fig4_HTML.jpg" loading="lazy" id="d33e1364" height="659" width="683" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Total proteins were extracted from <em>Arabidopsis</em> plants grown under high light (HL) or normal light (NL) conditions for 3, 4 or 5 weeks. UVR8 dimers were separated by non-denaturing PAGE, and the gel was irradiated with UV-B (30 min, 300 µW/cm<sup>2</sup>) before transfer to a nylon membrane (non-denaturing, upper blots in each panel). Denatured samples (denaturing, center blots in each panel) were run by SDS-PAGE and transferred to nylon. Blots were incubated with the antibody specific to <em>Arabidopsis</em> UVR8 (α-UVR8), and the blot corresponding to the gel run in denaturing conditions was incubated with an antibody specific to <em>Arabidopsis</em> actin (α-Actin, lower blots in each panel) for protein loading normalization. <strong>b</strong> Col-0, <em>tt5-2</em>, and <em>tt4-11 tt5-2</em> mutant plants grown for 3 weeks under HL (T<sub>0</sub>) were treated with UV-B (200 µW/cm<sup>2</sup>, 60 min, T<sub>1</sub>) and transferred back to HL conditions for 2 h (T<sub>3</sub>), 5 h (T<sub>6</sub>), 11 h (T<sub>12</sub>), 23 h (T<sub>24</sub>), 47 h (T<sub>48</sub>), and 71 h (T<sub>72</sub>). Samples were collected in a time series, and total proteins were extracted to determine the distribution of dimeric and monomeric UVR8s. Upper panel, non-denatured condition with the irradiation of protein gels to detect both dimeric and monomeric UVR8s. Middle panel, denatured samples indicating the amounts of total UVR8s. Actin levels were used as a loading control (lower panel). D indicates the dimeric form of UVR8, M<sub>1</sub> and M<sub>2</sub> correspond to two UVR8 monomer (M) conformers, and the open white arrow marks the position of a non-specific protein recognized by α-UVR8. The experiments were independently carried out twice with similar results. Source data are provided as a Source Data file.</p></figcaption></figure><p id="Par14">UV-B rapidly and reversibly induces the formation of UVR8 monomers in vivo<sup><a href="#CR18" class="usa-link" aria-describedby="CR18">18</a>,<a href="#CR21" class="usa-link" aria-describedby="CR21">21</a></sup>. To gain a better understanding of the mechanisms associated with the increase of UVR8 monomer in <em>tt5</em> mutants, we investigated the response to UV-B of UVR8 in <em>tt5-2</em>, compared to Col-0 and the <em>tt4-11 tt5-</em>2 double mutant. We treated plants (HL grown without UV-B, T<sub>0</sub>) with UV-B (200 µW/cm<sup>2</sup>) for 60 min (T<sub>1</sub>) and then we transferred plants back to HL conditions for 2 h (T<sub>3</sub>), 5 h (T<sub>6</sub>), 11 h (T<sub>12</sub>), 23 h (T<sub>24</sub>), 47 hours (T<sub>48</sub>), and 71 h (T<sub>72</sub>) (Fig. <a href="#Fig4" class="usa-link">4b</a>). We extracted protein and analyzed by Western blot the accumulation of monomeric and dimeric UVR8 as described above. The UV-B treatment slightly induced total UVR8 levels in all lines (compare denaturing conditions signals with α-UVR8 and α-Actin, Fig. <a href="#Fig4" class="usa-link">4b</a>), but most prominently induced the formation of UVR8 monomers (M<sub>1</sub> and M<sub>2</sub>) in Col-0 (lane 2, Fig. <a href="#Fig4" class="usa-link">4b</a>), <em>tt5-2</em> (lane 10, Fig. <a href="#Fig4" class="usa-link">4b</a>) and <em>tt4-11 tt5-2</em> (lane 18, Fig. <a href="#Fig4" class="usa-link">4b</a>). Upon transfer back to HL conditions, the monomer rapidly disappeared in Col-0 and <em>tt4-11 tt5-</em>2, but although reduced, did not completely vanish in <em>tt5-2</em> (compare lanes 3, 11 and 19, Fig. <a href="#Fig4" class="usa-link">4b</a>). However, the levels of UVR8 monomer dramatically increased 11 h after recovery from UV-B (lane 13, Fig. <a href="#Fig4" class="usa-link">4b</a>), but only in <em>tt5-2</em>, and continued high for 72 h, the duration of the experiment (lane 16, Fig. <a href="#Fig4" class="usa-link">4b</a>). Noticeable was that, while UV-B significantly induced the formation of monomer M<sub>2</sub> in both Col-0 and <em>tt5-2</em> (lanes 2 and 10, Fig. <a href="#Fig4" class="usa-link">4b</a>), the appearance of M<sub>1</sub> at longer times of recovery in HL was unique to <em>tt5-2</em> (lanes 14–16, Fig. <a href="#Fig4" class="usa-link">4b</a>). These results suggest that UV-B and NGC or its derivative promote formation of different UVR8 monomer conformers.</p>
<p id="Par15">Compared to other <em>Arabidopsis</em> flavonoid mutants, <em>tt5</em> is hypersensitive to UV-B<sup><a href="#CR40" class="usa-link" aria-describedby="CR40">40</a></sup>. This increased sensitivity of <em>tt5</em> to UV-B is evident, for example, in the appearance of a shiny leaf surface as early as T<sub>12</sub> and the development of more severe phenotypes including pale and curled leaves at later times after UV-B, phenotypes absent in the double <em>tt4-11 tt5-2</em> mutant (Supplementary Fig. <a href="#MOESM1" class="usa-link">16</a>). Based on our results, it is possible that NGC or its derivative directly or indirectly increases the levels of monomeric UVR8, sustaining a response to UV-B even after UV-B is gone, and that this is responsible for the observed UV-B sensitivity of <em>tt5</em>.</p></section><section id="Sec7"><h3 class="pmc_sec_title">Accumulation of NGC derivatives in <em>tt5</em> mutants in HL</h3>
<p id="Par16">To understand the accumulation of NGC or its derivative in <em>tt5</em> under HL potentially responsible for the increase in monomeric UVR8, we analyzed the plant samples described in the previous section (Supplementary Fig. <a href="#MOESM1" class="usa-link">14</a>), by performing untargeted metabolic profiling of methanol extracts by LC-MS/MS, focusing on compounds exhibiting a fragmentation product of <em>m/z</em> 273.08, characteristic of NGC or its derivatives. In extracts of <em>tt5-2</em> mutant with 4 weeks in HL, four major peaks absent in Col-0 and corresponding to NGC derivatives were identified (Fig. <a href="#Fig5" class="usa-link">5</a>). Remarkably, NGC itself was not found at significant levels in any of the samples, indicating that the naringenin identified in hydrolyzed samples (Supplementary Fig. <a href="#MOESM1" class="usa-link">9</a>) corresponded most likely to NGC derived from <em>O</em>-linked derivative(s) easily hydrolysable in hot acid. Subsequent tandem MS analysis of these four peaks indicated that Peak 1 (<em>m/z</em> 614.21; retention time 4.04 min) is consistent with a double glycosylation of NGC forming an NH<sub>4</sub> adduct during ionization (Hexose-NGC-Hexose_1, 273.08 + 162.05 + 162.05 + 17.03 = 614.21); Peak 2 (<em>m/z</em> 597.18; retention time 4.51 min) agrees with another NGC-diglycoside (Hexose-NGC-Hexose_2, 273.08 + 162.05 + 162.05 = 597.18); Peak 3 (<em>m/z</em> 435.13; retention time 5.02 min) is likely a single glycoside (hexose) linked to NGC (NGC-Hexose, 273.08 + 162.05 = 435.13); and Peak 4 (<em>m/z</em> 521.13; retention time 5.28 min) corresponds most likely to NGC conjugated to a glycoside and a malonyl group (NGC-Hexose-Mal, 273.08 + 162.05 + 86.01 = 521.13) (Fig. <a href="#Fig5" class="usa-link">5</a>).</p>
<figure class="fig xbox font-sm" id="Fig5"><h4 class="obj_head">Fig. 5. Identification of naringenin chalcone (NGC) derivatives in <em>tt5-2</em> plants.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371036_41467_2025_63010_Fig5_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9400/12371036/728cda46a311/41467_2025_63010_Fig5_HTML.jpg" loading="lazy" id="d33e1574" height="523" width="700" alt="Fig. 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>NGC derivatives in <em>tt5-2</em> plants. Four major peaks, resulting in the <em>m/z</em> 273.08 daughter ion from LC-MS/MS, were detected in the <em>tt5-2</em> mutant (black line) under high light (HL) conditions for 4 weeks, but not in Col-0 (red line). Tandem MS profiles of these peaks were annotated as follows: Peak 1 (<em>m/z</em> 614.21, Hex-NGC-Hex_1), Peak 2 (<em>m/z</em> 597.18, Hex-NGC-Hex_2), Peak 3 (<em>m/z</em> 435.13, NGC-Hex), and Peak 4 (<em>m/z</em> 521.13, NGC-Hex-Mal). Hex, hexose group; Mal, malonyl moiety. The experiments were independently repeated twice with similar results, each time using different sets of plants. Source data are provided as a Source Data file.</p></figcaption></figure><p id="Par17">Using the diagnostic <em>m/z</em> peaks, we determined that all the NGC derivatives accumulate at very low levels, if at all, in <em>tt5-2</em> under NL (Supplementary Fig. <a href="#MOESM1" class="usa-link">17a</a>). When grown under HL conditions, Peaks 1 and 2 continued to increase during development in the <em>tt5-2</em> mutant and in the <em>tt5 uvr8</em> suppressor lines, while Peaks 3 and 4 accumulated at a maximum after four weeks of HL exposure, to then decrease at five weeks in the <em>tt5-2</em> and <em>tt5 uvr8</em> suppressor lines under HL. These compounds were present at low levels in Col-0 plants grown in HL and were absent in <em>tt4-11</em> or <em>tt4-11 tt5-2</em> plants (Supplementary Fig. <a href="#MOESM1" class="usa-link">17b</a>). The absence of NGC in all the samples indicates that all the NGC that accumulates in <em>tt5-2</em> in HL gets rapidly glycosylated and/or malonylated.</p>
<p id="Par18">We then analyzed by LC-MS/MS the accumulation of NGC derivatives in plants treated with UV-B to determine whether a particular compound was associated with the sustained levels of UVR8 monomer observed in <em>tt5-2</em> following recovery from UV-B treatment (Fig. <a href="#Fig4" class="usa-link">4b</a>). All four NGC derivatives are present in <em>tt5-2</em> plants grown in HL for three weeks (T<sub>0</sub>, Supplementary Fig. <a href="#MOESM1" class="usa-link">18</a>) with UV-B significantly enhancing the accumulation of all four compounds at ~ 5 h after treatment but only in <em>tt5-2</em> (T<sub>6</sub>, Supplementary Fig. <a href="#MOESM1" class="usa-link">18</a>). This is consistent with previous studies demonstrating the induction of <em>Arabidopsis</em> flavonoid biosynthesis by UV light after 7 h of treatment<sup><a href="#CR41" class="usa-link" aria-describedby="CR41">41</a></sup>. At subsequent recovery times, the levels of the four NGC derivatives returned to levels comparable to those present before the UV-B treatment (Supplementary Fig. <a href="#MOESM1" class="usa-link">18</a>, compare T<sub>0</sub> and T<sub>12</sub>).</p>
<p id="Par19">These results show that UV-B induces UVR8 monomer formation irrespective of whether NGC derivatives are present or not, although the monomer levels in the flavonoid mutants might be higher than in Col-0 because of the absence of UV-protecting flavonoid sunscreen in <em>tt5-2</em> and <em>tt4-11 tt5-2</em> (compare lanes 2, 10 and 18 in Fig. <a href="#Fig4" class="usa-link">4b</a>). Once the UV-B treatment is finished, UVR8 rapidly reverts to the dimeric form in Col-0 and <em>tt4-11 tt5-2</em> (lanes 3 and 19, Fig. <a href="#Fig4" class="usa-link">4b</a>), but the presence of NGC derivatives in <em>tt5-2</em> sustains a low but significant amount of UVR8 monomer (lanes 11 and 12, Fig. <a href="#Fig4" class="usa-link">4b</a>). The striking increase of monomeric UVR8 in <em>tt5-2</em> about 11 h after the UV-B treatment (Lanes 13–16, Fig. <a href="#Fig4" class="usa-link">4b</a>) could be a consequence of the UV-B induced increase in NGC derivatives in <em>tt5-2</em>, which can be observed at around 5 h following the UV-B treatment (T<sub>6</sub>, Supplementary Fig. <a href="#MOESM1" class="usa-link">18</a>). It is interesting that all four NGC derivatives return to the levels found prior to the UV-B treatment by 11 h of recovery (T<sub>12</sub>, Supplementary Fig. <a href="#MOESM1" class="usa-link">18</a>), while the amount of monomeric UVR8 continues to increase (M, lanes 1–16, Fig. <a href="#Fig4" class="usa-link">4b</a>).</p>
<p id="Par20">To assess the significance of our findings under more natural <em>Arabidopsis</em> growth conditions and their physiological roles in wild-type plants, we cultivated Col-0 and mutant lines (<em>tt4-11, tt5-2, tt4-11 tt5-2, tt5-2 uvr8-31, PAP1-D, tt6-2</em>, and <em>tt19-8</em>) outdoors in a courtyard under natural Hawai’i September-October sunlight and temperature conditions, with and without a UV-A/B filter to block natural UV-A/B, for five weeks (Supplementary Fig. <a href="#MOESM1" class="usa-link">19</a>). <em>PAP1-D</em> corresponds to a line in which the <em>PAP1</em> anthocyanin regulator is constitutively expressed derived from an activation tagging screen<sup><a href="#CR42" class="usa-link" aria-describedby="CR42">42</a></sup>. After five weeks outside, total proteins were extracted from the plants and analyzed via Western blot to examine the distribution of monomeric and dimeric UVR8 (Supplementary Fig. <a href="#MOESM1" class="usa-link">20a</a>). Our analysis showed that the total UVR8 protein levels were comparable across Col-0<em>, tt4-11, tt5-2, tt4-11 tt5-2, PAP1-D, tt6-2</em>, and <em>tt19-8</em>, regardless of whether the plants were grown with or without UV-A/B filtration (Supplementary Fig. <a href="#MOESM1" class="usa-link">20a</a>, α-UVR8 Denaturing). Under non-denaturing conditions, when the polyacrylamide gels were exposed to UV-B, the UVR8 dimer levels were detected and showed no significant differences between the samples (Supplementary Fig. <a href="#MOESM1" class="usa-link">20a</a>, α-UVR8 Non-denaturing, +UV-B). Monomeric UVR8 levels were evaluated under non-denaturing conditions without UV-B exposure of the polyacrylamide gel, where UVR8 dimers are barely detectable (Supplementary Fig. <a href="#MOESM1" class="usa-link">20a</a>, α-UVR8 Non-denaturing, -UV-B). Comparing all samples, we observed accumulation of UVR8 monomer in Col-0 at moderate levels and <em>tt5-2</em> at the highest levels, regardless of whether UV-A/B radiation was eliminated or not during the growth of the plants (Supplementary Fig. <a href="#MOESM1" class="usa-link">20a</a>, α-UVR8 Non-denaturing, -UV-B, lanes 1, 3, 9, and 11).</p>
<p id="Par21">To determine the impact of sunlight conditions on the levels of NGC and derivatives, we analyzed by LC-MS/MS the accumulation of NGC derivatives in Col-0<em>, tt4-11 tt5-2, tt5-2</em>, and <em>tt5-2 uvr8-31</em> plants grown with and without a UV-A/B filter for 5 weeks (Supplementary Fig. <a href="#MOESM1" class="usa-link">20b</a>). The results showed that Peak 1 (Hex-NGC-Hex), Peak 3 (NGC-Hex), and Peak 4 (NGC-Hex-Mal) accumulated in Col-0 plants at meaningful levels irrespective of whether UV-A/B was present or not, were significantly (~ 10-fold) higher in <em>tt5-2</em> with a modest positive impact of UV-A/B, and absent in <em>tt4-11 tt5-2</em> (Supplementary Fig. <a href="#MOESM1" class="usa-link">20b</a>). Notably, the levels of Peaks 1 – 3 were indistinguishable in <em>tt5-2</em> and <em>tt5-2 uvr8-31</em> in the presence of natural UV-A/B radiation (<em>i.e</em>., -UV-A/B filter, Supplementary Fig. <a href="#MOESM1" class="usa-link">20b</a>) suggesting that UVR8 under sunlight conditions has a modest effect on flavonoid accumulation. Overall, our findings indicate that while NGC derivatives are undetectable in Col-0 growth chamber conditions, even in HL, their accumulation is significant when plants grow outdoors, regardless of UV-A/B exposure, and correlates with increased levels of monomeric UVR8.</p></section><section id="Sec8"><h3 class="pmc_sec_title">NGC and derivatives increase UVR8 monomer formation in vitro</h3>
<p id="Par22">Our results so far indicate that the presence of NGC derivatives correlates with the increase of UVR8 monomer accumulation. To uncover the mechanism driving this correlation, we incubated protein extracts obtained from Col-0 plants grown under HL for 4 weeks with methanol extracts of <em>tt5-2</em> plants grown under HL for 4 weeks (lanes 1 and 2, Supplementary Fig. <a href="#MOESM1" class="usa-link">21a</a>), pure commercial NGC (lanes 3 and 4, Supplementary Fig. <a href="#MOESM1" class="usa-link">21a</a>), acid hydrolyzed methanol extracts of <em>tt5-2</em> plants grown under HL for 4 weeks (lanes 5 and 6, Supplementary Fig. <a href="#MOESM1" class="usa-link">21a</a>) and compared with the intensity of the UVR8 monomer band in the absence of any exogenous small molecule (lane 7, Supplementary Fig. <a href="#MOESM1" class="usa-link">21a</a>). Because we are focusing specifically on the UVR8 monomer (M), we did not treat the gel with UV prior to transfer, hence the UVR8 dimer (D) band is barely visible under non-denaturing conditions, but the total amount of UVR8 is comparable between all the samples (α-UVR8 Denaturing, Supplementary Fig. <a href="#MOESM1" class="usa-link">21a</a>). The methanol extracts visibly increased the amount of UVR8 monomer, an increase that could be observed also with pure NGC, but not with hydrolyzed extracts (keep in mind that acid hydrolysis converts NGC into naringenin). To quantify this effect, we repeated this experiment with three different protein and methanol extracts (Supplementary Fig. <a href="#MOESM1" class="usa-link">21b</a>), and determined that NGC or methanol extracts, but not hydrolyzed extracts, increased the level of monomeric UVR8 by about 2-fold (Supplementary Fig. <a href="#MOESM1" class="usa-link">21b</a>). This is only a fraction of the UVR8 capable of forming monomer, evidenced by the finding that a 60 min brief treatment with UV-B (300 µW/cm<sup>2</sup>) of the protein extracts increased the amount of monomer by about 7.5-fold (lane 9 in Supplementary Fig. 21a; arrow in Supplementary Fig. <a href="#MOESM1" class="usa-link">21c</a>). Taken together, our results demonstrate that NGC (and NGC derivatives) enhance UVR8 monomer accumulation.</p></section><section id="Sec9"><h3 class="pmc_sec_title">NGC physically interacts with UVR8 monomer</h3>
<p id="Par23">To determine whether the correlation between NGC and UVR8 monomer results from a direct interaction rather than an indirect effect, for example requiring other cellular factors, we purified recombinant wild-type UVR8, UVR8<sup>D96N/D107N</sup> (a constitutive monomer<sup><a href="#CR43" class="usa-link" aria-describedby="CR43">43</a></sup>), and UVR8<sup>W285F</sup> (a constitutive dimer<sup><a href="#CR44" class="usa-link" aria-describedby="CR44">44</a></sup>) and validated their in vitro UV-B response properties by size exclusion chromatography (SEC). As expected, wild-type UVR8 dimers dissociated into monomers upon UV-B treatment, while UVR8<sup>D96N/D107N</sup> remained monomeric and UVR8<sup>W285F</sup> remained dimeric, regardless of UV-B exposure (Supplementary Fig. <a href="#MOESM1" class="usa-link">22</a>). These functionally validated UVR8 variants were then used for protein-ligand binding assays. In nano differential scanning fluorimetry (nanoDSF) experiments, NGC and naringenin were incubated with various UVR8 forms: wild-type UVR8 (dimer), wild-type UVR8 + UV-B (UVR8 monomer), UVR8<sup>D96N/D107N</sup> (UVR8 monomer), UVR8<sup>D96N/D107N</sup> + UV-B (UVR8 monomer), UVR8<sup>W285F</sup> (UVR8 dimer), and UVR8<sup>W285F</sup> + UV-B (UVR8 dimer). The thermal denaturation temperature (T<sub>m</sub>) was measured to assess protein conformational stability in each condition. The results showed no significant differences in T<sub>m</sub> among the three dimeric forms (wild-type UVR8, UVR8<sup>W285F</sup>, and UVR8<sup>W285F</sup> + UV-B) when incubated with DMSO (control), NGC, or naringenin (Supplementary Fig. <a href="#MOESM1" class="usa-link">23</a> and Supplementary Table <a href="#MOESM1" class="usa-link">2</a>). However, a significant increase in T<sub>m</sub> was observed when NGC was incubated with the three monomeric forms (wild-type UVR8 + UV-B, UVR8<sup>D96N/D107N</sup>, and UVR8<sup>D96N/D107N</sup> + UV-B) compared to incubation with DMSO or naringenin (Supplementary Fig. <a href="#MOESM1" class="usa-link">23</a> and Supplementary Table <a href="#MOESM1" class="usa-link">2</a>). These findings indicate that NGC, but not naringenin, physically interacts with monomeric UVR8 but not with dimeric UVR8. To further investigate the specificity of UVR8 monomer interactions with small molecules, we tested a collection of flavonoids, including NGC, naringenin, kaempferol, dihydrokaempferol, dihydroquercetin, leucocyanidin, and cyanidin. Among these, only NGC incubation resulted in a significantly increased T<sub>m</sub> for monomeric UVR8 (Supplementary Fig. <a href="#MOESM1" class="usa-link">24</a> and Supplementary Table <a href="#MOESM1" class="usa-link">3</a>). These results demonstrate that the UVR8 monomer specifically interacts with NGC, but not with other flavonoids or flavonoid pathway intermediates.</p>
<p id="Par24">To further characterize the interaction, we conducted microscale thermophoresis (MST) experiments to produce binding curves by incubating UVR8 in its dimeric or monomeric form with NGC or naringenin (Supplementary Fig. <a href="#MOESM1" class="usa-link">25</a>). These binding curves were then used to calculate the apparent equilibrium dissociation constant (<em>K</em><sub>d</sub>). The results showed no binding for dimeric UVR8 incubated with either NGC or naringenin. In contrast, a calculated <em>K</em><sub>d</sub> value (in triplicate) for NGC binding to UVR8 monomer was estimated to be ~ 12 ± 9.2 μM, demonstrating a strong interaction. As anticipated from our previous results, naringenin exhibited a much weaker interaction with UVR8 monomer (apparent <em>K</em><sub>d</sub> &gt; 1.6 mM). These findings demonstrate that NGC physically interacts with UVR8 in its monomeric form with high affinity. Unfortunately, we have not yet succeeded in purifying the NGC derivatives that are found in vivo in quantities large enough to test their affinity for UVR8 monomers.</p>
<p id="Par25">To predict the potential binding site(s) of NGC on the UVR8 monomer, we utilized Protenix<sup><a href="#CR45" class="usa-link" aria-describedby="CR45">45</a></sup>, a simulation tool that models protein-ligand interactions through a comprehensive AlphaFold3 reproduction. We tested different molar ratios of NGC to UVR8 monomer (1:1, 2:1, 3:1, and 4:1) and found that the 1:1 ratio produced the highest interface predicted template modeling (ipTM) score (the only score exceeding 0.9, indicating a confident, high-quality prediction; Supplementary Table <a href="#MOESM1" class="usa-link">4</a>). These results suggest that one NGC molecule binding to one UVR8 monomer is the most probable interaction model. We could not recover a UVR8-naringenin model in which the solved structure of UVR8 was preserved<sup><a href="#CR26" class="usa-link" aria-describedby="CR26">26</a>,<a href="#CR46" class="usa-link" aria-describedby="CR46">46</a></sup>, consistent with the experimental results that show no naringenin high-affinity binding to monomeric UVR8 (Supplementary Fig. <a href="#MOESM1" class="usa-link">25c, d</a>). To visualize the potential NGC binding site on the UVR8 monomer, we analyzed the highest-scoring model using PyMOL<sup><a href="#CR47" class="usa-link" aria-describedby="CR47">47</a></sup>, which revealed four candidate amino acids in UVR8 that may interact with NGC: G283, C335, W337, and R439 (Supplementary Fig. <a href="#MOESM1" class="usa-link">26</a>). The variation in free energy (Δ<em>G</em>) of the binding of NGC to the UVR8 monomer was estimated by PyFepRestr<sup><a href="#CR48" class="usa-link" aria-describedby="CR48">48</a></sup> at ~ 30 kJ/mol. These analyses predict a specific and high-affinity interaction between NGC and the UVR8 monomer and identify the potential key UVR8 residues involved.</p></section><section id="Sec10"><h3 class="pmc_sec_title">UVR8 may contribute to processes beyond UV-B signaling</h3>
<p id="Par26">Our results suggest that NGC (and its derivatives) modulate a UVR8 function that does not require UV-A/B. To gain insight into this function, we analyzed mRNA accumulation changes in plants with the null <em>uvr8-6</em> allele grown for 4 weeks under NL and HL conditions with a UV-A/B filter (Supplementary Fig. <a href="#MOESM1" class="usa-link">6b</a>), comparing them to Col-0. DEG analyses between Col-0 and <em>uvr8-6</em> under HL (with UV-A/B filter) identified 43 genes higher in Col-0 (potentially positively regulated by <em>UVR8</em> independently of UV-B) and 54 genes higher in <em>uvr8-6</em> (potentially negatively regulated by <em>UVR8</em> independently of UV-B, Fig. <a href="#Fig6" class="usa-link">6</a> and Supplementary Data <a href="#MOESM9" class="usa-link">6</a>). These results were further compared to those from <em>tt5-2</em>, <em>tt4-11</em>, and <em>tt4-11 tt5-2</em> mutants grown under the same conditions (Fig. <a href="#Fig2" class="usa-link">2</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">7</a>). We found that more than half of the <em>UVR8</em>-dependent UV-B independent DEGs were also unique to the response of <em>tt5</em> to four weeks of HL (DEG7, Fig. <a href="#Fig2" class="usa-link">2b</a>), yet mRNAs that accumulated lower in <em>uvr8-6</em> (presumably under positive UVB8 regulation) accumulated higher in <em>tt5-2</em> (Fig. <a href="#Fig6" class="usa-link">6</a>). The opposite trend is observed for genes that are higher in <em>uvr8-6</em> (<em>i.e</em>., they are lower in <em>tt5</em> grown for four weeks in HL), but the effect is less marked (Supplementary Data <a href="#MOESM10" class="usa-link">7</a>). An analysis of the potential function of these genes revealed that the vast majority are associated with plant defense or immunity responses, including genes encoding pathogen-associated molecular patterns, receptor-like proteins, chitinases, and disease resistance proteins (highlighted with asterisks in Fig. <a href="#Fig6" class="usa-link">6</a>). These results reveal that NGC and its derivatives modulate UVR8 functions beyond UV-B signaling, influencing stress responses and immunity. Moreover, they uncover unrecognized roles for UVR8, broadening our understanding of its versatility in plant biology.</p>
<figure class="fig xbox font-sm" id="Fig6"><h4 class="obj_head">Fig. 6. Identification of UVR8-dependent genes positively affected by the accumulation of naringenin chalcone (NGC) derivatives.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371036_41467_2025_63010_Fig6_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9400/12371036/89042056a4ea/41467_2025_63010_Fig6_HTML.jpg" loading="lazy" id="d33e2040" height="871" width="700" alt="Fig. 6"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Differentially expressed gene (DEG) analysis of Col-0 HL versus <em>uvr8-6</em> HL and <em>tt5-2</em> HL versus <em>tt5-2</em> NL were conducted on RNA-seq data with resulting in the identification of 97 and 5251 DEGs, respectively. The Venn diagram illustrates 55 shared genes between DEG9 and DEG7. The gene IDs, fold change, and description for the 27 genes upregulated in <em>tt5-2</em> HL are indicated. Color gradient indicates the log<sub>2</sub>Fold Change. The asterisks indicate 10 genes associated with plant defense or immune responses. Source data are provided as a Source Data file.</p></figcaption></figure></section></section><section id="Sec11"><h2 class="pmc_sec_title">Discussion</h2>
<p id="Par27">Mutations of flavonoid biosynthesis genes have not conventionally been associated with pleiotropic phenotypic effects, like those observed here for <em>tt5</em> under HL conditions (Supplementary Fig. <a href="#MOESM1" class="usa-link">3</a>). The finding that the pathway intermediate NGC and likely NGC derivatives that accumulate in WT plants under normal solar light, physically interact with the UVR8 photoreceptor increasing the monomeric form of UVR8, even in the absence of detectable UV-A/B radiation, provides an unexpected link between light-induced metabolites and photoreceptor-dependent developmental programs. Indeed, many of the developmental effects observed for <em>tt5</em> grown in HL resemble moderate manifestations of the phenotypes described for plants overexpressing a constitutively monomeric UVR8 protein<sup><a href="#CR26" class="usa-link" aria-describedby="CR26">26</a></sup>. While we initially considered the possibility that the developmental phenotypes of <em>tt5</em> under HL resulting from the accumulation of NGC derivatives resemble a constitutive response to UV-B (even in the absence of UV-B), our results indicate that this is not the case. When we compared the UV-B responsive <em>UVR8</em>-dependent genes from the literature<sup><a href="#CR49" class="usa-link" aria-describedby="CR49">49</a></sup> with those uniquely accumulating in <em>tt5</em> under HL (Fig. <a href="#Fig2" class="usa-link">2</a>), we found no statistically significant overlap (Supplementary Fig. <a href="#MOESM1" class="usa-link">27</a>). However, when we investigated <em>UVR8</em> regulated genes under HL conditions in the absence of UV-B, we found that they significantly overlap with those uniquely induced in <em>tt5</em> under HL (Fig. <a href="#Fig6" class="usa-link">6</a>). In fact, genes expressed uniquely higher in <em>tt5-2</em> under HL show lower mRNA accumulation in the <em>uvr8-6</em> mutant under HL (Fig. <a href="#Fig6" class="usa-link">6</a>), indicating that their expression is dependent on active UVR8. The enrichment of these genes in functions related to stress responses and immunity was unexpected and indicates a <em>UVR8</em> function that goes beyond UV-B perception and integration of light signals. In this regard, it is noteworthy the induction of SA and SAG in <em>tt5</em> plants under HL conditions, with the concomitant changes in gene expression (Fig. <a href="#Fig2" class="usa-link">2d</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">13a</a>). Indeed, about a third (642/2111) of the genes induced in <em>tt5-2</em> under HL (DEG7, Fig. <a href="#Fig2" class="usa-link">2b</a>) correspond to genes described as induced by one hour of SA treatment<sup><a href="#CR31" class="usa-link" aria-describedby="CR31">31</a></sup> indicating that most likely they result from the increased SA accumulation in <em>tt5-2</em> under HL. This increased SA accumulation under HL may provide <em>tt5</em> enhanced resistance to plant pathogens<sup><a href="#CR50" class="usa-link" aria-describedby="CR50">50</a></sup>, providing an additional piece to the puzzle of how light influences plant-pathogen interactions<sup><a href="#CR51" class="usa-link" aria-describedby="CR51">51</a></sup>. Our results, however, show that this increase in SA and SAG is independent of <em>UVR8</em> (Supplementary Fig. <a href="#MOESM1" class="usa-link">13b</a>), indicating that the induction by light of NGC and derivatives targets the pathogen response/immunity axis by UVR8-dependent and UVR8-independent mechanisms. Given that the induction of flavonoids by UV light is conserved across the plant kingdom<sup><a href="#CR17" class="usa-link" aria-describedby="CR17">17</a></sup>, our results may help explain the elusive mechanisms by which UV-A/B light contributes to reducing disease and pest incidence in many horticultural settings<sup><a href="#CR52" class="usa-link" aria-describedby="CR52">52</a></sup>. Constitutively high SA accumulation results in stunted <em>Arabidopsis</em> growth<sup><a href="#CR34" class="usa-link" aria-describedby="CR34">34</a>,<a href="#CR53" class="usa-link" aria-describedby="CR53">53</a>,<a href="#CR54" class="usa-link" aria-describedby="CR54">54</a></sup>, but the lack of similar defects in the <em>tt5 uvr8</em> double mutant suppressor lines (Supplementary Fig. <a href="#MOESM1" class="usa-link">12</a>) indicates that either the growth defect of <em>tt5</em> in HL is independent of high SA accumulation, or that UVR8 function is somehow required for the SA hyperaccumulation phenotypes. A mutation in the CDK8 component of the Mediator complex similarly offset SA hyperaccumulation phenotypes<sup><a href="#CR55" class="usa-link" aria-describedby="CR55">55</a></sup>, perhaps implying a relationship between the Mediator complex and UVR8.</p>
<p id="Par28">Under normal growth chamber conditions, most UVR8 is dimeric (believed to correspond to the cytoplasmic ground state) and upon UV-B irradiation, a significant fraction of UVR8 can be found in the monomeric form (believed to correspond to the nuclear signaling state) where it interacts with COP1 and various regulatory proteins<sup><a href="#CR18" class="usa-link" aria-describedby="CR18">18</a>,<a href="#CR23" class="usa-link" aria-describedby="CR23">23</a>–<a href="#CR25" class="usa-link" aria-describedby="CR25">25</a>,<a href="#CR56" class="usa-link" aria-describedby="CR56">56</a>,<a href="#CR57" class="usa-link" aria-describedby="CR57">57</a></sup> (Fig. <a href="#Fig7" class="usa-link">7a</a>). Consistent with a role of NGC (and derivatives) in stabilizing monomeric UVR8, the observed phenotypes of <em>tt5</em> under HL conditions resemble to some extent those of <em>cop1</em> mutants<sup><a href="#CR58" class="usa-link" aria-describedby="CR58">58</a>,<a href="#CR59" class="usa-link" aria-describedby="CR59">59</a></sup>.</p>
<figure class="fig xbox font-sm" id="Fig7"><h3 class="obj_head">Fig. 7. Proposed mechanism for stabilization of monomeric UVR8 by naringenin chalcone (NGC) derivatives and its effects on UVR8-mediated gene regulation.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371036_41467_2025_63010_Fig7_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9400/12371036/972b6f702659/41467_2025_63010_Fig7_HTML.jpg" loading="lazy" id="d33e2225" height="659" width="751" alt="Fig. 7"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig7/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Under growth chamber light conditions, UVR8 predominantly exists in the cytosol as an inactive homodimer. A minor fraction of UVR8 is present in the nucleus in its monomeric form, where it interacts with COP1 (E3 ubiquitin ligase) to inhibit COP1-mediated degradation of HY5. Defense-related genes I are activated by endogenous SA. <strong>b</strong> Under natural sunlight conditions, accumulated NGC derivatives elevate SA levels and directly bind to UVR8 monomers. This SA accumulation leads to the activation of defense-related genes II. The formation of monomeric UVR8-NGC derivative complex promotes the activation of a subset of UVR8-regulated genes that don’t require UV-B signaling. COP1, CONSTITUTIVELY PHOTOMORPHOGENIC1; HY5, ELONGATED HYPOCOTYL5; NGC, naringenin chalcone; SA, salicylic acid; UVR8, UV RESISTANCE LOCUS8.</p></figcaption></figure><p id="Par29">The importance of <em>UVR8</em> under growth chamber conditions likely lies in its long-term effects, such as mitigating UV-B induced DNA damage<sup><a href="#CR19" class="usa-link" aria-describedby="CR19">19</a>,<a href="#CR60" class="usa-link" aria-describedby="CR60">60</a></sup>. Solar light clearly induces monomeric UVR8, an induction that is observed even when UV-A/B light is filtered (Supplementary Fig. <a href="#MOESM1" class="usa-link">20</a>). Our results suggest that this is a consequence of the increase in NGC derivatives that physically interact with monomeric UVR8 (Supplementary Figs. <a href="#MOESM1" class="usa-link">23</a>–<a href="#MOESM1" class="usa-link">26</a>) and presumably results in higher levels of nuclear stabilized monomeric UVR8 (Fig. <a href="#Fig7" class="usa-link">7b</a>). How are such UVR8 monomers formed in the absence of UV radiation? Putting aside the possibility that there is some UV fluence that is below detection levels (yet could very slowly contribute to UVR8 monomer formation, as is the case in the experiments here described, Supplementary Fig. <a href="#MOESM1" class="usa-link">14</a>), it is important to remember that even though the dissociation constant for UVR8 dimer (without UV) is low<sup><a href="#CR20" class="usa-link" aria-describedby="CR20">20</a></sup> (yet has not been measured), monomer and dimer forms are in equilibrium, which would be displaced towards monomer formation when the monomer is stabilized by NGC (or derivatives). Because the accumulation of NGC derivatives in <em>tt5</em> grown in HL does not resemble the gene expression changes associated with UV-B exposure<sup><a href="#CR61" class="usa-link" aria-describedby="CR61">61</a></sup>, we argue that the interaction of UVR8 with NGC derivatives is important for a subset of the <em>UVR8</em> functions that don’t require UV-A/B radiation, directly implicating UVR8 with stress responses and immunity (Fig. <a href="#Fig7" class="usa-link">7b</a>). The interaction of UVR8 monomers with NGC and presumably one or more of the NGC derivatives identified in this study (Fig. <a href="#Fig5" class="usa-link">5</a>) is both robust (low micromolar range <em>K</em><sub>d</sub>, Supplementary Fig. <a href="#MOESM1" class="usa-link">25</a>) and specific, at least within the flavonoid pathway intermediates available (Supplementary Fig. <a href="#MOESM1" class="usa-link">24</a>). The possibility that NGC interacts in the UVR8 monomer with some of the same residues that are involved in dimerization<sup><a href="#CR62" class="usa-link" aria-describedby="CR62">62</a>,<a href="#CR63" class="usa-link" aria-describedby="CR63">63</a></sup> (Supplementary Fig. <a href="#MOESM1" class="usa-link">26</a>) is consistent with the model that it stabilizes the UVR8 monomer (Fig. <a href="#Fig7" class="usa-link">7b</a>). This discovery potentially identifies an interesting interaction pocket for small molecules, presenting exciting opportunities to engineer UVR8 mutants with tailored responsiveness to NGC and its derivatives. Our findings also highlight the need for caution when interpreting the phenotypes of mutants in metabolic pathways. Accumulation of pathway intermediates at significantly elevated levels in these mutants may exhibit unforeseen biological activities, potentially confounding phenotype analysis and revealing unexpected roles for these intermediates. Taken together, our results identify developmental functions for UVR8 that extend beyond just UV-A/B sensing and can be modulated by flavonoid pathway intermediates, which themselves are to some extent controlled by UVR8.</p></section><section id="Sec12"><h2 class="pmc_sec_title">Methods</h2>
<section id="Sec13"><h3 class="pmc_sec_title">Chemicals and instruments</h3>
<p id="Par30">Chemical standards naringenin chalcone (HY-N3007), naringenin (HY-N0100), kaempferol (HY-14590), dihydrokaempferol (HY-N2897), leucocyanidin (HY-119580), and dihydroquercetin (HY-N0136) were purchased from MedChemExpress (Monmouth Junction, NJ). Cyanidin (chloride) No. 14803 was purchased from Cayman Chemical (Ann Arbor, MI). Salicylic acid (Product No. 247588), MWGF1000 Gel Filtration Markers Kit, and HPLC-grade solvents were obtained from MilliporeSigma (Burlington, MA). Salicylic acid glucoside (Catalog No. sc-473013) and [<sup>13</sup>C<sub>6</sub>]-salicylic acid (Catalog No. sc-220088) were sourced from Santa Cruz Biotechnology (Dallas, TX). Antibodies to <em>Arabidopsis thaliana</em> UVR8 (PHY0809A), Actin (AT3G12110, PHY0001), goat anti-rabbit IgG H&amp;L (HRP, PHY6000), and goat anti-mouse IgG H&amp;L (HRP, PHY6006) were acquired from PhytoAB (San Jose, CA). The anti-plant actin monoclonal antibody (A17309) was purchased from Antibodies.com LLC (St. Louis, MO). The IRDye<sup>®</sup> 680RD goat anti-mouse IgG secondary antibody was obtained from LI-COR Biotechnology (Lincoln, NE). Platinum Taq DNA Polymerase High Fidelity and pCR Blunt II TOPO vector were purchased from ThermoFisher Scientific (Waltham, MA). <em>Bam</em>HI, <em>Hind</em>III, and T4 DNA ligase were purchased from Bio-Rad Laboratories (Hercules, CA). Ni-NTA resin for His-tag purification was purchased from QIAGEN (Hilden, Germany).</p>
<p id="Par31"><em>Arabidopsis</em> plant pictures were captured using a Canon PowerShot G12 camera. Light spectra were measured with a LI-180 Spectrometer from LI-COR Biosciences (Lincoln, NE). Detection of sinapoylmalate was carried out using Portable UVP 3UV Lamps from Analytik Jena (Upland, CA). To block UV-B light, an Acrylic UV Filter Plexiglass Sheet OP3/UF-5 from ePlastics (San Diego, CA) was employed. The amount of UV light was quantified using a Digital UVA/UVB Meter #UV513AB (280–400 nm) from General Tools (New York City, NY). Digital Ultraviolet Radiometer Model 6.2 UVB from Solar Light Company (Glenside, PA) was used to measure UV-B intensity. An additional source of UV-B was provided by Philips UVB Broadband TL 40 W/12 RS SLV/25 tubes. Western blot results were processed using the Mini-Med 90 X-Ray film processor from AFP manufacturing (Peachtree City, GA) with GeneMate blue basic autorad film, the iBright<sup>TM</sup> CL1500 Imaging System from Invitrogen (Waltham, MA), or the Sapphire Biomolecular Imager from Azure Biosystems (Dublin, CA). Induced proteins were solubilized using a Misonix S-4000 sonicator ultrasonic liquid processor (Newtown, CT). Size-exclusion chromatography (SEC) was conducted on a Cytiva ÄKTA GO protein purification system from Danaher Corporation (Washington, D.C.). NanoDSF and MST were performed on Monolith NT.Automated instrument from NanoTemper Technologies (San Francisco, CA). Chemical structures were created using ChemDraw 19.0, and graphs were generated using Prism 10.1.1.</p></section><section id="Sec14"><h3 class="pmc_sec_title">Plant materials and growth conditions</h3>
<p id="Par32"><em>Arabidopsis thaliana</em> (L.) accession Columbia-0 (Col-0), along with <em>tt4-11</em> (SALK_020583), <em>tt5-2</em> (GK_176H03), <em>tt5-4</em> (SAIL_641_E10)<sup><a href="#CR64" class="usa-link" aria-describedby="CR64">64</a></sup>, <em>tt8-6</em> (SALK_033468), <em>PAP1-D</em> (CS3884), <em>tt6-2</em> (GK_292E08), and <em>tt19-8</em> (SALK_105779) lines, were obtained from the Arabidopsis Biological Resource Center (Columbus, OH). All plants were cultivated in Suremix growth medium (Michigan Grower Products Inc, MI) under controlled conditions (22 °C and a 16 h light/8 h dark photoperiod) unless otherwise specified. For experiments involving HL or natural Sunlight, seedlings were grown for 7 days in MS media and then transferred to soil under the appropriate light conditions. The combination of light bulbs in High R + FR chamber consists of four fluorescent tubes (S-19983, 48” T8 Daylight, SYLVANIA<sup>®</sup>, Wilmington, MA) and two incandescent bulbs (S3810, A15 40 Watt-Clear-Medium Base, Satco, Brentwood, NY); whereas, the Low R + FR chamber has five fluorescent tubes but no incandescent bulbs. High-light conditions were obtained by the action of eight fluorescent tubes and four incandescent bulbs.</p></section><section id="Sec15"><h3 class="pmc_sec_title">Constructs, plasmid generation, and plant transformation</h3>
<p id="Par33">To generate the <em>TT5pro::TT5-mCherry</em> complemented lines, three constructs containing the <em>TT5</em> promoter region (161 nt upstream of TSS), the <em>TT5</em> CDS (741 nt), and <em>mCherry</em> CDS (711 nt) were synthesized by GeneUniversal (Dewark, DE). These constructs were integrated into the pGWB1 binary vector, resulting in the <em>TT5pro::TT5-mCherry</em> construct through a MultiSite Gateway LR reaction. For the Am4’CGT overexpression lines, the 4’CGT CDS from <em>Antirrhinum majus</em> (<a href="https://www.ncbi.nlm.nih.gov/nuccore/AB198665.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">AB198665.1</a>, 1374 bp) were synthesized by GeneUniversal and integrated into the pGWB502 binary vector. This resulted in the <em>35S::Am4’CGT</em> construct through a Gateway LR reaction. The resulting <em>TT5pro::TT5-mCherry</em> and <em>35S::Am4’CGT</em> constructs were introduced into <em>Agrobacterium tumefaciens</em> (GV3101) and used for <em>Arabidopsis</em> plant transformation via the floral dip method<sup><a href="#CR65" class="usa-link" aria-describedby="CR65">65</a></sup>. Transformed lines were selected by culturing surface-sterilized seeds on Murashige and Skoog (MS) solid media with 5% sucrose and hygromycin (50 mg/L). Primers for confirming T-DNA insertions (listed in Supplementary Table <a href="#MOESM1" class="usa-link">5</a>) were synthesized by Integrated DNA Technologies (Skokie, Il).</p>
<p id="Par34">For His-SUMO-UVR8/UVR8<sup>D96N/D107N</sup>/UVR8<sup>W285F</sup> purification, the wild-type UVR8 CDS (1323 nt) with restriction enzyme recognition sites (<em>Bam</em>HI and <em>Hind</em>III) was amplified from the <a href="https://www.ncbi.nlm.nih.gov/nuccore/U22080" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">U22080</a> construct that obtained from the ABRC and then subcloned into the pCR Blunt II TOPO vector. The constitutive monomer UVR8<sup>D96N/D107N</sup> and constitutive dimer UVR8<sup>W285F</sup> constructs were synthesized by GeneUniversal with flanking <em>Bam</em>HI and <em>Hind</em>III sites into the pUC57 vector. Restriction enzyme-released UVR8 fragments and linearized vector (modified pET28b vector with 6xHis-SUMO tag at N-terminus) were ligated and then transformed into <em>E. coli</em> (BL21) for subsequent protein induction and purification.</p></section><section id="Sec16"><h3 class="pmc_sec_title">Salicylic acid and salicylic acid glucoside quantification</h3>
<p id="Par35"><em>Arabidopsis</em> tissues were ground in liquid nitrogen (N<sub>2</sub>) and lyophilized before storage. In brief, 5 mg of dried samples were soaked in 100 μL of extraction buffer (80:20 v/v methanol:water, 0.1% formic acid, and 100 nM of [<sup>13</sup>C<sub>6</sub>]-SA as an internal standard) at 4 °C for 16 h. The sample was then diluted with an equal volume (100 μL) of water containing 0.1% formic acid. After centrifugation at 12,000 x <em>g</em> for 10 min, the supernatant was analyzed by LC-MS/MS. The amounts of SA and SAG in each sample were detected by multiple reaction monitoring (MRM) and quantified by comparing with SA and SAG standards. For creating standard curves of SA and SAG, the SA and SAG standards were dissolved in 80% of methanol with 0.1% formic acid. The UHPLC-MS/MS run lasted 5 min with a flow rate of 0.4 ml/min at 40 °C. The gradient was as follows with solvent A (0.1% formic acid in water) and solvent B (methanol): 0-0.5 min, 98% A and 2% B; 0.5-3.0 min, 30% A and 70% B; 3.0-4.0 min, 100% B; 4.0-5.0 min, 98% A and 2% B.</p></section><section id="Sec17"><h3 class="pmc_sec_title">Untargeted metabolic profiling and LC-MS/MS data analysis</h3>
<p id="Par36">For acid hydrolysis, 10 mg of dry weight tissue was resuspended in 100 μL of 2 M HCl containing 1 μM telmisartan (used as an internal standard) and incubated at 100 °C for 20 min. Aglycone flavonoids were then extracted in 50 μL isoamyl alcohol. To extract total metabolites, lyophilized <em>Arabidopsis</em> tissue was extracted in methanol/water (1:1, v/v) containing 0.1% formic acid and 1 μM telmisartan at a ratio of 1 mg dry tissue in 20 μL extraction buffer overnight at room temperature. After centrifugation at 10,000 x <em>g</em> for 5 min, the supernatant was diluted ten times with 80% of methanol containing 0.1% formic acid for LC-MS/MS analysis.</p>
<p id="Par37">LC-MS/MS analyses were conducted using an Acquity UHPLC system with a HSS T3 C18 column (100 × 2.1 mm, 1.8 μM, Waters, Milford, MA) connected to the mass spectrometer (Xevo G2-XS QTOF, Waters). Runs were performed at a flow rate of 0.3 mL/min and column temperature of 40 °C. The run involved a 10 min gradient with solvent A (0.1% formic acid in water) and solvent B (acetonitrile): 0-0.5 min, 100% A; 0.5-6 min, 50% A and 50% B; 6.01-6.5 min, 1% A and 99% B; 6.51-8.5 min, 1% A and 99% B; and 8.51-10 min 99% A and 1% B. Mass spectra acquisition was in positive mode with an <em>m/z</em> scan range of 50– 2000. Absorbance was recorded by a diode array detector (DAD) with a range of 200–800 nm. All MS raw data files (.raw) were processed using Waters Progenesis QI v3.0. NGC (<em>m/z</em> 273.08_6.31 min) and naringenin (<em>m/z</em> 273.08_6.43 min) were confirmed by tandem MS profiles, and MS intensity was recorded by Progenesis. The identification of NGC derivatives was achieved by analyzing the mass spectra using Waters MassLynx v4.2 SCN966. The MS raw data files were scanned by the mass chromatogram function with description <em>m/z</em> 273.08.</p></section><section id="Sec18"><h3 class="pmc_sec_title">EMS mutagenesis and whole genome re-sequencing to identify <em>tt5</em> mutant suppressors</h3>
<p id="Par38">EMS-treated <em>tt5-2</em> seeds (2.0 g, ~ 100,000) were planted on soil, grown to maturity, and collected into ~ 100 pools of M<sub>1</sub> seeds. Recessive suppressors that restored the leaf early senescence phenotype were isolated from 4-week-old M<sub>2</sub> plants under high light conditions. Genomic DNA from pooled leaves of ~ 50 <em>Arabidopsis</em> BC<sub>1</sub>F<sub>3</sub>
<em>tt5</em> suppressor plants and ~ 50 <em>tt5</em>-like plants were extracted using the Wizard Genomic DNA Purification Kit (Promega, WI). The pooled gDNA samples were then submitted to BGI genomic services (<a href="http://www.genomics.cn/en/index" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://www.genomics.cn/en/index</a>) for 100 bp paired-end Illumina sequencing. Raw data quality was assessed using FastQC (v0.11.5)<sup><a href="#CR66" class="usa-link" aria-describedby="CR66">66</a></sup>. Reads were aligned to the <em>Arabidopsis</em> reference genome (TAIR10) using Bowtie2 (v2.3.4)<sup><a href="#CR67" class="usa-link" aria-describedby="CR67">67</a></sup>. In total, ~ 236 million paired-end reads were obtained. The alignment files were sorted and converted to BAM files<sup><a href="#CR68" class="usa-link" aria-describedby="CR68">68</a></sup>. SNP calling was performed using the Single Sample Variants Detector module of NGSEP<sup><a href="#CR69" class="usa-link" aria-describedby="CR69">69</a></sup>, and the conversion into a VCF matrix was carried out using bcftools (v1.2)<sup><a href="#CR68" class="usa-link" aria-describedby="CR68">68</a></sup>. Variants were parsed using vcftools (v0.1.15)<sup><a href="#CR68" class="usa-link" aria-describedby="CR68">68</a></sup> to include only high-quality single SNPs. High-quality SNPs were defined as those with a minimum read depth of 5, a maximum read depth of 200, and a minimum mapping quality greater than 20 (--minDP 5; --MaxDP 200; --MinMQ 20; --remove-indels). SNP frequency was then calculated and used to filter each of the suppressor lines. The filtering criteria were based on common SNPs in each suppressor line, with an allele frequency greater than 0.8 in the samples from <em>tt5</em> suppressor plants and less than 0.4 in the corresponding samples from <em>tt5</em>-like plants.</p></section><section id="Sec19"><h3 class="pmc_sec_title">Extraction of total proteins and immunoblot analysis</h3>
<p id="Par39">Tissue samples were ground into powder with liquid nitrogen (N<sub>2</sub>) and resuspended in protein extraction buffer (150 mM NaCl, 2 mM EDTA, 10% Glycerol, 50 mM Tris-HCl, pH 7.6, and cOmplete™, EDTA-free Protease Inhibitor Cocktail). After 10 min of incubation on ice, samples were centrifuged at 13,000 x <em>g</em> for 10 min at 4 °C. Protein concentration in the supernatant was determined using the Bradford assay<sup><a href="#CR70" class="usa-link" aria-describedby="CR70">70</a></sup>.</p>
<p id="Par40">For NGC incubation, 50 or 200 µM of NGC was mixed with total protein extracts (~ 4 µg) at 4 °C for 16 h. For incubation with compound extracts, compounds extracted from ~ 0.25 or ~ 1.0 mg dry weight of <em>tt5-2</em> mutant under HL with or without acid hydrolysis were lyophilized and then resuspended with total protein extracts (~ 4 µg) at 4 °C for 16 h.</p>
<p id="Par41">Protein extracts (~ 4 µg total protein) were loaded on 10% polyacrylamide gels without heating and without a reducing agent, and the protein gel was exposed to 300 µW/cm<sup>2</sup> UV-B for 30 min before transfer to a membrane to detect UVR8 dimers<sup><a href="#CR18" class="usa-link" aria-describedby="CR18">18</a></sup> or under denatured conditions (samples were boiled for 10 min) to detect total UVR8s using anti-UVR8 (PhytoAB Inc, CA, catalog # PHY0809A) (1:4000) and goat anti-rabbit IgG H&amp;L (HRP, 1:5000) with ECL Prime Western Blotting Detection Reagents (Cytiva, Marlborough, MA). Membranes were probed for plant Actin levels using anti-plant Actin monoclonal antibody (1:5000) and IRDye 680RD goat anti-mouse IgG secondary antibody (1:10,000) with the Sapphire Biomolecular Imager.</p></section><section id="Sec20"><h3 class="pmc_sec_title">Size-exclusion chromatography, NanoDSF, and MST</h3>
<p id="Par42">Recombinant proteins His-SUMO-UVR8/UVR8<sup>D96N/D107N</sup>/UVR8<sup>W285F</sup> were induced by 0.2 mM IPTG at 18 °C for 16 h. Proteins were resuspended by sonication with lysis buffer (150 mM NaCl, 25 mM Tris, pH 8.0, 2 mM β-Mercaptoethanol, and 10 mM Imidazole) and then purified through Ni-NTA resin following the manufacturer’s recommendations with elution buffer (150 mM NaCl, 25 mM Tris, pH 8.0, 2 mM 2-Mercaptoethanol, and 250 mM Imidazole). The eluted samples were buffer-changed (150 mM NaCl, 25 mM Tris, pH 8.0, and 5 mM DTT) by size exclusion chromatography using Superdex 200 Increase 10/300 GL (flow rate at 0.4 mL/min) and collected as 0.25 mL fractions. The fractions containing His-SUMO-UVR8/UVR8<sup>D96N/D107N</sup>/UVR8<sup>W285F</sup> were combined and then concentrated through Amicon Ultra-15 centrifugal filters (MilliporeSigma). Protein concentration was determined by Bradford and adjusted to ~ 0.6 mg/mL. Purified His-SUMO-UVR8/UVR8<sup>D96N/D107N</sup>/UVR8<sup>W285F</sup> were treated with UV-B (300 µW/cm<sup>2</sup>, 60 min) on ice. For nanoDSF and MST experiments, 100 µM of flavonoid compounds (NGC, naringenin, kaempferol, dihydrokaempferol, leucocyanidin, dihydroquercetin, or cyanidin) were incubated with purified UVR8 proteins (~ 1.5 µg) at 4 °C for an hour.</p></section><section id="Sec21"><h3 class="pmc_sec_title">Prediction the binding sites of NGC on UVR8 proteins</h3>
<p id="Par43">The Protenix Server can be accessed at <a href="http://101.126.11.40:8000" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://101.126.11.40:8000</a>. To generate the predicted interactions between NGC and UVR8, we enabled both Multiple Sequence Alignment (MSA) and Evolutionary Scale Modeling (ESM) by selecting the “true” options. The input was uploaded as a CIF file, with MSA features activated to enhance folding accuracy by incorporating evolutionary information. The sequences for UVR8 and NGC were added along with the desired number of copies for the interaction prediction model. For molecular classification, UVR8 was designated as a protein, while NGC was entered using its Ligand Smiles representation (C1 = CC( = CC = C1/C = C/C( = O)C2 = C(C = C(C = C2O)O)O)O). Once all components were included, the interaction prediction request was submitted, and the results were received via email. The server provided five samples of predicted interactions in CIF and JSON file formats. CIF files were used to visualize molecular binding in PyMOL, while JSON files contained additional information such as the interface predicted template modeling (ipTM) score for each model.</p>
<p id="Par44">To analyze and visualize the NGC-UVR8 interaction and binding sites, we used PyMOL. The NGC molecule was selected, and by navigating to Action &gt; Modify &gt; Around &gt; residues within 4 Å of the NGC molecule were identified. Additionally, all polar contacts were located by selecting Action &gt; Find &gt; Polar Contact &gt; to any atoms, where binding sites were highlighted using red dashed lines.</p>
<p id="Par45">For binding free energy calculations, PyFepRestr was utilized as a PyMOL plug-in. The input parameters, including temperature (300 Kelvin) and force constant values (K raA, K θa, K θA, K φba, K φaA, and K φAB), were set to 10 kcal/mol/Å² and 10 kcal/mol/rad², respectively. Using the “Select 6 Atoms” function within the plug-in, three atoms from binding amino acids in UVR8 and three atoms from NGC were chosen. The plug-in then computed free energy values (Δ<em>G</em>_off and Δ<em>G</em>_on) in kJ/mol, providing insights into the interaction strength between NGC and UVR8.</p></section><section id="Sec22"><h3 class="pmc_sec_title">RNA-seq analysis</h3>
<p id="Par46">Total RNA was extracted from <em>Arabidopsis</em> plants using QIAGEN RNeasy Plant Mini Kit following the manufacturer’s instructions. Approximately 1 μg of RNA was utilized for constructing the RNA-seq libraries, which were then sequenced with 100 bp paired-ends on an Illumina sequencing platform by BGI. Each sample yielded ~ 24 million paired-end reads. RNA-seq reads underwent quality assessment using FastQC (v0.11.5)<sup><a href="#CR66" class="usa-link" aria-describedby="CR66">66</a></sup>. Subsequently, the reads were aligned to the <em>Arabidopsis thaliana</em> reference genome (TAIR 10) using Hisat2 (v2.1)<sup><a href="#CR71" class="usa-link" aria-describedby="CR71">71</a></sup>. Raw read counts for gene features were quantified from these alignments using FeatureCounts (v1.5)<sup><a href="#CR72" class="usa-link" aria-describedby="CR72">72</a></sup>. Transcript levels were calculated in transcripts per million (TPM) with StringTie (v2.1.1)<sup><a href="#CR73" class="usa-link" aria-describedby="CR73">73</a></sup>. The R package DESeq2 (v1.36.0)<sup><a href="#CR74" class="usa-link" aria-describedby="CR74">74</a></sup> was employed to test for differential expression (DE) through paired contrasts between HL and NL. The resulting <em>P</em>-values were adjusted using Benjamini and Hochberg’s method<sup><a href="#CR75" class="usa-link" aria-describedby="CR75">75</a></sup>. Genes with an adjusted <em>P</em>-value &lt; 0.05, as determined by DESeq2, were considered differentially expressed, regardless of the fold change value, unless otherwise specified. Venn diagrams were generated using InteractiVenn (<a href="http://www.interactivenn.net/index.html" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://www.interactivenn.net/index.html#</a>)<sup><a href="#CR76" class="usa-link" aria-describedby="CR76">76</a></sup>. Gene ontology (GO) enrichment analyses were performed using the online tool ShinyGO v0.76: Gene Ontology Enrichment Analysis (<a href="http://bioinformatics.sdstate.edu/goc/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://bioinformatics.sdstate.edu/goc/</a>)<sup><a href="#CR77" class="usa-link" aria-describedby="CR77">77</a></sup>.</p></section><section id="Sec23"><h3 class="pmc_sec_title">Statistical analyses</h3>
<p id="Par47">Statistical analyses were conducted using GraphPad Prism 10.1.1 (San Diego, California).</p></section><section id="Sec24"><h3 class="pmc_sec_title">Reporting summary</h3>
<p id="Par48">Further information on research design is available in the <a href="#MOESM11" class="usa-link">Nature Portfolio Reporting Summary</a> linked to this article.</p></section></section><section id="Sec25"><h2 class="pmc_sec_title">Supplementary information</h2>
<section class="sm xbox font-sm" id="MOESM1"><div class="media p"><div class="caption">
<a href="/articles/instance/12371036/bin/41467_2025_63010_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information</a><sup> (3.3MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM2"><div class="media p"><div class="caption">
<a href="/articles/instance/12371036/bin/41467_2025_63010_MOESM2_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Peer Review file</a><sup> (1.2MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM3"><div class="media p"><div class="caption">
<a href="/articles/instance/12371036/bin/41467_2025_63010_MOESM3_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">41467_2025_63010_MOESM3_ESM.pdf</a><sup> (6.9KB, pdf) </sup><p>Description of Additional Supplementary Files</p>
</div></div></section><section class="sm xbox font-sm" id="MOESM4"><div class="media p"><div class="caption">
<a href="/articles/instance/12371036/bin/41467_2025_63010_MOESM4_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 1</a><sup> (118KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM5"><div class="media p"><div class="caption">
<a href="/articles/instance/12371036/bin/41467_2025_63010_MOESM5_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 2</a><sup> (89.7KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM6"><div class="media p"><div class="caption">
<a href="/articles/instance/12371036/bin/41467_2025_63010_MOESM6_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 3</a><sup> (85.4KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM7"><div class="media p"><div class="caption">
<a href="/articles/instance/12371036/bin/41467_2025_63010_MOESM7_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 4</a><sup> (40.9KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM8"><div class="media p"><div class="caption">
<a href="/articles/instance/12371036/bin/41467_2025_63010_MOESM8_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 5</a><sup> (37.5KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM9"><div class="media p"><div class="caption">
<a href="/articles/instance/12371036/bin/41467_2025_63010_MOESM9_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 6</a><sup> (23.8KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM10"><div class="media p"><div class="caption">
<a href="/articles/instance/12371036/bin/41467_2025_63010_MOESM10_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 7</a><sup> (32.4KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM11"><div class="media p"><div class="caption">
<a href="/articles/instance/12371036/bin/41467_2025_63010_MOESM11_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Reporting Summary</a><sup> (107.2KB, pdf) </sup>
</div></div></section></section><section id="Sec26"><h2 class="pmc_sec_title">Source data</h2>
<section class="sm xbox font-sm" id="MOESM12"><div class="media p"><div class="caption">
<a href="/articles/instance/12371036/bin/41467_2025_63010_MOESM12_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Source Data</a><sup> (289.9MB, xlsx) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>We thank Rob Last for insightful comments on this study, and members of his research team for helpful suggestions. We are grateful to Marisa Otegui and Gregg Howe for critical comments and valuable suggestions on this work. We appreciate the technical assistance provided by Julieta Petrich and Suresh Kumar Gupta, and the advice provided by the staff of the MSU Metabolomics Core. Even though we ended not using them in this study, we thank Gareth Jenkins for providing RUP1 and RUP2 antibodies, and Gayani Ekanayake and David Mackey for providing <em>Pseudomonas syringae-</em>treated <em>Arabidopsis</em> samples. This research was supported in part by grants from the National Science Foundation MCB-1822343 to E.G. and IOS-1733633 to E.G.</p></section><section id="notes1"><h2 class="pmc_sec_title">Author contributions</h2>
<p>N.J. and E.G. designed the research; N.J. performed most of the research and analyses; T.G.N. contributed with the sequencing of the mutants and RNA-seq analysis; Y.S.L. performed nanoDSF and MST experiments; E.P. conducted protein expression and purification; S.P. contributed with <em>Arabidopsis</em> growth in Hawai’i and simulation between protein and compound; N.J. and E.G. wrote the manuscript.</p></section><section id="notes2"><h2 class="pmc_sec_title">Peer review</h2>
<section id="FPar1"><h3 class="pmc_sec_title">Peer review information</h3>
<p id="Par49"><em>Nature Communications</em> thanks Hongtao Liu, Andreas Richter and the other anonymous reviewer(s) for their contribution to the peer review of this work. A peer review file is available.</p></section></section><section id="notes3"><h2 class="pmc_sec_title">Data availability</h2>
<p>Data supporting the findings of this work are available within the paper and the respective Supplementary Information files. A reporting summary for this article is available as a Supplementary Information file. The raw RNA-seq reads and whole genome re-sequencing data generated in this study have been deposited in the NCBI BioProject database under accession codes <a href="https://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA1065785" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PRJNA1065785</a>, <a href="https://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA1231375" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PRJNA1231375</a>, and <a href="https://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA1065513" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PRJNA1065513</a>. The metabolomics data have been deposited to MetaboLights repository with the study identifier <a href="https://www.ebi.ac.uk/metabolights/MTBLS12792" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">MTBLS12792</a>. <a href="#Sec26" class="usa-link">Source data</a> are provided in this paper.</p></section><section id="FPar2"><h2 class="pmc_sec_title">Competing interests</h2>
<p id="Par50">The authors declare no competing interests.</p></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="fn1"><p><strong>Publisher’s note</strong> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></div></div></section><section id="sec27"><h2 class="pmc_sec_title">Supplementary information</h2>
<p>The online version contains supplementary material available at 10.1038/s41467-025-63010-3.</p></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>McCree, K. J. Test of current definitions of photosynthetically active radiation against leaf photosynthesis data. <em>Agric. Meteorol.</em><strong>10</strong>, 443–453 (1972).</cite> [<a href="https://scholar.google.com/scholar_lookup?McCree,%20K.%20J.%20Test%20of%20current%20definitions%20of%20photosynthetically%20active%20radiation%20against%20leaf%20photosynthesis%20data.%20Agric.%20Meteorol.10,%20443%E2%80%93453%20(1972)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>Kami, C., Lorrain, S., Hornitschek, P. &amp; Fankhauser, C. Light-regulated plant growth and development. <em>Curr. Top. Dev. Biol.</em><strong>91</strong>, 29–66 (2010).
</cite> [<a href="https://doi.org/10.1016/S0070-2153(10)91002-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20705178/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kami,%20C.,%20Lorrain,%20S.,%20Hornitschek,%20P.%20&amp;%20Fankhauser,%20C.%20Light-regulated%20plant%20growth%20and%20development.%20Curr.%20Top.%20Dev.%20Biol.91,%2029%E2%80%9366%20(2010)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>Möglich, A., Yang, X., Ayers, R. A. &amp; Moffat, K. Structure and function of plant photoreceptors. <em>Annu. Rev. Plant Biol.</em><strong>61</strong>, 21–47 (2010).
</cite> [<a href="https://doi.org/10.1146/annurev-arplant-042809-112259" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20192744/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?M%C3%B6glich,%20A.,%20Yang,%20X.,%20Ayers,%20R.%20A.%20&amp;%20Moffat,%20K.%20Structure%20and%20function%20of%20plant%20photoreceptors.%20Annu.%20Rev.%20Plant%20Biol.61,%2021%E2%80%9347%20(2010)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Li, X., Liang, T. &amp; Liu, H. How plants coordinate their development in response to light and temperature signals. <em>Plant Cell</em><strong>34</strong>, 955–966 (2022).
</cite> [<a href="https://doi.org/10.1093/plcell/koab302" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8894937/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34904672/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li,%20X.,%20Liang,%20T.%20&amp;%20Liu,%20H.%20How%20plants%20coordinate%20their%20development%20in%20response%20to%20light%20and%20temperature%20signals.%20Plant%20Cell34,%20955%E2%80%93966%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Rai, N., Morales, L. O. &amp; Aphalo, P. J. Perception of solar UV radiation by plants: photoreceptors and mechanisms. <em>Plant Physiol.</em><strong>186</strong>, 1382–1396 (2021).
</cite> [<a href="https://doi.org/10.1093/plphys/kiab162" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8260113/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33826733/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rai,%20N.,%20Morales,%20L.%20O.%20&amp;%20Aphalo,%20P.%20J.%20Perception%20of%20solar%20UV%20radiation%20by%20plants:%20photoreceptors%20and%20mechanisms.%20Plant%20Physiol.186,%201382%E2%80%931396%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Caldwell, M. M. Solar UV irradiation and the growth and development of higher plants. <em>Photophysiology</em><strong>6</strong>, 131–177 (1971).</cite> [<a href="https://scholar.google.com/scholar_lookup?Caldwell,%20M.%20M.%20Solar%20UV%20irradiation%20and%20the%20growth%20and%20development%20of%20higher%20plants.%20Photophysiology6,%20131%E2%80%93177%20(1971)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Rozema, J., van de Staaij, J., Björn, L. O. &amp; Caldwell, M. UV-B as an environmental factor in plant life: stress and regulation. <em>Trends Ecol. Evol.</em><strong>12</strong>, 22–28 (1997).
</cite> [<a href="https://doi.org/10.1016/s0169-5347(96)10062-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21237957/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rozema,%20J.,%20van%20de%20Staaij,%20J.,%20Bj%C3%B6rn,%20L.%20O.%20&amp;%20Caldwell,%20M.%20UV-B%20as%20an%20environmental%20factor%20in%20plant%20life:%20stress%20and%20regulation.%20Trends%20Ecol.%20Evol.12,%2022%E2%80%9328%20(1997)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>Stapleton, A. E. Ultraviolet radiation and plants: burning questions. <em>Plant Cell</em><strong>4</strong>, 1353 (1992).
</cite> [<a href="https://doi.org/10.1105/tpc.4.11.1353" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC160223/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12297637/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Stapleton,%20A.%20E.%20Ultraviolet%20radiation%20and%20plants:%20burning%20questions.%20Plant%20Cell4,%201353%20(1992)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>Frohnmeyer, H. &amp; Staiger, D. Ultraviolet-B radiation-mediated responses in plants. Balancing damage and protection. <em>Plant Physiol.</em><strong>133</strong>, 1420–1428 (2003).
</cite> [<a href="https://doi.org/10.1104/pp.103.030049" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1540342/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/14681524/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Frohnmeyer,%20H.%20&amp;%20Staiger,%20D.%20Ultraviolet-B%20radiation-mediated%20responses%20in%20plants.%20Balancing%20damage%20and%20protection.%20Plant%20Physiol.133,%201420%E2%80%931428%20(2003)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>Britt, A. B. Repair of DNA damage induced by ultraviolet radiation. <em>Plant Physiol.</em><strong>108</strong>, 891 (1995).
</cite> [<a href="https://doi.org/10.1104/pp.108.3.891" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC157437/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/7630970/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Britt,%20A.%20B.%20Repair%20of%20DNA%20damage%20induced%20by%20ultraviolet%20radiation.%20Plant%20Physiol.108,%20891%20(1995)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Foyer, C. H., Lelandais, M. &amp; Kunert, K. J. Photooxidative stress in plants. <em>Physiol. Plant.</em><strong>92</strong>, 696–717 (1994).</cite> [<a href="https://scholar.google.com/scholar_lookup?Foyer,%20C.%20H.,%20Lelandais,%20M.%20&amp;%20Kunert,%20K.%20J.%20Photooxidative%20stress%20in%20plants.%20Physiol.%20Plant.92,%20696%E2%80%93717%20(1994)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Hahlbrock, K. &amp; Scheel, D. Physiology and molecular biology of phenylpropanoid metabolism. <em>Annu. Rev. Plant Biol.</em><strong>40</strong>, 347–369 (1989).</cite> [<a href="https://scholar.google.com/scholar_lookup?Hahlbrock,%20K.%20&amp;%20Scheel,%20D.%20Physiology%20and%20molecular%20biology%20of%20phenylpropanoid%20metabolism.%20Annu.%20Rev.%20Plant%20Biol.40,%20347%E2%80%93369%20(1989)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>Landry, L. G., Chapple, C. C. S. &amp; Last, R. L. Arabidopsis mutants lacking phenolic sunscreens exhibit enhanced ultraviolet-B injury and oxidative damage. <em>Plant Physiol.</em><strong>109</strong>, 1159–1166 (1995).
</cite> [<a href="https://doi.org/10.1104/pp.109.4.1159" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC157646/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/8539286/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Landry,%20L.%20G.,%20Chapple,%20C.%20C.%20S.%20&amp;%20Last,%20R.%20L.%20Arabidopsis%20mutants%20lacking%20phenolic%20sunscreens%20exhibit%20enhanced%20ultraviolet-B%20injury%20and%20oxidative%20damage.%20Plant%20Physiol.109,%201159%E2%80%931166%20(1995)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>Winkel-Shirley, B. Biosynthesis of flavonoids and effects of stress. <em>Curr. Opin. Plant Biol.</em><strong>5</strong>, 218–223 (2002).
</cite> [<a href="https://doi.org/10.1016/s1369-5266(02)00256-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11960739/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Winkel-Shirley,%20B.%20Biosynthesis%20of%20flavonoids%20and%20effects%20of%20stress.%20Curr.%20Opin.%20Plant%20Biol.5,%20218%E2%80%93223%20(2002)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>Fiscus, E. L., Philbeck, R., Britt, A. B. &amp; Booker, F. L. Growth of Arabidopsis flavonoid mutants under solar radiation and UV filters. <em>Environ. Exp. Bot.</em><strong>41</strong>, 231–245 (1999).</cite> [<a href="https://scholar.google.com/scholar_lookup?Fiscus,%20E.%20L.,%20Philbeck,%20R.,%20Britt,%20A.%20B.%20&amp;%20Booker,%20F.%20L.%20Growth%20of%20Arabidopsis%20flavonoid%20mutants%20under%20solar%20radiation%20and%20UV%20filters.%20Environ.%20Exp.%20Bot.41,%20231%E2%80%93245%20(1999)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Kliebenstein, D. J., Lim, J. E., Landry, L. G. &amp; Last, R. L. Arabidopsis UVR8 regulates ultraviolet-B signal transduction and tolerance and contains sequence similarity to human regulator of chromatin condensation 1. <em>Plant Physiol.</em><strong>130</strong>, 234–243 (2002).
</cite> [<a href="https://doi.org/10.1104/pp.005041" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC166556/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12226503/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kliebenstein,%20D.%20J.,%20Lim,%20J.%20E.,%20Landry,%20L.%20G.%20&amp;%20Last,%20R.%20L.%20Arabidopsis%20UVR8%20regulates%20ultraviolet-B%20signal%20transduction%20and%20tolerance%20and%20contains%20sequence%20similarity%20to%20human%20regulator%20of%20chromatin%20condensation%201.%20Plant%20Physiol.130,%20234%E2%80%93243%20(2002)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>Clayton, W. A. et al. UVR8-mediated induction of flavonoid biosynthesis for UVB tolerance is conserved between the liverwort Marchantia polymorpha and flowering plants. <em>Plant J.</em><strong>96</strong>, 503–517 (2018).
</cite> [<a href="https://doi.org/10.1111/tpj.14044" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30044520/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Clayton,%20W.%20A.%20et%20al.%20UVR8-mediated%20induction%20of%20flavonoid%20biosynthesis%20for%20UVB%20tolerance%20is%20conserved%20between%20the%20liverwort%20Marchantia%20polymorpha%20and%20flowering%20plants.%20Plant%20J.96,%20503%E2%80%93517%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Rizzini, L. et al. Perception of UV-B by the Arabidopsis UVR8 protein. <em>Science</em><strong>332</strong>, 103–106 (2011).
</cite> [<a href="https://doi.org/10.1126/science.1200660" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21454788/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rizzini,%20L.%20et%20al.%20Perception%20of%20UV-B%20by%20the%20Arabidopsis%20UVR8%20protein.%20Science332,%20103%E2%80%93106%20(2011)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Jenkins, G. I. The UV-B photoreceptor UVR8: from structure to physiology. <em>Plant Cell</em><strong>26</strong>, 21–37 (2014).
</cite> [<a href="https://doi.org/10.1105/tpc.113.119446" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3963570/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24481075/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jenkins,%20G.%20I.%20The%20UV-B%20photoreceptor%20UVR8:%20from%20structure%20to%20physiology.%20Plant%20Cell26,%2021%E2%80%9337%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>Findlay, K. M. W. &amp; Jenkins, G. I. Regulation of UVR8 photoreceptor dimer/monomer photo-equilibrium in Arabidopsis plants grown under photoperiodic conditions. <em>Plant Cell Environ.</em><strong>39</strong>, 1706–1714 (2016).
</cite> [<a href="https://doi.org/10.1111/pce.12724" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5103188/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26864532/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Findlay,%20K.%20M.%20W.%20&amp;%20Jenkins,%20G.%20I.%20Regulation%20of%20UVR8%20photoreceptor%20dimer/monomer%20photo-equilibrium%20in%20Arabidopsis%20plants%20grown%20under%20photoperiodic%20conditions.%20Plant%20Cell%20Environ.39,%201706%E2%80%931714%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>aiserli, E. &amp; Jenkins, G. I. UV-B Promotes rapid nuclear translocation of the arabidopsis UV-B–specific signaling component UVR8 and activates its function in the nucleus. <em>Plant Cell</em><strong>19</strong>, 2662–2673 (2007).
</cite> [<a href="https://doi.org/10.1105/tpc.107.053330" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2002606/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17720867/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?aiserli,%20E.%20&amp;%20Jenkins,%20G.%20I.%20UV-B%20Promotes%20rapid%20nuclear%20translocation%20of%20the%20arabidopsis%20UV-B%E2%80%93specific%20signaling%20component%20UVR8%20and%20activates%20its%20function%20in%20the%20nucleus.%20Plant%20Cell19,%202662%E2%80%932673%20(2007)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>Araguirang, G. E. &amp; Richter, A. S. Activation of anthocyanin biosynthesis in high light–what is the initial signal?. <em>N. Phytol.</em><strong>236</strong>, 2037–2043 (2022).</cite> [<a href="https://doi.org/10.1111/nph.18488" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36110042/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Araguirang,%20G.%20E.%20&amp;%20Richter,%20A.%20S.%20Activation%20of%20anthocyanin%20biosynthesis%20in%20high%20light%E2%80%93what%20is%20the%20initial%20signal?.%20N.%20Phytol.236,%202037%E2%80%932043%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>Favory, J. J. et al. Interaction of COP1 and UVR8 regulates UV-B-induced photomorphogenesis and stress acclimation in Arabidopsis. <em>EMBO J.</em><strong>28</strong>, 591–601 (2009).
</cite> [<a href="https://doi.org/10.1038/emboj.2009.4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2657586/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19165148/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Favory,%20J.%20J.%20et%20al.%20Interaction%20of%20COP1%20and%20UVR8%20regulates%20UV-B-induced%20photomorphogenesis%20and%20stress%20acclimation%20in%20Arabidopsis.%20EMBO%20J.28,%20591%E2%80%93601%20(2009)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>Yin, R., Skvortsova, M. Y., Loubéry, S. &amp; Ulm, R. COP1 is required for UV-B–induced nuclear accumulation of the UVR8 photoreceptor. <em>Proc. Natl. Acad. Sci. USA</em><strong>113</strong>, E4415–E4422 (2016).
</cite> [<a href="https://doi.org/10.1073/pnas.1607074113" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4968759/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27407149/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yin,%20R.,%20Skvortsova,%20M.%20Y.,%20Loub%C3%A9ry,%20S.%20&amp;%20Ulm,%20R.%20COP1%20is%20required%20for%20UV-B%E2%80%93induced%20nuclear%20accumulation%20of%20the%20UVR8%20photoreceptor.%20Proc.%20Natl.%20Acad.%20Sci.%20USA113,%20E4415%E2%80%93E4422%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>Heijde, M. &amp; Ulm, R. Reversion of the Arabidopsis UV-B photoreceptor UVR8 to the homodimeric ground state. <em>Proc. Natl. Acad. Sci. USA</em><strong>110</strong>, 1113–1118 (2013).
</cite> [<a href="https://doi.org/10.1073/pnas.1214237110" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3549095/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23277547/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Heijde,%20M.%20&amp;%20Ulm,%20R.%20Reversion%20of%20the%20Arabidopsis%20UV-B%20photoreceptor%20UVR8%20to%20the%20homodimeric%20ground%20state.%20Proc.%20Natl.%20Acad.%20Sci.%20USA110,%201113%E2%80%931118%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>Podolec, R., Lau, K., Wagnon, T. B., Hothorn, M. &amp; Ulm, R. A constitutively monomeric UVR8 photoreceptor confers enhanced UV-B photomorphogenesis. <em>Proc. Natl. Acad. Sci. USA</em><strong>118</strong>, e2017284118 (2021).
</cite> [<a href="https://doi.org/10.1073/pnas.2017284118" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8017708/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33542100/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Podolec,%20R.,%20Lau,%20K.,%20Wagnon,%20T.%20B.,%20Hothorn,%20M.%20&amp;%20Ulm,%20R.%20A%20constitutively%20monomeric%20UVR8%20photoreceptor%20confers%20enhanced%20UV-B%20photomorphogenesis.%20Proc.%20Natl.%20Acad.%20Sci.%20USA118,%20e2017284118%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Liu, Z. et al. Phytochrome-interacting factors PIF4 and PIF5 negatively regulate anthocyanin biosynthesis under red light in Arabidopsis seedlings. <em>Plant Sci.</em><strong>238</strong>, 64–72 (2015).
</cite> [<a href="https://doi.org/10.1016/j.plantsci.2015.06.001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26259175/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Liu,%20Z.%20et%20al.%20Phytochrome-interacting%20factors%20PIF4%20and%20PIF5%20negatively%20regulate%20anthocyanin%20biosynthesis%20under%20red%20light%20in%20Arabidopsis%20seedlings.%20Plant%20Sci.238,%2064%E2%80%9372%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Casal, J. J. Shade avoidance. <em>Arabidopsis Book</em><strong>10</strong>, e0157 (2012).
</cite> [<a href="https://doi.org/10.1199/tab.0157" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3350169/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22582029/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Casal,%20J.%20J.%20Shade%20avoidance.%20Arabidopsis%20Book10,%20e0157%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>Gonzalez, A., Zhao, M., Leavitt, J. M. &amp; Lloyd, A. M. Regulation of the anthocyanin biosynthetic pathway by the TTG1/bHLH/Myb transcriptional complex in Arabidopsis seedlings. <em>Plant J.</em><strong>53</strong>, 814–827 (2008).
</cite> [<a href="https://doi.org/10.1111/j.1365-313X.2007.03373.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18036197/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gonzalez,%20A.,%20Zhao,%20M.,%20Leavitt,%20J.%20M.%20&amp;%20Lloyd,%20A.%20M.%20Regulation%20of%20the%20anthocyanin%20biosynthetic%20pathway%20by%20the%20TTG1/bHLH/Myb%20transcriptional%20complex%20in%20Arabidopsis%20seedlings.%20Plant%20J.53,%20814%E2%80%93827%20(2008)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR30">
<span class="label">30.</span><cite>Hoshino, A. et al. Generation of yellow flowers of the Japanese morning glory by engineering its flavonoid biosynthetic pathway toward aurones. <em>Plant Cell Physiol.</em><strong>60</strong>, 1871–1879 (2019).
</cite> [<a href="https://doi.org/10.1093/pcp/pcz101" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31135027/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hoshino,%20A.%20et%20al.%20Generation%20of%20yellow%20flowers%20of%20the%20Japanese%20morning%20glory%20by%20engineering%20its%20flavonoid%20biosynthetic%20pathway%20toward%20aurones.%20Plant%20Cell%20Physiol.60,%201871%E2%80%931879%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>Ding, Y. et al. Opposite roles of salicylic acid receptors NPR1 and NPR3/NPR4 in transcriptional regulation of plant immunity. <em>Cell</em><strong>173</strong>, 1454–1467 (2018).
</cite> [<a href="https://doi.org/10.1016/j.cell.2018.03.044" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29656896/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ding,%20Y.%20et%20al.%20Opposite%20roles%20of%20salicylic%20acid%20receptors%20NPR1%20and%20NPR3/NPR4%20in%20transcriptional%20regulation%20of%20plant%20immunity.%20Cell173,%201454%E2%80%931467%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>Genger, R. K. et al. Signaling pathways that regulate the enhanced disease resistance of arabidopsis “Defense, No Death” mutants. <em>Mol. Plant Microbe Interact.</em><strong>21</strong>, 1285–1296 (2008).
</cite> [<a href="https://doi.org/10.1094/MPMI-21-10-1285" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2923831/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18785824/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Genger,%20R.%20K.%20et%20al.%20Signaling%20pathways%20that%20regulate%20the%20enhanced%20disease%20resistance%20of%20arabidopsis%20%E2%80%9CDefense,%20No%20Death%E2%80%9D%20mutants.%20Mol.%20Plant%20Microbe%20Interact.21,%201285%E2%80%931296%20(2008)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Du, L. et al. Ca2+/calmodulin regulates salicylic-acid-mediated plant immunity. <em>Nature</em><strong>457</strong>, 1154–1158 (2009).
</cite> [<a href="https://doi.org/10.1038/nature07612" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19122675/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Du,%20L.%20et%20al.%20Ca2+/calmodulin%20regulates%20salicylic-acid-mediated%20plant%20immunity.%20Nature457,%201154%E2%80%931158%20(2009)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>Shirano, Y., Kachroo, P., Shah, J. &amp; Klessig, D. F. A gain-of-function mutation in an Arabidopsis Toll Interleukin1 Receptor–Nucleotide Binding Site–Leucine-Rich Repeat type R gene triggers defense responses and results in enhanced disease resistance. <em>Plant Cell</em><strong>14</strong>, 3149–3162 (2002).
</cite> [<a href="https://doi.org/10.1105/tpc.005348" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC151208/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12468733/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Shirano,%20Y.,%20Kachroo,%20P.,%20Shah,%20J.%20&amp;%20Klessig,%20D.%20F.%20A%20gain-of-function%20mutation%20in%20an%20Arabidopsis%20Toll%20Interleukin1%20Receptor%E2%80%93Nucleotide%20Binding%20Site%E2%80%93Leucine-Rich%20Repeat%20type%20R%20gene%20triggers%20defense%20responses%20and%20results%20in%20enhanced%20disease%20resistance.%20Plant%20Cell14,%203149%E2%80%933162%20(2002)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>Hertog, M. G. L., Hollman, P. C. H. &amp; Venema, D. P. Optimization of a quantitative HPLC determination of potentially anticarcinogenic flavonoids in vegetables and fruits. <em>J. Agric. Food Chem.</em><strong>40</strong>, 1591–1598 (2002).</cite> [<a href="https://scholar.google.com/scholar_lookup?Hertog,%20M.%20G.%20L.,%20Hollman,%20P.%20C.%20H.%20&amp;%20Venema,%20D.%20P.%20Optimization%20of%20a%20quantitative%20HPLC%20determination%20of%20potentially%20anticarcinogenic%20flavonoids%20in%20vegetables%20and%20fruits.%20J.%20Agric.%20Food%20Chem.40,%201591%E2%80%931598%20(2002)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR36">
<span class="label">36.</span><cite>Lv, H., Wang, X., He, Y., Wang, H. &amp; Suo, Y. Identification and quantification of flavonoid aglycones in rape bee pollen from Qinghai-Tibetan Plateau by HPLC-DAD-APCI/MS. <em>J. Food Compos. Anal.</em><strong>38</strong>, 49–54 (2015).</cite> [<a href="https://scholar.google.com/scholar_lookup?Lv,%20H.,%20Wang,%20X.,%20He,%20Y.,%20Wang,%20H.%20&amp;%20Suo,%20Y.%20Identification%20and%20quantification%20of%20flavonoid%20aglycones%20in%20rape%20bee%20pollen%20from%20Qinghai-Tibetan%20Plateau%20by%20HPLC-DAD-APCI/MS.%20J.%20Food%20Compos.%20Anal.38,%2049%E2%80%9354%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR37">
<span class="label">37.</span><cite>Gurdon, C. et al. Genetic and phytochemical characterization of lettuce flavonoid biosynthesis mutants. <em>Sci. Rep.</em><strong>9</strong>, 3305 (2019).
</cite> [<a href="https://doi.org/10.1038/s41598-019-39287-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6397293/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30824720/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gurdon,%20C.%20et%20al.%20Genetic%20and%20phytochemical%20characterization%20of%20lettuce%20flavonoid%20biosynthesis%20mutants.%20Sci.%20Rep.9,%203305%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR38">
<span class="label">38.</span><cite>Yin, R., Arongaus, A. B., Binkert, M. &amp; Ulm, R. Two distinct domains of the UVR8 photoreceptor interact with COP1 to initiate UV-B signaling in arabidopsis. <em>Plant Cell</em><strong>27</strong>, 202–213 (2015).
</cite> [<a href="https://doi.org/10.1105/tpc.114.133868" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4330580/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25627067/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yin,%20R.,%20Arongaus,%20A.%20B.,%20Binkert,%20M.%20&amp;%20Ulm,%20R.%20Two%20distinct%20domains%20of%20the%20UVR8%20photoreceptor%20interact%20with%20COP1%20to%20initiate%20UV-B%20signaling%20in%20arabidopsis.%20Plant%20Cell27,%20202%E2%80%93213%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR39">
<span class="label">39.</span><cite>Camacho, I. S. et al. Native mass spectrometry reveals the conformational diversity of the UVR8 photoreceptor. <em>Proc. Natl. Acad. Sci. USA</em><strong>116</strong>, 1116–1125 (2019).
</cite> [<a href="https://doi.org/10.1073/pnas.1813254116" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6347689/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30610174/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Camacho,%20I.%20S.%20et%20al.%20Native%20mass%20spectrometry%20reveals%20the%20conformational%20diversity%20of%20the%20UVR8%20photoreceptor.%20Proc.%20Natl.%20Acad.%20Sci.%20USA116,%201116%E2%80%931125%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR40">
<span class="label">40.</span><cite>Li, J., Ou-Lee, T. M., Raba, R., Amundson, R. G. &amp; Last, R. L. Arabidopsis Flavonoid Mutants Are Hypersensitive to UV-B Irradiation. <em>Plant Cell</em><strong>5</strong>, 171–179 (1993).
</cite> [<a href="https://doi.org/10.1105/tpc.5.2.171" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC160260/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12271060/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li,%20J.,%20Ou-Lee,%20T.%20M.,%20Raba,%20R.,%20Amundson,%20R.%20G.%20&amp;%20Last,%20R.%20L.%20Arabidopsis%20Flavonoid%20Mutants%20Are%20Hypersensitive%20to%20UV-B%20Irradiation.%20Plant%20Cell5,%20171%E2%80%93179%20(1993)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR41">
<span class="label">41.</span><cite>Lois, R. Accumulation of UV-absorbing flavonoids induced by UV-B radiation in Arabidopsis thaliana L. <em>Planta</em><strong>194</strong>, 498–503 (1994).</cite> [<a href="https://scholar.google.com/scholar_lookup?Lois,%20R.%20Accumulation%20of%20UV-absorbing%20flavonoids%20induced%20by%20UV-B%20radiation%20in%20Arabidopsis%20thaliana%20L.%20Planta194,%20498%E2%80%93503%20(1994)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR42">
<span class="label">42.</span><cite>Borevitz, J. O., Xia, Y., Blount, J., Dixon, R. A. &amp; Lamb, C. Activation tagging identifies a conserved MYB regulator of phenylpropanoid biosynthesis. <em>Plant Cell</em><strong>12</strong>, 2383–2393 (2000).
</cite> [<a href="https://doi.org/10.1105/tpc.12.12.2383" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC102225/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11148285/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Borevitz,%20J.%20O.,%20Xia,%20Y.,%20Blount,%20J.,%20Dixon,%20R.%20A.%20&amp;%20Lamb,%20C.%20Activation%20tagging%20identifies%20a%20conserved%20MYB%20regulator%20of%20phenylpropanoid%20biosynthesis.%20Plant%20Cell12,%202383%E2%80%932393%20(2000)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR43">
<span class="label">43.</span><cite>Heilmann, M. et al. Dimer/monomer status and in vivo function of salt-bridge mutants of the plant UV-B photoreceptor UVR8. <em>Plant J.</em><strong>88</strong>, 71–81 (2016).
</cite> [<a href="https://doi.org/10.1111/tpj.13260" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5091643/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27385642/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Heilmann,%20M.%20et%20al.%20Dimer/monomer%20status%20and%20in%20vivo%20function%20of%20salt-bridge%20mutants%20of%20the%20plant%20UV-B%20photoreceptor%20UVR8.%20Plant%20J.88,%2071%E2%80%9381%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR44">
<span class="label">44.</span><cite>Heijde, M. et al. Constitutively active UVR8 photoreceptor variant in Arabidopsis. <em>Proc. Natl. Acad. Sci. USA</em><strong>110</strong>, 20326–20331 (2013).
</cite> [<a href="https://doi.org/10.1073/pnas.1314336110" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3864333/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24277841/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Heijde,%20M.%20et%20al.%20Constitutively%20active%20UVR8%20photoreceptor%20variant%20in%20Arabidopsis.%20Proc.%20Natl.%20Acad.%20Sci.%20USA110,%2020326%E2%80%9320331%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR45">
<span class="label">45.</span><cite>Chen, X. et al. Protenix - advancing structure prediction through a comprehensive AlphaFold3 reproduction. Preprint at 10.1101/2025.01.08.631967 (2025).</cite>
</li>
<li id="CR46">
<span class="label">46.</span><cite>Wang, Y. et al. Structural insight into UV-B–activated UVR8 bound to COP1. <em>Sci. Adv</em>. <strong>8</strong>, 10.1126/sciadv.abn3337 (2022).</cite> [<a href="https://doi.org/10.1126/sciadv.abn3337" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9020657/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35442727/" class="usa-link">PubMed</a>]</li>
<li id="CR47">
<span class="label">47.</span><cite>Schrödinger, L. A. D., W. PyMOL, <a href="http://www.pymol.org/pymol" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://www.pymol.org/pymol</a> (2000).</cite>
</li>
<li id="CR48">
<span class="label">48.</span><cite>Lashkov, A. A., Tolmachev, I. V., Eistrikh-Heller, P. A. &amp; Rubinsky, S. V. PyFepRestr: Plugin to PyMOL molecular graphics system for calculating the free energy of ligand‒receptor binding. <em>Crystallogr. Rep.</em><strong>66</strong>, 861–865 (2021).</cite> [<a href="https://scholar.google.com/scholar_lookup?Lashkov,%20A.%20A.,%20Tolmachev,%20I.%20V.,%20Eistrikh-Heller,%20P.%20A.%20&amp;%20Rubinsky,%20S.%20V.%20PyFepRestr:%20Plugin%20to%20PyMOL%20molecular%20graphics%20system%20for%20calculating%20the%20free%20energy%20of%20ligand%E2%80%92receptor%20binding.%20Crystallogr.%20Rep.66,%20861%E2%80%93865%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR49">
<span class="label">49.</span><cite>Brown, B. A. et al. A UV-B-specific signaling component orchestrates plant UV protection. <em>Proc. Natl. Acad. Sci. USA</em><strong>102</strong>, 18225–18230 (2005).
</cite> [<a href="https://doi.org/10.1073/pnas.0507187102" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1312397/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16330762/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Brown,%20B.%20A.%20et%20al.%20A%20UV-B-specific%20signaling%20component%20orchestrates%20plant%20UV%20protection.%20Proc.%20Natl.%20Acad.%20Sci.%20USA102,%2018225%E2%80%9318230%20(2005)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR50">
<span class="label">50.</span><cite>de Albuquerque, I. The Arabidopsis tt5 mutant accumulates naringenin chalcone and exhibits enhanced resistance to fugal pathogens when grown under long day conditions. <a href="https://hdl.handle.net/10388/13637" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://hdl.handle.net/10388/13637</a> (2021).</cite>
</li>
<li id="CR51">
<span class="label">51.</span><cite>Roden, L. C. &amp; Ingle, R. A. Lights, rhythms, infection: the role of light and the circadian clock in determining the outcome of plant–pathogen interactions. <em>Plant Cell</em><strong>21</strong>, 2546–2552 (2009).
</cite> [<a href="https://doi.org/10.1105/tpc.109.069922" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2768925/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19789275/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Roden,%20L.%20C.%20&amp;%20Ingle,%20R.%20A.%20Lights,%20rhythms,%20infection:%20the%20role%20of%20light%20and%20the%20circadian%20clock%20in%20determining%20the%20outcome%20of%20plant%E2%80%93pathogen%20interactions.%20Plant%20Cell21,%202546%E2%80%932552%20(2009)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR52">
<span class="label">52.</span><cite>Meyer, P., Van de Poel, B. &amp; De Coninck, B. UV-B light and its application potential to reduce disease and pest incidence in crops. <em>Hortic. Res</em>. <strong>8</strong>, 194 (2021).</cite> [<a href="https://doi.org/10.1038/s41438-021-00629-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8408258/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34465753/" class="usa-link">PubMed</a>]</li>
<li id="CR53">
<span class="label">53.</span><cite>Shah, J., Kachroo, P., Nandi, A. &amp; Klessig, D. F. A recessive mutation in the Arabidopsis SSI2 gene confers SA-and NPR1-independent expression of PR genes and resistance against bacterial and oomycete pathogens. <em>Plant J.</em><strong>25</strong>, 563–574 (2001).
</cite> [<a href="https://doi.org/10.1046/j.1365-313x.2001.00992.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11309146/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Shah,%20J.,%20Kachroo,%20P.,%20Nandi,%20A.%20&amp;%20Klessig,%20D.%20F.%20A%20recessive%20mutation%20in%20the%20Arabidopsis%20SSI2%20gene%20confers%20SA-and%20NPR1-independent%20expression%20of%20PR%20genes%20and%20resistance%20against%20bacterial%20and%20oomycete%20pathogens.%20Plant%20J.25,%20563%E2%80%93574%20(2001)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR54">
<span class="label">54.</span><cite>Bowling, S. A. et al. A mutation in Arabidopsis that leads to constitutive expression of systemic acquired resistance. <em>Plant Cell</em><strong>6</strong>, 1845–1857 (1994).
</cite> [<a href="https://doi.org/10.1105/tpc.6.12.1845" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC160566/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/7866028/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bowling,%20S.%20A.%20et%20al.%20A%20mutation%20in%20Arabidopsis%20that%20leads%20to%20constitutive%20expression%20of%20systemic%20acquired%20resistance.%20Plant%20Cell6,%201845%E2%80%931857%20(1994)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR55">
<span class="label">55.</span><cite>Mao, X. et al. Mutation of Mediator subunit CDK 8 counteracts the stunted growth and salicylic acid hyperaccumulation phenotypes of an Arabidopsis MED 5 mutant. <em>N. Phytol.</em><strong>223</strong>, 233–245 (2019).</cite> [<a href="https://doi.org/10.1111/nph.15741" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30756399/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mao,%20X.%20et%20al.%20Mutation%20of%20Mediator%20subunit%20CDK%208%20counteracts%20the%20stunted%20growth%20and%20salicylic%20acid%20hyperaccumulation%20phenotypes%20of%20an%20Arabidopsis%20MED%205%20mutant.%20N.%20Phytol.223,%20233%E2%80%93245%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR56">
<span class="label">56.</span><cite>Yang, Y. et al. UVR8 interacts with WRKY36 to regulate HY5 transcription and hypocotyl elongation in Arabidopsis. <em>Nat. Plants</em><strong>4</strong>, 98–107 (2018).
</cite> [<a href="https://doi.org/10.1038/s41477-017-0099-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29379156/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yang,%20Y.%20et%20al.%20UVR8%20interacts%20with%20WRKY36%20to%20regulate%20HY5%20transcription%20and%20hypocotyl%20elongation%20in%20Arabidopsis.%20Nat.%20Plants4,%2098%E2%80%93107%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR57">
<span class="label">57.</span><cite>Fang, F. et al. Mechanisms of UV-B light-induced photoreceptor UVR8 nuclear localization dynamics. <em>N. Phytol.</em><strong>236</strong>, 1824–1837 (2022).</cite> [<a href="https://doi.org/10.1111/nph.18468" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9825989/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36089828/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Fang,%20F.%20et%20al.%20Mechanisms%20of%20UV-B%20light-induced%20photoreceptor%20UVR8%20nuclear%20localization%20dynamics.%20N.%20Phytol.236,%201824%E2%80%931837%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR58">
<span class="label">58.</span><cite>McNellis, T. W. et al. Genetic and molecular analysis of an allelic series of cop1 mutants suggests functional roles for the multiple protein domains. <em>Plant Cell</em><strong>6</strong>, 487–500 (1994).
</cite> [<a href="https://doi.org/10.1105/tpc.6.4.487" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC160452/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/8205001/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?McNellis,%20T.%20W.%20et%20al.%20Genetic%20and%20molecular%20analysis%20of%20an%20allelic%20series%20of%20cop1%20mutants%20suggests%20functional%20roles%20for%20the%20multiple%20protein%20domains.%20Plant%20Cell6,%20487%E2%80%93500%20(1994)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR59">
<span class="label">59.</span><cite>Deng, X. W. &amp; Quail, P. H. Genetic and phenotypic characterization of cop1 mutants of Arabidopsis thaliana. <em>Plant J.</em><strong>2</strong>, 83–95 (1992).</cite> [<a href="https://scholar.google.com/scholar_lookup?Deng,%20X.%20W.%20&amp;%20Quail,%20P.%20H.%20Genetic%20and%20phenotypic%20characterization%20of%20cop1%20mutants%20of%20Arabidopsis%20thaliana.%20Plant%20J.2,%2083%E2%80%9395%20(1992)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR60">
<span class="label">60.</span><cite>Liu, W., Jenkins, G. I. &amp; Datta, S. Recent advances in UV-B signalling: interaction of proteins with the UVR8 photoreceptor. <em>J. Exp. Bot.</em><strong>76</strong>, 873–881 (2025).
</cite> [<a href="https://doi.org/10.1093/jxb/erae132" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11805588/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38525857/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Liu,%20W.,%20Jenkins,%20G.%20I.%20&amp;%20Datta,%20S.%20Recent%20advances%20in%20UV-B%20signalling:%20interaction%20of%20proteins%20with%20the%20UVR8%20photoreceptor.%20J.%20Exp.%20Bot.76,%20873%E2%80%93881%20(2025)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR61">
<span class="label">61.</span><cite>Brown, B. A. &amp; Jenkins, G. I. UV-B Signaling pathways with different fluence-rate response profiles are distinguished in mature arabidopsis leaf tissue by requirement for UVR8, HY5, and HYH. <em>Plant Physiol.</em><strong>146</strong>, 323–324 (2008).</cite> [<a href="https://doi.org/10.1104/pp.107.108456" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2245850/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18055587/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Brown,%20B.%20A.%20&amp;%20Jenkins,%20G.%20I.%20UV-B%20Signaling%20pathways%20with%20different%20fluence-rate%20response%20profiles%20are%20distinguished%20in%20mature%20arabidopsis%20leaf%20tissue%20by%20requirement%20for%20UVR8,%20HY5,%20and%20HYH.%20Plant%20Physiol.146,%20323%E2%80%93324%20(2008)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR62">
<span class="label">62.</span><cite>Christie, J. M. et al. Plant UVR8 photoreceptor senses UV-B by tryptophan-mediated disruption of cross-dimer salt bridges. <em>Science</em><strong>335</strong>, 1492–1496 (2012).
</cite> [<a href="https://doi.org/10.1126/science.1218091" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3505452/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22323738/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Christie,%20J.%20M.%20et%20al.%20Plant%20UVR8%20photoreceptor%20senses%20UV-B%20by%20tryptophan-mediated%20disruption%20of%20cross-dimer%20salt%20bridges.%20Science335,%201492%E2%80%931496%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR63">
<span class="label">63.</span><cite>Di, W. et al. Structural basis of ultraviolet-B perception by UVR8. <em>Nature</em><strong>484</strong>, 214–219 (2012).
</cite> [<a href="https://doi.org/10.1038/nature10931" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22388820/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Di,%20W.%20et%20al.%20Structural%20basis%20of%20ultraviolet-B%20perception%20by%20UVR8.%20Nature484,%20214%E2%80%93219%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR64">
<span class="label">64.</span><cite>Jiang, N. et al. Diversity of genetic lesions characterizes new Arabidopsis flavonoid pigment mutant alleles from T-DNA collections. <em>Plant Sci</em>. <strong>291</strong>, 10.1016/j.plantsci.2019.110335 (2020).</cite> [<a href="https://doi.org/10.1016/j.plantsci.2019.110335" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31928687/" class="usa-link">PubMed</a>]</li>
<li id="CR65">
<span class="label">65.</span><cite>Zhang, X., Henriques, R., Lin, S. S., Niu, Q. W. &amp; Chua, N. H. Agrobacterium-mediated transformation of Arabidopsis thaliana using the floral dip method. <em>Nat. Protoc.</em><strong>1</strong>, 641–646 (2006).
</cite> [<a href="https://doi.org/10.1038/nprot.2006.97" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17406292/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhang,%20X.,%20Henriques,%20R.,%20Lin,%20S.%20S.,%20Niu,%20Q.%20W.%20&amp;%20Chua,%20N.%20H.%20Agrobacterium-mediated%20transformation%20of%20Arabidopsis%20thaliana%20using%20the%20floral%20dip%20method.%20Nat.%20Protoc.1,%20641%E2%80%93646%20(2006)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR66">
<span class="label">66.</span><cite>Andrews, S. FastQC: a quality control tool for high throughput sequence data. <a href="http://www.bioinformatics.babraham.ac.uk/projects/fastqc" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://www.bioinformatics.babraham.ac.uk/projects/fastqc</a> (2010).</cite>
</li>
<li id="CR67">
<span class="label">67.</span><cite>Langmead, B. &amp; Salzberg, S. L. Fast gapped-read alignment with Bowtie 2. <em>Nat. Methods</em><strong>9</strong>, 357–359 (2012).
</cite> [<a href="https://doi.org/10.1038/nmeth.1923" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3322381/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22388286/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Langmead,%20B.%20&amp;%20Salzberg,%20S.%20L.%20Fast%20gapped-read%20alignment%20with%20Bowtie%202.%20Nat.%20Methods9,%20357%E2%80%93359%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR68">
<span class="label">68.</span><cite>Danecek, P. et al. Twelve years of SAMtools and BCFtools. <em>Gigascience</em><strong>10</strong>, giab008, 10.1093/gigascience/giab008 (2021).</cite> [<a href="https://doi.org/10.1093/gigascience/giab008" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7931819/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33590861/" class="usa-link">PubMed</a>]</li>
<li id="CR69">
<span class="label">69.</span><cite>Tello, D. et al. NGSEP3: accurate variant calling across species and sequencing protocols. <em>Bioinformatics</em><strong>35</strong>, 4716–4723 (2019).
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btz275" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6853766/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31099384/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tello,%20D.%20et%20al.%20NGSEP3:%20accurate%20variant%20calling%20across%20species%20and%20sequencing%20protocols.%20Bioinformatics35,%204716%E2%80%934723%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR70">
<span class="label">70.</span><cite>Kruger, N. J. The Bradford method for protein quantitation. <em>Methods Mol. Biol.</em><strong>32</strong>, 9–15 (1994).
</cite> [<a href="https://doi.org/10.1385/0-89603-268-X:9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/7951753/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kruger,%20N.%20J.%20The%20Bradford%20method%20for%20protein%20quantitation.%20Methods%20Mol.%20Biol.32,%209%E2%80%9315%20(1994)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR71">
<span class="label">71.</span><cite>Siren, J., Valimaki, N. &amp; Makinen, V. Indexing graphs for path queries with applications in genome research. <em>IEEE ACM Trans. Comput. Biol. Bioinform.</em><strong>11</strong>, 375–388 (2014).
</cite> [<a href="https://doi.org/10.1109/TCBB.2013.2297101" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26355784/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Siren,%20J.,%20Valimaki,%20N.%20&amp;%20Makinen,%20V.%20Indexing%20graphs%20for%20path%20queries%20with%20applications%20in%20genome%20research.%20IEEE%20ACM%20Trans.%20Comput.%20Biol.%20Bioinform.11,%20375%E2%80%93388%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR72">
<span class="label">72.</span><cite>Liao, Y., Smyth, G. K. &amp; Shi, W. featureCounts: an efficient general purpose program for assigning sequence reads to genomic features. <em>Bioinformatics</em><strong>30</strong>, 923–930 (2014).
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btt656" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24227677/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Liao,%20Y.,%20Smyth,%20G.%20K.%20&amp;%20Shi,%20W.%20featureCounts:%20an%20efficient%20general%20purpose%20program%20for%20assigning%20sequence%20reads%20to%20genomic%20features.%20Bioinformatics30,%20923%E2%80%93930%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR73">
<span class="label">73.</span><cite>Pertea, M. et al. StringTie enables improved reconstruction of a transcriptome from RNA-seq reads. <em>Nat. Biotechnol.</em><strong>33</strong>, 290–295 (2015).
</cite> [<a href="https://doi.org/10.1038/nbt.3122" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4643835/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25690850/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Pertea,%20M.%20et%20al.%20StringTie%20enables%20improved%20reconstruction%20of%20a%20transcriptome%20from%20RNA-seq%20reads.%20Nat.%20Biotechnol.33,%20290%E2%80%93295%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR74">
<span class="label">74.</span><cite>Love, M. I., Huber, W. &amp; Anders, S. Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2. <em>Genome Biol.</em><strong>15</strong>, 550 (2014).
</cite> [<a href="https://doi.org/10.1186/s13059-014-0550-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4302049/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25516281/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Love,%20M.%20I.,%20Huber,%20W.%20&amp;%20Anders,%20S.%20Moderated%20estimation%20of%20fold%20change%20and%20dispersion%20for%20RNA-seq%20data%20with%20DESeq2.%20Genome%20Biol.15,%20550%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR75">
<span class="label">75.</span><cite>Benjamini, Y. &amp; Hochberg, Y. Controlling the false discovery rate: A practical and powerful approach to multiple testing. <em>J. R. Stat. Soc. Ser. B Methodol.</em><strong>57</strong>, 289–300 (1995).</cite> [<a href="https://scholar.google.com/scholar_lookup?Benjamini,%20Y.%20&amp;%20Hochberg,%20Y.%20Controlling%20the%20false%20discovery%20rate:%20A%20practical%20and%20powerful%20approach%20to%20multiple%20testing.%20J.%20R.%20Stat.%20Soc.%20Ser.%20B%20Methodol.57,%20289%E2%80%93300%20(1995)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR76">
<span class="label">76.</span><cite>Heberle, H., Meirelles, G. V., da Silva, F. R., Telles, G. P. &amp; Minghim, R. InteractiVenn: a web-based tool for the analysis of sets through Venn diagrams. <em>BMC Bioinform.</em><strong>16</strong>, 10.1186/s12859-015-0611-3 (2015).</cite> [<a href="https://doi.org/10.1186/s12859-015-0611-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4455604/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25994840/" class="usa-link">PubMed</a>]</li>
<li id="CR77">
<span class="label">77.</span><cite>Ge, S. X., Jung, D., Yao, R. &amp; Valencia, A. ShinyGO: a graphical gene-set enrichment tool for animals and plants. <em>Bioinformatics</em><strong>36</strong>, 2628–2629 (2020).
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btz931" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7178415/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31882993/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ge,%20S.%20X.,%20Jung,%20D.,%20Yao,%20R.%20&amp;%20Valencia,%20A.%20ShinyGO:%20a%20graphical%20gene-set%20enrichment%20tool%20for%20animals%20and%20plants.%20Bioinformatics36,%202628%E2%80%932629%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12371036/bin/41467_2025_63010_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information</a><sup> (3.3MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12371036/bin/41467_2025_63010_MOESM2_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Peer Review file</a><sup> (1.2MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12371036/bin/41467_2025_63010_MOESM3_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">41467_2025_63010_MOESM3_ESM.pdf</a><sup> (6.9KB, pdf) </sup><p>Description of Additional Supplementary Files</p>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material4_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12371036/bin/41467_2025_63010_MOESM4_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 1</a><sup> (118KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material5_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12371036/bin/41467_2025_63010_MOESM5_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 2</a><sup> (89.7KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material6_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12371036/bin/41467_2025_63010_MOESM6_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 3</a><sup> (85.4KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material7_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12371036/bin/41467_2025_63010_MOESM7_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 4</a><sup> (40.9KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material8_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12371036/bin/41467_2025_63010_MOESM8_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 5</a><sup> (37.5KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material9_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12371036/bin/41467_2025_63010_MOESM9_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 6</a><sup> (23.8KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material10_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12371036/bin/41467_2025_63010_MOESM10_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 7</a><sup> (32.4KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material11_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12371036/bin/41467_2025_63010_MOESM11_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Reporting Summary</a><sup> (107.2KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material12_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12371036/bin/41467_2025_63010_MOESM12_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Source Data</a><sup> (289.9MB, xlsx) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>Data supporting the findings of this work are available within the paper and the respective Supplementary Information files. A reporting summary for this article is available as a Supplementary Information file. The raw RNA-seq reads and whole genome re-sequencing data generated in this study have been deposited in the NCBI BioProject database under accession codes <a href="https://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA1065785" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PRJNA1065785</a>, <a href="https://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA1231375" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PRJNA1231375</a>, and <a href="https://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA1065513" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PRJNA1065513</a>. The metabolomics data have been deposited to MetaboLights repository with the study identifier <a href="https://www.ebi.ac.uk/metabolights/MTBLS12792" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">MTBLS12792</a>. <a href="#Sec26" class="usa-link">Source data</a> are provided in this paper.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Nature Communications are provided here courtesy of <strong>Nature Publishing Group</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1038/s41467-025-63010-3"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/41467_2025_Article_63010.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (3.8 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12371036/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12371036/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12371036%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371036/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12371036/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12371036/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40841532/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12371036/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40841532/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12371036/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12371036/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="Rg7mPePftvKDGUyBw4CUaxZtjUNE0Rxlt5ReXsdrfwdkGxxeLUQFs8aaqupmFWQo">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
