
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Microbiota metabolite taurodeoxycholic acid maintains intestinal tissue residency of innate lymphoid cells via engagement with P2Y10 receptor - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="15253A818AF20C13053A81001CAEE263.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="sciadv">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12372891/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Science Advances">
<meta name="citation_title" content="Microbiota metabolite taurodeoxycholic acid maintains intestinal tissue residency of innate lymphoid cells via engagement with P2Y10 receptor">
<meta name="citation_author" content="Yuwei Xu">
<meta name="citation_author_institution" content="State Key Laboratory of Epigenetic Regulation and Intervention, Institute of Biophysics, Chinese Academy of Sciences, Beijing, China.">
<meta name="citation_author" content="Zhen Xiong">
<meta name="citation_author_institution" content="State Key Laboratory of Epigenetic Regulation and Intervention, Institute of Biophysics, Chinese Academy of Sciences, Beijing, China.">
<meta name="citation_author" content="Peikang Zhang">
<meta name="citation_author_institution" content="State Key Laboratory of Epigenetic Regulation and Intervention, Institute of Biophysics, Chinese Academy of Sciences, Beijing, China.">
<meta name="citation_author_institution" content="University of Chinese Academy of Sciences, Beijing, China.">
<meta name="citation_author" content="Runyuan Wu">
<meta name="citation_author_institution" content="State Key Laboratory of Epigenetic Regulation and Intervention, Institute of Biophysics, Chinese Academy of Sciences, Beijing, China.">
<meta name="citation_author_institution" content="University of Chinese Academy of Sciences, Beijing, China.">
<meta name="citation_author" content="Cunzhen Li">
<meta name="citation_author_institution" content="State Key Laboratory of Epigenetic Regulation and Intervention, Institute of Biophysics, Chinese Academy of Sciences, Beijing, China.">
<meta name="citation_author_institution" content="University of Chinese Academy of Sciences, Beijing, China.">
<meta name="citation_author" content="Hui Guo">
<meta name="citation_author_institution" content="State Key Laboratory of Epigenetic Regulation and Intervention, Institute of Biophysics, Chinese Academy of Sciences, Beijing, China.">
<meta name="citation_author" content="Ying Du">
<meta name="citation_author_institution" content="State Key Laboratory of Epigenetic Regulation and Intervention, Institute of Biophysics, Chinese Academy of Sciences, Beijing, China.">
<meta name="citation_author" content="Xiaoxiao Zhu">
<meta name="citation_author_institution" content="State Key Laboratory of Epigenetic Regulation and Intervention, Institute of Biophysics, Chinese Academy of Sciences, Beijing, China.">
<meta name="citation_author" content="Dongdong Fan">
<meta name="citation_author_institution" content="State Key Laboratory of Epigenetic Regulation and Intervention, Institute of Biophysics, Chinese Academy of Sciences, Beijing, China.">
<meta name="citation_author" content="Hongzhe Fan">
<meta name="citation_author_institution" content="State Key Laboratory of Epigenetic Regulation and Intervention, Institute of Biophysics, Chinese Academy of Sciences, Beijing, China.">
<meta name="citation_author" content="Yong Tian">
<meta name="citation_author_institution" content="State Key Laboratory of Epigenetic Regulation and Intervention, Institute of Biophysics, Chinese Academy of Sciences, Beijing, China.">
<meta name="citation_author_institution" content="University of Chinese Academy of Sciences, Beijing, China.">
<meta name="citation_author" content="Yun Chen">
<meta name="citation_author_institution" content="The Affiliated Wuxi People’s Hospital of Nanjing Medical University, Wuxi People’s Hospital, Wuxi Medical Center, Wuxi; Department of Immunology, Jiangsu Key Lab of Cancer Biomarkers, Prevention and Treatment, Collaborative Innovation Center for Cancer Personalized Medicine, Nanjing Medical University, Nanjing, China.">
<meta name="citation_author" content="Zusen Fan">
<meta name="citation_author_institution" content="State Key Laboratory of Epigenetic Regulation and Intervention, Institute of Biophysics, Chinese Academy of Sciences, Beijing, China.">
<meta name="citation_author_institution" content="University of Chinese Academy of Sciences, Beijing, China.">
<meta name="citation_author_institution" content="Faculty of Pharmaceutical Sciences, Shenzhen University of Advanced Technology, Shenzhen, China.">
<meta name="citation_publication_date" content="2025 Aug 22">
<meta name="citation_volume" content="11">
<meta name="citation_issue" content="34">
<meta name="citation_firstpage" content="eadt9645">
<meta name="citation_doi" content="10.1126/sciadv.adt9645">
<meta name="citation_pmid" content="40845099">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12372891/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12372891/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12372891/pdf/sciadv.adt9645.pdf">
<meta name="description" content="Innate lymphoid cells (ILCs) play critical roles in innate immunity, epithelial barrier protection, and tissue homeostasis. However, the maintenance machinery of intestinal tissue residency of ILCs remains elusive. Here, we show that gut microbiota ...">
<meta name="og:title" content="Microbiota metabolite taurodeoxycholic acid maintains intestinal tissue residency of innate lymphoid cells via engagement with P2Y10 receptor">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Innate lymphoid cells (ILCs) play critical roles in innate immunity, epithelial barrier protection, and tissue homeostasis. However, the maintenance machinery of intestinal tissue residency of ILCs remains elusive. Here, we show that gut microbiota ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12372891/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12372891">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1126/sciadv.adt9645"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/sciadv.adt9645.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12372891%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12372891/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12372891/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12372891/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-sciadv.gif" alt="Science Advances logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Science Advances" title="Link to Science Advances" shape="default" href="http://advances.sciencemag.org/" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Sci Adv</button></div>. 2025 Aug 22;11(34):eadt9645. doi: <a href="https://doi.org/10.1126/sciadv.adt9645" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1126/sciadv.adt9645</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Sci%20Adv%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sci%20Adv%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Sci%20Adv%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Sci%20Adv%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Microbiota metabolite taurodeoxycholic acid maintains intestinal tissue residency of innate lymphoid cells via engagement with P2Y10 receptor</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Xu%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Yuwei Xu</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Yuwei Xu</span></h3>
<div class="p">
<sup><sup>1</sup></sup>State Key Laboratory of Epigenetic Regulation and Intervention, Institute of Biophysics, Chinese Academy of Sciences, Beijing, China.</div>
<div>Conceptualization, Data curation, Formal analysis, Investigation, Methodology, Software, Validation, Visualization, Writing - original draft, Writing - review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Xu%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yuwei Xu</span></a>
</div>
</div>
<sup>1,</sup><sup>†</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Xiong%20Z%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Zhen Xiong</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Zhen Xiong</span></h3>
<div class="p">
<sup><sup>1</sup></sup>State Key Laboratory of Epigenetic Regulation and Intervention, Institute of Biophysics, Chinese Academy of Sciences, Beijing, China.</div>
<div>Conceptualization, Funding acquisition, Methodology, Writing - original draft, Writing - review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Xiong%20Z%22%5BAuthor%5D" class="usa-link"><span class="name western">Zhen Xiong</span></a>
</div>
</div>
<sup>1,</sup><sup>†</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20P%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Peikang Zhang</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Peikang Zhang</span></h3>
<div class="p">
<sup><sup>1</sup></sup>State Key Laboratory of Epigenetic Regulation and Intervention, Institute of Biophysics, Chinese Academy of Sciences, Beijing, China.</div>
<div class="p">
<sup><sup>2</sup></sup>University of Chinese Academy of Sciences, Beijing, China.</div>
<div>Formal analysis, Investigation, Validation</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20P%22%5BAuthor%5D" class="usa-link"><span class="name western">Peikang Zhang</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>†</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wu%20R%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Runyuan Wu</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Runyuan Wu</span></h3>
<div class="p">
<sup><sup>1</sup></sup>State Key Laboratory of Epigenetic Regulation and Intervention, Institute of Biophysics, Chinese Academy of Sciences, Beijing, China.</div>
<div class="p">
<sup><sup>2</sup></sup>University of Chinese Academy of Sciences, Beijing, China.</div>
<div>Formal analysis</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wu%20R%22%5BAuthor%5D" class="usa-link"><span class="name western">Runyuan Wu</span></a>
</div>
</div>
<sup>1,</sup><sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Li%20C%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Cunzhen Li</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Cunzhen Li</span></h3>
<div class="p">
<sup><sup>1</sup></sup>State Key Laboratory of Epigenetic Regulation and Intervention, Institute of Biophysics, Chinese Academy of Sciences, Beijing, China.</div>
<div class="p">
<sup><sup>2</sup></sup>University of Chinese Academy of Sciences, Beijing, China.</div>
<div>Formal analysis, Investigation</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Li%20C%22%5BAuthor%5D" class="usa-link"><span class="name western">Cunzhen Li</span></a>
</div>
</div>
<sup>1,</sup><sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Guo%20H%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Hui Guo</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Hui Guo</span></h3>
<div class="p">
<sup><sup>1</sup></sup>State Key Laboratory of Epigenetic Regulation and Intervention, Institute of Biophysics, Chinese Academy of Sciences, Beijing, China.</div>
<div>Resources, Software</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Guo%20H%22%5BAuthor%5D" class="usa-link"><span class="name western">Hui Guo</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Du%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Ying Du</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Ying Du</span></h3>
<div class="p">
<sup><sup>1</sup></sup>State Key Laboratory of Epigenetic Regulation and Intervention, Institute of Biophysics, Chinese Academy of Sciences, Beijing, China.</div>
<div>Methodology, Software</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Du%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Ying Du</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhu%20X%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Xiaoxiao Zhu</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Xiaoxiao Zhu</span></h3>
<div class="p">
<sup><sup>1</sup></sup>State Key Laboratory of Epigenetic Regulation and Intervention, Institute of Biophysics, Chinese Academy of Sciences, Beijing, China.</div>
<div>Resources, Validation</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhu%20X%22%5BAuthor%5D" class="usa-link"><span class="name western">Xiaoxiao Zhu</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Fan%20D%22%5BAuthor%5D" class="usa-link" aria-describedby="id9"><span class="name western">Dongdong Fan</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">Dongdong Fan</span></h3>
<div class="p">
<sup><sup>1</sup></sup>State Key Laboratory of Epigenetic Regulation and Intervention, Institute of Biophysics, Chinese Academy of Sciences, Beijing, China.</div>
<div>Resources</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Fan%20D%22%5BAuthor%5D" class="usa-link"><span class="name western">Dongdong Fan</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Fan%20H%22%5BAuthor%5D" class="usa-link" aria-describedby="id10"><span class="name western">Hongzhe Fan</span></a><div hidden="hidden" id="id10">
<h3><span class="name western">Hongzhe Fan</span></h3>
<div class="p">
<sup><sup>1</sup></sup>State Key Laboratory of Epigenetic Regulation and Intervention, Institute of Biophysics, Chinese Academy of Sciences, Beijing, China.</div>
<div>Formal analysis, Investigation, Validation</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Fan%20H%22%5BAuthor%5D" class="usa-link"><span class="name western">Hongzhe Fan</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tian%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id11"><span class="name western">Yong Tian</span></a><div hidden="hidden" id="id11">
<h3><span class="name western">Yong Tian</span></h3>
<div class="p">
<sup><sup>1</sup></sup>State Key Laboratory of Epigenetic Regulation and Intervention, Institute of Biophysics, Chinese Academy of Sciences, Beijing, China.</div>
<div class="p">
<sup><sup>2</sup></sup>University of Chinese Academy of Sciences, Beijing, China.</div>
<div>Formal analysis, Investigation</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tian%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yong Tian</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>*</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chen%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id12"><span class="name western">Yun Chen</span></a><div hidden="hidden" id="id12">
<h3><span class="name western">Yun Chen</span></h3>
<div class="p">
<sup><sup>3</sup></sup>The Affiliated Wuxi People’s Hospital of Nanjing Medical University, Wuxi People’s Hospital, Wuxi Medical Center, Wuxi; Department of Immunology, Jiangsu Key Lab of Cancer Biomarkers, Prevention and Treatment, Collaborative Innovation Center for Cancer Personalized Medicine, Nanjing Medical University, Nanjing, China.</div>
<div>Formal analysis, Methodology, Supervision, Validation, Writing - review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chen%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yun Chen</span></a>
</div>
</div>
<sup>3,</sup><sup>*</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Fan%20Z%22%5BAuthor%5D" class="usa-link" aria-describedby="id13"><span class="name western">Zusen Fan</span></a><div hidden="hidden" id="id13">
<h3><span class="name western">Zusen Fan</span></h3>
<div class="p">
<sup><sup>1</sup></sup>State Key Laboratory of Epigenetic Regulation and Intervention, Institute of Biophysics, Chinese Academy of Sciences, Beijing, China.</div>
<div class="p">
<sup><sup>2</sup></sup>University of Chinese Academy of Sciences, Beijing, China.</div>
<div class="p">
<sup><sup>4</sup></sup>Faculty of Pharmaceutical Sciences, Shenzhen University of Advanced Technology, Shenzhen, China.</div>
<div>Conceptualization, Data curation, Funding acquisition, Methodology, Project administration, Resources, Supervision, Validation, Visualization, Writing - original draft, Writing - review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Fan%20Z%22%5BAuthor%5D" class="usa-link"><span class="name western">Zusen Fan</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>4,</sup><sup>*</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="aff1">
<sup><sup>1</sup></sup>State Key Laboratory of Epigenetic Regulation and Intervention, Institute of Biophysics, Chinese Academy of Sciences, Beijing, China.</div>
<div id="aff2">
<sup><sup>2</sup></sup>University of Chinese Academy of Sciences, Beijing, China.</div>
<div id="aff3">
<sup><sup>3</sup></sup>The Affiliated Wuxi People’s Hospital of Nanjing Medical University, Wuxi People’s Hospital, Wuxi Medical Center, Wuxi; Department of Immunology, Jiangsu Key Lab of Cancer Biomarkers, Prevention and Treatment, Collaborative Innovation Center for Cancer Personalized Medicine, Nanjing Medical University, Nanjing, China.</div>
<div id="aff4">
<sup><sup>4</sup></sup>Faculty of Pharmaceutical Sciences, Shenzhen University of Advanced Technology, Shenzhen, China.</div>
<div class="author-notes p">
<div class="fn" id="cor1">
<sup>*</sup><p class="display-inline">Corresponding author. Email: <span>fanz@moon.ibp.ac.cn</span> (Z.F.); <span>chenyun@njmu.edu.cn</span> (Y.C.); <span>ytian@ibp.ac.cn</span> (Y.T.)</p>
</div>
<div class="fn" id="afn1">
<sup>†</sup><p class="display-inline">These authors contributed equally to this work.</p>
</div>
</div>
<h4 class="font-secondary">Roles</h4>
<div class="p">
<strong class="contrib"><span class="name western">Yuwei Xu</span></strong>: <span class="role">Conceptualization, Data curation, Formal analysis, Investigation, Methodology, Software, Validation, Visualization, Writing - original draft, Writing - review &amp; editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">Zhen Xiong</span></strong>: <span class="role">Conceptualization, Funding acquisition, Methodology, Writing - original draft, Writing - review &amp; editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">Peikang Zhang</span></strong>: <span class="role">Formal analysis, Investigation, Validation</span>
</div>
<div>
<strong class="contrib"><span class="name western">Runyuan Wu</span></strong>: <span class="role">Formal analysis</span>
</div>
<div>
<strong class="contrib"><span class="name western">Cunzhen Li</span></strong>: <span class="role">Formal analysis, Investigation</span>
</div>
<div>
<strong class="contrib"><span class="name western">Hui Guo</span></strong>: <span class="role">Resources, Software</span>
</div>
<div>
<strong class="contrib"><span class="name western">Ying Du</span></strong>: <span class="role">Methodology, Software</span>
</div>
<div>
<strong class="contrib"><span class="name western">Xiaoxiao Zhu</span></strong>: <span class="role">Resources, Validation</span>
</div>
<div>
<strong class="contrib"><span class="name western">Dongdong Fan</span></strong>: <span class="role">Resources</span>
</div>
<div>
<strong class="contrib"><span class="name western">Hongzhe Fan</span></strong>: <span class="role">Formal analysis, Investigation, Validation</span>
</div>
<div>
<strong class="contrib"><span class="name western">Yong Tian</span></strong>: <span class="role">Formal analysis, Investigation</span>
</div>
<div>
<strong class="contrib"><span class="name western">Yun Chen</span></strong>: <span class="role">Formal analysis, Methodology, Supervision, Validation, Writing - review &amp; editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">Zusen Fan</span></strong>: <span class="role">Conceptualization, Data curation, Funding acquisition, Methodology, Project administration, Resources, Supervision, Validation, Visualization, Writing - original draft, Writing - review &amp; editing</span>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2024 Oct 17; Accepted 2025 Jul 23; Collection date 2025 Aug 22.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>Copyright © 2025 The Authors, some rights reserved; exclusive licensee American Association for the Advancement of Science. No claim to original U.S. Government Works. Distributed under a Creative Commons Attribution NonCommercial License 4.0 (CC BY-NC).</div>
<p>This is an open-access article distributed under the terms of the <a href="https://creativecommons.org/licenses/by-nc/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Creative Commons Attribution-NonCommercial license</a>, which permits use, distribution, and reproduction in any medium, so long as the resultant use is <strong>not</strong> for commercial advantage and provided the original work is properly cited.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12372891  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40845099/" class="usa-link">40845099</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>Abstract</h2>
<p>Innate lymphoid cells (ILCs) play critical roles in innate immunity, epithelial barrier protection, and tissue homeostasis. However, the maintenance machinery of intestinal tissue residency of ILCs remains elusive. Here, we show that gut microbiota is necessary for the maintenance of intestinal tissue residency of ILCs. Microbiota metabolite taurodeoxycholic acid (TDCA) binds to P2Y10 receptor on ILCs to initiate downstream Ca<sup>2+</sup> and RhoA signaling pathways. TDCA-P2Y10 engagement induces <em>Zfp414</em> transcription to prime expression of CD69 and integrin αE on ILCs, leading to intestinal residency of ILCs. Moreover, decreased levels of TDCA or P2Y10 deficiency abrogates the intestinal residency of ILCs, resulting in severer intestinal inflammation. Of note, TDCA administration can enhance intestinal tissue residency of ILCs and promote protection against intestinal inflammation. Thus, TDCA might be used as a potential drug to treat patients with inflammatory bowel disease.</p></section><section class="abstract" id="abstract2"><hr class="headless">
<p>Microbiota metabolite TDCA binds P2Y10 on intestinal ILCs to maintain tissue residency through CD69 and integrin alphaE.</p></section><section id="sec1"><h2 class="pmc_sec_title">INTRODUCTION</h2>
<p>Innate lymphoid cells (ILCs) are important guardians in mucosal immunity, contributing to barrier immunity, tissue homeostasis, and immune regulation. ILCs are classified into three groups on the basis of their development and functional characteristics: group 1 (ILC1s), group 2 (ILC2s), and group 3 (ILC3s) (<a href="#R1" class="usa-link" aria-describedby="R1"><em>1</em></a>). We also recently defined a subpopulation of regulatory ILCs called ILCreg that exist in the gut and suppress activation of ILC1s and ILC3s via interleukin-10 (IL-10) secretion, leading to protection against intestinal inflammation (<a href="#R2" class="usa-link" aria-describedby="R2"><em>2</em></a>). Based on their secretion of specific cytokines, each group of ILCs specializes in unique innate immunity against different types of pathogens, such as intracellular pathogens, helminths, and extracellular pathogens, respectively (<a href="#R3" class="usa-link" aria-describedby="R3"><em>3</em></a>). Tissue residency is a hallmark of intestinal ILCs. Upon seeding intestinal tissues during fetal development, ILCs maintain tissue sedentary lifestyle and replenish their populations predominantly by local expansion (<a href="#R4" class="usa-link" aria-describedby="R4"><em>4</em></a>, <a href="#R5" class="usa-link" aria-describedby="R5"><em>5</em></a>). Although long-term parabiosis experiments revealed that, under a steady-state condition, resident ILCs keep tissue dwelling, how the permanent resident state of ILC populations in the intestine is maintained and where ILCs detaching from intestinal tissues migrate remain elusive.</p>
<p>Gut microbiota plays a critical role in shaping an immune system and maintaining intestinal homeostasis. We previously showed that bacterial metabolite <em>N</em>-undecanoylglycine induces intestinal Tuft-2 cell expansion for antimicrobial immunity (<a href="#R6" class="usa-link" aria-describedby="R6"><em>6</em></a>). Recently, accumulating evidence suggests that metabolites produced by microbiota regulate the function of ILCs (<a href="#R7" class="usa-link" aria-describedby="R7"><em>7</em></a>). For example, tryptophan metabolites derived from microbiota promote IL-22 production of ILC3s via activating aryl hydrocarbon receptor (<a href="#R8" class="usa-link" aria-describedby="R8"><em>8</em></a>). Likewise, short-chain fatty acids (SCFAs) produced by commensal microbiota from dietary fibers differentially affect intestinal ILCs in a subset-specific manner (<a href="#R9" class="usa-link" aria-describedby="R9"><em>9</em></a>). They promote proliferation and IL-22 production of ILC3s, while they inhibit expansion of ILC2s through G protein–coupled receptor 41 (GPR41) (<a href="#R10" class="usa-link" aria-describedby="R10"><em>10</em></a>). Recent studies have reported the role of another microbiota metabolite, bile acid metabolite, in regulating ILCs’ function and intestinal homeostasis. Increased levels of cholic acid activate intestinal ILC2 to produce IL-5, leading to increased infiltration of eosinophils into intestinal tissues and inducing protective type II inflammatory response against worm infection (<a href="#R11" class="usa-link" aria-describedby="R11"><em>11</em></a>). Glycodeoxycholic acid binds to TGR5 receptor on ILC3 and activates downstream cyclic adenosine 3′,5′-monophosphate signaling pathways, resulting in activation of transcription factor (TF)GATA binding protein 3 (GATA3) and promotion of IL-22 secretion (<a href="#R12" class="usa-link" aria-describedby="R12"><em>12</em></a>). However, whether microbiota and their metabolites sustain retention of ILCs in the intestine is unclear.</p>
<p>As environmental sensors, G protein–coupled receptors (GPCRs) have been recognized to maintain retention of tissue-resident lymphocytes. For example, adhesion GPCR family member-E5 (Adgre5 or CD97) ensures retention of cDC2 in the spleen through mechanosensing red blood cells (<a href="#R13" class="usa-link" aria-describedby="R13"><em>13</em></a>). P2Y receptors belong to the GPCR superfamily and are expressed in various lymphocytes, implicating their regulatory effects on immune system in many ways, such as degranulation, phagocytic clearance, cytokine production, and lymphocytes migration (<a href="#R14" class="usa-link" aria-describedby="R14"><em>14</em></a>–<a href="#R16" class="usa-link" aria-describedby="R16"><em>16</em></a>). However, how P2Y receptors regulate ILCs residency remains unknown. Tissue-resident lymphocytes display unique phenotypic characteristics and rely on chemokine receptors and adhesion molecules for their retention within tissues, including CXCR6, CCR6, integrin α1, and CD69 (<a href="#R5" class="usa-link" aria-describedby="R5"><em>5</em></a>, <a href="#R17" class="usa-link" aria-describedby="R17"><em>17</em></a>, <a href="#R18" class="usa-link" aria-describedby="R18"><em>18</em></a>). Notably, the lectin receptor CD69, a marker for tissue-resident cells, is widely expressed in various lymphocytes, including tissue-resident memory T (Trm), B (Brm), and natural killer (NK) cells (<a href="#R17" class="usa-link" aria-describedby="R17"><em>17</em></a>, <a href="#R19" class="usa-link" aria-describedby="R19"><em>19</em></a>, <a href="#R20" class="usa-link" aria-describedby="R20"><em>20</em></a>). Here, we showed that gut microbiota is necessary for the maintenance of intestinal tissue residency of ILCs. Microbiota metabolite taurodeoxycholic acid (TDCA) is a ligand to P2Y10 receptor that is highly expressed on intestinal ILCs. TDCA-P2Y10 engagement initiates downstream Ca<sup>2+</sup> and RhoA signaling pathways to promote <em>Zfp414</em> transcription, leading to elevated expression of CD69 and integrin αE (ITAE; also called CD103) on ILCs. TDCA administration can enhance intestinal tissue residency of ILCs and promote protection against intestinal inflammation.</p></section><section id="sec2"><h2 class="pmc_sec_title">RESULTS</h2>
<section id="sec3"><h3 class="pmc_sec_title">Gut microbiota is necessary for the maintenance of ILCs intestinal residency</h3>
<p>To explore the effect of gut microbiota on regulating tissue residency of ILCs, we analyzed ILCs in intestinal tissues and determined their localization in germ-free (GF) mice. We found that intestinal ILCs within intestinal epithelia and lamina propria were much lower in GF mice compared with those in conventional specific pathogen–free (Ctrl) mice (<a href="#F1" class="usa-link">Fig. 1A</a> and fig. S1A). In addition, intestinal immunofluorescence staining showed a notable reduction in numbers of three ILC subsets in GF mice (fig. S1, B to D). Consistent with GF mice, frequencies and numbers of intestinal ILC subsets were decreased in mice treated with broad-spectrum antibiotic (Abx) (<a href="#F1" class="usa-link">Fig. 1B</a> and fig. S1, A and E). In addition, antibiotic treatment decreased ILC2s and ILC3s in Peyer’s patches (PPs) but had no obvious impact on ILC1s (fig. S1F). Furthermore, we found that proliferation and apoptosis rates were comparable, suggesting that ILC reduction in intestinal tissues is not caused by changes of cell proliferation and apoptosis (fig. S1G). Given that intestinal and peripheral lymphoid tissues are closely interconnected, intestinal lymphocytes are capable of trafficking between them. We then examined whether reduced intestinal ILCs migrate to lymphoid tissues such as mesenteric lymph nodes (mLNs) and spleen. We observed that percentages and numbers of ILCs from mLN and spleen were increased in GF (<a href="#F1" class="usa-link">Fig. 1A</a>) and Abx-treated mice (<a href="#F1" class="usa-link">Fig. 1, C and D</a>; and fig. S2, A and B) compared with those in Ctrl mice. It has been reported that KLRG1<sup>+</sup> ILC2s represent activated ILC2s. To verify whether the reduction of intestinal ILC2s caused by Abx was due to their migration rather than activation, we sorted intestinal ILC2s for in vitro activation assays. We found that antibiotic treatment did not affect the activation of ILC2s (fig. S2, C and D). Meanwhile, we also used GATA3 as the marker for total ILC2s. Consistent with the findings on KLRG1<sup>+</sup>Sca-1<sup>+</sup> ILC2s, antibiotic treatment decreased numbers of intestinal GATA3<sup>+</sup> ILC2s, while increased in mLN and spleen (fig. S2E).</p>
<figure class="fig xbox font-sm" id="F1"><h4 class="obj_head">Fig. 1. Gut microbiota is necessary for the intestinal maintenance of ILCs.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12372891_sciadv.adt9645-f1.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/3fdc/12372891/7f95deea56b4/sciadv.adt9645-f1.jpg" loading="lazy" height="785" width="746" alt="Fig. 1."></a></p>
<div class="p text-right font-secondary"><a href="figure/F1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>(<strong>A</strong>) Flow cytometry analysis of total ILCs (Lin<sup>−</sup>Thy1.2<sup>+</sup>CD127<sup>+</sup>) from intraepithelial lymphocytes (IELs), lamina propria lymphocytes (LPLs), mesenteric lymph node (mLN), and spleen in GF and Ctrl mice. Numbers in flow cytometry plots represent percentage of ILCs in each gate. <em>n</em> = 4. (<strong>B</strong> to <strong>D</strong>) Flow cytometry analysis of ILC1s (Lin<sup>−</sup>CD127<sup>+</sup>CD45.2<sup>+</sup>NK1.1<sup>+</sup>NKp46<sup>+</sup>), ILC2s (Lin<sup>−</sup>CD127<sup>+</sup>Thy1.2<sup>+</sup>KLRG1<sup>+</sup>Sca-1<sup>+</sup>), and ILC3s (Lin<sup>−</sup>CD127<sup>+</sup>Thy1.2<sup>+</sup>RORγt<sup>+</sup>) in the SI (B), mLN (C), and spleen (D) from Ctrl and Abx-treated mice that were fed water containing a combination of antibiotics (Abx) for 2 weeks. <em>n</em> = 6. (<strong>E</strong> to <strong>G</strong>) Intestinal ILC1s (E), ILC2s (F), and ILC3s (G) located in lymphatics from Ctrl and Abx-treated mice were visualized by three-dimensional (3D) immunofluorescence staining. Lyve1 and EpCAM denote the lymphatic vessels and intestinal epithelia separately. White arrows indicate ILC subsets within lymphatic vessels. Migratory ILCs subsets were regarded as cells within lymphatic vessels. Scale bars, 70 μm. Frequencies of migratory ILC subsets in total ILCs located in a villus were calculated. Each symbol in the dot plots indicates one villus. <em>n</em> = 11. Data are representative of at least three independent experiments and are shown as the means ± SD. Statistical analysis was performed by using unpaired two-tailed Student’s <em>t</em> test (*<em>P</em> &lt; 0.05; **<em>P</em> &lt; 0.01; ***<em>P</em> &lt; 0.001). SSC, side scatter; W, width.</p></figcaption></figure><p>As lymphocytes egress from small intestine (SI) and enter into lymphoid tissues via lymphatic vessels (<a href="#R21" class="usa-link" aria-describedby="R21"><em>21</em></a>), we conducted three-dimensional (3D) immunofluorescence of intestinal villi to visualize ILCs localization. Lyve1 staining was used to delineate lymphatic vessels (<a href="#R22" class="usa-link" aria-describedby="R22"><em>22</em></a>). In a steady state, ILCs predominantly resided in lamina propria, and only a rare fraction entered into lymphatics. After Abx treatment, much more ILCs exited intestine and entered into lymphatic vessels. All of three ILC subsets displayed the same phenomena (<a href="#F1" class="usa-link">Fig. 1, E to G</a>). Collectively, these data indicate that microbiota environment is necessary for the maintenance of tissue residency of intestinal ILCs.</p></section><section id="sec4"><h3 class="pmc_sec_title">Microbiota disruption causes migration of resident ILCs from SI to a circulation system</h3>
<p>Previous studies have reported that S1PR1 receptor regulates migration of central memory T cells (Tcm) from peripheral nonlymphoid tissues to lymphoid tissues, and the function of S1PR1 can be antagonized by CD69 (<a href="#R23" class="usa-link" aria-describedby="R23"><em>23</em></a>). To determine whether CD69 makes similar contribution to tissue residency of intestinal ILCs in a microbiota-dependent manner, we first examined expression of CD69 and S1PR1 on ILCs in Abx-treated versus Ctrl mice. We noticed that CD69<sup>+</sup> ILCs were markedly decreased in Abx-treated mice, whereas S1PR1<sup>+</sup> ILCs were increased (<a href="#F2" class="usa-link">Fig. 2A</a> and fig. S2, F and G). To determine that the reduction of intestinal ILCs caused by Abx treatment is due to their reentry into circulation and migration out of SI, we simultaneously treated mice with Abx and S1PR1 inhibitor FTY720 to prevent the entrance into lymphatic vessels. Flow cytometry analysis confirmed that intestinal ILC1s, ILC2s, and ILC3s were increased in Abx and FTY720 cotreated mice compared to those in mice treated with Abx alone (<a href="#F2" class="usa-link">Fig. 2B</a>). In contrast, three ILC subsets from mLN (<a href="#F2" class="usa-link">Fig. 2C</a>) and spleen (<a href="#F2" class="usa-link">Fig. 2D</a>) were decreased in these cotreated mice. In addition, after Abx treatment, numbers of S1PR1<sup>+</sup> ILCs in the mLN and spleen increased substantially, while blockade of S1PR1 with FTY720 reduced accumulation of migratory ILCs (fig. S3A). Moreover, we sorted intestinal ILCs from tdTomato-reporter mice and performed adoptive transfer experiments. We found that, with increasing duration of antibiotic treatment, numbers of tdTomato<sup>+</sup> ILCs decreased in the intestine but increased in the spleen (fig. S3, B to D). These results indicate that microbiota disruption abrogates the tissue residency of intestinal ILCs and enhances their S1PR1-dependent egression.</p>
<figure class="fig xbox font-sm" id="F2"><h4 class="obj_head">Fig. 2. Microbiota disruption causes migration of intestinal ILCs to a circulation system.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12372891_sciadv.adt9645-f2.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/3fdc/12372891/76e76e43843b/sciadv.adt9645-f2.jpg" loading="lazy" height="914" width="732" alt="Fig. 2."></a></p>
<div class="p text-right font-secondary"><a href="figure/F2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>(<strong>A</strong>) 3D Immunofluorescence staining of CD69<sup>+</sup> and S1PR1<sup>+</sup> ILCs in SI from Ctrl and Abx-treated mice. White arrows denote CD69<sup>+</sup> and S1PR1<sup>+</sup> ILCs. Scale bars, 100 μm. (<strong>B</strong> to <strong>D</strong>) Mice were intraperitoneally injected FTY720 every other day and treated with Abx for 2 weeks. ILC1s, ILC2s, and ILC3s in SI (B), mLN (C), and spleen (D) derived from Ctrl, Abx-treated, and Abx + FTY720–treated mice were analyzed by flow cytometry. <em>n</em> = 5. (<strong>E</strong> to <strong>G</strong>) PE-labeled anti-CD45 antibody (2 μg) was intravenously injected into mice. Ten minutes later, PE-labeled intestinal ILC1s (E), ILC2s (F), and ILC3s (G) from Ctrl and Abx-treated mice were detected by flow cytometry. Frequencies of PE-labeled ILC1s, ILC2s, and ILC3s were calculated. <em>n</em> = 4. (<strong>H</strong>) Lin<sup>−</sup> intestinal lymphocytes (1 × 10<sup>7</sup>) from CD45.2<sup>+</sup> donor mice were sorted and intravenously injected into lethally irradiated CD45.1<sup>+</sup> Ctrl or Abx-treated recipients. Eighteen hours later, donor-derived ILC subsets homing to intestine were analyzed by flow cytometry. Frequencies and numbers of intestinal CD45.2<sup>+</sup> ILC subsets in indicated recipients are shown in the right panel. <em>n</em> = 4. (<strong>I</strong>) Bone marrow (BM) cells (5 × 10<sup>6</sup>) from CD45.2<sup>+</sup> donor mice were intravenously injected into lethally irradiated CD45.1<sup>+</sup> Ctrl or Abx-treated recipients. Eight weeks later, homing and stable localization of donor-derived ILC subsets in the intestine were analyzed by flow cytometry. Frequencies and numbers of intestinal CD45.2<sup>+</sup> ILC subsets in indicated recipients are shown in the right panel. <em>n</em> = 4. Data are representative of at least three independent experiments and are shown as the means ± SD. Statistical analysis was performed by using unpaired two-tailed Student’s <em>t</em> test (*<em>P</em> &lt; 0.05; **<em>P</em> &lt; 0.01; ***<em>P</em> &lt; 0.001; n.s., not significant). SSC, side scatter; W, width.</p></figcaption></figure><p>To directly detect the circulatory ILCs, we performed intravascular antibody staining to label cells in vascular systems. Consistent with our above observations, much more ILC1s (<a href="#F2" class="usa-link">Fig. 2E</a>), ILC2s (<a href="#F2" class="usa-link">Fig. 2F</a>), and ILC3s (<a href="#F2" class="usa-link">Fig. 2G</a>) from Abx-treated mice were labeled by intravascular antibody. Next, we determined whether a stable microbiota environment is necessary for homing and positioning of ILCs in SI. Intestinal Lin<sup>−</sup> lymphocytes isolated from CD45.2<sup>+</sup> donor mice were injected intravenously into CD45.1<sup>+</sup> recipients for a short-term homing assay. In Abx recipients, ILC subsets were lower efficient in migrating to SI compared to Abx-untreated (Ctrl) recipients (<a href="#F2" class="usa-link">Fig. 2H</a>). Furthermore, we transferred CD45.2<sup>+</sup> bone marrow (BM) cells into same CD45.1<sup>+</sup> Abx recipients for a long-term homing assay to determine the importance of gut microbiota on retention of intestinal ILCs. Eight weeks later, Abx recipients showed a lower proportion and fewer numbers of intestinal ILC subsets (<a href="#F2" class="usa-link">Fig. 2I</a>), whereas higher percentages and greater numbers of ILC subsets were observed in mLN and spleen (fig. S3, E and F), suggesting that the intestinal environment with damaged microbiota is not conducive to homing and residency of ILCs. Together, our results reveal that disruption of gut microbiota promotes intestinal resident ILCs to exit their dwelling tissues and enter a circulation system.</p></section><section id="sec5"><h3 class="pmc_sec_title">P2Y10 is highly expressed on ILCs and mediates intestinal tissue residency</h3>
<p>Intestinal ILCs sense different kinds of contents derived from gut microbiota, which plays a critical role in maintaining tissue homeostasis (<a href="#R10" class="usa-link" aria-describedby="R10"><em>10</em></a>, <a href="#R24" class="usa-link" aria-describedby="R24"><em>24</em></a>). However, it remains unclear whether gut microbiota metabolites contribute to regulate the long-term stability of intestinal ILCs in position. To test the effect of gut microbiota on regulation of ILCs transcriptome, we conducted RNA sequencing (RNA-seq) analysis of ILCs from Ctrl versus Abx-treated mice. We noticed that ILCs from Ctrl mice markedly up-regulated Trm signature genes, whereas ILCs from Abx-treated mice were more similar to Tcm (fig. S4A). We also conducted same transcriptome analysis of ILC1, ILC2, and ILC3 from single-cell RNA sequencing (scRNA-seq) data (<a href="#R25" class="usa-link" aria-describedby="R25"><em>25</em></a>) and found that Trm signature genes were enriched in all three ILC subsets from Ctrl mice (fig. S4B). Additionally, we found that genes up-regulated in Ctrl ILCs were associated with tissue residency, mainly encoding integrins and some adhesive receptors. By contrast, genes enriched in Abx ILCs were related to cell migration and mobilization (fig. S4, C and D). Expectedly, gene ontology (GO) pathway enrichment analysis showed that pathways associated with innate immune response and cell-cell adhesion were enriched in Ctrl ILCs. GPCR activity and downstream signaling pathways displayed prominent enrichment (<a href="#F3" class="usa-link">Fig. 3A</a>). In recent years, numerous studies have shown that GPCRs bind to various microbial metabolites and participate in immune and metabolic regulation in the intestine. Gut microbiota typically exerts its functions by recognizing GPCRs and modulating downstream signaling (<a href="#R24" class="usa-link" aria-describedby="R24"><em>24</em></a>, <a href="#R26" class="usa-link" aria-describedby="R26"><em>26</em></a>, <a href="#R27" class="usa-link" aria-describedby="R27"><em>27</em></a>). Then, we analyzed expression of GPCRs and chose top 10 GPCR genes most highly expressed in Ctrl ILCs for further quantitative polymerase chain reaction (qPCR) validation (<a href="#F3" class="usa-link">Fig. 3B</a>). We found that several chemokine receptors were down-regulated in ILCs from Abx-treated mice, such as Cxcr6, Ccr5, and Cxcr4, suggesting that gut microbiota modulates chemotaxis and migration of ILCs (<a href="#F3" class="usa-link">Fig. 3C</a>). We further noticed that another GPCR gene, <em>P2ry10</em>, was highly expressed in Ctrl ILCs, while it was reduced after Abx treatment (<a href="#F3" class="usa-link">Fig. 3C</a> and fig. S4, E and F). Then, we sorted various immune cells from SI and detect the expression of <em>P2ry10</em>, and we found that <em>P2ry10</em> is most abundantly expressed in ILCs, suggesting that <em>P2ry10</em> plays an important role in intestinal ILCs (fig. S4G). Moreover, mature ILC1s, ILC2s, and ILC3s exhibited higher <em>P2ry10</em> expression compared to their precursors (fig. S4H). Compared with ILCs from lymphoid tissues such as spleen and mLN, SI-derived ILCs showed higher expression of <em>P2ry10</em> (fig. S4I). Consistently, more ILCs expressing P2Y10 were observed in Ctrl mice compared with those in microbiota-deficient mice (<a href="#F3" class="usa-link">Fig. 3D</a> and fig. S4, J to L). These data indicate that Abx treatment suppresses resident signatures of ILCs and down-regulates <em>P2ry10</em> expression on intestinal ILCs.</p>
<figure class="fig xbox font-sm" id="F3"><h4 class="obj_head">Fig. 3. P2Y10 is highly expressed on ILCs and mediates intestinal residency of ILCs.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12372891_sciadv.adt9645-f3.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/3fdc/12372891/05acdecd80bf/sciadv.adt9645-f3.jpg" loading="lazy" height="875" width="798" alt="Fig. 3."></a></p>
<div class="p text-right font-secondary"><a href="figure/F3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>(<strong>A</strong>) Transcriptome analysis of sorted intestinal ILCs (Lin<sup>−</sup>CD127<sup>+</sup>Thy1.2<sup>+</sup>) in Ctrl and Abx-treated mice. Highly expressed genes in Ctrl ILCs were performed GO enrichment analysis. (<strong>B</strong>) Expression of GPCRs was shown, and top 10 of the most specifically and highly expressed GPCRs in Ctrl ILCs were marked. (<strong>C</strong>) Relative mRNA levels of the candidate GPCRs in sorted Ctrl and Abx ILCs. Fold changes were normalized to <em>Actb</em>. (<strong>D</strong>) Immunofluorescence of P2Y10-expressing intestinal ILCs (white arrows). Scale bars, 30 μm. (<strong>E</strong> to <strong>G</strong>) Flow cytometry analysis of ILC1s, ILC2s, and ILC3s in the SI (E), mLN (F), and spleen (G) from <em>P2ry10</em><sup>+/+</sup> and <em>P2ry10</em><sup>−/−</sup> mice. <em>n</em> = 5. (<strong>H</strong>) Immunofluorescence of the SI from <em>P2ry10</em><sup>+/+</sup> and <em>P2ry10</em><sup>−/−</sup> mice for analysis of gut-associated lymphoid tissues. Scale bars, 2000 μm (left), 70 μm (right). <em>n</em> = 5. (<strong>I</strong>) Flow cytometry analysis of CD69<sup>+</sup> and S1PR1<sup>+</sup> ILCs derived from SI of <em>P2ry10</em><sup>+/+</sup> and <em>P2ry10</em><sup>−/−</sup> mice. <em>n</em> = 5. (<strong>J</strong>) <em>P2ry10</em><sup>+/+</sup> or <em>P2ry10</em><sup>−/−</sup> CD45.2<sup>+</sup> donor mice underwent parabiosis surgery with CD45.1<sup>+</sup> recipient mice. Flow cytometry analysis of donor-derived intestinal ILC1s, ILC2s, and ILC3s from the CD45.1<sup>+</sup> parabionts. <em>n</em> = 4. (<strong>K</strong>) Relative mRNA levels of <em>P2RY10</em> in ILCs from noninflamed and inflamed intestinal samples of patients with CD. Fold changes were normalized to <em>ACTB</em>. <em>n</em> = 4 independent experiments. (<strong>L</strong>) Flow cytometry analysis of the expression of P2Y10 in intestinal ILCs from patients with CD. <em>n</em> = 6. Data are representative of at least three independent experiments and are shown as the means ± SD. Statistical analysis was performed by using unpaired two-tailed Student’s <em>t</em> test (*<em>P</em> &lt; 0.05; **<em>P</em> &lt; 0.01; ***<em>P</em> &lt; 0.001; n.s., not significant). SSC, side scatter; W, width.</p></figcaption></figure><p>To further examine the impact of P2Y10 deficiency on ILCs retention, we generated <em>P2ry10</em>-deficient mice (fig. S5A). Consistent with the phenotype of Abx-treated mice, <em>P2ry10</em><sup>−/−</sup> mice showed decreased ILC1s, ILC2s, and ILC3s in SI (<a href="#F3" class="usa-link">Fig. 3E</a> and fig. S5B) but increased ILC subsets in mLN (<a href="#F3" class="usa-link">Fig. 3F</a> and fig. S5C) and spleen (<a href="#F3" class="usa-link">Fig. 3G</a> and fig. S5D) compared with their <em>P2ry10</em><sup>+/+</sup> littermates. Subsequent immunofluorescence staining further confirmed the reduction of ILC1s, ILC2s, and ILC3s in the intestine of <em>P2ry10</em><sup>−/−</sup> mice (fig. S5, E to G). Given the involvement of ILC3s, particularly CCR6<sup>+</sup> ILC3s in the formation of gut-associated lymphoid tissues (<a href="#R28" class="usa-link" aria-describedby="R28"><em>28</em></a>), we used immunofluorescence staining to visualize intestinal lymphoid structures and found reduced intestinal PPs and isolated lymphoid follicles in <em>P2ry10</em><sup>−/−</sup> mice (<a href="#F3" class="usa-link">Fig. 3H</a>). Meanwhile, flow cytometry analysis revealed that <em>P2ry10</em> knockout reduced numbers of ILC2s and ILC3s in PPs, but no impact on ILC1s (fig. S5H). In addition, Ki67 and annexin V analysis showed that P2Y10 deficiency did not disturb proliferation and apoptosis of ILCs (fig. S5, I and J). Moreover, P2Y10 deficiency caused decreased expression of CD69 on ILCs, while expression of S1PR1 was increased (<a href="#F3" class="usa-link">Fig. 3I</a> and fig. S5K). Then, we conducted BM transplantation experiments to investigate whether intestinal homing and long-term residency of ILCs are dependent on P2Y10 receptor. Because <em>P2ry10</em> knockout did not affect the proportion of ILC precursors (fig. S6A), an equal number (5 × 10<sup>6</sup>) of BM cells from <em>P2ry10</em><sup>+/+</sup> or <em>P2ry10</em><sup>−/−</sup> were, respectively, transplanted into irradiated recipients. <em>P2ry10</em>-deficient chimeric mice showed reduced homing ILC subpopulations in SI compared with control chimeric mice (fig. S6B). Collectively, we conclude that P2Y10 receptor is required for intestinal homing and subsequent formation of long-term resident ILC populations.</p>
<p>In addition, we generated parabiosis mice by jointing <em>P2ry10</em><sup>+/+</sup> or <em>P2ry10</em><sup>−/−</sup> CD45.2<sup>+</sup> mice with CD45.1<sup>+</sup> recipient mice to confirm the impact of P2Y10 deficiency on the residency of intestinal ILCs (fig. S6C). Due to a shared circulation system, circulating T and B cells in blood and spleen could reach equilibrium between parabionts (fig. S6D). Next, we analyzed proportions of ILC1s, ILC2s, and ILC3s derived from different donor mice in recipient parabionts. We found that, compared to recipients jointed with wild-type (WT) donor mice, recipients paired with <em>P2ry10</em><sup>−/−</sup> donor mice exhibited increased proportions of CD45.2<sup>+</sup> ILC subsets in SI (<a href="#F3" class="usa-link">Fig. 3J</a>) and blood (fig. S6E). These results indicate that loss of P2Y10 receptor disrupts residency of ILCs, leading to their entry into circulation and subsequent migration into paired parabionts via a shared circulation system. To further detect P2Y10 expression in human intestinal ILCs, we firstly analyzed RNA-seq data in patients with inflammatory bowel disease (IBD) from the Gene Expression Omnibus database (<a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE57945" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE57945</a>) and observed down-regulation of <em>P2RY10</em> in patients with Crohn’s disease (CD) (fig. S7A). Subsequently, we collected ileac biopsies from endoscopically inflamed and noninflamed sites in patients with CD and then analyzed P2Y10 expression in ILCs by qPCR assay, flow cytometry, and immunofluorescence staining. We observed that most of ILCs from noninflamed intestinal samples expressed P2Y10 and residency marker CD69. However, P2Y10<sup>+</sup> and CD69<sup>+</sup> ILCs were decreased in inflamed intestine, while S1PR1<sup>+</sup> ILCs were increased (<a href="#F3" class="usa-link">Fig. 3, K and L</a>; and fig. S7, B to E), suggesting that disturbed tissue residency of ILCs may contribute to the pathogenesis of intestinal inflammation. Together, P2Y10 is highly expressed on ILCs and mediates their intestinal tissue residency.</p></section><section id="sec6"><h3 class="pmc_sec_title">Microbiota metabolite TDCA is a ligand to P2Y10 receptor to mediate tissue-resident signaling of intestinal ILCs</h3>
<p>We next intended to identify microbiota metabolites that engaged with P2Y10 receptor. Through untargeted metabolomics analysis on gut contents from Ctrl and GF mice, we found that disruption of microbiota caused significant alterations in the abundance of numerous metabolites. We chose metabolites specifically and highly enriched in Ctrl mice but markedly decreased in GF mice (fold change &gt; 2) for further screening (<a href="#F4" class="usa-link">Fig. 4A</a>). Because P2Y10 receptor belongs to GPCRs, its activation can be assessed by detecting Ca<sup>2+</sup> flux with a calcium indicator, Fluo-4 AM (<a href="#R29" class="usa-link" aria-describedby="R29"><em>29</em></a>). Among these highly enriched metabolites, TDCA stimulation showed maximal activation of P2Y10 receptor (<a href="#F4" class="usa-link">Fig. 4B</a>). TDCA is a secondary bile acid that requires microbiota for its generation. We noticed that TDCA level was much lower in GF mice compared to that in Ctrl mice, while the levels of its precursors, TCDCA and TCA, were higher in GF mice (fig. S8A), suggesting that gut microbiota metabolizes TCDCA and TCA to form TDCA. In addition, TDCA was able to effectively activate P2Y10 compared to TCDCA or TCA (<a href="#F4" class="usa-link">Fig. 4C</a> and fig. S8B). Apart from calcium mobilization, previous studies have reported that P2Y10 receptor can also activate downstream RhoA signaling (<a href="#R14" class="usa-link" aria-describedby="R14"><em>14</em></a>). Subsequently, G-LISA and luciferase reporter assay were used to verify the activation effect of TDCA on RhoA signaling pathway (<a href="#F4" class="usa-link">Fig. 4D</a> and fig. S8, C and D). Furthermore, median effective concentration for binding of TDCA to P2Y10 was 0.27 μM (<a href="#F4" class="usa-link">Fig. 4D</a>).</p>
<figure class="fig xbox font-sm" id="F4"><h4 class="obj_head">Fig. 4. TDCA is a ligand to P2Y10 receptor.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12372891_sciadv.adt9645-f4.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/3fdc/12372891/836b41ac6ec1/sciadv.adt9645-f4.jpg" loading="lazy" height="913" width="759" alt="Fig. 4."></a></p>
<div class="p text-right font-secondary"><a href="figure/F4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>(<strong>A</strong>) Metabolome analysis of the intestinal contents from Ctrl and GF mice. Significantly differentiated metabolites were shown. (<strong>B</strong>) Response of P2Y10 to the metabolites was analyzed by detecting Ca<sup>2+</sup> flux with Fluo-4 AM (2 μM). Related fluorescence intensity (<em>F</em>/<em>F</em><sub>0</sub>) was calculated. <em>n</em> = 3. (<strong>C</strong>) Response of P2Y10 to TDCA and its metabolite precursors were analyzed as above. <em>n</em> = 3. (<strong>D</strong>) Response of P2Y10 to different doses of TDCA was analyzed by detecting active RhoA through luciferase reporter assay. Median effective concentration (EC<sub>50</sub>) was calculated. FC, fold change. (<strong>E</strong>) Flow cytometry analysis of RhoA activation in intestinal ILCs from <em>P2ry10</em><sup>+/+</sup> or <em>P2ry10</em><sup>−/−</sup> mice after TDCA stimulation. (<strong>F</strong>) Active RhoA staining in sorted intestinal ILCs from <em>P2ry10</em><sup>+/+</sup> or <em>P2ry10</em><sup>−/−</sup> mice. Scale bars, 5 μm. (<strong>G</strong>) Mice were orally administrated with TDCA (1 mg/20 g) daily for 2 weeks while maintained on Abx drinking. Flow cytometry analysis of intestinal ILC1s, ILC2s, and ILC3s. <em>n</em> = 4. (<strong>H</strong>) <em>P2ry10</em><sup>+/+</sup> and <em>P2ry10</em><sup>−/−</sup> mice kept on normal drinking were orally administrated TDCA (1 mg/20 g) daily for 2 weeks. Flow cytometry analysis of intestinal ILC1s, ILC2s, and ILC3s. <em>n</em> = 4. (<strong>I</strong> and <strong>J</strong>) Similar mice treatment as in (G) and (H) was conducted. Flow cytometry analysis of intestinal CD69<sup>+</sup> and S1PR1<sup>+</sup> ILCs. <em>n</em> = 4. (<strong>K</strong>) Intestinal ILCs were cultured with TDCA (10 μM), Ca<sup>2+</sup> inhibitor (U-73122, 1 μM), or ROCK inhibitor [(S)-H-1152, 1 μM], and, then, mRNA expression of <em>Cd69</em> and <em>S1pr1</em> was analyzed. Fold changes were normalized to <em>Actb</em>. Data are representative of at least three independent experiments and are shown as the means ± SD. Statistical analysis was performed by using unpaired two-tailed Student’s <em>t</em> test (*<em>P</em> &lt; 0.05; **<em>P</em> &lt; 0.01; ***<em>P</em> &lt; 0.001; n.s., not significant). SSC, side scatter; W, width.</p></figcaption></figure><p>In addition, we validated the effect of TDCA on intestinal ILCs. Flow cytometry analysis showed that TDCA stimulation enhanced the level of active RhoA in <em>P2ry10</em><sup>+/+</sup> ILCs, but not in <em>P2ry10</em><sup>−/−</sup> ILCs (<a href="#F4" class="usa-link">Fig. 4E</a>). Consistently, immunofluorescence staining showed that ILCs from <em>P2ry10</em><sup>+/+</sup> mice exhibited a marked increase in intracellular active RhoA level after TDCA stimulation (<a href="#F4" class="usa-link">Fig. 4F</a>). Next, we explored whether TDCA regulates the retention of intestinal ILCs. TDCA orally administration rescued a reduction in the number of intestinal ILC subsets resulted from Abx treatment (<a href="#F4" class="usa-link">Fig. 4G</a>), whereas such protective effect was lacking in <em>P2ry10</em><sup>−/−</sup> mice (<a href="#F4" class="usa-link">Fig. 4H</a>). Of note, compared with those in the untreated group, numbers of ILC1s, ILC2s, and ILC3s in intestines of <em>P2ry10</em><sup>+/+</sup> mice slightly increased with TDCA treatment (<a href="#F4" class="usa-link">Fig. 4H</a>).</p>
<p>To further determine whether TDCA-P2Y10 interaction regulates the expression of CD69 and S1PR1, we administered TDCA to Abx-treated mice followed by flow cytometry analysis. Compared to those in mice treated with Abx alone, CD69<sup>+</sup> ILCs were increased in Abx-treated mice supplemented with TDCA administration, while S1PR1<sup>+</sup> ILCs were decreased (<a href="#F4" class="usa-link">Fig. 4I</a> and fig. S8E). By contrast, in <em>P2ry10</em><sup>−/−</sup> mice, the positive regulatory effect of TDCA on CD69 expression was abolished (<a href="#F4" class="usa-link">Fig. 4J</a> and fig. S8F). To determine the role of P2Y10 downstream signaling in regulating expression of resident molecules, we sorted intestinal ILCs and stimulated them with TDCA. We found that TDCA increased <em>Cd69</em> expression in sorted ILCs, while decreased <em>S1pr1</em> expression (<a href="#F4" class="usa-link">Fig. 4K</a>). Given that TDCA engagement with P2Y10 receptor not only stimulated Ca<sup>2+</sup> flux but also activated RhoA signaling, we used a phospholipase (PLC) inhibitor (U-73122) and a ROCK inhibitor [(S)-H-1152] to separately inhibit Ca<sup>2+</sup> and RhoA signaling pathways. Both U-73122 and (S)-H-1152 eliminated the positive effect of TDCA on <em>Cd69</em> expression, while relieved the inhibition on <em>S1pr1</em> expression (<a href="#F4" class="usa-link">Fig. 4K</a>). Collectively, microbiota metabolite TDCA mediates tissue residency of intestinal ILCs via engagement with P2Y10 receptor.</p></section><section id="sec7"><h3 class="pmc_sec_title">TDCA-P2Y10 engagement initiates <em>Zfp414</em> transcription</h3>
<p>We intended to further define the mechanism by which TDCA-P2Y10–mediated signaling regulates long-term retention of intestinal resident ILCs. We performed RNA-seq of intestinal ILCs isolated from <em>P2ry10</em><sup>+/+</sup> and <em>P2ry10</em><sup>−/−</sup> mice. Gene set enrichment analysis (GSEA) showed that Tcm signature genes were enriched in <em>P2ry10</em><sup>−/−</sup> ILCs, while genes up-regulated in Abx-untreated ILCs were enriched in <em>P2ry10</em><sup>+/+</sup> ILCs (fig. S9A). In line with above results (<a href="#F3" class="usa-link">Fig. 3A</a>), several pathways such as positive regulation of cell adhesion, lymphocyte-mediated immunity, and lymphocyte activation involved in immune response were enriched in <em>P2ry10</em><sup>+/+</sup> ILCs, but not in <em>P2ry10</em><sup>−/−</sup> ILCs (<a href="#F5" class="usa-link">Fig. 5A</a>). Of note, the regulatory effects of TFs were attenuated in <em>P2ry10</em><sup>−/−</sup> ILCs (<a href="#F5" class="usa-link">Fig. 5A</a>). To further explore the TFs involved in regulation of tissue residency of ILCs, we identified 227 TFs that were simultaneously highly expressed in all ILC subsets from Abx-untreated mice. Subsequently, we analyzed expression of the 227 TFs in intestinal ILCs from <em>P2ry10</em><sup>+/+</sup> and <em>P2ry10</em><sup>−/−</sup> mice and picked out eight TFs with higher expression in <em>P2ry10</em><sup>+/+</sup> ILCs (<a href="#F5" class="usa-link">Fig. 5B</a>). Further qPCR validation revealed that <em>Zfp414</em> was the most highly expressed TF in Ctrl ILCs, while its expression was down-regulated after Abx treatment and <em>P2ry10</em> knockout (<a href="#F5" class="usa-link">Fig. 5C</a>). Subsequently, to investigate the role of ZFP414 in regulating tissue residency of intestinal ILCs, we constructed BM chimeric mice with <em>Zfp414</em> knockout using CRISPR-based mutagenesis in BM cells. <em>Zfp414</em> deficiency caused impaired homing of ILC subsets, and reintroduction of <em>Zfp414</em> rescued reduced numbers of intestinal ILC1s, ILC2s, and ILC3s (<a href="#F5" class="usa-link">Fig. 5D</a> and fig. S9, B and C). We further performed in vitro differentiation assay to test the effect of Zfp414 deficiency on ILC differentiation. We found that <em>Zfp414</em> depletion did not affect ILC differentiation (fig. S9D), indicating that the impaired homing of ILC subsets caused by <em>Zfp414</em> deficiency is not due to the defect of ILC differentiation. In addition, we observed that <em>P2ry10</em> reintroduction or <em>Zfp414</em> overexpression (fig. S9E) in <em>P2ry10</em><sup>−/−</sup> ILCs could increase the proportion of long-term resident ILC subsets (<a href="#F5" class="usa-link">Fig. 5E</a>), suggesting that ZFP414 is necessary for tissue residency of intestinal ILCs. Moreover, inhibition of P2Y10 downstream signaling pathways with U-73122 and (S)-H-1152 blocked the beneficial effect of TDCA on <em>Zfp414</em> expression (<a href="#F5" class="usa-link">Fig. 5F</a>). Consistent with the expression of P2Y10, ZFP414 was expressed in human ILCs derived from noninflamed intestine but down-regulated in inflamed ILCs (<a href="#F5" class="usa-link">Fig. 5, G and H</a>). These results indicate that TDCA-P2Y10–mediated signaling promotes the transcription of <em>Zfp414</em> for tissue residency of intestinal ILCs.</p>
<figure class="fig xbox font-sm" id="F5"><h4 class="obj_head">Fig. 5. TDCA-P2Y10 engagement initiates <em>Zfp414</em> transcription.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12372891_sciadv.adt9645-f5.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/3fdc/12372891/962f20c4adc0/sciadv.adt9645-f5.jpg" loading="lazy" height="907" width="773" alt="Fig. 5."></a></p>
<div class="p text-right font-secondary"><a href="figure/F5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>(<strong>A</strong>) Transcriptome analysis of sorted intestinal ILCs from <em>P2ry10</em><sup>+/+</sup> and <em>P2ry10</em><sup>−/−</sup> mice. Down-regulated genes in <em>P2ry10</em><sup>−/−</sup> ILCs were performed GO enrichment analysis. (<strong>B</strong>) Venn diagram of transcription factors (TFs) that were simultaneously enriched in Ctrl total ILCs and ILC subsets. The expression of TFs in intestinal ILCs from <em>P2ry10</em><sup>+/+</sup> and <em>P2ry10</em><sup>−/−</sup> mice was analyzed. FC = <em>P2ry10</em><sup>−/−</sup> ILCs/<em>P2ry10</em><sup>+/+</sup> ILCs. (<strong>C</strong>) Relative mRNA levels of the marked TFs above in <em>P2ry10</em><sup>+/+</sup> and <em>P2ry10</em><sup>−/−</sup> ILCs. Fold changes were normalized to <em>Actb</em>. (<strong>D</strong>) Construct BM chimeric mice with <em>Zfp414</em> deletion and reintroduction. Flow cytometry analysis of CD45.2<sup>+</sup> intestinal ILC1s, ILC2s, and ILC3s in different BM chimeras. <em>n</em> = 4. (<strong>E</strong>) Construct <em>P2ry10</em><sup>−/−</sup> BM chimeras with <em>P2ry10</em> or <em>Zfp414</em> overexpression. Flow cytometry analysis of CD45.2<sup>+</sup> intestinal ILC1s, ILC2s, and ILC3s in different BM chimeras. <em>n</em> = 4. (<strong>F</strong>) Sorted Intestinal ILCs were cultured as in <a href="#F4" class="usa-link">Fig. 4K</a>, and, then, the mRNA expression of <em>Zfp414</em> was analyzed. Fold changes were normalized to <em>Actb</em>. (<strong>G</strong>) Relative mRNA levels of <em>ZFP414</em> in ILCs from noninflamed and inflamed intestinal samples of patients with CD. Fold changes were normalized to <em>ACTB</em>. <em>n</em> = 4 independent experiments. (<strong>H</strong>) Immunofluorescence staining of intestinal ILCs expressing ZFP414 in noninflamed and inflamed intestinal samples of patients with CD. White arrows indicate intestinal ZFP414<sup>+</sup> ILCs. Scale bars, 20 μm. Percentages of ZFP414<sup>+</sup> ILCs in CD127<sup>+</sup> cells per field are shown in the right panel. <em>n</em> = 6 fields. Data are representative of at least three independent experiments and are shown as the means ± SD. Statistical analysis was performed by using unpaired two-tailed Student’s <em>t</em> test (*<em>P</em> &lt; 0.05; **<em>P</em> &lt; 0.01; ***<em>P</em> &lt; 0.001; n.s., not significant).</p></figcaption></figure></section><section id="sec8"><h3 class="pmc_sec_title">ZFP414 activates expression of integrin αE and CD69</h3>
<p>We next aimed to explore how TDCA-P2Y10-ZFP414 axis mediates the intestinal residency of ILCs. We observed that, besides TF activity, cell adhesion mediated by integrins showed a difference after Abx treatment (<a href="#F3" class="usa-link">Fig. 3A</a>). Integrins are a family of cell adhesion receptors that play a critical role in cell-cell adhesion. Previous reports have indicated that integrins play an important role in mediating tissue residency of lymphocytes (<a href="#R30" class="usa-link" aria-describedby="R30"><em>30</em></a>, <a href="#R31" class="usa-link" aria-describedby="R31"><em>31</em></a>). To determine the role of integrins in intestinal resident ILCs, we analyzed expression of integrins and identified five integrins that were down-regulated in Abx-treated and P2Y10-deficient ILCs (<a href="#F6" class="usa-link">Fig. 6A</a>). Further qPCR validation showed that <em>Itgae</em> (coding Integrin αE) and its interactor <em>Itgb7</em> displayed higher expression in untreated WT ILCs, whereas they were markedly decreased by Abx treatment and P2Y10 deficiency (<a href="#F6" class="usa-link">Fig. 6B</a>). Integrin αE typically forms a heterodimer by binding to integrin β7, which has been identified as a key molecule in maintaining the tissue-resident property of CD8<sup>+</sup> T cells by facilitating adhesion of CD8<sup>+</sup> T cells to epithelial cells (<a href="#R17" class="usa-link" aria-describedby="R17"><em>17</em></a>). However, it is unknown whether integrin αE regulates the tissue residency of intestinal ILCs. Integrin αE exhibited lower expression in intestinal ILCs from Abx and <em>P2ry10</em><sup>−/−</sup> mice compared to those from their control littermates (<a href="#F6" class="usa-link">Fig. 6, C and D</a>; and fig. S9, F and G). Consistently, reduced interaction of ITAE<sup>+</sup> ILCs and E-cadherin<sup>+</sup> intestinal epithelia was observed in Abx and <em>P2ry10</em><sup>−/−</sup> mice (fig. S9, H and I).</p>
<figure class="fig xbox font-sm" id="F6"><h4 class="obj_head">Fig. 6. ZFP414 primes expression of integrin αE on ILCs.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12372891_sciadv.adt9645-f6.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/3fdc/12372891/046a20714072/sciadv.adt9645-f6.jpg" loading="lazy" height="780" width="748" alt="Fig. 6."></a></p>
<div class="p text-right font-secondary"><a href="figure/F6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>(<strong>A</strong>) Venn diagram of integrins that were simultaneously down-regulated in Abx and <em>P2ry10</em><sup>−/−</sup> ILCs. (<strong>B</strong>) Relative mRNA levels of integrins that were enriched in Ctrl and <em>P2ry10</em><sup>+/+</sup> ILCs. Fold changes were normalized to <em>Actb</em>. (<strong>C</strong> and <strong>D</strong>) Flow cytometry analysis of the expression of ITAE on intestinal ILCs derived from Abx-treated (C) and <em>p2ry10</em><sup>−/−</sup> mice (D). <em>n</em> = 5. (<strong>E</strong>) Flow cytometry analysis of the expression of ITAE on CD45.2<sup>+</sup> intestinal ILCs from different BM chimeras with <em>Zfp414</em> deletion and reintroduction. <em>n</em> = 4. (<strong>F</strong>) Construct <em>P2ry10</em><sup>−/−</sup> BM chimeras with <em>P2ry10</em> or <em>Zfp414</em> overexpression. Flow cytometry analysis of the expression of ITAE on CD45.2<sup>+</sup> intestinal ILCs from different BM chimeras. <em>n</em> = 4. (<strong>G</strong> and <strong>H</strong>) Mice were treated as in <a href="#F4" class="usa-link">Fig. 4 (G and H)</a>. Flow cytometry analysis of the expression of ITAE on intestinal ILCs from indicated mice. <em>n</em> = 4. (<strong>I</strong>) Sorted intestinal ILCs were cultured as in <a href="#F4" class="usa-link">Fig. 4K</a>, and, then, <em>Itgae</em> expression was analyzed. Fold changes were normalized to <em>Actb</em>. (<strong>J</strong>) ILCs with <em>Itgae</em> silencing were labeled with CFSE (5 μM) and transferred (5 × 10<sup>6</sup>) into immunodeficient mice. Immunofluorescence of SI was performed. Scale bars, 70 μm. <em>n</em> = 5. (<strong>K</strong>) Relative mRNA levels of <em>ITGAE</em> in ILCs from noninflamed and inflamed intestinal samples of patients with CD. Fold changes were normalized to <em>ACTB</em>. <em>n</em> = 4. (<strong>L</strong>) Immunofluorescence of intestinal ITAE<sup>+</sup> ILCs in noninflamed and inflamed intestinal samples of patients with CD. Scale bars, 20 μm. Percentages of ITAE<sup>+</sup> ILCs in CD127<sup>+</sup> cells per field are shown. <em>n</em> = 6 fields. Data are representative of at least three independent experiments and are shown as the means ± SD. Statistical analysis was performed by using unpaired two-tailed Student’s <em>t</em> test (*<em>P</em> &lt; 0.05; **<em>P</em> &lt; 0.01; ***<em>P</em> &lt; 0.001; n.s., not significant). SSC, side scatter; W, width.</p></figcaption></figure><p>To further explore whether ZFP414 regulated the expression of ITAE, we performed a chromatin immunoprecipitation (ChIP) assay with anti-ZFP414 antibody and found that ZFP414 was enriched in the <em>Itgae</em> promoter region (−400 to ~−200), suggesting that ZFP414 enriches onto the <em>Itgae</em> locus to regulate its transcription (fig. S9J). As expected, <em>Zfp414</em> deficiency resulted in decreased expression of integrin αE and CD69 in intestinal ILCs, which was restored by <em>Zfp414</em> reintroduction (<a href="#F6" class="usa-link">Fig. 6E</a> and fig. S10, A and B). Additionally, reintroducing <em>P2ry10</em> or overexpressing <em>Zfp414</em> in <em>P2ry10</em><sup>−/−</sup> ILCs partially rescued integrin αE and CD69 expression (<a href="#F6" class="usa-link">Fig. 6F</a> and fig. S10C). Consistent with the effect on CD69, TDCA administration restored expression of integrin αE in Abx ILCs, but not in <em>P2ry10</em><sup>−/−</sup> ILCs (<a href="#F6" class="usa-link">Fig. 6, G and H</a>; and fig. S10, D and E). Moreover, blockade of P2Y10 downstream signals eliminated the beneficial influence of TDCA on <em>Itgae</em> expression (<a href="#F6" class="usa-link">Fig. 6I</a>). To directly verify the function of integrin αE on ILCs positioning in the intestine, we silenced <em>Itgae</em> expression in intestinal ILCs and then transferred these ILCs into immunodeficient recipient NOD-SCID-Il2rg<sup>−/−</sup> (NSG) mice to analyze their homing and resident ability. Our results showed that <em>Itgae</em> knockdown decreased ILCs homing to intestine and adhesion to epithelial cells (<a href="#F6" class="usa-link">Fig. 6J</a> and fig. S10F). Subsequently, we analyzed the expression of integrin αE on human ILCs with qPCR assay and immunofluorescence staining. Compared to noninflamed intestine, inflamed-intestine derived ILCs displayed much lower expression of integrin αE (<a href="#F6" class="usa-link">Fig. 6, K and L</a>). Together, the TDCA-P2Y10-ZFP414 axis mediates tissue residency of intestinal ILCs through integrin αE and CD69.</p></section><section id="sec9"><h3 class="pmc_sec_title">Maintenance of intestinal resident ILCs contributes to protection against intestinal inflammation</h3>
<p>Intestinal resident ILCs play a key role in regulating the function of intestinal epithelial barrier, contributing to tissue homeostasis and host defense (<a href="#R32" class="usa-link" aria-describedby="R32"><em>32</em></a>). To explore the physiological significance for maintaining stable residency of ILCs in the intestine, we generated <em>Zfp414</em>, <em>Itgae</em>-deficient mice and triple-deficient mice (sh<em>Zfp414</em>; sh<em>Itgae</em>; <em>P2ry10</em><sup>−/−</sup>) through short hairpin RNA (shRNA)–based gene silencing (fig. S10G). Then, we used a dextran sulfate sodium (DSS)–induced intestinal injury model to determine whether the P2Y10-ZFP414-ITAE axis in ILCs contributes to regulation of intestinal injury and inflammation. Deficiency of <em>P2ry10</em>, <em>Zfp414</em>, or <em>Itgae</em>, respectively, showed exacerbated intestinal inflammation with DSS treatment, as evidenced by increased epithelial damage and mucosal thickness (<a href="#F7" class="usa-link">Fig. 7A</a>), higher pathological scores (<a href="#F7" class="usa-link">Fig. 7B</a>), and faster weight loss (<a href="#F7" class="usa-link">Fig. 7C</a>). Furthermore, simultaneous deficiencies of these three molecules caused most severe intestinal inflammation. Next, we investigated whether maintaining stable residency of ILCs by TDCA contributes to intestinal homeostasis. With DSS treatment, compared to that in the Abx-treated mice, intestinal inflammation in the control mice was slightly exacerbated, despite no notable difference in weight loss. TDCA treatment obviously alleviated intestinal inflammation in both control and Abx-treated mice (<a href="#F7" class="usa-link">Fig. 7, D to F</a>). In addition, to evaluate the effect of TDCA on host defense, <em>Citrobacter rodentium</em> infection model was used (<a href="#R33" class="usa-link" aria-describedby="R33"><em>33</em></a>). <em>P2ry10</em><sup>−/−</sup> mice infected with <em>C. rodentium</em> showed more severe colitis (<a href="#F7" class="usa-link">Fig. 7, G and H</a>), more rapid weight loss (<a href="#F7" class="usa-link">Fig. 7I</a>) and more bacteria load in colons (<a href="#F7" class="usa-link">Fig. 7J</a>). Of note, TDCA administration protected against <em>C. rodentium</em> infection in <em>P2ry10</em><sup>+/+</sup> mice, but not in <em>P2ry10</em><sup>−/−</sup> mice (<a href="#F7" class="usa-link">Fig. 7, G to J</a>). To definitively validate that the increased susceptibility to intestinal inflammation in <em>C. rodentium</em> models is attributed to the decreased tissue residency of ILCs, rather than other cell types expressing P2Y10, we transferred <em>P2ry10</em><sup>+/+</sup> or <em>P2ry10</em><sup>−/−</sup> ILCs into immunodeficient NSG mice and subsequently infected mice with <em>C. rodentium</em>. We observed that <em>P2ry10</em><sup>−/−</sup> ILCs were less effective than <em>P2ry10</em><sup>+/+</sup> ILCs on protecting NSG mice and clearing <em>C. rodentium</em>, displaying more severe colitis (<a href="#F7" class="usa-link">Fig. 7, K and L</a>), increased weight loss (<a href="#F7" class="usa-link">Fig. 7M</a>), and higher bacteria load (<a href="#F7" class="usa-link">Fig. 7N</a>). These results indicate that maintaining tissue residency of intestinal ILCs is beneficial for intestinal homeostasis and host defense against pathogen infection.</p>
<figure class="fig xbox font-sm" id="F7"><h4 class="obj_head">Fig. 7. TDCA administration promotes intestinal residency of ILCs and protection against intestinal inflammation.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12372891_sciadv.adt9645-f7.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/3fdc/12372891/e86058ae4d03/sciadv.adt9645-f7.jpg" loading="lazy" height="819" width="766" alt="Fig. 7."></a></p>
<div class="p text-right font-secondary"><a href="figure/F7/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>(<strong>A</strong>) Mice were treated with 2% (w/v) DSS in drinking water for 5 days and followed by regular drinking water for 5 days. At day 10, colons were collected and fixed with 4% PFA for paraffin section and H&amp;E staining. Scale bars, 50 μm. (<strong>B</strong> and <strong>C</strong>) Pathological scores (B) and weight change (C) of indicated mice. <em>n</em> = 6. (<strong>D</strong>) Mice were treated as in <a href="#F4" class="usa-link">Fig. 4G</a>, and, then, intestinal injury was induced as above with 2% (w/v) DSS. At day 10, colons were collected and fixed with 4% PFA for paraffin section and H&amp;E staining. Scale bars, 50 μm. (<strong>E</strong> and <strong>F</strong>) Pathological scores (E) and weight change (F) of indicated mice. <em>n</em> = 5. (<strong>G</strong>) Mice were pretreated as in <a href="#F4" class="usa-link">Fig. 4H</a> and then were orally infected with <em>C. rodentium</em> (5 × 10<sup>9</sup> CFU). At day 10, colons were collected for paraffin section and H&amp;E staining. Scale bars, 50 μm. (<strong>H</strong> and <strong>I</strong>) Pathological scores (H) and body weight change (I) of indicated mice. <em>n</em> = 5. (<strong>J</strong>) Bacteria loads in feces were calculated. <em>n</em> = 5. (<strong>K</strong>) ILCs were isolated from <em>P2ry10</em><sup>+/+</sup> or <em>P2ry10</em><sup>−/−</sup> mice by flow cytometry and transferred (1 × 10<sup>6</sup>) into NSG mice on day −2 and 2 postinfection with <em>C. rodentium</em> (1 × 10<sup>9</sup> CFU). Ten days later, colons were collected for paraffin section and H&amp;E staining. Scale bars, 50 μm. (<strong>L</strong> and <strong>M</strong>) Pathological scores (L) and body weight change (M) of indicated mice. <em>n</em> = 5. (<strong>N</strong>) Bacteria loads in feces were calculated. <em>n</em> = 5. Data are representative of at least three independent experiments and are shown as the means ± SD. Statistical analysis was performed by using unpaired two-tailed Student’s <em>t</em> test (*<em>P</em> &lt; 0.05; ***<em>P</em> &lt; 0.001; n.s., not significant).</p></figcaption></figure></section></section><section id="sec10"><h2 class="pmc_sec_title">DISCUSSION</h2>
<p>Tissue residency is a remarkable characteristic of ILCs. Upon reaching their destination, tissue-resident ILCs develop unique molecular characteristics influenced by local microenvironment, irrespective of their origin. However, little is known about how the tissue environment, such as the abundant microbiota in the gut, affects the stable residency of ILCs. In this study, we showed that gut microbiota is necessary for the maintenance of intestinal tissue residency of ILCs. Microbiota metabolite TDCA binds to P2Y10 receptor on ILCs to initiate downstream Ca<sup>2+</sup> and RhoA signaling pathways. TDCA-P2Y10 engagement induces <em>Zfp414</em> transcription to prime expression of resident molecules CD69 and integrin αE on ILCs. These two molecules mediate the maintenance of intestinal tissue residency of ILCs by tethering ILCs to intestinal epithelial cells and restrict them out of tissue.</p>
<p>Microbiota and their metabolites represent an additional layer of regulation for the intestinal immune system. Microbiota-mediated signals have been defined to modulate tissue-resident lymphocytes, such as Trm and liver-resident Kupffer cells (<a href="#R34" class="usa-link" aria-describedby="R34"><em>34</em></a>). Intestinal microbiota-derived SCFAs like acetate can directly regulate glycolytic activity of CD8<sup>+</sup> T cells by GPCRs to prolong Trm survival (<a href="#R35" class="usa-link" aria-describedby="R35"><em>35</em></a>). In addition, intestinal commensal bacteria elicit tissue-resident intestinal T helper 17 (T<sub>H</sub>17) cells, which are homeostatic and beneficial for barrier maintenance (<a href="#R36" class="usa-link" aria-describedby="R36"><em>36</em></a>). Dysregulation of microbiota causes less butyrate in the liver and reduces IL-18 production in hepatocytes, leading to defective maturation of tissue-resident NK in the liver (<a href="#R37" class="usa-link" aria-describedby="R37"><em>37</em></a>). The involvement of microbiota in regulating the function and development of intestinal ILCs were observed (<a href="#R38" class="usa-link" aria-describedby="R38"><em>38</em></a>, <a href="#R39" class="usa-link" aria-describedby="R39"><em>39</em></a>), but how the microbiota modulates intestinal tissue residency of ILCs has not yet been investigated. Herein, we found that all three ILC subsets were decreased in the SI of GF and Abx-treated mice, while ILC subsets were increased in lymphoid organs such as mLNs and spleen. Although total numbers of ILCs were reduced in the intestinal tracts, the proportion of ILCs in the intestinal lymphatic vessels was increased, suggesting that ILCs are no longer in a resident state and have acquired characteristics of circulation. Although conventional NK cells belong to ILC1s, we excluded them from our study and distinguished ILC1s from NK cells with CD127 expression. Now, ILC1s are known to be enriched in tissues and are generally considered to be tissue-resident cells, whereas NK cells are often regarded as circulatory cells (<a href="#R40" class="usa-link" aria-describedby="R40"><em>40</em></a>). Furthermore, our results indicate that the expression levels of <em>P2ry10</em> in intestinal NK cells are lower than those in ILCs (fig. S3G), suggesting that P2Y10 may not play a major regulatory role in intestinal NK cells.</p>
<p>P2Y10 belongs to a purinergic receptor family and is classified as a GPCR due to its sevenfold transmembrane structure. Previous studies have reported that P2Y10 plays a role in the regulation of T cell migration and is expressed by other immune cells (<a href="#R14" class="usa-link" aria-describedby="R14"><em>14</em></a>, <a href="#R15" class="usa-link" aria-describedby="R15"><em>15</em></a>, <a href="#R41" class="usa-link" aria-describedby="R41"><em>41</em></a>). We showed that, among intestinal immune cell populations, <em>P2ry10</em> exhibited the highest expression levels in ILCs, suggesting its important regulatory function in ILCs. In addition, we used an ILC adoptive transfer model combined with a <em>C. rodentium</em>–induced intestinal inflammation model in immunodeficient mice to exclude the role of T cells in the susceptibility to inflammation caused by <em>P2ry10</em> deficiency. We identified the gut microbiota metabolite TDCA, a secondary bile acid whose metabolism and abundance are modulated by gut microbiota (<a href="#R42" class="usa-link" aria-describedby="R42"><em>42</em></a>), as a ligand for the P2Y10 receptor. Emerging evidence suggests that microbiota-derived secondary bile acids play a critical role in regulation of intestinal immunity (<a href="#R43" class="usa-link" aria-describedby="R43"><em>43</em></a>, <a href="#R44" class="usa-link" aria-describedby="R44"><em>44</em></a>). As derivatives of chlorogenic acid (LCA) and deoxycholic acid (DCA), 3-oxoLCA can directly bind to retinoid-related orphan receptor γt (RORγt) and inhibit differentiation of T<sub>H</sub>17 cells, while isoallo-LCA enhances the differentiation of regulatory T cells with anti-inflammatory function by promotion of Foxp3 expression (<a href="#R45" class="usa-link" aria-describedby="R45"><em>45</em></a>). Here, we demonstrated that TDCA acts as a ligand for P2Y10 receptor. TDCA-P2Y10 engagement activates its downstream Ca<sup>2+</sup> and RhoA signaling pathways to mediate tissue residency of ILCs.</p>
<p>Activation of GPCRs downstream Ca<sup>2+</sup> and RhoA signaling pathways is involved in various biological processes. For instance, GPR55 on macrophage responds to <span class="font-variant-small-caps">l</span>-α-lysophosphatidylinositol, leading to release of Ca<sup>2+</sup> and activation of nuclear factor of activated T cells (<a href="#R46" class="usa-link" aria-describedby="R46"><em>46</em></a>). The RhoA signaling pathway is also involved in the regulation of TFs transcription. RhoA activates protein kinase N to regulate the activity of myocardin-related TF A (<a href="#R47" class="usa-link" aria-describedby="R47"><em>47</em></a>). Additionally, hyperactivate RhoA signaling is both necessary and sufficient to drive oncogenic TEAD/YAP transcription in tumor (<a href="#R48" class="usa-link" aria-describedby="R48"><em>48</em></a>). In our study, we showed that TDCA-P2Y10–mediated Ca<sup>2+</sup> and RhoA signaling pathways initiate <em>Zfp414</em> transcription to promote expression of resident molecules CD69 and integrin αE on ILCs.</p>
<p>In SI, CD8<sup>+</sup> Trm cells inhibit SIPR1 by expressing unique molecules such as CD69 to prevent cell egression. Integrin αE binds E-cadherin on epithelial cells to tether Trm cells in the intestinal tissue. Similarly, resident T cells in the lamina propria also express CD69 and CD49a for their retention in a transforming growth factor–β–dependent manner (<a href="#R49" class="usa-link" aria-describedby="R49"><em>49</em></a>). In this study, we showed that disruption of TDCA-P2Y10 axis down-regulates CD69 and integrin αE on intestinal ILCs, leading to their tissue egress. A recent study showed that human intestinal ILCs also exhibit resident characteristics, with low expression of <em>S1PR1</em> and <em>SELL</em> but with high expression of <em>CD69</em> (<a href="#R50" class="usa-link" aria-describedby="R50"><em>50</em></a>). Reduced TDCA has been defined in the gut of CD (<a href="#R51" class="usa-link" aria-describedby="R51"><em>51</em></a>). We validated that patients with CD lowly expressed P2Y10 and resident molecules, suggesting that the TDCA-P2Y10 pathway could be involved in the pathogenesis of CD. Of note, TDCA administration can enhance intestinal tissue residency of ILCs and promote protection against intestinal inflammation. Thus, TDCA could be used as a potential drug to treat patients with IBD.</p></section><section id="sec11"><h2 class="pmc_sec_title">MATERIALS AND METHODS</h2>
<section id="sec12"><h3 class="pmc_sec_title">Antibodies and reagents</h3>
<p>Anti-human CD3–fluorescein isothiocyanate (FITC; UCHT1, RRID: AB_314059), anti-human CD3-allophycocyanin (APC;HIT3a, RRID: AB_314047), anti-Thy1.2-APC/cyanine7 (30-H12, RRID: AB_10613280), anti-CD127-APC (SB/199, RRID: AB_1595577), anti-CD127-Brilliant Violet 711 (A7R34, RRID: AB_2564577), anti-CD127-phycoerythrin/cyanine7 (PE-cyanine7; A7R34, RRID: AB_1937266), anti-NKp46 FITC (29A1.4, RRID: AB_2149150), anti-NKp46 (29A1.4, RRID: AB_10551441), anti-CD45.1-PE-cyanine7 (A20, RRID: AB_1134170), anti-CD45.2-PE-cyanine7 (104, RRID: AB_313444), anti-CD45.2-peridinin chlorophyll protein-cyanine5.5 (PerCP-cyanine5.5; 104, RRID: AB_893352), anti–GATA3–Alexa Fluor 647 (16E10A23, RRID: AB_2563216), and anti–epithelial cell adhesion molecule (EpCAM)–Alexa Fluor 647 (G8.8, RRID: AB_1134101) were purchased from BioLegend. Anti–CD3–eFluor 450 (17A2, RRID: AB_1272193), mouse hematopoietic lineage eFluor 450 Cocktail (RRID: AB_10426799), mouse hematopoietic lineage FITC cocktail (RRID: AB_2644066), human hematopoietic lineage eFluor 450 cocktail (RRID: AB_2642588), anti–CD19–eFluor 450 (eBio1D3 (1D3), RRID: AB_2734905), anti-Thy1.2-FITC (30-H12, RRID: AB_2735033), anti-human CD90–APC–eFluor 780 (5E10, RRID: AB_2848360), anti-B220-PE (RA3-6B2, RRID: AB_465671), anti-CCR6-APC (R6H1, RRID: AB_10733388), anti-human CD127-PE (eBioRDR5, RRID: AB_10853334), anti-CD127-PE (A7R34, RRID: AB_465843), anti-CD127 (A7R34, RRID: AB_467457), anti–CD127–PerCP–eFluor 710 (SB/199, RRID: AB_2573710), anti–NK1.1–eFluor 450 (PK136, RRID: AB_2043878), anti-NK1.1-APC (PK136, RRID: AB_469479), anti-NKp46-PE-cyanine7 (29A1.4, RRID: AB_2573441), anti-NKp46-PE (29A1.4, RRID: AB_1210743), anti–Sca-1–PerCP–cyanine5.5 (D7, RRID: AB_914370), anti–Sca-1–APC (D7, RRID: AB_469486), anti-KLRG1-PE (2F1, RRID: AB_10597431), anti-KLRG1-APC (2F1, RRID: AB_469469), anti-KLRG1 (2F1, RRID: AB_469131), anti-RORγt-APC (B2D, RRID: AB_2573253), anti-RORγt (AFKJS-9, RRID: AB_1311291), anti-PLZF–Alexa Fluor 488 (Mags.21F7, RRID: AB_2574444), anti-ID2-PE (ILCID2, RRID: AB_2735055), anti–c-Kit–PE (2B8, RRID: AB_465813), anti–integrin α4β7–APC (DATK32, RRID: AB_1210577), anti–integrin α4β7–biotin (DATK32, RRID: AB_529602), anti-CD25-PE-cyanine7 (PC61.5, RRID: AB_469608), anti-Flt3-PE (A2F10, RRID: AB_465859), anti-CD45.1-FITC (A20, RRID: AB_465057), anti-CD45.1-PerCP-cyanine5.5 (A20, RRID: AB_925750), anti–CD45.1–APC–eFluor 780 (A20. RRID: AB_1582229), anti-CD45.2-PE (104, RRID: AB_465678), anti–CD45.2–APC–eFluor 780 (104, RRID: AB_1272211), anti-CD11C-FITC (N418, RRID: AB_464940), anti–CD11b–eFluor 450 (M1/70, RRID: AB_1582236), anti-F4/80-PE (BM8, RRID: AB_465923), anti-CD69-PE-cyanine7 (H1.2F3, RRID: AB_469636), anti–integrin alpha E–FITC (2E7, RRID: AB_465175), anti-Ki67-FITC (SolA15, RRID: AB_11151689), anti–IL-13–PE (eBio13A, RRID: AB_763559), anti–LYVE1–Alexa Fluor 488 (ALY7, RRID: AB_1633417), ant–LYVE1–eFluor 450 (ALY7, RRID: AB_2784723), goat anti-rabbit immunoglobulin G (IgG) (H+L)–Alexa Fluor 647 (RRID: AB_2576217), donkey anti-rabbit IgG (H+L)–Alexa Fluor 488 (RRID: AB_2535792), donkey anti-rabbit IgG (H+L)–Alexa Fluor 594 (RRID: AB_141637), goat anti-rat IgG (H+L)–Alexa Fluor 594 (RRID: AB_2535795), and goat anti-rat IgG (H+L)–Alexa Fluor 488 (RRID: AB_2535794) were purchased from eBioscience. Anti–RORγt–Brilliant Violet 421 (Q31-378, RRID: AB_2687545) and anti–IL-5–PE (TRFK-5, RRID: AB_10894193) were purchased from BD Horizon. Anti-CD69 antibody was purchased from Bioss (RRID: AB_10855614). Anti-S1PR1-PE (T4-H28, RRID: AB_10994187) was purchased from R&amp;D Systems. Purified anti-S1PR1 antibody (RRID: AB_789722) was purchased from Novus Biologicals. Anti–E-cadherin (Ag14973, RRID: AB_10697811) was purchased from Proteintech. Purified anti-P2Y10 polyclonal antibody was purchased from Signalway Antibody and Novus. Anti-ZNF414 antibody (RRID: AB_10722362) was purchased from GeneTex. Anti–β-actin, goat anti-mouse IgG (H+L)–horseradish peroxidase (HRP) and goat anti-rabbit IgG (H+L)–HRP were purchased from Sungene Biotech. Recombinant murine IL-3, recombinant murine IL-6, recombinant murine IL-7, recombinant murine IL-2, recombinant murine IL-25, recombinant murine IL-33, and recombinant murine stem cell factor (SCF) were purchased from PeproTech. Fluo-4 AM was purchased from Yeasen. Type II collagenase and type III collagenase were purchased from Worthington. 4′,6-Diamidino-2-phenylindole (DAPI), Percoll, and paraformaldehyde (PFA) were purchased from Sigma-Aldrich. TRIzol, Lipofectamine 3000 transfection reagent, and the FoxP3 Transcription Factor Staining Kit were purchased from Thermo Fisher Scientific. Hispidol, spermidine, and dihydrostreptomycin were purchased from Psaitong. <span class="font-variant-small-caps">l</span>-α-Glycerophosphorylcholine was purchased from Sigma-Aldrich. Butobarbital was purchased from Toronto Research Chemicals. Propoxyphenyl thiosildenafil and creatinine were purchased from Aladdin. Linoleoyl ethanolamide was purchased from TargetMol. Taurochenodeoxycholic acid, glutathione reduced, taurocholic acid, and fibronectin (FN) were purchased from Solarbio. TDCA was purchased from APExBIO. (S)-H-1152 was purchased from Cayman. U-73122 was purchased from MedChemExpress. DSS was purchased from MP Biomedicals. An RhoA G-LISA activation kit was purchased from Cytoskeleton. ONE-Glo Luciferase Assay System was purchased from Promega. The Annexin V Apoptosis Detection Kit was purchased from eBioscience.</p></section><section id="sec13"><h3 class="pmc_sec_title">Mice</h3>
<p>GF C57BL/6 and NSG mice were purchased from GemPharmatech Co. Ltd. (Nanjing, China). <em>P2ry10</em><sup>−/−</sup> mice were generated on the basis of the methods described before (<a href="#R6" class="usa-link" aria-describedby="R6"><em>6</em></a>). In brief, mixtures of Cas9 mRNA (100 ng/ml) and single guide RNA (sgRNA; 50 ng/ml) (table S1) were microinjected into the cytoplasm of C57BL/6 fertilized eggs, followed by transferring to the uterus of pseudo-pregnant ICR females, from which viable founder mice were obtained. The following primers were used for deleted locus detection: 5′-TGTGCTGTGGGTCCTGTGTCCAAAGT-3′ and 5′-TATTGAAACTTTTGTGTCATTTTGTC-3′.</p>
<p>For generation of mice with <em>Itgae</em> and <em>Zfp414</em> silencing, sh<em>Itgae</em> (5′-GGTCATGAACTGCAAGATT-3′) and sh<em>Zfp414</em> (5′-GCTACTTCAAGTGTGAGAA-3′) were designed on online with RNAi designer (Merck or Thermo Fisher Scientific) and cloned into pLVshRNA-Puro lentivirus vector. Human embryonic kidney (HEK) 293T cell line (American Type Culture Collection, CRL-3216) was used for lentivirus packaging. When cells were 80 to 90% confluent, 8 μg of pLVshRNA-Puro plasmid, 6 μg of packaging plasmid psPAX2, and 2 μg of envelope plasmid pMD2.G were mixed and transfected into HEK293T cells by using Lipofectamine 3000 reagent. Cell supernatants were collected on the first, second, and third days after transfection and filtered through a 0.45-μm filter. Then, supernatants containing virus particles were ultracentrifuged at 25,000 rpm for 2 hours at 4°C for viral concentration. The virus pellets were resuspended with phosphate-buffered saline (PBS) and injected intravenously into <em>P2ry10</em><sup>+/+</sup> or <em>P2ry10</em><sup>−/−</sup> mice. Virus injections were performed every other day and continued for 2 weeks.</p>
<p>Cas9-KI, PLZF<sup>GFPcre</sup>, and <em>Id2</em><sup>+/GFP</sup> mice were purchased from the Jackson Laboratory. All the mice were C57BL/6 background and 8 to 10 weeks old. We used littermates with the same age and gender for each group. Animal use and protocols were approved by the Institutional Animal Care and Use Committees at the Institute of Biophysics, Chinese Academy of Sciences.</p></section><section id="sec14"><h3 class="pmc_sec_title">Mouse treatment</h3>
<p>For antibiotic (Abx) treatment, 8-week mice were fed water containing vancomycin (0.5 g/liter), ampicillin (1 g/liter), kanamycin (1 g/liter), and metronidazole (1 g/liter) for 2 weeks. Fresh Abx was replaced every other day. For microbiota metabolite treatment, metabolites were dissolved in drinking water. Mice were treated with 200 μl of indicated concentration of metabolites once a day by gavage administration at the beginning of antibiotic treatment. Treatment continued for 2 weeks.</p></section><section id="sec15"><h3 class="pmc_sec_title">IBD sample collection and ethical approval</h3>
<p>All patients are from The Affiliated Wuxi People’s Hospital of Nanjing Medical University. The patients were treatment naive and included upon diagnostic colonoscopy. IBD colitis diagnosis was endoscopically and histologically verified. Biopsies were obtained from two areas in the ileum of patients with IBD: the endoscopically most affected area (inflamed) and the endoscopically least-affected area (noninflamed). All participants provided written informed consent to sample collection and data analysis. This study was approved by the Research Ethics Committee at The Affiliated Wuxi People’s Hospital of Nanjing Medical University.</p></section><section id="sec16"><h3 class="pmc_sec_title">Parabiosis surgery</h3>
<p>Female 8-week-old congenic CD45.1 and CD45.2 mice were surgically connected in parabiosis according to a published protocol (<a href="#R52" class="usa-link" aria-describedby="R52"><em>52</em></a>). In brief, the mice were first anesthetized with tribromoethanol (300 mg/kg). Then, the skin along a continuous line from the elbow to the knee on the side to be jointed was shaved and sterilized with iodine and alcohol. After corresponding lateral skin incisions were made from elbow to knee in each mouse, forelimbs and hindlimbs were tied together using nylon suture, and the skin incisions were closed using stainless steel wound clips. The mice were placed on a heated pad in a supine position and allowed to wake up in ambient air. After the surgery, each pair of mice was housed separately, and subcutaneous injections of 0.25% bupivacaine (2.5 mg/ml, 8 mg/kg) were given daily for 3 days for pain management. The mice remained surgically jointed for at least 2 months. T cell and B cell equilibration in the spleen and blood was confirmed by assessing presence of CD45.1/CD45.2 congenic markers. Last, the parabionts were euthanized, and tissues from CD45.1<sup>+</sup> parabionts were processed and analyzed by flow cytometry for the presence of CD45.2<sup>+</sup> ILC subsets.</p></section><section id="sec17"><h3 class="pmc_sec_title">Isolation of mononuclear cells</h3>
<p>BM cells were flushed out from femurs in staining buffer [PBS buffer containing 1% fetal bovine serum (FBS)] and filtered through 40-μm cell strainers. Spleens and mLN were physically dissociated using ground glass slides and filtered through a 40-μm strainer. Red blood cells in spleen were lysed using red lysis buffer. The SI was dissected, and the fat and PPs were removed. Next, the SI was open up lengthwise, and the lumen contents were washed out. Then, SI tissue was cut into 0.5 cm-long pieces, placed in solution I buffer [10 mM Hepes and 5 mM EDTA in Hanks’ balanced salt solution (HBSS), pH 7.2] to incubate three times for 15 min each at 37°C. The cell suspension was collected and centrifuged after three rounds of digestion. Density gradient step was done by resuspending cells in 44% Percoll and carefully laid on 67% Percoll solution and centrifuged at 600<em>g</em> and room temperature for 20 min. Intraepithelial lymphocytes (IELs) were at the 44 to 67% interface. For lamina propria lymphocyte (LPL) isolation, remaining intestinal fragments were cut into fine pieces and digested three times for 20 min each at 37°C with solution II buffer [2% FBS and collagenase II and collagenase III (0.5 mg/ml)], and, then, cell suspension was collected after three rounds of digestion. Collected IELs and LPLs were filtered through 40-μm strainers before being centrifuged and resuspended in an appropriate amount of staining buffer. For cell isolation from ileac biopsies, biopsies were cut into pieces and digested three times for 20 min each at 37°C with digestion buffer [2% FBS and collagenase II and collagenase III (0.5 mg/ml)], and, then, cell suspension was collected and filtered through 40-μm strainers. After centrifugation, the cells were resuspended in staining buffer.</p></section><section id="sec18"><h3 class="pmc_sec_title">Flow cytometry</h3>
<p>For flow cytometry, cells were isolated and stained with antibodies targeting surface antigens for 30 to 40 min on ice, followed by washing with staining buffer (PBS containing 1% FBS). For intracellular staining, cells were harvested after surface marker staining, then fixed, and permeabilized with the FoxP3 Transcription Factor Staining Kit (Thermo Fisher Scientific) according to the manufacturer’s instructions. Antibodies for intracellular staining were incubated with cells on ice for 30 to 40 min in 1× permeabilization buffer. For flow cytometry analysis, ILCs (Lin<sup>−</sup>Thy1.2<sup>+</sup>CD127<sup>+</sup>), ILC1s (Lin<sup>−</sup>CD127<sup>+</sup>CD45.2<sup>+</sup>NK1.1<sup>+</sup>NKp46<sup>+</sup>), ILC2s (Lin<sup>−</sup>CD127<sup>+</sup>Thy1.2<sup>+</sup>KLRG1<sup>+</sup>Sca-1<sup>+</sup>), ILC3s (Lin<sup>−</sup>CD127<sup>+</sup>Thy1.2<sup>+</sup>RORγt<sup>+</sup>), common lymphoid progenitor (CLP, Lin<sup>−</sup>CD127<sup>+</sup>c-Kit<sup>lo</sup>Sca-1<sup>lo</sup>Flt3<sup>+</sup>α4β7<sup>−</sup>), common ILC progenitor (CILP, Lin<sup>−</sup>CD25<sup>−</sup>CD127<sup>+</sup>Flt3<sup>−</sup>α4β7<sup>+</sup>), common helper-like ILC progenitor (CHILP, Lin<sup>−</sup>CD25<sup>−</sup>CD127<sup>+</sup>Flt3<sup>−</sup>α4β7<sup>+</sup>Id2<sup>+</sup>), and ILC progenitor (ILCP, Lin<sup>−</sup>CD127<sup>+</sup>Flt3<sup>−</sup>α4β7<sup>+</sup>c-kit<sup>+</sup>PLZF<sup>+</sup>) were analyzed with a FACSAria III instrument (BD Biosciences). Results were analyzed by FlowJo 10.</p></section><section id="sec19"><h3 class="pmc_sec_title">Quantitative real-time PCR</h3>
<p>For analysis of sorted cell populations, total RNA was extracted through the RNA Miniprep Kit (Tiangen, Beijing, China) following the manufacturer’s instructions. For analysis of SI tissues and IBD samples, tissues or samples were homogenized directly into TRIzol (QIAGEN), and RNA was extracted via chloroform extraction. The quantity and quality of RNA were determined by using a NanoDrop (Thermo Fisher Scientific). For both methods, cDNA was synthesized with 5× All-In-one RT MasterMix (Abm, Vancouver, Canada) and analyzed on QuantStudio1 qPCR system using specific primer pairs listed in table S2. Relative expression was calculated and normalized to endogenous <em>Actb</em>.</p></section><section id="sec20"><h3 class="pmc_sec_title">Immunofluorescence staining</h3>
<p>SIs and colons were cut open longitudinally and rinsed with precold 1× PBS. The tissues were then rolled into a “Swiss roll” shape and fixed in 4% PFA for 1 hour at room temperature. After fixation, the samples were dehydrated in a 30% sucrose solution for 24 hours and then embedded in optimal cutting temperature (OCT) freezing medium for sectioning. For immunofluorescence staining, the tissue sections were briefly rehydrated in PBS and then permeabilized with 1% Triton X-100 solution (PBS diluted) for 1 hour, followed with blocking in 10% donkey serum (diluted in PBS) for 30 min. The blocked sections were incubated with prediluted primary antibodies overnight at 4°C on a shaker. After washing with PBST (0.05% Tween 20 in PBS) three times, 5 min each time, sections were incubated with secondary antibodies for 1 hour at room temperature and then washed with PBST. Last, stained sections were mounted with Fluormount G (SouthernBiotech) and examined on a Nikon laser scanning confocal microscope. Images were analyzed by Imaris 9 software.</p></section><section id="sec21"><h3 class="pmc_sec_title">ILC2s’ in vitro stimulation</h3>
<p>Sort-purified small intestinal ILC2s were incubated in complete RPMI 1640 medium [containing 10% FBS, 50 μM 2-mercaptoethanol, 1 mM <span class="font-variant-small-caps">l</span>-glutamine, 1 mM sodium pyruvate, penicillin (100 U/ml), and streptomycin (100 μg/ml)] for 24 hours at 37°C and 5% CO<sub>2</sub> with IL-2 (10 ng/ml) and IL-7 (10 ng/ml). IL-25 (50 ng/ml) and IL-33 (50 ng/ml) were added if indicated. For intracellular cytokine staining, the culture was supplemented with brefeldin A (10 μg/ml) for 4 hours.</p></section><section id="sec22"><h3 class="pmc_sec_title">In vivo antibody labeling</h3>
<p>PE-conjugated anti-CD45.2 antibody (2 μg) was intravenously injected (intravenously) into mice, which had been pretreated with antibiotics for 2 weeks. Ten minutes later, mice were euthanized, and SI was harvested to isolate mononuclear cells for further flow cytometry analysis.</p></section><section id="sec23"><h3 class="pmc_sec_title">Intestinal homing of ILCs</h3>
<p>To analyze short-term homing of ILCs, intestinal lymphocytes were isolated from CD45.2<sup>+</sup> WT mice as described above, and, then, Lin<sup>−</sup> lymphocytes were sorted. Sorted CD45.2<sup>+</sup> Lin<sup>−</sup> lymphocytes (1 × 10<sup>7</sup>) were transplanted into lethally irradiated (2 × 5 gray with a 4-hour interval) recipient mice (CD45.1<sup>+</sup>) pretreated with or without antibiotics for 2 weeks. The recipients were euthanized 18 hours later, and SI was harvested and examined by flow cytometry for frequencies of transferred ILCs.</p></section><section id="sec24"><h3 class="pmc_sec_title">3D fluorescence imaging</h3>
<p>3D fluorescence imaging was performed by using a slightly modified protocol previously published (<a href="#R53" class="usa-link" aria-describedby="R53"><em>53</em></a>). In brief, mice were perfused with cold PBS, followed by 4% PFA. Then, strips of SI were pinned onto agarose plates and fixed overnight with 4% PFA in PBS. After several rinses in PBS, samples were passed through a glycerol and sucrose gradient, permeabilized, and blocked in permeabilizing solution [0.1% Tween 20, 0.5% Triton X-100, 0.1% deoxycholate, 0.1% NP-40, and 10% bovine serum albumin (BSA) in PBS] at 4°C for 1 day with gentle shaking. An appropriate primary antibody mixture in blocking buffer (0.5 Triton X-100 and 3% BSA in PBS) was performed on samples and incubated for 1 day on a shaker at 4°C. After washing in washing buffer (1% Triton X-100 in PBS) for 5 hours with a change of buffer every 30 min on a shaker at 4°C, secondary antibodies were diluted in blocking buffer and performed on samples for 1 day on a shaker at 4°C, and, then, samples were washed in washing buffer as mentioned above. For confocal imaging, sample was properly sliced to 1-mm fragments with blades and placed on glass slides preadded with mounting medium Fluormount G (SouthernBiotech) and then covered by a coverslip. Samples were imaged by using confocal scanning (Nikon), and <em>z</em>-stacks projections were compiled by using Imaris 9 software.</p></section><section id="sec25"><h3 class="pmc_sec_title">Ca<sup>2+</sup> mobilization assay</h3>
<p><em>P2ry10</em> sequence was cloned into pCDNA4 plasmid for overexpression. Gα12, Gα13, Gα15, and Gα16 sequences were cloned into p3 × FLAG-CMV plasmid. Promiscuous expression vectors were cotransfected into HEK293T cells using Lipofectamine 3000 (Invitrogen) according to the manufacturer’s instructions. The transfected cells were cultured for 24 hours at 37°C and then were collected and seeded in fibronectin (Sigma-Aldrich)–coated 96-well plates. After continuing to culture for 16 hours, cells were washed with washing buffer [1× HBSS with Ca<sup>2+</sup>/Mg<sup>2+</sup>, 10 mM Hepes, and 0.1% BSA (pH 7.4)]. Then, cells were loaded with 2 μM Ca<sup>2+</sup> indicator (Fluo-4 AM) diluted with washing buffer supplemented with 0.05% pluronic-F127 (Sigma-Aldrich) and 1 mM probenecid (MCE) and then cultured in 37°C incubator for 30 min. Later, loading buffer with indicator was replaced with washing buffer to incubate for another 30 min, and, then, cells were washed again and loaded with washing buffer. Intracellular Ca<sup>2+</sup> flux induced by stimulation of different metabolites was analyzed by Nikon confocal through collecting fluorescence images at one frame per second after the metabolite was added into cells. Collection process was lasted for 180 s, and, then, the fluorescence intensity of every image was analyzed by NIS Elements AR software, and fluorescence/background (<em>F</em>/<em>F</em><sub>0</sub>) was quantified over time as an index of P2Y10 receptor activation.</p></section><section id="sec26"><h3 class="pmc_sec_title">Analysis of activated RhoA signaling</h3>
<p>Active RhoA (guanosine 5′-triphosphate bound) was determined by RhoA G-LISA activation kits (Cytoskeleton). HEK293T cells overexpressing P2Y10 and combined G proteins were obtained as above and then stimulated by indicated metabolite. Basal levels and metabolite-induced changes in activation were determined.</p>
<p>For luciferase reporter assay to determine the activation of RhoA signaling, HEK293T cells overexpressing P2Y10 and combined G proteins were seeded into six-well plates. Subsequently, 2 μg of pGL4.34[luc2P/SRF-RE/Hygro] plasmid (ZHUANGMO Bioscience, ZK39238) was transfected into the cells, which were then collected after 6 hours and seeded into 96-well plates. Following a 12-hour incubation, the culture medium was replaced with serum-free Dulbecco’s modified Eagle’s medium (DMEM) (100 μl per well) and continued to incubate for 8 hours. To stimulate the cells, the metabolites with indicated concentrations were diluted in DMEM containing 10% FBS and added to each well after removing 50 μl of the original culture medium. The cells were then incubated at 37°C for 6 hours. A control group without stimulation was set up. The relative light units (RLU) of luminescence signals were detected with the ONE-Glo Luciferase Assay System (Promega, E6110) according to the manufacturer’s protocol. Fold induction = RLU<sub>induced</sub>/RLU<sub>noninduced</sub>.</p></section><section id="sec27"><h3 class="pmc_sec_title">Analysis of activated RhoA signaling in ILCs</h3>
<p>To analyze the active RhoA levels in intestinal ILCs by flow cytometry, LPLs were isolated and stained with specific antibodies against lineage, Thy1.2, and CD127. After washing, the cells were stimulated with 10 μM TDCA for 5 min. Subsequently, the cells were fixed with 4% PFA and permeabilized with 0.1% Triton X-100 for 30 min on ice. The cells were then incubated with glutathione <em>S</em>-transferase (GST)–tagged Rho binding domain (RBD) of rhotekin for 3 hours at room temperature, followed by washing with TBST and incubation with rabbit anti-GST antibodies (no. ab111947, Abcam; 1:200) for 2 hours at room temperature. After washing, the cells were incubated with Alexa Fluor 488–labeled goat anti-rabbit secondary antibodies for 1 hour at room temperature. The flow cytometry analysis was performed to determine the active RhoA levels in ILCs.</p>
<p>To analyze the intracellular distribution of active RhoA using immunofluorescence, total ILCs were sorted from the SI of <em>P2ry10</em><sup>+/+</sup> and <em>P2ry10</em><sup>−/−</sup> mice. The sorted cells were seeded on chambered coverslips coated with 10% poly-<span class="font-variant-small-caps">l</span>-lysine solution. The cells were then stimulated with 10 μM TDCA for 5 min. After fixation with 4% PFA and permeabilization with 0.1% Triton X-100 for 10 min at 4°C, the cells were blocked with 10% donkey serum in PBS at room temperature. Subsequently, the cells were incubated with GST-tagged RBD of rhotekin for 24 hours at 4°C, then washed, and incubated with rabbit anti-GST antibodies overnight at 4°C. After washing, the cells were incubated with Alexa Fluor 488–labeled goat anti-rabbit secondary antibodies for 1 hour at room temperature. Last, the cells were stained with DAPI and imaged by using a Nikon confocal microscope.</p></section><section id="sec28"><h3 class="pmc_sec_title">DSS-induced inflammation model</h3>
<p><em>P2ry10</em><sup>+/+</sup> and <em>P2ry10</em><sup>−/−</sup> mice were treated with 2% (w/v) DSS (MP Biomedicals) in the drinking water for 5 days and followed by regular drinking water for 5 days. For TDCA-treated mice, mice were treated with TDCA (1 mg/20 g of body weight, diluted in drinking water) via orally gavage daily for 2 weeks while maintained on Abx drinking for 2 weeks, and, then, mice with different treatment were treated with 2% (w/v) DSS as above. Body weight was measured every day and mice were euthanized at day 10. Colons were collected for pathological analysis.</p></section><section id="sec29"><h3 class="pmc_sec_title">BM transductions and transplantation</h3>
<p>For <em>Zfp414</em> knockout in BM cells, sg<em>Zfp414s</em> (table S1) were cloned into the adeno-associated virus (AAV) vector with Cre recombinase. sgRNA sequences were selected by using Benchling’s CRISPR Guide tool. HEK293T cells were used for adenovirus packaging. AAV plasmid (8 8 μg), packaging plasmid pHelper (4 μg), and pAnc80L65 (4 μg) were mixed and transfected into HEK293T cells by using Lipofectamine 3000 reagent. On the second day after transfection, intracellular AAV was collected by freeze thawing with liquid nitrogen and then concentrated via ultracentrifuged at 25,000 rpm for 2 hours at 4°C. For <em>Zfp414</em> overexpression, the CDS sequence of <em>Zfp414</em> was cloned into PLVX-IERS-Puro lentivirus vector. Concentrated lentivirus was obtained as above. For BM transduction, Rosa26-cas9-P2A-EGFP mice were injected intravenously with 3 mg of 5-fluorouracil (Sigma-Aldrich). After 4 to 5 days, BM was collected and cultured in complete RPMI 1640 medium supplemented with IL-3 (20 ng/ml), IL-6 (50 ng/ml), and SCF (100 ng/ml). BM cells were spin infected (1000<em>g</em>, 2 hours, 32°C) at days 1 and 2 to introduce viral vectors. After the second infection, the infected BM cells (5 × 10<sup>6</sup>) were 1:1 mixed with BM cells from CD45.1<sup>+</sup> mice. The mixed cells were then transplanted into lethally irradiated CD45.1<sup>+</sup> recipients. Two months later, flow cytometry analysis was performed.</p></section><section id="sec30"><h3 class="pmc_sec_title">In vitro differentiation assay and ILCs labeling</h3>
<p>Intestinal ILC in vitro differentiation assay was performed as previously described (<a href="#R54" class="usa-link" aria-describedby="R54"><em>54</em></a>). Briefly, cells from femurs were flushed out by using PBS containing 1% FBS and filtered through 40-μm strainers. Collected cells were treated with red blood cell lysis buffer (Tiangen, Beijing) to exclude red cells. CHILPs (Lin<sup>−</sup>CD25<sup>−</sup>CD127<sup>+</sup>Flt3<sup>−</sup>α4β7<sup>+</sup>Id2<sup>GFP</sup>) and ILCPs (Lin<sup>−</sup>CD127<sup>+</sup>Flt3<sup>−</sup>α4β7<sup>+</sup>c-Kit<sup>+</sup>PLZF<sup>GFP</sup>) were sorted by FACSAria III instrument and transduced by lentivirus containing sh<em>Itgae</em> through spin infection (1000<em>g</em>, 2 hours, 32°C) at days 1 and 2. OP9-DL1 cells were maintained in complete minimum essential medium alpha (αMEM) medium (supplemented with 10% FBS, 1% streptomycin, and 1% penicillin). Before seeding of progenitor cells, OP9-DL1 cells were treated with mitomycin (4 μg/ml) for 2 hours to inhibit cell division. Then, cells were digested and seeded at the density of 1 × 10<sup>6</sup> cells per well in a 24-well plate. After OP9-DL1 cells were adhered, and indicated progenitor cells were inoculated on OP9- DL1 feeder cells in a complete RPMI 1640 medium [supplemented with 10% FBS, 1% streptomycin, 1% penicillin, IL-7 (20 ng/ml), and SCF (20 ng/ml)]. After 14 days, differentiated ILCs were harvested and labeled with carboxyfluorescein diacetate succinimidyl ester (CFSE; 5 μM; Thermo Fisher Scientific, <a href="https://www.ncbi.nlm.nih.gov/nuccore/C34570" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">C34570</a>) for 15 min at 37°C, and, then, CFSE was quenched with PBS for 3 min. Labeling ILCs (5 × 10<sup>6</sup>) were resuspended with complete RPMI 1640 medium and injected intravenously into immunodeficient NSG mice. The recipients were euthanized 18 hours later for further immunofluorescence.</p></section><section id="sec31"><h3 class="pmc_sec_title">
<em>C. rodentium</em> infection</h3>
<p><em>C. rodentium</em> was a gift from B. Ge (Shanghai Institutes for Biological Sciences, Chinese Academy of Sciences). Mice were fasted for 5 hours and then orally infected with 5 × 10<sup>9</sup> colony-forming units (CFU) of <em>C. rodentium</em>. Mice were weighed daily; on day 10 after infection, mice were euthanized, and colons were collected for pathological analysis. Feces were collected and homogenized for bacterial load measurement.</p></section><section id="sec32"><h3 class="pmc_sec_title">Histopathological analysis</h3>
<p>Colons from indicated mice were cut open longitudinally and rinsed with PBS before fixation in 4% PFA followed by paraffin sectioning and hematoxylin and eosin (H&amp;E) staining. The histopathological score was evaluated and analyzed according to a previous study (<a href="#R55" class="usa-link" aria-describedby="R55"><em>55</em></a>). Briefly, the sections were evaluated for leukocyte infiltration (0 for absent, 1 for mild, 2 for moderate, and 3 for severe infiltration), epithelial injury (0 for normal villus structure, 1 for mild epithelium loss, 2 for moderate epithelium loss, and 3 for severe epithelium loss), submucosal edema (0 for no edema, 1 for &lt;50% submucosal area and &lt;100 mm wide, 2 for 50 to 80% submucosal area and &lt;200 mm wide, and 3 for &gt;80% submucosal area and &gt; 200 mm wide), and the number of goblet cells (0 for &gt;40 goblet cells per visual field, 1 for 26 to 40 goblet cells per visual field, 2 for 16 to 25 goblet cells per visual field, and 3 for 0 to 15 goblet cells per visual field).</p></section><section id="sec33"><h3 class="pmc_sec_title">Inhibition of RhoA and Ca<sup>2+</sup> signaling cascade</h3>
<p>Sorted intestinal ILCs were rested for 2 hours in RPMI 1640 medium at 37°C. Then, cells were incubated with 1 μM Ca<sup>2+</sup> inhibitor U-73122 (HY-13419-5, MCE), 1 μM ROCK inhibitor (S)-H-1152, 1 μM (10007653, Cayman), or vehicle (dimethyl sulfoxide) during overnight stimulation with TDCA at 37°C. Then, the mRNA expression of <em>Cd69</em>, <em>S1pr1</em>, and <em>Itgae</em> was analyzed by qPCR.</p></section><section id="sec34"><h3 class="pmc_sec_title">ChIP assay</h3>
<p>Intestinal ILCs were isolated and crosslinked with 1% formaldehyde at 37°C for 10 min. After two PBS washes, cells were lysed with SDS-containing buffer and subjected to sonication to generate DNA fragments between 200 and 500 base pairs. Lysates underwent preclearance with Protein A/G magnetic beads before overnight incubation at 4°C with 2 μg of ZFP414 antibodies. Following immunoprecipitation and sequential washes, bound DNA complexes were eluted and purified for subsequent analysis. Quantitative detection was performed using qPCR with primer pairs detailed in table S3.</p></section><section id="sec35"><h3 class="pmc_sec_title">Transcriptome and bioinformatic analysis</h3>
<p>ILCs (Lin<sup>−</sup>Thy1.2<sup>+</sup>CD127<sup>+</sup>; 1 × 10<sup>6</sup>) from Ctrl and Abx-treated or <em>P2ry10</em><sup>+/+</sup> and <em>P2ry10</em><sup>−/−</sup> mice were sorted by using flow cytometry, and cells were lysed with TRIzol reagent, and, then, total RNA was extracted and subjected to RNA-seq by BGI Tech Company. R packages of “clusterProfiler” in Bioconductor were used for generating GO analysis. For bioinformatic analysis of RNA-seq of Ctrl and Abx-treated ILC1, ILC2s, and ILC3s, the scRNA-seq data were from <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE85154" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE85154</a> (<a href="#R25" class="usa-link" aria-describedby="R25"><em>25</em></a>).</p></section><section id="sec36"><h3 class="pmc_sec_title">Liquid chromatography–mass spectrometry</h3>
<p>For screening metabolites affected by microbiota deficiency, the contents of intestinal lumen were obtained and homogenized. Then, 400 μl of precooled methanol/acetonitrile/water solution (4:4:2, v/v/v) was added to 1 ml of sample. The mixture was vortexed, incubated at −20°C for 60 min, and then centrifuged at 14,000<em>g</em> for 20 min at 4°C. The supernatant was collected and dried in vacuum. For mass spectrometry (MS) analysis, 100 μl of acetonitrile-water solution (acetonitrile:water = 1:1, v/v) was added to reconstitute the samples. The mixture was vortexed and centrifuged at 14,000<em>g</em> for 15 min at 4°C, and 2 μl of the supernatant was taken for analysis. The chromatographic separation was performed by using an ACQUITY UPLC BEH C18 column (100 mm by 2.1 mm, 1.7 μm, Waters, USA) at a column temperature of 40°C and a flow rate of 0.3 ml/min. The mobile phase consisted of water with 0.1% formic acid as solvent A and acetonitrile as solvent B. The following gradient was used for metabolite elution: 0 to 0.5 min, 5% B; 0.5 to 1.0 min, 5% B; 1.0 to 9.0 min, 5 to 100% B; 9.0 to 12.0 min, 100% B; 12.0 to 15.0 min, 5% B. The injection volume for each sample was 5 μl. After separation by ultrahigh performance liquid chromatography (UHPLC), the samples were subjected to MS analysis using a Q-Exactive quadrupole-Orbitrap high-resolution mass spectrometer (Thermo Fisher Scientific). The analysis was operated in positive and negative electrospray ionization (ESI) mode. ESI settings were as follows: Ion Source Gas1, 60; Ion Source Gas2, 60; Curtain gas, 30; source temperature, 320°C; and IonSapary Voltage Floating, ±3500 V for both positive and negative ion modes. The MS scan mass/charge ratio range was set from 80 to 1200 Da, with a product ion scan resolution of 17,500. The MS scan accumulation time was 0.20 s per spectra, and the product ion scan accumulation time was 0.05 s per spectra. The second-stage MS was performed by using information-dependent acquisition in high-sensitivity mode. The following settings were used: exclude isotopes within 4 Da; candidate ions to monitor per cycle, 6. The declustering potential was set at ±60 V for both positive and negative ion modes, and the collision energy was set at 35 ± 15 eV. Data from liquid chromatography–MS/MS was analyzed with Compound Discoverer 3.0 (Thermo Fisher Scientific) and searched in BioCyc, HMDB, METLIN, HFMDB, and LIPID MAPS databases.</p></section><section id="sec37"><h3 class="pmc_sec_title">Statistics and reproducibility</h3>
<p>The numbers of mice and experiments, along with the statistical tests, are reported in each figure legend. For statistical evaluation, data were analyzed by GraphPad Prism 8.0. Two-tailed unpaired Student’s <em>t</em> test was used in this study. Results are shown as means ±SD. <em>P</em> values of ≤ 0.05 were considered significant (*<em>P</em> &lt; 0.05; **<em>P</em> &lt; 0.01; ***<em>P</em> &lt; 0.001), and <em>P</em> &gt; 0.05 was considered nonsignificant (n.s.). All flow cytometry data were analyzed by FlowJo 10.</p></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgments</h2>
<p>We thank Y. Xu, Z. Wang, and J. Jia for technical support and X. Gao, Z. Zhao, H. Liang, M. Hao, and Z. Liu for animal procedures. We also thank J. Li (Cnkingbio Company Ltd., Beijing, China) for technical support.</p>
<section id="sec38"><p><strong>Funding:</strong> Z.F. is supported by the National Key R&amp;D Program of China (2020YFA0803501, 2022YFF0710700, O2KY760402, and 2019YFA0508501), National Natural Science Foundation of China (82130088, 31930036, and 81921003), Strategic Priority Research Programs of the Chinese Academy of Sciences (XDB0570301), and Natural Science Foundation of Beijing (Z231100007223013). Z.X. is supported by the National Natural Science Foundation of China (82271785), Natural Science Foundation of Beijing (5222023), and Postdoctoral Grant (2020 M680712). Y.X. is supported by the Postdoctoral Grant (2022 T150685).</p></section><section id="sec39"><p><strong>Author contributions:</strong> Conceptualization: Y.X., Z.X., and Z.F. Methodology: Y.X., Z.X., Y.C., Y.D., and Z.F. Software: Y.X., H.G., and Y.D. Validation: Y.X., Y.C., X.Z., P.Z., H.F, and Z.F. Formal analysis: Y.X., C.L., Y.C., P.Z., H.F., R.W., and Y.T. Investigation: Y.X., P.Z., C.L., H.F., and Y.T. Resources: H.G., Z.F., X.Z., and D.F. Data curation: Y.X. and Z.F. Writing—original draft: Y.X., Z.X., and Z.F. Writing—review and editing: Y.X., Z.X., Y.C., and Z.F. Visualization: Y.X. and Z.F. Supervision: Y.C. and Z.F. Project administration: Z.F. Funding acquisition: Z.X. and Z.F.</p></section><section id="sec40"><p><strong>Competing interests:</strong> The authors declare that they have no competing interests.</p></section><section id="sec41"><p><strong>Data and materials availability:</strong> All data needed to evaluate the conclusions in the paper are present in the paper and/or the Supplementary Materials.</p></section></section><section id="sec42"><h2 class="pmc_sec_title">Supplementary Materials</h2>
<section id="sec43"><h3 class="pmc_sec_title">This PDF file includes:</h3>
<section class="sm xbox font-sm" id="supplementary-material1"><div class="caption p">
<p>Figs. S1 to S10</p>
<p>Tables S1 to S3</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12372891/bin/sciadv.adt9645_sm.pdf" data-ga-action="click_feat_suppl" class="usa-link">sciadv.adt9645_sm.pdf</a><sup> (3.8MB, pdf) </sup>
</div></div></section></section></section><section id="ref-list1" class="ref-list"><h2 class="pmc_sec_title">REFERENCES AND NOTES</h2>
<section id="ref-list1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="R1">
<span class="label">1.</span><cite>Vivier E., Artis D., Colonna M., Diefenbach A., Di Santo J. P., Eberl G., Koyasu S., Locksley R. M., McKenzie A. N. J., Mebius R. E., Powrie F., Spits H., 
Innate lymphoid cells: 10 Years on. Cell
174, 
1054–1066 (2018).
</cite> [<a href="https://doi.org/10.1016/j.cell.2018.07.017" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30142344/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Innate%20lymphoid%20cells:%2010%20Years%20on&amp;volume=174&amp;publication_year=2018&amp;pages=1054-1066&amp;pmid=30142344&amp;doi=10.1016/j.cell.2018.07.017&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R2">
<span class="label">2.</span><cite>Wang S., Xia P., Chen Y., Qu Y., Xiong Z., Ye B., Du Y., Tian Y., Yin Z., Xu Z., Fan Z., 
Regulatory innate lymphoid cells control innate intestinal inflammation. Cell
171, 
201–216.e18 (2017).
</cite> [<a href="https://doi.org/10.1016/j.cell.2017.07.027" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28844693/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Regulatory%20innate%20lymphoid%20cells%20control%20innate%20intestinal%20inflammation&amp;volume=171&amp;publication_year=2017&amp;pages=201-216.e18&amp;pmid=28844693&amp;doi=10.1016/j.cell.2017.07.027&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R3">
<span class="label">3.</span><cite>Jacquelot N., Seillet C., Vivier E., Belz G. T., 
Innate lymphoid cells and cancer. Nat. Immunol.
23, 
371–379 (2022).
</cite> [<a href="https://doi.org/10.1038/s41590-022-01127-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35228695/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Immunol.&amp;title=Innate%20lymphoid%20cells%20and%20cancer&amp;volume=23&amp;publication_year=2022&amp;pages=371-379&amp;pmid=35228695&amp;doi=10.1038/s41590-022-01127-z&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R4">
<span class="label">4.</span><cite>Gasteiger G., Fan X., Dikiy S., Lee S. Y., Rudensky A. Y., 
Tissue residency of innate lymphoid cells in lymphoid and nonlymphoid organs. Science
350, 
981–985 (2015).
</cite> [<a href="https://doi.org/10.1126/science.aac9593" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4720139/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26472762/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Tissue%20residency%20of%20innate%20lymphoid%20cells%20in%20lymphoid%20and%20nonlymphoid%20organs&amp;volume=350&amp;publication_year=2015&amp;pages=981-985&amp;pmid=26472762&amp;doi=10.1126/science.aac9593&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R5">
<span class="label">5.</span><cite>Gray J. I., Farber D. L., 
Tissue-resident immune cells in humans. Annu. Rev. Immunol.
40, 
195–220 (2022).
</cite> [<a href="https://doi.org/10.1146/annurev-immunol-093019-112809" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11194112/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35044795/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Annu.%20Rev.%20Immunol.&amp;title=Tissue-resident%20immune%20cells%20in%20humans&amp;volume=40&amp;publication_year=2022&amp;pages=195-220&amp;pmid=35044795&amp;doi=10.1146/annurev-immunol-093019-112809&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R6">
<span class="label">6.</span><cite>Xiong Z., Zhu X., Geng J., Xu Y., Wu R., Li C., Fan D., Qin X., Du Y., Tian Y., Fan Z., 
Intestinal Tuft-2 cells exert antimicrobial immunity via sensing bacterial metabolite N-undecanoylglycine. Immunity
55, 
686–700.e7 (2022).
</cite> [<a href="https://doi.org/10.1016/j.immuni.2022.03.001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35320705/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Immunity&amp;title=Intestinal%20Tuft-2%20cells%20exert%20antimicrobial%20immunity%20via%20sensing%20bacterial%20metabolite%20N-undecanoylglycine&amp;volume=55&amp;publication_year=2022&amp;pages=686-700.e7&amp;pmid=35320705&amp;doi=10.1016/j.immuni.2022.03.001&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R7">
<span class="label">7.</span><cite>Zhou L., Lin Q., Sonnenberg G. F., 
Metabolic control of innate lymphoid cells in health and disease. Nat. Metab.
4, 
1650–1659 (2022).
</cite> [<a href="https://doi.org/10.1038/s42255-022-00685-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9789197/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36424470/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Metab.&amp;title=Metabolic%20control%20of%20innate%20lymphoid%20cells%20in%20health%20and%20disease&amp;volume=4&amp;publication_year=2022&amp;pages=1650-1659&amp;pmid=36424470&amp;doi=10.1038/s42255-022-00685-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R8">
<span class="label">8.</span><cite>Qiu J., Heller J. J., Guo X., Chen Z.-m. E., Fish K., Fu Y.-X., Zhou L., 
The aryl hydrocarbon receptor regulates gut immunity through modulation of innate lymphoid cells. Immunity
36, 
92–104 (2012).
</cite> [<a href="https://doi.org/10.1016/j.immuni.2011.11.011" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3268875/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22177117/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Immunity&amp;title=The%20aryl%20hydrocarbon%20receptor%20regulates%20gut%20immunity%20through%20modulation%20of%20innate%20lymphoid%20cells&amp;volume=36&amp;publication_year=2012&amp;pages=92-104&amp;pmid=22177117&amp;doi=10.1016/j.immuni.2011.11.011&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R9">
<span class="label">9.</span><cite>Kim C. H., 
Control of lymphocyte functions by gut microbiota-derived short-chain fatty acids. Cell. Mol. Immunol.
18, 
1161–1171 (2021).
</cite> [<a href="https://doi.org/10.1038/s41423-020-00625-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8093302/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33850311/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell.%20Mol.%20Immunol.&amp;title=Control%20of%20lymphocyte%20functions%20by%20gut%20microbiota-derived%20short-chain%20fatty%20acids&amp;volume=18&amp;publication_year=2021&amp;pages=1161-1171&amp;pmid=33850311&amp;doi=10.1038/s41423-020-00625-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R10">
<span class="label">10.</span><cite>Yang W., Yu T., Huang X., Bilotta A. J., Xu L., Lu Y., Sun J., Pan F., Zhou J., Zhang W., Yao S., Maynard C. L., Singh N., Dann S. M., Liu Z., Cong Y., 
Intestinal microbiota-derived short-chain fatty acids regulation of immune cell IL-22 production and gut immunity. Nat. Commun.
11, 
4457 (2020).
</cite> [<a href="https://doi.org/10.1038/s41467-020-18262-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7478978/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32901017/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Commun.&amp;title=Intestinal%20microbiota-derived%20short-chain%20fatty%20acids%20regulation%20of%20immune%20cell%20IL-22%20production%20and%20gut%20immunity&amp;volume=11&amp;publication_year=2020&amp;pages=4457&amp;pmid=32901017&amp;doi=10.1038/s41467-020-18262-6&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R11">
<span class="label">11.</span><cite>Arifuzzaman M., Won T. H., Li T. T., Yano H., Digumarthi S., Heras A. F., Zhang W., Parkhurst C. N., Kashyap S., Jin W. B., Putzel G. G., Tsou A. M., Chu C., Wei Q., Grier A., Worgall S., Guo C. J., Schroeder F. C., Artis D., 
Inulin fibre promotes microbiota-derived bile acids and type 2 inflammation. Nature
611, 
578–584 (2022).
</cite> [<a href="https://doi.org/10.1038/s41586-022-05380-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10576985/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36323778/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Inulin%20fibre%20promotes%20microbiota-derived%20bile%20acids%20and%20type%202%20inflammation&amp;volume=611&amp;publication_year=2022&amp;pages=578-584&amp;pmid=36323778&amp;doi=10.1038/s41586-022-05380-y&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R12">
<span class="label">12.</span><cite>Qi X., Yun C., Sun L., Xia J., Wu Q., Wang Y., Wang L., Zhang Y., Liang X., Wang L., Gonzalez F. J., Patterson A. D., Liu H., Mu L., Zhou Z., Zhao Y., Li R., Liu P., Zhong C., Pang Y., Jiang C., Qiao J., 
Gut microbiota–bile acid–interleukin-22 axis orchestrates polycystic ovary syndrome. Nat. Med.
25, 
1225–1233 (2019).
</cite> [<a href="https://doi.org/10.1038/s41591-019-0509-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7376369/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31332392/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Med.&amp;title=Gut%20microbiota%E2%80%93bile%20acid%E2%80%93interleukin-22%20axis%20orchestrates%20polycystic%20ovary%20syndrome&amp;volume=25&amp;publication_year=2019&amp;pages=1225-1233&amp;pmid=31332392&amp;doi=10.1038/s41591-019-0509-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R13">
<span class="label">13.</span><cite>Liu D., Duan L., Rodda L. B., Lu E., Xu Y., An J., Qiu L., Liu F., Looney M. R., Yang Z., Allen C. D. C., Li Z., Marson A., Cyster J. G., 
CD97 promotes spleen dendritic cell homeostasis through the mechanosensing of red blood cells. Science
375, 
eabi5965 (2022).
</cite> [<a href="https://doi.org/10.1126/science.abi5965" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9310086/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35143305/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=CD97%20promotes%20spleen%20dendritic%20cell%20homeostasis%20through%20the%20mechanosensing%20of%20red%20blood%20cells&amp;volume=375&amp;publication_year=2022&amp;pages=eabi5965&amp;pmid=35143305&amp;doi=10.1126/science.abi5965&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R14">
<span class="label">14.</span><cite>Gurusamy M., Tischner D., Shao J., Klatt S., Zukunft S., Bonnavion R., Gunther S., Siebenbrodt K., Kestner R.-I., Kuhlmann T., Fleming I., Offermanns S., Wettschureck N., 
G-protein-coupled receptor P2Y10 facilitates chemokine-induced CD4 T cell migration through autocrine/paracrine mediators. Nat. Commun.
12, 
6798 (2021).
</cite> [<a href="https://doi.org/10.1038/s41467-021-26882-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8611058/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34815397/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Commun.&amp;title=G-protein-coupled%20receptor%20P2Y10%20facilitates%20chemokine-induced%20CD4%20T%20cell%20migration%20through%20autocrine/paracrine%20mediators&amp;volume=12&amp;publication_year=2021&amp;pages=6798&amp;pmid=34815397&amp;doi=10.1038/s41467-021-26882-9&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R15">
<span class="label">15.</span><cite>Otake-Kasamoto Y., Kayama H., Kishikawa T., Shinzaki S., Tashiro T., Amano T., Tani M., Yoshihara T., Li B., Tani H., Liu L., Hayashi A., Okuzaki D., Motooka D., Nakamura S., Okada Y., Iijima H., Takeda K., Takehara T., 
Lysophosphatidylserines derived from microbiota in Crohn’s disease elicit pathological Th1 response. J. Exp. Med.
219, 
e20211291 (2022).
</cite> [<a href="https://doi.org/10.1084/jem.20211291" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9134096/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35608941/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Exp.%20Med.&amp;title=Lysophosphatidylserines%20derived%20from%20microbiota%20in%20Crohn%E2%80%99s%20disease%20elicit%20pathological%20Th1%20response&amp;volume=219&amp;publication_year=2022&amp;pages=e20211291&amp;pmid=35608941&amp;doi=10.1084/jem.20211291&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R16">
<span class="label">16.</span><cite>Werder R. B., Ullah M. A., Rahman M. M., Simpson J., Lynch J. P., Collinson N., Rittchen S., Rashid R. B., Sikder M. A. A., Handoko H. Y., Curren B. F., Sebina I., Hartel G., Bissell A., Ngo S., Yarlagadda T., Hasnain S. Z., Lu W., Sohal S. S., Martin M., Bowler S., Burr L. D., Martinez L. O., Robaye B., Spann K., Ferreira M. A. R., Phipps S., 
Targeting the P2Y<sub>13</sub> receptor suppresses IL-33 and HMGB1 release and ameliorates experimental asthma. Am. J. Respir. Crit. Care Med.
205, 
300–312 (2022).
</cite> [<a href="https://doi.org/10.1164/rccm.202009-3686OC" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC12042653/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34860143/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Respir.%20Crit.%20Care%20Med.&amp;title=Targeting%20the%20P2Y13%20receptor%20suppresses%20IL-33%20and%20HMGB1%20release%20and%20ameliorates%20experimental%20asthma&amp;volume=205&amp;publication_year=2022&amp;pages=300-312&amp;pmid=34860143&amp;doi=10.1164/rccm.202009-3686OC&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R17">
<span class="label">17.</span><cite>Masopust D., Soerens A. G., 
Tissue-resident T cells and other resident leukocytes. Annu. Rev. Immunol.
37, 
521–546 (2019).
</cite> [<a href="https://doi.org/10.1146/annurev-immunol-042617-053214" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7175802/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30726153/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Annu.%20Rev.%20Immunol.&amp;title=Tissue-resident%20T%20cells%20and%20other%20resident%20leukocytes&amp;volume=37&amp;publication_year=2019&amp;pages=521-546&amp;pmid=30726153&amp;doi=10.1146/annurev-immunol-042617-053214&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R18">
<span class="label">18.</span><cite>Fan X., Rudensky A. Y., 
Hallmarks of tissue-resident lymphocytes. Cell
164, 
1198–1211 (2016).
</cite> [<a href="https://doi.org/10.1016/j.cell.2016.02.048" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4973889/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26967286/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Hallmarks%20of%20tissue-resident%20lymphocytes&amp;volume=164&amp;publication_year=2016&amp;pages=1198-1211&amp;pmid=26967286&amp;doi=10.1016/j.cell.2016.02.048&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R19">
<span class="label">19.</span><cite>Torcellan T., Friedrich C., Doucet-Ladevèze R., Ossner T., Solé V. V., Riedmann S., Ugur M., Imdahl F., Rosshart S. P., Arnold S. J., Gomez de Agüero M., Gagliani N., Flavell R. A., Backes S., Kastenmüller W., Gasteiger G., 
Circulating NK cells establish tissue residency upon acute infection of skin and mediate accelerated effector responses to secondary infection. Immunity
57, 
124–140.e7 (2024).
</cite> [<a href="https://doi.org/10.1016/j.immuni.2023.11.018" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10783803/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38157853/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Immunity&amp;title=Circulating%20NK%20cells%20establish%20tissue%20residency%20upon%20acute%20infection%20of%20skin%20and%20mediate%20accelerated%20effector%20responses%20to%20secondary%20infection&amp;volume=57&amp;publication_year=2024&amp;pages=124-140.e7&amp;pmid=38157853&amp;doi=10.1016/j.immuni.2023.11.018&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R20">
<span class="label">20.</span><cite>Tan H. X., Juno J. A., Esterbauer R., Kelly H. G., Wragg K. M., Konstandopoulos P., Alcantara S., Alvarado C., Jones R., Starkey G., Wang B. Z., Yoshino O., Tiang T., Grayson M. L., Opdam H., D’Costa R., Vago A., Austin Liver Transplant Perfusionist Group, Mackay L. K., Gordon C. L., Masopust D., Groom J. R., Kent S. J., Wheatley A. K., 
Lung-resident memory B cells established after pulmonary influenza infection display distinct transcriptional and phenotypic profiles. Sci. Immunol.
7, 
eabf5314 (2022).
</cite> [<a href="https://doi.org/10.1126/sciimmunol.abf5314" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35089815/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sci.%20Immunol.&amp;title=Lung-resident%20memory%20B%20cells%20established%20after%20pulmonary%20influenza%20infection%20display%20distinct%20transcriptional%20and%20phenotypic%20profiles&amp;volume=7&amp;publication_year=2022&amp;pages=eabf5314&amp;pmid=35089815&amp;doi=10.1126/sciimmunol.abf5314&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R21">
<span class="label">21.</span><cite>Kästele V., Mayer J., Lee E. S., Papazian N., Cole J. J., Cerovic V., Belz G., Tomura M., Eberl G., Goodyear C., Maciewicz R. A., Wall D., Cupedo T., Withers D. R., Milling S., 
Intestinal-derived ILCs migrating in lymph increase IFNγ production in response to <em>Salmonella</em> Typhimurium infection. Mucosal Immunol.
14, 
717–727 (2021).
</cite> [<a href="https://doi.org/10.1038/s41385-020-00366-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8075955/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33414524/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mucosal%20Immunol.&amp;title=Intestinal-derived%20ILCs%20migrating%20in%20lymph%20increase%20IFN%CE%B3%20production%20in%20response%20to%20Salmonella%20Typhimurium%20infection&amp;volume=14&amp;publication_year=2021&amp;pages=717-727&amp;pmid=33414524&amp;doi=10.1038/s41385-020-00366-3&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R22">
<span class="label">22.</span><cite>Petrova T. V., Koh G. Y., 
Biological functions of lymphatic vessels. Science
369, 
eaax4063 (2020).
</cite> [<a href="https://doi.org/10.1126/science.aax4063" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32646971/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Biological%20functions%20of%20lymphatic%20vessels&amp;volume=369&amp;publication_year=2020&amp;pages=eaax4063&amp;pmid=32646971&amp;doi=10.1126/science.aax4063&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R23">
<span class="label">23.</span><cite>Skon C. N., Lee J.-Y., Anderson K. G., Masopust D., Hogquist K. A., Jameson S. C., 
Transcriptional downregulation of <em>S1pr1</em> is required for the establishment of resident memory CD8<sup>+</sup> T cells. Nat. Immunol.
14, 
1285–1293 (2013).
</cite> [<a href="https://doi.org/10.1038/ni.2745" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3844557/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24162775/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Immunol.&amp;title=Transcriptional%20downregulation%20of%20S1pr1%20is%20required%20for%20the%20establishment%20of%20resident%20memory%20CD8+%20T%20cells&amp;volume=14&amp;publication_year=2013&amp;pages=1285-1293&amp;pmid=24162775&amp;doi=10.1038/ni.2745&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R24">
<span class="label">24.</span><cite>Chun E., Lavoie S., Fonseca-Pereira D., Bae S., Michaud M., Hoveyda H. R., Fraser G. L., Gallini Comeau C. A., Glickman J. N., Fuller M. H., Layden B. T., Garrett W. S., 
Metabolite-sensing receptor Ffar2 regulates colonic group 3 innate lymphoid cells and gut immunity. Immunity
51, 
871–884.e6 (2019).
</cite> [<a href="https://doi.org/10.1016/j.immuni.2019.09.014" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6901086/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31628054/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Immunity&amp;title=Metabolite-sensing%20receptor%20Ffar2%20regulates%20colonic%20group%203%20innate%20lymphoid%20cells%20and%20gut%20immunity&amp;volume=51&amp;publication_year=2019&amp;pages=871-884.e6&amp;pmid=31628054&amp;doi=10.1016/j.immuni.2019.09.014&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R25">
<span class="label">25.</span><cite>Gury-BenAri M., Thaiss C. A., Serafini N., Winter D. R., Giladi A., Lara-Astiaso D., Levy M., Salame T. M., Weiner A., David E., Shapiro H., Dori-Bachash M., Pevsner-Fischer M., Lorenzo-Vivas E., Keren-Shaul H., Paul F., Harmelin A., Eberl G., Itzkovitz S., Tanay A., Di Santo J. P., Elinav E., Amit I., 
The spectrum and regulatory landscape of intestinal innate lymphoid cells are shaped by the microbiome. Cell
166, 
1231–1246.e13 (2016).
</cite> [<a href="https://doi.org/10.1016/j.cell.2016.07.043" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27545347/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=The%20spectrum%20and%20regulatory%20landscape%20of%20intestinal%20innate%20lymphoid%20cells%20are%20shaped%20by%20the%20microbiome&amp;volume=166&amp;publication_year=2016&amp;pages=1231-1246.e13&amp;pmid=27545347&amp;doi=10.1016/j.cell.2016.07.043&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R26">
<span class="label">26.</span><cite>Colosimo D. A., Kohn J. A., Luo P. M., Piscotta F. J., Han S. M., Pickard A. J., Rao A., Cross J. R., Cohen L. J., Brady S. F., 
Mapping interactions of microbial metabolites with human G-protein-coupled receptors. Cell Host Microbe
26, 
273–282.e7 (2019).
</cite> [<a href="https://doi.org/10.1016/j.chom.2019.07.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6706627/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31378678/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Host%20Microbe&amp;title=Mapping%20interactions%20of%20microbial%20metabolites%20with%20human%20G-protein-coupled%20receptors&amp;volume=26&amp;publication_year=2019&amp;pages=273-282.e7&amp;pmid=31378678&amp;doi=10.1016/j.chom.2019.07.002&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R27">
<span class="label">27.</span><cite>Husted A. S., Trauelsen M., Rudenko O., Hjorth S. A., Schwartz T. W., 
GPCR-mediated signaling of metabolites. Cell Metab.
25, 
777–796 (2017).
</cite> [<a href="https://doi.org/10.1016/j.cmet.2017.03.008" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28380372/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Metab.&amp;title=GPCR-mediated%20signaling%20of%20metabolites&amp;volume=25&amp;publication_year=2017&amp;pages=777-796&amp;pmid=28380372&amp;doi=10.1016/j.cmet.2017.03.008&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R28">
<span class="label">28.</span><cite>Kiss E. A., Vonarbourg C., Kopfmann S., Hobeika E., Finke D., Esser C., Diefenbach A., 
Natural aryl hydrocarbon receptor ligands control organogenesis of intestinal lymphoid follicles. Science
334, 
1561–1565 (2011).
</cite> [<a href="https://doi.org/10.1126/science.1214914" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22033518/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Natural%20aryl%20hydrocarbon%20receptor%20ligands%20control%20organogenesis%20of%20intestinal%20lymphoid%20follicles&amp;volume=334&amp;publication_year=2011&amp;pages=1561-1565&amp;pmid=22033518&amp;doi=10.1126/science.1214914&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R29">
<span class="label">29.</span><cite>Meixiong J., Vasavda C., Snyder S. H., Dong X., 
MRGPRX4 is a G protein-coupled receptor activated by bile acids that may contribute to cholestatic pruritus. Proc. Natl. Acad. Sci. U.S.A.
116, 
10525–10530 (2019).
</cite> [<a href="https://doi.org/10.1073/pnas.1903316116" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6535009/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31068464/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20U.S.A.&amp;title=MRGPRX4%20is%20a%20G%20protein-coupled%20receptor%20activated%20by%20bile%20acids%20that%20may%20contribute%20to%20cholestatic%20pruritus&amp;volume=116&amp;publication_year=2019&amp;pages=10525-10530&amp;pmid=31068464&amp;doi=10.1073/pnas.1903316116&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R30">
<span class="label">30.</span><cite>Song W., Antao O. Q., Condiff E., Sanchez G. M., Chernova I., Zembrzuski K., Steach H., Rubtsova K., Angeletti D., Lemenze A., Laidlaw B. J., Craft J., Weinstein J. S., 
Development of Tbet- and CD11c-expressing B cells in a viral infection requires T follicular helper cells outside of germinal centers. Immunity
55, 
290–307.e5 (2022).
</cite> [<a href="https://doi.org/10.1016/j.immuni.2022.01.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8965751/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35090581/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Immunity&amp;title=Development%20of%20Tbet-%20and%20CD11c-expressing%20B%20cells%20in%20a%20viral%20infection%20requires%20T%20follicular%20helper%20cells%20outside%20of%20germinal%20centers&amp;volume=55&amp;publication_year=2022&amp;pages=290-307.e5&amp;pmid=35090581&amp;doi=10.1016/j.immuni.2022.01.002&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R31">
<span class="label">31.</span><cite>Dustin M. L., 
Integrins and their role in immune cell adhesion. Cell
177, 
499–501 (2019).
</cite> [<a href="https://doi.org/10.1016/j.cell.2019.03.038" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30952447/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Integrins%20and%20their%20role%20in%20immune%20cell%20adhesion&amp;volume=177&amp;publication_year=2019&amp;pages=499-501&amp;pmid=30952447&amp;doi=10.1016/j.cell.2019.03.038&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R32">
<span class="label">32.</span><cite>Diefenbach A., Gnafakis S., Shomrat O., 
Innate lymphoid cell-epithelial cell modules sustain intestinal homeostasis. Immunity
52, 
452–463 (2020).
</cite> [<a href="https://doi.org/10.1016/j.immuni.2020.02.016" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32187516/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Immunity&amp;title=Innate%20lymphoid%20cell-epithelial%20cell%20modules%20sustain%20intestinal%20homeostasis&amp;volume=52&amp;publication_year=2020&amp;pages=452-463&amp;pmid=32187516&amp;doi=10.1016/j.immuni.2020.02.016&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R33">
<span class="label">33.</span><cite>Zheng Y., Valdez P. A., Danilenko D. M., Hu Y., Sa S. M., Gong Q., Abbas A. R., Modrusan Z., Ghilardi N., de Sauvage F. J., Ouyang W., 
Interleukin-22 mediates early host defense against attaching and effacing bacterial pathogens. Nat. Med.
14, 
282–289 (2008).
</cite> [<a href="https://doi.org/10.1038/nm1720" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18264109/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Med.&amp;title=Interleukin-22%20mediates%20early%20host%20defense%20against%20attaching%20and%20effacing%20bacterial%20pathogens&amp;volume=14&amp;publication_year=2008&amp;pages=282-289&amp;pmid=18264109&amp;doi=10.1038/nm1720&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R34">
<span class="label">34.</span><cite>Gola A., Dorrington M. G., Speranza E., Sala C., Shih R. M., Radtke A. J., Wong H. S., Baptista A. P., Hernandez J. M., Castellani G., Fraser I. D. C., Germain R. N., 
Commensal-driven immune zonation of the liver promotes host defence. Nature
589, 
131–136 (2021).
</cite> [<a href="https://doi.org/10.1038/s41586-020-2977-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8691525/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33239787/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Commensal-driven%20immune%20zonation%20of%20the%20liver%20promotes%20host%20defence&amp;volume=589&amp;publication_year=2021&amp;pages=131-136&amp;pmid=33239787&amp;doi=10.1038/s41586-020-2977-2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R35">
<span class="label">35.</span><cite>Balmer M. L., Ma E. H., Bantug G. R., Grählert J., Pfister S., Glatter T., Jauch A., Dimeloe S., Slack E., Dehio P., Krzyzaniak M. A., King C. G., Burgener A. V., Fischer M., Develioglu L., Belle R., Recher M., Bonilla W. V., Macpherson A. J., Hapfelmeier S., Jones R. G., Hess C., 
Memory CD8<sup>+</sup> T cells require increased concentrations of acetate induced by stress for optimal function. Immunity
44, 
1312–1324 (2016).
</cite> [<a href="https://doi.org/10.1016/j.immuni.2016.03.016" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27212436/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Immunity&amp;title=Memory%20CD8+%20T%20cells%20require%20increased%20concentrations%20of%20acetate%20induced%20by%20stress%20for%20optimal%20function&amp;volume=44&amp;publication_year=2016&amp;pages=1312-1324&amp;pmid=27212436&amp;doi=10.1016/j.immuni.2016.03.016&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R36">
<span class="label">36.</span><cite>Omenetti S., Bussi C., Metidji A., Iseppon A., Lee S., Tolaini M., Li Y., Kelly G., Chakravarty P., Shoaie S., Gutierrez M. G., Stockinger B., 
The intestine harbors functionally distinct homeostatic tissue-resident and inflammatory Th17 cells. Immunity
51, 
77–89.e6 (2019).
</cite> [<a href="https://doi.org/10.1016/j.immuni.2019.05.004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6642154/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31229354/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Immunity&amp;title=The%20intestine%20harbors%20functionally%20distinct%20homeostatic%20tissue-resident%20and%20inflammatory%20Th17%20cells&amp;volume=51&amp;publication_year=2019&amp;pages=77-89.e6&amp;pmid=31229354&amp;doi=10.1016/j.immuni.2019.05.004&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R37">
<span class="label">37.</span><cite>Tian P., Yang W., Guo X., Wang T., Tan S., Sun R., Xiao R., Wang Y., Jiao D., Xu Y., Wei Y., Wu Z., Li C., Gao L., Ma C., Liang X., 
Early life gut microbiota sustains liver-resident natural killer cells maturation via the butyrate-IL-18 axis. Nat. Commun.
14, 
1710 (2023).
</cite> [<a href="https://doi.org/10.1038/s41467-023-37419-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10043027/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36973277/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Commun.&amp;title=Early%20life%20gut%20microbiota%20sustains%20liver-resident%20natural%20killer%20cells%20maturation%20via%20the%20butyrate-IL-18%20axis&amp;volume=14&amp;publication_year=2023&amp;pages=1710&amp;pmid=36973277&amp;doi=10.1038/s41467-023-37419-7&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R38">
<span class="label">38.</span><cite>Koren O., Konnikova L., Brodin P., Mysorekar I. U., Collado M. C., 
The maternal gut microbiome in pregnancy: Implications for the developing immune system. Nat. Rev. Gastroenterol. Hepatol.
21, 
35–45 (2024).
</cite> [<a href="https://doi.org/10.1038/s41575-023-00864-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38097774/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Rev.%20Gastroenterol.%20Hepatol.&amp;title=The%20maternal%20gut%20microbiome%20in%20pregnancy:%20Implications%20for%20the%20developing%20immune%20system&amp;volume=21&amp;publication_year=2024&amp;pages=35-45&amp;pmid=38097774&amp;doi=10.1038/s41575-023-00864-2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R39">
<span class="label">39.</span><cite>Horn V., Sonnenberg G. F., 
Group 3 innate lymphoid cells in intestinal health and disease. Nat. Rev. Gastroenterol. Hepatol.
21, 
428–443 (2024).
</cite> [<a href="https://doi.org/10.1038/s41575-024-00906-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11144103/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38467885/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Rev.%20Gastroenterol.%20Hepatol.&amp;title=Group%203%20innate%20lymphoid%20cells%20in%20intestinal%20health%20and%20disease&amp;volume=21&amp;publication_year=2024&amp;pages=428-443&amp;pmid=38467885&amp;doi=10.1038/s41575-024-00906-3&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R40">
<span class="label">40.</span><cite>Sudan R., Gilfillan S., Colonna M., 
Group 1 ILCs: Heterogeneity, plasticity, and transcriptional regulation. Immunol. Rev.
323, 
107–117 (2024).
</cite> [<a href="https://doi.org/10.1111/imr.13327" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11102297/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38563448/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Immunol.%20Rev.&amp;title=Group%201%20ILCs:%20Heterogeneity,%20plasticity,%20and%20transcriptional%20regulation&amp;volume=323&amp;publication_year=2024&amp;pages=107-117&amp;pmid=38563448&amp;doi=10.1111/imr.13327&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R41">
<span class="label">41.</span><cite>Hwang S. M., Kim H. J., Kim S. M., Jung Y., Park S. W., Chung I. Y., 
Lysophosphatidylserine receptor P2Y10: A G protein-coupled receptor that mediates eosinophil degranulation. Clin. Exp. Allergy
48, 
990–999 (2018).
</cite> [<a href="https://doi.org/10.1111/cea.13162" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29700886/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Clin.%20Exp.%20Allergy&amp;title=Lysophosphatidylserine%20receptor%20P2Y10:%20A%20G%20protein-coupled%20receptor%20that%20mediates%20eosinophil%20degranulation&amp;volume=48&amp;publication_year=2018&amp;pages=990-999&amp;pmid=29700886&amp;doi=10.1111/cea.13162&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R42">
<span class="label">42.</span><cite>Bai X., Wei H., Liu W., Coker O. O., Gou H., Liu C., Zhao L., Li C., Zhou Y., Wang G., Kang W., Ng E. K.-w., Yu J., 
Cigarette smoke promotes colorectal cancer through modulation of gut microbiota and related metabolites. Gut
71, 
2439–2450 (2022).
</cite> [<a href="https://doi.org/10.1136/gutjnl-2021-325021" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9664112/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35387878/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Gut&amp;title=Cigarette%20smoke%20promotes%20colorectal%20cancer%20through%20modulation%20of%20gut%20microbiota%20and%20related%20metabolites&amp;volume=71&amp;publication_year=2022&amp;pages=2439-2450&amp;pmid=35387878&amp;doi=10.1136/gutjnl-2021-325021&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R43">
<span class="label">43.</span><cite>Song X., Sun X., Oh S. F., Wu M., Zhang Y., Zheng W., Geva-Zatorsky N., Jupp R., Mathis D., Benoist C., Kasper D. L., 
Microbial bile acid metabolites modulate gut RORγ<sup>+</sup> regulatory T cell homeostasis. Nature
577, 
410–415 (2020).
</cite> [<a href="https://doi.org/10.1038/s41586-019-1865-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7274525/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31875848/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Microbial%20bile%20acid%20metabolites%20modulate%20gut%20ROR%CE%B3+%20regulatory%20T%20cell%20homeostasis&amp;volume=577&amp;publication_year=2020&amp;pages=410-415&amp;pmid=31875848&amp;doi=10.1038/s41586-019-1865-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R44">
<span class="label">44.</span><cite>Campbell C., McKenney P. T., Konstantinovsky D., Isaeva O. I., Schizas M., Verter J., Mai C., Jin W. B., Guo C. J., Violante S., Ramos R. J., Cross J. R., Kadaveru K., Hambor J., Rudensky A. Y., 
Bacterial metabolism of bile acids promotes generation of peripheral regulatory T cells. Nature
581, 
475–479 (2020).
</cite> [<a href="https://doi.org/10.1038/s41586-020-2193-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7540721/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32461639/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Bacterial%20metabolism%20of%20bile%20acids%20promotes%20generation%20of%20peripheral%20regulatory%20T%20cells&amp;volume=581&amp;publication_year=2020&amp;pages=475-479&amp;pmid=32461639&amp;doi=10.1038/s41586-020-2193-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R45">
<span class="label">45.</span><cite>Hang S., Paik D., Yao L., Kim E., Trinath J., Lu J., Ha S., Nelson B. N., Kelly S. P., Wu L., Zheng Y., Longman R. S., Rastinejad F., Devlin A. S., Krout M. R., Fischbach M. A., Littman D. R., Huh J. R., 
Bile acid metabolites control T<sub>H</sub>17 and T<sub>reg</sub> cell differentiation. Nature
576, 
143–148 (2019).
</cite> [<a href="https://doi.org/10.1038/s41586-019-1785-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6949019/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31776512/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Bile%20acid%20metabolites%20control%20TH17%20and%20Treg%20cell%20differentiation&amp;volume=576&amp;publication_year=2019&amp;pages=143-148&amp;pmid=31776512&amp;doi=10.1038/s41586-019-1785-z&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R46">
<span class="label">46.</span><cite>Feng Z., Sun R., Cong Y., Liu Z., 
Critical roles of G protein-coupled receptors in regulating intestinal homeostasis and inflammatory bowel disease. Mucosal Immunol.
15, 
819–828 (2022).
</cite> [<a href="https://doi.org/10.1038/s41385-022-00538-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35732818/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mucosal%20Immunol.&amp;title=Critical%20roles%20of%20G%20protein-coupled%20receptors%20in%20regulating%20intestinal%20homeostasis%20and%20inflammatory%20bowel%20disease&amp;volume=15&amp;publication_year=2022&amp;pages=819-828&amp;pmid=35732818&amp;doi=10.1038/s41385-022-00538-3&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R47">
<span class="label">47.</span><cite>Sakaguchi T., Takefuji M., Wettschureck N., Hamaguchi T., Amano M., Kato K., Tsuda T., Eguchi S., Ishihama S., Mori Y., Yura Y., Yoshida T., Unno K., Okumura T., Ishii H., Shimizu Y., Bando Y. K., Ohashi K., Ouchi N., Enomoto A., Offermanns S., Kaibuchi K., Murohara T., 
Protein kinase N promotes stress-induced cardiac dysfunction through phosphorylation of myocardin-related transcription factor A and disruption of its interaction with actin. Circulation
140, 
1737–1752 (2019).
</cite> [<a href="https://doi.org/10.1161/CIRCULATIONAHA.119.041019" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31564129/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Circulation&amp;title=Protein%20kinase%20N%20promotes%20stress-induced%20cardiac%20dysfunction%20through%20phosphorylation%20of%20myocardin-related%20transcription%20factor%20A%20and%20disruption%20of%20its%20interaction%20with%20actin&amp;volume=140&amp;publication_year=2019&amp;pages=1737-1752&amp;pmid=31564129&amp;doi=10.1161/CIRCULATIONAHA.119.041019&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R48">
<span class="label">48.</span><cite>Esposito D., Pant I., Shen Y., Qiao R. F., Yang X., Bai Y., Jin J., Poulikakos P. I., Aaronson S. A., 
ROCK1 mechano-signaling dependency of human malignancies driven by TEAD/YAP activation. Nat. Commun.
13, 
703 (2022).
</cite> [<a href="https://doi.org/10.1038/s41467-022-28319-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8817028/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35121738/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Commun.&amp;title=ROCK1%20mechano-signaling%20dependency%20of%20human%20malignancies%20driven%20by%20TEAD/YAP%20activation&amp;volume=13&amp;publication_year=2022&amp;pages=703&amp;pmid=35121738&amp;doi=10.1038/s41467-022-28319-3&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R49">
<span class="label">49.</span><cite>Chen W., 
TGF-β regulation of T cells. Annu. Rev. Immunol.
41, 
483–512 (2023).
</cite> [<a href="https://doi.org/10.1146/annurev-immunol-101921-045939" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36750317/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Annu.%20Rev.%20Immunol.&amp;title=TGF-%CE%B2%20regulation%20of%20T%20cells&amp;volume=41&amp;publication_year=2023&amp;pages=483-512&amp;pmid=36750317&amp;doi=10.1146/annurev-immunol-101921-045939&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R50">
<span class="label">50.</span><cite>Mazzurana L., Czarnewski P., Jonsson V., Wigge L., Ringnér M., Williams T. C., Ravindran A., Björklund Å. K., Säfholm J., Nilsson G., Dahlén S. E., Orre A. C., Al-Ameri M., Höög C., Hedin C., Szczegielniak S., Almer S., Mjösberg J., 
Tissue-specific transcriptional imprinting and heterogeneity in human innate lymphoid cells revealed by full-length single-cell RNA-sequencing. Cell Res.
31, 
554–568 (2021).
</cite> [<a href="https://doi.org/10.1038/s41422-020-00445-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8089104/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33420427/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Res.&amp;title=Tissue-specific%20transcriptional%20imprinting%20and%20heterogeneity%20in%20human%20innate%20lymphoid%20cells%20revealed%20by%20full-length%20single-cell%20RNA-sequencing&amp;volume=31&amp;publication_year=2021&amp;pages=554-568&amp;pmid=33420427&amp;doi=10.1038/s41422-020-00445-x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R51">
<span class="label">51.</span><cite>Feng L., Zhou N., Li Z., Fu D., Guo Y., Gao X., Liu X., 
Co-occurrence of gut microbiota dysbiosis and bile acid metabolism alteration is associated with psychological disorders in Crohn’s disease. FASEB J.
36, 
e22100 (2022).
</cite> [<a href="https://doi.org/10.1096/fj.202101088RRR" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34939244/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=FASEB%20J.&amp;title=Co-occurrence%20of%20gut%20microbiota%20dysbiosis%20and%20bile%20acid%20metabolism%20alteration%20is%20associated%20with%20psychological%20disorders%20in%20Crohn%E2%80%99s%20disease&amp;volume=36&amp;publication_year=2022&amp;pages=e22100&amp;pmid=34939244&amp;doi=10.1096/fj.202101088RRR&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R52">
<span class="label">52.</span><cite>Kamran P., Sereti K. I., Zhao P., Ali S. R., Weissman I. L., Ardehali R., 
Parabiosis in mice: A detailed protocol. J. Vis. Exp., 
e50556 (2013).</cite> [<a href="https://doi.org/10.3791/50556" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3938334/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24145664/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Vis.%20Exp.&amp;title=Parabiosis%20in%20mice:%20A%20detailed%20protocol&amp;publication_year=2013&amp;pages=e50556&amp;pmid=24145664&amp;doi=10.3791/50556&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R53">
<span class="label">53.</span><cite>Bernier-Latmani J., Petrova T. V., 
High-resolution 3D analysis of mouse small-intestinal stroma. Nat. Protoc.
11, 
1617–1629 (2016).
</cite> [<a href="https://doi.org/10.1038/nprot.2016.092" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27560169/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Protoc.&amp;title=High-resolution%203D%20analysis%20of%20mouse%20small-intestinal%20stroma&amp;volume=11&amp;publication_year=2016&amp;pages=1617-1629&amp;pmid=27560169&amp;doi=10.1038/nprot.2016.092&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R54">
<span class="label">54.</span><cite>Liu N., He J., Fan D., Gu Y., Wang J., Li H., Zhu X., Du Y., Tian Y., Liu B., Fan Z., 
Circular RNA <em>circTmem241</em> drives group III innate lymphoid cell differentiation via initiation of <em>Elk3</em> transcription. Nat. Commun.
13, 
4711 (2022).
</cite> [<a href="https://doi.org/10.1038/s41467-022-32322-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9372085/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35953472/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Commun.&amp;title=Circular%20RNA%20circTmem241%20drives%20group%20III%20innate%20lymphoid%20cell%20differentiation%20via%20initiation%20of%20Elk3%20transcription&amp;volume=13&amp;publication_year=2022&amp;pages=4711&amp;pmid=35953472&amp;doi=10.1038/s41467-022-32322-z&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R55">
<span class="label">55.</span><cite>Erben U., Loddenkemper C., Doerfel K., Spieckermann S., Haller D., Heimesaat M. M., Zeitz M., Siegmund B., Kühl A. A., 
A guide to histomorphological evaluation of intestinal inflammation in mouse models. Int. J. Clin. Exp. Pathol.
7, 
4557–4576 (2014).
</cite> [<a href="/articles/PMC4152019/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25197329/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Int.%20J.%20Clin.%20Exp.%20Pathol.&amp;title=A%20guide%20to%20histomorphological%20evaluation%20of%20intestinal%20inflammation%20in%20mouse%20models&amp;volume=7&amp;publication_year=2014&amp;pages=4557-4576&amp;pmid=25197329&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="caption p">
<p>Figs. S1 to S10</p>
<p>Tables S1 to S3</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12372891/bin/sciadv.adt9645_sm.pdf" data-ga-action="click_feat_suppl" class="usa-link">sciadv.adt9645_sm.pdf</a><sup> (3.8MB, pdf) </sup>
</div></div></section></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Science Advances are provided here courtesy of <strong>American Association for the Advancement of Science</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1126/sciadv.adt9645"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/sciadv.adt9645.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (6.4 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12372891/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12372891/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12372891%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12372891/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12372891/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12372891/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40845099/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12372891/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40845099/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12372891/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12372891/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="3KXTDkW22G5LS16q286a3KJr5uiAB20XRRc2rhErmjvsv8OFCZ2n4H3BDARXKxbp">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
