
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            KAT6A chimeras form a self-reinforcing epigenetic module with NURF and MLL/COMPASS to sustain AML - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="D2E4AECC8AF310A305AECC0054103208.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="genbio">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366150/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Genome Biology">
<meta name="citation_title" content="KAT6A chimeras form a self-reinforcing epigenetic module with NURF and MLL/COMPASS to sustain AML">
<meta name="citation_author" content="Junhui Lv">
<meta name="citation_author_institution" content="State Key Laboratory of Metabolism and Regulation in Complex Organisms, TaiKang Center for Life and Medical Sciences, School of Basic Medical Sciences, Wuhan University, 115 DongHu Road, Research Building III, Room 404, Wuchang District, Wuhan, 430071 China">
<meta name="citation_author" content="Zhinang Yin">
<meta name="citation_author_institution" content="State Key Laboratory of Metabolism and Regulation in Complex Organisms, TaiKang Center for Life and Medical Sciences, School of Basic Medical Sciences, Wuhan University, 115 DongHu Road, Research Building III, Room 404, Wuchang District, Wuhan, 430071 China">
<meta name="citation_author" content="Conghui Li">
<meta name="citation_author_institution" content="State Key Laboratory of Metabolism and Regulation in Complex Organisms, TaiKang Center for Life and Medical Sciences, School of Basic Medical Sciences, Wuhan University, 115 DongHu Road, Research Building III, Room 404, Wuchang District, Wuhan, 430071 China">
<meta name="citation_author" content="Honglin Wen">
<meta name="citation_author_institution" content="State Key Laboratory of Metabolism and Regulation in Complex Organisms, TaiKang Center for Life and Medical Sciences, School of Basic Medical Sciences, Wuhan University, 115 DongHu Road, Research Building III, Room 404, Wuchang District, Wuhan, 430071 China">
<meta name="citation_author" content="Jian Ni">
<meta name="citation_author_institution" content="State Key Laboratory of Virology and Biosafety, Frontier Science Center for Immunology and Metabolism, School of Pharmaceutical Sciences, Wuhan University, Wuhan, 430071 China">
<meta name="citation_author" content="Peiyuan Yang">
<meta name="citation_author_institution" content="State Key Laboratory of Metabolism and Regulation in Complex Organisms, TaiKang Center for Life and Medical Sciences, School of Basic Medical Sciences, Wuhan University, 115 DongHu Road, Research Building III, Room 404, Wuchang District, Wuhan, 430071 China">
<meta name="citation_author" content="Zemin Song">
<meta name="citation_author_institution" content="State Key Laboratory of Metabolism and Regulation in Complex Organisms, TaiKang Center for Life and Medical Sciences, School of Basic Medical Sciences, Wuhan University, 115 DongHu Road, Research Building III, Room 404, Wuchang District, Wuhan, 430071 China">
<meta name="citation_author" content="Ying Xiang">
<meta name="citation_author_institution" content="State Key Laboratory of Metabolism and Regulation in Complex Organisms, TaiKang Center for Life and Medical Sciences, School of Basic Medical Sciences, Wuhan University, 115 DongHu Road, Research Building III, Room 404, Wuchang District, Wuhan, 430071 China">
<meta name="citation_author" content="Honghong Wang">
<meta name="citation_author_institution" content="State Key Laboratory of Metabolism and Regulation in Complex Organisms, TaiKang Center for Life and Medical Sciences, School of Basic Medical Sciences, Wuhan University, 115 DongHu Road, Research Building III, Room 404, Wuchang District, Wuhan, 430071 China">
<meta name="citation_author" content="Rui Lu">
<meta name="citation_author_institution" content="State Key Laboratory of Metabolism and Regulation in Complex Organisms, TaiKang Center for Life and Medical Sciences, School of Basic Medical Sciences, Wuhan University, 115 DongHu Road, Research Building III, Room 404, Wuchang District, Wuhan, 430071 China">
<meta name="citation_author" content="Li Huang">
<meta name="citation_author_institution" content="Research Center for Medicine and Structural Biology, School of Basic Medical Sciences, Wuhan University, Wuhan, China">
<meta name="citation_author" content="Ying Zhou">
<meta name="citation_author_institution" content="Research Center for Medicine and Structural Biology, School of Basic Medical Sciences, Wuhan University, Wuhan, China">
<meta name="citation_author" content="Hai-Bing Zhou">
<meta name="citation_author_institution" content="State Key Laboratory of Virology and Biosafety, Frontier Science Center for Immunology and Metabolism, School of Pharmaceutical Sciences, Wuhan University, Wuhan, 430071 China">
<meta name="citation_author" content="Ruijing Xiao">
<meta name="citation_author_institution" content="State Key Laboratory of Metabolism and Regulation in Complex Organisms, TaiKang Center for Life and Medical Sciences, School of Basic Medical Sciences, Wuhan University, 115 DongHu Road, Research Building III, Room 404, Wuchang District, Wuhan, 430071 China">
<meta name="citation_author" content="Pingping Fang">
<meta name="citation_author_institution" content="State Key Laboratory of Metabolism and Regulation in Complex Organisms, TaiKang Center for Life and Medical Sciences, School of Basic Medical Sciences, Wuhan University, 115 DongHu Road, Research Building III, Room 404, Wuchang District, Wuhan, 430071 China">
<meta name="citation_author" content="Kaiwei Liang">
<meta name="citation_author_institution" content="State Key Laboratory of Metabolism and Regulation in Complex Organisms, TaiKang Center for Life and Medical Sciences, School of Basic Medical Sciences, Wuhan University, 115 DongHu Road, Research Building III, Room 404, Wuchang District, Wuhan, 430071 China">
<meta name="citation_author_institution" content="Hubei Province Key Laboratory of Allergy and Immunology, Department of Pathophysiology, School of Basic Medical Sciences, Wuhan University, Wuhan, China">
<meta name="citation_publication_date" content="2025 Aug 19">
<meta name="citation_volume" content="26">
<meta name="citation_firstpage" content="253">
<meta name="citation_doi" content="10.1186/s13059-025-03743-y">
<meta name="citation_pmid" content="40830799">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366150/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366150/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366150/pdf/13059_2025_Article_3743.pdf">
<meta name="description" content="KAT6A-CBP (K/C) and KAT6A-P300 (K/P) fusions are recurrent genetic alterations in acute myeloid leukemia (AML) associated with poor prognosis. Despite their strong oncogenic potential, the mechanisms underlying their genomic targeting and ...">
<meta name="og:title" content="KAT6A chimeras form a self-reinforcing epigenetic module with NURF and MLL/COMPASS to sustain AML">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="KAT6A-CBP (K/C) and KAT6A-P300 (K/P) fusions are recurrent genetic alterations in acute myeloid leukemia (AML) associated with poor prognosis. Despite their strong oncogenic potential, the mechanisms underlying their genomic targeting and ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366150/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Searchâ€¦
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12366150">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1186/s13059-025-03743-y"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/13059_2025_Article_3743.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12366150%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12366150/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12366150/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366150/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-genbio.png" alt="Genome Biology logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Genome Biology" title="Link to Genome Biology" shape="default" href="https://genomebiology.biomedcentral.com/" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Genome Biol</button></div>. 2025 Aug 19;26:253. doi: <a href="https://doi.org/10.1186/s13059-025-03743-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1186/s13059-025-03743-y</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Genome%20Biol%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Genome%20Biol%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Genome%20Biol%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Genome%20Biol%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>KAT6A chimeras form a self-reinforcing epigenetic module with NURF and MLL/COMPASS to sustain AML</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lv%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Junhui Lv</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Junhui Lv</span></h3>
<div class="p">
<sup>1</sup>State Key Laboratory of Metabolism and Regulation in Complex Organisms, TaiKang Center for Life and Medical Sciences, School of Basic Medical Sciences, Wuhan University, 115 DongHu Road, Research Building III, Room 404, Wuchang District, Wuhan, 430071 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lv%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Junhui Lv</span></a>
</div>
</div>
<sup>1,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yin%20Z%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Zhinang Yin</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Zhinang Yin</span></h3>
<div class="p">
<sup>1</sup>State Key Laboratory of Metabolism and Regulation in Complex Organisms, TaiKang Center for Life and Medical Sciences, School of Basic Medical Sciences, Wuhan University, 115 DongHu Road, Research Building III, Room 404, Wuchang District, Wuhan, 430071 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yin%20Z%22%5BAuthor%5D" class="usa-link"><span class="name western">Zhinang Yin</span></a>
</div>
</div>
<sup>1,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Li%20C%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Conghui Li</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Conghui Li</span></h3>
<div class="p">
<sup>1</sup>State Key Laboratory of Metabolism and Regulation in Complex Organisms, TaiKang Center for Life and Medical Sciences, School of Basic Medical Sciences, Wuhan University, 115 DongHu Road, Research Building III, Room 404, Wuchang District, Wuhan, 430071 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Li%20C%22%5BAuthor%5D" class="usa-link"><span class="name western">Conghui Li</span></a>
</div>
</div>
<sup>1,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wen%20H%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Honglin Wen</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Honglin Wen</span></h3>
<div class="p">
<sup>1</sup>State Key Laboratory of Metabolism and Regulation in Complex Organisms, TaiKang Center for Life and Medical Sciences, School of Basic Medical Sciences, Wuhan University, 115 DongHu Road, Research Building III, Room 404, Wuchang District, Wuhan, 430071 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wen%20H%22%5BAuthor%5D" class="usa-link"><span class="name western">Honglin Wen</span></a>
</div>
</div>
<sup>1,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ni%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Jian Ni</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Jian Ni</span></h3>
<div class="p">
<sup>2</sup>State Key Laboratory of Virology and Biosafety, Frontier Science Center for Immunology and Metabolism, School of Pharmaceutical Sciences, Wuhan University, Wuhan, 430071 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ni%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Jian Ni</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yang%20P%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Peiyuan Yang</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Peiyuan Yang</span></h3>
<div class="p">
<sup>1</sup>State Key Laboratory of Metabolism and Regulation in Complex Organisms, TaiKang Center for Life and Medical Sciences, School of Basic Medical Sciences, Wuhan University, 115 DongHu Road, Research Building III, Room 404, Wuchang District, Wuhan, 430071 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yang%20P%22%5BAuthor%5D" class="usa-link"><span class="name western">Peiyuan Yang</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Song%20Z%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Zemin Song</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Zemin Song</span></h3>
<div class="p">
<sup>1</sup>State Key Laboratory of Metabolism and Regulation in Complex Organisms, TaiKang Center for Life and Medical Sciences, School of Basic Medical Sciences, Wuhan University, 115 DongHu Road, Research Building III, Room 404, Wuchang District, Wuhan, 430071 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Song%20Z%22%5BAuthor%5D" class="usa-link"><span class="name western">Zemin Song</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Xiang%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Ying Xiang</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Ying Xiang</span></h3>
<div class="p">
<sup>1</sup>State Key Laboratory of Metabolism and Regulation in Complex Organisms, TaiKang Center for Life and Medical Sciences, School of Basic Medical Sciences, Wuhan University, 115 DongHu Road, Research Building III, Room 404, Wuchang District, Wuhan, 430071 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Xiang%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Ying Xiang</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20H%22%5BAuthor%5D" class="usa-link" aria-describedby="id9"><span class="name western">Honghong Wang</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">Honghong Wang</span></h3>
<div class="p">
<sup>1</sup>State Key Laboratory of Metabolism and Regulation in Complex Organisms, TaiKang Center for Life and Medical Sciences, School of Basic Medical Sciences, Wuhan University, 115 DongHu Road, Research Building III, Room 404, Wuchang District, Wuhan, 430071 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20H%22%5BAuthor%5D" class="usa-link"><span class="name western">Honghong Wang</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lu%20R%22%5BAuthor%5D" class="usa-link" aria-describedby="id10"><span class="name western">Rui Lu</span></a><div hidden="hidden" id="id10">
<h3><span class="name western">Rui Lu</span></h3>
<div class="p">
<sup>1</sup>State Key Laboratory of Metabolism and Regulation in Complex Organisms, TaiKang Center for Life and Medical Sciences, School of Basic Medical Sciences, Wuhan University, 115 DongHu Road, Research Building III, Room 404, Wuchang District, Wuhan, 430071 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lu%20R%22%5BAuthor%5D" class="usa-link"><span class="name western">Rui Lu</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Huang%20L%22%5BAuthor%5D" class="usa-link" aria-describedby="id11"><span class="name western">Li Huang</span></a><div hidden="hidden" id="id11">
<h3><span class="name western">Li Huang</span></h3>
<div class="p">
<sup>3</sup>Research Center for Medicine and Structural Biology, School of Basic Medical Sciences, Wuhan University, Wuhan, China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Huang%20L%22%5BAuthor%5D" class="usa-link"><span class="name western">Li Huang</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhou%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id12"><span class="name western">Ying Zhou</span></a><div hidden="hidden" id="id12">
<h3><span class="name western">Ying Zhou</span></h3>
<div class="p">
<sup>3</sup>Research Center for Medicine and Structural Biology, School of Basic Medical Sciences, Wuhan University, Wuhan, China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhou%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Ying Zhou</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhou%20HB%22%5BAuthor%5D" class="usa-link" aria-describedby="id13"><span class="name western">Hai-Bing Zhou</span></a><div hidden="hidden" id="id13">
<h3><span class="name western">Hai-Bing Zhou</span></h3>
<div class="p">
<sup>2</sup>State Key Laboratory of Virology and Biosafety, Frontier Science Center for Immunology and Metabolism, School of Pharmaceutical Sciences, Wuhan University, Wuhan, 430071 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhou%20HB%22%5BAuthor%5D" class="usa-link"><span class="name western">Hai-Bing Zhou</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Xiao%20R%22%5BAuthor%5D" class="usa-link" aria-describedby="id14"><span class="name western">Ruijing Xiao</span></a><div hidden="hidden" id="id14">
<h3><span class="name western">Ruijing Xiao</span></h3>
<div class="p">
<sup>1</sup>State Key Laboratory of Metabolism and Regulation in Complex Organisms, TaiKang Center for Life and Medical Sciences, School of Basic Medical Sciences, Wuhan University, 115 DongHu Road, Research Building III, Room 404, Wuchang District, Wuhan, 430071 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Xiao%20R%22%5BAuthor%5D" class="usa-link"><span class="name western">Ruijing Xiao</span></a>
</div>
</div>
<sup>1,</sup><sup>âœ‰</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Fang%20P%22%5BAuthor%5D" class="usa-link" aria-describedby="id15"><span class="name western">Pingping Fang</span></a><div hidden="hidden" id="id15">
<h3><span class="name western">Pingping Fang</span></h3>
<div class="p">
<sup>1</sup>State Key Laboratory of Metabolism and Regulation in Complex Organisms, TaiKang Center for Life and Medical Sciences, School of Basic Medical Sciences, Wuhan University, 115 DongHu Road, Research Building III, Room 404, Wuchang District, Wuhan, 430071 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Fang%20P%22%5BAuthor%5D" class="usa-link"><span class="name western">Pingping Fang</span></a>
</div>
</div>
<sup>1,</sup><sup>âœ‰</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Liang%20K%22%5BAuthor%5D" class="usa-link" aria-describedby="id16"><span class="name western">Kaiwei Liang</span></a><div hidden="hidden" id="id16">
<h3><span class="name western">Kaiwei Liang</span></h3>
<div class="p">
<sup>1</sup>State Key Laboratory of Metabolism and Regulation in Complex Organisms, TaiKang Center for Life and Medical Sciences, School of Basic Medical Sciences, Wuhan University, 115 DongHu Road, Research Building III, Room 404, Wuchang District, Wuhan, 430071 China </div>
<div class="p">
<sup>4</sup>Hubei Province Key Laboratory of Allergy and Immunology, Department of Pathophysiology, School of Basic Medical Sciences, Wuhan University, Wuhan, China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Liang%20K%22%5BAuthor%5D" class="usa-link"><span class="name western">Kaiwei Liang</span></a>
</div>
</div>
<sup>1,</sup><sup>4,</sup><sup>âœ‰</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>State Key Laboratory of Metabolism and Regulation in Complex Organisms, TaiKang Center for Life and Medical Sciences, School of Basic Medical Sciences, Wuhan University, 115 DongHu Road, Research Building III, Room 404, Wuchang District, Wuhan, 430071 China </div>
<div id="Aff2">
<sup>2</sup>State Key Laboratory of Virology and Biosafety, Frontier Science Center for Immunology and Metabolism, School of Pharmaceutical Sciences, Wuhan University, Wuhan, 430071 China </div>
<div id="Aff3">
<sup>3</sup>Research Center for Medicine and Structural Biology, School of Basic Medical Sciences, Wuhan University, Wuhan, China </div>
<div id="Aff4">
<sup>4</sup>Hubei Province Key Laboratory of Allergy and Immunology, Department of Pathophysiology, School of Basic Medical Sciences, Wuhan University, Wuhan, China </div>
<div class="author-notes p">
<div class="fn" id="_fncrsp93pmc__">
<sup>âœ‰</sup><p class="display-inline">Corresponding author.</p>
</div>
<div class="fn" id="_eqcntrb93pmc__">
<sup>#</sup><p class="display-inline">Contributed equally.</p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 Mar 5; Accepted 2025 Aug 12; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>Â© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12366150Â Â PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40830799/" class="usa-link">40830799</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<section id="sec1"><h3 class="pmc_sec_title">Background</h3>
<p id="Par1">KAT6A-CBP (K/C) and KAT6A-P300 (K/P) fusions are recurrent genetic alterations in acute myeloid leukemia (AML) associated with poor prognosis. Despite their strong oncogenic potential, the mechanisms underlying their genomic targeting and leukemogenic function remain unclear. A major challenge has been their large size, which has impeded preclinical model development and mechanistic studies.</p></section><section id="sec2"><h3 class="pmc_sec_title">Results</h3>
<p id="Par2">We employ a domain-focused truncation strategy to generate de novo murine models of K/C and K/P fusions, which faithfully recapitulate the morphological, immunophenotypic, and transcriptomic features of KAT6A-rearranged AML. Genomic profiling reveals that KAT6A fusions preferentially localize to H3K4me2/3-marked regions, while biochemical analyses uncover that KAT6A interacts with the Nucleosome Remodeling Factor (NURF), a key H3K4me2/3 reader. Disrupting NURF-chromatin interactions via depletion or small-molecule inhibition of its subunit, Bromodomain PHD Finger Transcription Factor (BPTF), impairs K/C recruitment and disrupts MLL/COMPASS-mediated H3K4me2 deposition, defining a functional epigenetic module involving KAT6A chimeras, NURF, and MLL/COMPASS. Notably, CBP/P300 inhibition reduces histone acetylation and chromatin accessibility, further impairing the recruitment of this epigenetic module. Targeting this module via NURF or CBP/P300 inhibition demonstrates efficacy in K/C leukemia models, with enhanced therapeutic effects observed when combined.</p></section><section id="sec3"><h3 class="pmc_sec_title">Conclusions</h3>
<p id="Par3">Our study identifies a self-reinforcing epigenetic module of histone modifiers and readers in KAT6A-rearranged AML, providing mechanistic insights into the genomic targeting of KAT6A chimeras and highlighting promising combinatorial therapeutic strategies.</p></section><section id="sec4"><h3 class="pmc_sec_title">Supplementary Information</h3>
<p>The online version contains supplementary material available at 10.1186/s13059-025-03743-y.</p></section><section id="kwd-group1" lang="en" class="kwd-group"><p><strong>Keywords:</strong> Acute myeloid leukemia, AML, KAT6A, KAT6A-CBP, KAT6A-P300, Nucleosome Remodeling Factor, NURF, MLL/COMPASS, Chromatin accessibility</p></section></section><section id="Sec1"><h2 class="pmc_sec_title">Background</h2>
<p id="Par4">Acute myeloid leukemia (AML) is the most common leukemia in adults, yet long-term survival rates remain dismally low despite significant therapeutic advancements [<a href="#CR1" class="usa-link" aria-describedby="CR1">1</a>, <a href="#CR2" class="usa-link" aria-describedby="CR2">2</a>]. A defining feature of AML is the frequent occurrence of chromosomal translocations that produce oncogenic fusion proteins, driving leukemogenesis from hematopoietic stem and progenitor cells (HSPCs) [<a href="#CR3" class="usa-link" aria-describedby="CR3">3</a>â€“<a href="#CR5" class="usa-link" aria-describedby="CR5">5</a>]. Among these, Lysine Acetyltransferase 6â€‰A (KAT6A, also known as MOZ) fusions are particularly prominent [<a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>â€“<a href="#CR8" class="usa-link" aria-describedby="CR8">8</a>]. KAT6A, a member of the MYST family of histone acetyltransferases, is a critical regulator of HOX gene expression, essential for embryogenesis and hematopoiesis [<a href="#CR9" class="usa-link" aria-describedby="CR9">9</a>â€“<a href="#CR11" class="usa-link" aria-describedby="CR11">11</a>]. It achieves this through interactions with chromatin regulators, including bromodomain-PHD finger protein 1 (BRPF1), inhibitor of growth (ING) proteins, MYST/Esa1-associated factor 6 (MEAF6), as well as through collaboration with the H3K4 methyltransferase MLL/COMPASS [<a href="#CR12" class="usa-link" aria-describedby="CR12">12</a>â€“<a href="#CR14" class="usa-link" aria-describedby="CR14">14</a>].</p>
<p id="Par5">In AML, recurrent chromosomal rearrangements fuse KAT6A to transcriptional coactivators, including P300 (EP300), CBP (CREBBP), and nuclear receptor coactivators (NCOA3 and TIF2). These fusions are associated with distinct clinical features such as monocytic or myelomonocytic differentiation, leukemia cutis, erythrophagocytosis, as well as high-risk outcomes including disseminated intravascular coagulation and poor survival [<a href="#CR7" class="usa-link" aria-describedby="CR7">7</a>, <a href="#CR15" class="usa-link" aria-describedby="CR15">15</a>]. CBP and P300, paralogous histone acetyltransferases with conserved domain architectures, serve as transcriptional coactivators by bridging chromatin remodelers and transcription factors. Despite their critical oncogenic role,Â KAT6AÂ translocations are rare (&lt;â€‰1%), with the most frequent subtype,Â KAT6A-CBP(K/C), occurring in only 0.2â€“0.4% of AML cases [<a href="#CR16" class="usa-link" aria-describedby="CR16">16</a>, <a href="#CR17" class="usa-link" aria-describedby="CR17">17</a>]. CBP and P300 bridge transcription factors to the general transcription machinery and promote chromatin relaxation through intrinsic acetyltransferase activity [<a href="#CR18" class="usa-link" aria-describedby="CR18">18</a>, <a href="#CR19" class="usa-link" aria-describedby="CR19">19</a>]. Conserved domains, such as the cysteine-histidine-rich (C/H) regions and functional modules like TAZ2, KIX, bromodomain, Q/P-rich region, and HAT domain, may mediate the oncogenic functions of KAT6A fusions [<a href="#CR20" class="usa-link" aria-describedby="CR20">20</a>]. However, whether these domains are truly essential for leukemogenesis remains unknown, and further investigation is needed to determine their specific contributions to the pathogenesis of KAT6A-rearranged AML.</p>
<p id="Par6">Despite their potent oncogenic activity, the mechanisms by which KAT6A fusion proteins achieve genomic specificity and drive leukemogenesis remain poorly understood [<a href="#CR21" class="usa-link" aria-describedby="CR21">21</a>]. Previous studies suggest that the leukemogenic potential of KAT6A-TIF2 depends on its chromatin-binding ability rather than its histone acetyltransferase (HAT) activity, with oncogenicity further amplified by CBP recruitment through its fusion partners [<a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>, <a href="#CR23" class="usa-link" aria-describedby="CR23">23</a>]. The winged helix domain (WH1) of KAT6A can bind unmethylated CpG islands and interact with P300/CBP via the TAZ2 domain [<a href="#CR24" class="usa-link" aria-describedby="CR24">24</a>â€“<a href="#CR26" class="usa-link" aria-describedby="CR26">26</a>]. However, the widespread presence of unmethylated CpG islands across the genome [<a href="#CR27" class="usa-link" aria-describedby="CR27">27</a>] does not fully explain the genomic specificity of KAT6A-P300 (K/P) and K/C fusions, leaving key mechanistic gaps unexplored.</p>
<p id="Par7">Adding to these challenges is the size of CBP and P300, each exceeding 2400 amino acids and forming fusion proteins of approximately 3500 amino acids when fused with KAT6A. These large proteins pose significant technical barriers for traditional retroviral expression systems due to their limited packaging capacity, hampering efficient expression and robust in vivo modeling [<a href="#CR28" class="usa-link" aria-describedby="CR28">28</a>, <a href="#CR29" class="usa-link" aria-describedby="CR29">29</a>]. To overcome these limitations, we employed a domain-focused truncation strategy to generate functional K/C and K/P fusion proteins. These truncated constructs, preserving the essential N-terminal domain of KAT6A and critical CBP/P300 domains, facilitated the development of robust murine AML models. These models faithfully recapitulate the morphological, transcriptomic, and immunophenotypic features of KAT6A-rearranged AML. Leveraging these systems, we uncovered a critical epigenetic transcriptional module comprising KAT6A fusions, the Nucleosome Remodeling Factor (NURF) complex, and MLL/COMPASS. This self-reinforcing module sustains oncogenic transcriptional activation and drives AML progression. Notably, our findings identify this module as a promising therapeutic target in these rare but aggressive KAT6A-rearranged AML.</p></section><section id="Sec2"><h2 class="pmc_sec_title">Results</h2>
<section id="Sec3"><h3 class="pmc_sec_title">Development of K/C and K/P Murine Leukemia Models Recapitulating Primary AML Morphology</h3>
<p id="Par8">To address the challenges posed by large sizes of full-length K/C and K/P fusion proteins (~â€‰3500 amino acids), we employed a domain-focused truncation strategy to generate four truncated KAT6A fusion variants incorporating key functional domains of CBP or P300 (Fig.Â <a href="#Fig1" class="usa-link">1</a>A). Specifically, K/C-I included the KIX, bromodomain, PHD, and HAT domains; K/C-II comprised the ZZ-TAZ2 and Q/P-rich regions; K/C-III incorporated the bromodomain, PHD, HAT, and ZZ-TAZ2 domains; while K/C-IV omitted the ZZ-TAZ2 domain from K/C-III (Fig.Â <a href="#Fig1" class="usa-link">1</a>A). Using a optimized retrovirus-based system with HaloTag tracking in our lab [<a href="#CR30" class="usa-link" aria-describedby="CR30">30</a>], we transduced c-Kit-enriched murine HSPCs and enhanced transduction efficiency via RetroNectin-coated plates with spinoculation (Fig.Â <a href="#Fig1" class="usa-link">1</a>B; Additional file 1: Fig. S1A). Transduced cells were transplanted into lethally irradiated C57BL/6 mice for leukemia development monitoring.</p>
<figure class="fig xbox font-sm" id="Fig1"><h4 class="obj_head">Fig.Â 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12366150_13059_2025_3743_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6f0e/12366150/9a5874008fee/13059_2025_3743_Fig1_HTML.jpg" loading="lazy" id="MO1" height="734" width="709" alt="Fig.Â 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Establishment of de novo K/C and K/P murine leukemia models using a domain-focused truncation strategy. <strong>A</strong> Design of KAT6A-CBP (K/C) and KAT6A-P300 (K/P) truncations featuring different domains to circumvent the retroviral cargo limit. DPF: double PHD finger; WH1: winged helix (WH) domains. <strong>B</strong> Schematic overview of a retrovirus-based murine leukemia model derived from c-Kit-enriched HSPCs, utilizing K/C or K/P truncations to induce leukemia in mice. <strong>C</strong>, <strong>D</strong> Kaplanâ€“Meier survival curves for C57BL/6J mice transplanted with HSPCs transduced with K/C constructs, assessing oncogenic potential (<em>n</em>â€‰=â€‰5 mice/group). Endpoint examinations of leukemic mice revealed splenomegaly and pale bone marrow in both the femur and tibia. <strong>E</strong>, <strong>F</strong> Kaplanâ€“Meier survival analysis of C57BL/6J mice transplanted with HSPCs expressing K/P truncations (<em>n</em>â€‰=â€‰5 mice/group). Endpoint analysis confirmed leukemic characteristics in the leukemic cohort. <strong>G</strong> Flow cytometry analysis of HaloTag signals confirmed extensive infiltration of leukemic blasts in the spleens of leukemic mice. <strong>H</strong> H&amp;E-stained spleen sections from the indicated cohorts, shown at 20â€‰Ã—â€‰magnification. The white pulp (WP) is outlined in white for samples from mice transplanted with MSCV empty vector-infected HSPCs (top). Notably, the clear demarcation between white pulp and red pulp (RP) is lost in mice receiving K/C-III and K/P-III due to excessive expansion of transformed leukemia cells, resulting in the splenomegaly observed in Fig. 1D and 1F. <strong>I</strong> Secondary transplantation study using primary leukemic spleen cells from mice initially transduced with K/C-III or K/P-III, assessing leukemogenic capacity (<em>n</em>â€‰=â€‰6). <strong>J</strong> Wright-Giemsa staining of leukemic cells revealed both mature and immature myeloid cells, many with monocytic features, alongside some blast forms. Erythrophagocytosis was observed in both K/C-III and K/P-III leukemic cells. <strong>K</strong> RT-qPCR analysis of Meis1, Hoxa9, and Hoxa10 gene expression in c-Kit-enriched HSPCs, normal bone marrow cells, and K/C-III and K/P-III leukemic cells. Both K/C-III and K/P-III leukemic cells exhibited elevated expression of these known KAT6A chimera target genes. One-way ANOVA was used for statistical analysis</p></figcaption></figure><p id="Par9">Strikingly, all mice (5/5) transplanted with K/C-III-transduced cells developed AML within 100Â days post-transplantation, whereas mice receiving other K/C variants or controls remained disease-free (Fig.Â <a href="#Fig1" class="usa-link">1</a>C; Additional file 1: Fig. S1B and C). Leukemic mice displayed hallmark AML features, including splenomegaly and pale, leukemic bone marrow in femurs and tibiae (Fig.Â <a href="#Fig1" class="usa-link">1</a>D). Consistent results were observed with K/P truncation constructs (Fig.Â <a href="#Fig1" class="usa-link">1</a>A and E). Mice expressing K/P-III developed classic AML features, such as pale bone marrow, splenomegaly, and infiltration of HaloTag-positive leukemic blasts (Fig.Â <a href="#Fig1" class="usa-link">1</a>F and G). These findings underscore the critical roles of the HAT and ZZ-TAZ2 domains of CBP/P300 in leukemogenesis, likely attributable to their histone-binding capacity and regulation of HAT specificity and activity of CBP/P300 [<a href="#CR31" class="usa-link" aria-describedby="CR31">31</a>, <a href="#CR32" class="usa-link" aria-describedby="CR32">32</a>]. In contrast, the KIX domain and Q/P-rich region were found to be dispensable.</p>
<p id="Par10">Histological examination revealed extensive leukemic infiltration and disrupted splenic architecture (Fig.Â <a href="#Fig1" class="usa-link">1</a>H). Secondary transplantation experiments confirmed the aggressive and persistent nature of leukemia driven by K/C-III and K/P-III constructs (Fig.Â <a href="#Fig1" class="usa-link">1</a>I). Giemsa staining of leukemic cells demonstrated differentiation at the myelomonocytic stage, along with erythrophagocytosis (Fig.Â <a href="#Fig1" class="usa-link">1</a>J), closely resembling the morphology observed in human KAT6A-rearranged AML [<a href="#CR7" class="usa-link" aria-describedby="CR7">7</a>, <a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>]. Furthermore, transcriptomic analysis showed significant upregulation of key transcriptional targets of KAT6A fusions [<a href="#CR8" class="usa-link" aria-describedby="CR8">8</a>], including <em>Meis1</em>, <em>Hoxa9</em>, and <em>Hoxa10</em>, in K/C-III and K/P-III leukemic cells compared to myeloid progenitors (MPs, Lin<sup>âˆ’</sup>Sca1<sup>âˆ’</sup>c-Kit<sup>+</sup>) and normal bone marrow cells (Fig.Â <a href="#Fig1" class="usa-link">1</a>K). These results underscore the fidelity of our K/C and K/P murine models in recapitulating the key features of primary KAT6A-rearranged AML.</p></section><section id="Sec4"><h3 class="pmc_sec_title">Immunotyping and Transcriptional Profiling of K/C and K/P Leukemic Cells</h3>
<p id="Par11">To characterize K/C and K/P leukemic cells, we conducted 15-fluorochrome spectral cytometry on bone marrow samples, using key markers to define the hematopoietic compartment architecture [<a href="#CR33" class="usa-link" aria-describedby="CR33">33</a>]. Phenotypic analysis revealed distinct immunoprofiles for K/C and K/P leukemia (Fig.Â <a href="#Fig2" class="usa-link">2</a>A and B). Leukemic cells were distinguished from normal murine hematopoietic populations by markers such as CD34, FcÎ³R, Flk2, and CD48, with subsets expressing c-Kit, CD49b, and CD41 (Fig.Â <a href="#Fig2" class="usa-link">2</a>C; Additional file 1: Fig. S2A and B). Both subtypes were positive for lineage specification markers â€”a combination of Ter119, Mac1, Gr1, B220, CD5, CD3, CD4, and CD8 (Fig.Â <a href="#Fig2" class="usa-link">2</a>D). Individual analysis of lineage markers showed predominant expression of myeloid markers Mac1 and Gr1 in the AML cells (Fig.Â <a href="#Fig2" class="usa-link">2</a>E), which aligns with Giemsa staining that indicated differentiation at the myelomonocytic stage (Fig.Â <a href="#Fig1" class="usa-link">1</a>J).</p>
<figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig.Â 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12366150_13059_2025_3743_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6f0e/12366150/14fab6f622c9/13059_2025_3743_Fig2_HTML.jpg" loading="lazy" id="MO2" height="682" width="708" alt="Fig.Â 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Immunotyping and transcriptome analysis of K/C and K/P murine leukemia cells. <strong>A</strong>, <strong>B</strong> Spectral flow cytometry of leukemia cells induced by K/C-III and K/P-III constructs. High-dimensional opt-SNE analysis, using all parameters except lineage and live/dead status, effectively distinguishes K/C-III and K/P-III AML cells (red) from normal murine hematopoietic cells. <strong>C</strong> Surface marker analysis of K/C-III and K/P-III AML cells, showing positive expression for CD34, FcÎ³R, Flk2, CD41, CD48, and c-Kit (<em>n</em>â€‰=â€‰2). <strong>D</strong> K/C-III and K/P-III AML cells exhibit lineage specification markers, including Ter119, Mac1, Gr1, B220, CD5, CD3, CD4, and CD8. The orange arrows indicate K/C-III and K/P-III leukemia cells. <strong>E</strong> Individual lineage marker analysis reveals strong expression of myeloid markers Mac1 and Gr1. <strong>F</strong> Workflow depicting isolation of K/C-III and K/P-III leukemia cells and subsequent mRNA sequencing. <strong>G</strong> Principal component analysis of mRNA-seq data from K/C-III and K/P-III leukemia cells, illustrating distinct transcriptional landscapes compared to MPs and c-Kit-enriched HSPCs (<em>n</em>â€‰=â€‰2). <strong>H</strong> Heatmap of shared differentially expressed genes in K/C-III and K/P-III leukemia cells versus MPs, highlighting distinct expression profiles. Gene ontology analysis of these differentially expressed genes is also shown. C1 and C2 represent the downregulated and upregulated gene clusters, respectively, compared to MPs. <strong>I</strong> Enrichment analysis of the KAT6A-rearranged AML transcriptional signature in comparison to AML cases with normal karyotypes. RNA-seq data from the TARGET AML Project (TARGET-AML) includes cases with KAT6A rearrangements (<em>n</em>â€‰=â€‰7) and normal karyotypes (<em>n</em>â€‰=â€‰111). <strong>J</strong> GSEA analysis of K/C-III and K/P-III leukemia cells compared to MPs, showing enrichment of the KAT6A-rearranged signature genes. <strong>K</strong> RNA-seq tracks demonstrate upregulation of <em>Hoxa</em> cluster genes in K/C-III and K/P-III leukemia cells compared to c-Kit-enriched HSPCs and MPs</p></figcaption></figure><p id="Par12">For transcriptional profiling, we performed mRNA sequencing and principal component analysis (PCA) on sorted leukemia cells, c-Kit-enriched HSPCs, and MPs. Leukemia cells displayed significant transcriptional divergence from normal HSPCs and MPs, with highly similar profiles between K/C-III and K/P-III variants (Fig.Â <a href="#Fig2" class="usa-link">2</a>F and G). Differential expression analysis identified 1,735 downregulated and 2,547 upregulated genes in K/C-III and K/P-III leukemic cells compared to MPs (Fig.Â <a href="#Fig2" class="usa-link">2</a>H; Additional file 2: Table S1). Gene ontology analysis of upregulated genes revealed enrichment in pathways related to cell activation, immune response, hematopoiesis, and cell proliferation, while downregulated genes were associated with cellular catabolism, myeloid homeostasis, and DNA metabolic processes (Fig.Â <a href="#Fig2" class="usa-link">2</a>H).</p>
<p id="Par13">As KAT6A-rearranged AML is characterized by a distinct transcriptional signature, including upregulation of the HOXA gene cluster [<a href="#CR8" class="usa-link" aria-describedby="CR8">8</a>], we analyzed the TARGET AML expression dataset. This confirmed a unique gene signature in KAT6A-rearranged AML cases (<em>n</em>â€‰=â€‰7) compared to AML cases with normal karyotypes (<em>n</em>â€‰=â€‰111) (Fig.Â <a href="#Fig2" class="usa-link">2</a>I). Gene set enrichment analysis (GSEA) of murine K/C and K/P leukemia cells showed significant enrichment of the KAT6A-rearranged AML signature (Fig.Â <a href="#Fig2" class="usa-link">2</a>J). Furthermore, tracks of individual genes verified robust upregulation of <em>Hoxa7</em>, <em>Hoxa9</em>, and <em>Hoxa10</em> in K/C-III and K/P-III leukemic cells compared to c-Kit -enriched HSPCs or MPs (Fig.Â <a href="#Fig2" class="usa-link">2</a>K). These findings confirm that our K/C and K/P murine leukemia models faithfully replicate the immunophenotypic and transcriptional features of primary KAT6A-rearranged AML, providing robust systems for investigating disease mechanisms and therapeutic strategies.</p></section><section id="Sec5"><h3 class="pmc_sec_title">KAT6A Co-Localizes with H3K4me2/3 and Interacts with the NURF Complex</h3>
<p id="Par14">To investigate the genomic targeting of KAT6A chimeras, we performed CUT&amp;RUN analysis on K/C-III, K/P-III, and the KAT6A N-terminus, examining their genome-wide binding patterns alongside Pol II and histone modifications, including H3K4me1/2/3, H3K36me3, H3K27me3, and H3K27ac. KAT6A N-terminus, K/C-III, and K/P-III proteins showed extensive promoter binding across the genome (Fig.Â <a href="#Fig3" class="usa-link">3</a>A; Additional file 1: Fig. S3A), with distribution extending beyond CpG islands, indicating alternative mechanisms beyond WH1 domain-mediated recognition of unmethylated CpG islands [<a href="#CR24" class="usa-link" aria-describedby="CR24">24</a>, <a href="#CR25" class="usa-link" aria-describedby="CR25">25</a>]. Heatmap and Pearson correlation analyses revealed a strong association with H3K4me2 and a moderate association with H3K4me3 (Fig.Â <a href="#Fig3" class="usa-link">3</a>B and C). Similar patterns for KAT6A-TIF2 and KAT6A-NCOA3 chimeras supported the genome-wide colocalization of KAT6A with H3K4me2/3 (Additional file 1: Fig. S3B). Correlations between the KAT6A N-terminus and its chimeric variants demonstrated that the N-terminus dictates their genomic distribution (Additional file 1: Fig. S3C-F).</p>
<figure class="fig xbox font-sm" id="Fig3"><h4 class="obj_head">Fig.Â 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12366150_13059_2025_3743_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6f0e/12366150/d184e926b605/13059_2025_3743_Fig3_HTML.jpg" loading="lazy" id="MO3" height="697" width="708" alt="Fig.Â 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>KAT6A associates with H3K4me2/3 and directly interact with the NURF complex. <strong>A</strong> Genome browser tracks showing the distribution of CpG islands, methyl-C levels, KAT6A-NT1, K/C-III, K/P-III, Pol II, and histone markers in HEK293T cells. CUT&amp;RUN analyses of KAT6A-NT1, K/C-III, and K/P-III reveal similar binding profiles with occupancy extending beyond CpG islands, overlapping with H3K4me3 and H3K4me2. <strong>B</strong>, <strong>C</strong> Heatmap analysis of KAT6A-NT1, K/C-III, K/P-III, Pol II, and histone markers in HEK293T cells (B). KAT6A-NT1 peaks (<em>N</em>â€‰=â€‰10,770) were ranked by decreasing occupancy, with Pearson correlation analysis (C) showing positive correlations between KAT6A-NT1, K/C-III, and K/P-III with H3K4me2 and H3K4me3 across the genome. <strong>D</strong> IP-MS analysis comparing KAT6A-NT1, K/C-III, and K/P-III to MSCV vector control in HEK293T cell lysates. Silver staining of IP products is displayed on the right. <strong>E</strong> Mass spectrometry analysis of KAT6A-NT1, K/C-III, and K/P-III immunoprecipitates. The peptide and sum PEP score for the BRPF1-KAT6A acetyltransferase complex and NURF complex subunits are presented. MSCV vector serves as the binding control. The list of interacting proteins was included in Additional file 2: Table S2. <strong>F</strong> Confirmation of the NURF complex interaction with KAT6A in HEK293T cells, demonstrated through anti-SMARCA1 immunoprecipitation followed by immunoblotting. KAT6A mutants were included, showing that deletion of the KAT6A MYST domain impairs the interaction with the NURF complex. <strong>G</strong> Colocalization of the NURF complex and K/C-III in murine K/C-III leukemia cells. Examples of genome browser tracks for BPTF, SMARCA1, HaloTag-K/C-III CUT&amp;RUN, and ChIP-seq signals for H3K4me2, H3K4me3, and H3K27ac at the <em>Metap1</em> and <em>Adh5</em> loci are shown. <strong>H</strong> Genome-wide analysis demonstrating the colocalization of K/C-III with the NURF complex and H3K4me2 and H3K4me3. Heatmaps are sorted by decreasing K/C-III occupancy. <strong>I</strong> Metaplots of BPTF, SMARCA1, HaloTag-K/C-III, H3K4me2, H3K4me3, and H3K27ac at K/C-III peaks in murine leukemia cells</p></figcaption></figure><p id="Par15">To uncover protein interactions underlying these genomic associations, we performed immunoprecipitation and mass spectrometry (IP-MS) on KAT6A N-terminus and chimeras (Fig.Â <a href="#Fig3" class="usa-link">3</a>D). This analysis identified core components of the KAT6A complex, including BRPF1, ING5, and MEAF6 (Fig.Â <a href="#Fig3" class="usa-link">3</a>E; Additional file 2: Table S2; Additional file 1: Fig. S3G). Notably, we noticed that KAT6A and its chimeras interacted with the NURF complex, comprising BPTF, SMARCA1, RBBP4, and RBBP7 (Fig.Â <a href="#Fig3" class="usa-link">3</a>E; Additional file 1: Fig. S3G; Additional file 2: Table S2). Deletion mutant studies revealed that the MYST domain of KAT6A is essential for binding the NURF complex (Fig.Â <a href="#Fig3" class="usa-link">3</a>F). Notably, the BPTF subunit of the NURF complex binds to H3K4me2/3 via its PHD finger domain [<a href="#CR34" class="usa-link" aria-describedby="CR34">34</a>, <a href="#CR35" class="usa-link" aria-describedby="CR35">35</a>], likely explaining the observed colocalization of KAT6A with these histone markers.</p>
<p id="Par16">To confirm the genomic associations among the NURF complex, KAT6A chimeras, and H3K4me2/3 in leukemia cells, we conducted CUT&amp;RUN and ChIP-seq analyses in K/C-III leukemia cells. These analyses revealed significant colocalization of K/C-III with NURF components (BPTF and SMARCA1) and the histone modifications H3K4me2/3 (Fig.Â <a href="#Fig3" class="usa-link">3</a>G and I). Scatter plot analyses further demonstrated strong positive correlations between K/C-III binding and NURF complex components (Additional file 1: Fig. S3H and I), confirming a genome-wide association of K/C-III with the NURF complex. Together, these findings suggest that KAT6A chimeras collaborate with the NURF complex, likely leveraging interactions with NURF to regulate leukemogenic gene expression programs.</p></section><section id="Sec6"><h3 class="pmc_sec_title">Disruption of NURF-Chromatin Interaction Impairs Genomic Targeting of KAT6A Chimeras</h3>
<p id="Par17">Given the interaction between the NURF complex and the MYST domain of KAT6A (Fig.Â <a href="#Fig3" class="usa-link">3</a>F), as well as the NURF complexâ€™s ability to bind histone modifications, we first examined how KAT6A mutants affect chromatin association. CUT&amp;RUN analysis revealed that the KAT6A-Î”MYST mutant exhibited significantly reducedâ€‰~â€‰50% genome-wide binding compared to wild-type KAT6A, whereas a catalytically inactive mutant did not show this effect (Additional file 1: Fig. S4A-C). These findings indicate that the MYST domain, independent of its catalytic activity, is critical for KAT6Aâ€™s chromatin association.</p>
<p id="Par18">Further investigation using BPTF knockdown demonstrated that BPTF depletion reduced KAT6A chromatin occupancy, as shown by chromatin fractionation and CUT&amp;RUN analyses (Fig.Â <a href="#Fig4" class="usa-link">4</a>A-C). Metaplot analyses revealed aâ€‰~â€‰50% reduction in KAT6A occupancy following BPTF knockdown (Fig.Â <a href="#Fig4" class="usa-link">4</a>D and E). To disrupt BPTF-chromatin interaction, we used BZ1, a small-molecule inhibitor that blocks BPTF binding to H4K16ac [<a href="#CR36" class="usa-link" aria-describedby="CR36">36</a>] (Fig.Â <a href="#Fig4" class="usa-link">4</a>F). BZ1 treatment resulted in dose-dependent inhibition of cell proliferation in K/C-III and K/P-III leukemia cells, with IC50 values of 0.54Â ÂµM and 0.70Â ÂµM, respectively (Fig.Â <a href="#Fig4" class="usa-link">4</a>G and H). Additionally, BZ1 suppressed colony formation and induced G1 cell cycle arrest in both leukemia models, with minimal effects on cell differentiation (Fig.Â <a href="#Fig4" class="usa-link">4</a>I and J; Additional file 1: Fig. S5A and B). In contrast, HSPCs exhibited an IC50 of 6.9Â ÂµM for BZ1â€”over tenfold higher than that observed in K/C-III and K/P-III leukemia cells (IC50â€‰=â€‰0.54â€“0.70Â ÂµM)â€”and showed minimal toxicity to normal hematopoietic progenitors (Additional file 1: Fig. S5C and D), indicating a favorable therapeutic window.</p>
<figure class="fig xbox font-sm" id="Fig4"><h4 class="obj_head">Fig.Â 4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12366150_13059_2025_3743_Fig4_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6f0e/12366150/353273a28381/13059_2025_3743_Fig4_HTML.jpg" loading="lazy" id="MO4" height="694" width="708" alt="Fig.Â 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>BPTF sustains the genome-wide targeting of KAT6A and KAT6A chimeras. <strong>A</strong> BPTF knockdown reduces KAT6A chromatin association. shRNA-mediated knockdown of BPTF in Flag-KAT6A-NT1-expressing HEK293T cells followed by chromatin fractionation and immunoblotting using anti-BPTF, anti-SMARCA1, and anti-Flag antibodies. <strong>B</strong> CUT&amp;RUN analysis of BPTF and Flag-KAT6A occupancy after BPTF knockdown in Flag-KAT6A-NT1-expressing HEK293T cells. BPTF depletion decreases KAT6A occupancy at the <em>NUP153</em> locus. <strong>C</strong> Genome-wide analysis of BPTF-occupied promoters (<em>n</em>â€‰=â€‰12,428) showing a global reduction in Flag-KAT6A occupancy following BPTF knockdown. Heatmaps are ranked by decreasing BPTF occupancy. <strong>D</strong>, <strong>E</strong> Metaplots of CUT&amp;RUN signals for BPTF and Flag-KAT6A at TSS following BPTF knockdown. Depletion of BPTF reduces KAT6A occupancy across the genome. <strong>F</strong> Chemical structure of BZ1, a pyridazinone-based inhibitor targeting the BPTF bromodomain [<a href="#CR36" class="usa-link" aria-describedby="CR36">36</a>]. <strong>G</strong>, <strong>H</strong> Doseâ€“response analysis of BZ1 in K/C-III and K/P-III leukemia cells, showing IC50 values. <strong>I</strong> BZ1 treatment inhibits colony formation in K/C-III and K/P-III leukemia cells. Leukemia cells were treated with 2 Î¼M BZ1 for one week. <strong>J</strong> BZ1 induces G1 cell cycle arrest in K/C-III and K/P-III leukemia cells. Leukemia cells were treated with 2 Î¼M BZ1 for 48 h (two-way ANOVA). <strong>K</strong>-<strong>M</strong> Genome-wide analysis of BPTF and K/C-III occupancy in murine K/C-III leukemia cells following 2 Î¼M BZ1 treatment for 6 h. Heatmaps of BPTF peaks (<em>n</em>â€‰=â€‰9,407) show reduced occupancy of BPTF and K/C-III. <strong>N</strong> Scatter plot illustrating log<sub>2</sub> fold changes of BPTF and K/C-III signals at BPTF peaks after 6 h BZ1 treatment</p></figcaption></figure><p id="Par19">To generalize these findings, we tested BZ1 in murine models of KAT6A-rearranged leukemia, including KAT6A-TIF2 and KAT6A-NCOA3 (Additional file 1: Fig. S6). Consistent with observations in K/C-III and K/P-III models, BZ1 inhibited cell proliferation and colony formation, and induced G1 arrest in these leukemia cells as well (Additional file 1: Fig. S6I-L). A six-hour BZ1 treatment of K/C-III leukemia cells led to reduced genome-wide occupancy of both BPTF and K/C-III at BPTF-bound promoter regions (Fig.Â <a href="#Fig4" class="usa-link">4</a>K-M). Log2 fold-change scatter plots of BPTF and K/C-III signals at BPTF peaks further demonstrated a reduction in their occupancy following BZ1 treatment (Fig.Â <a href="#Fig4" class="usa-link">4</a>N). Collectively, these results highlight the critical role of the NURF complex, via BPTF, in mediating the genomic targeting of KAT6A chimeras in leukemia cells.</p></section><section id="Sec7"><h3 class="pmc_sec_title">NURF Sustains WDR5-MLL/COMPASS-deposited H3K4 Methylation</h3>
<p id="Par20">To investigate the effects of BPTF inhibition on gene expression, we performed mRNA-seq on K/C-III leukemia cells following 6 and 12Â h of BZ1 treatment (Fig.Â <a href="#Fig5" class="usa-link">5</a>A; Additional file 2: Table S3). Gene ontology analysis of differentially expressed genes revealed enrichment in cell cycle-related pathways (Additional file 1: Fig. S7A), consistent with the observed G1 cell cycle arrest (Fig.Â <a href="#Fig4" class="usa-link">4</a>J). Among K/C-III-bound genes, we identified two distinct clusters based on their response to BZ1. Cluster 1, which comprised the majority, exhibited a trend toward downregulation, whereas cluster 2 showed minimal changes in expression (Fig.Â <a href="#Fig5" class="usa-link">5</a>B). Genes in cluster 2 had lower promoter occupancy by K/C-III and BPTF compared to cluster 1, suggesting that BZ1 preferentially affects genes with higher K/C-III and BPTF occupancy (Fig.Â <a href="#Fig5" class="usa-link">5</a>C and D). Notably, chromatin accessibility remained largely unchanged at both cluster 1 and cluster 2 genes (Additional file 1: Fig. S7B and C).</p>
<figure class="fig xbox font-sm" id="Fig5"><h4 class="obj_head">Fig.Â 5.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12366150_13059_2025_3743_Fig5_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6f0e/12366150/0cbc7736509d/13059_2025_3743_Fig5_HTML.jpg" loading="lazy" id="MO5" height="694" width="708" alt="Fig.Â 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>BPTF Inhibition diminishes H3K4 methylation and the chromatin association of WDR5-MLL/COMPASS. <strong>A</strong> CUT&amp;RUN and RNA-seq tracks of BPTF and K/C-III in murine leukemia cells after 6 h of 2 Î¼M BZ1 treatment, showing changes in chromatin occupancy. <strong>B</strong> RNA-seq analysis of K/C-III leukemia cells treated with 2 Î¼M BZ1 for 6 or 12 h. K/C-III-bound genes are categorized into two clusters, with most genes in Cluster 1 exhibiting downregulation following BZ1 treatment. <strong>C</strong>, <strong>D</strong> Box plots showing K/C-III and BPTF signal intensities in K/C-III leukemia cells post-BZ1 treatment. Cluster 1 genes display higher K/C-III and BPTF signals compared to Cluster 2 genes. <strong>E</strong> Schematic of ChIP-MNase profiling of nucleosomes using an H3K4me2 antibody. <strong>F</strong> Example ChIP-MNase track at the <em>NUP153</em> locus, illustrating nucleosome positioning. HEK293T cells expressing K/C-III were treated with 2 Î¼M BZ1 for 6 or 12 h. <strong>G</strong> ChIP-MNase mapping of nucleosome positions at promoter regions using an H3K4me2 antibody. Dashed lines indicate the positions of nucleosomes. <strong>H</strong>, <strong>I</strong> Immunoblot analysis showing a global reduction in H3K4me2 and H3K4me1 levels in murine K/C-III leukemia cells and K/C-III-expressing HEK293T cells by 2 Î¼M BZ1 treatments for 6 or 12 h. <strong>J</strong>, <strong>K</strong> ChIP-seq and CUT&amp;RUN analyses of K/C-III, BPTF, WDR5, MLL, and H3K4me1/2 in K/C-III leukemia cells after 6 h of BZ1 treatment. <strong>L</strong> Metaplot analysis of K/C-III, BPTF, WDR5, MLL, and H3K4me1/2 signals at TSS. BZ1 treatment reduces WDR5 and MLL occupancy, indicating disrupted chromatin association</p></figcaption></figure><p id="Par21">As a chromatin remodeling complex, NURF facilitates ATP-dependent nucleosome sliding to regulate chromatin accessibility and transcription [<a href="#CR37" class="usa-link" aria-describedby="CR37">37</a>, <a href="#CR38" class="usa-link" aria-describedby="CR38">38</a>]. BZ1 treatment reduced BPTF and K/C-III occupancy byâ€‰~â€‰50% (Fig.Â <a href="#Fig4" class="usa-link">4</a>L and M). To assess the functional consequences of this reduction, we performed ChIP-MNase assays targeting H3K4me2-marked promoters colocalized with BPTF and K/C-III (Fig.Â <a href="#Fig5" class="usa-link">5</a>E). Genome browser tracks revealed that BZ1 treatment significantly decreased H3K4me2 levels at the <em>NUP153</em> promoter (Fig.Â <a href="#Fig5" class="usa-link">5</a>F). However, nucleosome profiling near transcription start sites (TSS) showed no significant nucleosome repositioning. Instead, it is essential for maintaining H3K4me2 methylation levels at promoters (Fig.Â <a href="#Fig5" class="usa-link">5</a>G).</p>
<p id="Par22">Further validation in K/C-III-expressing HEK293T cells and murine leukemia cells confirmed that BZ1 treatment reduced bulk levels of H3K4me1 and H3K4me2 (Fig.Â <a href="#Fig5" class="usa-link">5</a>H and I). Given the established interactions between the NURF complex and the MLL/COMPASS complex component WDR5, as well as between KAT6A and the MLL complex [<a href="#CR34" class="usa-link" aria-describedby="CR34">34</a>, <a href="#CR39" class="usa-link" aria-describedby="CR39">39</a>], we also confirmed that KAT6A and KAT6A chimeras interacts with WDR5 and RBBP5 in our IP-MS experiments (Additional file 2: Table S2). Next, we examined WDR5 and MLL occupancy at K/C-III-bound promoters using CUT&amp;RUN (Fig.Â <a href="#Fig5" class="usa-link">5</a>J and K). Following 6Â h of BZ1 treatment, both WDR5 and MLL showed reduced binding, indicating that BPTF inhibition disrupts the recruitment of the WDR5-MLL/COMPASS complex, resulting in decreased H3K4me2 levels (Fig.Â <a href="#Fig5" class="usa-link">5</a>L).</p>
<p id="Par23">To determine whether the menin-MLL interaction is essential in KAT6A-rearranged leukemia, we evaluated the therapeutic potential of Revumenib, a clinical-stage menin-MLL inhibitor effective in MLL-rearranged AML [<a href="#CR40" class="usa-link" aria-describedby="CR40">40</a>]. KAT6A fusion-expressing leukemic cells treated with Revumenib (72 h) showed limited reduction in viability compared to control HSPCs (Additional file 1: Fig. S7D and E). Notably, high-dose Revumenib (6Â Î¼M) failed to induce cell cycle arrest (Additional file 1: Fig. S7F and G). These results suggest that MLL/COMPASS dependency in KAT6A-fusion leukemias may bypass the menin-MLL interaction, and underscores a distinct self-reinforcing mechanism whereby the NURF complex facilitates the recruitment of the WDR5-MLL/COMPASS complex to sustain H3K4 methylation. This epigenetic landscape likely enhances the occupancy of both NURF and KAT6A, supporting transcriptional regulation critical for leukemogenesis.</p></section><section id="Sec8"><h3 class="pmc_sec_title">CBP/P300 Inhibition Reduces Chromatin Accessibility and Recruitment of the KAT6A-NURF-MLL Module</h3>
<p id="Par24">To determine whether CBP/P300 HAT activity is essential for leukemia cell maintenance, we utilized the selective acetyltransferase inhibitor A485 [<a href="#CR19" class="usa-link" aria-describedby="CR19">19</a>]. Mass spectrometry analysis of post-translational modifications showed that A485 treatment of K/C-III-expressing HEK293T cells for 6 and 12Â h rapidly reduced multiple histone acetylation marks, including H3K27ac, H3K79ac, H4K5ac, H4K8ac, H4K12ac, H4K16ac, H2AK5ac, H2AK9ac, H2AK4ac, H2AK7ac, and H2AK11ac (Fig.Â <a href="#Fig6" class="usa-link">6</a>A and B). Notably, H4K16ac, a mark critical for interaction with BPTF to facilitate chromatin occupancy, was markedly affected. Immunoblotting confirmed decreases in H3K27ac and H4K16ac in K/C-III-expressing HEK293T cells and K/C-III leukemia cells upon A485 treatment (Fig.Â <a href="#Fig6" class="usa-link">6</a>C and D).</p>
<figure class="fig xbox font-sm" id="Fig6"><h4 class="obj_head">Fig.Â 6.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12366150_13059_2025_3743_Fig6_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6f0e/12366150/0406536989e8/13059_2025_3743_Fig6_HTML.jpg" loading="lazy" id="MO6" height="991" width="708" alt="Fig.Â 6"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>A485 disrupts histone acetylation and chromatin accessibility critical for the KAT6A-NURF-MLL moduleâ€™s chromatin association. <strong>A</strong> Schematic illustrating the experimental workflow for profiling histone modifications in K/C-III 293â€‰T cells following treatment of CBP/P300 acetyltransferase inhibitors A485 [<a href="#CR19" class="usa-link" aria-describedby="CR19">19</a>]. K/C-III 293â€‰T cells were treated with 2 Î¼M A485 for 6 or 12 h. <strong>B</strong> Heatmap showing a marked reduction in multiple histone acetylation marks in response to A485 treatments. <strong>C</strong>, <strong>D</strong> Immunoblot analysis confirms significant decreases in H3K27ac and H4K16ac levels in K/C-III 293â€‰T cells and leukemia cells treated with 2Î¼M A485 for 6 or 12 h. <strong>E</strong>, <strong>F</strong> A485 induces differentiation in K/C and K/P leukemia cells, as demonstrated by Wright-Giemsa staining and reduced expression of CD34 and c-Kit. <strong>G</strong> ATAC-seq, ChIP-seq, and CUT&amp;RUN analyses of Pol II, K/C-III, BPTF, MLL, and H3K27ac in K/C-III leukemia cells after 6 h of 2 Î¼M A485 treatment, showing reduced chromatin association at the <em>Nup153</em> and <em>Hoxa</em> cluster. <strong>H</strong> Metaplot analysis of ATAC-seq, Pol II, K/C-III, BPTF, MLL, and H3K27ac signals at promoters occupied by K/C-III. A485 treatment decreases chromatin accessibility, Pol II recruitment, and the chromatin association of the KAT6A-NURF-MLL module. <strong>I</strong> RNA-seq analysis of K/C-III leukemia cells treated with 2 Î¼M A485 for 6 and 12 h. A485 downregulates genes involved in hematopoiesis, cell proliferation, and MYC activity pathways, while upregulating genes related to interferon signaling, cellular catabolic processes, and actin filament-based processes. <strong>J</strong> Identification of K/C core target genes. These 770 genes are bound by K/C-III and sensitive to A485 treatments. <strong>K</strong> GO analysis of these 770 core target genes revealed enrichment in leukemia stem cell maintenance, hemopoiesis, negative regulation of cell differentiation, and cell proliferation</p></figcaption></figure><p id="Par25">Since A485 inhibits both endogenous CBP/P300 and the K/C fusion, we tested A485 in HEK293T isogenic cells to disentangle the contribution of K/C to A485. We found cells expressing K/C-III exhibited elevated H3K27ac levels compared to KAT6A-NT1 or MSCV controls. Short-term A485 treatment (6Â h) preferentially reduced H3K27ac and H4K16ac marks in K/C-III cells (Additional file 1: Fig. S8A), suggesting rapid inhibition of fusion-dependent HAT activity. Doseâ€“response assays further revealed aÂ twofold lower IC50Â for A485 in K/C-III cells (2.90Â Î¼M) compared to KAT6A-NT1 (5.37Â Î¼M) or MSCV controls (5.99Â Î¼M; Additional file 1: Fig. S8B), indicating heightened sensitivity to the A485 inhibitor. To validate these findings in primary HSPCs, we found K/C-III-expressing HSPCs showed twofold greater sensitivity to A485 in viability assays (IC50â€‰=â€‰2.37Â Î¼M vs. 6.0Â Î¼M for KAT6A-NT; Additional file 1: Fig. S8C). These results collectively suggest that K/C expression sensitizes cells to A485, likely due to the fusionâ€™s dependence on CBP/P300 HAT activity. However, residual effects of A485 in control cells (e.g., MSCV, KAT6A-NT1) indicate that endogenous CBP/P300 inhibition may contribute to the phenotype at higher doses.</p>
<p id="Par26">A485 treatment induced differentiation in leukemia cells, as evidenced by Giemsa staining, which revealed morphological changes starting from 4Â days of treatment (Fig.Â <a href="#Fig6" class="usa-link">6</a>E and F). Flow cytometry analysis showed reduced surface expression of c-Kit and CD34, markers of leukemic stemness, and similar differentiation effects were observed in KAT6A-TIF2 and KAT6A-NCOA3 leukemia cells, underscoring the broad impact of CBP/P300 inhibition (Additional file 1: Fig. S8D and E). To further explore the impact of CBP/P300 inhibition on chromatin dynamics, ATAC-seq was performed to assess chromatin accessibility [<a href="#CR41" class="usa-link" aria-describedby="CR41">41</a>]. Analysis of key factors, including Pol II, H3K27ac, BPTF, K/C, and MLL, in K/C-III leukemia cells revealed that A485 treatment for 6Â h significantly decreased chromatin accessibility, Pol II loading, and the recruitment of the KAT6A-NURF-MLL module at <em>Nup153</em> and <em>Hoxa</em> cluster loci (Fig.Â <a href="#Fig6" class="usa-link">6</a>G; Additional file 1: Fig. S8F). Genome-wide analyses confirmed similar reductions at all K/C-III-occupied promoter regions (Fig.Â <a href="#Fig6" class="usa-link">6</a>H), indicating that CBP/P300 inhibition broadly disrupts chromatin accessibility and the recruitment of the KAT6A-NURF-MLL epigenetic module.</p>
<p id="Par27">RNA-seq analysis of A485-treated K/C-III leukemia cells further revealed significant downregulation of key hematopoietic factors, including <em>Hoxa9</em>, <em>Hoxa10</em>, <em>Meis1</em>, <em>Cd34</em>, <em>c-Kit</em>, and <em>Csf1r</em> (Fig.Â <a href="#Fig6" class="usa-link">6</a>I; Additional file 2: Table S4). To define the transcriptional networks directly regulated by KAT6A fusions, we integrated K/C-III CUT&amp;RUN and RNA-seq datasets, and identified the core target genes of KAT6A fusion meeting two criteria: 1) Direct binding: occupied by KAT6A fusions in CUT&amp;RUN data, 2) Transcriptional downregulation by A485. This integrative analysis identified 770 core target genes (Fig.Â <a href="#Fig6" class="usa-link">6</a>J). K/C-III co-localizes with MLL and BPTF at these 770 core target genes (Fig.Â <a href="#Fig6" class="usa-link">6</a>J), indicating cooperative interactions within the epigenetic module to amplify oncogenic transcription. GO analysis of these 770 core target genes revealed enrichment in leukemia stem cell maintenance (e.g., <em>Hoxa9, Meis1, Flt3</em>), hemopoiesis (e.g., <em>Runx1, Runx2, Cd34, Kit</em>), negative regulation of cell differentiation (e.g., <em>Gata2, Ldb1, Ptk2b</em>), and cell proliferation (e.g., <em>Ccnd1, Cdkn2c</em>) (Fig.Â <a href="#Fig6" class="usa-link">6</a>K; Additional file 2: Table S5), which is consistent with oncogenic potential of KAT6A fusions.</p>
<p id="Par28">Furthermore, to clarify why BZ1 causes cell cycle arrest but not differentiation (unlike A485), we analyzed the A485-downregulated genes and found that BPTF inhibition by BZ1 only slightly reduced occupancy of K/C-III, BPTF, and MLL, whereas A485 treatment resulted in a substantial reduction of these signals (Additional file 1: Fig. S8G and H). Similarly, BPTF inhibition via BZ1 did not significantly decrease chromatin accessibility or RNA Pol II occupancy at these loci, whereas A485 caused significant reductions (Additional file 1: Fig. S8G and H).</p></section><section id="Sec9"><h3 class="pmc_sec_title">Therapeutic Targeting of the KAT6A-NURF-MLL Module Delays K/C Leukemia Progression</h3>
<p id="Par29">Building on the inhibitory effects of BZ1 in KAT6A-rearranged leukemia cells, we evaluated the therapeutic potential of targeting BPTF in the K/C leukemia model. BPTF was depleted in K/C-III leukemia cells using shRNA, and 2â€‰Ã—â€‰10<sup>5</sup> cells were transplanted into recipient mice for secondary transplantation (Fig.Â <a href="#Fig7" class="usa-link">7</a>A). Disease monitoring revealed that BPTF knockdown significantly delayed leukemia progression compared to GFP-shRNA (shGFP) controls. By day 52 post-transplantation, mice receiving shGFP cells displayed advanced leukemia symptoms, including splenomegaly and pale bone marrow (Fig.Â <a href="#Fig7" class="usa-link">7</a>B). In contrast, recipients of BPTF-depleted cells exhibited no such abnormalities. Quantitative analysis showed a marked enrichment of HaloTag-positive leukemia cells in the shGFP group, which was nearly absent in the BPTF knockdown group (Fig.Â <a href="#Fig7" class="usa-link">7</a>C). Histological analysis confirmed disrupted splenic architecture in GFP-shRNA mice, while spleens from BPTF-depleted mice retained normal morphology (Fig.Â <a href="#Fig7" class="usa-link">7</a>D). BPTF depletion also reduced colony formation in KAT6A-NCOA3A and KAT6A-TIF2 leukemic cells versus controls (Additional file 1: Fig. S9A and B), confirming that NURF dependency is conserved across KAT6A fusion subtypes. These results underscore the essential role of BPTF in K/C leukemia progression and its potential as a therapeutic target.</p>
<figure class="fig xbox font-sm" id="Fig7"><h4 class="obj_head">Fig.Â 7.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12366150_13059_2025_3743_Fig7_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6f0e/12366150/cedfd8919a1b/13059_2025_3743_Fig7_HTML.jpg" loading="lazy" id="MO7" height="629" width="708" alt="Fig.Â 7"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig7/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Therapeutic targeting of KAT6A-rearranged leukemia with BPTF and CBP/P300 inhibitors <strong>A</strong> Kaplanâ€“Meier survival analysis of mice injected with K/C-III leukemic splenocytes with or without <em>Bptf</em> knockdown. <em>Bptf</em> depletion significantly extends survival (<em>n</em>â€‰=â€‰5â€“6 mice per group; log-rank test). <strong>B</strong> Endpoint analysis shows reduced leukemia progression in recipient mice with <em>Bptf</em> knockdown. <strong>C</strong> Quantification of HaloTag<sup>+</sup> leukemia cells in the spleen of recipient mice demonstrates a reduced leukemia burden in the <em>Bptf</em>-depleted group. <strong>D</strong> H&amp;E-stained spleen Sects.Â (20â€‰Ã—â€‰magnification) reveal preserved spleen architecture in mice with <em>Bptf</em> knockdown. <strong>E</strong> In vivo treatment model for K/C-III leukemia using BZ1 and A485. Mice were treated with vehicle, BZ1 (50Â mg/kg), A485 (50Â mg/kg), or their combination via intraperitoneal injections for 10 times. Kaplanâ€“Meier survival curves indicate prolonged survival in all treatment groups, with the combination therapy showing the most pronounced effect (log-rank test). <strong>F</strong>, <strong>G</strong> Post-treatment leukemia burden analysis: (F) Percentage of HaloTag<sup>+</sup> leukemia cells in peripheral blood and (G) spleen measured on day 24 post-engraftment, showing reduced leukemia burden with BZ1, A485, and combination treatments. <strong>H</strong> Body weight monitoring of K/C-III leukemia mice during treatment. BZ1 (50 mg/kg), A485 (50 mg/kg), and combination treatment were well-tolerated, with no significant weight loss observed. <strong>I</strong> Proposed model for the self-reinforcing KAT6A-NURF-MLL module in KAT6A-rearranged leukemia. KAT6A associates preferentially with H3K4me2/3-marked chromatin and interacts with the NURF complex, a reader of H3K4me2/3. This interaction enhances KAT6Aâ€™s chromatin association, while the NURF complex further interacts with the MLL/COMPASS complex to sustain H3K4 methylation. CBP/P300-mediated histone acetylation enhances chromatin accessibility, enhancing the KAT6A-NURF-MLL module at target genomic loci</p></figcaption></figure><p id="Par30">To further explore therapeutic strategies, we evaluated the efficacy of small-molecule inhibitors targeting BPTF (BZ1) and CBP/P300 (A485) in a K/C-III secondary transplantation model (Fig.Â <a href="#Fig7" class="usa-link">7</a>E). Mice were injected with 8â€‰Ã—â€‰10<sup>5</sup> leukemia cells and treated intraperitoneally with vehicle (corn oil), BZ1 (50Â mg/kg), A485 (50Â mg/kg), or a combination of both inhibitors. Kaplanâ€“Meier survival analysis demonstrated that both BZ1 and A485 prolonged survival (Fig.Â <a href="#Fig7" class="usa-link">7</a>E), reduced peripheral blood leukemic blasts (Fig.Â <a href="#Fig7" class="usa-link">7</a>F), and decreased spleen volumes by day 24 after tail veil injection (Fig.Â <a href="#Fig7" class="usa-link">7</a>G). Notably, the combination treatment exhibited superior efficacy in delaying leukemia progression compared to either treatment alone (Fig.Â <a href="#Fig7" class="usa-link">7</a>E). Importantly, all treatments were well-tolerated, with no significant body weight loss observed (Fig.Â <a href="#Fig7" class="usa-link">7</a>H).</p>
<p id="Par31">To determine whether their combined effects are synergistic or additive, we conducted co-treatment with BZ1 (0.5â€“2Â Î¼M) and A-485 (0.5â€“1Â Î¼M) in K/C-III leukemic cells, which yielded a Bliss synergy score ofâ€‰âˆ’â€‰0.11 (Additional file 1: Fig. S9C), indicating additive effects (score â‰ˆ 0) rather than synergy (scoreâ€‰&gt;â€‰10). Besides, BZ1 co-treatment did not further reduce the population of c-Kit<sup>+</sup>CD34<sup>+</sup> leukemic cells compared to A485 treatment alone (Additional file 1: Fig. S9D), suggesting no enhanced differentiation effect. Moreover, RNA-seq of K/C-III leukemic cells treated with low-dose BZ1 (1Â Î¼M) and A485 (0.5Â Î¼M) revealed that co-treated cells shared most of the downregulated genes with A485-treated cells and no unique pathways or gene sets were specifically suppressed in co-treated cells (Additional file 1: Fig. S9E). These data demonstrate that BZ1 and A485 exert additive, not synergistic, antileukemic effects in K/C-III leukemic cells. The absence of unique transcriptional programs or enhanced differentiation in co-treated cells may reflect their shared mechanistic targeting of the KAT6A-NURF-MLL module. Furthermore, we evaluated the therapeutic potential of BZ1 and A485 in common non-KAT6A-rearranged human leukemia cell lines and found that that BZ1 and A485 do not broadly inhibit leukemia cell proliferation but exhibit selective efficacy (Additional file 1: Fig. S10A and B). Together, these findings underscore the critical oncogenic dependency of KAT6A-rearranged leukemia on BPTF and CBP/P300. They validate these factors as promising therapeutic targets, with the combination approach offering enhanced efficacy for treating this rare but aggressive leukemia subtype.</p></section></section><section id="Sec10"><h2 class="pmc_sec_title">Discussion</h2>
<p id="Par32">This study addresses the longstanding challenge of studying large oncogenic fusion proteins by employing a domain-focused truncation strategy. This innovative approach enabled the generation of robust murine AML models for K/C and K/P fusions, faithfully recapitulating the transcriptomic, morphological, and immunophenotypic characteristics of KAT6A-rearranged AML. Beyond their immediate relevance to KAT6A fusions, these models offer a scalable framework for investigating other large oncogenic fusions, broadening therapeutic insights across diverse AML subtypes. We demonstrate that KAT6A preferentially associates with H3K4me2/3-marked chromatin and interacts with the NURF complex, a known reader of these marks [<a href="#CR35" class="usa-link" aria-describedby="CR35">35</a>, <a href="#CR42" class="usa-link" aria-describedby="CR42">42</a>]. This interaction enhances KAT6Aâ€™s chromatin association and works with the MLL/COMPASS complex to sustain H3K4 methylation. Additionally, CBP/P300 catalyzes histone acetylation at multiple sites, including H4K16ac and H3K27ac, thereby enhancing chromatin accessibility and facilitating the recruitment of the KAT6A-NURF-MLL module to targeted genomic loci. Collectively, these findings define a critical self-reinforcing module that underpins the leukemogenic activity of K/C and K/P fusions (Fig.Â <a href="#Fig7" class="usa-link">7</a>I).</p>
<p id="Par33">KAT6A, a critical acetyltransferase, operates within protein complexes including BRPF1, ING4/5, and MEAF6, playing pivotal roles in development and hematopoiesis [<a href="#CR10" class="usa-link" aria-describedby="CR10">10</a>, <a href="#CR13" class="usa-link" aria-describedby="CR13">13</a>, <a href="#CR43" class="usa-link" aria-describedby="CR43">43</a>]. Interestingly, while KAT6Aâ€™s catalytic activity supports acetylation of marks like H3K9ac, H3K23ac, and H3K23 propionylation [<a href="#CR44" class="usa-link" aria-describedby="CR44">44</a>, <a href="#CR45" class="usa-link" aria-describedby="CR45">45</a>], deletion of the MYST domain, but not its catalytic activity, abolishes colony formation and leukemogenesis in mouse AML models (21). Consistent with this, our results demonstrate that catalytically inactive KAT6A retains genomic occupancy comparable to its wild-type form. This underscores that KAT6Aâ€™s chromatin-binding ability is independent of its enzymatic activity. Furthermore, we identify the NURF complex as a key interactor of the KAT6A MYST domain, facilitating its genomic occupancy. This interaction underscores the critical role of structural partnerships over KAT6A enzymatic activity in leukemogenesis, unveiling a KAT6A catalytic-independent mechanism through which KAT6A fusions contributes to AML pathogenesis.</p>
<p id="Par34">Beyond serving as an H3K4me2/3 reader, the NURF complex facilitates transcriptional regulation by catalyzing nucleosome sliding at promoters, thereby enhancing transcriptional access [<a href="#CR37" class="usa-link" aria-describedby="CR37">37</a>, <a href="#CR46" class="usa-link" aria-describedby="CR46">46</a>, <a href="#CR47" class="usa-link" aria-describedby="CR47">47</a>]. Disrupting this process using the small-molecule inhibitor BZ1 reduces BPTF-chromatin interactions byâ€‰~â€‰50%, yet ChIP-MNase and ATAC-seq analyses reveal no significant changes in nucleosome positioning and chromatin accessibility at promoters. These findings suggest that partial NURF disruption is insufficient to induce detectable nucleosome repositioning, highlighting the resilience of chromatin architecture. These observations illuminate the nuanced role of NURF in maintaining the oncogenic epigenetic landscape in AML and emphasize the need for further exploration into the thresholds and dynamics of NURF activity. The partial displacement of the fusion by BPTF inhibition may disrupt proliferative programs but retains residual fusion activity sufficient to maintain the differentiation blockade. In contrast, A485 could directly inhibits the fusionâ€™s enzymatic activity, eliminating both proliferation- and differentiation-blocking signals, thereby enabling cell differentiation.</p>
<p id="Par35">In contrast to KAT6Aâ€™s acetylation activity, CBP/P300 HAT activity is indeed indispensable for KAT6A-rearranged AML. CBP/P300 catalyzes histone acetylation at multiple sites, contributing to a dynamic and extensive acetylome [<a href="#CR19" class="usa-link" aria-describedby="CR19">19</a>, <a href="#CR48" class="usa-link" aria-describedby="CR48">48</a>]. Acetylation marks likely enhance chromatin accessibility, facilitating RNA Pol II loading and the recruitment of the KAT6A-NURF-MLL module to chromatin. Besides, the bromodomain of BPTF, a core component of the NURF complex, specifically recognizes H4K16ac, while its PHD domain binds H3K4me2/3 [<a href="#CR35" class="usa-link" aria-describedby="CR35">35</a>, <a href="#CR36" class="usa-link" aria-describedby="CR36">36</a>, <a href="#CR49" class="usa-link" aria-describedby="CR49">49</a>]. These interactions establish a unique trans-histone modification pattern within a single nucleosome, where BPTFâ€™s bivalent binding co-localizes with these marks across the genome [<a href="#CR49" class="usa-link" aria-describedby="CR49">49</a>]. Such co-localization suggests a coordinated mechanism whereby distinct histone modifications converge to regulate chromatin dynamics and selectively recruit the KAT6A-NURF-MLL module. This interplay likely enhances the genomic specificity of KAT6A fusions through the combined actions of histone reader proteins and modifying enzymes. These findings provide deeper insights into how chromatin architecture and histone modifications orchestrate oncogenic transcriptional programs, while also uncovering critical dependencies that may serve as therapeutic vulnerabilities in KAT6A-rearranged AML.</p>
<p id="Par36">Our findings address a critical challenge in treating KAT6A-rearranged AML, a high-risk subtype with poor prognosis [<a href="#CR7" class="usa-link" aria-describedby="CR7">7</a>], by highlighting the therapeutic potential of targeting BPTF and CBP/P300 activity. BZ1, a BPTF-H4K16ac inhibitor, induces cell cycle arrest and suppresses leukemia cell proliferation but exhibits weaker efficacy than BPTF knockdown, highlighting the need for further optimization. In contrast, the acetyltransferase inhibitor A485 effectively promotes leukemia cell differentiation. The differential effects of BZ1 and A485 may stem from A485â€™s inhibition of CBP/P300 acetyltransferase activity, leading to a rapid and dramatic reduction in chromatin accessibility, or its stronger suppression of K/C-III, BPTF, and MLL occupancy. Notably, the combination of A485 and BZ1 yields superior therapeutic outcomes, emphasizing the potential of dual targeting within the KAT6A-NURF-MLL module for KAT6A-rearranged AML. Future studies should evaluate the therapeutic efficacy of targeting this moduleâ€”both as single agents and in combination therapiesâ€”in primary AML patient-derived samples to advance their clinical potential. Moreover, we acknowledge that our animal models are based on overexpression of truncates of KAT6A chimeras, which may lead KAT6A chimeras to bind sites that might not be occupied in a more physiological setting or potentially exclude critical genomic binding sites or protein interactions. Collectively, by elucidating the interplay between chromatin dynamics, histone modifications, and transcriptional regulation in KAT6A-rearranged AML, this study provides a foundation for innovative therapeutic strategies targeting epigenetic vulnerabilities in this rare but aggressive leukemia subtype.</p></section><section id="Sec11"><h2 class="pmc_sec_title">Conclusions</h2>
<p id="Par37">In summary, we established murine leukemia models for large KAT6A fusions and characterized their morphological, immunophenotypic, and transcriptomic features. We found that KAT6A chimeras colocalize with H3K4me2/3-marked regions and interact with the H3K4me2/3 reader complex NURF. Furthermore, we identified a self-reinforcing epigenetic module involving KAT6A chimeras, NURF, and MLL/COMPASS, which sustains leukemogenic transcriptional programs through H3K4me2 deposition and histone acetylation. Targeting this module via NURF or CBP/P300 inhibition demonstrates efficacy in K/C leukemia models, with enhanced therapeutic effects when combined. More importantly, our findings establish robust models for investigating disease mechanisms and therapeutic strategies, offering broader implications for epigenetics and blood cancers.</p></section><section id="Sec12"><h2 class="pmc_sec_title">Methods</h2>
<section id="Sec13"><h3 class="pmc_sec_title">Cell culture and DNA construction</h3>
<p id="Par38">HEK293T (ATCC CRL-3216) and Mouse Embryonic Fibroblast (MEF; ATCC CRL-2752) cells were cultured in high-glucose DMEM (Life Technologies) supplemented with 10% FBS (LONSERA S711-001), 1% penicillin/streptomycin (Gibco), and L-glutamine (Gibco) at 37Â Â°C in a 5% CO<sub>2</sub> atmosphere. Drosophila S2 cells (Gibco <a href="https://www.ncbi.nlm.nih.gov/nuccore/R69007" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">R69007</a>) from Invitrogen were maintained in Schneider's medium (Sigma-Aldrich) at 25Â Â°C. Cell lines were routinely tested for mycoplasma contamination using MycoAlert (Lonza) and authenticated by DNA fingerprinting (ATCC). Primary murine HSPCs, isolated using CD117 MicroBeads, and K/C-III and K/P-III leukemia cells were cultured in Retronectin-coated plates (Takata T100A) with RPMI 1640 medium (Gibco) supplemented with 1% penicillin/streptomycin, 10% FBS (Gibco), and murine SCF, IL-6, and IL-3. Cell viability was assessed with a Bio-Rad TC20 automated cell counter.</p>
<p id="Par39">KAT6A truncates, K/C, and K/P fusion constructs were cloned into the MSCV retroviral vector (Addgene #68,469), incorporating N-terminal Flag and HaloTag tags for detection and purification purposes. Detailed sequences of these constructs are provided in Additional file 2: Table S6. For shRNA targeting human or mouse <em>Bptf</em>, oligonucleotides were designed and cloned into the PLKO shRNA vector. The sequences for the shRNA constructs are listed in Additional file 2: Table S7.</p></section><section id="Sec14"><h3 class="pmc_sec_title">Generation of Antibodies</h3>
<p id="Par40">Antibodies were generated by immunizing New Zealand White rabbits with recombinant proteins. Anti-BPTF serum was produced by immunizing rabbits with a BPTF fragment (residues 121â€“340) purified from <em>E. coli</em> BL21 cells. Similarly, anti-HaloTag serum was raised using recombinant HaloTag protein purified from <em>E. coli</em> BL21 cells. The serum was subsequently aliquoted and stored at âˆ’20Â Â°C or lower for long-term use. Commercial antibodies were listed in Additional file 2: Table S8.</p></section><section id="Sec15"><h3 class="pmc_sec_title">Isolation of c-Kit-enriched HSPCs</h3>
<p id="Par41">Bone marrow was collected from the femurs and tibias of 6-week-old female C57BL/6Â J mice. Cells were flushed from the bones with ice-cold PBS using a 25-gauge needle and syringe. The suspension was filtered through a 40-Âµm nylon mesh cell strainer to remove debris and aggregates. Red blood cells were lysed with Erythrocyte Lysate for 2Â min at 4Â Â°C. HSPCs were enriched using CD117 MicroBeads Kit (Miltenyi Biotec) and magnetic-activated cell sorting. Cells were incubated with CD117 beads for 15Â min at 4Â Â°C with gentle rotation, followed by magnetic separation using a Miltenyi Biotec column per the manufacturerâ€™s instructions. Isolated HSPCs were either used for transcriptome analysis or cultured.</p></section><section id="Sec16"><h3 class="pmc_sec_title">Retrovirus preparation and HSPCs transduction</h3>
<p id="Par42">HEK293T cells were transfected with an MSCV retroviral vector containing various HaloTag-KAT6A fusions, along with the pCL-Eco retrovirus packaging vector (Addgene #12,371), using Polyethyleneimine. After 48Â h, the retroviral supernatant was collected, filtered through a 0.45-Âµm filter, and concentrated by centrifugation at 18,000Â rpm for 90Â min at 4Â Â°C in a Himac CR21N rotor using Beckman Ultra-Clear tubes. The viral pellet was used immediately for transduction. c-Kit-enriched HSPCs were cultured on RetroNectin-coated plates (Takata T100A) and underwent spinoculation at 1,000Â g at 37Â Â°C for 60Â min, then incubated for 24Â h. After spinoculation, cells were resuspended in fresh cytokine-supplemented medium. Transduction efficiency was assessed by flow cytometry (Beckman Coulter CytoFLEX S) to detect HaloTag-tagged protein expression.</p></section><section id="Sec17"><h3 class="pmc_sec_title">Murine K/C and K/P leukemia models</h3>
<p id="Par43">Transduced HSPCs (5â€‰Ã—â€‰10<sup>6</sup> cells) were intravenously injected into the tail vein of primary recipient female C57BL/6 mice (8Â weeks old) that had been previously irradiated with 900Â cGy. The mice were monitored closely for signs of acute leukemia, including weight loss, reduced mobility, malaise, splenomegaly, and a hunched posture. Upon observing clear symptoms of leukemia, the mice were euthanized, and their spleens and bone marrow were harvested for subsequent analyses. Spleen homogenates were cultured in RPMI 1640 supplemented with 1% penicillin/streptomycin, 10% FBS, murine SCF, IL-6, and IL-3. For secondary transplantation, 2â€‰Ã—â€‰10<sup>5</sup> leukemia cells from primary recipients were injected into sublethally irradiated (450Â cGy) female C57BL/6 mice (8â€“10Â weeks old). Kaplanâ€“Meier survival analysis was conducted using the log-rank test in GraphPad Prism.</p></section><section id="Sec18"><h3 class="pmc_sec_title">Giemsa-Wright staining and H&amp;E staining</h3>
<p id="Par44">Wright-Giemsa staining was performed on murine AML cells harvested from the bone marrow of recipient mice. Blast cells and differentiated leukemia cells were visualized and photographed using a ZEISS Axio Imager 2 microscope with a 40â€‰Ã—â€‰objective lens. For histological examination, mouse spleens were dissected, fixed in 4% paraformaldehyde (PFA), and embedded in paraffin. Paraffin-embedded spleen sections were stained with hematoxylin and eosin (H&amp;E) and analyzed for AML cell infiltration.</p></section><section id="Sec19"><h3 class="pmc_sec_title">RT-qPCR</h3>
<p id="Par45">Murine MP cells (Lin<sup>âˆ’</sup>c-Kit<sup>+</sup>) were sorted by flow cytometry. RNA was extracted from murine leukemia cells, MP cells, or bone marrow cells and reverse transcribed using ReverTra Aceâ„¢ qPCR RT Master Mix (Toyobo). RTâ€’qPCR was conducted using SYBR Green Master Mix (Vazyme). The sequences of primers used are provided in Additional file 2: Table S9.</p></section><section id="Sec20"><h3 class="pmc_sec_title">In Vivo Treatment with BPTF and CBP/P300 Inhibitors</h3>
<p id="Par46">To evaluate the therapeutic effects of BPTF and CBP/P300 inhibitors, 8â€‰Ã—â€‰10<sup>5</sup>Â K/C-III leukemia cells were transplanted into sublethally irradiated (450Â cGy) 8-week-old female C57BL/6 mice. BPTF inhibitor BZ1 (MCE HY-132889) and CBP/P300 inhibitor A485 (Aladdin A287597) were dissolved in corn oil. Nine days post-transplantation, mice received intraperitoneal injections of either vehicle, 50Â mg/kg BZ1, 50Â mg/kg A485, or both inhibitors for 10 times. Body weights and overall survival rates of the recipient mice were monitored and compared across treatment groups.</p></section><section id="Sec21"><h3 class="pmc_sec_title">Spectral Flow Cytometry</h3>
<p id="Par47">Spectral flow cytometry was conducted using an adapted protocol [<a href="#CR33" class="usa-link" aria-describedby="CR33">33</a>]. The details of antibodies are provided in Additional file 2: Table S8. Data acquisition was performed on a 4-laser Cytek Aurora spectral flow cytometer, capturing approximately 1â€‰Ã—â€‰10<sup>7</sup> events. SpectroFlo software (Cytek Biosciences) was used for analysis, employing ordinary least squares linear unmixing. Data visualization was carried out using the OMIQ cytometry analysis platform. For lineage-specific marker analysis, leukemia cells were incubated with individual antibodies targeting lineage markers (Ter-119, Mac1, Gr-1, B220, CD5, CD3, CD4, and CD8), stained with F(ab')2-IgG, detected via flow cytometry, and analyzed with FlowJo software to generate histograms.</p></section><section id="Sec22"><h3 class="pmc_sec_title">Analysis for clinical data</h3>
<p id="Par48">For the analysis of KAT6A-rearranged AML patients, RNA expression data were extracted from the TCGA-LAML database (<a href="https://portal.gdc.cancer.gov" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://portal.gdc.cancer.gov</a>). AML patients were classified into two subgroups based on KAT6A rearrangement or normal karyotypes based on clinical profiles from the University of California, Santa Cruz (UCSC) Xena project (TCGA) Acute Myeloid Leukemia (LAML), <a href="http://xena.ucsc.edu" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://xena.ucsc.edu</a>). To investigate the gene expression signatures associated with KAT6A rearrangement, upregulated signature genes were obtained from a previously published study [<a href="#CR8" class="usa-link" aria-describedby="CR8">8</a>]. GSEA was then applied to test the expression of this signature in the two patient groups, allowing for comparison between KAT6A-rearranged and normal karyotype AML patients.</p></section><section id="Sec23"><h3 class="pmc_sec_title">CFU, proliferation, and cell cycle analysis</h3>
<p id="Par49">For the CFU assay, 1â€‰Ã—â€‰10<sup>4</sup> leukemic cells were cultured in M3434 medium in 12-well plates. After 7Â days of incubation, colonies containing more than 10 cells were counted using an inverted tissue culture microscope. In the proliferation assay, 1â€‰Ã—â€‰10<sup>5</sup> leukemic cells from murine models were seeded into 96-well plates and treated with DMSO or the BPTF inhibitor for 4Â days. Cell viability was assessed using the Promega CellTiter-GloÂ® Luminescent Cell Viability Assay according to the manufacturer's protocol. Bliss synergy score was calculated by SynergyFinder web version 3.0 (<a href="https://synergyfinder.fimm.fi" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://synergyfinder.fimm.fi</a>). For cell cycle analysis, leukemia cells were harvested, washed once with pre-cooled PBS, and fixed overnight in pre-cooled 70% ethanol. The fixed cells were stained with a propidium iodide solution containing 10% Triton X-100, 10Â mg/mL RNase A, and 1Â mg/mL propidium iodide. Following a 15-min incubation at 37Â Â°C, cell cycle distribution was analyzed using a CytoFLEX S flow cytometer.</p></section><section id="Sec24"><h3 class="pmc_sec_title">Cell differentiation assays</h3>
<p id="Par50">5â€‰Ã—â€‰10<sup>6</sup> splenocytes from leukemia mice were treated with DMSO or A485 (1Â Î¼M) for one week. For Giemsa-Wright staining, 1â€‰Ã—â€‰10<sup>6</sup> cells were seeded on coverslips and photographed using a ZEISS Axio Imager 2 microscope. For flow cytometry analysis, cells were washed with PBS and stained with Biolegend antibodies (Lin, CD127, Sca1, CD34, c-Kit, FcÎ³R), and analyzed by spectral flow cytometry. The L-GMP cell marker profile was used as CD127<sup>âˆ’</sup>Sca1<sup>âˆ’</sup>CD34<sup>+</sup>c-Kit<sup>+</sup>FcÎ³R<sup>+</sup>Lin<sup>âˆ’</sup>.</p></section><section id="Sec25"><h3 class="pmc_sec_title">IP-MS</h3>
<p id="Par51">HEK293T cells transfected with either KAT6A N-terminus or KAT6A fusion constructs were grown in five 150Â mm dishes. After harvesting, the cells were resuspended in Dignam buffer (10Â mM Trisâ€“HCl pH 7.9, 1.5Â mM MgCl<sub>2</sub>, 10Â mM KCl, with protease inhibitors) and washed once in the same buffer. The cell pellets were centrifuged at 600Â g for 5Â min at 4Â Â°C, and the supernatant was discarded. Cells were then resuspended in MRIPA buffer (25Â mM Trisâ€“HCl, pH 7.4, 1% NP-40, 0.25% Na-deoxycholate, 300Â mM NaCl, 5% glycerol) with Benzonase and MgCl<sub>2</sub> to lyse the nuclei at 4Â Â°C for 30Â min. Following nuclear lysis, the lysates were centrifuged at 13,500Â rpm for 30Â min at 4Â Â°C. The supernatants were incubated overnight at 4Â Â°C with anti-FLAG magnetic beads for protein capture. The bead-protein complexes were then washed three times with MRIPA buffer. Purified proteins were eluted with 0.1Â M glycine (pH 2.0), with 10% of the eluted samples analyzed by silver staining.</p>
<p id="Par52">The remaining protein samples were precipitated by adding pre-cooled acetone at âˆ’20Â Â°C overnight. The samples were then alkylated with iodoacetamide and digested with trypsin (Promega V528A) for 16Â h at 37Â Â°C. Digestion was stopped with 10% TFA, and peptides were desalted using C18 StageTips. Mass spectrometry analysis was conducted using an Orbitrap Exploris 480 equipped with a FAIMS Pro interface. Data processing was performed with Proteome Discoverer 2.4, with a four-stage program. Proteins with a q-valueâ€‰â‰¤â€‰0.01 were assigned high confidence, while those with a q-valueâ€‰â‰¤â€‰0.05 were assigned medium confidence.</p></section><section id="Sec26"><h3 class="pmc_sec_title">BPTF depletion and chromatin extraction</h3>
<p id="Par53">HEK293T cells were infected with lentivirus encoding shRNA sequences targeting human BPTF (GTC TAC CAC AAT CAA TAC TC or CAG GCA ATA CTA ATG TGA AC) and subsequently selected with puromycin for two days. Cells were then harvested, and cell membrane lysis was carried out in 0.5Â mL of ice-cold E1 buffer (50Â mM HEPESâ€“KOH, pH 7.5, 140Â mM NaCl, 1Â mM EDTA, 10% glycerol, 0.5% NP-40, 0.25% Triton X-100, 1Â mM DTT, and protease inhibitors). The nuclear pellet was isolated by centrifugation and resuspended in 0.5Â mL of ice-cold E2 buffer (10Â mM Trisâ€“HCl, 200Â mM NaCl, 1Â mM EDTA, 0.5Â mM EGTA, and protease inhibitors) to further purify the nuclear fraction. After discarding the supernatant, the pellet was resuspended in 100 Î¼L of ice-cold E3 buffer (50Â mM Trisâ€“HCl, 20Â mM NaCl, 1Â mM MgCl<sub>2</sub>, 1% NP-40, and protease inhibitors). Benzonase was added at a 1:1,000 dilution to digest nucleic acids, and the solution was rotated for 30Â min at 4Â Â°C to ensure thorough digestion. Following a final centrifugation at 16,000Â g for 10Â min at 4Â Â°C, the supernatant containing the chromatin fractions was collected for further analyses.</p></section><section id="Sec27"><h3 class="pmc_sec_title">Western blot analysis</h3>
<p id="Par54">Cells or chromatin fractions were lysed in 1â€‰Ã—â€‰SDS loading buffer and boiled at 100Â Â°C for 5Â min. Equal amounts of protein from whole-cell lysates or chromatin fractions were then resolved by SDS-PAGE and transferred onto nitrocellulose membranes for immunoblotting. Band intensity was quantified using ImageJ (NIH) and normalized to histone H3 as loading controls. Uncropped Western blot and gel images were provided in Additional file 3.</p></section><section id="Sec28"><h3 class="pmc_sec_title">CUT&amp;RUN analysis</h3>
<p id="Par55">CUT&amp;RUN reactions were performed as described [<a href="#CR50" class="usa-link" aria-describedby="CR50">50</a>], following the â€œStandard CUT&amp;RUNâ€ protocol. Briefly, freshly harvested cells (1â€‰Ã—â€‰10<sup>6</sup>) were bound to CoA beads 4FF (Smart Lifesciences), resuspended in antibody binding buffer, and incubated overnight with primary antibodies. CUT&amp;RUN experiments with leukemia cells were performed using ex vivo-isolated AML cells from leukemic mice. The samples were then washed and bound to homemade pA/G-MNase. Chromatin digestion was initiated by the addition of CaCl<sub>2</sub> and stopped after 30Â min with STOP buffer containing chelating agents. Samples were incubated for 30Â min at 37Â Â°C to release CUT&amp;RUN fragments and then treated for 1Â h at 50Â Â°C with proteinase K. A purification step was performed using SRPI beads. Library preparation was done using the NEBNext Ultra II DNA Library Prep Kit for Illumina, followed by sequencing on a NovaSeq 6000 to achieve a depth of at least 10 million paired-end 150Â bp reads per sample. CUT&amp;RUN reads were aligned to the human or mouse genome with Bowtie2 with the following options: â€”very-sensitive â€”no-discordant â€”no-mixed â€“X 700 â€“dovetail. Peak calling was performed with MACS2 using IgG samples as background. Coverage signal profiles in bigwig format were generated using the bamCoverage function from deepTools with a Counts-Per-Million per-sample normalization. These profiles were visualized using deepTools computeMatrix and plotHeatmap functions.</p></section><section id="Sec29"><h3 class="pmc_sec_title">ChIP-seq</h3>
<p id="Par56">ChIP-seq was performed as previously described [<a href="#CR51" class="usa-link" aria-describedby="CR51">51</a>, <a href="#CR52" class="usa-link" aria-describedby="CR52">52</a>]. Library preparation was carried out using the NEBNext Ultra II DNA Library Prep Kit for Illumina, followed by sequencing on a NovaSeq 6000. ChIP-seq reads were aligned to the human or mouse genome using Bowtie version 1.1.2. Only uniquely mapping reads with up to two mismatches within the 150Â bp reads were considered for further analysis.</p></section><section id="Sec30"><h3 class="pmc_sec_title">ChIP-MNase</h3>
<p id="Par57">ChIP-MNase was performed following an established protocol [<a href="#CR53" class="usa-link" aria-describedby="CR53">53</a>]. Briefly, HEK293T cells expressing K/C-III were fixed with 4% formaldehyde for 10Â min at room temperature and then quenched with glycine. The fixed cells were lysed in L1 buffer (50Â mM Trisâ€“HCl pH 8.0, 2Â mM EDTA, 0.1% NP-40, 10% glycerol, 1â€‰Ã—â€‰protease inhibitors, 1Â mM PMSF, 2Â mM DTT) on ice for 5Â min. Following centrifugation, the nuclear pellet was resuspended in ice-cold wash buffer D (50Â mM Trisâ€“HCl pH 8.0, 0.1Â mM EDTA, 5Â mMÂ Mg acetate, 5Â mM DTT, 25% glycerol) and incubated for 5Â min before centrifuging at 2,000Â g for 5Â min. The pellet was resuspended in 400 Î¼L MNase buffer (10Â mM Trisâ€“HCl pH 7.4, 15Â mM NaCl, 60Â mM KCl, 1Â mM CaCl<sub>2</sub>, 250Â mM sucrose, 0.5Â mM DTT) and underwent two freezeâ€“thaw cycles in liquid nitrogen to disrupt nuclei. Chromatin was sonicated with a Diagenode Bruptor Plus (low-power mode, 8 cycles of 30Â s ON and 30Â s OFF).</p>
<p id="Par58">Next, 2 U of MNase was used to digest fixed chromatin at 25Â Â°C for 30Â min with shaking at 1200Â rpm. Digestion was halted with EDTA, and 1Â mL of buffer V (50Â mM Tris pH 7.4, 50Â mM NaCl, 5Â mM EDTA) was added. The supernatant was pre-cleaned with rProtein A/G beads (Smart Lifesciences, SA032005). Samples were incubated overnight at 4Â Â°C with 5Â Î¼g H3K4me2 antibody and 15 Î¼L rProtein A/G beads. Beads were washed three times with MNase buffer, followed by a second MNase digestion (15 Î¼L MNase buffer containing 2 U MNase) at 25Â Â°C for 30Â min with shaking. The reaction was stopped with 34Â mM EDTA and treated with proteinase K. DNA was purified by phenolâ€“chloroform extraction and ethanol precipitation, and ChIP-MNase libraries were prepared using the NEBNext Ultra II DNA library prep kit for Illumina.</p></section><section id="Sec31"><h3 class="pmc_sec_title">ATAC-seq</h3>
<p id="Par59">Standard libraries were prepared following a previously established protocol [<a href="#CR54" class="usa-link" aria-describedby="CR54">54</a>, <a href="#CR55" class="usa-link" aria-describedby="CR55">55</a>]. Briefly, 2.5â€‰Ã—â€‰10<sup>4</sup> mouse leukemia cells were treated and centrifuged at 600Â g for 5Â min at room temperature for each reaction. The cell pellet was resuspended in 50 Î¼L lysis buffer containing 10Â mM Trisâ€“HCl (pH 7.5), 10Â mM NaCl, 3Â mM MgCl<sub>2</sub>, 0.1% NP-40, 0.1% Tween-20, and 0.02% digitonin, and centrifuged at 500Â g for 10Â min at 4Â Â°C. After centrifugation, the cell pellet was processed immediately for the transposition reaction, resuspended in 50 Î¼L transposase mixture containing 25 Î¼L 2â€‰Ã—â€‰TD buffer, 22.5 Î¼L nuclease-free water, and 2.5 Î¼L Tn5 transposase. The mixture was incubated for 30Â min at 37Â Â°C with mixing at 1,000Â rpm. The transposed DNA was purified and amplified using NEBNext high-fidelity 2â€‰Ã—â€‰PCR master mix with custom Nextera PCR primers. The optimal number of PCR cycles was determined by qPCR, and the DNA libraries were size-selected and sequenced on the NovaSeq 6000 platform. ATAC-seq reads were aligned to the mouse genome, and alignments were processed with Bowtie using the command options -m 1 -v 2, allowing only uniquely mapping reads with up to two mismatches within the 150Â bp reads. BigWig files were generated using the â€˜bamCoverageâ€™ function in Deeptools.</p></section><section id="Sec32"><h3 class="pmc_sec_title">Histone modification analysis</h3>
<p id="Par60">Histone modification analysis was conducted following a previously published protocol [<a href="#CR56" class="usa-link" aria-describedby="CR56">56</a>]. Briefly, 293Â T cells were transfected with MSCV-KAT6A-CBP for 48Â h, followed by treatment with DMSO or A485 for 0, 6, or 12Â h. Cell pellets (~â€‰50 ÂµL) were collected by centrifugation at 300â€‰Ã—â€‰g for 5Â min and washed twice with PBS. The cell pellets were resuspended in Nuclear Isolation Buffer (NIB; 15Â mM Tris, 60Â mM KCl, 15Â mM NaCl, 5Â mM MgCl<sub>2</sub>, 1Â mM CaCl<sub>2</sub>, 250Â mM sucrose, pH 7.5) at a 1:10 (v/v) ratio and washed twice. The pellets were then resuspended in NIB containing 0.2% NP40, incubated on ice for 10Â min, and centrifuged at 1,000â€‰Ã—â€‰g for 10Â min at 4Â Â°C. The nuclei were washed twice with NIB (without NP40) and resuspended in chilled 0.2Â M H<sub>2</sub>SO4 at a 1:5 (v/v) ratio. Samples were incubated with gentle shaking at 4Â Â°C for 4Â h, followed by centrifugation at 3,400â€‰Ã—â€‰g for 5Â min at 4Â Â°C. The supernatant was collected and mixed with 100% TCA at a 1:3 (v/v) ratio, incubated on ice for 1Â h, and centrifuged at 3,400â€‰Ã—â€‰g for 5Â min.</p>
<p id="Par61">The resulting pellet was washed once with ice-cold acetone containing 0.1% HCl and once with 100% ice-cold acetone. The pellet was dried using a vacuum centrifuge and dissolved in minimal ddH<sub>2</sub>O to completely solubilize the white layer. After centrifugation, the supernatant was collected, and protein concentrations were measured using a BCA assay. Histone proteins were analyzed by SDS-PAGE on 15% acrylamide gels. For chemical derivatization, histones were adjusted to 50Â mM NH<sub>4</sub>HCO<sub>3</sub> (pH 8.0), and a fresh propionylation reagent (propionic anhydride:acetonitrile, 1:3 v/v) was added at a 1:4 (v/v) ratio. NH<sub>4</sub>OH was used to re-establish pH 8.0, and the mixture was incubated at room temperature for 15Â min. Samples were dried to 10â€“20 ÂµL in a vacuum concentrator and diluted to a final volume of 40 ÂµL with ddH<sub>2</sub>O. The propionylation step was repeated once. Histones were resuspended in 50Â mM NH<sub>4</sub>HCO<sub>3</sub> at a concentration ofâ€‰â‰¥â€‰1Â Âµg/ÂµL (pH 8.0) and digested with trypsin at a 1:20 (w/w) ratio overnight at 37Â Â°C. Digestion was stopped by freezing at âˆ’80Â Â°C, and samples were dried to 10â€“20 ÂµL. Samples were resuspended in 30 ÂµL of 100Â mM NH<sub>4</sub>HCO3 and subjected to a second round of propionylation. Finally, samples were diluted with 50â€“100 ÂµL of 0.1% TFA in ddH<sub>2</sub>O, desalted using C18 desalting tips, dried in a vacuum concentrator, and analyzed by LC-FAIMS-MS/MS.</p></section><section id="Sec33"><h3 class="pmc_sec_title">RNA-seq analysis</h3>
<p id="Par62">Mouse leukemia cells isolated from leukemic mice, MP cells (Lin-, c-Kitâ€‰+), and c-Kit-enriched HSPCs were lysed with Trizol (Ambion 15,596,018) for total RNA extraction. mRNA from 2Â Âµg of total RNA was enriched with VAHTS mRNA capture beads and fragmented to 200â€“500 nt lengths using 2â€‰Ã—â€‰ProtoScript II buffer. Strand-specific RNA-seq libraries were prepared and sequenced with 150Â bp paired-end reads on the NovaSeq 6000 platform. The sequencing data were aligned to the mm10 mouse genome, normalized (RPM), and visualized using the UCSC genome browser. Differential gene expression analysis was performed with HTSeq-count (version 0.12.3) and DESeq2 (version 1.32.0) [<a href="#CR57" class="usa-link" aria-describedby="CR57">57</a>]. Gene ontology enrichment analysis was conducted using Metascape.</p></section><section id="Sec34"><h3 class="pmc_sec_title">Statistical analysis</h3>
<p id="Par63">Statistical analysis was performed using GraphPad Prism 8 software. Data are expressed as Meanâ€‰Â±â€‰SD. The statistical tests employed, including one-way ANOVA, Student's t-test, and log-rank test, were chosen based on the experimental design and data characteristics. Statistical significance was set at <em>P</em>â€‰&lt;â€‰0.05 (*, <em>P</em>â€‰&lt;â€‰0.05; **, <em>P</em>â€‰&lt;â€‰0.01; ***, <em>P</em>â€‰&lt;â€‰0.001).</p></section><section id="Sec35"><h3 class="pmc_sec_title">Data and materials availability</h3>
<p id="Par64">All data are available in the main text or the supplementary materials. The H3K9ac and H3K27me3 ChIP-seq data from 293Â T cells were originally deposited into GEO under the accession number: SRR19676715 [<a href="#CR25" class="usa-link" aria-describedby="CR25">25</a>] and SRR9604322 [<a href="#CR58" class="usa-link" aria-describedby="CR58">58</a>]. The H3K9me3 293Â T CUT&amp;RUN were obtained under the accession number: SRR21837874 and SRR21862863 [<a href="#CR59" class="usa-link" aria-describedby="CR59">59</a>]. The raw and processed mRNA-seq, ATAC-seq, and CUT&amp;RUN datasets supporting the conclusions of this article have been deposited in the Gene Expression Omnibus (GEO) repository under accession numbers: <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE299379" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE299379</a> [<a href="#CR60" class="usa-link" aria-describedby="CR60">60</a>], <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE299380" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE299380</a> [<a href="#CR61" class="usa-link" aria-describedby="CR61">61</a>], <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE299381" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE299381</a> [<a href="#CR62" class="usa-link" aria-describedby="CR62">62</a>], <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE299382" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE299382</a> [<a href="#CR63" class="usa-link" aria-describedby="CR63">63</a>], and <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE299383" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE299383</a> [<a href="#CR64" class="usa-link" aria-describedby="CR64">64</a>]. The mass spectrometry proteomics data generated in this study have been submitted to the ProteomeXchange Consortium via the PRIDE partner repository under the dataset identifier PXD061491 [<a href="#CR65" class="usa-link" aria-describedby="CR65">65</a>] and PXD061565 [<a href="#CR66" class="usa-link" aria-describedby="CR66">66</a>]. All unique materials generated are available from the corresponding author upon reasonable request. No custom scripts and software were used besides those mentioned in the Methods section.</p></section></section><section id="Sec36"><h2 class="pmc_sec_title">Supplementary Information</h2>
<section class="sm xbox font-sm" id="MOESM1"><div class="media p"><div class="caption">
<a href="/articles/instance/12366150/bin/13059_2025_3743_MOESM1_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link">Additional file 1: Figures S1-S10</a><sup> (42.9MB, docx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM2"><div class="media p"><div class="caption">
<a href="/articles/instance/12366150/bin/13059_2025_3743_MOESM2_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">13059_2025_3743_MOESM2_ESM.xlsx</a><sup> (7MB, xlsx) </sup><p>Additional file 2: Tables S1-S9.Â Table S1. mRNA-seq analysis of c-Kit<sup>+</sup> bone marrow, MPs, K/C-III, and K/P-III leukemia cells. Table S2. IP-MS analysis of KAT6A and KAT6A fusions. Table S3. mRNA-seq analysis of K/C-III leukemia cells after BZ1 treatments. Table S4. mRNA-seq analysis of K/C-III leukemia cells after A485 treatments. Table S5. List of 770 core target genes of K/C-III. Table S6. List of KAT6A truncates and KAT6A fusion sequence. Table S7. Primer sequences of BPTF shRNA. Table S8. List of antibodies used in this study. Table S9. List of RT-qPCR primers.</p>
</div></div></section><section class="sm xbox font-sm" id="MOESM3"><div class="media p"><div class="caption">
<a href="/articles/instance/12366150/bin/13059_2025_3743_MOESM3_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link">13059_2025_3743_MOESM3_ESM.docx</a><sup> (1.2MB, docx) </sup><p>Additional file 3: Uncropped western blot and gel images</p>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>We express our sincere gratitude to Dr. Edwin Smith for his valuable discussions. We thank the Research Center for Medicine and Structural Biology of Wuhan University for their technical support.</p>
<section id="FPar1"><h3 class="pmc_sec_title">Peer review information</h3>
<p id="Par65">Chunliang Li and Tim Sands were the primary editors of this article and managed its editorial process and peer review in collaboration with the rest of the editorial team. The peer-review history is available in the online version of this article.</p></section></section><section id="notes1"><h2 class="pmc_sec_title">Authorsâ€™ contributions</h2>
<p>J.L., H.W., C.L., P.Y., Y.X., R.L., R.X., Z.S., and P.F. carried-out experiments and data analysis. Z.Y., R.X., and K.L. performed statistical and bioinformatic analyses. R.X., L.H., P.F., and Y.Z. provided mass spectrometry analysis. K.L. and P.F. wrote the manuscript with inputs from all authors. K.L., P.F., R.X., and H.Z. supervised the experiments. K.L. and P.F. conceived and oversaw the study. All authors discussed the results and contributed to preparing the manuscript. All authors read and approved the final manuscript.</p></section><section id="notes2"><h2 class="pmc_sec_title">Funding</h2>
<p>This study was supported by the Noncommunicable Chronic Diseases-National Science and Technology Major Project (2023ZD0500600), National Outstanding Youth Science Fund Project of National Natural Science Foundation of China (82122006 to K.L.), National Natural Science Foundation of China (32170572 and 32470597 to P.F., 82172641 to K.L., and 82400141 to C.L.), Postdoctor Project of Hubei Province (2004HBBHCXA066 to C.L.), Natural Science Foundation of Hubei Province of China (2024AFB204 to C.L.), and Postdoctoral Fellowship Program of CPSF (GZB20230540).</p></section><section id="notes3"><h2 class="pmc_sec_title">Data availability</h2>
<p>All data are available in the main text or the supplementary materials. The H3K9ac and H3K27me3 ChIP-seq data from 293Â T cells were originally deposited into GEO under the accession number: SRR19676715 [<a href="#CR25" class="usa-link" aria-describedby="CR25">25</a>] and SRR9604322 [<a href="#CR58" class="usa-link" aria-describedby="CR58">58</a>]. The H3K9me3 293Â T CUT&amp;RUN were obtained under the accession number: SRR21837874 and SRR21862863 [<a href="#CR59" class="usa-link" aria-describedby="CR59">59</a>]. The raw and processed mRNA-seq, ATAC-seq, and CUT&amp;RUN datasets supporting the conclusions of this article have been deposited in the Gene Expression Omnibus (GEO) repository under accession numbers: <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE299379" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE299379</a> [<a href="#CR60" class="usa-link" aria-describedby="CR60">60</a>], <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE299380" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE299380</a> [<a href="#CR61" class="usa-link" aria-describedby="CR61">61</a>], <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE299381" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE299381</a> [<a href="#CR62" class="usa-link" aria-describedby="CR62">62</a>], <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE299382" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE299382</a> [<a href="#CR63" class="usa-link" aria-describedby="CR63">63</a>], and <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE299383" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE299383</a> [<a href="#CR64" class="usa-link" aria-describedby="CR64">64</a>]. The mass spectrometry proteomics data generated in this study have been submitted to the ProteomeXchange Consortium via the PRIDE partner repository under the dataset identifier PXD061491 [<a href="#CR65" class="usa-link" aria-describedby="CR65">65</a>] and PXD061565 [<a href="#CR66" class="usa-link" aria-describedby="CR66">66</a>]. All unique materials generated are available from the corresponding author upon reasonable request. No custom scripts and software were used besides those mentioned in the Methods section.</p></section><section id="notes4"><h2 class="pmc_sec_title">Declarations</h2>
<section id="FPar2"><h3 class="pmc_sec_title">Ethics approval and consent to participate</h3>
<p id="Par66">Our study employed only 6â€“8-week-old female C57BL/6Â J mice, obtained from Charles River. All procedures complied with the ethical standards set by the Wuhan University Animal Facility Ethics Committee (No. WAEF-2024â€“0062) and adhered to established guidelines for animal care.</p></section><section id="FPar3"><h3 class="pmc_sec_title">Consent for publication</h3>
<p id="Par67">Not applicable.</p></section><section id="FPar4"><h3 class="pmc_sec_title">Competing interests</h3>
<p id="Par68">The authors declare no competing interests.</p></section></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm">
<div class="fn p" id="fn1">
<p><strong>Publisherâ€™s Note</strong></p>
<p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p>
</div>
<div class="fn p" id="fn2"><p>Junhui Lv, Zhinang Yin, Conghui Li and Honglin Wen contributed equally to this work.</p></div>
</div></section><section id="_ci93_" lang="en" class="contrib-info"><h2 class="pmc_sec_title">Contributor Information</h2>
<p>Ruijing Xiao, Email: xrj7619@whu.edu.cn.</p>
<p>Pingping Fang, Email: pingpingfang@whu.edu.cn.</p>
<p>Kaiwei Liang, Email: kwliang@whu.edu.cn.</p></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>DiNardo CD, Erba HP, Freeman SD, Wei AH. Acute myeloid leukaemia. Lancet. 2023;401:2073â€“86.
</cite> [<a href="https://doi.org/10.1016/S0140-6736(23)00108-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37068505/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?DiNardo%20CD,%20Erba%20HP,%20Freeman%20SD,%20Wei%20AH.%20Acute%20myeloid%20leukaemia.%20Lancet.%202023;401:2073%E2%80%9386." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>Dohner H, Weisdorf DJ, Bloomfield CD. Acute myeloid leukemia. N Engl J Med. 2015;373:1136â€“52.
</cite> [<a href="https://doi.org/10.1056/NEJMra1406184" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26376137/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Dohner%20H,%20Weisdorf%20DJ,%20Bloomfield%20CD.%20Acute%20myeloid%20leukemia.%20N%20Engl%20J%20Med.%202015;373:1136%E2%80%9352." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>Papaemmanuil E, Gerstung M, Bullinger L, Gaidzik VI, Paschka P, Roberts ND, Potter NE, Heuser M, Thol F, Bolli N, et al. Genomic classification and prognosis in acute myeloid leukemia. N Engl J Med. 2016;374:2209â€“21.
</cite> [<a href="https://doi.org/10.1056/NEJMoa1516192" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4979995/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27276561/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Papaemmanuil%20E,%20Gerstung%20M,%20Bullinger%20L,%20Gaidzik%20VI,%20Paschka%20P,%20Roberts%20ND,%20Potter%20NE,%20Heuser%20M,%20Thol%20F,%20Bolli%20N,%20et%20al.%20Genomic%20classification%20and%20prognosis%20in%20acute%20myeloid%20leukemia.%20N%20Engl%20J%20Med.%202016;374:2209%E2%80%9321." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Brien GL, Stegmaier K, Armstrong SA. Targeting chromatin complexes in fusion protein-driven malignancies. Nat Rev Cancer. 2019;19:255â€“69.
</cite> [<a href="https://doi.org/10.1038/s41568-019-0132-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30962549/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Brien%20GL,%20Stegmaier%20K,%20Armstrong%20SA.%20Targeting%20chromatin%20complexes%20in%20fusion%20protein-driven%20malignancies.%20Nat%20Rev%20Cancer.%202019;19:255%E2%80%9369." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Song Z, Cui Y, Xin L, Xiao R, Feng J, Li C, Yin Z, Wang H, Li Q, Wang M, et al. Mechano-oncogenic cytoskeletal remodeling drives leukemic transformation with mitochondrial vesicle-mediated STING activation. Cell Stem Cell. 2025;32(581â€“597):e511.</cite> [<a href="https://doi.org/10.1016/j.stem.2025.01.013" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39986274/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Song%20Z,%20Cui%20Y,%20Xin%20L,%20Xiao%20R,%20Feng%20J,%20Li%20C,%20Yin%20Z,%20Wang%20H,%20Li%20Q,%20Wang%20M,%20et%20al.%20Mechano-oncogenic%20cytoskeletal%20remodeling%20drives%20leukemic%20transformation%20with%20mitochondrial%20vesicle-mediated%20STING%20activation.%20Cell%20Stem%20Cell.%202025;32(581%E2%80%93597):e511." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Huang F, Abmayr SM, Workman JL. Regulation of KAT6 acetyltransferases and their roles in cell cycle progression, stem cell maintenance, and human disease. Mol Cell Biol. 2016;36:1900â€“7.
</cite> [<a href="https://doi.org/10.1128/MCB.00055-16" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4936061/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27185879/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Huang%20F,%20Abmayr%20SM,%20Workman%20JL.%20Regulation%20of%20KAT6%20acetyltransferases%20and%20their%20roles%20in%20cell%20cycle%20progression,%20stem%20cell%20maintenance,%20and%20human%20disease.%20Mol%20Cell%20Biol.%202016;36:1900%E2%80%937." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Borrow J, Stanton VP Jr, Andresen JM, Becher R, Behm FG, Chaganti RS, Civin CI, Disteche C, Dube I, Frischauf AM, et al. The translocation t(8;16)(p11;p13) of acute myeloid leukaemia fuses a putative acetyltransferase to the CREB-binding protein. Nat Genet. 1996;14:33â€“41.
</cite> [<a href="https://doi.org/10.1038/ng0996-33" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/8782817/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Borrow%20J,%20Stanton%20VP%20Jr,%20Andresen%20JM,%20Becher%20R,%20Behm%20FG,%20Chaganti%20RS,%20Civin%20CI,%20Disteche%20C,%20Dube%20I,%20Frischauf%20AM,%20et%20al.%20The%20translocation%20t(8;16)(p11;p13)%20of%20acute%20myeloid%20leukaemia%20fuses%20a%20putative%20acetyltransferase%20to%20the%20CREB-binding%20protein.%20Nat%20Genet.%201996;14:33%E2%80%9341." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>Camos M, Esteve J, Jares P, Colomer D, Rozman M, Villamor N, Costa D, Carrio A, Nomdedeu J, Montserrat E, Campo E. Gene expression profiling of acute myeloid leukemia with translocation t(8;16)(p11;p13) and MYST3-CREBBP rearrangement reveals a distinctive signature with a specific pattern of HOX gene expression. Cancer Res. 2006;66:6947â€“54.
</cite> [<a href="https://doi.org/10.1158/0008-5472.CAN-05-4601" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16849538/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Camos%20M,%20Esteve%20J,%20Jares%20P,%20Colomer%20D,%20Rozman%20M,%20Villamor%20N,%20Costa%20D,%20Carrio%20A,%20Nomdedeu%20J,%20Montserrat%20E,%20Campo%20E.%20Gene%20expression%20profiling%20of%20acute%20myeloid%20leukemia%20with%20translocation%20t(8;16)(p11;p13)%20and%20MYST3-CREBBP%20rearrangement%20reveals%20a%20distinctive%20signature%20with%20a%20specific%20pattern%20of%20HOX%20gene%20expression.%20Cancer%20Res.%202006;66:6947%E2%80%9354." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>Baell JB, Leaver DJ, Hermans SJ, Kelly GL, Brennan MS, Downer NL, Nguyen N, Wichmann J, McRae HM, Yang Y, et al. Inhibitors of histone acetyltransferases KAT6A/B induce senescence and arrest tumour growth. Nature. 2018;560:253â€“7.
</cite> [<a href="https://doi.org/10.1038/s41586-018-0387-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30069049/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Baell%20JB,%20Leaver%20DJ,%20Hermans%20SJ,%20Kelly%20GL,%20Brennan%20MS,%20Downer%20NL,%20Nguyen%20N,%20Wichmann%20J,%20McRae%20HM,%20Yang%20Y,%20et%20al.%20Inhibitors%20of%20histone%20acetyltransferases%20KAT6A/B%20induce%20senescence%20and%20arrest%20tumour%20growth.%20Nature.%202018;560:253%E2%80%937." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>Sheikh BN, Yang Y, Schreuder J, Nilsson SK, Bilardi R, Carotta S, McRae HM, Metcalf D, Voss AK, Thomas T. MOZ (KAT6A) is essential for the maintenance of classically defined adult hematopoietic stem cells. Blood. 2016;128:2307â€“18.
</cite> [<a href="https://doi.org/10.1182/blood-2015-10-676072" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27663673/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sheikh%20BN,%20Yang%20Y,%20Schreuder%20J,%20Nilsson%20SK,%20Bilardi%20R,%20Carotta%20S,%20McRae%20HM,%20Metcalf%20D,%20Voss%20AK,%20Thomas%20T.%20MOZ%20(KAT6A)%20is%20essential%20for%20the%20maintenance%20of%20classically%20defined%20adult%20hematopoietic%20stem%20cells.%20Blood.%202016;128:2307%E2%80%9318." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Katsumoto T, Aikawa Y, Iwama A, Ueda S, Ichikawa H, Ochiya T, Kitabayashi I. MOZ is essential for maintenance of hematopoietic stem cells. Genes Dev. 2006;20:1321â€“30.
</cite> [<a href="https://doi.org/10.1101/gad.1393106" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1472906/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16702405/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Katsumoto%20T,%20Aikawa%20Y,%20Iwama%20A,%20Ueda%20S,%20Ichikawa%20H,%20Ochiya%20T,%20Kitabayashi%20I.%20MOZ%20is%20essential%20for%20maintenance%20of%20hematopoietic%20stem%20cells.%20Genes%20Dev.%202006;20:1321%E2%80%9330." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Klein BJ, Jang SM, Lachance C, Mi W, Lyu J, Sakuraba S, Krajewski K, Wang WW, Sidoli S, Liu J, et al. Histone H3K23-specific acetylation by MORF is coupled to H3K14 acylation. Nat Commun. 2019;10:4724.
</cite> [<a href="https://doi.org/10.1038/s41467-019-12551-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6797804/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31624313/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Klein%20BJ,%20Jang%20SM,%20Lachance%20C,%20Mi%20W,%20Lyu%20J,%20Sakuraba%20S,%20Krajewski%20K,%20Wang%20WW,%20Sidoli%20S,%20Liu%20J,%20et%20al.%20Histone%20H3K23-specific%20acetylation%20by%20MORF%20is%20coupled%20to%20H3K14%20acylation.%20Nat%20Commun.%202019;10:4724." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>Ullah M, Pelletier N, Xiao L, Zhao SP, Wang K, Degerny C, Tahmasebi S, Cayrou C, Doyon Y, Goh SL, et al. Molecular architecture of quartet MOZ/MORF histone acetyltransferase complexes. Mol Cell Biol. 2008;28:6828â€“43.
</cite> [<a href="https://doi.org/10.1128/MCB.01297-08" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2573306/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18794358/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ullah%20M,%20Pelletier%20N,%20Xiao%20L,%20Zhao%20SP,%20Wang%20K,%20Degerny%20C,%20Tahmasebi%20S,%20Cayrou%20C,%20Doyon%20Y,%20Goh%20SL,%20et%20al.%20Molecular%20architecture%20of%20quartet%20MOZ/MORF%20histone%20acetyltransferase%20complexes.%20Mol%20Cell%20Biol.%202008;28:6828%E2%80%9343." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>Piunti A, Shilatifard A. Epigenetic balance of gene expression by Polycomb and COMPASS families. Science. 2016;352:aad9780.
</cite> [<a href="https://doi.org/10.1126/science.aad9780" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27257261/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Piunti%20A,%20Shilatifard%20A.%20Epigenetic%20balance%20of%20gene%20expression%20by%20Polycomb%20and%20COMPASS%20families.%20Science.%202016;352:aad9780." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>Carapeti M, Aguiar RC, Goldman JM, Cross NC. A novel fusion between MOZ and the nuclear receptor coactivator TIF2 in acute myeloid leukemia. Blood. 1998;91:3127â€“33.
</cite> [<a href="https://pubmed.ncbi.nlm.nih.gov/9558366/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Carapeti%20M,%20Aguiar%20RC,%20Goldman%20JM,%20Cross%20NC.%20A%20novel%20fusion%20between%20MOZ%20and%20the%20nuclear%20receptor%20coactivator%20TIF2%20in%20acute%20myeloid%20leukemia.%20Blood.%201998;91:3127%E2%80%9333." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Gervais C, Murati A, Helias C, Struski S, Eischen A, Lippert E, Tigaud I, Penther D, Bastard C, Mugneret F, et al. Acute myeloid leukaemia with 8p11 (MYST3) rearrangement: an integrated cytologic, cytogenetic and molecular study by the groupe francophone de cytogenetique hematologique. Leukemia. 2008;22:1567â€“75.
</cite> [<a href="https://doi.org/10.1038/leu.2008.128" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18528428/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gervais%20C,%20Murati%20A,%20Helias%20C,%20Struski%20S,%20Eischen%20A,%20Lippert%20E,%20Tigaud%20I,%20Penther%20D,%20Bastard%20C,%20Mugneret%20F,%20et%20al.%20Acute%20myeloid%20leukaemia%20with%208p11%20(MYST3)%20rearrangement:%20an%20integrated%20cytologic,%20cytogenetic%20and%20molecular%20study%20by%20the%20groupe%20francophone%20de%20cytogenetique%20hematologique.%20Leukemia.%202008;22:1567%E2%80%9375." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>Xie W, Hu S, Xu J, Chen Z, Medeiros LJ, Tang G. Acute myeloid leukemia with t(8;16)(p11.2;p13.3)/KAT6A-CREBBP in adults. Ann Hematol. 2019;98:1149â€“57.
</cite> [<a href="https://doi.org/10.1007/s00277-019-03637-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30759270/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Xie%20W,%20Hu%20S,%20Xu%20J,%20Chen%20Z,%20Medeiros%20LJ,%20Tang%20G.%20Acute%20myeloid%20leukemia%20with%20t(8;16)(p11.2;p13.3)/KAT6A-CREBBP%20in%20adults.%20Ann%20Hematol.%202019;98:1149%E2%80%9357." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Chan HM, La Thangue NB. p300/CBP proteins: HATs for transcriptional bridges and scaffolds. J Cell Sci. 2001;114:2363â€“73.
</cite> [<a href="https://doi.org/10.1242/jcs.114.13.2363" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11559745/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chan%20HM,%20La%20Thangue%20NB.%20p300/CBP%20proteins:%20HATs%20for%20transcriptional%20bridges%20and%20scaffolds.%20J%20Cell%20Sci.%202001;114:2363%E2%80%9373." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Weinert BT, Narita T, Satpathy S, Srinivasan B, Hansen BK, Scholz C, Hamilton WB, Zucconi BE, Wang WW, Liu WR, et al. Time-Resolved Analysis Reveals Rapid Dynamics and Broad Scope of the CBP/p300 Acetylome. Cell. 2018;174(231â€“244):e212.</cite> [<a href="https://doi.org/10.1016/j.cell.2018.04.033" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6078418/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29804834/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Weinert%20BT,%20Narita%20T,%20Satpathy%20S,%20Srinivasan%20B,%20Hansen%20BK,%20Scholz%20C,%20Hamilton%20WB,%20Zucconi%20BE,%20Wang%20WW,%20Liu%20WR,%20et%20al.%20Time-Resolved%20Analysis%20Reveals%20Rapid%20Dynamics%20and%20Broad%20Scope%20of%20the%20CBP/p300%20Acetylome.%20Cell.%202018;174(231%E2%80%93244):e212." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>Teufel DP, Freund SM, Bycroft M, Fersht AR. Four domains of p300 each bind tightly to a sequence spanning both transactivation subdomains of p53. Proc Natl Acad Sci U S A. 2007;104:7009â€“14.
</cite> [<a href="https://doi.org/10.1073/pnas.0702010104" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1855428/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17438265/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Teufel%20DP,%20Freund%20SM,%20Bycroft%20M,%20Fersht%20AR.%20Four%20domains%20of%20p300%20each%20bind%20tightly%20to%20a%20sequence%20spanning%20both%20transactivation%20subdomains%20of%20p53.%20Proc%20Natl%20Acad%20Sci%20U%20S%20A.%202007;104:7009%E2%80%9314." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>Troke PJ, Kindle KB, Collins HM, Heery DM. MOZ fusion proteins in acute myeloid leukaemia. Biochem Soc Symp. 2006;73:23â€“39.Â 10.1042/bss0730023.</cite> [<a href="https://doi.org/10.1042/bss0730023" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16626284/" class="usa-link">PubMed</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>Huntly BJ, Shigematsu H, Deguchi K, Lee BH, Mizuno S, Duclos N, Rowan R, Amaral S, Curley D, Williams IR, et al. MOZ-TIF2, but not BCR-ABL, confers properties of leukemic stem cells to committed murine hematopoietic progenitors. Cancer Cell. 2004;6:587â€“96.
</cite> [<a href="https://doi.org/10.1016/j.ccr.2004.10.015" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15607963/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Huntly%20BJ,%20Shigematsu%20H,%20Deguchi%20K,%20Lee%20BH,%20Mizuno%20S,%20Duclos%20N,%20Rowan%20R,%20Amaral%20S,%20Curley%20D,%20Williams%20IR,%20et%20al.%20MOZ-TIF2,%20but%20not%20BCR-ABL,%20confers%20properties%20of%20leukemic%20stem%20cells%20to%20committed%20murine%20hematopoietic%20progenitors.%20Cancer%20Cell.%202004;6:587%E2%80%9396." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>Deguchi K, Ayton PM, Carapeti M, Kutok JL, Snyder CS, Williams IR, Cross NC, Glass CK, Cleary ML, Gilliland DG. MOZ-TIF2-induced acute myeloid leukemia requires the MOZ nucleosome binding motif and TIF2-mediated recruitment of CBP. Cancer Cell. 2003;3:259â€“71.
</cite> [<a href="https://doi.org/10.1016/s1535-6108(03)00051-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12676584/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Deguchi%20K,%20Ayton%20PM,%20Carapeti%20M,%20Kutok%20JL,%20Snyder%20CS,%20Williams%20IR,%20Cross%20NC,%20Glass%20CK,%20Cleary%20ML,%20Gilliland%20DG.%20MOZ-TIF2-induced%20acute%20myeloid%20leukemia%20requires%20the%20MOZ%20nucleosome%20binding%20motif%20and%20TIF2-mediated%20recruitment%20of%20CBP.%20Cancer%20Cell.%202003;3:259%E2%80%9371." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>Becht DC, Klein BJ, Kanai A, Jang SM, Cox KL, Zhou BR, Phanor SK, Zhang Y, Chen RW, Ebmeier CC, et al. MORF and MOZ acetyltransferases target unmethylated CpG islands through the winged helix domain. Nat Commun. 2023;14:697.
</cite> [<a href="https://doi.org/10.1038/s41467-023-36368-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9908889/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36754959/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Becht%20DC,%20Klein%20BJ,%20Kanai%20A,%20Jang%20SM,%20Cox%20KL,%20Zhou%20BR,%20Phanor%20SK,%20Zhang%20Y,%20Chen%20RW,%20Ebmeier%20CC,%20et%20al.%20MORF%20and%20MOZ%20acetyltransferases%20target%20unmethylated%20CpG%20islands%20through%20the%20winged%20helix%20domain.%20Nat%20Commun.%202023;14:697." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>Weber LM, Jia Y, Stielow B, Gisselbrecht SS, Cao Y, Ren Y, Rohner I, King J, Rothman E, Fischer S, et al. The histone acetyltransferase KAT6A is recruited to unmethylated CpG islands via a DNA binding winged helix domain. Nucleic Acids Res. 2023;51:574â€“94.
</cite> [<a href="https://doi.org/10.1093/nar/gkac1188" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9881136/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36537216/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Weber%20LM,%20Jia%20Y,%20Stielow%20B,%20Gisselbrecht%20SS,%20Cao%20Y,%20Ren%20Y,%20Rohner%20I,%20King%20J,%20Rothman%20E,%20Fischer%20S,%20et%20al.%20The%20histone%20acetyltransferase%20KAT6A%20is%20recruited%20to%20unmethylated%20CpG%20islands%20via%20a%20DNA%20binding%20winged%20helix%20domain.%20Nucleic%20Acids%20Res.%202023;51:574%E2%80%9394." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>Becht DC, Kanai A, Biswas S, Halawa M, Zeng L, Cox KL, Poirier MG, Zhou MM, Shi X, Yokoyama A, Kutateladze TG. The winged helix domain of MORF binds CpG islands and the TAZ2 domain of p300. iScience. 2024;27:109367.
</cite> [<a href="https://doi.org/10.1016/j.isci.2024.109367" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10946326/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38500836/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Becht%20DC,%20Kanai%20A,%20Biswas%20S,%20Halawa%20M,%20Zeng%20L,%20Cox%20KL,%20Poirier%20MG,%20Zhou%20MM,%20Shi%20X,%20Yokoyama%20A,%20Kutateladze%20TG.%20The%20winged%20helix%20domain%20of%20MORF%20binds%20CpG%20islands%20and%20the%20TAZ2%20domain%20of%20p300.%20iScience.%202024;27:109367." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Deaton AM, Bird A. Cpg islands and the regulation of transcription. Genes Dev. 2011;25:1010â€“22.
</cite> [<a href="https://doi.org/10.1101/gad.2037511" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3093116/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21576262/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Deaton%20AM,%20Bird%20A.%20Cpg%20islands%20and%20the%20regulation%20of%20transcription.%20Genes%20Dev.%202011;25:1010%E2%80%9322." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Kumar M, Keller B, Makalou N, Sutton RE. Systematic determination of the packaging limit of lentiviral vectors. Hum Gene Ther. 2001;12:1893â€“905.
</cite> [<a href="https://doi.org/10.1089/104303401753153947" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11589831/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kumar%20M,%20Keller%20B,%20Makalou%20N,%20Sutton%20RE.%20Systematic%20determination%20of%20the%20packaging%20limit%20of%20lentiviral%20vectors.%20Hum%20Gene%20Ther.%202001;12:1893%E2%80%93905." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>Hawley RG, Fong AZ, Burns BF, Hawley TS. Transplantable myeloproliferative disease induced in mice by an interleukin 6 retrovirus. J Exp Med. 1992;176:1149â€“63.
</cite> [<a href="https://doi.org/10.1084/jem.176.4.1149" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2119383/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/1402659/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hawley%20RG,%20Fong%20AZ,%20Burns%20BF,%20Hawley%20TS.%20Transplantable%20myeloproliferative%20disease%20induced%20in%20mice%20by%20an%20interleukin%206%20retrovirus.%20J%20Exp%20Med.%201992;176:1149%E2%80%9363." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR30">
<span class="label">30.</span><cite>Song Z, Cui Y, Xin L, Xiao R, Feng J, Li C, Yin Z, Wang H, Li Q, Wang M, et al. Mechano-oncogenic cytoskeletal remodeling drives leukemic transformation with mitochondrial vesicle-mediated STING activation. Cell Stem Cell. 2025. 10.1016/j.stem.2025.01.013.
</cite> [<a href="https://doi.org/10.1016/j.stem.2025.01.013" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39986274/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Song%20Z,%20Cui%20Y,%20Xin%20L,%20Xiao%20R,%20Feng%20J,%20Li%20C,%20Yin%20Z,%20Wang%20H,%20Li%20Q,%20Wang%20M,%20et%20al.%20Mechano-oncogenic%20cytoskeletal%20remodeling%20drives%20leukemic%20transformation%20with%20mitochondrial%20vesicle-mediated%20STING%20activation.%20Cell%20Stem%20Cell.%202025.%2010.1016/j.stem.2025.01.013." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>Zhang Y, Xue Y, Shi J, Ahn J, Mi W, Ali M, Wang X, Klein BJ, Wen H, Li W, et al. The ZZ domain of p300 mediates specificity of the adjacent HAT domain for histone H3. Nat Struct Mol Biol. 2018;25:841â€“9.
</cite> [<a href="https://doi.org/10.1038/s41594-018-0114-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6482957/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30150647/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhang%20Y,%20Xue%20Y,%20Shi%20J,%20Ahn%20J,%20Mi%20W,%20Ali%20M,%20Wang%20X,%20Klein%20BJ,%20Wen%20H,%20Li%20W,%20et%20al.%20The%20ZZ%20domain%20of%20p300%20mediates%20specificity%20of%20the%20adjacent%20HAT%20domain%20for%20histone%20H3.%20Nat%20Struct%20Mol%20Biol.%202018;25:841%E2%80%939." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>Xu L, Xuan H, He W, Zhang L, Huang M, Li K, Wen H, Xu H, Shi X. TAZ2 truncation confers overactivation of p300 and cellular vulnerability to HDAC inhibition. Nat Commun. 2023;14:5362.
</cite> [<a href="https://doi.org/10.1038/s41467-023-41245-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10475075/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37660055/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Xu%20L,%20Xuan%20H,%20He%20W,%20Zhang%20L,%20Huang%20M,%20Li%20K,%20Wen%20H,%20Xu%20H,%20Shi%20X.%20TAZ2%20truncation%20confers%20overactivation%20of%20p300%20and%20cellular%20vulnerability%20to%20HDAC%20inhibition.%20Nat%20Commun.%202023;14:5362." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Solomon M, DeLay M, Reynaud D. Phenotypic analysis of the mouse hematopoietic hierarchy using spectral cytometry: from stem cell subsets to early progenitor compartments. Cytometry A. 2020;97:1057â€“65.
</cite> [<a href="https://doi.org/10.1002/cyto.a.24041" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8630828/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32449586/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Solomon%20M,%20DeLay%20M,%20Reynaud%20D.%20Phenotypic%20analysis%20of%20the%20mouse%20hematopoietic%20hierarchy%20using%20spectral%20cytometry:%20from%20stem%20cell%20subsets%20to%20early%20progenitor%20compartments.%20Cytometry%20A.%202020;97:1057%E2%80%9365." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>Wysocka J, Swigut T, Xiao H, Milne TA, Kwon SY, Landry J, Kauer M, Tackett AJ, Chait BT, Badenhorst P, et al. A PHD finger of NURF couples histone H3 lysine 4 trimethylation with chromatin remodelling. Nature. 2006;442:86â€“90.
</cite> [<a href="https://doi.org/10.1038/nature04815" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16728976/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wysocka%20J,%20Swigut%20T,%20Xiao%20H,%20Milne%20TA,%20Kwon%20SY,%20Landry%20J,%20Kauer%20M,%20Tackett%20AJ,%20Chait%20BT,%20Badenhorst%20P,%20et%20al.%20A%20PHD%20finger%20of%20NURF%20couples%20histone%20H3%20lysine%204%20trimethylation%20with%20chromatin%20remodelling.%20Nature.%202006;442:86%E2%80%9390." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>Li H, Ilin S, Wang W, Duncan EM, Wysocka J, Allis CD, Patel DJ. Molecular basis for site-specific read-out of histone H3K4me3 by the BPTF PHD finger of NURF. Nature. 2006;442:91â€“5.
</cite> [<a href="https://doi.org/10.1038/nature04802" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4690523/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16728978/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li%20H,%20Ilin%20S,%20Wang%20W,%20Duncan%20EM,%20Wysocka%20J,%20Allis%20CD,%20Patel%20DJ.%20Molecular%20basis%20for%20site-specific%20read-out%20of%20histone%20H3K4me3%20by%20the%20BPTF%20PHD%20finger%20of%20NURF.%20Nature.%202006;442:91%E2%80%935." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR36">
<span class="label">36.</span><cite>Zahid H, Buchholz CR, Singh M, Ciccone MF, Chan A, Nithianantham S, Shi K, Aihara H, Fischer M, Schonbrunn E, et al. New design rules for developing potent cell-active inhibitors of the nucleosome remodeling factor (NURF) via BPTF bromodomain inhibition. J Med Chem. 2021;64:13902â€“17.
</cite> [<a href="https://doi.org/10.1021/acs.jmedchem.1c01294" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9012132/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34515477/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zahid%20H,%20Buchholz%20CR,%20Singh%20M,%20Ciccone%20MF,%20Chan%20A,%20Nithianantham%20S,%20Shi%20K,%20Aihara%20H,%20Fischer%20M,%20Schonbrunn%20E,%20et%20al.%20New%20design%20rules%20for%20developing%20potent%20cell-active%20inhibitors%20of%20the%20nucleosome%20remodeling%20factor%20(NURF)%20via%20BPTF%20bromodomain%20inhibition.%20J%20Med%20Chem.%202021;64:13902%E2%80%9317." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR37">
<span class="label">37.</span><cite>Hamiche A, Sandaltzopoulos R, Gdula DA, Wu C. ATP-dependent histone octamer sliding mediated by the chromatin remodeling complex NURF. Cell. 1999;97:833â€“42.
</cite> [<a href="https://doi.org/10.1016/s0092-8674(00)80796-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10399912/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hamiche%20A,%20Sandaltzopoulos%20R,%20Gdula%20DA,%20Wu%20C.%20ATP-dependent%20histone%20octamer%20sliding%20mediated%20by%20the%20chromatin%20remodeling%20complex%20NURF.%20Cell.%201999;97:833%E2%80%9342." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR38">
<span class="label">38.</span><cite>Hargreaves DC, Crabtree GR. ATP-dependent chromatin remodeling: genetics, genomics and mechanisms. Cell Res. 2011;21:396â€“420.
</cite> [<a href="https://doi.org/10.1038/cr.2011.32" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3110148/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21358755/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hargreaves%20DC,%20Crabtree%20GR.%20ATP-dependent%20chromatin%20remodeling:%20genetics,%20genomics%20and%20mechanisms.%20Cell%20Res.%202011;21:396%E2%80%93420." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR39">
<span class="label">39.</span><cite>Miyamoto R, Okuda H, Kanai A, Takahashi S, Kawamura T, Matsui H, Kitamura T, Kitabayashi I, Inaba T, Yokoyama A. Activation of CpG-rich promoters mediated by MLL drives MOZ-rearranged leukemia. Cell Rep. 2020;32:108200.
</cite> [<a href="https://doi.org/10.1016/j.celrep.2020.108200" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32997997/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Miyamoto%20R,%20Okuda%20H,%20Kanai%20A,%20Takahashi%20S,%20Kawamura%20T,%20Matsui%20H,%20Kitamura%20T,%20Kitabayashi%20I,%20Inaba%20T,%20Yokoyama%20A.%20Activation%20of%20CpG-rich%20promoters%20mediated%20by%20MLL%20drives%20MOZ-rearranged%20leukemia.%20Cell%20Rep.%202020;32:108200." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR40">
<span class="label">40.</span><cite>Issa GC, Aldoss I, Thirman MJ, DiPersio J, Arellano M, Blachly JS, Mannis GN, Perl A, Dickens DS, McMahon CM, et al. Menin inhibition with revumenib for KMT2A-rearranged relapsed or refractory acute leukemia (AUGMENT-101). J Clin Oncol. 2025;43:75â€“84.
</cite> [<a href="https://doi.org/10.1200/JCO.24.00826" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11687943/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39121437/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Issa%20GC,%20Aldoss%20I,%20Thirman%20MJ,%20DiPersio%20J,%20Arellano%20M,%20Blachly%20JS,%20Mannis%20GN,%20Perl%20A,%20Dickens%20DS,%20McMahon%20CM,%20et%20al.%20Menin%20inhibition%20with%20revumenib%20for%20KMT2A-rearranged%20relapsed%20or%20refractory%20acute%20leukemia%20(AUGMENT-101).%20J%20Clin%20Oncol.%202025;43:75%E2%80%9384." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR41">
<span class="label">41.</span><cite>Volk A, Liang K, Suraneni P, Li X, Zhao J, Bulic M, Marshall S, Pulakanti K, Malinge S, Taub J, et al. A CHAF1B-Dependent Molecular Switch in Hematopoiesis and Leukemia Pathogenesis. Cancer Cell. 2018;34(707â€“723):e707.</cite> [<a href="https://doi.org/10.1016/j.ccell.2018.10.004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6235627/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30423293/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Volk%20A,%20Liang%20K,%20Suraneni%20P,%20Li%20X,%20Zhao%20J,%20Bulic%20M,%20Marshall%20S,%20Pulakanti%20K,%20Malinge%20S,%20Taub%20J,%20et%20al.%20A%20CHAF1B-Dependent%20Molecular%20Switch%20in%20Hematopoiesis%20and%20Leukemia%20Pathogenesis.%20Cancer%20Cell.%202018;34(707%E2%80%93723):e707." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR42">
<span class="label">42.</span><cite>Yun M, Wu J, Workman JL, Li B. Readers of histone modifications. Cell Res. 2011;21:564â€“78.
</cite> [<a href="https://doi.org/10.1038/cr.2011.42" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3131977/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21423274/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yun%20M,%20Wu%20J,%20Workman%20JL,%20Li%20B.%20Readers%20of%20histone%20modifications.%20Cell%20Res.%202011;21:564%E2%80%9378." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR43">
<span class="label">43.</span><cite>You L, Yan K, Zou J, Zhao H, Bertos NR, Park M, Wang E, Yang XJ. The chromatin regulator Brpf1 regulates embryo development and cell proliferation. J Biol Chem. 2015;290:11349â€“64.
</cite> [<a href="https://doi.org/10.1074/jbc.M115.643189" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4416840/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25773539/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?You%20L,%20Yan%20K,%20Zou%20J,%20Zhao%20H,%20Bertos%20NR,%20Park%20M,%20Wang%20E,%20Yang%20XJ.%20The%20chromatin%20regulator%20Brpf1%20regulates%20embryo%20development%20and%20cell%20proliferation.%20J%20Biol%20Chem.%202015;290:11349%E2%80%9364." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR44">
<span class="label">44.</span><cite>Yan K, Rousseau J, Machol K, Cross LA, Agre KE, Gibson CF, Goverde A, Engleman KL, Verdin H, De Baere E, et al. Deficient histone H3 propionylation by BRPF1-KAT6 complexes in neurodevelopmental disorders and cancer. Sci Adv. 2020;6:eaax0021.
</cite> [<a href="https://doi.org/10.1126/sciadv.aax0021" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6976298/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32010779/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yan%20K,%20Rousseau%20J,%20Machol%20K,%20Cross%20LA,%20Agre%20KE,%20Gibson%20CF,%20Goverde%20A,%20Engleman%20KL,%20Verdin%20H,%20De%20Baere%20E,%20et%20al.%20Deficient%20histone%20H3%20propionylation%20by%20BRPF1-KAT6%20complexes%20in%20neurodevelopmental%20disorders%20and%20cancer.%20Sci%20Adv.%202020;6:eaax0021." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR45">
<span class="label">45.</span><cite>Kebede AF, Nieborak A, Shahidian LZ, Le Gras S, Richter F, Gomez DA, Baltissen MP, Meszaros G, Magliarelli HF, Taudt A, et al. Histone propionylation is a mark of active chromatin. Nat Struct Mol Biol. 2017;24:1048â€“56.
</cite> [<a href="https://doi.org/10.1038/nsmb.3490" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29058708/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kebede%20AF,%20Nieborak%20A,%20Shahidian%20LZ,%20Le%20Gras%20S,%20Richter%20F,%20Gomez%20DA,%20Baltissen%20MP,%20Meszaros%20G,%20Magliarelli%20HF,%20Taudt%20A,%20et%20al.%20Histone%20propionylation%20is%20a%20mark%20of%20active%20chromatin.%20Nat%20Struct%20Mol%20Biol.%202017;24:1048%E2%80%9356." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR46">
<span class="label">46.</span><cite>Kwon SY, Grisan V, Jang B, Herbert J, Badenhorst P. Genome-wide mapping targets of the metazoan chromatin remodeling factor NURF reveals nucleosome remodeling at enhancers, core promoters and gene insulators. PLoS Genet. 2016;12:e1005969.
</cite> [<a href="https://doi.org/10.1371/journal.pgen.1005969" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4821604/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27046080/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kwon%20SY,%20Grisan%20V,%20Jang%20B,%20Herbert%20J,%20Badenhorst%20P.%20Genome-wide%20mapping%20targets%20of%20the%20metazoan%20chromatin%20remodeling%20factor%20NURF%20reveals%20nucleosome%20remodeling%20at%20enhancers,%20core%20promoters%20and%20gene%20insulators.%20PLoS%20Genet.%202016;12:e1005969." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR47">
<span class="label">47.</span><cite>Becker PB, Workman JL. Nucleosome remodeling and epigenetics. Cold Spring Harb Perspect Biol. 2013. 10.1101/cshperspect.a017905.
</cite> [<a href="https://doi.org/10.1101/cshperspect.a017905" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3753709/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24003213/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Becker%20PB,%20Workman%20JL.%20Nucleosome%20remodeling%20and%20epigenetics.%20Cold%20Spring%20Harb%20Perspect%20Biol.%202013.%2010.1101/cshperspect.a017905." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR48">
<span class="label">48.</span><cite>Lasko LM, Jakob CG, Edalji RP, Qiu W, Montgomery D, Digiammarino EL, Hansen TM, Risi RM, Frey R, Manaves V, et al. Discovery of a selective catalytic p300/CBP inhibitor that targets lineage-specific tumours. Nature. 2017;550:128â€“32.
</cite> [<a href="https://doi.org/10.1038/nature24028" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6050590/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28953875/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lasko%20LM,%20Jakob%20CG,%20Edalji%20RP,%20Qiu%20W,%20Montgomery%20D,%20Digiammarino%20EL,%20Hansen%20TM,%20Risi%20RM,%20Frey%20R,%20Manaves%20V,%20et%20al.%20Discovery%20of%20a%20selective%20catalytic%20p300/CBP%20inhibitor%20that%20targets%20lineage-specific%20tumours.%20Nature.%202017;550:128%E2%80%9332." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR49">
<span class="label">49.</span><cite>Ruthenburg AJ, Li H, Milne TA, Dewell S, McGinty RK, Yuen M, Ueberheide B, Dou Y, Muir TW, Patel DJ, Allis CD. Recognition of a mononucleosomal histone modification pattern by BPTF via multivalent interactions. Cell. 2011;145:692â€“706.
</cite> [<a href="https://doi.org/10.1016/j.cell.2011.03.053" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3135172/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21596426/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ruthenburg%20AJ,%20Li%20H,%20Milne%20TA,%20Dewell%20S,%20McGinty%20RK,%20Yuen%20M,%20Ueberheide%20B,%20Dou%20Y,%20Muir%20TW,%20Patel%20DJ,%20Allis%20CD.%20Recognition%20of%20a%20mononucleosomal%20histone%20modification%20pattern%20by%20BPTF%20via%20multivalent%20interactions.%20Cell.%202011;145:692%E2%80%93706." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR50">
<span class="label">50.</span><cite>Meers MP, Bryson TD, Henikoff JG, Henikoff S. Improved CUT&amp;RUN chromatin profiling tools. Elife. 2019;8:e46314. 10.7554/eLife.46314.</cite> [<a href="https://doi.org/10.7554/eLife.46314" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6598765/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31232687/" class="usa-link">PubMed</a>]</li>
<li id="CR51">
<span class="label">51.</span><cite>Liang K, Smith ER, Aoi Y, Stoltz KL, Katagi H, Woodfin AR, Rendleman EJ, Marshall SA, Murray DC, Wang L, et al. Targeting Processive Transcription Elongation via SEC Disruption for MYC-Induced Cancer Therapy. Cell. 2018;175(766â€“779):e717.</cite> [<a href="https://doi.org/10.1016/j.cell.2018.09.027" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6422358/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30340042/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Liang%20K,%20Smith%20ER,%20Aoi%20Y,%20Stoltz%20KL,%20Katagi%20H,%20Woodfin%20AR,%20Rendleman%20EJ,%20Marshall%20SA,%20Murray%20DC,%20Wang%20L,%20et%20al.%20Targeting%20Processive%20Transcription%20Elongation%20via%20SEC%20Disruption%20for%20MYC-Induced%20Cancer%20Therapy.%20Cell.%202018;175(766%E2%80%93779):e717." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR52">
<span class="label">52.</span><cite>Qiu M, Yin Z, Wang H, Lei L, Li C, Cui Y, Dai R, Yang P, Xiang Y, Li Q, et al. CDK12 and Integrator-PP2A complex modulates LEO1 phosphorylation for processive transcription elongation. Sci Adv. 2023;9:eadf8698.
</cite> [<a href="https://doi.org/10.1126/sciadv.adf8698" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10198629/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37205756/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Qiu%20M,%20Yin%20Z,%20Wang%20H,%20Lei%20L,%20Li%20C,%20Cui%20Y,%20Dai%20R,%20Yang%20P,%20Xiang%20Y,%20Li%20Q,%20et%20al.%20CDK12%20and%20Integrator-PP2A%20complex%20modulates%20LEO1%20phosphorylation%20for%20processive%20transcription%20elongation.%20Sci%20Adv.%202023;9:eadf8698." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR53">
<span class="label">53.</span><cite>van Essen D, Oruba A, Saccani S. High-Resolution ChIP-MNase Mapping of Nucleosome Positions at Selected Genomic Loci and Alleles. Methods Mol Biol. 2021;2351:123â€“45.
</cite> [<a href="https://doi.org/10.1007/978-1-0716-1597-3_7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34382187/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?van%20Essen%20D,%20Oruba%20A,%20Saccani%20S.%20High-Resolution%20ChIP-MNase%20Mapping%20of%20Nucleosome%20Positions%20at%20Selected%20Genomic%20Loci%20and%20Alleles.%20Methods%20Mol%20Biol.%202021;2351:123%E2%80%9345." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR54">
<span class="label">54.</span><cite>Buenrostro JD, Giresi PG, Zaba LC, Chang HY, Greenleaf WJ. Transposition of native chromatin for fast and sensitive epigenomic profiling of open chromatin, DNA-binding proteins and nucleosome position. Nat Methods. 2013;10:1213â€“8.
</cite> [<a href="https://doi.org/10.1038/nmeth.2688" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3959825/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24097267/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Buenrostro%20JD,%20Giresi%20PG,%20Zaba%20LC,%20Chang%20HY,%20Greenleaf%20WJ.%20Transposition%20of%20native%20chromatin%20for%20fast%20and%20sensitive%20epigenomic%20profiling%20of%20open%20chromatin,%20DNA-binding%20proteins%20and%20nucleosome%20position.%20Nat%20Methods.%202013;10:1213%E2%80%938." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR55">
<span class="label">55.</span><cite>Li C, Yin Z, Xiao R, Huang B, Cui Y, Wang H, Xiang Y, Wang L, Lei L, Ye J, et al. G-quadruplexes sense natural porphyrin metabolites for regulation of gene transcription and chromatin landscapes. Genome Biol. 2022;23:259.
</cite> [<a href="https://doi.org/10.1186/s13059-022-02830-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9753424/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36522639/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li%20C,%20Yin%20Z,%20Xiao%20R,%20Huang%20B,%20Cui%20Y,%20Wang%20H,%20Xiang%20Y,%20Wang%20L,%20Lei%20L,%20Ye%20J,%20et%20al.%20G-quadruplexes%20sense%20natural%20porphyrin%20metabolites%20for%20regulation%20of%20gene%20transcription%20and%20chromatin%20landscapes.%20Genome%20Biol.%202022;23:259." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR56">
<span class="label">56.</span><cite>Sidoli S, Kori Y, Lopes M, Yuan ZF, Kim HJ, Kulej K, Janssen KA, Agosto LM, Cunha J, Andrews AJ, Garcia BA. One minute analysis of 200 histone posttranslational modifications by direct injection mass spectrometry. Genome Res. 2019;29:978â€“87.
</cite> [<a href="https://doi.org/10.1101/gr.247353.118" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6581051/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31123082/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sidoli%20S,%20Kori%20Y,%20Lopes%20M,%20Yuan%20ZF,%20Kim%20HJ,%20Kulej%20K,%20Janssen%20KA,%20Agosto%20LM,%20Cunha%20J,%20Andrews%20AJ,%20Garcia%20BA.%20One%20minute%20analysis%20of%20200%20histone%20posttranslational%20modifications%20by%20direct%20injection%20mass%20spectrometry.%20Genome%20Res.%202019;29:978%E2%80%9387." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR57">
<span class="label">57.</span><cite>Love MI, Huber W, Anders S. Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2. Genome Biol. 2014;15:550.
</cite> [<a href="https://doi.org/10.1186/s13059-014-0550-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4302049/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25516281/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Love%20MI,%20Huber%20W,%20Anders%20S.%20Moderated%20estimation%20of%20fold%20change%20and%20dispersion%20for%20RNA-seq%20data%20with%20DESeq2.%20Genome%20Biol.%202014;15:550." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR58">
<span class="label">58.</span><cite>Lamb KN, Bsteh D, Dishman SN, Moussa HF, Fan H, Stuckey JI, Norris JL, Cholensky SH, Li D, Wang J, et al. Discovery and Characterization of a Cellular Potent Positive Allosteric Modulator of the Polycomb Repressive Complex 1 Chromodomain, CBX7. Cell Chem Biol. 2019;26(1365â€“1379):e1322.</cite> [<a href="https://doi.org/10.1016/j.chembiol.2019.07.013" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6800648/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31422906/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lamb%20KN,%20Bsteh%20D,%20Dishman%20SN,%20Moussa%20HF,%20Fan%20H,%20Stuckey%20JI,%20Norris%20JL,%20Cholensky%20SH,%20Li%20D,%20Wang%20J,%20et%20al.%20Discovery%20and%20Characterization%20of%20a%20Cellular%20Potent%20Positive%20Allosteric%20Modulator%20of%20the%20Polycomb%20Repressive%20Complex%201%20Chromodomain,%20CBX7.%20Cell%20Chem%20Biol.%202019;26(1365%E2%80%931379):e1322." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR59">
<span class="label">59.</span><cite>Stoll GA, Pandiloski N, Douse CH, Modis Y. Structure and functional mapping of the KRAB-KAP1 repressor complex. EMBO J. 2022;41:e111179.
</cite> [<a href="https://doi.org/10.15252/embj.2022111179" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9753469/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36341546/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Stoll%20GA,%20Pandiloski%20N,%20Douse%20CH,%20Modis%20Y.%20Structure%20and%20functional%20mapping%20of%20the%20KRAB-KAP1%20repressor%20complex.%20EMBO%20J.%202022;41:e111179." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR60">
<span class="label">60.</span><cite>Liang K: KAT6A Chimeras Forms a Self-Reinforcing Epigenetic Module with NURF and MLL/COMPASS in AML [RNA-Seq]. Gene Expression Omnibus 2025. <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE299379" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE299379</a>.</cite> [<a href="https://doi.org/10.1186/s13059-025-03743-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC12366150/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40830799/" class="usa-link">PubMed</a>]</li>
<li id="CR61">
<span class="label">61.</span><cite>Liang K: KAT6A Chimeras Forms a Self-Reinforcing Epigenetic Module with NURF and MLL/COMPASS in AML [CUT&amp;Run]. Gene Expression Omnibus 2025. <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE299380" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE299380</a>.</cite> [<a href="https://doi.org/10.1186/s13059-025-03743-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC12366150/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40830799/" class="usa-link">PubMed</a>]</li>
<li id="CR62">
<span class="label">62.</span><cite>Liang K: KAT6A Chimeras Forms a Self-Reinforcing Epigenetic Module with NURF and MLL/COMPASS in AML [ChIP-Seq]. Gene Expression Omnibus 2025. <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE299381" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE299381</a>.</cite> [<a href="https://doi.org/10.1186/s13059-025-03743-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC12366150/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40830799/" class="usa-link">PubMed</a>]</li>
<li id="CR63">
<span class="label">63.</span><cite>Liang K: KAT6A Chimeras Forms a Self-Reinforcing Epigenetic Module with NURF and MLL/COMPASS in AML [ATAC-Seq]. Gene Expression Omnibus 2025. <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE299382" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE299382</a>.</cite> [<a href="https://doi.org/10.1186/s13059-025-03743-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC12366150/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40830799/" class="usa-link">PubMed</a>]</li>
<li id="CR64">
<span class="label">64.</span><cite>Liang K: KAT6A Chimeras Forms a Self-Reinforcing Epigenetic Module with NURF and MLL/COMPASS in AML [ChIPMNase]. Gene Expression Omnibus 2025. <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE299383" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE299383</a>.</cite> [<a href="https://doi.org/10.1186/s13059-025-03743-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC12366150/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40830799/" class="usa-link">PubMed</a>]</li>
<li id="CR65">
<span class="label">65.</span><cite>Lv J, Liang K: Immunoprecipitation and mass spectrometry (IP-MS) on KAT6A chimeras. ProteomeXchange Consortium; 2025. <a href="https://www.ebi.ac.uk/pride/archive/projects/PXD061491" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ebi.ac.uk/pride/archive/projects/PXD061491</a>.</cite>
</li>
<li id="CR66">
<span class="label">66.</span><cite>Lv J, Liang K: Mass spectrometry analysis of post-translational modifications of K/C-III-expressing HEK293T cells treated by A485 for 6 and 12 hours. ProteomeXchange Consortium; 2025. <a href="https://www.ebi.ac.uk/pride/archive/projects/PXD061565" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ebi.ac.uk/pride/archive/projects/PXD061565</a>.</cite>
</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12366150/bin/13059_2025_3743_MOESM1_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link">Additional file 1: Figures S1-S10</a><sup> (42.9MB, docx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12366150/bin/13059_2025_3743_MOESM2_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">13059_2025_3743_MOESM2_ESM.xlsx</a><sup> (7MB, xlsx) </sup><p>Additional file 2: Tables S1-S9.Â Table S1. mRNA-seq analysis of c-Kit<sup>+</sup> bone marrow, MPs, K/C-III, and K/P-III leukemia cells. Table S2. IP-MS analysis of KAT6A and KAT6A fusions. Table S3. mRNA-seq analysis of K/C-III leukemia cells after BZ1 treatments. Table S4. mRNA-seq analysis of K/C-III leukemia cells after A485 treatments. Table S5. List of 770 core target genes of K/C-III. Table S6. List of KAT6A truncates and KAT6A fusion sequence. Table S7. Primer sequences of BPTF shRNA. Table S8. List of antibodies used in this study. Table S9. List of RT-qPCR primers.</p>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12366150/bin/13059_2025_3743_MOESM3_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link">13059_2025_3743_MOESM3_ESM.docx</a><sup> (1.2MB, docx) </sup><p>Additional file 3: Uncropped western blot and gel images</p>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>All data are available in the main text or the supplementary materials. The H3K9ac and H3K27me3 ChIP-seq data from 293Â T cells were originally deposited into GEO under the accession number: SRR19676715 [<a href="#CR25" class="usa-link" aria-describedby="CR25">25</a>] and SRR9604322 [<a href="#CR58" class="usa-link" aria-describedby="CR58">58</a>]. The H3K9me3 293Â T CUT&amp;RUN were obtained under the accession number: SRR21837874 and SRR21862863 [<a href="#CR59" class="usa-link" aria-describedby="CR59">59</a>]. The raw and processed mRNA-seq, ATAC-seq, and CUT&amp;RUN datasets supporting the conclusions of this article have been deposited in the Gene Expression Omnibus (GEO) repository under accession numbers: <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE299379" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE299379</a> [<a href="#CR60" class="usa-link" aria-describedby="CR60">60</a>], <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE299380" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE299380</a> [<a href="#CR61" class="usa-link" aria-describedby="CR61">61</a>], <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE299381" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE299381</a> [<a href="#CR62" class="usa-link" aria-describedby="CR62">62</a>], <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE299382" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE299382</a> [<a href="#CR63" class="usa-link" aria-describedby="CR63">63</a>], and <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE299383" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE299383</a> [<a href="#CR64" class="usa-link" aria-describedby="CR64">64</a>]. The mass spectrometry proteomics data generated in this study have been submitted to the ProteomeXchange Consortium via the PRIDE partner repository under the dataset identifier PXD061491 [<a href="#CR65" class="usa-link" aria-describedby="CR65">65</a>] and PXD061565 [<a href="#CR66" class="usa-link" aria-describedby="CR66">66</a>]. All unique materials generated are available from the corresponding author upon reasonable request. No custom scripts and software were used besides those mentioned in the Methods section.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Genome Biology are provided here courtesy of <strong>BMC</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1186/s13059-025-03743-y"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/13059_2025_Article_3743.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (6.3Â MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12366150/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12366150/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12366150%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366150/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12366150/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12366150/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40830799/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12366150/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40830799/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12366150/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12366150/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="AsuUactUQv7jq8LNREWVk1z0BKVHvePocFHsVVmUNXZCAikz9qpRNxVWOQV7R0yw">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
