
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Stacking ensemble learning models diagnose pulmonary infections using host transcriptome data from metatranscriptomics - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="D2E4AF818AF31EA305AF81000FF5E88B.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="scirep">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12368225/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Scientific Reports">
<meta name="citation_title" content="Stacking ensemble learning models diagnose pulmonary infections using host transcriptome data from metatranscriptomics">
<meta name="citation_author" content="Tian Zhang">
<meta name="citation_author_institution" content="Tsinghua Shenzhen International Graduate School, Tsinghua University, Shenzhen, People’s Republic of China">
<meta name="citation_author_institution" content="Peng Cheng Laboratory, Shenzhen, China">
<meta name="citation_author" content="Ying Deng">
<meta name="citation_author_institution" content="Tsinghua Shenzhen International Graduate School, Tsinghua University, Shenzhen, People’s Republic of China">
<meta name="citation_author" content="Wentao Wang">
<meta name="citation_author_institution" content="Zhengzhou KingMed Center for Clinical Laboratory, Zhengzhou, Henan People’s Republic of China">
<meta name="citation_author" content="Zhe Zhao">
<meta name="citation_author_institution" content="Zhengzhou KingMed Center for Clinical Laboratory, Zhengzhou, Henan People’s Republic of China">
<meta name="citation_author" content="Yiling Wu">
<meta name="citation_author_institution" content="Peng Cheng Laboratory, Shenzhen, China">
<meta name="citation_author" content="Haoqian Wang">
<meta name="citation_author_institution" content="Tsinghua Shenzhen International Graduate School, Tsinghua University, Shenzhen, People’s Republic of China">
<meta name="citation_author" content="Shutao Xia">
<meta name="citation_author_institution" content="Tsinghua Shenzhen International Graduate School, Tsinghua University, Shenzhen, People’s Republic of China">
<meta name="citation_author_institution" content="Peng Cheng Laboratory, Shenzhen, China">
<meta name="citation_author" content="Weifang Liao">
<meta name="citation_author_institution" content="College of life science and technology, Wuhan Polytechnic University, Wuhan, People’s Republic of China">
<meta name="citation_author" content="Weijie Liao">
<meta name="citation_author_institution" content="Tsinghua Shenzhen International Graduate School, Tsinghua University, Shenzhen, People’s Republic of China">
<meta name="citation_author_institution" content="Department of Hematology and Oncology, International Cancer Center, Shenzhen University General Hospital, Shenzhen University, Shenzhen, People’s Republic of China">
<meta name="citation_publication_date" content="2025 Aug 20">
<meta name="citation_volume" content="15">
<meta name="citation_firstpage" content="30516">
<meta name="citation_doi" content="10.1038/s41598-025-15914-9">
<meta name="citation_pmid" content="40835893">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12368225/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12368225/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12368225/pdf/41598_2025_Article_15914.pdf">
<meta name="description" content="The prompt diagnosis of pulmonary infections with unknown etiology in patients in severe condition remains a challenge due to the lack of rapid and effective diagnostic methods. While metatranscriptomic sequencing offers a powerful approach, its ...">
<meta name="og:title" content="Stacking ensemble learning models diagnose pulmonary infections using host transcriptome data from metatranscriptomics">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="The prompt diagnosis of pulmonary infections with unknown etiology in patients in severe condition remains a challenge due to the lack of rapid and effective diagnostic methods. While metatranscriptomic sequencing offers a powerful approach, its ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12368225/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12368225">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1038/s41598-025-15914-9"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/41598_2025_Article_15914.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12368225%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12368225/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12368225/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12368225/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-scirep.jpg" alt="Scientific Reports logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Scientific Reports" title="Link to Scientific Reports" shape="default" href="http://www.nature.com/srep" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Sci Rep</button></div>. 2025 Aug 20;15:30516. doi: <a href="https://doi.org/10.1038/s41598-025-15914-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1038/s41598-025-15914-9</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Sci%20Rep%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sci%20Rep%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Sci%20Rep%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Sci%20Rep%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Stacking ensemble learning models diagnose pulmonary infections using host transcriptome data from metatranscriptomics</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20T%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Tian Zhang</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Tian Zhang</span></h3>
<div class="p">
<sup>1</sup>Tsinghua Shenzhen International Graduate School, Tsinghua University, Shenzhen, People’s Republic of China </div>
<div class="p">
<sup>2</sup>Peng Cheng Laboratory, Shenzhen, China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20T%22%5BAuthor%5D" class="usa-link"><span class="name western">Tian Zhang</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Deng%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Ying Deng</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Ying Deng</span></h3>
<div class="p">
<sup>1</sup>Tsinghua Shenzhen International Graduate School, Tsinghua University, Shenzhen, People’s Republic of China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Deng%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Ying Deng</span></a>
</div>
</div>
<sup>1,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20W%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Wentao Wang</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Wentao Wang</span></h3>
<div class="p">
<sup>5</sup>Zhengzhou KingMed Center for Clinical Laboratory, Zhengzhou, Henan People’s Republic of China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20W%22%5BAuthor%5D" class="usa-link"><span class="name western">Wentao Wang</span></a>
</div>
</div>
<sup>5</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhao%20Z%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Zhe Zhao</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Zhe Zhao</span></h3>
<div class="p">
<sup>5</sup>Zhengzhou KingMed Center for Clinical Laboratory, Zhengzhou, Henan People’s Republic of China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhao%20Z%22%5BAuthor%5D" class="usa-link"><span class="name western">Zhe Zhao</span></a>
</div>
</div>
<sup>5</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wu%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Yiling Wu</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Yiling Wu</span></h3>
<div class="p">
<sup>2</sup>Peng Cheng Laboratory, Shenzhen, China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wu%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yiling Wu</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20H%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Haoqian Wang</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Haoqian Wang</span></h3>
<div class="p">
<sup>1</sup>Tsinghua Shenzhen International Graduate School, Tsinghua University, Shenzhen, People’s Republic of China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20H%22%5BAuthor%5D" class="usa-link"><span class="name western">Haoqian Wang</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Xia%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Shutao Xia</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Shutao Xia</span></h3>
<div class="p">
<sup>1</sup>Tsinghua Shenzhen International Graduate School, Tsinghua University, Shenzhen, People’s Republic of China </div>
<div class="p">
<sup>2</sup>Peng Cheng Laboratory, Shenzhen, China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Xia%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Shutao Xia</span></a>
</div>
</div>
<sup>1,</sup><sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Liao%20W%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Weifang Liao</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Weifang Liao</span></h3>
<div class="p">
<sup>4</sup>College of life science and technology, Wuhan Polytechnic University, Wuhan, People’s Republic of China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Liao%20W%22%5BAuthor%5D" class="usa-link"><span class="name western">Weifang Liao</span></a>
</div>
</div>
<sup>4,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Liao%20W%22%5BAuthor%5D" class="usa-link" aria-describedby="id9"><span class="name western">Weijie Liao</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">Weijie Liao</span></h3>
<div class="p">
<sup>1</sup>Tsinghua Shenzhen International Graduate School, Tsinghua University, Shenzhen, People’s Republic of China </div>
<div class="p">
<sup>3</sup>Department of Hematology and Oncology, International Cancer Center, Shenzhen University General Hospital, Shenzhen University, Shenzhen, People’s Republic of China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Liao%20W%22%5BAuthor%5D" class="usa-link"><span class="name western">Weijie Liao</span></a>
</div>
</div>
<sup>1,</sup><sup>3,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>Tsinghua Shenzhen International Graduate School, Tsinghua University, Shenzhen, People’s Republic of China </div>
<div id="Aff2">
<sup>2</sup>Peng Cheng Laboratory, Shenzhen, China </div>
<div id="Aff3">
<sup>3</sup>Department of Hematology and Oncology, International Cancer Center, Shenzhen University General Hospital, Shenzhen University, Shenzhen, People’s Republic of China </div>
<div id="Aff4">
<sup>4</sup>College of life science and technology, Wuhan Polytechnic University, Wuhan, People’s Republic of China </div>
<div id="Aff5">
<sup>5</sup>Zhengzhou KingMed Center for Clinical Laboratory, Zhengzhou, Henan People’s Republic of China </div>
<div class="author-notes p">
<div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div>
<div class="fn" id="_eqcntrb93pmc__">
<sup>#</sup><p class="display-inline">Contributed equally.</p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2024 Jun 18; Accepted 2025 Aug 11; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by-nc-nd/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12368225  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40835893/" class="usa-link">40835893</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<p id="Par12">The prompt diagnosis of pulmonary infections with unknown etiology in patients in severe condition remains a challenge due to the lack of rapid and effective diagnostic methods. While metatranscriptomic sequencing offers a powerful approach, its clinical utility is often limited by issues of timeliness. In this study, we conducted metatranscriptomic sequencing on bronchoalveolar lavage fluid (BALF) collected from critically ill, severely ill, and ICU patients. Based on microbial detection results, patients were classified into four types: negative, bacterial infection, viral infection, and fungal infection. To identify host gene expression signatures associated with infection, we screened characteristic genes from human metatranscriptomic data by comparing 70% of patients with confirmed infections vs. non-infections. Leveraging these characteristic genes, we constructed classification sub-models employing 13 types of machine learning algorithms, and we further integrated these sub-models into stacking-based ensemble models with Lasso regression, resulting in diagnostic models that required only a small set of gene expression inputs. The average performance of five-fold cross-validation demonstrated high diagnostic accuracy: distinguishing infection from non-infection (AUC = 0.984), bacterial infection from non-bacterial infection (AUC = 0.98), and viral infection from non- viral infection (AUC = 0.98). Test cohorts’ results demonstrated the method’s high diagnostic accuracy consistency with metatranscriptomic sequencing in discerning patient infection status (AUC = 0.865) and the type of infection (viral: AUC = 0.934, bacterial: AUC = 0.871). Our study presented a rapid and inexpensive adjunctive diagnostic strategy that achieves diagnostic accuracy comparable to metatranscriptomic sequencing, enabling timely identification of both infection status and type in pulmonary infections.</p>
<section id="sec1"><h3 class="pmc_sec_title">Supplementary Information</h3>
<p>The online version contains supplementary material available at 10.1038/s41598-025-15914-9.</p></section><section id="kwd-group1" lang="en" class="kwd-group"><p><strong>Keywords:</strong> Patients in severe condition, Pulmonary infections, Prompt diagnosis, Metatranscriptomic sequencing, Machine learning models</p></section><section id="kwd-group2" class="kwd-group"><p><strong>Subject terms:</strong> Computational biology and bioinformatics, Genetics</p></section></section><section id="Sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p id="Par13">As a vital organ for gas exchange between the human body and the external environment, the lungs are highly susceptible to infection due to various factors, including microbial invasion, environmental exposure, and iatrogenic causes. However, the histopathological features of pulmonary infections, such as inflammatory infiltration and tissue necrosis, often closely resemble those of non-infectious diseases, posing a significant diagnostic challenge for surgical pathologists. Failure to establish a timely and accurate diagnosis may lead to disease progression, including pulmonary fibrosis or even systemic dissemination to other organs. In severe cases, patients can rapidly deteriorate into respiratory failure or sepsis, ultimately resulting in death<sup><a href="#CR1" class="usa-link" aria-describedby="CR1">1</a></sup>. Therefore, timely and precise diagnosis, followed by targeted intervention, is critical for improving clinical outcomes.</p>
<p id="Par14">Patients with severe illness, critical condition, and those admitted to the ICU frequently often face life-threatening pulmonary infections of undetermined etiology, posing major diagnostic and therapeutic challenges in clinical management<sup><a href="#CR2" class="usa-link" aria-describedby="CR2">2</a>,<a href="#CR3" class="usa-link" aria-describedby="CR3">3</a></sup>. While conventional methods for detecting pulmonary infections encompass physical examinations, imaging studies, microbial cultures, and blood tests, they are encumbered by issues such as low specificity, delays, restricted detection capabilities, and intricate operational procedures<sup><a href="#CR4" class="usa-link" aria-describedby="CR4">4</a>,<a href="#CR5" class="usa-link" aria-describedby="CR5">5</a></sup>. In the absence of a confirmed diagnosis, empirical therapy predominates, typically involving the combination use of broad-spectrum antibiotics, antiviral agents, and/or antifungal drugs. This approach is associated with considerable costs, lacks specificity, and contributes to antibiotic misuse, drug resistance, and potential adverse effects<sup><a href="#CR3" class="usa-link" aria-describedby="CR3">3</a></sup>.</p>
<p id="Par15">In recent years, metagenomic sequencing has emerged as a promising strategy for diagnosing pulmonary infections<sup><a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>,<a href="#CR7" class="usa-link" aria-describedby="CR7">7</a></sup>. Both metagenomic and metatranscriptomic detection technologies offer substantial improvements in detection accuracy<sup><a href="#CR8" class="usa-link" aria-describedby="CR8">8</a></sup> and microbial spectrum coverage including bacteria<sup><a href="#CR9" class="usa-link" aria-describedby="CR9">9</a></sup> viruses<sup><a href="#CR10" class="usa-link" aria-describedby="CR10">10</a></sup> and fungi, thereby enabling personalized treatment<sup><a href="#CR11" class="usa-link" aria-describedby="CR11">11</a></sup>. Nonetheless, these techniques present notable limitations, including data analysis complexity, and the necessity for specialized knowledge and skills, resulting in prolonged turnaround times for results<sup><a href="#CR12" class="usa-link" aria-describedby="CR12">12</a></sup>. These limitations could result in the oversight of optimal treatment opportunities for patients with severe illness, critical conditions, and those admitted to the ICU. Consequently, the development of rapid and inexpensive diagnostic approaches with diagnostic precision akin to metagenomic sequencing, capable of discerning infection status and type, assumes paramount importance. Such advancements would be pivotal in guiding timely, targeted antibiotic, antiviral, or antifungal interventions for these patients. This not only can significantly decrease drug selection pressure and adverse drug reactions, but also contribute to improving patient outcomes.</p>
<p id="Par16">To address this critical need, we collected BALF samples from critically ill and ICU patients for metatranscriptomic sequencing analysis. Based on the metatranscriptomic microbial detection results, patients were categorized into negative, bacterial infection, viral infection, and fungal infection, and characteristic gene screening was conducted on the transcriptome data of human genes. Using the transcriptome data of these characteristic genes, classification diagnostic sub-models were constructed employing various machine learning algorithms. However, a solitary machine learning model may encounter limitations stemming from data quality, risks of overfitting, and deficiencies in model generalization within the biological context. In contrast, ensemble learning amalgamates the strengths of diverse models, thereby reducing model bias and overfitting risk, ensuring generalizability across subsets, laying the foundation for robust ensemble learning and enhancing the robustness of feature interpretation. Therefore, stacking-based ensemble learning models for each sub-model were developed using Lasso, ultimately yielding a rapid, cost-effective classification diagnostic method that only necessitates the detection of a few characteristic genes. Test results indicated that the developed method exhibited high consistency in diagnostic accuracy with metatranscriptomic sequencing in distinguishing whether patients were infected and the type of infection (viral and bacterial), thus serving as a potential auxiliary diagnostic method for pulmonary infections of unknown etiology in patients with severe illness, critical condition, and those admitted to the ICU.</p></section><section id="Sec2"><h2 class="pmc_sec_title">Methods</h2>
<section id="Sec3"><h3 class="pmc_sec_title">Study design and participants</h3>
<p id="Par17">This retrospective study analyzed BALF specimens collected from 180 critically ill patients admitted to the intensive care unit (ICU) between 2021 and 2023 from Zhengzhou Kingmed Diagnostics. Patients with respiratory system infections of unknown etiology were included, while those under 18 years of age were excluded. All specimens underwent metatranscriptomic sequencing. Based on metatranscriptomic data analysis (Supplementary Table <a href="#MOESM1" class="usa-link">S1</a>), data from non-infected patients (<em>n</em> = 41) and infected patients (<em>n</em> = 139) were included in subsequent analyses. Infected patients were further stratified into three diagnostic groups: bacterial infection (<em>n</em> = 91), viral infection (<em>n</em> = 33), and fungal infection (<em>n</em> = 15). The pathogenic microbial infection model incorporated 41 non-infected cases alongside 139 confirmed infected cases. For pathogenic bacterial infection classification, the cohort comprised 91 bacterial infected patients compared with 89 non-bacterial patients. Similarly, the pathogenic viral infection model analyzed 33 viral infected cases against 147 non-viral infected cases. This study was approved by the ethics committee of Kingmed Diagnostics (No.2023160), and all experiments were performed in accordance with relevant guidelines and regulations. Due to the retrospective nature of the study, the ethics committee of Kingmed Diagnostics waived the need of obtaining informed consent.</p></section><section id="Sec4"><h3 class="pmc_sec_title">Metatranscriptomic sequencing</h3>
<p id="Par18">The Metatranscriptomic sequencing was carried out by KingMed Diagnostics. General DNA/RNA extraction kit (TR202-50) from GENSTONE Biotech was employed to extract total RNA from bronchoalveolar lavage fluid. Pathogenic microbial RNA detection kits (KS619-RNAmN48) from KingCreate Biotechnology were used to construct sequencing libraries, with library lengths ranging from 250 to 400 base pairs. Sequencing was performed using the Illumina NextSeq 550 platform. Each sample yielded over 2 Gb of raw data.</p></section><section id="Sec5"><h3 class="pmc_sec_title">Bioinformatics analysis and statistical tests</h3>
<p id="Par19">Metatranscriptomic data of microbiota processing: After undergoing quality control with the <em>fastp</em> software<sup><a href="#CR13" class="usa-link" aria-describedby="CR13">13</a></sup> the raw data underwent filtering to remove low-quality and low-complexity reads. Subsequently, adapter sequences and duplicate reads were eliminated, followed by sequence alignment. Reads aligned to microorganisms were then quantified. The proprietary database developed by KingMed Diagnostics comprised over 11,000 bacterial species, 7,300 viral species, and 1,600 fungal species.</p>
<p id="Par20">Metatranscriptomic data of human processing: FastQC<sup><a href="#CR14" class="usa-link" aria-describedby="CR14">14</a></sup> was employed to assess the quality of raw sequencing reads, followed by alignment using Hisat2 software<sup><a href="#CR15" class="usa-link" aria-describedby="CR15">15</a>,<a href="#CR16" class="usa-link" aria-describedby="CR16">16</a></sup> and deduplication and sorting using Samtools software<sup><a href="#CR17" class="usa-link" aria-describedby="CR17">17</a></sup>. Subsequently, featureCounts software<sup><a href="#CR18" class="usa-link" aria-describedby="CR18">18</a></sup> was utilized to obtain count data and calculate Transcripts Per Million (TPM). Utilizing the <em>cmdscale</em> function and Vegan package<sup><a href="#CR19" class="usa-link" aria-describedby="CR19">19</a></sup> both t-distributed Stochastic Neighbor Embedding (tSNE) and Principal Component Analysis (PCA) were conducted on TPM data derived from two sample groups. Differential analysis on count data from the same two sample groups was carried out using DESeq2<sup>20</sup>, EdgeR<sup><a href="#CR21" class="usa-link" aria-describedby="CR21">21</a></sup>and Limma packages<sup><a href="#CR22" class="usa-link" aria-describedby="CR22">22</a></sup>. Simultaneously, statistical analysis on Transcripts Per Million (TPM) data from the two groups was performed using Wilcoxon tests. Hyperparamter details were set as follows: DESeq2: fitType = “mean”, minReplicatesForReplace = 7, parallel = FALSE. Genes with padj &lt; 0.05 and |log2FC| &gt; 1 were considered significant; limma: The design matrix was constructed using model.matrix(~ GroupLabel), and empirical Bayes moderation was applied via eBayes(). Significance thresholds: P.Value &lt; 0.05, |logFC| &gt; 1; Wilcoxon test: Non-parametric rank-sum test applied on a per-gene basis, using default settings. Genes with <em>p</em> &lt; 0.05 and |log2FC| &gt; 1 were retained; edgeR: Normalization with calcNormFactors(), dispersion estimated using estimateDisp(), followed by glmFit() and glmLRT() for model fitting and hypothesis testing. Design matrix: model.matrix(~ GroupLabel). Genes with P.Value &lt; 0.05 and |logFC| &gt; 1 were selected. Subsequently, intersecting genes identified from the four differential analysis methods were selected to determine differentially expressed genes, and the <em>ggpubr</em> package<sup><a href="#CR23" class="usa-link" aria-describedby="CR23">23</a></sup> was employed to generate a Venn diagram. Heatmaps were generated using the <em>pheatmap</em> package<sup><a href="#CR24" class="usa-link" aria-describedby="CR24">24</a></sup>. Additionally, the <em>clusterProfiler</em> package<sup><a href="#CR25" class="usa-link" aria-describedby="CR25">25</a></sup> was utilized for Gene Ontology (GO) and Kyoto Encyclopedia of Genes and Genomes (KEGG)<sup><a href="#CR26" class="usa-link" aria-describedby="CR26">26</a>–<a href="#CR28" class="usa-link" aria-describedby="CR28">28</a></sup> pathway analysis, while the <em>IOBR</em> package<sup><a href="#CR29" class="usa-link" aria-describedby="CR29">29</a></sup> was employed for immune infiltration analysis.</p></section><section id="Sec6"><h3 class="pmc_sec_title">Machine learning model construction</h3>
<p id="Par21">This study included a total of 180 samples for a binary classification task (infection vs. non-infection, bacterial infection vs. non-bacterial infection or viral infection vs. non- viral infection). The samples were randomly divided into training set and testing set at a 7:3 ratio, resulting in 126 samples used for training and 54 for testing. Differential genes were analyzed using the training set and further screened using three machine learning algorithms: Support Vector Machine (SVM), Random Forest (RF), and Least Absolute Shrinkage and Selection Operator (Lasso). The intersection genes, representing the input features for the model, were obtained through analysis using the <em>ggpubr</em> package. Subsequently, thirteen algorithms, including Gradient Boosting Machine (GBM), Naive Bayes, Neural Network, RF, Elastic Net, Ridge Regression, Decision Tree, Support Vector Regression (SVR), Linear Model (LM), SVM, RPart, Generalized Linear Models (GLM), and Lasso, were employed to construct diagnostic sub-models. The performance of each model was assessed, and finally, the efficiency of each sub-model was integrated using the Lasso algorithm to build a classification diagnostic model based on ensemble learning. The hyperparamter details of each model were shown in Supplementary Table <a href="#MOESM2" class="usa-link">S2</a>. The effectiveness of the model was evaluated using 5-fold cross-validation on the training set samples, visualized via <em>pROC</em> (Receiver Operating Characteristic Curve) package<sup><a href="#CR30" class="usa-link" aria-describedby="CR30">30</a></sup> and further tested on the test set samples to assess its performance (Fig. <a href="#Fig1" class="usa-link">1</a>). The key components of the entire modeling pipeline are as follows:</p>
<figure class="fig xbox font-sm" id="Fig1"><h4 class="obj_head">Fig. 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12368225_41598_2025_15914_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/157b/12368225/b0851413a06b/41598_2025_15914_Fig1_HTML.jpg" loading="lazy" id="d33e440" height="945" width="762" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>The workflow of this study. This study presents a comprehensive procedure for analyzing host transcriptional profiles in severe respiratory infections. BALF specimens from 180 critically ill patients were subjected to metatranscriptomic analysis. Samples were categorized into four groups based on microbiological findings: microbial infection, bacterial infection, viral infection, and non-infected. Each group was randomly divided into training and testing sets at a 7:3 ratio. Due to limited fungal samples and substantial interindividual variability, fungal specimens were excluded from subsequent analysis. Differential analysis of host transcriptomes identified differentially expressed genes. SVM, RF, and Lasso methods were utilized for host biomarker selection. Integration learning was employed to construct a final diagnostic model by combining 13 machine learning sub-models. The effectiveness and validation of the model were assessed using five-fold cross-validation on the training set and validation on the testing set.</p></figcaption></figure><ol class="list" style="list-style-type:decimal">
<li><p id="Par24">Dependent Variable (Target): The outcome variable (AllLable) is a binary classification label indicating clinical infection status: - Class 0: non-infected, - Class 1: infected. We constructed four types of datasets: negative, bacteria, virus, and fungus. This binary variable serves as the target (y) for all supervised classification models.</p></li>
<li><p id="Par25">Predictor Variables (Features): Input features are selected from differentially expressed genes (DEGs), identified through statistical comparisons (DESeq2, limma, Wilcoxon). Prior to modeling, genes with low variance were removed. Features were ranked using standard deviation, and selected genes were assembled into the predictor matrix: x_train &lt;- as. matrix (gene_expression [, colnames(gene_expression) %in% ModelGene$ENSG]).</p></li>
<li><p id="Par26">Feature Selection and Cross-Validation: Each model incorporates feature selection and validation were showed in Supplementary Table S3.</p></li>
<li><p id="Par27">Classification Task Type: All models were built for binary classification, aiming to distinguish between infected and non-infected samples using gene expression data.</p></li>
<li><p id="Par28">Evaluation Metrics: To assess model performance and robustness, we applied Accuracy (Proportion of correctly predicted samples), AUC (Reflects model discrimination ability), Mean Predicted Probability (Average model confidence for each sample), Cross-validation average score (Stability across folds), four assessment criteria.</p></li>
</ol></section></section><section id="Sec7"><h2 class="pmc_sec_title">Results</h2>
<section id="Sec8"><h3 class="pmc_sec_title">Model construction for pathogenic microbial infection</h3>
<p id="Par29">To develop a diagnostic model for pathogenic microbial infections, we first established comparative cohorts based on metagenomic sequencing results. There were no significant differences in gender and age between the pathogenic microbial infected and non-infected cohorts (Fig. <a href="#Fig2" class="usa-link">2</a>A). tSNE analysis revealed significant differentiation between the two groups of samples (<em>p</em> = 0.002), and consistent results were obtained through PCA analysis (<em>p</em> &lt; 0.001) (Fig. <a href="#Fig2" class="usa-link">2</a>B and Figure <a href="#MOESM1" class="usa-link">S1</a>A). Immune cell composition analysis in the BALF showed a significant increase in immune cells promoting the immune response, including B cells, eosinophils, CD4 + T cells, CD8 + T cells, Th2 cells, and plasmacytoid dendritic cells, in the cohort with pathogenic microbial infections compared to those without, whereas the population of M2-type macrophages suppressing the immune response was significantly reduced. This suggested a heightened immune response in patients with detected pathogenic microbes compared to those without (Fig. <a href="#Fig2" class="usa-link">2</a>C).</p>
<figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12368225_41598_2025_15914_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/157b/12368225/a33eab66872a/41598_2025_15914_Fig2_HTML.jpg" loading="lazy" id="d33e514" height="416" width="669" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Model construction for pathogenic microbial infections. (A) Analysis of age and gender disparities in patients with detected and undetected pathogenic microorganisms. (B) tSNE of host genes in the metatranscriptomic data of patients with detected (<em>n</em> = 139) and undetected (<em>n</em> = 41) pathogenic microorganisms. (C) Analysis of immune cell composition in BALF from two groups of patients using the deconvolution algorithm. (D) Venn diagram analysis of differential genes commonly identified by four differential gene algorithms, Wilcoxon screening threshold as adjust-<em>p</em> &lt; 0.05. (E) GO enrichment analysis of differential genes, all the terms’ pvalue &lt; 0.05. (F) KEGG enrichment analysis of differential genes, all the pathways’ pvalue &lt; 0.05. (G) Input genes for feature selection by SVM, RF, and Lasso models. (H-I) Coefficient path of integrated model based on Lasso (H) and 10-fold cross-validation regularization path (I). (J) The coefficients of each sub-model in the Lasso linear regression integrated model. (K) Five-fold cross-validation of training set samples for model evaluation and (L) testing set samples for model test.</p></figcaption></figure><p id="Par31">To identify DEGs between the two cohorts, we employed three algorithms (Figure <a href="#MOESM1" class="usa-link">S1</a>B-D). Furthermore, we performed DEG analysis using Wilcoxon test. By intersecting the results from these four methods, we identified a set of DEGs between the two cohorts, with 132 genes downregulation, compared to the pathogenic microbial non- infection cohort (Fig. <a href="#Fig2" class="usa-link">2</a>D and Figure <a href="#MOESM1" class="usa-link">S1</a>E). GO enrichment analysis indicated that these DEGs are associated with cellular translation, oxidative phosphorylation, cellular respiration, electron transport chain, respiratory complexes, cytochrome c oxidase activity, as well as cell adhesion and migration (Fig. <a href="#Fig2" class="usa-link">2</a>E). KEGG enrichment analysis revealed that the differential genes mainly affect ribosome biogenesis, mitochondrial energy metabolism and quality control, and intracellular oxidative stress (Fig. <a href="#Fig2" class="usa-link">2</a>F). The combined results of GO and KEGG enrichment analyses suggested that these DEGs mainly impact cellular energy metabolism and substance metabolism, which are significantly affected following pathogenic microbial infections.</p>
<p id="Par32">Subsequently, we conducted screening of DEGs related to the presence or absence of pathogenic microbial infections. SVM algorithm identified 66 genes, RF algorithm identified 66 genes, and Lasso algorithm identified 9 genes. By taking the intersection of these three sets of genes, FIS1, MARCKSL1, RPL8, RPS15, RPS5, and STUB1 were selected as input genes for the sub-model (Fig. <a href="#Fig2" class="usa-link">2</a>G). We then constructed thirteen classification sub-models using thirteen algorithms, and the prediction outputs of each sample in the test set across all 13 models are presented in Supplementary Table <a href="#MOESM4" class="usa-link">S4</a>. The SHAP values of each input gene were shown in Figure <a href="#MOESM1" class="usa-link">S1</a>F. Subsequently, we integrated these sub-models using Lasso algorithm and selected the best regularization parameter (Lambda) through coefficient calculation (Fig. <a href="#Fig2" class="usa-link">2</a>H) and 10-fold cross-validation (Fig. <a href="#Fig2" class="usa-link">2</a>I), obtaining the optimal Lambda value of 3.3e-03. The determined lambda value was subsequently employed to construct a classification model, formally expressed as: S = −0.13 + 0.14E<sub>MLP</sub> + 0.04 E<sub>NaiveBayes</sub> + 0.11 E<sub>NeuralNet</sub> + 0.49 E<sub>RF</sub> + 0.48 E<sub>GBM</sub> (Fig. <a href="#Fig2" class="usa-link">2</a>J). Five-fold cross-validation results on the training set demonstrated the model’s robust diagnostic performance, with an AUC value of 0.984 (Fig. <a href="#Fig2" class="usa-link">2</a>K). Furthermore, test using the test set yielded an AUC value of 0.865 (Fig. <a href="#Fig2" class="usa-link">2</a>L), indicating good consistency of the metatranscriptomic detection of pathogenic microbial infections.</p></section><section id="Sec9"><h3 class="pmc_sec_title">Model construction for pathogenic bacterial infection</h3>
<p id="Par33">In this part of the research, we aimed to establish a diagnostic model that aligned with the results of metatranscriptomic detection of solely bacterial infection. Cases without bacterial infection and with bacterial infection were identified (Fig. <a href="#Fig3" class="usa-link">3</a>A). A substantial disparity emerged between the cohorts, as evidenced by tSNE (<em>p</em> &lt; 0.001) and PCA(<em>p</em> &lt; 0.001) (Fig. <a href="#Fig3" class="usa-link">3</a>B and Figure <a href="#MOESM2" class="usa-link">S2</a>A). DEGs were discerned using the same method as described above, with 1719 genes downregulation and 44 genes upregulation compared to cases without infection (Fig. <a href="#Fig3" class="usa-link">3</a>C and S2B-E). GO enrichment analysis unveiled associations with bacterial infection, such as regulation of innate immune response, response to lipopolysaccharide, response to molecule of bacterial origin, focal adhesion, MHC protein complex binding (Fig. <a href="#Fig3" class="usa-link">3</a>D). KEGG enrichment analysis revealed differential gene impacted on TNF signaling, NF-kappa B signaling, Tight junction, Necroptosis, MAPK signaling and bacterial invasion of epithelial cells, which were all associated with bacterial infection (Fig. <a href="#Fig3" class="usa-link">3</a>E). Besides, Gene Set Enrichment Analysis (GSEA) enrichment analysis also identified numerous pathways associated with bacterial infection, for instance antigen processing and presentation, natural killer cell mediated cytotoxicity, lysosome, apoptosis, endocytosis and MAPK signaling pathway (Fig. <a href="#Fig3" class="usa-link">3</a>F). These results suggested that the screened DEGs were highly correlated with bacterial infection.</p>
<figure class="fig xbox font-sm" id="Fig3"><h4 class="obj_head">Fig. 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12368225_41598_2025_15914_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/157b/12368225/00403bb08f3e/41598_2025_15914_Fig3_HTML.jpg" loading="lazy" id="d33e660" height="460" width="669" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Model construction for pathogenic bacterial infections. (<strong>A</strong>) Examination of demographic variations in individuals infected with Bacteria (<em>n</em> = 91) and without pathogenic microorganisms (<em>n</em> = 89). (<strong>B</strong>) Utilization of tSNE to analyze the genetic makeup of hosts in metatranscriptomic datasets of patients harboring bacterial infection and non-bacterial infection. (<strong>C</strong>) Investigation into the overlap of significant gene differences derived from four distinct algorithms using Venn diagrams, employing a Wilcoxon threshold with adjust-<em>p</em> &lt; 0.05. (<strong>D</strong>) GO enrichment analyses on disparate gene sets, with significance established at a p-value threshold of &lt; 0.05 for all terms. (<strong>E</strong>) Assessment of KEGG enrichment on differential gene sets, setting a significance threshold of &lt; 0.05 for all pathways. (F) GSEA enrichment analysis, setting a significance threshold of &lt; 0.05 for all pathways. (<strong>G</strong>) Selection of feature genes for predictive modeling through SVM, RF, and Lasso methodologies, the Mann-Whitney U was used to define significant differences, <sup>***</sup><em>p</em> &lt; 0.001. (<strong>H</strong>-<strong>I</strong>) Visualization of coefficient paths within an integrated model based on Lasso regularization (H) and elucidation of regularization paths via 10-fold cross-validation (<strong>I</strong>). (<strong>J</strong>) Determination of individual coefficients within the integrated Lasso linear regression model. (K) Application of five-fold cross-validation to assess model performance on training set samples and (<strong>L</strong>) Test model efficacy on independent testing set samples.</p></figcaption></figure><p id="Par35">DEGs underwent further screening, resulting in 10 genes identified by SVM, 10 genes selected by RF, and 17 genes determined by Lasso. By identifying the intersecting genes, AKT1, CRIP1, GTF3A, HDGF, HLA-DPB1, OGFR, PEBP1 and RPL13 genes were obtained and as input genes for next model constructions (Fig. <a href="#Fig3" class="usa-link">3</a>G). The 13 classification sub-models were constructed, and the prediction outputs of each sample in the test set across all 13 models are presented in Supplementary Table <a href="#MOESM4" class="usa-link">S4</a>. The SHAP value of each input gene were shown in Figure <a href="#MOESM2" class="usa-link">S2</a>F. The 13 sub-models were integrated via the Lasso algorithm, coupled with selection of the optimal regularization parameter (Lambda) through coefficient calculation (Fig. <a href="#Fig3" class="usa-link">3</a>H) and 10-fold cross-validation (Fig. <a href="#Fig3" class="usa-link">3</a>I), yielded an optimal Lambda value of 5.63e-04. Leveraging this Lambda value, an integrated learning model was constructed, as represented by: S = −0.05 + 0.06E<sub>MLP</sub> + 0.25 E<sub>NeuralNet</sub> + 0.47 E<sub>RF</sub> + 0.33 E<sub>GBM</sub> (Fig. <a href="#Fig3" class="usa-link">3</a>J). Five-fold cross-validation results on the training set underscored the model’s robust diagnostic prowess, with an AUC value of 0.98 (Fig. <a href="#Fig3" class="usa-link">3</a>K). Furthermore, test utilizing the test set yielded an AUC value of 0.871 (Fig. <a href="#Fig3" class="usa-link">3</a>L), indicating substantial concordance with the metatranscriptomic detection of pathogenic bacteria infections.</p></section></section><section id="Sec10"><h2 class="pmc_sec_title">Model construction for pathogenic viral infection</h2>
<p id="Par36">Following our investigation, we endeavored to formulate a diagnostic framework congruent with findings from metatranscriptomic assessments exclusive to viral infections. Among our cohort, delineation revealed 147 cases devoid of viral infection and 33 cases characterized by viral infection (Fig. <a href="#Fig4" class="usa-link">4</a>A). A conspicuous dichotomy surfaced between these cohorts, as substantiated by tSNE (<em>p</em> &lt; 0.001) and PCA (<em>p</em> &lt; 0.001) (Fig. <a href="#Fig4" class="usa-link">4</a>B and Figure <a href="#MOESM3" class="usa-link">S3</a>A). Employing analogous methodologies as aforementioned, we identified 598 genes exhibiting downregulation and 2 genes displaying upregulation in comparison to non-viral infection cases (Fig. <a href="#Fig4" class="usa-link">4</a>C and Figure <a href="#MOESM3" class="usa-link">S3</a>B-E). GO enrichment analysis illuminated associations indicative of viral infection, encompassing the viral process, antigen processing and presentation, MHC protein complex assembly, aerobic respiration, immune receptor activity and peptide antigen binding (Fig. <a href="#Fig4" class="usa-link">4</a>D). Moreover, KEGG enrichment analysis unveiled perturbed genes implicated in TH17 cell differentiation, Th1 and Th2 cell differentiation, Oxidative Phosphorylation, influenza A, COVID-19, Necroptosis, lysosome, antigen processing and presentation (Fig. <a href="#Fig4" class="usa-link">4</a>E). Furthermore, GSEA identified numerous pathways linked to mitochondrial function, including Huntington disease, Parkinson disease and Oxidative phosphorylation (Fig. <a href="#Fig4" class="usa-link">4</a>F). These findings collectively indicated a correlation between the identified DEGs and viral infection.</p>
<figure class="fig xbox font-sm" id="Fig4"><h3 class="obj_head">Fig. 4.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12368225_41598_2025_15914_Fig4_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/157b/12368225/59891a2d56ac/41598_2025_15914_Fig4_HTML.jpg" loading="lazy" id="d33e815" height="435" width="669" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Model construction for pathogenic viral infections. (<strong>A</strong>) Exploration of demographic disparities between individuals infected with Viral infection (<em>n</em> = 33) and those lacking pathogenic microorganism infection (<em>n</em> = 147). (<strong>B</strong>) Utilization of tSNE to analyze the genetic composition of hosts in metatranscriptomic datasets from patients with viral infection and non-viral infection. (<strong>C</strong>) Investigation into overlapping gene disparities using Venn diagrams derived from four distinct algorithms, employing a Wilcoxon threshold with adjust-<em>p</em> &lt; 0.05. (<strong>D</strong>) Conducting GO enrichment analyses on diverse gene sets, with significance determined at a p-value threshold of &lt; 0.05 for all terms. (<strong>E</strong>) Assessment of KEGG enrichment on differential gene sets, setting a significance threshold of &lt; 0.05 for all pathways. (<strong>F</strong>) Analysis using GSEA, with a significance threshold of &lt; 0.05 for all pathways. (<strong>G</strong>) Selection of feature genes for predictive model via SVM, RF, and Lasso methodologies, with significant differences defined using Mann-Whitney U test, <sup>*</sup><em>p</em> &lt; 0.05, <sup>**</sup><em>p</em> &lt; 0.01, <sup>***</sup><em>p</em> &lt; 0.001. (<strong>H</strong>-<strong>I</strong>) Representation of coefficient paths within an integrated model utilizing Lasso regularization (<strong>H</strong>) and elucidation of regularization paths via 10-fold cross-validation (<strong>I</strong>). (<strong>J</strong>) Determination of individual coefficients within the integrated Lasso linear regression model. (<strong>K</strong>-<strong>L</strong>) Application of five-fold cross-validation to assess model performance on training set samples (<strong>K</strong>) and test model efficacy on independent testing set samples (<strong>L</strong>).</p></figcaption></figure><p id="Par38">Subsequently, DEGs underwent further screening, yielding CPVL, DENND4C, MAFB, PTGER4, SLAMF8, TSPO and UBE2Q2 genes, which served as input features for subsequent model construction (Fig. <a href="#Fig4" class="usa-link">4</a>G). Thirteen classification sub-models were assembled and the prediction outputs of each sample in the test set across all 13 models are presented in Supplementary Table <a href="#MOESM4" class="usa-link">S4</a>. The SHAP value of each input gene were shown in Figure <a href="#MOESM3" class="usa-link">S3</a>F. The sub-models were integrated through the Lasso algorithm, incorporating the optimal regularization parameter (Lambda) determined via coefficient calculation (Fig. <a href="#Fig4" class="usa-link">4</a>H) and 10-fold cross-validation (Fig. <a href="#Fig4" class="usa-link">4</a>I), resulting in an optimal Lambda value of 1.29e-03. Leveraging this Lambda value, a composite classification model was devised, represented by the equation: S = −0.20 + 0.28E<sub>MLP</sub> + 0.15 E<sub>NeuralNet</sub> + 0.91 E<sub>RF</sub> + 0.05 E<sub>GBM</sub> + 0.10 E<sub>Elastic</sub> − 0.05 E<sub>NaiveBayes</sub> (Fig. <a href="#Fig4" class="usa-link">4</a>J). Five-fold cross-validation on the training set underscored the robust diagnostic efficacy of the model, yielding an AUC value of 0.98 (Fig. <a href="#Fig4" class="usa-link">4</a>K). Furthermore, test using an independent test dataset yielded an AUC value of 0.934 (Fig. <a href="#Fig4" class="usa-link">4</a>L), indicating significant concordance with metatranscriptomic detection of pathogenic viral infections.</p>
<section id="Sec11"><h3 class="pmc_sec_title">Robustness assessment of three stacking-based models</h3>
<p id="Par39">To further evaluate the robustness, we designed a comprehensive evaluation framework to quantitatively compare the performance stability of the three-stacking model against 13 conventional classifiers using the test cohorts. Specifically, we introduced a 5-dimensional evaluation scheme capturing both accuracy and generalization performance across heterogeneous subsets: AUC (measures overall discrimination ability); Accuracy (measures performance at a fixed threshold); Subset Accuracy Std (reflects prediction stability across different subgroups); Subset AUC Std (captures discrimination consistency across subsets); Subset Top3 Score (quantifies how often the model ranks in the top three across key subsets). All test set samples were used to evaluate the three stacking models and their 13 constituent sub-models, and average of the five metrics were calculated (Supplementary Table <a href="#MOESM5" class="usa-link">S5</a>). We further formulated the following scoring function to aggregate these metrics into a single robustness score: 0.2 × AUC<sub>average</sub> + 0.2 × Accuracy<sub>average</sub> + 0.2 × (1 − Subset Accuracy Std) <sub>average</sub> + 0.2 × (1 − AUC Std across Subsets) <sub>average</sub> + 0.2 × Subset Top3 Score (Normalized)<sub>average</sub>. This balanced scoring system prioritizes both global predictive quality and cross-scenario stability, with a modest boost from localized subset performance. The results showed that stacking models achieved the highest robustness score 0.93, outperforming all single model. This was further illustrated with visualizations including radar plots (Fig. <a href="#Fig5" class="usa-link">5</a>A) and score distribution (Fig. <a href="#Fig5" class="usa-link">5</a>B), highlighting its superior stability and resilience in diverse clinical conditions. In summary, we quantitatively and visually demonstrated that the stacking model was not only accurate but also robust across multiple data partitions, substantiating its advantage over individual models.</p>
<figure class="fig xbox font-sm" id="Fig5"><h4 class="obj_head">Fig. 5.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12368225_41598_2025_15914_Fig5_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/157b/12368225/a226fa200288/41598_2025_15914_Fig5_HTML.jpg" loading="lazy" id="d33e896" height="187" width="669" alt="Fig. 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Robustness assessment of three stacking-based models. (<strong>A</strong>) Radar plot visualization of the average scores across five evaluation metrics for the three stacking models and each individual sub-model on all test set samples. (<strong>B</strong>) Robustness scoring comparison between the stacking models and 13 single machine learning models.</p></figcaption></figure></section></section><section id="Sec12"><h2 class="pmc_sec_title">Discussion</h2>
<p id="Par41">Particularly concerning are patients in critical conditions, including those in ICU, where delayed diagnosis of severe pulmonary infections can lead to organ dysfunction and increased mortality risk<sup><a href="#CR31" class="usa-link" aria-describedby="CR31">31</a></sup>. Broad-spectrum antibiotics, antiviral agents, and antifungal medications are frequently employed as empiric therapies. However, up to half of ICU patients receiving empirical antimicrobial therapy lack definitive confirmation of infection. Administration of antimicrobial agents to non-infected patients not only fails to confer benefits but may also introduce potential risks of adverse events, secondary opportunistic pathogen infections, and drug resistance. Traditional microbiological detection methods, primarily microbial culture, suffer from limitations such as stringent technical requirements and conditions, resulting in unstable culture outcomes and poor timeliness, which may impede disease progression monitoring in critically ill patients awaiting results. The advent of sequencing technologies has introduced metagenomic and metatranscriptomic analyses, enabling discrimination between infection and non-infection states and accurate identification of microbial taxa. However, the technical complexity of these assays, combined with prolonged turnaround times for data analysis and result interpretation (typically 2–7 days in clinical settings), often compromises timely clinical decision-making. In China’s diagnostic landscape, the substantial cost (approximately 3,000 RMB per sample) and time requirements for BALF metatranscriptomic sequencing further limit its routine clinical application for critically ill, severely ill, and ICU patients. On the other hand, compared to metagenomic analyses, metatranscriptomic sequencing can identify the impact of infecting microorganisms on host gene expression<sup><a href="#CR32" class="usa-link" aria-describedby="CR32">32</a></sup>facilitating the identification of host gene biomarkers for infection diagnosis<sup><a href="#CR33" class="usa-link" aria-describedby="CR33">33</a></sup>. However, effective host biomarkers have yet to be fully elucidated, and diagnostic models based on basic biomarkers require further development.</p>
<p id="Par42">In this study, we proposed a targeted framework that simultaneously addresses. Retrospective samples of BALF from respiratory and critically ill patients were collected, followed by metatranscriptomic sequencing. Based on metatranscriptomic microbial detection results, patients were categorized into 41 negative cases, 33 bacterial infections, 91 viral infections, and 15 fungal infections. Due to the limited number of fungal infection cases and substantial inter-individual variations (<em>p</em> = 0.475) (Figure <a href="#MOESM4" class="usa-link">S4</a>A-B), further analysis was not pursued, constituting a major limitation of this study. However, further investigation will be conducted as more samples of fungal infections are continuously collected. Transcriptomic data of host genes were selected as characteristic genes from patients’ data with microbial detection results indicating bacterial or viral infections and those without detection. Using these characteristic gene transcriptomic data, various machine learning algorithms were employed to construct classification diagnostic sub-models. Subsequently, a stacking-based ensemble learning model was constructed using Lasso to integrate the sub-models, ultimately forming a rapid and inexpensive classification diagnostic method requiring the detection of only a few host gene biomarkers. Compared to utilizing a single model, ensemble learning models allowed for the consideration of the efficacy of multiple models in result evaluation, thereby enhancing predictive performance, mitigating the risk of overfitting, bolstering robustness, and providing increased flexibility and interpretability<sup><a href="#CR34" class="usa-link" aria-describedby="CR34">34</a>,<a href="#CR35" class="usa-link" aria-describedby="CR35">35</a></sup>. We emphasized that the choice to apply a unified set of 13 models across all datasets was not arbitrary, but rather driven by a need to maintain methodological consistency and ensure fair cross-cohort evaluation. The final stacking model was thus selected based on its superior performance and robustness across all datasets, balancing predictive power with technical uniformity. The average performance of the five-fold cross-validation models for distinguishing infection from non-infection was AUC = 0.984, for distinguishing bacterial infection from non-bacterial infection was AUC = 0.98, and for distinguishing viral infection from non- viral infection was AUC = 0.98. Moreover, test set results demonstrated that the developed method exhibited high consistency in diagnostic performance compared to metatranscriptomic sequencing in differentiating patients with infection (AUC = 0.865) and infection types (viral AUC = 0.934 and bacterial AUC = 0.871), serving as a potential adjunctive diagnostic method for unidentified pulmonary infections in critically ill, severely ill, and ICU patients.</p>
<p id="Par43">Our study presents a prospective adjunctive diagnostic approach based on qPCR detection of 21 genes, which requires only approximately 210 RMB (∼93% reduction compared to conventional metatranscriptomic sequencing) with a turnaround time of less than 6 h while maintaining diagnostic accuracy comparable to metatranscriptomic sequencing, enabling the identification of infection status and type in pulmonary infections of undetermined etiology among critically ill, severely ill, and ICU patients. Our developed detection method holds promise for reducing medical costs, mitigating drug selection pressures and adverse effects, and improving patient outcomes. Future directions of this study could involve integrated analysis linking pathogen metagenomic features with host immune signatures. A multi-omics data fusion approach would help elucidate critical molecular patterns of pathogen-host interactions, ultimately enhancing clinical diagnostic performance.</p></section><section id="Sec15"><h2 class="pmc_sec_title">Supplementary Information</h2>
<p>Below is the link to the electronic supplementary material.</p>
<section class="sm xbox font-sm" id="MOESM1"><div class="media p"><div class="caption">
<a href="/articles/instance/12368225/bin/41598_2025_15914_MOESM1_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 1</a><sup> (5.8MB, docx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM2"><div class="media p"><div class="caption">
<a href="/articles/instance/12368225/bin/41598_2025_15914_MOESM2_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 2</a><sup> (20KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM3"><div class="media p"><div class="caption">
<a href="/articles/instance/12368225/bin/41598_2025_15914_MOESM3_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 3</a><sup> (9.9KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM4"><div class="media p"><div class="caption">
<a href="/articles/instance/12368225/bin/41598_2025_15914_MOESM4_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 4</a><sup> (9.5KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM5"><div class="media p"><div class="caption">
<a href="/articles/instance/12368225/bin/41598_2025_15914_MOESM5_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 5</a><sup> (35.8KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM6"><div class="media p"><div class="caption">
<a href="/articles/instance/12368225/bin/41598_2025_15914_MOESM6_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 6</a><sup> (11KB, xlsx) </sup>
</div></div></section></section><section id="notes1"><h2 class="pmc_sec_title">Author contributions</h2>
<p>W.J.-L. and W.F.-L. designed and performed the project, wrote and revised the manuscript, supervised the project. T.-Z. and Y.-D. designed this project, analyzed the data and revised the manuscript. W.T.-W. and Z.-Z. collected and interpretated the samples and analyzed the data. H.Q.-W. revised the manuscript and supervised the project. Y.L.-W. and S.T.-X. assisted in revising the manuscript.</p></section><section id="notes2"><h2 class="pmc_sec_title">Funding</h2>
<p>This work was supported by the National Natural Science Foundation of Young Scientists of China (32200438).</p></section><section id="notes3"><h2 class="pmc_sec_title">Data availability</h2>
<p>The metatranscriptomic raw data generated and analyzed in this project are included in the article/Supplementary Material, further inquiries can be directed to the corresponding author/s.</p></section><section id="notes4"><h2 class="pmc_sec_title">Declarations</h2>
<section id="FPar1"><h3 class="pmc_sec_title">Competing interests</h3>
<p id="Par44">The authors declare no competing interests.</p></section><section id="FPar2"><h3 class="pmc_sec_title">Ethic approval and consent to participate</h3>
<p id="Par45">This study was approved by the ethics committee of Kingmed Diagnostics (2023160), and all experiments were performed in accordance with relevant guidelines and regulations. Due to the retrospective nature of the study, the ethics committee of Kingmed Diagnostics waived the need of obtaining informed consent.</p></section><section id="FPar3"><h3 class="pmc_sec_title">Code availability</h3>
<p id="Par46">Code supporting the study are available in 10.5281/zenodo.15655782.</p></section></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm">
<div class="fn p" id="fn1">
<p><strong>Publisher’s note</strong></p>
<p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p>
</div>
<div class="fn p" id="fn2"><p>Tian Zhang and Ying Deng: Co-first author.</p></div>
</div></section><section id="_ci93_" lang="en" class="contrib-info"><h2 class="pmc_sec_title">Contributor Information</h2>
<p>Weifang Liao, Email: leesalwf89@126.com.</p>
<p>Weijie Liao, Email: liaoweijie@szu.edu.cn.</p></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Kradin, R. L. &amp; Mark, E. J. <em>8 - Pathology of Pulmonary Infection</em> (Elsevier, 2018).</cite>
</li>
<li id="CR2">
<span class="label">2.</span><cite>Whiteside, S. A., McGinniss, J. E. &amp; Collman, R. G. The lung microbiome: progress and promise. <em>J. Clin. Invest.</em><strong>131</strong>10.1172/JCI150473 (2021).</cite> [<a href="https://doi.org/10.1172/JCI150473" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8321564/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34338230/" class="usa-link">PubMed</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>Timsit, J. F. et al. Rationalizing antimicrobial therapy in the ICU: a narrative review. <em>Intensive Care Med.</em><strong>45</strong>, 172–189. 10.1007/s00134-019-05520-5 (2019).
</cite> [<a href="https://doi.org/10.1007/s00134-019-05520-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30659311/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Timsit,%20J.%20F.%20et%20al.%20Rationalizing%20antimicrobial%20therapy%20in%20the%20ICU:%20a%20narrative%20review.%20Intensive%20Care%20Med.45,%20172%E2%80%93189.%2010.1007/s00134-019-05520-5%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Jain, S. et al. Community-Acquired pneumonia requiring hospitalization among U.S. Adults. <em>N Engl. J. Med.</em><strong>373</strong>, 415–427. 10.1056/NEJMoa1500245 (2015).
</cite> [<a href="https://doi.org/10.1056/NEJMoa1500245" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4728150/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26172429/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jain,%20S.%20et%20al.%20Community-Acquired%20pneumonia%20requiring%20hospitalization%20among%20U.S.%20Adults.%20N%20Engl.%20J.%20Med.373,%20415%E2%80%93427.%2010.1056/NEJMoa1500245%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Versporten, A. et al. Antimicrobial consumption and resistance in adult hospital inpatients in 53 countries: results of an internet-based global point prevalence survey. <em>Lancet Glob Health</em>. <strong>6</strong>, e619–e629. 10.1016/S2214-109X(18)30186-4 (2018).
</cite> [<a href="https://doi.org/10.1016/S2214-109X(18)30186-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29681513/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Versporten,%20A.%20et%20al.%20Antimicrobial%20consumption%20and%20resistance%20in%20adult%20hospital%20inpatients%20in%2053%20countries:%20results%20of%20an%20internet-based%20global%20point%20prevalence%20survey.%20Lancet%20Glob%20Health.%206,%20e619%E2%80%93e629.%2010.1016/S2214-109X(18)30186-4%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Guo, Y. et al. Metagenomic next-generation sequencing to identify pathogens and cancer in lung biopsy tissue. <em>EBioMedicine</em><strong>73</strong>, 103639. 10.1016/j.ebiom.2021.103639 (2021).
</cite> [<a href="https://doi.org/10.1016/j.ebiom.2021.103639" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8554462/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34700283/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Guo,%20Y.%20et%20al.%20Metagenomic%20next-generation%20sequencing%20to%20identify%20pathogens%20and%20cancer%20in%20lung%20biopsy%20tissue.%20EBioMedicine73,%20103639.%2010.1016/j.ebiom.2021.103639%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Xu, H. et al. Clinical application and evaluation of metagenomic Next-Generation sequencing in pulmonary infection with pleural effusion. <em>Infect. Drug Resist.</em><strong>15</strong>, 2813–2824. 10.2147/IDR.S365757 (2022).
</cite> [<a href="https://doi.org/10.2147/IDR.S365757" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9167844/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35677528/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Xu,%20H.%20et%20al.%20Clinical%20application%20and%20evaluation%20of%20metagenomic%20Next-Generation%20sequencing%20in%20pulmonary%20infection%20with%20pleural%20effusion.%20Infect.%20Drug%20Resist.15,%202813%E2%80%932824.%2010.2147/IDR.S365757%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>Huang, J. et al. Metagenomic Next-Generation sequencing versus traditional pathogen detection in the diagnosis of peripheral pulmonary infectious lesions. <em>Infect. Drug Resist.</em><strong>13</strong>, 567–576. 10.2147/IDR.S235182 (2020).
</cite> [<a href="https://doi.org/10.2147/IDR.S235182" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7036976/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32110067/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Huang,%20J.%20et%20al.%20Metagenomic%20Next-Generation%20sequencing%20versus%20traditional%20pathogen%20detection%20in%20the%20diagnosis%20of%20peripheral%20pulmonary%20infectious%20lesions.%20Infect.%20Drug%20Resist.13,%20567%E2%80%93576.%2010.2147/IDR.S235182%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>Charalampous, T. et al. Nanopore metagenomics enables rapid clinical diagnosis of bacterial lower respiratory infection. <em>Nat. Biotechnol.</em><strong>37</strong>, 783–792. 10.1038/s41587-019-0156-5 (2019).
</cite> [<a href="https://doi.org/10.1038/s41587-019-0156-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31235920/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Charalampous,%20T.%20et%20al.%20Nanopore%20metagenomics%20enables%20rapid%20clinical%20diagnosis%20of%20bacterial%20lower%20respiratory%20infection.%20Nat.%20Biotechnol.37,%20783%E2%80%93792.%2010.1038/s41587-019-0156-5%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>Zhu, N. et al. A novel coronavirus from patients with pneumonia in china, 2019. <em>N Engl. J. Med.</em><strong>382</strong>, 727–733. 10.1056/NEJMoa2001017 (2020).
</cite> [<a href="https://doi.org/10.1056/NEJMoa2001017" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7092803/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31978945/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhu,%20N.%20et%20al.%20A%20novel%20coronavirus%20from%20patients%20with%20pneumonia%20in%20china,%202019.%20N%20Engl.%20J.%20Med.382,%20727%E2%80%93733.%2010.1056/NEJMoa2001017%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Carr, V. R. &amp; Chaguza, C. Metagenomics for surveillance of respiratory pathogens. <em>Nat. Rev. Microbiol.</em><strong>19</strong>, 285. 10.1038/s41579-021-00541-8 (2021).
</cite> [<a href="https://doi.org/10.1038/s41579-021-00541-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7950416/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33707741/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Carr,%20V.%20R.%20&amp;%20Chaguza,%20C.%20Metagenomics%20for%20surveillance%20of%20respiratory%20pathogens.%20Nat.%20Rev.%20Microbiol.19,%20285.%2010.1038/s41579-021-00541-8%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Diao, Z., Han, D., Zhang, R. &amp; Li, J. Metagenomics next-generation sequencing tests take the stage in the diagnosis of lower respiratory tract infections. <em>J. Adv. Res.</em><strong>38</strong>, 201–212. 10.1016/j.jare.2021.09.012 (2022).
</cite> [<a href="https://doi.org/10.1016/j.jare.2021.09.012" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9091713/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35572406/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Diao,%20Z.,%20Han,%20D.,%20Zhang,%20R.%20&amp;%20Li,%20J.%20Metagenomics%20next-generation%20sequencing%20tests%20take%20the%20stage%20in%20the%20diagnosis%20of%20lower%20respiratory%20tract%20infections.%20J.%20Adv.%20Res.38,%20201%E2%80%93212.%2010.1016/j.jare.2021.09.012%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>Chen, S., Zhou, Y., Chen, Y. &amp; Gu, J. Fastp: an ultra-fast all-in-one FASTQ preprocessor. <em>Bioinformatics</em><strong>34</strong>, i884–i890. 10.1093/bioinformatics/bty560 (2018).
</cite> [<a href="https://doi.org/10.1093/bioinformatics/bty560" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6129281/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30423086/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chen,%20S.,%20Zhou,%20Y.,%20Chen,%20Y.%20&amp;%20Gu,%20J.%20Fastp:%20an%20ultra-fast%20all-in-one%20FASTQ%20preprocessor.%20Bioinformatics34,%20i884%E2%80%93i890.%2010.1093/bioinformatics/bty560%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>Andrews, S. (Cambridge, United Kingdom, (2010).</cite>
</li>
<li id="CR15">
<span class="label">15.</span><cite>Kim, D., Langmead, B. &amp; Salzberg, S. L. HISAT: a fast spliced aligner with low memory requirements. <em>Nat. Methods</em>. <strong>12</strong>, 357–360. 10.1038/nmeth.3317 (2015).
</cite> [<a href="https://doi.org/10.1038/nmeth.3317" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4655817/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25751142/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kim,%20D.,%20Langmead,%20B.%20&amp;%20Salzberg,%20S.%20L.%20HISAT:%20a%20fast%20spliced%20aligner%20with%20low%20memory%20requirements.%20Nat.%20Methods.%2012,%20357%E2%80%93360.%2010.1038/nmeth.3317%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Kim, D., Paggi, J. M., Park, C., Bennett, C. &amp; Salzberg, S. L. Graph-based genome alignment and genotyping with HISAT2 and HISAT-genotype. <em>Nat. Biotechnol.</em><strong>37</strong>, 907–915. 10.1038/s41587-019-0201-4 (2019).
</cite> [<a href="https://doi.org/10.1038/s41587-019-0201-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7605509/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31375807/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kim,%20D.,%20Paggi,%20J.%20M.,%20Park,%20C.,%20Bennett,%20C.%20&amp;%20Salzberg,%20S.%20L.%20Graph-based%20genome%20alignment%20and%20genotyping%20with%20HISAT2%20and%20HISAT-genotype.%20Nat.%20Biotechnol.37,%20907%E2%80%93915.%2010.1038/s41587-019-0201-4%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>Li, H. et al. The sequence alignment/map format and samtools. <em>Bioinformatics</em><strong>25</strong>, 2078–2079. 10.1093/bioinformatics/btp352 (2009).
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btp352" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2723002/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19505943/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li,%20H.%20et%20al.%20The%20sequence%20alignment/map%20format%20and%20samtools.%20Bioinformatics25,%202078%E2%80%932079.%2010.1093/bioinformatics/btp352%20(2009)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Liao, Y., Smyth, G. K. &amp; Shi, W. FeatureCounts: an efficient general purpose program for assigning sequence reads to genomic features. <em>Bioinformatics</em><strong>30</strong>, 923–930. 10.1093/bioinformatics/btt656 (2014).
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btt656" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24227677/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Liao,%20Y.,%20Smyth,%20G.%20K.%20&amp;%20Shi,%20W.%20FeatureCounts:%20an%20efficient%20general%20purpose%20program%20for%20assigning%20sequence%20reads%20to%20genomic%20features.%20Bioinformatics30,%20923%E2%80%93930.%2010.1093/bioinformatics/btt656%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Dixon, P. VEGAN, a package of R functions for community ecology. <em>J. Veg. Sci.</em><strong>14</strong>, 927–930 (2003).</cite> [<a href="https://scholar.google.com/scholar_lookup?Dixon,%20P.%20VEGAN,%20a%20package%20of%20R%20functions%20for%20community%20ecology.%20J.%20Veg.%20Sci.14,%20927%E2%80%93930%20(2003)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>Love, M. I., Huber, W. &amp; Anders, S. Moderated Estimation of fold change and dispersion for RNA-seq data with DESeq2. <em>Genome Biol.</em><strong>15</strong>, 550. 10.1186/s13059-014-0550-8 (2014).
</cite> [<a href="https://doi.org/10.1186/s13059-014-0550-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4302049/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25516281/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Love,%20M.%20I.,%20Huber,%20W.%20&amp;%20Anders,%20S.%20Moderated%20Estimation%20of%20fold%20change%20and%20dispersion%20for%20RNA-seq%20data%20with%20DESeq2.%20Genome%20Biol.15,%20550.%2010.1186/s13059-014-0550-8%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>Robinson, M. D., McCarthy, D. J. &amp; Smyth, G. K. EdgeR: a bioconductor package for differential expression analysis of digital gene expression data. <em>Bioinformatics</em><strong>26</strong>, 139–140. 10.1093/bioinformatics/btp616 (2010).
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btp616" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2796818/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19910308/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Robinson,%20M.%20D.,%20McCarthy,%20D.%20J.%20&amp;%20Smyth,%20G.%20K.%20EdgeR:%20a%20bioconductor%20package%20for%20differential%20expression%20analysis%20of%20digital%20gene%20expression%20data.%20Bioinformatics26,%20139%E2%80%93140.%2010.1093/bioinformatics/btp616%20(2010)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>Smyth, G. (New York: Springer, (2005).</cite>
</li>
<li id="CR23">
<span class="label">23.</span><cite>Hornik, K. The comprehensive R archive network. <em>Wiley Interdisciplinary Reviews: Comput. Stat.</em><strong>4</strong>, 394–398 (2012).</cite> [<a href="https://scholar.google.com/scholar_lookup?Hornik,%20K.%20The%20comprehensive%20R%20archive%20network.%20Wiley%20Interdisciplinary%20Reviews:%20Comput.%20Stat.4,%20394%E2%80%93398%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>Kolde, R. Pheatmap: pretty heatmaps. <em>R Package Version</em>. <strong>1</strong>, 726 (2019).</cite> [<a href="https://scholar.google.com/scholar_lookup?Kolde,%20R.%20Pheatmap:%20pretty%20heatmaps.%20R%20Package%20Version.%201,%20726%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>Yu, G., Wang, L. G., Han, Y. &amp; He, Q. Y. ClusterProfiler: an R package for comparing biological themes among gene clusters. <em>Omics: J. Integr. Biology</em>. <strong>16</strong>, 284–287 (2012).</cite> [<a href="https://doi.org/10.1089/omi.2011.0118" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3339379/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22455463/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yu,%20G.,%20Wang,%20L.%20G.,%20Han,%20Y.%20&amp;%20He,%20Q.%20Y.%20ClusterProfiler:%20an%20R%20package%20for%20comparing%20biological%20themes%20among%20gene%20clusters.%20Omics:%20J.%20Integr.%20Biology.%2016,%20284%E2%80%93287%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>Kanehisa, M. Toward Understanding the origin and evolution of cellular organisms. <em>Protein Sci.</em><strong>28</strong>, 1947–1951. 10.1002/pro.3715 (2019).
</cite> [<a href="https://doi.org/10.1002/pro.3715" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6798127/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31441146/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kanehisa,%20M.%20Toward%20Understanding%20the%20origin%20and%20evolution%20of%20cellular%20organisms.%20Protein%20Sci.28,%201947%E2%80%931951.%2010.1002/pro.3715%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Kanehisa, M., Furumichi, M., Sato, Y., Matsuura, Y. &amp; Ishiguro-Watanabe, M. KEGG: biological systems database as a model of the real world. <em>Nucleic Acids Res.</em><strong>53</strong>, D672–D677. 10.1093/nar/gkae909 (2024).</cite> [<a href="https://doi.org/10.1093/nar/gkae909" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11701520/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39417505/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kanehisa,%20M.,%20Furumichi,%20M.,%20Sato,%20Y.,%20Matsuura,%20Y.%20&amp;%20Ishiguro-Watanabe,%20M.%20KEGG:%20biological%20systems%20database%20as%20a%20model%20of%20the%20real%20world.%20Nucleic%20Acids%20Res.53,%20D672%E2%80%93D677.%2010.1093/nar/gkae909%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Kanehisa, M. &amp; Goto, S. K. E. G. G. Kyoto encyclopedia of genes and genomes. <em>Nucleic Acids Res.</em><strong>28</strong>, 27–30. 10.1093/nar/28.1.27 (2000).
</cite> [<a href="https://doi.org/10.1093/nar/28.1.27" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC102409/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10592173/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kanehisa,%20M.%20&amp;%20Goto,%20S.%20K.%20E.%20G.%20G.%20Kyoto%20encyclopedia%20of%20genes%20and%20genomes.%20Nucleic%20Acids%20Res.28,%2027%E2%80%9330.%2010.1093/nar/28.1.27%20(2000)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>Zeng, D. et al. Multi-Omics Immuno-Oncology biological research to Decode tumor microenvironment and signatures. <em>Front. Immunol.</em><strong>12</strong>, 687975. 10.3389/fimmu.2021.687975 (2021).
</cite> [<a href="https://doi.org/10.3389/fimmu.2021.687975" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8283787/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34276676/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zeng,%20D.%20et%20al.%20Multi-Omics%20Immuno-Oncology%20biological%20research%20to%20Decode%20tumor%20microenvironment%20and%20signatures.%20Front.%20Immunol.12,%20687975.%2010.3389/fimmu.2021.687975%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR30">
<span class="label">30.</span><cite>Robin, X. et al. pROC: an open-source package for R and S + to analyze and compare ROC curves. <em>BMC Bioinform.</em><strong>12</strong>, 1–8 (2011).</cite> [<a href="https://doi.org/10.1186/1471-2105-12-77" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3068975/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21414208/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Robin,%20X.%20et%20al.%20pROC:%20an%20open-source%20package%20for%20R%20and%20S%E2%80%89+%E2%80%89to%20analyze%20and%20compare%20ROC%20curves.%20BMC%20Bioinform.12,%201%E2%80%938%20(2011)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>Detsis, M., Karanika, S., Mylonakis, E. I. C. U. &amp; Acquisition Rate Risk factors, and clinical significance of digestive tract colonization with Extended-Spectrum Beta-Lactamase-Producing enterobacteriaceae: A systematic review and Meta-Analysis. <em>Crit. Care Med.</em><strong>45</strong>, 705–714. 10.1097/CCM.0000000000002253 (2017).
</cite> [<a href="https://doi.org/10.1097/CCM.0000000000002253" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28157141/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Detsis,%20M.,%20Karanika,%20S.,%20Mylonakis,%20E.%20I.%20C.%20U.%20&amp;%20Acquisition%20Rate%20Risk%20factors,%20and%20clinical%20significance%20of%20digestive%20tract%20colonization%20with%20Extended-Spectrum%20Beta-Lactamase-Producing%20enterobacteriaceae:%20A%20systematic%20review%20and%20Meta-Analysis.%20Crit.%20Care%20Med.45,%20705%E2%80%93714.%2010.1097/CCM.0000000000002253%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>Ramachandran, P. S. et al. Integrating central nervous system metagenomics and host response for diagnosis of tuberculosis meningitis and its mimics. <em>Nat. Commun.</em><strong>13</strong>, 1675. 10.1038/s41467-022-29353-x (2022).
</cite> [<a href="https://doi.org/10.1038/s41467-022-29353-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8967864/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35354815/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ramachandran,%20P.%20S.%20et%20al.%20Integrating%20central%20nervous%20system%20metagenomics%20and%20host%20response%20for%20diagnosis%20of%20tuberculosis%20meningitis%20and%20its%20mimics.%20Nat.%20Commun.13,%201675.%2010.1038/s41467-022-29353-x%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Fan, S. et al. Integrating host transcriptomic signatures for distinguishing autoimmune encephalitis in cerebrospinal fluid by metagenomic sequencing. <em>Cell. Biosci.</em><strong>13</strong>, 111. 10.1186/s13578-023-01047-x (2023).
</cite> [<a href="https://doi.org/10.1186/s13578-023-01047-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10278324/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37332019/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Fan,%20S.%20et%20al.%20Integrating%20host%20transcriptomic%20signatures%20for%20distinguishing%20autoimmune%20encephalitis%20in%20cerebrospinal%20fluid%20by%20metagenomic%20sequencing.%20Cell.%20Biosci.13,%20111.%2010.1186/s13578-023-01047-x%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>Polikar, R. Ensemble based systems in decision making. <em>IEEE Circuits Syst. Mag.</em><strong>6</strong>, 21–45 (2006).</cite> [<a href="https://scholar.google.com/scholar_lookup?Polikar,%20R.%20Ensemble%20based%20systems%20in%20decision%20making.%20IEEE%20Circuits%20Syst.%20Mag.6,%2021%E2%80%9345%20(2006)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>Kuncheva, L. I. &amp; Rodríguez, J. J. A weighted voting framework for classifiers ensembles. <em>Knowl. Inf. Syst.</em><strong>38</strong>, 259–275 (2014).</cite> [<a href="https://scholar.google.com/scholar_lookup?Kuncheva,%20L.%20I.%20&amp;%20Rodr%C3%ADguez,%20J.%20J.%20A%20weighted%20voting%20framework%20for%20classifiers%20ensembles.%20Knowl.%20Inf.%20Syst.38,%20259%E2%80%93275%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12368225/bin/41598_2025_15914_MOESM1_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 1</a><sup> (5.8MB, docx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12368225/bin/41598_2025_15914_MOESM2_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 2</a><sup> (20KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12368225/bin/41598_2025_15914_MOESM3_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 3</a><sup> (9.9KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material4_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12368225/bin/41598_2025_15914_MOESM4_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 4</a><sup> (9.5KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material5_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12368225/bin/41598_2025_15914_MOESM5_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 5</a><sup> (35.8KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material6_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12368225/bin/41598_2025_15914_MOESM6_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 6</a><sup> (11KB, xlsx) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>The metatranscriptomic raw data generated and analyzed in this project are included in the article/Supplementary Material, further inquiries can be directed to the corresponding author/s.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Scientific Reports are provided here courtesy of <strong>Nature Publishing Group</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1038/s41598-025-15914-9"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/41598_2025_Article_15914.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (2.6 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12368225/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12368225/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12368225%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12368225/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12368225/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12368225/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40835893/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12368225/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40835893/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12368225/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12368225/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="NyrOcmW1NSCvWglPwf5c2b2AjEW2TMVKYZc1qV9MHMoMKLBUfBVQbk28OfqzF27x">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
