
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            An intelligent method for detection of small target fungal wheat spores based on an improved YOLOv5 with microscopic images - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="AE53291A8AF18EC305291A006B35A785.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="plantmeth">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12372353/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Plant Methods">
<meta name="citation_title" content="An intelligent method for detection of small target fungal wheat spores based on an improved YOLOv5 with microscopic images">
<meta name="citation_author" content="Zhizhou Ren">
<meta name="citation_author_institution" content="School of Smart Agriculture (Artificial Intelligence), Nanjing Agricultural University, Nanjing, 210031 China">
<meta name="citation_author" content="Kun Liang">
<meta name="citation_author_institution" content="School of Smart Agriculture (Artificial Intelligence), Nanjing Agricultural University, Nanjing, 210031 China">
<meta name="citation_author" content="Yingqi Zhang">
<meta name="citation_author_institution" content="School of Smart Agriculture (Artificial Intelligence), Nanjing Agricultural University, Nanjing, 210031 China">
<meta name="citation_author" content="Jinpeng Song">
<meta name="citation_author_institution" content="College of Engineering, Nanjing Agricultural University, Nanjing, 210031 China">
<meta name="citation_author" content="Xiaoxiao Wu">
<meta name="citation_author_institution" content="Key Laboratory of Biotoxin Analysis &amp; Assessment for State Market Regulation, Nanjing Institute of Product Quality Inspection &amp; Testing, Nanjing, 210019 China">
<meta name="citation_author" content="Chi Zhang">
<meta name="citation_author_institution" content="Key Laboratory of Biotoxin Analysis &amp; Assessment for State Market Regulation, Nanjing Institute of Product Quality Inspection &amp; Testing, Nanjing, 210019 China">
<meta name="citation_author" content="Xiuming Mei">
<meta name="citation_author_institution" content="Key Laboratory of Biotoxin Analysis &amp; Assessment for State Market Regulation, Nanjing Institute of Product Quality Inspection &amp; Testing, Nanjing, 210019 China">
<meta name="citation_author" content="Yi Zhang">
<meta name="citation_author_institution" content="Jiangsu Grain and Oil Quality Monitoring Center, Nanjing, 210031 China">
<meta name="citation_author" content="Xin Liu">
<meta name="citation_author_institution" content="Institute of Food Quality and Safety Testing, Jiangsu Academy of Agricultural Sciences, Nanjing, 210014 China">
<meta name="citation_publication_date" content="2025 Aug 22">
<meta name="citation_volume" content="21">
<meta name="citation_firstpage" content="117">
<meta name="citation_doi" content="10.1186/s13007-025-01436-y">
<meta name="citation_pmid" content="40847309">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12372353/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12372353/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12372353/pdf/13007_2025_Article_1436.pdf">
<meta name="description" content="Wheat is significantly impacted by fungal diseases, which result in severe economic losses. These diseases result from pathogenic spores invading wheat. Rapid and accurate detection of these spores is essential for post-harvest contamination risk ...">
<meta name="og:title" content="An intelligent method for detection of small target fungal wheat spores based on an improved YOLOv5 with microscopic images">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Wheat is significantly impacted by fungal diseases, which result in severe economic losses. These diseases result from pathogenic spores invading wheat. Rapid and accurate detection of these spores is essential for post-harvest contamination risk ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12372353/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12372353">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1186/s13007-025-01436-y"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/13007_2025_Article_1436.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12372353%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12372353/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12372353/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12372353/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-plantmeth.png" alt="Plant Methods logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Plant Methods" title="Link to Plant Methods" shape="default" href="https://plantmethods.biomedcentral.com/" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Plant Methods</button></div>. 2025 Aug 22;21:117. doi: <a href="https://doi.org/10.1186/s13007-025-01436-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1186/s13007-025-01436-y</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Plant%20Methods%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Plant%20Methods%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Plant%20Methods%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Plant%20Methods%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><div class="lang-switcher font-secondary font-xs">
<label>Show available content in</label><ul class="lang-switcher__list">
<li class="lang-switcher__item"><button class="lang-switcher__button" aria-expanded="true" disabled data-lang="en">en</button></li>
<li class="lang-switcher__item"><button class="lang-switcher__button" aria-expanded="false" data-lang="de">de</button></li>
</ul>
</div>
<section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>An intelligent method for detection of small target fungal wheat spores based on an improved YOLOv5 with microscopic images</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ren%20Z%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Zhizhou Ren</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Zhizhou Ren</span></h3>
<div class="p">
<sup>1</sup>School of Smart Agriculture (Artificial Intelligence), Nanjing Agricultural University, Nanjing, 210031 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ren%20Z%22%5BAuthor%5D" class="usa-link"><span class="name western">Zhizhou Ren</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Liang%20K%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Kun Liang</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Kun Liang</span></h3>
<div class="p">
<sup>1</sup>School of Smart Agriculture (Artificial Intelligence), Nanjing Agricultural University, Nanjing, 210031 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Liang%20K%22%5BAuthor%5D" class="usa-link"><span class="name western">Kun Liang</span></a>
</div>
</div>
<sup>1,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Yingqi Zhang</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Yingqi Zhang</span></h3>
<div class="p">
<sup>1</sup>School of Smart Agriculture (Artificial Intelligence), Nanjing Agricultural University, Nanjing, 210031 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yingqi Zhang</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Song%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Jinpeng Song</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Jinpeng Song</span></h3>
<div class="p">
<sup>2</sup>College of Engineering, Nanjing Agricultural University, Nanjing, 210031 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Song%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Jinpeng Song</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wu%20X%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Xiaoxiao Wu</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Xiaoxiao Wu</span></h3>
<div class="p">
<sup>3</sup>Key Laboratory of Biotoxin Analysis &amp; Assessment for State Market Regulation, Nanjing Institute of Product Quality Inspection &amp; Testing, Nanjing, 210019 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wu%20X%22%5BAuthor%5D" class="usa-link"><span class="name western">Xiaoxiao Wu</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20C%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Chi Zhang</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Chi Zhang</span></h3>
<div class="p">
<sup>3</sup>Key Laboratory of Biotoxin Analysis &amp; Assessment for State Market Regulation, Nanjing Institute of Product Quality Inspection &amp; Testing, Nanjing, 210019 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20C%22%5BAuthor%5D" class="usa-link"><span class="name western">Chi Zhang</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mei%20X%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Xiuming Mei</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Xiuming Mei</span></h3>
<div class="p">
<sup>3</sup>Key Laboratory of Biotoxin Analysis &amp; Assessment for State Market Regulation, Nanjing Institute of Product Quality Inspection &amp; Testing, Nanjing, 210019 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mei%20X%22%5BAuthor%5D" class="usa-link"><span class="name western">Xiuming Mei</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Yi Zhang</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Yi Zhang</span></h3>
<div class="p">
<sup>4</sup>Jiangsu Grain and Oil Quality Monitoring Center, Nanjing, 210031 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yi Zhang</span></a>
</div>
</div>
<sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Liu%20X%22%5BAuthor%5D" class="usa-link" aria-describedby="id9"><span class="name western">Xin Liu</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">Xin Liu</span></h3>
<div class="p">
<sup>5</sup>Institute of Food Quality and Safety Testing, Jiangsu Academy of Agricultural Sciences, Nanjing, 210014 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Liu%20X%22%5BAuthor%5D" class="usa-link"><span class="name western">Xin Liu</span></a>
</div>
</div>
<sup>5</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>School of Smart Agriculture (Artificial Intelligence), Nanjing Agricultural University, Nanjing, 210031 China </div>
<div id="Aff2">
<sup>2</sup>College of Engineering, Nanjing Agricultural University, Nanjing, 210031 China </div>
<div id="Aff3">
<sup>3</sup>Key Laboratory of Biotoxin Analysis &amp; Assessment for State Market Regulation, Nanjing Institute of Product Quality Inspection &amp; Testing, Nanjing, 210019 China </div>
<div id="Aff4">
<sup>4</sup>Jiangsu Grain and Oil Quality Monitoring Center, Nanjing, 210031 China </div>
<div id="Aff5">
<sup>5</sup>Institute of Food Quality and Safety Testing, Jiangsu Academy of Agricultural Sciences, Nanjing, 210014 China </div>
<div class="author-notes p"><div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div></div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 May 6; Accepted 2025 Aug 17; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by-nc-nd/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12372353  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40847309/" class="usa-link">40847309</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract trans" lang="en" id="Abs1"><h2>Abstract</h2>
<p id="Par1">Wheat is significantly impacted by fungal diseases, which result in severe economic losses. These diseases result from pathogenic spores invading wheat. Rapid and accurate detection of these spores is essential for post-harvest contamination risk assessment and early warning. Traditional detection methods are time-consuming and labor-intensive, and difficult to detect small target spores in complex environments. Therefore, a YOLO-ASF-MobileViT detection algorithm is proposed to detect pathogenic wheat spores with varying sizes, shapes, and textures. Four types of common pathogenic wheat spores are used as the study object, including <em>Fusarium graminearum</em>, <em>Aspergillus flavus</em>, <em>Tilletia foetida</em> (sporidium maturum), and <em>Tilletia foetida</em> (sporidium immaturum). The Attentional Scale Sequence Fusion (ASF) is integrated into the original YOLOv5s to enhance the capture of small details in spore images and fuse multi-scale feature information of spores. Additionally, the Mobile Vision Transformer (MobileViT) attention mechanism is incorporated to enhance both local and global feature extraction for small spores. Experimental results show that the proposed YOLO-ASF-MobileViT model achieves an overall mAP@0.5 of 97.0%, outperforming advanced detectors such as TPH-YOLO (95.6%) and MG-YOLO (95.5%). Compared to the baseline YOLOv5s model, it improves the average detection accuracy by 1.6%, with a notable 4.3% increase in detecting small Aspergillus flavus spores (reaching 90.8%). The model maintains high robustness in challenging scenarios such as spore adhesion, occlusion, blur, and noise. This approach enables efficient and accurate detection of wheat fungal spores, supporting early contamination warning in post-harvest management.</p></section><section class="abstract trans" hidden="hidden" lang="de" id="Abs2"><h2>Graphical Abstract</h2>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12372353_13007_2025_1436_Figa_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d88d/12372353/d6b7d344d172/13007_2025_1436_Figa_HTML.jpg" loading="lazy" id="d33e269" height="327" width="669" alt="graphic file with name 13007_2025_1436_Figa_HTML.jpg"></a></p>
<section id="kwd-group1" lang="de" class="kwd-group"><p><strong>Keywords:</strong> Wheat, Fungal spore detection, Feature fusion, Attention mechanism, YOLO</p></section></section><section id="Sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p id="Par3">Wheat is an important food crop that provides an abundance of carbohydrates, essential amino acids, and minerals, making it a crucial source of energy and nutrients for humans [<a href="#CR1" class="usa-link" aria-describedby="CR1">1</a>, <a href="#CR2" class="usa-link" aria-describedby="CR2">2</a>]. Wheat is widely grown worldwide, and its healthy growth is crucial for food security [<a href="#CR3" class="usa-link" aria-describedby="CR3">3</a>]. The growth process of wheat is susceptible to a variety of fungal stresses that can lead to disease [<a href="#CR4" class="usa-link" aria-describedby="CR4">4</a>]. The three diseases explored in this paper—Fusarium head blight (FHB), common bunt, and <em>Aspergillus flavus</em> (<em>A. flavus</em>) infection—not only significantly reduce the yield and quality of wheat but also pose severe health risks and have widespread, lasting impacts due to spore transmission. FHB is caused by <em>Fusarium graminearum</em> (<em>F. graminearum</em>). FHB manifests as pink-orange sporodochia on infected spikelets and may cause shriveled kernels and weight loss [<a href="#CR5" class="usa-link" aria-describedby="CR5">5</a>]. Historical data from 1919 to the mid-1990s revealed yield losses of 15–29% in India, 30–40% in the Yangtze River Basin of China, and even 70% in parts of Romania [<a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>, <a href="#CR7" class="usa-link" aria-describedby="CR7">7</a>]. Wheat infected with FHB also produces mycotoxins, especially deoxynivalenol (DON), which seriously threaten the health of animals and humans [<a href="#CR8" class="usa-link" aria-describedby="CR8">8</a>]. Common wheat bunt is caused by <em>Tilletia foetida</em> (<em>T. foetida</em>). It is characterized by bunt balls—infected kernels filled with dark brown to black teliospores that emit a fetid odour at harvest [<a href="#CR9" class="usa-link" aria-describedby="CR9">9</a>, <a href="#CR10" class="usa-link" aria-describedby="CR10">10</a>]. Common bunt in wheat also affects the weight, germination effect and vigour of seeds [<a href="#CR11" class="usa-link" aria-describedby="CR11">11</a>]. In addition, wheat is infected with <em>A. flavus</em>, which is characterized by the appearance of yellow mould on the surfaces of wheat kernels. <em>A. flavus</em> produces aflatoxins B1 and B2 (AFB1 and AFB2, respectively), which form a stable class of carcinogenic mycotoxins that pose a deadly threat to human health [<a href="#CR12" class="usa-link" aria-describedby="CR12">12</a>]. Fungal spores are a major source of inoculum in wheat fungal diseases. Other transmission pathways include infected seeds, crop residues, and rain-splash dispersal [<a href="#CR13" class="usa-link" aria-describedby="CR13">13</a>, <a href="#CR14" class="usa-link" aria-describedby="CR14">14</a>]. This study focuses on the post-harvest stage, where rapid spore detection supports early warning and contamination risk assessment during storage and procurement.</p>
<p id="Par4">Manual visual rating of wheat ears and grains is labour‑intensive, low‑throughput, and prone to subjective bias, with inter‑rater inconsistencies and frequent underestimation or overestimation of disease severity [<a href="#CR15" class="usa-link" aria-describedby="CR15">15</a>]. This method is inefficient [<a href="#CR16" class="usa-link" aria-describedby="CR16">16</a>, <a href="#CR17" class="usa-link" aria-describedby="CR17">17</a>]. To solve this problem, in recent years, methods that use image processing and deep learning to detect the severity of diseases in wheat have gradually attracted attention [<a href="#CR18" class="usa-link" aria-describedby="CR18">18</a>]. A spore, as a propagule in the life cycle of a fungus, can provide reliable early warning of potential contamination during post-harvest handling [<a href="#CR19" class="usa-link" aria-describedby="CR19">19</a>]. Traditional methods such as PCR and ELISA offer high specificity and sensitivity [<a href="#CR20" class="usa-link" aria-describedby="CR20">20</a>, <a href="#CR21" class="usa-link" aria-describedby="CR21">21</a>]. However, their reliance on multi-step preparation, specialised equipment, and trained personnel, along with turnaround times of several hours, limits their use for rapid, large-scale screening in post-harvest settings [<a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>]. In contrast, image-based spore detection offers a faster and more accessible alternative for on-site early warning [<a href="#CR17" class="usa-link" aria-describedby="CR17">17</a>].</p>
<p id="Par5">Deep learning algorithms have been widely used to classify spore microscopy images. Qamar et al. conducted a classification study on bacterial spore images acquired via transmission electron microscopy (TEM) [<a href="#CR23" class="usa-link" aria-describedby="CR23">23</a>]. They used a convolutional neural network (CNN) for feature extraction and combined it with a random forest (RF) for classification purposes. Ultimately, they achieved 73% accuracy. They proposed an innovative idea combining deep learning-based feature extraction and machine learning-based classification for spores. Crespo-Michel et al. presented novel deep learning algorithms for classifying four species of grapevine wood fungi [<a href="#CR24" class="usa-link" aria-describedby="CR24">24</a>]. They used four different neural network architectures, including ResNet-50, VGG-16, MobileNet, and InceptionV3, to classify spore microscopy images with accuracies of up to 97.4%. The above methods have made some progress in terms of spore image classification. However, depending only on deep learning is insufficient for large-scale spore detection and counting tasks [<a href="#CR25" class="usa-link" aria-describedby="CR25">25</a>]. In contrast, a target detection approach based on deep learning has the potential to achieve multi-target classification and counting. This method enables the completion large-scale, diverse spore detection tasks [<a href="#CR17" class="usa-link" aria-describedby="CR17">17</a>].</p>
<p id="Par6">In recent years, target detection algorithms have made significant progress in fungal spore microscopy image detection, but most of them have focused on detecting a single type of pathogenic spore. For example, Zhang et al. proposed an improved You Only Look Once version 5 (YOLOv5) model for detecting <em>F. graminearum</em> spores [<a href="#CR26" class="usa-link" aria-describedby="CR26">26</a>]. They incorporated efficient channel attention (ECA) and adaptive spatial feature fusion (ASFF) mechanisms into their model to effectively address the small sizes and limited features of spores. The method achieved remarkable results in terms of detecting <em>F. graminearum</em> spores, with an average accuracy of 98.57%. Li et al. proposed a high-precision, lightweight detection method based on YOLOv5 for detecting downy mildew spores in complex backgrounds [<a href="#CR27" class="usa-link" aria-describedby="CR27">27</a>]. They introduced a normalization-based attention module (NAM) and receptive field blocks (RFB-s) with dilated convolution to improve the detection accuracy attained for small target spores, and their approach ultimately achieved a mean average precision (mAP) of 95.6%. Wei et al. introduced the Faster region-based CNN (R-CNN), which used ResNet-50 and two small-scale region proposal networks (RPNs) to extract features for detecting microscopic hyperspectral images of thick-wall spores, yielding an average accuracy of 94.68% [<a href="#CR28" class="usa-link" aria-describedby="CR28">28</a>]. The authors also used the model to detect small spores in complex backgrounds. The above studies show that the existing methods have achieved significant improvements in terms of detecting a single type of pathogenic spore, especially small spores, in complex contexts. Wheat is often exposed to various fungal propagules, including airborne spores of different shapes and sizes, which are the main targets of image-based detection in post-harvest scenarios [<a href="#CR29" class="usa-link" aria-describedby="CR29">29</a>]. Larger spores are relatively easy to detect, whereas smaller spores are easily overlooked. Therefore, we need to introduce a method for detecting multiple spores in complex environments and improve the accuracy of small spore detection, thereby providing technical support for providing early warnings concerning multiple fungal diseases in wheat.</p>
<p id="Par7">Recent studies have attained improved small spore detection performance by enhancing the feature extraction capabilities of models [<a href="#CR30" class="usa-link" aria-describedby="CR30">30</a>]. Due to the differences among the sizes, morphologies and textures of diverse spores, the performance of the existing methods in diverse small fungal spore detection tasks still needs to be improved [<a href="#CR31" class="usa-link" aria-describedby="CR31">31</a>]. In this paper, we propose a novel YOLO-ASF-MobileViT spore detection method to solve these problems and realize rapid detection for small pathogenic wheat spores. The main objectives include the following:</p>
<ol class="list" style="list-style-type:decimal">
<li><p id="Par9">We propose a neck with the introduction of the Attentional Scale Sequence Fusion (ASF). The ASF significantly enhance the capture of tiny details in spore images and fuse multi-scale feature information of spores.</p></li>
<li><p id="Par10">The Mobile Vision Transformer (MobileViT) attention is inserted before the P3 detection head to capture both local and global features of small spores. The method reduces missed small fungal spore detection.</p></li>
<li><p id="Par11">A novel detection method called YOLO-ASF-MobileViT is proposed for detecting small diverse pathogenic wheat spores; extensive experiments show that YOLO-ASF-MobileViT maintains high robustness even when faced with spore adhesion, overlap, blurred contours, and impurity interference.</p></li>
</ol>
<p id="Par12">The method proposed in this paper not only provides a novel technique for detecting diverse pathogenic wheat spores but also effectively detects small spores, which provides a new idea for the early detection of fungal wheat diseases.</p></section><section id="Sec2"><h2 class="pmc_sec_title">Materials and methods</h2>
<section id="Sec3"><h3 class="pmc_sec_title">Data acquisition</h3>
<p id="Par13">Wheat is susceptible to fungal contamination throughout the crop cycle, including sowing, vegetative and reproductive growth stages, as well as harvest, transportation, and storage [<a href="#CR32" class="usa-link" aria-describedby="CR32">32</a>, <a href="#CR33" class="usa-link" aria-describedby="CR33">33</a>]. To simulate the actual detection process, we collected microscopy images specifically of teliospores from <em>T. foetida</em>, macroconidia from <em>F. graminearum</em>, and conidia from <em>A. flavus</em>. <em>For T. foetida</em>, sori were dissected from diseased wheat samples, soaked in sterile distilled water, and shaken overnight. The suspension was filtered through double-layer gauze, washed twice with 0.25% sodium hypochlorite, and rinsed repeatedly with 0.01% sterile water. <em>F. graminearum</em> and <em>A. flavus</em> were cultured on potato dextrose agar (PDA) at 25–28 °C for 7–10 days. Spore suspensions were obtained by washing sporulated cultures with 0.01% sterile water and filtering through gauze. Spore concentration was determined using a hemocytometer and calculated following standard protocols. The final concentration of each spore type was adjusted to 1 × 10⁶ spores/mL, and equal volumes of the three spore suspensions were mixed to prepare the dataset. The mixed spore solution was obtained as shown in Fig. <a href="#Fig1" class="usa-link">1</a> (a). Subsequently, to acquire image data, spore suspensions were prepared in this study. The fungal samples used in this study were obtained from the Key Laboratory of Biotoxin Analysis &amp; Assessment for State Market Regulation. An inverted microscopy system was used to capture microscopy images of the fungal spores. As shown in Fig. <a href="#Fig1" class="usa-link">1</a> (b), the system consisted of a CKX53 inverted fluorescence microscope (OLYMPUS, Japan), an EP50 microscopic digital camera (OLYMPUS, Japan), and a host computer. All microscopic images were captured at a fixed 40× magnification using an inverted microscope. The average sizes of the spores captured under 40× are: teliospore<em>s</em> of <em>T. foetida</em>, 16–25 μm [<a href="#CR34" class="usa-link" aria-describedby="CR34">34</a>]; macroconidia of <em>F. graminearum</em>, 25–50 μm × 3–5 μm [<a href="#CR35" class="usa-link" aria-describedby="CR35">35</a>]; and conidia of <em>A. flavus</em>, 3–6 μm [<a href="#CR36" class="usa-link" aria-describedby="CR36">36</a>].</p>
<figure class="fig xbox font-sm" id="Fig1"><h4 class="obj_head">Fig. 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12372353_13007_2025_1436_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d88d/12372353/7eae515f6313/13007_2025_1436_Fig1_HTML.jpg" loading="lazy" id="d33e523" height="714" width="762" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Image acquisition process. (<strong>A</strong>) Spore suspension to be tested (<strong>B</strong>) The inverted microscopy system (<strong>C</strong>) Microscopic images of spores</p></figcaption></figure></section></section><section id="Sec4"><h2 class="pmc_sec_title">Dataset preparation</h2>
<section id="Sec5"><h3 class="pmc_sec_title">Image labelling</h3>
<p id="Par15">To satisfy the model training requirements, we used Labelme software to label each image for distinguishing diverse spores. We manually classified the spores contained in the images into four categories: <em>T. foetida</em> (<em>sporidium immaturum</em>) spores were labelled “UNGXHF”; <em>T. foetida</em> (<em>sporidia maturum</em>) spores were labelled “GXHF”; <em>F. graminearum</em> spores were labelled “HGLD”; and <em>A. flavus</em> spores were labelled “HQM”. In this study, a total of 1015 microscopic images were acquired at 40× magnification using an inverted microscope, with each image having a resolution of 5540 × 3648 pixels. Although the visual distribution may vary in individual microscopy images due to differences in spore morphology and settling, all spore types were mixed in equal concentrations (1 × 10<sup>6</sup> spores/mL) and homogenized before sample preparation.</p></section></section><section id="Sec6"><h2 class="pmc_sec_title">Data cleaning and augmentation</h2>
<p id="Par16">During the image acquisition process, improper slide preparation or imaging may have resulted in the presence of foreign objects within the slides, severe spore adhesion, image defocusing or ghosting. Ultimately, we screened 959 microscopy images of fungal wheat spores. Given the relatively small number of obtained spore images, data augmentation was used in this study to expand the dataset. We used several data augmentation methods: flipping (horizontal and vertical), luminance transformation, and contrast transformation [<a href="#CR37" class="usa-link" aria-describedby="CR37">37</a>]. Through these methods, we expanded the dataset to 2330 images, which increased the complexity and diversity of the data samples. These methods also effectively prevented model overfitting and improved the generalization performance of the model.</p></section><section id="Sec7"><h2 class="pmc_sec_title">Dataset partitioning</h2>
<p id="Par17">The enhanced wheat spore images were divided into training, validation and test sets at a 6:2:2 ratio. The dataset comprised 1,398 training images, 466 validation images, and 466 test images. This division scheme ensured that sufficient images were available for efficiently constructing the model. The validation set was used to adjust the hyperparameters of the model and improve its generalizability, whereas the test set was used to evaluate the robustness and accuracy of the model.</p>
<section id="Sec8"><h3 class="pmc_sec_title">The proposed YOLO-ASF-MobileViT architecture for spore detection</h3>
<p id="Par18">In this study, we proposed a novel YOLO model to more efficiently and accurately detect pathogenic fungal wheat spores, especially small spores such as <em>A. flavus</em>. The ASF was adopted to replace the original FPN in YOLOv5s. This modification allowed the model to capture local detail features. It also focused on global features, which improved the detection accuracy achieved for spores, especially small spores. The MobileViT attention mechanism was added in front of the P3 detection head. The P3 detection head could receive feature information from the TFE module and the SSFF module in the shallow layer. Therefore, this part contained local detail features. The MobileViT attention mechanism was added before the detection head of the shallow layer to finally optimize the features. The improved YOLO-ASF-MobileViT is shown in Fig. <a href="#Fig2" class="usa-link">2</a>. By combining these two methods, the detection accuracy attained for <em>A. flavus</em> spores was significantly improved. The pathogenic wheat spore detection performance also improved overall.</p>
<figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12372353_13007_2025_1436_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d88d/12372353/084c627110ac/13007_2025_1436_Fig2_HTML.jpg" loading="lazy" id="d33e584" height="666" width="708" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>The YOLO-ASF-MobileViT architecture</p></figcaption></figure></section></section><section id="Sec9"><h2 class="pmc_sec_title">Attentional scale sequence fusion mechanism</h2>
<p id="Par20">In this study, an ASF mechanism was introduced to fuse multiscale spatial features [<a href="#CR38" class="usa-link" aria-describedby="CR38">38</a>]. This mechanism captured spore features of different sizes. The feature fusion mechanism comprised two key modules: a TFE module was designed to capture the features of dense, small spores, and the SSFF module was utilized to integrate multiscale semantic information. This approach effectively improved the feature extraction performance achieved for spore images. It provided more comprehensive and accurate feature information for the subsequent detection step. To verify the effectiveness of the introduced ASF mechanism, we compared the performance of the original YOLOv5s model to that of YOLO-ASF on the same test set.</p>
<section id="Sec10"><h3 class="pmc_sec_title">Triple feature encoding module</h3>
<p id="Par21">The traditional feature pyramid network (FPN) only up-sampled small-sized feature maps and then added them to the previous layer, thus ignoring the detailed information of large-sized feature maps. Therefore, the TFE module was introduced to capture both the detailed and overall features of spore images for accurately localizing and regressing the target bounding boxes. The TFE module extracted the P3, P4, and P5 feature maps from the backbone of YOLOv5s. Among these maps, the P3 feature map had the largest scale and smallest receptive field. It primarily captured the local detailed features of small spore images. The P4 feature map had a moderate scale and a medium-sized receptive field. It focused on the local detail features of large spore images. The P5 feature map had the smallest scale and the largest receptive field. It mainly represented the global semantic features of spore images.</p>
<p id="Par22">The specific structure of the TFE module is shown in Fig. <a href="#Fig3" class="usa-link">3</a>. After extracting features from the P3 and P5 feature maps via 2D convolution, the feature maps were normalized in scale through downsampling and upsampling, respectively. A fusion operation combining global pooling and average pooling was employed to downsample the P3 feature map, which ensured the diversity of the high-resolution detail features. The nearest-neighbour interpolation method was used to upsample the P5 feature map, which mitigated the loss of information from the low-resolution features. The normalized P3 and P5 feature maps were consistent in scale with the P4 feature map. Convolution and concatenation were used to extract the information from the normalized P3 and P5 feature maps and fuse it, respectively, with the P4 feature map. The fused feature map channels were three times as numerous as those in the original feature maps (P3, P4, and P5).</p>
<figure class="fig xbox font-sm" id="Fig3"><h4 class="obj_head">Fig. 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12372353_13007_2025_1436_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d88d/12372353/aa234a519a6e/13007_2025_1436_Fig3_HTML.jpg" loading="lazy" id="d33e610" height="323" width="669" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Triple feature encoding module</p></figcaption></figure></section><section id="Sec11"><h3 class="pmc_sec_title">Scale sequence feature fusion</h3>
<p id="Par24">The wheat spore images were characterized by their sizes, shapes, and colours. The <em>A. flavus</em> spores were smaller than the <em>T. foetida</em> spores, whereas the <em>F. graminearum</em> spores were more elongated. The tiny features of the <em>A. flavus</em> spores could be clearly recognized in the high-resolution images. However, as the depth increased, the image resolution decreased, which led to the loss of tiny features. The traditional FPN fusion method failed to effectively utilize the relationships between different levels of pyramid feature maps, which resulted in a lack of communication between the local and global information. Therefore, we introduced SSFF. This module was inspired by video frame processing methods [<a href="#CR39" class="usa-link" aria-describedby="CR39">39</a>]. It performed interactive fusion on multiscale information by stitching multiscale feature maps. SSFF also utilized 3D convolution for feature extraction purposes. The specific structure of this module is shown in Fig. <a href="#Fig4" class="usa-link">4</a>. First, we used 1 × 1 2D convolutions to adjust the numbers of channels in the P4 and P5 feature maps, which ensured that both maps had 256 channels. Next, the nearest-neighbour up-sampling method was used to normalize the sizes of the P3, P4, and P5 feature maps to match the feature map with the highest resolution (P3). Then, the 3D [height, width, channel] tensors were converted into 4D [depth, height, width, channel] tensors. 3D convolution, batch normalization, and the leaky rectified linear unit (LeakyReLU) activation function were used to extract features from the 4D tensors. Finally, maximum pooling was used to down-sample the feature maps, and the down-sampled outputs were fused with the output of the TFE module.</p>
<figure class="fig xbox font-sm" id="Fig4"><h4 class="obj_head">Fig. 4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12372353_13007_2025_1436_Fig4_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d88d/12372353/2bfafb944fa3/13007_2025_1436_Fig4_HTML.jpg" loading="lazy" id="d33e643" height="574" width="762" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Scale sequence feature fusion</p></figcaption></figure></section></section><section id="Sec12"><h2 class="pmc_sec_title">MobileViT attention mechanism</h2>
<p id="Par26">The traditional attention mechanism focused on the channel and spatial aspects of images, whereas the ASF mechanism fused multichannel and multiscale information. Therefore, we shifted the focus of the attention mechanism to global semantic information and local detail information. We introduced the MobileViT attention mechanism, which is based on the ViT [<a href="#CR40" class="usa-link" aria-describedby="CR40">40</a>]. Compared with the traditional ViT, in the MobileViT, the transformer extracted global features, whereas the CNN extracted local features. The MobileViT endowed the model with spatial inductive bias and global representation capabilities [<a href="#CR41" class="usa-link" aria-describedby="CR41">41</a>]. The structure of the MobileViT is shown in Fig. <a href="#Fig5" class="usa-link">5</a>. It included local feature representation, global feature representation, and feature fusion components. In the local feature representation module, an n×n convolution was used to extract local features from the input feature map, and a 1 × 1 convolution was employed to adjust the number of channels. In the global feature representation module, the feature maps were divided into multiple fixed-size blocks, and each block was flattened into a one-dimensional vector. Through linear transformation, these one-dimensional vectors were mapped to higher-dimensional feature vectors. These feature vectors with positional encoding information were subsequently concatenated and fed into the transformer encoder. The transformer encoder incorporated a multihead attention mechanism and a positional feedforward neural network. The self-attention mechanism was used to capture the dependencies between different positions in the input image. The positional feedforward neural network contained multiple hidden layers to perform nonlinear mapping and feature extraction on the feature information at each position. Finally, the feature image processed by the ViT module was fused with the original image. Through the MobileViT attention mechanism, the local features were better preserved, and global features were obtained. This enabled the model to fully utilize both global semantic information and local details [<a href="#CR42" class="usa-link" aria-describedby="CR42">42</a>]. This approach improved the accuracy of wheat spore detection, particularly for smaller <em>A. flavus</em> spores.</p>
<figure class="fig xbox font-sm" id="Fig5"><h3 class="obj_head">Fig. 5.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12372353_13007_2025_1436_Fig5_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d88d/12372353/e50a53f2f841/13007_2025_1436_Fig5_HTML.jpg" loading="lazy" id="d33e686" height="455" width="669" alt="Fig. 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>MobileViT attention mechanism</p></figcaption></figure><p id="Par27">In this study, we conducted ablation experiments to validate the performance of the introduced MobileViT attention mechanism in terms of spore detection, particularly its ability to detect small spores. We compared it with the SE (Squeeze-and-Excitation) network [<a href="#CR43" class="usa-link" aria-describedby="CR43">43</a>], SA (Spatial Attention) [<a href="#CR44" class="usa-link" aria-describedby="CR44">44</a>], ECA (Efficient Channel Attention) [<a href="#CR45" class="usa-link" aria-describedby="CR45">45</a>] and the CBAM (Convolutional Block Attention Module) [<a href="#CR46" class="usa-link" aria-describedby="CR46">46</a>].</p>
<section id="Sec13"><h3 class="pmc_sec_title">Model evaluation</h3>
<p id="Par29">In this study, we compared the improved YOLOv5s algorithm (YOLOv5s-ASFF-MobileViT) with the original YOLOv5s algorithm. First, we compared the mAP_0.5 iteration curves of these models to evaluate their convergence performance during the training process. Second, to verify the robustness of the proposed model, we evaluated its ability to detect spores under different extreme environmental conditions, which included adhesive spores, overlapping spores, blurry spores, and spores located in environments with interference from impurities. Finally, we verified the effectiveness of the improved YOLOv5s algorithm in terms of spore detection through a comparison with nine detection algorithms: SSD [<a href="#CR47" class="usa-link" aria-describedby="CR47">47</a>], YOLOv5s [<a href="#CR48" class="usa-link" aria-describedby="CR48">48</a>], CenterNet [<a href="#CR49" class="usa-link" aria-describedby="CR49">49</a>], the Faster R-CNN [<a href="#CR50" class="usa-link" aria-describedby="CR50">50</a>], the, TPH-YOLO [<a href="#CR51" class="usa-link" aria-describedby="CR51">51</a>], MG-YOLO [<a href="#CR52" class="usa-link" aria-describedby="CR52">52</a>], SPD-YOLO [<a href="#CR53" class="usa-link" aria-describedby="CR53">53</a>], YOLO-ECA-ASFF [<a href="#CR26" class="usa-link" aria-describedby="CR26">26</a>], and YOLO-CG-HS [<a href="#CR37" class="usa-link" aria-describedby="CR37">37</a>]. Each model was trained, validated, and tested under the same conditions. several metrics were utilized to comprehensively evaluate the performance of the tested models, including precision, recall, average precision (AP), and mAP. Precision was used to measure the classification accuracy achieved for positive categories. Recall was used to measure the ability of each model to detect positive samples. The AP was evaluated by calculating the precision‒recall curve for each category, reflecting the precision of the corresponding model at different recall levels. AP represents the average precision attained by a model for a single category. mAP represents the overall average precision achieved by a model across multiple categories. The precision, recall, AP, and mAP metrics are calculated as follows:</p>
<table class="disp-formula p" id="Equ1"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d88d/12372353/3b1c04a88f43/d33e720.gif" loading="lazy" id="d33e720" alt="graphic file with name d33e720.gif"></td>
<td class="label">1</td>
</tr></table>
<table class="disp-formula p" id="Equ2"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d88d/12372353/418da906828e/d33e726.gif" loading="lazy" id="d33e726" alt="graphic file with name d33e726.gif"></td>
<td class="label">2</td>
</tr></table>
<table class="disp-formula p" id="Equ3"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d88d/12372353/668e4a82276e/d33e732.gif" loading="lazy" id="d33e732" alt="graphic file with name d33e732.gif"></td>
<td class="label">3</td>
</tr></table>
<table class="disp-formula p" id="Equ4"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d88d/12372353/3c4dff948060/d33e739.gif" loading="lazy" id="d33e739" alt="graphic file with name d33e739.gif"></td>
<td class="label">4</td>
</tr></table>
<p id="Par30">where TP represents the number of true-positive samples and FP represents the number of false-positive samples. Similarly, TN represents the number of true-negative samples, and FN represents the number of false-negative samples. Additionally, r denotes the recall, and P(r) represents the precision corresponding to r. N denotes the number of data categories detected in this study.</p></section><section id="Sec14"><h3 class="pmc_sec_title">Experimental platform</h3>
<p id="Par31">In this study, all the models were trained and tested in the following experimental environment: the deep learning framework was PyTorch 1.12.1, and the operating system was Ubuntu 16.04. The computer was equipped with an Intel(R) Core(TM) i7-9700 K 8-core processor at 3.60 GHz and 16 GB of RAM. To increase the training speed, we utilized an NVIDIA GeForce GTX 2080Ti GPU with 11 GB of video memory, CUDA version 11.3, and cuDNN version 8.2.1.</p></section></section><section id="Sec15"><h2 class="pmc_sec_title">Results and discussion</h2>
<section id="Sec16"><h3 class="pmc_sec_title">Comparison between the original and improved feature fusion mechanisms</h3>
<p id="Par32">The results of a comparison between the original and improved feature fusion mechanisms are shown in Table <a href="#Tab1" class="usa-link">1</a>. Compared with that of the original model, the overall precision of YOLO-ASF increased by 3%, and its overall recall increased by 0.6%. Regarding the detection of <em>F. graminearum</em>, <em>A. flavus</em>, and <em>T. foetida</em> (<em>sporidium immaturum</em>), the mAP_0.5 values increased by 2%, 2.7%, and 0.3%, respectively. However, for <em>T. foetida</em> (<em>sporidium maturum</em>), the mAP_0.5 of the improved model did not increase. Overall, mAP_0.5 increased by 1.1%. The results showed that the YOLO-ASF model more accurately detected spores than did the other methods. Notably, for small and dense <em>A. flavus</em> spores, the introduced ASF mechanism increased the mAP_0.5 of the proposed model by 2.7%, which significantly enhanced its performance in terms of detecting small target spores. This was attributed to the TFE and SSFF modules contained in the ASF mechanism. In the TFE module, three feature maps with large, medium, and small sizes were convolved once and spliced in the channel dimension, which prevented the loss of small spore features. Moreover, the global semantic information contained in the deep feature maps was combined with the local detail information of the SSFF module, which enabled the model to more comprehensively understand the image content. The ASF mechanism significantly improved the ability of the model to extract target features, which in turn improved its detection performance. However, the original model achieved a high detection accuracy of 99% because of the relatively large sizes and distinctive colour texture features of <em>T. foetida</em> (<em>sporidium maturum</em>) spores. The detection performance was close to the optimal level. It was difficult to further improve the detection accuracy under the current conditions.</p>
<section class="tw xbox font-sm" id="Tab1"><h4 class="obj_head">Table 1.</h4>
<div class="caption p"><p>Comparison of feature fusion mechanisms before and after improvement</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead>
<tr>
<th align="left" rowspan="2" colspan="1">Models</th>
<th align="left" rowspan="2" colspan="1">
<em>P</em> (%)</th>
<th align="left" rowspan="2" colspan="1">
<em>R</em> (%)</th>
<th align="left" colspan="5" rowspan="1">mAP_0.5 (%)</th>
</tr>
<tr>
<th align="left" colspan="1" rowspan="1">GXHF</th>
<th align="left" colspan="1" rowspan="1">HGLD</th>
<th align="left" colspan="1" rowspan="1">HQM</th>
<th align="left" colspan="1" rowspan="1">UNGXHF</th>
<th align="left" colspan="1" rowspan="1">Total</th>
</tr>
</thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">Original YOLOv5s</td>
<td align="center" colspan="1" rowspan="1">91.8</td>
<td align="center" colspan="1" rowspan="1">94.2</td>
<td align="center" colspan="1" rowspan="1">99.1</td>
<td align="center" colspan="1" rowspan="1">97.5</td>
<td align="center" colspan="1" rowspan="1">86.5</td>
<td align="center" colspan="1" rowspan="1">98.3</td>
<td align="center" colspan="1" rowspan="1">95.4</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">YOLO-ASF</td>
<td align="center" colspan="1" rowspan="1">94.8</td>
<td align="center" colspan="1" rowspan="1">94.8</td>
<td align="center" colspan="1" rowspan="1">99.1</td>
<td align="center" colspan="1" rowspan="1">99.5</td>
<td align="center" colspan="1" rowspan="1">89.2</td>
<td align="center" colspan="1" rowspan="1">98.6</td>
<td align="center" colspan="1" rowspan="1">96.5</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><p id="Par33">We applied gradient-weighted class activation mapping (Grad-CAM) to further analyse the performance of YOLO-ASF. Grad-CAM is a visualization technique that is used to interpret neural network prediction results [<a href="#CR54" class="usa-link" aria-describedby="CR54">54</a>]. The gradient computation process in Grad-CAM was used to show where the model was interested in different objects. The detection weights yielded by the original YOLOv5s model and the YOLO-ASF model for the four spore classes included in the same image are visualized in Fig. <a href="#Fig6" class="usa-link">6</a>. The four left panels of Fig. <a href="#Fig6" class="usa-link">6</a> visualize the weights of the original YOLOv5s model, whereas the four right panels visualize the weights of the YOLO-ASF model. The first and third images of Fig. <a href="#Fig6" class="usa-link">6</a> show that the original YOLOv5s model did not sufficiently focus on spores, and its weights covered a wide and imprecise area, even including areas without spores. In the case with a dense spore distribution, this model also ignored certain spores. In contrast, the two images in the fourth row of Fig. <a href="#Fig6" class="usa-link">6</a> show that the original YOLOv5s model did not accurately focus on small objects. Many small impurities were also incorrectly detected as spores. However, the YOLO-ASF model could focus on detecting small spores and selectively filtering out impurities. The TFE module integrated small spore features derived from three different resolution feature maps, enriching the spore features. The SSFF module combined the detailed information and global information acquired from the feature maps. This allowed for the precise localization of small spores. In summary, the ASF mechanism comprehensively improved the detection performance of the proposed model, especially in terms of detecting small spores.</p>
<figure class="fig xbox font-sm" id="Fig6"><h4 class="obj_head">Fig. 6.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12372353_13007_2025_1436_Fig6_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d88d/12372353/545bbd331cf0/13007_2025_1436_Fig6_HTML.jpg" loading="lazy" id="d33e899" height="922" width="762" alt="Fig. 6"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Grad-CAM Visualization Comparison. (<strong>A</strong>) and (<strong>B</strong>) show the Grad-CAM visualization images of YOLOv5s and YOLO-ASF-MobileViT, respectively</p></figcaption></figure></section><section id="Sec17"><h3 class="pmc_sec_title">Comparison among different attention mechanisms</h3>
<p id="Par36">A comparison among different attention mechanisms is shown in Table <a href="#Tab2" class="usa-link">2</a>. All performance metrics reported in Table <a href="#Tab2" class="usa-link">2</a> were evaluated on the independent test set. Compared with the YOLO-ASF model, only YOLO-ASF-CBAM and YOLO-ASF-MobileViT improved the <em>A. flavus</em> spore detection process, with their mAP_0.5 values increasing by 0.7% and 1.6%, respectively. This occurred because the attention mechanisms, such as SE, SA and ECA, focused on the dependencies between channels but ignored spatial information. In contrast, the CBAM combined a spatial attention mechanism with a channel attention mechanism, which enabled the model to obtain more effective features from both the channel and spatial dimensions. Both the CBAM and MobileViT focused on spatial and channel information, which significantly improved the resulting small target detection performance. Notably, the precision and recall of the YOLO-ASF-CBAM model were superior to those of the YOLO-ASF-MobileViT model, but the mAP_0.5 of the former was lower than that of the latter. The CBAM could correctly detect the targets, but it failed to precisely regress the bounding boxes because of its inaccurate localization effect. The MobileViT did not involve pooling operations. Therefore, it better preserved the global semantic information, which allowed for the target boxes to be more accurately predicted. In object detection tasks, we paid more attention to mAP_0.5, as the precision and recall values achieved at different confidence levels were considered in this metric, which better reflected the overall performance of the models. The YOLO-ASF-MobileViT model had the best performance, with an overall detection mAP_0.5 of 97%, which was 1.6% better than that of the original YOLOv5s model. The MobileViT attention mechanism combines the local feature extraction ability of a CNN with the global feature extraction ability of a ViT. This approach was more suitable for detecting various spores with significant size differences.</p>
<section class="tw xbox font-sm" id="Tab2"><h4 class="obj_head">Table 2.</h4>
<div class="caption p"><p>Comparisons of the three different feature fusion mechanism modules</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead>
<tr>
<th align="left" rowspan="2" colspan="1">Models</th>
<th align="left" rowspan="2" colspan="1">
<em>P</em> (%)</th>
<th align="left" rowspan="2" colspan="1">
<em>R</em> (%)</th>
<th align="left" colspan="5" rowspan="1">mAP_0.5 (%)</th>
</tr>
<tr>
<th align="left" colspan="1" rowspan="1">GXHF</th>
<th align="left" colspan="1" rowspan="1">HGLD</th>
<th align="left" colspan="1" rowspan="1">HQM</th>
<th align="left" colspan="1" rowspan="1">UNGXHF</th>
<th align="left" colspan="1" rowspan="1">Total</th>
</tr>
</thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">YOLO-ASF</td>
<td align="center" colspan="1" rowspan="1">94.8</td>
<td align="center" colspan="1" rowspan="1">95.4</td>
<td align="center" colspan="1" rowspan="1">99.0</td>
<td align="center" colspan="1" rowspan="1">
<strong>99.5</strong>
</td>
<td align="center" colspan="1" rowspan="1">89.2</td>
<td align="center" colspan="1" rowspan="1">98.6</td>
<td align="center" colspan="1" rowspan="1">96.5</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">YOLO-ASF-SE</td>
<td align="center" colspan="1" rowspan="1">94.5</td>
<td align="center" colspan="1" rowspan="1">95.4</td>
<td align="center" colspan="1" rowspan="1">98.8</td>
<td align="center" colspan="1" rowspan="1">99.2</td>
<td align="center" colspan="1" rowspan="1">88.4</td>
<td align="center" colspan="1" rowspan="1">98.4</td>
<td align="center" colspan="1" rowspan="1">96.2</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">YOLO-ASF-SA</td>
<td align="center" colspan="1" rowspan="1">94.8</td>
<td align="center" colspan="1" rowspan="1">95.1</td>
<td align="center" colspan="1" rowspan="1">98.7</td>
<td align="center" colspan="1" rowspan="1">99.4</td>
<td align="center" colspan="1" rowspan="1">89.1</td>
<td align="center" colspan="1" rowspan="1">98.7</td>
<td align="center" colspan="1" rowspan="1">96.5</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">YOLO-ASF-ECA</td>
<td align="center" colspan="1" rowspan="1">95.0</td>
<td align="center" colspan="1" rowspan="1">94.8</td>
<td align="center" colspan="1" rowspan="1">98.5</td>
<td align="center" colspan="1" rowspan="1">99.3</td>
<td align="center" colspan="1" rowspan="1">88.9</td>
<td align="center" colspan="1" rowspan="1">98.1</td>
<td align="center" colspan="1" rowspan="1">96.2</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">YOLO-ASF-CBAM</td>
<td align="center" colspan="1" rowspan="1">
<strong>95.1</strong>
</td>
<td align="center" colspan="1" rowspan="1">
<strong>95.5</strong>
</td>
<td align="center" colspan="1" rowspan="1">98.8</td>
<td align="center" colspan="1" rowspan="1">99.4</td>
<td align="center" colspan="1" rowspan="1">89.9</td>
<td align="center" colspan="1" rowspan="1">98.3</td>
<td align="center" colspan="1" rowspan="1">96.6</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">YOLO-ASF-MobileViT</td>
<td align="center" colspan="1" rowspan="1">94.8</td>
<td align="center" colspan="1" rowspan="1">95.4</td>
<td align="center" colspan="1" rowspan="1">
<strong>99.1</strong>
</td>
<td align="center" colspan="1" rowspan="1">99.4</td>
<td align="center" colspan="1" rowspan="1">
<strong>90.8</strong>
</td>
<td align="center" colspan="1" rowspan="1">
<strong>98.8</strong>
</td>
<td align="center" colspan="1" rowspan="1">
<strong>97.0</strong>
</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section></section><section id="Sec18"><h3 class="pmc_sec_title">Comparison between the performances achieved before and after improving the model</h3>
<section id="Sec19"><h4 class="pmc_sec_title">Model training and convergence performance comparison</h4>
<p id="Par38">A comparison between the training iteration curves produced before and after improving the model is shown in Fig. <a href="#Fig7" class="usa-link">7</a>. Both models exhibited significant fluctuations in their mAP_0.5 values during the first 150 iterations. Because the pretrained model was not used in this experiment, the model parameters were randomly initialized, resulting in a poor data fitting ability. Consequently, the initial training process was unstable. During the last 50 iterations, the mAP_0.5 values of both models gradually stabilized. Notably, after stabilization, the mAP_0.5 of the improved YOLOv5s model consistently remained higher than that of the original YOLOv5s model. Additionally, the improved model exhibited smaller fluctuations in its mAP_0.5 data and more quickly converged to a relatively stable state. Figure <a href="#Fig7" class="usa-link">7</a>. shows the training and validation performance curves, reflecting intermediate results on the validation set. In contrast, all metrics in Table <a href="#Tab2" class="usa-link">2</a> were computed on an independent test set. Slight differences are expected, as the test set evaluates the fully converged model on unseen data, demonstrating better generalization.</p>
<figure class="fig xbox font-sm" id="Fig7"><h5 class="obj_head">Fig. 7.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d88d/12372353/b3c75e8a82f1/13007_2025_1436_Fig7_HTML.jpg" loading="lazy" id="d33e1112" height="443" width="646" alt="Fig. 7"></p>
<div class="p text-right font-secondary"><a href="figure/Fig7/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>The comparison of training iteration curves before and after model improvement</p></figcaption></figure></section></section><section id="Sec20"><h3 class="pmc_sec_title">Model robustness performance comparison</h3>
<p id="Par40">Figures <a href="#Fig8" class="usa-link">8</a>, <a href="#Fig9" class="usa-link">9</a> and <a href="#Fig10" class="usa-link">10</a> show the detection performance of the model both before and after improvements, under four different extreme conditions. First, spore adhesion was shown in Fig. <a href="#Fig8" class="usa-link">8</a>. In sample 1 of Fig. <a href="#Fig8" class="usa-link">8</a>, T. <em>foetida</em> spores were adhered to each other. The original model misidentified the two adhered spores as three separate spores. In sample 2 of Fig. <a href="#Fig8" class="usa-link">8</a>, T. <em>caries</em> spores were adhered to <em>F. graminearum</em> spores. The original model misidentified a single <em>F. graminearum</em> spore as two separate spores. However, the improved model significantly reduced the false detection rate attained for adherent spores. The original model was overly concerned with the local features of the spores and ignored their overall features, thus misidentifying them as new spores. A case with overlapping spores is shown in Fig. <a href="#Fig9" class="usa-link">9</a>. In sample 1 of Fig. <a href="#Fig9" class="usa-link">9</a>, two <em>F. graminearum</em> spores overlapped. The original model could not accurately determine their target boxes and attempted to detect targets with multiple target boxes. In sample 2 of Fig. <a href="#Fig9" class="usa-link">9</a>, three <em>T. foetida</em> spores overlapped, with the middle spore partially covering the other two spores, which caused them to form a single cluster. It was difficult for the original model to accurately locate their positions, which resulted in false detections. This occurred because the original model identified multiple local features within the overlapping area and attempted to label these features with multiple target boxes. These target boxes had different sizes, and the overlap region between them was small. The intersection over union (IoU) between the adjacent target boxes did not reach the filtering threshold of the nonmaximum suppression (NMS) algorithm, which led to false detection results. Blurry spore images are shown in Fig. <a href="#Fig10" class="usa-link">10</a>. In sample 1 of Fig. <a href="#Fig10" class="usa-link">10</a>, the upper-right corner of the image shows a typical <em>F. graminearum</em> spore outline. However, owing to the influence of background lighting, its phenotypic features were not distinct. Therefore, the original model missed this target spore, whereas the improved model accurately detected the blurry spore. Similarly, in sample 2 of Fig. <a href="#Fig10" class="usa-link">10</a>, the upper-right corner shows the outlines of two <em>F. graminearum</em> spores. The original model detected the two spores as a single entity, whereas the improved model correctly detected both spores. During model training, the size and resolution of the feature map decreased as the model depth and the step size of the convolution operation increased. Therefore, the blurred spores lacked clear edges and texture features. This caused the deep feature maps of the original model to easily lose detail information. The impurity interference case is shown in Fig. <a href="#Fig11" class="usa-link">11</a>. In sample 1 of Fig. <a href="#Fig11" class="usa-link">11</a>, impurities appear in the lower-right corner, and the original model misclassified them as <em>F. graminearum</em>. In sample 2 of Fig. <a href="#Fig11" class="usa-link">11</a>, the same spore-like impurity appears in the lower-right corner, resembling <em>A. flavus</em>. The original model misclassified them as both <em>A. flavus</em> and <em>T. foetida</em>. In the impurity interference case, the backbone layers of the backbone layer performed poorly in terms of describing the shape and texture features of spores, leading to misdetection.</p>
<figure class="fig xbox font-sm" id="Fig8"><h4 class="obj_head">Fig. 8.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12372353_13007_2025_1436_Fig8_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d88d/12372353/4c9e88dbb3c3/13007_2025_1436_Fig8_HTML.jpg" loading="lazy" id="d33e1225" height="625" width="762" alt="Fig. 8"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig8/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>The results of detecting adhesive spores</p></figcaption></figure><figure class="fig xbox font-sm" id="Fig9"><h4 class="obj_head">Fig. 9.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12372353_13007_2025_1436_Fig9_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d88d/12372353/3e4c2dfeb727/13007_2025_1436_Fig9_HTML.jpg" loading="lazy" id="d33e1235" height="613" width="762" alt="Fig. 9"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig9/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>The results of detecting overlapping spores</p></figcaption></figure><figure class="fig xbox font-sm" id="Fig10"><h4 class="obj_head">Fig. 10.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12372353_13007_2025_1436_Fig10_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d88d/12372353/78c8c69e039d/13007_2025_1436_Fig10_HTML.jpg" loading="lazy" id="d33e1245" height="610" width="762" alt="Fig. 10"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig10/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>The results of detecting blurry spores</p></figcaption></figure><figure class="fig xbox font-sm" id="Fig11"><h4 class="obj_head">Fig. 11.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12372353_13007_2025_1436_Fig11_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d88d/12372353/9eaf5faf4a00/13007_2025_1436_Fig11_HTML.jpg" loading="lazy" id="d33e1255" height="605" width="762" alt="Fig. 11"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig11/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>The results of detecting spores in impurity interference environment</p></figcaption></figure><p id="Par41">The ASF mechanism and MobileViT attention mechanism introduced in this paper effectively solved the above problems. The MobileViT was added to the backbone layer. The self-attention mechanisms contained in the MobileViT were employed to capture the global relationships within features. Unlike traditional CNN algorithms with fixed receptive fields, this approach enabled the model to dynamically learn the long-range dependencies between different regions. The MobileViT incorporated CNNs as local feature extractors. This effectively integrated the local and global features. This approach improved the feature representation ability of the model and effectively addressed the misdetection issues caused by spore adhesion, intersection, and overlapping. In the ASF mechanism, the TFE module spliced features with three different sizes in the spatial dimension to capture detailed small spore information. The SSFF mechanism effectively fused the multichannel feature maps of P3, P4, and P5, which captured different spatial scales covering diverse spores with various sizes and shapes. This method mitigated the loss of detail information. Compared with the original model, the improved model exhibited greater robustness. It demonstrated better detection capabilities in terms of handling extreme conditions such as spore adhesion, overlap, impurity interference, and blurry spores. However, in real-world samples, interference may arise from fungal spores of other genera commonly found in cereals, such as <em>Alternaria</em>, <em>Bipolaris</em>, and <em>Penicillium</em> [<a href="#CR55" class="usa-link" aria-describedby="CR55">55</a>]. This may affect the model’s specificity. Thus, expanding the spore classes used for training will be essential to further enhance detection performance under complex field conditions.</p></section><section id="Sec21"><h3 class="pmc_sec_title">Performance comparisons with state-of-the-art detection methods</h3>
<p id="Par46">The quantitative results of the state-of-the-art detection methods in the test are shown in Table <a href="#Tab3" class="usa-link">3</a>, which contains the precision, recall and mAP_0.5 values produced by each model. Regarding the traditional one-stage detection networks, the mAP_0.5 of the improved model was 1.6%, 24.9%, and 5.1% higher than those of YOLOv5s, the SSD, and CenterNet, respectively. With respect to the traditional two-stage detection network, the mAP_0.5 of the improved model was 23.3% higher than that of the Faster R-CNN. The performance of the SSD in terms of detecting <em>A. flavus</em> spores was poor, with a recall of only 60.1% and an mAP_0.5 value of just 18.1%. This occurred because the SSD algorithm lacked a feature resampling step, which easily led to missed detections [<a href="#CR46" class="usa-link" aria-describedby="CR46">46</a>]. CenterNet significantly surpassed the SSD in overall detection performance, but its ability to detect <em>A. flavus</em> spores was relatively poor, with an mAP_0.5 of only 76.4%. In the two-stage detection networks, such as the Faster R-CNN, a base neural network was utilized for feature extraction purposes, and an RPN was employed on deeper feature maps to generate target boxes. The Faster R-CNN had the lowest mAP_0.5 for <em>A. flavus</em> spores at only 17.2% because of the small spore features lost from the depth feature maps.</p>
<section class="tw xbox font-sm" id="Tab3"><h4 class="obj_head">Table 3.</h4>
<div class="caption p"><p>Performance comparisons with state-of-the-art detection methods</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead>
<tr>
<th align="left" rowspan="2" colspan="1">Models</th>
<th align="left" rowspan="2" colspan="1">
<em>P</em> (%)</th>
<th align="left" rowspan="2" colspan="1">
<em>R</em> (%)</th>
<th align="left" colspan="5" rowspan="1">AP_0.5 (%)</th>
<th align="left" rowspan="2" colspan="1">Inference time (ms)</th>
<th align="left" rowspan="2" colspan="1">Params (M)</th>
</tr>
<tr>
<th align="left" colspan="1" rowspan="1">GXHF</th>
<th align="left" colspan="1" rowspan="1">HGLD</th>
<th align="left" colspan="1" rowspan="1">HQM</th>
<th align="left" colspan="1" rowspan="1">UNGXHF</th>
<th align="left" colspan="1" rowspan="1">Mean</th>
</tr>
</thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">YOLOv5s</td>
<td align="center" colspan="1" rowspan="1">91.8</td>
<td align="center" colspan="1" rowspan="1">94.2</td>
<td align="center" colspan="1" rowspan="1">99.1</td>
<td align="center" colspan="1" rowspan="1">97.5</td>
<td align="center" colspan="1" rowspan="1">86.5</td>
<td align="center" colspan="1" rowspan="1">98.3</td>
<td align="center" colspan="1" rowspan="1">95.4</td>
<td align="center" colspan="1" rowspan="1">5.8</td>
<td align="center" colspan="1" rowspan="1">7.0</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">CenterNet</td>
<td align="center" colspan="1" rowspan="1">93.3</td>
<td align="center" colspan="1" rowspan="1">86.1</td>
<td align="center" colspan="1" rowspan="1">99.2</td>
<td align="center" colspan="1" rowspan="1">97.3</td>
<td align="center" colspan="1" rowspan="1">76.4</td>
<td align="center" colspan="1" rowspan="1">94.9</td>
<td align="center" colspan="1" rowspan="1">91.9</td>
<td align="center" colspan="1" rowspan="1">24.8</td>
<td align="center" colspan="1" rowspan="1">54.6</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Faster R-CNN</td>
<td align="center" colspan="1" rowspan="1">58.5</td>
<td align="center" colspan="1" rowspan="1">83.3</td>
<td align="center" colspan="1" rowspan="1">97.3</td>
<td align="center" colspan="1" rowspan="1">93.0</td>
<td align="center" colspan="1" rowspan="1">17.2</td>
<td align="center" colspan="1" rowspan="1">87.4</td>
<td align="center" colspan="1" rowspan="1">73.7</td>
<td align="center" colspan="1" rowspan="1">38.5</td>
<td align="center" colspan="1" rowspan="1">112.2</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">SSD</td>
<td align="center" colspan="1" rowspan="1">84.9</td>
<td align="center" colspan="1" rowspan="1">60.1</td>
<td align="center" colspan="1" rowspan="1">97.5</td>
<td align="center" colspan="1" rowspan="1">88.2</td>
<td align="center" colspan="1" rowspan="1">18.1</td>
<td align="center" colspan="1" rowspan="1">84.5</td>
<td align="center" colspan="1" rowspan="1">72.1</td>
<td align="center" colspan="1" rowspan="1">11.2</td>
<td align="center" colspan="1" rowspan="1">25.1</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">TPH-YOLO</td>
<td align="center" colspan="1" rowspan="1">96.3</td>
<td align="center" colspan="1" rowspan="1">95.3</td>
<td align="center" colspan="1" rowspan="1">99.2</td>
<td align="center" colspan="1" rowspan="1">94.8</td>
<td align="center" colspan="1" rowspan="1">89.1</td>
<td align="center" colspan="1" rowspan="1">
<strong>99.1</strong>
</td>
<td align="center" colspan="1" rowspan="1">95.6</td>
<td align="center" colspan="1" rowspan="1">
<strong>5.6</strong>
</td>
<td align="center" colspan="1" rowspan="1">7.3</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">MG-YOLO</td>
<td align="center" colspan="1" rowspan="1">93.7</td>
<td align="center" colspan="1" rowspan="1">94.8</td>
<td align="center" colspan="1" rowspan="1">99.1</td>
<td align="center" colspan="1" rowspan="1">95.3</td>
<td align="center" colspan="1" rowspan="1">89.3</td>
<td align="center" colspan="1" rowspan="1">98.3</td>
<td align="center" colspan="1" rowspan="1">95.5</td>
<td align="center" colspan="1" rowspan="1">5.8</td>
<td align="center" colspan="1" rowspan="1">6.5</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">SPD-YOLO</td>
<td align="center" colspan="1" rowspan="1">93.5</td>
<td align="center" colspan="1" rowspan="1">94.8</td>
<td align="center" colspan="1" rowspan="1">98.7</td>
<td align="center" colspan="1" rowspan="1">97.6</td>
<td align="center" colspan="1" rowspan="1">85.7</td>
<td align="center" colspan="1" rowspan="1">97.0</td>
<td align="center" colspan="1" rowspan="1">94.8</td>
<td align="center" colspan="1" rowspan="1">8.0</td>
<td align="center" colspan="1" rowspan="1">7.4</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">YOLO-ECA-ASFF</td>
<td align="center" colspan="1" rowspan="1">
<strong>95.0</strong>
</td>
<td align="center" colspan="1" rowspan="1">93.2</td>
<td align="center" colspan="1" rowspan="1">
<strong>99.3</strong>
</td>
<td align="center" colspan="1" rowspan="1">94.8</td>
<td align="center" colspan="1" rowspan="1">88.0</td>
<td align="center" colspan="1" rowspan="1">98.2</td>
<td align="center" colspan="1" rowspan="1">95.0</td>
<td align="center" colspan="1" rowspan="1">8.8</td>
<td align="center" colspan="1" rowspan="1">7.6</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">YOLO-CG-HS</td>
<td align="center" colspan="1" rowspan="1">91.2</td>
<td align="center" colspan="1" rowspan="1">90.7</td>
<td align="center" colspan="1" rowspan="1">
<strong>99.3</strong>
</td>
<td align="center" colspan="1" rowspan="1">95.4</td>
<td align="center" colspan="1" rowspan="1">89.8</td>
<td align="center" colspan="1" rowspan="1">98.8</td>
<td align="center" colspan="1" rowspan="1">95.8</td>
<td align="center" colspan="1" rowspan="1">9.3</td>
<td align="center" colspan="1" rowspan="1">
<strong>3.8</strong>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Ours</td>
<td align="center" colspan="1" rowspan="1">94.8</td>
<td align="center" colspan="1" rowspan="1">
<strong>95.4</strong>
</td>
<td align="center" colspan="1" rowspan="1">99.1</td>
<td align="center" colspan="1" rowspan="1">
<strong>99.4</strong>
</td>
<td align="center" colspan="1" rowspan="1">
<strong>90.8</strong>
</td>
<td align="center" colspan="1" rowspan="1">98.8</td>
<td align="center" colspan="1" rowspan="1">
<strong>97.0</strong>
</td>
<td align="center" colspan="1" rowspan="1">
<strong>10.8</strong>
</td>
<td align="center" colspan="1" rowspan="1">8.4</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><p id="Par47">Furthermore, the mAP of TPH-YOLO, MG-YOLO, SPD-YOLO, YOLO-ECA-ASFF, and YOLO-CG-HS in detecting total fungal spores was 95.6%, 95.5%, 94.8%, 95.0%, and 95.8%, respectively, and the mAP of the proposed model was at least 1.2% higher than theirs. Particularly, these cutting-edge spore detection methods had relatively low accuracy in detecting <em>A. flavus</em> spores (AP_0.5 &lt; 90%). By comparing mAP, Precision, and Recall, YOLO-ASF-MobileViT showed higher accuracy, especially in detecting small spores. This was because the ASF module effectively fused multi-scale feature information of spores, enhancing fine details. Meanwhile, MobileViT better captured the local and global features of microspores, which significantly reduced the missed detection rate. Our model has more parameters (8.4 M) and a longer inference time (10.8 ms) due to its global attention design. However, it achieves the highest mAP (97.0%) among all models. Lightweight models like TPH-YOLO and MG-YOLO run faster (&lt; 7 ms) but perform worse, especially in detecting <em>A. flavus</em> small spores. Overall, our model balances accuracy and speed well for post-harvest early warning.</p></section></section><section id="Sec22"><h2 class="pmc_sec_title">Conclusion</h2>
<p id="Par49">This study introduced the YOLO-ASF-MobileViT network, which is an improved detection method that combines an ASF mechanism and the MobileViT attention mechanism. It effectively identifies morphologically diverse wheat fungal spores, with particular improvements in detecting <em>A. flavus spores</em>. The model achieved a mAP@0.5 of 97.0%, outperforming advanced detectors such as TPH-YOLO (95.6%) and MG-YOLO (95.5%). Although it introduces more parameters (8.4 M) and a longer inference time (10.8 ms), the inclusion of global attention significantly boosts detection robustness. The proposed model maintained high accuracy even under challenging conditions such as adhesion, occlusion, and noise. Overall, this method aims to provide early warning of potential fungal contamination in the post-harvest phase. It can assist in the risk assessment of seed-borne or storage-related fungal threats, thereby supporting timely intervention and quality control measures in wheat management.</p>
<p id="Par50">In the future research, we will focus on collecting more microscopic images to expand the model’s application scenarios. We also plan to incorporate additional cereal-associated fungal spore types into the training dataset to improve the model’s generalizability and reduce false positives in mixed-contamination environments.</p></section><section id="notes1"><h2 class="pmc_sec_title">Author contributions</h2>
<p>Zhizhou Ren made substantial contributions to the creation of new software; the acquisition, analysis, and interpretation of data; and drafted the work; Kun Liang made substantial contributions to the conception and design of the work; methodology and validation; and substantively revised the manuscript; Yingqi Zhang and Jinpeng Song made substantial contributions to the acquisition, analysis, and interpretation of data; Xiaoxiao Wu, Chi Zhang, Xiuming Mei, Yi Zhang and Xin Liu made substantial contributions to the acquisition and curation of data; All authors have read and approved the final manuscript and agree to be accountable for all aspects of the work.</p></section><section id="notes2"><h2 class="pmc_sec_title">Funding</h2>
<p>This work was supported by Postgraduate Research &amp; Practice Innovation Program of Jiangsu Province (Grant No. SJCX25_0286), Natural Science Foundation of Jiangsu Province (Grant No. BK20221518), National Undergraduate Innovation and Entrepreneurship Training Program (Grant No. 202410307100Z), Major Program of Jiangsu Provincial Administration for Market Regulation (Grant No. KJ2025016), and Jiangsu Agriculture Science and Technology Innovation Fund (Grant No. CX (23)1002).</p></section><section id="notes3"><h2 class="pmc_sec_title">Data availability</h2>
<p>No datasets were generated or analysed during the current study.</p></section><section id="notes4"><h2 class="pmc_sec_title">Declarations</h2>
<section id="FPar1"><h3 class="pmc_sec_title">Ethics approval and consent to participate</h3>
<p id="Par51">Not applicable.</p></section><section id="FPar2"><h3 class="pmc_sec_title">Consent for publication</h3>
<p id="Par52">Not applicable.</p></section><section id="FPar3"><h3 class="pmc_sec_title">Competing interests</h3>
<p id="Par53">The authors declare no competing interests.</p></section></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="fn1">
<p><strong>Publisher’s note</strong></p>
<p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p>
</div></div></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>El Houssni I, Zahidi A, Khedid K, Hassikou R. Nutrient and anti-nutrient composition of durum, soft and red wheat landraces: implications for nutrition and mineral bioavailability. J Agric Food Res. 2024;15:101078.</cite> [<a href="https://scholar.google.com/scholar_lookup?El%20Houssni%20I,%20Zahidi%20A,%20Khedid%20K,%20Hassikou%20R.%20Nutrient%20and%20anti-nutrient%20composition%20of%20durum,%20soft%20and%20red%20wheat%20landraces:%20implications%20for%20nutrition%20and%20mineral%20bioavailability.%20J%20Agric%20Food%20Res.%202024;15:101078." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>Kaur N, Singh B, Kaur A, Yadav MP, Singh N, Ahlawat AK, et al. Effect of growing conditions on proximate, mineral, amino acid, phenolic composition and antioxidant properties of wheatgrass from different wheat (Triticum aestivum L.) varieties. Food Chem. 2021;341:128201.
</cite> [<a href="https://doi.org/10.1016/j.foodchem.2020.128201" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33038775/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kaur%20N,%20Singh%20B,%20Kaur%20A,%20Yadav%20MP,%20Singh%20N,%20Ahlawat%20AK,%20et%20al.%20Effect%20of%20growing%20conditions%20on%20proximate,%20mineral,%20amino%20acid,%20phenolic%20composition%20and%20antioxidant%20properties%20of%20wheatgrass%20from%20different%20wheat%20(Triticum%20aestivum%20L.)%20varieties.%20Food%20Chem.%202021;341:128201." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>Kettlewell P, Byrne R, Jeffery S. Wheat area expansion into Northern higher latitudes and global food security. Agric Ecosyst Environ. 2023;351:108499.</cite> [<a href="https://scholar.google.com/scholar_lookup?Kettlewell%20P,%20Byrne%20R,%20Jeffery%20S.%20Wheat%20area%20expansion%20into%20Northern%20higher%20latitudes%20and%20global%20food%20security.%20Agric%20Ecosyst%20Environ.%202023;351:108499." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Chai Y, Senay S, Horvath D, Pardey P. Multi-peril pathogen risks to global wheat production: a probabilistic loss and investment assessment. Front Plant Sci. 2022;13:1034600.
</cite> [<a href="https://doi.org/10.3389/fpls.2022.1034600" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9659964/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36388575/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chai%20Y,%20Senay%20S,%20Horvath%20D,%20Pardey%20P.%20Multi-peril%20pathogen%20risks%20to%20global%20wheat%20production:%20a%20probabilistic%20loss%20and%20investment%20assessment.%20Front%20Plant%20Sci.%202022;13:1034600." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Mielniczuk E, Skwaryło-Bednarz B. Fusarium head blight, Mycotoxins and strategies for their reduction. Agronomy. 2020;10:509.</cite> [<a href="https://scholar.google.com/scholar_lookup?Mielniczuk%20E,%20Skwary%C5%82o-Bednarz%20B.%20Fusarium%20head%20blight,%20Mycotoxins%20and%20strategies%20for%20their%20reduction.%20Agronomy.%202020;10:509." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Almoujahed MB, Rangarajan AK, Whetton RL, Vincke D, Eylenbosch D, Vermeulen P, et al. Detection of fusarium head blight in wheat under field conditions using a hyperspectral camera and machine learning. Comput Electron Agric. 2022;203:107456.</cite> [<a href="https://scholar.google.com/scholar_lookup?Almoujahed%20MB,%20Rangarajan%20AK,%20Whetton%20RL,%20Vincke%20D,%20Eylenbosch%20D,%20Vermeulen%20P,%20et%20al.%20Detection%20of%20fusarium%20head%20blight%20in%20wheat%20under%20field%20conditions%20using%20a%20hyperspectral%20camera%20and%20machine%20learning.%20Comput%20Electron%20Agric.%202022;203:107456." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Scott P, Strange R, Korsten L, Gullino ML, editors. Plant diseases and food security in the 21st century. Cham: Springer International Publishing; 2021.</cite> [<a href="https://scholar.google.com/scholar_lookup?Scott%20P,%20Strange%20R,%20Korsten%20L,%20Gullino%20ML,%20editors.%20Plant%20diseases%20and%20food%20security%20in%20the%2021st%20century.%20Cham:%20Springer%20International%20Publishing;%202021." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>Abaya A, Xue A, Hsiang T. Selection and screening of fungal endophytes against wheat pathogens. Biol Control. 2021;154:104511.</cite> [<a href="https://scholar.google.com/scholar_lookup?Abaya%20A,%20Xue%20A,%20Hsiang%20T.%20Selection%20and%20screening%20of%20fungal%20endophytes%20against%20wheat%20pathogens.%20Biol%20Control.%202021;154:104511." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>Mathre DE. Bunts and Smuts revisited: has the air been cleared? Plant Health Prog. 2000;1:0622–02.</cite> [<a href="https://scholar.google.com/scholar_lookup?Mathre%20DE.%20Bunts%20and%20Smuts%20revisited:%20has%20the%20air%20been%20cleared?%20Plant%20Health%20Prog.%202000;1:0622%E2%80%9302." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>Sharma P, Chauhan R, Pande V, Basu T, Rajesh, Kumar A. Rapid sensing of Tilletia indica teliospore in wheat extract by a piezoelectric label-free immunosensor. Bioelectrochemistry. 2022;147:108175.
</cite> [<a href="https://doi.org/10.1016/j.bioelechem.2022.108175" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35749887/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sharma%20P,%20Chauhan%20R,%20Pande%20V,%20Basu%20T,%20Rajesh,%20Kumar%20A.%20Rapid%20sensing%20of%20Tilletia%20indica%20teliospore%20in%20wheat%20extract%20by%20a%20piezoelectric%20label-free%20immunosensor.%20Bioelectrochemistry.%202022;147:108175." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Kumar S, Singroha G, Singh GP, Sharma P. Karnal bunt of wheat: etiology, breeding and integrated management. Crop Prot. 2021;139:105376.</cite> [<a href="https://scholar.google.com/scholar_lookup?Kumar%20S,%20Singroha%20G,%20Singh%20GP,%20Sharma%20P.%20Karnal%20bunt%20of%20wheat:%20etiology,%20breeding%20and%20integrated%20management.%20Crop%20Prot.%202021;139:105376." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Noroozi R, Kobarfard F, Rezaei M, Ayatollahi SA, Paimard G, Eslamizad S, et al. Occurrence and exposure assessment of aflatoxin B1 in Iranian breads and wheat-based products considering effects of traditional processing. Food Control. 2022;138:108985.</cite> [<a href="https://scholar.google.com/scholar_lookup?Noroozi%20R,%20Kobarfard%20F,%20Rezaei%20M,%20Ayatollahi%20SA,%20Paimard%20G,%20Eslamizad%20S,%20et%20al.%20Occurrence%20and%20exposure%20assessment%20of%20aflatoxin%20B1%20in%20Iranian%20breads%20and%20wheat-based%20products%20considering%20effects%20of%20traditional%20processing.%20Food%20Control.%202022;138:108985." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>Hoffmann A, Funk R, Müller MEH. Blowin’ in the wind: wind dispersal ability of phytopathogenic fusarium in a wind tunnel experiment. Atmosphere. 2021;12:1653.</cite> [<a href="https://scholar.google.com/scholar_lookup?Hoffmann%20A,%20Funk%20R,%20M%C3%BCller%20MEH.%20Blowin%E2%80%99%20in%20the%20wind:%20wind%20dispersal%20ability%20of%20phytopathogenic%20fusarium%20in%20a%20wind%20tunnel%20experiment.%20Atmosphere.%202021;12:1653." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>Hernandez Nopsa JF, Daglish GJ, Hagstrum DW, Leslie JF, Phillips TW, Scoglio C, et al. Ecological networks in stored grain: key post-harvest nodes for emerging pests, pathogens, and Mycotoxins. Bioscience. 2015;65:985–1002.
</cite> [<a href="https://doi.org/10.1093/biosci/biv122" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4718207/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26955074/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hernandez%20Nopsa%20JF,%20Daglish%20GJ,%20Hagstrum%20DW,%20Leslie%20JF,%20Phillips%20TW,%20Scoglio%20C,%20et%20al.%20Ecological%20networks%20in%20stored%20grain:%20key%20post-harvest%20nodes%20for%20emerging%20pests,%20pathogens,%20and%20Mycotoxins.%20Bioscience.%202015;65:985%E2%80%931002." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>Gwinn KD, Leung MCK, Stephens AB, Punja ZK. Fungal and Mycotoxin contaminants in cannabis and hemp flowers: implications for consumer health and directions for further research. Front Microbiol. 2023;14:1278189.
</cite> [<a href="https://doi.org/10.3389/fmicb.2023.1278189" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10620813/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37928692/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gwinn%20KD,%20Leung%20MCK,%20Stephens%20AB,%20Punja%20ZK.%20Fungal%20and%20Mycotoxin%20contaminants%20in%20cannabis%20and%20hemp%20flowers:%20implications%20for%20consumer%20health%20and%20directions%20for%20further%20research.%20Front%20Microbiol.%202023;14:1278189." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Anwar H, Khan SU, Ghaffar MM, Fayyaz M, Khan MJ, Weis C, et al. The NWRD dataset: an open-source annotated segmentation dataset of diseased wheat crop. Sensors. 2023;23:6942.
</cite> [<a href="https://doi.org/10.3390/s23156942" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10422341/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37571726/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Anwar%20H,%20Khan%20SU,%20Ghaffar%20MM,%20Fayyaz%20M,%20Khan%20MJ,%20Weis%20C,%20et%20al.%20The%20NWRD%20dataset:%20an%20open-source%20annotated%20segmentation%20dataset%20of%20diseased%20wheat%20crop.%20Sensors.%202023;23:6942." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>Paul N, Sunil GC, Horvath D, Sun X. Deep learning for plant stress detection: a comprehensive review of technologies, challenges, and future directions. Comput Electron Agric. 2025;229:109734.</cite> [<a href="https://scholar.google.com/scholar_lookup?Paul%20N,%20Sunil%20GC,%20Horvath%20D,%20Sun%20X.%20Deep%20learning%20for%20plant%20stress%20detection:%20a%20comprehensive%20review%20of%20technologies,%20challenges,%20and%20future%20directions.%20Comput%20Electron%20Agric.%202025;229:109734." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Catal Reis H, Turk V. Integrated deep learning and ensemble learning model for deep feature-based wheat disease detection. Microchem J. 2024;197:109790.</cite> [<a href="https://scholar.google.com/scholar_lookup?Catal%20Reis%20H,%20Turk%20V.%20Integrated%20deep%20learning%20and%20ensemble%20learning%20model%20for%20deep%20feature-based%20wheat%20disease%20detection.%20Microchem%20J.%202024;197:109790." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Zhu Y, Yao Y, Xi J, Tang C, Wu L. Modelling the effect of pH and H2S on the germination of F. graminearum spores under different temperature conditions. LWT. 2023;177:114530.</cite> [<a href="https://scholar.google.com/scholar_lookup?Zhu%20Y,%20Yao%20Y,%20Xi%20J,%20Tang%20C,%20Wu%20L.%20Modelling%20the%20effect%20of%20pH%20and%20H2S%20on%20the%20germination%20of%20F.%20graminearum%20spores%20under%20different%20temperature%20conditions.%20LWT.%202023;177:114530." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>Wakeham A, Kennedy R, McCartney A. The collection and retention of a range of common airborne spore types trapped directly into microtiter wells for enzyme-linked immunosorbent analysis. J Aerosol Sci. 2004;35:835–50.</cite> [<a href="https://scholar.google.com/scholar_lookup?Wakeham%20A,%20Kennedy%20R,%20McCartney%20A.%20The%20collection%20and%20retention%20of%20a%20range%20of%20common%20airborne%20spore%20types%20trapped%20directly%20into%20microtiter%20wells%20for%20enzyme-linked%20immunosorbent%20analysis.%20J%20Aerosol%20Sci.%202004;35:835%E2%80%9350." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>Pilo P, Tiley AMM, Lawless C, Karki SJ, Burke J, Feechan A. A rapid fungal DNA extraction method suitable for PCR screening fungal mutants, infected plant tissue and spore trap samples. Physiol Mol Plant Pathol. 2022;117:101758.</cite> [<a href="https://scholar.google.com/scholar_lookup?Pilo%20P,%20Tiley%20AMM,%20Lawless%20C,%20Karki%20SJ,%20Burke%20J,%20Feechan%20A.%20A%20rapid%20fungal%20DNA%20extraction%20method%20suitable%20for%20PCR%20screening%20fungal%20mutants,%20infected%20plant%20tissue%20and%20spore%20trap%20samples.%20Physiol%20Mol%20Plant%20Pathol.%202022;117:101758." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>Harpaz D, Duanis-Assaf D, Alkan N, Eltzov E. Detection of post-harvest pathogenic fungi by RNA markers in a high-throughput platform on Streptavidin plate. Postharvest Biol Technol. 2022;183:111728.</cite> [<a href="https://scholar.google.com/scholar_lookup?Harpaz%20D,%20Duanis-Assaf%20D,%20Alkan%20N,%20Eltzov%20E.%20Detection%20of%20post-harvest%20pathogenic%20fungi%20by%20RNA%20markers%20in%20a%20high-throughput%20platform%20on%20Streptavidin%20plate.%20Postharvest%20Biol%20Technol.%202022;183:111728." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>Qamar S, Öberg R, Malyshev D, Andersson M. A hybrid CNN–random forest algorithm for bacterial spore segmentation and classification in TEM images. Sci Rep. 2023;13:18758.
</cite> [<a href="https://doi.org/10.1038/s41598-023-44212-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10618482/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37907463/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Qamar%20S,%20%C3%96berg%20R,%20Malyshev%20D,%20Andersson%20M.%20A%20hybrid%20CNN%E2%80%93random%20forest%20algorithm%20for%20bacterial%20spore%20segmentation%20and%20classification%20in%20TEM%20images.%20Sci%20Rep.%202023;13:18758." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>Crespo-Michel A, Alonso-Arévalo MA, Hernández-Martínez R. Developing a microscope image dataset for fungal spore classification in grapevine using deep learning. J Agric Food Res. 2023;14:100805.</cite> [<a href="https://scholar.google.com/scholar_lookup?Crespo-Michel%20A,%20Alonso-Ar%C3%A9valo%20MA,%20Hern%C3%A1ndez-Mart%C3%ADnez%20R.%20Developing%20a%20microscope%20image%20dataset%20for%20fungal%20spore%20classification%20in%20grapevine%20using%20deep%20learning.%20J%20Agric%20Food%20Res.%202023;14:100805." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>Zhou H, Lai Q, Huang Q, Cai D, Huang D, Wu B. Automatic detection of rice blast fungus spores by deep learning-based object detection: models, benchmarks and quantitative analysis. Agriculture. 2024;14:290.</cite> [<a href="https://scholar.google.com/scholar_lookup?Zhou%20H,%20Lai%20Q,%20Huang%20Q,%20Cai%20D,%20Huang%20D,%20Wu%20B.%20Automatic%20detection%20of%20rice%20blast%20fungus%20spores%20by%20deep%20learning-based%20object%20detection:%20models,%20benchmarks%20and%20quantitative%20analysis.%20Agriculture.%202024;14:290." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>Zhang D-Y, Zhang W, Cheng T, Zhou X-G, Yan Z, Wu Y, et al. Detection of wheat scab fungus spores utilizing the YOLOv5-ECA-ASFF network structure. Comput Electron Agric. 2023;210:107953.</cite> [<a href="https://scholar.google.com/scholar_lookup?Zhang%20D-Y,%20Zhang%20W,%20Cheng%20T,%20Zhou%20X-G,%20Yan%20Z,%20Wu%20Y,%20et%20al.%20Detection%20of%20wheat%20scab%20fungus%20spores%20utilizing%20the%20YOLOv5-ECA-ASFF%20network%20structure.%20Comput%20Electron%20Agric.%202023;210:107953." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Li K, Qiao C, Zhu X, Song Y, Zhang L, Gao W, et al. Lightweight fungal spore detection based on improved YOLOv5 in natural scenes. Int J Mach Learn Cybern. 2023;14:02026–x.</cite> [<a href="https://scholar.google.com/scholar_lookup?Li%20K,%20Qiao%20C,%20Zhu%20X,%20Song%20Y,%20Zhang%20L,%20Gao%20W,%20et%20al.%20Lightweight%20fungal%20spore%20detection%20based%20on%20improved%20YOLOv5%20in%20natural%20scenes.%20Int%20J%20Mach%20Learn%20Cybern.%202023;14:02026%E2%80%93x." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Wei X, Liu Y, Song Q, Zou J, Wen Z, Li J, et al. Microscopic hyperspectral imaging and an improved detection model based on detection of mycogone perniciosa chlamydospore in soil. Eur J Agron. 2024;152:127007.</cite> [<a href="https://scholar.google.com/scholar_lookup?Wei%20X,%20Liu%20Y,%20Song%20Q,%20Zou%20J,%20Wen%20Z,%20Li%20J,%20et%20al.%20Microscopic%20hyperspectral%20imaging%20and%20an%20improved%20detection%20model%20based%20on%20detection%20of%20mycogone%20perniciosa%20chlamydospore%20in%20soil.%20Eur%20J%20Agron.%202024;152:127007." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>Giovenzana V, Beghi R, Buratti S, Civelli R, Guidetti R. Monitoring of fresh-cut Valerianella locusta shelf life by electronic nose and VIS–NIR spectroscopy. Talanta. 2014;120:368–75.
</cite> [<a href="https://doi.org/10.1016/j.talanta.2013.12.014" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24468384/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Giovenzana%20V,%20Beghi%20R,%20Buratti%20S,%20Civelli%20R,%20Guidetti%20R.%20Monitoring%20of%20fresh-cut%20Valerianella%20locusta%20shelf%20life%20by%20electronic%20nose%20and%20VIS%E2%80%93NIR%20spectroscopy.%20Talanta.%202014;120:368%E2%80%9375." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR30">
<span class="label">30.</span><cite>Cao S, Wang T, Li T, Mao Z. UAV small-target detection algorithm based on an improved YOLOv5s model. J Vis Commun Image Represent. 2023;97:103936.</cite> [<a href="https://scholar.google.com/scholar_lookup?Cao%20S,%20Wang%20T,%20Li%20T,%20Mao%20Z.%20UAV%20small-target%20detection%20algorithm%20based%20on%20an%20improved%20YOLOv5s%20model.%20J%20Vis%20Commun%20Image%20Represent.%202023;97:103936." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>Shi Y, Wang B, Yin C, Li Z, Yu Y. Performance improvement: a lightweight gas information classification method combined with an electronic nose system. Sens Actuators B Chem. 2023;396:134551.</cite> [<a href="https://scholar.google.com/scholar_lookup?Shi%20Y,%20Wang%20B,%20Yin%20C,%20Li%20Z,%20Yu%20Y.%20Performance%20improvement:%20a%20lightweight%20gas%20information%20classification%20method%20combined%20with%20an%20electronic%20nose%20system.%20Sens%20Actuators%20B%20Chem.%202023;396:134551." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>Weaver MA, Park LC, Brewer MJ, Grodowitz MJ, Abbas HK. Detection, quantification, and characterization of airborne Aspergillus flavus within the corn canopy. Mycotoxin Res. 2025;41:267–76.
</cite> [<a href="https://doi.org/10.1007/s12550-025-00581-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11757857/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39808409/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Weaver%20MA,%20Park%20LC,%20Brewer%20MJ,%20Grodowitz%20MJ,%20Abbas%20HK.%20Detection,%20quantification,%20and%20characterization%20of%20airborne%20Aspergillus%20flavus%20within%20the%20corn%20canopy.%20Mycotoxin%20Res.%202025;41:267%E2%80%9376." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Ma P, Li C, Rahaman MM, Yao Y, Zhang J, Zou S, et al. A state-of-the-art survey of object detection techniques in microorganism image analysis: from classical methods to deep learning approaches. Artif Intell Rev. 2023;56:1627–98.
</cite> [<a href="https://doi.org/10.1007/s10462-022-10209-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9170564/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35693000/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ma%20P,%20Li%20C,%20Rahaman%20MM,%20Yao%20Y,%20Zhang%20J,%20Zou%20S,%20et%20al.%20A%20state-of-the-art%20survey%20of%20object%20detection%20techniques%20in%20microorganism%20image%20analysis:%20from%20classical%20methods%20to%20deep%20learning%20approaches.%20Artif%20Intell%20Rev.%202023;56:1627%E2%80%9398." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>Nguyen HDT, Sultana T, Kesanakurti P, Hambleton S. Genome sequencing and comparison of five Tilletia species to identify candidate genes for the detection of regulated species infecting wheat. IMA Fungus. 2019;10:0011–9.</cite> [<a href="https://doi.org/10.1186/s43008-019-0011-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7184893/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32355611/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Nguyen%20HDT,%20Sultana%20T,%20Kesanakurti%20P,%20Hambleton%20S.%20Genome%20sequencing%20and%20comparison%20of%20five%20Tilletia%20species%20to%20identify%20candidate%20genes%20for%20the%20detection%20of%20regulated%20species%20infecting%20wheat.%20IMA%20Fungus.%202019;10:0011%E2%80%939." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>Mansour MB, Goh YK, Vujanovic V. Rapid macroconidia production in fusarium graminearum 3- and 15-acetyldeoxynivalenol chemotypes using sucrose-water medium. Ann Microbiol. 2012;62:965–71.</cite> [<a href="https://scholar.google.com/scholar_lookup?Mansour%20MB,%20Goh%20YK,%20Vujanovic%20V.%20Rapid%20macroconidia%20production%20in%20fusarium%20graminearum%203-%20and%2015-acetyldeoxynivalenol%20chemotypes%20using%20sucrose-water%20medium.%20Ann%20Microbiol.%202012;62:965%E2%80%9371." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR36">
<span class="label">36.</span><cite>Son Y-E, Park H-S. SscA is required for fungal development, aflatoxin production, and pathogenicity in Aspergillus flavus. Int J Food Microbiol. 2024;413:110607.
</cite> [<a href="https://doi.org/10.1016/j.ijfoodmicro.2024.110607" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38308877/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Son%20Y-E,%20Park%20H-S.%20SscA%20is%20required%20for%20fungal%20development,%20aflatoxin%20production,%20and%20pathogenicity%20in%20Aspergillus%20flavus.%20Int%20J%20Food%20Microbiol.%202024;413:110607." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR37">
<span class="label">37.</span><cite>Cheng T, Zhang D, Gu C, Zhou X-G, Qiao H, Guo W, et al. YOLO-CG-HS: a lightweight spore detection method for wheat airborne fungal pathogens. Comput Electron Agric. 2024;227:109544.</cite> [<a href="https://scholar.google.com/scholar_lookup?Cheng%20T,%20Zhang%20D,%20Gu%20C,%20Zhou%20X-G,%20Qiao%20H,%20Guo%20W,%20et%20al.%20YOLO-CG-HS:%20a%20lightweight%20spore%20detection%20method%20for%20wheat%20airborne%20fungal%20pathogens.%20Comput%20Electron%20Agric.%202024;227:109544." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR38">
<span class="label">38.</span><cite>Li M, Liu J, Yao T, Gao Z, Gong J. Deep-learning based in-situ micrograph analysis of high-density crystallization slurry using image and data enhancement strategy. Powder Technol. 2024;437:119582.</cite> [<a href="https://scholar.google.com/scholar_lookup?Li%20M,%20Liu%20J,%20Yao%20T,%20Gao%20Z,%20Gong%20J.%20Deep-learning%20based%20in-situ%20micrograph%20analysis%20of%20high-density%20crystallization%20slurry%20using%20image%20and%20data%20enhancement%20strategy.%20Powder%20Technol.%202024;437:119582." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR39">
<span class="label">39.</span><cite>Kang M, Ting C-M, Ting FF, Phan R-CW, et al. ASF-YOLO: a novel YOLO model with attentional scale sequence fusion for cell instance segmentation. Image Vis Comput. 2024;147:105057.</cite> [<a href="https://scholar.google.com/scholar_lookup?Kang%20M,%20Ting%20C-M,%20Ting%20FF,%20Phan%20R-CW,%20et%20al.%20ASF-YOLO:%20a%20novel%20YOLO%20model%20with%20attentional%20scale%20sequence%20fusion%20for%20cell%20instance%20segmentation.%20Image%20Vis%20Comput.%202024;147:105057." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR40">
<span class="label">40.</span><cite>Dündar N, Keçeli AS, Kaya A, Sever H. A shallow 3D convolutional neural network for violence detection in videos. Egypt Inf J. 2024;26:100455.</cite> [<a href="https://scholar.google.com/scholar_lookup?D%C3%BCndar%20N,%20Ke%C3%A7eli%20AS,%20Kaya%20A,%20Sever%20H.%20A%20shallow%203D%20convolutional%20neural%20network%20for%20violence%20detection%20in%20videos.%20Egypt%20Inf%20J.%202024;26:100455." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR41">
<span class="label">41.</span><cite>Dosovitskiy A, Beyer L, Kolesnikov A, Weissenborn D, Zhai X, Unterthiner T et al. An image is worth 16×16 words: transformers for image recognition at scale. ArXiv [Preprint]. 2020; arXiv:2010.11929.</cite>
</li>
<li id="CR42">
<span class="label">42.</span><cite>Mehta S, Rastegari M. MobileViT: light-weight, general-purpose, and mobile-friendly vision transformer. ArXiv [Preprint]. 2022. arXiv:2110.02178.</cite>
</li>
<li id="CR43">
<span class="label">43.</span><cite>Hu J, Shen L, Sun G. Squeeze-and-excitation networks. In: Proceedings of the IEEE Conference on Computer Vision and Pattern Recognition; 2018. pp. 7132–7141.</cite>
</li>
<li id="CR44">
<span class="label">44.</span><cite>Zhang QL, Yang YB. SA-Net: shuffle attention for deep convolutional neural networks. In: ICASSP 2021 – IEEE International Conference on Acoustics, Speech and Signal Processing; 2021. pp. 2235–2239.</cite>
</li>
<li id="CR45">
<span class="label">45.</span><cite>Wang Q, Wu B, Zhu P, Li P, Zuo W, Hu Q et al. ECA-Net: efficient channel attention for deep convolutional neural networks. In: Proceedings of the IEEE/CVF Conference on Computer Vision and Pattern Recognition; 2020. pp. 11534–11542.</cite>
</li>
<li id="CR46">
<span class="label">46.</span><cite>Woo S, Park J, Lee J–Y, Kweon ISCBAM. Convolutional Block Attention Module. In: Proceedings of the European Conference on Computer Vision (ECCV); 2018. pp. 3–19.</cite>
</li>
<li id="CR47">
<span class="label">47.</span><cite>Liu W, Anguelov D, Erhan D, Szegedy C, Reed S, Fu C-Y et al. SSD: single shot multibox detector. In: Computer Vision – ECCV 2016: 14th European Conference, Amsterdam, The Netherlands, October 11–14, 2016, Proceedings, Part I; Springer International Publishing; 2016. pp. 21–37.</cite>
</li>
<li id="CR48">
<span class="label">48.</span><cite>Redmon J, Divvala S, Girshick R, Farhadi A, Kokkinos I, Maziarka B et al. You only look once: unified, real-time object detection. In: Proceedings of the IEEE Conference on Computer Vision and Pattern Recognition; 2016. pp. 779–788.</cite>
</li>
<li id="CR49">
<span class="label">49.</span><cite>Zhou X, Wang D, Krähenbühl P. Objects as points. ArXiv. 2019. ArXiv:1904.07850. Preprint.</cite>
</li>
<li id="CR50">
<span class="label">50.</span><cite>Ren S, He K, Girshick R, Sun J, Zhang X, Ioffe S, et al. Faster R-CNN: towards real-time object detection with region proposal networks. IEEE Trans Pattern Anal Mach Intell. 2016;39(6):1137–49.
</cite> [<a href="https://doi.org/10.1109/TPAMI.2016.2577031" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27295650/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ren%20S,%20He%20K,%20Girshick%20R,%20Sun%20J,%20Zhang%20X,%20Ioffe%20S,%20et%20al.%20Faster%20R-CNN:%20towards%20real-time%20object%20detection%20with%20region%20proposal%20networks.%20IEEE%20Trans%20Pattern%20Anal%20Mach%20Intell.%202016;39(6):1137%E2%80%9349." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR51">
<span class="label">51.</span><cite>Zhu X, Lyu S, Wang X, Zhao Q. TPH-YOLOv5: improved YOLOv5 based on transformer prediction head for object detection on drone-captured scenarios. In: 2021 IEEE/CVF International Conference on Computer Vision Workshops; 2021. pp. 2778–88.</cite>
</li>
<li id="CR52">
<span class="label">52.</span><cite>Li K, Zhu X, Qiao C, Zhang L, Gao W, Wang Y. The Gray mold spore detection of cucumber based on microscopic image and deep learning. Plant Phenomics. 2023;5:0011.
</cite> [<a href="https://doi.org/10.34133/plantphenomics.0011" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10013786/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36930758/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li%20K,%20Zhu%20X,%20Qiao%20C,%20Zhang%20L,%20Gao%20W,%20Wang%20Y.%20The%20Gray%20mold%20spore%20detection%20of%20cucumber%20based%20on%20microscopic%20image%20and%20deep%20learning.%20Plant%20Phenomics.%202023;5:0011." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR53">
<span class="label">53.</span><cite>Sunkara R, Luo T. No more strided convolutions or pooling: a new CNN building block for low-resolution images and small objects. In: Amini M-R, Canu S, Fischer A, Guns T, Kralj Novak P, Tsoumakas G, editors. Machine Learning and Knowledge Discovery in Databases; 2023. pp. 443–59.</cite>
</li>
<li id="CR54">
<span class="label">54.</span><cite>Selvaraju RR, Cogswell M, Das A, Vedantam R, Parikh D, Batra D. Grad-CAM: visual explanations from deep networks via gradient-based localization. Int J Comput Vis. 2017;128:336–59.</cite> [<a href="https://scholar.google.com/scholar_lookup?Selvaraju%20RR,%20Cogswell%20M,%20Das%20A,%20Vedantam%20R,%20Parikh%20D,%20Batra%20D.%20Grad-CAM:%20visual%20explanations%20from%20deep%20networks%20via%20gradient-based%20localization.%20Int%20J%20Comput%20Vis.%202017;128:336%E2%80%9359." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR55">
<span class="label">55.</span><cite>Pilo P, Lawless C, Tiley AMM, Karki SJ, Burke JI, Feechan A. Comparison of microscopic and metagenomic approaches to identify cereal pathogens and track fungal spore release in the field. Front Plant Sci. 2022;13:1039090.
</cite> [<a href="https://doi.org/10.3389/fpls.2022.1039090" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9630935/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36340419/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Pilo%20P,%20Lawless%20C,%20Tiley%20AMM,%20Karki%20SJ,%20Burke%20JI,%20Feechan%20A.%20Comparison%20of%20microscopic%20and%20metagenomic%20approaches%20to%20identify%20cereal%20pathogens%20and%20track%20fungal%20spore%20release%20in%20the%20field.%20Front%20Plant%20Sci.%202022;13:1039090." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>No datasets were generated or analysed during the current study.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Plant Methods are provided here courtesy of <strong>BMC</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1186/s13007-025-01436-y"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/13007_2025_Article_1436.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (3.1 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12372353/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12372353/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12372353%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12372353/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12372353/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12372353/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40847309/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12372353/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40847309/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12372353/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12372353/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="tFqZalxMRXPgz1XGycbJRnPwl476EGLy7RorVTsAaP4C4MGZCvWm5Xe2vXY3zsEa">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
