
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            TrimNN: characterizing cellular community motifs for studying multicellular topological organization in complex tissues - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="AE533C948AF36233053C94001F7B3EDC.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="ncomms">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365081/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Nature Communications">
<meta name="citation_title" content="TrimNN: characterizing cellular community motifs for studying multicellular topological organization in complex tissues">
<meta name="citation_author" content="Yang Yu">
<meta name="citation_author_institution" content="Department of Electrical Engineering and Computer Science, Christopher S. Bond Life Sciences Center, University of Missouri, Columbia, MO USA">
<meta name="citation_author" content="Shuang Wang">
<meta name="citation_author_institution" content="Department of Computer Science, Luddy School of Informatics, Computing, and Engineering, Indiana University Bloomington, Bloomington, IN USA">
<meta name="citation_author" content="Jinpu Li">
<meta name="citation_author_institution" content="Institute for Data Science and Informatics, University of Missouri, Columbia, MO USA">
<meta name="citation_author" content="Meichen Yu">
<meta name="citation_author_institution" content="Indiana Alzheimer’s Disease Research Center, Center for Neuroimaging, Department of Radiology and Imaging Sciences, Stark Neurosciences Research Institute, Indiana University School of Medicine, Indianapolis, IN USA">
<meta name="citation_author" content="Kyle McCrocklin">
<meta name="citation_author_institution" content="Department of Biomedical Engineering and Informatics, Luddy School of Informatics, Computing, and Engineering, Indiana University Indianapolis, Indianapolis, IN USA">
<meta name="citation_author" content="Jing-Qiong Kang">
<meta name="citation_author_institution" content="Department of Neurology &amp; Pharmacology, Vanderbilt University Medical Center, Vanderbilt Kennedy Center of Human Development, Vanderbilt University, Nashville, TN USA">
<meta name="citation_author" content="Anjun Ma">
<meta name="citation_author_institution" content="Department of Biomedical Informatics, College of Medicine, The Ohio State University, Columbus, OH USA">
<meta name="citation_author_institution" content="Pelotonia Institute for Immuno-Oncology, The James Comprehensive Cancer Center, The Ohio State University, Columbus, OH USA">
<meta name="citation_author" content="Qin Ma">
<meta name="citation_author_institution" content="Department of Biomedical Informatics, College of Medicine, The Ohio State University, Columbus, OH USA">
<meta name="citation_author_institution" content="Pelotonia Institute for Immuno-Oncology, The James Comprehensive Cancer Center, The Ohio State University, Columbus, OH USA">
<meta name="citation_author" content="Dong Xu">
<meta name="citation_author_institution" content="Department of Electrical Engineering and Computer Science, Christopher S. Bond Life Sciences Center, University of Missouri, Columbia, MO USA">
<meta name="citation_author_institution" content="Institute for Data Science and Informatics, University of Missouri, Columbia, MO USA">
<meta name="citation_author" content="Juexin Wang">
<meta name="citation_author_institution" content="Department of Biomedical Engineering and Informatics, Luddy School of Informatics, Computing, and Engineering, Indiana University Indianapolis, Indianapolis, IN USA">
<meta name="citation_publication_date" content="2025 Aug 19">
<meta name="citation_volume" content="16">
<meta name="citation_firstpage" content="7737">
<meta name="citation_doi" content="10.1038/s41467-025-63141-7">
<meta name="citation_pmid" content="40830137">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365081/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365081/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365081/pdf/41467_2025_Article_63141.pdf">
<meta name="description" content="The spatial organization of cells plays a pivotal role in shaping tissue functions and phenotypes in various biological systems and diseased microenvironments. However, the topological principles governing interactions among cell types within ...">
<meta name="og:title" content="TrimNN: characterizing cellular community motifs for studying multicellular topological organization in complex tissues">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="The spatial organization of cells plays a pivotal role in shaping tissue functions and phenotypes in various biological systems and diseased microenvironments. However, the topological principles governing interactions among cell types within ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365081/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12365081">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1038/s41467-025-63141-7"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/41467_2025_Article_63141.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12365081%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12365081/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12365081/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365081/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-ncomms.gif" alt="Nature Communications logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Nature Communications" title="Link to Nature Communications" shape="default" href="http://www.nature.com/ncomms/index.html" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Nat Commun</button></div>. 2025 Aug 19;16:7737. doi: <a href="https://doi.org/10.1038/s41467-025-63141-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1038/s41467-025-63141-7</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Nat%20Commun%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nat%20Commun%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Nat%20Commun%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Nat%20Commun%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>TrimNN: characterizing cellular community motifs for studying multicellular topological organization in complex tissues</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yu%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Yang Yu</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Yang Yu</span></h3>
<div class="p">
<sup>1</sup>Department of Electrical Engineering and Computer Science, Christopher S. Bond Life Sciences Center, University of Missouri, Columbia, MO USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yu%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yang Yu</span></a>
</div>
</div>
<sup>1,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Shuang Wang</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Shuang Wang</span></h3>
<div class="p">
<sup>2</sup>Department of Computer Science, Luddy School of Informatics, Computing, and Engineering, Indiana University Bloomington, Bloomington, IN USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Shuang Wang</span></a>
</div>
</div>
<sup>2,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Li%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Jinpu Li</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Jinpu Li</span></h3>
<div class="p">
<sup>3</sup>Institute for Data Science and Informatics, University of Missouri, Columbia, MO USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Li%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Jinpu Li</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yu%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Meichen Yu</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Meichen Yu</span></h3>
<div class="p">
<sup>4</sup>Indiana Alzheimer’s Disease Research Center, Center for Neuroimaging, Department of Radiology and Imaging Sciences, Stark Neurosciences Research Institute, Indiana University School of Medicine, Indianapolis, IN USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yu%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Meichen Yu</span></a>
</div>
</div>
<sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22McCrocklin%20K%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Kyle McCrocklin</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Kyle McCrocklin</span></h3>
<div class="p">
<sup>5</sup>Department of Biomedical Engineering and Informatics, Luddy School of Informatics, Computing, and Engineering, Indiana University Indianapolis, Indianapolis, IN USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22McCrocklin%20K%22%5BAuthor%5D" class="usa-link"><span class="name western">Kyle McCrocklin</span></a>
</div>
</div>
<sup>5</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kang%20JQ%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Jing-Qiong Kang</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Jing-Qiong Kang</span></h3>
<div class="p">
<sup>6</sup>Department of Neurology &amp; Pharmacology, Vanderbilt University Medical Center, Vanderbilt Kennedy Center of Human Development, Vanderbilt University, Nashville, TN USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kang%20JQ%22%5BAuthor%5D" class="usa-link"><span class="name western">Jing-Qiong Kang</span></a>
</div>
</div>
<sup>6</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ma%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Anjun Ma</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Anjun Ma</span></h3>
<div class="p">
<sup>7</sup>Department of Biomedical Informatics, College of Medicine, The Ohio State University, Columbus, OH USA </div>
<div class="p">
<sup>8</sup>Pelotonia Institute for Immuno-Oncology, The James Comprehensive Cancer Center, The Ohio State University, Columbus, OH USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ma%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Anjun Ma</span></a>
</div>
</div>
<sup>7,</sup><sup>8</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ma%20Q%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Qin Ma</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Qin Ma</span></h3>
<div class="p">
<sup>7</sup>Department of Biomedical Informatics, College of Medicine, The Ohio State University, Columbus, OH USA </div>
<div class="p">
<sup>8</sup>Pelotonia Institute for Immuno-Oncology, The James Comprehensive Cancer Center, The Ohio State University, Columbus, OH USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ma%20Q%22%5BAuthor%5D" class="usa-link"><span class="name western">Qin Ma</span></a>
</div>
</div>
<sup>7,</sup><sup>8,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Xu%20D%22%5BAuthor%5D" class="usa-link" aria-describedby="id9"><span class="name western">Dong Xu</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">Dong Xu</span></h3>
<div class="p">
<sup>1</sup>Department of Electrical Engineering and Computer Science, Christopher S. Bond Life Sciences Center, University of Missouri, Columbia, MO USA </div>
<div class="p">
<sup>3</sup>Institute for Data Science and Informatics, University of Missouri, Columbia, MO USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Xu%20D%22%5BAuthor%5D" class="usa-link"><span class="name western">Dong Xu</span></a>
</div>
</div>
<sup>1,</sup><sup>3,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id10"><span class="name western">Juexin Wang</span></a><div hidden="hidden" id="id10">
<h3><span class="name western">Juexin Wang</span></h3>
<div class="p">
<sup>5</sup>Department of Biomedical Engineering and Informatics, Luddy School of Informatics, Computing, and Engineering, Indiana University Indianapolis, Indianapolis, IN USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Juexin Wang</span></a>
</div>
</div>
<sup>5,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>Department of Electrical Engineering and Computer Science, Christopher S. Bond Life Sciences Center, University of Missouri, Columbia, MO USA </div>
<div id="Aff2">
<sup>2</sup>Department of Computer Science, Luddy School of Informatics, Computing, and Engineering, Indiana University Bloomington, Bloomington, IN USA </div>
<div id="Aff3">
<sup>3</sup>Institute for Data Science and Informatics, University of Missouri, Columbia, MO USA </div>
<div id="Aff4">
<sup>4</sup>Indiana Alzheimer’s Disease Research Center, Center for Neuroimaging, Department of Radiology and Imaging Sciences, Stark Neurosciences Research Institute, Indiana University School of Medicine, Indianapolis, IN USA </div>
<div id="Aff5">
<sup>5</sup>Department of Biomedical Engineering and Informatics, Luddy School of Informatics, Computing, and Engineering, Indiana University Indianapolis, Indianapolis, IN USA </div>
<div id="Aff6">
<sup>6</sup>Department of Neurology &amp; Pharmacology, Vanderbilt University Medical Center, Vanderbilt Kennedy Center of Human Development, Vanderbilt University, Nashville, TN USA </div>
<div id="Aff7">
<sup>7</sup>Department of Biomedical Informatics, College of Medicine, The Ohio State University, Columbus, OH USA </div>
<div id="Aff8">
<sup>8</sup>Pelotonia Institute for Immuno-Oncology, The James Comprehensive Cancer Center, The Ohio State University, Columbus, OH USA </div>
<div class="author-notes p">
<div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div>
<div class="fn" id="_eqcntrb93pmc__">
<sup>#</sup><p class="display-inline">Contributed equally.</p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 Jan 2; Accepted 2025 Aug 4; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12365081  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40830137/" class="usa-link">40830137</a>
</div>
<div class="ra xbox p" role="complementary" aria-label="Related or updated information about this article"><div>
<strong>Previous version available:</strong> This article is based on a previously available preprint posted on Research Square on January 17, 2025: "<a href="/articles/PMC11774463/" class="usa-link">TrimNN: Characterizing cellular community motifs for studying multicellular topological organization in complex tissues</a>".</div></div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<p id="Par1">The spatial organization of cells plays a pivotal role in shaping tissue functions and phenotypes in various biological systems and diseased microenvironments. However, the topological principles governing interactions among cell types within spatial patterns remain poorly understood. Here, we present the <strong>tri</strong>angulation cellular community <strong>m</strong>otif <strong>n</strong>eural <strong>n</strong>etwork (TrimNN), a graph-based deep learning framework designed to identify conserved spatial cell organization patterns, termed cellular community (CC) motifs, from spatial transcriptomics and proteomics data. TrimNN employs a semi–divide-and-conquer approach to efficiently detect overrepresented topological motifs of varying sizes in a triangulated space. By uncovering CC motifs, TrimNN reveals key associations between spatially distributed cell-type patterns and diverse phenotypes. These insights provide a foundation for understanding biological and disease mechanisms and offer potential biomarkers for diagnosis and therapeutic interventions.</p>
<section id="kwd-group1" class="kwd-group"><p><strong>Subject terms:</strong> Computational models, Machine learning, Data mining, Network topology</p></section></section><section class="abstract" id="Abs2"><hr class="headless">
<p id="Par2">Cellular spatial organisation is crucial for shaping tissue functions and phenotypes. Here, authors present TrimNN, a graph-based deep learning framework to identify conserved cellular community motifs, revealing links between spatially distributed cell-type patterns and diverse phenotypes.</p></section><section id="Sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p id="Par3">Various cells work together within spatial arrangements in tissue to support organ homeostasis and function<sup><a href="#CR1" class="usa-link" aria-describedby="CR1">1</a></sup>. Deciphering the multicellular organization is key to understanding the relationship between spatial structure and tissue biological and pathological functions<sup><a href="#CR2" class="usa-link" aria-describedby="CR2">2</a></sup>. Emerging spatial omics approaches, including spatially resolved transcriptomics<sup><a href="#CR3" class="usa-link" aria-describedby="CR3">3</a></sup> and spatial proteomics<sup><a href="#CR4" class="usa-link" aria-describedby="CR4">4</a></sup>, enable investigation of the mechanisms governing the spatial organization of different cell types in a specific tissue. Within a region of interest (ROI) in spatial omics, cellular neighborhoods (CNs) define local cell type enrichment patterns in cellular communities (CCs), and decoding function-related conservative spatial features in CNs is one of the primary spatial omics data analysis tasks<sup><a href="#CR4" class="usa-link" aria-describedby="CR4">4</a></sup>.</p>
<p id="Par4">Most existing data analysis approaches adopt the top-down strategy to describe the cell organizations. This strategy mainly relies on clustering strategies to identify the cell type compositions as common patterns. Deep learning approaches, including SPACE-GM<sup><a href="#CR5" class="usa-link" aria-describedby="CR5">5</a></sup>, CytoCommunity<sup><a href="#CR6" class="usa-link" aria-describedby="CR6">6</a></sup>, CellCharter<sup><a href="#CR7" class="usa-link" aria-describedby="CR7">7</a></sup>, and BANKSY<sup><a href="#CR8" class="usa-link" aria-describedby="CR8">8</a></sup>, typically learn low-dimensional embeddings of the nodes in corresponding CNs and then apply clustering approaches to these embeddings. However, clustering approaches suffer the following challenges in dissecting and interpreting highly heterogeneous, dynamically evolving cell systems<sup><a href="#CR9" class="usa-link" aria-describedby="CR9">9</a></sup>. First, clustering results usually become less stable when samples contain cells under active state transition, which is common in disease or developmental processes<sup><a href="#CR10" class="usa-link" aria-describedby="CR10">10</a></sup>. Second, clusters identified by these top-down approaches are often described as percentages of cell-type compositions. These clustering presentations lack formulations in topologically representing the geometrical cell-type interactions or are difficult to interpret biologically. Last, these top-down results essentially depend on the presence of batch effects, where CNs separate primarily by samples as technical covariates rather than biological features<sup><a href="#CR3" class="usa-link" aria-describedby="CR3">3</a></sup>. These batch effects make it easy to overfit the models but difficult to validate across different datasets<sup><a href="#CR5" class="usa-link" aria-describedby="CR5">5</a></sup>.</p>
<p id="Par5">Considering the preceding limitations of top-down strategies, we instead use a bottom-up strategy to identify CC motifs as recurring significant interconnections between cells. In the spatial omics–derived CC, we hypothesize that CC motifs can be represented as topological building blocks of multicellular organization consistent across different samples and associated with key biological processes and functions. CC motifs are biologically interpretable spatial patterns of the combined cell types, which provide topological information beyond clusters and explicitly link to the biological and pathological mechanisms through distinct cell–cell communications, highly expressed genes and pathways<sup><a href="#CR11" class="usa-link" aria-describedby="CR11">11</a></sup>. This concept is related to the functional tissue units (FTUs)<sup><a href="#CR12" class="usa-link" aria-describedby="CR12">12</a></sup>, but CC motifs are even smaller in the scale of cell locations and cell types, which provides more details for understanding and modeling the healthy physiological function of the organ and functional-related changes during disease states. Currently, size 1–3 motif analysis<sup><a href="#CR4" class="usa-link" aria-describedby="CR4">4</a></sup> makes up most of the spatial omics studies, where size-1 motifs are single nodes that can be treated as cell-type compositions, size-2 motifs are double nodes linked by edges, and size-3 motifs are triple nodes within triangles. Nevertheless, biologists have found that sizable CC motifs with more nodes than triangles substantially correlate with patient survival and phenotypical features in colorectal cancer (CRC)<sup><a href="#CR13" class="usa-link" aria-describedby="CR13">13</a></sup>, kidney diseases<sup><a href="#CR14" class="usa-link" aria-describedby="CR14">14</a></sup>, maternal–fetal interface<sup><a href="#CR15" class="usa-link" aria-describedby="CR15">15</a></sup>, and many other biological contexts.</p>
<p id="Par6">In practice, identifying the most overrepresented CC motifs composing multicellular organization is still computationally expensive with (<em>i</em>) subgraph matching<sup><a href="#CR16" class="usa-link" aria-describedby="CR16">16</a></sup>, which counts the occurrence of a given motif on the query graph, and (<em>ii</em>) pattern growth<sup><a href="#CR17" class="usa-link" aria-describedby="CR17">17</a></sup>, which finds the motifs with the most significant occurrence. It is known that subgraph matching is NP-complete<sup><a href="#CR16" class="usa-link" aria-describedby="CR16">16</a></sup>, which makes the node type combination alone super-exponential. Existing approaches include permutation<sup><a href="#CR11" class="usa-link" aria-describedby="CR11">11</a></sup>, edge sampling (e.g., mfinder<sup><a href="#CR18" class="usa-link" aria-describedby="CR18">18</a></sup>), node sampling (e.g., FANMOD<sup><a href="#CR19" class="usa-link" aria-describedby="CR19">19</a></sup>), and global pruning (e.g., Ullmann<sup><a href="#CR20" class="usa-link" aria-describedby="CR20">20</a></sup> and VF2<sup><a href="#CR21" class="usa-link" aria-describedby="CR21">21</a></sup>). A computationally feasible approach is still lacking to analytically identify conservative, interpretable, and generalizable spatial rules of cellular organization in different sizes across different samples of spatial omics.</p>
<p id="Par7">Here, we propose the triangulation cellular community motif neural network (TrimNN), a graph-based deep learning approach to analyze spatial transcriptomics and proteomics data using a bottom-up strategy (Supplementary Fig. <a href="#MOESM1" class="usa-link">1</a>). Within the input spatial omics samples, CC is defined based on the cells as nodes, the node types represent different cell types, and the edges encode physical proximity inferred unidirectional as the spatial cell-cell relation from Delaunay triangulation<sup><a href="#CR22" class="usa-link" aria-describedby="CR22">22</a></sup> based on nodes coordinates from ROI. TrimNN estimates overrepresented size-<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq1"><math id="d33e453"><mi>K</mi></math></span> CC motifs in the CC of spatial omics using graph isomorphism networks<sup><a href="#CR23" class="usa-link" aria-describedby="CR23">23</a></sup> (GIN) empowered by positional encoding<sup><a href="#CR24" class="usa-link" aria-describedby="CR24">24</a></sup> (PE). In various spatial transcriptomics and spatial proteomics case studies, TrimNN identifies computationally significant and biologically meaningful CC motifs to differentiate patient survival in CRC studies and represents pathologically related cell type organization in neurodegenerative diseases and colorectal carcinoma studies. Notably, the identified sizable CC motifs demonstrate their potential as interpretable topological prognostic biomarkers linking the topological structural organization of cell types at microscopic levels to phenotypes at macroscopic levels, which cannot be inferred by other existing tools. The source code of TrimNN is publicly available at <a href="https://github.com/yuyang-0825/TrimNN" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/yuyang-0825/TrimNN</a>.</p></section><section id="Sec2"><h2 class="pmc_sec_title">Results</h2>
<section id="Sec3"><h3 class="pmc_sec_title">TrimNN quantifies multicellular organization with sizable CC motifs</h3>
<p id="Par8">A schematic diagram of the proposed TrimNN and its analytic workflow is shown in Fig. <a href="#Fig1" class="usa-link">1A</a>. These identified CC motifs are biologically interpretable through a set of downstream analyses, including motif visualization, cellular-level interpretation within cell–cell communication analysis, gene-level interpretation within differentially expressed gene and pathway analysis, and phenotypical analysis within the availability of phenotypical information (Fig. <a href="#Fig1" class="usa-link">1B</a>). TrimNN is constructed on an empowered GIN to estimate the occurrence of the query on the target graph. On the CC as a triangulated graph built from spatial omics, TrimNN builds a supervised graph learning model by simplifying the graph constraints and incorporating the inductive bias within triangles derived from Delaunay triangulation. TrimNN decomposes the regression task in occurrence counting of the query graphs into many trackable binary classification tasks modeled by the sub-TrimNN module. Inspired by the idea of NSIC<sup><a href="#CR25" class="usa-link" aria-describedby="CR25">25</a></sup>, this method is trained on representative pairs of the predefined query subgraphs and the target triangulated cell graphs as a binary classification task. This graph representation framework builds upon GIN and adopts a shortest distance–based PE<sup><a href="#CR24" class="usa-link" aria-describedby="CR24">24</a></sup>, modeling the symmetric space to increase the expressive power. Additionally, TrimNN adopts a semi–divide-and-conquer strategy to estimate the abundance of the query by summarizing the enumeration of single classification tasks by a sub-TrimNN module on each node’s enclosed graph. Given the size of the query subgraph, our framework uses an enumeration approach to estimate the most overrepresented CC motifs with possible cell types and topology. Then, we search to infer CC motifs in different sizes incrementally. The details of the architecture of TrimNN are shown in Supplementary Fig. <a href="#MOESM1" class="usa-link">2</a>.</p>
<figure class="fig xbox font-sm" id="Fig1"><h4 class="obj_head">Fig. 1. TrimNN analysis workflow.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365081_41467_2025_63141_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2a5b/12365081/ddd26821e381/41467_2025_63141_Fig1_HTML.jpg" loading="lazy" id="d33e531" height="451" width="788" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>A</strong> Spatially resolved transcriptomics (e.g., STARmap PLUS and 10X Xenium) and spatial proteomics data (e.g., MIBI-TOF and CODEX) are used as input to generate corresponding CCs with spatial coordinates and Delaunay triangulation. TrimNN is trained on representative pairs of query motifs and target triangulated graphs at scale. Given a specific query, TrimNN identifies its occurrence in the target CC in the subgraph matching process by decomposing this regression task into many binary classification problems, where each classification predicts whether the query exists in the target graph as the enclosed graph of each node. Enumerating possible motifs at size-<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq2"><math id="d33e508"><mi>K</mi></math></span>, TrimNN identifies the most overrepresented motifs. Then, the pattern growth process adopts a heuristic search for their successor size-<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq3"><math id="d33e517"><mi>k</mi><mo>+</mo><mn>1</mn></math></span> motifs. Here, we take size-3 CC motifs as an example. After subgraph matching and pattern growth, TrimNN estimates overrepresented CC motifs. Created in BioRender. Yu, Y. (2025) <a href="http://BioRender.com/mfm4ta4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">BioRender.com/mfm4ta4</a>. <strong>B</strong> These CC motifs can be biologically interpreted in the downstream analysis, including visualization, cellular-level interpretation within cell–cell communication analysis, gene-level interpretation within differentially expressed gene analysis (e.g., GO enrichment analysis and pathway enrichment analysis), and phenotypical analysis within the availability of phenotypical information (e.g., survival curve and phenotypic classification analysis). CC: cellular community.</p></figcaption></figure><p id="Par9">We hypothesize CC motifs as the countable recurring spatial patterns of various cell types are robust within noises to represent and quantify multicellular organization. We performed simulations to mimic different levels of noises, including cell missing from the cell capture imperfection of sequencing technology (Fig. <a href="#Fig2" class="usa-link">2A</a>), cell coordinate shifting from technological errors (Fig. <a href="#Fig2" class="usa-link">2B</a>), and cell type misclassification from annotation errors in data analytics (Fig. <a href="#Fig2" class="usa-link">2C</a>). We noticed diverse noises do not influence the relative ranking of CC motif abundance, which remained robustly consistent in most scenarios (Fig. <a href="#Fig2" class="usa-link">2A</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">3</a>). Even under extreme cases with a noise ratio of 0.4 and 0.5, the Spearman correlation between abundance rankings before and after poised noise remained relatively stable in cell dropout and cell coordinate perturbations. When the noise level is high, the correlation values deteriorate in large motif sizes for cell type misclassification, which is unlikely to occur in practical scenarios.</p>
<figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2. The performance of TrimNN on spatial omics.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365081_41467_2025_63141_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2a5b/12365081/cfc37f34c33a/41467_2025_63141_Fig2_HTML.jpg" loading="lazy" id="d33e617" height="732" width="667" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>A</strong> Simulations of missing cell effects on CC motifs, represented as the Spearman correlation between abundance rankings of all the possible motifs before and after simulated noises at cell proportions of 0.01, 0.05, 0.1, 0.2, 0.3, 0.4, and 0.5 within CC motifs in size-1, size-2, size-3, size-4, and size-5 (<em>n</em> = 100). <strong>B</strong> Simulations of cell coordinate shifting effects on CC motifs, represented as the Spearman correlation between abundance rankings of all the possible motifs before and after simulated noises with different levels of noises of 0.01, 0.05, 0.1, 0.2, 0.3, 0.4, and 0.5 at cell proportions of 0.01, 0.05, 0.1, 0.2, 0.3, 0.4, and 0.5 within CC motifs in size-1, size-2, size-3, size-4, and size-5 (<em>n</em> = 100). <strong>C</strong> Simulations of cell-type misclassification effects on CC motifs, represented as the Spearman correlation between abundance rankings of all the possible motifs before and after simulated noises at cell proportions of 0.01, 0.05, 0.1, 0.2, 0.3, 0.4, and 0.5 within CC motifs in size-1, size-2, size-3, size-4, and size-5 (<em>n</em> = 100). <strong>D</strong> Benchmarking the performance of TrimNN, TrimNN-RGIN, and NSIC on independent simulated data for subgraph matching (<em>n</em> = 3000). The <em>X</em>-axis represents different sizes of CC motifs, and the <em>Y</em>-axis indicates the MCC (Matthews Correlation Coefficient) values. <strong>E</strong> Performance comparison of TrimNN, TrimNN-RGIN, and NSIC in identifying occurrences of CC motifs in diverse simulated datasets. The <em>Y</em>-axis is the RMSE (Root Mean Square Error) value. <strong>F</strong> Scalability of TrimNN. The <em>X</em>-axis represents the size of the triangulated graph, and the <em>Y</em>-axis indicates the runtime on a workstation equipped with an Intel Xeon Gold 6338 CPU and 80 G RAM. <strong>G</strong> Ablation tests on performance comparison adding the positional encoding of TrimNN model (<em>n</em> = 3000). The <em>X</em>-axis represents different sizes of CC motifs, and the <em>Y</em>-axis indicates the MCC values. CC: cellular community. On each box, the central mark indicates the median, and the bottom and top edges of the box indicate the 25th and 75th percentiles. The whiskers extend to the most extreme data points without outliers, and the outliers are plotted individually as circles. Source data are provided as a Source Data file.</p></figcaption></figure></section><section id="Sec4"><h3 class="pmc_sec_title">TrimNN accurately identifies overrepresented CC motifs in Cellular Neighborhoods</h3>
<p id="Par10">On a modified subgraph matching task as a binary classification of motif existence in a triangulated graph, TrimNN outperformed the competitive methods in all scenarios in most criteria in synthetic spatial omics data, including VF2, the original regression-based neural network method NSIC<sup><a href="#CR25" class="usa-link" aria-describedby="CR25">25</a></sup>, and TrimNN-RGIN with the proposed formulation but using NSIC’s RGIN network architecture. Especially on large-size CC motifs, TrimNN demonstrated significant performance improvements with TrimNN-RGIN, highlighting its architecture’s capacity (Fig. <a href="#Fig2" class="usa-link">2D</a> and Supplementary Data <a href="#MOESM3" class="usa-link">1</a>).</p>
<p id="Par11">TrimNN accurately identified the top overrepresented CC motifs. On a pattern growth challenge to determine the ranking of CC motif abundance, TrimNN outperformed competitive methods consistently in different sizes and cell types in synthetic spatial omics data. Both TrimNN and TrimNN-RGIN outperformed NSIC by a large margin in most scenarios and criteria, which highlights the capability of the proposed problem formulation. Notably, TrimNN demonstrated an average improvement over NSIC by approximately 20 to 60 times in root mean square error (RMSE) (Fig. <a href="#Fig2" class="usa-link">2E</a> and Supplementary Data <a href="#MOESM3" class="usa-link">2</a>). Besides the criteria in absolute occurrence value, the relative value of the ranking index also supported TrimNN’s capacity in Supplementary Fig. <a href="#MOESM1" class="usa-link">4A</a> and Supplementary Data <a href="#MOESM3" class="usa-link">3</a>.</p>
<p id="Par12">TrimNN is highly scalable in identifying large-size CC motifs. Because scalability plays a vital role in the study, we compared the computational time on target-triangulated graphs with varying node sizes. We observed that TrimNN, TrimNN-RGIN, and NSIC exhibit linear scalability with increasing node sizes, while TrimNN continuously consumed less computational time (Fig. <a href="#Fig2" class="usa-link">2F</a>). TrimNN was especially more efficient than TrimNN-RGIN with a simpler network architecture using the same problem setting. In contrast, the classical enumeration-based VF2 method grew exponentially, where its runtime made it unacceptable in most scenarios. In practical usage, on typical spatial omics data with thousands of cells of dozens of cell types, TrimNN robustly infers large-size CC motifs accurately in seconds, which is unattainable through conventional methods.</p>
<p id="Par13">Together with GIN, PE increases the expressive power of TrimNN. In challenging tasks with larger-sized motifs, ablation tests showed that integrating PE improved GNN (Graph Neural Network) performance compared with TrimNN-RGIN without PE and a complex GRU module (Fig. <a href="#Fig2" class="usa-link">2G</a> and Supplementary Data <a href="#MOESM3" class="usa-link">4</a>). In addition, GIN, as the critical component in TrimNN, was effective by replacing it with other graph neural network models, including Graph Convolutional Networks and Graph Transformer<sup><a href="#CR26" class="usa-link" aria-describedby="CR26">26</a></sup>, keeping other components and parameters constant (Supplementary Fig. <a href="#MOESM1" class="usa-link">4B</a> and Supplementary Data <a href="#MOESM3" class="usa-link">5</a>). This result aligned with theoretical analyses that GIN is a powerful 1-order graph neural network<sup><a href="#CR23" class="usa-link" aria-describedby="CR23">23</a></sup>. Meanwhile, it was shown that TrimNN requires sufficient training data to learn the complex relationships (Supplementary Fig. <a href="#MOESM1" class="usa-link">4C</a> and Supplementary Data <a href="#MOESM3" class="usa-link">6</a>).</p></section><section id="Sec5"><h3 class="pmc_sec_title">TrimNN identifies representative CC motifs that accurately differentiate the severity of colorectal cancer patients</h3>
<p id="Par14">In addition to the above simulation studies, we showed that the CC motifs inferred by TrimNN are intrinsic representations to differentiate phenotypes of the CC. In a proteomics study comprising 17 low-risk (Crohn’s-like lymphoid reaction, CLR) and 18 high-risk (diffuse inflammatory infiltration, DII) patients, using Co-Detection by Indexing (CODEX)<sup><a href="#CR13" class="usa-link" aria-describedby="CR13">13</a></sup> on CRC, we performed a CC motif analysis using TrimNN on 140 tissue regions and identified the most abundant CC motifs in size-1 to size-4. Traditional machine learning approaches, such as logistic regression (LR), were adopted using relative ranking indices to quantify motif occurrence as features. Because the original publication annotated 29 cell types, we chose 29 as the fixed number of features in supervised learning to classify CLR and DII. Within tenfold cross-validation following the same protocol as CytoCommunity<sup><a href="#CR6" class="usa-link" aria-describedby="CR6">6</a></sup>, the ROC-AUC results of LR were 0.77, 0.76, 0.79, and 0.76 (Fig. <a href="#Fig3" class="usa-link">3A</a>) for size-1 to size-4 CC motifs, respectively. This LR model with 29 CC motif features outperformed CytoCommunity’s extensive GNN computation performance using a default of 512 dimensions of embeddings as features (ROC-AUC: 0.71). Notably, if the feature number increased to the top 100, the LR model on size-3 motifs achieved an ROC-AUC of 0.81. To investigate the robustness of the model against potential overfitting, additional experiments were performed using 10 times 10-fold cross-validation on the LR model with the top 5, 10, 15, and 20 size-3 CC motif features, as well as CytoCommunity using reduced 29-dimensional embedding (Supplementary Fig. <a href="#MOESM1" class="usa-link">5</a> and Supplementary Data <a href="#MOESM3" class="usa-link">7</a>). The performance of the LR model with diverse small numbers of CC motif features is relatively stable, suggesting it encounters limited influences from overfitting with LR. In addition, other classical machine learning models, such as Random Forest and Support Vector Machine, were applied to the same classification tasks with the same settings. These models performed similarly to LR, further supporting the representational power of CC motifs (Supplementary Data <a href="#MOESM3" class="usa-link">8</a>-<a href="#MOESM3" class="usa-link">10</a>). The evaluation of the comprehensive performance comparison to CytoCommunity and SPACE-GM is provided in Supplementary Data <a href="#MOESM3" class="usa-link">11</a>.</p>
<figure class="fig xbox font-sm" id="Fig3"><h4 class="obj_head">Fig. 3. TrimNN analysis in a colorectal cancer study using CODEX.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365081_41467_2025_63141_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2a5b/12365081/576233ef4f01/41467_2025_63141_Fig3_HTML.jpg" loading="lazy" id="d33e797" height="999" width="794" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>A</strong> The ROC curves of the LR model classify CLR and DII patients using the top CC motifs of size-1 to size-4 as features, and the competitive method CytoCommunity uses learned dimension. The LR model uses features as motif counts from TrimNN and scales between 0 and 1. <strong>B</strong> Visualization of all the samples using the top two principal components from the 29 top size-2 CC motifs. Blue spots denote the CLR patient group and red spots denote the DII patient group. <strong>C</strong> Generalizability of the trained model testing on random cropping of ROI in the samples (<em>n</em> = 100). The <em>X</em>-axis is the ratio of width and height of the original ROI, and the <em>Y</em>-axis is ROC-AUC. On each box, the central mark indicates the median, and the bottom and top edges of the box indicate the 25th and 75th percentiles. The whiskers extend to the most extreme data points without outliers, and the outliers are plotted individually as circles. Survival curves of DII patients with and without enriched motifs, including <strong>D</strong> size-2 “A &amp; B”. Here, cell type CD68+CD163+ macrophages are denoted as “A” and smooth muscles are denoted as “B”. <strong>E</strong> size-3 “A &amp; A &amp; B” and <strong>F</strong> size-4 “A &amp; A &amp; B &amp; B”. The visualization of spatial localization of size-2 CC motif “A &amp; B” on the CC in <strong>G</strong> patient 3 (DII) on spot 5A and <strong>H</strong> patient 8 (CLR) on spot 16 A. The visualization of the spatial locations of the size-3 motif “A &amp; A &amp; B” in <strong>I</strong> DII spot and <strong>J</strong> CLR spot (same spots as <strong>G</strong> and <strong>H</strong>). The visualization of spatial localization of the size-4 motif “A &amp; A &amp; B &amp; B” in <strong>K</strong> DII spot and <strong>L</strong> CLR spot (same spots as <strong>G</strong> and <strong>H</strong>). All motifs are marked as blue, nodes of cell type “A” are red, and nodes of cell type “B” are orange. The plot of LR coefficients ranked by Cox PH <em>p</em>-value of the top 29 CC motifs in <strong>M</strong> size-2, <strong>N</strong> size-3, and <strong>O</strong> size-4. The extent of the blue color represents the Cox PH <em>p</em>-value. The <em>p</em>-values were derived from the two-sided Wald test. * marks the highlighted motif. LR: logistic regression, CC: cellular community. Source data are provided as a Source Data file.</p></figcaption></figure><p id="Par15">Besides supervised learning, these CC motifs seemed to capture some intrinsic characteristics in CNs, where CLR and DII demonstrated good visual separation using the top two principal components inferred from the top 29 size-2 motifs (Fig. <a href="#Fig3" class="usa-link">3B</a>). Further unsupervised hierarchical clustering showed the different distributions of the top 29 motif abundances among the CLR and DII groups in Supplementary Fig. <a href="#MOESM1" class="usa-link">6A</a> and <a href="#MOESM1" class="usa-link">6B</a> (size-2), Supplementary Fig. <a href="#MOESM1" class="usa-link">6C</a> and <a href="#MOESM1" class="usa-link">6D</a> (size-3), and Supplementary Fig. <a href="#MOESM1" class="usa-link">6E</a> and <a href="#MOESM1" class="usa-link">6F</a> (size-4).</p>
<p id="Par16">CC motifs in simpler models and fewer numbers of features showed better generalizability across multiple samples in machine learning. When only parts of the CC were available by random cropping the samples, CC motif–based LR methods were very robust in generalizability compared to competitive methods (Fig. <a href="#Fig3" class="usa-link">3C</a>). The same trends were observed in distorted samples with simulated noises in cell missing, cell coordinate shifting, and cell type misclassification (Supplementary Fig. <a href="#MOESM1" class="usa-link">7</a>).</p>
<p id="Par17">In addition, the enrichment of sizable CC motifs can be used to differentiate patient survival. We identified several size-2, size-3, and size-4 CC motifs that significantly differentiate survival (Cox PH <em>p</em> &lt; 0.05) between enriched and non-enriched DII patients, while cell type composition (size-1 motifs) may not necessarily succeed (Supplementary Data <a href="#MOESM3" class="usa-link">12</a>). With cell type “CD68+CD163+ macrophages” (denoted by “A”) and cell type “smooth muscle” (denoted by “B”), the survival curves showed that size-2 motif “A &amp; B” may not have adequately separated survival in the DII patient group (Cox PH <em>p</em> = 0.63, shown in Fig. <a href="#Fig3" class="usa-link">3D</a>), but including more adjacent nodes with the same cell types, patients with enrichment of size-3 (“A &amp; A &amp; B”) and size-4 (“A &amp; A &amp; B &amp; B”) CC motifs showed significant lower survival rates (Cox PH <em>p</em> = 0.016, shown in Fig. <a href="#Fig3" class="usa-link">3E</a>, and Cox PH <em>p</em> = 0.0093, shown in Fig. <a href="#Fig3" class="usa-link">3F</a>, respectively). To validate the results, we performed additional survival analyses in both COAD and READ cohorts of The Cancer Genome Atlas (TCGA) associated with CD68+CD163+ macrophage marker genes: <em>CD68</em>, <em>CD163</em>, <em>CD14</em>, and <em>ITGAM</em><sup><a href="#CR27" class="usa-link" aria-describedby="CR27">27</a></sup>, and smooth muscle marker genes: <em>ACTA2</em>, <em>MYH11</em>, and <em>MYL9</em><sup><a href="#CR28" class="usa-link" aria-describedby="CR28">28</a></sup>. We observed no significant differences in survival between patients associated with either cell type (Cox PH <em>p</em> &gt; 0.05, Supplementary Figs. <a href="#MOESM1" class="usa-link">8</a> and <a href="#MOESM1" class="usa-link">9</a>). This independent analysis showed consistent results, indicating that cell type compositions, such as size-1 CC motifs, have limited effectiveness in differentiating patient survival in CRC. In addition, the occurrence numbers of these CC motifs among DII and CLR patients were 14,415 and 7004 (ratio 2.06) for size-2 “A &amp; B”; 4176 and 1548 (ratio 2.70) for size-3 “A &amp; A &amp; B”; and 6946 and 2276 (ratio 3.05) for size-4 “A &amp; A &amp; B &amp; B,” respectively in each case. All were inferred as significant through the Benjamini-Hochberg adjusted Fisher’s exact test. The different distribution of these CC motifs on CCs among DII and CLR spots can be visualized in Fig. <a href="#Fig3" class="usa-link">3G and H</a> (size-2), Fig. <a href="#Fig3" class="usa-link">3</a>I and J (size-3), and Fig. <a href="#Fig3" class="usa-link">3K, L</a> (size-4).</p>
<p id="Par18">Notably, spatial topology plays a crucial role in linking phenotypes and survival. There were two types of size-3 motifs with cell types CD68+CD163+macrophages (“A”) and smooth muscle (“B”). Compared with “A &amp; A &amp; B”, the alternative motif “A &amp; B &amp; B” occurred 4602 and 2255 times among DII and CLR patients, respectively, with a lower ratio of 2.04, and it cannot differentiate survival well (Cox PH <em>p</em> = 0.2975). Apparently, these topological differences among the spatial localization of cells in different cell types played different roles biologically and pathologically, where conventional top-down approaches with cell type composition failed to distinguish (Supplementary Fig. <a href="#MOESM1" class="usa-link">10</a> and Supplementary Data <a href="#MOESM3" class="usa-link">13</a>–<a href="#MOESM3" class="usa-link">16</a>).</p>
<p id="Par19">Furthermore, an LR model provides intrinsic interpretability when differentiating phenotypes. The coefficients of each feature from the LR model demonstrated the importance of CC motifs quantitatively, making the model interpretable (Fig. <a href="#Fig3" class="usa-link">3</a>M–O). Notably, all macrophage-related, muscle-related, and significant Cox PH <em>p</em>-value motifs in different sizes tended to have high absolute coefficient values. The same interpretable results can also be cross-validated by Shapley value<sup><a href="#CR29" class="usa-link" aria-describedby="CR29">29</a></sup> in Supplementary Fig. <a href="#MOESM1" class="usa-link">6</a>G–I, showing that these macrophage-related and muscle-related CC motifs were essential to differentiating DII patients from CLR patients. Biologically, it was evidenced that macrophages facilitate pancreatic cancer to induce muscle wasting via promoting TWEAK (TNF-like weak inducer of apoptosis) secretion from the tumor<sup><a href="#CR30" class="usa-link" aria-describedby="CR30">30</a></sup>. After carefully checking these top motifs in different sizes, we also identified biologically meaningful tumor cells and B cells, which were known to be related to the severity of CRC<sup><a href="#CR31" class="usa-link" aria-describedby="CR31">31</a></sup>. Representative tumor and B cell enrichments in DII and CLR samples are shown in Supplementary Fig. <a href="#MOESM1" class="usa-link">6J</a>, <a href="#MOESM1" class="usa-link">6K</a>, <a href="#MOESM1" class="usa-link">6L</a>, and 6M. Our analysis validated the crosstalk between macrophages, muscle wasting, and cancer cachexia through an independent spatial omics study, and TrimNN identified CC motifs in a data-driven approach as robust interpretable representations in CNs.</p></section><section id="Sec6"><h3 class="pmc_sec_title">TrimNN identifies CC motifs revealing diverse roles in Alzheimer’s disease using spatial transcriptomics data</h3>
<p id="Par20">Next, we showed TrimNN’s capability to identify diverse spatially distributed CC motifs corresponding to multiple biological and pathological mechanisms in complex diseases. It is known that the interaction between CTX (Cortex) excitatory neurons and Microglia is significantly disrupted by neuroinflammation in Alzheimer’s disease (AD)<sup><a href="#CR32" class="usa-link" aria-describedby="CR32">32</a></sup>. However, their topological combinations, particularly their relationship with amyloid-β on the cellular level, are still unknown<sup><a href="#CR33" class="usa-link" aria-describedby="CR33">33</a></sup>. In a study, we performed TrimNN on an AD mouse brain with 8-month-old and 13-month-old samples sequenced by STARmap PLUS spatially resolved transcriptomics<sup><a href="#CR34" class="usa-link" aria-describedby="CR34">34</a></sup>. There were two replicates for both disease and control conditions at each time point. The transcriptomics data included 2,766 genes and two proteomics channels representing AD markers of amyloid-β and tau pathologies at subcellular resolution.</p>
<p id="Par21">On the derived CCs, size-3 triangle-like CC motifs composed of cell types CTX excitatory neurons and Microglia were identified significant between AD (Fig. <a href="#Fig4" class="usa-link">4A</a>) and control (Fig. <a href="#Fig4" class="usa-link">4B</a>). These significant CC motifs included CTX excitatory neurons-CTX excitatory neurons-CTX excitatory neurons (CCC), CTX excitatory neurons-CTX excitatory neurons-Microglia (CCM), CTX excitatory neurons-Microglia-Microglia (CMM), and Microglia-Microglia-Microglia (MMM) (Benjamini–Hochberg adjusted Fisher’s exact test in 8-month-old replicate 1 with <em>p</em> = 6.12e–32, <em>p</em> = 3.23e–20, <em>p</em> = 4.30e–34, and <em>p</em> = 9.74e–07, respectively). Visualization of an exemplary CC motif “MMM” demonstrated uneven spatial distribution that differed in AD (Fig. <a href="#Fig4" class="usa-link">4C</a>) and control (Fig. <a href="#Fig4" class="usa-link">4D</a>). Supplementary Fig. <a href="#MOESM1" class="usa-link">11</a> shows the spatial occurrence distribution of the other three motifs. The CCs inferred from all samples are shown in Supplementary Fig. <a href="#MOESM1" class="usa-link">12</a> and Supplementary Data <a href="#MOESM3" class="usa-link">17</a>–<a href="#MOESM3" class="usa-link">28</a>.</p>
<figure class="fig xbox font-sm" id="Fig4"><h4 class="obj_head">Fig. 4. TrimNN analysis in an AD mouse study sequenced by STARmap PLUS.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365081_41467_2025_63141_Fig4_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2a5b/12365081/dd72bd80ab72/41467_2025_63141_Fig4_HTML.jpg" loading="lazy" id="d33e1053" height="846" width="800" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>CCs of 13-month-old of <strong>A</strong>. AD and <strong>B</strong> control sample replicate 1 are obtained using Delaunay triangulation, where black spots are amyloid-β in the AD sample. The spatial locations of the identified motif with all Microglia cells (“MMM” motif, where “M” denotes cell type Microglia) in 13-month-old replicate 1 of <strong>C</strong>. AD and <strong>D</strong> control mouse samples. “MMM” motifs are marked as purple. Cell–cell communication analysis demonstrates the ligand–receptor differences between motif regions and non-motif regions as river plots, including <strong>E</strong> cell type CTX excitatory neurons (denoted as “C”) as source (left) and target (right), <strong>F</strong> cell type Microglia as source (left) and target (right) in regions with and without “CCC” motif. Similarly, <strong>G</strong> and <strong>H</strong> are cell types of CTX excitatory neurons and Microglia as source and target in regions with and without the “MMM” motif. <strong>I</strong> GO enrichment analysis of Biological Processes and <strong>J</strong> Pathway enrichment analysis on DEGs between regions containing and not containing the “MMM” motif in 13-month-old AD samples. <strong>K</strong> Expression of marker genes for cell types “C” and “M” related size-3 and size-4 motifs. <strong>L</strong> Spatial co-occurrence of different CC motifs with respect to amyloid-β as computed using Squidpy. Microglia-related motifs have an even higher spatial co-occurrence probability compared to the amyloid-β plaque, and CTX excitatory neurons–related motifs have lower spatial co-occurrence probabilities compared to the amyloid-β plaque. CC: cellular community. Source data are provided as a Source Data file.</p></figcaption></figure><p id="Par22">Here, we defined motif-enriched regions as expanded regions within three hops of CC motifs in the CC. From the perspective of cell–cell communications, unique ligand–receptor signaling pathway patterns between “CCC” (Fig. <a href="#Fig4" class="usa-link">4</a>E, F) and “MMM” (Fig. <a href="#Fig4" class="usa-link">4G and H</a>) were identified by motif-enriched and complementary regions on 13-month-old samples using CellChat<sup><a href="#CR35" class="usa-link" aria-describedby="CR35">35</a></sup>. Specific to cell type Microglia, motifs “CCC” and “MMM” had dominant ligand–receptor pairs GRN<sup><a href="#CR21" class="usa-link" aria-describedby="CR21">21</a></sup> and PMCH to distinguish motif regions from the complementary regions. AD-related ligand–receptors, including GRN, VEGF, PDGF, CCL, VIP, NRG, and SEMA3, were significantly enriched in CC motifs associated with CTX excitatory neurons or Microglia (Supplementary Data <a href="#MOESM3" class="usa-link">29</a>–<a href="#MOESM3" class="usa-link">32</a>). All the cell–cell communication results on CC motifs are detailed in Supplementary Figs. <a href="#MOESM1" class="usa-link">13</a>–<a href="#MOESM1" class="usa-link">21</a> and Supplementary Data <a href="#MOESM3" class="usa-link">33</a>–<a href="#MOESM3" class="usa-link">40</a>. These differences in cell–cell communications were independently validated by DeepTalk<sup><a href="#CR36" class="usa-link" aria-describedby="CR36">36</a></sup>, incorporating long-range cellular interactions. Particularly, CSF and VEGF pathways exhibited significant differences between “CCC” and “ non- CCC” regions in Microglia-to-CTX excitatory neurons cell–cell communication (Mann–Whitney <em>U</em> test, <em>p</em> = 0.033, and <em>p</em> = 0.003, Supplementary Figs. <a href="#MOESM1" class="usa-link">22</a>–<a href="#MOESM1" class="usa-link">25</a>). Considering cell–cell communication among all the cell types, all ligand–receptor pairs identified by CellChat show significant differences with and outside the identified motif regions (Mann–Whitney <em>U</em> test, <em>p</em>-value &lt; 0.001, Supplementary Figs. <a href="#MOESM1" class="usa-link">26</a> and <a href="#MOESM1" class="usa-link">27</a>).</p>
<p id="Par23">Then we analyzed the gene-level characteristics of identified CC motifs. Comparing “MMM” motif-enriched and complementary regions, differentially expressed genes (DEGs) were identified as significant (<em>p</em>-value &lt; 0.05) using the Wilcoxon rank-sum test, including <em>Plekha1</em>, <em>Ctsb</em>, and <em>Sort1</em> in 8-month-old samples (Supplementary Data <a href="#MOESM3" class="usa-link">44</a>), and <em>App</em>, <em>Plekha1</em>, <em>Clu</em>, <em>Ptk2b</em>, <em>Sort1</em>, <em>Bin1</em>, and <em><span class="text-underline">Ctsb</span></em> in 13-month-old samples (Supplementary Data <a href="#MOESM3" class="usa-link">48</a>). On DEGs in 13-month-old samples, Gene Ontology (GO) enrichment analysis showed significant vesicle-mediated transport in synapse (<em>q</em>-value = 1.68E–107), regulation of synapse structure or activity (<em>q</em>-value = 6.45e–106), learning or memory (<em>q</em>-value = 1.20e–56), and cognition (<em>q</em>-value = 5.54e–56) (Fig. <a href="#Fig4" class="usa-link">4I</a>). Neural systems (<em>q</em>-value = 3.63e–51), transmission across chemical synapses (<em>q</em>-value = 2.05e–33), neurotransmitter receptors and postsynaptic signal transmission (<em>q</em>-value = 8.68e–20), and nervous system development (<em>q</em>-value = 1.33e–15) were enriched with pathway enrichment analysis (Fig. <a href="#Fig4" class="usa-link">4J</a>). For all the detailed results of CC motifs “CCC”, “CCM”, “CMM”, and “MMM”, a similar analysis was performed for DEGs (Supplementary Data <a href="#MOESM3" class="usa-link">41</a>–<a href="#MOESM3" class="usa-link">48</a>), including both GO and pathway enrichment analyses (Supplementary Figs. <a href="#MOESM1" class="usa-link">28</a> and <a href="#MOESM1" class="usa-link">29</a>).</p>
<p id="Par24">To validate their relations with AD, we compared these motif-related DEGs with 77 AD-associated genes identified from large-scale GWAS analysis<sup><a href="#CR37" class="usa-link" aria-describedby="CR37">37</a></sup>. On CC motif “CCC”, the <em>Trem2</em> gene was exclusively observed in the replicates of the 13-month-old but not 8-month-old AD mouse model, consistent with its role in the late-onset of AD<sup><a href="#CR38" class="usa-link" aria-describedby="CR38">38</a></sup>. Similarly, for the motif “MMM”, the <em>Clu</em> gene was highlighted only in the 13-month-old mouse model, aligning with its direct involvement in the formation process of amyloid-β<sup><a href="#CR39" class="usa-link" aria-describedby="CR39">39</a></sup> (Supplementary Fig. <a href="#MOESM1" class="usa-link">30</a>).</p>
<p id="Par25">Based on the identified size-3 motifs, we performed pattern growth to identify size-4 motifs using TrimNN (Supplementary Notes <a href="#MOESM1" class="usa-link">1</a>). Among all the size-4 “CCC” expanded motifs, “CCCM” showed the most significant difference between AD and control samples, while “MMMM” was the most significant size-4 motif expanded from “MMM”. Similar to the analysis on size-3 motifs, checking significantly enriched ligand–receptors (Supplementary Data <a href="#MOESM3" class="usa-link">49</a>–<a href="#MOESM3" class="usa-link">51</a>) and DEGs (Supplementary Data <a href="#MOESM3" class="usa-link">52</a>–<a href="#MOESM3" class="usa-link">57</a>), these size-4 motifs were related to AD in cell–cell communication analysis (Supplementary Data <a href="#MOESM3" class="usa-link">58</a>–<a href="#MOESM3" class="usa-link">63</a>), GO enrichment analysis (Supplementary Fig. <a href="#MOESM1" class="usa-link">31</a>), and pathway enrichment analysis (Supplementary Fig. <a href="#MOESM1" class="usa-link">32</a>).</p>
<p id="Par26">Further investigation on DEGs showed diverse groups of CC motifs with expressed markers (Fig. <a href="#Fig4" class="usa-link">4K</a>). Size-3 “MMM” and size-4 “MMMM” motifs with homogeneous Microglia had divergent expression patterns. For example, <em>Hexb</em> had a higher average expression than the other CC motifs. <em>Hexb</em> is known to induce toxic and progressive neuronal damage, which may relate to neurodegenerative dementia<sup><a href="#CR40" class="usa-link" aria-describedby="CR40">40</a></sup>.</p>
<p id="Par27">In addition to examining the diversity of CC motifs at the gene level, we investigated whether the identified CC motifs were spatially co-localized with amyloid-β by computing their co-occurrence probabilities using Squidpy<sup><a href="#CR41" class="usa-link" aria-describedby="CR41">41</a></sup>. The results showed that Microglia-related CC motifs had an even higher co-occurrence probability with amyloid-β than the spatial expectation, distinguishing them from other CC motifs associated with CTX excitatory neurons (Fig. <a href="#Fig4" class="usa-link">4L</a>). Interestingly, the extent of homogeneity of Microglia regions seemed to correspond to a larger co-occurrence probability of amyloid-β. In contrast, the extent of homogeneity of CTX excitatory neurons tallied to a lower co-occurrence probability. This trend prevailed across the whole spectrum of CC motifs composed of Microglia and CTX excitatory neurons in multiple sizes, from a very high ratio of size-4 “MMMM” to a very low ratio of size-3 “CCC”.</p>
<p id="Par28">Differences in both DEGs and spatial co-occurrence suggest the presence of two distinct types of CC motifs related to amyloid-β in AD. One type of CC motif (i.e., “CCC”, “CCM”, “CMM”, and “CCCM”) was reluctant to co-localize with amyloid-β. Another kind of CC motif (i.e., “MMM” and “MMMM”) was closely co-localized with amyloid-β. These results were consistent with the observation that Microglia, as key mediators in the brain, activate inflammation in the vicinity of amyloid-β deposits, which are directly toxic to the adjacent neurons<sup><a href="#CR42" class="usa-link" aria-describedby="CR42">42</a></sup>. Activated Microglia release pro-inflammatory cytokines, such as tumor necrosis factor-alpha (TNF-α) and interleukin-1 beta (IL-1β), can damage excitatory neurons or alter their function<sup><a href="#CR43" class="usa-link" aria-describedby="CR43">43</a></sup>.</p>
<p id="Par29">In this case study, TrimNN confirmed known knowledge of AD-related cell types and provided some new insights into spatial biology. As an unbiased data-driven approach, TrimNN independently identified pathologically related spatial characteristics of Microglia and CTX excitatory neurons, along with their topological relations with diverse cell types, as two distinguished CC motifs differ in levels of cell type, gene, and cell–cell communications. Analysis enabled by CC motifs demonstrates an unprecedented spectrum of the spatial relationships between the homogeneity of CTX excitatory neurons/Microglia cell types and the location of amyloid-β. TrimNN accurately captured these spatial co-localization patterns with amyloid-β deposits, providing insights into the onset of AD as the result of interactions between multiple cell types<sup><a href="#CR44" class="usa-link" aria-describedby="CR44">44</a></sup>, which clustering-based tools may overlook (Supplementary Figs. <a href="#MOESM1" class="usa-link">33</a> and <a href="#MOESM1" class="usa-link">34</a>).</p></section><section id="Sec7"><h3 class="pmc_sec_title">TrimNN identifies cell type–specific spatial tendencies in a colorectal carcinoma study on spatial proteomics data</h3>
<p id="Par30">Besides the AD study, we also performed TrimNN analysis to explore cell type–specific spatial tendencies in one colorectal carcinoma study. It is known that the tumor microenvironment can significantly influence the interactions between T-cells and epithelial cells through antigen presentation, T-cell activation, and modulation of the tumor microenvironment. However, it is still unknown how the spatial arrangement of these cells is related to effective immune surveillance and the potential for therapeutic interventions<sup><a href="#CR45" class="usa-link" aria-describedby="CR45">45</a></sup>. The adopted colorectal carcinoma study investigated 40 ROIs in two colorectal cancer patients and 18 ROIs in two healthy controls using spatial proteomics of multiplexed ion beam imaging using time of flight (MIBI-TOF)<sup><a href="#CR46" class="usa-link" aria-describedby="CR46">46</a></sup>.</p>
<p id="Par31">After a comprehensive analysis of size-3 and their related size-4 CC motifs with TrimNN, we defined two types of CC motifs: Shifted Interaction Motifs and Homeostatic Interaction Motifs (Fig. <a href="#Fig5" class="usa-link">5A</a>). Shifted Interaction Motifs demonstrated a shift of CC motif abundance from control-enriched (more occurrence in control than disease samples) to disease-enriched (more occurrence in disease than control samples) when expanding from size-3 to size-4. The exemplary size-3 motif “ABC” (Fig. <a href="#Fig5" class="usa-link">5</a>B, C) suggested disease progression when involving other immune cells to form a size-4 motif “ABCD” (Fig. <a href="#Fig5" class="usa-link">5</a>D, E), where “A” denotes CD4 T-cells, “B” denotes CD8 T-cells, “C” denotes epithelial, and “D” denotes other immune cells (other CD45+) annotated by the original publication. Proportion tests showed that this size-4 “ABCD” motif significantly differed from the size-3 “ABC” motif in abundance between disease and control samples (<em>p</em> = 2.58e–12). In contrast, Homeostatic Interaction Motif remained consistent in abundance between disease and control groups when expanding its sizes (e.g., size-3 motif “AEC”) (Fig. <a href="#Fig5" class="usa-link">5</a>F, G), where “E” denotes endothelial concatenating another epithelial (“C”) as a size-4 motif “AECC” (Fig. <a href="#Fig5" class="usa-link">5</a>H, I). Proportion tests showed consistency between disease and control ratios among this pair of size-3 and size-4 motifs (<em>p</em> = 0.55). The expression level of antibodies also confirmed differences between these two groups of CC motifs (Fig. <a href="#Fig5" class="usa-link">5J</a>, Supplementary Fig. <a href="#MOESM1" class="usa-link">35A</a> and <a href="#MOESM1" class="usa-link">35B</a>). A similar analysis demonstrated that these two groups of CC motifs also existed in AD studies (Supplementary Notes <a href="#MOESM1" class="usa-link">2</a>).</p>
<figure class="fig xbox font-sm" id="Fig5"><h4 class="obj_head">Fig. 5. TrimNN analysis in a colorectal carcinoma study using MIBI-TOF.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365081_41467_2025_63141_Fig5_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2a5b/12365081/86fc9a71fcb2/41467_2025_63141_Fig5_HTML.jpg" loading="lazy" id="d33e1425" height="862" width="800" alt="Fig. 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>A</strong> Schematic of Shifted Interaction Motif and Homeostatic Interaction Motif as two types of size-4 motifs. Shifted Interaction Motifs: size-3 motif “ABC” (purple) in exemplary <strong>B</strong> spot 33 (disease) and <strong>C</strong> spot 56 (control), the successor size-4 motif “ABCD” (red) in the same <strong>D</strong> spot 33 (disease) and <strong>E</strong> spot 56 (control). Homeostatic Interaction Motifs: exemplary size-3 motif “AEC” (purple) in exemplary <strong>F</strong> spot 8 (disease) and <strong>G</strong> spot 50 (control), the successor size-4 motif “AECC” (red) in the same <strong>H</strong> spot 8 (disease) and <strong>I</strong> spot 50 (control). <strong>J</strong> Heatmap of antibody expression ratio between disease and control samples in Shifted Interaction Motif and Homeostatic Interaction Motif. <strong>K</strong> Ranking of effective size between all cell types in colon tissue samples. Abundance of size-2 CC motifs as occurrences in <strong>L</strong> colorectal carcinoma and <strong>M</strong> healthy control samples. <strong>N</strong>
<em>P</em>-value of size-2 CC motifs between disease and control by the two-sided Benjamini-Hochberg adjusted Fisher’s exact test. <strong>O</strong> Heatmap on sender rank from NCEM-type coupling analysis in colon tissue samples. Heatmap of NCEM-type coupling analysis in <strong>P</strong> colorectal carcinoma and <strong>Q</strong> healthy control samples. <strong>R</strong> Difference values from NCEM-type coupling analysis between disease and control samples. Cell type “A” denotes CD4 T-cells, “B” denotes CD8 T-cells, “C” denotes epithelial, “D” denotes other immune cells annotated by the original publication, and “E” denotes endothelial. The star symbol marks the paired cell type composition of the “AEC” motif. CC: cellular community. Source data are provided as a Source Data file.</p></figcaption></figure><p id="Par32">To better explain the biological relevance of these patterns, we analyzed the types of proteins showing increased expression in the expanded motifs. Interestingly, the increased disease/control fold-change observed in “ABCD” compared to “ABC” suggests that the involvement of the fourth node (other immune cells) is associated with heightened immune activity or immune cell engagement in the tumor microenvironment. Many of the upregulated proteins in this motif, such as CD39, CD11C, CD45, CD14, and H3, are related to immune infiltration, antigen presentation, or immune cell function. However, the presence of CD39 (involved in immunosuppressive adenosine signaling) and a concurrent rise in metabolic enzymes (e.g., HK1, G6PD, and VDAC1) and proliferation markers (e.g., Ki67) suggests that this immune activity may be skewed or dysfunctional. Rather than a classical immune-activated phenotype, the tumor microenvironment in “ABCD” may represent an immune-infiltrated but metabolically constrained immunoregulatory niche.</p>
<p id="Par33">In contrast, although the Homeostatic Interaction Motifs “AEC” and “AECC” do not change in abundance across conditions, their protein expression reveals different biological characteristics. Notably, PD1 and GLUT1 are upregulated in the size-4 motif “AECC” suggesting that immune checkpoint pathways and metabolic competition may play important roles even in spatially stable interaction contexts. PD1 upregulation indicates a checkpoint-mediated immune inhibitory signal, while GLUT1 elevation supports increased glycolytic activity, both of which can further restrict immune function through resource competition. We also observed that motif-specific differences in PD1 and CD3 did not change between “ABC” and “ABCD”, but both showed increased expression in “AECC” versus “AEC”. This implies that the involvement of endothelial cells may contribute to a niche promoting immune checkpoint activation or T cell exhaustion. CD36, a lipid metabolism regulator, showed a moderate decrease in the “ABCD” motif, suggesting lipid metabolic shifts associated with certain spatial configurations.</p>
<p id="Par34">Together, these comparisons provide evidence that different motif expansions not only reflect structural reorganization but also correspond to distinct biological states. The Shifted Interaction Motif represents a transition toward a metabolically active, immune-suppressive tumor niche, consistent with tumor progression and immune evasion. The Homeostatic Interaction Motif, while stable in structure, still shows signs of immune checkpoint engagement, potentially limiting effective immune surveillance.</p>
<p id="Par35">Next, we explored cell-type preferences using TrimNN analysis and identified CD4 T-cells that played key spatial roles in differentiating patients and healthy controls. Scrutinizing spatial co-localizations between cells in different cell types, homogeneous CD4 T-cells were most abundant in healthy control samples but ranked fifth in disease samples (purple rectangle in Fig. <a href="#Fig5" class="usa-link">5K</a>) by effective sizes, with the <em>p-</em>value &lt; 10e–30, using the Benjamini-Hochberg adjusted Fisher’s test (Fig. <a href="#Fig5" class="usa-link">5</a>L, M and Supplementary Data <a href="#MOESM3" class="usa-link">64</a>). This result was also supported by differences in the sender–receiver effect by NCEM<sup><a href="#CR47" class="usa-link" aria-describedby="CR47">47</a></sup>, which estimates cell interactions of the effects of niche composition (Fig. <a href="#Fig5" class="usa-link">5</a>O–Q). In addition, CD4 T-cells and epithelial cells tended to be more likely to be localized together than other combinations (yellow rectangle in Fig. <a href="#Fig5" class="usa-link">5K</a>). This combination was also significantly differentiated between disease and control (adjusted <em>p-</em>value &lt; 10e–30), supported by the observation that epithelial cells as the receiver significantly differ in antibody expression in CD4 T-cells, sender effect (Supplementary Fig. <a href="#MOESM1" class="usa-link">35C</a>, D) from NCEM. Statistical coupling analysis within all the cell types between disease and control samples (Fig. <a href="#Fig5" class="usa-link">5N</a>) showed that many were consistent with NCEM-type coupling analysis (Fig. <a href="#Fig5" class="usa-link">5R</a>). All these observations were consistent with the original studies<sup><a href="#CR46" class="usa-link" aria-describedby="CR46">46</a></sup>.</p>
<p id="Par36">We also investigated the motifs with sizes larger than 2 where NCEM cannot be inferred directly. The abundance of CC motif “AEC” (“CD4 T-cells &amp; Endothelial &amp; Epithelial”) was observed to be significantly different (<em>p</em> = 4.61e–15) between disease and control (Supplementary Data <a href="#MOESM3" class="usa-link">65</a>), where their composed pairs of cell types as size-2 motifs were confirmed as significant by TrimNN (Fig. <a href="#Fig5" class="usa-link">5N</a>). However, NCEM failed to support this observation, for none of these pairs were shown to be significantly different (Fig. <a href="#Fig5" class="usa-link">5R</a> marked as red “*”). Based on these analyses, TrimNN also found that motif “AEC”, a Homeostatic Interaction Motif, concatenating another cell type, epithelial, as a size-4 motif, was significant in disease and control. Similar analyses were performed to identify Shifted Interaction Motifs in Supplementary Notes <a href="#MOESM1" class="usa-link">3</a>. These results were also validated by an independent colorectal carcinoma study<sup><a href="#CR48" class="usa-link" aria-describedby="CR48">48</a></sup> using CODEX. The Homeostatic Interaction Motif remained consistent in abundance between the disease and control groups during the growth from size-3 to size-4, which was consistent with our observations (Supplementary Data <a href="#MOESM3" class="usa-link">67</a> and <a href="#MOESM3" class="usa-link">68</a>). Using TrimNN, we recognized that the decline of dominant CD4 T-cells in spatial space may be linked to colorectal carcinoma where CD4 T-cells had more spatial relations with other cell types<sup><a href="#CR49" class="usa-link" aria-describedby="CR49">49</a></sup>.</p></section></section><section id="Sec8"><h2 class="pmc_sec_title">Discussion</h2>
<p id="Par37">Spatial omics has significantly advanced our understanding of the nuanced cell organization within tissues at the cellular level. Being complementary to top-down approaches, such as clustering methods, TrimNN enables the characterization of sizable CC motifs and provides a new bottom-up angle in spatial omics analysis. (1) It overcomes the limitation of clustering approaches. Without arbitrary parameters in clustering, the bottom-up approach identifies topological building blocks as countable CC motifs to represent CNs. (2) It provides intrinsic interpretability of the topological building blocks of CNs. Easily interpretable biologically at the cellular and gene levels, and pathologically intertwined with clinical phenotypes, the results of TrimNN demonstrate the co-localization differences between repeated motifs and disease markers, which may correspond to different biological and pathological hypotheses. (3) It ensures better generalizability across samples. Without a complex machine learning model with uninterpretable embeddings dependent on trained samples, CC motifs as explicit interpretable representations are robust in differentiating CNs and noises.</p>
<p id="Par38">In many ways, the relationship between clustering-based approaches and CC motifs is similar to that between functional domains (such as Pfam<sup><a href="#CR50" class="usa-link" aria-describedby="CR50">50</a></sup>) and sequence motifs (such as PROSITE<sup><a href="#CR51" class="usa-link" aria-describedby="CR51">51</a></sup>) in protein research. For example, a typical protein sequence motif is a short, conserved amino acid pattern, often 10 to 20 residues in length, exemplified by motifs like the Zn-finger. These motifs are particularly useful for identifying fine-scale evolutionary differences, such as the gain or loss of protein functions resulting from motif-level changes. Functional domains represent larger, conserved sequences functioning as independent structural and functional units, providing broad functional annotations. In spatial biology, like protein functional domains, clustering provides a top-down view of tissue organization by identifying broad groupings of cell types based on composition and co-localization. In contrast, CC motifs, like protein sequence motifs, offer a bottom-up, fine-grained perspective by capturing conserved topological patterns of cell interactions. While clustering reveals high-level structural features, CC motifs uncover subtle, biologically meaningful differences, such as those between healthy and diseased tissues, enhancing sensitivity in comparative analyses and enabling deeper mechanistic insights.</p>
<p id="Par39">Mathematically, TrimNN offers an accurate, unbiased, efficient, and robust approach to quantifying CC motifs as interpretable building blocks of cell organization. The proposed work formulates the pattern quantification problem in counting subgraph occurrences, simplifying the task to a biologically constrained problem that can be solved by a supervised topological representation learning framework using triangles as the inductive bias. TrimNN’s effectiveness depends on its ability to (1) simplify the NP-complete subgraph matching problem on the universal graph to a well-defined set of biologically meaningful triangulated graphs; (2) decompose the challenging isomorphic counting regression problem on the entire graph into many straightforward binary present/absent prediction problems on small graphs, which makes it possible to estimate biologically meaningful top overrepresented CC motifs with relative values; (3) use PE-based graph representation methods to empower the expressive power of the model in spatial omics; and (4) use the supervised learning approach to move the computational tasks to the training process to accelerate the inference to facilitate users’ practice. This work paves the way for disclosing the biological mechanisms underlying multicellular differentiation, development, and disease progression.</p>
<p id="Par40">Biologically, TrimNN opens new opportunities for discovering complex mechanisms in complex diseases using spatial omics. The idea of CC motifs can be treated as an explicit representation that simplifies the cell organization and preserves the topological relations in the context of triangles derived from Delaunay triangulation, a fast and reliable mathematical process on spatial spaces as a CC. Using an interpretable LR model in the CRC study, the results of TrimNN robustly differentiate phenotypical and pathological characteristics of spatial omics samples. With TrimNN, we identified diverse CC motifs in spatial localization and cellular, gene, and pathway features in AD and colorectal carcinoma studies, corresponding to diverse biological and pathological mechanisms in complex diseases, which were often overlooked by conventional analysis.</p>
<p id="Par41">Furthermore, finding CC motifs is also related to mining FTUs at the atlas level. For example, identifying tumor lysis syndrome<sup><a href="#CR4" class="usa-link" aria-describedby="CR4">4</a></sup> in CNs in cancer research helps to illustrate how the immune microenvironment plays a role in cancer progression. Using topology-based cell type combinations in the spatial space, the abundance of CC motifs describes the dispersed and coherent cell organizations as shown in a demo system in Supplementary Fig. <a href="#MOESM1" class="usa-link">36</a>. In complex systems such as the tumor microenvironment, these abundances can be used to describe the cell type boundary and gradient. Especially, the abundance of specific cell types of tumor-immune mixing is also known to distinguish different organization scenarios, such as a cold tumor, mixed tumor, and compartmentalized tumor, which are directly related to survival in triple-negative breast cancer<sup><a href="#CR52" class="usa-link" aria-describedby="CR52">52</a></sup>.</p>
<p id="Par42">Finally, we note that both the theory and practice of determining the optimal spatial scale for describing observed data in spatial biology remain in their early stages. We comprehensively investigated the relationship between motif size and its impact on performance in Supplementary Note 4, which includes experiments exploring the robustness simulation to size-5 (Fig. <a href="#Fig2" class="usa-link">2</a>A–C), size-12 in the context of AD research (Supplementary Figs. <a href="#MOESM1" class="usa-link">37</a>–<a href="#MOESM1" class="usa-link">39</a> and Supplementary Data <a href="#MOESM3" class="usa-link">69</a>–<a href="#MOESM3" class="usa-link">71</a>), and size-7 motifs in a demo system (Supplementary Fig. <a href="#MOESM1" class="usa-link">40</a> and Supplementary Data <a href="#MOESM3" class="usa-link">72</a>). From the limited experiments and observations, there may not be one optimal size of CC motifs that uniformly describes CCs within the complex biological context revealed from the spatial omics. The selection of an optimal motif size requires extensive statistical studies, which will be our future work. For example, if a reliable approach can be established to assess the <em>p</em>-value of each motif between case and control, the motifs with the smallest <em>p</em>-values should be used as motif markers.</p>
<p id="Par43">In practice, there are trade-offs between large and small sizes of CC motifs in presenting the characteristics of CNs in spatial biology. (1) Complex large-size CC motifs may demonstrate better prediction power, but they are hard to identify. In the case study of CRC, we confirmed that size-3 CC motifs perform better in classification than size-1 CC motifs. However, the number of motifs increases exponentially with motif size, which makes identifying the most overrepresented motifs in large sizes computationally expensive. (2) CC motifs in large sizes have similar biological interpretability to smaller sizes but lack an established statistical framework in scientific rigor. There are no limits on size in order to use biological interpretation approaches on these motifs. However, to the best of our knowledge, no single statistical test is comparable to Fisher’s exact test (or the chi-squared test) or the CMH test that universally applies to four-way or higher-dimensional contingency tables. (<a href="#MOESM1" class="usa-link">3</a>) More caution needs to be taken when applying complex large-size CC motifs in machine learning because of overfitting. When applying these motifs to current spatial omics datasets with a limited number of samples, users need to be aware of selecting the appropriate number of top CC motifs as features in order to perform machine learning tasks to prevent overfitting.</p>
<p id="Par44">This work still has some limitations. First, TrimNN enumerates the possible graph topologies and greedy search strategy to identify the large-size motifs, which has room for improvement with deep reinforcement learning approaches. Second, TrimNN still estimates the relative rankings in abundance and cannot be guaranteed to be exact. Last, current settings in CCs may oversimplify the problem without any features on the edges and nodes. We may add essential features (e.g., morphology features) for specific applications. Future work will include analyzing large-scale spatial omics data to connect the idea of CC motifs and FTU, developing systematic evaluation metrics in multiple sizes of CC motifs, and assessing the results in different categories of diseases from multiple independent data sources.</p></section><section id="Sec9"><h2 class="pmc_sec_title">Methods</h2>
<section id="Sec10"><h3 class="pmc_sec_title">Problem setting</h3>
<p id="Par45">Formally, we define the triangulated graph <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq4"><math id="d33e1585"><mi>G</mi></math></span> as the CC inferred from spatial omics using Delaunay triangulation<sup><a href="#CR13" class="usa-link" aria-describedby="CR13">13</a></sup>, where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq5"><math id="d33e1598"><mi>G</mi><mo>=</mo><mrow><mo>{</mo><mrow><mi>V</mi><mo>,</mo><mi>E</mi></mrow><mo>}</mo></mrow></math></span> with <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq6"><math id="d33e1615"><mfenced close="∣" open="∣"><mrow><mi>V</mi></mrow></mfenced><mo>=</mo><mi>n</mi></math></span> nodes and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq7"><math id="d33e1628"><mfenced close="∣" open="∣"><mrow><mi>E</mi></mrow></mfenced></math></span> edges. The size-<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq8"><math id="d33e1640"><mi>k</mi></math></span> CC motif is a subgraph with <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq9"><math id="d33e1649"><mi>k</mi></math></span> nodes as <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq10"><math id="d33e1658"><msub><mrow><mi>m</mi></mrow><mrow><mi>k</mi></mrow></msub></math></span>, where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq11"><math id="d33e1671"><msub><mrow><mi>m</mi></mrow><mrow><mi>k</mi></mrow></msub></math></span> is an induced subgraph of <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq12"><math id="d33e1684"><mi>G</mi></math></span>. Here, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq13"><math id="d33e1693"><msub><mrow><mi>m</mi></mrow><mrow><mi>k</mi></mrow></msub><mo>=</mo><mfenced close=")" open="("><mrow><msup><mrow><mi>V</mi></mrow><mrow><mo>′</mo></mrow></msup><mo>,</mo><msup><mrow><mi>E</mi></mrow><mrow><mo>′</mo></mrow></msup></mrow></mfenced></math></span> is defined as an induced subgraph if and only if when <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq14"><math id="d33e1721"><msup><mrow><mi>V</mi></mrow><mrow><mo>′</mo></mrow></msup><mo>⊆</mo><mi>V</mi></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq15"><math id="d33e1736"><msup><mrow><mi>E</mi></mrow><mrow><mo>′</mo></mrow></msup><mo>=</mo><mrow><mo>{</mo><mrow><mrow><mo>(</mo><mrow><mi>u</mi><mo>,</mo><mi>v</mi></mrow><mo>)</mo></mrow><mo>∈</mo><mi>E</mi><mo>∣</mo><mi>u</mi><mo>,</mo><mi>v</mi><mo>∈</mo><msup><mrow><mi>V</mi></mrow><mrow><mo>′</mo></mrow></msup></mrow><mo>}</mo></mrow></math></span>. <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq16"><math id="d33e1773"><msub><mrow><mi>M</mi></mrow><mrow><mi>k</mi></mrow></msub></math></span> is the set of all <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq17"><math id="d33e1786"><msub><mrow><mi>m</mi></mrow><mrow><mi>k</mi></mrow></msub></math></span> of size-<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq18"><math id="d33e1799"><mi>k</mi></math></span>, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq19"><math id="d33e1808"><msub><mrow><mi>M</mi></mrow><mrow><mi>k</mi></mrow></msub><mo>⊆</mo><mi>G</mi></math></span>. The biological problem of identifying the overrepresented CC motifs can be modeled mathematically in finding the most overrepresented subgraph <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq20"><math id="d33e1824"><msub><mrow><mi>m</mi></mrow><mrow><mi>k</mi></mrow></msub><mspace width="0.3em"></mspace><mo>*</mo></math></span> in <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq21"><math id="d33e1839"><mi>G</mi></math></span>, where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq22"><math id="d33e1848"><msub><mrow><mi>m</mi></mrow><mrow><mi>k</mi></mrow></msub><mspace width="0.3em"></mspace><mo>*</mo><mo>⊆</mo><mi>G</mi></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq23"><math id="d33e1865"><msub><mrow><mi>m</mi></mrow><mrow><mi>k</mi></mrow></msub><mspace width="0.3em"></mspace><mo>*</mo><mo>∈</mo><msub><mrow><mi>M</mi></mrow><mrow><mi>k</mi></mrow></msub></math></span>. This challenge consists of a subgraph matching problem and a pattern growth problem built on it.</p>
<p id="Par46">TrimNN aims to address the subgraph matching problem in the context of spatial omics, which seeks to define a function <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq24"><math id="d33e1888"><mi>F</mi><mfenced close=")" open="("><mrow><mi>G</mi><mo>,</mo><msub><mrow><mi>m</mi></mrow><mrow><mi>k</mi></mrow></msub></mrow></mfenced><mi mathvariant="double-struck">\in</mi><mi mathvariant="double-struck">N</mi></math></span>, estimating the relative occurrence of the given <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq25"><math id="d33e1910"><msub><mrow><mi>m</mi></mrow><mrow><mi>k</mi></mrow></msub></math></span> in <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq26"><math id="d33e1923"><mi>G</mi></math></span>. In our setting, this problem can be semi divided and conquered by summarizing many sub-TrimNN problems. The goal of sub-TrimNN is to build a reliable binary prediction model <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq27"><math id="d33e1932"><mi>f</mi><mfenced close=")" open="("><mrow><mi>g</mi><mo>,</mo><msub><mrow><mi>m</mi></mrow><mrow><mi>k</mi></mrow></msub></mrow></mfenced><mo>∈</mo><mrow><mo>[</mo><mrow><mi mathvariant="normal">0, 1</mi></mrow><mo>]</mo></mrow></math></span>, where 0 represents that <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq28"><math id="d33e1957"><msub><mrow><mi>m</mi></mrow><mrow><mi>k</mi></mrow></msub></math></span> is absent in graph <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq29"><math id="d33e1971"><mi>g</mi></math></span> and 1 represents presence, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq30"><math id="d33e1980"><mi>g</mi><mo>⊆</mo><mi>G</mi></math></span>. With sub-TrimNN on enclosed graphs centered by each node, TrimNN is the summarization of results from all sub-TrimNN in the graph, as Eq. (<a href="#Equ1" class="usa-link">1</a>):</p>
<table class="disp-formula p" id="Equ1"><tr>
<td class="formula"><math id="d33e1995" display="block"><mi>F</mi><mfenced close=")" open="("><mrow><mi>G</mi><mo>,</mo><msub><mrow><mi>m</mi></mrow><mrow><mi>k</mi></mrow></msub></mrow></mfenced><mo>=</mo><msub><mrow><mo mathsize="big">∑</mo></mrow><mrow><mi>v</mi><mo>∈</mo><mi>G</mi></mrow></msub><mspace width="0.25em"></mspace><mi>f</mi><mfenced close=")" open="("><mrow><mi>g</mi><mfenced close=")" open="("><mrow><mi>v</mi><mo>,</mo><mi>h</mi><mi>o</mi><mi>p</mi></mrow></mfenced><mo>,</mo><msub><mrow><mi>m</mi></mrow><mrow><mi>k</mi></mrow></msub></mrow></mfenced></math></td>
<td class="label">1</td>
</tr></table>
<p>where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq31"><math id="d33e2040"><mi>g</mi><mrow><mo>(</mo><mrow><mi>v</mi><mo>,</mo><mi>h</mi><mi>o</mi><mi>p</mi></mrow><mo>)</mo></mrow></math></span> is the enclosed graph as the neighborhoods of node <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq32"><math id="d33e2058"><mi>v</mi><mo>∈</mo><mi>V</mi></math></span> with <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq33"><math id="d33e2070"><mi>h</mi><mi>o</mi><mi>p</mi><mo>∈</mo><mrow><mo>[</mo><mrow><mi mathvariant="normal">1, 2, 3</mi><mo>,</mo><mo>…</mo></mrow><mo>]</mo></mrow></math></span>, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq34"><math id="d33e2090"><mi>g</mi><mrow><mo>(</mo><mrow><mi>v</mi><mo>,</mo><mi>h</mi><mi>o</mi><mi>p</mi></mrow><mo>)</mo></mrow><mo>⊆</mo><mi>G</mi></math></span>. Generally, the value of <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq35"><math id="d33e2110"><mi>h</mi><mi>o</mi><mi>p</mi></math></span> is related to the length of the longest path of <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq36"><math id="d33e2121"><msub><mrow><mi>m</mi></mrow><mrow><mi>k</mi></mrow></msub></math></span>. Here, we use <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq37"><math id="d33e2134"><mi>h</mi><mi>o</mi><mi>p</mi><mo>=</mo><mn>2</mn></math></span> in all the analyses to make enclosed graphs <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq38"><math id="d33e2147"><mi>g</mi></math></span> is in a similar size of <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq39"><math id="d33e2157"><msub><mrow><mi>m</mi></mrow><mrow><mi>k</mi></mrow></msub></math></span>.</p>
<p id="Par47">We use a fast and reliable <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq40"><math id="d33e2172"><mi>F</mi><mfenced close=")" open="("><mrow><mi>G</mi><mo>,</mo><msub><mrow><mi>m</mi></mrow><mrow><mi>k</mi></mrow></msub></mrow></mfenced></math></span> from TrimNN to address the problem of pattern growth. Using searching processes, the goal is to get a top overrepresented set <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq41"><math id="d33e2190"><msub><mrow><mi>m</mi></mrow><mrow><mi>k</mi></mrow></msub><mspace width="0.3em"></mspace><mo>*</mo></math></span> has the maximum relative abundance in Eq. (<a href="#Equ2" class="usa-link">2</a>):</p>
<table class="disp-formula p" id="Equ2"><tr>
<td class="formula"><math id="d33e2209" display="block"><mi>F</mi><mfenced close=")" open="("><mrow><mi>G</mi><mo>,</mo><msub><mrow><mi>m</mi></mrow><mrow><mi>k</mi></mrow></msub><mo>*</mo></mrow></mfenced><mo>=</mo><msub><mrow><mi>max</mi></mrow><mrow><msub><mrow><mi>m</mi></mrow><mrow><mi>k</mi></mrow></msub><mo>∈</mo><msub><mrow><mi>M</mi></mrow><mrow><mi>k</mi></mrow></msub></mrow></msub><mfenced close=")" open="("><mrow><mi>F</mi><mrow><mo>(</mo><mrow><mi>G</mi><mo>,</mo><msub><mrow><mi>m</mi></mrow><mrow><mi>k</mi></mrow></msub></mrow><mo>)</mo></mrow></mrow></mfenced></math></td>
<td class="label">2</td>
</tr></table>
<p id="Par48">If both case and control samples are available, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq42"><math id="d33e2259"><mi>F</mi><mo>′</mo><mfenced close=")" open="("><mrow><msub><mrow><mi>G</mi></mrow><mrow><mi>c</mi><mi>a</mi><mi>s</mi><mi>e</mi></mrow></msub><mo>,</mo><msub><mrow><mi>G</mi></mrow><mrow><mi>c</mi><mi>o</mi><mi>n</mi><mi>t</mi><mi>r</mi><mi>o</mi><mi>l</mi></mrow></msub><mo>,</mo><msub><mrow><mi>m</mi></mrow><mrow><mi>k</mi></mrow></msub></mrow></mfenced></math></span> is defined to find CC motif <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq43"><math id="d33e2297"><msub><mrow><mi>m</mi></mrow><mrow><mi>k</mi></mrow></msub></math></span> that differentiates <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq44"><math id="d33e2310"><msub><mrow><mi>G</mi></mrow><mrow><mi>c</mi><mi>a</mi><mi>s</mi><mi>e</mi></mrow></msub></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq45"><math id="d33e2326"><msub><mrow><mi>G</mi></mrow><mrow><mi>c</mi><mi>o</mi><mi>n</mi><mi>t</mi><mi>r</mi><mi>o</mi><mi>l</mi></mrow></msub></math></span> in Eq. (<a href="#Equ3" class="usa-link">3</a>), where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq46"><math id="d33e2349"><msub><mrow><mi>G</mi></mrow><mrow><mi>c</mi><mi>a</mi><mi>s</mi><mi>e</mi></mrow></msub></math></span> represents CC from case samples and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq47"><math id="d33e2365"><msub><mrow><mi>G</mi></mrow><mrow><mi>c</mi><mi>o</mi><mi>n</mi><mi>t</mi><mi>r</mi><mi>o</mi><mi>l</mi></mrow></msub></math></span> represents CC from control samples. <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq48"><math id="d33e2384"><mi>F</mi><mo>′</mo></math></span> can be any function to describe the differences, including Fisher’s exact test or effective size. <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq49"><math id="d33e2394"><msub><mrow><mi>m</mi></mrow><mrow><mi>k</mi></mrow></msub><mspace width="0.3em"></mspace><mo>*</mo></math></span> is the top overrepresented set mostly differentiated conditions in Eq. (<a href="#Equ4" class="usa-link">4</a>).</p>
<table class="disp-formula p" id="Equ3"><tr>
<td class="formula"><math id="d33e2413" display="block"><mi>F</mi><mo>′</mo><mfenced close=")" open="("><mrow><msub><mrow><mi>G</mi></mrow><mrow><mi>c</mi><mi>a</mi><mi>s</mi><mi>e</mi></mrow></msub><mo>,</mo><msub><mrow><mi>G</mi></mrow><mrow><mi>c</mi><mi>o</mi><mi>n</mi><mi>t</mi><mi>r</mi><mi>o</mi><mi>l</mi></mrow></msub><mo>,</mo><msub><mrow><mi>m</mi></mrow><mrow><mi>k</mi></mrow></msub></mrow></mfenced><mo>=</mo><msup><mrow><mi>F</mi></mrow><mrow><mo>′</mo></mrow></msup><mfenced close=")" open="("><mrow><mi>F</mi><mrow><mo>(</mo><mrow><msub><mrow><mi>G</mi></mrow><mrow><mi>c</mi><mi>a</mi><mi>s</mi><mi>e</mi></mrow></msub><mo>,</mo><msub><mrow><mi>m</mi></mrow><mrow><mi>k</mi></mrow></msub></mrow><mo>)</mo></mrow><mo>,</mo><mi>F</mi><mrow><mo>(</mo><mrow><msub><mrow><mi>G</mi></mrow><mrow><mi>c</mi><mi>o</mi><mi>n</mi><mi>t</mi><mi>r</mi><mi>o</mi><mi>l</mi></mrow></msub><mo>,</mo><msub><mrow><mi>m</mi></mrow><mrow><mi>k</mi></mrow></msub></mrow><mo>)</mo></mrow></mrow></mfenced></math></td>
<td class="label">3</td>
</tr></table>
<table class="disp-formula p" id="Equ4"><tr>
<td class="formula"><math id="d33e2502" display="block"><mi>F</mi><mo>′</mo><mfenced close=")" open="("><mrow><msub><mrow><mi>G</mi></mrow><mrow><mi>c</mi><mi>a</mi><mi>s</mi><mi>e</mi></mrow></msub><mo>,</mo><msub><mrow><mi>G</mi></mrow><mrow><mi>c</mi><mi>o</mi><mi>n</mi><mi>t</mi><mi>r</mi><mi>o</mi><mi>l</mi></mrow></msub><mo>,</mo><msub><mrow><mi>m</mi></mrow><mrow><mi>k</mi></mrow></msub><mspace width="0.3em"></mspace><mo>*</mo></mrow></mfenced><mo>=</mo><msub><mrow><mi>max</mi></mrow><mrow><msub><mrow><mi>m</mi></mrow><mrow><mi>k</mi></mrow></msub><mo>∈</mo><msub><mrow><mi>M</mi></mrow><mrow><mi>k</mi></mrow></msub></mrow></msub><mfenced close=")" open="("><mrow><msup><mrow><mi>F</mi></mrow><mrow><mo>′</mo></mrow></msup><mrow><mo>(</mo><mrow><msub><mrow><mi>G</mi></mrow><mrow><mi>c</mi><mi>a</mi><mi>s</mi><mi>e</mi></mrow></msub><mo>,</mo><msub><mrow><mi>G</mi></mrow><mrow><mi>c</mi><mi>o</mi><mi>n</mi><mi>t</mi><mi>r</mi><mi>o</mi><mi>l</mi></mrow></msub><mo>,</mo><msub><mrow><mi>m</mi></mrow><mrow><mi>k</mi></mrow></msub></mrow><mo>)</mo></mrow></mrow></mfenced></math></td>
<td class="label">4</td>
</tr></table></section><section id="Sec11"><h3 class="pmc_sec_title">TrimNN in subgraph matching</h3>
<p id="Par49">We decomposed the regression problem of TrimNN in the CC into many binary classification problems in enclosed graphs centered by each node of the triangulated graph. This classification problem on each enclosed graph is solved by sub-TrimNN. The input of sub-TrimNN is a pair of subgraphs of query <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq50"><math id="d33e2598"><msub><mrow><mi>m</mi></mrow><mrow><mi>k</mi></mrow></msub></math></span> and the target triangulated graph <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq51"><math id="d33e2611"><mi>g</mi></math></span>. To better represent the topological information of subgraphs and graphs, we use an empowered GNN based on GIN and a shortest distance positional encoding <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq52"><math id="d33e2620"><mi>P</mi><mi>E</mi></math></span>, which can be denoted as Eq. (<a href="#Equ5" class="usa-link">5</a>):</p>
<table class="disp-formula p" id="Equ5"><tr>
<td class="formula"><math id="d33e2634" display="block"><msubsup><mrow><mi>h</mi></mrow><mrow><mi>v</mi></mrow><mrow><mfenced close=")" open="("><mrow><mi>l</mi></mrow></mfenced></mrow></msubsup><mo>=</mo><msup><mrow><mi mathvariant="normal">MLP</mi></mrow><mrow><mfenced close=")" open="("><mrow><mi>l</mi></mrow></mfenced></mrow></msup><mrow><mo>(</mo><mrow><mrow><mo>(</mo><mrow><mn>1</mn><mo>+</mo><msup><mrow><mi>ε</mi></mrow><mrow><mfenced close=")" open="("><mrow><mi>l</mi></mrow></mfenced></mrow></msup></mrow><mo>)</mo></mrow><mo>*</mo><msubsup><mrow><mi>h</mi></mrow><mrow><mi>v</mi></mrow><mrow><mfenced close=")" open="("><mrow><mi>l</mi><mo>−</mo><mn>1</mn></mrow></mfenced></mrow></msubsup><mo>+</mo><msub><mrow><mo mathsize="big">∑</mo></mrow><mrow><mi>u</mi><mo>∈</mo><mi>N</mi><mfenced close=")" open="("><mrow><mi>v</mi></mrow></mfenced></mrow></msub><msubsup><mrow><mi>h</mi></mrow><mrow><mi>u</mi></mrow><mrow><mfenced close=")" open="("><mrow><mi>l</mi><mo>−</mo><mn>1</mn></mrow></mfenced></mrow></msubsup><mo>+</mo><mi>a</mi><mo>*</mo><mi>P</mi><mi>E</mi></mrow><mo>)</mo></mrow></math></td>
<td class="label">5</td>
</tr></table>
<p>where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq53"><math id="d33e2718"><msubsup><mrow><mi>h</mi></mrow><mrow><mi>v</mi></mrow><mrow><mfenced close=")" open="("><mrow><mi>l</mi></mrow></mfenced></mrow></msubsup></math></span> is the learned embedding of node <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq54"><math id="d33e2735"><mi>v</mi></math></span> at the <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq55"><math id="d33e2744"><mi>l</mi></math></span>th layer. MLPs are multi-layer perceptrons, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq56"><math id="d33e2753"><mi>ε</mi></math></span> is a fixed scalar and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq57"><math id="d33e2762"><mi>N</mi><mfenced close=")" open="("><mrow><mi>v</mi></mrow></mfenced></math></span> is a set of nodes adjacent to <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq58"><math id="d33e2774"><mi>v</mi></math></span>, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq59"><math id="d33e2784"><mi>a</mi></math></span> is the scaling factor for controlling the strength of <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq60"><math id="d33e2793"><mi>P</mi><mi>E</mi></math></span>. Here, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq61"><math id="d33e2803"><mi>P</mi><mi>E</mi></math></span> as the shortest distance positional encoding<sup><a href="#CR24" class="usa-link" aria-describedby="CR24">24</a></sup> is adapted to encode the relative positions of nodes in a graph based on the shortest path distances between them, described as Eq. (<a href="#Equ6" class="usa-link">6</a>):</p>
<table class="disp-formula p" id="Equ6"><tr>
<td class="formula"><math id="d33e2821" display="block"><mi>P</mi><mi>E</mi><mo>=</mo><mfenced close="]" open="["><mrow><mi>d</mi><mfenced close=")" open="("><mrow><msub><mrow><mi>v</mi></mrow><mrow><mn>0</mn></mrow></msub><mo>,</mo><msub><mrow><mi>v</mi></mrow><mrow><mn>0</mn></mrow></msub></mrow></mfenced><mo>,</mo><mi>d</mi><mfenced close=")" open="("><mrow><msub><mrow><mi>v</mi></mrow><mrow><mn>0</mn></mrow></msub><mo>,</mo><msub><mrow><mi>v</mi></mrow><mrow><mn>1</mn></mrow></msub></mrow></mfenced><mo>,</mo><mi>d</mi><mfenced close=")" open="("><mrow><msub><mrow><mi>v</mi></mrow><mrow><mn>0</mn></mrow></msub><mo>,</mo><msub><mrow><mi>v</mi></mrow><mrow><mn>2</mn></mrow></msub></mrow></mfenced><mo>,</mo><mo>…</mo><mo>,</mo><mi>d</mi><mfenced close=")" open="("><mrow><msub><mrow><mi>v</mi></mrow><mrow><mn>0</mn></mrow></msub><mo>,</mo><msub><mrow><mi>v</mi></mrow><mrow><mi>n</mi></mrow></msub></mrow></mfenced></mrow></mfenced></math></td>
<td class="label">6</td>
</tr></table>
<p>where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq62"><math id="d33e2897"><mi>d</mi><mrow><mo>(</mo><mrow><mi>u</mi><mo>,</mo><mi>v</mi></mrow><mo>)</mo></mrow></math></span> denotes the shortest distance between two nodes <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq63"><math id="d33e2913"><mi>u</mi></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq64"><math id="d33e2922"><mi>v</mi></math></span>. For all nodes <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq65"><math id="d33e2931"><mi>v</mi></math></span> in the graph <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq66"><math id="d33e2940"><mi>g</mi></math></span>, we first selected one endpoint <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq67"><math id="d33e2949"><msub><mrow><mi>v</mi></mrow><mrow><mn>0</mn></mrow></msub></math></span> of the shortest path in the whole graph <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq68"><math id="d33e2963"><mi>g</mi></math></span> as the starting point. Then the shortest distance from this point was calculated for all other vertices.</p>
<p id="Par50">After obtaining <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq69"><math id="d33e2974"><mi>P</mi><mi>E</mi></math></span> for the paired query <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq70"><math id="d33e2984"><msub><mrow><mi>m</mi></mrow><mrow><mi>k</mi></mrow></msub></math></span> and target <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq71"><math id="d33e2997"><mi>g</mi></math></span>, we extended the positional encoding of each node to match the dimensionality of the learned graph embeddings from GIN and then added them together. Then we passed learned node representations through a graph max pooling layer<sup><a href="#CR53" class="usa-link" aria-describedby="CR53">53</a></sup> to get the graph representations. After the sigmoid function activates linear layers, sub-TrimNN outputs the binary predictions: <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq72"><math id="d33e3010"><mn>1</mn></math></span> as presence and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq73"><math id="d33e3020"><mn>0</mn></math></span> as absence. The whole training process aims to minimize the cross-entropy loss function of known presence/absence relations as Eq. (<a href="#Equ7" class="usa-link">7</a>):</p>
<table class="disp-formula p" id="Equ7"><tr>
<td class="formula"><math id="d33e3033" display="block"><mi>L</mi><mo>=</mo><mo>−</mo><mfrac><mrow><mn>1</mn></mrow><mrow><mi>N</mi></mrow></mfrac><msubsup><mrow><mo mathsize="big">∑</mo></mrow><mrow><mi>i</mi><mo>=</mo><mn>1</mn></mrow><mrow><mi>N</mi></mrow></msubsup><mstyle mathsize="1.61em"><mfenced open="["><mrow></mrow></mfenced></mstyle><msub><mrow><mi>y</mi></mrow><mrow><mi>i</mi></mrow></msub><mi>log</mi><mfenced close=")" open="("><mrow><msub><mrow><mover accent="true"><mrow><mi>y</mi></mrow><mrow><mo>^</mo></mrow></mover></mrow><mrow><mi>i</mi></mrow></msub></mrow></mfenced><mo>+</mo><mrow><mo>(</mo><mrow><mn>1</mn><mo>−</mo><msub><mrow><mi>y</mi></mrow><mrow><mi>i</mi></mrow></msub></mrow><mo>)</mo></mrow><mi>log</mi><mfenced close=")" open="("><mrow><msub><mrow><mn>1</mn><mo>−</mo><mover accent="true"><mrow><mi>y</mi></mrow><mrow><mo>^</mo></mrow></mover></mrow><mrow><mi>i</mi></mrow></msub></mrow></mfenced><mstyle mathsize="1.61em"><mfenced open="]"><mrow></mrow></mfenced></mstyle></math></td>
<td class="label">7</td>
</tr></table>
<p>where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq74"><math id="d33e3109"><mi>N</mi></math></span> is the number of samples, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq75"><math id="d33e3118"><msub><mrow><mi>y</mi></mrow><mrow><mi>i</mi></mrow></msub></math></span> is the true label for the <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq76"><math id="d33e3131"><mi>i</mi></math></span>-th sample (0 or 1), and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq77"><math id="d33e3141"><msub><mrow><mover accent="true"><mrow><mi>y</mi></mrow><mrow><mo>^</mo></mrow></mover></mrow><mrow><mi>i</mi></mrow></msub></math></span> is the predicted probability for the <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq78"><math id="d33e3158"><mi>i</mi></math></span>-th sample.</p>
<p id="Par51">After trained sub-TrimNN <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq79"><math id="d33e3169"><mi>f</mi><mfenced close=")" open="("><mrow><mi>g</mi><mo>,</mo><msub><mrow><mi>m</mi></mrow><mrow><mi>k</mi></mrow></msub></mrow></mfenced></math></span>, TrimNN estimates the abundances of <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq80"><math id="d33e3187"><mi>F</mi><mfenced close=")" open="("><mrow><mi>G</mi><mo>,</mo><msub><mrow><mi>m</mi></mrow><mrow><mi>k</mi></mrow></msub></mrow></mfenced></math></span> by summarizing sub-TrimNN predictions on each node’s enclosed graph as Eq. (<a href="#Equ1" class="usa-link">1</a>).</p>
<p id="Par52">Theoretically, both the time and space complexity of sub-TrimNN inference are <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq81"><math id="d33e3210"><mi>O</mi><mrow><mo>(</mo><mrow><mo>∣</mo><mi>V</mi><mo>∣</mo><mo>+</mo><mi>k</mi></mrow><mo>)</mo></mrow></math></span>, which is linear to the node sizes of the input subgraph and the triangulated graph. The time complexity of the entire TrimNN inference is the graph node size multiples the sub-TrimNN on all the enclosed graphs, which is <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq82"><math id="d33e3228"><mi>O</mi><mrow><mo>(</mo><mrow><mfenced close="∣" open="∣"><mrow><mi>V</mi></mrow></mfenced><mo>*</mo><mrow><mo>(</mo><mrow><mi>h</mi><mi>o</mi><mi>p</mi><mo>*</mo><mi>k</mi><mo>+</mo><mi>k</mi></mrow><mo>)</mo></mrow></mrow><mo>)</mo></mrow></math></span>, and the time complexity of building an enclosed graph is <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq83"><math id="d33e3256"><mi>O</mi><mrow><mo>(</mo><mrow><msup><mrow><mfenced close="∣" open="∣"><mrow><mi>V</mi></mrow></mfenced></mrow><mrow><mi>h</mi><mi>o</mi><mi>p</mi></mrow></msup></mrow><mo>)</mo></mrow></math></span>. The space complexity of the entire TrimNN inference is <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq84"><math id="d33e3278"><mi>O</mi><mrow><mo>(</mo><mrow><msup><mrow><mfenced close="∣" open="∣"><mrow><mi>V</mi></mrow></mfenced></mrow><mrow><mi>h</mi><mi>o</mi><mi>p</mi></mrow></msup><mo>*</mo><mrow><mo>(</mo><mrow><mi>h</mi><mi>o</mi><mi>p</mi><mo>*</mo><mi>k</mi><mo>+</mo><mi>k</mi></mrow><mo>)</mo></mrow></mrow><mo>)</mo></mrow></math></span>, where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq85"><math id="d33e3312"><mi>h</mi><mi>o</mi><mi>p</mi></math></span> is defined in generating the enclosed graph. On the other hand, the space complexity of VF2 is of order <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq86"><math id="d33e3324"><mi>O</mi><mrow><mo>(</mo><mrow><mi>V</mi></mrow><mo>)</mo></mrow></math></span>, and the time complexity is <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq87"><math id="d33e3338"><mi>O</mi><mrow><mo>(</mo><mrow><mi>V</mi><mo>!</mo><mo>*</mo><mi>V</mi></mrow><mo>)</mo></mrow></math></span>.</p></section><section id="Sec12"><h3 class="pmc_sec_title">Greedy strategy in pattern growth</h3>
<p id="Par53">In the pattern growth process, we use the function <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq88"><math id="d33e3359"><mi>F</mi><mfenced close=")" open="("><mrow><mi>G</mi><mo>,</mo><msub><mrow><mi>m</mi></mrow><mrow><mi>k</mi></mrow></msub></mrow></mfenced></math></span> from the subgraph matching in Eq. (<a href="#Equ2" class="usa-link">2</a>) to find <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq89"><math id="d33e3380"><msub><mrow><mi>m</mi></mrow><mrow><mi>K</mi></mrow></msub><mo>*</mo></math></span> for a specified size <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq90"><math id="d33e3394"><mi>K</mi></math></span> with a serial of greedy search. Starting from small sizes of CC motifs of size-<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq91"><math id="d33e3403"><mi>k</mi></math></span>, where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq92"><math id="d33e3413"><mi>k</mi><mo>=</mo><mi mathvariant="normal">1, 2</mi><mo>,</mo><mi>o</mi><mi>r</mi><mn>3</mn></math></span>, we enumerate all possible subgraphs <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq93"><math id="d33e3429"><msub><mrow><mi>m</mi></mrow><mrow><mi>k</mi></mrow></msub></math></span> and then obtain their corresponding predicted occurrence values using trained <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq94"><math id="d33e3442"><mi>F</mi><mfenced close=")" open="("><mrow><mi>G</mi><mo>,</mo><msub><mrow><mi>m</mi></mrow><mrow><mi>k</mi></mrow></msub></mrow></mfenced></math></span>. For studies with case and control conditions, we calculate the <em>p</em>-value of Fisher’s exact test to identify the most significant CC motifs between different conditions. If no case–control information is available, we select the subgraph with the maximum relative abundance as <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq95"><math id="d33e3463"><msub><mrow><mi>m</mi></mrow><mrow><mi>k</mi></mrow></msub><mspace width="0.3em"></mspace><mo>*</mo></math></span>. After obtaining <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq96"><math id="d33e3478"><msub><mrow><mi>m</mi></mrow><mrow><mi>k</mi></mrow></msub><mspace width="0.3em"></mspace><mo>*</mo></math></span> at size-<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq97"><math id="d33e3494"><mi>k</mi></math></span>, we use it as a seed to enumerate all possible size-<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq98"><math id="d33e3503"><mi>k</mi><mo>+</mo><mn>1</mn></math></span> subgraphs <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq99"><math id="d33e3514"><msub><mrow><mi>M</mi></mrow><mrow><mi>k</mi><mo>+</mo><mn>1</mn></mrow></msub></math></span> based on <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq100"><math id="d33e3529"><msub><mrow><mi>m</mi></mrow><mrow><mi>k</mi></mrow></msub><mspace width="0.3em"></mspace><mo>*</mo></math></span>.</p>
<table class="disp-formula p" id="Equ8"><tr>
<td class="formula"><math id="d33e3545" display="block"><msub><mrow><mi>M</mi></mrow><mrow><mi>k</mi><mo>+</mo><mn>1</mn></mrow></msub><mo>=</mo><msub><mrow><mo mathsize="big">⋃</mo></mrow><mrow><mi>v</mi><mo>∈</mo><mi>N</mi><mfenced close=")" open="("><mrow><msub><mrow><mi>m</mi></mrow><mrow><mi>k</mi></mrow></msub><mspace width="0.3em"></mspace><mo>*</mo></mrow></mfenced><mo>,</mo><mi>e</mi><mo>∈</mo><mi>E</mi><mfenced close=")" open="("><mrow><msub><mrow><mi>m</mi></mrow><mrow><mi>k</mi></mrow></msub><mspace width="0.3em"></mspace><mo>*</mo></mrow></mfenced></mrow></msub><mrow><mo>{</mo><mrow><msub><mrow><mi>m</mi></mrow><mrow><mi>k</mi></mrow></msub><mspace width="0.3em"></mspace><mo>*</mo><mo>,</mo><mi>v</mi><mo>,</mo><mi>e</mi></mrow><mo>}</mo></mrow></math></td>
<td class="label">8</td>
</tr></table>
<p>where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq101"><math id="d33e3606"><mi>E</mi><mfenced close=")" open="("><mrow><msub><mrow><mi>m</mi></mrow><mrow><mi>k</mi></mrow></msub><mspace width="0.3em"></mspace><mo>*</mo></mrow></mfenced></math></span> is a set of edges linked to the graph <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq102"><math id="d33e3625"><msub><mrow><mi>m</mi></mrow><mrow><mi>k</mi></mrow></msub><mspace width="0.3em"></mspace><mo>*</mo></math></span>. Each node type as a new node is selected with <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq103"><math id="d33e3640"><msub><mrow><mi>m</mi></mrow><mrow><mi>k</mi></mrow></msub><mspace width="0.3em"></mspace><mo>*</mo></math></span> to get a new size-<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq104"><math id="d33e3655"><mi>k</mi><mo>+</mo><mn>1</mn></math></span> graph. Similar to Eq. (<a href="#Equ1" class="usa-link">1</a>), <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq105"><math id="d33e3669"><msub><mrow><mi>m</mi></mrow><mrow><mi>k</mi><mo>+</mo><mn>1</mn></mrow></msub><mo>*</mo></math></span> is defined as Eq. (<a href="#Equ9" class="usa-link">9</a>) in <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq106"><math id="d33e3689"><msub><mrow><mi>M</mi></mrow><mrow><mi>k</mi><mo>+</mo><mn>1</mn></mrow></msub></math></span>.</p>
<table class="disp-formula p" id="Equ9"><tr>
<td class="formula"><math id="d33e3705" display="block"><mi>F</mi><mfenced close=")" open="("><mrow><mi>G</mi><mo>,</mo><msubsup><mrow><mi>m</mi></mrow><mrow><mi>k</mi><mo>+</mo><mn>1</mn></mrow><mrow><mo>*</mo></mrow></msubsup></mrow></mfenced><mo>=</mo><msub><mrow><mi>max</mi></mrow><mrow><msub><mrow><mi>m</mi></mrow><mrow><mi>k</mi><mo>+</mo><mn>1</mn></mrow></msub><mo>∈</mo><msub><mrow><mi>M</mi></mrow><mrow><mi>k</mi><mo>+</mo><mn>1</mn></mrow></msub></mrow></msub><mfenced close=")" open="("><mrow><mi>F</mi><mrow><mo>(</mo><mrow><mi>G</mi><mo>,</mo><msub><mrow><mi>m</mi></mrow><mrow><mi>k</mi><mo>+</mo><mn>1</mn></mrow></msub></mrow><mo>)</mo></mrow></mrow></mfenced></math></td>
<td class="label">9</td>
</tr></table>
<p id="Par54">If both case and control samples are available, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq107"><math id="d33e3764"><msubsup><mrow><mi>m</mi></mrow><mrow><mi>k</mi><mo>+</mo><mn>1</mn></mrow><mrow><mo>*</mo></mrow></msubsup></math></span> is defined as Eq. (<a href="#Equ10" class="usa-link">10</a>) in <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq108"><math id="d33e3784"><msub><mrow><mi>M</mi></mrow><mrow><mi>k</mi><mo>+</mo><mn>1</mn></mrow></msub></math></span>.</p>
<table class="disp-formula p" id="Equ10"><tr>
<td class="formula"><math id="d33e3800" display="block"><msup><mrow><mi>F</mi></mrow><mrow><mo>′</mo></mrow></msup><mfenced close=")" open="("><mrow><msub><mrow><mi>G</mi></mrow><mrow><mi>c</mi><mi>a</mi><mi>s</mi><mi>e</mi></mrow></msub><mo>,</mo><msub><mrow><mi>G</mi></mrow><mrow><mi>c</mi><mi>o</mi><mi>n</mi><mi>t</mi><mi>r</mi><mi>o</mi><mi>l</mi></mrow></msub><mo>,</mo><msubsup><mrow><mi>m</mi></mrow><mrow><mi>k</mi><mo>+</mo><mn>1</mn></mrow><mrow><mo>*</mo></mrow></msubsup></mrow></mfenced><mo>=</mo><msub><mrow><mi>max</mi></mrow><mrow><msub><mrow><mi>m</mi></mrow><mrow><mi>k</mi><mo>+</mo><mn>1</mn></mrow></msub><mo>∈</mo><msub><mrow><mi>M</mi></mrow><mrow><mi>k</mi><mo>+</mo><mn>1</mn></mrow></msub></mrow></msub><mfenced close=")" open="("><mrow><msup><mrow><mi>F</mi></mrow><mrow><mo>′</mo></mrow></msup><mrow><mo>(</mo><mrow><msub><mrow><mi>G</mi></mrow><mrow><mi>c</mi><mi>a</mi><mi>s</mi><mi>e</mi></mrow></msub><mo>,</mo><msub><mrow><mi>G</mi></mrow><mrow><mi>c</mi><mi>o</mi><mi>n</mi><mi>t</mi><mi>r</mi><mi>o</mi><mi>l</mi></mrow></msub><mo>,</mo><msub><mrow><mi>m</mi></mrow><mrow><mi>k</mi><mo>+</mo><mn>1</mn></mrow></msub></mrow><mo>)</mo></mrow></mrow></mfenced></math></td>
<td class="label">10</td>
</tr></table>
<p id="Par55">By iterating the process of Eq. (<a href="#Equ8" class="usa-link">8</a>) with incremental <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq109"><math id="d33e3908"><mi>k</mi></math></span>, we can find <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq110"><math id="d33e3917"><msub><mrow><mi>m</mi></mrow><mrow><mi>K</mi></mrow></msub><mo>*</mo></math></span> at specific size <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq111"><math id="d33e3931"><mi>K</mi></math></span> in Eq. (<a href="#Equ9" class="usa-link">9</a>) or Eq. (10) if case and control samples are available, where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq112"><math id="d33e3944"><mi>K</mi><mo>∈</mo><msup><mrow><mi mathvariant="double-struck">N</mi></mrow><mrow><mo>+</mo></mrow></msup></math></span>.</p></section><section id="Sec13"><h3 class="pmc_sec_title">Constructing the training dataset</h3>
<p id="Par56">In spatial omics samples, the spatial relations between the cells can be modeled as a CC in a cell graph using the Delaunay triangulation<sup><a href="#CR13" class="usa-link" aria-describedby="CR13">13</a></sup> on their spatial coordinates. In the generated cell graph composed of triangles after triangulation, each node denotes a cell and is labeled with a cell type, and each edge represents a hypothetical spatial relation between two cells.</p>
<p id="Par57">We build a comprehensive synthetic training set with ground truth presence/absence relations between pairs of the query CC motif (subgraph) and the target triangulated graph. The classical tool VF2<sup><a href="#CR21" class="usa-link" aria-describedby="CR21">21</a></sup> generated the ground truth occurrences by enumerating all the possibilities and guaranteeing the exact results with a substantial computational cost. To preserve biologically meaningful diversity, we simulated 7 distinct CC motifs from size-3 to size-9 in various topologies (Supplementary Fig. <a href="#MOESM1" class="usa-link">41</a>). Given the context of routine spatial omics in ROIs for each CC motif, we constructed the corresponding triangulated graphs with varying node types of 8, 16, and 32. The node types were randomly assigned to one of the cell types in the CC under a uniform distribution. To simulate different sizes of CC motifs corresponding to varying sizes of target graphs, we generated triangulated graphs of size-16 and size-32 for size-3 to size-6 CC motifs, and triangulated graphs of size-32 and size-64 for size-7 to size-9 CC motifs. Each pair of the query and the target has the same number of node types. To preserve the diversity, we generated 50 extended subgraphs with permutated node types for each CC motif. In total, we generated corresponding 1000 triangulated graphs permutating node types for each extended subgraph. To ensure a balanced ratio of positive and negative relations in presence and absence, we controlled the proportion of positive to negative samples at 1:1 in data generation. We split and set the generated data into training, validation, and testing sets in a ratio of 8:1:1. Noteworthily, to fairly test model performance and objectively evaluate the model’s actual generalization ability, we constructed an independent test set. For each type of CC motif, we selected 50 entirely new permutations and generated 100 triangulated graphs for each permutation.</p></section><section id="Sec14"><h3 class="pmc_sec_title">Evaluation performance on synthetic data</h3>
<p id="Par58">We selected CC motif sizes ranging from size-3 to size-9 in the generated synthetic data to demonstrate the power of TrimNN with ground truth. As a binary prediction task of subgraph matching in the synthetic dataset, the performance was evaluated by precision, recall, F1 score, and MCC on the generated test set of varying sizes and node types. For NSIC regresses continuous occurrence, we treat NSIC’s prediction of 0 as the query absent in the target graph, and any value larger than 0 as present in the target graph. We omitted VF2 in the performance comparison, for it has already been used to generate the ground truth at a substantial computational cost.</p>
<p id="Par59">To evaluate the quantitative performance in the synthetic dataset, we used metrics related to recommendation tasks, such as RMSE and mean absolute error (MAE), to quantify the accuracy of the predicted occurrences. We selected three distinct CC motifs, one in size-3, one in size-4, and one in size-5 (Supplementary Data <a href="#MOESM3" class="usa-link">2</a> and <a href="#MOESM3" class="usa-link">3</a>). Then we enumerated all the possibilities in both 8 cell types and 16 cell types with VF2 using significant computational resources. Because we value the biologically meaningful top overrepresented CC motifs in practice, we highlighted whether these methods can successfully identify the top 5 and top 10 overrepresented candidates as CC motifs.</p>
<p id="Par60">In scalability analysis, the query subgraph contains 9 nodes, and both the subgraph and the triangulated graph have 32 node types. All the experiments were performed on a workstation equipped with an Intel Xeon Gold 6338 CPU with one NVIDIA A100 GPU and 80 G RAM.</p></section><section id="Sec15"><h3 class="pmc_sec_title">Data preprocessing</h3>
<section id="Sec16"><h4 class="pmc_sec_title">Constructing triangulated graphs</h4>
<p id="Par61">We applied Delaunay triangulation to construct the cellular community based on the localization of cells with spatial coordinates. We added an optional step for noise reduction with unnecessary edges to prevent potential artifact edges from being included in certain boundary regions. We computed the lengths of all edges and used the 99th percentile as a threshold to remove outlier edges that were too long.</p></section><section id="Sec17"><h4 class="pmc_sec_title">Processing spatial proteomics in colorectal cancer study</h4>
<p id="Par62">The CRC study<sup><a href="#CR13" class="usa-link" aria-describedby="CR13">13</a></sup> included in the analysis uses the spatial proteomics approach CODEX at the single-cell resolution, which contains 140 tissue regions from 35 advanced-stage CRC patients with 56 protein markers and 29 distinct cell types. This study consists of 17 patients labeled as “Crohn’s-like reaction” (CLR) and the remaining 18 patients as “diffuse inflammatory infiltration” (DII). Each patient has CODEX data with two spots, and each spot has two regions. Coordinates and cell type annotations of the cells were from the original publication. For all 140 samples, the summarized occurrence numbers of size-1 to size-4 CC motifs were used in the classification tasks to predict the corresponding patients in either CLR or DII. Here, we chose a unified number 29 as the number of features in size-2 to size-4 CC motif analysis, in alignment with the size-1 29 features as the total number of cell types. To get the top size-4 motifs, we performed TrimNN based on the top 29 size-3 CC motifs as triangles.</p></section><section id="Sec18"><h4 class="pmc_sec_title">Processing spatial transcriptomics in an Alzheimer’s disease study</h4>
<p id="Par63">In the AD datasets using STARmap PLUS spatial transcriptomics in this study<sup><a href="#CR34" class="usa-link" aria-describedby="CR34">34</a></sup>, 8 AD samples of mouse brain tissues—two replicates of an 8-month-old AD and control, together with a 13-month-old AD and control—were utilized for analysis. The original study provided each cell’s coordinates and its cell type annotation. The CC was built using Delaunay triangulation, size 1 to size-3 CC motifs were identified by enumeration, and the size-4 CC motif was inferred through TrimNN. In the downstream analysis on the cellular level, we took only the disease samples with amyloid-β regions and combined replicates 1 and 2. We categorized the samples’ regions containing specific CC motifs and their extended 3-hop as motif regions, and all the remaining regions were complementary non-motif regions. Then we performed the cell–cell communication analysis using CellChat and DEG analysis using the Wilcoxon rank-sum test. The spatial localization of amyloid-β and cells’ spatial coordinates were used to compute the shortest distance from amyloid-β to motifs.</p></section><section id="Sec19"><h4 class="pmc_sec_title">Processing spatial proteomics in colorectal carcinoma study</h4>
<p id="Par64">This study<sup><a href="#CR46" class="usa-link" aria-describedby="CR46">46</a></sup> uses MIBI-TOF with 36 antibodies on colorectal carcinoma and comprises 58 ROIs within the spatial information and expression at the proteomics level. 40 ROIs were from two disease patients, and 18 ROIs were from two control patients. Both NCEM and TrimNN were used to analyze these 58 ROIs.</p></section></section><section id="Sec20"><h3 class="pmc_sec_title">Noise simulation</h3>
<p id="Par65">To further validate the robustness of CC motifs, we applied three types of simulations based on STARmap PLUS data in the AD study<sup><a href="#CR34" class="usa-link" aria-describedby="CR34">34</a></sup>. These simulations manually added noise to mimic limitations arising from sequencing technologies and data processing in practical analysis, including (1) cell missing, (2) cell coordinate shifting, and (3) cell type misclassification.</p>
<p id="Par66">(1) Cell missing targets to mimic the limited sequencing capacity to identify cells in the spatial omics samples. Proportions of cells in the input at rates of 0.01, 0.05, 0.1, 0.2, and 0.5 were randomly removed.</p>
<p id="Par67">(2) Cell coordinate shifting aims to mimic errors in sequencing or shifting in sample preparation to identify the spatial localization of the cells. For proportions of cells at rates of 0.01, 0.05, 0.1, 0.2, and 0.5, their coordinates were shifted proportion of 0.01, 0.05, 0.1, 0.2, and 0.5 with their average distances to the nearest neighboring cells in random directions.</p>
<p id="Par68">(3) Cell type misclassification aims to mimic errors in annotating cell types from spatial omics samples, possibly due to insufficient cell type annotations. In the simulation, original cell types were randomly shuffled in cells at proportions of 0.01, 0.05, 0.1, 0.2, and 0.5.</p>
<p id="Par69">To test the robustness of CC motifs in abundance rankings, all the simulations were randomly generated 100 times. Spearman correlation was adopted to compare the abundance ranks of all CC motifs between the original and the noisy datasets.</p>
<p id="Par70">To test the generalizability of machine learning models, random cropping was proposed in addition to noises from cell missing, cell coordinate shifting, and cell type misclassification. Random cropping aims to simulate smaller ROIs due to limited sequencing capacity or incomplete sample preparation, in that only a portion of the original tissue is captured. In this simulation, new patches as part of the original samples were randomly cropped with 0.1, 0.2, 0.3, 0.4, 0.5, 0.6, 0.7, 0.8, and 0.9 proportions of the original width and height. Random cropping was performed 10 times in each parameter setting. As part of the original samples, these newly generated small patches had identical phenotypes and survival of the original data. In generalization analysis, the machine learning model was trained on all the original CRC data and tested on the distorted samples.</p></section><section id="Sec21"><h3 class="pmc_sec_title">Statistics summary</h3>
<section id="Sec22"><h4 class="pmc_sec_title">False discovery rate (FDR)</h4>
<p id="Par71">To assess the significance of CC motifs across samples, Fisher’s exact test and Chi-squared test were adapted in the study, and they were adjusted using Bonferroni correction and the Benjamini-Hochberg method.</p></section><section id="Sec23"><h4 class="pmc_sec_title">Effect size</h4>
<p id="Par72">In large sample sizes, even small differences can become statistically significant (i.e., the <em>p</em>-value can be very small), which might not be practically significant. Therefore, we used the Cramér’s V effect size with a moderate effect size threshold of 0.21 in addition to the <em>p</em>-values to determine the significance of the Chi-squared tests<sup><a href="#CR54" class="usa-link" aria-describedby="CR54">54</a></sup>. For the hypothesis test that the occurrence of cell type “A” is independent of the occurrence of cell type “B”, the effect size of the chi-squared test is in Eq. (<a href="#Equ11" class="usa-link">11</a>):</p>
<table class="disp-formula p" id="Equ11"><tr>
<td class="formula"><math id="d33e4063" display="block"><msub><mrow><mi>E</mi><mi>S</mi></mrow><mrow><mi>A</mi><mi>B</mi></mrow></msub><mo>=</mo><msqrt><mrow><mfrac><mrow><msup><mrow><mi>χ</mi></mrow><mrow><mn>2</mn></mrow></msup></mrow><mrow><mi>n</mi><mo>⋅</mo><mfenced close=")" open="("><mrow><mi>k</mi><mo>−</mo><mn>1</mn></mrow></mfenced></mrow></mfrac></mrow></msqrt></math></td>
<td class="label">11</td>
</tr></table>
<p>where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq113"><math id="d33e4097"><msup><mrow><mi>χ</mi></mrow><mrow><mn>2</mn></mrow></msup></math></span> is the chi-squared statistic, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq114"><math id="d33e4110"><mi>n</mi></math></span> is the total number of edges involved in each test, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq115"><math id="d33e4119"><mi>k</mi></math></span> is 2 for the two-by-two contingency table. For the triangle of three types of cells in size-3 CC motifs, denoted as “A”, “B”, and “C”, we fixed one type of cell in the triangle and calculated the effect size for the test between the other two types of cells as in Eq. (<a href="#Equ12" class="usa-link">12</a>):</p>
<table class="disp-formula p" id="Equ12"><tr>
<td class="formula"><math id="d33e4132" display="block"><mi>E</mi><msub><mrow><mi>S</mi></mrow><mrow><mi>A</mi><mi>B</mi><mo>∣</mo><mi>C</mi></mrow></msub><mo>=</mo><msqrt><mrow><mfrac><mrow><msup><mrow><mi>χ</mi></mrow><mrow><mn>2</mn></mrow></msup></mrow><mrow><msub><mrow><mi>n</mi></mrow><mrow><mi>C</mi></mrow></msub><mo>⋅</mo><mfenced close=")" open="("><mrow><mi>k</mi><mo>−</mo><mn>1</mn></mrow></mfenced></mrow></mfrac></mrow></msqrt></math></td>
<td class="label">12</td>
</tr></table>
<p>where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq116"><math id="d33e4171"><msub><mrow><mi>n</mi></mrow><mrow><mi>C</mi></mrow></msub></math></span> denotes the total number of triangles that include at least one cell of “C”, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq117"><math id="d33e4185"><mi>k</mi></math></span> is 2. The overall effect size was determined as the minimum value of <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq118"><math id="d33e4194"><mi>E</mi><msub><mrow><mi>S</mi></mrow><mrow><mi>A</mi><mi>B</mi><mo>∣</mo><mi>C</mi></mrow></msub></math></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq119"><math id="d33e4211"><mi>E</mi><msub><mrow><mi>S</mi></mrow><mrow><mi>A</mi><mi>C</mi><mo>∣</mo><mi>B</mi></mrow></msub></math></span>, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq120"><math id="d33e4228"><mi>E</mi><msub><mrow><mi>S</mi></mrow><mrow><mi>B</mi><mi>C</mi><mo>∣</mo><mi>A</mi></mrow></msub></math></span>, i.e., <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq121"><math id="d33e4245"><mi>E</mi><msub><mrow><mi>S</mi></mrow><mrow><mi>A</mi><mi>B</mi><mi>C</mi></mrow></msub><mo>=</mo><mi>min</mi><mrow><mo>(</mo><mrow><mi>E</mi><msub><mrow><mi>S</mi></mrow><mrow><mi>A</mi><mi>B</mi><mo>∣</mo><mi>C</mi></mrow></msub><mo>,</mo><mi>E</mi><msub><mrow><mi>S</mi></mrow><mrow><mi>A</mi><mi>C</mi><mo>∣</mo><mi>B</mi></mrow></msub><mo>,</mo><mi>E</mi><msub><mrow><mi>S</mi></mrow><mrow><mi>B</mi><mi>C</mi><mo>∣</mo><mi>A</mi></mrow></msub></mrow><mo>)</mo></mrow></math></span>, to ensure that the effect size of each chi-squared test is above the threshold.</p></section><section id="Sec24"><h4 class="pmc_sec_title">Proportion test</h4>
<p id="Par73">We performed a two-sample, two-sided proportion test to test the significance between size-3 CC motifs and size-4 CC motifs. The null hypothesis (<em>H</em><sub>0</sub>) was that the proportions were equal between the two groups (i.e., <em>H</em><sub>0</sub>: the proportion of size-4 motifs within size-3 motifs is the same in both the disease and control groups).</p></section></section><section id="Sec25"><h3 class="pmc_sec_title">Parameter settings</h3>
<section id="Sec26"><h4 class="pmc_sec_title">Phenotypic classification in supervised learning</h4>
<p id="Par74">The top abundant CC motifs in size-1 to size-4 were selected as the features to represent CNs. Specifically, the dense rank of the occurrence was scaled to [0, 1] by the function min_max. Classical machine learning models Logistic Regression, Random Forest, and Support Vector Machine were adopted to perform classification tasks using 10 times tenfold cross-validation following the same protocol as CytoCommunity. Comprehensive criteria such as F1 score, precision, recall, MCC, area under the precision curve, and area under the Receiver Operating Characteristics Curve (ROC-AUC) were used to measure the binary classification performances.</p></section><section id="Sec27"><h4 class="pmc_sec_title">Survival curve</h4>
<p id="Par75">In survival analysis, we also summarized the overall occurrence of CC motifs through 140 regions for each size. The definition of whether a CC motif is enriched in a patient is by identifying whether it is among the top 29 motifs of its own size. Kaplan–Meier curves showing survival as a function of time for patients with and without CC motifs enriched in cell organizations using R packages “<em>survival</em>” and “<em>survminer</em>”. The hazard ratio (HR) and <em>p</em>-value (P) were calculated using Cox regression analysis.</p></section><section id="Sec28"><h4 class="pmc_sec_title">Cell–cell communication analysis</h4>
<p id="Par76">In the study of spatial transcriptomics datasets, CellChat<sup><a href="#CR35" class="usa-link" aria-describedby="CR35">35</a></sup> performed ligand–receptor analysis using the “<em>TruncatedMean</em>” method suggested by the official tutorial. In the cross-sample analysis, we retrieved the pathways related to targeted cell types of the single cell–cell interaction (targeted cell types as source or as target interacting with all other cell types) if the difference of cross-sample values is over 1. In the study of spatial proteomics datasets, we performed NCEM<sup><a href="#CR47" class="usa-link" aria-describedby="CR47">47</a></sup> type coupling and gene-wise analysis to supplement our size-2 triangulation analysis. All the parameters followed official tutorials.</p></section><section id="Sec29"><h4 class="pmc_sec_title">Gene-level analysis</h4>
<p id="Par77">We used the Wilcoxon rank-sum test in Scikit-learn<sup><a href="#CR55" class="usa-link" aria-describedby="CR55">55</a></sup> package to infer DEGs between motif and non-motif regions. Genes with <em>p</em>-values less than 0.05 were inferred as DEGs. GO and pathway enrichment analyses were adopted by clusterProfiler<sup><a href="#CR56" class="usa-link" aria-describedby="CR56">56</a></sup>. Then we applied the R package “<em>wilcoxauc</em>” to perform the Wilcoxon rank-sum test to identify marker genes in different motifs.</p></section><section id="Sec30"><h4 class="pmc_sec_title">Spatial co-occurrence analysis</h4>
<p id="Par78">We used Squidpy<sup><a href="#CR41" class="usa-link" aria-describedby="CR41">41</a></sup> function “<em>co_occurrence</em>” to calculate spatial co-occurrence probability between CC motifs and amyloid-β. The coordinates of CC motifs are the averaged coordinates of each node in the motif.</p></section><section id="Sec31"><h4 class="pmc_sec_title">Functional evaluation for multisize motifs</h4>
<p id="Par79">We extended the motif size up to 10 triangles (typically size-12) (shown in Supplementary Fig. <a href="#MOESM1" class="usa-link">37</a>) and benchmarked the exact occurrences of each motif using the enumerative searching approach VF2 with guaranteed accuracy. For each motif in multiple sizes, we identified its unique DEGs by comparing cells within the motif regions, as identified by VF2, to those outside the regions. GO and pathway enrichment analyses were then performed on these motif-related DEGs. The quality of CC motifs was evaluated and compared by <em>q</em>-values of enriched GO terms and pathways. To establish a baseline for AD relevance, we selected the top 300 GO terms found in the intersection of significant terms from four size-3 motifs: “CCC”, “CCM”, “CMM”, and “MMM” in 13-month-old AD disease samples. For each motif, the Cauchy meta-analysis<sup><a href="#CR57" class="usa-link" aria-describedby="CR57">57</a></sup> was applied to combine the <em>p</em>-values of these selected GO terms, with the resulting -log(<em>p</em>-value) as a quantitative measurement of relevance to AD.</p></section><section id="Sec32"><h4 class="pmc_sec_title">CytoCommunity and Space-GM</h4>
<p id="Par80">We used CytoCommunity and Space-GM to perform the supervised learning as the benchmark using the default parameter setting. Using the same protocol in CytoCommunity, we used the same 10 times tenfold cross-validation and the same evaluation metrics for all the methods. It is notable that CytoCommunity performs training within in default 20 epochs. The output from the last epoch was shown as the final results.</p></section><section id="Sec33"><h4 class="pmc_sec_title">DeepTalk</h4>
<p id="Par81">Following the official tutorial of DeepTalk, we first integrated the STARmap PLUS AD dataset and scRNA-seq data <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE176032" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE176032</a> downloaded from Gene Expression Omnibus (GEO). We identified the cell–cell communication score and focused on the ligand–receptor pairs: Ccl3-Ccr5, Csf1-Csf1r, Gdf15-Tgfbr2, Nrg3-Erbb4, Pdgfa-Pdgfra, Pdgfa-Pdgfrb, Pyy-Npy1r, Vegfa-Flt1, and Vip-Vipr2.</p></section></section><section id="Sec34"><h3 class="pmc_sec_title">Reporting summary</h3>
<p id="Par82">Further information on research design is available in the <a href="#MOESM4" class="usa-link">Nature Portfolio Reporting Summary</a> linked to this article.</p></section></section><section id="Sec35"><h2 class="pmc_sec_title">Supplementary information</h2>
<section class="sm xbox font-sm" id="MOESM1"><div class="media p"><div class="caption">
<a href="/articles/instance/12365081/bin/41467_2025_63141_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information</a><sup> (167.2MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM2"><div class="media p"><div class="caption">
<a href="/articles/instance/12365081/bin/41467_2025_63141_MOESM2_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">41467_2025_63141_MOESM2_ESM.pdf</a><sup> (101.7KB, pdf) </sup><p>Description of Additional Supplementary Files</p>
</div></div></section><section class="sm xbox font-sm" id="MOESM3"><div class="media p"><div class="caption">
<a href="/articles/instance/12365081/bin/41467_2025_63141_MOESM3_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Datasets 1-72</a><sup> (2.3MB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM4"><div class="media p"><div class="caption">
<a href="/articles/instance/12365081/bin/41467_2025_63141_MOESM4_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Reporting Summary</a><sup> (77.7KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM5"><div class="media p"><div class="caption">
<a href="/articles/instance/12365081/bin/41467_2025_63141_MOESM5_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Transparent Peer Review file</a><sup> (100.8MB, pdf) </sup>
</div></div></section></section><section id="Sec36"><h2 class="pmc_sec_title">Source data</h2>
<section class="sm xbox font-sm" id="MOESM6"><div class="media p"><div class="caption">
<a href="/articles/instance/12365081/bin/41467_2025_63141_MOESM6_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Source Data</a><sup> (63.6MB, xlsx) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>We thank Jiacheng Xie and Abdul Muqeeth for assisting with some of the analyses. This work is supported by National Institutes of Health grants R01DK138504 (to J.W. and Q.M.), R35GM126985 (to D.X.), R01GM152585, P01CA278732, U54AG075931, and P01AI177687 (to Q.M.), U19AG074879, R01AG019771, P30AG072976, U01AG072177, U01AG068057 (to M.Y.), NS121718 (to J-Q.K), R25HG012325 (to K.M.), R21DK140693 (to A.M.), the AnalytiXIN initiative (to J.W.), and the Alzheimer’s Association grants AARF-22-722571 (to M.Y.), as well as the Pelotonia Institute of Immuno-Oncology (PIIO) (to Q.M.).</p></section><section id="notes1"><h2 class="pmc_sec_title">Author contributions</h2>
<p>Conceptualization: J.W., Q.M., and D.X.; methodology: J.W. and D.X.; software coding: Y.Y. and S.W.; data collection and investigation: S.W., Y.Y., and K.M.; data analysis: S.W., Y.Y., J.L., M.Y., K.M., and A.M.; software testing and tutorial: Y.Y. and S.W.; manuscript writing, review, and editing: J.W., Y.Y., S.W., A.M., Q.M., J-Q.K., and D.X.</p></section><section id="notes2"><h2 class="pmc_sec_title">Peer review</h2>
<section id="FPar1"><h3 class="pmc_sec_title">Peer review information</h3>
<p id="Par83"><em>Nature Communications</em> thanks Wanxiang Shen and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. A peer review file is available.</p></section></section><section id="notes3"><h2 class="pmc_sec_title">Data availability</h2>
<p>All relevant data supporting the key findings of this study are available within the article and its supplementary information files. The human CRC CODEX dataset used in the colorectal cancer case study is available at <a href="https://data.mendeley.com/datasets/mpjzbtfgfr/1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://data.mendeley.com/datasets/mpjzbtfgfr/1</a>. The STARmap PLUS sequencing data in the Alzheimer’s disease case study are available at <a href="https://zenodo.org/records/7332091" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://zenodo.org/records/7332091</a>. The MIBI-TOF imaging data of colorectal carcinoma and healthy colon in the spatial proteomics case study are available at 10.5281/zenodo.3951613. The generated simulation data and analysis code has been deposited in Figshare database at 10.6084/m9.figshare.27958218. <a href="#Sec36" class="usa-link">Source data</a> are provided with this paper.</p></section><section id="notes4"><h2 class="pmc_sec_title">Code availability</h2>
<p>The source code of TrimNN is freely available at <a href="https://github.com/yuyang-0825/TrimNN" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/yuyang-0825/TrimNN</a> under MIT license. The specific version of the code associated with this publication is archived in Zenodo and is accessible via 10.5281/zenodo.15884334<sup><a href="#CR58" class="usa-link" aria-describedby="CR58">58</a></sup>.</p></section><section id="FPar2"><h2 class="pmc_sec_title">Competing interests</h2>
<p id="Par84">The authors declare no competing interests.</p></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm">
<div class="fn p" id="fn1"><p><strong>Publisher’s note</strong> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></div>
<div class="fn p" id="fn2"><p>These authors contributed equally: Yang Yu, Shuang Wang.</p></div>
</div></section><section id="_ci93_" lang="en" class="contrib-info"><h2 class="pmc_sec_title">Contributor Information</h2>
<p>Qin Ma, Email: qin.ma@osumc.edu.</p>
<p>Dong Xu, Email: xudong@missouri.edu.</p>
<p>Juexin Wang, Email: wangjuex@iu.edu.</p></section><section id="sec37"><h2 class="pmc_sec_title">Supplementary information</h2>
<p>The online version contains supplementary material available at 10.1038/s41467-025-63141-7.</p></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Ruitenberg, M. J. &amp; Nguyen, Q. H. Cellular neighborhood analysis in spatial omics reveals new tissue domains and cell subtypes. <em>Nat. Genet.</em><strong>56</strong>, 362–364 (2024).
</cite> [<a href="https://doi.org/10.1038/s41588-023-01646-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38413724/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ruitenberg,%20M.%20J.%20&amp;%20Nguyen,%20Q.%20H.%20Cellular%20neighborhood%20analysis%20in%20spatial%20omics%20reveals%20new%20tissue%20domains%20and%20cell%20subtypes.%20Nat.%20Genet.56,%20362%E2%80%93364%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>Palla, G., Fischer, D. S., Regev, A. &amp; Theis, F. J. Spatial components of molecular tissue biology. <em>Nat. Biotechnol.</em><strong>40</strong>, 308–318 (2022).
</cite> [<a href="https://doi.org/10.1038/s41587-021-01182-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35132261/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Palla,%20G.,%20Fischer,%20D.%20S.,%20Regev,%20A.%20&amp;%20Theis,%20F.%20J.%20Spatial%20components%20of%20molecular%20tissue%20biology.%20Nat.%20Biotechnol.40,%20308%E2%80%93318%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>Gulati, G. S., D’Silva, J. P., Liu, Y., Wang, L. &amp; Newman, A. M. Profiling cell identity and tissue architecture with single-cell and spatial transcriptomics. <em>Nat. Rev. Mol. Cell Biol.</em>10.1038/s41580-024-00768-2 (2024).</cite> [<a href="https://doi.org/10.1038/s41580-024-00768-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39169166/" class="usa-link">PubMed</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>de Souza, N., Zhao, S. &amp; Bodenmiller, B. Multiplex protein imaging in tumour biology. <em>Nat. Rev. Cancer</em><strong>24</strong>, 171–191 (2024).
</cite> [<a href="https://doi.org/10.1038/s41568-023-00657-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38316945/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?de%20Souza,%20N.,%20Zhao,%20S.%20&amp;%20Bodenmiller,%20B.%20Multiplex%20protein%20imaging%20in%20tumour%20biology.%20Nat.%20Rev.%20Cancer24,%20171%E2%80%93191%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Wu, Z. et al. Graph deep learning for the characterization of tumour microenvironments from spatial protein profiles in tissue specimens. <em>Nat. Biomed. Eng.</em><strong>6</strong>, 1435–1448 (2022).
</cite> [<a href="https://doi.org/10.1038/s41551-022-00951-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36357512/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wu,%20Z.%20et%20al.%20Graph%20deep%20learning%20for%20the%20characterization%20of%20tumour%20microenvironments%20from%20spatial%20protein%20profiles%20in%20tissue%20specimens.%20Nat.%20Biomed.%20Eng.6,%201435%E2%80%931448%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Hu, Y. et al. Unsupervised and supervised discovery of tissue cellular neighborhoods from cell phenotypes. <em>Nat. Methods</em><strong>21</strong>, 267–278 (2024).
</cite> [<a href="https://doi.org/10.1038/s41592-023-02124-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10864185/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38191930/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hu,%20Y.%20et%20al.%20Unsupervised%20and%20supervised%20discovery%20of%20tissue%20cellular%20neighborhoods%20from%20cell%20phenotypes.%20Nat.%20Methods21,%20267%E2%80%93278%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Varrone, M., Tavernari, D., Santamaria-Martinez, A., Walsh, L. A. &amp; Ciriello, G. CellCharter reveals spatial cell niches associated with tissue remodeling and cell plasticity. <em>Nat. Genet.</em><strong>56</strong>, 74–84 (2024).
</cite> [<a href="https://doi.org/10.1038/s41588-023-01588-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38066188/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Varrone,%20M.,%20Tavernari,%20D.,%20Santamaria-Martinez,%20A.,%20Walsh,%20L.%20A.%20&amp;%20Ciriello,%20G.%20CellCharter%20reveals%20spatial%20cell%20niches%20associated%20with%20tissue%20remodeling%20and%20cell%20plasticity.%20Nat.%20Genet.56,%2074%E2%80%9384%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>Singhal, V. et al. BANKSY unifies cell typing and tissue domain segmentation for scalable spatial omics data analysis. <em>Nat. Genet.</em><strong>56</strong>, 431–441 (2024).
</cite> [<a href="https://doi.org/10.1038/s41588-024-01664-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10937399/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38413725/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Singhal,%20V.%20et%20al.%20BANKSY%20unifies%20cell%20typing%20and%20tissue%20domain%20segmentation%20for%20scalable%20spatial%20omics%20data%20analysis.%20Nat.%20Genet.56,%20431%E2%80%93441%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>Liang, Q., Huang, Y., He, S. &amp; Chen, K. Pathway centric analysis for single-cell RNA-seq and spatial transcriptomics data with GSDensity. <em>Nat. Commun.</em><strong>14</strong>, 8416 (2023).
</cite> [<a href="https://doi.org/10.1038/s41467-023-44206-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10728201/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38110427/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Liang,%20Q.,%20Huang,%20Y.,%20He,%20S.%20&amp;%20Chen,%20K.%20Pathway%20centric%20analysis%20for%20single-cell%20RNA-seq%20and%20spatial%20transcriptomics%20data%20with%20GSDensity.%20Nat.%20Commun.14,%208416%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>Schiebinger, G. et al. Optimal-transport analysis of single-cell gene expression identifies developmental trajectories in reprogramming. <em>Cell</em><strong>176</strong>, 928–943 e922 (2019).
</cite> [<a href="https://doi.org/10.1016/j.cell.2019.01.006" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6402800/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30712874/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Schiebinger,%20G.%20et%20al.%20Optimal-transport%20analysis%20of%20single-cell%20gene%20expression%20identifies%20developmental%20trajectories%20in%20reprogramming.%20Cell176,%20928%E2%80%93943%20e922%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Bhate, S. S., Barlow, G. L., Schurch, C. M. &amp; Nolan, G. P. Tissue schematics map the specialization of immune tissue motifs and their appropriation by tumors. <em>Cell Syst.</em><strong>13</strong>, 109–130 e106 (2022).
</cite> [<a href="https://doi.org/10.1016/j.cels.2021.09.012" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34653369/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bhate,%20S.%20S.,%20Barlow,%20G.%20L.,%20Schurch,%20C.%20M.%20&amp;%20Nolan,%20G.%20P.%20Tissue%20schematics%20map%20the%20specialization%20of%20immune%20tissue%20motifs%20and%20their%20appropriation%20by%20tumors.%20Cell%20Syst.13,%20109%E2%80%93130%20e106%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Jain, Y. et al. Segmenting functional tissue units across human organs using community-driven development of generalizable machine learning algorithms. <em>Nat. Commun.</em><strong>14</strong>, 4656 (2023).
</cite> [<a href="https://doi.org/10.1038/s41467-023-40291-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10400613/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37537179/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jain,%20Y.%20et%20al.%20Segmenting%20functional%20tissue%20units%20across%20human%20organs%20using%20community-driven%20development%20of%20generalizable%20machine%20learning%20algorithms.%20Nat.%20Commun.14,%204656%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>Schurch, C. M. et al. Coordinated cellular neighborhoods orchestrate antitumoral immunity at the colorectal cancer invasive front. <em>Cell</em><strong>182</strong>, 1341–1359 e1319 (2020).
</cite> [<a href="https://doi.org/10.1016/j.cell.2020.07.005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7479520/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32763154/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Schurch,%20C.%20M.%20et%20al.%20Coordinated%20cellular%20neighborhoods%20orchestrate%20antitumoral%20immunity%20at%20the%20colorectal%20cancer%20invasive%20front.%20Cell182,%201341%E2%80%931359%20e1319%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>Lake, B. B. et al. An atlas of healthy and injured cell states and niches in the human kidney. <em>Nature</em><strong>619</strong>, 585–594 (2023).
</cite> [<a href="https://doi.org/10.1038/s41586-023-05769-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10356613/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37468583/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lake,%20B.%20B.%20et%20al.%20An%20atlas%20of%20healthy%20and%20injured%20cell%20states%20and%20niches%20in%20the%20human%20kidney.%20Nature619,%20585%E2%80%93594%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>Greenbaum, S. et al. A spatially resolved timeline of the human maternal-fetal interface. <em>Nature</em><strong>619</strong>, 595–605 (2023).
</cite> [<a href="https://doi.org/10.1038/s41586-023-06298-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10356615/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37468587/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Greenbaum,%20S.%20et%20al.%20A%20spatially%20resolved%20timeline%20of%20the%20human%20maternal-fetal%20interface.%20Nature619,%20595%E2%80%93605%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Wong, E., Baur, B., Quader, S. &amp; Huang, C. H. Biological network motif detection: principles and practice. <em>Brief. Bioinform.</em><strong>13</strong>, 202–215 (2012).
</cite> [<a href="https://doi.org/10.1093/bib/bbr033" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3294240/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22396487/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wong,%20E.,%20Baur,%20B.,%20Quader,%20S.%20&amp;%20Huang,%20C.%20H.%20Biological%20network%20motif%20detection:%20principles%20and%20practice.%20Brief.%20Bioinform.13,%20202%E2%80%93215%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>Patra, S. &amp; Mohapatra, A. Review of tools and algorithms for network motif discovery in biological networks. <em>IET Syst. Biol.</em><strong>14</strong>, 171–189 (2020).
</cite> [<a href="https://doi.org/10.1049/iet-syb.2020.0004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8687426/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32737276/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Patra,%20S.%20&amp;%20Mohapatra,%20A.%20Review%20of%20tools%20and%20algorithms%20for%20network%20motif%20discovery%20in%20biological%20networks.%20IET%20Syst.%20Biol.14,%20171%E2%80%93189%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Kashtan, N., Itzkovitz, S., Milo, R. &amp; Alon, U. Efficient sampling algorithm for estimating subgraph concentrations and detecting network motifs. <em>Bioinformatics</em><strong>20</strong>, 1746–1758 (2004).
</cite> [<a href="https://doi.org/10.1093/bioinformatics/bth163" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15001476/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kashtan,%20N.,%20Itzkovitz,%20S.,%20Milo,%20R.%20&amp;%20Alon,%20U.%20Efficient%20sampling%20algorithm%20for%20estimating%20subgraph%20concentrations%20and%20detecting%20network%20motifs.%20Bioinformatics20,%201746%E2%80%931758%20(2004)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Wernicke, S. &amp; Rasche, F. FANMOD: a tool for fast network motif detection. <em>Bioinformatics</em><strong>22</strong>, 1152–1153 (2006).
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btl038" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16455747/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wernicke,%20S.%20&amp;%20Rasche,%20F.%20FANMOD:%20a%20tool%20for%20fast%20network%20motif%20detection.%20Bioinformatics22,%201152%E2%80%931153%20(2006)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>Ullmann, J. R. An algorithm for subgraph isomorphism. <em>J. ACM</em><strong>23</strong>, 31–42 (1976).</cite> [<a href="https://scholar.google.com/scholar_lookup?Ullmann,%20J.%20R.%20An%20algorithm%20for%20subgraph%20isomorphism.%20J.%20ACM23,%2031%E2%80%9342%20(1976)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>Cordella, L. P., Foggia, P., Sansone, C. &amp; Vento, M. A (sub) graph isomorphism algorithm for matching large graphs. <em>IEEE Trans. Pattern Anal. Mach. Intell.</em><strong>26</strong>, 1367–1372 (2004).
</cite> [<a href="https://doi.org/10.1109/TPAMI.2004.75" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15641723/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cordella,%20L.%20P.,%20Foggia,%20P.,%20Sansone,%20C.%20&amp;%20Vento,%20M.%20A%20(sub)%20graph%20isomorphism%20algorithm%20for%20matching%20large%20graphs.%20IEEE%20Trans.%20Pattern%20Anal.%20Mach.%20Intell.26,%201367%E2%80%931372%20(2004)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>Qi, M., Cao, T. T. &amp; Tan, T. S. Computing 2D constrained delaunay triangulation using the GPU. <em>IEEE Trans. Vis. Comput. Graph.</em><strong>19</strong>, 736–748 (2013).
</cite> [<a href="https://doi.org/10.1109/TVCG.2012.307" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23492377/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Qi,%20M.,%20Cao,%20T.%20T.%20&amp;%20Tan,%20T.%20S.%20Computing%202D%20constrained%20delaunay%20triangulation%20using%20the%20GPU.%20IEEE%20Trans.%20Vis.%20Comput.%20Graph.19,%20736%E2%80%93748%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>Xu, K., Hu, W., Leskovec, J. &amp; Jegelka, S. How powerful are graph neural networks? Preprint at <a href="http://arxiv.org/abs/1810.00826" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://arxiv.org/abs/1810.00826</a> (2018).</cite>
</li>
<li id="CR24">
<span class="label">24.</span><cite>Li, P., Wang, Y., Wang, H. &amp; Leskovec, J. Distance encoding: Design provably more powerful neural networks for graph representation learning. <em>Adv. Neural Inf. Process. Syst.</em><strong>33</strong>, 4465–4478 (2020).</cite> [<a href="https://scholar.google.com/scholar_lookup?Li,%20P.,%20Wang,%20Y.,%20Wang,%20H.%20&amp;%20Leskovec,%20J.%20Distance%20encoding:%20Design%20provably%20more%20powerful%20neural%20networks%20for%20graph%20representation%20learning.%20Adv.%20Neural%20Inf.%20Process.%20Syst.33,%204465%E2%80%934478%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>Liu, X. et al. Neural subgraph isomorphism counting. <em>Proc. 26th ACM SIGKDD International Conference on Knowledge Discovery &amp; Data Mining</em>. 1959–1969 (Association for Computing Machinery, 2020).</cite>
</li>
<li id="CR26">
<span class="label">26.</span><cite>Ying, C. et al. Do transformers really perform badly for graph representation?. <em>Adv. Neural Inf. Process. Syst.</em><strong>34</strong>, 28877–28888 (2021).</cite> [<a href="https://scholar.google.com/scholar_lookup?Ying,%20C.%20et%20al.%20Do%20transformers%20really%20perform%20badly%20for%20graph%20representation?.%20Adv.%20Neural%20Inf.%20Process.%20Syst.34,%2028877%E2%80%9328888%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Wynn, T. A., Chawla, A. &amp; Pollard, J. W. Macrophage biology in development, homeostasis and disease. <em>Nature</em><strong>496</strong>, 445–455 (2013).
</cite> [<a href="https://doi.org/10.1038/nature12034" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3725458/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23619691/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wynn,%20T.%20A.,%20Chawla,%20A.%20&amp;%20Pollard,%20J.%20W.%20Macrophage%20biology%20in%20development,%20homeostasis%20and%20disease.%20Nature496,%20445%E2%80%93455%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Latif, N., Sarathchandra, P., Chester, A. H. &amp; Yacoub, M. H. Expression of smooth muscle cell markers and co-activators in calcified aortic valves. <em>Eur. Heart J.</em><strong>36</strong>, 1335–1345 (2015).
</cite> [<a href="https://doi.org/10.1093/eurheartj/eht547" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24419809/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Latif,%20N.,%20Sarathchandra,%20P.,%20Chester,%20A.%20H.%20&amp;%20Yacoub,%20M.%20H.%20Expression%20of%20smooth%20muscle%20cell%20markers%20and%20co-activators%20in%20calcified%20aortic%20valves.%20Eur.%20Heart%20J.36,%201335%E2%80%931345%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>Chen, H., Covert, I. C., Lundberg, S. M. &amp; Lee, S.-I. Algorithms to estimate Shapley value feature attributions. <em>Nat. Mach. Intell.</em><strong>5</strong>, 590–601 (2023).</cite> [<a href="https://scholar.google.com/scholar_lookup?Chen,%20H.,%20Covert,%20I.%20C.,%20Lundberg,%20S.%20M.%20&amp;%20Lee,%20S.-I.%20Algorithms%20to%20estimate%20Shapley%20value%20feature%20attributions.%20Nat.%20Mach.%20Intell.5,%20590%E2%80%93601%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR30">
<span class="label">30.</span><cite>Liu, M. et al. The crosstalk between macrophages and cancer cells potentiates pancreatic cancer cachexia. <em>Cancer Cell</em><strong>42</strong>, 885–903 e884 (2024).
</cite> [<a href="https://doi.org/10.1016/j.ccell.2024.03.009" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11162958/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38608702/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Liu,%20M.%20et%20al.%20The%20crosstalk%20between%20macrophages%20and%20cancer%20cells%20potentiates%20pancreatic%20cancer%20cachexia.%20Cancer%20Cell42,%20885%E2%80%93903%20e884%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>Laumont, C. M., Banville, A. C., Gilardi, M., Hollern, D. P. &amp; Nelson, B. H. Tumour-infiltrating B cells: immunological mechanisms, clinical impact and therapeutic opportunities. <em>Nat. Rev. Cancer</em><strong>22</strong>, 414–430 (2022).
</cite> [<a href="https://doi.org/10.1038/s41568-022-00466-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9678336/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35393541/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Laumont,%20C.%20M.,%20Banville,%20A.%20C.,%20Gilardi,%20M.,%20Hollern,%20D.%20P.%20&amp;%20Nelson,%20B.%20H.%20Tumour-infiltrating%20B%20cells:%20immunological%20mechanisms,%20clinical%20impact%20and%20therapeutic%20opportunities.%20Nat.%20Rev.%20Cancer22,%20414%E2%80%93430%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>Prater, K. E. et al. Human microglia show unique transcriptional changes in Alzheimer’s disease. <em>Nat. Aging</em><strong>3</strong>, 1–14 (2023).</cite> [<a href="https://doi.org/10.1038/s43587-023-00424-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10353942/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37248328/" class="usa-link">PubMed</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Zhang, Y., Chen, H., Li, R., Sterling, K. &amp; Song, W. Amyloid beta-based therapy for Alzheimer’s disease: challenges, successes and future. <em>Signal Transduct. Target Ther.</em><strong>8</strong>, 248 (2023).
</cite> [<a href="https://doi.org/10.1038/s41392-023-01484-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10310781/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37386015/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhang,%20Y.,%20Chen,%20H.,%20Li,%20R.,%20Sterling,%20K.%20&amp;%20Song,%20W.%20Amyloid%20beta-based%20therapy%20for%20Alzheimer%E2%80%99s%20disease:%20challenges,%20successes%20and%20future.%20Signal%20Transduct.%20Target%20Ther.8,%20248%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>Zeng, H. et al. Integrative in situ mapping of single-cell transcriptional states and tissue histopathology in a mouse model of Alzheimer’s disease. <em>Nat. Neurosci.</em><strong>26</strong>, 430–446 (2023).
</cite> [<a href="https://doi.org/10.1038/s41593-022-01251-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11332722/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36732642/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zeng,%20H.%20et%20al.%20Integrative%20in%20situ%20mapping%20of%20single-cell%20transcriptional%20states%20and%20tissue%20histopathology%20in%20a%20mouse%20model%20of%20Alzheimer%E2%80%99s%20disease.%20Nat.%20Neurosci.26,%20430%E2%80%93446%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>Jin, S. et al. Inference and analysis of cell-cell communication using CellChat. <em>Nat. Commun.</em><strong>12</strong>, 1088 (2021).
</cite> [<a href="https://doi.org/10.1038/s41467-021-21246-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7889871/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33597522/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jin,%20S.%20et%20al.%20Inference%20and%20analysis%20of%20cell-cell%20communication%20using%20CellChat.%20Nat.%20Commun.12,%201088%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR36">
<span class="label">36.</span><cite>Yang, W. et al. Deciphering cell-cell communication at single-cell resolution for spatial transcriptomics with subgraph-based graph attention network. <em>Nat. Commun.</em><strong>15</strong>, 7101 (2024).
</cite> [<a href="https://doi.org/10.1038/s41467-024-51329-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11330978/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39155292/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yang,%20W.%20et%20al.%20Deciphering%20cell-cell%20communication%20at%20single-cell%20resolution%20for%20spatial%20transcriptomics%20with%20subgraph-based%20graph%20attention%20network.%20Nat.%20Commun.15,%207101%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR37">
<span class="label">37.</span><cite>Bellenguez, C. et al. New insights into the genetic etiology of Alzheimer’s disease and related dementias. <em>Nat. Genet.</em><strong>54</strong>, 412–436 (2022).
</cite> [<a href="https://doi.org/10.1038/s41588-022-01024-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9005347/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35379992/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bellenguez,%20C.%20et%20al.%20New%20insights%20into%20the%20genetic%20etiology%20of%20Alzheimer%E2%80%99s%20disease%20and%20related%20dementias.%20Nat.%20Genet.54,%20412%E2%80%93436%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR38">
<span class="label">38.</span><cite>Ulland, T. K. &amp; Colonna, M. TREM2 - a key player in microglial biology and Alzheimer disease. <em>Nat. Rev. Neurol.</em><strong>14</strong>, 667–675 (2018).
</cite> [<a href="https://doi.org/10.1038/s41582-018-0072-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30266932/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ulland,%20T.%20K.%20&amp;%20Colonna,%20M.%20TREM2%20-%20a%20key%20player%20in%20microglial%20biology%20and%20Alzheimer%20disease.%20Nat.%20Rev.%20Neurol.14,%20667%E2%80%93675%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR39">
<span class="label">39.</span><cite>Thambisetty, M. et al. Alzheimer risk variant CLU and brain function during aging. <em>Biol. Psychiatry</em><strong>73</strong>, 399–405 (2013).
</cite> [<a href="https://doi.org/10.1016/j.biopsych.2012.05.026" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3488132/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22795969/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Thambisetty,%20M.%20et%20al.%20Alzheimer%20risk%20variant%20CLU%20and%20brain%20function%20during%20aging.%20Biol.%20Psychiatry73,%20399%E2%80%93405%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR40">
<span class="label">40.</span><cite>Whyte, L. S. et al. Lysosomal gene Hexb displays haploinsufficiency in a knock-in mouse model of Alzheimer’s disease. <em>IBRO Neurosci. Rep.</em><strong>12</strong>, 131–141 (2022).
</cite> [<a href="https://doi.org/10.1016/j.ibneur.2022.01.004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8819126/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35146484/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Whyte,%20L.%20S.%20et%20al.%20Lysosomal%20gene%20Hexb%20displays%20haploinsufficiency%20in%20a%20knock-in%20mouse%20model%20of%20Alzheimer%E2%80%99s%20disease.%20IBRO%20Neurosci.%20Rep.12,%20131%E2%80%93141%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR41">
<span class="label">41.</span><cite>Palla, G. et al. Squidpy: a scalable framework for spatial omics analysis. <em>Nat. Methods</em><strong>19</strong>, 171–178 (2022).
</cite> [<a href="https://doi.org/10.1038/s41592-021-01358-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8828470/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35102346/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Palla,%20G.%20et%20al.%20Squidpy:%20a%20scalable%20framework%20for%20spatial%20omics%20analysis.%20Nat.%20Methods19,%20171%E2%80%93178%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR42">
<span class="label">42.</span><cite>Kumari, S., Dhapola, R., Sharma, P., Singh, S. K. &amp; Reddy, D. H. Implicative role of Cytokines in Neuroinflammation mediated AD and associated signaling pathways: current Progress in molecular signaling and therapeutics. <em>Ageing Res. Rev.</em><strong>92</strong>, 102098 (2023).</cite> [<a href="https://doi.org/10.1016/j.arr.2023.102098" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39492425/" class="usa-link">PubMed</a>]</li>
<li id="CR43">
<span class="label">43.</span><cite>Sobue, A., Komine, O. &amp; Yamanaka, K. Neuroinflammation in Alzheimer’s disease: microglial signature and their relevance to disease. <em>Inflamm. Regen.</em><strong>43</strong>, 26 (2023).
</cite> [<a href="https://doi.org/10.1186/s41232-023-00277-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10170691/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37165437/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sobue,%20A.,%20Komine,%20O.%20&amp;%20Yamanaka,%20K.%20Neuroinflammation%20in%20Alzheimer%E2%80%99s%20disease:%20microglial%20signature%20and%20their%20relevance%20to%20disease.%20Inflamm.%20Regen.43,%2026%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR44">
<span class="label">44.</span><cite>d’Errico, P. et al. Microglia contribute to the propagation of Abeta into unaffected brain tissue. <em>Nat. Neurosci.</em><strong>25</strong>, 20–25 (2022).
</cite> [<a href="https://doi.org/10.1038/s41593-021-00951-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8737330/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34811521/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?d%E2%80%99Errico,%20P.%20et%20al.%20Microglia%20contribute%20to%20the%20propagation%20of%20Abeta%20into%20unaffected%20brain%20tissue.%20Nat.%20Neurosci.25,%2020%E2%80%9325%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR45">
<span class="label">45.</span><cite>Lei, X. et al. Immune cells within the tumor microenvironment: Biological functions and roles in cancer immunotherapy. <em>Cancer Lett.</em><strong>470</strong>, 126–133 (2020).
</cite> [<a href="https://doi.org/10.1016/j.canlet.2019.11.009" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31730903/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lei,%20X.%20et%20al.%20Immune%20cells%20within%20the%20tumor%20microenvironment:%20Biological%20functions%20and%20roles%20in%20cancer%20immunotherapy.%20Cancer%20Lett.470,%20126%E2%80%93133%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR46">
<span class="label">46.</span><cite>Hartmann, F. J. et al. Single-cell metabolic profiling of human cytotoxic T cells. <em>Nat. Biotechnol.</em><strong>39</strong>, 186–197 (2021).
</cite> [<a href="https://doi.org/10.1038/s41587-020-0651-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7878201/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32868913/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hartmann,%20F.%20J.%20et%20al.%20Single-cell%20metabolic%20profiling%20of%20human%20cytotoxic%20T%20cells.%20Nat.%20Biotechnol.39,%20186%E2%80%93197%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR47">
<span class="label">47.</span><cite>Fischer, D. S., Schaar, A. C. &amp; Theis, F. J. Modeling intercellular communication in tissues using spatial graphs of cells. <em>Nat. Biotechnol.</em><strong>41</strong>, 332–336 (2023).
</cite> [<a href="https://doi.org/10.1038/s41587-022-01467-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10017508/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36302986/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Fischer,%20D.%20S.,%20Schaar,%20A.%20C.%20&amp;%20Theis,%20F.%20J.%20Modeling%20intercellular%20communication%20in%20tissues%20using%20spatial%20graphs%20of%20cells.%20Nat.%20Biotechnol.41,%20332%E2%80%93336%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR48">
<span class="label">48.</span><cite>Li, N. et al. Mapping and modeling human colorectal carcinoma interactions with the tumor microenvironment. <em>Nat. Commun.</em><strong>14</strong>, 7915 (2023).
</cite> [<a href="https://doi.org/10.1038/s41467-023-43746-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10689473/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38036590/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li,%20N.%20et%20al.%20Mapping%20and%20modeling%20human%20colorectal%20carcinoma%20interactions%20with%20the%20tumor%20microenvironment.%20Nat.%20Commun.14,%207915%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR49">
<span class="label">49.</span><cite>Valdeolivas, A. et al. Profiling the heterogeneity of colorectal cancer consensus molecular subtypes using spatial transcriptomics. <em>NPJ Precis. Oncol.</em><strong>8</strong>, 10 (2024).
</cite> [<a href="https://doi.org/10.1038/s41698-023-00488-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10781769/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38200223/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Valdeolivas,%20A.%20et%20al.%20Profiling%20the%20heterogeneity%20of%20colorectal%20cancer%20consensus%20molecular%20subtypes%20using%20spatial%20transcriptomics.%20NPJ%20Precis.%20Oncol.8,%2010%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR50">
<span class="label">50.</span><cite>Punta, M. et al. The Pfam protein families database. <em>Nucleic Acids Res.</em><strong>40</strong>, D290–D301 (2012).
</cite> [<a href="https://doi.org/10.1093/nar/gkr1065" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3245129/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22127870/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Punta,%20M.%20et%20al.%20The%20Pfam%20protein%20families%20database.%20Nucleic%20Acids%20Res.40,%20D290%E2%80%93D301%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR51">
<span class="label">51.</span><cite>Sigrist, C. J. et al. PROSITE, a protein domain database for functional characterization and annotation. <em>Nucleic Acids Res.</em><strong>38</strong>, D161–D166 (2010).
</cite> [<a href="https://doi.org/10.1093/nar/gkp885" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2808866/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19858104/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sigrist,%20C.%20J.%20et%20al.%20PROSITE,%20a%20protein%20domain%20database%20for%20functional%20characterization%20and%20annotation.%20Nucleic%20Acids%20Res.38,%20D161%E2%80%93D166%20(2010)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR52">
<span class="label">52.</span><cite>Keren, L. et al. A structured tumor-immune microenvironment in triple negative breast cancer revealed by multiplexed ion beam imaging. <em>Cell</em><strong>174</strong>, 1373–1387 e1319 (2018).
</cite> [<a href="https://doi.org/10.1016/j.cell.2018.08.039" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6132072/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30193111/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Keren,%20L.%20et%20al.%20A%20structured%20tumor-immune%20microenvironment%20in%20triple%20negative%20breast%20cancer%20revealed%20by%20multiplexed%20ion%20beam%20imaging.%20Cell174,%201373%E2%80%931387%20e1319%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR53">
<span class="label">53.</span><cite>Wang, Y. et al. Graph pooling in graph neural networks: methods and their applications in omics studies. <em>Artif. Intell. Rev.</em><strong>57</strong>, 294 (2024).</cite> [<a href="https://scholar.google.com/scholar_lookup?Wang,%20Y.%20et%20al.%20Graph%20pooling%20in%20graph%20neural%20networks:%20methods%20and%20their%20applications%20in%20omics%20studies.%20Artif.%20Intell.%20Rev.57,%20294%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR54">
<span class="label">54.</span><cite>Lin, M., Lucas, H. C. Jr &amp; Shmueli, G. Research commentary—too big to fail: large samples and the p-value problem. <em>Inf. Syst. Res.</em><strong>24</strong>, 906–917 (2013).</cite> [<a href="https://scholar.google.com/scholar_lookup?Lin,%20M.,%20Lucas,%20H.%20C.%20Jr%20&amp;%20Shmueli,%20G.%20Research%20commentary%E2%80%94too%20big%20to%20fail:%20large%20samples%20and%20the%20p-value%20problem.%20Inf.%20Syst.%20Res.24,%20906%E2%80%93917%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR55">
<span class="label">55.</span><cite>Pedregosa, F. et al. Scikit-learn: Machine learning in Python. <em> J. Mach. Learn. Res.</em><strong>12</strong>, 2825–2830 (2011).</cite> [<a href="https://scholar.google.com/scholar_lookup?Pedregosa,%20F.%20et%20al.%20Scikit-learn:%20Machine%20learning%20in%20Python.%20J.%20Mach.%20Learn.%20Res.12,%202825%E2%80%932830%20(2011)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR56">
<span class="label">56.</span><cite>Xu, S. et al. Using clusterProfiler to characterize multiomics data. <em>Nat. Protoc.</em><strong>19</strong>, 3292–3320 (2024).</cite> [<a href="https://doi.org/10.1038/s41596-024-01020-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39019974/" class="usa-link">PubMed</a>]</li>
<li id="CR57">
<span class="label">57.</span><cite>Liu, Y. &amp; Xie, J. Cauchy combination test: a powerful test with analytic p-value calculation under arbitrary dependency structures. <em>J. Am. Stat. Assoc.</em><strong>115</strong>, 393–402 (2020).
</cite> [<a href="https://doi.org/10.1080/01621459.2018.1554485" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7531765/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33012899/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Liu,%20Y.%20&amp;%20Xie,%20J.%20Cauchy%20combination%20test:%20a%20powerful%20test%20with%20analytic%20p-value%20calculation%20under%20arbitrary%20dependency%20structures.%20J.%20Am.%20Stat.%20Assoc.115,%20393%E2%80%93402%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR58">
<span class="label">58.</span><cite>Yu, Y. et al. TrimNN: Characterizing cellular community motifs for studying multicellular topological organization in complex tissues, yuyang-0825/TrimNN, 10.5281/zenodo.15884334 (2025).</cite> [<a href="https://doi.org/10.1038/s41467-025-63141-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC12365081/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40830137/" class="usa-link">PubMed</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12365081/bin/41467_2025_63141_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information</a><sup> (167.2MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12365081/bin/41467_2025_63141_MOESM2_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">41467_2025_63141_MOESM2_ESM.pdf</a><sup> (101.7KB, pdf) </sup><p>Description of Additional Supplementary Files</p>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12365081/bin/41467_2025_63141_MOESM3_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Datasets 1-72</a><sup> (2.3MB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material4_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12365081/bin/41467_2025_63141_MOESM4_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Reporting Summary</a><sup> (77.7KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material5_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12365081/bin/41467_2025_63141_MOESM5_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Transparent Peer Review file</a><sup> (100.8MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material6_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12365081/bin/41467_2025_63141_MOESM6_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Source Data</a><sup> (63.6MB, xlsx) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>All relevant data supporting the key findings of this study are available within the article and its supplementary information files. The human CRC CODEX dataset used in the colorectal cancer case study is available at <a href="https://data.mendeley.com/datasets/mpjzbtfgfr/1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://data.mendeley.com/datasets/mpjzbtfgfr/1</a>. The STARmap PLUS sequencing data in the Alzheimer’s disease case study are available at <a href="https://zenodo.org/records/7332091" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://zenodo.org/records/7332091</a>. The MIBI-TOF imaging data of colorectal carcinoma and healthy colon in the spatial proteomics case study are available at 10.5281/zenodo.3951613. The generated simulation data and analysis code has been deposited in Figshare database at 10.6084/m9.figshare.27958218. <a href="#Sec36" class="usa-link">Source data</a> are provided with this paper.</p>
<p>The source code of TrimNN is freely available at <a href="https://github.com/yuyang-0825/TrimNN" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/yuyang-0825/TrimNN</a> under MIT license. The specific version of the code associated with this publication is archived in Zenodo and is accessible via 10.5281/zenodo.15884334<sup><a href="#CR58" class="usa-link" aria-describedby="CR58">58</a></sup>.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Nature Communications are provided here courtesy of <strong>Nature Publishing Group</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1038/s41467-025-63141-7"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/41467_2025_Article_63141.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (5.0 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12365081/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12365081/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12365081%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365081/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12365081/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12365081/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40830137/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12365081/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40830137/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12365081/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12365081/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="ZACJaU5dxus0ZmqSZTH0xyL1dcOyxya3qHFOHseeQaMcWBaftYV8srUMgS0Z1IIW">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
        <script type="module" crossorigin="" src="/static/assets/math-574fdcc6.js"></script>
    
    

    </body>
</html>
