
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Integrating miRNA profiling and machine learning for improved prostate cancer diagnosis - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="D2E4AF3E8AF3180305AF3E003A38982C.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="scirep">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365097/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Scientific Reports">
<meta name="citation_title" content="Integrating miRNA profiling and machine learning for improved prostate cancer diagnosis">
<meta name="citation_author" content="Shweta Singh">
<meta name="citation_author_institution" content="MIRNOW, BIONEST, Banaras Hindu University, Varanasi, India">
<meta name="citation_author" content="Abhay Kumar Pathak">
<meta name="citation_author_institution" content="DST-CIMS, Institute of Science, Banaras Hindu University, Varanasi, India">
<meta name="citation_author" content="Sukhad Kural">
<meta name="citation_author_institution" content="Department of Urology, Institute of Medical Sciences, Banaras Hindu University, Varanasi, India">
<meta name="citation_author" content="Lalit Kumar">
<meta name="citation_author_institution" content="Department of Urology, Institute of Medical Sciences, Banaras Hindu University, Varanasi, India">
<meta name="citation_author" content="Madan Gopal Bhardwaj">
<meta name="citation_author_institution" content="Department of Urology, Institute of Medical Sciences, Banaras Hindu University, Varanasi, India">
<meta name="citation_author" content="Mahima Yadav">
<meta name="citation_author_institution" content="Department of Pathology, Institute of Medical Sciences, Banaras Hindu University, Varanasi, India">
<meta name="citation_author" content="Sameer Trivedi">
<meta name="citation_author_institution" content="Department of Urology, Institute of Medical Sciences, Banaras Hindu University, Varanasi, India">
<meta name="citation_author" content="Parimal Das">
<meta name="citation_author_institution" content="Centre for Genetic Disorders, Institute of Science, Banaras Hindu University, Varanasi, India">
<meta name="citation_author" content="Manjari Gupta">
<meta name="citation_author_institution" content="DST-CIMS, Institute of Science, Banaras Hindu University, Varanasi, India">
<meta name="citation_author" content="Garima Jain">
<meta name="citation_author_institution" content="Centre for Genetic Disorders, Institute of Science, Banaras Hindu University, Varanasi, India">
<meta name="citation_author_institution" content="MIRNOW, BIONEST, Banaras Hindu University, Varanasi, India">
<meta name="citation_publication_date" content="2025 Aug 20">
<meta name="citation_volume" content="15">
<meta name="citation_firstpage" content="30477">
<meta name="citation_doi" content="10.1038/s41598-025-99754-7">
<meta name="citation_pmid" content="40830224">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365097/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365097/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365097/pdf/41598_2025_Article_99754.pdf">
<meta name="description" content="Prostate cancer (PCa) diagnosis remains challenging due to overlapping clinical features with benign prostatic hyperplasia (BPH) and limitations of existing diagnostic tools like PSA tests, which yield high false-positive rates. This study ...">
<meta name="og:title" content="Integrating miRNA profiling and machine learning for improved prostate cancer diagnosis">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Prostate cancer (PCa) diagnosis remains challenging due to overlapping clinical features with benign prostatic hyperplasia (BPH) and limitations of existing diagnostic tools like PSA tests, which yield high false-positive rates. This study ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365097/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12365097">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1038/s41598-025-99754-7"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/41598_2025_Article_99754.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12365097%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12365097/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12365097/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365097/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-scirep.jpg" alt="Scientific Reports logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Scientific Reports" title="Link to Scientific Reports" shape="default" href="http://www.nature.com/srep" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Sci Rep</button></div>. 2025 Aug 20;15:30477. doi: <a href="https://doi.org/10.1038/s41598-025-99754-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1038/s41598-025-99754-7</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Sci%20Rep%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sci%20Rep%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Sci%20Rep%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Sci%20Rep%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Integrating miRNA profiling and machine learning for improved prostate cancer diagnosis</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Singh%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Shweta Singh</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Shweta Singh</span></h3>
<div class="p">
<sup>5</sup>MIRNOW, BIONEST, Banaras Hindu University, Varanasi, India </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Singh%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Shweta Singh</span></a>
</div>
</div>
<sup>5,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Pathak%20AK%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Abhay Kumar Pathak</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Abhay Kumar Pathak</span></h3>
<div class="p">
<sup>2</sup>DST-CIMS, Institute of Science, Banaras Hindu University, Varanasi, India </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Pathak%20AK%22%5BAuthor%5D" class="usa-link"><span class="name western">Abhay Kumar Pathak</span></a>
</div>
</div>
<sup>2,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kural%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Sukhad Kural</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Sukhad Kural</span></h3>
<div class="p">
<sup>3</sup>Department of Urology, Institute of Medical Sciences, Banaras Hindu University, Varanasi, India </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kural%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Sukhad Kural</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kumar%20L%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Lalit Kumar</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Lalit Kumar</span></h3>
<div class="p">
<sup>3</sup>Department of Urology, Institute of Medical Sciences, Banaras Hindu University, Varanasi, India </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kumar%20L%22%5BAuthor%5D" class="usa-link"><span class="name western">Lalit Kumar</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bhardwaj%20MG%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Madan Gopal Bhardwaj</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Madan Gopal Bhardwaj</span></h3>
<div class="p">
<sup>3</sup>Department of Urology, Institute of Medical Sciences, Banaras Hindu University, Varanasi, India </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bhardwaj%20MG%22%5BAuthor%5D" class="usa-link"><span class="name western">Madan Gopal Bhardwaj</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yadav%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Mahima Yadav</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Mahima Yadav</span></h3>
<div class="p">
<sup>4</sup>Department of Pathology, Institute of Medical Sciences, Banaras Hindu University, Varanasi, India </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yadav%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Mahima Yadav</span></a>
</div>
</div>
<sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Trivedi%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Sameer Trivedi</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Sameer Trivedi</span></h3>
<div class="p">
<sup>3</sup>Department of Urology, Institute of Medical Sciences, Banaras Hindu University, Varanasi, India </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Trivedi%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Sameer Trivedi</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Das%20P%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Parimal Das</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Parimal Das</span></h3>
<div class="p">
<sup>1</sup>Centre for Genetic Disorders, Institute of Science, Banaras Hindu University, Varanasi, India </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Das%20P%22%5BAuthor%5D" class="usa-link"><span class="name western">Parimal Das</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gupta%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id9"><span class="name western">Manjari Gupta</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">Manjari Gupta</span></h3>
<div class="p">
<sup>2</sup>DST-CIMS, Institute of Science, Banaras Hindu University, Varanasi, India </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gupta%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Manjari Gupta</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Jain%20G%22%5BAuthor%5D" class="usa-link" aria-describedby="id10"><span class="name western">Garima Jain</span></a><div hidden="hidden" id="id10">
<h3><span class="name western">Garima Jain</span></h3>
<div class="p">
<sup>1</sup>Centre for Genetic Disorders, Institute of Science, Banaras Hindu University, Varanasi, India </div>
<div class="p">
<sup>5</sup>MIRNOW, BIONEST, Banaras Hindu University, Varanasi, India </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Jain%20G%22%5BAuthor%5D" class="usa-link"><span class="name western">Garima Jain</span></a>
</div>
</div>
<sup>1,</sup><sup>5,</sup><sup>✉,</sup><sup>#</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>Centre for Genetic Disorders, Institute of Science, Banaras Hindu University, Varanasi, India </div>
<div id="Aff2">
<sup>2</sup>DST-CIMS, Institute of Science, Banaras Hindu University, Varanasi, India </div>
<div id="Aff3">
<sup>3</sup>Department of Urology, Institute of Medical Sciences, Banaras Hindu University, Varanasi, India </div>
<div id="Aff4">
<sup>4</sup>Department of Pathology, Institute of Medical Sciences, Banaras Hindu University, Varanasi, India </div>
<div id="Aff5">
<sup>5</sup>MIRNOW, BIONEST, Banaras Hindu University, Varanasi, India </div>
<div class="author-notes p">
<div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div>
<div class="fn" id="_eqcntrb93pmc__">
<sup>#</sup><p class="display-inline">Contributed equally.</p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 Jan 18; Accepted 2025 Apr 22; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by-nc-nd/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12365097  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40830224/" class="usa-link">40830224</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<p id="Par1">Prostate cancer (PCa) diagnosis remains challenging due to overlapping clinical features with benign prostatic hyperplasia (BPH) and limitations of existing diagnostic tools like PSA tests, which yield high false-positive rates. This study investigates the potential of microRNA (miRNA) biomarkers, analyzed via reverse transcription polymerase chain reaction and machine learning (ML), to enhance diagnostic accuracy. miRNAs such as miR-21-5p, miR-141-3p, and miR-221-3p were identified as significant discriminators between PCa and BPH through a prospective cohort study. Whole blood miRNA profiling offered a robust systemic representation of disease states. A random forest ML model was trained on expression data, achieving notable performance metrics: an accuracy of 77.42%, AUC of 0.78 during verification, and 74.07% accuracy and 0.75 AUC in validation. The model’s use of miRNA expression ratios, such as miR-141-3p/miR-221-3p, demonstrated superior sensitivity and specificity over traditional PSA testing. Bioinformatics analysis confirmed the association of selected miRNAs with cancer pathways, including PD-L1/PD-1 checkpoint and androgen receptor signaling, validating the biological relevance of the findings. This novel integration of miRNA profiling and machine learning holds great potential for the clinical translation of miRNA-based non-invasive diagnostics, enhancing diagnostic precision. However, broader population studies and standardization of protocols are needed to ensure scalability and clinical applicability. This research provides a foundational framework for advancing miRNA-based diagnostics, bridging discovery and clinical implementation.</p>
<section id="sec1"><h3 class="pmc_sec_title">Supplementary Information</h3>
<p>The online version contains supplementary material available at 10.1038/s41598-025-99754-7.</p></section><section id="kwd-group1" lang="en" class="kwd-group"><p><strong>Keywords:</strong> Liquid biopsy, Random forest, Prostate cancer, miRNAs, Biomarker, Cancer diagnostics</p></section><section id="kwd-group2" class="kwd-group"><p><strong>Subject terms:</strong> Biochemistry, Cancer, Computational biology and bioinformatics, Genetics, Molecular biology, Health care, Urology</p></section></section><section id="Sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p id="Par2">MicroRNA (miRNA) biomarkers have emerged as critical players in disease diagnostics, particularly in cancer research. Known for their stability and differential expression in disease versus healthy subjects, miRNA biomarkers are promising candidates for early cancer diagnosis and monitoring<sup><a href="#CR1" class="usa-link" aria-describedby="CR1">1</a>,<a href="#CR2" class="usa-link" aria-describedby="CR2">2</a></sup>. The advent of high-throughput technologies like next-generation sequencing (NGS) has revolutionized miRNA biomarker discovery, providing comprehensive datasets that reveal miRNA expression profiles across various disease conditions<sup><a href="#CR3" class="usa-link" aria-describedby="CR3">3</a></sup>. Despite these advancements, the clinical application of miRNA biomarkers remains limited<sup><a href="#CR4" class="usa-link" aria-describedby="CR4">4</a>,<a href="#CR5" class="usa-link" aria-describedby="CR5">5</a></sup>, primarily due to the high cost and complexity of NGS, which makes it unsuitable for routine diagnostics in resource-limited settings<sup><a href="#CR6" class="usa-link" aria-describedby="CR6">6</a></sup>. Consequently, more affordable and accessible technologies such as reverse transcription polymerase chain reaction (RT-PCR) has become the primary tools for validating discovered biomarkers<sup><a href="#CR7" class="usa-link" aria-describedby="CR7">7</a>,<a href="#CR8" class="usa-link" aria-describedby="CR8">8</a></sup>. While PCR-based methods are sensitive for targeted miRNA quantification, they still exhibit relatively lower sensitivity compared to high-throughput approaches like NGS<sup><a href="#CR9" class="usa-link" aria-describedby="CR9">9</a></sup>. However, the diagnostic application of miRNA biomarkers using RT-PCR-based methods faces significant challenges<sup><a href="#CR10" class="usa-link" aria-describedby="CR10">10</a></sup> in terms of experimental design, internal controls, sample preparation, and data analysis<sup><a href="#CR11" class="usa-link" aria-describedby="CR11">11</a></sup>. A major concern is the variability introduced by differences in devices, methodologies, and sample handling. Such inconsistencies can result in data variations that do not accurately reflect the biological state of the patient but instead stem from technical artifacts. This lack of uniformity makes it difficult to establish a precise cutoff for differentiation based on Ct values that undermines the reliability of miRNA-based diagnostics in clinical settings. Variability and low sensitivity has hindered the translation of promising research findings into clinical trials and diagnostic applications. Furthermore, the necessity for large-scale and prospective clinical studies for validation remains indispensable, posing significant resource and ethical challenges. Addressing these challenges is essential for improving diagnostic accuracy and enhancing the clinical utility of miRNA biomarkers.</p>
<p id="Par3">Machine learning (ML) is an artificial intelligence (AI) subset that enables systems to learn from data without explicit programming. ML models use algorithms to find patterns in data, enabling them to make predictions or decisions. These models can be supervised, unsupervised, semi-supervised, or reinforcement learning-based. As they process more data, their performance improves, making ML valuable across various fields, including healthcare<sup><a href="#CR12" class="usa-link" aria-describedby="CR12">12</a></sup>.</p>
<p id="Par4">Linear methods, such as linear regression, assume a direct relationship between inputs and outputs. These models can struggle when data shows high variance or complex relationships, leading to underfitting or overfitting. ML models, especially those like decision trees, random forests, and neural networks, are more flexible and equipped to handle variance<sup><a href="#CR13" class="usa-link" aria-describedby="CR13">13</a></sup>. They capture non-linear relationships and interactions between features more effectively than linear models. For example, ensemble techniques such as random forests use multiple decision trees to reduce variance by averaging predictions, making ML models robust in noisy and complex datasets where traditional linear models would fail<sup><a href="#CR14" class="usa-link" aria-describedby="CR14">14</a></sup>.</p>
<p id="Par5">ML models are revolutionizing biological research by analyzing complex datasets like genomics, proteomics, and medical imaging. In biology, datasets often exhibit high dimensionality, non-linearity, and noise, where ML excels<sup><a href="#CR15" class="usa-link" aria-describedby="CR15">15</a></sup>. For instance, ML models have extensively been used in biomarker discovery, such as in cancer diagnostics, by analyzing RNA sequencing or microarray data<sup><a href="#CR16" class="usa-link" aria-describedby="CR16">16</a></sup>. Supervised learning models, which excel in classifying labeled data, however remain underutilized in the context of RT-PCR data. To the best of our knowledge, very few studies have utilized RT-PCR data to effectively leverage supervised learning for enhancing the diagnostic accuracy<sup><a href="#CR17" class="usa-link" aria-describedby="CR17">17</a>,<a href="#CR18" class="usa-link" aria-describedby="CR18">18</a></sup>. By training ML models on a large set of patient-specific RT-PCR data, ML models can capture patterns in multiple miRNA expressions that may go undetected by traditional analysis methods, to improve the sensitivity (the ability to accurately identify true positives) and specificity (the ability to accurately identify true negatives)<strong>.</strong></p>
<p id="Par6">The aim of our study was to assess the expression levels of a miRNA panel in the blood of prostate cancer (PCa) patients compared to those with benign prostatic hyperplasia (BPH) and apply supervised ML algorithms to enhance the sensitivity and specificity of these biomarkers. By leveraging the strengths of ML alongside the cost-effectiveness and accessibility of RT-PCR, we aimed to not only improve biomarker panel performance but also bridge the gap between discovery research and clinical application.</p>
<p id="Par7">The standard screening method for PCa is the serum prostate-specific antigen (PSA) test. However, while PSA levels often rise in prostate-related conditions, it is not specific to PCa<sup><a href="#CR19" class="usa-link" aria-describedby="CR19">19</a></sup>, leading to high false-positive rates, overdiagnosis, and unnecessary biopsies. The gold standard for distinguishing PCa from BPH remains a transrectal prostate biopsy, which is invasive and uncomfortable, causing many patients to avoid it<sup><a href="#CR20" class="usa-link" aria-describedby="CR20">20</a></sup>. Since PCa and BPH differ significantly in their life-threatening risk, treatment, and intervention strategies<sup><a href="#CR21" class="usa-link" aria-describedby="CR21">21</a></sup>, early differentiation is crucial for informed decision-making. Therefore, there is an urgent need for more accurate, non-invasive diagnostic tools to distinguish between these conditions.</p>
<p id="Par8">The discovery of miRNAs in the plasma of PCa patients sparked significant interest in their potential as liquid biopsy biomarkers for PCa<sup><a href="#CR22" class="usa-link" aria-describedby="CR22">22</a></sup>. Numerous studies have identified and validated various miRNAs as potential biomarkers; however, the results have not been consistent<sup><a href="#CR23" class="usa-link" aria-describedby="CR23">23</a>,<a href="#CR24" class="usa-link" aria-describedby="CR24">24</a></sup>.</p>
<p id="Par9">Based on previous studies, we selected a group of promising miRNAs for investigation in our study: miR-19b-3p, miR-21-5p, miR-93-5p, miR-141-3p, miR-221-3p, and miR-375-3p<sup><a href="#CR25" class="usa-link" aria-describedby="CR25">25</a>–<a href="#CR29" class="usa-link" aria-describedby="CR29">29</a></sup>. Although, urine, plasma and serum are commonly utilized samples in both the studies mentioned above and other related research, whole blood also presents a good source of biomarkers with distinct advantages, including a higher miRNA yield, reduced susceptibility to technical errors, and a more comprehensive representation of the systemic profile<sup><a href="#CR30" class="usa-link" aria-describedby="CR30">30</a></sup>. In this study, we focused on miRNA profiling from whole blood using the RT-PCR method, conducted in three distinct phases with separate cohorts. In the first phase, the most significant miRNAs were identified from the discovery cohort. The verification phase involved generating RT-PCR data from the verification cohort to train the ML model. During the final validation phase, the ML model was evaluated using a dataset that had not been previously exposed to the model (Figs. <a href="#Fig1" class="usa-link">1</a> and <a href="#Fig2" class="usa-link">2</a>). The results showed that combining ML with miRNA biomarkers significantly improved diagnostic efficiency compared to using individual miRNAs alone.</p>
<figure class="fig xbox font-sm" id="Fig1"><h3 class="obj_head">Fig. 1.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365097_41598_2025_99754_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6fe0/12365097/b493e74188a6/41598_2025_99754_Fig1_HTML.jpg" loading="lazy" id="d33e387" height="465" width="799" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Venn diagram showing the overview of sample distribution across discovery, verification and validation cohorts.</p></figcaption></figure><figure class="fig xbox font-sm" id="Fig2"><h3 class="obj_head">Fig. 2.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365097_41598_2025_99754_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6fe0/12365097/9379c1044ff8/41598_2025_99754_Fig2_HTML.jpg" loading="lazy" id="d33e411" height="453" width="673" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Workflow of the study design (<strong>A</strong>) discovery cohort; (<strong>B</strong>) verification cohort; (<strong>C</strong>) validation cohort figure created using Biorender (<a href="https://biorender.com/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://biorender.com/</a>).</p></figcaption></figure></section><section id="Sec2"><h2 class="pmc_sec_title">Materials and methods</h2>
<section id="Sec3"><h3 class="pmc_sec_title">Study population</h3>
<p id="Par10">We conducted a prospective cohort study of 86 participants from November 2022 to August 2024 in collaboration with the Centre for Genetic Disorders and the Department of Urology at the Institute of Medical Sciences, Banaras Hindu University. This study was conducted in accordance with relevant guidelines and regulations (ICMR National Ethical Guidelines for Biomedical and Health Research). The experimental protocols were reviewed and approved by the Ethics Committee, Institute of Science, Banaras Hindu University (Registration No.: ECR/226/Indt/UP/2014/RR22) under Approval No: I.Sc./ECM-XIV/2022-23. Written informed consent was obtained from all participants and/or their legal guardians before sample collection and data analysis. The study examined a treatment-naïve patients who were clinically suspected of having PCa and scheduled for a prostate biopsy based on digital rectal examination (DRE) findings and elevated PSA levels. Exclusion criteria involved: (a) lack of complete medical records; (b) non-urothelial malignancies or other urological; (c) refusal to consent; and (d) prior androgen deprivation therapy (ADT) or radiation therapy. In the second phase, de-identification of collected samples was performed, followed by classification into two groups based on histopathological examination of 12-core prostate biopsies resulting in PCa (n = 51) and BPH (n = 35) (Table <a href="#Tab1" class="usa-link">1</a>). The BPH cohort includes five samples from patients who presented with similar LUTS (lower urinary tract symptoms) but were classified as PIN (Prostatic Intraepithelial Neoplasia) or histopathologically negative upon evaluation.</p>
<section class="tw xbox font-sm" id="Tab1"><h4 class="obj_head">Table 1.</h4>
<div class="caption p"><p>Clinicopathological characteristics of prostate biopsy patients.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Clinical parameters</th>
<th align="left" colspan="1" rowspan="1">Sample size</th>
<th align="left" colspan="1" rowspan="1">PCa</th>
<th align="left" colspan="1" rowspan="1">BPH</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">Total number</td>
<td align="left" colspan="1" rowspan="1">86</td>
<td align="left" colspan="1" rowspan="1">51</td>
<td align="left" colspan="1" rowspan="1">35</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Age (Mean ± SD)</td>
<td align="left" colspan="1" rowspan="1">68.771 ± 10.70</td>
<td align="left" colspan="1" rowspan="1">70.35 ± 10.15</td>
<td align="left" colspan="1" rowspan="1">61.34 ± 11.04</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Range</td>
<td align="left" colspan="1" rowspan="1">43–87</td>
<td align="left" colspan="1" rowspan="1">48–87</td>
<td align="left" colspan="1" rowspan="1">43–85</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">PSA (Mean ± SD)</td>
<td align="left" colspan="1" rowspan="1">31.43 ± 29.48</td>
<td align="left" colspan="1" rowspan="1">41.8 ± 31.53</td>
<td align="left" colspan="1" rowspan="1">21.92 ± 31.54</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Range</td>
<td align="left" colspan="1" rowspan="1">1–114.5</td>
<td align="left" colspan="1" rowspan="1">2–114.5</td>
<td align="left" colspan="1" rowspan="1">1–53</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section></section><section id="Sec4"><h3 class="pmc_sec_title">Blood collection, storage and RNA isolation</h3>
<p id="Par11">Peripheral venous blood was collected from the patient into collection tubes coated with EDTA prior to the prostate biopsy. Total RNA was extracted from 400 µL of blood using 750 µL of Trizol reagent (Cat No. 15596026). 200 µL of chloroform was added, the sample was partitioned into three distinct phases. The upper aqueous phase was carefully separated, combined with 500 µL of isopropanol, to precipitate RNA. The pellet was resuspended in 20 μL of DEPC-treated water. The concentration and quality of the extracted RNA were assessed using NanoDrop (Thermo Scientific, USA).</p></section><section id="Sec5"><h3 class="pmc_sec_title">Reverse transcription and RT-PCR</h3>
<p id="Par12">The RevertAid First Strand cDNA Synthesis Kit (Catalog #K1622, Thermo Scientific, USA) was used for the reverse transcription of miRNA samples. miRNA-targeted stem-loop primer was used for reverse transcription (Supplementary Table <a href="#MOESM1" class="usa-link">1</a>). For quantitative PCR (RT-PCR), reactions in triplicate were carried out in the Applied Biosystem QuantStudio 6 Flex Real-Time PCR System (Thermo Scientific, USA) using Maxima SYBR Green/ROX RT-PCR Master mix (2X, Catalog #K0221). Delta Ct values were then calculated by subtracting the Ct value of the endogenous control, RNU6 for each sample.</p></section><section id="Sec6"><h3 class="pmc_sec_title">Study design</h3>
<p id="Par13">Due to the limited availability of studies investigating miRNA expression in whole blood for PCa, we selected candidate miRNAs (miR-19b-3p, miR-21-5p, miR-93-5p, miR-141-3p, miR-221-3p, and miR-375-3p) based on existing literature on serum and plasma. These miRNAs have been consistently associated with PCa and key oncogenic pathways. Our study aims to validate their relevance in whole blood, which provides distinct advantages as higher miRNA yield, reduced susceptibility to pre-analytical variability, and a more comprehensive representation of systemic disease states.</p>
<p id="Par14">The study was designed in three cohorts: discovery, verification, and validation (Fig. <a href="#Fig2" class="usa-link">2</a>). The discovery cohort comprised of 20 samples, including BPH (n = 9) and PCa (n = 11) cases, and was used to analyze the expression patterns of six selected miRNAs using RT-PCR. The hypothesis test (Mann–Whitney U) was employed to identify the top 3 differentially expressed miRNAs among the selected group, resulting in the selection of three miRNAs: miR-21-5p, miR-141-3p, and miR-221-3p (Supplementary Table <a href="#MOESM1" class="usa-link">2</a>). In the verification cohort (n = 59; inclusive of samples used in the discovery cohort) composed of BPH (n = 36) and PCa (n = 23) (Fig. <a href="#Fig2" class="usa-link">2</a>), the expression data for these 3 selected miRNAs was collected and this data was used to build and train a ML model. Finally, the validation cohort (n = 27), consisting of BPH (n = 7), PIN (n = 5) and PCa (n = 15) samples, included cases that had not been previously exposed to the trained model. The aim of the validation cohort was to evaluate its performance of the trained ML model.</p></section><section id="Sec7"><h3 class="pmc_sec_title">ML-model development</h3>
<section id="Sec8"><h4 class="pmc_sec_title">Preprocessing</h4>
<p id="Par15">For the preprocessing step, NULL values have been imputed by the median value. All the operations have been performed on Delta Ct values.</p></section><section id="Sec9"><h4 class="pmc_sec_title">Statistical analysis</h4>
<p id="Par16">The Mann–Whitney U test, a nonparametric statistical method, was used for hypothesis testing to compare differences between two independent groups. Its ability to operate without assuming a normal distribution makes it a versatile analytical tool. We have used the ROC curve to evaluate the predictive capability of individual features in distinguishing the BPH and PCa classes. ROC (receiver operating characteristic) curve is a graphical representation that shows the performance of a binary model at different threshold settings. Using two values of TPR (true positive rate) and FPR (false positive rate) it plots two parameters at different threshold levels. The AUC area under the ROC curve quantifies the overall ability of the model to differentiate between negative (BPH) and positive (PCa) classes.</p>
<p id="Par17">Decision curve analysis (DCA) was utilized to assess the clinical utility of the predictive models by quantifying their net benefit across a range of threshold probabilities. DCA compares the proposed model with alternative strategies, such as using PSA alone, treating all patients, or treating none.</p>
<p id="Par18">The threshold probability represents the minimum predicted probability at which a patient would opt for further intervention. It allows for assessment of whether a model provides a higher benefit than treating all patients or none, and how it performs across decision thresholds relevant to clinical practice. The DCA was implemented using predicted probabilities from the trained models, and results were presented as net benefit curves across threshold probabilities ranging from 0.01 to 0.99.</p></section><section id="Sec10"><h4 class="pmc_sec_title">Feature selection</h4>
<p id="Par19">Each biomarker in the discovery cohort was assessed using the Mann–Whitney U test to identify the most significant ones based on their discriminative power between the two classes.</p></section><section id="Sec11"><h4 class="pmc_sec_title">Machine learning</h4>
<p id="Par20">In this study, random forest has been employed as a powerful classifier, combining an ensemble of decision trees to capture the complex patterns of RT-PCR data. In this model, no hyperparameter optimization methods were used in this study, to ensure the reproducibility of the proposed model we have utilised the defaults hyperparameter values from scikit-learn library. The detailed steps used in the modelling has been presented (Supplementary Fig. <a href="#MOESM1" class="usa-link">s1</a>). Each tree within the ensemble is constructed from bootstrapped samples and guided by a randomly selected subset of features, which helps to reduce the correlation between trees and enhances the overall diversity of the model. By working collectively, these decision trees independently vote on each classification, minimizing the influence of any single outlier and improving the robustness of the final classification. The randomness introduced in both sampling and feature selection mitigates overfitting, leading to improved stability and accuracy. Ultimately, the aggregate voting of these trees forms the final prediction in PCa classification. To validate the generalizability and avoid overfitting of the model, a fivefold cross validation has been employed on the verification cohort. The dataset was randomly spitted in 5 different equal subsets, where the model was trained on four folds and tested on the remaining fold iteratively. The final performance metrics were averaged across all five folds.</p></section></section><section id="Sec12"><h3 class="pmc_sec_title">Bioinformatics analysis</h3>
<p id="Par21">To predict the target genes of the 3 miRNAs, we used databases like miRTarBase and miRBase<sup><a href="#CR31" class="usa-link" aria-describedby="CR31">31</a>,<a href="#CR32" class="usa-link" aria-describedby="CR32">32</a></sup>. Target gene lists from both the sources were combined by removing any duplicates. All selected target genes were listed in a common list referred to as miRNA Target Gene Panel (mTGP). This list was used to analyze functional aspects by Gene ontology based functional analysis using ShinyGO tool<sup><a href="#CR33" class="usa-link" aria-describedby="CR33">33</a></sup>. A PCa specific gene panel was created using GeneCards, KEGG and String database<sup><a href="#CR34" class="usa-link" aria-describedby="CR34">34</a></sup>. We employed the keyword “PCA” (Prostate Cancer) to retrieve gene entries from the databases to ensure the inclusion of genes associated specifically with PCa. All the genes reflected from the search were included in the PCa Gene Panel (PGP). To improve the reliability of target gene results, we identified the overlapping target genes from mTGP and PGP by using the VennDiagram package of R software. Then, these overlapping target genes were used to study protein–protein interaction and cancer context specific network construction using PINAv3.0.</p></section></section><section id="Sec13"><h2 class="pmc_sec_title">Results</h2>
<section id="Sec14"><h3 class="pmc_sec_title">miRNA expression analysis in discovery cohort</h3>
<p id="Par22">In the discovery cohort, expression analysis of six selected miRNAs was done using RT-PCR, in both BPH and PCa groups. The Mann–Whitney U test showed that all 6 miRNAs had p-values below 0.05, indicating significance of selected miRNAs. miR-21-5p, miR-221-3p, and miR-141-3p exhibiting the most significant differences, with <em>p</em> values of 0.000224, 0.000153, and 0.004293, respectively, were selected for further study in the verification cohort. Although the distribution pattern showed the bimodal nature of data we still selected these miRNAs for training, because the random forest model has the capability to address non-linearity in the data Additionally miR-19b-3p, miR-93-5p, and miR-375-3p, also displayed significance with <em>p</em> values ranging from 0.004747 to 0.037008 (Fig. <a href="#Fig3" class="usa-link">3</a>, Supplementary Table <a href="#MOESM1" class="usa-link">2</a>). These findings, illustrated by the boxplots of ΔCt values, highlight the potential of these miRNAs as diagnostic biomarkers for PCa.</p>
<figure class="fig xbox font-sm" id="Fig3"><h4 class="obj_head">Fig. 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365097_41598_2025_99754_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6fe0/12365097/215404a44e26/41598_2025_99754_Fig3_HTML.jpg" loading="lazy" id="d33e587" height="367" width="737" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Boxplot showing the Delta Ct value expression in different miRNA’s. Where ‘0’ represents BPH, ‘1’ represents the PCa.</p></figcaption></figure></section><section id="Sec15"><h3 class="pmc_sec_title">ML model development on verification cohort</h3>
<p id="Par23">The verification cohort was used to generate expression data of 3 miRNAs (miR-21-5p, miR-141-3p, miR-221-3p) using RT-PCR. In the verification cohort, 6 features were created—3 individual miRNAs as features (miR-21-5p, miR-141-3p, miR-221-3p) and their ratio-based combinations (21_141, 141_221, and 221_21). The performance of each feature was assessed using ROC curve analysis (Supplementary Fig. <a href="#MOESM1" class="usa-link">s2</a>). The resulting AUC scores ranged from 0.48 to 0.52, indicating that the both individual features and ratios performed poorly when used as independent features suggesting their limited standalone utility in diagnostic predictions.</p>
<p id="Par24">In the verification cohort, a random forest model was applied using all features. The trained ML model showed an accuracy of 77.42%, precision of 86.21%, recall of 71.43%, and an F1 score of 78.12%, as illustrated by the confusion matrix (Fig. <a href="#Fig4" class="usa-link">4</a>a). The ROC curve for the random forest model showed an AUC of 0.78 (Fig. <a href="#Fig4" class="usa-link">4</a>b) as compared to PSA that showed an AUC of 0.52 (Fig. <a href="#Fig4" class="usa-link">4</a>b) similar to random classifier.</p>
<figure class="fig xbox font-sm" id="Fig4"><h4 class="obj_head">Fig. 4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365097_41598_2025_99754_Fig4_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6fe0/12365097/84bd972311d7/41598_2025_99754_Fig4_HTML.jpg" loading="lazy" id="d33e628" height="599" width="780" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>(<strong>a</strong>) Confusion matrix for the model trained with differentially expressed miRNAs on test set in verification cohort; (<strong>b</strong>) ROC AUC graph of proposed miRNA-based model and PSA based model; (<strong>c</strong>) Feature importance employed by random forest model. 141_221 refers to expression ratio of miR-141-3p/miR-221-3p, 21_141 refers to expression ratio of miR-21-5p/miR-141-3p, and 221_21 refers to expression ratio of miR-221-3p/miR-21-5p (<strong>d</strong>) Decision curve analysis of proposed model over PSA.</p></figcaption></figure><p id="Par25">To validate the stability and generalizability of the proposed model a fivefold cross validation was performed on the verification cohort. The cross validated result demonstrated consistent performance across all the folds with average accuracy of 81.67%, average precision of 83.28%, recall of 84%, f1 score of 83.54% and AUC score of 0.81. To rigorously evaluate the proposed method, baseline ML models were implemented using the same feature set for comparative analysis. The random forest model consistently demonstrated superior performance over these alternatives (Supplementary Table <a href="#MOESM1" class="usa-link">3</a>), further substantiating its suitability for the given classification task.</p>
<p id="Par26">The ensemble model, being composed of multiple decision trees, assigns importance to features based on their ability to improve classification accuracy. Further analysis of feature importance, generated by the random forest classifier, revealed miRNAs and feature combinations that contributed most to the model’s predictive accuracy and are the key drivers behind its discriminative performance. As shown in the feature importance plot (Fig. <a href="#Fig4" class="usa-link">4</a>c), the ratio feature of miR-141-3p and miR-221-3P was the most significant contributor, followed closely by the ratio of miR-21-3p and miR-141-3p. Further, decision curve analysis (Fig. <a href="#Fig4" class="usa-link">4</a>d) clearly shows the proposed miRNA based model shows greater clinical net benefit than the PSA based model. Especially in the threshold probability range between 0.2 and 0.7, where it demonstrates realistic clinical decision making.</p></section><section id="Sec16"><h3 class="pmc_sec_title">Validation of designed ML-model on validation cohort</h3>
<p id="Par27">To validate the performance of the trained model, we used an independent cohort that was previously unexposed to the ML-model. When expression data from this validation cohort is employed to the trained ML-model. The testing of the model using data from the validation cohort showed an accuracy of 74.07%. It exhibited a precision of 71.43%, meaning it effectively identified true positive cases, while maintaining a recall (sensitivity) of 76.92%, which reflects the model’s capacity to capture most of the true positive instances of PCa. Furthermore, the model achieved a specificity of 74.07% (Fig. <a href="#Fig5" class="usa-link">5</a>a) and ROC-AUC score of 0.75 (Fig. <a href="#Fig5" class="usa-link">5</a>b).</p>
<figure class="fig xbox font-sm" id="Fig5"><h4 class="obj_head">Fig. 5.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365097_41598_2025_99754_Fig5_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6fe0/12365097/2e55ca8bda45/41598_2025_99754_Fig5_HTML.jpg" loading="lazy" id="d33e668" height="281" width="669" alt="Fig. 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>(<strong>a</strong>) Confusion matrix for the model trained with differentially expressed miRNAs on test set in validation cohort; (<strong>b</strong>) ROC AUC graph for validation cohort.</p></figcaption></figure></section><section id="Sec17"><h3 class="pmc_sec_title">Association of miRNA and their target genes with PCa pathogenesis</h3>
<p id="Par28">We predicted target genes for the selected miRNAs (miR-21-5p, miR-141-3p, and miR-221-3p), identifying 278 genes (mTGP panel). Using ShinyGO, we conducted Gene Ontology (GO) enrichment analysis of mTGP to explore key biological processes (BP), cellular components (CC), and molecular functions (MF), presenting the top 20 in each category. The BP analysis revealed that the genes are primarily involved in regulating metabolic processes related to nitrogen compounds, macromolecules, and nucleobase compounds (Fig. <a href="#Fig6" class="usa-link">6</a>a). In the MF category, the genes were linked to chromatin binding, DNA binding, and kinase activity (Fig. <a href="#Fig6" class="usa-link">6</a>c). The CC analysis showed strong associations with ribonucleoprotein complexes, including the RNA-induced silencing complex (RISC) and transcription regulator complexes (Fig. <a href="#Fig6" class="usa-link">6</a>b). Additionally, KEGG pathway analysis highlighted enrichment in cancer-related pathways, such as miRNAs in cancer, TNF, JAK-STAT, MAPK, and Akt signaling, as well as growth hormone and prolactin synthesis—processes relevant to cancer pathogenesis (Fig. <a href="#Fig6" class="usa-link">6</a>).</p>
<figure class="fig xbox font-sm" id="Fig6"><h4 class="obj_head">Fig. 6.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365097_41598_2025_99754_Fig6_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6fe0/12365097/afc301ad3825/41598_2025_99754_Fig6_HTML.jpg" loading="lazy" id="d33e707" height="460" width="756" alt="Fig. 6"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Gene ontology and KEGG enrichment analysis of target genes for miR-21-5p, miR-141-3p, miR-221-3p. (<strong>a</strong>) Biological processes: shows regulation of metabolic pathways crucial for cellular function and cancer progression. (<strong>b</strong>) Cellular component: reveals key roles in transcription factor binding and kinase activity. (<strong>c</strong>) Molecular function: reveals key roles in transcription factor binding and kinase activity. (<strong>d</strong>) KEGG Pathway analysis highlights enrichment in cancer-related signaling pathways.</p></figcaption></figure><p id="Par29">To determine association of selected miRNAs with PCa progression, identified target genes (mTGP) were screened against PCa gene panel (PGP), revealing 32 shared genes. Result of the chord diagram shows that all 3 miRNA interacted with at least one gene from PGP (Fig. <a href="#Fig7" class="usa-link">7</a><strong>)</strong>. The analysis highlighted miR-21-5p targeting the most PCa-associated genes, followed by miR-141-3p and miR-221-3p, suggesting these miRNAs are critical for PCa biology.</p>
<figure class="fig xbox font-sm" id="Fig7"><h4 class="obj_head">Fig. 7.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365097_41598_2025_99754_Fig7_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6fe0/12365097/47982360a762/41598_2025_99754_Fig7_HTML.jpg" loading="lazy" id="d33e724" height="606" width="643" alt="Fig. 7"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig7/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Chord diagram analysis of miR-21-5p, miR-141-3p, and miR-221-3p and their target genes associated with PCa (Connections between miRNAs and their respective target genes are illustrated using colored arcs).</p></figcaption></figure><p id="Par30">KEGG pathway analysis on 15 genes out of 32 genes that interacted with miR-21-5p showed involvement of genes in general cancer related pathways such as PD-L1 and PD-1 checkpoint, Prolactin, HIF-1, NF-kB etc. Whereas combined analysis of 17 genes interacting with miR-141-3p and miR-221-3p showed more PCa specific pathways involvement such as Endocrine Resistance (Supplementary Fig. <a href="#MOESM1" class="usa-link">s3</a>).</p>
<p id="Par31">Interestingly, the 32 identified genes are functionally interconnected within cancer-specific protein–protein interaction (PPI) networks, as constructed using PINAv3.0 (Supplementary Fig. <a href="#MOESM1" class="usa-link">s4</a>). This analysis revealed seven hub genes—RASA1 (RAS P21 Protein Activator 1), RHOB (Ras Homolog Family Member B), CDKN1B (Cyclin Dependent Kinase Inhibitor 1B), EPHA2 (EPH Receptor A2), ARID1A (AT-Rich Interaction Domain 1A), OGT (O-GlcNAc transferase), and CBX4 (Chromobox 4)—all associated with tumorigenesis. Specifically, RASA1 and ARID1A were strongly linked to the androgen receptor (AR) pathway, which is frequently disrupted in PCa. Other notable interactions showed connections between key oncogenes, such as SMAD2 (SMAD Family Member 2), TNFAIP1(TNF alpha induced protein 1), and CBX8 (Chromobox homolog 8), with these hub genes (Supplementary Fig. <a href="#MOESM1" class="usa-link">s4</a>). Expanding this analysis using STRING for the complete mTGP gene list identified additional critical hub genes, like STAT3, PTEN, CDKN1B (Cyclin Dependent Kinase Inhibitor 1B), and FOS, indicating roles of these proteins in systemic regulations (Supplementary Fig. <a href="#MOESM1" class="usa-link">s5</a>). Overall, the miRNA targets exhibited extensive interactions with both PCa specific genes and generalised oncogenes, underscoring their significant role in the development and progression of the disease through target gene regulation. While these findings support a strong functional link between the selected miRNAs and key regulatory pathways, it is important to note that our analysis was conducted at the individual miRNA level and does not directly account for the miRNA ratios used in the machine learning model. Additionally, further functional validation is required to elucidate the role of these genes in PCa pathogenesis.</p></section></section><section id="Sec18"><h2 class="pmc_sec_title">Discussion and conclusion</h2>
<p id="Par32">PCa diagnosis remains challenging, particularly in differentiating it from BPH, as both conditions share overlapping clinical features. The reliance on PSA tests has led to high false-positive rates, resulting in unnecessary biopsies and significant patient anxiety<sup><a href="#CR35" class="usa-link" aria-describedby="CR35">35</a>,<a href="#CR36" class="usa-link" aria-describedby="CR36">36</a></sup>. These limitations highlight the urgent need for reliable, non-invasive biomarkers capable of accurately distinguishing between PCa and BPH.</p>
<section id="Sec19"><h3 class="pmc_sec_title">Integration of miRNA profiling and machine learning</h3>
<p id="Par33">miRNAs are promising candidates for non-invasive diagnostics due to their stability in circulation and ability to reflect tumor biology. Most studies on miRNA biomarkers for PCa have primarily examined serum and plasma<sup><a href="#CR37" class="usa-link" aria-describedby="CR37">37</a></sup>, with limited available research on whole blood. However, promising results have been reported for other cancers such as breast, pancreatic, and lung cancers using whole blood-based miRNA profiling. Our study utilized miRNAs previously identified in studies and evaluated their diagnostic potential in whole blood. Whole blood offers key advantages, including higher miRNA yield and robust systemic representation of disease states, making it a valuable biofluid for biomarker discovery. The complexity of whole blood, which contains miRNAs from multiple cellular sources, could possibly introduce noise. To improve standardization and reproducibility, future studies should systematically compare miRNA expression across different biofluids, ensuring consistency in diagnostic applications. However, the ensemble-based random forest method used in the study mitigates this challenge by handling non-linear relationships and reducing sensitivity to noise<sup><a href="#CR38" class="usa-link" aria-describedby="CR38">38</a></sup>. Further, the fivefold validation proves generalizability of the proposed model, supporting that the model is able to handle unseen data and avoid overfitting. This study leverages a novel combination of miRNA profiling and ML to enhance diagnostic precision for PCa. While individual miRNAs, such as miR-21-5p, miR-141-3p, and miR-221-3p, have been implicated in PCa progression in previous studies<sup><a href="#CR39" class="usa-link" aria-describedby="CR39">39</a>–<a href="#CR41" class="usa-link" aria-describedby="CR41">41</a></sup>, the current work innovates by exploring the application of machine learning tools and miRNA’s expression profile showing that use of ML demonstrated superior discriminatory power in distinguishing PCa from BPH, capturing synergistic effects overlooked in standalone miRNA analyses and linear models respectively.</p>
<p id="Par34">The random forest classifier was chosen in this study due to its ability to capture non-linear relationships and complex feature interactions, leading to an AUC-ROC score of 0.78. The results demonstrated that our machine learning model outperformed PSA, which suffers from high false-positive rates, achieving a higher AUC-ROC score and offering greater net benefit across a range of threshold probabilities in DCA. Unlike models reliant on fixed Ct-value thresholds, our ML approach dynamically adjusts to data variability, improving sensitivity and specificity. These findings suggest that miRNA-based diagnostics, when integrated with ML approaches, could provide a more accurate and clinically relevant tool for PCa detection and risk stratification, reducing unnecessary biopsies while maintaining high sensitivity.</p></section><section id="Sec20"><h3 class="pmc_sec_title">Biological interpretation of miRNA findings</h3>
<p id="Par35">To address concerns regarding the “black box” nature of ML models, we incorporated feature importance rankings and bioinformatics analyses to validate the biological relevance of the key features identified by the model. The miRNA ratios miR-141-3p/miR-221-3p and miR-21-5p/miR-141-3p were confirmed as critical features for distinguishing PCa from BPH.</p>
<p id="Par36">KEGG pathway enrichment analysis linked miR-21-5p to cancer-related pathways, including PD-L1/PD-1 checkpoint regulation, prolactin signaling, HIF-1, and NF-κB signaling, all of which play crucial roles in immune evasion, angiogenesis, and inflammation. In contrast, miR-141-3p and miR-221-3p were associated with androgen receptor (AR) signaling and endocrine resistance, which are critical pathways in hormone-sensitive and castration-resistant PCa. These findings suggest a potential regulatory role of these miRNAs in PCa progression, but further functional validation is required to confirm their direct involvement in tumor development and progression. Interestingly, target gene analysis revealed both oncogenes and tumor suppressors within the hub gene network, including: EPHA2, CBX8, STAT3, SMAD2 (context-dependent), TNFAIP1 as oncogenes and RASA1, RHOB, CDKN1B, ARID1A, OGT, CBX4, PTEN, FOS (context-dependent) as tumor suppressors. To better understand how these hub genes influence tumor progression and response to therapy, future research should focus on longitudinal expression studies and functional assays. The coexistence of both oncogenes and tumor suppressors in the hub gene network may initially appear counterintuitive. However, this reflects the complex regulatory interactions within cancer biology, where genes can have dual roles depending on cellular context, mutation status, and signaling interactions. These findings also emphasize the complex regulatory landscape of miRNA expression in PCa and suggest that miRNA profiling in exosomal fractions or immune cell subsets may provide deeper insights.</p></section><section id="Sec21"><h3 class="pmc_sec_title">Limitations and future directions</h3>
<p id="Par37">While our model demonstrated the ability to generalize, an essential requirement for real-world clinical applications, several limitations must be acknowledged. The study’s findings are based on a limited cohort predominantly from a single population, necessitating further validation across diverse genetic, environmental, and clinical settings. In the future we are focused on large-scale, multi-center study to validate the model across various populations. Other models, such as XGBoost, support vector machines (SVM), and deep learning, could be explored for scalability and improved predictive power in larger datasets. However, such deep learning methods typically require larger training datasets and extensive computational resources, which were beyond the scope of this study.</p>
<p id="Par38">One of the major challenges in translating miRNA-based diagnostics into clinical practice is the lack of standardized protocols. Variations in sample processing methods, RT-PCR platforms, cut-off values, and reference genes can introduce inconsistencies that hinder cross-study validation. Standardization efforts, including unified Ct value normalization methods and consensus guidelines for miRNA biomarker validation, are crucial to improving reproducibility and clinical utility. Future work should focus on establishing standardized protocols for reproducibility across different platforms.</p>
<p id="Par39">To further strengthen the external validation of our findings, future studies should consider leveraging publicly available datasets. These datasets provide valuable large-scale transcriptomic data across diverse patient populations and could help assess the generalizability of our model. However, integrating such datasets presents challenges, as they often involve heterogeneous sample types (e.g., plasma, serum, urine) and different profiling platforms (e.g., RNA sequencing, microarrays, RT-PCR), leading to technical variability. Addressing these discrepancies would require robust normalization strategies and cross-platform data harmonization to ensure comparability with the current model. A research effort focused on developing computational approaches for cross-platform normalization strategies could be highly valuable.</p>
<p id="Par40">The integration of miRNA-based ML models with multiparametric MRI (mpMRI) is also a promising avenue for enhancing PCa diagnosis and risk stratification. While mpMRI is widely used to assess prostate lesions and guide biopsies, its accuracy can be limited by inter-reader variability and false-positive findings. Combining molecular biomarkers, such as miRNA signatures, with radiological features (e.g., lesion morphology, diffusion-weighted imaging parameters) could improve diagnostic precision.</p>
<p id="Par41">In Conclusion, The integration of miRNA profiling with ML offers a promising approach for improving PCa diagnostics. By leveraging miRNA expression ratios and ensemble-based models, this study demonstrated enhanced diagnostic accuracy, surpassing traditional PSA-based approaches. The biological validation of key miRNA biomarkers supports their clinical potential, while model validation underscores its reliability. Future research focused on large-scale validation, standardization, and multimodal integration will be crucial in advancing this approach toward clinical implementation.</p></section></section><section id="Sec22"><h2 class="pmc_sec_title">Electronic supplementary material</h2>
<p>Below is the link to the electronic supplementary material.</p>
<section class="sm xbox font-sm" id="MOESM1"><div class="media p"><div class="caption">
<a href="/articles/instance/12365097/bin/41598_2025_99754_MOESM1_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 1</a><sup> (2.3MB, docx) </sup>
</div></div></section></section><section id="notes1"><h2 class="pmc_sec_title">Author contributions</h2>
<p>Research conceptualization and funding acquisition: Jain G; Supervision and validation: Jain G; Sample collection and/or data curation: Singh S and Kural S; miRNAs analysis workflow: Singh S; Formal data analysis and interpretation: Singh S, Jain G, Pathak AK, Gupta M and Kural S; Visualization: Singh S, Pathak AK and Kural S; Writing-original draft: Singh S, Pathak AK and Jain G; Resources: Yadav M and Bhardwaj MG; Clinical advisory: Trivedi S, Kumar L and Yadav M; Model optimization assistance: Pathak AK and Gupta M Writing-review and editing: Singh S, Pathak AK, Kural S, Jain G, Trivedi S and Das P.</p></section><section id="notes2"><h2 class="pmc_sec_title">Funding</h2>
<p>This research was supported by the grant from BIRAC-India, under AGC-JanCare scheme to MIRNOW. The Institute of Eminence (IoE) Scheme at BHU, Fellowship/Grant support to Dr. Garima Jain, Banaras Hindu University. Non-Net fellowship support to AP and SK, and MIRNOW employee support to SS. We would also like to acknowledge the Centre for Genetic Disorders, BHU for providing laboratory resources and facilities as well as the Department of Urology, BHU for facilitating sample collection. We extend our sincere gratitude to all collaborators and patients involved in this study.</p></section><section id="notes3"><h2 class="pmc_sec_title">Data availability</h2>
<p>The data generated and analyzed in this study are available from the corresponding author upon reasonable request. To ensure privacy and comply with ethical guidelines, access to patient related data is restricted. However, anonymized datasets and supplementary materials used in this study will be provided on request for research and verification purposes, subject to institutional approval and appropriate data-sharing agreements.</p></section><section id="notes4"><h2 class="pmc_sec_title">Declarations</h2>
<section id="FPar1"><h3 class="pmc_sec_title">Competing interests</h3>
<p id="Par42">The authors declare no competing interests.</p></section></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm">
<div class="fn p" id="fn1">
<p><strong>Publisher’s note</strong></p>
<p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p>
</div>
<div class="fn p" id="fn2"><p>Shweta Singh, Abhay Kumar Pathak, and Garima Jain contributed equally as first authors.</p></div>
</div></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Condrat, C. E. et al. miRNAs as biomarkers in disease: Latest findings regarding their role in diagnosis and prognosis. <em>Cells</em><strong>9</strong>, 276 (2020).
</cite> [<a href="https://doi.org/10.3390/cells9020276" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7072450/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31979244/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Condrat,%20C.%20E.%20et%20al.%20miRNAs%20as%20biomarkers%20in%20disease:%20Latest%20findings%20regarding%20their%20role%20in%20diagnosis%20and%20prognosis.%20Cells9,%20276%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>Sempere, L. F., Azmi, A. S. &amp; Moore, A. microRNA-based diagnostic and therapeutic applications in cancer medicine. <em>WIREs RNA</em><strong>12</strong>, e1662 (2021).
</cite> [<a href="https://doi.org/10.1002/wrna.1662" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8519065/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33998154/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sempere,%20L.%20F.,%20Azmi,%20A.%20S.%20&amp;%20Moore,%20A.%20microRNA-based%20diagnostic%20and%20therapeutic%20applications%20in%20cancer%20medicine.%20WIREs%20RNA12,%20e1662%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>Khamina, K. et al. A microRNA next-generation-sequencing discovery assay (miND) for genome-scale analysis and absolute quantitation of circulating microRNA biomarkers. <em>Int. J. Mol. Sci.</em><strong>23</strong>, 1226 (2022).
</cite> [<a href="https://doi.org/10.3390/ijms23031226" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8835905/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35163149/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Khamina,%20K.%20et%20al.%20A%20microRNA%20next-generation-sequencing%20discovery%20assay%20(miND)%20for%20genome-scale%20analysis%20and%20absolute%20quantitation%20of%20circulating%20microRNA%20biomarkers.%20Int.%20J.%20Mol.%20Sci.23,%201226%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Constâncio, V., Tavares, N. T., Henrique, R., Jerónimo, C. &amp; Lobo, J. MiRNA biomarkers in cancers of the male reproductive system: Are we approaching clinical application?. <em>Andrology</em><strong>11</strong>, 651–667 (2023).
</cite> [<a href="https://doi.org/10.1111/andr.13258" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35930290/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Const%C3%A2ncio,%20V.,%20Tavares,%20N.%20T.,%20Henrique,%20R.,%20Jer%C3%B3nimo,%20C.%20&amp;%20Lobo,%20J.%20MiRNA%20biomarkers%20in%20cancers%20of%20the%20male%20reproductive%20system:%20Are%20we%20approaching%20clinical%20application?.%20Andrology11,%20651%E2%80%93667%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Muñoz, J. P., Pérez-Moreno, P., Pérez, Y. &amp; Calaf, G. M. The role of MicroRNAs in breast cancer and the challenges of their clinical application. <em>Diagnostics</em><strong>13</strong>, 3072 (2023).
</cite> [<a href="https://doi.org/10.3390/diagnostics13193072" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10572677/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37835815/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mu%C3%B1oz,%20J.%20P.,%20P%C3%A9rez-Moreno,%20P.,%20P%C3%A9rez,%20Y.%20&amp;%20Calaf,%20G.%20M.%20The%20role%20of%20MicroRNAs%20in%20breast%20cancer%20and%20the%20challenges%20of%20their%20clinical%20application.%20Diagnostics13,%203072%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Pritchard, C. C., Cheng, H. H. &amp; Tewari, M. MicroRNA profiling: Approaches and considerations. <em>Nat. Rev. Genet.</em><strong>13</strong>, 358–369 (2012).
</cite> [<a href="https://doi.org/10.1038/nrg3198" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4517822/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22510765/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Pritchard,%20C.%20C.,%20Cheng,%20H.%20H.%20&amp;%20Tewari,%20M.%20MicroRNA%20profiling:%20Approaches%20and%20considerations.%20Nat.%20Rev.%20Genet.13,%20358%E2%80%93369%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Seputra, K. P., Purnomo, B. B., Susianti, H., Kalim, H. &amp; Purnomo, A. F. miRNA-21 serum evaluation in BPH, hormone sensitive prostate cancer, and castrate resistant prostate cancer: Attempt for diagnostic biomarker evaluation. <em>Acta Inform. Med.</em><strong>29</strong>, 266 (2021).
</cite> [<a href="https://doi.org/10.5455/aim.2021.29.266-269" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8800573/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35197661/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Seputra,%20K.%20P.,%20Purnomo,%20B.%20B.,%20Susianti,%20H.,%20Kalim,%20H.%20&amp;%20Purnomo,%20A.%20F.%20miRNA-21%20serum%20evaluation%20in%20BPH,%20hormone%20sensitive%20prostate%20cancer,%20and%20castrate%20resistant%20prostate%20cancer:%20Attempt%20for%20diagnostic%20biomarker%20evaluation.%20Acta%20Inform.%20Med.29,%20266%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>Li, C. et al. MiRNA-199a-3p in plasma as a potential diagnostic biomarker for gastric cancer. <em>Ann. Surg. Oncol.</em><strong>20</strong>, 397–405 (2013).</cite> [<a href="https://doi.org/10.1245/s10434-012-2600-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22956063/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li,%20C.%20et%20al.%20MiRNA-199a-3p%20in%20plasma%20as%20a%20potential%20diagnostic%20biomarker%20for%20gastric%20cancer.%20Ann.%20Surg.%20Oncol.20,%20397%E2%80%93405%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>Chen, M. &amp; Zhao, H. Next-generation sequencing in liquid biopsy: Cancer screening and early detection. <em>Hum. Genom.</em><strong>13</strong>, 34 (2019).</cite> [<a href="https://doi.org/10.1186/s40246-019-0220-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6669976/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31370908/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chen,%20M.%20&amp;%20Zhao,%20H.%20Next-generation%20sequencing%20in%20liquid%20biopsy:%20Cancer%20screening%20and%20early%20detection.%20Hum.%20Genom.13,%2034%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>McDonald, J. S., Milosevic, D., Reddi, H. V., Grebe, S. K. &amp; Algeciras-Schimnich, A. Analysis of circulating microRNA: Preanalytical and analytical challenges. <em>Clin. Chem.</em><strong>57</strong>, 833–840 (2011).
</cite> [<a href="https://doi.org/10.1373/clinchem.2010.157198" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21487102/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?McDonald,%20J.%20S.,%20Milosevic,%20D.,%20Reddi,%20H.%20V.,%20Grebe,%20S.%20K.%20&amp;%20Algeciras-Schimnich,%20A.%20Analysis%20of%20circulating%20microRNA:%20Preanalytical%20and%20analytical%20challenges.%20Clin.%20Chem.57,%20833%E2%80%93840%20(2011)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Pathak, A. K. et al. Development of a robust and generalizable algorithm “gQuant” for accurate normalizer gene selection in qRT-PCR analysis. <em>Sci. Rep.</em><strong>14</strong>, 18774 (2024).
</cite> [<a href="https://doi.org/10.1038/s41598-024-66770-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11322313/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39138232/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Pathak,%20A.%20K.%20et%20al.%20Development%20of%20a%20robust%20and%20generalizable%20algorithm%20%E2%80%9CgQuant%E2%80%9D%20for%20accurate%20normalizer%20gene%20selection%20in%20qRT-PCR%20analysis.%20Sci.%20Rep.14,%2018774%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Greener, J. G., Kandathil, S. M., Moffat, L. &amp; Jones, D. T. A guide to machine learning for biologists. <em>Nat. Rev. Mol. Cell Biol.</em><strong>23</strong>, 40–55 (2022).
</cite> [<a href="https://doi.org/10.1038/s41580-021-00407-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34518686/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Greener,%20J.%20G.,%20Kandathil,%20S.%20M.,%20Moffat,%20L.%20&amp;%20Jones,%20D.%20T.%20A%20guide%20to%20machine%20learning%20for%20biologists.%20Nat.%20Rev.%20Mol.%20Cell%20Biol.23,%2040%E2%80%9355%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>Jiang, T., Gradus, J. L. &amp; Rosellini, A. J. Supervised machine learning: A brief primer. <em>Behav. Ther.</em><strong>51</strong>, 675–687 (2020).
</cite> [<a href="https://doi.org/10.1016/j.beth.2020.05.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7431677/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32800297/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jiang,%20T.,%20Gradus,%20J.%20L.%20&amp;%20Rosellini,%20A.%20J.%20Supervised%20machine%20learning:%20A%20brief%20primer.%20Behav.%20Ther.51,%20675%E2%80%93687%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>Rigatti, S. J. Random forest. <em>J. Insur. Med. N. Y.</em><strong>47</strong>, 31–39 (2017).</cite> [<a href="https://doi.org/10.17849/insm-47-01-31-39.1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28836909/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rigatti,%20S.%20J.%20Random%20forest.%20J.%20Insur.%20Med.%20N.%20Y.47,%2031%E2%80%9339%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>Reel, P. S., Reel, S., Pearson, E., Trucco, E. &amp; Jefferson, E. Using machine learning approaches for multi-omics data analysis: A review. <em>Biotechnol. Adv.</em><strong>49</strong>, 107739 (2021).
</cite> [<a href="https://doi.org/10.1016/j.biotechadv.2021.107739" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33794304/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Reel,%20P.%20S.,%20Reel,%20S.,%20Pearson,%20E.,%20Trucco,%20E.%20&amp;%20Jefferson,%20E.%20Using%20machine%20learning%20approaches%20for%20multi-omics%20data%20analysis:%20A%20review.%20Biotechnol.%20Adv.49,%20107739%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Bostanci, E. et al. Machine learning analysis of RNA-seq data for diagnostic and prognostic prediction of colon cancer. <em>Sensors</em><strong>23</strong>, 3080 (2023).
</cite> [<a href="https://doi.org/10.3390/s23063080" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10052105/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36991790/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bostanci,%20E.%20et%20al.%20Machine%20learning%20analysis%20of%20RNA-seq%20data%20for%20diagnostic%20and%20prognostic%20prediction%20of%20colon%20cancer.%20Sensors23,%203080%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>Sharova, E. et al. A circulating miRNA assay as a first-line test for prostate cancer screening. <em>Br. J. Cancer</em><strong>114</strong>, 1362–1366 (2016).
</cite> [<a href="https://doi.org/10.1038/bjc.2016.151" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4984473/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27228285/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sharova,%20E.%20et%20al.%20A%20circulating%20miRNA%20assay%20as%20a%20first-line%20test%20for%20prostate%20cancer%20screening.%20Br.%20J.%20Cancer114,%201362%E2%80%931366%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Pawelka, D. et al. Machine-learning-based analysis identifies miRNA expression profile for diagnosis and prediction of colorectal cancer: A preliminary study. <em>Cancer Genom. Proteom.</em><strong>19</strong>, 503–511 (2022).</cite> [<a href="https://doi.org/10.21873/cgp.20336" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9247881/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35732322/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Pawelka,%20D.%20et%20al.%20Machine-learning-based%20analysis%20identifies%20miRNA%20expression%20profile%20for%20diagnosis%20and%20prediction%20of%20colorectal%20cancer:%20A%20preliminary%20study.%20Cancer%20Genom.%20Proteom.19,%20503%E2%80%93511%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Balk, S. P., Ko, Y.-J. &amp; Bubley, G. J. Biology of prostate-specific antigen. <em>J. Clin. Oncol.</em><strong>21</strong>, 383–391 (2003).
</cite> [<a href="https://doi.org/10.1200/JCO.2003.02.083" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12525533/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Balk,%20S.%20P.,%20Ko,%20Y.-J.%20&amp;%20Bubley,%20G.%20J.%20Biology%20of%20prostate-specific%20antigen.%20J.%20Clin.%20Oncol.21,%20383%E2%80%93391%20(2003)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>McNally, C. J., Ruddock, M. W., Moore, T. &amp; McKenna, D. J. Biomarkers that differentiate benign prostatic hyperplasia from prostate cancer: A literature review. <em>Cancer Manag. Res.</em><strong>12</strong>, 5225–5241 (2020).
</cite> [<a href="https://doi.org/10.2147/CMAR.S250829" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7335899/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32669872/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?McNally,%20C.%20J.,%20Ruddock,%20M.%20W.,%20Moore,%20T.%20&amp;%20McKenna,%20D.%20J.%20Biomarkers%20that%20differentiate%20benign%20prostatic%20hyperplasia%20from%20prostate%20cancer:%20A%20literature%20review.%20Cancer%20Manag.%20Res.12,%205225%E2%80%935241%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>Nickel, J. C. Inflammation and benign prostatic hyperplasia. <em>Urol. Clin. N. Am.</em><strong>35</strong>, 109–115 (2008).</cite> [<a href="https://doi.org/10.1016/j.ucl.2007.09.012" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2198849/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18061029/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Nickel,%20J.%20C.%20Inflammation%20and%20benign%20prostatic%20hyperplasia.%20Urol.%20Clin.%20N.%20Am.35,%20109%E2%80%93115%20(2008)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>Munteanu, V. C. et al. MiRNA-based inspired approach in diagnosis of prostate cancer. <em>Medicina (Mex.)</em><strong>56</strong>, 94 (2020).</cite> [<a href="https://doi.org/10.3390/medicina56020094" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7074198/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32102477/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Munteanu,%20V.%20C.%20et%20al.%20MiRNA-based%20inspired%20approach%20in%20diagnosis%20of%20prostate%20cancer.%20Medicina%20(Mex.)56,%2094%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>Fabris, L. et al. The potential of microRNAs as prostate cancer biomarkers. <em>Eur. Urol.</em><strong>70</strong>, 312–322 (2016).
</cite> [<a href="https://doi.org/10.1016/j.eururo.2015.12.054" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4927364/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26806656/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Fabris,%20L.%20et%20al.%20The%20potential%20of%20microRNAs%20as%20prostate%20cancer%20biomarkers.%20Eur.%20Urol.70,%20312%E2%80%93322%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>Jain, G. et al. Urinary extracellular vesicles miRNA—A new era of prostate cancer biomarkers. <em>Front. Genet.</em><strong>14</strong>, 1065757 (2023).
</cite> [<a href="https://doi.org/10.3389/fgene.2023.1065757" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9895092/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36741322/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jain,%20G.%20et%20al.%20Urinary%20extracellular%20vesicles%20miRNA%E2%80%94A%20new%20era%20of%20prostate%20cancer%20biomarkers.%20Front.%20Genet.14,%201065757%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>Yaman Agaoglu, F. et al. Investigation of miR-21, miR-141, and miR-221 in blood circulation of patients with prostate cancer. <em>Tumour Biol. J. Int. Soc. Oncodev. Biol. Med.</em><strong>32</strong>, 583–588 (2011).</cite> [<a href="https://doi.org/10.1007/s13277-011-0154-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21274675/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yaman%20Agaoglu,%20F.%20et%20al.%20Investigation%20of%20miR-21,%20miR-141,%20and%20miR-221%20in%20blood%20circulation%20of%20patients%20with%20prostate%20cancer.%20Tumour%20Biol.%20J.%20Int.%20Soc.%20Oncodev.%20Biol.%20Med.32,%20583%E2%80%93588%20(2011)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>Joković, S. M. et al. MiR-375 and miR-21 as potential biomarkers of prostate cancer: Comparison of matching samples of plasma and exosomes. <em>Genes</em><strong>13</strong>, 2320 (2022).
</cite> [<a href="https://doi.org/10.3390/genes13122320" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9778022/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36553586/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jokovi%C4%87,%20S.%20M.%20et%20al.%20MiR-375%20and%20miR-21%20as%20potential%20biomarkers%20of%20prostate%20cancer:%20Comparison%20of%20matching%20samples%20of%20plasma%20and%20exosomes.%20Genes13,%202320%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Porzycki, P., Ciszkowicz, E., Semik, M. &amp; Tyrka, M. Combination of three miRNA (miR-141, miR-21, and miR-375) as potential diagnostic tool for prostate cancer recognition. <em>Int. Urol. Nephrol.</em><strong>50</strong>, 1619–1626 (2018).
</cite> [<a href="https://doi.org/10.1007/s11255-018-1938-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6133127/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30014459/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Porzycki,%20P.,%20Ciszkowicz,%20E.,%20Semik,%20M.%20&amp;%20Tyrka,%20M.%20Combination%20of%20three%20miRNA%20(miR-141,%20miR-21,%20and%20miR-375)%20as%20potential%20diagnostic%20tool%20for%20prostate%20cancer%20recognition.%20Int.%20Urol.%20Nephrol.50,%201619%E2%80%931626%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Konoshenko, M. Y. et al. The panel of 12 cell-free microRNAs as potential biomarkers in prostate neoplasms. <em>Diagnostics</em><strong>10</strong>, 38 (2020).
</cite> [<a href="https://doi.org/10.3390/diagnostics10010038" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7168237/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31936850/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Konoshenko,%20M.%20Y.%20et%20al.%20The%20panel%20of%2012%20cell-free%20microRNAs%20as%20potential%20biomarkers%20in%20prostate%20neoplasms.%20Diagnostics10,%2038%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>Ciszkowicz, E., Porzycki, P., Semik, M., Kaznowska, E. &amp; Tyrka, M. MiR-93/miR-375: Diagnostic potential, aggressiveness correlation and common target genes in prostate cancer. <em>Int. J. Mol. Sci.</em><strong>21</strong>, 5667 (2020).
</cite> [<a href="https://doi.org/10.3390/ijms21165667" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7460886/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32784653/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ciszkowicz,%20E.,%20Porzycki,%20P.,%20Semik,%20M.,%20Kaznowska,%20E.%20&amp;%20Tyrka,%20M.%20MiR-93/miR-375:%20Diagnostic%20potential,%20aggressiveness%20correlation%20and%20common%20target%20genes%20in%20prostate%20cancer.%20Int.%20J.%20Mol.%20Sci.21,%205667%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR30">
<span class="label">30.</span><cite>Wen, W. et al. Identification of two microRNA signatures in whole blood as novel biomarkers for diagnosis of nasopharyngeal carcinoma. <em>J. Transl. Med.</em><strong>17</strong>, 186 (2019).
</cite> [<a href="https://doi.org/10.1186/s12967-019-1923-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6547589/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31159814/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wen,%20W.%20et%20al.%20Identification%20of%20two%20microRNA%20signatures%20in%20whole%20blood%20as%20novel%20biomarkers%20for%20diagnosis%20of%20nasopharyngeal%20carcinoma.%20J.%20Transl.%20Med.17,%20186%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>Huang, H.-Y. et al. miRTarBase 2020: Updates to the experimentally validated microRNA-target interaction database. <em>Nucleic Acids Res.</em><strong>48</strong>, D148–D154 (2020).
</cite> [<a href="https://doi.org/10.1093/nar/gkz896" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7145596/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31647101/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Huang,%20H.-Y.%20et%20al.%20miRTarBase%202020:%20Updates%20to%20the%20experimentally%20validated%20microRNA-target%20interaction%20database.%20Nucleic%20Acids%20Res.48,%20D148%E2%80%93D154%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>Griffiths-Jones, S., Saini, H. K., van Dongen, S. &amp; Enright, A. J. miRBase: Tools for microRNA genomics. <em>Nucleic Acids Res.</em><strong>36</strong>, D154-158 (2008).
</cite> [<a href="https://doi.org/10.1093/nar/gkm952" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2238936/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17991681/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Griffiths-Jones,%20S.,%20Saini,%20H.%20K.,%20van%20Dongen,%20S.%20&amp;%20Enright,%20A.%20J.%20miRBase:%20Tools%20for%20microRNA%20genomics.%20Nucleic%20Acids%20Res.36,%20D154-158%20(2008)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Ge, S. X., Jung, D. &amp; Yao, R. ShinyGO: A graphical gene-set enrichment tool for animals and plants. <em>Bioinformatics</em><strong>36</strong>, 2628–2629 (2020).
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btz931" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7178415/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31882993/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ge,%20S.%20X.,%20Jung,%20D.%20&amp;%20Yao,%20R.%20ShinyGO:%20A%20graphical%20gene-set%20enrichment%20tool%20for%20animals%20and%20plants.%20Bioinformatics36,%202628%E2%80%932629%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>Safran, M. et al. GeneCards Version 3: The human gene integrator. <em>Database J. Biol. Databases Curation</em><strong>2010</strong>, 20 (2010).</cite> [<a href="https://doi.org/10.1093/database/baq020" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2938269/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20689021/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Safran,%20M.%20et%20al.%20GeneCards%20Version%203:%20The%20human%20gene%20integrator.%20Database%20J.%20Biol.%20Databases%20Curation2010,%2020%20(2010)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>Lumbreras, B. et al. Variables associated with false-positive PSA results: A cohort study with real-world data. <em>Cancers</em><strong>15</strong>, 261 (2022).
</cite> [<a href="https://doi.org/10.3390/cancers15010261" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9818944/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36612257/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lumbreras,%20B.%20et%20al.%20Variables%20associated%20with%20false-positive%20PSA%20results:%20A%20cohort%20study%20with%20real-world%20data.%20Cancers15,%20261%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR36">
<span class="label">36.</span><cite>Kumar, L., Kumar, S. &amp; Agarwal, S. To find the role of DNA damage marker 8-hydroxy 2-deoxy guanosine in patients of prostate cancer, benign prostatic hyperplasia and its association to other prognostic factors of prostate cancer. <em>J. Integr. Nephrol. Androl.</em><strong>4</strong>, 55–55 (2017).</cite> [<a href="https://scholar.google.com/scholar_lookup?Kumar,%20L.,%20Kumar,%20S.%20&amp;%20Agarwal,%20S.%20To%20find%20the%20role%20of%20DNA%20damage%20marker%208-hydroxy%202-deoxy%20guanosine%20in%20patients%20of%20prostate%20cancer,%20benign%20prostatic%20hyperplasia%20and%20its%20association%20to%20other%20prognostic%20factors%20of%20prostate%20cancer.%20J.%20Integr.%20Nephrol.%20Androl.4,%2055%E2%80%9355%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR37">
<span class="label">37.</span><cite>Endzeliņš, E. et al. Diagnostic, prognostic and predictive value of cell-free miRNAs in prostate cancer: A systematic review. <em>Mol. Cancer</em><strong>15</strong>, 41 (2016).
</cite> [<a href="https://doi.org/10.1186/s12943-016-0523-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4870749/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27189160/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Endzeli%C5%86%C5%A1,%20E.%20et%20al.%20Diagnostic,%20prognostic%20and%20predictive%20value%20of%20cell-free%20miRNAs%20in%20prostate%20cancer:%20A%20systematic%20review.%20Mol.%20Cancer15,%2041%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR38">
<span class="label">38.</span><cite>Breiman, L. Random forests. <em>Mach. Learn.</em><strong>45</strong>, 5–32 (2001).</cite> [<a href="https://scholar.google.com/scholar_lookup?Breiman,%20L.%20Random%20forests.%20Mach.%20Learn.45,%205%E2%80%9332%20(2001)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR39">
<span class="label">39.</span><cite>Huang, S. et al. Downregulation of miR-141-3p promotes bone metastasis via activating NF-κB signaling in prostate cancer. <em>J. Exp. Clin. Cancer Res.</em><strong>36</strong>, 173 (2017).
</cite> [<a href="https://doi.org/10.1186/s13046-017-0645-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5716366/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29202848/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Huang,%20S.%20et%20al.%20Downregulation%20of%20miR-141-3p%20promotes%20bone%20metastasis%20via%20activating%20NF-%CE%BAB%20signaling%20in%20prostate%20cancer.%20J.%20Exp.%20Clin.%20Cancer%20Res.36,%20173%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR40">
<span class="label">40.</span><cite>Li, D. et al. Elevated miR-221-3p inhibits epithelial–mesenchymal transition and biochemical recurrence of prostate cancer via targeting KPNA2: An evidence-based and knowledge-guided strategy. <em>BMC Cancer</em><strong>25</strong>, 34 (2025).
</cite> [<a href="https://doi.org/10.1186/s12885-025-13444-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11708077/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39780096/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li,%20D.%20et%20al.%20Elevated%20miR-221-3p%20inhibits%20epithelial%E2%80%93mesenchymal%20transition%20and%20biochemical%20recurrence%20of%20prostate%20cancer%20via%20targeting%20KPNA2:%20An%20evidence-based%20and%20knowledge-guided%20strategy.%20BMC%20Cancer25,%2034%20(2025)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR41">
<span class="label">41.</span><cite>Ribas, J. et al. miR-21: An androgen receptor-regulated microRNA that promotes hormone-dependent and hormone-independent prostate cancer growth. <em>Cancer Res.</em><strong>69</strong>, 7165–7169 (2009).
</cite> [<a href="https://doi.org/10.1158/0008-5472.CAN-09-1448" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2861586/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19738047/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ribas,%20J.%20et%20al.%20miR-21:%20An%20androgen%20receptor-regulated%20microRNA%20that%20promotes%20hormone-dependent%20and%20hormone-independent%20prostate%20cancer%20growth.%20Cancer%20Res.69,%207165%E2%80%937169%20(2009)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12365097/bin/41598_2025_99754_MOESM1_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 1</a><sup> (2.3MB, docx) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>The data generated and analyzed in this study are available from the corresponding author upon reasonable request. To ensure privacy and comply with ethical guidelines, access to patient related data is restricted. However, anonymized datasets and supplementary materials used in this study will be provided on request for research and verification purposes, subject to institutional approval and appropriate data-sharing agreements.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Scientific Reports are provided here courtesy of <strong>Nature Publishing Group</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1038/s41598-025-99754-7"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/41598_2025_Article_99754.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (2.1 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12365097/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12365097/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12365097%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365097/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12365097/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12365097/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40830224/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12365097/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40830224/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12365097/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12365097/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="kY4BTsQH8QtZVuuZLREEEPbfmiV9geQUnky6M6oBCR4Bv5gBShp0XkyAKIKx1ceT">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
