
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Detergent‐free isolation and characterization of amyloid precursor protein C99 in E. coli native lipid‐nanodiscs using non‐ionic polymer - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="AE530E498AEEC2B3050E4900364D74E1.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="prosci">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12363408/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Protein Science : A Publication of the Protein Society">
<meta name="citation_title" content="Detergent‐free isolation and characterization of amyloid precursor protein C99 in E. coli native lipid‐nanodiscs using non‐ionic polymer">
<meta name="citation_author" content="Gaurav Sharma">
<meta name="citation_author_institution" content="Biophysics Program, The University of Michigan, Ann Arbor, Michigan, USA">
<meta name="citation_author_institution" content="Department of Chemistry, The University of Michigan, Ann Arbor, Michigan, USA">
<meta name="citation_author_institution" content="Biomedical Engineering, Michigan Institute for Neuroscience, Macromolecular Science and Engineering, The University of Michigan, Ann Arbor, Michigan, USA">
<meta name="citation_author_institution" content="Present address: T.N. SYS Meryl Pharma Ujjain India">
<meta name="citation_author" content="Bankala Krishnarjuna">
<meta name="citation_author_institution" content="Biophysics Program, The University of Michigan, Ann Arbor, Michigan, USA">
<meta name="citation_author_institution" content="Department of Chemistry, The University of Michigan, Ann Arbor, Michigan, USA">
<meta name="citation_author_institution" content="Biomedical Engineering, Michigan Institute for Neuroscience, Macromolecular Science and Engineering, The University of Michigan, Ann Arbor, Michigan, USA">
<meta name="citation_author_institution" content="Present address: Department of Biochemistry All India Institute of Medical Sciences Guwahati India">
<meta name="citation_author" content="Volodymyr M Hiiuk">
<meta name="citation_author_institution" content="Biophysics Program, The University of Michigan, Ann Arbor, Michigan, USA">
<meta name="citation_author_institution" content="Department of Chemistry, The University of Michigan, Ann Arbor, Michigan, USA">
<meta name="citation_author" content="Magdalena I Ivanova">
<meta name="citation_author_institution" content="Biophysics Program, The University of Michigan, Ann Arbor, Michigan, USA">
<meta name="citation_author_institution" content="Department of Neurology, The University of Michigan, Ann Arbor, Michigan, USA">
<meta name="citation_author" content="Pavel Nagorny">
<meta name="citation_author_institution" content="Department of Chemistry, The University of Michigan, Ann Arbor, Michigan, USA">
<meta name="citation_author" content="Ayyalusamy Ramamoorthy">
<meta name="citation_author_institution" content="Biophysics Program, The University of Michigan, Ann Arbor, Michigan, USA">
<meta name="citation_author_institution" content="Department of Chemistry, The University of Michigan, Ann Arbor, Michigan, USA">
<meta name="citation_author_institution" content="Biomedical Engineering, Michigan Institute for Neuroscience, Macromolecular Science and Engineering, The University of Michigan, Ann Arbor, Michigan, USA">
<meta name="citation_author_institution" content="Department of Chemical and Biomedical Engineering, FAMU‐FSU College of Engineering, Florida State University, Tallahassee, Florida, USA">
<meta name="citation_author_institution" content="National High Magnetic Field Laboratory, Florida State University, Tallahassee, Florida, USA">
<meta name="citation_author_institution" content="Institute of Molecular Biophysics, Florida State University, Tallahassee, Florida, USA">
<meta name="citation_publication_date" content="2025 Aug 19">
<meta name="citation_volume" content="34">
<meta name="citation_issue" content="9">
<meta name="citation_firstpage" content="e70276">
<meta name="citation_doi" content="10.1002/pro.70276">
<meta name="citation_pmid" content="40828514">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12363408/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12363408/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12363408/pdf/PRO-34-e70276.pdf">
<meta name="description" content="Alzheimer&#x27;s disease (AD), a progressive neurodegenerative disorder, is characterized by cognitive decline resulting from neuronal cell death. A key contributor to AD pathology is C99, a membrane‐bound β‐secretase‐cleaved fragment of amyloid ...">
<meta name="og:title" content="Detergent‐free isolation and characterization of amyloid precursor protein C99 in E. coli native lipid‐nanodiscs using non‐ionic polymer">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Alzheimer&#x27;s disease (AD), a progressive neurodegenerative disorder, is characterized by cognitive decline resulting from neuronal cell death. A key contributor to AD pathology is C99, a membrane‐bound β‐secretase‐cleaved fragment of amyloid ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12363408/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12363408">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1002/pro.70276"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/PRO-34-e70276.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12363408%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12363408/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12363408/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12363408/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-prosci.gif" alt="Protein Science : A Publication of the Protein Society logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Protein Science : A Publication of the Protein Society" title="Link to Protein Science : A Publication of the Protein Society" shape="default" href="http://www.proteinscience.org/" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Protein Sci</button></div>. 2025 Aug 19;34(9):e70276. doi: <a href="https://doi.org/10.1002/pro.70276" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1002/pro.70276</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Protein%20Sci%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Protein%20Sci%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Protein%20Sci%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Protein%20Sci%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Detergent‐free isolation and characterization of amyloid precursor protein C99 in <em>E. coli</em> native lipid‐nanodiscs using non‐ionic polymer</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sharma%20G%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Gaurav Sharma</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Gaurav Sharma</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Biophysics Program, The University of Michigan, Ann Arbor, Michigan, USA</div>
<div class="p">
<sup>
<sup>2</sup>
</sup>Department of Chemistry, The University of Michigan, Ann Arbor, Michigan, USA</div>
<div class="p">
<sup>
<sup>3</sup>
</sup>Biomedical Engineering, Michigan Institute for Neuroscience, Macromolecular Science and Engineering, The University of Michigan, Ann Arbor, Michigan, USA</div>
<div class="p">
<sup>
<sup>8</sup>
</sup>Present address:
T.N. SYS Meryl Pharma
Ujjain
India
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sharma%20G%22%5BAuthor%5D" class="usa-link"><span class="name western">Gaurav Sharma</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>3,</sup><sup>8</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Krishnarjuna%20B%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Bankala Krishnarjuna</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Bankala Krishnarjuna</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Biophysics Program, The University of Michigan, Ann Arbor, Michigan, USA</div>
<div class="p">
<sup>
<sup>2</sup>
</sup>Department of Chemistry, The University of Michigan, Ann Arbor, Michigan, USA</div>
<div class="p">
<sup>
<sup>3</sup>
</sup>Biomedical Engineering, Michigan Institute for Neuroscience, Macromolecular Science and Engineering, The University of Michigan, Ann Arbor, Michigan, USA</div>
<div class="p">
<sup>
<sup>9</sup>
</sup>Present address:
Department of Biochemistry
All India Institute of Medical Sciences
Guwahati
India
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Krishnarjuna%20B%22%5BAuthor%5D" class="usa-link"><span class="name western">Bankala Krishnarjuna</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>3,</sup><sup>9</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hiiuk%20VM%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Volodymyr M Hiiuk</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Volodymyr M Hiiuk</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Biophysics Program, The University of Michigan, Ann Arbor, Michigan, USA</div>
<div class="p">
<sup>
<sup>2</sup>
</sup>Department of Chemistry, The University of Michigan, Ann Arbor, Michigan, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hiiuk%20VM%22%5BAuthor%5D" class="usa-link"><span class="name western">Volodymyr M Hiiuk</span></a>
</div>
</div>
<sup>1,</sup><sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ivanova%20MI%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Magdalena I Ivanova</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Magdalena I Ivanova</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Biophysics Program, The University of Michigan, Ann Arbor, Michigan, USA</div>
<div class="p">
<sup>
<sup>4</sup>
</sup>Department of Neurology, The University of Michigan, Ann Arbor, Michigan, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ivanova%20MI%22%5BAuthor%5D" class="usa-link"><span class="name western">Magdalena I Ivanova</span></a>
</div>
</div>
<sup>1,</sup><sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nagorny%20P%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Pavel Nagorny</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Pavel Nagorny</span></h3>
<div class="p">
<sup>
<sup>2</sup>
</sup>Department of Chemistry, The University of Michigan, Ann Arbor, Michigan, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nagorny%20P%22%5BAuthor%5D" class="usa-link"><span class="name western">Pavel Nagorny</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ramamoorthy%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Ayyalusamy Ramamoorthy</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Ayyalusamy Ramamoorthy</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Biophysics Program, The University of Michigan, Ann Arbor, Michigan, USA</div>
<div class="p">
<sup>
<sup>2</sup>
</sup>Department of Chemistry, The University of Michigan, Ann Arbor, Michigan, USA</div>
<div class="p">
<sup>
<sup>3</sup>
</sup>Biomedical Engineering, Michigan Institute for Neuroscience, Macromolecular Science and Engineering, The University of Michigan, Ann Arbor, Michigan, USA</div>
<div class="p">
<sup>
<sup>5</sup>
</sup>Department of Chemical and Biomedical Engineering, FAMU‐FSU College of Engineering, Florida State University, Tallahassee, Florida, USA</div>
<div class="p">
<sup>
<sup>6</sup>
</sup>National High Magnetic Field Laboratory, Florida State University, Tallahassee, Florida, USA</div>
<div class="p">
<sup>
<sup>7</sup>
</sup>Institute of Molecular Biophysics, Florida State University, Tallahassee, Florida, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ramamoorthy%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Ayyalusamy Ramamoorthy</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>3,</sup><sup>5,</sup><sup>6,</sup><sup>7,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="pro70276-aff-0001">
<sup>
<sup>1</sup>
</sup>Biophysics Program, The University of Michigan, Ann Arbor, Michigan, USA</div>
<div id="pro70276-aff-0002">
<sup>
<sup>2</sup>
</sup>Department of Chemistry, The University of Michigan, Ann Arbor, Michigan, USA</div>
<div id="pro70276-aff-0003">
<sup>
<sup>3</sup>
</sup>Biomedical Engineering, Michigan Institute for Neuroscience, Macromolecular Science and Engineering, The University of Michigan, Ann Arbor, Michigan, USA</div>
<div id="pro70276-aff-0004">
<sup>
<sup>4</sup>
</sup>Department of Neurology, The University of Michigan, Ann Arbor, Michigan, USA</div>
<div id="pro70276-aff-0005">
<sup>
<sup>5</sup>
</sup>Department of Chemical and Biomedical Engineering, FAMU‐FSU College of Engineering, Florida State University, Tallahassee, Florida, USA</div>
<div id="pro70276-aff-0006">
<sup>
<sup>6</sup>
</sup>National High Magnetic Field Laboratory, Florida State University, Tallahassee, Florida, USA</div>
<div id="pro70276-aff-0007">
<sup>
<sup>7</sup>
</sup>Institute of Molecular Biophysics, Florida State University, Tallahassee, Florida, USA</div>
<div id="pro70276-curr-0001">
<sup>
<sup>8</sup>
</sup>Present address:
T.N. SYS Meryl Pharma
Ujjain
India
</div>
<div id="pro70276-curr-0002">
<sup>
<sup>9</sup>
</sup>Present address:
Department of Biochemistry
All India Institute of Medical Sciences
Guwahati
India
</div>
<div class="author-notes p">
<div class="fn" id="correspondenceTo">
<sup>*</sup><p class="display-inline">

<strong>Correspondence</strong>
, 
Ayyalusamy Ramamoorthy, Department of Chemical and Biomedical Engineering, FAMU‐FSU College of Engineering, Florida State University, Tallahassee, FL 32310, USA. 
Email: <span>aramamoorthy@fsu.edu</span>

</p>
</div>
<div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Revised 2025 Aug 1; Received 2025 Jun 13; Accepted 2025 Aug 7; Collection date 2025 Sep.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© 2025 The Author(s). <em>Protein Science</em> published by Wiley Periodicals LLC on behalf of The Protein Society.</div>
<p>This is an open access article under the terms of the <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by/4.0/</a> License, which permits use, distribution and reproduction in any medium, provided the original work is properly cited.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12363408  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40828514/" class="usa-link">40828514</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>Abstract</h2>
<p>Alzheimer's disease (AD), a progressive neurodegenerative disorder, is characterized by cognitive decline resulting from neuronal cell death. A key contributor to AD pathology is C99, a membrane‐bound β‐secretase‐cleaved fragment of amyloid precursor protein (APP). C99 plays a central role in generating amyloid‐beta (Aβ) isomers, which are directly implicated in disease progression. Understanding its structure and lipid interactions is essential for elucidating its mechanistic role in AD and guiding therapeutic development. C99 has been studied in membrane mimetics such as micelles, bicelles, and reconstituted nanodiscs. Although reconstituted nanodiscs provide a native‐like lipid‐bilayer environment, the use of detergents prior to reconstitution has been reported to disrupt native folding and lipid‐protein interactions. In this study, we successfully isolated and purified C99 along with its associated lipids directly from <em>E. coli</em> cell membranes using a non‐ionic pentyl‐inulin polymer, avoiding the need for detergents. The purified C99‐containing pentyl‐inulin nanodiscs were characterized using SDS‐PAGE, Western blotting, dynamic light scattering (DLS), <sup>1</sup>H NMR spectroscopy, matrix‐assisted laser desorption/ionization time‐of‐flight (MALDI‐TOF) mass spectrometry, and liquid chromatography–mass spectrometry (LC–MS). Notably, we observed SDS‐stable oligomers of C99. DLS and <sup>1</sup>H NMR confirmed the presence of large particles composed of pentyl‐inulin and <em>E. coli</em> lipids. MALDI‐TOF and LC–MS verified the molecular mass and amino acid sequence of C99, respectively. We propose that this detergent‐free method for the direct isolation of C99 and native lipids using non‐ionic pentyl‐inulin may serve as a valuable tool for investigating the C99‐secretase complex and for developing compounds aimed at inhibiting the production of amyloid‐beta isomers.</p>
<section id="kwd-group1" class="kwd-group"><p><strong>Keywords:</strong> Alzheimer's disease, amyloid precursor protein (APP‐C99), C99 oligomer, detergent‐free purification, nanodisc, non‐ionic polymer</p></section></section><section id="pro70276-sec-0001"><h2 class="pmc_sec_title">1. INTRODUCTION</h2>
<p>Membrane proteins constitute approximately 60% of the drug targets and 30% of the mammalian proteome (Arinaminpathy et al., <a href="#pro70276-bib-0003" class="usa-link" aria-describedby="pro70276-bib-0003">2009</a>; Errey &amp; Fiez‐Vandal, <a href="#pro70276-bib-0019" class="usa-link" aria-describedby="pro70276-bib-0019"><sup>2020</sup></a>; Levental &amp; Lyman, <a href="#pro70276-bib-0037" class="usa-link" aria-describedby="pro70276-bib-0037"><sup>2023</sup></a>; Overington et al., <a href="#pro70276-bib-0046" class="usa-link" aria-describedby="pro70276-bib-0046"><sup>2006</sup></a>). Being gate‐keepers, membrane proteins are essential to carry out important cellular functions, including transmitting molecular information and transporting essential molecules across the cell membrane (Ma et al., <a href="#pro70276-bib-0039" class="usa-link" aria-describedby="pro70276-bib-0039">2024</a>; Wu et al., <a href="#pro70276-bib-0068" class="usa-link" aria-describedby="pro70276-bib-0068"><sup>2023</sup></a>). Many devastating diseases are also caused by the functional impairment of membrane proteins (Wisedchaisri et al., <a href="#pro70276-bib-0067" class="usa-link" aria-describedby="pro70276-bib-0067">2023</a>; Yu et al., <a href="#pro70276-bib-0072" class="usa-link" aria-describedby="pro70276-bib-0072"><sup>2023</sup></a>). Therefore, understanding their structure–function relationship holds much potential for drug design and development (Batool et al., <a href="#pro70276-bib-0005" class="usa-link" aria-describedby="pro70276-bib-0005">2019</a>; Bou‐Assaf &amp; Marshall, <a href="#pro70276-bib-0011" class="usa-link" aria-describedby="pro70276-bib-0011"><sup>2020</sup></a>; Yang et al., <a href="#pro70276-bib-0069" class="usa-link" aria-describedby="pro70276-bib-0069"><sup>2021</sup></a>).</p>
<p>While the importance of understanding the dynamic structure and function of membrane proteins is well recognized, significant challenges remain in achieving successful structural studies despite recent advances in cryo‐electron microscopy (cryoEM) and solid‐state NMR spectroscopy (Zhou et al., <a href="#pro70276-bib-0075" class="usa-link" aria-describedby="pro70276-bib-0075">2025</a>). One of the primary obstacles to obtaining atomic‐resolution structural information lies in the reconstitution of membrane proteins into membrane mimetics that preserve their native fold and functional integrity, which is essential for biophysical characterization (Carpenter et al., <a href="#pro70276-bib-0012" class="usa-link" aria-describedby="pro70276-bib-0012">2008</a>; Miles &amp; Wallace, <a href="#pro70276-bib-0042" class="usa-link" aria-describedby="pro70276-bib-0042"><sup>2016</sup></a>; Opella, <a href="#pro70276-bib-0045" class="usa-link" aria-describedby="pro70276-bib-0045"><sup>2013</sup></a>; Tiefenauer &amp; Demarche, <a href="#pro70276-bib-0061" class="usa-link" aria-describedby="pro70276-bib-0061"><sup>2012</sup></a>).</p>
<p>Traditionally, membrane proteins have been purified utilizing various detergents and then reconstituted into membrane mimetics for in vitro characterization (Bayburt et al., <a href="#pro70276-bib-0006" class="usa-link" aria-describedby="pro70276-bib-0006">2002</a>; Bayburt &amp; Sligar, <a href="#pro70276-bib-0007" class="usa-link" aria-describedby="pro70276-bib-0007"><sup>2010</sup></a>; Krishnarjuna &amp; Ramamoorthy, <a href="#pro70276-bib-0029" class="usa-link" aria-describedby="pro70276-bib-0029"><sup>2022</sup></a>; Ritchie et al., <a href="#pro70276-bib-0054" class="usa-link" aria-describedby="pro70276-bib-0054"><sup>2009</sup></a>; Shen et al., <a href="#pro70276-bib-0055" class="usa-link" aria-describedby="pro70276-bib-0055"><sup>2013</sup></a>). Studies have reported that using detergents in the purification protocols disrupts the native lipid‐protein interactions as all the cell membrane lipids are washed out (Bao et al., <a href="#pro70276-bib-0004" class="usa-link" aria-describedby="pro70276-bib-0004">2013</a>; Julien et al., <a href="#pro70276-bib-0025" class="usa-link" aria-describedby="pro70276-bib-0025"><sup>2021</sup></a>; Krishnarjuna &amp; Ramamoorthy, <a href="#pro70276-bib-0029" class="usa-link" aria-describedby="pro70276-bib-0029"><sup>2022</sup></a>; Rawlings, <a href="#pro70276-bib-0053" class="usa-link" aria-describedby="pro70276-bib-0053"><sup>2016</sup></a>; Urner et al., <a href="#pro70276-bib-0063" class="usa-link" aria-describedby="pro70276-bib-0063"><sup>2022</sup></a>; Yang et al., <a href="#pro70276-bib-0070" class="usa-link" aria-describedby="pro70276-bib-0070"><sup>2014</sup></a>; Yeung &amp; Stanley, <a href="#pro70276-bib-0071" class="usa-link" aria-describedby="pro70276-bib-0071"><sup>2010</sup></a>). The subsequent reconstitution into micelles, vesicles, or bicelles may not represent a native‐like environment for a membrane protein. This is because these membrane‐mimicking systems are less stable due to uncontrolled fusion (Stępień et al., <a href="#pro70276-bib-0058" class="usa-link" aria-describedby="pro70276-bib-0058">2023</a>) or due to the presence of detergents and curvature (Krishnarjuna &amp; Ramamoorthy, <a href="#pro70276-bib-0029" class="usa-link" aria-describedby="pro70276-bib-0029">2022</a>).</p>
<p>Recent studies have demonstrated the benefits of using nanodiscs that contain flat lipid‐bilayered discs surrounded by protective amphipathic belts (Bayburt et al., <a href="#pro70276-bib-0006" class="usa-link" aria-describedby="pro70276-bib-0006">2002</a>; Denisov &amp; Sligar, <a href="#pro70276-bib-0015" class="usa-link" aria-describedby="pro70276-bib-0015"><sup>2016</sup></a>; Sligar &amp; Denisov, <a href="#pro70276-bib-0057" class="usa-link" aria-describedby="pro70276-bib-0057"><sup>2021</sup></a>). Different types of nanodisc‐forming belts have been reported, which include membrane scaffold protein (MSP), short peptides, and synthetic polymers. Although these nanodiscs containing planar lipid bilayers are better suited, the commonly used approach exposes a membrane protein to detergents during purification, which is not desirable for stable and functional reconstitution of many types of membrane proteins. Contrarily, recently demonstrated detergent‐free membrane protein isolation methods, where the cell membranes are solubilized using lipid‐solubilizing synthetic polymers or peptides/proteins, do not remove native cell membrane lipids; thus, the native lipid‐protein interactions are retained to render high stability and function (Drulyte et al., <a href="#pro70276-bib-0017" class="usa-link" aria-describedby="pro70276-bib-0017">2023</a>; Krishnarjuna, Im, et al., <a href="#pro70276-bib-0028" class="usa-link" aria-describedby="pro70276-bib-0028"><sup>2022</sup></a>; Krishnarjuna &amp; Ramamoorthy, <a href="#pro70276-bib-0029" class="usa-link" aria-describedby="pro70276-bib-0029"><sup>2022</sup></a>; Krishnarjuna, Ravula, &amp; Ramamoorthy, <a href="#pro70276-bib-0030" class="usa-link" aria-describedby="pro70276-bib-0030"><sup>2022</sup></a>; Krishnarjuna, Sharma, Hiiuk, et al., <a href="#pro70276-bib-0031" class="usa-link" aria-describedby="pro70276-bib-0031"><sup>2024</sup></a>; Krishnarjuna, Sharma, Im, et al., <a href="#pro70276-bib-0032" class="usa-link" aria-describedby="pro70276-bib-0032"><sup>2024</sup></a>).</p>
<p>Although several studies have reported the use of detergent‐free membrane protein isolation methods for functional and structural studies in native/near‐native membrane environments, it is still not straightforward as each membrane protein is unique and poses many difficulties, especially those with low expression levels and with self‐assembling properties (Krishnarjuna &amp; Ramamoorthy, <a href="#pro70276-bib-0029" class="usa-link" aria-describedby="pro70276-bib-0029">2022</a>). One such protein is the amyloid precursor protein‐C99 (APP‐C99 or C99), which is associated with AD (Beel et al., <a href="#pro70276-bib-0009" class="usa-link" aria-describedby="pro70276-bib-0009">2010</a>; Koch et al., <a href="#pro70276-bib-0026" class="usa-link" aria-describedby="pro70276-bib-0026"><sup>2023</sup></a>; Lane et al., <a href="#pro70276-bib-0034" class="usa-link" aria-describedby="pro70276-bib-0034"><sup>2018</sup></a>; Shen &amp; Kelleher, <a href="#pro70276-bib-0056" class="usa-link" aria-describedby="pro70276-bib-0056"><sup>2007</sup></a>). APP undergoes two sequential proteolytic cleavages by β‐ and γ‐secretases before forming amyloids‐β (Aβ) isomers (Abraham et al., <a href="#pro70276-bib-0001" class="usa-link" aria-describedby="pro70276-bib-0001">2023</a>; Chen et al., <a href="#pro70276-bib-0013" class="usa-link" aria-describedby="pro70276-bib-0013"><sup>2017</sup></a>; Davra &amp; Benzeroual, <a href="#pro70276-bib-0014" class="usa-link" aria-describedby="pro70276-bib-0014"><sup>2023</sup></a>; Koch et al., <a href="#pro70276-bib-0026" class="usa-link" aria-describedby="pro70276-bib-0026"><sup>2023</sup></a>). Firstly, APP is cleaved by β‐secretase to form C99, a 99‐residue C‐terminal protein (known as β‐CTF), which undergoes a second cleavage by γ‐secretase to form Aβ isomers. The accumulation of Aβ peptide aggregates has been linked to AD, a condition that is becoming increasingly prevalent with the aging population (Alzheimer's A, <a href="#pro70276-bib-0002" class="usa-link" aria-describedby="pro70276-bib-0002">2024</a>; Lane et al., <a href="#pro70276-bib-0034" class="usa-link" aria-describedby="pro70276-bib-0034"><sup>2018</sup></a>; Zilioli et al., <a href="#pro70276-bib-0076" class="usa-link" aria-describedby="pro70276-bib-0076"><sup>2024</sup></a>). Moreover, some studies have directly implicated C99 in AD pathogenesis (Jordà‐Siquier et al., <a href="#pro70276-bib-0024" class="usa-link" aria-describedby="pro70276-bib-0024">2022</a>; Lauritzen et al., <a href="#pro70276-bib-0035" class="usa-link" aria-describedby="pro70276-bib-0035"><sup>2019</sup></a>; Tompa, <a href="#pro70276-bib-0062" class="usa-link" aria-describedby="pro70276-bib-0062"><sup>2002</sup></a>), making it an important membrane protein for mechanistic and drug‐development studies (Takasugi et al., <a href="#pro70276-bib-0060" class="usa-link" aria-describedby="pro70276-bib-0060">2023</a>) Previous studies have characterized C99 by reconstituting it in membrane‐mimicking environments that utilized detergent‐based methods to extract C99; hence, the native lipid‐protein interactions are likely to be lost (Krishnarjuna, Sharma, Hiiuk, et al., <a href="#pro70276-bib-0031" class="usa-link" aria-describedby="pro70276-bib-0031">2024</a>; Motoki et al., <a href="#pro70276-bib-0044" class="usa-link" aria-describedby="pro70276-bib-0044"><sup>2012</sup></a>; Vetrivel et al., <a href="#pro70276-bib-0064" class="usa-link" aria-describedby="pro70276-bib-0064"><sup>2005</sup></a>). Therefore, it is important to study C99 with its associated native lipids to understand the role of native lipids on the structure and function of C99. A recent cryo‐EM study reported that γ‐secretase processes C99 through stepwise helix unwinding in a cholesterol‐ and phospholipid‐rich environment (Guo et al., <a href="#pro70276-bib-0022" class="usa-link" aria-describedby="pro70276-bib-0022">2024</a>). While not directly focused on lipid interactions, the study emphasized the importance of membrane composition in C99 processing. Replica exchange molecular dynamics (REMD) simulations of the full‐length C99 in membranes of varying thickness revealed that the structural states of its N‐ and C‐terminal domains are strongly influenced by the surrounding lipid environment (Pantelopulos et al., <a href="#pro70276-bib-0049" class="usa-link" aria-describedby="pro70276-bib-0049">2018</a>). Specifically, thinner membranes promote β‐strand formation at the N‐terminus, which may facilitate Aβ aggregation. In contrast, thicker membranes favor α‐helical conformations that stabilize interactions with γ‐secretase and cytoplasmic binding partners. These observations underscore the importance of preserving native lipid‐protein interactions when studying C99 structure and function.</p>
<p>Recent studies suggest that cholesterol's influence on Aβ production and AD pathogenesis is largely mediated by its ability to shape membrane domains and modulate protein trafficking (Pantelopulos et al., <a href="#pro70276-bib-0047" class="usa-link" aria-describedby="pro70276-bib-0047">2024</a>). Simulations and biophysical studies further demonstrated that membrane properties, including bilayer thickness and lipid composition, modulate C99 dimer topology and conformational dynamics (Dominguez et al., <a href="#pro70276-bib-0016" class="usa-link" aria-describedby="pro70276-bib-0016">2016</a>; Li et al., <a href="#pro70276-bib-0038" class="usa-link" aria-describedby="pro70276-bib-0038"><sup>2022</sup></a>; Viswanath et al., <a href="#pro70276-bib-0065" class="usa-link" aria-describedby="pro70276-bib-0065"><sup>2015</sup></a>). Notably, C99 itself can remodel its surrounding membrane environment (Pantelopulos et al., <a href="#pro70276-bib-0048" class="usa-link" aria-describedby="pro70276-bib-0048">2022</a>), and recent evidence indicates that neuronal cholesterol levels regulate C99 oligomerization and its amyloidogenic potential (Mesa et al., <a href="#pro70276-bib-0041" class="usa-link" aria-describedby="pro70276-bib-0041">2024</a>). These findings position C99 as both a sensor and effector of membrane lipid composition. Therefore, a method that preserves native lipid–protein interactions can serve as a powerful platform to study how cholesterol influences C99 monomer‐dimer equilibrium and its structure–function relationships in AD.</p>
<p>Detergents such as DDM and CHAPS have been shown to destabilize transmembrane domains, alter protein oligomerization (Bao et al., <a href="#pro70276-bib-0004" class="usa-link" aria-describedby="pro70276-bib-0004">2013</a>; Julien et al., <a href="#pro70276-bib-0025" class="usa-link" aria-describedby="pro70276-bib-0025"><sup>2021</sup></a>), and disrupt native lipids that are essential for maintaining the conformational integrity of membrane proteins like C99 and their interaction with processing enzymes (Krishnarjuna, Sharma, Hiiuk, et al., <a href="#pro70276-bib-0031" class="usa-link" aria-describedby="pro70276-bib-0031">2024</a>; Kumar et al., <a href="#pro70276-bib-0033" class="usa-link" aria-describedby="pro70276-bib-0033"><sup>2024</sup></a>). These disruptions can compromise the structural and functional fidelity of C99, potentially leading to incomplete or misleading mechanistic insights. This underscores the need for alternative purification strategies that preserve the native lipid environment.</p>
<p>This study introduces a new method for isolating C99 from bacterial cell membranes using a charge‐free polymer. This approach is also adaptable to brain tissue samples from AD patients, enabling the isolation of C99‐native lipid complexes. As such, the method has significant potential for elucidating the role of native lipids in regulating the structure and function of C99. To implement this strategy, C99‐rich cell membranes were solubilized using the pentyl‐inulin polymer, and C99‐enriched <em>E. coli</em> lipid nanodiscs were subsequently purified and characterized using biophysical techniques.</p></section><section id="pro70276-sec-0002"><h2 class="pmc_sec_title">2. MATERIALS AND METHODS</h2>
<section id="pro70276-sec-0003"><h3 class="pmc_sec_title">2.1. Expression of recombinant C99 using IPTG and autoinduction</h3>
<p>The plasmid with the C99 gene was kindly provided by Professor Charles R. Sanders (Vanderbilt University School of Medicine). The plasmid was chemically transformed into a C41 bacterial strain prior to colony selection on a carbenicillin agar plate. A single colony was inoculated in LB medium, and the cells were grown overnight at 200 rpm, 37°C. Pre‐culture was inoculated in ZYM‐5052 autoinduction medium (Bentley et al., <a href="#pro70276-bib-0010" class="usa-link" aria-describedby="pro70276-bib-0010">2020</a>; Krishnarjuna, Sharma, Hiiuk, et al., <a href="#pro70276-bib-0031" class="usa-link" aria-describedby="pro70276-bib-0031"><sup>2024</sup></a>; Studier, <a href="#pro70276-bib-0059" class="usa-link" aria-describedby="pro70276-bib-0059"><sup>2005</sup></a>), supplemented with 1 mM MgSO<sub>4</sub>, 1× trace metal mix (#T1001, Teknova), 0.1× BME vitamin (#B6891, Sigma‐Aldrich), and 100 μg/mL carbenicillin at the ratio of 0.3:100. The cells were grown at 37°C until OD<sub>600</sub> reached between 0.8 and 1.0. Then, the temperature was lowered to 16–18°C, and the culture was grown for an additional 16–20 h to overexpress the protein by autoinduction. The cells were harvested by centrifugation at 8000 rpm for 10 min, 4°C, and the pellets were stored at −80°C. In the case of IPTG induction, cells were induced with 1 mM IPTG and allowed to express at 16°C for 16 h and at 37°C for 5 h. A 0.8 OD<sub>600</sub> sample (equal concentration) was loaded onto the gel to verify expression under both IPTG‐induced and autoinduction conditions. Negative control samples (cells without protein expression) were cultured to confirm that the observed protein signals were specific to the expressed construct and not due to background expression or contamination.</p></section><section id="pro70276-sec-0004"><h3 class="pmc_sec_title">2.2. Cell lysis and preparation of C99‐enriched membranes</h3>
<p>Two protease inhibitor tablets (#11836170001, cOmplete™, Mini, EDTA‐free protease inhibitor cocktail, Roche) were dissolved in a 50 mL resuspension buffer (75 mM Tris—pH 7.8, 300 mM NaCl, 0.2 mM EDTA), and the cell pellets (12 g) were resuspended in the resuspension buffer. Cell lysis with Lysozyme (New Brunswick, NJ, USA) (5 mg/1 g cell paste) and DNase (Sigma‐Aldrich, St. Louis, Missouri, USA) (1 mg/mL – dissolved in 5 mM MgCl<sub>2</sub>) was done on ice for 10 min before sonication was carried out using a 13 mm sonicator probe (Thomas Scientific, LLC, NJ) at 40% amplitude. The physical lysis was performed by sonicating the cells for 9 cycles, with a 20 s pulse and 1 min rest on ice between each cycle. The soluble components were removed by centrifugation for 35 min at 18,500 rpm in a JA20 rotor (Beckman) and the insoluble C99‐rich cell membranes (pellet) were collected and washed once with the 50 mL resuspension buffer, followed by a final wash with resuspension buffer without EDTA. All the steps to prepare C99‐rich cell membranes were performed on ice, and protease inhibitors were used throughout the membrane preparation steps.</p></section><section id="pro70276-sec-0005"><h3 class="pmc_sec_title">2.3. Pentyl‐inulin polymer synthesis and analysis</h3>
<p>The pentyl‐inulin polymer (also called INPEN) with a degree of substitution (DS) value of 30% was synthesized following a previously published protocol (Ravula &amp; Ramamoorthy, <a href="#pro70276-bib-0052" class="usa-link" aria-describedby="pro70276-bib-0052">2021</a>). The degree of substitution, which represents the average number of pentyl groups attached per fructose unit, was estimated using the method outlined in the same protocol.</p>
<p>1.0 g of inulin from chicory root (5.55 mmol of fructose, 1.0 equiv.) purchased from Sigma‐Aldrich (#I2255) with an average degree of polymerization (DP) of n ≈ 14 was added to 30 mL of <em>N,N‐</em>Dimethylacetamide (DMAc;#803,235) and stirred at 50°C until inulin was dissolved completely. The resulting reaction mixture was then cooled to room temperature, and 444 mg of sodium hydride (11.10 mmol, 2.0 equiv., 60% NaH in mineral oil) was added in three portions over 1 h. The resulting mixture was reheated to 50°C and stirred for 45 min before cooling down to room temperature again. Subsequently, 1.03 mL of pentyl bromide (8.33 mmol, 1.5 equiv.) was added, and the reaction mixture was stirred at room temperature for ~2–2.5 days. The progress of the reaction was monitored by <sup>1</sup>H NMR spectroscopy, and the reaction was stopped once the desired degree of substitution (30%) was achieved. To quench the reaction, ice‐cold ethanol was added, followed by diethyl ether. The formed precipitate was separated by slow centrifugation (3000 rpm for 10 min) and then washed with diethyl ether and ethanol. The resulting crude pentyl‐inulin polymer was dissolved in water and subjected to dialysis using a Spectra Por dialysis tubing (MWCO: 500–1000 Da; #131096, Cole‐Parmer), followed by lyophilization to obtain a white powder. The polymer was characterized by <sup>1</sup>H NMR spectroscopy (Figure <a href="#pro70276-supitem-0001" class="usa-link">S1</a>). <sup>1</sup>H NMR spectrum of pentyl‐inulin in D<sub>2</sub>O was recorded on a Varian Vnmrs 700 (700 MHz) spectrometer.</p></section><section id="pro70276-sec-0006"><h3 class="pmc_sec_title">2.4. Membrane solubilization with pentyl‐inulin polymer</h3>
<p>We first verified the presence of the C99 in the membranes. Briefly, the membranes that were prepared at a concentration of 25 mg/mL (by dissolving 25 mg of membranes in 1 mL of buffer) in solubilization buffer (10 mM Tris pH 7.4, 100 mM NaCl, and protease inhibitor) were added to the pentyl inulin polymer (prepared in solubilization buffer) at a 1:1 (w/w) membrane to polymer ratio. The membranes‐polymer mixture, prepared in a total volume of 1 mL, was then incubated on a rotor at 4°C overnight for solubilization. On the next day, the mixture was spun at 10,000 rpm for 45 min at 4°C, and the supernatant, representing the solubilized nanodiscs sample, was gently pipetted out before confirming it using sodium dodecyl‐sulfate polyacrylamide gel electrophoresis (SDS‐PAGE). Membranes without polymer were used as a negative control. The efficiency of the polymer in forming empty nanodiscs was also verified (Figures <a href="#pro70276-supitem-0001" class="usa-link">S2–S4</a>).</p>
<p>For a large‐scale solubilization, 3.6 g of membranes were resuspended in 50 mL of solubilization buffer (10 mM Tris pH 7.4, 100 mM NaCl, and two protease inhibitor tablets). After resuspension, 1.8 g of pentyl‐inulin polymer (1:0.5 membrane‐to‐polymer ratio) was added to the membrane solution. The sample volume was then adjusted to 150 mL, and the mixture was incubated overnight at 4°C with gentle mixing.</p></section><section id="pro70276-sec-0007"><h3 class="pmc_sec_title">2.5. 
SDS‐PAGE and Western blot</h3>
<p>Protein detection in the solubilized fractions was accomplished using SDS‐PAGE on pre‐cast GenScript gradient gels (4–20%) (Piscataway, NJ). The samples were mixed in a 1:1 ratio with 2x dye (#1610737, Bio‐Rad) and heated for 4 min at 90°C before loading onto the gel. The gel electrophoresis was performed using a Bio‐Rad SDS‐PAGE gel casting system (Bio‐Rad Laboratories, Inc.; Hercules, CA, USA) at 100 V for 10 min at room temperature first and then at 150 V for 60 min at room temperature. The proteins were stained using a Coomassie stain (0.1% Coomassie Brilliant Blue, 30% ethanol, and 10% acetic acid) and destained using destaining buffer (30% methanol and 10% acetic acid) before capturing them on a Bio‐Rad ChemiDoc MP imaging system.</p>
<p>For the Western blot, the gel was run as explained above. During gel electrophoresis, a PVDF membrane (Immun‐Blot Bio‐Rad 1,620,177, 26 cm × 3.3 m, 0.2 μm) was soaked in 100% methanol for 30 min. The membrane was then soaked in the transfer buffer (0.25 M Tris, 1.92 M glycine, and 20% methanol) for 10 min. The proteins from the SDS‐PAGE gel were transferred onto the PVDF membrane at 4°C, 110 V for 1.5 h. The PVDF membrane was blocked for 1 h in blocking buffer (TBS‐0.02 M Tris, 0.15 M NaCl, 0.1% Tween 20, 10% milk) at room temperature. After blocking, the membrane was incubated with the primary antibody (6E10; Cat# 803004, Bio Legend, USA, prepared at a 1:500 dilution in TBST with 5% milk) overnight at 4°C with gentle shaking. The blot was then washed 3 times with 5% milk in TBST before incubating with the secondary antibody (goat anti‐mouse horseradish peroxidase; Thermo Fisher Scientific; Cat#32230, prepared at a dilution of 1:5000 in the same buffer as primary antibody). The membrane was washed 3 times as previously, and the protein bands were visualized by the GENESys imaging system in the presence of substrate Eco bright Femto HRP (# EBFH100, Innovative Solutions, USA).</p></section><section id="pro70276-sec-0008"><h3 class="pmc_sec_title">2.6. Affinity purification of C99
</h3>
<p>All the purification steps were performed at 4°C. The overnight solubilized membranes (150 mL) were spun down at 18,000 rpm, 4°C for 30–50 min. The supernatant was carefully collected and filtered using a 1.2‐micron cellulose acetate filter (Findlay, OH).</p>
<p>Five milliliter of nickel‐chelated resin was equilibrated with His binding buffer A (30 mM Tris, pH 8.0, 100 mM NaCl) prior to the addition of the solubilized protein. The mixture was incubated for 2 h at room temperature with gentle rotation. The bound C99 was loaded on a chromatography column (Bio‐Rad #7372512) to separate the protein from the filtrate. The beads were washed with 10 column volumes (CV) of binding buffer to remove unbound polypeptides. C99 was eluted using a stepwise gradient of imidazole ranging from 20 to 300 mM. The collected fractions were analyzed by SDS‐PAGE; those containing C99 were pooled for further use.</p></section><section id="pro70276-sec-0009"><h3 class="pmc_sec_title">2.7. Size‐exclusion chromatography (SEC)</h3>
<p>The affinity‐purified C99‐containing nanodiscs were concentrated to 4.5 mL using an Amicon 10 kDa concentrator. The size‐exclusion column (HiLoad 16/600 Superdex® 200 pg, Cytiva, #28‐9893‐35) was washed with 2 CV of milli‐Q and equilibrated with 2 CV of size‐ex buffer (20 mM potassium phosphate pH 7.4, 100 mM NaCl) before injecting the sample into the column. Both the milli‐Q and the buffer were stored on ice during chromatography. The mobile phase was run at a 1 mL/min flow rate on an ӒKTA Purifier FPLC system (GE Healthcare, Chicago, IL). The milli‐Q and the buffer were degassed (Ultrasonic cleaner, Cole‐Parmer, Chicago) by filtering with a 0.2‐μ filter (Millipore Steritop Vaccum filter, Merck) before purification.</p></section><section id="pro70276-sec-0010"><h3 class="pmc_sec_title">2.8. Matrix‐assisted laser desorption/ionization time‐of‐flight (MALDI‐TOF) mass spectrometry</h3>
<p>Matrix‐assisted laser desorption/ionization time‐of‐flight (MALDI‐TOF) mass spectrometry measurements were performed using a Bruker Autoflex mass spectrometer instrument under linear positive ion mode to determine the mass of the C99. On a 384‐well MTP stainless‐steel plate, a sample (0.75 μL) was spotted with an equal volume of the matrices (2,5‐dihydroxybenzoic acid (DHB) or sinapinic acid (SA)). After the matrix was added to the sample, the mixture was thoroughly mixed and allowed to air dry. Prior to data acquisition, the mass spectrometer was calibrated using mid‐mass range protein standards: insulin (5734.52 Da) and myoglobin (16952.31 Da). Most of the samples were desalted with either C4 or C18 resin ZipTip 10 μL tips (Merck Millipore, USA) before preparing spots for MALDI‐TOF. The data was analyzed using Bruker flex Analysis software.</p></section><section id="pro70276-sec-0011"><h3 class="pmc_sec_title">2.9. Liquid chromatography‐mass spectrometry (LC–MS)</h3>
<p>To further verify the protein integrity and to support the MALDI‐TOF data, liquid chromatography–mass spectrometry (LC–MS) was performed.</p>
<section id="pro70276-sec-0012"><h4 class="pmc_sec_title">2.9.1. 
Sample preparation
</h4>
<p>The band from the destained SDS‐PAGE gel was cut and treated for disulfide bond reduction and alkylation with 10 mM DTT and 10 mM IAA in 50 mM ammonium acetate and 4 M urea. The sample was further diluted to 2 M urea with 50 mM ammonium bicarbonate, and the in‐gel trypsin digestion was allowed at 37°C overnight. Tryptic peptides were extracted with 80% acetonitrile before drying them using Speedvac. The sample was reconstituted in 0.1% formic acid for desalting using C18 pipette tips (Pierce C18 Pipette Tips, Thermo Scientific).</p></section><section id="pro70276-sec-0013"><h4 class="pmc_sec_title">2.9.2. 
Data collection
</h4>
<p>LC–MS analysis was performed on an Ultimate 3600 nan‐UPLC (Thermo Scientific) coupled to an Orbitrap Fusion Lumos Tribrid mass spectrometer (Thermo Scientific). The reconstituted samples (~5 μL) were separated on a C18 column (Acclaim PepMap 100, C18, 75 μm × 50 cm, 2 μm, Thermo Scientific) over a 90 min gradient of H<sub>2</sub>O with 0.1% formic acid (solvent A) and 80% acetonitrile in H<sub>2</sub>O with 0.1% formic acid (solvent B) at a flow rate of 300 nL/min (0–5 min, 5% B; 5–70 min, 5%‐50% B; 70–80 min, 50%‐95 %B; 80–82 min, 95% B; 82–90 min, 5% B). The peptides were introduced into the mass spectrometer in positive ion mode on‐line via nano‐ESI. MS and tandem mass spectrometry (MS/MS) data were collected using a data‐dependent approach to perform high‐resolution peptide analysis. MS1 spectra were recorded within the m/z range of 375–1500 at a resolution of 120 K using the Orbitrap; the subsequent MS/MS spectra for peptide sequencing were generated by applying HCD (higher‐energy collisional dissociation in a multipole collision cell) at 32% normalized energy.</p></section><section id="pro70276-sec-0014"><h4 class="pmc_sec_title">2.9.3. 
Data analysis
</h4>
<p>MS and MS/MS spectra were searched for peptide and protein identification using Proteome Discoverer 2.2 (Thermo Scientific). Using the Sequest search algorithm, the search was conducted against protein sequences of <em>E. coli</em> (UniProt, reviewed, 2023.02). Mass tolerances of 5 ppm (MS1) and 0.02 Da (MS/MS) were applied. The database search was allowed for both static modifications, carbamidomethylation of cysteine (C) and dynamic modifications, including methionine oxidation (M), asparagine deamidation (N), and N‐terminal protein acetylation. Peptide and protein identifications were filtered at a 1% false discovery rate (FDR). To assess the reliability of the LC–MS/MS data, several key parameters were examined. The cross‐correlation score (XCorr), which reflects the quality of the match between experimental data and theoretical values; an XCorr value of 2 is generally considered indicative of a reliable identification. Additionally, a Sequest HT score above 100 is typically regarded as optimal for confident protein identification.</p></section></section><section id="pro70276-sec-0015"><h3 class="pmc_sec_title">2.10. Dynamic light scattering (DLS)</h3>
<p>The size distribution of C99 nanodiscs was measured on the Wyatt Technology DynaPro® NanoStar® DLS instrument. Five μicroliter of the sample purified using SEC was loaded in the quartz MicroCuvette (Wyatt Technology, CA), and the DLS measurements were recorded at 25°C. Ten acquisitions of 5 s each were taken and averaged to plot the size distribution profile.</p></section><section id="pro70276-sec-0016"><h3 class="pmc_sec_title">2.11. 
NMR spectroscopy</h3>
<p>The formation of nanodiscs and the interaction between C99 and nanodiscs were studied through 1D <sup>1</sup>H NMR. The SEC‐purified protein (in 10 mM Kpi, 50 mM NaCl pH 7.4) was loaded in a 5 mm NMR tube, and the spectra were recorded at 298 K on a Bruker 500 MHz NMR spectrometer (Billerica, MA, USA) using a TXI‐probe. The NMR data were processed and analyzed using Bruker Topspin software (version 3.6.2).</p></section></section><section id="pro70276-sec-0017"><h2 class="pmc_sec_title">3. RESULTS AND DISCUSSION</h2>
<section id="pro70276-sec-0018"><h3 class="pmc_sec_title">3.1. High C99 yields achieved using the autoinduction method</h3>
<p>C99 was expressed in <em>E. coli</em> using autoinduction and IPTG induction methods (Figure <a href="#pro70276-fig-0001" class="usa-link">1a</a>). The autoinduction was done at 16°C, whereas induction with IPTG was at 37°C. IPTG induction at a lower temperature (16°C) was also tested; however, it did not yield the optimal cell density required for the direct extraction method. Equal concentrations of total protein were loaded onto SDS‐PAGE gel to compare expression levels of C99 protein under different induction conditions. Autoinduction resulted in a markedly higher yield of C99 protein compared to IPTG induction, as evidenced by a more intense C99 protein band in the autoinduction sample (Figure <a href="#pro70276-fig-0001" class="usa-link">1a</a>). Densitometric analysis indicated approximately a 1.5‐fold increase in the intensity of the autoinduction band. This result was corroborated by Western blot analysis (Figure <a href="#pro70276-fig-0001" class="usa-link">1b</a>). The low protein yield in the IPTG‐induced samples is likely due to the predominant localization of the expressed protein in inclusion bodies rather than in the cell membrane (Figure <a href="#pro70276-supitem-0001" class="usa-link">S7</a>). In contrast, autoinduction enhanced the localization of the protein in the membrane fraction, making it a more effective method for overexpressing C99.</p>
<figure class="fig xbox font-sm" id="pro70276-fig-0001"><h4 class="obj_head">FIGURE 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fd/12363408/61419f39369b/PRO-34-e70276-g004.jpg" loading="lazy" id="jats-graphic-1" height="415" width="661" alt="FIGURE 1"></p>
<div class="p text-right font-secondary"><a href="figure/pro70276-fig-0001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>SDS‐PAGE/Western blot analysis of recombinant C99. a) C99 expression was analyzed by SDS‐PAGE. Lane 1: Protein marker; lane 2: Non‐induced negative control; lane 3: C99 overexpressed by the autoinduction method; and lane 4: C99 overexpressed by the IPTG‐induction method. b) Detection of C99 by Western blot; wells loaded with the same samples as in Figure <a href="#pro70276-fig-0001" class="usa-link">1a</a>. m, Monomer; d, Dimer; d’, Degraded dimer; t, Trimer; and o, Oligomer. The dimer and oligomer bands in lane 3 are enlarged for clarity. Samples were loaded at a normalized concentration of 0.8 OD<sub>600</sub> to assess protein expression levels. The protein band that appeared below the monomer is due to C99 degradation.</p></figcaption></figure><p>Dimers, trimers, and higher‐order oligomers of C99 were also detected by Western blot (Figure <a href="#pro70276-fig-0001" class="usa-link">1b</a>). These findings are consistent with previous studies reporting the presence of dimers, trimers, oligomers, and multimers of APP‐C99 (Dominguez et al., <a href="#pro70276-bib-0016" class="usa-link" aria-describedby="pro70276-bib-0016">2016</a>; Fawzi et al., <a href="#pro70276-bib-0020" class="usa-link" aria-describedby="pro70276-bib-0020"><sup>2011</sup></a>; Gerber et al., <a href="#pro70276-bib-0021" class="usa-link" aria-describedby="pro70276-bib-0021"><sup>2017</sup></a>; Krishnarjuna, Sharma, Hiiuk, et al., <a href="#pro70276-bib-0031" class="usa-link" aria-describedby="pro70276-bib-0031"><sup>2024</sup></a>; Perrin et al., <a href="#pro70276-bib-0050" class="usa-link" aria-describedby="pro70276-bib-0050"><sup>2020</sup></a>). A recent study conducted on five membrane proteins proposed that oligomerization in membrane proteins is often driven by weak, transient interactions (Zhang et al., <a href="#pro70276-bib-0074" class="usa-link" aria-describedby="pro70276-bib-0074">2025</a>; Zhang &amp; MacKinnon, <a href="#pro70276-bib-0073" class="usa-link" aria-describedby="pro70276-bib-0073"><sup>2025</sup></a>). The trimer band appeared at an approximate size of 42 kDa (Figure <a href="#pro70276-fig-0001" class="usa-link">1b</a>). The dimer band was expected to appear near 28 kDa; however, two bands were observed at a similar size. We believe that the band labeled with d corresponds to dimer and the band d' could be a degraded product of d.</p>
<p>Our observations under native‐like conditions are consistent with previous findings showing that membrane properties, including bilayer thickness, lipid composition, and cholesterol content, strongly influence C99 dimerization and conformational states (Beel et al., <a href="#pro70276-bib-0008" class="usa-link" aria-describedby="pro70276-bib-0008">2008</a>; Dominguez et al., <a href="#pro70276-bib-0016" class="usa-link" aria-describedby="pro70276-bib-0016"><sup>2016</sup></a>; Pantelopulos et al., <a href="#pro70276-bib-0047" class="usa-link" aria-describedby="pro70276-bib-0047"><sup>2024</sup></a>; Viswanath et al., <a href="#pro70276-bib-0065" class="usa-link" aria-describedby="pro70276-bib-0065"><sup>2015</sup></a>). Notably, Pantelopulos et al. (<a href="#pro70276-bib-0049" class="usa-link" aria-describedby="pro70276-bib-0049">2018</a>) used REMD simulations to show that membrane thickness modulates C99 oligomerization by increasing flexibility in extramembrane regions and exposing interaction‐prone sites such as the G37G38 hinge. These simulations revealed multiple dimerization motifs, with varying helix crossing angles giving rise to distinct transmembrane dimer conformations whose stability is lipid dependent (Pantelopulos et al., <a href="#pro70276-bib-0049" class="usa-link" aria-describedby="pro70276-bib-0049">2018</a>). More recent studies further showed that C99 can remodel its surrounding membrane, thereby altering its own oligomerization behavior and processing efficiency (Pantelopulos et al., <a href="#pro70276-bib-0048" class="usa-link" aria-describedby="pro70276-bib-0048">2022</a>). Mesa et al. (<a href="#pro70276-bib-0041" class="usa-link" aria-describedby="pro70276-bib-0041">2024</a>) also reported that neuronal cholesterol levels modulate C99 oligomerization and amyloidogenic activity, emphasizing the importance of membrane composition in AD progression (Mesa et al., <a href="#pro70276-bib-0041" class="usa-link" aria-describedby="pro70276-bib-0041">2024</a>).</p>
<p>Our detergent‐free extraction strategy preserves native lipid‐protein interactions and yields C99 in monomeric, dimeric, and higher‐order oligomeric forms. Unlike reconstitution‐based approaches, this method does not allow precise control over the lipid‐to‐protein ratio due to co‐extraction of endogenous lipids. However, downstream biophysical methods such as high‐resolution SEC and density gradient ultracentrifugation can be used to resolve and enrich specific oligomeric populations based on size and mass. Additionally, chemical cross‐linking under mild, native‐like conditions, followed by native PAGE or mass spectrometry, has been shown to stabilize transient C99 dimers (Hutchison et al., <a href="#pro70276-bib-0023" class="usa-link" aria-describedby="pro70276-bib-0023">2020</a>). Site‐directed mutagenesis targeting key dimerization motifs (e.g., GXXXG, GXXXA) provides a complementary approach for modulating the oligomeric equilibrium (Pantelopulos et al., <a href="#pro70276-bib-0048" class="usa-link" aria-describedby="pro70276-bib-0048">2022</a>). While the present study utilizes bacterial membranes, this strategy is extendable to cholesterol‐rich mammalian systems, including neurons and AD patient‐derived tissues.</p></section><section id="pro70276-sec-0019"><h3 class="pmc_sec_title">3.2. Western blotting confirmed the C99 expression and its anomalous migration on SDS‐PAGE
</h3>
<p>We verified the C99 expression by Western blot before proceeding with the direct extraction approach (Figure <a href="#pro70276-fig-0001" class="usa-link">1b</a>). The theoretical molecular weight of the C99 is 13.8 kDa; however, on SDS‐PAGE gel, the protein band was observed slightly above 15 kDa (Figure <a href="#pro70276-fig-0001" class="usa-link">1</a>) as we reported earlier (Krishnarjuna, Sharma, Hiiuk, et al., <a href="#pro70276-bib-0031" class="usa-link" aria-describedby="pro70276-bib-0031">2024</a>). The apparent increase in molecular weight of C99 observed on SDS‐PAGE is attributed to its anomalous migration, commonly referred to as “gel shifting”. Previous studies have documented the atypical migration behavior of transmembrane domains in varying concentrations of SDS (Emmanuel et al., <a href="#pro70276-bib-0018" class="usa-link" aria-describedby="pro70276-bib-0018">2025</a>; Krishnarjuna et al., <a href="#pro70276-bib-0027" class="usa-link" aria-describedby="pro70276-bib-0027"><sup>2016</sup></a>; Lee et al., <a href="#pro70276-bib-0036" class="usa-link" aria-describedby="pro70276-bib-0036"><sup>2013</sup></a>; MacRaild et al., <a href="#pro70276-bib-0040" class="usa-link" aria-describedby="pro70276-bib-0040"><sup>2015</sup></a>; Morales et al., <a href="#pro70276-bib-0043" class="usa-link" aria-describedby="pro70276-bib-0043"><sup>2015</sup></a>;Rath et al., <a href="#pro70276-bib-0051" class="usa-link" aria-describedby="pro70276-bib-0051"><sup>2009</sup></a>; Walkenhorst et al., <a href="#pro70276-bib-0066" class="usa-link" aria-describedby="pro70276-bib-0066"><sup>2009</sup></a>). Rath et al. and others have proposed that gel shifting arises from altered detergent binding and differential solvation by SDS, which disrupt protein–protein interactions and favor protein‐detergent complexes (Rath et al., <a href="#pro70276-bib-0051" class="usa-link" aria-describedby="pro70276-bib-0051">2009</a>; Walkenhorst et al., <a href="#pro70276-bib-0066" class="usa-link" aria-describedby="pro70276-bib-0066"><sup>2009</sup></a>). The authenticity of the C99 protein was confirmed by MALDI‐TOF and LC–MS/MS analyses, as detailed in Sections <a href="#pro70276-sec-0024" class="usa-link">3.7</a> and <a href="#pro70276-sec-0025" class="usa-link">3.8</a>.</p></section><section id="pro70276-sec-0020"><h3 class="pmc_sec_title">3.3. Small‐scale C99 pentyl‐inulin solubilization</h3>
<p>Before directly extracting C99 at a large scale, a small‐scale solubilization experiment was performed using pentyl‐inulin polymer; this would prove the presence of C99 in the <em>E. coli</em> cell membranes prepared from the autoinduction system. The efficiency of the pentyl‐inulin polymer was evaluated by SEC analysis of nanodiscs assembled using a 7:3 (w/w) mixture of DMPC and DMPG lipids derived from commercial <em>E. coli</em> sources. The resulting SEC profile is shown in Figure <a href="#pro70276-supitem-0001" class="usa-link">S3</a>. The pentyl‐inulin polymer was able to extract C99, as observed in lane 3 of Figure <a href="#pro70276-fig-0002" class="usa-link">2a</a>. The cell membranes without the addition of polymer (representing negative control, lane 2) showed no band, confirming that the C99 band in lane 3 was due to pentyl‐inulin‐driven solubilization. C99 was also observed in the insoluble membrane fractions (in the insoluble pellet, lanes 4 and 5) due to incomplete solubilization of C99‐rich cell membranes.</p>
<figure class="fig xbox font-sm" id="pro70276-fig-0002"><h4 class="obj_head">FIGURE 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fd/12363408/2c36c19f9d3b/PRO-34-e70276-g002.jpg" loading="lazy" id="jats-graphic-3" height="317" width="709" alt="FIGURE 2"></p>
<div class="p text-right font-secondary"><a href="figure/pro70276-fig-0002/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>(a) SDS‐PAGE analysis of pentyl‐inulin‐solubilized <em>E. coli</em> cell membranes enriched with C99. Lanes 1 and 6 are protein markers; lane 2 (negative control) represents the supernatant of the membranes that were not treated with the polymer for solubilization; lane 3 contains the supernatant of C99 membranes that were solubilized with a pentyl‐inulin polymer (labeled with red box); lanes 4 and 5 represent pellets (insoluble fractions) from samples without and with polymer solubilization, respectively. (b) SDS‐PAGE analysis of detergent‐free extraction of C99 using Ni<sup>2+</sup> affinity purification. Lanes 1 and 11 contain the protein molecular weight markers. Lanes 2–10 show eluted fractions at increasing imidazole concentrations of 20, 40, 50, 80, 120, 160, 200, 250, and 300 mM, respectively. Lane 12 corresponds to a second elution fraction using 300 mM imidazole.</p></figcaption></figure></section><section id="pro70276-sec-0021"><h3 class="pmc_sec_title">3.4. Purification of C99 in bacterial nanodiscs using Ni‐NTA affinity chromatography</h3>
<p>During an attempt to purify C99 using the Ni‐NTA chromatography column, we encountered difficulties due to the presence of large aggregates of crude cell membranes. Therefore, we opted to purify C99 using unpacked resin in a 50 mL centrifugation tube. The fractions eluted with an imidazole gradient were then analyzed using SDS‐PAGE (Figure <a href="#pro70276-fig-0002" class="usa-link">2b</a>). Most unwanted proteins were eluted at imidazole concentrations ranging from 50 to 80 mM. Subsequently, at concentrations up to 160 mM of imidazole the band intensity of C99 on the SDS‐PAGE gel was not substantial. However, we observed high‐intensity protein bands at imidazole concentrations above 200 mM, indicating efficient binding of C99 to the resin. The purification profile obtained from the detergent‐free isolation method in this study is comparable to that attained through the detergent‐based method (Krishnarjuna, Sharma, Hiiuk, et al., <a href="#pro70276-bib-0031" class="usa-link" aria-describedby="pro70276-bib-0031">2024</a>), suggesting excellent compatibility of pentyl‐inulin with metal‐chelated resins (Krishnarjuna &amp; Ramamoorthy, <a href="#pro70276-bib-0029" class="usa-link" aria-describedby="pro70276-bib-0029">2022</a>; Krishnarjuna, Ravula, &amp; Ramamoorthy, <a href="#pro70276-bib-0030" class="usa-link" aria-describedby="pro70276-bib-0030"><sup>2022</sup></a>). The highest purity was achieved in the 300 mM imidazole elution fraction (second elution with 300 mM imidazole, lane 12), which was used for all subsequent experiments.</p></section><section id="pro70276-sec-0022"><h3 class="pmc_sec_title">3.5. Purification of C99 in bacterial nanodiscs using SEC
</h3>
<p>SEC was performed to further purify the C99‐containing polymer nanodiscs (Figure <a href="#pro70276-fig-0003" class="usa-link">3a</a>). SEC showed a single, mostly symmetric peak representing pentyl‐inulin‐C99 nanodiscs. The position of the peak in the SEC chromatogram resembled the peak position that was obtained through the reconstitution method (Krishnarjuna, Sharma, Hiiuk, et al., <a href="#pro70276-bib-0031" class="usa-link" aria-describedby="pro70276-bib-0031">2024</a>), suggesting similar sizes of C99‐rich pentyl‐inulin nanodiscs. SDS‐PAGE analysis of the SEC fractions between volumes 42 mL to 62 mL indicated the protein bands corresponding to C99 (Figure <a href="#pro70276-fig-0003" class="usa-link">3b</a>), and Western blotting confirmed the presence of C99 (Figure <a href="#pro70276-fig-0003" class="usa-link">3c</a>). Western blot analysis revealed C99 bands corresponding to monomers (m), trimers (t), and higher‐order oligomers (o). Notably, a band above 20 kDa, likely representing a dimer (denoted as d’), was detected, although its precise position on the blot remains unclear. A band at a similar position was also observed in the overexpression samples (Figure <a href="#pro70276-fig-0001" class="usa-link">1b</a>) Interestingly, the expected dimer band (d) is absent in Figure <a href="#pro70276-fig-0003" class="usa-link">3c</a>, while only the putative degraded form (d’) is present. This may indicate that the dimer was degraded completely during or after purification, or alternatively, that it migrated differently due to distinct aggregation states. Intriguingly, oligomers are the predominant species in the detergent‐free isolated samples (Figures <a href="#pro70276-fig-0001" class="usa-link">1b</a> and <a href="#pro70276-fig-0003" class="usa-link">3c</a>), whereas higher‐order species (multimers) are more prominent in the detergent‐purified and nanodisc‐reconstituted C99 (Krishnarjuna, Sharma, Hiiuk, et al., <a href="#pro70276-bib-0031" class="usa-link" aria-describedby="pro70276-bib-0031">2024</a>). However, low levels of oligomers were also detected in those later preparations. These differences highlight the uniqueness of the detergent‐free membrane protein isolation method over detergent‐based methods. Although challenging, further investigations to isolate and characterize these oligomers and higher‐order structures could provide valuable insights into the mechanisms underlying C99 stability.</p>
<figure class="fig xbox font-sm" id="pro70276-fig-0003"><h4 class="obj_head">FIGURE 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fd/12363408/3d9b825addf9/PRO-34-e70276-g003.jpg" loading="lazy" id="jats-graphic-5" height="831" width="665" alt="FIGURE 3"></p>
<div class="p text-right font-secondary"><a href="figure/pro70276-fig-0003/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>(a) SEC elution profile of directly extracted C99 nanodiscs, performed on the nickel‐purified fractions. The peak represents C99‐rich pentyl‐inulin nanodiscs. (b) SDS‐PAGE analysis on the SEC fractions collected between volumes 42 to 62 mL. The fractions loaded on the gel are shown with blue arrows. c) Western blot analysis on the first four fractions (Arinaminpathy et al., <a href="#pro70276-bib-0003" class="usa-link" aria-describedby="pro70276-bib-0003">2009</a>; Errey &amp; Fiez‐Vandal, <a href="#pro70276-bib-0019" class="usa-link" aria-describedby="pro70276-bib-0019"><sup>2020</sup></a>; Levental &amp; Lyman, <a href="#pro70276-bib-0037" class="usa-link" aria-describedby="pro70276-bib-0037"><sup>2023</sup></a>; Overington et al., <a href="#pro70276-bib-0046" class="usa-link" aria-describedby="pro70276-bib-0046"><sup>2006</sup></a>) of SDS‐PAGE. m, monomers; d’, dimers (most likely a dimer); t, trimers; and o, oligomers.</p></figcaption></figure></section><section id="pro70276-sec-0023"><h3 class="pmc_sec_title">3.6. Characterization of C99‐containing nanodiscs by DLS and 
<sup>1</sup>H NMR
</h3>
<p>The C99‐rich nanodiscs were analyzed using DLS measurements. The major nanodisc species, with a hydrodynamic radius of 10 ± 1 nm, were observed (Figure <a href="#pro70276-fig-0004" class="usa-link">4a</a>). Again, the DLS results are in agreement with our previous study (Krishnarjuna, Sharma, Hiiuk, et al., <a href="#pro70276-bib-0031" class="usa-link" aria-describedby="pro70276-bib-0031">2024</a>).</p>
<figure class="fig xbox font-sm" id="pro70276-fig-0004"><h4 class="obj_head">FIGURE 4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fd/12363408/d0b3047b928d/PRO-34-e70276-g001.jpg" loading="lazy" id="jats-graphic-7" height="295" width="709" alt="FIGURE 4"></p>
<div class="p text-right font-secondary"><a href="figure/pro70276-fig-0004/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>DLS and NMR analysis of C99 in <em>E. coli</em> lipid‐pentyl‐inulin nanodiscs. (a) DLS profile showing a hydrodynamic radius of ~10 nm. (b) <sup>1</sup>H NMR spectrum of C99‐rich pentyl‐inulin nanodiscs. The amide region (violet bar), <em>E. coli</em> lipids and the pentyl‐inulin polymer (highlighted by red box and green bar).</p></figcaption></figure><p>The composition of the SEC‐purified sample was analyzed by <sup>1</sup>H NMR. The peaks representing polymer (from pentyl group and carbohydrate moiety) and bacterial lipids were observed (Figure <a href="#pro70276-fig-0004" class="usa-link">4b</a>). However, the peaks corresponding to the C99 protein were not prominent due to line broadening from its interaction with lipids in nanometer‐sized polymer nanodiscs. To confirm the presence of C99, additional verification was performed using mass spectrometry.</p></section><section id="pro70276-sec-0024"><h3 class="pmc_sec_title">3.7. 
MALDI analysis of C99‐nanodiscs</h3>
<p>The molecular mass of C99 in bacterial native nanodiscs was verified utilizing MALDI‐TOF. The expected mass of C99 is 13,779.76 Da; however, the mass values obtained through MALDI were 14,276 Da, 14,491 Da, and 14,764 Da (Figure <a href="#pro70276-fig-0005" class="usa-link">5a</a>), which are higher than expected. The observed increase in molecular weight (~500 to 1000 Da) may result from the association of one or more bacterial lipids with the C99 protein, particularly since C99 was isolated from <em>E. coli</em> native lipid membranes without denaturation during extraction. While strong interactions with other biomolecules or self‐aggregation cannot be ruled out, these possibilities are beyond the scope of this study. To investigate and confirm the mass difference indicated by the C99 peaks, the samples were treated with the detergent Empigen, both in the presence and absence of denaturants such as guanidine hydrochloride (GdnHCl) and urea.</p>
<figure class="fig xbox font-sm" id="pro70276-fig-0005"><h4 class="obj_head">FIGURE 5.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fd/12363408/d5cb1f1678e7/PRO-34-e70276-g005.jpg" loading="lazy" id="jats-graphic-9" height="563" width="709" alt="FIGURE 5"></p>
<div class="p text-right font-secondary"><a href="figure/pro70276-fig-0005/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>MALDI‐TOF MS spectra of C99 nanodiscs. (a). A peak corresponding to a molecular weight of 14,276 Da was observed, along with weak‐intensity peaks at higher molecular weights. The matrix used was SA. (b) After treatment with 3% Empigen, new peaks at molecular weights of 13,747.5 Da and 13,921.5 Da emerged. The matrix used was SA. (c) Following 3% Empigen treatment and analysis with the DHB matrix, the new peaks observed in Figure <a href="#pro70276-fig-0005" class="usa-link">5b</a> showed increased intensity in Figure <a href="#pro70276-fig-0005" class="usa-link">5c</a>, indicating improved resolution with the DHB matrix. (d) After treating C99 nanodiscs with 3% Empigen and 6 M GdnHCl (1:1) treatment, a significant suppression in the 14,269 Da peak was observed. The samples were desalted with C4 or C18 resin ZipTip 10 μL tips (Merck Millipore, USA) and mixed with SA, DHB, or CHCA matrices in a 1:1 ratio before spots on 384‐well stainless‐steel targets were prepared. Ionization was performed on a Bruker Autoflex mass spectrometer, and data were analyzed using Bruker flexAnalysis software.</p></figcaption></figure><p>Two news peaks at 13,747.5 Da and 13,921.5 Da (in the near‐expected region), along with a peak at 14,271.9 Da (with a decrease in intensity by 3‐fold) were observed (Figure <a href="#pro70276-fig-0005" class="usa-link">5b</a>). The intensity‐specific correlation indicates a possible interaction between lipids and C99 or C99 self‐aggregation. Upon changing the matrix to DHB while keeping all the other parameters the same, as kept for Figure <a href="#pro70276-fig-0005" class="usa-link">5b</a>, high‐intensity peaks in the near‐expected area emerged (Figure <a href="#pro70276-fig-0005" class="usa-link">5c</a>). The peaks at 13,745 and 13,916 Da were higher (~3.8‐ and 3.9‐fold, respectively) in intensity than what was obtained with SA. The peak at 14,269 went further down to an intensity of 87, inferring a 1.75‐fold decrease when compared with Figure <a href="#pro70276-fig-0005" class="usa-link">5b</a>. Compared to Figure <a href="#pro70276-fig-0005" class="usa-link">5a</a>, a drastic decrement of 5‐fold was observed in the ~14,200 Da peak.</p>
<p>To completely remove the possible interactions, C99 was treated with 6 M GdnHCl (1:1) and 8 M Urea (1:1) before analyzing by MALDI‐TOF (Figure <a href="#pro70276-fig-0005" class="usa-link">5d</a> and <a href="#pro70276-supitem-0001" class="usa-link">S5B</a>). Interestingly, the 14,269 peak almost disappeared, and the intensity of the 14,269 peak significantly went down to 44 (Figure <a href="#pro70276-fig-0005" class="usa-link">5d</a>) from 87 (Figure <a href="#pro70276-fig-0005" class="usa-link">5c</a>), decreasing by two‐fold. Compared to Figure <a href="#pro70276-fig-0005" class="usa-link">5a</a>, the fold difference in Figure <a href="#pro70276-fig-0005" class="usa-link">5d</a> is highly significant (10‐fold). The urea‐treated C99 (Figure <a href="#pro70276-supitem-0001" class="usa-link">S5B</a>, red peaks) was not significantly different from the Empigen only sample (Figure <a href="#pro70276-fig-0005" class="usa-link">5c</a>); however, the 6 M GdnHCl sample was clearly better than 8 M Urea in the case of C99. Interestingly, when the samples were analyzed without desalting them, no signals were observed in MALDI‐TOF (Figure <a href="#pro70276-supitem-0001" class="usa-link">S5</a>).</p>
<p>The presence of two peaks (13,749 and ~13,919 Da) suggests a heterogeneous nature of the sample. Using the reconstitution method, we recently reported the C99 size as 13,741 Da (Krishnarjuna, Sharma, Hiiuk, et al., <a href="#pro70276-bib-0031" class="usa-link" aria-describedby="pro70276-bib-0031">2024</a>). In that study, both detergent and denaturant were used before reconstituting C99 into nanodiscs. Hence, the molecular mass of C99 would be more accurate as only one prominent species was detected. In the current study, the observed molecular mass of C99 was ~13,745 and ~13,916 Da (Figure <a href="#pro70276-fig-0005" class="usa-link">5c</a>). The first species closely matches the reconstitution study, and the difference of a few Da could be due to the addition of adduct H<sup>+</sup> ions or the intrinsic sensitivity of the instrument used. However, the second species (~13,916 Da, approximately 171 Da larger) could not be conclusively characterized, even after treating the sample with both detergents and denaturants. Future studies could explore alternative combinations of solubilizing agents to isolate a single homogeneous species. Promising conditions may include 6 M GdnHCl in combination with DTT, CHAPS, or SDS, as well as SDS treatment coupled with heat and reducing agents such as DTT. Additionally, biochemical assays could help elucidate the identity of the higher‐mass species; for example, deglycosylation using PNGase F or Endo H, or phosphatase treatment may reveal post‐translational modifications. The structural and functional relevance of this second species could also be investigated using biophysical approaches, such as circular dichroism (CD) spectroscopy or NMR, particularly following detergent or denaturant treatment. Binding assays with lipids, APP‐processing proteins, or conformation‐specific antibodies may uncover differential interaction profiles between the two forms. However, such analyses would require the isolation of individual species, a technically challenging task that falls outside the scope of the current study.</p>
<p>To validate our observations and further check the presence of C99 in the tested sample for MALDI‐TOF, LC–MS experiments were performed.</p></section><section id="pro70276-sec-0025"><h3 class="pmc_sec_title">3.8. 
C99 sequence analysis by tandem LC–MS/MS
</h3>
<p>To further validate the presence of C99 in our direct‐extraction sample, tandem mass spectrometry (LC–MS/MS) was performed. LC–MS/MS analysis demonstrated substantial sequence coverage (Figure <a href="#pro70276-supitem-0001" class="usa-link">S6B</a>), providing strong evidence for the presence of C99. Peptides bearing carbamylation (C), deamidation (D), and oxidation (M) modifications were also identified (Figure <a href="#pro70276-supitem-0001" class="usa-link">S65B</a>). An XCorr value of 6.8, together with an HT score of 140, indicates a good fit and strong confidence in the data. Additionally, the peptide‐spectrum match (PSM) count was 54, and detection of nine unique peptides, both within optimal range, further supports the reliability of the results. A detailed summary of these parameters is provided in Figure <a href="#pro70276-supitem-0001" class="usa-link">S6A</a>.</p>
<p>Several high‐confidence MS/MS spectra corresponding to distinct tryptic peptides were acquired (Figure <a href="#pro70276-supitem-0001" class="usa-link">S6B</a>). A particularly illustrative example is the tryptic peptide “QYTSIHHGVVEVDAAVTPEER,” corresponding to residues 57–77 of C99. The fragmentation evidence for this peptide is presented in Figure <a href="#pro70276-supitem-0001" class="usa-link">S6D</a>, where an annotated tandem MS/MS spectrum displays both b and y ion series. This detailed fragmentation pattern supports confident peptide identification and sequence validation. A comprehensive tabulation of all observed b and y ions for this peptide is included in Figure <a href="#pro70276-supitem-0001" class="usa-link">S6C</a>. Taken together, the tandem LC–MS/MS analysis, coupled with Western blot validation, provides robust confirmation of C99 presence in the directly isolated sample, demonstrating high analytical concordance.</p></section></section><section id="pro70276-sec-0026"><h2 class="pmc_sec_title">4. CONCLUSION</h2>
<p>BC99 has been implicated in the pathophysiology of Alzheimer's disease (Jordà‐Siquier et al., <a href="#pro70276-bib-0024" class="usa-link" aria-describedby="pro70276-bib-0024">2022</a>; Lauritzen et al., <a href="#pro70276-bib-0035" class="usa-link" aria-describedby="pro70276-bib-0035"><sup>2019</sup></a>). Although C99 has been extensively studied in various membrane‐mimetic systems, its purification has traditionally relied on the use of detergents or denaturing agents, which disrupt protein–protein interactions (Yang et al., <a href="#pro70276-bib-0070" class="usa-link" aria-describedby="pro70276-bib-0070">2014</a>) and compromise native lipid‐protein interactions essential for its functional conformation (Krishnarjuna &amp; Ramamoorthy, <a href="#pro70276-bib-0029" class="usa-link" aria-describedby="pro70276-bib-0029">2022</a>; Yang et al., <a href="#pro70276-bib-0070" class="usa-link" aria-describedby="pro70276-bib-0070"><sup>2014</sup></a>). In this study, we directly isolated C99 along with its associated lipids from <em>E. coli</em> using a non‐ionic pentyl‐inulin polymer, thereby preserving its near‐native lipid environment. The resulting C99‐rich nanodiscs were subsequently purified, maintaining the integrity of the lipid‐protein complex. The presence of isolated C99 was confirmed by SDS‐PAGE and Western blot, where SDS‐stable dimers and oligomers were observed. The formation of C99‐containing nanodiscs was further supported by DLS, which showed a hydrodynamic radius of ~10 nm. NMR spectroscopy analysis verified the presence of <em>E. coli</em> lipids and pentyl‐inulin, confirming nanodiscs formation.</p>
<p>MALDI‐TOF confirmed the expected molecular mass of C99, consistent with a previous C99 reconstitution study (Krishnarjuna, Sharma, Hiiuk, et al., <a href="#pro70276-bib-0031" class="usa-link" aria-describedby="pro70276-bib-0031">2024</a>), and also revealed a new species at ~13,900 Da that remains in equilibrium with the main species (~13,745 Da). Finally, the amino acid sequence of the isolated C99 was validated by tandem LC–MS/MS analysis. We propose that this optimized detergent‐free isolation method could be broadly applicable for the isolation and study of other challenging membrane proteins, including amyloid precursor protein.</p></section><section id="pro70276-sec-0030"><h2 class="pmc_sec_title">AUTHOR CONTRIBUTIONS</h2>
<p>
<strong>Gaurav Sharma:</strong> Methodology; data curation; investigation; validation; formal analysis; visualization; writing – original draft; writing – review and editing. <strong>Bankala Krishnarjuna:</strong> Conceptualization; methodology; investigation; validation; formal analysis; visualization; data curation; writing – review and editing. <strong>Volodymyr M. Hiiuk:</strong> Investigation. <strong>Magdalena I. Ivanova:</strong> Supervision; investigation; writing – review and editing; validation; resources; formal analysis. <strong>Pavel Nagorny:</strong> Supervision; resources. <strong>Ayyalusamy Ramamoorthy:</strong> Conceptualization; methodology; investigation; supervision; funding acquisition; project administration; resources; writing – review and editing; data curation; formal analysis.</p></section><section id="sec28"><h2 class="pmc_sec_title">Supporting information</h2>
<section class="sm xbox font-sm" id="pro70276-supitem-0001"><div class="caption p"><p>
<strong>DATA S1.</strong> Supporting Information.</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12363408/bin/PRO-34-e70276-s001.pdf" data-ga-action="click_feat_suppl" class="usa-link">PRO-34-e70276-s001.pdf</a><sup> (2.4MB, pdf) </sup>
</div></div></section></section><section id="pro70276-sec-0027" class="ack"><h2 class="pmc_sec_title">ACKNOWLEDGMENTS</h2>
<p>This study was supported by the National Institutes of Health (NIH) (R35 GM139573 and RO1 DK132214). Volodymyr Hiiuk was supported by funds from R35 GM139573 and R35 GM136341. The Orbitrap Fusion Lumos instrument was supported by the Office of the Director, National Institutes of Health, under Award Number S10OD021619. We thank Mr. Joseph Marte for his participation in protein expression at the beginning period of the project, and Dr. Thirupathi Ravula (University of Wisconsin) for help with polymer synthesis.</p></section><section id="notes1"><p>

Sharma G, Krishnarjuna B, Hiiuk VM, Ivanova MI, Nagorny P, Ramamoorthy A. Detergent‐free isolation and characterization of amyloid precursor protein C99 in <em>E. coli</em> native lipid‐nanodiscs using non‐ionic polymer. Protein Science. 2025;34(9):e70276. 10.1002/pro.70276


</p>
<section id="pro70276-ntgp-0001" class="fn-group"><div class="fn-group p font-secondary-light font-sm">
<div class="fn p" id="pro70276-note-0001"><p>Gaurav Sharma and Bankala Krishnarjuna have contributed equally to this study.</p></div>
<div class="fn p" id="pro70276-note-0002"><p>
<strong>Review Editor:</strong> Hideo Akutsu</p></div>
</div></section></section><section id="pro70276-sec-0029"><h2 class="pmc_sec_title">DATA AVAILABILITY STATEMENT</h2>
<p>The data that support the findings of this study are available from the corresponding author upon reasonable request.</p></section><section id="pro70276-bibl-0001" class="ref-list"><h2 class="pmc_sec_title">REFERENCES</h2>
<section id="pro70276-bibl-0001_sec2"><ol class="ref-list font-sm">
<li id="pro70276-bib-0001">
<cite>
Abraham CB, Xu L, Pantelopulos GA, Straub JE. Characterizing the transmembrane domains of ADAM10 and BACE1 and the impact of membrane composition. Biophysical Journal. 2023;122:3999–4010.
</cite> [<a href="https://doi.org/10.1016/j.bpj.2023.08.025" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10560698/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37658602/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biophysical%20Journal&amp;title=Characterizing%20the%20transmembrane%20domains%20of%20ADAM10%20and%20BACE1%20and%20the%20impact%20of%20membrane%20composition&amp;author=CB%20Abraham&amp;author=L%20Xu&amp;author=GA%20Pantelopulos&amp;author=JE%20Straub&amp;volume=122&amp;publication_year=2023&amp;pages=3999-4010&amp;pmid=37658602&amp;doi=10.1016/j.bpj.2023.08.025&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70276-bib-0002">
<cite>
Alzheimer's A
. Alzheimer's disease facts and figures. Alzheimers Dement. 2024;20:3708–3821.
</cite> [<a href="https://doi.org/10.1002/alz.13809" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11095490/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38689398/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Alzheimers%20Dement&amp;title=Alzheimer's%20disease%20facts%20and%20figures&amp;volume=20&amp;publication_year=2024&amp;pages=3708-3821&amp;pmid=38689398&amp;doi=10.1002/alz.13809&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70276-bib-0003">
<cite>
Arinaminpathy Y, Khurana E, Engelman DM, Gerstein MB. Computational analysis of membrane proteins: the largest class of drug targets. Drug Discovery Today. 2009;14:1130–1135.
</cite> [<a href="https://doi.org/10.1016/j.drudis.2009.08.006" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2796609/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19733256/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Drug%20Discovery%20Today&amp;title=Computational%20analysis%20of%20membrane%20proteins:%20the%20largest%20class%20of%20drug%20targets&amp;author=Y%20Arinaminpathy&amp;author=E%20Khurana&amp;author=DM%20Engelman&amp;author=MB%20Gerstein&amp;volume=14&amp;publication_year=2009&amp;pages=1130-1135&amp;pmid=19733256&amp;doi=10.1016/j.drudis.2009.08.006&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70276-bib-0004">
<cite>
Bao H, Dalal K, Wang V, Rouiller I, Duong F. The maltose ABC transporter: action of membrane lipids on the transporter stability, coupling and ATPase activity. Biochimica et Biophysica Acta. 2013;1828:1723–1730.
</cite> [<a href="https://doi.org/10.1016/j.bbamem.2013.03.024" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23562402/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biochimica%20et%20Biophysica%20Acta&amp;title=The%20maltose%20ABC%20transporter:%20action%20of%20membrane%20lipids%20on%20the%20transporter%20stability,%20coupling%20and%20ATPase%20activity&amp;author=H%20Bao&amp;author=K%20Dalal&amp;author=V%20Wang&amp;author=I%20Rouiller&amp;author=F%20Duong&amp;volume=1828&amp;publication_year=2013&amp;pages=1723-1730&amp;pmid=23562402&amp;doi=10.1016/j.bbamem.2013.03.024&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70276-bib-0005">
<cite>
Batool M, Ahmad B, Choi S. A structure‐based drug discovery paradigm. International Journal of Molecular Sciences. 2019;20:2783.
</cite> [<a href="https://doi.org/10.3390/ijms20112783" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6601033/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31174387/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=International%20Journal%20of%20Molecular%20Sciences&amp;title=A%20structure%E2%80%90based%20drug%20discovery%20paradigm&amp;author=M%20Batool&amp;author=B%20Ahmad&amp;author=S%20Choi&amp;volume=20&amp;publication_year=2019&amp;pages=2783&amp;pmid=31174387&amp;doi=10.3390/ijms20112783&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70276-bib-0006">
<cite>
Bayburt TH, Grinkova YV, Sligar SG. Self‐assembly of discoidal phospholipid bilayer nanoparticles with membrane scaffold proteins. Nano Letters. 2002;2:853–856.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Nano%20Letters&amp;title=Self%E2%80%90assembly%20of%20discoidal%20phospholipid%20bilayer%20nanoparticles%20with%20membrane%20scaffold%20proteins&amp;author=TH%20Bayburt&amp;author=YV%20Grinkova&amp;author=SG%20Sligar&amp;volume=2&amp;publication_year=2002&amp;pages=853-856&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70276-bib-0007">
<cite>
Bayburt TH, Sligar SG. Membrane protein assembly into Nanodiscs. FEBS Letters. 2010;584:1721–1727.
</cite> [<a href="https://doi.org/10.1016/j.febslet.2009.10.024" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4758813/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19836392/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=FEBS%20Letters&amp;title=Membrane%20protein%20assembly%20into%20Nanodiscs&amp;author=TH%20Bayburt&amp;author=SG%20Sligar&amp;volume=584&amp;publication_year=2010&amp;pages=1721-1727&amp;pmid=19836392&amp;doi=10.1016/j.febslet.2009.10.024&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70276-bib-0008">
<cite>
Beel AJ, Mobley CK, Kim HJ, Tian F, Hadziselimovic A, Jap B, et al. Structural studies of the transmembrane c‐terminal domain of the amyloid precursor protein (APP): does APP function as a cholesterol sensor?
Biochemistry. 2008;47:9428–9446.
</cite> [<a href="https://doi.org/10.1021/bi800993c" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2572687/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18702528/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biochemistry&amp;title=Structural%20studies%20of%20the%20transmembrane%20c%E2%80%90terminal%20domain%20of%20the%20amyloid%20precursor%20protein%20(APP):%20does%20APP%20function%20as%20a%20cholesterol%20sensor?&amp;author=AJ%20Beel&amp;author=CK%20Mobley&amp;author=HJ%20Kim&amp;author=F%20Tian&amp;author=A%20Hadziselimovic&amp;volume=47&amp;publication_year=2008&amp;pages=9428-9446&amp;pmid=18702528&amp;doi=10.1021/bi800993c&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70276-bib-0009">
<cite>
Beel AJ, Sakakura M, Barrett PJ, Sanders CR. Direct binding of cholesterol to the amyloid precursor protein: An important interaction in lipid‐Alzheimer's disease relationships?
Biochimica et Biophysica Acta. 2010;1801:975–982.
</cite> [<a href="https://doi.org/10.1016/j.bbalip.2010.03.008" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2886191/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20304095/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biochimica%20et%20Biophysica%20Acta&amp;title=Direct%20binding%20of%20cholesterol%20to%20the%20amyloid%20precursor%20protein:%20An%20important%20interaction%20in%20lipid%E2%80%90Alzheimer's%20disease%20relationships?&amp;author=AJ%20Beel&amp;author=M%20Sakakura&amp;author=PJ%20Barrett&amp;author=CR%20Sanders&amp;volume=1801&amp;publication_year=2010&amp;pages=975-982&amp;pmid=20304095&amp;doi=10.1016/j.bbalip.2010.03.008&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70276-bib-0010">
<cite>
Bentley MR, Ilyichova OV, Wang G, Williams ML, Sharma G, Alwan WS, et al. Rapid elaboration of fragments into leads by X‐ray crystallographic screening of parallel chemical libraries (REFiLX). Journal of Medicinal Chemistry. 2020;63:6863–6875.
</cite> [<a href="https://doi.org/10.1021/acs.jmedchem.0c00111" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32529824/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Medicinal%20Chemistry&amp;title=Rapid%20elaboration%20of%20fragments%20into%20leads%20by%20X%E2%80%90ray%20crystallographic%20screening%20of%20parallel%20chemical%20libraries%20(REFiLX)&amp;author=MR%20Bentley&amp;author=OV%20Ilyichova&amp;author=G%20Wang&amp;author=ML%20Williams&amp;author=G%20Sharma&amp;volume=63&amp;publication_year=2020&amp;pages=6863-6875&amp;pmid=32529824&amp;doi=10.1021/acs.jmedchem.0c00111&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70276-bib-0011">
<cite>
Bou‐Assaf GM, Marshall AG. Chapter 12 ‐ biophysical mass spectrometry for biopharmaceutical process development: focus on hydrogen/deuterium exchange. In: Houde DJ, Berkowitz SA, editors. Biophysical characterization of proteins in developing biopharmaceuticals (second edition). Amsterdam: Netherlands, Elsevier; 2020. p. 333–374.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=Biophysical%20characterization%20of%20proteins%20in%20developing%20biopharmaceuticals%20(second%20edition)&amp;author=GM%20Bou%E2%80%90Assaf&amp;author=AG%20Marshall&amp;publication_year=2020&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70276-bib-0012">
<cite>
Carpenter EP, Beis K, Cameron AD, Iwata S. Overcoming the challenges of membrane protein crystallography. Current Opinion in Structural Biology. 2008;18:581–586.
</cite> [<a href="https://doi.org/10.1016/j.sbi.2008.07.001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2580798/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18674618/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Current%20Opinion%20in%20Structural%20Biology&amp;title=Overcoming%20the%20challenges%20of%20membrane%20protein%20crystallography&amp;author=EP%20Carpenter&amp;author=K%20Beis&amp;author=AD%20Cameron&amp;author=S%20Iwata&amp;volume=18&amp;publication_year=2008&amp;pages=581-586&amp;pmid=18674618&amp;doi=10.1016/j.sbi.2008.07.001&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70276-bib-0013">
<cite>
Chen GF, Xu TH, Yan Y, Zhou YR, Jiang Y, Melcher K, et al. Amyloid beta: structure, biology and structure‐based therapeutic development. Acta Pharmacologica Sinica. 2017;38:1205–1235.
</cite> [<a href="https://doi.org/10.1038/aps.2017.28" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5589967/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28713158/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Acta%20Pharmacologica%20Sinica&amp;title=Amyloid%20beta:%20structure,%20biology%20and%20structure%E2%80%90based%20therapeutic%20development&amp;author=GF%20Chen&amp;author=TH%20Xu&amp;author=Y%20Yan&amp;author=YR%20Zhou&amp;author=Y%20Jiang&amp;volume=38&amp;publication_year=2017&amp;pages=1205-1235&amp;pmid=28713158&amp;doi=10.1038/aps.2017.28&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70276-bib-0014">
<cite>
Davra V, Benzeroual KE. Flavonoids and fibrate modulate apoE4‐induced processing of amyloid precursor protein in neuroblastoma cells. Frontiers in Neuroscience. 2023;17:1245895.
</cite> [<a href="https://doi.org/10.3389/fnins.2023.1245895" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10777729/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38204816/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Neuroscience&amp;title=Flavonoids%20and%20fibrate%20modulate%20apoE4%E2%80%90induced%20processing%20of%20amyloid%20precursor%20protein%20in%20neuroblastoma%20cells&amp;author=V%20Davra&amp;author=KE%20Benzeroual&amp;volume=17&amp;publication_year=2023&amp;pages=1245895&amp;pmid=38204816&amp;doi=10.3389/fnins.2023.1245895&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70276-bib-0015">
<cite>
Denisov IG, Sligar SG. Nanodiscs for structural and functional studies of membrane proteins. Nature Structural &amp; Molecular Biology. 2016;23:481–486.</cite> [<a href="https://doi.org/10.1038/nsmb.3195" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8934039/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27273631/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Structural%20&amp;%20Molecular%20Biology&amp;title=Nanodiscs%20for%20structural%20and%20functional%20studies%20of%20membrane%20proteins&amp;author=IG%20Denisov&amp;author=SG%20Sligar&amp;volume=23&amp;publication_year=2016&amp;pages=481-486&amp;pmid=27273631&amp;doi=10.1038/nsmb.3195&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70276-bib-0016">
<cite>
Dominguez L, Foster L, Straub JE, Thirumalai D. Impact of membrane lipid composition on the structure and stability of the transmembrane domain of amyloid precursor protein. Proceedings of the National Academy of Sciences of the United States of America. 2016;113:E5281–E5287.
</cite> [<a href="https://doi.org/10.1073/pnas.1606482113" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5018773/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27559086/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proceedings%20of%20the%20National%20Academy%20of%20Sciences%20of%20the%20United%20States%20of%20America&amp;title=Impact%20of%20membrane%20lipid%20composition%20on%20the%20structure%20and%20stability%20of%20the%20transmembrane%20domain%20of%20amyloid%20precursor%20protein&amp;author=L%20Dominguez&amp;author=L%20Foster&amp;author=JE%20Straub&amp;author=D%20Thirumalai&amp;volume=113&amp;publication_year=2016&amp;pages=E5281-E5287&amp;pmid=27559086&amp;doi=10.1073/pnas.1606482113&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70276-bib-0017">
<cite>
Drulyte I, Gutgsell AR, Lloris‐Garcerá P, Liss M, Geschwindner S, Radjainia M, et al. Direct cell extraction of membrane proteins for structure‐function analysis. Scientific Reports. 2023;13:1420.
</cite> [<a href="https://doi.org/10.1038/s41598-023-28455-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9876986/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36697499/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Scientific%20Reports&amp;title=Direct%20cell%20extraction%20of%20membrane%20proteins%20for%20structure%E2%80%90function%20analysis&amp;author=I%20Drulyte&amp;author=AR%20Gutgsell&amp;author=P%20Lloris%E2%80%90Garcer%C3%A1&amp;author=M%20Liss&amp;author=S%20Geschwindner&amp;volume=13&amp;publication_year=2023&amp;pages=1420&amp;pmid=36697499&amp;doi=10.1038/s41598-023-28455-w&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70276-bib-0018">
<cite>
Emmanuel N, He Q, Kang Y, Zhang D, Gao M, Wang M, et al. Hydrophobicity causes anomalous migration of cystine/glutamate antiporter SLC7A11 in SDS‐PAGE with low acrylamide concentration. FEBS Open Bio. 2025;15:994–1008.</cite> [<a href="https://doi.org/10.1002/2211-5463.70019" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC12127883/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40125830/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=FEBS%20Open%20Bio&amp;title=Hydrophobicity%20causes%20anomalous%20migration%20of%20cystine/glutamate%20antiporter%20SLC7A11%20in%20SDS%E2%80%90PAGE%20with%20low%20acrylamide%20concentration&amp;author=N%20Emmanuel&amp;author=Q%20He&amp;author=Y%20Kang&amp;author=D%20Zhang&amp;author=M%20Gao&amp;volume=15&amp;publication_year=2025&amp;pages=994-1008&amp;pmid=40125830&amp;doi=10.1002/2211-5463.70019&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70276-bib-0019">
<cite>
Errey JC, Fiez‐Vandal C. Production of membrane proteins in industry: the example of GPCRs. Protein Expression and Purification. 2020;169:105569.
</cite> [<a href="https://doi.org/10.1016/j.pep.2020.105569" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31945417/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Protein%20Expression%20and%20Purification&amp;title=Production%20of%20membrane%20proteins%20in%20industry:%20the%20example%20of%20GPCRs&amp;author=JC%20Errey&amp;author=C%20Fiez%E2%80%90Vandal&amp;volume=169&amp;publication_year=2020&amp;pages=105569&amp;pmid=31945417&amp;doi=10.1016/j.pep.2020.105569&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70276-bib-0020">
<cite>
Fawzi NL, Ying J, Ghirlando R, Torchia DA, Clore GM. Atomic‐resolution dynamics on the surface of amyloid‐β protofibrils probed by solution NMR. Nature. 2011;480:268–272.
</cite> [<a href="https://doi.org/10.1038/nature10577" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3237923/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22037310/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Atomic%E2%80%90resolution%20dynamics%20on%20the%20surface%20of%20amyloid%E2%80%90%CE%B2%20protofibrils%20probed%20by%20solution%20NMR&amp;author=NL%20Fawzi&amp;author=J%20Ying&amp;author=R%20Ghirlando&amp;author=DA%20Torchia&amp;author=GM%20Clore&amp;volume=480&amp;publication_year=2011&amp;pages=268-272&amp;pmid=22037310&amp;doi=10.1038/nature10577&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70276-bib-0021">
<cite>
Gerber H, Wu F, Dimitrov M, Garcia Osuna GM, Fraering PC. Zinc and copper differentially modulate amyloid precursor protein processing by γ‐secretase and amyloid‐β peptide production. The Journal of Biological Chemistry. 2017;292:3751–3767.
</cite> [<a href="https://doi.org/10.1074/jbc.M116.754101" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5339758/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28096459/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=The%20Journal%20of%20Biological%20Chemistry&amp;title=Zinc%20and%20copper%20differentially%20modulate%20amyloid%20precursor%20protein%20processing%20by%20%CE%B3%E2%80%90secretase%20and%20amyloid%E2%80%90%CE%B2%20peptide%20production&amp;author=H%20Gerber&amp;author=F%20Wu&amp;author=M%20Dimitrov&amp;author=GM%20Garcia%20Osuna&amp;author=PC%20Fraering&amp;volume=292&amp;publication_year=2017&amp;pages=3751-3767&amp;pmid=28096459&amp;doi=10.1074/jbc.M116.754101&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70276-bib-0022">
<cite>
Guo X, Li H, Yan C, Lei J, Zhou R, Shi Y. Molecular mechanism of substrate recognition and cleavage by human γ‐secretase. Science. 2024;384:1091–1095.
</cite> [<a href="https://doi.org/10.1126/science.adn5820" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38843321/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Molecular%20mechanism%20of%20substrate%20recognition%20and%20cleavage%20by%20human%20%CE%B3%E2%80%90secretase&amp;author=X%20Guo&amp;author=H%20Li&amp;author=C%20Yan&amp;author=J%20Lei&amp;author=R%20Zhou&amp;volume=384&amp;publication_year=2024&amp;pages=1091-1095&amp;pmid=38843321&amp;doi=10.1126/science.adn5820&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70276-bib-0023">
<cite>
Hutchison JM, Shih K‐C, Scheidt HA, Fantin SM, Parson KF, Pantelopulos GA, et al. Bicelles rich in both sphingolipids and cholesterol and their use in studies of membrane proteins. Journal of the American Chemical Society. 2020;142:12715–12729.
</cite> [<a href="https://doi.org/10.1021/jacs.0c04669" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7924963/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32575981/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20the%20American%20Chemical%20Society&amp;title=Bicelles%20rich%20in%20both%20sphingolipids%20and%20cholesterol%20and%20their%20use%20in%20studies%20of%20membrane%20proteins&amp;author=JM%20Hutchison&amp;author=K%E2%80%90C%20Shih&amp;author=HA%20Scheidt&amp;author=SM%20Fantin&amp;author=KF%20Parson&amp;volume=142&amp;publication_year=2020&amp;pages=12715-12729&amp;pmid=32575981&amp;doi=10.1021/jacs.0c04669&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70276-bib-0024">
<cite>
Jordà‐Siquier T, Petrel M, Kouskoff V, Smailovic U, Cordelières F, Frykman S, et al. APP accumulates with presynaptic proteins around amyloid plaques: a role for presynaptic mechanisms in Alzheimer's disease?
Alzheimers Dement. 2022;18:2099–2116.
</cite> [<a href="https://doi.org/10.1002/alz.12546" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9786597/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35076178/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Alzheimers%20Dement&amp;title=APP%20accumulates%20with%20presynaptic%20proteins%20around%20amyloid%20plaques:%20a%20role%20for%20presynaptic%20mechanisms%20in%20Alzheimer's%20disease?&amp;author=T%20Jord%C3%A0%E2%80%90Siquier&amp;author=M%20Petrel&amp;author=V%20Kouskoff&amp;author=U%20Smailovic&amp;author=F%20Cordeli%C3%A8res&amp;volume=18&amp;publication_year=2022&amp;pages=2099-2116&amp;pmid=35076178&amp;doi=10.1002/alz.12546&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70276-bib-0025">
<cite>
Julien JA, Fernandez MG, Brandmier KM, Del Mundo JT, Bator CM, Loftus LA, et al. Rapid preparation of nanodiscs for biophysical studies. Archives of Biochemistry and Biophysics. 2021;712:109051.
</cite> [<a href="https://doi.org/10.1016/j.abb.2021.109051" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8596956/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34610337/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Archives%20of%20Biochemistry%20and%20Biophysics&amp;title=Rapid%20preparation%20of%20nanodiscs%20for%20biophysical%20studies&amp;author=JA%20Julien&amp;author=MG%20Fernandez&amp;author=KM%20Brandmier&amp;author=JT%20Del%20Mundo&amp;author=CM%20Bator&amp;volume=712&amp;publication_year=2021&amp;pages=109051&amp;pmid=34610337&amp;doi=10.1016/j.abb.2021.109051&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70276-bib-0026">
<cite>
Koch M, Enzlein T, Chen SY, Petit D, Lismont S, Zacharias M, et al. APP substrate ectodomain defines amyloid‐β peptide length by restraining γ‐secretase processivity and facilitating product release. The EMBO Journal. 2023;42:e114372.
</cite> [<a href="https://doi.org/10.15252/embj.2023114372" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10690472/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37853914/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=The%20EMBO%20Journal&amp;title=APP%20substrate%20ectodomain%20defines%20amyloid%E2%80%90%CE%B2%20peptide%20length%20by%20restraining%20%CE%B3%E2%80%90secretase%20processivity%20and%20facilitating%20product%20release&amp;author=M%20Koch&amp;author=T%20Enzlein&amp;author=SY%20Chen&amp;author=D%20Petit&amp;author=S%20Lismont&amp;volume=42&amp;publication_year=2023&amp;pages=e114372&amp;pmid=37853914&amp;doi=10.15252/embj.2023114372&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70276-bib-0027">
<cite>
Krishnarjuna B, Andrew D, MacRaild CA, Morales RA, Beeson JG, Anders RF, et al. Strain‐transcending immune response generated by chimeras of the malaria vaccine candidate merozoite surface protein 2. Scientific Reports. 2016;6:20613.
</cite> [<a href="https://doi.org/10.1038/srep20613" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4749986/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26865062/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Scientific%20Reports&amp;title=Strain%E2%80%90transcending%20immune%20response%20generated%20by%20chimeras%20of%20the%20malaria%20vaccine%20candidate%20merozoite%20surface%20protein%202&amp;author=B%20Krishnarjuna&amp;author=D%20Andrew&amp;author=CA%20MacRaild&amp;author=RA%20Morales&amp;author=JG%20Beeson&amp;volume=6&amp;publication_year=2016&amp;pages=20613&amp;pmid=26865062&amp;doi=10.1038/srep20613&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70276-bib-0028">
<cite>
Krishnarjuna B, Im SC, Ravula T, Marte J, Auchus RJ, Ramamoorthy A. Ionic inulin‐based polymer nanodiscs enable functional reconstitution of a redox complex composed of oppositely charged CYP450 and CPR in a lipid bilayer membrane. Analytical Chemistry. 2022;94:11908–11915.
</cite> [<a href="https://doi.org/10.1021/acs.analchem.2c02489" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10851674/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35977417/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Analytical%20Chemistry&amp;title=Ionic%20inulin%E2%80%90based%20polymer%20nanodiscs%20enable%20functional%20reconstitution%20of%20a%20redox%20complex%20composed%20of%20oppositely%20charged%20CYP450%20and%20CPR%20in%20a%20lipid%20bilayer%20membrane&amp;author=B%20Krishnarjuna&amp;author=SC%20Im&amp;author=T%20Ravula&amp;author=J%20Marte&amp;author=RJ%20Auchus&amp;volume=94&amp;publication_year=2022&amp;pages=11908-11915&amp;pmid=35977417&amp;doi=10.1021/acs.analchem.2c02489&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70276-bib-0029">
<cite>
Krishnarjuna B, Ramamoorthy A. Detergent‐free isolation of membrane proteins and strategies to study them in a near‐native membrane environment. Biomolecules. 2022;12:1076.
</cite> [<a href="https://doi.org/10.3390/biom12081076" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9406181/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36008970/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biomolecules&amp;title=Detergent%E2%80%90free%20isolation%20of%20membrane%20proteins%20and%20strategies%20to%20study%20them%20in%20a%20near%E2%80%90native%20membrane%20environment&amp;author=B%20Krishnarjuna&amp;author=A%20Ramamoorthy&amp;volume=12&amp;publication_year=2022&amp;pages=1076&amp;pmid=36008970&amp;doi=10.3390/biom12081076&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70276-bib-0030">
<cite>
Krishnarjuna B, Ravula T, Ramamoorthy A. Detergent‐free isolation of CYP450‐reductase's FMN‐binding domain in E. coli lipid‐nanodiscs using a charge‐free polymer. Chemical Communications. 2022;58:4913–4916.
</cite> [<a href="https://doi.org/10.1039/d1cc07193a" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9578324/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35356954/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Chemical%20Communications&amp;title=Detergent%E2%80%90free%20isolation%20of%20CYP450%E2%80%90reductase's%20FMN%E2%80%90binding%20domain%20in%20E.%20coli%20lipid%E2%80%90nanodiscs%20using%20a%20charge%E2%80%90free%20polymer&amp;author=B%20Krishnarjuna&amp;author=T%20Ravula&amp;author=A%20Ramamoorthy&amp;volume=58&amp;publication_year=2022&amp;pages=4913-4916&amp;pmid=35356954&amp;doi=10.1039/d1cc07193a&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70276-bib-0031">
<cite>
Krishnarjuna B, Sharma G, Hiiuk VM, Struppe J, Nagorny P, Ivanova MI, et al. Nanodisc reconstitution and characterization of amyloid‐β precursor protein C99. Analytical Chemistry. 2024;96:9362–9369.
</cite> [<a href="https://doi.org/10.1021/acs.analchem.3c05727" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38826107/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Analytical%20Chemistry&amp;title=Nanodisc%20reconstitution%20and%20characterization%20of%20amyloid%E2%80%90%CE%B2%20precursor%20protein%20C99&amp;author=B%20Krishnarjuna&amp;author=G%20Sharma&amp;author=VM%20Hiiuk&amp;author=J%20Struppe&amp;author=P%20Nagorny&amp;volume=96&amp;publication_year=2024&amp;pages=9362-9369&amp;pmid=38826107&amp;doi=10.1021/acs.analchem.3c05727&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70276-bib-0032">
<cite>
Krishnarjuna B, Sharma G, Im SC, Auchus R, Anantharamaiah GM, Ramamoorthy A. Characterization of nanodisc‐forming peptides for membrane protein studies. Journal of Colloid and Interface Science. 2024;653:1402–1414.
</cite> [<a href="https://doi.org/10.1016/j.jcis.2023.09.162" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10864042/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37801850/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Colloid%20and%20Interface%20Science&amp;title=Characterization%20of%20nanodisc%E2%80%90forming%20peptides%20for%20membrane%20protein%20studies&amp;author=B%20Krishnarjuna&amp;author=G%20Sharma&amp;author=SC%20Im&amp;author=R%20Auchus&amp;author=GM%20Anantharamaiah&amp;volume=653&amp;publication_year=2024&amp;pages=1402-1414&amp;pmid=37801850&amp;doi=10.1016/j.jcis.2023.09.162&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70276-bib-0033">
<cite>
Kumar S, Stover L, Wang L, Bahramimoghaddam H, Zhou M, Russell DH, et al. Native mass spectrometry of membrane protein–lipid interactions in different detergent environments. Analytical Chemistry. 2024;96:16768–16776.
</cite> [<a href="https://doi.org/10.1021/acs.analchem.4c03312" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11503522/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39394983/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Analytical%20Chemistry&amp;title=Native%20mass%20spectrometry%20of%20membrane%20protein%E2%80%93lipid%20interactions%20in%20different%20detergent%20environments&amp;author=S%20Kumar&amp;author=L%20Stover&amp;author=L%20Wang&amp;author=H%20Bahramimoghaddam&amp;author=M%20Zhou&amp;volume=96&amp;publication_year=2024&amp;pages=16768-16776&amp;pmid=39394983&amp;doi=10.1021/acs.analchem.4c03312&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70276-bib-0034">
<cite>
Lane CA, Hardy J, Schott JM. Alzheimer's disease. European Journal of Neurology. 2018;25:59–70.
</cite> [<a href="https://doi.org/10.1111/ene.13439" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28872215/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=European%20Journal%20of%20Neurology&amp;title=Alzheimer's%20disease&amp;author=CA%20Lane&amp;author=J%20Hardy&amp;author=JM%20Schott&amp;volume=25&amp;publication_year=2018&amp;pages=59-70&amp;pmid=28872215&amp;doi=10.1111/ene.13439&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70276-bib-0035">
<cite>
Lauritzen I, Pardossi‐Piquard R, Bourgeois A, Bécot A, Checler F. Does intraneuronal accumulation of carboxyl‐terminal fragments of the amyloid precursor protein trigger early neurotoxicity in Alzheimer's disease?
Current Alzheimer Research. 2019;16:453–457.
</cite> [<a href="https://doi.org/10.2174/1567205016666190325092841" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30907322/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Current%20Alzheimer%20Research&amp;title=Does%20intraneuronal%20accumulation%20of%20carboxyl%E2%80%90terminal%20fragments%20of%20the%20amyloid%20precursor%20protein%20trigger%20early%20neurotoxicity%20in%20Alzheimer's%20disease?&amp;author=I%20Lauritzen&amp;author=R%20Pardossi%E2%80%90Piquard&amp;author=A%20Bourgeois&amp;author=A%20B%C3%A9cot&amp;author=F%20Checler&amp;volume=16&amp;publication_year=2019&amp;pages=453-457&amp;pmid=30907322&amp;doi=10.2174/1567205016666190325092841&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70276-bib-0036">
<cite>
Lee C‐R, Park Y‐H, Kim Y‐R, Peterkofsky A, Seok Y‐J. Phosphorylation‐dependent mobility shift of proteins on SDS‐PAGE is due to decreased binding of SDS. Bulletin of Korean Chemical Society. 2013;34:2063–2066.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Bulletin%20of%20Korean%20Chemical%20Society&amp;title=Phosphorylation%E2%80%90dependent%20mobility%20shift%20of%20proteins%20on%20SDS%E2%80%90PAGE%20is%20due%20to%20decreased%20binding%20of%20SDS&amp;author=C%E2%80%90R%20Lee&amp;author=Y%E2%80%90H%20Park&amp;author=Y%E2%80%90R%20Kim&amp;author=A%20Peterkofsky&amp;author=Y%E2%80%90J%20Seok&amp;volume=34&amp;publication_year=2013&amp;pages=2063-2066&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70276-bib-0037">
<cite>
Levental I, Lyman E. Regulation of membrane protein structure and function by their lipid nano‐environment. Nature Reviews. Molecular Cell Biology. 2023;24:107–122.
</cite> [<a href="https://doi.org/10.1038/s41580-022-00524-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9892264/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36056103/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Reviews.%20Molecular%20Cell%20Biology&amp;title=Regulation%20of%20membrane%20protein%20structure%20and%20function%20by%20their%20lipid%20nano%E2%80%90environment&amp;author=I%20Levental&amp;author=E%20Lyman&amp;volume=24&amp;publication_year=2023&amp;pages=107-122&amp;pmid=36056103&amp;doi=10.1038/s41580-022-00524-4&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70276-bib-0038">
<cite>
Li CD, Junaid M, Shan X, Wang Y, Wang X, Khan A, et al. Effect of cholesterol on C99 dimerization: revealed by molecular dynamics simulations. Frontiers in Molecular Biosciences. 2022;9:872385.
</cite> [<a href="https://doi.org/10.3389/fmolb.2022.872385" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9343951/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35928227/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Molecular%20Biosciences&amp;title=Effect%20of%20cholesterol%20on%20C99%20dimerization:%20revealed%20by%20molecular%20dynamics%20simulations&amp;author=CD%20Li&amp;author=M%20Junaid&amp;author=X%20Shan&amp;author=Y%20Wang&amp;author=X%20Wang&amp;volume=9&amp;publication_year=2022&amp;pages=872385&amp;pmid=35928227&amp;doi=10.3389/fmolb.2022.872385&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70276-bib-0039">
<cite>
Ma C, Zheng F, Xu W, Liu W, Xu C, Chen Y, et al. Surface roughness effects on confined nanoscale transport of ions and biomolecules. Small Methods. 2024;8:2301485.</cite> [<a href="https://doi.org/10.1002/smtd.202301485" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38150654/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Small%20Methods&amp;title=Surface%20roughness%20effects%20on%20confined%20nanoscale%20transport%20of%20ions%20and%20biomolecules&amp;author=C%20Ma&amp;author=F%20Zheng&amp;author=W%20Xu&amp;author=W%20Liu&amp;author=C%20Xu&amp;volume=8&amp;publication_year=2024&amp;pages=2301485&amp;pmid=38150654&amp;doi=10.1002/smtd.202301485&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70276-bib-0040">
<cite>
MacRaild CA, Zachrdla M, Andrew D, Krishnarjuna B, Nováček J, Žídek L, et al. Conformational dynamics and antigenicity in the disordered malaria antigen merozoite surface protein 2. PLoS One. 2015;10:0119899.</cite> [<a href="https://doi.org/10.1371/journal.pone.0119899" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4351039/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25742002/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20One&amp;title=Conformational%20dynamics%20and%20antigenicity%20in%20the%20disordered%20malaria%20antigen%20merozoite%20surface%20protein%202&amp;author=CA%20MacRaild&amp;author=M%20Zachrdla&amp;author=D%20Andrew&amp;author=B%20Krishnarjuna&amp;author=J%20Nov%C3%A1%C4%8Dek&amp;volume=10&amp;publication_year=2015&amp;pages=0119899&amp;pmid=25742002&amp;doi=10.1371/journal.pone.0119899&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70276-bib-0041">
<cite>
Mesa H, Zhang EY, Wang Y, Zhang Q. Human neurons lacking amyloid precursor protein exhibit cholesterol‐associated developmental and presynaptic deficits. Journal of Cellular Physiology. 2024;239:e30999.
</cite> [<a href="https://doi.org/10.1002/jcp.30999" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36966431/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Cellular%20Physiology&amp;title=Human%20neurons%20lacking%20amyloid%20precursor%20protein%20exhibit%20cholesterol%E2%80%90associated%20developmental%20and%20presynaptic%20deficits&amp;author=H%20Mesa&amp;author=EY%20Zhang&amp;author=Y%20Wang&amp;author=Q%20Zhang&amp;volume=239&amp;publication_year=2024&amp;pages=e30999&amp;pmid=36966431&amp;doi=10.1002/jcp.30999&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70276-bib-0042">
<cite>
Miles AJ, Wallace BA. Circular dichroism spectroscopy of membrane proteins. Chemical Society Reviews. 2016;45:4859–4872.
</cite> [<a href="https://doi.org/10.1039/c5cs00084j" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27347568/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Chemical%20Society%20Reviews&amp;title=Circular%20dichroism%20spectroscopy%20of%20membrane%20proteins&amp;author=AJ%20Miles&amp;author=BA%20Wallace&amp;volume=45&amp;publication_year=2016&amp;pages=4859-4872&amp;pmid=27347568&amp;doi=10.1039/c5cs00084j&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70276-bib-0043">
<cite>
Morales RAV, MacRaild CA, Seow J, Krishnarjuna B, Drinkwater N, Rouet R, et al. Structural basis for epitope masking and strain specificity of a conserved epitope in an intrinsically disordered malaria vaccine candidate. Scientific Reports. 2015;5:10103.
</cite> [<a href="https://doi.org/10.1038/srep10103" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4428071/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25965408/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Scientific%20Reports&amp;title=Structural%20basis%20for%20epitope%20masking%20and%20strain%20specificity%20of%20a%20conserved%20epitope%20in%20an%20intrinsically%20disordered%20malaria%20vaccine%20candidate&amp;author=RAV%20Morales&amp;author=CA%20MacRaild&amp;author=J%20Seow&amp;author=B%20Krishnarjuna&amp;author=N%20Drinkwater&amp;volume=5&amp;publication_year=2015&amp;pages=10103&amp;pmid=25965408&amp;doi=10.1038/srep10103&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70276-bib-0044">
<cite>
Motoki K, Kume H, Oda A, Tamaoka A, Hosaka A, Kametani F, et al. Neuronal β‐amyloid generation is independent of lipid raft association of β‐secretase BACE1: analysis with a palmitoylation‐deficient mutant. Brain and Behavior: A Cognitive Neuroscience Perspective. 2012;2:270–282.</cite> [<a href="https://doi.org/10.1002/brb3.52" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3381632/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22741101/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Brain%20and%20Behavior:%20A%20Cognitive%20Neuroscience%20Perspective&amp;title=Neuronal%20%CE%B2%E2%80%90amyloid%20generation%20is%20independent%20of%20lipid%20raft%20association%20of%20%CE%B2%E2%80%90secretase%20BACE1:%20analysis%20with%20a%20palmitoylation%E2%80%90deficient%20mutant&amp;author=K%20Motoki&amp;author=H%20Kume&amp;author=A%20Oda&amp;author=A%20Tamaoka&amp;author=A%20Hosaka&amp;volume=2&amp;publication_year=2012&amp;pages=270-282&amp;pmid=22741101&amp;doi=10.1002/brb3.52&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70276-bib-0045">
<cite>
Opella SJ. Structure determination of membrane proteins by nuclear magnetic resonance spectroscopy. Annual Review of Analytical Chemistry (Palo Alto, California). 2013;6:305–328.</cite> [<a href="https://doi.org/10.1146/annurev-anchem-062012-092631" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3980955/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23577669/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Annual%20Review%20of%20Analytical%20Chemistry%20(Palo%20Alto,%20California)&amp;title=Structure%20determination%20of%20membrane%20proteins%20by%20nuclear%20magnetic%20resonance%20spectroscopy&amp;author=SJ%20Opella&amp;volume=6&amp;publication_year=2013&amp;pages=305-328&amp;pmid=23577669&amp;doi=10.1146/annurev-anchem-062012-092631&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70276-bib-0046">
<cite>
Overington JP, Al‐Lazikani B, Hopkins AL. How many drug targets are there?
Nature Reviews. Drug Discovery. 2006;5:993–996.
</cite> [<a href="https://doi.org/10.1038/nrd2199" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17139284/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Reviews.%20Drug%20Discovery&amp;title=How%20many%20drug%20targets%20are%20there?&amp;author=JP%20Overington&amp;author=B%20Al%E2%80%90Lazikani&amp;author=AL%20Hopkins&amp;volume=5&amp;publication_year=2006&amp;pages=993-996&amp;pmid=17139284&amp;doi=10.1038/nrd2199&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70276-bib-0047">
<cite>
Pantelopulos GA, Abraham CB, Straub JE. Cholesterol and lipid rafts in the biogenesis of amyloid‐β protein and Alzheimer's disease. Annual Review of Biophysics. 2024;53:455–486.</cite> [<a href="https://doi.org/10.1146/annurev-biophys-062823-023436" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11575466/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38382114/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Annual%20Review%20of%20Biophysics&amp;title=Cholesterol%20and%20lipid%20rafts%20in%20the%20biogenesis%20of%20amyloid%E2%80%90%CE%B2%20protein%20and%20Alzheimer's%20disease&amp;author=GA%20Pantelopulos&amp;author=CB%20Abraham&amp;author=JE%20Straub&amp;volume=53&amp;publication_year=2024&amp;pages=455-486&amp;pmid=38382114&amp;doi=10.1146/annurev-biophys-062823-023436&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70276-bib-0048">
<cite>
Pantelopulos GA, Matsuoka D, Hutchison JM, Sanders CR, Sugita Y, Straub JE, et al. Formation of extramembrane β‐strands controls dimerization of transmembrane helices in amyloid precursor protein C99. Proceedings of the National Academy of Sciences of the United States of America. 2022;119:e2212207119.
</cite> [<a href="https://doi.org/10.1073/pnas.2212207119" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9907117/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36538482/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proceedings%20of%20the%20National%20Academy%20of%20Sciences%20of%20the%20United%20States%20of%20America&amp;title=Formation%20of%20extramembrane%20%CE%B2%E2%80%90strands%20controls%20dimerization%20of%20transmembrane%20helices%20in%20amyloid%20precursor%20protein%20C99&amp;author=GA%20Pantelopulos&amp;author=D%20Matsuoka&amp;author=JM%20Hutchison&amp;author=CR%20Sanders&amp;author=Y%20Sugita&amp;volume=119&amp;publication_year=2022&amp;pages=e2212207119&amp;pmid=36538482&amp;doi=10.1073/pnas.2212207119&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70276-bib-0049">
<cite>
Pantelopulos GA, Straub JE, Thirumalai D, Sugita Y. Structure of APP‐C99<sub>1–99</sub> and implications for role of extra‐membrane domains in function and oligomerization. Biochimica et Biophysica Acta. 2018;1860:1698–1708.
</cite> [<a href="https://doi.org/10.1016/j.bbamem.2018.04.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6202261/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29702072/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biochimica%20et%20Biophysica%20Acta&amp;title=Structure%20of%20APP%E2%80%90C991%E2%80%9399%20and%20implications%20for%20role%20of%20extra%E2%80%90membrane%20domains%20in%20function%20and%20oligomerization&amp;author=GA%20Pantelopulos&amp;author=JE%20Straub&amp;author=D%20Thirumalai&amp;author=Y%20Sugita&amp;volume=1860&amp;publication_year=2018&amp;pages=1698-1708&amp;pmid=29702072&amp;doi=10.1016/j.bbamem.2018.04.002&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70276-bib-0050">
<cite>
Perrin F, Papadopoulos N, Suelves N, Opsomer R, Vadukul DM, Vrancx C, et al. Dimeric transmembrane orientations of APP/C99 regulate γ‐secretase processing line impacting signaling and oligomerization. iScience. 2020;23:101887.
</cite> [<a href="https://doi.org/10.1016/j.isci.2020.101887" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7749410/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33367225/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=iScience&amp;title=Dimeric%20transmembrane%20orientations%20of%20APP/C99%20regulate%20%CE%B3%E2%80%90secretase%20processing%20line%20impacting%20signaling%20and%20oligomerization&amp;author=F%20Perrin&amp;author=N%20Papadopoulos&amp;author=N%20Suelves&amp;author=R%20Opsomer&amp;author=DM%20Vadukul&amp;volume=23&amp;publication_year=2020&amp;pages=101887&amp;pmid=33367225&amp;doi=10.1016/j.isci.2020.101887&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70276-bib-0051">
<cite>
Rath A, Glibowicka M, Nadeau VG, Chen G, Deber CM. Detergent binding explains anomalous SDS‐PAGE migration of membrane proteins. Proceedings of the National Academy of Sciences of the United States of America. 2009;106:1760–1765.
</cite> [<a href="https://doi.org/10.1073/pnas.0813167106" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2644111/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19181854/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proceedings%20of%20the%20National%20Academy%20of%20Sciences%20of%20the%20United%20States%20of%20America&amp;title=Detergent%20binding%20explains%20anomalous%20SDS%E2%80%90PAGE%20migration%20of%20membrane%20proteins&amp;author=A%20Rath&amp;author=M%20Glibowicka&amp;author=VG%20Nadeau&amp;author=G%20Chen&amp;author=CM%20Deber&amp;volume=106&amp;publication_year=2009&amp;pages=1760-1765&amp;pmid=19181854&amp;doi=10.1073/pnas.0813167106&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70276-bib-0052">
<cite>
Ravula T, Ramamoorthy A. Synthesis, characterization, and nanodisc formation of non‐ionic polymers. Angewandte Chemie. 2021;60:16885–16888.
</cite> [<a href="https://doi.org/10.1002/anie.202101950" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8292192/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33998111/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Angewandte%20Chemie&amp;title=Synthesis,%20characterization,%20and%20nanodisc%20formation%20of%20non%E2%80%90ionic%20polymers&amp;author=T%20Ravula&amp;author=A%20Ramamoorthy&amp;volume=60&amp;publication_year=2021&amp;pages=16885-16888&amp;pmid=33998111&amp;doi=10.1002/anie.202101950&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70276-bib-0053">
<cite>
Rawlings AE. Membrane proteins: always an insoluble problem?
Biochemical Society Transactions. 2016;44:790–795.
</cite> [<a href="https://doi.org/10.1042/BST20160025" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4900757/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27284043/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biochemical%20Society%20Transactions&amp;title=Membrane%20proteins:%20always%20an%20insoluble%20problem?&amp;author=AE%20Rawlings&amp;volume=44&amp;publication_year=2016&amp;pages=790-795&amp;pmid=27284043&amp;doi=10.1042/BST20160025&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70276-bib-0054">
<cite>
Ritchie TK, Grinkova YV, Bayburt TH, Denisov IG, Zolnerciks JK, Atkins WM, Sligar SG (2009) Chapter 11 ‐ reconstitution of membrane proteins in phospholipid bilayer nanodiscs. Methods in Enzymology
464:211–231, Elsevier.
</cite> [<a href="https://doi.org/10.1016/S0076-6879(09)64011-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4196316/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19903557/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?title=Methods%20in%20Enzymology&amp;author=TK%20Ritchie&amp;author=YV%20Grinkova&amp;author=TH%20Bayburt&amp;author=IG%20Denisov&amp;author=JK%20Zolnerciks&amp;publication_year=2009&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70276-bib-0055">
<cite>
Shen HH, Lithgow T, Martin L. Reconstitution of membrane proteins into model membranes: seeking better ways to retain protein activities. International Journal of Molecular Sciences. 2013;14:1589–1607.
</cite> [<a href="https://doi.org/10.3390/ijms14011589" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3565336/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23344058/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=International%20Journal%20of%20Molecular%20Sciences&amp;title=Reconstitution%20of%20membrane%20proteins%20into%20model%20membranes:%20seeking%20better%20ways%20to%20retain%20protein%20activities&amp;author=HH%20Shen&amp;author=T%20Lithgow&amp;author=L%20Martin&amp;volume=14&amp;publication_year=2013&amp;pages=1589-1607&amp;pmid=23344058&amp;doi=10.3390/ijms14011589&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70276-bib-0056">
<cite>
Shen J, Kelleher RJ 3rd. The presenilin hypothesis of Alzheimer's disease: evidence for a loss‐of‐function pathogenic mechanism. Proceedings of the National Academy of Sciences of the United States of America. 2007;104:403–409.
</cite> [<a href="https://doi.org/10.1073/pnas.0608332104" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1766397/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17197420/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proceedings%20of%20the%20National%20Academy%20of%20Sciences%20of%20the%20United%20States%20of%20America&amp;title=The%20presenilin%20hypothesis%20of%20Alzheimer's%20disease:%20evidence%20for%20a%20loss%E2%80%90of%E2%80%90function%20pathogenic%20mechanism&amp;author=J%20Shen&amp;author=RJ%20Kelleher&amp;volume=104&amp;publication_year=2007&amp;pages=403-409&amp;pmid=17197420&amp;doi=10.1073/pnas.0608332104&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70276-bib-0057">
<cite>
Sligar SG, Denisov IG. Nanodiscs: a toolkit for membrane protein science. Protein Science. 2021;30:297–315.
</cite> [<a href="https://doi.org/10.1002/pro.3994" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7784751/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33165998/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Protein%20Science&amp;title=Nanodiscs:%20a%20toolkit%20for%20membrane%20protein%20science&amp;author=SG%20Sligar&amp;author=IG%20Denisov&amp;volume=30&amp;publication_year=2021&amp;pages=297-315&amp;pmid=33165998&amp;doi=10.1002/pro.3994&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70276-bib-0058">
<cite>
Stępień P, Świątek S, Robles MYY, Markiewicz‐Mizera J, Balakrishnan D, Inaba‐Inoue S, et al. CRAFTing delivery of membrane proteins into protocells using nanodiscs. ACS Applied Materials &amp; Interfaces. 2023;15:56689–56701.
</cite> [<a href="https://doi.org/10.1021/acsami.3c11894" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10726305/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38015973/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=ACS%20Applied%20Materials%20&amp;%20Interfaces&amp;title=CRAFTing%20delivery%20of%20membrane%20proteins%20into%20protocells%20using%20nanodiscs&amp;author=P%20St%C4%99pie%C5%84&amp;author=S%20%C5%9Awi%C4%85tek&amp;author=MYY%20Robles&amp;author=J%20Markiewicz%E2%80%90Mizera&amp;author=D%20Balakrishnan&amp;volume=15&amp;publication_year=2023&amp;pages=56689-56701&amp;pmid=38015973&amp;doi=10.1021/acsami.3c11894&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70276-bib-0059">
<cite>
Studier FW. Protein production by auto‐induction in high density shaking cultures. Protein Expression and Purification. 2005;41:207–234.
</cite> [<a href="https://doi.org/10.1016/j.pep.2005.01.016" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15915565/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Protein%20Expression%20and%20Purification&amp;title=Protein%20production%20by%20auto%E2%80%90induction%20in%20high%20density%20shaking%20cultures&amp;author=FW%20Studier&amp;volume=41&amp;publication_year=2005&amp;pages=207-234&amp;pmid=15915565&amp;doi=10.1016/j.pep.2005.01.016&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70276-bib-0060">
<cite>
Takasugi N, Komai M, Kaneshiro N, Ikeda A, Kamikubo Y, Uehara T. The pursuit of the “inside” of the amyloid hypothesis‐is C99 a promising therapeutic target for Alzheimer's disease?
Cells. 2023;12:454.
</cite> [<a href="https://doi.org/10.3390/cells12030454" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9914381/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36766796/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cells&amp;title=The%20pursuit%20of%20the%20%E2%80%9Cinside%E2%80%9D%20of%20the%20amyloid%20hypothesis%E2%80%90is%20C99%20a%20promising%20therapeutic%20target%20for%20Alzheimer's%20disease?&amp;author=N%20Takasugi&amp;author=M%20Komai&amp;author=N%20Kaneshiro&amp;author=A%20Ikeda&amp;author=Y%20Kamikubo&amp;volume=12&amp;publication_year=2023&amp;pages=454&amp;pmid=36766796&amp;doi=10.3390/cells12030454&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70276-bib-0061">
<cite>
Tiefenauer L, Demarche S. Challenges in the development of functional assays of membrane proteins. Materials. 2012;5:2205–2242.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Materials&amp;title=Challenges%20in%20the%20development%20of%20functional%20assays%20of%20membrane%20proteins&amp;author=L%20Tiefenauer&amp;author=S%20Demarche&amp;volume=5&amp;publication_year=2012&amp;pages=2205-2242&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70276-bib-0062">
<cite>
Tompa P. Intrinsically unstructured proteins. Trends in Biochemical Sciences. 2002;27:527–533.
</cite> [<a href="https://doi.org/10.1016/s0968-0004(02)02169-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12368089/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Trends%20in%20Biochemical%20Sciences&amp;title=Intrinsically%20unstructured%20proteins&amp;author=P%20Tompa&amp;volume=27&amp;publication_year=2002&amp;pages=527-533&amp;pmid=12368089&amp;doi=10.1016/s0968-0004(02)02169-2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70276-bib-0063">
<cite>
Urner LH, Liko I, Pagel K, Haag R, Robinson CV. Non‐ionic hybrid detergents for protein delipidation. Biochimica et Biophysica Acta. 2022;1864:183958.
</cite> [<a href="https://doi.org/10.1016/j.bbamem.2022.183958" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35551920/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biochimica%20et%20Biophysica%20Acta&amp;title=Non%E2%80%90ionic%20hybrid%20detergents%20for%20protein%20delipidation&amp;author=LH%20Urner&amp;author=I%20Liko&amp;author=K%20Pagel&amp;author=R%20Haag&amp;author=CV%20Robinson&amp;volume=1864&amp;publication_year=2022&amp;pages=183958&amp;pmid=35551920&amp;doi=10.1016/j.bbamem.2022.183958&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70276-bib-0064">
<cite>
Vetrivel KS, Cheng H, Kim SH, Chen Y, Barnes NY, Parent AT, et al. Spatial segregation of gamma‐secretase and substrates in distinct membrane domains. The Journal of Biological Chemistry. 2005;280:25892–25900.
</cite> [<a href="https://doi.org/10.1074/jbc.M503570200" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1201532/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15886206/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=The%20Journal%20of%20Biological%20Chemistry&amp;title=Spatial%20segregation%20of%20gamma%E2%80%90secretase%20and%20substrates%20in%20distinct%20membrane%20domains&amp;author=KS%20Vetrivel&amp;author=H%20Cheng&amp;author=SH%20Kim&amp;author=Y%20Chen&amp;author=NY%20Barnes&amp;volume=280&amp;publication_year=2005&amp;pages=25892-25900&amp;pmid=15886206&amp;doi=10.1074/jbc.M503570200&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70276-bib-0065">
<cite>
Viswanath S, Dominguez L, Foster LS, Straub JE, Elber R. Extension of a protein docking algorithm to membranes and applications to amyloid precursor protein dimerization. Proteins. 2015;83:2170–2185.
</cite> [<a href="https://doi.org/10.1002/prot.24934" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4715591/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26404856/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proteins&amp;title=Extension%20of%20a%20protein%20docking%20algorithm%20to%20membranes%20and%20applications%20to%20amyloid%20precursor%20protein%20dimerization&amp;author=S%20Viswanath&amp;author=L%20Dominguez&amp;author=LS%20Foster&amp;author=JE%20Straub&amp;author=R%20Elber&amp;volume=83&amp;publication_year=2015&amp;pages=2170-2185&amp;pmid=26404856&amp;doi=10.1002/prot.24934&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70276-bib-0066">
<cite>
Walkenhorst WF, Merzlyakov M, Hristova K, Wimley WC. Polar residues in transmembrane helices can decrease electrophoretic mobility in polyacrylamide gels without causing helix dimerization. Biochimica et Biophysica Acta. 2009;1788:1321–1331.
</cite> [<a href="https://doi.org/10.1016/j.bbamem.2009.02.017" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2935849/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19265670/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biochimica%20et%20Biophysica%20Acta&amp;title=Polar%20residues%20in%20transmembrane%20helices%20can%20decrease%20electrophoretic%20mobility%20in%20polyacrylamide%20gels%20without%20causing%20helix%20dimerization&amp;author=WF%20Walkenhorst&amp;author=M%20Merzlyakov&amp;author=K%20Hristova&amp;author=WC%20Wimley&amp;volume=1788&amp;publication_year=2009&amp;pages=1321-1331&amp;pmid=19265670&amp;doi=10.1016/j.bbamem.2009.02.017&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70276-bib-0067">
<cite>
Wisedchaisri G, Gamal El‐Din TM, Powell NM, Zheng N, Catterall WA. Structural basis for severe pain caused by mutations in the voltage sensors of sodium channel NaV1.7. The Journal of General Physiology. 2023;155:2219624120.</cite> [<a href="https://doi.org/10.1085/jgp.202313450" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10616507/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37903281/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=The%20Journal%20of%20General%20Physiology&amp;title=Structural%20basis%20for%20severe%20pain%20caused%20by%20mutations%20in%20the%20voltage%20sensors%20of%20sodium%20channel%20NaV1.7&amp;author=G%20Wisedchaisri&amp;author=TM%20Gamal%20El%E2%80%90Din&amp;author=NM%20Powell&amp;author=N%20Zheng&amp;author=WA%20Catterall&amp;volume=155&amp;publication_year=2023&amp;pages=2219624120&amp;pmid=37903281&amp;doi=10.1085/jgp.202313450&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70276-bib-0068">
<cite>
Wu W, Zheng J, Wang R, Wang Y. Ion channels regulate energy homeostasis and the progression of metabolic disorders: novel mechanisms and pharmacology of their modulators. Biochemical Pharmacology. 2023;218:115863.
</cite> [<a href="https://doi.org/10.1016/j.bcp.2023.115863" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37863328/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biochemical%20Pharmacology&amp;title=Ion%20channels%20regulate%20energy%20homeostasis%20and%20the%20progression%20of%20metabolic%20disorders:%20novel%20mechanisms%20and%20pharmacology%20of%20their%20modulators&amp;author=W%20Wu&amp;author=J%20Zheng&amp;author=R%20Wang&amp;author=Y%20Wang&amp;volume=218&amp;publication_year=2023&amp;pages=115863&amp;pmid=37863328&amp;doi=10.1016/j.bcp.2023.115863&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70276-bib-0069">
<cite>
Yang D, Zhou Q, Labroska V, Qin S, Darbalaei S, Wu Y, et al. G protein‐coupled receptors: structure‐ and function‐based drug discovery. Signal Transduction and Targeted Therapy. 2021;6:7.
</cite> [<a href="https://doi.org/10.1038/s41392-020-00435-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7790836/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33414387/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Signal%20Transduction%20and%20Targeted%20Therapy&amp;title=G%20protein%E2%80%90coupled%20receptors:%20structure%E2%80%90%20and%20function%E2%80%90based%20drug%20discovery&amp;author=D%20Yang&amp;author=Q%20Zhou&amp;author=V%20Labroska&amp;author=S%20Qin&amp;author=S%20Darbalaei&amp;volume=6&amp;publication_year=2021&amp;pages=7&amp;pmid=33414387&amp;doi=10.1038/s41392-020-00435-w&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70276-bib-0070">
<cite>
Yang Z, Wang C, Zhou Q, An J, Hildebrandt E, Aleksandrov LA, et al. Membrane protein stability can be compromised by detergent interactions with the extramembranous soluble domains. Protein Science. 2014;23:769–789.
</cite> [<a href="https://doi.org/10.1002/pro.2460" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4093953/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24652590/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Protein%20Science&amp;title=Membrane%20protein%20stability%20can%20be%20compromised%20by%20detergent%20interactions%20with%20the%20extramembranous%20soluble%20domains&amp;author=Z%20Yang&amp;author=C%20Wang&amp;author=Q%20Zhou&amp;author=J%20An&amp;author=E%20Hildebrandt&amp;volume=23&amp;publication_year=2014&amp;pages=769-789&amp;pmid=24652590&amp;doi=10.1002/pro.2460&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70276-bib-0071">
<cite>
Yeung YG, Stanley ER. Rapid detergent removal from peptide samples with ethyl acetate for mass spectrometry analysis. Current Protocols in Protein Science. 2010;59: 16.12.11–16.12.15.</cite> [<a href="https://doi.org/10.1002/0471140864.ps1612s59" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2852680/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20155730/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Current%20Protocols%20in%20Protein%20Science&amp;title=Rapid%20detergent%20removal%20from%20peptide%20samples%20with%20ethyl%20acetate%20for%20mass%20spectrometry%20analysis&amp;author=YG%20Yeung&amp;author=ER%20Stanley&amp;volume=59&amp;publication_year=2010&amp;pmid=20155730&amp;doi=10.1002/0471140864.ps1612s59&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70276-bib-0072">
<cite>
Yu X, Ton AN, Niu Z, Morales BM, Chen J, Braz J, et al. ACVR1‐activating mutation causes neuropathic pain and sensory neuron hyperexcitability in humans. Pain. 2023;164:43–58.
</cite> [<a href="https://doi.org/10.1097/j.pain.0000000000002656" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9582048/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35442931/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Pain&amp;title=ACVR1%E2%80%90activating%20mutation%20causes%20neuropathic%20pain%20and%20sensory%20neuron%20hyperexcitability%20in%20humans&amp;author=X%20Yu&amp;author=AN%20Ton&amp;author=Z%20Niu&amp;author=BM%20Morales&amp;author=J%20Chen&amp;volume=164&amp;publication_year=2023&amp;pages=43-58&amp;pmid=35442931&amp;doi=10.1097/j.pain.0000000000002656&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70276-bib-0073">
<cite>
Zhang Y, MacKinnon R. Higher‐order transient structures and the principle of dynamic connectivity in membrane signaling. Proceedings of the National Academy of Sciences of the United States of America. 2025;122:2421280121.</cite> [<a href="https://doi.org/10.1073/pnas.2421280121" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11725812/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39739805/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proceedings%20of%20the%20National%20Academy%20of%20Sciences%20of%20the%20United%20States%20of%20America&amp;title=Higher%E2%80%90order%20transient%20structures%20and%20the%20principle%20of%20dynamic%20connectivity%20in%20membrane%20signaling&amp;author=Y%20Zhang&amp;author=R%20MacKinnon&amp;volume=122&amp;publication_year=2025&amp;pages=2421280121&amp;pmid=39739805&amp;doi=10.1073/pnas.2421280121&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70276-bib-0074">
<cite>
Zhang Y, Mazal H, Mandala VS, Pérez‐Mitta G, Sondoghdar V, Haselwandter CA, et al. Higher‐order transient membrane protein structures. Proceedings of the National Academy of Sciences of the United States of America. 2025;122:2421275121.</cite> [<a href="https://doi.org/10.1073/pnas.2421275121" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11725870/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39739811/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proceedings%20of%20the%20National%20Academy%20of%20Sciences%20of%20the%20United%20States%20of%20America&amp;title=Higher%E2%80%90order%20transient%20membrane%20protein%20structures&amp;author=Y%20Zhang&amp;author=H%20Mazal&amp;author=VS%20Mandala&amp;author=G%20P%C3%A9rez%E2%80%90Mitta&amp;author=V%20Sondoghdar&amp;volume=122&amp;publication_year=2025&amp;pages=2421275121&amp;pmid=39739811&amp;doi=10.1073/pnas.2421275121&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70276-bib-0075">
<cite>
Zhou Z, Chen Z, Li Y, Mao X, Chen J, Zhou X, et al. Advances in solubilization and stabilization techniques for structural and functional studies of membrane proteins. PeerJ. 2025;13:19211.</cite> [<a href="https://doi.org/10.7717/peerj.19211" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11974516/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40196297/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PeerJ&amp;title=Advances%20in%20solubilization%20and%20stabilization%20techniques%20for%20structural%20and%20functional%20studies%20of%20membrane%20proteins&amp;author=Z%20Zhou&amp;author=Z%20Chen&amp;author=Y%20Li&amp;author=X%20Mao&amp;author=J%20Chen&amp;volume=13&amp;publication_year=2025&amp;pages=19211&amp;pmid=40196297&amp;doi=10.7717/peerj.19211&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70276-bib-0076">
<cite>
Zilioli A, Misirocchi F, Pancaldi B, Mutti C, Ganazzoli C, Florindo I, et al. A novel presenilin‐1 mutation (Leu226Val) in early onset Alzheimer's disease with parkinsonism. The Canadian Journal of Neurological Sciences. 2024;51:703–706.
</cite> [<a href="https://doi.org/10.1017/cjn.2023.285" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37694526/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=The%20Canadian%20Journal%20of%20Neurological%20Sciences&amp;title=A%20novel%20presenilin%E2%80%901%20mutation%20(Leu226Val)%20in%20early%20onset%20Alzheimer's%20disease%20with%20parkinsonism&amp;author=A%20Zilioli&amp;author=F%20Misirocchi&amp;author=B%20Pancaldi&amp;author=C%20Mutti&amp;author=C%20Ganazzoli&amp;volume=51&amp;publication_year=2024&amp;pages=703-706&amp;pmid=37694526&amp;doi=10.1017/cjn.2023.285&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ol></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="caption p"><p>
<strong>DATA S1.</strong> Supporting Information.</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12363408/bin/PRO-34-e70276-s001.pdf" data-ga-action="click_feat_suppl" class="usa-link">PRO-34-e70276-s001.pdf</a><sup> (2.4MB, pdf) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>The data that support the findings of this study are available from the corresponding author upon reasonable request.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Protein Science : A Publication of the Protein Society are provided here courtesy of <strong>The Protein Society</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1002/pro.70276"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/PRO-34-e70276.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (2.1 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12363408/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12363408/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12363408%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12363408/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12363408/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12363408/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40828514/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12363408/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40828514/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12363408/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12363408/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="JReoKyx2P3Onz37AzP66ERJGLMPpwOVFaZIgeaVNp8M7rWipPMKTDSiEMHDxbMHS">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
