
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Lignin Unlocks Stealth Carbon Sinks in Cold Seeps via Microbial Enzymatic Gatekeeping - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="152539378AF1F963053937004613019E.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="research">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12377529/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Research">
<meta name="citation_title" content="Lignin Unlocks Stealth Carbon Sinks in Cold Seeps via Microbial Enzymatic Gatekeeping">
<meta name="citation_author" content="Jialing Li">
<meta name="citation_author_institution" content="Southern Marine Science and Engineering Guangdong Laboratory (Guangzhou), Guangzhou 511458, China.">
<meta name="citation_author" content="Jingchun Feng">
<meta name="citation_author_institution" content="Southern Marine Science and Engineering Guangdong Laboratory (Guangzhou), Guangzhou 511458, China.">
<meta name="citation_author_institution" content="School of Ecology, Environmental, and Resources, Guangdong University of Technology, Guangzhou 510006, China.">
<meta name="citation_author" content="Pandeng Wang">
<meta name="citation_author_institution" content="School of Pharmacy, Shenzhen University Medical School, Shenzhen University, Shenzhen 518055, China.">
<meta name="citation_author" content="Mengzhuo Zhu">
<meta name="citation_author_institution" content="School of Ecology, Environmental, and Resources, Guangdong University of Technology, Guangzhou 510006, China.">
<meta name="citation_author" content="Yongji Huang">
<meta name="citation_author_institution" content="South China Sea Institute of Oceanology, Chinese Academy of Sciences, Guangzhou 510301, China.">
<meta name="citation_author" content="Ying Wu">
<meta name="citation_author_institution" content="State Key Laboratory of Estuarine and Coastal Research East China Normal University, Shanghai 200662, China.">
<meta name="citation_author" content="Junning Fan">
<meta name="citation_author_institution" content="State Key Laboratory of Estuarine and Coastal Research East China Normal University, Shanghai 200662, China.">
<meta name="citation_author" content="Junlin Hu">
<meta name="citation_author_institution" content="Southern Marine Science and Engineering Guangdong Laboratory (Guangzhou), Guangzhou 511458, China.">
<meta name="citation_author" content="Xiyang Dong">
<meta name="citation_author_institution" content="Key Laboratory of Marine Genetic Resources, Third Institute of Oceanography, Ministry of Natural Resources, Xiamen 361005, China.">
<meta name="citation_author" content="Yingli Zhou">
<meta name="citation_author_institution" content="Southern Marine Science and Engineering Guangdong Laboratory (Guangzhou), Guangzhou 511458, China.">
<meta name="citation_author" content="Xuanyu Tao">
<meta name="citation_author_institution" content="Southern Marine Science and Engineering Guangdong Laboratory (Guangzhou), Guangzhou 511458, China.">
<meta name="citation_author" content="Si Zhang">
<meta name="citation_author_institution" content="Southern Marine Science and Engineering Guangdong Laboratory (Guangzhou), Guangzhou 511458, China.">
<meta name="citation_publication_date" content="2025 Aug 25">
<meta name="citation_volume" content="8">
<meta name="citation_firstpage" content="0848">
<meta name="citation_doi" content="10.34133/research.0848">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12377529/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12377529/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12377529/pdf/research.0848.pdf">
<meta name="description" content="Cold seep ecosystems serve as critical hubs in marine carbon cycling through methane emissions and organic matter processing. While terrestrial lignin constitutes a major fraction of persistent organic carbon in cold seep sediments, its microbial ...">
<meta name="og:title" content="Lignin Unlocks Stealth Carbon Sinks in Cold Seeps via Microbial Enzymatic Gatekeeping">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Cold seep ecosystems serve as critical hubs in marine carbon cycling through methane emissions and organic matter processing. While terrestrial lignin constitutes a major fraction of persistent organic carbon in cold seep sediments, its microbial ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12377529/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12377529">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.34133/research.0848"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/research.0848.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12377529%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12377529/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12377529/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12377529/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-research.png" alt="Research logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Research" title="Link to Research" shape="default" href="https://spj.sciencemag.org/research/" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Research (Wash D C)</button></div>. 2025 Aug 25;8:0848. doi: <a href="https://doi.org/10.34133/research.0848" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.34133/research.0848</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Research%20(Wash%20D%20C)%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Research%20(Wash%20D%20C)%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Research%20(Wash%20D%20C)%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Research%20(Wash%20D%20C)%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Lignin Unlocks Stealth Carbon Sinks in Cold Seeps via Microbial Enzymatic Gatekeeping</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Li%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Jialing Li</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Jialing Li</span></h3>
<div class="p">
<sup><sup>1</sup></sup>Southern Marine Science and Engineering Guangdong Laboratory (Guangzhou), Guangzhou 511458, China.</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Li%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Jialing Li</span></a>
</div>
</div>
<sup>1,</sup><sup>†</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Feng%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Jingchun Feng</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Jingchun Feng</span></h3>
<div class="p">
<sup><sup>1</sup></sup>Southern Marine Science and Engineering Guangdong Laboratory (Guangzhou), Guangzhou 511458, China.</div>
<div class="p">
<sup><sup>2</sup></sup>School of Ecology, Environmental, and Resources, 
Guangdong University of Technology, Guangzhou 510006, China.</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Feng%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Jingchun Feng</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>*</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20P%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Pandeng Wang</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Pandeng Wang</span></h3>
<div class="p">
<sup><sup>3</sup></sup>School of Pharmacy, 
Shenzhen University
Medical School, Shenzhen University, Shenzhen 518055, China.</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20P%22%5BAuthor%5D" class="usa-link"><span class="name western">Pandeng Wang</span></a>
</div>
</div>
<sup>3,</sup><sup>†</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhu%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Mengzhuo Zhu</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Mengzhuo Zhu</span></h3>
<div class="p">
<sup><sup>2</sup></sup>School of Ecology, Environmental, and Resources, 
Guangdong University of Technology, Guangzhou 510006, China.</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhu%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Mengzhuo Zhu</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Huang%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Yongji Huang</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Yongji Huang</span></h3>
<div class="p">
<sup><sup>4</sup></sup>South China Sea Institute of Oceanology, 
Chinese Academy of Sciences, Guangzhou 510301, China.</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Huang%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yongji Huang</span></a>
</div>
</div>
<sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wu%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Ying Wu</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Ying Wu</span></h3>
<div class="p">
<sup><sup>5</sup></sup>
State Key Laboratory of Estuarine and Coastal Research East China Normal University, Shanghai 200662, China.</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wu%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Ying Wu</span></a>
</div>
</div>
<sup>5</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Fan%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Junning Fan</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Junning Fan</span></h3>
<div class="p">
<sup><sup>5</sup></sup>
State Key Laboratory of Estuarine and Coastal Research East China Normal University, Shanghai 200662, China.</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Fan%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Junning Fan</span></a>
</div>
</div>
<sup>5</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hu%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Junlin Hu</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Junlin Hu</span></h3>
<div class="p">
<sup><sup>1</sup></sup>Southern Marine Science and Engineering Guangdong Laboratory (Guangzhou), Guangzhou 511458, China.</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hu%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Junlin Hu</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Dong%20X%22%5BAuthor%5D" class="usa-link" aria-describedby="id9"><span class="name western">Xiyang Dong</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">Xiyang Dong</span></h3>
<div class="p">
<sup><sup>6</sup></sup>Key Laboratory of Marine Genetic Resources, 
Third Institute of Oceanography, Ministry of Natural Resources, Xiamen 361005, China.</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Dong%20X%22%5BAuthor%5D" class="usa-link"><span class="name western">Xiyang Dong</span></a>
</div>
</div>
<sup>6</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhou%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id10"><span class="name western">Yingli Zhou</span></a><div hidden="hidden" id="id10">
<h3><span class="name western">Yingli Zhou</span></h3>
<div class="p">
<sup><sup>1</sup></sup>Southern Marine Science and Engineering Guangdong Laboratory (Guangzhou), Guangzhou 511458, China.</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhou%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yingli Zhou</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tao%20X%22%5BAuthor%5D" class="usa-link" aria-describedby="id11"><span class="name western">Xuanyu Tao</span></a><div hidden="hidden" id="id11">
<h3><span class="name western">Xuanyu Tao</span></h3>
<div class="p">
<sup><sup>1</sup></sup>Southern Marine Science and Engineering Guangdong Laboratory (Guangzhou), Guangzhou 511458, China.</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tao%20X%22%5BAuthor%5D" class="usa-link"><span class="name western">Xuanyu Tao</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id12"><span class="name western">Si Zhang</span></a><div hidden="hidden" id="id12">
<h3><span class="name western">Si Zhang</span></h3>
<div class="p">
<sup><sup>1</sup></sup>Southern Marine Science and Engineering Guangdong Laboratory (Guangzhou), Guangzhou 511458, China.</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Si Zhang</span></a>
</div>
</div>
<sup>1,</sup><sup>*</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="aff1">
<sup><sup>1</sup></sup>Southern Marine Science and Engineering Guangdong Laboratory (Guangzhou), Guangzhou 511458, China.</div>
<div id="aff2">
<sup><sup>2</sup></sup>School of Ecology, Environmental, and Resources, 
Guangdong University of Technology, Guangzhou 510006, China.</div>
<div id="aff3">
<sup><sup>3</sup></sup>School of Pharmacy, 
Shenzhen University
Medical School, Shenzhen University, Shenzhen 518055, China.</div>
<div id="aff4">
<sup><sup>4</sup></sup>South China Sea Institute of Oceanology, 
Chinese Academy of Sciences, Guangzhou 510301, China.</div>
<div id="aff5">
<sup><sup>5</sup></sup>
State Key Laboratory of Estuarine and Coastal Research East China Normal University, Shanghai 200662, China.</div>
<div id="aff6">
<sup><sup>6</sup></sup>Key Laboratory of Marine Genetic Resources, 
Third Institute of Oceanography, Ministry of Natural Resources, Xiamen 361005, China.</div>
<div class="author-notes p">
<div class="fn" id="corr1">
<sup>*</sup><p class="display-inline">Address correspondence to: <span>zhsimd@scsio.ac.cn</span> (S.Z.); <span>fengjc@gdut.edu.cn</span> (J.F.)</p>
</div>
<div class="fn" id="afn1">
<sup>†</sup><p class="display-inline">These authors contributed equally to this work.</p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 May 24; Revised 2025 Jul 19; Accepted 2025 Jul 29; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>Copyright © 2025 Jialing Li et al.</div>
<p>Exclusive licensee Science and Technology Review Publishing House. No claim to original U.S. Government Works. Distributed under a <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Creative Commons Attribution License (CC BY 4.0)</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12377529</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>Abstract</h2>
<p>Cold seep ecosystems serve as critical hubs in marine carbon cycling through methane emissions and organic matter processing. While terrestrial lignin constitutes a major fraction of persistent organic carbon in cold seep sediments, its microbial transformation pathways in deep-sea cold seep environments remain unexplored. Here, we present the first comprehensive analysis of lignin distribution across sediment horizons at the Haima cold seep, coupled with a multi-omics investigation of microbial lignin metabolism. Laboratory enrichment of sediment communities employing lignin as the exclusive carbon substrate revealed substantial microbial community restructuring dominated by <em>Burkholderiales</em>, <em>Pseudomonadales</em>, and <em>Rhizobiales</em> lineages. Integrated omics resolved 2-tiered metabolic cascades: (a) enzymatic depolymerization via dyP-type peroxidases and LigEFG-mediated β-aryl ether cleavage, targeting syringyl and diarylpropane subunits; (b) funneling of aromatic intermediates through 4,5-/3,4-PDOG (protocatechuate dioxygenase) pathways into central carbon metabolism. Although direct methanogenesis was undetected, methylotrophic potential was evidenced through methane cycle gene expression patterns by lignin decomposers. Phylogenetic surveys further demonstrated the global prevalence of lignin decomposers across 12 major cold seep systems. These decomposers showed marked divergence in enzymatic repertoires compared to degraders from other ecosystems. Our findings establish 3 paradigm shifts: (a) The turnover rates of terrestrial organic carbon are likely underestimated in deep-sea ecosystems; (b) microbial consortia employ combinatorial enzymatic strategies distinct from terrestrial decomposition regimes; (c) methyl shunting from lignin breakdown primes methanogenic precursors, revealing cryptic linkages between refractory carbon cycling and greenhouse gas reservoirs.</p></section><section id="sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p>Deep-sea cold seeps function as biogeochemical reactors, orchestrating the mineralization and sequestration of organic carbon through tightly coupled microbial processes that shape regional carbon budgets [<a href="#B1" class="usa-link" aria-describedby="B1">1</a>–<a href="#B3" class="usa-link" aria-describedby="B3">3</a>]. While CH<sub>4</sub> and CO<sub>2</sub> represent the terminal phase of carbon mineralization [<a href="#B4" class="usa-link" aria-describedby="B4">4</a>,<a href="#B5" class="usa-link" aria-describedby="B5">5</a>], the upstream enzymatic machinery governing macromolecular organic carbon breakdown—particularly the catabolism of aromatic polymers—remains a critical blind spot in current carbon flux models [<a href="#B6" class="usa-link" aria-describedby="B6">6</a>].</p>
<p>Vertical stratification analyses reveal substantial heterogeneity in organic carbon composition across sediment horizons [<a href="#B7" class="usa-link" aria-describedby="B7">7</a>–<a href="#B9" class="usa-link" aria-describedby="B9">9</a>], reflecting microbial niche partitioning and substrate specialization. The catabolism of organic carbon through microbial activity constitutes an intricate biochemical network, wherein polymeric macromolecules first undergo enzymatic depolymerization to yield monomeric building blocks, which are then channeled into acidogenesis pathways producing characteristic metabolites including hydrogen gas, short-chain fatty acids (e.g., acetate, propionate, and butyrate), and alcohol derivatives. These intermediates are ultimately metabolized into CH<sub>4</sub> and CO<sub>2</sub> [<a href="#B10" class="usa-link" aria-describedby="B10">10</a>]. Current paradigms disproportionately emphasize terminal methane cycling, neglecting the rate-limiting initial steps of high-molecular-weight organic carbon processing [<a href="#B11" class="usa-link" aria-describedby="B11">11</a>]. This knowledge gap becomes particularly acute when considering lignin as the most abundant terrestrial organic carbon, which accumulates in shelf-margin sediments through terrestrial runoff [<a href="#B12" class="usa-link" aria-describedby="B12">12</a>,<a href="#B13" class="usa-link" aria-describedby="B13">13</a>].</p>
<p>Lignin is composed of hydroxyphenylpropane monomers linked through various ether and carbon–carbon bonds [<a href="#B14" class="usa-link" aria-describedby="B14">14</a>–<a href="#B16" class="usa-link" aria-describedby="B16">16</a>], resulting from dehydrogenative polymerization [<a href="#B17" class="usa-link" aria-describedby="B17">17</a>]. The initial step in lignin biodegradation involves the action of extracellular lignin-degrading enzymes that catalyze the depolymerization of lignin [<a href="#B18" class="usa-link" aria-describedby="B18">18</a>–<a href="#B20" class="usa-link" aria-describedby="B20">20</a>]. Microorganisms utilize peroxidases such as laccase and dye-decolorizing peroxidase (DyP) [<a href="#B21" class="usa-link" aria-describedby="B21">21</a>,<a href="#B22" class="usa-link" aria-describedby="B22">22</a>]; coupled with supplementary enzyme systems, these biocatalysts orchestrate regioselective transformations: cleavage of intermonomeric β-O-4 bonds, separation of alkyl groups from aromatic moieties, methyl group elimination, and disruption of cyclic structures, thereby generating fragmented aromatic molecules. But lignin’s operational dynamics in oxic–suboxic transition zones remain enigmatic in cold seeps.</p>
<p>Previous efforts primarily characterized isolated enzyme systems [<a href="#B23" class="usa-link" aria-describedby="B23">23</a>–<a href="#B26" class="usa-link" aria-describedby="B26">26</a>], failing to capture how microbial community orchestrates the lignin degradation cascade (macromolecular depolymerization → aromatic monomer conversion → methanogenic precursor formation). This disconnect obscures critical linkages between terrestrial carbon export and marine methane reservoirs. Through laboratory enrichments and in situ analyses, we address 2 fundamental questions: (a) What enzymatic systems enable extracellular lignin modification in suboxic conditions? (b) How do degradation intermediates interface with methane-genic metabolic networks? Here, we employ a tripartite omics framework (metagenomics, metatranscriptomics, and metaproteomics) to dissect lignin transformation pathways in South China Sea cold seep surface sediments.</p></section><section id="sec2"><h2 class="pmc_sec_title">Results</h2>
<section id="sec3"><h3 class="pmc_sec_title">Lignin parameter reveals sustained allochthonous input in Haima cold seep sediments</h3>
<p>Quantitative lignin analysis across the Haima cold seep (0 to 25 cm) revealed substantial depleted lignin concentrations [vanillyl (V) phenols: 4.53 to 14.46 μg/g; syringyl (S) phenols: 0.09 to 4.33 μg/g; cinnamyl (C) phenols: 4.46 to 11.47 μg/g] compared to nearshore systems [<a href="#B27" class="usa-link" aria-describedby="B27">27</a>–<a href="#B29" class="usa-link" aria-describedby="B29">29</a>], establishing cold seeps as distinct lignin reservoirs. Molecular signature analysis (Table <a href="#supplementary-material-1" class="usa-link">S2</a>) identified nonwoody angiosperm sources as the predominant lignin origin (&gt;0 of S/V and C/V ratios), contrasting with terrestrial-dominated coastal inputs. Vertical profiling demonstrated pronounced oxidative modification [P/(S +V): 0.55 to 2.78] across 5 sediment horizons, with the uppermost layer of HOV3 exhibiting relatively lower oxidative degradation and a higher presence of fresh tissues [(Ad/Al) v &lt; 0.3]. This indicates the continuous input and accumulation of lignin in the study area.</p></section><section id="sec4"><h3 class="pmc_sec_title">Temporal succession of lignin-responsive microbes reveals niche partitioning in Haima cold seep sediments</h3>
<p>Following the enrichment of surface sediment divided into 5 layers using lignin as the exclusive carbon substrate for enrichment (Fig. <a href="#F1" class="usa-link">1</a>A), the mean residual solid lignin concentration across all microcosms ranged from 2.8 to 4.1 g at day 90. Notably, layer-specific differences in residual lignin were observed: Mean degradation rates were 34.0% ± 9.5% for layer 1, 40.0% ± 4.0% for layer 2, 22.0% ± 4.0% for layer 3, 26.0% ± 4.0% for layer 4, and 34.0% ± 4.0% for layer 5. Furthermore, distinct shifts in microbial communities were observed across a continuous time gradient. The PCoA (principal coordinates analysis) ordination based on Bray–Curtis distances (Fig. <a href="#F1" class="usa-link">1</a>B) revealed progressive temporal divergence in community structure during enrichment, with trajectories shifting toward assemblages enriched with lignin-degrading taxa (e.g., <em>Sphingomonadales</em> and <em>Burkholderiales</em>), likely driven by functional adaptation to lignin utilization as the sole carbon source.</p>
<figure class="fig xbox font-sm" id="F1"><h4 class="obj_head">Fig. 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1abc/12377529/ff8817e347ad/research.0848.fig.001.jpg" loading="lazy" height="445" width="660" alt="Fig. 1."></p>
<div class="p text-right font-secondary"><a href="figure/F1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>(A) Experimental procedure diagram. The sediments from the Haima cold seep were divided into 5 layers and enriched each with lignin, sampling at 7 intervals; after sampling, a portion of the samples was transferred to hermetically sealed bottles for gas testing of methane content on alternate days. (B) PCoA analysis based on Bray–Curtis distances illustrating the dissimilarities among various samples. (C) Distribution of microbial communities at the order level based on average abundance in different layers (Others &lt;1%). Five layers served as replicates; different time points were analyzed separately in (B) and (C).</p></figcaption></figure><p>The number of core phyla decreased over the course of enrichment (Fig. <a href="#supplementary-material-1" class="usa-link">S1</a>), with microbial communities progressively converging toward core groups potentially capable of lignin utilization. With increasing incubation time, the dominant shared taxa were primarily affiliated with <em>Gammaproteobacteria</em>, <em>Alphaproteobacteria</em>, and <em>Bacteroidia</em> (Fig. <a href="#supplementary-material-1" class="usa-link">S1</a>). Notably, orders such as <em>Burkholderiales</em>, <em>Pseudomonadales</em>, <em>Parvibaculales</em>, and <em>Rhizobiales</em> exhibited significant dominance in average relative abundance [Fig. <a href="#F1" class="usa-link">1</a>C; analysis of variance (ANOVA), <em>P</em> &lt; 0.05], and some of these were groups previously reported to be involved in lignin degradation [<a href="#B30" class="usa-link" aria-describedby="B30">30</a>–<a href="#B32" class="usa-link" aria-describedby="B32">32</a>]. Cluster analysis indicated that the variation in microbial communities along the time gradient was more pronounced than the differences between communities in different layers based on Bray–Curtis distances (Fig. <a href="#supplementary-material-1" class="usa-link">S1</a>). This could be attributed to the increased abundance of potential lignin decomposers.</p></section><section id="sec5"><h3 class="pmc_sec_title">Lignin decomposers in Haima cold seep sediments</h3>
<p>Our analysis revealed dynamic lignin degradation patterns in cold seep sediments (Fig. <a href="#F1" class="usa-link">1</a> and Fig.<a href="#supplementary-material-1" class="usa-link">S2</a>). While quantitative lignin concentration data were excluded due to evaporation-induced volume variations in shake flask experiments, microbial taxonomic profiling identified key lignin-degrading consortia through functional gene analysis. Phylogenetic reconstruction based on 16<em>S</em> ribosomal RNA (rRNA) gene amplicons and metagenome-assembled genomes (MAGs) demonstrated conserved taxonomic patterns of lignin-degrading capacity across 21 bacterial orders, with <em>Parvibaculales</em>, <em>Burkholderiales</em>, <em>Pseudomonadales</em>, and <em>Rhizobiales</em> emerging as dominant degraders (Fig. <a href="#F2" class="usa-link">2</a>A and B).</p>
<figure class="fig xbox font-sm" id="F2"><h4 class="obj_head">Fig. 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1abc/12377529/9d49b571c4c3/research.0848.fig.002.jpg" loading="lazy" height="540" width="660" alt="Fig. 2."></p>
<div class="p text-right font-secondary"><a href="figure/F2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Bacterial communities involved in lignin decomposition. (A) Relative abundance of lignin degradation genes. (B) Transcript abundance of lignin degradation genes. Bars represent the relative abundance of individual decomposing genes. The communities presented here were annotated based on contigs from metagenomes and metatranscriptomes that contain lignin metabolism genes, showing the degradation groups at the order level for microbial community.</p></figcaption></figure><p>We observed a degradation change in lignin (Fig. <a href="#supplementary-material-1" class="usa-link">S2</a>). Due to the evaporation of the enriched liquid within the shake flasks, which resulted in significant differences in the liquid volumes inside the flasks, we did not present the lignin concentration. We analyzed microbial taxa possessing genetic components involved in metabolizing lignin and its low-molecular-weight derivatives. This result was similar to that of taxa resulting from 16<em>S</em> rRNA gene amplicon. At the order level, decomposers were primarily distributed across 21 orders, with <em>Parvibaculales</em>, <em>Burkholderiales</em>, <em>Pseudomonadales</em>, and <em>Rhizobiales</em> being the predominant degrading groups (Fig. <a href="#F2" class="usa-link">2</a>A and B). The order <em>Parvibaculales</em> contains genes for lignin-decomposing DyPs, but due to their low abundance, high-quality MAGs were not obtained in this order; the order <em>Oceanibaculales</em> contains genes for glutathione-dependent β-etherases (LigEFG), and the MAGs (bin2, bin24, bin25, and bin29) suggest that <em>Oceanibaculum nanhaiense</em> plays a role in lignin degradation at the macromolecular level in the environment (Table <a href="#supplementary-material-1" class="usa-link">S3</a>).</p>
<p>The order <em>Burkholderiales</em> primarily participated in the metabolism of syringyl lignin fragments and the 3,4-PDOG pathway. Concurrently, we recovered 4 high-quality bins (bin15, bin22, bin3, and bin60) from the <em>Burkholderiales</em> order. These bins were highly abundant across the 5-layer samples and belonged to the genera <em>Nitrogeniibacter</em> and <em>Pusillimonas</em>, both of which contained genes from these 2 decomposition pathways (Table <a href="#supplementary-material-1" class="usa-link">S3</a>). The order <em>Pseudomonadales</em> participated in various steps of lignin fragment decomposition, such as the ferulic acid pathway (FerAB/B<sub>2</sub>) and the 3,4-PDOG pathway, and was also a highly abundant and metabolically active group. We assembled a total of 23 bins belonging to the <em>Pseudomonadales</em> order, of which 17 bins contained genes associated with lignin and its degradation, distributed across 8 different genera.</p>
<p>These microorganisms (<em>Parvibaculales</em>, <em>Burkholderiales</em>, <em>Pseudomonadales</em>, and <em>Rhizobiales</em>) constituted a relatively high proportion in the South China Sea cold seep ecosystem [<a href="#B1" class="usa-link" aria-describedby="B1">1</a>,<a href="#B13" class="usa-link" aria-describedby="B13">13</a>,<a href="#B33" class="usa-link" aria-describedby="B33">33</a>]. The diverse lignin degraders among them possessed significantly different degradation genes (Fig. <a href="#supplementary-material-1" class="usa-link">S3</a>). These microbial groups serve as pivotal contributors to degradation of lignin macromolecules within the cold seep ecosystem.</p></section><section id="sec6"><h3 class="pmc_sec_title">Lignin decomposition pathways in Haima cold seep sediments</h3>
<p>Reconstruction of lignin and aromatic fragment degradation pathways revealed stratified enzymatic strategies in cold seep sediments (Fig. <a href="#F3" class="usa-link">3</a>). Genes encoding DyPs and glutathione-dependent β-etherases (LigEFG) were ubiquitously detected across all 5 sediment layers, implicating these enzymes as core mediators of lignin depolymerization. Notably, canonical pathways dependent on multicopper oxidases or Cα-dehydrogenases (LACs) were absent, suggesting niche-specific adaptation of oxidative machinery.</p>
<figure class="fig xbox font-sm" id="F3"><h4 class="obj_head">Fig. 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1abc/12377529/673c316d8b79/research.0848.fig.003.jpg" loading="lazy" height="660" width="638" alt="Fig. 3."></p>
<div class="p text-right font-secondary"><a href="figure/F3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>(A) Major degradation pathways of lignin macromolecules in cold seep sediments, with degraders presented at the species level through assembled genomes; key enzymes involved in the degradation process were listed, and Fe<sup>3+</sup> and O<sub>2</sub> play roles in electron transfer during lignin degradation. (B) Ring cleavage pathways of lignin-derived aromatic compounds in cold seep sediments. (C) Abundance distribution of these lignin degradation genes in metagenomes and metatranscriptomes, with the <em>x</em> axis indicating gene names and the <em>y</em> axis representing different sediment layers. Superscript circles represent assigned step numbers for different lignin metabolic pathways. All corresponding gene sources, taxa, functional annotations, and additional details for these numbered steps are comprehensively documented in Table <a href="#supplementary-material-1" class="usa-link">S1</a>.</p></figcaption></figure><p>Beyond these depolymerases, multi-omics analysis profiling identified auxiliary oxidoreductases—including catalase, quinone oxidoreductase, aryl alcohol oxidase, and pyranose oxidase—acting as redox partners (Fig. <a href="#supplementary-material-1" class="usa-link">S4</a> and Table <a href="#supplementary-material-1" class="usa-link">S4</a>) [<a href="#B34" class="usa-link" aria-describedby="B34">34</a>–<a href="#B36" class="usa-link" aria-describedby="B36">36</a>]. Catalase and quinone oxidoreductase exhibited consistent activity across layers, resolving a critical constraint in this system: While DyPs and lignin peroxidases (LiPs) require H<sub>2</sub>O<sub>2</sub> to initiate radical-based cleavage [<a href="#B35" class="usa-link" aria-describedby="B35">35</a>], its intracellular toxicity necessitates tight regulation via catalase-mediated disproportionation. Concurrently, quinone oxidoreductases likely sustain extracellular redox balance during aromatic fragment processing [<a href="#B35" class="usa-link" aria-describedby="B35">35</a>], highlighting layered metabolic cooperation between depolymerization and detoxification modules.</p>
<p>Lignin, after extracellular depolymerization, yields various small molecular aromatic compounds, which are transported into the cell and undergo a series of peripheral catabolic pathways and central metabolic pathways [<a href="#B37" class="usa-link" aria-describedby="B37">37</a>]. In this study, lignin small molecular fragments were primarily degraded through 3 pathways: β-aryl ether fragments (⑥ and ⑦), syringyl lignin fragments (㉗ to ㉟), and diarylpropane fragments (⑦, ⑨, and ⑩; Fig. <a href="#F3" class="usa-link">3</a>A), as the genes for these pathways presented high abundance in the metagenome and metatranscriptome (Fig. <a href="#F3" class="usa-link">3</a>C). Genes for the biphenyl fragment pathway (LigXYZ) were essentially absent in both the metagenome and metatranscriptome. Genes for the ferulic acid fragments presented at low abundance in the metagenome of the 5-layer samples, but no transcriptional activity was detected across the layers.</p>
<p>Lignin biotransformation proceeds through sequential biochemical phases: initial extracellular depolymerization via oxidative enzymes (e.g., laccases/peroxidases) to yield aromatic oligomers, followed by intracellular funneling of these metabolites where aromatic ring fission proceeds, ultimately generating acetyl-CoA and succinyl-CoA for assimilation into central carbon metabolism. In this study, the degradation of smaller lignin molecule ring cleavage proceeds through the 4,5-PDOG and 3,4-PDOG pathways to the TCA cycle (tricarboxylic acid cycle), with relatively complete gene sets in both the metagenome and metatranscriptome (Fig. <a href="#F3" class="usa-link">3</a>B). The metaproteomic results also indicate a relatively intact basic metabolic pathway (Fig. <a href="#supplementary-material-1" class="usa-link">S2</a>A).</p></section><section id="sec7"><h3 class="pmc_sec_title">Lignin decomposers involved in the broad carbon cycling</h3>
<p>We also interrogated the Carbon Cycle Database to demonstrate the broad carbon cycling capabilities of these taxa (Table <a href="#supplementary-material-1" class="usa-link">S4</a>). This additional comparison ensures that our analysis encompasses the wide-ranging potential of these microbial groups in carbon metabolism, which serves as a valuable resource for understanding the role of microorganisms in global carbon cycling.</p></section><section id="sec8"><h3 class="pmc_sec_title">Lignin decomposers involved in the methane process</h3>
<p>CH<sub>4</sub> is typically produced in anoxic sediments but can also be produced in oxic conditions [<a href="#B38" class="usa-link" aria-describedby="B38">38</a>]. To identify whether lignin-degrading bacteria can utilize intermediate products of lignin to produce methane under microoxic conditions, we transferred the enriched samples cultured for 90 d into sealed bottles and measured the methane production in 5-layer enrichment system. Methane gas production was monitored daily for 3 consecutive days. Methane gas was undetectable throughout the vertically stratified experimental system, maintaining consistency with control group under equivalent analytical protocols (Table <a href="#supplementary-material-1" class="usa-link">S5</a>; ANOVA, <em>P</em> &lt; 0.05). We were unable to detect the key methane-producing genes <em>mcrABG</em> in enriched samples based on metagenomic (<em>n</em> = 5) and transcriptomic data (<em>n</em> = 5). This indicates that these lignin-degrading bacteria do not possess the capability to produce methane through the known pathway involving the <em>mcr</em> gene.</p>
<p>However, lignin-degrading enzymes facilitate the demethylation of methoxy (-OCH<sub>3</sub>) and methylated (-CH<sub>3</sub>) compounds within lignin [<a href="#B39" class="usa-link" aria-describedby="B39">39</a>], leading to the production of methylated small molecular compounds. These small molecules may serve as precursors for methane production in cold seep areas. We have reconstructed a metabolic pathway for methane production by the enriched microbial community (Fig. <a href="#F4" class="usa-link">4</a> and Fig. <a href="#supplementary-material-1" class="usa-link">S3</a>). We identified methane production precursor genes associated with lignin-degrading taxa, such as <em>mtbC</em> (found in <em>Rhodobacterales</em> and <em>Streptosporangiales</em>) and <em>torACZ</em> genes (present in <em>Burkholderiales</em>, <em>Pseudomonadales</em>, and <em>Rhizobiales</em>) (Fig. <a href="#F4" class="usa-link">4</a>). The torACZ gene cluster catalyzed the enzymatic conversion of TMAO (trimethylamine-N-oxide) to TMA(trimethylamine) through a reductive biochemical pathway, representing a branch of methylotrophic methane generation [<a href="#B40" class="usa-link" aria-describedby="B40">40</a>]. The metaproteomic analysis revealed a considerable abundance of enzymes involved in these methane cycle processes (Fig. <a href="#supplementary-material-1" class="usa-link">S4</a>).</p>
<figure class="fig xbox font-sm" id="F4"><h4 class="obj_head">Fig. 4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1abc/12377529/7df0c14a4629/research.0848.fig.004.jpg" loading="lazy" height="461" width="660" alt="Fig. 4."></p>
<div class="p text-right font-secondary"><a href="figure/F4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Contribution of lignin degraders to methane production progress. Analysis of methane production genes within lignin-degrading microbial communities reveals their involvement in the methane generation process. It illustrates the annotation of 15 high-quality species-level bins, each containing genes associated with the methane generation process and capable of lignin degradation. The contributions of these species to methane production are indicated adjacent to the corresponding genes in the figure, following the removal of redundant entries. The top panel displays the chemical structure of lignin, and the right side lists the species of microbes involved in lignin degradation.</p></figcaption></figure><p>We identified 15 high-quality bins in species level (containing methane generation precursor process genes) with the capability to degrade lignin, after removing repetition. These species’ contributions to the methane generation precursors were annotated next to the related genes in Fig. <a href="#F4" class="usa-link">4</a>. Unlike the anaerobic methanogenesis pathways in deep sedimentary that utilize nonmethylated precursors, lignin decomposition releases methyl group donors, establishing a unique shallow CH<sub>4</sub> source with contributions to sediment–water interface methane fluxes.</p></section><section id="sec9"><h3 class="pmc_sec_title">Lignin decomposer distribution in global representative cold seeps</h3>
<p>Utilizing a global collection of cold seep metagenomes and metagenomes, we explore lignin-degrading communities based on the 1,495 bins (Table <a href="#supplementary-material-1" class="usa-link">S6</a>). Our findings revealed that 395 of these bins contained genes associated with lignin degradation (Table <a href="#supplementary-material-1" class="usa-link">S7</a>). Constructing a phylogenetic tree with these bins, we observed that these degrading communities were broad distribution within the orders <em>Burkholderiales</em>, <em>Pseudomonadales</em>, and <em>Rhizobiales</em> (Fig. <a href="#F5" class="usa-link">5</a>B). Genomic analysis revealed a bipartite distribution of lignin-processing capacity: While a limited subset of bins harbored genetic determinants for macromolecular lignin depolymerization, the predominant fraction encoded enzymatic machinery specialized for catabolizing lignin-derived monomers, potentially through cross-pathway recruitment of conserved aromatic degradation orthologs. These degrading communities were distributed across 77 phyla, indicating a broad distribution of lignin degraders in globally representative cold seeps.</p>
<figure class="fig xbox font-sm" id="F5"><h4 class="obj_head">Fig. 5.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1abc/12377529/7d56a26d6c3b/research.0848.fig.005.jpg" loading="lazy" height="660" width="606" alt="Fig. 5."></p>
<div class="p text-right font-secondary"><a href="figure/F5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>(A) Geographic locations of globally representative cold seeps. (B) Phylogenetic tree of the bins (whole genomes) of lignin-degrading microbial communities in globally representative cold seeps. The outer circle illustrates the distribution of these lignin degraders at the phylum level, with the names of the top 10 most abundant phyla shown; the inner circle shows the distribution of lignin degraders at the order level.</p></figcaption></figure></section></section><section id="sec10"><h2 class="pmc_sec_title">Discussion</h2>
<p>Our systematic dissection of lignin processing in the Haima cold seep reveals how microbial consortia govern specific refractory carbon turnover in these cold seep ecosystems. Quantitative sediment profiling identified fresh lignin inputs alongside degradation intermediates, confirming active processing of terrestrial organic matter in marine sediments. Genomically resolved degraders spanning <em>Burkholderiales</em>, <em>Pseudomonadales</em>, and <em>Rhizobiales</em> (Fig. <a href="#F2" class="usa-link">2</a>) employ complementary oxidative strategies—DyPs and glutathione-dependent β-etherases (LigEFG)—to depolymerize lignin into β-aryl ether, syringyl, and diarylpropane fragments. These products subsequently enter central metabolism via convergent 4,5-PDOG and 3,4-PDOG pathways (Fig. <a href="#F2" class="usa-link">2</a>A and B), with methylated derivatives potentially subsidizing near-surface methanogenesis through methylotrophic priming (Fig. <a href="#F4" class="usa-link">4</a>). Crucially, the global prevalence of these degraders across cold seeps [<a href="#B41" class="usa-link" aria-describedby="B41">41</a>–<a href="#B44" class="usa-link" aria-describedby="B44">44</a>] suggests that lignin processing represents a ubiquitous but overlooked carbon flux in deep-sea cold seeps.</p>
<p>The temporal community divergence may stem from lignin-driven selective pressures that progressively favored specialized taxa through resource partitioning, metabolic adaptation, and cooperative interactions [<a href="#B45" class="usa-link" aria-describedby="B45">45</a>,<a href="#B46" class="usa-link" aria-describedby="B46">46</a>]. As lignin became the sole carbon source, competitive exclusion shifted communities from generalist heterotrophs (day 0) to lignin-specialized assemblages (days 7, 30, 45, 60, 75, and 90) (Fig. <a href="#F1" class="usa-link">1</a>B), dominated by taxa (e.g., <em>Sphingomonadales</em> and <em>Burkholderiales</em>) with enzymatic machinery (e.g., laccases/peroxidases) for cleaving rate-limiting β-aryl ether linkages (Figs. <a href="#F1" class="usa-link">1</a>C and <a href="#F2" class="usa-link">2</a>). This metabolic specialization was amplified by in situ enzymatic induction and cross-feeding networks, where primary degraders converted complex lignin into metabolites that sustained secondary utilizers [<a href="#B47" class="usa-link" aria-describedby="B47">47</a>], creating syntrophic feedback loops that reinforced the community’s functional adaptation to lignin utilization.</p>
<p>The first step of lignin biodegradation is to depolymerize lignin through extracellular lignin depolymerase. Ligninolytic enzymes employ diverse catalytic mechanisms for polymer decomposition. Laccases (LACs), classified as multicopper oxidoreductases, facilitate lignin depolymerization through radical-mediated oxidation processes mediated by their tetra-nuclear copper cluster [<a href="#B48" class="usa-link" aria-describedby="B48">48</a>]. The β-O-4 aryl ether linkages—predominant structural motifs in lignin—undergo specific cleavage through 2 distinct enzymatic systems: NAD<sup>+</sup>-dependent Cα-dehydrogenase (LigD) catalyzes α-carbon oxidation, while the glutathione-dependent LigEFG complex mediates β-ether bond cleavage through nucleophilic thiolysis [<a href="#B49" class="usa-link" aria-describedby="B49">49</a>–<a href="#B51" class="usa-link" aria-describedby="B51">51</a>]. Furthermore, peroxidase family enzymes demonstrate significant oxidative capacity in lignin modification through hydrogen peroxide-dependent radical chain reactions [<a href="#B46" class="usa-link" aria-describedby="B46">46</a>,<a href="#B52" class="usa-link" aria-describedby="B52">52</a>]. Previous studies indicate that the most frequently detected lignin-degrading gene families in terrestrial ecosystems include <em>dypA</em>, <em>dypB</em>, <em>laCs</em>, and <em>mnP</em>, among others. Terrestrial ecosystems harbor both aerobic and anaerobic lignin degradation pathways, with aerobic pathways being more abundant [<a href="#B53" class="usa-link" aria-describedby="B53">53</a>]. In contrast, our study discovered that in the Haima cold seep and globally representative cold seeps, genes encoding enzymes (LigEFG, DyPs) for lignin depolymerization. Degradation proceeds via both aerobic and anaerobic processes, and analysis of enrichment cultures enables the detection of gene signatures associated with anaerobic pathways such as <em>aor</em>, <em>phhB</em>, and <em>tyrB</em> (Fig. <a href="#supplementary-material-1" class="usa-link">S5</a> and Table <a href="#supplementary-material-1" class="usa-link">S9</a>). We found that a total of 169 bins contained the <em>phhB</em> gene, 336 bins contained the <em>tyrB</em> gene, and no bins contained the <em>aor</em> gene, which is a key gene for the anaerobic process of lignin degradation (Table <a href="#supplementary-material-1" class="usa-link">S10</a>).</p>
<p>In previous estuarine studies, genes such as <em>lacCs</em>, <em>dyPs</em>, and <em>ligD</em> were found to be involved in the depolymerization of lignin [<a href="#B54" class="usa-link" aria-describedby="B54">54</a>,<a href="#B55" class="usa-link" aria-describedby="B55">55</a>]. In coastal ecosystems, the laccase gene family exhibits high abundance, highlighting its critical role in lignin decomposition within these environments [<a href="#B26" class="usa-link" aria-describedby="B26">26</a>,<a href="#B53" class="usa-link" aria-describedby="B53">53</a>]. While the abundance of gene families associated with aerobic and anaerobic degradation pathways remains relatively balanced in coastal ecosystems, an increasing trend in the abundance of anaerobic pathway-related gene families has been observed [<a href="#B26" class="usa-link" aria-describedby="B26">26</a>,<a href="#B56" class="usa-link" aria-describedby="B56">56</a>]. In Haima cold seep, canonical pathways dependent on multicopper oxidases or Cα-dehydrogenases (LACs) were absent, suggesting niche-specific adaptation of oxidative machinery. These findings underscore the intricate variations in lignin depolymerization processes observed among distinct ecosystems. Furthermore, the metabolic routes involved in breaking down lignin-derived aromatic compounds exhibit notable parallels to decomposition pathways documented in estuarine sediments, geothermal wetlands, and terrestrial soil matrices [<a href="#B57" class="usa-link" aria-describedby="B57">57</a>–<a href="#B62" class="usa-link" aria-describedby="B62">62</a>], with protocatechuic acid being cleaved through 2 distinct routes: the 4,5-PDOG pathway and the 3,4-PDOG pathway.</p>
<p>In terrestrial ecosystems, lignin degradation is primarily mediated by synergistic interactions between fungi and bacteria, with mechanisms exhibiting significant ecosystem specificity [<a href="#B63" class="usa-link" aria-describedby="B63">63</a>]. White-rot fungi (e.g., <em>Phanerochaete chrysosporium</em>) are the dominant degraders in forest ecosystems, employing secreted ligninolytic enzymes such as LiP, manganese peroxidase (MnP), and laccase to oxidatively dismantle lignin via nonspecific radical reactions that disrupt its aromatic backbone [<a href="#B63" class="usa-link" aria-describedby="B63">63</a>,<a href="#B64" class="usa-link" aria-describedby="B64">64</a>]. In temperate forests, enzyme activity correlates strongly with climatic conditions; for example, beech forests exhibit concurrent MnP, LiP, and laccase activity, while pine forests rely predominantly on LiP, with degradation rates positively linked to temperature [<a href="#B65" class="usa-link" aria-describedby="B65">65</a>]. In contrast, grassland and agricultural systems depend more on bacterial degraders (e.g., <em>Rhodococcus</em> spp.), which utilize dioxygenases and demethylases to break down lignin monomers and employ microbial consortia to degrade lignin–cellulose complexes [<a href="#B66" class="usa-link" aria-describedby="B66">66</a>]. Notably, the low pH of forest soils favors fungal activity, resulting in higher lignin oxidation rates compared to grasslands and farmlands [<a href="#B66" class="usa-link" aria-describedby="B66">66</a>].</p>
<p>In previously studied marine ecosystems, lignin degradation is predominantly bacterial, with <em>Gammaproteobacteria</em> (e.g., <em>Alteromonadaceae</em> and <em>Idiomarinaceae</em>) and <em>Alphaproteobacteria</em> excelling in oligotrophic and high-salinity environments through phenylacetyl-CoA (PA-CoA) pathways [<a href="#B67" class="usa-link" aria-describedby="B67">67</a>]. Unlike terrestrial systems, marine fungi (e.g., <em>Ascomycota</em> and <em>Basidiomycota</em>) are scarce, constituting less than 2% of microbial communities [<a href="#B67" class="usa-link" aria-describedby="B67">67</a>]. Studies show that marine bacteria achieve lignin mineralization rates of 10% to 20%, relying on dynamic interspecies interactions [<a href="#B68" class="usa-link" aria-describedby="B68">68</a>]. In anaerobic sediments, <em>Bathyarchaeota</em> contribute to lignin metabolism through demethylation and aromatic ring cleavage, producing acetate for methanogens and forming carbon cycling networks [<a href="#B69" class="usa-link" aria-describedby="B69">69</a>,<a href="#B70" class="usa-link" aria-describedby="B70">70</a>]. Coastal sediments harbor Proteobacteria and Firmicutes that drive carbon cycling via reconstructed lignin-degrading networks (e.g., β-ketoadipate and protocatechuate branches) [<a href="#B71" class="usa-link" aria-describedby="B71">71</a>]. In deep-sea cold seep ecosystems, our analysis based on species-level bins revealed the involvement of certain groups, such as <em>Methylophaga thalassica</em>, which had not been previously reported to possess lignin degradation functions. <em>M. thalassica</em> carries genes (<em>praD</em>, <em>praF</em>, <em>praG</em>) involved in degrading late-stage lignin-derived small molecules. Global seep analysis identified archaeal bins harboring lignin-degrading genes, mainly including <em>Asgardarchaeota</em> and <em>Thermoplasmatota</em> (Tables <a href="#supplementary-material-1" class="usa-link">S8</a> and <a href="#supplementary-material-1" class="usa-link">S10</a>). Notably, no archaeal bins were found to participate in active lignin-utilizing microbial consortia or their associated metabolic pathways under aerobic enrichment conditions (Tables <a href="#supplementary-material-1" class="usa-link">S8</a> and <a href="#supplementary-material-1" class="usa-link">S10</a>).</p>
<p>In this study, we combined methane gas detection and functional gene analysis, revealing that lignin-degrading bacteria enriched under aerobic conditions lack complete methanogenesis pathways [<a href="#B13" class="usa-link" aria-describedby="B13">13</a>,<a href="#B72" class="usa-link" aria-describedby="B72">72</a>]. However, these bacteria play a critical role in generating methyl group precursors (e.g., methanol and methylamines) through lignin depolymerization. Under oxygen-limited conditions, isolates like <em>Klebsiella</em> sp. LEA1 metabolize lignin into methylated phenols (e.g., 4-vinyl guaiacol) and aromatic acids [<a href="#B73" class="usa-link" aria-describedby="B73">73</a>], which may serve as substrates for methylotrophic methanogens. In cold seeps, hydrogen-dependent methylotrophic methanogens (e.g., <em>Methanomassiliococcales</em>) utilize these methyl compounds coupled with H₂ [<a href="#B2" class="usa-link" aria-describedby="B2">2</a>,<a href="#B4" class="usa-link" aria-describedby="B4">4</a>]. Our study identified 15 species-level MAGs encoding key enzymes for methyl transfer (e.g., <em>mtaB</em> and <em>mttC</em>) and corrinoid cofactor metabolism (Fig. <a href="#F4" class="usa-link">4</a>), consistent with methylotrophic methanogenesis reported in seep sediments [<a href="#B33" class="usa-link" aria-describedby="B33">33</a>]. Persistent deposition of lignocellulose provides sustained carbon input, while microbial redox cycling (e.g., Fe<sup>3+</sup>/Fe<sup>2+</sup> and S<sup>0</sup>/HS<sup>−</sup>) generates methylotrophic byproducts [<a href="#B74" class="usa-link" aria-describedby="B74">74</a>]. This may create anoxic microniches conducive to shallow subsurface methanogenesis near the sediment–water interface. The contribution of this methyl-shunt to CH₄ emissions may be further amplified in shallow zones where methanotrophic activity is suppressed, rendering its impact on sediment-surface CH₄ fluxes non-negligible.</p>
<p>Previous laboratory and natural condition studies have indicated that the degradation period for most microbes targeting lignin was relatively long, ranging from 30 to 60 d, with degradation rates between 20% and 50% [<a href="#B37" class="usa-link" aria-describedby="B37">37</a>,<a href="#B46" class="usa-link" aria-describedby="B46">46</a>,<a href="#B75" class="usa-link" aria-describedby="B75">75</a>,<a href="#B76" class="usa-link" aria-describedby="B76">76</a>]. Our cultivation time conditions were conducive to the enrichment of lignin degraders, and changes in the community were detected through PCoA analysis. Taxa such as <em>Burkholderiales</em>, <em>Pseudomonadales</em>, and <em>Parvibaculales</em>, which possess lignin degradation genes, increased significantly over time, demonstrating the effectiveness of our enrichment conditions. We have recovered a group of high-quality bins with lignin degradation capabilities, allowing for a more precise identification of lignin degraders at the species level, which is relatively innovative in the context of cold seep studies. Our laboratory-based experiments could not fully replicate complex in situ conditions, which may limit direct quantitative extrapolation to natural environments. Specifically, the absence of native environmental pressures (e.g., high hydrostatic pressure characteristic of cold seeps) and potential differences in substrate availability/accessibility within the sediment matrix in our enrichment system could alter microbial activity, community dynamics, and, thus, the representativeness of the enriched degraders and their degradation kinetics compared to the true in situ process. While these limitations may affect kinetic rates or absolute degradation efficiencies, our qualitative findings on lignin-degrading consortia and key functional taxa (e.g., dominant bacterial degraders and enzymatic pathways) should remain robust. To address these constraints and improve our understanding of in situ lignin degradation in cold seeps, future studies should employ in situ approaches. This will include deploying high-pressure cultivation systems to simulate the deep-sea conditions, combined with stable isotope probing (SIP) using isotopically labeled lignin substrates. SIP will allow us to directly link metabolic activity to taxonomic identity under relevant pressure conditions, providing a more authentic picture of the key lignin-degrading microorganisms and their functional roles in situ.</p></section><section id="sec11"><h2 class="pmc_sec_title">Conclusion</h2>
<p>Cold seep ecosystems possess a remarkably robust capacity for lignin degradation. Identifying and confirming the primary microbial pathways for lignin carbon decomposition within these environments is an essential foundational work. Our research has uncovered a suite of cold seep microbial communities with the potential for lignin degradation, which are distributed across taxa such as <em>Burkholderiales</em>, <em>Pseudomonadales</em>, and <em>Rhizobiales</em>. These groups acted on the depolymerization of lignin through genes encoding <em>dyPs</em> and <em>ligEFG</em>, and subsequently, the breakdown primarily followed 3 pathways: β-aryl ether fragments, syringyl lignin fragments, and diarylpropane fragment. The degradation of smaller lignin molecules then proceeded through the 4,5-PDOG and 3,4-PDOG pathways, entering the TCA cycle. Methylated small-molecule compounds produced from lignin degradation may serve as precursors for methane production in cold seep regions. This evidence the diverse ecological strategies of cold seep microbes and broadens our understanding. The lignin-degrading communities we have identified were widely distributed across global cold seeps and exhibit relatively high abundance, indicating the widespread capacity for lignin degradation among sedimentary microbes in cold seeps.</p></section><section id="sec12"><h2 class="pmc_sec_title">Methods</h2>
<section id="sec13"><h3 class="pmc_sec_title">Sampling in situ</h3>
<p>Sediment sampling was conducted at the Haima cold seep site (16°43′N, 110°28′E) located on the northern continental shelf of the South China Sea during April 2023. The collection was performed at a water depth of 1,384 m using a box core sampler with 30-cm penetration depth during the scientific cruise organized by R/V “Kexue”. Seafloor surface sediments were successfully retrieved through standardized box corer deployment procedures, ensuring minimal disturbance to sediment–water interface integrity. Immediately following core retrieval, sediment samples were stabilized through prompt transfer to 4 °C temperature-controlled shipboard storage chambers, where controlled preservation effectively suppressed exogenous microbial metabolisms.</p></section><section id="sec14"><h3 class="pmc_sec_title">Incubation experiments</h3>
<p>Surface sediments from the cold seep (0 to 25 cm) were collected in 5 stratified layers, with 5 g of each layer mixed with 20 ml of sterilized water to form a slurry. The slurry was equally distributed into conical flasks in triplicate, with 6 ml allocated to each flask. Each flask received 5 g of alkaline lignin (99.9% purity, CAS 8068-05-1; Sigma-Aldrich) as the sole carbon source in a layered cultivation system. Samples were placed into flasks and diluted with sterile water to achieve 150-ml total volume for enrichment culture. Triplicate samples were taken for each layer, yielding a total of 15 treatments. Enrichment conditions were maintained at pH 6 with a shaking speed of 110 rpm in the dark. Samples were collected at 7 time points—0, 7, 30, 45, 60, 75, and 90 d—over a 3-month experimental period. This resulted in a total of 105 samples, with each time point having 5 layers and 3 replicates.</p></section><section id="sec15"><h3 class="pmc_sec_title">Lignin parameters</h3>
<p>The in situ collected surface sediments (0 to 25 cm) were equally divided into 5 layers, and each layer was measured separately for lignin parameters. After the sediment was naturally air-dried to constant weight, the total organic carbon content of 1-g sediment was determined using a TOC (total organic carbon) analyzer (TOC-VCPN; Shimadzu Corp., Tokyo, Japan). Lignin biomarker quantification was performed following standardized protocols outlined in prior studies [<a href="#B77" class="usa-link" aria-describedby="B77">77</a>,<a href="#B78" class="usa-link" aria-describedby="B78">78</a>], with method validation through spike recovery tests. The analytical workflow commenced with microwave-assisted alkaline oxidative hydrolysis (165 °C × 3.5 h) using a CEM MARS5 microwave reactor (300 W maximum power). Approximately 2 g of homogenized sediment aliquots was combined in Teflon-lined digestion vessels with reagent-grade CuO, (NH<sub>4</sub>)Fe(SO<sub>4</sub>)<sub>2</sub> solution, and 2 M NaOH. After undergoing nitrogen sparging, reaction vessels underwent controlled thermal decomposition under inert atmosphere. Post-hydrolysis cooling to ambient temperature facilitated addition of ethyl vanillin as quantitative internal standard, achieving 95% ± 2% mean recovery across triplicate analyses.</p>
<p>Phase separation was achieved through sequential liquid–liquid extraction with ethyl acetate:petroleum ether (99:1 v/v). The combined organic phases were concentrated under N<sub>2</sub> stream and derivatized with BSTFA [<em>N</em>,<em>O</em>-bis(trimethylsilyl)tri fluoroacetamide]:TMCS (trimethylchlorosilane) (99:1). Final analysis employed an Agilent 6890N GC-FID system equipped with a DB-1 capillary column, as detailed in [<a href="#B79" class="usa-link" aria-describedby="B79">79</a>].</p>
<p>Following incubation, lignin degradation was quantified gravimetrically. After 90 d of treatment, samples were filtered through 0.2-μm membrane filters (Millipore, USA). Lignin retained on the filters was collected, dried overnight at 45 °C in a drying oven, and weighed. The percentage of lignin weight loss was calculated as follows:</p>
<table class="disp-formula p" id="EQ1"><tr>
<td class="formula"><math id="M1" display="block" overflow="linebreak"><msub><mi>P</mi><mi mathvariant="normal">W</mi></msub><mo>=</mo><mfenced open="[" close="]"><mrow><mfenced open="(" close=")"><mrow><msub><mi>W</mi><mn>0</mn></msub><mo>−</mo><msub><mi>W</mi><mi mathvariant="normal">T</mi></msub></mrow></mfenced><mo>/</mo><msub><mi>W</mi><mn>0</mn></msub></mrow></mfenced><mo>×</mo><mn>100</mn><mi mathvariant="normal">%</mi></math></td>
<td class="label">(1)</td>
</tr></table>
<p>where <em>W</em><sub>0</sub> represents the initial lignin weight (g) and <em>W</em><sub>T</sub> denotes the dry weight (g) at the endpoint (90 d).</p>
<p>All measurements were performed in triplicate.</p></section><section id="sec16"><h3 class="pmc_sec_title">CH<sub>4</sub> flux measurement</h3>
<p>On day 90, 15 ml of lignin-enriched suspension samples from the remaining materials was transferred into sealed, light-protected 50-ml glass bottles, with 3 replicates set for each layer and placed in a shaker for cultivation at 25 °C with a rotation speed of 110 rpm. A control was also set up, with the addition of 15 ml of sterilized water. Headspace gas sampling (10 ml/day) was conducted using evacuated aluminized gas sampling bags. To maintain isobaric conditions during gas extraction, a dual-syringe system was implemented: One syringe withdrew gas samples, while a second simultaneously delivered 10 ml of ultrapure helium as pressure compensation. Following sampling, vessels were uncapped and aerated for 30 min under ambient conditions. Methane gas quantification was performed via wavelength-scanned cavity ring-down spectroscopy (WS-CRDS, Picarro G2201-i) with 1-Hz sampling frequency [<a href="#B80" class="usa-link" aria-describedby="B80">80</a>]. For 5 treatment groups and 1 control group, samples were collected continuously over 3 d, resulting in a total of 54 gas samples (3 d * 6 treatments * 3 replicates).</p></section><section id="sec17"><h3 class="pmc_sec_title">Total protein extraction and detection</h3>
<p>Protein extraction from day 90 samples (<em>n</em> = 5) was conducted following cryogenic protocols. After ice-thawing, samples were homogenized in BPP buffer (200 mM Tris-Base, 200 mM NaCl, 50 mM vitamin C, 1% sodium deoxycholate, 1% NP-40, 1% Tween 20, 5 mM EDTA, 30% sugar, 50 mM sodium tetraborate, 1% PVPP) using mechanical disruption and centrifuged (12,000<em>g</em>, 4 °C, 20 min). Sequential phenol partitioning was performed with tris-saturated phenol under vortexing (10 min, 4 °C), followed by phase separation through successive centrifugations. Proteins were precipitated overnight in ammonium acetate–methanol (−20 °C) and washed twice with 90% acetone. Pellets were resolubilized in denaturing buffer [8 M urea, 1% SDS (sodium dodecyl sulfate), protease inhibitors] with ultrasonic assistance (2-min ice bath). Post-centrifugation supernatants underwent BCA (bicinchoninic acid) quantification (Thermo Scientific kit) against bovine serum albumin standards. Electrophoretic validation was performed using Laemmli SDS–polyacrylamide gel electrophoresis [<a href="#B63" class="usa-link" aria-describedby="B63">63</a>] under reducing conditions (150 V, 60 min) [<a href="#B81" class="usa-link" aria-describedby="B81">81</a>].</p>
<p>Protein samples (100 μg) underwent sequential processing in TEAB (triethylammonium bicarbonate) buffer: reduction with 10 mM TCEP [Tris(2-carboxyethyl)phosphine; 37 °C, 60 min], cysteine blocking via 40 mM IAM (iodoacetamide; room temperature, dark, 40 min), and clarification by centrifugation (10,000<em>g</em>, 4 °C, 20 min). Trypsin digestion (1:50 w/w, 37 °C, 16 h) preceded peptide desalting (HLB cartridges, Oasis HLB 96-well Plate, Waters) after vacuum drying. Reconstituted peptides in 0.1% TFA (trifluoroacetic acid) were quantified spectrophotometrically (NanoDrop One) with external calibration [<a href="#B81" class="usa-link" aria-describedby="B81">81</a>].</p></section><section id="sec18"><h3 class="pmc_sec_title">Nucleic acid extraction and sequencing</h3>
<p>Total nucleic acid from 105 samples was co-isolated via Total RNA Isolation kit (QIAGEN, Germany). RNA was collected from RNA Capture Columns. Retained columns enabled sequential DNA elution (RNA QIAGEN DNA Elution Accessory kit). After merging duplicates, 16<em>S</em> rRNA gene amplicons (515F-Y and 926R) from samples (<em>n</em> = 35) were sequenced [<a href="#B82" class="usa-link" aria-describedby="B82">82</a>]. For the 90th day sampling, a subset underwent metagenome (<em>n</em> = 5) and metatranscriptome (<em>n</em> = 5) sequencing. Following preprocessing steps, amplicon and metagenome/metatranscriptome sequencing were performed using Illumina NovaSeq 6000 systems (PE250 and PE150) via MAGIGENE (Guangzhou, China), respectively.</p></section><section id="sec19"><h3 class="pmc_sec_title">Amplicon analyses</h3>
<p>The raw sequencing data were processed as per established protocols [<a href="#B83" class="usa-link" aria-describedby="B83">83</a>]. Specifically, paired-end reads were consolidated using USEARCH v11, with primers excised employing CUTADAPT v2.4 software, operationalized with the parameters “--no-indels-e 0 --discard-unrimmed” [<a href="#B84" class="usa-link" aria-describedby="B84">84</a>]. Thereafter, low-quality reads defined as length &lt;350 base pairs (bp) or maximum expected error &gt;1.0 were stringently removed. The retained high-quality reads were grouped into amplicon sequence variants (ASVs) using the unoise3 algorithm [<a href="#B85" class="usa-link" aria-describedby="B85">85</a>]. Taxonomic assignment of ASVs was conducted with the naïve Bayes classifier [<a href="#B86" class="usa-link" aria-describedby="B86">86</a>] through SILVA 138 database analysis in the QIIME2 environment [<a href="#B87" class="usa-link" aria-describedby="B87">87</a>]. After removing ASVs unclassified as bacterial or archaeal, a curated ASV table was generated by mapping primer-trimmed reads to the ASVs’ reference sequences, thereby elucidating the microbial taxonomy with precision.</p></section><section id="sec20"><h3 class="pmc_sec_title">Metagenomic and metatranscriptomic analyses</h3>
<p>Metagenomic sequencing data were subjected to processing in alignment with prior documentation [<a href="#B88" class="usa-link" aria-describedby="B88">88</a>]. Low-quality sequences and adapter contaminants were trimmed from raw data using fastp (v0.23.4) [<a href="#B89" class="usa-link" aria-describedby="B89">89</a>], followed by scaffold reconstruction of processed reads via SPAdes (v3.15.2) [<a href="#B90" class="usa-link" aria-describedby="B90">90</a>]. Open reading frames (ORFs) were computationally predicted for scaffolds &gt;500 bp using Prodigal (v2.6.3) [<a href="#B91" class="usa-link" aria-describedby="B91">91</a>]. These ORFs were compiled into a deduplicated gene catalog with CD-HIT-EST (v4.8.1) at 95% identity and 90% coverage thresholds. For metatranscriptomic data, we deployed fastp v0.23.4 to purge low-quality sequences and adapter remnants, and RiboDetector [<a href="#B92" class="usa-link" aria-describedby="B92">92</a>] to eliminate rRNA sequences with the parameter “-e rrna”. ORF abundance and transcriptional activity [expressed as reads per kilobase per million mapped reads (RPKM)] were determined by aligning metagenomic/metatranscriptomic reads to the deduplicated gene catalog via Bowtie2 v2.5 [<a href="#B93" class="usa-link" aria-describedby="B93">93</a>] and CoverM v0.6.1, with thresholds of 95% identity, 80% read alignment, and 60% coverage. MAGs were extracted from metagenomic datasets and polished using MetaWRAP [<a href="#B94" class="usa-link" aria-describedby="B94">94</a>]. MAG quality was evaluated via CheckM v1.0.12 [<a href="#B95" class="usa-link" aria-describedby="B95">95</a>], while PROKKA v1.12 annotated MAGs meeting high-quality criteria (&gt;60% completeness, &lt;5% contamination) [<a href="#B96" class="usa-link" aria-describedby="B96">96</a>].</p></section><section id="sec21"><h3 class="pmc_sec_title">Reconstruction of lignin decomposition pathway</h3>
<p>We constructed a lignin database comprising a comprehensive collection of lignolytic enzyme sequences, curated from published literature [<a href="#B53" class="usa-link" aria-describedby="B53">53</a>,<a href="#B97" class="usa-link" aria-describedby="B97">97</a>]. The deduplicated gene dataset was screened against these reference proteins using DIAMOND v0.9 with thresholds “--id 50 --query-cover 80” [<a href="#B98" class="usa-link" aria-describedby="B98">98</a>]. Cumulative RPKM values for ORFs encoding target enzymes were calculated to quantify ligninolytic potential. Gene expression patterns were analyzed to estimate pathway activation status. Bacterial genomes (MAGs) associated with lignin degradation were identified through protein sequence comparison against the reference database using DIAMOND v0.9. Genomic assignments to specific metabolic pathways were determined through systematic evaluation of enzymatic gene inventories within each MAG. To determine taxonomic assignments of ligninolytic-associated genes, alignments were performed against the GTDB r95 proteome database using DIAMOND v0.9 with “—evalue 0.00001 -k 10” parameters. Taxonomic classification for each gene was established by applying the Last Common Ancestor (LCA) algorithm to the 10 highest-ranking matches. Taxonomic consistency between databases was ensured by converting GTDB classifications to National Center for Biotechnology Information (NCBI) taxonomy through the GTDB-provided mapping file.</p>
<p>Additionally, we employed the LCdb database to identify lignin-degrading genes from deduplicated gene dataset. Protein sequences were aligned against the LCdb reference database using DIAMOND v2.0.14. Then, alignment results were functionally annotated using a custom Python script. To investigate the distribution of lignin-degrading genes in global cold seep ecosystems, we analyzed MAGs identified through protein sequence comparison against the LCdb database [<a href="#B53" class="usa-link" aria-describedby="B53">53</a>].</p></section><section id="sec22"><h3 class="pmc_sec_title">Methane metabolic pathway</h3>
<p>We downloaded the carbon cycling database CcycDB (<a href="https://ccycdb.github.io/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://ccycdb.github.io/</a>) and the methane cycling database McycDB (<a href="https://github.com/qichao1984/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/qichao1984/</a>). Subsequently, using DIAMOND v0.9 with parameters set as described in the literature [<a href="#B40" class="usa-link" aria-describedby="B40">40</a>], we interrogated the nonredundant gene catalog obtained above against the sequences in these databases. Gene expression patterns of these target genes were analyzed to assess corresponding pathway activation states. Furthermore, MAG-derived ORFs were identified through database sequence alignment. Systematic evaluation of enzymatic gene inventories enabled precise mapping of MAGs to methane metabolism pathways.</p></section><section id="sec23"><h3 class="pmc_sec_title">Metaproteomic identification and analyses</h3>
<p>For the data-independent acquisition (DIA) raw data analysis, Spectronaut software (Version 18) was utilized [<a href="#B99" class="usa-link" aria-describedby="B99">99</a>]. Quantitative profiling utilized 6 peptides per protein with 3 fragment ions per peptide, under the following parameters: protein false discovery rate (FDR) cutoff ≤0.01%, peptide FDR ≤0.01%, identification confidence ≥99%, and XIC mass tolerance ≤75 parts per million [<a href="#B99" class="usa-link" aria-describedby="B99">99</a>]. Peptides that were shared or modified were excluded from the analysis. Quantification results were derived through integration and summation of chromatographic peak intensities [<a href="#B100" class="usa-link" aria-describedby="B100">100</a>]. Protein identification was conducted solely for those proteins possessing at least one unique peptide. Proteomic data processing was conducted via the Majorbio Cloud Platform (<a href="https://cloud.majorbio.com" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://cloud.majorbio.com</a>), followed by metagenome-informed functional annotation of all detected proteins [<a href="#B101" class="usa-link" aria-describedby="B101">101</a>].</p></section><section id="sec24"><h3 class="pmc_sec_title">Lignin decomposers in representative global cold seeps</h3>
<p>Metagenomic assemblies (MAGs) derived from 68 sediment specimens obtained from 6 globally representative cold seep ecosystems were retrieved from public repositories [<a href="#B7" class="usa-link" aria-describedby="B7">7</a>,<a href="#B41" class="usa-link" aria-describedby="B41">41</a>,<a href="#B72" class="usa-link" aria-describedby="B72">72</a>]. Sediment cores encompassed vertical redox zonation across 430-cm subseafloor depth, transitioning from oxic surficial zones to sulfidic strata at the sediment–water interface. These MAG data were species-delineated based on a 95% average nucleotide identity threshold, recovering a total of 1,427 species-level clusters. In combination with the 68 bins from our lignin-enriched samples, we obtained 1,495 species-level clusters in total (1,214 bacteria and 281 archaea; Table <a href="#supplementary-material-1" class="usa-link">S6</a>), which were discerned against a lignin database. Metagenomic bins harboring genetic determinants for lignin metabolism were subjected to phylogenetic reconstruction through maximum likelihood analysis implemented in the Phylophlan 3.0 platform (v3.7.1) [<a href="#B102" class="usa-link" aria-describedby="B102">102</a>]. Subsequent topological visualization of the refined phylogenetic tree (RAxML_bestTree.refined.tre) was executed using the Interactive Tree of Life interface (iTOL v6.7.3) [<a href="#B103" class="usa-link" aria-describedby="B103">103</a>].</p></section><section id="sec25"><h3 class="pmc_sec_title">Statistical analysis</h3>
<p>Statistical evaluation of lignin composition (components, sources, and characteristics), temporal stratification effects on microbiome community, and methane accumulation gas was performed using one-way ANOVA. The computational framework incorporated customized analytical pipelines executed in R version 4.2.2, leveraging both base statistical functions and specialized ecological analysis modules from the vegan package (v2.6-4) for comprehensive multivariate comparisons across 5 distinct sedimentary horizons.</p></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgments</h2>
<p><strong>Funding:</strong> Research funding was obtained through competitive grants from the following agencies: National Natural Science Foundation of China (NSFC grant IDs 42494884, 42325603, 42449303, and 32200090), Guangdong Provincial Basic Research Program (project no. 2025A1515010959), and the PI project of (GML20190609) administered by the Southern Marine Science and Engineering Guangdong Laboratory (Guangzhou).</p>
<p><strong>Author contributions:</strong> Experimental execution and analytical workflows were coordinated by J.L., including design of laboratory protocols, sample processing, statistical interpretation, and graphical synthesis. P.W. contributed to multivariate data analysis. Y.W. and J. Fan measured the content and composition of lignin in cold seep sediments. J.H. participated in laboratory sampling. X.D. provided global cold seep data. J. Feng and X.T. were responsible for manuscript revision. M.Z. participated in sea sampling. Y.H. and Y.Z. participated in gas testing. S.Z. provided manuscript revisions, instrumental support, and funding.</p>
<p><strong>Competing interests:</strong> The authors declare that they have no competing interests.</p></section><section id="sec26"><h2 class="pmc_sec_title">Data Availability</h2>
<p>Multi-omics sequencing datasets (16<em>S</em> rRNA gene amplicon sequencing, metagenomics, and metatranscriptomics) have been archived in the NCBI Sequence Read Archive repository under BioProject accessions PRJNA1205479-PRJNA1206677. Complementary experimental datasets supporting the conclusions of this work are maintained by the corresponding investigator team and can be accessed through formal data request protocols.</p></section><section id="supplementary-material-1"><h2 class="pmc_sec_title">Supplementary Materials</h2>
<section class="sm xbox font-sm" id="supp-1"><div class="caption p">
<span>Supplementary 1</span><p>Figs. S1 to S5</p>
<p>Tables S1 to S10</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12377529/bin/research.0848.f1.zip" data-ga-action="click_feat_suppl" class="usa-link">research.0848.f1.zip</a><sup> (3.7MB, zip) </sup>
</div></div></section></section><section id="ref-list1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="ref-list1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="B1">
<span class="label">1.</span><cite>Jiang Q, Jing H, Jiang Q, Zhang Y. 
Insights into carbon-fixation pathways through metagonomics in the sediments of deep-sea cold seeps. Mar Pollut Bull. 2022;176:
Article 113458.
</cite> [<a href="https://doi.org/10.1016/j.marpolbul.2022.113458" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35217425/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mar%20Pollut%20Bull&amp;title=Insights%20into%20carbon-fixation%20pathways%20through%20metagonomics%20in%20the%20sediments%20of%20deep-sea%20cold%20seeps&amp;volume=176&amp;publication_year=2022&amp;pages=Article%20113458&amp;pmid=35217425&amp;doi=10.1016/j.marpolbul.2022.113458&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B2">
<span class="label">2.</span><cite>Lv Y, Yang S, Xiao X, Zhang Y. 
Stimulated organic carbon cycling and microbial community shift driven by a simulated cold-seep eruption. MBio. 2022;13(2):
Article e0008722.
</cite> [<a href="https://doi.org/10.1128/mbio.00087-22" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8941925/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35229641/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=MBio&amp;title=Stimulated%20organic%20carbon%20cycling%20and%20microbial%20community%20shift%20driven%20by%20a%20simulated%20cold-seep%20eruption&amp;volume=13&amp;issue=2&amp;publication_year=2022&amp;pages=Article%20e0008722&amp;pmid=35229641&amp;doi=10.1128/mbio.00087-22&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B3">
<span class="label">3.</span><cite>Yang S, Lv Y, Liu X, Wang Y, Fan Q, Yang Z, Boon N, Wang F, Xiao X, Zhang Y. 
Genomic and enzymatic evidence of acetogenesis by anaerobic methanotrophic archaea. Nat Commun. 2020;11(1):
Article 3941.
</cite> [<a href="https://doi.org/10.1038/s41467-020-17860-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7414198/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32770005/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Commun&amp;title=Genomic%20and%20enzymatic%20evidence%20of%20acetogenesis%20by%20anaerobic%20methanotrophic%20archaea&amp;volume=11&amp;issue=1&amp;publication_year=2020&amp;pages=Article%203941&amp;pmid=32770005&amp;doi=10.1038/s41467-020-17860-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B4">
<span class="label">4.</span><cite>Boetius A, Wenzhöfer F. 
Seafloor oxygen consumption fuelled by methane from cold seeps. Nat Geosci. 2013;6(9):725–734.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Geosci&amp;title=Seafloor%20oxygen%20consumption%20fuelled%20by%20methane%20from%20cold%20seeps&amp;volume=6&amp;issue=9&amp;publication_year=2013&amp;pages=725-734&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B5">
<span class="label">5.</span><cite>Jiang Q, Jing H, Liu H, Du M. 
Biogeographic distributions of microbial communities associated with anaerobic methane oxidation in the surface sediments of deep-sea cold seeps in the South China Sea. Front Microbiol. 2022;13:
Article 1060206.
</cite> [<a href="https://doi.org/10.3389/fmicb.2022.1060206" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9822730/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36620029/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Front%20Microbiol&amp;title=Biogeographic%20distributions%20of%20microbial%20communities%20associated%20with%20anaerobic%20methane%20oxidation%20in%20the%20surface%20sediments%20of%20deep-sea%20cold%20seeps%20in%20the%20South%20China%20Sea&amp;volume=13&amp;publication_year=2022&amp;pages=Article%201060206&amp;pmid=36620029&amp;doi=10.3389/fmicb.2022.1060206&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B6">
<span class="label">6.</span><cite>Feng J, Yan J, Wang Y, Yang Z, Zhang S, Liang S, Li X. 
Methane mitigation: Learning from the natural marine environment. Innovation. 2022;3(5):
Article 100297.</cite> [<a href="https://doi.org/10.1016/j.xinn.2022.100297" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9425072/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36051819/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Innovation&amp;title=Methane%20mitigation:%20Learning%20from%20the%20natural%20marine%20environment&amp;volume=3&amp;issue=5&amp;publication_year=2022&amp;pages=Article%20100297&amp;pmid=36051819&amp;doi=10.1016/j.xinn.2022.100297&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B7">
<span class="label">7.</span><cite>Dong X, Peng Y, Wang M, Woods L, Wu W, Wang Y, Xiao X, Li J, Jia K, Greening C, et al. 
Evolutionary ecology of microbial populations inhabiting deep sea sediments associated with cold seeps. Nat Commun. 2023;14(1):
Article 1127.
</cite> [<a href="https://doi.org/10.1038/s41467-023-36877-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9974965/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36854684/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Commun&amp;title=Evolutionary%20ecology%20of%20microbial%20populations%20inhabiting%20deep%20sea%20sediments%20associated%20with%20cold%20seeps&amp;volume=14&amp;issue=1&amp;publication_year=2023&amp;pages=Article%201127&amp;pmid=36854684&amp;doi=10.1038/s41467-023-36877-3&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B8">
<span class="label">8.</span><cite>Wu X, Feng J, Chen X, Li C, Zhang S. 
Exploring carbon content variation in microplastics sequestrated from seawater to sediment in the Haima cold seep area. J Hazard Mater. 2024;462:
Article 132742.
</cite> [<a href="https://doi.org/10.1016/j.jhazmat.2023.132742" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37871440/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Hazard%20Mater&amp;title=Exploring%20carbon%20content%20variation%20in%20microplastics%20sequestrated%20from%20seawater%20to%20sediment%20in%20the%20Haima%20cold%20seep%20area&amp;volume=462&amp;publication_year=2024&amp;pages=Article%20132742&amp;pmid=37871440&amp;doi=10.1016/j.jhazmat.2023.132742&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B9">
<span class="label">9.</span><cite>Zhang X, Du Z, Zheng R, Luan Z, Qi F, Cheng K, Wang B, Ye W, Liu X, Lian C, et al. 
Development of a new deep-sea hybrid Raman insertion probe and its application to the geochemistry of hydrothermal vent and cold seep fluids. Deep-Sea Res Pt I Oceanogr Res Pap. 2017;123:1–12.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Deep-Sea%20Res%20Pt%20I%20Oceanogr%20Res%20Pap&amp;title=Development%20of%20a%20new%20deep-sea%20hybrid%20Raman%20insertion%20probe%20and%20its%20application%20to%20the%20geochemistry%20of%20hydrothermal%20vent%20and%20cold%20seep%20fluids&amp;volume=123&amp;publication_year=2017&amp;pages=1-12&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B10">
<span class="label">10.</span><cite>Zhai X, Shi X, Cheng H, Yao P, Zhao B, Chen L, Liu J, Cao L, Wang M, Fu L, et al. 
Horizontal and vertical heterogeneity of sediment microbial community in Site F cold seep, the South China Sea. Front Mar Sci. 2022;9:
Article 957762.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Front%20Mar%20Sci&amp;title=Horizontal%20and%20vertical%20heterogeneity%20of%20sediment%20microbial%20community%20in%20Site%20F%20cold%20seep,%20the%20South%20China%20Sea&amp;volume=9&amp;publication_year=2022&amp;pages=Article%20957762&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B11">
<span class="label">11.</span><cite>Muyzer G, Stams AJM. 
The ecology and biotechnology of sulphate-reducing bacteria. Nat Rev Microbiol. 2008;6(6):441–454.
</cite> [<a href="https://doi.org/10.1038/nrmicro1892" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18461075/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Microbiol&amp;title=The%20ecology%20and%20biotechnology%20of%20sulphate-reducing%20bacteria&amp;volume=6&amp;issue=6&amp;publication_year=2008&amp;pages=441-454&amp;pmid=18461075&amp;doi=10.1038/nrmicro1892&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B12">
<span class="label">12.</span><cite>Bhattarai S, Cassarini C, Lens PNL. 
Physiology and distribution of archaeal methanotrophs that couple anaerobic oxidation of methane with sulfate reduction. Microbiol Mol Biol Rev. 2019;83(3):
Article e00074-18.</cite> [<a href="https://doi.org/10.1128/MMBR.00074-18" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6710461/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31366606/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Microbiol%20Mol%20Biol%20Rev&amp;title=Physiology%20and%20distribution%20of%20archaeal%20methanotrophs%20that%20couple%20anaerobic%20oxidation%20of%20methane%20with%20sulfate%20reduction&amp;volume=83&amp;issue=3&amp;publication_year=2019&amp;pages=Article%20e00074-18&amp;pmid=31366606&amp;doi=10.1128/MMBR.00074-18&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B13">
<span class="label">13.</span><cite>Feng D, Qiu J, Hu Y, Peckmann J, Guan H, Tong H, Chen C, Chen J, Gong S, Li N, et al. 
Cold seep systems in the South China Sea: An overview. J Asian Earth Sci. 2018;168:3–16.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Asian%20Earth%20Sci&amp;title=Cold%20seep%20systems%20in%20the%20South%20China%20Sea:%20An%20overview&amp;volume=168&amp;publication_year=2018&amp;pages=3-16&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B14">
<span class="label">14.</span><cite>Kandasamy S, Nath BN. 
Perspectives on the terrestrial organic matter transport and burial along the land-deep sea continuum: Caveats in our understanding of biogeochemical processes and future needs. Front Mar Sci. 2016;3:
Article 00259.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Front%20Mar%20Sci&amp;title=Perspectives%20on%20the%20terrestrial%20organic%20matter%20transport%20and%20burial%20along%20the%20land-deep%20sea%20continuum:%20Caveats%20in%20our%20understanding%20of%20biogeochemical%20processes%20and%20future%20needs&amp;volume=3&amp;publication_year=2016&amp;pages=Article%2000259&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B15">
<span class="label">15.</span><cite>Schlunz B, Schneider RR. 
Transport of terrestrial organic carbon to the oceans by rivers: Re-estimating flux- and burial rates. Int J Earth Sci. 2000;88(4):599–606.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Int%20J%20Earth%20Sci&amp;title=Transport%20of%20terrestrial%20organic%20carbon%20to%20the%20oceans%20by%20rivers:%20Re-estimating%20flux-%20and%20burial%20rates&amp;volume=88&amp;issue=4&amp;publication_year=2000&amp;pages=599-606&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B16">
<span class="label">16.</span><cite>Yu S, Zhou Y, Gan M, Chen L, Xie Y, Zhong Y, Feng Q, Chen C. 
Lignocellulose-based optical biofilter with high near-infrared transmittance via lignin capturing–fusing approach. Research. 2023;6:
Article 0250.
</cite> [<a href="https://doi.org/10.34133/research.0250" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10585486/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37869743/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Research&amp;title=Lignocellulose-based%20optical%20biofilter%20with%20high%20near-infrared%20transmittance%20via%20lignin%20capturing%E2%80%93fusing%20approach&amp;volume=6&amp;publication_year=2023&amp;pages=Article%200250&amp;pmid=37869743&amp;doi=10.34133/research.0250&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B17">
<span class="label">17.</span><cite>Kamimura N, Sakamoto S, Mitsuda N, Masai E, Kajita S. 
Advances in microbial lignin degradation and its applications. Curr Opin Biotechnol. 2019;56:179–186.
</cite> [<a href="https://doi.org/10.1016/j.copbio.2018.11.011" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30530243/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Curr%20Opin%20Biotechnol&amp;title=Advances%20in%20microbial%20lignin%20degradation%20and%20its%20applications&amp;volume=56&amp;publication_year=2019&amp;pages=179-186&amp;pmid=30530243&amp;doi=10.1016/j.copbio.2018.11.011&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B18">
<span class="label">18.</span><cite>Chen Z, Wan C. 
Biological valorization strategies for converting lignin into fuels and chemicals. Renew Sust Energ Rev. 2017;73:610–621.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Renew%20Sust%20Energ%20Rev&amp;title=Biological%20valorization%20strategies%20for%20converting%20lignin%20into%20fuels%20and%20chemicals&amp;volume=73&amp;publication_year=2017&amp;pages=610-621&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B19">
<span class="label">19.</span><cite>Lou Y, Sun X, Yu Y, Zeng S, Li Y, Liu Y, Yu H. 
One-pot protolignin extraction by targeted unlocking lignin–carbohydrate esters via nucleophilic addition–elimination strategy. Research. 2023;6:
Article 0069.
</cite> [<a href="https://doi.org/10.34133/research.0069" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10013968/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36930767/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Research&amp;title=One-pot%20protolignin%20extraction%20by%20targeted%20unlocking%20lignin%E2%80%93carbohydrate%20esters%20via%20nucleophilic%20addition%E2%80%93elimination%20strategy&amp;volume=6&amp;publication_year=2023&amp;pages=Article%200069&amp;pmid=36930767&amp;doi=10.34133/research.0069&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B20">
<span class="label">20.</span><cite>Qu X, Zhao Y, Za C, Wang S, Ren Y, Wang Q, Shao J, Wang W, Dong X. 
Thermoresponsive lignin-reinforced poly(ionic liquid) hydrogel wireless strain sensor. Research. 2021;2021:
Article 9845482.
</cite> [<a href="https://doi.org/10.34133/2021/9845482" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8674648/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34957404/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Research&amp;title=Thermoresponsive%20lignin-reinforced%20poly(ionic%20liquid)%20hydrogel%20wireless%20strain%20sensor&amp;volume=2021&amp;publication_year=2021&amp;pages=Article%209845482&amp;pmid=34957404&amp;doi=10.34133/2021/9845482&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B21">
<span class="label">21.</span><cite>Palazzolo MA, Kurina-Sanz M. 
Microbial utilization of lignin: Available biotechnologies for its degradation and valorization. World J Microbiol Biotechnol. 2016;32(10):
Article 173.
</cite> [<a href="https://doi.org/10.1007/s11274-016-2128-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27565783/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=World%20J%20Microbiol%20Biotechnol&amp;title=Microbial%20utilization%20of%20lignin:%20Available%20biotechnologies%20for%20its%20degradation%20and%20valorization&amp;volume=32&amp;issue=10&amp;publication_year=2016&amp;pages=Article%20173&amp;pmid=27565783&amp;doi=10.1007/s11274-016-2128-y&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B22">
<span class="label">22.</span><cite>Abdelaziz OY, Brink DP, Prothmann J, Ravi K, Sun M, García-Hidalgo J, Sandahl M, Hulteberg CP, Turner C, Lidén G, et al. 
Biological valorization of low molecular weight lignin. Biotechnol Adv. 2016;34(8):1318–1346.
</cite> [<a href="https://doi.org/10.1016/j.biotechadv.2016.10.001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27720980/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biotechnol%20Adv&amp;title=Biological%20valorization%20of%20low%20molecular%20weight%20lignin&amp;volume=34&amp;issue=8&amp;publication_year=2016&amp;pages=1318-1346&amp;pmid=27720980&amp;doi=10.1016/j.biotechadv.2016.10.001&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B23">
<span class="label">23.</span><cite>Bonin P, Portas A, Hardy J, Guasco S, Bianchi TS, Ward ND, Rontani J-F. 
Biological mechanisms underlying priming of vascular plant material in the presence of diatoms. Aquat Microb Ecol. 2023;89:99–117.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Aquat%20Microb%20Ecol&amp;title=Biological%20mechanisms%20underlying%20priming%20of%20vascular%20plant%20material%20in%20the%20presence%20of%20diatoms&amp;volume=89&amp;publication_year=2023&amp;pages=99-117&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B24">
<span class="label">24.</span><cite>Dang Y, Cha Q, Liu S, Wang S, Li P, Li C, Wang P, Chen X, Tian J, Xin Y, et al. 
Phytoplankton-derived polysaccharides and microbial peptidoglycans are key nutrients for deep-sea microbes in the Mariana Trench. Microbiome. 2024;12(1):
Article 77.
</cite> [<a href="https://doi.org/10.1186/s40168-024-01789-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11044484/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38664737/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Microbiome&amp;title=Phytoplankton-derived%20polysaccharides%20and%20microbial%20peptidoglycans%20are%20key%20nutrients%20for%20deep-sea%20microbes%20in%20the%20Mariana%20Trench&amp;volume=12&amp;issue=1&amp;publication_year=2024&amp;pages=Article%2077&amp;pmid=38664737&amp;doi=10.1186/s40168-024-01789-x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B25">
<span class="label">25.</span><cite>Leadbeater DR, Oates NC, Bennett JP, Li Y, Dowle AA, Taylor JD, Alponti JS, Setchfield AT, Alessi AM, Helgason T, et al. 
Mechanistic strategies of microbial communities regulating lignocellulose deconstruction in a UK salt marsh. Microbiome. 2021;9(1):
Article 48.</cite> [<a href="https://doi.org/10.1186/s40168-020-00964-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7890819/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33597033/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Microbiome&amp;title=Mechanistic%20strategies%20of%20microbial%20communities%20regulating%20lignocellulose%20deconstruction%20in%20a%20UK%20salt%20marsh&amp;volume=9&amp;issue=1&amp;publication_year=2021&amp;pages=Article%2048&amp;pmid=33597033&amp;doi=10.1186/s40168-020-00964-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B26">
<span class="label">26.</span><cite>Peng Q, Lin L, Tu Q, Wang X, Zhou Y, Chen J, Jiao N, Zhou J. 
Unraveling the roles of coastal bacterial consortia in degradation of various lignocellulosic substrates. mSystems. 2023;8(4):
Article e0128322.
</cite> [<a href="https://doi.org/10.1128/msystems.01283-22" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10469889/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37417747/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=mSystems&amp;title=Unraveling%20the%20roles%20of%20coastal%20bacterial%20consortia%20in%20degradation%20of%20various%20lignocellulosic%20substrates&amp;volume=8&amp;issue=4&amp;publication_year=2023&amp;pages=Article%20e0128322&amp;pmid=37417747&amp;doi=10.1128/msystems.01283-22&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B27">
<span class="label">27.</span><cite>Bao H, Wu Y, Zhan X, Wang X, Spencer RGM, Hernes PJJ, Feng X, Lee L, Huang J, Zhang J, et al. 
Global riverine export of dissolved lignin constrained by hydrology, geomorphology, and land-cover. Glob Biogeochem Cycles. 2023;37(4):
Article e2022GB007607.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Glob%20Biogeochem%20Cycles&amp;title=Global%20riverine%20export%20of%20dissolved%20lignin%20constrained%20by%20hydrology,%20geomorphology,%20and%20land-cover&amp;volume=37&amp;issue=4&amp;publication_year=2023&amp;pages=Article%20e2022GB007607&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B28">
<span class="label">28.</span><cite>Spencer RGM, Stubbins A, Hernes PJ, Baker A, Mopper K, Aufdenkampe AK, Dyda RY, Mwamba VL, Mangangu AM, Wabakanghanzi JN, et al. 
Photochemical degradation of dissolved organic matter and dissolved lignin phenols from the Congo River. J Geophys Res Biogeosci. 2009;114(G3):
Article 2009JG000968.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Geophys%20Res%20Biogeosci&amp;title=Photochemical%20degradation%20of%20dissolved%20organic%20matter%20and%20dissolved%20lignin%20phenols%20from%20the%20Congo%20River&amp;volume=114&amp;issue=G3&amp;publication_year=2009&amp;pages=Article%202009JG000968&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B29">
<span class="label">29.</span><cite>Dittmar T, Hertkorn N, Kattner G, Lara RJ. 
Mangroves, a major source of dissolved organic carbon to the oceans. Glob Biogeochem Cycles. 2006;20(1):
Article 2005GB002570.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Glob%20Biogeochem%20Cycles&amp;title=Mangroves,%20a%20major%20source%20of%20dissolved%20organic%20carbon%20to%20the%20oceans&amp;volume=20&amp;issue=1&amp;publication_year=2006&amp;pages=Article%202005GB002570&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B30">
<span class="label">30.</span><cite>Ahmad M, Taylor CR, Pink D, Burton K, Eastwood D, Bending GD, Bugg TD. 
Development of novel assays for lignin degradation: Comparative analysis of bacterial and fungal lignin degraders. Mol BioSyst. 2010;6(5):815–821.
</cite> [<a href="https://doi.org/10.1039/b908966g" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20567767/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol%20BioSyst&amp;title=Development%20of%20novel%20assays%20for%20lignin%20degradation:%20Comparative%20analysis%20of%20bacterial%20and%20fungal%20lignin%20degraders&amp;volume=6&amp;issue=5&amp;publication_year=2010&amp;pages=815-821&amp;pmid=20567767&amp;doi=10.1039/b908966g&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B31">
<span class="label">31.</span><cite>Brink DP, Ravi K, Lidén G, Gorwa-Grauslund MF. 
Mapping the diversity of microbial lignin catabolism: Experiences from the eLignin database. Appl Microbiol Biotechnol. 2019;103(10):3979–4002.
</cite> [<a href="https://doi.org/10.1007/s00253-019-09692-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6486533/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30963208/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Appl%20Microbiol%20Biotechnol&amp;title=Mapping%20the%20diversity%20of%20microbial%20lignin%20catabolism:%20Experiences%20from%20the%20eLignin%20database&amp;volume=103&amp;issue=10&amp;publication_year=2019&amp;pages=3979-4002&amp;pmid=30963208&amp;doi=10.1007/s00253-019-09692-4&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B32">
<span class="label">32.</span><cite>Bugg TD, Ahmad M, Hardiman EM, Rahmanpour R. 
Pathways for degradation of lignin in bacteria and fungi. Nat Prod Rep. 2011;28(12):1883–1896.
</cite> [<a href="https://doi.org/10.1039/c1np00042j" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21918777/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Prod%20Rep&amp;title=Pathways%20for%20degradation%20of%20lignin%20in%20bacteria%20and%20fungi&amp;volume=28&amp;issue=12&amp;publication_year=2011&amp;pages=1883-1896&amp;pmid=21918777&amp;doi=10.1039/c1np00042j&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B33">
<span class="label">33.</span><cite>Zhang H, Wang M, Wang H, Chen H, Cao L, Zhong Z, Lian C, Zhou L, Li C. 
Metagenome sequencing and 768 microbial genomes from cold seep in South China Sea. Sci Data. 2022;9(1):
Article 480.
</cite> [<a href="https://doi.org/10.1038/s41597-022-01586-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9357000/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35933411/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sci%20Data&amp;title=Metagenome%20sequencing%20and%20768%20microbial%20genomes%20from%20cold%20seep%20in%20South%20China%20Sea&amp;volume=9&amp;issue=1&amp;publication_year=2022&amp;pages=Article%20480&amp;pmid=35933411&amp;doi=10.1038/s41597-022-01586-x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B34">
<span class="label">34.</span><cite>Brown ME, Walker MC, Nakashige TG, Iavarone AT, Chang MCY. 
Discovery and characterization of heme enzymes from unsequenced bacteria: Application to microbial lignin degradation. J Am Chem Soc. 2011;133(45):18006–18009.
</cite> [<a href="https://doi.org/10.1021/ja203972q" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21671563/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Am%20Chem%20Soc&amp;title=Discovery%20and%20characterization%20of%20heme%20enzymes%20from%20unsequenced%20bacteria:%20Application%20to%20microbial%20lignin%20degradation&amp;volume=133&amp;issue=45&amp;publication_year=2011&amp;pages=18006-18009&amp;pmid=21671563&amp;doi=10.1021/ja203972q&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B35">
<span class="label">35.</span><cite>Yamada Y, Wang J, Kawagishi H, Hirai H. 
Improvement of ligninolytic properties by recombinant expression of glyoxal oxidase gene in hyper lignin-degrading fungus Phanerochaete sordida YK-624. Biosci Biotechnol Biochem. 2014;78(12):2128–2133.
</cite> [<a href="https://doi.org/10.1080/09168451.2014.946398" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25117933/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biosci%20Biotechnol%20Biochem&amp;title=Improvement%20of%20ligninolytic%20properties%20by%20recombinant%20expression%20of%20glyoxal%20oxidase%20gene%20in%20hyper%20lignin-degrading%20fungus%20Phanerochaete%20sordida%20YK-624&amp;volume=78&amp;issue=12&amp;publication_year=2014&amp;pages=2128-2133&amp;pmid=25117933&amp;doi=10.1080/09168451.2014.946398&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B36">
<span class="label">36.</span><cite>Ma J, Zhang K, Liao H, Hector SB, Shi X, Li J, Liu B, Xu T, Tong C, Liu X, et al. 
Genomic and secretomic insight into lignocellulolytic system of an endophytic bacterium Pantoea ananatis Sd-1. Biotechnol Biofuels. 2016;9(1):
Article 25.</cite> [<a href="https://doi.org/10.1186/s13068-016-0439-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4736469/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26839588/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biotechnol%20Biofuels&amp;title=Genomic%20and%20secretomic%20insight%20into%20lignocellulolytic%20system%20of%20an%20endophytic%20bacterium%20Pantoea%20ananatis%20Sd-1&amp;volume=9&amp;issue=1&amp;publication_year=2016&amp;pages=Article%2025&amp;pmid=26839588&amp;doi=10.1186/s13068-016-0439-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B37">
<span class="label">37.</span><cite>Atiwesh G, Parrish CC, Banoub J, Le TT. 
Lignin degradation by microorganisms: A review. Biotechnol Prog. 2022;38(2):
Article e3226.
</cite> [<a href="https://doi.org/10.1002/btpr.3226" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34854261/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biotechnol%20Prog&amp;title=Lignin%20degradation%20by%20microorganisms:%20A%20review&amp;volume=38&amp;issue=2&amp;publication_year=2022&amp;pages=Article%20e3226&amp;pmid=34854261&amp;doi=10.1002/btpr.3226&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B38">
<span class="label">38.</span><cite>Peeters F, Hofmann H. 
Oxic methanogenesis is only a minor source of lake-wide diffusive CH<sub>4</sub> emissions from lakes. Nat Commun. 2021;12(1):
Article 1206.</cite> [<a href="https://doi.org/10.1038/s41467-021-21215-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7900097/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33619253/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Commun&amp;title=Oxic%20methanogenesis%20is%20only%20a%20minor%20source%20of%20lake-wide%20diffusive%20CH4%20emissions%20from%20lakes&amp;volume=12&amp;issue=1&amp;publication_year=2021&amp;pages=Article%201206&amp;pmid=33619253&amp;doi=10.1038/s41467-021-21215-2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B39">
<span class="label">39.</span><cite>Mallinson SJB, Machovina MM, Silveira RL, Garcia-Borràs M, Gallup N, Johnson CW, Allen MD, Skaf MS, Crowley MF, Neidle EL, et al. 
A promiscuous cytochrome P450 aromatic O-demethylase for lignin bioconversion. Nat Commun. 2018;9(1):
Article 2487.
</cite> [<a href="https://doi.org/10.1038/s41467-018-04878-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6021390/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29950589/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Commun&amp;title=A%20promiscuous%20cytochrome%20P450%20aromatic%20O-demethylase%20for%20lignin%20bioconversion&amp;volume=9&amp;issue=1&amp;publication_year=2018&amp;pages=Article%202487&amp;pmid=29950589&amp;doi=10.1038/s41467-018-04878-2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B40">
<span class="label">40.</span><cite>Qian L, Yu X, Zhou J, Gu H, Ding J, Peng Y, He Q, Tian Y, Liu J, Wang S, et al. 
MCycDB: A curated database for comprehensively profiling methane cycling processes of environmental microbiomes. Mol Ecol Resour. 2022;22(5):1803–1823.
</cite> [<a href="https://doi.org/10.1111/1755-0998.13589" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35080120/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol%20Ecol%20Resour&amp;title=MCycDB:%20A%20curated%20database%20for%20comprehensively%20profiling%20methane%20cycling%20processes%20of%20environmental%20microbiomes&amp;volume=22&amp;issue=5&amp;publication_year=2022&amp;pages=1803-1823&amp;pmid=35080120&amp;doi=10.1111/1755-0998.13589&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B41">
<span class="label">41.</span><cite>Dong X, Rattray JE, Campbell DC, Webb J, Chakraborty A, Adebayo O, Matthews S, Li C, Fowler M, Morrison NM, et al. 
Thermogenic hydrocarbon biodegradation by diverse depth-stratified microbial populations at a Scotian Basin cold seep. Nat Commun. 2020;11(1):
Article 5825.
</cite> [<a href="https://doi.org/10.1038/s41467-020-19648-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7673041/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33203858/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Commun&amp;title=Thermogenic%20hydrocarbon%20biodegradation%20by%20diverse%20depth-stratified%20microbial%20populations%20at%20a%20Scotian%20Basin%20cold%20seep&amp;volume=11&amp;issue=1&amp;publication_year=2020&amp;pages=Article%205825&amp;pmid=33203858&amp;doi=10.1038/s41467-020-19648-2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B42">
<span class="label">42.</span><cite>Han Y, Zhang C, Zhao Z, Peng Y, Liao J, Jiang Q, Liu Q, Shao Z, Dong X. 
A comprehensive genomic catalog from global cold seeps. Sci Data. 2023;10(1):
Article 596.
</cite> [<a href="https://doi.org/10.1038/s41597-023-02521-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10491686/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37684262/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sci%20Data&amp;title=A%20comprehensive%20genomic%20catalog%20from%20global%20cold%20seeps&amp;volume=10&amp;issue=1&amp;publication_year=2023&amp;pages=Article%20596&amp;pmid=37684262&amp;doi=10.1038/s41597-023-02521-4&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B43">
<span class="label">43.</span><cite>Quan Q, Liu J, Li C, Ke Z, Tan Y. 
Insights into prokaryotic communities and their potential functions in biogeochemical cycles in cold seep. mSphere. 2024;9(10):
Article e0054924.
</cite> [<a href="https://doi.org/10.1128/msphere.00549-24" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11524163/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39269181/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=mSphere&amp;title=Insights%20into%20prokaryotic%20communities%20and%20their%20potential%20functions%20in%20biogeochemical%20cycles%20in%20cold%20seep&amp;volume=9&amp;issue=10&amp;publication_year=2024&amp;pages=Article%20e0054924&amp;pmid=39269181&amp;doi=10.1128/msphere.00549-24&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B44">
<span class="label">44.</span><cite>Chen J, Jia Y, Sun Y, Liu K, Zhou C, Liu C, Li D, Liu G, Zhang C, Yang T, et al. 
Global marine microbial diversity and its potential in bioprospecting. Nature. 2024;633(8029):371–379.
</cite> [<a href="https://doi.org/10.1038/s41586-024-07891-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11390488/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39232160/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Global%20marine%20microbial%20diversity%20and%20its%20potential%20in%20bioprospecting&amp;volume=633&amp;issue=8029&amp;publication_year=2024&amp;pages=371-379&amp;pmid=39232160&amp;doi=10.1038/s41586-024-07891-2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B45">
<span class="label">45.</span><cite>Levy-Booth DJ, Navas LE, Fetherolf MM, Liu LY, Dalhuisen T, Renneckar S, Eltis LD, Mohn WW. 
Discovery of lignin-transforming bacteria and enzymes in thermophilic environments using stable isotope probing. ISME J. 2022;16(8):1944–1956.
</cite> [<a href="https://doi.org/10.1038/s41396-022-01241-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9296663/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35501417/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=ISME%20J&amp;title=Discovery%20of%20lignin-transforming%20bacteria%20and%20enzymes%20in%20thermophilic%20environments%20using%20stable%20isotope%20probing&amp;volume=16&amp;issue=8&amp;publication_year=2022&amp;pages=1944-1956&amp;pmid=35501417&amp;doi=10.1038/s41396-022-01241-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B46">
<span class="label">46.</span><cite>Zhao L, Zhang J, Zhao D, Jia L, Qin B, Cao X, Zang L, Lu F, Liu F. 
Biological degradation of lignin: A critical review on progress and perspectives. Ind Crop Prod. 2022;188:
Article 115715.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Ind%20Crop%20Prod&amp;title=Biological%20degradation%20of%20lignin:%20A%20critical%20review%20on%20progress%20and%20perspectives&amp;volume=188&amp;publication_year=2022&amp;pages=Article%20115715&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B47">
<span class="label">47.</span><cite>Zhang R, Wang J, Milligan S, Yan Y. 
Microbial utilization of lignin-derived aromatics via a synthetic catechol meta-cleavage pathway. Green Chem. 2021;23(20):8238–8250.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Green%20Chem&amp;title=Microbial%20utilization%20of%20lignin-derived%20aromatics%20via%20a%20synthetic%20catechol%20meta-cleavage%20pathway&amp;volume=23&amp;issue=20&amp;publication_year=2021&amp;pages=8238-8250&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B48">
<span class="label">48.</span><cite>Sirim D, Wagner F, Wang L, Schmid RD, Pleiss J. 
The laccase engineering database: A classification and analysis system for laccases and related multicopper oxidases. Database. 2011;2011:
Article bar006.
</cite> [<a href="https://doi.org/10.1093/database/bar006" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3077825/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21498547/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Database&amp;title=The%20laccase%20engineering%20database:%20A%20classification%20and%20analysis%20system%20for%20laccases%20and%20related%20multicopper%20oxidases&amp;volume=2011&amp;publication_year=2011&amp;pages=Article%20bar006&amp;pmid=21498547&amp;doi=10.1093/database/bar006&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B49">
<span class="label">49.</span><cite>Wang X, Lin L, Zhou J. 
Links among extracellular enzymes, lignin degradation and cell growth establish the models to identify marine lignin-utilizing bacteria. Environ Microbiol. 2021;23(1):160–173.
</cite> [<a href="https://doi.org/10.1111/1462-2920.15289" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33107668/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Environ%20Microbiol&amp;title=Links%20among%20extracellular%20enzymes,%20lignin%20degradation%20and%20cell%20growth%20establish%20the%20models%20to%20identify%20marine%20lignin-utilizing%20bacteria&amp;volume=23&amp;issue=1&amp;publication_year=2021&amp;pages=160-173&amp;pmid=33107668&amp;doi=10.1111/1462-2920.15289&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B50">
<span class="label">50.</span><cite>Marinović M, Nousiainen P, Dilokpimol A, Kontro J, Moore R, Sipilä J, Vries RP, Mäkelä MR, Hildén K. 
Selective cleavage of lignin β-O-4 aryl ether bond by β-etherase of the white-rot fungus Dichomitus squalens. ACS Sustain Chem Eng. 2018;6(3):2878–2882.
</cite> [<a href="https://doi.org/10.1021/acssuschemeng.7b03619" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6156110/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30271687/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=ACS%20Sustain%20Chem%20Eng&amp;title=Selective%20cleavage%20of%20lignin%20%CE%B2-O-4%20aryl%20ether%20bond%20by%20%CE%B2-etherase%20of%20the%20white-rot%20fungus%20Dichomitus%20squalens&amp;volume=6&amp;issue=3&amp;publication_year=2018&amp;pages=2878-2882&amp;pmid=30271687&amp;doi=10.1021/acssuschemeng.7b03619&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B51">
<span class="label">51.</span><cite>Pereira JH, Heins RA, Gall DL, McAndrew RP, Deng K, Holland KC, Donohue TJ, Noguera DR, Simmons BA, Sale KL, et al. 
Structural and biochemical characterization of the early and late enzymes in the lignin β-aryl ether cleavage pathway from Sphingobium sp. SYK-6*. J Biol Chem. 2016;291(19):10228–10238.
</cite> [<a href="https://doi.org/10.1074/jbc.M115.700427" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4858972/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26940872/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&amp;title=Structural%20and%20biochemical%20characterization%20of%20the%20early%20and%20late%20enzymes%20in%20the%20lignin%20%CE%B2-aryl%20ether%20cleavage%20pathway%20from%20Sphingobium%20sp.%20SYK-6*&amp;volume=291&amp;issue=19&amp;publication_year=2016&amp;pages=10228-10238&amp;pmid=26940872&amp;doi=10.1074/jbc.M115.700427&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B52">
<span class="label">52.</span><cite>Hendel B, Sinsabaugh RL, Marxsen J. Lignin-degrading enzymes: Phenoloxidase and peroxidase. In: Bärlocher F, Gessner MO, Graça MAS, editors. <em>Methods to study litter decomposition: A practical guide</em>. Cham: Springer International Publishing; 2020. p. 425–431. </cite> [<a href="https://scholar.google.com/scholar_lookup?Hendel%20B,%20Sinsabaugh%20RL,%20Marxsen%20J.%20Lignin-degrading%20enzymes:%20Phenoloxidase%20and%20peroxidase.%20In:%20B%C3%A4rlocher%20F,%20Gessner%20MO,%20Gra%C3%A7a%20MAS,%20editors.%20Methods%20to%20study%20litter%20decomposition:%20A%20practical%20guide.%20Cham:%20Springer%20International%20Publishing;%202020.%20p.%20425%E2%80%93431." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B53">
<span class="label">53.</span><cite>Chen J, Lin L, Tu Q, Peng Q, Wang X, Liang C, Zhou J, Yu X. 
Metagenomic-based discovery and comparison of the lignin degrading potential of microbiomes in aquatic and terrestrial ecosystems via the LCdb database. Mol Ecol Resour. 2024;24(5):
Article e13950.
</cite> [<a href="https://doi.org/10.1111/1755-0998.13950" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38567644/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol%20Ecol%20Resour&amp;title=Metagenomic-based%20discovery%20and%20comparison%20of%20the%20lignin%20degrading%20potential%20of%20microbiomes%20in%20aquatic%20and%20terrestrial%20ecosystems%20via%20the%20LCdb%20database&amp;volume=24&amp;issue=5&amp;publication_year=2024&amp;pages=Article%20e13950&amp;pmid=38567644&amp;doi=10.1111/1755-0998.13950&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B54">
<span class="label">54.</span><cite>Dittmar T, Lara RJ. 
Molecular evidence for lignin degradation in sulfate-reducing mangrove sediments (Amazônia, Brazil). Geochim Cosmochim Acta. 2001;65(9):1417–1428.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Geochim%20Cosmochim%20Acta&amp;title=Molecular%20evidence%20for%20lignin%20degradation%20in%20sulfate-reducing%20mangrove%20sediments%20(Amaz%C3%B4nia,%20Brazil)&amp;volume=65&amp;issue=9&amp;publication_year=2001&amp;pages=1417-1428&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B55">
<span class="label">55.</span><cite>Li J, Duan L, Wu Y, Ahmad M, Yin L, Luo X, Wang X, Fang B, Li S, Huang L, et al. 
Unraveling microbe-mediated degradation of lignin and lignin-derived aromatic fragments in the Pearl River Estuary sediments. Chemosphere. 2022;296:
Article 133995.
</cite> [<a href="https://doi.org/10.1016/j.chemosphere.2022.133995" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35176304/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Chemosphere&amp;title=Unraveling%20microbe-mediated%20degradation%20of%20lignin%20and%20lignin-derived%20aromatic%20fragments%20in%20the%20Pearl%20River%20Estuary%20sediments&amp;volume=296&amp;publication_year=2022&amp;pages=Article%20133995&amp;pmid=35176304&amp;doi=10.1016/j.chemosphere.2022.133995&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B56">
<span class="label">56.</span><cite>Ma W, Lin L, Peng Q. 
Origin, selection, and succession of coastal intertidal zone-derived bacterial communities associated with the degradation of various lignocellulose substrates. Microb Ecol. 2023;86(3):1589–1603.
</cite> [<a href="https://doi.org/10.1007/s00248-023-02170-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36717391/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Microb%20Ecol&amp;title=Origin,%20selection,%20and%20succession%20of%20coastal%20intertidal%20zone-derived%20bacterial%20communities%20associated%20with%20the%20degradation%20of%20various%20lignocellulose%20substrates&amp;volume=86&amp;issue=3&amp;publication_year=2023&amp;pages=1589-1603&amp;pmid=36717391&amp;doi=10.1007/s00248-023-02170-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B57">
<span class="label">57.</span><cite>Levy-Booth DJ, Hashimi A, Roccor R, Liu L-Y, Renneckar S, Eltis LD, Mohn WW. 
Genomics and metatranscriptomics of biogeochemical cycling and degradation of lignin-derived aromatic compounds in thermal swamp sediment. ISME J. 2021;15(3):879–893.
</cite> [<a href="https://doi.org/10.1038/s41396-020-00820-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8027834/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33139871/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=ISME%20J&amp;title=Genomics%20and%20metatranscriptomics%20of%20biogeochemical%20cycling%20and%20degradation%20of%20lignin-derived%20aromatic%20compounds%20in%20thermal%20swamp%20sediment&amp;volume=15&amp;issue=3&amp;publication_year=2021&amp;pages=879-893&amp;pmid=33139871&amp;doi=10.1038/s41396-020-00820-x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B58">
<span class="label">58.</span><cite>Ferrer A, Heath KD, Canam T, Flores HD, Dalling JW. 
Contribution of fungal and invertebrate communities to wood decay in tropical terrestrial and aquatic habitats. Ecology. 2020;101(9):
Article e03097.
</cite> [<a href="https://doi.org/10.1002/ecy.3097" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32415862/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Ecology&amp;title=Contribution%20of%20fungal%20and%20invertebrate%20communities%20to%20wood%20decay%20in%20tropical%20terrestrial%20and%20aquatic%20habitats&amp;volume=101&amp;issue=9&amp;publication_year=2020&amp;pages=Article%20e03097&amp;pmid=32415862&amp;doi=10.1002/ecy.3097&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B59">
<span class="label">59.</span><cite>Chen J, Luo Y, García-Palacios P, Cao J, Dacal M, Zhou X, Li J, Xia J, Niu S, Yang H, et al. 
Differential responses of carbon-degrading enzyme activities to warming: Implications for soil respiration. Glob Chang Biol. 2018;24(10):4816–4826.
</cite> [<a href="https://doi.org/10.1111/gcb.14394" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29999577/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Glob%20Chang%20Biol&amp;title=Differential%20responses%20of%20carbon-degrading%20enzyme%20activities%20to%20warming:%20Implications%20for%20soil%20respiration&amp;volume=24&amp;issue=10&amp;publication_year=2018&amp;pages=4816-4826&amp;pmid=29999577&amp;doi=10.1111/gcb.14394&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B60">
<span class="label">60.</span><cite>Woo HL, Hazen TC, Simmons BA, DeAngelis KM. 
Enzyme activities of aerobic lignocellulolytic bacteria isolated from wet tropical forest soils. Syst Appl Microbiol. 2014;37(1):60–67.
</cite> [<a href="https://doi.org/10.1016/j.syapm.2013.10.001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24238986/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Syst%20Appl%20Microbiol&amp;title=Enzyme%20activities%20of%20aerobic%20lignocellulolytic%20bacteria%20isolated%20from%20wet%20tropical%20forest%20soils&amp;volume=37&amp;issue=1&amp;publication_year=2014&amp;pages=60-67&amp;pmid=24238986&amp;doi=10.1016/j.syapm.2013.10.001&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B61">
<span class="label">61.</span><cite>Wilhelm RC, Singh R, Eltis LD, Mohn WW. 
Bacterial contributions to delignification and lignocellulose degradation in forest soils with metagenomic and quantitative stable isotope probing. ISME J. 2019;13(2):413–429.
</cite> [<a href="https://doi.org/10.1038/s41396-018-0279-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6331573/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30258172/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=ISME%20J&amp;title=Bacterial%20contributions%20to%20delignification%20and%20lignocellulose%20degradation%20in%20forest%20soils%20with%20metagenomic%20and%20quantitative%20stable%20isotope%20probing&amp;volume=13&amp;issue=2&amp;publication_year=2019&amp;pages=413-429&amp;pmid=30258172&amp;doi=10.1038/s41396-018-0279-6&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B62">
<span class="label">62.</span><cite>Janvier M, Grimont PA. 
The genus Methylophaga, a new line of descent within phylogenetic branch gamma of Proteobacteria. Res Microbiol. 1995;146(7):543–550.
</cite> [<a href="https://doi.org/10.1016/0923-2508(96)80560-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/8577995/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Res%20Microbiol&amp;title=The%20genus%20Methylophaga,%20a%20new%20line%20of%20descent%20within%20phylogenetic%20branch%20gamma%20of%20Proteobacteria&amp;volume=146&amp;issue=7&amp;publication_year=1995&amp;pages=543-550&amp;pmid=8577995&amp;doi=10.1016/0923-2508(96)80560-2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B63">
<span class="label">63.</span><cite>Fujii K, Nakada Y, Umezawa K, Yoshida M, Shibata M, Hayakawa C, Inagaki Y, Kosaki T, Hangs R. 
A comparison of lignin-degrading enzyme activities in forest floor layers across a global climatic gradient. Soil Ecol Lett. 2020;2(4):281–294.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Soil%20Ecol%20Lett&amp;title=A%20comparison%20of%20lignin-degrading%20enzyme%20activities%20in%20forest%20floor%20layers%20across%20a%20global%20climatic%20gradient&amp;volume=2&amp;issue=4&amp;publication_year=2020&amp;pages=281-294&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B64">
<span class="label">64.</span><cite>Boyle CD, Kropp BR, Reid ID. 
Solubilization and mineralization of lignin by white rot fungi. Appl Environ Microbiol. 1992;58(10):3217–3224.
</cite> [<a href="https://doi.org/10.1128/aem.58.10.3217-3224.1992" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC183083/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16348781/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Appl%20Environ%20Microbiol&amp;title=Solubilization%20and%20mineralization%20of%20lignin%20by%20white%20rot%20fungi&amp;volume=58&amp;issue=10&amp;publication_year=1992&amp;pages=3217-3224&amp;pmid=16348781&amp;doi=10.1128/aem.58.10.3217-3224.1992&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B65">
<span class="label">65.</span><cite>Chio C, Sain M, Qin W. 
Lignin utilization: A review of lignin depolymerization from various aspects. Renew Sust Energ Rev. 2019;107:232–249.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Renew%20Sust%20Energ%20Rev&amp;title=Lignin%20utilization:%20A%20review%20of%20lignin%20depolymerization%20from%20various%20aspects&amp;volume=107&amp;publication_year=2019&amp;pages=232-249&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B66">
<span class="label">66.</span><cite>Angst G, Mueller KE, Nierop KGJ, Simpson MJ. 
Plant- or microbial-derived? A review on the molecular composition of stabilized soil organic matter. Soil Biol Biochem. 2021;156:
Article 108189.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Soil%20Biol%20Biochem&amp;title=Plant-%20or%20microbial-derived?%20A%20review%20on%20the%20molecular%20composition%20of%20stabilized%20soil%20organic%20matter&amp;volume=156&amp;publication_year=2021&amp;pages=Article%20108189&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B67">
<span class="label">67.</span><cite>Su J, Li D, Chen Q, Li M, Su L, Luo T, Liang D, Lai G, Shuai O, Jiao C, et al. 
Anti-breast cancer enhancement of a polysaccharide from spore of Ganoderma lucidum with paclitaxel: Suppression on tumor metabolism with gut microbiota reshaping. Front Microbiol. 2018;9:
Article 3099.
</cite> [<a href="https://doi.org/10.3389/fmicb.2018.03099" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6304348/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30619178/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Front%20Microbiol&amp;title=Anti-breast%20cancer%20enhancement%20of%20a%20polysaccharide%20from%20spore%20of%20Ganoderma%20lucidum%20with%20paclitaxel:%20Suppression%20on%20tumor%20metabolism%20with%20gut%20microbiota%20reshaping&amp;volume=9&amp;publication_year=2018&amp;pages=Article%203099&amp;pmid=30619178&amp;doi=10.3389/fmicb.2018.03099&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B68">
<span class="label">68.</span><cite>Peng Q, Zhao C, Wang X, Cheng K, Wang C, Xu X, Lin L. 
Modeling bacterial interactions uncovers the importance of outliers in the coastal lignin-degrading consortium. Nat Commun. 2025;16(1):
Article 639.
</cite> [<a href="https://doi.org/10.1038/s41467-025-56012-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11733112/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39809803/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Commun&amp;title=Modeling%20bacterial%20interactions%20uncovers%20the%20importance%20of%20outliers%20in%20the%20coastal%20lignin-degrading%20consortium&amp;volume=16&amp;issue=1&amp;publication_year=2025&amp;pages=Article%20639&amp;pmid=39809803&amp;doi=10.1038/s41467-025-56012-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B69">
<span class="label">69.</span><cite>Yu T, Wu W, Liang W, Wang Y, Hou J, Chen Y, Elvert M, Hinrichs K-U, Wang F. 
Anaerobic degradation of organic carbon supports uncultured microbial populations in estuarine sediments. Microbiome. 2023;11(1):
Article 81.
</cite> [<a href="https://doi.org/10.1186/s40168-023-01531-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10116835/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37081504/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Microbiome&amp;title=Anaerobic%20degradation%20of%20organic%20carbon%20supports%20uncultured%20microbial%20populations%20in%20estuarine%20sediments&amp;volume=11&amp;issue=1&amp;publication_year=2023&amp;pages=Article%2081&amp;pmid=37081504&amp;doi=10.1186/s40168-023-01531-z&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B70">
<span class="label">70.</span><cite>Yu T, Hu H, Zeng X, Wang Y, Pan D, Deng L, Liang L, Hou J, Wang F. 
Widespread Bathyarchaeia encode a novel methyltransferase utilizing lignin-derived aromatics. mLife. 2023;2(3):272–282.
</cite> [<a href="https://doi.org/10.1002/mlf2.12082" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10989822/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38817817/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=mLife&amp;title=Widespread%20Bathyarchaeia%20encode%20a%20novel%20methyltransferase%20utilizing%20lignin-derived%20aromatics&amp;volume=2&amp;issue=3&amp;publication_year=2023&amp;pages=272-282&amp;pmid=38817817&amp;doi=10.1002/mlf2.12082&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B71">
<span class="label">71.</span><cite>Ley Y, Cheng X-Y, Ying Z-Y, Zhou N-Y, Xu Y. 
Characterization of two marine lignin-degrading consortia and the potential microbial lignin degradation network in nearshore regions. Microbiol Spectr. 2023;11(3):
Article e0442422.
</cite> [<a href="https://doi.org/10.1128/spectrum.04424-22" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10269927/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37042774/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Microbiol%20Spectr&amp;title=Characterization%20of%20two%20marine%20lignin-degrading%20consortia%20and%20the%20potential%20microbial%20lignin%20degradation%20network%20in%20nearshore%20regions&amp;volume=11&amp;issue=3&amp;publication_year=2023&amp;pages=Article%20e0442422&amp;pmid=37042774&amp;doi=10.1128/spectrum.04424-22&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B72">
<span class="label">72.</span><cite>Dong X, Zhang C, Peng Y, Zhang H, Shi L, Wei G, Hubert CRJ, Wang Y, Greening C. 
Phylogenetically and catabolically diverse diazotrophs reside in deep-sea cold seep sediments. Nat Commun. 2022;13(1):
Article 4885.
</cite> [<a href="https://doi.org/10.1038/s41467-022-32503-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9391474/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35985998/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Commun&amp;title=Phylogenetically%20and%20catabolically%20diverse%20diazotrophs%20reside%20in%20deep-sea%20cold%20seep%20sediments&amp;volume=13&amp;issue=1&amp;publication_year=2022&amp;pages=Article%204885&amp;pmid=35985998&amp;doi=10.1038/s41467-022-32503-w&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B73">
<span class="label">73.</span><cite>Sumranwanich T, Amosu E, Chankhamhaengdecha S, Phetruen T, Loktumraks W, Ounjai P, Harnvoravongchai P. 
Evaluating lignin degradation under limited oxygen conditions by bacterial isolates from forest soil. Sci Rep. 2024;14(1):
Article 13350.
</cite> [<a href="https://doi.org/10.1038/s41598-024-64237-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11164938/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38858437/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sci%20Rep&amp;title=Evaluating%20lignin%20degradation%20under%20limited%20oxygen%20conditions%20by%20bacterial%20isolates%20from%20forest%20soil&amp;volume=14&amp;issue=1&amp;publication_year=2024&amp;pages=Article%2013350&amp;pmid=38858437&amp;doi=10.1038/s41598-024-64237-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B74">
<span class="label">74.</span><cite>Bhattarai S, Cassarini C, Rene ER, Kummel S, Esposito G, Lens P. Enrichment of ANME-2 dominated anaerobic oxidation of methane coupled to sulfate reduction consortia from cold seep sediment (Ginsburg Mud Volcano, Gulf of Cadiz) in a membrane bioreactor. In: <em>Performance assessment and enrichment of anaerobic methane oxidizing microbial communities from marine sediments in bioreactors</em>. London (UK): CRC Press; 2018. p. 369–378. .</cite>
</li>
<li id="B75">
<span class="label">75.</span><cite>Tuomela M, Vikman M, Hatakka A, Itävaara M. 
Biodegradation of lignin in a compost environment: A review. Bioresour Technol. 2000;72(2):169–183.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Bioresour%20Technol&amp;title=Biodegradation%20of%20lignin%20in%20a%20compost%20environment:%20A%20review&amp;volume=72&amp;issue=2&amp;publication_year=2000&amp;pages=169-183&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B76">
<span class="label">76.</span><cite>Thevenot M, Dignac M-F, Rumpel C. 
Fate of lignins in soils: A review. Soil Biol Biochem. 2010;42(8):1200–1211.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Soil%20Biol%20Biochem&amp;title=Fate%20of%20lignins%20in%20soils:%20A%20review&amp;volume=42&amp;issue=8&amp;publication_year=2010&amp;pages=1200-1211&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B77">
<span class="label">77.</span><cite>Hedges JI, Mann DC. 
The characterization of plant tissues by their lignin oxidation products. Geochim Cosmochim Acta. 1979;43(11):1803–1807.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Geochim%20Cosmochim%20Acta&amp;title=The%20characterization%20of%20plant%20tissues%20by%20their%20lignin%20oxidation%20products&amp;volume=43&amp;issue=11&amp;publication_year=1979&amp;pages=1803-1807&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B78">
<span class="label">78.</span><cite>Goñi MA, Montgomery S. 
Alkaline CuO oxidation with a microwave digestion system: Lignin analyses of geochemical samples. Anal Chem. 2000;72(14):3116–3121.
</cite> [<a href="https://doi.org/10.1021/ac991316w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10939375/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Anal%20Chem&amp;title=Alkaline%20CuO%20oxidation%20with%20a%20microwave%20digestion%20system:%20Lignin%20analyses%20of%20geochemical%20samples&amp;volume=72&amp;issue=14&amp;publication_year=2000&amp;pages=3116-3121&amp;pmid=10939375&amp;doi=10.1021/ac991316w&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B79">
<span class="label">79.</span><cite>Yu H, Wu Y, Zhang J, Deng B, Zhu Z. 
Impact of extreme drought and the Three Gorges Dam on transport of particulate terrestrial organic carbon in the Changjiang (Yangtze), River. J Geophys Res Earth. 2011;116(F4):
Article 2011JF002012.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Geophys%20Res%20Earth&amp;title=Impact%20of%20extreme%20drought%20and%20the%20Three%20Gorges%20Dam%20on%20transport%20of%20particulate%20terrestrial%20organic%20carbon%20in%20the%20Changjiang%20(Yangtze),%20River&amp;volume=116&amp;issue=F4&amp;publication_year=2011&amp;pages=Article%202011JF002012&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B80">
<span class="label">80.</span><cite>Li X, Yao H, Yu Y, Cao Y, Tang C. 
Greenhouse gases in an urban river: Trend, isotopic evidence for underlying processes, and the impact of in-river structures. J Hydrol. 2020;591:
Article 125290.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Hydrol&amp;title=Greenhouse%20gases%20in%20an%20urban%20river:%20Trend,%20isotopic%20evidence%20for%20underlying%20processes,%20and%20the%20impact%20of%20in-river%20structures&amp;volume=591&amp;publication_year=2020&amp;pages=Article%20125290&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B81">
<span class="label">81.</span><cite>Xu M, Deng J, Xu K, Zhu T, Han L, Yan Y, Yao D, Deng H, Wang D, Sun Y, et al. 
In-depth serum proteomics reveals biomarkers of psoriasis severity and response to traditional Chinese medicine. Theranostics. 2019;9(9):2475–2488.
</cite> [<a href="https://doi.org/10.7150/thno.31144" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6526001/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31131048/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Theranostics&amp;title=In-depth%20serum%20proteomics%20reveals%20biomarkers%20of%20psoriasis%20severity%20and%20response%20to%20traditional%20Chinese%20medicine&amp;volume=9&amp;issue=9&amp;publication_year=2019&amp;pages=2475-2488&amp;pmid=31131048&amp;doi=10.7150/thno.31144&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B82">
<span class="label">82.</span><cite>Yang N, Tian C, Lv Y, Hou J, Yang Z, Xiao X, Zhang Y. 
Novel primers for 16S rRNA gene-based archaeal and bacterial community analysis in oceanic trench sediments. Appl Microbiol Biotechnol. 2022;106(7):2795–2809.
</cite> [<a href="https://doi.org/10.1007/s00253-022-11893-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35348850/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Appl%20Microbiol%20Biotechnol&amp;title=Novel%20primers%20for%2016S%20rRNA%20gene-based%20archaeal%20and%20bacterial%20community%20analysis%20in%20oceanic%20trench%20sediments&amp;volume=106&amp;issue=7&amp;publication_year=2022&amp;pages=2795-2809&amp;pmid=35348850&amp;doi=10.1007/s00253-022-11893-3&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B83">
<span class="label">83.</span><cite>Wang P, Li J, Luo X, Ahmad M, Duan L, Yin L, Fang B, Li S, Yang Y, Jiang L, et al. 
Biogeographical distributions of nitrogen-cycling functional genes in a subtropical estuary. Funct Ecol. 2022;36(1):187–201.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Funct%20Ecol&amp;title=Biogeographical%20distributions%20of%20nitrogen-cycling%20functional%20genes%20in%20a%20subtropical%20estuary&amp;volume=36&amp;issue=1&amp;publication_year=2022&amp;pages=187-201&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B84">
<span class="label">84.</span><cite>Martin M. 
CUTADAPT removes adapter sequences from high-throughput sequencing reads. EMBnet J. 2011;17(1):10–12.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=EMBnet%20J&amp;title=CUTADAPT%20removes%20adapter%20sequences%20from%20high-throughput%20sequencing%20reads&amp;volume=17&amp;issue=1&amp;publication_year=2011&amp;pages=10-12&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B85">
<span class="label">85.</span><cite>Edgar RC. UNOISE2: improved error-correction for Illumina 16S and ITS amplicon sequencing. bioRxiv. 2016. 10.1101/081257</cite> [<a href="https://doi.org/10.1101/081257" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>]</li>
<li id="B86">
<span class="label">86.</span><cite>Bokulich NA, Kaehler BD, Rideout JR, Dillon M, Bolyen E, Knight R, Huttley GA, Gregory Caporaso J. 
Optimizing taxonomic classification of marker-gene amplicon sequences with QIIME 2’s q2-feature-classifier plugin. Microbiome. 2018;6(1):
Article 90.
</cite> [<a href="https://doi.org/10.1186/s40168-018-0470-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5956843/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29773078/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Microbiome&amp;title=Optimizing%20taxonomic%20classification%20of%20marker-gene%20amplicon%20sequences%20with%20QIIME%202%E2%80%99s%20q2-feature-classifier%20plugin&amp;volume=6&amp;issue=1&amp;publication_year=2018&amp;pages=Article%2090&amp;pmid=29773078&amp;doi=10.1186/s40168-018-0470-z&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B87">
<span class="label">87.</span><cite>Bolyen E, Rideout JR, Dillon MR, Bokulich NA, Abnet CC, Al-Ghalith GA, Alexander H, Alm EJ, Arumugam M, Asnicar F, et al. 
Reproducible, interactive, scalable and extensible microbiome data science using QIIME 2. Nat Biotechnol. 2019;37(8):852–857.
</cite> [<a href="https://doi.org/10.1038/s41587-019-0209-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7015180/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31341288/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Biotechnol&amp;title=Reproducible,%20interactive,%20scalable%20and%20extensible%20microbiome%20data%20science%20using%20QIIME%202&amp;volume=37&amp;issue=8&amp;publication_year=2019&amp;pages=852-857&amp;pmid=31341288&amp;doi=10.1038/s41587-019-0209-9&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B88">
<span class="label">88.</span><cite>Luo X, Wang P, Li J, Ahmad M, Duan L, Yin L, Deng Q, Fang B, Li S, Li W. 
Viral community-wide auxiliary metabolic genes differ by lifestyles, habitats, and hosts. Microbiome. 2022;10(1):
Article 190.
</cite> [<a href="https://doi.org/10.1186/s40168-022-01384-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9636769/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36333738/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Microbiome&amp;title=Viral%20community-wide%20auxiliary%20metabolic%20genes%20differ%20by%20lifestyles,%20habitats,%20and%20hosts&amp;volume=10&amp;issue=1&amp;publication_year=2022&amp;pages=Article%20190&amp;pmid=36333738&amp;doi=10.1186/s40168-022-01384-y&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B89">
<span class="label">89.</span><cite>Chen S, Zhou Y, Chen Y, Gu J. 
Fastp: An ultra-fast all-in-one FASTQ preprocessor. Bioinformatics. 2018;34(17):i884–i890.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/bty560" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6129281/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30423086/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Fastp:%20An%20ultra-fast%20all-in-one%20FASTQ%20preprocessor&amp;volume=34&amp;issue=17&amp;publication_year=2018&amp;pages=i884-i890&amp;pmid=30423086&amp;doi=10.1093/bioinformatics/bty560&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B90">
<span class="label">90.</span><cite>Nurk S, Meleshko D, Korobeynikov A, Pevzner PA. 
metaSPAdes: A new versatile metagenomic assembler. Genome Res. 2017;27(5):824–834.
</cite> [<a href="https://doi.org/10.1101/gr.213959.116" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5411777/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28298430/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genome%20Res&amp;title=metaSPAdes:%20A%20new%20versatile%20metagenomic%20assembler&amp;volume=27&amp;issue=5&amp;publication_year=2017&amp;pages=824-834&amp;pmid=28298430&amp;doi=10.1101/gr.213959.116&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B91">
<span class="label">91.</span><cite>Hyatt D, Chen G-L, LoCascio PF, Land ML, Larimer FW, Hauser LJ. 
Prodigal: Prokaryotic gene recognition and translation initiation site identification. BMC Bioinformatics. 2010;11(1):
Article 119.
</cite> [<a href="https://doi.org/10.1186/1471-2105-11-119" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2848648/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20211023/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Bioinformatics&amp;title=Prodigal:%20Prokaryotic%20gene%20recognition%20and%20translation%20initiation%20site%20identification&amp;volume=11&amp;issue=1&amp;publication_year=2010&amp;pages=Article%20119&amp;pmid=20211023&amp;doi=10.1186/1471-2105-11-119&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B92">
<span class="label">92.</span><cite>Deng Z, Münch PC, Mreches R, McHardy AC. 
Rapid and accurate identification of ribosomal RNA sequences via deep learning. Nucleic Acids Res. 2022;50(10):
Article e60.
</cite> [<a href="https://doi.org/10.1093/nar/gkac112" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9177968/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35188571/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=Rapid%20and%20accurate%20identification%20of%20ribosomal%20RNA%20sequences%20via%20deep%20learning&amp;volume=50&amp;issue=10&amp;publication_year=2022&amp;pages=Article%20e60&amp;pmid=35188571&amp;doi=10.1093/nar/gkac112&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B93">
<span class="label">93.</span><cite>Langmead B, Salzberg SL. 
Fast gapped-read alignment with Bowtie 2. Nat Methods. 2012;9(4):357–359.
</cite> [<a href="https://doi.org/10.1038/nmeth.1923" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3322381/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22388286/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Methods&amp;title=Fast%20gapped-read%20alignment%20with%20Bowtie%202&amp;volume=9&amp;issue=4&amp;publication_year=2012&amp;pages=357-359&amp;pmid=22388286&amp;doi=10.1038/nmeth.1923&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B94">
<span class="label">94.</span><cite>Uritskiy GV, DiRuggiero J, Taylor J. 
MetaWRAP—A flexible pipeline for genome-resolved metagenomic data analysis. Microbiome. 2018;6(1):
Article 158.
</cite> [<a href="https://doi.org/10.1186/s40168-018-0541-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6138922/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30219103/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Microbiome&amp;title=MetaWRAP%E2%80%94A%20flexible%20pipeline%20for%20genome-resolved%20metagenomic%20data%20analysis&amp;volume=6&amp;issue=1&amp;publication_year=2018&amp;pages=Article%20158&amp;pmid=30219103&amp;doi=10.1186/s40168-018-0541-1&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B95">
<span class="label">95.</span><cite>Parks DH, Imelfort M, Skennerton CT, Hugenholtz P, Tyson GW. 
CheckM: Assessing the quality of microbial genomes recovered from isolates, single cells, and metagenomes. Genome Res. 2015;25(7):1043–1055.
</cite> [<a href="https://doi.org/10.1101/gr.186072.114" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4484387/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25977477/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genome%20Res&amp;title=CheckM:%20Assessing%20the%20quality%20of%20microbial%20genomes%20recovered%20from%20isolates,%20single%20cells,%20and%20metagenomes&amp;volume=25&amp;issue=7&amp;publication_year=2015&amp;pages=1043-1055&amp;pmid=25977477&amp;doi=10.1101/gr.186072.114&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B96">
<span class="label">96.</span><cite>Seemann T. 
Prokka: Rapid prokaryotic genome annotation. Bioinformatics. 2014;30(14):2068–2069.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btu153" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24642063/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Prokka:%20Rapid%20prokaryotic%20genome%20annotation&amp;volume=30&amp;issue=14&amp;publication_year=2014&amp;pages=2068-2069&amp;pmid=24642063&amp;doi=10.1093/bioinformatics/btu153&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B97">
<span class="label">97.</span><cite>Li J, Sun W, Cao Y, Wu J, Duan L, Zhang M, Luo X, Deng Q, Peng Z, Mou X, et al. 
Increased temperature enhances microbial-mediated lignin decomposition in river sediment. Microbiome. 2025;13(1):
Article 89.
</cite> [<a href="https://doi.org/10.1186/s40168-025-02076-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11959967/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40170118/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Microbiome&amp;title=Increased%20temperature%20enhances%20microbial-mediated%20lignin%20decomposition%20in%20river%20sediment&amp;volume=13&amp;issue=1&amp;publication_year=2025&amp;pages=Article%2089&amp;pmid=40170118&amp;doi=10.1186/s40168-025-02076-z&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B98">
<span class="label">98.</span><cite>Buchfink B, Xie C, Huson DH. 
Fast and sensitive protein alignment using diamond. Nat Methods. 2015;12(1):59–60.
</cite> [<a href="https://doi.org/10.1038/nmeth.3176" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25402007/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Methods&amp;title=Fast%20and%20sensitive%20protein%20alignment%20using%20diamond&amp;volume=12&amp;issue=1&amp;publication_year=2015&amp;pages=59-60&amp;pmid=25402007&amp;doi=10.1038/nmeth.3176&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B99">
<span class="label">99.</span><cite>Zhu T, Zhu Y, Xuan Y, Gao H, Cai X, Piersma SR, Pham TV, Schelfhorst T, Haas RRGD, Bijnsdorp IV, et al. 
DPHL: A DIA pan-human protein mass spectrometry library for robust biomarker discovery. Genomics Proteomics Bioinformatics. 2020;18(2):104–119.
</cite> [<a href="https://doi.org/10.1016/j.gpb.2019.11.008" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7646093/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32795611/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genomics%20Proteomics%20Bioinformatics&amp;title=DPHL:%20A%20DIA%20pan-human%20protein%20mass%20spectrometry%20library%20for%20robust%20biomarker%20discovery&amp;volume=18&amp;issue=2&amp;publication_year=2020&amp;pages=104-119&amp;pmid=32795611&amp;doi=10.1016/j.gpb.2019.11.008&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B100">
<span class="label">100.</span><cite>Lou R, Shui W. 
Acquisition and analysis of DIA-based proteomic data: A comprehensive survey in 2023. Mol Cell Proteomics. 2024;23(2):
Article 100712.
</cite> [<a href="https://doi.org/10.1016/j.mcpro.2024.100712" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10847697/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38182042/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol%20Cell%20Proteomics&amp;title=Acquisition%20and%20analysis%20of%20DIA-based%20proteomic%20data:%20A%20comprehensive%20survey%20in%202023&amp;volume=23&amp;issue=2&amp;publication_year=2024&amp;pages=Article%20100712&amp;pmid=38182042&amp;doi=10.1016/j.mcpro.2024.100712&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B101">
<span class="label">101.</span><cite>Ren Y, Yu G, Shi C, Liu L, Guo Q, Han C, Zhang D, Zhang L, Liu B, Gao H, et al. 
Majorbio Cloud: A one-stop, comprehensive bioinformatic platform for multiomics analyses. iMeta. 2022;1(2):
Article e12.
</cite> [<a href="https://doi.org/10.1002/imt2.12" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10989754/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38868573/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=iMeta&amp;title=Majorbio%20Cloud:%20A%20one-stop,%20comprehensive%20bioinformatic%20platform%20for%20multiomics%20analyses&amp;volume=1&amp;issue=2&amp;publication_year=2022&amp;pages=Article%20e12&amp;pmid=38868573&amp;doi=10.1002/imt2.12&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B102">
<span class="label">102.</span><cite>Asnicar F, Thomas AM, Beghini F, Mengoni C, Manara S, Manghi P, Zhu Q, Bolzan M, Cumbo F, May U, et al. 
Precise phylogenetic analysis of microbial isolates and genomes from metagenomes using PhyloPhlAn 3.0. Nat Commun. 2020;11(1):
Article 2500.
</cite> [<a href="https://doi.org/10.1038/s41467-020-16366-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7237447/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32427907/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Commun&amp;title=Precise%20phylogenetic%20analysis%20of%20microbial%20isolates%20and%20genomes%20from%20metagenomes%20using%20PhyloPhlAn%203.0&amp;volume=11&amp;issue=1&amp;publication_year=2020&amp;pages=Article%202500&amp;pmid=32427907&amp;doi=10.1038/s41467-020-16366-7&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B103">
<span class="label">103.</span><cite>Letunic I, Bork P. 
Interactive Tree Of Life v2: Online annotation and display of phylogenetic trees made easy. Nucleic Acids Res. 2011;39:W475–W478.
</cite> [<a href="https://doi.org/10.1093/nar/gkr201" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3125724/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21470960/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=Interactive%20Tree%20Of%20Life%20v2:%20Online%20annotation%20and%20display%20of%20phylogenetic%20trees%20made%20easy&amp;volume=39&amp;publication_year=2011&amp;pages=W475-W478&amp;pmid=21470960&amp;doi=10.1093/nar/gkr201&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="caption p">
<span>Supplementary 1</span><p>Figs. S1 to S5</p>
<p>Tables S1 to S10</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12377529/bin/research.0848.f1.zip" data-ga-action="click_feat_suppl" class="usa-link">research.0848.f1.zip</a><sup> (3.7MB, zip) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>Multi-omics sequencing datasets (16<em>S</em> rRNA gene amplicon sequencing, metagenomics, and metatranscriptomics) have been archived in the NCBI Sequence Read Archive repository under BioProject accessions PRJNA1205479-PRJNA1206677. Complementary experimental datasets supporting the conclusions of this work are maintained by the corresponding investigator team and can be accessed through formal data request protocols.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Research are provided here courtesy of <strong>American Association for the Advancement of Science (AAAS) and Science and Technology Review Publishing House</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.34133/research.0848"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/research.0848.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (5.1 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12377529/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12377529/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12377529%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12377529/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12377529/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12377529/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12377529/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12377529/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12377529/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="DXdfqIUp2gbrjVR3I57Ysj1dXPsPRBfeCZNz8eIUk0ywEHTo2dV9eoH50WBefICv">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
        <script type="module" crossorigin="" src="/static/assets/math-574fdcc6.js"></script>
    
    

    </body>
</html>
