
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Improving automated deep phenotyping through large language models using retrieval-augmented generation - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="AE53464D8AF43D1305464D0014C9636F.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="genmed">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12359922/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Genome Medicine">
<meta name="citation_title" content="Improving automated deep phenotyping through large language models using retrieval-augmented generation">
<meta name="citation_author" content="Brandon T Garcia">
<meta name="citation_author_institution" content="Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, TX 77030 USA">
<meta name="citation_author_institution" content="Medical Scientist Training Program, Baylor College of Medicine, Houston, TX 77030 USA">
<meta name="citation_author_institution" content="Genetics and Genomics Graduate Program, Baylor College of Medicine, Houston, TX 77030 USA">
<meta name="citation_author" content="Lauren Westerfield">
<meta name="citation_author_institution" content="Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, TX 77030 USA">
<meta name="citation_author_institution" content="Texas Children’s Hospital, Houston, TX 77030 USA">
<meta name="citation_author_institution" content="Department of Obstetrics and Gynecology, Baylor College of Medicine, Houston, TX 77030 USA">
<meta name="citation_author" content="Priya Yelemali">
<meta name="citation_author_institution" content="Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, TX 77030 USA">
<meta name="citation_author_institution" content="School of Medicine, Meharry Medical College, Nashville, TN 37208 USA">
<meta name="citation_author" content="Nikhita Gogate">
<meta name="citation_author_institution" content="Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, TX 77030 USA">
<meta name="citation_author_institution" content="Genetics and Genomics Graduate Program, Baylor College of Medicine, Houston, TX 77030 USA">
<meta name="citation_author" content="E Andres Rivera-Munoz">
<meta name="citation_author_institution" content="Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, TX 77030 USA">
<meta name="citation_author_institution" content="Genetics and Genomics Graduate Program, Baylor College of Medicine, Houston, TX 77030 USA">
<meta name="citation_author" content="Haowei Du">
<meta name="citation_author_institution" content="Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, TX 77030 USA">
<meta name="citation_author" content="Moez Dawood">
<meta name="citation_author_institution" content="Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, TX 77030 USA">
<meta name="citation_author_institution" content="Medical Scientist Training Program, Baylor College of Medicine, Houston, TX 77030 USA">
<meta name="citation_author_institution" content="Genetics and Genomics Graduate Program, Baylor College of Medicine, Houston, TX 77030 USA">
<meta name="citation_author_institution" content="Human Genome Sequencing Center, Baylor College of Medicine, Houston, TX 77030 USA">
<meta name="citation_author" content="Angad Jolly">
<meta name="citation_author_institution" content="Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, TX 77030 USA">
<meta name="citation_author" content="James R Lupski">
<meta name="citation_author_institution" content="Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, TX 77030 USA">
<meta name="citation_author_institution" content="Texas Children’s Hospital, Houston, TX 77030 USA">
<meta name="citation_author_institution" content="Human Genome Sequencing Center, Baylor College of Medicine, Houston, TX 77030 USA">
<meta name="citation_author_institution" content="Department of Pediatrics, Baylor College of Medicine, Houston, TX 77030 USA">
<meta name="citation_author" content="Jennifer E Posey">
<meta name="citation_author_institution" content="Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, TX 77030 USA">
<meta name="citation_author_institution" content="Department of Pediatrics, Columbia University Vagelos College of Physicians and Surgeons, Columbia University Irving Medical Center, New York, NY 10032 USA">
<meta name="citation_author_institution" content="Department of Medicine, Columbia University Vagelos College of Physicians and Surgeons, Columbia University Irving Medical Center, New York, NY 10032 USA">
<meta name="citation_publication_date" content="2025 Aug 18">
<meta name="citation_volume" content="17">
<meta name="citation_firstpage" content="91">
<meta name="citation_doi" content="10.1186/s13073-025-01521-w">
<meta name="citation_pmid" content="40826123">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12359922/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12359922/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12359922/pdf/13073_2025_Article_1521.pdf">
<meta name="description" content="Diagnosing rare genetic disorders relies on precise phenotypic and genotypic analysis, with the Human Phenotype Ontology (HPO) providing a standardized language for capturing clinical phenotypes. Rule-based HPO extraction tools use concept ...">
<meta name="og:title" content="Improving automated deep phenotyping through large language models using retrieval-augmented generation">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Diagnosing rare genetic disorders relies on precise phenotypic and genotypic analysis, with the Human Phenotype Ontology (HPO) providing a standardized language for capturing clinical phenotypes. Rule-based HPO extraction tools use concept ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12359922/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12359922">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1186/s13073-025-01521-w"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/13073_2025_Article_1521.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12359922%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12359922/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12359922/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12359922/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-genmed.png" alt="Genome Medicine logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Genome Medicine" title="Link to Genome Medicine" shape="default" href="https://genomemedicine.biomedcentral.com/" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Genome Med</button></div>. 2025 Aug 18;17:91. doi: <a href="https://doi.org/10.1186/s13073-025-01521-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1186/s13073-025-01521-w</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Genome%20Med%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Genome%20Med%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Genome%20Med%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Genome%20Med%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Improving automated deep phenotyping through large language models using retrieval-augmented generation</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Garcia%20BT%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Brandon T Garcia</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Brandon T Garcia</span></h3>
<div class="p">
<sup>1</sup>Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, TX 77030 USA </div>
<div class="p">
<sup>2</sup>Medical Scientist Training Program, Baylor College of Medicine, Houston, TX 77030 USA </div>
<div class="p">
<sup>3</sup>Genetics and Genomics Graduate Program, Baylor College of Medicine, Houston, TX 77030 USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Garcia%20BT%22%5BAuthor%5D" class="usa-link"><span class="name western">Brandon T Garcia</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Westerfield%20L%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Lauren Westerfield</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Lauren Westerfield</span></h3>
<div class="p">
<sup>1</sup>Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, TX 77030 USA </div>
<div class="p">
<sup>4</sup>Texas Children’s Hospital, Houston, TX 77030 USA </div>
<div class="p">
<sup>5</sup>Department of Obstetrics and Gynecology, Baylor College of Medicine, Houston, TX 77030 USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Westerfield%20L%22%5BAuthor%5D" class="usa-link"><span class="name western">Lauren Westerfield</span></a>
</div>
</div>
<sup>1,</sup><sup>4,</sup><sup>5</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yelemali%20P%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Priya Yelemali</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Priya Yelemali</span></h3>
<div class="p">
<sup>1</sup>Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, TX 77030 USA </div>
<div class="p">
<sup>6</sup>School of Medicine, Meharry Medical College, Nashville, TN 37208 USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yelemali%20P%22%5BAuthor%5D" class="usa-link"><span class="name western">Priya Yelemali</span></a>
</div>
</div>
<sup>1,</sup><sup>6</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gogate%20N%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Nikhita Gogate</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Nikhita Gogate</span></h3>
<div class="p">
<sup>1</sup>Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, TX 77030 USA </div>
<div class="p">
<sup>3</sup>Genetics and Genomics Graduate Program, Baylor College of Medicine, Houston, TX 77030 USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gogate%20N%22%5BAuthor%5D" class="usa-link"><span class="name western">Nikhita Gogate</span></a>
</div>
</div>
<sup>1,</sup><sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Rivera-Munoz%20EA%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">E Andres Rivera-Munoz</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">E Andres Rivera-Munoz</span></h3>
<div class="p">
<sup>1</sup>Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, TX 77030 USA </div>
<div class="p">
<sup>3</sup>Genetics and Genomics Graduate Program, Baylor College of Medicine, Houston, TX 77030 USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Rivera-Munoz%20EA%22%5BAuthor%5D" class="usa-link"><span class="name western">E Andres Rivera-Munoz</span></a>
</div>
</div>
<sup>1,</sup><sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Du%20H%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Haowei Du</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Haowei Du</span></h3>
<div class="p">
<sup>1</sup>Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, TX 77030 USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Du%20H%22%5BAuthor%5D" class="usa-link"><span class="name western">Haowei Du</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Dawood%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Moez Dawood</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Moez Dawood</span></h3>
<div class="p">
<sup>1</sup>Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, TX 77030 USA </div>
<div class="p">
<sup>2</sup>Medical Scientist Training Program, Baylor College of Medicine, Houston, TX 77030 USA </div>
<div class="p">
<sup>3</sup>Genetics and Genomics Graduate Program, Baylor College of Medicine, Houston, TX 77030 USA </div>
<div class="p">
<sup>7</sup>Human Genome Sequencing Center, Baylor College of Medicine, Houston, TX 77030 USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Dawood%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Moez Dawood</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>3,</sup><sup>7</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Jolly%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Angad Jolly</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Angad Jolly</span></h3>
<div class="p">
<sup>1</sup>Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, TX 77030 USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Jolly%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Angad Jolly</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lupski%20JR%22%5BAuthor%5D" class="usa-link" aria-describedby="id9"><span class="name western">James R Lupski</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">James R Lupski</span></h3>
<div class="p">
<sup>1</sup>Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, TX 77030 USA </div>
<div class="p">
<sup>4</sup>Texas Children’s Hospital, Houston, TX 77030 USA </div>
<div class="p">
<sup>7</sup>Human Genome Sequencing Center, Baylor College of Medicine, Houston, TX 77030 USA </div>
<div class="p">
<sup>8</sup>Department of Pediatrics, Baylor College of Medicine, Houston, TX 77030 USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lupski%20JR%22%5BAuthor%5D" class="usa-link"><span class="name western">James R Lupski</span></a>
</div>
</div>
<sup>1,</sup><sup>4,</sup><sup>7,</sup><sup>8</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Posey%20JE%22%5BAuthor%5D" class="usa-link" aria-describedby="id10"><span class="name western">Jennifer E Posey</span></a><div hidden="hidden" id="id10">
<h3><span class="name western">Jennifer E Posey</span></h3>
<div class="p">
<sup>1</sup>Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, TX 77030 USA </div>
<div class="p">
<sup>9</sup>Department of Pediatrics, Columbia University Vagelos College of Physicians and Surgeons, Columbia University Irving Medical Center, New York, NY 10032 USA </div>
<div class="p">
<sup>10</sup>Department of Medicine, Columbia University Vagelos College of Physicians and Surgeons, Columbia University Irving Medical Center, New York, NY 10032 USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Posey%20JE%22%5BAuthor%5D" class="usa-link"><span class="name western">Jennifer E Posey</span></a>
</div>
</div>
<sup>1,</sup><sup>9,</sup><sup>10,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, TX 77030 USA </div>
<div id="Aff2">
<sup>2</sup>Medical Scientist Training Program, Baylor College of Medicine, Houston, TX 77030 USA </div>
<div id="Aff3">
<sup>3</sup>Genetics and Genomics Graduate Program, Baylor College of Medicine, Houston, TX 77030 USA </div>
<div id="Aff4">
<sup>4</sup>Texas Children’s Hospital, Houston, TX 77030 USA </div>
<div id="Aff5">
<sup>5</sup>Department of Obstetrics and Gynecology, Baylor College of Medicine, Houston, TX 77030 USA </div>
<div id="Aff6">
<sup>6</sup>School of Medicine, Meharry Medical College, Nashville, TN 37208 USA </div>
<div id="Aff7">
<sup>7</sup>Human Genome Sequencing Center, Baylor College of Medicine, Houston, TX 77030 USA </div>
<div id="Aff8">
<sup>8</sup>Department of Pediatrics, Baylor College of Medicine, Houston, TX 77030 USA </div>
<div id="Aff9">
<sup>9</sup>Department of Pediatrics, Columbia University Vagelos College of Physicians and Surgeons, Columbia University Irving Medical Center, New York, NY 10032 USA </div>
<div id="Aff10">
<sup>10</sup>Department of Medicine, Columbia University Vagelos College of Physicians and Surgeons, Columbia University Irving Medical Center, New York, NY 10032 USA </div>
<div class="author-notes p"><div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div></div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2024 Nov 6; Accepted 2025 Jul 29; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by-nc-nd/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12359922  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40826123/" class="usa-link">40826123</a>
</div>
<div class="ra xbox p" role="complementary" aria-label="Related or updated information about this article"><div>
<strong>Previous version available:</strong> This article is based on a previously available preprint posted on medRxiv on December 2, 2024: "<a href="/articles/PMC11643181/" class="usa-link">Improving Automated Deep Phenotyping Through Large Language Models Using Retrieval Augmented Generation</a>".</div></div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<section id="sec1"><h3 class="pmc_sec_title">Background</h3>
<p id="Par11">Diagnosing rare genetic disorders relies on precise phenotypic and genotypic analysis, with the Human Phenotype Ontology (HPO) providing a standardized language for capturing clinical phenotypes. Rule-based HPO extraction tools use concept recognition to automatically identify phenotypes, but they often struggle with incomplete phenotype assignment, requiring significant manual review. While large language models (LLMs) hold promise for more context-driven phenotype extraction, they are prone to errors and “hallucinations,” making them less reliable without further refinement. We present RAG-HPO, a Python-based tool that leverages retrieval-augmented generation (RAG) to elevate accuracy of HPO term assignment by LLM. This approach bypasses the limitations of baseline models and eliminates the need for time- and resource-intensive fine-tuning. RAG-HPO integrates a dynamic vector database, containing &gt; 54,000 phenotypic phrases mapped to HPO IDs, which allows real-time retrieval and contextual matching. The RAG-HPO workflow begins by extracting phenotypic phrases from clinical text via an LLM and then matching them via semantic similarity to entries within the database. The best term matches are returned to the LLM as context for final HPO term assignment of each phrase.</p></section><section id="sec2"><h3 class="pmc_sec_title">Results</h3>
<p id="Par12">Performance was benchmarked on 112 published case reports with 1792 manually assigned HPO terms and compared to Doc2HPO, ClinPhen, and FastHPOCR. In evaluations, RAG-HPO + LLaMa-3.1 70B achieved a mean precision of 0.81, recall of 0.76, and an F1 score of 0.78—significantly surpassing conventional tools (<em>p</em> &lt; 0.00001). RAG-HPO returned 1648 terms, of which 19.1% (315) were false positives that did not exactly match our manually annotated standard. Among these, &lt; 1% (1/315) represented hallucinations, and 1.3% (4/315) represented terms with no ontological relationship to the desired target; the remaining false positives (95.2%, 300/315) were broader ancestor terms of the target term, which may still be relevant to users in many contexts.</p></section><section id="sec3"><h3 class="pmc_sec_title">Conclusions</h3>
<p id="Par13">RAG-HPO is a user-friendly, adaptable tool designed for secure evaluation of clinical text and outperforms standard HPO-matching tools in precision, recall, and F1. Its enhanced precision and recall represent a substantial advancement in phenotypic analysis, accelerating the identification of genetic mechanisms underlying rare diseases and driving progress in genetic research and clinical genomics. RAG-HPO is available at <a href="https://github.com/PoseyPod/RAG-HPO" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/PoseyPod/RAG-HPO</a>.</p></section><section id="sec4"><h3 class="pmc_sec_title">Supplementary Information</h3>
<p>The online version contains supplementary material available at 10.1186/s13073-025-01521-w.</p></section><section id="kwd-group1" lang="en" class="kwd-group"><p><strong>Keywords:</strong> Large language models (LLMs), Retrieval-augmented generation (RAG), Phenotyping, Human Phenotype Ontology (HPO), Natural language processing (NLP), Clinical genomics, Generative pre-trained transformer (GPT), Generative AI, LLaMa-3</p></section></section><section id="Sec1"><h2 class="pmc_sec_title">Background</h2>
<p id="Par22">In genomic medicine and research, phenotypic and genotypic analyses are critical for achieving accurate molecular diagnoses. Deep phenotyping allows for a detailed understanding of a patient’s clinical presentation, which can then be matched to potential genetic causes [<a href="#CR1" class="usa-link" aria-describedby="CR1">1</a>]. Genomic analysis provides the molecular insights necessary to identify pathogenic variants that may be contributing to disease [<a href="#CR2" class="usa-link" aria-describedby="CR2">2</a>]. Together, these approaches facilitate a comprehensive evaluation of patients, particularly those with rare or undiagnosed conditions, and offer the possibility of uncovering genetic etiologies that might otherwise remain elusive [<a href="#CR3" class="usa-link" aria-describedby="CR3">3</a>]. Integrating clinical and molecular data is essential for providing patients with concrete answers, guiding their treatment, and improving outcomes [<a href="#CR4" class="usa-link" aria-describedby="CR4">4</a>–<a href="#CR7" class="usa-link" aria-describedby="CR7">7</a>].</p>
<p id="Par23">The Human Phenotype Ontology (HPO) is a standardized, hierarchically structured vocabulary that underpins modern deep-phenotyping workflows [<a href="#CR8" class="usa-link" aria-describedby="CR8">8</a>–<a href="#CR10" class="usa-link" aria-describedby="CR10">10</a>]. Its hierarchical structure links broad categories to increasingly specific descriptors, enabling clinicians and researchers to capture subtle phenotypic differences—even when clinical terminology varies—while preserving computational comparability [<a href="#CR11" class="usa-link" aria-describedby="CR11">11</a>]. Each concept is assigned a stable identifier (e.g., HP:0004322 “short stature”), allowing rapid, large-scale matching of phenotypically similar individuals to candidate genetic variants [<a href="#CR10" class="usa-link" aria-describedby="CR10">10</a>, <a href="#CR12" class="usa-link" aria-describedby="CR12">12</a>, <a href="#CR13" class="usa-link" aria-describedby="CR13">13</a>]. Over time, revision and expansion of the HPO term set has led to a modest number of obsolete terms [<a href="#CR10" class="usa-link" aria-describedby="CR10">10</a>].
</p>
<p id="Par24">Figure <a href="#Fig1" class="usa-link">1</a> illustrates both the breadth and depth of the HPO as a resource for identifying patient phenotypes. Figure <a href="#Fig1" class="usa-link">1</a>A shows that nearly half of the 19,024 non-obsolete terms belong to the musculoskeletal system, while only 245 (&lt; 2%) HPO terms within the database describe developmental biology-related phenotypes. Figure <a href="#Fig1" class="usa-link">1</a>B highlights the ontology’s specificity by organ system, with median nodal depths clustering between six and ten across organ systems. However, every branch also contains range of nodal depths from general (&lt; 3-level) to ultra-specific (&gt; 12-level) terms (e.g., HP:0001507 “growth abnormality” vs. HP:0008845 “mesomelic short stature”). This combination of wide anatomic coverage and variable specificity makes the HPO uniquely suited for machine-readable phenotypic data, laying the groundwork for advanced genotype–phenotype matching tools that drive precision genomic medicine today [<a href="#CR14" class="usa-link" aria-describedby="CR14">14</a>–<a href="#CR16" class="usa-link" aria-describedby="CR16">16</a>].</p>
<figure class="fig xbox font-sm" id="Fig1"><h3 class="obj_head">Fig. 1.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12359922_13073_2025_1521_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed34/12359922/977c2908c7c2/13073_2025_1521_Fig1_HTML.jpg" loading="lazy" id="MO1" height="728" width="778" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Distribution of HPO terms by organ system and nodal distribution. <strong>A</strong> Absolute counts of HPO terms per organ system. <strong>B</strong> Nodal-depth distribution within each system representing term specificity. Musculoskeletal descriptors account for the largest share of HPO concepts, whereas developmental-biology terms form the smallest branch. The box-and-whisker plots in <strong>B</strong> show that all systems possess a long “tail” of deeply nested (&gt; 12-level) terms, illustrating the ontology’s ability to capture highly granular phenotypes. Counts reflect the 19,024 non-obsolete HPO terms available in the October 2024 release; multi-system annotations are tallied once per system</p></figcaption></figure><p id="Par25">In rare disease research, HPO terms have become central to phenotype-driven variant prioritization and genotype–phenotype correlation [<a href="#CR17" class="usa-link" aria-describedby="CR17">17</a>]. Tools such as Exomiser use patient-specific HPO profiles to rank candidate variants by phenotypic similarity, improving diagnostic yield and reducing the time to diagnosis [<a href="#CR18" class="usa-link" aria-describedby="CR18">18</a>, <a href="#CR19" class="usa-link" aria-describedby="CR19">19</a>]. At the research level, platforms like the Monarch Initiative integrate HPO annotations across human and model organism datasets to support discovery of novel gene–disease associations [<a href="#CR10" class="usa-link" aria-describedby="CR10">10</a>, <a href="#CR20" class="usa-link" aria-describedby="CR20">20</a>]. Beyond genetics, HPO also enables the stratification of patients with overlapping clinical features in studies of complex traits, treatment response, and multi-system diseases, underscoring its broad utility in biomedical research [<a href="#CR21" class="usa-link" aria-describedby="CR21">21</a>, <a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>].</p>
<p id="Par26">While HPO terms provide a standardized framework for cataloging and investigating patient phenotypes, the process of extracting deep phenotyping information from clinical text remains labor-intensive and reliant on clinical expertise. Many tools, such as Doc2HPO, ClinPhen, and FastHPOCR, seek to automate the extraction of relevant phenotypic phrases from clinical records using dictionary-based, concept recognition methods [<a href="#CR12" class="usa-link" aria-describedby="CR12">12</a>, <a href="#CR13" class="usa-link" aria-describedby="CR13">13</a>, <a href="#CR23" class="usa-link" aria-describedby="CR23">23</a>]. This approach involves comparing segments of text against a predefined lexicon of domain-specific terms, such as HPO terms, to identify phenotypes within clinical text. While straightforward and interpretable, concept recognition lacks the ability to interpret nuanced contexts, such as family history and pertinent negatives, leading to the incorrect assignment of HPO IDs. These tools often miss a substantial portion of patient phenotypes, necessitating thorough manual cross-examination to ensure accurate phenotype annotation.</p>
<p id="Par27">Large language models (LLMs) are advanced computational programs trained to understand and generate natural language based on patterns present in human-generated text [<a href="#CR24" class="usa-link" aria-describedby="CR24">24</a>, <a href="#CR25" class="usa-link" aria-describedby="CR25">25</a>]. As part of the increased popularity of generative artificial intelligence (AI), these programs offer a promising opportunity to significantly improve automated deep phenotyping [<a href="#CR26" class="usa-link" aria-describedby="CR26">26</a>, <a href="#CR27" class="usa-link" aria-describedby="CR27">27</a>]. Within the biomedical field, remarkable advances in LLM technology have led to rapid growth in the use of technology in various clinical and basic science applications, including interpretation of radiological results or analyzing medical text [<a href="#CR28" class="usa-link" aria-describedby="CR28">28</a>–<a href="#CR32" class="usa-link" aria-describedby="CR32">32</a>]. The ability of LLMs to understand natural language context is a critical component for improving the ability to extract relevant clinical phenotypes from patient data. Consequently, LLMs have entered the phenotypic analysis space through tools like PhenoTagger, PhenoBERT, and PhenoGPT [<a href="#CR33" class="usa-link" aria-describedby="CR33">33</a>–<a href="#CR35" class="usa-link" aria-describedby="CR35">35</a>].</p>
<p id="Par28">However, the integration of LLMs into phenotypic analysis presents new challenges. As other authors have noted, currently available LLMs are resource-intensive and slow in their reasoning compared to other machine learning methods [<a href="#CR23" class="usa-link" aria-describedby="CR23">23</a>]. Importantly, they are also prone to hallucinations, in which the model generates incorrect information and confidently asserts it as fact [<a href="#CR36" class="usa-link" aria-describedby="CR36">36</a>]. Many LLM-based tools for phenotypic analysis address these concerns through fine-tuning, which involves further training of a model with additional data [<a href="#CR35" class="usa-link" aria-describedby="CR35">35</a>, <a href="#CR37" class="usa-link" aria-describedby="CR37">37</a>, <a href="#CR38" class="usa-link" aria-describedby="CR38">38</a>]. While this process increases the accuracy of LLM responses, it is computationally expensive (requiring substantial GPU and RAM), time-consuming, and necessitates specialized expertise in working with generative AI models [<a href="#CR26" class="usa-link" aria-describedby="CR26">26</a>, <a href="#CR36" class="usa-link" aria-describedby="CR36">36</a>, <a href="#CR39" class="usa-link" aria-describedby="CR39">39</a>]. Additionally, fine-tuning may not always improve LLM performance in the desired way, especially with large-scale models [<a href="#CR40" class="usa-link" aria-describedby="CR40">40</a>, <a href="#CR41" class="usa-link" aria-describedby="CR41">41</a>]. With the rapid advancement of LLM technology and continual updates to the HPO lexicon, fine-tuning LLMs for phenotypic analysis becomes impractical for most clinicians and researchers involved in diagnosing rare genetic diseases.</p>
<p id="Par29">Alternatively, retrieval-augmented generation (RAG) is a practical solution to the limitations of fine-tuning. RAG uses vector databases to retrieve relevant information from source documents in real-time, making it easier and faster to update the system with new information [<a href="#CR42" class="usa-link" aria-describedby="CR42">42</a>–<a href="#CR44" class="usa-link" aria-describedby="CR44">44</a>]. Users can refresh the underlying vector database with minimal effort, allowing the system to stay current without the need to retrain the LLM. This makes RAG more cost-effective and adaptable, particularly for users without advanced technical expertise and who use a resource that constantly updates, while still leveraging the power of LLMs to enhance the accuracy and precision of results.</p>
<p id="Par30">We have developed a Python-based program, RAG-HPO [<a href="#CR45" class="usa-link" aria-describedby="CR45">45</a>], to apply RAG to deep phenotyping, dramatically increasing LLMs’ ability to accurately assign HPO terms to patient phenotypes. This enables us to leverage the LLMs’ superior ability to extract phenotypic information from clinical data, improving the process of automated deep phenotyping. RAG-HPO is simple to use, does not require extensive understanding of LLMs, works with any language model, and does not require computationally intense resources like GPUs and large amounts of RAM for use. By utilizing a vector database, RAG-HPO can be easily updated with new information from the HPO database and user input. When paired with LLaMa-3, RAG-HPO demonstrates superior precision and recall compared to popular dictionary-based concept recognition tools.</p></section><section id="Sec2"><h2 class="pmc_sec_title">Implementation</h2>
<p id="Par31">RAG-HPO is a Python-based tool designed to extract clinical phenotypes from medical free text and assign HPO terms to those phrases using RAG. Users can employ any LLM of their choosing by providing an Application Programming Interface (API) key. Below, we describe implementation of this tool with the LLaMa-3.1 70B for the benchmarking of RAG-HPO.</p>
<section id="Sec3"><h3 class="pmc_sec_title">Data preparation, embedding, and vectorization</h3>
<p id="Par32">The vector database used by RAG-HPO utilizes a Python dictionary with key-value pairs, where keys are clinical words and phrases that reliably match to HPO ID numerical values (e.g., furrowed tongue: HP:0000221). The initial dictionary was extracted from the HPO database and includes term titles, names, definitions, and synonyms paired with their respective HPO IDs. To expand its coverage, we generated additional synonyms and short phrases for each HPO term using LLaMa-3.1 70B, with instructions focused on medical practice and current literature. After removing duplicates, we submitted each candidate phrase for validation using Doc2HPO and ClinPhen, retaining only those that accurately mapped back to the original HPO ID. This process added several thousand custom entries, each reliably associated with the correct HPO term, resulting in a final custom dictionary of over 54,000 unique phrases that correspond to individual HPO IDs stored in JavaScript Object Notation (JSON) format (Additional file 1: Vector database dictionary). The refined dataset serves as the basis for the vector database used in RAG-HPO. Fastembed was employed to convert each term within the JSON file to a high-dimensional vector to capture semantic relationships between phenotypic terms and factor together lineage information to bring like terms closer to one another, enabling enhanced similarity searches during phenotypic matching.</p>
<p id="Par33">The embedded database is stored as a NumPy array for use by RAG-HPO, serving as a resource to support the LLM in making accurate assignments. When the analysis program is initiated, the NumPy array is indexed in a vector database optimized for dense vector retrieval, allowing for efficient approximate nearest-neighbor searches. Relevant metadata is also indexed within the database to assist the LLM in reasoning through the assignment process.</p></section><section id="Sec4"><h3 class="pmc_sec_title">RAG-HPO workflow</h3>
<p id="Par34">The program processes free text provided as strings, converting them into HPO terms. Input text can either be supplied directly by the user or batch-processed from a CSV file containing clinical notes. The text is first passed to the user’s chosen LLM via an API call, where a custom system prompt instructs the LLM to extract phenotypic phrases that describe the patient’s health status, while disregarding non-relevant information such as administrative details or general observations that do not pertain to the patient’s health (Fig. <a href="#Fig2" class="usa-link">2</a>A, Additional file 2: Fig. S1).</p>
<figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed34/12359922/cff9b7bf8918/13073_2025_1521_Fig2_HTML.jpg" loading="lazy" id="MO2" height="293" width="787" alt="Fig. 2"></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>RAG-HPO extracts phenotypic information and returns HPO terms. RAG-HPO assigns HPO terms to free-text clinical notes in two main phases. <strong>A</strong> First, clinical information is provided to the LLM, which extracts relevant clinical abnormalities as text phrases. <strong>B</strong> Next, each extracted phrase is vectorized and compared to entries in an HPO vector database using semantic similarity search. <strong>C</strong> For each phrase, the top 20 most similar HPO terms are returned to the LLM, which selects the most appropriate HPO term. <strong>D</strong> After all phrases are processed, the final list of assigned HPO terms is returned to the user for verification and downstream analysis</p></figcaption></figure><p id="Par35">Once extracted, these phenotypic phrases are prepared for the HPO term assignment process, which begins with semantic similarity search (Fig. <a href="#Fig2" class="usa-link">2</a>B). To optimize computational efficiency and reduce the workload on the LLM, each extracted phrase is first compared to the metadata within the embeddings array using fuzzy matching. Exact matches are directly included in the results without further processing, while phrases that do not have an exact match are converted into high-dimensional semantic vectors using Fastembed. The generated embeddings, along with associated metadata (such as HPO terms, lineage information, and organ system), are compared to the developed HPO database in a similarity search via Facebook AI Similarity Search, an efficient package for indexing and searching vector data, to identify related HPO terms (Fig. <a href="#Fig2" class="usa-link">2</a>B). The program retrieves the top 20 most semantically similar vectors, along with their associated metadata, including the relevant HPO terms, lineage, and other contextual information. Then, the surrounding sentence from the original clinical text is retrieved to provide additional context for understanding the extracted phrase.</p>
<p id="Par36">The extracted clinical phrases are resubmitted to the LLM, which uses the additional context provided by the metadata to select the most appropriate and detailed HPO term representing the most distal matching node (highest possible information content) in the ontology that fits the level of information provided by the phenotypic phrase (Fig. <a href="#Fig2" class="usa-link">2</a>C). After each extracted phrase and metadata block are passed through the LLM, a completed list of HPO terms for the whole passage is returned to the user (Fig. <a href="#Fig2" class="usa-link">2</a>D). When analyzing batches, the results are saved as a JSON object list within a copy of the original CSV file under a new column.</p></section><section id="Sec5"><h3 class="pmc_sec_title">Selection of evaluation material and criteria for comparative analysis</h3>
<p id="Par37">RAG-HPO, ClinPhen, Doc2HPO, and FASTHPOCR were evaluated based on their ability to accurately extract HPO terms from previously published case reports. This case study cohort (CSC) comprised 120 case reports sourced from two peer-reviewed journals: <em>BMJ Case Reports</em> and <em>Oxford Medical Case Reports</em>. These journals were selected for their rigorous editorial standards and comprehensive coverage of diverse medical specialties, including gastroenterology, musculoskeletal, dermatology, hematology, cardiovascular, pulmonary, renal, endocrinology, reproductive health, neuroanatomy, and psychiatry. Both journals are widely recognized in the medical community and are indexed in PubMed Central, ensuring accessibility and credibility. A third-year medical student, independent of the evaluation team, screened the reports to minimize selection bias.</p>
<p id="Par38">Each case within the study described one to two patients with a distinct set of phenotypes for a particular illness. Manual annotation of HPO terms was initiated by the medical student, who extracted and matched phenotypic features from each case report to specific HPO IDs. Subsequently, additional team members, including a certified genetic counselor and a physician-scientist trainee, reviewed and completed the annotations to ensure accuracy and consistency across the dataset. Discrepancies were resolved through consensus discussions involving the annotators and medical genetics faculty, who selected the most appropriate HPO term based on clinical expertise and the context of each case. Reports describing fewer than four discernible phenotypic features were excluded, leaving 112 cases with 1794 (1208 unique) assignable HPO terms across the cohort with a mean of 15.8 HPO terms per case (Table <a href="#Tab1" class="usa-link">1</a>). Additional file 2: Fig. S2 demonstrates the organ system distribution and nodal depth of the terms found within the CSC.
</p>
<section class="tw xbox font-sm" id="Tab1"><h4 class="obj_head">Table 1.</h4>
<div class="caption p"><p>Composition of the evaluation datasets compared to HPO database. The composition of HPO terms found in the case study cohort (CSC) compared to gold standard corpora (GSC) and original HPO database. The CSC has nearly triple the number of unique HPO terms and almost twice as many terms per case as the GSC, offering more opportunity to test HPO analysis tools on a wide variety of phenotypic categories</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead>
<tr>
<th align="left" rowspan="2" colspan="1">Cohort</th>
<th align="left" colspan="3" rowspan="1">HPO terms present</th>
<th align="left" colspan="2" rowspan="1">Nodal depth</th>
<th align="left" colspan="2" rowspan="1">Organ systems</th>
</tr>
<tr>
<th align="left" colspan="1" rowspan="1">Total</th>
<th align="left" colspan="1" rowspan="1">Unique</th>
<th align="left" colspan="1" rowspan="1">Per case (mean)</th>
<th align="left" colspan="1" rowspan="1">Cohort (mean)</th>
<th align="left" colspan="1" rowspan="1">Per case (mean)</th>
<th align="left" colspan="1" rowspan="1">Total</th>
<th align="left" colspan="1" rowspan="1">Per case (mean)</th>
</tr>
</thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">HPO</td>
<td align="left" colspan="1" rowspan="1">19,024</td>
<td align="left" colspan="1" rowspan="1">19,024</td>
<td align="left" colspan="1" rowspan="1">N/A</td>
<td align="left" colspan="1" rowspan="1">9.58</td>
<td align="left" colspan="1" rowspan="1">N/A</td>
<td align="left" colspan="1" rowspan="1">30</td>
<td align="left" colspan="1" rowspan="1">N/A</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">CSC</td>
<td align="left" colspan="1" rowspan="1">1794</td>
<td align="left" colspan="1" rowspan="1">1208</td>
<td align="left" colspan="1" rowspan="1">15.8</td>
<td align="left" colspan="1" rowspan="1">7.1</td>
<td align="left" colspan="1" rowspan="1">6.8</td>
<td align="left" colspan="1" rowspan="1">28</td>
<td align="left" colspan="1" rowspan="1">6.7</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">GSC</td>
<td align="left" colspan="1" rowspan="1">1013</td>
<td align="left" colspan="1" rowspan="1">415</td>
<td align="left" colspan="1" rowspan="1">8.9</td>
<td align="left" colspan="1" rowspan="1">7.1</td>
<td align="left" colspan="1" rowspan="1">6.7</td>
<td align="left" colspan="1" rowspan="1">21</td>
<td align="left" colspan="1" rowspan="1">5.0</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">GSC (original)</td>
<td align="left" colspan="1" rowspan="1">1323</td>
<td align="left" colspan="1" rowspan="1">447</td>
<td align="left" colspan="1" rowspan="1">11.6</td>
<td align="left" colspan="1" rowspan="1">6.9</td>
<td align="left" colspan="1" rowspan="1">6.5</td>
<td align="left" colspan="1" rowspan="1">23</td>
<td align="left" colspan="1" rowspan="1">5.6</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><p id="Par39">We also evaluated RAG-HPO and other HPO analysis software using the gold standard corpora (GSC), which contains 229 entries describing genetic conditions [<a href="#CR15" class="usa-link" aria-describedby="CR15">15</a>]. After applying the same filtering criteria as for our case study cohort (i.e., removing entries with fewer than four distinct phenotypes and manually assigning HPO terms), we curated a final dataset of 114 entries comprising 1013 HPO terms, of which 415 were unique, averaging 8.9 terms per case (Table <a href="#Tab1" class="usa-link">1</a>). We then compared our manually assigned HPO terms for each GSC entry to those comprising the original annotation. Additional file 2: Fig. S3 illustrates entries for which we excluded overly general terms present in the original annotations. Additional file 2: Fig. S4 displays the organ system distribution and nodal depth of the terms found within the GSC, comparable to Fig. <a href="#Fig1" class="usa-link">1</a> and Additional file 2: Fig. S2 for the HPO database and CSC.</p>
<p id="Par40">Figure <a href="#Fig3" class="usa-link">3</a> demonstrates key differences between the full HPO dataset, the GSC, and our CSC. While the GSC covers a range of organ systems, it is heavily skewed toward a small subset of systems, much like the HPO database (Fig. <a href="#Fig3" class="usa-link">3</a>A and B, respectively). Over 56.5% of the annotated HPO terms in the GSC fall within just three systems: the nervous system (23.7%), musculoskeletal system (22.2%), and head/neck/special senses (10.6%). Other systems such as the endocrine/metabolic (5.8%), genitourinary (3.5%), cardiovascular (3.1%), and immune (5.4%) systems are markedly underrepresented. This distribution reflects a bias inherent to the OMIM-derived source material, which often emphasizes neurodevelopmental and congenital disorders.</p>
<figure class="fig xbox font-sm" id="Fig3"><h4 class="obj_head">Fig. 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12359922_13073_2025_1521_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed34/12359922/c588ed46ec81/13073_2025_1521_Fig3_HTML.jpg" loading="lazy" id="MO3" height="728" width="778" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Comparative organ system representation across HPO, GSC, and case study cohorts. <strong>A</strong> Distribution of HPO terms by organ system in the full HPO database, where musculoskeletal terms comprise the largest proportion. <strong>B</strong> Distribution in the gold standard corpus (GSC), showing a more balanced representation across systems, though still skewed toward musculoskeletal and nervous system terms. <strong>C</strong> Distribution in the case study cohort (CSC), reflecting broader organ system diversity, with increased representation of cardiovascular, endocrine, and neurologic terms</p></figcaption></figure><p id="Par41">Such imbalances reduce the generalizability of the GSC as a benchmark for phenotype extraction tools aimed at broad clinical application. In contrast, our CSC demonstrates a more even and realistic distribution of phenotypic systems (Fig. <a href="#Fig3" class="usa-link">3</a>C), enhancing its value as a performance benchmark for general-purpose tools like RAG-HPO.</p></section><section id="Sec6"><h3 class="pmc_sec_title">ClinPhen, Doc2HPO, and FASTHPOCR employ dictionary-based concept recognition to identify HPO terms</h3>
<p id="Par42">Briefly, ClinPhen segments clinical notes into sentences and subsentences, normalizing and matching phrases against an HPO synonym dictionary while filtering out irrelevant or negated mentions [<a href="#CR13" class="usa-link" aria-describedby="CR13">13</a>]. Doc2HPO integrates several natural language processing engines—including string-based matching via the Aho–Corasick algorithm and MetaMap-based methods—to provide an interactive interface for real-time curation and negation detection [<a href="#CR12" class="usa-link" aria-describedby="CR12">12</a>]. FASTHPOCR uses a fast dictionary-based approach that first consolidates morphologically equivalent tokens into clusters, creating an index of HPO concept signatures for rapid candidate matching [<a href="#CR23" class="usa-link" aria-describedby="CR23">23</a>]. Detailed descriptions of these methods are available in their respective publications.</p>
<p id="Par43">For our evaluation, ClinPhen [<a href="#CR46" class="usa-link" aria-describedby="CR46">46</a>] was downloaded from <a href="http://bejerano.stanford.edu/clinphen/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://bejerano.stanford.edu/clinphen/</a> and Doc2HPO [<a href="#CR47" class="usa-link" aria-describedby="CR47">47</a>] was accessed via its web interface at <a href="https://doc2hpo.wglab.org/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://doc2hpo.wglab.org/</a>. FASTHPOCR [<a href="#CR48" class="usa-link" aria-describedby="CR48">48</a>] was obtained from its GitHub repository (<a href="https://github.com/tudorgroza/fast_hpo_cr" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/tudorgroza/fast_hpo_cr</a>) in June 2024. It is noteworthy that both ClinPhen and Doc2HPO utilize an older version of the HPO dataset, which may lead to the extraction of obsolete terms.</p></section><section id="Sec7"><h3 class="pmc_sec_title">Performance evaluation of RAG-HPO using multiple open-source LLMs</h3>
<p id="Par44">In contrast, RAG-HPO’s performance was evaluated using an updated HPO version accessed in August 2024 (v2024-08-13). For the evaluation, we tested RAG-HPO using several LLMs available for use through groq.com, including LLaMa-3 70B, LLaMa-3.1 8B, LLaMa-3.1 70B, LLaMa-4 Scout (16B), Misra 24B, and Deepseek-R1. These LLMs represent a wide variety of open-source LLMs that are reasonably available for researchers and clinicians to access for their own evaluations and use in research. Testing across several versions of LLaMa language models allowed us to demonstrate the changes in performance with incremental improvements of LLMs over time.</p>
<p id="Par45">In our analysis, we considered any HPO term called by an analysis tool to be a true positive if it matched the manually annotated standard. False positives were then any called HPO term that did not match manually annotated terms. Failure to identify an HPO term from the standard was considered a false negative. The number of true positives, false positives, and false negatives were used to calculate three key metrics: precision, recall, and the F1 score. These scores were used in determining the performance of the HPO analysis tools.</p>
<p id="Par46">Precision measures the accuracy of each tool in returning relevant HPO terms and is calculated as:</p>
<table class="disp-formula p" id="Equa"><tr><td class="formula"><math id="d33e873" display="block"><mrow><mtext>Precision</mtext><mo>=</mo><mfrac><mrow><mtext>True Positive</mtext></mrow><mrow><mtext>True Positive</mtext><mo>+</mo><mtext>False Positive</mtext></mrow></mfrac></mrow></math></td></tr></table>
<p id="Par47">Recall measures the ability of the tool to retrieve all HPO terms present within a case note and is calculated as:</p>
<table class="disp-formula p" id="Equb"><tr><td class="formula"><math id="d33e892" display="block"><mrow><mtext>Recall</mtext><mo>=</mo><mfrac><mrow><mtext>True Positive</mtext></mrow><mrow><mtext>True Positive</mtext><mo>+</mo><mtext>False Negative</mtext></mrow></mfrac></mrow></math></td></tr></table>
<p id="Par48">The F1 score is a weighted average of precision and recall on a scale of 0–1 and was calculated using the following formula:</p>
<table class="disp-formula p" id="Equc"><tr><td class="formula"><math id="d33e911" display="block"><mrow><mtext>F</mtext><mn>1</mn><mo>=</mo><mn>2</mn><mo>×</mo><mfrac><mrow><mtext>Precision</mtext><mo>×</mo><mtext>Recall</mtext></mrow><mrow><mtext>Precision</mtext><mo>+</mo><mtext>Recall</mtext></mrow></mfrac></mrow></math></td></tr></table>
<p id="Par49">Python scripts were used to evaluate performance and develop figures. To keep evaluations objective and consistent, we developed a Python script to accurately compared the output of each program against the manually annotated standards. The script automatically split multi-term entries, removed invalid rows (entries without HPO term IDs), and counted the true positives, false positives, and false negatives for each case. These counts were used to calculate precision, recall, and F1 scores per case and we reported the average of these scores for the evaluated cohorts. Additionally, we built Python scripts to generate the figures within this paper using a variety of packages, including <em>numpy</em> (2.2.5), <em>pandas</em> (2.2.3), <em>scipy</em> (1.15.2), <em>statsmodels</em> (0.14.4), and <em>matplotlib</em> (3.10.1). Original verbatim inputs for the CSC and GSC, the manual annotations, and all outputs for this study are available for review in Additional file 3: RAG-HPO tests and data analysis.</p></section></section><section id="Sec8"><h2 class="pmc_sec_title">Results</h2>
<p id="Par50">RAG greatly increases LLM’s ability to identify correct HPO terms. The goal of RAG is to improve a language model’s capacity to retrieve and generate contextually relevant information, thereby reducing “hallucinations.” In HPO analysis, hallucinations typically appear as non-existent or inappropriate HPO identifiers that do not match the underlying clinical phrases. Without RAG (or similar informational scaffolding), current LLMs struggle to assign HPO terms accurately.</p>
<p id="Par51">To illustrate this challenge, we evaluated several open- and closed-source LLMs on a subset of 20 notes from our case study cohort. Below is an example clinical note with the available phenotypic phrases highlighted according to ChatGPT (o3) output:</p>
<blockquote class="text-italic"><p id="Par52">A 44-year- old <em>super-morbidly- obese</em> man body mass index (BMI 63) underwent sleeve gastrectomy for weight loss and was found to have <em>multiple adenomatous fundic gland polyps</em> on final pathology. Subsequent workup included esophagogastroduodenoscopy which revealed <em>innumerable polyps of the remaining gastric fundus and body</em> consistent with fundic gland polyps, normal duodenum without polyps, and <em>Barrett’s oesophagus</em>. Colonoscopy was significant for <em>innumerable polyps of varying sizes up to 1.5 cm throughout the colon,</em> with relative rectal sparing. Biopsies were consistent with <em>tubular adenoma and hyperplastic polyps</em><em>.</em> Thyroid ultrasound was within normal limits and abdominal CT was significant for <em>left-sided 3.4 cm mesenteric mass</em> representing scarring versus possible desmoid. Family history was significant for colon cancer diagnosed in his maternal grandfather at age 72, paternal grandmother who died of metastatic cancer at age 50 with unknown primary and mother diagnosed with thyroid cancer at age 40. Physical examination was notable for <em>morbid obesity</em> (BMI 45), cardiopulmonary examination within normal limits and abdomen with well-healing surgical scars from prior sleeve gastrectomy.</p></blockquote>
<p id="Par53">As shown in this example, most LLMs capture the majority of clinical phenotypes available in a particular case (see Additional file 2: Fig. S5 for full results). However, they frequently failed to assign proper HPO codes, yielding uniformly low precision, recall, and F1 scores (Table <a href="#Tab2" class="usa-link">2</a>).
</p>
<section class="tw xbox font-sm" id="Tab2"><h3 class="obj_head">Table 2.</h3>
<div class="caption p"><p>Comparison of baseline LLM performance in HPO analysis of case studies. Models were evaluated based on the number of true positives, false positives, and false negatives, with corresponding precision, recall, and F1 scores calculated. No retrieval augmentation or fine-tuning was applied. Results demonstrate substantial variability in performance across models, with generally low precision and recall in the absence of additional context or refinement</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Analysis tool</th>
<th align="left" colspan="1" rowspan="1">True positive</th>
<th align="left" colspan="1" rowspan="1">False positive</th>
<th align="left" colspan="1" rowspan="1">False negative</th>
<th align="left" colspan="1" rowspan="1">Precision</th>
<th align="left" colspan="1" rowspan="1">Recall</th>
<th align="left" colspan="1" rowspan="1">F1</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">LLaMA-3.1 70b</td>
<td align="left" colspan="1" rowspan="1">29</td>
<td align="left" colspan="1" rowspan="1">181</td>
<td align="left" colspan="1" rowspan="1">264</td>
<td align="left" colspan="1" rowspan="1">0.13</td>
<td align="left" colspan="1" rowspan="1">0.09</td>
<td align="left" colspan="1" rowspan="1">0.10</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">LLaMA-4 Scout</td>
<td align="left" colspan="1" rowspan="1">8</td>
<td align="left" colspan="1" rowspan="1">187</td>
<td align="left" colspan="1" rowspan="1">285</td>
<td align="left" colspan="1" rowspan="1">0.04</td>
<td align="left" colspan="1" rowspan="1">0.03</td>
<td align="left" colspan="1" rowspan="1">0.03</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">ChatGPT (o1)</td>
<td align="left" colspan="1" rowspan="1">57</td>
<td align="left" colspan="1" rowspan="1">149</td>
<td align="left" colspan="1" rowspan="1">236</td>
<td align="left" colspan="1" rowspan="1">0.24</td>
<td align="left" colspan="1" rowspan="1">0.18</td>
<td align="left" colspan="1" rowspan="1">0.20</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">ChatGPT (4o)</td>
<td align="left" colspan="1" rowspan="1">60</td>
<td align="left" colspan="1" rowspan="1">212</td>
<td align="left" colspan="1" rowspan="1">233</td>
<td align="left" colspan="1" rowspan="1">0.20</td>
<td align="left" colspan="1" rowspan="1">0.19</td>
<td align="left" colspan="1" rowspan="1">0.19</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">ChatGPT (o3)</td>
<td align="left" colspan="1" rowspan="1">68</td>
<td align="left" colspan="1" rowspan="1">128</td>
<td align="left" colspan="1" rowspan="1">225</td>
<td align="left" colspan="1" rowspan="1">0.32</td>
<td align="left" colspan="1" rowspan="1">0.22</td>
<td align="left" colspan="1" rowspan="1">0.26</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Copilot</td>
<td align="left" colspan="1" rowspan="1">62</td>
<td align="left" colspan="1" rowspan="1">155</td>
<td align="left" colspan="1" rowspan="1">231</td>
<td align="left" colspan="1" rowspan="1">0.26</td>
<td align="left" colspan="1" rowspan="1">0.20</td>
<td align="left" colspan="1" rowspan="1">0.22</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Gemini 2</td>
<td align="left" colspan="1" rowspan="1">38</td>
<td align="left" colspan="1" rowspan="1">239</td>
<td align="left" colspan="1" rowspan="1">255</td>
<td align="left" colspan="1" rowspan="1">0.13</td>
<td align="left" colspan="1" rowspan="1">0.12</td>
<td align="left" colspan="1" rowspan="1">0.13</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><p id="Par54">By contrast, RAG-HPO greatly improved the ability of LLMs to accurately assign HPO terms. To evaluate this improvement, we compared the performance of LLaMa-3 70B both individually and as part of RAG-HPO (Table <a href="#Tab3" class="usa-link">3</a>) in the assignment of HPO terms for the case study cohort. When using the LLaMa-3 70B model alone (“baseline”), precision, recall, and F1 scores were 0.12, 0.09, and 0.10, respectively. However, when combined with RAG-HPO, these metrics improved dramatically, yielding a precision of 0.71, recall of 0.61, and an F1 score of 0.64. These results confirm that RAG significantly enhances LLM performance in HPO analysis, with the RAG component directly responsible for the improvements in accuracy.
</p>
<section class="tw xbox font-sm" id="Tab3"><h3 class="obj_head">Table 3.</h3>
<div class="caption p"><p>LLaMA-3 70B with vs. without RAG-HPO for HPO assignment. In the baseline condition, LLaMA-3 70B assigned only 164 correct terms across the entire case study cohort (CSC), demonstrating limited standalone utility. When paired with RAG-HPO, the same model achieved 1083 true positives, substantially improving precision, recall, and F1 score. Paired <em>t</em>-tests confirmed that the improvements in all three metrics were statistically significant (***<em>p</em> &lt; 0.001)</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead>
<tr>
<th align="left" rowspan="2" colspan="1">Analysis tool</th>
<th align="left" rowspan="2" colspan="1">True positive</th>
<th align="left" rowspan="2" colspan="1">False positive</th>
<th align="left" rowspan="2" colspan="1">False negative</th>
<th align="left" rowspan="2" colspan="1">Precision</th>
<th align="left" rowspan="2" colspan="1">Recall</th>
<th align="left" rowspan="2" colspan="1">F1</th>
<th align="left" colspan="3" rowspan="1">
<em>t</em>-test comparison</th>
</tr>
<tr>
<th align="left" colspan="1" rowspan="1"><strong>Precision</strong></th>
<th align="left" colspan="1" rowspan="1"><strong>Recall</strong></th>
<th align="left" colspan="1" rowspan="1"><strong>F1</strong></th>
</tr>
</thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">RAG-HPO</td>
<td align="left" colspan="1" rowspan="1">1083</td>
<td align="left" colspan="1" rowspan="1">486</td>
<td align="left" colspan="1" rowspan="1">711</td>
<td align="left" colspan="1" rowspan="1">0.71</td>
<td align="left" colspan="1" rowspan="1">0.61</td>
<td align="left" colspan="1" rowspan="1">0.64</td>
<td align="left" rowspan="2" colspan="1">***</td>
<td align="left" rowspan="2" colspan="1">***</td>
<td align="left" rowspan="2" colspan="1">***</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Baseline</td>
<td align="left" colspan="1" rowspan="1">164</td>
<td align="left" colspan="1" rowspan="1">1149</td>
<td align="left" colspan="1" rowspan="1">1630</td>
<td align="left" colspan="1" rowspan="1">0.12</td>
<td align="left" colspan="1" rowspan="1">0.09</td>
<td align="left" colspan="1" rowspan="1">0.10</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><section id="Sec9"><h3 class="pmc_sec_title">RAG-HPO versus established HPO analysis tools</h3>
<p id="Par55">The performance of RAG-HPO was benchmarked against ClinPhen, Doc2HPO, and FastHPOCR using our case study cohort (Table <a href="#Tab4" class="usa-link">4</a>). To assess the influence of LLM choice on performance, we paired RAG-HPO with six different back-end models. With the sole exception of DeepSeek R1, every LLM configuration of RAG-HPO performed on par with, or significantly better than, the conventional dictionary-based concept recognition tools (pair wise statistics, Additional file 2: Fig. S6). RAG-HPO performance improved with increased LLM parameter scaling and improvements to underlying architecture, context windows, and other back-end improvements made to LLMs over time.
</p>
<section class="tw xbox font-sm" id="Tab4"><h4 class="obj_head">Table 4.</h4>
<div class="caption p"><p>Comparative performance of RAG-HPO and benchmark tools in CSC analysis. Each model was evaluated on the number of true positives, false positives, and false negatives, along with average precision, recall, and F1 score across all cases. The combination of RAG-HPO with LLaMA-3.1 70B achieved the highest overall performance, outperforming both smaller LLMs and rule-based tools. Results highlight the effectiveness of retrieval-augmented generation in improving accuracy and reducing error rates in automated HPO term assignment</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">LLM + RAG-HPO</th>
<th align="left" colspan="1" rowspan="1">True positive</th>
<th align="left" colspan="1" rowspan="1">False positive</th>
<th align="left" colspan="1" rowspan="1">False negative</th>
<th align="left" colspan="1" rowspan="1">Average precision</th>
<th align="left" colspan="1" rowspan="1">Average recall</th>
<th align="left" colspan="1" rowspan="1">Average F1</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">LLaMA-3.1 70B</td>
<td align="left" colspan="1" rowspan="1">1333</td>
<td align="left" colspan="1" rowspan="1">315</td>
<td align="left" colspan="1" rowspan="1">461</td>
<td align="left" colspan="1" rowspan="1">0.81</td>
<td align="left" colspan="1" rowspan="1">0.76</td>
<td align="left" colspan="1" rowspan="1">0.78</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">LLaMA-3 70B</td>
<td align="left" colspan="1" rowspan="1">1083</td>
<td align="left" colspan="1" rowspan="1">486</td>
<td align="left" colspan="1" rowspan="1">711</td>
<td align="left" colspan="1" rowspan="1">0.71</td>
<td align="left" colspan="1" rowspan="1">0.61</td>
<td align="left" colspan="1" rowspan="1">0.64</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">LLaMA-4 Scout</td>
<td align="left" colspan="1" rowspan="1">1257</td>
<td align="left" colspan="1" rowspan="1">634</td>
<td align="left" colspan="1" rowspan="1">537</td>
<td align="left" colspan="1" rowspan="1">0.65</td>
<td align="left" colspan="1" rowspan="1">0.69</td>
<td align="left" colspan="1" rowspan="1">0.66</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">LLaMA-3 8B</td>
<td align="left" colspan="1" rowspan="1">852</td>
<td align="left" colspan="1" rowspan="1">727</td>
<td align="left" colspan="1" rowspan="1">942</td>
<td align="left" colspan="1" rowspan="1">0.53</td>
<td align="left" colspan="1" rowspan="1">0.46</td>
<td align="left" colspan="1" rowspan="1">0.48</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Misra 24B</td>
<td align="left" colspan="1" rowspan="1">581</td>
<td align="left" colspan="1" rowspan="1">174</td>
<td align="left" colspan="1" rowspan="1">1213</td>
<td align="left" colspan="1" rowspan="1">0.73</td>
<td align="left" colspan="1" rowspan="1">0.31</td>
<td align="left" colspan="1" rowspan="1">0.42</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Deepseek-R1</td>
<td align="left" colspan="1" rowspan="1">682</td>
<td align="left" colspan="1" rowspan="1">1944</td>
<td align="left" colspan="1" rowspan="1">1112</td>
<td align="left" colspan="1" rowspan="1">0.15</td>
<td align="left" colspan="1" rowspan="1">0.35</td>
<td align="left" colspan="1" rowspan="1">0.20</td>
</tr>
<tr><td align="left" colspan="7" rowspan="1">Other HPO analysis tools</td></tr>
<tr>
<td align="left" colspan="1" rowspan="1"> FASTHPOCR</td>
<td align="left" colspan="1" rowspan="1">816</td>
<td align="left" colspan="1" rowspan="1">726</td>
<td align="left" colspan="1" rowspan="1">978</td>
<td align="left" colspan="1" rowspan="1">0.53</td>
<td align="left" colspan="1" rowspan="1">0.45</td>
<td align="left" colspan="1" rowspan="1">0.49</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> Doc2HPO</td>
<td align="left" colspan="1" rowspan="1">634</td>
<td align="left" colspan="1" rowspan="1">260</td>
<td align="left" colspan="1" rowspan="1">1159</td>
<td align="left" colspan="1" rowspan="1">0.71</td>
<td align="left" colspan="1" rowspan="1">0.35</td>
<td align="left" colspan="1" rowspan="1">0.47</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> ClinPhen</td>
<td align="left" colspan="1" rowspan="1">635</td>
<td align="left" colspan="1" rowspan="1">370</td>
<td align="left" colspan="1" rowspan="1">1159</td>
<td align="left" colspan="1" rowspan="1">0.63</td>
<td align="left" colspan="1" rowspan="1">0.35</td>
<td align="left" colspan="1" rowspan="1">0.45</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><p id="Par56">RAG-HPO + LLaMa-3.1 70B achieved the strongest overall results, posting the highest precision (0.81), recall (0.76), and F1 score (0.78) in the evaluation (Fig. <a href="#Fig4" class="usa-link">4</a>, Table <a href="#Tab4" class="usa-link">4</a>). This configuration identified 1333 correct HPO terms, which was more than twice the true-positive count of ClinPhen (635) and Doc2HPO (634), and an almost 1.6-fold increase compared to FastHPOCR (816). Although FastHPOCR retrieved a larger number of terms per case than the other traditional tools, its larger false-positive burden (FP = 726) reduced both precision (0.53) and F1 (0.49).</p>
<figure class="fig xbox font-sm" id="Fig4"><h4 class="obj_head">Fig. 4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed34/12359922/72ad6e01d3be/13073_2025_1521_Fig4_HTML.jpg" loading="lazy" id="MO4" height="611" width="786" alt="Fig. 4"></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>RAG-HPO has more superior recall and precision than established HPO analysis tools. We compared the output of LLaMA-3 70B alone to RAG-HPO paired with LLaMA-3 70B in assigning HPO terms to 20 previously published case reports. RAG significantly improved performance (F1: 0.12 vs. 0.78, <em>p</em> &lt; 0.0001). RAG-HPO also outperformed ClinPhen, FastHPOCR, Doc2HPO, and baseline LLaMA across the CSC in precision, recall, and F1 score</p></figcaption></figure><p id="Par57">Figure <a href="#Fig5" class="usa-link">5</a> compares each tool’s output with the manually annotated case study reference set. RAG-HPO + LLaMA-3.1 70B recovered the largest share of ground-truth terms while adding the fewest false positives among RAG configurations, yielding the overall best error balance. Among stand-alone concept recognition tools, FastHPOCR retrieved the most correct terms but incurred a high false positive rate while ClinPhen and Doc2HPO only captured about one-third of the gold standard, but with a much lower rate of false positives. Together, these results underscore that even the best traditional tools lag behind RAG-HPO, both by missing true phenotypes and by misattributing HPO assignments.</p>
<figure class="fig xbox font-sm" id="Fig5"><h4 class="obj_head">Fig. 5.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12359922_13073_2025_1521_Fig5_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed34/12359922/c9d35eef201f/13073_2025_1521_Fig5_HTML.jpg" loading="lazy" id="MO5" height="324" width="709" alt="Fig. 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Error composition across RAG-HPO LLM variants and benchmark tools. Stacked bar plot comparing the distribution of true positives (TP), false negatives (FN), and false positives (FP) in HPO term extraction across multiple RAG-HPO configurations (with different large language models) and established tools (FastHPOCR, ClinPhen, and Doc2HPO). The y-axis represents the total percentage of output terms normalized per method. RAG-HPO models with LLaMA-3 70B and LLaMA-4 Scout achieve a more favorable balance of true positive predictions, while methods like DeepSeek R1 and some traditional tools demonstrate a higher burden of false positives and false negatives. A vertical dashed line separates RAG-HPO variants from non-RAG benchmark tools</p></figcaption></figure></section><section id="Sec10"><h3 class="pmc_sec_title">Evaluation of performance by organ system and nodal depth</h3>
<p id="Par58">We modeled the probability that a program assigns the correct HPO term using a binomial generalized estimating equation with patient-level clustering. Predictors were program, organ system, and nodal depth. Model coefficients were converted to odds ratios and summarized in three panels: a forest plot of organ-system effects (Fig. <a href="#Fig6" class="usa-link">6</a>A), a fitted-probability curve across nodal depths (Fig. <a href="#Fig6" class="usa-link">6</a>B), and box plots of predicted hit probabilities for each program (Fig. <a href="#Fig6" class="usa-link">6</a>C).</p>
<figure class="fig xbox font-sm" id="Fig6"><h4 class="obj_head">Fig. 6.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12359922_13073_2025_1521_Fig6_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed34/12359922/2e1821b66aea/13073_2025_1521_Fig6_HTML.jpg" loading="lazy" id="MO6" height="551" width="778" alt="Fig. 6"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Factors influencing HPO assignment accuracy in RAG-HPO and benchmark tools. GEE models were used to estimate the probability of correct HPO term assignment, accounting for clustering by patient. <strong>A</strong> Forest plot of organ system effects, shown as odds ratios with 95% confidence intervals. <strong>B</strong> Fitted probability curves by nodal depth, illustrating a decline in assignment accuracy as HPO terms became more specific. <strong>C</strong> Predicted hit probability distributions by tool, based on marginal effects from the model. RAG-HPO configurations consistently outperformed traditional concept recognition tools, showing higher and more stable accuracy across the full ontology depth</p></figcaption></figure><p id="Par59">Organ system had a limited influence on accuracy (Fig. <a href="#Fig6" class="usa-link">6</a>A). Cardiovascular, genitourinary, and head/neck terms showed moderately higher odds of correct assignment (odds ratio ≈ 8–11 versus reference), while most other systems clustered around unity. All tools exhibited a monotonic decline in accuracy as terms became more specific (Fig. <a href="#Fig6" class="usa-link">6</a>B); predicted hit probability fell steadily from shallow (depth ≤ 3) to deep nodes (depth ≥ 10). However, tool choice remained the strongest determinant of success (Fig. <a href="#Fig6" class="usa-link">6</a>C). Almost every RAG-HPO configuration out-performed the concept recognition tools FastHPOCR, ClinPhen, and Doc2HPO, with RAG-HPO + LLaMa-3.1 70B achieving the highest predicted hit probability across the depth range.</p></section><section id="Sec11"><h3 class="pmc_sec_title">Evaluation of false positive results among HPO analysis tools</h3>
<p id="Par60">In our analysis, we accepted only the most specific term (highest information content) to describe each phenotype. However, there is value in identifying directly related ancestor terms, which are often broader descriptors of the phenotypes observed (Fig. <a href="#Fig7" class="usa-link">7</a>). To better understand the meaning of these incorrectly attributed terms, we conducted a deeper analysis of the relationship of false positives to the manually annotated HPO terms, specifically focusing on the ontological connections between the HPO terms selected by the programs and those identified through manual annotation. We categorized the false positive HPO terms into three distinct groups: (1) terms that were direct ancestors of the target HPO terms, (2) terms that were indirectly related to the target terms through a shared ancestor, and (3) terms that were completely unrelated to the correct terms. Additionally, for RAG-HPO, we classified any false positive HPO terms that did not exist in the HPO database as “hallucinations” generated by the system (Fig. <a href="#Fig7" class="usa-link">7</a>A).</p>
<figure class="fig xbox font-sm" id="Fig7"><h4 class="obj_head">Fig. 7.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12359922_13073_2025_1521_Fig7_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed34/12359922/b2d990aae69d/13073_2025_1521_Fig7_HTML.jpg" loading="lazy" id="MO7" height="417" width="709" alt="Fig. 7"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig7/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Analysis of incorrect HPO terms assigned to cases by analysis tools. Examination of the HPO terms returned that did not match the manually annotated standard were considered false positives. <strong>A</strong> Depicts the false positive types in relation to the target term (in purple) with direct and indirect relatives being on the same phenotypic abnormality branch, unrelated terms belonging to completely different branches of the HPO, and hallucinations having no existence within the HPO database. <strong>B</strong> We completed a deeper analysis of these terms to better understand their nature and categorized them into distinct groups based on their relationship to the standard: direct ancestors, indirect relatives, hallucinations, and irrelevant terms</p></figcaption></figure><p id="Par61">RAG-HPO paired with LLaMa-3.1 70B generated 315 false positives: 70 direct ancestors (22%), 230 distant relatives (73%), 14 unrelated terms including deprecated codes (4%), and only a single hallucination (&lt; 1%) (Fig. <a href="#Fig7" class="usa-link">7</a>B). The other RAG variants showed a similar pattern, although the less-aligned models (e.g., LLaMA-3 70B and LLaMA-4 Scout) contained a larger unrelated fraction. Among concept recognition tools, FASTHPOCR recorded 729 false positives, dominated by distant relatives (79%) with 13 unrelated terms (2%); ClinPhen and Doc2HPO produced 376 and 264 FPs, respectively, with distant relatives comprising 62–66%. True hallucinations were absent in all three rule-based tools and remained rare for RAG-HPO.</p>
<p id="Par62">Unrelated HPO terms assigned by RAG-HPO rarely included errors attributable to poor semantic similarity search. When RAG-HPO identifies a phenotypic phrase for term assignment, the phrase is vectorized and compared to the custom vector database we created. In rare cases, the search returned database entries that were completely unrelated to the extracted clinical phrases. As currently written, the LLM must choose an HPO term from the list generated by similarity search, a constraint designed to prevent hallucinations, but which can lead to nonsensical HPO choices.</p>
<p id="Par63">A deeper understanding of the false positive population helps us to better understand the reasoning for their development and determine ways to reduce their occurrence and improve our precision. This analysis also highlights the importance of understanding the level of detail needed for specific downstream applications. In some instances, less specific but related terms may be acceptable.</p></section><section id="Sec12"><h3 class="pmc_sec_title">RAG-HPO performance in other datasets</h3>
<p id="Par64">As previously mentioned, we curated the GSC down to 114 notes each containing at least four distinct phenotypes for a total of 1013 HPO terms (415 unique). With this dataset, we evaluated the performance of RAG-HPO, using both LLaMa-3 70B and LLaMa-4 Scout back-ends, against FastHPOCR, Doc2HPO, and ClinPhen.</p>
<p id="Par65">RAG-HPO + LLaMA-3 70B returned 766 true positives, 358 false positives, and 245 false negatives, corresponding to a mean precision of 0.69, recall 0.77, and F1 0.71 (Table <a href="#Tab5" class="usa-link">5</a>). Substituting the larger-context LLaMA-4 Scout backbone increased true positives to 783 but also raised false positives to 490, lowering precision to 0.62 and F1 to 0.68 while maintaining the highest recall (0.79).
</p>
<section class="tw xbox font-sm" id="Tab5"><h4 class="obj_head">Table 5.</h4>
<div class="caption p"><p>Comparison of RAG-HPO performance to other tools in analysis of GSC. While RAG-HPO combined with LLaMA-3 70B achieved F1 scores comparable to FASTHPOCR, it identified a greater number of relevant HPO terms. The performance of LLaMA-4 Scout, despite its smaller size, suggests that recent architectural advances may enable smaller LLMs to perform competitively in phenotypic extraction tasks, potentially reducing computational burden in future applications</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">LLM + RAG-HPO</th>
<th align="left" colspan="1" rowspan="1">True positive</th>
<th align="left" colspan="1" rowspan="1">False positive</th>
<th align="left" colspan="1" rowspan="1">False negative</th>
<th align="left" colspan="1" rowspan="1">Average precision</th>
<th align="left" colspan="1" rowspan="1">Average recall</th>
<th align="left" colspan="1" rowspan="1">Average F1</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">LLaMA-3 70B</td>
<td align="left" colspan="1" rowspan="1">766</td>
<td align="left" colspan="1" rowspan="1">358</td>
<td align="left" colspan="1" rowspan="1">245</td>
<td align="left" colspan="1" rowspan="1">0.69</td>
<td align="left" colspan="1" rowspan="1">0.77</td>
<td align="left" colspan="1" rowspan="1">0.71</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">LLaMA-4 Scout</td>
<td align="left" colspan="1" rowspan="1">783</td>
<td align="left" colspan="1" rowspan="1">490</td>
<td align="left" colspan="1" rowspan="1">228</td>
<td align="left" colspan="1" rowspan="1">0.62</td>
<td align="left" colspan="1" rowspan="1">0.79</td>
<td align="left" colspan="1" rowspan="1">0.68</td>
</tr>
<tr><td align="left" colspan="7" rowspan="1">Other HPO analysis tools</td></tr>
<tr>
<td align="left" colspan="1" rowspan="1"> FASTHPOCR</td>
<td align="left" colspan="1" rowspan="1">696</td>
<td align="left" colspan="1" rowspan="1">241</td>
<td align="left" colspan="1" rowspan="1">315</td>
<td align="left" colspan="1" rowspan="1">0.74</td>
<td align="left" colspan="1" rowspan="1">0.70</td>
<td align="left" colspan="1" rowspan="1">0.70</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> Doc2HPO</td>
<td align="left" colspan="1" rowspan="1">438</td>
<td align="left" colspan="1" rowspan="1">89</td>
<td align="left" colspan="1" rowspan="1">573</td>
<td align="left" colspan="1" rowspan="1">0.80</td>
<td align="left" colspan="1" rowspan="1">0.44</td>
<td align="left" colspan="1" rowspan="1">0.56</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> ClinPhen</td>
<td align="left" colspan="1" rowspan="1">428</td>
<td align="left" colspan="1" rowspan="1">204</td>
<td align="left" colspan="1" rowspan="1">583</td>
<td align="left" colspan="1" rowspan="1">0.61</td>
<td align="left" colspan="1" rowspan="1">0.42</td>
<td align="left" colspan="1" rowspan="1">0.46</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><p id="Par66">Among the concept recognition tools, FastHPOCR achieved the next-best balance (precision 0.74, recall 0.70, F1 0.70), whereas Doc2HPO and ClinPhen prioritized precision (0.80 and 0.61, respectively) at the cost of markedly reduced recall (0.44 and 0.42) and lower F1 scores (0.56 and 0.46).</p>
<p id="Par67">Compared with the CSC, RAG-HPO’s precision decreased by almost 12%, likely because GSC entries are research summaries rather than clinical notes, introducing broader language and citation artifacts that inflate distant-relative matches. However, RAG-HPO retained the highest overall F1 and the strongest recall across this markedly different text genre, confirming the generalizability of the retrieval-augmented framework.</p></section></section><section id="Sec13"><h2 class="pmc_sec_title">Discussion</h2>
<section id="Sec14"><h3 class="pmc_sec_title">Summary</h3>
<p id="Par68">We developed RAG-HPO, a RAG framework designed to enable precise and comprehensive extraction of HPO terms from clinical case narratives using LLMs. RAG-HPO consistently outperforms existing tools in both precision and recall while maintaining flexibility and ease of use (Fig. <a href="#Fig3" class="usa-link">3</a>). While LLMs are capable of identifying phenotypic descriptions in text, they frequently misassign or incompletely map terms to HPO identifiers when operating in isolation (Table <a href="#Tab3" class="usa-link">3</a>). RAG-HPO mitigates these errors by grounding generation in a curated knowledge base during inference. This integration of retrieval reduces the incidence of hallucinated or invalid terms and promotes accurate alignment between clinical language and ontology-based concepts. When evaluated on two independent datasets, the combination of RAG-HPO + LLaMA-3.1 70B achieved the highest F1 scores among all tested tools. It more than doubled the true positive rate of traditional dictionary-based concept recognition systems, with hallucinations accounting for less than 1% of the output (Table <a href="#Tab4" class="usa-link">4</a>).</p></section><section id="Sec15"><h3 class="pmc_sec_title">Strengths</h3>
<p id="Par69">One of RAG-HPO’s key strengths is its adaptability and ease of use, even for users with limited experience in machine learning or language model deployment. Unlike traditional fine-tuning approaches, which demand significant time, computational resources, and specialized expertise, our program leverages RAG to enable high-performance HPO term extraction with minimal technical burden. Users can easily update the vector database with just a few clicks, ensuring that the tool remains current and effective without the need for retraining or model modification.</p>
<p id="Par70">The program is also simple to run, making it accessible to researchers and clinicians with limited technical expertise. At the same time, its modular Python structure allows for easy integration, modification, and ongoing improvement. RAG-HPO enables users to submit plain text without needing to preprocess data or integrate into electronic health records (EHR). This adaptability allows it to handle free-form medical text, making it especially useful in rare disease research, where patient descriptions often vary in format (Fig. <a href="#Fig2" class="usa-link">2</a>A).</p>
<p id="Par71">This flexibility extends to model deployment. RAG-HPO is compatible with any LLM supporting the OpenAI API framework, including those hosted locally for HIPAA-compliant workflows. This model-agnostic design allows researchers and clinicians to select backends that best meet their infrastructure and privacy needs. As demonstrated by our cross-platform analysis of LLM performance, model selection is a key factor in optimizing precision and recall. For example, replacing the baseline LLaMA-3 70B with the alignment-tuned LLaMA-3.1 70B led to notable gains in both precision (0.71 to 0.81) and recall (0.61 to 0.76), resulting in an F1 score of 0.78 (Table <a href="#Tab3" class="usa-link">3</a>). Interestingly, the lightweight LLaMA-4 Scout (16B parameter size) approached a similar level of performance (F1 = 0.66) to LLaMA-3 70B, despite being smaller. These findings highlight the value of backbone selection and suggest that newer model families may continue to improve with future releases.</p>
<p id="Par72">Performance profiling revealed additional insights into program behavior. Organ system had a minimal overall effect on performance, with odds ratios clustering around 1 (Fig. <a href="#Fig6" class="usa-link">6</a>A). Slight boosts were observed for cardiovascular, genitourinary, and head and neck terms, with some reaching odds ratios between 8 and 11. However, the organ system type was not the dominant driver of performance. Nodal depth, which can be used as a measure of HPO term specificity, had a more consistent and monotonic effect: deeper terms reduced hit probability across all tools, reflecting the increased difficulty of matching complex or distal ontology nodes (Fig. <a href="#Fig6" class="usa-link">6</a>B) with described phenotypes. Although all variants of RAG-HPO displayed a similar performance drop rate as other tools, most had a consistently higher F1 compared to the more traditional dictionary-based concept recognition tools.</p></section><section id="Sec16"><h3 class="pmc_sec_title">Tool-to-tool comparisons further underscored RAG-HPO’s advantages</h3>
<p id="Par73">RAG-HPO + LLaMA-3.1 70B tripled the odds of correct HPO assignment compared to Doc2HPO (Fig. <a href="#Fig6" class="usa-link">6</a>C). Even the weakest RAG variant performed as well as or better than the strongest rule-based tool. A trade-off was observed between backbone strength and hit probability behavior: stronger models generally yielded higher recall and captured more distant matches, while weaker models offered slightly higher precision but lower sensitivity overall. Larger context windows, such as those used in LLaMa-4 Scout, appeared to lift recall but also slightly inflated false positives, reflecting a precision-recall trade-off.</p>
<p id="Par74">Finally, RAG-HPO’s structural modularity enables incremental improvement over time. Its components, including system prompts, embedding generation, similarity search, and LLM integration, can be independently modified, swapped, or upgraded. The retrieval database can be updated without retraining the underlying model, ensuring compatibility with evolving versions of the HPO as long as the embedding process for the database is the same as the embedding process used during the deep phenotype extraction process. This makes RAG-HPO not only performant and flexible, but also capable of keeping up with changes in the LLM field.</p>
<p id="Par75">Recently, others have used a predetermined dataset, the GSC, to evaluate their analysis programs [<a href="#CR15" class="usa-link" aria-describedby="CR15">15</a>, <a href="#CR23" class="usa-link" aria-describedby="CR23">23</a>, <a href="#CR35" class="usa-link" aria-describedby="CR35">35</a>]. The GSC was derived from entries in the Online Mendelian Inheritance in Man (OMIM) database and contains a mix of content types, including structured disease descriptions and a subset of passages formatted to resemble clinical notes. While this dataset has served as a comparative benchmark for several tools, it presents significant limitations for evaluating systems designed for clinical NLP and deep phenotype extraction.</p>
<p id="Par76">Many of the GSC entries resemble scientific abstracts or informational summaries describing well-known genetic conditions such as neurofibromatosis, Gorlin syndrome, and branchio-oto-renal syndrome. However, this format often results in annotated truth sets populated with general or lower information-content HPO terms, even when more specific alternatives are appropriate. For instance, in case 1,347,096, the phrase “basal cell carcinoma” appears in the source text, yet the annotation includes both HP:0002671 (“basal cell carcinoma”) and the more general HP:0030731 (“carcinoma”). In contrast, our approach to deep phenotyping emphasizes selecting the single most descriptive and specific term, with the understanding that downstream analyses can leverage the HPO’s hierarchical structure to trace lineages and identify ancestral terms as needed. As a result, reliance on the GSC as a primary evaluation benchmark may underestimate the capabilities of tools like RAG-HPO, which are designed to prioritize specificity, precision, and contextual depth in phenotype assignment.</p>
<p id="Par77">Although we had access to the GSC, we deliberately developed a separate cohort composed of published clinical case reports—the CSC. Unlike the condensed, citation-style summaries found in the GSC, these reports more closely resemble real-world clinical documentation, capturing greater variability, language, and contextual detail more reflective of medical records. To ensure data quality, we applied strict inclusion criteria and employed multilayered manual annotation, with review by medical genetics professionals to promote consistency and accuracy.</p>
<p id="Par78">Beyond differences in size and term count (the GSC contained 114 entries with 415 unique HPO terms (8.9 per case on average), while our CSC included 112 reports with 1208 unique terms and a higher annotation density (15.8 terms per case)), the structural composition of the two datasets further underscores their divergence. The GSC is heavily skewed toward a narrow set of organ systems—primarily nervous, musculoskeletal, and head/neck domains—while significantly underrepresenting others such as endocrine, cardiovascular, and genitourinary systems (Additional file 2: Fig. S4B). This bias reflects the GSC’s origin in genetic reference databases and its focus on well-characterized monogenic conditions. However, the HPO database has matured to support a much broader array of clinical concepts, extending beyond neurological and congenital disorders. In contrast, our case study cohort offers a more balanced and comprehensive distribution across organ systems (Fig. <a href="#Fig3" class="usa-link">3</a>), providing a more robust foundation for evaluating phenotype extraction tools intended for general clinical application.</p>
<p id="Par79">Despite these limitations, RAG-HPO demonstrated strong performance on the GSC dataset, achieving the highest recall (0.77–0.79) and F1 score (0.71) among all tools tested (Table <a href="#Tab4" class="usa-link">4</a>). This underscores the robustness of the RAG-HPO architecture and its ability to generalize across dataset types. However, the relative success of FASTHPOCR on this benchmark may reflect alignment with the dataset’s structure, as both were developed by the same research group—a factor that may inflate apparent performance. These findings highlight the broader challenge of overfitting to familiar benchmarks and reinforce the need for diverse, heterogeneous evaluation sets.</p></section><section id="Sec17"><h3 class="pmc_sec_title">Limitations</h3>
<p id="Par80">One limitation of RAG-HPO is its processing speed. While individual clinical notes may take up to 45 s to analyze, other tools, such as FastHPOCR, can process the entire 112 case cohort in that time. However, this longer runtime for RAG-HPO is offset by its superior precision and comprehensive output, which reduces the need for downstream correction and ultimately supports more accurate deep phenotype extraction.</p>
<p id="Par81">Another challenge is that the precision of RAG-HPO depends heavily on the quality of the vector database used for semantic similarity searches. While our current database is carefully curated and regularly updated, it remains largely oriented toward genetic disease phenotypes, reflecting the historical focus of HPO itself. However, it is also limited by the available HPO terms, which are tailored toward features recognized in medical genetics. This introduces the potential for rare mismatches—particularly when semantic similarity scores rank related but suboptimal terms highly. While retrieval constraints minimize outright hallucinations, they may still return less-than-ideal terms if the embedding space misfires. However, most false positives returned by RAG-HPO are closely related terms rather than completely irrelevant matches. These near misses may still hold clinical value, particularly in differential diagnosis workflows or patient comparative studies where broader phenotypic context is useful.</p>
<p id="Par82">As the HPO database matures, we expect precision to improve. To support this ongoing process, we have made the full program for vector database creation, as well as the supplementary metadata used to enrich each term, publicly available on GitHub. A spreadsheet-format supplement document enables users to view and update curated entries, which are merged with the current hp.obo to regenerate the full index. This FAIR-compliant structure allows end users to integrate their own validated entries and optionally submit them back to the authors for inclusion in future versions after review. Our long-term goal is to promote this as a community-supported effort, enabling sustainable, open-source improvement of the database for as long as the tool remains clinically or scientifically valuable. However, the subjective nature of phenotype interpretation can still influence results, as different users may prioritize varying levels of specificity. While RAG-HPO is designed to identify the most precise HPO term for a given phenotypic phrase, some users may seek a broader characterization of a patient’s phenotype.</p>
<p id="Par83">The inherent subjectivity of clinical evaluations, through which different physicians may arrive at different interpretations of the same patient, can also impact the phenotyping process. Although HPO terms aim to standardize patient presentations, variability in clinical assessment may still affect the final set of phenotypes attributed to an individual patient. While minimized with the use of LLMs in the automated deep phenotyping process, this subjectivity will persist when comparing cases of individuals seen by different physicians. Further downstream analyses focused on comparing HPO term lineages along with the specific HPO terms can further mitigate the impact of clinician subjectivity in patient evaluation.</p>
<p id="Par84">Finally, RAG-HPO’s recall is influenced by the choice of LLM backend. More powerful models with broader contextual understanding generally yield higher recall and better interpretation of complex or obliquely phrased phenotypes. Fortunately, the system is designed to be model-agnostic, allowing users to substitute any OpenAI-compatible LLM. We selected LLaMa-3 70B as a balance between performance and accessibility, but ongoing improvements in open-source language models are likely to yield continued gains in speed, accuracy, and flexibility.</p></section><section id="Sec18"><h3 class="pmc_sec_title">Future directions</h3>
<p id="Par85">Our primary objective in developing RAG-HPO is to create an efficient and user-friendly tool for automated deep phenotyping. Looking ahead, we plan to further improve the vector database by incorporating user contributions of verified phrases that consistently map to established HPO terms. This community-driven approach will support ongoing refinement and ensure the system remains responsive to evolving clinical language.</p>
<p id="Par86">Although RAG-HPO was initially designed for deep phenotype extraction in the context of rare genetic variant discovery, its potential applications extend to other areas of medicine that rely on patient comparison or characterization based on clinical presentation. Future efforts will focus on expanding these use cases to include phenotype clustering, cohort matching, and differential diagnosis. To support these broader goals, we are also exploring hybrid pipelines that combine high-precision dictionary filters with retrieval-based augmentation to balance specificity and recall.</p>
<p id="Par87">As RAG-HPO continues to evolve, we aim to establish it as a generalizable, open-source clinical phenotyping tool. We plan to promote ontology-aware evaluation frameworks that classify errors by term relationships (e.g., ancestor, relative, unrelated), enabling more nuanced assessments of tool performance. These strategies will support more scalable, adaptable applications of RAG-HPO across research and clinical settings.</p></section></section><section id="Sec19"><h2 class="pmc_sec_title">Conclusions</h2>
<p id="Par88">RAG-HPO represents a significant advancement in the field of automated deep phenotyping, demonstrating a marked improvement over traditional HPO tools in both precision and recall. By leveraging retrieval-augmented generation (RAG) and a dynamic vector database of over 54,000 phenotypic terms, RAG-HPO offers clinicians and researchers an accessible, computationally efficient solution for phenotype extraction that is both adaptable and resource-light. Unlike conventional LLM-based tools, RAG-HPO minimizes the need for fine-tuning and reduces the risks of “hallucinations,” making it a reliable and scalable tool for phenotypic analysis, particularly in rare disease diagnosis and clinical genomics.</p>
<p id="Par89">Availability and requirements:</p>
<ul class="list" style="list-style-type:disc">
<li><p id="Par90">Project name: RAG-HPO [<a href="#CR45" class="usa-link" aria-describedby="CR45">45</a>]</p></li>
<li><p id="Par91">Home page: <a href="https://github.com/PoseyPod/RAG-HPO" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/PoseyPod/RAG-HPO</a></p></li>
<li><p id="Par92">Operating system(s): MacOS and Windows</p></li>
<li><p id="Par93">Programming language: Python</p></li>
<li><p id="Par94">Other requirements:</p></li>
<li><p id="Par95">License: MIT, CC BY 4.0 Attribution</p></li>
<li><p id="Par96">Any restrictions to use by non-academics:as per MIT license with appropriate attribution</p></li>
</ul></section><section id="Sec20"><h2 class="pmc_sec_title">Supplementary Information</h2>
<section class="sm xbox font-sm" id="MOESM1"><div class="media p"><div class="caption">
<a href="/articles/instance/12359922/bin/13073_2025_1521_MOESM1_ESM.csv" data-ga-action="click_feat_suppl" class="usa-link">13073_2025_1521_MOESM1_ESM.csv</a><sup> (163.4MB, csv) </sup><p>Additional file 1. Vector database dictionary. This spreadsheet file contains a table of all key-value pairs, mapping each clinical word or phrase (“key”) to an HPO ID (“value”).</p>
</div></div></section><section class="sm xbox font-sm" id="MOESM2"><div class="media p"><div class="caption">
<a href="/articles/instance/12359922/bin/13073_2025_1521_MOESM2_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link">13073_2025_1521_MOESM2_ESM.docx</a><sup> (1.6MB, docx) </sup><p>Additional file 2. Supplemental text and figures. This text file includes supplemental text demonstrated the system messages generated by RAG-HPO and provided to the user, as well as supplemental figures.</p>
</div></div></section><section class="sm xbox font-sm" id="MOESM3"><div class="media p"><div class="caption">
<a href="/articles/instance/12359922/bin/13073_2025_1521_MOESM3_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">13073_2025_1521_MOESM3_ESM.xlsx</a><sup> (361.1KB, xlsx) </sup><p>Additional file 3. RAG-HPO tests and data analysis. This spreadsheet file contains the HPO IDs for the case sets against which benchmarking was performed, as well as the benchmarking data itself, across all tested tools.</p>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements
</h2>
<p>We would like to thank the following for their help and support in creating RAG-HPO: Thanks to Steve Ludke, PhD, Maxim Seferovic, PhD, and Shinya Yamamoto DVM, PhD for your guidance in coding in Python; thanks to Gwendolyn Hummel for biostatistics guidance; and thanks to Cole Deisseroth for being a resource for information on HPO analysis and the application of NLP to deep phenotyping.</p></section><section id="glossary1" class="glossary"><h2 class="pmc_sec_title">Abbreviations</h2>
<dl class="def-list">
<dt>API</dt>
<dd><p id="Par14">Application Programming Interface</p></dd>
<dt>CSC</dt>
<dd><p id="Par15">Case study cohort</p></dd>
<dt>GSC</dt>
<dd><p id="Par16">Gold standardized corpora</p></dd>
<dt>HPO</dt>
<dd><p id="Par17">Human Phenotype Ontology</p></dd>
<dt>JSON</dt>
<dd><p id="Par18">JavaScript Object Notation</p></dd>
<dt>LLM</dt>
<dd><p id="Par19">Large language model</p></dd>
<dt>OMIM</dt>
<dd><p id="Par20">Online Mendelian Inheritance in Man</p></dd>
<dt>RAG</dt>
<dd><p id="Par21">Retrieval-augmented generation</p></dd>
</dl></section><section id="notes1"><h2 class="pmc_sec_title">Authors’ contributions
</h2>
<p>BTG designed the study, created the software, collected, analyzed and interpreted the data, and drafted the manuscript. PY selected the test cohort independently of other authors. BTG and LW created the manually annotated standard. LW, PY, NG, EARM, HD, MD, and AJ assisted with data collection, analysis, and interpretation. JRL assisted with data interpretation. JEP supervised the study design, data collection and analysis. All authors read and approved the final manuscript.</p></section><section id="notes2"><h2 class="pmc_sec_title">Funding</h2>
<p>This study was supported by the US NIH National Human Genome Research Institute (NHGRI) U01 <a href="https://www.ncbi.nlm.nih.gov/nuccore/HG011758" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">HG011758</a> to the Baylor College of Medicine Genomic Research to Elucidate the Genetics of Rare disease center (BCM-GREGoR).</p></section><section id="notes3"><h2 class="pmc_sec_title">Data availability
</h2>
<p>Published case reports and data used to benchmark RAG-HPO are provided within the Supplemental Files. RAG-HPO is available for download at https://github.com/PoseyPod/RAG-HPO.</p></section><section id="notes4"><h2 class="pmc_sec_title">Declarations</h2>
<section id="FPar1"><h3 class="pmc_sec_title">Ethics approval and consent to participate</h3>
<p id="Par97">All clinical texts and data used herein were derived from published case reports available in the medical literature or the published gold standardized corpora. No unpublished clinical or research patient data were implemented in this study.</p></section><section id="FPar2"><h3 class="pmc_sec_title">Consent for publication</h3>
<p id="Par98">Not applicable. No unpublished clinical or research patient data were implemented in this study.</p></section><section id="FPar3"><h3 class="pmc_sec_title">Competing interests
</h3>
<p id="Par99">The Department of Molecular &amp; Human Genetics at Baylor College of Medicine receives revenue from clinical genetic testing conducted at Baylor Genetics Laboratories. JRL is a co-inventor on multiple United States and European patents related to molecular diagnostics for inherited neuropathies, eye diseases, genomic disorders and bacterial genomic fingerprinting. J.E.P. is a member of the advisory board for MaddieBio. The remaining authors declare that they do not have any competing interests.</p></section></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="fn1">
<p><strong>Publisher’s Note</strong></p>
<p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p>
</div></div></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Posey JE, Harel T, Liu P, Rosenfeld JA, James RA, Coban Akdemir ZH, et al. Resolution of disease phenotypes resulting from multilocus genomic variation. N Engl J Med. 2017;376:21–31.
</cite> [<a href="https://doi.org/10.1056/NEJMoa1516767" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5335876/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27959697/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Posey%20JE,%20Harel%20T,%20Liu%20P,%20Rosenfeld%20JA,%20James%20RA,%20Coban%20Akdemir%20ZH,%20et%20al.%20Resolution%20of%20disease%20phenotypes%20resulting%20from%20multilocus%20genomic%20variation.%20N%20Engl%20J%20Med.%202017;376:21%E2%80%9331." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>Posey JE. Genome sequencing and implications for rare disorders. Orphanet J Rare Dis. 2019;14: 153.
</cite> [<a href="https://doi.org/10.1186/s13023-019-1127-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6591893/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31234920/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Posey%20JE.%20Genome%20sequencing%20and%20implications%20for%20rare%20disorders.%20Orphanet%20J%20Rare%20Dis.%202019;14:%20153." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>Liu P, Meng L, Normand EA, Xia F, Song X, Ghazi A, et al. Reanalysis of clinical exome sequencing data. N Engl J Med. 2019;380:2478–80.
</cite> [<a href="https://doi.org/10.1056/NEJMc1812033" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6934160/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31216405/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Liu%20P,%20Meng%20L,%20Normand%20EA,%20Xia%20F,%20Song%20X,%20Ghazi%20A,%20et%20al.%20Reanalysis%20of%20clinical%20exome%20sequencing%20data.%20N%20Engl%20J%20Med.%202019;380:2478%E2%80%9380." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Posey JE, O’Donnell-Luria AH, Chong JX, Harel T, Jhangiani SN, Coban Akdemir ZH, et al. Insights into genetics, human biology and disease gleaned from family based genomic studies. Genet Med. 2019;21:798–812.
</cite> [<a href="https://doi.org/10.1038/s41436-018-0408-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6691975/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30655598/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Posey%20JE,%20O%E2%80%99Donnell-Luria%20AH,%20Chong%20JX,%20Harel%20T,%20Jhangiani%20SN,%20Coban%20Akdemir%20ZH,%20et%20al.%20Insights%20into%20genetics,%20human%20biology%20and%20disease%20gleaned%20from%20family%20based%20genomic%20studies.%20Genet%20Med.%202019;21:798%E2%80%93812." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Posey JE, Lupski JR. Genomics in clinical practice. N Engl J Med. 2023;388:1619-20.
</cite> [<a href="https://doi.org/10.1056/NEJMe2302643" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37043638/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Posey%20JE,%20Lupski%20JR.%20Genomics%20in%20clinical%20practice.%20N%20Engl%20J%20Med.%202023;388:1619-20." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Wright CF, Campbell P, Eberhardt RY, Aitken S, Perrett D, Brent S, et al. Genomic diagnosis of rare pediatric disease in the United Kingdom and Ireland. N Engl J Med. 2023;388:1559–71.
</cite> [<a href="https://doi.org/10.1056/NEJMoa2209046." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7614484/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37043637/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wright%20CF,%20Campbell%20P,%20Eberhardt%20RY,%20Aitken%20S,%20Perrett%20D,%20Brent%20S,%20et%20al.%20Genomic%20diagnosis%20of%20rare%20pediatric%20disease%20in%20the%20United%20Kingdom%20and%20Ireland.%20N%20Engl%20J%20Med.%202023;388:1559%E2%80%9371." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Rehm HL, Berg JS, Brooks LD, Bustamante CD, Evans JP, Landrum MJ, et al. ClinGen–the clinical genome resource. N Engl J Med. 2015;372:2235–42.
</cite> [<a href="https://doi.org/10.1056/NEJMsr1406261" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4474187/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26014595/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rehm%20HL,%20Berg%20JS,%20Brooks%20LD,%20Bustamante%20CD,%20Evans%20JP,%20Landrum%20MJ,%20et%20al.%20ClinGen%E2%80%93the%20clinical%20genome%20resource.%20N%20Engl%20J%20Med.%202015;372:2235%E2%80%9342." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>Robinson PN, Mundlos S. The human phenotype ontology. Clin Genet. 2010;77:525–34.
</cite> [<a href="https://doi.org/10.1111/j.1399-0004.2010.01436.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20412080/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Robinson%20PN,%20Mundlos%20S.%20The%20human%20phenotype%20ontology.%20Clin%20Genet.%202010;77:525%E2%80%9334." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>Robinson PN, Köhler S, Bauer S, Seelow D, Horn D, Mundlos S. The human phenotype ontology: a tool for annotating and analyzing human hereditary disease. Am J Hum Genet. 2008;83:610–5.
</cite> [<a href="https://doi.org/10.1016/j.ajhg.2008.09.017" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2668030/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18950739/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Robinson%20PN,%20K%C3%B6hler%20S,%20Bauer%20S,%20Seelow%20D,%20Horn%20D,%20Mundlos%20S.%20The%20human%20phenotype%20ontology:%20a%20tool%20for%20annotating%20and%20analyzing%20human%20hereditary%20disease.%20Am%20J%20Hum%20Genet.%202008;83:610%E2%80%935." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>Putman TE, Schaper K, Matentzoglu N, Rubinetti VP, Alquaddoomi FS, Cox C, et al. The Monarch Initiative in 2024: an analytic platform integrating phenotypes, genes and diseases across species. Nucleic Acids Res. 2024;52:D938–49.
</cite> [<a href="https://doi.org/10.1093/nar/gkad1082" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10767791/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38000386/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Putman%20TE,%20Schaper%20K,%20Matentzoglu%20N,%20Rubinetti%20VP,%20Alquaddoomi%20FS,%20Cox%20C,%20et%20al.%20The%20Monarch%20Initiative%20in%202024:%20an%20analytic%20platform%20integrating%20phenotypes,%20genes%20and%20diseases%20across%20species.%20Nucleic%20Acids%20Res.%202024;52:D938%E2%80%9349." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Son JH, Xie G, Yuan C, Ena L, Li Z, Goldstein A, et al. Deep phenotyping on electronic health records facilitates genetic diagnosis by clinical exomes. Am J Hum Genet. 2018;103:58–73.
</cite> [<a href="https://doi.org/10.1016/j.ajhg.2018.05.010" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6035281/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29961570/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Son%20JH,%20Xie%20G,%20Yuan%20C,%20Ena%20L,%20Li%20Z,%20Goldstein%20A,%20et%20al.%20Deep%20phenotyping%20on%20electronic%20health%20records%20facilitates%20genetic%20diagnosis%20by%20clinical%20exomes.%20Am%20J%20Hum%20Genet.%202018;103:58%E2%80%9373." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Liu C, Peres Kury FS, Li Z, Ta C, Wang K, Weng C. Doc2hpo: a web application for efficient and accurate HPO concept curation. Nucleic Acids Res. 2019;47:W566–70.
</cite> [<a href="https://doi.org/10.1093/nar/gkz386" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6602487/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31106327/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Liu%20C,%20Peres%20Kury%20FS,%20Li%20Z,%20Ta%20C,%20Wang%20K,%20Weng%20C.%20Doc2hpo:%20a%20web%20application%20for%20efficient%20and%20accurate%20HPO%20concept%20curation.%20Nucleic%20Acids%20Res.%202019;47:W566%E2%80%9370." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>Deisseroth CA, Birgmeier J, Bodle EE, Kohler JN, Matalon DR, Nazarenko Y, et al. ClinPhen extracts and prioritizes patient phenotypes directly from medical records to expedite genetic disease diagnosis. Genet Med. 2019;21:1585–93.
</cite> [<a href="https://doi.org/10.1038/s41436-018-0381-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6551315/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30514889/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Deisseroth%20CA,%20Birgmeier%20J,%20Bodle%20EE,%20Kohler%20JN,%20Matalon%20DR,%20Nazarenko%20Y,%20et%20al.%20ClinPhen%20extracts%20and%20prioritizes%20patient%20phenotypes%20directly%20from%20medical%20records%20to%20expedite%20genetic%20disease%20diagnosis.%20Genet%20Med.%202019;21:1585%E2%80%9393." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>Karaca E, Posey JE, Coban Akdemir Z, Pehlivan D, Harel T, Jhangiani SN, et al. Phenotypic expansion illuminates multilocus pathogenic variation. Genet Med. 2018;20:1528–37.
</cite> [<a href="https://doi.org/10.1038/gim.2018.33" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6450542/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29790871/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Karaca%20E,%20Posey%20JE,%20Coban%20Akdemir%20Z,%20Pehlivan%20D,%20Harel%20T,%20Jhangiani%20SN,%20et%20al.%20Phenotypic%20expansion%20illuminates%20multilocus%20pathogenic%20variation.%20Genet%20Med.%202018;20:1528%E2%80%9337." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>Groza T, Köhler S, Doelken S, Collier N, Oellrich A, Smedley D, et al. Automatic concept recognition using the human phenotype ontology reference and test suite corpora. Database (Oxford). 2015;2015:bav005.</cite> [<a href="https://doi.org/10.1093/database/bav005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4343077/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25725061/" class="usa-link">PubMed</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Köhler S, Øien NC, Buske OJ, Groza T, Jacobsen JOB, McNamara C, et al. Encoding clinical data with the human phenotype ontology for computational differential diagnostics. Curr Protoc Hum Genet. 2019;103: e92.
</cite> [<a href="https://doi.org/10.1002/cphg.92" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6814016/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31479590/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?K%C3%B6hler%20S,%20%C3%98ien%20NC,%20Buske%20OJ,%20Groza%20T,%20Jacobsen%20JOB,%20McNamara%20C,%20et%20al.%20Encoding%20clinical%20data%20with%20the%20human%20phenotype%20ontology%20for%20computational%20differential%20diagnostics.%20Curr%20Protoc%20Hum%20Genet.%202019;103:%20e92." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>Bone WP, Washington NL, Buske OJ, Adams DR, Davis J, Draper D, et al. Computational evaluation of exome sequence data using human and model organism phenotypes improves diagnostic efficiency. Genet Med. 2016;18:608–17.
</cite> [<a href="https://doi.org/10.1038/gim.2015.137" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4916229/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26562225/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bone%20WP,%20Washington%20NL,%20Buske%20OJ,%20Adams%20DR,%20Davis%20J,%20Draper%20D,%20et%20al.%20Computational%20evaluation%20of%20exome%20sequence%20data%20using%20human%20and%20model%20organism%20phenotypes%20improves%20diagnostic%20efficiency.%20Genet%20Med.%202016;18:608%E2%80%9317." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Cipriani V, Pontikos N, Arno G, Sergouniotis PI, Lenassi E, Thawong P, et al. An improved phenotype-driven tool for rare Mendelian variant prioritization: benchmarking Exomiser on real patient whole-exome data. Genes. 2020;11:460.
</cite> [<a href="https://doi.org/10.3390/genes11040460" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7230372/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32340307/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cipriani%20V,%20Pontikos%20N,%20Arno%20G,%20Sergouniotis%20PI,%20Lenassi%20E,%20Thawong%20P,%20et%20al.%20An%20improved%20phenotype-driven%20tool%20for%20rare%20Mendelian%20variant%20prioritization:%20benchmarking%20Exomiser%20on%20real%20patient%20whole-exome%20data.%20Genes.%202020;11:460." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Smedley D, Jacobsen JOB, Jäger M, Köhler S, Holtgrewe M, Schubach M, et al. Next-generation diagnostics and disease-gene discovery with the Exomiser. Nat Protoc. 2015;10:2004–15.
</cite> [<a href="https://doi.org/10.1038/nprot.2015.124" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5467691/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26562621/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Smedley%20D,%20Jacobsen%20JOB,%20J%C3%A4ger%20M,%20K%C3%B6hler%20S,%20Holtgrewe%20M,%20Schubach%20M,%20et%20al.%20Next-generation%20diagnostics%20and%20disease-gene%20discovery%20with%20the%20Exomiser.%20Nat%20Protoc.%202015;10:2004%E2%80%9315." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>Doğan T. HPO2GO: prediction of human phenotype ontology term associations for proteins using cross ontology annotation co-occurrences. PeerJ. 2018;6: e5298.
</cite> [<a href="https://doi.org/10.7717/peerj.5298" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6076985/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30083448/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Do%C4%9Fan%20T.%20HPO2GO:%20prediction%20of%20human%20phenotype%20ontology%20term%20associations%20for%20proteins%20using%20cross%20ontology%20annotation%20co-occurrences.%20PeerJ.%202018;6:%20e5298." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>Xue H, Peng J, Shang X. Predicting disease-related phenotypes using an integrated phenotype similarity measurement based on HPO. BMC Syst Biol. 2019;13(Suppl 2): 34.
</cite> [<a href="https://doi.org/10.1186/s12918-019-0697-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6449884/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30953559/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Xue%20H,%20Peng%20J,%20Shang%20X.%20Predicting%20disease-related%20phenotypes%20using%20an%20integrated%20phenotype%20similarity%20measurement%20based%20on%20HPO.%20BMC%20Syst%20Biol.%202019;13(Suppl%202):%2034." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>Dohi E, Takatsuki T, Tateisi Y, Fujiwara T, Yamamoto Y. Examining HPO by organ and system to facilitate practical use by clinicians. Genomics Inform. 2024;22: 23.
</cite> [<a href="https://doi.org/10.1186/s44342-024-00024-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11559069/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39533429/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Dohi%20E,%20Takatsuki%20T,%20Tateisi%20Y,%20Fujiwara%20T,%20Yamamoto%20Y.%20Examining%20HPO%20by%20organ%20and%20system%20to%20facilitate%20practical%20use%20by%20clinicians.%20Genomics%20Inform.%202024;22:%2023." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>Groza T, Gration D, Baynam G, Robinson PN. FastHPOCR: pragmatic, fast, and accurate concept recognition using the human phenotype ontology. Bioinformatics. 2024;40:btae406.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btae406" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11227366/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38913850/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Groza%20T,%20Gration%20D,%20Baynam%20G,%20Robinson%20PN.%20FastHPOCR:%20pragmatic,%20fast,%20and%20accurate%20concept%20recognition%20using%20the%20human%20phenotype%20ontology.%20Bioinformatics.%202024;40:btae406." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>Yan C, Ong HH, Grabowska ME, Krantz MS, Su W-C, Dickson AL, et al. Large language models facilitate the generation of electronic health record phenotyping algorithms. J Am Med Inform Assoc. 2024;31(9):1994–2001.
</cite> [<a href="https://doi.org/10.1093/jamia/ocae072" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11339509/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38613820/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yan%20C,%20Ong%20HH,%20Grabowska%20ME,%20Krantz%20MS,%20Su%20W-C,%20Dickson%20AL,%20et%20al.%20Large%20language%20models%20facilitate%20the%20generation%20of%20electronic%20health%20record%20phenotyping%20algorithms.%20J%20Am%20Med%20Inform%20Assoc.%202024;31(9):1994%E2%80%932001." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>Zeng Z, Deng Y, Li X, Naumann T, Luo Y. Natural language processing for EHR-based computational phenotyping. IEEE ACM Trans Comput Biol Bioinform. 2019;16:139–53.
</cite> [<a href="https://doi.org/10.1109/TCBB.2018.2849968" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6388621/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29994486/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zeng%20Z,%20Deng%20Y,%20Li%20X,%20Naumann%20T,%20Luo%20Y.%20Natural%20language%20processing%20for%20EHR-based%20computational%20phenotyping.%20IEEE%20ACM%20Trans%20Comput%20Biol%20Bioinform.%202019;16:139%E2%80%9353." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>Kim J, Wang K, Weng C, Liu C. Assessing the utility of large language models for phenotype-driven gene prioritization in the diagnosis of rare genetic disease. Am J Hum Genet. 2024;111:2190–202.
</cite> [<a href="https://doi.org/10.1016/j.ajhg.2024.08.010" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11480789/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39255797/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kim%20J,%20Wang%20K,%20Weng%20C,%20Liu%20C.%20Assessing%20the%20utility%20of%20large%20language%20models%20for%20phenotype-driven%20gene%20prioritization%20in%20the%20diagnosis%20of%20rare%20genetic%20disease.%20Am%20J%20Hum%20Genet.%202024;111:2190%E2%80%93202." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Groza T, Caufield H, Gration D, Baynam G, Haendel MA, Robinson PN, et al. An evaluation of GPT models for phenotype concept recognition. BMC Med Inform Decis Mak. 2024;24: 30.
</cite> [<a href="https://doi.org/10.1186/s12911-024-02439-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10829255/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38297371/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Groza%20T,%20Caufield%20H,%20Gration%20D,%20Baynam%20G,%20Haendel%20MA,%20Robinson%20PN,%20et%20al.%20An%20evaluation%20of%20GPT%20models%20for%20phenotype%20concept%20recognition.%20BMC%20Med%20Inform%20Decis%20Mak.%202024;24:%2030." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Gargari OK, Fatehi F, Mohammadi I, Firouzabadi SR, Shafiee A, Habibi G. Diagnostic accuracy of large language models in psychiatry. Asian J Psychiatr. 2024;100: 104168.
</cite> [<a href="https://doi.org/10.1016/j.ajp.2024.104168" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39111087/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gargari%20OK,%20Fatehi%20F,%20Mohammadi%20I,%20Firouzabadi%20SR,%20Shafiee%20A,%20Habibi%20G.%20Diagnostic%20accuracy%20of%20large%20language%20models%20in%20psychiatry.%20Asian%20J%20Psychiatr.%202024;100:%20104168." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>Young CC, Enichen E, Rivera C, Auger CA, Grant N, Rao A, et al. Diagnostic accuracy of a custom large language model on rare pediatric disease case reports. Am J Med Genet A. 2025;197: e63878.
</cite> [<a href="https://doi.org/10.1002/ajmg.a.63878" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC12123583/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39268988/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Young%20CC,%20Enichen%20E,%20Rivera%20C,%20Auger%20CA,%20Grant%20N,%20Rao%20A,%20et%20al.%20Diagnostic%20accuracy%20of%20a%20custom%20large%20language%20model%20on%20rare%20pediatric%20disease%20case%20reports.%20Am%20J%20Med%20Genet%20A.%202025;197:%20e63878." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR30">
<span class="label">30.</span><cite>Li Y, Zhao J, Li M, Dang Y, Yu E, Li J, et al. RefAI: a GPT-powered retrieval-augmented generative tool for biomedical literature recommendation and summarization. J Am Med Inform Assoc. 2024;31:2030–9.
</cite> [<a href="https://doi.org/10.1093/jamia/ocae129" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11339508/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38857454/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li%20Y,%20Zhao%20J,%20Li%20M,%20Dang%20Y,%20Yu%20E,%20Li%20J,%20et%20al.%20RefAI:%20a%20GPT-powered%20retrieval-augmented%20generative%20tool%20for%20biomedical%20literature%20recommendation%20and%20summarization.%20J%20Am%20Med%20Inform%20Assoc.%202024;31:2030%E2%80%939." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>Cui H, Wang C, Maan H, Pang K, Luo F, Duan N, et al. ScGPT: toward building a foundation model for single-cell multi-omics using generative AI. Nat Methods. 2024;21:1470–80.
</cite> [<a href="https://doi.org/10.1038/s41592-024-02201-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38409223/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cui%20H,%20Wang%20C,%20Maan%20H,%20Pang%20K,%20Luo%20F,%20Duan%20N,%20et%20al.%20ScGPT:%20toward%20building%20a%20foundation%20model%20for%20single-cell%20multi-omics%20using%20generative%20AI.%20Nat%20Methods.%202024;21:1470%E2%80%9380." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>Li Q, Li L, Li Y. Developing ChatGPT for biology and medicine: a complete review of biomedical question answering. Biophys Rep. 2024;10:152–71.
</cite> [<a href="https://doi.org/10.52601/bpr.2024.240004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11252240/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39027317/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li%20Q,%20Li%20L,%20Li%20Y.%20Developing%20ChatGPT%20for%20biology%20and%20medicine:%20a%20complete%20review%20of%20biomedical%20question%20answering.%20Biophys%20Rep.%202024;10:152%E2%80%9371." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Luo L, Yan S, Lai P-T, Veltri D, Oler A, Xirasagar S, et al. PhenoTagger: a hybrid method for phenotype concept recognition using human phenotype ontology. Bioinformatics. 2021;37:1884–90.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btab019" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11025364/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33471061/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Luo%20L,%20Yan%20S,%20Lai%20P-T,%20Veltri%20D,%20Oler%20A,%20Xirasagar%20S,%20et%20al.%20PhenoTagger:%20a%20hybrid%20method%20for%20phenotype%20concept%20recognition%20using%20human%20phenotype%20ontology.%20Bioinformatics.%202021;37:1884%E2%80%9390." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>Feng Y, Qi L, Tian W. Phenobert: a combined deep learning method for automated recognition of human phenotype ontology. IEEE ACM Trans Comput Biol Bioinform. 2023;20:1269–77.
</cite> [<a href="https://doi.org/10.1109/TCBB.2022.3170301" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35471885/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Feng%20Y,%20Qi%20L,%20Tian%20W.%20Phenobert:%20a%20combined%20deep%20learning%20method%20for%20automated%20recognition%20of%20human%20phenotype%20ontology.%20IEEE%20ACM%20Trans%20Comput%20Biol%20Bioinform.%202023;20:1269%E2%80%9377." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>Yang J, Liu C, Deng W, Wu D, Weng C, Zhou Y, et al. Enhancing phenotype recognition in clinical notes using large language models: PhenoBCBERT and PhenoGPT. Patterns. 2024;5: 100887.
</cite> [<a href="https://doi.org/10.1016/j.patter.2023.100887" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10801236/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38264716/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yang%20J,%20Liu%20C,%20Deng%20W,%20Wu%20D,%20Weng%20C,%20Zhou%20Y,%20et%20al.%20Enhancing%20phenotype%20recognition%20in%20clinical%20notes%20using%20large%20language%20models:%20PhenoBCBERT%20and%20PhenoGPT.%20Patterns.%202024;5:%20100887." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR36">
<span class="label">36.</span><cite>Soong D, Sridhar S, Si H, Wagner J-S, Sá ACC, Yu CY, et al. Improving accuracy of GPT-3/4 results on biomedical data using a retrieval-augmented language model. PLoS Digit Health. 2024;3: e0000568.
</cite> [<a href="https://doi.org/10.1371/journal.pdig.0000568" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11338460/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39167594/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Soong%20D,%20Sridhar%20S,%20Si%20H,%20Wagner%20J-S,%20S%C3%A1%20ACC,%20Yu%20CY,%20et%20al.%20Improving%20accuracy%20of%20GPT-3/4%20results%20on%20biomedical%20data%20using%20a%20retrieval-augmented%20language%20model.%20PLoS%20Digit%20Health.%202024;3:%20e0000568." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR37">
<span class="label">37.</span><cite> Yang Y, Bean AM, McCraith R, Mahdi A. Evaluating fine-tuning efficiency of human-inspired learning strategies in medical question Answering. arXiv. 2024.</cite>
</li>
<li id="CR38">
<span class="label">38.</span><cite> Wang A, Liu C, Yang J, Weng C. Fine-tuning large language models for rare disease concept normalization. BioRxiv. 2024.</cite> [<a href="https://doi.org/10.1093/jamia/ocae133" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11339522/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38829731/" class="usa-link">PubMed</a>]</li>
<li id="CR39">
<span class="label">39.</span><cite>Jeong M, Sohn J, Sung M, Kang J. Improving medical reasoning through retrieval and self-reflection with retrieval-augmented large language models. Bioinformatics. 2024;40(Suppl 1):i119–29.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btae238" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11211826/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38940167/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jeong%20M,%20Sohn%20J,%20Sung%20M,%20Kang%20J.%20Improving%20medical%20reasoning%20through%20retrieval%20and%20self-reflection%20with%20retrieval-augmented%20large%20language%20models.%20Bioinformatics.%202024;40(Suppl%201):i119%E2%80%9329." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR40">
<span class="label">40.</span><cite> Qi X, Zeng Y, Xie T, Chen P-Y, Jia R, Mittal P, et al. Fine-tuning aligned language models compromises safety, even when users do not intend to! arXiv. 2023.</cite>
</li>
<li id="CR41">
<span class="label">41.</span><cite> Luo Y, Yang Z, Meng F, Li Y, Zhou J, Zhang Y. An empirical study of catastrophic forgetting in large language models during continual fine-tuning. arXiv. 2023.</cite>
</li>
<li id="CR42">
<span class="label">42.</span><cite> Lewis P, Perez E, Piktus A, Petroni F, Karpukhin V, Goyal N, et al. Retrieval-augmented generation for knowledge-intensive NLP tasks. arXiv. 2020.</cite>
</li>
<li id="CR43">
<span class="label">43.</span><cite>Thomo A. PubMed retrieval with RAG techniques. Stud Health Technol Inform. 2024;316:652–3.
</cite> [<a href="https://doi.org/10.3233/SHTI240498" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39176826/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Thomo%20A.%20PubMed%20retrieval%20with%20RAG%20techniques.%20Stud%20Health%20Technol%20Inform.%202024;316:652%E2%80%933." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR44">
<span class="label">44.</span><cite> Gupta S, Ranjan R, Singh SN. A comprehensive survey of retrieval-augmented generation (RAG): evolution, current landscape and future directions. arXiv. 2024.</cite>
</li>
<li id="CR45">
<span class="label">45.</span><cite> Garcia B, Posey JE. RAG-HPO. GitHub. <a href="https://github.com/PoseyPod/RAG-HPO" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/PoseyPod/RAG-HPO</a> (2024).</cite>
</li>
<li id="CR46">
<span class="label">46.</span><cite> Deisseroth CA, Birgmeier J, Bodle EE, Kohler J, Matalon D, Nazarenko Y, Genetti CA, Brownstein CA, Schmitz-Abe K, Schoch K, Cope H, Signer R; Undiagnosed Diseases Network, Martinez-Agosto JA, Shashi V, Beggs AH, Wheeler MT, Bernstein JA, Bejerano G. ClinPhen. Computer software. <a href="http://bejerano.stanford.edu/clinphen/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://bejerano.stanford.edu/clinphen/</a> (2018).</cite>
</li>
<li id="CR47">
<span class="label">47.</span><cite> Liu C, Kury FSP, Li Z, Ta C, Wang K, Weng C. Doc2Hpo. Github. <a href="https://github.com/stormliucong/doc2hpo" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/stormliucong/doc2hpo</a> (2019).</cite>
</li>
<li id="CR48">
<span class="label">48.</span><cite> Groza T. FastHPOCR. Github. <a href="https://github.com/tudorgroza/fast_hpo_cr" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/tudorgroza/fast_hpo_cr</a> (2024).</cite>
</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12359922/bin/13073_2025_1521_MOESM1_ESM.csv" data-ga-action="click_feat_suppl" class="usa-link">13073_2025_1521_MOESM1_ESM.csv</a><sup> (163.4MB, csv) </sup><p>Additional file 1. Vector database dictionary. This spreadsheet file contains a table of all key-value pairs, mapping each clinical word or phrase (“key”) to an HPO ID (“value”).</p>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12359922/bin/13073_2025_1521_MOESM2_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link">13073_2025_1521_MOESM2_ESM.docx</a><sup> (1.6MB, docx) </sup><p>Additional file 2. Supplemental text and figures. This text file includes supplemental text demonstrated the system messages generated by RAG-HPO and provided to the user, as well as supplemental figures.</p>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12359922/bin/13073_2025_1521_MOESM3_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">13073_2025_1521_MOESM3_ESM.xlsx</a><sup> (361.1KB, xlsx) </sup><p>Additional file 3. RAG-HPO tests and data analysis. This spreadsheet file contains the HPO IDs for the case sets against which benchmarking was performed, as well as the benchmarking data itself, across all tested tools.</p>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>Published case reports and data used to benchmark RAG-HPO are provided within the Supplemental Files. RAG-HPO is available for download at https://github.com/PoseyPod/RAG-HPO.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Genome Medicine are provided here courtesy of <strong>BMC</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1186/s13073-025-01521-w"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/13073_2025_Article_1521.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (1.8 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12359922/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12359922/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12359922%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12359922/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12359922/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12359922/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40826123/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12359922/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40826123/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12359922/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12359922/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="JmVxip1jxjFIBM6W93RgAG2UMNvo3X2ko8QOlSy2hIvLSZidd5yGg2eoccBdCMVF">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
        <script type="module" crossorigin="" src="/static/assets/math-574fdcc6.js"></script>
    
    

    </body>
</html>
