
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Stabilization of norovirus GII.3 virus-like particles by rational disulfide engineering - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="15254C408AF3B163054C40003E8259EF.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="npjvac">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365219/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="NPJ Vaccines">
<meta name="citation_title" content="Stabilization of norovirus GII.3 virus-like particles by rational disulfide engineering">
<meta name="citation_author" content="Christopher Warren">
<meta name="citation_author_institution" content="Infectious Diseases and Vaccines Discovery, Merck &amp; Co., Inc., West Point, PA USA">
<meta name="citation_author" content="Jennifer D Galli">
<meta name="citation_author_institution" content="Infectious Diseases and Vaccines Discovery, Merck &amp; Co., Inc., West Point, PA USA">
<meta name="citation_author" content="Karin Bystol">
<meta name="citation_author_institution" content="Infectious Diseases and Vaccines Discovery, Merck &amp; Co., Inc., West Point, PA USA">
<meta name="citation_author" content="Gregory O’Donnell">
<meta name="citation_author_institution" content="Quantitative Biosciences, Merck &amp; Co., Inc., West Point, PA USA">
<meta name="citation_author" content="Andrew R Swartz">
<meta name="citation_author_institution" content="Process Research &amp; Development, Merck &amp; Co., Inc., West Point, PA USA">
<meta name="citation_author" content="Emily A Dewar">
<meta name="citation_author_institution" content="Process Research &amp; Development, Merck &amp; Co., Inc., West Point, PA USA">
<meta name="citation_author" content="Corey May Fulton">
<meta name="citation_author_institution" content="Quantitative Biosciences, Merck &amp; Co., Inc., West Point, PA USA">
<meta name="citation_author" content="Pamela Shen">
<meta name="citation_author_institution" content="Quantitative Biosciences, Merck &amp; Co., Inc., West Point, PA USA">
<meta name="citation_author" content="Estibaliz Gonzalez-Fernandez">
<meta name="citation_author_institution" content="Analytical Research and Development, Merck &amp; Co., Inc, West Point, PA USA">
<meta name="citation_author" content="Lizzy Aurora DeWitt">
<meta name="citation_author_institution" content="Infectious Diseases and Vaccines Discovery, Merck &amp; Co., Inc., West Point, PA USA">
<meta name="citation_author" content="Uijin Jeong">
<meta name="citation_author_institution" content="Quantitative Biosciences, Merck &amp; Co., Inc., West Point, PA USA">
<meta name="citation_author" content="Oscar Chi-Chien Pan">
<meta name="citation_author_institution" content="Infectious Diseases and Vaccines Discovery, Merck &amp; Co., Inc., West Point, PA USA">
<meta name="citation_author" content="Sean Miller">
<meta name="citation_author_institution" content="Analytical Research and Development, Merck &amp; Co., Inc, West Point, PA USA">
<meta name="citation_author" content="Arthur Fridman">
<meta name="citation_author_institution" content="Data Science and Scientific Informatics, RaDS-IT, Merck &amp; Co., Inc., Rahway, NJ USA">
<meta name="citation_author" content="Courtney David">
<meta name="citation_author_institution" content="Discovery Pharmaceutical Sciences, Merck &amp; Co., Inc., West Point, PA USA">
<meta name="citation_author" content="Zhifeng Chen">
<meta name="citation_author_institution" content="Infectious Diseases and Vaccines Discovery, Merck &amp; Co., Inc., West Point, PA USA">
<meta name="citation_author" content="Jiajie Wei">
<meta name="citation_author_institution" content="Infectious Diseases and Vaccines Discovery, Merck &amp; Co., Inc., West Point, PA USA">
<meta name="citation_publication_date" content="2025 Aug 19">
<meta name="citation_volume" content="10">
<meta name="citation_firstpage" content="196">
<meta name="citation_doi" content="10.1038/s41541-025-01254-2">
<meta name="citation_pmid" content="40830611">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365219/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365219/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365219/pdf/41541_2025_Article_1254.pdf">
<meta name="description" content="Noroviruses are non-enveloped, single-stranded positive-sense RNA viruses and the leading cause of gastroenteritis worldwide. The major capsid protein, VP1, can self-assemble into non-infectious virus-like particles (VLPs), representing an ...">
<meta name="og:title" content="Stabilization of norovirus GII.3 virus-like particles by rational disulfide engineering">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Noroviruses are non-enveloped, single-stranded positive-sense RNA viruses and the leading cause of gastroenteritis worldwide. The major capsid protein, VP1, can self-assemble into non-infectious virus-like particles (VLPs), representing an ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365219/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12365219">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1038/s41541-025-01254-2"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/41541_2025_Article_1254.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12365219%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12365219/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12365219/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365219/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-npjvac.jpg" alt="NPJ Vaccines logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to NPJ Vaccines" title="Link to NPJ Vaccines" shape="default" href="https://www.nature.com/npjvaccines/" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">NPJ Vaccines</button></div>. 2025 Aug 19;10:196. doi: <a href="https://doi.org/10.1038/s41541-025-01254-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1038/s41541-025-01254-2</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22NPJ%20Vaccines%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22NPJ%20Vaccines%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22NPJ%20Vaccines%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22NPJ%20Vaccines%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Stabilization of norovirus GII.3 virus-like particles by rational disulfide engineering</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Warren%20C%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Christopher Warren</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Christopher Warren</span></h3>
<div class="p">
<sup>1</sup>Infectious Diseases and Vaccines Discovery, Merck &amp; Co., Inc., West Point, PA USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Warren%20C%22%5BAuthor%5D" class="usa-link"><span class="name western">Christopher Warren</span></a>
</div>
</div>
<sup>1,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Galli%20JD%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Jennifer D Galli</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Jennifer D Galli</span></h3>
<div class="p">
<sup>1</sup>Infectious Diseases and Vaccines Discovery, Merck &amp; Co., Inc., West Point, PA USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Galli%20JD%22%5BAuthor%5D" class="usa-link"><span class="name western">Jennifer D Galli</span></a>
</div>
</div>
<sup>1,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bystol%20K%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Karin Bystol</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Karin Bystol</span></h3>
<div class="p">
<sup>1</sup>Infectious Diseases and Vaccines Discovery, Merck &amp; Co., Inc., West Point, PA USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bystol%20K%22%5BAuthor%5D" class="usa-link"><span class="name western">Karin Bystol</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22O%E2%80%99Donnell%20G%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Gregory O’Donnell</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Gregory O’Donnell</span></h3>
<div class="p">
<sup>2</sup>Quantitative Biosciences, Merck &amp; Co., Inc., West Point, PA USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22O%E2%80%99Donnell%20G%22%5BAuthor%5D" class="usa-link"><span class="name western">Gregory O’Donnell</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Swartz%20AR%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Andrew R Swartz</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Andrew R Swartz</span></h3>
<div class="p">
<sup>3</sup>Process Research &amp; Development, Merck &amp; Co., Inc., West Point, PA USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Swartz%20AR%22%5BAuthor%5D" class="usa-link"><span class="name western">Andrew R Swartz</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Dewar%20EA%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Emily A Dewar</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Emily A Dewar</span></h3>
<div class="p">
<sup>3</sup>Process Research &amp; Development, Merck &amp; Co., Inc., West Point, PA USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Dewar%20EA%22%5BAuthor%5D" class="usa-link"><span class="name western">Emily A Dewar</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Fulton%20CM%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Corey May Fulton</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Corey May Fulton</span></h3>
<div class="p">
<sup>2</sup>Quantitative Biosciences, Merck &amp; Co., Inc., West Point, PA USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Fulton%20CM%22%5BAuthor%5D" class="usa-link"><span class="name western">Corey May Fulton</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Shen%20P%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Pamela Shen</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Pamela Shen</span></h3>
<div class="p">
<sup>2</sup>Quantitative Biosciences, Merck &amp; Co., Inc., West Point, PA USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Shen%20P%22%5BAuthor%5D" class="usa-link"><span class="name western">Pamela Shen</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gonzalez-Fernandez%20E%22%5BAuthor%5D" class="usa-link" aria-describedby="id9"><span class="name western">Estibaliz Gonzalez-Fernandez</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">Estibaliz Gonzalez-Fernandez</span></h3>
<div class="p">
<sup>4</sup>Analytical Research and Development, Merck &amp; Co., Inc, West Point, PA USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gonzalez-Fernandez%20E%22%5BAuthor%5D" class="usa-link"><span class="name western">Estibaliz Gonzalez-Fernandez</span></a>
</div>
</div>
<sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22DeWitt%20LA%22%5BAuthor%5D" class="usa-link" aria-describedby="id10"><span class="name western">Lizzy Aurora DeWitt</span></a><div hidden="hidden" id="id10">
<h3><span class="name western">Lizzy Aurora DeWitt</span></h3>
<div class="p">
<sup>1</sup>Infectious Diseases and Vaccines Discovery, Merck &amp; Co., Inc., West Point, PA USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22DeWitt%20LA%22%5BAuthor%5D" class="usa-link"><span class="name western">Lizzy Aurora DeWitt</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Jeong%20U%22%5BAuthor%5D" class="usa-link" aria-describedby="id11"><span class="name western">Uijin Jeong</span></a><div hidden="hidden" id="id11">
<h3><span class="name western">Uijin Jeong</span></h3>
<div class="p">
<sup>2</sup>Quantitative Biosciences, Merck &amp; Co., Inc., West Point, PA USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Jeong%20U%22%5BAuthor%5D" class="usa-link"><span class="name western">Uijin Jeong</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Pan%20OCC%22%5BAuthor%5D" class="usa-link" aria-describedby="id12"><span class="name western">Oscar Chi-Chien Pan</span></a><div hidden="hidden" id="id12">
<h3><span class="name western">Oscar Chi-Chien Pan</span></h3>
<div class="p">
<sup>1</sup>Infectious Diseases and Vaccines Discovery, Merck &amp; Co., Inc., West Point, PA USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Pan%20OCC%22%5BAuthor%5D" class="usa-link"><span class="name western">Oscar Chi-Chien Pan</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Miller%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id13"><span class="name western">Sean Miller</span></a><div hidden="hidden" id="id13">
<h3><span class="name western">Sean Miller</span></h3>
<div class="p">
<sup>4</sup>Analytical Research and Development, Merck &amp; Co., Inc, West Point, PA USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Miller%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Sean Miller</span></a>
</div>
</div>
<sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Fridman%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id14"><span class="name western">Arthur Fridman</span></a><div hidden="hidden" id="id14">
<h3><span class="name western">Arthur Fridman</span></h3>
<div class="p">
<sup>5</sup>Data Science and Scientific Informatics, RaDS-IT, Merck &amp; Co., Inc., Rahway, NJ USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Fridman%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Arthur Fridman</span></a>
</div>
</div>
<sup>5</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22David%20C%22%5BAuthor%5D" class="usa-link" aria-describedby="id15"><span class="name western">Courtney David</span></a><div hidden="hidden" id="id15">
<h3><span class="name western">Courtney David</span></h3>
<div class="p">
<sup>6</sup>Discovery Pharmaceutical Sciences, Merck &amp; Co., Inc., West Point, PA USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22David%20C%22%5BAuthor%5D" class="usa-link"><span class="name western">Courtney David</span></a>
</div>
</div>
<sup>6</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chen%20Z%22%5BAuthor%5D" class="usa-link" aria-describedby="id16"><span class="name western">Zhifeng Chen</span></a><div hidden="hidden" id="id16">
<h3><span class="name western">Zhifeng Chen</span></h3>
<div class="p">
<sup>1</sup>Infectious Diseases and Vaccines Discovery, Merck &amp; Co., Inc., West Point, PA USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chen%20Z%22%5BAuthor%5D" class="usa-link"><span class="name western">Zhifeng Chen</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wei%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id17"><span class="name western">Jiajie Wei</span></a><div hidden="hidden" id="id17">
<h3><span class="name western">Jiajie Wei</span></h3>
<div class="p">
<sup>1</sup>Infectious Diseases and Vaccines Discovery, Merck &amp; Co., Inc., West Point, PA USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wei%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Jiajie Wei</span></a>
</div>
</div>
<sup>1,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>Infectious Diseases and Vaccines Discovery, Merck &amp; Co., Inc., West Point, PA USA </div>
<div id="Aff2">
<sup>2</sup>Quantitative Biosciences, Merck &amp; Co., Inc., West Point, PA USA </div>
<div id="Aff3">
<sup>3</sup>Process Research &amp; Development, Merck &amp; Co., Inc., West Point, PA USA </div>
<div id="Aff4">
<sup>4</sup>Analytical Research and Development, Merck &amp; Co., Inc, West Point, PA USA </div>
<div id="Aff5">
<sup>5</sup>Data Science and Scientific Informatics, RaDS-IT, Merck &amp; Co., Inc., Rahway, NJ USA </div>
<div id="Aff6">
<sup>6</sup>Discovery Pharmaceutical Sciences, Merck &amp; Co., Inc., West Point, PA USA </div>
<div class="author-notes p">
<div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div>
<div class="fn" id="_eqcntrb93pmc__">
<sup>#</sup><p class="display-inline">Contributed equally.</p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 Apr 15; Accepted 2025 Aug 5; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12365219  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40830611/" class="usa-link">40830611</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<p id="Par1">Noroviruses are non-enveloped, single-stranded positive-sense RNA viruses and the leading cause of gastroenteritis worldwide. The major capsid protein, VP1, can self-assemble into non-infectious virus-like particles (VLPs), representing an attractive vaccine platform. It was demonstrated that engineered disulfide bonds within VP1 could significantly stabilize VLPs of the archetypal GI.1 strain. Here, we apply a similar strategy to VLPs of multiple circulating GII genotypes. We find that engineered disulfide mutations can significantly stabilize VLPs of the GII.3 strain, but not the closely related GII.6 strain. Disulfide-stabilized GII.3 VLPs (GII.3-DS1) exhibit increased yields, greater homogeneity, and higher thermal stability compared to wild-type GII.3 VLPs. GII.3-DS1 VLPs are a superior reagent in immunological assays compared to the wild-type counterpart. Importantly, mRNA encoding GII.3-DS1 elicits superior humoral immune responses compared to wild-type GII.3 mRNA in mice. These results demonstrate the utility of rational VLP stabilization for advancing vaccine development efforts.</p>
<section id="kwd-group1" class="kwd-group"><p><strong>Subject terms:</strong> Infectious diseases, Vaccines</p></section></section><section id="Sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p id="Par2">Noroviruses are a leading cause of acute gastroenteritis (AGE) across all age groups and are responsible for 18% of diarrheal disease globally<sup><a href="#CR1" class="usa-link" aria-describedby="CR1">1</a></sup>. Norovirus is highly contagious and transmitted by the fecal-oral route<sup><a href="#CR2" class="usa-link" aria-describedby="CR2">2</a>–<a href="#CR4" class="usa-link" aria-describedby="CR4">4</a></sup>. Infections have a broad clinical spectrum that ranges from asymptomatic or self-limiting illness to severe or fatal disease. The public health impact generated by noroviruses is significant, with more than 20 million cases in the United States, 680 million cases worldwide and 219 thousand deaths annually. The disease burden is especially significant among children, with more than 200 million infections reported in children under 5 years of age and 50,000 cases resulting in death every year<sup><a href="#CR5" class="usa-link" aria-describedby="CR5">5</a>–<a href="#CR7" class="usa-link" aria-describedby="CR7">7</a></sup>. The total economic burden of noroviruses is estimated to be $64.5 billion annually<sup><a href="#CR5" class="usa-link" aria-describedby="CR5">5</a></sup>. Despite the prevalence of norovirus infection and associated disease burden, there are no approved prophylactics or targeted therapies. Treatments remain only palliative, resulting in a significant unmet medical need.</p>
<p id="Par3">Noroviruses are non-enveloped, single-stranded RNA viruses belonging to the <em>Caliciviridae</em> family. Noroviruses are classified into 10 genogroups (GI-GX) and 48 genotypes<sup><a href="#CR8" class="usa-link" aria-describedby="CR8">8</a></sup>. The GII genogroup is responsible for 90% of global outbreaks, with GII.4 at greater than 50% in all age groups since the mid-1990s<sup><a href="#CR9" class="usa-link" aria-describedby="CR9">9</a></sup>. While GII.4 is most common among all age groups, GII.3 and GII.6 are one of the most common genotypes infecting children and infants<sup><a href="#CR10" class="usa-link" aria-describedby="CR10">10</a>–<a href="#CR13" class="usa-link" aria-describedby="CR13">13</a></sup>. Based on data from NoroSurv, a global network for norovirus strain surveillance among children, from January 2015 to March 2025, 15% and 6% of submitted sequences were typed as GII.3 and GII.6, respectively<sup><a href="#CR13" class="usa-link" aria-describedby="CR13">13</a>,<a href="#CR14" class="usa-link" aria-describedby="CR14">14</a></sup>.</p>
<p id="Par4">Human noroviruses contain three overlapping open reading frames (ORF) encoding 6 non-structural proteins and 2 structural proteins, including major capsid protein VP1 and minor capsid protein VP2. VP1 is a 60 kDa protein that can assemble into virus-like particles (VLPs), which morphologically and antigenically mimic native virions<sup><a href="#CR15" class="usa-link" aria-describedby="CR15">15</a>–<a href="#CR17" class="usa-link" aria-describedby="CR17">17</a></sup>. VP2 is not required for virion or VLP formation but is thought to play a role in viral genome encapsidation, homogeneity and infectivity<sup><a href="#CR18" class="usa-link" aria-describedby="CR18">18</a>–<a href="#CR20" class="usa-link" aria-describedby="CR20">20</a></sup>. The norovirus capsid is primarily a <em>T</em> = 3 icosahedral shell approximately 35 to 50 nm in diameter and composed of 180 copies of VP1, organized into 90 dimers<sup><a href="#CR21" class="usa-link" aria-describedby="CR21">21</a></sup>. Structural studies have defined two major domains in VP1, the shell (S) and the protruding (P) domains, linked by a flexible hinge region. The P-domain can be further divided into P1 and P2 subdomains, the latter being critical for norovirus infection by interacting with carbohydrates present on various histoblood group antigens (HBGAs) in the host<sup><a href="#CR15" class="usa-link" aria-describedby="CR15">15</a>,<a href="#CR22" class="usa-link" aria-describedby="CR22">22</a></sup>.</p>
<p id="Par5">Norovirus vaccine development has been challenging, partially due to the lack of an in vitro culture system for human norovirus. VLPs formed by VP1 mimic native virions morphologically and immunologically, representing a potential vaccine platform. VP1-based vaccine candidates, including VLPs, P particles, and recombinant adenoviruses, elicit HBGA blocking antibodies, a potential correlate of protection for norovirus infection and disease identified by human challenge studies<sup><a href="#CR23" class="usa-link" aria-describedby="CR23">23</a>–<a href="#CR25" class="usa-link" aria-describedby="CR25">25</a></sup>. In addition, clinical trial data demonstrate that VP1-based vaccines can stimulate other immunologic markers potentially associated with the protection of norovirus disease and infection, such as serum immunoglobulin G (IgG), serum immunoglobulin A (IgA), fecal IgA, salivary IgA, and norovirus-specific memory IgG B-cells<sup><a href="#CR23" class="usa-link" aria-describedby="CR23">23</a>,<a href="#CR25" class="usa-link" aria-describedby="CR25">25</a>,<a href="#CR26" class="usa-link" aria-describedby="CR26">26</a></sup>. Regardless of the vaccine modality, VLPs are a required reagent to support vaccine characterization and immunological assays.</p>
<p id="Par6">Structure-guided protein design has been particularly impactful in the field of vaccines, as many viral antigens are unstable, metastable, or sample multiple distinct conformations requiring rational mutagenesis for suitable vaccine development. Structure-guided stabilization approaches have been successfully applied to multiple clinically approved vaccines, including for SARS-CoV2 and RSV. The SARS-CoV2 Spike (S) protein was stabilized in the prefusion conformation by introducing a double proline (2 P) mutation within the S2 domain<sup><a href="#CR27" class="usa-link" aria-describedby="CR27">27</a></sup>. These mutations were based on extensive research on the structure of the S protein of MERS<sup><a href="#CR28" class="usa-link" aria-describedby="CR28">28</a></sup>, allowing for the rapid development of vaccines against SARS-CoV2. Second-generation constructs containing additional proline mutations and disulfide bonds have further increased stability<sup><a href="#CR29" class="usa-link" aria-describedby="CR29">29</a>,<a href="#CR30" class="usa-link" aria-describedby="CR30">30</a></sup>. In the case of RSV, structure-guided stabilization of the prefusion conformation of the fusion (F) glycoprotein has paved the way for the recent development of multiple clinically approved RSV vaccines<sup><a href="#CR31" class="usa-link" aria-describedby="CR31">31</a></sup>. The basis of these RSV vaccines is the DS-Cav1 construct, which stabilized the prefusion conformation by a combination of disulfide and cavity filling mutations<sup><a href="#CR32" class="usa-link" aria-describedby="CR32">32</a></sup>. These foundational examples represent a new paradigm of rational stabilization of viral antigens for successful vaccine development.</p>
<p id="Par7">Recently, it was demonstrated that a single engineered disulfide bond increased the stability of norovirus VLPs of the archetypical GI.1 strain<sup><a href="#CR33" class="usa-link" aria-describedby="CR33">33</a></sup>. Here, we apply a similar protein engineering strategy to VLPs of multiple currently circulating GII genotypes. We show that these design principles are not generalizable across multiple norovirus strains. Engineered disulfide mutations can significantly stabilize VLPs of the GII.3 strain, but not the closely related GII.6 strain. Disulfide stabilization of GII.3 (GII.3-DS1) results in increased VLP yield from mammalian cell expression, along with more compact and less polydisperse VLPs with significantly higher thermal stability compared to the wild-type counterpart. Negative-stain electron microscopy (nsEM) further confirms that engineered disulfides increase GII.3 VLP stability, resulting in more homogenous particles. We show that stabilized GII.3-DS1 VLPs can be recognized by human sera in ELISA assays and can be used in Histo-Blood Group Antigen (HBGA) assays to test functional antibodies against VP1. Finally, mice vaccinated with GII.3-DS1 mRNA show superior ELISA binding and HBGA blocking titers compared to those vaccinated with wild-type GII.3 mRNA. These results demonstrate the utility of rational VLP stabilization and pave the way for vaccine development efforts.</p></section><section id="Sec2"><h2 class="pmc_sec_title">Results</h2>
<section id="Sec3"><h3 class="pmc_sec_title">Design and expression of disulfide-stabilized GII VLPs</h3>
<p id="Par8">We sought to engineer a disulfide bond that could stabilize the predominant structure, <em>T</em> = 3 icosahedron, of VLPs from GII genotypes. Previous structural studies indicate that VLPs from various GII genotypes can assemble into <em>T</em> = 1, <em>T</em> = 3, and <em>T</em> = 4 icosahedrons<sup><a href="#CR21" class="usa-link" aria-describedby="CR21">21</a></sup>. An earlier study showed that engineering a single disulfide bond between residues 116 and 193 at the 3-fold and 5-fold symmetry axes could significantly stabilize <em>T</em> = 3 VLPs derived from the GI.1 genotype (Fig. <a href="#Fig1" class="usa-link">1B</a>)<sup><a href="#CR33" class="usa-link" aria-describedby="CR33">33</a></sup>. We sought to identify analogous positions in circulating norovirus strains from the GII genotypes. Despite significant sequence variability in these regions, we identified residues 112 and 189 in genotypes GII.3, GII.4 and GII.6 as candidate sites for analogous stabilizing mutations (Fig. <a href="#Fig1" class="usa-link">1A, C, D</a>). We next modeled the structures of the <em>T</em> = 3 shells from these genotypes using a combination of AlphaFold2 and PyRosetta, and subsequently measured intermolecular distances between 2 loops containing residues 105–120 and 181–196 (Fig. <a href="#Fig1" class="usa-link">1E</a>)<sup><a href="#CR34" class="usa-link" aria-describedby="CR34">34</a>,<a href="#CR35" class="usa-link" aria-describedby="CR35">35</a></sup>. This structural modeling indicated that residues 112 and 189 are likely in close proximity at the 3-fold symmetry axis of these genotypes in the modeled <em>T</em> = 3 arrangement. The models also predict that residues 112 and 189 may be closer together in GII.3, compared to GII.4 and GII.6. Based on both the sequence alignment and this structural modeling, we attempted to stabilize these GII genotypes by introducing N112C/N189C mutations, hereafter referred to as DS1.</p>
<figure class="fig xbox font-sm" id="Fig1"><h4 class="obj_head">Fig. 1. Structure-guided stabilization of norovirus VLPs.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365219_41541_2025_1254_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/b09c/12365219/d40f6dbb3f4e/41541_2025_1254_Fig1_HTML.jpg" loading="lazy" id="d33e509" height="309" width="691" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>A</strong> Domain map of the norovirus VP1 protein showing the shell (S), hinge, and protruding (P1/P2) domains. Protease cleavage site present in GII.3 and GII.6 shown at residue 302 (lightning bolt). Engineered disulfide bond (DS1) between interprotomer residues N112 and N189 shown in purple. <strong>B</strong> Structure of the disulfide-stabilized GI.1 shell (PDB 7KJP) colored by chain. Left shows the intact <em>T</em> = 3 VLP. Asymmetric unit shown on the right with the interprotomer disulfide highlighted. <strong>C</strong> Sequence identity matrix of full-length VP1 based on the alignment of 4 norovirus genotypes. <strong>D</strong> Sequence alignment of VP1 from 4 norovirus genotypes in the regions of the engineered disulfide bond. Cysteine mutation positions are in bold. <strong>E</strong> Alphafold models of the shell domains of three GII genotypes colored by pLDDT. Monomeric models were aligned to the GI.1 <em>T</em> = 3 asymmetric unit (PDB 7KJP) and their energies were minimized. Interprotomer Ca-Ca distance matrices were calculated between loops containing residues 105–120 (<em>y</em>-axis) and 181–196 (<em>x</em>-axis) to predict proximity of residue pairs within the <em>T</em> = 3 VLP structure.</p></figcaption></figure><p id="Par9">To determine VP1 protein expression, plasmids encoding VP1 protein from either GII.3, GII.4, GII.6 or their DS1 variants were transfected transiently into mammalian cells. In the supernatant, VP1 protein for GII.4 and GII.4-DS1 was readily visible by Coomassie-stained SDS-PAGE gel and anti-VP1 western blot whereas VP1 protein for GII.3 and GII.6 and their DS1 variants was primarily visible by anti-VP1 western blot only (Fig. <a href="#Fig2" class="usa-link">2A</a>). In the cell lysate, VP1 protein is visible by both Coomassie-stained SDS-PAGE gel and anti-VP1 western blot for all constructs tested (Fig. <a href="#Fig2" class="usa-link">2B</a>). Protease cleavage fragments are present for wildtype and DS1 variants for GII.3 and GII.6 in both the supernatant and cell lysate fractions (Fig. <a href="#Fig2" class="usa-link">2A, B</a>), confirming earlier publications<sup><a href="#CR36" class="usa-link" aria-describedby="CR36">36</a>–<a href="#CR40" class="usa-link" aria-describedby="CR40">40</a></sup>. Purified GII.4 yielded an intact VLP of the expected size as determined by sucrose gradient, DLS and nsEM (Supplementary Fig. <a href="#MOESM1" class="usa-link">1A, C, D</a>). Additionally, purified GII.4 VLPs were thermostable as shown by nanoDSF (Supplementary Fig. <a href="#MOESM1" class="usa-link">1B</a>). Owing to the superior quality of GII.4 wildtype VLPs, we did not further evaluate GII.4-DS1 and focused our efforts on the DS1 variants of GII.3 and GII.6 VLPs.</p>
<figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2. VP1 protein is expressed successfully from transiently transfected Expi293 cells.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365219_41541_2025_1254_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/b09c/12365219/6f77d25bc8fb/41541_2025_1254_Fig2_HTML.jpg" loading="lazy" id="d33e549" height="443" width="700" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Cells were transfected with expression plasmids encoding various strains of norovirus VP1. Clarified supernatant (2.6 mg/lane, <strong>A</strong>) and lysate (15 mg/lane, <strong>B</strong>) fractions were separated via SDS-PAGE and either stained with SimplyBlue Safestain (upper panels) or blotted onto membranes and probed with an anti-VP1 antibody (lower panels). The position of full-length VP1 is denoted by black arrows. The positions of cleaved fragments of strains GII.3 and GII.6 are denoted by gray arrows. Marker proteins in kilodaltons (kDa) are indicated.</p></figcaption></figure></section><section id="Sec4"><h3 class="pmc_sec_title">Engineered GII.3-DS1 and GII.6-DS1 assemble into VLPs and can form interprotomer disulfide bonds</h3>
<p id="Par10">Bulk harvested and clarified Expi293 cell lysates were analyzed via sucrose gradient ultracentrifugation to determine if the expressed GII.3 and GII.6 VP1 proteins and their DS1 counterparts form VLPs. Norovirus VLPs typically elute near the center of a 15–36% gradient (~25% sucrose), and we observed VLPs for all four samples (Fig. <a href="#Fig3" class="usa-link">3A</a>). Lower molecular weight products of the protease-cleaved VP1 co-eluted with full-length VP1, indicating that these cleaved VP1 species are incorporated into VLPs. To determine if VP1 cleavage is required for VLP assembly and function, we also identified a conserved caspase cleavage site within the P-domains of GII.3 and GII.6 (Supplementary Fig. <a href="#MOESM1" class="usa-link">2A–C</a>). We show that mutations at this site in GII.3 (D302A/S) result in VLPs containing only full-length VP1 (Supplementary Fig. <a href="#MOESM1" class="usa-link">2D</a>). These VLPs have higher polydispersity and decreased saliva binding compared to wild-type GII.3 and GII.3-DS1 VLPs, consistent with previous studies (Supplementary Fig. <a href="#MOESM1" class="usa-link">2F, G</a>)<sup><a href="#CR40" class="usa-link" aria-describedby="CR40">40</a></sup>. Notably, compared to the GII.3 wild-type sequence, GII.3-DS1 showed much stronger bands in VLP fractions (Fig. <a href="#Fig3" class="usa-link">3A</a>), indicating better expression and/or more efficient VLP formation. VLP fractions were pooled, dialyzed to remove sucrose, and concentrated to 0.5 mg/mL. Comparison of the relative post-sucrose gradient purification yields of GII.3 and GII.6 to their DS1 counterparts showed that indeed GII.3-DS1 generated approximately twice as much VLP as GII.3 while GII.6-DS1 and GII.6 yields were comparable (Fig. <a href="#Fig3" class="usa-link">3F</a>).</p>
<figure class="fig xbox font-sm" id="Fig3"><h4 class="obj_head">Fig. 3. DS1 mutation stabilizes GII.3 VLPs.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365219_41541_2025_1254_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/b09c/12365219/fe560eaaf4c2/41541_2025_1254_Fig3_HTML.jpg" loading="lazy" id="d33e613" height="319" width="691" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>A</strong> Sucrose gradient purification of GII.3 and GII.6 VLPs with and without DS1 mutation. Fractions 5–7 (red line) were pooled for downstream analyses. Gradient input denoted input. <strong>B</strong> Dynamic light scattering (DLS) of GII.3 and GII.6 VLPs. Z-average (z-avg) and polydispersity index (PdI) values are shown in the insert. <strong>C</strong> SDS-PAGE analysis of GII.3 and GII.6 VLPs in either the untreated condition (top) or after treatment with 20 mM BME (middle) or 20 mM diamide (bottom) for 1 h at room temperature. <strong>D</strong> nanoDSF measurements of GII.3 and GII.6 VLPs. The line represents the average of 3 technical replicates. <strong>E</strong> Quantification of melting temperature (<em>T</em><sub>m</sub>) from nanoDSF experiments, showing a significant increase in thermal stability of GII.3 VLPs upon introduction of the DS1 mutation. Significant values determined by unpaired <em>t</em>-test (<em>p</em> &lt; 0.0001). <strong>F</strong> Relative yields of DS1 mutant compared to wild-type after sucrose gradient purification of VLPs.</p></figcaption></figure><p id="Par11">To determine if disulfide bonds are formed in the DS1 mutants, VLPs were run on SDS-PAGE under 3 conditions: (1) without additional treatment, (2) after treatment with 20 mM beta-mercaptoethanol (reducing condition) for 1 h, or (3) after treatment with 20 mM diamide (oxidizing condition) for 1 h. We observed a distinct shift in the VP1 bands from GII.3-DS1 and GII.6-DS1 into the wells under oxidizing conditions (Fig. <a href="#Fig3" class="usa-link">3C</a>). This shift is indicative of engineered disulfide bond formation. Disulfide bond formation was only observed on SDS-PAGE after treatment with a strong oxidizing agent; however, there may be a minor amount of VP1 material in the well in the untreated condition. This observation is consistent with previously published data on the GI.1-DS1 VLP<sup><a href="#CR33" class="usa-link" aria-describedby="CR33">33</a></sup>. All subsequent experiments were performed without diamide pretreatment.</p></section><section id="Sec5"><h3 class="pmc_sec_title">Disulfide stabilizing mutation enhances VLP formation, yield, and stability of strain GII.3 but not the closely related strain GII.6</h3>
<p id="Par12">To measure the effects of the DS1 mutation on VLP stability, we performed dynamic light scattering (DLS) and differential scanning fluorimetry (DSF) experiments. Compared to wild-type GII.3, GII.3-DS1 showed reductions in both the hydrodynamic diameter (58.34 to 50.45 nm) and polydispersity index (0.180 to 0.102), indicating GII.3-DS1 VLPs are more compact and more homogenous compared to wild-type GII.3 VLPs (Fig. <a href="#Fig3" class="usa-link">3B</a>). The DS1 mutation did not appear to have a significant effect in the GII.6 background as measured by DLS. Compared to wild-type GII.6, GII.6-DS1 led to a slight reduction in the hydrodynamic diameter (53.04 to 50.94 nm) and a marginal increase in the polydispersity index (0.141 to 0.155) (Fig. <a href="#Fig3" class="usa-link">3B</a>). We further conducted nanoDSF experiments to measure the thermal stability of VLPs. Wild-type GII.3 and GII.6 VLPs both showed monophasic transitions with melting temperatures(<em>T</em><sub>m</sub>) of 64.0 and 66.8 °C, respectively (Fig. <a href="#Fig3" class="usa-link">3D, E</a>). GII.3-DS1 showed a significant increase in <em>T</em><sub>m</sub> of 4.2 °C. In contrast, GII.6-DS1 marginally increased the <em>T</em><sub>m</sub> by 1.0 °C (Fig. <a href="#Fig3" class="usa-link">3D, E</a>). Taken together, these data indicate that the DS1 mutation stabilizes GII.3 VLPs, whereas disulfide-induced stabilization was not observed for GII.6 VLPs.</p>
<p id="Par13">To further assess VLP formation and stability, we imaged purified VLPs via negative-stain electron microscopy (nsEM). We directly compared sucrose gradient purified VLPs from genotypes GII.3, GII.3-DS1, GII.6 and GII.6-DS1 after a single freeze-thaw (Fig. <a href="#Fig4" class="usa-link">4</a>) or without freezing (Supplementary Fig. <a href="#MOESM1" class="usa-link">3</a>). Intact VLPs were observed in all samples having diameters consistent with either <em>T</em> = 3 or <em>T</em> = 4 icosahedral symmetry, though we are unable to differentiate these two assembly types using nsEM. Micrographs from the 1x freeze thawed samples indicated a higher proportion of intact VLPs for GII.3-DS1 (Fig. <a href="#Fig4" class="usa-link">4B</a>) compared to GII.3 (Fig. <a href="#Fig4" class="usa-link">4A</a>), which showed a substantial amount of breakdown products in the micrographs. 2D classes generated from manually selected VLPs also confirmed more homogenous VLPs in the GII.3-DS1 sample compared to GII.3. This stabilizing effect was less pronounced when comparing wild-type GII.6 (Fig. <a href="#Fig4" class="usa-link">4C</a>) and GII.6-DS1 (Fig. <a href="#Fig4" class="usa-link">4D</a>) samples. The stabilizing effect was also less pronounced when comparing the VLPs without freeze-thaw (Supplementary Fig. <a href="#MOESM1" class="usa-link">3</a>), which showed fewer breakdown products in the wild-type GII.3 sample. Together, these biophysical and structural data indicate that the DS1 mutation stabilizes VLPs from the GII.3 genotype but has little effect on the closely related GII.6 genotype.</p>
<figure class="fig xbox font-sm" id="Fig4"><h4 class="obj_head">Fig. 4. DS1 mutation leads to more uniform VLPs.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365219_41541_2025_1254_Fig4_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/b09c/12365219/5e591f63f8a8/41541_2025_1254_Fig4_HTML.jpg" loading="lazy" id="d33e715" height="347" width="691" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Representative negative-stain electron micrographs (nsEM) and 2D classes of GII.3 (<strong>A</strong>), GII.3-DS1 (<strong>B</strong>), GII.6 (<strong>C</strong>), and GII.6-DS1 (<strong>D</strong>) VLPs. 2D class averages of intact VLPs are in the panel set on the right. nsEM magnification is 58,000X. VLPs were stored frozen at -80⁰C until processed for imaging. <strong>A</strong> GII.3 2D class averages derived from 139 particles picked from 27 micrographs. <strong>B</strong> GII.3-DS1 2D class averages derived from 97 particles picked from 26 micrographs. <strong>C</strong> GII.6 2D class averages derived from 205 particles picked from 27 micrographs. <strong>D</strong> GII.6-DS1 2D class averages derived from 314 particles picked from 24 micrographs.</p></figcaption></figure></section><section id="Sec6"><h3 class="pmc_sec_title">GII.3 and GII.3-DS1 VLPs bind HBGA glycans and strain-specific monoclonal antibodies similarly</h3>
<p id="Par14">To assess the functionality of the GII.3 and GII.3-DS1 VLPs, we evaluated their ability to bind to histoblood group glycans present in human saliva. VLPs were used to bind a panel of human saliva samples on ELISA plates (<em>n</em> = 51). Results indicate that GII.3 and GII.3-DS1 VLPs bind to human saliva samples similarly (<em>R</em><sup>2</sup> = 0.9208) (Fig. <a href="#Fig5" class="usa-link">5A</a>). Of note, consistent with previous observations, neither GII.3 nor GII.3-DS1 binds strongly to most salivary samples; however, we identified a few samples that showed high binding signals (Supplementary Fig. <a href="#MOESM1" class="usa-link">4</a>)<sup><a href="#CR41" class="usa-link" aria-describedby="CR41">41</a>,<a href="#CR42" class="usa-link" aria-describedby="CR42">42</a></sup>. The saliva that bound strongly to both GII.3 and GII.3-DS1 VLPs was used for all ELISA and HBGA assays hereafter. Furthermore, we screened a panel of GII.3-specific monoclonal antibodies (mAbs, <em>n</em> = 19) for their ability to bind GII.3 and GII.3-DS1 VLPs. We observed highly similar interpolated endpoint titers for both VLPs (<em>R</em><sup>2</sup> = 0.8031) (Fig. <a href="#Fig5" class="usa-link">5B</a>). These similarities in the binding profiles indicate that GII.3-DS1 VLPs functionally mimic their wild-type counterpart.</p>
<figure class="fig xbox font-sm" id="Fig5"><h4 class="obj_head">Fig. 5. GII.3 and GII.3-DS1 VLPs bind to human saliva samples and monoclonal antibodies.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365219_41541_2025_1254_Fig5_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/b09c/12365219/b79edbbe11cf/41541_2025_1254_Fig5_HTML.jpg" loading="lazy" id="d33e776" height="242" width="700" alt="Fig. 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>A</strong> ELISA plates were coated with 51 human saliva samples and incubated with GII.3 or GII.3-DS1 VLPs at 20 µg/mL. Bound VLPs were detected using polyclonal sera against GII.3. Relative luminescent units (RLU) of GII.3 and GII.3-DS1 are reported on the <em>x-</em> and <em>y</em>-axes, respectively. Data are fitted into a simple linear regression model using GraphPad Prism software. <strong>B</strong> Plates were coated with 50 ng of either GII.3 or GII.3-DS1 VLP and incubated with 19 unique human monoclonal antibodies starting at 4 µg/mL and diluted in a four-fold, ten-point dilution series. Bound antibodies were detected using goat anti-human Ig-Fc fragment conjugated to horseradish peroxidase (HRP). Interpolated endpoint titer for each antibody against either GII.3 or GII.3-DS1 is reported on the <em>x</em>- and <em>y</em>-axes, respectively. Data are fitted into a simple linear regression model using GraphPad Prism software.</p></figcaption></figure></section><section id="Sec7"><h3 class="pmc_sec_title">GII.3-DS1 mRNA elicited higher humoral responses in mice</h3>
<p id="Par15">We demonstrated that the DS1 mutation increases VLP quantities and confers increased stability to the GII.3 VLP without affecting glycan binding. To investigate whether the advantages conferred to the VLP by the DS1 mutation could improve the vaccine antigen, we vaccinated mice with mRNAs encoding GII.3 or GII.3-DS1 VP1 encapsulated in LNPs<sup><a href="#CR43" class="usa-link" aria-describedby="CR43">43</a></sup>. Mice were immunized twice, at four-week intervals, with three dose groups of each mRNA. Serum was collected at 2 weeks post doses 1 and 2 and evaluated for antibody binding and HBGA blockage titers (Fig. <a href="#Fig6" class="usa-link">6A</a>). At both weeks 2 and 6, mRNA encoding GII.3-DS1 at low (0.1 µg) and mid (0.5 µg) doses elicited statistically significantly higher VLP binding ELISA titers than mRNAs encoding GII.3, although this difference was not observed at the high (2 µg) dose (Fig. <a href="#Fig6" class="usa-link">6B</a> and Supplementary Table <a href="#MOESM1" class="usa-link">1</a>). While ELISA titers were detected in both low dose groups at week 2, neither GII.3 nor GII.3-DS1 mRNAs elicited measurable HBGA blocking titers at week 2 (Fig. <a href="#Fig6" class="usa-link">6C</a>). However, at week 2, both the mid and high dose groups showed statistically significant differences, with GII.3-DS1 mRNA inducing higher HBGA blockage titers than GII.3 (Fig. <a href="#Fig6" class="usa-link">6C</a> and Supplementary Table <a href="#MOESM1" class="usa-link">1</a>). By week 6, the low dose of GII.3-DS1 mRNAs elicited statistically significant HBGA blocking titers, while the mid and high doses of both GII.3-DS1 and GII.3 mRNAs performed similarly (Fig. <a href="#Fig6" class="usa-link">6C</a>). Week 6 data remained consistent whether the assay regent used was GII.3 or GII-DS1 VLPs (Supplementary Fig. <a href="#MOESM1" class="usa-link">5</a>).</p>
<figure class="fig xbox font-sm" id="Fig6"><h4 class="obj_head">Fig. 6. Mouse immunogenicity study of mRNAs encoding either GII.3 or GII.3-DS1, using assay reagent GII.3-DS1 VLP.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365219_41541_2025_1254_Fig6_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/b09c/12365219/5e2e572b1f70/41541_2025_1254_Fig6_HTML.jpg" loading="lazy" id="d33e854" height="315" width="691" alt="Fig. 6"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>A</strong> In vivo immunogenicity study design. BALB/c mice (<em>N</em> = 8 per group) were immunized with 0.1, 0.5, or 2 µg of mRNA at weeks 0 and 4 (black arrows). An empty lipid nanoparticle (LNP) was immunized as a negative control. Blood was drawn at weeks 2 and 6 (red arrows). Serum antibody ELISA titers (<strong>B</strong>) and HBGA blockade antibody titers (<strong>C</strong>) against GII.3-DS1 VLP. Serum of mice immunized with mRNAs encoding either GII.3 (red dots) or GII.3-DS1 (blue dots) was assayed. <strong>B</strong>, <strong>C</strong> Limits of detection are indicated with horizontal dotted lines (lower limit titer = 50 in (<strong>B</strong>) and 20 in (<strong>C</strong>), upper limit titer = 51,200 in (<strong>C</strong>). Geometric mean titers with geometric standard deviation are shown in a scatter dot plot. P values were determined using GraphPad Prism software (<em>p</em> &lt; 0.05*, <em>p</em> &lt; 0.01**, <em>p</em> &lt; 0.001***).</p></figcaption></figure><p id="Par16">We next performed a series of mouse immunizations using purified VLPs to determine if the enhanced immunogenicity of GII.3-DS1 mRNA is due to increased VLP stability or if this effect can be more simply explained by higher expression of GII.3-DS1. Mice were immunized twice, at four-week intervals, with purified GII.3 or GII.3-DS1 VLPs at two doses: 0.2 and 2 µg. Immunizations were performed both with and without an aluminum salt (AAHS) adjuvant. In the high dose groups, we observed slight increases in both ELISA and HBGA blocking titers, both with and without adjuvant present (Supplementary Fig. <a href="#MOESM1" class="usa-link">6A</a>, C). However, in the low dose groups, we observed no statistically significant differences in the adjuvanted groups and even a modest decrease in immunogenicity of GII.3-DS1 VLPs in groups without AAHS adjuvant (Supplementary Fig. <a href="#MOESM1" class="usa-link">6B, D</a>). Despite these inconsistencies, the small magnitude of these differences from wild-type GII.3 VLPs strongly suggests that the robust increase in immunogenicity observed from the GII.3-DS1 mRNA can be largely attributed to increased expression of the antigen and not greater VLP stability in vivo. Taken together, these data suggest that, as an mRNA vaccine candidate, GII.3-DS1 is more immunogenic than the wild-type GII.3 at non-saturating dose levels in mice.</p></section></section><section id="Sec8"><h2 class="pmc_sec_title">Discussion</h2>
<p id="Par17">VLPs represent an attractive vaccine platform due to their structural mimicry of native virions and multivalent display of viral epitopes. Currently, recombinantly produced VLPs have been successfully implemented in two FDA-approved vaccines: Gardasil (HPV) and Engerix-B (HBV)<sup><a href="#CR44" class="usa-link" aria-describedby="CR44">44</a></sup>. Multiple enveloped and non-enveloped viruses may be amendable to future VLP-based vaccine development<sup><a href="#CR45" class="usa-link" aria-describedby="CR45">45</a>,<a href="#CR46" class="usa-link" aria-describedby="CR46">46</a></sup>. Independent of the vaccine modality, recombinant proteins are typically required for immunological assays to support vaccine development. In the case of norovirus, VLPs have been shown as promising vaccine candidates<sup><a href="#CR47" class="usa-link" aria-describedby="CR47">47</a></sup>. High-quality VLPs are required to support ELISA and HBGA functional assays, even if the vaccine platform is mRNA or adenovirus-based<sup><a href="#CR25" class="usa-link" aria-describedby="CR25">25</a>,<a href="#CR48" class="usa-link" aria-describedby="CR48">48</a>,<a href="#CR49" class="usa-link" aria-describedby="CR49">49</a></sup>. The manufacturing challenges posed by the inherent differences of VLPs across genotypes present an opportunity for the application of rational VLP engineering.</p>
<p id="Par18">Here we demonstrate that a rationally engineered disulfide mutation (DS1) can significantly stabilize VLPs of the GII.3 genotype, leading to increased VLP yield and a significantly more thermostable product compared to wild-type. nsEM analysis further demonstrates the increased stability of GII.3-DS1 VLPs through the stress of a single freeze-thaw cycle. Our results are consistent with an earlier publication of the DS1 mutation in the archetypal GI.1 genotype, which showed increased thermostability of VLPs. These VLPs, in the absence of an adjuvant, also yielded superior HBGA blocking titers in an animal study<sup><a href="#CR33" class="usa-link" aria-describedby="CR33">33</a></sup>. We note that these GI.1-DS1 VLPs were treated with a strong oxidizing agent (diamide) during their purification, and all subsequent biophysical, structural, and immunological experiments were completed using the diamide-treated VLPs. In contrast, the GII.3-DS1 VLPs generated here show significantly enhanced stability without diamide treatment (Figs.<a href="#Fig3" class="usa-link"> 3</a> and <a href="#Fig4" class="usa-link">4</a>). Consistent with this observation, we also demonstrated superior immune responses in a mouse immunogenicity study using mRNAs encoding GII.3-DS1 compared to wild-type GII.3 (Fig. <a href="#Fig6" class="usa-link">6</a>). Due to the increased biophysical stability of untreated GII.3-DS1 VLPs, it was unexpected that a strong disulfide-induced band shift in SDS-PAGE was not observed unless VLPs were treated with diamide (Fig. <a href="#Fig3" class="usa-link">3C</a>). We hypothesize that, in the untreated condition, GII.3-DS1 VLPs may transiently form disulfide bonds. In a <em>T</em> = 3 icosahedron with 180 copies of VP1, the DS1 mutation can form intermolecular disulfides at both the 5-fold and 3-fold symmetry axes, leading to a maximum of 180 possible intermolecular disulfide bonds formed per VLP. We theorize that even a small fraction of disulfides formed, while not detected by SDS-PAGE analysis, could increase overall VLP stability.</p>
<p id="Par19">The advantages provided by the DS1 mutation are genotype-specific. Unlike GII.3, we did not observe a significant stabilizing effect of the DS1 mutation when introduced into the closely related GII.6 genotype, although GII.6-DS1 also showed disulfide bond formation after treatment with diamide (Fig. <a href="#Fig3" class="usa-link">3C</a>). The reason for this difference is unclear; however, wild-type GII.6 VLPs were significantly more compact and less polydisperse than wild-type GII.3 by DLS (Fig. <a href="#Fig3" class="usa-link">3B</a>) and showed higher thermal stability compared to wild-type GII.3 by nanoDSF (Fig. <a href="#Fig3" class="usa-link">3D, E</a>). In addition, our structural modeling of the GII.6 shell in the <em>T</em> = 3 arrangement indicates that residues 112 and 189 may be less optimally positioned for disulfide bond engineering compared to GII.3 (Fig. <a href="#Fig1" class="usa-link">1E</a>). Recently published cryoEM structures of multiple norovirus VLPs show significant variations in both the conformations of the shell domain and copy number, with the GII.4 Minerva strain predominantly existing as a <em>T</em> = 4 icosahedron containing 240 copies of VP1<sup><a href="#CR21" class="usa-link" aria-describedby="CR21">21</a></sup>. While this form represents a 33% increase in the number of VP1 protomers per VLP (240 vs. 180), the <em>T</em> = 4 VLPs are only approximately 16% larger in diameter compared to <em>T</em> = 3 VLPs from other GII genotypes (490 Å compared to 420 Å, respectively). This modest increase in diameter may not be detected by the DLS or nsEM methods employed here. Additionally, the DS1 mutation positions, N112C and N189C, are less optimal for stabilizing a <em>T</em> = 4 VLP, based on the GII.4 Minerva structure, compared to a <em>T</em> = 3 VLP<sup><a href="#CR21" class="usa-link" aria-describedby="CR21">21</a></sup>. Finally, multiple studies have demonstrated the existence of <em>T</em> = 1 VLPs, containing only 60 copies of VP1<sup><a href="#CR21" class="usa-link" aria-describedby="CR21">21</a>,<a href="#CR50" class="usa-link" aria-describedby="CR50">50</a>,<a href="#CR51" class="usa-link" aria-describedby="CR51">51</a></sup>. Among these studies, the GII.4 S<sub>60</sub> nanoparticle was further stabilized by introducing a triple cysteine mutation within the shell domain (V57C/Q58C/S136C), which occurs at the two-fold symmetry axis and may be able to stabilize VLPs in multiple different arrangements. A similar mutation in GII.3 (Q58C/A136C) was screened in the current study but did not lead to significant increases in VP1 expression and therefore was not included in downstream analyses (data not shown). Taken together, we conclude that a universal approach to stabilize norovirus VLPs across genotypes is unlikely to be successful. Instead, precise modeling of local structure, VP1 dynamics and overall VLP arrangement would yield superior rationally designed stabilizing mutations for other genotypes.</p>
<p id="Par20">We established that the DS1 mutation can stabilize GII.3 VLPs used as assay reagents. We sought to determine whether this stabilizing effect could be applied to improve GII.3 vaccine antigen design. In mice, mRNA encoding for GII.3-DS1 elicited significantly higher serum antibody titers compared to wild-type GII.3 mRNA (Fig. <a href="#Fig6" class="usa-link">6</a>). We hypothesize that this result could be due to three possibilities: (1) increased VP1 protein production, (2) more efficient VLP formation, and/or (3) increased half-life of VLPs due to their enhanced stability. Immunizations with purified GII.3 and GII.3-DS1 VLPs show only very modest and inconsistent differences in ELISA and HBGA blocking titers among groups, suggesting that the increased immunogenicity of the GII.3-DS1 mRNA can be largely attributed to higher expression (Supplementary Fig. <a href="#MOESM1" class="usa-link">6</a>). Further supporting this hypothesis, we showed that GII.3-DS1 yielded greater amounts of VLP compared to GII.3 in transiently transfected Expi 293 cells (Fig. <a href="#Fig3" class="usa-link">3F</a>). While it is unknown whether VLPs form within a mouse after vaccination with an mRNA-encoded VP1 sequence, previous structural studies of GII.3 show that the HBGA binding cleft is formed at the interprotomer interface of two P2 domains of assembled VP1 protein dimers, similar to HBGA binding sites in other GII strains<sup><a href="#CR52" class="usa-link" aria-describedby="CR52">52</a>,<a href="#CR53" class="usa-link" aria-describedby="CR53">53</a></sup>. Therefore, the presence of strong HBGA blocking titers, in addition to high ELISA titers, after mRNA vaccination is consistent with supramolecular assembly of VP1 proteins after mRNA vaccination.</p>
<p id="Par21">Taken together, these data highlight the utility of structure-guided disulfide bond engineering for the stabilization of norovirus VLPs. These data also indicate that a universal approach to stabilize VLPs from different norovirus strains has a low probability of success due to intrinsic variations in VLP assembly and VP1 structure and dynamics. Finally, we demonstrate that the stabilizing mutations within GII.3-DS1 VLPs can be applied as an mRNA vaccine antigen, emphasizing the potential of rational vaccine design and development.</p></section><section id="Sec9"><h2 class="pmc_sec_title">Methods</h2>
<section id="Sec10"><h3 class="pmc_sec_title">Structural modeling of <em>T</em> = 3 GII.3, GII.4, and GII.6 VP1 shell domains</h3>
<p id="Par22">Monomeric models of the shell domains from VP1 proteins (residues 1–220) were generated with AlphaFold2 using sequences for GII.3 (Genbank Accession <a href="https://www.ncbi.nlm.nih.gov/nuccore/MT409884" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">MT409884</a>), GII.4 (Genbank Accession <a href="https://www.ncbi.nlm.nih.gov/nuccore/MK754446" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">MK754446</a>), and GII.6 (Genbank Accession <a href="https://www.ncbi.nlm.nih.gov/nuccore/MH260487" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">MH260487</a>)<sup><a href="#CR34" class="usa-link" aria-describedby="CR34">34</a></sup>. High-confidence predictions (pLDDT &gt;85) were generated for each shell sequence and three copies were subsequently aligned to the asymmetric unit of the GI.1-DS1 <em>T</em> = 3 shell structure (PDB 7kjp). These <em>T</em> = 3 models were subjected to energy minimization and FastRelax protocols in PyRosetta to remove any clashes that may have been generated during the structure alignment<sup><a href="#CR35" class="usa-link" aria-describedby="CR35">35</a></sup>. Intermolecular distances were measured between all Cα atoms from 2 loops containing residues 105–120 and 181–196 in the modeled <em>T</em> = 3 arrangements. 2D matrices corresponding to these intermolecular distances were plotted as a heatmap for visualization.</p></section><section id="Sec11"><h3 class="pmc_sec_title">Plasmid construction and VLP expression</h3>
<p id="Par23">Protein sequences of norovirus VP1 from genotypes GII.3 (Genbank Accession <a href="https://www.ncbi.nlm.nih.gov/nuccore/MT409884" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">MT409884</a>), GII.6 (Genbank Accession <a href="https://www.ncbi.nlm.nih.gov/nuccore/MH260487" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">MH260487</a>) and GII.4 (Genbank Accession <a href="https://www.ncbi.nlm.nih.gov/nuccore/MK754446" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">MK754446</a>) were mammalian codon-optimized by Genewiz (South Plainfield, NJ) and subcloned into a eukaryotic-expression vector under the control of the CMV promoter. DS1 variants were generated by introducing cysteine substitutions at amino acid positions N112 and N189 within the VP1 protein of each genotype. Caspase cleavage mutants (GII.3-D302A and GII.3-D302S) were generated by introducing alanine or serine substitutions at amino acid position D302 within VP1 of GII.3. Plasmids were transiently transfected into Expi 293 F cells (ThermoFisher, Waltham, MA) using Expifectamine (ThermoFisher, Waltham, MA) following the manufacturer’s recommended protocol. At 72 h post-transfection, a 1 mL sample of each culture was harvested into a microcentrifuge tube and centrifuged at 10,000 × <em>g</em>. Clarified supernatant was decanted into a fresh tube. The cell pellet was solubilized in 0.5 mL lysis buffer, centrifuged at 10,000 × <em>g</em>, and the clarified cell lysate was decanted into a fresh tube. Cell lysis buffer contained 0.5% Triton X-100, 40 mM Tris-HCl, pH 8.0, 120 mM NaCl, and 1X EDTA-free protease inhibitor (Roche Diagnostics, Indianapolis, IN). Clarified supernatant and cell lysate were used for SDS-PAGE and western blot analysis. Remaining bulk cell cultures were harvested 72 h post-transfection and frozen at −70 °C. To generate clarified bulk supernatant, the bulk harvests were thawed at room temperature (RT), mixed thoroughly by inversion, and centrifuged at 3500 × <em>g</em> for 10 min at RT. The resulting clarified bulk supernatant was transferred to a clean, sterile tube and stored at −70 °C.</p></section><section id="Sec12"><h3 class="pmc_sec_title">VLP purification</h3>
<p id="Par24">The frozen bulk cell culture was thawed and clarified through a Sartopure GF+ (Sartorius, Bohemia, NY) depth filter. The clarified VLPs were precipitated using 1.5 M ammonium sulfate (AS) and pelleted by centrifugation at 5000 × <em>g</em> for 10 min. The pellet was resuspended in an anion exchange chromatography (AEX) loading buffer and filtered through a 0.2 µm PVDF membrane (MilliporeSigma, Burlington, MA). The filtered AS precipitation product was purified by AEX using a Sartobind Q membrane (Sartorius, Bohemia, NY) and the VLPs were eluted in an elevated ionic strength buffer. The AEX eluates were further purified by size exclusion chromatography (SEC) using a Sepharose 6 FF (Cytiva, Marlborough, MA) column. Selected SEC fractions were pooled, concentrated, and exchanged into a formulation buffer using a 300 kDa UF/DF membrane (Repligen, Waltham, MA). The UF/DF product was spiked with a cryoprotectant and filtered through a 0.2 µm PVDF membrane (MilliporeSigma, Burlington, MA). All purified VLPs were frozen and stored at −70 °C. The final concentration of the VLPs was 1 mg/mL as measured by the Bradford Assay (ThermoFisher, Waltham, MA) using the manufacturer’s recommended protocol.</p></section><section id="Sec13"><h3 class="pmc_sec_title">SDS-PAGE and western blot</h3>
<p id="Par25">Clarified cell lysate and clarified supernatant were resolved on a 4–12% gradient, 1.5 mm, 15-well Bis-Tris NuPAGE gel in 1X MES buffer (ThermoFisher, Waltham, MA). For SDS-PAGE, where equal protein was loaded, the protein concentration was determined by Bradford assay. SeeBlue Plus2 pre-stained protein standard (ThermoFisher, Waltham, MA) was included on the SDS-PAGE for molecular weight reference. For Coomassie staining, SDS-PAGE gels were rinsed with water, stained with SimplyBlue Safestain (ThermoFisher, Waltham, MA) for 1 h at RT and destained with water at RT overnight. For Western blot analysis, proteins were transferred onto nitrocellulose membranes using the iBlot western blotting system (ThermoFisher, Waltham, MA). Membranes were transferred to the Bandmate Automated Western Blot Processor (ThermoFisher, Waltham, MA) for blocking and probing. Membranes were blocked for 1 h at RT with 30 mL of 5% non-fat dry milk (Blotting-grade blocker, Bio-Rad, Hercules, CA) in 1X tris-buffered saline containing 0.1% Tween-20 (1X TBST). Membranes were probed with a primary antibody cocktail of 2 mouse norovirus VP1 monoclonal antibodies at a dilution of 1:1000 in 5% non-fat dry milk in 1X TBST for 1 h at RT. The primary antibody cocktail consisted of a 1:1 mixture of two mouse norovirus VP1 monoclonal antibodies: My BioSource.com clone # M120539 (MyBioSource.com, San Diego, CA, cat # MBS832466) and Abcam clone # NVGC-01 (Abcam, Waltham, MA, cat # ab272687). Membranes were probed with an alkaline-phosphatase-conjugated affinipure goat anti-mouse IgG secondary antibody (Jackson ImmunoResearch, West Grove, PA, cat # 1115-055-146) diluted 1:1000 in 5% non-fat dry milk in 1X TBST for 1 h at RT. Membranes were washed 3 times for 5 min each at RT with 1X TBST. Membranes were developed using 1-step NBT/BCIP (Pierce, ThermoFisher, Waltham, MA) according to the manufacturer’s recommendations. Coomassie-stained SDS-PAGE gels and western blots were imaged using an Epson Perfection V370 Photo Scanner and Epson Scan Software version 3.9.2.5US.</p></section><section id="Sec14"><h3 class="pmc_sec_title">Sucrose gradient purification</h3>
<p id="Par26">30 mL frozen bulk lysates were thawed in a water bath, clarified by centrifugation, and concentrated to 1 mL using an Amicon 100 kDa MWCO centrifugal filter unit (MilliporeSigma, Burlington, MA). 5 μL of input material was saved for SDS-PAGE analysis. 16 mL, 15–36% continuous sucrose gradients in phosphate-buffered saline (PBS) were made on the day of purification using a Hoefer SG15 gradient maker (Hoefer, Bridgewater, MA) connected to a Cole Palmer Master Flex L/S Digital peristaltic pump flowed at 3.6 mL/min using size 16 tubing capped with a blunt 4-inch 14-gauge needle placed at the bottom of a 17 mL ultracentrifuge tube. 1 mL of concentrated bulk lysate was gently layered on top, and gradients were placed in SW28.1 buckets and loaded into a SW28Ti swinging bucket rotor (Beckman Coulter, Indianapolis, IN). Samples were centrifuged for 3 h at 150,000 × <em>g</em> in a Beckman Optima CL-100K Ultracentrifuge (Beckman Coulter, Indianapolis, IN) operated at 4 °C under vacuum. Gradients were manually fractionated from the top down using a pipette into 12 × 1.34 mL fractions. 15 μL of each fraction was analyzed on a 4–12% SDS-PAGE gel, and norovirus VLPs consistently eluted at the center of the gradient. Fractions 5–7 were pooled and dialyzed overnight using a 25 kDa MWCO dialysis cassette against norovirus VLP storage buffer containing: 25 mM Tris pH 7.5, 150 mM NaCl, 0.02% PS-80, and 5% sucrose. Protein concentration was estimated by Bradford assay using a BSA standard curve, and concentrations were adjusted to approximately 0.5 mg/mL for use in subsequent assays. VLPs were aliquoted, flash frozen in liquid nitrogen and stored at –80 °C.</p></section><section id="Sec15"><h3 class="pmc_sec_title">Dynamic light scattering (DLS)</h3>
<p id="Par27">The hydrodynamic diameter and polydispersity index (PdI) of purified VLPs were measured using a Malvern Zetasizer Nano ZS instrument (Malvern Panalytical, Westborough, MA). Measurements were recorded using 0.1 mL of purified VLPs at 0.5 mg/mL in norovirus VLP storage buffer (25 mM Tris, pH 7.5, 150 mM NaCl, 0.02% PS-80, and 5% (w/v) sucrose) in a ZEN040 cuvette (Malvern Panalytical, Westborough, MA) using the following instrument parameters:</p>
<ul class="list" style="list-style-type:disc">
<li><p id="Par28">Material = Protein (RI = 1.450, Absorption = 0.001)</p></li>
<li><p id="Par29">Dispersant = 25 mM Tris, 150 mM NaCl, 5% (w/v) sucrose (Viscosity = 1.0454 cP, RI = 1.339)</p></li>
<li><p id="Par30">Temperature = 25 °C</p></li>
<li><p id="Par31">Method = Mark-Houwink</p></li>
<li><p id="Par32">Cell type = ZEN0040</p></li>
<li><p id="Par33">Equilibration time = 60 s</p></li>
<li><p id="Par34">Back scatter = 173⁰</p></li>
<li><p id="Par35">3 measurements each (automatic duration)</p></li>
</ul>
<p id="Par36">General-purpose data analysis was performed in Zetasizer software. Intensity distribution values of the 3 replicates were plotted in GraphPad Prism software for visualization.</p></section><section id="Sec16"><h3 class="pmc_sec_title">VLP treatment</h3>
<p id="Par37">100 mM stocks of diamide (Sigma) and beta-mercaptoethanol (Acros Organics) were freshly prepared in PBS buffer. 2.5 µg of VLPs from each genotype were treated with either diamide or BME at a final concentration of 20 mM at RT in PBS buffer for 1 h. Samples were mixed with 1X LDS buffer (ThermoFisher, Waltham, MA) and boiled for 10 min prior to analysis via SDS-PAGE as described above.</p></section><section id="Sec17"><h3 class="pmc_sec_title">Nano differential scanning fluorimetry (nanoDSF)</h3>
<p id="Par38">Thermal stability assays were performed using a NanoTemper Prometheus NT.48 nanoDSF instrument (Nanotemper, Watertown, MA). Purified VLPs at a concentration of 0.5 mg/mL in norovirus VLP storage buffer were loaded into 3 NanoTemper high-sensitivity capillaries (3 technical replicates each). An initial discovery scan was used to optimize excitation power at 280 nm. Temperature was ramped from 20 °C to 95 °C at a rate of 1 °C per minute, and fluorescence was recorded at 330 nm and 350 nm. The 330/350 fluorescence ratio and its derivative were used to fit the inflection point (<em>T</em><sub>m</sub>). The average 330/350 fluorescence signal of the 3 replicates was plotted in GraphPad Prism software for visualization.</p></section><section id="Sec18"><h3 class="pmc_sec_title">Electron microscopy</h3>
<p id="Par39">Purified VLP samples, along with norovirus VLP storage buffer, were shipped to Nano Imaging Services (NIS, San Diego, CA) for negative-stain electron microscopy analysis. The samples were diluted to a final concentration of 0.012 mg/mL with buffer. Samples were deposited on a layer of continuous carbon supported by nitrocellulose on a 400-mesh copper grid. The grids were prepared by applying 3 μL of sample suspension to a cleaned grid, blotting away with filter paper, and immediately staining with uranyl formate. Samples were imaged on an FEI Tecnai T12 electron microscope (serial number D1100), operating at 120 keV and equipped with an FEI Eagle 4k x 4k CCD camera. Negative-stain grids were transferred into the electron microscope using an RT stage. Data collection is carried out using Leginon software, where high magnification images are acquired by selecting targets at a lower magnification<sup><a href="#CR54" class="usa-link" aria-describedby="CR54">54</a>,<a href="#CR55" class="usa-link" aria-describedby="CR55">55</a></sup>. Images of each grid were acquired at multiple scales to assess the overall distribution of the specimen. After identifying potentially suitable target areas for imaging at lower magnifications, high magnification images were acquired at nominal magnifications of 110,000x (0.099 nm/pixel), 52,000x (0.212 nm/pixel) and 21,000x (0.513 nm/pixel). The images were acquired at a nominal underfocus of −5.0 μm to −1.2 μm and electron doses of ~10–25 e-/Å2. Representative micrographs for each sample collected at 52,000x were selected for qualitative comparisons. All micrographs from the 52,000x magnification were imported into RELION (v4.0.1)<sup><a href="#CR56" class="usa-link" aria-describedby="CR56">56</a></sup> and CTF corrected using CTFFIND (v4.1.14)<sup><a href="#CR57" class="usa-link" aria-describedby="CR57">57</a></sup> VLPs were manually picked from micrographs and were extracted in a 300 ×300 pixel (636 × 636 Å) box and subjected to 2D classification using 4 classes and a particle diameter of 500 Å.</p></section><section id="Sec19"><h3 class="pmc_sec_title">Saliva binding affinity assay</h3>
<p id="Par40">Maxisorp 384-well assay plates (Thermo Fisher, Waltham, MA) were coated with 25 μL of human saliva (Precision for Medicine, Frederick, MD) diluted 1:500 in Dulbecco’s phosphate-buffered saline (DPBS). Plates were sealed, centrifuged at 1500 × <em>g</em> for 1 min and incubated overnight at 4 °C in a humidity chamber. A norovirus VLP masterblock was generated by diluting VLPs in 5% non-fat dry milk powder (NFDMP) in phosphate-buffered saline containing 0.1% tween-20 (PBS-T) to a starting concentration of 20 μg/mL, followed by a two-fold, ten-point serial dilution. The masterblock was sealed, centrifuged at 1500 × <em>g</em> for 1 min and stored overnight at 4 °C in a humidity chamber. The assay plates were washed 6x with 1X PBS-T (100 µL/well) for 1 min each and blocked with 5% NFDMP in PBS-T (80 μL) for 30 min in a humidified incubator (21 °C, 80% RH). VLPs were added to 384-well assay plates and incubated for 1 h in a humidified incubator to allow for VLP binding of human saliva (21 °C, 80% RH). Following incubation, the assay plates were washed 6x with 1X PBS-T (100 µL/well) for 1 min each. Saliva-bound VLP was detected using GII.3-specific polyclonal rabbit serum in 5% NFDMP in PBS-T (1:3000) for 1 h in a humidified incubator (21 °C, 80% RH). Assay plates were washed 6x with PBS-T (100 μL/well) followed by the addition of goat anti-rabbit-HRP IgG (Jackson ImmunoResearch, West Grove, PA) diluted to 1:5000 in 5% NFDMP in PBS-T and incubated for 1 h in a humidified incubator (21 °C, 80% RH). After incubation, assay plates were washed 6x with PBS-T (100 μL/well). Luminescence was developed using 20 μL/well of luminescent substrate (Pierce West Pico PLUS, ThermoFisher, Waltham, MA) for 15 min at RT Ultrasensitive luminescence was read on an Envision plate reader at 0.1 s per well.</p></section><section id="Sec20"><h3 class="pmc_sec_title">HBGA blocking assay</h3>
<p id="Par41">Maxisorp 384-well assay plates (Thermo Fisher, Waltham, MA) were coated with human saliva (Precision for Medicine, Frederick, MD) diluted 1:500 in Dulbecco’s phosphate-buffered saline (DPBS). Assay plates were centrifuged at 1500 × <em>g</em> for 1 min and incubated overnight at 4 °C in a humidity chamber. Ten, two-fold serial dilutions of mouse sera were prepared in 5% non-fat dry milk powder in phosphate-buffered saline with Tween-20 (NFDMP-PBS-T) in 384-well masterblocks (Greiner Bio-One, Monroe, NC) with a starting dilution of 1:10. VLPs prepared at 1 µg/mL in 5% NFDMP-PBS-T were added in a 1:1 dilution to the mouse sera serial dilutions. Masterblocks were sealed, centrifuged at 1500 × <em>g</em> for 1 min and stored overnight at 4 °C in a humidity chamber. Assay plates were rinsed 6x with PBS-T and blocked with 5% NFDMP in PBS-T for 30 min in a humidified incubator (21 °C, 80% RH). VLP-mouse sera mixtures were transferred to assay plates and incubated for 2 h in a humidified incubator (21 °C, 80% RH). Assay plates were washed 6x with PBS-T. VLP was detected by incubation with GII.3-specific polyclonal rabbit serum diluted with 5% NFDMP in PBS-T (1:3000) for 1 h in a humidified incubator (21 °C, 80% RH). Assay plates were washed 6x with PBS-T. Goat anti-rabbit-HRP (Fc)-IgG (1:5000 in 5% NFDMP in PBS-T) (Jackson ImmunoResearch, West Grove, PA) was added to assay plates and incubated for 1 h in a humidified incubator (21 °C, 80% RH). Assay plates were washed 6x with PBS-T. Luminescent signal was developed for 15 min at RT with luminescent substrate (Pierce West Pico PLUS, ThermoFisher, Waltham, MA). BT50 values were defined as the titer at which luminescence readings were 50% of the positive control. A value of 10 was assigned to samples with a BT50 less than the starting dilution of 20. For all experiments, an anti-GII.3 mouse serum sample was included as a blocking control, and plates were rejected if the BT50 for the blocking control was greater than one dilution above or below the known BT50.</p></section><section id="Sec21"><h3 class="pmc_sec_title">Serum IgG and human GII.3 monoclonal antibody ELISA binding assays</h3>
<p id="Par42">Maxisorp 384-well plates (Thermo Fisher, Waltham, MA) were coated with 50 ng/well of VLP diluted in Dulbecco’s phosphate-buffered saline (DPBS). Plates were sealed, centrifuged at 1500 × <em>g</em> for 1 min and incubated overnight at 4 °C in a humidity chamber. Human IgG1 monoclonal antibodies, discovered internally by panning against a human Fab phage display library with norovirus VLPs as antigens, or mouse study serum samples, were diluted in DPBS to a starting concentration of 200 μg/mL (antibody) or 1:50 (serum) and subsequently diluted in masterblocks in a four-fold, ten-point series in 3% non-fat dry milk powder (NFDMP) in PBS containing 0.1% tween-20 (PBS-T). Dilution masterblocks were sealed, centrifuged at 1500 × <em>g</em> for 1 min and stored overnight at 4 °C in a humidity chamber. Assay plates were washed 6x with 1X PBS-T (100 μL/well) and blocked with 3% NFDMP in PBS-T (80 μL) for 30 min in a humidified incubator (21 °C, 80% RH). Monoclonal antibodies or serum were added to 384-well assay plates and incubated for 2 h in a humidified incubator to allow for VLP binding (21 °C, 80% RH). Following incubation, the assay plates were washed 6x with PBS-T (100 μL/well). VLP-bound antibodies were detected with HRP-goat anti-human IgG, Fcγ (min X) (Jackson ImmunoResearch, West Grove, PA, Catalog #109-035-098) or goat anti-mouse IgG (Fc), HRP conjugated (ThermoFisher Catalog #A16084) in 3% NFDMP in PBS-T (1:10,000) for 1 h in a humidified incubator (21 °C, 80% RH). Assay plates were washed 6x with PBS-T (100 μL/well) and developed with luminescent substrate (Pierce West Pico PLUS, ThermoFisher, Waltham, MA) for 15 min at RT. Ultrasensitive luminescence was read on an Envision plate reader (PerkinElmer, Waltham, MA) at 0.1 s per well. Interpolated serum antibody endpoint titers were calculated as the highest dilution where the relative light unit (RLU) signal is above a 50,000 RLU cutoff. Samples where no dilution crossed the threshold were given a placeholder titer of “25”. For samples above the threshold at the highest dilution tested (1:13107200), the value 13107200 was used for the titer. Samples where a well value was the same as 50,000 RLU were given the interpolated titer value at that exact dilution. If a sample crossed the 50000 RLU threshold between two dilutions, the titer at the crossing point was calculated by connecting the nearest points above and below the threshold, solving for the dilution at which this line crossed the threshold.</p></section><section id="Sec22"><h3 class="pmc_sec_title">mRNA/LNP generation</h3>
<p id="Par43">VP1-expressing mRNAs were generated by Trilink Biotechnologies (San Diego, CA) with N1-methyl-pseudouridine triphosphate modification and clean-cap. LNP-encapsulating mRNA was prepared by a rapid precipitation process as previously described<sup><a href="#CR58" class="usa-link" aria-describedby="CR58">58</a></sup>. The lipid components of the LNP comprised an asymmetric ionizable amino lipid, 1,2-distearoyl-sn-glycero-3-phosphocholine (DSPC), cholesterol and poly(ethylene glycol)2000-dimyristoylglycerol (PEG2000-DMG) in a molar ratio of 58:30:10:2, respectively.</p></section><section id="Sec23"><h3 class="pmc_sec_title">Rabbit polyclonal sera generation</h3>
<p id="Par44">The Institutional Animal Care and Use Committee at Merck &amp; Co., Inc., Rahway, NJ, USA, approved the rabbit studies. Three female New Zealand white rabbits (approx. four months old at receipt) from Jackson Labs (Bar Harbor, ME) were housed singly and randomized into two groups. Animals were inoculated three times (two weeks between doses) intramuscularly in the quadriceps bilaterally. Inoculations consisted of 50 μg of appropriate VLP and 225 μg of amorphous aluminum hydroxyphosphate sulfate (AAHS) in 250 μL total (remaining volume was phosphate-buffered saline), administered once in each quadriceps at each time point. Blood draws were performed at weeks 1, 3, 5, and 6 after the first inoculation. All blood draws were from the ear artery, and the final was a terminal bleed. For all survival blood draws, animals were dosed with acepromazine (1–3 mg/kg subcutaneously) and approximately 2 mL of blood was drawn from the ear artery. For terminal bleeds, animals were administered ketamine (100 mg) and xylazine (20 mg) intramuscularly, then were bled from the ear artery until no more blood could be acquired. A cardiac puncture was performed for the remaining blood, and then euthanasia was performed by administration of Euthasol intravenously. Euthanasia was confirmed by cutting the diaphragm to cause pneumothorax. All blood was spun at 4000 ×<em>g</em> for 5 min to separate serum, which was transferred to new tubes.</p></section><section id="Sec24"><h3 class="pmc_sec_title">Mouse immunization studies</h3>
<p id="Par45">The Institutional Animal Care and Use Committee at Merck &amp; Co., Inc., Rahway, NJ, USA, approved the mouse studies. BALB/c mice aged 6–8 weeks were obtained from Charles River Laboratory (Malvern, PA). Animals were housed in a company animal facility, in accordance with the Guide for the Care and Use of Laboratory Animals, and the facility is credentialed by the Association for Assessment and Accreditation of Laboratory Animal Care<sup><a href="#CR59" class="usa-link" aria-describedby="CR59">59</a></sup>. Seven groups of mice (<em>N</em> = 8/group) were immunized at weeks 0 and 4 with intramuscular injections of mRNAs in a lipid nanoparticle formulation. Mice were injected with 100 μL total volume per vaccination with 50 μL injected per thigh. Two weeks after the first immunization, blood was drawn via the tail vein. Specifically, a small cut was made on the tail vein with a scalpel blade (Bard-Parker, Size 15, Fisher Catalog #0268880) and around 100–150 μL blood was collected into serum separator tubes (Sarstedt AG &amp; Co. KG Microvette 500 Z-Gel Catalog #20.1344). Two weeks after the second immunization, all mice were euthanized via CO<sub>2</sub> exposure. Blood collection was performed via cardiac puncture using a 1 mL syringe with a 25G needle (BD Catalog #309626). Blood was transferred into serum separator tubes. Euthanasia was confirmed by cervical dislocation. For VLP immunogen studies, nine groups of mice (<em>N</em> = 10 /group) were immunized with 2 μg or 0.2 μg of either GII.3 or GII.3-DS1 VLPs with and without amorphous aluminum hydroxyphosphate sulfate (AAHS) at weeks 0 and 4. Mice were injected intramuscularly with 100 μL total volume, 50 μL in each thigh. Blood was drawn at weeks 2, 6, and 10.</p></section><section id="Sec25"><h3 class="pmc_sec_title">Statistical analysis</h3>
<p id="Par46">Statistical analyses were completed using GraphPad Prism version 10.2.2. Data were tested for normality using a Shapiro–Wilk test. If the normality of both groups is satisfied, then statistical significance was determined by an unpaired two-sample <em>t</em>-test with Welch’s correction. If normality is not satisfied, statistical significance was determined using a Mann–Whitney <em>U</em>-test.</p></section><section id="Sec26"><h3 class="pmc_sec_title">Edman degradation sequencing</h3>
<p id="Par47">N-terminal Edman degradation sequencing was outsourced to Evotec (Princeton, NJ). Purified GII.3 VLPs were shipped at 4 °C to Evotec. The N-termini of the full-length VP1 protein and 28 kDa cleavage product were sequenced to determine the site of VP1 cleavage within the cell during expression.</p></section></section><section id="Sec27"><h2 class="pmc_sec_title">Supplementary information</h2>
<section class="sm xbox font-sm" id="MOESM1"><div class="media p"><div class="caption">
<a href="/articles/instance/12365219/bin/41541_2025_1254_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">8July2025Supp</a><sup> (1.3MB, pdf) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>The authors would like to thank Eberhard Durr and Lan Zhang, Merck &amp; Co., Inc., Rahway, NJ, USA for their thoughtful reviews of the manuscript and helpful suggestions. The work is fully funded by Merck &amp; Co., Inc., Rahway, NJ, USA.</p></section><section id="notes1"><h2 class="pmc_sec_title">Author contributions</h2>
<p>C.W., J.D.G., K.B., G.O., and J.W. were responsible for manuscript conceptualization. C.W., J.D.G., K.B., G.O., A.R.S., E.A.D., C.M.F., P.S., E.G.-F., L.A.D., U.J., and J.W. were involved in experimental investigation, experimental methodology, and formal data analysis. O.C.P., S.M., A.F., and Z.C. were responsible for the generation and characterization of the anti-GII.3-VP1 monoclonal antibodies. A.F. was responsible for mRNA design. C.D. was responsible for mRNA procurement and formulation. C.M.F. and P.S. were responsible for animal experiment design and oversight. C.W., J.D.G., K.B., G.O., C.M.F., A.R.S., and J.W. were responsible for original draft preparation and data curation. J.W. was responsible for project administration. All authors have read and agreed to the published version of the manuscript.</p></section><section id="notes2"><h2 class="pmc_sec_title">Data availability</h2>
<p>Data were plotted and visualized using GraphPad Prism version 10.2.2. Data generated and analyzed in the current study are included in this published article and its supplementary information files. All nucleotide sequences have been deposited in GenBank (<a href="https://www.ncbi.nlm.nih.gov/genbank/" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/genbank/</a>) under accession numbers <a href="https://www.ncbi.nlm.nih.gov/nuccore/?term=PV921589:PV921598%5Bpacc%5D" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PV921589-PV921598</a>. nsEM datasets are available from the corresponding author upon request. Non-commercial critical reagents are available upon request. Readers may contact the corresponding author to request reagents or materials via a Material Transfer Agreement (MTA).</p></section><section id="notes3"><h2 class="pmc_sec_title">Code availability</h2>
<p>No custom scripts were generated in this study. Electron micrographs were analyzed using RELION (v4.0.1) for CTF correction, particle picking, and 2D classification.</p></section><section id="FPar1"><h2 class="pmc_sec_title">Competing interests</h2>
<p id="Par48">All co-authors are employees of Merck Sharp &amp; Dohme LLC, a subsidiary of Merck &amp; Co., Inc., Rahway, NJ, USA and may own stock or hold stock options in Merck &amp; Co., Inc., Rahway, NJ, USA. This does not alter the authors’ adherence to all journal policies on competing interests and sharing data and materials. In addition, a subset of the authors is listed as inventors on a patent surrounding this work held by Merck &amp; Co., Inc., Rahway, NJ, USA.</p></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm">
<div class="fn p" id="fn1"><p><strong>Publisher’s note</strong> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></div>
<div class="fn p" id="fn2"><p>These authors contributed equally: Christopher Warren, Jennifer D. Galli.</p></div>
</div></section><section id="sec28"><h2 class="pmc_sec_title">Supplementary information</h2>
<p>The online version contains supplementary material available at 10.1038/s41541-025-01254-2.</p></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Lopman, B. A., Steele, D., Kirkwood, C. D. &amp; Parashar, U. D. The vast and varied global burden of norovirus: prospects for prevention and control. <em>PLoS Med.</em><strong>13</strong>, e1001999 (2016).
</cite> [<a href="https://doi.org/10.1371/journal.pmed.1001999" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4846155/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27115709/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lopman,%20B.%20A.,%20Steele,%20D.,%20Kirkwood,%20C.%20D.%20&amp;%20Parashar,%20U.%20D.%20The%20vast%20and%20varied%20global%20burden%20of%20norovirus:%20prospects%20for%20prevention%20and%20control.%20PLoS%20Med.13,%20e1001999%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>Teunis, P. F. et al. Norwalk virus: how infectious is it?. <em>J. Med. Virol.</em><strong>80</strong>, 1468–1476 (2008).
</cite> [<a href="https://doi.org/10.1002/jmv.21237" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18551613/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Teunis,%20P.%20F.%20et%20al.%20Norwalk%20virus:%20how%20infectious%20is%20it?.%20J.%20Med.%20Virol.80,%201468%E2%80%931476%20(2008)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>de Graaf, M., van Beek, J. &amp; Koopmans, M. P. Human norovirus transmission and evolution in a changing world. <em>Nat. Rev. Microbiol.</em><strong>14</strong>, 421–433 (2016).
</cite> [<a href="https://doi.org/10.1038/nrmicro.2016.48" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27211790/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?de%20Graaf,%20M.,%20van%20Beek,%20J.%20&amp;%20Koopmans,%20M.%20P.%20Human%20norovirus%20transmission%20and%20evolution%20in%20a%20changing%20world.%20Nat.%20Rev.%20Microbiol.14,%20421%E2%80%93433%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Verhoef, L. et al. Norovirus genotype profiles associated with foodborne transmission, 1999–2012. <em>Emerg. Infect. Dis.</em><strong>21</strong>, 592–599 (2015).
</cite> [<a href="https://doi.org/10.3201/eid2104.141073" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4378480/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25811368/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Verhoef,%20L.%20et%20al.%20Norovirus%20genotype%20profiles%20associated%20with%20foodborne%20transmission,%201999%E2%80%932012.%20Emerg.%20Infect.%20Dis.21,%20592%E2%80%93599%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Bartsch, S. M., Lopman, B. A., Ozawa, S., Hall, A. J. &amp; Lee, B. Y. Global economic burden of norovirus gastroenteritis. <em>PLoS One</em><strong>11</strong>, e0151219 (2016).
</cite> [<a href="https://doi.org/10.1371/journal.pone.0151219" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4846012/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27115736/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bartsch,%20S.%20M.,%20Lopman,%20B.%20A.,%20Ozawa,%20S.,%20Hall,%20A.%20J.%20&amp;%20Lee,%20B.%20Y.%20Global%20economic%20burden%20of%20norovirus%20gastroenteritis.%20PLoS%20One11,%20e0151219%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Ahmed, S. M. et al. Global prevalence of norovirus in cases of gastroenteritis: a systematic review and meta-analysis. <em>Lancet Infect. Dis.</em><strong>14</strong>, 725–730 (2014).
</cite> [<a href="https://doi.org/10.1016/S1473-3099(14)70767-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8006533/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24981041/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ahmed,%20S.%20M.%20et%20al.%20Global%20prevalence%20of%20norovirus%20in%20cases%20of%20gastroenteritis:%20a%20systematic%20review%20and%20meta-analysis.%20Lancet%20Infect.%20Dis.14,%20725%E2%80%93730%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>U.S. Centers for Disease Control and Prevention. <em>Norovirus Facts and Stats</em><a href="https://www.cdc.gov/norovirus/data-research/?CDC_AAref_Val=" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.cdc.gov/norovirus/data-research/?CDC_AAref_Val=</a>, <a href="https://www.cdc.gov/norovirus/burden.html" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.cdc.gov/norovirus/burden.html</a> (U.S. Centers for Disease Control and Prevention, 2024, May 8).</cite>
</li>
<li id="CR8">
<span class="label">8.</span><cite>Chhabra, P. et al. Corrigendum: Updated classification of norovirus genogroups and genotypes. <em>J. Gen. Virol.</em><strong>101</strong>, 893 (2020).
</cite> [<a href="https://doi.org/10.1099/jgv.0.001475" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7641392/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32854814/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chhabra,%20P.%20et%20al.%20Corrigendum:%20Updated%20classification%20of%20norovirus%20genogroups%20and%20genotypes.%20J.%20Gen.%20Virol.101,%20893%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>Hall, A. J. et al. Norovirus disease in the United States. <em>Emerg. Infect. Dis.</em><strong>19</strong>, 1198–1205 (2013).
</cite> [<a href="https://doi.org/10.3201/eid1908.130465" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3739528/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23876403/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hall,%20A.%20J.%20et%20al.%20Norovirus%20disease%20in%20the%20United%20States.%20Emerg.%20Infect.%20Dis.19,%201198%E2%80%931205%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>Wangchuk, S., Matsumoto, T., Iha, H. &amp; Ahmed, K. Surveillance of norovirus among children with diarrhea in four major hospitals in Bhutan: replacement of GII.21 by GII.3 as a dominant genotype. <em>PLoS ONE</em><strong>12</strong>, e0184826 (2017).
</cite> [<a href="https://doi.org/10.1371/journal.pone.0184826" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5599041/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28910371/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wangchuk,%20S.,%20Matsumoto,%20T.,%20Iha,%20H.%20&amp;%20Ahmed,%20K.%20Surveillance%20of%20norovirus%20among%20children%20with%20diarrhea%20in%20four%20major%20hospitals%20in%20Bhutan:%20replacement%20of%20GII.21%20by%20GII.3%20as%20a%20dominant%20genotype.%20PLoS%20ONE12,%20e0184826%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Boonchan, M. et al. The dynamics of norovirus genotypes and genetic analysis of a novel recombinant GII.P12-GII.3 among infants and children in Bangkok, Thailand between 2014 and 2016. <em>Infect. Genet. Evol.</em><strong>60</strong>, 133–139 (2018).
</cite> [<a href="https://doi.org/10.1016/j.meegid.2018.02.028" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29471118/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Boonchan,%20M.%20et%20al.%20The%20dynamics%20of%20norovirus%20genotypes%20and%20genetic%20analysis%20of%20a%20novel%20recombinant%20GII.P12-GII.3%20among%20infants%20and%20children%20in%20Bangkok,%20Thailand%20between%202014%20and%202016.%20Infect.%20Genet.%20Evol.60,%20133%E2%80%93139%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Utsumi, T. et al. Molecular epidemiology and genetic diversity of norovirus infection in children hospitalized with acute gastroenteritis in East Java, Indonesia in 2015-2019. <em>Infect. Genet Evol.</em><strong>88</strong>, 104703. 10.1016/j.meegid.2020.104703 (2021).
</cite> [<a href="https://doi.org/10.1016/j.meegid.2020.104703" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33401005/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Utsumi,%20T.%20et%20al.%20Molecular%20epidemiology%20and%20genetic%20diversity%20of%20norovirus%20infection%20in%20children%20hospitalized%20with%20acute%20gastroenteritis%20in%20East%20Java,%20Indonesia%20in%202015-2019.%20Infect.%20Genet%20Evol.88,%20104703.%2010.1016/j.meegid.2020.104703%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>Cannon, J. L. et al. Global trends in norovirus genotype distribution among children with acute gastroenteritis. <em>Emerg. Infect. Dis.</em><strong>27</strong>, 1438–1445 (2021).
</cite> [<a href="https://doi.org/10.3201/eid2705.204756" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8084493/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33900173/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cannon,%20J.%20L.%20et%20al.%20Global%20trends%20in%20norovirus%20genotype%20distribution%20among%20children%20with%20acute%20gastroenteritis.%20Emerg.%20Infect.%20Dis.27,%201438%E2%80%931445%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>NoroSurv. <em>Norosurv.org</em>, <a href="https://www.norosurv.org/login" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.norosurv.org/login</a> (NoroSurv., 2025).</cite>
</li>
<li id="CR15">
<span class="label">15.</span><cite>Jiang, X., Wang, M., Graham, D. Y. &amp; Estes, M. K. Expression, self-assembly, and antigenicity of the Norwalk virus capsid protein. <em>J. Virol.</em><strong>66</strong>, 6527–6532 (1992).
</cite> [<a href="https://doi.org/10.1128/jvi.66.11.6527-6532.1992" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC240146/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/1328679/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jiang,%20X.,%20Wang,%20M.,%20Graham,%20D.%20Y.%20&amp;%20Estes,%20M.%20K.%20Expression,%20self-assembly,%20and%20antigenicity%20of%20the%20Norwalk%20virus%20capsid%20protein.%20J.%20Virol.66,%206527%E2%80%936532%20(1992)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Prasad, B. V. et al. X-ray crystallographic structure of the Norwalk virus capsid. <em>Science</em><strong>286</strong>, 287–290 (1999).
</cite> [<a href="https://doi.org/10.1126/science.286.5438.287" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10514371/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Prasad,%20B.%20V.%20et%20al.%20X-ray%20crystallographic%20structure%20of%20the%20Norwalk%20virus%20capsid.%20Science286,%20287%E2%80%93290%20(1999)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>Prasad, B. V., Rothnagel, R., Jiang, X. &amp; Estes, M. K. Three-dimensional structure of baculovirus-expressed Norwalk virus capsids. <em>J. Virol.</em><strong>68</strong>, 5117–5125 (1994).
</cite> [<a href="https://doi.org/10.1128/jvi.68.8.5117-5125.1994" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC236455/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/8035511/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Prasad,%20B.%20V.,%20Rothnagel,%20R.,%20Jiang,%20X.%20&amp;%20Estes,%20M.%20K.%20Three-dimensional%20structure%20of%20baculovirus-expressed%20Norwalk%20virus%20capsids.%20J.%20Virol.68,%205117%E2%80%935125%20(1994)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Bertolotti-Ciarlet, A., White, L. J., Chen, R., Prasad, B. V. &amp; Estes, M. K. Structural requirements for the assembly of Norwalk virus-like particles. <em>J. Virol.</em><strong>76</strong>, 4044–4055 (2002).
</cite> [<a href="https://doi.org/10.1128/JVI.76.8.4044-4055.2002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC136079/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11907243/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bertolotti-Ciarlet,%20A.,%20White,%20L.%20J.,%20Chen,%20R.,%20Prasad,%20B.%20V.%20&amp;%20Estes,%20M.%20K.%20Structural%20requirements%20for%20the%20assembly%20of%20Norwalk%20virus-like%20particles.%20J.%20Virol.76,%204044%E2%80%934055%20(2002)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Vongpunsawad, S., Venkataram Prasad, B. V. &amp; Estes, M. K. Norwalk virus minor capsid protein VP2 associates within the VP1 shell domain. <em>J. Virol.</em><strong>87</strong>, 4818–4825 (2013).
</cite> [<a href="https://doi.org/10.1128/JVI.03508-12" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3624303/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23408637/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Vongpunsawad,%20S.,%20Venkataram%20Prasad,%20B.%20V.%20&amp;%20Estes,%20M.%20K.%20Norwalk%20virus%20minor%20capsid%20protein%20VP2%20associates%20within%20the%20VP1%20shell%20domain.%20J.%20Virol.87,%204818%E2%80%934825%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>Bertolotti-Ciarlet, A., Crawford, S. E., Hutson, A. M. &amp; Estes, M. K. The 3’ end of Norwalk virus mRNA contains determinants that regulate the expression and stability of the viral capsid protein VP1: a novel function for the VP2 protein. <em>J. Virol.</em><strong>77</strong>, 11603–11615 (2003).
</cite> [<a href="https://doi.org/10.1128/JVI.77.21.11603-11615.2003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC229252/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/14557646/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bertolotti-Ciarlet,%20A.,%20Crawford,%20S.%20E.,%20Hutson,%20A.%20M.%20&amp;%20Estes,%20M.%20K.%20The%203%E2%80%99%20end%20of%20Norwalk%20virus%20mRNA%20contains%20determinants%20that%20regulate%20the%20expression%20and%20stability%20of%20the%20viral%20capsid%20protein%20VP1:%20a%20novel%20function%20for%20the%20VP2%20protein.%20J.%20Virol.77,%2011603%E2%80%9311615%20(2003)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>Jung, J. et al. High-resolution cryo-EM structures of outbreak strain human norovirus shells reveal size variations. <em>Proc. Natl. Acad. Sci. USA</em><strong>116</strong>, 12828–12832 (2019).
</cite> [<a href="https://doi.org/10.1073/pnas.1903562116" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6601263/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31182604/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jung,%20J.%20et%20al.%20High-resolution%20cryo-EM%20structures%20of%20outbreak%20strain%20human%20norovirus%20shells%20reveal%20size%20variations.%20Proc.%20Natl.%20Acad.%20Sci.%20USA116,%2012828%E2%80%9312832%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>Choi, J. M., Hutson, A. M., Estes, M. K. &amp; Prasad, B. V. Atomic resolution structural characterization of recognition of histo-blood group antigens by Norwalk virus. <em>Proc. Natl. Acad. Sci. USA</em><strong>105</strong>, 9175–9180 (2008).
</cite> [<a href="https://doi.org/10.1073/pnas.0803275105" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2453692/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18599458/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Choi,%20J.%20M.,%20Hutson,%20A.%20M.,%20Estes,%20M.%20K.%20&amp;%20Prasad,%20B.%20V.%20Atomic%20resolution%20structural%20characterization%20of%20recognition%20of%20histo-blood%20group%20antigens%20by%20Norwalk%20virus.%20Proc.%20Natl.%20Acad.%20Sci.%20USA105,%209175%E2%80%939180%20(2008)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>Treanor, J. J. et al. A novel intramuscular bivalent norovirus virus-like particle vaccine candidate-reactogenicity, safety, and immunogenicity in a phase 1 trial in healthy adults. <em>J. Infect. Dis.</em><strong>210</strong>, 1763–1771 (2014).
</cite> [<a href="https://doi.org/10.1093/infdis/jiu337" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8483568/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24951828/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Treanor,%20J.%20J.%20et%20al.%20A%20novel%20intramuscular%20bivalent%20norovirus%20virus-like%20particle%20vaccine%20candidate-reactogenicity,%20safety,%20and%20immunogenicity%20in%20a%20phase%201%20trial%20in%20healthy%20adults.%20J.%20Infect.%20Dis.210,%201763%E2%80%931771%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>Fang, H., Tan, M., Xia, M., Wang, L. &amp; Jiang, X. Norovirus P particle efficiently elicits innate, humoral and cellular immunity. <em>PLoS One</em><strong>8</strong>, e63269 (2013).
</cite> [<a href="https://doi.org/10.1371/journal.pone.0063269" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3639243/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23638188/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Fang,%20H.,%20Tan,%20M.,%20Xia,%20M.,%20Wang,%20L.%20&amp;%20Jiang,%20X.%20Norovirus%20P%20particle%20efficiently%20elicits%20innate,%20humoral%20and%20cellular%20immunity.%20PLoS%20One8,%20e63269%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>Kim, L. et al. Safety and immunogenicity of an oral tablet norovirus vaccine, a phase I randomized, placebo-controlled trial. <em>JCI Insight</em><strong>3</strong>10.1172/jci.insight.121077 (2018).</cite> [<a href="https://doi.org/10.1172/jci.insight.121077" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6124525/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29997294/" class="usa-link">PubMed</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>Atmar, R. L. et al. An exploratory study of the salivary immunoglobulin A responses to 1 dose of a norovirus virus-like particle candidate vaccine in healthy adults. <em>J. Infect. Dis.</em><strong>219</strong>, 410–414 (2019).
</cite> [<a href="https://doi.org/10.1093/infdis/jiy529" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8483565/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30203081/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Atmar,%20R.%20L.%20et%20al.%20An%20exploratory%20study%20of%20the%20salivary%20immunoglobulin%20A%20responses%20to%201%20dose%20of%20a%20norovirus%20virus-like%20particle%20candidate%20vaccine%20in%20healthy%20adults.%20J.%20Infect.%20Dis.219,%20410%E2%80%93414%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Wrapp, D. et al. Cryo-EM structure of the 2019-nCoV Spike in the prefusion conformation. <em>bioRxiv</em>10.1101/2020.02.11.944462 (2020).
</cite> [<a href="https://doi.org/10.1126/science.abb2507" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7164637/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32075877/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wrapp,%20D.%20et%20al.%20Cryo-EM%20structure%20of%20the%202019-nCoV%20Spike%20in%20the%20prefusion%20conformation.%20bioRxiv10.1101/2020.02.11.944462%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Pallesen, J. et al. Immunogenicity and structures of a rationally designed prefusion MERS-CoV spike antigen. <em>Proc. Natl. Acad. Sci. USA</em><strong>114</strong>, E7348–E7357 (2017).
</cite> [<a href="https://doi.org/10.1073/pnas.1707304114" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5584442/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28807998/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Pallesen,%20J.%20et%20al.%20Immunogenicity%20and%20structures%20of%20a%20rationally%20designed%20prefusion%20MERS-CoV%20spike%20antigen.%20Proc.%20Natl.%20Acad.%20Sci.%20USA114,%20E7348%E2%80%93E7357%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>Hsieh, C. L. et al. Structure-based design of prefusion-stabilized SARS-CoV-2 spikes. <em>Science</em><strong>369</strong>, 1501–1505 (2020).
</cite> [<a href="https://doi.org/10.1126/science.abd0826" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7402631/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32703906/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hsieh,%20C.%20L.%20et%20al.%20Structure-based%20design%20of%20prefusion-stabilized%20SARS-CoV-2%20spikes.%20Science369,%201501%E2%80%931505%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR30">
<span class="label">30.</span><cite>Tan, T. J. C. et al. High-throughput identification of prefusion-stabilizing mutations in SARS-CoV-2 spike. <em>Nat. Commun.</em><strong>14</strong>, 2003 (2023).
</cite> [<a href="https://doi.org/10.1038/s41467-023-37786-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10086000/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37037866/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tan,%20T.%20J.%20C.%20et%20al.%20High-throughput%20identification%20of%20prefusion-stabilizing%20mutations%20in%20SARS-CoV-2%20spike.%20Nat.%20Commun.14,%202003%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>Papi, A. et al. Respiratory syncytial virus prefusion F protein vaccine in older adults. <em>N. Engl. J. Med.</em><strong>388</strong>, 595–608 (2023).
</cite> [<a href="https://doi.org/10.1056/NEJMoa2209604" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36791160/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Papi,%20A.%20et%20al.%20Respiratory%20syncytial%20virus%20prefusion%20F%20protein%20vaccine%20in%20older%20adults.%20N.%20Engl.%20J.%20Med.388,%20595%E2%80%93608%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>McLellan, J. S. et al. Structure-based design of a fusion glycoprotein vaccine for respiratory syncytial virus. <em>Science</em><strong>342</strong>, 592–598 (2013).
</cite> [<a href="https://doi.org/10.1126/science.1243283" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4461862/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24179220/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?McLellan,%20J.%20S.%20et%20al.%20Structure-based%20design%20of%20a%20fusion%20glycoprotein%20vaccine%20for%20respiratory%20syncytial%20virus.%20Science342,%20592%E2%80%93598%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Verardi, R. et al. Disulfide stabilization of human norovirus GI.1 virus-like particles focuses immune response toward blockade epitopes. <em>NPJ Vaccines</em><strong>5</strong>, 110 (2020).
</cite> [<a href="https://doi.org/10.1038/s41541-020-00260-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7736355/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33318483/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Verardi,%20R.%20et%20al.%20Disulfide%20stabilization%20of%20human%20norovirus%20GI.1%20virus-like%20particles%20focuses%20immune%20response%20toward%20blockade%20epitopes.%20NPJ%20Vaccines5,%20110%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>Jumper, J. et al. Highly accurate protein structure prediction with AlphaFold. <em>Nature</em><strong>596</strong>, 583–589 (2021).
</cite> [<a href="https://doi.org/10.1038/s41586-021-03819-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8371605/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34265844/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jumper,%20J.%20et%20al.%20Highly%20accurate%20protein%20structure%20prediction%20with%20AlphaFold.%20Nature596,%20583%E2%80%93589%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>Chaudhury, S., Lyskov, S. &amp; Gray, J. J. PyRosetta: a script-based interface for implementing molecular modeling algorithms using Rosetta. <em>Bioinformatics</em><strong>26</strong>, 689–691 (2010).
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btq007" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2828115/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20061306/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chaudhury,%20S.,%20Lyskov,%20S.%20&amp;%20Gray,%20J.%20J.%20PyRosetta:%20a%20script-based%20interface%20for%20implementing%20molecular%20modeling%20algorithms%20using%20Rosetta.%20Bioinformatics26,%20689%E2%80%93691%20(2010)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR36">
<span class="label">36.</span><cite>Ma, S. et al. Chimeric GII.3/GII.6 norovirus capsid (VP1) proteins: characterization by electron microscopy, trypsin sensitivity and binding to histo-blood group antigens. <em>Arch. Virol.</em><strong>163</strong>, 3265–3273 (2018).
</cite> [<a href="https://doi.org/10.1007/s00705-018-4002-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30143876/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ma,%20S.%20et%20al.%20Chimeric%20GII.3/GII.6%20norovirus%20capsid%20(VP1)%20proteins:%20characterization%20by%20electron%20microscopy,%20trypsin%20sensitivity%20and%20binding%20to%20histo-blood%20group%20antigens.%20Arch.%20Virol.163,%203265%E2%80%933273%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR37">
<span class="label">37.</span><cite>Hardy, M. E., White, L. J., Ball, J. M. &amp; Estes, M. K. Specific proteolytic cleavage of recombinant Norwalk virus capsid protein. <em>J. Virol.</em><strong>69</strong>, 1693–1698 (1995).
</cite> [<a href="https://doi.org/10.1128/jvi.69.3.1693-1698.1995" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC188770/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/7853506/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hardy,%20M.%20E.,%20White,%20L.%20J.,%20Ball,%20J.%20M.%20&amp;%20Estes,%20M.%20K.%20Specific%20proteolytic%20cleavage%20of%20recombinant%20Norwalk%20virus%20capsid%20protein.%20J.%20Virol.69,%201693%E2%80%931698%20(1995)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR38">
<span class="label">38.</span><cite>Kumar, S., Ochoa, W., Kobayashi, S. &amp; Reddy, V. S. Presence of a surface-exposed loop facilitates trypsinization of particles of Sinsiro virus, a genogroup II.3 norovirus. <em>J. Virol.</em><strong>81</strong>, 1119–1128 (2007).
</cite> [<a href="https://doi.org/10.1128/JVI.01909-06" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1797490/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17079293/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kumar,%20S.,%20Ochoa,%20W.,%20Kobayashi,%20S.%20&amp;%20Reddy,%20V.%20S.%20Presence%20of%20a%20surface-exposed%20loop%20facilitates%20trypsinization%20of%20particles%20of%20Sinsiro%20virus,%20a%20genogroup%20II.3%20norovirus.%20J.%20Virol.81,%201119%E2%80%931128%20(2007)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR39">
<span class="label">39.</span><cite>Zheng, L. et al. Comprehensive characterization of a major capsid protein derived from a documented GII.6 norovirus strain. <em>Arch. Virol.</em><strong>162</strong>, 3863–3868 (2017).
</cite> [<a href="https://doi.org/10.1007/s00705-017-3537-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28866835/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zheng,%20L.%20et%20al.%20Comprehensive%20characterization%20of%20a%20major%20capsid%20protein%20derived%20from%20a%20documented%20GII.6%20norovirus%20strain.%20Arch.%20Virol.162,%203863%E2%80%933868%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR40">
<span class="label">40.</span><cite>Huo, Y. et al. Enzymatic cleavage promotes disassembly of GII.3 norovirus virus like particles and its binding to salivary histo-blood group antigens. <em>Virus Res</em>. <strong>240</strong>, 18–24 (2017).
</cite> [<a href="https://doi.org/10.1016/j.virusres.2017.07.017" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28754559/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Huo,%20Y.%20et%20al.%20Enzymatic%20cleavage%20promotes%20disassembly%20of%20GII.3%20norovirus%20virus%20like%20particles%20and%20its%20binding%20to%20salivary%20histo-blood%20group%20antigens.%20Virus%20Res.%20240,%2018%E2%80%9324%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR41">
<span class="label">41.</span><cite>Huo, Y., Wan, X., Ling, T. &amp; Shen, S. Biological and immunological characterization of norovirus major capsid proteins from three different genotypes. <em>Micro Pathog.</em><strong>90</strong>, 78–83 (2016).</cite> [<a href="https://doi.org/10.1016/j.micpath.2015.11.022" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26616166/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Huo,%20Y.,%20Wan,%20X.,%20Ling,%20T.%20&amp;%20Shen,%20S.%20Biological%20and%20immunological%20characterization%20of%20norovirus%20major%20capsid%20proteins%20from%20three%20different%20genotypes.%20Micro%20Pathog.90,%2078%E2%80%9383%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR42">
<span class="label">42.</span><cite>Ayouni, S. et al. Relationship between GII.3 norovirus infections and blood group antigens in young children in Tunisia. <em>Clin. Microbiol Infect.</em><strong>21</strong>, 874.e871–874.e878 (2015).</cite> [<a href="https://doi.org/10.1016/j.cmi.2015.05.015" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26003283/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ayouni,%20S.%20et%20al.%20Relationship%20between%20GII.3%20norovirus%20infections%20and%20blood%20group%20antigens%20in%20young%20children%20in%20Tunisia.%20Clin.%20Microbiol%20Infect.21,%20874.e871%E2%80%93874.e878%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR43">
<span class="label">43.</span><cite>Swaminathan, G. et al. A novel lipid nanoparticle adjuvant significantly enhances B cell and T cell responses to sub-unit vaccine antigens. <em>Vaccine</em><strong>34</strong>, 110–119 (2016).
</cite> [<a href="https://doi.org/10.1016/j.vaccine.2015.10.132" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26555351/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Swaminathan,%20G.%20et%20al.%20A%20novel%20lipid%20nanoparticle%20adjuvant%20significantly%20enhances%20B%20cell%20and%20T%20cell%20responses%20to%20sub-unit%20vaccine%20antigens.%20Vaccine34,%20110%E2%80%93119%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR44">
<span class="label">44.</span><cite>Kheirvari, M., Liu, H. &amp; Tumban, E. Virus-like particle vaccines and platforms for vaccine development. <em>Viruses</em><strong>15</strong>10.3390/v15051109 (2023).</cite> [<a href="https://doi.org/10.3390/v15051109" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10223759/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37243195/" class="usa-link">PubMed</a>]</li>
<li id="CR45">
<span class="label">45.</span><cite>Nooraei, S. et al. Virus-like particles: preparation, immunogenicity and their roles as nanovaccines and drug nanocarriers. <em>J. Nanobiotechnol.</em><strong>19</strong>, 59 (2021).</cite> [<a href="https://doi.org/10.1186/s12951-021-00806-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7905985/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33632278/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Nooraei,%20S.%20et%20al.%20Virus-like%20particles:%20preparation,%20immunogenicity%20and%20their%20roles%20as%20nanovaccines%20and%20drug%20nanocarriers.%20J.%20Nanobiotechnol.19,%2059%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR46">
<span class="label">46.</span><cite>Chaudhary, N., Weissman, D. &amp; Whitehead, K. A. mRNA vaccines for infectious diseases: principles, delivery and clinical translation. <em>Nat. Rev. Drug Discov.</em><strong>20</strong>, 817–838 (2021).
</cite> [<a href="https://doi.org/10.1038/s41573-021-00283-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8386155/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34433919/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chaudhary,%20N.,%20Weissman,%20D.%20&amp;%20Whitehead,%20K.%20A.%20mRNA%20vaccines%20for%20infectious%20diseases:%20principles,%20delivery%20and%20clinical%20translation.%20Nat.%20Rev.%20Drug%20Discov.20,%20817%E2%80%93838%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR47">
<span class="label">47.</span><cite>Leroux-Roels, G., Atmar, R. L., Cramer, J. P., Escudero, I. &amp; Borkowski, A. Persistence of the immune response to an intramuscular bivalent (GI.1/GII.4) Norovirus vaccine in adults. <em>Vaccines</em><strong>13</strong>10.3390/vaccines13010082 (2025).</cite> [<a href="https://doi.org/10.3390/vaccines13010082" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11768790/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39852862/" class="usa-link">PubMed</a>]</li>
<li id="CR48">
<span class="label">48.</span><cite>Bollman, B., Jorquera, P. &amp; Arunkumar, G. Norovirus mRNA Vaccines. <em>World Intellectual Property Organization</em> WO2024015890 (2024).</cite>
</li>
<li id="CR49">
<span class="label">49.</span><cite>Schoofs, T. et al. 575. Safety and immunogenicity of mRNA-1403, a multivalent norovirus mRNA vaccine, in Healthy Adults: Interim Results of a Phase 1/2, Randomized, Observer-Blind, Placebo-Controlled, Dose-Ranging Trial. <em>Open Forum Infect. Dis.</em><strong>12</strong>10.1093/ofid/ofae631.013 (2025).</cite>
</li>
<li id="CR50">
<span class="label">50.</span><cite>Xia, M. et al. Bioengineered norovirus S(60) nanoparticles as a multifunctional vaccine platform. <em>ACS Nano</em><strong>12</strong>, 10665–10682 (2018).
</cite> [<a href="https://doi.org/10.1021/acsnano.8b02776" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6261714/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30234973/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Xia,%20M.%20et%20al.%20Bioengineered%20norovirus%20S(60)%20nanoparticles%20as%20a%20multifunctional%20vaccine%20platform.%20ACS%20Nano12,%2010665%E2%80%9310682%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR51">
<span class="label">51.</span><cite>Devant, J. M. &amp; Hansman, G. S. Structural heterogeneity of a human norovirus vaccine candidate. <em>Virology</em><strong>553</strong>, 23–34 (2021).
</cite> [<a href="https://doi.org/10.1016/j.virol.2020.10.005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33202318/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Devant,%20J.%20M.%20&amp;%20Hansman,%20G.%20S.%20Structural%20heterogeneity%20of%20a%20human%20norovirus%20vaccine%20candidate.%20Virology553,%2023%E2%80%9334%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR52">
<span class="label">52.</span><cite>Yang, Y. et al. Structural basis of host ligand specificity change of GII porcine noroviruses from their closely related GII human noroviruses. <em>Emerg. Microbes Infect.</em><strong>8</strong>, 1642–1657 (2019).
</cite> [<a href="https://doi.org/10.1080/22221751.2019.1686335" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6853222/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31711377/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yang,%20Y.%20et%20al.%20Structural%20basis%20of%20host%20ligand%20specificity%20change%20of%20GII%20porcine%20noroviruses%20from%20their%20closely%20related%20GII%20human%20noroviruses.%20Emerg.%20Microbes%20Infect.8,%201642%E2%80%931657%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR53">
<span class="label">53.</span><cite>Hansman, G. S. et al. Crystal structures of GII.10 and GII.12 norovirus protruding domains in complex with histo-blood group antigens reveal details for a potential site of vulnerability. <em>J. Virol.</em><strong>85</strong>, 6687–6701 (2011).
</cite> [<a href="https://doi.org/10.1128/JVI.00246-11" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3126497/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21525337/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hansman,%20G.%20S.%20et%20al.%20Crystal%20structures%20of%20GII.10%20and%20GII.12%20norovirus%20protruding%20domains%20in%20complex%20with%20histo-blood%20group%20antigens%20reveal%20details%20for%20a%20potential%20site%20of%20vulnerability.%20J.%20Virol.85,%206687%E2%80%936701%20(2011)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR54">
<span class="label">54.</span><cite>Suloway, C. et al. Automated molecular microscopy: the new Leginon system. <em>J. Struct. Biol.</em><strong>151</strong>, 41–60 (2005).
</cite> [<a href="https://doi.org/10.1016/j.jsb.2005.03.010" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15890530/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Suloway,%20C.%20et%20al.%20Automated%20molecular%20microscopy:%20the%20new%20Leginon%20system.%20J.%20Struct.%20Biol.151,%2041%E2%80%9360%20(2005)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR55">
<span class="label">55.</span><cite>Cheng, A. et al. Leginon: new features and applications. <em>Protein Sci.</em><strong>30</strong>, 136–150 (2021).
</cite> [<a href="https://doi.org/10.1002/pro.3967" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7737759/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33030237/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cheng,%20A.%20et%20al.%20Leginon:%20new%20features%20and%20applications.%20Protein%20Sci.30,%20136%E2%80%93150%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR56">
<span class="label">56.</span><cite>Scheres, S. H. RELION: implementation of a Bayesian approach to cryo-EM structure determination. <em>J Struct Biol</em><strong>180</strong>, 519–530 (2012).
</cite> [<a href="https://doi.org/10.1016/j.jsb.2012.09.006" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3690530/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23000701/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Scheres,%20S.%20H.%20RELION:%20implementation%20of%20a%20Bayesian%20approach%20to%20cryo-EM%20structure%20determination.%20J%20Struct%20Biol180,%20519%E2%80%93530%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR57">
<span class="label">57.</span><cite>Rohou, A. &amp; Grigorieff, N. CTFFIND4: Fast and accurate defocus estimation from electron micrographs. <em>J Struct Biol</em><strong>192</strong>, 216–221 (2015).
</cite> [<a href="https://doi.org/10.1016/j.jsb.2015.08.008" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6760662/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26278980/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rohou,%20A.%20&amp;%20Grigorieff,%20N.%20CTFFIND4:%20Fast%20and%20accurate%20defocus%20estimation%20from%20electron%20micrographs.%20J%20Struct%20Biol192,%20216%E2%80%93221%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR58">
<span class="label">58.</span><cite>Gindy, M. E. et al. Stabilization of Ostwald ripening in low molecular weight amino lipid nanoparticles for systemic delivery of siRNA therapeutics. <em>Mol. Pharm.</em><strong>11</strong>, 4143–4153 (2014).
</cite> [<a href="https://doi.org/10.1021/mp500367k" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25317715/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gindy,%20M.%20E.%20et%20al.%20Stabilization%20of%20Ostwald%20ripening%20in%20low%20molecular%20weight%20amino%20lipid%20nanoparticles%20for%20systemic%20delivery%20of%20siRNA%20therapeutics.%20Mol.%20Pharm.11,%204143%E2%80%934153%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR59">
<span class="label">59.</span><cite>National Research Council Committee for the Update of the Guide for the Care and Use of Laboratory Animals. <em>The National Academies Collection: Reports funded by National Institutes of Health</em>. (National Academies Press (US), 2011).</cite>
</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12365219/bin/41541_2025_1254_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">8July2025Supp</a><sup> (1.3MB, pdf) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>Data were plotted and visualized using GraphPad Prism version 10.2.2. Data generated and analyzed in the current study are included in this published article and its supplementary information files. All nucleotide sequences have been deposited in GenBank (<a href="https://www.ncbi.nlm.nih.gov/genbank/" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/genbank/</a>) under accession numbers <a href="https://www.ncbi.nlm.nih.gov/nuccore/?term=PV921589:PV921598%5Bpacc%5D" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PV921589-PV921598</a>. nsEM datasets are available from the corresponding author upon request. Non-commercial critical reagents are available upon request. Readers may contact the corresponding author to request reagents or materials via a Material Transfer Agreement (MTA).</p>
<p>No custom scripts were generated in this study. Electron micrographs were analyzed using RELION (v4.0.1) for CTF correction, particle picking, and 2D classification.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from NPJ Vaccines are provided here courtesy of <strong>Nature Publishing Group</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1038/s41541-025-01254-2"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/41541_2025_Article_1254.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (2.4 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12365219/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12365219/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12365219%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365219/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12365219/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12365219/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40830611/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12365219/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40830611/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12365219/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12365219/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="xis3tWfDlxUBQXyNc7XzPIQo2lxYM9KRDzSzNuCuWaNEO0ePgitKGX0EwPm1xQbI">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
