
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            From ageing clocks to human digital twins in personalising healthcare through biological age analysis - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="152538DA8AF1EB630538DA0074643A47.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="npjdigitmed">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371080/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="NPJ Digital Medicine">
<meta name="citation_title" content="From ageing clocks to human digital twins in personalising healthcare through biological age analysis">
<meta name="citation_author" content="Murih Pusparum">
<meta name="citation_author_institution" content="Environmental Intelligence, Flemish Institute for Technological Research (VITO), Mol, Belgium">
<meta name="citation_author_institution" content="Data Science Institute, Hasselt University, Hasselt, Belgium">
<meta name="citation_author" content="Olivier Thas">
<meta name="citation_author_institution" content="Data Science Institute, Hasselt University, Hasselt, Belgium">
<meta name="citation_author_institution" content="Department of Applied Mathematics, Computer Science and Statistics, Ghent University, Ghent, Belgium">
<meta name="citation_author_institution" content="National Institute for Applied Statistics Research Australia (NIASRA), University of Wollongong, Wollongong, NSW Australia">
<meta name="citation_author" content="Stephan Beck">
<meta name="citation_author_institution" content="UCL Cancer Institute, University College London, London, UK">
<meta name="citation_author" content="Simone Ecker">
<meta name="citation_author_institution" content="UCL Cancer Institute, University College London, London, UK">
<meta name="citation_author" content="Gökhan Ertaylan">
<meta name="citation_author_institution" content="Environmental Intelligence, Flemish Institute for Technological Research (VITO), Mol, Belgium">
<meta name="citation_publication_date" content="2025 Aug 21">
<meta name="citation_volume" content="8">
<meta name="citation_firstpage" content="537">
<meta name="citation_doi" content="10.1038/s41746-025-01911-9">
<meta name="citation_pmid" content="40841468">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371080/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371080/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371080/pdf/41746_2025_Article_1911.pdf">
<meta name="description" content="Age is the most important risk factor for the majority human diseases, leading to the exploration of innovative approaches, including the development of predictors to estimate biological age (BA). These predictors offer promising insights into the ...">
<meta name="og:title" content="From ageing clocks to human digital twins in personalising healthcare through biological age analysis">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Age is the most important risk factor for the majority human diseases, leading to the exploration of innovative approaches, including the development of predictors to estimate biological age (BA). These predictors offer promising insights into the ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371080/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12371080">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1038/s41746-025-01911-9"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/41746_2025_Article_1911.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12371080%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12371080/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12371080/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371080/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-npjdigitmed.jpg" alt="NPJ Digital Medicine logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to NPJ Digital Medicine" title="Link to NPJ Digital Medicine" shape="default" href="https://www.nature.com/npjdigitalmed/" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">NPJ Digit Med</button></div>. 2025 Aug 21;8:537. doi: <a href="https://doi.org/10.1038/s41746-025-01911-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1038/s41746-025-01911-9</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22NPJ%20Digit%20Med%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22NPJ%20Digit%20Med%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22NPJ%20Digit%20Med%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22NPJ%20Digit%20Med%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>From ageing clocks to human digital twins in personalising healthcare through biological age analysis</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Pusparum%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Murih Pusparum</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Murih Pusparum</span></h3>
<div class="p">
<sup>1</sup>Environmental Intelligence, Flemish Institute for Technological Research (VITO), Mol, Belgium </div>
<div class="p">
<sup>2</sup>Data Science Institute, Hasselt University, Hasselt, Belgium </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Pusparum%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Murih Pusparum</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Thas%20O%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Olivier Thas</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Olivier Thas</span></h3>
<div class="p">
<sup>2</sup>Data Science Institute, Hasselt University, Hasselt, Belgium </div>
<div class="p">
<sup>3</sup>Department of Applied Mathematics, Computer Science and Statistics, Ghent University, Ghent, Belgium </div>
<div class="p">
<sup>4</sup>National Institute for Applied Statistics Research Australia (NIASRA), University of Wollongong, Wollongong, NSW Australia </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Thas%20O%22%5BAuthor%5D" class="usa-link"><span class="name western">Olivier Thas</span></a>
</div>
</div>
<sup>2,</sup><sup>3,</sup><sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Beck%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Stephan Beck</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Stephan Beck</span></h3>
<div class="p">
<sup>5</sup>UCL Cancer Institute, University College London, London, UK </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Beck%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Stephan Beck</span></a>
</div>
</div>
<sup>5</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ecker%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Simone Ecker</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Simone Ecker</span></h3>
<div class="p">
<sup>5</sup>UCL Cancer Institute, University College London, London, UK </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ecker%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Simone Ecker</span></a>
</div>
</div>
<sup>5,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ertaylan%20G%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Gökhan Ertaylan</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Gökhan Ertaylan</span></h3>
<div class="p">
<sup>1</sup>Environmental Intelligence, Flemish Institute for Technological Research (VITO), Mol, Belgium </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ertaylan%20G%22%5BAuthor%5D" class="usa-link"><span class="name western">Gökhan Ertaylan</span></a>
</div>
</div>
<sup>1,</sup><sup>✉,</sup><sup>#</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>Environmental Intelligence, Flemish Institute for Technological Research (VITO), Mol, Belgium </div>
<div id="Aff2">
<sup>2</sup>Data Science Institute, Hasselt University, Hasselt, Belgium </div>
<div id="Aff3">
<sup>3</sup>Department of Applied Mathematics, Computer Science and Statistics, Ghent University, Ghent, Belgium </div>
<div id="Aff4">
<sup>4</sup>National Institute for Applied Statistics Research Australia (NIASRA), University of Wollongong, Wollongong, NSW Australia </div>
<div id="Aff5">
<sup>5</sup>UCL Cancer Institute, University College London, London, UK </div>
<div class="author-notes p">
<div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div>
<div class="fn" id="_eqcntrb93pmc__">
<sup>#</sup><p class="display-inline">Contributed equally.</p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2024 Aug 30; Accepted 2025 Jul 24; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12371080  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40841468/" class="usa-link">40841468</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<p id="Par1">Age is the most important risk factor for the majority human diseases, leading to the exploration of innovative approaches, including the development of predictors to estimate biological age (BA). These predictors offer promising insights into the ageing process and age-related diseases. With real-time, multi-modal data streams and continuous patient monitoring, these BA can also inform the construction of ‘human digital twins’, quantifying how age-related changes impact health trajectories. This study highlights the significance of BA within a deeply phenotyped longitudinal cohort, using omics-based approaches alongside gold-standard clinical risk predictors. BA and health traits predictions were computed from 29 epigenetics, 4 clinical-biochemistry, 2 proteomics, and 3 metabolomics clocks. The study reveals that ageing is different between individuals but relatively stable within individuals. We suggest that BA should be considered crucial biomarkers complementing routine clinical tests. Regular updates of BA predictions within digital twin frameworks can also help guiding individualised treatment plans.</p>
<section id="kwd-group1" class="kwd-group"><p><strong>Subject terms:</strong> Personalized medicine, Epigenetics, Proteomics, Computational models</p></section></section><section id="Sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p id="Par2">Age is the most important shared risk factor for the majority of human diseases. Hence, there are strong efforts towards attenuating ageing-related disease risk via lifestyle, pharmacological, and clinical interventions to slow or reverse ageing. ‘Chronological age’ (CA), defined by the time passed since an individual’s birth, falls short of reflecting interindividual and environmental differences acting upon a biological system during this period. A critical prerequisite in the endeavour to improve healthy ageing is to quantify an individual’s ‘wellness,’ which covers not only the absence of sickness but also their resilience to future disease, general satisfaction with their health, and having sufficient energy levels for activities that enrich one’s life. While a variety of signals related to individual health and well-being can be collected, validating their contribution to clinically relevant outcomes remains an open issue. The hallmarks of ageing include genomic instability, telomere attrition, epigenetic alterations, loss of proteostasis, disabled macroautophagy, deregulated nutrient-sensing, mitochondrial dysfunction, cellular senescence, stem cell exhaustion, altered intercellular communication, chronic inflammation and dysbiosis<sup><a href="#CR1" class="usa-link" aria-describedby="CR1">1</a></sup>. Each hallmark contributes to the ageing process. The major challenge is to dissect the interconnectedness between these hallmarks and their relative contributions to ageing<sup><a href="#CR1" class="usa-link" aria-describedby="CR1">1</a></sup>.</p>
<p id="Par3">To achieve significant progress in the wellness and longevity area to improve human health span, we need to develop tools and methodologies for standard collection, harmonisation, analysis, integration, and interpretation of this information at the individual level. A variety of biological age (BA) predictors have been proposed. They share the common approach of using large cross-sectional ‘healthy’ discovery population cohorts with age as a phenotype to construct a predictive model for an organism’s age and subsequently validate it in a separate cohort. Integrating these BA predictors with digital representations of individuals, often termed ‘human digital twins,’ refers to creating a dynamic computational model that captures an individual’s (near) real-time clinical, molecular, and lifestyle data. This model is continuously updated as new data become available, enabling healthcare providers and researchers to simulate potential interventions, predict disease risk, and personalise treatment strategies.</p>
<p id="Par4">Human digital twins can incorporate data from wearable devices (e.g. heart rate, physical activity, and sleep trackers), electronic health records and multi-omics analyses (e.g. genomics, proteomics and epigenetics). By bringing these data streams together, digital twins allow for continuous modelling, simulation and prediction of health outcomes under various interventions or environmental exposures<sup><a href="#CR2" class="usa-link" aria-describedby="CR2">2</a></sup>. For instance, a digital twin can project how an individual’s BA will respond to a specific change in diet or exercise, thereby guiding preventive or therapeutic strategies in a timely manner.</p>
<p id="Par5">However, implementing human digital twins in real-world healthcare settings poses several technical challenges. These include the need for secure data integration across disparate platforms, the development of scalable computational frameworks capable of handling high-volume longitudinal data, and the establishment of standards for data interoperability and privacy. Additionally, machine learning and systems biology approaches are required to dynamically update and validate the models as individuals’ data profiles evolve over time.</p>
<p id="Par6">A critical element that enhances the precision of human digital twins is the incorporation of BA predictors. These predictors can act as <strong>quantitative anchors</strong> for the <strong>digital twin’s ‘virtual health state,’</strong>
<em>allowing clinicians to track deviations from expected trajectories more accurately</em>. By regularly updating these predictors, clinicians and researchers can detect early signs of accelerated or decelerated ageing and refine intervention strategies accordingly, improving the predictive power of the twin. This synergy between digital twin technology and validated and robust BA measures stands to accelerate preventive medicine and decision-making, ultimately improving health span and quality of life<sup><a href="#CR3" class="usa-link" aria-describedby="CR3">3</a></sup>.</p>
<p id="Par7">Based on the type of molecular data employed, these BA predictors can be classified into six categories: Telomere length (TL), epigenetic clocks, transcriptomic predictors, proteomic predictors, metabolomics-based predictors, and composite biomarker predictors. Among these, epigenetic clocks are considered to be the most accurate in predicting BA and other health phenotypes<sup><a href="#CR4" class="usa-link" aria-describedby="CR4">4</a></sup>.</p>
<p id="Par8">Telomeres are repetitive DNA sequences capping chromosomes that shorten every time a cell divides; thus, TL is a conventional marker of biological ageing across various biological domains<sup><a href="#CR5" class="usa-link" aria-describedby="CR5">5</a></sup>. TL has been associated with BA, wellness and mortality risk of an individual<sup><a href="#CR4" class="usa-link" aria-describedby="CR4">4</a>–<a href="#CR6" class="usa-link" aria-describedby="CR6">6</a></sup>. Furthermore, TL has been proposed for some specific types of cancer<sup><a href="#CR6" class="usa-link" aria-describedby="CR6">6</a></sup> and cardiovascular mortality predictions<sup><a href="#CR7" class="usa-link" aria-describedby="CR7">7</a>,<a href="#CR8" class="usa-link" aria-describedby="CR8">8</a></sup>. However, TL is hard to measure in clinical practice, and recent reports comparing TL with epigenetic clocks on non-symptomatic (or healthy) individuals found TL to be less informative<sup><a href="#CR4" class="usa-link" aria-describedby="CR4">4</a>,<a href="#CR9" class="usa-link" aria-describedby="CR9">9</a>–<a href="#CR11" class="usa-link" aria-describedby="CR11">11</a></sup>.</p>
<p id="Par9">Epigenetic clocks are molecular tools based on 5mC methylation changes to a person’s DNA over time. These modifications, which can be influenced by various factors, including environmental exposures and lifestyle choices, can change over time and, therefore, be used to predict a person’s age. The ‘epigenetic clock’ premise is to link developmental and maintenance processes to biological ageing, giving rise to a unified theory of the life course of an organism<sup><a href="#CR12" class="usa-link" aria-describedby="CR12">12</a></sup>. Epigenetic clocks are used in ageing research to identify potential interventions that could delay or reverse age-related changes and to understand the biological processes underlying ageing. They are also employed to study the relationship between<sup><a href="#CR2" class="usa-link" aria-describedby="CR2">2</a></sup> epigenetic changes and various age-related diseases and conditions, such as cancer and cardiovascular disease. Epigenetic clocks are not yet widely used in clinical practice but show promise as a way to measure biological ageing and identify interventions that may be able to improve health outcomes. There is a rapidly growing number of epigenetic clock estimators built for distinct purposes<sup><a href="#CR13" class="usa-link" aria-describedby="CR13">13</a>,<a href="#CR14" class="usa-link" aria-describedby="CR14">14</a></sup>, and their application potential, together with the other omics clocks, has also been discussed<sup><a href="#CR15" class="usa-link" aria-describedby="CR15">15</a>,<a href="#CR16" class="usa-link" aria-describedby="CR16">16</a></sup>. Proteomics, metabolomics, and multiple clinical biomarker readouts are used to predict BA, specific disease risks or phenotypes<sup><a href="#CR4" class="usa-link" aria-describedby="CR4">4</a>,<a href="#CR17" class="usa-link" aria-describedby="CR17">17</a></sup>.</p>
<p id="Par10">Many of those predictors of age and fitness have already been proposed to be used in healthcare practice, and several companies have started to offer direct-to-consumer products in this context<sup><a href="#CR18" class="usa-link" aria-describedby="CR18">18</a></sup>. Their product implementation in clinical practice requires rigorous validation and simple and clear recommendations for both the healthcare practitioner and the individual. Hence, despite the large consensus of their premise, there are significant challenges to overcome to transfer scientific health and wellness tools into clinical practice<sup><a href="#CR19" class="usa-link" aria-describedby="CR19">19</a></sup>.</p>
<p id="Par11">In this study, we aim to demonstrate the value of epigenetic clocks, proteomics, metabolomics, and multi-biomarker predictors in a unique longitudinal pilot cohort<sup><a href="#CR20" class="usa-link" aria-describedby="CR20">20</a>,<a href="#CR21" class="usa-link" aria-describedby="CR21">21</a></sup> where all required data types (clinical parameters, epigenomics, proteomics, metabolomics, along with deep phenotyping and metadata) are available across multiple time points in a 13-month period. Aligned with the Physiome<sup><a href="#CR22" class="usa-link" aria-describedby="CR22">22</a></sup> and EDITH<sup><a href="#CR23" class="usa-link" aria-describedby="CR23">23</a></sup> projects we recognise the significance of the multiscale modelling hierarchy that forms system physiology and whole-body functions. Therefore, in alignment with multi-scale models of physiology, we also aim to explore how incorporating multi-omics-based ageing clocks into human digital twins can offer a hierarchical understanding of an individual’s state—from cellular and molecular processes to organ-level function and overall clinical risk. Our objective is <em>to assess the feasibility of operationalising the concept of biological ageing in conjunction with human digital twins</em>, ultimately enhancing our ability to provide tailored and adaptive insights into individual health trajectories.</p></section><section id="Sec2"><h2 class="pmc_sec_title">Results</h2>
<section id="Sec3"><h3 class="pmc_sec_title">Study design and clocks overview</h3>
<p id="Par12">From March 2019 to March 2020, the IAM Frontier study collected monthly samples and data (via online questionnaires and wearable sensors) from 30 healthy (no diagnosed chronic diseases, no self-reported illnesses except hypertension) and highly motivated individuals. An intake interview was performed to check whether the volunteers fit the inclusion and exclusion criteria (health, age, sex balance) and to assess their motivation to join and stay within the study, as the study required monthly site visits for sample collection, continuous wearing of sensors and weekly questionnaires. All individuals were followed up with study doctor visits to inform them about their health status (from the clinical grade biomarkers), as well as provided with genetic counselling with certified personnel when necessary. The study design is illustrated in Fig. <a href="#Fig1" class="usa-link">1</a>.</p>
<figure class="fig xbox font-sm" id="Fig1"><h4 class="obj_head">Fig. 1. Study design of the IAM Frontier.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f693/12371080/e77b98e34fb2/41746_2025_1911_Fig1_HTML.jpg" loading="lazy" id="d33e381" height="336" width="630" alt="Fig. 1"></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Longitudinal comprehensive data were collected over the course of the study including monthly physiological and clinical biochemistry data (13 time points), bimonthly proteomics and metabolomics data (7 time points), and 6-monthly DNA methylation and microbiome data (3 time points). The study participants consist of 15 males and 15 females, with a (chronological) age range of 45–59 years old.</p></figcaption></figure><p id="Par13">In this study, we first explore the global pattern of the values resulting from a multitude of ageing clocks and examine their utility in predicting personal wellness, health, and BA. These clocks were computed for all the IAM Frontier individuals, utilising their clinical biochemistry and physiological data (such as blood pressure, weight and height), DNA methylation, metabolomics, and proteomics measurements. Further, we investigate how these clocks predict outcomes across different time points, leveraging the longitudinal nature of the IAM Frontier study. Individuals with interesting findings are subjected to further investigation to demonstrate the importance and the relevance of the omics clocks as personal health assistants, capable of monitoring and assisting the clinical examination and diagnosis.</p>
<p id="Par14">Figure <a href="#Fig2" class="usa-link">2</a> shows the overview of ageing clocks that can be estimated from various omics measurements. We categorise these clocks according to their utility in predicting (1) BA, (2) blood counts, (3) health traits such as smoking status, alcohol consumption and body mass index (BMI) and (4) plasma protein levels. By using DNA methylation data, 13 clocks aim to predict age, one clock to predict blood counts, 11 clocks to predict health traits, and seven clocks to predict plasma protein levels. In addition, three clinical biochemistry clocks, three metabolomics, and two proteomics clocks can also be used for predicting BA from clinical biochemistry data, NMR metabolomics, and Olink proteomics measurements. In this study, we focus on investigating the ageing clocks that are particularly useful in predicting BA.</p>
<figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2. The overview of published clinical and omics ageing clocks and predictors discussed in this article.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371080_41746_2025_1911_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f693/12371080/16c030533c23/41746_2025_1911_Fig2_HTML.jpg" loading="lazy" id="d33e396" height="373" width="707" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>The predictors are grouped based on their purpose on predicting age, health traits, and telomere length. Each node represents one predictor connected to another. In the left-hand panel, the edges represent the ratio of shared parameters, e.g. there are 26 shared parameters between the MetaboAge-vdA clock and the MetaboAge-MD clock, hence the thickest edge. The size of each node represents the total number of parameters used in each clock. The right-hand panel shows all DNA methylation predictors discussed in this study. Edges indicate the parameters used for each predictor. Due to the high number of DNA methylation sites used by the predictors, nodes and edges are shown in standard sizes, with no relation to the number of features used or shared.</p></figcaption></figure><p id="Par15">In general, in each data type, different ageing clocks use a different set of features. The features used in ageing clocks will further be referred to as variables. Figure <a href="#Fig2" class="usa-link">2</a> shows that there are some variables that are shared and utilised in more than one clock, as demonstrated by the edges between clocks (see multiple linear regression (MLR)-Levine and Klemera and Doubal method (KDM)). It is also possible that these variables are shared across different data types, for example between the MLR-Levine and the Metabo-MD, where albumin is utilised in both clocks. Albumin and several other variables are present in the clinical biochemistry and metabolomics data; they refer to the same clinical compounds but are measured by different technologies. Since our study focuses on ageing clocks that aim to predict BA, it is also worth noting that there are several omics ageing clocks that include CA in their algorithm, for example, GrimAge and KDM. GrimAge also includes sex and plasma protein levels predicted from the DNA methylation data in its calculation.</p>
<p id="Par16">Epigenomic, metabolomic and proteomic data are high dimensional. The IAM Frontier study collected the DNA methylation measurements of more than 850,000 sites per sample using Illumina’s Infinium MethylationEPIC BeadChips as well as 1068 proteins and 249 metabolites were measured in the metabolomics and proteomics data of IAM Frontier, respectively. Large numbers of variables are used in ageing clocks working with these data types. Although several epigenetic age clocks based on only a few loci have been proposed<sup><a href="#CR24" class="usa-link" aria-describedby="CR24">24</a>,<a href="#CR25" class="usa-link" aria-describedby="CR25">25</a></sup>, they tend to be less accurate. The most widely used epigenetic clocks and predictors incorporate between 71<sup><a href="#CR26" class="usa-link" aria-describedby="CR26">26</a></sup> and 1030 CpGs<sup><a href="#CR27" class="usa-link" aria-describedby="CR27">27</a></sup>. In comparison, ageing clocks based on clinical biochemistry data or other omics data (metabolomics, proteomics) require nine to 203 parameters<sup><a href="#CR28" class="usa-link" aria-describedby="CR28">28</a>–<a href="#CR31" class="usa-link" aria-describedby="CR31">31</a></sup>.</p></section><section id="Sec4"><h3 class="pmc_sec_title">Global analysis of omics clocks and predictors</h3>
<p id="Par17">We implemented 26 BA clocks in this study, see Table <a href="#Tab3" class="usa-link">3</a> in ‘Methods’. Some of the clocks were not further investigated due to showing unrealistic predictions, unavailable model coefficients (e.g. the model intercept coefficients are not published in the MetaboAge-MD clock), or unavailable variables (e.g. due to differences in the omics technology, around 80% of the required metabolites used in the Metabonomics clock were not measured in the IAM Frontier dataset)<sup><a href="#CR25" class="usa-link" aria-describedby="CR25">25</a>,<a href="#CR29" class="usa-link" aria-describedby="CR29">29</a>,<a href="#CR32" class="usa-link" aria-describedby="CR32">32</a>–<a href="#CR38" class="usa-link" aria-describedby="CR38">38</a></sup>. Different sets of variables with different sizes are involved in each clock. The included algorithms do not constitute an exhaustive list but were selected in their applicability to the IAM Frontier study with high confidence due to data availability and matching underlying assumptions.</p>
<section class="tw xbox font-sm" id="Tab3"><h4 class="obj_head">Table 3.</h4>
<div class="caption p"><p>List of applied clinical and omics ageing clocks</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th colspan="1" rowspan="1">Clock</th>
<th colspan="1" rowspan="1">Reference</th>
<th colspan="1" rowspan="1">Data type</th>
<th colspan="1" rowspan="1">Remarks</th>
</tr></thead>
<tbody>
<tr>
<td colspan="1" rowspan="1">
<sup>‡</sup>MLR-Levine<sup><a href="#CR28" class="usa-link" aria-describedby="CR28">28</a></sup>
</td>
<td rowspan="2" colspan="1">Levine, ME. 2013. Gerontol A Biol Sci Med Sci.</td>
<td rowspan="3" colspan="1">Clinical</td>
<td colspan="1" rowspan="1">Predicts chronological/ biological age</td>
</tr>
<tr>
<td colspan="1" rowspan="1">
<sup>‡</sup>PCA-Levine<sup><a href="#CR28" class="usa-link" aria-describedby="CR28">28</a></sup>
</td>
<td rowspan="2" colspan="1">Predicts chronological/ biological age</td>
</tr>
<tr>
<td colspan="1" rowspan="1">
<sup>‡</sup>KDM<sup><a href="#CR31" class="usa-link" aria-describedby="CR31">31</a></sup>
</td>
<td colspan="1" rowspan="1">Klemera, P. and Doubal, S. 2006. Mech Ageing Dev.</td>
</tr>
<tr>
<td colspan="1" rowspan="1">
<sup>‡</sup>ProtAge-Tanaka<sup><a href="#CR30" class="usa-link" aria-describedby="CR30">30</a></sup>
</td>
<td colspan="1" rowspan="1">Tanaka, T., et al. 2018. Aging Cell.</td>
<td rowspan="2" colspan="1">Proteomics</td>
<td colspan="1" rowspan="1">Predicts chronological/ biological age</td>
</tr>
<tr>
<td colspan="1" rowspan="1">
<sup>‡</sup>ProtAge-MD<sup><a href="#CR29" class="usa-link" aria-describedby="CR29">29</a></sup>
</td>
<td colspan="1" rowspan="1">Macdonald-Dunlop, E., et al. 2022. Aging.</td>
<td colspan="1" rowspan="1">Predicts chronological/ biological age</td>
</tr>
<tr>
<td colspan="1" rowspan="1">ClinAge-MD<sup><a href="#CR29" class="usa-link" aria-describedby="CR29">29</a></sup>
</td>
<td colspan="1" rowspan="1">Macdonald-Dunlop, E., et al. 2022. Aging.</td>
<td colspan="1" rowspan="1">Clinical</td>
<td colspan="1" rowspan="1">Overestimated BA (median BA 750 years), not shown</td>
</tr>
<tr>
<td colspan="1" rowspan="1">MetaboAge-MD<sup><a href="#CR29" class="usa-link" aria-describedby="CR29">29</a></sup>
</td>
<td colspan="1" rowspan="1">Macdonald-Dunlop, E., et al. 2022. Aging.</td>
<td rowspan="3" colspan="1">Metabolomics</td>
<td colspan="1" rowspan="1">Overestimated BA (median BA 600 years), not shown</td>
</tr>
<tr>
<td colspan="1" rowspan="1">MetaboAge-vdA<sup><a href="#CR32" class="usa-link" aria-describedby="CR32">32</a></sup>
</td>
<td colspan="1" rowspan="1">van den Akker, EB., et al. 2020. Circ Genom Precis Med.</td>
<td colspan="1" rowspan="1">Underestimated BA (median BA –5000 years), not shown</td>
</tr>
<tr>
<td colspan="1" rowspan="1">Metabonomics<sup><a href="#CR33" class="usa-link" aria-describedby="CR33">33</a></sup>
</td>
<td colspan="1" rowspan="1">Hertel, J., et al. 2016. Journal of Proteome Research.</td>
<td colspan="1" rowspan="1">Missing variables (only 22% vars. are available), not shown</td>
</tr>
<tr>
<td colspan="1" rowspan="1">
<sup>‡</sup>Skin &amp; blood clock<sup><a href="#CR52" class="usa-link" aria-describedby="CR52">52</a></sup>
</td>
<td colspan="1" rowspan="1">Horvath, S., et al. 2018. Aging.</td>
<td rowspan="13" colspan="1">DNA methylation</td>
<td colspan="1" rowspan="1">Predicts chronological age</td>
</tr>
<tr>
<td colspan="1" rowspan="1">
<sup>‡</sup>MethylDetectRAge<sup><a href="#CR43" class="usa-link" aria-describedby="CR43">43</a></sup>
</td>
<td colspan="1" rowspan="1">Hillary, R. and Marioni, R. Wellcome Open Res.</td>
<td colspan="1" rowspan="1">Predicts chronological age</td>
</tr>
<tr>
<td colspan="1" rowspan="1">
<sup>‡</sup>Multi-tissue clock<sup><a href="#CR53" class="usa-link" aria-describedby="CR53">53</a></sup>
</td>
<td colspan="1" rowspan="1">Horvath, S. 2013. Genome Bioloy.</td>
<td colspan="1" rowspan="1">Predicts chronological/biological age</td>
</tr>
<tr>
<td colspan="1" rowspan="1">
<sup>‡</sup>Hannum clock<sup><a href="#CR26" class="usa-link" aria-describedby="CR26">26</a></sup>
</td>
<td colspan="1" rowspan="1">Hannum, G., et al. 2013. Molecular Cell.</td>
<td colspan="1" rowspan="1">Predicts chronological/biological age</td>
</tr>
<tr>
<td colspan="1" rowspan="1">mPoA<sup><a href="#CR25" class="usa-link" aria-describedby="CR25">25</a></sup>
</td>
<td colspan="1" rowspan="1">Belsky, D., et al. 2020. eLife.</td>
<td colspan="1" rowspan="1">Predicts the pace of ageing</td>
</tr>
<tr>
<td colspan="1" rowspan="1">DunedinPACE<sup><a href="#CR56" class="usa-link" aria-describedby="CR56">56</a></sup>
</td>
<td colspan="1" rowspan="1">Belsky, D., et al. 2022. eLife.</td>
<td colspan="1" rowspan="1">Predicts the pace of ageing</td>
</tr>
<tr>
<td colspan="1" rowspan="1">
<sup>‡</sup>PhenoAge<sup><a href="#CR54" class="usa-link" aria-describedby="CR54">54</a></sup>
</td>
<td colspan="1" rowspan="1">Levine, M., et al. 2018. Aging.</td>
<td colspan="1" rowspan="1">Predicts health- and lifespan</td>
</tr>
<tr>
<td colspan="1" rowspan="1">
<sup>‡</sup>GrimAge<sup><a href="#CR27" class="usa-link" aria-describedby="CR27">27</a></sup>
</td>
<td colspan="1" rowspan="1">Lu, A., et al. 2019. Aging.</td>
<td colspan="1" rowspan="1">Predicts health- and lifespan</td>
</tr>
<tr>
<td colspan="1" rowspan="1">
<sup>‡</sup>GrimAge2<sup><a href="#CR55" class="usa-link" aria-describedby="CR55">55</a></sup>
</td>
<td colspan="1" rowspan="1">Lu, A., et al. 2022. Aging.</td>
<td colspan="1" rowspan="1">Predicts health- and lifespan</td>
</tr>
<tr>
<td colspan="1" rowspan="1">Metaclock<sup><a href="#CR34" class="usa-link" aria-describedby="CR34">34</a></sup>
</td>
<td colspan="1" rowspan="1">Liu, Z., et al. 2020. Aging Cell.</td>
<td colspan="1" rowspan="1">Predicts mortality</td>
</tr>
<tr>
<td colspan="1" rowspan="1">EpiTOC<sup><a href="#CR35" class="usa-link" aria-describedby="CR35">35</a></sup>
</td>
<td colspan="1" rowspan="1">Yang, Z., et al. 2016. Genome Biology.</td>
<td colspan="1" rowspan="1">Predicts mitotic age</td>
</tr>
<tr>
<td colspan="1" rowspan="1">EpiTOC2<sup><a href="#CR36" class="usa-link" aria-describedby="CR36">36</a></sup>
</td>
<td colspan="1" rowspan="1">Teschendorff, A.E. 2020. Genome Medicine</td>
<td colspan="1" rowspan="1">Predicts mitotic age</td>
</tr>
<tr>
<td colspan="1" rowspan="1">MiAge<sup><a href="#CR37" class="usa-link" aria-describedby="CR37">37</a></sup>
</td>
<td colspan="1" rowspan="1">Youn, A., and Wang, S. 2018. Epigenetics.</td>
<td colspan="1" rowspan="1">Predicts mitotic age</td>
</tr>
<tr>
<td colspan="1" rowspan="1">DNAmTL<sup><a href="#CR38" class="usa-link" aria-describedby="CR38">38</a></sup>
</td>
<td colspan="1" rowspan="1">Lu, A., et al. 2019. Aging.</td>
<td rowspan="4" colspan="1">DNA methylation</td>
<td colspan="1" rowspan="1">Predicts telomere length</td>
</tr>
<tr>
<td colspan="1" rowspan="1">
<sup>‡</sup>CausAge<sup><a href="#CR61" class="usa-link" aria-describedby="CR61">61</a></sup>
</td>
<td colspan="1" rowspan="1">Ying, K, et al. 2024. Nature Aging.</td>
<td colspan="1" rowspan="1">Predicts chronological/ biological age</td>
</tr>
<tr>
<td colspan="1" rowspan="1">DamAge<sup><a href="#CR61" class="usa-link" aria-describedby="CR61">61</a></sup>
</td>
<td colspan="1" rowspan="1">Ying, K, et al. 2024. Nature Aging.</td>
<td colspan="1" rowspan="1">Predicts chronological/ biological age</td>
</tr>
<tr>
<td colspan="1" rowspan="1">AdaptAge<sup><a href="#CR61" class="usa-link" aria-describedby="CR61">61</a></sup>
</td>
<td colspan="1" rowspan="1">Ying, K, et al. 2024. Nature Aging.</td>
<td colspan="1" rowspan="1">Predicts chronological/ biological age</td>
</tr>
<tr>
<td colspan="1" rowspan="1">GOLD BioAge<sup><a href="#CR71" class="usa-link" aria-describedby="CR71">71</a></sup>
</td>
<td rowspan="3" colspan="1">Hao, et al. 2025. Adv. Sci.</td>
<td colspan="1" rowspan="1">Clinical</td>
<td colspan="1" rowspan="1">Predicts morbidity &amp; mortality</td>
</tr>
<tr>
<td colspan="1" rowspan="1">GOLD ProtAge<sup><a href="#CR71" class="usa-link" aria-describedby="CR71">71</a></sup>
</td>
<td colspan="1" rowspan="1">Metabolomics</td>
<td colspan="1" rowspan="1">Predicts morbidity &amp; mortality</td>
</tr>
<tr>
<td colspan="1" rowspan="1">GOLD MetAge<sup><a href="#CR71" class="usa-link" aria-describedby="CR71">71</a></sup>
</td>
<td colspan="1" rowspan="1">Proteomics</td>
<td colspan="1" rowspan="1">Predicts morbidity &amp; mortality</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="_fn_p55"><p><sup>‡</sup>Rows with double dagger symbol (‡) refer to clocks that are further investigated.</p></div></div></section><p id="Par18">Although the study’s main objective is focused on BA prediction, CA is still incorporated in some analyses. While CA may not provide the most robust description of human ageing, it offers the most convenient way to calculate age and is perceived as a standard measure of ageing. Stratification by CA group is also often done, for example in clinical reference intervals<sup><a href="#CR39" class="usa-link" aria-describedby="CR39">39</a>,<a href="#CR40" class="usa-link" aria-describedby="CR40">40</a></sup>. For this reason, we believe it is valid to compare the predictions of BA with the subjects’ CA; see the Pearson correlation coefficients in Table <a href="#Tab1" class="usa-link">1</a>. The majority of the clocks shown in Table <a href="#Tab1" class="usa-link">1</a> are significantly correlated with CA; MethylDetectR and the Skin &amp; Blood clock give the highest correlation coefficients, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq1"><math id="d33e466"><mi>R</mi></math></span>= 0.90 and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq2"><math id="d33e475"><mi>R</mi></math></span>= 0.87, respectively. This is not unexpected, as both clocks were developed to predict CA. The kernel densities of the predicted BA as well as CA are shown in Fig. <a href="#Fig3" class="usa-link">3</a>. The predicted Skin &amp; Blood clock age gives a similar kernel density as the CA, both in the position as well as the shape. From this perspective, we may suggest this clock as the best tool to measure individuals’ chronological ageing. However, the correlation between CA and predicted age among all measured biological clocks is the highest for MethylDetectRAge, another CA predictor (Pearson’s <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq3"><math id="d33e488"><mi>R</mi></math></span>= 0.91), although its predictions appear to be slightly shifted towards older ages in our dataset. GrimAge, with <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq4"><math id="d33e497"><mi>R</mi></math></span>= 0.85, gives a position of the kernel similar to the CA, but with a different shape. GrimAge aims to predict lifespan and healthspan, and as a remark, it also incorporates CA in its calculation, unlike the other clocks, which are solely based on DNA methylation markers, other omics, or clinical measurements. The same remark also applies to the KDM-Levine clock, with <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq5"><math id="d33e506"><mi>R</mi></math></span>= 0.76 and an identical position of the kernel density as compared to the CA.</p>
<section class="tw xbox font-sm" id="Tab1"><h4 class="obj_head">Table 1.</h4>
<div class="caption p"><p>Correlation between predicted age and chronological age</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th colspan="1" rowspan="1">Predictor</th>
<th colspan="1" rowspan="1">Corr. coefficent</th>
<th colspan="1" rowspan="1">
<em>p</em>-value</th>
</tr></thead>
<tbody>
<tr>
<td colspan="1" rowspan="1">MethylDetectRAge</td>
<td colspan="1" rowspan="1">0.90</td>
<td colspan="1" rowspan="1">2.0E-30*</td>
</tr>
<tr>
<td colspan="1" rowspan="1">Skin &amp; blood clock</td>
<td colspan="1" rowspan="1">0.87</td>
<td colspan="1" rowspan="1">3.5E-26*</td>
</tr>
<tr>
<td colspan="1" rowspan="1">GrimAge</td>
<td colspan="1" rowspan="1">0.85</td>
<td colspan="1" rowspan="1">1.1E-23*</td>
</tr>
<tr>
<td colspan="1" rowspan="1">GrimAge2</td>
<td colspan="1" rowspan="1">0.84</td>
<td colspan="1" rowspan="1">2.7E-22*</td>
</tr>
<tr>
<td colspan="1" rowspan="1">Multi-tissue clock</td>
<td colspan="1" rowspan="1">0.81</td>
<td colspan="1" rowspan="1">1.6E-19*</td>
</tr>
<tr>
<td colspan="1" rowspan="1">KDM</td>
<td colspan="1" rowspan="1">0.76</td>
<td colspan="1" rowspan="1">2.2E-16*</td>
</tr>
<tr>
<td colspan="1" rowspan="1">ProtAge-MD</td>
<td colspan="1" rowspan="1">0.67</td>
<td colspan="1" rowspan="1">2.2E-16*</td>
</tr>
<tr>
<td colspan="1" rowspan="1">PhenoAge</td>
<td colspan="1" rowspan="1">0.66</td>
<td colspan="1" rowspan="1">3.8E-11*</td>
</tr>
<tr>
<td colspan="1" rowspan="1">Hannum clock</td>
<td colspan="1" rowspan="1">0.65</td>
<td colspan="1" rowspan="1">4.8E-11*</td>
</tr>
<tr>
<td colspan="1" rowspan="1">ProtAge-Tanaka</td>
<td colspan="1" rowspan="1">0.60</td>
<td colspan="1" rowspan="1">2.2E-16*</td>
</tr>
<tr>
<td colspan="1" rowspan="1">MetaClock</td>
<td colspan="1" rowspan="1">0.56</td>
<td colspan="1" rowspan="1">8.7E-08*</td>
</tr>
<tr>
<td colspan="1" rowspan="1">PCA-Levine</td>
<td colspan="1" rowspan="1">0.48</td>
<td colspan="1" rowspan="1">2.2E-16*</td>
</tr>
<tr>
<td colspan="1" rowspan="1">MLR-Levine</td>
<td colspan="1" rowspan="1">0.40</td>
<td colspan="1" rowspan="1">1.4E-15*</td>
</tr>
<tr>
<td colspan="1" rowspan="1">ClinAge-MD</td>
<td colspan="1" rowspan="1">0.37</td>
<td colspan="1" rowspan="1">1.2E-13*</td>
</tr>
<tr>
<td colspan="1" rowspan="1">EpiTOC</td>
<td colspan="1" rowspan="1">0.25</td>
<td colspan="1" rowspan="1">2.4E-02*</td>
</tr>
<tr>
<td colspan="1" rowspan="1">MiAge</td>
<td colspan="1" rowspan="1">0.20</td>
<td colspan="1" rowspan="1">7.6E-02</td>
</tr>
<tr>
<td colspan="1" rowspan="1">MetaboAge-vdA</td>
<td colspan="1" rowspan="1">0.18</td>
<td colspan="1" rowspan="1">8.6E-03*</td>
</tr>
<tr>
<td colspan="1" rowspan="1">mPoA</td>
<td colspan="1" rowspan="1">0.18</td>
<td colspan="1" rowspan="1">1.1E-01</td>
</tr>
<tr>
<td colspan="1" rowspan="1">DunedinPACE</td>
<td colspan="1" rowspan="1">0.15</td>
<td colspan="1" rowspan="1">1.7E-01</td>
</tr>
<tr>
<td colspan="1" rowspan="1">MetaboAge-MD</td>
<td colspan="1" rowspan="1">0.08</td>
<td colspan="1" rowspan="1">2.6E-01</td>
</tr>
<tr>
<td colspan="1" rowspan="1">EpiTOC2</td>
<td colspan="1" rowspan="1">0.04</td>
<td colspan="1" rowspan="1">7.4E-01</td>
</tr>
<tr>
<td colspan="1" rowspan="1">CausAge</td>
<td colspan="1" rowspan="1">0.69</td>
<td colspan="1" rowspan="1">1.8E-12*</td>
</tr>
<tr>
<td colspan="1" rowspan="1">DamAge</td>
<td colspan="1" rowspan="1">0.62</td>
<td colspan="1" rowspan="1">1.3E-09*</td>
</tr>
<tr>
<td colspan="1" rowspan="1">AdaptAge</td>
<td colspan="1" rowspan="1">0.49</td>
<td colspan="1" rowspan="1">3.6E-06*</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="_fn_p23"><p>*Significant correlations are displayed with an asterisk (at <em>p</em> &lt; 0.05). GrimAge and KDM incorporate chronological age in their calculation.</p></div></div></section><figure class="fig xbox font-sm" id="Fig3"><h4 class="obj_head">Fig. 3. Density plots and correlations of BA predictions.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371080_41746_2025_1911_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f693/12371080/d713aec6c6e4/41746_2025_1911_Fig3_HTML.jpg" loading="lazy" id="d33e517" height="453" width="686" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>MethylDetectRAge and Skin &amp; blood clock have the highest correlations with the chronological age (CA) (left panel); the Skin &amp; blood clock predicted age gives the most similar kernel density as CA, both in position and shape (right panel).</p></figcaption></figure><p id="Par19">We performed different data exploration approaches to compare the BA predictions of each individual who participated in the IAM Frontier study. At each BA predictor, we observe smaller within-person predictions at different time points (focusing on <em>inter-individual variability</em>) than the between-person predictions at the same time point (focusing on <em>intra-individual variability</em>). The small inter-individual variabilities, which imply to large intra-class correlation (ICC) values, reflected by high ICC values, indicate that predictions for the same individual are more consistent over time than those observed between peers. This pattern aligns with the characteristics of standard clinical biomarkers measured longitudinally.</p>
<p id="Par20">Our results show ICC values ranging from 0.70 to 0.94, while the ratios of intra- to inter-individual variability (<em>Var_ratio</em>) span from 0.06 to 0.43, as illustrated in Fig. <a href="#Fig4" class="usa-link">4b</a>. In contrast to ICC, a higher <em>Var_ratio</em> suggests greater divergence in within-person predictions. Consistent patterns were also observed in the unsupervised analyses (Fig. <a href="#Fig4" class="usa-link">4a</a>), where BA predictions from the same individual consistently clustered together, reinforcing the notion that within-person predictions are more similar than those between individuals. When we examine individual physical condition, clocks with lower ICC, such as PCA-Levine and ProtAge-Tanaka appear more sensitive to fluctuations in self-reported health status, as measured through weekly questionnaires on physical complaints. This is further supported by the pairwise correlation analysis between health trends and the average BA predictions from each clock (Supplementary Fig. <a href="#MOESM1" class="usa-link">5b</a>). While these clocks show less within-person stability, they may be better suited for capturing short-term biological changes associated with acute physiological changes. These findings support an analytical approach that emphasises inter-individual differences while also exploring the variability within individuals across time. For instance, Fig. <a href="#Fig4" class="usa-link">4c</a> presents age acceleration—defined as the difference between predicted BA and CA—at time points two, eight, and 13, which are common across all omics and clinical data. Distinct predictions were evident across different clocks, with observable fluctuations across time points.</p>
<figure class="fig xbox font-sm" id="Fig4"><h4 class="obj_head">Fig. 4. Longitudinal exploration of BA predictions in three different omics modalities.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371080_41746_2025_1911_Fig4_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f693/12371080/f0a350c39cbb/41746_2025_1911_Fig4_HTML.jpg" loading="lazy" id="d33e824" height="765" width="799" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Unsupervised analysis of BA predictions (PCA and cluster analysis) for all subjects in three shared time points, <strong>b</strong> ICC and <em>Var_ratio</em> of each BA clocks; large ICC and small <em>Var_ratio</em> indicate high within-person similarity, <strong>c</strong> Age acceleration of all IAM Frontier participants in different clocks; epigenetics clocks with <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq6"><math id="d33e770"><mrow><msub><mrow><mover accent="true"><mrow><mi>S</mi></mrow><mo>®</mo></mover></mrow><mrow><mi>E</mi></mrow></msub><mo>=</mo><mn>0.164</mn></mrow></math></span>: multi-tissue clock (pink), skin &amp; blood clock (yellow), henoAge (green), GrimAge2 (orange), MethylDetectR (steel blue, top), CausAge(light blue, bottom) and Hannum clock (black). Clinical clocks with <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq7"><math id="d33e790"><mrow><msub><mrow><mover accent="true"><mrow><mi>S</mi></mrow><mo>®</mo></mover></mrow><mrow><mi>C</mi></mrow></msub><mo>=</mo><mn>0.118</mn></mrow></math></span>: MLR-Levine clock (brown), PCA-Levine clock (light pink), and KDM clock (blue). Proteomics clocks with <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq8"><math id="d33e809"><mrow><msub><mrow><mover accent="true"><mrow><mi>S</mi></mrow><mo>®</mo></mover></mrow><mrow><mi>P</mi></mrow></msub><mo>=</mo><mn>0.116</mn></mrow></math></span>: ProtAge-Tanaka (grey) and ProtAge-MD (tosca).</p></figcaption></figure><p id="Par21">Within the same clock, we also observe that the between-individual fluctuations are larger than the within-individual’. We calculated the stability index (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq9"><math id="d33e832"><mrow><msub><mrow><mover accent="true"><mrow><mi>S</mi></mrow><mo>®</mo></mover></mrow><mrow><mi>k</mi></mrow></msub><mo>,</mo></mrow></math></span> for each clock category <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq10"><math id="d33e850"><mi>k</mi></math></span>) based on these fluctuations,</p>
<table class="disp-formula p" id="Equ1"><tr>
<td class="formula"><math id="d33e860" display="block"><mrow><msub><mrow><mover accent="true"><mrow><mi>S</mi></mrow><mo>®</mo></mover></mrow><mrow><mi>k</mi></mrow></msub><mo>=</mo><mfrac><mrow><mn>1</mn></mrow><mrow><mi>N</mi></mrow></mfrac><munderover><mo>∑</mo><mrow><mi>i</mi><mo>=</mo><mn>1</mn></mrow><mrow><mi>N</mi></mrow></munderover><mrow><msub><mrow><mover accent="true"><mrow><mi>S</mi></mrow><mo>~</mo></mover></mrow><mrow><mi>i</mi></mrow></msub></mrow><mo>,</mo><mi mathvariant="normal">with</mi><msub><mrow><mover accent="true"><mrow><mi>S</mi></mrow><mo>~</mo></mover></mrow><mrow><mi>i</mi></mrow></msub><mo>=</mo><mfrac><mrow><mn>1</mn></mrow><mrow><mi mathvariant="italic">mean</mi><mo>(</mo><msubsup><mrow><mi>d</mi></mrow><mrow><mn>12</mn></mrow><mrow><mi>i</mi></mrow></msubsup><mo>,</mo><mspace width="0.25em"></mspace><msubsup><mrow><mi>d</mi></mrow><mrow><mn>13</mn></mrow><mrow><mi>i</mi></mrow></msubsup><mo>,</mo><mspace width="0.25em"></mspace><mo>…</mo><mo>,</mo><mspace width="0.25em"></mspace><msubsup><mrow><mi>d</mi></mrow><mrow><mi>n</mi><mo>−</mo><mn>1</mn><mo>,</mo><mi>n</mi></mrow><mrow><mi>i</mi></mrow></msubsup><mo>)</mo></mrow></mfrac></mrow></math></td>
<td class="label">1</td>
</tr></table>
<p>and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq11"><math id="d33e953"><mrow><msubsup><mrow><mi>d</mi></mrow><mrow><mn>12</mn></mrow><mrow><mi>i</mi></mrow></msubsup><mo>,</mo><mspace width="0.25em"></mspace><msubsup><mrow><mi>d</mi></mrow><mrow><mn>13</mn></mrow><mrow><mi>i</mi></mrow></msubsup><mo>,</mo><mspace width="0.25em"></mspace><mo>…</mo><mo>,</mo><mspace width="0.25em"></mspace><msubsup><mrow><mi>d</mi></mrow><mrow><mi>n</mi><mo>−</mo><mn>1</mn><mo>,</mo><mi mathvariant="italic"> n</mi></mrow><mrow><mi>i</mi></mrow></msubsup></mrow></math></span> are the pairwise differences of BA from different time points at each subject <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq12"><math id="d33e995"><mi>i</mi></math></span>. For easier interpretations, we normalised the score to 0–1 range with a larger index indicating less fluctuations and hence, more stable predictions. The results reveal that epigenetic clocks provide the most stable predictions, followed by clinical data and proteomics. This suggests that the rate of change in each layer affects the reliability of BA predictions over time. Therefore, DNA methylation data should be used in conjunction with more recent clinical and proteomics data to produce stable BA predictions. The individual stability index, calculated for each clock category, further supports the finding that epigenetic clocks consistently yield more stable within-individual predictions (see Supplementary Fig. <a href="#MOESM1" class="usa-link">6</a>). Clinical and proteomic measures appear more sensitive to short-term influences such as lifestyle, medication, and other external factors. The age acceleration predictions of GrimAge, the Skin &amp; Blood clock, KDM-Levine, PCA-Levine, and ProtAge-Tanaka are close to zero, i.e. their predicted BA values are close to the corresponding CAs. Referring to the estimated kernel density in Fig. <a href="#Fig3" class="usa-link">3</a>, the predicted values of these clocks are similar to the CA with reference to the location.</p></section><section id="Sec5"><h3 class="pmc_sec_title">Individual analysis of omics clocks and predictors</h3>
<p id="Par22">We continue the analyses by investigating the BA and health traits predictions in all IAM Frontier participants. Due to the small sample nature of the IAM Frontier study, we are able to examine the individual predictions resulting from all clocks. Figure <a href="#Fig5" class="usa-link">5a</a> shows the ordered heatmaps of the predicted BAs as well as the health trait predictions in all individuals at time points two, eight, and 13. These were selected because the results of all clocks are available at these time points. The MAD thresholds were computed and were used to give the colour annotations. Therefore, only deviating predictions appear coloured in the figure. We can observe distinct patterns for ID06, ID08 and ID27, detailed below. In Fig. <a href="#Fig5" class="usa-link">5c</a>, we show the clinical laboratory profiles of these individuals.</p>
<figure class="fig xbox font-sm" id="Fig5"><h4 class="obj_head">Fig. 5. Individual analysis of BA predictions in all available time points.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371080_41746_2025_1911_Fig5_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f693/12371080/bb99b46e9363/41746_2025_1911_Fig5_HTML.jpg" loading="lazy" id="d33e1032" height="1245" width="743" alt="Fig. 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Reduced ordered heatmap based on MAD thresholds. <strong>b</strong> Age acceleration of ID06, ID08, and ID27. Prediction values that are significantly different from the rest of the cohort are marked with full circles. <strong>c</strong> Clinical and blood cell profiles of ID06, ID08, and ID27. Measurements that are outside the cohort reference intervals are marked with full circles. The blood cell counts are shown in percentages.</p></figcaption></figure><p id="Par23">ID06 shows very high smoking predictions. ID06 is a current smoker, indeed, and hence has a higher risk of developing cardiovascular diseases (CVD) and diabetes (Supplementary Fig. <a href="#MOESM1" class="usa-link">3</a> of CVD and diabetes disease risk scores). From the individual plots in Fig. <a href="#Fig5" class="usa-link">5b</a>, we also observe that the BA predictions of ID06 increase over time for some of the ageing clocks, such as in the Skin &amp; Blood clock and PhenoAge. This individual was also biologically older at all time points according to the KDM and ProtAge-Tanaka clocks. However, the clinical laboratory profiles of ID06 show an increasing trend of the lipid measurements, in line with this individual’s increased risk of CVD and diabetes. Interestingly, ID06’s B-cell percentages are higher than the normal range, and the CD4 T-cell and lymphocyte counts are low. This could show that this individual might also have problems with antibodies and the immune system. In the questionnaires, ID06 did not report major health burdens except that this individual felt having low energy, tense muscles, and constant fatigue, with the latter two commonly reported among the study participants.</p>
<p id="Par24">ID08 appears as the only individual with predicted shortened telomeres across both measured time points based on the DNAmTL clock, as seen in Fig. <a href="#Fig5" class="usa-link">5b</a>. This result may indicate premature ageing. In addition, in almost all the applied epigenetic clocks, this individual was also predicted to be epigenetically older at the second time point compared to the first (see Fig. <a href="#Fig5" class="usa-link">5b</a>). The lipid (cholesterol) profiles of ID08 (see Fig. <a href="#Fig5" class="usa-link">5c</a>) show an alarming status; the measurements are out-of-normal range at almost all time points. Other clinical parameters are still within the normal limits, although the percentages of CD8 T-cell and lymphocyte are high compared to the peers. Based on the weekly questionnaires, during the course of this study, this individual experienced constant fatigue, diarrhoea, numbness, indigestion, sleeping problems, stuffy nose, and blurry vision, see Supplementary Fig. <a href="#MOESM1" class="usa-link">5a</a>. Although ID08 does not appear to report all of these health burdens at a significantly higher level than the other study participants, together with the rapid epigenetic ageing and high lipid profiles observed, they might provide additional support to the premature ageing condition predicted by the DNAmTL clock.</p>
<p id="Par25">ID27 appears to be different in our analyses in many ways, although most of the epigenetic, proteomic and clinical ageing clocks predict this individual to be in a healthy state—healthier and biologically younger than the peers. However, ID27 shows very strong outlier BA predictions corresponding to the Hannum clock as well as very different smoking and health trait predictions. It is the only subject with a completely distinct Hannum age acceleration compared to the peers at all time points, epigenetically younger at the second time point as compared to the first across all clocks, and with conflicting smoking predictions (this individual was not a current smoker). The BA predictions of the clinical and proteomics clocks also show a general decreasing trend over time, where ID27 was biologically younger than chronologically, according to some of those clocks. In addition, from the DNA methylation and proteomics analyses, this subject is discovered to be far outside the main cluster in the corresponding PCA plots (Supplementary Fig. <a href="#MOESM1" class="usa-link">4a, b</a>). When looking at DNA methylation predicted blood cell composition, we observe a strongly increased proportion of B cell (Supplementary Fig. <a href="#MOESM1" class="usa-link">7</a>). The finding is confirmed by actual blood counts showing the same abnormalities, clearly indicating a haematological problem. This explains why ID27’s age predictions appear as extreme outliers according to Hannum’s clock, as it is sensitive to blood composition changes, and may also explain the conflicting and unexpected outlier predictions obtained for several health traits such as smoking and alcohol intake, among others. Throughout the study period, ID27 reported constant medium backpain, a slight headache and blurry vision.</p></section></section><section id="Sec6"><h2 class="pmc_sec_title">Discussion</h2>
<p id="Par26">Age stands as the principal risk determinant for ailments, impairments, and diseases. The pursuit of mitigating age-related illnesses and extending the healthful years of life has led to the innovative idea of directly targeting the ageing process to restore physiological functionality. Achieving this ambitious goal requires the precise assessment of BA and the pace of ageing at the molecular level. Building on the latest breakthroughs in high-throughput omics technologies, a novel suite of tools has emerged for the quantitative analysis of biological ageing. By leveraging data from various domains—such as epigenomics, proteomics, and metabolomics—and employing machine learning techniques, ‘biological ageing clocks’ have been constructed. These clocks have proven their ability to pinpoint potential biomarkers of biological ageing, offering unprecedented insights into the ageing process and laying the groundwork for anchoring digital twin models to health trajectories.</p>
<p id="Par27">A recent review paints today’s landscape of BA prediction algorithms using omics technologies<sup><a href="#CR41" class="usa-link" aria-describedby="CR41">41</a></sup>. These methods are claimed to better represent the biological state of an organism than the CA. Ageing clocks for BA are also starting to appear as commercial products, serving as a window into the personal health status. In this manuscript, we use a unique deep-phenotyping dataset that allows us to perform and analyse multiple BA predictors using different data types side by side. The longitudinal nature of the data enables us to study the individuals’ fluctuations between time points, exploring their changes in BA predictions over time. We optimise our analyses using the extensive set of technical replicates generated to estimate technical variation in our dataset and design our analyses correspondingly (see ‘Methods’ and Supplementary Material Section <a href="#MOESM1" class="usa-link">1</a>).</p>
<p id="Par28">The BA analyses in the IAM Frontier data shows that <strong>BA predictions can fluctuate over time</strong>; both an increase and a decrease might happen depending on the biological conditions <strong>rather than monotonically increase by time, as in CA</strong>. Repeated measurements over time are highly valuable, as outliers can also be caused by normal biological fluctuations. The stability index, which quantifies the consistency of BA predictions across different time points, also indicates that <strong>ageing likely manifests at different rates across various omics layers</strong>. As represented within the multi-dimensional modelling of digital twins, ageing is a multi-dimensional—or multilayer—process. In the physiome hierarchical framework, organs are mapped from cells to functional tissue units (FTUs) and then to the organ level (a similar approach is taken in EDITH and other digital twin frameworks)<sup><a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>,<a href="#CR42" class="usa-link" aria-describedby="CR42">42</a></sup>. Extending this multi-level structure into <strong>BA</strong> enables real-time simulation of age-related changes across various biological scales. In the context of ageing, <em>BA predictions can serve as quantifiable indicators for analysing deviations in ageing at the cellular or even FTU levels, thereby refining the digital twin’s ability to capture early signs of physiological decline</em>. Understanding stability is crucial for determining the appropriate periodicity of data collection in each omics layer to ensure accurate, multi-scale integration into digital twin models.</p>
<p id="Par29">In our individual analysis, most BA predictions were ‘<strong>stable</strong>’, that is, within the expected range of variability across the available time points. However, there were individuals whose predictions do change more than expected over time, and where this is the case, they are often consistent across multiple BA predictors and time points. A comprehensive exploration of the BA prediction differences between individuals of our study leads to <strong>three individuals with distinct BA predictions</strong> as compared to their peers.</p>
<p id="Par30">The BA predictions of <em>ID06</em>, <em>ID08</em>, and <em>ID27</em> provide a compelling illustration of how multi-omics data can not only support standard laboratory findings but also pinpoint subtle physiological deviations well before overt clinical symptoms emerge. In a digital twin context, these detailed omics layers and BA predictors could be integrated into a computational model that flags abnormal trajectories in real time, prompting earlier diagnostic follow-up. For example, <strong><em>ID08</em></strong><strong>’s consistently short telomeres and abnormal lipid profiles could trigger predictive simulations about cardiovascular risks</strong>, while <strong><em>ID27</em></strong><strong>’s epigenetic deviations could inform more targeted immunological assessments</strong>. These instances highlight the promise of digital twin models to capture and interpret diverse biomarker changes as they unfold.</p>
<p id="Par31">These strong alterations may be what impacted some of the reported epigenetic health trait predictions. Of note, according to the developers of those health trait predictors, they should currently only be used at a population level and are not yet supposed to provide reliable predictions at the individual level. Training data from larger-scale cohorts, <em>including diseased individuals</em>, will be required to refine these additional health trait predictors and enable their clinical use<sup><a href="#CR43" class="usa-link" aria-describedby="CR43">43</a></sup>. Irrespectively, the clearly abnormal epigenetic measurements and predictions of <em>ID27</em> would undoubtedly also have led to further clinical tests if detected in a wellness- or preventative healthcare setting, revealing the ongoing but previously undetected pathological process.</p>
<p id="Par32">Our findings demonstrate the value and potential of epigenetic predictors and BA estimators, particularly for risk assessment and early detection. Several newer versions of these epigenetic predictors, introduced and integrated in this study, are at least on par with their earlier models, while still offering valuable new benefits. Our integrative analysis with self-reported health complaints further shows important distinctions between clock types. Clinical and proteomics-based clocks, while exhibited slightly lower overall stability (with the lowest ICC of 0.76), more accurately captured short-term fluctuations in health status and physical complaints compared to epigenetic clocks. A greater number of health complaints showed significant associations with changes in these clocks’ BA predictions, suggesting they may be particularly sensitive to acute, short-term physiological states. In contrast, epigenetic clocks appeared considerably more stable within individuals but were less reflective of week-to-week physical variations, possibly reflecting more stable, long-term biological aging processes.</p>
<p id="Par33">Taken together, these BA clocks can finally reveal unexpected deviations and inform about the probability of having or developing a disease, for example shown here by individual <em>ID27</em>. Moreover, they can serve as the cornerstone in anchoring the implementation of human digital twins that model and simulate biological processes in real-time.</p>
<p id="Par34">Recent advancements demonstrate the growing potential of digital twins in capturing the complex dynamics of human ageing<sup><a href="#CR44" class="usa-link" aria-describedby="CR44">44</a></sup>. Incorporating BA clocks into digital twins demands the creation of ‘<strong>biological-age-sensitive</strong>’ modelling paradigms in which normal age-related changes are distinguished from genuine disease progression. As illustrated by our cohort findings, clocks derived from epigenetics, proteomics, or metabolomics can parameterise tissue- and organ-level simulations, improving model fidelity. By regularly updating these BA-based parameters, digital twins can track the natural ageing continuum while highlighting deviations that may signal the onset of pathology. Hence, considering that BA clocks have the potential to reflect the age/health status of the tissue or organ from the perspective of the modality (methylation, proteomics), they have the potential to parameterise the tissue or organ level models better.</p>
<p id="Par35">In the future, when epigenetics and multi-omics data of patients are routinely captured, researchers and clinicians could integrate these data into the digital twin’s analytics pipeline, calibrating BA models using validated clinical outcomes, and finally performing iterative updates as new biomarker measurements become available. This would us to optimise personalised medical treatments, and refine preventive healthcare strategies. However, several challenges remain before the seamless deployment of biological-age-sensitive human digital twins in clinical practice, including (i) data standardisation and interoperability across disparate omics platforms, (ii) ensuring model accuracy and generalisability given high-dimensional datasets, (iii) managing the computational burden of real-time or near-real-time updates, and (iv) establishing robust data privacy, security, and regulatory frameworks. Addressing these technical and ethical hurdles will be essential for enabling BA-driven digital twins to evolve from conceptual prototypes into mainstream clinical tools.</p>
<p id="Par36">Currently, the extensive and complex preprocessing procedures required to obtain high quality results from omics data are still hindering many of the corresponding predictors from developing their full clinical potential, particularly at the individual level. However, with the field growing and maturing, we argue that BA predictions should be considered as crucial biomarkers that can anchor digital twin models and well-complement routine medical tests and CA.</p>
<p id="Par37">Omics data have already begun to enter clinical practice<sup><a href="#CR45" class="usa-link" aria-describedby="CR45">45</a></sup>, making omics-based BA predictions feasible. However, despite their high-dimensionality, omics datasets often suffer from limited sample sizes. In this IAM Frontier study, for instance, the small sample size (<em>n</em> = 30) presents challenges for generalising the results and reduces statistical power. To enhance the reliability of the analyses, several measures were implemented, including the use of the median absolute deviation (MAD) to identify and mitigate outliers. This approach is more robust and less influenced by extreme values. Additionally, downstream analyses were not solely reliant on aggregate results, which can be skewed by outliers, but also incorporated subject-level analyses. This ensures that potential biological findings are not overlooked or diluted, as can happen in population-level analyses. By adopting this dual approach, we mitigate some of the risks associated with small sample sizes and improve the robustness of the conclusions.</p>
<p id="Par38">Once omics technologies have been fully incorporated into the medical routine, BA predictions will likely become standard measurements regularly discussed between patients and medical professionals. A novel personalised value to identify the ‘normal’ BA for an individual could also be estimated, together with common clinical measures, to provide precise individual interpretations<sup><a href="#CR46" class="usa-link" aria-describedby="CR46">46</a>,<a href="#CR47" class="usa-link" aria-describedby="CR47">47</a></sup>. In line with the clinical trial frameworks, epigenetic age measurements are also being implemented more frequently in studies investigating interventions for age-related diseases<sup><a href="#CR48" class="usa-link" aria-describedby="CR48">48</a></sup>. In oncology practices, consistent findings have been observed that individuals with accelerated epigenetic ageing often display a higher risk of developing various cancers, suggesting that regular BA monitoring may help identify high-risk patients for targeted screening<sup><a href="#CR49" class="usa-link" aria-describedby="CR49">49</a>,<a href="#CR50" class="usa-link" aria-describedby="CR50">50</a></sup>. In industry, this momentum is also already apparent for startups and biotech companies that offer epigenetic-based BA testing panels (<a href="https://gero.ai/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://gero.ai/</a>, <a href="https://www.elysiumhealth.com/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.elysiumhealth.com/</a>, <a href="https://glycanage.com/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://glycanage.com/</a>), demonstrating commercial readiness and patient demand for these emerging biomarkers. Despite this promise, the field remains in its early stages. Nevertheless, through continued large-scale validation, alignment with healthcare policies, and education of both clinicians and patients—BA predictions are poised to become essential tools for preventive and precision-oriented medical care.</p>
<p id="Par39">Further multi-scale longitudinal studies—including broader and more diverse populations, as well as external factors such as environment, lifestyle and real-time physiological metrics—will undoubtedly strengthen the validation of BA predictions at the personal level. In a digital twin context, such datasets would refine the capacity to model complex interactions and generate highly individualised forecasts. By integrating epigenomic, proteomic, and metabolomic data into machine-learning-enhanced simulations, future digital twins could differentiate natural ageing processes from disease-driven changes with greater precision. Ultimately, these innovations hold the potential to accelerate risk stratification, guide interventions, and transform how clinicians and patients collaborate on preventive and precision-oriented healthcare. Such studies would allow for simulations in complex scenarios where multiple variables interact, providing insights into the intricate dynamics that influence health and ageing. As the field of biological ageing advances, the construction of BA using ageing clocks employing diverse data sources such as epigenomics, proteomics, and metabolomics could be proven effective in uncovering novel biomarkers of biological ageing. Future endeavours to weave multi-omics into ageing clocks are poised to not only broaden our grasp of the molecular signatures that characterise ageing but also enhance the predictive powers of these models. The involvement of machine learning in harnessing diverse datasets—both molecular and environmental—would also advance the models and more accurately reflect how individuals respond to interventions or health risks. Moreover, as demonstrated by this study, where individual lifestyle factors were shown to influence ageing, incorporating these and potentially other external factors will be crucial. The continued expansion of this integrative approach is expected to yield more precise and actionable insights, solidifying the role of ageing clocks as indispensable tools. Ultimately, this will lead to robust multi-scale modelling of human digital twins, further advancing the evolving landscape of personalised medicine and ageing research.</p>
<p id="Par40">In summary, the convergence of BA clocks with human digital twin methodologies offers a powerful new paradigm for personalised healthcare. Through the dynamic integration of molecular and clinical data, digital twins can highlight early biomarkers of ageing and disease, optimise treatment strategies, and ultimately extend healthspan. Our findings underscore the feasibility of this approach and pave the way for future research into scalable, interoperable, and privacy-conscious platforms that will bring BA-driven digital twin models closer to routine clinical practice.</p></section><section id="Sec7"><h2 class="pmc_sec_title">Methods</h2>
<section id="Sec8"><h3 class="pmc_sec_title">Study design and participants</h3>
<p id="Par41">The IAM Frontier study is a unique longitudinal cohort study that ran for 13 months in 30 healthy individuals, consisting of 15 male and 15 female participants<sup><a href="#CR21" class="usa-link" aria-describedby="CR21">21</a></sup>. The 13-month study duration was chosen to cover the seasonal fluctuations that might occur over a 1-year period. The study specifically targeted the employees of the research organisation VITO within the age range of 45–59. A major reason for the selection of this group of employees was that as they are part of a research organisation, they are expected to be more open to research-grade technologies and interventions. The age range was selected because the highest prevalence of onset of chronic diseases occurs from the age of 45–65<sup><a href="#CR51" class="usa-link" aria-describedby="CR51">51</a></sup>. Individuals were selected based on the following inclusion criteria: not suffering from a chronic disease, diagnosed and currently followed-up by a medical specialist: asthma, chronic bronchitis, chronic obstructive pulmonary disease, emphysema, myocardial infarction, coronary heart disease (angina pectoris), other serious heart diseases, stroke (cerebral haemorrhage, cerebral thrombosis), diabetes, cancer (malignant tumour, also including leukaemia and lymphoma). At monthly visits, a range of samples (whole blood, plasma, urine, stool) were collected and sent to accredited laboratories and comprehensive multi-omics and clinical biochemistry data were assessed. Self-administered questionnaires on, for example, health conditions and physical activity were also completed by the participants. In this article, we analysed data from the IAM Frontier study, which included DNA methylation data, clinical biochemistry, proteomics, metabolomics and physical examinations data.</p></section><section id="Sec9"><h3 class="pmc_sec_title">Sample collection</h3>
<p id="Par42">The sampling of the IAM Frontier study took place between March 2019 and March 2020 and included the collection of human biospecimen and digital data. During the 13 months of the study, the participants donated blood, urine and stool samples at monthly visits. These samples were collected after overnight fasting for at least 8 h. The urine and blood samples (in EDTA-, citrate-, and serum-vacutainers) were transported to the clinical laboratory at room temperature within 6 h after collection. Peripheral blood mononuclear cells (PBMC) were isolated from EDTA-blood samples, and PBMC pellets were stored at −80 °C till the DNA extraction. At monthly visits, clinical tests and health examinations such as blood pressure, body height, weight, and abdominal circumference measurements were performed by accredited labs and appointed doctors. At bi-monthly visits, plasma samples were taken, and omics (proteomics and metabolomics) measurements were assessed. At months 1, 6 and 13, the PBMC samples were used to measure DNA methylation. At month 13, only 20 participants were able to donate samples due to the start of the COVID-19 pandemic. Table <a href="#Tab2" class="usa-link">2</a> presents an overview of the sample collection. All samples have been collected in accordance with the applicable Belgian regulations regarding the use of human body material for scientific research (Belgian Law on use of human body material, 2008) and the Belgian Royal Decree on biobanks (<em>Het Koninklijk Besluit betreffende de biobanken. Belgisch Staatsblad 05.02.2018. Brussels (2018)</em>). All participants signed an informed consent and the study was approved by the ethical committee of the Antwerp University Hospital (RegN°: B300201938600). The research was conducted in compliance with the Declaration of Helsinki and all relevant ethical guidelines.</p>
<section class="tw xbox font-sm" id="Tab2"><h4 class="obj_head">Table 2.</h4>
<div class="caption p"><p>Data collection of the IAM Frontier study</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th colspan="1" rowspan="1">Data</th>
<th colspan="1" rowspan="1">Measurement technique</th>
<th colspan="1" rowspan="1">Timepoints – month</th>
<th colspan="1" rowspan="1">Type of sample</th>
<th colspan="1" rowspan="1">Total samples</th>
<th colspan="1" rowspan="1">Remarks</th>
</tr></thead>
<tbody>
<tr>
<td colspan="1" rowspan="1">DNA methylation</td>
<td colspan="1" rowspan="1">Illumina Infinium MethylationEPIC BeadChips</td>
<td colspan="1" rowspan="1">2-8-13</td>
<td colspan="1" rowspan="1">PBMC</td>
<td colspan="1" rowspan="1">96</td>
<td colspan="1" rowspan="1">20 samples at month 13, 16 technical replicates</td>
</tr>
<tr>
<td colspan="1" rowspan="1">Clinical and physiological</td>
<td colspan="1" rowspan="1">Clinical biochemistry, blood cell counts, health examination</td>
<td colspan="1" rowspan="1">1-2-3-4-5-6-7-8-9-10-11-12-13</td>
<td colspan="1" rowspan="1">Whole blood, serum PB, fasting urine</td>
<td colspan="1" rowspan="1">380</td>
<td rowspan="3" colspan="1">20 samples at month 13</td>
</tr>
<tr>
<td colspan="1" rowspan="1">Metabolomics</td>
<td colspan="1" rowspan="1">NMR - MS</td>
<td colspan="1" rowspan="1">1-3-5-7-9-11-13</td>
<td colspan="1" rowspan="1">Plasma</td>
<td colspan="1" rowspan="1">200</td>
</tr>
<tr>
<td colspan="1" rowspan="1">Proteomics</td>
<td colspan="1" rowspan="1">LC-MSMS -PEA</td>
<td colspan="1" rowspan="1">1-3-5-7-9-11-13</td>
<td colspan="1" rowspan="1">Plasma</td>
<td colspan="1" rowspan="1">200</td>
</tr>
<tr>
<td colspan="1" rowspan="1">Microbiome</td>
<td colspan="1" rowspan="1">Illumina MiSeq</td>
<td colspan="1" rowspan="1">1-6-13</td>
<td colspan="1" rowspan="1">Stool</td>
<td colspan="1" rowspan="1">20</td>
<td colspan="1" rowspan="1">16 samples at month 13</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section></section><section id="Sec10"><h3 class="pmc_sec_title">DNA methylation assay</h3>
<p id="Par43">The DNA methylation assay was carried out using Diagenode Epigenomic Services (Vienna, Austria, Cat No. G02090000). PBMC samples were sent for DNA methylation profiling using the Illumina Infinium MethylationEPIC array BeadChip (850 K) platform to analyse the methylation status of more than 850,000 CpGs per sample. This microarray covers ∼96% of CpG Islands and 99% of annotated RefSeq genes. We performed the DNA methylation data pre-processing and the corresponding details can be found in Supplementary Document Section <a href="#MOESM1" class="usa-link">1</a>.</p></section><section id="Sec11"><h3 class="pmc_sec_title">Analysis of epigenetic clocks and predictors</h3>
<p id="Par44">We applied 34 different epigenetic age and health trait predictors (see Fig. <a href="#Fig2" class="usa-link">2</a>). The Skin &amp; Blood Clock<sup><a href="#CR52" class="usa-link" aria-describedby="CR52">52</a></sup>, Multi-tissue Clock<sup><a href="#CR53" class="usa-link" aria-describedby="CR53">53</a></sup>, HannumAge<sup><a href="#CR26" class="usa-link" aria-describedby="CR26">26</a></sup>, DNAmTL<sup><a href="#CR38" class="usa-link" aria-describedby="CR38">38</a></sup>, PhenoAge<sup><a href="#CR54" class="usa-link" aria-describedby="CR54">54</a></sup>, GrimAge<sup><a href="#CR27" class="usa-link" aria-describedby="CR27">27</a></sup>, GrimAge DNAmPACKYRS and GrimAge protein levels (DNAmADM, B2M, CystatinC, GDF15, Leptin, PAI1, TIMP1) were obtained by Steve Horvath’s DNA Methylation Age Calculator available on <a href="http://dnamage.genetics.ucla.edu/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://dnamage.genetics.ucla.edu/</a>. We also included the new version of GrimaAge, GrimAge2<sup><a href="#CR55" class="usa-link" aria-describedby="CR55">55</a></sup>. The MethylDetectR predictions (Age, Alcohol, BMI, HDL, BodyFat, Waist:Hip Ratio and Smoking)<sup><a href="#CR43" class="usa-link" aria-describedby="CR43">43</a></sup> were calculated using the code available at <a href="https://zenodo.org/record/4646300" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://zenodo.org/record/4646300</a>. The methylation Pace of Age (mPoA)<sup><a href="#CR25" class="usa-link" aria-describedby="CR25">25</a></sup> was estimated using the code available at <a href="https://github.com/danbelsky/DunedinPoAm38" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/danbelsky/DunedinPoAm38</a>. The refined version of mPoA, DunedinPACE<sup><a href="#CR56" class="usa-link" aria-describedby="CR56">56</a></sup> was also estimated. The MetaClock<sup><a href="#CR34" class="usa-link" aria-describedby="CR34">34</a></sup> code was received by e-mail from the author Morgan E Levine. EpiTOC scores were calculated using the code available in the corresponding publication<sup><a href="#CR35" class="usa-link" aria-describedby="CR35">35</a></sup>. EpiTOC2<sup><a href="#CR36" class="usa-link" aria-describedby="CR36">36</a></sup> scores were calculated using the code available from 10.5281/zenodo.2632938, and MiAge<sup><a href="#CR37" class="usa-link" aria-describedby="CR37">37</a></sup> scores were calculated using the code available from <a href="http://www.columbia.edu/~sw2206/softwares.htm" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://www.columbia.edu/~sw2206/softwares.htm</a>. Alcohol predictions<sup><a href="#CR57" class="usa-link" aria-describedby="CR57">57</a></sup> were generated using the <em>dnamlci</em> R package available from <a href="https://github.com/yousefi138/dnamalci" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/yousefi138/dnamalci</a>. Elliot’s smoking score<sup><a href="#CR58" class="usa-link" aria-describedby="CR58">58</a></sup>, Zhang’s smoking score<sup><a href="#CR59" class="usa-link" aria-describedby="CR59">59</a></sup> and EpiSmokEr’s smoking status<sup><a href="#CR60" class="usa-link" aria-describedby="CR60">60</a></sup> were obtained using the R package <em>EpiSmokEr</em> available at <a href="https://github.com/sailalithabollepalli/EpiSmokEr" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/sailalithabollepalli/EpiSmokEr</a>. Causality clocks<sup><a href="#CR61" class="usa-link" aria-describedby="CR61">61</a></sup> including CausAge, DamAge, and AdaptAge were estimated using the <em>biolearn</em> Python library available from <a href="https://github.com/bio-learn/biolearn/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/bio-learn/biolearn/</a>.</p>
<p id="Par45">GrimAge, HannumAge, MethylDetectRAge, mPoA, as well as the smoking, alcohol and health trait predictors, were originally developed for whole blood measurements but have been shown to work well with PBMCs in our study and others<sup><a href="#CR62" class="usa-link" aria-describedby="CR62">62</a>–<a href="#CR65" class="usa-link" aria-describedby="CR65">65</a></sup>. Table <a href="#Tab3" class="usa-link">3</a> shows the list of clocks implemented in this study, including clocks predicted from clinical, metabolomics, and proteomics data (see the next section).</p>
<p id="Par46">We also predicted blood counts using minfi<sup><a href="#CR66" class="usa-link" aria-describedby="CR66">66</a></sup> and the Reinius reference dataset<sup><a href="#CR67" class="usa-link" aria-describedby="CR67">67</a></sup>, as well as IDOL using the Salas reference dataset<sup><a href="#CR68" class="usa-link" aria-describedby="CR68">68</a></sup> with the code available from <a href="https://github.com/immunomethylomics/FlowSorted.Blood.EPIC" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/immunomethylomics/FlowSorted.Blood.EPIC</a>.</p>
<p id="Par47">The correlation between predicted and CA was calculated using Pearson’s correlation. We excluded technical replicates from the calculation of correlation coefficients.</p>
<p id="Par48">For the longitudinal analyses of epigenetic predictions, we used the 16 technical replicates available from our study to estimate technical variation. Two technical replicates were generated within each of the three DNA methylation time points, and ten additional technical replicates of the two previous time points were generated at the third time point, providing detailed insight into technical variation influencing the predictions. We used the maximum absolute difference observed between technical replicates as a threshold to define potential biological differences across time points after also adding the chronological time that passed between time points. For example, the maximum absolute differences observed for the Skin &amp; Blood Clock across technical replicates was 2.56 years (the reported median error in blood is 2.5 years<sup><a href="#CR52" class="usa-link" aria-describedby="CR52">52</a></sup>), and the threshold we used represents the maximum differences observed among technical replicates plus the chronological time that passed between the time points which are maximum 343 days (0.94 years).</p>
<p id="Par49">For the global analyses of our dataset and the identification of biological outliers of interest based on our cohort, we introduced a mean absolute deviation (MAD) threshold of ±2*MAD and ±3*MAD across all biological replicates. In particular, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq13"><math id="d33e1872"><mrow><msub><mrow><mover accent="true"><mrow><mi>y</mi></mrow><mo>ˆ</mo></mover></mrow><mrow><mi mathvariant="italic">ij</mi></mrow></msub><mo>±</mo><mn>2</mn><mi mathvariant="italic">MAD</mi></mrow></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq14"><math id="d33e1894"><mrow><msub><mrow><mover accent="true"><mrow><mi>y</mi></mrow><mo>ˆ</mo></mover></mrow><mrow><mi mathvariant="italic">ij</mi></mrow></msub><mo>±</mo><mn>3</mn><mi mathvariant="italic">MAD</mi></mrow></math></span>, where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq15"><math id="d33e1916"><msub><mrow><mover accent="true"><mrow><mi>y</mi></mrow><mo>ˆ</mo></mover></mrow><mrow><mi mathvariant="italic">ij</mi></mrow></msub></math></span> is the individual BA prediction of subject <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq16"><math id="d33e1933"><mi>i</mi></math></span> at time point <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq17"><math id="d33e1942"><mi>j</mi></math></span>. The MAD thresholds correspond to the 95th and 99th standard normal percentiles and are selected for their robustness in describing data with a relatively small time series as in the IAM Frontier study.</p></section><section id="Sec12"><h3 class="pmc_sec_title">Analysis of clinical, metabolomics, and proteomics clocks</h3>
<p id="Par50">The monthly clinical and bi-monthly metabolomics as well as proteomics data were used for predicting longitudinal BA using several published calculators. We applied the MLR model and principal component analysis (PCA) developed by Levine et al.<sup><a href="#CR28" class="usa-link" aria-describedby="CR28">28</a></sup>—they will be referred to as MLR-Levine and PCA-Levine—to the clinical data that consist of samples from 12 to 13 time points per individual. The model prediction involves both clinical and physiological measurements, including total cholesterol level, glycated haemoglobin, C-reactive protein, systolic blood pressure, forced expiratory volume (FEV) and cytomegalovirus (CMV). In the IAM Frontier data, FEV and CMV were not measured, but we imputed the values with the corresponding median from the original study<sup><a href="#CR69" class="usa-link" aria-describedby="CR69">69</a></sup>. The same clinical variables as in the PCA-Levine model were further used to predict the BA using the KDM<sup><a href="#CR31" class="usa-link" aria-describedby="CR31">31</a></sup>. Unlike the MLR-Levine and the PCA-Levine, KDM also incorporates CA in their estimation procedure. Similarly, we also computed other BA predictions based on similar clinical biochemistry variables developed by McDonald-Dunlop et al.<sup><a href="#CR29" class="usa-link" aria-describedby="CR29">29</a></sup>.</p>
<p id="Par51">The proteomics and NMR metabolomics data consist of samples from six to seven time points per individual, where the samples were sent to the laboratory in four different batches. For the proteomics data, batch correction normalisation was done prior to the analysis to reduce the technical variation between batches/plates<sup><a href="#CR70" class="usa-link" aria-describedby="CR70">70</a></sup>. We performed PCA and multidimensional scaling analyses for the metabolomics data, where we did not observe any batch effects, so the raw metabolite abundances were used. In both datasets, there are twenty subjects with technical replicates spread across different time points. We performed a procedure for selecting the samples (between the originals and the replicates) by computing cosine similarity coefficients for all samples. The samples with the closest similarity to the rest of the individuals’ measurements were selected. Further, we predicted the BA using other published proteomics and metabolomics clocks: ProtAge-MD (from McDonald-Dunlop et al.<sup><a href="#CR14" class="usa-link" aria-describedby="CR14">14</a></sup>), MetaboAge-MD, ProtAge-Tanaka, MetaboAge-vdA, and Metabonomics<sup><a href="#CR29" class="usa-link" aria-describedby="CR29">29</a>,<a href="#CR30" class="usa-link" aria-describedby="CR30">30</a>,<a href="#CR32" class="usa-link" aria-describedby="CR32">32</a>,<a href="#CR33" class="usa-link" aria-describedby="CR33">33</a></sup>. Recently, the GOLD BioAge model was proposed as a Gompertz-based BA estimator using clinical and omics data<sup><a href="#CR71" class="usa-link" aria-describedby="CR71">71</a></sup>. We reference it here to highlight ongoing developments in this area.</p></section><section id="Sec13"><h3 class="pmc_sec_title">Software tools and programming language</h3>
<p id="Par52">The network (Fig. <a href="#Fig2" class="usa-link">2</a>) was created in Gephi version 0.9.5<sup><a href="#CR72" class="usa-link" aria-describedby="CR72">72</a></sup> and Cytoscape version 3.8.2<sup><a href="#CR73" class="usa-link" aria-describedby="CR73">73</a></sup>. All data analyses were conducted in the R statistical environment version 4.1.3.</p></section></section><section id="Sec14"><h2 class="pmc_sec_title">Supplementary information</h2>
<section class="sm xbox font-sm" id="MOESM1"><div class="media p"><div class="caption">
<a href="/articles/instance/12371080/bin/41746_2025_1911_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary document_v2</a><sup> (1.3MB, pdf) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>We acknowledge the contribution of researchers and laboratory staff at VITO NV for operationalising the IAM Frontier cohort. M.P. is funded through VITO and the Research Foundation—Flanders (FWO), as a postdoctoral fellow in fundamental research (grant number: 12AMD24N).</p></section><section id="notes1"><h2 class="pmc_sec_title">Author contributions</h2>
<p>Conceptualisation, M.P., G.E. and S.E.; Formal analysis, M.P. and S.E.; Visualisation, M.P. and S.E.; Writing—Original draft, M.P., G.E. and S.E.; Writing—review and editing, G.E. and S.E.; Supervision; G.E., S.E., O.T. and S.B. All authors have read, reviewed, and approved the manuscript.</p></section><section id="notes2"><h2 class="pmc_sec_title">Data availability</h2>
<p>Due to participants’ privacy, the data is available upon request to the I AM Frontier Project Data Access Committee for further research (DataAccess.IAF@vito.be). The sensitive nature of the data does not allow it to be deposited into public repositories. We welcome collaboration with other research groups focused on personalised prevention, offering data sharing opportunities to advance research and improve healthcare outcomes.</p></section><section id="notes3"><h2 class="pmc_sec_title">Code availability</h2>
<p>The codes are available upon request from the corresponding authors.</p></section><section id="FPar1"><h2 class="pmc_sec_title">Competing interests</h2>
<p id="Par53">The authors declare no competing interests.</p></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm">
<div class="fn p" id="fn1"><p><strong>Publisher’s note</strong> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></div>
<div class="fn p" id="fn2"><p>These authors contributed equally: Simone Ecker, Gökhan Ertaylan.</p></div>
</div></section><section id="_ci93_" lang="en" class="contrib-info"><h2 class="pmc_sec_title">Contributor Information</h2>
<p>Murih Pusparum, Email: murih.pusparum@vito.be.</p>
<p>Gökhan Ertaylan, Email: gokhan.ertaylan@vito.be.</p></section><section id="sec15"><h2 class="pmc_sec_title">Supplementary information</h2>
<p>The online version contains supplementary material available at 10.1038/s41746-025-01911-9.</p></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>López-Otín, C., Blasco, M. A., Partridge, L., Serrano, M. &amp; Kroemer, G. The hallmarks of aging. <em>Cell</em><strong>153</strong>, 1194–1217 (2013).
</cite> [<a href="https://doi.org/10.1016/j.cell.2013.05.039" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3836174/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23746838/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?L%C3%B3pez-Ot%C3%ADn,%20C.,%20Blasco,%20M.%20A.,%20Partridge,%20L.,%20Serrano,%20M.%20&amp;%20Kroemer,%20G.%20The%20hallmarks%20of%20aging.%20Cell153,%201194%E2%80%931217%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>Bruynseels, K., Santoni de Sio, F. &amp; van den Hoven, J. Digital twins in health care: ethical implications of an emerging engineering paradigm. <em>Front. Genet.</em><strong>9</strong>, 31 (2018).</cite> [<a href="https://doi.org/10.3389/fgene.2018.00031" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5816748/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29487613/" class="usa-link">PubMed</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>Corral-Acero, J. et al. The ‘Digital Twin’ to enable the vision of precision cardiology. <em>Eur. Heart J.</em><strong>41</strong>, 4556–4564 (2020).
</cite> [<a href="https://doi.org/10.1093/eurheartj/ehaa159" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7774470/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32128588/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Corral-Acero,%20J.%20et%20al.%20The%20%E2%80%98Digital%20Twin%E2%80%99%20to%20enable%20the%20vision%20of%20precision%20cardiology.%20Eur.%20Heart%20J.41,%204556%E2%80%934564%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Jylhävä, J., Pedersen, N. L. &amp; Hägg, S. Biological age predictors. <em>EBioMedicine</em><strong>21</strong>, 29–36 (2017).
</cite> [<a href="https://doi.org/10.1016/j.ebiom.2017.03.046" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5514388/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28396265/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jylh%C3%A4v%C3%A4,%20J.,%20Pedersen,%20N.%20L.%20&amp;%20H%C3%A4gg,%20S.%20Biological%20age%20predictors.%20EBioMedicine21,%2029%E2%80%9336%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Blackburn, E. H., Greider, C. W. &amp; Szostak, J. W. Telomeres and telomerase: the path from maize, Tetrahymena and yeast to human cancer and aging. <em>Nat. Med.</em><strong>12</strong>, 1133–1138 (2006).
</cite> [<a href="https://doi.org/10.1038/nm1006-1133" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17024208/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Blackburn,%20E.%20H.,%20Greider,%20C.%20W.%20&amp;%20Szostak,%20J.%20W.%20Telomeres%20and%20telomerase:%20the%20path%20from%20maize,%20Tetrahymena%20and%20yeast%20to%20human%20cancer%20and%20aging.%20Nat.%20Med.12,%201133%E2%80%931138%20(2006)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Zhu, X. et al. The association between telomere length and cancer risk in population studies. <em>Sci. Rep.</em><strong>6</strong>, 22243 (2016).
</cite> [<a href="https://doi.org/10.1038/srep22243" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4768100/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26915412/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhu,%20X.%20et%20al.%20The%20association%20between%20telomere%20length%20and%20cancer%20risk%20in%20population%20studies.%20Sci.%20Rep.6,%2022243%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Haycock, P. C. et al. Leucocyte telomere length and risk of cardiovascular disease: systematic review and meta-analysis. <em>BMJ</em><strong>349</strong>, g4227–g4227 (2014).
</cite> [<a href="https://doi.org/10.1136/bmj.g4227" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4086028/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25006006/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Haycock,%20P.%20C.%20et%20al.%20Leucocyte%20telomere%20length%20and%20risk%20of%20cardiovascular%20disease:%20systematic%20review%20and%20meta-analysis.%20BMJ349,%20g4227%E2%80%93g4227%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>Scheller Madrid, A., Rode, L., Nordestgaard, B. G. &amp; Bojesen, S. E. Short telomere length and ischemic heart disease: observational and genetic studies in 290 022 individuals. <em>Clin. Chem.</em><strong>62</strong>, 1140–1149 (2016).
</cite> [<a href="https://doi.org/10.1373/clinchem.2016.258566" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27259814/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Scheller%20Madrid,%20A.,%20Rode,%20L.,%20Nordestgaard,%20B.%20G.%20&amp;%20Bojesen,%20S.%20E.%20Short%20telomere%20length%20and%20ischemic%20heart%20disease:%20observational%20and%20genetic%20studies%20in%20290%20022%20individuals.%20Clin.%20Chem.62,%201140%E2%80%931149%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>Breitling, L. P. et al. Frailty is associated with the epigenetic clock but not with telomere length in a German cohort. <em>Clin. Epigenetics</em><strong>8</strong>, 21 (2016).
</cite> [<a href="https://doi.org/10.1186/s13148-016-0186-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4768341/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26925173/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Breitling,%20L.%20P.%20et%20al.%20Frailty%20is%20associated%20with%20the%20epigenetic%20clock%20but%20not%20with%20telomere%20length%20in%20a%20German%20cohort.%20Clin.%20Epigenetics8,%2021%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>Vetter, V. M. et al. Epigenetic clock and relative telomere length represent largely different aspects of aging in the Berlin Aging Study II (BASE-II). <em>J. Gerontol. Ser. A</em><strong>74</strong>, 27–32 (2019).</cite> [<a href="https://doi.org/10.1093/gerona/gly184" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30137208/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Vetter,%20V.%20M.%20et%20al.%20Epigenetic%20clock%20and%20relative%20telomere%20length%20represent%20largely%20different%20aspects%20of%20aging%20in%20the%20Berlin%20Aging%20Study%20II%20(BASE-II).%20J.%20Gerontol.%20Ser.%20A74,%2027%E2%80%9332%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Marioni, R. E. et al. The epigenetic clock and telomere length are independently associated with chronological age and mortality. <em>Int. J. Epidemiol.</em><strong>47</strong>, 356–356 (2018).
</cite> [<a href="https://doi.org/10.1093/ije/dyx233" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5837660/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29190382/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Marioni,%20R.%20E.%20et%20al.%20The%20epigenetic%20clock%20and%20telomere%20length%20are%20independently%20associated%20with%20chronological%20age%20and%20mortality.%20Int.%20J.%20Epidemiol.47,%20356%E2%80%93356%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Bocklandt, S. et al. Epigenetic predictor of age. <em>PLoS ONE</em><strong>6</strong>, e14821 (2011).
</cite> [<a href="https://doi.org/10.1371/journal.pone.0014821" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3120753/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21731603/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bocklandt,%20S.%20et%20al.%20Epigenetic%20predictor%20of%20age.%20PLoS%20ONE6,%20e14821%20(2011)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>Ecker, S. &amp; Beck, S. The epigenetic clock: a molecular crystal ball for human aging? <em>Aging</em><strong>11</strong>, 833–835 (2019).
</cite> [<a href="https://doi.org/10.18632/aging.101712" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6366965/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30669120/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ecker,%20S.%20&amp;%20Beck,%20S.%20The%20epigenetic%20clock:%20a%20molecular%20crystal%20ball%20for%20human%20aging?%20Aging11,%20833%E2%80%93835%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>Horvath, S. &amp; Raj, K. DNA methylation-based biomarkers and the epigenetic clock theory of ageing. <em>Nat. Rev. Genet.</em><strong>19</strong>, 371–384 (2018).
</cite> [<a href="https://doi.org/10.1038/s41576-018-0004-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29643443/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Horvath,%20S.%20&amp;%20Raj,%20K.%20DNA%20methylation-based%20biomarkers%20and%20the%20epigenetic%20clock%20theory%20of%20ageing.%20Nat.%20Rev.%20Genet.19,%20371%E2%80%93384%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>Jansen, R. et al. An integrative study of five biological clocks in somatic and mental health. <em>Elife</em><strong>10</strong>, e59479 (2021).</cite> [<a href="https://doi.org/10.7554/eLife.59479" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7872513/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33558008/" class="usa-link">PubMed</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Crimmins, E. M., Thyagarajan, B., Levine, M. E., Weir, D. R. &amp; Faul, J. Associations of age, sex, race/ethnicity, and education with 13 epigenetic clocks in a nationally representative U.S. sample: the health and retirement study. <em>J. Gerontol. Ser. A</em><strong>76</strong>, 1117–1123 (2021).</cite> [<a href="https://doi.org/10.1093/gerona/glab016" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8140049/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33453106/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Crimmins,%20E.%20M.,%20Thyagarajan,%20B.,%20Levine,%20M.%20E.,%20Weir,%20D.%20R.%20&amp;%20Faul,%20J.%20Associations%20of%20age,%20sex,%20race/ethnicity,%20and%20education%20with%2013%20epigenetic%20clocks%20in%20a%20nationally%20representative%20U.S.%20sample:%20the%20health%20and%20retirement%20study.%20J.%20Gerontol.%20Ser.%20A76,%201117%E2%80%931123%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>Argentieri, M. A. et al. Proteomic aging clock predicts mortality and risk of common age-related diseases in diverse populations. <em>Nat. Med.</em><strong>30</strong>, 2450–2460 (2024).
</cite> [<a href="https://doi.org/10.1038/s41591-024-03164-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11405266/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39117878/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Argentieri,%20M.%20A.%20et%20al.%20Proteomic%20aging%20clock%20predicts%20mortality%20and%20risk%20of%20common%20age-related%20diseases%20in%20diverse%20populations.%20Nat.%20Med.30,%202450%E2%80%932460%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Knoppers, T. et al. The omics of our lives: practices and policies of direct-to-consumer epigenetic and microbiomic testing companies. <em>N. Genet. Soc.</em><strong>40</strong>, 541–569 (2021).</cite> [<a href="https://scholar.google.com/scholar_lookup?Knoppers,%20T.%20et%20al.%20The%20omics%20of%20our%20lives:%20practices%20and%20policies%20of%20direct-to-consumer%20epigenetic%20and%20microbiomic%20testing%20companies.%20N.%20Genet.%20Soc.40,%20541%E2%80%93569%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Carbonara, K., MacNeil, A. J., O’Leary, D. D. &amp; Coorssen, J. R. Profit versus quality: the enigma of scientific wellness. <em>J. Pers. Med.</em><strong>12</strong>, 34 (2022).
</cite> [<a href="https://doi.org/10.3390/jpm12010034" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8779909/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35055349/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Carbonara,%20K.,%20MacNeil,%20A.%20J.,%20O%E2%80%99Leary,%20D.%20D.%20&amp;%20Coorssen,%20J.%20R.%20Profit%20versus%20quality:%20the%20enigma%20of%20scientific%20wellness.%20J.%20Pers.%20Med.12,%2034%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>VITO Health Belgium. I AM Frontier Study. <a href="https://iammyhealth.eu/en/i-am-frontier" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://iammyhealth.eu/en/i-am-frontier</a>.</cite>
</li>
<li id="CR21">
<span class="label">21.</span><cite>Heylen, D. et al. Cohort profile: The Belgian I AM frontier prospective cohort study for comprehensive health outcome exploration. <em>PLoS ONE</em><strong>20</strong>, e0326024 (2025).</cite> [<a href="https://doi.org/10.1371/journal.pone.0326024" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC12161581/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40504832/" class="usa-link">PubMed</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>Bassingthwaighte, J. B. Strategies for the physiome project. <em>Ann. Biomed. Eng.</em><strong>28</strong>, 1043–1058 (2000).
</cite> [<a href="https://doi.org/10.1114/1.1313771" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3425440/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11144666/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bassingthwaighte,%20J.%20B.%20Strategies%20for%20the%20physiome%20project.%20Ann.%20Biomed.%20Eng.28,%201043%E2%80%931058%20(2000)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>EDITH. European Virtual Human Twin. <a href="https://www.edith-csa.eu/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.edith-csa.eu/</a>.</cite>
</li>
<li id="CR24">
<span class="label">24.</span><cite>Weidner, C. et al. Aging of blood can be tracked by DNA methylation changes at just three CpG sites. <em>Genome Biol.</em><strong>15</strong>, R24 (2014).
</cite> [<a href="https://doi.org/10.1186/gb-2014-15-2-r24" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4053864/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24490752/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Weidner,%20C.%20et%20al.%20Aging%20of%20blood%20can%20be%20tracked%20by%20DNA%20methylation%20changes%20at%20just%20three%20CpG%20sites.%20Genome%20Biol.15,%20R24%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>Belsky, D. W. et al. Quantification of the pace of biological aging in humans through a blood test, the DunedinPoAm DNA methylation algorithm. <em>Elife</em><strong>9</strong>, e54870 (2020).</cite> [<a href="https://doi.org/10.7554/eLife.54870" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7282814/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32367804/" class="usa-link">PubMed</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>Hannum, G. et al. Genome-wide methylation profiles reveal quantitative views of human aging rates. <em>Mol. Cell</em><strong>49</strong>, 359–367 (2013).
</cite> [<a href="https://doi.org/10.1016/j.molcel.2012.10.016" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3780611/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23177740/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hannum,%20G.%20et%20al.%20Genome-wide%20methylation%20profiles%20reveal%20quantitative%20views%20of%20human%20aging%20rates.%20Mol.%20Cell49,%20359%E2%80%93367%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Lu, A. T. et al. DNA methylation GrimAge strongly predicts lifespan and healthspan. <em>Aging</em><strong>11</strong>, 303–327 (2019).
</cite> [<a href="https://doi.org/10.18632/aging.101684" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6366976/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30669119/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lu,%20A.%20T.%20et%20al.%20DNA%20methylation%20GrimAge%20strongly%20predicts%20lifespan%20and%20healthspan.%20Aging11,%20303%E2%80%93327%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Levine, M. E. Modeling the rate of senescence: can estimated biological age predict mortality more accurately than chronological age? <em>J. Gerontol. A Biol. Sci. Med. Sci.</em><strong>68</strong>, 667–674 (2013).
</cite> [<a href="https://doi.org/10.1093/gerona/gls233" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3660119/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23213031/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Levine,%20M.%20E.%20Modeling%20the%20rate%20of%20senescence:%20can%20estimated%20biological%20age%20predict%20mortality%20more%20accurately%20than%20chronological%20age?%20J.%20Gerontol.%20A%20Biol.%20Sci.%20Med.%20Sci.68,%20667%E2%80%93674%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>Macdonald-Dunlop, E. et al. A catalogue of omics biological ageing clocks reveals substantial commonality and associations with disease risk. <em>Aging</em><strong>14</strong>, 623–659 (2022).
</cite> [<a href="https://doi.org/10.18632/aging.203847" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8833109/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35073279/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Macdonald-Dunlop,%20E.%20et%20al.%20A%20catalogue%20of%20omics%20biological%20ageing%20clocks%20reveals%20substantial%20commonality%20and%20associations%20with%20disease%20risk.%20Aging14,%20623%E2%80%93659%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR30">
<span class="label">30.</span><cite>Tanaka, T. et al. Plasma proteomic signature of age in healthy humans. <em>Aging Cell</em><strong>17</strong>, e12799 (2018).
</cite> [<a href="https://doi.org/10.1111/acel.12799" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6156492/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29992704/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tanaka,%20T.%20et%20al.%20Plasma%20proteomic%20signature%20of%20age%20in%20healthy%20humans.%20Aging%20Cell17,%20e12799%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>Klemera, P. &amp; Doubal, S. A new approach to the concept and computation of biological age. <em>Mech. Ageing Dev.</em><strong>127</strong>, 240–248 (2006).
</cite> [<a href="https://doi.org/10.1016/j.mad.2005.10.004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16318865/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Klemera,%20P.%20&amp;%20Doubal,%20S.%20A%20new%20approach%20to%20the%20concept%20and%20computation%20of%20biological%20age.%20Mech.%20Ageing%20Dev.127,%20240%E2%80%93248%20(2006)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>van den Akker, E. B. et al. Metabolic age based on the BBMRI-NL <sup>1</sup> H-NMR metabolomics repository as biomarker of age-related disease. <em>Circ. Genom. Precis. Med.</em><strong>13</strong>, 541–547 (2020).
</cite> [<a href="https://doi.org/10.1161/CIRCGEN.119.002610" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33079603/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?van%20den%20Akker,%20E.%20B.%20et%20al.%20Metabolic%20age%20based%20on%20the%20BBMRI-NL%201%20H-NMR%20metabolomics%20repository%20as%20biomarker%20of%20age-related%20disease.%20Circ.%20Genom.%20Precis.%20Med.13,%20541%E2%80%93547%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Hertel, J. et al. Measuring biological age via metabonomics: the metabolic age score. <em>J. Proteome Res.</em><strong>15</strong>, 400–410 (2016).
</cite> [<a href="https://doi.org/10.1021/acs.jproteome.5b00561" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26652958/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hertel,%20J.%20et%20al.%20Measuring%20biological%20age%20via%20metabonomics:%20the%20metabolic%20age%20score.%20J.%20Proteome%20Res.15,%20400%E2%80%93410%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>Liu, Z. et al. Underlying features of epigenetic aging clocks in vivo and in vitro. <em>Aging Cell</em><strong>19</strong>, e13229 (2020).</cite> [<a href="https://doi.org/10.1111/acel.13229" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7576259/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32930491/" class="usa-link">PubMed</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>Yang, Z. et al. Correlation of an epigenetic mitotic clock with cancer risk. <em>Genome Biol.</em><strong>17</strong>, 205 (2016).
</cite> [<a href="https://doi.org/10.1186/s13059-016-1064-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5046977/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27716309/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yang,%20Z.%20et%20al.%20Correlation%20of%20an%20epigenetic%20mitotic%20clock%20with%20cancer%20risk.%20Genome%20Biol.17,%20205%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR36">
<span class="label">36.</span><cite>Teschendorff, A. E. A comparison of epigenetic mitotic-like clocks for cancer risk prediction. <em>Genome Med.</em><strong>12</strong>, 56 (2020).
</cite> [<a href="https://doi.org/10.1186/s13073-020-00752-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7315560/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32580750/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Teschendorff,%20A.%20E.%20A%20comparison%20of%20epigenetic%20mitotic-like%20clocks%20for%20cancer%20risk%20prediction.%20Genome%20Med.12,%2056%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR37">
<span class="label">37.</span><cite>Youn, A. &amp; Wang, S. The MiAge calculator: a DNA methylation-based mitotic age calculator of human tissue types. <em>Epigenetics</em><strong>13</strong>, 192–206 (2018).
</cite> [<a href="https://doi.org/10.1080/15592294.2017.1389361" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5873367/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29160179/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Youn,%20A.%20&amp;%20Wang,%20S.%20The%20MiAge%20calculator:%20a%20DNA%20methylation-based%20mitotic%20age%20calculator%20of%20human%20tissue%20types.%20Epigenetics13,%20192%E2%80%93206%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR38">
<span class="label">38.</span><cite>Lu, A. T. et al. DNA methylation-based estimator of telomere length. <em>Aging</em><strong>11</strong>, 5895–5923 (2019).
</cite> [<a href="https://doi.org/10.18632/aging.102173" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6738410/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31422385/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lu,%20A.%20T.%20et%20al.%20DNA%20methylation-based%20estimator%20of%20telomere%20length.%20Aging11,%205895%E2%80%935923%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR39">
<span class="label">39.</span><cite>Cooper, J. G. et al. Age specific reference intervals for plasma biomarkers of neurodegeneration and neurotrauma in a Canadian population. <em>Clin. Biochem.</em><strong>121–122</strong>, 110680 (2023).
</cite> [<a href="https://doi.org/10.1016/j.clinbiochem.2023.110680" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37884086/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cooper,%20J.%20G.%20et%20al.%20Age%20specific%20reference%20intervals%20for%20plasma%20biomarkers%20of%20neurodegeneration%20and%20neurotrauma%20in%20a%20Canadian%20population.%20Clin.%20Biochem.121%E2%80%93122,%20110680%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR40">
<span class="label">40.</span><cite>Swainson, M. G., Batterham, A. M. &amp; Hind, K. Age- and sex-specific reference intervals for visceral fat mass in adults. <em>Int. J. Obes.</em><strong>44</strong>, 289–296 (2020).</cite> [<a href="https://doi.org/10.1038/s41366-019-0393-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31201361/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Swainson,%20M.%20G.,%20Batterham,%20A.%20M.%20&amp;%20Hind,%20K.%20Age-%20and%20sex-specific%20reference%20intervals%20for%20visceral%20fat%20mass%20in%20adults.%20Int.%20J.%20Obes.44,%20289%E2%80%93296%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR41">
<span class="label">41.</span><cite>Rutledge, J., Oh, H. &amp; Wyss-Coray, T. Measuring biological age using omics data. <em>Nat. Rev. Genet.</em><strong>23</strong>, 715–727 (2022).
</cite> [<a href="https://doi.org/10.1038/s41576-022-00511-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10048602/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35715611/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rutledge,%20J.,%20Oh,%20H.%20&amp;%20Wyss-Coray,%20T.%20Measuring%20biological%20age%20using%20omics%20data.%20Nat.%20Rev.%20Genet.23,%20715%E2%80%93727%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR42">
<span class="label">42.</span><cite>Physiome Project. The Physiome Project. <a href="https://physiomeproject.org/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://physiomeproject.org/</a>.</cite>
</li>
<li id="CR43">
<span class="label">43.</span><cite>Hillary, R. F. &amp; Marioni, R. E. MethylDetectR: a software for methylation-based health profiling. <em>Wellcome Open Res.</em><strong>5</strong>, 283 (2021).
</cite> [<a href="https://doi.org/10.12688/wellcomeopenres.16458.1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8080939/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33969230/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hillary,%20R.%20F.%20&amp;%20Marioni,%20R.%20E.%20MethylDetectR:%20a%20software%20for%20methylation-based%20health%20profiling.%20Wellcome%20Open%20Res.5,%20283%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR44">
<span class="label">44.</span><cite>Tang, C. et al. A roadmap for the development of human body digital twins. <em>Nat. Rev. Electr. Eng.</em><strong>1</strong>, 199–207 (2024).</cite> [<a href="https://scholar.google.com/scholar_lookup?Tang,%20C.%20et%20al.%20A%20roadmap%20for%20the%20development%20of%20human%20body%20digital%20twins.%20Nat.%20Rev.%20Electr.%20Eng.1,%20199%E2%80%93207%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR45">
<span class="label">45.</span><cite>Karczewski, K. J. &amp; Snyder, M. P. Integrative omics for health and disease. <em>Nat. Rev. Genet.</em><strong>19</strong>, 299–310 (2018).
</cite> [<a href="https://doi.org/10.1038/nrg.2018.4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5990367/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29479082/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Karczewski,%20K.%20J.%20&amp;%20Snyder,%20M.%20P.%20Integrative%20omics%20for%20health%20and%20disease.%20Nat.%20Rev.%20Genet.19,%20299%E2%80%93310%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR46">
<span class="label">46.</span><cite>Pusparum, M., Ertaylan, G. &amp; Thas, O. From population to subject-specific reference intervals. <em>Lecture Notes in Computer Science (including subseries Lecture Notes in Artificial Intelligence and Lecture Notes in Bioinformatics</em>) <strong>12140 LNCS</strong>, 468–482 (2020).</cite>
</li>
<li id="CR47">
<span class="label">47.</span><cite>Pusparum, M., Ertaylan, G. &amp; Thas, O. Individual reference intervals for personalised interpretation of clinical and metabolomics measurements. <em>J. Biomed. Inf.</em><strong>131</strong>, 104111 (2022).</cite> [<a href="https://doi.org/10.1016/j.jbi.2022.104111" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35671939/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Pusparum,%20M.,%20Ertaylan,%20G.%20&amp;%20Thas,%20O.%20Individual%20reference%20intervals%20for%20personalised%20interpretation%20of%20clinical%20and%20metabolomics%20measurements.%20J.%20Biomed.%20Inf.131,%20104111%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR48">
<span class="label">48.</span><cite>Justice, J. et al. Frameworks for Proof-of-Concept Clinical Trials of Interventions That Target Fundamental Aging Processes. <em>J. Gerontol. A Biol. Sci. Med. Sci.</em><strong>71</strong>, 1415–1423 (2016).
</cite> [<a href="https://doi.org/10.1093/gerona/glw126" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5055651/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27535966/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Justice,%20J.%20et%20al.%20Frameworks%20for%20Proof-of-Concept%20Clinical%20Trials%20of%20Interventions%20That%20Target%20Fundamental%20Aging%20Processes.%20J.%20Gerontol.%20A%20Biol.%20Sci.%20Med.%20Sci.71,%201415%E2%80%931423%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR49">
<span class="label">49.</span><cite>Levine, M. E. et al. DNA methylation age of blood predicts future onset of lung cancer in the Women’s Health Initiative. <em>Aging</em><strong>7</strong>, 690–700 (2015).
</cite> [<a href="https://doi.org/10.18632/aging.100809" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4600626/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26411804/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Levine,%20M.%20E.%20et%20al.%20DNA%20methylation%20age%20of%20blood%20predicts%20future%20onset%20of%20lung%20cancer%20in%20the%20Women%E2%80%99s%20Health%20Initiative.%20Aging7,%20690%E2%80%93700%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR50">
<span class="label">50.</span><cite>Kresovich, J. K. et al. Epigenetic mortality predictors and incidence of breast cancer. <em>Aging</em><strong>11</strong>, 11975–11987 (2019).
</cite> [<a href="https://doi.org/10.18632/aging.102523" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6949084/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31848323/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kresovich,%20J.%20K.%20et%20al.%20Epigenetic%20mortality%20predictors%20and%20incidence%20of%20breast%20cancer.%20Aging11,%2011975%E2%80%9311987%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR51">
<span class="label">51.</span><cite>MacNee, W., Rabinovich, R. A. &amp; Choudhury, G. Ageing and the border between health and disease. <em>Eur. Respir. J.</em><strong>44</strong>, 1332–1352 (2014).
</cite> [<a href="https://doi.org/10.1183/09031936.00134014" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25323246/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?MacNee,%20W.,%20Rabinovich,%20R.%20A.%20&amp;%20Choudhury,%20G.%20Ageing%20and%20the%20border%20between%20health%20and%20disease.%20Eur.%20Respir.%20J.44,%201332%E2%80%931352%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR52">
<span class="label">52.</span><cite>Horvath, S. et al. Epigenetic clock for skin and blood cells applied to Hutchinson Gilford Progeria Syndrome and ex vivo studies. <em>Aging</em><strong>10</strong>, 1758–1775 (2018).
</cite> [<a href="https://doi.org/10.18632/aging.101508" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6075434/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30048243/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Horvath,%20S.%20et%20al.%20Epigenetic%20clock%20for%20skin%20and%20blood%20cells%20applied%20to%20Hutchinson%20Gilford%20Progeria%20Syndrome%20and%20ex%20vivo%20studies.%20Aging10,%201758%E2%80%931775%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR53">
<span class="label">53.</span><cite>Horvath, S. DNA methylation age of human tissues and cell types. <em>Genome Biol.</em><strong>14</strong>, R115 (2013).
</cite> [<a href="https://doi.org/10.1186/gb-2013-14-10-r115" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4015143/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24138928/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Horvath,%20S.%20DNA%20methylation%20age%20of%20human%20tissues%20and%20cell%20types.%20Genome%20Biol.14,%20R115%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR54">
<span class="label">54.</span><cite>Levine, M. E. et al. An epigenetic biomarker of aging for lifespan and healthspan. <em>Aging</em><strong>10</strong>, 573–591 (2018).
</cite> [<a href="https://doi.org/10.18632/aging.101414" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5940111/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29676998/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Levine,%20M.%20E.%20et%20al.%20An%20epigenetic%20biomarker%20of%20aging%20for%20lifespan%20and%20healthspan.%20Aging10,%20573%E2%80%93591%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR55">
<span class="label">55.</span><cite>Lu, A. T. et al. DNA methylation GrimAge version 2. <em>Aging</em>10.18632/aging.204434 (2022).</cite> [<a href="https://doi.org/10.18632/aging.204434" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9792204/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36516495/" class="usa-link">PubMed</a>]</li>
<li id="CR56">
<span class="label">56.</span><cite>Belsky, D. W. et al. DunedinPACE, a DNA methylation biomarker of the pace of aging. <em>Elife</em><strong>11</strong>, e73420 (2022).</cite> [<a href="https://doi.org/10.7554/eLife.73420" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8853656/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35029144/" class="usa-link">PubMed</a>]</li>
<li id="CR57">
<span class="label">57.</span><cite>Liu, C. et al. A DNA methylation biomarker of alcohol consumption. <em>Mol. Psychiatry</em><strong>23</strong>, 422–433 (2018).
</cite> [<a href="https://doi.org/10.1038/mp.2016.192" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5575985/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27843151/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Liu,%20C.%20et%20al.%20A%20DNA%20methylation%20biomarker%20of%20alcohol%20consumption.%20Mol.%20Psychiatry23,%20422%E2%80%93433%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR58">
<span class="label">58.</span><cite>Elliott, H. R. et al. Differences in smoking associated DNA methylation patterns in South Asians and Europeans. <em>Clin. Epigenetics</em><strong>6</strong>, 4 (2014).
</cite> [<a href="https://doi.org/10.1186/1868-7083-6-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3915234/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24485148/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Elliott,%20H.%20R.%20et%20al.%20Differences%20in%20smoking%20associated%20DNA%20methylation%20patterns%20in%20South%20Asians%20and%20Europeans.%20Clin.%20Epigenetics6,%204%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR59">
<span class="label">59.</span><cite>Zhang, Y. et al. Smoking-associated DNA methylation markers predict lung cancer incidence. <em>Clin. Epigenetics</em><strong>8</strong>, 127 (2016).
</cite> [<a href="https://doi.org/10.1186/s13148-016-0292-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5123284/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27924164/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhang,%20Y.%20et%20al.%20Smoking-associated%20DNA%20methylation%20markers%20predict%20lung%20cancer%20incidence.%20Clin.%20Epigenetics8,%20127%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR60">
<span class="label">60.</span><cite>Bollepalli, S., Korhonen, T., Kaprio, J., Anders, S. &amp; Ollikainen, M. EpiSmokEr: a robust classifier to determine smoking status from DNA methylation data. <em>Epigenomics</em><strong>11</strong>, 1469–1486 (2019).
</cite> [<a href="https://doi.org/10.2217/epi-2019-0206" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31466478/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bollepalli,%20S.,%20Korhonen,%20T.,%20Kaprio,%20J.,%20Anders,%20S.%20&amp;%20Ollikainen,%20M.%20EpiSmokEr:%20a%20robust%20classifier%20to%20determine%20smoking%20status%20from%20DNA%20methylation%20data.%20Epigenomics11,%201469%E2%80%931486%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR61">
<span class="label">61.</span><cite>Ying, K. et al. Causality-enriched epigenetic age uncouples damage and adaptation. <em>Nat. Aging</em><strong>4</strong>, 231–246 (2024).
</cite> [<a href="https://doi.org/10.1038/s43587-023-00557-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11070280/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38243142/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ying,%20K.%20et%20al.%20Causality-enriched%20epigenetic%20age%20uncouples%20damage%20and%20adaptation.%20Nat.%20Aging4,%20231%E2%80%93246%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR62">
<span class="label">62.</span><cite>Sadahiro, R. et al. Major surgery induces acute changes in measured DNA methylation associated with immune response pathways. <em>Sci. Rep.</em><strong>10</strong>, 5743 (2020).
</cite> [<a href="https://doi.org/10.1038/s41598-020-62262-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7113299/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32238836/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sadahiro,%20R.%20et%20al.%20Major%20surgery%20induces%20acute%20changes%20in%20measured%20DNA%20methylation%20associated%20with%20immune%20response%20pathways.%20Sci.%20Rep.10,%205743%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR63">
<span class="label">63.</span><cite>Sehl, M. E. et al. The effects of anti-retroviral therapy on epigenetic age acceleration observed in HIV-1-infected adults. <em>Pathog. Immun.</em><strong>5</strong>, 291 (2020).
</cite> [<a href="https://doi.org/10.20411/pai.v5i1.376" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7815056/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33501399/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sehl,%20M.%20E.%20et%20al.%20The%20effects%20of%20anti-retroviral%20therapy%20on%20epigenetic%20age%20acceleration%20observed%20in%20HIV-1-infected%20adults.%20Pathog.%20Immun.5,%20291%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR64">
<span class="label">64.</span><cite>Corley, M. J. et al. Genome-wide DNA methylation profiling of peripheral blood reveals an epigenetic signature associated with severe COVID-19. <em>J. Leukoc. Biol.</em><strong>110</strong>, 21–26 (2021).
</cite> [<a href="https://doi.org/10.1002/JLB.5HI0720-466R" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8013321/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33464637/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Corley,%20M.%20J.%20et%20al.%20Genome-wide%20DNA%20methylation%20profiling%20of%20peripheral%20blood%20reveals%20an%20epigenetic%20signature%20associated%20with%20severe%20COVID-19.%20J.%20Leukoc.%20Biol.110,%2021%E2%80%9326%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR65">
<span class="label">65.</span><cite>Dugué, P.-A. et al. Biological aging measures based on blood DNA methylation and risk of cancer: a prospective study. <em>JNCI Cancer Spectr.</em><strong>5</strong>, pkaa109 (2021).</cite> [<a href="https://doi.org/10.1093/jncics/pkaa109" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7791618/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33442664/" class="usa-link">PubMed</a>]</li>
<li id="CR66">
<span class="label">66.</span><cite>Aryee, M. J. et al. Minfi: a flexible and comprehensive Bioconductor package for the analysis of Infinium DNA methylation microarrays. <em>Bioinformatics</em><strong>30</strong>, 1363–1369 (2014).
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btu049" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4016708/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24478339/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Aryee,%20M.%20J.%20et%20al.%20Minfi:%20a%20flexible%20and%20comprehensive%20Bioconductor%20package%20for%20the%20analysis%20of%20Infinium%20DNA%20methylation%20microarrays.%20Bioinformatics30,%201363%E2%80%931369%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR67">
<span class="label">67.</span><cite>Reinius, L. E. et al. Differential DNA methylation in purified human blood cells: implications for cell lineage and studies on disease susceptibility. <em>PLoS ONE</em><strong>7</strong>, e41361 (2012).
</cite> [<a href="https://doi.org/10.1371/journal.pone.0041361" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3405143/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22848472/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Reinius,%20L.%20E.%20et%20al.%20Differential%20DNA%20methylation%20in%20purified%20human%20blood%20cells:%20implications%20for%20cell%20lineage%20and%20studies%20on%20disease%20susceptibility.%20PLoS%20ONE7,%20e41361%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR68">
<span class="label">68.</span><cite>Salas, L. A. et al. An optimized library for reference-based deconvolution of whole-blood biospecimens assayed using the Illumina HumanMethylationEPIC BeadArray. <em>Genome Biol.</em><strong>19</strong>, 64 (2018).
</cite> [<a href="https://doi.org/10.1186/s13059-018-1448-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5975716/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29843789/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Salas,%20L.%20A.%20et%20al.%20An%20optimized%20library%20for%20reference-based%20deconvolution%20of%20whole-blood%20biospecimens%20assayed%20using%20the%20Illumina%20HumanMethylationEPIC%20BeadArray.%20Genome%20Biol.19,%2064%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR69">
<span class="label">69.</span><cite>Berkelmans, G. F. N. et al. Population median imputation was noninferior to complex approaches for imputing missing values in cardiovascular prediction models in clinical practice. <em>J. Clin. Epidemiol.</em><strong>145</strong>, 70–80 (2022).
</cite> [<a href="https://doi.org/10.1016/j.jclinepi.2022.01.011" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35066115/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Berkelmans,%20G.%20F.%20N.%20et%20al.%20Population%20median%20imputation%20was%20noninferior%20to%20complex%20approaches%20for%20imputing%20missing%20values%20in%20cardiovascular%20prediction%20models%20in%20clinical%20practice.%20J.%20Clin.%20Epidemiol.145,%2070%E2%80%9380%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR70">
<span class="label">70.</span><cite>Olink Proteomics. <em>Data Normalization and Standardization</em>. <a href="https://www.olink.com/content/uploads/2021/09/olink-data-normalization-white-paper-v2.0.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.olink.com/content/uploads/2021/09/olink-data-normalization-white-paper-v2.0.pdf</a> (2021).</cite>
</li>
<li id="CR71">
<span class="label">71.</span><cite>Hao, M. et al. Gompertz law‐based biological age (GOLD BioAge): A simple and practical measurement of biological ageing to capture morbidity and mortality risks. <em>Adv. Sci.</em> e01765 (2025).</cite> [<a href="https://doi.org/10.1002/advs.202501765" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40600465/" class="usa-link">PubMed</a>]</li>
<li id="CR72">
<span class="label">72.</span><cite>Bastian, M., Heymann, S. &amp; Jacomy, M. Gephi: an open source software for exploring and manipulating networks. In <em>Proc.</em><em>International AAAI Conference on Web and Social Media,</em><strong>3</strong>, 361–362 (<em>AAAI Press,</em> 2009).</cite>
</li>
<li id="CR73">
<span class="label">73.</span><cite>Shannon, P. et al. Cytoscape: a software environment for integrated models of biomolecular interaction networks. <em>Genome Res.</em><strong>13</strong>, 2498–2504 (2003).
</cite> [<a href="https://doi.org/10.1101/gr.1239303" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC403769/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/14597658/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Shannon,%20P.%20et%20al.%20Cytoscape:%20a%20software%20environment%20for%20integrated%20models%20of%20biomolecular%20interaction%20networks.%20Genome%20Res.13,%202498%E2%80%932504%20(2003)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12371080/bin/41746_2025_1911_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary document_v2</a><sup> (1.3MB, pdf) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>Due to participants’ privacy, the data is available upon request to the I AM Frontier Project Data Access Committee for further research (DataAccess.IAF@vito.be). The sensitive nature of the data does not allow it to be deposited into public repositories. We welcome collaboration with other research groups focused on personalised prevention, offering data sharing opportunities to advance research and improve healthcare outcomes.</p>
<p>The codes are available upon request from the corresponding authors.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from NPJ Digital Medicine are provided here courtesy of <strong>Nature Publishing Group</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1038/s41746-025-01911-9"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/41746_2025_Article_1911.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (1.3 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12371080/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12371080/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12371080%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371080/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12371080/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12371080/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40841468/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12371080/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40841468/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12371080/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12371080/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="3xHhRGr78z1fECN0yizvB9JSpDvggYD8OoyetlnsUdKg7lWXllaXq3L7jsZw91r1">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
        <script type="module" crossorigin="" src="/static/assets/math-574fdcc6.js"></script>
    
    

    </body>
</html>
