
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Phenome‐to‐genome insights for evaluating root system architecture in field studies of maize - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="D2E484818AEF0D7305848100480D9CB4.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="blackwellopen">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12375851/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="The Plant Genome">
<meta name="citation_title" content="Phenome‐to‐genome insights for evaluating root system architecture in field studies of maize">
<meta name="citation_author" content="Kirsten M Hein">
<meta name="citation_author_institution" content="Department of Soil and Crop Sciences, Colorado State University, Fort Collins, Colorado, USA">
<meta name="citation_author_institution" content="Donald Danforth Plant Science Center, Saint Louis, Missouri, USA">
<meta name="citation_author" content="Alexander E Liu">
<meta name="citation_author_institution" content="Donald Danforth Plant Science Center, Saint Louis, Missouri, USA">
<meta name="citation_author" content="Jack L Mullen">
<meta name="citation_author_institution" content="Department of Soil and Crop Sciences, Colorado State University, Fort Collins, Colorado, USA">
<meta name="citation_author" content="Mon‐Ray Shao">
<meta name="citation_author_institution" content="Donald Danforth Plant Science Center, Saint Louis, Missouri, USA">
<meta name="citation_author_institution" content="Dümmen Orange, De Lier, The Netherlands">
<meta name="citation_author" content="Christopher N Topp">
<meta name="citation_author_institution" content="Donald Danforth Plant Science Center, Saint Louis, Missouri, USA">
<meta name="citation_author" content="John K McKay">
<meta name="citation_author_institution" content="Department of Soil and Crop Sciences, Colorado State University, Fort Collins, Colorado, USA">
<meta name="citation_publication_date" content="2025 Aug 24">
<meta name="citation_volume" content="18">
<meta name="citation_issue" content="3">
<meta name="citation_firstpage" content="e70100">
<meta name="citation_doi" content="10.1002/tpg2.70100">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12375851/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12375851/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12375851/pdf/TPG2-18-e70100.pdf">
<meta name="description" content="Understanding the genetic basis of root system architecture (RSA) in crops requires innovative approaches that enable both high‐throughput and precise phenotyping in field conditions. In this study, we evaluated multiple phenotyping and analytical ...">
<meta name="og:title" content="Phenome‐to‐genome insights for evaluating root system architecture in field studies of maize">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Understanding the genetic basis of root system architecture (RSA) in crops requires innovative approaches that enable both high‐throughput and precise phenotyping in field conditions. In this study, we evaluated multiple phenotyping and analytical ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12375851/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12375851">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1002/tpg2.70100"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/TPG2-18-e70100.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12375851%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12375851/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12375851/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12375851/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-blackwellopen.png" alt="Wiley Open Access Collection logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Wiley Open Access Collection" title="Link to Wiley Open Access Collection" shape="default" href="https://doi.org/10.1002/tpg2.70100" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Plant Genome</button></div>. 2025 Aug 24;18(3):e70100. doi: <a href="https://doi.org/10.1002/tpg2.70100" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1002/tpg2.70100</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Plant%20Genome%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Plant%20Genome%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Plant%20Genome%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Plant%20Genome%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Phenome‐to‐genome insights for evaluating root system architecture in field studies of maize</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hein%20KM%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Kirsten M Hein</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Kirsten M Hein</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Department of Soil and Crop Sciences, Colorado State University, Fort Collins, Colorado, USA</div>
<div class="p">
<sup>
<sup>2</sup>
</sup>Donald Danforth Plant Science Center, Saint Louis, Missouri, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hein%20KM%22%5BAuthor%5D" class="usa-link"><span class="name western">Kirsten M Hein</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Liu%20AE%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Alexander E Liu</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Alexander E Liu</span></h3>
<div class="p">
<sup>
<sup>2</sup>
</sup>Donald Danforth Plant Science Center, Saint Louis, Missouri, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Liu%20AE%22%5BAuthor%5D" class="usa-link"><span class="name western">Alexander E Liu</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mullen%20JL%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Jack L Mullen</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Jack L Mullen</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Department of Soil and Crop Sciences, Colorado State University, Fort Collins, Colorado, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mullen%20JL%22%5BAuthor%5D" class="usa-link"><span class="name western">Jack L Mullen</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Shao%20MR%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Mon‐Ray Shao</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Mon‐Ray Shao</span></h3>
<div class="p">
<sup>
<sup>2</sup>
</sup>Donald Danforth Plant Science Center, Saint Louis, Missouri, USA</div>
<div class="p">
<sup>
<sup>3</sup>
</sup>Dümmen Orange, De Lier, The Netherlands</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Shao%20MR%22%5BAuthor%5D" class="usa-link"><span class="name western">Mon‐Ray Shao</span></a>
</div>
</div>
<sup>2,</sup><sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Topp%20CN%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Christopher N Topp</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Christopher N Topp</span></h3>
<div class="p">
<sup>
<sup>2</sup>
</sup>Donald Danforth Plant Science Center, Saint Louis, Missouri, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Topp%20CN%22%5BAuthor%5D" class="usa-link"><span class="name western">Christopher N Topp</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22McKay%20JK%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">John K McKay</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">John K McKay</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Department of Soil and Crop Sciences, Colorado State University, Fort Collins, Colorado, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22McKay%20JK%22%5BAuthor%5D" class="usa-link"><span class="name western">John K McKay</span></a>
</div>
</div>
<sup>1</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="tpg270100-aff-0001">
<sup>
<sup>1</sup>
</sup>Department of Soil and Crop Sciences, Colorado State University, Fort Collins, Colorado, USA</div>
<div id="tpg270100-aff-0002">
<sup>
<sup>2</sup>
</sup>Donald Danforth Plant Science Center, Saint Louis, Missouri, USA</div>
<div id="tpg270100-aff-0003">
<sup>
<sup>3</sup>
</sup>Dümmen Orange, De Lier, The Netherlands</div>
<div class="author-notes p">
<div class="fn" id="correspondenceTo">
<sup>*</sup><p class="display-inline">

<strong>Correspondence</strong>
, Kirsten M. Hein, Donald Danforth Plant Science Center, 975 North Warson Rd, Saint Louis, Missouri, USA. Email: <span>khein@danforthcenter.org</span>

</p>
</div>
<div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Revised 2025 Jun 6; Received 2024 Dec 28; Accepted 2025 Jul 15; Issue date 2025 Sep.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© 2025 The Author(s). <em>The Plant Genome</em> published by Wiley Periodicals LLC on behalf of Crop Science Society of America.</div>
<p>This is an open access article under the terms of the <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by/4.0/</a> License, which permits use, distribution and reproduction in any medium, provided the original work is properly cited.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12375851</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>Abstract</h2>
<p>Understanding the genetic basis of root system architecture (RSA) in crops requires innovative approaches that enable both high‐throughput and precise phenotyping in field conditions. In this study, we evaluated multiple phenotyping and analytical frameworks for quantifying RSA in mature, field‐grown maize in three field experiments. We used forward and reverse genetic approaches to evaluate &gt;1700 maize root crowns, including a diversity panel, a biparental mapping population, and maize mutant and wild‐type alleles at two known RSA genes, <em>DEEPER ROOTING 1</em> (<em>DRO1</em>) and <em>Rootless1</em> (<em>Rt1</em>). We show the utility of increasing the dimensionality of traditional two‐dimensional (2D) techniques, referred to as the “2D multi‐view” method, to improve the capture of whole root system information for mapping genetic variation influencing RSA. Comparison of univariate and multivariate genome‐wide association study (GWAS) approaches revealed that multivariate traits were effective at dissecting complex RSA phenotypes and identifying pleiotropic quantitative trait loci (QTLs). Overall, three‐dimensional (3D) root models generated from X‐ray computed tomography and digital phenotyping captured a larger proportion of RSA trait variations compared to other methods of root phenotyping, as evidenced by both genome‐wide and single‐gene analyses. Among the individual root traits, root pulling force emerged as a highly heritable estimate of RSA that identified the largest number of shared QTLs with 3D phenotypes. Our study shows that integrating complementary phenotyping technologies helps to provide a more comprehensive understanding of the genetic architecture of RSA in field‐grown maize.</p></section><section class="abstract" id="abstract2"><h2>Core Ideas</h2>
<ul id="tpg270100-list-10001" class="list" style="list-style-type:disc">
<li><p>Complementary phenotyping approaches are powerful for studying the genetic basis of root system architecture.</p></li>
<li><p>Multi‐view imaging enhances root trait estimation in two‐dimensional root crown phenotyping for genotype–phenotype mapping.</p></li>
<li><p>Multivariate approaches effectively capture complex root traits and their underlying genetic architecture.</p></li>
<li><p>Three‐dimensional phenotyping offers detailed insights to root crown analysis for forward and reverse genetic studies.</p></li>
<li><p>Root pulling force is highly heritable and cost‐effective for large‐scale root crown field phenotyping.</p></li>
</ul></section><section class="abstract" id="abstract3"><h2>Plain Language Summary</h2>
<p>Understanding the genetic and molecular mechanisms that regulate root system growth and function in plants is crucial for enhancing crop resilience to environmental stress and for promoting sustainable agriculture. However, the investigation of complex root system dynamics under field conditions remains limited by the challenges of large‐scale sampling and precise phenotyping. This study used forward and reverse genetics in three maize field experiments to assess different root phenotyping methods in capturing features of root crown phenome and identifying genotype–phenotype relationships. High‐dimensional phenotyping, including two‐dimensional multi‐view and three‐dimensional image analysis, combined with multivariate analysis, offered a more detailed understanding of the complex trait relationships and genetic architecture underlying maize root crowns. These findings underscore the value of integrating complementary approaches to bridge the phenotype‐to‐genotype gap in field studies of belowground traits.</p></section><hr class="headless">
<h3>Abbreviations</h3>
<dl class="def-list" id="tpg270100-dl-0001">
<dt>BLUP</dt>
<dd><p>best linear unbiased predictor</p></dd>
<dt>DIRT</dt>
<dd><p>digital imaging of root traits</p></dd>
<dt>
<em>DRO1</em>
</dt>
<dd><p>
<em>DEEPER ROOTING 1</em>
</p></dd>
<dt>FarmCPU</dt>
<dd><p>fixed and random model circulating probability</p></dd>
<dt>GWAS</dt>
<dd><p>genome‐wide association study</p></dd>
<dt>MANOVA</dt>
<dd><p>multivariate analysis of variance</p></dd>
<dt>NAM</dt>
<dd><p>nested association mapping</p></dd>
<dt>PCA</dt>
<dd><p>principal component analysis</p></dd>
<dt>PVE</dt>
<dd><p>phenotypic variance explained</p></dd>
<dt>QTL</dt>
<dd><p>quantitative trait locus</p></dd>
<dt>RIL</dt>
<dd><p>recombinant inbred line</p></dd>
<dt>RPF</dt>
<dd><p>root pulling force</p></dd>
<dt>RSA</dt>
<dd><p>root system architecture</p></dd>
<dt>
<em>Rt1</em>
</dt>
<dd><p>
<em>Rootless1</em>
</p></dd>
<dt>SAM</dt>
<dd><p>shoot apical meristem</p></dd>
<dt>SNP</dt>
<dd><p>single‐nucleotide polymorphism</p></dd>
<dt>SCD</dt>
<dd><p>steep, cheap, and deep</p></dd>
<dt>XRT</dt>
<dd><p>X‐ray tomography</p></dd>
</dl>
<section id="tpg270100-sec-0010"><h2 class="pmc_sec_title">1. INTRODUCTION</h2>
<p>Root systems play critical roles in plant acclimation by modulating the timing and location of growth in response to water and nutrient availability. Genetic variation in root systems is also important for adaptation across species, germplasm, and environments. Such genetic variation in root traits can be utilized to achieve breeding targets for enhancing crop resiliency. Despite their importance in plant development and fitness, the quantitative genetic control of root traits, particularly in field‐grown crops, remains underexplored. This gap in understanding is primarily due to the difficulty of obtaining high‐quality measurements from hundreds or thousands of samples. With the emergence of various high‐throughput root phenotyping technologies in field environments, we investigate the relative value of these methods to expand our current understanding of the overall root system and its underlying genetic architecture.</p>
<p>Root system architecture (RSA) is a term that describes the spatial organization of root systems, integrating developmental, anatomical, and topological aspects of root–environment interactions (Fitter, <a href="#tpg270100-bib-0020" class="usa-link" aria-describedby="tpg270100-bib-0020">1987</a>). Concerted efforts have been made to develop standardized terminology and classification schemes for root systems, led by the International Society of Root Research and Plant Ontology Consortium (Feix et al., <a href="#tpg270100-bib-0016" class="usa-link" aria-describedby="tpg270100-bib-0016">2002</a>; Ilic et al., <a href="#tpg270100-bib-0034" class="usa-link" aria-describedby="tpg270100-bib-0034">2007</a>; Zobel &amp; Waisel, <a href="#tpg270100-bib-0099" class="usa-link" aria-describedby="tpg270100-bib-0099">2010</a>). These classifications provide precise descriptions of root function and structure, accommodating the inherent complexity and plasticity of root systems, across species and environments. RSA demonstrates notable plasticity in response to environmental factors such as nutrient and water availability, soil composition, and rhizosphere biotic interactions (Clarke et al., <a href="#tpg270100-bib-0011" class="usa-link" aria-describedby="tpg270100-bib-0011">2019</a>; Johnson &amp; Rasmann, <a href="#tpg270100-bib-0037" class="usa-link" aria-describedby="tpg270100-bib-0037">2015</a>; Karlova et al., <a href="#tpg270100-bib-0040" class="usa-link" aria-describedby="tpg270100-bib-0040">2021</a>; Lareen et al., <a href="#tpg270100-bib-0045" class="usa-link" aria-describedby="tpg270100-bib-0045">2016</a>). The regulation of specific root traits, such as root angle and crown root number, has been shown to improve mobile and immobile nutrient acquisition in response to different environmental scenarios (Bonser et al., <a href="#tpg270100-bib-0003" class="usa-link" aria-describedby="tpg270100-bib-0003">1996</a>; de Jong et al., <a href="#tpg270100-bib-0013" class="usa-link" aria-describedby="tpg270100-bib-0013">2019</a>; Gaudin et al., <a href="#tpg270100-bib-0023" class="usa-link" aria-describedby="tpg270100-bib-0023">2011</a>; J. P. Lynch &amp; Brown, <a href="#tpg270100-bib-0056" class="usa-link" aria-describedby="tpg270100-bib-0056">2001</a>; Saengwilai et al., <a href="#tpg270100-bib-0073" class="usa-link" aria-describedby="tpg270100-bib-0073">2014</a>; Sun et al., <a href="#tpg270100-bib-0079" class="usa-link" aria-describedby="tpg270100-bib-0079">2018</a>; Trachsel et al., <a href="#tpg270100-bib-0085" class="usa-link" aria-describedby="tpg270100-bib-0085">2013</a>; York &amp; Lynch, <a href="#tpg270100-bib-0097" class="usa-link" aria-describedby="tpg270100-bib-0097">2015</a>). By integrating systematic classifications with ongoing research, our understanding of RSA can facilitate innovative breeding strategies that optimize plant productivity and adaptation to specific production environments and management (Schneider &amp; Lynch, <a href="#tpg270100-bib-0076" class="usa-link" aria-describedby="tpg270100-bib-0076">2020</a>).</p>
<p>Root crowns are the nexus of the entire root system and are therefore information‐rich targets of RSA investigations. Advances in phenotyping technologies and their application in field studies have enabled detailed measurements of complex traits across time and space (e.g., Bucksch et al., <a href="#tpg270100-bib-0006" class="usa-link" aria-describedby="tpg270100-bib-0006">2014</a>; Landi et al., <a href="#tpg270100-bib-0043" class="usa-link" aria-describedby="tpg270100-bib-0043">2010</a>; J. P. Lynch et al., <a href="#tpg270100-bib-0057" class="usa-link" aria-describedby="tpg270100-bib-0057">2014</a>; Rajurkar et al., <a href="#tpg270100-bib-0069" class="usa-link" aria-describedby="tpg270100-bib-0069">2022</a>; Shao et al., <a href="#tpg270100-bib-0077" class="usa-link" aria-describedby="tpg270100-bib-0077">2021</a>; Woods et al., <a href="#tpg270100-bib-0095" class="usa-link" aria-describedby="tpg270100-bib-0095">2022</a>). Historically, field studies have relied on manual scoring methods to assess root crown architecture, such as Shovelomics (Trachsel et al., <a href="#tpg270100-bib-0084" class="usa-link" aria-describedby="tpg270100-bib-0084">2011</a>) and root pulling force (RPF) (Landi et al., <a href="#tpg270100-bib-0044" class="usa-link" aria-describedby="tpg270100-bib-0044">2002</a>; Ortman et al., <a href="#tpg270100-bib-0064" class="usa-link" aria-describedby="tpg270100-bib-0064">1968</a>; Woods et al., <a href="#tpg270100-bib-0095" class="usa-link" aria-describedby="tpg270100-bib-0095">2022</a>). RPF quantifies the vertical force required to extract root crowns from the soil and has recently been scaled for higher throughput applications through mechanization (McKay et al., <a href="#tpg270100-bib-0061" class="usa-link" aria-describedby="tpg270100-bib-0061">2023</a>). Semiautomated imaging and analysis pipelines, including optical two‐dimensional (2D), and optical and X‐ray‐based three‐dimensional (3D), have enhanced the data quality obtained from mature root crowns, providing insights into root morphology and function (Bray &amp; Topp, <a href="#tpg270100-bib-0004" class="usa-link" aria-describedby="tpg270100-bib-0004">2018</a>; Das et al., <a href="#tpg270100-bib-0012" class="usa-link" aria-describedby="tpg270100-bib-0012">2015</a>; Jiang et al., <a href="#tpg270100-bib-0036" class="usa-link" aria-describedby="tpg270100-bib-0036">2019</a>; S. Liu et al., <a href="#tpg270100-bib-0052" class="usa-link" aria-describedby="tpg270100-bib-0052">2021</a>; Lobet, <a href="#tpg270100-bib-0054" class="usa-link" aria-describedby="tpg270100-bib-0054">2017</a>). However, these methods present trade‐offs between capturing phenotypic variation under agronomically relevant conditions and obtaining high‐throughput detailed data (York &amp; Lobet, <a href="#tpg270100-bib-0096" class="usa-link" aria-describedby="tpg270100-bib-0096">2017</a>). For example, X‐ray tomography (XRT) provides detailed in situ or post‐extraction insights into plant structure‐function relationships that most other root phenotyping methods cannot achieve; however, its application is limited by higher resource costs and lower throughput (Duncan et al., <a href="#tpg270100-bib-0015" class="usa-link" aria-describedby="tpg270100-bib-0015">2019</a>; Helliwell et al., <a href="#tpg270100-bib-0026" class="usa-link" aria-describedby="tpg270100-bib-0026">2017</a>; Ju et al., <a href="#tpg270100-bib-0038" class="usa-link" aria-describedby="tpg270100-bib-0038">2024</a>; Mooney et al., <a href="#tpg270100-bib-0062" class="usa-link" aria-describedby="tpg270100-bib-0062">2012</a>; Zeng et al., <a href="#tpg270100-bib-0098" class="usa-link" aria-describedby="tpg270100-bib-0098">2021</a>). While numerous methods exist for measuring RSA, no single technique captures the full phenotypic complexity of the root crown architecture. Therefore, we hypothesize that integrating multiple phenotyping technologies can improve genotype–phenotype mapping in field studies of maize root crowns, enabling more robust analyses of complex traits in agronomically relevant conditions.</p>
<p>RSA is a complex trait controlled by numerous genetic and environmental factors (Cai et al., <a href="#tpg270100-bib-0007" class="usa-link" aria-describedby="tpg270100-bib-0007">2012</a>; Champoux et al., <a href="#tpg270100-bib-0009" class="usa-link" aria-describedby="tpg270100-bib-0009">1995</a>; Rogers &amp; Benfey, <a href="#tpg270100-bib-0072" class="usa-link" aria-describedby="tpg270100-bib-0072">2015</a>; Stuber, <a href="#tpg270100-bib-0078" class="usa-link" aria-describedby="tpg270100-bib-0078">1995</a>). In rice, two genes have been identified as major contributors to phenotypic variation in RSA: <em>DEEPER ROOTING 1</em> (<em>DRO1</em>; Uga et al., <a href="#tpg270100-bib-0090" class="usa-link" aria-describedby="tpg270100-bib-0090">2013</a>), which regulates root gravitropism, resulting in steeper growth angles and deeper root systems characteristic of the “steep, cheap, and deep” (SCD; J. P. Lynch, <a href="#tpg270100-bib-0055" class="usa-link" aria-describedby="tpg270100-bib-0055">2013</a>) ideotype, and <em>Phosphorus‐Starvation Tolerance 1</em> (Gamuyao et al., <a href="#tpg270100-bib-0022" class="usa-link" aria-describedby="tpg270100-bib-0022">2012</a>), which promotes early crown root formation under low phosphorus conditions. In maize, several genes influencing abnormalities in root development have been cloned; however, these studies have largely focused on seedlings, and their effects on mature root systems remain poorly understood (reviewed thoroughly by Feix et al., <a href="#tpg270100-bib-0016" class="usa-link" aria-describedby="tpg270100-bib-0016">2002</a> and Hochholdinger et al., <a href="#tpg270100-bib-0030" class="usa-link" aria-describedby="tpg270100-bib-0030">2018</a>). While numerous RSA QTLs have been identified in mature maize through field studies, only a small number of these have been cloned. These include <em>Rootless1</em> (<em>Rt1</em>), which has a large effect on shoot‐borne and lateral root development across growth stages (Jenkins, <a href="#tpg270100-bib-0035" class="usa-link" aria-describedby="tpg270100-bib-0035">1930</a>), as well as the root‐expressed <em>CBL‐interacting serine/threonine‐protein kinase 15</em> (<em>ZmCIPK15</em>; Schneider et al., <a href="#tpg270100-bib-0075" class="usa-link" aria-describedby="tpg270100-bib-0075">2022</a>) and auxin‐related genes <em>ZmRSA3.1</em> and <em>ZmRSA3.2</em> (Ren et al., <a href="#tpg270100-bib-0070" class="usa-link" aria-describedby="tpg270100-bib-0070">2022</a>), which regulate rooting angle and depth. This study builds upon previous findings by providing detailed field annotations of mutant phenotypes in mature maize root crowns. Specifically, we examine allelic variants of <em>DRO1</em> (“<em>dro1‐1</em>”) and <em>Rt1</em> (“<em>rt1‐2</em>”; Ju et al., <a href="#tpg270100-bib-0038" class="usa-link" aria-describedby="tpg270100-bib-0038">2024</a>).</p>
<p>Bridging the gap between the identified RSA QTLs and functionally validated genes from field studies remains challenging, primarily because of the difficulty in capturing the full phenotypic space. In this study, we systematically evaluated phenome‐to‐genome variation in mature maize root crowns to assess how different field phenotyping approaches quantitatively capture RSA and to advance our understanding of its genetic architecture through genome‐wide and single‐gene analyses. To achieve this, we employed a diverse set of phenotyping approaches in three independent field experiments, including manual measurements of root biomass and RPF, as well as measurements of RSA obtained through 2D and 3D imaging and feature extraction. We hypothesized that employing multi‐view 2D and 3D imaging techniques will enhance the resolution and accuracy of extracting RSA information from field‐excavated root crowns, thereby improving the detection and characterization of key RSA QTL. Through this integrative approach, we aimed to provide insights into the use of high‐throughput root phenotyping technologies to survey root crown architecture for future field studies.</p>
<section class="bt xbox font-sm" id="boxed-text1"><section id="tpg270100-sec-0020"><h3 class="pmc_sec_title">Core Ideas</h3>
<ul id="tpg270100-list-0001" class="list" style="list-style-type:disc">
<li><p>Complementary phenotyping approaches are powerful for studying the genetic basis of root system architecture.</p></li>
<li><p>Multi‐view imaging enhances root trait estimation in two‐dimensional root crown phenotyping for genotype–phenotype mapping.</p></li>
<li><p>Multivariate approaches effectively capture complex root traits and their underlying genetic architecture.</p></li>
<li><p>Three‐dimensional phenotyping offers detailed insights to root crown analysis for forward and reverse genetic studies.</p></li>
<li><p>Root pulling force is highly heritable and cost‐effective for large‐scale root crown field phenotyping.</p></li>
</ul></section></section></section><section id="tpg270100-sec-0030"><h2 class="pmc_sec_title">2. MATERIALS AND METHODS</h2>
<section id="tpg270100-sec-0040"><h3 class="pmc_sec_title">2.1. Germplasm and experimental design</h3>
<p>We investigated the genetic basis of RSA phenotypes across three field experiments conducted at the Colorado State University Agricultural Research Development and Education Center in Fort Collins, CO (40.649°N, −105.000°W) during the summers of 2019, 2022, and 2023.</p>
<p>In the first experiment (2019), we evaluated quantitative variation in root traits measured in 312 inbred lines from the shoot apical meristem (SAM) maize diversity panel (Leiboff et al., <a href="#tpg270100-bib-0047" class="usa-link" aria-describedby="tpg270100-bib-0047">2015</a>). Seeds were sown on 14 May using a split‐plot design with two treatment levels of factor irrigation: full irrigation (wet) or limited irrigation (dry), each consisting of three field replicates per genotype. Roots were sampled in both irrigation treatments when approximately 30% of the lines reached anthesis, hereafter referred to as “mid‐flowering” at 13 weeks post‐planting. Details of the field experiment have been described previously (Woods et al., <a href="#tpg270100-bib-0095" class="usa-link" aria-describedby="tpg270100-bib-0095">2022</a>).</p>
<p>In the second experiment (2022), we employed a maize population of recombinant inbred lines (RILs) that segregated for root trait variations to compare RSA heritability estimates derived from 2D single‐ and multi‐view imaging and analysis of maize root crowns. The RIL population was obtained from a cross between two nested association mapping (NAM) parental lines, B73 and CML69, as described by Buckler et al. (<a href="#tpg270100-bib-0005" class="usa-link" aria-describedby="tpg270100-bib-0005">2009</a>). Based on soil nutrient analysis, the field was fertilized to recommendations for 200 bu/ac yield, amounting to 53.9 kg/ha monoammonium phosphate and 401.4 kg/ha urea prior to planting. Lines were planted in two‐row, 6.1‐m plots, with 76‐cm row spacing and 22‐cm seed spacing. Seeds were sown on May 19 in duplicate blocks under full irrigation, averaging approximately 2.5 cm per week. These root crowns were sampled prior to anthesis at 11 weeks post‐planting.</p>
<p>In the third experiment (2023), two maize root development mutants and their respective wild‐type allele lines were used to evaluate the effects of single‐gene mutations on quantitative RSA phenotypes. The <em>dro1‐1</em> allele, a <em>Mu</em> transposon insertion in <em>DRO1</em>, was introgressed into the W22 genetic background through a single generation of backcrossing to reduce background mutations (Feng et al., <a href="#tpg270100-bib-0017" class="usa-link" aria-describedby="tpg270100-bib-0017">2022</a>). Similarly, the <em>rt1‐2</em> allele, a <em>Ds</em> insertion in <em>Rt1</em>, was introgressed into the T43 genetic background through one generation of backcrossing (Ju et al., <a href="#tpg270100-bib-0038" class="usa-link" aria-describedby="tpg270100-bib-0038">2024</a>). The resulting plants from each population were self‐pollinated for three generations and selected for mutant and wild‐type alleles homozygous for each gene. For wild‐type comparisons in the field, the original W22 inbred line provided the wild‐type control for <em>dro1‐1</em>, while a wild‐type segregant replaced the original T43 inbred as the control for <em>rt1‐2</em>. Prior to planting the experiment, soil samples were collected, and nutrient levels were quantified. The field was then fertilized according to the recommendations for a targeted yield of 200 bu/ac, amounting to 107.8 kg/ha monoammonium phosphate and 339.7 kg/ha urea. Planting occurred on May 24, with seeds sown at 22‐cm spacing in two‐row, 6.1‐m plots with a row spacing of 76 cm. Throughout the growing season, the plots received approximately 2.5 cm of weekly irrigation and were regularly weeded by hand. Phenotyping for root system size was conducted post‐anthesis at 21 weeks post‐planting.</p></section><section id="tpg270100-sec-0050"><h3 class="pmc_sec_title">2.2. Field phenotyping</h3>
<p>In the 2019 field experiment, maize root crowns were extracted from soil using manual root pulls. During this process, the maximum force required to extract the root system from the ground was measured using handheld force gauges and designated as the RPF (Woods et al., <a href="#tpg270100-bib-0095" class="usa-link" aria-describedby="tpg270100-bib-0095">2022</a>). The throughput of this method was improved in 2022, where root crown excavations were conducted using a custom root pulling device mounted onto a high clearance tractor (McKay et al., <a href="#tpg270100-bib-0061" class="usa-link" aria-describedby="tpg270100-bib-0061">2023</a>). This mechanical device utilizes a vision system to identify maize stalks and securely grasps them by closing a gripper around each stalk and then vertically extracting the plants from the soil. Root systems in 2023 were shoveled out manually. The excavated root systems were washed with water and air‐dried by hanging, yielding a total sample size of <em>N</em>
<sub>SAM</sub> = 1546, <em>N</em>
<sub>RIL</sub> = 193, and <em>N</em>
<sub>MU/WT</sub> = 29 cleaned root crowns for 2019, 2022, and 2023, respectively. Root biomass measurements were taken across all three field experiments after the root crowns were air‐dried to a constant weight.</p></section><section id="tpg270100-sec-0060"><h3 class="pmc_sec_title">2.3. Root imaging and feature extraction</h3>
<p>We performed image‐based analyses of the maize root systems to quantify and compare field‐grown variations in root architectural traits using three imaging pipelines (Figure <a href="#tpg270100-fig-0001" class="usa-link">1</a>). Note that 2D images were captured by suspending the root samples vertically in front of a dark background using a digital single‐lens reflex camera. These images were then processed and analyzed using the 2D digital imaging of root traits (DIRT) analysis software (Das et al., <a href="#tpg270100-bib-0012" class="usa-link" aria-describedby="tpg270100-bib-0012">2015</a>). Note that 3D root imaging and analysis were conducted using XRT and a custom image processing and feature extraction pipeline (Donald Danforth Plant Science Center), as previously described (Shao et al., <a href="#tpg270100-bib-0077" class="usa-link" aria-describedby="tpg270100-bib-0077">2021</a>), referred to here as the root crown analysis pipeline (RCAP). Detailed descriptions of RCAP trait implementations and related resources are available at: <a href="https://github.com/Topp-Roots-Lab/3d-root-crown-analysis-pipeline/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/Topp‐Roots‐Lab/3d‐root‐crown‐analysis‐pipeline/</a>. Finally, we reproduced a cost‐effective multi‐view imaging system, as described in Tovar et al. (<a href="#tpg270100-bib-0083" class="usa-link" aria-describedby="tpg270100-bib-0083">2018</a>), to simultaneously capture the features of root crowns from three angles (2D multi‐view). Raspberry Pi computers controlled the individual 6 MP cameras, which were positioned with a rotation around the root system of 45° for one camera pair and a 90° separation between the second and third cameras, resulting in 135° of rotation between the furthest pair of cameras (Figure <a href="#tpg270100-fig-0001" class="usa-link">1</a>). For detailed information and protocols regarding software and hardware implementation, refer to Tovar et al. (<a href="#tpg270100-bib-0083" class="usa-link" aria-describedby="tpg270100-bib-0083">2018</a>). The 2D multi‐view RSA measurements were obtained through 2D feature extraction of maize root crowns using DIRT image analysis software (Das et al., <a href="#tpg270100-bib-0012" class="usa-link" aria-describedby="tpg270100-bib-0012">2015</a>) and were computed as the aggregate genotype averages based on three rotated viewpoints.</p>
<figure class="fig xbox font-sm" id="tpg270100-fig-0001"><h4 class="obj_head">FIGURE 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8569/12375851/a12a0bb6b6d3/TPG2-18-e70100-g001.jpg" loading="lazy" id="jats-graphic-1" height="249" width="700" alt="FIGURE 1"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270100-fig-0001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Simplified schematic of the relative amount of root crown data captured by each imaging platform. 2D, two‐dimensional; 3D, three‐dimensional; XRT, X‐ray computed tomography.</p></figcaption></figure></section><section id="tpg270100-sec-0070"><h3 class="pmc_sec_title">2.4. Descriptive statistics of phenotypic space</h3>
<p>Statistical analyses were conducted using R version 4.3.0 (R Core Team, <a href="#tpg270100-bib-0068" class="usa-link" aria-describedby="tpg270100-bib-0068">2023</a>) and JMP Pro 18 (JMP Statistical Discovery). To determine which RSA traits exhibited sufficient genetic variation for variant discovery in 2D phenotyping, we analyzed root crowns from the 2022 NAM RIL maize population. Traits derived from 2D single‐view and multi‐view imaging were tested for significantly nonzero genotypic variance components using a linear mixed model with genotype treated as a random effect. Significance was determined using a Wald test (<em>p </em>&lt; 0.05). For traits showing a significant genotypic variance, broad‐sense heritability (<em>H</em>
<sup>2</sup>) was calculated as the proportion of genotypic variance to total phenotypic variance, and estimates were then compared between the single‐view and multi‐view approaches. To further assess the contribution of individual camera perspectives to the multi‐view results, heritability was also estimated separately for each of the three camera perspectives. These results were visualized using R/ggplot2 (Wickham, <a href="#tpg270100-bib-0094" class="usa-link" aria-describedby="tpg270100-bib-0094">2016</a>).</p>
<p>In the 2019 experiment, we examined RSA in 312 inbred lines from the SAM maize diversity panel. Outliers within each treatment and genotype group were identified and excluded using the Bonferroni outlier test in R/car (Fox &amp; Weisberg, <a href="#tpg270100-bib-0021" class="usa-link" aria-describedby="tpg270100-bib-0021">2019</a>). Observations with Bonferroni‐adjusted <em>p</em>‐values &lt; 0.05 were considered mean‐shift outliers. Trait heritability was estimated as the proportion of genotypic variance to total variance using a random effect one‐way analysis of variance model. Specifically, the phenotypic data were partitioned by treatment and then analyzed using the “lmer” function in R/lme4 (Bates et al., <a href="#tpg270100-bib-0001" class="usa-link" aria-describedby="tpg270100-bib-0001">2015</a>), treating genotype as a random effect in the model.</p>
<p>Genotype averages for the maize diversity panel were calculated as best linear unbiased predictors (BLUPs), treating genotype as a random effect within treatments using the R/lme4 package (Bates et al., <a href="#tpg270100-bib-0001" class="usa-link" aria-describedby="tpg270100-bib-0001">2015</a>). Spearman's rank genetic correlations were then computed from the BLUPs using R/cor (R Core Team, <a href="#tpg270100-bib-0068" class="usa-link" aria-describedby="tpg270100-bib-0068">2023</a>). Principal component analysis (PCA) was performed on the remaining trait variables by employing the function “PCA” in R/FactoMineR (Lê et al., <a href="#tpg270100-bib-0046" class="usa-link" aria-describedby="tpg270100-bib-0046">2008</a>). Axis selection was visually determined by analyzing a scree plot of the canonical eigenvalues. Clustering analyses were visualized using R/factoextra software (Kassambara &amp; Mundt, <a href="#tpg270100-bib-0041" class="usa-link" aria-describedby="tpg270100-bib-0041">2020</a>).</p></section><section id="tpg270100-sec-0080"><h3 class="pmc_sec_title">2.5. Quantitative genetic control of RSA in SAM maize genotypes</h3>
<p>We implemented a genome‐wide association study (GWAS) on 312 SAM lines to identify genetic variation associated with root phenotypes and assess how different methods of quantifying RSA captured this variation. Prior to analysis, the phenotypic data were partitioned by irrigation treatment, and BLUP values were subsequently calculated for each trait by treating genotype as a random effect using the “lmer” function in R/lme4 (Bates et al., <a href="#tpg270100-bib-0001" class="usa-link" aria-describedby="tpg270100-bib-0001">2015</a>). These values were used as the univariate GWAS response variables. PCA was then used to generate multivariate RSA traits, referred to as “PCs,” using the scaled BLUP values from each set of imaging traits (2D single‐view and 3D) in R/prcomp (R Core Team, <a href="#tpg270100-bib-0068" class="usa-link" aria-describedby="tpg270100-bib-0068">2023</a>). The data derived from the genotypes on the first two PCA axes were used as GWAS multivariate response variables in the subsequent analyses. These variables were denoted as “2D PC1,” “2D PC2,” “3D PC1,” and “3D PC2” for each environment.</p>
<p>GWAS was conducted on 115 univariate and four multivariate (PC) traits derived from BLUPs (see Table <a href="#tpg270100-supinfo-0001" class="usa-link">S1</a>) under full and limited irrigation, using the fixed and random model circulating probability (FarmCPU) method in GAPIT (Lipka et al., <a href="#tpg270100-bib-0050" class="usa-link" aria-describedby="tpg270100-bib-0050">2012</a>; X. Liu et al., <a href="#tpg270100-bib-0053" class="usa-link" aria-describedby="tpg270100-bib-0053">2016</a>). The initial HapMap genotype matrix for the SAM panel contained 1.2 million single nucleotide polymorphisms (SNPs) (Leiboff et al., <a href="#tpg270100-bib-0047" class="usa-link" aria-describedby="tpg270100-bib-0047">2015</a>), which was refined to 860,000 after imposing a minor allele frequency filter of 5%. The first three principal components (PCs) and a kinship matrix calculated using GAPIT were used as covariates to control for population structure. Only traits with <em>Q–Q</em> plots displaying <em>p</em>‐value distributions that approximated normality were included in the FarmCPU results. SNPs whose significance passed the Benjamini–Hochberg false discovery rate threshold (0.10) were further investigated to determine the presence of co‐located QTL across methods, including the list of 2019 RSA‐associated SNPs previously identified by Woods et al. (<a href="#tpg270100-bib-0095" class="usa-link" aria-describedby="tpg270100-bib-0095">2022</a>).</p>
<p>RSA‐associated SNPs were fitted as random effects to estimate the SNP‐attributed and residual variance components using the R package lme4 v1.1‐34 (Bates et al., <a href="#tpg270100-bib-0001" class="usa-link" aria-describedby="tpg270100-bib-0001">2015</a>). The proportion of phenotypic variance explained (PVE) by each SNP was calculated as the ratio of their corresponding variance to the total variance. Candidate regions of SNP colocalization were identified using 1‐Mb bins within each chromosome. Bin indices were calculated based on the number of complete bins that occur before the current SNP position within each chromosome, using the following formula:
</p>
<table class="disp-formula p" id="tpg270100-disp-0001"><tr><td class="formula"><math id="jats-math-1" display="block"><mrow><mrow><mrow><mi>Bin</mi><mspace width="0.28em"></mspace><mi>Inde</mi></mrow><msub><mi mathvariant="normal">x</mi><mi>Chromsome</mi></msub><mo linebreak="badbreak">=</mo><mfenced separators="" open="|" close="|"><mfrac><mrow><mrow><mi>Position</mi><mspace width="0.28em"></mspace></mrow><mo>−</mo><mspace width="0.28em"></mspace><mn>1</mn></mrow><mrow><mn>1</mn><mspace width="0.28em"></mspace><mi>Mb</mi></mrow></mfrac></mfenced><mo linebreak="goodbreak">+</mo><mn>1</mn><mo>,</mo></mrow></mrow></math></td></tr></table>
<p>where “Position” represents the SNP position mapped to the v2 B73 reference genome, and the bin size in the denominator, set to 1 Mb, is specified in base pairs (bp). The results were visualized using R/ComplexHeatmap, R/factoextra, and R/VennDiagram (Chen, <a href="#tpg270100-bib-0010" class="usa-link" aria-describedby="tpg270100-bib-0010">2022</a>; Gu, <a href="#tpg270100-bib-0024" class="usa-link" aria-describedby="tpg270100-bib-0024">2022</a>; Gu et al., <a href="#tpg270100-bib-0025" class="usa-link" aria-describedby="tpg270100-bib-0025">2016</a>; Kassambara &amp; Mundt, <a href="#tpg270100-bib-0041" class="usa-link" aria-describedby="tpg270100-bib-0041">2020</a>).</p></section><section id="tpg270100-sec-0090"><h3 class="pmc_sec_title">2.6. Evaluation of single‐gene perturbed maize genotypes</h3>
<p>In the 2023 experiment, two maize root development mutants and their corresponding wild‐type allele lines were used to examine how 2D single‐view, 2D multi‐view, and 3D imaging approaches detect the effects of single‐gene mutations on individual RSA phenotypes and patterns of among‐trait covariances. Differences in average phenotype response between mutant and wild‐type alleles were assessed using a two‐sample <em>t</em>‐test in R/t.test (R Core Team, <a href="#tpg270100-bib-0068" class="usa-link" aria-describedby="tpg270100-bib-0068">2023</a>), with a significance level (<em>α</em>) of 0.05.</p>
<p>To estimate mutational covariance in the maize mutant and wild‐type allele lines, we conducted a Pearson correlation analysis using R/cor.test (R Core Team, <a href="#tpg270100-bib-0068" class="usa-link" aria-describedby="tpg270100-bib-0068">2023</a>) to calculate the genetic correlation between RSA phenotypes within and among imaging methods. In addition, we applied a multivariate analysis of variance (MANOVA) to test for pairwise interactions among all trait combinations in individual models. We evaluated the results of the MANOVA models at a significance level (<em>α</em>) of 0.05.</p></section></section><section id="tpg270100-sec-0100"><h2 class="pmc_sec_title">3. RESULTS</h2>
<section id="tpg270100-sec-0110"><h3 class="pmc_sec_title">3.1. Multi‐view 2D imaging enhances detection of heritable variation in maize root crown architecture under field conditions</h3>
<p>In the 2022 CML69 × B73 NAM RIL population, 18 of the 33 RSA traits showed significant genotypic variance (Wald test, <em>p</em> &lt; 0.05) in at least one of the 2D phenotyping approaches. Heritability estimates were generally higher under the multi‐view setup, which also identified a broader range of significant traits, including those associated with root crown area, density, depth, and width (Figure <a href="#tpg270100-fig-0002" class="usa-link">2</a>; Table <a href="#tpg270100-supinfo-0001" class="usa-link">S2</a>). In contrast, half of the traits detected by the single‐view approach were limited to depth‐derived traits, including accumulated width at 10%–60% depth (2D D10‐D60). To better understand the contribution of multiple viewpoints to the multi‐view approach, we compared heritability estimates from each individual camera perspective to those derived from the aggregated multi‐view trait values. The aggregated values used in the 2D multi‐view approach consistently provided better estimates than the individual camera perspectives from that setup, indicating added value for field‐driven RSA variant discovery pipelines by integrating multiple root crown perspectives (Table <a href="#tpg270100-supinfo-0001" class="usa-link">S3</a>). Overall, the 2D multi‐view imaging approach captured a larger proportion of genetic variation underlying a broader array of geometrical and topological root crown traits compared to the single‐view approach.</p>
<figure class="fig xbox font-sm" id="tpg270100-fig-0002"><h4 class="obj_head">FIGURE 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8569/12375851/ee2219482262/TPG2-18-e70100-g006.jpg" loading="lazy" id="jats-graphic-3" height="424" width="760" alt="FIGURE 2"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270100-fig-0002/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Comparison of broad‐sense heritability (<em>H</em>
<sup>2</sup>) between two‐dimensional (2D) single‐view and multi‐view approaches for root traits in the 2022 CML69 × B73 recombinant inbred line (RIL) population. Traits were extracted from root crown images using the DIRT (digital imaging of root traits) software (see Das et al., <a href="#tpg270100-bib-0012" class="usa-link" aria-describedby="tpg270100-bib-0012">2015</a>, for a complete list of trait descriptions). The 18 traits shown were selected based on significant <em>H</em>
<sup>2</sup> estimates in at least one method (Wald test, <em>p</em> &lt; 0.05).</p></figcaption></figure></section><section id="tpg270100-sec-0120"><h3 class="pmc_sec_title">3.2. Belowground phenotypic variation is largely driven by root system size and spread</h3>
<p>We found a broad distribution of genotypic correlations among the different phenotyping methods in the 2019 SAM maize diversity panel (Figure <a href="#tpg270100-supinfo-0001" class="usa-link">S1</a>). RPF showed the greatest positive correlations (<em>r</em> ≤ 0.70) with traits associated with overall root system size, including root biomass, 2D Area, 3D Volume, and fractal dimension side‐view (3D FractalDimensionS). These results were consistent with the findings of Shao et al. (<a href="#tpg270100-bib-0077" class="usa-link" aria-describedby="tpg270100-bib-0077">2021</a>) and Woods et al. (<a href="#tpg270100-bib-0095" class="usa-link" aria-describedby="tpg270100-bib-0095">2022</a>). Groups of related traits (i.e., those with multiple measurements) were both visually and statistically evaluated for removal based on weak overall correlations (<em>|r|</em> ≤ 0.30) and low variability (SD &lt; 0.05) in relationships with traits measured using other phenotyping approaches (see Figure <a href="#tpg270100-supinfo-0001" class="usa-link">S1</a>). For individual traits, we balanced the statistical considerations with biological relevance to maintain the integrity and interpretability of the analysis. The removal process aimed to minimize redundancy while retaining traits that captured unique variation or meaningfully contributed to the relationships between phenotyping approaches, resulting in the selection of 115 RSA traits for downstream analysis.</p>
<p>PCA was used to investigate the phenotypic relationships among the remaining 115 RSA traits (Table <a href="#tpg270100-supinfo-0001" class="usa-link">S4</a>). The first two PCs accounted for 32.30% and 9.90% of the total variation, respectively (Figure <a href="#tpg270100-supinfo-0001" class="usa-link">S2</a>), with no discrete separation among individuals between treatment groups (Figure <a href="#tpg270100-supinfo-0001" class="usa-link">S3</a>). We found a clear separation between the types of RSA traits that were strongly loaded on PC1 compared to PC2 (Figure <a href="#tpg270100-fig-0003" class="usa-link">3A</a>). PC1 predominantly captured variation in overall root system size and distribution, with higher scores positively correlated with 2D Area, 3D ConvexVolume, and 3D biomass distributions (vhist 13–16), indicative of larger root sizes. In contrast, PC2 captured differences in root density (3D Density S2, S6, T2, T4–T6), and in maximum root width (3D HorEqDiameter). Further evidence of these relationships was depicted by Figure <a href="#tpg270100-fig-0003" class="usa-link">3B</a>, which illustrated a sample of the variations in maize root crown size, shape, and distribution through XRT scans of four individuals from genotypes that were representative of each quadrant of the multidimensional space.</p>
<figure class="fig xbox font-sm" id="tpg270100-fig-0003"><h4 class="obj_head">FIGURE 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8569/12375851/bc4406155975/TPG2-18-e70100-g002.jpg" loading="lazy" id="jats-graphic-5" height="1000" width="775" alt="FIGURE 3"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270100-fig-0003/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Relationship of 115 root system architecture (RSA) traits across 312 genotypes from the 2019 shoot apical meristem (SAM) maize panel. (A) Principal component analysis (PCA) biplot of the scaled first two principal components (PCs) for roots excavated at mid‐flowering. Open circles represent individual genotypes, and vectors indicate the top 20 RSA traits with the highest loadings on PC1 and PC2. Highlighted individuals are further illustrated in panel (B). (B) Comparison of maize root crown features through X‐ray computed tomography (XRT) scans of four distinct genotypes, contrasting the physical attributes of genotypes collected of the same replicate from each quadrant (I–IV) in the biplot. 2D, two‐dimensional; 3D, three‐dimensional.</p></figcaption></figure><p>Estimates of <em>H</em>
<sup>2</sup> in the SAM maize lines were generally higher for all evaluated RSA traits under wet conditions than under dry conditions (Figures <a href="#tpg270100-supinfo-0001" class="usa-link">S4</a> and <a href="#tpg270100-supinfo-0001" class="usa-link">S5</a>; Table <a href="#tpg270100-supinfo-0001" class="usa-link">S5</a>). This finding supports the prediction that low‐stress environments can increase the range of observable phenotypic variation, thereby facilitating clearer distinctions between genotypes (reviewed by Visscher et al. [<a href="#tpg270100-bib-0091" class="usa-link" aria-describedby="tpg270100-bib-0091">2008</a>]). Within the wet treatment, heritability estimates for 2D RSA traits highly correlated with PC1 ranged from 0.33 to 0.43, with 2D Area exhibiting the highest estimate. In comparison, 3D RSA traits associated with the same PC1 axis exhibited a broader distribution of heritability ranging from 0.19 to 0.45, with 3D FractalDimensionS showing the highest estimate. For traits associated with PC2, which primarily captured root density characteristics across 2D and 3D RSA traits, heritability ranged from 0.32 to 0.41, with the highest estimate observed for 3D Density T5. Although RPF and root biomass were not strongly correlated with PC1 and PC2, they demonstrated a higher average heritability than the traits obtained through 2D single‐view and 3D imaging (Figures <a href="#tpg270100-supinfo-0001" class="usa-link">S4</a> and <a href="#tpg270100-supinfo-0001" class="usa-link">S5</a>).</p></section><section id="tpg270100-sec-0130"><h3 class="pmc_sec_title">3.3. Multivariate analysis identifies genome‐wide relationships among individual RSA traits</h3>
<p>The FarmCPU GWAS models identified 405 significant SNPs across 288 distinct 1‐Mb bins associated with multivariate (PC) traits and univariate BLUP traits, including 2D single‐view, 3D, and RPF measurements. These findings build on the 22 SNPs previously reported by Woods et al. (<a href="#tpg270100-bib-0095" class="usa-link" aria-describedby="tpg270100-bib-0095">2022</a>), resulting in a cumulative total of 427 significant SNPs across 297 distinct 1‐Mb bins (Figure <a href="#tpg270100-fig-0004" class="usa-link">4</a>; Table <a href="#tpg270100-supinfo-0001" class="usa-link">S6</a>). Incorporating both multivariate and univariate RSA traits as response variables in the GWAS models enabled better detection of trait‐specific and pleiotropic genomic signals. The combined set of significant SNPs were predominantly identified by 3D traits (83.84%), with 3D biomass distributions and 3D convex hull distributions accounting for the majority of these associations. This was followed by the RPF (7.03%), 2D (single‐view; 5.62%), and PC (3.51%) traits. A greater proportion of SNPs associated with individual RSA traits corresponded to RPF (7.03%), 3D Solidity (4.45%), and 3D Density S6 (3.75%). Notably, RPF shared the largest proportion of 1‐Mb bins with SNPs associated with other traits. RPF measurements were moderately heritable in both environments (wet treatment: <em>H</em>
<sup>2</sup> = 0.35; dry treatment: <em>H</em>
<sup>2</sup> = 0.49) and predominantly colocalized with 3D traits that had lower heritability. Specifically, these SNPs were associated with traits representative of root system density, and biomass and convex hull distributions within the upper 7 cm of the root crown, which cumulatively displayed higher values of PVE than RPF (Figure <a href="#tpg270100-supinfo-0001" class="usa-link">S6</a>). Overall, 3D RSA traits accounted for the largest proportion of phenotypic variation and its corresponding genetic loci.</p>
<figure class="fig xbox font-sm" id="tpg270100-fig-0004"><h4 class="obj_head">FIGURE 4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8569/12375851/b57ca12d351e/TPG2-18-e70100-g004.jpg" loading="lazy" id="jats-graphic-7" height="236" width="400" alt="FIGURE 4"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270100-fig-0004/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Venn diagram of genome‐wide association study (GWAS) single‐nucleotide polymorphism (SNP)‐associated 1‐Mb bins partitioned by phenotyping method. Overlapping regions indicate 1‐Mb bins that were enriched in two or more comparisons. 2D, two‐dimensional; 3D, three‐dimensional; RPF, root pulling force.</p></figcaption></figure><p>The use of multivariate RSA traits was instrumental in investigating key phenotype‐to‐genotype relationships. Approximately 60.0% (nine SNPs) of the SNPs associated with the multivariate traits were co‐located with 19 univariate traits. PC1 and PC2 explained substantial phenotypic variation, accounting for approximately 51% of the cumulative variation in 2D multivariate traits and 32% in 3D multivariate traits under both treatments. We decomposed the trait space of the eight multivariate traits to identify phenotypic relationships among genotypes along each principal axis (Table <a href="#tpg270100-supinfo-0001" class="usa-link">S7</a>). Among the 2D multivariate traits, 2D PC1 primarily captured variations in root size and spatial characteristics, such as root area and lateral extent, while 2D PC2 largely described variations in root angle and depth. For the 3D multivariate traits, 3D PC1 represented variations in root distribution, including 3D biomass and convex hull distributions in the lower 50% (vhist 13–16) of the root crown, while 3D PC2 described variations in root density and uniformity. These relationships were captured on the genome‐wide level as SNPs associated with 2D PC1 were identified in multiple 1‐Mb bins that contained SNPs associated with individual 2D single‐view and 3D root density, area, and volume‐related traits. Similarly, the SNPs identified by 3D PC2 colocalized with individual 3D density and solidity traits (Figure <a href="#tpg270100-supinfo-0001" class="usa-link">S6</a>). We estimated the phenotypic correlations between the multivariate traits and the breeding values for root biomass to obtain a biological ground truth for data interpretation and found significant positive correlations for root biomass with 2D PC1 (<em>r</em> = 0.69; <em>p </em>&lt; 0.05) and 3D PC1 (<em>r</em> = 0.67; <em>p </em>&lt; 0.05). In contrast, weaker correlations were observed with 2D PC2 (<em>r</em> = −0.04; <em>p </em>= 0.30) and 3D PC2 (<em>r</em> = −0.28; <em>p </em>&lt; 0.05) (Figure <a href="#tpg270100-fig-0005" class="usa-link">5</a>). These results indicate that the first PC axes for both 2D and 3D RSA traits were strong predictors of belowground biomass.</p>
<figure class="fig xbox font-sm" id="tpg270100-fig-0005"><h4 class="obj_head">FIGURE 5.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8569/12375851/2cce125264c2/TPG2-18-e70100-g003.jpg" loading="lazy" id="jats-graphic-9" height="509" width="800" alt="FIGURE 5"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270100-fig-0005/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Correlation between genotypic values (best linear unbiased predictors [BLUPs]) of two‐dimensional (2D) and three‐dimensional (3D) genome‐wide association study (GWAS) multivariate root traits (PC1 and PC2) and root biomass in the 2019 shoot apical meristem (SAM) maize panel. <em>p</em>, probability value; <em>r</em>, Spearman's rank correlation coefficient. BLUP, best linear unbiased predictor; PC, principal component.</p></figcaption></figure></section><section id="tpg270100-sec-0140"><h3 class="pmc_sec_title">3.4. Two‐dimensional multi‐view and 3D root phenotyping improve resolution in detecting phenotypic effects of two known‐RSA genes in mature maize root crowns</h3>
<p>
<em>Rt1</em> is a classic maize mutation known to reduce the number of lateral and nodal roots (Hochholdinger, <a href="#tpg270100-bib-0028" class="usa-link" aria-describedby="tpg270100-bib-0028">2009</a>; Jenkins, <a href="#tpg270100-bib-0035" class="usa-link" aria-describedby="tpg270100-bib-0035">1930</a>; Taramino &amp; Sakai, <a href="#tpg270100-bib-0080" class="usa-link" aria-describedby="tpg270100-bib-0080">2009</a>). However, the <em>rt1‐2</em> mutant allele has been shown to induce supernumerary nodal roots near the soil line, contrasting with the original <em>Rt1</em> phenotype and resulting in changes to the overall root crown architecture that resemble characteristics of the SCD ideotype (Ju et al., <a href="#tpg270100-bib-0038" class="usa-link" aria-describedby="tpg270100-bib-0038">2024</a>; A. Liu et al., <a href="#tpg270100-bib-0051" class="usa-link" aria-describedby="tpg270100-bib-0051">2023</a>; J. P. Lynch, <a href="#tpg270100-bib-0055" class="usa-link" aria-describedby="tpg270100-bib-0055">2013</a>). In the 2023 field experiment, all three imaging approaches consistently captured the predicted geometrical root crown phenotypes for each allele. The <em>rt1‐2</em> mutation was associated with increased root growth angles and significantly reduced root system area and width‐to‐depth ratios (<em>p</em> ≤ 0.05) compared to wild‐type plants. However, significant differences in genotype averages for root system width were only observed in root crown features extracted from 2D single‐view and 3D imaging (Table <a href="#tpg270100-tbl-0001" class="usa-link">1</a>).</p>
<section class="tw xbox font-sm" id="tpg270100-tbl-0001"><h4 class="obj_head">TABLE 1.</h4>
<div class="caption p"><p>Statistical comparison of average root system architecture (RSA) traits between <em>dro1‐1</em> and <em>rt1‐2</em> maize mutant lines and their respective wild‐type allele lines across imaging approaches. Bold values denote statistical significance at the <em>p</em> ≤ 0.05 level.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col align="left" span="1">
<col align="left" span="1">
<col align="char" char="." span="1">
<col align="char" char="." span="1">
<col align="char" char="." span="1">
<col align="char" char="." span="1">
<col align="char" char="." span="1">
<thead>
<tr><th align="left" colspan="7" style="border-bottom:solid 1px #000000" rowspan="1">
<em>dro1‐1</em>
</th></tr>
<tr style="border-bottom:solid 1px #000000">
<th align="left" rowspan="1" colspan="1">Imaging</th>
<th align="left" rowspan="1" colspan="1">Trait</th>
<th align="left" rowspan="1" colspan="1">Wt</th>
<th align="left" rowspan="1" colspan="1">Mu</th>
<th align="left" rowspan="1" colspan="1">
<em>df</em>
</th>
<th align="left" rowspan="1" colspan="1">
<em>t</em>
</th>
<th align="left" rowspan="1" colspan="1">
<em>p</em>
</th>
</tr>
</thead>
<tbody>
<tr>
<td rowspan="5" align="left" colspan="1">2D single‐view<a href="#tpg270100-tbl1-note-0003" class="usa-link">
<sup>a</sup>
</a>
</td>
<td align="left" rowspan="1" colspan="1">Area</td>
<td align="center" rowspan="1" colspan="1">8568.94</td>
<td align="center" rowspan="1" colspan="1">10,315.35</td>
<td align="center" rowspan="1" colspan="1">8</td>
<td align="center" rowspan="1" colspan="1">−1.69</td>
<td align="center" rowspan="1" colspan="1">0.13</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">SklWidth</td>
<td align="center" rowspan="1" colspan="1">172.33</td>
<td align="center" rowspan="1" colspan="1">186.30</td>
<td align="center" rowspan="1" colspan="1">17</td>
<td align="center" rowspan="1" colspan="1">−0.82</td>
<td align="center" rowspan="1" colspan="1">0.42</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">RTPCount</td>
<td align="center" rowspan="1" colspan="1">450.28</td>
<td align="center" rowspan="1" colspan="1">555.83</td>
<td align="center" rowspan="1" colspan="1">17</td>
<td align="center" rowspan="1" colspan="1">−1.48</td>
<td align="center" rowspan="1" colspan="1">0.16</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">AngTop</td>
<td align="center" rowspan="1" colspan="1">57.13</td>
<td align="center" rowspan="1" colspan="1">45.20</td>
<td align="center" rowspan="1" colspan="1">10</td>
<td align="center" rowspan="1" colspan="1">2.53</td>
<td align="center" rowspan="1" colspan="1">
<strong>0.03</strong>
<a href="#tpg270100-tbl1-note-0005" class="usa-link">
<sup>*</sup>
</a>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">AvgDensity</td>
<td align="center" rowspan="1" colspan="1">2.05</td>
<td align="center" rowspan="1" colspan="1">3.81</td>
<td align="center" rowspan="1" colspan="1">6</td>
<td align="center" rowspan="1" colspan="1">−2.58</td>
<td align="center" rowspan="1" colspan="1">
<strong>0.04</strong>
<a href="#tpg270100-tbl1-note-0005" class="usa-link">
<sup>*</sup>
</a>
</td>
</tr>
<tr>
<td rowspan="5" align="left" colspan="1">2D multi‐view<a href="#tpg270100-tbl1-note-0003" class="usa-link">
<sup>a</sup>
</a>
</td>
<td align="left" rowspan="1" colspan="1">Area</td>
<td align="center" rowspan="1" colspan="1">8568.95</td>
<td align="center" rowspan="1" colspan="1">10,315.34</td>
<td align="center" rowspan="1" colspan="1">8</td>
<td align="center" rowspan="1" colspan="1">−1.69</td>
<td align="center" rowspan="1" colspan="1">0.13</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">SklWidth</td>
<td align="center" rowspan="1" colspan="1">161.12</td>
<td align="center" rowspan="1" colspan="1">184.03</td>
<td align="center" rowspan="1" colspan="1">17</td>
<td align="center" rowspan="1" colspan="1">−2.06</td>
<td align="center" rowspan="1" colspan="1">
<strong>0.05</strong>
<a href="#tpg270100-tbl1-note-0005" class="usa-link">
<sup>*</sup>
</a>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">RTPCount</td>
<td align="center" rowspan="1" colspan="1">111.14</td>
<td align="center" rowspan="1" colspan="1">145.22</td>
<td align="center" rowspan="1" colspan="1">15</td>
<td align="center" rowspan="1" colspan="1">−2.70</td>
<td align="center" rowspan="1" colspan="1">
<strong>0.02</strong>
<a href="#tpg270100-tbl1-note-0005" class="usa-link">
<sup>*</sup>
</a>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">AngTop</td>
<td align="center" rowspan="1" colspan="1">37.01</td>
<td align="center" rowspan="1" colspan="1">41.08</td>
<td align="center" rowspan="1" colspan="1">17</td>
<td align="center" rowspan="1" colspan="1">−0.81</td>
<td align="center" rowspan="1" colspan="1">0.43</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">AvgDensity</td>
<td align="center" rowspan="1" colspan="1">7.66</td>
<td align="center" rowspan="1" colspan="1">8.93</td>
<td align="center" rowspan="1" colspan="1">14</td>
<td align="center" rowspan="1" colspan="1">−0.94</td>
<td align="center" rowspan="1" colspan="1">0.36</td>
</tr>
<tr>
<td rowspan="6" align="left" colspan="1">3D (XRT)<a href="#tpg270100-tbl1-note-0004" class="usa-link">
<sup>b</sup>
</a>
</td>
<td align="left" rowspan="1" colspan="1">HorEqDiameter</td>
<td align="center" rowspan="1" colspan="1">407.81</td>
<td align="center" rowspan="1" colspan="1">521.08</td>
<td align="center" rowspan="1" colspan="1">8</td>
<td align="center" rowspan="1" colspan="1">−3.39</td>
<td align="center" rowspan="1" colspan="1">
<strong>≤0.01</strong>
<a href="#tpg270100-tbl1-note-0005" class="usa-link">
<sup>*</sup>
</a>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">WDRatio</td>
<td align="center" rowspan="1" colspan="1">0.48</td>
<td align="center" rowspan="1" colspan="1">0.66</td>
<td align="center" rowspan="1" colspan="1">6</td>
<td align="center" rowspan="1" colspan="1">−2.72</td>
<td align="center" rowspan="1" colspan="1">
<strong>0.03</strong>
<a href="#tpg270100-tbl1-note-0005" class="usa-link">
<sup>*</sup>
</a>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">TotalLength</td>
<td align="center" rowspan="1" colspan="1">92,214.50</td>
<td align="center" rowspan="1" colspan="1">117,418.30</td>
<td align="center" rowspan="1" colspan="1">9</td>
<td align="center" rowspan="1" colspan="1">−2.32</td>
<td align="center" rowspan="1" colspan="1">
<strong>0.04</strong>
<a href="#tpg270100-tbl1-note-0005" class="usa-link">
<sup>*</sup>
</a>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">LengthDistr</td>
<td align="center" rowspan="1" colspan="1">0.12</td>
<td align="center" rowspan="1" colspan="1">0.14</td>
<td align="center" rowspan="1" colspan="1">6</td>
<td align="center" rowspan="1" colspan="1">−0.35</td>
<td align="center" rowspan="1" colspan="1">0.74</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">NumberTips</td>
<td align="center" rowspan="1" colspan="1">1672.43</td>
<td align="center" rowspan="1" colspan="1">2327.67</td>
<td align="center" rowspan="1" colspan="1">10</td>
<td align="center" rowspan="1" colspan="1">−2.82</td>
<td align="center" rowspan="1" colspan="1">
<strong>0.02</strong>
<a href="#tpg270100-tbl1-note-0005" class="usa-link">
<sup>*</sup>
</a>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Density T2</td>
<td align="center" rowspan="1" colspan="1">0.09</td>
<td align="center" rowspan="1" colspan="1">0.10</td>
<td align="center" rowspan="1" colspan="1">15</td>
<td align="center" rowspan="1" colspan="1">−2.07</td>
<td align="center" rowspan="1" colspan="1">
<strong>0.05</strong>
<a href="#tpg270100-tbl1-note-0005" class="usa-link">
<sup>*</sup>
</a>
</td>
</tr>
</tbody>
</table></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col align="left" span="1">
<col align="left" span="1">
<col align="char" char="." span="1">
<col align="char" char="." span="1">
<col align="char" char="." span="1">
<col align="char" char="." span="1">
<col align="char" char="." span="1">
<thead>
<tr><th align="left" colspan="7" style="border-bottom:solid 1px #000000" rowspan="1">
<em>rt1‐2</em>
</th></tr>
<tr style="border-bottom:solid 1px #000000">
<th align="left" rowspan="1" colspan="1">Imaging</th>
<th align="left" rowspan="1" colspan="1">Trait</th>
<th align="left" rowspan="1" colspan="1">Wt</th>
<th align="left" rowspan="1" colspan="1">Mu</th>
<th align="left" rowspan="1" colspan="1">
<em>df</em>
</th>
<th align="left" rowspan="1" colspan="1">
<em>t</em>
</th>
<th align="left" rowspan="1" colspan="1">
<em>p</em>
</th>
</tr>
</thead>
<tbody>
<tr>
<td rowspan="5" align="left" colspan="1">2D single‐view<a href="#tpg270100-tbl1-note-0003" class="usa-link">
<sup>a</sup>
</a>
</td>
<td align="left" rowspan="1" colspan="1">Area</td>
<td align="center" rowspan="1" colspan="1">9748.62</td>
<td align="center" rowspan="1" colspan="1">6999.19</td>
<td align="center" rowspan="1" colspan="1">11</td>
<td align="center" rowspan="1" colspan="1">3.96</td>
<td align="center" rowspan="1" colspan="1">
<strong>≤0.01</strong>
<a href="#tpg270100-tbl1-note-0005" class="usa-link">
<sup>*</sup>
</a>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">SklWidth</td>
<td align="center" rowspan="1" colspan="1">168.61</td>
<td align="center" rowspan="1" colspan="1">125.04</td>
<td align="center" rowspan="1" colspan="1">10</td>
<td align="center" rowspan="1" colspan="1">2.32</td>
<td align="center" rowspan="1" colspan="1">
<strong>0.04</strong>
<a href="#tpg270100-tbl1-note-0005" class="usa-link">
<sup>*</sup>
</a>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">RTPCount</td>
<td align="center" rowspan="1" colspan="1">573.29</td>
<td align="center" rowspan="1" colspan="1">359.71</td>
<td align="center" rowspan="1" colspan="1">12</td>
<td align="center" rowspan="1" colspan="1">3.00</td>
<td align="center" rowspan="1" colspan="1">
<strong>≤0.01</strong>
<a href="#tpg270100-tbl1-note-0005" class="usa-link">
<sup>*</sup>
</a>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">AngTop</td>
<td align="center" rowspan="1" colspan="1">44.46</td>
<td align="center" rowspan="1" colspan="1">45.45</td>
<td align="center" rowspan="1" colspan="1">10</td>
<td align="center" rowspan="1" colspan="1">−0.17</td>
<td align="center" rowspan="1" colspan="1">0.87</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">AvgDensity</td>
<td align="center" rowspan="1" colspan="1">3.21</td>
<td align="center" rowspan="1" colspan="1">3.10</td>
<td align="center" rowspan="1" colspan="1">11</td>
<td align="center" rowspan="1" colspan="1">0.14</td>
<td align="center" rowspan="1" colspan="1">0.89</td>
</tr>
<tr>
<td rowspan="5" align="left" colspan="1">2D multi‐view<a href="#tpg270100-tbl1-note-0003" class="usa-link">
<sup>a</sup>
</a>
</td>
<td align="left" rowspan="1" colspan="1">Area</td>
<td align="center" rowspan="1" colspan="1">14,492.11</td>
<td align="center" rowspan="1" colspan="1">10,649.19</td>
<td align="center" rowspan="1" colspan="1">10</td>
<td align="center" rowspan="1" colspan="1">2.94</td>
<td align="center" rowspan="1" colspan="1">
<strong>0.02</strong>
<a href="#tpg270100-tbl1-note-0005" class="usa-link">
<sup>*</sup>
</a>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">SklWidth</td>
<td align="center" rowspan="1" colspan="1">163.29</td>
<td align="center" rowspan="1" colspan="1">136.02</td>
<td align="center" rowspan="1" colspan="1">7</td>
<td align="center" rowspan="1" colspan="1">1.47</td>
<td align="center" rowspan="1" colspan="1">0.18</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">RTPCount</td>
<td align="center" rowspan="1" colspan="1">102.88</td>
<td align="center" rowspan="1" colspan="1">60.17</td>
<td align="center" rowspan="1" colspan="1">10</td>
<td align="center" rowspan="1" colspan="1">5.19</td>
<td align="center" rowspan="1" colspan="1">
<strong>≤0.01</strong>
<a href="#tpg270100-tbl1-note-0005" class="usa-link">
<sup>*</sup>
</a>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">AngTop</td>
<td align="center" rowspan="1" colspan="1">37.59</td>
<td align="center" rowspan="1" colspan="1">44.00</td>
<td align="center" rowspan="1" colspan="1">10</td>
<td align="center" rowspan="1" colspan="1">−1.21</td>
<td align="center" rowspan="1" colspan="1">0.25</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">AvgDensity</td>
<td align="center" rowspan="1" colspan="1">10.47</td>
<td align="center" rowspan="1" colspan="1">8.58</td>
<td align="center" rowspan="1" colspan="1">10</td>
<td align="center" rowspan="1" colspan="1">1.17</td>
<td align="center" rowspan="1" colspan="1">0.27</td>
</tr>
<tr>
<td rowspan="6" align="left" colspan="1">3D (XRT)<a href="#tpg270100-tbl1-note-0004" class="usa-link">
<sup>b</sup>
</a>
</td>
<td align="left" rowspan="1" colspan="1">HorEqDiameter</td>
<td align="center" rowspan="1" colspan="1">466.00</td>
<td align="center" rowspan="1" colspan="1">363.87</td>
<td align="center" rowspan="1" colspan="1">9</td>
<td align="center" rowspan="1" colspan="1">5.50</td>
<td align="center" rowspan="1" colspan="1">
<strong>≤0.01</strong>
<a href="#tpg270100-tbl1-note-0005" class="usa-link">
<sup>*</sup>
</a>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">WDRatio</td>
<td align="center" rowspan="1" colspan="1">0.50</td>
<td align="center" rowspan="1" colspan="1">0.40</td>
<td align="center" rowspan="1" colspan="1">9</td>
<td align="center" rowspan="1" colspan="1">5.12</td>
<td align="center" rowspan="1" colspan="1">
<strong>≤0.01</strong>
<a href="#tpg270100-tbl1-note-0005" class="usa-link">
<sup>*</sup>
</a>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">TotalLength</td>
<td align="center" rowspan="1" colspan="1">127,777.70</td>
<td align="center" rowspan="1" colspan="1">83,140.00</td>
<td align="center" rowspan="1" colspan="1">11</td>
<td align="center" rowspan="1" colspan="1">4.52</td>
<td align="center" rowspan="1" colspan="1">
<strong>≤0.01</strong>
<a href="#tpg270100-tbl1-note-0005" class="usa-link">
<sup>*</sup>
</a>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">LengthDistr</td>
<td align="center" rowspan="1" colspan="1">0.29</td>
<td align="center" rowspan="1" colspan="1">0.08</td>
<td align="center" rowspan="1" colspan="1">6</td>
<td align="center" rowspan="1" colspan="1">2.50</td>
<td align="center" rowspan="1" colspan="1">
<strong>0.05</strong>
<a href="#tpg270100-tbl1-note-0005" class="usa-link">
<sup>*</sup>
</a>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">NumberTips</td>
<td align="center" rowspan="1" colspan="1">2385.57</td>
<td align="center" rowspan="1" colspan="1">1225.00</td>
<td align="center" rowspan="1" colspan="1">11</td>
<td align="center" rowspan="1" colspan="1">6.32</td>
<td align="center" rowspan="1" colspan="1">
<strong>≤0.01</strong>
<a href="#tpg270100-tbl1-note-0005" class="usa-link">
<sup>*</sup>
</a>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Density T2</td>
<td align="center" rowspan="1" colspan="1">0.09</td>
<td align="center" rowspan="1" colspan="1">0.09</td>
<td align="center" rowspan="1" colspan="1">11</td>
<td align="center" rowspan="1" colspan="1">0.01</td>
<td align="center" rowspan="1" colspan="1">0.99</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/tpg270100-tbl-0001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p">
<div class="fn" id="tpg270100-tbl1-note-0001"><p>
<em>Note</em>: AngTop = upper root system angle (°); Area = root area (mm<sup>2</sup>); AvgDensity = average root system density (unitless); Density T2 = top‐view root system density (unitless); HorEqDiameter = maximum root model width (voxel); LengthDistr = ratio of root length in the upper one‐third to the lower two‐thirds of model; Mu = mutant; NumberTips = number of root tips; RTPCount = root tip count (count); SklWidth = skeleton width (mm); TotalLength = cumulative root length (voxel); WDRatio = width‐to‐depth ratio (unitless); Wt = wild‐type.</p></div>
<div class="fn" id="tpg270100-tbl1-note-0002"><p>Abbreviations: 2D, two‐dimensional; 3D, three‐dimensional; RCAP, root crown analysis pipeline; XRT, X‐ray tomography.</p></div>
<div class="fn" id="tpg270100-tbl1-note-0003">
<sup>
<sup>a</sup>
</sup><p class="display-inline">Image analysis using DIRT (digital imaging of root traits).</p>
</div>
<div class="fn" id="tpg270100-tbl1-note-0004">
<sup>
<sup>b</sup>
</sup><p class="display-inline">Image analysis using RCAP.</p>
</div>
<div class="fn" id="tpg270100-tbl1-note-0005">
<sup>*</sup><p class="display-inline">Welch's <em>t</em>‐test (<em>p</em> ≤ 0.05).</p>
</div>
</div></section><p>Similarly, root crown imaging and digital phenotyping revealed significant variations (<em>p</em> ≤ 0.05) in root system size and horizontal spread between <em>dro1‐1</em> mutant and wild‐type allele lines (Table <a href="#tpg270100-tbl-0001" class="usa-link">1</a>). Based on prior observations of the <em>DRO1</em> gene in maize (Feng et al., <a href="#tpg270100-bib-0017" class="usa-link" aria-describedby="tpg270100-bib-0017">2022</a>) and rice (Uga et al., <a href="#tpg270100-bib-0090" class="usa-link" aria-describedby="tpg270100-bib-0090">2013</a>), wild‐type plants were predicted to exhibit increased root gravitropism compared to mutant plants. As expected, the <em>dro1‐1</em> mutation was associated with significantly reduced root growth angles, as well as increased root system area and width‐to‐depth ratios compared to wild‐type plants (Table <a href="#tpg270100-tbl-0001" class="usa-link">1</a>). In contrast to <em>rt1‐2</em>, both 2D multi‐view and 3D phenotyping improved genotype discrimination (<em>p</em> ≤ 0.05) for geometrical features in <em>dro1‐1</em> maize lines, such as root system width and number of root tips, which are typically constrained by the orthogonal positioning of the root crown in 2D single‐view imaging (Figure <a href="#tpg270100-fig-0006" class="usa-link">6</a>).</p>
<figure class="fig xbox font-sm" id="tpg270100-fig-0006"><h4 class="obj_head">FIGURE 6.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8569/12375851/48369616d6bc/TPG2-18-e70100-g005.jpg" loading="lazy" id="jats-graphic-11" height="495" width="700" alt="FIGURE 6"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270100-fig-0006/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Genotype averages for root system width ± SE in <em>dro1‐1</em> mutant and wild‐type allele lines, measured across three imaging and analysis pipelines. Root system width was extracted as skeleton root width (two‐dimensional [2D] SklWidth) using DIRT (digital imaging of root traits) for both 2D single‐view and 2D multi‐view imaging approaches, and as maximum root system width (three‐dimensional [3D] HorEqDiameter) using root crown analysis pipeline (RCAP) for 3D imaging. Significant differences between genotypes are denoted by asterisks (*), with <em>p</em> ≤ 0.05.</p></figcaption></figure><p>The analysis of mutational covariance of 3D RSA traits revealed detailed mutation‐specific relationships underlying differences in root gravitropism for the <em>dro1‐1</em> and <em>rt1‐2</em> maize lines. Two‐way MANOVA, accounting for genetic background, demonstrated significant genotype effects on 3D WDRatio and other traits, including total root length (3D TotalLength) (Pillai's Trace = 0.84, <em>F</em>(3, 33) = 7.18, <em>p </em>&lt; 0.001) and root length distribution (3D LengthDistr) (Pillai's Trace = 0.81, F(3, 33) = 6.79, <em>p</em> &lt; 0.001). In the <em>dro1‐1</em> lines, significant positive correlations were observed between 3D WDRatio and 3D TotalLength (<em>r </em>= 0.57; <em>p</em> = 0.009) as well as 3D LengthDistr (<em>r</em> = 0.73; <em>p </em>= 0.003). Similarly, <em>rt1‐2</em> lines exhibited significant positive correlations between 3D WDRatio and both 3D TotalLength (<em>r </em>= 0.73; <em>p</em> = 0.003) and 3D LengthDistr (<em>r</em> = 0.70; <em>p</em> = 0.005). The root crowns of both the <em>dro1‐1</em> wild‐type plants and the <em>rt1‐2</em> mutant plants displayed smaller width‐to‐depth ratios, reduced total root length, and a greater proportion of root length concentrated in the lower two‐thirds of the root systems. Overall, these results suggest that higher dimensional root phenotyping provides a more refined understanding of gene function and annotation of polymorphisms that influence root crown characteristics in the field.</p></section></section><section id="tpg270100-sec-0150"><h2 class="pmc_sec_title">4. DISCUSSION</h2>
<p>The field of phenomics aims to enhance our understanding of genotype–phenotype maps through the systematic capture of whole‐organism phenotype information and multidimensional data analysis (Houle et al., <a href="#tpg270100-bib-0033" class="usa-link" aria-describedby="tpg270100-bib-0033">2010</a>). Research in this domain has expanded the plant phenome and has identified new underlying gene models (Li et al., <a href="#tpg270100-bib-0048" class="usa-link" aria-describedby="tpg270100-bib-0048">2018</a>, <a href="#tpg270100-bib-0049" class="usa-link" aria-describedby="tpg270100-bib-0049">2023</a>; Ubbens et al., <a href="#tpg270100-bib-0088" class="usa-link" aria-describedby="tpg270100-bib-0088">2020</a>). In this study, we evaluated multiple phenotyping and analytical frameworks for quantifying RSA in mature maize root crowns across three field experiments. Specifically, we assessed how different quantitative measurements of RSA describe the broader root crown phenome and determine the significance of this information for mapping genotype–phenotype relationships.</p>
<p>Overall, our results suggest that increasing the number of rotational viewpoints in 2D imaging using the 2D multi‐view approach enhances the capture of genetic variation controlling different aspects of maize RSA. In the 2019 SAM maize panel, phenome‐ and genome‐wide analyses indicated that 3D traits captured the largest proportion of phenotypic and genetic variation in field root crown architecture across different phenotyping methods. Among individual root traits, measurements of RPF were moderately heritable and strongly correlated with 3D traits, highlighting its potential as a cost‐effective alternative to XRT 3D imaging for large‐scale field mapping studies. Our findings also support the use of multivariate traits as an effective method for quantifying complex RSA phenotypes and identifying putative pleiotropic loci that are often missed by canonical univariate GWAS approaches. Analysis of the maize root development mutants, <em>dro1‐1</em> and <em>rt1‐2</em>, further validated the application of 3D and 2D multi‐view imaging within the context of a reverse genetics framework. Both methods detected statistically significant phenotypic variations that aligned with previous qualitative descriptions of RSA in their wild‐type and mutant allele lines—variations not captured by the 2D single‐view approach alone.</p>
<p>Trait‐based heritability estimates often vary based on measurement methods and environmental conditions (Hill et al., <a href="#tpg270100-bib-0027" class="usa-link" aria-describedby="tpg270100-bib-0027">1983</a>; Hoffmann &amp; Merilä, <a href="#tpg270100-bib-0031" class="usa-link" aria-describedby="tpg270100-bib-0031">1999</a>; Macgregor et al., <a href="#tpg270100-bib-0059" class="usa-link" aria-describedby="tpg270100-bib-0059">2006</a>). Our analysis of root crowns from the 2022 NAM RIL population demonstrated that integrating multiple viewpoints in 2D phenotyping enhances the detection of significant heritable variation in RSA under field conditions (Tables <a href="#tpg270100-supinfo-0001" class="usa-link">S2</a> and <a href="#tpg270100-supinfo-0001" class="usa-link">S3</a>). As illustrated in Figure <a href="#tpg270100-fig-0002" class="usa-link">2</a>, the 2D multi‐view approach yielded relatively higher heritability estimates than the single‐view approach for various RSA features that broadly characterize root crown geometry and topology. For instance, 2D WidthMax, a direct measurement of maximum root system width (see Das et al., <a href="#tpg270100-bib-0012" class="usa-link" aria-describedby="tpg270100-bib-0012">2015</a>), exhibited significant genotypic variance (<em>H</em>
<sup>2</sup> = 0.27; <em>p </em>&lt; 0.05) in the multi‐view setup, whereas it did not in the single‐view setup (<em>H</em>
<sup>2</sup> = 0.10; <em>p </em>= 0.38) or in any other individual camera perspectives (Table <a href="#tpg270100-supinfo-0001" class="usa-link">S3</a>). These findings were further supported by our analysis of the maize development mutants, where the multi‐view trait estimates provided better statistical power for detecting subtle differences in RSA, including root system width, between <em>dro1‐1</em> wild‐type and mutant plants (Table <a href="#tpg270100-tbl-0001" class="usa-link">1</a>; Figure <a href="#tpg270100-fig-0006" class="usa-link">6</a>). For the <em>D</em>‐value traits (2D‐D60), where the 2D single‐view approach produced higher heritability estimates, these traits are associated with brace root orientation and root system spread and are derived from measurements of maximum root width, depth, and width at specific depth intervals (Bucksch et al., <a href="#tpg270100-bib-0006" class="usa-link" aria-describedby="tpg270100-bib-0006">2014</a>). Similar to the <em>D</em>‐value traits, these associated traits showed differences among imaging perspectives (see Table <a href="#tpg270100-supinfo-0001" class="usa-link">S3</a>).</p>
<p>Single‐view 2D imaging is a widely used and relatively straightforward approach to digital root phenotyping; however, it can introduce sampling bias to root feature extraction because root crowns are rarely symmetrical and are typically positioned with the widest part of the root system orthogonal to the camera (Tuberosa, <a href="#tpg270100-bib-0086" class="usa-link" aria-describedby="tpg270100-bib-0086">2012</a>). These results suggest that in single‐view imaging, root crowns are often preferentially arranged to provide an orientation that optimally displays brace root orientation and root system spread, leading to higher end trait estimates. In contrast, the inclusion of additional perspectives in the multi‐view approach may summarize more variability around the sampled root crowns. Therefore, we recommend the 2D multi‐view approach as a readily implementable improvement to 2D single‐view for addressing forward and reverse genetic hypotheses, especially when investigating RSA traits that are greatly influenced by asymmetry and root crown orientation. Future research should focus on optimizing the integration of multi‐view image data into consolidated phenotypic values, as simple averages across images may fail to accurately capture the complexity of the field‐excavated root crowns.</p>
<p>Advances in 3D imaging technology have significantly enhanced our ability to interrogate root systems and capture the complexity of RSA in agricultural systems (Bohn et al., <a href="#tpg270100-bib-0002" class="usa-link" aria-describedby="tpg270100-bib-0002">2006</a>; Dowd et al., <a href="#tpg270100-bib-0014" class="usa-link" aria-describedby="tpg270100-bib-0014">2022</a>; Jung &amp; McCouch, <a href="#tpg270100-bib-0039" class="usa-link" aria-describedby="tpg270100-bib-0039">2013</a>; Li et al., <a href="#tpg270100-bib-0049" class="usa-link" aria-describedby="tpg270100-bib-0049">2023</a>; S. Liu et al., <a href="#tpg270100-bib-0052" class="usa-link" aria-describedby="tpg270100-bib-0052">2021</a>; Mattupalli et al., <a href="#tpg270100-bib-0060" class="usa-link" aria-describedby="tpg270100-bib-0060">2018</a>; Morris et al., <a href="#tpg270100-bib-0063" class="usa-link" aria-describedby="tpg270100-bib-0063">2017</a>; Shao et al., <a href="#tpg270100-bib-0077" class="usa-link" aria-describedby="tpg270100-bib-0077">2021</a>; Topp et al., <a href="#tpg270100-bib-0082" class="usa-link" aria-describedby="tpg270100-bib-0082">2013</a>). Our study reinforced this progress by showing that 3D measurements not only provided detailed insights at the phenome level but also had a greater capacity to detect significant variations in both genome‐wide and single‐gene analyses. PCA of SAM maize lines revealed substantial phenotypic variation driven by 3D traits, particularly those linked to root size, distribution, and density (Figure <a href="#tpg270100-fig-0003" class="usa-link">3A</a>). XRT scans of maize root crowns further validated these results, with root models visually reflecting size variations along PC1 and density variations along PC2, consistent with the vectors in the PCA biplot (Figure <a href="#tpg270100-fig-0003" class="usa-link">3B</a>). Furthermore, genome‐wide association analyses identified the largest number of RSA‐associated loci using 3D traits, with these loci showing higher average PVE values compared to those identified with 2D, RPF, and multivariate traits (Figure <a href="#tpg270100-fig-0004" class="usa-link">4</a>; Figure <a href="#tpg270100-supinfo-0001" class="usa-link">S6</a>). Analyses of mutant and wild‐type allele lines for the <em>dro1‐1</em> and <em>rt1‐2</em> RSA genes provided evidence for the ability of 3D phenotyping to capture fine‐scale root allocation trade‐offs, consistent with previously reported qualitative descriptions (Figure <a href="#tpg270100-fig-0006" class="usa-link">6</a>; Table <a href="#tpg270100-tbl-0001" class="usa-link">1</a>) (Feng et al., <a href="#tpg270100-bib-0017" class="usa-link" aria-describedby="tpg270100-bib-0017">2022</a>; Hochholdinger et al., <a href="#tpg270100-bib-0030" class="usa-link" aria-describedby="tpg270100-bib-0030">2018</a>; Jenkins, <a href="#tpg270100-bib-0035" class="usa-link" aria-describedby="tpg270100-bib-0035">1930</a>; Uga et al., <a href="#tpg270100-bib-0090" class="usa-link" aria-describedby="tpg270100-bib-0090">2013</a>). Overall, these findings underscore the value of 3D phenotyping as a powerful tool for capturing complex root system dynamics and advancing our understanding of genetic contributions to RSA in field studies of maize.</p>
<p>Although 3D phenotyping offers substantial advantages for genome‐wide studies of RSA in maize, its widespread application is often constrained by its associated overhead costs and training. In contrast, RPF measurements, which demonstrated high heritability and strong correlations with 3D traits, may serve as a more cost‐effective method for quantifying RSA for large‐scale field phenotyping (Shao et al., <a href="#tpg270100-bib-0077" class="usa-link" aria-describedby="tpg270100-bib-0077">2021</a>; Woods et al., <a href="#tpg270100-bib-0095" class="usa-link" aria-describedby="tpg270100-bib-0095">2022</a>). However, the suitability of its application varies depending on the system, root archetype, and experimental design. Notably, genome‐wide association analyses revealed numerous SNPs associated with lower heritable 3D traits that co‐located with RPF and explained a higher proportion of phenotypic variance for each bin (Figure <a href="#tpg270100-supinfo-0001" class="usa-link">S6</a>). Based on the quantitative genetic theory, we hypothesized that the genetic architecture of RPF may be more polygenic in nature, with many genes contributing to small effects (Fisher, <a href="#tpg270100-bib-0018" class="usa-link" aria-describedby="tpg270100-bib-0018">1919</a>, <a href="#tpg270100-bib-0019" class="usa-link" aria-describedby="tpg270100-bib-0019">1930</a>). In contrast, the larger PVE values found among information‐rich 3D traits could be controlled by fewer loci with larger effects. Genome‐wide associations with RPF could have also been confounded by nonadditive genetic variation, as estimates of <em>H</em>
<sup>2</sup> reflect the total genetic variation comprising additive, dominance, and epistatic effects (M. Lynch &amp; Walsh, <a href="#tpg270100-bib-0058" class="usa-link" aria-describedby="tpg270100-bib-0058">1998</a>).</p>
<p>Complex phenotypes in biological organisms emerge from interactions and contributions of individual traits. A common approach to studying the genetic underpinnings of these phenotypes involves using data reduction techniques, such as PCs, to generate “composite traits” (Hotelling, <a href="#tpg270100-bib-0032" class="usa-link" aria-describedby="tpg270100-bib-0032">1933</a>; Klei et al., <a href="#tpg270100-bib-0042" class="usa-link" aria-describedby="tpg270100-bib-0042">2008</a>; Li et al., <a href="#tpg270100-bib-0049" class="usa-link" aria-describedby="tpg270100-bib-0049">2023</a>; Rice et al., <a href="#tpg270100-bib-0071" class="usa-link" aria-describedby="tpg270100-bib-0071">2020</a>; Topp et al., <a href="#tpg270100-bib-0082" class="usa-link" aria-describedby="tpg270100-bib-0082">2013</a>). Our study utilized 2D and 3D multivariate traits derived from PCA to test for genome‐wide associations. We found that these traits, particularly the 2D PC1 and 3D PC1, had strong positive associations with root biomass accumulation (Figure <a href="#tpg270100-fig-0005" class="usa-link">5</a>). Additionally, the loci identified by multivariate traits were primarily located within regions that were also associated with individual traits (Figure <a href="#tpg270100-fig-0004" class="usa-link">4</a>). Notably, the combination of individual traits for these regions reflected both the correlated (e.g., 3D PC2 and 3D root density traits) and orthogonal (e.g., 2D PC1 and 2D AvgDensity) phenotypic relationships observed in the PCA (Figure <a href="#tpg270100-fig-0003" class="usa-link">3A</a>; Figure <a href="#tpg270100-supinfo-0001" class="usa-link">S6</a>). One explanation for these patterns is that similar and distinct elements of maize root architecture are controlled by the same genes, which is indicative of a pleiotropic genetic architecture. Although multivariate traits do not directly provide biological insights into pleiotropic loci, they are advantageous for elucidating the relationships between low‐ and highly heritable traits at a given locus, which are often missed by univariate approaches (Rice et al., <a href="#tpg270100-bib-0071" class="usa-link" aria-describedby="tpg270100-bib-0071">2020</a>). These results further demonstrate that the multivariate approach has the power to replicate univariate results, reinforcing its value in forward genetic studies (Rice et al., <a href="#tpg270100-bib-0071" class="usa-link" aria-describedby="tpg270100-bib-0071">2020</a>; Topp et al., <a href="#tpg270100-bib-0082" class="usa-link" aria-describedby="tpg270100-bib-0082">2013</a>).</p>
<p>Understanding the genetic and molecular mechanisms that regulate root system development and architecture is critical for enhancing plant resilience to environmental stress, improving resource use efficiency, and supporting ecosystem services such as soil stabilization and carbon sequestration. Historically, much of our understanding of this area has been limited to studies in controlled environments, focusing on seedlings or model systems. While the dicot model organism <em>Arabidopsis thaliana</em> has provided a foundation for understanding root development, it does not directly translate to the genetic architecture of monocot crops, such as maize, rice, and sorghum, which differ in the development and morphology of their root systems (Jung &amp; McCouch, <a href="#tpg270100-bib-0039" class="usa-link" aria-describedby="tpg270100-bib-0039">2013</a>; Péret et al., <a href="#tpg270100-bib-0065" class="usa-link" aria-describedby="tpg270100-bib-0065">2009</a>; Petricka et al., <a href="#tpg270100-bib-0067" class="usa-link" aria-describedby="tpg270100-bib-0067">2012</a>; Scheres et al., <a href="#tpg270100-bib-0074" class="usa-link" aria-describedby="tpg270100-bib-0074">1996</a>; Tuberosa et al., <a href="#tpg270100-bib-0087" class="usa-link" aria-describedby="tpg270100-bib-0087">2003</a>; Ueda et al., <a href="#tpg270100-bib-0089" class="usa-link" aria-describedby="tpg270100-bib-0089">2005</a>). This has created a gap in our ability to study complex, mature root systems in crops grown under agronomic environments, primarily because of the lack of appropriate phenotyping tools and analytical methods. Our research addressed this by applying advanced RSA phenotyping frameworks to mature maize root crowns, revealing that capturing multidimensional data, such as through 2D multi‐view, 3D phenotyping, and multivariate approaches, significantly improves the precision of phenotype–genotype mapping. These findings underscore the importance of employing complementary approaches to fully explore the phenotype–genotype gap. However, it is worth noting that the relationship between phenotyping methods and their effectiveness is not necessarily static. As feature extraction technologies—such as AI‐based tools, topological data analysis, and advanced 3D graphics—continue to evolve, the potential for capturing more comprehensive and accurate root phenotypes will grow. Therefore, future research should not only build on current methods but also remain adaptable, integrating cutting‐edge technologies to further refine our understanding of the root phenome and its genetic drivers under diverse agricultural conditions.</p></section><section id="tpg270100-sec-0160"><h2 class="pmc_sec_title">AUTHOR CONTRIBUTIONS</h2>
<p>
<strong>Kirsten M. Hein</strong>: Conceptualization; data curation; formal analysis; funding acquisition; investigation; methodology; project administration; validation; visualization; writing—original draft; writing—review and editing. <strong>Alexander E. Liu</strong>: Methodology; resources; writing—review and editing. <strong>Jack L. Mullen</strong>: Conceptualization; data curation; formal analysis; funding acquisition; investigation; methodology; supervision; validation; writing—review and editing. <strong>Mon‐Ray Shao</strong>: Methodology; resources; writing—review and editing. <strong>Christopher N. Topp</strong>: Conceptualization; data curation; funding acquisition; investigation; methodology; resources; supervision; writing—review and editing. <strong>John K. McKay</strong>: Conceptualization; data curation; funding acquisition; investigation; project administration; resources; supervision; writing—review and editing.</p></section><section id="tpg270100-sec-0180"><h2 class="pmc_sec_title">CONFLICT OF INTEREST STATEMENT</h2>
<p>The authors declare no conflicts of interest.</p></section><section id="sec18"><h2 class="pmc_sec_title">Supporting information</h2>
<section class="sm xbox font-sm" id="tpg270100-supinfo-0001"></section><section class="sm xbox font-sm" id="tpg270100-supitem-0001"><div class="caption p">
<p>
<strong>Supplemental Figure S1</strong> Heatmap of pairwise genotypic correlations among 155 RSA traits grouped by collection method (2D single‐view imaging, 3D imaging, and manual measurements) in the 2019 SAM maize panel.</p>
<p>
<strong>Supplemental Figure S2</strong> Percentage of explained variances by individual principal components associated with the PCA of 115 RSA traits measured in the 2019 SAM maize panel.</p>
<p>
<strong>Supplemental Figure S3</strong> PCA of RSA traits partitioned by irrigation treatment measured in the 2019 SAM maize panel.</p>
<p>
<strong>Supplemental Figure S4</strong> Distribution of broad‐sense heritability estimates of the 2019 SAM maize panel RSA traits among phenotyping methods and irrigation treatments.</p>
<p>
<strong>Supplemental Figure S5</strong> Broad‐sense heritability estimates for root mass (g) and RPF (kg) across irrigation treatments in the 2019 SAM maize panel.</p>
<p>
<strong>Supplemental Figure S6</strong> Heatmap of the percentage of phenotypic variance explained (PVE) by 1‐Mb genomic regions associated with traits colocalized across imaging, RPF, and multivariate approaches.</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12375851/bin/TPG2-18-e70100-s002.pdf" data-ga-action="click_feat_suppl" class="usa-link">TPG2-18-e70100-s002.pdf</a><sup> (467.9KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="tpg270100-supitem-0002"><div class="caption p">
<p>
<strong>Supplemental Table S1</strong> Genotypic values (BLUPs) used in the 2019 FarmCPU GWAS on the SAM maize diversity panel, with RSA univariate and multivariate traits partitioned by phenotyping method.</p>
<p>
<strong>Supplemental Table S2</strong> Comparison of broad‐sense heritability for all RSA traits between 2D single‐view and multi‐view root crown imaging in the 2022 NAM RIL population.</p>
<p>
<strong>Supplemental Table S3</strong> Comparison of 18 RSA traits that showed significant broad‐sense heritability (H2) in at least one 2D imaging method in the 2022 NAM RIL population.</p>
<p>
<strong>Supplemental Table S4</strong> 2019 PCA trait loadings associated with the axes represented in Figure 3.</p>
<p>
<strong>Supplemental Table S5</strong> Broad‐sense heritability and variance estimates for RSA traits in the 2019 SAM maize panel.</p>
<p>
<strong>Supplemental Table S6</strong> List of significant SNPs from the 2019 FarmCPU GWAS.</p>
<p>
<strong>Supplemental Table S7</strong> Multivariate GWAS trait loadings by phenotyping method and irrigation treatment in the 2019 SAM maize diversity panel.</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12375851/bin/TPG2-18-e70100-s001.xlsx" data-ga-action="click_feat_suppl" class="usa-link">TPG2-18-e70100-s001.xlsx</a><sup> (967.4KB, xlsx) </sup>
</div></div></section></section><section id="tpg270100-sec-0170" class="ack"><h2 class="pmc_sec_title">ACKNOWLEDGMENTS</h2>
<p>We would like to thank Kyle Evans, Anne Howard, Jake Keenan, and Stephanie Goldin for their contributions to managing the field trials and sample processing. We also extend our thanks to the many undergraduate students at Colorado State University who assisted with field maintenance and data collection as part of our summer field crews. Special thanks to Keith Duncan and Clara Lebow at the Donald Danforth Plant Science Center for their support and expertise in X‐ray computed tomography and RCAP data processing. This research was supported by the Department of Energy under award number DE‐AR0000826 to John K. McKay, Jack L. Mullen, and Christopher N. Topp; National Science Foundation award number 2220726 to John K. McKay; and National Science Foundation Graduate Research Fellowship Program award numbers 1840343 and 2234690 to Kirsten M. Hein.</p></section><section id="notes1"><p>


Hein, K. M.
, 
Liu, A. E.
, 
Mullen, J. L.
, 
Shao, M.‐R.
, 
Topp, C. N.
, &amp; 
McKay, J. K.
 (2025). Phenome‐to‐genome insights for evaluating root system architecture in field studies of maize. The Plant Genome, 18, e70100. 10.1002/tpg2.70100

</p>
<section id="fn-group1" class="fn-group"><div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="tpg270100-note-0001"><p>Assigned to Associate Editor Roberto Tuberosa.</p></div></div></section></section><section id="tpg270100-sec-0200"><h2 class="pmc_sec_title">DATA AVAILABILITY STATEMENT</h2>
<p>Raw phenotypic metadata are available on the Dryad Digital Repository (<a href="https://doi.org/10.5061/dryad.z34tmpgq4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://doi.org/10.5061/dryad.z34tmpgq4</a>, <a href="http://datadryad.org/share/HeNYoxNMdN_GrHMyZHFN3rUTN1UiG8OFhU-B107E7mM" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://datadryad.org/share/HeNYoxNMdN_GrHMyZHFN3rUTN1UiG8OFhU-B107E7mM</a>). Phenotypic values used in the 2019 genome‐wide association analysis, along with the identified significant RSA‐associated SNPs, are provided in Tables <a href="#tpg270100-supinfo-0001" class="usa-link">S1</a>, <a href="#tpg270100-supinfo-0001" class="usa-link">S6</a>, and <a href="#tpg270100-supinfo-0001" class="usa-link">S7</a>.</p></section><section id="tpg270100-bibl-0001" class="ref-list"><h2 class="pmc_sec_title">REFERENCES</h2>
<section id="tpg270100-bibl-0001_sec2"><ol class="ref-list font-sm">
<li id="tpg270100-bib-0001">
<cite>

Bates, D.
, 
Mächler, M.
, 
Bolker, B.
, &amp; 
Walker, S.
 (2015). Fitting linear mixed‐effects models using lme4. Journal of Statistical Software, 67(1), 1–48. 10.18637/jss.v067.i01
</cite> [<a href="https://doi.org/10.18637/jss.v067.i01" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Statistical%20Software&amp;title=Fitting%20linear%20mixed%E2%80%90effects%20models%20using%20lme4&amp;author=D.%20Bates&amp;author=M.%20M%C3%A4chler&amp;author=B.%20Bolker&amp;author=S.%20Walker&amp;volume=67&amp;issue=1&amp;publication_year=2015&amp;pages=1-48&amp;doi=10.18637/jss.v067.i01&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270100-bib-0002">
<cite>

Bohn, M.
, 
Novais, J.
, 
Fonseca, R.
, 
Tuberosa, R.
, &amp; 
Grift, T. E.
 (2006). Genetic evaluation of root complexity in maize. Acta Agronomica Hungarica, 54(3), 291–303. 10.1556/AAgr.54.2006.3.3
</cite> [<a href="https://doi.org/10.1556/AAgr.54.2006.3.3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Acta%20Agronomica%20Hungarica&amp;title=Genetic%20evaluation%20of%20root%20complexity%20in%20maize&amp;author=M.%20Bohn&amp;author=J.%20Novais&amp;author=R.%20Fonseca&amp;author=R.%20Tuberosa&amp;author=T.%20E.%20Grift&amp;volume=54&amp;issue=3&amp;publication_year=2006&amp;pages=291-303&amp;doi=10.1556/AAgr.54.2006.3.3&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270100-bib-0003">
<cite>

Bonser, A. M. Y. M.
, 
Lynch, J.
, &amp; 
Snapp, S.
 (1996). Effect of phosphorus deficiency on growth angle of basal roots in <em>Phaseolus vulgaris</em>
. New Phytologist, 132(2), 281–288. 10.1111/j.1469-8137.1996.tb01847.x

</cite> [<a href="https://doi.org/10.1111/j.1469-8137.1996.tb01847.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11541132/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=New%20Phytologist&amp;title=Effect%20of%20phosphorus%20deficiency%20on%20growth%20angle%20of%20basal%20roots%20in%20Phaseolus%20vulgaris&amp;author=A.%20M.%20Y.%20M.%20Bonser&amp;author=J.%20Lynch&amp;author=S.%20Snapp&amp;volume=132&amp;issue=2&amp;publication_year=1996&amp;pages=281-288&amp;pmid=11541132&amp;doi=10.1111/j.1469-8137.1996.tb01847.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270100-bib-0004">
<cite>

Bray, A. L.
, &amp; 
Topp, C. N.
 (2018). The quantitative genetic control of root architecture in maize. Plant and Cell Physiology, 59(10), 1919–1930. 10.1093/pcp/pcy141

</cite> [<a href="https://doi.org/10.1093/pcp/pcy141" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6178961/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30020530/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20and%20Cell%20Physiology&amp;title=The%20quantitative%20genetic%20control%20of%20root%20architecture%20in%20maize&amp;author=A.%20L.%20Bray&amp;author=C.%20N.%20Topp&amp;volume=59&amp;issue=10&amp;publication_year=2018&amp;pages=1919-1930&amp;pmid=30020530&amp;doi=10.1093/pcp/pcy141&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270100-bib-0005">
<cite>

Buckler, E. S.
, 
Holland, J. B.
, 
Bradbury, P. J.
, 
Acharya, C. B.
, 
Brown, P. J.
, 
Browne, C.
, 
Ersoz, E.
, 
Flint‐Garcia, S.
, 
Garcia, A.
, 
Glaubitz, J. C.
, 
Goodman, M. M.
, 
Harjes, C.
, 
Guill, K.
, 
Kroon, D. E.
, 
Larsson, S.
, 
Lepak, N. K.
, 
Li, H.
, 
Mitchell, S. E.
, 
Pressoir, G.
, … 
McMullen, M. D.
 (2009). The genetic architecture of maize flowering time. Science, 325(5941), 714–718. 10.1126/science.1174276

</cite> [<a href="https://doi.org/10.1126/science.1174276" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19661422/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=The%20genetic%20architecture%20of%20maize%20flowering%20time&amp;author=E.%20S.%20Buckler&amp;author=J.%20B.%20Holland&amp;author=P.%20J.%20Bradbury&amp;author=C.%20B.%20Acharya&amp;author=P.%20J.%20Brown&amp;volume=325&amp;issue=5941&amp;publication_year=2009&amp;pages=714-718&amp;pmid=19661422&amp;doi=10.1126/science.1174276&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270100-bib-0006">
<cite>

Bucksch, A.
, 
Burridge, J.
, 
York, L. M.
, 
Das, A.
, 
Nord, E.
, 
Weitz, J. S.
, &amp; 
Lynch, J. P.
 (2014). Image‐based high‐throughput field phenotyping of crop roots. Plant Physiology, 166(2), 470–486. 10.1104/pp.114.243519

</cite> [<a href="https://doi.org/10.1104/pp.114.243519" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4213080/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25187526/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Physiology&amp;title=Image%E2%80%90based%20high%E2%80%90throughput%20field%20phenotyping%20of%20crop%20roots&amp;author=A.%20Bucksch&amp;author=J.%20Burridge&amp;author=L.%20M.%20York&amp;author=A.%20Das&amp;author=E.%20Nord&amp;volume=166&amp;issue=2&amp;publication_year=2014&amp;pages=470-486&amp;pmid=25187526&amp;doi=10.1104/pp.114.243519&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270100-bib-0007">
<cite>

Cai, H.
, 
Chen, F.
, 
Mi, G.
, 
Zhang, F.
, 
Maurer, H. P.
, 
Liu, W.
, 
Reif, J. C.
, &amp; 
Yuan, L.
 (2012). Mapping QTLs for root system architecture of maize (<em>Zea mays</em> L.) in the field at different developmental stages. Theoretical and Applied Genetics, 125(6), 1313–1324. 10.1007/s00122-012-1915-6

</cite> [<a href="https://doi.org/10.1007/s00122-012-1915-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22718302/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Theoretical%20and%20Applied%20Genetics&amp;title=Mapping%20QTLs%20for%20root%20system%20architecture%20of%20maize%20(Zea%20mays%20L.)%20in%20the%20field%20at%20different%20developmental%20stages&amp;author=H.%20Cai&amp;author=F.%20Chen&amp;author=G.%20Mi&amp;author=F.%20Zhang&amp;author=H.%20P.%20Maurer&amp;volume=125&amp;issue=6&amp;publication_year=2012&amp;pages=1313-1324&amp;pmid=22718302&amp;doi=10.1007/s00122-012-1915-6&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270100-bib-0009">
<cite>

Champoux, M. C.
, 
Wang, G.
, 
Sarkarung, S.
, 
Mackill, D. J.
, 
O'Toole, J. C.
, 
Huang, N.
, &amp; 
McCouch, S. R.
 (1995). Locating genes associated with root morphology and drought avoidance in rice via linkage to molecular markers. Theoretical and Applied Genetics, 90(7), 969–981. 10.1007/BF00222910

</cite> [<a href="https://doi.org/10.1007/BF00222910" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24173051/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Theoretical%20and%20Applied%20Genetics&amp;title=Locating%20genes%20associated%20with%20root%20morphology%20and%20drought%20avoidance%20in%20rice%20via%20linkage%20to%20molecular%20markers&amp;author=M.%20C.%20Champoux&amp;author=G.%20Wang&amp;author=S.%20Sarkarung&amp;author=D.%20J.%20Mackill&amp;author=J.%20C.%20O'Toole&amp;volume=90&amp;issue=7&amp;publication_year=1995&amp;pages=969-981&amp;pmid=24173051&amp;doi=10.1007/BF00222910&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270100-bib-0010">
<cite>

Chen, H.
 (2022). VennDiagram: Generate High‐Resolution Venn and Euler Plots [Computer software]. CRAN. <a href="https://cran.r-project.org/package=VennDiagram" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://cran.r‐project.org/package=VennDiagram</a>
</cite> [<a href="https://scholar.google.com/scholar_lookup?Chen,%20H.%20(2022).%20VennDiagram:%20Generate%20High%E2%80%90Resolution%20Venn%20and%20Euler%20Plots%20%5BComputer%20software%5D.%20CRAN.%20https://cran.r%E2%80%90project.org/package=VennDiagram" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270100-bib-0011">
<cite>

Clarke, C. R.
, 
Timko, M. P.
, 
Yoder, J. I.
, 
Axtell, M. J.
, &amp; 
Westwood, J. H.
 (2019). Molecular dialog between parasitic plants and their hosts. Annual Review of Phytopathology, 57, 279–299. 10.1146/annurev-phyto-082718-100043
</cite> [<a href="https://doi.org/10.1146/annurev-phyto-082718-100043" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31226021/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Annual%20Review%20of%20Phytopathology&amp;title=Molecular%20dialog%20between%20parasitic%20plants%20and%20their%20hosts&amp;author=C.%20R.%20Clarke&amp;author=M.%20P.%20Timko&amp;author=J.%20I.%20Yoder&amp;author=M.%20J.%20Axtell&amp;author=J.%20H.%20Westwood&amp;volume=57&amp;publication_year=2019&amp;pages=279-299&amp;pmid=31226021&amp;doi=10.1146/annurev-phyto-082718-100043&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270100-bib-0012">
<cite>

Das, A.
, 
Schneider, H.
, 
Burridge, J.
, 
Ascanio, A. K. M.
, 
Wojciechowski, T.
, 
Topp, C. N.
, 
Lynch, J. P.
, 
Weitz, J. S.
, &amp; 
Bucksch, A.
 (2015). Digital imaging of root traits (DIRT): A high‐throughput computing and collaboration platform for field‐based root phenomics. Plant Methods, 11(1), Article 51. 10.1186/s13007-015-0093-3

</cite> [<a href="https://doi.org/10.1186/s13007-015-0093-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4630929/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26535051/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Methods&amp;title=Digital%20imaging%20of%20root%20traits%20(DIRT):%20A%20high%E2%80%90throughput%20computing%20and%20collaboration%20platform%20for%20field%E2%80%90based%20root%20phenomics&amp;author=A.%20Das&amp;author=H.%20Schneider&amp;author=J.%20Burridge&amp;author=A.%20K.%20M.%20Ascanio&amp;author=T.%20Wojciechowski&amp;volume=11&amp;issue=1&amp;publication_year=2015&amp;pages=51&amp;pmid=26535051&amp;doi=10.1186/s13007-015-0093-3&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270100-bib-0013">
<cite>

de Jong, M.
, 
Tavares, H.
, 
Pasam, R. K.
, 
Butler, R.
, 
Ward, S.
, 
George, G.
, 
Melnyk, C. W.
, 
Challis, R.
, 
Kover, P. X.
, &amp; 
Leyser, O.
 (2019). Natural variation in Arabidopsis shoot branching plasticity in response to nitrate supply affects fitness. PLoS Genetics, 15(9), e1008366. 10.1371/journal.pgen.1008366

</cite> [<a href="https://doi.org/10.1371/journal.pgen.1008366" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6774567/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31539368/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20Genetics&amp;title=Natural%20variation%20in%20Arabidopsis%20shoot%20branching%20plasticity%20in%20response%20to%20nitrate%20supply%20affects%20fitness&amp;author=M.%20de%20Jong&amp;author=H.%20Tavares&amp;author=R.%20K.%20Pasam&amp;author=R.%20Butler&amp;author=S.%20Ward&amp;volume=15&amp;issue=9&amp;publication_year=2019&amp;pages=e1008366&amp;pmid=31539368&amp;doi=10.1371/journal.pgen.1008366&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270100-bib-0014">
<cite>

Dowd, T. G.
, 
Li, M.
, 
Bagnall, G. C.
, 
Johnston, A.
, &amp; 
Topp, C. N.
 (2022). Root system architecture and environmental flux analysis in mature crops using 3D root mesocosms. Frontiers in Plant Science, 13(12), 1041404. 10.3389/fpls.2022.1041404

</cite> [<a href="https://doi.org/10.3389/fpls.2022.1041404" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9800027/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36589101/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Plant%20Science&amp;title=Root%20system%20architecture%20and%20environmental%20flux%20analysis%20in%20mature%20crops%20using%203D%20root%20mesocosms&amp;author=T.%20G.%20Dowd&amp;author=M.%20Li&amp;author=G.%20C.%20Bagnall&amp;author=A.%20Johnston&amp;author=C.%20N.%20Topp&amp;volume=13&amp;issue=12&amp;publication_year=2022&amp;pages=1041404&amp;pmid=36589101&amp;doi=10.3389/fpls.2022.1041404&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270100-bib-0015">
<cite>

Duncan, K. E.
, 
Bray, A. L.
, 
Dowd, T. G.
, &amp; 
Topp, C. N.
 (2019). Using 3D X‐ray microscopy to study crown root development and primary root tip growth in diverse maize (<em>Zea mays</em> L.) Lines. Microscopy and Microanalysis, 25(2), 1032–1033. 10.1017/S1431927619005890
31134876
</cite> [<a href="https://doi.org/10.1017/S1431927619005890" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Microscopy%20and%20Microanalysis&amp;title=Using%203D%20X%E2%80%90ray%20microscopy%20to%20study%20crown%20root%20development%20and%20primary%20root%20tip%20growth%20in%20diverse%20maize%20(Zea%20mays%20L.)%20Lines&amp;author=K.%20E.%20Duncan&amp;author=A.%20L.%20Bray&amp;author=T.%20G.%20Dowd&amp;author=C.%20N.%20Topp&amp;volume=25&amp;issue=2&amp;publication_year=2019&amp;pages=1032-1033&amp;doi=10.1017/S1431927619005890&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270100-bib-0016">
<cite>

Feix, G.
, 
Hochholdinger, F.
, &amp; 
June Park, W.
 (2002). Maize root system and genetic analysis of its formation. In Waisel Y., Eshel A., Beeckman T., &amp; Kafkafi U. (Eds.), Plant roots: The hidden half (3rd ed., pp. 239–248). CRC Press. 10.1201/9780203909423
</cite> [<a href="https://doi.org/10.1201/9780203909423" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?title=Plant%20roots:%20The%20hidden%20half&amp;author=G.%20Feix&amp;author=F.%20Hochholdinger&amp;author=W.%20June%20Park&amp;publication_year=2002&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270100-bib-0017">
<cite>

Feng, X.
, 
Jia, L.
, 
Cai, Y.
, 
Guan, H.
, 
Zheng, D.
, 
Zhang, W.
, 
Xiong, H.
, 
Zhou, H.
, 
Wen, Y.
, 
Hu, Y.
, 
Zhang, X.
, 
Wang, Q.
, 
Wu, F.
, 
Xu, J.
, &amp; 
Lu, Y.
 (2022). ABA‐inducible <em>DEEPER ROOTING 1</em> improves adaptation of maize to water deficiency. Plant Biotechnology Journal, 20(11), 2077–2088. 10.1111/pbi.13889

</cite> [<a href="https://doi.org/10.1111/pbi.13889" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9616520/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35796628/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Biotechnology%20Journal&amp;title=ABA%E2%80%90inducible%20DEEPER%20ROOTING%201%20improves%20adaptation%20of%20maize%20to%20water%20deficiency&amp;author=X.%20Feng&amp;author=L.%20Jia&amp;author=Y.%20Cai&amp;author=H.%20Guan&amp;author=D.%20Zheng&amp;volume=20&amp;issue=11&amp;publication_year=2022&amp;pages=2077-2088&amp;pmid=35796628&amp;doi=10.1111/pbi.13889&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270100-bib-0018">
<cite>

Fisher, R. A.
 (1919). XV—The correlation between relatives on the supposition of Mendelian inheritance. Transactions of the Royal Society of Edinburgh, 52(2), 399–433. 10.1017/S0080456800012163
</cite> [<a href="https://doi.org/10.1017/S0080456800012163" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Transactions%20of%20the%20Royal%20Society%20of%20Edinburgh&amp;title=XV%E2%80%94The%20correlation%20between%20relatives%20on%20the%20supposition%20of%20Mendelian%20inheritance&amp;author=R.%20A.%20Fisher&amp;volume=52&amp;issue=2&amp;publication_year=1919&amp;pages=399-433&amp;doi=10.1017/S0080456800012163&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270100-bib-0019">
<cite>

Fisher, R. A.
 (1930). The genetical theory of natural selection. Clarendon Press. 10.5962/bhl.title.27468
</cite> [<a href="https://doi.org/10.5962/bhl.title.27468" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?title=The%20genetical%20theory%20of%20natural%20selection&amp;author=R.%20A.%20Fisher&amp;publication_year=1930&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270100-bib-0020">
<cite>

Fitter, A. H.
 (1987). An architectural approach to the comparative ecology of plant root systems. New Phytologist, 106, 61–77. 10.1111/j.1469-8137.1987.tb04683.x
</cite> [<a href="https://doi.org/10.1111/j.1469-8137.1987.tb04683.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=New%20Phytologist&amp;title=An%20architectural%20approach%20to%20the%20comparative%20ecology%20of%20plant%20root%20systems&amp;author=A.%20H.%20Fitter&amp;volume=106&amp;publication_year=1987&amp;pages=61-77&amp;doi=10.1111/j.1469-8137.1987.tb04683.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270100-bib-0021">
<cite>

Fox, J.
, &amp; 
Weisberg, S.
 (2019). An R companion to applied regression (3rd ed.). Sage. <a href="https://socialsciences.mcmaster.ca/jfox/Books/Companion/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://socialsciences.mcmaster.ca/jfox/Books/Companion/</a>
</cite> [<a href="https://scholar.google.com/scholar_lookup?title=An%20R%20companion%20to%20applied%20regression&amp;author=J.%20Fox&amp;author=S.%20Weisberg&amp;publication_year=2019&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270100-bib-0022">
<cite>

Gamuyao, R.
, 
Chin, J. H.
, 
Pariasca‐Tanaka, J.
, 
Pesaresi, P.
, 
Catausan, S.
, 
Dalid, C.
, 
Slamet‐Loedin, I.
, 
Tecson‐Mendoza, E. M.
, 
Wissuwa, M.
, &amp; 
Heuer, S.
 (2012). The protein kinase <em>PSTOL 1</em> from traditional rice confers tolerance of phosphorus deficiency. Nature, 488(7412), 535–539. 10.1038/nature11346

</cite> [<a href="https://doi.org/10.1038/nature11346" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22914168/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=The%20protein%20kinase%20PSTOL%201%20from%20traditional%20rice%20confers%20tolerance%20of%20phosphorus%20deficiency&amp;author=R.%20Gamuyao&amp;author=J.%20H.%20Chin&amp;author=J.%20Pariasca%E2%80%90Tanaka&amp;author=P.%20Pesaresi&amp;author=S.%20Catausan&amp;volume=488&amp;issue=7412&amp;publication_year=2012&amp;pages=535-539&amp;pmid=22914168&amp;doi=10.1038/nature11346&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270100-bib-0023">
<cite>

Gaudin, A. C. M.
, 
McClymont, S. A.
, 
Holmes, B. M.
, 
Lyons, E.
, &amp; 
Raizada, M. N.
 (2011). Novel temporal, fine‐scale and growth variation phenotypes in roots of adult‐stage maize (<em>Zea mays</em> L.) in response to low nitrogen stress. Plant, Cell &amp; Environment, 34(12), 2122–2137. 10.1111/j.1365-3040.2011.02409.x
</cite> [<a href="https://doi.org/10.1111/j.1365-3040.2011.02409.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21848860/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant,%20Cell%20&amp;%20Environment&amp;title=Novel%20temporal,%20fine%E2%80%90scale%20and%20growth%20variation%20phenotypes%20in%20roots%20of%20adult%E2%80%90stage%20maize%20(Zea%20mays%20L.)%20in%20response%20to%20low%20nitrogen%20stress&amp;author=A.%20C.%20M.%20Gaudin&amp;author=S.%20A.%20McClymont&amp;author=B.%20M.%20Holmes&amp;author=E.%20Lyons&amp;author=M.%20N.%20Raizada&amp;volume=34&amp;issue=12&amp;publication_year=2011&amp;pages=2122-2137&amp;pmid=21848860&amp;doi=10.1111/j.1365-3040.2011.02409.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270100-bib-0024">
<cite>

Gu, Z.
 (2022). Complex heatmap visualization. IMeta, 1(3), e43. 10.1002/imt2.43

</cite> [<a href="https://doi.org/10.1002/imt2.43" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10989952/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38868715/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=IMeta&amp;title=Complex%20heatmap%20visualization&amp;author=Z.%20Gu&amp;volume=1&amp;issue=3&amp;publication_year=2022&amp;pages=e43&amp;pmid=38868715&amp;doi=10.1002/imt2.43&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270100-bib-0025">
<cite>

Gu, Z.
, 
Eils, R.
, &amp; 
Schlesner, M.
 (2016). Complex heatmaps reveal patterns and correlations in multidimensional genomic data. Bioinformatics, 32(18), 2847–2849. 10.1093/bioinformatics/btw313

</cite> [<a href="https://doi.org/10.1093/bioinformatics/btw313" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27207943/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Complex%20heatmaps%20reveal%20patterns%20and%20correlations%20in%20multidimensional%20genomic%20data&amp;author=Z.%20Gu&amp;author=R.%20Eils&amp;author=M.%20Schlesner&amp;volume=32&amp;issue=18&amp;publication_year=2016&amp;pages=2847-2849&amp;pmid=27207943&amp;doi=10.1093/bioinformatics/btw313&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270100-bib-0026">
<cite>

Helliwell, J. R.
, 
Sturrock, C. J.
, 
Mairhofer, S.
, 
Craigon, J.
, 
Ashton, R. W.
, 
Miller, A. J.
, 
Whalley, W. R.
, &amp; 
Mooney, S. J.
 (2017). The emergent rhizosphere: Imaging the development of the porous architecture at the root‐soil interface. Scientific Reports, 7(1), Article 14875. 10.1038/s41598-017-14904-w

</cite> [<a href="https://doi.org/10.1038/s41598-017-14904-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5665926/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29093533/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Scientific%20Reports&amp;title=The%20emergent%20rhizosphere:%20Imaging%20the%20development%20of%20the%20porous%20architecture%20at%20the%20root%E2%80%90soil%20interface&amp;author=J.%20R.%20Helliwell&amp;author=C.%20J.%20Sturrock&amp;author=S.%20Mairhofer&amp;author=J.%20Craigon&amp;author=R.%20W.%20Ashton&amp;volume=7&amp;issue=1&amp;publication_year=2017&amp;pages=14875&amp;pmid=29093533&amp;doi=10.1038/s41598-017-14904-w&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270100-bib-0027">
<cite>

Hill, W. G.
, 
Edwards, M. R.
, 
Ahmed, M.‐K. A.
, &amp; 
Thompson, R.
 (1983). Heritability of milk yield and composition at different levels and variability of production. Animal Science, 36(1), 59–68. 10.1017/S0003356100039933
</cite> [<a href="https://doi.org/10.1017/S0003356100039933" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Animal%20Science&amp;title=Heritability%20of%20milk%20yield%20and%20composition%20at%20different%20levels%20and%20variability%20of%20production&amp;author=W.%20G.%20Hill&amp;author=M.%20R.%20Edwards&amp;author=M.%E2%80%90K.%20A.%20Ahmed&amp;author=R.%20Thompson&amp;volume=36&amp;issue=1&amp;publication_year=1983&amp;pages=59-68&amp;doi=10.1017/S0003356100039933&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270100-bib-0028">
<cite>

Hochholdinger, F.
 (2009). The maize root system: Morphology, anatomy, and genetics. In Bennetzen J. L. &amp; Hake S. C. (Eds.), Handbook of maize: Its biology. Springer. 10.1007/978-0-387-79418-1_8
</cite> [<a href="https://doi.org/10.1007/978-0-387-79418-1_8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?title=Handbook%20of%20maize:%20Its%20biology&amp;author=F.%20Hochholdinger&amp;publication_year=2009&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270100-bib-0030">
<cite>

Hochholdinger, F.
, 
Yu, P.
, &amp; 
Marcon, C.
 (2018). Genetic control of root system development in maize. Trends in Plant Science, 23(1), 79–88. 10.1016/j.tplants.2017.10.004

</cite> [<a href="https://doi.org/10.1016/j.tplants.2017.10.004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29170008/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Trends%20in%20Plant%20Science&amp;title=Genetic%20control%20of%20root%20system%20development%20in%20maize&amp;author=F.%20Hochholdinger&amp;author=P.%20Yu&amp;author=C.%20Marcon&amp;volume=23&amp;issue=1&amp;publication_year=2018&amp;pages=79-88&amp;pmid=29170008&amp;doi=10.1016/j.tplants.2017.10.004&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270100-bib-0031">
<cite>

Hoffmann, A. A.
, &amp; 
Merilä, J.
 (1999). Heritable variation and evolution under favourable and unfavourable conditions. Trends in Ecology &amp; Evolution, 14(3), 96–101. 10.1016/S0169-5347(99)01595-5

</cite> [<a href="https://doi.org/10.1016/S0169-5347(99)01595-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10322508/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Trends%20in%20Ecology%20&amp;%20Evolution&amp;title=Heritable%20variation%20and%20evolution%20under%20favourable%20and%20unfavourable%20conditions&amp;author=A.%20A.%20Hoffmann&amp;author=J.%20Meril%C3%A4&amp;volume=14&amp;issue=3&amp;publication_year=1999&amp;pages=96-101&amp;pmid=10322508&amp;doi=10.1016/S0169-5347(99)01595-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270100-bib-0032">
<cite>

Hotelling, H.
 (1933). Analysis of a complex of statistical variables into principal components. Journal of Educational Psychology, 24(6), 417–441. 10.1037/h0071325
</cite> [<a href="https://doi.org/10.1037/h0071325" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Educational%20Psychology&amp;title=Analysis%20of%20a%20complex%20of%20statistical%20variables%20into%20principal%20components&amp;author=H.%20Hotelling&amp;volume=24&amp;issue=6&amp;publication_year=1933&amp;pages=417-441&amp;doi=10.1037/h0071325&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270100-bib-0033">
<cite>

Houle, D.
, 
Govindaraju, D. R.
, &amp; 
Omholt, S.
 (2010). Phenomics: The next challenge. Nature Reviews Genetics, 11(12), 855–866. 10.1038/nrg2897
</cite> [<a href="https://doi.org/10.1038/nrg2897" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21085204/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Reviews%20Genetics&amp;title=Phenomics:%20The%20next%20challenge&amp;author=D.%20Houle&amp;author=D.%20R.%20Govindaraju&amp;author=S.%20Omholt&amp;volume=11&amp;issue=12&amp;publication_year=2010&amp;pages=855-866&amp;pmid=21085204&amp;doi=10.1038/nrg2897&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270100-bib-0034">
<cite>

Ilic, K.
, 
Kellogg, E. A.
, 
Jaiswal, P.
, 
Zapata, F.
, 
Stevens, P. F.
, 
Vincent, L. P.
, 
Avraham, S.
, 
Reiser, L.
, 
Pujar, A.
, 
Sachs, M. M.
, 
Whitman, N. T.
, 
McCouch, S. R.
, 
Schaeffer, M. L.
, 
Ware, D. H.
, 
Stein, L. D.
, &amp; 
Rhee, S. Y.
 (2007). The plant structure ontology, a unified vocabulary of anatomy and morphology of a flowering plant. Plant Physiology, 143(2), 587–599. 10.1104/pp.106.092825

</cite> [<a href="https://doi.org/10.1104/pp.106.092825" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1803752/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17142475/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Physiology&amp;title=The%20plant%20structure%20ontology,%20a%20unified%20vocabulary%20of%20anatomy%20and%20morphology%20of%20a%20flowering%20plant&amp;author=K.%20Ilic&amp;author=E.%20A.%20Kellogg&amp;author=P.%20Jaiswal&amp;author=F.%20Zapata&amp;author=P.%20F.%20Stevens&amp;volume=143&amp;issue=2&amp;publication_year=2007&amp;pages=587-599&amp;pmid=17142475&amp;doi=10.1104/pp.106.092825&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270100-bib-0035">
<cite>

Jenkins, M.
 (1930). Heritable characters of maize. Journal of Heredity, 21(2), 79–80. 10.1093/oxfordjournals.jhered.a103287
</cite> [<a href="https://doi.org/10.1093/oxfordjournals.jhered.a103287" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Heredity&amp;title=Heritable%20characters%20of%20maize&amp;author=M.%20Jenkins&amp;volume=21&amp;issue=2&amp;publication_year=1930&amp;pages=79-80&amp;doi=10.1093/oxfordjournals.jhered.a103287&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270100-bib-0036">
<cite>

Jiang, N.
, 
Floro, E.
, 
Bray, A. L.
, 
Laws, B.
, 
Duncan, K. E.
, &amp; 
Topp, C. N.
 (2019). Three‐dimensional time‐lapse analysis reveals multiscale relationships in maize root systems with contrasting architectures. The Plant Cell, 31(8), 1708–1722. 10.1105/tpc.19.00015

</cite> [<a href="https://doi.org/10.1105/tpc.19.00015" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6713302/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31123089/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=The%20Plant%20Cell&amp;title=Three%E2%80%90dimensional%20time%E2%80%90lapse%20analysis%20reveals%20multiscale%20relationships%20in%20maize%20root%20systems%20with%20contrasting%20architectures&amp;author=N.%20Jiang&amp;author=E.%20Floro&amp;author=A.%20L.%20Bray&amp;author=B.%20Laws&amp;author=K.%20E.%20Duncan&amp;volume=31&amp;issue=8&amp;publication_year=2019&amp;pages=1708-1722&amp;pmid=31123089&amp;doi=10.1105/tpc.19.00015&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270100-bib-0037">
<cite>

Johnson, S. N.
, &amp; 
Rasmann, S.
 (2015). Root‐feeding insects and their interactions with organisms in the rhizosphere. Annual Review of Entomology, 60, 517–535. 10.1146/annurev-ento-010814-020608
</cite> [<a href="https://doi.org/10.1146/annurev-ento-010814-020608" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25564744/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Annual%20Review%20of%20Entomology&amp;title=Root%E2%80%90feeding%20insects%20and%20their%20interactions%20with%20organisms%20in%20the%20rhizosphere&amp;author=S.%20N.%20Johnson&amp;author=S.%20Rasmann&amp;volume=60&amp;publication_year=2015&amp;pages=517-535&amp;pmid=25564744&amp;doi=10.1146/annurev-ento-010814-020608&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270100-bib-0038">
<cite>

Ju, Y.
, 
Liu, A. E.
, 
Oestreich, K.
, 
Wang, T.
, 
Topp, C. N.
, &amp; 
Ju, T.
 (2024). TopoRoot+: Computing whorl and soil line traits of field‐excavated maize roots from CT imaging. Plant Methods, 20(1), Article 132. 10.1186/s13007-024-01240-0

</cite> [<a href="https://doi.org/10.1186/s13007-024-01240-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11348750/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39187896/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Methods&amp;title=TopoRoot+:%20Computing%20whorl%20and%20soil%20line%20traits%20of%20field%E2%80%90excavated%20maize%20roots%20from%20CT%20imaging&amp;author=Y.%20Ju&amp;author=A.%20E.%20Liu&amp;author=K.%20Oestreich&amp;author=T.%20Wang&amp;author=C.%20N.%20Topp&amp;volume=20&amp;issue=1&amp;publication_year=2024&amp;pages=132&amp;pmid=39187896&amp;doi=10.1186/s13007-024-01240-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270100-bib-0039">
<cite>

Jung, J. K. H.
, &amp; 
McCouch, S.
 (2013). Getting to the roots of it: Genetic and hormonal control of root architecture. Frontiers in Plant Science, 4(6), Article 186. 10.3389/fpls.2013.00186

</cite> [<a href="https://doi.org/10.3389/fpls.2013.00186" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3685011/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23785372/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Plant%20Science&amp;title=Getting%20to%20the%20roots%20of%20it:%20Genetic%20and%20hormonal%20control%20of%20root%20architecture&amp;author=J.%20K.%20H.%20Jung&amp;author=S.%20McCouch&amp;volume=4&amp;issue=6&amp;publication_year=2013&amp;pages=186&amp;pmid=23785372&amp;doi=10.3389/fpls.2013.00186&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270100-bib-0040">
<cite>

Karlova, R.
, 
Boer, D.
, 
Hayes, S.
, &amp; 
Testerink, C.
 (2021). Root plasticity under abiotic stress. Plant Physiology, 187(3), 1057–1070. 10.1093/plphys/kiab392

</cite> [<a href="https://doi.org/10.1093/plphys/kiab392" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8566202/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34734279/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Physiology&amp;title=Root%20plasticity%20under%20abiotic%20stress&amp;author=R.%20Karlova&amp;author=D.%20Boer&amp;author=S.%20Hayes&amp;author=C.%20Testerink&amp;volume=187&amp;issue=3&amp;publication_year=2021&amp;pages=1057-1070&amp;pmid=34734279&amp;doi=10.1093/plphys/kiab392&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270100-bib-0041">
<cite>

Kassambara, A.
, &amp; 
Mundt, F.
 (2020). factoextra: Extract and Visualize the Results of Multivariate Data Analyses [Computer software]. CRAN. <a href="https://cran.r-project.org/package=factoextra" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://cran.r‐project.org/package=factoextra</a>
</cite> [<a href="https://scholar.google.com/scholar_lookup?Kassambara,%20A.%20,%20&amp;%20Mundt,%20F.%20(2020).%20factoextra:%20Extract%20and%20Visualize%20the%20Results%20of%20Multivariate%20Data%20Analyses%20%5BComputer%20software%5D.%20CRAN.%20https://cran.r%E2%80%90project.org/package=factoextra" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270100-bib-0042">
<cite>

Klei, L.
, 
Luca, D.
, 
Devlin, B.
, &amp; 
Roeder, K.
 (2008). Pleiotropy and principal components of heritability combine to increase power for association analysis. Genetic Epidemiology, 32(1), 9–19. 10.1002/gepi.20257

</cite> [<a href="https://doi.org/10.1002/gepi.20257" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17922480/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genetic%20Epidemiology&amp;title=Pleiotropy%20and%20principal%20components%20of%20heritability%20combine%20to%20increase%20power%20for%20association%20analysis&amp;author=L.%20Klei&amp;author=D.%20Luca&amp;author=B.%20Devlin&amp;author=K.%20Roeder&amp;volume=32&amp;issue=1&amp;publication_year=2008&amp;pages=9-19&amp;pmid=17922480&amp;doi=10.1002/gepi.20257&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270100-bib-0043">
<cite>

Landi, P.
, 
Giuliani, S.
, 
Salvi, S.
, 
Ferri, M.
, 
Tuberosa, R.
, &amp; 
Sanguineti, M. C.
 (2010). Characterization of root‐yield‐1.06, a major constitutive QTL for root and agronomic traits in maize across water regimes. Journal of Experimental Botany, 61(13), 3553–3562. 10.1093/jxb/erq192

</cite> [<a href="https://doi.org/10.1093/jxb/erq192" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20627896/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Experimental%20Botany&amp;title=Characterization%20of%20root%E2%80%90yield%E2%80%901.06,%20a%20major%20constitutive%20QTL%20for%20root%20and%20agronomic%20traits%20in%20maize%20across%20water%20regimes&amp;author=P.%20Landi&amp;author=S.%20Giuliani&amp;author=S.%20Salvi&amp;author=M.%20Ferri&amp;author=R.%20Tuberosa&amp;volume=61&amp;issue=13&amp;publication_year=2010&amp;pages=3553-3562&amp;pmid=20627896&amp;doi=10.1093/jxb/erq192&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270100-bib-0044">
<cite>

Landi, P.
, 
Sanguineti, M. C.
, 
Darrah, L. L.
, 
Giuliani, M. M.
, 
Salvi, S.
, 
Conti, S.
, &amp; 
Tuberosa, R.
 (2002). Detection of QTLs for vertical root pulling resistance in maize and overlap with QTLs for root traits in hydroponics and for grain yield under different water regimes. Maydica, 47(3–4), 233–243.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Maydica&amp;title=Detection%20of%20QTLs%20for%20vertical%20root%20pulling%20resistance%20in%20maize%20and%20overlap%20with%20QTLs%20for%20root%20traits%20in%20hydroponics%20and%20for%20grain%20yield%20under%20different%20water%20regimes&amp;author=P.%20Landi&amp;author=M.%20C.%20Sanguineti&amp;author=L.%20L.%20Darrah&amp;author=M.%20M.%20Giuliani&amp;author=S.%20Salvi&amp;volume=47&amp;issue=3%E2%80%934&amp;publication_year=2002&amp;pages=233-243&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270100-bib-0045">
<cite>

Lareen, A.
, 
Burton, F.
, &amp; 
Schäfer, P.
 (2016). Plant root‐microbe communication in shaping root microbiomes. Plant Molecular Biology, 90(6), 575–587. 10.1007/s11103-015-0417-8

</cite> [<a href="https://doi.org/10.1007/s11103-015-0417-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4819777/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26729479/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Molecular%20Biology&amp;title=Plant%20root%E2%80%90microbe%20communication%20in%20shaping%20root%20microbiomes&amp;author=A.%20Lareen&amp;author=F.%20Burton&amp;author=P.%20Sch%C3%A4fer&amp;volume=90&amp;issue=6&amp;publication_year=2016&amp;pages=575-587&amp;pmid=26729479&amp;doi=10.1007/s11103-015-0417-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270100-bib-0046">
<cite>

Lê, S.
, 
Josse, J.
, &amp; 
Husson, F.
 (2008). {FactoMineR}: A package for multivariate analysis. Journal of Statistical Software, 25(1), 1–18. 10.18637/jss.v025.i01
</cite> [<a href="https://doi.org/10.18637/jss.v025.i01" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Statistical%20Software&amp;title=%7BFactoMineR%7D:%20A%20package%20for%20multivariate%20analysis&amp;author=S.%20L%C3%AA&amp;author=J.%20Josse&amp;author=F.%20Husson&amp;volume=25&amp;issue=1&amp;publication_year=2008&amp;pages=1-18&amp;doi=10.18637/jss.v025.i01&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270100-bib-0047">
<cite>

Leiboff, S.
, 
Li, X.
, 
Hu, H. C.
, 
Todt, N.
, 
Yang, J.
, 
Li, X.
, 
Yu, X.
, 
Muehlbauer, G. J.
, 
Timmermans, M. C. P.
, 
Yu, J.
, 
Schnable, P. S.
, &amp; 
Scanlon, M. J.
 (2015). Genetic control of morphometric diversity in the maize shoot apical meristem. Nature Communications, 6, Article 8974. 10.1038/ncomms9974
</cite> [<a href="https://doi.org/10.1038/ncomms9974" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4673881/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26584889/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Communications&amp;title=Genetic%20control%20of%20morphometric%20diversity%20in%20the%20maize%20shoot%20apical%20meristem&amp;author=S.%20Leiboff&amp;author=X.%20Li&amp;author=H.%20C.%20Hu&amp;author=N.%20Todt&amp;author=J.%20Yang&amp;volume=6&amp;publication_year=2015&amp;pages=8974&amp;pmid=26584889&amp;doi=10.1038/ncomms9974&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270100-bib-0048">
<cite>

Li, M.
, 
Frank, M. H.
, 
Coneva, V.
, 
Mio, W.
, 
Chitwood, D. H.
, &amp; 
Topp, C. N.
 (2018). The persistent homology mathematical framework provides enhanced genotype‐to‐phenotype associations for plant morphology. Plant Physiology, 177(4), 1382–1395. 10.1104/pp.18.00104

</cite> [<a href="https://doi.org/10.1104/pp.18.00104" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6084663/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29871979/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Physiology&amp;title=The%20persistent%20homology%20mathematical%20framework%20provides%20enhanced%20genotype%E2%80%90to%E2%80%90phenotype%20associations%20for%20plant%20morphology&amp;author=M.%20Li&amp;author=M.%20H.%20Frank&amp;author=V.%20Coneva&amp;author=W.%20Mio&amp;author=D.%20H.%20Chitwood&amp;volume=177&amp;issue=4&amp;publication_year=2018&amp;pages=1382-1395&amp;pmid=29871979&amp;doi=10.1104/pp.18.00104&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270100-bib-0049">
<cite>

Li, M.
, 
Liu, Z.
, 
Jiang, N.
, 
Laws, B.
, 
Tiskevich, C.
, 
Moose, S. P.
, &amp; 
Topp, C. N.
 (2023). Topological data analysis expands the genotype to phenotype map for 3D maize root system architecture. Frontiers in Plant Science, 14(1), 1260005. 10.3389/fpls.2023.1260005

</cite> [<a href="https://doi.org/10.3389/fpls.2023.1260005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10822944/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38288407/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Plant%20Science&amp;title=Topological%20data%20analysis%20expands%20the%20genotype%20to%20phenotype%20map%20for%203D%20maize%20root%20system%20architecture&amp;author=M.%20Li&amp;author=Z.%20Liu&amp;author=N.%20Jiang&amp;author=B.%20Laws&amp;author=C.%20Tiskevich&amp;volume=14&amp;issue=1&amp;publication_year=2023&amp;pages=1260005&amp;pmid=38288407&amp;doi=10.3389/fpls.2023.1260005&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270100-bib-0050">
<cite>

Lipka, A. E.
, 
Tian, F.
, 
Wang, Q.
, 
Peiffer, J.
, 
Li, M.
, 
Bradbury, P. J.
, 
Gore, M. A.
, 
Buckler, E. S.
, &amp; 
Zhang, Z.
 (2012). GAPIT: Genome association and prediction integrated tool. Bioinformatics, 28(18), 2397–2399. 10.1093/bioinformatics/bts444

</cite> [<a href="https://doi.org/10.1093/bioinformatics/bts444" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22796960/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=GAPIT:%20Genome%20association%20and%20prediction%20integrated%20tool&amp;author=A.%20E.%20Lipka&amp;author=F.%20Tian&amp;author=Q.%20Wang&amp;author=J.%20Peiffer&amp;author=M.%20Li&amp;volume=28&amp;issue=18&amp;publication_year=2012&amp;pages=2397-2399&amp;pmid=22796960&amp;doi=10.1093/bioinformatics/bts444&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270100-bib-0051">
<cite>

Liu, A.
, 
Bagnall, G. C.
, 
Bray, A. L.
, &amp; 
Topp, C.
 (2023, October 29–November 1). A rootless1 knockdown allele affects maize nodal root development, changing root system architecture and function [Conference abstract]. In 
ASA, CSSA, SSSA International Annual Meeting
, St. Louis, MO. <a href="https://scisoc.confex.com/scisoc/2023am/meetingapp.cgi/Paper/154360" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://scisoc.confex.com/scisoc/2023am/meetingapp.cgi/Paper/154360</a>
</cite> [<a href="https://scholar.google.com/scholar_lookup?Liu,%20A.%20,%20Bagnall,%20G.%20C.%20,%20Bray,%20A.%20L.%20,%20&amp;%20Topp,%20C.%20(2023,%20October%2029%E2%80%93November%201).%20A%20rootless1%20knockdown%20allele%20affects%20maize%20nodal%20root%20development,%20changing%20root%20system%20architecture%20and%20function%20%5BConference%20abstract%5D.%20In%20ASA,%20CSSA,%20SSSA%20International%20Annual%20Meeting%20,%20St.%20Louis,%20MO.%20https://scisoc.confex.com/scisoc/2023am/meetingapp.cgi/Paper/154360" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270100-bib-0052">
<cite>

Liu, S.
, 
Barrow, C. S.
, 
Hanlon, M.
, 
Lynch, J. P.
, &amp; 
Bucksch, A.
 (2021). DIRT/3D: 3D root phenotyping for field‐grown maize (<em>Zea mays</em>). Plant Physiology, 187(2), 739–757. 10.1093/plphys/kiab311

</cite> [<a href="https://doi.org/10.1093/plphys/kiab311" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8491025/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34608967/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Physiology&amp;title=DIRT/3D:%203D%20root%20phenotyping%20for%20field%E2%80%90grown%20maize%20(Zea%20mays)&amp;author=S.%20Liu&amp;author=C.%20S.%20Barrow&amp;author=M.%20Hanlon&amp;author=J.%20P.%20Lynch&amp;author=A.%20Bucksch&amp;volume=187&amp;issue=2&amp;publication_year=2021&amp;pages=739-757&amp;pmid=34608967&amp;doi=10.1093/plphys/kiab311&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270100-bib-0053">
<cite>

Liu, X.
, 
Huang, M.
, 
Fan, B.
, 
Buckler, E. S.
, &amp; 
Zhang, Z.
 (2016). Iterative usage of fixed and random effect models for powerful and efficient genome‐wide association studies. PLOS Genetics, 12(3), e1005957. 10.1371/journal.pgen.1005767

</cite> [<a href="https://doi.org/10.1371/journal.pgen.1005767" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4734661/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26828793/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLOS%20Genetics&amp;title=Iterative%20usage%20of%20fixed%20and%20random%20effect%20models%20for%20powerful%20and%20efficient%20genome%E2%80%90wide%20association%20studies&amp;author=X.%20Liu&amp;author=M.%20Huang&amp;author=B.%20Fan&amp;author=E.%20S.%20Buckler&amp;author=Z.%20Zhang&amp;volume=12&amp;issue=3&amp;publication_year=2016&amp;pages=e1005957&amp;pmid=26828793&amp;doi=10.1371/journal.pgen.1005767&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270100-bib-0054">
<cite>

Lobet, G.
 (2017). Image analysis in plant sciences: Publish then perish. Trends in Plant Science, 22(7), 559–566. 10.1016/j.tplants.2017.05.002

</cite> [<a href="https://doi.org/10.1016/j.tplants.2017.05.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28571940/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Trends%20in%20Plant%20Science&amp;title=Image%20analysis%20in%20plant%20sciences:%20Publish%20then%20perish&amp;author=G.%20Lobet&amp;volume=22&amp;issue=7&amp;publication_year=2017&amp;pages=559-566&amp;pmid=28571940&amp;doi=10.1016/j.tplants.2017.05.002&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270100-bib-0055">
<cite>

Lynch, J. P.
 (2013). Steep, cheap and deep: An ideotype to optimize water and N acquisition by maize root systems. Annals of Botany, 112(2), 347–357. 10.1093/aob/mcs293

</cite> [<a href="https://doi.org/10.1093/aob/mcs293" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3698384/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23328767/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Annals%20of%20Botany&amp;title=Steep,%20cheap%20and%20deep:%20An%20ideotype%20to%20optimize%20water%20and%20N%20acquisition%20by%20maize%20root%20systems&amp;author=J.%20P.%20Lynch&amp;volume=112&amp;issue=2&amp;publication_year=2013&amp;pages=347-357&amp;pmid=23328767&amp;doi=10.1093/aob/mcs293&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270100-bib-0056">
<cite>

Lynch, J. P.
, &amp; 
Brown, K. M.
 (2001). Topsoil foraging—An architectural adaptation of plants to low phosphorus availability. Plant and Soil, 237(2), 225–237. 10.1023/A:1013324727040
</cite> [<a href="https://doi.org/10.1023/A:1013324727040" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20and%20Soil&amp;title=Topsoil%20foraging%E2%80%94An%20architectural%20adaptation%20of%20plants%20to%20low%20phosphorus%20availability&amp;author=J.%20P.%20Lynch&amp;author=K.%20M.%20Brown&amp;volume=237&amp;issue=2&amp;publication_year=2001&amp;pages=225-237&amp;doi=10.1023/A:1013324727040&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270100-bib-0057">
<cite>

Lynch, J. P.
, 
Chimungu, J. G.
, &amp; 
Brown, K. M.
 (2014). Root anatomical phenes associated with water acquisition from drying soil: Targets for crop improvement. Journal of Experimental Botany, 65(21), 6155–6166. 10.1093/jxb/eru162

</cite> [<a href="https://doi.org/10.1093/jxb/eru162" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24759880/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Experimental%20Botany&amp;title=Root%20anatomical%20phenes%20associated%20with%20water%20acquisition%20from%20drying%20soil:%20Targets%20for%20crop%20improvement&amp;author=J.%20P.%20Lynch&amp;author=J.%20G.%20Chimungu&amp;author=K.%20M.%20Brown&amp;volume=65&amp;issue=21&amp;publication_year=2014&amp;pages=6155-6166&amp;pmid=24759880&amp;doi=10.1093/jxb/eru162&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270100-bib-0058">
<cite>

Lynch, M.
, &amp; 
Walsh, B.
 (1998). Genetics and analysis of quantitative traits. Sinauer.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=Genetics%20and%20analysis%20of%20quantitative%20traits&amp;author=M.%20Lynch&amp;author=B.%20Walsh&amp;publication_year=1998&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270100-bib-0059">
<cite>

Macgregor, S.
, 
Cornes, B. K.
, 
Martin, N. G.
, &amp; 
Visscher, P. M.
 (2006). Bias, precision and heritability of self‐reported and clinically measured height in Australian twins. Human Genetics, 120(4), 571–580. 10.1007/s00439-006-0240-z

</cite> [<a href="https://doi.org/10.1007/s00439-006-0240-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16933140/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Human%20Genetics&amp;title=Bias,%20precision%20and%20heritability%20of%20self%E2%80%90reported%20and%20clinically%20measured%20height%20in%20Australian%20twins&amp;author=S.%20Macgregor&amp;author=B.%20K.%20Cornes&amp;author=N.%20G.%20Martin&amp;author=P.%20M.%20Visscher&amp;volume=120&amp;issue=4&amp;publication_year=2006&amp;pages=571-580&amp;pmid=16933140&amp;doi=10.1007/s00439-006-0240-z&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270100-bib-0060"><cite>

Mattupalli, C.
, 
Seethepalli, A.
, 
York, L. M.
, &amp; 
Young, C. A.
 (2018). Digital imaging to evaluate root system architectural changes associated with soil biotic factors. <em>BioRxiv</em>. <a href="https://api.semanticscholar.org/CorpusID:92292738" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://api.semanticscholar.org/CorpusID:92292738</a>
</cite></li>
<li id="tpg270100-bib-0061">
<cite>

McKay, J. K.
, 
Alvarado, C. A.
, 
Babbitt, G. R.
, 
Palmiscno, K. J.
, 
Turner, C. W.
, &amp; 
Whitehill, B. C.
 (2023). Device for automated crop root sampling (U.S. Patent No. 11,703,404). U.S. Patent and Trademark Office. <a href="https://www.uspto.gov/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.uspto.gov/</a>
</cite> [<a href="https://scholar.google.com/scholar_lookup?title=Device%20for%20automated%20crop%20root%20sampling&amp;author=J.%20K.%20McKay&amp;author=C.%20A.%20Alvarado&amp;author=G.%20R.%20Babbitt&amp;author=K.%20J.%20Palmiscno&amp;author=C.%20W.%20Turner&amp;publication_year=2023&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270100-bib-0062">
<cite>

Mooney, S. J.
, 
Pridmore, T. P.
, 
Helliwell, J.
, &amp; 
Bennett, M. J.
 (2012). Developing X‐ray computed tomography to non‐invasively image 3‐D root systems architecture in soil. Plant and Soil, 352(1), 1–22. 10.1007/s11104-011-1039-9
</cite> [<a href="https://doi.org/10.1007/s11104-011-1039-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20and%20Soil&amp;title=Developing%20X%E2%80%90ray%20computed%20tomography%20to%20non%E2%80%90invasively%20image%203%E2%80%90D%20root%20systems%20architecture%20in%20soil&amp;author=S.%20J.%20Mooney&amp;author=T.%20P.%20Pridmore&amp;author=J.%20Helliwell&amp;author=M.%20J.%20Bennett&amp;volume=352&amp;issue=1&amp;publication_year=2012&amp;pages=1-22&amp;doi=10.1007/s11104-011-1039-9&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270100-bib-0063">
<cite>

Morris, E. C.
, 
Griffiths, M.
, 
Golebiowska, A.
, 
Mairhofer, S.
, 
Burr‐Hersey, J.
, 
Goh, T.
, 
von Wangenheim, D.
, 
Atkinson, B.
, 
Sturrock, C. J.
, 
Lynch, J. P.
, 
Vissenberg, K.
, 
Ritz, K.
, 
Wells, D. M.
, 
Mooney, S. J.
, &amp; 
Bennett, M. J.
 (2017). Shaping 3D root system architecture. Current Biology, 27(17), 919–930. 10.1016/j.cub.2017.06.043
</cite> [<a href="https://doi.org/10.1016/j.cub.2017.06.043" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28898665/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Current%20Biology&amp;title=Shaping%203D%20root%20system%20architecture&amp;author=E.%20C.%20Morris&amp;author=M.%20Griffiths&amp;author=A.%20Golebiowska&amp;author=S.%20Mairhofer&amp;author=J.%20Burr%E2%80%90Hersey&amp;volume=27&amp;issue=17&amp;publication_year=2017&amp;pages=919-930&amp;pmid=28898665&amp;doi=10.1016/j.cub.2017.06.043&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270100-bib-0064">
<cite>

Ortman, E. E.
, 
Peters, D. C.
, &amp; 
Fitzgerald, P. J.
 (1968). Vertical‐pull technique for evaluating tolerance of corn root systems to northern and western corn rootworms. Journal of Economic Entomology, 61(2), 373–375. 10.1093/jee/61.2.373
</cite> [<a href="https://doi.org/10.1093/jee/61.2.373" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Economic%20Entomology&amp;title=Vertical%E2%80%90pull%20technique%20for%20evaluating%20tolerance%20of%20corn%20root%20systems%20to%20northern%20and%20western%20corn%20rootworms&amp;author=E.%20E.%20Ortman&amp;author=D.%20C.%20Peters&amp;author=P.%20J.%20Fitzgerald&amp;volume=61&amp;issue=2&amp;publication_year=1968&amp;pages=373-375&amp;doi=10.1093/jee/61.2.373&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270100-bib-0065">
<cite>

Péret, B.
, 
De Rybel, B.
, 
Casimiro, I.
, 
Benková, E.
, 
Swarup, R.
, 
Laplaze, L.
, 
Beeckman, T.
, &amp; 
Bennett, M. J.
 (2009). Arabidopsis lateral root development: An emerging story. Trends in Plant Science, 14(7), 399–408. 10.1016/j.tplants.2009.05.002

</cite> [<a href="https://doi.org/10.1016/j.tplants.2009.05.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19559642/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Trends%20in%20Plant%20Science&amp;title=Arabidopsis%20lateral%20root%20development:%20An%20emerging%20story&amp;author=B.%20P%C3%A9ret&amp;author=B.%20De%20Rybel&amp;author=I.%20Casimiro&amp;author=E.%20Benkov%C3%A1&amp;author=R.%20Swarup&amp;volume=14&amp;issue=7&amp;publication_year=2009&amp;pages=399-408&amp;pmid=19559642&amp;doi=10.1016/j.tplants.2009.05.002&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270100-bib-0067">
<cite>

Petricka, J. J.
, 
Winter, C. M.
, &amp; 
Benfey, P. N.
 (2012). Control of Arabidopsis root development. Annual Review of Plant Biology, 63, 563–590. 10.1146/annurev-arplant-042811-105501
</cite> [<a href="https://doi.org/10.1146/annurev-arplant-042811-105501" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3646660/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22404466/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Annual%20Review%20of%20Plant%20Biology&amp;title=Control%20of%20Arabidopsis%20root%20development&amp;author=J.%20J.%20Petricka&amp;author=C.%20M.%20Winter&amp;author=P.%20N.%20Benfey&amp;volume=63&amp;publication_year=2012&amp;pages=563-590&amp;pmid=22404466&amp;doi=10.1146/annurev-arplant-042811-105501&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270100-bib-0068">
<cite>
R Core Team
. (2023). 
R: A language and environment for statistical computing
. R Foundation for Statistical Computing. <a href="https://www.r-project.org/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.r‐project.org/</a>
</cite> [<a href="https://scholar.google.com/scholar_lookup?R%20Core%20Team%20.%20(2023).%20R:%20A%20language%20and%20environment%20for%20statistical%20computing%20.%20R%20Foundation%20for%20Statistical%20Computing.%20https://www.r%E2%80%90project.org/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270100-bib-0069">
<cite>

Rajurkar, A. B.
, 
McCoy, S. M.
, 
Ruhter, J.
, 
Mulcrone, J.
, 
Freyfogle, L.
, &amp; 
Leakey, A. D. B.
 (2022). Installation and imaging of thousands of minirhizotrons to phenotype root systems of field‐grown plants. Plant Methods, 18(1), Article 39. 10.1186/s13007-022-00874-2

</cite> [<a href="https://doi.org/10.1186/s13007-022-00874-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8958774/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35346269/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Methods&amp;title=Installation%20and%20imaging%20of%20thousands%20of%20minirhizotrons%20to%20phenotype%20root%20systems%20of%20field%E2%80%90grown%20plants&amp;author=A.%20B.%20Rajurkar&amp;author=S.%20M.%20McCoy&amp;author=J.%20Ruhter&amp;author=J.%20Mulcrone&amp;author=L.%20Freyfogle&amp;volume=18&amp;issue=1&amp;publication_year=2022&amp;pages=39&amp;pmid=35346269&amp;doi=10.1186/s13007-022-00874-2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270100-bib-0070">
<cite>

Ren, W.
, 
Zhao, L.
, 
Liang, J.
, 
Wang, L.
, 
Chen, L.
, 
Li, P.
, 
Liu, Z.
, 
Li, X.
, 
Zhang, Z.
, 
Li, J.
, 
He, K.
, 
Zhao, Z.
, 
Ali, F.
, 
Mi, G.
, 
Yan, J.
, 
Zhang, F.
, 
Chen, F.
, 
Yuan, L.
, &amp; 
Pan, Q.
 (2022). Genome‐wide dissection of changes in maize root system architecture during modern breeding. Nature Plants, 8(12), 1408–1422. 10.1038/s41477-022-01274-z

</cite> [<a href="https://doi.org/10.1038/s41477-022-01274-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36396706/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Plants&amp;title=Genome%E2%80%90wide%20dissection%20of%20changes%20in%20maize%20root%20system%20architecture%20during%20modern%20breeding&amp;author=W.%20Ren&amp;author=L.%20Zhao&amp;author=J.%20Liang&amp;author=L.%20Wang&amp;author=L.%20Chen&amp;volume=8&amp;issue=12&amp;publication_year=2022&amp;pages=1408-1422&amp;pmid=36396706&amp;doi=10.1038/s41477-022-01274-z&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270100-bib-0071">
<cite>

Rice, B. R.
, 
Fernandes, S. B.
, &amp; 
Lipka, A. E.
 (2020). Multi‐trait genome‐wide association studies reveal loci associated with maize inflorescence and leaf architecture. Plant and Cell Physiology, 61(8), 1427–1437. 10.1093/pcp/pcaa039

</cite> [<a href="https://doi.org/10.1093/pcp/pcaa039" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32186727/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20and%20Cell%20Physiology&amp;title=Multi%E2%80%90trait%20genome%E2%80%90wide%20association%20studies%20reveal%20loci%20associated%20with%20maize%20inflorescence%20and%20leaf%20architecture&amp;author=B.%20R.%20Rice&amp;author=S.%20B.%20Fernandes&amp;author=A.%20E.%20Lipka&amp;volume=61&amp;issue=8&amp;publication_year=2020&amp;pages=1427-1437&amp;pmid=32186727&amp;doi=10.1093/pcp/pcaa039&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270100-bib-0072">
<cite>

Rogers, E. D.
, &amp; 
Benfey, P. N.
 (2015). Regulation of plant root system architecture: Implications for crop advancement. Current Opinion in Biotechnology, 32, 93–98. 10.1016/j.copbio.2014.11.015

</cite> [<a href="https://doi.org/10.1016/j.copbio.2014.11.015" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25448235/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Current%20Opinion%20in%20Biotechnology&amp;title=Regulation%20of%20plant%20root%20system%20architecture:%20Implications%20for%20crop%20advancement&amp;author=E.%20D.%20Rogers&amp;author=P.%20N.%20Benfey&amp;volume=32&amp;publication_year=2015&amp;pages=93-98&amp;pmid=25448235&amp;doi=10.1016/j.copbio.2014.11.015&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270100-bib-0073">
<cite>

Saengwilai, P.
, 
Tian, X.
, &amp; 
Lynch, J. P.
 (2014). Low crown root number enhances nitrogen acquisition from low‐nitrogen soils in maize. Plant Physiology, 166(2), 581–589. 10.1104/pp.113.232603

</cite> [<a href="https://doi.org/10.1104/pp.113.232603" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4213090/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24706553/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Physiology&amp;title=Low%20crown%20root%20number%20enhances%20nitrogen%20acquisition%20from%20low%E2%80%90nitrogen%20soils%20in%20maize&amp;author=P.%20Saengwilai&amp;author=X.%20Tian&amp;author=J.%20P.%20Lynch&amp;volume=166&amp;issue=2&amp;publication_year=2014&amp;pages=581-589&amp;pmid=24706553&amp;doi=10.1104/pp.113.232603&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270100-bib-0074">
<cite>

Scheres, B.
, 
McKhann, H. I.
, &amp; 
Van Den Berg, C.
 (1996). Roots redefined: Anatomical and genetic analysis of root development. Plant Physiology, 111(4), 959–964. 10.1104/pp.111.4.959

</cite> [<a href="https://doi.org/10.1104/pp.111.4.959" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC160962/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12226340/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Physiology&amp;title=Roots%20redefined:%20Anatomical%20and%20genetic%20analysis%20of%20root%20development&amp;author=B.%20Scheres&amp;author=H.%20I.%20McKhann&amp;author=C.%20Van%20Den%20Berg&amp;volume=111&amp;issue=4&amp;publication_year=1996&amp;pages=959-964&amp;pmid=12226340&amp;doi=10.1104/pp.111.4.959&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270100-bib-0075">
<cite>

Schneider, H. M.
, 
Lor, V. S. N.
, 
Hanlon, M. T.
, 
Perkins, A.
, 
Kaeppler, S. M.
, 
Borkar, A. N.
, 
Bhosale, R.
, 
Zhang, X.
, 
Rodriguez, J.
, 
Bucksch, A.
, 
Bennett, M. J.
, 
Brown, K. M.
, &amp; 
Lynch, J. P.
 (2022). Root angle in maize influences nitrogen capture and is regulated by calcineurin B‐like protein (CBL)‐interacting serine/threonine‐protein kinase 15 (ZmCIPK15). Plant Cell and Environment, 45(3), 837–853. 10.1111/pce.14135
</cite> [<a href="https://doi.org/10.1111/pce.14135" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9544310/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34169548/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Cell%20and%20Environment&amp;title=Root%20angle%20in%20maize%20influences%20nitrogen%20capture%20and%20is%20regulated%20by%20calcineurin%20B%E2%80%90like%20protein%20(CBL)%E2%80%90interacting%20serine/threonine%E2%80%90protein%20kinase%2015%20(ZmCIPK15)&amp;author=H.%20M.%20Schneider&amp;author=V.%20S.%20N.%20Lor&amp;author=M.%20T.%20Hanlon&amp;author=A.%20Perkins&amp;author=S.%20M.%20Kaeppler&amp;volume=45&amp;issue=3&amp;publication_year=2022&amp;pages=837-853&amp;pmid=34169548&amp;doi=10.1111/pce.14135&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270100-bib-0076">
<cite>

Schneider, H. M.
, &amp; 
Lynch, J. P.
 (2020). Should root plasticity be a crop breeding target?
Frontiers in Plant Science, 11(5), Article 546. 10.3389/fpls.2020.00546

</cite> [<a href="https://doi.org/10.3389/fpls.2020.00546" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7243933/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32499798/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Plant%20Science&amp;title=Should%20root%20plasticity%20be%20a%20crop%20breeding%20target?&amp;author=H.%20M.%20Schneider&amp;author=J.%20P.%20Lynch&amp;volume=11&amp;issue=5&amp;publication_year=2020&amp;pages=546&amp;pmid=32499798&amp;doi=10.3389/fpls.2020.00546&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270100-bib-0077">
<cite>

Shao, M. R.
, 
Jiang, N.
, 
Li, M.
, 
Howard, A.
, 
Lehner, K.
, 
Mullen, J. L.
, 
Gunn, S. L.
, 
McKay, J. K.
, &amp; 
Topp, C. N.
 (2021). Complementary phenotyping of maize root system architecture by root pulling force and X‐ray imaging. Plant Phenomics, 2021, 9859254. 10.34133/2021/9859254

</cite> [<a href="https://doi.org/10.34133/2021/9859254" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8603028/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34870229/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Phenomics&amp;title=Complementary%20phenotyping%20of%20maize%20root%20system%20architecture%20by%20root%20pulling%20force%20and%20X%E2%80%90ray%20imaging&amp;author=M.%20R.%20Shao&amp;author=N.%20Jiang&amp;author=M.%20Li&amp;author=A.%20Howard&amp;author=K.%20Lehner&amp;volume=2021&amp;publication_year=2021&amp;pages=9859254&amp;pmid=34870229&amp;doi=10.34133/2021/9859254&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270100-bib-0078">
<cite>

Stuber, C. W.
 (1995). Mapping and manipulating quantitative traits in maize. Trends in Genetics, 11(12), 477–481. 10.1016/S0168-9525(00)89156-8

</cite> [<a href="https://doi.org/10.1016/S0168-9525(00)89156-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/8533163/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Trends%20in%20Genetics&amp;title=Mapping%20and%20manipulating%20quantitative%20traits%20in%20maize&amp;author=C.%20W.%20Stuber&amp;volume=11&amp;issue=12&amp;publication_year=1995&amp;pages=477-481&amp;pmid=8533163&amp;doi=10.1016/S0168-9525(00)89156-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270100-bib-0079">
<cite>

Sun, B.
, 
Gao, Y.
, &amp; 
Lynch, J. P.
 (2018). Large crown root number improves topsoil foraging and phosphorus acquisition. Plant Physiology, 177(1), 90–104. 10.1104/pp.18.00234

</cite> [<a href="https://doi.org/10.1104/pp.18.00234" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5933112/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29618638/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Physiology&amp;title=Large%20crown%20root%20number%20improves%20topsoil%20foraging%20and%20phosphorus%20acquisition&amp;author=B.%20Sun&amp;author=Y.%20Gao&amp;author=J.%20P.%20Lynch&amp;volume=177&amp;issue=1&amp;publication_year=2018&amp;pages=90-104&amp;pmid=29618638&amp;doi=10.1104/pp.18.00234&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270100-bib-0080">
<cite>

Taramino, G.
, &amp; 
Sakai, H.
 (2009). Plants with altered root architecture, involving the rt1 gene, related constructs and methods (U.S. Patent No. 20,090,044,293). U.S. Patent and Trademark Office. <a href="https://www.uspto.gov/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.uspto.gov/</a>
</cite> [<a href="https://scholar.google.com/scholar_lookup?title=Plants%20with%20altered%20root%20architecture,%20involving%20the%20rt1%20gene,%20related%20constructs%20and%20methods&amp;author=G.%20Taramino&amp;author=H.%20Sakai&amp;publication_year=2009&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270100-bib-0082">
<cite>

Topp, C. N.
, 
Iyer‐Pascuzzi, A. S.
, 
Anderson, J. T.
, 
Lee, C. R.
, 
Zurek, P. R.
, 
Symonova, O.
, 
Zheng, Y.
, 
Bucksch, A.
, 
Mileyko, Y.
, 
Galkovskyi, T.
, 
Moore, B. T.
, 
Harer, J.
, 
Edelsbrunner, H.
, 
Mitchell‐Olds, T.
, 
Weitz, J. S.
, &amp; 
Benfey, P. N.
 (2013). 3D phenotyping and quantitative trait locus mapping identify core regions of the rice genome controlling root architecture. Proceedings of the National Academy of Sciences of the United States of America, 110(18), E1695–E1704. 10.1073/pnas.1304354110

</cite> [<a href="https://doi.org/10.1073/pnas.1304354110" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3645568/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23580618/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proceedings%20of%20the%20National%20Academy%20of%20Sciences%20of%20the%20United%20States%20of%20America&amp;title=3D%20phenotyping%20and%20quantitative%20trait%20locus%20mapping%20identify%20core%20regions%20of%20the%20rice%20genome%20controlling%20root%20architecture&amp;author=C.%20N.%20Topp&amp;author=A.%20S.%20Iyer%E2%80%90Pascuzzi&amp;author=J.%20T.%20Anderson&amp;author=C.%20R.%20Lee&amp;author=P.%20R.%20Zurek&amp;volume=110&amp;issue=18&amp;publication_year=2013&amp;pages=E1695-E1704&amp;pmid=23580618&amp;doi=10.1073/pnas.1304354110&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270100-bib-0083">
<cite>

Tovar, J. C.
, 
Hoyer, J. S.
, 
Lin, A.
, 
Tielking, A.
, 
Callen, S. T.
, 
Elizabeth Castillo, S.
, 
Miller, M.
, 
Tessman, M.
, 
Fahlgren, N.
, 
Carrington, J. C.
, 
Nusinow, D. A.
, &amp; 
Gehan, M. A.
 (2018). Raspberry Pi–powered imaging for plant phenotyping. Applications in Plant Sciences, 6(3), e1031. 10.1002/aps3.1031

</cite> [<a href="https://doi.org/10.1002/aps3.1031" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5895192/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29732261/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Applications%20in%20Plant%20Sciences&amp;title=Raspberry%20Pi%E2%80%93powered%20imaging%20for%20plant%20phenotyping&amp;author=J.%20C.%20Tovar&amp;author=J.%20S.%20Hoyer&amp;author=A.%20Lin&amp;author=A.%20Tielking&amp;author=S.%20T.%20Callen&amp;volume=6&amp;issue=3&amp;publication_year=2018&amp;pages=e1031&amp;pmid=29732261&amp;doi=10.1002/aps3.1031&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270100-bib-0084">
<cite>

Trachsel, S.
, 
Kaeppler, S. M.
, 
Brown, K. M.
, &amp; 
Lynch, J. P.
 (2011). Shovelomics: High throughput phenotyping of maize (<em>Zea mays</em> L.) root architecture in the field. Plant and Soil, 341(1), 75–87. 10.1007/s11104-010-0623-8
</cite> [<a href="https://doi.org/10.1007/s11104-010-0623-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20and%20Soil&amp;title=Shovelomics:%20High%20throughput%20phenotyping%20of%20maize%20(Zea%20mays%20L.)%20root%20architecture%20in%20the%20field&amp;author=S.%20Trachsel&amp;author=S.%20M.%20Kaeppler&amp;author=K.%20M.%20Brown&amp;author=J.%20P.%20Lynch&amp;volume=341&amp;issue=1&amp;publication_year=2011&amp;pages=75-87&amp;doi=10.1007/s11104-010-0623-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270100-bib-0085">
<cite>

Trachsel, S.
, 
Kaeppler, S. M.
, 
Brown, K. M.
, &amp; 
Lynch, J. P.
 (2013). Maize root growth angles become steeper under low N conditions. Field Crops Research, 140, 18–31. 10.1016/j.fcr.2012.09.010
</cite> [<a href="https://doi.org/10.1016/j.fcr.2012.09.010" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Field%20Crops%20Research&amp;title=Maize%20root%20growth%20angles%20become%20steeper%20under%20low%20N%20conditions&amp;author=S.%20Trachsel&amp;author=S.%20M.%20Kaeppler&amp;author=K.%20M.%20Brown&amp;author=J.%20P.%20Lynch&amp;volume=140&amp;publication_year=2013&amp;pages=18-31&amp;doi=10.1016/j.fcr.2012.09.010&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270100-bib-0086">
<cite>

Tuberosa, R.
 (2012). Phenotyping for drought tolerance of crops in the genomics era. Frontiers in Physiology, 3(9), Article 347. 10.3389/fphys.2012.00347

</cite> [<a href="https://doi.org/10.3389/fphys.2012.00347" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3446691/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23049510/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Physiology&amp;title=Phenotyping%20for%20drought%20tolerance%20of%20crops%20in%20the%20genomics%20era&amp;author=R.%20Tuberosa&amp;volume=3&amp;issue=9&amp;publication_year=2012&amp;pages=347&amp;pmid=23049510&amp;doi=10.3389/fphys.2012.00347&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270100-bib-0087">
<cite>

Tuberosa, R.
, 
Salvi, S.
, 
Sanguineti, M. C.
, 
Maccaferri, M.
, 
Giuliani, S.
, &amp; 
Landi, P.
 (2003). Searching for quantitative trait loci controlling root traits in maize: A critical appraisal. Plant and Soil, 255(1), 35–54. 10.1023/A:1026146615248
</cite> [<a href="https://doi.org/10.1023/A:1026146615248" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20and%20Soil&amp;title=Searching%20for%20quantitative%20trait%20loci%20controlling%20root%20traits%20in%20maize:%20A%20critical%20appraisal&amp;author=R.%20Tuberosa&amp;author=S.%20Salvi&amp;author=M.%20C.%20Sanguineti&amp;author=M.%20Maccaferri&amp;author=S.%20Giuliani&amp;volume=255&amp;issue=1&amp;publication_year=2003&amp;pages=35-54&amp;doi=10.1023/A:1026146615248&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270100-bib-0088">
<cite>

Ubbens, J.
, 
Cieslak, M.
, 
Prusinkiewicz, P.
, 
Parkin, I.
, 
Ebersbach, J.
, &amp; 
Stavness, I.
 (2020). Latent space phenotyping: Automatic image‐based phenotyping for treatment studies. Plant Phenomics, 2020, 5801869. 10.34133/2020/5801869

</cite> [<a href="https://doi.org/10.34133/2020/5801869" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7706325/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33313558/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Phenomics&amp;title=Latent%20space%20phenotyping:%20Automatic%20image%E2%80%90based%20phenotyping%20for%20treatment%20studies&amp;author=J.%20Ubbens&amp;author=M.%20Cieslak&amp;author=P.%20Prusinkiewicz&amp;author=I.%20Parkin&amp;author=J.%20Ebersbach&amp;volume=2020&amp;publication_year=2020&amp;pages=5801869&amp;pmid=33313558&amp;doi=10.34133/2020/5801869&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270100-bib-0089">
<cite>

Ueda, M.
, 
Koshino‐Kimura, Y.
, &amp; 
Okada, K.
 (2005). Stepwise understanding of root development. Current Opinion in Plant Biology, 8(1), 71–76. 10.1016/j.pbi.2004.11.014

</cite> [<a href="https://doi.org/10.1016/j.pbi.2004.11.014" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15653403/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Current%20Opinion%20in%20Plant%20Biology&amp;title=Stepwise%20understanding%20of%20root%20development&amp;author=M.%20Ueda&amp;author=Y.%20Koshino%E2%80%90Kimura&amp;author=K.%20Okada&amp;volume=8&amp;issue=1&amp;publication_year=2005&amp;pages=71-76&amp;pmid=15653403&amp;doi=10.1016/j.pbi.2004.11.014&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270100-bib-0090">
<cite>

Uga, Y.
, 
Yamamoto, E.
, 
Kanno, N.
, 
Kawai, S.
, 
Mizubayashi, T.
, &amp; 
Fukuoka, S.
 (2013). A major QTL controlling deep rooting on rice chromosome 4. Scientific Reports, 3, Article 3040. 10.1038/srep03040

</cite> [<a href="https://doi.org/10.1038/srep03040" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3807109/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24154623/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Scientific%20Reports&amp;title=A%20major%20QTL%20controlling%20deep%20rooting%20on%20rice%20chromosome%204&amp;author=Y.%20Uga&amp;author=E.%20Yamamoto&amp;author=N.%20Kanno&amp;author=S.%20Kawai&amp;author=T.%20Mizubayashi&amp;volume=3&amp;publication_year=2013&amp;pages=3040&amp;pmid=24154623&amp;doi=10.1038/srep03040&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270100-bib-0091">
<cite>

Visscher, P. M.
, 
Hill, W. G.
, &amp; 
Wray, N. R.
 (2008). Heritability in the genomics era—Concepts and misconceptions. Nature Reviews Genetics, 9(4), 255–266. 10.1038/nrg2322
</cite> [<a href="https://doi.org/10.1038/nrg2322" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18319743/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Reviews%20Genetics&amp;title=Heritability%20in%20the%20genomics%20era%E2%80%94Concepts%20and%20misconceptions&amp;author=P.%20M.%20Visscher&amp;author=W.%20G.%20Hill&amp;author=N.%20R.%20Wray&amp;volume=9&amp;issue=4&amp;publication_year=2008&amp;pages=255-266&amp;pmid=18319743&amp;doi=10.1038/nrg2322&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270100-bib-0094">
<cite>

Wickham, H.
 (2016). ggplot2: Elegant graphics for data analysis. Springer‐Verlag. <a href="https://ggplot2.tidyverse.org" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://ggplot2.tidyverse.org</a>
</cite> [<a href="https://scholar.google.com/scholar_lookup?title=ggplot2:%20Elegant%20graphics%20for%20data%20analysis&amp;author=H.%20Wickham&amp;publication_year=2016&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270100-bib-0095">
<cite>

Woods, P.
, 
Lehner, K. R.
, 
Hein, K.
, 
Mullen, J. L.
, &amp; 
McKay, J. K.
 (2022). Root pulling force across drought in maize reveals genotype by environment interactions and candidate genes. Frontiers in Plant Science, 13(4), Article 883209. 10.3389/fpls.2022.883209

</cite> [<a href="https://doi.org/10.3389/fpls.2022.883209" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9051544/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35498695/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Plant%20Science&amp;title=Root%20pulling%20force%20across%20drought%20in%20maize%20reveals%20genotype%20by%20environment%20interactions%20and%20candidate%20genes&amp;author=P.%20Woods&amp;author=K.%20R.%20Lehner&amp;author=K.%20Hein&amp;author=J.%20L.%20Mullen&amp;author=J.%20K.%20McKay&amp;volume=13&amp;issue=4&amp;publication_year=2022&amp;pages=883209&amp;pmid=35498695&amp;doi=10.3389/fpls.2022.883209&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270100-bib-0096">
<cite>

York, L. M.
, &amp; 
Lobet, G.
 (2017). Phenomics of root system architecture: Measuring and analyzing root phenes. The Plant Cell, 29(9), 1–9. 10.1105/tpc.117.tt0917

</cite> [<a href="https://doi.org/10.1105/tpc.117.tt0917" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5635994/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29018159/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=The%20Plant%20Cell&amp;title=Phenomics%20of%20root%20system%20architecture:%20Measuring%20and%20analyzing%20root%20phenes&amp;author=L.%20M.%20York&amp;author=G.%20Lobet&amp;volume=29&amp;issue=9&amp;publication_year=2017&amp;pages=1-9&amp;pmid=29018159&amp;doi=10.1105/tpc.117.tt0917&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270100-bib-0097">
<cite>

York, L. M.
, &amp; 
Lynch, J. P.
 (2015). Intensive field phenotyping of maize (<em>Zea mays</em> L.) root crowns identifies phenes and phene integration associated with plant growth and nitrogen acquisition. Journal of Experimental Botany, 66(18), 5493–5505. 10.1093/jxb/erv241

</cite> [<a href="https://doi.org/10.1093/jxb/erv241" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4585417/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26041317/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Experimental%20Botany&amp;title=Intensive%20field%20phenotyping%20of%20maize%20(Zea%20mays%20L.)%20root%20crowns%20identifies%20phenes%20and%20phene%20integration%20associated%20with%20plant%20growth%20and%20nitrogen%20acquisition&amp;author=L.%20M.%20York&amp;author=J.%20P.%20Lynch&amp;volume=66&amp;issue=18&amp;publication_year=2015&amp;pages=5493-5505&amp;pmid=26041317&amp;doi=10.1093/jxb/erv241&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270100-bib-0098">
<cite>

Zeng, D.
, 
Li, M.
, 
Jiang, N.
, 
Ju, Y.
, 
Schreiber, H.
, 
Chambers, E.
, 
Letscher, D.
, 
Ju, T.
, &amp; 
Topp, C. N.
 (2021). TopoRoot: A method for computing hierarchy and fine‐grained traits of maize roots from 3D imaging. Plant Methods, 17(1), Article 127. 10.1186/s13007-021-00829-z

</cite> [<a href="https://doi.org/10.1186/s13007-021-00829-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8667396/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34903248/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Methods&amp;title=TopoRoot:%20A%20method%20for%20computing%20hierarchy%20and%20fine%E2%80%90grained%20traits%20of%20maize%20roots%20from%203D%20imaging&amp;author=D.%20Zeng&amp;author=M.%20Li&amp;author=N.%20Jiang&amp;author=Y.%20Ju&amp;author=H.%20Schreiber&amp;volume=17&amp;issue=1&amp;publication_year=2021&amp;pages=127&amp;pmid=34903248&amp;doi=10.1186/s13007-021-00829-z&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270100-bib-0099">
<cite>

Zobel, R. W.
, &amp; 
Waisel, Y.
 (2010). A plant root system architectural taxonomy: A framework for root nomenclature. Plant Biosystems—An International Journal Dealing with All Aspects of Plant Biology, 144(2), 507–512. 10.1080/11263501003764483
</cite> [<a href="https://doi.org/10.1080/11263501003764483" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Biosystems%E2%80%94An%20International%20Journal%20Dealing%20with%20All%20Aspects%20of%20Plant%20Biology&amp;title=A%20plant%20root%20system%20architectural%20taxonomy:%20A%20framework%20for%20root%20nomenclature&amp;author=R.%20W.%20Zobel&amp;author=Y.%20Waisel&amp;volume=144&amp;issue=2&amp;publication_year=2010&amp;pages=507-512&amp;doi=10.1080/11263501003764483&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ol></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="caption p">
<p>
<strong>Supplemental Figure S1</strong> Heatmap of pairwise genotypic correlations among 155 RSA traits grouped by collection method (2D single‐view imaging, 3D imaging, and manual measurements) in the 2019 SAM maize panel.</p>
<p>
<strong>Supplemental Figure S2</strong> Percentage of explained variances by individual principal components associated with the PCA of 115 RSA traits measured in the 2019 SAM maize panel.</p>
<p>
<strong>Supplemental Figure S3</strong> PCA of RSA traits partitioned by irrigation treatment measured in the 2019 SAM maize panel.</p>
<p>
<strong>Supplemental Figure S4</strong> Distribution of broad‐sense heritability estimates of the 2019 SAM maize panel RSA traits among phenotyping methods and irrigation treatments.</p>
<p>
<strong>Supplemental Figure S5</strong> Broad‐sense heritability estimates for root mass (g) and RPF (kg) across irrigation treatments in the 2019 SAM maize panel.</p>
<p>
<strong>Supplemental Figure S6</strong> Heatmap of the percentage of phenotypic variance explained (PVE) by 1‐Mb genomic regions associated with traits colocalized across imaging, RPF, and multivariate approaches.</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12375851/bin/TPG2-18-e70100-s002.pdf" data-ga-action="click_feat_suppl" class="usa-link">TPG2-18-e70100-s002.pdf</a><sup> (467.9KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="caption p">
<p>
<strong>Supplemental Table S1</strong> Genotypic values (BLUPs) used in the 2019 FarmCPU GWAS on the SAM maize diversity panel, with RSA univariate and multivariate traits partitioned by phenotyping method.</p>
<p>
<strong>Supplemental Table S2</strong> Comparison of broad‐sense heritability for all RSA traits between 2D single‐view and multi‐view root crown imaging in the 2022 NAM RIL population.</p>
<p>
<strong>Supplemental Table S3</strong> Comparison of 18 RSA traits that showed significant broad‐sense heritability (H2) in at least one 2D imaging method in the 2022 NAM RIL population.</p>
<p>
<strong>Supplemental Table S4</strong> 2019 PCA trait loadings associated with the axes represented in Figure 3.</p>
<p>
<strong>Supplemental Table S5</strong> Broad‐sense heritability and variance estimates for RSA traits in the 2019 SAM maize panel.</p>
<p>
<strong>Supplemental Table S6</strong> List of significant SNPs from the 2019 FarmCPU GWAS.</p>
<p>
<strong>Supplemental Table S7</strong> Multivariate GWAS trait loadings by phenotyping method and irrigation treatment in the 2019 SAM maize diversity panel.</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12375851/bin/TPG2-18-e70100-s001.xlsx" data-ga-action="click_feat_suppl" class="usa-link">TPG2-18-e70100-s001.xlsx</a><sup> (967.4KB, xlsx) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>Raw phenotypic metadata are available on the Dryad Digital Repository (<a href="https://doi.org/10.5061/dryad.z34tmpgq4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://doi.org/10.5061/dryad.z34tmpgq4</a>, <a href="http://datadryad.org/share/HeNYoxNMdN_GrHMyZHFN3rUTN1UiG8OFhU-B107E7mM" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://datadryad.org/share/HeNYoxNMdN_GrHMyZHFN3rUTN1UiG8OFhU-B107E7mM</a>). Phenotypic values used in the 2019 genome‐wide association analysis, along with the identified significant RSA‐associated SNPs, are provided in Tables <a href="#tpg270100-supinfo-0001" class="usa-link">S1</a>, <a href="#tpg270100-supinfo-0001" class="usa-link">S6</a>, and <a href="#tpg270100-supinfo-0001" class="usa-link">S7</a>.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from The Plant Genome are provided here courtesy of <strong>Wiley</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1002/tpg2.70100"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/TPG2-18-e70100.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (1.6 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12375851/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12375851/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12375851%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12375851/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12375851/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12375851/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12375851/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12375851/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12375851/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="E6eJrcaGK1ul2hEJDRfo1jtoQggdMkq4Zy8eUrvwzFArplydbLxsJBzfcbcMPTNJ">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
        <script type="module" crossorigin="" src="/static/assets/math-574fdcc6.js"></script>
    
    

    </body>
</html>
