
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Genetic mapping and haplotype analysis identify novel candidate genes for high night temperature tolerance in winter wheat - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="D2E484428AEF08F3058442002693E5F1.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="blackwellopen">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12256648/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="The Plant Genome">
<meta name="citation_title" content="Genetic mapping and haplotype analysis identify novel candidate genes for high night temperature tolerance in winter wheat">
<meta name="citation_author" content="Dinesh Kumar Saini">
<meta name="citation_author_institution" content="Department of Plant and Soil Science, Texas Tech University, Lubbock, Texas, USA">
<meta name="citation_author" content="Nathan T Hein">
<meta name="citation_author_institution" content="Department of Agronomy, Kansas State University, Manhattan, Kansas, USA">
<meta name="citation_author" content="Impa Somayanda">
<meta name="citation_author_institution" content="Department of Plant and Soil Science, Texas Tech University, Lubbock, Texas, USA">
<meta name="citation_author_institution" content="Department of Agronomy, Kansas State University, Manhattan, Kansas, USA">
<meta name="citation_author" content="Raju Bheemanahalli">
<meta name="citation_author_institution" content="Department of Plant and Soil Sciences, Mississippi State University, Mississippi State, Mississippi, USA">
<meta name="citation_author" content="Troy Ostmeyer">
<meta name="citation_author_institution" content="Department of Agronomy, Kansas State University, Manhattan, Kansas, USA">
<meta name="citation_author" content="S V Krishna Jagadish">
<meta name="citation_author_institution" content="Department of Plant and Soil Science, Texas Tech University, Lubbock, Texas, USA">
<meta name="citation_author_institution" content="Department of Agronomy, Kansas State University, Manhattan, Kansas, USA">
<meta name="citation_publication_date" content="2025 Jul 13">
<meta name="citation_volume" content="18">
<meta name="citation_issue" content="3">
<meta name="citation_firstpage" content="e70075">
<meta name="citation_doi" content="10.1002/tpg2.70075">
<meta name="citation_pmid" content="40653599">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12256648/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12256648/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12256648/pdf/TPG2-18-e70075.pdf">
<meta name="description" content="A diverse panel of 320 winter wheat (Triticum aestivum L.) genotypes was evaluated for yield‐related parameters under control and high night‐time temperature (HNT) stress using custom‐designed, movable field‐based heat tents over 2 years. ...">
<meta name="og:title" content="Genetic mapping and haplotype analysis identify novel candidate genes for high night temperature tolerance in winter wheat">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="A diverse panel of 320 winter wheat (Triticum aestivum L.) genotypes was evaluated for yield‐related parameters under control and high night‐time temperature (HNT) stress using custom‐designed, movable field‐based heat tents over 2 years. ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12256648/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12256648">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1002/tpg2.70075"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/TPG2-18-e70075.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12256648%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12256648/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12256648/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12256648/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-blackwellopen.png" alt="Wiley Open Access Collection logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Wiley Open Access Collection" title="Link to Wiley Open Access Collection" shape="default" href="https://doi.org/10.1002/tpg2.70075" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Plant Genome</button></div>. 2025 Jul 13;18(3):e70075. doi: <a href="https://doi.org/10.1002/tpg2.70075" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1002/tpg2.70075</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Plant%20Genome%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Plant%20Genome%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Plant%20Genome%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Plant%20Genome%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Genetic mapping and haplotype analysis identify novel candidate genes for high night temperature tolerance in winter wheat</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kumar%20Saini%20D%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Dinesh Kumar Saini</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Dinesh Kumar Saini</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Department of Plant and Soil Science, Texas Tech University, Lubbock, Texas, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kumar%20Saini%20D%22%5BAuthor%5D" class="usa-link"><span class="name western">Dinesh Kumar Saini</span></a>
</div>
</div>
<sup>1,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hein%20NT%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Nathan T Hein</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Nathan T Hein</span></h3>
<div class="p">
<sup>
<sup>2</sup>
</sup>Department of Agronomy, Kansas State University, Manhattan, Kansas, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hein%20NT%22%5BAuthor%5D" class="usa-link"><span class="name western">Nathan T Hein</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Somayanda%20I%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Impa Somayanda</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Impa Somayanda</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Department of Plant and Soil Science, Texas Tech University, Lubbock, Texas, USA</div>
<div class="p">
<sup>
<sup>2</sup>
</sup>Department of Agronomy, Kansas State University, Manhattan, Kansas, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Somayanda%20I%22%5BAuthor%5D" class="usa-link"><span class="name western">Impa Somayanda</span></a>
</div>
</div>
<sup>1,</sup><sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bheemanahalli%20R%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Raju Bheemanahalli</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Raju Bheemanahalli</span></h3>
<div class="p">
<sup>
<sup>3</sup>
</sup>Department of Plant and Soil Sciences, Mississippi State University, Mississippi State, Mississippi, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bheemanahalli%20R%22%5BAuthor%5D" class="usa-link"><span class="name western">Raju Bheemanahalli</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ostmeyer%20T%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Troy Ostmeyer</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Troy Ostmeyer</span></h3>
<div class="p">
<sup>
<sup>2</sup>
</sup>Department of Agronomy, Kansas State University, Manhattan, Kansas, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ostmeyer%20T%22%5BAuthor%5D" class="usa-link"><span class="name western">Troy Ostmeyer</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Jagadish%20SVK%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">S V Krishna Jagadish</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">S V Krishna Jagadish</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Department of Plant and Soil Science, Texas Tech University, Lubbock, Texas, USA</div>
<div class="p">
<sup>
<sup>2</sup>
</sup>Department of Agronomy, Kansas State University, Manhattan, Kansas, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Jagadish%20SVK%22%5BAuthor%5D" class="usa-link"><span class="name western">S V Krishna Jagadish</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="tpg270075-aff-0001">
<sup>
<sup>1</sup>
</sup>Department of Plant and Soil Science, Texas Tech University, Lubbock, Texas, USA</div>
<div id="tpg270075-aff-0002">
<sup>
<sup>2</sup>
</sup>Department of Agronomy, Kansas State University, Manhattan, Kansas, USA</div>
<div id="tpg270075-aff-0003">
<sup>
<sup>3</sup>
</sup>Department of Plant and Soil Sciences, Mississippi State University, Mississippi State, Mississippi, USA</div>
<div class="author-notes p">
<div class="fn" id="correspondenceTo">
<sup>*</sup><p class="display-inline">

<strong>Correspondence</strong>
, Dinesh Kumar Saini and S. V. Krishna Jagadish, Department of Plant and Soil Science, Texas Tech University, Lubbock, TX, USA. Email: <span>disaini@ttu.edu</span> and <span>kjagadish.sv@ttu.edu</span>

</p>
</div>
<div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Revised 2025 May 13; Received 2025 Feb 8; Accepted 2025 May 30; Issue date 2025 Sep.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© 2025 The Author(s). <em>The Plant Genome</em> published by Wiley Periodicals LLC on behalf of Crop Science Society of America.</div>
<p>This is an open access article under the terms of the <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by-nc-nd/4.0/</a> License, which permits use and distribution in any medium, provided the original work is properly cited, the use is non‐commercial and no modifications or adaptations are made.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12256648  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40653599/" class="usa-link">40653599</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>Abstract</h2>
<p>A diverse panel of 320 winter wheat (<em>Triticum aestivum</em> L.) genotypes was evaluated for yield‐related parameters under control and high night‐time temperature (HNT) stress using custom‐designed, movable field‐based heat tents over 2 years. Genome‐wide association mapping identified several significant single nucleotide polymorphisms (SNPs) across environments, with SNPs under HNT conditions, based on absolute trait values and stress tolerance indices, grouped into 19 HNT quantitative trait loci (QTLs). Additionally, significant SNPs identified under both (control and HNT) conditions contributed to seven condition‐independent QTLs. Thirteen haplotype blocks, derived from some of these QTLs, significantly influenced traits, with nine specific to HNT stress and four shared across conditions. Candidate gene mining identified 17 high‐confidence genes involved in the HNT response, including those encoding ATP‐dependent RNA helicase, GCN5‐related N‐acetyltransferase, oxidoreductase FAD/NAD(P)‐binding, RNA recognition motif, and sugar/inositol transporter. Gene‐based haplotype analysis identified four haplotypes with significant differences under HNT conditions and one block with differences under both conditions. For instance, Hap1<em>
<sup>TraesCS1A02G305700</sup>
</em> exhibited higher biomass and spike number under HNT, Hap1<em>
<sup>TraesCS2B02G599800</sup>
</em> showed higher biomass under HNT, and Hap2<em>
<sup>TraesCS4B02G264300</sup>
</em> demonstrated higher biomass under both conditions. This study elucidates the genetic control of traits contributing to HNT stress responses in wheat, supporting the selection of lines with favorable alleles for crosses and enabling marker‐assisted and genomic selections for HNT stress tolerance. The identified candidate genes and haplotypes provide valuable targets for functional validation, breeding strategies, and gene‐editing approaches to enhance HNT resilience in wheat.</p></section><section class="abstract" id="abstract2"><h2>Core Ideas</h2>
<ul id="tpg270075-list-10001" class="list" style="list-style-type:disc">
<li><p>A panel of 320 winter wheat genotypes was assessed under control and high night‐time temperature (HNT) stress.</p></li>
<li><p>Genome‐wide association mapping revealed significant single nucleotide polymorphisms (SNPs) linked to HNT response, grouped into 19 HNT‐specific quantitative trait loci (QTLs).</p></li>
<li><p>Thirteen haplotype blocks significantly influenced traits, with nine specific to HNT and four shared across conditions.</p></li>
<li><p>Candidate gene mining identified 17 high‐confidence genes related to HNT stress responses in wheat.</p></li>
<li><p>These findings can support breeding programs in developing wheat varieties that sustain high yields despite warm nights.</p></li>
</ul></section><hr class="headless">
<h3>Abbreviations</h3>
<dl class="def-list" id="tpg270075-dl-0001">
<dt>GWAS</dt>
<dd><p>genome‐wide association study</p></dd>
<dt>HDT</dt>
<dd><p>high day‐time temperature</p></dd>
<dt>HNT</dt>
<dd><p>high night‐time temperature</p></dd>
<dt>HWWAMP</dt>
<dd><p>Hard Winter Wheat Association Mapping Panel</p></dd>
<dt>PCA</dt>
<dd><p>principal component analysis</p></dd>
<dt>QTL</dt>
<dd><p>quantitative trait loci</p></dd>
<dt>SNP</dt>
<dd><p>single nucleotide polymorphism</p></dd>
</dl>
<section id="tpg270075-sec-0010"><h2 class="pmc_sec_title">1. INTRODUCTION</h2>
<p>Bread wheat (<em>Triticum aestivum</em> L.) serves as a crucial staple grain, providing a significant portion of the world's population with essential calories. Global wheat yield is increasing at just 0.9% annually, significantly below the 2.4% required to double production by 2050, which may result in only a 38% increase, which is insufficient to meet projected demand (Ray et al., <a href="#tpg270075-bib-0103" class="usa-link" aria-describedby="tpg270075-bib-0103">2013</a>). As we strive to meet the demands of a growing population, the changing climate poses a significant risk to the future productivity of important staple crops like wheat. Since the pre‐industrial era, the global average surface temperature has increased by 1°C, and projections suggest it will rise by an additional 0.2°C per decade, with average warming anticipated to reach 1.5°C between 2030 and 2052 (Ogunbode et al., <a href="#tpg270075-bib-0088" class="usa-link" aria-describedby="tpg270075-bib-0088">2020</a>). Studies indicate that with every degree rise in global mean temperature, there will be corresponding decreases in yields of wheat, rice (<em>Oryza sativa</em> L.), maize (<em>Zea mays</em> L.), and soybean (<em>Glycine max</em> (L.) Merr.) by 6%, 3.2%, 7.4%, and 3.1%, respectively, posing a significant threat to food security (Zhao et al., <a href="#tpg270075-bib-0137" class="usa-link" aria-describedby="tpg270075-bib-0137">2017</a>). Retrospective analyses have indicated that night temperature has risen at a faster pace than day temperature in recent decades (Alexander et al., <a href="#tpg270075-bib-0004" class="usa-link" aria-describedby="tpg270075-bib-0004">2006</a>; Sillmann, Kharin, Zhang, et al., <a href="#tpg270075-bib-0115" class="usa-link" aria-describedby="tpg270075-bib-0115">2013</a>). Multi‐model ensembles project a higher increase in the frequency of warm nights compared to hot days, suggesting ongoing asymmetric warming (Sillmann, Kharin, Zwiers, et al., <a href="#tpg270075-bib-0116" class="usa-link" aria-describedby="tpg270075-bib-0116">2013</a>).</p>
<p>Warm nights have been observed in different cropping regions, predominantly in temperate regions, including the Argentinean Pampas and the Australian wheat belt in the Southern Hemisphere (Alexander et al., <a href="#tpg270075-bib-0003" class="usa-link" aria-describedby="tpg270075-bib-0003">2007</a>; Fernández‐Long et al., <a href="#tpg270075-bib-0029" class="usa-link" aria-describedby="tpg270075-bib-0029">2013</a>), as well as in the North American wheat belt and Central and South Asia in the Northern Hemisphere (Klein Tank et al., <a href="#tpg270075-bib-0056" class="usa-link" aria-describedby="tpg270075-bib-0056">2006</a>; Vincent &amp; Mekis, <a href="#tpg270075-bib-0122" class="usa-link" aria-describedby="tpg270075-bib-0122">2006</a>). Consequently, most of the crops, including wheat, are presently exposed to, and are likely to continue to face, warm nights during their growing seasons. This rapid nocturnal temperature increase poses a considerable challenge for maintaining crop yields, impacting wheat (Hein et al., <a href="#tpg270075-bib-0043" class="usa-link" aria-describedby="tpg270075-bib-0043">2019</a>; Impa et al., <a href="#tpg270075-bib-0048" class="usa-link" aria-describedby="tpg270075-bib-0048">2020</a>; Kundu et al., <a href="#tpg270075-bib-0062" class="usa-link" aria-describedby="tpg270075-bib-0062">2024</a>), rice (Bahuguna et al., <a href="#tpg270075-bib-0009" class="usa-link" aria-describedby="tpg270075-bib-0009">2017</a>; S. Peng et al., <a href="#tpg270075-bib-0094" class="usa-link" aria-describedby="tpg270075-bib-0094">2004</a>; Sharma et al., <a href="#tpg270075-bib-0112" class="usa-link" aria-describedby="tpg270075-bib-0112">2025</a>), and barley (<em>Hordeum vulgare</em> L.) (García et al., <a href="#tpg270075-bib-0033" class="usa-link" aria-describedby="tpg270075-bib-0033">2016</a>; <a href="#tpg270075-bib-0032" class="usa-link" aria-describedby="tpg270075-bib-0032">2018</a>), among others. Selecting and/or breeding for resilience to high day‐time temperature (HDT) stress will not compensate for losses induced by high night‐time temperature (HNT), given the distinctiveness in adaptive mechanisms for these two conditions (Impa et al., <a href="#tpg270075-bib-0046" class="usa-link" aria-describedby="tpg270075-bib-0046">2021</a>; Prasad et al., <a href="#tpg270075-bib-0099" class="usa-link" aria-describedby="tpg270075-bib-0099">2008</a>; Shi et al., <a href="#tpg270075-bib-0114" class="usa-link" aria-describedby="tpg270075-bib-0114">2017</a>).</p>
<p>In wheat, the post‐flowering period is particularly vulnerable to HNT, which accelerates senescence, shortens grain‐filling duration, and reduces yield and quality by up to 4% per degree Celsius increase (García et al., <a href="#tpg270075-bib-0033" class="usa-link" aria-describedby="tpg270075-bib-0033">2016</a>; Hein et al., <a href="#tpg270075-bib-0042" class="usa-link" aria-describedby="tpg270075-bib-0042">2020</a>). Contrastingly, some studies suggest that HNT during early spring delays senescence, enhancing grain‐filling rate and duration (Djanaguiraman et al., <a href="#tpg270075-bib-0023" class="usa-link" aria-describedby="tpg270075-bib-0023">2020</a>). While wheat can adapt to HDT by initiating flowering early in the morning to escape or employ transpiration cooling (Aiqing et al., <a href="#tpg270075-bib-0002" class="usa-link" aria-describedby="tpg270075-bib-0002">2018</a>; Jagadish, <a href="#tpg270075-bib-0049" class="usa-link" aria-describedby="tpg270075-bib-0049">2020</a>), it lacks similar mechanisms to counteract the adverse effects of HNT. Studies conducted both in controlled environments and field settings have shown that HDT stress adversely affects reproductive processes in wheat, leading to spikelet sterility (Bergkamp et al., <a href="#tpg270075-bib-0011" class="usa-link" aria-describedby="tpg270075-bib-0011">2018</a>; Bheemanahalli et al., <a href="#tpg270075-bib-0013" class="usa-link" aria-describedby="tpg270075-bib-0013">2019</a>; Mirosavljević et al., <a href="#tpg270075-bib-0084" class="usa-link" aria-describedby="tpg270075-bib-0084">2021</a>). Surprisingly, subjecting wheat to HNT from panicle initiation to physiological maturity in field conditions did not significantly increase spikelet sterility or reduce grain number compared to HDT (Narayanan et al., <a href="#tpg270075-bib-0087" class="usa-link" aria-describedby="tpg270075-bib-0087">2015</a>). However, any decrease in grain number could result from fewer spikes per square meter, potentially leading to a 7% reduction in grain yield per degree Celsius increase in night temperature (García et al., <a href="#tpg270075-bib-0031" class="usa-link" aria-describedby="tpg270075-bib-0031">2015</a>).</p>
<p>Based on extensive literature, it is concluded that the accelerated growth rate induced by post‐flowering HDT stress fails to compensate for the reduction in growth duration, ultimately leading to decreased dry‐matter accumulation (B. Liu et al., <a href="#tpg270075-bib-0075" class="usa-link" aria-describedby="tpg270075-bib-0075">2020</a>; Ullah et al., <a href="#tpg270075-bib-0120" class="usa-link" aria-describedby="tpg270075-bib-0120">2022</a>). Reductions in biomass accumulation due to HNT have also been observed in wheat experiments conducted across controlled environment chambers, with reductions ranging from 2% to 46% (Impa et al., <a href="#tpg270075-bib-0047" class="usa-link" aria-describedby="tpg270075-bib-0047">2019</a>; Narayanan et al., <a href="#tpg270075-bib-0087" class="usa-link" aria-describedby="tpg270075-bib-0087">2015</a>; Posch et al., <a href="#tpg270075-bib-0097" class="usa-link" aria-describedby="tpg270075-bib-0097">2022</a>; Prasad et al., <a href="#tpg270075-bib-0099" class="usa-link" aria-describedby="tpg270075-bib-0099">2008</a>) and field‐based studies (with reductions ranging from 8.5% to 23.4%; García et al., <a href="#tpg270075-bib-0031" class="usa-link" aria-describedby="tpg270075-bib-0031">2015</a>; Parveen et al., <a href="#tpg270075-bib-0091" class="usa-link" aria-describedby="tpg270075-bib-0091">2022</a>). However, some studies suggest that HNT promotes wheat biomass accumulation by increasing tillering, grains per spike, photosynthetic activity, and/or pre‐anthesis respiration (Y. Fan, Qin, et al., <a href="#tpg270075-bib-0027" class="usa-link" aria-describedby="tpg270075-bib-0027">2024</a>; Lazăr &amp; Lazăr, <a href="#tpg270075-bib-0065" class="usa-link" aria-describedby="tpg270075-bib-0065">2010</a>; G. Xiao et al., <a href="#tpg270075-bib-0127" class="usa-link" aria-describedby="tpg270075-bib-0127">2010</a>). Similarly, a meta‐analysis by Jing et al. (<a href="#tpg270075-bib-0050" class="usa-link" aria-describedby="tpg270075-bib-0050">2016</a>) found that HNT exposure during vegetative stages in rice accelerated development without adversely affecting total biomass. Also, rice exposed to HNT during vegetative stage exhibits significant leaf blade growth stimulation compared to lower temperature, leading to higher biomass accumulation (Kanno et al., <a href="#tpg270075-bib-0052" class="usa-link" aria-describedby="tpg270075-bib-0052">2009</a>). These findings highlight a lack of consensus regarding the effects of HNT on biomass accumulation and effective tillers/spikes per unit area. Overall, these findings suggest that while trait mapping has made strides in addressing HDT stress in wheat, the same strategies may not directly translate to mitigating HNT stress.</p>
<p>Furthermore, genetic mapping for traits contributing to HNT tolerance will require the evaluation of a large number of genotypes under HNT stress conditions. Previous studies on HNT stress in wheat mainly utilized controlled environment growth chambers (Impa et al., <a href="#tpg270075-bib-0047" class="usa-link" aria-describedby="tpg270075-bib-0047">2019</a>; <a href="#tpg270075-bib-0048" class="usa-link" aria-describedby="tpg270075-bib-0048">2020</a>; Narayanan et al., <a href="#tpg270075-bib-0087" class="usa-link" aria-describedby="tpg270075-bib-0087">2015</a>) or small chambers under field conditions (García et al., <a href="#tpg270075-bib-0031" class="usa-link" aria-describedby="tpg270075-bib-0031">2015</a>, <a href="#tpg270075-bib-0033" class="usa-link" aria-describedby="tpg270075-bib-0033">2016</a>). However, these facilities struggled to capture large genetic diversity and faced challenges such as variations in light, wind speed, and humidity compared to field conditions. Until 2019, there was no large mobile infrastructure for imposing controlled HNT stress on crops. We addressed this by developing a field‐based prototype for winter wheat (Hein et al., <a href="#tpg270075-bib-0043" class="usa-link" aria-describedby="tpg270075-bib-0043">2019</a>) and subsequently a scaled‐up version incorporating an advanced cyber‐physical system to apply post‐anthesis HNT stress until physiological maturity (Hein et al., <a href="#tpg270075-bib-0042" class="usa-link" aria-describedby="tpg270075-bib-0042">2020</a>), enabling phenotyping under real‐world field conditions. On the other hand, genome‐wide association study (GWAS) has emerged as a powerful tool for unraveling the genetic architecture of complex traits, especially with the advent of high‐density single nucleotide polymorphism (SNP) arrays and next‐generation sequencing technologies (Saini et al., <a href="#tpg270075-bib-0108" class="usa-link" aria-describedby="tpg270075-bib-0108">2022</a>). Moreover, GWAS is highly efficient at identifying numerous natural allelic variations across diverse individuals, surpassing traditional quantitative trait locus (QTL) mapping methods (X. Huang &amp; Han, <a href="#tpg270075-bib-0045" class="usa-link" aria-describedby="tpg270075-bib-0045">2014</a>; Saini et al., <a href="#tpg270075-bib-0107" class="usa-link" aria-describedby="tpg270075-bib-0107">2025</a>).</p>
<section class="bt xbox font-sm" id="boxed-text1"><section id="tpg270075-sec-0020"><h3 class="pmc_sec_title">Core Ideas</h3>
<ul id="tpg270075-list-0001" class="list" style="list-style-type:disc">
<li><p>A panel of 320 winter wheat genotypes was assessed under control and high night‐time temperature (HNT) stress.</p></li>
<li><p>Genome‐wide association mapping revealed significant single nucleotide polymorphisms (SNPs) linked to HNT response, grouped into 19 HNT‐specific quantitative trait loci (QTLs).</p></li>
<li><p>Thirteen haplotype blocks significantly influenced traits, with nine specific to HNT and four shared across conditions.</p></li>
<li><p>Candidate gene mining identified 17 high‐confidence genes related to HNT stress responses in wheat.</p></li>
<li><p>These findings can support breeding programs in developing wheat varieties that sustain high yields despite warm nights.</p></li>
</ul></section></section><p>Over 2500 genes/QTLs/meta‐QTLs/marker‐trait associations have been identified in tetraploid and hexaploid wheat through various approaches, including GWAS and interval mapping, covering traits related to HDT stress (WheatQTLdb, <a href="http://www.wheatqtldb.net/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://www.wheatqtldb.net/</a>; S. Kumar et al., <a href="#tpg270075-bib-0060" class="usa-link" aria-describedby="tpg270075-bib-0060">2024</a>, <a href="#tpg270075-bib-0061" class="usa-link" aria-describedby="tpg270075-bib-0061">2021</a>; Singh et al., <a href="#tpg270075-bib-0118" class="usa-link" aria-describedby="tpg270075-bib-0118">2022</a>). However, HNT stress in wheat has received limited attention. In a previous study, we conducted GWAS on a diverse indica rice panel, identifying 20 significant SNPs associated with HNT tolerance traits, and prioritized 20 causal genes through network‐based models based on their expression patterns under HNT stress (Bheemanahalli, Knight, et al., <a href="#tpg270075-bib-0012" class="usa-link" aria-describedby="tpg270075-bib-0012">2021</a>). By contrast, the genetic basis of HNT tolerance in wheat remains largely unexplored, with only one study on spring wheat (Kahlon et al., <a href="#tpg270075-bib-0051" class="usa-link" aria-describedby="tpg270075-bib-0051">2024</a>) identifying 19 QTLs for yield‐related traits under greenhouse conditions. However, controlled environments like greenhouses create microclimates distinct from field conditions, limiting the applicability of these findings (Langstroff et al., <a href="#tpg270075-bib-0064" class="usa-link" aria-describedby="tpg270075-bib-0064">2022</a>). Further, this study prioritized candidate genes (CGs) based on functional domains associated with HDT stress and tissue‐specific expression under HDT conditions. However, since the tolerance mechanisms for HNT and HDT are shown to differ (Impa et al., <a href="#tpg270075-bib-0046" class="usa-link" aria-describedby="tpg270075-bib-0046">2021</a>; Prasad et al., <a href="#tpg270075-bib-0099" class="usa-link" aria-describedby="tpg270075-bib-0099">2008</a>; Shi et al., <a href="#tpg270075-bib-0114" class="usa-link" aria-describedby="tpg270075-bib-0114">2017</a>), the selected genes may not be suitable candidates for HNT tolerance. Furthermore, no superior haplotypes were identified that could be utilized for haplotype‐assisted forward breeding.</p>
<p>This study seeks to address these gaps by identifying key QTLs, CGs, and haplotypes associated with HNT tolerance under field conditions, facilitating the development of resilient wheat varieties suited to current and future climates. Specifically, a Hard Winter Wheat Association Mapping Panel (HWWAMP) was phenotyped for yield‐related traits such as spike number, spike weight, and plant biomass per unit area using custom‐designed, movable, field‐based heat tents. The specific objectives of this study were to (i) measure the phenotypic responses of wheat genotypes for various yield traits under both control and HNT conditions; (ii) identify QTLs and superior haplotypes linked to yield traits under both conditions, and assess the overlap of genetic loci across environments; (iii) evaluate the effects of individual and combined alleles on traits; and (iv) identify and prioritize CGs and CG‐based haplotypes contributing to natural variation for HNT tolerance in wheat.</p></section><section id="tpg270075-sec-0030"><h2 class="pmc_sec_title">2. MATERIALS AND METHODS</h2>
<section id="tpg270075-sec-0040"><h3 class="pmc_sec_title">2.1. Plant materials</h3>
<p>A total of 320 genotypes were considered for this study, comprising winter wheat genotypes that are part of the HWWAMP (consisting of 299 winter wheat genotypes developed under the USDA Triticeae Coordinated Agricultural Project [TCAP]), as well as wheat varieties extensively grown in Kansas, breeding lines from the Kansas Wheat Breeding Program, and exotic accessions previously demonstrated to exhibit differential responses to heat stress (Guttieri et al., <a href="#tpg270075-bib-0038" class="usa-link" aria-describedby="tpg270075-bib-0038">2015</a>; Hein et al., <a href="#tpg270075-bib-0043" class="usa-link" aria-describedby="tpg270075-bib-0043">2019</a>, <a href="#tpg270075-bib-0042" class="usa-link" aria-describedby="tpg270075-bib-0042">2020</a>). HWWAMP includes 258 hard red and 41 hard white winter wheat types, comprising landraces, advanced breeding lines, and varieties developed between 1874 and 2014 through public and private breeding programs across Colorado, Kansas, Montana, Nebraska, North Dakota, Oklahoma, South Dakota, and Texas (Grogan et al., <a href="#tpg270075-bib-0035" class="usa-link" aria-describedby="tpg270075-bib-0035">2016</a>). Additional information regarding the HWWAMP genotypes can be found in the T3/Wheat database (<a href="https://triticeaetoolbox.org/wheat/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://triticeaetoolbox.org/wheat/</a>).</p></section><section id="tpg270075-sec-0050"><h3 class="pmc_sec_title">2.2. Field infrastructure</h3>
<p>Custom‐designed and movable tents were utilized to evaluate the winter wheat genotypes. Details regarding field infrastructure, temperature control system, and operation of the tents, along with stress impositions, have already been provided (Hein et al., <a href="#tpg270075-bib-0042" class="usa-link" aria-describedby="tpg270075-bib-0042">2020</a>). In brief, these tents, measuring 9.1 m wide, 14.6 m long, and 4.4 m tall, feature 1.9‐m‐tall sidewalls joined by vertical trusses and are mounted on 15.2‐m‐long skids for effortless relocation via tractor. To regulate temperature, motorized roll‐up systems (Advancing Alternatives) are installed on sidewalls, end walls, and roofs, enabling ventilation control to mitigate HDT. Additionally, heating systems equipped with propane heaters (Modine HDB100 Hot Dawg propane heaters; Ferguson Plumbing and Heating), convection tubing, and circulation fans facilitate temperature adjustment within the tents. Control tents with similar setups but without heat imposition serve as comparison environments, utilizing power tube fans and convection tubing to replicate air movement observed in stress tents. Electricity is supplied by a centrally located electric generator (Caterpillar XQ30 electric generator; Foley Power Solutions), with diesel tanks and distribution systems ensuring power availability to each tent. The thermostat controller system, comprising a Raspberry Pi (Raspberry Pi Foundation), six MCP9808 temperature sensors (Adafruit), and one DS3231 Real‐Time Clock module (Adafruit), monitors the temperature in real‐time within each tent. It employs wireless transmission to ensure consistent conditions are maintained across both control and stress environments.</p>
<p>HNT stress was applied at night (from 8:00 p.m. until 6:00 a.m.), after 50% anthesis, and continued until physiological maturity. During this period, the tents were sealed, heaters activated, and stress levels monitored wirelessly, and the system was shut down in the morning. Multiple sensor sets tracked temperature, relative humidity, and CO<sub>2</sub> levels inside the tents, providing comprehensive data for detailed analysis. The system recorded a 0.6°C differential between six temperature sensors within the stress tents. HOBO UA 002–64 Pendent data loggers confirmed uniform heat distribution, measuring a 0.2°C temperature difference at any given time within both control and stress tents. Genotypes in stress tents were exposed to HNT conditions, averaging +3.8°C compared to the average night‐time temperature of the control tents (Hein, <a href="#tpg270075-bib-0041" class="usa-link" aria-describedby="tpg270075-bib-0041">2022</a>; Hein et al., <a href="#tpg270075-bib-0042" class="usa-link" aria-describedby="tpg270075-bib-0042">2020</a>). The average temperature during the stress period was 22.1°C in the stress tents and 18.4°C in the control tents. Temperature divergence started at 6:00 p.m., reaching +2.8°C and +3.5°C by 7 and 8:00 p.m., respectively, and maintained at ∼3.8°C from 10:00 p.m. until the stress imposition ended the next morning and tents were opened at 6:00 a.m. (Hein, <a href="#tpg270075-bib-0041" class="usa-link" aria-describedby="tpg270075-bib-0041">2022</a>; Hein et al., <a href="#tpg270075-bib-0042" class="usa-link" aria-describedby="tpg270075-bib-0042">2020</a>).</p></section><section id="tpg270075-sec-0060"><h3 class="pmc_sec_title">2.3. Experimental trials and crop management</h3>
<p>A total of six custom‐designed movable tents (three control tents and three heat tents), located at the Kansas State University Agronomy North Farm in Manhattan, KS (39° 12′ 47.3″ N, 96° 35′ 35.0″ W), were utilized. Each tent consisted of eight blocks accommodating all 320 genotypes, with 40 rows per block. Each genotype was planted in a single row across three control and HNT stress replications in the 2018–2019 and 2019–2020 seasons. The tent space measured 8.2 m × 10.7 m, with each block containing 40 rows, spaced 19.1 cm apart and with a 0.5‐m alley between blocks. Trenches were dug to a depth of 3.8 cm using a tool with eight equally spaced prongs to ensure uniform depth and spacing throughout the blocks and tents. Seeds were hand‐planted at a rate of 88 seeds per row or 383 seeds/m<sup>2</sup>, equivalent to 4.45 million seeds per ha, after treatment with a seed mixture. Prior to planting each year, 47.07 kg N/ha of urea ammonium nitrate solution was applied. Post‐planting, 0.88 L/ha of MCPA herbicide and 2.19 cL/ha of Finesse Cereal and Fallow herbicide were applied, followed by 57.16 kg N/ha of urea ammonium nitrate solution. Additionally, 0.49 L/ha of Approach Prima fungicide was applied to manage Fusarium head blight. The stress period started when nearly all genotypes reached 50% anthesis, with only a 12‐day difference between the earliest and latest flowering genotypes (data not shown), and continued until physiological maturity. Optimal water levels, maintained through irrigation and rainfall, prevented water‐limited stress throughout both seasons.</p></section><section id="tpg270075-sec-0070"><h3 class="pmc_sec_title">2.4. Phenotypic trait evaluation and statistical analysis</h3>
<p>Spike numbers per line were recorded by counting the spikes in each row. Additionally, a 0.5 m central strip of each genotype was hand‐harvested at physiological maturity (Feekes 11.4). Spikes were separated from the stem and dried at 40°C for 96 h, while the biomass (all plant parts except spikes) was dried separately at 60°C for 168 h. Spike weight was measured, and the dried biomass was then combined with the spike weight (excluding grains) to determine the total biomass.</p>
<p>Combined analysis of variance (ANOVA) was performed across years to evaluate the effects of year, treatment, replication, and genotype, as well as their interactions, on biomass, spike number, and spike weight. Replication was included as a blocking factor. The analysis was conducted in R (R Core Team, <a href="#tpg270075-bib-0102" class="usa-link" aria-describedby="tpg270075-bib-0102">2024</a>) using the following packages: “stats” (for model fitting with lm) and “car” (for Type III ANOVA with the Anova function). META‐R (Multi Environment Trial Analysis with R) v6.04 (Alvarado et al., <a href="#tpg270075-bib-0006" class="usa-link" aria-describedby="tpg270075-bib-0006">2020</a>) was used to calculate best linear unbiased estimates (BLUEs) for all traits. For individual treatment‐year combinations (e.g., control_2018‐19, HNT_2018‐19), BLUEs were calculated using the following linear model:</p>
<table class="disp-formula p" id="tpg270075-disp-0001"><tr><td class="formula"><math id="jats-math-1" display="block"><mrow><mrow><msub><mi>Y</mi><mrow><mi>i</mi><mi>j</mi></mrow></msub><mo linebreak="badbreak">=</mo><mi mathvariant="normal">μ</mi><mo linebreak="goodbreak">+</mo><msub><mi>G</mi><mi>i</mi></msub><mo linebreak="goodbreak">+</mo><msub><mi>R</mi><mi>j</mi></msub><mo linebreak="goodbreak">+</mo><msub><mi>e</mi><mrow><mi>i</mi><mi>j</mi></mrow></msub></mrow></mrow></math></td></tr></table>
<p>where <em>Y<sub>ij</sub>
</em> is the observed trait value of genotype <em>i</em> in replicate <em>j</em>, <em>μ</em> is the overall mean, <em>G<sub>i</sub>
</em> is the fixed effect of genotype, <em>R<sub>j</sub>
</em> is the random effect of replicate, and <em>e<sub>ij</sub>
</em> is the residual error. Additionally, to estimate genotype performance across both years within each treatment (referred to as the control_combined and HNT_combined datasets), a linear mixed model was fitted as follows:
</p>
<table class="disp-formula p" id="tpg270075-disp-0002"><tr><td class="formula"><math id="jats-math-2" display="block"><mrow><mrow><msub><mi>Y</mi><mrow><mi>i</mi><mi>j</mi><mi>k</mi></mrow></msub><mo linebreak="badbreak">=</mo><mi mathvariant="normal">μ</mi><mo linebreak="goodbreak">+</mo><msub><mi>G</mi><mi>i</mi></msub><mo linebreak="goodbreak">+</mo><msub><mi>Y</mi><mi>j</mi></msub><mo linebreak="goodbreak">+</mo><msub><mi>R</mi><mrow><mi>k</mi><mo stretchy="false">(</mo><mi>j</mi><mo stretchy="false">)</mo></mrow></msub><mo linebreak="goodbreak">+</mo><mi>G</mi><msub><mi>Y</mi><mrow><mi>i</mi><mi>j</mi></mrow></msub><mo linebreak="goodbreak">+</mo><msub><mi>e</mi><mrow><mi>i</mi><mi>j</mi><mi>k</mi></mrow></msub></mrow></mrow></math></td></tr></table>
<p>where <em>Y<sub>ijk</sub>
</em> is the observed phenotype for genotype <em>i</em> in year <em>j</em> and replicate <em>k</em>, <em>μ</em> is the overall mean, <em>G<sub>i</sub>
</em> is the fixed effect of genotype, <em>Y<sub>j</sub>
</em> is the random effect of year, <em>R<sub>k​(j)</sub>
</em> is the random effect of replicate nested within year, <em>GY<sub>ij</sub>
</em> is the random genotype × year interaction, and <em>e<sub>ijk</sub>
</em> is the residual error. Furthermore, the broad‐sense heritability (<em>H</em>
<sup>2</sup>) for the three traits under control and HNT stress was estimated separately for both years as follows: <em>H</em>
<sup>2</sup> = <em>σ</em>
<sup>2</sup>
<sub>g</sub>/(<em>σ</em>
<sup>2</sup>
<sub>g</sub> + <em>σ</em>
<sup>2</sup>
<sub>e</sub>/<em>n</em>
<sub>Rep</sub>), where <em>σ</em>
<sup>2</sup>
<sub>g</sub> and <em>σ</em>
<sup>2</sup>
<sub>e</sub> are the estimated genotype and residual variance components, respectively, and <em>n</em>
<sub>Rep</sub> is the number of replicates.</p>
<p>Incorporating stress tolerance indices (STIs) alongside absolute trait values enhances trait mapping by integrating performance across control and stress environments, capturing genotype × environment (G × E) interactions, minimizing environmental noise, and improving the detection of adaptive QTLs linked to resilience and stability (Fernandez, <a href="#tpg270075-bib-0028" class="usa-link" aria-describedby="tpg270075-bib-0028">1992</a>; MajidiMehr et al., <a href="#tpg270075-bib-0078" class="usa-link" aria-describedby="tpg270075-bib-0078">2024</a>). Therefore, STIs for different genotypes were also calculated using the iPASTIC toolkit (Pour‐Aboughadareh et al., <a href="#tpg270075-bib-0098" class="usa-link" aria-describedby="tpg270075-bib-0098">2019</a>), which utilizes the following formula: (<em>Y<sub>S</sub>
</em> × <em>Y<sub>P</sub>
</em>)/(<em>X<sub>P</sub>
</em>)<sup>2</sup>, where <em>Y<sub>P</sub>
</em> is the trait value under control conditions, <em>Y<sub>S</sub>
</em> is the trait value under HNT conditions, and <em>X<sub>P</sub>
</em> is the grand mean of the trait in the control environment.</p>
<p>Overlapping histograms were generated for trait data collected over two growing seasons using the R package “ggplot” (<a href="https://cran.r-project.org/web/packages/ggplot2/index.html" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://cran.r‐project.org/web/packages/ggplot2/index.html</a>). Additionally, the estimated BLUEs were used to calculate correlations for various combinations between (i) traits within and across environments and (ii) the same traits across different environments. Correlation analysis was performed using the “corrplot” package (<a href="https://cran.r-project.org/web/packages/corrplot/index.html" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://cran.r‐project.org/web/packages/corrplot/index.html</a>) in R. Further, the percentage changes in phenotypic values under HNT stress compared to the control were calculated to assess its impact on the phenotypic performance of the genotypes.</p></section><section id="tpg270075-sec-0080"><h3 class="pmc_sec_title">2.5. SNP genotyping and population structure</h3>
<p>Winter wheat genotypes used in the study were previously genotyped using the wheat Infinium 90K iSelect array (Illumina Inc.) as part of the USDA–TCAP initiative. Genotypic data were acquired from the T3 Toolbox (<a href="https://wheat.triticeaetoolbox.org/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://wheat.triticeaetoolbox.org/</a>). The dataset was filtered to exclude SNPs with &gt;10% heterozygosity, over 30% missing data, and a minor allele frequency of &lt;5%. Following filtration, the genotypic dataset comprised 15,913 high‐quality SNPs, which were subsequently imputed using TASSEL v5.0 (Bradbury et al., <a href="#tpg270075-bib-0015" class="usa-link" aria-describedby="tpg270075-bib-0015">2007</a>). The physical positions of these imputed high‐quality SNP markers were retrieved from the IWGSCv1.0 genome assembly, accessible at Wheat‐URGI (<a href="https://wheat-urgi.versailles.inra.fr/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://wheat‐urgi.versailles.inra.fr/</a>). SNPs that were unassigned to any chromosomes were eliminated, as were those lacking physical positions. Ultimately, 261 genotypes were selected for further analysis based on the availability of high‐quality genotypic data and the presence of phenotypic data across both treatments and years.</p>
<p>The imputed genotypic data were utilized for principal component analysis (PCA) in R to investigate population structure, using the Genomic Association and Prediction Integrated Tool (GAPIT) version 3.0 (J. Wang &amp; Zhang, <a href="#tpg270075-bib-0124" class="usa-link" aria-describedby="tpg270075-bib-0124">2021</a>). Population structure was also assessed using STRUCTURE version 2.3.4, a Bayesian model‐based clustering program assuming an admixture model (Pritchard et al., <a href="#tpg270075-bib-0101" class="usa-link" aria-describedby="tpg270075-bib-0101">2000</a>). Ten subgroups (<em>K</em> = 1–10) were analyzed, with each undergoing five independent runs. Each run included a burn‐in period of 10,000 iterations followed by 10,000 Monte Carlo iterations. To determine the most probable number of subpopulations, an ad hoc statistic (i.e., DeltaK) was utilized, which assessed the rate of change in log probability across successive K‐values (Evanno et al., <a href="#tpg270075-bib-0025" class="usa-link" aria-describedby="tpg270075-bib-0025">2005</a>) with STRUCTURE HARVESTER (Earl &amp; VonHoldt, <a href="#tpg270075-bib-0024" class="usa-link" aria-describedby="tpg270075-bib-0024">2012</a>).</p></section><section id="tpg270075-sec-0090"><h3 class="pmc_sec_title">2.6. Genome‐wide association mapping</h3>
<p>Genome‐wide association analyses were conducted using a panel of 261 genotypes with physically anchored high‐quality SNPs to identify significant SNPs associated with different traits across nine conditions under two scenarios: (i) using absolute trait values (Control_2018‐2019, Control_2019‐2020, Control_combined, HNT_2018‐2019, HNT_2019‐2020, and HNT_combined) and (ii) using STI values of genotypes (STI_2018‐19, STI_2019‐20, and STI_combined). Initially, we employed three different single‐locus and three multi‐locus GWAS algorithms: single‐locus models including generalized linear model (Price et al., <a href="#tpg270075-bib-0100" class="usa-link" aria-describedby="tpg270075-bib-0100">2006</a>), mixed linear model (Yu et al., <a href="#tpg270075-bib-0131" class="usa-link" aria-describedby="tpg270075-bib-0131">2006</a>), and compressed mixed linear model (Z. Zhang et al., <a href="#tpg270075-bib-0136" class="usa-link" aria-describedby="tpg270075-bib-0136">2010</a>); and multi‐locus models including multi‐locus mixed linear model (Segura et al., <a href="#tpg270075-bib-0110" class="usa-link" aria-describedby="tpg270075-bib-0110">2012</a>), fixed and random model circulating probability unification (FarmCPU; X. Liu et al., <a href="#tpg270075-bib-0076" class="usa-link" aria-describedby="tpg270075-bib-0076">2016</a>), and Bayesian‐information and linkage‐disequilibrium iteratively nested keyway (M. Huang et al., <a href="#tpg270075-bib-0044" class="usa-link" aria-describedby="tpg270075-bib-0044">2019</a>). FarmCPU outperformed the other five methods in controlling <em>p</em>‐value inflation (as visualized by quantile–quantile plots) and in detecting significant SNPs that overlapped with known loci. Therefore, we selected FarmCPU to report the GWAS results in this study. Previous studies utilizing the HWWAMP panel and genotypic data from the wheat Infinium 90K iSelect array also concluded that FarmCPU outperforms other GWAS algorithms (AlTameemi et al., <a href="#tpg270075-bib-0005" class="usa-link" aria-describedby="tpg270075-bib-0005">2021</a>; Halder et al., <a href="#tpg270075-bib-0040" class="usa-link" aria-describedby="tpg270075-bib-0040">2023</a>). FarmCPU, an improved multiple‐locus model, effectively controls false positives by incorporating the associated markers identified from iterations as cofactors for marker tests within a fixed‐effect model (X. Liu et al., <a href="#tpg270075-bib-0076" class="usa-link" aria-describedby="tpg270075-bib-0076">2016</a>; Saini et al., <a href="#tpg270075-bib-0108" class="usa-link" aria-describedby="tpg270075-bib-0108">2022</a>).</p>
<p>FarmCPU‐based GWAS analysis was performed using the GAPIT version 3.0 in the R environment (J. Wang &amp; Zhang, <a href="#tpg270075-bib-0124" class="usa-link" aria-describedby="tpg270075-bib-0124">2021</a>). Additionally, to address population structure, the first two principal components were included based on visual examination of the scree plot, the DeltaK statistic from STRUCTURE analysis, and PCA results obtained from PCA analysis. The Bonferroni correction‐based threshold for declaring a significant association is generally considered overly stringent because it accounts for all SNPs in the dataset rather than independent tests. Therefore, many studies opt for either an exploratory threshold or a corrected Bonferroni threshold based on independent tests (AlTameemi et al., <a href="#tpg270075-bib-0005" class="usa-link" aria-describedby="tpg270075-bib-0005">2021</a>; Bheemanahalli, Wang, et al., <a href="#tpg270075-bib-0014" class="usa-link" aria-describedby="tpg270075-bib-0014">2021</a>; Halder et al., <a href="#tpg270075-bib-0040" class="usa-link" aria-describedby="tpg270075-bib-0040">2023</a>; Kaur et al., <a href="#tpg270075-bib-0053" class="usa-link" aria-describedby="tpg270075-bib-0053">2023</a>; Reddy et al., <a href="#tpg270075-bib-0105" class="usa-link" aria-describedby="tpg270075-bib-0105">2023</a>). In our study, we utilized an exploratory threshold of −log10(<em>P</em>) = 3.00, which is generally considered appropriate for a multi‐locus model that typically does not require multiple corrections (Y. M. Zhang, Jia, et al., <a href="#tpg270075-bib-0135" class="usa-link" aria-describedby="tpg270075-bib-0135">2019</a>). Significant SNPs that passed the threshold were categorized as (i) <em>multi‐trait SNPs</em> associated with two or more traits under a condition; (ii) SNPs detected in at least two control environments (absolute values); (iii) SNPs identified using both absolute values (control) and indices. SNPs from categories (ii) and (iii) were grouped as <em>QTLs under control</em>; (iv) SNPs detected in at least two HNT environments (absolute values); (v) SNPs identified using both absolute values (HNT) and indices. SNPs from categories (iv) and (v) were grouped as <em>HNT QTLs</em>; and (vi) SNPs identified using both absolute values (control and HNT) and indices, grouped as <em>condition‐independent QTLs</em>.</p></section><section id="tpg270075-sec-0100"><h3 class="pmc_sec_title">2.7. Individual allelic effects, joint allelic effects, and haplotype analyses</h3>
<p>Alleles from significant SNPs clustered within HNT QTL and environment‐independent QTL regions, identified through post‐GWAS analysis, were used to assess their effects on the traits of interest. For each significant SNP, trait values for two groups of alleles (favorable and unfavorable) were compared using two‐tailed <em>t</em>‐tests. We also aimed to analyze the impact of accumulated favorable alleles on trait phenotypes. Genotypes were grouped based on the alleles of significant SNPs identified for different traits, considering all significant SNPs for each trait collectively. Groups were compared using the “ggstatsplot” R package (Patil, <a href="#tpg270075-bib-0092" class="usa-link" aria-describedby="tpg270075-bib-0092">2021</a>) with the Welch <em>F</em>‐test to identify significant differences between means, addressing unequal variances effectively (Delacre et al., <a href="#tpg270075-bib-0018" class="usa-link" aria-describedby="tpg270075-bib-0018">2019</a>). Following this, the Games–Howell test, a post hoc analysis similar to Tukey's HSD (Honestly Significant Difference) but not assuming equal variances or sample sizes, was used to identify significantly different means (S. Lee &amp; Lee, <a href="#tpg270075-bib-0067" class="usa-link" aria-describedby="tpg270075-bib-0067">2018</a>).</p>
<p>Furthermore, the significant SNPs clustered within HNT QTL and environment‐independent QTL regions were utilized for detecting QTL‐based haplotypes. To generate and visualize linkage disequilibrium (LD)‐based haplotypes in specific regions, Haploview version 4.2 software (Barrett et al., <a href="#tpg270075-bib-0010" class="usa-link" aria-describedby="tpg270075-bib-0010">2005</a>) was used. The wheat genotypes were grouped according to the identified haplotypes. Trait data for each haplotype were analyzed using the “ggstatsplot” R package (Patil, <a href="#tpg270075-bib-0092" class="usa-link" aria-describedby="tpg270075-bib-0092">2021</a>) to test the statistical significance of effect sizes, utilizing the Welch <em>F</em>‐test and Games–Howell test, as mentioned above. For this analysis, combined phenotypic datasets (Control_combined and HNT_combined) were utilized.</p></section><section id="tpg270075-sec-0110"><h3 class="pmc_sec_title">2.8. CG mining, prioritization, and gene‐based haplotype analysis</h3>
<p>A common approach for identifying CGs associated with significant SNPs is either to mine gene models within the LD regions of the SNPs on different chromosomes separately (Gudi et al., <a href="#tpg270075-bib-0036" class="usa-link" aria-describedby="tpg270075-bib-0036">2023</a>; Pang et al., <a href="#tpg270075-bib-0089" class="usa-link" aria-describedby="tpg270075-bib-0089">2020</a>) or to mine CGs in fixed genomic regions surrounding the SNPs (Halder et al., <a href="#tpg270075-bib-0040" class="usa-link" aria-describedby="tpg270075-bib-0040">2023</a>; Kaur et al., <a href="#tpg270075-bib-0053" class="usa-link" aria-describedby="tpg270075-bib-0053">2023</a>). In the present study, gene models located within 1 Mb upstream and downstream of the peak positions of both HNT‐specific and condition‐independent QTLs were extracted using the BioMart tool from the EnsemblPlants database (<a href="http://plants.ensembl.org/index.html" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://plants.ensembl.org/index.html</a>), based on the IWGSC RefSeq v1.0 genome assembly of wheat. Functional annotations of the gene models were conducted based on the domains found in their corresponding protein sequences, obtained from the InterPro database (<a href="https://www.ebi.ac.uk/interpro/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ebi.ac.uk/interpro/</a>).</p>
<p>Once genes are identified, candidates for each locus are typically prioritized by examining their functional roles reported in the available literature and conducting in silico expression analyses using transcriptomics datasets generated under targeted stress conditions. However, research on genes influencing the genetic architecture of traits under HNT stress in wheat is limited, and no expression databases provide in silico data for wheat tissues under HNT. Therefore, we employed a different yet more reliable approach to prioritize high‐confidence CGs. A cross‐species genomic analysis was first conducted, focusing on transcriptomics and genetic studies on HNT in rice (Bheemanahalli, Knight, et al., <a href="#tpg270075-bib-0012" class="usa-link" aria-describedby="tpg270075-bib-0012">2021</a>; Desai et al., <a href="#tpg270075-bib-0020" class="usa-link" aria-describedby="tpg270075-bib-0020">2021</a>; J. Fan, Zhang, et al., <a href="#tpg270075-bib-0026" class="usa-link" aria-describedby="tpg270075-bib-0026">2024</a>; Glaubitz et al., <a href="#tpg270075-bib-0034" class="usa-link" aria-describedby="tpg270075-bib-0034">2017</a>; A. Kumar et al., <a href="#tpg270075-bib-0058" class="usa-link" aria-describedby="tpg270075-bib-0058">2021</a>; Liao et al., <a href="#tpg270075-bib-0074" class="usa-link" aria-describedby="tpg270075-bib-0074">2015</a>; Yang et al., <a href="#tpg270075-bib-0130" class="usa-link" aria-describedby="tpg270075-bib-0130">2024</a>). This analysis explored whether orthologs of the identified genes regulate HNT responses in the mentioned rice studies and compared their functional annotations with those of previously characterized rice genes. Next, the expression patterns of the shortlisted gene models were examined using the WheatOmics database (<a href="http://wheatomics.sdau.edu.cn/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://wheatomics.sdau.edu.cn/</a>) across various developmental stages, along with an analysis of transcript abundance in the target tissue. The shortlisted gene models were subjected to CG‐based haplotype analysis. For this, we considered the SNPs available within each gene and developed haplotypes based on their allelic variations across genotypes. Wheat genotypes were grouped according to the identified haplotypes, and trait data for each haplotype were analyzed using the “ggstatsplot” R package to assess the statistical significance of effect sizes, utilizing the Welch <em>F</em>‐test and Games–Howell test. Gene structures for the targeted CGs were retrieved from the EnsemblPlants database.</p></section></section><section id="tpg270075-sec-0120"><h2 class="pmc_sec_title">3. RESULTS</h2>
<section id="tpg270075-sec-0130"><h3 class="pmc_sec_title">3.1. Phenotypic responses of wheat genotypes and correlation among different traits</h3>
<p>All traits under control and HNT conditions exhibited a broad range of phenotypic variations, with continuous, near‐normal distributions across both years (Table <a href="#tpg270075-supinfo-0001" class="usa-link">S1</a>; Figure <a href="#tpg270075-fig-0001" class="usa-link">1</a>). The histograms showed overlapping patterns with shifts toward lower values under stress. Under HNT stress, wheat genotypes showed 4.2%–13.6% reductions in biomass, 6.7%–14.4% fewer spikes, and 12.1%–13.4% decreases in spike weight compared to the control, across different years (Figure <a href="#tpg270075-supinfo-0001" class="usa-link">S1</a>). Combined ANOVA revealed highly significant genotype effects for all three traits, along with significant treatment × genotype and year × treatment × genotype interactions (Table <a href="#tpg270075-supinfo-0001" class="usa-link">S2</a>). The coefficient of variation (CV%) for each trait varied across environments (Table <a href="#tpg270075-supinfo-0001" class="usa-link">S3</a>). All traits exhibited moderate to high broad‐sense heritability, ranging from 0.56 to 0.81 under HNT conditions and 0.61 to 0.85 under control conditions, with spike number showing the highest values across years (Table <a href="#tpg270075-supinfo-0001" class="usa-link">S3</a>). Significant positive correlations ( <em>P</em> &lt; 0.001) were observed between traits within and across years, with slightly stronger correlations under HNT conditions compared to control. Similarly, significant correlations (<em>P</em> &lt; 0.001) were found between the same traits across different conditions (Table <a href="#tpg270075-supinfo-0001" class="usa-link">S4</a>). The STIs indicated variation in trait distribution among genotypes across years, with biomass ranging from 0.25 (2019–2020) to 3.84 (2018–2019), spike number from 0.28 (2018–2019) to 2.09 (2019–2020), and spike weight from 0.30 to 1.98 (2019–2020) (Table <a href="#tpg270075-supinfo-0001" class="usa-link">S5</a>).</p>
<figure class="fig xbox font-sm" id="tpg270075-fig-0001"><h4 class="obj_head">FIGURE 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/0b36/12256648/f2eece8ed3f7/TPG2-18-e70075-g005.jpg" loading="lazy" id="jats-graphic-1" height="374" width="800" alt="FIGURE 1"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270075-fig-0001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Overlapping histograms showing the distribution of (A–C) biomass, spike number, and spike weight per unit area in wheat genotypes under control and high night‐time temperature (HNT) stress during 2018–2019, and (D–F) the same traits during 2019–2020. Dark gray columns represent phenotypic data under control conditions, white columns under HNT stress, and light gray areas indicate an overlap between the two conditions.</p></figcaption></figure></section><section id="tpg270075-sec-0140"><h3 class="pmc_sec_title">3.2. SNP genotyping and population structure analysis</h3>
<p>Genotypic data for 261 genotypes, consisting of 16,058 SNPs, were obtained from the T3 wheat database. After filtering out 145 SNPs with an MAF &lt; 5% and 3,632 SNPs without chromosomal assignments or physical positions, 12,281 high‐quality SNP markers were retained for further analysis. These SNPs covered approximately 13,424 Mbp across all 21 wheat chromosomes, with an average marker density of one SNP per 1.09 Mbp. The sub‐genomes A, B, and D contained 4,649, 5,234, and 2,398 SNP markers, respectively. Marker distribution per chromosome ranged from 47 markers on 7D to 1,041 markers on 5B (Table <a href="#tpg270075-supinfo-0001" class="usa-link">S6</a>). PCA indicated substantial genotype admixture, with the first two PCs explaining the majority of the variance (Figure <a href="#tpg270075-supinfo-0001" class="usa-link">S2A</a>). The DeltaK statistic from the STRUCTURE analysis revealed a single peak at <em>K</em> = 2, suggesting the presence of two subgroups within the association panel (Figure <a href="#tpg270075-supinfo-0001" class="usa-link">S2B</a>).</p></section><section id="tpg270075-sec-0150"><h3 class="pmc_sec_title">3.3. Genomic regions governing yield‐related traits under control and HNT stress conditions</h3>
<p>Genome‐wide association mapping identified 479 significant SNPs, with some SNPs detected multiple times, linked to three traits across nine conditions: (i) 46 SNPs under control_2018‐2019, (ii) 70 SNPs under control_2019‐2020, (iii) 42 SNPs under control_combined, (iv) 34 SNPs under HNT_2018‐2019, (v) 109 SNPs under HNT_2019‐2020, (vi) 26 SNPs under HNT_combined, (vii) 40 SNPs under STI_2018‐19, (viii) 67 SNPs under STI_2019‐20, and (ix) 45 SNPs under STI_combined (Table <a href="#tpg270075-tbl-0001" class="usa-link">1</a>). These SNPs were distributed across all wheat chromosomes (except 7D), with a minimum of 5 SNPs located each on 5B and 5D and a maximum of 109 SNPs detected on 4B (Table <a href="#tpg270075-supinfo-0001" class="usa-link">S7</a>). The number of significant SNPs also varied by trait (Table <a href="#tpg270075-tbl-0001" class="usa-link">1</a>). Of the 479 SNPs, nine were linked to multiple traits in the same environment, such as BS00012081_51, which was associated with spike weight and biomass under HNT in 2018–2019 (Table <a href="#tpg270075-supinfo-0001" class="usa-link">S8</a>). Overall, a total of 158 SNPs were associated with traits under control conditions, 169 SNPs under HNT conditions, and 152 SNPs were identified using STIs.</p>
<section class="tw xbox font-sm" id="tpg270075-tbl-0001"><h4 class="obj_head">TABLE 1.</h4>
<div class="caption p"><p>Summary of significant single nucleotide polymorphisms (SNPs) associated with plant biomass, spike number, and spike weight in wheat.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<thead>
<tr>
<th align="left" rowspan="1" colspan="1"></th>
<th align="left" rowspan="1" colspan="1"></th>
<th align="left" rowspan="1" colspan="1"></th>
<th align="left" rowspan="1" colspan="1"></th>
<th colspan="2" style="border-bottom:solid 1px #000000" align="left" rowspan="1">−log10(<em>P</em>)</th>
</tr>
<tr style="border-bottom:solid 1px #000000">
<th align="left" rowspan="1" colspan="1">Trait</th>
<th align="left" rowspan="1" colspan="1">Environment</th>
<th align="left" rowspan="1" colspan="1">Chr</th>
<th align="left" rowspan="1" colspan="1">Significant SNPs</th>
<th align="left" rowspan="1" colspan="1">Minimum</th>
<th align="left" rowspan="1" colspan="1">Maximum</th>
</tr>
</thead>
<tbody>
<tr>
<td align="left" rowspan="1" colspan="1">Biomass</td>
<td align="left" rowspan="1" colspan="1">Control_2018‐19</td>
<td align="left" rowspan="1" colspan="1">1B, 3A, 4A, 4B, 4D, 5A, 6A, 7B</td>
<td align="left" rowspan="1" colspan="1">9</td>
<td align="left" rowspan="1" colspan="1">3.11</td>
<td align="left" rowspan="1" colspan="1">5.66</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">HNT_2018‐19</td>
<td align="left" rowspan="1" colspan="1">1A, 2B, 3B, 4B, 4D, 5A, 5B, 5D, 7B</td>
<td align="left" rowspan="1" colspan="1">14</td>
<td align="left" rowspan="1" colspan="1">3.09</td>
<td align="left" rowspan="1" colspan="1">7.5</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">STI_2018‐19</td>
<td align="left" rowspan="1" colspan="1">2B, 3A, 3D, 4B, 5A, 5B, 6B, 7B</td>
<td align="left" rowspan="1" colspan="1">12</td>
<td align="left" rowspan="1" colspan="1">3.11</td>
<td align="left" rowspan="1" colspan="1">5.26</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">Control_2019‐20</td>
<td align="left" rowspan="1" colspan="1">2D, 4B, 4D, 5A, 6A, 6B, 6D, 7A</td>
<td align="left" rowspan="1" colspan="1">16</td>
<td align="left" rowspan="1" colspan="1">3.04</td>
<td align="left" rowspan="1" colspan="1">4.22</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">HNT_2019‐20</td>
<td align="left" rowspan="1" colspan="1">1B, 4A, 4B, 4D, 6A, 7A</td>
<td align="left" rowspan="1" colspan="1">36</td>
<td align="left" rowspan="1" colspan="1">3.02</td>
<td align="left" rowspan="1" colspan="1">3.81</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">STI_2019‐20</td>
<td align="left" rowspan="1" colspan="1">2D, 3A, 3B, 4A, 4B, 4D, 6A</td>
<td align="left" rowspan="1" colspan="1">49</td>
<td align="left" rowspan="1" colspan="1">3.08</td>
<td align="left" rowspan="1" colspan="1">3.97</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">Control_combined</td>
<td align="left" rowspan="1" colspan="1">3A, 3B, 4B, 5A, 7A, 7B</td>
<td align="left" rowspan="1" colspan="1">7</td>
<td align="left" rowspan="1" colspan="1">3.04</td>
<td align="left" rowspan="1" colspan="1">4.75</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">HNT_combined</td>
<td align="left" rowspan="1" colspan="1">1B, 2B, 3D, 4B, 4D, 5B, 5D, 6A, 7A, 7B</td>
<td align="left" rowspan="1" colspan="1">12</td>
<td align="left" rowspan="1" colspan="1">3.09</td>
<td align="left" rowspan="1" colspan="1">7.05</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">STI_Combined</td>
<td align="left" rowspan="1" colspan="1">2B, 2D, 4B, 6A, 6B, 7A, 7B</td>
<td align="left" rowspan="1" colspan="1">12</td>
<td align="left" rowspan="1" colspan="1">3.07</td>
<td align="left" rowspan="1" colspan="1">5.86</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Spike number</td>
<td align="left" rowspan="1" colspan="1">Control_2018‐19</td>
<td align="left" rowspan="1" colspan="1">3B, 4A, 6A, 6D, 7A</td>
<td align="left" rowspan="1" colspan="1">11</td>
<td align="left" rowspan="1" colspan="1">3.22</td>
<td align="left" rowspan="1" colspan="1">4.01</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">HNT_2018‐19</td>
<td align="left" rowspan="1" colspan="1">1B, 1D, 3B, 7A</td>
<td align="left" rowspan="1" colspan="1">5</td>
<td align="left" rowspan="1" colspan="1">3.08</td>
<td align="left" rowspan="1" colspan="1">3.33</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">STI_2018‐19</td>
<td align="left" rowspan="1" colspan="1">1B, 1D, 3B, 6A, 7A</td>
<td align="left" rowspan="1" colspan="1">9</td>
<td align="left" rowspan="1" colspan="1">3.12</td>
<td align="left" rowspan="1" colspan="1">3.49</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">Control_2019‐20</td>
<td align="left" rowspan="1" colspan="1">3B, 3D, 4D</td>
<td align="left" rowspan="1" colspan="1">7</td>
<td align="left" rowspan="1" colspan="1">3.02</td>
<td align="left" rowspan="1" colspan="1">3.52</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">HNT_2019‐20</td>
<td align="left" rowspan="1" colspan="1">1A, 1B, 1D, 2A, 2B, 2D, 3B, 4A, 5D</td>
<td align="left" rowspan="1" colspan="1">57</td>
<td align="left" rowspan="1" colspan="1">3.07</td>
<td align="left" rowspan="1" colspan="1">4.9</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">STI_2019‐20</td>
<td align="left" rowspan="1" colspan="1">1A, 4B, 4D</td>
<td align="left" rowspan="1" colspan="1">3</td>
<td align="left" rowspan="1" colspan="1">3.2</td>
<td align="left" rowspan="1" colspan="1">3.59</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">Control_combined</td>
<td align="left" rowspan="1" colspan="1">3B, 4D, 6A, 7A</td>
<td align="left" rowspan="1" colspan="1">9</td>
<td align="left" rowspan="1" colspan="1">3.02</td>
<td align="left" rowspan="1" colspan="1">3.52</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">HNT_combined</td>
<td align="left" rowspan="1" colspan="1">1A, 2D, 5D, 7A</td>
<td align="left" rowspan="1" colspan="1">4</td>
<td align="left" rowspan="1" colspan="1">3.02</td>
<td align="left" rowspan="1" colspan="1">3.48</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">STI_Combined</td>
<td align="left" rowspan="1" colspan="1">3B, 4B, 4D, 6A, 6D, 7A</td>
<td align="left" rowspan="1" colspan="1">13</td>
<td align="left" rowspan="1" colspan="1">3.02</td>
<td align="left" rowspan="1" colspan="1">3.6</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Spike weight</td>
<td align="left" rowspan="1" colspan="1">Control_2018‐19</td>
<td align="left" rowspan="1" colspan="1">1A, 1B, 1D, 3B, 4A, 4D, 6A, 6B, 6D, 7A</td>
<td align="left" rowspan="1" colspan="1">26</td>
<td align="left" rowspan="1" colspan="1">3.08</td>
<td align="left" rowspan="1" colspan="1">5.22</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">HNT_2018‐19</td>
<td align="left" rowspan="1" colspan="1">2B, 2D, 4B, 4D, 5B</td>
<td align="left" rowspan="1" colspan="1">15</td>
<td align="left" rowspan="1" colspan="1">3.17</td>
<td align="left" rowspan="1" colspan="1">3.97</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">STI_2018‐19</td>
<td align="left" rowspan="1" colspan="1">2B, 4B, 4D, 6A, 6D, 7B</td>
<td align="left" rowspan="1" colspan="1">19</td>
<td align="left" rowspan="1" colspan="1">3.33</td>
<td align="left" rowspan="1" colspan="1">4.49</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">Control_2019‐20</td>
<td align="left" rowspan="1" colspan="1">2D, 3A, 3B, 3D, 7B</td>
<td align="left" rowspan="1" colspan="1">47</td>
<td align="left" rowspan="1" colspan="1">3.04</td>
<td align="left" rowspan="1" colspan="1">4.3</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">HNT_2019‐20</td>
<td align="left" rowspan="1" colspan="1">1A, 4B, 4D, 5D, 6A</td>
<td align="left" rowspan="1" colspan="1">16</td>
<td align="left" rowspan="1" colspan="1">3.07</td>
<td align="left" rowspan="1" colspan="1">4.38</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">STI_2019‐20</td>
<td align="left" rowspan="1" colspan="1">1A, 1B, 2A, 3D, 4B, 6A</td>
<td align="left" rowspan="1" colspan="1">15</td>
<td align="left" rowspan="1" colspan="1">3.02</td>
<td align="left" rowspan="1" colspan="1">4.61</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">Control_combined</td>
<td align="left" rowspan="1" colspan="1">1D, 3A, 3D, 5A, 6A, 6B, 6D, 7A, 7B</td>
<td align="left" rowspan="1" colspan="1">26</td>
<td align="left" rowspan="1" colspan="1">3.01</td>
<td align="left" rowspan="1" colspan="1">4.12</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">HNT_combined</td>
<td align="left" rowspan="1" colspan="1">1A, 1B, 2B, 3B, 4B, 4D, 6A, 6D, 7A, 7B</td>
<td align="left" rowspan="1" colspan="1">10</td>
<td align="left" rowspan="1" colspan="1">3.04</td>
<td align="left" rowspan="1" colspan="1">9.77</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">STI_Combined</td>
<td align="left" rowspan="1" colspan="1">2B, 4B, 4D, 6A, 6D, 7B</td>
<td align="left" rowspan="1" colspan="1">20</td>
<td align="left" rowspan="1" colspan="1">3.07</td>
<td align="left" rowspan="1" colspan="1">4.2</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/tpg270075-tbl-0001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="tpg270075-tbl1-note-0001"><p>Abbreviations: HNT, high night temperature; STI, stress tolerance index.</p></div></div></section><p>A total of 29 unique SNPs were detected in at least two control conditions (absolute values), while 24 unique SNPs were identified using both absolute trait values (under control) and STIs, resulting in 20 control QTLs (<em>Q1</em>–<em>Q20</em>) (Table <a href="#tpg270075-supinfo-0001" class="usa-link">S9</a>). In a similar manner, two unique SNPs were detected in at least two HNT conditions (absolute values), and 37 unique SNPs were identified using both absolute trait values (under HNT) and STIs, leading to the identification of 19 HNT QTLs (<em>Q21</em>–<em>Q39</em>) (Table <a href="#tpg270075-tbl-0002" class="usa-link">2</a>). Furthermore, 28 SNPs were identified using both absolute trait values (under both control and HNT conditions) and STIs, which were grouped into seven condition‐independent QTLs (Table <a href="#tpg270075-tbl-0002" class="usa-link">2</a>). Among the 19 HNT QTLs, five were associated with biomass, five with spike number, and two with spike weight. Seven QTLs were linked to two or more traits: <em>Q24</em> to biomass and spike number, <em>Q27</em>, <em>Q32</em>, <em>Q34</em>, and <em>Q36</em> to biomass and spike weight, <em>Q22</em> to spike number and weight, and <em>Q31</em> to all three traits. The −log10(<em>P</em>) values for these QTLs ranged from 3.02 to 7.67 (Table <a href="#tpg270075-tbl-0002" class="usa-link">2</a>). Among the seven condition‐independent QTLs, two (<em>Q42</em> and <em>Q46</em>) were associated with biomass, four (<em>Q40</em>, <em>Q41</em>, <em>Q44</em>, and <em>Q45</em>) were associated with both biomass and spike weight, and one QTL (<em>Q43</em>) was linked to all three traits. The −log10(<em>P</em>) values for these QTLs ranged from 3.02 to 9.77 (Table <a href="#tpg270075-tbl-0002" class="usa-link">2</a>).</p>
<section class="tw xbox font-sm" id="tpg270075-tbl-0002"><h4 class="obj_head">TABLE 2.</h4>
<div class="caption p"><p>High night‐time temperature (HNT) quantitative trait loci (QTLs) and condition‐independent QTLs are identified in the present study.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<thead><tr style="border-bottom:solid 1px #000000">
<th align="left" rowspan="1" colspan="1">QTL ID</th>
<th align="left" rowspan="1" colspan="1">Chr</th>
<th align="left" rowspan="1" colspan="1">Traits</th>
<th align="left" rowspan="1" colspan="1">QTL peak region</th>
<th align="left" rowspan="1" colspan="1">SNPs included</th>
<th align="left" rowspan="1" colspan="1">−log10(<em>P</em>)</th>
<th align="left" rowspan="1" colspan="1">Environments</th>
</tr></thead>
<tbody>
<tr><td colspan="7" align="left" rowspan="1">
<strong>(A) HNT QTLs</strong>
</td></tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<em>Q21</em>
</td>
<td align="left" rowspan="1" colspan="1">2D</td>
<td align="left" rowspan="1" colspan="1">Spike number</td>
<td align="left" rowspan="1" colspan="1">645,046,980</td>
<td align="left" rowspan="1" colspan="1">GENE‐0762_808</td>
<td align="left" rowspan="1" colspan="1">3.08–4.90</td>
<td align="left" rowspan="1" colspan="1">HNT_2019‐20, HNT_combined</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<em>Q22</em>
</td>
<td align="left" rowspan="1" colspan="1">5D</td>
<td align="left" rowspan="1" colspan="1">Spike number and spike weight</td>
<td align="left" rowspan="1" colspan="1">453,807,686</td>
<td align="left" rowspan="1" colspan="1">Excalibur_c34793_1260</td>
<td align="left" rowspan="1" colspan="1">3.21–4.63</td>
<td align="left" rowspan="1" colspan="1">HNT_2019‐20, HNT_combined</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<em>Q23</em>
</td>
<td align="left" rowspan="1" colspan="1">1A</td>
<td align="left" rowspan="1" colspan="1">Spike weight</td>
<td align="left" rowspan="1" colspan="1">303,853,976</td>
<td align="left" rowspan="1" colspan="1">Kukri_c49091_371</td>
<td align="left" rowspan="1" colspan="1">3.04–3.90</td>
<td align="left" rowspan="1" colspan="1">HNT_2019‐20, HNT_combined, STI_2019‐20</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<em>Q24</em>
</td>
<td align="left" rowspan="1" colspan="1">1A</td>
<td align="left" rowspan="1" colspan="1">Biomass and spike number</td>
<td align="left" rowspan="1" colspan="1">498,048,808</td>
<td align="left" rowspan="1" colspan="1">wsnp_Ex_rep_c109742_92411838</td>
<td align="left" rowspan="1" colspan="1">3.02–5.02</td>
<td align="left" rowspan="1" colspan="1">HNT_2018‐19, HNT_2019‐20, HNT_combined, STI_2019‐20</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<em>Q25</em>
</td>
<td align="left" rowspan="1" colspan="1">1B</td>
<td align="left" rowspan="1" colspan="1">Spike number</td>
<td align="left" rowspan="1" colspan="1">3,657,935</td>
<td align="left" rowspan="1" colspan="1">Excalibur_c35316_137</td>
<td align="left" rowspan="1" colspan="1">3.23–3.33</td>
<td align="left" rowspan="1" colspan="1">HNT_2018‐19, STI_2018‐19</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<em>Q26</em>
</td>
<td align="left" rowspan="1" colspan="1">1D</td>
<td align="left" rowspan="1" colspan="1">Spike number</td>
<td align="left" rowspan="1" colspan="1">578,942—579,017</td>
<td align="left" rowspan="1" colspan="1">Excalibur_c12254_2003, Kukri_c20775_56</td>
<td align="left" rowspan="1" colspan="1">3.23–3.33</td>
<td align="left" rowspan="1" colspan="1">HNT_2018‐19, STI_2018‐19</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<em>Q27</em>
</td>
<td align="left" rowspan="1" colspan="1">2B</td>
<td align="left" rowspan="1" colspan="1">Biomass and spike weight</td>
<td align="left" rowspan="1" colspan="1">41,144,291–41,145,744</td>
<td align="left" rowspan="1" colspan="1">BS00012081_51, RAC875_c15844_348, RAC875_c31358_214</td>
<td align="left" rowspan="1" colspan="1">3.07–5.52</td>
<td align="left" rowspan="1" colspan="1">HNT_2018‐19, STI_combined</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<em>Q28</em>
</td>
<td align="left" rowspan="1" colspan="1">2B</td>
<td align="left" rowspan="1" colspan="1">Biomass</td>
<td align="left" rowspan="1" colspan="1">782,534,064</td>
<td align="left" rowspan="1" colspan="1">CAP11_c1820_244</td>
<td align="left" rowspan="1" colspan="1">3.41–5.47</td>
<td align="left" rowspan="1" colspan="1">HNT_2018‐19, HNT_combined, STI_2018‐19, STI_combined</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<em>Q29</em>
</td>
<td align="left" rowspan="1" colspan="1">3B</td>
<td align="left" rowspan="1" colspan="1">Spike number</td>
<td align="left" rowspan="1" colspan="1">59,064,210</td>
<td align="left" rowspan="1" colspan="1">BS00003822_51</td>
<td align="left" rowspan="1" colspan="1">3.08–3.12</td>
<td align="left" rowspan="1" colspan="1">HNT_2018‐19, STI_2018‐19</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<em>Q30</em>
</td>
<td align="left" rowspan="1" colspan="1">4A</td>
<td align="left" rowspan="1" colspan="1">Biomass</td>
<td align="left" rowspan="1" colspan="1">30,397,789</td>
<td align="left" rowspan="1" colspan="1">GENE‐2361_194</td>
<td align="left" rowspan="1" colspan="1">3.05–3.37</td>
<td align="left" rowspan="1" colspan="1">HNT_2019‐20, STI_2019‐20</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<em>Q31</em>
</td>
<td align="left" rowspan="1" colspan="1">4B</td>
<td align="left" rowspan="1" colspan="1">Biomass, spike number, and spike weight</td>
<td align="left" rowspan="1" colspan="1">13,977,166–15,413,216</td>
<td align="left" rowspan="1" colspan="1">BobWhite_c162_145, Excalibur_c7581_1266, Tdurum_contig76559_124, Excalibur_c7581_791, Tdurum_contig50625_2342, Excalibur_rep_c79414_306, IACX8647, BS00010115_51, BS00009480_51, IACX7746, GENE‐4933_1095, GENE‐4933_1085, Tdurum_contig76213_958, Tdurum_contig76213_1156</td>
<td align="left" rowspan="1" colspan="1">3.07–3.75</td>
<td align="left" rowspan="1" colspan="1">HNT_2018‐19, HNT_2019‐20, STI_2018‐19, STI_2019‐20, STI_combined</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<em>Q32</em>
</td>
<td align="left" rowspan="1" colspan="1">4B</td>
<td align="left" rowspan="1" colspan="1">Biomass and spike weight</td>
<td align="left" rowspan="1" colspan="1">26,491,532</td>
<td align="left" rowspan="1" colspan="1">Tdurum_contig64772_417</td>
<td align="left" rowspan="1" colspan="1">3.07–3.74</td>
<td align="left" rowspan="1" colspan="1">HNT_2018‐19, HNT_combined, STI_combined</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<em>Q33</em>
</td>
<td align="left" rowspan="1" colspan="1">4B</td>
<td align="left" rowspan="1" colspan="1">Biomass</td>
<td align="left" rowspan="1" colspan="1">167,346,674</td>
<td align="left" rowspan="1" colspan="1">Excalibur_c52517_464</td>
<td align="left" rowspan="1" colspan="1">4.03–6.07</td>
<td align="left" rowspan="1" colspan="1">HNT_2018‐19, HNT_combined, STI_2018‐19, STI_combined</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<em>Q34</em>
</td>
<td align="left" rowspan="1" colspan="1">4D</td>
<td align="left" rowspan="1" colspan="1">Biomass and spike weight</td>
<td align="left" rowspan="1" colspan="1">7,646,744–80,84,249</td>
<td align="left" rowspan="1" colspan="1">wsnp_Ra_c1022_2067517, GENE‐4933_489, BS00009426_51, Excalibur_c7377_445, Kukri_c10445_225</td>
<td align="left" rowspan="1" colspan="1">3.09–7.67</td>
<td align="left" rowspan="1" colspan="1">HNT_2018‐19, HNT_2019‐20, HNT_combined, STI_2018‐19, STI_combined</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<em>Q35</em>
</td>
<td align="left" rowspan="1" colspan="1">5A</td>
<td align="left" rowspan="1" colspan="1">Biomass</td>
<td align="left" rowspan="1" colspan="1">1,225,706</td>
<td align="left" rowspan="1" colspan="1">wsnp_Ex_c6209_10838456</td>
<td align="left" rowspan="1" colspan="1">4.46–4.87</td>
<td align="left" rowspan="1" colspan="1">HNT_2018‐19, STI_2018‐19</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<em>Q36</em>
</td>
<td align="left" rowspan="1" colspan="1">5B</td>
<td align="left" rowspan="1" colspan="1">Biomass and spike weight</td>
<td align="left" rowspan="1" colspan="1">595,662,002</td>
<td align="left" rowspan="1" colspan="1">BS00023064_51</td>
<td align="left" rowspan="1" colspan="1">3.11–7.50</td>
<td align="left" rowspan="1" colspan="1">HNT_2018‐19, HNT_combined, STI_2018‐19</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<em>Q37</em>
</td>
<td align="left" rowspan="1" colspan="1">6A</td>
<td align="left" rowspan="1" colspan="1">Spike weight</td>
<td align="left" rowspan="1" colspan="1">65,96,867</td>
<td align="left" rowspan="1" colspan="1">Tdurum_contig55193_296</td>
<td align="left" rowspan="1" colspan="1">3.07–3.59</td>
<td align="left" rowspan="1" colspan="1">HNT_2019‐20, STI_2019‐20</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<em>Q38</em>
</td>
<td align="left" rowspan="1" colspan="1">7A</td>
<td align="left" rowspan="1" colspan="1">Biomass</td>
<td align="left" rowspan="1" colspan="1">669,728,991</td>
<td align="left" rowspan="1" colspan="1">IACX2471</td>
<td align="left" rowspan="1" colspan="1">3.02–4.38</td>
<td align="left" rowspan="1" colspan="1">HNT_2019‐20, STI_combined</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<em>Q39</em>
</td>
<td align="left" rowspan="1" colspan="1">7A</td>
<td align="left" rowspan="1" colspan="1">Spike number</td>
<td align="left" rowspan="1" colspan="1">688,967,553</td>
<td align="left" rowspan="1" colspan="1">Excalibur_c113078_320</td>
<td align="left" rowspan="1" colspan="1">3.24–3.48</td>
<td align="left" rowspan="1" colspan="1">HNT_2018‐19, HNT_combined, STI_2018‐19, STI_combined</td>
</tr>
<tr><td colspan="7" align="left" rowspan="1">
<strong>(B) Condition‐independent QTLs</strong>
</td></tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<em>Q40</em>
</td>
<td align="left" rowspan="1" colspan="1">2D</td>
<td align="left" rowspan="1" colspan="1">Biomass and spike weight</td>
<td align="left" rowspan="1" colspan="1">25,622,027</td>
<td align="left" rowspan="1" colspan="1">Tdurum_contig42153_1190</td>
<td align="left" rowspan="1" colspan="1">3.05–3.26</td>
<td align="left" rowspan="1" colspan="1">Control_2019‐20, HNT_2018‐19, STI_2019‐20</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<em>Q41</em>
</td>
<td align="left" rowspan="1" colspan="1">3D</td>
<td align="left" rowspan="1" colspan="1">Biomass and spike weight</td>
<td align="left" rowspan="1" colspan="1">603,603,057–604,666,729</td>
<td align="left" rowspan="1" colspan="1">RAC875_c111148_324, RAC875_c61950_377, wsnp_Ex_c12424_19815550, Kukri_c23354_183, RAC875_c63331_327</td>
<td align="left" rowspan="1" colspan="1">3.02–5.34</td>
<td align="left" rowspan="1" colspan="1">Control_2019‐20, HNT_combined, STI_2019‐20</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<em>Q42</em>
</td>
<td align="left" rowspan="1" colspan="1">4B</td>
<td align="left" rowspan="1" colspan="1">Biomass</td>
<td align="left" rowspan="1" colspan="1">535,049,553–536,919,272</td>
<td align="left" rowspan="1" colspan="1">BobWhite_c7157_633, GENE‐2636_193, Kukri_c5502_2513, wsnp_Ku_c5502_9765942, wsnp_JD_c33645_25575463, wsnp_Ku_c5502_9764246, BS00046613_51, wsnp_Ex_c50195_54565006, Kukri_c89906_197, RAC875_c5087_310, Tdurum_contig55374_309</td>
<td align="left" rowspan="1" colspan="1">3.43–5.86</td>
<td align="left" rowspan="1" colspan="1">Control_2019‐20, Control_combined, HNT_2019‐20, HNT_combined, STI_2018‐19, STI_2019‐20, STI_combined</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<em>Q43</em>
</td>
<td align="left" rowspan="1" colspan="1">6A</td>
<td align="left" rowspan="1" colspan="1">Biomass, spike number, and spike weight</td>
<td align="left" rowspan="1" colspan="1">3,746,733–5,364,009</td>
<td align="left" rowspan="1" colspan="1">Kukri_c39873_66, Tdurum_contig12045_868, RAC875_c4668_771, Tdurum_contig12045_246, TA005787‐0140, Tdurum_contig29878_59</td>
<td align="left" rowspan="1" colspan="1">3.04–5.66</td>
<td align="left" rowspan="1" colspan="1">Control_2018‐19, Control_combined, HNT_2019‐20, STI_2018‐19, STI_2019‐20</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<em>Q44</em>
</td>
<td align="left" rowspan="1" colspan="1">6A</td>
<td align="left" rowspan="1" colspan="1">Biomass and spike weight</td>
<td align="left" rowspan="1" colspan="1">613,485,873–615,468,963</td>
<td align="left" rowspan="1" colspan="1">wsnp_Ex_c28973_38050174, Kukri_c494_1479</td>
<td align="left" rowspan="1" colspan="1">3.53–4.87</td>
<td align="left" rowspan="1" colspan="1">Control_2019‐20, HNT_combined, STI_2019‐20</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<em>Q45</em>
</td>
<td align="left" rowspan="1" colspan="1">7B</td>
<td align="left" rowspan="1" colspan="1">Biomass and spike weight</td>
<td align="left" rowspan="1" colspan="1">17,656,905</td>
<td align="left" rowspan="1" colspan="1">BobWhite_c44404_312</td>
<td align="left" rowspan="1" colspan="1">3.04–9.77</td>
<td align="left" rowspan="1" colspan="1">Control_combined, HNT_2018‐19, HNT_combined, STI_2018‐19, STI_combined</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<em>Q46</em>
</td>
<td align="left" rowspan="1" colspan="1">7B</td>
<td align="left" rowspan="1" colspan="1">Biomass</td>
<td align="left" rowspan="1" colspan="1">701,188,928–701,872,166</td>
<td align="left" rowspan="1" colspan="1">Kukri_c7036_87, wsnp_Ex_c8400_14157318</td>
<td align="left" rowspan="1" colspan="1">4.03–5.19</td>
<td align="left" rowspan="1" colspan="1">Control_combined, HNT_combined, STI_combined</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/tpg270075-tbl-0002/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="tpg270075-tbl2-note-0001"><p>Abbreviations: SNPs, single nucleotide polymorphisms; STI: stress tolerance index.</p></div></div></section></section><section id="tpg270075-sec-0160"><h3 class="pmc_sec_title">3.4. Impact of allelic variation and additive effects on traits</h3>
<p>The study also examined how different alleles of SNPs within various HNT and condition‐independent QTLs influence trait variation. Two‐tailed <em>t</em>‐tests revealed significant mean trait differences for most SNPs in HNT and condition‐independent QTLs (<em>P</em> ≤ 0.001), except for a few SNPs in some QTLs (Table <a href="#tpg270075-supinfo-0001" class="usa-link">S10</a>). The frequency of favorable alleles across different loci ranged from 8.81% to 94.25%, with over 60% of loci showing favorable alleles in &lt;50% of the genotypes (Table <a href="#tpg270075-supinfo-0001" class="usa-link">S10</a>). Genotypes were further grouped based on the number of favorable alleles per trait to evaluate their cumulative effect. In some cases, significant SNPs clustered in proximal QTLs showed similar distributions of favorable and non‐favorable alleles among genotypes (Table <a href="#tpg270075-supinfo-0001" class="usa-link">S10</a>). For biomass, the mean trait value was 55.1 g without any favorable allele. The presence of one favorable allele increased the mean to 67.9 g, reaching 98.5 g with 33 favorable alleles. However, the increase in mean value was not linear with the number of favorable alleles; for instance, no consistent rise was observed when the number of favorable alleles increased from 10 to 20 (Figure <a href="#tpg270075-fig-0002" class="usa-link">2A</a>). A similar pattern was observed for spike number, with mean values increasing from 77.9 (no favorable alleles) to 112.1 (18 alleles), though additional alleles did not always correspond to higher trait values (Figure <a href="#tpg270075-fig-0002" class="usa-link">2B</a>). For spike weight, the mean value increased from 61.4 g (no favorable alleles) to 69.5 g with one favorable allele, peaking at 83.4 g with 22–23 favorable alleles (Figure <a href="#tpg270075-fig-0002" class="usa-link">2C</a>).</p>
<figure class="fig xbox font-sm" id="tpg270075-fig-0002"><h4 class="obj_head">FIGURE 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/0b36/12256648/598d40b9f7e9/TPG2-18-e70075-g003.jpg" loading="lazy" id="jats-graphic-3" height="210" width="800" alt="FIGURE 2"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270075-fig-0002/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Effect of accumulating favorable alleles from high night‐time temperature (HNT) and condition‐independent quantitative trait loci (QTLs) on various traits: (A) plant biomass; (B) spike number; (C) spike weight. Joint effect analysis included QTLs detected in at least two HNT conditions (<em>Q21</em>, <em>Q22</em>, <em>Q23</em>, <em>Q24</em>, <em>Q28</em>, <em>Q31</em>, <em>Q32</em>, <em>Q33</em>, <em>Q34</em>, <em>Q36</em>, and <em>Q39</em>) and those found in both CNT (control) and HNT across at least two environments each (<em>Q42</em>). The <em>x</em>‐axis represents favorable allele count, and the <em>y</em>‐axis represents the corresponding phenotypic value. In some cases, significant single nucleotide polymorphisms (SNPs) clustered in proximal QTLs showed similar distribution of favorable and non‐favorable alleles among genotypes. For instance, in panel (A), alleles 12–20 exhibited identical patterns across the genotypes. Significantly different groups are indicated with asterisks (above the boxplots) based on the Games–Howell post hoc test: <sup>*</sup>
<em>P</em> ≤ 0.05, <sup>**</sup>
<em>P</em> ≤ 0.01, and <sup>***</sup>
<em>P</em> ≤ 0.001.</p></figcaption></figure></section><section id="tpg270075-sec-0170"><h3 class="pmc_sec_title">3.5. Haplotypes associated with different traits contributing to HNT tolerance</h3>
<p>Initially, all significant SNPs from 19 HNT and 7 condition‐independent QTLs were considered for haplotype analysis. Some SNPs were not in LD with others, while some were in LD with only one SNP, leading to their exclusion. Nine haplotype blocks with 48 unique SNPs from nine HNT QTLs were identified, eight of which showed significant associations with trait variations under HNT conditions (<em>P</em>‐values &lt; 0.05 to &lt; 0.001) (Table <a href="#tpg270075-supinfo-0001" class="usa-link">S11</a>). Five haplotype blocks were identified from five of the seven condition‐independent QTLs, with four showing significant associations under both conditions and one only under HNT. Some multi‐trait QTLs resulted in haplotype blocks linked to multiple traits, though not all associations were significant (Table <a href="#tpg270075-supinfo-0001" class="usa-link">S11</a>).</p>
<p>Some of the key haplotypes are as follows: (i) The <em>Q24</em>‐based haplotype block (involving four SNPs within the 498,536,695–498,048,808 bp region) associated with biomass and spike number had three haplotypes (Hap1, Hap2, and Hap3), occurring at frequencies of 19.2%, 64.4%, and 12.6%, respectively. Hap2 exhibited the highest average biomass (69.2 g) and spike number (90.2), while Hap3 had the lowest biomass (59.9 g) and spike number (78.1) under HNT conditions (Table <a href="#tpg270075-supinfo-0001" class="usa-link">S11</a>). (ii) The <em>Q31</em>‐based haplotype block on chromosome 4B (involving 12 SNPs within the 14,110,534–14,124,098 bp region), linked to all three traits, had three haplotypes with frequencies of 30.7%, 43.3%, and 25.3% (Figure <a href="#tpg270075-fig-0003" class="usa-link">3A</a>). Hap2 showed the highest average biomass (70.8 g), spike number (89.4), and spike weight (71.3), whereas Hap1 had the lowest biomass (61.7 g), spike number (83.7), and spike weight (64.5) under HNT conditions (Figure <a href="#tpg270075-fig-0003" class="usa-link">3B–D</a>). (iii) Another haplotype block on chromosome 4B (involving nine SNPs within the 534,736,640–535,085,059 bp region), based on a condition‐independent QTL (i.e., <em>Q42</em>), included three haplotypes (Hap1, Hap2, and Hap3), occurring at frequencies of 26.2%, 64.2%, and 8.1%, respectively (Figure <a href="#tpg270075-fig-0003" class="usa-link">3A</a>). Hap3 had the highest average biomass (85.7 g under control; 80.0 g under HNT), followed by Hap1 (81.8 g under control; 74.6 g under HNT) and Hap2 (67.0 g under control; 61.0 g under HNT). These superior and inferior haplotypes were significantly different from each other, mostly at <em>P</em> &lt; 0.001 (Figure <a href="#tpg270075-fig-0003" class="usa-link">3E,F</a>).</p>
<figure class="fig xbox font-sm" id="tpg270075-fig-0003"><h4 class="obj_head">FIGURE 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/0b36/12256648/f9277d205eda/TPG2-18-e70075-g004.jpg" loading="lazy" id="jats-graphic-5" height="329" width="800" alt="FIGURE 3"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270075-fig-0003/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Representative quantitative trait locus (QTL)‐based haplotypes identified in this study: (A) Chromosome map of 4B showing the positions of two QTLs, <em>Q31</em> and <em>Q42</em>. Linkage disequilibrium (LD) blocks for regions containing significant single nucleotide polymorphisms (SNPs): a 13,564 bp block (<em>Q31</em>) with three haplotypes from 12 SNPs and a 348,419 bp block (<em>Q42</em>) with three haplotypes from nine SNPs, both with haplotype frequencies. Differences in (B) biomass, (C) spike number, and (D) spike weight among the three haplotypes from the <em>Q31</em> block, and in (E) control (CNT) biomass and (F) high night‐time temperature (HNT) biomass among the three haplotypes from the <em>Q42</em> block. *<em>P</em> ≤ 0.05, **<em>P</em> ≤ 0.01, *<em>P</em> ≤ 0.001.</p></figcaption></figure></section><section id="tpg270075-sec-0180"><h3 class="pmc_sec_title">3.6. CGs and gene‐based haplotypes associated with HNT stress response</h3>
<p>An in silico search within QTL regions identified 766 gene models: 623 associated with HNT QTLs and 143 associated with condition‐independent QTLs. Among the 766 gene models, 652 had functional annotations, while 114 were uncharacterized (Table <a href="#tpg270075-supinfo-0001" class="usa-link">S12</a>). These gene models were unevenly distributed across the different chromosomes, with the minimum number of genes (15) on 4A and the maximum number (115) on 6A (Table <a href="#tpg270075-supinfo-0001" class="usa-link">S12</a>). Several gene models with similar functional descriptions were repeatedly identified across different QTL regions, including 36 F‐box domain proteins, 33 protein kinases, 15 BTB/POZ domain proteins, 15 cytochrome P450s, 12 glycoside hydrolases, 12 SANT/Myb domain proteins, 12 zinc finger proteins, and 10 RNA recognition motif domain proteins (Table <a href="#tpg270075-supinfo-0001" class="usa-link">S12</a>).</p>
<p>Wheat–rice comparative genomics analysis identified 176 gene models as potential contributors to HNT response, showing functional similarities to previously characterized HNT stress genes in rice (Table <a href="#tpg270075-supinfo-0001" class="usa-link">S12</a>). Expression patterns across different wheat tissues narrowed this list to 44 genes with at least 2 TPM expression in one tissue. Among these, 17 genes (13 from HNT QTLs and four from condition‐independent QTLs) were designated as high‐confidence CGs based on two criteria: (i) very high expression in target tissues, such as spikes for genes available from HNT QTL linked to spike weight and (ii) highly effective under HNT stress response based on corresponding rice gene studies. These 17 genes encode key proteins, including ATP‐dependent RNA helicase DEAD‐box, GCN5‐related N‐acetyltransferase, oxidoreductase FAD/NAD(P)‐binding, protein kinases, ribose 5‐phosphate isomerase, RNA recognition motif domain proteins, and sugar/inositol transporters (Table <a href="#tpg270075-tbl-0003" class="usa-link">3</a>). The expression patterns of these selected high‐confidence CGs across various tissues are shown in Figure <a href="#tpg270075-supinfo-0001" class="usa-link">S3</a>.</p>
<section class="tw xbox font-sm" id="tpg270075-tbl-0003"><h4 class="obj_head">TABLE 3.</h4>
<div class="caption p"><p>High‐confidence candidate genes available from high night‐time temperature (HNT) quantitative trait loci (QTLs) and condition‐independent QTLs.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<thead><tr style="border-bottom:solid 1px #000000">
<th align="left" rowspan="1" colspan="1">QTL ID</th>
<th align="left" rowspan="1" colspan="1">Trait</th>
<th align="left" rowspan="1" colspan="1">Gene stable ID</th>
<th align="left" rowspan="1" colspan="1">Gene start (bp)</th>
<th align="left" rowspan="1" colspan="1">Gene end (bp)</th>
<th align="left" rowspan="1" colspan="1">Functional annotation</th>
</tr></thead>
<tbody>
<tr><td colspan="6" align="left" rowspan="1">
<strong>(A) HNT QTLs</strong>
</td></tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<em>Q22</em>
</td>
<td align="left" rowspan="1" colspan="1">Spike number and spike weight</td>
<td align="left" rowspan="1" colspan="1">
<em>TraesCS5D02G385300</em>
</td>
<td align="left" rowspan="1" colspan="1">454,414,059</td>
<td align="left" rowspan="1" colspan="1">454,419,771</td>
<td align="left" rowspan="1" colspan="1">Homeobox domain</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<em>Q24</em>
</td>
<td align="left" rowspan="1" colspan="1">Biomass and spike number</td>
<td align="left" rowspan="1" colspan="1">
<em>TraesCS1A02G305700</em>
</td>
<td align="left" rowspan="1" colspan="1">498,138,711</td>
<td align="left" rowspan="1" colspan="1">498,142,127</td>
<td align="left" rowspan="1" colspan="1">Oxidoreductase FAD/NAD(P)‐binding</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<em>Q28</em>
</td>
<td align="left" rowspan="1" colspan="1">Biomass</td>
<td align="left" rowspan="1" colspan="1">
<em>TraesCS2B02G599800</em>
</td>
<td align="left" rowspan="1" colspan="1">782,533,511</td>
<td align="left" rowspan="1" colspan="1">782,538,118</td>
<td align="left" rowspan="1" colspan="1">Glycoside hydrolase family 1</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<em>Q28</em>
</td>
<td align="left" rowspan="1" colspan="1">Biomass</td>
<td align="left" rowspan="1" colspan="1">
<em>TraesCS2B02G600200</em>
</td>
<td align="left" rowspan="1" colspan="1">783,226,574</td>
<td align="left" rowspan="1" colspan="1">783,231,181</td>
<td align="left" rowspan="1" colspan="1">Glycoside hydrolase family 1</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<em>Q29</em>
</td>
<td align="left" rowspan="1" colspan="1">Spike number</td>
<td align="left" rowspan="1" colspan="1">
<em>TraesCS3B02G089200</em>
</td>
<td align="left" rowspan="1" colspan="1">59,063,552</td>
<td align="left" rowspan="1" colspan="1">59,065,965</td>
<td align="left" rowspan="1" colspan="1">F‐box domain</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<em>Q31</em>
</td>
<td align="left" rowspan="1" colspan="1">Biomass, spike number, and spike weight</td>
<td align="left" rowspan="1" colspan="1">
<em>TraesCS4B02G020100</em>
</td>
<td align="left" rowspan="1" colspan="1">14,265,762</td>
<td align="left" rowspan="1" colspan="1">14,268,621</td>
<td align="left" rowspan="1" colspan="1">Glycoside hydrolase family 17</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<em>Q32</em>
</td>
<td align="left" rowspan="1" colspan="1">Biomass and spike weight</td>
<td align="left" rowspan="1" colspan="1">
<em>TraesCS4B02G035400</em>
</td>
<td align="left" rowspan="1" colspan="1">25,834,624</td>
<td align="left" rowspan="1" colspan="1">25,837,112</td>
<td align="left" rowspan="1" colspan="1">Proteasome, subunit alpha/beta</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<em>Q32</em>
</td>
<td align="left" rowspan="1" colspan="1">Biomass and spike weight</td>
<td align="left" rowspan="1" colspan="1">
<em>TraesCS4B02G037300</em>
</td>
<td align="left" rowspan="1" colspan="1">27,104,920</td>
<td align="left" rowspan="1" colspan="1">27,110,487</td>
<td align="left" rowspan="1" colspan="1">BRCT domain</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<em>Q34</em>
</td>
<td align="left" rowspan="1" colspan="1">Biomass and spike weight</td>
<td align="left" rowspan="1" colspan="1">
<em>TraesCS4D02G018500</em>
</td>
<td align="left" rowspan="1" colspan="1">7,910,944</td>
<td align="left" rowspan="1" colspan="1">7,911,929</td>
<td align="left" rowspan="1" colspan="1">RNA recognition motif domain</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<em>Q35</em>
</td>
<td align="left" rowspan="1" colspan="1">Biomass</td>
<td align="left" rowspan="1" colspan="1">
<em>TraesCS5A02G000600</em>
</td>
<td align="left" rowspan="1" colspan="1">638,578</td>
<td align="left" rowspan="1" colspan="1">644,527</td>
<td align="left" rowspan="1" colspan="1">Protein kinase domain</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<em>Q36</em>
</td>
<td align="left" rowspan="1" colspan="1">Biomass and spike weight</td>
<td align="left" rowspan="1" colspan="1">
<em>TraesCS5B02G418600</em>
</td>
<td align="left" rowspan="1" colspan="1">594,867,010</td>
<td align="left" rowspan="1" colspan="1">594,872,193</td>
<td align="left" rowspan="1" colspan="1">CBS domain</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<em>Q36</em>
</td>
<td align="left" rowspan="1" colspan="1">Biomass and spike weight</td>
<td align="left" rowspan="1" colspan="1">
<em>TraesCS5B02G419300</em>
</td>
<td align="left" rowspan="1" colspan="1">594,922,735</td>
<td align="left" rowspan="1" colspan="1">594,925,178</td>
<td align="left" rowspan="1" colspan="1">Ribose 5‐phosphate isomerase, type A</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<em>Q41</em>
</td>
<td align="left" rowspan="1" colspan="1">Biomass and spike weight</td>
<td align="left" rowspan="1" colspan="1">
<em>TraesCS3D02G524400</em>
</td>
<td align="left" rowspan="1" colspan="1">604,367,637</td>
<td align="left" rowspan="1" colspan="1">604,371,395</td>
<td align="left" rowspan="1" colspan="1">Leucine‐rich repeat</td>
</tr>
<tr><td colspan="6" align="left" rowspan="1">
<strong>(B) Condition‐independent QTLs</strong>
</td></tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<em>Q42</em>
</td>
<td align="left" rowspan="1" colspan="1">Biomass</td>
<td align="left" rowspan="1" colspan="1">
<em>TraesCS4B02G264300</em>
</td>
<td align="left" rowspan="1" colspan="1">535,070,273</td>
<td align="left" rowspan="1" colspan="1">535,080,559</td>
<td align="left" rowspan="1" colspan="1">GCN5‐related N‐acetyltransferase</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<em>Q43</em>
</td>
<td align="left" rowspan="1" colspan="1">Biomass, spike number, and spike weight</td>
<td align="left" rowspan="1" colspan="1">
<em>TraesCS6A02G009200</em>
</td>
<td align="left" rowspan="1" colspan="1">3,851,157</td>
<td align="left" rowspan="1" colspan="1">3,853,368</td>
<td align="left" rowspan="1" colspan="1">Oxidoreductase FAD/NAD(P)‐binding</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<em>Q44</em>
</td>
<td align="left" rowspan="1" colspan="1">Biomass and spike weight</td>
<td align="left" rowspan="1" colspan="1">
<em>TraesCS6A02G416700</em>
</td>
<td align="left" rowspan="1" colspan="1">615,589,220</td>
<td align="left" rowspan="1" colspan="1">615,584,492</td>
<td align="left" rowspan="1" colspan="1">ATP‐dependent RNA helicase DEAD‐box, conserved site</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<em>Q46</em>
</td>
<td align="left" rowspan="1" colspan="1">Biomass</td>
<td align="left" rowspan="1" colspan="1">
<em>TraesCS7B02G432900</em>
</td>
<td align="left" rowspan="1" colspan="1">700,570,401</td>
<td align="left" rowspan="1" colspan="1">700,579,131</td>
<td align="left" rowspan="1" colspan="1">Sugar/inositol transporter</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/tpg270075-tbl-0003/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><p>Of the 17 genes analyzed for haplotypes, six were excluded due to lack of SNPs or having only one SNP. Six genes with two to four SNPs each resulted in the identification of six gene‐based haplotype blocks (Table <a href="#tpg270075-supinfo-0001" class="usa-link">S13</a>). Haplotypes from four blocks (based on genes <em>TraesCS1A02G305700</em>, <em>TraesCS2B02G599800</em>, <em>TraesCS2B02G600200</em>, and <em>TraesCS3B02G089200</em>) showed significant differences under HNT conditions, while one block (based on <em>TraesCS4B02G264300</em>) exhibited significant differences under both control and HNT conditions (<em>P</em>‐values &lt; 0.05 to &lt; 0.001). In contrast, another block showed no significant difference in either condition (Table <a href="#tpg270075-supinfo-0001" class="usa-link">S13</a>). Focusing on the haplotype block based on <em>TraesCS1A02G305700</em>, two haplotypes were identified: Hap1<em>
<sup>TraesCS1A02G305700</sup>
</em> and Hap2<em>
<sup>TraesCS1A02G305700</sup>
</em>, with frequencies of 68.2% and 31.8%, respectively. Hap1 exhibited higher biomass (68.8 g) and spike number (89.6), while Hap2 recorded lower biomass (60.8 g) and spike number (81.7) under HNT conditions (Figure <a href="#tpg270075-fig-0004" class="usa-link">4A</a>). Similarly, the <em>TraesCS4B02G264300</em> block comprised two haplotypes: Hap1<em>
<sup>TraesCS4B02G264300</sup>
</em> and Hap2<em>
<sup>TraesCS4B02G264300</sup>
</em>, with frequencies of 64% and 34.1%. Here, Hap2 demonstrated higher biomass (82.7 g under control and 75.9 g under HNT), whereas Hap1 had lower biomass (67.0 g under control and 61.0 g under HNT) (Figure <a href="#tpg270075-fig-0004" class="usa-link">4B</a>).</p>
<figure class="fig xbox font-sm" id="tpg270075-fig-0004"><h4 class="obj_head">FIGURE 4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/0b36/12256648/c82a5686526a/TPG2-18-e70075-g002.jpg" loading="lazy" id="jats-graphic-7" height="372" width="800" alt="FIGURE 4"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270075-fig-0004/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Representative gene‐based haplotypes identified in this study: (A) <em>TraesCS1A02G305700</em>, located in the <em>Q24</em> region, encodes an oxidoreductase FAD/NAD(P)‐binding protein. Two single nucleotide polymorphisms (SNPs) within its genic region were used to develop two haplotypes, whose effects on biomass and spike number under high night‐time temperature (HNT) are presented. (B) <em>TraesCS4B02G264300</em>, found in the <em>Q42</em> region, encodes a GCN5‐related N‐acetyltransferase protein. Four SNPs within its genic region were used to create two haplotypes, with their effects on biomass under both control (CNT) and HNT conditions shown. Gene structures were retrieved from the EnsemblPlants database (<a href="https://plants.ensembl.org/index.html" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://plants.ensembl.org/index.html</a>). *<em>P</em> ≤ 0.05, **<em>P</em> ≤ 0.01, *<em>P</em> ≤ 0.001.</p></figcaption></figure></section></section><section id="tpg270075-sec-0190"><h2 class="pmc_sec_title">4. DISCUSSION</h2>
<p>Adapting crop yield to climate change is crucial for food security and environmental sustainability. Understanding crop responses to environmental variations is essential for effective adaptation strategies (Lobell, <a href="#tpg270075-bib-0077" class="usa-link" aria-describedby="tpg270075-bib-0077">2014</a>; Raza et al., <a href="#tpg270075-bib-0104" class="usa-link" aria-describedby="tpg270075-bib-0104">2019</a>). Studies in controlled and field environments have shown significant wheat yield losses under HNT conditions (Hein et al., <a href="#tpg270075-bib-0043" class="usa-link" aria-describedby="tpg270075-bib-0043">2019</a>, <a href="#tpg270075-bib-0042" class="usa-link" aria-describedby="tpg270075-bib-0042">2020</a>; Impa et al., <a href="#tpg270075-bib-0047" class="usa-link" aria-describedby="tpg270075-bib-0047">2019</a>, <a href="#tpg270075-bib-0048" class="usa-link" aria-describedby="tpg270075-bib-0048">2020</a>, <a href="#tpg270075-bib-0046" class="usa-link" aria-describedby="tpg270075-bib-0046">2021</a>; Narayanan et al., <a href="#tpg270075-bib-0087" class="usa-link" aria-describedby="tpg270075-bib-0087">2015</a>; Prasad et al., <a href="#tpg270075-bib-0099" class="usa-link" aria-describedby="tpg270075-bib-0099">2008</a>). Grain yield relies on the combined effects of traits like spike and grain characteristics, and breeders are keen to identify novel genes and QTLs linked to these traits. The genetic basis of yield traits contributing to HNT tolerance in wheat remains largely unexplored, with only one recent study (Kahlon et al., <a href="#tpg270075-bib-0051" class="usa-link" aria-describedby="tpg270075-bib-0051">2024</a>) available, which was not conducted under field conditions. To address this gap, we investigated the genetics of key yield‐contributing traits, including biomass, spike number, and spike weight, under HNT stress using a diverse panel from US Great Plains hard winter wheat programs. This panel, carefully curated by breeders in the region, includes genotypes significant in production or foundational to contemporary germplasm pedigrees (Grogan et al., <a href="#tpg270075-bib-0035" class="usa-link" aria-describedby="tpg270075-bib-0035">2016</a>).</p>
<section id="tpg270075-sec-0200"><h3 class="pmc_sec_title">4.1. Winter wheat responses to HNT stress</h3>
<p>Wheat genotypes exhibited significant variation in all traits under control and HNT conditions, with HNT stress leading to marked reductions in biomass, spike number, and spike weight, consistent with previous studies (García et al., <a href="#tpg270075-bib-0031" class="usa-link" aria-describedby="tpg270075-bib-0031">2015</a>; Impa et al., <a href="#tpg270075-bib-0047" class="usa-link" aria-describedby="tpg270075-bib-0047">2019</a>; Narayanan et al., <a href="#tpg270075-bib-0087" class="usa-link" aria-describedby="tpg270075-bib-0087">2015</a>; Parveen et al., <a href="#tpg270075-bib-0091" class="usa-link" aria-describedby="tpg270075-bib-0091">2022</a>; Posch et al., <a href="#tpg270075-bib-0097" class="usa-link" aria-describedby="tpg270075-bib-0097">2022</a>; Prasad et al., <a href="#tpg270075-bib-0099" class="usa-link" aria-describedby="tpg270075-bib-0099">2008</a>). A 4.2%–13.6% reduction in biomass under HNT stress was observed, consistent with earlier reports of 2%–46% reductions under controlled and field conditions (García et al., <a href="#tpg270075-bib-0031" class="usa-link" aria-describedby="tpg270075-bib-0031">2015</a>; Impa et al., <a href="#tpg270075-bib-0047" class="usa-link" aria-describedby="tpg270075-bib-0047">2019</a>; Narayanan et al., <a href="#tpg270075-bib-0087" class="usa-link" aria-describedby="tpg270075-bib-0087">2015</a>; Parveen et al., <a href="#tpg270075-bib-0091" class="usa-link" aria-describedby="tpg270075-bib-0091">2022</a>; Posch et al., <a href="#tpg270075-bib-0097" class="usa-link" aria-describedby="tpg270075-bib-0097">2022</a>; Prasad et al., <a href="#tpg270075-bib-0099" class="usa-link" aria-describedby="tpg270075-bib-0099">2008</a>). Similarly, a 6.7%–14.4% reduction in spike number (or effective tillers per unit area) aligns with reports of reduced tillers in different crops (Bahuguna et al., <a href="#tpg270075-bib-0008" class="usa-link" aria-describedby="tpg270075-bib-0008">2022</a>; Xu et al., <a href="#tpg270075-bib-0129" class="usa-link" aria-describedby="tpg270075-bib-0129">2020</a>). These reductions likely result from increased night‐time respiration (Bahuguna et al., <a href="#tpg270075-bib-0009" class="usa-link" aria-describedby="tpg270075-bib-0009">2017</a>; Shi et al., <a href="#tpg270075-bib-0113" class="usa-link" aria-describedby="tpg270075-bib-0113">2013</a>), accelerated senescence, and disrupted carbon allocation (Impa et al., <a href="#tpg270075-bib-0047" class="usa-link" aria-describedby="tpg270075-bib-0047">2019</a>, <a href="#tpg270075-bib-0048" class="usa-link" aria-describedby="tpg270075-bib-0048">2020</a>; Sadok &amp; Jagadish, <a href="#tpg270075-bib-0106" class="usa-link" aria-describedby="tpg270075-bib-0106">2020</a>). Reduced spike weight (12.1%–13.4%) under HNT stress may be attributed to limited carbohydrate availability from increased respiration and reduced assimilate supply to grains (Impa et al., <a href="#tpg270075-bib-0046" class="usa-link" aria-describedby="tpg270075-bib-0046">2021</a>, <a href="#tpg270075-bib-0047" class="usa-link" aria-describedby="tpg270075-bib-0047">2019</a>; McAusland et al., <a href="#tpg270075-bib-0081" class="usa-link" aria-describedby="tpg270075-bib-0081">2023</a>). Warmer nights impair starch synthesis by affecting key enzymes, reducing grain size, while also accelerating chlorophyll degradation and inhibiting its biosynthesis (Abdelrahman et al., <a href="#tpg270075-bib-0001" class="usa-link" aria-describedby="tpg270075-bib-0001">2020</a>; Bahuguna et al., <a href="#tpg270075-bib-0009" class="usa-link" aria-describedby="tpg270075-bib-0009">2017</a>; Schaarschmidt et al., <a href="#tpg270075-bib-0109" class="usa-link" aria-describedby="tpg270075-bib-0109">2020</a>). These disruptions in source‐sink balance deplete reserves essential for optimal grain development under HNT stress (Impa et al., <a href="#tpg270075-bib-0047" class="usa-link" aria-describedby="tpg270075-bib-0047">2019</a>, <a href="#tpg270075-bib-0048" class="usa-link" aria-describedby="tpg270075-bib-0048">2020</a>). However, not all genotypes showed reductions; some maintained or even increased trait values, suggesting inherent genetic resilience or adaptive mechanisms. Previous studies have also reported increases in biomass, tillering, grains per spike, photosynthetic activity, and pre‐anthesis respiration in some wheat and other crop genotypes under HNT stress (Y. Fan, Qin, et al., <a href="#tpg270075-bib-0027" class="usa-link" aria-describedby="tpg270075-bib-0027">2024</a>; Jing et al., <a href="#tpg270075-bib-0050" class="usa-link" aria-describedby="tpg270075-bib-0050">2016</a>; Kanno et al., <a href="#tpg270075-bib-0052" class="usa-link" aria-describedby="tpg270075-bib-0052">2009</a>; Lazăr &amp; Lazăr, <a href="#tpg270075-bib-0065" class="usa-link" aria-describedby="tpg270075-bib-0065">2010</a>; Mohammed &amp; Tarpley, <a href="#tpg270075-bib-0086" class="usa-link" aria-describedby="tpg270075-bib-0086">2010</a>; Parveen et al., <a href="#tpg270075-bib-0091" class="usa-link" aria-describedby="tpg270075-bib-0091">2022</a>; G. Xiao et al., <a href="#tpg270075-bib-0127" class="usa-link" aria-describedby="tpg270075-bib-0127">2010</a>).</p>
<p>A positive correlation (with <em>P </em>&lt; 0.001) was observed between spike number and spike weight, but increasing spike number can reduce individual spike weight, affecting yield and quality (Guo et al., <a href="#tpg270075-bib-0037" class="usa-link" aria-describedby="tpg270075-bib-0037">2017</a>; Philipp et al., <a href="#tpg270075-bib-0095" class="usa-link" aria-describedby="tpg270075-bib-0095">2018</a>). Despite this, the total spike weight per unit area can increase if the spike number increases while the individual spike weight remains constant or changes minimally (Gaju et al., <a href="#tpg270075-bib-0030" class="usa-link" aria-describedby="tpg270075-bib-0030">2009</a>). Biomass was positively correlated with both spike number and weight (<em>P </em>&lt; 0.001). Increased spike number or weight per unit area contributes to higher biomass (Deng et al., <a href="#tpg270075-bib-0019" class="usa-link" aria-describedby="tpg270075-bib-0019">2011</a>; Ding et al., <a href="#tpg270075-bib-0021" class="usa-link" aria-describedby="tpg270075-bib-0021">2023</a>). A strong positive correlation was also found between traits under control and HNT conditions, indicating consistent genetic characteristics, though variability in stress response was observed (Table <a href="#tpg270075-supinfo-0001" class="usa-link">S4</a>).</p></section><section id="tpg270075-sec-0210"><h3 class="pmc_sec_title">4.2. Association mapping reveals novel genetic targets for enhancing wheat productivity under HNT stress</h3>
<p>A total of 19 HNT QTLs and seven condition‐independent QTLs were identified during the current study (Table <a href="#tpg270075-tbl-0002" class="usa-link">2</a>). While rice studies have identified genomic regions for different yield‐related traits under HNT stress (Bheemanahalli, Knight, et al., <a href="#tpg270075-bib-0012" class="usa-link" aria-describedby="tpg270075-bib-0012">2021</a>; A. Kumar et al., <a href="#tpg270075-bib-0058" class="usa-link" aria-describedby="tpg270075-bib-0058">2021</a>; A. Kumar et al., <a href="#tpg270075-bib-0059" class="usa-link" aria-describedby="tpg270075-bib-0059">2023</a>), only one study has focused on wheat (Kahlon et al., <a href="#tpg270075-bib-0051" class="usa-link" aria-describedby="tpg270075-bib-0051">2024</a>), highlighting a gap in research. Importantly, none of the HNT QTLs or condition‐independent QTLs identified here overlapped with those in Kahlon et al. (<a href="#tpg270075-bib-0051" class="usa-link" aria-describedby="tpg270075-bib-0051">2024</a>). However, both studies mapped QTLs on chromosomes 1B, 1D, 3B, 5D, and 7B, along with additional QTLs on other chromosomes. The lack of overlap may be attributed to several factors: (i) the previous study (Kahlon et al., <a href="#tpg270075-bib-0051" class="usa-link" aria-describedby="tpg270075-bib-0051">2024</a>) used greenhouse conditions, which generally differ from field environments in light, wind speed, and humidity, creating a distinct microclimate that can alter plant responses through strong G × E interactions (Langstroff et al., <a href="#tpg270075-bib-0064" class="usa-link" aria-describedby="tpg270075-bib-0064">2022</a>).In contrast, this study employed movable field‐based heat tents that better replicate real‐world conditions (Hein et al., <a href="#tpg270075-bib-0043" class="usa-link" aria-describedby="tpg270075-bib-0043">2019</a>, <a href="#tpg270075-bib-0042" class="usa-link" aria-describedby="tpg270075-bib-0042">2020</a>), providing more reliable insights into the genetic control of HNT tolerance traits; (ii) the previous study applied stress from planting, while our study applied it when nearly all genotypes reached 50% anthesis; (iii) the two studies utilized different plant materials and there were variations in the measured traits; (iv) interval mapping and GWAS have varying sensitivities to detect genetic variants, leading to different QTL results due to differences in statistical power and the ability to capture genetic diversity. Future research should validate these QTLs in diverse genetic backgrounds and environments for potential utilization in breeding programs.</p>
<p>In this study, QTLs such as <em>Q6</em> and <em>Q7</em> (spike weight under control), <em>Q37</em> (spike weight under HNT), and <em>Q43</em> (all traits under both conditions) clustered within the 0.63–5.36 Mb region on chromosome 6A (Figure <a href="#tpg270075-fig-0005" class="usa-link">5</a>). Some clusters were near known genes (Table <a href="#tpg270075-supinfo-0001" class="usa-link">S14</a>) with potential functional relevance, such as <em>TaDWF4‐1</em>, <em>TaDWF4‐2</em> (carbon assimilation), and <em>WPCL1‐3D</em> (flowering time) in the 603.6–614.2 Mb region on chromosome 3D (Milner et al., <a href="#tpg270075-bib-0082" class="usa-link" aria-describedby="tpg270075-bib-0082">2022</a>; Mizuno et al., <a href="#tpg270075-bib-0085" class="usa-link" aria-describedby="tpg270075-bib-0085">2016</a>). Similarly, <em>Q15</em> (biomass under control), <em>Q31</em> (all traits under HNT), and <em>Q32</em> (biomass and spike weight under HNT) were located within the 13.97–26.49 Mb region on chromosome 4B, near genes like <em>TaGRP2</em> (days to anthesis; J. Xiao et al., <a href="#tpg270075-bib-0128" class="usa-link" aria-describedby="tpg270075-bib-0128">2014</a>), <em>FLO6‐4B</em> (starch synthesis; Chia et al., <a href="#tpg270075-bib-0017" class="usa-link" aria-describedby="tpg270075-bib-0017">2020</a>), <em>TaTB1‐4B</em> (inflorescence architecture; Dixon et al., <a href="#tpg270075-bib-0022" class="usa-link" aria-describedby="tpg270075-bib-0022">2018</a>), and <em>Rht‐B1</em> (plant height; J. Peng et al., <a href="#tpg270075-bib-0093" class="usa-link" aria-describedby="tpg270075-bib-0093">1999</a>). In the homoeologous 4D region, <em>Q34</em> (biomass and spike weight under HNT) mapped close to <em>FLO6‐4D</em> (starch synthesis; Chia et al., <a href="#tpg270075-bib-0017" class="usa-link" aria-describedby="tpg270075-bib-0017">2020</a>), <em>TaTB1‐4D</em> (inflorescence architecture; Dixon et al., <a href="#tpg270075-bib-0022" class="usa-link" aria-describedby="tpg270075-bib-0022">2018</a>), and <em>Rht‐D1</em> (plant height; J. Peng et al., <a href="#tpg270075-bib-0093" class="usa-link" aria-describedby="tpg270075-bib-0093">1999</a>). <em>Q28</em> (biomass under HNT) on chromosome 2B was in close proximity to genes such as <em>TaARF12‐2B</em> (plant height, spike length, seeds per spike, and grain weight; A. Li et al., <a href="#tpg270075-bib-0068" class="usa-link" aria-describedby="tpg270075-bib-0068">2022</a>) and <em>TaSTT3b‐2B</em> (grain weight; Zhu et al., <a href="#tpg270075-bib-0138" class="usa-link" aria-describedby="tpg270075-bib-0138">2022</a>). <em>Q22</em> (spike number and spike weight under HNT) was close to <em>TaPIL1‐5D</em>, which regulates tiller number in wheat (L. Zhang, He, et al., <a href="#tpg270075-bib-0132" class="usa-link" aria-describedby="tpg270075-bib-0132">2022</a>). Additionally, <em>Q38</em> (biomass under HNT) and <em>Q39</em> (spike number under HNT) on chromosome 7A were near genes like <em>WAPO1‐7A</em> (spikelets per spike, Paraiso et al., <a href="#tpg270075-bib-0090" class="usa-link" aria-describedby="tpg270075-bib-0090">2024</a>), <em>TaCol‐B5‐7A</em> (spike architecture and grain yield; X. Zhang, Jia, et al., <a href="#tpg270075-bib-0134" class="usa-link" aria-describedby="tpg270075-bib-0134">2022</a>), and <em>Pinb2‐1</em>, <em>Pinb2‐2</em>, and <em>Pinb2‐3</em> (grain hardness and kernel texture; X. Li et al., <a href="#tpg270075-bib-0071" class="usa-link" aria-describedby="tpg270075-bib-0071">2020</a>).</p>
<figure class="fig xbox font-sm" id="tpg270075-fig-0005"><h4 class="obj_head">FIGURE 5.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/0b36/12256648/76c47174c99e/TPG2-18-e70075-g006.jpg" loading="lazy" id="jats-graphic-9" height="600" width="800" alt="FIGURE 5"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270075-fig-0005/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Phenogram plot displaying high night‐time temperature (HNT) and condition‐independent quantitative trait loci (QTLs) (distinguished using blue font and slightly enlarged text) alongside previously identified major genes that govern yield‐related traits across different wheat chromosomes. Each QTL and gene is represented by a distinct color. Horizontal bands indicate their genomic positions on chromosomes, with the start and end positions of each chromosome labeled. Rectangular boxes highlight the locations of clusters containing known genes and QTLs. Details of these known genes, including their IDs and genomic positions, can be found in Table <a href="#tpg270075-supinfo-0001" class="usa-link">S14</a>. This figure was created using Phenogram (<a href="https://visualization.ritchielab.org/phenograms/plot" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://visualization.ritchielab.org/phenograms/plot</a>).</p></figcaption></figure><p>These QTL clusters offer targets for breeding stable‐yield varieties under optimal and stress conditions, though functional studies are required to explore their roles and potential trade‐offs under HNT stress. For instance, <em>Q27</em> (biomass and spike weight under HNT) is located near the <em>Rht8</em> gene, which is associated with reduced plant height, spike length, and grain number (Chai et al., <a href="#tpg270075-bib-0139" class="usa-link" aria-describedby="tpg270075-bib-0139">2022</a>; Kowalski et al., <a href="#tpg270075-bib-0057" class="usa-link" aria-describedby="tpg270075-bib-0057">2016</a>). Consequently, the overlapping QTL may lead to a trade‐off, where increases in biomass and spike weight could come at the expense of spike length or number, ultimately impacting yield components.</p>
<p>Out of 19 HNT QTLs, seven were associated with multiple traits, and five of the seven condition‐independent QTLs showed similar associations (Table <a href="#tpg270075-tbl-0002" class="usa-link">2</a>). This suggests possible pleiotropy and shared pathways, offering breeding opportunities to improve multiple traits simultaneously (A. Li, Hao, et al., <a href="#tpg270075-bib-0068" class="usa-link" aria-describedby="tpg270075-bib-0068">2022</a>). Alternatively, these QTLs may be located in regions with linked genes, facilitating the co‐inheritance of desirable traits (Taagen et al., <a href="#tpg270075-bib-0119" class="usa-link" aria-describedby="tpg270075-bib-0119">2022</a>). The detection of condition‐independent QTLs suggests several possibilities: these QTLs may contribute to plasticity, adapting their effects based on environmental conditions and influencing traits differently under stress compared to control (Lacaze et al., <a href="#tpg270075-bib-0063" class="usa-link" aria-describedby="tpg270075-bib-0063">2009</a>; X. Li et al., <a href="#tpg270075-bib-0070" class="usa-link" aria-describedby="tpg270075-bib-0070">2018</a>). Alternatively, they could have stable effects, consistently impacting traits regardless of environmental changes (Simmonds et al., <a href="#tpg270075-bib-0117" class="usa-link" aria-describedby="tpg270075-bib-0117">2014</a>). These QTLs may also participate in adaptive mechanisms that help plants manage stress while maintaining performance under optimal conditions (Mao et al., <a href="#tpg270075-bib-0080" class="usa-link" aria-describedby="tpg270075-bib-0080">2023</a>; Pinto et al., <a href="#tpg270075-bib-0096" class="usa-link" aria-describedby="tpg270075-bib-0096">2010</a>).</p>
<p>The frequency of favorable alleles ranged from 8.81% to 94.25%, with most loci present in few genotypes. If a favorable allele is both major and prevalent, it may already be fixed, limiting further genetic improvement through traditional breeding (Khalid et al., <a href="#tpg270075-bib-0054" class="usa-link" aria-describedby="tpg270075-bib-0054">2019</a>). In contrast, minor favorable alleles offer more potential for selective breeding or genetic manipulation (Malosetti et al., <a href="#tpg270075-bib-0079" class="usa-link" aria-describedby="tpg270075-bib-0079">2021</a>) to enhance performance under HNT stress. Changes in allele frequencies at significant SNPs may indicate shifts in QTL allele frequencies due to selection, but these SNP alleles are not necessarily causative; they are assumed to be in coupling phase with the causative (unobserved) favorable QTL allele. Integrating multiple favorable loci into the same genetic background can optimize plant traits (Figure <a href="#tpg270075-fig-0002" class="usa-link">2</a>), making it an effective approach for modern wheat improvement. Genotypes with more favorable alleles generally showed better performance under HNT stress, though some inconsistencies were observed, possibly due to gene interactions (epistasis) or the influence of the overall genetic background on trait expression.</p></section><section id="tpg270075-sec-0220"><h3 class="pmc_sec_title">4.3. Wheat‐rice comparative genomics identifies novel genes for HNT tolerance in wheat</h3>
<p>Validating the causal gene for each GWAS‐identified locus is crucial, but experimentally testing all genes in these regions is impractical, necessitating the prioritization of target genes. Although several genetic and transcriptomic studies, particularly in rice, have identified and characterized numerous genes for HNT tolerance (Bheemanahalli, Knight, et al., <a href="#tpg270075-bib-0012" class="usa-link" aria-describedby="tpg270075-bib-0012">2021</a>; Desai et al., <a href="#tpg270075-bib-0020" class="usa-link" aria-describedby="tpg270075-bib-0020">2021</a>; J. Fan, Zhang, et al., <a href="#tpg270075-bib-0026" class="usa-link" aria-describedby="tpg270075-bib-0026">2024</a>; Glaubitz et al., <a href="#tpg270075-bib-0034" class="usa-link" aria-describedby="tpg270075-bib-0034">2017</a>; A. Kumar et al., <a href="#tpg270075-bib-0058" class="usa-link" aria-describedby="tpg270075-bib-0058">2021</a>; Liao et al., <a href="#tpg270075-bib-0074" class="usa-link" aria-describedby="tpg270075-bib-0074">2015</a>; Yang et al., <a href="#tpg270075-bib-0130" class="usa-link" aria-describedby="tpg270075-bib-0130">2024</a>), only limited research has been conducted in wheat. We prioritized 17 high‐confidence CGs from 766 genes through a cross‐species genomic analysis of rice genes for HNT responses, expression analysis across developmental tissues, and gene‐based haplotype development from SNP variations.</p>
<p>Among these genes, <em>TraesCS6A02G416700</em>, which encodes an ATP‐dependent RNA helicase (DEAD‐box), corresponds to the rice <em>TOGR1</em> gene (D. Wang et al., <a href="#tpg270075-bib-0123" class="usa-link" aria-describedby="tpg270075-bib-0123">2016</a>). Regulated by temperature and the circadian clock, <em>TOGR1</em>’s helicase activity is promoted by heat, which supports rRNA homeostasis and improves plant growth under high temperature (D. Wang et al., <a href="#tpg270075-bib-0123" class="usa-link" aria-describedby="tpg270075-bib-0123">2016</a>). Similarly, RNA‐binding proteins are known to play a key role in post‐transcriptional RNA metabolism during plant growth, development, and stress responses (K. Lee &amp; Kang, <a href="#tpg270075-bib-0066" class="usa-link" aria-describedby="tpg270075-bib-0066">2016</a>). <em>TraesCS4D02G018500</em>, which encodes glycine‐rich RNA‐binding proteins, is orthologous to rice genes <em>OsGRP3</em> and <em>OsGRP162</em>, respectively (Yang et al., <a href="#tpg270075-bib-0130" class="usa-link" aria-describedby="tpg270075-bib-0130">2024</a>). <em>OsGRP3</em> and <em>OsGRP162</em> are vital for night‐time thermotolerance in rice, peaking at midnight and increasing under HNT stress, regulated by <em>OsELF3‐2</em> (Yang et al., <a href="#tpg270075-bib-0130" class="usa-link" aria-describedby="tpg270075-bib-0130">2024</a>). Double mutants show reduced survival and seed‐setting rates, with increased sensitivity to night‐time heat stress and lower expression of heat‐stress‐responsive genes (Yang et al., <a href="#tpg270075-bib-0130" class="usa-link" aria-describedby="tpg270075-bib-0130">2024</a>). Similarly, <em>TraesCS4B02G264300</em> encodes a GCN5‐related N‐acetyltransferase. In rice, the corresponding gene shows differential expression under HNT stress and is associated with circadian clock components, potentially disrupting the plant's natural rhythms and physiological processes (Desai et al., <a href="#tpg270075-bib-0020" class="usa-link" aria-describedby="tpg270075-bib-0020">2021</a>; Kim et al., <a href="#tpg270075-bib-0055" class="usa-link" aria-describedby="tpg270075-bib-0055">2017</a>). <em>TraesCS4B02G035400</em> corresponds to the rice gene <em>Os03g0387100</em> (<em>TT1</em> locus), encoding the α2 subunit of the 26S proteasome. This gene is known to protect cells from heat stress by efficiently eliminating cytotoxic denatured proteins and maintaining heat‐response processes (X. M. Li et al., <a href="#tpg270075-bib-0072" class="usa-link" aria-describedby="tpg270075-bib-0072">2015</a>).</p>
<p>Additionally, glycoside hydrolases, such as those encoded by <em>TraesCS2B02G599800</em>, <em>TraesCS2B02G600200</em>, and <em>TraesCS4B02G020100</em>, which promote cell size increase, were significantly upregulated in rice panicles under HNT stress, underscoring their role in the stress response (Liao et al., <a href="#tpg270075-bib-0074" class="usa-link" aria-describedby="tpg270075-bib-0074">2015</a>; Minic &amp; Jouanin, <a href="#tpg270075-bib-0083" class="usa-link" aria-describedby="tpg270075-bib-0083">2006</a>). A rice GWAS study identified an F‐box protein‐associated SNP under HNT stress (Bheemanahalli, Knight, et al., <a href="#tpg270075-bib-0012" class="usa-link" aria-describedby="tpg270075-bib-0012">2021</a>). Similarly, our study identified <em>TraesCS3B02G089200</em>, encoding an F‐box protein, known for critical roles in abiotic stress responses (X. Zhang, Gonzalez‐Carranza, et al., <a href="#tpg270075-bib-0133" class="usa-link" aria-describedby="tpg270075-bib-0133">2019</a>). <em>TraesCS6A02G009200</em> encodes an oxidoreductase FAD/NAD(P)‐binding protein, corresponding to the <em>OJ1353_F08.4</em> gene, which was differentially expressed between HNT‐tolerant and sensitive rice genotypes (Liao et al., <a href="#tpg270075-bib-0074" class="usa-link" aria-describedby="tpg270075-bib-0074">2015</a>). This protein, crucial for nitrate reduction and redox balance, supports nitrogen metabolism and energy homeostasis, potentially enhancing wheat's resilience and yield under HNT stress. The gene <em>TraesCS7B02G432900</em>, encoding a sugar/inositol transporter, may play a role in maintaining sugar transport and osmotic balance under HNT stress. A similar transporter in rice (<em>P0567H04.25</em>) was upregulated in both HNT‐tolerant and susceptible lines, with higher expression in the sensitive genotype, suggesting that its regulation may influence stress sensitivity (Liao et al., <a href="#tpg270075-bib-0074" class="usa-link" aria-describedby="tpg270075-bib-0074">2015</a>). In wheat, this transporter could impact energy allocation and stress response, potentially contributing to differential tolerance under HNT conditions. The prioritized high‐confidence CGs are key targets for further characterization and functional validation under HNT stress in wheat.</p></section><section id="tpg270075-sec-0230"><h3 class="pmc_sec_title">4.4. Haplotype blocks as predictive tools for HNT stress adaptation in wheat</h3>
<p>Haplotype blocks with more than two SNPs in strong LD are more informative than biallelic SNPs due to their multiallelic nature (Brinton et al., <a href="#tpg270075-bib-0016" class="usa-link" aria-describedby="tpg270075-bib-0016">2020</a>). Haplotype‐assisted forward breeding, which combines superior haplotypes to develop tailor‐made varieties, has emerged as a new approach in crop improvement programs (Varshney et al., <a href="#tpg270075-bib-0121" class="usa-link" aria-describedby="tpg270075-bib-0121">2021</a>). In this study, we identified 13 haplotype blocks significantly affecting the traits, with nine associated with HNT conditions and four with both control and HNT conditions (Table <a href="#tpg270075-supinfo-0001" class="usa-link">S11</a>). Genotypes carrying superior haplotypes are believed to have higher HNT tolerance, as they maintained higher trait values under stress conditions compared to genotypes with non‐favorable haplotypes.</p>
<p>The <em>Q31</em> haplotype block, associated with all three traits, comprised three major haplotypes. Hap2, with the highest average values, was found in ∼40% of genotypes, mainly from the Southern region, indicating potential for breeding efforts to enhance its frequency in Northern genotypes. The <em>Q42</em> block, containing Hap3, which performed best under both conditions, was found in only 8% of genotypes, evenly distributed between the Northern and Southern regions. This presents a valuable opportunity to enhance its prevalence and improve yield stability under HNT stress in both regions. Additionally, gene‐based haplotypes from four blocks exhibited significant differences under HNT conditions, while one block showed notable differences under both conditions. In the <em>TraesCS1A02G305700</em> block, linked to biomass and spike number, Hap1<em>
<sup>TraesCS1A02G305700</sup>
</em> showed higher values under HNT and was present in over 50% of genotypes, mainly from the Southern region. In the <em>TraesCS4B02G264300</em> block, Hap2<em>
<sup>TraesCS4B02G264300</sup>
</em> exhibited higher biomass in ∼30% of genotypes, slightly more prevalent in the Northern region. These regional differences in haplotype prevalence may reflect adaptation to environmental conditions, with cooler Northern regions experiencing less selection pressure for HNT tolerance (Webb, <a href="#tpg270075-bib-0125" class="usa-link" aria-describedby="tpg270075-bib-0125">2022</a>). In contrast, warmer Southern regions, characterized by warmer days and nights, may have exerted greater selection pressure for HNT tolerance (Webb, <a href="#tpg270075-bib-0125" class="usa-link" aria-describedby="tpg270075-bib-0125">2022</a>). These adaptations may have been shaped by underlying genetic variation (Haasl &amp; Payseur, <a href="#tpg270075-bib-0039" class="usa-link" aria-describedby="tpg270075-bib-0039">2016</a>; Sertse et al., <a href="#tpg270075-bib-0111" class="usa-link" aria-describedby="tpg270075-bib-0111">2023</a>). Haplotype‐based genomic selection offers greater information than biallelic SNPs and can improve prediction accuracy and selection gains by capturing complex genotype‐phenotype relationships more effectively (Weber et al., <a href="#tpg270075-bib-0126" class="usa-link" aria-describedby="tpg270075-bib-0126">2023</a>).</p>
<p>Further, to validate the identified haplotypes, we selected the top 20 contrasting genotypes based on STI scores and phenotypic values. Genotypes common across all three traits in each group were then analyzed for haplotype distribution from blocks with significant trait differences. HNT‐sensitive genotypes often carried inferior haplotypes, while tolerant genotypes predominantly had superior ones (Figure <a href="#tpg270075-fig-0006" class="usa-link">6A</a>). This analysis validated the haplotypes and identified key genotypes: sensitive genotypes included Arlin, Overley, Shocker, TX01M5009‐28, TX03A0148, and TX99A0153‐1, while tolerant genotypes included Arapahoe, Century, Crimson, Genou, and Rose. Sensitive genotypes consistently exhibited lower trait values, while tolerant genotypes consistently displayed significantly higher trait values across all three traits (Figure <a href="#tpg270075-fig-0006" class="usa-link">6B</a>). HNT‐tolerant genotypes can serve as valuable donors in developing HNT‐tolerant wheat varieties. By crossing these tolerant genotypes with HNT‐sensitive ones, researchers can also develop trait‐focused biparental populations to facilitate more effective trait mapping. Furthermore, these contrasting sets of genotypes can be utilized for further omics studies and functional validation of key QTLs and CGs.</p>
<figure class="fig xbox font-sm" id="tpg270075-fig-0006"><h4 class="obj_head">FIGURE 6.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/0b36/12256648/a069b8a74f8e/TPG2-18-e70075-g001.jpg" loading="lazy" id="jats-graphic-11" height="238" width="800" alt="FIGURE 6"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270075-fig-0006/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Figure illustrating the analysis of haplotype distribution among contrasting genotypes. (A) Quantitative trait locus (QTL)‐based and gene‐based haplotypes are classified as superior, intermediate, or inferior based on their effects on traits, represented by green, light blue, and dark orange font colors, respectively. Sensitive genotypes include Arlin, Overley, Shocker, TX01M5009‐28, TX03A0148, and TX99A0153‐1, while tolerant genotypes consist of Arapahoe, Century, Crimson, Genou, and Rose. (B) Trait values of susceptible and tolerant wheat genotypes. Susceptible genotypes exhibited lower trait values across all three traits, whereas tolerant genotypes consistently showed higher trait values.</p></figcaption></figure></section></section><section id="tpg270075-sec-0240"><h2 class="pmc_sec_title">5. CONCLUSIONS</h2>
<p>HNT stress significantly reduced wheat biomass, spike number, and spike weight. However, some genotypes exhibited genetic resilience that enabled them to maintain or improve yield‐related traits under HNT stress, likely due to adaptive physiological and molecular mechanisms. Significant genetic variation and strong correlations between these traits suggest they can be jointly selected to improve yield under HNT stress. This study is the first of its kind in wheat to dissect the genetic control of traits for HNT tolerance under field conditions, identifying key QTLs, superior QTL‐based and gene‐based haplotypes, and high‐confidence CGs linked to yield‐related traits under HNT stress. Further validation and functional studies of the identified genomic regions could pave the way for their use in breeding programs to improve HNT tolerance in wheat. Developing kompetitive allele‐specific PCR assays for candidate SNPs can streamline their use in early‐stage breeding through marker‐assisted selection. Superior haplotypes can enhance breeding for HNT stress tolerance, and genomic data from this study can improve genomic prediction models for selecting resilient wheat varieties. Overall, this study advances understanding of HNT tolerance in wheat and guides future research and breeding for climate resilience.</p></section><section id="tpg270075-sec-0250"><h2 class="pmc_sec_title">AUTHOR CONTRIBUTIONS</h2>
<p>
<strong>Dinesh Kumar Saini</strong>: Data curation; formal analysis; software; visualization; writing—original draft; writing—review and editing. <strong>Nathan T. Hein</strong>: Conceptualization; data curation; methodology. <strong>Impa Somayanda</strong>: Conceptualization; data curation; methodology. <strong>Raju Bheemanahalli</strong>: Conceptualization; data curation; methodology. <strong>Troy Ostmeyer</strong>: Conceptualization; data curation; methodology. <strong>S. V. Krishna Jagadish</strong>: Conceptualization; project administration; resources; supervision; writing—review and editing.</p></section><section id="tpg270075-sec-0270"><h2 class="pmc_sec_title">CONFLICT OF INTEREST STATEMENT</h2>
<p>The authors declare no conflicts of interest.</p></section><section id="sec27"><h2 class="pmc_sec_title">Supporting information</h2>
<section class="sm xbox font-sm" id="tpg270075-supinfo-0001"></section><section class="sm xbox font-sm" id="tpg270075-supitem-0001"><div class="caption p">
<p>
<strong>Figure S1</strong> Bar plot showing year‐wise reductions in trait values under HNT stress for biomass, spike number, and spike weight in wheat during the 2018–19 and 2019–20 seasons. Solid‐filled bars represent control conditions, while pattern‐filled bars indicate HNT stress. Percent reductions under HNT stress are annotated above each pair of bars. Trait values are averages across the genotypes, and percentage reductions were calculated using the formula: (Average trait value under control – Average trait value under HNT stress) / Average trait value under control × 100.</p>
<p>
<strong>Figure S2</strong> (A) 3D scatter plot of 261 wheat genotypes based on the first three principal components (PC1, PC2, and PC3). The axes represent PC1 (5.26% variance explained), PC2 (4.22% variance explained), and PC3 (3.47% variance explained). Each dot represents an accession. (B) Evanno plot of the delta‐K statistic revealing the number of sub‐populations in the association panel.</p>
<p>
<strong>Figure S3</strong> Expression patterns of high‐confidence genes across developmental stages, with values shown in TPM (Transcripts Per Million). Data was obtained from the WheatOmics database (<a href="http://wheatomics.sdau.edu.cn/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://wheatomics.sdau.edu.cn/</a>). The heatmap was generated using TBtools (<a href="https://github.com/CJ-Chen/TBtools-II/releases" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/CJ‐Chen/TBtools‐II/releases</a>). Z10: First leaf through coleoptile; Z13: Three leaves emerged; Z23: Main stem with three tillers; Z30: Pseudostem (youngest leaf sheath erect) Z32: Second node detectable; Z39: Flag leaf ligule just visible; Z65: Anthesis halfway (anthers halfway between tip and base of ear); Z71: Kernel water ripe (no starch); Z75: Medium milk; Z85: Soft dough. Further details on the Zadoks growth scale are available at <a href="https://www.agric.wa.gov.au/grains/zadoks-growth-scale" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.agric.wa.gov.au/grains/zadoks‐growth‐scale</a>.</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12256648/bin/TPG2-18-e70075-s001.docx" data-ga-action="click_feat_suppl" class="usa-link">TPG2-18-e70075-s001.docx</a><sup> (730.8KB, docx) </sup>
</div></div></section><section class="sm xbox font-sm" id="tpg270075-supitem-0002"><div class="caption p">
<p>
<strong>Table S1</strong> Best linear unbiased estimates for trait values across control and high night temperature environments.</p>
<p>
<strong>Table S2</strong> Combined ANOVA showing mean squares for main and interaction sources of variation for different traits recorded under control and HNT conditions.</p>
<p>
<strong>Table S3</strong> Descriptive statistics, coefficient of variations (CV%), and heritability estimates of different traits across the environments.</p>
<p>
<strong>Table S4</strong> Correlation patterns observed among different traits within and across environments.</p>
<p>
<strong>Table S5</strong> Stress tolerance indices for different genotypes in the panel.</p>
<p>
<strong>Table S6</strong> Number of SNPs available on different wheat chromosomes.</p>
<p>
<strong>Table S7</strong> Details on significant SNPs associated with various yield‐related traits in different environments.</p>
<p>
<strong>Table S8</strong> Details on significant SNPs associated with multiple traits (within the individual environments) identified in the present study.</p>
<p>
<strong>Table S9</strong> Details on QTLs associated with different traits under control conditions.</p>
<p>
<strong>Table S10</strong> Patterns of favorable and non‐favorable alleles at various loci and their differential impact on trait values.</p>
<p>
<strong>Table S11</strong> Haplotypes associated with different traits identified during the present study.</p>
<p>
<strong>Table S12</strong> Gene models available from different QTL regions.</p>
<p>
<strong>Table S13</strong> Haplotype analysis of key candidate genes prioritized from various QTL regions identified in the present study.</p>
<p>
<strong>Table S14</strong> Major known genes regulating yield‐related traits in wheat.</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12256648/bin/TPG2-18-e70075-s002.xlsx" data-ga-action="click_feat_suppl" class="usa-link">TPG2-18-e70075-s002.xlsx</a><sup> (224.1KB, xlsx) </sup>
</div></div></section></section><section id="tpg270075-sec-0260" class="ack"><h2 class="pmc_sec_title">ACKNOWLEDGMENTS</h2>
<p>The authors thank Dan Wagner, Amaranatha Vennapusa, Carlos Bustamante, Meghnath Pokharel, Anuj Chiluwal, Jianming Fu, and Mitchell Neilsen for their assistance during the trial and high‐night temperature stress imposition. The Thornton Distinguished Fund is gratefully acknowledged for supporting Dr. Jagadish's program at Texas Tech University.</p></section><section id="notes1"><p>


Kumar Saini, D.
, 
Hein, N. T.
, 
Somayanda, I.
, 
Bheemanahalli, R.
, 
Ostmeyer, T.
, &amp; 
Jagadish, S. V. K.
 (2025). Genetic mapping and haplotype analysis identify novel candidate genes for high night temperature tolerance in winter wheat. The Plant Genome, 18, e70075. 10.1002/tpg2.70075


</p>
<section id="fn-group1" class="fn-group"><div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="tpg270075-note-0001"><p>Assigned to Associate Editor Devinder Sandhu.</p></div></div></section></section><section id="_ci93_" lang="en" class="contrib-info"><h2 class="pmc_sec_title">Contributor Information</h2>
<p>Dinesh Kumar Saini, Email: disaini@ttu.edu.</p>
<p>S. V. Krishna Jagadish, Email: kjagadish.sv@ttu.edu.</p></section><section id="tpg270075-sec-0290"><h2 class="pmc_sec_title">DATA AVAILABILITY STATEMENT</h2>
<p>Additional supporting information can be found online in the Supporting Information section at the end of this article.</p></section><section id="tpg270075-bibl-0001" class="ref-list"><h2 class="pmc_sec_title">REFERENCES</h2>
<section id="tpg270075-bibl-0001_sec2"><ol class="ref-list font-sm">
<li id="tpg270075-bib-0001">
<cite>

Abdelrahman, M.
, 
Burritt, D. J.
, 
Gupta, A.
, 
Tsujimoto, H.
, &amp; 
Tran, L. S. P.
 (2020). Heat stress effects on source–sink relationships and metabolome dynamics in wheat. Journal of Experimental Botany, 71(2), 543–554. 10.1093/jxb/erz296

</cite> [<a href="https://doi.org/10.1093/jxb/erz296" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31232445/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Experimental%20Botany&amp;title=Heat%20stress%20effects%20on%20source%E2%80%93sink%20relationships%20and%20metabolome%20dynamics%20in%20wheat&amp;author=M.%20Abdelrahman&amp;author=D.%20J.%20Burritt&amp;author=A.%20Gupta&amp;author=H.%20Tsujimoto&amp;author=L.%20S.%20P.%20Tran&amp;volume=71&amp;issue=2&amp;publication_year=2020&amp;pages=543-554&amp;pmid=31232445&amp;doi=10.1093/jxb/erz296&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270075-bib-0002">
<cite>

Aiqing, S.
, 
Somayanda, I.
, 
Sebastian, S. V.
, 
Singh, K.
, 
Gill, K.
, 
Prasad, P. V. V.
, &amp; 
Jagadish, S. K.
 (2018). Heat stress during flowering affects time of day of flowering, seed set, and grain quality in spring wheat. Crop Science, 58(1), 380–392. 10.2135/cropsci2017.04.0221
</cite> [<a href="https://doi.org/10.2135/cropsci2017.04.0221" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Crop%20Science&amp;title=Heat%20stress%20during%20flowering%20affects%20time%20of%20day%20of%20flowering,%20seed%20set,%20and%20grain%20quality%20in%20spring%20wheat&amp;author=S.%20Aiqing&amp;author=I.%20Somayanda&amp;author=S.%20V.%20Sebastian&amp;author=K.%20Singh&amp;author=K.%20Gill&amp;volume=58&amp;issue=1&amp;publication_year=2018&amp;pages=380-392&amp;doi=10.2135/cropsci2017.04.0221&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270075-bib-0003">
<cite>

Alexander, L. V.
, 
Hope, P.
, 
Collins, D.
, 
Trewin, B.
, 
Lynch, A.
, &amp; 
Nicholls, N.
 (2007). Trends in Australia's climate means and extremes: A global context. Australian Meteorological Magazine, 56(1), 1–18. 10.1029/2005JD006119Alexander
</cite> [<a href="https://doi.org/10.1029/2005JD006119Alexander" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Australian%20Meteorological%20Magazine&amp;title=Trends%20in%20Australia's%20climate%20means%20and%20extremes:%20A%20global%20context&amp;author=L.%20V.%20Alexander&amp;author=P.%20Hope&amp;author=D.%20Collins&amp;author=B.%20Trewin&amp;author=A.%20Lynch&amp;volume=56&amp;issue=1&amp;publication_year=2007&amp;pages=1-18&amp;doi=10.1029/2005JD006119Alexander&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270075-bib-0004">
<cite>

Alexander, L. V.
, 
Zhang, X.
, 
Peterson, T. C.
, 
Caesar, J.
, 
Gleason, B.
, 
Klein Tank, A. M. G.
, 
Haylock, M.
, 
Collins, D.
, 
Trewin, B.
, 
Rahimzadeh, F.
, &amp; 
Tagipour, A.
 (2006). Global observed changes in daily climate extremes of temperature and precipitation. Journal of Geophysical Research: Atmospheres, 111(D5). 10.1029/2005JD006290
</cite> [<a href="https://doi.org/10.1029/2005JD006290" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Geophysical%20Research:%20Atmospheres&amp;title=Global%20observed%20changes%20in%20daily%20climate%20extremes%20of%20temperature%20and%20precipitation&amp;author=L.%20V.%20Alexander&amp;author=X.%20Zhang&amp;author=T.%20C.%20Peterson&amp;author=J.%20Caesar&amp;author=B.%20Gleason&amp;volume=111&amp;issue=D5&amp;publication_year=2006&amp;doi=10.1029/2005JD006290&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270075-bib-0005">
<cite>

AlTameemi, R.
, 
Gill, H. S.
, 
Ali, S.
, 
Ayana, G.
, 
Halder, J.
, 
Sidhu, J. S.
, 
Gill, U. S.
, 
Turnipseed, B.
, 
Hernandez, J. L. G.
, &amp; 
Sehgal, S. K.
 (2021). Genome‐wide association analysis permits characterization of Stagonospora nodorum blotch (SNB) resistance in hard winter wheat. Scientific Reports, 11(1), Article 12570. 10.1038/s41598-021-91515-6

</cite> [<a href="https://doi.org/10.1038/s41598-021-91515-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8206080/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34131169/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Scientific%20Reports&amp;title=Genome%E2%80%90wide%20association%20analysis%20permits%20characterization%20of%20Stagonospora%20nodorum%20blotch%20(SNB)%20resistance%20in%20hard%20winter%20wheat&amp;author=R.%20AlTameemi&amp;author=H.%20S.%20Gill&amp;author=S.%20Ali&amp;author=G.%20Ayana&amp;author=J.%20Halder&amp;volume=11&amp;issue=1&amp;publication_year=2021&amp;pages=12570&amp;pmid=34131169&amp;doi=10.1038/s41598-021-91515-6&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270075-bib-0006">
<cite>

Alvarado, G.
, 
Rodríguez, F. M.
, 
Pacheco, A.
, 
Burgueño, J.
, 
Crossa, J.
, 
Vargas, M.
, 
Pérez‐Rodríguez, P.
, &amp; 
Lopez‐Cruz, M. A.
 (2020). META‐R: A software to analyze data from multi‐environment plant breeding trials. The Crop Journal, 8(5), 745–756. 10.1016/j.cj.2020.03.010
</cite> [<a href="https://doi.org/10.1016/j.cj.2020.03.010" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=The%20Crop%20Journal&amp;title=META%E2%80%90R:%20A%20software%20to%20analyze%20data%20from%20multi%E2%80%90environment%20plant%20breeding%20trials&amp;author=G.%20Alvarado&amp;author=F.%20M.%20Rodr%C3%ADguez&amp;author=A.%20Pacheco&amp;author=J.%20Burgue%C3%B1o&amp;author=J.%20Crossa&amp;volume=8&amp;issue=5&amp;publication_year=2020&amp;pages=745-756&amp;doi=10.1016/j.cj.2020.03.010&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270075-bib-0008">
<cite>

Bahuguna, R. N.
, 
Chaturvedi, A. K.
, 
Pal, M.
, 
Viswanathan, C.
, 
Jagadish, S. K.
, &amp; 
Pareek, A.
 (2022). Carbon dioxide responsiveness mitigates rice yield loss under high night temperature. Plant Physiology, 188(1), 285–300. 10.1093/plphys/kiab470

</cite> [<a href="https://doi.org/10.1093/plphys/kiab470" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8774858/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34643728/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Physiology&amp;title=Carbon%20dioxide%20responsiveness%20mitigates%20rice%20yield%20loss%20under%20high%20night%20temperature&amp;author=R.%20N.%20Bahuguna&amp;author=A.%20K.%20Chaturvedi&amp;author=M.%20Pal&amp;author=C.%20Viswanathan&amp;author=S.%20K.%20Jagadish&amp;volume=188&amp;issue=1&amp;publication_year=2022&amp;pages=285-300&amp;pmid=34643728&amp;doi=10.1093/plphys/kiab470&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270075-bib-0009">
<cite>

Bahuguna, R. N.
, 
Solis, C. A.
, 
Shi, W.
, &amp; 
Jagadish, K. S.
 (2017). Post‐flowering night respiration and altered sink activity account for high night temperature‐induced grain yield and quality loss in rice (<em>Oryza sativa</em> L.). Physiologia Plantarum, 159(1), 59–73. 10.1111/ppl.12485

</cite> [<a href="https://doi.org/10.1111/ppl.12485" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27513992/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Physiologia%20Plantarum&amp;title=Post%E2%80%90flowering%20night%20respiration%20and%20altered%20sink%20activity%20account%20for%20high%20night%20temperature%E2%80%90induced%20grain%20yield%20and%20quality%20loss%20in%20rice%20(Oryza%20sativa%20L.)&amp;author=R.%20N.%20Bahuguna&amp;author=C.%20A.%20Solis&amp;author=W.%20Shi&amp;author=K.%20S.%20Jagadish&amp;volume=159&amp;issue=1&amp;publication_year=2017&amp;pages=59-73&amp;pmid=27513992&amp;doi=10.1111/ppl.12485&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270075-bib-0010">
<cite>

Barrett, J. C.
, 
Fry, B.
, 
Maller, J. D. M. J.
, &amp; 
Daly, M. J.
 (2005). Haploview: Analysis and visualization of LD and haplotype maps. Bioinformatics, 21(2), 263–265. 10.1093/bioinformatics/bth457

</cite> [<a href="https://doi.org/10.1093/bioinformatics/bth457" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15297300/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Haploview:%20Analysis%20and%20visualization%20of%20LD%20and%20haplotype%20maps&amp;author=J.%20C.%20Barrett&amp;author=B.%20Fry&amp;author=J.%20D.%20M.%20J.%20Maller&amp;author=M.%20J.%20Daly&amp;volume=21&amp;issue=2&amp;publication_year=2005&amp;pages=263-265&amp;pmid=15297300&amp;doi=10.1093/bioinformatics/bth457&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270075-bib-0011">
<cite>

Bergkamp, B.
, 
Impa, S. M.
, 
Asebedo, A. R.
, 
Fritz, A. K.
, &amp; 
Jagadish, S. K.
 (2018). Prominent winter wheat varieties response to post‐flowering heat stress under controlled chambers and field based heat tents. Field Crops Research, 222, 143–152. 10.1016/j.fcr.2018.03.009
</cite> [<a href="https://doi.org/10.1016/j.fcr.2018.03.009" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Field%20Crops%20Research&amp;title=Prominent%20winter%20wheat%20varieties%20response%20to%20post%E2%80%90flowering%20heat%20stress%20under%20controlled%20chambers%20and%20field%20based%20heat%20tents&amp;author=B.%20Bergkamp&amp;author=S.%20M.%20Impa&amp;author=A.%20R.%20Asebedo&amp;author=A.%20K.%20Fritz&amp;author=S.%20K.%20Jagadish&amp;volume=222&amp;publication_year=2018&amp;pages=143-152&amp;doi=10.1016/j.fcr.2018.03.009&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270075-bib-0012">
<cite>

Bheemanahalli, R.
, 
Knight, M.
, 
Quinones, C.
, 
Doherty, C. J.
, &amp; 
Jagadish, S. K.
 (2021). Genome‐wide association study and gene network analyses reveal potential candidate genes for high night temperature tolerance in rice. Scientific Reports, 11(1), Article 6747. 10.1038/s41598-021-85921-z

</cite> [<a href="https://doi.org/10.1038/s41598-021-85921-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7991035/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33762605/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Scientific%20Reports&amp;title=Genome%E2%80%90wide%20association%20study%20and%20gene%20network%20analyses%20reveal%20potential%20candidate%20genes%20for%20high%20night%20temperature%20tolerance%20in%20rice&amp;author=R.%20Bheemanahalli&amp;author=M.%20Knight&amp;author=C.%20Quinones&amp;author=C.%20J.%20Doherty&amp;author=S.%20K.%20Jagadish&amp;volume=11&amp;issue=1&amp;publication_year=2021&amp;pages=6747&amp;pmid=33762605&amp;doi=10.1038/s41598-021-85921-z&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270075-bib-0013">
<cite>

Bheemanahalli, R.
, 
Sunoj, V. J.
, 
Saripalli, G.
, 
Prasad, P. V.
, 
Balyan, H. S.
, 
Gupta, P. K.
, 
Grant, N.
, 
Gill, K. S.
, &amp; 
Jagadish, S. K.
 (2019). Quantifying the impact of heat stress on pollen germination, seed set, and grain filling in spring wheat. Crop Science, 59(2), 684–696. 10.2135/cropsci2018.05.0292
</cite> [<a href="https://doi.org/10.2135/cropsci2018.05.0292" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Crop%20Science&amp;title=Quantifying%20the%20impact%20of%20heat%20stress%20on%20pollen%20germination,%20seed%20set,%20and%20grain%20filling%20in%20spring%20wheat&amp;author=R.%20Bheemanahalli&amp;author=V.%20J.%20Sunoj&amp;author=G.%20Saripalli&amp;author=P.%20V.%20Prasad&amp;author=H.%20S.%20Balyan&amp;volume=59&amp;issue=2&amp;publication_year=2019&amp;pages=684-696&amp;doi=10.2135/cropsci2018.05.0292&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270075-bib-0014">
<cite>

Bheemanahalli, R.
, 
Wang, C.
, 
Bashir, E.
, 
Chiluwal, A.
, 
Pokharel, M.
, 
Perumal, R.
, 
Moghimi, N.
, 
Ostmeyer, T.
, 
Caragea, D.
, &amp; 
Jagadish, S. K.
 (2021). Classical phenotyping and deep learning concur on genetic control of stomatal density and area in sorghum. Plant Physiology, 186(3), 1562–1579. 10.1093/plphys/kiab174

</cite> [<a href="https://doi.org/10.1093/plphys/kiab174" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8260133/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33856488/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Physiology&amp;title=Classical%20phenotyping%20and%20deep%20learning%20concur%20on%20genetic%20control%20of%20stomatal%20density%20and%20area%20in%20sorghum&amp;author=R.%20Bheemanahalli&amp;author=C.%20Wang&amp;author=E.%20Bashir&amp;author=A.%20Chiluwal&amp;author=M.%20Pokharel&amp;volume=186&amp;issue=3&amp;publication_year=2021&amp;pages=1562-1579&amp;pmid=33856488&amp;doi=10.1093/plphys/kiab174&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270075-bib-0015">
<cite>

Bradbury, P. J.
, 
Zhang, Z.
, 
Kroon, D. E.
, 
Casstevens, T. M.
, 
Ramdoss, Y.
, &amp; 
Buckler, E. S.
 (2007). TASSEL: Software for association mapping of complex traits in diverse samples. Bioinformatics, 23(19), 2633–2635. 10.1093/bioinformatics/btm308

</cite> [<a href="https://doi.org/10.1093/bioinformatics/btm308" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17586829/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=TASSEL:%20Software%20for%20association%20mapping%20of%20complex%20traits%20in%20diverse%20samples&amp;author=P.%20J.%20Bradbury&amp;author=Z.%20Zhang&amp;author=D.%20E.%20Kroon&amp;author=T.%20M.%20Casstevens&amp;author=Y.%20Ramdoss&amp;volume=23&amp;issue=19&amp;publication_year=2007&amp;pages=2633-2635&amp;pmid=17586829&amp;doi=10.1093/bioinformatics/btm308&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270075-bib-0016">
<cite>

Brinton, J.
, 
Ramirez‐Gonzalez, R. H.
, 
Simmonds, J.
, 
Wingen, L.
, 
Orford, S.
, 
Griffiths, S.
, 
Haberer, G.
, 
Spannagl, M.
, 
Walkowiak, S.
, 
Pozniak, C.
, &amp; 10 Wheat Genome Project
. (2020). A haplotype‐led approach to increase the precision of wheat breeding. Communications Biology, 3(1), Article 712. 10.1038/s42003-020-01413-2

</cite> [<a href="https://doi.org/10.1038/s42003-020-01413-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7689427/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33239669/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Communications%20Biology&amp;title=A%20haplotype%E2%80%90led%20approach%20to%20increase%20the%20precision%20of%20wheat%20breeding&amp;author=J.%20Brinton&amp;author=R.%20H.%20Ramirez%E2%80%90Gonzalez&amp;author=J.%20Simmonds&amp;author=L.%20Wingen&amp;author=S.%20Orford&amp;volume=3&amp;issue=1&amp;publication_year=2020&amp;pages=712&amp;pmid=33239669&amp;doi=10.1038/s42003-020-01413-2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270075-bib-0139">
<cite>

Chai, L.
, 
Xin, M.
, 
Dong, C.
, 
Chen, Z.
, 
Zhai, H.
, 
Zhuang, J.
, 
Cheng, X.
, 
Wang, N.
, 
Geng, J.
, 
Wang, X.
, &amp; 
Bian, R.
 (2022). A natural variation in Ribonuclease H‐like gene underlies Rht8 to confer “Green Revolution” trait in wheat. Molecular Plant, 15(3), 377–380. 10.1016/j.molp.2022.01.013

</cite> [<a href="https://doi.org/10.1016/j.molp.2022.01.013" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35063659/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Plant&amp;title=A%20natural%20variation%20in%20Ribonuclease%20H%E2%80%90like%20gene%20underlies%20Rht8%20to%20confer%20%E2%80%9CGreen%20Revolution%E2%80%9D%20trait%20in%20wheat&amp;author=L.%20Chai&amp;author=M.%20Xin&amp;author=C.%20Dong&amp;author=Z.%20Chen&amp;author=H.%20Zhai&amp;volume=15&amp;issue=3&amp;publication_year=2022&amp;pages=377-380&amp;pmid=35063659&amp;doi=10.1016/j.molp.2022.01.013&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270075-bib-0017">
<cite>

Chia, T.
, 
Chirico, M.
, 
King, R.
, 
Ramirez‐Gonzalez, R.
, 
Saccomanno, B.
, 
Seung, D.
, 
Simmonds, J.
, 
Trick, M.
, 
Uauy, C.
, 
Verhoeven, T.
, &amp; 
Trafford, K.
 (2020). A carbohydrate‐binding protein, B‐GRANULE CONTENT 1, influences starch granule size distribution in a dose‐dependent manner in polyploid wheat. Journal of Experimental Botany, 71(1), 105–115. 10.1093/jxb/erz405

</cite> [<a href="https://doi.org/10.1093/jxb/erz405" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31633795/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Experimental%20Botany&amp;title=A%20carbohydrate%E2%80%90binding%20protein,%20B%E2%80%90GRANULE%20CONTENT%201,%20influences%20starch%20granule%20size%20distribution%20in%20a%20dose%E2%80%90dependent%20manner%20in%20polyploid%20wheat&amp;author=T.%20Chia&amp;author=M.%20Chirico&amp;author=R.%20King&amp;author=R.%20Ramirez%E2%80%90Gonzalez&amp;author=B.%20Saccomanno&amp;volume=71&amp;issue=1&amp;publication_year=2020&amp;pages=105-115&amp;pmid=31633795&amp;doi=10.1093/jxb/erz405&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270075-bib-0018">
<cite>

Delacre, M.
, 
Leys, C.
, 
Mora, Y. L.
, &amp; 
Lakens, D.
 (2019). Taking parametric assumptions seriously: Arguments for the use of Welch's F‐test instead of the classical F‐test in one‐way ANOVA. International Review of Social Psychology, 32(1), 13. 10.5334/irsp.198
</cite> [<a href="https://doi.org/10.5334/irsp.198" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=International%20Review%20of%20Social%20Psychology&amp;title=Taking%20parametric%20assumptions%20seriously:%20Arguments%20for%20the%20use%20of%20Welch's%20F%E2%80%90test%20instead%20of%20the%20classical%20F%E2%80%90test%20in%20one%E2%80%90way%20ANOVA&amp;author=M.%20Delacre&amp;author=C.%20Leys&amp;author=Y.%20L.%20Mora&amp;author=D.%20Lakens&amp;volume=32&amp;issue=1&amp;publication_year=2019&amp;pages=13&amp;doi=10.5334/irsp.198&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270075-bib-0019">
<cite>

Deng, S.
, 
Wu, X.
, 
Wu, Y.
, 
Zhou, R.
, 
Wang, H.
, 
Jia, J.
, &amp; 
Liu, S.
 (2011). Characterization and precise mapping of a QTL increasing spike number with pleiotropic effects in wheat. Theoretical and Applied Genetics, 122, 281–289. 10.1007/s00122-010-1443-1

</cite> [<a href="https://doi.org/10.1007/s00122-010-1443-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20872211/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Theoretical%20and%20Applied%20Genetics&amp;title=Characterization%20and%20precise%20mapping%20of%20a%20QTL%20increasing%20spike%20number%20with%20pleiotropic%20effects%20in%20wheat&amp;author=S.%20Deng&amp;author=X.%20Wu&amp;author=Y.%20Wu&amp;author=R.%20Zhou&amp;author=H.%20Wang&amp;volume=122&amp;publication_year=2011&amp;pages=281-289&amp;pmid=20872211&amp;doi=10.1007/s00122-010-1443-1&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270075-bib-0020">
<cite>

Desai, J. S.
, 
Lawas, L. M. F.
, 
Valente, A. M.
, 
Leman, A. R.
, 
Grinevich, D. O.
, 
Jagadish, S. K.
, &amp; 
Doherty, C. J.
 (2021). Warm nights disrupt transcriptome rhythms in field‐grown rice panicles. Proceedings of the National Academy of Sciences, 118(25), e2025899118. 10.1073/pnas.2025899118
</cite> [<a href="https://doi.org/10.1073/pnas.2025899118" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8237568/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34155145/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proceedings%20of%20the%20National%20Academy%20of%20Sciences&amp;title=Warm%20nights%20disrupt%20transcriptome%20rhythms%20in%20field%E2%80%90grown%20rice%20panicles&amp;author=J.%20S.%20Desai&amp;author=L.%20M.%20F.%20Lawas&amp;author=A.%20M.%20Valente&amp;author=A.%20R.%20Leman&amp;author=D.%20O.%20Grinevich&amp;volume=118&amp;issue=25&amp;publication_year=2021&amp;pages=e2025899118&amp;pmid=34155145&amp;doi=10.1073/pnas.2025899118&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270075-bib-0021">
<cite>

Ding, Y. G.
, 
Zhang, X. B.
, 
Quan, M. A.
, 
Li, F. J.
, 
Tao, R. R.
, 
Min, Z. H. U.
, 
Li, C. Y.
, 
Zhu, X. K.
, 
Guo, W. S.
, &amp; 
Ding, J. F.
 (2023). Tiller fertility is critical for improving grain yield, photosynthesis, and nitrogen efficiency in wheat. Journal of Integrative Agriculture, 22(7), 2054–2066. 10.1016/j.jia.2022.10.005
</cite> [<a href="https://doi.org/10.1016/j.jia.2022.10.005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Integrative%20Agriculture&amp;title=Tiller%20fertility%20is%20critical%20for%20improving%20grain%20yield,%20photosynthesis,%20and%20nitrogen%20efficiency%20in%20wheat&amp;author=Y.%20G.%20Ding&amp;author=X.%20B.%20Zhang&amp;author=M.%20A.%20Quan&amp;author=F.%20J.%20Li&amp;author=R.%20R.%20Tao&amp;volume=22&amp;issue=7&amp;publication_year=2023&amp;pages=2054-2066&amp;doi=10.1016/j.jia.2022.10.005&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270075-bib-0022">
<cite>

Dixon, L. E.
, 
Greenwood, J. R.
, 
Bencivenga, S.
, 
Zhang, P.
, 
Cockram, J.
, 
Mellers, G.
, 
Ramm, K.
, 
Cavanagh, C.
, 
Swain, S. M.
, &amp; 
Boden, S. A.
 (2018). 
<em>TEOSINTE BRANCHED1</em> regulates inflorescence architecture and development in bread wheat (<em>Triticum aestivum</em>). The Plant Cell, 30(3), 563–581. 10.1105/tpc.17.00961

</cite> [<a href="https://doi.org/10.1105/tpc.17.00961" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5894836/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29444813/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=The%20Plant%20Cell&amp;title=TEOSINTE%20BRANCHED1%20regulates%20inflorescence%20architecture%20and%20development%20in%20bread%20wheat%20(Triticum%20aestivum)&amp;author=L.%20E.%20Dixon&amp;author=J.%20R.%20Greenwood&amp;author=S.%20Bencivenga&amp;author=P.%20Zhang&amp;author=J.%20Cockram&amp;volume=30&amp;issue=3&amp;publication_year=2018&amp;pages=563-581&amp;pmid=29444813&amp;doi=10.1105/tpc.17.00961&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270075-bib-0023">
<cite>

Djanaguiraman, M.
, 
Narayanan, S.
, 
Erdayani, E.
, &amp; 
Prasad, P. V.
 (2020). Effects of high temperature stress during anthesis and grain filling periods on photosynthesis, lipids and grain yield in wheat. BMC Plant Biology, 20, Article 268. 10.1186/s12870-020-02479-0

</cite> [<a href="https://doi.org/10.1186/s12870-020-02479-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7285450/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32517754/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Plant%20Biology&amp;title=Effects%20of%20high%20temperature%20stress%20during%20anthesis%20and%20grain%20filling%20periods%20on%20photosynthesis,%20lipids%20and%20grain%20yield%20in%20wheat&amp;author=M.%20Djanaguiraman&amp;author=S.%20Narayanan&amp;author=E.%20Erdayani&amp;author=P.%20V.%20Prasad&amp;volume=20&amp;publication_year=2020&amp;pages=268&amp;pmid=32517754&amp;doi=10.1186/s12870-020-02479-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270075-bib-0024">
<cite>

Earl, D. A.
, &amp; 
VonHoldt, B. M.
 (2012). STRUCTURE HARVESTER: A website and program for visualizing STRUCTURE output and implementing the Evanno method. Conservation genetics resources, 4, 359–361. 10.1007/s12686-011-9548-7
</cite> [<a href="https://doi.org/10.1007/s12686-011-9548-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Conservation%20genetics%20resources&amp;title=STRUCTURE%20HARVESTER:%20A%20website%20and%20program%20for%20visualizing%20STRUCTURE%20output%20and%20implementing%20the%20Evanno%20method&amp;author=D.%20A.%20Earl&amp;author=B.%20M.%20VonHoldt&amp;volume=4&amp;publication_year=2012&amp;pages=359-361&amp;doi=10.1007/s12686-011-9548-7&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270075-bib-0025">
<cite>

Evanno, G.
, 
Regnaut, S.
, &amp; 
Goudet, J.
 (2005). Detecting the number of clusters of individuals using the software STRUCTURE: A simulation study. Molecular Ecology, 14(8), 2611–2620. 10.1111/j.1365-294X.2005.02553.x

</cite> [<a href="https://doi.org/10.1111/j.1365-294X.2005.02553.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15969739/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Ecology&amp;title=Detecting%20the%20number%20of%20clusters%20of%20individuals%20using%20the%20software%20STRUCTURE:%20A%20simulation%20study&amp;author=G.%20Evanno&amp;author=S.%20Regnaut&amp;author=J.%20Goudet&amp;volume=14&amp;issue=8&amp;publication_year=2005&amp;pages=2611-2620&amp;pmid=15969739&amp;doi=10.1111/j.1365-294X.2005.02553.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270075-bib-0026">
<cite>

Fan, J.
, 
Zhang, H.
, 
Shi, Y.
, 
Li, Y.
, 
He, Y.
, 
Wang, Q.
, 
Liu, S.
, 
Yao, Y.
, 
Zhou, X.
, 
Liao, J.
, 
Huang, Y.
, &amp; 
Wang, Z.
 (2024). Systematic identification and characterization of microRNAs with target genes involved in high night temperature stress at the filling stage of rice. Physiologia Plantarum, 176(2), e14305. 10.1111/ppl.14305

</cite> [<a href="https://doi.org/10.1111/ppl.14305" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38659134/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Physiologia%20Plantarum&amp;title=Systematic%20identification%20and%20characterization%20of%20microRNAs%20with%20target%20genes%20involved%20in%20high%20night%20temperature%20stress%20at%20the%20filling%20stage%20of%20rice&amp;author=J.%20Fan&amp;author=H.%20Zhang&amp;author=Y.%20Shi&amp;author=Y.%20Li&amp;author=Y.%20He&amp;volume=176&amp;issue=2&amp;publication_year=2024&amp;pages=e14305&amp;pmid=38659134&amp;doi=10.1111/ppl.14305&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270075-bib-0027">
<cite>

Fan, Y.
, 
Qin, B.
, 
Yang, J.
, 
Ma, L.
, 
Cui, G.
, 
He, W.
, 
Tang, Y.
, 
Zhang, W.
, 
Ma, S.
, 
Ma, C.
, &amp; 
Huang, Z.
 (2024). Night warming increases wheat yield by improving pre‐anthesis plant growth and post‐anthesis grain starch biosynthesis. Journal of Integrative Agriculture, 23(2), 536–550. 10.1016/j.jia.2023.06.024
</cite> [<a href="https://doi.org/10.1016/j.jia.2023.06.024" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Integrative%20Agriculture&amp;title=Night%20warming%20increases%20wheat%20yield%20by%20improving%20pre%E2%80%90anthesis%20plant%20growth%20and%20post%E2%80%90anthesis%20grain%20starch%20biosynthesis&amp;author=Y.%20Fan&amp;author=B.%20Qin&amp;author=J.%20Yang&amp;author=L.%20Ma&amp;author=G.%20Cui&amp;volume=23&amp;issue=2&amp;publication_year=2024&amp;pages=536-550&amp;doi=10.1016/j.jia.2023.06.024&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270075-bib-0028">
<cite>

Fernandez, G. C. J.
 (1992). Effective selection criteria for assessing plant stress tolerance. In Kuo C. G. (Ed.), 
Proceedings of the international symposium on adaptation of vegetables and other food crops in temperature and water stress
 (pp. 257–270). AVRDC Publication.</cite> [<a href="https://scholar.google.com/scholar_lookup?Fernandez,%20G.%20C.%20J.%20(1992).%20Effective%20selection%20criteria%20for%20assessing%20plant%20stress%20tolerance.%20In%20Kuo%20C.%20G.%20(Ed.),%20Proceedings%20of%20the%20international%20symposium%20on%20adaptation%20of%20vegetables%20and%20other%20food%20crops%20in%20temperature%20and%20water%20stress%20(pp.%20257%E2%80%93270).%20AVRDC%20Publication." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270075-bib-0029">
<cite>

Fernández Long, M. E.
, 
Muller, G. V.
, 
Przekurat, B. A.
, &amp; 
Scarpati, O. E.
 (2013). Long‐term and recent changes in temperature‐based agroclimatic indices in Argentina. International Journal of Climatology, 33, 1673–1686. 10.1002/joc.3541
</cite> [<a href="https://doi.org/10.1002/joc.3541" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=International%20Journal%20of%20Climatology&amp;title=Long%E2%80%90term%20and%20recent%20changes%20in%20temperature%E2%80%90based%20agroclimatic%20indices%20in%20Argentina&amp;author=M.%20E.%20Fern%C3%A1ndez%20Long&amp;author=G.%20V.%20Muller&amp;author=B.%20A.%20Przekurat&amp;author=O.%20E.%20Scarpati&amp;volume=33&amp;publication_year=2013&amp;pages=1673-1686&amp;doi=10.1002/joc.3541&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270075-bib-0030">
<cite>

Gaju, O.
, 
Reynolds, M. P.
, 
Sparkes, D. L.
, &amp; 
Foulkes, M. J.
 (2009). Relationships between large‐spike phenotype, grain number, and yield potential in spring wheat. Crop Science, 49(3), 961–973. 10.2135/cropsci2008.05.0285
</cite> [<a href="https://doi.org/10.2135/cropsci2008.05.0285" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Crop%20Science&amp;title=Relationships%20between%20large%E2%80%90spike%20phenotype,%20grain%20number,%20and%20yield%20potential%20in%20spring%20wheat&amp;author=O.%20Gaju&amp;author=M.%20P.%20Reynolds&amp;author=D.%20L.%20Sparkes&amp;author=M.%20J.%20Foulkes&amp;volume=49&amp;issue=3&amp;publication_year=2009&amp;pages=961-973&amp;doi=10.2135/cropsci2008.05.0285&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270075-bib-0031">
<cite>

García, G. A.
, 
Dreccer, M. F.
, 
Miralles, D. J.
, &amp; 
Serrago, R. A.
 (2015). High night temperatures during grain number determination reduce wheat and barley grain yield: A field study. Global Change Biology, 21(11), 4153–4164. 10.1111/gcb.13009

</cite> [<a href="https://doi.org/10.1111/gcb.13009" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26111197/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Global%20Change%20Biology&amp;title=High%20night%20temperatures%20during%20grain%20number%20determination%20reduce%20wheat%20and%20barley%20grain%20yield:%20A%20field%20study&amp;author=G.%20A.%20Garc%C3%ADa&amp;author=M.%20F.%20Dreccer&amp;author=D.%20J.%20Miralles&amp;author=R.%20A.%20Serrago&amp;volume=21&amp;issue=11&amp;publication_year=2015&amp;pages=4153-4164&amp;pmid=26111197&amp;doi=10.1111/gcb.13009&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270075-bib-0032">
<cite>

García, G. A.
, 
Miralles, D. J.
, 
Serrago, R. A.
, 
Alzueta, I.
, 
Huth, N.
, &amp; 
Dreccer, M. F.
 (2018). Warm nights in the Argentine Pampas: Modelling its impact on wheat and barley shows yield reductions. Agricultural Systems, 162, 259–268. 10.1016/j.agsy.2017.12.009
</cite> [<a href="https://doi.org/10.1016/j.agsy.2017.12.009" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Agricultural%20Systems&amp;title=Warm%20nights%20in%20the%20Argentine%20Pampas:%20Modelling%20its%20impact%20on%20wheat%20and%20barley%20shows%20yield%20reductions&amp;author=G.%20A.%20Garc%C3%ADa&amp;author=D.%20J.%20Miralles&amp;author=R.%20A.%20Serrago&amp;author=I.%20Alzueta&amp;author=N.%20Huth&amp;volume=162&amp;publication_year=2018&amp;pages=259-268&amp;doi=10.1016/j.agsy.2017.12.009&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270075-bib-0033">
<cite>

García, G. A.
, 
Serrago, R. A.
, 
Dreccer, M. F.
, &amp; 
Miralles, D. J.
 (2016). Post‐anthesis warm nights reduce grain weight in field‐grown wheat and barley. Field Crops Research, 195, 50–59. 10.1016/j.fcr.2016.06.002
</cite> [<a href="https://doi.org/10.1016/j.fcr.2016.06.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Field%20Crops%20Research&amp;title=Post%E2%80%90anthesis%20warm%20nights%20reduce%20grain%20weight%20in%20field%E2%80%90grown%20wheat%20and%20barley&amp;author=G.%20A.%20Garc%C3%ADa&amp;author=R.%20A.%20Serrago&amp;author=M.%20F.%20Dreccer&amp;author=D.%20J.%20Miralles&amp;volume=195&amp;publication_year=2016&amp;pages=50-59&amp;doi=10.1016/j.fcr.2016.06.002&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270075-bib-0034">
<cite>

Glaubitz, U.
, 
Li, X.
, 
Schaedel, S.
, 
Erban, A.
, 
Sulpice, R.
, 
Kopka, J.
, 
Hincha, D. K.
, &amp; 
Zuther, E.
 (2017). Integrated analysis of rice transcriptomic and metabolomic responses to elevated night temperatures identifies sensitivity‐and tolerance‐related profiles. Plant, Cell &amp; Environment, 40(1), 121–137. 10.1111/pce.12850
</cite> [<a href="https://doi.org/10.1111/pce.12850" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27761892/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant,%20Cell%20&amp;%20Environment&amp;title=Integrated%20analysis%20of%20rice%20transcriptomic%20and%20metabolomic%20responses%20to%20elevated%20night%20temperatures%20identifies%20sensitivity%E2%80%90and%20tolerance%E2%80%90related%20profiles&amp;author=U.%20Glaubitz&amp;author=X.%20Li&amp;author=S.%20Schaedel&amp;author=A.%20Erban&amp;author=R.%20Sulpice&amp;volume=40&amp;issue=1&amp;publication_year=2017&amp;pages=121-137&amp;pmid=27761892&amp;doi=10.1111/pce.12850&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270075-bib-0035">
<cite>

Grogan, S. M.
, 
Brown‐Guedira, G.
, 
Haley, S. D.
, 
McMaster, G. S.
, 
Reid, S. D.
, 
Smith, J.
, &amp; 
Byrne, P. F.
 (2016). Allelic variation in developmental genes and effects on winter wheat heading date in the US Great Plains. PLoS One, 11(4), e0152852. 10.1371/journal.pone.0152852

</cite> [<a href="https://doi.org/10.1371/journal.pone.0152852" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4825937/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27058239/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20One&amp;title=Allelic%20variation%20in%20developmental%20genes%20and%20effects%20on%20winter%20wheat%20heading%20date%20in%20the%20US%20Great%20Plains&amp;author=S.%20M.%20Grogan&amp;author=G.%20Brown%E2%80%90Guedira&amp;author=S.%20D.%20Haley&amp;author=G.%20S.%20McMaster&amp;author=S.%20D.%20Reid&amp;volume=11&amp;issue=4&amp;publication_year=2016&amp;pages=e0152852&amp;pmid=27058239&amp;doi=10.1371/journal.pone.0152852&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270075-bib-0036">
<cite>

Gudi, S.
, 
Saini, D. K.
, 
Halladakeri, P.
, 
Singh, G.
, 
Singh, S.
, 
Kaur, S.
, 
Goyal, P.
, 
Srivastava, P.
, 
Mavi, G. S.
, &amp; 
Sharma, A.
 (2023). Genome‐wide association study unravels genomic regions associated with chlorophyll fluorescence parameters in wheat (<em>Triticum aestivum</em> L.) under different sowing conditions. Plant Cell Reports, 42(9), 1453–1472. 10.1007/s00299-023-03041-6

</cite> [<a href="https://doi.org/10.1007/s00299-023-03041-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37338572/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Cell%20Reports&amp;title=Genome%E2%80%90wide%20association%20study%20unravels%20genomic%20regions%20associated%20with%20chlorophyll%20fluorescence%20parameters%20in%20wheat%20(Triticum%20aestivum%20L.)%20under%20different%20sowing%20conditions&amp;author=S.%20Gudi&amp;author=D.%20K.%20Saini&amp;author=P.%20Halladakeri&amp;author=G.%20Singh&amp;author=S.%20Singh&amp;volume=42&amp;issue=9&amp;publication_year=2023&amp;pages=1453-1472&amp;pmid=37338572&amp;doi=10.1007/s00299-023-03041-6&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270075-bib-0037">
<cite>

Guo, Z.
, 
Chen, D.
, 
Alqudah, A. M.
, 
Röder, M. S.
, 
Ganal, M. W.
, &amp; 
Schnurbusch, T.
 (2017). Genome‐wide association analyses of 54 traits identified multiple loci for the determination of floret fertility in wheat. New Phytologist, 214(1), 257–270. 10.1111/nph.14342

</cite> [<a href="https://doi.org/10.1111/nph.14342" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27918076/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=New%20Phytologist&amp;title=Genome%E2%80%90wide%20association%20analyses%20of%2054%20traits%20identified%20multiple%20loci%20for%20the%20determination%20of%20floret%20fertility%20in%20wheat&amp;author=Z.%20Guo&amp;author=D.%20Chen&amp;author=A.%20M.%20Alqudah&amp;author=M.%20S.%20R%C3%B6der&amp;author=M.%20W.%20Ganal&amp;volume=214&amp;issue=1&amp;publication_year=2017&amp;pages=257-270&amp;pmid=27918076&amp;doi=10.1111/nph.14342&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270075-bib-0038">
<cite>

Guttieri, M. J.
, 
Baenziger, P. S.
, 
Frels, K.
, 
Carver, B.
, 
Arnall, B.
, &amp; 
Waters, B. M.
 (2015). Variation for grain mineral concentration in a diversity panel of current and historical Great Plains hard winter wheat germplasm. Crop Science, 55(3), 1035–1052. 10.2135/cropsci2014.07.0506
</cite> [<a href="https://doi.org/10.2135/cropsci2014.07.0506" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Crop%20Science&amp;title=Variation%20for%20grain%20mineral%20concentration%20in%20a%20diversity%20panel%20of%20current%20and%20historical%20Great%20Plains%20hard%20winter%20wheat%20germplasm&amp;author=M.%20J.%20Guttieri&amp;author=P.%20S.%20Baenziger&amp;author=K.%20Frels&amp;author=B.%20Carver&amp;author=B.%20Arnall&amp;volume=55&amp;issue=3&amp;publication_year=2015&amp;pages=1035-1052&amp;doi=10.2135/cropsci2014.07.0506&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270075-bib-0039">
<cite>

Haasl, R. J.
, &amp; 
Payseur, B. A.
 (2016). Fifteen years of genomewide scans for selection: Trends, lessons and unaddressed genetic sources of complication. Molecular Ecology, 25(1), 5–23. 10.1111/mec.13339

</cite> [<a href="https://doi.org/10.1111/mec.13339" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4868130/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26224644/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Ecology&amp;title=Fifteen%20years%20of%20genomewide%20scans%20for%20selection:%20Trends,%20lessons%20and%20unaddressed%20genetic%20sources%20of%20complication&amp;author=R.%20J.%20Haasl&amp;author=B.%20A.%20Payseur&amp;volume=25&amp;issue=1&amp;publication_year=2016&amp;pages=5-23&amp;pmid=26224644&amp;doi=10.1111/mec.13339&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270075-bib-0040">
<cite>

Halder, J.
, 
Gill, H. S.
, 
Zhang, J.
, 
Altameemi, R.
, 
Olson, E.
, 
Turnipseed, B.
, &amp; 
Sehgal, S. K.
 (2023). Genome‐wide association analysis of spike and kernel traits in the US hard winter wheat. The Plant Genome, 16(1), e20300. 10.1002/tpg2.20300

</cite> [<a href="https://doi.org/10.1002/tpg2.20300" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36636831/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=The%20Plant%20Genome&amp;title=Genome%E2%80%90wide%20association%20analysis%20of%20spike%20and%20kernel%20traits%20in%20the%20US%20hard%20winter%20wheat&amp;author=J.%20Halder&amp;author=H.%20S.%20Gill&amp;author=J.%20Zhang&amp;author=R.%20Altameemi&amp;author=E.%20Olson&amp;volume=16&amp;issue=1&amp;publication_year=2023&amp;pages=e20300&amp;pmid=36636831&amp;doi=10.1002/tpg2.20300&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270075-bib-0041">
<cite>

Hein, N. T.
 (2022). Field‐based phenotyping for the effects of high night‐time temperature stress in wheat and maize [PhD thesis]. Kansas State University. <a href="https://krex.k-state.edu/items/198da612-5888-4549-a446-31ce18a8e0d2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://krex.k‐state.edu/items/198da612‐5888‐4549‐a446‐31ce18a8e0d2</a>
</cite> [<a href="https://scholar.google.com/scholar_lookup?title=Field%E2%80%90based%20phenotyping%20for%20the%20effects%20of%20high%20night%E2%80%90time%20temperature%20stress%20in%20wheat%20and%20maize&amp;author=N.%20T.%20Hein&amp;publication_year=2022&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270075-bib-0042">
<cite>

Hein, N. T.
, 
Bheemanahalli, R.
, 
Wagner, D.
, 
Vennapusa, A. R.
, 
Bustamante, C.
, 
Ostmeyer, T.
, 
Pokharel, M.
, 
Chiluwal, A.
, 
Fu, J.
, 
Srikanthan, D. S.
, &amp; 
Jagadish, S. K.
 (2020). Improved cyber‐physical system captured post‐flowering high night temperature impact on yield and quality of field grown wheat. Scientific Reports, 10(1), Article 22213. 10.1038/s41598-020-79179-0

</cite> [<a href="https://doi.org/10.1038/s41598-020-79179-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7747627/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33335185/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Scientific%20Reports&amp;title=Improved%20cyber%E2%80%90physical%20system%20captured%20post%E2%80%90flowering%20high%20night%20temperature%20impact%20on%20yield%20and%20quality%20of%20field%20grown%20wheat&amp;author=N.%20T.%20Hein&amp;author=R.%20Bheemanahalli&amp;author=D.%20Wagner&amp;author=A.%20R.%20Vennapusa&amp;author=C.%20Bustamante&amp;volume=10&amp;issue=1&amp;publication_year=2020&amp;pages=22213&amp;pmid=33335185&amp;doi=10.1038/s41598-020-79179-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270075-bib-0043">
<cite>

Hein, N. T.
, 
Wagner, D.
, 
Bheemanahalli, R.
, 
Šebela, D.
, 
Bustamante, C.
, 
Chiluwal, A.
, 
Neilsen, M. L.
, &amp; 
Jagadish, S. K.
 (2019). Integrating field‐based heat tents and cyber‐physical system technology to phenotype high night‐time temperature impact on winter wheat. Plant Methods, 15, Article 41. 10.1186/s13007-019-0424-x

</cite> [<a href="https://doi.org/10.1186/s13007-019-0424-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6480702/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31044000/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Methods&amp;title=Integrating%20field%E2%80%90based%20heat%20tents%20and%20cyber%E2%80%90physical%20system%20technology%20to%20phenotype%20high%20night%E2%80%90time%20temperature%20impact%20on%20winter%20wheat&amp;author=N.%20T.%20Hein&amp;author=D.%20Wagner&amp;author=R.%20Bheemanahalli&amp;author=D.%20%C5%A0ebela&amp;author=C.%20Bustamante&amp;volume=15&amp;publication_year=2019&amp;pages=41&amp;pmid=31044000&amp;doi=10.1186/s13007-019-0424-x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270075-bib-0044">
<cite>

Huang, M.
, 
Liu, X.
, 
Zhou, Y.
, 
Summers, R. M.
, &amp; 
Zhang, Z.
 (2019). BLINK: A package for the next level of genome‐wide association studies with both individuals and markers in the millions. Gigascience, 8(2), giy154. 10.1093/gigascience/giy154

</cite> [<a href="https://doi.org/10.1093/gigascience/giy154" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6365300/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30535326/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Gigascience&amp;title=BLINK:%20A%20package%20for%20the%20next%20level%20of%20genome%E2%80%90wide%20association%20studies%20with%20both%20individuals%20and%20markers%20in%20the%20millions&amp;author=M.%20Huang&amp;author=X.%20Liu&amp;author=Y.%20Zhou&amp;author=R.%20M.%20Summers&amp;author=Z.%20Zhang&amp;volume=8&amp;issue=2&amp;publication_year=2019&amp;pages=giy154&amp;pmid=30535326&amp;doi=10.1093/gigascience/giy154&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270075-bib-0045">
<cite>

Huang, X.
, &amp; 
Han, B.
 (2014). Natural variations and genome‐wide association studies in crop plants. Annual Review of Plant Biology, 65(1), 531–551. 10.1146/annurev-arplant-050213-035715
</cite> [<a href="https://doi.org/10.1146/annurev-arplant-050213-035715" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24274033/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Annual%20Review%20of%20Plant%20Biology&amp;title=Natural%20variations%20and%20genome%E2%80%90wide%20association%20studies%20in%20crop%20plants&amp;author=X.%20Huang&amp;author=B.%20Han&amp;volume=65&amp;issue=1&amp;publication_year=2014&amp;pages=531-551&amp;pmid=24274033&amp;doi=10.1146/annurev-arplant-050213-035715&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270075-bib-0046">
<cite>

Impa, S. M.
, 
Raju, B.
, 
Hein, N. T.
, 
Sandhu, J.
, 
Prasad, P. V.
, 
Walia, H.
, &amp; 
Jagadish, S. K.
 (2021). High night temperature effects on wheat and rice: Current status and way forward. Plant, Cell &amp; Environment, 44(7), 2049–2065. 10.1111/pce.14028
</cite> [<a href="https://doi.org/10.1111/pce.14028" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33576033/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant,%20Cell%20&amp;%20Environment&amp;title=High%20night%20temperature%20effects%20on%20wheat%20and%20rice:%20Current%20status%20and%20way%20forward&amp;author=S.%20M.%20Impa&amp;author=B.%20Raju&amp;author=N.%20T.%20Hein&amp;author=J.%20Sandhu&amp;author=P.%20V.%20Prasad&amp;volume=44&amp;issue=7&amp;publication_year=2021&amp;pages=2049-2065&amp;pmid=33576033&amp;doi=10.1111/pce.14028&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270075-bib-0047">
<cite>

Impa, S. M.
, 
Sunoj, V. J.
, 
Krassovskaya, I.
, 
Bheemanahalli, R.
, 
Obata, T.
, &amp; 
Jagadish, S. K.
 (2019). Carbon balance and source‐sink metabolic changes in winter wheat exposed to high night‐time temperature. Plant, Cell &amp; Environment, 42(4), 1233–1246. 10.1111/pce.13488
</cite> [<a href="https://doi.org/10.1111/pce.13488" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30471235/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant,%20Cell%20&amp;%20Environment&amp;title=Carbon%20balance%20and%20source%E2%80%90sink%20metabolic%20changes%20in%20winter%20wheat%20exposed%20to%20high%20night%E2%80%90time%20temperature&amp;author=S.%20M.%20Impa&amp;author=V.%20J.%20Sunoj&amp;author=I.%20Krassovskaya&amp;author=R.%20Bheemanahalli&amp;author=T.%20Obata&amp;volume=42&amp;issue=4&amp;publication_year=2019&amp;pages=1233-1246&amp;pmid=30471235&amp;doi=10.1111/pce.13488&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270075-bib-0048">
<cite>

Impa, S. M.
, 
Vennapusa, A. R.
, 
Bheemanahalli, R.
, 
Sabela, D.
, 
Boyle, D.
, 
Walia, H.
, &amp; 
Jagadish, S. K.
 (2020). High night temperature induced changes in grain starch metabolism alters starch, protein, and lipid accumulation in winter wheat. Plant, Cell &amp; Environment, 43(2), 431–447. 10.1111/pce.13671
</cite> [<a href="https://doi.org/10.1111/pce.13671" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31702834/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant,%20Cell%20&amp;%20Environment&amp;title=High%20night%20temperature%20induced%20changes%20in%20grain%20starch%20metabolism%20alters%20starch,%20protein,%20and%20lipid%20accumulation%20in%20winter%20wheat&amp;author=S.%20M.%20Impa&amp;author=A.%20R.%20Vennapusa&amp;author=R.%20Bheemanahalli&amp;author=D.%20Sabela&amp;author=D.%20Boyle&amp;volume=43&amp;issue=2&amp;publication_year=2020&amp;pages=431-447&amp;pmid=31702834&amp;doi=10.1111/pce.13671&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270075-bib-0049">
<cite>

Jagadish, S. K.
 (2020). Heat stress during flowering in cereals–effects and adaptation strategies. New Phytologist, 226(6), 1567–1572. 10.1111/nph.16429

</cite> [<a href="https://doi.org/10.1111/nph.16429" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31943230/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=New%20Phytologist&amp;title=Heat%20stress%20during%20flowering%20in%20cereals%E2%80%93effects%20and%20adaptation%20strategies&amp;author=S.%20K.%20Jagadish&amp;volume=226&amp;issue=6&amp;publication_year=2020&amp;pages=1567-1572&amp;pmid=31943230&amp;doi=10.1111/nph.16429&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270075-bib-0050">
<cite>

Jing, P.
, 
Wang, D.
, 
Zhu, C.
, &amp; 
Chen, J.
 (2016). Plant physiological, morphological and yield‐related responses to night temperature changes across different species and plant functional types. Frontiers in Plant Science, 7, 1774. 10.3389/fpls.2016.01774

</cite> [<a href="https://doi.org/10.3389/fpls.2016.01774" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5121221/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27933085/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Plant%20Science&amp;title=Plant%20physiological,%20morphological%20and%20yield%E2%80%90related%20responses%20to%20night%20temperature%20changes%20across%20different%20species%20and%20plant%20functional%20types&amp;author=P.%20Jing&amp;author=D.%20Wang&amp;author=C.%20Zhu&amp;author=J.%20Chen&amp;volume=7&amp;publication_year=2016&amp;pages=1774&amp;pmid=27933085&amp;doi=10.3389/fpls.2016.01774&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270075-bib-0051">
<cite>

Kahlon, K. S.
, 
Rawale, K. S.
, 
Kumar, S.
, &amp; 
Gill, K. S.
 (2024). Identification and mapping of QTLs and their corresponding candidate genes controlling high night‐time temperature stress tolerance in wheat (<em>Triticum aestivum</em> L.). The Plant Genome, 17(4), e20517. 10.1002/tpg2.20517

</cite> [<a href="https://doi.org/10.1002/tpg2.20517" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11628910/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39318199/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=The%20Plant%20Genome&amp;title=Identification%20and%20mapping%20of%20QTLs%20and%20their%20corresponding%20candidate%20genes%20controlling%20high%20night%E2%80%90time%20temperature%20stress%20tolerance%20in%20wheat%20(Triticum%20aestivum%20L.)&amp;author=K.%20S.%20Kahlon&amp;author=K.%20S.%20Rawale&amp;author=S.%20Kumar&amp;author=K.%20S.%20Gill&amp;volume=17&amp;issue=4&amp;publication_year=2024&amp;pages=e20517&amp;pmid=39318199&amp;doi=10.1002/tpg2.20517&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270075-bib-0052">
<cite>

Kanno, K.
, 
Mae, T.
, &amp; 
Makino, A.
 (2009). High night temperature stimulates photosynthesis, biomass production and growth during the vegetative stage of rice plants. Soil Science and Plant Nutrition, 55(1), 124–131. 10.1111/j.1747-0765.2008.00343.x
</cite> [<a href="https://doi.org/10.1111/j.1747-0765.2008.00343.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Soil%20Science%20and%20Plant%20Nutrition&amp;title=High%20night%20temperature%20stimulates%20photosynthesis,%20biomass%20production%20and%20growth%20during%20the%20vegetative%20stage%20of%20rice%20plants&amp;author=K.%20Kanno&amp;author=T.%20Mae&amp;author=A.%20Makino&amp;volume=55&amp;issue=1&amp;publication_year=2009&amp;pages=124-131&amp;doi=10.1111/j.1747-0765.2008.00343.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270075-bib-0053">
<cite>

Kaur, S.
, 
Gill, H. S.
, 
Breiland, M.
, 
Kolmer, J. A.
, 
Gupta, R.
, 
Sehgal, S. K.
, &amp; 
Gill, U.
 (2023). Identification of leaf rust resistance loci in a geographically diverse panel of wheat using genome‐wide association analysis. Frontiers in Plant Science, 14, 1090163. 10.3389/fpls.2023.1090163

</cite> [<a href="https://doi.org/10.3389/fpls.2023.1090163" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9929074/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36818858/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Plant%20Science&amp;title=Identification%20of%20leaf%20rust%20resistance%20loci%20in%20a%20geographically%20diverse%20panel%20of%20wheat%20using%20genome%E2%80%90wide%20association%20analysis&amp;author=S.%20Kaur&amp;author=H.%20S.%20Gill&amp;author=M.%20Breiland&amp;author=J.%20A.%20Kolmer&amp;author=R.%20Gupta&amp;volume=14&amp;publication_year=2023&amp;pages=1090163&amp;pmid=36818858&amp;doi=10.3389/fpls.2023.1090163&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270075-bib-0054">
<cite>

Khalid, M.
, 
Afzal, F.
, 
Gul, A.
, 
Amir, R.
, 
Subhani, A.
, 
Ahmed, Z.
, 
Mahmood, Z.
, 
Xia, X.
, 
Rasheed, A.
, &amp; 
He, Z.
 (2019). Molecular characterization of 87 functional genes in wheat diversity panel and their association with phenotypes under well‐watered and water‐limited conditions. Frontiers in Plant Science, 10, 717. 10.3389/fpls.2019.00717

</cite> [<a href="https://doi.org/10.3389/fpls.2019.00717" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6558208/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31214230/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Plant%20Science&amp;title=Molecular%20characterization%20of%2087%20functional%20genes%20in%20wheat%20diversity%20panel%20and%20their%20association%20with%20phenotypes%20under%20well%E2%80%90watered%20and%20water%E2%80%90limited%20conditions&amp;author=M.%20Khalid&amp;author=F.%20Afzal&amp;author=A.%20Gul&amp;author=R.%20Amir&amp;author=A.%20Subhani&amp;volume=10&amp;publication_year=2019&amp;pages=717&amp;pmid=31214230&amp;doi=10.3389/fpls.2019.00717&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270075-bib-0055">
<cite>

Kim, J. A.
, 
Kim, H. S.
, 
Choi, S. H.
, 
Jang, J. Y.
, 
Jeong, M. J.
, &amp; 
Lee, S. I.
 (2017). The importance of the circadian clock in regulating plant metabolism. International Journal of Molecular Sciences, 18(12), 2680. 10.3390/ijms18122680

</cite> [<a href="https://doi.org/10.3390/ijms18122680" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5751282/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29232921/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=International%20Journal%20of%20Molecular%20Sciences&amp;title=The%20importance%20of%20the%20circadian%20clock%20in%20regulating%20plant%20metabolism&amp;author=J.%20A.%20Kim&amp;author=H.%20S.%20Kim&amp;author=S.%20H.%20Choi&amp;author=J.%20Y.%20Jang&amp;author=M.%20J.%20Jeong&amp;volume=18&amp;issue=12&amp;publication_year=2017&amp;pages=2680&amp;pmid=29232921&amp;doi=10.3390/ijms18122680&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270075-bib-0056">
<cite>

Klein Tank, A. M.
, 
Peterson, T. C.
, 
Quadir, D. A.
, 
Dorji, S.
, 
Zou, X.
, 
Tang, H.
, 
Santhosh, K.
, 
Joshi, U. R.
, 
Jaswal, A. K.
, 
Kolli, R. K.
, &amp; 
Spektorman, T.
 (2006). Changes in daily temperature and precipitation extremes in central and south Asia. Journal of Geophysical Research: Atmospheres, 111(D16), D16105. 10.1029/2005JD006316
</cite> [<a href="https://doi.org/10.1029/2005JD006316" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Geophysical%20Research:%20Atmospheres&amp;title=Changes%20in%20daily%20temperature%20and%20precipitation%20extremes%20in%20central%20and%20south%20Asia&amp;author=A.%20M.%20Klein%20Tank&amp;author=T.%20C.%20Peterson&amp;author=D.%20A.%20Quadir&amp;author=S.%20Dorji&amp;author=X.%20Zou&amp;volume=111&amp;issue=D16&amp;publication_year=2006&amp;pages=D16105&amp;doi=10.1029/2005JD006316&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270075-bib-0057">
<cite>

Kowalski, A. M.
, 
Gooding, M.
, 
Ferrante, A.
, 
Slafer, G. A.
, 
Orford, S.
, 
Gasperini, D.
, &amp; 
Griffiths, S.
 (2016). Agronomic assessment of the wheat semi‐dwarfing gene <em>Rht8</em> in contrasting nitrogen treatments and water regimes. Field Crops Research, 191, 150–160. 10.1016/j.fcr.2016.02.026

</cite> [<a href="https://doi.org/10.1016/j.fcr.2016.02.026" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4862442/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27212788/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Field%20Crops%20Research&amp;title=Agronomic%20assessment%20of%20the%20wheat%20semi%E2%80%90dwarfing%20gene%20Rht8%20in%20contrasting%20nitrogen%20treatments%20and%20water%20regimes&amp;author=A.%20M.%20Kowalski&amp;author=M.%20Gooding&amp;author=A.%20Ferrante&amp;author=G.%20A.%20Slafer&amp;author=S.%20Orford&amp;volume=191&amp;publication_year=2016&amp;pages=150-160&amp;pmid=27212788&amp;doi=10.1016/j.fcr.2016.02.026&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270075-bib-0058">
<cite>

Kumar, A.
, 
Gupta, C.
, 
Thomas, J.
, &amp; 
Pereira, A.
 (2021). Genetic dissection of grain yield component traits under high nighttime temperature stress in a rice diversity panel. Frontiers in Plant Science, 12, 712167. 10.3389/fpls.2021.712167

</cite> [<a href="https://doi.org/10.3389/fpls.2021.712167" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8508263/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34650575/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Plant%20Science&amp;title=Genetic%20dissection%20of%20grain%20yield%20component%20traits%20under%20high%20nighttime%20temperature%20stress%20in%20a%20rice%20diversity%20panel&amp;author=A.%20Kumar&amp;author=C.%20Gupta&amp;author=J.%20Thomas&amp;author=A.%20Pereira&amp;volume=12&amp;publication_year=2021&amp;pages=712167&amp;pmid=34650575&amp;doi=10.3389/fpls.2021.712167&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270075-bib-0059">
<cite>

Kumar, A.
, 
Thomas, J.
, 
Gill, N.
, 
Dwiningsih, Y.
, 
Ruiz, C.
, 
Famoso, A.
, &amp; 
Pereira, A.
 (2023). Molecular mapping and characterization of QTLs for grain quality traits in a RIL population of US rice under high nighttime temperature stress. Scientific Reports, 13(1), Article 4880. 10.1038/s41598-023-31399-w

</cite> [<a href="https://doi.org/10.1038/s41598-023-31399-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10039871/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36966148/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Scientific%20Reports&amp;title=Molecular%20mapping%20and%20characterization%20of%20QTLs%20for%20grain%20quality%20traits%20in%20a%20RIL%20population%20of%20US%20rice%20under%20high%20nighttime%20temperature%20stress&amp;author=A.%20Kumar&amp;author=J.%20Thomas&amp;author=N.%20Gill&amp;author=Y.%20Dwiningsih&amp;author=C.%20Ruiz&amp;volume=13&amp;issue=1&amp;publication_year=2023&amp;pages=4880&amp;pmid=36966148&amp;doi=10.1038/s41598-023-31399-w&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270075-bib-0060">
<cite>

Kumar, S.
, 
Kumar, S.
, 
Sharma, H.
, 
Singh, V. P.
, 
Rawale, K. S.
, 
Kahlon, K. S.
, 
Gupta, V.
, 
Bhatt, S. K.
, 
Vairamani, R.
, 
Gill, K. S.
, &amp; 
Balyan, H. S.
 (2024). Physical map of QTL for eleven agronomic traits across fifteen environments, identification of related candidate genes, and development of KASP markers with emphasis on terminal heat stress tolerance in common wheat. Theoretical and Applied Genetics, 137(10), Article 235. 10.1007/s00122-024-04748-0

</cite> [<a href="https://doi.org/10.1007/s00122-024-04748-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39333356/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Theoretical%20and%20Applied%20Genetics&amp;title=Physical%20map%20of%20QTL%20for%20eleven%20agronomic%20traits%20across%20fifteen%20environments,%20identification%20of%20related%20candidate%20genes,%20and%20development%20of%20KASP%20markers%20with%20emphasis%20on%20terminal%20heat%20stress%20tolerance%20in%20common%20wheat&amp;author=S.%20Kumar&amp;author=S.%20Kumar&amp;author=H.%20Sharma&amp;author=V.%20P.%20Singh&amp;author=K.%20S.%20Rawale&amp;volume=137&amp;issue=10&amp;publication_year=2024&amp;pages=235&amp;pmid=39333356&amp;doi=10.1007/s00122-024-04748-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270075-bib-0061">
<cite>

Kumar, S.
, 
Singh, V. P.
, 
Saini, D. K.
, 
Sharma, H.
, 
Saripalli, G.
, 
Kumar, S.
, 
Balyan, H. S.
, &amp; 
Gupta, P. K.
 (2021). Meta‐QTLs, ortho‐MQTLs, and candidate genes for thermotolerance in wheat (<em>Triticum aestivum</em> L.). Molecular Breeding, 41, 1–22. 10.1007/s11032-021-01264-7

</cite> [<a href="https://doi.org/10.1007/s11032-021-01264-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10236124/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37309361/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Breeding&amp;title=Meta%E2%80%90QTLs,%20ortho%E2%80%90MQTLs,%20and%20candidate%20genes%20for%20thermotolerance%20in%20wheat%20(Triticum%20aestivum%20L.)&amp;author=S.%20Kumar&amp;author=V.%20P.%20Singh&amp;author=D.%20K.%20Saini&amp;author=H.%20Sharma&amp;author=G.%20Saripalli&amp;volume=41&amp;publication_year=2021&amp;pages=1-22&amp;pmid=37309361&amp;doi=10.1007/s11032-021-01264-7&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270075-bib-0062">
<cite>

Kundu, S.
, 
Padile, G. P.
, 
Meena, B.
, 
Das, S.
, 
Singh, S. K.
, &amp; 
Bahuguna, R. N.
 (2024). Warmer nights predominantly alter yield and biomass in wheat experiencing heat waves at the terminal growth stage. Acta Physiologiae Plantarum, 46(4), Article 38. 10.1007/s11738-024-03670-8
</cite> [<a href="https://doi.org/10.1007/s11738-024-03670-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Acta%20Physiologiae%20Plantarum&amp;title=Warmer%20nights%20predominantly%20alter%20yield%20and%20biomass%20in%20wheat%20experiencing%20heat%20waves%20at%20the%20terminal%20growth%20stage&amp;author=S.%20Kundu&amp;author=G.%20P.%20Padile&amp;author=B.%20Meena&amp;author=S.%20Das&amp;author=S.%20K.%20Singh&amp;volume=46&amp;issue=4&amp;publication_year=2024&amp;pages=38&amp;doi=10.1007/s11738-024-03670-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270075-bib-0063">
<cite>

Lacaze, X.
, 
Hayes, P. M.
, &amp; 
Korol, A.
 (2009). Genetics of phenotypic plasticity: QTL analysis in barley, <em>Hordeum vulgare</em>
. Heredity, 102(2), 163–173. 10.1038/hdy.2008.76

</cite> [<a href="https://doi.org/10.1038/hdy.2008.76" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18941472/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Heredity&amp;title=Genetics%20of%20phenotypic%20plasticity:%20QTL%20analysis%20in%20barley,%20Hordeum%20vulgare&amp;author=X.%20Lacaze&amp;author=P.%20M.%20Hayes&amp;author=A.%20Korol&amp;volume=102&amp;issue=2&amp;publication_year=2009&amp;pages=163-173&amp;pmid=18941472&amp;doi=10.1038/hdy.2008.76&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270075-bib-0064">
<cite>

Langstroff, A.
, 
Heuermann, M. C.
, 
Stahl, A.
, &amp; 
Junker, A.
 (2022). Opportunities and limits of controlled‐environment plant phenotyping for climate response traits. Theoretical and Applied Genetics, 135(1), 1–16. 10.1007/s00122-021-03892-1
</cite> [<a href="https://doi.org/10.1007/s00122-021-03892-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8741719/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34302493/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Theoretical%20and%20Applied%20Genetics&amp;title=Opportunities%20and%20limits%20of%20controlled%E2%80%90environment%20plant%20phenotyping%20for%20climate%20response%20traits&amp;author=A.%20Langstroff&amp;author=M.%20C.%20Heuermann&amp;author=A.%20Stahl&amp;author=A.%20Junker&amp;volume=135&amp;issue=1&amp;publication_year=2022&amp;pages=1-16&amp;pmid=34302493&amp;doi=10.1007/s00122-021-03892-1&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270075-bib-0065">
<cite>

Lazăr, C.
, &amp; 
Lazăr, D. A.
 (2010). Simulation of temperature increase influence on winter wheat yields and development in South‐Eastern Romania. Romanian Agricultural Research, 27, 7–15.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Romanian%20Agricultural%20Research&amp;title=Simulation%20of%20temperature%20increase%20influence%20on%20winter%20wheat%20yields%20and%20development%20in%20South%E2%80%90Eastern%20Romania&amp;author=C.%20Laz%C4%83r&amp;author=D.%20A.%20Laz%C4%83r&amp;volume=27&amp;publication_year=2010&amp;pages=7-15&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270075-bib-0066">
<cite>

Lee, K.
, &amp; 
Kang, H.
 (2016). Emerging roles of RNA‐binding proteins in plant growth, development, and stress responses. Molecules and Cells, 39(3), 179–185. 10.14348/molcells.2016.2359

</cite> [<a href="https://doi.org/10.14348/molcells.2016.2359" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4794599/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26831454/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecules%20and%20Cells&amp;title=Emerging%20roles%20of%20RNA%E2%80%90binding%20proteins%20in%20plant%20growth,%20development,%20and%20stress%20responses&amp;author=K.%20Lee&amp;author=H.%20Kang&amp;volume=39&amp;issue=3&amp;publication_year=2016&amp;pages=179-185&amp;pmid=26831454&amp;doi=10.14348/molcells.2016.2359&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270075-bib-0067">
<cite>

Lee, S.
, &amp; 
Lee, D. K.
 (2018). What is the proper way to apply the multiple comparison test?. Korean Journal of Anesthesiology, 71(5), 353–360. 10.4097/kja.d.18.00242

</cite> [<a href="https://doi.org/10.4097/kja.d.18.00242" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6193594/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30157585/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Korean%20Journal%20of%20Anesthesiology&amp;title=What%20is%20the%20proper%20way%20to%20apply%20the%20multiple%20comparison%20test?&amp;author=S.%20Lee&amp;author=D.%20K.%20Lee&amp;volume=71&amp;issue=5&amp;publication_year=2018&amp;pages=353-360&amp;pmid=30157585&amp;doi=10.4097/kja.d.18.00242&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270075-bib-0068">
<cite>

Li, A.
, 
Hao, C.
, 
Wang, Z.
, 
Geng, S.
, 
Jia, M.
, 
Wang, F.
, 
Han, X.
, 
Kong, X.
, 
Yin, L.
, 
Tao, S.
, &amp; 
Mao, L.
 (2022). Wheat breeding history reveals synergistic selection of pleiotropic genomic sites for plant architecture and grain yield. Molecular Plant, 15(3), 504–519. 10.1016/j.molp.2022.01.004

</cite> [<a href="https://doi.org/10.1016/j.molp.2022.01.004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35026438/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Plant&amp;title=Wheat%20breeding%20history%20reveals%20synergistic%20selection%20of%20pleiotropic%20genomic%20sites%20for%20plant%20architecture%20and%20grain%20yield&amp;author=A.%20Li&amp;author=C.%20Hao&amp;author=Z.%20Wang&amp;author=S.%20Geng&amp;author=M.%20Jia&amp;volume=15&amp;issue=3&amp;publication_year=2022&amp;pages=504-519&amp;pmid=35026438&amp;doi=10.1016/j.molp.2022.01.004&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270075-bib-0070">
<cite>

Li, X.
, 
Guo, T.
, 
Mu, Q.
, 
Li, X.
, &amp; 
Yu, J.
 (2018). Genomic and environmental determinants and their interplay underlying phenotypic plasticity. Proceedings of the National Academy of Sciences, 115(26), 6679–6684. 10.1073/pnas.1718326115
</cite> [<a href="https://doi.org/10.1073/pnas.1718326115" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6042117/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29891664/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proceedings%20of%20the%20National%20Academy%20of%20Sciences&amp;title=Genomic%20and%20environmental%20determinants%20and%20their%20interplay%20underlying%20phenotypic%20plasticity&amp;author=X.%20Li&amp;author=T.%20Guo&amp;author=Q.%20Mu&amp;author=X.%20Li&amp;author=J.%20Yu&amp;volume=115&amp;issue=26&amp;publication_year=2018&amp;pages=6679-6684&amp;pmid=29891664&amp;doi=10.1073/pnas.1718326115&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270075-bib-0071">
<cite>

Li, X.
, 
Li, Y.
, 
Yu, X.
, 
Sun, F.
, 
Yang, G.
, &amp; 
He, G.
 (2020). Genomics‐enabled analysis of Puroindoline b2 genes identifies new alleles in wheat and related Triticeae species. International Journal of Molecular Sciences, 21(4), 1304. 10.3390/ijms21041304

</cite> [<a href="https://doi.org/10.3390/ijms21041304" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7072932/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32075191/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=International%20Journal%20of%20Molecular%20Sciences&amp;title=Genomics%E2%80%90enabled%20analysis%20of%20Puroindoline%20b2%20genes%20identifies%20new%20alleles%20in%20wheat%20and%20related%20Triticeae%20species&amp;author=X.%20Li&amp;author=Y.%20Li&amp;author=X.%20Yu&amp;author=F.%20Sun&amp;author=G.%20Yang&amp;volume=21&amp;issue=4&amp;publication_year=2020&amp;pages=1304&amp;pmid=32075191&amp;doi=10.3390/ijms21041304&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270075-bib-0072">
<cite>

Li, X. M.
, 
Chao, D. Y.
, 
Wu, Y.
, 
Huang, X.
, 
Chen, K.
, 
Cui, L. G.
, 
Su, L.
, 
Ye, W. W.
, 
Chen, H.
, 
Chen, H. C.
, &amp; 
Dong, N. Q.
 (2015). Natural alleles of a proteasome α2 subunit gene contribute to thermotolerance and adaptation of African rice. Nature Genetics, 47(7), 827–833. 10.1038/ng.3305

</cite> [<a href="https://doi.org/10.1038/ng.3305" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25985140/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Genetics&amp;title=Natural%20alleles%20of%20a%20proteasome%20%CE%B12%20subunit%20gene%20contribute%20to%20thermotolerance%20and%20adaptation%20of%20African%20rice&amp;author=X.%20M.%20Li&amp;author=D.%20Y.%20Chao&amp;author=Y.%20Wu&amp;author=X.%20Huang&amp;author=K.%20Chen&amp;volume=47&amp;issue=7&amp;publication_year=2015&amp;pages=827-833&amp;pmid=25985140&amp;doi=10.1038/ng.3305&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270075-bib-0074">
<cite>

Liao, J. L.
, 
Zhou, H. W.
, 
Peng, Q.
, 
Zhong, P. A.
, 
Zhang, H. Y.
, 
He, C.
, &amp; 
Huang, Y. J.
 (2015). Transcriptome changes in rice (<em>Oryza sativa</em> L.) in response to high night temperature stress at the early milky stage. BMC Genomics, 16, Article 18. 10.1186/s12864-015-1222-0

</cite> [<a href="https://doi.org/10.1186/s12864-015-1222-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4369907/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25928563/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Genomics&amp;title=Transcriptome%20changes%20in%20rice%20(Oryza%20sativa%20L.)%20in%20response%20to%20high%20night%20temperature%20stress%20at%20the%20early%20milky%20stage&amp;author=J.%20L.%20Liao&amp;author=H.%20W.%20Zhou&amp;author=Q.%20Peng&amp;author=P.%20A.%20Zhong&amp;author=H.%20Y.%20Zhang&amp;volume=16&amp;publication_year=2015&amp;pages=18&amp;pmid=25928563&amp;doi=10.1186/s12864-015-1222-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270075-bib-0075">
<cite>

Liu, B.
, 
Liu, L.
, 
Asseng, S.
, 
Zhang, D.
, 
Ma, W.
, 
Tang, L.
, 
Cao, W.
, &amp; 
Zhu, Y.
 (2020). Modelling the effects of post‐heading heat stress on biomass partitioning, and grain number and weight of wheat. Journal of Experimental Botany, 71(19), 6015–6031. 10.1093/jxb/eraa310

</cite> [<a href="https://doi.org/10.1093/jxb/eraa310" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32964926/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Experimental%20Botany&amp;title=Modelling%20the%20effects%20of%20post%E2%80%90heading%20heat%20stress%20on%20biomass%20partitioning,%20and%20grain%20number%20and%20weight%20of%20wheat&amp;author=B.%20Liu&amp;author=L.%20Liu&amp;author=S.%20Asseng&amp;author=D.%20Zhang&amp;author=W.%20Ma&amp;volume=71&amp;issue=19&amp;publication_year=2020&amp;pages=6015-6031&amp;pmid=32964926&amp;doi=10.1093/jxb/eraa310&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270075-bib-0076">
<cite>

Liu, X.
, 
Huang, M.
, 
Fan, B.
, 
Buckler, E. S.
, &amp; 
Zhang, Z.
 (2016). Iterative usage of fixed and random effect models for powerful and efficient genome‐wide association studies. PLoS Genetics, 12(2), e1005767. 10.1371/journal.pgen.1005767

</cite> [<a href="https://doi.org/10.1371/journal.pgen.1005767" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4734661/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26828793/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20Genetics&amp;title=Iterative%20usage%20of%20fixed%20and%20random%20effect%20models%20for%20powerful%20and%20efficient%20genome%E2%80%90wide%20association%20studies&amp;author=X.%20Liu&amp;author=M.%20Huang&amp;author=B.%20Fan&amp;author=E.%20S.%20Buckler&amp;author=Z.%20Zhang&amp;volume=12&amp;issue=2&amp;publication_year=2016&amp;pages=e1005767&amp;pmid=26828793&amp;doi=10.1371/journal.pgen.1005767&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270075-bib-0077">
<cite>

Lobell, D. B.
 (2014). Climate change adaptation in crop production: Beware of illusions. Global Food Security, 3(2), 72–76. 10.1016/j.gfs.2014.05.002
</cite> [<a href="https://doi.org/10.1016/j.gfs.2014.05.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Global%20Food%20Security&amp;title=Climate%20change%20adaptation%20in%20crop%20production:%20Beware%20of%20illusions&amp;author=D.%20B.%20Lobell&amp;volume=3&amp;issue=2&amp;publication_year=2014&amp;pages=72-76&amp;doi=10.1016/j.gfs.2014.05.002&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270075-bib-0078">
<cite>

MajidiMehr, A.
, 
Pahlavani, M. H.
, 
El Gataa, Z.
, &amp; 
Amiri‐Fahliani, R.
 (2024). Genome‐wide association mapping and genomic prediction of water deficit stress tolerance indices in spring bread wheat. Gene Reports, 37, 102054. 10.1016/j.genrep.2024.102054
</cite> [<a href="https://doi.org/10.1016/j.genrep.2024.102054" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Gene%20Reports&amp;title=Genome%E2%80%90wide%20association%20mapping%20and%20genomic%20prediction%20of%20water%20deficit%20stress%20tolerance%20indices%20in%20spring%20bread%20wheat&amp;author=A.%20MajidiMehr&amp;author=M.%20H.%20Pahlavani&amp;author=Z.%20El%20Gataa&amp;author=R.%20Amiri%E2%80%90Fahliani&amp;volume=37&amp;publication_year=2024&amp;pages=102054&amp;doi=10.1016/j.genrep.2024.102054&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270075-bib-0079">
<cite>

Malosetti, M.
, 
Zwep, L. B.
, 
Forrest, K.
, 
van Eeuwijk, F. A.
, &amp; 
Dieters, M.
 (2021). Lessons from a GWAS study of a wheat pre‐breeding program: Pyramiding resistance alleles to Fusarium crown rot. Theoretical and Applied Genetics, 134, 897–908. 10.1007/s00122-020-03740-8

</cite> [<a href="https://doi.org/10.1007/s00122-020-03740-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7925461/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33367942/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Theoretical%20and%20Applied%20Genetics&amp;title=Lessons%20from%20a%20GWAS%20study%20of%20a%20wheat%20pre%E2%80%90breeding%20program:%20Pyramiding%20resistance%20alleles%20to%20Fusarium%20crown%20rot&amp;author=M.%20Malosetti&amp;author=L.%20B.%20Zwep&amp;author=K.%20Forrest&amp;author=F.%20A.%20van%20Eeuwijk&amp;author=M.%20Dieters&amp;volume=134&amp;publication_year=2021&amp;pages=897-908&amp;pmid=33367942&amp;doi=10.1007/s00122-020-03740-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270075-bib-0080">
<cite>

Mao, H.
, 
Jiang, C.
, 
Tang, C.
, 
Nie, X.
, 
Du, L.
, 
Liu, Y.
, 
Cheng, P.
, 
Wu, Y.
, 
Liu, H.
, 
Kang, Z.
, &amp; 
Wang, X.
 (2023). Wheat adaptation to environmental stresses under climate change: Molecular basis and genetic improvement. Molecular Plant, 16(10), 1564–1589. 10.1016/j.molp.2023.09.001

</cite> [<a href="https://doi.org/10.1016/j.molp.2023.09.001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37671604/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Plant&amp;title=Wheat%20adaptation%20to%20environmental%20stresses%20under%20climate%20change:%20Molecular%20basis%20and%20genetic%20improvement&amp;author=H.%20Mao&amp;author=C.%20Jiang&amp;author=C.%20Tang&amp;author=X.%20Nie&amp;author=L.%20Du&amp;volume=16&amp;issue=10&amp;publication_year=2023&amp;pages=1564-1589&amp;pmid=37671604&amp;doi=10.1016/j.molp.2023.09.001&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270075-bib-0081">
<cite>

McAusland, L.
, 
Acevedo‐Siaca, L. G.
, 
Pinto, R. S.
, 
Pinto, F.
, 
Molero, G.
, 
Garatuza‐Payan, J.
, 
Reynolds, M. P.
, 
Murchie, E. H.
, &amp; 
Yepez, E. A.
 (2023). Night‐time warming in the field reduces nocturnal stomatal conductance and grain yield but does not alter daytime physiological responses. New Phytologist, 239(5), 1622–1636. 10.1111/nph.19075

</cite> [<a href="https://doi.org/10.1111/nph.19075" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10952344/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37430457/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=New%20Phytologist&amp;title=Night%E2%80%90time%20warming%20in%20the%20field%20reduces%20nocturnal%20stomatal%20conductance%20and%20grain%20yield%20but%20does%20not%20alter%20daytime%20physiological%20responses&amp;author=L.%20McAusland&amp;author=L.%20G.%20Acevedo%E2%80%90Siaca&amp;author=R.%20S.%20Pinto&amp;author=F.%20Pinto&amp;author=G.%20Molero&amp;volume=239&amp;issue=5&amp;publication_year=2023&amp;pages=1622-1636&amp;pmid=37430457&amp;doi=10.1111/nph.19075&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270075-bib-0082">
<cite>

Milner, M. J.
, 
Swarbreck, S. M.
, 
Craze, M.
, 
Bowden, S.
, 
Griffiths, H.
, 
Bentley, A. R.
, &amp; 
Wallington, E. J.
 (2022). Over‐expression of <em>TaDWF4</em> increases wheat productivity under low and sufficient nitrogen through enhanced carbon assimilation. Communications Biology, 5(1), Article 193. 10.1038/s42003-022-03139-9

</cite> [<a href="https://doi.org/10.1038/s42003-022-03139-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8894359/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35241776/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Communications%20Biology&amp;title=Over%E2%80%90expression%20of%20TaDWF4%20increases%20wheat%20productivity%20under%20low%20and%20sufficient%20nitrogen%20through%20enhanced%20carbon%20assimilation&amp;author=M.%20J.%20Milner&amp;author=S.%20M.%20Swarbreck&amp;author=M.%20Craze&amp;author=S.%20Bowden&amp;author=H.%20Griffiths&amp;volume=5&amp;issue=1&amp;publication_year=2022&amp;pages=193&amp;pmid=35241776&amp;doi=10.1038/s42003-022-03139-9&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270075-bib-0083">
<cite>

Minic, Z.
, &amp; 
Jouanin, L.
 (2006). Plant glycoside hydrolases involved in cell wall polysaccharide degradation. Plant physiology and Biochemistry, 44(7–9), 435–449. 10.1016/j.plaphy.2006.08.001

</cite> [<a href="https://doi.org/10.1016/j.plaphy.2006.08.001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17023165/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20physiology%20and%20Biochemistry&amp;title=Plant%20glycoside%20hydrolases%20involved%20in%20cell%20wall%20polysaccharide%20degradation&amp;author=Z.%20Minic&amp;author=L.%20Jouanin&amp;volume=44&amp;issue=7%E2%80%939&amp;publication_year=2006&amp;pages=435-449&amp;pmid=17023165&amp;doi=10.1016/j.plaphy.2006.08.001&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270075-bib-0084">
<cite>

Mirosavljević, M.
, 
Mikić, S.
, 
Kondić‐Špika, A.
, 
Župunski, V.
, 
Zhou, R.
, 
Abdelhakim, L.
, &amp; 
Ottosen, C. O.
 (2021). The effect of heat stress on some main spike traits in 12 wheat cultivars at anthesis and mid‐grain filling stage. Plant, Soil and Environment, 67(2), 71–76. 10.17221/457/2020-PSE
</cite> [<a href="https://doi.org/10.17221/457/2020-PSE" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant,%20Soil%20and%20Environment&amp;title=The%20effect%20of%20heat%20stress%20on%20some%20main%20spike%20traits%20in%2012%20wheat%20cultivars%20at%20anthesis%20and%20mid%E2%80%90grain%20filling%20stage&amp;author=M.%20Mirosavljevi%C4%87&amp;author=S.%20Miki%C4%87&amp;author=A.%20Kondi%C4%87%E2%80%90%C5%A0pika&amp;author=V.%20%C5%BDupunski&amp;author=R.%20Zhou&amp;volume=67&amp;issue=2&amp;publication_year=2021&amp;pages=71-76&amp;doi=10.17221/457/2020-PSE&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270075-bib-0085">
<cite>

Mizuno, N.
, 
Kinoshita, M.
, 
Kinoshita, S.
, 
Nishida, H.
, 
Fujita, M.
, 
Kato, K.
, 
Murai, K.
, &amp; 
Nasuda, S.
 (2016). Loss‐of‐function mutations in three homoeologous <em>PHYTOCLOCK 1</em> genes in common wheat are associated with the extra‐early flowering phenotype. PLoS One, 11(10), e0165618. 10.1371/journal.pone.0165618

</cite> [<a href="https://doi.org/10.1371/journal.pone.0165618" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5082820/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27788250/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20One&amp;title=Loss%E2%80%90of%E2%80%90function%20mutations%20in%20three%20homoeologous%20PHYTOCLOCK%201%20genes%20in%20common%20wheat%20are%20associated%20with%20the%20extra%E2%80%90early%20flowering%20phenotype&amp;author=N.%20Mizuno&amp;author=M.%20Kinoshita&amp;author=S.%20Kinoshita&amp;author=H.%20Nishida&amp;author=M.%20Fujita&amp;volume=11&amp;issue=10&amp;publication_year=2016&amp;pages=e0165618&amp;pmid=27788250&amp;doi=10.1371/journal.pone.0165618&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270075-bib-0086">
<cite>

Mohammed, A. R.
, &amp; 
Tarpley, L.
 (2010). Effects of high night temperature and spikelet position on yield‐related parameters of rice (<em>Oryza sativa</em> L.) plants. European Journal of Agronomy, 33(2), 117–123. 10.1016/j.eja.2009.11.006
</cite> [<a href="https://doi.org/10.1016/j.eja.2009.11.006" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=European%20Journal%20of%20Agronomy&amp;title=Effects%20of%20high%20night%20temperature%20and%20spikelet%20position%20on%20yield%E2%80%90related%20parameters%20of%20rice%20(Oryza%20sativa%20L.)%20plants&amp;author=A.%20R.%20Mohammed&amp;author=L.%20Tarpley&amp;volume=33&amp;issue=2&amp;publication_year=2010&amp;pages=117-123&amp;doi=10.1016/j.eja.2009.11.006&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270075-bib-0087">
<cite>

Narayanan, S.
, 
Prasad, P. V. V.
, 
Fritz, A. K.
, 
Boyle, D. L.
, &amp; 
Gill, B. S.
 (2015). Impact of high night‐time and high daytime temperature stress on winter wheat. Journal of Agronomy and Crop Science, 201(3), 206–218. 10.1111/jac.12101
</cite> [<a href="https://doi.org/10.1111/jac.12101" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Agronomy%20and%20Crop%20Science&amp;title=Impact%20of%20high%20night%E2%80%90time%20and%20high%20daytime%20temperature%20stress%20on%20winter%20wheat&amp;author=S.%20Narayanan&amp;author=P.%20V.%20V.%20Prasad&amp;author=A.%20K.%20Fritz&amp;author=D.%20L.%20Boyle&amp;author=B.%20S.%20Gill&amp;volume=201&amp;issue=3&amp;publication_year=2015&amp;pages=206-218&amp;doi=10.1111/jac.12101&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270075-bib-0088">
<cite>

Ogunbode, C. A.
, 
Doran, R.
, &amp; 
Böhm, G.
 (2020). Exposure to the IPCC special report on 1.5 C global warming is linked to perceived threat and increased concern about climate change. Climatic Change, 158, 361–375. 10.1007/s10584-019-02609-0
</cite> [<a href="https://doi.org/10.1007/s10584-019-02609-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Climatic%20Change&amp;title=Exposure%20to%20the%20IPCC%20special%20report%20on%201.5%20C%20global%20warming%20is%20linked%20to%20perceived%20threat%20and%20increased%20concern%20about%20climate%20change&amp;author=C.%20A.%20Ogunbode&amp;author=R.%20Doran&amp;author=G.%20B%C3%B6hm&amp;volume=158&amp;publication_year=2020&amp;pages=361-375&amp;doi=10.1007/s10584-019-02609-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270075-bib-0089">
<cite>

Pang, Y.
, 
Liu, C.
, 
Wang, D.
, 
Amand, P. S.
, 
Bernardo, A.
, 
Li, W.
, 
He, F.
, 
Li, L.
, 
Wang, L.
, 
Yuan, X.
, &amp; 
Liu, S.
 (2020). High‐resolution genome‐wide association study identifies genomic regions and candidate genes for important agronomic traits in wheat. Molecular Plant, 13(9), 1311–1327. 10.1016/j.molp.2020.07.008

</cite> [<a href="https://doi.org/10.1016/j.molp.2020.07.008" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32702458/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Plant&amp;title=High%E2%80%90resolution%20genome%E2%80%90wide%20association%20study%20identifies%20genomic%20regions%20and%20candidate%20genes%20for%20important%20agronomic%20traits%20in%20wheat&amp;author=Y.%20Pang&amp;author=C.%20Liu&amp;author=D.%20Wang&amp;author=P.%20S.%20Amand&amp;author=A.%20Bernardo&amp;volume=13&amp;issue=9&amp;publication_year=2020&amp;pages=1311-1327&amp;pmid=32702458&amp;doi=10.1016/j.molp.2020.07.008&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270075-bib-0090">
<cite>

Paraiso, F.
, 
Lin, H.
, 
Li, C.
, 
Woods, D. P.
, 
Lan, T.
, 
Tumelty, C.
, 
Debernardi, J. M.
, 
Joe, A.
, &amp; 
Dubcovsky, J.
 (2024). 
<em>LEAFY</em> and <em>WAPO1</em> jointly regulate spikelet number per spike and floret development in wheat. Development, 151(15), dev202803. 10.1242/dev.202803

</cite> [<a href="https://doi.org/10.1242/dev.202803" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11317094/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39082949/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Development&amp;title=LEAFY%20and%20WAPO1%20jointly%20regulate%20spikelet%20number%20per%20spike%20and%20floret%20development%20in%20wheat&amp;author=F.%20Paraiso&amp;author=H.%20Lin&amp;author=C.%20Li&amp;author=D.%20P.%20Woods&amp;author=T.%20Lan&amp;volume=151&amp;issue=15&amp;publication_year=2024&amp;pages=dev202803&amp;pmid=39082949&amp;doi=10.1242/dev.202803&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270075-bib-0091">
<cite>

Parveen, S.
, 
Rudra, S. G.
, 
Singh, B.
, &amp; 
Anand, A.
 (2022). Impact of high night temperature on yield and pasting properties of flour in early and late‐maturing wheat genotypes. Plants, 11(22), 3096. 10.3390/plants11223096

</cite> [<a href="https://doi.org/10.3390/plants11223096" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9693585/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36432825/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plants&amp;title=Impact%20of%20high%20night%20temperature%20on%20yield%20and%20pasting%20properties%20of%20flour%20in%20early%20and%20late%E2%80%90maturing%20wheat%20genotypes&amp;author=S.%20Parveen&amp;author=S.%20G.%20Rudra&amp;author=B.%20Singh&amp;author=A.%20Anand&amp;volume=11&amp;issue=22&amp;publication_year=2022&amp;pages=3096&amp;pmid=36432825&amp;doi=10.3390/plants11223096&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270075-bib-0092">
<cite>

Patil, I.
 (2021). Visualizations with statistical details: The ‘ggstatsplot’ approach. Journal of Open Source Software, 6(61), 3167. 10.21105/joss.03167
</cite> [<a href="https://doi.org/10.21105/joss.03167" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Open%20Source%20Software&amp;title=Visualizations%20with%20statistical%20details:%20The%20%E2%80%98ggstatsplot%E2%80%99%20approach&amp;author=I.%20Patil&amp;volume=6&amp;issue=61&amp;publication_year=2021&amp;pages=3167&amp;doi=10.21105/joss.03167&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270075-bib-0093">
<cite>

Peng, J.
, 
Richards, D. E.
, 
Hartley, N. M.
, 
Murphy, G. P.
, 
Devos, K. M.
, 
Flintham, J. E.
, 
Beales, J.
, 
Fish, L. J.
, 
Worland, A. J.
, 
Pelica, F.
, &amp; 
Harberd, N. P.
 (1999). Green revolution’ genes encode mutant gibberellin response modulators. Nature, 400(6741), 256–261. 10.1038/22307

</cite> [<a href="https://doi.org/10.1038/22307" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10421366/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Green%20revolution%E2%80%99%20genes%20encode%20mutant%20gibberellin%20response%20modulators&amp;author=J.%20Peng&amp;author=D.%20E.%20Richards&amp;author=N.%20M.%20Hartley&amp;author=G.%20P.%20Murphy&amp;author=K.%20M.%20Devos&amp;volume=400&amp;issue=6741&amp;publication_year=1999&amp;pages=256-261&amp;pmid=10421366&amp;doi=10.1038/22307&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270075-bib-0094">
<cite>

Peng, S.
, 
Huang, J.
, 
Sheehy, J. E.
, 
Laza, R. C.
, 
Visperas, R. M.
, 
Zhong, X.
, 
Centeno, G. S.
, 
Khush, G. S.
, &amp; 
Cassman, K. G.
 (2004). Rice yields decline with higher night temperature from global warming. Proceedings of the National Academy of Sciences, 101(27), 9971–9975. 10.1073/pnas.0403720101
</cite> [<a href="https://doi.org/10.1073/pnas.0403720101" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC454199/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15226500/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proceedings%20of%20the%20National%20Academy%20of%20Sciences&amp;title=Rice%20yields%20decline%20with%20higher%20night%20temperature%20from%20global%20warming&amp;author=S.%20Peng&amp;author=J.%20Huang&amp;author=J.%20E.%20Sheehy&amp;author=R.%20C.%20Laza&amp;author=R.%20M.%20Visperas&amp;volume=101&amp;issue=27&amp;publication_year=2004&amp;pages=9971-9975&amp;pmid=15226500&amp;doi=10.1073/pnas.0403720101&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270075-bib-0095">
<cite>

Philipp, N.
, 
Weichert, H.
, 
Bohra, U.
, 
Weschke, W.
, 
Schulthess, A. W.
, &amp; 
Weber, H.
 (2018). Grain number and grain yield distribution along the spike remain stable despite breeding for high yield in winter wheat. PLoS One, 13(10), e0205452. 10.1371/journal.pone.0205452

</cite> [<a href="https://doi.org/10.1371/journal.pone.0205452" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6179273/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30304020/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20One&amp;title=Grain%20number%20and%20grain%20yield%20distribution%20along%20the%20spike%20remain%20stable%20despite%20breeding%20for%20high%20yield%20in%20winter%20wheat&amp;author=N.%20Philipp&amp;author=H.%20Weichert&amp;author=U.%20Bohra&amp;author=W.%20Weschke&amp;author=A.%20W.%20Schulthess&amp;volume=13&amp;issue=10&amp;publication_year=2018&amp;pages=e0205452&amp;pmid=30304020&amp;doi=10.1371/journal.pone.0205452&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270075-bib-0096">
<cite>

Pinto, R. S.
, 
Reynolds, M. P.
, 
Mathews, K. L.
, 
McIntyre, C. L.
, 
Olivares‐Villegas, J. J.
, &amp; 
Chapman, S. C.
 (2010). Heat and drought adaptive QTL in a wheat population designed to minimize confounding agronomic effects. Theoretical and Applied Genetics, 121, 1001–1021. 10.1007/s00122-010-1351-4

</cite> [<a href="https://doi.org/10.1007/s00122-010-1351-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2938441/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20523964/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Theoretical%20and%20Applied%20Genetics&amp;title=Heat%20and%20drought%20adaptive%20QTL%20in%20a%20wheat%20population%20designed%20to%20minimize%20confounding%20agronomic%20effects&amp;author=R.%20S.%20Pinto&amp;author=M.%20P.%20Reynolds&amp;author=K.%20L.%20Mathews&amp;author=C.%20L.%20McIntyre&amp;author=J.%20J.%20Olivares%E2%80%90Villegas&amp;volume=121&amp;publication_year=2010&amp;pages=1001-1021&amp;pmid=20523964&amp;doi=10.1007/s00122-010-1351-4&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270075-bib-0097">
<cite>

Posch, B. C.
, 
Zhai, D.
, 
Coast, O.
, 
Scafaro, A. P.
, 
Bramley, H.
, 
Reich, P.
, 
Ruan, Y. L.
, 
Trethowan, R.
, 
Way, D. A.
, &amp; 
Atkin, O.
 (2022). Wheat respiratory O2 consumption falls with night warming alongside greater respiratory CO<sub>2</sub> loss and reduced biomass. Journal of Experimental Botany, 73(3), 915–926. 10.1093/jxb/erab454

</cite> [<a href="https://doi.org/10.1093/jxb/erab454" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34652413/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Experimental%20Botany&amp;title=Wheat%20respiratory%20O2%20consumption%20falls%20with%20night%20warming%20alongside%20greater%20respiratory%20CO2%20loss%20and%20reduced%20biomass&amp;author=B.%20C.%20Posch&amp;author=D.%20Zhai&amp;author=O.%20Coast&amp;author=A.%20P.%20Scafaro&amp;author=H.%20Bramley&amp;volume=73&amp;issue=3&amp;publication_year=2022&amp;pages=915-926&amp;pmid=34652413&amp;doi=10.1093/jxb/erab454&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270075-bib-0098">
<cite>

Pour‐Aboughadareh, A.
, 
Yousefian, M.
, 
Moradkhani, H.
, 
Moghaddam Vahed, M.
, 
Poczai, P.
, &amp; 
Siddique, K. H.
 (2019). iPASTIC: An online toolkit to estimate plant abiotic stress indices. Applications in Plant Sciences, 7(7), e11278. 10.1002/aps3.11278

</cite> [<a href="https://doi.org/10.1002/aps3.11278" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6636621/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31346510/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Applications%20in%20Plant%20Sciences&amp;title=iPASTIC:%20An%20online%20toolkit%20to%20estimate%20plant%20abiotic%20stress%20indices&amp;author=A.%20Pour%E2%80%90Aboughadareh&amp;author=M.%20Yousefian&amp;author=H.%20Moradkhani&amp;author=M.%20Moghaddam%20Vahed&amp;author=P.%20Poczai&amp;volume=7&amp;issue=7&amp;publication_year=2019&amp;pages=e11278&amp;pmid=31346510&amp;doi=10.1002/aps3.11278&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270075-bib-0099">
<cite>

Prasad, P. V.
, 
Pisipati, S. R.
, 
Ristic, Z.
, 
Bukovnik, U. R. S. K. A.
, &amp; 
Fritz, A. K.
 (2008). Impact of nighttime temperature on physiology and growth of spring wheat. Crop Science, 48(6), 2372–2380. 10.2135/cropsci2007.12.0717
</cite> [<a href="https://doi.org/10.2135/cropsci2007.12.0717" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Crop%20Science&amp;title=Impact%20of%20nighttime%20temperature%20on%20physiology%20and%20growth%20of%20spring%20wheat&amp;author=P.%20V.%20Prasad&amp;author=S.%20R.%20Pisipati&amp;author=Z.%20Ristic&amp;author=U.%20R.%20S.%20K.%20A.%20Bukovnik&amp;author=A.%20K.%20Fritz&amp;volume=48&amp;issue=6&amp;publication_year=2008&amp;pages=2372-2380&amp;doi=10.2135/cropsci2007.12.0717&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270075-bib-0100">
<cite>

Price, A. L.
, 
Patterson, N. J.
, 
Plenge, R. M.
, 
Weinblatt, M. E.
, 
Shadick, N. A.
, &amp; 
Reich, D.
 (2006). Principal components analysis corrects for stratification in genome‐wide association studies. Nature Genetics, 38(8), 904–909. 10.1038/ng1847

</cite> [<a href="https://doi.org/10.1038/ng1847" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16862161/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Genetics&amp;title=Principal%20components%20analysis%20corrects%20for%20stratification%20in%20genome%E2%80%90wide%20association%20studies&amp;author=A.%20L.%20Price&amp;author=N.%20J.%20Patterson&amp;author=R.%20M.%20Plenge&amp;author=M.%20E.%20Weinblatt&amp;author=N.%20A.%20Shadick&amp;volume=38&amp;issue=8&amp;publication_year=2006&amp;pages=904-909&amp;pmid=16862161&amp;doi=10.1038/ng1847&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270075-bib-0101">
<cite>

Pritchard, J. K.
, 
Stephens, M.
, &amp; 
Donnelly, P.
 (2000). Inference of population structure using multilocus genotype data. Genetics, 155(2), 945–959. 10.1093/genetics/155.2.945

</cite> [<a href="https://doi.org/10.1093/genetics/155.2.945" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1461096/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10835412/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genetics&amp;title=Inference%20of%20population%20structure%20using%20multilocus%20genotype%20data&amp;author=J.%20K.%20Pritchard&amp;author=M.%20Stephens&amp;author=P.%20Donnelly&amp;volume=155&amp;issue=2&amp;publication_year=2000&amp;pages=945-959&amp;pmid=10835412&amp;doi=10.1093/genetics/155.2.945&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270075-bib-0102">
<cite>
R Core Team
. (2024). R: A language and environment for statistical computing. R Foundation for Statistical Computing. <a href="https://www.R-project.org/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.R‐project.org/</a>
</cite> [<a href="https://scholar.google.com/scholar_lookup?title=R:%20A%20language%20and%20environment%20for%20statistical%20computing&amp;publication_year=2024&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270075-bib-0103">
<cite>

Ray, D. K.
, 
Mueller, N. D.
, 
West, P. C.
, &amp; 
Foley, J. A.
 (2013). Yield trends are insufficient to double global crop production by 2050. PLoS One, 8(6), e66428. 10.1371/journal.pone.0066428

</cite> [<a href="https://doi.org/10.1371/journal.pone.0066428" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3686737/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23840465/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20One&amp;title=Yield%20trends%20are%20insufficient%20to%20double%20global%20crop%20production%20by%202050&amp;author=D.%20K.%20Ray&amp;author=N.%20D.%20Mueller&amp;author=P.%20C.%20West&amp;author=J.%20A.%20Foley&amp;volume=8&amp;issue=6&amp;publication_year=2013&amp;pages=e66428&amp;pmid=23840465&amp;doi=10.1371/journal.pone.0066428&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270075-bib-0104">
<cite>

Raza, A.
, 
Razzaq, A.
, 
Mehmood, S. S.
, 
Zou, X.
, 
Zhang, X.
, 
Lv, Y.
, &amp; 
Xu, J.
 (2019). Impact of climate change on crops adaptation and strategies to tackle its outcome: A review. Plants, 8(2), 34. 10.3390/plants8020034

</cite> [<a href="https://doi.org/10.3390/plants8020034" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6409995/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30704089/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plants&amp;title=Impact%20of%20climate%20change%20on%20crops%20adaptation%20and%20strategies%20to%20tackle%20its%20outcome:%20A%20review&amp;author=A.%20Raza&amp;author=A.%20Razzaq&amp;author=S.%20S.%20Mehmood&amp;author=X.%20Zou&amp;author=X.%20Zhang&amp;volume=8&amp;issue=2&amp;publication_year=2019&amp;pages=34&amp;pmid=30704089&amp;doi=10.3390/plants8020034&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270075-bib-0105">
<cite>

Reddy, S. S.
, 
Saini, D. K.
, 
Singh, G. M.
, 
Sharma, S.
, 
Mishra, V. K.
, &amp; 
Joshi, A. K.
 (2023). Genome‐wide association mapping of genomic regions associated with drought stress tolerance at seedling and reproductive stages in bread wheat. Frontiers in Plant Science, 14, 1166439. 10.3389/fpls.2023.1166439

</cite> [<a href="https://doi.org/10.3389/fpls.2023.1166439" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10213333/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37251775/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Plant%20Science&amp;title=Genome%E2%80%90wide%20association%20mapping%20of%20genomic%20regions%20associated%20with%20drought%20stress%20tolerance%20at%20seedling%20and%20reproductive%20stages%20in%20bread%20wheat&amp;author=S.%20S.%20Reddy&amp;author=D.%20K.%20Saini&amp;author=G.%20M.%20Singh&amp;author=S.%20Sharma&amp;author=V.%20K.%20Mishra&amp;volume=14&amp;publication_year=2023&amp;pages=1166439&amp;pmid=37251775&amp;doi=10.3389/fpls.2023.1166439&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270075-bib-0106">
<cite>

Sadok, W.
, &amp; 
Jagadish, S. K.
 (2020). The hidden costs of nighttime warming on yields. Trends in Plant Science, 25(7), 644–651. 10.1016/j.tplants.2020.02.003

</cite> [<a href="https://doi.org/10.1016/j.tplants.2020.02.003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32526169/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Trends%20in%20Plant%20Science&amp;title=The%20hidden%20costs%20of%20nighttime%20warming%20on%20yields&amp;author=W.%20Sadok&amp;author=S.%20K.%20Jagadish&amp;volume=25&amp;issue=7&amp;publication_year=2020&amp;pages=644-651&amp;pmid=32526169&amp;doi=10.1016/j.tplants.2020.02.003&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270075-bib-0107">
<cite>

Saini, D. K.
, 
Bahuguna, R. N.
, 
Pal, M.
, 
Chaturvedi, A. K.
, &amp; 
Krishna Jagadish, S. V.
 (2025). Genome‐wide mapping, allelic fingerprinting, and haplotypes validation provide insights into the genetic control of phenotypic plasticity in rice. Plant, Cell &amp; Environment. 10.1111/pce.15477
</cite> [<a href="https://doi.org/10.1111/pce.15477" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40108857/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant,%20Cell%20&amp;%20Environment&amp;title=Genome%E2%80%90wide%20mapping,%20allelic%20fingerprinting,%20and%20haplotypes%20validation%20provide%20insights%20into%20the%20genetic%20control%20of%20phenotypic%20plasticity%20in%20rice&amp;author=D.%20K.%20Saini&amp;author=R.%20N.%20Bahuguna&amp;author=M.%20Pal&amp;author=A.%20K.%20Chaturvedi&amp;author=S.%20V.%20Krishna%20Jagadish&amp;publication_year=2025&amp;pmid=40108857&amp;doi=10.1111/pce.15477&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270075-bib-0108">
<cite>

Saini, D. K.
, 
Chopra, Y.
, 
Singh, J.
, 
Sandhu, K. S.
, 
Kumar, A.
, 
Bazzer, S.
, &amp; 
Srivastava, P.
 (2022). Comprehensive evaluation of mapping complex traits in wheat using genome‐wide association studies. Molecular Breeding, 42, Article 1. 10.1007/s11032-021-01272-7

</cite> [<a href="https://doi.org/10.1007/s11032-021-01272-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10248672/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37309486/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Breeding&amp;title=Comprehensive%20evaluation%20of%20mapping%20complex%20traits%20in%20wheat%20using%20genome%E2%80%90wide%20association%20studies&amp;author=D.%20K.%20Saini&amp;author=Y.%20Chopra&amp;author=J.%20Singh&amp;author=K.%20S.%20Sandhu&amp;author=A.%20Kumar&amp;volume=42&amp;publication_year=2022&amp;pages=1&amp;pmid=37309486&amp;doi=10.1007/s11032-021-01272-7&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270075-bib-0109">
<cite>

Schaarschmidt, S.
, 
Lawas, L. M. F.
, 
Glaubitz, U.
, 
Li, X.
, 
Erban, A.
, 
Kopka, J.
, 
Jagadish, S. K.
, 
Hincha, D. K.
, &amp; 
Zuther, E.
 (2020). Season affects yield and metabolic profiles of rice (<em>Oryza sativa</em>) under high night temperature stress in the field. International Journal of Molecular Sciences, 21(9), 3187. 10.3390/ijms21093187

</cite> [<a href="https://doi.org/10.3390/ijms21093187" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7247591/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32366031/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=International%20Journal%20of%20Molecular%20Sciences&amp;title=Season%20affects%20yield%20and%20metabolic%20profiles%20of%20rice%20(Oryza%20sativa)%20under%20high%20night%20temperature%20stress%20in%20the%20field&amp;author=S.%20Schaarschmidt&amp;author=L.%20M.%20F.%20Lawas&amp;author=U.%20Glaubitz&amp;author=X.%20Li&amp;author=A.%20Erban&amp;volume=21&amp;issue=9&amp;publication_year=2020&amp;pages=3187&amp;pmid=32366031&amp;doi=10.3390/ijms21093187&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270075-bib-0110">
<cite>

Segura, V.
, 
Vilhjálmsson, B. J.
, 
Platt, A.
, 
Korte, A.
, 
Seren, Ü.
, 
Long, Q.
, &amp; 
Nordborg, M.
 (2012). An efficient multi‐locus mixed‐model approach for genome‐wide association studies in structured populations. Nature Genetics, 44(7), 825–830. 10.1038/ng.2314

</cite> [<a href="https://doi.org/10.1038/ng.2314" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3386481/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22706313/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Genetics&amp;title=An%20efficient%20multi%E2%80%90locus%20mixed%E2%80%90model%20approach%20for%20genome%E2%80%90wide%20association%20studies%20in%20structured%20populations&amp;author=V.%20Segura&amp;author=B.%20J.%20Vilhj%C3%A1lmsson&amp;author=A.%20Platt&amp;author=A.%20Korte&amp;author=%C3%9C.%20Seren&amp;volume=44&amp;issue=7&amp;publication_year=2012&amp;pages=825-830&amp;pmid=22706313&amp;doi=10.1038/ng.2314&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270075-bib-0111">
<cite>

Sertse, D.
, 
You, F. M.
, 
Klymiuk, V.
, 
Haile, J. K.
, 
N'Diaye, A.
, 
Pozniak, C. J.
, 
Cloutier, S.
, &amp; 
Kagale, S.
 (2023). Historical selection, adaptation signatures, and ambiguity of introgressions in wheat. International Journal of Molecular Sciences, 24(9), 8390. 10.3390/ijms24098390

</cite> [<a href="https://doi.org/10.3390/ijms24098390" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10179502/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37176097/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=International%20Journal%20of%20Molecular%20Sciences&amp;title=Historical%20selection,%20adaptation%20signatures,%20and%20ambiguity%20of%20introgressions%20in%20wheat&amp;author=D.%20Sertse&amp;author=F.%20M.%20You&amp;author=V.%20Klymiuk&amp;author=J.%20K.%20Haile&amp;author=A.%20N'Diaye&amp;volume=24&amp;issue=9&amp;publication_year=2023&amp;pages=8390&amp;pmid=37176097&amp;doi=10.3390/ijms24098390&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270075-bib-0112">
<cite>

Sharma, N.
, 
Saini, D. K.
, 
Pushkar, S.
, 
Somayanda, I.
, 
Jagadish, S. K.
, &amp; 
Anand, A.
 (2025). Reprogramming assimilate partitioning in the second half of the night supports grain filling in inferior spikelets under high night temperature stress in rice. Plant Stress, 15, 100773. 10.1016/j.stress.2025.100773
</cite> [<a href="https://doi.org/10.1016/j.stress.2025.100773" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Stress&amp;title=Reprogramming%20assimilate%20partitioning%20in%20the%20second%20half%20of%20the%20night%20supports%20grain%20filling%20in%20inferior%20spikelets%20under%20high%20night%20temperature%20stress%20in%20rice&amp;author=N.%20Sharma&amp;author=D.%20K.%20Saini&amp;author=S.%20Pushkar&amp;author=I.%20Somayanda&amp;author=S.%20K.%20Jagadish&amp;volume=15&amp;publication_year=2025&amp;pages=100773&amp;doi=10.1016/j.stress.2025.100773&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270075-bib-0113">
<cite>

Shi, W.
, 
Muthurajan, R.
, 
Rahman, H.
, 
Selvam, J.
, 
Peng, S.
, 
Zou, Y.
, &amp; 
Jagadish, K. S.
 (2013). Source–sink dynamics and proteomic reprogramming under elevated night temperature and their impact on rice yield and grain quality. New Phytologist, 197(3), 825–837. 10.1111/nph.12088

</cite> [<a href="https://doi.org/10.1111/nph.12088" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23252708/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=New%20Phytologist&amp;title=Source%E2%80%93sink%20dynamics%20and%20proteomic%20reprogramming%20under%20elevated%20night%20temperature%20and%20their%20impact%20on%20rice%20yield%20and%20grain%20quality&amp;author=W.%20Shi&amp;author=R.%20Muthurajan&amp;author=H.%20Rahman&amp;author=J.%20Selvam&amp;author=S.%20Peng&amp;volume=197&amp;issue=3&amp;publication_year=2013&amp;pages=825-837&amp;pmid=23252708&amp;doi=10.1111/nph.12088&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270075-bib-0114">
<cite>

Shi, W.
, 
Yin, X.
, 
Struik, P. C.
, 
Solis, C.
, 
Xie, F.
, 
Schmidt, R. C.
, 
Huang, M.
, 
Zou, Y.
, 
Ye, C.
, &amp; 
Jagadish, S. K.
 (2017). High day‐and night‐time temperatures affect grain growth dynamics in contrasting rice genotypes. Journal of Experimental Botany, 68(18), 5233–5245. 10.1093/jxb/erx344

</cite> [<a href="https://doi.org/10.1093/jxb/erx344" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5853565/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29106621/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Experimental%20Botany&amp;title=High%20day%E2%80%90and%20night%E2%80%90time%20temperatures%20affect%20grain%20growth%20dynamics%20in%20contrasting%20rice%20genotypes&amp;author=W.%20Shi&amp;author=X.%20Yin&amp;author=P.%20C.%20Struik&amp;author=C.%20Solis&amp;author=F.%20Xie&amp;volume=68&amp;issue=18&amp;publication_year=2017&amp;pages=5233-5245&amp;pmid=29106621&amp;doi=10.1093/jxb/erx344&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270075-bib-0115">
<cite>

Sillmann, J.
, 
Kharin, V. V.
, 
Zhang, X.
, 
Zwiers, F. W.
, &amp; 
Bronaugh, D.
 (2013). Climate extremes indices in the CMIP5 multimodel ensemble: Part 1. Model evaluation in the present climate. Journal of Geophysical Research: Atmospheres, 118(4), 1716–1733. 10.1002/jgrd.50203
</cite> [<a href="https://doi.org/10.1002/jgrd.50203" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Geophysical%20Research:%20Atmospheres&amp;title=Climate%20extremes%20indices%20in%20the%20CMIP5%20multimodel%20ensemble:%20Part%201.%20Model%20evaluation%20in%20the%20present%20climate&amp;author=J.%20Sillmann&amp;author=V.%20V.%20Kharin&amp;author=X.%20Zhang&amp;author=F.%20W.%20Zwiers&amp;author=D.%20Bronaugh&amp;volume=118&amp;issue=4&amp;publication_year=2013&amp;pages=1716-1733&amp;doi=10.1002/jgrd.50203&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270075-bib-0116">
<cite>

Sillmann, J.
, 
Kharin, V. V.
, 
Zwiers, F. W.
, 
Zhang, X.
, &amp; 
Bronaugh, D.
 (2013). Climate extremes indices in the CMIP5 multimodel ensemble: Part 2. Future climate projections. Journal of Geophysical Research: Atmospheres, 118(6), 2473–2493. 10.1002/jgrd.50188
</cite> [<a href="https://doi.org/10.1002/jgrd.50188" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Geophysical%20Research:%20Atmospheres&amp;title=Climate%20extremes%20indices%20in%20the%20CMIP5%20multimodel%20ensemble:%20Part%202.%20Future%20climate%20projections&amp;author=J.%20Sillmann&amp;author=V.%20V.%20Kharin&amp;author=F.%20W.%20Zwiers&amp;author=X.%20Zhang&amp;author=D.%20Bronaugh&amp;volume=118&amp;issue=6&amp;publication_year=2013&amp;pages=2473-2493&amp;doi=10.1002/jgrd.50188&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270075-bib-0117">
<cite>

Simmonds, J.
, 
Scott, P.
, 
Leverington‐Waite, M.
, 
Turner, A. S.
, 
Brinton, J.
, 
Korzun, V.
, 
Snape, J.
, &amp; 
Uauy, C.
 (2014). Identification and independent validation of a stable yield and thousand grain weight QTL on chromosome 6A of hexaploid wheat (<em>Triticum aestivum</em> L.). BMC Plant Biology, 14, Article 191. 10.1186/s12870-014-0191-9

</cite> [<a href="https://doi.org/10.1186/s12870-014-0191-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4105860/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25034643/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Plant%20Biology&amp;title=Identification%20and%20independent%20validation%20of%20a%20stable%20yield%20and%20thousand%20grain%20weight%20QTL%20on%20chromosome%206A%20of%20hexaploid%20wheat%20(Triticum%20aestivum%20L.)&amp;author=J.%20Simmonds&amp;author=P.%20Scott&amp;author=M.%20Leverington%E2%80%90Waite&amp;author=A.%20S.%20Turner&amp;author=J.%20Brinton&amp;volume=14&amp;publication_year=2014&amp;pages=191&amp;pmid=25034643&amp;doi=10.1186/s12870-014-0191-9&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270075-bib-0118">
<cite>

Singh, K.
, 
Saini, D. K.
, 
Saripalli, G.
, 
Batra, R.
, 
Gautam, T.
, 
Singh, R.
, 
Pal, S.
, 
Kumar, M.
, 
Jan, I.
, 
Singh, S.
, &amp; 
Gupta, P. K.
 (2022). WheatQTLdb V2. 0: A supplement to the database for wheat QTL. Molecular Breeding, 42(10), Article 56. 10.1007/s11032-022-01329-1

</cite> [<a href="https://doi.org/10.1007/s11032-022-01329-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10248696/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37313017/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Breeding&amp;title=WheatQTLdb%20V2.%200:%20A%20supplement%20to%20the%20database%20for%20wheat%20QTL&amp;author=K.%20Singh&amp;author=D.%20K.%20Saini&amp;author=G.%20Saripalli&amp;author=R.%20Batra&amp;author=T.%20Gautam&amp;volume=42&amp;issue=10&amp;publication_year=2022&amp;pages=56&amp;pmid=37313017&amp;doi=10.1007/s11032-022-01329-1&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270075-bib-0119">
<cite>

Taagen, E.
, 
Jordan, K.
, 
Akhunov, E.
, 
Sorrells, M. E.
, &amp; 
Jannink, J. L.
 (2022). If it ain't broke, don't fix it: Evaluating the effect of increased recombination on response to selection for wheat breeding. G3 Genes|Genomes|Genetics, 12(12), jkac291. 10.1093/g3journal/jkac291

</cite> [<a href="https://doi.org/10.1093/g3journal/jkac291" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9713416/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36331396/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=G3%20Genes%7CGenomes%7CGenetics&amp;title=If%20it%20ain't%20broke,%20don't%20fix%20it:%20Evaluating%20the%20effect%20of%20increased%20recombination%20on%20response%20to%20selection%20for%20wheat%20breeding&amp;author=E.%20Taagen&amp;author=K.%20Jordan&amp;author=E.%20Akhunov&amp;author=M.%20E.%20Sorrells&amp;author=J.%20L.%20Jannink&amp;volume=12&amp;issue=12&amp;publication_year=2022&amp;pages=jkac291&amp;pmid=36331396&amp;doi=10.1093/g3journal/jkac291&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270075-bib-0120">
<cite>

Ullah, A.
, 
Nadeem, F.
, 
Nawaz, A.
, 
Siddique, K. H.
, &amp; 
Farooq, M.
 (2022). Heat stress effects on the reproductive physiology and yield of wheat. Journal of Agronomy and Crop Science, 208(1), 1–17. 10.1111/jac.12572
</cite> [<a href="https://doi.org/10.1111/jac.12572" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Agronomy%20and%20Crop%20Science&amp;title=Heat%20stress%20effects%20on%20the%20reproductive%20physiology%20and%20yield%20of%20wheat&amp;author=A.%20Ullah&amp;author=F.%20Nadeem&amp;author=A.%20Nawaz&amp;author=K.%20H.%20Siddique&amp;author=M.%20Farooq&amp;volume=208&amp;issue=1&amp;publication_year=2022&amp;pages=1-17&amp;doi=10.1111/jac.12572&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270075-bib-0121">
<cite>

Varshney, R. K.
, 
Bohra, A.
, 
Roorkiwal, M.
, 
Barmukh, R.
, 
Cowling, W. A.
, 
Chitikineni, A.
, 
Lam, H. M.
, 
Hickey, L. T.
, 
Croser, J. S.
, 
Bayer, P. E.
, &amp; 
Siddique, K. H.
 (2021). Fast‐forward breeding for a food‐secure world. Trends in Genetics, 37(12), 1124–1136. 10.1016/j.tig.2021.08.002

</cite> [<a href="https://doi.org/10.1016/j.tig.2021.08.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34531040/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Trends%20in%20Genetics&amp;title=Fast%E2%80%90forward%20breeding%20for%20a%20food%E2%80%90secure%20world&amp;author=R.%20K.%20Varshney&amp;author=A.%20Bohra&amp;author=M.%20Roorkiwal&amp;author=R.%20Barmukh&amp;author=W.%20A.%20Cowling&amp;volume=37&amp;issue=12&amp;publication_year=2021&amp;pages=1124-1136&amp;pmid=34531040&amp;doi=10.1016/j.tig.2021.08.002&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270075-bib-0122">
<cite>

Vincent, L. A.
, &amp; 
Mekis, E.
 (2006). Changes in daily and extreme temperature and precipitation indices for Canada over the twentieth century. Atmosphere‐Ocean, 44, 177–193. 10.3137/ao.440205
</cite> [<a href="https://doi.org/10.3137/ao.440205" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Atmosphere%E2%80%90Ocean&amp;title=Changes%20in%20daily%20and%20extreme%20temperature%20and%20precipitation%20indices%20for%20Canada%20over%20the%20twentieth%20century&amp;author=L.%20A.%20Vincent&amp;author=E.%20Mekis&amp;volume=44&amp;publication_year=2006&amp;pages=177-193&amp;doi=10.3137/ao.440205&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270075-bib-0123">
<cite>

Wang, D.
, 
Qin, B.
, 
Li, X.
, 
Tang, D.
, 
Zhang, Y. E.
, 
Cheng, Z.
, &amp; 
Xue, Y.
 (2016). Nucleolar DEAD‐box RNA helicase <em>TOGR1</em> regulates thermotolerant growth as a pre‐rRNA chaperone in rice. PLoS Genetics, 12(2), e1005844. 10.1371/journal.pgen.1005844

</cite> [<a href="https://doi.org/10.1371/journal.pgen.1005844" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4743921/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26848586/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20Genetics&amp;title=Nucleolar%20DEAD%E2%80%90box%20RNA%20helicase%20TOGR1%20regulates%20thermotolerant%20growth%20as%20a%20pre%E2%80%90rRNA%20chaperone%20in%20rice&amp;author=D.%20Wang&amp;author=B.%20Qin&amp;author=X.%20Li&amp;author=D.%20Tang&amp;author=Y.%20E.%20Zhang&amp;volume=12&amp;issue=2&amp;publication_year=2016&amp;pages=e1005844&amp;pmid=26848586&amp;doi=10.1371/journal.pgen.1005844&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270075-bib-0124">
<cite>

Wang, J.
, &amp; 
Zhang, Z.
 (2021). GAPIT version 3: Boosting power and accuracy for genomic association and prediction. Genomics, Proteomics and Bioinformatics, 19(4), 629–640. 10.1016/j.gpb.2021.08.005
</cite> [<a href="https://doi.org/10.1016/j.gpb.2021.08.005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9121400/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34492338/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genomics,%20Proteomics%20and%20Bioinformatics&amp;title=GAPIT%20version%203:%20Boosting%20power%20and%20accuracy%20for%20genomic%20association%20and%20prediction&amp;author=J.%20Wang&amp;author=Z.%20Zhang&amp;volume=19&amp;issue=4&amp;publication_year=2021&amp;pages=629-640&amp;pmid=34492338&amp;doi=10.1016/j.gpb.2021.08.005&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270075-bib-0125">
<cite>

Webb, W. P.
 (2022). The great plains. University of Nebraska Press.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=The%20great%20plains&amp;author=W.%20P.%20Webb&amp;publication_year=2022&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270075-bib-0126">
<cite>

Weber, S. E.
, 
Frisch, M.
, 
Snowdon, R. J.
, &amp; 
Voss‐Fels, K. P.
 (2023). Haplotype blocks for genomic prediction: A comparative evaluation in multiple crop datasets. Frontiers in Plant Science, 14, 1217589. 10.3389/fpls.2023.1217589

</cite> [<a href="https://doi.org/10.3389/fpls.2023.1217589" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10507710/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37731980/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Plant%20Science&amp;title=Haplotype%20blocks%20for%20genomic%20prediction:%20A%20comparative%20evaluation%20in%20multiple%20crop%20datasets&amp;author=S.%20E.%20Weber&amp;author=M.%20Frisch&amp;author=R.%20J.%20Snowdon&amp;author=K.%20P.%20Voss%E2%80%90Fels&amp;volume=14&amp;publication_year=2023&amp;pages=1217589&amp;pmid=37731980&amp;doi=10.3389/fpls.2023.1217589&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270075-bib-0127">
<cite>

Xiao, G.
, 
Zhang, Q.
, 
Li, Y.
, 
Wang, R.
, 
Yao, Y.
, 
Zhao, H.
, &amp; 
Bai, H.
 (2010). Impact of temperature increase on the yield of winter wheat at low and high altitudes in semiarid northwestern China. Agricultural Water Management, 97(9), 1360–1364. 10.1016/j.agwat.2010.04.002
</cite> [<a href="https://doi.org/10.1016/j.agwat.2010.04.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Agricultural%20Water%20Management&amp;title=Impact%20of%20temperature%20increase%20on%20the%20yield%20of%20winter%20wheat%20at%20low%20and%20high%20altitudes%20in%20semiarid%20northwestern%20China&amp;author=G.%20Xiao&amp;author=Q.%20Zhang&amp;author=Y.%20Li&amp;author=R.%20Wang&amp;author=Y.%20Yao&amp;volume=97&amp;issue=9&amp;publication_year=2010&amp;pages=1360-1364&amp;doi=10.1016/j.agwat.2010.04.002&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270075-bib-0128">
<cite>

Xiao, J.
, 
Xu, S.
, 
Li, C.
, 
Xu, Y.
, 
Xing, L.
, 
Niu, Y.
, 
Huan, Q.
, 
Tang, Y.
, 
Zhao, C.
, 
Wagner, D.
, &amp; 
Chong, K.
 (2014). O‐GlcNAc‐mediated interaction between <em>VER2</em> and <em>TaGRP2</em> elicits <em>TaVRN1</em> mRNA accumulation during vernalization in winter wheat. Nature Communications, 5(1), Article 4572. 10.1038/ncomms5572
</cite> [<a href="https://doi.org/10.1038/ncomms5572" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4143922/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25091017/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Communications&amp;title=O%E2%80%90GlcNAc%E2%80%90mediated%20interaction%20between%20VER2%20and%20TaGRP2%20elicits%20TaVRN1%20mRNA%20accumulation%20during%20vernalization%20in%20winter%20wheat&amp;author=J.%20Xiao&amp;author=S.%20Xu&amp;author=C.%20Li&amp;author=Y.%20Xu&amp;author=L.%20Xing&amp;volume=5&amp;issue=1&amp;publication_year=2014&amp;pages=4572&amp;pmid=25091017&amp;doi=10.1038/ncomms5572&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270075-bib-0129">
<cite>

Xu, J.
, 
Henry, A.
, &amp; 
Sreenivasulu, N.
 (2020). Rice yield formation under high day and night temperatures—A prerequisite to ensure future food security. Plant, Cell &amp; Environment, 43(7), 1595–1608. 10.1111/pce.13748
</cite> [<a href="https://doi.org/10.1111/pce.13748" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32112422/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant,%20Cell%20&amp;%20Environment&amp;title=Rice%20yield%20formation%20under%20high%20day%20and%20night%20temperatures%E2%80%94A%20prerequisite%20to%20ensure%20future%20food%20security&amp;author=J.%20Xu&amp;author=A.%20Henry&amp;author=N.%20Sreenivasulu&amp;volume=43&amp;issue=7&amp;publication_year=2020&amp;pages=1595-1608&amp;pmid=32112422&amp;doi=10.1111/pce.13748&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270075-bib-0130">
<cite>

Yang, C.
, 
Luo, A.
, 
Lu, H. P.
, 
Davis, S. J.
, &amp; 
Liu, J. X.
 (2024). Diurnal regulation of alternative splicing associated with thermotolerance in rice by two glycine‐rich RNA‐binding proteins. Science Bulletin, 69(1), 59–71. 10.1016/j.scib.2023.11.046

</cite> [<a href="https://doi.org/10.1016/j.scib.2023.11.046" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38044192/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science%20Bulletin&amp;title=Diurnal%20regulation%20of%20alternative%20splicing%20associated%20with%20thermotolerance%20in%20rice%20by%20two%20glycine%E2%80%90rich%20RNA%E2%80%90binding%20proteins&amp;author=C.%20Yang&amp;author=A.%20Luo&amp;author=H.%20P.%20Lu&amp;author=S.%20J.%20Davis&amp;author=J.%20X.%20Liu&amp;volume=69&amp;issue=1&amp;publication_year=2024&amp;pages=59-71&amp;pmid=38044192&amp;doi=10.1016/j.scib.2023.11.046&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270075-bib-0131">
<cite>

Yu, J.
, 
Pressoir, G.
, 
Briggs, W. H.
, 
Vroh Bi, I.
, 
Yamasaki, M.
, 
Doebley, J. F.
, 
McMullen, M. D.
, 
Gaut, B. S.
, 
Nielsen, D. M.
, 
Holland, J. B.
, &amp; 
Buckler, E. S.
 (2006). A unified mixed‐model method for association mapping that accounts for multiple levels of relatedness. Nature Genetics, 38(2), 203–208. 10.1038/ng1702

</cite> [<a href="https://doi.org/10.1038/ng1702" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16380716/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Genetics&amp;title=A%20unified%20mixed%E2%80%90model%20method%20for%20association%20mapping%20that%20accounts%20for%20multiple%20levels%20of%20relatedness&amp;author=J.%20Yu&amp;author=G.%20Pressoir&amp;author=W.%20H.%20Briggs&amp;author=I.%20Vroh%20Bi&amp;author=M.%20Yamasaki&amp;volume=38&amp;issue=2&amp;publication_year=2006&amp;pages=203-208&amp;pmid=16380716&amp;doi=10.1038/ng1702&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270075-bib-0132">
<cite>

Zhang, L.
, 
He, G.
, 
Li, Y.
, 
Yang, Z.
, 
Liu, T.
, 
Xie, X.
, 
Kong, X.
, &amp; 
Sun, J.
 (2022). PIL transcription factors directly interact with SPLs and repress tillering/branching in plants. New Phytologist, 233(3), 1414–1425. 10.1111/nph.17872

</cite> [<a href="https://doi.org/10.1111/nph.17872" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34800046/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=New%20Phytologist&amp;title=PIL%20transcription%20factors%20directly%20interact%20with%20SPLs%20and%20repress%20tillering/branching%20in%20plants&amp;author=L.%20Zhang&amp;author=G.%20He&amp;author=Y.%20Li&amp;author=Z.%20Yang&amp;author=T.%20Liu&amp;volume=233&amp;issue=3&amp;publication_year=2022&amp;pages=1414-1425&amp;pmid=34800046&amp;doi=10.1111/nph.17872&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270075-bib-0133">
<cite>

Zhang, X.
, 
Gonzalez‐Carranza, Z. H.
, 
Zhang, S.
, 
Miao, Y.
, 
Liu, C. J.
, &amp; 
Roberts, J. A.
 (2019). F‐Box proteins in plants. Annual Plant Review, 2, 307–328. 10.1002/9781119312994.apr0701
</cite> [<a href="https://doi.org/10.1002/9781119312994.apr0701" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Annual%20Plant%20Review&amp;title=F%E2%80%90Box%20proteins%20in%20plants&amp;author=X.%20Zhang&amp;author=Z.%20H.%20Gonzalez%E2%80%90Carranza&amp;author=S.%20Zhang&amp;author=Y.%20Miao&amp;author=C.%20J.%20Liu&amp;volume=2&amp;publication_year=2019&amp;pages=307-328&amp;doi=10.1002/9781119312994.apr0701&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270075-bib-0134">
<cite>

Zhang, X.
, 
Jia, H.
, 
Li, T.
, 
Wu, J.
, 
Nagarajan, R.
, 
Lei, L.
, 
Powers, C.
, 
Kan, C. C.
, 
Hua, W.
, 
Liu, Z.
, &amp; 
Yan, L.
 (2022). 
<em>TaCol‐B5</em> modifies spike architecture and enhances grain yield in wheat. Science, 376(6589), 180–183. 10.1126/science.abm0717

</cite> [<a href="https://doi.org/10.1126/science.abm0717" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35389775/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=TaCol%E2%80%90B5%20modifies%20spike%20architecture%20and%20enhances%20grain%20yield%20in%20wheat&amp;author=X.%20Zhang&amp;author=H.%20Jia&amp;author=T.%20Li&amp;author=J.%20Wu&amp;author=R.%20Nagarajan&amp;volume=376&amp;issue=6589&amp;publication_year=2022&amp;pages=180-183&amp;pmid=35389775&amp;doi=10.1126/science.abm0717&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270075-bib-0135">
<cite>

Zhang, Y. M.
, 
Jia, Z.
, &amp; 
Dunwell, J. M.
 (2019). The applications of new multi‐locus GWAS methodologies in the genetic dissection of complex traits. Frontiers in Plant Science, 10, 100. 10.3389/fpls.2019.00100

</cite> [<a href="https://doi.org/10.3389/fpls.2019.00100" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6378272/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30804969/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Plant%20Science&amp;title=The%20applications%20of%20new%20multi%E2%80%90locus%20GWAS%20methodologies%20in%20the%20genetic%20dissection%20of%20complex%20traits&amp;author=Y.%20M.%20Zhang&amp;author=Z.%20Jia&amp;author=J.%20M.%20Dunwell&amp;volume=10&amp;publication_year=2019&amp;pages=100&amp;pmid=30804969&amp;doi=10.3389/fpls.2019.00100&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270075-bib-0136">
<cite>

Zhang, Z.
, 
Ersoz, E.
, 
Lai, C. Q.
, 
Todhunter, R. J.
, 
Tiwari, H. K.
, 
Gore, M. A.
, 
Bradbury, P. J.
, 
Yu, J.
, 
Arnett, D. K.
, 
Ordovas, J. M.
, &amp; 
Buckler, E. S.
 (2010). Mixed linear model approach adapted for genome‐wide association studies. Nature Genetics, 42(4), 355–360. 10.1038/ng.546

</cite> [<a href="https://doi.org/10.1038/ng.546" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2931336/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20208535/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Genetics&amp;title=Mixed%20linear%20model%20approach%20adapted%20for%20genome%E2%80%90wide%20association%20studies&amp;author=Z.%20Zhang&amp;author=E.%20Ersoz&amp;author=C.%20Q.%20Lai&amp;author=R.%20J.%20Todhunter&amp;author=H.%20K.%20Tiwari&amp;volume=42&amp;issue=4&amp;publication_year=2010&amp;pages=355-360&amp;pmid=20208535&amp;doi=10.1038/ng.546&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270075-bib-0137">
<cite>

Zhao, C.
, 
Liu, B.
, 
Piao, S.
, 
Wang, X.
, 
Lobell, D. B.
, 
Huang, Y.
, 
Huang, M.
, 
Yao, Y.
, 
Bassu, S.
, 
Ciais, P.
, &amp; 
Asseng, S.
 (2017). Temperature increase reduces global yields of major crops in four independent estimates. Proceedings of the National Academy of Sciences, 114(35), 9326–9331. 10.1073/pnas.1701762114
</cite> [<a href="https://doi.org/10.1073/pnas.1701762114" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5584412/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28811375/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proceedings%20of%20the%20National%20Academy%20of%20Sciences&amp;title=Temperature%20increase%20reduces%20global%20yields%20of%20major%20crops%20in%20four%20independent%20estimates&amp;author=C.%20Zhao&amp;author=B.%20Liu&amp;author=S.%20Piao&amp;author=X.%20Wang&amp;author=D.%20B.%20Lobell&amp;volume=114&amp;issue=35&amp;publication_year=2017&amp;pages=9326-9331&amp;pmid=28811375&amp;doi=10.1073/pnas.1701762114&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270075-bib-0138">
<cite>

Zhu, X.
, 
Rong, W.
, 
Wang, K.
, 
Guo, W.
, 
Zhou, M.
, 
Wu, J.
, 
Ye, X.
, 
Wei, X.
, &amp; 
Zhang, Z.
 (2022). Overexpression of <em>TaSTT3b‐2B</em> improves resistance to sharp eyespot and increases grain weight in wheat. Plant Biotechnology Journal, 20(4), 777–793. 10.1111/pbi.13760

</cite> [<a href="https://doi.org/10.1111/pbi.13760" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8989504/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34873799/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Biotechnology%20Journal&amp;title=Overexpression%20of%20TaSTT3b%E2%80%902B%20improves%20resistance%20to%20sharp%20eyespot%20and%20increases%20grain%20weight%20in%20wheat&amp;author=X.%20Zhu&amp;author=W.%20Rong&amp;author=K.%20Wang&amp;author=W.%20Guo&amp;author=M.%20Zhou&amp;volume=20&amp;issue=4&amp;publication_year=2022&amp;pages=777-793&amp;pmid=34873799&amp;doi=10.1111/pbi.13760&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ol></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="caption p">
<p>
<strong>Figure S1</strong> Bar plot showing year‐wise reductions in trait values under HNT stress for biomass, spike number, and spike weight in wheat during the 2018–19 and 2019–20 seasons. Solid‐filled bars represent control conditions, while pattern‐filled bars indicate HNT stress. Percent reductions under HNT stress are annotated above each pair of bars. Trait values are averages across the genotypes, and percentage reductions were calculated using the formula: (Average trait value under control – Average trait value under HNT stress) / Average trait value under control × 100.</p>
<p>
<strong>Figure S2</strong> (A) 3D scatter plot of 261 wheat genotypes based on the first three principal components (PC1, PC2, and PC3). The axes represent PC1 (5.26% variance explained), PC2 (4.22% variance explained), and PC3 (3.47% variance explained). Each dot represents an accession. (B) Evanno plot of the delta‐K statistic revealing the number of sub‐populations in the association panel.</p>
<p>
<strong>Figure S3</strong> Expression patterns of high‐confidence genes across developmental stages, with values shown in TPM (Transcripts Per Million). Data was obtained from the WheatOmics database (<a href="http://wheatomics.sdau.edu.cn/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://wheatomics.sdau.edu.cn/</a>). The heatmap was generated using TBtools (<a href="https://github.com/CJ-Chen/TBtools-II/releases" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/CJ‐Chen/TBtools‐II/releases</a>). Z10: First leaf through coleoptile; Z13: Three leaves emerged; Z23: Main stem with three tillers; Z30: Pseudostem (youngest leaf sheath erect) Z32: Second node detectable; Z39: Flag leaf ligule just visible; Z65: Anthesis halfway (anthers halfway between tip and base of ear); Z71: Kernel water ripe (no starch); Z75: Medium milk; Z85: Soft dough. Further details on the Zadoks growth scale are available at <a href="https://www.agric.wa.gov.au/grains/zadoks-growth-scale" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.agric.wa.gov.au/grains/zadoks‐growth‐scale</a>.</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12256648/bin/TPG2-18-e70075-s001.docx" data-ga-action="click_feat_suppl" class="usa-link">TPG2-18-e70075-s001.docx</a><sup> (730.8KB, docx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="caption p">
<p>
<strong>Table S1</strong> Best linear unbiased estimates for trait values across control and high night temperature environments.</p>
<p>
<strong>Table S2</strong> Combined ANOVA showing mean squares for main and interaction sources of variation for different traits recorded under control and HNT conditions.</p>
<p>
<strong>Table S3</strong> Descriptive statistics, coefficient of variations (CV%), and heritability estimates of different traits across the environments.</p>
<p>
<strong>Table S4</strong> Correlation patterns observed among different traits within and across environments.</p>
<p>
<strong>Table S5</strong> Stress tolerance indices for different genotypes in the panel.</p>
<p>
<strong>Table S6</strong> Number of SNPs available on different wheat chromosomes.</p>
<p>
<strong>Table S7</strong> Details on significant SNPs associated with various yield‐related traits in different environments.</p>
<p>
<strong>Table S8</strong> Details on significant SNPs associated with multiple traits (within the individual environments) identified in the present study.</p>
<p>
<strong>Table S9</strong> Details on QTLs associated with different traits under control conditions.</p>
<p>
<strong>Table S10</strong> Patterns of favorable and non‐favorable alleles at various loci and their differential impact on trait values.</p>
<p>
<strong>Table S11</strong> Haplotypes associated with different traits identified during the present study.</p>
<p>
<strong>Table S12</strong> Gene models available from different QTL regions.</p>
<p>
<strong>Table S13</strong> Haplotype analysis of key candidate genes prioritized from various QTL regions identified in the present study.</p>
<p>
<strong>Table S14</strong> Major known genes regulating yield‐related traits in wheat.</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12256648/bin/TPG2-18-e70075-s002.xlsx" data-ga-action="click_feat_suppl" class="usa-link">TPG2-18-e70075-s002.xlsx</a><sup> (224.1KB, xlsx) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>Additional supporting information can be found online in the Supporting Information section at the end of this article.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from The Plant Genome are provided here courtesy of <strong>Wiley</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1002/tpg2.70075"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/TPG2-18-e70075.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (2.3 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12256648/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12256648/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12256648%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12256648/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12256648/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12256648/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40653599/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12256648/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40653599/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12256648/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12256648/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="tOWEwU6BzWwZ9wn9FFHlomzHWuqIz1U7ExnfNE7apFREdLnGFdmkTJOQ0m1arFdE">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
        <script type="module" crossorigin="" src="/static/assets/math-574fdcc6.js"></script>
    
    

    </body>
</html>
