
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Identifying new targets for improving terpenoid biosynthesis in Yarrowia lipolytica through random genomic cytosine base editing - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="AE53347B8AF2968305347B007C620E04.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="micrcellf">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366007/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Microbial Cell Factories">
<meta name="citation_title" content="Identifying new targets for improving terpenoid biosynthesis in Yarrowia lipolytica through random genomic cytosine base editing">
<meta name="citation_author" content="Zhenxia Li">
<meta name="citation_author_institution" content="College of Horticulture and Landscape, Henan Institute of Science and Technology, Xinxiang, 453003 Henan China">
<meta name="citation_author" content="Bo Liu">
<meta name="citation_author_institution" content="College of Horticulture and Landscape, Henan Institute of Science and Technology, Xinxiang, 453003 Henan China">
<meta name="citation_author_institution" content="Tianjin Institute of Industrial Biotechnology, Chinese of Academy of Sciences, 32 Xiqidao, Tianjin Airport Economic Park, Tianjin, 300308 China">
<meta name="citation_author" content="Rongtao Lv">
<meta name="citation_author_institution" content="Tianjin Institute of Industrial Biotechnology, Chinese of Academy of Sciences, 32 Xiqidao, Tianjin Airport Economic Park, Tianjin, 300308 China">
<meta name="citation_author_institution" content="College of Biotechnology, Tianjin University of Sciences and Technology, Tianjin, 300457 China">
<meta name="citation_author" content="Zhe Sun">
<meta name="citation_author_institution" content="Tianjin Institute of Industrial Biotechnology, Chinese of Academy of Sciences, 32 Xiqidao, Tianjin Airport Economic Park, Tianjin, 300308 China">
<meta name="citation_author_institution" content="National Technology Innovation Center of Synthetic Biology, Tianjin, 300308 China">
<meta name="citation_author" content="QingYan Li">
<meta name="citation_author_institution" content="Tianjin Institute of Industrial Biotechnology, Chinese of Academy of Sciences, 32 Xiqidao, Tianjin Airport Economic Park, Tianjin, 300308 China">
<meta name="citation_author_institution" content="National Technology Innovation Center of Synthetic Biology, Tianjin, 300308 China">
<meta name="citation_author" content="XueLi Zhang">
<meta name="citation_author_institution" content="Tianjin Institute of Industrial Biotechnology, Chinese of Academy of Sciences, 32 Xiqidao, Tianjin Airport Economic Park, Tianjin, 300308 China">
<meta name="citation_author_institution" content="State Key Laboratory of Engineering Biology for Low-Carbon Manufacturing, Chinese Academy of Sciences, Tianjin, 300308 P.R. China">
<meta name="citation_author_institution" content="National Technology Innovation Center of Synthetic Biology, Tianjin, 300308 China">
<meta name="citation_publication_date" content="2025 Aug 20">
<meta name="citation_volume" content="24">
<meta name="citation_firstpage" content="192">
<meta name="citation_doi" content="10.1186/s12934-025-02819-5">
<meta name="citation_pmid" content="40830463">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366007/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366007/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366007/pdf/12934_2025_Article_2819.pdf">
<meta name="description" content="Genome-scale mutagenesis integrated with high-throughput phenotypic screening and causal mutation mapping serves as a robust paradigm for systemic genetic dissection. Despite the application of non-homologous end joining (NHEJ)-mediated genome ...">
<meta name="og:title" content="Identifying new targets for improving terpenoid biosynthesis in Yarrowia lipolytica through random genomic cytosine base editing">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Genome-scale mutagenesis integrated with high-throughput phenotypic screening and causal mutation mapping serves as a robust paradigm for systemic genetic dissection. Despite the application of non-homologous end joining (NHEJ)-mediated genome ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366007/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12366007">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1186/s12934-025-02819-5"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/12934_2025_Article_2819.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12366007%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12366007/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12366007/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366007/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-micrcellf.png" alt="Microbial Cell Factories logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Microbial Cell Factories" title="Link to Microbial Cell Factories" shape="default" href="https://microbialcellfactories.biomedcentral.com/" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Microb Cell Fact</button></div>. 2025 Aug 20;24:192. doi: <a href="https://doi.org/10.1186/s12934-025-02819-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1186/s12934-025-02819-5</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Microb%20Cell%20Fact%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Microb%20Cell%20Fact%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Microb%20Cell%20Fact%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Microb%20Cell%20Fact%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Identifying new targets for improving terpenoid biosynthesis in <em>Yarrowia lipolytica</em> through random genomic cytosine base editing</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Li%20Z%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Zhenxia Li</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Zhenxia Li</span></h3>
<div class="p">
<sup>1</sup>College of Horticulture and Landscape, Henan Institute of Science and Technology, Xinxiang, 453003 Henan China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Li%20Z%22%5BAuthor%5D" class="usa-link"><span class="name western">Zhenxia Li</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Liu%20B%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Bo Liu</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Bo Liu</span></h3>
<div class="p">
<sup>1</sup>College of Horticulture and Landscape, Henan Institute of Science and Technology, Xinxiang, 453003 Henan China </div>
<div class="p">
<sup>2</sup>Tianjin Institute of Industrial Biotechnology, Chinese of Academy of Sciences, 32 Xiqidao, Tianjin Airport Economic Park, Tianjin, 300308 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Liu%20B%22%5BAuthor%5D" class="usa-link"><span class="name western">Bo Liu</span></a>
</div>
</div>
<sup>1,</sup><sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lv%20R%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Rongtao Lv</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Rongtao Lv</span></h3>
<div class="p">
<sup>2</sup>Tianjin Institute of Industrial Biotechnology, Chinese of Academy of Sciences, 32 Xiqidao, Tianjin Airport Economic Park, Tianjin, 300308 China </div>
<div class="p">
<sup>5</sup>College of Biotechnology, Tianjin University of Sciences and Technology, Tianjin, 300457 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lv%20R%22%5BAuthor%5D" class="usa-link"><span class="name western">Rongtao Lv</span></a>
</div>
</div>
<sup>2,</sup><sup>5</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sun%20Z%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Zhe Sun</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Zhe Sun</span></h3>
<div class="p">
<sup>2</sup>Tianjin Institute of Industrial Biotechnology, Chinese of Academy of Sciences, 32 Xiqidao, Tianjin Airport Economic Park, Tianjin, 300308 China </div>
<div class="p">
<sup>4</sup>National Technology Innovation Center of Synthetic Biology, Tianjin, 300308 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sun%20Z%22%5BAuthor%5D" class="usa-link"><span class="name western">Zhe Sun</span></a>
</div>
</div>
<sup>2,</sup><sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Li%20Q%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">QingYan Li</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">QingYan Li</span></h3>
<div class="p">
<sup>2</sup>Tianjin Institute of Industrial Biotechnology, Chinese of Academy of Sciences, 32 Xiqidao, Tianjin Airport Economic Park, Tianjin, 300308 China </div>
<div class="p">
<sup>4</sup>National Technology Innovation Center of Synthetic Biology, Tianjin, 300308 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Li%20Q%22%5BAuthor%5D" class="usa-link"><span class="name western">QingYan Li</span></a>
</div>
</div>
<sup>2,</sup><sup>4,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20X%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">XueLi Zhang</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">XueLi Zhang</span></h3>
<div class="p">
<sup>2</sup>Tianjin Institute of Industrial Biotechnology, Chinese of Academy of Sciences, 32 Xiqidao, Tianjin Airport Economic Park, Tianjin, 300308 China </div>
<div class="p">
<sup>3</sup>State Key Laboratory of Engineering Biology for Low-Carbon Manufacturing, Chinese Academy of Sciences, Tianjin, 300308 P.R. China </div>
<div class="p">
<sup>4</sup>National Technology Innovation Center of Synthetic Biology, Tianjin, 300308 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20X%22%5BAuthor%5D" class="usa-link"><span class="name western">XueLi Zhang</span></a>
</div>
</div>
<sup>2,</sup><sup>3,</sup><sup>4,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>College of Horticulture and Landscape, Henan Institute of Science and Technology, Xinxiang, 453003 Henan China </div>
<div id="Aff2">
<sup>2</sup>Tianjin Institute of Industrial Biotechnology, Chinese of Academy of Sciences, 32 Xiqidao, Tianjin Airport Economic Park, Tianjin, 300308 China </div>
<div id="Aff3">
<sup>3</sup>State Key Laboratory of Engineering Biology for Low-Carbon Manufacturing, Chinese Academy of Sciences, Tianjin, 300308 P.R. China </div>
<div id="Aff4">
<sup>4</sup>National Technology Innovation Center of Synthetic Biology, Tianjin, 300308 China </div>
<div id="Aff5">
<sup>5</sup>College of Biotechnology, Tianjin University of Sciences and Technology, Tianjin, 300457 China </div>
<div class="author-notes p"><div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div></div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 Apr 9; Accepted 2025 Aug 5; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by-nc-nd/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12366007  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40830463/" class="usa-link">40830463</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<section id="sec1"><h3 class="pmc_sec_title">Background</h3>
<p id="Par1">Genome-scale mutagenesis integrated with high-throughput phenotypic screening and causal mutation mapping serves as a robust paradigm for systemic genetic dissection. Despite the application of non-homologous end joining (NHEJ)-mediated genome editing in <em>Yarrowia lipolytica</em>, the development of alternative genome-wide mutagenesis strategies remains unexplored in this industrially relevant oleaginous yeast.</p></section><section id="sec2"><h3 class="pmc_sec_title">Results</h3>
<p id="Par2">We developed the Helicase-Assisted (Helicase-CDA) system, a genome-wide mutagenesis platform integrating the helicase domain of <em>Yarrowia</em> MCM5 (Encoded by <em>YALI1_A01766g</em>) with cytidine deaminase (CDA). This system enables continuous C-to-T specific mutations at random genomic loci. Applied to an industrial β-carotene-producing <em>Y. lipolytica</em> strain, Helicase-CDA system generated a mutagenized library through 7-day subculturing. Through high-throughput screening, we successfully isolated the mutant strain CDA-14, which demonstrated a 25% enhancement in β-carotene production (448.1 mg/L) compared to the wild-type strain. Notably, its productivity of β-carotene reached 6.15 g/L in fed-batch fermentation. Whole-genome sequencing revealed a G1637A substitution in <em>YALI1_B16239g</em>, which encodes a membrane protein showing homology to sterol biosynthesis regulator MGA2. This mutation led to reduced ERG1 expression level and redirected central carbon flux toward carotenoid synthesis by perturbing isoprenoid precursor partitioning.</p></section><section id="sec3"><h3 class="pmc_sec_title">Conclusion</h3>
<p id="Par3">Helicase-CDA system circumvents the dependency on NHEJ-mediated whole-genome mutation approach, offering a robust tool for continuous genome evolution in pre-engineered industrial strains. This study not only enhances genome editing in <em>Y. lipolytica</em> but also identifies a practical target for optimizing terpenoid biosynthesis, demonstrating significant potential for applications in metabolic engineering and synthetic biology.</p></section><section id="sec4"><h3 class="pmc_sec_title">Supplementary Information</h3>
<p>The online version contains supplementary material available at 10.1186/s12934-025-02819-5.</p></section><section id="kwd-group1" lang="en" class="kwd-group"><p><strong>Keywords:</strong> <em>Yarrowia lipolytica</em>, Genome-wide mutagenesis, Helicase-CDA system, Β-carotene, Random base editing</p></section></section><section id="Sec1"><h2 class="pmc_sec_title">Background</h2>
<p id="Par4">Metabolic engineering and synthetic biology have emerged as powerful tools for improving the production of high-value biochemicals in microbial hosts [<a href="#CR1" class="usa-link" aria-describedby="CR1">1</a>]. Among these, <em>Y. lipolytica</em> has gained prominence as a model organism due to its robust metabolism, high tolerance to stress, and ability to accumulate lipophilic compounds [<a href="#CR2" class="usa-link" aria-describedby="CR2">2</a>]. It has been widely applied in the synthesis of bulk chemicals, fatty acids, aromatic compounds, terpenoids, etc [<a href="#CR3" class="usa-link" aria-describedby="CR3">3</a>–<a href="#CR7" class="usa-link" aria-describedby="CR7">7</a>]. However, the complexity of cellular metabolic networks and incomplete understanding of context-dependent regulation continue to challenge rational engineering strategies for industrial trait improvement.</p>
<p id="Par5">Genome-scale evolution has become an indispensable approach for developing robust cell factories, particularly when coupled with high-throughput screening methodologies [<a href="#CR8" class="usa-link" aria-describedby="CR8">8</a>]. While adaptive laboratory evolution (ALE) remains widely employed for phenotypic enhancement through spontaneous mutagenesis during serial passaging [<a href="#CR9" class="usa-link" aria-describedby="CR9">9</a>], recent advances in molecular biology tools have enabled more sophisticated genome editing strategies. Notable developments include RNAi-assisted genome evolution (RAGE) [<a href="#CR10" class="usa-link" aria-describedby="CR10">10</a>], CRISPR/Cas9-mediated homology-directed genome-scale engineering (CHAnGE), multifunctional genome-wide CRISPR systems (MAGIC) [<a href="#CR11" class="usa-link" aria-describedby="CR11">11</a>], and helicase-assisted cytidine deaminase fusion technologies [<a href="#CR12" class="usa-link" aria-describedby="CR12">12</a>]. A particularly promising approach involves the fusion of DNA replication-associated helicases with cytidine deaminases (CADs), exemplified by the Helicase-CDA base editing [<a href="#CR12" class="usa-link" aria-describedby="CR12">12</a>]. This technology leverages ssDNA-binding proteins fused with CADs to generate genome-wide C-to-T mutations, achieving remarkable success in enhancing industrial phenotypes across multiple microbial platforms - including β-carotene overproduction in <em>Saccharomyces cerevisiae</em> [<a href="#CR12" class="usa-link" aria-describedby="CR12">12</a>, <a href="#CR13" class="usa-link" aria-describedby="CR13">13</a>], isobutanol tolerance in <em>Escherichia coli</em> [<a href="#CR12" class="usa-link" aria-describedby="CR12">12</a>], and stress resistance improvements in <em>Corynebacterium glutamicum</em> [<a href="#CR14" class="usa-link" aria-describedby="CR14">14</a>] <em>and</em> high concentrations of lovastatin production in <em>Monascus purpureus</em> [<a href="#CR15" class="usa-link" aria-describedby="CR15">15</a>]. Despite these advances, the application of such genome-wide mutagenesis tools remains unexplored in <em>Y. lipolytica</em>, representing a critical technological gap for this industrially important yeast.</p>
<p id="Par6">In this study, we developed a novel genome editing platform for <em>Y. lipolytica.</em> We engineered a synthetic fusion protein combining a conserved helicase homolog YlMCM5, functionally analogous to <em>S. cerevisiae</em> MCM5, with cytosine deaminase (CDA) to enable programmable genome-wide C-to-T transitions via processive DNA unwinding-coupled base editing. Through continuous mutagenesis and fluorescence-activated cell sorting (FACS)-based high-throughput screening, we successfully isolated strains with significantly enhanced β-carotene production. Furthermore, through whole-genome resequencing and reverse genetic validation, we identified novel molecular targets contributing to carotenoid overproduction. This work not only establishes a powerful genome editing toolkit (Helicase-CDA system) for <em>Y. lipolytica</em> but also provides fundamental insights into the metabolic regulation of isoprenoid biosynthesis in oleaginous yeasts.</p></section><section id="Sec2"><h2 class="pmc_sec_title">Materials and methods</h2>
<section id="Sec3"><h3 class="pmc_sec_title">Strains, media and growth conditions</h3>
<p id="Par7"><em>Escherichia coli</em> Trans1 T1 was used for plasmids construction and was cultivated at 37 °C in lysogeny broth (LB) medium supplemented with 100 µg/mL ampicillin or 50 µg/mL kanamycin. For flask fermentation, the <em>Y. lipolytica</em> strains were cultivated at 30 °C with shaking at 250 rpm in YPD medium with 30 g/L glucose (10 g/L yeast extract, 20 g/L peptone, and 30 g/L glucose) or YPDE medium (10 g/L yeast extract, 20 g/L peptone, 15 g/L glucose and 15 g/L erythritol) for 4 days. Synthetic complete medium (SC medium: 20 g/L glucose, 8 g/L SC (YGM003A; FUNGENOME; China), 18 g/L agar) without uracil was used to screen yeast transformants. Synthetic complete medium supplemented with 1 g/L 5-flfluoroorotic acid (5-FOA) was used to screen URA3 deficient strains.</p></section><section id="Sec4"><h3 class="pmc_sec_title">Plasmids and strains construction</h3>
<p id="Par8">The plasmids used in this study are presented in table <a href="#MOESM2" class="usa-link">S2</a>, and their construction is described in detail below and in Text S1. The primers are listed in table <a href="#MOESM1" class="usa-link">S1</a>. The gRNA sequence used in this study are listed in table <a href="#MOESM3" class="usa-link">S3</a>.</p>
<p id="Par9">Plasmids were assembled using the Seamless Cloning Kit (Beyotime, China) or Golden Gate method [<a href="#CR16" class="usa-link" aria-describedby="CR16">16</a>]. To construct the plasmid pURA-MCM5-CDA, the backbone was PCR-amplified from pURA-YL-IntC2 using the primer pair Cen1-R/AmpP-cyc1-F. The GPD promoter and <em>YLMCM5</em>(<em>YALI1_A01766g</em>) were amplified from the chromosomal DNA of <em>Y. lipolytica</em> using the primer pairs GPD-CEN-F/GPD-R and MCM5-GPD-F/MCM5-V40-R, respectively. The CDA fragment, which includes a large peptide linker (100 amino acids) and an SV40 nuclear localization signal (NLS) at the N-terminus, was amplified from pRS315e_pGal-dCas9-pmCDA1 (Addgene #79615) using the primer pair SV40-NLS-F/CDMA-CYC1-R. The CYC1 terminator was amplified from <em>S. cerevisiae</em> using the primer pair SCCyc1T-F/SCCyc1T-R. These five PCR products were digested with <em>Dpn</em>I and ligated using the Seamless Cloning Kit to generate pURA-MCM5-CDA. The sequence of pURA-MCM5-CDA is provided in Table S4. The main primers and plasmids are listed in Table <a href="#MOESM1" class="usa-link">S1</a> and Table <a href="#MOESM2" class="usa-link">S2</a>.</p>
<p id="Par10">All yeast strains used in this study (Table <a href="#Tab1" class="usa-link">1</a>) were derived from YL002 [<a href="#CR17" class="usa-link" aria-describedby="CR17">17</a>]. The methods for transformation and removal of the URA3 selection marker were described previously [<a href="#CR18" class="usa-link" aria-describedby="CR18">18</a>]. The YL002 strain was engineered with a rational metabolic engineering strategy to enhance β-carotene biosynthesis through the co-overexpression of key enzymes involved in the mevalonate pathway components <em>ERG12</em>, <em>ERG20</em>, <em>IDI</em>, <em>HMG1</em> and <em>GGS</em> from <em>Y. lipolytica</em>, and the carotenoid biosynthesis pathway GGPPSxd from <em>Phaffia rhodozyma</em>, three copies of <em>CarB</em> and <em>CarRP</em> from <em>Mucor circinelloides</em>(SEQ ID N0.3) (Fig. <a href="#Fig1" class="usa-link">1</a>A) were overexpressed to improve β-carotene production. A flowchart describing the construction of β-carotene-producing <em>Y. lipolytica</em> is shown in Fig. <a href="#Fig1" class="usa-link">1</a>B. All <em>Y. lipolytica</em> strains constructed in this study are listed in Table <a href="#Tab1" class="usa-link">1</a>. The genes were integrated into the yeast genome using CRISPR/cas9 [<a href="#CR17" class="usa-link" aria-describedby="CR17">17</a>]. The construction methods for plasmids containing donor DNA or gRNA are described in Text S1. The key genes abbreviations and identifiers used in this study were listed in table S5.</p>
<section class="tw xbox font-sm" id="Tab1"><h4 class="obj_head">Table 1.</h4>
<div class="caption p"><p>Strains used in this study</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Strains</th>
<th align="left" colspan="1" rowspan="1">Relative characteristics</th>
<th align="left" colspan="1" rowspan="1">Sources</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">YL002</td>
<td align="left" colspan="1" rowspan="1">Po1g-Δ<em>ku70::Cas9-Hyg</em>, Δ<em>ura3</em>
</td>
<td align="left" colspan="1" rowspan="1">[<a href="#CR17" class="usa-link" aria-describedby="CR17">17</a>]</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">YL045</td>
<td align="left" colspan="1" rowspan="1">Yl002-intE3::ERG12-ERG20</td>
<td align="left" colspan="1" rowspan="1">This work</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">YL046</td>
<td align="left" colspan="1" rowspan="1">Yl045-intE1::GGPPxd-IDI</td>
<td align="left" colspan="1" rowspan="1">This work</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">YL061</td>
<td align="left" colspan="1" rowspan="1">Yl049-intC2::CarRP-CarB</td>
<td align="left" colspan="1" rowspan="1">This work</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">YL062</td>
<td align="left" colspan="1" rowspan="1">Yl061-intF1::tHMG1-GGS</td>
<td align="left" colspan="1" rowspan="1">This work</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">YL063</td>
<td align="left" colspan="1" rowspan="1">Yl062-intF2::CarRP-CarB</td>
<td align="left" colspan="1" rowspan="1">This work</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">YL064</td>
<td align="left" colspan="1" rowspan="1">Yl063-intA8::tHMG1</td>
<td align="left" colspan="1" rowspan="1">This work</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">YL065</td>
<td align="left" colspan="1" rowspan="1">Yl064-intF17::CarRP-CarB</td>
<td align="left" colspan="1" rowspan="1">This work</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">CDA-14</td>
<td align="left" colspan="1" rowspan="1">Screened from library</td>
<td align="left" colspan="1" rowspan="1">This work</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">CDA-44</td>
<td align="left" colspan="1" rowspan="1">Screened from library</td>
<td align="left" colspan="1" rowspan="1">This work</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">YL065-MGA2</td>
<td align="left" colspan="1" rowspan="1">YL065-intC3::MGA2</td>
<td align="left" colspan="1" rowspan="1">This work</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">CDA-14-MGA2</td>
<td align="left" colspan="1" rowspan="1">CDA-14-intC3::MGA2</td>
<td align="left" colspan="1" rowspan="1">This work</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">YL065-△MGA2</td>
<td align="left" colspan="1" rowspan="1">YL065 with MGA2 deletion</td>
<td align="left" colspan="1" rowspan="1">This work</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><figure class="fig xbox font-sm" id="Fig1"><h4 class="obj_head">Fig. 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12366007_12934_2025_2819_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/7481/12366007/0f5295be246a/12934_2025_2819_Fig1_HTML.jpg" loading="lazy" id="MO1" height="281" width="774" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Reconstituted β-Carotene Biosynthetic Pathway and Strain Engineering Workflow in this study. <strong>A</strong>: Genes highlighted in red indicate overexpression, while genes marked in blue represent suppressed expression. Metabolites: G6P, Glucose-6-phosphate; F6P, Fructose-6-phosphate; FBP, Fructose-1,6-bisphosphate; GAP, Glyceraldehyde-3-phosphate; Acety-CoA, Acetyl-coenzyme A; MVA, Mevalonic acid; MVA-5-P, Mevalonate-5-phosphate; MVA-5-PP, Mevalonate-5-diphosphate; HMG-CoA, 3-hydroxy-3-methylglutaryl-CoA; IPP, isopentenyl diphosphate; DMAPP, dimethylallyl diphosphate; GPP, geranyl diphosphate; FPP, farnesyl diphosphate; GGPP, geranylgeranyl diphosphate; DMAPP, dimethylallyl diphosphate. Enzymes: ERG10, acetyl-CoA acetyltransferase; ERG13, HMG-CoA synthase; ERG12, mevalonate kinase; ERG8, phosphomevalonate kinase; ERG19, mevalonate diphosphate decarboxylase; IDI, IPP isomerase; ERG20, farnesyl pyrophosphate synthetase; ERG9, squalene synthase; ERG1, squalene epoxidase; GGPPxd, geranylgeranyl diphosphate synthase; CarRP, phytoene synthase/lycopene cyclase; CarB, phytoene dehydrogenase; tHMGR, truncated 3-Hydroxy-3-Methylglutaryl-CoA Reductase; PDH, Pyruvate Dehydrogenase; ALDO, Fructose-bisphosphate Aldolase; PFK, Phosphofructokinase; PGI, Phosphoglucose Isomerase; HK, Hexokinase. <strong>B</strong>: Strain Engineering Workflow in this study.</p></figcaption></figure></section><section id="Sec5"><h3 class="pmc_sec_title">Random base editing procedure</h3>
<p id="Par12">The β-carotene-producing <em>Y. lipolytica</em> strains YL065 was used for chromosome random base editing. YL065 colonies with pURA-YLMCM5-CDA or without plasmid were first inoculated in a 2% glucose SD-URA<sup>−</sup> medium and grown at 30 °C overnight, which was then inoculated into fresh medium at a 1% inoculation ratio and transferred every 24 h. After 5 and 7 consecutive transfers, strains exhibiting high fluorescence intensity within the 510–530 nm range were selected using flow cytometry due to the β-carotene’s characteristic fluorescence profile (excitation: 488 nm; emission: 535 ± 25 nm). The mutagenesis protocol and high-throughput screening schematic for β-carotene-producing strains are diagrammatically illustrated in Fig. <a href="#Fig4" class="usa-link">4</a>A.</p>
<figure class="fig xbox font-sm" id="Fig4"><h4 class="obj_head">Fig. 4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12366007_12934_2025_2819_Fig4_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/7481/12366007/2cc636ea3560/12934_2025_2819_Fig4_HTML.jpg" loading="lazy" id="MO7" height="712" width="797" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Flow Cytometry-Based Screening (FACS)and Fermentation Optimization of β-Carotene-Producing Microbial Strains. <strong>A</strong>: FACS Workflow for Helicase-CDA System-Generated Mutant Libraries; <strong>B</strong>: Growth and β-carotene yield of High-Fluorescence Strains Enriched by Flow Cytometry; <strong>C</strong>: Growth and β-carotene yield of screened high β-carotene-producing strain from Mutant Libraries. Error bar represents standard deviation of triplicate experiments</p></figcaption></figure></section><section id="Sec6"><h3 class="pmc_sec_title">Preliminary screening of high-β-carotene-producing mutants</h3>
<p id="Par13">The cytometric analysis and sorting of the chromosome random mutagenesis library were performed following the protocol described in a previous report [<a href="#CR19" class="usa-link" aria-describedby="CR19">19</a>]. Cytometric analysis and sorting of YL065 chromosome random mutagenesis library were executed with a flow cytometer (BD FACS Aria Fusion) using a 50-mW argon ion laser emitting at 488 nm. The cells were washed twice with 10 mM potassium phosphate buffer (pH 7.4), filtered through a 40-mm nylon mesh (assembled in a 5 mL polystyrene tube; cell-strainer cap; Nippon Becton, Dickinson) to remove accumulated cells, and then diluted with the same buffer to an optimum concentration. The fluorescence emission was measured at 510 nm to 560 nm. The self-defined high fluorescein area cells were sorted (around 15,000 cells). Fluorescence data were analyzed by Accuri CFlow software (Accuri Cytometers, Inc., Ann Arbor, MI, USA) and Flowjo software (FlowJo, LLC, Ashland, OR, USA) for statistical analysis.</p>
<p id="Par14">The mutagenized libraries from days 5 and 7 were sorted using flow cytometry, and the top 0.05% of cells with the highest fluorescence intensity were collected as high-yield strains. For each sample, 2,000 cells were pooled and mixed, while an additional 100 colonies were streaked onto plates for further analysis.</p></section><section id="Sec7"><h3 class="pmc_sec_title">Whole genome sequencing of evolved strains</h3>
<p id="Par15">Whole genome sequencing of CDA14 and CDA44 was performed together with YL065. Genomic DNA was extracted from the strains using a cetyltrimethylammonium bromide (CTAB)-based method and sequencing libraries were prepared using the standard library building process of the Illumina TruSeq DNAPCR-Free Prep Kit reagent. Libraries with a 350 insert were constructed for each sample and were subjected to paired-end sequencing using next-generation sequencing (NGS) based on the Illumina/BGI sequencing platform. There were, in total, 7,842,520, 8,072,614, and 7,588,142 reads for YL065, YL065-CDA-14, and YL065-CDA-44, respectively. The GATK software was used to analyze SNP and InDels. and were filtered with the following parameters: QD &lt; 2.0 || MQ &lt; 40.0 || FS &gt; 60.0 || QUAL &lt; 30.0 || MQrankSum &lt; −12.5 || ReadPosRankSum &lt; −8.0 -clusterSize 2 -clusterWindowSize 5. The SNPs identified by GATK were further filtered. Finally, High-quality SNPs were obtained. Exonic SNPs identified in all evolved strains as compared with <em>Y. lipolytica</em> (Yarrowia lipolytica genome assembly ASM176148v1 - NCBI - NLM) are provided in Supplementary Excel S1.</p></section><section id="Sec8"><h3 class="pmc_sec_title">Fed-batch fermentation of the strains YL065 and CDA-14</h3>
<p id="Par16">Strains YL065 and CDA-14 were utilized for β-carotene production through fed-batch fermentation. Fed-batch fermentation was performed as described [<a href="#CR20" class="usa-link" aria-describedby="CR20">20</a>, <a href="#CR21" class="usa-link" aria-describedby="CR21">21</a>] with the following modification. A two-stage system was used for preparing seed culture. In the first stage, fresh colonies were inoculated into a 100 mL flask containing 10 mL of medium, incubated at 30 °C with shaking at 250 rpm until reaching an OD<sub>600</sub> of 1.0–4.0. Subsequently, 1 mL from this initial seed culture was transferred to a 1 L flask containing 200 mL of medium and further incubated under the same conditions until the OD<sub>600</sub> reached 15. This second-stage seed solution was transferred to a 5 L bioreactor containing 2 L substrate medium. The fermentation substrate medium contained (per liter): 100 g glucose, 50 g yeast extract, 100 g peptone.</p>
<p id="Par17">During the early fermentation phase, dissolved oxygen (DO) concentration was maintained at 30% by cascading with agitation speed and ventilation, the temperature was controlled at 30℃, and pH was maintained at 5.5 by automatic addition of 13% ammonia. Glucose content was measured every 4–6 h. Once glucose concentration dropped below 10 g/L, feed solution containing 20 g/L of yeast extract, 50 g/L of (NH<sub>4</sub>)<sub>2</sub>SO<sub>4</sub> and 500 g/L of glucose was added to restore the glucose concentration within the bioreactor to 80 g/L. The airflow rate and agitation speed were kept at the maximum value. Then the concentration of DO was settled at 30%.</p></section><section id="Sec9"><h3 class="pmc_sec_title">Determination of squalene levels</h3>
<p id="Par18">Squalene content in YL065, CDA-14, and YL065-ΔMGA2 strains was measured according to the method reported previous [<a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>] with some modification. YL065, CDA-14, and YL065-ΔMGA2 were cultured in YPD medium, and squalene levels were determined in samples collected at 48 h and 72 h of fermentation. Briefly, a 200 µL aliquot of the culture broth was centrifuged (12,000×g, 5 min), mixed with 3–4 mm glass beads, and resuspended in 1 mL acetone. Cells were disrupted by vortexing at 3,000 rpm for 10 min, followed by sonication for 5 min in an ultrasonic cleaner. The lysate was then incubated at 55 °C for 15 min in the dark. Afterward, the samples were centrifuged at 12,000×g for 10 min to obtain the supernatant containing carotenoids, which was used for HPLC analysis as described before [<a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>].</p></section><section id="Sec10"><h3 class="pmc_sec_title">RNA isolation and qRT-PCR</h3>
<p id="Par19">Flask cultures were inoculated into YPD liquid medium containing 3% (w/v) glucose and cultured for 24 h. After cultivation, 1 mL of cultures were collected by centrifugation for 5 min at 12,000 g at 4℃. Total RNA was extracted using the RNeasy Mini Kit (QIAGEN, Germany) according to the manufacturer’s protocol, with gDNA elimination performed using DNase I (Thermo Scientific, USA). cDNA was synthesized using the <em>EasyScript</em> One-Step gDNA Removal and cDNA Synthesis SuperMix (TransGen Biotech, China). RT-qPCR analysis was performed using ChamQ Universal SYBR qPCR Master Mix (Vazyme, China) on an Eco Real-Time PCR System (Illumina, USA).</p>
<p id="Par20">Primers for qRT-PCR are listed in Supplementary Table <a href="#Tab1" class="usa-link">1</a>.The <em>ACT1</em> gene (actin gene) served as the internal control for data normalization. Gene relative expression was calculated via the 2<sup>−ΔΔCT</sup> method [<a href="#CR23" class="usa-link" aria-describedby="CR23">23</a>]. All experiments included biological triplicates.</p></section><section id="Sec11"><h3 class="pmc_sec_title">Measurement of carotenoid production</h3>
<p id="Par21">Carotenoid extraction was performed as described before with the following modification [<a href="#CR24" class="usa-link" aria-describedby="CR24">24</a>]. Carotenoids were extracted using acetone following the same method as for squalene extraction, and the supernatant containing carotenoids was obtained and used for HPLC analysis.</p>
<p id="Par22">The analytes were separated on a Symmetry C18 column (250 mm×4.6 mm, 5 μm, Waters, Ireland), which was kept at 30 °C, and were detected at 476 nm using a Shimadzu UV-2550 spectrophotometer (Shimadzu, Kyoto, Japan). The mobile phase and flow rate in measurement of carotenoids production were the same as that described previously [<a href="#CR25" class="usa-link" aria-describedby="CR25">25</a>]. The results represent the means ± SD of three independent experiments. Dry cell weight (DCW) was calculated from the optical density at 600 nm (OD<sub>600</sub>) using the empirical formula 1 OD<sub>600</sub> = 0.39 g DCW L<sup>−1</sup>.</p></section></section><section id="Sec12"><h2 class="pmc_sec_title">Results</h2>
<section id="Sec21"><h3 class="pmc_sec_title">Constructing the Helicase-CDA molecular device in <strong><em>Y. lipolytica</em></strong>
</h3>
<p id="Par23">To establish a CDA-mediated targeted mutagenesis system in <em>Y. lipolytica</em>, we employed the previously developed Target-AID (activation-induced cytidine deaminase) base editor [<a href="#CR26" class="usa-link" aria-describedby="CR26">26</a>]. For this system, we selected a helicase as the docking platform to fuse with the cytidine deaminase. By attaching the cytidine deaminase to the helicase, we aimed to exploit this unwinding process, enabling the enzyme to act directly on the exposed single-stranded DNA [<a href="#CR12" class="usa-link" aria-describedby="CR12">12</a>]. Mcm2-7 complex is the replicative helicase in eukaryotes, which is constituted of 6 different subunits [<a href="#CR27" class="usa-link" aria-describedby="CR27">27</a>]. One of the subunits, MCM5, was selected and fused to CDA to construct MCM5-AID complex expressed by plasmid pMCM5-AID [<a href="#CR12" class="usa-link" aria-describedby="CR12">12</a>].</p>
<p id="Par24">Genomic alignment revealed that the protein encoded by <em>YALI1_A01766g</em> in <em>Y. lipolytica</em> shares 53% sequence similarity with the MCM5 helicase subunit of <em>S. cerevisiae</em>, suggesting evolutionary conservation of DNA replication fork unwinding functionality. Based on this phylogenetic evidence, we designated this protein as YLMCM5 and engineered a fusion construct with cytidine deaminase (CDA) from <em>Petromyzon marinus</em> using flexible peptide linkers, resulting in the pMCM5-CDA plasmid (Fig. <a href="#Fig2" class="usa-link">2</a>A) for genome-wide C→T mutagenesis (Fig. <a href="#Fig2" class="usa-link">2</a>B). The molecular mechanism involves YLMCM5-mediated recruitment of CDA to single-stranded DNA regions at replication forks during chromosomal DNA synthesis. CDA catalyzes deamination of cytosine to uracil (C→U) on nascent DNA strands, which is subsequently fixed as thymine (T) through replication, thereby achieving directional C/G→T/A base substitutions (Fig. <a href="#Fig2" class="usa-link">2</a>C).</p>
<figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12366007_12934_2025_2819_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/7481/12366007/d1de49ce5fb6/12934_2025_2819_Fig2_HTML.jpg" loading="lazy" id="MO3" height="474" width="797" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Mechanistic Basis of Helicase-CDA Genome Editing. <strong>A</strong>: Diagram of the plasmid of pURA-MCM5-CDA and Helicase-CDA Fusion Protein, The SV40 nuclear localization signal (NLS) directs efficient nuclear targeting of the fusion protein, while the helicase simultaneously unwinds duplex DNA and catalyzes CDA1-mediated deamination of cytosine (C) to uracil (U) during DNA replication. <strong>B</strong>: Schematic Diagram of Cytosine Deamination to Uracil. <strong>C</strong>: The mechanism of Helicase-CDA system mediated random base editing. The schematic is hypothesis, and may not reflect the actual mechanism</p></figcaption></figure><p id="Par25">To validate the mutagenic capacity of this system while simultaneously uncovering novel targets for improving carotenoid biosynthesis, pMCM5-CDA would be transformed into a β-carotene-producing chassis strain. Then a genome-wide saturation mutagenesis library was constructed and subjected to high-throughput screening via fluorescence-activated cell sorting (FACS), leveraging the intrinsic autofluorescence of carotenoids to establish a phenotype-genotype correlation platform.</p></section><section id="Sec13"><h3 class="pmc_sec_title">Strain design for high β-carotene production</h3>
<p id="Par26">To achieve dual objectives of functional validation for the MCM5-guided CDA mutagenesis system and discovery of novel β-carotene biosynthesis enhancers, a high-titer β-carotene-producing <em>Y. lipolytica</em> chassis strain was engineered. The previously engineered <em>Y. lipolytica</em> strain YL002, in which <em>KU70</em> was deleted through the <em>Cas9</em> expression cassette insertion and <em>URA3</em> was deleted using CRISPR/Cas9 system, was used to construct β-carotene-producing strain [<a href="#CR17" class="usa-link" aria-describedby="CR17">17</a>]. The β-carotene synthetic pathway in yeast can be decomposed to the endogenous mevalonate (MVA) pathway section generating geranylgeranyl pyrophosphate (GGPP) as the precursor for carotenoids. In previous studies, overexpression of the genes <em>ERG12</em>, <em>ERG20</em>, <em>IDI</em>, <em>HMG1</em>, and <em>GGS</em> have been reported to enhance the supply of IPP and DMAPP, thereby increasing carotenoid production [<a href="#CR7" class="usa-link" aria-describedby="CR7">7</a>, <a href="#CR28" class="usa-link" aria-describedby="CR28">28</a>]. Moreover, the <em>GGPP</em>xd derived from <em>Xanthophyllomyces dendrorhous</em> exhibits superior catalytic activity [<a href="#CR20" class="usa-link" aria-describedby="CR20">20</a>]. Additionally, the <em>CarRP</em> and <em>CarB</em> genes from <em>Mucor circinelloides</em>, encoding phytoene synthase/lycopene cyclase and phytoene dehydrogenase were used to catalyze the synthesis of β-carotene from GGPP. Notably, we employed an engineered CarRP variant (Y27R) that completely abolishes substrate inhibition while preserving catalytic efficiency, as previously characterized through structural enzymology studies [<a href="#CR20" class="usa-link" aria-describedby="CR20">20</a>]. Therefore, <em>ERG12</em>, <em>ERG20</em>, <em>IDI</em>,<em> HMG1</em>,<em> GGS</em> and the codon optimized <em>GGPP</em>xd, <em>CarB</em>, <em>CarRP</em> were overexpressed in <em>Y lipolytica</em> for improve carotenoid production.</p>
<p id="Par27">Specifically, the <em>ERG12</em> and <em>ERG20</em> genes are inserted at the <em>intE3</em> locus, the <em>GGPPxd</em> and <em>IDI</em> genes at the <em>intE1</em> locus, and the <em>HMG1</em> and <em>GGS</em> genes at the <em>intF1</em> locus. The gene clusters (<em>CarRP</em> and <em>CarB</em>) were integrated into the genome of <em>Y. lipolytica</em> at the <em>intC2</em>, <em>intF2</em> and <em>intF17</em> loci to enhance β-carotene production (Figs. <a href="#Fig1" class="usa-link">1</a>B and <a href="#Fig3" class="usa-link">3</a>A). The engineered strain YL061 with co-expression of <em>CarRP</em>, <em>CarB</em>,<em> ERG12</em>,<em> ERG20</em>,<em> GGPPxd</em> and <em>IDI</em> successfully produced 31.81 mg/L, 2.84 mg/g DCW of β-carotene (Fig. <a href="#Fig3" class="usa-link">3</a>B).</p>
<figure class="fig xbox font-sm" id="Fig3"><h4 class="obj_head">Fig. 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12366007_12934_2025_2819_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/7481/12366007/b240044e3000/12934_2025_2819_Fig3_HTML.jpg" loading="lazy" id="MO5" height="333" width="797" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>β-carotene production in engineered strains of <em>Y. lipolytica</em>
<strong>A</strong>: Schematic representation of high-expression genetic elements, including gene, promoter, terminator, and insertion loci. <strong>B</strong>: Cell growth (OD<sub>600</sub>), β-carotene production (mg/L), and content (mg/g DCW) in β-carotene-producing strains after 4-day cultivation in YPD medium. Error bar represents standard deviation of triplicate experiments</p></figcaption></figure><p id="Par28">Previous research demonstrated that overexpression of 3-hydroxy-3-methylglutaryl-CoA (HMG-CoA) reductase (HMGR1) significantly enhances terpenoid production [<a href="#CR29" class="usa-link" aria-describedby="CR29">29</a>, <a href="#CR30" class="usa-link" aria-describedby="CR30">30</a>]. Furthermore, a truncated form of HMG1 (tHMG1), lacking the N-terminal 500 amino acids, has been shown to markedly increase β-carotene production and other terpenoids in <em>Y. lipolytica</em> [<a href="#CR31" class="usa-link" aria-describedby="CR31">31</a>]. In addition to tHmgR, expression of geranylgeranyl diphosphate synthase (GGS1) has also been reported to elevate the synthesis of carotenoid precursors [<a href="#CR30" class="usa-link" aria-describedby="CR30">30</a>, <a href="#CR31" class="usa-link" aria-describedby="CR31">31</a>]. Based on these findings, the construct P<sub>EXP</sub>-tHmgR-P<sub>GPD</sub>-ggs1 was integrated into the YL061 genome at the <em>intF1</em> locus, generating strain YL062 (Fig. <a href="#Fig3" class="usa-link">3</a>A). As anticipated, strain YL062 demonstrated β-carotene production levels of 119.94 mg/L and 10.05 mg/g DCW, corresponding to 3.77-fold and 3.54-fold increases, respectively, compared to YL061 (Fig. <a href="#Fig3" class="usa-link">3</a>B).</p>
<p id="Par29">In addition, <em>CarB</em>, <em>CarRP</em>, and <em>tHMG1</em> have been identified as bottlenecks in β-carotene synthesis in <em>Y. lipolytica</em>. To further enhance β-carotene production, additional copies of <em>CarRP</em> and <em>CarB</em> were expressed along with <em>tHMG1</em> in strain YL062, resulting in the development of strains YL063, YL064, and YL065 (Fig. <a href="#Fig1" class="usa-link">1</a>B). After the introduction of a third copy of CarRP and CarB into YL062, stain YL063 produced 236.71 mg/L and 18.69 mg/g DCW of β-carotene, representing improvements of 97.4% and 86.1%, respectively, compared to the parent strain YL062. The incorporation of tHMG1 into the YL063 strain further increased β-carotene production and yield to 247.27 mg/L and 20.66 mg/g DCW, reflecting improvements of 4.5% and 10.5% over the parent strain. Subsequently, the integration of a third copy of <em>CarRP</em> and <em>CarB</em> further enhanced β-carotene production and yield to 352.79 mg/L and 27.92 mg/g DCW, corresponding to increases of 42.7% and 35.1% relative to the parent strain. When compared to YL061, the production and yield in YL065 exhibited 11.1-fold and 9.8-fold enhancements, respectively (Fig. <a href="#Fig3" class="usa-link">3</a>B).</p>
<p id="Par30">YL065 contained three copies of <em>CarRP</em> and <em>CarB</em>, providing a sufficient pathway for β-carotene synthesis. Mutations that enhance precursor supply can be reflected in increased β-carotene production. Therefore, starting from this strain, whole-genome mutagenesis was performed to identify new targets for improving carotenoid synthesis in <em>Y. lipolytica</em>.</p></section><section id="Sec14"><h3 class="pmc_sec_title">Constructing and sorting the chromosome random mutagenesis library</h3>
<p id="Par31">The <em>pMCM5-CDA</em> plasmid was transformed into the <em>YL065</em> strain, and the transformants were cultured on <em>SD-URA</em><sup><strong>−</strong></sup> medium to induce whole-genome random mutagenesis. A fluorescence-activated cell sorting (FACS) strategy was previously established for efficient high-throughput screening of β-carotene-producing strains [<a href="#CR19" class="usa-link" aria-describedby="CR19">19</a>]. This approach utilizes the green fluorescence characteristics of β-carotene, which exhibits an excitation peak at 488 nm and an emission wavelength of 535 ± 25 nm. In this study, flow cytometry was employed to isolate high β-carotene-producing strains from mutagenized libraries generated via genome-wide mutation introduced by p<em>MCM5-CDA.</em></p>
<p id="Par32">First, cells mix of 2000 colonies sorted by flow cytometry from the random chromosomal mutagenesis library after 5/7 days continuous cultivation were cultured, and their β-carotene production was quantified to evaluate the impact of mutagenesis duration on yield. Second, a defined number of colonies with intense pigmentation from plate were selected and subjected to shake-flask fermentation to identify high-yield β-carotene-producing strains.</p>
<p id="Par33">The results of fermentation are shown in Fig. <a href="#Fig4" class="usa-link">4</a>B. Strains obtained after 5 days of continuous culture, the β-carotene production reached 398 mg/L, representing a 13% increase compared to the control strain. For strains obtained for 7 days, β-carotene production reached 438.43 mg/L, a 24% increase over the control strain. These results demonstrate that flow cytometry can effectively sort and enrich strains with high β-carotene production.</p>
<p id="Par34">Fifty visually yellowish colonies were selected from the plates and fermented in shake flasks to measure β-carotene production. The production levels ranged from 214.84 to 447 mg/L, corresponding to 0.60–1.24 times the production of the original YL065 strain (Fig. <a href="#MOESM1" class="usa-link">S1</a>). High-yield strains, including CDA-37, CDA-15, CDA-45, CDA-44, CDA-9, CDA-12, and CDA-14, were selected for repeated fermentation to obtain accurate data on cell growth and β-carotene production.</p>
<p id="Par35">The results are shown in Fig. <a href="#Fig4" class="usa-link">4</a>C. The selected strains exhibited similar cell growth, with OD<sub>600</sub> values ranging from 28 to 34.8, and their β-carotene production ranged from 88 to 125% of the parental strain. Among these, the β-carotene production of CDA-12, CDA-14, and CDA-44 exceeded that of the control strain, reaching 412.23 mg/L, 448.1 mg/L, and 416.95 mg/L, representing increases of 15%, 25%, and 17%, respectively.</p>
<p id="Par36">The CDA-44 and CDA-14 were functionally validated to investigate if Helicase-CDA system could mediate genome-wide C→T mutation, and identify new target for improving β-carotene biosynthesis.</p></section><section id="Sec15"><h3 class="pmc_sec_title">Performance validation of the Helicase-CDA system by whole-genome sequencing</h3>
<p id="Par37">Whole-genome analysis was conducted on CDA-14 and CDA-44, which exhibited significant and moderate improvements in β-carotene production, respectively, using YL065 as the control strain (Excel 1 in supplementary). Compared to the reference strain W29, after excluding the initial mutations introduced in YL065 (i.e., the three strains share identical bases but differ from the reference sequence), base deletions and insertions (indels), 90, 77, and 4 base substitutions were accumulated in CDA-14, CDA-44, and YL065, respectively(Fig. <a href="#Fig5" class="usa-link">5</a> and Excel 2 in supplementary).</p>
<figure class="fig xbox font-sm" id="Fig5"><h4 class="obj_head">Fig. 5.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12366007_12934_2025_2819_Fig5_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/7481/12366007/e4d8647078ff/12934_2025_2819_Fig5_HTML.jpg" loading="lazy" id="MO9" height="618" width="797" alt="Fig. 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Analysis and Validation of Nucleotide Mutations in Screened Microbial Strains <strong>A</strong>: Proportional Distribution of SNP Counts in CDA-14, CDA-44, and YL065; Pre-analysis Filtering of Shared Mutations in Strains CDA-14, CDA-44, and YL065 Relative to the Reference Genome; <strong>B</strong>: Mutation type of SNP in CDA-14, CDA-44, and YL065; <strong>C</strong>: Cell Growth and β-Carotene Production of Strain with Reverse Genetics Validation of YALI1_B16239g; <strong>D</strong>: β-Carotene yield of Strain with Reverse Genetics Validation of YALI1_B16239g; Error bar represents standard deviation of triplicate experiments.</p></figcaption></figure><p id="Par38">Among the substitutions identified in strains CDA-14, CDA-44, and YL065, C: G → T: A transitions accounted for 61, 55, and 0 base changes, representing 68%, 71%, and 0% of their total substitutions, respectively. This pattern aligns with the known catalytic activity of cytidine deaminase (CDA), which specifically promotes C-to-T conversions, confirming the functional efficacy of the Helicase-CDA fusion in <em>Y. lipolytica</em> for inducing genome-wide C-to-T mutations. Notably, C:G → G: C transversions were undetectable in both CDA-14 and CDA-44, while the frequency of T: A → A: T transversions remained similar to that observed in the absence of Helicase-CDA. These results indicate that Helicase-CDA selectively suppresses C: G → G: C transversions without significantly affecting the rate of T: A → A: T transversions (Fig. <a href="#Fig5" class="usa-link">5</a>B).</p></section><section id="Sec16"><h3 class="pmc_sec_title">Analysis of mutations in high β-carotene-producing strain CDA-14</h3>
<p id="Par39">Given that the β-carotene yield of CDA-14 significantly exceeded that of CDA-44, we prioritized mutation analysis in CDA-14. Among all identified SNPs (Table <a href="#Tab2" class="usa-link">2</a>), 27 and 25 mutations were located in upstream and downstream regions, respectively, while 17 were synonymous mutations. Intergenic, intronic, and stop-lost mutations accounted for 2, 1, and 1 instances, respectively, all of which are unlikely to substantially impact strain phenotype. Notably, 15 non-synonymous coding mutations and 3 nonsense mutations were identified, some of which may influence β-carotene production by altering transcriptional/translational efficiency or enzymatic activity. These 18 mutations mapped to 17 genes, categorized as follows: 1 storage function, 2 DNA/RNA-related, 1 flavin-linked, 4 membrane-associated, 1 inorganic ion transport, 2 molecular chaperones, 5 functionally uncharacterized, and 1 involved in metabolic pathways (Table <a href="#Tab3" class="usa-link">3</a>). All the nonsynonymous, upstream region of genes and termination codon base conversions are listed in Table <a href="#MOESM3" class="usa-link">S3</a>, along with their kegg suggested functions (<a href="https://www.kegg.jp/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.kegg.jp/</a>). Given the lipophilic nature of β-carotene, we prioritized functional analysis of four membrane-associated proteins. Bioinformatics annotation identified <em>YALI1_B16239g</em> as a putative MGA2 homolog.</p>
<section class="tw xbox font-sm" id="Tab2"><h4 class="obj_head">Table 2.</h4>
<div class="caption p"><p>Distribution of SNP variant types and their counts in CDA-14</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Mutation Category</th>
<th align="left" colspan="1" rowspan="1">Numbers</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">DOWNSTREAM</td>
<td align="center" colspan="1" rowspan="1">25</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">NON_SYNONYMOUS_CODING</td>
<td align="center" colspan="1" rowspan="1">15</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">stop</td>
<td align="center" colspan="1" rowspan="1">3</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">INTERGENIC</td>
<td align="center" colspan="1" rowspan="1">2</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">intron</td>
<td align="center" colspan="1" rowspan="1">1</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">SYNONYMOUS_CODING</td>
<td align="center" colspan="1" rowspan="1">17</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">UPSTREAM</td>
<td align="center" colspan="1" rowspan="1">27</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><section class="tw xbox font-sm" id="Tab3"><h4 class="obj_head">Table 3.</h4>
<div class="caption p"><p>Gene functional classification and annotations of the NON_SYNONYMOUS_CODING and stop mutation of CDA-14</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Functionl Category</th>
<th align="left" colspan="1" rowspan="1">Base Mutation</th>
<th align="left" colspan="1" rowspan="1">Gene ID</th>
<th align="left" colspan="1" rowspan="1">Related Annotation</th>
</tr></thead>
<tbody>
<tr>
<td align="left" rowspan="2" colspan="1">Storage Function</td>
<td align="left" colspan="1" rowspan="1">Ctc/Atc</td>
<td align="left" colspan="1" rowspan="1">Gene-YALI1_C14738g</td>
<td align="left" colspan="1" rowspan="1">Storage vacuole, lytic vacuole</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Ctc/Atc</td>
<td align="left" colspan="1" rowspan="1">gene-YALI1_C14738g</td>
<td align="left" colspan="1" rowspan="1">Storage vacuole, lytic vacuole</td>
</tr>
<tr>
<td align="left" rowspan="2" colspan="1">DNA and RNA Related</td>
<td align="left" colspan="1" rowspan="1">Gaa/Aaa</td>
<td align="left" colspan="1" rowspan="1">gene-YALI1_C13302g</td>
<td align="left" colspan="1" rowspan="1">DNA replication initiation</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Ctt/Ttt</td>
<td align="left" colspan="1" rowspan="1">gene-YALI1_D00577g</td>
<td align="left" colspan="1" rowspan="1">RNA processing</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">flavin-linked</td>
<td align="left" colspan="1" rowspan="1">Gat/Aat</td>
<td align="left" colspan="1" rowspan="1">gene-YALI1_D34258g</td>
<td align="left" colspan="1" rowspan="1">flavin-linked sulfhydryl oxidase activity</td>
</tr>
<tr>
<td align="left" rowspan="4" colspan="1">Membrane-related</td>
<td align="left" colspan="1" rowspan="1">Gtt/Att</td>
<td align="left" colspan="1" rowspan="1">gene-YALI1_F25309g</td>
<td align="left" colspan="1" rowspan="1">transmembrane transport</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Ttt/Ctt</td>
<td align="left" colspan="1" rowspan="1">gene-YALI1_D16990g</td>
<td align="left" colspan="1" rowspan="1">integral component of membrane</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">cGa/cAa</td>
<td align="left" colspan="1" rowspan="1">gene-YALI1_B16239g</td>
<td align="left" colspan="1" rowspan="1">integral component of membrane</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">cGc/cAc</td>
<td align="left" colspan="1" rowspan="1">gene-YALI1_E27273g</td>
<td align="left" colspan="1" rowspan="1">protein-cysteine S-palmitoyltransferase activity</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Inorganic ion transport</td>
<td align="left" colspan="1" rowspan="1">Ggt/Agt</td>
<td align="left" colspan="1" rowspan="1">gene-YALI1_B28659g</td>
<td align="left" colspan="1" rowspan="1">Inorganic ion transport and metabolism</td>
</tr>
<tr>
<td align="left" rowspan="2" colspan="1">Molecular Chaperone</td>
<td align="left" colspan="1" rowspan="1">Ggc/Agc</td>
<td align="left" colspan="1" rowspan="1">gene-YALI1_D00231g</td>
<td align="left" colspan="1" rowspan="1">Posttranslational modification, protein turnover, chaperones</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Cag/Tag</td>
<td align="left" colspan="1" rowspan="1">gene-YALI1_C29052g</td>
<td align="left" colspan="1" rowspan="1">Posttranslational modification, protein turnover, chaperones</td>
</tr>
<tr>
<td align="left" rowspan="6" colspan="1">Function Unknown</td>
<td align="left" colspan="1" rowspan="1">cGa/cAa</td>
<td align="left" colspan="1" rowspan="1">gene-YALI1_D27547g</td>
<td align="left" colspan="1" rowspan="1">Uncharacterized protein</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">gCa/gTa</td>
<td align="left" colspan="1" rowspan="1">gene-YALI1_F25212g</td>
<td align="left" colspan="1" rowspan="1">Uncharacterized protein</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Tcc/Gcc</td>
<td align="left" colspan="1" rowspan="1">gene-YALI1_D00040g</td>
<td align="left" colspan="1" rowspan="1">Uncharacterized protein</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Cca/Tca</td>
<td align="left" colspan="1" rowspan="1">gene-YALI1_B00711g</td>
<td align="left" colspan="1" rowspan="1">Uncharacterized protein</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Caa/Taa</td>
<td align="left" colspan="1" rowspan="1">gene-YALI1_F25212g</td>
<td align="left" colspan="1" rowspan="1">Uncharacterized protein</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Cag/Tag</td>
<td align="left" colspan="1" rowspan="1">gene-YALI1_D01796g</td>
<td align="left" colspan="1" rowspan="1">Uncharacterized protein</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><p id="Par42">Given the lipophilic nature of β-carotene, we prioritized functional analysis of four membrane-associated proteins. Bioinformatics annotation identified <em>YALI1_B16239g</em> as a putative MGA2 homolog. Sequence alignment revealed weak homology between <em>YALI1_B16239g</em> and <em>S. cerevisiae</em> S288c MGA2, a membrane-anchored transcriptional regulator that coordinates unsaturated fatty acid biosynthesis through dual modulation of OLE1 (encoding Δ9 fatty acid desaturase) transcription and mRNA stability [<a href="#CR32" class="usa-link" aria-describedby="CR32">32</a>, <a href="#CR33" class="usa-link" aria-describedby="CR33">33</a>]. Notably, <em>S. cerevisiae mga2Δ</em> mutants exhibited a 5.5-fold increase in squalene accumulation compared to wild-type strains. Previous studies have demonstrated that MGA2 regulates sterol biosynthesis by activating ERG1 (squalene epoxidase) expression [<a href="#CR34" class="usa-link" aria-describedby="CR34">34</a>], a pathway competing with carotenoid synthesis for shared precursors (IPP and DMAPP). To investigate whether <em>YALI1_B16239g</em> serves as a novel engineering target for carotenoid overproduction, we performed reverse genetic manipulation of this gene to assess its impact on β-carotene yield, with particular focus on its potential role in redirecting terpenoid precursor flux.</p>
<p id="Par43">we overexpressed <em>YALI1_B16239g</em> in parental strain YL065 and high-producing strain CDA-14 and the result stains named YL065-<em>MGA2</em> and CDA-14-<em>MGA2</em>. As shown in Fig. <a href="#Fig5" class="usa-link">5</a>C, overexpression strains YL065-<em>MGA2</em> and CDA-14-<em>MGA2</em> produced 269.14 mg/L and 275.18 mg/L β-carotene, representing 75% and 61% of their respective parental strains’ β-carotene production (Fig. <a href="#Fig5" class="usa-link">5</a>C). Conversely, <em>MGA2</em> knockout in YL065 (YL065-Δ<em>MGA2</em>) reduced total β-carotene production to 85% of wild-type levels, while demonstrating a 27% increase in β-carotene yield per dry cell weight (36.18 mg/g DCW) (Fig. <a href="#Fig5" class="usa-link">5</a>D).</p></section><section id="Sec17"><h3 class="pmc_sec_title">The levels of squalene accumulation and ERG1 transcription in YL065, YL065-ΔMGA2 and CDA-14</h3>
<p id="Par44">To investigate the relationship between ERG1 transcription levels, squalene accumulation and MGA2 alteration, squalene levels were measured in YL065, YL065-ΔMGA2, and CDA-14 strains at 48 and 72 h of fermentation, while ERG1 transcription was analyzed by qRT-PCR at 24 h.</p>
<p id="Par45">The results showed that, regardless of the fermentation time, the squalene production in YL065-ΔMGA2 and CDA-14 strains was higher than that of the YL065 strain. Specifically, the squalene accumulation in CDA-14 at 48 h and 72 h was 5.6% and 8.0% higher, respectively, compared to the control strain. The squalene accumulation in YL065-ΔMGA2 increased by 65.4% and 24.6% at these two time points, respectively(Fig. <a href="#Fig6" class="usa-link">6</a>AB). Squalene accumulation followed the order: YL065 &lt; CDA-14 &lt; YL065-ΔMGA2.</p>
<figure class="fig xbox font-sm" id="Fig6"><h4 class="obj_head">Fig. 6.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12366007_12934_2025_2819_Fig6_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/7481/12366007/5c8a6b66fb96/12934_2025_2819_Fig6_HTML.jpg" loading="lazy" id="MO10" height="222" width="797" alt="Fig. 6"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>The levels of squalene accumulation and ERG1 transcription in YL065, YL065-ΔMGA2 and CDA-14. <strong>A</strong>: Squalene yield after 48 h fermentation; <strong>B</strong>: Squalene yield after 72 h fermentation; <strong>C</strong>: Relative expression levels of ERG1 after 24 h fermentation</p></figcaption></figure><p id="Par46">The qRT-PCR analysis revealed that after 24-h fermentation, ERG1 transcription in CDA-14 and YL065-ΔMGA2 was reduced by 15% and 68%(Fig. <a href="#Fig6" class="usa-link">6</a> C), respectively, relative to YL065, indicating that <em>MGA2</em> deletion represses ERG1 transcription, thus likely enhancing β-carotene synthesis.</p></section><section id="Sec22"><h3 class="pmc_sec_title">Fed-batch fermentation of strains YL065 and CDA-14 for β-carotene production</h3>
<p id="Par47">To validate enhanced β-carotene production by strain CDA-14 under fed-batch fermentation, YL065 and CDA-14 were cultivated in 5-L bioreactors at pH 5.5. Prior to conducting fed-batch fermentation, the <em>LEU</em> and <em>URA3</em> genes were supplemented into the YL065 and CDA-14 strains to eliminate the need for amino acid supplementation during fermentation. YL065 produced 4.26 g/L and 28.44 mg/g DCW of β-carotene within 110 h (Fig. <a href="#Fig7" class="usa-link">7</a>AC). CDA-14 produced 6.15 g/L and 39.28 mg/g DCW of β-carotene within 110 h (Fig. <a href="#Fig7" class="usa-link">7</a>BC), representing 31% and 28% increases over YL065, respectively.</p>
<figure class="fig xbox font-sm" id="Fig7"><h4 class="obj_head">Fig. 7.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12366007_12934_2025_2819_Fig7_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/7481/12366007/3191f857885a/12934_2025_2819_Fig7_HTML.jpg" loading="lazy" id="MO11" height="550" width="797" alt="Fig. 7"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig7/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Fed-batch fermentation results for β-carotene production by strains YL065 and CDA-14. <strong>A</strong>: Strain YL065; <strong>B</strong>: Strain CDA-14; <strong>C</strong>: The β-carotene yield of YL065 and CDA-14; <strong>A</strong>: β-carotene cultures displayed a deepred-orange color after 110 h cultivation. Error bar represents standard deviation of triplicate experiments</p></figcaption></figure></section></section><section id="Sec18"><h2 class="pmc_sec_title">Discussion</h2>
<p id="Par48">Current research on genome-scale mutagenesis technologies in <em>Y. lipolytica</em> remains limited, with existing studies primarily exploiting the yeast’s robust non-homologous end joining (NHEJ) repair mechanism to construct genome-wide mutant libraries through NHEJ-mediated integration [<a href="#CR35" class="usa-link" aria-describedby="CR35">35</a>]. Unlike other mutagenesis approaches, NHEJ-mediated insertional mutagenesis facilitates straightforward library generation while eliminating the need for heterologous transposons or CRISPR systems, and enables rapid mutation tracking. Leveraging this NHEJ-driven insertional mutagenesis platform, researchers have efficiently identified novel targets for enhancing β-carotene biosynthesis and acetic acid tolerance, demonstrating its utility for functional genomics in this industrially relevant yeast. However, a critical technological paradox emerges when applying this strategy to pre-optimized industrial strains. The prevailing paradigm for constructing <em>Y. lipolytica</em> cell factories requires suppression of NHEJ activity (typically through KU70 deletion) to enhance homologous recombination efficiency for precision genome editing. This essential optimization directly conflicts with the fundamental requirement of NHEJ-mediated mutagenesis approaches, creating incompatibility between random insertion library strategies and continuous strain improvement in engineered production platforms.</p>
<p id="Par49">To overcome this technological barrier, we successfully developed a Helicase-Assisted mutation (Helicase-CDA) system, a genome-wide mutagenesis platform based on a helicase-deaminase complex [<a href="#CR12" class="usa-link" aria-describedby="CR12">12</a>]. By functionally fusing the MCM5 helicase of <em>Y. lipolytica</em> with cytidine deaminase (CDA), this innovative system enables the establishment of a controllable, continuous mutagenesis mechanism in engineered strains through a single transformation of a plasmid carrying the MCM5-CDA fusion module. MCM5 represents an optimal fusion subunit within the MCM2-7 helicase complex, attributed to its dual structural roles in replication fork anchoring and gating regulation [<a href="#CR27" class="usa-link" aria-describedby="CR27">27</a>]. The molecular basis for its precise replication fork localization involves two domain-specific mechanisms: (1) electrostatic docking of its C-terminal domain (C5) onto Orc3 of the ORC complex for origin positioning [<a href="#CR36" class="usa-link" aria-describedby="CR36">36</a>] and (2) interlocking of its N-terminal zinc finger domain with adjacent hexamers to stabilize the double hexamer (DH) interface encircling dsDNA. Helicase activation is initiated by ORC-Cdc6 ATP hydrolysis, which triggers closure of the MCM2-MCM5 gating gap to form a topologically closed ring entrapping dsDNA [<a href="#CR37" class="usa-link" aria-describedby="CR37">37</a>]. This structural transition releases Cdt1 and converts the inactive DH into a processive helicase stably anchored at the fork. The spatial precision afforded by MCM5-mediated localization (via ORC interaction and DH stabilization) ensures targeted delivery of fused enzymatic payloads [<a href="#CR37" class="usa-link" aria-describedby="CR37">37</a>]. This mechanistic rationale is functionally validated in <em>S. cerevisiae</em>, where MCM5-AID fusion increased editing efficiency by (2.1 ± 0.4) × 10³-fold over genomic background rates, demonstrating its capacity for replication-coupled enzymatic action [<a href="#CR34" class="usa-link" aria-describedby="CR34">34</a>]. Following a 7-day continuous subculturing regimen (once-daily passages), the system achieves C-to-T site-specific base substitutions across the entire genome via helicase-mediated localized DNA unwinding and deaminase targeting.</p>
<p id="Par50">When applied the Helicase-CDA system to an industrial β-carotene-producing <em>Y. lipolytica</em> strain, a 7-day continuous culture and mutagenesis process successfully yielded the superior mutant CDA-14 with 25% increased β-carotene production. This system, which enables continuous mutagenesis in engineered strains without the need to restore high NHEJ efficiency, provides a novel platform for rapid directed evolution of metabolically engineered strains.</p>
<p id="Par51">Whole-genome resequencing of the CDA-14 strain which screened from library that subjected to 7-day continuous mutagenesis revealed only 96 accumulated SNPs, indicating suboptimal mutagenesis efficiency. We hypothesize this limitation arises from two molecular mechanisms: (1) The absence of codon optimization for the <em>CDA</em> gene in <em>Y. lipolytica</em> likely reduced translational efficiency, resulting in insufficient MCM5-CDA fusion protein expression [<a href="#CR38" class="usa-link" aria-describedby="CR38">38</a>]. (2) Lack of uracil glycosylase inhibitor (UGI) co-expression may permit repair of CDA-induced C-to-U mutations through base excision repair (BER) pathways [<a href="#CR39" class="usa-link" aria-describedby="CR39">39</a>]. Strategic improvements including codon usage adaptation and UGI overexpression could therefore enhance mutation rates. Crucially, mutagenesis intensity requires systematic optimization to balance genetic burden and phenotypic screening efficiency – a principle critical for evolutionary engineering success. The optimal mutation efficiency can be achieved through the molecular optimization of Helicase-CDA system components, including modulating the expression of the Helicase-CDA fusion protein to regulate editing activity, or by precisely controlling the duration of exposure to active mutagenesis phases.</p>
<p id="Par52">The G-to-A mutation at position 1637 of the <em>YALI1_B16239g</em> gene (encoding MGA2) results in a codon change from CGA (arginine) to CAA (glutamine). This mutation not only alters the amino acid but also replaces a high-frequency codon (41.4‰) with a low-frequency codon (0.7‰). In strains with G-to-A mutations or deletion of <em>YALI1_B16239g</em>, ERG1 transcription levels decreased, while squalene accumulation and β-carotene yield increased. We hypothesized that the introduction of this rare codon reduces the expression of the <em>YALI1_B16239g</em>-encoded protein, leading to lower ERG1 activity. This decrease in ERG1 expression reduces the metabolic flux toward sterols and potentially redirects it toward β-carotene. The increase in squalene accumulation due to ERG1 downregulation is partially offset by the diversion of metabolic flux toward β-carotene. As a result, the overall increase in squalene production in this study is only 65% higher than in the wild-type, which is much lower than the 5.5-fold increase observed upon <em>MGA2</em> deletion in <em>S. cerevisiae</em> [<a href="#CR34" class="usa-link" aria-describedby="CR34">34</a>]. Furthermore, squalene accumulation varied significantly between different fermentation time points, with lower accumulation at 72 h compared to 48 h. This suggests that while the <em>MGA2 g</em>ene regulates squalene accumulation, other factors may also influence its levels during fermentation.</p>
<p id="Par53">Although the YL065-ΔMGA2 strain exhibits a 15% reduction in total β-carotene production due to a significant biomass decrease (49% lower OD<sub>600</sub>), it paradoxically achieves a 27% higher β-carotene yield per unit biomass. This yield increase stems from enhanced carbon flux efficiency toward β-carotene biosynthesis. Knocking out MGA2 suppresses ERG1 expression and sterol biosynthesis. While sterol depletion likely limits growth and biomass [<a href="#CR40" class="usa-link" aria-describedby="CR40">40</a>], it simultaneously redirects metabolic flux away from the competing sterol pathway and into β-carotene production, boosting its accumulation per cell mass.</p>
<p id="Par54">Notably, <em>YALI1_B16239g</em> represents a novel target, and there are no reports yet regarding its influence on β-carotene production in <em>Y. lipolytica</em> or any other species. Future research can further explore this hypothesis.</p></section><section id="Sec19"><h2 class="pmc_sec_title">Conclusion</h2>
<p id="Par55">In this study, we developed the Helicase-CDA base editing system to enhance C: G-to-T: A mutation efficiency in <em>Y. lipolytica</em>. Functional validation demonstrated that the Helicase-CDA system significantly increased the occurrence of C: G-to-T: A substitutions compared to the parental strain. Through targeted mutagenesis, screening and whole genome sequencing, we identified a novel genetic target, <em>YALI1_B16239g</em> (a putative membrane protein involved in OLE1 transcriptional regulation), where a specific C-to-T mutation correlated with a modest increase in β-carotene production. This discovery not only validates the utility of the Helicase-CDA system for uncovering functional genetic elements but also reveals a previously uncharacterized regulatory node in carotenoid biosynthesis.</p>
<p id="Par56">CDA-14 exhibited 25% and 31% increases in β-carotene production under shake-flask and 5-L bioreactor conditions, respectively, demonstrating the Helicase-CDA system’s robust efficacy as a high-performance tool for microbial strain engineering. The system’s unique capability to enable continuous mutagenesis without restoring NHEJ pathway activity provides a critical advantage for iterative strain optimization in genetically engineered industrial microbes. The Helicase-CDA system establishes a versatile platform for advancing microbial metabolic engineering and accelerating the development of high-performance cell factories.</p></section><section id="Sec20"><h2 class="pmc_sec_title">Supplementary Information</h2>
<section class="sm xbox font-sm" id="MOESM1"><div class="media p"><div class="caption">
<a href="/articles/instance/12366007/bin/12934_2025_2819_MOESM1_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">12934_2025_2819_MOESM1_ESM.xlsx</a><sup> (136KB, xlsx) </sup><p>Supplementary Material 1. Excel. S1 Overview of Mutation Types. </p>
</div></div></section><section class="sm xbox font-sm" id="MOESM2"><div class="media p"><div class="caption">
<a href="/articles/instance/12366007/bin/12934_2025_2819_MOESM2_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">12934_2025_2819_MOESM2_ESM.xlsx</a><sup> (79.6KB, xlsx) </sup><p>Supplementary Material 2. Excel. S2. Overview of Mutation Types Post-Filtering.</p>
</div></div></section><section class="sm xbox font-sm" id="MOESM3"><div class="media p"><div class="caption">
<a href="/articles/instance/12366007/bin/12934_2025_2819_MOESM3_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link">12934_2025_2819_MOESM3_ESM.docx</a><sup> (1.6MB, docx) </sup><p>Supplementary Material 3. Text S1 Construction of plasmids for the chromosomal integration of heterologous genes. Table S1. Primers used in this study. Table S2. Plasmids used in this study. Table S3. Integration Sites and Corresponding sgRNA Sequences. Table S4. DNA sequence of plasmid pURA-MCM5-CDA, mutant gene <em>YALI1_B16239g</em>, and exogenous inserted genes; Table S5. Key Gene Abbreviations and Identifiers Used in This Study; Fig. S1. Primary Fermentation of 50 strains form Library; Fig. S2. Genome-wide distribution of mutation Sites. </p>
</div></div></section></section><section id="notes1"><h2 class="pmc_sec_title">Author contributions</h2>
<p>Zhenxia Li and Bo Liu: Data curation, Formal analysis, Writing - original draft. Rongtao Lv: Data curation, Methodology. Sun Zhe: Conceptualization, Formal analysis，Writing - review &amp; editing. QingYan Li: Supervision, Writing - review &amp; editing. XueLi Zhang: Supervision, Funding acquisition.</p></section><section id="notes2"><h2 class="pmc_sec_title">Funding</h2>
<p>This research was supported by grants from the National Science Fund for Distinguished Young Scholar (No.32225031), the Basic Science Center Project of the National Natural Science Foundation of China (No. 32488301).</p></section><section id="notes3"><h2 class="pmc_sec_title">Data availability</h2>
<p>No datasets were generated or analysed during the current study.</p></section><section id="notes4"><h2 class="pmc_sec_title">Declarations</h2>
<section id="FPar1"><h3 class="pmc_sec_title">Competing interests</h3>
<p id="Par63">The authors declare no competing interests.</p></section></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="fn1">
<p><strong>Publisher’s Note</strong></p>
<p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p>
</div></div></section><section id="_ci93_" lang="en" class="contrib-info"><h2 class="pmc_sec_title">Contributor Information</h2>
<p>QingYan Li, Email: li_qy@tib.cas.cn.</p>
<p>XueLi Zhang, Email: zhang_xl@tib.cas.cn.</p></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Keasling JD. Manufacturing molecules through metabolic engineering. Science. 2010;330:1355–8.
</cite> [<a href="https://doi.org/10.1126/science.1193990" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21127247/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Keasling%20JD.%20Manufacturing%20molecules%20through%20metabolic%20engineering.%20Science.%202010;330:1355%E2%80%938." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>Abdel-Mawgoud AM, Markham KA, Palmer CM, Liu N, Stephanopoulos G, Alper HS. Metabolic engineering in the host <em>Yarrowia lipolytica</em>. Metab Eng. 2018;50:192–208.
</cite> [<a href="https://doi.org/10.1016/j.ymben.2018.07.016" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30056205/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Abdel-Mawgoud%20AM,%20Markham%20KA,%20Palmer%20CM,%20Liu%20N,%20Stephanopoulos%20G,%20Alper%20HS.%20Metabolic%20engineering%20in%20the%20host%20Yarrowia%20lipolytica.%20Metab%20Eng.%202018;50:192%E2%80%93208." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>Zhong YT, Shang CY, Tao HL, Hou J, Cui ZY, Qi QS. Boosting succinic acid production of at low pH through enhancing product tolerance and glucose metabolism. Microb Cell Fact. 2024;23(1):291.
</cite> [<a href="https://doi.org/10.1186/s12934-024-02565-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11515616/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39443950/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhong%20YT,%20Shang%20CY,%20Tao%20HL,%20Hou%20J,%20Cui%20ZY,%20Qi%20QS.%20Boosting%20succinic%20acid%20production%20of%20at%20low%20pH%20through%20enhancing%20product%20tolerance%20and%20glucose%20metabolism.%20Microb%20Cell%20Fact.%202024;23(1):291." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Gajdos P, Urbanikova V, Vicenova M, Certik M. Enhancing very long chain fatty acids production in <em>Yarrowia lipolytica</em>. Microb Cell Fact. 2022;21:138.
</cite> [<a href="https://doi.org/10.1186/s12934-022-01866-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9275168/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35818073/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gajdos%20P,%20Urbanikova%20V,%20Vicenova%20M,%20Certik%20M.%20Enhancing%20very%20long%20chain%20fatty%20acids%20production%20in%20Yarrowia%20lipolytica.%20Microb%20Cell%20Fact.%202022;21:138." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Messina E, de Souza CP, Cappella C, Barile SN, Scarcia P, Pisano I, Palmieri L, Nicaud JM, Agrimi G. Genetic inactivation of the carnitine/acetyl-carnitine mitochondrial carrier of <em>Yarrowia lipolytica</em> leads to enhanced odd-chain fatty acid production. Microb Cell Fact. 2023;22:128.
</cite> [<a href="https://doi.org/10.1186/s12934-023-02137-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10339547/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37443049/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Messina%20E,%20de%20Souza%20CP,%20Cappella%20C,%20Barile%20SN,%20Scarcia%20P,%20Pisano%20I,%20Palmieri%20L,%20Nicaud%20JM,%20Agrimi%20G.%20Genetic%20inactivation%20of%20the%20carnitine/acetyl-carnitine%20mitochondrial%20carrier%20of%20Yarrowia%20lipolytica%20leads%20to%20enhanced%20odd-chain%20fatty%20acid%20production.%20Microb%20Cell%20Fact.%202023;22:128." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Shang Y, Zhang P, Wei W, Li J, Ye BC. Metabolic engineering for the high-yield production of Polydatin in <em>Yarrowia lipolytica</em>. Bioresour Technol. 2023;381: 129129.
</cite> [<a href="https://doi.org/10.1016/j.biortech.2023.129129" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37146696/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Shang%20Y,%20Zhang%20P,%20Wei%20W,%20Li%20J,%20Ye%20BC.%20Metabolic%20engineering%20for%20the%20high-yield%20production%20of%20Polydatin%20in%20Yarrowia%20lipolytica.%20Bioresour%20Technol.%202023;381:%20129129." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Arnesen JA, Kildegaard KR, Cernuda Pastor M, Jayachandran S, Kristensen M, Borodina I. <em>Yarrowia lipolytica</em> strains engineered for the production of terpenoids. Front Bioeng Biotechnol. 2020;8:945.
</cite> [<a href="https://doi.org/10.3389/fbioe.2020.00945" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7456906/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32923433/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Arnesen%20JA,%20Kildegaard%20KR,%20Cernuda%20Pastor%20M,%20Jayachandran%20S,%20Kristensen%20M,%20Borodina%20I.%20Yarrowia%20lipolytica%20strains%20engineered%20for%20the%20production%20of%20terpenoids.%20Front%20Bioeng%20Biotechnol.%202020;8:945." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>Santos CN, Stephanopoulos G. Combinatorial engineering of microbes for optimizing cellular phenotype. Curr Opin Chem Biol. 2008;12:168–76.
</cite> [<a href="https://doi.org/10.1016/j.cbpa.2008.01.017" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18275860/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Santos%20CN,%20Stephanopoulos%20G.%20Combinatorial%20engineering%20of%20microbes%20for%20optimizing%20cellular%20phenotype.%20Curr%20Opin%20Chem%20Biol.%202008;12:168%E2%80%9376." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>Sandberg TE, Salazar MJ, Weng LL, Palsson BO, Feist AM. The emergence of adaptive laboratory evolution as an efficient tool for biological discovery and industrial biotechnology. Metab Eng. 2019;56:1–16.
</cite> [<a href="https://doi.org/10.1016/j.ymben.2019.08.004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6944292/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31401242/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sandberg%20TE,%20Salazar%20MJ,%20Weng%20LL,%20Palsson%20BO,%20Feist%20AM.%20The%20emergence%20of%20adaptive%20laboratory%20evolution%20as%20an%20efficient%20tool%20for%20biological%20discovery%20and%20industrial%20biotechnology.%20Metab%20Eng.%202019;56:1%E2%80%9316." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>Bao Z, HamediRad M, Xue P, Xiao H, Tasan I, Chao R, Liang J, Zhao H. Genome-scale engineering of <em>Saccharomyces cerevisiae</em> with single-nucleotide precision. Nat Biotechnol. 2018;36:505–8.
</cite> [<a href="https://doi.org/10.1038/nbt.4132" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29734295/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bao%20Z,%20HamediRad%20M,%20Xue%20P,%20Xiao%20H,%20Tasan%20I,%20Chao%20R,%20Liang%20J,%20Zhao%20H.%20Genome-scale%20engineering%20of%20Saccharomyces%20cerevisiae%20with%20single-nucleotide%20precision.%20Nat%20Biotechnol.%202018;36:505%E2%80%938." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Dong C, Schultz JC, Liu W, Lian J, Huang L, Xu Z, Zhao H. Identification of novel metabolic engineering targets for S-adenosyl-L-methionine production in <em>Saccharomyces cerevisiae</em> via genome-scale engineering. Metab Eng. 2021;66:319–27.
</cite> [<a href="https://doi.org/10.1016/j.ymben.2021.03.005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33713797/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Dong%20C,%20Schultz%20JC,%20Liu%20W,%20Lian%20J,%20Huang%20L,%20Xu%20Z,%20Zhao%20H.%20Identification%20of%20novel%20metabolic%20engineering%20targets%20for%20S-adenosyl-L-methionine%20production%20in%20Saccharomyces%20cerevisiae%20via%20genome-scale%20engineering.%20Metab%20Eng.%202021;66:319%E2%80%9327." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Wang J, Zhao D, Li J, Hu M, Xin X, Price MA, Li Q, Liu L, Li S, Rosser SJ, et al. Helicase-AID: a novel molecular device for base editing at random genomic loci. Metab Eng. 2021;67:396–402.
</cite> [<a href="https://doi.org/10.1016/j.ymben.2021.08.005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34411701/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang%20J,%20Zhao%20D,%20Li%20J,%20Hu%20M,%20Xin%20X,%20Price%20MA,%20Li%20Q,%20Liu%20L,%20Li%20S,%20Rosser%20SJ,%20et%20al.%20Helicase-AID:%20a%20novel%20molecular%20device%20for%20base%20editing%20at%20random%20genomic%20loci.%20Metab%20Eng.%202021;67:396%E2%80%93402." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>Pan Y, Xia S, Dong C, Pan H, Cai J, Huang L, Xu Z, Lian J. Random base editing for genome evolution in <em>Saccharomyces cerevisiae</em>. ACS Synth Biol. 2021;10:2440–6.
</cite> [<a href="https://doi.org/10.1021/acssynbio.1c00217" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34542280/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Pan%20Y,%20Xia%20S,%20Dong%20C,%20Pan%20H,%20Cai%20J,%20Huang%20L,%20Xu%20Z,%20Lian%20J.%20Random%20base%20editing%20for%20genome%20evolution%20in%20Saccharomyces%20cerevisiae.%20ACS%20Synth%20Biol.%202021;10:2440%E2%80%936." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>Ju Y, Zhang H, Du X, Wei J, Liu J, Wei L, Liu Q, Xu N. Dragon: harnessing the power of DNA repair for accelerating genome evolution in <em>Corynebacterium glutamicum</em>. Metab Eng. 2023;79:182–91.
</cite> [<a href="https://doi.org/10.1016/j.ymben.2023.08.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37579915/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ju%20Y,%20Zhang%20H,%20Du%20X,%20Wei%20J,%20Liu%20J,%20Wei%20L,%20Liu%20Q,%20Xu%20N.%20Dragon:%20harnessing%20the%20power%20of%20DNA%20repair%20for%20accelerating%20genome%20evolution%20in%20Corynebacterium%20glutamicum.%20Metab%20Eng.%202023;79:182%E2%80%9391." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>Duan Y, Tan Y, Chen X, Pei X, Li M. Modular and flexible molecular device for simultaneous cytosine and adenine base editing at random genomic loci in filamentous fungi. ACS Synth Biol. 2023;12:2147–56.
</cite> [<a href="https://doi.org/10.1021/acssynbio.3c00229" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37428865/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Duan%20Y,%20Tan%20Y,%20Chen%20X,%20Pei%20X,%20Li%20M.%20Modular%20and%20flexible%20molecular%20device%20for%20simultaneous%20cytosine%20and%20adenine%20base%20editing%20at%20random%20genomic%20loci%20in%20filamentous%20fungi.%20ACS%20Synth%20Biol.%202023;12:2147%E2%80%9356." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Engler C, Kandzia R, Marillonnet S. A one pot, one step, precision cloning method with high throughput capability. PLoS One. 2008;3:e3647.
</cite> [<a href="https://doi.org/10.1371/journal.pone.0003647" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2574415/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18985154/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Engler%20C,%20Kandzia%20R,%20Marillonnet%20S.%20A%20one%20pot,%20one%20step,%20precision%20cloning%20method%20with%20high%20throughput%20capability.%20PLoS%20One.%202008;3:e3647." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>Zhang M, Li QY, Fan FY, Lu FP, Zhang XL. Combinatorial metabolic engineering and tolerance evolution of for improved γ-decalactone synthesis. Process Biochem. 2023;132:248–54.</cite> [<a href="https://scholar.google.com/scholar_lookup?Zhang%20M,%20Li%20QY,%20Fan%20FY,%20Lu%20FP,%20Zhang%20XL.%20Combinatorial%20metabolic%20engineering%20and%20tolerance%20evolution%20of%20for%20improved%20%CE%B3-decalactone%20synthesis.%20Process%20Biochem.%202023;132:248%E2%80%9354." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Holkenbrink C, Dam MI, Kildegaard KR, Beder J, Dahlin J, Belda DD, Borodina I. EasyCloneYALI: CRISPR/Cas9-Based Synthetic Toolbox for Engineering of the Yeast. Biotechnol J. 2018;13(9):1700543.</cite> [<a href="https://doi.org/10.1002/biot.201700543" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29377615/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Holkenbrink%20C,%20Dam%20MI,%20Kildegaard%20KR,%20Beder%20J,%20Dahlin%20J,%20Belda%20DD,%20Borodina%20I.%20EasyCloneYALI:%20CRISPR/Cas9-Based%20Synthetic%20Toolbox%20for%20Engineering%20of%20the%20Yeast.%20Biotechnol%20J.%202018;13(9):1700543." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Liu M, Zhang J, Liu X, Hou J, Qi Q. Rapid gene target tracking for enhancing beta-carotene production using flow cytometry-based high-throughput screening in <em>Yarrowia lipolytica</em>. Appl Environ Microbiol. 2022;88: e0114922.
</cite> [<a href="https://doi.org/10.1128/aem.01149-22" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9552598/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36094200/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Liu%20M,%20Zhang%20J,%20Liu%20X,%20Hou%20J,%20Qi%20Q.%20Rapid%20gene%20target%20tracking%20for%20enhancing%20beta-carotene%20production%20using%20flow%20cytometry-based%20high-throughput%20screening%20in%20Yarrowia%20lipolytica.%20Appl%20Environ%20Microbiol.%202022;88:%20e0114922." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>Ma Y, Liu N, Greisen P, Li J, Qiao K, Huang S, Stephanopoulos G. Removal of lycopene substrate inhibition enables high carotenoid productivity in <em>Yarrowia lipolytica</em>. Nat Commun. 2022;13:572.
</cite> [<a href="https://doi.org/10.1038/s41467-022-28277-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8803881/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35102143/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ma%20Y,%20Liu%20N,%20Greisen%20P,%20Li%20J,%20Qiao%20K,%20Huang%20S,%20Stephanopoulos%20G.%20Removal%20of%20lycopene%20substrate%20inhibition%20enables%20high%20carotenoid%20productivity%20in%20Yarrowia%20lipolytica.%20Nat%20Commun.%202022;13:572." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>Fu J, Zaghen S, Lu H, Konzock O, Poorinmohammad N, Kornberg A, Ledesma-Amaro R, Koseto D, Wentzel A, Di Bartolomeo F, Kerkhoven EJ. Reprogramming <em>Yarrowia lipolytica</em> metabolism for efficient synthesis of itaconic acid from flask to semipilot scale. Sci Adv. 2024;10: eadn0414.
</cite> [<a href="https://doi.org/10.1126/sciadv.adn0414" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11313960/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39121230/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Fu%20J,%20Zaghen%20S,%20Lu%20H,%20Konzock%20O,%20Poorinmohammad%20N,%20Kornberg%20A,%20Ledesma-Amaro%20R,%20Koseto%20D,%20Wentzel%20A,%20Di%20Bartolomeo%20F,%20Kerkhoven%20EJ.%20Reprogramming%20Yarrowia%20lipolytica%20metabolism%20for%20efficient%20synthesis%20of%20itaconic%20acid%20from%20flask%20to%20semipilot%20scale.%20Sci%20Adv.%202024;10:%20eadn0414." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>Ning Y, Liu M, Ru Z, Zeng W, Liu S, Zhou J. Efficient synthesis of squalene by cytoplasmic-peroxisomal engineering and regulating lipid metabolism in <em>Yarrowia lipolytica</em>. Bioresour Technol. 2024;395: 130379.
</cite> [<a href="https://doi.org/10.1016/j.biortech.2024.130379" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38281547/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ning%20Y,%20Liu%20M,%20Ru%20Z,%20Zeng%20W,%20Liu%20S,%20Zhou%20J.%20Efficient%20synthesis%20of%20squalene%20by%20cytoplasmic-peroxisomal%20engineering%20and%20regulating%20lipid%20metabolism%20in%20Yarrowia%20lipolytica.%20Bioresour%20Technol.%202024;395:%20130379." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>Mironczuk AM, Biegalska A, Zugaj K, Rzechonek DA, Dobrowolski A. A role of a newly identified isomerase from <em>Yarrowia lipolytica</em> in erythritol catabolism. Front Microbiol. 2018;9: 1122.
</cite> [<a href="https://doi.org/10.3389/fmicb.2018.01122" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5992420/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29910781/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mironczuk%20AM,%20Biegalska%20A,%20Zugaj%20K,%20Rzechonek%20DA,%20Dobrowolski%20A.%20A%20role%20of%20a%20newly%20identified%20isomerase%20from%20Yarrowia%20lipolytica%20in%20erythritol%20catabolism.%20Front%20Microbiol.%202018;9:%201122." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>Zhang M, Gong Z, Tang J, Lu F, Li Q, Zhang X. Improving astaxanthin production in <em>Escherichia coli</em> by co-utilizing CrtZ enzymes with different substrate preference. Microb Cell Fact. 2022;21:71.
</cite> [<a href="https://doi.org/10.1186/s12934-022-01798-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9036794/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35468798/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhang%20M,%20Gong%20Z,%20Tang%20J,%20Lu%20F,%20Li%20Q,%20Zhang%20X.%20Improving%20astaxanthin%20production%20in%20Escherichia%20coli%20by%20co-utilizing%20CrtZ%20enzymes%20with%20different%20substrate%20preference.%20Microb%20Cell%20Fact.%202022;21:71." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>Zhao J, Li Q, Sun T, Zhu X, Xu H, Tang J, Zhang X, Ma Y. Engineering central metabolic modules of <em>Escherichia coli</em> for improving beta-carotene production. Metab Eng. 2013;17:42–50.
</cite> [<a href="https://doi.org/10.1016/j.ymben.2013.02.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23500001/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhao%20J,%20Li%20Q,%20Sun%20T,%20Zhu%20X,%20Xu%20H,%20Tang%20J,%20Zhang%20X,%20Ma%20Y.%20Engineering%20central%20metabolic%20modules%20of%20Escherichia%20coli%20for%20improving%20beta-carotene%20production.%20Metab%20Eng.%202013;17:42%E2%80%9350." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>Nishida K, Arazoe T, Yachie N, Banno S, Kakimoto M, Tabata M, Mochizuki M, Miyabe A, Araki M, Hara KY, et al. Targeted nucleotide editing using hybrid prokaryotic and vertebrate adaptive immune systems. Science. 2016. 10.1126/science.aaf8729.
</cite> [<a href="https://doi.org/10.1126/science.aaf8729" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27492474/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Nishida%20K,%20Arazoe%20T,%20Yachie%20N,%20Banno%20S,%20Kakimoto%20M,%20Tabata%20M,%20Mochizuki%20M,%20Miyabe%20A,%20Araki%20M,%20Hara%20KY,%20et%20al.%20Targeted%20nucleotide%20editing%20using%20hybrid%20prokaryotic%20and%20vertebrate%20adaptive%20immune%20systems.%20Science.%202016.%2010.1126/science.aaf8729." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Vijayraghavan S, Schwacha A. The eukaryotic Mcm2-7 replicative helicase. Subcell Biochem. 2012;62:113–34.
</cite> [<a href="https://doi.org/10.1007/978-94-007-4572-8_7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22918583/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Vijayraghavan%20S,%20Schwacha%20A.%20The%20eukaryotic%20Mcm2-7%20replicative%20helicase.%20Subcell%20Biochem.%202012;62:113%E2%80%9334." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Zhang XK, Wang DN, Chen J, Liu ZJ, Wei LJ, Hua Q. Metabolic engineering of beta-carotene biosynthesis in <em>Yarrowia lipolytica</em>. Biotechnol Lett. 2020;42:945–56.
</cite> [<a href="https://doi.org/10.1007/s10529-020-02844-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32090297/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhang%20XK,%20Wang%20DN,%20Chen%20J,%20Liu%20ZJ,%20Wei%20LJ,%20Hua%20Q.%20Metabolic%20engineering%20of%20beta-carotene%20biosynthesis%20in%20Yarrowia%20lipolytica.%20Biotechnol%20Lett.%202020;42:945%E2%80%9356." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>Li SL, Rong LX, Wang SH, Liu SQ, Lu ZH, Miao L, Zhao BX, Zhang CY, Xiao DG, Pushpanathan K, et al. Enhanced limonene production by metabolically engineered from cheap carbon sources. Chem Eng Sci. 2022;249.</cite>
</li>
<li id="CR30">
<span class="label">30.</span><cite>Matthäus F, Ketelhot M, Gatter M, Barth G. Production of lycopene in the Non-Carotenoid-Producing yeast. Appl Environ Microbiol. 2014;80:1660–9.
</cite> [<a href="https://doi.org/10.1128/AEM.03167-13" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3957620/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24375130/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Matth%C3%A4us%20F,%20Ketelhot%20M,%20Gatter%20M,%20Barth%20G.%20Production%20of%20lycopene%20in%20the%20Non-Carotenoid-Producing%20yeast.%20Appl%20Environ%20Microbiol.%202014;80:1660%E2%80%939." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>Gao S, Tong Y, Zhu L, Ge M, Zhang Y, Chen D, Jiang Y, Yang S. Iterative integration of multiple-copy pathway genes in <em>Yarrowia lipolytica</em> for heterologous beta-carotene production. Metab Eng. 2017;41:192–201.
</cite> [<a href="https://doi.org/10.1016/j.ymben.2017.04.004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28414174/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gao%20S,%20Tong%20Y,%20Zhu%20L,%20Ge%20M,%20Zhang%20Y,%20Chen%20D,%20Jiang%20Y,%20Yang%20S.%20Iterative%20integration%20of%20multiple-copy%20pathway%20genes%20in%20Yarrowia%20lipolytica%20for%20heterologous%20beta-carotene%20production.%20Metab%20Eng.%202017;41:192%E2%80%93201." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>Chellappa R, Kandasamy P, Oh CS, Jiang Y, Vemula M, Martin CE. The membrane proteins, Spt23p and Mga2p, play distinct roles in the activation of <em>Saccharomyces cerevisiae</em> OLE1 gene expression. Fatty acid-mediated regulation of Mga2p activity is independent of its proteolytic processing into a soluble transcription activator. J Biol Chem. 2001;276:43548–56.
</cite> [<a href="https://doi.org/10.1074/jbc.M107845200" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11557770/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chellappa%20R,%20Kandasamy%20P,%20Oh%20CS,%20Jiang%20Y,%20Vemula%20M,%20Martin%20CE.%20The%20membrane%20proteins,%20Spt23p%20and%20Mga2p,%20play%20distinct%20roles%20in%20the%20activation%20of%20Saccharomyces%20cerevisiae%20OLE1%20gene%20expression.%20Fatty%20acid-mediated%20regulation%20of%20Mga2p%20activity%20is%20independent%20of%20its%20proteolytic%20processing%20into%20a%20soluble%20transcription%20activator.%20J%20Biol%20Chem.%202001;276:43548%E2%80%9356." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Zhang S, Skalsky Y, Garfinkel DJ. MGA2 or SPT23 is required for transcription of the delta9 fatty acid desaturase gene, OLE1, and nuclear membrane integrity in <em>Saccharomyces cerevisiae</em>. Genetics. 1999;151:473–83.
</cite> [<a href="https://doi.org/10.1093/genetics/151.2.473" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1460504/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9927444/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhang%20S,%20Skalsky%20Y,%20Garfinkel%20DJ.%20MGA2%20or%20SPT23%20is%20required%20for%20transcription%20of%20the%20delta9%20fatty%20acid%20desaturase%20gene,%20OLE1,%20and%20nuclear%20membrane%20integrity%20in%20Saccharomyces%20cerevisiae.%20Genetics.%201999;151:473%E2%80%9383." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>Rice C, Cooke M, Treloar N, Vollbrecht P, Stukey J, McDonough V. A role for MGA2, but not SPT23, in activation of transcription of ERG1 in <em>Saccharomyces cerevisiae</em>. Biochem Biophys Res Commun. 2010;403:293–7.
</cite> [<a href="https://doi.org/10.1016/j.bbrc.2010.11.020" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21075079/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rice%20C,%20Cooke%20M,%20Treloar%20N,%20Vollbrecht%20P,%20Stukey%20J,%20McDonough%20V.%20A%20role%20for%20MGA2,%20but%20not%20SPT23,%20in%20activation%20of%20transcription%20of%20ERG1%20in%20Saccharomyces%20cerevisiae.%20Biochem%20Biophys%20Res%20Commun.%202010;403:293%E2%80%937." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>Liu X, Liu M, Zhang J, Chang Y, Cui Z, Ji B, Nielsen J, Qi Q, Hou J. Mapping of nonhomologous end joining-mediated integration facilitates genome-scale trackable mutagenesis in <em>Yarrowia lipolytica</em>. ACS Synth Biol. 2022;11:216–27.
</cite> [<a href="https://doi.org/10.1021/acssynbio.1c00390" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34958561/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Liu%20X,%20Liu%20M,%20Zhang%20J,%20Chang%20Y,%20Cui%20Z,%20Ji%20B,%20Nielsen%20J,%20Qi%20Q,%20Hou%20J.%20Mapping%20of%20nonhomologous%20end%20joining-mediated%20integration%20facilitates%20genome-scale%20trackable%20mutagenesis%20in%20Yarrowia%20lipolytica.%20ACS%20Synth%20Biol.%202022;11:216%E2%80%9327." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR36">
<span class="label">36.</span><cite>Ramey CJ, Sclafani RA. Functional conservation of the pre-sensor one beta-finger hairpin (PS1-hp) structures in mini-chromosome maintenance proteins of <em>Saccharomyces cerevisiae</em> and archaea. G3 Genes|Genomes|Genetics. 2014;4:1319–26.
</cite> [<a href="https://doi.org/10.1534/g3.114.011668" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4455780/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24875627/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ramey%20CJ,%20Sclafani%20RA.%20Functional%20conservation%20of%20the%20pre-sensor%20one%20beta-finger%20hairpin%20(PS1-hp)%20structures%20in%20mini-chromosome%20maintenance%20proteins%20of%20Saccharomyces%20cerevisiae%20and%20archaea.%20G3%20Genes%7CGenomes%7CGenetics.%202014;4:1319%E2%80%9326." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR37">
<span class="label">37.</span><cite>Eickhoff P, Kose HB, Martino F, Petojevic T, Abid Ali F, Locke J, Tamberg N, Nans A, Berger JM, Botchan MR, et al. Molecular basis for ATP-Hydrolysis-Driven DNA translocation by the CMG helicase of the eukaryotic replisome. Cell Rep. 2019;28:2673–e26882678.
</cite> [<a href="https://doi.org/10.1016/j.celrep.2019.07.104" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6737378/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31484077/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Eickhoff%20P,%20Kose%20HB,%20Martino%20F,%20Petojevic%20T,%20Abid%20Ali%20F,%20Locke%20J,%20Tamberg%20N,%20Nans%20A,%20Berger%20JM,%20Botchan%20MR,%20et%20al.%20Molecular%20basis%20for%20ATP-Hydrolysis-Driven%20DNA%20translocation%20by%20the%20CMG%20helicase%20of%20the%20eukaryotic%20replisome.%20Cell%20Rep.%202019;28:2673%E2%80%93e26882678." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR38">
<span class="label">38.</span><cite>Gustafsson C, Govindarajan S, Minshull J. Codon bias and heterologous protein expression. Trends Biotechnol. 2004;22:346–53.
</cite> [<a href="https://doi.org/10.1016/j.tibtech.2004.04.006" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15245907/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gustafsson%20C,%20Govindarajan%20S,%20Minshull%20J.%20Codon%20bias%20and%20heterologous%20protein%20expression.%20Trends%20Biotechnol.%202004;22:346%E2%80%9353." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR39">
<span class="label">39.</span><cite>Komor AC, Kim YB, Packer MS, Zuris JA, Liu DR. Programmable editing of a target base in genomic DNA without double-stranded DNA cleavage. Nature. 2016;533:420–4.
</cite> [<a href="https://doi.org/10.1038/nature17946" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4873371/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27096365/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Komor%20AC,%20Kim%20YB,%20Packer%20MS,%20Zuris%20JA,%20Liu%20DR.%20Programmable%20editing%20of%20a%20target%20base%20in%20genomic%20DNA%20without%20double-stranded%20DNA%20cleavage.%20Nature.%202016;533:420%E2%80%934." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR40">
<span class="label">40.</span><cite>Jiang Y, Sun Z, Lu K, Wu Z, Xue H, Zhu L, Li G, Feng Y, Wu M, Lin J, et al. Manipulation of sterol homeostasis for the production of 24-epi-ergosterol in industrial yeast. Nat Commun. 2023;14:437.
</cite> [<a href="https://doi.org/10.1038/s41467-023-36007-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9883489/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36707526/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jiang%20Y,%20Sun%20Z,%20Lu%20K,%20Wu%20Z,%20Xue%20H,%20Zhu%20L,%20Li%20G,%20Feng%20Y,%20Wu%20M,%20Lin%20J,%20et%20al.%20Manipulation%20of%20sterol%20homeostasis%20for%20the%20production%20of%2024-epi-ergosterol%20in%20industrial%20yeast.%20Nat%20Commun.%202023;14:437." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12366007/bin/12934_2025_2819_MOESM1_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">12934_2025_2819_MOESM1_ESM.xlsx</a><sup> (136KB, xlsx) </sup><p>Supplementary Material 1. Excel. S1 Overview of Mutation Types. </p>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12366007/bin/12934_2025_2819_MOESM2_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">12934_2025_2819_MOESM2_ESM.xlsx</a><sup> (79.6KB, xlsx) </sup><p>Supplementary Material 2. Excel. S2. Overview of Mutation Types Post-Filtering.</p>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12366007/bin/12934_2025_2819_MOESM3_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link">12934_2025_2819_MOESM3_ESM.docx</a><sup> (1.6MB, docx) </sup><p>Supplementary Material 3. Text S1 Construction of plasmids for the chromosomal integration of heterologous genes. Table S1. Primers used in this study. Table S2. Plasmids used in this study. Table S3. Integration Sites and Corresponding sgRNA Sequences. Table S4. DNA sequence of plasmid pURA-MCM5-CDA, mutant gene <em>YALI1_B16239g</em>, and exogenous inserted genes; Table S5. Key Gene Abbreviations and Identifiers Used in This Study; Fig. S1. Primary Fermentation of 50 strains form Library; Fig. S2. Genome-wide distribution of mutation Sites. </p>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>No datasets were generated or analysed during the current study.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Microbial Cell Factories are provided here courtesy of <strong>BMC</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1186/s12934-025-02819-5"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/12934_2025_Article_2819.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (3.1 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12366007/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12366007/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12366007%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366007/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12366007/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12366007/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40830463/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12366007/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40830463/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12366007/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12366007/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="dDbhqb2VR5hoQpcCMDUtoL3MiXJ0yFpHeGVLbMBra3COPZGLcNmR8xeQ1Sx344iY">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
